Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-09-08 22:57:54.309708262 -0700
+++ /tmp/foob	2016-09-08 22:57:54.313708267 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc5-next-20160908-141518-g4affa54 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Sep 7 23:07:31 PDT 2016
+Linux version 4.8.0-rc5-next-20160909-141747-gd221eb9 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Sep 8 22:52:56 PDT 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -18,15 +18,15 @@
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
-Memory: 1000060K/1048576K available (7996K kernel code, 820K rwdata, 3792K rodata, 960K init, 274K bss, 32132K reserved, 16384K cma-reserved)
+Memory: 1000060K/1048576K available (7996K kernel code, 822K rwdata, 3792K rodata, 960K init, 272K bss, 32132K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008850000   (  8000 KB)
     .rodata : 0xffff000008850000 - 0xffff000008c10000   (  3840 KB)
       .init : 0xffff000008c10000 - 0xffff000008d00000   (   960 KB)
-      .data : 0xffff000008d00000 - 0xffff000008dcd200   (   821 KB)
-       .bss : 0xffff000008dcd200 - 0xffff000008e11dc0   (   275 KB)
+      .data : 0xffff000008d00000 - 0xffff000008dcda00   (   823 KB)
+       .bss : 0xffff000008dcda00 - 0xffff000008e11dc0   (   273 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -101,7 +101,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.716:1): initialized
+audit: type=2000 audit(0.720:1): initialized
 workingset: timestamp_bits=46 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -166,7 +166,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2016-09-08 06:08:10 UTC (1473314890)
+rtc-pl031 9010000.pl031: setting system clock to 2016-09-09 05:53:33 UTC (1473400413)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-09-08 22:57:54.317708271 -0700
+++ /tmp/foob	2016-09-08 22:57:54.317708271 -0700
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra114 (Dalmore) # 
 Tegra114 (Dalmore) # usb reset
 (Re)start USB...
@@ -38,21 +38,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ################################
-	 3.1 MiB/s
+	 #################################
+	 3 MiB/s
 done
-Bytes transferred = 7148912 (6d1570 hex)
+Bytes transferred = 7157096 (6d3568 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.1 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6d1570 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6d3568 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc5-next-20160908-117085-g4affa54 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Sep 7 23:03:47 PDT 2016
+Linux version 4.8.0-rc5-next-20160909-117314-gd221eb9 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Sep 8 22:49:14 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c1420c00, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1425b00, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1995080K/2096128K available (9216K kernel code, 1162K rwdata, 4028K rodata, 2048K init, 342K bss, 35512K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1995060K/2096128K available (9216K kernel code, 1182K rwdata, 4032K rodata, 2048K init, 342K bss, 35532K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,8 +92,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc14229e0   (1163 kB)
-       .bss : 0xc1424000 - 0xc1479ac8   ( 343 kB)
+      .data : 0xc1300000 - 0xc14278e0   (1183 kB)
+       .bss : 0xc1429000 - 0xc147eac8   ( 343 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -323,7 +323,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-07-02 17:30:57 UTC (962559057)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-07-03 17:16:32 UTC (962644592)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -335,4 +335,4 @@
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.410248] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.415273] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-09-08 22:57:54.321708276 -0700
+++ /tmp/foob	2016-09-08 22:57:54.325708282 -0700
@@ -37,9 +37,9 @@
 	 #################################################################
 	 #################################################################
 	 #############################################
-	 3.6 MiB/s
+	 3.1 MiB/s
 done
-Bytes transferred = 5429400 (52d898 hex)
+Bytes transferred = 5429872 (52da70 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x52d898 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x52da70 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc5-next-20160908-117085-g4affa54 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Sep 7 23:03:35 PDT 2016
+Linux version 4.8.0-rc5-next-20160909-117314-gd221eb9 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Sep 8 22:48:58 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0d5c580, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0d5c700, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999936K/2096128K available (8192K kernel code, 375K rwdata, 2444K rodata, 1024K init, 366K bss, 30656K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1999932K/2096128K available (8192K kernel code, 376K rwdata, 2444K rodata, 1024K init, 366K bss, 30660K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,8 +92,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d5df40   ( 376 kB)
-       .bss : 0xc0d5f000 - 0xc0dbaac0   ( 367 kB)
+      .data : 0xc0d00000 - 0xc0d5e0c0   ( 377 kB)
+       .bss : 0xc0d60000 - 0xc0dbbac0   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -322,7 +322,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-07-02 17:28:15 UTC (962558895)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-07-03 17:13:41 UTC (962644421)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -336,5 +336,5 @@
 Freeing unused kernel memory: 1024K (c0c00000 - c0d00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
-root@(none):/# [    6.715300] asix 1-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0d:b5:16
+root@(none):/# [    6.018879] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+asix 1-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0d:b5:16
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-09-08 22:57:54.329708286 -0700
+++ /tmp/foob	2016-09-08 22:57:54.329708286 -0700
@@ -33,29 +33,29 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ################################
+	 #################################
 	 4 MiB/s
 done
-Bytes transferred = 7148912 (6d1570 hex)
+Bytes transferred = 7157096 (6d3568 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.7 MiB/s
+	 1.4 MiB/s
 done
-Bytes transferred = 66621 (1043d hex)
+Bytes transferred = 66405 (10365 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6d1570 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x6d3568 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
-   Using Device Tree in place at 83000000, end 8301343c
+   Using Device Tree in place at 83000000, end 83013364
 
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc5-next-20160908-117085-g4affa54 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Sep 7 23:03:47 PDT 2016
+Linux version 4.8.0-rc5-next-20160909-117314-gd221eb9 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Sep 8 22:49:14 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -64,7 +64,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c1420c00, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1425b00, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -77,7 +77,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1994960K/2096128K available (9216K kernel code, 1162K rwdata, 4028K rodata, 2048K init, 342K bss, 35632K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1994940K/2096128K available (9216K kernel code, 1182K rwdata, 4032K rodata, 2048K init, 342K bss, 35652K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -87,8 +87,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc14229e0   (1163 kB)
-       .bss : 0xc1424000 - 0xc1479ac8   ( 343 kB)
+      .data : 0xc1300000 - 0xc14278e0   (1183 kB)
+       .bss : 0xc1429000 - 0xc147eac8   ( 343 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -195,13 +195,13 @@
 tegra-pmc 7000e400.pmc: emergency thermal reset enabled
 Serial: 8250/16550 driver, 4 ports, IRQ sharing disabled
 console [ttyS0] disabled
-70006300.serial: ttyS0 at MMIO 0x70006300 (irq = 333, base_baud = 25500000) is a Tegra
+70006300.serial: ttyS0 at MMIO 0x70006300 (irq = 335, base_baud = 25500000) is a Tegra
 console [ttyS0] enabled
 SuperH (H)SCI(F) driver initialized
 msm_serial: driver initialized
 STMicroelectronics ASC driver initialized
-serial-tegra serial@0,70006000: failed to get alias id, errno -19
-serial-tegra serial@0,70006040: failed to get alias id, errno -19
+70006000.serial: ttyTHS1 at MMIO 0x70006000 (irq = 333, base_baud = 0) is a TEGRA_UART
+70006040.serial: ttyTHS2 at MMIO 0x70006040 (irq = 334, base_baud = 0) is a TEGRA_UART
 [drm] Initialized drm 1.1.0 20060810
 tegra-hdmi 54280000.hdmi: failed to get HDMI regulator
 brd: module loaded
@@ -255,7 +255,7 @@
 tegra-ehci: Tegra EHCI driver
 tegra-ehci 7d004000.usb: EHCI Host Controller
 tegra-ehci 7d004000.usb: new USB bus registered, assigned bus number 1
-tegra-ehci 7d004000.usb: irq 352, io mem 0x7d004000
+tegra-ehci 7d004000.usb: irq 354, io mem 0x7d004000
 tegra-ehci 7d004000.usb: USB 2.0 started, EHCI 1.10
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
@@ -269,7 +269,7 @@
 usbcore: registered new interface driver usb-storage
 mousedev: PS/2 mouse device common for all mice
 as3722-rtc as3722-rtc: rtc core: registered as3722-rtc as rtc0
-as3722-rtc as3722-rtc: RTC interrupt 375
+as3722-rtc as3722-rtc: RTC interrupt 377
 tegra_rtc 7000e000.rtc: rtc core: registered 7000e000.rtc as rtc1
 tegra_rtc 7000e000.rtc: Tegra internal Real Time Clock
 i2c /dev entries driver
@@ -297,12 +297,6 @@
 can: netlink gateway (rev 20130117) max_hops=1
 Key type dns_resolver registered
 ThumbEE CPU extension supported.
-mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
- mmcblk0: p1
 Registering SWP/SWPB emulation handler
 +3.3V_RUN: supplied by +3.3V_SYS
 +3.3V_AVDD_HDMI_AP_GATED: supplied by +3.3V_RUN
@@ -318,6 +312,7 @@
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
 tegra-pcie 1003000.pcie-controller: Failed to get supply 'avddio-pex': -517
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
+mmc0: new high speed MMC card at address 0001
 as3722-sd2: supplied by +5V_SYS
 as3722-sd3: supplied by +5V_SYS
 as3722-sd4: supplied by +5V_SYS
@@ -342,11 +337,11 @@
 tegra-ahci 70027000.sata: flags: 64bit ncq sntf pm led pio slum part sadm sds apst 
 scsi host0: tegra-ahci
 scsi host1: tegra-ahci
-ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 343
+ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 345
 ata2: DUMMY
 tegra-ehci 7d008000.usb: EHCI Host Controller
 tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 2
-tegra-ehci 7d008000.usb: irq 353, io mem 0x7d008000
+tegra-ehci 7d008000.usb: irq 355, io mem 0x7d008000
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
@@ -377,7 +372,6 @@
 pci 0000:01:00.0: PME# supported from D0 D3hot D3cold
 PCI: bus1: Fast back to back transfers disabled
 pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
-ata1: SATA link down (SStatus 0 SControl 300)
 pci 0000:02:00.0: [10ec:8168] type 00 class 0x020000
 pci 0000:02:00.0: reg 0x10: [io  0x0000-0x00ff]
 pci 0000:02:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit]
@@ -412,15 +406,22 @@
 pcie_pme 0000:00:02.0:pcie001: service driver pcie_pme loaded
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
-r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0a4d000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 393
+r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0a43000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 395
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
-mmc1: new high speed SDHC card at address aaaa
-mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
+drm drm: No connectors reported connected with modes
+[drm] Cannot find any crtc or sizes - going 1024x768
+ata1: SATA link down (SStatus 0 SControl 300)
+mmc1: new high speed SDHC card at address aaaa
+mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
  mmcblk1: p1 p2 p3
-tegra-hdmi 54280000.hdmi: cannot set audio to 48000 Hz at 297000000 Hz pixel clock
-Console: switching to colour frame buffer device 512x135
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+Console: switching to colour frame buffer device 128x48
+ mmcblk0: p1
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 of: dev_pm_opp_of_cpumask_add_table: couldn't find opp table for cpu:0, -19
@@ -439,7 +440,7 @@
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:1.
+VFS: Mounted root (ext3 filesystem) on device 179:17.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-09-08 22:57:54.337708297 -0700
+++ /tmp/foob	2016-09-08 22:57:54.337708297 -0700
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -34,7 +34,7 @@
 	 #############################################
 	 4 MiB/s
 done
-Bytes transferred = 5429400 (52d898 hex)
+Bytes transferred = 5429872 (52da70 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -43,18 +43,18 @@
 Loading: *#####
 	 4 MiB/s
 done
-Bytes transferred = 66621 (1043d hex)
+Bytes transferred = 66405 (10365 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x52d898 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x52da70 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
-   Using Device Tree in place at 83000000, end 8301343c
+   Using Device Tree in place at 83000000, end 83013364
 
 Starting kernel ...
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc5-next-20160908-117085-g4affa54 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Sep 7 23:03:35 PDT 2016
+Linux version 4.8.0-rc5-next-20160909-117314-gd221eb9 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Sep 8 22:48:58 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -64,7 +64,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0d5c580, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0d5c700, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -77,7 +77,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999816K/2096128K available (8192K kernel code, 375K rwdata, 2444K rodata, 1024K init, 366K bss, 30776K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1999812K/2096128K available (8192K kernel code, 376K rwdata, 2444K rodata, 1024K init, 366K bss, 30780K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -87,8 +87,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d5df40   ( 376 kB)
-       .bss : 0xc0d5f000 - 0xc0dbaac0   ( 367 kB)
+      .data : 0xc0d00000 - 0xc0d5e0c0   ( 377 kB)
+       .bss : 0xc0d60000 - 0xc0dbbac0   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -193,13 +193,13 @@
 tegra-pmc 7000e400.pmc: emergency thermal reset enabled
 Serial: 8250/16550 driver, 4 ports, IRQ sharing disabled
 console [ttyS0] disabled
-70006300.serial: ttyS0 at MMIO 0x70006300 (irq = 333, base_baud = 25500000) is a Tegra
+70006300.serial: ttyS0 at MMIO 0x70006300 (irq = 335, base_baud = 25500000) is a Tegra
 console [ttyS0] enabled
 console [ttyS0] enabled
 bootconsole [earlycon0] disabled
 bootconsole [earlycon0] disabled
-serial-tegra serial@0,70006000: failed to get alias id, errno -19
-serial-tegra serial@0,70006040: failed to get alias id, errno -19
+70006000.serial: ttyTHS1 at MMIO 0x70006000 (irq = 333, base_baud = 0) is a TEGRA_UART
+70006040.serial: ttyTHS2 at MMIO 0x70006040 (irq = 334, base_baud = 0) is a TEGRA_UART
 [drm] Initialized drm 1.1.0 20060810
 tegra-hdmi 54280000.hdmi: failed to get HDMI regulator
 loop: module loaded
@@ -245,7 +245,7 @@
 tegra-ehci: Tegra EHCI driver
 tegra-ehci 7d004000.usb: EHCI Host Controller
 tegra-ehci 7d004000.usb: new USB bus registered, assigned bus number 1
-tegra-ehci 7d004000.usb: irq 352, io mem 0x7d004000
+tegra-ehci 7d004000.usb: irq 354, io mem 0x7d004000
 tegra-ehci 7d004000.usb: USB 2.0 started, EHCI 1.10
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
@@ -254,9 +254,9 @@
 usbcore: registered new interface driver cdc_wdm
 usbcore: registered new interface driver usb-storage
 mousedev: PS/2 mouse device common for all mice
-random: fast init done
 as3722-rtc as3722-rtc: rtc core: registered as3722-rtc as rtc0
-as3722-rtc as3722-rtc: RTC interrupt 375
+as3722-rtc as3722-rtc: RTC interrupt 377
+random: fast init done
 tegra_rtc 7000e000.rtc: rtc core: registered 7000e000.rtc as rtc1
 tegra_rtc 7000e000.rtc: Tegra internal Real Time Clock
 i2c /dev entries driver
@@ -346,11 +346,11 @@
 tegra-ahci 70027000.sata: flags: 64bit ncq sntf pm led pio slum part sadm sds apst 
 scsi host0: tegra-ahci
 scsi host1: tegra-ahci
-ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 343
+ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 345
 ata2: DUMMY
 tegra-ehci 7d008000.usb: EHCI Host Controller
 tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 2
-tegra-ehci 7d008000.usb: irq 353, io mem 0x7d008000
+tegra-ehci 7d008000.usb: irq 355, io mem 0x7d008000
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
@@ -358,7 +358,7 @@
 tegra-xusb 70090000.usb: xHCI Host Controller
 tegra-xusb 70090000.usb: new USB bus registered, assigned bus number 3
 tegra-xusb 70090000.usb: hcc params 0x0184f525 hci version 0x100 quirks 0x00010010
-tegra-xusb 70090000.usb: irq 345, io mem 0x70090000
+tegra-xusb 70090000.usb: irq 347, io mem 0x70090000
 hub 3-0:1.0: USB hub found
 hub 3-0:1.0: 6 ports detected
 tegra-xusb 70090000.usb: xHCI Host Controller
@@ -373,8 +373,8 @@
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
 tegra-pcie 1003000.pcie-controller: probing port 0, using 2 lanes
-ata1: SATA link down (SStatus 0 SControl 300)
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
+ata1: SATA link down (SStatus 0 SControl 300)
 tegra-pcie 1003000.pcie-controller: probing port 1, using 1 lanes
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000000
 tegra-pcie 1003000.pcie-controller: PCI host bridge to bus 0000:00
@@ -411,6 +411,7 @@
 pci 0000:01:00.0: BAR 0: assigned [mem 0x13000000-0x13001fff 64bit]
 pci 0000:00:01.0: PCI bridge to [bus 01]
 pci 0000:00:01.0:   bridge window [mem 0x13000000-0x130fffff]
+ mmcblk1: p1 p2 p3
 pci 0000:02:00.0: BAR 4: assigned [mem 0x20000000-0x20003fff 64bit pref]
 pci 0000:02:00.0: BAR 2: assigned [mem 0x13100000-0x13100fff 64bit]
 pci 0000:02:00.0: BAR 0: assigned [io  0x1000-0x10ff]
@@ -418,7 +419,6 @@
 pci 0000:00:02.0:   bridge window [io  0x1000-0x1fff]
 pci 0000:00:02.0:   bridge window [mem 0x13100000-0x131fffff]
 pci 0000:00:02.0:   bridge window [mem 0x20000000-0x200fffff 64bit pref]
- mmcblk1: p1 p2 p3
 pci 0000:00:01.0: nv_msi_ht_cap_quirk didn't locate host bridge
 pcieport 0000:00:01.0: enabling device (0140 -> 0142)
 pcieport 0000:00:01.0: Signaling PME through PCIe PME interrupt
@@ -431,7 +431,7 @@
 pcie_pme 0000:00:02.0:pcie001: service driver pcie_pme loaded
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
-r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf09f9000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 393
+r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf09f9000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 395
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
@@ -453,7 +453,7 @@
 +2.8V_RUN_CAM_AF: disabling
 +1.8V_RUN_VPP_FUSE: disabling
 ALSA device list:
-  #0: tegra-hda at 0x70038000 irq 344
+  #0: tegra-hda at 0x70038000 irq 346
   #1: NVIDIA Tegra Jetson TK1
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk0p1): recovery complete
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-09-08 22:57:54.341708301 -0700
+++ /tmp/foob	2016-09-08 22:57:54.345708306 -0700
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -40,21 +40,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ################################
-	 3.6 MiB/s
+	 #################################
+	 3.4 MiB/s
 done
-Bytes transferred = 7148912 (6d1570 hex)
+Bytes transferred = 7157096 (6d3568 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.2 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 66821 (10505 hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6d1570 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x6d3568 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013504
@@ -62,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc5-next-20160908-117085-g4affa54 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Sep 7 23:03:47 PDT 2016
+Linux version 4.8.0-rc5-next-20160909-117314-gd221eb9 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Sep 8 22:49:14 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -72,7 +72,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c1420c00, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1425b00, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -85,7 +85,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1995980K/2097148K available (9216K kernel code, 1162K rwdata, 4028K rodata, 2048K init, 342K bss, 35632K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1995960K/2097148K available (9216K kernel code, 1182K rwdata, 4032K rodata, 2048K init, 342K bss, 35652K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -95,8 +95,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc14229e0   (1163 kB)
-       .bss : 0xc1424000 - 0xc1479ac8   ( 343 kB)
+      .data : 0xc1300000 - 0xc14278e0   (1183 kB)
+       .bss : 0xc1429000 - 0xc147eac8   ( 343 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -354,7 +354,7 @@
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:22 UTC (946684822)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:23 UTC (946684823)
 +1.8V_RUN_CAM: disabling
 +3.3V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2016-09-08 22:57:54.349708312 -0700
+++ /tmp/foob	2016-09-08 22:57:54.349708312 -0700
@@ -33,27 +33,26 @@
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'zImage.tegra_defconfig'.
 Load address: 0x82000000
-Loading: *###################EHCI timed out on TD - token=0x88008d80
-T T ##############################################
-	 #################################################################
+Loading: *#################################################################
 	 #################################################################
 	 #################################################################
 	 #################################################################
+	 #######################################################T T ##########
 	 #############################################
-	 392.6 KiB/s
+	 445.3 KiB/s
 done
-Bytes transferred = 5429400 (52d898 hex)
+Bytes transferred = 5429872 (52da70 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.5 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 66821 (10505 hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x52d898 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x52da70 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013504
@@ -168,11 +167,11 @@
 Bluetooth: HIDP (Human Interface Emulation) ver 1.2
 Bluetooth: HIDP socket layer initialized
 Key type dns_resolver registered
-input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input3
 Registering SWP/SWPB emulation handler
 +3.3V_RUN: supplied by +3.3V_SYS
 +3.3V_AVDD_HDMI_AP_GATED: supplied by +3.3V_RUN
 +1.8V_RUN_CAM: supplied by +3.3V_RUN
+input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input3
 +VDDIO_SDMMC3: supplied by +3.3V_RUN
 +1.8V_RUN_VPP_FUSE: supplied by +3.3V_RUN
 +VDD_LED: supplied by +VDD_MUX
@@ -225,7 +224,7 @@
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input4
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:36 UTC (946684836)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:35 UTC (946684835)
 +1.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
 +1.05V_RUN_CAM_REAR: disabling
@@ -247,7 +246,7 @@
 Freeing unused kernel memory: 1024K (c0c00000 - c0d00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    6.444997] asix 1-3:1.0 eth0: register 'asix' at usb-70090000.usb-3, ASIX AX88772 USB 2.0 Ethernet, c8:d7:19:90:fa:65
+root@(none):/# [    6.413576] asix 1-3:1.0 eth0: register 'asix' at usb-70090000.usb-3, ASIX AX88772 USB 2.0 Ethernet, c8:d7:19:90:fa:65
 usb 1-1: new high-speed USB device number 4 using tegra-xusb
 usb-storage 1-1:1.0: USB Mass Storage device detected
 scsi host0: usb-storage 1-1:1.0
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2016-09-08 22:57:54.353708317 -0700
+++ /tmp/foob	2016-09-08 22:57:54.357708321 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1801006 us
+Bootloader-Cpu Init at (time stamp): 1801566 us
 
 Pinmux changes applied in kernel way
 
@@ -62,22 +62,22 @@
 Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1800980  us
-Bootloader (AosInit) time: 734230 us
-Bootloader Main Init time: 180076 us
-BlockDevMgrInit time: 157539 us
+Miscellaneous avp time: 1801540  us
+Bootloader (AosInit) time: 733508 us
+Bootloader Main Init time: 180198 us
+BlockDevMgrInit time: 157546 us
 PartMgrInit time: 8 us
 StorMgrInit time: 31 us
 FsMgrInit time: 2 us
-Partition Table load: 76683 us 
-Display init time: 574681 us
-Kernel read time: 378 msec
-Load Kernel Size:   16375808 Bytes
-Bl Kernel ReadSpeed: 43322000 Bytes/sec
-Pre-Kernel Jump time: 1740475 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5890576 us
+Partition Table load: 76988 us 
+Display init time: 574469 us
+Kernel read time: 379 msec
+Load Kernel Size:   16377856 Bytes
+Bl Kernel ReadSpeed: 43213000 Bytes/sec
+Pre-Kernel Jump time: 1740400 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5847506 us
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc5-next-20160908-141518-g4affa54 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Sep 7 23:07:31 PDT 2016
+Linux version 4.8.0-rc5-next-20160909-141747-gd221eb9 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Sep 8 22:52:56 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
 Memory limited to 1734MB
 efi: Getting EFI parameters from FDT:
@@ -91,15 +91,15 @@
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 Dentry cache hash table entries: 262144 (order: 9, 2097152 bytes)
 Inode-cache hash table entries: 131072 (order: 8, 1048576 bytes)
-Memory: 1706652K/1775616K available (7996K kernel code, 820K rwdata, 3792K rodata, 960K init, 274K bss, 52580K reserved, 16384K cma-reserved)
+Memory: 1706652K/1775616K available (7996K kernel code, 822K rwdata, 3792K rodata, 960K init, 272K bss, 52580K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008850000   (  8000 KB)
     .rodata : 0xffff000008850000 - 0xffff000008c10000   (  3840 KB)
       .init : 0xffff000008c10000 - 0xffff000008d00000   (   960 KB)
-      .data : 0xffff000008d00000 - 0xffff000008dcd200   (   821 KB)
-       .bss : 0xffff000008dcd200 - 0xffff000008e11dc0   (   275 KB)
+      .data : 0xffff000008d00000 - 0xffff000008dcda00   (   823 KB)
+       .bss : 0xffff000008dcda00 - 0xffff000008e11dc0   (   273 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -185,7 +185,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(1.099:1): initialized
+audit: type=2000 audit(1.095:1): initialized
 workingset: timestamp_bits=46 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -247,8 +247,8 @@
 usbhid: USB HID core driver
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-NET: Registered protocol family 17
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+NET: Registered protocol family 17
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
 9pnet: Installing 9P2000 support
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-09-08 22:57:54.361708327 -0700
+++ /tmp/foob	2016-09-08 22:57:54.365708332 -0700
@@ -46,21 +46,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ################################
-	 4 MiB/s
+	 #################################
+	 4.1 MiB/s
 done
-Bytes transferred = 7148912 (6d1570 hex)
+Bytes transferred = 7157096 (6d3568 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x6d1570 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x6d3568 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102b3
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc5-next-20160908-117085-g4affa54 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Sep 7 23:03:47 PDT 2016
+Linux version 4.8.0-rc5-next-20160909-117314-gd221eb9 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Sep 8 22:49:14 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: Compulab TrimSlice board
@@ -76,7 +76,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c1420c00, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c1425b00, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -89,7 +89,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 955904K/1048576K available (9216K kernel code, 1162K rwdata, 4028K rodata, 2048K init, 342K bss, 27136K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 955884K/1048576K available (9216K kernel code, 1182K rwdata, 4032K rodata, 2048K init, 342K bss, 27156K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -99,8 +99,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc14229e0   (1163 kB)
-       .bss : 0xc1424000 - 0xc1479ac8   ( 343 kB)
+      .data : 0xc1300000 - 0xc14278e0   (1183 kB)
+       .bss : 0xc1429000 - 0xc147eac8   ( 343 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -322,29 +322,28 @@
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 mmc0: host does not support reading read-only switch, assuming write-enable
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
 mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
  mmcblk0: p1
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
+mmc1: new high speed SD card at address b368
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
+ mmcblk1: p1 p2
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2015-02-05 17:24:54 UTC (1423157094)
+rtc-em3027 2-0056: setting system clock to 2015-02-05 17:28:23 UTC (1423157303)
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
-mmc1: new high speed SD card at address b368
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
- mmcblk1: p1 p2
-hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
-usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
-asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
-random: fast init done
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
+hub 1-1:1.0: USB hub found
+hub 1-1:1.0: 4 ports detected
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
+random: fast init done
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# 
\ No newline at end of file
+usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
+root@(none):/# [    3.047121] asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-09-08 22:57:54.369708336 -0700
+++ /tmp/foob	2016-09-08 22:57:54.369708336 -0700
@@ -45,9 +45,9 @@
 	 #################################################################
 	 #################################################################
 	 #############################################
-	 4.1 MiB/s
+	 3.9 MiB/s
 done
-Bytes transferred = 5429400 (52d898 hex)
+Bytes transferred = 5429872 (52da70 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x52d898 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x52da70 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102b3
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc5-next-20160908-117085-g4affa54 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Sep 7 23:03:35 PDT 2016
+Linux version 4.8.0-rc5-next-20160909-117314-gd221eb9 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Sep 8 22:48:58 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: Compulab TrimSlice board
@@ -76,7 +76,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0d5c580, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0d5c700, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -89,7 +89,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 960760K/1048576K available (8192K kernel code, 375K rwdata, 2444K rodata, 1024K init, 366K bss, 22280K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 960756K/1048576K available (8192K kernel code, 376K rwdata, 2444K rodata, 1024K init, 366K bss, 22284K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -99,8 +99,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d5df40   ( 376 kB)
-       .bss : 0xc0d5f000 - 0xc0dbaac0   ( 367 kB)
+      .data : 0xc0d00000 - 0xc0d5e0c0   ( 377 kB)
+       .bss : 0xc0d60000 - 0xc0dbbac0   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
@@ -342,7 +342,7 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2015-02-05 17:23:13 UTC (1423156993)
+rtc-em3027 2-0056: setting system clock to 2015-02-05 17:26:28 UTC (1423157188)
 ALSA device list:
   #0: tegra-trimslice
 asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2016-09-08 22:57:54.373708341 -0700
+++ /tmp/foob	2016-09-08 22:57:54.377708347 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 815416 us
-Basic modules init took 349054 us
+Tegraboot started after 767244 us
+Basic modules init took 348467 us
 Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14361 us
-Carveout took 252721 us
-CPU initialization took 92325 us
-Total time taken by TegraBoot 708461 us
+Next stage binary read took 14382 us
+Carveout took 252842 us
+CPU initialization took 92322 us
+Total time taken by TegraBoot 708013 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.653] LPDDR4 Training: Number of tables = 10
+64b[0001.605] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,7 +117,7 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-13947392 bytes read in 774 ms (17.2 MiB/s)
+13949440 bytes read in 771 ms (17.3 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
 42034 bytes read in 453 ms (89.8 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc5-next-20160908-141518-g4affa54 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Sep 7 23:07:31 PDT 2016
+Linux version 4.8.0-rc5-next-20160909-141747-gd221eb9 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Sep 8 22:52:56 PDT 2016
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -166,15 +166,15 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffff80007a000000-ffff80007dffffff]
-Memory: 2979008K/3131392K available (7996K kernel code, 820K rwdata, 3792K rodata, 960K init, 274K bss, 136000K reserved, 16384K cma-reserved)
+Memory: 2979008K/3131392K available (7996K kernel code, 822K rwdata, 3792K rodata, 960K init, 272K bss, 136000K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008850000   (  8000 KB)
     .rodata : 0xffff000008850000 - 0xffff000008c10000   (  3840 KB)
       .init : 0xffff000008c10000 - 0xffff000008d00000   (   960 KB)
-      .data : 0xffff000008d00000 - 0xffff000008dcd200   (   821 KB)
-       .bss : 0xffff000008dcd200 - 0xffff000008e11dc0   (   275 KB)
+      .data : 0xffff000008d00000 - 0xffff000008dcda00   (   823 KB)
+       .bss : 0xffff000008dcda00 - 0xffff000008e11dc0   (   273 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -267,7 +267,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.463:1): initialized
+audit: type=2000 audit(0.475:1): initialized
 workingset: timestamp_bits=46 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -346,6 +346,6 @@
 devtmpfs: mounted
 Freeing unused kernel memory: 960K (ffff800000c10000 - ffff800000d00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
-bash: no job contr[    1.343419] random: fast init done
+bash: no job contr[    1.334499] random: fast init done
 ol in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2016-09-08 22:57:54.381708351 -0700
+++ /tmp/foob	2016-09-08 22:57:54.381708351 -0700
@@ -1,4 +1,38 @@
-B maximum)
+ootconsole [uart0] enabled
+efi: Getting EFI parameters from FDT:
+efi: UEFI not found.
+cma: Reserved 16 MiB at 0x00000000f5400000
+psci: probing for conduit method from DT.
+psci: PSCIv1.0 detected in firmware.
+psci: Using standard PSCI v0.2 function IDs
+psci: MIGRATE_INFO_UP_CPU reported invalid physical ID (0x80000000)
+percpu: Embedded 21 pages/cpu @ffff8000bff3e000 s47488 r8192 d30336 u86016
+Detected PIPT I-cache on CPU0
+CPU features: enabling workaround for ARM erratum 832075
+CPU features: enabling workaround for ARM erratum 834220
+Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 735336
+Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
+log_buf_len individual max cpu contribution: 4096 bytes
+log_buf_len total cpu_extra contributions: 12288 bytes
+log_buf_len min size: 16384 bytes
+log_buf_len: 32768 bytes
+early log buf free: 14408(87%)
+PID hash table entries: 4096 (order: 3, 32768 bytes)
+Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
+Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
+software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
+Memory: 2833196K/2988032K available (7996K kernel code, 822K rwdata, 3792K rodata, 960K init, 272K bss, 138452K reserved, 16384K cma-reserved)
+Virtual kernel memory layout:
+    modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
+    vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
+      .text : 0xffff000008080000 - 0xffff000008850000   (  8000 KB)
+    .rodata : 0xffff000008850000 - 0xffff000008c10000   (  3840 KB)
+      .init : 0xffff000008c10000 - 0xffff000008d00000   (   960 KB)
+      .data : 0xffff000008d00000 - 0xffff000008dcda00   (   823 KB)
+       .bss : 0xffff000008dcda00 - 0xffff000008e11dc0   (   273 KB)
+    fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
+    PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
+    vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
               0xffff7e0000000000 - 0xffff7e0003000000   (    48 MB actual)
     memory  : 0xffff800000000000 - 0xffff8000c0000000   (  3072 MB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
@@ -24,7 +58,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x8868: expected 0x2004f400
+boot misc3 0x88c0: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -100,7 +134,7 @@
 RPC: Registered tcp transport module.
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 Unpacking initramfs...
-Freeing initrd memory: 2364K (ffff800009ff6000 - ffff80000a245000)
+Freeing initrd memory: 2360K (ffff800009ff7000 - ffff80000a245000)
 kvm [1]: IDMAP page: 8083d000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: 8-bit VMID
@@ -110,7 +144,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.607:1): initialized
+audit: type=2000 audit(0.659:1): initialized
 workingset: timestamp_bits=46 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -142,7 +176,6 @@
 PP1100_SOC_RTC: Bringing 1000000uV into 850000-850000uV
 PP2800L_FCAM: Bringing 3100000uV into 2800000-2800000uV
 PP_LDO6: Bringing 2800000uV into 1800000-1800000uV
-random: fast init done
 libphy: Fixed MDIO Bus: probed
 tun: Universal TUN/TAP device driver, 1.6
 tun: (C) 1999-2004 Max Krasnyansky <maxk@qualcomm.com>
@@ -177,6 +210,7 @@
 hub 2-0:1.0: 4 ports detected
 usbcore: registered new interface driver usb-storage
 mousedev: PS/2 mouse device common for all mice
+random: fast init done
 max77686-rtc max77620-rtc: rtc core: registered max77620-rtc as rtc0
 tegra_rtc 7000e000.rtc: rtc core: registered 7000e000.rtc as rtc1
 tegra_rtc 7000e000.rtc: Tegra internal Real Time Clock
@@ -203,8 +237,9 @@
 mmcblk0boot0: mmc0:0001 BGND3R partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB
+max77686-rtc max77620-rtc: setting system clock to 2000-06-20 11:22:43 UTC (961500163)
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
-max77686-rtc max77620-rtc: setting system clock to 2000-06-19 11:37:20 UTC (961414640)
+tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 PP1200_AVDD: disabling
 PP1200_RCAM: disabling
 PP2800L_RCAM: disabling
@@ -215,7 +250,6 @@
 PP1800_CAM: disabling
 ALSA device list:
   No soundcards found.
-tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 Freeing unused kernel memory: 960K (ffff800000c10000 - ffff800000d00000)
 
 System initialization...
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-09-08 22:57:54.389708362 -0700
+++ /tmp/foob	2016-09-08 22:57:54.389708362 -0700
@@ -38,21 +38,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ################################
-	 3.6 MiB/s
+	 #################################
+	 3.1 MiB/s
 done
-Bytes transferred = 7148912 (6d1570 hex)
+Bytes transferred = 7157096 (6d3568 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 51572 (c974 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6d1570 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6d3568 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff973 ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc5-next-20160908-117085-g4affa54 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Sep 7 23:03:47 PDT 2016
+Linux version 4.8.0-rc5-next-20160909-117314-gd221eb9 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Sep 8 22:49:14 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,7 +68,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c1420c00, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1425b00, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -81,7 +81,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1995000K/2096128K available (9216K kernel code, 1162K rwdata, 4028K rodata, 2048K init, 342K bss, 35592K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1994980K/2096128K available (9216K kernel code, 1182K rwdata, 4032K rodata, 2048K init, 342K bss, 35612K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -91,8 +91,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc14229e0   (1163 kB)
-       .bss : 0xc1424000 - 0xc1479ac8   ( 343 kB)
+      .data : 0xc1300000 - 0xc14278e0   (1183 kB)
+       .bss : 0xc1429000 - 0xc147eac8   ( 343 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -299,18 +299,18 @@
 sys_3v3,vdd_3v3_alw: supplied by vdd_5v_in
 +VDD_5V_HDMI: supplied by sys_3v3,vdd_3v3_alw
 sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
-mmc1: new high speed MMC card at address 0001
 tegra-pcie 3000.pcie-controller: 2x3 configuration
-mmcblk1: mmc1:0001 SEM16G 14.8 GiB 
 tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
+mmc1: new high speed MMC card at address 0001
+mmcblk1: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
-mmc0: new ultra high speed DDR50 SDHC card at address aaaa
 mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
-mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
 mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB
 Primary GPT is invalid, using alternate GPT.
  mmcblk1: p1 p2 p3 p4 p5
 tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
+mmc0: new ultra high speed DDR50 SDHC card at address aaaa
+mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
  mmcblk0: p1 p2 p3
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -382,4 +382,4 @@
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.863975] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.868584] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-09-08 22:57:54.393708366 -0700
+++ /tmp/foob	2016-09-08 22:57:54.397708371 -0700
@@ -39,18 +39,18 @@
 	 #############################################
 	 2.9 MiB/s
 done
-Bytes transferred = 5429400 (52d898 hex)
+Bytes transferred = 5429872 (52da70 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 51572 (c974 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x52d898 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x52da70 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff973 ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc5-next-20160908-117085-g4affa54 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Sep 7 23:03:35 PDT 2016
+Linux version 4.8.0-rc5-next-20160909-117314-gd221eb9 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Sep 8 22:48:58 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,7 +68,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0d5c580, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0d5c700, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -81,7 +81,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999856K/2096128K available (8192K kernel code, 375K rwdata, 2444K rodata, 1024K init, 366K bss, 30736K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1999852K/2096128K available (8192K kernel code, 376K rwdata, 2444K rodata, 1024K init, 366K bss, 30740K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -91,8 +91,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d5df40   ( 376 kB)
-       .bss : 0xc0d5f000 - 0xc0dbaac0   ( 367 kB)
+      .data : 0xc0d00000 - 0xc0d5e0c0   ( 377 kB)
+       .bss : 0xc0d60000 - 0xc0dbbac0   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -383,5 +383,4 @@
 Freeing unused kernel memory: 1024K (c0c00000 - c0d00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
-root@(none):/# [    8.512516] asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0f:67:60
+root@(none):/# [    7.730977] usb 2-1: new high-speed USB device number 2 using tegra-ehci
