Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2017-04-10 23:04:20.704944953 -0700
+++ /tmp/foob	2017-04-10 23:04:20.708944970 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc6-next-20170410-283965-gf8c97bd (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Apr 10 01:24:41 PDT 2017
+Linux version 4.11.0-rc6-next-20170411-284285-gc4e7b35 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Apr 10 23:01:41 PDT 2017
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -29,15 +29,15 @@
 Policy zone: DMA
 Kernel command line: root=/dev/vda2 rw rootwait mem=1024M console=ttyAMA0,38400n8 init=/bin/bash
 PID hash table entries: 4096 (order: 3, 32768 bytes)
-Memory: 999816K/1048576K available (8828K kernel code, 1080K rwdata, 4220K rodata, 1088K init, 396K bss, 32376K reserved, 16384K cma-reserved)
+Memory: 999816K/1048576K available (8828K kernel code, 1082K rwdata, 4224K rodata, 1088K init, 397K bss, 32376K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008920000   (  8832 KB)
     .rodata : 0xffff000008920000 - 0xffff000008d50000   (  4288 KB)
       .init : 0xffff000008d50000 - 0xffff000008e60000   (  1088 KB)
-      .data : 0xffff000008e60000 - 0xffff000008f6e200   (  1081 KB)
-       .bss : 0xffff000008f6e200 - 0xffff000008fd1438   (   397 KB)
+      .data : 0xffff000008e60000 - 0xffff000008f6ea00   (  1083 KB)
+       .bss : 0xffff000008f6ea00 - 0xffff000008fd1f38   (   398 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -178,7 +178,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2017-04-10 08:25:25 UTC (1491812725)
+rtc-pl031 9010000.pl031: setting system clock to 2017-04-11 06:02:28 UTC (1491890548)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-04-10 23:04:20.728945052 -0700
+++ /tmp/foob	2017-04-10 23:04:20.728945052 -0700
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra114 (Dalmore) # 
 Tegra114 (Dalmore) # usb reset
 (Re)start USB...
@@ -39,21 +39,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##################
-	 3.2 MiB/s
+	 #################
+	 3.3 MiB/s
 done
-Bytes transferred = 7883456 (784ac0 hex)
+Bytes transferred = 7882656 (7847a0 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.8 MiB/s
+	 2.6 MiB/s
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x784ac0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x7847a0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc6-next-20170410-165724-gf8c97bd (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Apr 10 01:21:22 PDT 2017
+Linux version 4.11.0-rc6-next-20170411-166044-gc4e7b35 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Apr 10 22:58:00 PDT 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -72,7 +72,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c157e380, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c157e880, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -85,7 +85,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993552K/2096128K available (10240K kernel code, 1559K rwdata, 4376K rodata, 2048K init, 407K bss, 37040K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993548K/2096128K available (10240K kernel code, 1561K rwdata, 4380K rodata, 2048K init, 407K bss, 37044K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -95,8 +95,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1585f28   (1560 kB)
-       .bss : 0xc158ed1c - 0xc15f4a08   ( 408 kB)
+      .data : 0xc1400000 - 0xc1586428   (1562 kB)
+       .bss : 0xc158fd1c - 0xc15f5a08   ( 408 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU debugfs-based tracing is enabled.
@@ -313,18 +313,18 @@
  mmcblk0: p1 p2 p3
 tegra-ehci 7d008000.usb: EHCI Host Controller
 tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 1
+tegra-ehci 7d008000.usb: irq 339, io mem 0x7d008000
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
  mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
-tegra-ehci 7d008000.usb: irq 339, io mem 0x7d008000
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2001-02-01 19:57:26 UTC (981057446)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2001-02-02 17:35:07 UTC (981135307)
 vdd_hdmi_5v0: disabling
 dvdd_ts: disabling
 usb1_vbus: disabling
@@ -338,4 +338,4 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.429543] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.419573] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2017-04-10 23:04:20.736945085 -0700
+++ /tmp/foob	2017-04-10 23:04:20.736945085 -0700
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra114 (Dalmore) # 
 Tegra114 (Dalmore) # usb reset
 (Re)start USB...
@@ -38,20 +38,20 @@
 	 #################################################################
 	 #################################################################
 	 #######################
-	 3 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 6062232 (5c8098 hex)
+Bytes transferred = 6062040 (5c7fd8 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.6 MiB/s
+	 2.4 MiB/s
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5c8098 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5c7fd8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -60,7 +60,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc6-next-20170410-165724-gf8c97bd (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Apr 10 01:20:13 PDT 2017
+Linux version 4.11.0-rc6-next-20170411-166044-gc4e7b35 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Apr 10 22:57:42 PDT 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -70,7 +70,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0eac840, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0ead6c0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -83,7 +83,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998572K/2096128K available (9216K kernel code, 731K rwdata, 2684K rodata, 1024K init, 304K bss, 32020K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998564K/2096128K available (9216K kernel code, 734K rwdata, 2688K rodata, 1024K init, 307K bss, 32028K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -93,8 +93,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0eb6d70   ( 732 kB)
-       .bss : 0xc0ec06dc - 0xc0f0c8b4   ( 305 kB)
+      .data : 0xc0e00000 - 0xc0eb7bf0   ( 735 kB)
+       .bss : 0xc0ec16dc - 0xc0f0e3b4   ( 308 kB)
 Preemptible hierarchical RCU implementation.
 NR_IRQS:16 nr_irqs:16 16
 /interrupt-controller@60004000: 160 interrupts forwarded to /interrupt-controller@50041000
@@ -275,15 +275,15 @@
 mmc1: SDHCI controller on 78000600.sdhci [78000600.sdhci] using ADMA
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
-mmc0: new high speed SDHC card at address aaaa
-mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
- mmcblk0: p1 p2 p3
+mmc0: new high speed SDHC card at address aaaa
+mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
  mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
+ mmcblk0: p1 p2 p3
 tegra30-i2s 70080400.i2s: DMA channels sourced from device 70080000.ahub
 tegra-snd-rt5640 sound: rt5640-aif1 <-> 70080400.i2s mapping ok
 input: NVIDIA Tegra Dalmore Headphones as /devices/soc0/sound/sound/card0/input0
@@ -316,7 +316,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2001-02-01 19:54:51 UTC (981057291)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2001-02-02 17:32:22 UTC (981135142)
 vdd_hdmi_5v0: disabling
 dvdd_ts: disabling
 usb1_vbus: disabling
@@ -327,9 +327,9 @@
 EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk1p1): recovery complete
 EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:17.
+VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.657809] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.647583] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-04-10 23:04:20.744945119 -0700
+++ /tmp/foob	2017-04-10 23:04:20.744945119 -0700
@@ -34,10 +34,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##################
+	 #################
 	 3.9 MiB/s
 done
-Bytes transferred = 7883456 (784ac0 hex)
+Bytes transferred = 7882656 (7847a0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -48,7 +48,7 @@
 done
 Bytes transferred = 67133 (1063d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x784ac0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x7847a0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301363c
@@ -56,7 +56,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc6-next-20170410-165724-gf8c97bd (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Apr 10 01:21:22 PDT 2017
+Linux version 4.11.0-rc6-next-20170411-166044-gc4e7b35 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Apr 10 22:58:00 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -67,7 +67,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c157e380, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c157e880, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -80,7 +80,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993432K/2096128K available (10240K kernel code, 1559K rwdata, 4376K rodata, 2048K init, 407K bss, 37160K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993428K/2096128K available (10240K kernel code, 1561K rwdata, 4380K rodata, 2048K init, 407K bss, 37164K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,8 +90,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1585f28   (1560 kB)
-       .bss : 0xc158ed1c - 0xc15f4a08   ( 408 kB)
+      .data : 0xc1400000 - 0xc1586428   (1562 kB)
+       .bss : 0xc158fd1c - 0xc15f5a08   ( 408 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU debugfs-based tracing is enabled.
@@ -380,9 +380,9 @@
 pci 0000:00:02.0: [10de:0e13] type 01 class 0x060400
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x88 may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x52 may corrupt adjacent RW1C bits
+ata1: SATA link down (SStatus 0 SControl 300)
 pci 0000:00:02.0: PME# supported from D0 D1 D2 D3hot D3cold
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4c may corrupt adjacent RW1C bits
-ata1: SATA link down (SStatus 0 SControl 300)
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x3e may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x3e may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x4 may corrupt adjacent RW1C bits
@@ -435,7 +435,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:02 UTC (946684922)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:01:37 UTC (946684897)
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +USB0_VBUS_SW: disabling
 +1.05V_RUN_AVDD_HDMI_PLL: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2017-04-10 23:04:20.752945151 -0700
+++ /tmp/foob	2017-04-10 23:04:20.752945151 -0700
@@ -35,7 +35,7 @@
 	 #######################
 	 4 MiB/s
 done
-Bytes transferred = 6062232 (5c8098 hex)
+Bytes transferred = 6062040 (5c7fd8 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -46,7 +46,7 @@
 done
 Bytes transferred = 67133 (1063d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5c8098 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5c7fd8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301363c
@@ -55,7 +55,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc6-next-20170410-165724-gf8c97bd (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Apr 10 01:20:13 PDT 2017
+Linux version 4.11.0-rc6-next-20170411-166044-gc4e7b35 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Apr 10 22:57:42 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -65,7 +65,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0eac840, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0ead6c0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -78,7 +78,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998452K/2096128K available (9216K kernel code, 731K rwdata, 2684K rodata, 1024K init, 304K bss, 32140K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998444K/2096128K available (9216K kernel code, 734K rwdata, 2688K rodata, 1024K init, 307K bss, 32148K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,8 +88,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0eb6d70   ( 732 kB)
-       .bss : 0xc0ec06dc - 0xc0f0c8b4   ( 305 kB)
+      .data : 0xc0e00000 - 0xc0eb7bf0   ( 735 kB)
+       .bss : 0xc0ec16dc - 0xc0f0e3b4   ( 308 kB)
 Preemptible hierarchical RCU implementation.
 NR_IRQS:16 nr_irqs:16 16
 /interrupt-controller@60004000: 160 interrupts forwarded to /interrupt-controller@50041000
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-04-10 23:04:20.760945184 -0700
+++ /tmp/foob	2017-04-10 23:04:20.760945184 -0700
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -42,21 +42,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##################
-	 3.5 MiB/s
+	 #################
+	 3.6 MiB/s
 done
-Bytes transferred = 7883456 (784ac0 hex)
+Bytes transferred = 7882656 (7847a0 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.2 MiB/s
+	 266.6 KiB/s
 done
 Bytes transferred = 68155 (10a3b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x784ac0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x7847a0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a3a
@@ -64,7 +64,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc6-next-20170410-165724-gf8c97bd (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Apr 10 01:21:22 PDT 2017
+Linux version 4.11.0-rc6-next-20170411-166044-gc4e7b35 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Apr 10 22:58:00 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -76,7 +76,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c157e380, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c157e880, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -89,7 +89,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1994448K/2097148K available (10240K kernel code, 1559K rwdata, 4376K rodata, 2048K init, 407K bss, 37164K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1994444K/2097148K available (10240K kernel code, 1561K rwdata, 4380K rodata, 2048K init, 407K bss, 37168K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -99,8 +99,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1585f28   (1560 kB)
-       .bss : 0xc158ed1c - 0xc15f4a08   ( 408 kB)
+      .data : 0xc1400000 - 0xc1586428   (1562 kB)
+       .bss : 0xc158fd1c - 0xc15f5a08   ( 408 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU debugfs-based tracing is enabled.
@@ -350,7 +350,7 @@
 mmc1: new high speed SDIO card at address 0001
 mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:23 UTC (946684823)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:24 UTC (946684824)
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +1.05V_RUN: disabling
 +1.8V_RUN_CAM: disabling
@@ -372,10 +372,10 @@
 EXT4-fs (mmcblk0p3): recovery complete
 EXT4-fs (mmcblk0p3): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:3.
-devtmpfs: mounted
-Freeing unused kernel memory: 2048K
 mmc2: new high speed SDHC card at address 0007
 mmcblk2: mmc2:0007 SD08G 7.42 GiB 
+devtmpfs: mounted
+Freeing unused kernel memory: 2048K
 GPT:Primary header alternate_lba != Alt. header my_lba
 GPT:3014591 != 15564799
 GPT:Primary header thinks Alt. header is not at the end of the disk.
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2017-04-10 23:04:20.768945218 -0700
+++ /tmp/foob	2017-04-10 23:04:20.772945233 -0700
@@ -41,9 +41,9 @@
 	 #################################################################
 	 #################################################################
 	 #######################
-	 3.6 MiB/s
+	 3.5 MiB/s
 done
-Bytes transferred = 6062232 (5c8098 hex)
+Bytes transferred = 6062040 (5c7fd8 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -54,7 +54,7 @@
 done
 Bytes transferred = 68155 (10a3b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5c8098 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5c7fd8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a3a
@@ -63,7 +63,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc6-next-20170410-165724-gf8c97bd (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Apr 10 01:20:13 PDT 2017
+Linux version 4.11.0-rc6-next-20170411-166044-gc4e7b35 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Apr 10 22:57:42 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -74,7 +74,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c0eac840, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0ead6c0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -87,7 +87,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999468K/2097148K available (9216K kernel code, 731K rwdata, 2684K rodata, 1024K init, 304K bss, 32144K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1999460K/2097148K available (9216K kernel code, 734K rwdata, 2688K rodata, 1024K init, 307K bss, 32152K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -97,8 +97,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0eb6d70   ( 732 kB)
-       .bss : 0xc0ec06dc - 0xc0f0c8b4   ( 305 kB)
+      .data : 0xc0e00000 - 0xc0eb7bf0   ( 735 kB)
+       .bss : 0xc0ec16dc - 0xc0f0e3b4   ( 308 kB)
 Preemptible hierarchical RCU implementation.
 NR_IRQS:16 nr_irqs:16 16
 /interrupt-controller@60004000: 160 interrupts forwarded to /interrupt-controller@50041000
@@ -293,11 +293,11 @@
 input: GoogleNyanBig Headphones as /devices/soc0/sound/sound/card1/input1
 input: GoogleNyanBig Mic Jack as /devices/soc0/sound/sound/card1/input2
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
-mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 NET: Registered protocol family 10
 Segment Routing with IPv6
 mip6: Mobile IPv6
+mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
 NET: Registered protocol family 17
 NET: Registered protocol family 15
@@ -370,24 +370,24 @@
 +5V_HDMI_CON: disabling
 ALSA device list:
   #1: GoogleNyanBig
-usb 1-2: new high-speed USB device number 2 using tegra-xusb
-input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input4
 EXT4-fs (mmcblk0p3): couldn't mount as ext3 due to feature incompatibilities
+input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input4
+usb 1-2: new high-speed USB device number 2 using tegra-xusb
 mmc2: new high speed SDHC card at address 0007
 mmcblk2: mmc2:0007 SD08G 7.42 GiB 
 GPT:Primary header alternate_lba != Alt. header my_lba
 GPT:3014591 != 15564799
 GPT:Primary header thinks Alt. header is not at the end of the disk.
 GPT:3014591 != 15564799
+GPT: Use GNU Parted to correct GPT errors.
+ mmcblk2: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 EXT4-fs (mmcblk0p3): recovery complete
 EXT4-fs (mmcblk0p3): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:3.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K
-GPT: Use GNU Parted to correct GPT errors.
- mmcblk2: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.641094] uvcvideo: Found UVC 1.00 device HD WebCam (1bcf:2c67)
+root@(none):/# [    2.610692] uvcvideo: Found UVC 1.00 device HD WebCam (1bcf:2c67)
 input: HD WebCam as /devices/soc0/70090000.usb/usb1/1-2/1-2:1.0/input/input5
 usb 1-3: new high-speed USB device number 3 using tegra-xusb
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2017-04-10 23:04:20.776945250 -0700
+++ /tmp/foob	2017-04-10 23:04:20.776945250 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1800665 us
+Bootloader-Cpu Init at (time stamp): 1800565 us
 
 Pinmux changes applied in kernel way
 
@@ -62,20 +62,20 @@
 Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1800639  us
-Bootloader (AosInit) time: 733450 us
-Bootloader Main Init time: 180289 us
-BlockDevMgrInit time: 157519 us
+Miscellaneous avp time: 1800539  us
+Bootloader (AosInit) time: 733509 us
+Bootloader Main Init time: 180167 us
+BlockDevMgrInit time: 157526 us
 PartMgrInit time: 8 us
-StorMgrInit time: 31 us
-FsMgrInit time: 2 us
-Partition Table load: 77069 us 
-Display init time: 574440 us
+StorMgrInit time: 30 us
+FsMgrInit time: 3 us
+Partition Table load: 77091 us 
+Display init time: 574593 us
 Kernel read time: 378 msec
 Load Kernel Size:   16304128 Bytes
 Bl Kernel ReadSpeed: 43132000 Bytes/sec
-Pre-Kernel Jump time: 1740786 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5844897 us
+Pre-Kernel Jump time: 1740913 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5842900 us
 Booting Linux on physical CPU 0x0
 Linux version 4.8.0-239303-g3477d16 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Fri Oct 7 01:12:37 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
@@ -184,7 +184,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(1.179:1): initialized
+audit: type=2000 audit(1.159:1): initialized
 workingset: timestamp_bits=46 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -251,11 +251,10 @@
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 +VDD_LED: supplied by +VDD_MUX
-+5V_USB_HS: supplied by +5V_SYS
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
++5V_USB_HS: supplied by +5V_SYS
 +5V_USB_SS: supplied by +5V_SYS
 +5V_HDMI_CON: supplied by +5V_SYS
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
@@ -273,6 +272,9 @@
 Freeing unused kernel memory: 960K (ffff800000c00000 - ffff800000cf0000)
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
+sdhci-tegra 700b0400.sdhci: Got CD GPIO
+sdhci-tegra 700b0400.sdhci: Got WP GPIO
 
 System initialization...
 
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-04-10 23:04:20.784945283 -0700
+++ /tmp/foob	2017-04-10 23:04:20.784945283 -0700
@@ -47,10 +47,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##################
+	 #################
 	 4.1 MiB/s
 done
-Bytes transferred = 7883456 (784ac0 hex)
+Bytes transferred = 7882656 (7847a0 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -61,7 +61,7 @@
 done
 Bytes transferred = 21426 (53b2 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x784ac0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x7847a0 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103b1
@@ -69,7 +69,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc6-next-20170410-165724-gf8c97bd (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Apr 10 01:21:22 PDT 2017
+Linux version 4.11.0-rc6-next-20170411-166044-gc4e7b35 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Apr 10 22:58:00 PDT 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -79,7 +79,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c157e380, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c157e880, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -92,7 +92,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 954380K/1048576K available (10240K kernel code, 1559K rwdata, 4376K rodata, 2048K init, 407K bss, 28660K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 954376K/1048576K available (10240K kernel code, 1561K rwdata, 4380K rodata, 2048K init, 407K bss, 28664K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -102,8 +102,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1585f28   (1560 kB)
-       .bss : 0xc158ed1c - 0xc15f4a08   ( 408 kB)
+      .data : 0xc1400000 - 0xc1586428   (1562 kB)
+       .bss : 0xc158fd1c - 0xc15f5a08   ( 408 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	RCU debugfs-based tracing is enabled.
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2017-04-10 23:04:20.792945316 -0700
+++ /tmp/foob	2017-04-10 23:04:20.792945316 -0700
@@ -48,7 +48,7 @@
 	 #######################
 	 4.1 MiB/s
 done
-Bytes transferred = 6062232 (5c8098 hex)
+Bytes transferred = 6062040 (5c7fd8 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -59,7 +59,7 @@
 done
 Bytes transferred = 21426 (53b2 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x5c8098 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x5c7fd8 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103b1
@@ -68,7 +68,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc6-next-20170410-165724-gf8c97bd (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Apr 10 01:20:13 PDT 2017
+Linux version 4.11.0-rc6-next-20170411-166044-gc4e7b35 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Apr 10 22:57:42 PDT 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -77,7 +77,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0eac840, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0ead6c0, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -90,7 +90,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 959400K/1048576K available (9216K kernel code, 731K rwdata, 2684K rodata, 1024K init, 304K bss, 23640K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 959392K/1048576K available (9216K kernel code, 734K rwdata, 2688K rodata, 1024K init, 307K bss, 23648K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -100,8 +100,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0eb6d70   ( 732 kB)
-       .bss : 0xc0ec06dc - 0xc0f0c8b4   ( 305 kB)
+      .data : 0xc0e00000 - 0xc0eb7bf0   ( 735 kB)
+       .bss : 0xc0ec16dc - 0xc0f0e3b4   ( 308 kB)
 Preemptible hierarchical RCU implementation.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
 RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=2
@@ -325,8 +325,8 @@
 Key type dns_resolver registered
 Registering SWP/SWPB emulation handler
 mmc1: new high speed SD card at address b368
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2017-04-10 23:04:20.800945349 -0700
+++ /tmp/foob	2017-04-10 23:04:20.800945349 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 748747 us
-Basic modules init took 350119 us
-Sec Bootdevice Read Time = 203 ms, Read Size = 8459 KB
-Next stage binary read took 14384 us
-Carveout took 252708 us
-CPU initialization took 93309 us
-Total time taken by TegraBoot 710520 us
+Tegraboot started after 741317 us
+Basic modules init took 350178 us
+Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
+Next stage binary read took 14385 us
+Carveout took 252723 us
+CPU initialization took 92360 us
+Total time taken by TegraBoot 709646 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.590] LPDDR4 Training: Number of tables = 10
+64b[0001.582] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-15655424 bytes read in 813 ms (18.4 MiB/s)
+15657472 bytes read in 811 ms (18.4 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-45142 bytes read in 460 ms (95.7 KiB/s)
+45142 bytes read in 459 ms (95.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 495 ms (3.1 MiB/s)
+1627787 bytes read in 497 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc6-next-20170410-283965-gf8c97bd (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Apr 10 01:24:41 PDT 2017
+Linux version 4.11.0-rc6-next-20170411-284285-gc4e7b35 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Apr 10 23:01:41 PDT 2017
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -174,15 +174,15 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffff80007a000000-ffff80007dffffff]
-Memory: 2983400K/3131392K available (8828K kernel code, 1080K rwdata, 4220K rodata, 1088K init, 396K bss, 131608K reserved, 16384K cma-reserved)
+Memory: 2983400K/3131392K available (8828K kernel code, 1082K rwdata, 4224K rodata, 1088K init, 397K bss, 131608K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008920000   (  8832 KB)
     .rodata : 0xffff000008920000 - 0xffff000008d50000   (  4288 KB)
       .init : 0xffff000008d50000 - 0xffff000008e60000   (  1088 KB)
-      .data : 0xffff000008e60000 - 0xffff000008f6e200   (  1081 KB)
-       .bss : 0xffff000008f6e200 - 0xffff000008fd1438   (   397 KB)
+      .data : 0xffff000008e60000 - 0xffff000008f6ea00   (  1083 KB)
+       .bss : 0xffff000008f6ea00 - 0xffff000008fd1f38   (   398 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -266,7 +266,7 @@
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 PCI: CLS 0 bytes, default 128
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 8090a000
+kvm [1]: IDMAP page: 80909000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
@@ -352,6 +352,6 @@
 devtmpfs: mounted
 Freeing unused kernel memory: 1088K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
-bash: no job contr[    1.356947] random: fast init done
+bash: no job contr[    1.358019] random: fast init done
 ol in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2017-04-10 23:04:20.808945382 -0700
+++ /tmp/foob	2017-04-10 23:04:20.808945382 -0700
@@ -1,4 +1,6 @@
-FI parameters from FDT:
+ Apr 10 23:01:41 PDT 2017
+Boot CPU: AArch64 Processor [411fd071]
+efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
 cma: Reserved 16 MiB at 0x00000000f5400000
 earlycon: uart0 at MMIO 0x0000000070006000 (options '115200n8')
@@ -30,15 +32,15 @@
 Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2837588K/2988032K available (8828K kernel code, 1080K rwdata, 4220K rodata, 1088K init, 396K bss, 134060K reserved, 16384K cma-reserved)
+Memory: 2837584K/2988032K available (8828K kernel code, 1082K rwdata, 4224K rodata, 1088K init, 397K bss, 134064K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008920000   (  8832 KB)
     .rodata : 0xffff000008920000 - 0xffff000008d50000   (  4288 KB)
       .init : 0xffff000008d50000 - 0xffff000008e60000   (  1088 KB)
-      .data : 0xffff000008e60000 - 0xffff000008f6e200   (  1081 KB)
-       .bss : 0xffff000008f6e200 - 0xffff000008fd1438   (   397 KB)
+      .data : 0xffff000008e60000 - 0xffff000008f6ea00   (  1083 KB)
+       .bss : 0xffff000008f6ea00 - 0xffff000008fd1f38   (   398 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -75,7 +77,7 @@
 console [tty0] enabled
 bootconsole [uart0] disabled
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc6-next-20170410-283965-gf8c97bd (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Apr 10 01:24:41 PDT 2017
+Linux version 4.11.0-rc6-next-20170411-284285-gc4e7b35 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Apr 10 23:01:41 PDT 2017
 Boot CPU: AArch64 Processor [411fd071]
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
@@ -109,15 +111,15 @@
 Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2837588K/2988032K available (8828K kernel code, 1080K rwdata, 4220K rodata, 1088K init, 396K bss, 134060K reserved, 16384K cma-reserved)
+Memory: 2837584K/2988032K available (8828K kernel code, 1082K rwdata, 4224K rodata, 1088K init, 397K bss, 134064K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008920000   (  8832 KB)
     .rodata : 0xffff000008920000 - 0xffff000008d50000   (  4288 KB)
       .init : 0xffff000008d50000 - 0xffff000008e60000   (  1088 KB)
-      .data : 0xffff000008e60000 - 0xffff000008f6e200   (  1081 KB)
-       .bss : 0xffff000008f6e200 - 0xffff000008fd1438   (   397 KB)
+      .data : 0xffff000008e60000 - 0xffff000008f6ea00   (  1083 KB)
+       .bss : 0xffff000008f6ea00 - 0xffff000008fd1f38   (   398 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -222,9 +224,9 @@
 RPC: Registered tcp transport module.
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 Unpacking initramfs...
-Freeing initrd memory: 2360K
+Freeing initrd memory: 2364K
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 8090a000
+kvm [1]: IDMAP page: 80909000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
@@ -315,15 +317,16 @@
 Key type dns_resolver registered
 registered taskstats version 1
 tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
-input: gpio-keys as /devices/platform/gpio-keys/input/input0
-tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 BGND3R 29.1 GiB 
 mmcblk0boot0: mmc0:0001 BGND3R partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
-max77686-rtc max77620-rtc: setting system clock to 2001-01-19 13:51:30 UTC (979912290)
+tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
+input: gpio-keys as /devices/platform/gpio-keys/input/input0
+tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
+max77686-rtc max77620-rtc: setting system clock to 2001-01-20 11:28:32 UTC (979990112)
 PP1200_AVDD: disabling
 PP1200_RCAM: disabling
 PP2800L_RCAM: disabling
@@ -335,7 +338,6 @@
 ALSA device list:
   No soundcards found.
 Freeing unused kernel memory: 1088K
-tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 
 System initialization...
 
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-04-10 23:04:20.816945414 -0700
+++ /tmp/foob	2017-04-10 23:04:20.816945414 -0700
@@ -39,10 +39,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##################
-	 3.2 MiB/s
+	 #################
+	 3.3 MiB/s
 done
-Bytes transferred = 7883456 (784ac0 hex)
+Bytes transferred = 7882656 (7847a0 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -53,7 +53,7 @@
 done
 Bytes transferred = 51804 (ca5c hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x784ac0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x7847a0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa5b ... OK
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc6-next-20170410-165724-gf8c97bd (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Apr 10 01:21:22 PDT 2017
+Linux version 4.11.0-rc6-next-20170411-166044-gc4e7b35 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Apr 10 22:58:00 PDT 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -71,7 +71,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c157e380, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c157e880, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -84,7 +84,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993472K/2096128K available (10240K kernel code, 1559K rwdata, 4376K rodata, 2048K init, 407K bss, 37120K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993468K/2096128K available (10240K kernel code, 1561K rwdata, 4380K rodata, 2048K init, 407K bss, 37124K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -94,8 +94,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1585f28   (1560 kB)
-       .bss : 0xc158ed1c - 0xc15f4a08   ( 408 kB)
+      .data : 0xc1400000 - 0xc1586428   (1562 kB)
+       .bss : 0xc158fd1c - 0xc15f5a08   ( 408 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU debugfs-based tracing is enabled.
@@ -318,8 +318,8 @@
  mmcblk1: p1 p2 p3 p4 p5
 mmc0: new ultra high speed DDR50 SDHC card at address aaaa
 mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
-tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
  mmcblk0: p1 p2 p3
+tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -391,5 +391,5 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.083706] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.063654] usb 2-1: new high-speed USB device number 2 using tegra-ehci
 asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0f:67:60
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2017-04-10 23:04:20.824945447 -0700
+++ /tmp/foob	2017-04-10 23:04:20.828945464 -0700
@@ -40,7 +40,7 @@
 	 #######################
 	 3.2 MiB/s
 done
-Bytes transferred = 6062232 (5c8098 hex)
+Bytes transferred = 6062040 (5c7fd8 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -51,7 +51,7 @@
 done
 Bytes transferred = 51804 (ca5c hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5c8098 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5c7fd8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa5b ... OK
@@ -60,7 +60,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc6-next-20170410-165724-gf8c97bd (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Apr 10 01:20:13 PDT 2017
+Linux version 4.11.0-rc6-next-20170411-166044-gc4e7b35 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Apr 10 22:57:42 PDT 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0eac840, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0ead6c0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998492K/2096128K available (9216K kernel code, 731K rwdata, 2684K rodata, 1024K init, 304K bss, 32100K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998484K/2096128K available (9216K kernel code, 734K rwdata, 2688K rodata, 1024K init, 307K bss, 32108K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,8 +92,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0eb6d70   ( 732 kB)
-       .bss : 0xc0ec06dc - 0xc0f0c8b4   ( 305 kB)
+      .data : 0xc0e00000 - 0xc0eb7bf0   ( 735 kB)
+       .bss : 0xc0ec16dc - 0xc0f0e3b4   ( 308 kB)
 Preemptible hierarchical RCU implementation.
 NR_IRQS:16 nr_irqs:16 16
 /interrupt-controller@60004000: 160 interrupts forwarded to /interrupt-controller@50041000
@@ -385,4 +385,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.360138] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.370129] usb 2-1: new high-speed USB device number 2 using tegra-ehci
