Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-04-14 22:30:48.638059123 -0700
+++ /tmp/foob	2016-04-14 22:30:48.638059123 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc3-next-20160414+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Apr 13 22:36:34 PDT 2016
+Linux version 4.6.0-rc3-next-20160415+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Apr 14 22:25:25 PDT 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -9,7 +9,7 @@
 psci: PSCIv0.2 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: Trusted OS migration not required
-percpu: Embedded 20 pages/cpu @ffffffc03efd3000 s42752 r8192 d30976 u81920
+percpu: Embedded 20 pages/cpu @ffffffc03efd2000 s42752 r8192 d30976 u81920
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
 CPU features: enabling workaround for ARM erratum 834220
@@ -19,14 +19,14 @@
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
 software IO TLB [mem 0x79c00000-0x7dc00000] (64MB) mapped at [ffffffc039c00000-ffffffc03dbfffff]
-Memory: 936272K/1048576K available (7036K kernel code, 625K rwdata, 2980K rodata, 856K init, 244K bss, 95920K reserved, 16384K cma-reserved)
+Memory: 936252K/1048576K available (7040K kernel code, 637K rwdata, 2980K rodata, 856K init, 244K bss, 95940K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffffff8000000000 - 0xffffff8008000000   (   128 MB)
     vmalloc : 0xffffff8008000000 - 0xffffffbdbfff0000   (   246 GB)
-      .text : 0xffffff8008080000 - 0xffffff800875d000   (  7028 KB)
-    .rodata : 0xffffff800875d000 - 0xffffff8008a4a000   (  2996 KB)
-      .init : 0xffffff8008a4a000 - 0xffffff8008b20000   (   856 KB)
-      .data : 0xffffff8008b20000 - 0xffffff8008bbc400   (   625 KB)
+      .text : 0xffffff8008080000 - 0xffffff800875e000   (  7032 KB)
+    .rodata : 0xffffff800875e000 - 0xffffff8008a4b000   (  2996 KB)
+      .init : 0xffffff8008a4b000 - 0xffffff8008b21000   (   856 KB)
+      .data : 0xffffff8008b21000 - 0xffffff8008bc0400   (   637 KB)
     vmemmap : 0xffffffbdc0000000 - 0xffffffbfc0000000   (     8 GB maximum)
               0xffffffbdc0000000 - 0xffffffbdc1000000   (    16 MB actual)
     fixed   : 0xffffffbffe7fd000 - 0xffffffbffec00000   (  4108 KB)
@@ -60,7 +60,7 @@
 pinctrl core: initialized pinctrl subsystem
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffffff8008763000, 1 data @ ffffff8008b24000)
+vdso: 2 pages (1 code @ ffffff8008764000, 1 data @ ffffff8008b28000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -94,7 +94,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.624:1): initialized
+audit: type=2000 audit(0.700:1): initialized
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -154,7 +154,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2016-04-14 05:37:05 UTC (1460612225)
+rtc-pl031 9010000.pl031: setting system clock to 2016-04-15 05:25:55 UTC (1460697955)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
@@ -163,7 +163,7 @@
 EXT4-fs (vda2): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 254:2.
 devtmpfs: mounted
-Freeing unused kernel memory: 856K (ffffff8008a4a000 - ffffff8008b20000)
+Freeing unused kernel memory: 856K (ffffff8008a4b000 - ffffff8008b21000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 bash-4.3# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-04-14 22:30:48.658059124 -0700
+++ /tmp/foob	2016-04-14 22:30:48.662059124 -0700
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra114 (Dalmore) # 
 Tegra114 (Dalmore) # usb reset
 (Re)start USB...
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 #######################
-	 3.2 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 7011520 (6afcc0 hex)
+Bytes transferred = 7013328 (6b03d0 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.6 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 34956 (888c hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6afcc0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6b03d0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff88b ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc3-next-20160414-86978-gd0591d2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Apr 13 22:34:42 PDT 2016
+Linux version 4.6.0-rc3-next-20160415-87276-gaec98d5 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Apr 14 22:23:29 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -90,7 +90,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc105b9ec   (14671 kB)
+      .text : 0xc0208000 - 0xc105ba14   (14671 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
       .data : 0xc1300000 - 0xc141a2c0   (1129 kB)
        .bss : 0xc141c000 - 0xc1470ad8   ( 339 kB)
@@ -124,18 +124,18 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 1 at arch/arm/mach-omap2/id.c:49 omap_rev+0x3c/0x50
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc3-next-20160414-86978-gd0591d2 #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc3-next-20160415-87276-gaec98d5 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c030f9a8>] (unwind_backtrace) from [<c030b088>] (show_stack+0x10/0x14)
-[<c030b088>] (show_stack) from [<c0582b28>] (dump_stack+0x84/0xa4)
-[<c0582b28>] (dump_stack) from [<c0341bf8>] (__warn+0xd4/0x100)
-[<c0341bf8>] (__warn) from [<c0341c44>] (warn_slowpath_null+0x20/0x28)
-[<c0341c44>] (warn_slowpath_null) from [<c03243fc>] (omap_rev+0x3c/0x50)
+[<c030b088>] (show_stack) from [<c05833a8>] (dump_stack+0x84/0xa4)
+[<c05833a8>] (dump_stack) from [<c0341c24>] (__warn+0xd4/0x100)
+[<c0341c24>] (__warn) from [<c0341c70>] (warn_slowpath_null+0x20/0x28)
+[<c0341c70>] (warn_slowpath_null) from [<c03243fc>] (omap_rev+0x3c/0x50)
 [<c03243fc>] (omap_rev) from [<c1114acc>] (__omap4_sar_ram_init+0x8/0x88)
 [<c1114acc>] (__omap4_sar_ram_init) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
 [<c0301e5c>] (do_one_initcall) from [<c1100ccc>] (kernel_init_freeable+0x70/0x1ec)
-[<c1100ccc>] (kernel_init_freeable) from [<c0b46c0c>] (kernel_init+0x8/0x118)
-[<c0b46c0c>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
+[<c1100ccc>] (kernel_init_freeable) from [<c0b477c0>] (kernel_init+0x8/0x118)
+[<c0b477c0>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
 ---[ end trace cb88537fdc8fa200 ]---
 Tegra Revision: A01 SKU: 0 CPU Process: 1 SoC Process: 0
 CPU1: thread -1, cpu 1, socket 0, mpidr 80000001
@@ -276,26 +276,26 @@
 ehci-atmel: EHCI Atmel driver
 tegra-ehci: Tegra EHCI driver
 ------------[ cut here ]------------
-WARNING: CPU: 1 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
+WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
 Modules linked in:
-CPU: 1 PID: 1 Comm: swapper/0 Tainted: G        W       4.6.0-rc3-next-20160414-86978-gd0591d2 #1
+CPU: 0 PID: 1 Comm: swapper/0 Tainted: G        W       4.6.0-rc3-next-20160415-87276-gaec98d5 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c030f9a8>] (unwind_backtrace) from [<c030b088>] (show_stack+0x10/0x14)
-[<c030b088>] (show_stack) from [<c0582b28>] (dump_stack+0x84/0xa4)
-[<c0582b28>] (dump_stack) from [<c0341bf8>] (__warn+0xd4/0x100)
-[<c0341bf8>] (__warn) from [<c0341c44>] (warn_slowpath_null+0x20/0x28)
-[<c0341c44>] (warn_slowpath_null) from [<c0694f48>] (__of_reset_control_get+0x1bc/0x228)
-[<c0694f48>] (__of_reset_control_get) from [<c0903100>] (tegra_ehci_probe+0x2d0/0x51c)
-[<c0903100>] (tegra_ehci_probe) from [<c076da5c>] (platform_drv_probe+0x50/0xb0)
-[<c076da5c>] (platform_drv_probe) from [<c076c3c0>] (driver_probe_device+0x1f4/0x2b0)
-[<c076c3c0>] (driver_probe_device) from [<c076c528>] (__driver_attach+0xac/0xb0)
-[<c076c528>] (__driver_attach) from [<c076a9d4>] (bus_for_each_dev+0x54/0x88)
-[<c076a9d4>] (bus_for_each_dev) from [<c076b9b4>] (bus_add_driver+0xe8/0x1f4)
-[<c076b9b4>] (bus_add_driver) from [<c076cd44>] (driver_register+0x78/0xf4)
-[<c076cd44>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
+[<c030b088>] (show_stack) from [<c05833a8>] (dump_stack+0x84/0xa4)
+[<c05833a8>] (dump_stack) from [<c0341c24>] (__warn+0xd4/0x100)
+[<c0341c24>] (__warn) from [<c0341c70>] (warn_slowpath_null+0x20/0x28)
+[<c0341c70>] (warn_slowpath_null) from [<c06957c8>] (__of_reset_control_get+0x1bc/0x228)
+[<c06957c8>] (__of_reset_control_get) from [<c0903658>] (tegra_ehci_probe+0x2d0/0x51c)
+[<c0903658>] (tegra_ehci_probe) from [<c076e2c8>] (platform_drv_probe+0x50/0xb0)
+[<c076e2c8>] (platform_drv_probe) from [<c076cc2c>] (driver_probe_device+0x1f4/0x2b0)
+[<c076cc2c>] (driver_probe_device) from [<c076cd94>] (__driver_attach+0xac/0xb0)
+[<c076cd94>] (__driver_attach) from [<c076b240>] (bus_for_each_dev+0x54/0x88)
+[<c076b240>] (bus_for_each_dev) from [<c076c220>] (bus_add_driver+0xe8/0x1f4)
+[<c076c220>] (bus_add_driver) from [<c076d5b0>] (driver_register+0x78/0xf4)
+[<c076d5b0>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
 [<c0301e5c>] (do_one_initcall) from [<c1100d7c>] (kernel_init_freeable+0x120/0x1ec)
-[<c1100d7c>] (kernel_init_freeable) from [<c0b46c0c>] (kernel_init+0x8/0x118)
-[<c0b46c0c>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
+[<c1100d7c>] (kernel_init_freeable) from [<c0b477c0>] (kernel_init+0x8/0x118)
+[<c0b477c0>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
 ---[ end trace cb88537fdc8fa201 ]---
 tegra-ehci 7d008000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d008000.usb: continuing, but please update your DT
@@ -342,15 +342,15 @@
 usb2_vbus: supplied by vdd-sys-5v0
 VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
-mmc0: new high speed SDHC card at address aaaa
 [drm] No driver support for vblank timestamp query.
+mmc0: new high speed SDHC card at address aaaa
 mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
- mmcblk0: p1 p2 p3
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
+ mmcblk0: p1 p2 p3
  mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
 Console: switching to colour frame buffer device 240x75
 drm drm: fb0:  frame buffer device
@@ -362,7 +362,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-06 16:53:57 UTC (949856037)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-07 16:42:43 UTC (949941763)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -374,4 +374,4 @@
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.645531] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.660547] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-04-14 22:30:48.670059124 -0700
+++ /tmp/foob	2016-04-14 22:30:48.670059124 -0700
@@ -39,18 +39,18 @@
 	 #######################################
 	 3.3 MiB/s
 done
-Bytes transferred = 5328888 (514ff8 hex)
+Bytes transferred = 5329528 (515278 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.8 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 34956 (888c hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x514ff8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x515278 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff88b ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc3-next-20160414-86978-gd0591d2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Apr 13 22:34:32 PDT 2016
+Linux version 4.6.0-rc3-next-20160415-87276-gaec98d5 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Apr 14 22:23:14 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -90,7 +90,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab4454   (10930 kB)
+      .text : 0xc0008000 - 0xc0ab4498   (10930 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
       .data : 0xc0c00000 - 0xc0c5dec0   ( 376 kB)
        .bss : 0xc0c5f000 - 0xc0cb9a4c   ( 363 kB)
@@ -252,25 +252,25 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1b4/0x218
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc3-next-20160414-86978-gd0591d2 #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc3-next-20160415-87276-gaec98d5 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c010f1f8>] (unwind_backtrace) from [<c010a864>] (show_stack+0x10/0x14)
-[<c010a864>] (show_stack) from [<c033a174>] (dump_stack+0x84/0xa4)
-[<c033a174>] (dump_stack) from [<c011f0b0>] (__warn+0xd4/0x100)
-[<c011f0b0>] (__warn) from [<c011f0fc>] (warn_slowpath_null+0x20/0x28)
-[<c011f0fc>] (warn_slowpath_null) from [<c03b8adc>] (__of_reset_control_get+0x1b4/0x218)
-[<c03b8adc>] (__of_reset_control_get) from [<c0509fbc>] (tegra_ehci_probe+0x2c8/0x514)
-[<c0509fbc>] (tegra_ehci_probe) from [<c04388c0>] (platform_drv_probe+0x50/0xb0)
-[<c04388c0>] (platform_drv_probe) from [<c04372ec>] (driver_probe_device+0x1f4/0x2b0)
-[<c04372ec>] (driver_probe_device) from [<c0437454>] (__driver_attach+0xac/0xb0)
-[<c0437454>] (__driver_attach) from [<c0435910>] (bus_for_each_dev+0x54/0x88)
-[<c0435910>] (bus_for_each_dev) from [<c04368e0>] (bus_add_driver+0xe8/0x1f4)
-[<c04368e0>] (bus_add_driver) from [<c0437c50>] (driver_register+0x78/0xf4)
-[<c0437c50>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
+[<c010f1f8>] (unwind_backtrace) from [<c010a86c>] (show_stack+0x10/0x14)
+[<c010a86c>] (show_stack) from [<c033a9d4>] (dump_stack+0x84/0xa4)
+[<c033a9d4>] (dump_stack) from [<c011f0dc>] (__warn+0xd4/0x100)
+[<c011f0dc>] (__warn) from [<c011f128>] (warn_slowpath_null+0x20/0x28)
+[<c011f128>] (warn_slowpath_null) from [<c03b933c>] (__of_reset_control_get+0x1b4/0x218)
+[<c03b933c>] (__of_reset_control_get) from [<c050a808>] (tegra_ehci_probe+0x2c8/0x514)
+[<c050a808>] (tegra_ehci_probe) from [<c043910c>] (platform_drv_probe+0x50/0xb0)
+[<c043910c>] (platform_drv_probe) from [<c0437b38>] (driver_probe_device+0x1f4/0x2b0)
+[<c0437b38>] (driver_probe_device) from [<c0437ca0>] (__driver_attach+0xac/0xb0)
+[<c0437ca0>] (__driver_attach) from [<c043615c>] (bus_for_each_dev+0x54/0x88)
+[<c043615c>] (bus_for_each_dev) from [<c043712c>] (bus_add_driver+0xe8/0x1f4)
+[<c043712c>] (bus_add_driver) from [<c043849c>] (driver_register+0x78/0xf4)
+[<c043849c>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
 [<c010179c>] (do_one_initcall) from [<c0b00d90>] (kernel_init_freeable+0x11c/0x1e8)
-[<c0b00d90>] (kernel_init_freeable) from [<c07de054>] (kernel_init+0x8/0x118)
-[<c07de054>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
----[ end trace 4b6de9fb4dedef15 ]---
+[<c0b00d90>] (kernel_init_freeable) from [<c07df378>] (kernel_init+0x8/0x118)
+[<c07df378>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
+---[ end trace c66e5a0f37ed6d5a ]---
 tegra-ehci 7d008000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d008000.usb: continuing, but please update your DT
 usbcore: registered new interface driver cdc_acm
@@ -344,7 +344,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-06 16:52:42 UTC (949855962)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-07 16:41:25 UTC (949941685)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -358,5 +358,4 @@
 Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
-root@(none):/# 
\ No newline at end of file
+root@(none):/# [    2.650138] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-04-14 22:30:48.678059125 -0700
+++ /tmp/foob	2016-04-14 22:30:48.682059125 -0700
@@ -26,7 +26,7 @@
        scanning usb for ethernet devices... 
 Warning: asx0 using MAC address from net device
 1 Ethernet Device(s) found
-Tegra124 (Jetson TK1) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash'
+Tegra124 (Jetson TK1) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=8683e640-b305-6c03-ae83-3c34ba91e1b7 rootwait gpt debug noinitrd earlyprintk init=/bin/bash'
 Tegra124 (Jetson TK1) # setenv ipaddr 192.168.100.102; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 zImage.multi_v7_defconfig
 Using asx0 device
@@ -43,7 +43,7 @@
 	 #######################
 	 2.8 MiB/s
 done
-Bytes transferred = 7011520 (6afcc0 hex)
+Bytes transferred = 7013328 (6b03d0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -54,7 +54,7 @@
 done
 Bytes transferred = 64488 (fbe8 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6afcc0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x6b03d0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012be7
@@ -62,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc3-next-20160414-86978-gd0591d2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Apr 13 22:34:42 PDT 2016
+Linux version 4.6.0-rc3-next-20160415-87276-gaec98d5 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Apr 14 22:23:29 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -80,7 +80,7 @@
 pcpu-alloc: s19648 r8192 d21312 u49152 alloc=12*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
-Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
+Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=8683e640-b305-6c03-ae83-3c34ba91e1b7 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
@@ -92,7 +92,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc105b9ec   (14671 kB)
+      .text : 0xc0208000 - 0xc105ba14   (14671 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
       .data : 0xc1300000 - 0xc141a2c0   (1129 kB)
        .bss : 0xc141c000 - 0xc1470ad8   ( 339 kB)
@@ -126,18 +126,18 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 1 at arch/arm/mach-omap2/id.c:49 omap_rev+0x3c/0x50
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc3-next-20160414-86978-gd0591d2 #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc3-next-20160415-87276-gaec98d5 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c030f9a8>] (unwind_backtrace) from [<c030b088>] (show_stack+0x10/0x14)
-[<c030b088>] (show_stack) from [<c0582b28>] (dump_stack+0x84/0xa4)
-[<c0582b28>] (dump_stack) from [<c0341bf8>] (__warn+0xd4/0x100)
-[<c0341bf8>] (__warn) from [<c0341c44>] (warn_slowpath_null+0x20/0x28)
-[<c0341c44>] (warn_slowpath_null) from [<c03243fc>] (omap_rev+0x3c/0x50)
+[<c030b088>] (show_stack) from [<c05833a8>] (dump_stack+0x84/0xa4)
+[<c05833a8>] (dump_stack) from [<c0341c24>] (__warn+0xd4/0x100)
+[<c0341c24>] (__warn) from [<c0341c70>] (warn_slowpath_null+0x20/0x28)
+[<c0341c70>] (warn_slowpath_null) from [<c03243fc>] (omap_rev+0x3c/0x50)
 [<c03243fc>] (omap_rev) from [<c1114acc>] (__omap4_sar_ram_init+0x8/0x88)
 [<c1114acc>] (__omap4_sar_ram_init) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
 [<c0301e5c>] (do_one_initcall) from [<c1100ccc>] (kernel_init_freeable+0x70/0x1ec)
-[<c1100ccc>] (kernel_init_freeable) from [<c0b46c0c>] (kernel_init+0x8/0x118)
-[<c0b46c0c>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
+[<c1100ccc>] (kernel_init_freeable) from [<c0b477c0>] (kernel_init+0x8/0x118)
+[<c0b477c0>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
 ---[ end trace cb88537fdc8fa200 ]---
 Tegra Revision: A01 SKU: 129 CPU Process: 1 SoC Process: 1
 CPU1: thread -1, cpu 1, socket 0, mpidr 80000001
@@ -267,24 +267,24 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Tainted: G        W       4.6.0-rc3-next-20160414-86978-gd0591d2 #1
+CPU: 0 PID: 1 Comm: swapper/0 Tainted: G        W       4.6.0-rc3-next-20160415-87276-gaec98d5 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c030f9a8>] (unwind_backtrace) from [<c030b088>] (show_stack+0x10/0x14)
-[<c030b088>] (show_stack) from [<c0582b28>] (dump_stack+0x84/0xa4)
-[<c0582b28>] (dump_stack) from [<c0341bf8>] (__warn+0xd4/0x100)
-[<c0341bf8>] (__warn) from [<c0341c44>] (warn_slowpath_null+0x20/0x28)
-[<c0341c44>] (warn_slowpath_null) from [<c0694f48>] (__of_reset_control_get+0x1bc/0x228)
-[<c0694f48>] (__of_reset_control_get) from [<c0903100>] (tegra_ehci_probe+0x2d0/0x51c)
-[<c0903100>] (tegra_ehci_probe) from [<c076da5c>] (platform_drv_probe+0x50/0xb0)
-[<c076da5c>] (platform_drv_probe) from [<c076c3c0>] (driver_probe_device+0x1f4/0x2b0)
-[<c076c3c0>] (driver_probe_device) from [<c076c528>] (__driver_attach+0xac/0xb0)
-[<c076c528>] (__driver_attach) from [<c076a9d4>] (bus_for_each_dev+0x54/0x88)
-[<c076a9d4>] (bus_for_each_dev) from [<c076b9b4>] (bus_add_driver+0xe8/0x1f4)
-[<c076b9b4>] (bus_add_driver) from [<c076cd44>] (driver_register+0x78/0xf4)
-[<c076cd44>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
+[<c030b088>] (show_stack) from [<c05833a8>] (dump_stack+0x84/0xa4)
+[<c05833a8>] (dump_stack) from [<c0341c24>] (__warn+0xd4/0x100)
+[<c0341c24>] (__warn) from [<c0341c70>] (warn_slowpath_null+0x20/0x28)
+[<c0341c70>] (warn_slowpath_null) from [<c06957c8>] (__of_reset_control_get+0x1bc/0x228)
+[<c06957c8>] (__of_reset_control_get) from [<c0903658>] (tegra_ehci_probe+0x2d0/0x51c)
+[<c0903658>] (tegra_ehci_probe) from [<c076e2c8>] (platform_drv_probe+0x50/0xb0)
+[<c076e2c8>] (platform_drv_probe) from [<c076cc2c>] (driver_probe_device+0x1f4/0x2b0)
+[<c076cc2c>] (driver_probe_device) from [<c076cd94>] (__driver_attach+0xac/0xb0)
+[<c076cd94>] (__driver_attach) from [<c076b240>] (bus_for_each_dev+0x54/0x88)
+[<c076b240>] (bus_for_each_dev) from [<c076c220>] (bus_add_driver+0xe8/0x1f4)
+[<c076c220>] (bus_add_driver) from [<c076d5b0>] (driver_register+0x78/0xf4)
+[<c076d5b0>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
 [<c0301e5c>] (do_one_initcall) from [<c1100d7c>] (kernel_init_freeable+0x120/0x1ec)
-[<c1100d7c>] (kernel_init_freeable) from [<c0b46c0c>] (kernel_init+0x8/0x118)
-[<c0b46c0c>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
+[<c1100d7c>] (kernel_init_freeable) from [<c0b477c0>] (kernel_init+0x8/0x118)
+[<c0b477c0>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
 ---[ end trace cb88537fdc8fa201 ]---
 tegra-ehci 7d004000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d004000.usb: continuing, but please update your DT
@@ -342,6 +342,13 @@
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
 tegra-pcie 1003000.pcie-controller: Failed to get supply 'avddio-pex': -517
 +5V_HDMI_CON: supplied by +5V_SYS
+mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+Primary GPT is invalid, using alternate GPT.
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
 tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
 tegra-ahci 70027000.sata: Failed to get regulators
@@ -383,46 +390,7 @@
 tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
 tegra-ahci 70027000.sata: Failed to get regulators
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-mmc0: new HS200 MMC card at address 0001
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
-Primary GPT is invalid, using alternate GPT.
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9
-sdhci-tegra 700b0400.sdhci: Got CD GPIO
-sdhci-tegra 700b0400.sdhci: Got WP GPIO
-mmc1: Unknown controller version (3). You may experience problems.
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 204000000, volt: 900000, enabled: 1. New: freq: 204000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 306000000, volt: 900000, enabled: 1. New: freq: 306000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 408000000, volt: 900000, enabled: 1. New: freq: 408000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 510000000, volt: 900000, enabled: 1. New: freq: 510000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 612000000, volt: 900000, enabled: 1. New: freq: 612000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 714000000, volt: 900000, enabled: 1. New: freq: 714000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 816000000, volt: 900000, enabled: 1. New: freq: 816000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 918000000, volt: 900000, enabled: 1. New: freq: 918000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1020000000, volt: 900000, enabled: 1. New: freq: 1020000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1122000000, volt: 900000, enabled: 1. New: freq: 1122000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1224000000, volt: 900000, enabled: 1. New: freq: 1224000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1326000000, volt: 900000, enabled: 1. New: freq: 1326000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1428000000, volt: 900000, enabled: 1. New: freq: 1428000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1530000000, volt: 900000, enabled: 1. New: freq: 1530000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1632000000, volt: 920000, enabled: 1. New: freq: 1632000000, volt: 920000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1734000000, volt: 960000, enabled: 1. New: freq: 1734000000, volt: 960000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1836000000, volt: 990000, enabled: 1. New: freq: 1836000000, volt: 990000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1938000000, volt: 1030000, enabled: 1. New: freq: 1938000000, volt: 1030000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2014500000, volt: 1060000, enabled: 1. New: freq: 2014500000, volt: 1060000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2116500000, volt: 1100000, enabled: 1. New: freq: 2116500000, volt: 1100000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2218500000, volt: 1140000, enabled: 1. New: freq: 2218500000, volt: 1140000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2320500000, volt: 1180000, enabled: 1. New: freq: 2320500000, volt: 1180000, enabled: 1
-tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
-tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
-tegra-pcie 1003000.pcie-controller: Failed to get supply 'avddio-pex': -517
-tegra-hdmi 54280000.hdmi: failed to get PLL regulator
-tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
-tegra-ahci 70027000.sata: Failed to get regulators
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 12:11:04 UTC (946728664)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:01:49 UTC (946684909)
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc1: Unknown controller version (3). You may experience problems.
@@ -460,46 +428,15 @@
 +5V_HDMI_CON: disabling
 +5V_SATA: disabling
 +12V_SATA: disabling
-asix 2-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:11:31:3b
 EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
-sdhci-tegra 700b0400.sdhci: Got CD GPIO
-sdhci-tegra 700b0400.sdhci: Got WP GPIO
-mmc1: Unknown controller version (3). You may experience problems.
 EXT4-fs (mmcblk0p1): couldn't mount as ext2 due to feature incompatibilities
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 204000000, volt: 900000, enabled: 1. New: freq: 204000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 306000000, volt: 900000, enabled: 1. New: freq: 306000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 408000000, volt: 900000, enabled: 1. New: freq: 408000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 510000000, volt: 900000, enabled: 1. New: freq: 510000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 612000000, volt: 900000, enabled: 1. New: freq: 612000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 714000000, volt: 900000, enabled: 1. New: freq: 714000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 816000000, volt: 900000, enabled: 1. New: freq: 816000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 918000000, volt: 900000, enabled: 1. New: freq: 918000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1020000000, volt: 900000, enabled: 1. New: freq: 1020000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1122000000, volt: 900000, enabled: 1. New: freq: 1122000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1224000000, volt: 900000, enabled: 1. New: freq: 1224000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1326000000, volt: 900000, enabled: 1. New: freq: 1326000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1428000000, volt: 900000, enabled: 1. New: freq: 1428000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1530000000, volt: 900000, enabled: 1. New: freq: 1530000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1632000000, volt: 920000, enabled: 1. New: freq: 1632000000, volt: 920000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1734000000, volt: 960000, enabled: 1. New: freq: 1734000000, volt: 960000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1836000000, volt: 990000, enabled: 1. New: freq: 1836000000, volt: 990000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1938000000, volt: 1030000, enabled: 1. New: freq: 1938000000, volt: 1030000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2014500000, volt: 1060000, enabled: 1. New: freq: 2014500000, volt: 1060000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2116500000, volt: 1100000, enabled: 1. New: freq: 2116500000, volt: 1100000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2218500000, volt: 1140000, enabled: 1. New: freq: 2218500000, volt: 1140000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2320500000, volt: 1180000, enabled: 1. New: freq: 2320500000, volt: 1180000, enabled: 1
-tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
-tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
-tegra-pcie 1003000.pcie-controller: Failed to get supply 'avddio-pex': -517
-tegra-hdmi 54280000.hdmi: failed to get PLL regulator
-tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
-tegra-ahci 70027000.sata: Failed to get regulators
 EXT4-fs (mmcblk0p1): warning: mounting fs with errors, running e2fsck is recommended
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-04-14 22:30:48.686059125 -0700
+++ /tmp/foob	2016-04-14 22:30:48.686059125 -0700
@@ -26,7 +26,7 @@
        scanning usb for ethernet devices... 
 Warning: asx0 using MAC address from net device
 1 Ethernet Device(s) found
-Tegra124 (Jetson TK1) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash'
+Tegra124 (Jetson TK1) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=8683e640-b305-6c03-ae83-3c34ba91e1b7 rootwait gpt debug noinitrd earlyprintk init=/bin/bash'
 Tegra124 (Jetson TK1) # setenv ipaddr 192.168.100.102; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 zImage.tegra_defconfig
 Using asx0 device
@@ -39,9 +39,9 @@
 	 #################################################################
 	 #################################################################
 	 #######################################
-	 3.4 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 5328888 (514ff8 hex)
+Bytes transferred = 5329528 (515278 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -52,7 +52,7 @@
 done
 Bytes transferred = 64488 (fbe8 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x514ff8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x515278 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012be7
@@ -61,7 +61,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc3-next-20160414-86978-gd0591d2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Apr 13 22:34:32 PDT 2016
+Linux version 4.6.0-rc3-next-20160415-87276-gaec98d5 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Apr 14 22:23:14 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -80,7 +80,7 @@
 pcpu-alloc: s19776 r8192 d21184 u49152 alloc=12*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
-Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
+Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=8683e640-b305-6c03-ae83-3c34ba91e1b7 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
@@ -92,7 +92,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab4454   (10930 kB)
+      .text : 0xc0008000 - 0xc0ab4498   (10930 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
       .data : 0xc0c00000 - 0xc0c5dec0   ( 376 kB)
        .bss : 0xc0c5f000 - 0xc0cb9a4c   ( 363 kB)
@@ -241,25 +241,25 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1b4/0x218
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc3-next-20160414-86978-gd0591d2 #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc3-next-20160415-87276-gaec98d5 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c010f1f8>] (unwind_backtrace) from [<c010a864>] (show_stack+0x10/0x14)
-[<c010a864>] (show_stack) from [<c033a174>] (dump_stack+0x84/0xa4)
-[<c033a174>] (dump_stack) from [<c011f0b0>] (__warn+0xd4/0x100)
-[<c011f0b0>] (__warn) from [<c011f0fc>] (warn_slowpath_null+0x20/0x28)
-[<c011f0fc>] (warn_slowpath_null) from [<c03b8adc>] (__of_reset_control_get+0x1b4/0x218)
-[<c03b8adc>] (__of_reset_control_get) from [<c0509fbc>] (tegra_ehci_probe+0x2c8/0x514)
-[<c0509fbc>] (tegra_ehci_probe) from [<c04388c0>] (platform_drv_probe+0x50/0xb0)
-[<c04388c0>] (platform_drv_probe) from [<c04372ec>] (driver_probe_device+0x1f4/0x2b0)
-[<c04372ec>] (driver_probe_device) from [<c0437454>] (__driver_attach+0xac/0xb0)
-[<c0437454>] (__driver_attach) from [<c0435910>] (bus_for_each_dev+0x54/0x88)
-[<c0435910>] (bus_for_each_dev) from [<c04368e0>] (bus_add_driver+0xe8/0x1f4)
-[<c04368e0>] (bus_add_driver) from [<c0437c50>] (driver_register+0x78/0xf4)
-[<c0437c50>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
+[<c010f1f8>] (unwind_backtrace) from [<c010a86c>] (show_stack+0x10/0x14)
+[<c010a86c>] (show_stack) from [<c033a9d4>] (dump_stack+0x84/0xa4)
+[<c033a9d4>] (dump_stack) from [<c011f0dc>] (__warn+0xd4/0x100)
+[<c011f0dc>] (__warn) from [<c011f128>] (warn_slowpath_null+0x20/0x28)
+[<c011f128>] (warn_slowpath_null) from [<c03b933c>] (__of_reset_control_get+0x1b4/0x218)
+[<c03b933c>] (__of_reset_control_get) from [<c050a808>] (tegra_ehci_probe+0x2c8/0x514)
+[<c050a808>] (tegra_ehci_probe) from [<c043910c>] (platform_drv_probe+0x50/0xb0)
+[<c043910c>] (platform_drv_probe) from [<c0437b38>] (driver_probe_device+0x1f4/0x2b0)
+[<c0437b38>] (driver_probe_device) from [<c0437ca0>] (__driver_attach+0xac/0xb0)
+[<c0437ca0>] (__driver_attach) from [<c043615c>] (bus_for_each_dev+0x54/0x88)
+[<c043615c>] (bus_for_each_dev) from [<c043712c>] (bus_add_driver+0xe8/0x1f4)
+[<c043712c>] (bus_add_driver) from [<c043849c>] (driver_register+0x78/0xf4)
+[<c043849c>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
 [<c010179c>] (do_one_initcall) from [<c0b00d90>] (kernel_init_freeable+0x11c/0x1e8)
-[<c0b00d90>] (kernel_init_freeable) from [<c07de054>] (kernel_init+0x8/0x118)
-[<c07de054>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
----[ end trace beb6578cce5d9783 ]---
+[<c0b00d90>] (kernel_init_freeable) from [<c07df378>] (kernel_init+0x8/0x118)
+[<c07df378>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
+---[ end trace 9c433b8511d035ea ]---
 tegra-ehci 7d004000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d004000.usb: continuing, but please update your DT
 tegra-ehci 7d004000.usb: EHCI Host Controller
@@ -295,7 +295,7 @@
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
 tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
-mmc0: new HS200 MMC card at address 0001
+mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
@@ -379,7 +379,7 @@
 tegra-ahci 70027000.sata: Failed to get regulators
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 12:09:32 UTC (946728572)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:15 UTC (946684815)
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc1: Unknown controller version (3). You may experience problems.
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 204000000, volt: 900000, enabled: 1. New: freq: 204000000, volt: 900000, enabled: 1
@@ -453,9 +453,9 @@
 tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
 tegra-ahci 70027000.sata: Failed to get regulators
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
+EXT4-fs (mmcblk0p1): warning: mounting fs with errors, running e2fsck is recommended
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc1: Unknown controller version (3). You may experience problems.
-EXT4-fs (mmcblk0p1): warning: mounting fs with errors, running e2fsck is recommended
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2016-04-14 22:30:48.694059125 -0700
+++ /tmp/foob	2016-04-14 22:30:48.694059125 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1799973 us
+Bootloader-Cpu Init at (time stamp): 1800331 us
 
 Pinmux changes applied in kernel way
 
@@ -59,32 +59,32 @@
 The proc BoardInfo: 0x0176:0x0001:0x01:0x43:0x00
 FastbootAddAtag: setting ATAG (0)
 
-Board Id Read Time: 4 us
+Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1799946  us
-Bootloader (AosInit) time: 734262 us
-Bootloader Main Init time: 180082 us
-BlockDevMgrInit time: 157546 us
+Miscellaneous avp time: 1800305  us
+Bootloader (AosInit) time: 733463 us
+Bootloader Main Init time: 180053 us
+BlockDevMgrInit time: 157562 us
 PartMgrInit time: 8 us
 StorMgrInit time: 30 us
 FsMgrInit time: 2 us
-Partition Table load: 76648 us 
-Display init time: 574819 us
+Partition Table load: 77093 us 
+Display init time: 574559 us
 Kernel read time: 312 msec
-Load Kernel Size:   13412352 Bytes
-Bl Kernel ReadSpeed: 42988000 Bytes/sec
-Pre-Kernel Jump time: 1515610 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5533567 us
+Load Kernel Size:   13428736 Bytes
+Bl Kernel ReadSpeed: 43040000 Bytes/sec
+Pre-Kernel Jump time: 1517368 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5534341 us
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc3-next-20160414+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Apr 13 22:36:34 PDT 2016
+Linux version 4.6.0-rc3-next-20160415+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Apr 14 22:25:25 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
 Memory limited to 1734MB
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
 cma: Reserved 16 MiB at 0x00000000eb400000
 /cpus/cpu@1: missing enable-method property
-percpu: Embedded 20 pages/cpu @ffffffc06c5a3000 s42752 r8192 d30976 u81920
+percpu: Embedded 20 pages/cpu @ffffffc06c5a2000 s42752 r8192 d30976 u81920
 Detected PIPT I-cache on CPU0
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 436968
 Kernel command line: tegraid=13.1.3.0.0 mem=1734M@2048M memtype=255 vpr=151M@3817M ddr_die=2048M@2048M section=256M commchip_id=0 usb_port_owner_info=0 lane_owner_info=0 emc_max_dvfs=0 touch_id=0@0 video=tegrafb no_console_suspend=1 console=ttyS0,115200n8 debug_uartport=lsport,0 earlyprintk=uart8250-32bit,0x70006000 usbcore.old_scheme_first=1 lp0_vec=0@0x0 tegra_fbmem=32899072@0xad012000 core_edp_mv=1150 core_edp_ma=4000 pmuboard=0x0176:0x0001:0x01:0x43:0x00 displayboard=0x0162:0x0001:0x00:0x42:0x00 power_supply=Adapter audio_codec=rt5640 board_info=0x0176:0x0001:0x01:0x43:0x00 tegraboot=sdmmc gpt gpt_sector=94207 modem_id=0 android.kerneltype=normal
@@ -92,14 +92,14 @@
 Dentry cache hash table entries: 262144 (order: 9, 2097152 bytes)
 Inode-cache hash table entries: 131072 (order: 8, 1048576 bytes)
 software IO TLB [mem 0xe5200000-0xe9200000] (64MB) mapped at [ffffffc065200000-ffffffc0691fffff]
-Memory: 1643640K/1775616K available (7036K kernel code, 625K rwdata, 2980K rodata, 856K init, 244K bss, 115592K reserved, 16384K cma-reserved)
+Memory: 1643620K/1775616K available (7040K kernel code, 637K rwdata, 2980K rodata, 856K init, 244K bss, 115612K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffffff8000000000 - 0xffffff8008000000   (   128 MB)
     vmalloc : 0xffffff8008000000 - 0xffffffbdbfff0000   (   246 GB)
-      .text : 0xffffff8008080000 - 0xffffff800875d000   (  7028 KB)
-    .rodata : 0xffffff800875d000 - 0xffffff8008a4a000   (  2996 KB)
-      .init : 0xffffff8008a4a000 - 0xffffff8008b20000   (   856 KB)
-      .data : 0xffffff8008b20000 - 0xffffff8008bbc400   (   625 KB)
+      .text : 0xffffff8008080000 - 0xffffff800875e000   (  7032 KB)
+    .rodata : 0xffffff800875e000 - 0xffffff8008a4b000   (  2996 KB)
+      .init : 0xffffff8008a4b000 - 0xffffff8008b21000   (   856 KB)
+      .data : 0xffffff8008b21000 - 0xffffff8008bc0400   (   637 KB)
     vmemmap : 0xffffffbdc0000000 - 0xffffffbfc0000000   (     8 GB maximum)
               0xffffffbdc0000000 - 0xffffffbdc1b18000   (    27 MB actual)
     fixed   : 0xffffffbffe7fd000 - 0xffffffbffec00000   (  4108 KB)
@@ -133,7 +133,7 @@
 pinctrl core: initialized pinctrl subsystem
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffffff8008763000, 1 data @ ffffff8008b24000)
+vdso: 2 pages (1 code @ ffffff8008764000, 1 data @ ffffff8008b28000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -181,7 +181,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.835:1): initialized
+audit: type=2000 audit(0.839:1): initialized
 workingset: timestamp_bits=44 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -239,21 +239,24 @@
 9pnet: Installing 9P2000 support
 Key type dns_resolver registered
 registered taskstats version 1
+mmc0: new high speed MMC card at address 0001
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
-sdhci-tegra 700b0400.sdhci: Got WP GPIO
-mmc1: Unknown controller version (3). You may experience problems.
-input: gpio-keys as /devices/platform/gpio-keys/input/input0
-mmc0: new HS200 MMC card at address 0001
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmc1: Unknown controller version (3). You may experience problems.
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+input: gpio-keys as /devices/platform/gpio-keys/input/input0
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc1: Unknown controller version (3). You may experience problems.
 hctosys: unable to open rtc device (rtc0)
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 +3.3V_AVDD_HDMI_AP_GATED: disabling
+sdhci-tegra 700b0400.sdhci: Got CD GPIO
+sdhci-tegra 700b0400.sdhci: Got WP GPIO
+mmc1: Unknown controller version (3). You may experience problems.
 +VDD_LED: disabling
 +5V_USB_HS: disabling
 +5V_USB_SS: disabling
@@ -261,15 +264,12 @@
 +5V_HDMI_CON: disabling
 ALSA device list:
   No soundcards found.
-Freeing unused kernel memory: 856K (ffffff8008a4a000 - ffffff8008b20000)
-sdhci-tegra 700b0400.sdhci: Got CD GPIO
+Freeing unused kernel memory: 856K (ffffff8008a4b000 - ffffff8008b21000)
 
 System initialization...
 
-Mounting /proc :[    2.583846] sdhci-tegra 700b0400.sdhci: Got WP GPIO
-[SUCCESS]
-Mounting /sys :[    2.599738] mmc1: Unknown controller version (3). You may experience problems.
-[SUCCESS]
+Mounting /proc :[SUCCESS]
+Mounting /sys :[SUCCESS]
 Mounting /dev :[SUCCESS]
 Mounting /dev/pts :[SUCCESS]
 Mounting /sys/kernel/debug :[SUCCESS]
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-04-14 22:30:48.702059125 -0700
+++ /tmp/foob	2016-04-14 22:30:48.702059125 -0700
@@ -47,9 +47,9 @@
 	 #################################################################
 	 #################################################################
 	 #######################
-	 3.6 MiB/s
+	 3.5 MiB/s
 done
-Bytes transferred = 7011520 (6afcc0 hex)
+Bytes transferred = 7013328 (6b03d0 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -60,7 +60,7 @@
 done
 Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x6afcc0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x6b03d0 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102b3
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc3-next-20160414-86978-gd0591d2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Apr 13 22:34:42 PDT 2016
+Linux version 4.6.0-rc3-next-20160415-87276-gaec98d5 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Apr 14 22:23:29 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -97,7 +97,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc105b9ec   (14671 kB)
+      .text : 0xc0208000 - 0xc105ba14   (14671 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
       .data : 0xc1300000 - 0xc141a2c0   (1129 kB)
        .bss : 0xc141c000 - 0xc1470ad8   ( 339 kB)
@@ -130,18 +130,18 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 1 at arch/arm/mach-omap2/id.c:49 omap_rev+0x3c/0x50
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc3-next-20160414-86978-gd0591d2 #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc3-next-20160415-87276-gaec98d5 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c030f9a8>] (unwind_backtrace) from [<c030b088>] (show_stack+0x10/0x14)
-[<c030b088>] (show_stack) from [<c0582b28>] (dump_stack+0x84/0xa4)
-[<c0582b28>] (dump_stack) from [<c0341bf8>] (__warn+0xd4/0x100)
-[<c0341bf8>] (__warn) from [<c0341c44>] (warn_slowpath_null+0x20/0x28)
-[<c0341c44>] (warn_slowpath_null) from [<c03243fc>] (omap_rev+0x3c/0x50)
+[<c030b088>] (show_stack) from [<c05833a8>] (dump_stack+0x84/0xa4)
+[<c05833a8>] (dump_stack) from [<c0341c24>] (__warn+0xd4/0x100)
+[<c0341c24>] (__warn) from [<c0341c70>] (warn_slowpath_null+0x20/0x28)
+[<c0341c70>] (warn_slowpath_null) from [<c03243fc>] (omap_rev+0x3c/0x50)
 [<c03243fc>] (omap_rev) from [<c1114acc>] (__omap4_sar_ram_init+0x8/0x88)
 [<c1114acc>] (__omap4_sar_ram_init) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
 [<c0301e5c>] (do_one_initcall) from [<c1100ccc>] (kernel_init_freeable+0x70/0x1ec)
-[<c1100ccc>] (kernel_init_freeable) from [<c0b46c0c>] (kernel_init+0x8/0x118)
-[<c0b46c0c>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
+[<c1100ccc>] (kernel_init_freeable) from [<c0b477c0>] (kernel_init+0x8/0x118)
+[<c0b477c0>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
 ---[ end trace cb88537fdc8fa200 ]---
 Tegra Revision: A03 prime SKU: 8 CPU Process: 1 SoC Process: 2
 CPU1: thread -1, cpu 1, socket 0, mpidr 80000001
@@ -287,26 +287,26 @@
 ehci-atmel: EHCI Atmel driver
 tegra-ehci: Tegra EHCI driver
 ------------[ cut here ]------------
-WARNING: CPU: 1 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
+WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
 Modules linked in:
-CPU: 1 PID: 1 Comm: swapper/0 Tainted: G        W       4.6.0-rc3-next-20160414-86978-gd0591d2 #1
+CPU: 0 PID: 1 Comm: swapper/0 Tainted: G        W       4.6.0-rc3-next-20160415-87276-gaec98d5 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c030f9a8>] (unwind_backtrace) from [<c030b088>] (show_stack+0x10/0x14)
-[<c030b088>] (show_stack) from [<c0582b28>] (dump_stack+0x84/0xa4)
-[<c0582b28>] (dump_stack) from [<c0341bf8>] (__warn+0xd4/0x100)
-[<c0341bf8>] (__warn) from [<c0341c44>] (warn_slowpath_null+0x20/0x28)
-[<c0341c44>] (warn_slowpath_null) from [<c0694f48>] (__of_reset_control_get+0x1bc/0x228)
-[<c0694f48>] (__of_reset_control_get) from [<c0903100>] (tegra_ehci_probe+0x2d0/0x51c)
-[<c0903100>] (tegra_ehci_probe) from [<c076da5c>] (platform_drv_probe+0x50/0xb0)
-[<c076da5c>] (platform_drv_probe) from [<c076c3c0>] (driver_probe_device+0x1f4/0x2b0)
-[<c076c3c0>] (driver_probe_device) from [<c076c528>] (__driver_attach+0xac/0xb0)
-[<c076c528>] (__driver_attach) from [<c076a9d4>] (bus_for_each_dev+0x54/0x88)
-[<c076a9d4>] (bus_for_each_dev) from [<c076b9b4>] (bus_add_driver+0xe8/0x1f4)
-[<c076b9b4>] (bus_add_driver) from [<c076cd44>] (driver_register+0x78/0xf4)
-[<c076cd44>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
+[<c030b088>] (show_stack) from [<c05833a8>] (dump_stack+0x84/0xa4)
+[<c05833a8>] (dump_stack) from [<c0341c24>] (__warn+0xd4/0x100)
+[<c0341c24>] (__warn) from [<c0341c70>] (warn_slowpath_null+0x20/0x28)
+[<c0341c70>] (warn_slowpath_null) from [<c06957c8>] (__of_reset_control_get+0x1bc/0x228)
+[<c06957c8>] (__of_reset_control_get) from [<c0903658>] (tegra_ehci_probe+0x2d0/0x51c)
+[<c0903658>] (tegra_ehci_probe) from [<c076e2c8>] (platform_drv_probe+0x50/0xb0)
+[<c076e2c8>] (platform_drv_probe) from [<c076cc2c>] (driver_probe_device+0x1f4/0x2b0)
+[<c076cc2c>] (driver_probe_device) from [<c076cd94>] (__driver_attach+0xac/0xb0)
+[<c076cd94>] (__driver_attach) from [<c076b240>] (bus_for_each_dev+0x54/0x88)
+[<c076b240>] (bus_for_each_dev) from [<c076c220>] (bus_add_driver+0xe8/0x1f4)
+[<c076c220>] (bus_add_driver) from [<c076d5b0>] (driver_register+0x78/0xf4)
+[<c076d5b0>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
 [<c0301e5c>] (do_one_initcall) from [<c1100d7c>] (kernel_init_freeable+0x120/0x1ec)
-[<c1100d7c>] (kernel_init_freeable) from [<c0b46c0c>] (kernel_init+0x8/0x118)
-[<c0b46c0c>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
+[<c1100d7c>] (kernel_init_freeable) from [<c0b477c0>] (kernel_init+0x8/0x118)
+[<c0b477c0>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
 ---[ end trace cb88537fdc8fa201 ]---
 tegra-ehci c5000000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci c5000000.usb: continuing, but please update your DT
@@ -360,29 +360,28 @@
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
-mmc0: host does not support reading read-only switch, assuming write-enable
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
+mmc0: host does not support reading read-only switch, assuming write-enable
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
+mmc0: new high speed SDHC card at address b368
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
-input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-28 16:31:11 UTC (1419784271)
-Waiting for root device PARTUUID=11eb23c1-01...
-mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
+input: gpio-keys as /devices/soc0/gpio-keys/input/input0
+rtc-em3027 2-0056: setting system clock to 2014-12-28 16:42:12 UTC (1419784932)
  mmcblk0: p1
+EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
-hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
-EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
+hub 1-1:1.0: USB hub found
+hub 1-1:1.0: 4 ports detected
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
-usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
+usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-04-14 22:30:48.710059126 -0700
+++ /tmp/foob	2016-04-14 22:30:48.714059126 -0700
@@ -45,20 +45,20 @@
 	 #################################################################
 	 #################################################################
 	 #######################################
-	 4.1 MiB/s
+	 901.4 KiB/s
 done
-Bytes transferred = 5328888 (514ff8 hex)
+Bytes transferred = 5329528 (515278 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 4 MiB/s
+	 861.3 KiB/s
 done
 Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x514ff8 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x515278 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102b3
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc3-next-20160414-86978-gd0591d2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Apr 13 22:34:32 PDT 2016
+Linux version 4.6.0-rc3-next-20160415-87276-gaec98d5 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Apr 14 22:23:14 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -97,7 +97,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab4454   (10930 kB)
+      .text : 0xc0008000 - 0xc0ab4498   (10930 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
       .data : 0xc0c00000 - 0xc0c5dec0   ( 376 kB)
        .bss : 0xc0c5f000 - 0xc0cb9a4c   ( 363 kB)
@@ -265,25 +265,25 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1b4/0x218
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc3-next-20160414-86978-gd0591d2 #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc3-next-20160415-87276-gaec98d5 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c010f1f8>] (unwind_backtrace) from [<c010a864>] (show_stack+0x10/0x14)
-[<c010a864>] (show_stack) from [<c033a174>] (dump_stack+0x84/0xa4)
-[<c033a174>] (dump_stack) from [<c011f0b0>] (__warn+0xd4/0x100)
-[<c011f0b0>] (__warn) from [<c011f0fc>] (warn_slowpath_null+0x20/0x28)
-[<c011f0fc>] (warn_slowpath_null) from [<c03b8adc>] (__of_reset_control_get+0x1b4/0x218)
-[<c03b8adc>] (__of_reset_control_get) from [<c0509fbc>] (tegra_ehci_probe+0x2c8/0x514)
-[<c0509fbc>] (tegra_ehci_probe) from [<c04388c0>] (platform_drv_probe+0x50/0xb0)
-[<c04388c0>] (platform_drv_probe) from [<c04372ec>] (driver_probe_device+0x1f4/0x2b0)
-[<c04372ec>] (driver_probe_device) from [<c0437454>] (__driver_attach+0xac/0xb0)
-[<c0437454>] (__driver_attach) from [<c0435910>] (bus_for_each_dev+0x54/0x88)
-[<c0435910>] (bus_for_each_dev) from [<c04368e0>] (bus_add_driver+0xe8/0x1f4)
-[<c04368e0>] (bus_add_driver) from [<c0437c50>] (driver_register+0x78/0xf4)
-[<c0437c50>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
+[<c010f1f8>] (unwind_backtrace) from [<c010a86c>] (show_stack+0x10/0x14)
+[<c010a86c>] (show_stack) from [<c033a9d4>] (dump_stack+0x84/0xa4)
+[<c033a9d4>] (dump_stack) from [<c011f0dc>] (__warn+0xd4/0x100)
+[<c011f0dc>] (__warn) from [<c011f128>] (warn_slowpath_null+0x20/0x28)
+[<c011f128>] (warn_slowpath_null) from [<c03b933c>] (__of_reset_control_get+0x1b4/0x218)
+[<c03b933c>] (__of_reset_control_get) from [<c050a808>] (tegra_ehci_probe+0x2c8/0x514)
+[<c050a808>] (tegra_ehci_probe) from [<c043910c>] (platform_drv_probe+0x50/0xb0)
+[<c043910c>] (platform_drv_probe) from [<c0437b38>] (driver_probe_device+0x1f4/0x2b0)
+[<c0437b38>] (driver_probe_device) from [<c0437ca0>] (__driver_attach+0xac/0xb0)
+[<c0437ca0>] (__driver_attach) from [<c043615c>] (bus_for_each_dev+0x54/0x88)
+[<c043615c>] (bus_for_each_dev) from [<c043712c>] (bus_add_driver+0xe8/0x1f4)
+[<c043712c>] (bus_add_driver) from [<c043849c>] (driver_register+0x78/0xf4)
+[<c043849c>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
 [<c010179c>] (do_one_initcall) from [<c0b00d90>] (kernel_init_freeable+0x11c/0x1e8)
-[<c0b00d90>] (kernel_init_freeable) from [<c07de054>] (kernel_init+0x8/0x118)
-[<c07de054>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
----[ end trace 5da649b188ddd4b4 ]---
+[<c0b00d90>] (kernel_init_freeable) from [<c07df378>] (kernel_init+0x8/0x118)
+[<c07df378>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
+---[ end trace 1248cd14dfdecbc1 ]---
 tegra-ehci c5000000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci c5000000.usb: continuing, but please update your DT
 ulpi_phy_power_on: ulpi write failed
@@ -328,16 +328,17 @@
 tlv320aic23-codec 2-001a: ASoC: no dapm match for MICIN --> NULL --> Mic Input
 tlv320aic23-codec 2-001a: ASoC: Failed to add route MICIN -> NULL -> Mic Input
 tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
-NET: Registered protocol family 10
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
-mip6: Mobile IPv6
-sit: IPv6 over IPv4 tunneling driver
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 mmc1: new high speed SD card at address b368
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
  mmcblk0: p1
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
+ mmcblk1: p1 p2
+NET: Registered protocol family 10
+mip6: Mobile IPv6
+sit: IPv6 over IPv4 tunneling driver
 NET: Registered protocol family 17
 NET: Registered protocol family 15
 can: controller area network core (rev 20120528 abi 9)
@@ -357,23 +358,22 @@
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
+hub 1-1:1.0: USB hub found
+hub 1-1:1.0: 4 ports detected
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-28 16:30:06 UTC (1419784206)
+rtc-em3027 2-0056: setting system clock to 2014-12-28 16:41:02 UTC (1419784862)
 ALSA device list:
   #0: tegra-trimslice
-hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
- mmcblk1: p1 p2
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
-usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
+usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.318913] asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
+root@(none):/# 
\ No newline at end of file
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2016-04-14 22:30:48.718059126 -0700
+++ /tmp/foob	2016-04-14 22:30:48.718059126 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 821045 us
-Basic modules init took 348760 us
+Tegraboot started after 812876 us
+Basic modules init took 349157 us
 Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14371 us
-Carveout took 252726 us
-CPU initialization took 92333 us
-Total time taken by TegraBoot 708190 us
+Next stage binary read took 14373 us
+Carveout took 252737 us
+CPU initialization took 92329 us
+Total time taken by TegraBoot 708596 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.658] LPDDR4 Training: Number of tables = 10
+64b[0001.650] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,20 +108,20 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
 Tegra210 (P2371-0000) # setenv devtype mmc
 Tegra210 (P2371-0000) # setenv devnum 0
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
-Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=/dev/mmcblk0p1 rootwait debug earlyprintk init=/bin/bash
+Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-11781120 bytes read in 727 ms (15.5 MiB/s)
+11797504 bytes read in 729 ms (15.4 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-37076 bytes read in 454 ms (79.1 KiB/s)
+37076 bytes read in 453 ms (79.1 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 496 ms (3.1 MiB/s)
+1627787 bytes read in 495 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc3-next-20160414+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Apr 13 22:36:34 PDT 2016
+Linux version 4.6.0-rc3-next-20160415+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Apr 14 22:25:25 PDT 2016
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -149,31 +149,31 @@
 psci: PSCIv0.2 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: MIGRATE_INFO_TYPE not supported.
-percpu: Embedded 20 pages/cpu @ffffffc0bff5a000 s42752 r8192 d30976 u81920
+percpu: Embedded 20 pages/cpu @ffffffc0bff59000 s42752 r8192 d30976 u81920
 pcpu-alloc: s42752 r8192 d30976 u81920 alloc=20*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
 CPU features: enabling workaround for ARM erratum 834220
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 770560
-Kernel command line: ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=/dev/mmcblk0p1 rootwait debug earlyprintk init=/bin/bash
+Kernel command line: ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 log_buf_len individual max cpu contribution: 4096 bytes
 log_buf_len total cpu_extra contributions: 12288 bytes
 log_buf_len min size: 16384 bytes
 log_buf_len: 32768 bytes
-early log buf free: 14304(87%)
+early log buf free: 14272(87%)
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffffffc07a000000-ffffffc07dffffff]
-Memory: 2981212K/3131392K available (7036K kernel code, 625K rwdata, 2980K rodata, 856K init, 244K bss, 133796K reserved, 16384K cma-reserved)
+Memory: 2981192K/3131392K available (7040K kernel code, 637K rwdata, 2980K rodata, 856K init, 244K bss, 133816K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffffff8000000000 - 0xffffff8008000000   (   128 MB)
     vmalloc : 0xffffff8008000000 - 0xffffffbdbfff0000   (   246 GB)
-      .text : 0xffffff8008080000 - 0xffffff800875d000   (  7028 KB)
-    .rodata : 0xffffff800875d000 - 0xffffff8008a4a000   (  2996 KB)
-      .init : 0xffffff8008a4a000 - 0xffffff8008b20000   (   856 KB)
-      .data : 0xffffff8008b20000 - 0xffffff8008bbc400   (   625 KB)
+      .text : 0xffffff8008080000 - 0xffffff800875e000   (  7032 KB)
+    .rodata : 0xffffff800875e000 - 0xffffff8008a4b000   (  2996 KB)
+      .init : 0xffffff8008a4b000 - 0xffffff8008b21000   (   856 KB)
+      .data : 0xffffff8008b21000 - 0xffffff8008bc0400   (   637 KB)
     vmemmap : 0xffffffbdc0000000 - 0xffffffbfc0000000   (     8 GB maximum)
               0xffffffbdc0000000 - 0xffffffbdc3000000   (    48 MB actual)
     fixed   : 0xffffffbffe7fd000 - 0xffffffbffec00000   (  4108 KB)
@@ -222,7 +222,7 @@
 pinctrl core: initialized pinctrl subsystem
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffffff8008763000, 1 data @ ffffff8008b24000)
+vdso: 2 pages (1 code @ ffffff8008764000, 1 data @ ffffff8008b28000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -319,7 +319,7 @@
 hctosys: unable to open rtc device (rtc0)
 ALSA device list:
   No soundcards found.
-Waiting for root device /dev/mmcblk0p1...
+Waiting for root device PARTUUID=00000000-0001-0000-788f-a75600000000...
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 SDW16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SDW16G partition 1 4.00 MiB
@@ -330,7 +330,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 856K (ffffff8008a4a000 - ffffff8008b20000)
+Freeing unused kernel memory: 856K (ffffff8008a4b000 - ffffff8008b21000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-04-14 22:30:48.726059126 -0700
+++ /tmp/foob	2016-04-14 22:30:48.726059126 -0700
@@ -24,7 +24,7 @@
        scanning usb for ethernet devices... 
 Warning: asx0 using MAC address from net device
 1 Ethernet Device(s) found
-Tegra30 (Beaver) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash'
+Tegra30 (Beaver) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=edbe1840-3633-c751-7993-b0ac834f81e0 rootwait gpt debug noinitrd earlyprintk init=/bin/bash'
 Tegra30 (Beaver) # setenv ipaddr 192.168.100.101; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra30 (Beaver) # tftpboot 0x81000000 zImage.multi_v7_defconfig
 Using asx0 device
@@ -41,18 +41,18 @@
 	 #######################
 	 3.4 MiB/s
 done
-Bytes transferred = 7011520 (6afcc0 hex)
+Bytes transferred = 7013328 (6b03d0 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.1 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 51511 (c937 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6afcc0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6b03d0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff936 ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc3-next-20160414-86978-gd0591d2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Apr 13 22:34:42 PDT 2016
+Linux version 4.6.0-rc3-next-20160415-87276-gaec98d5 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Apr 14 22:23:29 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -77,7 +77,7 @@
 pcpu-alloc: s19648 r8192 d21312 u49152 alloc=12*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
-Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
+Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=edbe1840-3633-c751-7993-b0ac834f81e0 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
@@ -89,7 +89,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc105b9ec   (14671 kB)
+      .text : 0xc0208000 - 0xc105ba14   (14671 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
       .data : 0xc1300000 - 0xc141a2c0   (1129 kB)
        .bss : 0xc141c000 - 0xc1470ad8   ( 339 kB)
@@ -122,18 +122,18 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 1 at arch/arm/mach-omap2/id.c:49 omap_rev+0x3c/0x50
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc3-next-20160414-86978-gd0591d2 #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc3-next-20160415-87276-gaec98d5 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c030f9a8>] (unwind_backtrace) from [<c030b088>] (show_stack+0x10/0x14)
-[<c030b088>] (show_stack) from [<c0582b28>] (dump_stack+0x84/0xa4)
-[<c0582b28>] (dump_stack) from [<c0341bf8>] (__warn+0xd4/0x100)
-[<c0341bf8>] (__warn) from [<c0341c44>] (warn_slowpath_null+0x20/0x28)
-[<c0341c44>] (warn_slowpath_null) from [<c03243fc>] (omap_rev+0x3c/0x50)
+[<c030b088>] (show_stack) from [<c05833a8>] (dump_stack+0x84/0xa4)
+[<c05833a8>] (dump_stack) from [<c0341c24>] (__warn+0xd4/0x100)
+[<c0341c24>] (__warn) from [<c0341c70>] (warn_slowpath_null+0x20/0x28)
+[<c0341c70>] (warn_slowpath_null) from [<c03243fc>] (omap_rev+0x3c/0x50)
 [<c03243fc>] (omap_rev) from [<c1114acc>] (__omap4_sar_ram_init+0x8/0x88)
 [<c1114acc>] (__omap4_sar_ram_init) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
 [<c0301e5c>] (do_one_initcall) from [<c1100ccc>] (kernel_init_freeable+0x70/0x1ec)
-[<c1100ccc>] (kernel_init_freeable) from [<c0b46c0c>] (kernel_init+0x8/0x118)
-[<c0b46c0c>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
+[<c1100ccc>] (kernel_init_freeable) from [<c0b477c0>] (kernel_init+0x8/0x118)
+[<c0b477c0>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
 ---[ end trace cb88537fdc8fa200 ]---
 Tegra Revision: A03 SKU: 128 CPU Process: 4 SoC Process: 0
 CPU1: thread -1, cpu 1, socket 0, mpidr 80000001
@@ -260,26 +260,26 @@
 ehci-atmel: EHCI Atmel driver
 tegra-ehci: Tegra EHCI driver
 ------------[ cut here ]------------
-WARNING: CPU: 1 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
+WARNING: CPU: 2 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
 Modules linked in:
-CPU: 1 PID: 1 Comm: swapper/0 Tainted: G        W       4.6.0-rc3-next-20160414-86978-gd0591d2 #1
+CPU: 2 PID: 1 Comm: swapper/0 Tainted: G        W       4.6.0-rc3-next-20160415-87276-gaec98d5 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c030f9a8>] (unwind_backtrace) from [<c030b088>] (show_stack+0x10/0x14)
-[<c030b088>] (show_stack) from [<c0582b28>] (dump_stack+0x84/0xa4)
-[<c0582b28>] (dump_stack) from [<c0341bf8>] (__warn+0xd4/0x100)
-[<c0341bf8>] (__warn) from [<c0341c44>] (warn_slowpath_null+0x20/0x28)
-[<c0341c44>] (warn_slowpath_null) from [<c0694f48>] (__of_reset_control_get+0x1bc/0x228)
-[<c0694f48>] (__of_reset_control_get) from [<c0903100>] (tegra_ehci_probe+0x2d0/0x51c)
-[<c0903100>] (tegra_ehci_probe) from [<c076da5c>] (platform_drv_probe+0x50/0xb0)
-[<c076da5c>] (platform_drv_probe) from [<c076c3c0>] (driver_probe_device+0x1f4/0x2b0)
-[<c076c3c0>] (driver_probe_device) from [<c076c528>] (__driver_attach+0xac/0xb0)
-[<c076c528>] (__driver_attach) from [<c076a9d4>] (bus_for_each_dev+0x54/0x88)
-[<c076a9d4>] (bus_for_each_dev) from [<c076b9b4>] (bus_add_driver+0xe8/0x1f4)
-[<c076b9b4>] (bus_add_driver) from [<c076cd44>] (driver_register+0x78/0xf4)
-[<c076cd44>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
+[<c030b088>] (show_stack) from [<c05833a8>] (dump_stack+0x84/0xa4)
+[<c05833a8>] (dump_stack) from [<c0341c24>] (__warn+0xd4/0x100)
+[<c0341c24>] (__warn) from [<c0341c70>] (warn_slowpath_null+0x20/0x28)
+[<c0341c70>] (warn_slowpath_null) from [<c06957c8>] (__of_reset_control_get+0x1bc/0x228)
+[<c06957c8>] (__of_reset_control_get) from [<c0903658>] (tegra_ehci_probe+0x2d0/0x51c)
+[<c0903658>] (tegra_ehci_probe) from [<c076e2c8>] (platform_drv_probe+0x50/0xb0)
+[<c076e2c8>] (platform_drv_probe) from [<c076cc2c>] (driver_probe_device+0x1f4/0x2b0)
+[<c076cc2c>] (driver_probe_device) from [<c076cd94>] (__driver_attach+0xac/0xb0)
+[<c076cd94>] (__driver_attach) from [<c076b240>] (bus_for_each_dev+0x54/0x88)
+[<c076b240>] (bus_for_each_dev) from [<c076c220>] (bus_add_driver+0xe8/0x1f4)
+[<c076c220>] (bus_add_driver) from [<c076d5b0>] (driver_register+0x78/0xf4)
+[<c076d5b0>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
 [<c0301e5c>] (do_one_initcall) from [<c1100d7c>] (kernel_init_freeable+0x120/0x1ec)
-[<c1100d7c>] (kernel_init_freeable) from [<c0b46c0c>] (kernel_init+0x8/0x118)
-[<c0b46c0c>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
+[<c1100d7c>] (kernel_init_freeable) from [<c0b477c0>] (kernel_init+0x8/0x118)
+[<c0b477c0>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
 ---[ end trace cb88537fdc8fa201 ]---
 tegra-ehci 7d004000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d004000.usb: continuing, but please update your DT
@@ -344,8 +344,8 @@
 mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB
-Primary GPT is invalid, using alternate GPT.
 tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
+Primary GPT is invalid, using alternate GPT.
  mmcblk1: p1 p2 p3 p4 p5
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -408,6 +408,13 @@
 tps65910-rtc tps65910-rtc: setting system clock to 2000-01-01 00:00:02 UTC (946684802)
 vdd_sata,avdd_plle: disabling
 usb1_vbus: disabling
-Waiting for root device /dev/mmcblk0p1...
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
-asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0f:67:60
+EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
+EXT4-fs (mmcblk1p1): warning: mounting fs with errors, running e2fsck is recommended
+EXT4-fs (mmcblk1p1): recovery complete
+EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
+VFS: Mounted root (ext3 filesystem) on device 179:1.
+devtmpfs: mounted
+Freeing unused kernel memory: 2048K (c1100000 - c1300000)
+bash: cannot set terminal process group (-1): Inappropriate ioctl for device
+bash: no job control in this shell
+root@(none):/# [    4.108915] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-04-14 22:30:48.734059127 -0700
+++ /tmp/foob	2016-04-14 22:30:48.734059127 -0700
@@ -24,7 +24,7 @@
        scanning usb for ethernet devices... 
 Warning: asx0 using MAC address from net device
 1 Ethernet Device(s) found
-Tegra30 (Beaver) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash'
+Tegra30 (Beaver) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=edbe1840-3633-c751-7993-b0ac834f81e0 rootwait gpt debug noinitrd earlyprintk init=/bin/bash'
 Tegra30 (Beaver) # setenv ipaddr 192.168.100.101; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra30 (Beaver) # tftpboot 0x81000000 zImage.tegra_defconfig
 Using asx0 device
@@ -37,9 +37,9 @@
 	 #################################################################
 	 #################################################################
 	 #######################################
-	 3.3 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 5328888 (514ff8 hex)
+Bytes transferred = 5329528 (515278 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 51511 (c937 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x514ff8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x515278 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff936 ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc3-next-20160414-86978-gd0591d2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Apr 13 22:34:32 PDT 2016
+Linux version 4.6.0-rc3-next-20160415-87276-gaec98d5 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Apr 14 22:23:14 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -77,7 +77,7 @@
 pcpu-alloc: s19776 r8192 d21184 u49152 alloc=12*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
-Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
+Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=edbe1840-3633-c751-7993-b0ac834f81e0 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
@@ -89,7 +89,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab4454   (10930 kB)
+      .text : 0xc0008000 - 0xc0ab4498   (10930 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
       .data : 0xc0c00000 - 0xc0c5dec0   ( 376 kB)
        .bss : 0xc0c5f000 - 0xc0cb9a4c   ( 363 kB)
@@ -234,27 +234,27 @@
 ehci-pci: EHCI PCI platform driver
 tegra-ehci: Tegra EHCI driver
 ------------[ cut here ]------------
-WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1b4/0x218
+WARNING: CPU: 2 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1b4/0x218
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc3-next-20160414-86978-gd0591d2 #1
+CPU: 2 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc3-next-20160415-87276-gaec98d5 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c010f1f8>] (unwind_backtrace) from [<c010a864>] (show_stack+0x10/0x14)
-[<c010a864>] (show_stack) from [<c033a174>] (dump_stack+0x84/0xa4)
-[<c033a174>] (dump_stack) from [<c011f0b0>] (__warn+0xd4/0x100)
-[<c011f0b0>] (__warn) from [<c011f0fc>] (warn_slowpath_null+0x20/0x28)
-[<c011f0fc>] (warn_slowpath_null) from [<c03b8adc>] (__of_reset_control_get+0x1b4/0x218)
-[<c03b8adc>] (__of_reset_control_get) from [<c0509fbc>] (tegra_ehci_probe+0x2c8/0x514)
-[<c0509fbc>] (tegra_ehci_probe) from [<c04388c0>] (platform_drv_probe+0x50/0xb0)
-[<c04388c0>] (platform_drv_probe) from [<c04372ec>] (driver_probe_device+0x1f4/0x2b0)
-[<c04372ec>] (driver_probe_device) from [<c0437454>] (__driver_attach+0xac/0xb0)
-[<c0437454>] (__driver_attach) from [<c0435910>] (bus_for_each_dev+0x54/0x88)
-[<c0435910>] (bus_for_each_dev) from [<c04368e0>] (bus_add_driver+0xe8/0x1f4)
-[<c04368e0>] (bus_add_driver) from [<c0437c50>] (driver_register+0x78/0xf4)
-[<c0437c50>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
+[<c010f1f8>] (unwind_backtrace) from [<c010a86c>] (show_stack+0x10/0x14)
+[<c010a86c>] (show_stack) from [<c033a9d4>] (dump_stack+0x84/0xa4)
+[<c033a9d4>] (dump_stack) from [<c011f0dc>] (__warn+0xd4/0x100)
+[<c011f0dc>] (__warn) from [<c011f128>] (warn_slowpath_null+0x20/0x28)
+[<c011f128>] (warn_slowpath_null) from [<c03b933c>] (__of_reset_control_get+0x1b4/0x218)
+[<c03b933c>] (__of_reset_control_get) from [<c050a808>] (tegra_ehci_probe+0x2c8/0x514)
+[<c050a808>] (tegra_ehci_probe) from [<c043910c>] (platform_drv_probe+0x50/0xb0)
+[<c043910c>] (platform_drv_probe) from [<c0437b38>] (driver_probe_device+0x1f4/0x2b0)
+[<c0437b38>] (driver_probe_device) from [<c0437ca0>] (__driver_attach+0xac/0xb0)
+[<c0437ca0>] (__driver_attach) from [<c043615c>] (bus_for_each_dev+0x54/0x88)
+[<c043615c>] (bus_for_each_dev) from [<c043712c>] (bus_add_driver+0xe8/0x1f4)
+[<c043712c>] (bus_add_driver) from [<c043849c>] (driver_register+0x78/0xf4)
+[<c043849c>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
 [<c010179c>] (do_one_initcall) from [<c0b00d90>] (kernel_init_freeable+0x11c/0x1e8)
-[<c0b00d90>] (kernel_init_freeable) from [<c07de054>] (kernel_init+0x8/0x118)
-[<c07de054>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
----[ end trace 7db85dcdf3346e83 ]---
+[<c0b00d90>] (kernel_init_freeable) from [<c07df378>] (kernel_init+0x8/0x118)
+[<c07df378>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
+---[ end trace 7d6deffe5e4b3a78 ]---
 tegra-ehci 7d004000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d004000.usb: continuing, but please update your DT
 usbcore: registered new interface driver cdc_acm
@@ -392,6 +392,14 @@
 usb1_vbus: disabling
 ALSA device list:
   #0: NVIDIA Tegra Beaver
-Waiting for root device /dev/mmcblk0p1...
+EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
+EXT4-fs (mmcblk1p1): warning: mounting fs with errors, running e2fsck is recommended
+EXT4-fs (mmcblk1p1): recovery complete
+EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
+VFS: Mounted root (ext3 filesystem) on device 179:1.
+devtmpfs: mounted
+Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
+bash: cannot set terminal process group (-1): Inappropriate ioctl for device
+bash: no job control in this shell
 usb 2-1: new high-speed USB device number 2 using tegra-ehci
-asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0f:67:60
+root@(none):/# 
\ No newline at end of file
