Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2017-12-07 01:50:15.215803985 -0800
+++ /tmp/foob	2017-12-07 01:50:15.215803985 -0800
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0000000000 [0x411fd070]
-Linux version 4.15.0-rc2-next-20171206-337197-gd5c52866f560 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Dec 6 09:04:15 PST 2017
+Linux version 4.15.0-rc2-next-20171207-337784-ge40fd8d6b4d9 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Thu Dec 7 01:37:31 PST 2017
 Machine model: linux,dummy-virt
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -20,22 +20,22 @@
 psci: Using standard PSCI v0.2 function IDs
 psci: Trusted OS migration not required
 random: fast init done
-percpu: Embedded 23 pages/cpu @        (ptrval) s55936 r8192 d30080 u94208
+percpu: Embedded 23 pages/cpu @        (ptrval) s55168 r8192 d30848 u94208
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
 CPU features: enabling workaround for ARM erratum 834220
 Built 1 zonelists, mobility grouping on.  Total pages: 258048
 Policy zone: DMA
 Kernel command line: root=/dev/vda2 rw rootwait mem=1024M console=ttyAMA0,38400n8 init=/bin/bash
-Memory: 998540K/1048576K available (9724K kernel code, 1162K rwdata, 4484K rodata, 1152K init, 394K bss, 33652K reserved, 16384K cma-reserved)
+Memory: 998540K/1048576K available (9724K kernel code, 1160K rwdata, 4484K rodata, 1152K init, 396K bss, 33652K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0x        (ptrval) - 0x        (ptrval)   (  9728 KB)
     .rodata : 0x        (ptrval) - 0x        (ptrval)   (  4544 KB)
       .init : 0x        (ptrval) - 0x        (ptrval)   (  1152 KB)
-      .data : 0x        (ptrval) - 0x        (ptrval)   (  1163 KB)
-       .bss : 0x        (ptrval) - 0x        (ptrval)   (   395 KB)
+      .data : 0x        (ptrval) - 0x        (ptrval)   (  1161 KB)
+       .bss : 0x        (ptrval) - 0x        (ptrval)   (   397 KB)
     fixed   : 0xffff7dfffe7fb000 - 0xffff7dfffec00000   (  4116 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -77,7 +77,7 @@
 audit: initializing netlink subsys (disabled)
 audit: type=2000 audit(0.188:1): state=initialized audit_enabled=0 res=1
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ 00000000fc1313f3, 1 data @ 00000000430ce140)
+vdso: 2 pages (1 code @ 000000003c9132d3, 1 data @ 0000000068ef1e67)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -184,7 +184,7 @@
 registered taskstats version 1
 Loading compiled-in X.509 certificates
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2017-12-06 17:10:30 UTC (1512580230)
+rtc-pl031 9010000.pl031: setting system clock to 2017-12-07 09:44:16 UTC (1512639856)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-12-07 01:50:15.251804145 -0800
+++ /tmp/foob	2017-12-07 01:50:15.255804163 -0800
@@ -40,20 +40,20 @@
 	 #################################################################
 	 #################################################################
 	 #################################
-	 2.9 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 8114688 (7bd200 hex)
+Bytes transferred = 8110592 (7bc200 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.7 MiB/s
+	 2.3 MiB/s
 done
 Bytes transferred = 34502 (86c6 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x7bd200 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x7bc200 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff6c5 ... OK
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-rc2-next-20171206-337197-gd5c52866f560 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Dec 6 08:57:44 PST 2017
+Linux version 4.15.0-rc2-next-20171207-337784-ge40fd8d6b4d9 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Dec 7 01:31:34 PST 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -77,14 +77,14 @@
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
 random: fast init done
-percpu: Embedded 17 pages/cpu @(ptrval) s39232 r8192 d22208 u69632
-pcpu-alloc: s39232 r8192 d22208 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @(ptrval) s39040 r8192 d22400 u69632
+pcpu-alloc: s39040 r8192 d22400 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=7369c667-ff51-ec4a-29cd-baabf2fbe346 rootwait gpt debug earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1992548K/2096128K available (11264K kernel code, 1603K rwdata, 4312K rodata, 2048K init, 404K bss, 38044K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1992548K/2096128K available (11264K kernel code, 1602K rwdata, 4312K rodata, 2048K init, 404K bss, 38044K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -94,7 +94,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0x(ptrval) - 0x(ptrval)   (12256 kB)
       .init : 0x(ptrval) - 0x(ptrval)   (2048 kB)
-      .data : 0x(ptrval) - 0x(ptrval)   (1604 kB)
+      .data : 0x(ptrval) - 0x(ptrval)   (1603 kB)
        .bss : 0x(ptrval) - 0x(ptrval)   ( 405 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -394,7 +394,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-04-28 23:59:37 UTC (956966377)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-04-29 16:34:10 UTC (957026050)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2017-12-07 01:50:15.259804181 -0800
+++ /tmp/foob	2017-12-07 01:50:15.263804199 -0800
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra114 (Dalmore) # 
 Tegra114 (Dalmore) # usb reset
 (Re)start USB...
@@ -40,7 +40,7 @@
 	 #################################
 	 3.2 MiB/s
 done
-Bytes transferred = 6204160 (5eab00 hex)
+Bytes transferred = 6203752 (5ea968 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -51,7 +51,7 @@
 done
 Bytes transferred = 34502 (86c6 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5eab00 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5ea968 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff6c5 ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-rc2-next-20171206-17248-gd5c5286 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Dec 6 08:56:59 PST 2017
+Linux version 4.15.0-rc2-next-20171207-17835-ge40fd8d (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Dec 7 01:30:48 PST 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -74,14 +74,14 @@
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
 random: fast init done
-percpu: Embedded 17 pages/cpu @(ptrval) s39296 r8192 d22144 u69632
-pcpu-alloc: s39296 r8192 d22144 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @(ptrval) s39104 r8192 d22336 u69632
+pcpu-alloc: s39104 r8192 d22336 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=7369c667-ff51-ec4a-29cd-baabf2fbe346 rootwait gpt debug earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998600K/2096128K available (9216K kernel code, 778K rwdata, 2564K rodata, 1024K init, 298K bss, 31992K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998600K/2096128K available (9216K kernel code, 777K rwdata, 2564K rodata, 1024K init, 298K bss, 31992K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -91,7 +91,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0x(ptrval) - 0x(ptrval)   (10208 kB)
       .init : 0x(ptrval) - 0x(ptrval)   (1024 kB)
-      .data : 0x(ptrval) - 0x(ptrval)   ( 779 kB)
+      .data : 0x(ptrval) - 0x(ptrval)   ( 778 kB)
        .bss : 0x(ptrval) - 0x(ptrval)   ( 299 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
@@ -343,10 +343,10 @@
 iommu: Adding device snd-soc-dummy to group 57
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
-mmc0: new high speed SDHC card at address aaaa
-mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
+mmc0: new high speed SDHC card at address aaaa
+mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB, chardev (247:0)
  mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
  mmcblk0: p1 p2 p3
@@ -389,7 +389,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-04-28 23:56:46 UTC (956966206)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-04-29 16:31:28 UTC (957025888)
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
@@ -407,4 +407,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.121235] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.100841] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-12-07 01:50:15.267804217 -0800
+++ /tmp/foob	2017-12-07 01:50:15.271804235 -0800
@@ -35,9 +35,9 @@
 	 #################################################################
 	 #################################################################
 	 #################################
-	 4 MiB/s
+	 3.9 MiB/s
 done
-Bytes transferred = 8114688 (7bd200 hex)
+Bytes transferred = 8110592 (7bc200 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -48,7 +48,7 @@
 done
 Bytes transferred = 66544 (103f0 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x7bd200 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x7bc200 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830133ef
@@ -56,7 +56,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-rc2-next-20171206-337197-gd5c52866f560 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Dec 6 08:57:44 PST 2017
+Linux version 4.15.0-rc2-next-20171207-337784-ge40fd8d6b4d9 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Dec 7 01:31:34 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -72,14 +72,14 @@
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
 random: fast init done
-percpu: Embedded 17 pages/cpu @(ptrval) s39232 r8192 d22208 u69632
-pcpu-alloc: s39232 r8192 d22208 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @(ptrval) s39040 r8192 d22400 u69632
+pcpu-alloc: s39040 r8192 d22400 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=b1791faa-84e8-4e27-83a1-9fb0a77e8803 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1992440K/2096128K available (11264K kernel code, 1603K rwdata, 4312K rodata, 2048K init, 404K bss, 38152K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1992440K/2096128K available (11264K kernel code, 1602K rwdata, 4312K rodata, 2048K init, 404K bss, 38152K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -89,7 +89,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0x(ptrval) - 0x(ptrval)   (12256 kB)
       .init : 0x(ptrval) - 0x(ptrval)   (2048 kB)
-      .data : 0x(ptrval) - 0x(ptrval)   (1604 kB)
+      .data : 0x(ptrval) - 0x(ptrval)   (1603 kB)
        .bss : 0x(ptrval) - 0x(ptrval)   ( 405 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -391,17 +391,12 @@
 +5V_USB_HS: supplied by +5V_SYS
 +3.3V_LP0: supplied by +3.3V_SYS
 +5V_HDMI_CON: supplied by +5V_SYS
-mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB, chardev (236:0)
- mmcblk0: p1
 +5V_SATA: supplied by +5V_SYS
 +12V_SATA: supplied by +VDD_MUX
  usb2-0: usb2-0 supply vbus not found, using dummy regulator
  usb2-1: usb2-1 supply vbus not found, using dummy regulator
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
+mmc0: new high speed MMC card at address 0001
 tegra-pcie 1003000.pcie: 2x1, 1x1 configuration
 tegra-pcie 1003000.pcie: Failed to get supply 'avddio-pex': -517
 tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
@@ -466,7 +461,6 @@
 pci 0000:00:02.0: enabling Extended Tags
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x88 may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x52 may corrupt adjacent RW1C bits
-ata1: SATA link down (SStatus 0 SControl 300)
 pci 0000:00:02.0: PME# supported from D0 D1 D2 D3hot D3cold
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4c may corrupt adjacent RW1C bits
 PCI: bus0: Fast back to back transfers disabled
@@ -505,23 +499,18 @@
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
 r8169 0000:02:00.0 (unnamed net_device) (uninitialized): not PCI Express
-mmc1: new high speed SDHC card at address aaaa
-mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
-r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0x6e5cb173, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 22
+r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xc2ba86e0, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 22
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
- mmcblk1: p1 p2 p3
-tegra-hdmi 54280000.hdmi: cannot set audio to 48000 Hz at 297000000 Hz pixel clock
-Console: switching to colour frame buffer device 512x135
-drm drm: fb0:  frame buffer device
+[drm] Cannot find any crtc or sizes
 [drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
 iommu: Adding device cpufreq-dt.0 to group 74
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:04:16 UTC (946685056)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:04:25 UTC (946685065)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
@@ -530,12 +519,24 @@
 +3.3V_RUN_TOUCH: disabling
 +2.8V_RUN_CAM_AF: disabling
 +1.8V_RUN_VPP_FUSE: disabling
+mmc1: new high speed SDHC card at address aaaa
+mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
+ mmcblk1: p1 p2 p3
+tegra-hdmi 54280000.hdmi: cannot set audio to 48000 Hz at 297000000 Hz pixel clock
+ata1: SATA link down (SStatus 0 SControl 300)
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB, chardev (236:0)
+ mmcblk0: p1
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:1.
+VFS: Mounted root (ext3 filesystem) on device 179:17.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K
+Console: switching to colour frame buffer device 512x135
+drm drm: fb0:  frame buffer device
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2017-12-07 01:50:15.275804253 -0800
+++ /tmp/foob	2017-12-07 01:50:15.279804270 -0800
@@ -33,20 +33,20 @@
 	 #################################################################
 	 #################################################################
 	 #################################
-	 940.4 KiB/s
+	 4 MiB/s
 done
-Bytes transferred = 6204160 (5eab00 hex)
+Bytes transferred = 6203752 (5ea968 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 927.7 KiB/s
+	 4 MiB/s
 done
 Bytes transferred = 66544 (103f0 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5eab00 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5ea968 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830133ef
@@ -54,7 +54,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-rc2-next-20171206-17248-gd5c5286 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Dec 6 08:56:59 PST 2017
+Linux version 4.15.0-rc2-next-20171207-17835-ge40fd8d (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Dec 7 01:30:48 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,14 +69,14 @@
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
 random: fast init done
-percpu: Embedded 17 pages/cpu @(ptrval) s39296 r8192 d22144 u69632
-pcpu-alloc: s39296 r8192 d22144 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @(ptrval) s39104 r8192 d22336 u69632
+pcpu-alloc: s39104 r8192 d22336 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=b1791faa-84e8-4e27-83a1-9fb0a77e8803 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998492K/2096128K available (9216K kernel code, 778K rwdata, 2564K rodata, 1024K init, 298K bss, 32100K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998492K/2096128K available (9216K kernel code, 777K rwdata, 2564K rodata, 1024K init, 298K bss, 32100K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -86,7 +86,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0x(ptrval) - 0x(ptrval)   (10208 kB)
       .init : 0x(ptrval) - 0x(ptrval)   (1024 kB)
-      .data : 0x(ptrval) - 0x(ptrval)   ( 779 kB)
+      .data : 0x(ptrval) - 0x(ptrval)   ( 778 kB)
        .bss : 0x(ptrval) - 0x(ptrval)   ( 299 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
@@ -460,8 +460,8 @@
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x52 may corrupt adjacent RW1C bits
 pci 0000:00:01.0: PME# supported from D0 D1 D2 D3hot D3cold
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x4c may corrupt adjacent RW1C bits
-ata1: SATA link down (SStatus 0 SControl 300)
 pci 0000:00:02.0: [10de:0e13] type 01 class 0x060400
+ata1: SATA link down (SStatus 0 SControl 300)
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4 may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4 may corrupt adjacent RW1C bits
 pci 0000:00:02.0: enabling Extended Tags
@@ -479,21 +479,21 @@
 pci 0000:02:00.0: [10ec:8168] type 00 class 0x020000
 pci 0000:02:00.0: reg 0x10: [io  0x0000-0x00ff]
 pci 0000:02:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit]
-mmc1: new high speed SDHC card at address aaaa
-mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 pci 0000:02:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
 PCI: bus2: Fast back to back transfers disabled
 pci_bus 0000:02: busn_res: [bus 02-ff] end is updated to 02
+mmc1: new high speed SDHC card at address aaaa
+mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 pci 0000:00:01.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
 pci 0000:00:02.0: BAR 8: assigned [mem 0x13100000-0x131fffff]
 pci 0000:00:02.0: BAR 9: assigned [mem 0x20000000-0x200fffff 64bit pref]
 pci 0000:00:02.0: BAR 7: assigned [io  0x1000-0x1fff]
 pci 0000:01:00.0: BAR 0: assigned [mem 0x13000000-0x13001fff 64bit]
 pci 0000:00:01.0: PCI bridge to [bus 01]
- mmcblk1: p1 p2 p3
 pci 0000:00:01.0:   bridge window [mem 0x13000000-0x130fffff]
 pci 0000:02:00.0: BAR 4: assigned [mem 0x20000000-0x20003fff 64bit pref]
 pci 0000:02:00.0: BAR 2: assigned [mem 0x13100000-0x13100fff 64bit]
+ mmcblk1: p1 p2 p3
 pci 0000:02:00.0: BAR 0: assigned [io  0x1000-0x10ff]
 pci 0000:00:02.0: PCI bridge to [bus 02]
 pci 0000:00:02.0:   bridge window [io  0x1000-0x1fff]
@@ -508,7 +508,7 @@
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
 r8169 0000:02:00.0 (unnamed net_device) (uninitialized): not PCI Express
-r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xaba29bca, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 22
+r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xd30176d5, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 22
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
@@ -521,7 +521,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:18 UTC (946684818)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:13 UTC (946684813)
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig%tegra-fw_log.txt
--- /tmp/fooa	2017-12-07 01:50:15.283804288 -0800
+++ /tmp/foob	2017-12-07 01:50:15.287804306 -0800
@@ -33,9 +33,9 @@
 	 #################################################################
 	 #################################################################
 	 ############################################
-	 3.6 MiB/s
+	 4 MiB/s
 done
-Bytes transferred = 6369032 (612f08 hex)
+Bytes transferred = 6368864 (612e60 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -46,7 +46,7 @@
 done
 Bytes transferred = 66544 (103f0 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x612f08 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x612e60 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830133ef
@@ -54,7 +54,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-rc2-next-20171206-218956-gd5c5286 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Dec 6 08:57:08 PST 2017
+Linux version 4.15.0-rc2-next-20171207-219543-ge40fd8d (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Dec 7 01:31:04 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,14 +69,14 @@
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
 random: fast init done
-percpu: Embedded 17 pages/cpu @(ptrval) s39296 r8192 d22144 u69632
-pcpu-alloc: s39296 r8192 d22144 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @(ptrval) s39104 r8192 d22336 u69632
+pcpu-alloc: s39104 r8192 d22336 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=b1791faa-84e8-4e27-83a1-9fb0a77e8803 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1997468K/2096128K available (9216K kernel code, 778K rwdata, 2824K rodata, 1024K init, 298K bss, 33124K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1997468K/2096128K available (9216K kernel code, 777K rwdata, 2824K rodata, 1024K init, 298K bss, 33124K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -86,7 +86,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0x(ptrval) - 0x(ptrval)   (10208 kB)
       .init : 0x(ptrval) - 0x(ptrval)   (1024 kB)
-      .data : 0x(ptrval) - 0x(ptrval)   ( 779 kB)
+      .data : 0x(ptrval) - 0x(ptrval)   ( 778 kB)
        .bss : 0x(ptrval) - 0x(ptrval)   ( 299 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
@@ -413,8 +413,8 @@
 as3722-ldo5: supplied by +1.8V_VDDIO
 as3722-ldo6: supplied by +3.3V_RUN
 as3722-ldo7: supplied by +1.8V_VDDIO
-+1.05V_RUN_CAM_REAR: Bringing 0uV into 1050000-1050000uV
 input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input1
++1.05V_RUN_CAM_REAR: Bringing 0uV into 1050000-1050000uV
 as3722-ldo9: supplied by +5V_SYS
 +3.3V_RUN_TOUCH: Bringing 0uV into 2800000-2800000uV
 as3722-ldo10: supplied by +5V_SYS
@@ -513,7 +513,7 @@
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
 r8169 0000:02:00.0 (unnamed net_device) (uninitialized): not PCI Express
-r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0x7ecb696e, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 22
+r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf04ce41a, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 22
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
@@ -524,7 +524,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:01:57 UTC (946684917)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:09 UTC (946684929)
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 +USB0_VBUS_SW: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-12-07 01:50:15.291804324 -0800
+++ /tmp/foob	2017-12-07 01:50:15.295804342 -0800
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -45,7 +45,7 @@
 	 #################################
 	 3.5 MiB/s
 done
-Bytes transferred = 8114688 (7bd200 hex)
+Bytes transferred = 8110592 (7bc200 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.104
@@ -56,7 +56,7 @@
 done
 Bytes transferred = 67482 (1079a hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x7bd200 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x7bc200 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013799
@@ -64,7 +64,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-rc2-next-20171206-337197-gd5c52866f560 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Dec 6 08:57:44 PST 2017
+Linux version 4.15.0-rc2-next-20171207-337784-ge40fd8d6b4d9 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Dec 7 01:31:34 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -81,14 +81,14 @@
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327679 pages, LIFO batch:31
 random: fast init done
-percpu: Embedded 17 pages/cpu @(ptrval) s39232 r8192 d22208 u69632
-pcpu-alloc: s39232 r8192 d22208 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @(ptrval) s39040 r8192 d22400 u69632
+pcpu-alloc: s39040 r8192 d22400 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists, mobility grouping on.  Total pages: 522751
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=4d36e82a-b3c0-4b4b-bfcd-afe5217ac023 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993456K/2097148K available (11264K kernel code, 1603K rwdata, 4312K rodata, 2048K init, 404K bss, 38156K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1993456K/2097148K available (11264K kernel code, 1602K rwdata, 4312K rodata, 2048K init, 404K bss, 38156K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -98,7 +98,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0x(ptrval) - 0x(ptrval)   (12256 kB)
       .init : 0x(ptrval) - 0x(ptrval)   (2048 kB)
-      .data : 0x(ptrval) - 0x(ptrval)   (1604 kB)
+      .data : 0x(ptrval) - 0x(ptrval)   (1603 kB)
        .bss : 0x(ptrval) - 0x(ptrval)   ( 405 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -371,7 +371,6 @@
 mmc0: Invalid maximum block size, assuming 512 bytes
 mmc0: SDHCI controller on 700b0600.sdhci [700b0600.sdhci] using ADMA
 ledtrig-cpu: registered to indicate activity on CPUs
-elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
 NET: Registered protocol family 10
@@ -399,12 +398,19 @@
 +3.3V_PANEL: supplied by +3.3V_RUN
 +3.3V_LP0: supplied by +3.3V_SYS
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
+elan_i2c 1-0015: fetching report descriptor failed.: -121
 +5V_HDMI_CON: supplied by +5V_SYS
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
  usb3-1: usb3-1 supply vbus not found, using dummy regulator
+mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
+mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
+mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
+mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB, chardev (236:0)
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
 sdhci-tegra 700b0000.sdhci: allocated mmc-pwrseq
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 mmc1: Unknown controller version (3). You may experience problems.
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: Switching to 3.3V signalling voltage failed
@@ -413,13 +419,14 @@
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc2: Unknown controller version (3). You may experience problems.
 mmc2: Invalid maximum block size, assuming 512 bytes
+mmc1: new high speed SDIO card at address 0001
 mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:26:57 UTC (946686417)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 17:02:11 UTC (946746131)
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +1.05V_RUN: disabling
 +1.8V_RUN_CAM: disabling
@@ -436,22 +443,6 @@
 +3.3V_PANEL: disabling
 +1.05V_RUN_AVDD_HDMI_PLL: disabling
 +5V_HDMI_CON: disabling
-[   d as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input1
-Waiting for root device PARTUUID=4d36e82a-b3c0-4b4b-bfcd-afe5217ac023...
-tegra-sor 54540000.sor: failed to probe output: -517
-panel-simple panel: panel supply power not found, using dummy regulator
-elan_i2c 1-0015: invalid report id data (ff)
-mmc1: new high speed SDIO card at address 0001
-mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
-mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
-mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
-mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB, chardev (236:0)
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
-tegra-sor 54540000.sor: failed to probe output: -517
-panel-simple panel: panel supply power not found, using dummy regulator
-EXT4-fs (mmcblk0p3): couldn't mount as ext3 due to feature incompatibilities
-EXT4-fs (mmcblk0p3): couldn't mount as ext2 due to feature incompatibilities
 mmc2: new high speed SDHC card at address 0007
 mmcblk2: mmc2:0007 SD08G 7.42 GiB 
 GPT:Primary header alternate_lba != Alt. header my_lba
@@ -462,6 +453,13 @@
  mmcblk2: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
+elan_i2c 1-0015: device initialize failed: -121
+elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
+input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input1
+EXT4-fs (mmcblk0p3): couldn't mount as ext3 due to feature incompatibilities
+tegra-sor 54540000.sor: failed to probe output: -517
+panel-simple panel: panel supply power not found, using dummy regulator
+EXT4-fs (mmcblk0p3): couldn't mount as ext2 due to feature incompatibilities
 EXT4-fs (mmcblk0p3): recovery complete
 EXT4-fs (mmcblk0p3): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:3.
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2017-12-07 01:50:15.299804359 -0800
+++ /tmp/foob	2017-12-07 01:50:15.303804377 -0800
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -41,20 +41,20 @@
 	 #################################################################
 	 #################################################################
 	 #################################
-	 3.6 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 6204160 (5eab00 hex)
+Bytes transferred = 6203752 (5ea968 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.104
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.4 MiB/s
+	 3.6 MiB/s
 done
 Bytes transferred = 67482 (1079a hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5eab00 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5ea968 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013799
@@ -62,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-rc2-next-20171206-17248-gd5c5286 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Dec 6 08:56:59 PST 2017
+Linux version 4.15.0-rc2-next-20171207-17835-ge40fd8d (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Dec 7 01:30:48 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -78,14 +78,14 @@
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327679 pages, LIFO batch:31
 random: fast init done
-percpu: Embedded 17 pages/cpu @(ptrval) s39296 r8192 d22144 u69632
-pcpu-alloc: s39296 r8192 d22144 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @(ptrval) s39104 r8192 d22336 u69632
+pcpu-alloc: s39104 r8192 d22336 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists, mobility grouping on.  Total pages: 522751
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=4d36e82a-b3c0-4b4b-bfcd-afe5217ac023 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999512K/2097148K available (9216K kernel code, 778K rwdata, 2564K rodata, 1024K init, 298K bss, 32100K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1999512K/2097148K available (9216K kernel code, 777K rwdata, 2564K rodata, 1024K init, 298K bss, 32100K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -95,7 +95,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0x(ptrval) - 0x(ptrval)   (10208 kB)
       .init : 0x(ptrval) - 0x(ptrval)   (1024 kB)
-      .data : 0x(ptrval) - 0x(ptrval)   ( 779 kB)
+      .data : 0x(ptrval) - 0x(ptrval)   ( 778 kB)
        .bss : 0x(ptrval) - 0x(ptrval)   ( 299 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
@@ -363,16 +363,16 @@
 iommu: Adding device snd-soc-dummy to group 74
 iommu: Adding device 70301100.i2s to group 75
 tegra30-i2s 70301100.i2s: DMA channels sourced from device 70300000.ahub
-max98090 0-0010: MAX98090 REVID=0x43
-max98090 0-0010: use default 2.8v micbias
-tegra-snd-max98090 sound: HiFi <-> 70301100.i2s mapping ok
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
-input: GoogleNyanBig Headphones as /devices/soc0/sound/sound/card1/input1
+max98090 0-0010: MAX98090 REVID=0x43
 mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
 mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB, chardev (247:0)
+max98090 0-0010: use default 2.8v micbias
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
+tegra-snd-max98090 sound: HiFi <-> 70301100.i2s mapping ok
+input: GoogleNyanBig Headphones as /devices/soc0/sound/sound/card1/input1
 input: GoogleNyanBig Mic Jack as /devices/soc0/sound/sound/card1/input2
 NET: Registered protocol family 10
 Segment Routing with IPv6
@@ -431,7 +431,7 @@
 input: gpio-keys as /devices/soc0/gpio-keys/input/input3
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:22:49 UTC (946686169)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 16:57:24 UTC (946745844)
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
@@ -456,11 +456,11 @@
 mmcblk2: mmc2:0007 SD08G 7.42 GiB 
 GPT:Primary header alternate_lba != Alt. header my_lba
 GPT:3014591 != 15564799
+GPT:Primary header thinks Alt. header is not at the end of the disk.
+GPT:3014591 != 15564799
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
 input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input4
-GPT:Primary header thinks Alt. header is not at the end of the disk.
-GPT:3014591 != 15564799
 GPT: Use GNU Parted to correct GPT errors.
  mmcblk2: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 tegra-sor 54540000.sor: failed to probe output: -517
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig%tegra-fw_log.txt
--- /tmp/fooa	2017-12-07 01:50:15.307804395 -0800
+++ /tmp/foob	2017-12-07 01:50:15.311804413 -0800
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -41,20 +41,20 @@
 	 #################################################################
 	 #################################################################
 	 ############################################
-	 3.5 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 6369032 (612f08 hex)
+Bytes transferred = 6368864 (612e60 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.104
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.6 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 67482 (1079a hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x612f08 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x612e60 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013799
@@ -62,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-rc2-next-20171206-218956-gd5c5286 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Dec 6 08:57:08 PST 2017
+Linux version 4.15.0-rc2-next-20171207-219543-ge40fd8d (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Dec 7 01:31:04 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -78,14 +78,14 @@
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327679 pages, LIFO batch:31
 random: fast init done
-percpu: Embedded 17 pages/cpu @(ptrval) s39296 r8192 d22144 u69632
-pcpu-alloc: s39296 r8192 d22144 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @(ptrval) s39104 r8192 d22336 u69632
+pcpu-alloc: s39104 r8192 d22336 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists, mobility grouping on.  Total pages: 522751
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=4d36e82a-b3c0-4b4b-bfcd-afe5217ac023 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998488K/2097148K available (9216K kernel code, 778K rwdata, 2824K rodata, 1024K init, 298K bss, 33124K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1998488K/2097148K available (9216K kernel code, 777K rwdata, 2824K rodata, 1024K init, 298K bss, 33124K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -95,7 +95,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0x(ptrval) - 0x(ptrval)   (10208 kB)
       .init : 0x(ptrval) - 0x(ptrval)   (1024 kB)
-      .data : 0x(ptrval) - 0x(ptrval)   ( 779 kB)
+      .data : 0x(ptrval) - 0x(ptrval)   ( 778 kB)
        .bss : 0x(ptrval) - 0x(ptrval)   ( 299 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
@@ -363,16 +363,16 @@
 iommu: Adding device snd-soc-dummy to group 74
 iommu: Adding device 70301100.i2s to group 75
 tegra30-i2s 70301100.i2s: DMA channels sourced from device 70300000.ahub
-max98090 0-0010: MAX98090 REVID=0x43
-max98090 0-0010: use default 2.8v micbias
-tegra-snd-max98090 sound: HiFi <-> 70301100.i2s mapping ok
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
-input: GoogleNyanBig Headphones as /devices/soc0/sound/sound/card1/input1
+max98090 0-0010: MAX98090 REVID=0x43
 mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
 mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB, chardev (247:0)
+max98090 0-0010: use default 2.8v micbias
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
+tegra-snd-max98090 sound: HiFi <-> 70301100.i2s mapping ok
+input: GoogleNyanBig Headphones as /devices/soc0/sound/sound/card1/input1
 input: GoogleNyanBig Mic Jack as /devices/soc0/sound/sound/card1/input2
 NET: Registered protocol family 10
 Segment Routing with IPv6
@@ -439,7 +439,7 @@
 input: gpio-keys as /devices/soc0/gpio-keys/input/input3
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:23:27 UTC (946686207)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 16:58:05 UTC (946745885)
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
@@ -458,10 +458,10 @@
 ALSA device list:
   #1: GoogleNyanBig
 cfg80211: failed to load regulatory.db
-tegra-sor 54540000.sor: failed to probe output: -517
-panel-simple panel: panel supply power not found, using dummy regulator
 input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input4
 EXT4-fs (mmcblk0p3): couldn't mount as ext3 due to feature incompatibilities
+tegra-sor 54540000.sor: failed to probe output: -517
+panel-simple panel: panel supply power not found, using dummy regulator
 usb 1-2: new high-speed USB device number 2 using tegra-xusb
 mmc2: new high speed SDHC card at address 0007
 mmcblk2: mmc2:0007 SD08G 7.42 GiB 
@@ -473,10 +473,6 @@
  mmcblk2: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
-uvcvideo: Found UVC 1.00 device HD WebCam (1bcf:2c67)
-input: HD WebCam: HD WebCam as /devices/soc0/70090000.usb/usb1/1-2/1-2:1.0/input/input5
-tegra-sor 54540000.sor: failed to probe output: -517
-panel-simple panel: panel supply power not found, using dummy regulator
 EXT4-fs (mmcblk0p3): recovery complete
 EXT4-fs (mmcblk0p3): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:3.
@@ -484,7 +480,7 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.424401] usb 1-3: new high-speed USB device number 3 using tegra-xusb
-asix 1-3:1.0 eth0: register 'asix' at usb-70090000.usb-3, ASIX AX88772 USB 2.0 Ethernet, c8:d7:19:90:fa:65
+root@(none):/# [    3.057012] uvcvideo: Found UVC 1.00 device HD WebCam (1bcf:2c67)
+input: HD WebCam: HD WebCam as /devices/soc0/70090000.usb/usb1/1-2/1-2:1.0/input/input5
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2017-12-07 01:50:15.315804431 -0800
+++ /tmp/foob	2017-12-07 01:50:15.319804450 -0800
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1801347 us
+Bootloader-Cpu Init at (time stamp): 1801359 us
 
 Pinmux changes applied in kernel way
 
@@ -59,25 +59,25 @@
 The proc BoardInfo: 0x0176:0x0001:0x01:0x43:0x00
 FastbootAddAtag: setting ATAG (0)
 
-Board Id Read Time: 3 us
+Board Id Read Time: 4 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1801321  us
-Bootloader (AosInit) time: 733472 us
-Bootloader Main Init time: 180216 us
-BlockDevMgrInit time: 157560 us
+Miscellaneous avp time: 1801332  us
+Bootloader (AosInit) time: 733468 us
+Bootloader Main Init time: 180045 us
+BlockDevMgrInit time: 157523 us
 PartMgrInit time: 8 us
 StorMgrInit time: 30 us
-FsMgrInit time: 3 us
-Partition Table load: 77015 us 
-Display init time: 574537 us
+FsMgrInit time: 2 us
+Partition Table load: 77044 us 
+Display init time: 574532 us
 Kernel read time: 1546 msec
-Load Kernel Size:   20484096 Bytes
-Bl Kernel ReadSpeed: 13249000 Bytes/sec
-Pre-Kernel Jump time: 4333564 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 9497655 us
+Load Kernel Size:   20482048 Bytes
+Bl Kernel ReadSpeed: 13248000 Bytes/sec
+Pre-Kernel Jump time: 4333346 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 9499517 us
 Booting Linux on physical CPU 0x0000000000 [0x4e0f0000]
-Linux version 4.15.0-rc2-next-20171206-337197-gd5c52866f560 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Dec 6 09:04:15 PST 2017
+Linux version 4.15.0-rc2-next-20171207-337784-ge40fd8d6b4d9 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Thu Dec 7 01:37:31 PST 2017
 Machine model: NVIDIA Tegra132 Norrin
 Memory limited to 1734MB
 efi: Getting EFI parameters from FDT:
@@ -95,20 +95,20 @@
 Initmem setup node 0 [mem 0x0000000080000000-0x00000000ec5fffff]
 /cpus/cpu@1: missing enable-method property
 random: fast init done
-percpu: Embedded 23 pages/cpu @        (ptrval) s55936 r8192 d30080 u94208
+percpu: Embedded 23 pages/cpu @        (ptrval) s55168 r8192 d30848 u94208
 Detected PIPT I-cache on CPU0
 Built 1 zonelists, mobility grouping on.  Total pages: 436968
 Policy zone: DMA
 Kernel command line: tegraid=13.1.3.0.0 mem=1734M@2048M memtype=255 vpr=151M@3817M ddr_die=2048M@2048M section=256M commchip_id=0 usb_port_owner_info=0 lane_owner_info=0 emc_max_dvfs=0 touch_id=0@0 video=tegrafb no_console_suspend=1 console=ttyS0,115200n8 debug_uartport=lsport,0 earlyprintk=uart8250-32bit,0x70006000 usbcore.old_scheme_first=1 lp0_vec=0@0x0 tegra_fbmem=32899072@0xad012000 core_edp_mv=1150 core_edp_ma=4000 pmuboard=0x0176:0x0001:0x01:0x43:0x00 displayboard=0x0162:0x0001:0x00:0x42:0x00 power_supply=Adapter audio_codec=rt5640 board_info=0x0176:0x0001:0x01:0x43:0x00 tegraboot=sdmmc gpt gpt_sector=192511 modem_id=0 android.kerneltype=normal
-Memory: 1705656K/1775616K available (9724K kernel code, 1162K rwdata, 4484K rodata, 1152K init, 394K bss, 53576K reserved, 16384K cma-reserved)
+Memory: 1705656K/1775616K available (9724K kernel code, 1160K rwdata, 4484K rodata, 1152K init, 396K bss, 53576K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0x        (ptrval) - 0x        (ptrval)   (  9728 KB)
     .rodata : 0x        (ptrval) - 0x        (ptrval)   (  4544 KB)
       .init : 0x        (ptrval) - 0x        (ptrval)   (  1152 KB)
-      .data : 0x        (ptrval) - 0x        (ptrval)   (  1163 KB)
-       .bss : 0x        (ptrval) - 0x        (ptrval)   (   395 KB)
+      .data : 0x        (ptrval) - 0x        (ptrval)   (  1161 KB)
+       .bss : 0x        (ptrval) - 0x        (ptrval)   (   397 KB)
     fixed   : 0xffff7dfffe7fb000 - 0xffff7dfffec00000   (  4116 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -148,9 +148,9 @@
 DMI not present or invalid.
 NET: Registered protocol family 16
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.136:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.132:1): state=initialized audit_enabled=0 res=1
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ 00000000db9e0688, 1 data @ 0000000095b0383e)
+vdso: 2 pages (1 code @ 000000003552e16d, 1 data @ 00000000291388af)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -283,9 +283,12 @@
 +5V_HDMI_CON: supplied by +5V_SYS
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
-input: gpio-keys as /devices/platform/gpio-keys/input/input0
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
+sdhci-tegra 700b0400.sdhci: Got WP GPIO
+input: gpio-keys as /devices/platform/gpio-keys/input/input0
 hctosys: unable to open rtc device (rtc0)
+sdhci-tegra 700b0400.sdhci: Got CD GPIO
+sdhci-tegra 700b0400.sdhci: Got WP GPIO
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +VDD_LED: disabling
 +5V_USB_HS: disabling
@@ -295,7 +298,6 @@
 ALSA device list:
   No soundcards found.
 Freeing unused kernel memory: 1152K
-sdhci-tegra 700b0400.sdhci: Got WP GPIO
 
 System initialization...
 
Diff for ./tegra186-p2771-0000/tegra186-p2771-0000/defconfig%jetson-tx2_log.txt
Binary files /tmp/fooa and /tmp/foob differ
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-12-07 01:50:15.335804520 -0800
+++ /tmp/foob	2017-12-07 01:50:15.339804538 -0800
@@ -48,20 +48,20 @@
 	 #################################################################
 	 #################################################################
 	 #################################
-	 3.6 MiB/s
+	 4.1 MiB/s
 done
-Bytes transferred = 8114688 (7bd200 hex)
+Bytes transferred = 8110592 (7bc200 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 2.9 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21160 (52a8 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x7bd200 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x7bc200 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102a7
@@ -69,7 +69,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-rc2-next-20171206-337197-gd5c52866f560 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Dec 6 08:57:44 PST 2017
+Linux version 4.15.0-rc2-next-20171207-337784-ge40fd8d6b4d9 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Dec 7 01:31:34 PST 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -84,14 +84,14 @@
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 65536 pages, LIFO batch:15
 random: fast init done
-percpu: Embedded 17 pages/cpu @(ptrval) s39232 r8192 d22208 u69632
-pcpu-alloc: s39232 r8192 d22208 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @(ptrval) s39040 r8192 d22400 u69632
+pcpu-alloc: s39040 r8192 d22400 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 
 Built 1 zonelists, mobility grouping on.  Total pages: 260608
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=11eb23c1-01 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 953376K/1048576K available (11264K kernel code, 1603K rwdata, 4312K rodata, 2048K init, 404K bss, 29664K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 953376K/1048576K available (11264K kernel code, 1602K rwdata, 4312K rodata, 2048K init, 404K bss, 29664K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -101,7 +101,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0x(ptrval) - 0x(ptrval)   (12256 kB)
       .init : 0x(ptrval) - 0x(ptrval)   (2048 kB)
-      .data : 0x(ptrval) - 0x(ptrval)   (1604 kB)
+      .data : 0x(ptrval) - 0x(ptrval)   (1603 kB)
        .bss : 0x(ptrval) - 0x(ptrval)   ( 405 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
@@ -267,7 +267,7 @@
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:01:00.0: enabling device (0140 -> 0143)
 r8169 0000:01:00.0 (unnamed net_device) (uninitialized): not PCI Express
-r8169 0000:01:00.0 eth0: RTL8168d/8111d at 0xffe241bc, 00:01:c0:13:7d:63, XID 081000c0 IRQ 286
+r8169 0000:01:00.0 eth0: RTL8168d/8111d at 0xed50d215, 00:01:c0:13:7d:63, XID 081000c0 IRQ 286
 r8169 0000:01:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 pegasus: v0.9.3 (2013/04/25), Pegasus/Pegasus II USB Ethernet driver
 usbcore: registered new interface driver pegasus
@@ -342,9 +342,9 @@
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
-tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
+tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
  mmcblk1: p1 p2
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2017-12-07 01:50:15.343804556 -0800
+++ /tmp/foob	2017-12-07 01:50:15.347804574 -0800
@@ -48,7 +48,7 @@
 	 #################################
 	 4.1 MiB/s
 done
-Bytes transferred = 6204160 (5eab00 hex)
+Bytes transferred = 6203752 (5ea968 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -59,7 +59,7 @@
 done
 Bytes transferred = 21160 (52a8 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x5eab00 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x5ea968 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102a7
@@ -67,7 +67,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-rc2-next-20171206-17248-gd5c5286 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Dec 6 08:56:59 PST 2017
+Linux version 4.15.0-rc2-next-20171207-17835-ge40fd8d (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Dec 7 01:30:48 PST 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -81,14 +81,14 @@
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 65536 pages, LIFO batch:15
 random: fast init done
-percpu: Embedded 17 pages/cpu @(ptrval) s39296 r8192 d22144 u69632
-pcpu-alloc: s39296 r8192 d22144 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @(ptrval) s39104 r8192 d22336 u69632
+pcpu-alloc: s39104 r8192 d22336 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 
 Built 1 zonelists, mobility grouping on.  Total pages: 260608
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=11eb23c1-01 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 959424K/1048576K available (9216K kernel code, 778K rwdata, 2564K rodata, 1024K init, 298K bss, 23616K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 959424K/1048576K available (9216K kernel code, 777K rwdata, 2564K rodata, 1024K init, 298K bss, 23616K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -98,7 +98,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0x(ptrval) - 0x(ptrval)   (10208 kB)
       .init : 0x(ptrval) - 0x(ptrval)   (1024 kB)
-      .data : 0x(ptrval) - 0x(ptrval)   ( 779 kB)
+      .data : 0x(ptrval) - 0x(ptrval)   ( 778 kB)
        .bss : 0x(ptrval) - 0x(ptrval)   ( 299 kB)
 Preemptible hierarchical RCU implementation.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
@@ -258,7 +258,7 @@
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:01:00.0: enabling device (0140 -> 0143)
 r8169 0000:01:00.0 (unnamed net_device) (uninitialized): not PCI Express
-r8169 0000:01:00.0 eth0: RTL8168d/8111d at 0xffbfc241, 00:01:c0:13:7d:63, XID 081000c0 IRQ 286
+r8169 0000:01:00.0 eth0: RTL8168d/8111d at 0x466d1bb9, 00:01:c0:13:7d:63, XID 081000c0 IRQ 286
 r8169 0000:01:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 pegasus: v0.9.3 (2013/04/25), Pegasus/Pegasus II USB Ethernet driver
 usbcore: registered new interface driver pegasus
@@ -329,16 +329,16 @@
 Registering SWP/SWPB emulation handler
 Loading compiled-in X.509 certificates
 mmc1: new high speed SD card at address b368
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
-hub 2-0:1.0: 1 port detected
  mmcblk1: p1 p2
+hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
 rtc-em3027 2-0056: hctosys: unable to read the hardware clock
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig%jetson-tx1_log.txt
--- /tmp/fooa	2017-12-07 01:50:15.355804610 -0800
+++ /tmp/foob	2017-12-07 01:50:15.355804610 -0800
@@ -1,7 +1,7 @@
 [TegraBoot] (version 23.00.2015.14-mobile-1ef66670)
 Processing in cold boot mode Bootloader 2
 A01Q Bootrom Patch rev = 3
-Power-up reason: on button
+Power-up reason: pmu watchdog
 No Battery Present
 Platform has Ddr4 type ram
 max77620 disabling SD1 Remote Sense
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 779959 us
-Basic modules init took 348471 us
-Sec Bootdevice Read Time = 204 ms, Read Size = 8459 KB
-Next stage binary read took 15664 us
-Carveout took 252662 us
-CPU initialization took 92470 us
-Total time taken by TegraBoot 709267 us
+Tegraboot started after 828673 us
+Basic modules init took 350928 us
+Sec Bootdevice Read Time = 203 ms, Read Size = 8459 KB
+Next stage binary read took 14365 us
+Carveout took 252663 us
+CPU initialization took 92450 us
+Total time taken by TegraBoot 710406 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.619] LPDDR4 Training: Number of tables = 10
+64b[0001.668] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig%jetson-tx1
-17115648 bytes read in 866 ms (18.8 MiB/s)
+17115648 bytes read in 868 ms (18.8 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
 44764 bytes read in 453 ms (95.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 495 ms (3.1 MiB/s)
+1627787 bytes read in 496 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0000000000 [0x411fd071]
-Linux version 4.15.0-rc2-next-20171206-218956-gd5c5286 (jonathanh@sellafield) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Dec 6 09:00:26 PST 2017
+Linux version 4.15.0-rc2-next-20171207-17835-ge40fd8d (jonathanh@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Thu Dec 7 01:34:29 PST 2017
 Machine model: NVIDIA Tegra210 P2371 (P2530/P2595) reference design
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -161,8 +161,8 @@
 psci: Using standard PSCI v0.2 function IDs
 psci: MIGRATE_INFO_TYPE not supported.
 random: fast init done
-percpu: Embedded 23 pages/cpu @        (ptrval) s55936 r8192 d30080 u94208
-pcpu-alloc: s55936 r8192 d30080 u94208 alloc=23*4096
+percpu: Embedded 23 pages/cpu @        (ptrval) s55168 r8192 d30848 u94208
+pcpu-alloc: s55168 r8192 d30848 u94208 alloc=23*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
@@ -233,7 +233,7 @@
 audit: initializing netlink subsys (disabled)
 audit: type=2000 audit(0.268:1): state=initialized audit_enabled=0 res=1
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ 0000000001e36287, 1 data @ 0000000055efed58)
+vdso: 2 pages (1 code @ 00000000e63e71e1, 1 data @ 000000007aa59d0c)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2017-12-07 01:50:15.363804645 -0800
+++ /tmp/foob	2017-12-07 01:50:15.367804664 -0800
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 736466 us
-Basic modules init took 348450 us
-Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14372 us
-Carveout took 252659 us
-CPU initialization took 92454 us
-Total time taken by TegraBoot 707935 us
+Tegraboot started after 796831 us
+Basic modules init took 348594 us
+Sec Bootdevice Read Time = 203 ms, Read Size = 8459 KB
+Next stage binary read took 14371 us
+Carveout took 253437 us
+CPU initialization took 92433 us
+Total time taken by TegraBoot 708835 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.575] LPDDR4 Training: Number of tables = 10
+64b[0001.635] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-16984576 bytes read in 850 ms (19.1 MiB/s)
+16982528 bytes read in 851 ms (19 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
 44764 bytes read in 453 ms (95.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 494 ms (3.1 MiB/s)
+1627787 bytes read in 495 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0000000000 [0x411fd071]
-Linux version 4.15.0-rc2-next-20171206-337197-gd5c52866f560 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Dec 6 09:04:15 PST 2017
+Linux version 4.15.0-rc2-next-20171207-337784-ge40fd8d6b4d9 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Thu Dec 7 01:37:31 PST 2017
 Machine model: NVIDIA Tegra210 P2371 (P2530/P2595) reference design
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -161,8 +161,8 @@
 psci: Using standard PSCI v0.2 function IDs
 psci: MIGRATE_INFO_TYPE not supported.
 random: fast init done
-percpu: Embedded 23 pages/cpu @        (ptrval) s55936 r8192 d30080 u94208
-pcpu-alloc: s55936 r8192 d30080 u94208 alloc=23*4096
+percpu: Embedded 23 pages/cpu @        (ptrval) s55168 r8192 d30848 u94208
+pcpu-alloc: s55168 r8192 d30848 u94208 alloc=23*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
@@ -171,15 +171,15 @@
 Policy zone: Normal
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [        (ptrval)-        (ptrval)]
-Memory: 2982164K/3131392K available (9724K kernel code, 1162K rwdata, 4484K rodata, 1152K init, 394K bss, 132844K reserved, 16384K cma-reserved)
+Memory: 2982164K/3131392K available (9724K kernel code, 1160K rwdata, 4484K rodata, 1152K init, 396K bss, 132844K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0x        (ptrval) - 0x        (ptrval)   (  9728 KB)
     .rodata : 0x        (ptrval) - 0x        (ptrval)   (  4544 KB)
       .init : 0x        (ptrval) - 0x        (ptrval)   (  1152 KB)
-      .data : 0x        (ptrval) - 0x        (ptrval)   (  1163 KB)
-       .bss : 0x        (ptrval) - 0x        (ptrval)   (   395 KB)
+      .data : 0x        (ptrval) - 0x        (ptrval)   (  1161 KB)
+       .bss : 0x        (ptrval) - 0x        (ptrval)   (   397 KB)
     fixed   : 0xffff7dfffe7fb000 - 0xffff7dfffec00000   (  4116 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -233,7 +233,7 @@
 audit: initializing netlink subsys (disabled)
 audit: type=2000 audit(0.268:1): state=initialized audit_enabled=0 res=1
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ 0000000016d2307e, 1 data @ 00000000b79b7f5a)
+vdso: 2 pages (1 code @ 00000000ff3a6727, 1 data @ 00000000cae6e1c7)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
Diff for ./tegra210-p2371-2180/tegra210-p2371-2180/defconfig%jetson-tx1_log.txt
Binary files /tmp/fooa and /tmp/foob differ
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2017-12-07 01:50:15.379804716 -0800
+++ /tmp/foob	2017-12-07 01:50:15.379804716 -0800
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0000000000 [0x411fd071]
-Linux version 4.15.0-rc2-next-20171206-337197-gd5c52866f560 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Dec 6 09:04:15 PST 2017
+Linux version 4.15.0-rc2-next-20171207-337784-ge40fd8d6b4d9 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Thu Dec 7 01:37:31 PST 2017
 Machine model: Google Pixel C
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
@@ -23,7 +23,7 @@
 psci: Using standard PSCI v0.2 function IDs
 psci: MIGRATE_INFO_UP_CPU reported invalid physical ID (0x80000000)
 random: get_random_bytes called from start_kernel+0xb0/0x410 with crng_init=0
-percpu: Embedded 23 pages/cpu @        (ptrval) s55936 r8192 d30080 u94208
+percpu: Embedded 23 pages/cpu @        (ptrval) s55168 r8192 d30848 u94208
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
 CPU features: enabling workaround for ARM erratum 834220
@@ -31,15 +31,15 @@
 Policy zone: Normal
 Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [        (ptrval)-        (ptrval)]
-Memory: 2835308K/2988032K available (9724K kernel code, 1162K rwdata, 4484K rodata, 1152K init, 394K bss, 136340K reserved, 16384K cma-reserved)
+Memory: 2835312K/2988032K available (9724K kernel code, 1160K rwdata, 4484K rodata, 1152K init, 396K bss, 136336K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0x        (ptrval) - 0x        (ptrval)   (  9728 KB)
     .rodata : 0x        (ptrval) - 0x        (ptrval)   (  4544 KB)
       .init : 0x        (ptrval) - 0x        (ptrval)   (  1152 KB)
-      .data : 0x        (ptrval) - 0x        (ptrval)   (  1163 KB)
-       .bss : 0x        (ptrval) - 0x        (ptrval)   (   395 KB)
+      .data : 0x        (ptrval) - 0x        (ptrval)   (  1161 KB)
+       .bss : 0x        (ptrval) - 0x        (ptrval)   (   397 KB)
     fixed   : 0xffff7dfffe7fb000 - 0xffff7dfffec00000   (  4116 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -110,7 +110,7 @@
 audit: initializing netlink subsys (disabled)
 audit: type=2000 audit(0.180:1): state=initialized audit_enabled=0 res=1
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ 00000000f66f83a1, 1 data @ 00000000dbb890bb)
+vdso: 2 pages (1 code @ 00000000817f72f3, 1 data @ 00000000c2b62099)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -151,7 +151,7 @@
 RPC: Registered tcp transport module.
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 Unpacking initramfs...
-Freeing initrd memory: 3412K
+Freeing initrd memory: 3408K
 kvm [1]: 8-bit VMID
 kvm [1]: IDMAP page: 809e9000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
@@ -267,7 +267,7 @@
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB, chardev (240:0)
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
-max77686-rtc max77620-rtc: setting system clock to 2001-09-16 22:33:08 UTC (1000679588)
+max77686-rtc max77620-rtc: setting system clock to 2001-09-17 15:07:41 UTC (1000739261)
 PP1200_AVDD: disabling
 PP1200_RCAM: disabling
 PP2800L_RCAM: disabling
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-12-07 01:50:15.383804734 -0800
+++ /tmp/foob	2017-12-07 01:50:15.387804752 -0800
@@ -40,20 +40,20 @@
 	 #################################################################
 	 #################################################################
 	 #################################
-	 2.9 MiB/s
+	 2.8 MiB/s
 done
-Bytes transferred = 8114688 (7bd200 hex)
+Bytes transferred = 8110592 (7bc200 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 163.1 KiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 51470 (c90e hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x7bd200 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x7bc200 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90d ... OK
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-rc2-next-20171206-337197-gd5c52866f560 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Dec 6 08:57:44 PST 2017
+Linux version 4.15.0-rc2-next-20171207-337784-ge40fd8d6b4d9 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Dec 7 01:31:34 PST 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -76,14 +76,14 @@
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
 random: fast init done
-percpu: Embedded 17 pages/cpu @(ptrval) s39232 r8192 d22208 u69632
-pcpu-alloc: s39232 r8192 d22208 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @(ptrval) s39040 r8192 d22400 u69632
+pcpu-alloc: s39040 r8192 d22400 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=edbe1840-3633-c751-7993-b0ac834f81e0 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1992476K/2096128K available (11264K kernel code, 1603K rwdata, 4312K rodata, 2048K init, 404K bss, 38116K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1992476K/2096128K available (11264K kernel code, 1602K rwdata, 4312K rodata, 2048K init, 404K bss, 38116K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -93,7 +93,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0x(ptrval) - 0x(ptrval)   (12256 kB)
       .init : 0x(ptrval) - 0x(ptrval)   (2048 kB)
-      .data : 0x(ptrval) - 0x(ptrval)   (1604 kB)
+      .data : 0x(ptrval) - 0x(ptrval)   (1603 kB)
        .bss : 0x(ptrval) - 0x(ptrval)   ( 405 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -433,7 +433,7 @@
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:01:00.0: enabling device (0140 -> 0143)
 r8169 0000:01:00.0 (unnamed net_device) (uninitialized): not PCI Express
-r8169 0000:01:00.0 eth0: RTL8168evl/8111evl at 0x161cc915, 02:04:4b:01:60:a6, XID 0c900800 IRQ 18
+r8169 0000:01:00.0 eth0: RTL8168evl/8111evl at 0x3e5caf55, 02:04:4b:01:60:a6, XID 0c900800 IRQ 18
 r8169 0000:01:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
@@ -464,4 +464,4 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.444087] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.434075] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2017-12-07 01:50:15.391804770 -0800
+++ /tmp/foob	2017-12-07 01:50:15.395804788 -0800
@@ -38,20 +38,20 @@
 	 #################################################################
 	 #################################################################
 	 #################################
-	 3.2 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 6204160 (5eab00 hex)
+Bytes transferred = 6203752 (5ea968 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 51470 (c90e hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5eab00 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5ea968 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90d ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-rc2-next-20171206-17248-gd5c5286 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Dec 6 08:56:59 PST 2017
+Linux version 4.15.0-rc2-next-20171207-17835-ge40fd8d (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Dec 7 01:30:48 PST 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -73,14 +73,14 @@
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
 random: fast init done
-percpu: Embedded 17 pages/cpu @(ptrval) s39296 r8192 d22144 u69632
-pcpu-alloc: s39296 r8192 d22144 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @(ptrval) s39104 r8192 d22336 u69632
+pcpu-alloc: s39104 r8192 d22336 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=edbe1840-3633-c751-7993-b0ac834f81e0 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998528K/2096128K available (9216K kernel code, 778K rwdata, 2564K rodata, 1024K init, 298K bss, 32064K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998528K/2096128K available (9216K kernel code, 777K rwdata, 2564K rodata, 1024K init, 298K bss, 32064K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,7 +90,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0x(ptrval) - 0x(ptrval)   (10208 kB)
       .init : 0x(ptrval) - 0x(ptrval)   (1024 kB)
-      .data : 0x(ptrval) - 0x(ptrval)   ( 779 kB)
+      .data : 0x(ptrval) - 0x(ptrval)   ( 778 kB)
        .bss : 0x(ptrval) - 0x(ptrval)   ( 299 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
@@ -428,7 +428,7 @@
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:01:00.0: enabling device (0140 -> 0143)
 r8169 0000:01:00.0 (unnamed net_device) (uninitialized): not PCI Express
-r8169 0000:01:00.0 eth0: RTL8168evl/8111evl at 0x8128bc93, 02:04:4b:01:60:a6, XID 0c900800 IRQ 18
+r8169 0000:01:00.0 eth0: RTL8168evl/8111evl at 0xe169829b, 02:04:4b:01:60:a6, XID 0c900800 IRQ 18
 r8169 0000:01:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
@@ -465,4 +465,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.687601] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.677669] usb 2-1: new high-speed USB device number 2 using tegra-ehci
