Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2015-09-17 21:53:46.996917612 -0700
+++ /tmp/foob	2015-09-17 21:53:47.000917608 -0700
@@ -1,6 +1,6 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
-Linux version 4.3.0-rc1-next-20150917+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Sep 16 20:49:04 PDT 2015
+Linux version 4.3.0-rc1-next-20150918+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Sep 17 21:48:56 PDT 2015
 CPU: AArch64 Processor [411fd070] revision 0
 Detected PIPT I-cache on CPU0
 alternatives: enabling workaround for ARM erratum 832075
@@ -20,7 +20,7 @@
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
 software IO TLB [mem 0x79c00000-0x7dc00000] (64MB) mapped at [ffffffc039c00000-ffffffc03dbfffff]
-Memory: 939156K/1048576K available (5397K kernel code, 471K rwdata, 2256K rodata, 512K init, 203K bss, 93036K reserved, 16384K cma-reserved)
+Memory: 939136K/1048576K available (5397K kernel code, 482K rwdata, 2260K rodata, 512K init, 204K bss, 93056K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     vmalloc : 0xffffff8000000000 - 0xffffffbdbfff0000   (   246 GB)
     vmemmap : 0xffffffbdc0000000 - 0xffffffbfc0000000   (     8 GB maximum)
@@ -29,9 +29,9 @@
     PCI I/O : 0xffffffbffae00000 - 0xffffffbffbe00000   (    16 MB)
     modules : 0xffffffbffc000000 - 0xffffffc000000000   (    64 MB)
     memory  : 0xffffffc000000000 - 0xffffffc040000000   (  1024 MB)
-      .init : 0xffffffc0007fc000 - 0xffffffc00087c000   (   512 KB)
-      .text : 0xffffffc000080000 - 0xffffffc0007fb524   (  7662 KB)
-      .data : 0xffffffc00088c000 - 0xffffffc000901c00   (   471 KB)
+      .init : 0xffffffc0007fd000 - 0xffffffc00087d000   (   512 KB)
+      .text : 0xffffffc000080000 - 0xffffffc0007fc564   (  7666 KB)
+      .data : 0xffffffc00088e000 - 0xffffffc000906800   (   482 KB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 64.
@@ -62,7 +62,7 @@
 NET: Registered protocol family 16
 cpuidle: using governor ladder
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffffffc000891000, 1 data @ ffffffc000890000)
+vdso: 2 pages (1 code @ ffffffc000895000, 1 data @ ffffffc000894000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -90,7 +90,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 2, 16384 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.610:1): initialized
+audit: type=2000 audit(0.590:1): initialized
 HugeTLB registered 2 MB page size, pre-allocated 0 pages
 VFS: Disk quotas dquot_6.6.0
 VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
@@ -134,8 +134,8 @@
 EXT4-fs (vda2): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 254:2.
 devtmpfs: mounted
-Freeing unused kernel memory: 512K (ffffffc0007fc000 - ffffffc00087c000)
-Freeing alternatives memory: 44K (ffffffc00087c000 - ffffffc000887000)
+Freeing unused kernel memory: 512K (ffffffc0007fd000 - ffffffc00087d000)
+Freeing alternatives memory: 48K (ffffffc00087d000 - ffffffc000889000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 bash-4.3# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-09-17 21:53:47.004917605 -0700
+++ /tmp/foob	2015-09-17 21:53:47.008917601 -0700
@@ -38,9 +38,9 @@
 	 #################################################################
 	 #################################################################
 	 #################################################
-	 3.2 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 6433872 (622c50 hex)
+Bytes transferred = 6433712 (622bb0 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -51,7 +51,7 @@
 done
 Bytes transferred = 34966 (8896 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x622c50 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x622bb0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff895 ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.3.0-rc1-next-20150917-42145-g01c8787 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Sep 16 20:47:46 PDT 2015
+Linux version 4.3.0-rc1-next-20150918-42418-g887e5d0 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Sep 17 21:47:41 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -67,30 +67,30 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f7aa00, node_mem_map ee7f8000
-  DMA zone: 1520 pages used for memmap
+free_area_init_node: node 0, pgdat c0f7ad80, node_mem_map eeff8000
+  DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
-  DMA zone: 194560 pages, LIFO batch:31
-  HighMem zone: 329472 pages, LIFO batch:31
-PERCPU: Embedded 12 pages/cpu @ee7a3000 s19200 r8192 d21760 u49152
+  DMA zone: 196608 pages, LIFO batch:31
+  HighMem zone: 327424 pages, LIFO batch:31
+PERCPU: Embedded 12 pages/cpu @eefa3000 s19200 r8192 d21760 u49152
 pcpu-alloc: s19200 r8192 d21760 u49152 alloc=12*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
-Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522512
+Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998880K/2096128K available (8115K kernel code, 1018K rwdata, 3624K rodata, 1016K init, 340K bss, 31712K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1998880K/2096128K available (8115K kernel code, 1019K rwdata, 3624K rodata, 1016K init, 340K bss, 31712K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
-    vmalloc : 0xf0000000 - 0xff000000   ( 240 MB)
-    lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
+    vmalloc : 0xf0800000 - 0xff800000   ( 240 MB)
+    lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d7fea0   (11744 kB)
+      .text : 0xc0208000 - 0xc0d7fec8   (11744 kB)
       .init : 0xc0d80000 - 0xc0e7e000   (1016 kB)
-      .data : 0xc0e7e000 - 0xc0f7c9e0   (1019 kB)
+      .data : 0xc0e7e000 - 0xc0f7cd60   (1020 kB)
        .bss : 0xc0f7f000 - 0xc0fd42e8   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -254,18 +254,18 @@
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
 tps65090 2-0048: No cache defaults, reading back from HW
-vdd-lcd-bl: supplied by vdd_ac_bat
 mmc0: MAN_BKOPS_EN bit is not set
-vdd-sys-5v0: supplied by vdd_ac_bat
-VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
-vdd_hdmi_5v0: supplied by vdd-sys-5v0
-[drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 SEM32G 29.1 GiB 
 mmcblk0boot0: mmc0:0001 SEM32G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM32G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM32G partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
+vdd-lcd-bl: supplied by vdd_ac_bat
+vdd-sys-5v0: supplied by vdd_ac_bat
+VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
+vdd_hdmi_5v0: supplied by vdd-sys-5v0
+[drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
 drm drm: No connectors reported connected with modes
 [drm] Cannot find any crtc or sizes - going 1024x768
@@ -289,7 +289,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 78000400.sdhci [78000400.sdhci] using ADMA
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-03-29 12:42:53 UTC (954333773)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-03-30 13:42:50 UTC (954423770)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2015-09-17 21:53:47.012917597 -0700
+++ /tmp/foob	2015-09-17 21:53:47.012917597 -0700
@@ -36,21 +36,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #####################
-	 3.3 MiB/s
+	 #############################
+	 3.4 MiB/s
 done
-Bytes transferred = 5064848 (4d4890 hex)
+Bytes transferred = 5184232 (4f1ae8 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.8 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 34966 (8896 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4d4890 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4f1ae8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff895 ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.3.0-rc1-next-20150917-42145-g01c8787 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Sep 16 20:48:03 PDT 2015
+Linux version 4.3.0-rc1-next-20150918-42418-g887e5d0 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Sep 17 21:48:02 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -69,31 +69,31 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c09c7880, node_mem_map ee7f7000
-  Normal zone: 1520 pages used for memmap
+free_area_init_node: node 0, pgdat c0a04540, node_mem_map eeff7000
+  Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
-  Normal zone: 194560 pages, LIFO batch:31
-  HighMem zone: 329472 pages, LIFO batch:31
-PERCPU: Embedded 12 pages/cpu @ee7a2000 s19456 r8192 d21504 u49152
+  Normal zone: 196608 pages, LIFO batch:31
+  HighMem zone: 327424 pages, LIFO batch:31
+PERCPU: Embedded 12 pages/cpu @eefa2000 s19456 r8192 d21504 u49152
 pcpu-alloc: s19456 r8192 d21504 u49152 alloc=12*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
-Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522512
+Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002640K/2096128K available (6932K kernel code, 341K rwdata, 2388K rodata, 324K init, 363K bss, 27952K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 2002392K/2096128K available (7106K kernel code, 352K rwdata, 2440K rodata, 332K init, 364K bss, 28200K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
-    vmalloc : 0xf0000000 - 0xff000000   ( 240 MB)
-    lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
+    vmalloc : 0xf0800000 - 0xff800000   ( 240 MB)
+    lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0922768   (9322 kB)
-      .init : 0xc0923000 - 0xc0974000   ( 324 kB)
-      .data : 0xc0974000 - 0xc09c9400   ( 341 kB)
-       .bss : 0xc09cc000 - 0xc0a26d20   ( 364 kB)
+      .text : 0xc0008000 - 0xc095af6c   (9548 kB)
+      .init : 0xc095b000 - 0xc09ae000   ( 332 kB)
+      .data : 0xc09ae000 - 0xc0a060c0   ( 353 kB)
+       .bss : 0xc0a09000 - 0xc0a64078   ( 365 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -175,6 +175,9 @@
 PCI: CLS 0 bytes, default 64
 futex hash table entries: 1024 (order: 4, 65536 bytes)
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
+NFS: Registering the id_resolver key type
+Key type id_resolver registered
+Key type id_legacy registered
 bounce: pool size: 64 pages
 io scheduler noop registered (default)
 tegra-apbdma 6000a000.dma: Tegra20 APB DMA driver register 32 channels
@@ -235,26 +238,27 @@
 sit: IPv6 over IPv4 tunneling driver
 NET: Registered protocol family 17
 mmc0: MAN_BKOPS_EN bit is not set
-mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 SEM32G 29.1 GiB 
-mmcblk0boot0: mmc0:0001 SEM32G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM32G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM32G partition 3 4.00 MiB
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 NET: Registered protocol family 15
 can: controller area network core (rev 20120528 abi 9)
 NET: Registered protocol family 29
 can: raw protocol (rev 20120528)
 can: broadcast manager protocol (rev 20120528 t)
+mmc0: new high speed MMC card at address 0001
 can: netlink gateway (rev 20130117) max_hops=1
 Bluetooth: RFCOMM socket layer initialized
+mmcblk0: mmc0:0001 SEM32G 29.1 GiB 
+mmcblk0boot0: mmc0:0001 SEM32G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM32G partition 2 4.00 MiB
 Bluetooth: RFCOMM ver 1.11
 Bluetooth: BNEP (Ethernet Emulation) ver 1.3
 Bluetooth: BNEP socket layer initialized
 Bluetooth: HIDP (Human Interface Emulation) ver 1.2
 Bluetooth: HIDP socket layer initialized
+Key type dns_resolver registered
 Registering SWP/SWPB emulation handler
+mmcblk0rpmb: mmc0:0001 SEM32G partition 3 4.00 MiB
 tps65090 2-0048: No cache defaults, reading back from HW
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 vdd-lcd-bl: supplied by vdd_ac_bat
 vdd-sys-5v0: supplied by vdd_ac_bat
 VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
@@ -286,7 +290,7 @@
 tegra-snd-rt5640 sound: rt5640-aif1 <-> 70080400.i2s mapping ok
 input: NVIDIA Tegra Dalmore Headphones as /devices/soc0/sound/sound/card0/input0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-03-29 12:43:25 UTC (954333805)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-03-30 13:43:22 UTC (954423802)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -299,7 +303,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 324K (c0923000 - c0974000)
+Freeing unused kernel memory: 332K (c095b000 - c09ae000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-09-17 21:53:47.020917589 -0700
+++ /tmp/foob	2015-09-17 21:53:47.020917589 -0700
@@ -29,7 +29,6 @@
 Tegra124 (Jetson TK1) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug init=/bin/bash noinitrd'
 Tegra124 (Jetson TK1) # setenv ipaddr 192.168.100.102; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra124 (Jetson TK1) # tftpboot 0x81000000 zImage.multi_v7_defconfig
-Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'zImage.multi_v7_defconfig'.
@@ -41,20 +40,20 @@
 	 #################################################################
 	 #################################################################
 	 #################################################
-	 3.2 MiB/s
+	 3.1 MiB/s
 done
-Bytes transferred = 6433872 (622c50 hex)
+Bytes transferred = 6433712 (622bb0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *#####
-	 2.9 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 64210 (fad2 hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x622c50 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x622bb0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012ad1
@@ -62,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.3.0-rc1-next-20150917-42145-g01c8787 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Sep 16 20:47:46 PDT 2015
+Linux version 4.3.0-rc1-next-20150918-42418-g887e5d0 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Sep 17 21:47:41 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -70,30 +69,30 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f7aa00, node_mem_map ee7f8000
-  DMA zone: 1520 pages used for memmap
+free_area_init_node: node 0, pgdat c0f7ad80, node_mem_map eeff8000
+  DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
-  DMA zone: 194560 pages, LIFO batch:31
-  HighMem zone: 329472 pages, LIFO batch:31
-PERCPU: Embedded 12 pages/cpu @ee78f000 s19200 r8192 d21760 u49152
+  DMA zone: 196608 pages, LIFO batch:31
+  HighMem zone: 327424 pages, LIFO batch:31
+PERCPU: Embedded 12 pages/cpu @eef8f000 s19200 r8192 d21760 u49152
 pcpu-alloc: s19200 r8192 d21760 u49152 alloc=12*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
-Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522512
+Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug init=/bin/bash noinitrd
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998772K/2096128K available (8115K kernel code, 1018K rwdata, 3624K rodata, 1016K init, 340K bss, 31820K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1998772K/2096128K available (8115K kernel code, 1019K rwdata, 3624K rodata, 1016K init, 340K bss, 31820K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
-    vmalloc : 0xf0000000 - 0xff000000   ( 240 MB)
-    lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
+    vmalloc : 0xf0800000 - 0xff800000   ( 240 MB)
+    lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d7fea0   (11744 kB)
+      .text : 0xc0208000 - 0xc0d7fec8   (11744 kB)
       .init : 0xc0d80000 - 0xc0e7e000   (1016 kB)
-      .data : 0xc0e7e000 - 0xc0f7c9e0   (1019 kB)
+      .data : 0xc0e7e000 - 0xc0f7cd60   (1020 kB)
        .bss : 0xc0f7f000 - 0xc0fd42e8   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -258,13 +257,6 @@
 NET: Registered protocol family 10
 sit: IPv6 over IPv4 tunneling driver
 mmc0: MAN_BKOPS_EN bit is not set
-mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 SEM16G 14.6 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
-Primary GPT is invalid, using alternate GPT.
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9
 NET: Registered protocol family 17
 can: controller area network core (rev 20120528 abi 9)
 NET: Registered protocol family 29
@@ -279,8 +271,15 @@
 +3.3V_LP0: supplied by +3.3V_SYS
 +1.35V_LP0(sd2): supplied by +5V_SYS
 +1.05V_RUN_AVDD: supplied by +1.35V_LP0(sd2)
+mmc0: new high speed MMC card at address 0001
 tegra-pcie 1003000.pcie-controller: probing port 0, using 2 lanes
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
+mmcblk0: mmc0:0001 SEM16G 14.6 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+Primary GPT is invalid, using alternate GPT.
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9
 tegra-pcie 1003000.pcie-controller: link 0 down, retrying
 tegra-pcie 1003000.pcie-controller: link 0 down, retrying
 tegra-pcie 1003000.pcie-controller: link 0 down, retrying
@@ -334,7 +333,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:19 UTC (946684819)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:18 UTC (946684818)
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
@@ -354,4 +353,4 @@
 Freeing unused kernel memory: 1016K (c0d80000 - c0e7e000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# 
\ No newline at end of file
+root@(none):/# [    5.960018] asix 2-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:11:31:3b
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2015-09-17 21:53:47.028917582 -0700
+++ /tmp/foob	2015-09-17 21:53:47.032917579 -0700
@@ -38,21 +38,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #####################
+	 #############################
 	 3.2 MiB/s
 done
-Bytes transferred = 5064848 (4d4890 hex)
+Bytes transferred = 5184232 (4f1ae8 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *#####
-	 3.1 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 64210 (fad2 hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4d4890 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4f1ae8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012ad1
@@ -63,7 +63,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.3.0-rc1-next-20150917-42145-g01c8787 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Sep 16 20:48:03 PDT 2015
+Linux version 4.3.0-rc1-next-20150918-42418-g887e5d0 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Sep 17 21:48:02 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -71,31 +71,31 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c09c7880, node_mem_map ee7f7000
-  Normal zone: 1520 pages used for memmap
+free_area_init_node: node 0, pgdat c0a04540, node_mem_map eeff7000
+  Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
-  Normal zone: 194560 pages, LIFO batch:31
-  HighMem zone: 329472 pages, LIFO batch:31
-PERCPU: Embedded 12 pages/cpu @ee78e000 s19456 r8192 d21504 u49152
+  Normal zone: 196608 pages, LIFO batch:31
+  HighMem zone: 327424 pages, LIFO batch:31
+PERCPU: Embedded 12 pages/cpu @eef8e000 s19456 r8192 d21504 u49152
 pcpu-alloc: s19456 r8192 d21504 u49152 alloc=12*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
-Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522512
+Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug init=/bin/bash noinitrd
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002532K/2096128K available (6932K kernel code, 341K rwdata, 2388K rodata, 324K init, 363K bss, 28060K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 2002284K/2096128K available (7106K kernel code, 352K rwdata, 2440K rodata, 332K init, 364K bss, 28308K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
-    vmalloc : 0xf0000000 - 0xff000000   ( 240 MB)
-    lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
+    vmalloc : 0xf0800000 - 0xff800000   ( 240 MB)
+    lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0922768   (9322 kB)
-      .init : 0xc0923000 - 0xc0974000   ( 324 kB)
-      .data : 0xc0974000 - 0xc09c9400   ( 341 kB)
-       .bss : 0xc09cc000 - 0xc0a26d20   ( 364 kB)
+      .text : 0xc0008000 - 0xc095af6c   (9548 kB)
+      .init : 0xc095b000 - 0xc09ae000   ( 332 kB)
+      .data : 0xc09ae000 - 0xc0a060c0   ( 353 kB)
+       .bss : 0xc0a09000 - 0xc0a64078   ( 365 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -174,6 +174,9 @@
 hw perfevents: enabled with armv7_cortex_a15 PMU driver, 7 counters available
 futex hash table entries: 1024 (order: 4, 65536 bytes)
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
+NFS: Registering the id_resolver key type
+Key type id_resolver registered
+Key type id_legacy registered
 bounce: pool size: 64 pages
 io scheduler noop registered (default)
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
@@ -267,6 +270,7 @@
 Bluetooth: BNEP socket layer initialized
 Bluetooth: HIDP (Human Interface Emulation) ver 1.2
 Bluetooth: HIDP socket layer initialized
+Key type dns_resolver registered
 Registering SWP/SWPB emulation handler
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
 +3.3V_SYS: supplied by +VDD_MUX
@@ -315,7 +319,7 @@
 pcie_pme 0000:00:02.0:pcie01: service driver pcie_pme loaded
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:01:00.0: enabling device (0140 -> 0143)
-r8169 0000:01:00.0 eth0: RTL8168g/8111g at 0xf00f6000, 00:04:4b:25:b3:b4, XID 0c000800 IRQ 388
+r8169 0000:01:00.0 eth0: RTL8168g/8111g at 0xf08f6000, 00:04:4b:25:b3:b4, XID 0c000800 IRQ 388
 r8169 0000:01:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 +5V_HDMI_CON: supplied by +5V_SYS
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
@@ -358,7 +362,7 @@
 tegra-snd-rt5640 sound: rt5640-aif1 <-> 70301100.i2s mapping ok
 input: NVIDIA Tegra Jetson TK1 Headphones as /devices/soc0/sound/sound/card1/input1
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:51 UTC (946684851)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:50 UTC (946684850)
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
@@ -377,7 +381,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 324K (c0923000 - c0974000)
+Freeing unused kernel memory: 332K (c095b000 - c09ae000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-09-17 21:53:47.036917575 -0700
+++ /tmp/foob	2015-09-17 21:53:47.036917575 -0700
@@ -46,9 +46,9 @@
 	 #################################################################
 	 #################################################################
 	 #################################################
-	 4.1 MiB/s
+	 4 MiB/s
 done
-Bytes transferred = 6433872 (622c50 hex)
+Bytes transferred = 6433712 (622bb0 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -59,7 +59,7 @@
 done
 Bytes transferred = 21202 (52d2 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x622c50 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x622bb0 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102d1
@@ -67,7 +67,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.3.0-rc1-next-20150917-42145-g01c8787 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Sep 16 20:47:46 PDT 2015
+Linux version 4.3.0-rc1-next-20150918-42418-g887e5d0 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Sep 17 21:47:41 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -75,30 +75,30 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0f7aa00, node_mem_map eeff8000
-  DMA zone: 1520 pages used for memmap
+free_area_init_node: node 0, pgdat c0f7ad80, node_mem_map ef7f8000
+  DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
-  DMA zone: 194560 pages, LIFO batch:31
-  HighMem zone: 67584 pages, LIFO batch:15
-PERCPU: Embedded 12 pages/cpu @eefca000 s19200 r8192 d21760 u49152
+  DMA zone: 196608 pages, LIFO batch:31
+  HighMem zone: 65536 pages, LIFO batch:15
+PERCPU: Embedded 12 pages/cpu @ef7cb000 s19200 r8192 d21760 u49152
 pcpu-alloc: s19200 r8192 d21760 u49152 alloc=12*4096
 pcpu-alloc: [0] 0 [0] 1 
-Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 260624
+Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 260608
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 959688K/1048576K available (8115K kernel code, 1018K rwdata, 3624K rodata, 1016K init, 340K bss, 23352K reserved, 65536K cma-reserved, 204800K highmem)
+Memory: 959688K/1048576K available (8115K kernel code, 1019K rwdata, 3624K rodata, 1016K init, 340K bss, 23352K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
-    vmalloc : 0xf0000000 - 0xff000000   ( 240 MB)
-    lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
+    vmalloc : 0xf0800000 - 0xff800000   ( 240 MB)
+    lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d7fea0   (11744 kB)
+      .text : 0xc0208000 - 0xc0d7fec8   (11744 kB)
       .init : 0xc0d80000 - 0xc0e7e000   (1016 kB)
-      .data : 0xc0e7e000 - 0xc0f7c9e0   (1019 kB)
+      .data : 0xc0e7e000 - 0xc0f7cd60   (1020 kB)
        .bss : 0xc0f7f000 - 0xc0fd42e8   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
@@ -241,7 +241,7 @@
 igb: Copyright (c) 2007-2014 Intel Corporation.
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:01:00.0: enabling device (0140 -> 0143)
-r8169 0000:01:00.0 eth0: RTL8168d/8111d at 0xf005e000, 00:01:c0:13:7d:63, XID 081000c0 IRQ 72
+r8169 0000:01:00.0 eth0: RTL8168d/8111d at 0xf085e000, 00:01:c0:13:7d:63, XID 081000c0 IRQ 72
 r8169 0000:01:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 pegasus: v0.9.3 (2013/04/25), Pegasus/Pegasus II USB Ethernet driver
 usbcore: registered new interface driver pegasus
@@ -330,19 +330,19 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on c8000600.sdhci [c8000600.sdhci] using ADMA
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-26 22:44:15 UTC (1419633855)
+rtc-em3027 2-0056: setting system clock to 2014-12-26 22:44:51 UTC (1419633891)
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
-mmc1: new high speed SD card at address b368
 devtmpfs: mounted
+mmc1: new high speed SD card at address b368
 Freeing unused kernel memory: 1016K (c0d80000 - c0e7e000)
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.290549] asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008
\ No newline at end of file
+root@(none):/# [    3.285751] asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2015-09-17 21:53:47.044917566 -0700
+++ /tmp/foob	2015-09-17 21:53:47.044917566 -0700
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -44,21 +44,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #####################
+	 #############################
 	 4.1 MiB/s
 done
-Bytes transferred = 5064848 (4d4890 hex)
+Bytes transferred = 5184232 (4f1ae8 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 4 MiB/s
 done
 Bytes transferred = 21202 (52d2 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x4d4890 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x4f1ae8 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102d1
@@ -69,7 +69,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.3.0-rc1-next-20150917-42145-g01c8787 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Sep 16 20:48:03 PDT 2015
+Linux version 4.3.0-rc1-next-20150918-42418-g887e5d0 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Sep 17 21:48:02 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -78,31 +78,31 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c09c7880, node_mem_map eeff7000
-  Normal zone: 1520 pages used for memmap
+free_area_init_node: node 0, pgdat c0a04540, node_mem_map ef7f7000
+  Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
-  Normal zone: 194560 pages, LIFO batch:31
-  HighMem zone: 67584 pages, LIFO batch:15
-PERCPU: Embedded 12 pages/cpu @eefc9000 s19456 r8192 d21504 u49152
+  Normal zone: 196608 pages, LIFO batch:31
+  HighMem zone: 65536 pages, LIFO batch:15
+PERCPU: Embedded 12 pages/cpu @ef7ca000 s19456 r8192 d21504 u49152
 pcpu-alloc: s19456 r8192 d21504 u49152 alloc=12*4096
 pcpu-alloc: [0] 0 [0] 1 
-Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 260624
+Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 260608
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 963448K/1048576K available (6932K kernel code, 341K rwdata, 2388K rodata, 324K init, 363K bss, 19592K reserved, 65536K cma-reserved, 204800K highmem)
+Memory: 963200K/1048576K available (7106K kernel code, 352K rwdata, 2440K rodata, 332K init, 364K bss, 19840K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
-    vmalloc : 0xf0000000 - 0xff000000   ( 240 MB)
-    lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
+    vmalloc : 0xf0800000 - 0xff800000   ( 240 MB)
+    lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0922768   (9322 kB)
-      .init : 0xc0923000 - 0xc0974000   ( 324 kB)
-      .data : 0xc0974000 - 0xc09c9400   ( 341 kB)
-       .bss : 0xc09cc000 - 0xc0a26d20   ( 364 kB)
+      .text : 0xc0008000 - 0xc095af6c   (9548 kB)
+      .init : 0xc095b000 - 0xc09ae000   ( 332 kB)
+      .data : 0xc09ae000 - 0xc0a060c0   ( 353 kB)
+       .bss : 0xc0a09000 - 0xc0a64078   ( 365 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
@@ -180,6 +180,9 @@
 hw perfevents: enabled with armv7_cortex_a9 PMU driver, 7 counters available
 futex hash table entries: 512 (order: 3, 32768 bytes)
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
+NFS: Registering the id_resolver key type
+Key type id_resolver registered
+Key type id_legacy registered
 bounce: pool size: 64 pages
 io scheduler noop registered (default)
 tegra-pcie 80003000.pcie-controller: dual-mode configuration
@@ -244,7 +247,7 @@
 igb: Copyright (c) 2007-2014 Intel Corporation.
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:01:00.0: enabling device (0140 -> 0143)
-r8169 0000:01:00.0 eth0: RTL8168d/8111d at 0xf005e000, 00:01:c0:13:7d:63, XID 081000c0 IRQ 72
+r8169 0000:01:00.0 eth0: RTL8168d/8111d at 0xf085e000, 00:01:c0:13:7d:63, XID 081000c0 IRQ 72
 r8169 0000:01:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 pegasus: v0.9.3 (2013/04/25), Pegasus/Pegasus II USB Ethernet driver
 usbcore: registered new interface driver pegasus
@@ -306,25 +309,26 @@
 NET: Registered protocol family 15
 can: controller area network core (rev 20120528 abi 9)
 NET: Registered protocol family 29
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 can: raw protocol (rev 20120528)
 can: broadcast manager protocol (rev 20120528 t)
 can: netlink gateway (rev 20130117) max_hops=1
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
 Bluetooth: RFCOMM socket layer initialized
 Bluetooth: RFCOMM ver 1.11
 Bluetooth: BNEP (Ethernet Emulation) ver 1.3
 Bluetooth: BNEP socket layer initialized
+mmc0: host does not support reading read-only switch, assuming write-enable
 Bluetooth: HIDP (Human Interface Emulation) ver 1.2
 Bluetooth: HIDP socket layer initialized
+Key type dns_resolver registered
+mmc0: new high speed SDHC card at address b368
 Registering SWP/SWPB emulation handler
-mmc0: host does not support reading read-only switch, assuming write-enable
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
+ mmcblk0: p1
 tegra-phy c5004000.usb-phy: no vbus regulator
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
-mmc0: new high speed SDHC card at address b368
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
- mmcblk0: p1
 tegra-ehci c5000000.usb: irq 64, io mem 0xc5000000
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
@@ -341,20 +345,20 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on c8000600.sdhci [c8000600.sdhci] using ADMA
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-26 22:44:34 UTC (1419633874)
+rtc-em3027 2-0056: setting system clock to 2014-12-26 22:45:10 UTC (1419633910)
 ALSA device list:
   #0: tegra-trimslice
 EXT2-fs (mmcblk0p1): error: couldn't mount because of unsupported optional features (4)
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
- mmcblk1: p1 p2
-usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
+usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 VFS: Mounted root (ext3 filesystem) on device 179:1.
+ mmcblk1: p1 p2
 devtmpfs: mounted
-Freeing unused kernel memory: 324K (c0923000 - c0974000)
+Freeing unused kernel memory: 332K (c095b000 - c09ae000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-09-17 21:53:47.052917559 -0700
+++ /tmp/foob	2015-09-17 21:53:47.052917559 -0700
@@ -40,7 +40,7 @@
 	 #################################################
 	 3.3 MiB/s
 done
-Bytes transferred = 6433872 (622c50 hex)
+Bytes transferred = 6433712 (622bb0 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -51,7 +51,7 @@
 done
 Bytes transferred = 51539 (c953 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x622c50 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x622bb0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff952 ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.3.0-rc1-next-20150917-42145-g01c8787 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Sep 16 20:47:46 PDT 2015
+Linux version 4.3.0-rc1-next-20150918-42418-g887e5d0 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Sep 17 21:47:41 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -67,30 +67,30 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f7aa00, node_mem_map ee7f8000
-  DMA zone: 1520 pages used for memmap
+free_area_init_node: node 0, pgdat c0f7ad80, node_mem_map eeff8000
+  DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
-  DMA zone: 194560 pages, LIFO batch:31
-  HighMem zone: 329472 pages, LIFO batch:31
-PERCPU: Embedded 12 pages/cpu @ee793000 s19200 r8192 d21760 u49152
+  DMA zone: 196608 pages, LIFO batch:31
+  HighMem zone: 327424 pages, LIFO batch:31
+PERCPU: Embedded 12 pages/cpu @eef93000 s19200 r8192 d21760 u49152
 pcpu-alloc: s19200 r8192 d21760 u49152 alloc=12*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
-Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522512
+Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug init=/bin/bash noinitrd
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998800K/2096128K available (8115K kernel code, 1018K rwdata, 3624K rodata, 1016K init, 340K bss, 31792K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1998800K/2096128K available (8115K kernel code, 1019K rwdata, 3624K rodata, 1016K init, 340K bss, 31792K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
-    vmalloc : 0xf0000000 - 0xff000000   ( 240 MB)
-    lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
+    vmalloc : 0xf0800000 - 0xff800000   ( 240 MB)
+    lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d7fea0   (11744 kB)
+      .text : 0xc0208000 - 0xc0d7fec8   (11744 kB)
       .init : 0xc0d80000 - 0xc0e7e000   (1016 kB)
-      .data : 0xc0e7e000 - 0xc0f7c9e0   (1019 kB)
+      .data : 0xc0e7e000 - 0xc0f7cd60   (1020 kB)
        .bss : 0xc0f7f000 - 0xc0fd42e8   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -267,18 +267,18 @@
 Key type dns_resolver registered
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
+tegra-pcie 3000.pcie-controller: 2x3 configuration
+sys_3v3,vdd_3v3_alw: supplied by vdd_5v_in
+sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
+tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 SEM16G 14.8 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 1.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 1.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 128 KiB
+tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 Primary GPT is invalid, using alternate GPT.
-tegra-pcie 3000.pcie-controller: 2x3 configuration
-sys_3v3,vdd_3v3_alw: supplied by vdd_5v_in
-sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
-tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
  mmcblk0: p1 p2 p3 p4 p5
-tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -315,7 +315,7 @@
 pcie_pme 0000:00:01.0:pcie01: service driver pcie_pme loaded
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:01:00.0: enabling device (0140 -> 0143)
-r8169 0000:01:00.0 eth0: RTL8168evl/8111evl at 0xf007e000, 02:04:4b:01:60:a6, XID 0c900800 IRQ 375
+r8169 0000:01:00.0 eth0: RTL8168evl/8111evl at 0xf087e000, 02:04:4b:01:60:a6, XID 0c900800 IRQ 375
 r8169 0000:01:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2015-09-17 21:53:47.060917552 -0700
+++ /tmp/foob	2015-09-17 21:53:47.060917552 -0700
@@ -36,21 +36,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #####################
+	 #############################
 	 3.3 MiB/s
 done
-Bytes transferred = 5064848 (4d4890 hex)
+Bytes transferred = 5184232 (4f1ae8 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.3 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 51539 (c953 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4d4890 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4f1ae8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff952 ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.3.0-rc1-next-20150917-42145-g01c8787 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Sep 16 20:48:03 PDT 2015
+Linux version 4.3.0-rc1-next-20150918-42418-g887e5d0 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Sep 17 21:48:02 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -69,31 +69,31 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c09c7880, node_mem_map ee7f7000
-  Normal zone: 1520 pages used for memmap
+free_area_init_node: node 0, pgdat c0a04540, node_mem_map eeff7000
+  Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
-  Normal zone: 194560 pages, LIFO batch:31
-  HighMem zone: 329472 pages, LIFO batch:31
-PERCPU: Embedded 12 pages/cpu @ee792000 s19456 r8192 d21504 u49152
+  Normal zone: 196608 pages, LIFO batch:31
+  HighMem zone: 327424 pages, LIFO batch:31
+PERCPU: Embedded 12 pages/cpu @eef92000 s19456 r8192 d21504 u49152
 pcpu-alloc: s19456 r8192 d21504 u49152 alloc=12*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
-Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522512
+Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug init=/bin/bash noinitrd
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002560K/2096128K available (6932K kernel code, 341K rwdata, 2388K rodata, 324K init, 363K bss, 28032K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 2002312K/2096128K available (7106K kernel code, 352K rwdata, 2440K rodata, 332K init, 364K bss, 28280K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
-    vmalloc : 0xf0000000 - 0xff000000   ( 240 MB)
-    lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
+    vmalloc : 0xf0800000 - 0xff800000   ( 240 MB)
+    lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0922768   (9322 kB)
-      .init : 0xc0923000 - 0xc0974000   ( 324 kB)
-      .data : 0xc0974000 - 0xc09c9400   ( 341 kB)
-       .bss : 0xc09cc000 - 0xc0a26d20   ( 364 kB)
+      .text : 0xc0008000 - 0xc095af6c   (9548 kB)
+      .init : 0xc095b000 - 0xc09ae000   ( 332 kB)
+      .data : 0xc09ae000 - 0xc0a060c0   ( 353 kB)
+       .bss : 0xc0a09000 - 0xc0a64078   ( 365 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -172,6 +172,9 @@
 hw perfevents: enabled with armv7_cortex_a9 PMU driver, 7 counters available
 futex hash table entries: 1024 (order: 4, 65536 bytes)
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
+NFS: Registering the id_resolver key type
+Key type id_resolver registered
+Key type id_legacy registered
 bounce: pool size: 64 pages
 io scheduler noop registered (default)
 tegra30-pinctrl 70000868.pinmux: invalid group "sdmmc3_dat6_pd3" for function "rsvd1"
@@ -255,20 +258,21 @@
 can: raw protocol (rev 20120528)
 can: broadcast manager protocol (rev 20120528 t)
 can: netlink gateway (rev 20130117) max_hops=1
-mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 SEM16G 14.8 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 1.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 1.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 128 KiB
-Primary GPT is invalid, using alternate GPT.
- mmcblk0: p1 p2 p3 p4 p5
 Bluetooth: RFCOMM socket layer initialized
 Bluetooth: RFCOMM ver 1.11
 Bluetooth: BNEP (Ethernet Emulation) ver 1.3
 Bluetooth: BNEP socket layer initialized
 Bluetooth: HIDP (Human Interface Emulation) ver 1.2
 Bluetooth: HIDP socket layer initialized
+Key type dns_resolver registered
+mmc0: new high speed MMC card at address 0001
 Registering SWP/SWPB emulation handler
+mmcblk0: mmc0:0001 SEM16G 14.8 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 1.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 1.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 128 KiB
+Primary GPT is invalid, using alternate GPT.
+ mmcblk0: p1 p2 p3 p4 p5
 tegra-pcie 3000.pcie-controller: 2x3 configuration
 sys_3v3,vdd_3v3_alw: supplied by vdd_5v_in
 sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
@@ -310,7 +314,7 @@
 pcie_pme 0000:00:01.0:pcie01: service driver pcie_pme loaded
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:01:00.0: enabling device (0140 -> 0143)
-r8169 0000:01:00.0 eth0: RTL8168evl/8111evl at 0xf007e000, 02:04:4b:01:60:a6, XID 0c900800 IRQ 375
+r8169 0000:01:00.0 eth0: RTL8168evl/8111evl at 0xf087e000, 02:04:4b:01:60:a6, XID 0c900800 IRQ 375
 r8169 0000:01:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
@@ -355,7 +359,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 324K (c0923000 - c0974000)
+Freeing unused kernel memory: 332K (c095b000 - c09ae000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
