Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-04-05 21:56:03.338093979 -0700
+++ /tmp/foob	2016-04-05 21:56:03.342093978 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc2-next-20160405+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Apr 4 22:21:01 PDT 2016
+Linux version 4.6.0-rc2-next-20160406+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Apr 5 21:50:26 PDT 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -94,7 +94,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.524:1): initialized
+audit: type=2000 audit(0.560:1): initialized
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -154,7 +154,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2016-04-05 05:21:32 UTC (1459833692)
+rtc-pl031 9010000.pl031: setting system clock to 2016-04-06 04:50:57 UTC (1459918257)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-04-05 21:56:03.346093977 -0700
+++ /tmp/foob	2016-04-05 21:56:03.350093976 -0700
@@ -39,28 +39,28 @@
 	 #################################################################
 	 #################################################################
 	 ######################
-	 3.3 MiB/s
+	 2.8 MiB/s
 done
-Bytes transferred = 6995840 (6abf80 hex)
+Bytes transferred = 6994480 (6aba30 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3.3 MiB/s
+	 2.8 MiB/s
 done
-Bytes transferred = 34912 (8860 hex)
+Bytes transferred = 34956 (888c hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6abf80 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6aba30 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
-   Loading Device Tree to 8fff4000, end 8ffff85f ... OK
+   Loading Device Tree to 8fff4000, end 8ffff88b ... OK
 
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc2-next-20160405-84844-gaa36efa (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Apr 4 22:19:20 PDT 2016
+Linux version 4.6.0-rc2-next-20160406-85171-g4c8df6e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Apr 5 21:48:53 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141cf40, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c141cf80, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -90,9 +90,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc105a58c   (14666 kB)
+      .text : 0xc0208000 - 0xc105a5b4   (14666 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141efa0   (1148 kB)
+      .data : 0xc1300000 - 0xc141efe0   (1148 kB)
        .bss : 0xc1420000 - 0xc1475558   ( 342 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -310,8 +310,8 @@
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
- mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
  mmcblk0: p1 p2 p3
+ mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
 Console: switching to colour frame buffer device 240x75
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
@@ -322,7 +322,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-28 16:37:02 UTC (949077422)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-29 16:07:56 UTC (949162076)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -334,4 +334,4 @@
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.345126] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.340231] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-04-05 21:56:03.354093974 -0700
+++ /tmp/foob	2016-04-05 21:56:03.358093973 -0700
@@ -37,29 +37,29 @@
 	 #################################################################
 	 #################################################################
 	 ######################################
-	 3.3 MiB/s
+	 3 MiB/s
 done
-Bytes transferred = 5322448 (5136d0 hex)
+Bytes transferred = 5321312 (513260 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.6 MiB/s
+	 2.8 MiB/s
 done
-Bytes transferred = 34912 (8860 hex)
+Bytes transferred = 34956 (888c hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5136d0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x513260 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
-   Loading Device Tree to 8fff4000, end 8ffff85f ... OK
+   Loading Device Tree to 8fff4000, end 8ffff88b ... OK
 
 Starting kernel ...
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc2-next-20160405-84844-gaa36efa (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Apr 4 22:19:17 PDT 2016
+Linux version 4.6.0-rc2-next-20160406-85171-g4c8df6e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Apr 5 21:48:43 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0c5c700, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0c5c740, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab32f8   (10925 kB)
+      .text : 0xc0008000 - 0xc0ab3308   (10925 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
-      .data : 0xc0c00000 - 0xc0c5e280   ( 377 kB)
-       .bss : 0xc0c60000 - 0xc0cbaa88   ( 363 kB)
+      .data : 0xc0c00000 - 0xc0c5e2c0   ( 377 kB)
+       .bss : 0xc0c60000 - 0xc0cbaa8c   ( 363 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -320,7 +320,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-28 16:39:33 UTC (949077573)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-29 16:06:30 UTC (949161990)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-04-05 21:56:03.362093972 -0700
+++ /tmp/foob	2016-04-05 21:56:03.366093971 -0700
@@ -29,6 +29,7 @@
 Tegra124 (Jetson TK1) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash'
 Tegra124 (Jetson TK1) # setenv ipaddr 192.168.100.102; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 zImage.multi_v7_defconfig
+Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'zImage.multi_v7_defconfig'.
@@ -41,28 +42,28 @@
 	 #################################################################
 	 #################################################################
 	 ######################
-	 3.2 MiB/s
+	 2.8 MiB/s
 done
-Bytes transferred = 6995840 (6abf80 hex)
+Bytes transferred = 6994480 (6aba30 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.1 MiB/s
+	 2.9 MiB/s
 done
-Bytes transferred = 64236 (faec hex)
+Bytes transferred = 64280 (fb18 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6abf80 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x6aba30 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
-   Using Device Tree in place at 83000000, end 83012aeb
+   Using Device Tree in place at 83000000, end 83012b17
 
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc2-next-20160405-84844-gaa36efa (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Apr 4 22:19:20 PDT 2016
+Linux version 4.6.0-rc2-next-20160406-85171-g4c8df6e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Apr 5 21:48:53 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -71,7 +72,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141cf40, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c141cf80, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -92,9 +93,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc105a58c   (14666 kB)
+      .text : 0xc0208000 - 0xc105a5b4   (14666 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141efa0   (1148 kB)
+      .data : 0xc1300000 - 0xc141efe0   (1148 kB)
        .bss : 0xc1420000 - 0xc1475558   ( 342 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -304,6 +305,13 @@
 +5V_HDMI_CON: supplied by +5V_SYS
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
 tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
+mmc0: new HS200 MMC card at address 0001
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+Primary GPT is invalid, using alternate GPT.
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9
 tegra-ahci 70027000.sata: Failed to get regulators
 tegra-ehci 7d008000.usb: EHCI Host Controller
 tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 2
@@ -315,13 +323,6 @@
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc1: Unknown controller version (3). You may experience problems.
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 204000000, volt: 900000, enabled: 1. New: freq: 204000000, volt: 900000, enabled: 1
-mmc0: new HS200 MMC card at address 0001
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
-Primary GPT is invalid, using alternate GPT.
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 306000000, volt: 900000, enabled: 1. New: freq: 306000000, volt: 900000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 408000000, volt: 900000, enabled: 1. New: freq: 408000000, volt: 900000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 510000000, volt: 900000, enabled: 1. New: freq: 510000000, volt: 900000, enabled: 1
@@ -349,6 +350,8 @@
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
 tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
 tegra-ahci 70027000.sata: Failed to get regulators
+input: gpio-keys as /devices/soc0/gpio-keys/input/input0
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:01:48 UTC (946684908)
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc1: Unknown controller version (3). You may experience problems.
@@ -367,7 +370,6 @@
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1428000000, volt: 900000, enabled: 1. New: freq: 1428000000, volt: 900000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1530000000, volt: 900000, enabled: 1. New: freq: 1530000000, volt: 900000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1632000000, volt: 920000, enabled: 1. New: freq: 1632000000, volt: 920000, enabled: 1
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1734000000, volt: 960000, enabled: 1. New: freq: 1734000000, volt: 960000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1836000000, volt: 990000, enabled: 1. New: freq: 1836000000, volt: 990000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1938000000, volt: 1030000, enabled: 1. New: freq: 1938000000, volt: 1030000, enabled: 1
@@ -376,22 +378,24 @@
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2218500000, volt: 1140000, enabled: 1. New: freq: 2218500000, volt: 1140000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2320500000, volt: 1180000, enabled: 1. New: freq: 2320500000, volt: 1180000, enabled: 1
 tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
-input: gpio-keys as /devices/soc0/gpio-keys/input/input0
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:16 UTC (946684816)
 tegra-pcie 1003000.pcie-controller: Failed to get supply 'avddio-pex': -517
+tegra-hdmi 54280000.hdmi: failed to get PLL regulator
+tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
+tegra-ahci 70027000.sata: Failed to get regulators
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +USB0_VBUS_SW: disabling
 +1.05V_RUN_AVDD_HDMI_PLL: disabling
 +5V_HDMI_CON: disabling
 +5V_SATA: disabling
 +12V_SATA: disabling
-tegra-hdmi 54280000.hdmi: failed to get PLL regulator
-tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
-tegra-ahci 70027000.sata: Failed to get regulators
+asix 2-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:11:31:3b
+EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc1: Unknown controller version (3). You may experience problems.
+EXT4-fs (mmcblk0p1): couldn't mount as ext2 due to feature incompatibilities
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 204000000, volt: 900000, enabled: 1. New: freq: 204000000, volt: 900000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 306000000, volt: 900000, enabled: 1. New: freq: 306000000, volt: 900000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 408000000, volt: 900000, enabled: 1. New: freq: 408000000, volt: 900000, enabled: 1
@@ -415,40 +419,11 @@
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2218500000, volt: 1140000, enabled: 1. New: freq: 2218500000, volt: 1140000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2320500000, volt: 1180000, enabled: 1. New: freq: 2320500000, volt: 1180000, enabled: 1
 tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
-asix 2-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:11:31:3b
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
 tegra-pcie 1003000.pcie-controller: Failed to get supply 'avddio-pex': -517
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
 tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
 tegra-ahci 70027000.sata: Failed to get regulators
-sdhci-tegra 700b0400.sdhci: Got CD GPIO
-sdhci-tegra 700b0400.sdhci: Got WP GPIO
-mmc1: Unknown controller version (3). You may experience problems.
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 204000000, volt: 900000, enabled: 1. New: freq: 204000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 306000000, volt: 900000, enabled: 1. New: freq: 306000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 408000000, volt: 900000, enabled: 1. New: freq: 408000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 510000000, volt: 900000, enabled: 1. New: freq: 510000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 612000000, volt: 900000, enabled: 1. New: freq: 612000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 714000000, volt: 900000, enabled: 1. New: freq: 714000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 816000000, volt: 900000, enabled: 1. New: freq: 816000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 918000000, volt: 900000, enabled: 1. New: freq: 918000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1020000000, volt: 900000, enabled: 1. New: freq: 1020000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1122000000, volt: 900000, enabled: 1. New: freq: 1122000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1224000000, volt: 900000, enabled: 1. New: freq: 1224000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1326000000, volt: 900000, enabled: 1. New: freq: 1326000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1428000000, volt: 900000, enabled: 1. New: freq: 1428000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1530000000, volt: 900000, enabled: 1. New: freq: 1530000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1632000000, volt: 920000, enabled: 1. New: freq: 1632000000, volt: 920000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1734000000, volt: 960000, enabled: 1. New: freq: 1734000000, volt: 960000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1836000000, volt: 990000, enabled: 1. New: freq: 1836000000, volt: 990000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1938000000, volt: 1030000, enabled: 1. New: freq: 1938000000, volt: 1030000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2014500000, volt: 1060000, enabled: 1. New: freq: 2014500000, volt: 1060000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2116500000, volt: 1100000, enabled: 1. New: freq: 2116500000, volt: 1100000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2218500000, volt: 1140000, enabled: 1. New: freq: 2218500000, volt: 1140000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2320500000, volt: 1180000, enabled: 1. New: freq: 2320500000, volt: 1180000, enabled: 1
-tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
-EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
-EXT4-fs (mmcblk0p1): couldn't mount as ext2 due to feature incompatibilities
 EXT4-fs (mmcblk0p1): warning: mounting fs with errors, running e2fsck is recommended
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-04-05 21:56:03.370093970 -0700
+++ /tmp/foob	2016-04-05 21:56:03.374093969 -0700
@@ -41,7 +41,7 @@
 	 ######################################
 	 3.2 MiB/s
 done
-Bytes transferred = 5322448 (5136d0 hex)
+Bytes transferred = 5321312 (513260 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -50,18 +50,18 @@
 Loading: *#####
 	 2.9 MiB/s
 done
-Bytes transferred = 64236 (faec hex)
+Bytes transferred = 64280 (fb18 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5136d0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x513260 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
-   Using Device Tree in place at 83000000, end 83012aeb
+   Using Device Tree in place at 83000000, end 83012b17
 
 Starting kernel ...
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc2-next-20160405-84844-gaa36efa (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Apr 4 22:19:17 PDT 2016
+Linux version 4.6.0-rc2-next-20160406-85171-g4c8df6e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Apr 5 21:48:43 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -71,7 +71,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0c5c700, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0c5c740, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -92,10 +92,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab32f8   (10925 kB)
+      .text : 0xc0008000 - 0xc0ab3308   (10925 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
-      .data : 0xc0c00000 - 0xc0c5e280   ( 377 kB)
-       .bss : 0xc0c60000 - 0xc0cbaa88   ( 363 kB)
+      .data : 0xc0c00000 - 0xc0c5e2c0   ( 377 kB)
+       .bss : 0xc0c60000 - 0xc0cbaa8c   ( 363 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -355,7 +355,7 @@
 tegra-ahci 70027000.sata: Failed to get regulators
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:21 UTC (946684941)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:15 UTC (946684815)
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc1: Unknown controller version (3). You may experience problems.
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 204000000, volt: 900000, enabled: 1. New: freq: 204000000, volt: 900000, enabled: 1
@@ -427,14 +427,14 @@
 tegra-pcie 1003000.pcie-controller: Failed to get supply 'avddio-pex': -517
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
 tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
+EXT4-fs (mmcblk0p1): warning: mounting fs with errors, running e2fsck is recommended
 tegra-ahci 70027000.sata: Failed to get regulators
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
-EXT4-fs (mmcblk0p1): warning: mounting fs with errors, running e2fsck is recommended
-sdhci-tegra 700b0400.sdhci: Got WP GPIO
-mmc1: Unknown controller version (3). You may experience problems.
 EXT4-fs (mmcblk0p1): recovery complete
+sdhci-tegra 700b0400.sdhci: Got WP GPIO
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
+mmc1: Unknown controller version (3). You may experience problems.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2016-04-05 21:56:03.378093968 -0700
+++ /tmp/foob	2016-04-05 21:56:03.382093967 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1799858 us
+Bootloader-Cpu Init at (time stamp): 1800426 us
 
 Pinmux changes applied in kernel way
 
@@ -59,25 +59,25 @@
 The proc BoardInfo: 0x0176:0x0001:0x01:0x43:0x00
 FastbootAddAtag: setting ATAG (0)
 
-Board Id Read Time: 3 us
+Board Id Read Time: 4 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1799832  us
-Bootloader (AosInit) time: 734254 us
-Bootloader Main Init time: 180064 us
-BlockDevMgrInit time: 157541 us
+Miscellaneous avp time: 1800399  us
+Bootloader (AosInit) time: 733469 us
+Bootloader Main Init time: 180263 us
+BlockDevMgrInit time: 157545 us
 PartMgrInit time: 8 us
 StorMgrInit time: 31 us
 FsMgrInit time: 2 us
-Partition Table load: 76710 us 
-Display init time: 574681 us
+Partition Table load: 77000 us 
+Display init time: 574552 us
 Kernel read time: 308 msec
 Load Kernel Size:   13377536 Bytes
 Bl Kernel ReadSpeed: 43433000 Bytes/sec
-Pre-Kernel Jump time: 1513447 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5526350 us
+Pre-Kernel Jump time: 1513557 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5526501 us
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc2-next-20160405+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Apr 4 22:21:01 PDT 2016
+Linux version 4.6.0-rc2-next-20160406+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Apr 5 21:50:26 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
 Memory limited to 1734MB
 efi: Getting EFI parameters from FDT:
@@ -181,7 +181,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.839:1): initialized
+audit: type=2000 audit(0.847:1): initialized
 workingset: timestamp_bits=44 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -245,33 +245,33 @@
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 hctosys: unable to open rtc device (rtc0)
-mmc0: new HS200 MMC card at address 0001
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
+sdhci-tegra 700b0400.sdhci: Got WP GPIO
+mmc1: Unknown controller version (3). You may experience problems.
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +VDD_LED: disabling
+mmc0: new HS200 MMC card at address 0001
 +5V_USB_HS: disabling
 +5V_USB_SS: disabling
 +1.05V_RUN_AVDD_HDMI_PLL_AP_GATE: disabling
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 +5V_HDMI_CON: disabling
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 ALSA device list:
   No soundcards found.
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
 Freeing unused kernel memory: 856K (ffffff8008a42000 - ffffff8008b18000)
-sdhci-tegra 700b0400.sdhci: Got WP GPIO
-mmc1: Unknown controller version (3). You may experience problems.
-sdhci-tegra 700b0400.sdhci: Got CD GPIO
-sdhci-tegra 700b0400.sdhci: Got WP GPIO
-mmc1: Unknown controller version (3). You may experience problems.
 
 System initialization...
 
-Mounting /proc :[SUCCESS]
+Mounting /proc :[    2.533577]  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
+[SUCCESS]
 Mounting /sys :[SUCCESS]
-Mounting /dev :[SUCCESS]
-Mounting /dev/pts :[SUCCESS]
+Mounting /dev :[    2.574424] sdhci-tegra 700b0400.sdhci: Got CD GPIO
+sdhci-tegra 700b0400.sdhci: Got WP GPIO
+[SUCCESS]
+Mounting /dev/pts :[    2.595555] mmc1: Unknown controller version (3). You may experience problems.
+[SUCCESS]
 Mounting /sys/kernel/debug :[SUCCESS]
 Mounting other filesystems :[SUCCESS]
 Enabling hot-plug :[SUCCESS]
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-04-05 21:56:03.386093966 -0700
+++ /tmp/foob	2016-04-05 21:56:03.390093965 -0700
@@ -47,28 +47,28 @@
 	 #################################################################
 	 #################################################################
 	 ######################
-	 4.1 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 6995840 (6abf80 hex)
+Bytes transferred = 6994480 (6aba30 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 4 MiB/s
+	 2.5 MiB/s
 done
-Bytes transferred = 21128 (5288 hex)
+Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x6abf80 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x6aba30 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
-   Using Device Tree in place at 02408000, end 02410287
+   Using Device Tree in place at 02408000, end 024102b3
 
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc2-next-20160405-84844-gaa36efa (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Apr 4 22:19:20 PDT 2016
+Linux version 4.6.0-rc2-next-20160406-85171-g4c8df6e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Apr 5 21:48:53 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -76,7 +76,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c141cf40, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c141cf80, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -97,9 +97,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc105a58c   (14666 kB)
+      .text : 0xc0208000 - 0xc105a5b4   (14666 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141efa0   (1148 kB)
+      .data : 0xc1300000 - 0xc141efe0   (1148 kB)
        .bss : 0xc1420000 - 0xc1475558   ( 342 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
@@ -330,7 +330,7 @@
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
 mmc1: new high speed SD card at address b368
-rtc-em3027 2-0056: setting system clock to 2014-12-28 13:41:45 UTC (1419774105)
+rtc-em3027 2-0056: setting system clock to 2014-12-28 13:52:29 UTC (1419774749)
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-04-05 21:56:03.394093963 -0700
+++ /tmp/foob	2016-04-05 21:56:03.398093963 -0700
@@ -45,29 +45,29 @@
 	 #################################################################
 	 #################################################################
 	 ######################################
-	 906.3 KiB/s
+	 4.1 MiB/s
 done
-Bytes transferred = 5322448 (5136d0 hex)
+Bytes transferred = 5321312 (513260 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 859.4 KiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 21128 (5288 hex)
+Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x5136d0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x513260 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
-   Using Device Tree in place at 02408000, end 02410287
+   Using Device Tree in place at 02408000, end 024102b3
 
 Starting kernel ...
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc2-next-20160405-84844-gaa36efa (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Apr 4 22:19:17 PDT 2016
+Linux version 4.6.0-rc2-next-20160406-85171-g4c8df6e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Apr 5 21:48:43 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -76,7 +76,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0c5c700, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0c5c740, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -97,10 +97,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab32f8   (10925 kB)
+      .text : 0xc0008000 - 0xc0ab3308   (10925 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
-      .data : 0xc0c00000 - 0xc0c5e280   ( 377 kB)
-       .bss : 0xc0c60000 - 0xc0cbaa88   ( 363 kB)
+      .data : 0xc0c00000 - 0xc0c5e2c0   ( 377 kB)
+       .bss : 0xc0c60000 - 0xc0cbaa8c   ( 363 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
@@ -340,7 +340,7 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-28 13:43:08 UTC (1419774188)
+rtc-em3027 2-0056: setting system clock to 2014-12-28 13:51:29 UTC (1419774689)
 ALSA device list:
   #0: tegra-trimslice
 EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2016-04-05 21:56:03.402093961 -0700
+++ /tmp/foob	2016-04-05 21:56:03.406093960 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 797799 us
-Basic modules init took 348898 us
-Sec Bootdevice Read Time = 203 ms, Read Size = 8459 KB
-Next stage binary read took 14371 us
-Carveout took 252726 us
-CPU initialization took 93278 us
-Total time taken by TegraBoot 709273 us
+Tegraboot started after 845543 us
+Basic modules init took 350860 us
+Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
+Next stage binary read took 14370 us
+Carveout took 252752 us
+CPU initialization took 92298 us
+Total time taken by TegraBoot 710280 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.636] LPDDR4 Training: Number of tables = 10
+64b[0001.684] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,21 +117,21 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=/dev/mmcblk0p1 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-11746816 bytes read in 738 ms (15.2 MiB/s)
+11746816 bytes read in 739 ms (15.2 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-37136 bytes read in 454 ms (79.1 KiB/s)
+37160 bytes read in 454 ms (79.1 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
 1627787 bytes read in 495 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
-   Using Device Tree in place at 0000000082000000, end 000000008200c10f
+   Using Device Tree in place at 0000000082000000, end 000000008200c127
 
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc2-next-20160405+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Apr 4 22:21:01 PDT 2016
+Linux version 4.6.0-rc2-next-20160406+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Apr 5 21:50:26 PDT 2016
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-04-05 21:56:03.410093959 -0700
+++ /tmp/foob	2016-04-05 21:56:03.410093959 -0700
@@ -39,9 +39,9 @@
 	 #################################################################
 	 #################################################################
 	 ######################
-	 3.3 MiB/s
+	 2.9 MiB/s
 done
-Bytes transferred = 6995840 (6abf80 hex)
+Bytes transferred = 6994480 (6aba30 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -50,17 +50,17 @@
 Loading: *####
 	 3.1 MiB/s
 done
-Bytes transferred = 51467 (c90b hex)
+Bytes transferred = 51511 (c937 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6abf80 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6aba30 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
-   Loading Device Tree to 8fff0000, end 8ffff90a ... OK
+   Loading Device Tree to 8fff0000, end 8ffff936 ... OK
 
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc2-next-20160405-84844-gaa36efa (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Apr 4 22:19:20 PDT 2016
+Linux version 4.6.0-rc2-next-20160406-85171-g4c8df6e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Apr 5 21:48:53 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,7 +68,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141cf40, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c141cf80, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -89,9 +89,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc105a58c   (14666 kB)
+      .text : 0xc0208000 - 0xc105a5b4   (14666 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141efa0   (1148 kB)
+      .data : 0xc1300000 - 0xc141efe0   (1148 kB)
        .bss : 0xc1420000 - 0xc1475558   ( 342 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -304,9 +304,9 @@
 mmcblk0boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk0boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk0rpmb: mmc1:0001 SEM16G partition 3 128 KiB
+tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 Primary GPT is invalid, using alternate GPT.
  mmcblk0: p1 p2 p3 p4 p5
-tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -377,4 +377,4 @@
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.803460] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.818449] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-04-05 21:56:03.418093957 -0700
+++ /tmp/foob	2016-04-05 21:56:03.418093957 -0700
@@ -37,29 +37,29 @@
 	 #################################################################
 	 #################################################################
 	 ######################################
-	 2.8 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 5322448 (5136d0 hex)
+Bytes transferred = 5321312 (513260 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.7 MiB/s
+	 728.5 KiB/s
 done
-Bytes transferred = 51467 (c90b hex)
+Bytes transferred = 51511 (c937 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5136d0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x513260 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
-   Loading Device Tree to 8fff0000, end 8ffff90a ... OK
+   Loading Device Tree to 8fff0000, end 8ffff936 ... OK
 
 Starting kernel ...
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc2-next-20160405-84844-gaa36efa (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Apr 4 22:19:17 PDT 2016
+Linux version 4.6.0-rc2-next-20160406-85171-g4c8df6e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Apr 5 21:48:43 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,7 +68,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0c5c700, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0c5c740, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -89,10 +89,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab32f8   (10925 kB)
+      .text : 0xc0008000 - 0xc0ab3308   (10925 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
-      .data : 0xc0c00000 - 0xc0c5e280   ( 377 kB)
-       .bss : 0xc0c60000 - 0xc0cbaa88   ( 363 kB)
+      .data : 0xc0c00000 - 0xc0c5e2c0   ( 377 kB)
+       .bss : 0xc0c60000 - 0xc0cbaa8c   ( 363 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -375,7 +375,6 @@
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
-bash: cannot set terminal process group (-1): Inappropriate ioctl for de[    4.091003] usb 2-1: new high-speed USB device number 2 using tegra-ehci
-vice
+bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# 
\ No newline at end of file
+root@(none):/# [    4.090599] usb 2-1: new high-speed USB device number 2 using tegra-ehci
