Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-04-19 22:56:21.695304989 -0700
+++ /tmp/foob	2016-04-19 22:56:21.699304989 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc4-next-20160419+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Apr 18 22:51:20 PDT 2016
+Linux version 4.6.0-rc4-next-20160420+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Apr 19 22:50:35 PDT 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -19,7 +19,7 @@
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
 software IO TLB [mem 0x79c00000-0x7dc00000] (64MB) mapped at [ffffffc039c00000-ffffffc03dbfffff]
-Memory: 936204K/1048576K available (7056K kernel code, 625K rwdata, 2988K rodata, 896K init, 244K bss, 95988K reserved, 16384K cma-reserved)
+Memory: 936204K/1048576K available (7060K kernel code, 625K rwdata, 2984K rodata, 896K init, 244K bss, 95988K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffffff8000000000 - 0xffffff8008000000   (   128 MB)
     vmalloc : 0xffffff8008000000 - 0xffffffbebfff0000   (   250 GB)
@@ -27,6 +27,7 @@
     .rodata : 0xffffff8008760000 - 0xffffff8008a50000   (  3008 KB)
       .init : 0xffffff8008a50000 - 0xffffff8008b30000   (   896 KB)
       .data : 0xffffff8008b30000 - 0xffffff8008bcc600   (   626 KB)
+       .bss : 0xffffff8008bcc600 - 0xffffff8008c097b0   (   245 KB)
     fixed   : 0xffffffbefe7fd000 - 0xffffffbefec00000   (  4108 KB)
     PCI I/O : 0xffffffbefee00000 - 0xffffffbeffe00000   (    16 MB)
     vmemmap : 0xffffffbf00000000 - 0xffffffc000000000   (     4 GB maximum)
@@ -94,7 +95,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.596:1): initialized
+audit: type=2000 audit(0.520:1): initialized
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -154,7 +155,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2016-04-19 05:51:52 UTC (1461045112)
+rtc-pl031 9010000.pl031: setting system clock to 2016-04-20 05:51:06 UTC (1461131466)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-04-19 22:56:21.719304992 -0700
+++ /tmp/foob	2016-04-19 22:56:21.719304992 -0700
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 ########################
-	 3.3 MiB/s
+	 3 MiB/s
 done
-Bytes transferred = 7018632 (6b1888 hex)
+Bytes transferred = 7017832 (6b1568 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 34956 (888c hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6b1888 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6b1568 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff88b ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc4-next-20160419-87959-gb20b2d5 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Apr 18 22:49:32 PDT 2016
+Linux version 4.6.0-rc4-next-20160420-88117-g356ed29 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Apr 19 22:49:08 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c1418600, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1417f40, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1995148K/2096128K available (9526K kernel code, 1129K rwdata, 4160K rodata, 2048K init, 338K bss, 35444K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1995152K/2096128K available (9526K kernel code, 1127K rwdata, 4160K rodata, 2048K init, 338K bss, 35440K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc105dbb8   (14679 kB)
+      .text : 0xc0208000 - 0xc105db50   (14679 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141a640   (1130 kB)
-       .bss : 0xc141c000 - 0xc1470b18   ( 339 kB)
+      .data : 0xc1300000 - 0xc1419f80   (1128 kB)
+       .bss : 0xc141b000 - 0xc146fb18   ( 339 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -121,22 +121,6 @@
 /cpus/cpu@3 missing clock-frequency property
 CPU0: thread -1, cpu 0, socket 0, mpidr 80000000
 Setting up static identity map for 0x80300000 - 0x80300098
-------------[ cut here ]------------
-WARNING: CPU: 0 PID: 1 at arch/arm/mach-omap2/id.c:49 omap_rev+0x3c/0x50
-Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc4-next-20160419-87959-gb20b2d5 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c030f998>] (unwind_backtrace) from [<c030b07c>] (show_stack+0x10/0x14)
-[<c030b07c>] (show_stack) from [<c0583528>] (dump_stack+0x84/0xa4)
-[<c0583528>] (dump_stack) from [<c0341c24>] (__warn+0xd4/0x100)
-[<c0341c24>] (__warn) from [<c0341c70>] (warn_slowpath_null+0x20/0x28)
-[<c0341c70>] (warn_slowpath_null) from [<c032435c>] (omap_rev+0x3c/0x50)
-[<c032435c>] (omap_rev) from [<c11149c8>] (__omap4_sar_ram_init+0x8/0x88)
-[<c11149c8>] (__omap4_sar_ram_init) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
-[<c0301e5c>] (do_one_initcall) from [<c1100ccc>] (kernel_init_freeable+0x70/0x1ec)
-[<c1100ccc>] (kernel_init_freeable) from [<c0b49b70>] (kernel_init+0x8/0x118)
-[<c0b49b70>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
----[ end trace cb88537fdc8fa200 ]---
 Tegra Revision: A01 SKU: 0 CPU Process: 1 SoC Process: 0
 CPU1: thread -1, cpu 1, socket 0, mpidr 80000001
 CPU2: thread -1, cpu 2, socket 0, mpidr 80000002
@@ -276,27 +260,27 @@
 ehci-atmel: EHCI Atmel driver
 tegra-ehci: Tegra EHCI driver
 ------------[ cut here ]------------
-WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
+WARNING: CPU: 1 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Tainted: G        W       4.6.0-rc4-next-20160419-87959-gb20b2d5 #1
+CPU: 1 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc4-next-20160420-88117-g356ed29 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c030f998>] (unwind_backtrace) from [<c030b07c>] (show_stack+0x10/0x14)
 [<c030b07c>] (show_stack) from [<c0583528>] (dump_stack+0x84/0xa4)
 [<c0583528>] (dump_stack) from [<c0341c24>] (__warn+0xd4/0x100)
 [<c0341c24>] (__warn) from [<c0341c70>] (warn_slowpath_null+0x20/0x28)
-[<c0341c70>] (warn_slowpath_null) from [<c0696040>] (__of_reset_control_get+0x1bc/0x228)
-[<c0696040>] (__of_reset_control_get) from [<c09048b4>] (tegra_ehci_probe+0x2d0/0x51c)
-[<c09048b4>] (tegra_ehci_probe) from [<c076edb0>] (platform_drv_probe+0x50/0xb0)
-[<c076edb0>] (platform_drv_probe) from [<c076d714>] (driver_probe_device+0x1f4/0x2b0)
-[<c076d714>] (driver_probe_device) from [<c076d87c>] (__driver_attach+0xac/0xb0)
-[<c076d87c>] (__driver_attach) from [<c076bd28>] (bus_for_each_dev+0x54/0x88)
-[<c076bd28>] (bus_for_each_dev) from [<c076cd08>] (bus_add_driver+0xe8/0x1f4)
-[<c076cd08>] (bus_add_driver) from [<c076e098>] (driver_register+0x78/0xf4)
-[<c076e098>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
+[<c0341c70>] (warn_slowpath_null) from [<c0695f64>] (__of_reset_control_get+0x1bc/0x228)
+[<c0695f64>] (__of_reset_control_get) from [<c0904968>] (tegra_ehci_probe+0x2d0/0x51c)
+[<c0904968>] (tegra_ehci_probe) from [<c076ed00>] (platform_drv_probe+0x50/0xb0)
+[<c076ed00>] (platform_drv_probe) from [<c076d664>] (driver_probe_device+0x1f4/0x2b0)
+[<c076d664>] (driver_probe_device) from [<c076d7cc>] (__driver_attach+0xac/0xb0)
+[<c076d7cc>] (__driver_attach) from [<c076bc78>] (bus_for_each_dev+0x54/0x88)
+[<c076bc78>] (bus_for_each_dev) from [<c076cc58>] (bus_add_driver+0xe8/0x1f4)
+[<c076cc58>] (bus_add_driver) from [<c076dfe8>] (driver_register+0x78/0xf4)
+[<c076dfe8>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
 [<c0301e5c>] (do_one_initcall) from [<c1100d7c>] (kernel_init_freeable+0x120/0x1ec)
-[<c1100d7c>] (kernel_init_freeable) from [<c0b49b70>] (kernel_init+0x8/0x118)
-[<c0b49b70>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
----[ end trace cb88537fdc8fa201 ]---
+[<c1100d7c>] (kernel_init_freeable) from [<c0b49634>] (kernel_init+0x8/0x118)
+[<c0b49634>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
+---[ end trace 4f2569947662aa82 ]---
 tegra-ehci 7d008000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d008000.usb: continuing, but please update your DT
 ohci_hcd: USB 1.1 'Open' Host Controller (OHCI) Driver
@@ -345,10 +329,10 @@
 mmc0: new high speed SDHC card at address aaaa
 mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 [drm] No driver support for vblank timestamp query.
+ mmcblk0: p1 p2 p3
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
- mmcblk0: p1 p2 p3
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
  mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
@@ -362,7 +346,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-11 17:07:58 UTC (950288878)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-12 17:08:41 UTC (950375321)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -374,5 +358,4 @@
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
-root@(none):/# 
\ No newline at end of file
+root@(none):/# [    2.520490] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-04-19 22:56:21.727304993 -0700
+++ /tmp/foob	2016-04-19 22:56:21.731304993 -0700
@@ -37,9 +37,9 @@
 	 #################################################################
 	 #################################################################
 	 #######################################
-	 3.3 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 5332440 (515dd8 hex)
+Bytes transferred = 5332928 (515fc0 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 34956 (888c hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x515dd8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x515fc0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff88b ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc4-next-20160419-87959-gb20b2d5 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Apr 18 22:49:32 PDT 2016
+Linux version 4.6.0-rc4-next-20160420-88117-g356ed29 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Apr 19 22:48:51 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0c5c400, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0c5bdc0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2000996K/2096128K available (7413K kernel code, 375K rwdata, 2528K rodata, 1024K init, 362K bss, 29596K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2000996K/2096128K available (7413K kernel code, 374K rwdata, 2528K rodata, 1024K init, 362K bss, 29596K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,9 +90,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab555c   (10934 kB)
+      .text : 0xc0008000 - 0xc0ab55a0   (10934 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
-      .data : 0xc0c00000 - 0xc0c5df80   ( 376 kB)
+      .data : 0xc0c00000 - 0xc0c5d940   ( 375 kB)
        .bss : 0xc0c5f000 - 0xc0cb9a4c   ( 363 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -252,25 +252,25 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1b4/0x218
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc4-next-20160419-87959-gb20b2d5 #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc4-next-20160420-88117-g356ed29 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c010f1f0>] (unwind_backtrace) from [<c010a860>] (show_stack+0x10/0x14)
 [<c010a860>] (show_stack) from [<c033ab54>] (dump_stack+0x84/0xa4)
 [<c033ab54>] (dump_stack) from [<c011f0dc>] (__warn+0xd4/0x100)
 [<c011f0dc>] (__warn) from [<c011f128>] (warn_slowpath_null+0x20/0x28)
 [<c011f128>] (warn_slowpath_null) from [<c03b96e0>] (__of_reset_control_get+0x1b4/0x218)
-[<c03b96e0>] (__of_reset_control_get) from [<c050ace4>] (tegra_ehci_probe+0x2c8/0x514)
-[<c050ace4>] (tegra_ehci_probe) from [<c04395f8>] (platform_drv_probe+0x50/0xb0)
-[<c04395f8>] (platform_drv_probe) from [<c0438024>] (driver_probe_device+0x1f4/0x2b0)
-[<c0438024>] (driver_probe_device) from [<c043818c>] (__driver_attach+0xac/0xb0)
-[<c043818c>] (__driver_attach) from [<c0436648>] (bus_for_each_dev+0x54/0x88)
-[<c0436648>] (bus_for_each_dev) from [<c0437618>] (bus_add_driver+0xe8/0x1f4)
-[<c0437618>] (bus_add_driver) from [<c0438988>] (driver_register+0x78/0xf4)
-[<c0438988>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
+[<c03b96e0>] (__of_reset_control_get) from [<c050adac>] (tegra_ehci_probe+0x2c8/0x514)
+[<c050adac>] (tegra_ehci_probe) from [<c0439624>] (platform_drv_probe+0x50/0xb0)
+[<c0439624>] (platform_drv_probe) from [<c0438050>] (driver_probe_device+0x1f4/0x2b0)
+[<c0438050>] (driver_probe_device) from [<c04381b8>] (__driver_attach+0xac/0xb0)
+[<c04381b8>] (__driver_attach) from [<c0436674>] (bus_for_each_dev+0x54/0x88)
+[<c0436674>] (bus_for_each_dev) from [<c0437644>] (bus_add_driver+0xe8/0x1f4)
+[<c0437644>] (bus_add_driver) from [<c04389b4>] (driver_register+0x78/0xf4)
+[<c04389b4>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
 [<c010179c>] (do_one_initcall) from [<c0b00d90>] (kernel_init_freeable+0x11c/0x1e8)
-[<c0b00d90>] (kernel_init_freeable) from [<c07df9f8>] (kernel_init+0x8/0x118)
-[<c07df9f8>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
----[ end trace 44cd4bee8ba31fce ]---
+[<c0b00d90>] (kernel_init_freeable) from [<c07dfc04>] (kernel_init+0x8/0x118)
+[<c07dfc04>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
+---[ end trace 30ceeae392aa68de ]---
 tegra-ehci 7d008000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d008000.usb: continuing, but please update your DT
 usbcore: registered new interface driver cdc_acm
@@ -344,7 +344,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-11 17:10:26 UTC (950289026)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-12 17:07:17 UTC (950375237)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-04-19 22:56:21.735304995 -0700
+++ /tmp/foob	2016-04-19 22:56:21.739304996 -0700
@@ -34,7 +34,7 @@
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'zImage.multi_v7_defconfig'.
 Load address: 0x82000000
-Loading: *T #################################################################
+Loading: *#################################################################
 	 #################################################################
 	 #################################################################
 	 #################################################################
@@ -42,21 +42,20 @@
 	 #################################################################
 	 #################################################################
 	 ########################
-	 793.9 KiB/s
+	 2.9 MiB/s
 done
-Bytes transferred = 7018632 (6b1888 hex)
+Bytes transferred = 7017832 (6b1568 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
-Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 2.9 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 64488 (fbe8 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6b1888 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x6b1568 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012be7
@@ -64,7 +63,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc4-next-20160419-87959-gb20b2d5 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Apr 18 22:49:32 PDT 2016
+Linux version 4.6.0-rc4-next-20160420-88117-g356ed29 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Apr 19 22:49:08 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -73,7 +72,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c1418600, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1417f40, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -86,7 +85,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1995040K/2096128K available (9526K kernel code, 1129K rwdata, 4160K rodata, 2048K init, 338K bss, 35552K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1995044K/2096128K available (9526K kernel code, 1127K rwdata, 4160K rodata, 2048K init, 338K bss, 35548K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -94,10 +93,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc105dbb8   (14679 kB)
+      .text : 0xc0208000 - 0xc105db50   (14679 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141a640   (1130 kB)
-       .bss : 0xc141c000 - 0xc1470b18   ( 339 kB)
+      .data : 0xc1300000 - 0xc1419f80   (1128 kB)
+       .bss : 0xc141b000 - 0xc146fb18   ( 339 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -125,22 +124,6 @@
 /cpus/cpu@3 missing clock-frequency property
 CPU0: thread -1, cpu 0, socket 0, mpidr 80000000
 Setting up static identity map for 0x80300000 - 0x80300098
-------------[ cut here ]------------
-WARNING: CPU: 0 PID: 1 at arch/arm/mach-omap2/id.c:49 omap_rev+0x3c/0x50
-Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc4-next-20160419-87959-gb20b2d5 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c030f998>] (unwind_backtrace) from [<c030b07c>] (show_stack+0x10/0x14)
-[<c030b07c>] (show_stack) from [<c0583528>] (dump_stack+0x84/0xa4)
-[<c0583528>] (dump_stack) from [<c0341c24>] (__warn+0xd4/0x100)
-[<c0341c24>] (__warn) from [<c0341c70>] (warn_slowpath_null+0x20/0x28)
-[<c0341c70>] (warn_slowpath_null) from [<c032435c>] (omap_rev+0x3c/0x50)
-[<c032435c>] (omap_rev) from [<c11149c8>] (__omap4_sar_ram_init+0x8/0x88)
-[<c11149c8>] (__omap4_sar_ram_init) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
-[<c0301e5c>] (do_one_initcall) from [<c1100ccc>] (kernel_init_freeable+0x70/0x1ec)
-[<c1100ccc>] (kernel_init_freeable) from [<c0b49b70>] (kernel_init+0x8/0x118)
-[<c0b49b70>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
----[ end trace cb88537fdc8fa200 ]---
 Tegra Revision: A01 SKU: 129 CPU Process: 1 SoC Process: 1
 CPU1: thread -1, cpu 1, socket 0, mpidr 80000001
 CPU2: thread -1, cpu 2, socket 0, mpidr 80000002
@@ -269,25 +252,25 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Tainted: G        W       4.6.0-rc4-next-20160419-87959-gb20b2d5 #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc4-next-20160420-88117-g356ed29 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c030f998>] (unwind_backtrace) from [<c030b07c>] (show_stack+0x10/0x14)
 [<c030b07c>] (show_stack) from [<c0583528>] (dump_stack+0x84/0xa4)
 [<c0583528>] (dump_stack) from [<c0341c24>] (__warn+0xd4/0x100)
 [<c0341c24>] (__warn) from [<c0341c70>] (warn_slowpath_null+0x20/0x28)
-[<c0341c70>] (warn_slowpath_null) from [<c0696040>] (__of_reset_control_get+0x1bc/0x228)
-[<c0696040>] (__of_reset_control_get) from [<c09048b4>] (tegra_ehci_probe+0x2d0/0x51c)
-[<c09048b4>] (tegra_ehci_probe) from [<c076edb0>] (platform_drv_probe+0x50/0xb0)
-[<c076edb0>] (platform_drv_probe) from [<c076d714>] (driver_probe_device+0x1f4/0x2b0)
-[<c076d714>] (driver_probe_device) from [<c076d87c>] (__driver_attach+0xac/0xb0)
-[<c076d87c>] (__driver_attach) from [<c076bd28>] (bus_for_each_dev+0x54/0x88)
-[<c076bd28>] (bus_for_each_dev) from [<c076cd08>] (bus_add_driver+0xe8/0x1f4)
-[<c076cd08>] (bus_add_driver) from [<c076e098>] (driver_register+0x78/0xf4)
-[<c076e098>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
+[<c0341c70>] (warn_slowpath_null) from [<c0695f64>] (__of_reset_control_get+0x1bc/0x228)
+[<c0695f64>] (__of_reset_control_get) from [<c0904968>] (tegra_ehci_probe+0x2d0/0x51c)
+[<c0904968>] (tegra_ehci_probe) from [<c076ed00>] (platform_drv_probe+0x50/0xb0)
+[<c076ed00>] (platform_drv_probe) from [<c076d664>] (driver_probe_device+0x1f4/0x2b0)
+[<c076d664>] (driver_probe_device) from [<c076d7cc>] (__driver_attach+0xac/0xb0)
+[<c076d7cc>] (__driver_attach) from [<c076bc78>] (bus_for_each_dev+0x54/0x88)
+[<c076bc78>] (bus_for_each_dev) from [<c076cc58>] (bus_add_driver+0xe8/0x1f4)
+[<c076cc58>] (bus_add_driver) from [<c076dfe8>] (driver_register+0x78/0xf4)
+[<c076dfe8>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
 [<c0301e5c>] (do_one_initcall) from [<c1100d7c>] (kernel_init_freeable+0x120/0x1ec)
-[<c1100d7c>] (kernel_init_freeable) from [<c0b49b70>] (kernel_init+0x8/0x118)
-[<c0b49b70>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
----[ end trace cb88537fdc8fa201 ]---
+[<c1100d7c>] (kernel_init_freeable) from [<c0b49634>] (kernel_init+0x8/0x118)
+[<c0b49634>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
+---[ end trace 6fe78bebac11aa6f ]---
 tegra-ehci 7d004000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d004000.usb: continuing, but please update your DT
 tegra-ehci 7d004000.usb: EHCI Host Controller
@@ -392,17 +375,11 @@
 tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
 tegra-ahci 70027000.sata: Failed to get regulators
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:01:50 UTC (946684910)
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc1: Unknown controller version (3). You may experience problems.
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:22 UTC (946684822)
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 204000000, volt: 900000, enabled: 1. New: freq: 204000000, volt: 900000, enabled: 1
-+3.3V_AVDD_HDMI_AP_GATED: disabling
-+USB0_VBUS_SW: disabling
-+1.05V_RUN_AVDD_HDMI_PLL: disabling
-+5V_HDMI_CON: disabling
-+5V_SATA: disabling
-+12V_SATA: disabling
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 306000000, volt: 900000, enabled: 1. New: freq: 306000000, volt: 900000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 408000000, volt: 900000, enabled: 1. New: freq: 408000000, volt: 900000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 510000000, volt: 900000, enabled: 1. New: freq: 510000000, volt: 900000, enabled: 1
@@ -431,11 +408,18 @@
 tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
 tegra-ahci 70027000.sata: Failed to get regulators
 usb 2-1: new high-speed USB device number 2 using tegra-ehci
++3.3V_AVDD_HDMI_AP_GATED: disabling
++USB0_VBUS_SW: disabling
++1.05V_RUN_AVDD_HDMI_PLL: disabling
++5V_HDMI_CON: disabling
++5V_SATA: disabling
++12V_SATA: disabling
 asix 2-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:11:31:3b
+EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc1: Unknown controller version (3). You may experience problems.
-EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
+EXT4-fs (mmcblk0p1): couldn't mount as ext2 due to feature incompatibilities
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 204000000, volt: 900000, enabled: 1. New: freq: 204000000, volt: 900000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 306000000, volt: 900000, enabled: 1. New: freq: 306000000, volt: 900000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 408000000, volt: 900000, enabled: 1. New: freq: 408000000, volt: 900000, enabled: 1
@@ -464,7 +448,6 @@
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
 tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
 tegra-ahci 70027000.sata: Failed to get regulators
-EXT4-fs (mmcblk0p1): couldn't mount as ext2 due to feature incompatibilities
 EXT4-fs (mmcblk0p1): warning: mounting fs with errors, running e2fsck is recommended
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-04-19 22:56:21.743304996 -0700
+++ /tmp/foob	2016-04-19 22:56:21.747304997 -0700
@@ -41,18 +41,18 @@
 	 #######################################
 	 3.2 MiB/s
 done
-Bytes transferred = 5332440 (515dd8 hex)
+Bytes transferred = 5332928 (515fc0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.1 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 64488 (fbe8 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x515dd8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x515fc0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012be7
@@ -61,7 +61,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc4-next-20160419-87959-gb20b2d5 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Apr 18 22:49:32 PDT 2016
+Linux version 4.6.0-rc4-next-20160420-88117-g356ed29 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Apr 19 22:48:51 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -71,7 +71,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0c5c400, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0c5bdc0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -84,7 +84,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2000888K/2096128K available (7413K kernel code, 375K rwdata, 2528K rodata, 1024K init, 362K bss, 29704K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2000888K/2096128K available (7413K kernel code, 374K rwdata, 2528K rodata, 1024K init, 362K bss, 29704K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,9 +92,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab555c   (10934 kB)
+      .text : 0xc0008000 - 0xc0ab55a0   (10934 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
-      .data : 0xc0c00000 - 0xc0c5df80   ( 376 kB)
+      .data : 0xc0c00000 - 0xc0c5d940   ( 375 kB)
        .bss : 0xc0c5f000 - 0xc0cb9a4c   ( 363 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -241,25 +241,25 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1b4/0x218
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc4-next-20160419-87959-gb20b2d5 #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc4-next-20160420-88117-g356ed29 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c010f1f0>] (unwind_backtrace) from [<c010a860>] (show_stack+0x10/0x14)
 [<c010a860>] (show_stack) from [<c033ab54>] (dump_stack+0x84/0xa4)
 [<c033ab54>] (dump_stack) from [<c011f0dc>] (__warn+0xd4/0x100)
 [<c011f0dc>] (__warn) from [<c011f128>] (warn_slowpath_null+0x20/0x28)
 [<c011f128>] (warn_slowpath_null) from [<c03b96e0>] (__of_reset_control_get+0x1b4/0x218)
-[<c03b96e0>] (__of_reset_control_get) from [<c050ace4>] (tegra_ehci_probe+0x2c8/0x514)
-[<c050ace4>] (tegra_ehci_probe) from [<c04395f8>] (platform_drv_probe+0x50/0xb0)
-[<c04395f8>] (platform_drv_probe) from [<c0438024>] (driver_probe_device+0x1f4/0x2b0)
-[<c0438024>] (driver_probe_device) from [<c043818c>] (__driver_attach+0xac/0xb0)
-[<c043818c>] (__driver_attach) from [<c0436648>] (bus_for_each_dev+0x54/0x88)
-[<c0436648>] (bus_for_each_dev) from [<c0437618>] (bus_add_driver+0xe8/0x1f4)
-[<c0437618>] (bus_add_driver) from [<c0438988>] (driver_register+0x78/0xf4)
-[<c0438988>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
+[<c03b96e0>] (__of_reset_control_get) from [<c050adac>] (tegra_ehci_probe+0x2c8/0x514)
+[<c050adac>] (tegra_ehci_probe) from [<c0439624>] (platform_drv_probe+0x50/0xb0)
+[<c0439624>] (platform_drv_probe) from [<c0438050>] (driver_probe_device+0x1f4/0x2b0)
+[<c0438050>] (driver_probe_device) from [<c04381b8>] (__driver_attach+0xac/0xb0)
+[<c04381b8>] (__driver_attach) from [<c0436674>] (bus_for_each_dev+0x54/0x88)
+[<c0436674>] (bus_for_each_dev) from [<c0437644>] (bus_add_driver+0xe8/0x1f4)
+[<c0437644>] (bus_add_driver) from [<c04389b4>] (driver_register+0x78/0xf4)
+[<c04389b4>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
 [<c010179c>] (do_one_initcall) from [<c0b00d90>] (kernel_init_freeable+0x11c/0x1e8)
-[<c0b00d90>] (kernel_init_freeable) from [<c07df9f8>] (kernel_init+0x8/0x118)
-[<c07df9f8>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
----[ end trace cfad813864a7e67e ]---
+[<c0b00d90>] (kernel_init_freeable) from [<c07dfc04>] (kernel_init+0x8/0x118)
+[<c07dfc04>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
+---[ end trace 3a01120f3ad9dd90 ]---
 tegra-ehci 7d004000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d004000.usb: continuing, but please update your DT
 tegra-ehci 7d004000.usb: EHCI Host Controller
@@ -379,7 +379,7 @@
 tegra-ahci 70027000.sata: Failed to get regulators
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:26 UTC (946684946)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:15 UTC (946684815)
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc1: Unknown controller version (3). You may experience problems.
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 204000000, volt: 900000, enabled: 1. New: freq: 204000000, volt: 900000, enabled: 1
@@ -425,8 +425,8 @@
 mmc1: Unknown controller version (3). You may experience problems.
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 204000000, volt: 900000, enabled: 1. New: freq: 204000000, volt: 900000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 306000000, volt: 900000, enabled: 1. New: freq: 306000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 408000000, volt: 900000, enabled: 1. New: freq: 408000000, volt: 900000, enabled: 1
 EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
+cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 408000000, volt: 900000, enabled: 1. New: freq: 408000000, volt: 900000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 510000000, volt: 900000, enabled: 1. New: freq: 510000000, volt: 900000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 612000000, volt: 900000, enabled: 1. New: freq: 612000000, volt: 900000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 714000000, volt: 900000, enabled: 1. New: freq: 714000000, volt: 900000, enabled: 1
@@ -447,20 +447,20 @@
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2218500000, volt: 1140000, enabled: 1. New: freq: 2218500000, volt: 1140000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2320500000, volt: 1180000, enabled: 1. New: freq: 2320500000, volt: 1180000, enabled: 1
 tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
-EXT4-fs (mmcblk0p1): warning: mounting fs with errors, running e2fsck is recommended
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
-EXT4-fs (mmcblk0p1): recovery complete
-EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext4 filesystem) on device 179:1.
-devtmpfs: mounted
-Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
 tegra-pcie 1003000.pcie-controller: Failed to get supply 'avddio-pex': -517
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
 tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
 tegra-ahci 70027000.sata: Failed to get regulators
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
+EXT4-fs (mmcblk0p1): warning: mounting fs with errors, running e2fsck is recommended
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc1: Unknown controller version (3). You may experience problems.
+EXT4-fs (mmcblk0p1): recovery complete
+EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
+VFS: Mounted root (ext4 filesystem) on device 179:1.
+devtmpfs: mounted
+Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2016-04-19 22:56:21.751304997 -0700
+++ /tmp/foob	2016-04-19 22:56:21.751304997 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1801053 us
+Bootloader-Cpu Init at (time stamp): 1801160 us
 
 Pinmux changes applied in kernel way
 
@@ -62,22 +62,22 @@
 Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1801027  us
-Bootloader (AosInit) time: 734248 us
-Bootloader Main Init time: 180100 us
-BlockDevMgrInit time: 157516 us
+Miscellaneous avp time: 1801134  us
+Bootloader (AosInit) time: 734254 us
+Bootloader Main Init time: 180088 us
+BlockDevMgrInit time: 157535 us
 PartMgrInit time: 8 us
 StorMgrInit time: 30 us
-FsMgrInit time: 3 us
-Partition Table load: 76752 us 
-Display init time: 574841 us
+FsMgrInit time: 2 us
+Partition Table load: 76710 us 
+Display init time: 574690 us
 Kernel read time: 313 msec
 Load Kernel Size:   13477888 Bytes
 Bl Kernel ReadSpeed: 43060000 Bytes/sec
-Pre-Kernel Jump time: 1520810 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5540654 us
+Pre-Kernel Jump time: 1520933 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5541949 us
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc4-next-20160419+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Apr 18 22:51:20 PDT 2016
+Linux version 4.6.0-rc4-next-20160420+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Apr 19 22:50:35 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
 Memory limited to 1734MB
 efi: Getting EFI parameters from FDT:
@@ -92,7 +92,7 @@
 Dentry cache hash table entries: 262144 (order: 9, 2097152 bytes)
 Inode-cache hash table entries: 131072 (order: 8, 1048576 bytes)
 software IO TLB [mem 0xe5200000-0xe9200000] (64MB) mapped at [ffffffc065200000-ffffffc0691fffff]
-Memory: 1643572K/1775616K available (7056K kernel code, 625K rwdata, 2988K rodata, 896K init, 244K bss, 115660K reserved, 16384K cma-reserved)
+Memory: 1643572K/1775616K available (7060K kernel code, 625K rwdata, 2984K rodata, 896K init, 244K bss, 115660K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffffff8000000000 - 0xffffff8008000000   (   128 MB)
     vmalloc : 0xffffff8008000000 - 0xffffffbebfff0000   (   250 GB)
@@ -100,6 +100,7 @@
     .rodata : 0xffffff8008760000 - 0xffffff8008a50000   (  3008 KB)
       .init : 0xffffff8008a50000 - 0xffffff8008b30000   (   896 KB)
       .data : 0xffffff8008b30000 - 0xffffff8008bcc600   (   626 KB)
+       .bss : 0xffffff8008bcc600 - 0xffffff8008c097b0   (   245 KB)
     fixed   : 0xffffffbefe7fd000 - 0xffffffbefec00000   (  4108 KB)
     PCI I/O : 0xffffffbefee00000 - 0xffffffbeffe00000   (    16 MB)
     vmemmap : 0xffffffbf00000000 - 0xffffffc000000000   (     4 GB maximum)
@@ -181,7 +182,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.851:1): initialized
+audit: type=2000 audit(0.847:1): initialized
 workingset: timestamp_bits=44 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -242,21 +243,21 @@
 mmc0: new high speed MMC card at address 0001
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmc1: Unknown controller version (3). You may experience problems.
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-input: gpio-keys as /devices/platform/gpio-keys/input/input0
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+sdhci-tegra 700b0400.sdhci: Got WP GPIO
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
+mmc1: Unknown controller version (3). You may experience problems.
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc1: Unknown controller version (3). You may experience problems.
-hctosys: unable to open rtc device (rtc0)
-+3.3V_AVDD_HDMI_AP_GATED: disabling
+input: gpio-keys as /devices/platform/gpio-keys/input/input0
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
+hctosys: unable to open rtc device (rtc0)
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc1: Unknown controller version (3). You may experience problems.
++3.3V_AVDD_HDMI_AP_GATED: disabling
 +VDD_LED: disabling
 +5V_USB_HS: disabling
 +5V_USB_SS: disabling
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-04-19 22:56:21.759304998 -0700
+++ /tmp/foob	2016-04-19 22:56:21.759304998 -0700
@@ -47,20 +47,20 @@
 	 #################################################################
 	 #################################################################
 	 ########################
-	 911.1 KiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 7018632 (6b1888 hex)
+Bytes transferred = 7017832 (6b1568 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 861.3 KiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x6b1888 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x6b1568 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102b3
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc4-next-20160419-87959-gb20b2d5 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Apr 18 22:49:32 PDT 2016
+Linux version 4.6.0-rc4-next-20160420-88117-g356ed29 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Apr 19 22:49:08 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -76,7 +76,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c1418600, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c1417f40, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -89,7 +89,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 955956K/1048576K available (9526K kernel code, 1129K rwdata, 4160K rodata, 2048K init, 338K bss, 27084K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 955960K/1048576K available (9526K kernel code, 1127K rwdata, 4160K rodata, 2048K init, 338K bss, 27080K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -97,10 +97,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc105dbb8   (14679 kB)
+      .text : 0xc0208000 - 0xc105db50   (14679 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141a640   (1130 kB)
-       .bss : 0xc141c000 - 0xc1470b18   ( 339 kB)
+      .data : 0xc1300000 - 0xc1419f80   (1128 kB)
+       .bss : 0xc141b000 - 0xc146fb18   ( 339 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -127,22 +127,6 @@
 CPU: Testing write buffer coherency: ok
 CPU0: thread -1, cpu 0, socket 0, mpidr 80000000
 Setting up static identity map for 0x300000 - 0x300098
-------------[ cut here ]------------
-WARNING: CPU: 0 PID: 1 at arch/arm/mach-omap2/id.c:49 omap_rev+0x3c/0x50
-Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc4-next-20160419-87959-gb20b2d5 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c030f998>] (unwind_backtrace) from [<c030b07c>] (show_stack+0x10/0x14)
-[<c030b07c>] (show_stack) from [<c0583528>] (dump_stack+0x84/0xa4)
-[<c0583528>] (dump_stack) from [<c0341c24>] (__warn+0xd4/0x100)
-[<c0341c24>] (__warn) from [<c0341c70>] (warn_slowpath_null+0x20/0x28)
-[<c0341c70>] (warn_slowpath_null) from [<c032435c>] (omap_rev+0x3c/0x50)
-[<c032435c>] (omap_rev) from [<c11149c8>] (__omap4_sar_ram_init+0x8/0x88)
-[<c11149c8>] (__omap4_sar_ram_init) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
-[<c0301e5c>] (do_one_initcall) from [<c1100ccc>] (kernel_init_freeable+0x70/0x1ec)
-[<c1100ccc>] (kernel_init_freeable) from [<c0b49b70>] (kernel_init+0x8/0x118)
-[<c0b49b70>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
----[ end trace cb88537fdc8fa200 ]---
 Tegra Revision: A03 prime SKU: 8 CPU Process: 1 SoC Process: 2
 CPU1: thread -1, cpu 1, socket 0, mpidr 80000001
 Brought up 2 CPUs
@@ -289,25 +273,25 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Tainted: G        W       4.6.0-rc4-next-20160419-87959-gb20b2d5 #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc4-next-20160420-88117-g356ed29 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c030f998>] (unwind_backtrace) from [<c030b07c>] (show_stack+0x10/0x14)
 [<c030b07c>] (show_stack) from [<c0583528>] (dump_stack+0x84/0xa4)
 [<c0583528>] (dump_stack) from [<c0341c24>] (__warn+0xd4/0x100)
 [<c0341c24>] (__warn) from [<c0341c70>] (warn_slowpath_null+0x20/0x28)
-[<c0341c70>] (warn_slowpath_null) from [<c0696040>] (__of_reset_control_get+0x1bc/0x228)
-[<c0696040>] (__of_reset_control_get) from [<c09048b4>] (tegra_ehci_probe+0x2d0/0x51c)
-[<c09048b4>] (tegra_ehci_probe) from [<c076edb0>] (platform_drv_probe+0x50/0xb0)
-[<c076edb0>] (platform_drv_probe) from [<c076d714>] (driver_probe_device+0x1f4/0x2b0)
-[<c076d714>] (driver_probe_device) from [<c076d87c>] (__driver_attach+0xac/0xb0)
-[<c076d87c>] (__driver_attach) from [<c076bd28>] (bus_for_each_dev+0x54/0x88)
-[<c076bd28>] (bus_for_each_dev) from [<c076cd08>] (bus_add_driver+0xe8/0x1f4)
-[<c076cd08>] (bus_add_driver) from [<c076e098>] (driver_register+0x78/0xf4)
-[<c076e098>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
+[<c0341c70>] (warn_slowpath_null) from [<c0695f64>] (__of_reset_control_get+0x1bc/0x228)
+[<c0695f64>] (__of_reset_control_get) from [<c0904968>] (tegra_ehci_probe+0x2d0/0x51c)
+[<c0904968>] (tegra_ehci_probe) from [<c076ed00>] (platform_drv_probe+0x50/0xb0)
+[<c076ed00>] (platform_drv_probe) from [<c076d664>] (driver_probe_device+0x1f4/0x2b0)
+[<c076d664>] (driver_probe_device) from [<c076d7cc>] (__driver_attach+0xac/0xb0)
+[<c076d7cc>] (__driver_attach) from [<c076bc78>] (bus_for_each_dev+0x54/0x88)
+[<c076bc78>] (bus_for_each_dev) from [<c076cc58>] (bus_add_driver+0xe8/0x1f4)
+[<c076cc58>] (bus_add_driver) from [<c076dfe8>] (driver_register+0x78/0xf4)
+[<c076dfe8>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
 [<c0301e5c>] (do_one_initcall) from [<c1100d7c>] (kernel_init_freeable+0x120/0x1ec)
-[<c1100d7c>] (kernel_init_freeable) from [<c0b49b70>] (kernel_init+0x8/0x118)
-[<c0b49b70>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
----[ end trace cb88537fdc8fa201 ]---
+[<c1100d7c>] (kernel_init_freeable) from [<c0b49634>] (kernel_init+0x8/0x118)
+[<c0b49634>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
+---[ end trace 8363059ac356f916 ]---
 tegra-ehci c5000000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci c5000000.usb: continuing, but please update your DT
 ulpi_phy_power_on: ulpi write failed
@@ -364,24 +348,24 @@
 mmc0: host does not support reading read-only switch, assuming write-enable
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 mmc0: new high speed SDHC card at address b368
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-28 17:19:55 UTC (1419787195)
+rtc-em3027 2-0056: setting system clock to 2014-12-28 17:32:11 UTC (1419787931)
  mmcblk0: p1
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
+hub 1-1:1.0: USB hub found
+hub 1-1:1.0: 4 ports detected
+usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
-hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-04-19 22:56:21.767305000 -0700
+++ /tmp/foob	2016-04-19 22:56:21.767305000 -0700
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -45,9 +45,9 @@
 	 #################################################################
 	 #################################################################
 	 #######################################
-	 904.3 KiB/s
+	 906.3 KiB/s
 done
-Bytes transferred = 5332440 (515dd8 hex)
+Bytes transferred = 5332928 (515fc0 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x515dd8 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x515fc0 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102b3
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc4-next-20160419-87959-gb20b2d5 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Apr 18 22:49:32 PDT 2016
+Linux version 4.6.0-rc4-next-20160420-88117-g356ed29 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Apr 19 22:48:51 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -76,7 +76,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0c5c400, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0c5bdc0, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -89,7 +89,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 961804K/1048576K available (7413K kernel code, 375K rwdata, 2528K rodata, 1024K init, 362K bss, 21236K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 961804K/1048576K available (7413K kernel code, 374K rwdata, 2528K rodata, 1024K init, 362K bss, 21236K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -97,9 +97,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab555c   (10934 kB)
+      .text : 0xc0008000 - 0xc0ab55a0   (10934 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
-      .data : 0xc0c00000 - 0xc0c5df80   ( 376 kB)
+      .data : 0xc0c00000 - 0xc0c5d940   ( 375 kB)
        .bss : 0xc0c5f000 - 0xc0cb9a4c   ( 363 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -265,25 +265,25 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1b4/0x218
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc4-next-20160419-87959-gb20b2d5 #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc4-next-20160420-88117-g356ed29 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c010f1f0>] (unwind_backtrace) from [<c010a860>] (show_stack+0x10/0x14)
 [<c010a860>] (show_stack) from [<c033ab54>] (dump_stack+0x84/0xa4)
 [<c033ab54>] (dump_stack) from [<c011f0dc>] (__warn+0xd4/0x100)
 [<c011f0dc>] (__warn) from [<c011f128>] (warn_slowpath_null+0x20/0x28)
 [<c011f128>] (warn_slowpath_null) from [<c03b96e0>] (__of_reset_control_get+0x1b4/0x218)
-[<c03b96e0>] (__of_reset_control_get) from [<c050ace4>] (tegra_ehci_probe+0x2c8/0x514)
-[<c050ace4>] (tegra_ehci_probe) from [<c04395f8>] (platform_drv_probe+0x50/0xb0)
-[<c04395f8>] (platform_drv_probe) from [<c0438024>] (driver_probe_device+0x1f4/0x2b0)
-[<c0438024>] (driver_probe_device) from [<c043818c>] (__driver_attach+0xac/0xb0)
-[<c043818c>] (__driver_attach) from [<c0436648>] (bus_for_each_dev+0x54/0x88)
-[<c0436648>] (bus_for_each_dev) from [<c0437618>] (bus_add_driver+0xe8/0x1f4)
-[<c0437618>] (bus_add_driver) from [<c0438988>] (driver_register+0x78/0xf4)
-[<c0438988>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
+[<c03b96e0>] (__of_reset_control_get) from [<c050adac>] (tegra_ehci_probe+0x2c8/0x514)
+[<c050adac>] (tegra_ehci_probe) from [<c0439624>] (platform_drv_probe+0x50/0xb0)
+[<c0439624>] (platform_drv_probe) from [<c0438050>] (driver_probe_device+0x1f4/0x2b0)
+[<c0438050>] (driver_probe_device) from [<c04381b8>] (__driver_attach+0xac/0xb0)
+[<c04381b8>] (__driver_attach) from [<c0436674>] (bus_for_each_dev+0x54/0x88)
+[<c0436674>] (bus_for_each_dev) from [<c0437644>] (bus_add_driver+0xe8/0x1f4)
+[<c0437644>] (bus_add_driver) from [<c04389b4>] (driver_register+0x78/0xf4)
+[<c04389b4>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
 [<c010179c>] (do_one_initcall) from [<c0b00d90>] (kernel_init_freeable+0x11c/0x1e8)
-[<c0b00d90>] (kernel_init_freeable) from [<c07df9f8>] (kernel_init+0x8/0x118)
-[<c07df9f8>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
----[ end trace 61f89d418f8bc350 ]---
+[<c0b00d90>] (kernel_init_freeable) from [<c07dfc04>] (kernel_init+0x8/0x118)
+[<c07dfc04>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
+---[ end trace b2160c623d1576ae ]---
 tegra-ehci c5000000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci c5000000.usb: continuing, but please update your DT
 ulpi_phy_power_on: ulpi write failed
@@ -358,22 +358,22 @@
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
+hub 1-1:1.0: 4 ports detected
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-28 17:21:22 UTC (1419787282)
+rtc-em3027 2-0056: setting system clock to 2014-12-28 17:31:09 UTC (1419787869)
 ALSA device list:
   #0: tegra-trimslice
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
+usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
-usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# 
\ No newline at end of file
+root@(none):/# [    3.306303] asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2016-04-19 22:56:21.775305001 -0700
+++ /tmp/foob	2016-04-19 22:56:21.775305001 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 721092 us
-Basic modules init took 349098 us
+Tegraboot started after 821221 us
+Basic modules init took 351338 us
 Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14387 us
-Carveout took 252714 us
-CPU initialization took 92324 us
-Total time taken by TegraBoot 708523 us
+Next stage binary read took 14370 us
+Carveout took 252728 us
+CPU initialization took 92326 us
+Total time taken by TegraBoot 710762 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.561] LPDDR4 Training: Number of tables = 10
+64b[0001.661] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-11847168 bytes read in 727 ms (15.5 MiB/s)
+11847168 bytes read in 728 ms (15.5 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-37076 bytes read in 454 ms (79.1 KiB/s)
+37076 bytes read in 453 ms (79.1 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 494 ms (3.1 MiB/s)
+1627787 bytes read in 495 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc4-next-20160419+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Apr 18 22:51:20 PDT 2016
+Linux version 4.6.0-rc4-next-20160420+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Apr 19 22:50:35 PDT 2016
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -166,7 +166,7 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffffffc07a000000-ffffffc07dffffff]
-Memory: 2981152K/3131392K available (7056K kernel code, 625K rwdata, 2988K rodata, 896K init, 244K bss, 133856K reserved, 16384K cma-reserved)
+Memory: 2981152K/3131392K available (7060K kernel code, 625K rwdata, 2984K rodata, 896K init, 244K bss, 133856K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffffff8000000000 - 0xffffff8008000000   (   128 MB)
     vmalloc : 0xffffff8008000000 - 0xffffffbebfff0000   (   250 GB)
@@ -174,6 +174,7 @@
     .rodata : 0xffffff8008760000 - 0xffffff8008a50000   (  3008 KB)
       .init : 0xffffff8008a50000 - 0xffffff8008b30000   (   896 KB)
       .data : 0xffffff8008b30000 - 0xffffff8008bcc600   (   626 KB)
+       .bss : 0xffffff8008bcc600 - 0xffffff8008c097b0   (   245 KB)
     fixed   : 0xffffffbefe7fd000 - 0xffffffbefec00000   (  4108 KB)
     PCI I/O : 0xffffffbefee00000 - 0xffffffbeffe00000   (    16 MB)
     vmemmap : 0xffffffbf00000000 - 0xffffffc000000000   (     4 GB maximum)
@@ -259,7 +260,7 @@
 kvm [1]: timer IRQ3
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.355:1): initialized
+audit: type=2000 audit(0.359:1): initialized
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-04-19 22:56:21.783305002 -0700
+++ /tmp/foob	2016-04-19 22:56:21.783305002 -0700
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 ########################
-	 3.3 MiB/s
+	 3.1 MiB/s
 done
-Bytes transferred = 7018632 (6b1888 hex)
+Bytes transferred = 7017832 (6b1568 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 51511 (c937 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6b1888 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6b1568 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff936 ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc4-next-20160419-87959-gb20b2d5 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Apr 18 22:49:32 PDT 2016
+Linux version 4.6.0-rc4-next-20160420-88117-g356ed29 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Apr 19 22:49:08 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,7 +68,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c1418600, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1417f40, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -81,7 +81,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1995068K/2096128K available (9526K kernel code, 1129K rwdata, 4160K rodata, 2048K init, 338K bss, 35524K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1995072K/2096128K available (9526K kernel code, 1127K rwdata, 4160K rodata, 2048K init, 338K bss, 35520K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -89,10 +89,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc105dbb8   (14679 kB)
+      .text : 0xc0208000 - 0xc105db50   (14679 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141a640   (1130 kB)
-       .bss : 0xc141c000 - 0xc1470b18   ( 339 kB)
+      .data : 0xc1300000 - 0xc1419f80   (1128 kB)
+       .bss : 0xc141b000 - 0xc146fb18   ( 339 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -119,22 +119,6 @@
 CPU: Testing write buffer coherency: ok
 CPU0: thread -1, cpu 0, socket 0, mpidr 80000000
 Setting up static identity map for 0x80300000 - 0x80300098
-------------[ cut here ]------------
-WARNING: CPU: 0 PID: 1 at arch/arm/mach-omap2/id.c:49 omap_rev+0x3c/0x50
-Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc4-next-20160419-87959-gb20b2d5 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c030f998>] (unwind_backtrace) from [<c030b07c>] (show_stack+0x10/0x14)
-[<c030b07c>] (show_stack) from [<c0583528>] (dump_stack+0x84/0xa4)
-[<c0583528>] (dump_stack) from [<c0341c24>] (__warn+0xd4/0x100)
-[<c0341c24>] (__warn) from [<c0341c70>] (warn_slowpath_null+0x20/0x28)
-[<c0341c70>] (warn_slowpath_null) from [<c032435c>] (omap_rev+0x3c/0x50)
-[<c032435c>] (omap_rev) from [<c11149c8>] (__omap4_sar_ram_init+0x8/0x88)
-[<c11149c8>] (__omap4_sar_ram_init) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
-[<c0301e5c>] (do_one_initcall) from [<c1100ccc>] (kernel_init_freeable+0x70/0x1ec)
-[<c1100ccc>] (kernel_init_freeable) from [<c0b49b70>] (kernel_init+0x8/0x118)
-[<c0b49b70>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
----[ end trace cb88537fdc8fa200 ]---
 Tegra Revision: A03 SKU: 128 CPU Process: 4 SoC Process: 0
 CPU1: thread -1, cpu 1, socket 0, mpidr 80000001
 CPU2: thread -1, cpu 2, socket 0, mpidr 80000002
@@ -260,27 +244,27 @@
 ehci-atmel: EHCI Atmel driver
 tegra-ehci: Tegra EHCI driver
 ------------[ cut here ]------------
-WARNING: CPU: 2 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
+WARNING: CPU: 1 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
 Modules linked in:
-CPU: 2 PID: 1 Comm: swapper/0 Tainted: G        W       4.6.0-rc4-next-20160419-87959-gb20b2d5 #1
+CPU: 1 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc4-next-20160420-88117-g356ed29 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c030f998>] (unwind_backtrace) from [<c030b07c>] (show_stack+0x10/0x14)
 [<c030b07c>] (show_stack) from [<c0583528>] (dump_stack+0x84/0xa4)
 [<c0583528>] (dump_stack) from [<c0341c24>] (__warn+0xd4/0x100)
 [<c0341c24>] (__warn) from [<c0341c70>] (warn_slowpath_null+0x20/0x28)
-[<c0341c70>] (warn_slowpath_null) from [<c0696040>] (__of_reset_control_get+0x1bc/0x228)
-[<c0696040>] (__of_reset_control_get) from [<c09048b4>] (tegra_ehci_probe+0x2d0/0x51c)
-[<c09048b4>] (tegra_ehci_probe) from [<c076edb0>] (platform_drv_probe+0x50/0xb0)
-[<c076edb0>] (platform_drv_probe) from [<c076d714>] (driver_probe_device+0x1f4/0x2b0)
-[<c076d714>] (driver_probe_device) from [<c076d87c>] (__driver_attach+0xac/0xb0)
-[<c076d87c>] (__driver_attach) from [<c076bd28>] (bus_for_each_dev+0x54/0x88)
-[<c076bd28>] (bus_for_each_dev) from [<c076cd08>] (bus_add_driver+0xe8/0x1f4)
-[<c076cd08>] (bus_add_driver) from [<c076e098>] (driver_register+0x78/0xf4)
-[<c076e098>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
+[<c0341c70>] (warn_slowpath_null) from [<c0695f64>] (__of_reset_control_get+0x1bc/0x228)
+[<c0695f64>] (__of_reset_control_get) from [<c0904968>] (tegra_ehci_probe+0x2d0/0x51c)
+[<c0904968>] (tegra_ehci_probe) from [<c076ed00>] (platform_drv_probe+0x50/0xb0)
+[<c076ed00>] (platform_drv_probe) from [<c076d664>] (driver_probe_device+0x1f4/0x2b0)
+[<c076d664>] (driver_probe_device) from [<c076d7cc>] (__driver_attach+0xac/0xb0)
+[<c076d7cc>] (__driver_attach) from [<c076bc78>] (bus_for_each_dev+0x54/0x88)
+[<c076bc78>] (bus_for_each_dev) from [<c076cc58>] (bus_add_driver+0xe8/0x1f4)
+[<c076cc58>] (bus_add_driver) from [<c076dfe8>] (driver_register+0x78/0xf4)
+[<c076dfe8>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
 [<c0301e5c>] (do_one_initcall) from [<c1100d7c>] (kernel_init_freeable+0x120/0x1ec)
-[<c1100d7c>] (kernel_init_freeable) from [<c0b49b70>] (kernel_init+0x8/0x118)
-[<c0b49b70>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
----[ end trace cb88537fdc8fa201 ]---
+[<c1100d7c>] (kernel_init_freeable) from [<c0b49634>] (kernel_init+0x8/0x118)
+[<c0b49634>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
+---[ end trace 07d97733c5de6ed3 ]---
 tegra-ehci 7d004000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d004000.usb: continuing, but please update your DT
 ohci_hcd: USB 1.1 'Open' Host Controller (OHCI) Driver
@@ -416,6 +400,5 @@
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
-bash: no j[    4.103926] usb 2-1: new high-speed USB device number 2 using tegra-ehci
-ob control in this shell
-root@(none):/# 
\ No newline at end of file
+bash: no job control in this shell
+root@(none):/# [    3.993906] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-04-19 22:56:21.791305003 -0700
+++ /tmp/foob	2016-04-19 22:56:21.791305003 -0700
@@ -37,9 +37,9 @@
 	 #################################################################
 	 #################################################################
 	 #######################################
-	 2.8 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 5332440 (515dd8 hex)
+Bytes transferred = 5332928 (515fc0 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 51511 (c937 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x515dd8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x515fc0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff936 ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc4-next-20160419-87959-gb20b2d5 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Apr 18 22:49:32 PDT 2016
+Linux version 4.6.0-rc4-next-20160420-88117-g356ed29 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Apr 19 22:48:51 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,7 +68,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0c5c400, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0c5bdc0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -81,7 +81,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2000916K/2096128K available (7413K kernel code, 375K rwdata, 2528K rodata, 1024K init, 362K bss, 29676K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2000916K/2096128K available (7413K kernel code, 374K rwdata, 2528K rodata, 1024K init, 362K bss, 29676K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -89,9 +89,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab555c   (10934 kB)
+      .text : 0xc0008000 - 0xc0ab55a0   (10934 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
-      .data : 0xc0c00000 - 0xc0c5df80   ( 376 kB)
+      .data : 0xc0c00000 - 0xc0c5d940   ( 375 kB)
        .bss : 0xc0c5f000 - 0xc0cb9a4c   ( 363 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -236,25 +236,25 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1b4/0x218
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc4-next-20160419-87959-gb20b2d5 #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc4-next-20160420-88117-g356ed29 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c010f1f0>] (unwind_backtrace) from [<c010a860>] (show_stack+0x10/0x14)
 [<c010a860>] (show_stack) from [<c033ab54>] (dump_stack+0x84/0xa4)
 [<c033ab54>] (dump_stack) from [<c011f0dc>] (__warn+0xd4/0x100)
 [<c011f0dc>] (__warn) from [<c011f128>] (warn_slowpath_null+0x20/0x28)
 [<c011f128>] (warn_slowpath_null) from [<c03b96e0>] (__of_reset_control_get+0x1b4/0x218)
-[<c03b96e0>] (__of_reset_control_get) from [<c050ace4>] (tegra_ehci_probe+0x2c8/0x514)
-[<c050ace4>] (tegra_ehci_probe) from [<c04395f8>] (platform_drv_probe+0x50/0xb0)
-[<c04395f8>] (platform_drv_probe) from [<c0438024>] (driver_probe_device+0x1f4/0x2b0)
-[<c0438024>] (driver_probe_device) from [<c043818c>] (__driver_attach+0xac/0xb0)
-[<c043818c>] (__driver_attach) from [<c0436648>] (bus_for_each_dev+0x54/0x88)
-[<c0436648>] (bus_for_each_dev) from [<c0437618>] (bus_add_driver+0xe8/0x1f4)
-[<c0437618>] (bus_add_driver) from [<c0438988>] (driver_register+0x78/0xf4)
-[<c0438988>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
+[<c03b96e0>] (__of_reset_control_get) from [<c050adac>] (tegra_ehci_probe+0x2c8/0x514)
+[<c050adac>] (tegra_ehci_probe) from [<c0439624>] (platform_drv_probe+0x50/0xb0)
+[<c0439624>] (platform_drv_probe) from [<c0438050>] (driver_probe_device+0x1f4/0x2b0)
+[<c0438050>] (driver_probe_device) from [<c04381b8>] (__driver_attach+0xac/0xb0)
+[<c04381b8>] (__driver_attach) from [<c0436674>] (bus_for_each_dev+0x54/0x88)
+[<c0436674>] (bus_for_each_dev) from [<c0437644>] (bus_add_driver+0xe8/0x1f4)
+[<c0437644>] (bus_add_driver) from [<c04389b4>] (driver_register+0x78/0xf4)
+[<c04389b4>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
 [<c010179c>] (do_one_initcall) from [<c0b00d90>] (kernel_init_freeable+0x11c/0x1e8)
-[<c0b00d90>] (kernel_init_freeable) from [<c07df9f8>] (kernel_init+0x8/0x118)
-[<c07df9f8>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
----[ end trace 860f0a518cb17497 ]---
+[<c0b00d90>] (kernel_init_freeable) from [<c07dfc04>] (kernel_init+0x8/0x118)
+[<c07dfc04>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
+---[ end trace 5ad4755e39d2ea2e ]---
 tegra-ehci 7d004000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d004000.usb: continuing, but please update your DT
 usbcore: registered new interface driver cdc_acm
@@ -399,7 +399,7 @@
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
-bash: no job control in this shell
+bash: no j[    4.270905] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+ob control in this shell
 root@(none):/# 
\ No newline at end of file
