Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2017-11-03 02:59:23.002983703 -0700
+++ /tmp/foob	2017-11-03 02:59:23.002983703 -0700
@@ -1 +0,0 @@
-script "/home/jonathanh/workdir/tegra/epmt/test/testscripts/qemu-vexpress64_defc
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-11-03 02:59:23.030983829 -0700
+++ /tmp/foob	2017-11-03 02:59:23.034983845 -0700
@@ -42,18 +42,18 @@
 	 ########################################
 	 3.2 MiB/s
 done
-Bytes transferred = 8208896 (7d4200 hex)
+Bytes transferred = 8212992 (7d5200 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.7 MiB/s
+	 38.1 KiB/s
 done
 Bytes transferred = 34502 (86c6 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x7d4200 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x7d5200 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff6c5 ... OK
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc7-next-20171102-212067-gfa8785e (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Nov 2 06:23:00 PDT 2017
+Linux version 4.14.0-rc7-next-20171103-212598-g5a3517e (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Fri Nov 3 02:48:31 PDT 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -85,7 +85,7 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=7369c667-ff51-ec4a-29cd-baabf2fbe346 rootwait gpt debug earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1992548K/2096128K available (11264K kernel code, 1600K rwdata, 4536K rodata, 2048K init, 404K bss, 38044K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1992548K/2096128K available (11264K kernel code, 1600K rwdata, 4548K rodata, 2048K init, 404K bss, 38044K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -95,8 +95,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0e00000   (12256 kB)
       .init : 0xc1300000 - 0xc1500000   (2048 kB)
-      .data : 0xc1500000 - 0xc16901bc   (1601 kB)
-       .bss : 0xc16901bc - 0xc16f5534   ( 405 kB)
+      .data : 0xc1500000 - 0xc169021c   (1601 kB)
+       .bss : 0xc169021c - 0xc16f55b4   ( 405 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -373,17 +373,17 @@
 mmc0: new high speed SDHC card at address aaaa
 usb1_vbus: supplied by vdd-sys-5v0
 usb2_vbus: supplied by vdd-sys-5v0
-mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
- mmcblk0: p1 p2 p3
+mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
+[drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
-[drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
-[drm] No driver support for vblank timestamp query.
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB, chardev (237:0)
+ mmcblk0: p1 p2 p3
  mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
+[drm] No driver support for vblank timestamp query.
 Console: switching to colour frame buffer device 240x75
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
@@ -395,7 +395,7 @@
 tegra-ehci 7d008000.usb: Failed to add USB HCD
 tegra-ehci: probe of 7d008000.usb failed with error -12
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-03-25 20:23:48 UTC (954015828)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-03-26 16:49:11 UTC (954089351)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2017-11-03 02:59:23.038983861 -0700
+++ /tmp/foob	2017-11-03 02:59:23.038983861 -0700
@@ -38,20 +38,20 @@
 	 #################################################################
 	 #################################################################
 	 #########################################
-	 3.3 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 6323784 (607e48 hex)
+Bytes transferred = 6324200 (607fe8 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.7 MiB/s
+	 42 KiB/s
 done
 Bytes transferred = 34502 (86c6 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x607e48 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x607fe8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff6c5 ... OK
@@ -60,7 +60,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc7-next-20171102-212067-gfa8785e (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Nov 2 06:22:48 PDT 2017
+Linux version 4.14.0-rc7-next-20171103-212598-g5a3517e (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Fri Nov 3 02:47:53 PDT 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -70,7 +70,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0faf140, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0faf180, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -93,8 +93,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
-      .data : 0xc0f00000 - 0xc0fc2694   ( 778 kB)
-       .bss : 0xc0fc2694 - 0xc100d334   ( 300 kB)
+      .data : 0xc0f00000 - 0xc0fc26d4   ( 778 kB)
+       .bss : 0xc0fc26d4 - 0xc100d374   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -390,7 +390,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-03-25 20:21:14 UTC (954015674)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-03-26 16:46:21 UTC (954089181)
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
@@ -408,4 +408,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.280934] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.271499] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-11-03 02:59:23.046983899 -0700
+++ /tmp/foob	2017-11-03 02:59:23.046983899 -0700
@@ -37,18 +37,18 @@
 	 ########################################
 	 3.9 MiB/s
 done
-Bytes transferred = 8208896 (7d4200 hex)
+Bytes transferred = 8212992 (7d5200 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 2.8 MiB/s
+	 3.7 MiB/s
 done
 Bytes transferred = 66544 (103f0 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x7d4200 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x7d5200 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830133ef
@@ -56,7 +56,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc7-next-20171102-212067-gfa8785e (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Nov 2 06:23:00 PDT 2017
+Linux version 4.14.0-rc7-next-20171103-212598-g5a3517e (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Fri Nov 3 02:48:31 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -80,7 +80,7 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=b1791faa-84e8-4e27-83a1-9fb0a77e8803 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1992440K/2096128K available (11264K kernel code, 1600K rwdata, 4536K rodata, 2048K init, 404K bss, 38152K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1992440K/2096128K available (11264K kernel code, 1600K rwdata, 4548K rodata, 2048K init, 404K bss, 38152K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,8 +90,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0e00000   (12256 kB)
       .init : 0xc1300000 - 0xc1500000   (2048 kB)
-      .data : 0xc1500000 - 0xc16901bc   (1601 kB)
-       .bss : 0xc16901bc - 0xc16f5534   ( 405 kB)
+      .data : 0xc1500000 - 0xc169021c   (1601 kB)
+       .bss : 0xc169021c - 0xc16f55b4   ( 405 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -428,7 +428,6 @@
 +5V_HDMI_CON: supplied by +5V_SYS
 +5V_SATA: supplied by +5V_SYS
 +12V_SATA: supplied by +VDD_MUX
-mmc0: new high speed MMC card at address 0001
  usb2-0: usb2-0 supply vbus not found, using dummy regulator
  usb2-1: usb2-1 supply vbus not found, using dummy regulator
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
@@ -455,6 +454,11 @@
 +2.8V_RUN_CAM_AF: Bringing 0uV into 2800000-2800000uV
 as3722-ldo11: supplied by +3.3V_RUN
 +1.8V_RUN_VPP_FUSE: Bringing 0uV into 1800000-1800000uV
+mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB, chardev (237:0)
 tegra-ahci 70027000.sata: AHCI 0001.0301 32 slots 2 ports 3 Gbps 0x1 impl platform mode
 tegra-ahci 70027000.sata: flags: 64bit ncq sntf pm led pio slum part sadm sds apst 
 cma: cma_alloc: alloc failed, req-size: 23 pages, ret: -16
@@ -467,6 +471,7 @@
 tegra-ehci 7d008000.usb: USB bus 2 deregistered
 tegra-ehci 7d008000.usb: Failed to add USB HCD
 tegra-ehci: probe of 7d008000.usb failed with error -12
+ mmcblk0: p1
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc1: Unknown controller version (3). You may experience problems.
@@ -536,19 +541,14 @@
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
 r8169 0000:02:00.0 (unnamed net_device) (uninitialized): not PCI Express
-cma: cma_alloc: alloc failed, req-size: 1 pages, ret: -16
 mmc1: new high speed SDHC card at address aaaa
 mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
- mmcblk1: p1 p2 p3
+cma: cma_alloc: alloc failed, req-size: 1 pages, ret: -16
 r8169: probe of 0000:02:00.0 failed with error -12
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB, chardev (237:0)
- mmcblk0: p1
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
+ mmcblk1: p1 p2 p3
 tegra-hdmi 54280000.hdmi: cannot set audio to 48000 Hz at 297000000 Hz pixel clock
 Console: switching to colour frame buffer device 512x135
 drm drm: fb0:  frame buffer device
@@ -557,7 +557,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:01:37 UTC (946684897)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:01:40 UTC (946684900)
 +USB0_VBUS_SW: disabling
 +5V_SATA: disabling
 +12V_SATA: disabling
@@ -571,7 +571,7 @@
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:17.
+VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2017-11-03 02:59:23.054983937 -0700
+++ /tmp/foob	2017-11-03 02:59:23.058983955 -0700
@@ -33,20 +33,20 @@
 	 #################################################################
 	 #################################################################
 	 #########################################
-	 3.9 MiB/s
+	 3.6 MiB/s
 done
-Bytes transferred = 6323784 (607e48 hex)
+Bytes transferred = 6324200 (607fe8 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.7 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 66544 (103f0 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x607e48 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x607fe8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830133ef
@@ -55,7 +55,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc7-next-20171102-212067-gfa8785e (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Nov 2 06:22:48 PDT 2017
+Linux version 4.14.0-rc7-next-20171103-212598-g5a3517e (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Fri Nov 3 02:47:53 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -65,7 +65,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0faf140, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0faf180, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -88,8 +88,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
-      .data : 0xc0f00000 - 0xc0fc2694   ( 778 kB)
-       .bss : 0xc0fc2694 - 0xc100d334   ( 300 kB)
+      .data : 0xc0f00000 - 0xc0fc26d4   ( 778 kB)
+       .bss : 0xc0fc26d4 - 0xc100d374   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -486,9 +486,9 @@
 pci 0000:02:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
 PCI: bus2: Fast back to back transfers disabled
 pci_bus 0000:02: busn_res: [bus 02-ff] end is updated to 02
-pci 0000:00:01.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
 mmc1: new high speed SDHC card at address aaaa
 mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
+pci 0000:00:01.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
 pci 0000:00:02.0: BAR 8: assigned [mem 0x13100000-0x131fffff]
 pci 0000:00:02.0: BAR 9: assigned [mem 0x20000000-0x200fffff 64bit pref]
 pci 0000:00:02.0: BAR 7: assigned [io  0x1000-0x1fff]
@@ -497,8 +497,8 @@
 pci 0000:00:01.0:   bridge window [mem 0x13000000-0x130fffff]
 pci 0000:02:00.0: BAR 4: assigned [mem 0x20000000-0x20003fff 64bit pref]
 pci 0000:02:00.0: BAR 2: assigned [mem 0x13100000-0x13100fff 64bit]
-pci 0000:02:00.0: BAR 0: assigned [io  0x1000-0x10ff]
  mmcblk1: p1 p2 p3
+pci 0000:02:00.0: BAR 0: assigned [io  0x1000-0x10ff]
 pci 0000:00:02.0: PCI bridge to [bus 02]
 pci 0000:00:02.0:   bridge window [io  0x1000-0x1fff]
 pci 0000:00:02.0:   bridge window [mem 0x13100000-0x131fffff]
@@ -529,7 +529,6 @@
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
-cfg80211: failed to load regulatory.db
 +USB0_VBUS_SW: disabling
 +5V_SATA: disabling
 +12V_SATA: disabling
@@ -542,6 +541,7 @@
 +1.8V_RUN_VPP_FUSE: disabling
 ALSA device list:
   #1: NVIDIA Tegra Jetson TK1
+cfg80211: failed to load regulatory.db
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-11-03 02:59:23.062983971 -0700
+++ /tmp/foob	2017-11-03 02:59:23.066983987 -0700
@@ -43,20 +43,20 @@
 	 #################################################################
 	 #################################################################
 	 ########################################
-	 3.1 MiB/s
+	 3.6 MiB/s
 done
-Bytes transferred = 8208896 (7d4200 hex)
+Bytes transferred = 8212992 (7d5200 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.4 MiB/s
+	 1.1 MiB/s
 done
 Bytes transferred = 67482 (1079a hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x7d4200 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x7d5200 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013799
@@ -64,7 +64,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc7-next-20171102-212067-gfa8785e (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Nov 2 06:23:00 PDT 2017
+Linux version 4.14.0-rc7-next-20171103-212598-g5a3517e (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Fri Nov 3 02:48:31 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -89,7 +89,7 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=4d36e82a-b3c0-4b4b-bfcd-afe5217ac023 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993456K/2097148K available (11264K kernel code, 1600K rwdata, 4536K rodata, 2048K init, 404K bss, 38156K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1993456K/2097148K available (11264K kernel code, 1600K rwdata, 4548K rodata, 2048K init, 404K bss, 38156K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -99,8 +99,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0e00000   (12256 kB)
       .init : 0xc1300000 - 0xc1500000   (2048 kB)
-      .data : 0xc1500000 - 0xc16901bc   (1601 kB)
-       .bss : 0xc16901bc - 0xc16f5534   ( 405 kB)
+      .data : 0xc1500000 - 0xc169021c   (1601 kB)
+       .bss : 0xc169021c - 0xc16f55b4   ( 405 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -369,6 +369,8 @@
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc0: Unknown controller version (3). You may experience problems.
 mmc0: Invalid maximum block size, assuming 512 bytes
+elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
+input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 mmc0: SDHCI controller on 700b0600.sdhci [700b0600.sdhci] using ADMA
 ledtrig-cpu: registered to indicate activity on CPUs
 usbcore: registered new interface driver usbhid
@@ -386,7 +388,6 @@
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
 Loading compiled-in X.509 certificates
-elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
 +3.3V_RUN: supplied by +3.3V_SYS
 +VDD_LED: supplied by +VDD_MUX
 +3.3V_AVDD_HDMI_AP_GATED: supplied by +3.3V_RUN
@@ -399,6 +400,13 @@
 +3.3V_PANEL: supplied by +3.3V_RUN
 +3.3V_LP0: supplied by +3.3V_SYS
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
+elan_i2c 1-0015: failed to read report data: -121
+mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
+mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
+mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
+mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB, chardev (237:0)
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 +5V_HDMI_CON: supplied by +5V_SYS
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
  usb3-1: usb3-1 supply vbus not found, using dummy regulator
@@ -406,14 +414,9 @@
 panel-simple panel: panel supply power not found, using dummy regulator
 sdhci-tegra 700b0000.sdhci: allocated mmc-pwrseq
 mmc1: Unknown controller version (3). You may experience problems.
-mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
-mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
-mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
-mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB, chardev (237:0)
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: Switching to 3.3V signalling voltage failed
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
+elan_i2c 1-0015: invalid report id data (ff)
 mmc1: SDHCI controller on 700b0000.sdhci [700b0000.sdhci] using ADMA
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
@@ -421,8 +424,6 @@
 mmc2: Invalid maximum block size, assuming 512 bytes
 mmc1: new high speed SDIO card at address 0001
 mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
-input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
-elan_i2c 1-0015: invalid report id data (ff)
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
 mmc2: new high speed SDHC card at address 0007
@@ -437,7 +438,7 @@
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-as3722-rtc as3722-rtc: setting system clock to 2000-01-02 22:44:51 UTC (946853091)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-03 19:09:48 UTC (946926588)
 +1.8V_RUN_CAM: disabling
 +3.3V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2017-11-03 02:59:23.070984003 -0700
+++ /tmp/foob	2017-11-03 02:59:23.070984003 -0700
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -41,20 +41,20 @@
 	 #################################################################
 	 #################################################################
 	 #########################################
-	 3.6 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 6323784 (607e48 hex)
+Bytes transferred = 6324200 (607fe8 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.4 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 67482 (1079a hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x607e48 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x607fe8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013799
@@ -63,7 +63,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc7-next-20171102-212067-gfa8785e (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Nov 2 06:22:48 PDT 2017
+Linux version 4.14.0-rc7-next-20171103-212598-g5a3517e (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Fri Nov 3 02:47:53 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -74,7 +74,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c0faf140, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0faf180, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -97,8 +97,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
-      .data : 0xc0f00000 - 0xc0fc2694   ( 778 kB)
-       .bss : 0xc0fc2694 - 0xc100d334   ( 300 kB)
+      .data : 0xc0f00000 - 0xc0fc26d4   ( 778 kB)
+       .bss : 0xc0fc26d4 - 0xc100d374   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -314,121 +314,121 @@
 kernel BUG at drivers/platform/chrome/cros_ec_proto.c:34!
 Internal error: Oops - BUG: 0 [#1] PREEMPT SMP ARM
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.14.0-rc7-next-20171102-212067-gfa8785e #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.14.0-rc7-next-20171103-212598-g5a3517e #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 task: ee890740 task.stack: ee8a6000
 PC is at cros_ec_prepare_tx+0x168/0x16c
 LR is at cros_ec_pkt_xfer_spi+0x18/0x474
-pc : [<c06e5bf4>]    lr : [<c0554800>]    psr: 20000013
+pc : [<c06e5c14>]    lr : [<c0554884>]    psr: 20000013
 sp : ee8a7bd8  ip : 00000000  fp : 00000000
-r10: c05547e8  r9 : 00000000  r8 : eefe7fb4
-r7 : ee2abb00  r6 : ee2abb00  r5 : ee2abb00  r4 : ee28de50
-r3 : 0000ffff  r2 : 00000000  r1 : 0000ffff  r0 : ee28de50
+r10: c055486c  r9 : 00000000  r8 : eefe7fb4
+r7 : ee28cac0  r6 : ee28cac0  r5 : ee28cac0  r4 : ee26fe50
+r3 : 0000ffff  r2 : 00000000  r1 : 0000ffff  r0 : ee26fe50
 Flags: nzCv  IRQs on  FIQs on  Mode SVC_32  ISA ARM  Segment none
 Control: 10c5387d  Table: 8000406a  DAC: 00000051
 Process swapper/0 (pid: 1, stack limit = 0xee8a6210)
 Stack: (0xee8a7bd8 to 0xee8a8000)
-7bc0:                                                       ee28de50 ee2abb00
-7be0: ee2abb90 c0554800 c0989c5c 00000004 ee8a7c5c ee8a7c5c ee2a9800 00000000
-7c00: c05a0d6c 00000000 00000008 00000008 00000000 ee8a7c14 ee8a7c14 00000000
-7c20: ee8a7c20 ee8a7c20 ee2abb50 ee2aba80 00000008 00000000 00000000 00000000
-7c40: c0e90e08 c023b2d4 40002800 00000560 eef94e08 c023d738 002dc6c0 ee8a7bf0
-7c60: ee8a7bf0 00000000 c06e5a24 005c5560 3b9aca00 000f4240 00000000 ee8a7ca8
-7c80: 00000001 00000501 c05547e8 c098e598 00000000 ee2abac0 a0000013 c06e5a24
-7ca0: ee2abb00 ee28de50 ee2abb00 ee2a9800 ee2abb00 eefe7fb4 00000000 c05547e8
-7cc0: 00000000 c06e5980 0000400b ee28de50 ee2abb00 ee2a9800 0000000c eefe7fb4
-7ce0: c0fecd3c c06e5d9c 00000000 ee28de50 ee2a9800 00000000 00000000 eefe7fb4
-7d00: c0f6c378 c055428c ee8a7d14 ee2a9800 00000000 00000001 ee2a9800 ee28de50
-7d20: ee2abb90 c055463c 00000000 00000000 00000000 000007d0 ee2a9800 00000000
-7d40: c0f6c368 00000000 00000000 c05a17b0 ee2a9800 c0ffdfe4 c0ffdfe8 c05302ec
-7d60: 00000000 ee8a7d98 c053047c 00000001 c0ffdfc0 00000000 00000000 c052e904
-7d80: ee9f1170 ee22d5b8 ee2a9800 ee2a9834 c0f72ea4 c0530024 ee2a9800 00000001
-7da0: 00000000 ee2a9800 ee2a9800 c0f72ea4 ee034000 c052f668 ee2a9800 00000000
-7dc0: ee2a9808 c052dcac ee2a9800 c05a471c 00000000 ee2a9800 ee2a9800 ee034000
-7de0: 00000000 eea8be10 eefe8004 00000001 00000000 c05a1604 002dc6c0 ee034000
-7e00: eefe7fb4 00000000 ee2a9800 c05a35a4 00000000 014000c0 00000000 c05a2f90
-7e20: 00000063 002dc6c0 ee034000 ee2ab650 eea8be00 eea8be10 40000000 00000153
-7e40: 00000000 c05a3ac8 eefe7d0c ee034350 ee034000 eea8be00 eea8be10 c05a4654
-7e60: 00000000 ee034350 ffffffed eea8be10 fffffdfb c0f73800 00000000 00000000
-7e80: c0f73800 c0531be8 c0531b98 eea8be10 c0ffdfe4 c0ffdfe8 00000000 c05302ec
-7ea0: eea8be10 c0f73800 eea8be44 00000000 0000010e c0e6a83c c0e00594 c0530444
-7ec0: 00000000 c0f73800 c0530398 c052e9a0 ee8cc55c eea14934 c0f73800 ee2a5e80
-7ee0: c0f6a968 c052f83c c0c206a0 c0f73800 00000006 c0f73800 00000006 c0e3869c
-7f00: 00000000 c0530fc4 c053183c c0e83324 00000006 c0101b8c 00000000 c0cbe00c
-7f20: 00000100 efffcc45 c0cbe00c 0000010e efffcc53 c013e858 c0cbcaf4 00000000
+7bc0:                                                       ee26fe50 ee28cac0
+7be0: ee28cb50 c0554884 c0989c7c 00000004 ee8a7c5c ee8a7c5c ee28b800 00000000
+7c00: c05a0df0 00000000 00000008 00000008 00000000 ee8a7c14 ee8a7c14 00000000
+7c20: ee8a7c20 ee8a7c20 ee28cb10 ee28ca40 00000008 00000000 00000000 00000000
+7c40: c0e90e08 c023b348 40002800 00000540 eef94e08 c023d7ac 002dc6c0 ee8a7bf0
+7c60: ee8a7bf0 00000000 c06e5a44 005c5180 3b9aca00 000f4240 00000000 ee8a7ca8
+7c80: 00000001 00000501 c055486c c098e5b8 00000000 ee28ca80 a0000013 c06e5a44
+7ca0: ee28cac0 ee26fe50 ee28cac0 ee28b800 ee28cac0 eefe7fb4 00000000 c055486c
+7cc0: 00000000 c06e59a0 0000400b ee26fe50 ee28cac0 ee28b800 0000000c eefe7fb4
+7ce0: c0fecd7c c06e5dbc 00000000 ee26fe50 ee28b800 00000000 00000000 eefe7fb4
+7d00: c0f6c3b8 c0554310 ee8a7d14 ee28b800 00000000 00000001 ee28b800 ee26fe50
+7d20: ee28cb50 c05546c0 00000000 00000000 00000000 000007d0 ee28b800 00000000
+7d40: c0f6c3a8 00000000 00000000 c05a1834 ee28b800 c0ffe024 c0ffe028 c0530370
+7d60: 00000000 ee8a7d98 c0530500 00000001 c0ffe000 00000000 00000000 c052e988
+7d80: ee9f1170 ee1f75b8 ee28b800 ee28b834 c0f72ee4 c05300a8 ee28b800 00000001
+7da0: 00000000 ee28b800 ee28b800 c0f72ee4 ee036000 c052f6ec ee28b800 00000000
+7dc0: ee28b808 c052dd30 ee28b800 c05a47a0 00000000 ee28b800 ee28b800 ee036000
+7de0: 00000000 eea8be10 eefe8004 00000001 00000000 c05a1688 002dc6c0 ee036000
+7e00: eefe7fb4 00000000 ee28b800 c05a3628 00000000 014000c0 00000000 c05a3014
+7e20: 00000063 002dc6c0 ee036000 ee28c610 eea8be00 eea8be10 40000000 00000153
+7e40: 00000000 c05a3b4c eefe7d0c ee036350 ee036000 eea8be00 eea8be10 c05a46d8
+7e60: 00000000 ee036350 ffffffed eea8be10 fffffdfb c0f73840 00000000 00000000
+7e80: c0f73840 c0531c6c c0531c1c eea8be10 c0ffe024 c0ffe028 00000000 c0530370
+7ea0: eea8be10 c0f73840 eea8be44 00000000 0000010e c0e6a83c c0e00594 c05304c8
+7ec0: 00000000 c0f73840 c053041c c052ea24 ee8cc55c eea14934 c0f73840 ee287e80
+7ee0: c0f6a9a8 c052f8c0 c0c20608 c0f73840 00000006 c0f73840 00000006 c0e3869c
+7f00: 00000000 c0531048 c05318c0 c0e83324 00000006 c0101b8c 00000000 c0cbe028
+7f20: 00000100 efffcc45 c0cbe028 0000010e efffcc53 c013e858 c0cbcb10 00000000
 7f40: 00000006 00000006 efffcc45 efffcc4a 00000000 efffcbc0 00000006 c0e6a82c
-7f60: c0e83324 00000006 c0e6a830 c0fc26c0 0000010e c0e00d94 00000006 00000006
-7f80: 00000000 c0e00594 00000000 c09878b4 00000000 00000000 00000000 00000000
-7fa0: 00000000 c09878bc 00000000 c0107cf0 00000000 00000000 00000000 00000000
+7f60: c0e83324 00000006 c0e6a830 c0fc2700 0000010e c0e00d94 00000006 00000006
+7f80: 00000000 c0e00594 00000000 c09878d4 00000000 00000000 00000000 00000000
+7fa0: 00000000 c09878dc 00000000 c0107cf0 00000000 00000000 00000000 00000000
 7fc0: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
 7fe0: 00000000 00000000 00000000 00000000 00000013 00000000 00000000 00000000
-[<c06e5bf4>] (cros_ec_prepare_tx) from [<c0554800>] (cros_ec_pkt_xfer_spi+0x18/0x474)
-[<c0554800>] (cros_ec_pkt_xfer_spi) from [<c06e5980>] (send_command+0x20/0xd8)
-[<c06e5980>] (send_command) from [<c06e5d9c>] (cros_ec_query_all+0x1a4/0x398)
-[<c06e5d9c>] (cros_ec_query_all) from [<c055428c>] (cros_ec_register+0x94/0x21c)
-[<c055428c>] (cros_ec_register) from [<c055463c>] (cros_ec_spi_probe+0x108/0x150)
-[<c055463c>] (cros_ec_spi_probe) from [<c05a17b0>] (spi_drv_probe+0x7c/0xa8)
-[<c05a17b0>] (spi_drv_probe) from [<c05302ec>] (driver_probe_device+0x240/0x2ec)
-[<c05302ec>] (driver_probe_device) from [<c052e904>] (bus_for_each_drv+0x44/0x8c)
-[<c052e904>] (bus_for_each_drv) from [<c0530024>] (__device_attach+0x9c/0x100)
-[<c0530024>] (__device_attach) from [<c052f668>] (bus_probe_device+0x84/0x8c)
-[<c052f668>] (bus_probe_device) from [<c052dcac>] (device_add+0x380/0x528)
-[<c052dcac>] (device_add) from [<c05a1604>] (spi_add_device+0x88/0x130)
-[<c05a1604>] (spi_add_device) from [<c05a35a4>] (spi_register_controller+0x428/0x918)
-[<c05a35a4>] (spi_register_controller) from [<c05a3ac8>] (devm_spi_register_controller+0x34/0x70)
-[<c05a3ac8>] (devm_spi_register_controller) from [<c05a4654>] (tegra_spi_probe+0x298/0x35c)
-[<c05a4654>] (tegra_spi_probe) from [<c0531be8>] (platform_drv_probe+0x50/0xb0)
-[<c0531be8>] (platform_drv_probe) from [<c05302ec>] (driver_probe_device+0x240/0x2ec)
-[<c05302ec>] (driver_probe_device) from [<c0530444>] (__driver_attach+0xac/0xb0)
-[<c0530444>] (__driver_attach) from [<c052e9a0>] (bus_for_each_dev+0x54/0x88)
-[<c052e9a0>] (bus_for_each_dev) from [<c052f83c>] (bus_add_driver+0xe4/0x1f0)
-[<c052f83c>] (bus_add_driver) from [<c0530fc4>] (driver_register+0x78/0xf4)
-[<c0530fc4>] (driver_register) from [<c0101b8c>] (do_one_initcall+0x3c/0x168)
+[<c06e5c14>] (cros_ec_prepare_tx) from [<c0554884>] (cros_ec_pkt_xfer_spi+0x18/0x474)
+[<c0554884>] (cros_ec_pkt_xfer_spi) from [<c06e59a0>] (send_command+0x20/0xd8)
+[<c06e59a0>] (send_command) from [<c06e5dbc>] (cros_ec_query_all+0x1a4/0x398)
+[<c06e5dbc>] (cros_ec_query_all) from [<c0554310>] (cros_ec_register+0x94/0x21c)
+[<c0554310>] (cros_ec_register) from [<c05546c0>] (cros_ec_spi_probe+0x108/0x150)
+[<c05546c0>] (cros_ec_spi_probe) from [<c05a1834>] (spi_drv_probe+0x7c/0xa8)
+[<c05a1834>] (spi_drv_probe) from [<c0530370>] (driver_probe_device+0x240/0x2ec)
+[<c0530370>] (driver_probe_device) from [<c052e988>] (bus_for_each_drv+0x44/0x8c)
+[<c052e988>] (bus_for_each_drv) from [<c05300a8>] (__device_attach+0x9c/0x100)
+[<c05300a8>] (__device_attach) from [<c052f6ec>] (bus_probe_device+0x84/0x8c)
+[<c052f6ec>] (bus_probe_device) from [<c052dd30>] (device_add+0x380/0x528)
+[<c052dd30>] (device_add) from [<c05a1688>] (spi_add_device+0x88/0x130)
+[<c05a1688>] (spi_add_device) from [<c05a3628>] (spi_register_controller+0x428/0x918)
+[<c05a3628>] (spi_register_controller) from [<c05a3b4c>] (devm_spi_register_controller+0x34/0x70)
+[<c05a3b4c>] (devm_spi_register_controller) from [<c05a46d8>] (tegra_spi_probe+0x298/0x35c)
+[<c05a46d8>] (tegra_spi_probe) from [<c0531c6c>] (platform_drv_probe+0x50/0xb0)
+[<c0531c6c>] (platform_drv_probe) from [<c0530370>] (driver_probe_device+0x240/0x2ec)
+[<c0530370>] (driver_probe_device) from [<c05304c8>] (__driver_attach+0xac/0xb0)
+[<c05304c8>] (__driver_attach) from [<c052ea24>] (bus_for_each_dev+0x54/0x88)
+[<c052ea24>] (bus_for_each_dev) from [<c052f8c0>] (bus_add_driver+0xe4/0x1f0)
+[<c052f8c0>] (bus_add_driver) from [<c0531048>] (driver_register+0x78/0xf4)
+[<c0531048>] (driver_register) from [<c0101b8c>] (do_one_initcall+0x3c/0x168)
 [<c0101b8c>] (do_one_initcall) from [<c0e00d94>] (kernel_init_freeable+0x10c/0x1d0)
-[<c0e00d94>] (kernel_init_freeable) from [<c09878bc>] (kernel_init+0x8/0x114)
-[<c09878bc>] (kernel_init) from [<c0107cf0>] (ret_from_fork+0x14/0x24)
+[<c0e00d94>] (kernel_init_freeable) from [<c09878dc>] (kernel_init+0x8/0x114)
+[<c09878dc>] (kernel_init) from [<c0107cf0>] (ret_from_fork+0x14/0x24)
 Code: e2800008 e8bd8070 e7f001f2 e7f001f2 (e7f001f2) 
----[ end trace 2495e607878277e1 ]---
+---[ end trace f1544cd2a6c32eec ]---
 Kernel panic - not syncing: Attempted to kill init! exitcode=0x0000000b
 
-CPU3: stopping
-CPU: 3 PID: 0 Comm: swapper/3 Tainted: G      D          4.14.0-rc7-next-20171102-212067-gfa8785e #1
+CPU1: stopping
+CPU: 1 PID: 0 Comm: swapper/1 Tainted: G      D          4.14.0-rc7-next-20171103-212598-g5a3517e #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c0111088>] (unwind_backtrace) from [<c010c074>] (show_stack+0x10/0x14)
-[<c010c074>] (show_stack) from [<c0973f9c>] (dump_stack+0x84/0xa4)
-[<c0973f9c>] (dump_stack) from [<c010f5e0>] (handle_IPI+0x18c/0x32c)
+[<c010c074>] (show_stack) from [<c0973fbc>] (dump_stack+0x84/0xa4)
+[<c0973fbc>] (dump_stack) from [<c010f5e0>] (handle_IPI+0x18c/0x32c)
 [<c010f5e0>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
-Exception stack(0xee8e3f70 to 0xee8e3fb8)
-3f60:                                     00000001 00000000 ee8e3fc8 c011a360
-3f80: 00000008 c0f04ccc 00000000 c0f04c68 00000000 c0e8c3f8 00000000 c0f04cd8
-3fa0: c06ceb3c ee8e3fc0 c0108758 c010875c 60000013 ffffffff
+Exception stack(0xee8c1f70 to 0xee8c1fb8)
+1f60:                                     00000001 00000000 ee8c1fc8 c011a360
+1f80: 00000002 c0f04ccc 00000000 c0f04c68 00000000 c0e8c3f8 00000000 c0f04cd8
+1fa0: c06ceb5c ee8c1fc0 c0108758 c010875c 60000013 ffffffff
 [<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
 [<c010875c>] (arch_cpu_idle) from [<c016254c>] (do_idle+0x140/0x204)
 [<c016254c>] (do_idle) from [<c0162898>] (cpu_startup_entry+0x18/0x1c)
 [<c0162898>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
-CPU1: stopping
-CPU: 1 PID: 0 Comm: swapper/1 Tainted: G      D          4.14.0-rc7-next-20171102-212067-gfa8785e #1
+CPU3: stopping
+CPU: 3 PID: 0 Comm: swapper/3 Tainted: G      D          4.14.0-rc7-next-20171103-212598-g5a3517e #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c0111088>] (unwind_backtrace) from [<c010c074>] (show_stack+0x10/0x14)
-[<c010c074>] (show_stack) from [<c0973f9c>] (dump_stack+0x84/0xa4)
-[<c0973f9c>] (dump_stack) from [<c010f5e0>] (handle_IPI+0x18c/0x32c)
+[<c010c074>] (show_stack) from [<c0973fbc>] (dump_stack+0x84/0xa4)
+[<c0973fbc>] (dump_stack) from [<c010f5e0>] (handle_IPI+0x18c/0x32c)
 [<c010f5e0>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
-Exception stack(0xee8c1f70 to 0xee8c1fb8)
-1f60:                                     00000001 00000000 ee8c1fc8 c011a360
-1f80: 00000002 c0f04ccc 00000000 c0f04c68 00000000 c0e8c3f8 00000000 c0f04cd8
-1fa0: c06ceb3c ee8c1fc0 c0108758 c010875c 60000013 ffffffff
+Exception stack(0xee8e3f70 to 0xee8e3fb8)
+3f60:                                     00000001 00000000 ee8e3fc8 c011a360
+3f80: 00000008 c0f04ccc 00000000 c0f04c68 00000000 c0e8c3f8 00000000 c0f04cd8
+3fa0: 00000002 ee8e3fc0 c0108758 c010875c 60000013 ffffffff
 [<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
 [<c010875c>] (arch_cpu_idle) from [<c016254c>] (do_idle+0x140/0x204)
 [<c016254c>] (do_idle) from [<c0162898>] (cpu_startup_entry+0x18/0x1c)
 [<c0162898>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
 CPU2: stopping
-CPU: 2 PID: 0 Comm: swapper/2 Tainted: G      D          4.14.0-rc7-next-20171102-212067-gfa8785e #1
+CPU: 2 PID: 0 Comm: swapper/2 Tainted: G      D          4.14.0-rc7-next-20171103-212598-g5a3517e #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c0111088>] (unwind_backtrace) from [<c010c074>] (show_stack+0x10/0x14)
-[<c010c074>] (show_stack) from [<c0973f9c>] (dump_stack+0x84/0xa4)
-[<c0973f9c>] (dump_stack) from [<c010f5e0>] (handle_IPI+0x18c/0x32c)
+[<c010c074>] (show_stack) from [<c0973fbc>] (dump_stack+0x84/0xa4)
+[<c0973fbc>] (dump_stack) from [<c010f5e0>] (handle_IPI+0x18c/0x32c)
 [<c010f5e0>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
 Exception stack(0xee8e1f70 to 0xee8e1fb8)
@@ -441,3 +441,19 @@
 [<c0162898>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
 ---[ end Kernel panic - not syncing: Attempted to kill init! exitcode=0x0000000b
 
+------------[ cut here ]------------
+WARNING: CPU: 0 PID: 8 at kernel/irq_work.c:68 irq_work_queue_on+0xbc/0xfc
+Modules linked in:
+CPU: 0 PID: 8 Comm: rcu_preempt Tainted: G      D          4.14.0-rc7-next-20171103-212598-g5a3517e #1
+Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
+[<c0111088>] (unwind_backtrace) from [<c010c074>] (show_stack+0x10/0x14)
+[<c010c074>] (show_stack) from [<c0973fbc>] (dump_stack+0x84/0xa4)
+[<c0973fbc>] (dump_stack) from [<c01219e4>] (__warn+0xd8/0x104)
+[<c01219e4>] (__warn) from [<c0121a30>] (warn_slowpath_null+0x20/0x28)
+[<c0121a30>] (warn_slowpath_null) from [<c01d6810>] (irq_work_queue_on+0xbc/0xfc)
+[<c01d6810>] (irq_work_queue_on) from [<c0183ca4>] (rcu_implicit_dynticks_qs+0x280/0x2a0)
+[<c0183ca4>] (rcu_implicit_dynticks_qs) from [<c01824a4>] (force_qs_rnp+0x118/0x1b8)
+[<c01824a4>] (force_qs_rnp) from [<c0182a9c>] (rcu_gp_kthread+0x558/0x9c0)
+[<c0182a9c>] (rcu_gp_kthread) from [<c013f104>] (kthread+0x120/0x158)
+[<c013f104>] (kthread) from [<c0107cf0>] (ret_from_fork+0x14/0x24)
+---[ end trace f1544cd2a6c32eed ]---
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2017-11-03 02:59:23.078984041 -0700
+++ /tmp/foob	2017-11-03 02:59:23.078984041 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1801815 us
+Bootloader-Cpu Init at (time stamp): 1801563 us
 
 Pinmux changes applied in kernel way
 
@@ -62,17 +62,17 @@
 Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1801789  us
-Bootloader (AosInit) time: 734236 us
-Bootloader Main Init time: 180080 us
-BlockDevMgrInit time: 157556 us
-PartMgrInit time: 8 us
+Miscellaneous avp time: 1801537  us
+Bootloader (AosInit) time: 733499 us
+Bootloader Main Init time: 180193 us
+BlockDevMgrInit time: 157525 us
+PartMgrInit time: 9 us
 StorMgrInit time: 30 us
 FsMgrInit time: 2 us
-Partition Table load: 76632 us 
-Display init time: 574959 us
-Kernel read time: 1364 msec
-Load Kernel Size:   18079744 Bytes
-Bl Kernel ReadSpeed: 13254000 Bytes/sec
-Pre-Kernel Jump time: 3883121 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 8867086 us
+Partition Table load: 77029 us 
+Display init time: 574572 us
+Kernel read time: 1366 msec
+Load Kernel Size:   18094080 Bytes
+Bl Kernel ReadSpeed: 13246000 Bytes/sec
+Pre-Kernel Jump time: 3886904 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 8870890 us
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-11-03 02:59:23.082984060 -0700
+++ /tmp/foob	2017-11-03 02:59:23.086984079 -0700
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -50,18 +50,18 @@
 	 ########################################
 	 4.1 MiB/s
 done
-Bytes transferred = 8208896 (7d4200 hex)
+Bytes transferred = 8212992 (7d5200 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 4 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21160 (52a8 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x7d4200 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x7d5200 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102a7
@@ -69,7 +69,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc7-next-20171102-212067-gfa8785e (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Nov 2 06:23:00 PDT 2017
+Linux version 4.14.0-rc7-next-20171103-212598-g5a3517e (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Fri Nov 3 02:48:31 PDT 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -92,7 +92,7 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=11eb23c1-01 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 953376K/1048576K available (11264K kernel code, 1600K rwdata, 4536K rodata, 2048K init, 404K bss, 29664K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 953376K/1048576K available (11264K kernel code, 1600K rwdata, 4548K rodata, 2048K init, 404K bss, 29664K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -102,8 +102,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0e00000   (12256 kB)
       .init : 0xc1300000 - 0xc1500000   (2048 kB)
-      .data : 0xc1500000 - 0xc16901bc   (1601 kB)
-       .bss : 0xc16901bc - 0xc16f5534   ( 405 kB)
+      .data : 0xc1500000 - 0xc169021c   (1601 kB)
+       .bss : 0xc169021c - 0xc16f55b4   ( 405 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -332,11 +332,7 @@
 Segment Routing with IPv6
 sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
 NET: Registered protocol family 17
-mmc0: host does not support reading read-only switch, assuming write-enable
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
-mmc0: new high speed SDHC card at address b368
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
- mmcblk0: p1
 can: controller area network core (rev 20170425 abi 9)
 NET: Registered protocol family 29
 can: raw protocol (rev 20170425)
@@ -346,6 +342,7 @@
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
 Loading compiled-in X.509 certificates
+mmc0: host does not support reading read-only switch, assuming write-enable
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
@@ -356,9 +353,13 @@
 tegra-ehci: probe of c5000000.usb failed with error -12
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
 rtc-em3027 2-0056: hctosys: unable to read the hardware clock
-EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
+Waiting for root device PARTUUID=11eb23c1-01...
+mmc0: new high speed SDHC card at address b368
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
+ mmcblk0: p1
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
+EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
  mmcblk1: p1 p2
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2017-11-03 02:59:23.090984097 -0700
+++ /tmp/foob	2017-11-03 02:59:23.094984113 -0700
@@ -46,20 +46,20 @@
 	 #################################################################
 	 #################################################################
 	 #########################################
-	 4 MiB/s
+	 3.9 MiB/s
 done
-Bytes transferred = 6323784 (607e48 hex)
+Bytes transferred = 6324200 (607fe8 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 4 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21160 (52a8 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x607e48 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x607fe8 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102a7
@@ -68,7 +68,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc7-next-20171102-212067-gfa8785e (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Nov 2 06:22:48 PDT 2017
+Linux version 4.14.0-rc7-next-20171103-212598-g5a3517e (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Fri Nov 3 02:47:53 PDT 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -77,7 +77,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0x3c000000
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0faf140, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0faf180, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -100,8 +100,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
-      .data : 0xc0f00000 - 0xc0fc2694   ( 778 kB)
-       .bss : 0xc0fc2694 - 0xc100d334   ( 300 kB)
+      .data : 0xc0f00000 - 0xc0fc26d4   ( 778 kB)
+       .bss : 0xc0fc26d4 - 0xc100d374   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
 	Tasks RCU enabled.
@@ -331,9 +331,9 @@
 Registering SWP/SWPB emulation handler
 Loading compiled-in X.509 certificates
 mmc1: new high speed SD card at address b368
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
  mmcblk1: p1 p2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
@@ -341,16 +341,16 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: hctosys: unable to read the hardware clock
-cfg80211: Loading compiled-in X.509 certificates for regulatory database
 hub 1-1:1.0: USB hub found
+rtc-em3027 2-0056: hctosys: unable to read the hardware clock
 hub 1-1:1.0: 4 ports detected
+cfg80211: Loading compiled-in X.509 certificates for regulatory database
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
+cfg80211: failed to load regulatory.db
 ALSA device list:
   #0: tegra-trimslice
 Waiting for root device PARTUUID=11eb23c1-01...
-cfg80211: failed to load regulatory.db
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2017-11-03 02:59:23.098984129 -0700
+++ /tmp/foob	2017-11-03 02:59:23.102984146 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 756331 us
-Basic modules init took 348648 us
+Tegraboot started after 735101 us
+Basic modules init took 348813 us
 Sec Bootdevice Read Time = 203 ms, Read Size = 8459 KB
 Next stage binary read took 14366 us
-Carveout took 253879 us
-CPU initialization took 92469 us
-Total time taken by TegraBoot 709362 us
+Carveout took 252913 us
+CPU initialization took 93098 us
+Total time taken by TegraBoot 709190 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.596] LPDDR4 Training: Number of tables = 10
+64b[0001.575] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-16759296 bytes read in 841 ms (19 MiB/s)
+16773632 bytes read in 841 ms (19 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
 44764 bytes read in 453 ms (95.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 494 ms (3.1 MiB/s)
+1627787 bytes read in 496 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0000000000 [0x411fd071]
-Linux version 4.14.0-rc7-next-20171102-330308-gfa8785e862ef (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Thu Nov 2 06:26:11 PDT 2017
+Linux version 4.14.0-rc7-next-20171103-330839-g5a3517e009e9 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Fri Nov 3 02:52:04 PDT 2017
 Machine model: NVIDIA Tegra210 P2371 (P2530/P2595) reference design
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -154,23 +154,23 @@
 [0000000000000000] user address but active_mm is swapper
 Internal error: Oops: 96000004 [#1] PREEMPT SMP
 Modules linked in:
-CPU: 0 PID: 0 Comm: swapper Not tainted 4.14.0-rc7-next-20171102-330308-gfa8785e862ef #1
+CPU: 0 PID: 0 Comm: swapper Not tainted 4.14.0-rc7-next-20171103-330839-g5a3517e009e9 #1
 Hardware name: NVIDIA Tegra210 P2371 (P2530/P2595) reference design (DT)
 task: ffff000008f72a80 task.stack: ffff000008f60000
-pstate: 800000c5 (Nzcv daIF -PAN -UAO)
+pstate: 80000085 (Nzcv daIf -PAN -UAO)
 pc : memory_present+0x68/0xf4
 lr : memory_present+0x34/0xf4
 sp : ffff000008f63e80
 x29: ffff000008f63e80 x28: 0000000080e50018 
 x27: 0000000000000000 x26: 0000000000000004 
 x25: 0000000000000002 x24: 0000000000000000 
-x23: 0000000000000000 x22: ffff0000090b3000 
-x21: 0000000000000000 x20: ffff0000090b3000 
-x19: 0000000000080000 x18: ffff0000090afb40 
+x23: 0000000000000000 x22: ffff0000090b7000 
+x21: 0000000000000000 x20: ffff0000090b7000 
+x19: 0000000000080000 x18: ffff0000090b3b40 
 x17: 0000000000000007 x16: 0000000000000001 
 x15: 0000000100000000 x14: 000000000003b6e4 
 x13: 000000013ffb8918 x12: 0000000000000001 
-x11: 000000000000000b x10: ffff000008f8c958 
+x11: 000000000000000b x10: ffff000008f8c980 
 x9 : 0000000000000000 x8 : ffff8000bffb8900 
 x7 : 0000000000000000 x6 : 0000000000000000 
 x5 : fffffffffffffffe x4 : ffffffffffffffff 
@@ -180,7 +180,7 @@
 Call trace:
  memory_present+0x68/0xf4
  bootmem_init+0x90/0x11c
- setup_arch+0x198/0x4e8
+ setup_arch+0x19c/0x4ec
  start_kernel+0x64/0x3ac
 Code: d35afe78 f9417e80 d37df318 d352fe79 (f8786800) 
 random: get_random_bytes called from print_oops_end_marker+0x48/0x64 with crng_init=0
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-11-03 02:59:23.110984182 -0700
+++ /tmp/foob	2017-11-03 02:59:23.114984201 -0700
@@ -40,20 +40,20 @@
 	 #################################################################
 	 #################################################################
 	 ########################################
-	 3.1 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 8208896 (7d4200 hex)
+Bytes transferred = 8212992 (7d5200 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.6 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 51470 (c90e hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x7d4200 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x7d5200 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90d ... OK
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc7-next-20171102-212067-gfa8785e (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Nov 2 06:23:00 PDT 2017
+Linux version 4.14.0-rc7-next-20171103-212598-g5a3517e (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Fri Nov 3 02:48:31 PDT 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -84,7 +84,7 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=edbe1840-3633-c751-7993-b0ac834f81e0 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1992476K/2096128K available (11264K kernel code, 1600K rwdata, 4536K rodata, 2048K init, 404K bss, 38116K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1992476K/2096128K available (11264K kernel code, 1600K rwdata, 4548K rodata, 2048K init, 404K bss, 38116K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -94,8 +94,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0e00000   (12256 kB)
       .init : 0xc1300000 - 0xc1500000   (2048 kB)
-      .data : 0xc1500000 - 0xc16901bc   (1601 kB)
-       .bss : 0xc16901bc - 0xc16f5534   ( 405 kB)
+      .data : 0xc1500000 - 0xc169021c   (1601 kB)
+       .bss : 0xc169021c - 0xc16f55b4   ( 405 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2017-11-03 02:59:23.122984237 -0700
+++ /tmp/foob	2017-11-03 02:59:23.122984237 -0700
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra30 (Beaver) # 
 Tegra30 (Beaver) # usb reset
 (Re)start USB...
@@ -38,20 +38,20 @@
 	 #################################################################
 	 #################################################################
 	 #########################################
-	 3.3 MiB/s
+	 2.9 MiB/s
 done
-Bytes transferred = 6323784 (607e48 hex)
+Bytes transferred = 6324200 (607fe8 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.9 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 51470 (c90e hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x607e48 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x607fe8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90d ... OK
@@ -60,7 +60,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc7-next-20171102-212067-gfa8785e (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Nov 2 06:22:48 PDT 2017
+Linux version 4.14.0-rc7-next-20171103-212598-g5a3517e (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Fri Nov 3 02:47:53 PDT 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -69,7 +69,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0faf140, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0faf180, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -92,8 +92,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
-      .data : 0xc0f00000 - 0xc0fc2694   ( 778 kB)
-       .bss : 0xc0fc2694 - 0xc100d334   ( 300 kB)
+      .data : 0xc0f00000 - 0xc0fc26d4   ( 778 kB)
+       .bss : 0xc0fc26d4 - 0xc100d374   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -453,11 +453,11 @@
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
+cfg80211: failed to load regulatory.db
 vdd_sata,avdd_plle: disabling
 usb1_vbus: disabling
 ALSA device list:
   #0: NVIDIA Tegra Beaver
-cfg80211: failed to load regulatory.db
 EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk1p1): warning: mounting fs with errors, running e2fsck is recommended
 EXT4-fs (mmcblk1p1): recovery complete
