Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2017-02-12 23:13:21.468228644 -0800
+++ /tmp/foob	2017-02-12 23:13:21.468228644 -0800
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc7-next-20170210-176877-g632571b (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Fri Feb 10 00:07:47 PST 2017
+Linux version 4.10.0-rc7-next-20170213-177255-g8813198 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Feb 12 23:07:37 PST 2017
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -21,7 +21,7 @@
 psci: PSCIv0.2 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: Trusted OS migration not required
-percpu: Embedded 23 pages/cpu @ffff80003efd0000 s56832 r8192 d29184 u94208
+percpu: Embedded 23 pages/cpu @ffff80003efd0000 s56960 r8192 d29056 u94208
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
 CPU features: enabling workaround for ARM erratum 834220
@@ -29,15 +29,15 @@
 Policy zone: DMA
 Kernel command line: root=/dev/vda2 rw rootwait mem=1024M console=ttyAMA0,38400n8 init=/bin/bash
 PID hash table entries: 4096 (order: 3, 32768 bytes)
-Memory: 1000120K/1048576K available (8700K kernel code, 970K rwdata, 4196K rodata, 1088K init, 396K bss, 32072K reserved, 16384K cma-reserved)
+Memory: 1000124K/1048576K available (8700K kernel code, 964K rwdata, 4196K rodata, 1088K init, 398K bss, 32068K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008900000   (  8704 KB)
     .rodata : 0xffff000008900000 - 0xffff000008d20000   (  4224 KB)
       .init : 0xffff000008d20000 - 0xffff000008e30000   (  1088 KB)
-      .data : 0xffff000008e30000 - 0xffff000008f22a00   (   971 KB)
-       .bss : 0xffff000008f22a00 - 0xffff000008f85bb8   (   397 KB)
+      .data : 0xffff000008e30000 - 0xffff000008f21200   (   965 KB)
+       .bss : 0xffff000008f21200 - 0xffff000008f84a38   (   399 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -45,8 +45,9 @@
     memory  : 0xffff800000000000 - 0xffff800040000000   (  1024 MB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
 Preemptible hierarchical RCU implementation.
+	Build-time adjustment of leaf fanout to 64.
 	RCU restricting CPUs from NR_CPUS=64 to nr_cpu_ids=1.
-RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=1
+RCU: Adjusting geometry for rcu_fanout_leaf=64, nr_cpu_ids=1
 NR_IRQS:64 nr_irqs:64 0
 GICv2m: range[mem 0x08020000-0x08020fff], SPI[80:143]
 arm_arch_timer: WARNING: Invalid trigger for IRQ3, assuming level low
@@ -113,7 +114,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.724:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.700:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -179,7 +180,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2017-02-10 08:08:33 UTC (1486714113)
+rtc-pl031 9010000.pl031: setting system clock to 2017-02-13 07:08:18 UTC (1486969698)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-02-12 23:13:21.488228722 -0800
+++ /tmp/foob	2017-02-12 23:13:21.488228722 -0800
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 #########################################################
-	 3.1 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 7510704 (729ab0 hex)
+Bytes transferred = 7513912 (72a738 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3.3 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x729ab0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x72a738 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc7-next-20170210-152444-g632571b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Fri Feb 10 00:04:49 PST 2017
+Linux version 4.10.0-rc7-next-20170213-152822-g8813198 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Feb 12 23:04:13 PST 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -71,7 +71,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c153a800, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c153a700, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -94,13 +94,14 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc153c1a0   (1265 kB)
-       .bss : 0xc153e000 - 0xc15c4178   ( 537 kB)
+      .data : 0xc1400000 - 0xc153c020   (1265 kB)
+       .bss : 0xc153e000 - 0xc15c41f8   ( 537 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU debugfs-based tracing is enabled.
+	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=16 to nr_cpu_ids=4.
-RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=4
+RCU: Adjusting geometry for rcu_fanout_leaf=32, nr_cpu_ids=4
 NR_IRQS:16 nr_irqs:16 16
 /interrupt-controller@60004000: 160 interrupts forwarded to /interrupt-controller@50041000
 Tegra clk 127: register failed with -17
@@ -310,11 +311,11 @@
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
 mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
- mmcblk0: p1 p2 p3
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
+ mmcblk0: p1 p2 p3
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
  mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
 Console: switching to colour frame buffer device 240x75
@@ -327,7 +328,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-12-04 19:38:57 UTC (975958737)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-12-07 18:38:39 UTC (976214319)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -339,4 +340,4 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.509558] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.519422] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2017-02-12 23:13:21.504228782 -0800
+++ /tmp/foob	2017-02-12 23:13:21.508228798 -0800
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 ######################################################
-	 3.2 MiB/s
+	 3.1 MiB/s
 done
-Bytes transferred = 5554352 (54c0b0 hex)
+Bytes transferred = 5553960 (54bf28 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3.3 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x54c0b0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x54bf28 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc7-next-20170210-152444-g632571b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Feb 10 00:04:29 PST 2017
+Linux version 4.10.0-rc7-next-20170213-152822-g8813198 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Feb 12 23:04:05 PST 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,20 +69,20 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0d5ed00, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0d5ec00, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 16 pages/cpu @eef9b000 s34624 r8192 d22720 u65536
-pcpu-alloc: s34624 r8192 d22720 u65536 alloc=16*4096
+percpu: Embedded 16 pages/cpu @eef9b000 s34688 r8192 d22656 u65536
+pcpu-alloc: s34688 r8192 d22656 u65536 alloc=16*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=7369c667-ff51-ec4a-29cd-baabf2fbe346 rootwait gpt debug earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999924K/2096128K available (8192K kernel code, 384K rwdata, 2488K rodata, 1024K init, 369K bss, 30668K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1999924K/2096128K available (8192K kernel code, 384K rwdata, 2492K rodata, 1024K init, 369K bss, 30668K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,9 +92,10 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d60260   ( 385 kB)
-       .bss : 0xc0d62000 - 0xc0dbe6ec   ( 370 kB)
+      .data : 0xc0d00000 - 0xc0d600e0   ( 385 kB)
+       .bss : 0xc0d62000 - 0xc0dbe66c   ( 370 kB)
 Preemptible hierarchical RCU implementation.
+	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
 /interrupt-controller@60004000: 160 interrupts forwarded to /interrupt-controller@50041000
 Tegra clk 127: register failed with -17
@@ -320,7 +321,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-12-04 19:36:15 UTC (975958575)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-12-07 18:35:58 UTC (976214158)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -334,4 +335,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.749221] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.740048] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-02-12 23:13:21.524228859 -0800
+++ /tmp/foob	2017-02-12 23:13:21.524228859 -0800
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -34,9 +34,9 @@
 	 #################################################################
 	 #################################################################
 	 #########################################################
-	 3.9 MiB/s
+	 3.8 MiB/s
 done
-Bytes transferred = 7510704 (729ab0 hex)
+Bytes transferred = 7513912 (72a738 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -47,7 +47,7 @@
 done
 Bytes transferred = 67133 (1063d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x729ab0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x72a738 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301363c
@@ -55,7 +55,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc7-next-20170210-152444-g632571b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Fri Feb 10 00:04:49 PST 2017
+Linux version 4.10.0-rc7-next-20170213-152822-g8813198 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Feb 12 23:04:13 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -66,7 +66,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c153a800, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c153a700, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -89,13 +89,14 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc153c1a0   (1265 kB)
-       .bss : 0xc153e000 - 0xc15c4178   ( 537 kB)
+      .data : 0xc1400000 - 0xc153c020   (1265 kB)
+       .bss : 0xc153e000 - 0xc15c41f8   ( 537 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU debugfs-based tracing is enabled.
+	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=16 to nr_cpu_ids=4.
-RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=4
+RCU: Adjusting geometry for rcu_fanout_leaf=32, nr_cpu_ids=4
 NR_IRQS:16 nr_irqs:16 16
 /interrupt-controller@60004000: 160 interrupts forwarded to /interrupt-controller@50041000
 Tegra clk 127: register failed with -17
@@ -312,6 +313,12 @@
 +5V_SATA: supplied by +5V_SYS
 +12V_SATA: supplied by +VDD_MUX
  usb2-0: usb2-0 supply vbus not found, using dummy regulator
+mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+ mmcblk0: p1
  usb2-1: usb2-1 supply vbus not found, using dummy regulator
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
@@ -336,22 +343,16 @@
 as3722-ldo10: supplied by +5V_SYS
 +2.8V_RUN_CAM_AF: Bringing 0uV into 2800000-2800000uV
 as3722-ldo11: supplied by +3.3V_RUN
-mmc0: new high speed MMC card at address 0001
 +1.8V_RUN_VPP_FUSE: Bringing 0uV into 1800000-1800000uV
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 tegra-ahci 70027000.sata: AHCI 0001.0301 32 slots 2 ports 3 Gbps 0x1 impl platform mode
 tegra-ahci 70027000.sata: flags: 64bit ncq sntf pm led pio slum part sadm sds apst 
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 scsi host0: tegra-ahci
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
 scsi host1: tegra-ahci
 ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 345
 ata2: DUMMY
 tegra-ehci 7d008000.usb: EHCI Host Controller
 tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci 7d008000.usb: irq 355, io mem 0x7d008000
- mmcblk0: p1
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
@@ -380,8 +381,8 @@
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4c may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x3e may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x3e may corrupt adjacent RW1C bits
-pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x4 may corrupt adjacent RW1C bits
 ata1: SATA link down (SStatus 0 SControl 300)
+pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x4 may corrupt adjacent RW1C bits
 pci_bus 0000:00: 1-byte config write to 0000:00:01.0 offset 0xc may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4 may corrupt adjacent RW1C bits
 pci_bus 0000:00: 1-byte config write to 0000:00:02.0 offset 0xc may corrupt adjacent RW1C bits
@@ -392,6 +393,7 @@
 pci 0000:01:00.0: reg 0x10: [mem 0x00000000-0x00001fff 64bit]
 pci 0000:01:00.0: PME# supported from D0 D3hot D3cold
 mmc1: new high speed SDHC card at address aaaa
+mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 PCI: bus1: Fast back to back transfers disabled
 pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
 pci 0000:02:00.0: [10ec:8168] type 00 class 0x020000
@@ -400,7 +402,6 @@
 pci 0000:02:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
 pci 0000:02:00.0: supports D1 D2
 pci 0000:02:00.0: PME# supported from D0 D1 D2 D3hot D3cold
-mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
  mmcblk1: p1 p2 p3
 PCI: bus2: Fast back to back transfers disabled
 pci_bus 0000:02: busn_res: [bus 02-ff] end is updated to 02
@@ -426,7 +427,7 @@
 pcieport 0000:00:02.0: Signaling PME with IRQ 394
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
-r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0b53000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 395
+r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0b54000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 395
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
@@ -439,7 +440,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:05 UTC (946684925)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:01:56 UTC (946684916)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2017-02-12 23:13:21.532228891 -0800
+++ /tmp/foob	2017-02-12 23:13:21.536228906 -0800
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MMC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -34,7 +34,7 @@
 	 ######################################################
 	 3.9 MiB/s
 done
-Bytes transferred = 5554352 (54c0b0 hex)
+Bytes transferred = 5553960 (54bf28 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -45,7 +45,7 @@
 done
 Bytes transferred = 67133 (1063d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x54c0b0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x54bf28 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301363c
@@ -54,7 +54,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc7-next-20170210-152444-g632571b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Feb 10 00:04:29 PST 2017
+Linux version 4.10.0-rc7-next-20170213-152822-g8813198 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Feb 12 23:04:05 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -64,20 +64,20 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0d5ed00, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0d5ec00, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 16 pages/cpu @eef85000 s34624 r8192 d22720 u65536
-pcpu-alloc: s34624 r8192 d22720 u65536 alloc=16*4096
+percpu: Embedded 16 pages/cpu @eef85000 s34688 r8192 d22656 u65536
+pcpu-alloc: s34688 r8192 d22656 u65536 alloc=16*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=b1791faa-84e8-4e27-83a1-9fb0a77e8803 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999804K/2096128K available (8192K kernel code, 384K rwdata, 2488K rodata, 1024K init, 369K bss, 30788K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1999804K/2096128K available (8192K kernel code, 384K rwdata, 2492K rodata, 1024K init, 369K bss, 30788K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -87,9 +87,10 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d60260   ( 385 kB)
-       .bss : 0xc0d62000 - 0xc0dbe6ec   ( 370 kB)
+      .data : 0xc0d00000 - 0xc0d600e0   ( 385 kB)
+       .bss : 0xc0d62000 - 0xc0dbe66c   ( 370 kB)
 Preemptible hierarchical RCU implementation.
+	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
 /interrupt-controller@60004000: 160 interrupts forwarded to /interrupt-controller@50041000
 Tegra clk 127: register failed with -17
@@ -407,6 +408,7 @@
 pci 0000:02:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
 pci 0000:02:00.0: supports D1 D2
 pci 0000:02:00.0: PME# supported from D0 D1 D2 D3hot D3cold
+ mmcblk1: p1 p2 p3
 PCI: bus2: Fast back to back transfers disabled
 pci_bus 0000:02: busn_res: [bus 02-ff] end is updated to 02
 pci 0000:00:01.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
@@ -431,14 +433,13 @@
 pcieport 0000:00:02.0: Signaling PME with IRQ 394
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
-r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0af7000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 395
+r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0c01000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 395
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
-[drm] Cannot find any crtc or sizes - going 1024x768
- mmcblk1: p1 p2 p3
-Console: switching to colour frame buffer device 128x48
+tegra-hdmi 54280000.hdmi: cannot set audio to 48000 Hz at 297000000 Hz pixel clock
+Console: switching to colour frame buffer device 512x135
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
 of: dev_pm_opp_of_cpumask_add_table: couldn't find opp table for cpu:0, -19
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-02-12 23:13:21.540228921 -0800
+++ /tmp/foob	2017-02-12 23:13:21.544228936 -0800
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -42,9 +42,9 @@
 	 #################################################################
 	 #################################################################
 	 #########################################################
-	 3.5 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 7510704 (729ab0 hex)
+Bytes transferred = 7513912 (72a738 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -55,7 +55,7 @@
 done
 Bytes transferred = 68155 (10a3b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x729ab0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x72a738 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a3a
@@ -63,7 +63,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc7-next-20170210-152444-g632571b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Fri Feb 10 00:04:49 PST 2017
+Linux version 4.10.0-rc7-next-20170213-152822-g8813198 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Feb 12 23:04:13 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -75,7 +75,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c153a800, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c153a700, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -98,13 +98,14 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc153c1a0   (1265 kB)
-       .bss : 0xc153e000 - 0xc15c4178   ( 537 kB)
+      .data : 0xc1400000 - 0xc153c020   (1265 kB)
+       .bss : 0xc153e000 - 0xc15c41f8   ( 537 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU debugfs-based tracing is enabled.
+	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=16 to nr_cpu_ids=4.
-RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=4
+RCU: Adjusting geometry for rcu_fanout_leaf=32, nr_cpu_ids=4
 NR_IRQS:16 nr_irqs:16 16
 /interrupt-controller@60004000: 160 interrupts forwarded to /interrupt-controller@50041000
 Tegra clk 127: register failed with -17
@@ -294,8 +295,6 @@
 mmc0: Unknown controller version (3). You may experience problems.
 mmc0: Invalid maximum block size, assuming 512 bytes
 mmc0: SDHCI controller on 700b0600.sdhci [700b0600.sdhci] using ADMA
-elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
-input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 ledtrig-cpu: registered to indicate activity on CPUs
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
@@ -304,6 +303,7 @@
 sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
 NET: Registered protocol family 17
 can: controller area network core (rev 20120528 abi 9)
+elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
 NET: Registered protocol family 29
 can: raw protocol (rev 20120528)
 can: broadcast manager protocol (rev 20161123 t)
@@ -326,14 +326,13 @@
 +5V_HDMI_CON: supplied by +5V_SYS
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
  usb3-1: usb3-1 supply vbus not found, using dummy regulator
-elan_i2c 1-0015: failed to read report data: -121
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
-mmc0: new high speed MMC card at address 0001
 sdhci-tegra 700b0000.sdhci: allocated mmc-pwrseq
 mmc1: Unknown controller version (3). You may experience problems.
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: Switching to 3.3V signalling voltage failed
+mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
 mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
@@ -343,10 +342,10 @@
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc2: Unknown controller version (3). You may experience problems.
-elan_i2c 1-0015: invalid report id data (ff)
 mmc2: Invalid maximum block size, assuming 512 bytes
 mmc1: new high speed SDIO card at address 0001
 mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
+input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
 mmc2: new high speed SDHC card at address 0007
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2017-02-12 23:13:21.552228967 -0800
+++ /tmp/foob	2017-02-12 23:13:21.552228967 -0800
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -42,18 +42,18 @@
 	 ######################################################
 	 3.6 MiB/s
 done
-Bytes transferred = 5554352 (54c0b0 hex)
+Bytes transferred = 5553960 (54bf28 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.6 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 68155 (10a3b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x54c0b0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x54bf28 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a3a
@@ -62,7 +62,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc7-next-20170210-152444-g632571b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Feb 10 00:04:29 PST 2017
+Linux version 4.10.0-rc7-next-20170213-152822-g8813198 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Feb 12 23:04:05 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -73,20 +73,20 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c0d5ed00, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0d5ec00, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327679 pages, LIFO batch:31
-percpu: Embedded 16 pages/cpu @eef84000 s34624 r8192 d22720 u65536
-pcpu-alloc: s34624 r8192 d22720 u65536 alloc=16*4096
+percpu: Embedded 16 pages/cpu @eef84000 s34688 r8192 d22656 u65536
+pcpu-alloc: s34688 r8192 d22656 u65536 alloc=16*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522751
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=4d36e82a-b3c0-4b4b-bfcd-afe5217ac023 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2000820K/2097148K available (8192K kernel code, 384K rwdata, 2488K rodata, 1024K init, 369K bss, 30792K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 2000820K/2097148K available (8192K kernel code, 384K rwdata, 2492K rodata, 1024K init, 369K bss, 30792K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -96,9 +96,10 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d60260   ( 385 kB)
-       .bss : 0xc0d62000 - 0xc0dbe6ec   ( 370 kB)
+      .data : 0xc0d00000 - 0xc0d600e0   ( 385 kB)
+       .bss : 0xc0d62000 - 0xc0dbe66c   ( 370 kB)
 Preemptible hierarchical RCU implementation.
+	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
 /interrupt-controller@60004000: 160 interrupts forwarded to /interrupt-controller@50041000
 Tegra clk 127: register failed with -17
@@ -292,15 +293,15 @@
 Segment Routing with IPv6
 mip6: Mobile IPv6
 mmc0: new high speed MMC card at address 0001
-sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
 mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
-mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
+sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
 NET: Registered protocol family 17
-NET: Registered protocol family 15
-can: controller area network core (rev 20120528 abi 9)
+mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
 mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
+NET: Registered protocol family 15
+can: controller area network core (rev 20120528 abi 9)
 NET: Registered protocol family 29
 can: raw protocol (rev 20120528)
 can: broadcast manager protocol (rev 20161123 t)
@@ -384,10 +385,10 @@
 EXT4-fs (mmcblk0p3): recovery complete
 EXT4-fs (mmcblk0p3): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:3.
+uvcvideo: Found UVC 1.00 device HD WebCam (1bcf:2c67)
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K
-uvcvideo: Found UVC 1.00 device HD WebCam (1bcf:2c67)
 input: HD WebCam as /devices/soc0/70090000.usb/usb1/1-2/1-2:1.0/input/input5
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.913638] usb 1-3: new high-speed USB device number 3 using tegra-xusb
+root@(none):/# [    2.903847] usb 1-3: new high-speed USB device number 3 using tegra-xusb
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2017-02-12 23:13:21.572229044 -0800
+++ /tmp/foob	2017-02-12 23:13:21.576229060 -0800
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1800688 us
+Bootloader-Cpu Init at (time stamp): 1800701 us
 
 Pinmux changes applied in kernel way
 
@@ -62,20 +62,20 @@
 Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1800662  us
-Bootloader (AosInit) time: 733443 us
-Bootloader Main Init time: 180251 us
-BlockDevMgrInit time: 157500 us
+Miscellaneous avp time: 1800675  us
+Bootloader (AosInit) time: 734234 us
+Bootloader Main Init time: 180077 us
+BlockDevMgrInit time: 157539 us
 PartMgrInit time: 8 us
-StorMgrInit time: 31 us
+StorMgrInit time: 30 us
 FsMgrInit time: 2 us
-Partition Table load: 77135 us 
-Display init time: 574531 us
-Kernel read time: 378 msec
+Partition Table load: 76651 us 
+Display init time: 574812 us
+Kernel read time: 377 msec
 Load Kernel Size:   16304128 Bytes
-Bl Kernel ReadSpeed: 43132000 Bytes/sec
-Pre-Kernel Jump time: 1740583 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5842491 us
+Bl Kernel ReadSpeed: 43247000 Bytes/sec
+Pre-Kernel Jump time: 1740817 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5844910 us
 Booting Linux on physical CPU 0x0
 Linux version 4.8.0-239303-g3477d16 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Fri Oct 7 01:12:37 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
@@ -184,7 +184,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(1.151:1): initialized
+audit: type=2000 audit(1.147:1): initialized
 workingset: timestamp_bits=46 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-02-12 23:13:21.580229076 -0800
+++ /tmp/foob	2017-02-12 23:13:21.584229090 -0800
@@ -47,20 +47,20 @@
 	 #################################################################
 	 #################################################################
 	 #########################################################
-	 4.1 MiB/s
+	 3.5 MiB/s
 done
-Bytes transferred = 7510704 (729ab0 hex)
+Bytes transferred = 7513912 (72a738 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 2.6 MiB/s
 done
 Bytes transferred = 21426 (53b2 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x729ab0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x72a738 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103b1
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc7-next-20170210-152444-g632571b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Fri Feb 10 00:04:49 PST 2017
+Linux version 4.10.0-rc7-next-20170213-152822-g8813198 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Feb 12 23:04:13 PST 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -78,7 +78,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c153a800, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c153a700, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -101,13 +101,14 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc153c1a0   (1265 kB)
-       .bss : 0xc153e000 - 0xc15c4178   ( 537 kB)
+      .data : 0xc1400000 - 0xc153c020   (1265 kB)
+       .bss : 0xc153e000 - 0xc15c41f8   ( 537 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	RCU debugfs-based tracing is enabled.
+	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=16 to nr_cpu_ids=2.
-RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=2
+RCU: Adjusting geometry for rcu_fanout_leaf=32, nr_cpu_ids=2
 NR_IRQS:16 nr_irqs:16 16
 /interrupt-controller@60004000: 128 interrupts forwarded to /interrupt-controller@50041000
 L2C: platform modifies aux control register: 0x02080000 -> 0x3e480001
@@ -330,23 +331,23 @@
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
-tegra-ehci c5000000.usb: EHCI Host Controller
-tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
-tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
+tegra-ehci c5000000.usb: EHCI Host Controller
  mmcblk0: p1
+tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
+tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
+rtc-em3027 2-0056: setting system clock to 2043-02-07 05:57:56 UTC (2306901476)
+EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
-rtc-em3027 2-0056: setting system clock to 2015-02-07 05:47:37 UTC (1423288057)
-EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2017-02-12 23:13:21.588229105 -0800
+++ /tmp/foob	2017-02-12 23:13:21.592229121 -0800
@@ -45,9 +45,9 @@
 	 #################################################################
 	 #################################################################
 	 ######################################################
-	 4.1 MiB/s
+	 4 MiB/s
 done
-Bytes transferred = 5554352 (54c0b0 hex)
+Bytes transferred = 5553960 (54bf28 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21426 (53b2 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x54c0b0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x54bf28 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103b1
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc7-next-20170210-152444-g632571b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Feb 10 00:04:29 PST 2017
+Linux version 4.10.0-rc7-next-20170213-152822-g8813198 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Feb 12 23:04:05 PST 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -76,20 +76,20 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0d5ed00, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0d5ec00, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 65536 pages, LIFO batch:15
-percpu: Embedded 16 pages/cpu @ef7c5000 s34624 r8192 d22720 u65536
-pcpu-alloc: s34624 r8192 d22720 u65536 alloc=16*4096
+percpu: Embedded 16 pages/cpu @ef7c5000 s34688 r8192 d22656 u65536
+pcpu-alloc: s34688 r8192 d22656 u65536 alloc=16*4096
 pcpu-alloc: [0] 0 [0] 1 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 260608
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=11eb23c1-01 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 960744K/1048576K available (8192K kernel code, 384K rwdata, 2488K rodata, 1024K init, 369K bss, 22296K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 960744K/1048576K available (8192K kernel code, 384K rwdata, 2492K rodata, 1024K init, 369K bss, 22296K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -99,11 +99,12 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d60260   ( 385 kB)
-       .bss : 0xc0d62000 - 0xc0dbe6ec   ( 370 kB)
+      .data : 0xc0d00000 - 0xc0d600e0   ( 385 kB)
+       .bss : 0xc0d62000 - 0xc0dbe66c   ( 370 kB)
 Preemptible hierarchical RCU implementation.
+	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
-RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=2
+RCU: Adjusting geometry for rcu_fanout_leaf=32, nr_cpu_ids=2
 NR_IRQS:16 nr_irqs:16 16
 /interrupt-controller@60004000: 128 interrupts forwarded to /interrupt-controller@50041000
 L2C: platform modifies aux control register: 0x02080000 -> 0x3e480001
@@ -313,22 +314,22 @@
 tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
 NET: Registered protocol family 10
 Segment Routing with IPv6
-mmc0: host does not support reading read-only switch, assuming write-enable
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
+mmc0: host does not support reading read-only switch, assuming write-enable
+mmc1: new high speed SD card at address b368
 mmc0: new high speed SDHC card at address b368
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
+ mmcblk1: p1 p2
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
  mmcblk0: p1
 mip6: Mobile IPv6
 sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
 NET: Registered protocol family 17
 NET: Registered protocol family 15
-mmc1: new high speed SD card at address b368
 can: controller area network core (rev 20120528 abi 9)
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 NET: Registered protocol family 29
 can: raw protocol (rev 20120528)
 can: broadcast manager protocol (rev 20161123 t)
- mmcblk1: p1 p2
 can: netlink gateway (rev 20130117) max_hops=1
 Bluetooth: RFCOMM socket layer initialized
 Bluetooth: RFCOMM ver 1.11
@@ -342,13 +343,13 @@
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
-hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
+hub 1-1:1.0: USB hub found
+hub 1-1:1.0: 4 ports detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2015-02-07 05:45:50 UTC (1423287950)
+rtc-em3027 2-0056: setting system clock to 2043-02-07 05:55:14 UTC (2306901314)
 ALSA device list:
   #0: tegra-trimslice
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
@@ -357,7 +358,7 @@
 asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:1.
+VFS: Mounted root (ext3 filesystem) on device 179:17.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2017-02-12 23:13:21.624229244 -0800
+++ /tmp/foob	2017-02-12 23:13:21.628229260 -0800
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 769316 us
-Basic modules init took 348534 us
-Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14370 us
-Carveout took 252717 us
-CPU initialization took 92319 us
-Total time taken by TegraBoot 707940 us
+Tegraboot started after 737721 us
+Basic modules init took 349187 us
+Sec Bootdevice Read Time = 203 ms, Read Size = 8459 KB
+Next stage binary read took 15529 us
+Carveout took 252726 us
+CPU initialization took 92306 us
+Total time taken by TegraBoot 709748 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.607] LPDDR4 Training: Number of tables = 10
+64b[0001.578] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,9 +117,9 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-15346176 bytes read in 804 ms (18.2 MiB/s)
+15340032 bytes read in 804 ms (18.2 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-44882 bytes read in 474 ms (91.8 KiB/s)
+44882 bytes read in 478 ms (90.8 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
 1627787 bytes read in 495 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc7-next-20170210-176877-g632571b (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Fri Feb 10 00:07:47 PST 2017
+Linux version 4.10.0-rc7-next-20170213-177255-g8813198 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Feb 12 23:07:37 PST 2017
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -163,8 +163,8 @@
 psci: PSCIv0.2 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: MIGRATE_INFO_TYPE not supported.
-percpu: Embedded 23 pages/cpu @ffff8000bff4e000 s56832 r8192 d29184 u94208
-pcpu-alloc: s56832 r8192 d29184 u94208 alloc=23*4096
+percpu: Embedded 23 pages/cpu @ffff8000bff4e000 s56960 r8192 d29056 u94208
+pcpu-alloc: s56960 r8192 d29056 u94208 alloc=23*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
@@ -174,15 +174,15 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffff80007a000000-ffff80007dffffff]
-Memory: 2983708K/3131392K available (8700K kernel code, 970K rwdata, 4196K rodata, 1088K init, 396K bss, 131300K reserved, 16384K cma-reserved)
+Memory: 2983712K/3131392K available (8700K kernel code, 964K rwdata, 4196K rodata, 1088K init, 398K bss, 131296K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008900000   (  8704 KB)
     .rodata : 0xffff000008900000 - 0xffff000008d20000   (  4224 KB)
       .init : 0xffff000008d20000 - 0xffff000008e30000   (  1088 KB)
-      .data : 0xffff000008e30000 - 0xffff000008f22a00   (   971 KB)
-       .bss : 0xffff000008f22a00 - 0xffff000008f85bb8   (   397 KB)
+      .data : 0xffff000008e30000 - 0xffff000008f21200   (   965 KB)
+       .bss : 0xffff000008f21200 - 0xffff000008f84a38   (   399 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -190,8 +190,9 @@
     memory  : 0xffff800000000000 - 0xffff8000c0000000   (  3072 MB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Preemptible hierarchical RCU implementation.
+	Build-time adjustment of leaf fanout to 64.
 	RCU restricting CPUs from NR_CPUS=64 to nr_cpu_ids=4.
-RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=4
+RCU: Adjusting geometry for rcu_fanout_leaf=64, nr_cpu_ids=4
 NR_IRQS:64 nr_irqs:64 0
 GIC: Using split EOI/Deactivate mode
 /interrupt-controller@60004000: 192 interrupts forwarded to /interrupt-controller@50041000
@@ -356,6 +357,6 @@
 devtmpfs: mounted
 Freeing unused kernel memory: 1088K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
-bash: no job contr[    1.344600] random: fast init done
+bash: no job contr[    1.372074] random: fast init done
 ol in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2017-02-12 23:13:21.644229321 -0800
+++ /tmp/foob	2017-02-12 23:13:21.648229337 -0800
@@ -21,7 +21,7 @@
 psci: PSCIv1.0 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: MIGRATE_INFO_UP_CPU reported invalid physical ID (0x80000000)
-percpu: Embedded 23 pages/cpu @ffff8000bff42000 s56832 r8192 d29184 u94208
+percpu: Embedded 23 pages/cpu @ffff8000bff42000 s56960 r8192 d29056 u94208
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
 CPU features: enabling workaround for ARM erratum 834220
@@ -30,15 +30,15 @@
 Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2837892K/2988032K available (8700K kernel code, 970K rwdata, 4196K rodata, 1088K init, 396K bss, 133756K reserved, 16384K cma-reserved)
+Memory: 2837900K/2988032K available (8700K kernel code, 964K rwdata, 4196K rodata, 1088K init, 398K bss, 133748K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008900000   (  8704 KB)
     .rodata : 0xffff000008900000 - 0xffff000008d20000   (  4224 KB)
       .init : 0xffff000008d20000 - 0xffff000008e30000   (  1088 KB)
-      .data : 0xffff000008e30000 - 0xffff000008f22a00   (   971 KB)
-       .bss : 0xffff000008f22a00 - 0xffff000008f85bb8   (   397 KB)
+      .data : 0xffff000008e30000 - 0xffff000008f21200   (   965 KB)
+       .bss : 0xffff000008f21200 - 0xffff000008f84a38   (   399 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -46,8 +46,9 @@
     memory  : 0xffff800000000000 - 0xffff8000c0000000   (  3072 MB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Preemptible hierarchical RCU implementation.
+	Build-time adjustment of leaf fanout to 64.
 	RCU restricting CPUs from NR_CPUS=64 to nr_cpu_ids=4.
-RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=4
+RCU: Adjusting geometry for rcu_fanout_leaf=64, nr_cpu_ids=4
 NR_IRQS:64 nr_irqs:64 0
 GIC: Using split EOI/Deactivate mode
 /interrupt-controller@60004000: 192 interrupts forwarded to /interrupt-controller@50041000
@@ -66,7 +67,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x4f28: expected 0x2004f400
+boot misc3 0x5288: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -79,7 +80,7 @@
 console [tty0] enabled
 bootconsole [uart0] disabled
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc7-next-20170210-176877-g632571b (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Fri Feb 10 00:07:47 PST 2017
+Linux version 4.10.0-rc7-next-20170213-177255-g8813198 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Feb 12 23:07:37 PST 2017
 Boot CPU: AArch64 Processor [411fd071]
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
@@ -104,7 +105,7 @@
 psci: PSCIv1.0 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: MIGRATE_INFO_UP_CPU reported invalid physical ID (0x80000000)
-percpu: Embedded 23 pages/cpu @ffff8000bff42000 s56832 r8192 d29184 u94208
+percpu: Embedded 23 pages/cpu @ffff8000bff42000 s56960 r8192 d29056 u94208
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
 CPU features: enabling workaround for ARM erratum 834220
@@ -113,15 +114,15 @@
 Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2837892K/2988032K available (8700K kernel code, 970K rwdata, 4196K rodata, 1088K init, 396K bss, 133756K reserved, 16384K cma-reserved)
+Memory: 2837900K/2988032K available (8700K kernel code, 964K rwdata, 4196K rodata, 1088K init, 398K bss, 133748K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008900000   (  8704 KB)
     .rodata : 0xffff000008900000 - 0xffff000008d20000   (  4224 KB)
       .init : 0xffff000008d20000 - 0xffff000008e30000   (  1088 KB)
-      .data : 0xffff000008e30000 - 0xffff000008f22a00   (   971 KB)
-       .bss : 0xffff000008f22a00 - 0xffff000008f85bb8   (   397 KB)
+      .data : 0xffff000008e30000 - 0xffff000008f21200   (   965 KB)
+       .bss : 0xffff000008f21200 - 0xffff000008f84a38   (   399 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -129,8 +130,9 @@
     memory  : 0xffff800000000000 - 0xffff8000c0000000   (  3072 MB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Preemptible hierarchical RCU implementation.
+	Build-time adjustment of leaf fanout to 64.
 	RCU restricting CPUs from NR_CPUS=64 to nr_cpu_ids=4.
-RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=4
+RCU: Adjusting geometry for rcu_fanout_leaf=64, nr_cpu_ids=4
 NR_IRQS:64 nr_irqs:64 0
 GIC: Using split EOI/Deactivate mode
 /interrupt-controller@60004000: 192 interrupts forwarded to /interrupt-controller@50041000
@@ -149,7 +151,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x4f28: expected 0x2004f400
+boot misc3 0x5288: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -229,7 +231,7 @@
 RPC: Registered tcp transport module.
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 Unpacking initramfs...
-Freeing initrd memory: 2364K
+Freeing initrd memory: 2360K
 kvm [1]: 8-bit VMID
 kvm [1]: IDMAP page: 808eb000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
@@ -239,7 +241,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.327:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.323:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -332,7 +334,7 @@
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
-max77686-rtc max77620-rtc: setting system clock to 2000-11-21 13:35:32 UTC (974813732)
+max77686-rtc max77620-rtc: setting system clock to 2000-11-24 12:35:17 UTC (975069317)
 PP1200_AVDD: disabling
 PP1200_RCAM: disabling
 PP2800L_RCAM: disabling
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-02-12 23:13:21.652229353 -0800
+++ /tmp/foob	2017-02-12 23:13:21.652229353 -0800
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 #########################################################
-	 3.2 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 7510704 (729ab0 hex)
+Bytes transferred = 7513912 (72a738 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.3 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 51804 (ca5c hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x729ab0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x72a738 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa5b ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc7-next-20170210-152444-g632571b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Fri Feb 10 00:04:49 PST 2017
+Linux version 4.10.0-rc7-next-20170213-152822-g8813198 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Feb 12 23:04:13 PST 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -70,7 +70,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c153a800, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c153a700, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -93,13 +93,14 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc153c1a0   (1265 kB)
-       .bss : 0xc153e000 - 0xc15c4178   ( 537 kB)
+      .data : 0xc1400000 - 0xc153c020   (1265 kB)
+       .bss : 0xc153e000 - 0xc15c41f8   ( 537 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU debugfs-based tracing is enabled.
+	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=16 to nr_cpu_ids=4.
-RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=4
+RCU: Adjusting geometry for rcu_fanout_leaf=32, nr_cpu_ids=4
 NR_IRQS:16 nr_irqs:16 16
 /interrupt-controller@60004000: 160 interrupts forwarded to /interrupt-controller@50041000
 L2C: platform modifies aux control register: 0x02080000 -> 0x3e480001
@@ -304,19 +305,19 @@
 sys_3v3,vdd_3v3_alw: supplied by vdd_5v_in
 sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
 +VDD_5V_HDMI: supplied by sys_3v3,vdd_3v3_alw
+mmc0: new ultra high speed DDR50 SDHC card at address aaaa
 tegra-pcie 3000.pcie-controller: 2x3 configuration
 tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
+mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB
-mmc0: new ultra high speed DDR50 SDHC card at address aaaa
-mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
-Primary GPT is invalid, using alternate GPT.
- mmcblk1: p1 p2 p3 p4 p5
 tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
  mmcblk0: p1 p2 p3
+Primary GPT is invalid, using alternate GPT.
+ mmcblk1: p1 p2 p3 p4 p5
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -389,10 +390,10 @@
 EXT4-fs (mmcblk1p1): warning: mounting fs with errors, running e2fsck is recommended
 EXT4-fs (mmcblk1p1): recovery complete
 EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:1.
+VFS: Mounted root (ext3 filesystem) on device 179:17.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.094010] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.084012] usb 2-1: new high-speed USB device number 2 using tegra-ehci
 asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0f:67:60
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2017-02-12 23:13:21.660229382 -0800
+++ /tmp/foob	2017-02-12 23:13:21.664229398 -0800
@@ -37,9 +37,9 @@
 	 #################################################################
 	 #################################################################
 	 ######################################################
-	 3.3 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 5554352 (54c0b0 hex)
+Bytes transferred = 5553960 (54bf28 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 51804 (ca5c hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x54c0b0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x54bf28 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa5b ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc7-next-20170210-152444-g632571b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Feb 10 00:04:29 PST 2017
+Linux version 4.10.0-rc7-next-20170213-152822-g8813198 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Feb 12 23:04:05 PST 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,20 +68,20 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0d5ed00, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0d5ec00, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 16 pages/cpu @eef8b000 s34624 r8192 d22720 u65536
-pcpu-alloc: s34624 r8192 d22720 u65536 alloc=16*4096
+percpu: Embedded 16 pages/cpu @eef8b000 s34688 r8192 d22656 u65536
+pcpu-alloc: s34688 r8192 d22656 u65536 alloc=16*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=edbe1840-3633-c751-7993-b0ac834f81e0 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999844K/2096128K available (8192K kernel code, 384K rwdata, 2488K rodata, 1024K init, 369K bss, 30748K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1999844K/2096128K available (8192K kernel code, 384K rwdata, 2492K rodata, 1024K init, 369K bss, 30748K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -91,9 +91,10 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d60260   ( 385 kB)
-       .bss : 0xc0d62000 - 0xc0dbe6ec   ( 370 kB)
+      .data : 0xc0d00000 - 0xc0d600e0   ( 385 kB)
+       .bss : 0xc0d62000 - 0xc0dbe66c   ( 370 kB)
 Preemptible hierarchical RCU implementation.
+	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
 /interrupt-controller@60004000: 160 interrupts forwarded to /interrupt-controller@50041000
 L2C: platform modifies aux control register: 0x02080000 -> 0x3e480001
@@ -389,4 +390,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.370058] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.370197] usb 2-1: new high-speed USB device number 2 using tegra-ehci
