Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2017-08-28 14:08:24.613467258 -0700
+++ /tmp/foob	2017-08-28 14:08:24.613467258 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc6-next-20170824-196160-g9506597d (jonathanh@sellafield) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Mon Aug 28 13:37:21 PDT 2017
+Linux version 4.13.0-rc6-next-20170825-314808-g7159188 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Mon Aug 28 14:04:57 PDT 2017
 Boot CPU: AArch64 Processor [411fd070]
 Machine model: linux,dummy-virt
 Memory limited to 1024MB
@@ -29,15 +29,15 @@
 Policy zone: DMA
 Kernel command line: root=/dev/vda2 rw rootwait mem=1024M console=ttyAMA0,38400n8 init=/bin/bash
 PID hash table entries: 4096 (order: 3, 32768 bytes)
-Memory: 998976K/1048576K available (9212K kernel code, 1142K rwdata, 4552K rodata, 1152K init, 402K bss, 33216K reserved, 16384K cma-reserved)
+Memory: 998972K/1048576K available (9212K kernel code, 1144K rwdata, 4544K rodata, 1152K init, 404K bss, 33220K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008980000   (  9216 KB)
     .rodata : 0xffff000008980000 - 0xffff000008e00000   (  4608 KB)
       .init : 0xffff000008e00000 - 0xffff000008f20000   (  1152 KB)
-      .data : 0xffff000008f20000 - 0xffff00000903da00   (  1143 KB)
-       .bss : 0xffff00000903da00 - 0xffff0000090a23f0   (   403 KB)
+      .data : 0xffff000008f20000 - 0xffff00000903e200   (  1145 KB)
+       .bss : 0xffff00000903e200 - 0xffff0000090a33f0   (   405 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -114,7 +114,7 @@
 hw perfevents: enabled with armv8_pmuv3 PMU driver, 1 counters available
 kvm [1]: HYP mode not available
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.796:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.820:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -179,7 +179,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2017-08-28 20:38:40 UTC (1503952720)
+rtc-pl031 9010000.pl031: setting system clock to 2017-08-28 21:06:01 UTC (1503954361)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-08-28 14:08:24.633467331 -0700
+++ /tmp/foob	2017-08-28 14:08:24.637467346 -0700
@@ -40,7 +40,7 @@
 	 #################################################################
 	 #################################################################
 	 ###################################
-	 2.8 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 8135176 (7c2208 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
@@ -49,7 +49,7 @@
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.6 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 35036 (88dc hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2017-08-28 14:08:24.641467361 -0700
+++ /tmp/foob	2017-08-28 14:08:24.645467376 -0700
@@ -38,9 +38,9 @@
 	 #################################################################
 	 #################################################################
 	 ######################################
-	 3 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 6280856 (5fd698 hex)
+Bytes transferred = 6282712 (5fddd8 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -51,7 +51,7 @@
 done
 Bytes transferred = 35036 (88dc hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5fd698 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5fddd8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8db ... OK
@@ -60,7 +60,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc6-next-20170824-196160-g9506597d (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Aug 28 13:33:59 PDT 2017
+Linux version 4.13.0-rc6-next-20170825-196567-g7159188 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Aug 28 14:00:18 PDT 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -70,7 +70,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0fb7e40, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0fb7d80, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -94,7 +94,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
-      .data : 0xc0f00000 - 0xc0fc2dd0   ( 780 kB)
+      .data : 0xc0f00000 - 0xc0fc2d10   ( 780 kB)
        .bss : 0xc0fcc5b0 - 0xc10176fc   ( 301 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
@@ -150,107 +150,107 @@
 [00000020] *pgd=00000000
 Internal error: Oops: 5 [#1] PREEMPT SMP ARM
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.13.0-rc6-next-20170824-196160-g9506597d #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.13.0-rc6-next-20170825-196567-g7159188 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 task: ee88f740 task.stack: ee8a2000
 PC is at sysfs_add_link_to_group+0x8/0x48
 LR is at iommu_device_link+0x3c/0x9c
-pc : [<c02a956c>]    lr : [<c048b4c4>]    psr: 60000013
+pc : [<c02a9388>]    lr : [<c048b580>]    psr: 60000013
 sp : ee8a3d40  ip : ee8a3d70  fp : 00000000
-r10: c0f8e39c  r9 : eea34a10  r8 : 00500050
+r10: c0f8e29c  r9 : eea34a10  r8 : 00500050
 r7 : ee9d8818  r6 : eefdde2c  r5 : ee9d8810  r4 : ee9d8818
-r3 : ee9db280  r2 : ee9d8818  r1 : c0bec2f8  r0 : 00000008
+r3 : ee9db280  r2 : ee9d8818  r1 : c0bec64c  r0 : 00000008
 Flags: nZCv  IRQs on  FIQs on  Mode SVC_32  ISA ARM  Segment none
 Control: 10c5387d  Table: 8000406a  DAC: 00000051
 Process swapper/0 (pid: 1, stack limit = 0xee8a2210)
 Stack: (0xee8a3d40 to 0xee8a4000)
-3d40: eea42650 ee9d8810 eefdde2c c048b4c4 eea3db90 00000001 ee8a3d70 eefdde2c
-3d60: ee9d8810 c048d6bc ee8a3d70 6d656d2e eeff1e80 00000001 00000001 c0007265
-3d80: eea3da40 c043cd64 c0489c68 a0000013 eea3d9c0 c0238424 000000a5 c0489c68
-3da0: ee9b8db8 ee8a3de0 ee9b8e34 00000000 c0519c9c eea34a10 c0f8e39c ee9d8810
-3dc0: c0a588cc c0488e64 00000000 c0488e90 00000000 00000000 ee8a3dfc c0519e40
-3de0: ee8c965c ee9b8db4 c0f69188 00000000 eea3d9c0 c0489cbc eea42610 c0a588cc
-3e00: 2000003f eea42610 20000020 2000003f eea34a10 c048e654 ee826100 ee8a3e30
-3e20: c0c48284 eea3db90 00000000 00000000 00000041 c06d6de0 00000000 00000000
+3d40: eea42650 ee9d8810 eefdde2c c048b580 eea3db90 00000001 ee8a3d70 eefdde2c
+3d60: ee9d8810 c048d778 ee8a3d70 6d656d2e eeff1e80 00000001 00000001 c0007265
+3d80: eea3da40 c043cd98 c0489d24 a0000013 eea3d9c0 c023824c 000000a5 c0489d24
+3da0: ee9b8db8 ee8a3de0 ee9b8e34 00000000 c0519d58 eea34a10 c0f8e29c ee9d8810
+3dc0: c0a588cc c0488f20 00000000 c0488f4c 00000000 00000000 ee8a3dfc c0519efc
+3de0: ee8c965c ee9b8db4 c0f69188 00000000 eea3d9c0 c0489d78 eea42610 c0a588cc
+3e00: 2000003f eea42610 20000020 2000003f eea34a10 c048e710 ee826100 ee8a3e30
+3e20: c0c48614 eea3db90 00000000 00000000 00000041 c06d6dbc 00000000 00000000
 3e40: 00000000 00000001 eea35cb8 eea43d98 eea34a00 00000003 00000000 eea43cb8
-3e60: 00000000 ffffffed eea34a10 fffffdfb c0f8e39c 00000000 00000000 c0f8e39c
-3e80: 00000000 c051d07c c051d02c eea34a10 c1009250 c1009254 00000000 c051b78c
-3ea0: eea34a10 c0f8e39c eea34a44 00000000 00000111 c0e6a83c c0e005a4 c051b8e4
-3ec0: 00000000 c0f8e39c c051b838 c0519e40 ee8c965c eea2f2b4 c0f8e39c eea42680
-3ee0: c0f69188 c051ace4 c0c483f0 c0f8e39c 00000003 c0f8e39c 00000003 c06d6ac8
-3f00: 00000000 c051c45c c051ccd4 c0e82f04 00000003 c0101b8c 00000000 c0cb9104
-3f20: 00000100 efffcc3c c0cb9104 00000111 efffcc4a c013e478 c0cb7bb0 00000000
+3e60: 00000000 ffffffed eea34a10 fffffdfb c0f8e29c 00000000 00000000 c0f8e29c
+3e80: 00000000 c051d138 c051d0e8 eea34a10 c1009250 c1009254 00000000 c051b848
+3ea0: eea34a10 c0f8e29c eea34a44 00000000 00000111 c0e6a83c c0e005a4 c051b9a0
+3ec0: 00000000 c0f8e29c c051b8f4 c0519efc ee8c965c eea2f2b4 c0f8e29c eea42680
+3ee0: c0f69188 c051ada0 c0c48780 c0f8e29c 00000003 c0f8e29c 00000003 c06d6aa4
+3f00: 00000000 c051c518 c051cd90 c0e82f04 00000003 c0101b8c 00000000 c0cb9108
+3f20: 00000100 efffcc3c c0cb9108 00000111 efffcc4a c013e478 c0cb7bb4 00000000
 3f40: 00000003 00000003 efffcc3c efffcc41 c0f0dcd0 efffcbc0 00000003 c0e6a820
 3f60: c0e82f04 00000003 c0e6a824 c0fcc5c0 00000111 c0e00da8 00000003 00000003
-3f80: 00000000 c0e005a4 00000000 c0970374 00000000 00000000 00000000 00000000
-3fa0: 00000000 c097037c 00000000 c0107ce0 00000000 00000000 00000000 00000000
+3f80: 00000000 c0e005a4 00000000 c09708f4 00000000 00000000 00000000 00000000
+3fa0: 00000000 c09708fc 00000000 c0107ce0 00000000 00000000 00000000 00000000
 3fc0: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
 3fe0: 00000000 00000000 00000000 00000000 00000013 00000000 ffffffff ffffffff
-[<c02a956c>] (sysfs_add_link_to_group) from [<c048b4c4>] (iommu_device_link+0x3c/0x9c)
-[<c048b4c4>] (iommu_device_link) from [<c048d6bc>] (tegra_smmu_add_device+0x9c/0xa8)
-[<c048d6bc>] (tegra_smmu_add_device) from [<c0488e90>] (add_iommu_group+0x2c/0x58)
-[<c0488e90>] (add_iommu_group) from [<c0519e40>] (bus_for_each_dev+0x54/0x88)
-[<c0519e40>] (bus_for_each_dev) from [<c0489cbc>] (bus_set_iommu+0x94/0xd8)
-[<c0489cbc>] (bus_set_iommu) from [<c048e654>] (tegra_smmu_probe+0x17c/0x264)
-[<c048e654>] (tegra_smmu_probe) from [<c06d6de0>] (tegra_mc_probe+0x308/0x51c)
-[<c06d6de0>] (tegra_mc_probe) from [<c051d07c>] (platform_drv_probe+0x50/0xb0)
-[<c051d07c>] (platform_drv_probe) from [<c051b78c>] (driver_probe_device+0x238/0x2e4)
-[<c051b78c>] (driver_probe_device) from [<c051b8e4>] (__driver_attach+0xac/0xb0)
-[<c051b8e4>] (__driver_attach) from [<c0519e40>] (bus_for_each_dev+0x54/0x88)
-[<c0519e40>] (bus_for_each_dev) from [<c051ace4>] (bus_add_driver+0xec/0x1f8)
-[<c051ace4>] (bus_add_driver) from [<c051c45c>] (driver_register+0x78/0xf4)
-[<c051c45c>] (driver_register) from [<c0101b8c>] (do_one_initcall+0x3c/0x168)
+[<c02a9388>] (sysfs_add_link_to_group) from [<c048b580>] (iommu_device_link+0x3c/0x9c)
+[<c048b580>] (iommu_device_link) from [<c048d778>] (tegra_smmu_add_device+0x9c/0xa8)
+[<c048d778>] (tegra_smmu_add_device) from [<c0488f4c>] (add_iommu_group+0x2c/0x58)
+[<c0488f4c>] (add_iommu_group) from [<c0519efc>] (bus_for_each_dev+0x54/0x88)
+[<c0519efc>] (bus_for_each_dev) from [<c0489d78>] (bus_set_iommu+0x94/0xd8)
+[<c0489d78>] (bus_set_iommu) from [<c048e710>] (tegra_smmu_probe+0x17c/0x264)
+[<c048e710>] (tegra_smmu_probe) from [<c06d6dbc>] (tegra_mc_probe+0x308/0x51c)
+[<c06d6dbc>] (tegra_mc_probe) from [<c051d138>] (platform_drv_probe+0x50/0xb0)
+[<c051d138>] (platform_drv_probe) from [<c051b848>] (driver_probe_device+0x238/0x2e4)
+[<c051b848>] (driver_probe_device) from [<c051b9a0>] (__driver_attach+0xac/0xb0)
+[<c051b9a0>] (__driver_attach) from [<c0519efc>] (bus_for_each_dev+0x54/0x88)
+[<c0519efc>] (bus_for_each_dev) from [<c051ada0>] (bus_add_driver+0xec/0x1f8)
+[<c051ada0>] (bus_add_driver) from [<c051c518>] (driver_register+0x78/0xf4)
+[<c051c518>] (driver_register) from [<c0101b8c>] (do_one_initcall+0x3c/0x168)
 [<c0101b8c>] (do_one_initcall) from [<c0e00da8>] (kernel_init_freeable+0x10c/0x1d0)
-[<c0e00da8>] (kernel_init_freeable) from [<c097037c>] (kernel_init+0x8/0x114)
-[<c097037c>] (kernel_init) from [<c0107ce0>] (ret_from_fork+0x14/0x34)
+[<c0e00da8>] (kernel_init_freeable) from [<c09708fc>] (kernel_init+0x8/0x114)
+[<c09708fc>] (kernel_init) from [<c0107ce0>] (ret_from_fork+0x14/0x34)
 Code: e8bd4038 eafff194 e92d4070 e1a04002 (e5900018) 
----[ end trace b363b6899a994dce ]---
+---[ end trace 890e39d41b48be84 ]---
 Kernel panic - not syncing: Attempted to kill init! exitcode=0x0000000b
 
-CPU3: stopping
-CPU: 3 PID: 0 Comm: swapper/3 Tainted: G      D         4.13.0-rc6-next-20170824-196160-g9506597d #1
+CPU1: stopping
+CPU: 1 PID: 0 Comm: swapper/1 Tainted: G      D         4.13.0-rc6-next-20170825-196567-g7159188 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c01110d0>] (unwind_backtrace) from [<c010c094>] (show_stack+0x10/0x14)
-[<c010c094>] (show_stack) from [<c095cd70>] (dump_stack+0x84/0xa4)
-[<c095cd70>] (dump_stack) from [<c010f620>] (handle_IPI+0x18c/0x39c)
+[<c010c094>] (show_stack) from [<c095d2f0>] (dump_stack+0x84/0xa4)
+[<c095d2f0>] (dump_stack) from [<c010f620>] (handle_IPI+0x18c/0x39c)
 [<c010f620>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
-Exception stack(0xee8e1f70 to 0xee8e1fb8)
-1f60:                                     00000001 00000000 ee8e1fc8 c011a3e0
-1f80: c0f03ccc c0f03c68 c0e8c378 c0f03cd8 c0fbdb64 00000000 00000001 00000000
-1fa0: 2e146000 ee8e1fc0 c0108758 c010875c 60000013 ffffffff
+Exception stack(0xee8ddf70 to 0xee8ddfb8)
+df60:                                     00000001 00000000 ee8ddfc8 c011a3e0
+df80: c0f03ccc c0f03c68 c0e8c378 c0f03cd8 c0fbdaa4 00000000 00000001 00000000
+dfa0: 2e124000 ee8ddfc0 c0108758 c010875c 60000013 ffffffff
 [<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
 [<c010875c>] (arch_cpu_idle) from [<c0160c0c>] (do_idle+0x148/0x204)
 [<c0160c0c>] (do_idle) from [<c0160f50>] (cpu_startup_entry+0x18/0x1c)
 [<c0160f50>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
-CPU2: stopping
-CPU: 2 PID: 0 Comm: swapper/2 Tainted: G      D         4.13.0-rc6-next-20170824-196160-g9506597d #1
+CPU3: stopping
+CPU: 3 PID: 0 Comm: swapper/3 Tainted: G      D         4.13.0-rc6-next-20170825-196567-g7159188 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c01110d0>] (unwind_backtrace) from [<c010c094>] (show_stack+0x10/0x14)
-[<c010c094>] (show_stack) from [<c095cd70>] (dump_stack+0x84/0xa4)
-[<c095cd70>] (dump_stack) from [<c010f620>] (handle_IPI+0x18c/0x39c)
+[<c010c094>] (show_stack) from [<c095d2f0>] (dump_stack+0x84/0xa4)
+[<c095d2f0>] (dump_stack) from [<c010f620>] (handle_IPI+0x18c/0x39c)
 [<c010f620>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
-Exception stack(0xee8dff70 to 0xee8dffb8)
-ff60:                                     00000001 00000000 ee8dffc8 c011a3e0
-ff80: c0f03ccc c0f03c68 c0e8c378 c0f03cd8 c0fbdb64 00000000 00000001 00000000
-ffa0: 2e135000 ee8dffc0 c0108758 c010875c 60000013 ffffffff
+Exception stack(0xee8e1f70 to 0xee8e1fb8)
+1f60:                                     00000001 00000000 ee8e1fc8 c011a3e0
+1f80: c0f03ccc c0f03c68 c0e8c378 c0f03cd8 c0fbdaa4 00000000 00000001 00000000
+1fa0: 2e146000 ee8e1fc0 c0108758 c010875c 60000013 ffffffff
 [<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
 [<c010875c>] (arch_cpu_idle) from [<c0160c0c>] (do_idle+0x148/0x204)
 [<c0160c0c>] (do_idle) from [<c0160f50>] (cpu_startup_entry+0x18/0x1c)
 [<c0160f50>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
-CPU1: stopping
-CPU: 1 PID: 0 Comm: swapper/1 Tainted: G      D         4.13.0-rc6-next-20170824-196160-g9506597d #1
+CPU2: stopping
+CPU: 2 PID: 0 Comm: swapper/2 Tainted: G      D         4.13.0-rc6-next-20170825-196567-g7159188 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c01110d0>] (unwind_backtrace) from [<c010c094>] (show_stack+0x10/0x14)
-[<c010c094>] (show_stack) from [<c095cd70>] (dump_stack+0x84/0xa4)
-[<c095cd70>] (dump_stack) from [<c010f620>] (handle_IPI+0x18c/0x39c)
+[<c010c094>] (show_stack) from [<c095d2f0>] (dump_stack+0x84/0xa4)
+[<c095d2f0>] (dump_stack) from [<c010f620>] (handle_IPI+0x18c/0x39c)
 [<c010f620>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
-Exception stack(0xee8ddf70 to 0xee8ddfb8)
-df60:                                     00000001 00000000 ee8ddfc8 c011a3e0
-df80: c0f03ccc c0f03c68 c0e8c378 c0f03cd8 c0fbdb64 00000000 00000001 00000000
-dfa0: 2e124000 ee8ddfc0 c0108758 c010875c 60000013 ffffffff
+Exception stack(0xee8dff70 to 0xee8dffb8)
+ff60:                                     00000001 00000000 ee8dffc8 c011a3e0
+ff80: c0f03ccc c0f03c68 c0e8c378 c0f03cd8 c0fbdaa4 00000000 00000001 00000000
+ffa0: 2e135000 ee8dffc0 c0108758 c010875c 60000013 ffffffff
 [<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
 [<c010875c>] (arch_cpu_idle) from [<c0160c0c>] (do_idle+0x148/0x204)
 [<c0160c0c>] (do_idle) from [<c0160f50>] (cpu_startup_entry+0x18/0x1c)
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-08-28 14:08:24.649467390 -0700
+++ /tmp/foob	2017-08-28 14:08:24.653467405 -0700
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MMC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -35,7 +35,7 @@
 	 #################################################################
 	 #################################################################
 	 ###################################
-	 3.6 MiB/s
+	 3.9 MiB/s
 done
 Bytes transferred = 8135176 (7c2208 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
@@ -44,7 +44,7 @@
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.6 MiB/s
+	 3.8 MiB/s
 done
 Bytes transferred = 67173 (10665 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2017-08-28 14:08:24.661467435 -0700
+++ /tmp/foob	2017-08-28 14:08:24.661467435 -0700
@@ -28,25 +28,25 @@
 Load address: 0x82000000
 Loading: *#################################################################
 	 #################################################################
-	 ##################################T T ###############################
+	 #################################################################
 	 #################################################################
 	 #################################################################
 	 #################################################################
 	 ######################################
-	 516.6 KiB/s
+	 3.9 MiB/s
 done
-Bytes transferred = 6280856 (5fd698 hex)
+Bytes transferred = 6282712 (5fddd8 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3 MiB/s
+	 3.8 MiB/s
 done
 Bytes transferred = 67173 (10665 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5fd698 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5fddd8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013664
@@ -55,7 +55,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc6-next-20170824-196160-g9506597d (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Aug 28 13:33:59 PDT 2017
+Linux version 4.13.0-rc6-next-20170825-196567-g7159188 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Aug 28 14:00:18 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -65,7 +65,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0fb7e40, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0fb7d80, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -89,7 +89,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
-      .data : 0xc0f00000 - 0xc0fc2dd0   ( 780 kB)
+      .data : 0xc0f00000 - 0xc0fc2d10   ( 780 kB)
        .bss : 0xc0fcc5b0 - 0xc10176fc   ( 301 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
@@ -142,107 +142,107 @@
 [00000020] *pgd=00000000
 Internal error: Oops: 5 [#1] PREEMPT SMP ARM
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.13.0-rc6-next-20170824-196160-g9506597d #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.13.0-rc6-next-20170825-196567-g7159188 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 task: ee891740 task.stack: ee8a6000
 PC is at sysfs_add_link_to_group+0x8/0x48
 LR is at iommu_device_link+0x3c/0x9c
-pc : [<c02a956c>]    lr : [<c048b4c4>]    psr: 60000013
+pc : [<c02a9388>]    lr : [<c048b580>]    psr: 60000013
 sp : ee8a7d40  ip : ee8a7d70  fp : eeaa6210
 r10: 00000000  r9 : eea8f410  r8 : 00000060
 r7 : eea1ac18  r6 : eefc8188  r5 : eea1ac10  r4 : eea1ac18
-r3 : eea31c80  r2 : eea1ac18  r1 : c0bec2f8  r0 : 00000008
+r3 : eea31c80  r2 : eea1ac18  r1 : c0bec64c  r0 : 00000008
 Flags: nZCv  IRQs on  FIQs on  Mode SVC_32  ISA ARM  Segment none
 Control: 10c5387d  Table: 8000406a  DAC: 00000051
 Process swapper/0 (pid: 1, stack limit = 0xee8a6210)
 Stack: (0xee8a7d40 to 0xee8a8000)
-7d40: eeaa6bd0 eea1ac10 eefc8188 c048b4c4 eeaaa750 00000001 ee8a7d70 eefc8188
-7d60: eea1ac10 c048d6bc ee8a7d70 c043c460 eefe6f58 00000001 00000006 c043cbdc
-7d80: eeaaa600 c043cd64 c0489c68 a0000013 eeaaa580 c0238424 000000a5 c0489c68
-7da0: eea36538 ee8a7de0 eea36634 00000000 c0519c9c eea8f410 00000000 eea1ac10
-7dc0: c0a588cc c0488e64 00000000 c0488e90 00000000 00000000 ee8a7dfc c0519e40
-7de0: ee8c955c eea36534 c0f69188 00000000 eeaaa580 c0489cbc 60000013 c0a588cc
-7e00: 00000001 eeaa6b90 30000020 2800003f eea8f410 c048e654 ee8a7e24 eea8f410
-7e20: eefe808c eeaaa750 eefe71b4 00000013 00000000 c06d6de0 00000000 00000000
+7d40: eeaa6bd0 eea1ac10 eefc8188 c048b580 eeaaa750 00000001 ee8a7d70 eefc8188
+7d60: eea1ac10 c048d778 ee8a7d70 c043c494 eefe6f58 00000001 00000006 c043cc10
+7d80: eeaaa600 c043cd98 c0489d24 a0000013 eeaaa580 c023824c 000000a5 c0489d24
+7da0: eea36538 ee8a7de0 eea36634 00000000 c0519d58 eea8f410 00000000 eea1ac10
+7dc0: c0a588cc c0488f20 00000000 c0488f4c 00000000 00000000 ee8a7dfc c0519efc
+7de0: ee8c955c eea36534 c0f69188 00000000 eeaaa580 c0489d78 60000013 c0a588cc
+7e00: 00000001 eeaa6b90 30000020 2800003f eea8f410 c048e710 ee8a7e24 eea8f410
+7e20: eefe808c eeaaa750 eefe71b4 00000013 00000000 c06d6dbc 00000000 00000000
 7e40: 00000000 00000001 eea93468 eeaa6268 eea8f400 00000003 00000003 37131f00
-7e60: 00000000 ffffffed eea8f410 fffffdfb c0f8e39c 00000000 00000000 c0f8e39c
-7e80: 00000000 c051d07c c051d02c eea8f410 c1009250 c1009254 00000000 c051b78c
-7ea0: eea8f410 c0f8e39c eea8f444 00000000 00000111 c0e6a83c c0e005a4 c051b8e4
-7ec0: 00000000 c0f8e39c c051b838 c0519e40 ee8c955c eea926b4 c0f8e39c eeaa6280
-7ee0: c0f69188 c051ace4 c0c483f0 c0f8e39c 00000003 c0f8e39c 00000003 c06d6ac8
-7f00: 00000000 c051c45c c051ccd4 c0e82f04 00000003 c0101b8c 00000065 c0cb9104
-7f20: 00000100 efffcc05 c0cb9104 00000111 efffcc13 c013e478 c0cb7bb0 00000000
+7e60: 00000000 ffffffed eea8f410 fffffdfb c0f8e29c 00000000 00000000 c0f8e29c
+7e80: 00000000 c051d138 c051d0e8 eea8f410 c1009250 c1009254 00000000 c051b848
+7ea0: eea8f410 c0f8e29c eea8f444 00000000 00000111 c0e6a83c c0e005a4 c051b9a0
+7ec0: 00000000 c0f8e29c c051b8f4 c0519efc ee8c955c eea926b4 c0f8e29c eeaa6280
+7ee0: c0f69188 c051ada0 c0c48780 c0f8e29c 00000003 c0f8e29c 00000003 c06d6aa4
+7f00: 00000000 c051c518 c051cd90 c0e82f04 00000003 c0101b8c 00000000 c0cb9108
+7f20: 00000100 efffcc05 c0cb9108 00000111 efffcc13 c013e478 c0cb7bb4 00000000
 7f40: 00000003 00000003 efffcc05 efffcc0a c0f0dcd0 efffcb80 00000003 c0e6a820
 7f60: c0e82f04 00000003 c0e6a824 c0fcc5c0 00000111 c0e00da8 00000003 00000003
-7f80: 00000000 c0e005a4 00000000 c0970374 00000000 00000000 00000000 00000000
-7fa0: 00000000 c097037c 00000000 c0107ce0 00000000 00000000 00000000 00000000
+7f80: 00000000 c0e005a4 00000000 c09708f4 00000000 00000000 00000000 00000000
+7fa0: 00000000 c09708fc 00000000 c0107ce0 00000000 00000000 00000000 00000000
 7fc0: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
 7fe0: 00000000 00000000 00000000 00000000 00000013 00000000 ffffffff ffffffff
-[<c02a956c>] (sysfs_add_link_to_group) from [<c048b4c4>] (iommu_device_link+0x3c/0x9c)
-[<c048b4c4>] (iommu_device_link) from [<c048d6bc>] (tegra_smmu_add_device+0x9c/0xa8)
-[<c048d6bc>] (tegra_smmu_add_device) from [<c0488e90>] (add_iommu_group+0x2c/0x58)
-[<c0488e90>] (add_iommu_group) from [<c0519e40>] (bus_for_each_dev+0x54/0x88)
-[<c0519e40>] (bus_for_each_dev) from [<c0489cbc>] (bus_set_iommu+0x94/0xd8)
-[<c0489cbc>] (bus_set_iommu) from [<c048e654>] (tegra_smmu_probe+0x17c/0x264)
-[<c048e654>] (tegra_smmu_probe) from [<c06d6de0>] (tegra_mc_probe+0x308/0x51c)
-[<c06d6de0>] (tegra_mc_probe) from [<c051d07c>] (platform_drv_probe+0x50/0xb0)
-[<c051d07c>] (platform_drv_probe) from [<c051b78c>] (driver_probe_device+0x238/0x2e4)
-[<c051b78c>] (driver_probe_device) from [<c051b8e4>] (__driver_attach+0xac/0xb0)
-[<c051b8e4>] (__driver_attach) from [<c0519e40>] (bus_for_each_dev+0x54/0x88)
-[<c0519e40>] (bus_for_each_dev) from [<c051ace4>] (bus_add_driver+0xec/0x1f8)
-[<c051ace4>] (bus_add_driver) from [<c051c45c>] (driver_register+0x78/0xf4)
-[<c051c45c>] (driver_register) from [<c0101b8c>] (do_one_initcall+0x3c/0x168)
+[<c02a9388>] (sysfs_add_link_to_group) from [<c048b580>] (iommu_device_link+0x3c/0x9c)
+[<c048b580>] (iommu_device_link) from [<c048d778>] (tegra_smmu_add_device+0x9c/0xa8)
+[<c048d778>] (tegra_smmu_add_device) from [<c0488f4c>] (add_iommu_group+0x2c/0x58)
+[<c0488f4c>] (add_iommu_group) from [<c0519efc>] (bus_for_each_dev+0x54/0x88)
+[<c0519efc>] (bus_for_each_dev) from [<c0489d78>] (bus_set_iommu+0x94/0xd8)
+[<c0489d78>] (bus_set_iommu) from [<c048e710>] (tegra_smmu_probe+0x17c/0x264)
+[<c048e710>] (tegra_smmu_probe) from [<c06d6dbc>] (tegra_mc_probe+0x308/0x51c)
+[<c06d6dbc>] (tegra_mc_probe) from [<c051d138>] (platform_drv_probe+0x50/0xb0)
+[<c051d138>] (platform_drv_probe) from [<c051b848>] (driver_probe_device+0x238/0x2e4)
+[<c051b848>] (driver_probe_device) from [<c051b9a0>] (__driver_attach+0xac/0xb0)
+[<c051b9a0>] (__driver_attach) from [<c0519efc>] (bus_for_each_dev+0x54/0x88)
+[<c0519efc>] (bus_for_each_dev) from [<c051ada0>] (bus_add_driver+0xec/0x1f8)
+[<c051ada0>] (bus_add_driver) from [<c051c518>] (driver_register+0x78/0xf4)
+[<c051c518>] (driver_register) from [<c0101b8c>] (do_one_initcall+0x3c/0x168)
 [<c0101b8c>] (do_one_initcall) from [<c0e00da8>] (kernel_init_freeable+0x10c/0x1d0)
-[<c0e00da8>] (kernel_init_freeable) from [<c097037c>] (kernel_init+0x8/0x114)
-[<c097037c>] (kernel_init) from [<c0107ce0>] (ret_from_fork+0x14/0x34)
+[<c0e00da8>] (kernel_init_freeable) from [<c09708fc>] (kernel_init+0x8/0x114)
+[<c09708fc>] (kernel_init) from [<c0107ce0>] (ret_from_fork+0x14/0x34)
 Code: e8bd4038 eafff194 e92d4070 e1a04002 (e5900018) 
----[ end trace 5a418a077247f17d ]---
+---[ end trace 103838b4c6580006 ]---
 Kernel panic - not syncing: Attempted to kill init! exitcode=0x0000000b
 
 CPU2: stopping
-CPU: 2 PID: 0 Comm: swapper/2 Tainted: G      D         4.13.0-rc6-next-20170824-196160-g9506597d #1
+CPU: 2 PID: 0 Comm: swapper/2 Tainted: G      D         4.13.0-rc6-next-20170825-196567-g7159188 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c01110d0>] (unwind_backtrace) from [<c010c094>] (show_stack+0x10/0x14)
-[<c010c094>] (show_stack) from [<c095cd70>] (dump_stack+0x84/0xa4)
-[<c095cd70>] (dump_stack) from [<c010f620>] (handle_IPI+0x18c/0x39c)
+[<c010c094>] (show_stack) from [<c095d2f0>] (dump_stack+0x84/0xa4)
+[<c095d2f0>] (dump_stack) from [<c010f620>] (handle_IPI+0x18c/0x39c)
 [<c010f620>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
 Exception stack(0xee8e1f70 to 0xee8e1fb8)
 1f60:                                     00000001 00000000 ee8e1fc8 c011a3e0
-1f80: c0f03ccc c0f03c68 c0e8c378 c0f03cd8 c0fbdb64 00000000 00000001 00000000
+1f80: c0f03ccc c0f03c68 c0e8c378 c0f03cd8 c0fbdaa4 00000000 00000001 00000000
 1fa0: 2e11f000 ee8e1fc0 c0108758 c010875c 60000013 ffffffff
 [<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
 [<c010875c>] (arch_cpu_idle) from [<c0160c0c>] (do_idle+0x148/0x204)
 [<c0160c0c>] (do_idle) from [<c0160f50>] (cpu_startup_entry+0x18/0x1c)
 [<c0160f50>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
-CPU3: stopping
-CPU: 3 PID: 0 Comm: swapper/3 Tainted: G      D         4.13.0-rc6-next-20170824-196160-g9506597d #1
+CPU1: stopping
+CPU: 1 PID: 0 Comm: swapper/1 Tainted: G      D         4.13.0-rc6-next-20170825-196567-g7159188 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c01110d0>] (unwind_backtrace) from [<c010c094>] (show_stack+0x10/0x14)
-[<c010c094>] (show_stack) from [<c095cd70>] (dump_stack+0x84/0xa4)
-[<c095cd70>] (dump_stack) from [<c010f620>] (handle_IPI+0x18c/0x39c)
+[<c010c094>] (show_stack) from [<c095d2f0>] (dump_stack+0x84/0xa4)
+[<c095d2f0>] (dump_stack) from [<c010f620>] (handle_IPI+0x18c/0x39c)
 [<c010f620>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
-Exception stack(0xee8e3f70 to 0xee8e3fb8)
-3f60:                                     00000001 00000000 ee8e3fc8 c011a3e0
-3f80: c0f03ccc c0f03c68 c0e8c378 c0f03cd8 c0fbdb64 00000000 00000001 00000000
-3fa0: 2e130000 ee8e3fc0 c0108758 c010875c 60000013 ffffffff
+Exception stack(0xee8dff70 to 0xee8dffb8)
+ff60:                                     00000001 00000000 ee8dffc8 c011a3e0
+ff80: c0f03ccc c0f03c68 c0e8c378 c0f03cd8 c0fbdaa4 00000000 00000001 00000000
+ffa0: 2e10e000 ee8dffc0 c0108758 c010875c 60000013 ffffffff
 [<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
 [<c010875c>] (arch_cpu_idle) from [<c0160c0c>] (do_idle+0x148/0x204)
 [<c0160c0c>] (do_idle) from [<c0160f50>] (cpu_startup_entry+0x18/0x1c)
 [<c0160f50>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
-CPU1: stopping
-CPU: 1 PID: 0 Comm: swapper/1 Tainted: G      D         4.13.0-rc6-next-20170824-196160-g9506597d #1
+CPU3: stopping
+CPU: 3 PID: 0 Comm: swapper/3 Tainted: G      D         4.13.0-rc6-next-20170825-196567-g7159188 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c01110d0>] (unwind_backtrace) from [<c010c094>] (show_stack+0x10/0x14)
-[<c010c094>] (show_stack) from [<c095cd70>] (dump_stack+0x84/0xa4)
-[<c095cd70>] (dump_stack) from [<c010f620>] (handle_IPI+0x18c/0x39c)
+[<c010c094>] (show_stack) from [<c095d2f0>] (dump_stack+0x84/0xa4)
+[<c095d2f0>] (dump_stack) from [<c010f620>] (handle_IPI+0x18c/0x39c)
 [<c010f620>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
-Exception stack(0xee8dff70 to 0xee8dffb8)
-ff60:                                     00000001 00000000 ee8dffc8 c011a3e0
-ff80: c0f03ccc c0f03c68 c0e8c378 c0f03cd8 c0fbdb64 00000000 00000001 00000000
-ffa0: 2e10e000 ee8dffc0 c0108758 c010875c 60000013 ffffffff
+Exception stack(0xee8e3f70 to 0xee8e3fb8)
+3f60:                                     00000001 00000000 ee8e3fc8 c011a3e0
+3f80: c0f03ccc c0f03c68 c0e8c378 c0f03cd8 c0fbdaa4 00000000 00000001 00000000
+3fa0: 2e130000 ee8e3fc0 c0108758 c010875c 60000013 ffffffff
 [<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
 [<c010875c>] (arch_cpu_idle) from [<c0160c0c>] (do_idle+0x148/0x204)
 [<c0160c0c>] (do_idle) from [<c0160f50>] (cpu_startup_entry+0x18/0x1c)
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-08-28 14:08:24.665467449 -0700
+++ /tmp/foob	2017-08-28 14:08:24.669467464 -0700
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -43,7 +43,7 @@
 	 #################################################################
 	 #################################################################
 	 ###################################
-	 2.8 MiB/s
+	 3.5 MiB/s
 done
 Bytes transferred = 8135176 (7c2208 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
@@ -52,7 +52,7 @@
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 2.7 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 68203 (10a6b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2017-08-28 14:08:24.673467479 -0700
+++ /tmp/foob	2017-08-28 14:08:24.677467494 -0700
@@ -41,20 +41,20 @@
 	 #################################################################
 	 #################################################################
 	 ######################################
-	 2.9 MiB/s
+	 3.5 MiB/s
 done
-Bytes transferred = 6280856 (5fd698 hex)
+Bytes transferred = 6282712 (5fddd8 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 2.6 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 68203 (10a6b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5fd698 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5fddd8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a6a
@@ -63,7 +63,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc6-next-20170824-196160-g9506597d (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Aug 28 13:33:59 PDT 2017
+Linux version 4.13.0-rc6-next-20170825-196567-g7159188 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Aug 28 14:00:18 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -74,7 +74,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c0fb7e40, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0fb7d80, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -98,7 +98,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
-      .data : 0xc0f00000 - 0xc0fc2dd0   ( 780 kB)
+      .data : 0xc0f00000 - 0xc0fc2d10   ( 780 kB)
        .bss : 0xc0fcc5b0 - 0xc10176fc   ( 301 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
@@ -151,107 +151,107 @@
 [00000020] *pgd=00000000
 Internal error: Oops: 5 [#1] PREEMPT SMP ARM
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.13.0-rc6-next-20170824-196160-g9506597d #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.13.0-rc6-next-20170825-196567-g7159188 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 task: ee891740 task.stack: ee8a6000
 PC is at sysfs_add_link_to_group+0x8/0x48
 LR is at iommu_device_link+0x3c/0x9c
-pc : [<c02a956c>]    lr : [<c048b4c4>]    psr: 60000013
+pc : [<c02a9388>]    lr : [<c048b580>]    psr: 60000013
 sp : ee8a7d40  ip : ee8a7d70  fp : 00000000
-r10: c0f8e39c  r9 : eea8f410  r8 : 00000050
+r10: c0f8e29c  r9 : eea8f410  r8 : 00000050
 r7 : eea1ce18  r6 : eefc770c  r5 : eea1ce10  r4 : eea1ce18
-r3 : eea36ac0  r2 : eea1ce18  r1 : c0bec2f8  r0 : 00000008
+r3 : eea36ac0  r2 : eea1ce18  r1 : c0bec64c  r0 : 00000008
 Flags: nZCv  IRQs on  FIQs on  Mode SVC_32  ISA ARM  Segment none
 Control: 10c5387d  Table: 8000406a  DAC: 00000051
 Process swapper/0 (pid: 1, stack limit = 0xee8a6210)
 Stack: (0xee8a7d40 to 0xee8a8000)
-7d40: eeaa63d0 eea1ce10 eefc770c c048b4c4 eeaa3a50 00000001 ee8a7d70 eefc770c
-7d60: eea1ce10 c048d6bc ee8a7d70 6d656d2e eefe70d8 00000001 00000006 c0007265
-7d80: eeaa3900 c043cd64 c0489c68 a0000013 eeaa3880 c0238424 000000a5 c0489c68
-7da0: eea185b8 ee8a7de0 ee9f4bb4 00000000 c0519c9c eea8f410 c0f8e39c eea1ce10
-7dc0: c0a588cc c0488e64 00000000 c0488e90 00000000 00000000 ee8a7dfc c0519e40
-7de0: ee8c955c eea185b4 c0f69188 00000000 eeaa3880 c0489cbc 60000013 c0a588cc
-7e00: 00000001 eeaa6390 30000020 2800003f eea8f410 c048e654 ee8a7e24 ee8a7e30
-7e20: c0c48284 eeaa3a50 00000000 00000000 00000043 c06d6de0 00000000 00000000
+7d40: eeaa63d0 eea1ce10 eefc770c c048b580 eeaa3a50 00000001 ee8a7d70 eefc770c
+7d60: eea1ce10 c048d778 ee8a7d70 6d656d2e eefe70d8 00000001 00000006 c0007265
+7d80: eeaa3900 c043cd98 c0489d24 a0000013 eeaa3880 c023824c 000000a5 c0489d24
+7da0: eea185b8 ee8a7de0 ee9f4bb4 00000000 c0519d58 eea8f410 c0f8e29c eea1ce10
+7dc0: c0a588cc c0488f20 00000000 c0488f4c 00000000 00000000 ee8a7dfc c0519efc
+7de0: ee8c955c eea185b4 c0f69188 00000000 eeaa3880 c0489d78 60000013 c0a588cc
+7e00: 00000001 eeaa6390 30000020 2800003f eea8f410 c048e710 ee8a7e24 ee8a7e30
+7e20: c0c48614 eeaa3a50 00000000 00000000 00000043 c06d6dbc 00000000 00000000
 7e40: 00000000 00000001 eea94a18 eeaa9e78 eea8f400 00000000 00000001 eeaab698
-7e60: 00000000 ffffffed eea8f410 fffffdfb c0f8e39c 00000000 00000000 c0f8e39c
-7e80: 00000000 c051d07c c051d02c eea8f410 c1009250 c1009254 00000000 c051b78c
-7ea0: eea8f410 c0f8e39c eea8f444 00000000 00000111 c0e6a83c c0e005a4 c051b8e4
-7ec0: 00000000 c0f8e39c c051b838 c0519e40 ee8c955c eea936b4 c0f8e39c eeaa6400
-7ee0: c0f69188 c051ace4 c0c483f0 c0f8e39c 00000003 c0f8e39c 00000003 c06d6ac8
-7f00: 00000000 c051c45c c051ccd4 c0e82f04 00000003 c0101b8c 00000000 c0cb9104
-7f20: 00000100 efffcc45 c0cb9104 00000111 efffcc53 c013e478 c0cb7bb0 00000000
+7e60: 00000000 ffffffed eea8f410 fffffdfb c0f8e29c 00000000 00000000 c0f8e29c
+7e80: 00000000 c051d138 c051d0e8 eea8f410 c1009250 c1009254 00000000 c051b848
+7ea0: eea8f410 c0f8e29c eea8f444 00000000 00000111 c0e6a83c c0e005a4 c051b9a0
+7ec0: 00000000 c0f8e29c c051b8f4 c0519efc ee8c955c eea936b4 c0f8e29c eeaa6400
+7ee0: c0f69188 c051ada0 c0c48780 c0f8e29c 00000003 c0f8e29c 00000003 c06d6aa4
+7f00: 00000000 c051c518 c051cd90 c0e82f04 00000003 c0101b8c 00000000 c0cb9108
+7f20: 00000100 efffcc45 c0cb9108 00000111 efffcc53 c013e478 c0cb7bb4 00000000
 7f40: 00000003 00000003 efffcc45 efffcc4a c0f0dcd0 efffcbc0 00000003 c0e6a820
 7f60: c0e82f04 00000003 c0e6a824 c0fcc5c0 00000111 c0e00da8 00000003 00000003
-7f80: 00000000 c0e005a4 00000000 c0970374 00000000 00000000 00000000 00000000
-7fa0: 00000000 c097037c 00000000 c0107ce0 00000000 00000000 00000000 00000000
+7f80: 00000000 c0e005a4 00000000 c09708f4 00000000 00000000 00000000 00000000
+7fa0: 00000000 c09708fc 00000000 c0107ce0 00000000 00000000 00000000 00000000
 7fc0: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
 7fe0: 00000000 00000000 00000000 00000000 00000013 00000000 00000000 00000000
-[<c02a956c>] (sysfs_add_link_to_group) from [<c048b4c4>] (iommu_device_link+0x3c/0x9c)
-[<c048b4c4>] (iommu_device_link) from [<c048d6bc>] (tegra_smmu_add_device+0x9c/0xa8)
-[<c048d6bc>] (tegra_smmu_add_device) from [<c0488e90>] (add_iommu_group+0x2c/0x58)
-[<c0488e90>] (add_iommu_group) from [<c0519e40>] (bus_for_each_dev+0x54/0x88)
-[<c0519e40>] (bus_for_each_dev) from [<c0489cbc>] (bus_set_iommu+0x94/0xd8)
-[<c0489cbc>] (bus_set_iommu) from [<c048e654>] (tegra_smmu_probe+0x17c/0x264)
-[<c048e654>] (tegra_smmu_probe) from [<c06d6de0>] (tegra_mc_probe+0x308/0x51c)
-[<c06d6de0>] (tegra_mc_probe) from [<c051d07c>] (platform_drv_probe+0x50/0xb0)
-[<c051d07c>] (platform_drv_probe) from [<c051b78c>] (driver_probe_device+0x238/0x2e4)
-[<c051b78c>] (driver_probe_device) from [<c051b8e4>] (__driver_attach+0xac/0xb0)
-[<c051b8e4>] (__driver_attach) from [<c0519e40>] (bus_for_each_dev+0x54/0x88)
-[<c0519e40>] (bus_for_each_dev) from [<c051ace4>] (bus_add_driver+0xec/0x1f8)
-[<c051ace4>] (bus_add_driver) from [<c051c45c>] (driver_register+0x78/0xf4)
-[<c051c45c>] (driver_register) from [<c0101b8c>] (do_one_initcall+0x3c/0x168)
+[<c02a9388>] (sysfs_add_link_to_group) from [<c048b580>] (iommu_device_link+0x3c/0x9c)
+[<c048b580>] (iommu_device_link) from [<c048d778>] (tegra_smmu_add_device+0x9c/0xa8)
+[<c048d778>] (tegra_smmu_add_device) from [<c0488f4c>] (add_iommu_group+0x2c/0x58)
+[<c0488f4c>] (add_iommu_group) from [<c0519efc>] (bus_for_each_dev+0x54/0x88)
+[<c0519efc>] (bus_for_each_dev) from [<c0489d78>] (bus_set_iommu+0x94/0xd8)
+[<c0489d78>] (bus_set_iommu) from [<c048e710>] (tegra_smmu_probe+0x17c/0x264)
+[<c048e710>] (tegra_smmu_probe) from [<c06d6dbc>] (tegra_mc_probe+0x308/0x51c)
+[<c06d6dbc>] (tegra_mc_probe) from [<c051d138>] (platform_drv_probe+0x50/0xb0)
+[<c051d138>] (platform_drv_probe) from [<c051b848>] (driver_probe_device+0x238/0x2e4)
+[<c051b848>] (driver_probe_device) from [<c051b9a0>] (__driver_attach+0xac/0xb0)
+[<c051b9a0>] (__driver_attach) from [<c0519efc>] (bus_for_each_dev+0x54/0x88)
+[<c0519efc>] (bus_for_each_dev) from [<c051ada0>] (bus_add_driver+0xec/0x1f8)
+[<c051ada0>] (bus_add_driver) from [<c051c518>] (driver_register+0x78/0xf4)
+[<c051c518>] (driver_register) from [<c0101b8c>] (do_one_initcall+0x3c/0x168)
 [<c0101b8c>] (do_one_initcall) from [<c0e00da8>] (kernel_init_freeable+0x10c/0x1d0)
-[<c0e00da8>] (kernel_init_freeable) from [<c097037c>] (kernel_init+0x8/0x114)
-[<c097037c>] (kernel_init) from [<c0107ce0>] (ret_from_fork+0x14/0x34)
+[<c0e00da8>] (kernel_init_freeable) from [<c09708fc>] (kernel_init+0x8/0x114)
+[<c09708fc>] (kernel_init) from [<c0107ce0>] (ret_from_fork+0x14/0x34)
 Code: e8bd4038 eafff194 e92d4070 e1a04002 (e5900018) 
----[ end trace 0a66eb2d25cc77b1 ]---
+---[ end trace 7aeb18944fca766f ]---
 Kernel panic - not syncing: Attempted to kill init! exitcode=0x0000000b
 
-CPU2: stopping
-CPU: 2 PID: 0 Comm: swapper/2 Tainted: G      D         4.13.0-rc6-next-20170824-196160-g9506597d #1
+CPU3: stopping
+CPU: 3 PID: 0 Comm: swapper/3 Tainted: G      D         4.13.0-rc6-next-20170825-196567-g7159188 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c01110d0>] (unwind_backtrace) from [<c010c094>] (show_stack+0x10/0x14)
-[<c010c094>] (show_stack) from [<c095cd70>] (dump_stack+0x84/0xa4)
-[<c095cd70>] (dump_stack) from [<c010f620>] (handle_IPI+0x18c/0x39c)
+[<c010c094>] (show_stack) from [<c095d2f0>] (dump_stack+0x84/0xa4)
+[<c095d2f0>] (dump_stack) from [<c010f620>] (handle_IPI+0x18c/0x39c)
 [<c010f620>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
-Exception stack(0xee8e1f70 to 0xee8e1fb8)
-1f60:                                     00000001 00000000 ee8e1fc8 c011a3e0
-1f80: c0f03ccc c0f03c68 c0e8c378 c0f03cd8 c0fbdb64 00000000 00000001 00000000
-1fa0: 2e11e000 ee8e1fc0 c0108758 c010875c 60000013 ffffffff
+Exception stack(0xee8e3f70 to 0xee8e3fb8)
+3f60:                                     00000001 00000000 ee8e3fc8 c011a3e0
+3f80: c0f03ccc c0f03c68 c0e8c378 c0f03cd8 c0fbdaa4 00000000 00000001 00000000
+3fa0: 2e12f000 ee8e3fc0 c0108758 c010875c 60000013 ffffffff
 [<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
 [<c010875c>] (arch_cpu_idle) from [<c0160c0c>] (do_idle+0x148/0x204)
 [<c0160c0c>] (do_idle) from [<c0160f50>] (cpu_startup_entry+0x18/0x1c)
 [<c0160f50>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
-CPU1: stopping
-CPU: 1 PID: 0 Comm: swapper/1 Tainted: G      D         4.13.0-rc6-next-20170824-196160-g9506597d #1
+CPU2: stopping
+CPU: 2 PID: 0 Comm: swapper/2 Tainted: G      D         4.13.0-rc6-next-20170825-196567-g7159188 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c01110d0>] (unwind_backtrace) from [<c010c094>] (show_stack+0x10/0x14)
-[<c010c094>] (show_stack) from [<c095cd70>] (dump_stack+0x84/0xa4)
-[<c095cd70>] (dump_stack) from [<c010f620>] (handle_IPI+0x18c/0x39c)
+[<c010c094>] (show_stack) from [<c095d2f0>] (dump_stack+0x84/0xa4)
+[<c095d2f0>] (dump_stack) from [<c010f620>] (handle_IPI+0x18c/0x39c)
 [<c010f620>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
-Exception stack(0xee8dff70 to 0xee8dffb8)
-ff60:                                     00000001 00000000 ee8dffc8 c011a3e0
-ff80: c0f03ccc c0f03c68 c0e8c378 c0f03cd8 c0fbdb64 00000000 00000001 00000000
-ffa0: 2e10d000 ee8dffc0 c0108758 c010875c 60000013 ffffffff
+Exception stack(0xee8e1f70 to 0xee8e1fb8)
+1f60:                                     00000001 00000000 ee8e1fc8 c011a3e0
+1f80: c0f03ccc c0f03c68 c0e8c378 c0f03cd8 c0fbdaa4 00000000 00000001 00000000
+1fa0: 2e11e000 ee8e1fc0 c0108758 c010875c 60000013 ffffffff
 [<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
 [<c010875c>] (arch_cpu_idle) from [<c0160c0c>] (do_idle+0x148/0x204)
 [<c0160c0c>] (do_idle) from [<c0160f50>] (cpu_startup_entry+0x18/0x1c)
 [<c0160f50>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
-CPU3: stopping
-CPU: 3 PID: 0 Comm: swapper/3 Tainted: G      D         4.13.0-rc6-next-20170824-196160-g9506597d #1
+CPU1: stopping
+CPU: 1 PID: 0 Comm: swapper/1 Tainted: G      D         4.13.0-rc6-next-20170825-196567-g7159188 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c01110d0>] (unwind_backtrace) from [<c010c094>] (show_stack+0x10/0x14)
-[<c010c094>] (show_stack) from [<c095cd70>] (dump_stack+0x84/0xa4)
-[<c095cd70>] (dump_stack) from [<c010f620>] (handle_IPI+0x18c/0x39c)
+[<c010c094>] (show_stack) from [<c095d2f0>] (dump_stack+0x84/0xa4)
+[<c095d2f0>] (dump_stack) from [<c010f620>] (handle_IPI+0x18c/0x39c)
 [<c010f620>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
-Exception stack(0xee8e3f70 to 0xee8e3fb8)
-3f60:                                     00000001 00000000 ee8e3fc8 c011a3e0
-3f80: c0f03ccc c0f03c68 c0e8c378 c0f03cd8 c0fbdb64 00000000 00000001 00000000
-3fa0: 2e12f000 ee8e3fc0 c0108758 c010875c 60000013 ffffffff
+Exception stack(0xee8dff70 to 0xee8dffb8)
+ff60:                                     00000001 00000000 ee8dffc8 c011a3e0
+ff80: c0f03ccc c0f03c68 c0e8c378 c0f03cd8 c0fbdaa4 00000000 00000001 00000000
+ffa0: 2e10d000 ee8dffc0 c0108758 c010875c 60000013 ffffffff
 [<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
 [<c010875c>] (arch_cpu_idle) from [<c0160c0c>] (do_idle+0x148/0x204)
 [<c0160c0c>] (do_idle) from [<c0160f50>] (cpu_startup_entry+0x18/0x1c)
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2017-08-28 14:08:24.681467508 -0700
+++ /tmp/foob	2017-08-28 14:08:24.681467508 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1801997 us
+Bootloader-Cpu Init at (time stamp): 1800961 us
 
 Pinmux changes applied in kernel way
 
@@ -62,22 +62,22 @@
 Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1801971  us
-Bootloader (AosInit) time: 734251 us
-Bootloader Main Init time: 180095 us
-BlockDevMgrInit time: 157521 us
+Miscellaneous avp time: 1800935  us
+Bootloader (AosInit) time: 734255 us
+Bootloader Main Init time: 180083 us
+BlockDevMgrInit time: 157532 us
 PartMgrInit time: 8 us
 StorMgrInit time: 30 us
 FsMgrInit time: 2 us
-Partition Table load: 76663 us 
-Display init time: 574798 us
+Partition Table load: 76695 us 
+Display init time: 574803 us
 Kernel read time: 1345 msec
-Load Kernel Size:   17825792 Bytes
-Bl Kernel ReadSpeed: 13253000 Bytes/sec
-Pre-Kernel Jump time: 3839312 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 8806452 us
+Load Kernel Size:   17827840 Bytes
+Bl Kernel ReadSpeed: 13254000 Bytes/sec
+Pre-Kernel Jump time: 3840013 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 8806160 us
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc6-next-20170824-196160-g9506597d (jonathanh@sellafield) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Mon Aug 28 13:37:21 PDT 2017
+Linux version 4.13.0-rc6-next-20170825-314808-g7159188 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Mon Aug 28 14:04:57 PDT 2017
 Boot CPU: AArch64 Processor [4e0f0000]
 Machine model: NVIDIA Tegra132 Norrin
 Memory limited to 1734MB
@@ -102,15 +102,15 @@
 Policy zone: DMA
 Kernel command line: tegraid=13.1.3.0.0 mem=1734M@2048M memtype=255 vpr=151M@3817M ddr_die=2048M@2048M section=256M commchip_id=0 usb_port_owner_info=0 lane_owner_info=0 emc_max_dvfs=0 touch_id=0@0 video=tegrafb no_console_suspend=1 console=ttyS0,115200n8 debug_uartport=lsport,0 earlyprintk=uart8250-32bit,0x70006000 usbcore.old_scheme_first=1 lp0_vec=0@0x0 tegra_fbmem=32899072@0xad012000 core_edp_mv=1150 core_edp_ma=4000 pmuboard=0x0176:0x0001:0x01:0x43:0x00 displayboard=0x0162:0x0001:0x00:0x42:0x00 power_supply=Adapter audio_codec=rt5640 board_info=0x0176:0x0001:0x01:0x43:0x00 tegraboot=sdmmc gpt gpt_sector=192511 modem_id=0 android.kerneltype=normal
 PID hash table entries: 4096 (order: 3, 32768 bytes)
-Memory: 1708200K/1775616K available (9212K kernel code, 1142K rwdata, 4552K rodata, 1152K init, 402K bss, 51032K reserved, 16384K cma-reserved)
+Memory: 1708196K/1775616K available (9212K kernel code, 1144K rwdata, 4544K rodata, 1152K init, 404K bss, 51036K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008980000   (  9216 KB)
     .rodata : 0xffff000008980000 - 0xffff000008e00000   (  4608 KB)
       .init : 0xffff000008e00000 - 0xffff000008f20000   (  1152 KB)
-      .data : 0xffff000008f20000 - 0xffff00000903da00   (  1143 KB)
-       .bss : 0xffff00000903da00 - 0xffff0000090a23f0   (   403 KB)
+      .data : 0xffff000008f20000 - 0xffff00000903e200   (  1145 KB)
+       .bss : 0xffff00000903e200 - 0xffff0000090a33f0   (   405 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -271,15 +271,14 @@
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 +VDD_LED: supplied by +VDD_MUX
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 +5V_USB_HS: supplied by +5V_SYS
 +5V_USB_SS: supplied by +5V_SYS
 +5V_HDMI_CON: supplied by +5V_SYS
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-sdhci-tegra 700b0400.sdhci: Got CD GPIO
 hctosys: unable to open rtc device (rtc0)
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +VDD_LED: disabling
@@ -290,6 +289,7 @@
 ALSA device list:
   No soundcards found.
 Freeing unused kernel memory: 1152K
+sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 
 System initialization...
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-08-28 14:08:24.689467537 -0700
+++ /tmp/foob	2017-08-28 14:08:24.689467537 -0700
@@ -48,7 +48,7 @@
 	 #################################################################
 	 #################################################################
 	 ###################################
-	 3.7 MiB/s
+	 4 MiB/s
 done
 Bytes transferred = 8135176 (7c2208 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
@@ -57,7 +57,7 @@
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 2.9 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21454 (53ce hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
@@ -69,7 +69,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc6-next-20170824-196160-g9506597d (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Mon Aug 28 13:33:59 PDT 2017
+Linux version 4.13.0-rc6-next-20170825-196567-g7159188 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Mon Aug 28 14:00:27 PDT 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -79,7 +79,7 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0x3c000000
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c158ab80, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c158a880, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -93,7 +93,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 954328K/1048576K available (10240K kernel code, 1611K rwdata, 4480K rodata, 2048K init, 405K bss, 28712K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 954328K/1048576K available (10240K kernel code, 1610K rwdata, 4476K rodata, 2048K init, 405K bss, 28712K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -103,7 +103,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1592e90   (1612 kB)
+      .data : 0xc1400000 - 0xc1592b90   (1611 kB)
        .bss : 0xc159bc20 - 0xc16013fc   ( 406 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
@@ -342,11 +342,11 @@
 can: raw protocol (rev 20170425)
 can: broadcast manager protocol (rev 20170425 t)
 can: netlink gateway (rev 20170425) max_hops=1
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
 Key type dns_resolver registered
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2017-08-28 14:08:24.697467567 -0700
+++ /tmp/foob	2017-08-28 14:08:24.697467567 -0700
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -46,20 +46,20 @@
 	 #################################################################
 	 #################################################################
 	 ######################################
-	 3.7 MiB/s
+	 3.9 MiB/s
 done
-Bytes transferred = 6280856 (5fd698 hex)
+Bytes transferred = 6282712 (5fddd8 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 2.6 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21454 (53ce hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x5fd698 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x5fddd8 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103cd
@@ -68,7 +68,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc6-next-20170824-196160-g9506597d (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Aug 28 13:33:59 PDT 2017
+Linux version 4.13.0-rc6-next-20170825-196567-g7159188 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Aug 28 14:00:18 PDT 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -77,7 +77,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0x3c000000
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0fb7e40, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0fb7d80, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -101,7 +101,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
-      .data : 0xc0f00000 - 0xc0fc2dd0   ( 780 kB)
+      .data : 0xc0f00000 - 0xc0fc2d10   ( 780 kB)
        .bss : 0xc0fcc5b0 - 0xc10176fc   ( 301 kB)
 Preemptible hierarchical RCU implementation.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
@@ -316,83 +316,23 @@
 mmc1: SDHCI controller on c8000600.sdhci [c8000600.sdhci] using ADMA
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
-------------[ cut here ]------------
-WARNING: CPU: 0 PID: 1 at fs/proc/generic.c:330 proc_register+0xec/0x10c
-proc_dir_entry 'asound/pcm' already registered
-Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.13.0-rc6-next-20170824-196160-g9506597d #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c01110d0>] (unwind_backtrace) from [<c010c094>] (show_stack+0x10/0x14)
-[<c010c094>] (show_stack) from [<c095cd70>] (dump_stack+0x84/0xa4)
-[<c095cd70>] (dump_stack) from [<c012194c>] (__warn+0xd8/0x104)
-[<c012194c>] (__warn) from [<c0121a18>] (warn_slowpath_fmt+0x38/0x48)
-[<c0121a18>] (warn_slowpath_fmt) from [<c029d518>] (proc_register+0xec/0x10c)
-[<c029d518>] (proc_register) from [<c029d5bc>] (proc_create_data+0x84/0xc8)
-[<c029d5bc>] (proc_create_data) from [<c06eae90>] (snd_info_register+0x60/0xc8)
-[<c06eae90>] (snd_info_register) from [<c06f0750>] (snd_pcm_new_stream+0x13c/0x400)
-[<c06f0750>] (snd_pcm_new_stream) from [<c06f0ac4>] (_snd_pcm_new+0xb0/0x13c)
-[<c06f0ac4>] (_snd_pcm_new) from [<c06f0ba0>] (snd_pcm_new+0x24/0x2c)
-[<c06f0ba0>] (snd_pcm_new) from [<c072ec8c>] (soc_new_pcm+0x13c/0x424)
-[<c072ec8c>] (soc_new_pcm) from [<c0722f2c>] (snd_soc_register_card+0xc14/0xd38)
-[<c0722f2c>] (snd_soc_register_card) from [<c073c970>] (tegra_snd_trimslice_probe+0x9c/0x110)
-[<c073c970>] (tegra_snd_trimslice_probe) from [<c051d07c>] (platform_drv_probe+0x50/0xb0)
-[<c051d07c>] (platform_drv_probe) from [<c051b78c>] (driver_probe_device+0x238/0x2e4)
-[<c051b78c>] (driver_probe_device) from [<c051b8e4>] (__driver_attach+0xac/0xb0)
-[<c051b8e4>] (__driver_attach) from [<c0519e40>] (bus_for_each_dev+0x54/0x88)
-[<c0519e40>] (bus_for_each_dev) from [<c051ace4>] (bus_add_driver+0xec/0x1f8)
-[<c051ace4>] (bus_add_driver) from [<c051c45c>] (driver_register+0x78/0xf4)
-[<c051c45c>] (driver_register) from [<c0101b8c>] (do_one_initcall+0x3c/0x168)
-[<c0101b8c>] (do_one_initcall) from [<c0e00da8>] (kernel_init_freeable+0x10c/0x1d0)
-[<c0e00da8>] (kernel_init_freeable) from [<c097037c>] (kernel_init+0x8/0x114)
-[<c097037c>] (kernel_init) from [<c0107ce0>] (ret_from_fork+0x14/0x34)
----[ end trace 5cc18940195e9016 ]---
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
-------------[ cut here ]------------
-WARNING: CPU: 0 PID: 1 at fs/proc/generic.c:330 proc_register+0xec/0x10c
-proc_dir_entry 'asound/pcm' already registered
-Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Tainted: G        W       4.13.0-rc6-next-20170824-196160-g9506597d #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c01110d0>] (unwind_backtrace) from [<c010c094>] (show_stack+0x10/0x14)
-[<c010c094>] (show_stack) from [<c095cd70>] (dump_stack+0x84/0xa4)
-[<c095cd70>] (dump_stack) from [<c012194c>] (__warn+0xd8/0x104)
-[<c012194c>] (__warn) from [<c0121a18>] (warn_slowpath_fmt+0x38/0x48)
-[<c0121a18>] (warn_slowpath_fmt) from [<c029d518>] (proc_register+0xec/0x10c)
-[<c029d518>] (proc_register) from [<c029d5bc>] (proc_create_data+0x84/0xc8)
-[<c029d5bc>] (proc_create_data) from [<c06eae90>] (snd_info_register+0x60/0xc8)
-[<c06eae90>] (snd_info_register) from [<c06f0750>] (snd_pcm_new_stream+0x13c/0x400)
-[<c06f0750>] (snd_pcm_new_stream) from [<c06f0adc>] (_snd_pcm_new+0xc8/0x13c)
-[<c06f0adc>] (_snd_pcm_new) from [<c06f0ba0>] (snd_pcm_new+0x24/0x2c)
-[<c06f0ba0>] (snd_pcm_new) from [<c072ec8c>] (soc_new_pcm+0x13c/0x424)
-[<c072ec8c>] (soc_new_pcm) from [<c0722f2c>] (snd_soc_register_card+0xc14/0xd38)
-[<c0722f2c>] (snd_soc_register_card) from [<c073c970>] (tegra_snd_trimslice_probe+0x9c/0x110)
-[<c073c970>] (tegra_snd_trimslice_probe) from [<c051d07c>] (platform_drv_probe+0x50/0xb0)
-[<c051d07c>] (platform_drv_probe) from [<c051b78c>] (driver_probe_device+0x238/0x2e4)
-[<c051b78c>] (driver_probe_device) from [<c051b8e4>] (__driver_attach+0xac/0xb0)
-[<c051b8e4>] (__driver_attach) from [<c0519e40>] (bus_for_each_dev+0x54/0x88)
-[<c0519e40>] (bus_for_each_dev) from [<c051ace4>] (bus_add_driver+0xec/0x1f8)
-[<c051ace4>] (bus_add_driver) from [<c051c45c>] (driver_register+0x78/0xf4)
-[<c051c45c>] (driver_register) from [<c0101b8c>] (do_one_initcall+0x3c/0x168)
-[<c0101b8c>] (do_one_initcall) from [<c0e00da8>] (kernel_init_freeable+0x10c/0x1d0)
-[<c0e00da8>] (kernel_init_freeable) from [<c097037c>] (kernel_init+0x8/0x114)
-[<c097037c>] (kernel_init) from [<c0107ce0>] (ret_from_fork+0x14/0x34)
----[ end trace 5cc18940195e9017 ]---
 tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
 NET: Registered protocol family 10
 Segment Routing with IPv6
 mip6: Mobile IPv6
-mmc1: new high speed SD card at address b368
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
- mmcblk1: p1 p2
 NET: Registered protocol family 17
 NET: Registered protocol family 15
 can: controller area network core (rev 20170425 abi 9)
 NET: Registered protocol family 29
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 can: raw protocol (rev 20170425)
 can: broadcast manager protocol (rev 20170425 t)
 can: netlink gateway (rev 20170425) max_hops=1
 Bluetooth: RFCOMM socket layer initialized
+mmc1: new high speed SD card at address b368
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
+ mmcblk1: p1 p2
 Bluetooth: RFCOMM ver 1.11
 Bluetooth: BNEP (Ethernet Emulation) ver 1.3
 Bluetooth: BNEP socket layer initialized
@@ -408,10 +348,10 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: hctosys: unable to read the hardware clock
-ALSA device list:
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
+rtc-em3027 2-0056: hctosys: unable to read the hardware clock
+ALSA device list:
   #0: tegra-trimslice
 Waiting for root device PARTUUID=11eb23c1-01...
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2017-08-28 14:08:24.705467596 -0700
+++ /tmp/foob	2017-08-28 14:08:24.705467596 -0700
@@ -1,7 +1,7 @@
 [TegraBoot] (version 23.00.2015.14-mobile-1ef66670)
 Processing in cold boot mode Bootloader 2
 A01Q Bootrom Patch rev = 3
-Power-up reason: pmu watchdog
+Power-up reason: on button
 No Battery Present
 Platform has Ddr4 type ram
 max77620 disabling SD1 Remote Sense
@@ -64,13 +64,13 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 723751 us
-Basic modules init took 349910 us
-Sec Bootdevice Read Time = 203 ms, Read Size = 8459 KB
-Next stage binary read took 14363 us
-Carveout took 252713 us
-CPU initialization took 92461 us
-Total time taken by TegraBoot 709447 us
+Tegraboot started after 721744 us
+Basic modules init took 350284 us
+Sec Bootdevice Read Time = 204 ms, Read Size = 8459 KB
+Next stage binary read took 15667 us
+Carveout took 252764 us
+CPU initialization took 92447 us
+Total time taken by TegraBoot 711162 us
 
 Starting CPU & Halting co-processor 
 
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,9 +117,9 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-16505344 bytes read in 836 ms (18.8 MiB/s)
+16507392 bytes read in 836 ms (18.8 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-45170 bytes read in 454 ms (96.7 KiB/s)
+45170 bytes read in 453 ms (96.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
 1627787 bytes read in 495 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc6-next-20170824-196160-g9506597d (jonathanh@sellafield) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Mon Aug 28 13:37:21 PDT 2017
+Linux version 4.13.0-rc6-next-20170825-314808-g7159188 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Mon Aug 28 14:04:57 PDT 2017
 Boot CPU: AArch64 Processor [411fd071]
 Machine model: NVIDIA Tegra210 P2371 (P2530/P2595) reference design
 debug: ignoring loglevel setting.
@@ -173,15 +173,15 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffff80007a000000-ffff80007dffffff]
-Memory: 2982564K/3131392K available (9212K kernel code, 1142K rwdata, 4552K rodata, 1152K init, 402K bss, 132444K reserved, 16384K cma-reserved)
+Memory: 2982560K/3131392K available (9212K kernel code, 1144K rwdata, 4544K rodata, 1152K init, 404K bss, 132448K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008980000   (  9216 KB)
     .rodata : 0xffff000008980000 - 0xffff000008e00000   (  4608 KB)
       .init : 0xffff000008e00000 - 0xffff000008f20000   (  1152 KB)
-      .data : 0xffff000008f20000 - 0xffff00000903da00   (  1143 KB)
-       .bss : 0xffff00000903da00 - 0xffff0000090a23f0   (   403 KB)
+      .data : 0xffff000008f20000 - 0xffff00000903e200   (  1145 KB)
+       .bss : 0xffff00000903e200 - 0xffff0000090a33f0   (   405 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -268,14 +268,14 @@
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 PCI: CLS 0 bytes, default 128
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 8096b000
+kvm [1]: IDMAP page: 8096c000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
 kvm [1]: vgic interrupt IRQ1
 kvm [1]: virtual timer IRQ4
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.519:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.507:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2017-08-28 14:08:24.713467626 -0700
+++ /tmp/foob	2017-08-28 14:08:24.713467626 -0700
@@ -1,8 +1,4 @@
- Google Pixel C
-efi: Getting EFI parameters from FDT:
-efi: UEFI not found.
-cma: Reserved 16 MiB at 0x00000000f5400000
-earlycon: uart0 at MMIO 0x0000000070006000 (options '115200n8')
+0000070006000 (options '115200n8')
 bootconsole [uart0] enabled
 NUMA: No NUMA configuration found
 NUMA: Faking a node at [mem 0x0000000000000000-0x000000013fffffff]
@@ -28,15 +24,15 @@
 Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2837828K/2988032K available (9212K kernel code, 1142K rwdata, 4552K rodata, 1152K init, 402K bss, 133820K reserved, 16384K cma-reserved)
+Memory: 2837828K/2988032K available (9212K kernel code, 1144K rwdata, 4544K rodata, 1152K init, 404K bss, 133820K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008980000   (  9216 KB)
     .rodata : 0xffff000008980000 - 0xffff000008e00000   (  4608 KB)
       .init : 0xffff000008e00000 - 0xffff000008f20000   (  1152 KB)
-      .data : 0xffff000008f20000 - 0xffff00000903da00   (  1143 KB)
-       .bss : 0xffff00000903da00 - 0xffff0000090a23f0   (   403 KB)
+      .data : 0xffff000008f20000 - 0xffff00000903e200   (  1145 KB)
+       .bss : 0xffff00000903e200 - 0xffff0000090a33f0   (   405 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -55,15 +51,15 @@
 boot misc2 0x1f720005: expected 0x1f720f05
  (comparison mask = 0xffffff17)
 PLL_C already enabled. Postponing set full defaults
-PLL_DP already enabled. Postponing set full defaults
 boot misc0 0x0: expected 0x20
  (comparison mask = 0x7ffffff)
-boot misc1 0x90000000: expected 0xc0000000
+boot misc1 0x10000000: expected 0xc0000000
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x8d2: expected 0x2004f400
+boot misc3 0x0: expected 0x2004f400
  (comparison mask = 0xffffffff)
+PLL_DP already enabled. Postponing set full defaults
 boot misc1 0x0: expected 0x20
  (comparison mask = 0xffffff)
 PLL_X already enabled. Postponing set full defaults
@@ -74,7 +70,7 @@
 console [tty0] enabled
 bootconsole [uart0] disabled
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc6-next-20170824-196160-g9506597d (jonathanh@sellafield) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Mon Aug 28 13:37:21 PDT 2017
+Linux version 4.13.0-rc6-next-20170825-314808-g7159188 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Mon Aug 28 14:04:57 PDT 2017
 Boot CPU: AArch64 Processor [411fd071]
 Machine model: Google Pixel C
 efi: Getting EFI parameters from FDT:
@@ -106,15 +102,15 @@
 Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2837828K/2988032K available (9212K kernel code, 1142K rwdata, 4552K rodata, 1152K init, 402K bss, 133820K reserved, 16384K cma-reserved)
+Memory: 2837828K/2988032K available (9212K kernel code, 1144K rwdata, 4544K rodata, 1152K init, 404K bss, 133820K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008980000   (  9216 KB)
     .rodata : 0xffff000008980000 - 0xffff000008e00000   (  4608 KB)
       .init : 0xffff000008e00000 - 0xffff000008f20000   (  1152 KB)
-      .data : 0xffff000008f20000 - 0xffff00000903da00   (  1143 KB)
-       .bss : 0xffff00000903da00 - 0xffff0000090a23f0   (   403 KB)
+      .data : 0xffff000008f20000 - 0xffff00000903e200   (  1145 KB)
+       .bss : 0xffff00000903e200 - 0xffff0000090a33f0   (   405 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -133,15 +129,15 @@
 boot misc2 0x1f720005: expected 0x1f720f05
  (comparison mask = 0xffffff17)
 PLL_C already enabled. Postponing set full defaults
-PLL_DP already enabled. Postponing set full defaults
 boot misc0 0x0: expected 0x20
  (comparison mask = 0x7ffffff)
-boot misc1 0x90000000: expected 0xc0000000
+boot misc1 0x10000000: expected 0xc0000000
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x8d2: expected 0x2004f400
+boot misc3 0x0: expected 0x2004f400
  (comparison mask = 0xffffffff)
+PLL_DP already enabled. Postponing set full defaults
 boot misc1 0x0: expected 0x20
  (comparison mask = 0xffffff)
 PLL_X already enabled. Postponing set full defaults
@@ -223,16 +219,16 @@
 RPC: Registered tcp transport module.
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 Unpacking initramfs...
-Freeing initrd memory: 1284K
+Freeing initrd memory: 1280K
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 8096b000
+kvm [1]: IDMAP page: 8096c000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
 kvm [1]: vgic interrupt IRQ1
 kvm [1]: virtual timer IRQ4
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.271:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.267:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -328,16 +324,15 @@
 Key type dns_resolver registered
 registered taskstats version 1
 tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
+input: gpio-keys as /devices/platform/gpio-keys/input/input0
+tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 BGND3R 29.1 GiB 
 mmcblk0boot0: mmc0:0001 BGND3R partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
-tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
-input: gpio-keys as /devices/platform/gpio-keys/input/input0
-tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
-max77686-rtc max77620-rtc: setting system clock to 2001-06-09 02:02:31 UTC (992052151)
+max77686-rtc max77620-rtc: setting system clock to 2001-06-09 02:29:46 UTC (992053786)
 PP1200_AVDD: disabling
 PP1200_RCAM: disabling
 PP2800L_RCAM: disabling
@@ -348,6 +343,7 @@
 PP1800_CAM: disabling
 ALSA device list:
   No soundcards found.
+tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 Freeing unused kernel memory: 1152K
 
 System initialization...
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-08-28 14:08:24.717467641 -0700
+++ /tmp/foob	2017-08-28 14:08:24.721467655 -0700
@@ -40,7 +40,7 @@
 	 #################################################################
 	 #################################################################
 	 ###################################
-	 2.8 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 8135176 (7c2208 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
@@ -49,7 +49,7 @@
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.6 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 51860 (ca94 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2017-08-28 14:08:24.725467670 -0700
+++ /tmp/foob	2017-08-28 14:08:24.729467685 -0700
@@ -27,6 +27,7 @@
 Tegra30 (Beaver) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=edbe1840-3633-c751-7993-b0ac834f81e0 rootwait gpt debug noinitrd earlyprintk init=/bin/bash'
 Tegra30 (Beaver) # setenv ipaddr 192.168.100.101; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra30 (Beaver) # tftpboot 0x81000000 zImage.tegra_defconfig
+Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'zImage.tegra_defconfig'.
@@ -38,9 +39,9 @@
 	 #################################################################
 	 #################################################################
 	 ######################################
-	 2.9 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 6280856 (5fd698 hex)
+Bytes transferred = 6282712 (5fddd8 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -51,7 +52,7 @@
 done
 Bytes transferred = 51860 (ca94 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5fd698 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5fddd8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa93 ... OK
@@ -60,7 +61,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc6-next-20170824-196160-g9506597d (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Aug 28 13:33:59 PDT 2017
+Linux version 4.13.0-rc6-next-20170825-196567-g7159188 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Aug 28 14:00:18 PDT 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -69,7 +70,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0fb7e40, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0fb7d80, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -93,7 +94,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
-      .data : 0xc0f00000 - 0xc0fc2dd0   ( 780 kB)
+      .data : 0xc0f00000 - 0xc0fc2d10   ( 780 kB)
        .bss : 0xc0fcc5b0 - 0xc10176fc   ( 301 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
@@ -153,107 +154,107 @@
 [00000020] *pgd=00000000
 Internal error: Oops: 5 [#1] PREEMPT SMP ARM
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.13.0-rc6-next-20170824-196160-g9506597d #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.13.0-rc6-next-20170825-196567-g7159188 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 task: ee88e740 task.stack: ee8a2000
 PC is at sysfs_add_link_to_group+0x8/0x48
 LR is at iommu_device_link+0x3c/0x9c
-pc : [<c02a956c>]    lr : [<c048b4c4>]    psr: 60000013
+pc : [<c02a9388>]    lr : [<c048b580>]    psr: 60000013
 sp : ee8a3d40  ip : ee8a3d70  fp : 00000000
-r10: c0f8e39c  r9 : eea75c10  r8 : 002a0042
+r10: c0f8e29c  r9 : eea75c10  r8 : 002a0042
 r7 : ee9fbc18  r6 : eefcf2ac  r5 : ee9fbc10  r4 : ee9fbc18
-r3 : ee9f97c0  r2 : ee9fbc18  r1 : c0bec2f8  r0 : 00000008
+r3 : ee9f97c0  r2 : ee9fbc18  r1 : c0bec64c  r0 : 00000008
 Flags: nZCv  IRQs on  FIQs on  Mode SVC_32  ISA ARM  Segment none
 Control: 10c5387d  Table: 8000404a  DAC: 00000051
 Process swapper/0 (pid: 1, stack limit = 0xee8a2210)
 Stack: (0xee8a3d40 to 0xee8a4000)
-3d40: eea76950 ee9fbc10 eefcf2ac c048b4c4 eea857d0 00000001 ee8a3d70 eefcf2ac
-3d60: ee9fbc10 c048d6bc ee8a3d70 6d656d2e eeff11a8 00000001 00000001 c0007265
-3d80: eea85680 c043cd64 c0489c68 a0000013 eea85600 c0238424 000000a5 c0489c68
-3da0: eea142b8 ee8a3de0 eea14334 00000000 c0519c9c eea75c10 c0f8e39c ee9fbc10
-3dc0: c0a588cc c0488e64 00000000 c0488e90 00000000 00000000 ee8a3dfc c0519e40
-3de0: ee8ffa5c eea142b4 c0f69188 00000000 eea85600 c0489cbc eea76910 c0a588cc
-3e00: 2000003f eea76910 20000010 2000003f eea75c10 c048e654 ee826480 ee8a3e30
-3e20: c0c48284 eea857d0 00000000 00000000 00000042 c06d6de0 00000000 00000000
+3d40: eea76950 ee9fbc10 eefcf2ac c048b580 eea857d0 00000001 ee8a3d70 eefcf2ac
+3d60: ee9fbc10 c048d778 ee8a3d70 6d656d2e eeff11a8 00000001 00000001 c0007265
+3d80: eea85680 c043cd98 c0489d24 a0000013 eea85600 c023824c 000000a5 c0489d24
+3da0: eea142b8 ee8a3de0 eea14334 00000000 c0519d58 eea75c10 c0f8e29c ee9fbc10
+3dc0: c0a588cc c0488f20 00000000 c0488f4c 00000000 00000000 ee8a3dfc c0519efc
+3de0: ee8ffa5c eea142b4 c0f69188 00000000 eea85600 c0489d78 eea76910 c0a588cc
+3e00: 2000003f eea76910 20000010 2000003f eea75c10 c048e710 ee826480 ee8a3e30
+3e20: c0c48614 eea857d0 00000000 00000000 00000042 c06d6dbc 00000000 00000000
 3e40: 00000000 00000001 eea770e8 eea880e8 eea75c00 00000003 00000000 eea88008
-3e60: 00000000 ffffffed eea75c10 fffffdfb c0f8e39c 00000000 00000000 c0f8e39c
-3e80: 00000000 c051d07c c051d02c eea75c10 c1009250 c1009254 00000000 c051b78c
-3ea0: eea75c10 c0f8e39c eea75c44 00000000 00000111 c0e6a83c c0e005a4 c051b8e4
-3ec0: 00000000 c0f8e39c c051b838 c0519e40 ee8ffa5c eea76734 c0f8e39c eea76980
-3ee0: c0f69188 c051ace4 c0c483f0 c0f8e39c 00000003 c0f8e39c 00000003 c06d6ac8
-3f00: 00000000 c051c45c c051ccd4 c0e82f04 00000003 c0101b8c 00000000 c0cb9104
-3f20: 00000100 efffcc45 c0cb9104 00000111 efffcc53 c013e478 c0cb7bb0 00000000
+3e60: 00000000 ffffffed eea75c10 fffffdfb c0f8e29c 00000000 00000000 c0f8e29c
+3e80: 00000000 c051d138 c051d0e8 eea75c10 c1009250 c1009254 00000000 c051b848
+3ea0: eea75c10 c0f8e29c eea75c44 00000000 00000111 c0e6a83c c0e005a4 c051b9a0
+3ec0: 00000000 c0f8e29c c051b8f4 c0519efc ee8ffa5c eea76734 c0f8e29c eea76980
+3ee0: c0f69188 c051ada0 c0c48780 c0f8e29c 00000003 c0f8e29c 00000003 c06d6aa4
+3f00: 00000000 c051c518 c051cd90 c0e82f04 00000003 c0101b8c 00000000 c0cb9108
+3f20: 00000100 efffcc45 c0cb9108 00000111 efffcc53 c013e478 c0cb7bb4 00000000
 3f40: 00000003 00000003 efffcc45 efffcc4a c0f0dcd0 efffcbc0 00000003 c0e6a820
 3f60: c0e82f04 00000003 c0e6a824 c0fcc5c0 00000111 c0e00da8 00000003 00000003
-3f80: 00000000 c0e005a4 00000000 c0970374 00000000 00000000 00000000 00000000
-3fa0: 00000000 c097037c 00000000 c0107ce0 00000000 00000000 00000000 00000000
+3f80: 00000000 c0e005a4 00000000 c09708f4 00000000 00000000 00000000 00000000
+3fa0: 00000000 c09708fc 00000000 c0107ce0 00000000 00000000 00000000 00000000
 3fc0: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
 3fe0: 00000000 00000000 00000000 00000000 00000013 00000000 ffffffff ffffffff
-[<c02a956c>] (sysfs_add_link_to_group) from [<c048b4c4>] (iommu_device_link+0x3c/0x9c)
-[<c048b4c4>] (iommu_device_link) from [<c048d6bc>] (tegra_smmu_add_device+0x9c/0xa8)
-[<c048d6bc>] (tegra_smmu_add_device) from [<c0488e90>] (add_iommu_group+0x2c/0x58)
-[<c0488e90>] (add_iommu_group) from [<c0519e40>] (bus_for_each_dev+0x54/0x88)
-[<c0519e40>] (bus_for_each_dev) from [<c0489cbc>] (bus_set_iommu+0x94/0xd8)
-[<c0489cbc>] (bus_set_iommu) from [<c048e654>] (tegra_smmu_probe+0x17c/0x264)
-[<c048e654>] (tegra_smmu_probe) from [<c06d6de0>] (tegra_mc_probe+0x308/0x51c)
-[<c06d6de0>] (tegra_mc_probe) from [<c051d07c>] (platform_drv_probe+0x50/0xb0)
-[<c051d07c>] (platform_drv_probe) from [<c051b78c>] (driver_probe_device+0x238/0x2e4)
-[<c051b78c>] (driver_probe_device) from [<c051b8e4>] (__driver_attach+0xac/0xb0)
-[<c051b8e4>] (__driver_attach) from [<c0519e40>] (bus_for_each_dev+0x54/0x88)
-[<c0519e40>] (bus_for_each_dev) from [<c051ace4>] (bus_add_driver+0xec/0x1f8)
-[<c051ace4>] (bus_add_driver) from [<c051c45c>] (driver_register+0x78/0xf4)
-[<c051c45c>] (driver_register) from [<c0101b8c>] (do_one_initcall+0x3c/0x168)
+[<c02a9388>] (sysfs_add_link_to_group) from [<c048b580>] (iommu_device_link+0x3c/0x9c)
+[<c048b580>] (iommu_device_link) from [<c048d778>] (tegra_smmu_add_device+0x9c/0xa8)
+[<c048d778>] (tegra_smmu_add_device) from [<c0488f4c>] (add_iommu_group+0x2c/0x58)
+[<c0488f4c>] (add_iommu_group) from [<c0519efc>] (bus_for_each_dev+0x54/0x88)
+[<c0519efc>] (bus_for_each_dev) from [<c0489d78>] (bus_set_iommu+0x94/0xd8)
+[<c0489d78>] (bus_set_iommu) from [<c048e710>] (tegra_smmu_probe+0x17c/0x264)
+[<c048e710>] (tegra_smmu_probe) from [<c06d6dbc>] (tegra_mc_probe+0x308/0x51c)
+[<c06d6dbc>] (tegra_mc_probe) from [<c051d138>] (platform_drv_probe+0x50/0xb0)
+[<c051d138>] (platform_drv_probe) from [<c051b848>] (driver_probe_device+0x238/0x2e4)
+[<c051b848>] (driver_probe_device) from [<c051b9a0>] (__driver_attach+0xac/0xb0)
+[<c051b9a0>] (__driver_attach) from [<c0519efc>] (bus_for_each_dev+0x54/0x88)
+[<c0519efc>] (bus_for_each_dev) from [<c051ada0>] (bus_add_driver+0xec/0x1f8)
+[<c051ada0>] (bus_add_driver) from [<c051c518>] (driver_register+0x78/0xf4)
+[<c051c518>] (driver_register) from [<c0101b8c>] (do_one_initcall+0x3c/0x168)
 [<c0101b8c>] (do_one_initcall) from [<c0e00da8>] (kernel_init_freeable+0x10c/0x1d0)
-[<c0e00da8>] (kernel_init_freeable) from [<c097037c>] (kernel_init+0x8/0x114)
-[<c097037c>] (kernel_init) from [<c0107ce0>] (ret_from_fork+0x14/0x34)
+[<c0e00da8>] (kernel_init_freeable) from [<c09708fc>] (kernel_init+0x8/0x114)
+[<c09708fc>] (kernel_init) from [<c0107ce0>] (ret_from_fork+0x14/0x34)
 Code: e8bd4038 eafff194 e92d4070 e1a04002 (e5900018) 
----[ end trace 513cadd921e7c108 ]---
+---[ end trace 8b3f83edd1fc5b59 ]---
 Kernel panic - not syncing: Attempted to kill init! exitcode=0x0000000b
 
-CPU3: stopping
-CPU: 3 PID: 0 Comm: swapper/3 Tainted: G      D         4.13.0-rc6-next-20170824-196160-g9506597d #1
+CPU1: stopping
+CPU: 1 PID: 0 Comm: swapper/1 Tainted: G      D         4.13.0-rc6-next-20170825-196567-g7159188 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c01110d0>] (unwind_backtrace) from [<c010c094>] (show_stack+0x10/0x14)
-[<c010c094>] (show_stack) from [<c095cd70>] (dump_stack+0x84/0xa4)
-[<c095cd70>] (dump_stack) from [<c010f620>] (handle_IPI+0x18c/0x39c)
+[<c010c094>] (show_stack) from [<c095d2f0>] (dump_stack+0x84/0xa4)
+[<c095d2f0>] (dump_stack) from [<c010f620>] (handle_IPI+0x18c/0x39c)
 [<c010f620>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
-Exception stack(0xee8e1f70 to 0xee8e1fb8)
-1f60:                                     00000001 00000000 ee8e1fc8 c011a3e0
-1f80: c0f03ccc c0f03c68 c0e8c378 c0f03cd8 c0fbdb64 00000000 00000001 00000000
-1fa0: 2e136000 ee8e1fc0 c0108758 c010875c 60000013 ffffffff
+Exception stack(0xee8bdf70 to 0xee8bdfb8)
+df60:                                     00000001 00000000 ee8bdfc8 c011a3e0
+df80: c0f03ccc c0f03c68 c0e8c378 c0f03cd8 c0fbdaa4 00000000 00000001 00000000
+dfa0: 2e114000 ee8bdfc0 c0108758 c010875c 60000013 ffffffff
 [<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
 [<c010875c>] (arch_cpu_idle) from [<c0160c0c>] (do_idle+0x148/0x204)
 [<c0160c0c>] (do_idle) from [<c0160f50>] (cpu_startup_entry+0x18/0x1c)
 [<c0160f50>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
 CPU2: stopping
-CPU: 2 PID: 0 Comm: swapper/2 Tainted: G      D         4.13.0-rc6-next-20170824-196160-g9506597d #1
+CPU: 2 PID: 0 Comm: swapper/2 Tainted: G      D         4.13.0-rc6-next-20170825-196567-g7159188 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c01110d0>] (unwind_backtrace) from [<c010c094>] (show_stack+0x10/0x14)
-[<c010c094>] (show_stack) from [<c095cd70>] (dump_stack+0x84/0xa4)
-[<c095cd70>] (dump_stack) from [<c010f620>] (handle_IPI+0x18c/0x39c)
+[<c010c094>] (show_stack) from [<c095d2f0>] (dump_stack+0x84/0xa4)
+[<c095d2f0>] (dump_stack) from [<c010f620>] (handle_IPI+0x18c/0x39c)
 [<c010f620>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
 Exception stack(0xee8dff70 to 0xee8dffb8)
 ff60:                                     00000001 00000000 ee8dffc8 c011a3e0
-ff80: c0f03ccc c0f03c68 c0e8c378 c0f03cd8 c0fbdb64 00000000 00000001 00000000
+ff80: c0f03ccc c0f03c68 c0e8c378 c0f03cd8 c0fbdaa4 00000000 00000001 00000000
 ffa0: 2e125000 ee8dffc0 c0108758 c010875c 60000013 ffffffff
 [<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
 [<c010875c>] (arch_cpu_idle) from [<c0160c0c>] (do_idle+0x148/0x204)
 [<c0160c0c>] (do_idle) from [<c0160f50>] (cpu_startup_entry+0x18/0x1c)
 [<c0160f50>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
-CPU1: stopping
-CPU: 1 PID: 0 Comm: swapper/1 Tainted: G      D         4.13.0-rc6-next-20170824-196160-g9506597d #1
+CPU3: stopping
+CPU: 3 PID: 0 Comm: swapper/3 Tainted: G      D         4.13.0-rc6-next-20170825-196567-g7159188 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c01110d0>] (unwind_backtrace) from [<c010c094>] (show_stack+0x10/0x14)
-[<c010c094>] (show_stack) from [<c095cd70>] (dump_stack+0x84/0xa4)
-[<c095cd70>] (dump_stack) from [<c010f620>] (handle_IPI+0x18c/0x39c)
+[<c010c094>] (show_stack) from [<c095d2f0>] (dump_stack+0x84/0xa4)
+[<c095d2f0>] (dump_stack) from [<c010f620>] (handle_IPI+0x18c/0x39c)
 [<c010f620>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
-Exception stack(0xee8bdf70 to 0xee8bdfb8)
-df60:                                     00000001 00000000 ee8bdfc8 c011a3e0
-df80: c0f03ccc c0f03c68 c0e8c378 c0f03cd8 c0fbdb64 00000000 00000001 00000000
-dfa0: 2e114000 ee8bdfc0 c0108758 c010875c 60000013 ffffffff
+Exception stack(0xee8e1f70 to 0xee8e1fb8)
+1f60:                                     00000001 00000000 ee8e1fc8 c011a3e0
+1f80: c0f03ccc c0f03c68 c0e8c378 c0f03cd8 c0fbdaa4 00000000 00000001 00000000
+1fa0: 2e136000 ee8e1fc0 c0108758 c010875c 60000013 ffffffff
 [<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
 [<c010875c>] (arch_cpu_idle) from [<c0160c0c>] (do_idle+0x148/0x204)
 [<c0160c0c>] (do_idle) from [<c0160f50>] (cpu_startup_entry+0x18/0x1c)
