Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2017-02-22 21:15:27.318144976 -0800
+++ /tmp/foob	2017-02-22 21:15:27.318144976 -0800
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-next-20170222-178980-g9fb8cda (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Feb 21 21:06:54 PST 2017
+Linux version 4.10.0-next-20170223-179007-g27fde84 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Feb 22 21:08:23 PST 2017
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -114,7 +114,7 @@
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 kvm [1]: HYP mode not available
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.704:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.712:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -180,7 +180,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2017-02-22 05:07:38 UTC (1487740058)
+rtc-pl031 9010000.pl031: setting system clock to 2017-02-23 05:09:05 UTC (1487826545)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-02-22 21:15:27.342145070 -0800
+++ /tmp/foob	2017-02-22 21:15:27.346145085 -0800
@@ -41,7 +41,7 @@
 	 ###########################################################
 	 3.1 MiB/s
 done
-Bytes transferred = 7545456 (732270 hex)
+Bytes transferred = 7545432 (732258 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -52,7 +52,7 @@
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x732270 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x732258 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-next-20170222-154547-g9fb8cda (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Feb 21 21:03:55 PST 2017
+Linux version 4.10.0-next-20170223-154574-g27fde84 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Feb 22 21:06:01 PST 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -305,12 +305,11 @@
 Key type dns_resolver registered
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
-usb1_vbus: supplied by vdd-sys-5v0
 mmc0: new high speed SDHC card at address aaaa
-mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
+usb1_vbus: supplied by vdd-sys-5v0
 usb2_vbus: supplied by vdd-sys-5v0
 VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
- mmcblk0: p1 p2 p3
+mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
 mmc1: new high speed MMC card at address 0001
@@ -318,6 +317,7 @@
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
+ mmcblk0: p1 p2 p3
  mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
 Console: switching to colour frame buffer device 240x75
 drm drm: fb0:  frame buffer device
@@ -329,7 +329,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-12-16 16:38:36 UTC (976984716)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-12-17 16:42:16 UTC (977071336)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -341,4 +341,4 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.529418] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.529538] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2017-02-22 21:15:27.354145117 -0800
+++ /tmp/foob	2017-02-22 21:15:27.354145117 -0800
@@ -39,18 +39,18 @@
 	 #######################################################
 	 3.2 MiB/s
 done
-Bytes transferred = 5566968 (54f1f8 hex)
+Bytes transferred = 5566288 (54ef50 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.8 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x54f1f8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x54ef50 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-next-20170222-154547-g9fb8cda (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Feb 21 21:03:38 PST 2017
+Linux version 4.10.0-next-20170223-154574-g27fde84 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Feb 22 21:05:29 PST 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -321,7 +321,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-12-16 16:35:57 UTC (976984557)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-12-17 16:39:37 UTC (977071177)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -335,4 +335,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.746551] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.737138] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-02-22 21:15:27.362145147 -0800
+++ /tmp/foob	2017-02-22 21:15:27.366145163 -0800
@@ -34,20 +34,20 @@
 	 #################################################################
 	 #################################################################
 	 ###########################################################
-	 3.3 MiB/s
+	 3.9 MiB/s
 done
-Bytes transferred = 7545456 (732270 hex)
+Bytes transferred = 7545432 (732258 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.6 MiB/s
+	 3.8 MiB/s
 done
 Bytes transferred = 67133 (1063d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x732270 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x732258 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301363c
@@ -55,7 +55,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-next-20170222-154547-g9fb8cda (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Feb 21 21:03:55 PST 2017
+Linux version 4.10.0-next-20170223-154574-g27fde84 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Feb 22 21:06:01 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -314,6 +314,12 @@
 +5V_SATA: supplied by +5V_SYS
 +12V_SATA: supplied by +VDD_MUX
  usb2-0: usb2-0 supply vbus not found, using dummy regulator
+mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+ mmcblk0: p1
  usb2-1: usb2-1 supply vbus not found, using dummy regulator
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
@@ -338,22 +344,16 @@
 as3722-ldo10: supplied by +5V_SYS
 +2.8V_RUN_CAM_AF: Bringing 0uV into 2800000-2800000uV
 as3722-ldo11: supplied by +3.3V_RUN
-mmc0: new high speed MMC card at address 0001
 +1.8V_RUN_VPP_FUSE: Bringing 0uV into 1800000-1800000uV
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 tegra-ahci 70027000.sata: AHCI 0001.0301 32 slots 2 ports 3 Gbps 0x1 impl platform mode
 tegra-ahci 70027000.sata: flags: 64bit ncq sntf pm led pio slum part sadm sds apst 
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 scsi host0: tegra-ahci
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
 scsi host1: tegra-ahci
 ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 345
 ata2: DUMMY
 tegra-ehci 7d008000.usb: EHCI Host Controller
 tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci 7d008000.usb: irq 355, io mem 0x7d008000
- mmcblk0: p1
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
@@ -382,8 +382,8 @@
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x52 may corrupt adjacent RW1C bits
 pci 0000:00:02.0: PME# supported from D0 D1 D2 D3hot D3cold
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4c may corrupt adjacent RW1C bits
-ata1: SATA link down (SStatus 0 SControl 300)
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x3e may corrupt adjacent RW1C bits
+ata1: SATA link down (SStatus 0 SControl 300)
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x3e may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x4 may corrupt adjacent RW1C bits
 pci_bus 0000:00: 1-byte config write to 0000:00:01.0 offset 0xc may corrupt adjacent RW1C bits
@@ -394,6 +394,7 @@
 pci 0000:01:00.0: reg 0x10: [mem 0x00000000-0x00001fff 64bit]
 pci 0000:01:00.0: PME# supported from D0 D3hot D3cold
 mmc1: new high speed SDHC card at address aaaa
+mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 PCI: bus1: Fast back to back transfers disabled
 pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
 pci 0000:02:00.0: [10ec:8168] type 00 class 0x020000
@@ -402,7 +403,6 @@
 pci 0000:02:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
 pci 0000:02:00.0: supports D1 D2
 pci 0000:02:00.0: PME# supported from D0 D1 D2 D3hot D3cold
-mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
  mmcblk1: p1 p2 p3
 PCI: bus2: Fast back to back transfers disabled
 pci_bus 0000:02: busn_res: [bus 02-ff] end is updated to 02
@@ -441,7 +441,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:24 UTC (946684944)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:06:13 UTC (946685173)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2017-02-22 21:15:27.370145179 -0800
+++ /tmp/foob	2017-02-22 21:15:27.374145194 -0800
@@ -32,9 +32,9 @@
 	 #################################################################
 	 #################################################################
 	 #######################################################
-	 4 MiB/s
+	 3.9 MiB/s
 done
-Bytes transferred = 5566968 (54f1f8 hex)
+Bytes transferred = 5566288 (54ef50 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -45,7 +45,7 @@
 done
 Bytes transferred = 67133 (1063d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x54f1f8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x54ef50 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301363c
@@ -54,7 +54,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-next-20170222-154547-g9fb8cda (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Feb 21 21:03:38 PST 2017
+Linux version 4.10.0-next-20170223-154574-g27fde84 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Feb 22 21:05:29 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -408,7 +408,6 @@
 pci 0000:02:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
 pci 0000:02:00.0: supports D1 D2
 pci 0000:02:00.0: PME# supported from D0 D1 D2 D3hot D3cold
- mmcblk1: p1 p2 p3
 PCI: bus2: Fast back to back transfers disabled
 pci_bus 0000:02: busn_res: [bus 02-ff] end is updated to 02
 pci 0000:00:01.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
@@ -433,13 +432,14 @@
 pcieport 0000:00:02.0: Signaling PME with IRQ 394
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
-r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0c01000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 395
+r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0af7000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 395
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
-tegra-hdmi 54280000.hdmi: cannot set audio to 48000 Hz at 297000000 Hz pixel clock
-Console: switching to colour frame buffer device 512x135
+[drm] Cannot find any crtc or sizes - going 1024x768
+ mmcblk1: p1 p2 p3
+Console: switching to colour frame buffer device 128x48
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
 of: dev_pm_opp_of_cpumask_add_table: couldn't find opp table for cpu:0, -19
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-02-22 21:15:27.390145257 -0800
+++ /tmp/foob	2017-02-22 21:15:27.394145273 -0800
@@ -42,20 +42,20 @@
 	 #################################################################
 	 #################################################################
 	 ###########################################################
-	 3.5 MiB/s
+	 2.9 MiB/s
 done
-Bytes transferred = 7545456 (732270 hex)
+Bytes transferred = 7545432 (732258 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.2 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 68155 (10a3b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x732270 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x732258 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a3a
@@ -63,7 +63,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-next-20170222-154547-g9fb8cda (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Feb 21 21:03:55 PST 2017
+Linux version 4.10.0-next-20170223-154574-g27fde84 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Feb 22 21:06:01 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -296,8 +296,6 @@
 mmc0: Unknown controller version (3). You may experience problems.
 mmc0: Invalid maximum block size, assuming 512 bytes
 mmc0: SDHCI controller on 700b0600.sdhci [700b0600.sdhci] using ADMA
-elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
-input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 ledtrig-cpu: registered to indicate activity on CPUs
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
@@ -315,6 +313,8 @@
 Registering SWP/SWPB emulation handler
 +3.3V_RUN: supplied by +3.3V_SYS
 +VDD_LED: supplied by +VDD_MUX
+elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
+input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 +3.3V_AVDD_HDMI_AP_GATED: supplied by +3.3V_RUN
 +1.8V_RUN_CAM: supplied by +3.3V_RUN
 +VDDIO_SDMMC3: supplied by +3.3V_RUN
@@ -323,29 +323,29 @@
 +5V_USB_HS: supplied by +5V_SYS
 +5V_USB_SS: supplied by +5V_SYS
 +3.3V_PANEL: supplied by +3.3V_RUN
+mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
+mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
+mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
+mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 +3.3V_LP0: supplied by +3.3V_SYS
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
+elan_i2c 1-0015: failed to read report data: -121
 +5V_HDMI_CON: supplied by +5V_SYS
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
  usb3-1: usb3-1 supply vbus not found, using dummy regulator
-elan_i2c 1-0015: failed to read report data: -121
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
-mmc0: new high speed MMC card at address 0001
 sdhci-tegra 700b0000.sdhci: allocated mmc-pwrseq
 mmc1: Unknown controller version (3). You may experience problems.
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: Switching to 3.3V signalling voltage failed
-mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
-mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
-mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
-mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 mmc1: SDHCI controller on 700b0000.sdhci [700b0000.sdhci] using ADMA
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
-mmc2: Unknown controller version (3). You may experience problems.
 elan_i2c 1-0015: invalid report id data (ff)
+mmc2: Unknown controller version (3). You may experience problems.
 mmc2: Invalid maximum block size, assuming 512 bytes
 mmc1: new high speed SDIO card at address 0001
 mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2017-02-22 21:15:27.398145288 -0800
+++ /tmp/foob	2017-02-22 21:15:27.402145304 -0800
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -40,9 +40,9 @@
 	 #################################################################
 	 #################################################################
 	 #######################################################
-	 3.6 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 5566968 (54f1f8 hex)
+Bytes transferred = 5566288 (54ef50 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -53,7 +53,7 @@
 done
 Bytes transferred = 68155 (10a3b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x54f1f8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x54ef50 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a3a
@@ -62,7 +62,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-next-20170222-154547-g9fb8cda (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Feb 21 21:03:38 PST 2017
+Linux version 4.10.0-next-20170223-154574-g27fde84 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Feb 22 21:05:29 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -356,10 +356,10 @@
 mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
-input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input3
 usb 1-2: new high-speed USB device number 2 using tegra-xusb
 mmc2: new high speed SDHC card at address 0007
 mmcblk2: mmc2:0007 SD08G 7.42 GiB 
+input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input3
 GPT:Primary header alternate_lba != Alt. header my_lba
 GPT:3014591 != 15564799
 GPT:Primary header thinks Alt. header is not at the end of the disk.
@@ -370,7 +370,7 @@
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input4
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:23 UTC (946684823)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:22 UTC (946684822)
 +1.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
 +1.05V_RUN_CAM_REAR: disabling
@@ -391,4 +391,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.910239] usb 1-3: new high-speed USB device number 3 using tegra-xusb
+root@(none):/# [    2.910361] usb 1-3: new high-speed USB device number 3 using tegra-xusb
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2017-02-22 21:15:27.406145320 -0800
+++ /tmp/foob	2017-02-22 21:15:27.406145320 -0800
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1800568 us
+Bootloader-Cpu Init at (time stamp): 1800666 us
 
 Pinmux changes applied in kernel way
 
@@ -62,20 +62,20 @@
 Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1800542  us
-Bootloader (AosInit) time: 733494 us
-Bootloader Main Init time: 180172 us
-BlockDevMgrInit time: 157528 us
+Miscellaneous avp time: 1800640  us
+Bootloader (AosInit) time: 733452 us
+Bootloader Main Init time: 180265 us
+BlockDevMgrInit time: 157502 us
 PartMgrInit time: 8 us
-StorMgrInit time: 31 us
+StorMgrInit time: 30 us
 FsMgrInit time: 2 us
-Partition Table load: 77010 us 
-Display init time: 574599 us
+Partition Table load: 77051 us 
+Display init time: 564539 us
 Kernel read time: 378 msec
 Load Kernel Size:   16304128 Bytes
 Bl Kernel ReadSpeed: 43132000 Bytes/sec
-Pre-Kernel Jump time: 1740435 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5842341 us
+Pre-Kernel Jump time: 1740372 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5834394 us
 Booting Linux on physical CPU 0x0
 Linux version 4.8.0-239303-g3477d16 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Fri Oct 7 01:12:37 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
@@ -184,7 +184,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(1.175:1): initialized
+audit: type=2000 audit(1.147:1): initialized
 workingset: timestamp_bits=46 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -254,6 +254,7 @@
 +VDD_LED: supplied by +VDD_MUX
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 +5V_USB_HS: supplied by +5V_SYS
 +5V_USB_SS: supplied by +5V_SYS
 +5V_HDMI_CON: supplied by +5V_SYS
@@ -272,9 +273,6 @@
 Freeing unused kernel memory: 960K (ffff800000c00000 - ffff800000cf0000)
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
-sdhci-tegra 700b0400.sdhci: Got CD GPIO
-sdhci-tegra 700b0400.sdhci: Got WP GPIO
 
 System initialization...
 
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-02-22 21:15:27.414145350 -0800
+++ /tmp/foob	2017-02-22 21:15:27.414145350 -0800
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -47,9 +47,9 @@
 	 #################################################################
 	 #################################################################
 	 ###########################################################
-	 4.1 MiB/s
+	 4 MiB/s
 done
-Bytes transferred = 7545456 (732270 hex)
+Bytes transferred = 7545432 (732258 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -60,7 +60,7 @@
 done
 Bytes transferred = 21426 (53b2 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x732270 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x732258 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103b1
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-next-20170222-154547-g9fb8cda (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Feb 21 21:03:55 PST 2017
+Linux version 4.10.0-next-20170223-154574-g27fde84 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Feb 22 21:06:01 PST 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -331,24 +331,24 @@
 Key type dns_resolver registered
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
-mmc0: host does not support reading read-only switch, assuming write-enable
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
+mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
  mmcblk0: p1
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
-mmc1: new high speed SD card at address b368
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
- mmcblk1: p1 p2
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2000-11-01 00:00:16 UTC (973036816)
+rtc-em3027 2-0056: setting system clock to 2000-11-01 00:00:17 UTC (973036817)
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
+mmc1: new high speed SD card at address b368
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
+ mmcblk1: p1 p2
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2017-02-22 21:15:27.418145366 -0800
+++ /tmp/foob	2017-02-22 21:15:27.422145382 -0800
@@ -45,20 +45,20 @@
 	 #################################################################
 	 #################################################################
 	 #######################################################
-	 4 MiB/s
+	 4.1 MiB/s
 done
-Bytes transferred = 5566968 (54f1f8 hex)
+Bytes transferred = 5566288 (54ef50 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 4.1 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21426 (53b2 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x54f1f8 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x54ef50 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103b1
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-next-20170222-154547-g9fb8cda (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Feb 21 21:03:38 PST 2017
+Linux version 4.10.0-next-20170223-154574-g27fde84 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Feb 22 21:05:29 PST 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -318,9 +318,9 @@
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
 mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
- mmcblk0: p1
 mip6: Mobile IPv6
 sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
+ mmcblk0: p1
 NET: Registered protocol family 17
 NET: Registered protocol family 15
 mmc1: new high speed SD card at address b368
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2017-02-22 21:15:27.426145397 -0800
+++ /tmp/foob	2017-02-22 21:15:27.430145413 -0800
@@ -1,7 +1,7 @@
 [TegraBoot] (version 23.00.2015.14-mobile-1ef66670)
 Processing in cold boot mode Bootloader 2
 A01Q Bootrom Patch rev = 3
-Power-up reason: pmu watchdog
+Power-up reason: on button
 No Battery Present
 Platform has Ddr4 type ram
 max77620 disabling SD1 Remote Sense
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 805446 us
-Basic modules init took 350811 us
-Sec Bootdevice Read Time = 203 ms, Read Size = 8459 KB
-Next stage binary read took 14363 us
-Carveout took 252721 us
-CPU initialization took 93295 us
-Total time taken by TegraBoot 711190 us
+Tegraboot started after 773843 us
+Basic modules init took 348290 us
+Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
+Next stage binary read took 14812 us
+Carveout took 252730 us
+CPU initialization took 92332 us
+Total time taken by TegraBoot 708164 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.646] LPDDR4 Training: Number of tables = 10
+64b[0001.612] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-15340032 bytes read in 805 ms (18.2 MiB/s)
+15340032 bytes read in 803 ms (18.2 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
 44882 bytes read in 461 ms (94.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 495 ms (3.1 MiB/s)
+1627787 bytes read in 496 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-next-20170222-178980-g9fb8cda (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Feb 21 21:06:54 PST 2017
+Linux version 4.10.0-next-20170223-179007-g27fde84 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Feb 22 21:08:23 PST 2017
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -277,7 +277,7 @@
 kvm [1]: vgic interrupt IRQ1
 kvm [1]: virtual timer IRQ4
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.535:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.523:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -357,6 +357,6 @@
 devtmpfs: mounted
 Freeing unused kernel memory: 1088K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
-bash: no job contr[    1.359178] random: fast init done
+bash: no job contr[    1.360198] random: fast init done
 ol in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2017-02-22 21:15:27.434145428 -0800
+++ /tmp/foob	2017-02-22 21:15:27.438145444 -0800
@@ -1,12 +1,4 @@
-:54 PST 2017
-Boot CPU: AArch64 Processor [411fd071]
-efi: Getting EFI parameters from FDT:
-efi: UEFI not found.
-cma: Reserved 16 MiB at 0x00000000f5400000
-earlycon: uart0 at MMIO 0x0000000070006000 (options '115200n8')
-bootconsole [uart0] enabled
-NUMA: No NUMA configuration found
-NUMA: Faking a node at [mem 0x0000000000000000-0x000000013fffffff]
+t [mem 0x0000000000000000-0x000000013fffffff]
 NUMA: Adding memblock [0x80000000 - 0xf65fffff] on node 0
 NUMA: Adding memblock [0x100000000 - 0x13fffffff] on node 0
 NUMA: Initmem setup node 0 [mem 0x80000000-0x13fffffff]
@@ -82,7 +74,7 @@
 console [tty0] enabled
 bootconsole [uart0] disabled
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-next-20170222-178980-g9fb8cda (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Feb 21 21:06:54 PST 2017
+Linux version 4.10.0-next-20170223-179007-g27fde84 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Feb 22 21:08:23 PST 2017
 Boot CPU: AArch64 Processor [411fd071]
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
@@ -243,7 +235,7 @@
 kvm [1]: vgic interrupt IRQ1
 kvm [1]: virtual timer IRQ4
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.327:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.331:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -272,8 +264,8 @@
 PP2800L_RCAM: Bringing 3100000uV into 2800000-2800000uV
 PP1100_SOC_RTC: Bringing 1000000uV into 850000-850000uV
 PP2800L_FCAM: Bringing 3100000uV into 2800000-2800000uV
-PP_LDO6: Bringing 2800000uV into 1800000-1800000uV
 random: fast init done
+PP_LDO6: Bringing 2800000uV into 1800000-1800000uV
 hisi_sas: driver version v1.6
 libphy: Fixed MDIO Bus: probed
 tun: Universal TUN/TAP device driver, 1.6
@@ -333,7 +325,7 @@
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 BGND3R 29.1 GiB 
 mmcblk0boot0: mmc0:0001 BGND3R partition 1 4.00 MiB
-max77686-rtc max77620-rtc: setting system clock to 2000-12-03 10:34:26 UTC (975839666)
+max77686-rtc max77620-rtc: setting system clock to 2000-12-04 10:36:43 UTC (975926203)
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-02-22 21:15:27.442145459 -0800
+++ /tmp/foob	2017-02-22 21:15:27.446145475 -0800
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 ###########################################################
-	 3.3 MiB/s
+	 2.8 MiB/s
 done
-Bytes transferred = 7545456 (732270 hex)
+Bytes transferred = 7545432 (732258 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 51804 (ca5c hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x732270 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x732258 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa5b ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-next-20170222-154547-g9fb8cda (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Feb 21 21:03:55 PST 2017
+Linux version 4.10.0-next-20170223-154574-g27fde84 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Feb 22 21:06:01 PST 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -313,10 +313,10 @@
 mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB
-Primary GPT is invalid, using alternate GPT.
- mmcblk1: p1 p2 p3 p4 p5
 mmc0: new ultra high speed DDR50 SDHC card at address aaaa
+Primary GPT is invalid, using alternate GPT.
 tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
+ mmcblk1: p1 p2 p3 p4 p5
 mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
  mmcblk0: p1 p2 p3
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -396,5 +396,5 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.073599] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.093662] usb 2-1: new high-speed USB device number 2 using tegra-ehci
 asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0f:67:60
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2017-02-22 21:15:27.450145491 -0800
+++ /tmp/foob	2017-02-22 21:15:27.454145507 -0800
@@ -39,18 +39,18 @@
 	 #######################################################
 	 3.3 MiB/s
 done
-Bytes transferred = 5566968 (54f1f8 hex)
+Bytes transferred = 5566288 (54ef50 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.9 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 51804 (ca5c hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x54f1f8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x54ef50 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa5b ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-next-20170222-154547-g9fb8cda (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Feb 21 21:03:38 PST 2017
+Linux version 4.10.0-next-20170223-154574-g27fde84 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Feb 22 21:05:29 PST 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -390,4 +390,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.380048] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.390118] usb 2-1: new high-speed USB device number 2 using tegra-ehci
