Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2015-08-06 00:51:39.217713338 -0700
+++ /tmp/foob	2015-08-06 00:51:39.221713334 -0700
@@ -1,6 +1,6 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
-Linux version 4.2.0-rc5-next-20150805+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Aug 5 01:52:28 PDT 2015
+Linux version 4.2.0-rc5-next-20150806+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Aug 6 00:49:01 PDT 2015
 CPU: AArch64 Processor [411fd070] revision 0
 Detected PIPT I-cache on CPU0
 alternatives: enabling workaround for ARM erratum 832075
@@ -19,7 +19,7 @@
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
 software IO TLB [mem 0x79c00000-0x7dc00000] (64MB) mapped at [ffffffc039c00000-ffffffc03dbfffff]
-Memory: 939144K/1048576K available (5409K kernel code, 478K rwdata, 2252K rodata, 508K init, 204K bss, 93048K reserved, 16384K cma-reserved)
+Memory: 939144K/1048576K available (5409K kernel code, 479K rwdata, 2252K rodata, 508K init, 203K bss, 93048K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     vmalloc : 0xffffff8000000000 - 0xffffffbdbfff0000   (   246 GB)
     vmemmap : 0xffffffbdc0000000 - 0xffffffbfc0000000   (     8 GB maximum)
@@ -30,7 +30,7 @@
     memory  : 0xffffffc000000000 - 0xffffffc040000000   (  1024 MB)
       .init : 0xffffffc0007fe000 - 0xffffffc00087d000   (   508 KB)
       .text : 0xffffffc000080000 - 0xffffffc0007fd534   (  7670 KB)
-      .data : 0xffffffc00088d000 - 0xffffffc000904a00   (   479 KB)
+      .data : 0xffffffc00088d000 - 0xffffffc000904c00   (   479 KB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 64.
@@ -88,7 +88,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 2, 16384 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.430:1): initialized
+audit: type=2000 audit(0.410:1): initialized
 HugeTLB registered 2 MB page size, pre-allocated 0 pages
 VFS: Disk quotas dquot_6.6.0
 VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-08-06 00:51:39.229713329 -0700
+++ /tmp/foob	2015-08-06 00:51:39.229713329 -0700
@@ -40,18 +40,18 @@
 	 #############################################
 	 3.3 MiB/s
 done
-Bytes transferred = 6371336 (613808 hex)
+Bytes transferred = 6371664 (613950 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 2.6 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x613808 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x613950 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc5-next-20150805-34012-ga5b4661 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Aug 5 01:51:10 PDT 2015
+Linux version 4.2.0-rc5-next-20150806-34434-gc6b169e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Aug 6 00:47:45 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -67,7 +67,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f54040, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0f561c0, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -80,7 +80,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999032K/2096128K available (8002K kernel code, 1016K rwdata, 3596K rodata, 1004K init, 340K bss, 31560K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1999024K/2096128K available (8002K kernel code, 1016K rwdata, 3600K rodata, 1008K init, 340K bss, 31568K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,10 +88,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d5c8f8   (11603 kB)
-      .init : 0xc0d5d000 - 0xc0e58000   (1004 kB)
-      .data : 0xc0e58000 - 0xc0f56040   (1017 kB)
-       .bss : 0xc0f59000 - 0xc0fae268   ( 341 kB)
+      .text : 0xc0208000 - 0xc0d5d9b4   (11607 kB)
+      .init : 0xc0d5e000 - 0xc0e5a000   (1008 kB)
+      .data : 0xc0e5a000 - 0xc0f58180   (1017 kB)
+       .bss : 0xc0f5b000 - 0xc0fb0268   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -296,7 +296,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-15 17:44:18 UTC (950636658)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-16 16:40:58 UTC (950719258)
 vdd_cam_1v8_reg: disabling
 usb1_vbus: disabling
 dvdd_ts: disabling
@@ -305,7 +305,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1004K (c0d5d000 - c0e58000)
+Freeing unused kernel memory: 1008K (c0d5e000 - c0e5a000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.204128] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.198915] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2015-08-06 00:51:39.237713322 -0700
+++ /tmp/foob	2015-08-06 00:51:39.237713322 -0700
@@ -36,21 +36,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##################
+	 ###################
 	 3.2 MiB/s
 done
-Bytes transferred = 5035184 (4cd4b0 hex)
+Bytes transferred = 5036576 (4cda20 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.8 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4cd4b0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4cda20 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc5-next-20150805-34012-ga5b4661 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Aug 5 01:51:41 PDT 2015
+Linux version 4.2.0-rc5-next-20150806-34434-gc6b169e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Aug 6 00:48:18 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c09b2700, node_mem_map ee7f7000
+free_area_init_node: node 0, pgdat c09b47c0, node_mem_map ee7f7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002724K/2096128K available (6868K kernel code, 344K rwdata, 2376K rodata, 312K init, 363K bss, 27868K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 2002716K/2096128K available (6868K kernel code, 344K rwdata, 2380K rodata, 316K init, 363K bss, 27876K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc090f58c   (9246 kB)
-      .init : 0xc0910000 - 0xc095e000   ( 312 kB)
-      .data : 0xc095e000 - 0xc09b42c0   ( 345 kB)
-       .bss : 0xc09b7000 - 0xc0a11da0   ( 364 kB)
+      .text : 0xc0008000 - 0xc09105dc   (9250 kB)
+      .init : 0xc0911000 - 0xc0960000   ( 316 kB)
+      .data : 0xc0960000 - 0xc09b6340   ( 345 kB)
+       .bss : 0xc09b9000 - 0xc0a13da0   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -292,7 +292,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-15 17:44:56 UTC (950636696)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-16 16:41:31 UTC (950719291)
 vdd_cam_1v8_reg: disabling
 usb1_vbus: disabling
 dvdd_ts: disabling
@@ -304,7 +304,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 312K (c0910000 - c095e000)
+Freeing unused kernel memory: 316K (c0911000 - c0960000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.749949] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.730013] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-08-06 00:51:39.245713315 -0700
+++ /tmp/foob	2015-08-06 00:51:39.245713315 -0700
@@ -40,20 +40,20 @@
 	 #################################################################
 	 #################################################################
 	 #############################################
-	 3.2 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 6371336 (613808 hex)
+Bytes transferred = 6371664 (613950 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *#####
-	 2.4 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x613808 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x613950 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012a89
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc5-next-20150805-34012-ga5b4661 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Aug 5 01:51:10 PDT 2015
+Linux version 4.2.0-rc5-next-20150806-34434-gc6b169e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Aug 6 00:47:45 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f54040, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0f561c0, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998924K/2096128K available (8002K kernel code, 1016K rwdata, 3596K rodata, 1004K init, 340K bss, 31668K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1998916K/2096128K available (8002K kernel code, 1016K rwdata, 3600K rodata, 1008K init, 340K bss, 31676K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d5c8f8   (11603 kB)
-      .init : 0xc0d5d000 - 0xc0e58000   (1004 kB)
-      .data : 0xc0e58000 - 0xc0f56040   (1017 kB)
-       .bss : 0xc0f59000 - 0xc0fae268   ( 341 kB)
+      .text : 0xc0208000 - 0xc0d5d9b4   (11607 kB)
+      .init : 0xc0d5e000 - 0xc0e5a000   (1008 kB)
+      .data : 0xc0e5a000 - 0xc0f58180   (1017 kB)
+       .bss : 0xc0f5b000 - 0xc0fb0268   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -291,6 +291,7 @@
 +1.05V_RUN_AVDD: supplied by +1.35V_LP0(sd2)
 mmc1: MAN_BKOPS_EN bit is not set
 tegra-pcie 1003000.pcie-controller: probing port 0, using 2 lanes
+tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
 mmc1: new high speed MMC card at address 0001
 mmcblk0: mmc1:0001 SEM16G 14.6 GiB 
 mmcblk0boot0: mmc1:0001 SEM16G partition 1 4.00 MiB
@@ -298,7 +299,6 @@
 mmcblk0rpmb: mmc1:0001 SEM16G partition 3 4.00 MiB
 Primary GPT is invalid, using alternate GPT.
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9
-tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
 tegra-pcie 1003000.pcie-controller: link 0 down, retrying
 tegra-pcie 1003000.pcie-controller: link 0 down, retrying
 tegra-pcie 1003000.pcie-controller: link 0 down, retrying
@@ -355,12 +355,12 @@
 ata1: SATA link down (SStatus 0 SControl 300)
 EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
 EXT4-fs (mmcblk0p1): couldn't mount as ext2 due to feature incompatibilities
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1004K (c0d5d000 - c0e58000)
+Freeing unused kernel memory: 1008K (c0d5e000 - c0e5a000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2015-08-06 00:51:39.253713308 -0700
+++ /tmp/foob	2015-08-06 00:51:39.253713308 -0700
@@ -38,10 +38,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##################
-	 3.1 MiB/s
+	 ###################
+	 3.2 MiB/s
 done
-Bytes transferred = 5035184 (4cd4b0 hex)
+Bytes transferred = 5036576 (4cda20 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -52,7 +52,7 @@
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4cd4b0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4cda20 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012a89
@@ -63,7 +63,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc5-next-20150805-34012-ga5b4661 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Aug 5 01:51:41 PDT 2015
+Linux version 4.2.0-rc5-next-20150806-34434-gc6b169e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Aug 6 00:48:18 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -71,7 +71,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c09b2700, node_mem_map ee7f7000
+free_area_init_node: node 0, pgdat c09b47c0, node_mem_map ee7f7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -84,7 +84,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002616K/2096128K available (6868K kernel code, 344K rwdata, 2376K rodata, 312K init, 363K bss, 27976K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 2002608K/2096128K available (6868K kernel code, 344K rwdata, 2380K rodata, 316K init, 363K bss, 27984K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,10 +92,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc090f58c   (9246 kB)
-      .init : 0xc0910000 - 0xc095e000   ( 312 kB)
-      .data : 0xc095e000 - 0xc09b42c0   ( 345 kB)
-       .bss : 0xc09b7000 - 0xc0a11da0   ( 364 kB)
+      .text : 0xc0008000 - 0xc09105dc   (9250 kB)
+      .init : 0xc0911000 - 0xc0960000   ( 316 kB)
+      .data : 0xc0960000 - 0xc09b6340   ( 345 kB)
+       .bss : 0xc09b9000 - 0xc0a13da0   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -340,7 +340,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-02-01 00:00:54 UTC (949363254)
+as3722-rtc as3722-rtc: setting system clock to 2000-02-01 00:00:19 UTC (949363219)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_VPP_FUSE: disabling
 +2.8V_RUN_CAM_AF: disabling
@@ -352,15 +352,15 @@
 ALSA device list:
   #0: tegra-hda at 0x70038000 irq 342
   #1: NVIDIA Tegra Jetson TK1
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 ata1: SATA link down (SStatus 0 SControl 300)
 EXT2-fs (mmcblk0p1): error: couldn't mount because of unsupported optional features (244)
 EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 312K (c0910000 - c095e000)
+Freeing unused kernel memory: 316K (c0911000 - c0960000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-08-06 00:51:39.261713301 -0700
+++ /tmp/foob	2015-08-06 00:51:39.261713301 -0700
@@ -48,7 +48,7 @@
 	 #############################################
 	 4.1 MiB/s
 done
-Bytes transferred = 6371336 (613808 hex)
+Bytes transferred = 6371664 (613950 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -59,7 +59,7 @@
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x613808 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x613950 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -67,7 +67,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc5-next-20150805-34012-ga5b4661 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Aug 5 01:51:10 PDT 2015
+Linux version 4.2.0-rc5-next-20150806-34434-gc6b169e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Aug 6 00:47:45 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -75,7 +75,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0f54040, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c0f561c0, node_mem_map eeff8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -88,7 +88,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 959840K/1048576K available (8002K kernel code, 1016K rwdata, 3596K rodata, 1004K init, 340K bss, 23200K reserved, 65536K cma-reserved, 204800K highmem)
+Memory: 959832K/1048576K available (8002K kernel code, 1016K rwdata, 3600K rodata, 1008K init, 340K bss, 23208K reserved, 65536K cma-reserved, 204800K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -96,10 +96,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d5c8f8   (11603 kB)
-      .init : 0xc0d5d000 - 0xc0e58000   (1004 kB)
-      .data : 0xc0e58000 - 0xc0f56040   (1017 kB)
-       .bss : 0xc0f59000 - 0xc0fae268   ( 341 kB)
+      .text : 0xc0208000 - 0xc0d5d9b4   (11607 kB)
+      .init : 0xc0d5e000 - 0xc0e5a000   (1008 kB)
+      .data : 0xc0e5a000 - 0xc0f58180   (1017 kB)
+       .bss : 0xc0f5b000 - 0xc0fb0268   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -191,6 +191,8 @@
 pci 0000:00:01.0: PME# supported from D0 D1 D2 D3hot D3cold
 PCI: bus0: Fast back to back transfers disabled
 pci 0000:00:01.0: bridge configuration invalid ([bus 00-00]), reconfiguring
+pci 0000:00:01.0: PCI bridge to [bus 01-ff]
+pci 0000:00:01.0:   bridge window [io  0x0000-0x0fff]
 pci 0000:01:00.0: [10ec:8168] type 00 class 0x020000
 pci 0000:01:00.0: reg 0x10: [io  0x0000-0x00ff]
 pci 0000:01:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit pref]
@@ -323,26 +325,27 @@
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
-tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 mmc0: host does not support reading read-only switch, assuming write-enable
-hub 2-0:1.0: USB hub found
-hub 2-0:1.0: 1 port detected
 mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
-input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-26 20:19:02 UTC (1419625142)
  mmcblk0: p1
-EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
+tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
+hub 2-0:1.0: USB hub found
+hub 2-0:1.0: 1 port detected
+input: gpio-keys as /devices/soc0/gpio-keys/input/input0
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
+rtc-em3027 2-0056: setting system clock to 2014-12-26 20:19:39 UTC (1419625179)
+EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
+mmc1: new high speed SD card at address b368
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
+ mmcblk1: p1 p2
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1004K (c0d5d000 - c0e58000)
-mmc1: new high speed SD card at address b368
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
- mmcblk1: p1 p2
+Freeing unused kernel memory: 1008K (c0d5e000 - c0e5a000)
+usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.417113] usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
+root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2015-08-06 00:51:39.269713294 -0700
+++ /tmp/foob	2015-08-06 00:51:39.269713294 -0700
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -44,10 +44,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##################
+	 ###################
 	 4.1 MiB/s
 done
-Bytes transferred = 5035184 (4cd4b0 hex)
+Bytes transferred = 5036576 (4cda20 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x4cd4b0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x4cda20 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -69,7 +69,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc5-next-20150805-34012-ga5b4661 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Aug 5 01:51:41 PDT 2015
+Linux version 4.2.0-rc5-next-20150806-34434-gc6b169e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Aug 6 00:48:18 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -78,7 +78,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c09b2700, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c09b47c0, node_mem_map eeff7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -91,7 +91,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 963532K/1048576K available (6868K kernel code, 344K rwdata, 2376K rodata, 312K init, 363K bss, 19508K reserved, 65536K cma-reserved, 204800K highmem)
+Memory: 963524K/1048576K available (6868K kernel code, 344K rwdata, 2380K rodata, 316K init, 363K bss, 19516K reserved, 65536K cma-reserved, 204800K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -99,10 +99,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc090f58c   (9246 kB)
-      .init : 0xc0910000 - 0xc095e000   ( 312 kB)
-      .data : 0xc095e000 - 0xc09b42c0   ( 345 kB)
-       .bss : 0xc09b7000 - 0xc0a11da0   ( 364 kB)
+      .text : 0xc0008000 - 0xc09105dc   (9250 kB)
+      .init : 0xc0911000 - 0xc0960000   ( 316 kB)
+      .data : 0xc0960000 - 0xc09b6340   ( 345 kB)
+       .bss : 0xc09b9000 - 0xc0a13da0   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
@@ -194,6 +194,8 @@
 pci 0000:00:01.0: PME# supported from D0 D1 D2 D3hot D3cold
 PCI: bus0: Fast back to back transfers disabled
 pci 0000:00:01.0: bridge configuration invalid ([bus 00-00]), reconfiguring
+pci 0000:00:01.0: PCI bridge to [bus 01-ff]
+pci 0000:00:01.0:   bridge window [io  0x0000-0x0fff]
 pci 0000:01:00.0: [10ec:8168] type 00 class 0x020000
 pci 0000:01:00.0: reg 0x10: [io  0x0000-0x00ff]
 pci 0000:01:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit pref]
@@ -341,7 +343,7 @@
 hub 2-0:1.0: 1 port detected
 hub 1-1:1.0: USB hub found
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-26 20:19:21 UTC (1419625161)
+rtc-em3027 2-0056: setting system clock to 2014-12-26 20:19:58 UTC (1419625198)
 ALSA device list:
   #0: tegra-trimslice
 hub 1-1:1.0: 4 ports detected
@@ -354,7 +356,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 312K (c0910000 - c095e000)
+Freeing unused kernel memory: 316K (c0911000 - c0960000)
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-08-06 00:51:39.273713291 -0700
+++ /tmp/foob	2015-08-06 00:51:39.277713287 -0700
@@ -38,9 +38,9 @@
 	 #################################################################
 	 #################################################################
 	 #############################################
-	 3.4 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 6371336 (613808 hex)
+Bytes transferred = 6371664 (613950 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -51,7 +51,7 @@
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x613808 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x613950 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc5-next-20150805-34012-ga5b4661 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Aug 5 01:51:10 PDT 2015
+Linux version 4.2.0-rc5-next-20150806-34434-gc6b169e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Aug 6 00:47:45 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -67,7 +67,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f54040, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0f561c0, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -80,7 +80,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998952K/2096128K available (8002K kernel code, 1016K rwdata, 3596K rodata, 1004K init, 340K bss, 31640K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1998944K/2096128K available (8002K kernel code, 1016K rwdata, 3600K rodata, 1008K init, 340K bss, 31648K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,10 +88,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d5c8f8   (11603 kB)
-      .init : 0xc0d5d000 - 0xc0e58000   (1004 kB)
-      .data : 0xc0e58000 - 0xc0f56040   (1017 kB)
-       .bss : 0xc0f59000 - 0xc0fae268   ( 341 kB)
+      .text : 0xc0208000 - 0xc0d5d9b4   (11607 kB)
+      .init : 0xc0d5e000 - 0xc0e5a000   (1008 kB)
+      .data : 0xc0e5a000 - 0xc0f58180   (1017 kB)
+       .bss : 0xc0f5b000 - 0xc0fb0268   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -306,6 +306,8 @@
 pci 0000:00:01.0: PME# supported from D0 D1 D2 D3hot D3cold
 PCI: bus0: Fast back to back transfers disabled
 pci 0000:00:01.0: bridge configuration invalid ([bus 00-00]), reconfiguring
+pci 0000:00:01.0: PCI bridge to [bus 01-ff]
+pci 0000:00:01.0:   bridge window [io  0x0000-0x0fff]
 pci 0000:01:00.0: [10ec:8168] type 00 class 0x020000
 pci 0000:01:00.0: reg 0x10: [io  0x0000-0x00ff]
 pci 0000:01:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit pref]
@@ -361,7 +363,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1004K (c0d5d000 - c0e58000)
+Freeing unused kernel memory: 1008K (c0d5e000 - c0e5a000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.738817] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.748787] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2015-08-06 00:51:39.281713284 -0700
+++ /tmp/foob	2015-08-06 00:51:39.285713280 -0700
@@ -36,21 +36,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##################
+	 ###################
 	 3.3 MiB/s
 done
-Bytes transferred = 5035184 (4cd4b0 hex)
+Bytes transferred = 5036576 (4cda20 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.3 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4cd4b0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4cda20 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc5-next-20150805-34012-ga5b4661 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Aug 5 01:51:41 PDT 2015
+Linux version 4.2.0-rc5-next-20150806-34434-gc6b169e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Aug 6 00:48:18 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c09b2700, node_mem_map ee7f7000
+free_area_init_node: node 0, pgdat c09b47c0, node_mem_map ee7f7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002644K/2096128K available (6868K kernel code, 344K rwdata, 2376K rodata, 312K init, 363K bss, 27948K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 2002636K/2096128K available (6868K kernel code, 344K rwdata, 2380K rodata, 316K init, 363K bss, 27956K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc090f58c   (9246 kB)
-      .init : 0xc0910000 - 0xc095e000   ( 312 kB)
-      .data : 0xc095e000 - 0xc09b42c0   ( 345 kB)
-       .bss : 0xc09b7000 - 0xc0a11da0   ( 364 kB)
+      .text : 0xc0008000 - 0xc09105dc   (9250 kB)
+      .init : 0xc0911000 - 0xc0960000   ( 316 kB)
+      .data : 0xc0960000 - 0xc09b6340   ( 345 kB)
+       .bss : 0xc09b9000 - 0xc0a13da0   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -303,6 +303,8 @@
 pci 0000:00:01.0: PME# supported from D0 D1 D2 D3hot D3cold
 PCI: bus0: Fast back to back transfers disabled
 pci 0000:00:01.0: bridge configuration invalid ([bus 00-00]), reconfiguring
+pci 0000:00:01.0: PCI bridge to [bus 01-ff]
+pci 0000:00:01.0:   bridge window [io  0x0000-0x0fff]
 pci 0000:01:00.0: [10ec:8168] type 00 class 0x020000
 pci 0000:01:00.0: reg 0x10: [io  0x0000-0x00ff]
 pci 0000:01:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit pref]
@@ -361,7 +363,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 312K (c0910000 - c095e000)
+Freeing unused kernel memory: 316K (c0911000 - c0960000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.386445] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.396426] usb 2-1: new high-speed USB device number 2 using tegra-ehci
