Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-05-04 23:56:04.724260055 -0700
+++ /tmp/foob	2016-05-04 23:56:04.728260056 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc6-next-20160504+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed May 4 00:35:49 PDT 2016
+Linux version 4.6.0-rc6-next-20160505+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed May 4 23:50:54 PDT 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -98,7 +98,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.536:1): initialized
+audit: type=2000 audit(0.600:1): initialized
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -158,7 +158,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2016-05-04 07:36:21 UTC (1462347381)
+rtc-pl031 9010000.pl031: setting system clock to 2016-05-05 06:51:26 UTC (1462431086)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-05-04 23:56:04.748260064 -0700
+++ /tmp/foob	2016-05-04 23:56:04.748260064 -0700
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 ##########################
-	 3.2 MiB/s
+	 3 MiB/s
 done
-Bytes transferred = 7056048 (6baab0 hex)
+Bytes transferred = 7055656 (6ba928 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.8 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 34956 (888c hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6baab0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6ba928 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff88b ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc6-next-20160504-92291-ga57cdca (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed May 4 00:33:38 PDT 2016
+Linux version 4.6.0-rc6-next-20160505-92543-ge3c95c2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed May 4 23:48:59 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c1417e00, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1417fc0, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1995132K/2096128K available (9528K kernel code, 1127K rwdata, 4188K rodata, 2048K init, 342K bss, 35460K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1995128K/2096128K available (9528K kernel code, 1128K rwdata, 4188K rodata, 2048K init, 343K bss, 35464K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc10651ec   (14709 kB)
+      .text : 0xc0208000 - 0xc1065268   (14709 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc1419e60   (1128 kB)
-       .bss : 0xc141b000 - 0xc1470bd8   ( 343 kB)
+      .data : 0xc1300000 - 0xc141a020   (1129 kB)
+       .bss : 0xc141c000 - 0xc1471c58   ( 344 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -260,27 +260,27 @@
 ehci-atmel: EHCI Atmel driver
 tegra-ehci: Tegra EHCI driver
 ------------[ cut here ]------------
-WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
+WARNING: CPU: 1 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc6-next-20160504-92291-ga57cdca #1
+CPU: 1 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc6-next-20160505-92543-ge3c95c2 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c030f9a0>] (unwind_backtrace) from [<c030b090>] (show_stack+0x10/0x14)
-[<c030b090>] (show_stack) from [<c0585828>] (dump_stack+0x84/0xa4)
-[<c0585828>] (dump_stack) from [<c0341b6c>] (__warn+0xd4/0x100)
+[<c030b090>] (show_stack) from [<c0585868>] (dump_stack+0x84/0xa4)
+[<c0585868>] (dump_stack) from [<c0341b6c>] (__warn+0xd4/0x100)
 [<c0341b6c>] (__warn) from [<c0341bb8>] (warn_slowpath_null+0x20/0x28)
-[<c0341bb8>] (warn_slowpath_null) from [<c069df60>] (__of_reset_control_get+0x1bc/0x228)
-[<c069df60>] (__of_reset_control_get) from [<c090fc64>] (tegra_ehci_probe+0x2d0/0x51c)
-[<c090fc64>] (tegra_ehci_probe) from [<c0777d6c>] (platform_drv_probe+0x50/0xb0)
-[<c0777d6c>] (platform_drv_probe) from [<c07766d0>] (driver_probe_device+0x1f4/0x2b0)
-[<c07766d0>] (driver_probe_device) from [<c0776838>] (__driver_attach+0xac/0xb0)
-[<c0776838>] (__driver_attach) from [<c0774ce4>] (bus_for_each_dev+0x54/0x88)
-[<c0774ce4>] (bus_for_each_dev) from [<c0775cc4>] (bus_add_driver+0xe8/0x1f4)
-[<c0775cc4>] (bus_add_driver) from [<c0777054>] (driver_register+0x78/0xf4)
-[<c0777054>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
+[<c0341bb8>] (warn_slowpath_null) from [<c069dfa0>] (__of_reset_control_get+0x1bc/0x228)
+[<c069dfa0>] (__of_reset_control_get) from [<c090e780>] (tegra_ehci_probe+0x2d0/0x51c)
+[<c090e780>] (tegra_ehci_probe) from [<c0776798>] (platform_drv_probe+0x50/0xb0)
+[<c0776798>] (platform_drv_probe) from [<c07750fc>] (driver_probe_device+0x1f4/0x2b0)
+[<c07750fc>] (driver_probe_device) from [<c0775264>] (__driver_attach+0xac/0xb0)
+[<c0775264>] (__driver_attach) from [<c0773710>] (bus_for_each_dev+0x54/0x88)
+[<c0773710>] (bus_for_each_dev) from [<c07746f0>] (bus_add_driver+0xe8/0x1f4)
+[<c07746f0>] (bus_add_driver) from [<c0775a80>] (driver_register+0x78/0xf4)
+[<c0775a80>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
 [<c0301e5c>] (do_one_initcall) from [<c1100d80>] (kernel_init_freeable+0x120/0x1ec)
-[<c1100d80>] (kernel_init_freeable) from [<c0b5813c>] (kernel_init+0x8/0x118)
-[<c0b5813c>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
----[ end trace 68fb5b30411ea7d0 ]---
+[<c1100d80>] (kernel_init_freeable) from [<c0b57138>] (kernel_init+0x8/0x118)
+[<c0b57138>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
+---[ end trace 7bd323041c8f5ed1 ]---
 tegra-ehci 7d008000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d008000.usb: continuing, but please update your DT
 ohci_hcd: USB 1.1 'Open' Host Controller (OHCI) Driver
@@ -325,13 +325,13 @@
 usb1_vbus: supplied by vdd-sys-5v0
 usb2_vbus: supplied by vdd-sys-5v0
 VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
-[drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 mmc0: new high speed SDHC card at address aaaa
-[drm] No driver support for vblank timestamp query.
+[drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
+[drm] No driver support for vblank timestamp query.
+ mmcblk0: p1 p2 p3
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
- mmcblk0: p1 p2 p3
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
@@ -346,7 +346,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-26 18:52:38 UTC (951591158)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-27 18:08:03 UTC (951674883)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -358,4 +358,5 @@
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.530143] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-05-04 23:56:04.752260065 -0700
+++ /tmp/foob	2016-05-04 23:56:04.756260067 -0700
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 #########################################
-	 3 MiB/s
+	 3.1 MiB/s
 done
-Bytes transferred = 5371024 (51f490 hex)
+Bytes transferred = 5369800 (51efc8 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.6 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 34956 (888c hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x51f490 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x51efc8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff88b ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc6-next-20160504-92291-ga57cdca (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed May 4 00:33:39 PDT 2016
+Linux version 4.6.0-rc6-next-20160505-92543-ge3c95c2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed May 4 23:48:57 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -90,7 +90,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0abce88   (10964 kB)
+      .text : 0xc0008000 - 0xc0abcee8   (10964 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
       .data : 0xc0c00000 - 0xc0c5d8c0   ( 375 kB)
        .bss : 0xc0c5f000 - 0xc0cbaad0   ( 367 kB)
@@ -252,25 +252,25 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1b4/0x218
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc6-next-20160504-92291-ga57cdca #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc6-next-20160505-92543-ge3c95c2 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c010f1f8>] (unwind_backtrace) from [<c010a86c>] (show_stack+0x10/0x14)
-[<c010a86c>] (show_stack) from [<c033d034>] (dump_stack+0x84/0xa4)
-[<c033d034>] (dump_stack) from [<c011f2c8>] (__warn+0xd4/0x100)
-[<c011f2c8>] (__warn) from [<c011f314>] (warn_slowpath_null+0x20/0x28)
-[<c011f314>] (warn_slowpath_null) from [<c03c0150>] (__of_reset_control_get+0x1b4/0x218)
-[<c03c0150>] (__of_reset_control_get) from [<c05121f8>] (tegra_ehci_probe+0x2c8/0x514)
-[<c05121f8>] (tegra_ehci_probe) from [<c04409c8>] (platform_drv_probe+0x50/0xb0)
-[<c04409c8>] (platform_drv_probe) from [<c043f3f4>] (driver_probe_device+0x1f4/0x2b0)
-[<c043f3f4>] (driver_probe_device) from [<c043f55c>] (__driver_attach+0xac/0xb0)
-[<c043f55c>] (__driver_attach) from [<c043da18>] (bus_for_each_dev+0x54/0x88)
-[<c043da18>] (bus_for_each_dev) from [<c043e9e8>] (bus_add_driver+0xe8/0x1f4)
-[<c043e9e8>] (bus_add_driver) from [<c043fd58>] (driver_register+0x78/0xf4)
-[<c043fd58>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
+[<c010a86c>] (show_stack) from [<c033d0b4>] (dump_stack+0x84/0xa4)
+[<c033d0b4>] (dump_stack) from [<c011f308>] (__warn+0xd4/0x100)
+[<c011f308>] (__warn) from [<c011f354>] (warn_slowpath_null+0x20/0x28)
+[<c011f354>] (warn_slowpath_null) from [<c03c01d0>] (__of_reset_control_get+0x1b4/0x218)
+[<c03c01d0>] (__of_reset_control_get) from [<c0511afc>] (tegra_ehci_probe+0x2c8/0x514)
+[<c0511afc>] (tegra_ehci_probe) from [<c0440290>] (platform_drv_probe+0x50/0xb0)
+[<c0440290>] (platform_drv_probe) from [<c043ecbc>] (driver_probe_device+0x1f4/0x2b0)
+[<c043ecbc>] (driver_probe_device) from [<c043ee24>] (__driver_attach+0xac/0xb0)
+[<c043ee24>] (__driver_attach) from [<c043d2e0>] (bus_for_each_dev+0x54/0x88)
+[<c043d2e0>] (bus_for_each_dev) from [<c043e2b0>] (bus_add_driver+0xe8/0x1f4)
+[<c043e2b0>] (bus_add_driver) from [<c043f620>] (driver_register+0x78/0xf4)
+[<c043f620>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
 [<c010179c>] (do_one_initcall) from [<c0b00d94>] (kernel_init_freeable+0x11c/0x1e8)
-[<c0b00d94>] (kernel_init_freeable) from [<c07eec84>] (kernel_init+0x8/0x118)
-[<c07eec84>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
----[ end trace 8daee6fe79a98f93 ]---
+[<c0b00d94>] (kernel_init_freeable) from [<c07eea6c>] (kernel_init+0x8/0x118)
+[<c07eea6c>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
+---[ end trace d015c4b9da1905d6 ]---
 tegra-ehci 7d008000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d008000.usb: continuing, but please update your DT
 usbcore: registered new interface driver cdc_acm
@@ -344,7 +344,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-26 18:55:13 UTC (951591313)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-27 18:10:41 UTC (951675041)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -356,7 +356,7 @@
 VFS: Mounted root (ext3 filesystem) on device 179:17.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-05-04 23:56:04.764260071 -0700
+++ /tmp/foob	2016-05-04 23:56:04.764260071 -0700
@@ -36,7 +36,7 @@
 	 ##########################
 	 4 MiB/s
 done
-Bytes transferred = 7056048 (6baab0 hex)
+Bytes transferred = 7055656 (6ba928 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -47,7 +47,7 @@
 done
 Bytes transferred = 66737 (104b1 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6baab0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x6ba928 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830134b0
@@ -55,7 +55,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc6-next-20160504-92291-ga57cdca (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed May 4 00:33:38 PDT 2016
+Linux version 4.6.0-rc6-next-20160505-92543-ge3c95c2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed May 4 23:48:59 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -64,7 +64,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c1417e00, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1417fc0, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -77,7 +77,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1995012K/2096128K available (9528K kernel code, 1127K rwdata, 4188K rodata, 2048K init, 342K bss, 35580K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1995008K/2096128K available (9528K kernel code, 1128K rwdata, 4188K rodata, 2048K init, 343K bss, 35584K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -85,10 +85,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc10651ec   (14709 kB)
+      .text : 0xc0208000 - 0xc1065268   (14709 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc1419e60   (1128 kB)
-       .bss : 0xc141b000 - 0xc1470bd8   ( 343 kB)
+      .data : 0xc1300000 - 0xc141a020   (1129 kB)
+       .bss : 0xc141c000 - 0xc1471c58   ( 344 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -252,27 +252,27 @@
 ehci-atmel: EHCI Atmel driver
 tegra-ehci: Tegra EHCI driver
 ------------[ cut here ]------------
-WARNING: CPU: 3 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
+WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
 Modules linked in:
-CPU: 3 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc6-next-20160504-92291-ga57cdca #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc6-next-20160505-92543-ge3c95c2 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c030f9a0>] (unwind_backtrace) from [<c030b090>] (show_stack+0x10/0x14)
-[<c030b090>] (show_stack) from [<c0585828>] (dump_stack+0x84/0xa4)
-[<c0585828>] (dump_stack) from [<c0341b6c>] (__warn+0xd4/0x100)
+[<c030b090>] (show_stack) from [<c0585868>] (dump_stack+0x84/0xa4)
+[<c0585868>] (dump_stack) from [<c0341b6c>] (__warn+0xd4/0x100)
 [<c0341b6c>] (__warn) from [<c0341bb8>] (warn_slowpath_null+0x20/0x28)
-[<c0341bb8>] (warn_slowpath_null) from [<c069df60>] (__of_reset_control_get+0x1bc/0x228)
-[<c069df60>] (__of_reset_control_get) from [<c090fc64>] (tegra_ehci_probe+0x2d0/0x51c)
-[<c090fc64>] (tegra_ehci_probe) from [<c0777d6c>] (platform_drv_probe+0x50/0xb0)
-[<c0777d6c>] (platform_drv_probe) from [<c07766d0>] (driver_probe_device+0x1f4/0x2b0)
-[<c07766d0>] (driver_probe_device) from [<c0776838>] (__driver_attach+0xac/0xb0)
-[<c0776838>] (__driver_attach) from [<c0774ce4>] (bus_for_each_dev+0x54/0x88)
-[<c0774ce4>] (bus_for_each_dev) from [<c0775cc4>] (bus_add_driver+0xe8/0x1f4)
-[<c0775cc4>] (bus_add_driver) from [<c0777054>] (driver_register+0x78/0xf4)
-[<c0777054>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
+[<c0341bb8>] (warn_slowpath_null) from [<c069dfa0>] (__of_reset_control_get+0x1bc/0x228)
+[<c069dfa0>] (__of_reset_control_get) from [<c090e780>] (tegra_ehci_probe+0x2d0/0x51c)
+[<c090e780>] (tegra_ehci_probe) from [<c0776798>] (platform_drv_probe+0x50/0xb0)
+[<c0776798>] (platform_drv_probe) from [<c07750fc>] (driver_probe_device+0x1f4/0x2b0)
+[<c07750fc>] (driver_probe_device) from [<c0775264>] (__driver_attach+0xac/0xb0)
+[<c0775264>] (__driver_attach) from [<c0773710>] (bus_for_each_dev+0x54/0x88)
+[<c0773710>] (bus_for_each_dev) from [<c07746f0>] (bus_add_driver+0xe8/0x1f4)
+[<c07746f0>] (bus_add_driver) from [<c0775a80>] (driver_register+0x78/0xf4)
+[<c0775a80>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
 [<c0301e5c>] (do_one_initcall) from [<c1100d80>] (kernel_init_freeable+0x120/0x1ec)
-[<c1100d80>] (kernel_init_freeable) from [<c0b5813c>] (kernel_init+0x8/0x118)
-[<c0b5813c>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
----[ end trace a4628697ec21f3b2 ]---
+[<c1100d80>] (kernel_init_freeable) from [<c0b57138>] (kernel_init+0x8/0x118)
+[<c0b57138>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
+---[ end trace c98059af8a3f3080 ]---
 tegra-ehci 7d004000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d004000.usb: continuing, but please update your DT
 tegra-ehci 7d004000.usb: EHCI Host Controller
@@ -401,8 +401,8 @@
 PCI: bus1: Fast back to back transfers disabled
 pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
 pci 0000:02:00.0: [10ec:8168] type 00 class 0x020000
-pci 0000:02:00.0: reg 0x10: [io  0x0000-0x00ff]
 ata1: SATA link down (SStatus 0 SControl 300)
+pci 0000:02:00.0: reg 0x10: [io  0x0000-0x00ff]
 pci 0000:02:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit]
 pci 0000:02:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
 pci 0000:02:00.0: supports D1 D2
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-05-04 23:56:04.772260073 -0700
+++ /tmp/foob	2016-05-04 23:56:04.772260073 -0700
@@ -32,20 +32,20 @@
 	 #################################################################
 	 #################################################################
 	 #########################################
-	 3.3 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 5371024 (51f490 hex)
+Bytes transferred = 5369800 (51efc8 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.2 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 66737 (104b1 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x51f490 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x51efc8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830134b0
@@ -54,7 +54,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc6-next-20160504-92291-ga57cdca (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed May 4 00:33:39 PDT 2016
+Linux version 4.6.0-rc6-next-20160505-92543-ge3c95c2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed May 4 23:48:57 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -85,7 +85,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0abce88   (10964 kB)
+      .text : 0xc0008000 - 0xc0abcee8   (10964 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
       .data : 0xc0c00000 - 0xc0c5d8c0   ( 375 kB)
        .bss : 0xc0c5f000 - 0xc0cbaad0   ( 367 kB)
@@ -244,25 +244,25 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1b4/0x218
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc6-next-20160504-92291-ga57cdca #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc6-next-20160505-92543-ge3c95c2 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c010f1f8>] (unwind_backtrace) from [<c010a86c>] (show_stack+0x10/0x14)
-[<c010a86c>] (show_stack) from [<c033d034>] (dump_stack+0x84/0xa4)
-[<c033d034>] (dump_stack) from [<c011f2c8>] (__warn+0xd4/0x100)
-[<c011f2c8>] (__warn) from [<c011f314>] (warn_slowpath_null+0x20/0x28)
-[<c011f314>] (warn_slowpath_null) from [<c03c0150>] (__of_reset_control_get+0x1b4/0x218)
-[<c03c0150>] (__of_reset_control_get) from [<c05121f8>] (tegra_ehci_probe+0x2c8/0x514)
-[<c05121f8>] (tegra_ehci_probe) from [<c04409c8>] (platform_drv_probe+0x50/0xb0)
-[<c04409c8>] (platform_drv_probe) from [<c043f3f4>] (driver_probe_device+0x1f4/0x2b0)
-[<c043f3f4>] (driver_probe_device) from [<c043f55c>] (__driver_attach+0xac/0xb0)
-[<c043f55c>] (__driver_attach) from [<c043da18>] (bus_for_each_dev+0x54/0x88)
-[<c043da18>] (bus_for_each_dev) from [<c043e9e8>] (bus_add_driver+0xe8/0x1f4)
-[<c043e9e8>] (bus_add_driver) from [<c043fd58>] (driver_register+0x78/0xf4)
-[<c043fd58>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
+[<c010a86c>] (show_stack) from [<c033d0b4>] (dump_stack+0x84/0xa4)
+[<c033d0b4>] (dump_stack) from [<c011f308>] (__warn+0xd4/0x100)
+[<c011f308>] (__warn) from [<c011f354>] (warn_slowpath_null+0x20/0x28)
+[<c011f354>] (warn_slowpath_null) from [<c03c01d0>] (__of_reset_control_get+0x1b4/0x218)
+[<c03c01d0>] (__of_reset_control_get) from [<c0511afc>] (tegra_ehci_probe+0x2c8/0x514)
+[<c0511afc>] (tegra_ehci_probe) from [<c0440290>] (platform_drv_probe+0x50/0xb0)
+[<c0440290>] (platform_drv_probe) from [<c043ecbc>] (driver_probe_device+0x1f4/0x2b0)
+[<c043ecbc>] (driver_probe_device) from [<c043ee24>] (__driver_attach+0xac/0xb0)
+[<c043ee24>] (__driver_attach) from [<c043d2e0>] (bus_for_each_dev+0x54/0x88)
+[<c043d2e0>] (bus_for_each_dev) from [<c043e2b0>] (bus_add_driver+0xe8/0x1f4)
+[<c043e2b0>] (bus_add_driver) from [<c043f620>] (driver_register+0x78/0xf4)
+[<c043f620>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
 [<c010179c>] (do_one_initcall) from [<c0b00d94>] (kernel_init_freeable+0x11c/0x1e8)
-[<c0b00d94>] (kernel_init_freeable) from [<c07eec84>] (kernel_init+0x8/0x118)
-[<c07eec84>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
----[ end trace 7e073cd202e08724 ]---
+[<c0b00d94>] (kernel_init_freeable) from [<c07eea6c>] (kernel_init+0x8/0x118)
+[<c07eea6c>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
+---[ end trace 94a4b7553a4c299a ]---
 tegra-ehci 7d004000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d004000.usb: continuing, but please update your DT
 tegra-ehci 7d004000.usb: EHCI Host Controller
@@ -400,9 +400,9 @@
 pci 0000:00:02.0: [10de:0e13] type 01 class 0x060400
 pci 0000:00:02.0: PME# supported from D0 D1 D2 D3hot D3cold
 PCI: bus0: Fast back to back transfers disabled
+ata1: SATA link down (SStatus 0 SControl 300)
 pci 0000:00:01.0: bridge configuration invalid ([bus 00-00]), reconfiguring
 pci 0000:00:02.0: bridge configuration invalid ([bus 00-00]), reconfiguring
-ata1: SATA link down (SStatus 0 SControl 300)
 pci 0000:01:00.0: [8086:4232] type 00 class 0x028000
 pci 0000:01:00.0: reg 0x10: [mem 0x00000000-0x00001fff 64bit]
 pci 0000:01:00.0: PME# supported from D0 D3hot D3cold
@@ -417,17 +417,17 @@
 mmc1: new high speed SDHC card at address aaaa
 PCI: bus2: Fast back to back transfers disabled
 pci_bus 0000:02: busn_res: [bus 02-ff] end is updated to 02
+mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 pci 0000:00:01.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
 pci 0000:00:02.0: BAR 8: assigned [mem 0x13100000-0x131fffff]
-mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 pci 0000:00:02.0: BAR 9: assigned [mem 0x20000000-0x200fffff 64bit pref]
 pci 0000:00:02.0: BAR 7: assigned [io  0x1000-0x1fff]
 pci 0000:01:00.0: BAR 0: assigned [mem 0x13000000-0x13001fff 64bit]
- mmcblk1: p1 p2 p3
 pci 0000:00:01.0: PCI bridge to [bus 01]
 pci 0000:00:01.0:   bridge window [mem 0x13000000-0x130fffff]
 pci 0000:02:00.0: BAR 4: assigned [mem 0x20000000-0x20003fff 64bit pref]
 pci 0000:02:00.0: BAR 2: assigned [mem 0x13100000-0x13100fff 64bit]
+ mmcblk1: p1 p2 p3
 pci 0000:02:00.0: BAR 0: assigned [io  0x1000-0x10ff]
 pci 0000:00:02.0: PCI bridge to [bus 02]
 pci 0000:00:02.0:   bridge window [io  0x1000-0x1fff]
@@ -461,7 +461,7 @@
 cpu cpu2: cpufreq_init: failed to get clk: -2
 cpu cpu3: cpufreq_init: failed to get clk: -2
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:01:36 UTC (946684896)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:06 UTC (946684926)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2016-05-04 23:56:04.780260077 -0700
+++ /tmp/foob	2016-05-04 23:56:04.780260077 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1800538 us
+Bootloader-Cpu Init at (time stamp): 1800962 us
 
 Pinmux changes applied in kernel way
 
@@ -62,22 +62,22 @@
 Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1800512  us
-Bootloader (AosInit) time: 733492 us
-Bootloader Main Init time: 180190 us
-BlockDevMgrInit time: 157541 us
-PartMgrInit time: 9 us
-StorMgrInit time: 30 us
+Miscellaneous avp time: 1800936  us
+Bootloader (AosInit) time: 734274 us
+Bootloader Main Init time: 180066 us
+BlockDevMgrInit time: 157536 us
+PartMgrInit time: 8 us
+StorMgrInit time: 31 us
 FsMgrInit time: 2 us
-Partition Table load: 77070 us 
-Display init time: 574444 us
-Kernel read time: 343 msec
+Partition Table load: 76687 us 
+Display init time: 574830 us
+Kernel read time: 344 msec
 Load Kernel Size:   14880768 Bytes
-Bl Kernel ReadSpeed: 43384000 Bytes/sec
-Pre-Kernel Jump time: 1627273 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5687418 us
+Bl Kernel ReadSpeed: 43258000 Bytes/sec
+Pre-Kernel Jump time: 1626996 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5686900 us
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc6-next-20160504+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed May 4 00:35:49 PDT 2016
+Linux version 4.6.0-rc6-next-20160505+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed May 4 23:50:54 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
 Memory limited to 1734MB
 efi: Getting EFI parameters from FDT:
@@ -185,7 +185,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(1.047:1): initialized
+audit: type=2000 audit(1.059:1): initialized
 workingset: timestamp_bits=44 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-05-04 23:56:04.784260078 -0700
+++ /tmp/foob	2016-05-04 23:56:04.788260079 -0700
@@ -47,20 +47,20 @@
 	 #################################################################
 	 #################################################################
 	 ##########################
-	 910.2 KiB/s
+	 901.4 KiB/s
 done
-Bytes transferred = 7056048 (6baab0 hex)
+Bytes transferred = 7055656 (6ba928 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 898.4 KiB/s
+	 861.3 KiB/s
 done
 Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x6baab0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x6ba928 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102b3
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc6-next-20160504-92291-ga57cdca (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed May 4 00:33:38 PDT 2016
+Linux version 4.6.0-rc6-next-20160505-92543-ge3c95c2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed May 4 23:48:59 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -76,7 +76,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c1417e00, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c1417fc0, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -89,7 +89,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 955948K/1048576K available (9528K kernel code, 1127K rwdata, 4188K rodata, 2048K init, 342K bss, 27092K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 955944K/1048576K available (9528K kernel code, 1128K rwdata, 4188K rodata, 2048K init, 343K bss, 27096K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -97,10 +97,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc10651ec   (14709 kB)
+      .text : 0xc0208000 - 0xc1065268   (14709 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc1419e60   (1128 kB)
-       .bss : 0xc141b000 - 0xc1470bd8   ( 343 kB)
+      .data : 0xc1300000 - 0xc141a020   (1129 kB)
+       .bss : 0xc141c000 - 0xc1471c58   ( 344 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -273,25 +273,25 @@
 ------------[ cut here ]------------
 WARNING: CPU: 1 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
 Modules linked in:
-CPU: 1 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc6-next-20160504-92291-ga57cdca #1
+CPU: 1 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc6-next-20160505-92543-ge3c95c2 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c030f9a0>] (unwind_backtrace) from [<c030b090>] (show_stack+0x10/0x14)
-[<c030b090>] (show_stack) from [<c0585828>] (dump_stack+0x84/0xa4)
-[<c0585828>] (dump_stack) from [<c0341b6c>] (__warn+0xd4/0x100)
+[<c030b090>] (show_stack) from [<c0585868>] (dump_stack+0x84/0xa4)
+[<c0585868>] (dump_stack) from [<c0341b6c>] (__warn+0xd4/0x100)
 [<c0341b6c>] (__warn) from [<c0341bb8>] (warn_slowpath_null+0x20/0x28)
-[<c0341bb8>] (warn_slowpath_null) from [<c069df60>] (__of_reset_control_get+0x1bc/0x228)
-[<c069df60>] (__of_reset_control_get) from [<c090fc64>] (tegra_ehci_probe+0x2d0/0x51c)
-[<c090fc64>] (tegra_ehci_probe) from [<c0777d6c>] (platform_drv_probe+0x50/0xb0)
-[<c0777d6c>] (platform_drv_probe) from [<c07766d0>] (driver_probe_device+0x1f4/0x2b0)
-[<c07766d0>] (driver_probe_device) from [<c0776838>] (__driver_attach+0xac/0xb0)
-[<c0776838>] (__driver_attach) from [<c0774ce4>] (bus_for_each_dev+0x54/0x88)
-[<c0774ce4>] (bus_for_each_dev) from [<c0775cc4>] (bus_add_driver+0xe8/0x1f4)
-[<c0775cc4>] (bus_add_driver) from [<c0777054>] (driver_register+0x78/0xf4)
-[<c0777054>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
+[<c0341bb8>] (warn_slowpath_null) from [<c069dfa0>] (__of_reset_control_get+0x1bc/0x228)
+[<c069dfa0>] (__of_reset_control_get) from [<c090e780>] (tegra_ehci_probe+0x2d0/0x51c)
+[<c090e780>] (tegra_ehci_probe) from [<c0776798>] (platform_drv_probe+0x50/0xb0)
+[<c0776798>] (platform_drv_probe) from [<c07750fc>] (driver_probe_device+0x1f4/0x2b0)
+[<c07750fc>] (driver_probe_device) from [<c0775264>] (__driver_attach+0xac/0xb0)
+[<c0775264>] (__driver_attach) from [<c0773710>] (bus_for_each_dev+0x54/0x88)
+[<c0773710>] (bus_for_each_dev) from [<c07746f0>] (bus_add_driver+0xe8/0x1f4)
+[<c07746f0>] (bus_add_driver) from [<c0775a80>] (driver_register+0x78/0xf4)
+[<c0775a80>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
 [<c0301e5c>] (do_one_initcall) from [<c1100d80>] (kernel_init_freeable+0x120/0x1ec)
-[<c1100d80>] (kernel_init_freeable) from [<c0b5813c>] (kernel_init+0x8/0x118)
-[<c0b5813c>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
----[ end trace 8fcec3018d1c0f5f ]---
+[<c1100d80>] (kernel_init_freeable) from [<c0b57138>] (kernel_init+0x8/0x118)
+[<c0b57138>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
+---[ end trace 1b445291a51c4c4e ]---
 tegra-ehci c5000000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci c5000000.usb: continuing, but please update your DT
 ulpi_phy_power_on: ulpi write failed
@@ -349,24 +349,24 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: hctosys: unable to read the hardware clock
-Waiting for root device PARTUUID=11eb23c1-01...
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
+rtc-em3027 2-0056: hctosys: unable to read the hardware clock
  mmcblk0: p1
+EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
-EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
+usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
+asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
-usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-05-04 23:56:04.792260081 -0700
+++ /tmp/foob	2016-05-04 23:56:04.796260083 -0700
@@ -45,20 +45,20 @@
 	 #################################################################
 	 #################################################################
 	 #########################################
-	 904.3 KiB/s
+	 906.3 KiB/s
 done
-Bytes transferred = 5371024 (51f490 hex)
+Bytes transferred = 5369800 (51efc8 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 861.3 KiB/s
+	 898.4 KiB/s
 done
 Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x51f490 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x51efc8 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102b3
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc6-next-20160504-92291-ga57cdca (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed May 4 00:33:39 PDT 2016
+Linux version 4.6.0-rc6-next-20160505-92543-ge3c95c2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed May 4 23:48:57 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -97,7 +97,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0abce88   (10964 kB)
+      .text : 0xc0008000 - 0xc0abcee8   (10964 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
       .data : 0xc0c00000 - 0xc0c5d8c0   ( 375 kB)
        .bss : 0xc0c5f000 - 0xc0cbaad0   ( 367 kB)
@@ -265,25 +265,25 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1b4/0x218
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc6-next-20160504-92291-ga57cdca #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc6-next-20160505-92543-ge3c95c2 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c010f1f8>] (unwind_backtrace) from [<c010a86c>] (show_stack+0x10/0x14)
-[<c010a86c>] (show_stack) from [<c033d034>] (dump_stack+0x84/0xa4)
-[<c033d034>] (dump_stack) from [<c011f2c8>] (__warn+0xd4/0x100)
-[<c011f2c8>] (__warn) from [<c011f314>] (warn_slowpath_null+0x20/0x28)
-[<c011f314>] (warn_slowpath_null) from [<c03c0150>] (__of_reset_control_get+0x1b4/0x218)
-[<c03c0150>] (__of_reset_control_get) from [<c05121f8>] (tegra_ehci_probe+0x2c8/0x514)
-[<c05121f8>] (tegra_ehci_probe) from [<c04409c8>] (platform_drv_probe+0x50/0xb0)
-[<c04409c8>] (platform_drv_probe) from [<c043f3f4>] (driver_probe_device+0x1f4/0x2b0)
-[<c043f3f4>] (driver_probe_device) from [<c043f55c>] (__driver_attach+0xac/0xb0)
-[<c043f55c>] (__driver_attach) from [<c043da18>] (bus_for_each_dev+0x54/0x88)
-[<c043da18>] (bus_for_each_dev) from [<c043e9e8>] (bus_add_driver+0xe8/0x1f4)
-[<c043e9e8>] (bus_add_driver) from [<c043fd58>] (driver_register+0x78/0xf4)
-[<c043fd58>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
+[<c010a86c>] (show_stack) from [<c033d0b4>] (dump_stack+0x84/0xa4)
+[<c033d0b4>] (dump_stack) from [<c011f308>] (__warn+0xd4/0x100)
+[<c011f308>] (__warn) from [<c011f354>] (warn_slowpath_null+0x20/0x28)
+[<c011f354>] (warn_slowpath_null) from [<c03c01d0>] (__of_reset_control_get+0x1b4/0x218)
+[<c03c01d0>] (__of_reset_control_get) from [<c0511afc>] (tegra_ehci_probe+0x2c8/0x514)
+[<c0511afc>] (tegra_ehci_probe) from [<c0440290>] (platform_drv_probe+0x50/0xb0)
+[<c0440290>] (platform_drv_probe) from [<c043ecbc>] (driver_probe_device+0x1f4/0x2b0)
+[<c043ecbc>] (driver_probe_device) from [<c043ee24>] (__driver_attach+0xac/0xb0)
+[<c043ee24>] (__driver_attach) from [<c043d2e0>] (bus_for_each_dev+0x54/0x88)
+[<c043d2e0>] (bus_for_each_dev) from [<c043e2b0>] (bus_add_driver+0xe8/0x1f4)
+[<c043e2b0>] (bus_add_driver) from [<c043f620>] (driver_register+0x78/0xf4)
+[<c043f620>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
 [<c010179c>] (do_one_initcall) from [<c0b00d94>] (kernel_init_freeable+0x11c/0x1e8)
-[<c0b00d94>] (kernel_init_freeable) from [<c07eec84>] (kernel_init+0x8/0x118)
-[<c07eec84>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
----[ end trace 42b269610ee79066 ]---
+[<c0b00d94>] (kernel_init_freeable) from [<c07eea6c>] (kernel_init+0x8/0x118)
+[<c07eea6c>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
+---[ end trace 8e2b49def8ccfe21 ]---
 tegra-ehci c5000000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci c5000000.usb: continuing, but please update your DT
 ulpi_phy_power_on: ulpi write failed
@@ -328,13 +328,13 @@
 tlv320aic23-codec 2-001a: ASoC: no dapm match for MICIN --> NULL --> Mic Input
 tlv320aic23-codec 2-001a: ASoC: Failed to add route MICIN -> NULL -> Mic Input
 tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
-mmc1: new high speed SD card at address b368
 mmc0: host does not support reading read-only switch, assuming write-enable
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk0: p1
+mmc1: new high speed SD card at address b368
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2016-05-04 23:56:04.800260085 -0700
+++ /tmp/foob	2016-05-04 23:56:04.804260086 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 754620 us
-Basic modules init took 348803 us
+Tegraboot started after 748872 us
+Basic modules init took 349527 us
 Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14371 us
-Carveout took 252725 us
-CPU initialization took 92305 us
-Total time taken by TegraBoot 708204 us
+Next stage binary read took 14340 us
+Carveout took 252756 us
+CPU initialization took 92309 us
+Total time taken by TegraBoot 708932 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.593] LPDDR4 Training: Number of tables = 10
+64b[0001.588] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -121,7 +121,7 @@
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
 37108 bytes read in 454 ms (79.1 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 495 ms (3.1 MiB/s)
+1627787 bytes read in 494 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc6-next-20160504+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed May 4 00:35:49 PDT 2016
+Linux version 4.6.0-rc6-next-20160505+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed May 4 23:50:54 PDT 2016
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-05-04 23:56:04.808260087 -0700
+++ /tmp/foob	2016-05-04 23:56:04.812260089 -0700
@@ -39,9 +39,9 @@
 	 #################################################################
 	 #################################################################
 	 ##########################
-	 3.2 MiB/s
+	 3.1 MiB/s
 done
-Bytes transferred = 7056048 (6baab0 hex)
+Bytes transferred = 7055656 (6ba928 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -52,7 +52,7 @@
 done
 Bytes transferred = 51627 (c9ab hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6baab0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6ba928 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff9aa ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc6-next-20160504-92291-ga57cdca (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed May 4 00:33:38 PDT 2016
+Linux version 4.6.0-rc6-next-20160505-92543-ge3c95c2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed May 4 23:48:59 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,7 +68,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c1417e00, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1417fc0, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -81,7 +81,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1995052K/2096128K available (9528K kernel code, 1127K rwdata, 4188K rodata, 2048K init, 342K bss, 35540K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1995048K/2096128K available (9528K kernel code, 1128K rwdata, 4188K rodata, 2048K init, 343K bss, 35544K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -89,10 +89,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc10651ec   (14709 kB)
+      .text : 0xc0208000 - 0xc1065268   (14709 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc1419e60   (1128 kB)
-       .bss : 0xc141b000 - 0xc1470bd8   ( 343 kB)
+      .data : 0xc1300000 - 0xc141a020   (1129 kB)
+       .bss : 0xc141c000 - 0xc1471c58   ( 344 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -239,27 +239,27 @@
 ehci-atmel: EHCI Atmel driver
 tegra-ehci: Tegra EHCI driver
 ------------[ cut here ]------------
-WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
+WARNING: CPU: 2 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc6-next-20160504-92291-ga57cdca #1
+CPU: 2 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc6-next-20160505-92543-ge3c95c2 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c030f9a0>] (unwind_backtrace) from [<c030b090>] (show_stack+0x10/0x14)
-[<c030b090>] (show_stack) from [<c0585828>] (dump_stack+0x84/0xa4)
-[<c0585828>] (dump_stack) from [<c0341b6c>] (__warn+0xd4/0x100)
+[<c030b090>] (show_stack) from [<c0585868>] (dump_stack+0x84/0xa4)
+[<c0585868>] (dump_stack) from [<c0341b6c>] (__warn+0xd4/0x100)
 [<c0341b6c>] (__warn) from [<c0341bb8>] (warn_slowpath_null+0x20/0x28)
-[<c0341bb8>] (warn_slowpath_null) from [<c069df60>] (__of_reset_control_get+0x1bc/0x228)
-[<c069df60>] (__of_reset_control_get) from [<c090fc64>] (tegra_ehci_probe+0x2d0/0x51c)
-[<c090fc64>] (tegra_ehci_probe) from [<c0777d6c>] (platform_drv_probe+0x50/0xb0)
-[<c0777d6c>] (platform_drv_probe) from [<c07766d0>] (driver_probe_device+0x1f4/0x2b0)
-[<c07766d0>] (driver_probe_device) from [<c0776838>] (__driver_attach+0xac/0xb0)
-[<c0776838>] (__driver_attach) from [<c0774ce4>] (bus_for_each_dev+0x54/0x88)
-[<c0774ce4>] (bus_for_each_dev) from [<c0775cc4>] (bus_add_driver+0xe8/0x1f4)
-[<c0775cc4>] (bus_add_driver) from [<c0777054>] (driver_register+0x78/0xf4)
-[<c0777054>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
+[<c0341bb8>] (warn_slowpath_null) from [<c069dfa0>] (__of_reset_control_get+0x1bc/0x228)
+[<c069dfa0>] (__of_reset_control_get) from [<c090e780>] (tegra_ehci_probe+0x2d0/0x51c)
+[<c090e780>] (tegra_ehci_probe) from [<c0776798>] (platform_drv_probe+0x50/0xb0)
+[<c0776798>] (platform_drv_probe) from [<c07750fc>] (driver_probe_device+0x1f4/0x2b0)
+[<c07750fc>] (driver_probe_device) from [<c0775264>] (__driver_attach+0xac/0xb0)
+[<c0775264>] (__driver_attach) from [<c0773710>] (bus_for_each_dev+0x54/0x88)
+[<c0773710>] (bus_for_each_dev) from [<c07746f0>] (bus_add_driver+0xe8/0x1f4)
+[<c07746f0>] (bus_add_driver) from [<c0775a80>] (driver_register+0x78/0xf4)
+[<c0775a80>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
 [<c0301e5c>] (do_one_initcall) from [<c1100d80>] (kernel_init_freeable+0x120/0x1ec)
-[<c1100d80>] (kernel_init_freeable) from [<c0b5813c>] (kernel_init+0x8/0x118)
-[<c0b5813c>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
----[ end trace 165c2f008bb80cc4 ]---
+[<c1100d80>] (kernel_init_freeable) from [<c0b57138>] (kernel_init+0x8/0x118)
+[<c0b57138>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
+---[ end trace 1116613d80a66229 ]---
 tegra-ehci 7d004000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d004000.usb: continuing, but please update your DT
 ohci_hcd: USB 1.1 'Open' Host Controller (OHCI) Driver
@@ -309,13 +309,13 @@
 tegra-pcie 3000.pcie-controller: 2x3 configuration
 tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
 mmc1: new high speed MMC card at address 0001
+tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 mmcblk1: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB
 Primary GPT is invalid, using alternate GPT.
  mmcblk1: p1 p2 p3 p4 p5
-tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -386,5 +386,4 @@
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
-root@(none):/# 
\ No newline at end of file
+root@(none):/# [    3.848555] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-05-04 23:56:04.816260091 -0700
+++ /tmp/foob	2016-05-04 23:56:04.820260093 -0700
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 #########################################
-	 3.2 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 5371024 (51f490 hex)
+Bytes transferred = 5369800 (51efc8 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.9 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 51627 (c9ab hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x51f490 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x51efc8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff9aa ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc6-next-20160504-92291-ga57cdca (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed May 4 00:33:39 PDT 2016
+Linux version 4.6.0-rc6-next-20160505-92543-ge3c95c2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed May 4 23:48:57 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -89,7 +89,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0abce88   (10964 kB)
+      .text : 0xc0008000 - 0xc0abcee8   (10964 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
       .data : 0xc0c00000 - 0xc0c5d8c0   ( 375 kB)
        .bss : 0xc0c5f000 - 0xc0cbaad0   ( 367 kB)
@@ -231,25 +231,25 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1b4/0x218
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc6-next-20160504-92291-ga57cdca #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc6-next-20160505-92543-ge3c95c2 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c010f1f8>] (unwind_backtrace) from [<c010a86c>] (show_stack+0x10/0x14)
-[<c010a86c>] (show_stack) from [<c033d034>] (dump_stack+0x84/0xa4)
-[<c033d034>] (dump_stack) from [<c011f2c8>] (__warn+0xd4/0x100)
-[<c011f2c8>] (__warn) from [<c011f314>] (warn_slowpath_null+0x20/0x28)
-[<c011f314>] (warn_slowpath_null) from [<c03c0150>] (__of_reset_control_get+0x1b4/0x218)
-[<c03c0150>] (__of_reset_control_get) from [<c05121f8>] (tegra_ehci_probe+0x2c8/0x514)
-[<c05121f8>] (tegra_ehci_probe) from [<c04409c8>] (platform_drv_probe+0x50/0xb0)
-[<c04409c8>] (platform_drv_probe) from [<c043f3f4>] (driver_probe_device+0x1f4/0x2b0)
-[<c043f3f4>] (driver_probe_device) from [<c043f55c>] (__driver_attach+0xac/0xb0)
-[<c043f55c>] (__driver_attach) from [<c043da18>] (bus_for_each_dev+0x54/0x88)
-[<c043da18>] (bus_for_each_dev) from [<c043e9e8>] (bus_add_driver+0xe8/0x1f4)
-[<c043e9e8>] (bus_add_driver) from [<c043fd58>] (driver_register+0x78/0xf4)
-[<c043fd58>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
+[<c010a86c>] (show_stack) from [<c033d0b4>] (dump_stack+0x84/0xa4)
+[<c033d0b4>] (dump_stack) from [<c011f308>] (__warn+0xd4/0x100)
+[<c011f308>] (__warn) from [<c011f354>] (warn_slowpath_null+0x20/0x28)
+[<c011f354>] (warn_slowpath_null) from [<c03c01d0>] (__of_reset_control_get+0x1b4/0x218)
+[<c03c01d0>] (__of_reset_control_get) from [<c0511afc>] (tegra_ehci_probe+0x2c8/0x514)
+[<c0511afc>] (tegra_ehci_probe) from [<c0440290>] (platform_drv_probe+0x50/0xb0)
+[<c0440290>] (platform_drv_probe) from [<c043ecbc>] (driver_probe_device+0x1f4/0x2b0)
+[<c043ecbc>] (driver_probe_device) from [<c043ee24>] (__driver_attach+0xac/0xb0)
+[<c043ee24>] (__driver_attach) from [<c043d2e0>] (bus_for_each_dev+0x54/0x88)
+[<c043d2e0>] (bus_for_each_dev) from [<c043e2b0>] (bus_add_driver+0xe8/0x1f4)
+[<c043e2b0>] (bus_add_driver) from [<c043f620>] (driver_register+0x78/0xf4)
+[<c043f620>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
 [<c010179c>] (do_one_initcall) from [<c0b00d94>] (kernel_init_freeable+0x11c/0x1e8)
-[<c0b00d94>] (kernel_init_freeable) from [<c07eec84>] (kernel_init+0x8/0x118)
-[<c07eec84>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
----[ end trace 1b9dbc9aa09ea3f2 ]---
+[<c0b00d94>] (kernel_init_freeable) from [<c07eea6c>] (kernel_init+0x8/0x118)
+[<c07eea6c>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
+---[ end trace b961602c67f7fe0b ]---
 tegra-ehci 7d004000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d004000.usb: continuing, but please update your DT
 usbcore: registered new interface driver cdc_acm
