Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2017-04-12 22:56:22.717518698 -0700
+++ /tmp/foob	2017-04-12 22:56:22.721518713 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc6-next-20170412-284657-g1267f8f (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Apr 12 01:08:47 PDT 2017
+Linux version 4.11.0-rc6-next-20170413-284853-gf4fa27cb (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Apr 12 22:54:44 PDT 2017
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -21,7 +21,7 @@
 psci: PSCIv0.2 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: Trusted OS migration not required
-percpu: Embedded 23 pages/cpu @ffff80003efd0000 s56960 r8192 d29056 u94208
+percpu: Embedded 24 pages/cpu @ffff80003efcf000 s58624 r8192 d31488 u98304
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
 CPU features: enabling workaround for ARM erratum 834220
@@ -29,15 +29,15 @@
 Policy zone: DMA
 Kernel command line: root=/dev/vda2 rw rootwait mem=1024M console=ttyAMA0,38400n8 init=/bin/bash
 PID hash table entries: 4096 (order: 3, 32768 bytes)
-Memory: 999804K/1048576K available (8828K kernel code, 1084K rwdata, 4228K rodata, 1088K init, 404K bss, 32388K reserved, 16384K cma-reserved)
+Memory: 999800K/1048576K available (8828K kernel code, 1086K rwdata, 4232K rodata, 1088K init, 403K bss, 32392K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008920000   (  8832 KB)
     .rodata : 0xffff000008920000 - 0xffff000008d50000   (  4288 KB)
       .init : 0xffff000008d50000 - 0xffff000008e60000   (  1088 KB)
-      .data : 0xffff000008e60000 - 0xffff000008f6f200   (  1085 KB)
-       .bss : 0xffff000008f6f200 - 0xffff000008fd4338   (   405 KB)
+      .data : 0xffff000008e60000 - 0xffff000008f6fa00   (  1087 KB)
+       .bss : 0xffff000008f6fa00 - 0xffff000008fd47b8   (   404 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -113,7 +113,7 @@
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 kvm [1]: HYP mode not available
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.728:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.713:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -179,7 +179,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2017-04-12 08:09:33 UTC (1491984573)
+rtc-pl031 9010000.pl031: setting system clock to 2017-04-13 05:55:30 UTC (1492062930)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-04-12 22:56:22.753518845 -0700
+++ /tmp/foob	2017-04-12 22:56:22.753518845 -0700
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra114 (Dalmore) # 
 Tegra114 (Dalmore) # usb reset
 (Re)start USB...
@@ -40,20 +40,20 @@
 	 #################################################################
 	 #################################################################
 	 ##################
-	 3 MiB/s
+	 3.1 MiB/s
 done
-Bytes transferred = 7884576 (784f20 hex)
+Bytes transferred = 7889992 (786448 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 36.1 KiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x784f20 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x786448 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc6-next-20170412-166416-g1267f8f (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Apr 12 01:04:46 PDT 2017
+Linux version 4.11.0-rc6-next-20170413-166612-gf4fa27cb (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Apr 12 22:50:01 PDT 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -72,13 +72,13 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c157e9c0, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c157ea80, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 17 pages/cpu @eef98000 s38592 r8192 d22848 u69632
-pcpu-alloc: s38592 r8192 d22848 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @eef98000 s39040 r8192 d22400 u69632
+pcpu-alloc: s39040 r8192 d22400 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=7369c667-ff51-ec4a-29cd-baabf2fbe346 rootwait gpt debug earlyprintk init=/bin/bash
@@ -95,8 +95,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1586568   (1562 kB)
-       .bss : 0xc158fd10 - 0xc15f5a08   ( 408 kB)
+      .data : 0xc1400000 - 0xc1586628   (1562 kB)
+       .bss : 0xc158fd10 - 0xc15f5b08   ( 408 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU debugfs-based tracing is enabled.
@@ -324,7 +324,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2001-02-03 19:41:35 UTC (981229295)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2001-02-04 17:26:56 UTC (981307616)
 vdd_hdmi_5v0: disabling
 dvdd_ts: disabling
 usb1_vbus: disabling
@@ -338,4 +338,4 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.419508] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.419646] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2017-04-12 22:56:22.761518878 -0700
+++ /tmp/foob	2017-04-12 22:56:22.761518878 -0700
@@ -27,7 +27,6 @@
 Tegra114 (Dalmore) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=7369c667-ff51-ec4a-29cd-baabf2fbe346 rootwait gpt debug earlyprintk init=/bin/bash'
 Tegra114 (Dalmore) # setenv ipaddr 192.168.100.100; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra114 (Dalmore) # tftpboot 0x81000000 zImage.tegra_defconfig
-Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'zImage.tegra_defconfig'.
@@ -38,10 +37,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #######################
-	 3.3 MiB/s
+	 ########################
+	 2.8 MiB/s
 done
-Bytes transferred = 6061720 (5c7e98 hex)
+Bytes transferred = 6066320 (5c9090 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -52,7 +51,7 @@
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5c7e98 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5c9090 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -61,7 +60,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc6-next-20170412-166416-g1267f8f (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Apr 12 01:04:26 PDT 2017
+Linux version 4.11.0-rc6-next-20170413-166612-gf4fa27cb (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Apr 12 22:49:33 PDT 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -71,13 +70,13 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0ead700, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0ead800, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 17 pages/cpu @eef97000 s38528 r8192 d22912 u69632
-pcpu-alloc: s38528 r8192 d22912 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @eef97000 s38912 r8192 d22528 u69632
+pcpu-alloc: s38912 r8192 d22528 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=7369c667-ff51-ec4a-29cd-baabf2fbe346 rootwait gpt debug earlyprintk init=/bin/bash
@@ -94,8 +93,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0eb7c30   ( 736 kB)
-       .bss : 0xc0ec1664 - 0xc0f0e334   ( 308 kB)
+      .data : 0xc0e00000 - 0xc0eb7d30   ( 736 kB)
+       .bss : 0xc0ec1664 - 0xc0f0e434   ( 308 kB)
 Preemptible hierarchical RCU implementation.
 NR_IRQS:16 nr_irqs:16 16
 /interrupt-controller@60004000: 160 interrupts forwarded to /interrupt-controller@50041000
@@ -317,7 +316,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2001-02-03 19:39:05 UTC (981229145)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2001-02-04 17:24:21 UTC (981307461)
 vdd_hdmi_5v0: disabling
 dvdd_ts: disabling
 usb1_vbus: disabling
@@ -333,4 +332,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.647248] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.647907] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-04-12 22:56:22.769518910 -0700
+++ /tmp/foob	2017-04-12 22:56:22.769518910 -0700
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -35,9 +35,9 @@
 	 #################################################################
 	 #################################################################
 	 ##################
-	 4 MiB/s
+	 3.9 MiB/s
 done
-Bytes transferred = 7884576 (784f20 hex)
+Bytes transferred = 7889992 (786448 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -48,7 +48,7 @@
 done
 Bytes transferred = 67133 (1063d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x784f20 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x786448 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301363c
@@ -56,7 +56,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc6-next-20170412-166416-g1267f8f (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Apr 12 01:04:46 PDT 2017
+Linux version 4.11.0-rc6-next-20170413-166612-gf4fa27cb (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Apr 12 22:50:01 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -67,13 +67,13 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c157e9c0, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c157ea80, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 17 pages/cpu @eef82000 s38592 r8192 d22848 u69632
-pcpu-alloc: s38592 r8192 d22848 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @eef82000 s39040 r8192 d22400 u69632
+pcpu-alloc: s39040 r8192 d22400 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=b1791faa-84e8-4e27-83a1-9fb0a77e8803 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
@@ -90,8 +90,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1586568   (1562 kB)
-       .bss : 0xc158fd10 - 0xc15f5a08   ( 408 kB)
+      .data : 0xc1400000 - 0xc1586628   (1562 kB)
+       .bss : 0xc158fd10 - 0xc15f5b08   ( 408 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU debugfs-based tracing is enabled.
@@ -314,17 +314,17 @@
 +5V_SATA: supplied by +5V_SYS
 +12V_SATA: supplied by +VDD_MUX
  usb2-0: usb2-0 supply vbus not found, using dummy regulator
+ usb2-1: usb2-1 supply vbus not found, using dummy regulator
+ usb3-0: usb3-0 supply vbus not found, using dummy regulator
 mmc0: new high speed MMC card at address 0001
+tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
+tegra-pcie 1003000.pcie-controller: Failed to get supply 'avddio-pex': -517
+tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
  mmcblk0: p1
- usb2-1: usb2-1 supply vbus not found, using dummy regulator
- usb3-0: usb3-0 supply vbus not found, using dummy regulator
-tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
-tegra-pcie 1003000.pcie-controller: Failed to get supply 'avddio-pex': -517
-tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
 as3722-sd2: supplied by +5V_SYS
 as3722-sd3: supplied by +5V_SYS
@@ -374,6 +374,7 @@
 pci_bus 0000:00: root bus resource [bus 00-ff]
 pci 0000:00:01.0: [10de:0e12] type 01 class 0x060400
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x88 may corrupt adjacent RW1C bits
+ata1: SATA link down (SStatus 0 SControl 300)
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x52 may corrupt adjacent RW1C bits
 pci 0000:00:01.0: PME# supported from D0 D1 D2 D3hot D3cold
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x4c may corrupt adjacent RW1C bits
@@ -382,7 +383,6 @@
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x52 may corrupt adjacent RW1C bits
 pci 0000:00:02.0: PME# supported from D0 D1 D2 D3hot D3cold
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4c may corrupt adjacent RW1C bits
-ata1: SATA link down (SStatus 0 SControl 300)
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x3e may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x3e may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x4 may corrupt adjacent RW1C bits
@@ -435,7 +435,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:02 UTC (946684922)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:30 UTC (946684950)
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +USB0_VBUS_SW: disabling
 +1.05V_RUN_AVDD_HDMI_PLL: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2017-04-12 22:56:22.777518944 -0700
+++ /tmp/foob	2017-04-12 22:56:22.777518944 -0700
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MMC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -32,21 +32,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #######################
-	 4 MiB/s
+	 ########################
+	 3.7 MiB/s
 done
-Bytes transferred = 6061720 (5c7e98 hex)
+Bytes transferred = 6066320 (5c9090 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.8 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 67133 (1063d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5c7e98 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5c9090 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301363c
@@ -55,7 +55,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc6-next-20170412-166416-g1267f8f (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Apr 12 01:04:26 PDT 2017
+Linux version 4.11.0-rc6-next-20170413-166612-gf4fa27cb (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Apr 12 22:49:33 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -65,13 +65,13 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0ead700, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0ead800, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 17 pages/cpu @eef81000 s38528 r8192 d22912 u69632
-pcpu-alloc: s38528 r8192 d22912 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @eef81000 s38912 r8192 d22528 u69632
+pcpu-alloc: s38912 r8192 d22528 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=b1791faa-84e8-4e27-83a1-9fb0a77e8803 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
@@ -88,8 +88,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0eb7c30   ( 736 kB)
-       .bss : 0xc0ec1664 - 0xc0f0e334   ( 308 kB)
+      .data : 0xc0e00000 - 0xc0eb7d30   ( 736 kB)
+       .bss : 0xc0ec1664 - 0xc0f0e434   ( 308 kB)
 Preemptible hierarchical RCU implementation.
 NR_IRQS:16 nr_irqs:16 16
 /interrupt-controller@60004000: 160 interrupts forwarded to /interrupt-controller@50041000
@@ -317,6 +317,7 @@
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
 tegra-pcie 1003000.pcie-controller: Failed to get supply 'avddio-pex': -517
 tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
+input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input1
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
 as3722-sd2: supplied by +5V_SYS
 as3722-sd3: supplied by +5V_SYS
@@ -329,7 +330,6 @@
 as3722-ldo4: supplied by +3.3V_SYS
 as3722-ldo5: supplied by +1.8V_VDDIO
 as3722-ldo6: supplied by +3.3V_RUN
-input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input1
 as3722-ldo7: supplied by +1.8V_VDDIO
 +1.05V_RUN_CAM_REAR: Bringing 0uV into 1050000-1050000uV
 as3722-ldo9: supplied by +5V_SYS
@@ -366,11 +366,11 @@
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc1: Unknown controller version (3). You may experience problems.
 mmc1: Invalid maximum block size, assuming 512 bytes
+ata1: SATA link down (SStatus 0 SControl 300)
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
 tegra-pcie 1003000.pcie-controller: probing port 0, using 2 lanes
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
-ata1: SATA link down (SStatus 0 SControl 300)
 tegra-pcie 1003000.pcie-controller: probing port 1, using 1 lanes
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000000
 tegra-pcie 1003000.pcie-controller: PCI host bridge to bus 0000:00
@@ -440,7 +440,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:13 UTC (946684813)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:16 UTC (946684816)
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +USB0_VBUS_SW: disabling
 +1.05V_RUN_AVDD_HDMI_PLL: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-04-12 22:56:22.785518976 -0700
+++ /tmp/foob	2017-04-12 22:56:22.785518976 -0700
@@ -43,20 +43,20 @@
 	 #################################################################
 	 #################################################################
 	 ##################
-	 2.8 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 7884576 (784f20 hex)
+Bytes transferred = 7889992 (786448 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 43 KiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 68155 (10a3b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x784f20 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x786448 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a3a
@@ -64,7 +64,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc6-next-20170412-166416-g1267f8f (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Apr 12 01:04:46 PDT 2017
+Linux version 4.11.0-rc6-next-20170413-166612-gf4fa27cb (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Apr 12 22:50:01 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -76,13 +76,13 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c157e9c0, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c157ea80, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327679 pages, LIFO batch:31
-percpu: Embedded 17 pages/cpu @eef81000 s38592 r8192 d22848 u69632
-pcpu-alloc: s38592 r8192 d22848 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @eef81000 s39040 r8192 d22400 u69632
+pcpu-alloc: s39040 r8192 d22400 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522751
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=4d36e82a-b3c0-4b4b-bfcd-afe5217ac023 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
@@ -99,8 +99,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1586568   (1562 kB)
-       .bss : 0xc158fd10 - 0xc15f5a08   ( 408 kB)
+      .data : 0xc1400000 - 0xc1586628   (1562 kB)
+       .bss : 0xc158fd10 - 0xc15f5b08   ( 408 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU debugfs-based tracing is enabled.
@@ -293,9 +293,9 @@
 sdhci-tegra 700b0000.sdhci: allocated mmc-pwrseq
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
+elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
 mmc0: Unknown controller version (3). You may experience problems.
 mmc0: Invalid maximum block size, assuming 512 bytes
-elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
 input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 mmc0: SDHCI controller on 700b0600.sdhci [700b0600.sdhci] using ADMA
 ledtrig-cpu: registered to indicate activity on CPUs
@@ -310,12 +310,6 @@
 can: raw protocol (rev 20120528)
 can: broadcast manager protocol (rev 20161123 t)
 can: netlink gateway (rev 20130117) max_hops=1
-mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
-mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
-mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
-mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 Key type dns_resolver registered
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
@@ -335,22 +329,28 @@
 +5V_HDMI_CON: supplied by +5V_SYS
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
  usb3-1: usb3-1 supply vbus not found, using dummy regulator
+mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
+mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
 tegra-sor 54540000.sor: failed to probe output: -517
+mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
+mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
 panel-simple panel: panel supply power not found, using dummy regulator
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 sdhci-tegra 700b0000.sdhci: allocated mmc-pwrseq
 mmc1: Unknown controller version (3). You may experience problems.
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: Switching to 3.3V signalling voltage failed
 mmc1: SDHCI controller on 700b0000.sdhci [700b0000.sdhci] using ADMA
-elan_i2c 1-0015: invalid report id data (ff)
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
+elan_i2c 1-0015: invalid report id data (ff)
 mmc2: Unknown controller version (3). You may experience problems.
 mmc2: Invalid maximum block size, assuming 512 bytes
 mmc1: new high speed SDIO card at address 0001
 mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 01:31:01 UTC (946690261)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:23 UTC (946684823)
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +1.05V_RUN: disabling
 +1.8V_RUN_CAM: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2017-04-12 22:56:22.793519009 -0700
+++ /tmp/foob	2017-04-12 22:56:22.793519009 -0700
@@ -40,21 +40,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #######################
-	 3.6 MiB/s
+	 ########################
+	 3.5 MiB/s
 done
-Bytes transferred = 6061720 (5c7e98 hex)
+Bytes transferred = 6066320 (5c9090 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.4 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 68155 (10a3b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5c7e98 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5c9090 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a3a
@@ -63,7 +63,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc6-next-20170412-166416-g1267f8f (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Apr 12 01:04:26 PDT 2017
+Linux version 4.11.0-rc6-next-20170413-166612-gf4fa27cb (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Apr 12 22:49:33 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -74,13 +74,13 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c0ead700, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0ead800, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327679 pages, LIFO batch:31
-percpu: Embedded 17 pages/cpu @eef80000 s38528 r8192 d22912 u69632
-pcpu-alloc: s38528 r8192 d22912 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @eef80000 s38912 r8192 d22528 u69632
+pcpu-alloc: s38912 r8192 d22528 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522751
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=4d36e82a-b3c0-4b4b-bfcd-afe5217ac023 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
@@ -97,8 +97,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0eb7c30   ( 736 kB)
-       .bss : 0xc0ec1664 - 0xc0f0e334   ( 308 kB)
+      .data : 0xc0e00000 - 0xc0eb7d30   ( 736 kB)
+       .bss : 0xc0ec1664 - 0xc0f0e434   ( 308 kB)
 Preemptible hierarchical RCU implementation.
 NR_IRQS:16 nr_irqs:16 16
 /interrupt-controller@60004000: 160 interrupts forwarded to /interrupt-controller@50041000
@@ -293,11 +293,11 @@
 input: GoogleNyanBig Headphones as /devices/soc0/sound/sound/card1/input1
 input: GoogleNyanBig Mic Jack as /devices/soc0/sound/sound/card1/input2
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
+mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 NET: Registered protocol family 10
 Segment Routing with IPv6
 mip6: Mobile IPv6
-mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
 NET: Registered protocol family 17
 NET: Registered protocol family 15
@@ -354,8 +354,8 @@
 mmc2: Invalid maximum block size, assuming 512 bytes
 mmc1: new high speed SDIO card at address 0001
 mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
-input: gpio-keys as /devices/soc0/gpio-keys/input/input3
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 01:27:48 UTC (946690068)
+input: gpio-keys as /devices/soc0/gpio-keys/input/input4
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:22 UTC (946684822)
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +1.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
@@ -365,14 +365,14 @@
 +1.8V_RUN_VPP_FUSE: disabling
 +VDD_LED: disabling
 +5V_VDD_TS_SW: disabling
+usb 1-2: new high-speed USB device number 2 using tegra-xusb
 +3.3V_PANEL: disabling
+input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input3
 +1.05V_RUN_AVDD_HDMI_PLL: disabling
 +5V_HDMI_CON: disabling
 ALSA device list:
+  #0: tegra-hda at 0x70038000 irq 343
   #1: GoogleNyanBig
-EXT4-fs (mmcblk0p3): couldn't mount as ext3 due to feature incompatibilities
-input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input4
-usb 1-2: new high-speed USB device number 2 using tegra-xusb
 mmc2: new high speed SDHC card at address 0007
 mmcblk2: mmc2:0007 SD08G 7.42 GiB 
 GPT:Primary header alternate_lba != Alt. header my_lba
@@ -381,13 +381,14 @@
 GPT:3014591 != 15564799
 GPT: Use GNU Parted to correct GPT errors.
  mmcblk2: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
+EXT4-fs (mmcblk0p3): couldn't mount as ext3 due to feature incompatibilities
+uvcvideo: Found UVC 1.00 device HD WebCam (1bcf:2c67)
+input: HD WebCam as /devices/soc0/70090000.usb/usb1/1-2/1-2:1.0/input/input5
 EXT4-fs (mmcblk0p3): recovery complete
 EXT4-fs (mmcblk0p3): mounted filesystem with ordered data mode. Opts: (null)
-uvcvideo: Found UVC 1.00 device HD WebCam (1bcf:2c67)
 VFS: Mounted root (ext4 filesystem) on device 179:3.
-input: HD WebCam as /devices/soc0/70090000.usb/usb1/1-2/1-2:1.0/input/input5
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.970319] usb 1-3: new high-speed USB device number 3 using tegra-xusb
+root@(none):/# [    3.090702] usb 1-3: new high-speed USB device number 3 using tegra-xusb
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2017-04-12 22:56:22.801519042 -0700
+++ /tmp/foob	2017-04-12 22:56:22.801519042 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1800590 us
+Bootloader-Cpu Init at (time stamp): 1800686 us
 
 Pinmux changes applied in kernel way
 
@@ -62,20 +62,20 @@
 Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1800564  us
-Bootloader (AosInit) time: 733511 us
-Bootloader Main Init time: 180171 us
-BlockDevMgrInit time: 157550 us
+Miscellaneous avp time: 1800660  us
+Bootloader (AosInit) time: 733467 us
+Bootloader Main Init time: 180251 us
+BlockDevMgrInit time: 157492 us
 PartMgrInit time: 8 us
-StorMgrInit time: 31 us
+StorMgrInit time: 30 us
 FsMgrInit time: 2 us
-Partition Table load: 77111 us 
-Display init time: 574343 us
+Partition Table load: 77090 us 
+Display init time: 574743 us
 Kernel read time: 378 msec
 Load Kernel Size:   16304128 Bytes
 Bl Kernel ReadSpeed: 43132000 Bytes/sec
-Pre-Kernel Jump time: 1740401 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5842282 us
+Pre-Kernel Jump time: 1740553 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5844630 us
 Booting Linux on physical CPU 0x0
 Linux version 4.8.0-239303-g3477d16 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Fri Oct 7 01:12:37 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
@@ -184,7 +184,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(1.175:1): initialized
+audit: type=2000 audit(1.131:1): initialized
 workingset: timestamp_bits=46 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -251,8 +251,8 @@
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-+VDD_LED: supplied by +VDD_MUX
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
++VDD_LED: supplied by +VDD_MUX
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 +5V_USB_HS: supplied by +5V_SYS
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-04-12 22:56:22.809519075 -0700
+++ /tmp/foob	2017-04-12 22:56:22.809519075 -0700
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -48,9 +48,9 @@
 	 #################################################################
 	 #################################################################
 	 ##################
-	 4 MiB/s
+	 2.7 MiB/s
 done
-Bytes transferred = 7884576 (784f20 hex)
+Bytes transferred = 7889992 (786448 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -61,7 +61,7 @@
 done
 Bytes transferred = 21426 (53b2 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x784f20 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x786448 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103b1
@@ -69,7 +69,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc6-next-20170412-166416-g1267f8f (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Apr 12 01:04:46 PDT 2017
+Linux version 4.11.0-rc6-next-20170413-166612-gf4fa27cb (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Apr 12 22:50:01 PDT 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -79,13 +79,13 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c157e9c0, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c157ea80, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 65536 pages, LIFO batch:15
-percpu: Embedded 17 pages/cpu @ef7c4000 s38592 r8192 d22848 u69632
-pcpu-alloc: s38592 r8192 d22848 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @ef7c4000 s39040 r8192 d22400 u69632
+pcpu-alloc: s39040 r8192 d22400 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 260608
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=11eb23c1-01 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
@@ -102,8 +102,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1586568   (1562 kB)
-       .bss : 0xc158fd10 - 0xc15f5a08   ( 408 kB)
+      .data : 0xc1400000 - 0xc1586628   (1562 kB)
+       .bss : 0xc158fd10 - 0xc15f5b08   ( 408 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	RCU debugfs-based tracing is enabled.
@@ -344,8 +344,8 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: hctosys: unable to read the hardware clock
 hub 1-1:1.0: USB hub found
+rtc-em3027 2-0056: hctosys: unable to read the hardware clock
 hub 1-1:1.0: 4 ports detected
 Waiting for root device PARTUUID=11eb23c1-01...
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2017-04-12 22:56:22.817519107 -0700
+++ /tmp/foob	2017-04-12 22:56:22.817519107 -0700
@@ -45,21 +45,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #######################
-	 4.1 MiB/s
+	 ########################
+	 3.8 MiB/s
 done
-Bytes transferred = 6061720 (5c7e98 hex)
+Bytes transferred = 6066320 (5c9090 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 4.1 MiB/s
 done
 Bytes transferred = 21426 (53b2 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x5c7e98 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x5c9090 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103b1
@@ -68,7 +68,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc6-next-20170412-166416-g1267f8f (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Apr 12 01:04:26 PDT 2017
+Linux version 4.11.0-rc6-next-20170413-166612-gf4fa27cb (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Apr 12 22:49:33 PDT 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -77,13 +77,13 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0ead700, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0ead800, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 65536 pages, LIFO batch:15
-percpu: Embedded 17 pages/cpu @ef7c3000 s38528 r8192 d22912 u69632
-pcpu-alloc: s38528 r8192 d22912 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @ef7c3000 s38912 r8192 d22528 u69632
+pcpu-alloc: s38912 r8192 d22528 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 260608
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=11eb23c1-01 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
@@ -100,8 +100,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0eb7c30   ( 736 kB)
-       .bss : 0xc0ec1664 - 0xc0f0e334   ( 308 kB)
+      .data : 0xc0e00000 - 0xc0eb7d30   ( 736 kB)
+       .bss : 0xc0ec1664 - 0xc0f0e434   ( 308 kB)
 Preemptible hierarchical RCU implementation.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
 RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=2
@@ -329,8 +329,8 @@
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
- mmcblk1: p1 p2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
+ mmcblk1: p1 p2
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2017-04-12 22:56:22.825519141 -0700
+++ /tmp/foob	2017-04-12 22:56:22.825519141 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 721360 us
-Basic modules init took 348740 us
+Tegraboot started after 750988 us
+Basic modules init took 348535 us
 Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14392 us
-Carveout took 252713 us
-CPU initialization took 92307 us
-Total time taken by TegraBoot 708152 us
+Next stage binary read took 14382 us
+Carveout took 252715 us
+CPU initialization took 92320 us
+Total time taken by TegraBoot 707952 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.561] LPDDR4 Training: Number of tables = 10
+64b[0001.589] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-15659520 bytes read in 811 ms (18.4 MiB/s)
+15661568 bytes read in 811 ms (18.4 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-45142 bytes read in 458 ms (95.7 KiB/s)
+45142 bytes read in 459 ms (95.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 494 ms (3.1 MiB/s)
+1627787 bytes read in 495 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc6-next-20170412-284657-g1267f8f (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Apr 12 01:08:47 PDT 2017
+Linux version 4.11.0-rc6-next-20170413-284853-gf4fa27cb (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Apr 12 22:54:44 PDT 2017
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -163,8 +163,8 @@
 psci: PSCIv0.2 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: MIGRATE_INFO_TYPE not supported.
-percpu: Embedded 23 pages/cpu @ffff8000bff4d000 s56960 r8192 d29056 u94208
-pcpu-alloc: s56960 r8192 d29056 u94208 alloc=23*4096
+percpu: Embedded 24 pages/cpu @ffff8000bff49000 s58624 r8192 d31488 u98304
+pcpu-alloc: s58624 r8192 d31488 u98304 alloc=24*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
@@ -174,15 +174,15 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffff80007a000000-ffff80007dffffff]
-Memory: 2983388K/3131392K available (8828K kernel code, 1084K rwdata, 4228K rodata, 1088K init, 404K bss, 131620K reserved, 16384K cma-reserved)
+Memory: 2983372K/3131392K available (8828K kernel code, 1086K rwdata, 4232K rodata, 1088K init, 403K bss, 131636K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008920000   (  8832 KB)
     .rodata : 0xffff000008920000 - 0xffff000008d50000   (  4288 KB)
       .init : 0xffff000008d50000 - 0xffff000008e60000   (  1088 KB)
-      .data : 0xffff000008e60000 - 0xffff000008f6f200   (  1085 KB)
-       .bss : 0xffff000008f6f200 - 0xffff000008fd4338   (   405 KB)
+      .data : 0xffff000008e60000 - 0xffff000008f6fa00   (  1087 KB)
+       .bss : 0xffff000008f6fa00 - 0xffff000008fd47b8   (   404 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -266,14 +266,14 @@
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 PCI: CLS 0 bytes, default 128
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 8090b000
+kvm [1]: IDMAP page: 8090d000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
 kvm [1]: vgic interrupt IRQ1
 kvm [1]: virtual timer IRQ4
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.507:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.531:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -353,6 +353,6 @@
 devtmpfs: mounted
 Freeing unused kernel memory: 1088K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
-bash: no job control in th[    1.349253] random: fast init done
-is shell
+bash: no job contr[    1.367051] random: fast init done
+ol in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2017-04-12 22:56:22.833519173 -0700
+++ /tmp/foob	2017-04-12 22:56:22.837519190 -0700
@@ -1,7 +1,4 @@
-EFI parameters from FDT:
-efi: UEFI not found.
-cma: Reserved 16 MiB at 0x00000000f5400000
-earlycon: uart0 at MMIO 0x0000000070006000 (options '115200n8')
+5200n8')
 bootconsole [uart0] enabled
 NUMA: No NUMA configuration found
 NUMA: Faking a node at [mem 0x0000000000000000-0x000000013fffffff]
@@ -21,7 +18,7 @@
 psci: PSCIv1.0 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: MIGRATE_INFO_UP_CPU reported invalid physical ID (0x80000000)
-percpu: Embedded 23 pages/cpu @ffff8000bff41000 s56960 r8192 d29056 u94208
+percpu: Embedded 24 pages/cpu @ffff8000bff3d000 s58624 r8192 d31488 u98304
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
 CPU features: enabling workaround for ARM erratum 834220
@@ -30,15 +27,15 @@
 Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2837576K/2988032K available (8828K kernel code, 1084K rwdata, 4228K rodata, 1088K init, 404K bss, 134072K reserved, 16384K cma-reserved)
+Memory: 2837556K/2988032K available (8828K kernel code, 1086K rwdata, 4232K rodata, 1088K init, 403K bss, 134092K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008920000   (  8832 KB)
     .rodata : 0xffff000008920000 - 0xffff000008d50000   (  4288 KB)
       .init : 0xffff000008d50000 - 0xffff000008e60000   (  1088 KB)
-      .data : 0xffff000008e60000 - 0xffff000008f6f200   (  1085 KB)
-       .bss : 0xffff000008f6f200 - 0xffff000008fd4338   (   405 KB)
+      .data : 0xffff000008e60000 - 0xffff000008f6fa00   (  1087 KB)
+       .bss : 0xffff000008f6fa00 - 0xffff000008fd47b8   (   404 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -75,7 +72,7 @@
 console [tty0] enabled
 bootconsole [uart0] disabled
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc6-next-20170412-284657-g1267f8f (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Apr 12 01:08:47 PDT 2017
+Linux version 4.11.0-rc6-next-20170413-284853-gf4fa27cb (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Apr 12 22:54:44 PDT 2017
 Boot CPU: AArch64 Processor [411fd071]
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
@@ -100,7 +97,7 @@
 psci: PSCIv1.0 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: MIGRATE_INFO_UP_CPU reported invalid physical ID (0x80000000)
-percpu: Embedded 23 pages/cpu @ffff8000bff41000 s56960 r8192 d29056 u94208
+percpu: Embedded 24 pages/cpu @ffff8000bff3d000 s58624 r8192 d31488 u98304
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
 CPU features: enabling workaround for ARM erratum 834220
@@ -109,15 +106,15 @@
 Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2837576K/2988032K available (8828K kernel code, 1084K rwdata, 4228K rodata, 1088K init, 404K bss, 134072K reserved, 16384K cma-reserved)
+Memory: 2837556K/2988032K available (8828K kernel code, 1086K rwdata, 4232K rodata, 1088K init, 403K bss, 134092K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008920000   (  8832 KB)
     .rodata : 0xffff000008920000 - 0xffff000008d50000   (  4288 KB)
       .init : 0xffff000008d50000 - 0xffff000008e60000   (  1088 KB)
-      .data : 0xffff000008e60000 - 0xffff000008f6f200   (  1085 KB)
-       .bss : 0xffff000008f6f200 - 0xffff000008fd4338   (   405 KB)
+      .data : 0xffff000008e60000 - 0xffff000008f6fa00   (  1087 KB)
+       .bss : 0xffff000008f6fa00 - 0xffff000008fd47b8   (   404 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -222,9 +219,9 @@
 RPC: Registered tcp transport module.
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 Unpacking initramfs...
-Freeing initrd memory: 2360K
+Freeing initrd memory: 2364K
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 8090b000
+kvm [1]: IDMAP page: 8090d000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
@@ -321,22 +318,22 @@
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 BGND3R 29.1 GiB 
 mmcblk0boot0: mmc0:0001 BGND3R partition 1 4.00 MiB
-max77686-rtc max77620-rtc: setting system clock to 2001-01-21 13:35:37 UTC (980084137)
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
+max77686-rtc max77620-rtc: setting system clock to 2001-01-22 11:21:33 UTC (980162493)
 PP1200_AVDD: disabling
 PP1200_RCAM: disabling
 PP2800L_RCAM: disabling
 PP2800L_FCAM: disabling
 PP_LDO6: disabling
 PPLCD_VDD: disabling
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
-tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 PP1800_LCDIO: disabling
 PP1800_CAM: disabling
 ALSA device list:
   No soundcards found.
 Freeing unused kernel memory: 1088K
+tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 
 System initialization...
 
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-04-12 22:56:22.841519206 -0700
+++ /tmp/foob	2017-04-12 22:56:22.845519222 -0700
@@ -42,18 +42,18 @@
 	 ##################
 	 3.2 MiB/s
 done
-Bytes transferred = 7884576 (784f20 hex)
+Bytes transferred = 7889992 (786448 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.5 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 51804 (ca5c hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x784f20 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x786448 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa5b ... OK
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc6-next-20170412-166416-g1267f8f (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Apr 12 01:04:46 PDT 2017
+Linux version 4.11.0-rc6-next-20170413-166612-gf4fa27cb (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Apr 12 22:50:01 PDT 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -71,13 +71,13 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c157e9c0, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c157ea80, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 17 pages/cpu @eef88000 s38592 r8192 d22848 u69632
-pcpu-alloc: s38592 r8192 d22848 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @eef88000 s39040 r8192 d22400 u69632
+pcpu-alloc: s39040 r8192 d22400 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=edbe1840-3633-c751-7993-b0ac834f81e0 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
@@ -94,8 +94,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1586568   (1562 kB)
-       .bss : 0xc158fd10 - 0xc15f5a08   ( 408 kB)
+      .data : 0xc1400000 - 0xc1586628   (1562 kB)
+       .bss : 0xc158fd10 - 0xc15f5b08   ( 408 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU debugfs-based tracing is enabled.
@@ -308,17 +308,17 @@
 sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
 +VDD_5V_HDMI: supplied by sys_3v3,vdd_3v3_alw
 tegra-pcie 3000.pcie-controller: 2x3 configuration
-tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
 mmc1: new high speed MMC card at address 0001
+tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
 mmcblk1: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
-mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB
 mmc0: new ultra high speed DDR50 SDHC card at address aaaa
+mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB
 mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
 Primary GPT is invalid, using alternate GPT.
- mmcblk1: p1 p2 p3 p4 p5
 tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
+ mmcblk1: p1 p2 p3 p4 p5
  mmcblk0: p1 p2 p3
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -391,5 +391,5 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.074042] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.084093] usb 2-1: new high-speed USB device number 2 using tegra-ehci
 asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0f:67:60
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2017-04-12 22:56:22.849519239 -0700
+++ /tmp/foob	2017-04-12 22:56:22.853519255 -0700
@@ -37,21 +37,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #######################
-	 3.2 MiB/s
+	 ########################
+	 3.1 MiB/s
 done
-Bytes transferred = 6061720 (5c7e98 hex)
+Bytes transferred = 6066320 (5c9090 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.3 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 51804 (ca5c hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5c7e98 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5c9090 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa5b ... OK
@@ -60,7 +60,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc6-next-20170412-166416-g1267f8f (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Apr 12 01:04:26 PDT 2017
+Linux version 4.11.0-rc6-next-20170413-166612-gf4fa27cb (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Apr 12 22:49:33 PDT 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -69,13 +69,13 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0ead700, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0ead800, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 17 pages/cpu @eef87000 s38528 r8192 d22912 u69632
-pcpu-alloc: s38528 r8192 d22912 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @eef87000 s38912 r8192 d22528 u69632
+pcpu-alloc: s38912 r8192 d22528 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=edbe1840-3633-c751-7993-b0ac834f81e0 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
@@ -92,8 +92,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0eb7c30   ( 736 kB)
-       .bss : 0xc0ec1664 - 0xc0f0e334   ( 308 kB)
+      .data : 0xc0e00000 - 0xc0eb7d30   ( 736 kB)
+       .bss : 0xc0ec1664 - 0xc0f0e434   ( 308 kB)
 Preemptible hierarchical RCU implementation.
 NR_IRQS:16 nr_irqs:16 16
 /interrupt-controller@60004000: 160 interrupts forwarded to /interrupt-controller@50041000
@@ -385,4 +385,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.360069] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.380135] usb 2-1: new high-speed USB device number 2 using tegra-ehci
