Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2017-09-17 19:57:36.911514483 -0700
+++ /tmp/foob	2017-09-17 19:57:36.915514499 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-next-20170915-318284-g1f183459b514 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Thu Sep 14 20:40:05 PDT 2017
+Linux version 4.14.0-rc1-next-20170918-318337-gfc2e8b1a47c1 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sun Sep 17 19:55:25 PDT 2017
 Boot CPU: AArch64 Processor [411fd070]
 Machine model: linux,dummy-virt
 Memory limited to 1024MB
@@ -116,7 +116,7 @@
 hw perfevents: enabled with armv8_pmuv3 PMU driver, 1 counters available
 kvm [1]: HYP mode not available
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.808:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.840:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -181,7 +181,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2017-09-15 03:41:17 UTC (1505446877)
+rtc-pl031 9010000.pl031: setting system clock to 2017-09-18 02:56:32 UTC (1505703392)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-09-17 19:57:36.947514637 -0700
+++ /tmp/foob	2017-09-17 19:57:36.951514655 -0700
@@ -40,7 +40,7 @@
 	 #################################################################
 	 #################################################################
 	 #####################################
-	 3.1 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 8167944 (7ca208 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-next-20170915-200043-g1f183459 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Sep 14 20:36:30 PDT 2017
+Linux version 4.14.0-rc1-next-20170918-200096-gfc2e8b1 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Sun Sep 17 19:49:29 PDT 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -72,7 +72,7 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c158eb80, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c158eb40, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -86,7 +86,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993476K/2096128K available (10240K kernel code, 1628K rwdata, 4508K rodata, 2048K init, 405K bss, 37116K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993480K/2096128K available (10240K kernel code, 1627K rwdata, 4508K rodata, 2048K init, 405K bss, 37112K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -96,8 +96,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1597028   (1629 kB)
-       .bss : 0xc15a0bc0 - 0xc160637c   ( 406 kB)
+      .data : 0xc1400000 - 0xc1596fe8   (1628 kB)
+       .bss : 0xc159fbc0 - 0xc160537c   ( 406 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -136,7 +136,7 @@
 SMP: Total of 4 processors activated (96.00 BogoMIPS).
 CPU: All CPU(s) started in SVC mode.
 devtmpfs: initialized
-Built 1 zonelists, mobility grouping on.  Total pages: 514753
+Built 1 zonelists, mobility grouping on.  Total pages: 514754
 VFP support v0.3: implementor 41 architecture 4 part 30 variant f rev 0
 clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 19112604462750000 ns
 futex hash table entries: 1024 (order: 4, 65536 bytes)
@@ -412,7 +412,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-06 10:31:51 UTC (949833111)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-09 09:42:57 UTC (950089377)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -424,5 +424,5 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.109131] usb 1-1: new high-speed USB device number 2 using tegra-ehci
-asix 1-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0d:b5:16
+root@(none):/# [    4.089109] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+[    4.
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2017-09-17 19:57:36.971514740 -0700
+++ /tmp/foob	2017-09-17 19:57:36.975514757 -0700
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra114 (Dalmore) # 
 Tegra114 (Dalmore) # usb reset
 (Re)start USB...
@@ -40,18 +40,18 @@
 	 #######################################
 	 3.3 MiB/s
 done
-Bytes transferred = 6293000 (600608 hex)
+Bytes transferred = 6292552 (600448 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.8 MiB/s
+	 2.1 MiB/s
 done
 Bytes transferred = 35036 (88dc hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x600608 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x600448 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8db ... OK
@@ -60,7 +60,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-next-20170915-200043-g1f183459 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Sep 14 20:35:58 PDT 2017
+Linux version 4.14.0-rc1-next-20170918-200096-gfc2e8b1 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Sun Sep 17 19:49:39 PDT 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -407,7 +407,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-06 10:29:17 UTC (949832957)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-09 09:45:26 UTC (950089526)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -421,4 +421,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# 
\ No newline at end of file
+root@(none):/# [    3.737232] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-09-17 19:57:36.983514790 -0700
+++ /tmp/foob	2017-09-17 19:57:36.987514807 -0700
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MMC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -35,7 +35,7 @@
 	 #################################################################
 	 #################################################################
 	 #####################################
-	 3.9 MiB/s
+	 3.5 MiB/s
 done
 Bytes transferred = 8167944 (7ca208 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
@@ -44,7 +44,7 @@
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.8 MiB/s
+	 3.6 MiB/s
 done
 Bytes transferred = 67173 (10665 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
@@ -56,7 +56,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-next-20170915-200043-g1f183459 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Sep 14 20:36:30 PDT 2017
+Linux version 4.14.0-rc1-next-20170918-200096-gfc2e8b1 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Sun Sep 17 19:49:29 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -67,7 +67,7 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c158eb80, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c158eb40, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -81,7 +81,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993356K/2096128K available (10240K kernel code, 1628K rwdata, 4508K rodata, 2048K init, 405K bss, 37236K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993360K/2096128K available (10240K kernel code, 1627K rwdata, 4508K rodata, 2048K init, 405K bss, 37232K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -91,8 +91,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1597028   (1629 kB)
-       .bss : 0xc15a0bc0 - 0xc160637c   ( 406 kB)
+      .data : 0xc1400000 - 0xc1596fe8   (1628 kB)
+       .bss : 0xc159fbc0 - 0xc160537c   ( 406 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -131,7 +131,7 @@
 SMP: Total of 4 processors activated (96.00 BogoMIPS).
 CPU: All CPU(s) started in SVC mode.
 devtmpfs: initialized
-Built 1 zonelists, mobility grouping on.  Total pages: 514723
+Built 1 zonelists, mobility grouping on.  Total pages: 514724
 VFP support v0.3: implementor 41 architecture 4 part 30 variant f rev 0
 clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 19112604462750000 ns
 futex hash table entries: 1024 (order: 4, 65536 bytes)
@@ -530,7 +530,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:10 UTC (946684930)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:14 UTC (946684814)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2017-09-17 19:57:37.003514875 -0700
+++ /tmp/foob	2017-09-17 19:57:37.007514892 -0700
@@ -35,18 +35,18 @@
 	 #######################################
 	 4 MiB/s
 done
-Bytes transferred = 6293000 (600608 hex)
+Bytes transferred = 6292552 (600448 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 4 MiB/s
+	 3.8 MiB/s
 done
 Bytes transferred = 67173 (10665 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x600608 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x600448 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013664
@@ -55,7 +55,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-next-20170915-200043-g1f183459 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Sep 14 20:35:58 PDT 2017
+Linux version 4.14.0-rc1-next-20170918-200096-gfc2e8b1 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Sun Sep 17 19:49:39 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -538,7 +538,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:14 UTC (946684814)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:22 UTC (946684942)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-09-17 19:57:37.027514978 -0700
+++ /tmp/foob	2017-09-17 19:57:37.031514995 -0700
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -64,7 +64,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-next-20170915-200043-g1f183459 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Sep 14 20:36:30 PDT 2017
+Linux version 4.14.0-rc1-next-20170918-200096-gfc2e8b1 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Sun Sep 17 19:49:29 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -76,7 +76,7 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c158eb80, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c158eb40, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -90,7 +90,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1994372K/2097148K available (10240K kernel code, 1628K rwdata, 4508K rodata, 2048K init, 405K bss, 37240K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1994376K/2097148K available (10240K kernel code, 1627K rwdata, 4508K rodata, 2048K init, 405K bss, 37236K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -100,8 +100,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1597028   (1629 kB)
-       .bss : 0xc15a0bc0 - 0xc160637c   ( 406 kB)
+      .data : 0xc1400000 - 0xc1596fe8   (1628 kB)
+       .bss : 0xc159fbc0 - 0xc160537c   ( 406 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -140,7 +140,7 @@
 SMP: Total of 4 processors activated (96.00 BogoMIPS).
 CPU: All CPU(s) started in SVC mode.
 devtmpfs: initialized
-Built 1 zonelists, mobility grouping on.  Total pages: 514977
+Built 1 zonelists, mobility grouping on.  Total pages: 514978
 VFP support v0.3: implementor 41 architecture 4 part 30 variant f rev 0
 clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 19112604462750000 ns
 futex hash table entries: 1024 (order: 4, 65536 bytes)
@@ -378,8 +378,6 @@
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc0: Unknown controller version (3). You may experience problems.
 mmc0: Invalid maximum block size, assuming 512 bytes
-elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
-input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 mmc0: SDHCI controller on 700b0600.sdhci [700b0600.sdhci] using ADMA
 ledtrig-cpu: registered to indicate activity on CPUs
 usbcore: registered new interface driver usbhid
@@ -407,30 +405,29 @@
 +5V_USB_SS: supplied by +5V_SYS
 +3.3V_PANEL: supplied by +3.3V_RUN
 +3.3V_LP0: supplied by +3.3V_SYS
-mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
-mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
-mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
-mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
-elan_i2c 1-0015: failed to read report data: -121
-elan_i2c 1-0015: failed to read report data: -121
+elan_i2c 1-0015: cannot get device descriptor: -121
 +5V_HDMI_CON: supplied by +5V_SYS
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
  usb3-1: usb3-1 supply vbus not found, using dummy regulator
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
 sdhci-tegra 700b0000.sdhci: allocated mmc-pwrseq
+mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
+mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
+mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
+mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
 mmc1: Unknown controller version (3). You may experience problems.
 mmc1: Invalid maximum block size, assuming 512 bytes
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 mmc1: Switching to 3.3V signalling voltage failed
 mmc1: SDHCI controller on 700b0000.sdhci [700b0000.sdhci] using ADMA
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
-elan_i2c 1-0015: invalid report id data (ff)
 mmc2: Unknown controller version (3). You may experience problems.
 mmc2: Invalid maximum block size, assuming 512 bytes
+elan_i2c 1-0015: device initialize failed: -121
 mmc1: new high speed SDIO card at address 0001
 mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
@@ -443,11 +440,13 @@
 GPT:3014591 != 15564799
 GPT: Use GNU Parted to correct GPT errors.
  mmcblk2: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
+elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
+input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 Console: switching to colour frame buffer device 170x48
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 03:49:30 UTC (946698570)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:23 UTC (946684823)
 +1.8V_RUN_CAM: disabling
 +3.3V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2017-09-17 19:57:37.047515063 -0700
+++ /tmp/foob	2017-09-17 19:57:37.051515079 -0700
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -41,20 +41,20 @@
 	 #################################################################
 	 #################################################################
 	 #######################################
-	 3.6 MiB/s
+	 2.9 MiB/s
 done
-Bytes transferred = 6293000 (600608 hex)
+Bytes transferred = 6292552 (600448 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.4 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 68203 (10a6b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x600608 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x600448 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a6a
@@ -63,7 +63,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-next-20170915-200043-g1f183459 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Sep 14 20:35:58 PDT 2017
+Linux version 4.14.0-rc1-next-20170918-200096-gfc2e8b1 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Sun Sep 17 19:49:39 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -313,89 +313,105 @@
 kernel BUG at drivers/platform/chrome/cros_ec_proto.c:34!
 Internal error: Oops - BUG: 0 [#1] PREEMPT SMP ARM
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.13.0-next-20170915-200043-g1f183459 #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.14.0-rc1-next-20170918-200096-gfc2e8b1 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 task: ee891740 task.stack: ee8a6000
 PC is at cros_ec_prepare_tx+0x168/0x16c
 LR is at cros_ec_pkt_xfer_spi+0x18/0x474
-pc : [<c06d7830>]    lr : [<c05453fc>]    psr: 20000013
+pc : [<c06d7878>]    lr : [<c0545464>]    psr: 20000013
 sp : ee8a7bd8  ip : 00000000  fp : 00000000
-r10: c05453e4  r9 : 00000000  r8 : eefe503c
-r7 : ee290000  r6 : ee290000  r5 : ee290000  r4 : ee247d90
-r3 : 0000ffff  r2 : 00000000  r1 : 0000ffff  r0 : ee247d90
+r10: c054544c  r9 : 00000000  r8 : eefe503c
+r7 : ee291000  r6 : ee291000  r5 : ee291000  r4 : ee248d90
+r3 : 0000ffff  r2 : 00000000  r1 : 0000ffff  r0 : ee248d90
 Flags: nzCv  IRQs on  FIQs on  Mode SVC_32  ISA ARM  Segment none
 Control: 10c5387d  Table: 8000406a  DAC: 00000051
 Process swapper/0 (pid: 1, stack limit = 0xee8a6210)
 Stack: (0xee8a7bd8 to 0xee8a8000)
-7bc0:                                                       ee247d90 ee290000
-7be0: ee290090 c05453fc c0976b2c 00000004 ee8a7c5c ee8a7c5c ee292c00 00000000
-7c00: c0591840 00000000 00000008 00000008 00000000 ee8a7c14 ee8a7c14 00000000
-7c20: ee8a7c20 ee8a7c20 ee290050 ee290780 00000008 00000000 00000000 00000000
-7c40: c0e91c50 c02385f8 40002800 000003e0 eef8cc50 c023aa68 002dc6c0 ee8a7bf0
-7c60: ee8a7bf0 00000000 c06d7660 005c5200 3b9aca00 000f4240 00000000 ee8a7ca8
-7c80: 00000001 00000501 c05453e4 c097b3bc 00000000 ee2907c0 a0000013 c06d7660
-7ca0: ee290000 ee247d90 ee290000 ee292c00 ee290000 eefe503c 00000000 c05453e4
-7cc0: 00000000 c06d75bc 0000400b ee247d90 ee290000 ee292c00 0000000c eefe503c
-7ce0: c0ff8e7c c06d79d8 00000010 ee247d90 ee292c00 00000000 00000000 eefe503c
-7d00: c0f6b7d8 c0544e88 ee8a7d14 ee292c00 00000000 00000001 ee292c00 ee247d90
-7d20: ee290090 c0545238 00000000 00000000 00000000 000007d0 ee292c00 00000000
-7d40: c0f6b7c8 00000000 00000000 c0592284 ee292c00 c100a204 c100a208 c051e144
-7d60: 00000000 ee8a7d98 c051e2d4 00000001 c100a1e0 00000000 00000000 c051c764
-7d80: ee9f7170 ee2375b8 ee292c00 ee292c34 c0f7249c c051de84 ee292c00 00000001
-7da0: 00000000 ee292c00 ee292c00 c0f7249c ee10d000 c051d4c8 ee292c00 00000000
-7dc0: ee292c08 c051bb0c ee292c00 c0595188 00000010 ee292c00 ee292c00 ee10d000
-7de0: 00000000 eea8be10 eefe508c 00000001 00000000 c05920d8 002dc6c0 ee10d000
-7e00: eefe503c 00000000 ee292c00 c0594074 00000000 014000c0 00000000 c0593a60
-7e20: 00000063 002dc6c0 ee10d000 ee290350 eea8be00 eea8be10 40000000 00000153
-7e40: 00000000 c0594534 eefe4d2c ee10d348 ee10d000 eea8be00 eea8be10 c05950c0
-7e60: 00000000 ee10d348 ffffffed eea8be10 fffffdfb c0f72dfc 00000000 00000000
-7e80: c0f72dfc c051fa34 c051f9e4 eea8be10 c100a204 c100a208 00000000 c051e144
-7ea0: eea8be10 c0f72dfc eea8be44 00000000 00000111 c0e6b83c c0e005a4 c051e29c
-7ec0: 00000000 c0f72dfc c051e1f0 c051c800 ee8c955c eea1c934 c0f72dfc ee289e80
-7ee0: c0f69d88 c051d69c c0c1ccac c0f72dfc 00000006 c0f72dfc 00000006 c0e38834
-7f00: 00000000 c051ee14 c051f68c c0e83c84 00000006 c0101b8c 00000000 c0cba380
-7f20: 00000100 efffcb45 c0cba380 00000111 efffcb53 c013e410 c0cb8e2c 00000000
+7bc0:                                                       ee248d90 ee291000
+7be0: ee291090 c0545464 c0976bcc 00000004 ee8a7c5c ee8a7c5c ee293c00 00000000
+7c00: c0591888 00000000 00000008 00000008 00000000 ee8a7c14 ee8a7c14 00000000
+7c20: ee8a7c20 ee8a7c20 ee291050 ee291780 00000008 00000000 00000000 00000000
+7c40: c0e91c50 c02385fc 40002800 000003e0 eef8cc50 c023aa6c 002dc6c0 ee8a7bf0
+7c60: ee8a7bf0 00000000 c06d76a8 005c5220 3b9aca00 000f4240 00000000 ee8a7ca8
+7c80: 00000001 00000501 c054544c c097b45c 00000000 ee2917c0 a0000013 c06d76a8
+7ca0: ee291000 ee248d90 ee291000 ee293c00 ee291000 eefe503c 00000000 c054544c
+7cc0: 00000000 c06d7604 0000400b ee248d90 ee291000 ee293c00 0000000c eefe503c
+7ce0: c0ff8e7c c06d7a20 00000010 ee248d90 ee293c00 00000000 00000000 eefe503c
+7d00: c0f6b7c0 c0544ef0 ee8a7d14 ee293c00 00000000 00000001 ee293c00 ee248d90
+7d20: ee291090 c05452a0 00000000 00000000 00000000 000007d0 ee293c00 00000000
+7d40: c0f6b7b0 00000000 00000000 c05922cc ee293c00 c100a204 c100a208 c051e1a8
+7d60: 00000000 ee8a7d98 c051e338 00000001 c100a1e0 00000000 00000000 c051c7c8
+7d80: ee9f7170 ee2385b8 ee293c00 ee293c34 c0f72484 c051dee8 ee293c00 00000001
+7da0: 00000000 ee293c00 ee293c00 c0f72484 ee07b000 c051d52c ee293c00 00000000
+7dc0: ee293c08 c051bb70 ee293c00 c05951d0 00000010 ee293c00 ee293c00 ee07b000
+7de0: 00000000 eea8be10 eefe508c 00000001 00000000 c0592120 002dc6c0 ee07b000
+7e00: eefe503c 00000000 ee293c00 c05940bc 00000000 014000c0 00000000 c0593aa8
+7e20: 00000063 002dc6c0 ee07b000 ee291350 eea8be00 eea8be10 40000000 00000153
+7e40: 00000000 c059457c eefe4d2c ee07b348 ee07b000 eea8be00 eea8be10 c0595108
+7e60: 00000000 ee07b348 ffffffed eea8be10 fffffdfb c0f72de4 00000000 00000000
+7e80: c0f72de4 c051fa98 c051fa48 eea8be10 c100a204 c100a208 00000000 c051e1a8
+7ea0: eea8be10 c0f72de4 eea8be44 00000000 00000111 c0e6b83c c0e005a4 c051e300
+7ec0: 00000000 c0f72de4 c051e254 c051c864 ee8c955c eea1c934 c0f72de4 ee28ae80
+7ee0: c0f69d70 c051d700 c0c1ccb8 c0f72de4 00000006 c0f72de4 00000006 c0e38834
+7f00: 00000000 c051ee78 c051f6f0 c0e83c84 00000006 c0101b8c 00000000 c0cba3ac
+7f20: 00000100 efffcb45 c0cba3ac 00000111 efffcb53 c013e410 c0cb8e58 00000000
 7f40: 00000006 00000006 efffcb45 efffcb4a 00000000 efffcac0 00000006 c0e6b82c
 7f60: c0e83c84 00000006 c0e6b830 c0fcd540 00000111 c0e00da8 00000006 00000006
-7f80: 00000000 c0e005a4 00000000 c097490c 00000000 00000000 00000000 00000000
-7fa0: 00000000 c0974914 00000000 c0107ce0 00000000 00000000 00000000 00000000
+7f80: 00000000 c0e005a4 00000000 c09749ac 00000000 00000000 00000000 00000000
+7fa0: 00000000 c09749b4 00000000 c0107ce0 00000000 00000000 00000000 00000000
 7fc0: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
 7fe0: 00000000 00000000 00000000 00000000 00000013 00000000 00000000 00000000
-[<c06d7830>] (cros_ec_prepare_tx) from [<c05453fc>] (cros_ec_pkt_xfer_spi+0x18/0x474)
-[<c05453fc>] (cros_ec_pkt_xfer_spi) from [<c06d75bc>] (send_command+0x20/0xd8)
-[<c06d75bc>] (send_command) from [<c06d79d8>] (cros_ec_query_all+0x1a4/0x398)
-[<c06d79d8>] (cros_ec_query_all) from [<c0544e88>] (cros_ec_register+0x94/0x21c)
-[<c0544e88>] (cros_ec_register) from [<c0545238>] (cros_ec_spi_probe+0x108/0x150)
-[<c0545238>] (cros_ec_spi_probe) from [<c0592284>] (spi_drv_probe+0x7c/0xa8)
-[<c0592284>] (spi_drv_probe) from [<c051e144>] (driver_probe_device+0x238/0x2e4)
-[<c051e144>] (driver_probe_device) from [<c051c764>] (bus_for_each_drv+0x44/0x8c)
-[<c051c764>] (bus_for_each_drv) from [<c051de84>] (__device_attach+0x9c/0x100)
-[<c051de84>] (__device_attach) from [<c051d4c8>] (bus_probe_device+0x84/0x8c)
-[<c051d4c8>] (bus_probe_device) from [<c051bb0c>] (device_add+0x380/0x528)
-[<c051bb0c>] (device_add) from [<c05920d8>] (spi_add_device+0x88/0x130)
-[<c05920d8>] (spi_add_device) from [<c0594074>] (spi_register_controller+0x428/0x8b4)
-[<c0594074>] (spi_register_controller) from [<c0594534>] (devm_spi_register_controller+0x34/0x70)
-[<c0594534>] (devm_spi_register_controller) from [<c05950c0>] (tegra_spi_probe+0x298/0x35c)
-[<c05950c0>] (tegra_spi_probe) from [<c051fa34>] (platform_drv_probe+0x50/0xb0)
-[<c051fa34>] (platform_drv_probe) from [<c051e144>] (driver_probe_device+0x238/0x2e4)
-[<c051e144>] (driver_probe_device) from [<c051e29c>] (__driver_attach+0xac/0xb0)
-[<c051e29c>] (__driver_attach) from [<c051c800>] (bus_for_each_dev+0x54/0x88)
-[<c051c800>] (bus_for_each_dev) from [<c051d69c>] (bus_add_driver+0xe4/0x1f0)
-[<c051d69c>] (bus_add_driver) from [<c051ee14>] (driver_register+0x78/0xf4)
-[<c051ee14>] (driver_register) from [<c0101b8c>] (do_one_initcall+0x3c/0x168)
+[<c06d7878>] (cros_ec_prepare_tx) from [<c0545464>] (cros_ec_pkt_xfer_spi+0x18/0x474)
+[<c0545464>] (cros_ec_pkt_xfer_spi) from [<c06d7604>] (send_command+0x20/0xd8)
+[<c06d7604>] (send_command) from [<c06d7a20>] (cros_ec_query_all+0x1a4/0x398)
+[<c06d7a20>] (cros_ec_query_all) from [<c0544ef0>] (cros_ec_register+0x94/0x21c)
+[<c0544ef0>] (cros_ec_register) from [<c05452a0>] (cros_ec_spi_probe+0x108/0x150)
+[<c05452a0>] (cros_ec_spi_probe) from [<c05922cc>] (spi_drv_probe+0x7c/0xa8)
+[<c05922cc>] (spi_drv_probe) from [<c051e1a8>] (driver_probe_device+0x238/0x2e4)
+[<c051e1a8>] (driver_probe_device) from [<c051c7c8>] (bus_for_each_drv+0x44/0x8c)
+[<c051c7c8>] (bus_for_each_drv) from [<c051dee8>] (__device_attach+0x9c/0x100)
+[<c051dee8>] (__device_attach) from [<c051d52c>] (bus_probe_device+0x84/0x8c)
+[<c051d52c>] (bus_probe_device) from [<c051bb70>] (device_add+0x380/0x528)
+[<c051bb70>] (device_add) from [<c0592120>] (spi_add_device+0x88/0x130)
+[<c0592120>] (spi_add_device) from [<c05940bc>] (spi_register_controller+0x428/0x8b4)
+[<c05940bc>] (spi_register_controller) from [<c059457c>] (devm_spi_register_controller+0x34/0x70)
+[<c059457c>] (devm_spi_register_controller) from [<c0595108>] (tegra_spi_probe+0x298/0x35c)
+[<c0595108>] (tegra_spi_probe) from [<c051fa98>] (platform_drv_probe+0x50/0xb0)
+[<c051fa98>] (platform_drv_probe) from [<c051e1a8>] (driver_probe_device+0x238/0x2e4)
+[<c051e1a8>] (driver_probe_device) from [<c051e300>] (__driver_attach+0xac/0xb0)
+[<c051e300>] (__driver_attach) from [<c051c864>] (bus_for_each_dev+0x54/0x88)
+[<c051c864>] (bus_for_each_dev) from [<c051d700>] (bus_add_driver+0xe4/0x1f0)
+[<c051d700>] (bus_add_driver) from [<c051ee78>] (driver_register+0x78/0xf4)
+[<c051ee78>] (driver_register) from [<c0101b8c>] (do_one_initcall+0x3c/0x168)
 [<c0101b8c>] (do_one_initcall) from [<c0e00da8>] (kernel_init_freeable+0x10c/0x1d0)
-[<c0e00da8>] (kernel_init_freeable) from [<c0974914>] (kernel_init+0x8/0x114)
-[<c0974914>] (kernel_init) from [<c0107ce0>] (ret_from_fork+0x14/0x34)
+[<c0e00da8>] (kernel_init_freeable) from [<c09749b4>] (kernel_init+0x8/0x114)
+[<c09749b4>] (kernel_init) from [<c0107ce0>] (ret_from_fork+0x14/0x34)
 Code: e2800008 e8bd8070 e7f001f2 e7f001f2 (e7f001f2) 
----[ end trace 6ee941a7c107634c ]---
+---[ end trace 4db06ce4eaceff3f ]---
 Kernel panic - not syncing: Attempted to kill init! exitcode=0x0000000b
 
+CPU1: stopping
+CPU: 1 PID: 0 Comm: swapper/1 Tainted: G      D         4.14.0-rc1-next-20170918-200096-gfc2e8b1 #1
+Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
+[<c01110d0>] (unwind_backtrace) from [<c010c090>] (show_stack+0x10/0x14)
+[<c010c090>] (show_stack) from [<c09612c0>] (dump_stack+0x84/0xa4)
+[<c09612c0>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
+[<c010f61c>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
+[<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
+Exception stack(0xee8dff70 to 0xee8dffb8)
+ff60:                                     00000001 00000000 ee8dffc8 c011a380
+ff80: c0f03ccc c0f03c68 c0e8d378 c0f03cd8 c0fbe23d 00000000 00000001 00000000
+ffa0: 2e10c000 ee8dffc0 c0108758 c010875c 60000013 ffffffff
+[<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
+[<c010875c>] (arch_cpu_idle) from [<c0160cc4>] (do_idle+0x148/0x204)
+[<c0160cc4>] (do_idle) from [<c0161008>] (cpu_startup_entry+0x18/0x1c)
+[<c0161008>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
 CPU3: stopping
-CPU: 3 PID: 0 Comm: swapper/3 Tainted: G      D         4.13.0-next-20170915-200043-g1f183459 #1
+CPU: 3 PID: 0 Comm: swapper/3 Tainted: G      D         4.14.0-rc1-next-20170918-200096-gfc2e8b1 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c01110d0>] (unwind_backtrace) from [<c010c090>] (show_stack+0x10/0x14)
-[<c010c090>] (show_stack) from [<c0961220>] (dump_stack+0x84/0xa4)
-[<c0961220>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
+[<c010c090>] (show_stack) from [<c09612c0>] (dump_stack+0x84/0xa4)
+[<c09612c0>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
 [<c010f61c>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
 Exception stack(0xee8e3f70 to 0xee8e3fb8)
@@ -403,15 +419,15 @@
 3f80: c0f03ccc c0f03c68 c0e8d378 c0f03cd8 c0fbe23d 00000000 00000001 00000000
 3fa0: 2e12e000 ee8e3fc0 c0108758 c010875c 60000013 ffffffff
 [<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
-[<c010875c>] (arch_cpu_idle) from [<c0160cc0>] (do_idle+0x148/0x204)
-[<c0160cc0>] (do_idle) from [<c0161004>] (cpu_startup_entry+0x18/0x1c)
-[<c0161004>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
+[<c010875c>] (arch_cpu_idle) from [<c0160cc4>] (do_idle+0x148/0x204)
+[<c0160cc4>] (do_idle) from [<c0161008>] (cpu_startup_entry+0x18/0x1c)
+[<c0161008>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
 CPU2: stopping
-CPU: 2 PID: 0 Comm: swapper/2 Tainted: G      D         4.13.0-next-20170915-200043-g1f183459 #1
+CPU: 2 PID: 0 Comm: swapper/2 Tainted: G      D         4.14.0-rc1-next-20170918-200096-gfc2e8b1 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c01110d0>] (unwind_backtrace) from [<c010c090>] (show_stack+0x10/0x14)
-[<c010c090>] (show_stack) from [<c0961220>] (dump_stack+0x84/0xa4)
-[<c0961220>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
+[<c010c090>] (show_stack) from [<c09612c0>] (dump_stack+0x84/0xa4)
+[<c09612c0>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
 [<c010f61c>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
 Exception stack(0xee8e1f70 to 0xee8e1fb8)
@@ -419,24 +435,8 @@
 1f80: c0f03ccc c0f03c68 c0e8d378 c0f03cd8 c0fbe23d 00000000 00000001 00000000
 1fa0: 2e11d000 ee8e1fc0 c0108758 c010875c 60000013 ffffffff
 [<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
-[<c010875c>] (arch_cpu_idle) from [<c0160cc0>] (do_idle+0x148/0x204)
-[<c0160cc0>] (do_idle) from [<c0161004>] (cpu_startup_entry+0x18/0x1c)
-[<c0161004>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
-CPU1: stopping
-CPU: 1 PID: 0 Comm: swapper/1 Tainted: G      D         4.13.0-next-20170915-200043-g1f183459 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c01110d0>] (unwind_backtrace) from [<c010c090>] (show_stack+0x10/0x14)
-[<c010c090>] (show_stack) from [<c0961220>] (dump_stack+0x84/0xa4)
-[<c0961220>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
-[<c010f61c>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
-[<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
-Exception stack(0xee8dff70 to 0xee8dffb8)
-ff60:                                     00000001 00000000 ee8dffc8 c011a380
-ff80: c0f03ccc c0f03c68 c0e8d378 c0f03cd8 c0fbe23d 00000000 00000001 00000000
-ffa0: 2e10c000 ee8dffc0 c0108758 c010875c 60000013 ffffffff
-[<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
-[<c010875c>] (arch_cpu_idle) from [<c0160cc0>] (do_idle+0x148/0x204)
-[<c0160cc0>] (do_idle) from [<c0161004>] (cpu_startup_entry+0x18/0x1c)
-[<c0161004>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
+[<c010875c>] (arch_cpu_idle) from [<c0160cc4>] (do_idle+0x148/0x204)
+[<c0160cc4>] (do_idle) from [<c0161008>] (cpu_startup_entry+0x18/0x1c)
+[<c0161008>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
 ---[ end Kernel panic - not syncing: Attempted to kill init! exitcode=0x0000000b
 
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2017-09-17 19:57:37.059515116 -0700
+++ /tmp/foob	2017-09-17 19:57:37.059515116 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1801270 us
+Bootloader-Cpu Init at (time stamp): 1801228 us
 
 Pinmux changes applied in kernel way
 
@@ -62,22 +62,22 @@
 Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1801244  us
-Bootloader (AosInit) time: 733426 us
-Bootloader Main Init time: 180323 us
-BlockDevMgrInit time: 157553 us
+Miscellaneous avp time: 1801202  us
+Bootloader (AosInit) time: 733462 us
+Bootloader Main Init time: 180288 us
+BlockDevMgrInit time: 157545 us
 PartMgrInit time: 8 us
 StorMgrInit time: 31 us
 FsMgrInit time: 2 us
-Partition Table load: 77060 us 
-Display init time: 574615 us
-Kernel read time: 1339 msec
+Partition Table load: 77137 us 
+Display init time: 574431 us
+Kernel read time: 1340 msec
 Load Kernel Size:   17760256 Bytes
-Bl Kernel ReadSpeed: 13263000 Bytes/sec
-Pre-Kernel Jump time: 3828162 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 8786264 us
+Bl Kernel ReadSpeed: 13253000 Bytes/sec
+Pre-Kernel Jump time: 3827636 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 8787785 us
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-next-20170915-318284-g1f183459b514 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Thu Sep 14 20:40:05 PDT 2017
+Linux version 4.14.0-rc1-next-20170918-318337-gfc2e8b1a47c1 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sun Sep 17 19:55:25 PDT 2017
 Boot CPU: AArch64 Processor [4e0f0000]
 Machine model: NVIDIA Tegra132 Norrin
 Memory limited to 1734MB
@@ -202,7 +202,7 @@
 Freeing initrd memory: 1284K
 kvm [1]: HYP mode not available
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.971:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.975:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -273,14 +273,15 @@
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
-+VDD_LED: supplied by +VDD_MUX
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
++VDD_LED: supplied by +VDD_MUX
 +5V_USB_HS: supplied by +5V_SYS
 +5V_USB_SS: supplied by +5V_SYS
 +5V_HDMI_CON: supplied by +5V_SYS
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
+sdhci-tegra 700b0400.sdhci: Got CD GPIO
 hctosys: unable to open rtc device (rtc0)
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +VDD_LED: disabling
@@ -291,7 +292,6 @@
 ALSA device list:
   No soundcards found.
 Freeing unused kernel memory: 1088K
-sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 
 System initialization...
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-09-17 19:57:37.091515250 -0700
+++ /tmp/foob	2017-09-17 19:57:37.095515267 -0700
@@ -48,7 +48,7 @@
 	 #################################################################
 	 #################################################################
 	 #####################################
-	 4.1 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 8167944 (7ca208 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
@@ -57,7 +57,7 @@
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 21454 (53ce hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
@@ -69,7 +69,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-next-20170915-200043-g1f183459 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Sep 14 20:36:30 PDT 2017
+Linux version 4.14.0-rc1-next-20170918-200096-gfc2e8b1 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Sun Sep 17 19:49:29 PDT 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -79,7 +79,7 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0x3c000000
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c158eb80, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c158eb40, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -93,7 +93,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 954304K/1048576K available (10240K kernel code, 1628K rwdata, 4508K rodata, 2048K init, 405K bss, 28736K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 954308K/1048576K available (10240K kernel code, 1627K rwdata, 4508K rodata, 2048K init, 405K bss, 28732K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -103,8 +103,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1597028   (1629 kB)
-       .bss : 0xc15a0bc0 - 0xc160637c   ( 406 kB)
+      .data : 0xc1400000 - 0xc1596fe8   (1628 kB)
+       .bss : 0xc159fbc0 - 0xc160537c   ( 406 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -140,7 +140,7 @@
 SMP: Total of 2 processors activated (4.00 BogoMIPS).
 CPU: All CPU(s) started in SVC mode.
 devtmpfs: initialized
-Built 1 zonelists, mobility grouping on.  Total pages: 254960
+Built 1 zonelists, mobility grouping on.  Total pages: 254961
 VFP support v0.3: implementor 41 architecture 3 part 30 variant 9 rev 1
 clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 19112604462750000 ns
 futex hash table entries: 512 (order: 3, 32768 bytes)
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2017-09-17 19:57:37.131515422 -0700
+++ /tmp/foob	2017-09-17 19:57:37.131515422 -0700
@@ -48,18 +48,18 @@
 	 #######################################
 	 4.1 MiB/s
 done
-Bytes transferred = 6293000 (600608 hex)
+Bytes transferred = 6292552 (600448 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 4.1 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21454 (53ce hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x600608 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x600448 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103cd
@@ -68,7 +68,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-next-20170915-200043-g1f183459 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Sep 14 20:35:58 PDT 2017
+Linux version 4.14.0-rc1-next-20170918-200096-gfc2e8b1 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Sun Sep 17 19:49:39 PDT 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -348,8 +348,8 @@
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
-hub 1-1:1.0: USB hub found
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
+hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
 rtc-em3027 2-0056: hctosys: unable to read the hardware clock
 ALSA device list:
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2017-09-17 19:57:37.155515524 -0700
+++ /tmp/foob	2017-09-17 19:57:37.159515541 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 799639 us
-Basic modules init took 349167 us
-Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14387 us
-Carveout took 252663 us
-CPU initialization took 92450 us
-Total time taken by TegraBoot 708667 us
+Tegraboot started after 877992 us
+Basic modules init took 348794 us
+Sec Bootdevice Read Time = 204 ms, Read Size = 8459 KB
+Next stage binary read took 15498 us
+Carveout took 252675 us
+CPU initialization took 93023 us
+Total time taken by TegraBoot 709990 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.638] LPDDR4 Training: Number of tables = 10
+64b[0001.715] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,7 +117,7 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-16439808 bytes read in 833 ms (18.8 MiB/s)
+16439808 bytes read in 834 ms (18.8 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
 45170 bytes read in 454 ms (96.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-next-20170915-318284-g1f183459b514 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Thu Sep 14 20:40:05 PDT 2017
+Linux version 4.14.0-rc1-next-20170918-318337-gfc2e8b1a47c1 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sun Sep 17 19:55:25 PDT 2017
 Boot CPU: AArch64 Processor [411fd071]
 Machine model: NVIDIA Tegra210 P2371 (P2530/P2595) reference design
 debug: ignoring loglevel setting.
@@ -277,7 +277,7 @@
 kvm [1]: vgic interrupt IRQ1
 kvm [1]: virtual timer IRQ4
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.499:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.515:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2017-09-17 19:57:37.179515626 -0700
+++ /tmp/foob	2017-09-17 19:57:37.179515626 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-next-20170915-318284-g1f183459b514 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Thu Sep 14 20:40:05 PDT 2017
+Linux version 4.14.0-rc1-next-20170918-318337-gfc2e8b1a47c1 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sun Sep 17 19:55:25 PDT 2017
 Boot CPU: AArch64 Processor [411fd071]
 Machine model: Google Pixel C
 efi: Getting EFI parameters from FDT:
@@ -262,7 +262,7 @@
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
-max77686-rtc max77620-rtc: setting system clock to 2001-06-26 09:04:46 UTC (993546286)
+max77686-rtc max77620-rtc: setting system clock to 2001-06-29 08:20:01 UTC (993802801)
 PP1200_AVDD: disabling
 PP1200_RCAM: disabling
 PP2800L_RCAM: disabling
@@ -273,8 +273,8 @@
 PP1800_CAM: disabling
 ALSA device list:
   No soundcards found.
-Freeing unused kernel memory: 1088K
 tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
+Freeing unused kernel memory: 1088K
 
 System initialization...
 
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-09-17 19:57:37.187515661 -0700
+++ /tmp/foob	2017-09-17 19:57:37.187515661 -0700
@@ -40,7 +40,7 @@
 	 #################################################################
 	 #################################################################
 	 #####################################
-	 3.2 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 8167944 (7ca208 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
@@ -49,7 +49,7 @@
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.9 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 51860 (ca94 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-next-20170915-200043-g1f183459 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Sep 14 20:36:30 PDT 2017
+Linux version 4.14.0-rc1-next-20170918-200096-gfc2e8b1 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Sun Sep 17 19:49:29 PDT 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -71,7 +71,7 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c158eb80, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c158eb40, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -85,7 +85,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993396K/2096128K available (10240K kernel code, 1628K rwdata, 4508K rodata, 2048K init, 405K bss, 37196K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993400K/2096128K available (10240K kernel code, 1627K rwdata, 4508K rodata, 2048K init, 405K bss, 37192K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -95,8 +95,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1597028   (1629 kB)
-       .bss : 0xc15a0bc0 - 0xc160637c   ( 406 kB)
+      .data : 0xc1400000 - 0xc1596fe8   (1628 kB)
+       .bss : 0xc159fbc0 - 0xc160537c   ( 406 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -134,7 +134,7 @@
 SMP: Total of 4 processors activated (8.00 BogoMIPS).
 CPU: All CPU(s) started in SVC mode.
 devtmpfs: initialized
-Built 1 zonelists, mobility grouping on.  Total pages: 514733
+Built 1 zonelists, mobility grouping on.  Total pages: 514734
 VFP support v0.3: implementor 41 architecture 3 part 30 variant 9 rev 4
 clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 19112604462750000 ns
 futex hash table entries: 1024 (order: 4, 65536 bytes)
@@ -474,4 +474,4 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.554439] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.554428] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2017-09-17 19:57:37.195515695 -0700
+++ /tmp/foob	2017-09-17 19:57:37.195515695 -0700
@@ -40,18 +40,18 @@
 	 #######################################
 	 3.2 MiB/s
 done
-Bytes transferred = 6293000 (600608 hex)
+Bytes transferred = 6292552 (600448 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 51860 (ca94 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x600608 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x600448 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa93 ... OK
@@ -60,7 +60,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-next-20170915-200043-g1f183459 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Sep 14 20:35:58 PDT 2017
+Linux version 4.14.0-rc1-next-20170918-200096-gfc2e8b1 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Sun Sep 17 19:49:39 PDT 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -471,4 +471,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.804434] usb 2-1: new h
\ No newline at end of file
+root@(none):/# [    4.804432] usb 2-1: new high-speed USB device number 2 using tegra-ehci
