Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-05-10 22:56:31.339099815 -0700
+++ /tmp/foob	2016-05-10 22:56:31.343099815 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc7-next-20160510+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon May 9 23:51:48 PDT 2016
+Linux version 4.6.0-rc7-next-20160511+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue May 10 22:51:04 PDT 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -19,15 +19,15 @@
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
 software IO TLB [mem 0x79c00000-0x7dc00000] (64MB) mapped at [ffff800039c00000-ffff80003dbfffff]
-Memory: 935588K/1048576K available (7400K kernel code, 640K rwdata, 3156K rodata, 960K init, 257K bss, 96604K reserved, 16384K cma-reserved)
+Memory: 935520K/1048576K available (7440K kernel code, 642K rwdata, 3180K rodata, 960K init, 260K bss, 96672K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff0000087b0000   (  7360 KB)
-    .rodata : 0xffff0000087b0000 - 0xffff000008ad0000   (  3200 KB)
-      .init : 0xffff000008ad0000 - 0xffff000008bc0000   (   960 KB)
-      .data : 0xffff000008bc0000 - 0xffff000008c60000   (   640 KB)
-       .bss : 0xffff000008c60000 - 0xffff000008ca07c0   (   258 KB)
+      .text : 0xffff000008080000 - 0xffff0000087c0000   (  7424 KB)
+    .rodata : 0xffff0000087c0000 - 0xffff000008ae0000   (  3200 KB)
+      .init : 0xffff000008ae0000 - 0xffff000008bd0000   (   960 KB)
+      .data : 0xffff000008bd0000 - 0xffff000008c70800   (   642 KB)
+       .bss : 0xffff000008c70800 - 0xffff000008cb1ac0   (   261 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -62,7 +62,7 @@
 pinctrl core: initialized pinctrl subsystem
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffff0000087b6000, 1 data @ ffff000008bc4000)
+vdso: 2 pages (1 code @ ffff0000087c6000, 1 data @ ffff000008bd4000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -98,7 +98,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.544:1): initialized
+audit: type=2000 audit(0.560:1): initialized
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -158,7 +158,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2016-05-10 06:52:19 UTC (1462863139)
+rtc-pl031 9010000.pl031: setting system clock to 2016-05-11 05:51:36 UTC (1462945896)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
@@ -167,7 +167,7 @@
 EXT4-fs (vda2): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 254:2.
 devtmpfs: mounted
-Freeing unused kernel memory: 960K (ffff800000ad0000 - ffff800000bc0000)
+Freeing unused kernel memory: 960K (ffff800000ae0000 - ffff800000bd0000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 bash-4.3# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-05-10 22:56:31.355099819 -0700
+++ /tmp/foob	2016-05-10 22:56:31.359099820 -0700
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 ###########################
-	 3.1 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 7070312 (6be268 hex)
+Bytes transferred = 7070584 (6be378 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 34956 (888c hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6be268 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6be378 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff88b ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc7-next-20160510-93572-g416636b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon May 9 23:50:03 PDT 2016
+Linux version 4.6.0-rc7-next-20160511-94083-g13425f1 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue May 10 22:49:08 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -90,7 +90,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc106a424   (14730 kB)
+      .text : 0xc0208000 - 0xc106a43c   (14730 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
       .data : 0xc1300000 - 0xc141c6e0   (1138 kB)
        .bss : 0xc141e000 - 0xc1473ca0   ( 344 kB)
@@ -260,27 +260,27 @@
 ehci-atmel: EHCI Atmel driver
 tegra-ehci: Tegra EHCI driver
 ------------[ cut here ]------------
-WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
+WARNING: CPU: 1 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc7-next-20160510-93572-g416636b #1
+CPU: 1 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc7-next-20160511-94083-g13425f1 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c030f990>] (unwind_backtrace) from [<c030b088>] (show_stack+0x10/0x14)
-[<c030b088>] (show_stack) from [<c0586208>] (dump_stack+0x84/0xa4)
-[<c0586208>] (dump_stack) from [<c0341b6c>] (__warn+0xd4/0x100)
+[<c030b088>] (show_stack) from [<c0586188>] (dump_stack+0x84/0xa4)
+[<c0586188>] (dump_stack) from [<c0341b6c>] (__warn+0xd4/0x100)
 [<c0341b6c>] (__warn) from [<c0341bb8>] (warn_slowpath_null+0x20/0x28)
-[<c0341bb8>] (warn_slowpath_null) from [<c069eb08>] (__of_reset_control_get+0x1bc/0x228)
-[<c069eb08>] (__of_reset_control_get) from [<c0910220>] (tegra_ehci_probe+0x2d0/0x51c)
-[<c0910220>] (tegra_ehci_probe) from [<c0777528>] (platform_drv_probe+0x50/0xb0)
-[<c0777528>] (platform_drv_probe) from [<c0775e8c>] (driver_probe_device+0x1f4/0x2b0)
-[<c0775e8c>] (driver_probe_device) from [<c0775ff4>] (__driver_attach+0xac/0xb0)
-[<c0775ff4>] (__driver_attach) from [<c07744a0>] (bus_for_each_dev+0x54/0x88)
-[<c07744a0>] (bus_for_each_dev) from [<c0775480>] (bus_add_driver+0xe8/0x1f4)
-[<c0775480>] (bus_add_driver) from [<c0776810>] (driver_register+0x78/0xf4)
-[<c0776810>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
+[<c0341bb8>] (warn_slowpath_null) from [<c069eb28>] (__of_reset_control_get+0x1bc/0x228)
+[<c069eb28>] (__of_reset_control_get) from [<c0910308>] (tegra_ehci_probe+0x2d0/0x51c)
+[<c0910308>] (tegra_ehci_probe) from [<c07775a8>] (platform_drv_probe+0x50/0xb0)
+[<c07775a8>] (platform_drv_probe) from [<c0775f0c>] (driver_probe_device+0x1f4/0x2b0)
+[<c0775f0c>] (driver_probe_device) from [<c0776074>] (__driver_attach+0xac/0xb0)
+[<c0776074>] (__driver_attach) from [<c0774520>] (bus_for_each_dev+0x54/0x88)
+[<c0774520>] (bus_for_each_dev) from [<c0775500>] (bus_add_driver+0xe8/0x1f4)
+[<c0775500>] (bus_add_driver) from [<c0776890>] (driver_register+0x78/0xf4)
+[<c0776890>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
 [<c0301e5c>] (do_one_initcall) from [<c1100d80>] (kernel_init_freeable+0x120/0x1ec)
-[<c1100d80>] (kernel_init_freeable) from [<c0b59d08>] (kernel_init+0x8/0x118)
-[<c0b59d08>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
----[ end trace 007ccd25635b2b7a ]---
+[<c1100d80>] (kernel_init_freeable) from [<c0b59f10>] (kernel_init+0x8/0x118)
+[<c0b59f10>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
+---[ end trace e64d37f22f6e0064 ]---
 tegra-ehci 7d008000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d008000.usb: continuing, but please update your DT
 ohci_hcd: USB 1.1 'Open' Host Controller (OHCI) Driver
@@ -326,15 +326,15 @@
 usb2_vbus: supplied by vdd-sys-5v0
 VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
 mmc0: new high speed SDHC card at address aaaa
+mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
-mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
- mmcblk0: p1 p2 p3
-mmc1: new high speed MMC card at address 0001
+mmc1: new MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
+ mmcblk0: p1 p2 p3
  mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
 Console: switching to colour frame buffer device 240x75
 drm drm: fb0:  frame buffer device
@@ -346,16 +346,18 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-03-03 18:09:20 UTC (952106960)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-03-04 17:08:27 UTC (952189707)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
 EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 EXT4-fs (mmcblk1p1): recovery complete
 EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:17.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
+asix 1-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0d:b5:16
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.535181] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-05-10 22:56:31.363099820 -0700
+++ /tmp/foob	2016-05-10 22:56:31.367099821 -0700
@@ -39,7 +39,7 @@
 	 ##########################################
 	 2.8 MiB/s
 done
-Bytes transferred = 5376256 (520900 hex)
+Bytes transferred = 5376968 (520bc8 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 34956 (888c hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x520900 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x520bc8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff88b ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc7-next-20160510-93572-g416636b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon May 9 23:50:01 PDT 2016
+Linux version 4.6.0-rc7-next-20160511-94083-g13425f1 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue May 10 22:49:05 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2000976K/2096128K available (7415K kernel code, 374K rwdata, 2556K rodata, 1024K init, 366K bss, 29616K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2000976K/2096128K available (7415K kernel code, 374K rwdata, 2560K rodata, 1024K init, 366K bss, 29616K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,7 +90,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0abcf8c   (10964 kB)
+      .text : 0xc0008000 - 0xc0abdfa4   (10968 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
       .data : 0xc0c00000 - 0xc0c5d8c0   ( 375 kB)
        .bss : 0xc0c5f000 - 0xc0cbab18   ( 367 kB)
@@ -252,25 +252,25 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1b4/0x218
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc7-next-20160510-93572-g416636b #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc7-next-20160511-94083-g13425f1 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c010f1e8>] (unwind_backtrace) from [<c010a864>] (show_stack+0x10/0x14)
-[<c010a864>] (show_stack) from [<c033da54>] (dump_stack+0x84/0xa4)
-[<c033da54>] (dump_stack) from [<c011f308>] (__warn+0xd4/0x100)
+[<c010a864>] (show_stack) from [<c033da34>] (dump_stack+0x84/0xa4)
+[<c033da34>] (dump_stack) from [<c011f308>] (__warn+0xd4/0x100)
 [<c011f308>] (__warn) from [<c011f354>] (warn_slowpath_null+0x20/0x28)
-[<c011f354>] (warn_slowpath_null) from [<c03c0b6c>] (__of_reset_control_get+0x1b4/0x218)
-[<c03c0b6c>] (__of_reset_control_get) from [<c05134c4>] (tegra_ehci_probe+0x2c8/0x514)
-[<c05134c4>] (tegra_ehci_probe) from [<c0440e54>] (platform_drv_probe+0x50/0xb0)
-[<c0440e54>] (platform_drv_probe) from [<c043f880>] (driver_probe_device+0x1f4/0x2b0)
-[<c043f880>] (driver_probe_device) from [<c043f9e8>] (__driver_attach+0xac/0xb0)
-[<c043f9e8>] (__driver_attach) from [<c043dea4>] (bus_for_each_dev+0x54/0x88)
-[<c043dea4>] (bus_for_each_dev) from [<c043ee74>] (bus_add_driver+0xe8/0x1f4)
-[<c043ee74>] (bus_add_driver) from [<c04401e4>] (driver_register+0x78/0xf4)
-[<c04401e4>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
+[<c011f354>] (warn_slowpath_null) from [<c03c0be8>] (__of_reset_control_get+0x1b4/0x218)
+[<c03c0be8>] (__of_reset_control_get) from [<c05135f4>] (tegra_ehci_probe+0x2c8/0x514)
+[<c05135f4>] (tegra_ehci_probe) from [<c0440ed0>] (platform_drv_probe+0x50/0xb0)
+[<c0440ed0>] (platform_drv_probe) from [<c043f8fc>] (driver_probe_device+0x1f4/0x2b0)
+[<c043f8fc>] (driver_probe_device) from [<c043fa64>] (__driver_attach+0xac/0xb0)
+[<c043fa64>] (__driver_attach) from [<c043df20>] (bus_for_each_dev+0x54/0x88)
+[<c043df20>] (bus_for_each_dev) from [<c043eef0>] (bus_add_driver+0xe8/0x1f4)
+[<c043eef0>] (bus_add_driver) from [<c0440260>] (driver_register+0x78/0xf4)
+[<c0440260>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
 [<c010179c>] (do_one_initcall) from [<c0b00d94>] (kernel_init_freeable+0x11c/0x1e8)
-[<c0b00d94>] (kernel_init_freeable) from [<c07f07b4>] (kernel_init+0x8/0x118)
-[<c07f07b4>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
----[ end trace 021f9fc2e301a748 ]---
+[<c0b00d94>] (kernel_init_freeable) from [<c07f09e8>] (kernel_init+0x8/0x118)
+[<c07f09e8>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
+---[ end trace 6aa43aef61bcba7e ]---
 tegra-ehci 7d008000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d008000.usb: continuing, but please update your DT
 usbcore: registered new interface driver cdc_acm
@@ -301,13 +301,13 @@
 usbhid: USB HID core driver
 mmc0: new high speed SDHC card at address aaaa
 mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
-mmc1: new high speed MMC card at address 0001
+mmc1: new MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
- mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
  mmcblk0: p1 p2 p3
+ mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
 tegra30-i2s 70080400.i2s: DMA channels sourced from device 70080000.ahub
 tegra-snd-rt5640 sound: rt5640-aif1 <-> 70080400.i2s mapping ok
 input: NVIDIA Tegra Dalmore Headphones as /devices/soc0/sound/sound/card0/input0
@@ -344,7 +344,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-03-03 18:10:48 UTC (952107048)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-03-04 17:09:57 UTC (952189797)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -354,9 +354,10 @@
 EXT4-fs (mmcblk1p1): recovery complete
 EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:17.
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
+asix 1-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0d:b5:16
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-05-10 22:56:31.375099824 -0700
+++ /tmp/foob	2016-05-10 22:56:31.375099824 -0700
@@ -34,9 +34,9 @@
 	 #################################################################
 	 #################################################################
 	 ###########################
-	 4 MiB/s
+	 3.9 MiB/s
 done
-Bytes transferred = 7070312 (6be268 hex)
+Bytes transferred = 7070584 (6be378 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -47,7 +47,7 @@
 done
 Bytes transferred = 66737 (104b1 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6be268 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x6be378 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830134b0
@@ -55,7 +55,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc7-next-20160510-93572-g416636b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon May 9 23:50:03 PDT 2016
+Linux version 4.6.0-rc7-next-20160511-94083-g13425f1 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue May 10 22:49:08 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -85,7 +85,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc106a424   (14730 kB)
+      .text : 0xc0208000 - 0xc106a43c   (14730 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
       .data : 0xc1300000 - 0xc141c6e0   (1138 kB)
        .bss : 0xc141e000 - 0xc1473ca0   ( 344 kB)
@@ -254,25 +254,25 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc7-next-20160510-93572-g416636b #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc7-next-20160511-94083-g13425f1 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c030f990>] (unwind_backtrace) from [<c030b088>] (show_stack+0x10/0x14)
-[<c030b088>] (show_stack) from [<c0586208>] (dump_stack+0x84/0xa4)
-[<c0586208>] (dump_stack) from [<c0341b6c>] (__warn+0xd4/0x100)
+[<c030b088>] (show_stack) from [<c0586188>] (dump_stack+0x84/0xa4)
+[<c0586188>] (dump_stack) from [<c0341b6c>] (__warn+0xd4/0x100)
 [<c0341b6c>] (__warn) from [<c0341bb8>] (warn_slowpath_null+0x20/0x28)
-[<c0341bb8>] (warn_slowpath_null) from [<c069eb08>] (__of_reset_control_get+0x1bc/0x228)
-[<c069eb08>] (__of_reset_control_get) from [<c0910220>] (tegra_ehci_probe+0x2d0/0x51c)
-[<c0910220>] (tegra_ehci_probe) from [<c0777528>] (platform_drv_probe+0x50/0xb0)
-[<c0777528>] (platform_drv_probe) from [<c0775e8c>] (driver_probe_device+0x1f4/0x2b0)
-[<c0775e8c>] (driver_probe_device) from [<c0775ff4>] (__driver_attach+0xac/0xb0)
-[<c0775ff4>] (__driver_attach) from [<c07744a0>] (bus_for_each_dev+0x54/0x88)
-[<c07744a0>] (bus_for_each_dev) from [<c0775480>] (bus_add_driver+0xe8/0x1f4)
-[<c0775480>] (bus_add_driver) from [<c0776810>] (driver_register+0x78/0xf4)
-[<c0776810>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
+[<c0341bb8>] (warn_slowpath_null) from [<c069eb28>] (__of_reset_control_get+0x1bc/0x228)
+[<c069eb28>] (__of_reset_control_get) from [<c0910308>] (tegra_ehci_probe+0x2d0/0x51c)
+[<c0910308>] (tegra_ehci_probe) from [<c07775a8>] (platform_drv_probe+0x50/0xb0)
+[<c07775a8>] (platform_drv_probe) from [<c0775f0c>] (driver_probe_device+0x1f4/0x2b0)
+[<c0775f0c>] (driver_probe_device) from [<c0776074>] (__driver_attach+0xac/0xb0)
+[<c0776074>] (__driver_attach) from [<c0774520>] (bus_for_each_dev+0x54/0x88)
+[<c0774520>] (bus_for_each_dev) from [<c0775500>] (bus_add_driver+0xe8/0x1f4)
+[<c0775500>] (bus_add_driver) from [<c0776890>] (driver_register+0x78/0xf4)
+[<c0776890>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
 [<c0301e5c>] (do_one_initcall) from [<c1100d80>] (kernel_init_freeable+0x120/0x1ec)
-[<c1100d80>] (kernel_init_freeable) from [<c0b59d08>] (kernel_init+0x8/0x118)
-[<c0b59d08>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
----[ end trace 96f5ff44f5944b8a ]---
+[<c1100d80>] (kernel_init_freeable) from [<c0b59f10>] (kernel_init+0x8/0x118)
+[<c0b59f10>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
+---[ end trace 952d6cd706e24bfd ]---
 tegra-ehci 7d004000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d004000.usb: continuing, but please update your DT
 tegra-ehci 7d004000.usb: EHCI Host Controller
@@ -330,13 +330,13 @@
 +5V_SATA: supplied by +5V_SYS
 +12V_SATA: supplied by +VDD_MUX
 usb2-0 supply vbus not found, using dummy regulator
-usb2-1 supply vbus not found, using dummy regulator
-mmc0: new high speed MMC card at address 0001
+mmc0: new MMC card at address 0001
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
  mmcblk0: p1
+usb2-1 supply vbus not found, using dummy regulator
 usb3-0 supply vbus not found, using dummy regulator
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
 tegra-pcie 1003000.pcie-controller: Failed to get supply 'avddio-pex': -517
@@ -409,10 +409,7 @@
 pci 0000:02:00.0: PME# supported from D0 D1 D2 D3hot D3cold
 PCI: bus2: Fast back to back transfers disabled
 pci_bus 0000:02: busn_res: [bus 02-ff] end is updated to 02
-mmc1: new high speed SDHC card at address aaaa
-mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 pci 0000:00:01.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
- mmcblk1: p1 p2 p3
 pci 0000:00:02.0: BAR 8: assigned [mem 0x13100000-0x131fffff]
 pci 0000:00:02.0: BAR 9: assigned [mem 0x20000000-0x200fffff 64bit pref]
 pci 0000:00:02.0: BAR 7: assigned [io  0x1000-0x1fff]
@@ -425,6 +422,8 @@
 pci 0000:00:02.0: PCI bridge to [bus 02]
 pci 0000:00:02.0:   bridge window [io  0x1000-0x1fff]
 pci 0000:00:02.0:   bridge window [mem 0x13100000-0x131fffff]
+mmc1: new high speed SDHC card at address aaaa
+mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 pci 0000:00:02.0:   bridge window [mem 0x20000000-0x200fffff 64bit pref]
 pci 0000:00:01.0: nv_msi_ht_cap_quirk didn't locate host bridge
 pcieport 0000:00:01.0: enabling device (0140 -> 0142)
@@ -438,6 +437,7 @@
 pcie_pme 0000:00:02.0:pcie001: service driver pcie_pme loaded
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
+ mmcblk1: p1 p2 p3
 r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0a5c000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 393
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-05-10 22:56:31.383099825 -0700
+++ /tmp/foob	2016-05-10 22:56:31.383099825 -0700
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+MMC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -32,20 +32,20 @@
 	 #################################################################
 	 #################################################################
 	 ##########################################
-	 4 MiB/s
+	 902.3 KiB/s
 done
-Bytes transferred = 5376256 (520900 hex)
+Bytes transferred = 5376968 (520bc8 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.7 MiB/s
+	 879.9 KiB/s
 done
 Bytes transferred = 66737 (104b1 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x520900 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x520bc8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830134b0
@@ -54,7 +54,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc7-next-20160510-93572-g416636b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon May 9 23:50:01 PDT 2016
+Linux version 4.6.0-rc7-next-20160511-94083-g13425f1 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue May 10 22:49:05 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -77,7 +77,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2000856K/2096128K available (7415K kernel code, 374K rwdata, 2556K rodata, 1024K init, 366K bss, 29736K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2000856K/2096128K available (7415K kernel code, 374K rwdata, 2560K rodata, 1024K init, 366K bss, 29736K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -85,7 +85,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0abcf8c   (10964 kB)
+      .text : 0xc0008000 - 0xc0abdfa4   (10968 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
       .data : 0xc0c00000 - 0xc0c5d8c0   ( 375 kB)
        .bss : 0xc0c5f000 - 0xc0cbab18   ( 367 kB)
@@ -242,27 +242,27 @@
 ehci-pci: EHCI PCI platform driver
 tegra-ehci: Tegra EHCI driver
 ------------[ cut here ]------------
-WARNING: CPU: 2 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1b4/0x218
+WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1b4/0x218
 Modules linked in:
-CPU: 2 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc7-next-20160510-93572-g416636b #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc7-next-20160511-94083-g13425f1 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c010f1e8>] (unwind_backtrace) from [<c010a864>] (show_stack+0x10/0x14)
-[<c010a864>] (show_stack) from [<c033da54>] (dump_stack+0x84/0xa4)
-[<c033da54>] (dump_stack) from [<c011f308>] (__warn+0xd4/0x100)
+[<c010a864>] (show_stack) from [<c033da34>] (dump_stack+0x84/0xa4)
+[<c033da34>] (dump_stack) from [<c011f308>] (__warn+0xd4/0x100)
 [<c011f308>] (__warn) from [<c011f354>] (warn_slowpath_null+0x20/0x28)
-[<c011f354>] (warn_slowpath_null) from [<c03c0b6c>] (__of_reset_control_get+0x1b4/0x218)
-[<c03c0b6c>] (__of_reset_control_get) from [<c05134c4>] (tegra_ehci_probe+0x2c8/0x514)
-[<c05134c4>] (tegra_ehci_probe) from [<c0440e54>] (platform_drv_probe+0x50/0xb0)
-[<c0440e54>] (platform_drv_probe) from [<c043f880>] (driver_probe_device+0x1f4/0x2b0)
-[<c043f880>] (driver_probe_device) from [<c043f9e8>] (__driver_attach+0xac/0xb0)
-[<c043f9e8>] (__driver_attach) from [<c043dea4>] (bus_for_each_dev+0x54/0x88)
-[<c043dea4>] (bus_for_each_dev) from [<c043ee74>] (bus_add_driver+0xe8/0x1f4)
-[<c043ee74>] (bus_add_driver) from [<c04401e4>] (driver_register+0x78/0xf4)
-[<c04401e4>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
+[<c011f354>] (warn_slowpath_null) from [<c03c0be8>] (__of_reset_control_get+0x1b4/0x218)
+[<c03c0be8>] (__of_reset_control_get) from [<c05135f4>] (tegra_ehci_probe+0x2c8/0x514)
+[<c05135f4>] (tegra_ehci_probe) from [<c0440ed0>] (platform_drv_probe+0x50/0xb0)
+[<c0440ed0>] (platform_drv_probe) from [<c043f8fc>] (driver_probe_device+0x1f4/0x2b0)
+[<c043f8fc>] (driver_probe_device) from [<c043fa64>] (__driver_attach+0xac/0xb0)
+[<c043fa64>] (__driver_attach) from [<c043df20>] (bus_for_each_dev+0x54/0x88)
+[<c043df20>] (bus_for_each_dev) from [<c043eef0>] (bus_add_driver+0xe8/0x1f4)
+[<c043eef0>] (bus_add_driver) from [<c0440260>] (driver_register+0x78/0xf4)
+[<c0440260>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
 [<c010179c>] (do_one_initcall) from [<c0b00d94>] (kernel_init_freeable+0x11c/0x1e8)
-[<c0b00d94>] (kernel_init_freeable) from [<c07f07b4>] (kernel_init+0x8/0x118)
-[<c07f07b4>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
----[ end trace cb25669927240518 ]---
+[<c0b00d94>] (kernel_init_freeable) from [<c07f09e8>] (kernel_init+0x8/0x118)
+[<c07f09e8>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
+---[ end trace 2c383eca725222d2 ]---
 tegra-ehci 7d004000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d004000.usb: continuing, but please update your DT
 tegra-ehci 7d004000.usb: EHCI Host Controller
@@ -298,7 +298,7 @@
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
 tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
-mmc0: new high speed MMC card at address 0001
+mmc0: new MMC card at address 0001
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
@@ -400,9 +400,9 @@
 pci 0000:00:02.0: [10de:0e13] type 01 class 0x060400
 pci 0000:00:02.0: PME# supported from D0 D1 D2 D3hot D3cold
 PCI: bus0: Fast back to back transfers disabled
-ata1: SATA link down (SStatus 0 SControl 300)
 pci 0000:00:01.0: bridge configuration invalid ([bus 00-00]), reconfiguring
 pci 0000:00:02.0: bridge configuration invalid ([bus 00-00]), reconfiguring
+ata1: SATA link down (SStatus 0 SControl 300)
 pci 0000:01:00.0: [8086:4232] type 00 class 0x028000
 pci 0000:01:00.0: reg 0x10: [mem 0x00000000-0x00001fff 64bit]
 pci 0000:01:00.0: PME# supported from D0 D3hot D3cold
@@ -419,10 +419,11 @@
 pci_bus 0000:02: busn_res: [bus 02-ff] end is updated to 02
 pci 0000:00:01.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
 pci 0000:00:02.0: BAR 8: assigned [mem 0x13100000-0x131fffff]
-mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 pci 0000:00:02.0: BAR 9: assigned [mem 0x20000000-0x200fffff 64bit pref]
+mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 pci 0000:00:02.0: BAR 7: assigned [io  0x1000-0x1fff]
 pci 0000:01:00.0: BAR 0: assigned [mem 0x13000000-0x13001fff 64bit]
+ mmcblk1: p1 p2 p3
 pci 0000:00:01.0: PCI bridge to [bus 01]
 pci 0000:00:01.0:   bridge window [mem 0x13000000-0x130fffff]
 pci 0000:02:00.0: BAR 4: assigned [mem 0x20000000-0x20003fff 64bit pref]
@@ -433,7 +434,6 @@
 pci 0000:00:02.0:   bridge window [mem 0x13100000-0x131fffff]
 pci 0000:00:02.0:   bridge window [mem 0x20000000-0x200fffff 64bit pref]
 pci 0000:00:01.0: nv_msi_ht_cap_quirk didn't locate host bridge
- mmcblk1: p1 p2 p3
 pcieport 0000:00:01.0: enabling device (0140 -> 0142)
 pcieport 0000:00:01.0: Signaling PME through PCIe PME interrupt
 pci 0000:01:00.0: Signaling PME through PCIe PME interrupt
@@ -460,7 +460,7 @@
 cpu cpu2: cpufreq_init: failed to get clk: -2
 cpu cpu3: cpufreq_init: failed to get clk: -2
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:08 UTC (946684928)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:06 UTC (946684926)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2016-05-10 22:56:31.391099827 -0700
+++ /tmp/foob	2016-05-10 22:56:31.391099827 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1801067 us
+Bootloader-Cpu Init at (time stamp): 1800544 us
 
 Pinmux changes applied in kernel way
 
@@ -62,22 +62,22 @@
 Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1801041  us
-Bootloader (AosInit) time: 734244 us
-Bootloader Main Init time: 180067 us
-BlockDevMgrInit time: 157529 us
+Miscellaneous avp time: 1800518  us
+Bootloader (AosInit) time: 733449 us
+Bootloader Main Init time: 180240 us
+BlockDevMgrInit time: 157535 us
 PartMgrInit time: 8 us
 StorMgrInit time: 30 us
-FsMgrInit time: 3 us
-Partition Table load: 76690 us 
-Display init time: 574690 us
-Kernel read time: 342 msec
-Load Kernel Size:   14878720 Bytes
-Bl Kernel ReadSpeed: 43505000 Bytes/sec
-Pre-Kernel Jump time: 1627560 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5687718 us
+FsMgrInit time: 2 us
+Partition Table load: 77101 us 
+Display init time: 574350 us
+Kernel read time: 343 msec
+Load Kernel Size:   14946304 Bytes
+Bl Kernel ReadSpeed: 43575000 Bytes/sec
+Pre-Kernel Jump time: 1632851 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5690795 us
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc7-next-20160510+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon May 9 23:51:48 PDT 2016
+Linux version 4.6.0-rc7-next-20160511+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue May 10 22:51:04 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
 Memory limited to 1734MB
 efi: Getting EFI parameters from FDT:
@@ -92,15 +92,15 @@
 Dentry cache hash table entries: 262144 (order: 9, 2097152 bytes)
 Inode-cache hash table entries: 131072 (order: 8, 1048576 bytes)
 software IO TLB [mem 0xe5200000-0xe9200000] (64MB) mapped at [ffff800065200000-ffff8000691fffff]
-Memory: 1642180K/1775616K available (7400K kernel code, 640K rwdata, 3156K rodata, 960K init, 257K bss, 117052K reserved, 16384K cma-reserved)
+Memory: 1642112K/1775616K available (7440K kernel code, 642K rwdata, 3180K rodata, 960K init, 260K bss, 117120K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff0000087b0000   (  7360 KB)
-    .rodata : 0xffff0000087b0000 - 0xffff000008ad0000   (  3200 KB)
-      .init : 0xffff000008ad0000 - 0xffff000008bc0000   (   960 KB)
-      .data : 0xffff000008bc0000 - 0xffff000008c60000   (   640 KB)
-       .bss : 0xffff000008c60000 - 0xffff000008ca07c0   (   258 KB)
+      .text : 0xffff000008080000 - 0xffff0000087c0000   (  7424 KB)
+    .rodata : 0xffff0000087c0000 - 0xffff000008ae0000   (  3200 KB)
+      .init : 0xffff000008ae0000 - 0xffff000008bd0000   (   960 KB)
+      .data : 0xffff000008bd0000 - 0xffff000008c70800   (   642 KB)
+       .bss : 0xffff000008c70800 - 0xffff000008cb1ac0   (   261 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -135,7 +135,7 @@
 pinctrl core: initialized pinctrl subsystem
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffff0000087b6000, 1 data @ ffff000008bc4000)
+vdso: 2 pages (1 code @ ffff0000087c6000, 1 data @ ffff000008bd4000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -185,7 +185,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(1.047:1): initialized
+audit: type=2000 audit(1.059:1): initialized
 workingset: timestamp_bits=44 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -243,7 +243,8 @@
 9pnet: Installing 9P2000 support
 Key type dns_resolver registered
 registered taskstats version 1
-mmc0: new high speed MMC card at address 0001
+mmc0: new MMC card at address 0001
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 +VDD_LED: supplied by +VDD_MUX
 +5V_USB_HS: supplied by +5V_SYS
 +5V_USB_SS: supplied by +5V_SYS
@@ -264,8 +265,7 @@
 +5V_HDMI_CON: disabling
 ALSA device list:
   No soundcards found.
-Freeing unused kernel memory: 960K (ffff800000ad0000 - ffff800000bc0000)
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+Freeing unused kernel memory: 960K (ffff800000ae0000 - ffff800000bd0000)
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-05-10 22:56:31.399099829 -0700
+++ /tmp/foob	2016-05-10 22:56:31.399099829 -0700
@@ -47,9 +47,9 @@
 	 #################################################################
 	 #################################################################
 	 ###########################
-	 895.5 KiB/s
+	 909.2 KiB/s
 done
-Bytes transferred = 7070312 (6be268 hex)
+Bytes transferred = 7070584 (6be378 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -60,7 +60,7 @@
 done
 Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x6be268 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x6be378 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102b3
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc7-next-20160510-93572-g416636b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon May 9 23:50:03 PDT 2016
+Linux version 4.6.0-rc7-next-20160511-94083-g13425f1 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue May 10 22:49:08 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -97,7 +97,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc106a424   (14730 kB)
+      .text : 0xc0208000 - 0xc106a43c   (14730 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
       .data : 0xc1300000 - 0xc141c6e0   (1138 kB)
        .bss : 0xc141e000 - 0xc1473ca0   ( 344 kB)
@@ -271,27 +271,27 @@
 ehci-atmel: EHCI Atmel driver
 tegra-ehci: Tegra EHCI driver
 ------------[ cut here ]------------
-WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
+WARNING: CPU: 1 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc7-next-20160510-93572-g416636b #1
+CPU: 1 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc7-next-20160511-94083-g13425f1 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c030f990>] (unwind_backtrace) from [<c030b088>] (show_stack+0x10/0x14)
-[<c030b088>] (show_stack) from [<c0586208>] (dump_stack+0x84/0xa4)
-[<c0586208>] (dump_stack) from [<c0341b6c>] (__warn+0xd4/0x100)
+[<c030b088>] (show_stack) from [<c0586188>] (dump_stack+0x84/0xa4)
+[<c0586188>] (dump_stack) from [<c0341b6c>] (__warn+0xd4/0x100)
 [<c0341b6c>] (__warn) from [<c0341bb8>] (warn_slowpath_null+0x20/0x28)
-[<c0341bb8>] (warn_slowpath_null) from [<c069eb08>] (__of_reset_control_get+0x1bc/0x228)
-[<c069eb08>] (__of_reset_control_get) from [<c0910220>] (tegra_ehci_probe+0x2d0/0x51c)
-[<c0910220>] (tegra_ehci_probe) from [<c0777528>] (platform_drv_probe+0x50/0xb0)
-[<c0777528>] (platform_drv_probe) from [<c0775e8c>] (driver_probe_device+0x1f4/0x2b0)
-[<c0775e8c>] (driver_probe_device) from [<c0775ff4>] (__driver_attach+0xac/0xb0)
-[<c0775ff4>] (__driver_attach) from [<c07744a0>] (bus_for_each_dev+0x54/0x88)
-[<c07744a0>] (bus_for_each_dev) from [<c0775480>] (bus_add_driver+0xe8/0x1f4)
-[<c0775480>] (bus_add_driver) from [<c0776810>] (driver_register+0x78/0xf4)
-[<c0776810>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
+[<c0341bb8>] (warn_slowpath_null) from [<c069eb28>] (__of_reset_control_get+0x1bc/0x228)
+[<c069eb28>] (__of_reset_control_get) from [<c0910308>] (tegra_ehci_probe+0x2d0/0x51c)
+[<c0910308>] (tegra_ehci_probe) from [<c07775a8>] (platform_drv_probe+0x50/0xb0)
+[<c07775a8>] (platform_drv_probe) from [<c0775f0c>] (driver_probe_device+0x1f4/0x2b0)
+[<c0775f0c>] (driver_probe_device) from [<c0776074>] (__driver_attach+0xac/0xb0)
+[<c0776074>] (__driver_attach) from [<c0774520>] (bus_for_each_dev+0x54/0x88)
+[<c0774520>] (bus_for_each_dev) from [<c0775500>] (bus_add_driver+0xe8/0x1f4)
+[<c0775500>] (bus_add_driver) from [<c0776890>] (driver_register+0x78/0xf4)
+[<c0776890>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
 [<c0301e5c>] (do_one_initcall) from [<c1100d80>] (kernel_init_freeable+0x120/0x1ec)
-[<c1100d80>] (kernel_init_freeable) from [<c0b59d08>] (kernel_init+0x8/0x118)
-[<c0b59d08>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
----[ end trace 0030b58d573f7314 ]---
+[<c1100d80>] (kernel_init_freeable) from [<c0b59f10>] (kernel_init+0x8/0x118)
+[<c0b59f10>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
+---[ end trace 390fd57a2828070f ]---
 tegra-ehci c5000000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci c5000000.usb: continuing, but please update your DT
 ulpi_phy_power_on: ulpi write failed
@@ -344,20 +344,20 @@
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
 mmc0: host does not support reading read-only switch, assuming write-enable
-tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
+ mmcblk0: p1
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
+tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: hctosys: unable to read the hardware clock
- mmcblk0: p1
-EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 mmc1: new high speed SD card at address b368
+rtc-em3027 2-0056: hctosys: unable to read the hardware clock
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
+EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-05-10 22:56:31.403099829 -0700
+++ /tmp/foob	2016-05-10 22:56:31.407099830 -0700
@@ -45,9 +45,9 @@
 	 #################################################################
 	 #################################################################
 	 ##########################################
-	 907.2 KiB/s
+	 903.3 KiB/s
 done
-Bytes transferred = 5376256 (520900 hex)
+Bytes transferred = 5376968 (520bc8 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x520900 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x520bc8 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102b3
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc7-next-20160510-93572-g416636b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon May 9 23:50:01 PDT 2016
+Linux version 4.6.0-rc7-next-20160511-94083-g13425f1 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue May 10 22:49:05 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -89,7 +89,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 961792K/1048576K available (7415K kernel code, 374K rwdata, 2556K rodata, 1024K init, 366K bss, 21248K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 961792K/1048576K available (7415K kernel code, 374K rwdata, 2560K rodata, 1024K init, 366K bss, 21248K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -97,7 +97,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0abcf8c   (10964 kB)
+      .text : 0xc0008000 - 0xc0abdfa4   (10968 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
       .data : 0xc0c00000 - 0xc0c5d8c0   ( 375 kB)
        .bss : 0xc0c5f000 - 0xc0cbab18   ( 367 kB)
@@ -265,25 +265,25 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1b4/0x218
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc7-next-20160510-93572-g416636b #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc7-next-20160511-94083-g13425f1 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c010f1e8>] (unwind_backtrace) from [<c010a864>] (show_stack+0x10/0x14)
-[<c010a864>] (show_stack) from [<c033da54>] (dump_stack+0x84/0xa4)
-[<c033da54>] (dump_stack) from [<c011f308>] (__warn+0xd4/0x100)
+[<c010a864>] (show_stack) from [<c033da34>] (dump_stack+0x84/0xa4)
+[<c033da34>] (dump_stack) from [<c011f308>] (__warn+0xd4/0x100)
 [<c011f308>] (__warn) from [<c011f354>] (warn_slowpath_null+0x20/0x28)
-[<c011f354>] (warn_slowpath_null) from [<c03c0b6c>] (__of_reset_control_get+0x1b4/0x218)
-[<c03c0b6c>] (__of_reset_control_get) from [<c05134c4>] (tegra_ehci_probe+0x2c8/0x514)
-[<c05134c4>] (tegra_ehci_probe) from [<c0440e54>] (platform_drv_probe+0x50/0xb0)
-[<c0440e54>] (platform_drv_probe) from [<c043f880>] (driver_probe_device+0x1f4/0x2b0)
-[<c043f880>] (driver_probe_device) from [<c043f9e8>] (__driver_attach+0xac/0xb0)
-[<c043f9e8>] (__driver_attach) from [<c043dea4>] (bus_for_each_dev+0x54/0x88)
-[<c043dea4>] (bus_for_each_dev) from [<c043ee74>] (bus_add_driver+0xe8/0x1f4)
-[<c043ee74>] (bus_add_driver) from [<c04401e4>] (driver_register+0x78/0xf4)
-[<c04401e4>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
+[<c011f354>] (warn_slowpath_null) from [<c03c0be8>] (__of_reset_control_get+0x1b4/0x218)
+[<c03c0be8>] (__of_reset_control_get) from [<c05135f4>] (tegra_ehci_probe+0x2c8/0x514)
+[<c05135f4>] (tegra_ehci_probe) from [<c0440ed0>] (platform_drv_probe+0x50/0xb0)
+[<c0440ed0>] (platform_drv_probe) from [<c043f8fc>] (driver_probe_device+0x1f4/0x2b0)
+[<c043f8fc>] (driver_probe_device) from [<c043fa64>] (__driver_attach+0xac/0xb0)
+[<c043fa64>] (__driver_attach) from [<c043df20>] (bus_for_each_dev+0x54/0x88)
+[<c043df20>] (bus_for_each_dev) from [<c043eef0>] (bus_add_driver+0xe8/0x1f4)
+[<c043eef0>] (bus_add_driver) from [<c0440260>] (driver_register+0x78/0xf4)
+[<c0440260>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
 [<c010179c>] (do_one_initcall) from [<c0b00d94>] (kernel_init_freeable+0x11c/0x1e8)
-[<c0b00d94>] (kernel_init_freeable) from [<c07f07b4>] (kernel_init+0x8/0x118)
-[<c07f07b4>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
----[ end trace b7ccbe10aa690c5c ]---
+[<c0b00d94>] (kernel_init_freeable) from [<c07f09e8>] (kernel_init+0x8/0x118)
+[<c07f09e8>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
+---[ end trace fc976a3c3d824d77 ]---
 tegra-ehci c5000000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci c5000000.usb: continuing, but please update your DT
 ulpi_phy_power_on: ulpi write failed
@@ -330,12 +330,12 @@
 tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
 mmc0: host does not support reading read-only switch, assuming write-enable
-mmc0: new high speed SDHC card at address b368
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
- mmcblk0: p1
+mmc0: new high speed SDHC card at address b368
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
  mmcblk1: p1 p2
+ mmcblk0: p1
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
 NET: Registered protocol family 10
@@ -371,7 +371,7 @@
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:1.
+VFS: Mounted root (ext3 filesystem) on device 179:17.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2016-05-10 22:56:31.411099832 -0700
+++ /tmp/foob	2016-05-10 22:56:31.415099833 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 773728 us
-Basic modules init took 348541 us
+Tegraboot started after 833930 us
+Basic modules init took 348501 us
 Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14370 us
+Next stage binary read took 14371 us
 Carveout took 252725 us
-CPU initialization took 92363 us
-Total time taken by TegraBoot 707999 us
+CPU initialization took 92313 us
+Total time taken by TegraBoot 707910 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.612] LPDDR4 Training: Number of tables = 10
+64b[0001.670] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,7 +117,7 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-12451840 bytes read in 754 ms (15.7 MiB/s)
+12519424 bytes read in 744 ms (16 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
 37108 bytes read in 454 ms (79.1 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc7-next-20160510+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon May 9 23:51:48 PDT 2016
+Linux version 4.6.0-rc7-next-20160511+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue May 10 22:51:04 PDT 2016
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -166,15 +166,15 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffff80007a000000-ffff80007dffffff]
-Memory: 2980536K/3131392K available (7400K kernel code, 640K rwdata, 3156K rodata, 960K init, 257K bss, 134472K reserved, 16384K cma-reserved)
+Memory: 2980468K/3131392K available (7440K kernel code, 642K rwdata, 3180K rodata, 960K init, 260K bss, 134540K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff0000087b0000   (  7360 KB)
-    .rodata : 0xffff0000087b0000 - 0xffff000008ad0000   (  3200 KB)
-      .init : 0xffff000008ad0000 - 0xffff000008bc0000   (   960 KB)
-      .data : 0xffff000008bc0000 - 0xffff000008c60000   (   640 KB)
-       .bss : 0xffff000008c60000 - 0xffff000008ca07c0   (   258 KB)
+      .text : 0xffff000008080000 - 0xffff0000087c0000   (  7424 KB)
+    .rodata : 0xffff0000087c0000 - 0xffff000008ae0000   (  3200 KB)
+      .init : 0xffff000008ae0000 - 0xffff000008bd0000   (   960 KB)
+      .data : 0xffff000008bd0000 - 0xffff000008c70800   (   642 KB)
+       .bss : 0xffff000008c70800 - 0xffff000008cb1ac0   (   261 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -224,7 +224,7 @@
 pinctrl core: initialized pinctrl subsystem
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffff0000087b6000, 1 data @ ffff000008bc4000)
+vdso: 2 pages (1 code @ ffff0000087c6000, 1 data @ ffff000008bd4000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -263,7 +263,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.367:1): initialized
+audit: type=2000 audit(0.371:1): initialized
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -324,7 +324,7 @@
 ALSA device list:
   No soundcards found.
 Waiting for root device PARTUUID=00000000-0001-0000-788f-a75600000000...
-mmc0: new high speed MMC card at address 0001
+mmc0: new MMC card at address 0001
 mmcblk0: mmc0:0001 SDW16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SDW16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SDW16G partition 2 4.00 MiB
@@ -334,7 +334,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 960K (ffff800000ad0000 - ffff800000bc0000)
+Freeing unused kernel memory: 960K (ffff800000ae0000 - ffff800000bd0000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-05-10 22:56:31.419099834 -0700
+++ /tmp/foob	2016-05-10 22:56:31.423099834 -0700
@@ -41,7 +41,7 @@
 	 ###########################
 	 3.2 MiB/s
 done
-Bytes transferred = 7070312 (6be268 hex)
+Bytes transferred = 7070584 (6be378 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -52,7 +52,7 @@
 done
 Bytes transferred = 51627 (c9ab hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6be268 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6be378 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff9aa ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc7-next-20160510-93572-g416636b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon May 9 23:50:03 PDT 2016
+Linux version 4.6.0-rc7-next-20160511-94083-g13425f1 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue May 10 22:49:08 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -89,7 +89,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc106a424   (14730 kB)
+      .text : 0xc0208000 - 0xc106a43c   (14730 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
       .data : 0xc1300000 - 0xc141c6e0   (1138 kB)
        .bss : 0xc141e000 - 0xc1473ca0   ( 344 kB)
@@ -239,27 +239,27 @@
 ehci-atmel: EHCI Atmel driver
 tegra-ehci: Tegra EHCI driver
 ------------[ cut here ]------------
-WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
+WARNING: CPU: 1 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc7-next-20160510-93572-g416636b #1
+CPU: 1 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc7-next-20160511-94083-g13425f1 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c030f990>] (unwind_backtrace) from [<c030b088>] (show_stack+0x10/0x14)
-[<c030b088>] (show_stack) from [<c0586208>] (dump_stack+0x84/0xa4)
-[<c0586208>] (dump_stack) from [<c0341b6c>] (__warn+0xd4/0x100)
+[<c030b088>] (show_stack) from [<c0586188>] (dump_stack+0x84/0xa4)
+[<c0586188>] (dump_stack) from [<c0341b6c>] (__warn+0xd4/0x100)
 [<c0341b6c>] (__warn) from [<c0341bb8>] (warn_slowpath_null+0x20/0x28)
-[<c0341bb8>] (warn_slowpath_null) from [<c069eb08>] (__of_reset_control_get+0x1bc/0x228)
-[<c069eb08>] (__of_reset_control_get) from [<c0910220>] (tegra_ehci_probe+0x2d0/0x51c)
-[<c0910220>] (tegra_ehci_probe) from [<c0777528>] (platform_drv_probe+0x50/0xb0)
-[<c0777528>] (platform_drv_probe) from [<c0775e8c>] (driver_probe_device+0x1f4/0x2b0)
-[<c0775e8c>] (driver_probe_device) from [<c0775ff4>] (__driver_attach+0xac/0xb0)
-[<c0775ff4>] (__driver_attach) from [<c07744a0>] (bus_for_each_dev+0x54/0x88)
-[<c07744a0>] (bus_for_each_dev) from [<c0775480>] (bus_add_driver+0xe8/0x1f4)
-[<c0775480>] (bus_add_driver) from [<c0776810>] (driver_register+0x78/0xf4)
-[<c0776810>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
+[<c0341bb8>] (warn_slowpath_null) from [<c069eb28>] (__of_reset_control_get+0x1bc/0x228)
+[<c069eb28>] (__of_reset_control_get) from [<c0910308>] (tegra_ehci_probe+0x2d0/0x51c)
+[<c0910308>] (tegra_ehci_probe) from [<c07775a8>] (platform_drv_probe+0x50/0xb0)
+[<c07775a8>] (platform_drv_probe) from [<c0775f0c>] (driver_probe_device+0x1f4/0x2b0)
+[<c0775f0c>] (driver_probe_device) from [<c0776074>] (__driver_attach+0xac/0xb0)
+[<c0776074>] (__driver_attach) from [<c0774520>] (bus_for_each_dev+0x54/0x88)
+[<c0774520>] (bus_for_each_dev) from [<c0775500>] (bus_add_driver+0xe8/0x1f4)
+[<c0775500>] (bus_add_driver) from [<c0776890>] (driver_register+0x78/0xf4)
+[<c0776890>] (driver_register) from [<c0301e5c>] (do_one_initcall+0x3c/0x16c)
 [<c0301e5c>] (do_one_initcall) from [<c1100d80>] (kernel_init_freeable+0x120/0x1ec)
-[<c1100d80>] (kernel_init_freeable) from [<c0b59d08>] (kernel_init+0x8/0x118)
-[<c0b59d08>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
----[ end trace dde69f0cdb61d1b6 ]---
+[<c1100d80>] (kernel_init_freeable) from [<c0b59f10>] (kernel_init+0x8/0x118)
+[<c0b59f10>] (kernel_init) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
+---[ end trace c0cde1ff25718475 ]---
 tegra-ehci 7d004000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d004000.usb: continuing, but please update your DT
 ohci_hcd: USB 1.1 'Open' Host Controller (OHCI) Driver
@@ -308,15 +308,15 @@
 sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
 tegra-pcie 3000.pcie-controller: 2x3 configuration
 tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
-mmc1: new high speed MMC card at address 0001
+mmc0: error -110 whilst initialising SD card
+mmc1: new MMC card at address 0001
 mmcblk1: mmc1:0001 SEM16G 14.8 GiB 
+tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB
 Primary GPT is invalid, using alternate GPT.
  mmcblk1: p1 p2 p3 p4 p5
-mmc0: error -110 whilst initialising SD card
-tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -379,12 +379,14 @@
 vdd_sata,avdd_plle: disabling
 usb1_vbus: disabling
 EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 EXT4-fs (mmcblk1p1): warning: mounting fs with errors, running e2fsck is recommended
 EXT4-fs (mmcblk1p1): recovery complete
 EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
+asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0f:67:60
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.964007] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# 
\ No newline at end of file
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-05-10 22:56:31.427099835 -0700
+++ /tmp/foob	2016-05-10 22:56:31.431099836 -0700
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 ##########################################
-	 3.1 MiB/s
+	 2.8 MiB/s
 done
-Bytes transferred = 5376256 (520900 hex)
+Bytes transferred = 5376968 (520bc8 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.6 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 51627 (c9ab hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x520900 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x520bc8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff9aa ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc7-next-20160510-93572-g416636b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon May 9 23:50:01 PDT 2016
+Linux version 4.6.0-rc7-next-20160511-94083-g13425f1 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue May 10 22:49:05 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -81,7 +81,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2000896K/2096128K available (7415K kernel code, 374K rwdata, 2556K rodata, 1024K init, 366K bss, 29696K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2000896K/2096128K available (7415K kernel code, 374K rwdata, 2560K rodata, 1024K init, 366K bss, 29696K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -89,7 +89,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0abcf8c   (10964 kB)
+      .text : 0xc0008000 - 0xc0abdfa4   (10968 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
       .data : 0xc0c00000 - 0xc0c5d8c0   ( 375 kB)
        .bss : 0xc0c5f000 - 0xc0cbab18   ( 367 kB)
@@ -231,25 +231,25 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1b4/0x218
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc7-next-20160510-93572-g416636b #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-rc7-next-20160511-94083-g13425f1 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c010f1e8>] (unwind_backtrace) from [<c010a864>] (show_stack+0x10/0x14)
-[<c010a864>] (show_stack) from [<c033da54>] (dump_stack+0x84/0xa4)
-[<c033da54>] (dump_stack) from [<c011f308>] (__warn+0xd4/0x100)
+[<c010a864>] (show_stack) from [<c033da34>] (dump_stack+0x84/0xa4)
+[<c033da34>] (dump_stack) from [<c011f308>] (__warn+0xd4/0x100)
 [<c011f308>] (__warn) from [<c011f354>] (warn_slowpath_null+0x20/0x28)
-[<c011f354>] (warn_slowpath_null) from [<c03c0b6c>] (__of_reset_control_get+0x1b4/0x218)
-[<c03c0b6c>] (__of_reset_control_get) from [<c05134c4>] (tegra_ehci_probe+0x2c8/0x514)
-[<c05134c4>] (tegra_ehci_probe) from [<c0440e54>] (platform_drv_probe+0x50/0xb0)
-[<c0440e54>] (platform_drv_probe) from [<c043f880>] (driver_probe_device+0x1f4/0x2b0)
-[<c043f880>] (driver_probe_device) from [<c043f9e8>] (__driver_attach+0xac/0xb0)
-[<c043f9e8>] (__driver_attach) from [<c043dea4>] (bus_for_each_dev+0x54/0x88)
-[<c043dea4>] (bus_for_each_dev) from [<c043ee74>] (bus_add_driver+0xe8/0x1f4)
-[<c043ee74>] (bus_add_driver) from [<c04401e4>] (driver_register+0x78/0xf4)
-[<c04401e4>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
+[<c011f354>] (warn_slowpath_null) from [<c03c0be8>] (__of_reset_control_get+0x1b4/0x218)
+[<c03c0be8>] (__of_reset_control_get) from [<c05135f4>] (tegra_ehci_probe+0x2c8/0x514)
+[<c05135f4>] (tegra_ehci_probe) from [<c0440ed0>] (platform_drv_probe+0x50/0xb0)
+[<c0440ed0>] (platform_drv_probe) from [<c043f8fc>] (driver_probe_device+0x1f4/0x2b0)
+[<c043f8fc>] (driver_probe_device) from [<c043fa64>] (__driver_attach+0xac/0xb0)
+[<c043fa64>] (__driver_attach) from [<c043df20>] (bus_for_each_dev+0x54/0x88)
+[<c043df20>] (bus_for_each_dev) from [<c043eef0>] (bus_add_driver+0xe8/0x1f4)
+[<c043eef0>] (bus_add_driver) from [<c0440260>] (driver_register+0x78/0xf4)
+[<c0440260>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
 [<c010179c>] (do_one_initcall) from [<c0b00d94>] (kernel_init_freeable+0x11c/0x1e8)
-[<c0b00d94>] (kernel_init_freeable) from [<c07f07b4>] (kernel_init+0x8/0x118)
-[<c07f07b4>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
----[ end trace c91586d6d00bf85b ]---
+[<c0b00d94>] (kernel_init_freeable) from [<c07f09e8>] (kernel_init+0x8/0x118)
+[<c07f09e8>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
+---[ end trace ec1ec0bf9e2cecba ]---
 tegra-ehci 7d004000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d004000.usb: continuing, but please update your DT
 usbcore: registered new interface driver cdc_acm
@@ -276,7 +276,7 @@
 mmc1: SDHCI controller on 78000600.sdhci [78000600.sdhci] using ADMA
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
-mmc1: new high speed MMC card at address 0001
+mmc1: new MMC card at address 0001
 mmcblk1: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
@@ -379,13 +379,14 @@
 ALSA device list:
   #0: NVIDIA Tegra Beaver
 EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 EXT4-fs (mmcblk1p1): warning: mounting fs with errors, running e2fsck is recommended
 EXT4-fs (mmcblk1p1): recovery complete
 EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
-bash: cannot set terminal process group (-1): Inappropri[    7.480918] usb 2-1: new high-speed USB device number 2 using tegra-ehci
-ate ioctl for device
+asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0f:67:60
+bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
