Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2015-10-20 23:53:54.349991578 -0700
+++ /tmp/foob	2015-10-20 23:53:54.353991574 -0700
@@ -1,6 +1,6 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
-Linux version 4.3.0-rc6-next-20151020+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Oct 19 22:49:04 PDT 2015
+Linux version 4.3.0-rc6-next-20151021+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Oct 20 23:49:14 PDT 2015
 CPU: AArch64 Processor [411fd070] revision 0
 Detected PIPT I-cache on CPU0
 alternatives: enabling workaround for ARM erratum 832075
@@ -19,7 +19,7 @@
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
 software IO TLB [mem 0x79c00000-0x7dc00000] (64MB) mapped at [ffffffc039c00000-ffffffc03dbfffff]
-Memory: 938888K/1048576K available (5538K kernel code, 493K rwdata, 2328K rodata, 540K init, 206K bss, 93304K reserved, 16384K cma-reserved)
+Memory: 938888K/1048576K available (5542K kernel code, 488K rwdata, 2328K rodata, 540K init, 206K bss, 93304K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     vmalloc : 0xffffff8000000000 - 0xffffffbdbfff0000   (   246 GB)
     vmemmap : 0xffffffbdc0000000 - 0xffffffbfc0000000   (     8 GB maximum)
@@ -28,9 +28,9 @@
     PCI I/O : 0xffffffbffae00000 - 0xffffffbffbe00000   (    16 MB)
     modules : 0xffffffbffc000000 - 0xffffffc000000000   (    64 MB)
     memory  : 0xffffffc000000000 - 0xffffffc040000000   (  1024 MB)
-      .init : 0xffffffc000831000 - 0xffffffc0008b8000   (   540 KB)
-      .text : 0xffffffc000080000 - 0xffffffc0008309e4   (  7875 KB)
-      .data : 0xffffffc0008c9000 - 0xffffffc000944400   (   493 KB)
+      .init : 0xffffffc000832000 - 0xffffffc0008b9000   (   540 KB)
+      .text : 0xffffffc000080000 - 0xffffffc0008319e4   (  7879 KB)
+      .data : 0xffffffc0008ca000 - 0xffffffc000944200   (   489 KB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 64.
@@ -86,7 +86,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 2, 16384 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.590:1): initialized
+audit: type=2000 audit(0.640:1): initialized
 HugeTLB registered 2 MB page size, pre-allocated 0 pages
 VFS: Disk quotas dquot_6.6.0
 VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
@@ -134,9 +134,9 @@
 EXT4-fs (vda2): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 254:2.
 devtmpfs: mounted
+Freeing unused kernel memory: 540K (ffffffc000832000 - ffffffc0008b9000)
+Freeing alternatives memory: 48K (ffffffc0008b9000 - ffffffc0008c5000)
  
-Freeing unused kernel memory: 540K (ffffffc000831000 - ffffffc0008b8000)
-Freeing alternatives memory: 48K (ffffffc0008b8000 - ffffffc0008c4000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 bash-4.3#  
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-10-20 23:53:54.377991551 -0700
+++ /tmp/foob	2015-10-20 23:53:54.381991547 -0700
@@ -40,18 +40,18 @@
 	 ######################################################
 	 3.2 MiB/s
 done
-Bytes transferred = 6517152 (6371a0 hex)
+Bytes transferred = 6517888 (637480 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 2.4 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6371a0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x637480 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.3.0-rc6-next-20151020-50777-g1c7a47a (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Oct 19 22:47:44 PDT 2015
+Linux version 4.3.0-rc6-next-20151021-51114-g5d491cc (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Oct 20 23:47:50 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -67,7 +67,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0fb5900, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c0fb58c0, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -80,7 +80,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998644K/2096128K available (8225K kernel code, 1038K rwdata, 3708K rodata, 1036K init, 340K bss, 31948K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998644K/2096128K available (8221K kernel code, 1038K rwdata, 3708K rodata, 1040K init, 340K bss, 31948K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,9 +88,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0db079c   (11938 kB)
-      .init : 0xc0db1000 - 0xc0eb4000   (1036 kB)
-      .data : 0xc0eb4000 - 0xc0fb7860   (1039 kB)
+      .text : 0xc0208000 - 0xc0daf7bc   (11934 kB)
+      .init : 0xc0db0000 - 0xc0eb4000   (1040 kB)
+      .data : 0xc0eb4000 - 0xc0fb7820   (1039 kB)
        .bss : 0xc0fba000 - 0xc100f3e8   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -286,7 +286,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 78000600.sdhci [78000600.sdhci] using ADMA
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-05-01 14:44:27 UTC (957192267)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-05-02 15:44:32 UTC (957282272)
 vdd_cam_1v8_reg: disabling
 dvdd_ts: disabling
 usb1_vbus: disabling
@@ -304,7 +304,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1036K (c0db1000 - c0eb4000)
+Freeing unused kernel memory: 1040K (c0db0000 - c0eb4000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2015-10-20 23:53:54.401991528 -0700
+++ /tmp/foob	2015-10-20 23:53:54.405991525 -0700
@@ -39,18 +39,18 @@
 	 ##############################
 	 3.4 MiB/s
 done
-Bytes transferred = 5205712 (4f6ed0 hex)
+Bytes transferred = 5208424 (4f7968 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.8 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4f6ed0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4f7968 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.3.0-rc6-next-20151020-50777-g1c7a47a (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Oct 19 22:48:01 PDT 2015
+Linux version 4.3.0-rc6-next-20151021-51114-g5d491cc (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Oct 20 23:48:06 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0a12480, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0a14540, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002340K/2096128K available (7151K kernel code, 351K rwdata, 2444K rodata, 336K init, 364K bss, 28252K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2002328K/2096128K available (7151K kernel code, 352K rwdata, 2448K rodata, 340K init, 364K bss, 28264K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0967150   (9597 kB)
-      .init : 0xc0968000 - 0xc09bc000   ( 336 kB)
-      .data : 0xc09bc000 - 0xc0a13fa0   ( 352 kB)
-       .bss : 0xc0a16000 - 0xc0a710c0   ( 365 kB)
+      .text : 0xc0008000 - 0xc096819c   (9601 kB)
+      .init : 0xc0969000 - 0xc09be000   ( 340 kB)
+      .data : 0xc09be000 - 0xc0a16060   ( 353 kB)
+       .bss : 0xc0a19000 - 0xc0a740c0   ( 365 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -296,7 +296,7 @@
 tegra-snd-rt5640 sound: rt5640-aif1 <-> 70080400.i2s mapping ok
 input: NVIDIA Tegra Dalmore Headphones as /devices/soc0/sound/sound/card0/input0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-05-01 14:45:01 UTC (957192301)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-05-02 15:45:04 UTC (957282304)
 vdd_cam_1v8_reg: disabling
 dvdd_ts: disabling
 usb1_vbus: disabling
@@ -309,7 +309,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 336K (c0968000 - c09bc000)
+Freeing unused kernel memory: 340K (c0969000 - c09be000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-10-20 23:53:54.409991521 -0700
+++ /tmp/foob	2015-10-20 23:53:54.413991517 -0700
@@ -40,10 +40,11 @@
 	 #################################################################
 	 #################################################################
 	 ######################################################
-	 3.3 MiB/s
+	 929.7 KiB/s
 done
-Bytes transferred = 6517152 (6371a0 hex)
+Bytes transferred = 6517888 (637480 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
+Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
@@ -53,7 +54,7 @@
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6371a0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x637480 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012a89
@@ -61,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.3.0-rc6-next-20151020-50777-g1c7a47a (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Oct 19 22:47:44 PDT 2015
+Linux version 4.3.0-rc6-next-20151021-51114-g5d491cc (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Oct 20 23:47:50 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -69,7 +70,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0fb5900, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c0fb58c0, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -82,7 +83,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998536K/2096128K available (8225K kernel code, 1038K rwdata, 3708K rodata, 1036K init, 340K bss, 32056K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998536K/2096128K available (8221K kernel code, 1038K rwdata, 3708K rodata, 1040K init, 340K bss, 32056K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,9 +91,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0db079c   (11938 kB)
-      .init : 0xc0db1000 - 0xc0eb4000   (1036 kB)
-      .data : 0xc0eb4000 - 0xc0fb7860   (1039 kB)
+      .text : 0xc0208000 - 0xc0daf7bc   (11934 kB)
+      .init : 0xc0db0000 - 0xc0eb4000   (1040 kB)
+      .data : 0xc0eb4000 - 0xc0fb7820   (1039 kB)
        .bss : 0xc0fba000 - 0xc100f3e8   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -330,14 +331,17 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0600.sdhci [700b0600.sdhci] using ADMA 64-bit
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:18 UTC (946684818)
-+1.8V_RUN_CAM: disabling
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:23 UTC (946684823)
 ata1: SATA link down (SStatus 0 SControl 300)
++1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
 +1.05V_RUN_CAM_REAR: disabling
 +3.3V_RUN_TOUCH: disabling
 +2.8V_RUN_CAM_AF: disabling
++1.8V_RUN_VPP_FUSE: disabling
++USB0_VBUS_SW: disabling
+Waiting for root device /dev/mmcblk0p1...
 mmc1: MAN_BKOPS_EN bit is not set
 mmc1: new high speed MMC card at address 0001
 mmcblk0: mmc1:0001 SEM16G 14.6 GiB 
@@ -346,16 +350,14 @@
 mmcblk0rpmb: mmc1:0001 SEM16G partition 3 4.00 MiB
 Primary GPT is invalid, using alternate GPT.
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9
-+1.8V_RUN_VPP_FUSE: disabling
-+USB0_VBUS_SW: disabling
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
 EXT4-fs (mmcblk0p1): couldn't mount as ext2 due to feature incompatibilities
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1036K (c0db1000 - c0eb4000)
+Freeing unused kernel memory: 1040K (c0db0000 - c0eb4000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2015-10-20 23:53:54.421991509 -0700
+++ /tmp/foob	2015-10-20 23:53:54.421991509 -0700
@@ -39,9 +39,9 @@
 	 #################################################################
 	 #################################################################
 	 ##############################
-	 3.2 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 5205712 (4f6ed0 hex)
+Bytes transferred = 5208424 (4f7968 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -52,7 +52,7 @@
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4f6ed0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4f7968 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012a89
@@ -63,7 +63,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.3.0-rc6-next-20151020-50777-g1c7a47a (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Oct 19 22:48:01 PDT 2015
+Linux version 4.3.0-rc6-next-20151021-51114-g5d491cc (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Oct 20 23:48:06 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -71,7 +71,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0a12480, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0a14540, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -84,7 +84,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002232K/2096128K available (7151K kernel code, 351K rwdata, 2444K rodata, 336K init, 364K bss, 28360K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2002220K/2096128K available (7151K kernel code, 352K rwdata, 2448K rodata, 340K init, 364K bss, 28372K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,10 +92,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0967150   (9597 kB)
-      .init : 0xc0968000 - 0xc09bc000   ( 336 kB)
-      .data : 0xc09bc000 - 0xc0a13fa0   ( 352 kB)
-       .bss : 0xc0a16000 - 0xc0a710c0   ( 365 kB)
+      .text : 0xc0008000 - 0xc096819c   (9601 kB)
+      .init : 0xc0969000 - 0xc09be000   ( 340 kB)
+      .data : 0xc09be000 - 0xc0a16060   ( 353 kB)
+       .bss : 0xc0a19000 - 0xc0a740c0   ( 365 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -327,9 +327,9 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0600.sdhci [700b0600.sdhci] using ADMA 64-bit
 tegra30-i2s 70301100.i2s: DMA channels sourced from device 70300000.ahub
+ata1: SATA link down (SStatus 0 SControl 300)
 tegra-snd-rt5640 sound: ASoC: CODEC DAI rt5640-aif1 not registered
 tegra-snd-rt5640 sound: snd_soc_register_card failed (-517)
-ata1: SATA link down (SStatus 0 SControl 300)
 mmc1: MAN_BKOPS_EN bit is not set
 mmc1: new high speed MMC card at address 0001
 mmcblk0: mmc1:0001 SEM16G 14.6 GiB 
@@ -342,7 +342,7 @@
 tegra-snd-rt5640 sound: rt5640-aif1 <-> 70301100.i2s mapping ok
 input: NVIDIA Tegra Jetson TK1 Headphones as /devices/soc0/sound/sound/card1/input0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:51 UTC (946684851)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:59 UTC (946684859)
 input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input1
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
@@ -362,7 +362,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 336K (c0968000 - c09bc000)
+Freeing unused kernel memory: 340K (c0969000 - c09be000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-10-20 23:53:54.429991503 -0700
+++ /tmp/foob	2015-10-20 23:53:54.429991503 -0700
@@ -46,20 +46,20 @@
 	 #################################################################
 	 #################################################################
 	 ######################################################
-	 4.1 MiB/s
+	 4 MiB/s
 done
-Bytes transferred = 6517152 (6371a0 hex)
+Bytes transferred = 6517888 (637480 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 4 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x6371a0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x637480 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -67,7 +67,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.3.0-rc6-next-20151020-50777-g1c7a47a (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Oct 19 22:47:44 PDT 2015
+Linux version 4.3.0-rc6-next-20151021-51114-g5d491cc (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Oct 20 23:47:50 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -75,7 +75,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0fb5900, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c0fb58c0, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -88,7 +88,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 959452K/1048576K available (8225K kernel code, 1038K rwdata, 3708K rodata, 1036K init, 340K bss, 23588K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 959452K/1048576K available (8221K kernel code, 1038K rwdata, 3708K rodata, 1040K init, 340K bss, 23588K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -96,9 +96,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0db079c   (11938 kB)
-      .init : 0xc0db1000 - 0xc0eb4000   (1036 kB)
-      .data : 0xc0eb4000 - 0xc0fb7860   (1039 kB)
+      .text : 0xc0208000 - 0xc0daf7bc   (11934 kB)
+      .init : 0xc0db0000 - 0xc0eb4000   (1040 kB)
+      .data : 0xc0eb4000 - 0xc0fb7820   (1039 kB)
        .bss : 0xc0fba000 - 0xc100f3e8   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
@@ -241,21 +241,21 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 0 at drivers/soc/tegra/pmc.c:607 tegra_pmc_enter_suspend_mode+0x194/0x1ac()
 Modules linked in:
-CPU: 0 PID: 0 Comm: swapper/0 Not tainted 4.3.0-rc6-next-20151020-50777-g1c7a47a #1
+CPU: 0 PID: 0 Comm: swapper/0 Not tainted 4.3.0-rc6-next-20151021-51114-g5d491cc #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c0218728>] (unwind_backtrace) from [<c0213e40>] (show_stack+0x10/0x14)
-[<c0213e40>] (show_stack) from [<c048308c>] (dump_stack+0x78/0x94)
-[<c048308c>] (dump_stack) from [<c024a98c>] (warn_slowpath_common+0x74/0xb0)
-[<c024a98c>] (warn_slowpath_common) from [<c024a9e4>] (warn_slowpath_null+0x1c/0x24)
-[<c024a9e4>] (warn_slowpath_null) from [<c05624a8>] (tegra_pmc_enter_suspend_mode+0x194/0x1ac)
-[<c05624a8>] (tegra_pmc_enter_suspend_mode) from [<c0241e3c>] (tegra_idle_lp2_last+0xc/0x40)
-[<c0241e3c>] (tegra_idle_lp2_last) from [<c02429a8>] (tegra20_idle_lp2_coupled+0xf0/0x1e8)
-[<c02429a8>] (tegra20_idle_lp2_coupled) from [<c0823404>] (cpuidle_enter_state+0xbc/0x260)
-[<c0823404>] (cpuidle_enter_state) from [<c08253d0>] (cpuidle_enter_state_coupled+0x33c/0x370)
-[<c08253d0>] (cpuidle_enter_state_coupled) from [<c027ee2c>] (cpu_startup_entry+0x174/0x24c)
-[<c027ee2c>] (cpu_startup_entry) from [<c0db1c1c>] (start_kernel+0x35c/0x3c4)
-[<c0db1c1c>] (start_kernel) from [<0020807c>] (0x20807c)
----[ end trace 194b0f320da8133b ]---
+[<c02185a8>] (unwind_backtrace) from [<c0213ca0>] (show_stack+0x10/0x14)
+[<c0213ca0>] (show_stack) from [<c048210c>] (dump_stack+0x78/0x94)
+[<c048210c>] (dump_stack) from [<c024998c>] (warn_slowpath_common+0x74/0xb0)
+[<c024998c>] (warn_slowpath_common) from [<c02499e4>] (warn_slowpath_null+0x1c/0x24)
+[<c02499e4>] (warn_slowpath_null) from [<c0561620>] (tegra_pmc_enter_suspend_mode+0x194/0x1ac)
+[<c0561620>] (tegra_pmc_enter_suspend_mode) from [<c0240e3c>] (tegra_idle_lp2_last+0xc/0x40)
+[<c0240e3c>] (tegra_idle_lp2_last) from [<c02419a8>] (tegra20_idle_lp2_coupled+0xf0/0x1e8)
+[<c02419a8>] (tegra20_idle_lp2_coupled) from [<c0822864>] (cpuidle_enter_state+0xbc/0x260)
+[<c0822864>] (cpuidle_enter_state) from [<c0824830>] (cpuidle_enter_state_coupled+0x33c/0x370)
+[<c0824830>] (cpuidle_enter_state_coupled) from [<c027de2c>] (cpu_startup_entry+0x174/0x24c)
+[<c027de2c>] (cpu_startup_entry) from [<c0db0c1c>] (start_kernel+0x35c/0x3c4)
+[<c0db0c1c>] (start_kernel) from [<0020807c>] (0x20807c)
+---[ end trace 9674da8d3fa56f7a ]---
 tegra-i2c 7000c500.i2c: i2c transfer timed out
 rtc-em3027 2-0056: em3027_get_time: read error
 rtc-em3027 2-0056: rtc core: registered rtc-em3027 as rtc0
@@ -344,7 +344,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on c8000600.sdhci [c8000600.sdhci] using ADMA
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-26 23:08:54 UTC (1419635334)
+rtc-em3027 2-0056: setting system clock to 2014-12-26 23:12:51 UTC (1419635571)
 Waiting for root device /dev/mmcblk0p1...
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
@@ -356,12 +356,12 @@
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
 hub 2-1:1.0: USB hub found
+hub 2-1:1.0: 4 ports detected
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1036K (c0db1000 - c0eb4000)
-hub 2-1:1.0: 4 ports detected
+Freeing unused kernel memory: 1040K (c0db0000 - c0eb4000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 usb 2-1.2: new high-speed USB device number 3 using tegra-ehci
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2015-10-20 23:53:54.437991494 -0700
+++ /tmp/foob	2015-10-20 23:53:54.441991490 -0700
@@ -47,7 +47,7 @@
 	 ##############################
 	 4.1 MiB/s
 done
-Bytes transferred = 5205712 (4f6ed0 hex)
+Bytes transferred = 5208424 (4f7968 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x4f6ed0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x4f7968 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -69,7 +69,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.3.0-rc6-next-20151020-50777-g1c7a47a (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Oct 19 22:48:01 PDT 2015
+Linux version 4.3.0-rc6-next-20151021-51114-g5d491cc (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Oct 20 23:48:06 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -78,7 +78,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0a12480, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0a14540, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -91,7 +91,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 963148K/1048576K available (7151K kernel code, 351K rwdata, 2444K rodata, 336K init, 364K bss, 19892K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 963136K/1048576K available (7151K kernel code, 352K rwdata, 2448K rodata, 340K init, 364K bss, 19904K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -99,10 +99,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0967150   (9597 kB)
-      .init : 0xc0968000 - 0xc09bc000   ( 336 kB)
-      .data : 0xc09bc000 - 0xc0a13fa0   ( 352 kB)
-       .bss : 0xc0a16000 - 0xc0a710c0   ( 365 kB)
+      .text : 0xc0008000 - 0xc096819c   (9601 kB)
+      .init : 0xc0969000 - 0xc09be000   ( 340 kB)
+      .data : 0xc09be000 - 0xc0a16060   ( 353 kB)
+       .bss : 0xc0a19000 - 0xc0a740c0   ( 365 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
@@ -239,7 +239,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.3.0-rc6-next-20151020-50777-g1c7a47a (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Oct 19 22:48:01 PDT 2015
+Linux version 4.3.0-rc6-next-20151021-51114-g5d491cc (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Oct 20 23:48:06 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -248,7 +248,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0a12480, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0a14540, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -261,7 +261,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 963148K/1048576K available (7151K kernel code, 351K rwdata, 2444K rodata, 336K init, 364K bss, 19892K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 963136K/1048576K available (7151K kernel code, 352K rwdata, 2448K rodata, 340K init, 364K bss, 19904K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -269,10 +269,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0967150   (9597 kB)
-      .init : 0xc0968000 - 0xc09bc000   ( 336 kB)
-      .data : 0xc09bc000 - 0xc0a13fa0   ( 352 kB)
-       .bss : 0xc0a16000 - 0xc0a710c0   ( 365 kB)
+      .text : 0xc0008000 - 0xc096819c   (9601 kB)
+      .init : 0xc0969000 - 0xc09be000   ( 340 kB)
+      .data : 0xc09be000 - 0xc0a16060   ( 353 kB)
+       .bss : 0xc0a19000 - 0xc0a740c0   ( 365 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
@@ -506,12 +506,12 @@
 tlv320aic23-codec 2-001a: ASoC: Failed to add route MICIN -> NULL -> Mic Input
 tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-26 23:09:15 UTC (1419635355)
+rtc-em3027 2-0056: setting system clock to 2014-12-26 23:13:14 UTC (1419635594)
 mmc0: host does not support reading read-only switch, assuming write-enable
+mmc0: new high speed SDHC card at address b368
 ALSA device list:
   #0: tegra-trimslice
 Waiting for root device /dev/mmcblk0p1...
-mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
  mmcblk0: p1
 usb 2-1: new high-speed USB device number 2 using tegra-ehci
@@ -526,7 +526,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 336K (c0968000 - c09bc000)
+Freeing unused kernel memory: 340K (c0969000 - c09be000)
 usb 2-1.2: new high-speed USB device number 3 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-10-20 23:53:54.445991487 -0700
+++ /tmp/foob	2015-10-20 23:53:54.449991483 -0700
@@ -40,18 +40,18 @@
 	 ######################################################
 	 3.3 MiB/s
 done
-Bytes transferred = 6517152 (6371a0 hex)
+Bytes transferred = 6517888 (637480 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6371a0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x637480 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.3.0-rc6-next-20151020-50777-g1c7a47a (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Oct 19 22:47:44 PDT 2015
+Linux version 4.3.0-rc6-next-20151021-51114-g5d491cc (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Oct 20 23:47:50 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -67,7 +67,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0fb5900, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c0fb58c0, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -80,7 +80,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998564K/2096128K available (8225K kernel code, 1038K rwdata, 3708K rodata, 1036K init, 340K bss, 32028K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998564K/2096128K available (8221K kernel code, 1038K rwdata, 3708K rodata, 1040K init, 340K bss, 32028K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,9 +88,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0db079c   (11938 kB)
-      .init : 0xc0db1000 - 0xc0eb4000   (1036 kB)
-      .data : 0xc0eb4000 - 0xc0fb7860   (1039 kB)
+      .text : 0xc0208000 - 0xc0daf7bc   (11934 kB)
+      .init : 0xc0db0000 - 0xc0eb4000   (1040 kB)
+      .data : 0xc0eb4000 - 0xc0fb7820   (1039 kB)
        .bss : 0xc0fba000 - 0xc100f3e8   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -350,14 +350,14 @@
 mmcblk0rpmb: mmc1:0001 SEM16G partition 3 128 KiB
 Primary GPT is invalid, using alternate GPT.
  mmcblk0: p1 p2 p3 p4 p5
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk0p1): warning: mounting fs with errors, running e2fsck is recommended
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1036K (c0db1000 - c0eb4000)
+Freeing unused kernel memory: 1040K (c0db0000 - c0eb4000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2015-10-20 23:53:54.453991479 -0700
+++ /tmp/foob	2015-10-20 23:53:54.457991475 -0700
@@ -27,6 +27,7 @@
 Tegra30 (Beaver) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug init=/bin/bash noinitrd'
 Tegra30 (Beaver) # setenv ipaddr 192.168.100.101; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra30 (Beaver) # tftpboot 0x81000000 zImage.tegra_defconfig
+Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'zImage.tegra_defconfig'.
@@ -37,9 +38,9 @@
 	 #################################################################
 	 #################################################################
 	 ##############################
-	 3.3 MiB/s
+	 3.5 MiB/s
 done
-Bytes transferred = 5205712 (4f6ed0 hex)
+Bytes transferred = 5208424 (4f7968 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -50,7 +51,7 @@
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4f6ed0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4f7968 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -61,7 +62,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.3.0-rc6-next-20151020-50777-g1c7a47a (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Oct 19 22:48:01 PDT 2015
+Linux version 4.3.0-rc6-next-20151021-51114-g5d491cc (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Oct 20 23:48:06 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -69,7 +70,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0a12480, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0a14540, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -82,7 +83,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002260K/2096128K available (7151K kernel code, 351K rwdata, 2444K rodata, 336K init, 364K bss, 28332K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2002248K/2096128K available (7151K kernel code, 352K rwdata, 2448K rodata, 340K init, 364K bss, 28344K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +91,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0967150   (9597 kB)
-      .init : 0xc0968000 - 0xc09bc000   ( 336 kB)
-      .data : 0xc09bc000 - 0xc0a13fa0   ( 352 kB)
-       .bss : 0xc0a16000 - 0xc0a710c0   ( 365 kB)
+      .text : 0xc0008000 - 0xc096819c   (9601 kB)
+      .init : 0xc0969000 - 0xc09be000   ( 340 kB)
+      .data : 0xc09be000 - 0xc0a16060   ( 353 kB)
+       .bss : 0xc0a19000 - 0xc0a740c0   ( 365 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -344,8 +345,8 @@
 mmcblk0: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk0boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk0boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
-mmcblk0rpmb: mmc1:0001 SEM16G partition 3 128 KiB
 usb 2-1: new high-speed USB device number 2 using tegra-ehci
+mmcblk0rpmb: mmc1:0001 SEM16G partition 3 128 KiB
 Primary GPT is invalid, using alternate GPT.
  mmcblk0: p1 p2 p3 p4 p5
 tegra-snd-rt5640 sound: rt5640-aif1 <-> 70080400.i2s mapping ok
@@ -363,7 +364,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 336K (c0968000 - c09bc000)
+Freeing unused kernel memory: 340K (c0969000 - c09be000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
