Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2017-11-28 19:34:40.747925415 -0800
+++ /tmp/foob	2017-11-28 19:34:40.751925433 -0800
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0000000000 [0x411fd070]
-Linux version 4.15.0-rc1-next-20171128-334872-g5bef2980adef (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Mon Nov 27 20:09:53 PST 2017
+Linux version 4.15.0-rc1-next-20171129-335140-gd127129e85a0 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Tue Nov 28 19:24:19 PST 2017
 Machine model: linux,dummy-virt
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -75,7 +75,7 @@
 DMI not present or invalid.
 NET: Registered protocol family 16
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.184:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.200:1): state=initialized audit_enabled=0 res=1
 cpuidle: using governor menu
 vdso: 2 pages (1 code @ ffff0000089f6000, 1 data @ ffff000008f75000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
@@ -182,7 +182,7 @@
 registered taskstats version 1
 Loading compiled-in X.509 certificates
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2017-11-28 04:16:51 UTC (1511842611)
+rtc-pl031 9010000.pl031: setting system clock to 2017-11-29 03:31:07 UTC (1511926267)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-11-28 19:34:40.783925577 -0800
+++ /tmp/foob	2017-11-28 19:34:40.787925596 -0800
@@ -40,7 +40,7 @@
 	 #################################################################
 	 #################################################################
 	 ###############################
-	 2.9 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 8081920 (7b5200 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-rc1-next-20171128-334872-g5bef2980adef (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Mon Nov 27 20:03:50 PST 2017
+Linux version 4.15.0-rc1-next-20171129-335140-gd127129e85a0 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Tue Nov 28 19:18:17 PST 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -94,8 +94,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0e00000   (12256 kB)
       .init : 0xc1300000 - 0xc1500000   (2048 kB)
-      .data : 0xc1500000 - 0xc1690b70   (1603 kB)
-       .bss : 0xc1690b70 - 0xc16f5de4   ( 405 kB)
+      .data : 0xc1500000 - 0xc1690af0   (1603 kB)
+       .bss : 0xc1690af0 - 0xc16f5d64   ( 405 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -370,19 +370,19 @@
 Registering SWP/SWPB emulation handler
 Loading compiled-in X.509 certificates
 mmc0: new high speed SDHC card at address aaaa
-mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 usb1_vbus: supplied by vdd-sys-5v0
 usb2_vbus: supplied by vdd-sys-5v0
 VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
-mmc1: new high speed MMC card at address 0001
+mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
+[drm] No driver support for vblank timestamp query.
  mmcblk0: p1 p2 p3
+mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB, chardev (237:0)
  mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
-[drm] No driver support for vblank timestamp query.
 Console: switching to colour frame buffer device 240x75
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
@@ -393,7 +393,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-04-20 11:05:26 UTC (956228726)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-04-21 10:19:53 UTC (956312393)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2017-11-28 19:34:40.795925632 -0800
+++ /tmp/foob	2017-11-28 19:34:40.795925632 -0800
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra114 (Dalmore) # 
 Tegra114 (Dalmore) # usb reset
 (Re)start USB...
@@ -38,20 +38,20 @@
 	 #################################################################
 	 #################################################################
 	 ################################
-	 3.1 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 6192624 (5e7df0 hex)
+Bytes transferred = 6193480 (5e8148 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.7 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 34502 (86c6 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5e7df0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5e8148 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff6c5 ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-rc1-next-20171128-14923-g5bef298 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Nov 27 20:02:56 PST 2017
+Linux version 4.15.0-rc1-next-20171129-15191-gd127129 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Tue Nov 28 19:17:25 PST 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -91,8 +91,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0ec2ba4   ( 779 kB)
-       .bss : 0xc0ec2ba4 - 0xc0f0d7ec   ( 300 kB)
+      .data : 0xc0e00000 - 0xc0ec2bb0   ( 779 kB)
+       .bss : 0xc0ec2bb0 - 0xc0f0d7ec   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -388,7 +388,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-04-20 11:02:54 UTC (956228574)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-04-21 10:17:19 UTC (956312239)
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
@@ -406,4 +406,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.111401] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.130900] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-11-28 19:34:40.803925667 -0800
+++ /tmp/foob	2017-11-28 19:34:40.803925667 -0800
@@ -44,7 +44,7 @@
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.7 MiB/s
+	 3.5 MiB/s
 done
 Bytes transferred = 66544 (103f0 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
@@ -56,7 +56,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-rc1-next-20171128-334872-g5bef2980adef (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Mon Nov 27 20:03:50 PST 2017
+Linux version 4.15.0-rc1-next-20171129-335140-gd127129e85a0 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Tue Nov 28 19:18:17 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -89,8 +89,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0e00000   (12256 kB)
       .init : 0xc1300000 - 0xc1500000   (2048 kB)
-      .data : 0xc1500000 - 0xc1690b70   (1603 kB)
-       .bss : 0xc1690b70 - 0xc16f5de4   ( 405 kB)
+      .data : 0xc1500000 - 0xc1690af0   (1603 kB)
+       .bss : 0xc1690af0 - 0xc16f5d64   ( 405 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -390,12 +390,6 @@
 +5V_USB_HS: supplied by +5V_SYS
 +3.3V_LP0: supplied by +3.3V_SYS
 +5V_HDMI_CON: supplied by +5V_SYS
-mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB, chardev (237:0)
- mmcblk0: p1
 +5V_SATA: supplied by +5V_SYS
 +12V_SATA: supplied by +VDD_MUX
  usb2-0: usb2-0 supply vbus not found, using dummy regulator
@@ -424,8 +418,13 @@
 +2.8V_RUN_CAM_AF: Bringing 0uV into 2800000-2800000uV
 as3722-ldo11: supplied by +3.3V_RUN
 +1.8V_RUN_VPP_FUSE: Bringing 0uV into 1800000-1800000uV
+mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 tegra-ahci 70027000.sata: AHCI 0001.0301 32 slots 2 ports 3 Gbps 0x1 impl platform mode
 tegra-ahci 70027000.sata: flags: 64bit ncq sntf pm led pio slum part sadm sds apst 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB, chardev (237:0)
 scsi host0: tegra-ahci
 scsi host1: tegra-ahci
 ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 345
@@ -433,6 +432,7 @@
 tegra-ehci 7d008000.usb: EHCI Host Controller
 tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci 7d008000.usb: irq 357, io mem 0x7d008000
+ mmcblk0: p1
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
@@ -520,7 +520,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:04:55 UTC (946685095)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:04:51 UTC (946685091)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2017-11-28 19:34:40.811925705 -0800
+++ /tmp/foob	2017-11-28 19:34:40.811925705 -0800
@@ -35,7 +35,7 @@
 	 ################################
 	 4 MiB/s
 done
-Bytes transferred = 6192624 (5e7df0 hex)
+Bytes transferred = 6193480 (5e8148 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -46,7 +46,7 @@
 done
 Bytes transferred = 66544 (103f0 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5e7df0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5e8148 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830133ef
@@ -54,7 +54,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-rc1-next-20171128-14923-g5bef298 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Nov 27 20:02:56 PST 2017
+Linux version 4.15.0-rc1-next-20171129-15191-gd127129 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Tue Nov 28 19:17:25 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -86,8 +86,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0ec2ba4   ( 779 kB)
-       .bss : 0xc0ec2ba4 - 0xc0f0d7ec   ( 300 kB)
+      .data : 0xc0e00000 - 0xc0ec2bb0   ( 779 kB)
+       .bss : 0xc0ec2bb0 - 0xc0f0d7ec   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -412,8 +412,8 @@
 as3722-ldo5: supplied by +1.8V_VDDIO
 as3722-ldo6: supplied by +3.3V_RUN
 as3722-ldo7: supplied by +1.8V_VDDIO
-+1.05V_RUN_CAM_REAR: Bringing 0uV into 1050000-1050000uV
 input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input1
++1.05V_RUN_CAM_REAR: Bringing 0uV into 1050000-1050000uV
 as3722-ldo9: supplied by +5V_SYS
 +3.3V_RUN_TOUCH: Bringing 0uV into 2800000-2800000uV
 as3722-ldo10: supplied by +5V_SYS
@@ -520,11 +520,10 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:14 UTC (946684814)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:13 UTC (946684813)
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
-cfg80211: failed to load regulatory.db
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
@@ -536,6 +535,7 @@
 ALSA device list:
   #0: tegra-hda at 0x70038000 irq 346
   #1: NVIDIA Tegra Jetson TK1
+cfg80211: failed to load regulatory.db
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig%tegra-fw_log.txt
--- /tmp/fooa	2017-11-28 19:34:40.819925741 -0800
+++ /tmp/foob	2017-11-28 19:34:40.819925741 -0800
@@ -28,25 +28,25 @@
 Load address: 0x82000000
 Loading: *#################################################################
 	 #################################################################
-	 ###########################################################T T ######
+	 #################################################################
 	 #################################################################
 	 #################################################################
 	 #################################################################
 	 ############################################
-	 531.3 KiB/s
+	 3.7 MiB/s
 done
-Bytes transferred = 6356632 (60fe98 hex)
+Bytes transferred = 6357376 (610180 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.3 MiB/s
+	 3.5 MiB/s
 done
 Bytes transferred = 66544 (103f0 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x60fe98 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x610180 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830133ef
@@ -54,7 +54,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-rc1-next-20171128-216631-g5bef298 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Nov 27 20:03:55 PST 2017
+Linux version 4.15.0-rc1-next-20171129-216899-gd127129 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Tue Nov 28 19:18:16 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -86,8 +86,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
-      .data : 0xc0f00000 - 0xc0fc2ba4   ( 779 kB)
-       .bss : 0xc0fc2ba4 - 0xc100d7ec   ( 300 kB)
+      .data : 0xc0f00000 - 0xc0fc2bb0   ( 779 kB)
+       .bss : 0xc0fc2bb0 - 0xc100d7ec   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -412,8 +412,8 @@
 as3722-ldo5: supplied by +1.8V_VDDIO
 as3722-ldo6: supplied by +3.3V_RUN
 as3722-ldo7: supplied by +1.8V_VDDIO
-+1.05V_RUN_CAM_REAR: Bringing 0uV into 1050000-1050000uV
 input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input1
++1.05V_RUN_CAM_REAR: Bringing 0uV into 1050000-1050000uV
 as3722-ldo9: supplied by +5V_SYS
 +3.3V_RUN_TOUCH: Bringing 0uV into 2800000-2800000uV
 as3722-ldo10: supplied by +5V_SYS
@@ -528,7 +528,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:41 UTC (946684961)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:27 UTC (946684947)
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-11-28 19:34:40.827925776 -0800
+++ /tmp/foob	2017-11-28 19:34:40.827925776 -0800
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -43,7 +43,7 @@
 	 #################################################################
 	 #################################################################
 	 ###############################
-	 2.9 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 8081920 (7b5200 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
@@ -52,7 +52,7 @@
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 1.6 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 67482 (1079a hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
@@ -64,7 +64,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-rc1-next-20171128-334872-g5bef2980adef (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Mon Nov 27 20:03:50 PST 2017
+Linux version 4.15.0-rc1-next-20171129-335140-gd127129e85a0 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Tue Nov 28 19:18:17 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -98,8 +98,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0e00000   (12256 kB)
       .init : 0xc1300000 - 0xc1500000   (2048 kB)
-      .data : 0xc1500000 - 0xc1690b70   (1603 kB)
-       .bss : 0xc1690b70 - 0xc16f5de4   ( 405 kB)
+      .data : 0xc1500000 - 0xc1690af0   (1603 kB)
+       .bss : 0xc1690af0 - 0xc16f5d64   ( 405 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -405,10 +405,10 @@
 +3.3V_PANEL: supplied by +3.3V_RUN
 +3.3V_LP0: supplied by +3.3V_SYS
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
-elan_i2c 1-0015: failed to read report data: -121
 +5V_HDMI_CON: supplied by +5V_SYS
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
  usb3-1: usb3-1 supply vbus not found, using dummy regulator
+elan_i2c 1-0015: failed to read report data: -121
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
 sdhci-tegra 700b0000.sdhci: allocated mmc-pwrseq
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2017-11-28 19:34:40.835925814 -0800
+++ /tmp/foob	2017-11-28 19:34:40.835925814 -0800
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -41,9 +41,9 @@
 	 #################################################################
 	 #################################################################
 	 ################################
-	 3.5 MiB/s
+	 3.6 MiB/s
 done
-Bytes transferred = 6192624 (5e7df0 hex)
+Bytes transferred = 6193480 (5e8148 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -54,7 +54,7 @@
 done
 Bytes transferred = 67482 (1079a hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5e7df0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5e8148 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013799
@@ -62,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-rc1-next-20171128-14923-g5bef298 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Nov 27 20:02:56 PST 2017
+Linux version 4.15.0-rc1-next-20171129-15191-gd127129 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Tue Nov 28 19:17:25 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -95,8 +95,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0ec2ba4   ( 779 kB)
-       .bss : 0xc0ec2ba4 - 0xc0f0d7ec   ( 300 kB)
+      .data : 0xc0e00000 - 0xc0ec2bb0   ( 779 kB)
+       .bss : 0xc0ec2bb0 - 0xc0f0d7ec   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -312,106 +312,90 @@
 kernel BUG at drivers/platform/chrome/cros_ec_proto.c:34!
 Internal error: Oops - BUG: 0 [#1] PREEMPT SMP ARM
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.15.0-rc1-next-20171128-14923-g5bef298 #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.15.0-rc1-next-20171129-15191-gd127129 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 task: ee890740 task.stack: ee8a6000
 PC is at cros_ec_prepare_tx+0x168/0x16c
 LR is at cros_ec_pkt_xfer_spi+0x18/0x474
-pc : [<c06ebf58>]    lr : [<c0559170>]    psr: 20000013
+pc : [<c06ec0cc>]    lr : [<c05592ac>]    psr: 20000013
 sp : ee8a7bb8  ip : 00000000  fp : 00000000
-r10: c0559158  r9 : 00000000  r8 : eefe7fb4
-r7 : ee2a6a40  r6 : ee2a6a40  r5 : ee2a6a40  r4 : ee28be50
+r10: c0559294  r9 : 00000000  r8 : eefe7fb4
+r7 : ee2a6a80  r6 : ee2a6a80  r5 : ee2a6a80  r4 : ee28be50
 r3 : 0000ffff  r2 : 00000000  r1 : 0000ffff  r0 : ee28be50
 Flags: nzCv  IRQs on  FIQs on  Mode SVC_32  ISA ARM  Segment none
 Control: 10c5387d  Table: 8000406a  DAC: 00000051
 Process swapper/0 (pid: 1, stack limit = 0xee8a6210)
 Stack: (0xee8a7bb8 to 0xee8a8000)
-7ba0:                                                       ee28be50 ee2a6a40
-7bc0: ee2a6ad0 c0559170 c0995d5c 00000004 ee8a7c3c ee8a7c3c ee2aac00 00000000
-7be0: c05a5e44 00000000 00000008 00000008 00000000 ee8a7bf4 ee8a7bf4 00000000
-7c00: ee8a7c00 ee8a7c00 ee2a6a90 ee2a69c0 00000008 00000000 00000000 00000000
-7c20: c0d91b50 c023c370 40002800 00000500 eef94b50 c023e7d4 002dc6c0 ee8a7bd0
-7c40: ee8a7bd0 00000000 c06ebd88 005c54c0 3b9aca00 000f4240 00000000 ee8a7c88
-7c60: 00000001 00000501 c0559158 c099a70c 00000000 ee2a6a00 a0000013 c06ebd88
-7c80: ee2a6a40 ee28be50 ee2a6a40 ee2aac00 ee2a6a40 eefe7fb4 00000000 c0559158
-7ca0: 00000000 c06ebce4 0000400b ee28be50 ee2a6a40 ee2aac00 0000000c eefe7fb4
-7cc0: c0eed23c c06ec100 00000000 ee28be50 ee2aac00 00000000 00000000 eefe7fb4
-7ce0: c0e6c768 c0558bfc ee8a7cf4 ee2aac00 00000000 00000001 ee2aac00 ee28be50
-7d00: ee2a6ad0 c0558fac 00000000 00000000 00000000 000007d0 ee2aac00 00000000
-7d20: c0e6c758 c0efe4d8 00000000 c05a6888 ee2aac00 c0efe4d4 00000000 c0534a48
-7d40: 00000000 ee8a7d78 c0534bf4 00000001 ee110000 00000000 c0efe4b0 c0533050
-7d60: ee9f1170 ee1d05b8 ee2aac00 ee2aac34 c0e73294 c0534770 ee2aac00 00000001
-7d80: 00000000 ee2aac00 ee2aac00 c0e73294 c0e04c48 c0533db4 ee2aac00 ee2aac08
-7da0: 00000000 c05323f0 40400000 00000000 ee110350 c05a9884 ee2aac00 e416be4f
-7dc0: 00000000 ee2aac00 ee110000 00000000 eea8be10 eefe8004 00000001 00000000
-7de0: 00000000 c05a66dc 002dc6c0 ee110000 eefe7fb4 00000000 ee2aac00 c05a867c
-7e00: 00000000 014000c0 00000000 c05a8068 00000063 002dc6c0 ee110000 ee2a6590
-7e20: eea8be00 eea8be10 40000000 00000153 00000000 c05a8ba0 eefe7d0c ee110348
-7e40: ee110000 eea8be00 eea8be10 c05a972c 00000000 ee110348 ffffffed eea8be10
-7e60: fffffdfb c0e73bf0 00000000 00000000 c0e73bf0 c0536360 c0536310 eea8be10
-7e80: c0efe4d4 00000000 c0efe4d8 c0534a48 eea8be10 c0e73bf0 eea8be44 00000000
-7ea0: 00000000 c0d6b83c c0d00594 c0534bbc 00000000 c0e73bf0 c0534b10 c05330ec
-7ec0: ee8cc55c eea14934 c0e73bf0 ee2a2e80 c0e6ad60 c0533f88 c0be0b5c c0e73bf0
-7ee0: 00000006 c0e73bf0 00000006 c0e04c48 c0d387ac c053573c c0535fb4 c0d8434c
-7f00: 00000006 c0101bbc 00000000 ee8a7f24 00000000 c0161d00 00000000 c0c7f290
-7f20: 0000010e efffcc45 c0c7f290 0000010e efffcc53 c013eb9c c0c7dd78 00000000
+7ba0:                                                       ee28be50 ee2a6a80
+7bc0: ee2a6b10 c05592ac c0995e7c 00000004 ee8a7c3c ee8a7c3c ee2aac00 00000000
+7be0: c05a5f80 00000000 00000008 00000008 00000000 ee8a7bf4 ee8a7bf4 00000000
+7c00: ee8a7c00 ee8a7c00 ee2a6ad0 ee2a6a00 00000008 00000000 00000000 00000000
+7c20: c0d91b50 c023c2e8 40002800 00000520 eef94b50 c023e74c 002dc6c0 ee8a7bd0
+7c40: ee8a7bd0 00000000 c06ebefc 005c54c0 3b9aca00 000f4240 00000000 ee8a7c88
+7c60: 00000001 00000501 c0559294 c099a82c 00000000 ee2a6a40 a0000013 c06ebefc
+7c80: ee2a6a80 ee28be50 ee2a6a80 ee2aac00 ee2a6a80 eefe7fb4 00000000 c0559294
+7ca0: 00000000 c06ebe58 0000400b ee28be50 ee2a6a80 ee2aac00 0000000c eefe7fb4
+7cc0: c0eed23c c06ec274 00000000 ee28be50 ee2aac00 00000000 00000000 eefe7fb4
+7ce0: c0e6c768 c0558d38 ee8a7cf4 ee2aac00 00000000 00000001 ee2aac00 ee28be50
+7d00: ee2a6b10 c05590e8 00000000 00000000 00000000 000007d0 ee2aac00 00000000
+7d20: c0e6c758 c0efe4d8 00000000 c05a69c4 ee2aac00 c0efe4d4 00000000 c0534afc
+7d40: 00000000 ee8a7d78 c0534ca8 00000001 ee124000 00000000 c0efe4b0 c0533104
+7d60: ee9f1170 ee1d05b8 ee2aac00 ee2aac34 c0e73294 c0534824 ee2aac00 00000001
+7d80: 00000000 ee2aac00 ee2aac00 c0e73294 c0e04c48 c0533e68 ee2aac00 ee2aac08
+7da0: 00000000 c05324a4 40400000 00000000 ee124350 c05a99c0 ee2aac00 e416be4f
+7dc0: 00000000 ee2aac00 ee124000 00000000 eea8be10 eefe8004 00000001 00000000
+7de0: 00000000 c05a6818 002dc6c0 ee124000 eefe7fb4 00000000 ee2aac00 c05a87b8
+7e00: 00000000 014000c0 00000000 c05a81a4 00000063 002dc6c0 ee124000 ee2a65d0
+7e20: eea8be00 eea8be10 40000000 00000153 00000000 c05a8cdc eefe7d0c ee124348
+7e40: ee124000 eea8be00 eea8be10 c05a9868 00000000 ee124348 ffffffed eea8be10
+7e60: fffffdfb c0e73bf0 00000000 00000000 c0e73bf0 c0536414 c05363c4 eea8be10
+7e80: c0efe4d4 00000000 c0efe4d8 c0534afc eea8be10 c0e73bf0 eea8be44 00000000
+7ea0: 00000000 c0d6b83c c0d00594 c0534c70 00000000 c0e73bf0 c0534bc4 c05331a0
+7ec0: ee8cc55c eea14934 c0e73bf0 ee2a2e80 c0e6ad60 c053403c c0be0c28 c0e73bf0
+7ee0: 00000006 c0e73bf0 00000006 c0e04c48 c0d38798 c05357f0 c0536068 c0d8434c
+7f00: 00000006 c0101bbc 00000000 ee8a7f24 00000000 c0161d00 00000000 c0c7f2a4
+7f20: 0000010e efffcc45 c0c7f2a4 0000010e efffcc53 c013eb9c c0c7dd8c 00000000
 7f40: 00000006 00000006 efffcc45 efffcc4a 00000000 e416be4f 00000006 c0d8434c
 7f60: 00000006 c0d6b830 c0ec2bc0 0000010e c0d00594 c0d00dd0 00000006 00000006
-7f80: 00000000 c0d00594 00000000 c099370c 00000000 00000000 00000000 00000000
-7fa0: 00000000 c0993714 00000000 c0107d88 00000000 00000000 00000000 00000000
+7f80: 00000000 c0d00594 00000000 c0993830 00000000 00000000 00000000 00000000
+7fa0: 00000000 c0993838 00000000 c0107d88 00000000 00000000 00000000 00000000
 7fc0: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
 7fe0: 00000000 00000000 00000000 00000000 00000013 00000000 00000000 00000000
-[<c06ebf58>] (cros_ec_prepare_tx) from [<c0559170>] (cros_ec_pkt_xfer_spi+0x18/0x474)
-[<c0559170>] (cros_ec_pkt_xfer_spi) from [<c06ebce4>] (send_command+0x20/0xd8)
-[<c06ebce4>] (send_command) from [<c06ec100>] (cros_ec_query_all+0x1a4/0x398)
-[<c06ec100>] (cros_ec_query_all) from [<c0558bfc>] (cros_ec_register+0x94/0x21c)
-[<c0558bfc>] (cros_ec_register) from [<c0558fac>] (cros_ec_spi_probe+0x108/0x150)
-[<c0558fac>] (cros_ec_spi_probe) from [<c05a6888>] (spi_drv_probe+0x7c/0xa8)
-[<c05a6888>] (spi_drv_probe) from [<c0534a48>] (driver_probe_device+0x250/0x318)
-[<c0534a48>] (driver_probe_device) from [<c0533050>] (bus_for_each_drv+0x44/0x8c)
-[<c0533050>] (bus_for_each_drv) from [<c0534770>] (__device_attach+0x9c/0x100)
-[<c0534770>] (__device_attach) from [<c0533db4>] (bus_probe_device+0x84/0x8c)
-[<c0533db4>] (bus_probe_device) from [<c05323f0>] (device_add+0x39c/0x54c)
-[<c05323f0>] (device_add) from [<c05a66dc>] (spi_add_device+0x88/0x130)
-[<c05a66dc>] (spi_add_device) from [<c05a867c>] (spi_register_controller+0x428/0x918)
-[<c05a867c>] (spi_register_controller) from [<c05a8ba0>] (devm_spi_register_controller+0x34/0x70)
-[<c05a8ba0>] (devm_spi_register_controller) from [<c05a972c>] (tegra_spi_probe+0x298/0x35c)
-[<c05a972c>] (tegra_spi_probe) from [<c0536360>] (platform_drv_probe+0x50/0xb0)
-[<c0536360>] (platform_drv_probe) from [<c0534a48>] (driver_probe_device+0x250/0x318)
-[<c0534a48>] (driver_probe_device) from [<c0534bbc>] (__driver_attach+0xac/0xb0)
-[<c0534bbc>] (__driver_attach) from [<c05330ec>] (bus_for_each_dev+0x54/0x88)
-[<c05330ec>] (bus_for_each_dev) from [<c0533f88>] (bus_add_driver+0xe4/0x1f0)
-[<c0533f88>] (bus_add_driver) from [<c053573c>] (driver_register+0x78/0xf4)
-[<c053573c>] (driver_register) from [<c0101bbc>] (do_one_initcall+0x48/0x18c)
+[<c06ec0cc>] (cros_ec_prepare_tx) from [<c05592ac>] (cros_ec_pkt_xfer_spi+0x18/0x474)
+[<c05592ac>] (cros_ec_pkt_xfer_spi) from [<c06ebe58>] (send_command+0x20/0xd8)
+[<c06ebe58>] (send_command) from [<c06ec274>] (cros_ec_query_all+0x1a4/0x398)
+[<c06ec274>] (cros_ec_query_all) from [<c0558d38>] (cros_ec_register+0x94/0x21c)
+[<c0558d38>] (cros_ec_register) from [<c05590e8>] (cros_ec_spi_probe+0x108/0x150)
+[<c05590e8>] (cros_ec_spi_probe) from [<c05a69c4>] (spi_drv_probe+0x7c/0xa8)
+[<c05a69c4>] (spi_drv_probe) from [<c0534afc>] (driver_probe_device+0x250/0x318)
+[<c0534afc>] (driver_probe_device) from [<c0533104>] (bus_for_each_drv+0x44/0x8c)
+[<c0533104>] (bus_for_each_drv) from [<c0534824>] (__device_attach+0x9c/0x100)
+[<c0534824>] (__device_attach) from [<c0533e68>] (bus_probe_device+0x84/0x8c)
+[<c0533e68>] (bus_probe_device) from [<c05324a4>] (device_add+0x39c/0x54c)
+[<c05324a4>] (device_add) from [<c05a6818>] (spi_add_device+0x88/0x130)
+[<c05a6818>] (spi_add_device) from [<c05a87b8>] (spi_register_controller+0x428/0x918)
+[<c05a87b8>] (spi_register_controller) from [<c05a8cdc>] (devm_spi_register_controller+0x34/0x70)
+[<c05a8cdc>] (devm_spi_register_controller) from [<c05a9868>] (tegra_spi_probe+0x298/0x35c)
+[<c05a9868>] (tegra_spi_probe) from [<c0536414>] (platform_drv_probe+0x50/0xb0)
+[<c0536414>] (platform_drv_probe) from [<c0534afc>] (driver_probe_device+0x250/0x318)
+[<c0534afc>] (driver_probe_device) from [<c0534c70>] (__driver_attach+0xac/0xb0)
+[<c0534c70>] (__driver_attach) from [<c05331a0>] (bus_for_each_dev+0x54/0x88)
+[<c05331a0>] (bus_for_each_dev) from [<c053403c>] (bus_add_driver+0xe4/0x1f0)
+[<c053403c>] (bus_add_driver) from [<c05357f0>] (driver_register+0x78/0xf4)
+[<c05357f0>] (driver_register) from [<c0101bbc>] (do_one_initcall+0x48/0x18c)
 [<c0101bbc>] (do_one_initcall) from [<c0d00dd0>] (kernel_init_freeable+0x10c/0x1d0)
-[<c0d00dd0>] (kernel_init_freeable) from [<c0993714>] (kernel_init+0x8/0x114)
-[<c0993714>] (kernel_init) from [<c0107d88>] (ret_from_fork+0x14/0x2c)
+[<c0d00dd0>] (kernel_init_freeable) from [<c0993838>] (kernel_init+0x8/0x114)
+[<c0993838>] (kernel_init) from [<c0107d88>] (ret_from_fork+0x14/0x2c)
 Code: e2800008 e8bd8070 e7f001f2 e7f001f2 (e7f001f2) 
----[ end trace 3ad3faf10fd6631d ]---
+---[ end trace 7677b8cbc447895a ]---
 Kernel panic - not syncing: Attempted to kill init! exitcode=0x0000000b
 
-CPU1: stopping
-CPU: 1 PID: 0 Comm: swapper/1 Tainted: G      D          4.15.0-rc1-next-20171128-14923-g5bef298 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c0111188>] (unwind_backtrace) from [<c010c188>] (show_stack+0x10/0x14)
-[<c010c188>] (show_stack) from [<c097fc1c>] (dump_stack+0x84/0xa4)
-[<c097fc1c>] (dump_stack) from [<c010f6e0>] (handle_IPI+0x18c/0x32c)
-[<c010f6e0>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
-[<c0101430>] (gic_handle_irq) from [<c010cd4c>] (__irq_svc+0x6c/0xa8)
-Exception stack(0xee8c1f70 to 0xee8c1fb8)
-1f60:                                     00000001 00000000 ee8c1fc8 c011a4a0
-1f80: 00000002 c0e04ce0 00000000 c0e04c6c 00000000 c0d8d278 00000000 c0e04cf0
-1fa0: c06d4e28 ee8c1fc0 c01087f8 c01087fc 60000013 ffffffff
-[<c010cd4c>] (__irq_svc) from [<c01087fc>] (arch_cpu_idle+0x30/0x3c)
-[<c01087fc>] (arch_cpu_idle) from [<c0162880>] (do_idle+0x140/0x204)
-[<c0162880>] (do_idle) from [<c0162bcc>] (cpu_startup_entry+0x18/0x1c)
-[<c0162bcc>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
 CPU2: stopping
-CPU: 2 PID: 0 Comm: swapper/2 Tainted: G      D          4.15.0-rc1-next-20171128-14923-g5bef298 #1
+CPU: 2 PID: 0 Comm: swapper/2 Tainted: G      D          4.15.0-rc1-next-20171129-15191-gd127129 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c0111188>] (unwind_backtrace) from [<c010c188>] (show_stack+0x10/0x14)
-[<c010c188>] (show_stack) from [<c097fc1c>] (dump_stack+0x84/0xa4)
-[<c097fc1c>] (dump_stack) from [<c010f6e0>] (handle_IPI+0x18c/0x32c)
+[<c010c188>] (show_stack) from [<c097fd1c>] (dump_stack+0x84/0xa4)
+[<c097fd1c>] (dump_stack) from [<c010f6e0>] (handle_IPI+0x18c/0x32c)
 [<c010f6e0>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cd4c>] (__irq_svc+0x6c/0xa8)
 Exception stack(0xee8e1f70 to 0xee8e1fb8)
@@ -419,24 +403,40 @@
 1f80: 00000004 c0e04ce0 00000000 c0e04c6c 00000000 c0d8d278 00000000 c0e04cf0
 1fa0: 00000000 ee8e1fc0 c01087f8 c01087fc 60000013 ffffffff
 [<c010cd4c>] (__irq_svc) from [<c01087fc>] (arch_cpu_idle+0x30/0x3c)
-[<c01087fc>] (arch_cpu_idle) from [<c0162880>] (do_idle+0x140/0x204)
-[<c0162880>] (do_idle) from [<c0162bcc>] (cpu_startup_entry+0x18/0x1c)
-[<c0162bcc>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
+[<c01087fc>] (arch_cpu_idle) from [<c01628a4>] (do_idle+0x140/0x204)
+[<c01628a4>] (do_idle) from [<c0162bf0>] (cpu_startup_entry+0x18/0x1c)
+[<c0162bf0>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
+CPU1: stopping
+CPU: 1 PID: 0 Comm: swapper/1 Tainted: G      D          4.15.0-rc1-next-20171129-15191-gd127129 #1
+Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
+[<c0111188>] (unwind_backtrace) from [<c010c188>] (show_stack+0x10/0x14)
+[<c010c188>] (show_stack) from [<c097fd1c>] (dump_stack+0x84/0xa4)
+[<c097fd1c>] (dump_stack) from [<c010f6e0>] (handle_IPI+0x18c/0x32c)
+[<c010f6e0>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
+[<c0101430>] (gic_handle_irq) from [<c010cd4c>] (__irq_svc+0x6c/0xa8)
+Exception stack(0xee8c1f70 to 0xee8c1fb8)
+1f60:                                     00000001 00000000 ee8c1fc8 c011a4a0
+1f80: 00000002 c0e04ce0 00000000 c0e04c6c 00000000 c0d8d278 00000000 c0e04cf0
+1fa0: c06d4f9c ee8c1fc0 c01087f8 c01087fc 60000013 ffffffff
+[<c010cd4c>] (__irq_svc) from [<c01087fc>] (arch_cpu_idle+0x30/0x3c)
+[<c01087fc>] (arch_cpu_idle) from [<c01628a4>] (do_idle+0x140/0x204)
+[<c01628a4>] (do_idle) from [<c0162bf0>] (cpu_startup_entry+0x18/0x1c)
+[<c0162bf0>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
 CPU3: stopping
-CPU: 3 PID: 0 Comm: swapper/3 Tainted: G      D          4.15.0-rc1-next-20171128-14923-g5bef298 #1
+CPU: 3 PID: 0 Comm: swapper/3 Tainted: G      D          4.15.0-rc1-next-20171129-15191-gd127129 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c0111188>] (unwind_backtrace) from [<c010c188>] (show_stack+0x10/0x14)
-[<c010c188>] (show_stack) from [<c097fc1c>] (dump_stack+0x84/0xa4)
-[<c097fc1c>] (dump_stack) from [<c010f6e0>] (handle_IPI+0x18c/0x32c)
+[<c010c188>] (show_stack) from [<c097fd1c>] (dump_stack+0x84/0xa4)
+[<c097fd1c>] (dump_stack) from [<c010f6e0>] (handle_IPI+0x18c/0x32c)
 [<c010f6e0>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cd4c>] (__irq_svc+0x6c/0xa8)
 Exception stack(0xee8e3f70 to 0xee8e3fb8)
 3f60:                                     00000001 00000000 ee8e3fc8 c011a4a0
 3f80: 00000008 c0e04ce0 00000000 c0e04c6c 00000000 c0d8d278 00000000 c0e04cf0
-3fa0: c0e12400 ee8e3fc0 c01087f8 c01087fc 60000013 ffffffff
+3fa0: c06d4f9c ee8e3fc0 c01087f8 c01087fc 60000013 ffffffff
 [<c010cd4c>] (__irq_svc) from [<c01087fc>] (arch_cpu_idle+0x30/0x3c)
-[<c01087fc>] (arch_cpu_idle) from [<c0162880>] (do_idle+0x140/0x204)
-[<c0162880>] (do_idle) from [<c0162bcc>] (cpu_startup_entry+0x18/0x1c)
-[<c0162bcc>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
+[<c01087fc>] (arch_cpu_idle) from [<c01628a4>] (do_idle+0x140/0x204)
+[<c01628a4>] (do_idle) from [<c0162bf0>] (cpu_startup_entry+0x18/0x1c)
+[<c0162bf0>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
 ---[ end Kernel panic - not syncing: Attempted to kill init! exitcode=0x0000000b
 
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig%tegra-fw_log.txt
--- /tmp/fooa	2017-11-28 19:34:40.843925850 -0800
+++ /tmp/foob	2017-11-28 19:34:40.843925850 -0800
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -41,20 +41,20 @@
 	 #################################################################
 	 #################################################################
 	 ############################################
-	 3.6 MiB/s
+	 3.5 MiB/s
 done
-Bytes transferred = 6356632 (60fe98 hex)
+Bytes transferred = 6357376 (610180 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.6 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 67482 (1079a hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x60fe98 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x610180 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013799
@@ -62,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-rc1-next-20171128-216631-g5bef298 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Nov 27 20:03:55 PST 2017
+Linux version 4.15.0-rc1-next-20171129-216899-gd127129 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Tue Nov 28 19:18:16 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -95,8 +95,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
-      .data : 0xc0f00000 - 0xc0fc2ba4   ( 779 kB)
-       .bss : 0xc0fc2ba4 - 0xc100d7ec   ( 300 kB)
+      .data : 0xc0f00000 - 0xc0fc2bb0   ( 779 kB)
+       .bss : 0xc0fc2bb0 - 0xc100d7ec   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -312,131 +312,131 @@
 kernel BUG at drivers/platform/chrome/cros_ec_proto.c:34!
 Internal error: Oops - BUG: 0 [#1] PREEMPT SMP ARM
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.15.0-rc1-next-20171128-216631-g5bef298 #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.15.0-rc1-next-20171129-216899-gd127129 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 task: ee890740 task.stack: ee8a6000
 PC is at cros_ec_prepare_tx+0x168/0x16c
 LR is at cros_ec_pkt_xfer_spi+0x18/0x474
-pc : [<c06ebf58>]    lr : [<c0559170>]    psr: 20000013
+pc : [<c06ec0cc>]    lr : [<c05592ac>]    psr: 20000013
 sp : ee8a7bb8  ip : 00000000  fp : 00000000
-r10: c0559158  r9 : 00000000  r8 : eefe7fb4
-r7 : ee2a6a80  r6 : ee2a6a80  r5 : ee2a6a80  r4 : ee28be50
-r3 : 0000ffff  r2 : 00000000  r1 : 0000ffff  r0 : ee28be50
+r10: c0559294  r9 : 00000000  r8 : eefe7fb4
+r7 : ee282e80  r6 : ee282e80  r5 : ee282e80  r4 : ee267e50
+r3 : 0000ffff  r2 : 00000000  r1 : 0000ffff  r0 : ee267e50
 Flags: nzCv  IRQs on  FIQs on  Mode SVC_32  ISA ARM  Segment none
 Control: 10c5387d  Table: 8000406a  DAC: 00000051
 Process swapper/0 (pid: 1, stack limit = 0xee8a6210)
 Stack: (0xee8a7bb8 to 0xee8a8000)
-7ba0:                                                       ee28be50 ee2a6a80
-7bc0: ee2a6b10 c0559170 c0995d5c 00000004 ee8a7c3c ee8a7c3c ee2aac00 00000000
-7be0: c05a5e44 00000000 00000008 00000008 00000000 ee8a7bf4 ee8a7bf4 00000000
-7c00: ee8a7c00 ee8a7c00 ee2a6ad0 ee2a6a00 00000008 00000000 00000000 00000000
-7c20: c0e91b50 c023c370 40002800 00000520 eef94b50 c023e7d4 002dc6c0 ee8a7bd0
-7c40: ee8a7bd0 00000000 c06ebd88 005c54c0 3b9aca00 000f4240 00000000 ee8a7c88
-7c60: 00000001 00000501 c0559158 c099a70c 00000000 ee2a6a40 a0000013 c06ebd88
-7c80: ee2a6a80 ee28be50 ee2a6a80 ee2aac00 ee2a6a80 eefe7fb4 00000000 c0559158
-7ca0: 00000000 c06ebce4 0000400b ee28be50 ee2a6a80 ee2aac00 0000000c eefe7fb4
-7cc0: c0fed23c c06ec100 00000000 ee28be50 ee2aac00 00000000 00000000 eefe7fb4
-7ce0: c0f6c768 c0558bfc ee8a7cf4 ee2aac00 00000000 00000001 ee2aac00 ee28be50
-7d00: ee2a6b10 c0558fac 00000000 00000000 00000000 000007d0 ee2aac00 00000000
-7d20: c0f6c758 c0ffe4d8 00000000 c05a6888 ee2aac00 c0ffe4d4 00000000 c0534a48
-7d40: 00000000 ee8a7d78 c0534bf4 00000001 ee110000 00000000 c0ffe4b0 c0533050
-7d60: ee9f1170 ee1d05b8 ee2aac00 ee2aac34 c0f73294 c0534770 ee2aac00 00000001
-7d80: 00000000 ee2aac00 ee2aac00 c0f73294 c0f04c48 c0533db4 ee2aac00 ee2aac08
-7da0: 00000000 c05323f0 40400000 00000000 ee110350 c05a9884 ee2aac00 e416be4f
-7dc0: 00000000 ee2aac00 ee110000 00000000 eea8be10 eefe8004 00000001 00000000
-7de0: 00000000 c05a66dc 002dc6c0 ee110000 eefe7fb4 00000000 ee2aac00 c05a867c
-7e00: 00000000 014000c0 00000000 c05a8068 00000063 002dc6c0 ee110000 ee2a65d0
-7e20: eea8be00 eea8be10 40000000 00000153 00000000 c05a8ba0 eefe7d0c ee110348
-7e40: ee110000 eea8be00 eea8be10 c05a972c 00000000 ee110348 ffffffed eea8be10
-7e60: fffffdfb c0f73bf0 00000000 00000000 c0f73bf0 c0536360 c0536310 eea8be10
-7e80: c0ffe4d4 00000000 c0ffe4d8 c0534a48 eea8be10 c0f73bf0 eea8be44 00000000
-7ea0: 00000000 c0e6b83c c0e00594 c0534bbc 00000000 c0f73bf0 c0534b10 c05330ec
-7ec0: ee8cc55c eea14934 c0f73bf0 ee2a2e80 c0f6ad60 c0533f88 c0c21fa0 c0f73bf0
-7ee0: 00000006 c0f73bf0 00000006 c0f04c48 c0e387ac c053573c c0535fb4 c0e8434c
-7f00: 00000006 c0101bbc 00000000 ee8a7f24 00000000 c0161d00 00000000 c0cc02a8
-7f20: 0000010e efffcc45 c0cc02a8 0000010e efffcc53 c013eb9c c0cbed90 00000000
+7ba0:                                                       ee267e50 ee282e80
+7bc0: ee282f10 c05592ac c0995e7c 00000004 ee8a7c3c ee8a7c3c ee29cc00 00000000
+7be0: c05a5f80 00000000 00000008 00000008 00000000 ee8a7bf4 ee8a7bf4 00000000
+7c00: ee8a7c00 ee8a7c00 ee282ed0 ee282dc0 00000008 00000000 00000000 00000000
+7c20: c0e91b50 c023c2e8 40002800 00000720 eef94b50 c023e74c 002dc6c0 ee8a7bd0
+7c40: ee8a7bd0 00000000 c06ebefc 005c5040 3b9aca00 000f4240 00000000 ee8a7c88
+7c60: 00000001 00000501 c0559294 c099a82c 00000000 ee282e40 a0000013 c06ebefc
+7c80: ee282e80 ee267e50 ee282e80 ee29cc00 ee282e80 eefe7fb4 00000000 c0559294
+7ca0: 00000000 c06ebe58 0000400b ee267e50 ee282e80 ee29cc00 0000000c eefe7fb4
+7cc0: c0fed23c c06ec274 00000000 ee267e50 ee29cc00 00000000 00000000 eefe7fb4
+7ce0: c0f6c768 c0558d38 ee8a7cf4 ee29cc00 00000000 00000001 ee29cc00 ee267e50
+7d00: ee282f10 c05590e8 00000000 00000000 00000000 000007d0 ee29cc00 00000000
+7d20: c0f6c758 c0ffe4d8 00000000 c05a69c4 ee29cc00 c0ffe4d4 00000000 c0534afc
+7d40: 00000000 ee8a7d78 c0534ca8 00000001 ee03c000 00000000 c0ffe4b0 c0533104
+7d60: ee9f1170 ee1ac5b8 ee29cc00 ee29cc34 c0f73294 c0534824 ee29cc00 00000001
+7d80: 00000000 ee29cc00 ee29cc00 c0f73294 c0f04c48 c0533e68 ee29cc00 ee29cc08
+7da0: 00000000 c05324a4 40400000 00000000 ee03c350 c05a99c0 ee29cc00 e416be4f
+7dc0: 00000000 ee29cc00 ee03c000 00000000 eea8be10 eefe8004 00000001 00000000
+7de0: 00000000 c05a6818 002dc6c0 ee03c000 eefe7fb4 00000000 ee29cc00 c05a87b8
+7e00: 00000000 014000c0 00000000 c05a81a4 00000063 002dc6c0 ee03c000 ee2825d0
+7e20: eea8be00 eea8be10 40000000 00000153 00000000 c05a8cdc eefe7d0c ee03c348
+7e40: ee03c000 eea8be00 eea8be10 c05a9868 00000000 ee03c348 ffffffed eea8be10
+7e60: fffffdfb c0f73bf0 00000000 00000000 c0f73bf0 c0536414 c05363c4 eea8be10
+7e80: c0ffe4d4 00000000 c0ffe4d8 c0534afc eea8be10 c0f73bf0 eea8be44 00000000
+7ea0: 00000000 c0e6b83c c0e00594 c0534c70 00000000 c0f73bf0 c0534bc4 c05331a0
+7ec0: ee8cc55c eea14934 c0f73bf0 ee27ee80 c0f6ad60 c053403c c0c220ac c0f73bf0
+7ee0: 00000006 c0f73bf0 00000006 c0f04c48 c0e38798 c05357f0 c0536068 c0e8434c
+7f00: 00000006 c0101bbc 00000000 ee8a7f24 00000000 c0161d00 00000000 c0cc02bc
+7f20: 0000010e efffcc45 c0cc02bc 0000010e efffcc53 c013eb9c c0cbeda4 00000000
 7f40: 00000006 00000006 efffcc45 efffcc4a 00000000 e416be4f 00000006 c0e8434c
 7f60: 00000006 c0e6b830 c0fc2bc0 0000010e c0e00594 c0e00dd0 00000006 00000006
-7f80: 00000000 c0e00594 00000000 c099370c 00000000 00000000 00000000 00000000
-7fa0: 00000000 c0993714 00000000 c0107d88 00000000 00000000 00000000 00000000
+7f80: 00000000 c0e00594 00000000 c0993830 00000000 00000000 00000000 00000000
+7fa0: 00000000 c0993838 00000000 c0107d88 00000000 00000000 00000000 00000000
 7fc0: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
 7fe0: 00000000 00000000 00000000 00000000 00000013 00000000 00000000 00000000
-[<c06ebf58>] (cros_ec_prepare_tx) from [<c0559170>] (cros_ec_pkt_xfer_spi+0x18/0x474)
-[<c0559170>] (cros_ec_pkt_xfer_spi) from [<c06ebce4>] (send_command+0x20/0xd8)
-[<c06ebce4>] (send_command) from [<c06ec100>] (cros_ec_query_all+0x1a4/0x398)
-[<c06ec100>] (cros_ec_query_all) from [<c0558bfc>] (cros_ec_register+0x94/0x21c)
-[<c0558bfc>] (cros_ec_register) from [<c0558fac>] (cros_ec_spi_probe+0x108/0x150)
-[<c0558fac>] (cros_ec_spi_probe) from [<c05a6888>] (spi_drv_probe+0x7c/0xa8)
-[<c05a6888>] (spi_drv_probe) from [<c0534a48>] (driver_probe_device+0x250/0x318)
-[<c0534a48>] (driver_probe_device) from [<c0533050>] (bus_for_each_drv+0x44/0x8c)
-[<c0533050>] (bus_for_each_drv) from [<c0534770>] (__device_attach+0x9c/0x100)
-[<c0534770>] (__device_attach) from [<c0533db4>] (bus_probe_device+0x84/0x8c)
-[<c0533db4>] (bus_probe_device) from [<c05323f0>] (device_add+0x39c/0x54c)
-[<c05323f0>] (device_add) from [<c05a66dc>] (spi_add_device+0x88/0x130)
-[<c05a66dc>] (spi_add_device) from [<c05a867c>] (spi_register_controller+0x428/0x918)
-[<c05a867c>] (spi_register_controller) from [<c05a8ba0>] (devm_spi_register_controller+0x34/0x70)
-[<c05a8ba0>] (devm_spi_register_controller) from [<c05a972c>] (tegra_spi_probe+0x298/0x35c)
-[<c05a972c>] (tegra_spi_probe) from [<c0536360>] (platform_drv_probe+0x50/0xb0)
-[<c0536360>] (platform_drv_probe) from [<c0534a48>] (driver_probe_device+0x250/0x318)
-[<c0534a48>] (driver_probe_device) from [<c0534bbc>] (__driver_attach+0xac/0xb0)
-[<c0534bbc>] (__driver_attach) from [<c05330ec>] (bus_for_each_dev+0x54/0x88)
-[<c05330ec>] (bus_for_each_dev) from [<c0533f88>] (bus_add_driver+0xe4/0x1f0)
-[<c0533f88>] (bus_add_driver) from [<c053573c>] (driver_register+0x78/0xf4)
-[<c053573c>] (driver_register) from [<c0101bbc>] (do_one_initcall+0x48/0x18c)
+[<c06ec0cc>] (cros_ec_prepare_tx) from [<c05592ac>] (cros_ec_pkt_xfer_spi+0x18/0x474)
+[<c05592ac>] (cros_ec_pkt_xfer_spi) from [<c06ebe58>] (send_command+0x20/0xd8)
+[<c06ebe58>] (send_command) from [<c06ec274>] (cros_ec_query_all+0x1a4/0x398)
+[<c06ec274>] (cros_ec_query_all) from [<c0558d38>] (cros_ec_register+0x94/0x21c)
+[<c0558d38>] (cros_ec_register) from [<c05590e8>] (cros_ec_spi_probe+0x108/0x150)
+[<c05590e8>] (cros_ec_spi_probe) from [<c05a69c4>] (spi_drv_probe+0x7c/0xa8)
+[<c05a69c4>] (spi_drv_probe) from [<c0534afc>] (driver_probe_device+0x250/0x318)
+[<c0534afc>] (driver_probe_device) from [<c0533104>] (bus_for_each_drv+0x44/0x8c)
+[<c0533104>] (bus_for_each_drv) from [<c0534824>] (__device_attach+0x9c/0x100)
+[<c0534824>] (__device_attach) from [<c0533e68>] (bus_probe_device+0x84/0x8c)
+[<c0533e68>] (bus_probe_device) from [<c05324a4>] (device_add+0x39c/0x54c)
+[<c05324a4>] (device_add) from [<c05a6818>] (spi_add_device+0x88/0x130)
+[<c05a6818>] (spi_add_device) from [<c05a87b8>] (spi_register_controller+0x428/0x918)
+[<c05a87b8>] (spi_register_controller) from [<c05a8cdc>] (devm_spi_register_controller+0x34/0x70)
+[<c05a8cdc>] (devm_spi_register_controller) from [<c05a9868>] (tegra_spi_probe+0x298/0x35c)
+[<c05a9868>] (tegra_spi_probe) from [<c0536414>] (platform_drv_probe+0x50/0xb0)
+[<c0536414>] (platform_drv_probe) from [<c0534afc>] (driver_probe_device+0x250/0x318)
+[<c0534afc>] (driver_probe_device) from [<c0534c70>] (__driver_attach+0xac/0xb0)
+[<c0534c70>] (__driver_attach) from [<c05331a0>] (bus_for_each_dev+0x54/0x88)
+[<c05331a0>] (bus_for_each_dev) from [<c053403c>] (bus_add_driver+0xe4/0x1f0)
+[<c053403c>] (bus_add_driver) from [<c05357f0>] (driver_register+0x78/0xf4)
+[<c05357f0>] (driver_register) from [<c0101bbc>] (do_one_initcall+0x48/0x18c)
 [<c0101bbc>] (do_one_initcall) from [<c0e00dd0>] (kernel_init_freeable+0x10c/0x1d0)
-[<c0e00dd0>] (kernel_init_freeable) from [<c0993714>] (kernel_init+0x8/0x114)
-[<c0993714>] (kernel_init) from [<c0107d88>] (ret_from_fork+0x14/0x2c)
+[<c0e00dd0>] (kernel_init_freeable) from [<c0993838>] (kernel_init+0x8/0x114)
+[<c0993838>] (kernel_init) from [<c0107d88>] (ret_from_fork+0x14/0x2c)
 Code: e2800008 e8bd8070 e7f001f2 e7f001f2 (e7f001f2) 
----[ end trace 4c7a5227979d4c21 ]---
+---[ end trace 1464629f06795d06 ]---
 Kernel panic - not syncing: Attempted to kill init! exitcode=0x0000000b
 
-CPU1: stopping
-CPU: 1 PID: 0 Comm: swapper/1 Tainted: G      D          4.15.0-rc1-next-20171128-216631-g5bef298 #1
+CPU3: stopping
+CPU: 3 PID: 0 Comm: swapper/3 Tainted: G      D          4.15.0-rc1-next-20171129-216899-gd127129 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c0111188>] (unwind_backtrace) from [<c010c188>] (show_stack+0x10/0x14)
-[<c010c188>] (show_stack) from [<c097fc1c>] (dump_stack+0x84/0xa4)
-[<c097fc1c>] (dump_stack) from [<c010f6e0>] (handle_IPI+0x18c/0x32c)
+[<c010c188>] (show_stack) from [<c097fd1c>] (dump_stack+0x84/0xa4)
+[<c097fd1c>] (dump_stack) from [<c010f6e0>] (handle_IPI+0x18c/0x32c)
 [<c010f6e0>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cd4c>] (__irq_svc+0x6c/0xa8)
-Exception stack(0xee8c1f70 to 0xee8c1fb8)
-1f60:                                     00000001 00000000 ee8c1fc8 c011a4a0
-1f80: 00000002 c0f04ce0 00000000 c0f04c6c 00000000 c0e8d278 00000000 c0f04cf0
-1fa0: c06d4e28 ee8c1fc0 c01087f8 c01087fc 60000013 ffffffff
+Exception stack(0xee8e3f70 to 0xee8e3fb8)
+3f60:                                     00000001 00000000 ee8e3fc8 c011a4a0
+3f80: 00000008 c0f04ce0 00000000 c0f04c6c 00000000 c0e8d278 00000000 c0f04cf0
+3fa0: 00000000 ee8e3fc0 c01087f8 c01087fc 60000013 ffffffff
 [<c010cd4c>] (__irq_svc) from [<c01087fc>] (arch_cpu_idle+0x30/0x3c)
-[<c01087fc>] (arch_cpu_idle) from [<c0162880>] (do_idle+0x140/0x204)
-[<c0162880>] (do_idle) from [<c0162bcc>] (cpu_startup_entry+0x18/0x1c)
-[<c0162bcc>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
+[<c01087fc>] (arch_cpu_idle) from [<c01628a4>] (do_idle+0x140/0x204)
+[<c01628a4>] (do_idle) from [<c0162bf0>] (cpu_startup_entry+0x18/0x1c)
+[<c0162bf0>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
 CPU2: stopping
-CPU: 2 PID: 0 Comm: swapper/2 Tainted: G      D          4.15.0-rc1-next-20171128-216631-g5bef298 #1
+CPU: 2 PID: 0 Comm: swapper/2 Tainted: G      D          4.15.0-rc1-next-20171129-216899-gd127129 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c0111188>] (unwind_backtrace) from [<c010c188>] (show_stack+0x10/0x14)
-[<c010c188>] (show_stack) from [<c097fc1c>] (dump_stack+0x84/0xa4)
-[<c097fc1c>] (dump_stack) from [<c010f6e0>] (handle_IPI+0x18c/0x32c)
+[<c010c188>] (show_stack) from [<c097fd1c>] (dump_stack+0x84/0xa4)
+[<c097fd1c>] (dump_stack) from [<c010f6e0>] (handle_IPI+0x18c/0x32c)
 [<c010f6e0>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cd4c>] (__irq_svc+0x6c/0xa8)
 Exception stack(0xee8e1f70 to 0xee8e1fb8)
 1f60:                                     00000001 00000000 ee8e1fc8 c011a4a0
 1f80: 00000004 c0f04ce0 00000000 c0f04c6c 00000000 c0e8d278 00000000 c0f04cf0
-1fa0: 00000000 ee8e1fc0 c01087f8 c01087fc 60000013 ffffffff
+1fa0: c0f12400 ee8e1fc0 c01087f8 c01087fc 60000013 ffffffff
 [<c010cd4c>] (__irq_svc) from [<c01087fc>] (arch_cpu_idle+0x30/0x3c)
-[<c01087fc>] (arch_cpu_idle) from [<c0162880>] (do_idle+0x140/0x204)
-[<c0162880>] (do_idle) from [<c0162bcc>] (cpu_startup_entry+0x18/0x1c)
-[<c0162bcc>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
-CPU3: stopping
-CPU: 3 PID: 0 Comm: swapper/3 Tainted: G      D          4.15.0-rc1-next-20171128-216631-g5bef298 #1
+[<c01087fc>] (arch_cpu_idle) from [<c01628a4>] (do_idle+0x140/0x204)
+[<c01628a4>] (do_idle) from [<c0162bf0>] (cpu_startup_entry+0x18/0x1c)
+[<c0162bf0>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
+CPU1: stopping
+CPU: 1 PID: 0 Comm: swapper/1 Tainted: G      D          4.15.0-rc1-next-20171129-216899-gd127129 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c0111188>] (unwind_backtrace) from [<c010c188>] (show_stack+0x10/0x14)
-[<c010c188>] (show_stack) from [<c097fc1c>] (dump_stack+0x84/0xa4)
-[<c097fc1c>] (dump_stack) from [<c010f6e0>] (handle_IPI+0x18c/0x32c)
+[<c010c188>] (show_stack) from [<c097fd1c>] (dump_stack+0x84/0xa4)
+[<c097fd1c>] (dump_stack) from [<c010f6e0>] (handle_IPI+0x18c/0x32c)
 [<c010f6e0>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cd4c>] (__irq_svc+0x6c/0xa8)
-Exception stack(0xee8e3f70 to 0xee8e3fb8)
-3f60:                                     00000001 00000000 ee8e3fc8 c011a4a0
-3f80: 00000008 c0f04ce0 00000000 c0f04c6c 00000000 c0e8d278 00000000 c0f04cf0
-3fa0: c06d4e28 ee8e3fc0 c01087f8 c01087fc 60000013 ffffffff
+Exception stack(0xee8c1f70 to 0xee8c1fb8)
+1f60:                                     00000001 00000000 ee8c1fc8 c011a4a0
+1f80: 00000002 c0f04ce0 00000000 c0f04c6c 00000000 c0e8d278 00000000 c0f04cf0
+1fa0: 00000001 ee8c1fc0 c01087f8 c01087fc 60000013 ffffffff
 [<c010cd4c>] (__irq_svc) from [<c01087fc>] (arch_cpu_idle+0x30/0x3c)
-[<c01087fc>] (arch_cpu_idle) from [<c0162880>] (do_idle+0x140/0x204)
-[<c0162880>] (do_idle) from [<c0162bcc>] (cpu_startup_entry+0x18/0x1c)
-[<c0162bcc>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
+[<c01087fc>] (arch_cpu_idle) from [<c01628a4>] (do_idle+0x140/0x204)
+[<c01628a4>] (do_idle) from [<c0162bf0>] (cpu_startup_entry+0x18/0x1c)
+[<c0162bf0>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
 ---[ end Kernel panic - not syncing: Attempted to kill init! exitcode=0x0000000b
 
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2017-11-28 19:34:40.851925885 -0800
+++ /tmp/foob	2017-11-28 19:34:40.851925885 -0800
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1801528 us
+Bootloader-Cpu Init at (time stamp): 1801221 us
 
 Pinmux changes applied in kernel way
 
@@ -59,25 +59,25 @@
 The proc BoardInfo: 0x0176:0x0001:0x01:0x43:0x00
 FastbootAddAtag: setting ATAG (0)
 
-Board Id Read Time: 4 us
+Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1801501  us
-Bootloader (AosInit) time: 733511 us
-Bootloader Main Init time: 180173 us
-BlockDevMgrInit time: 157543 us
-PartMgrInit time: 9 us
-StorMgrInit time: 30 us
+Miscellaneous avp time: 1801195  us
+Bootloader (AosInit) time: 733543 us
+Bootloader Main Init time: 180236 us
+BlockDevMgrInit time: 157547 us
+PartMgrInit time: 8 us
+StorMgrInit time: 31 us
 FsMgrInit time: 2 us
-Partition Table load: 77118 us 
-Display init time: 574566 us
-Kernel read time: 1536 msec
+Partition Table load: 77119 us 
+Display init time: 574666 us
+Kernel read time: 1535 msec
 Load Kernel Size:   20342784 Bytes
-Bl Kernel ReadSpeed: 13244000 Bytes/sec
-Pre-Kernel Jump time: 4306815 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 9461062 us
+Bl Kernel ReadSpeed: 13252000 Bytes/sec
+Pre-Kernel Jump time: 4307365 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 9461328 us
 Booting Linux on physical CPU 0x0000000000 [0x4e0f0000]
-Linux version 4.15.0-rc1-next-20171128-334872-g5bef2980adef (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Mon Nov 27 20:09:53 PST 2017
+Linux version 4.15.0-rc1-next-20171129-335140-gd127129e85a0 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Tue Nov 28 19:24:19 PST 2017
 Machine model: NVIDIA Tegra132 Norrin
 Memory limited to 1734MB
 efi: Getting EFI parameters from FDT:
@@ -266,23 +266,22 @@
 usbhid: USB HID core driver
 mmc0: new high speed MMC card at address 0001
 NET: Registered protocol family 17
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB, chardev (240:0)
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 9pnet: Installing 9P2000 support
 Key type dns_resolver registered
 registered taskstats version 1
 Loading compiled-in X.509 certificates
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB, chardev (240:0)
 +VDD_LED: supplied by +VDD_MUX
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 +5V_USB_HS: supplied by +5V_SYS
 +5V_USB_SS: supplied by +5V_SYS
 +5V_HDMI_CON: supplied by +5V_SYS
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-sdhci-tegra 700b0400.sdhci: Got CD GPIO
 hctosys: unable to open rtc device (rtc0)
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +VDD_LED: disabling
@@ -292,8 +291,9 @@
 +5V_HDMI_CON: disabling
 ALSA device list:
   No soundcards found.
-sdhci-tegra 700b0400.sdhci: Got WP GPIO
 Freeing unused kernel memory: 1152K
+sdhci-tegra 700b0400.sdhci: Got CD GPIO
+sdhci-tegra 700b0400.sdhci: Got WP GPIO
 
 System initialization...
 
Diff for ./tegra186-p2771-0000/tegra186-p2771-0000/defconfig%jetson-tx2_log.txt
Binary files /tmp/fooa and /tmp/foob differ
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-11-28 19:34:40.867925958 -0800
+++ /tmp/foob	2017-11-28 19:34:40.871925976 -0800
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -48,7 +48,7 @@
 	 #################################################################
 	 #################################################################
 	 ###############################
-	 3.6 MiB/s
+	 3.5 MiB/s
 done
 Bytes transferred = 8081920 (7b5200 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
@@ -69,7 +69,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-rc1-next-20171128-334872-g5bef2980adef (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Mon Nov 27 20:03:50 PST 2017
+Linux version 4.15.0-rc1-next-20171129-335140-gd127129e85a0 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Tue Nov 28 19:18:17 PST 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -101,8 +101,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0e00000   (12256 kB)
       .init : 0xc1300000 - 0xc1500000   (2048 kB)
-      .data : 0xc1500000 - 0xc1690b70   (1603 kB)
-       .bss : 0xc1690b70 - 0xc16f5de4   ( 405 kB)
+      .data : 0xc1500000 - 0xc1690af0   (1603 kB)
+       .bss : 0xc1690af0 - 0xc16f5d64   ( 405 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -341,11 +341,11 @@
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
+tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
- mmcblk1: p1 p2
-tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
+ mmcblk1: p1 p2
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
 rtc-em3027 2-0056: hctosys: unable to read the hardware clock
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2017-11-28 19:34:40.875925994 -0800
+++ /tmp/foob	2017-11-28 19:34:40.875925994 -0800
@@ -48,18 +48,18 @@
 	 ################################
 	 4.1 MiB/s
 done
-Bytes transferred = 6192624 (5e7df0 hex)
+Bytes transferred = 6193480 (5e8148 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 4 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21160 (52a8 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x5e7df0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x5e8148 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102a7
@@ -67,7 +67,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-rc1-next-20171128-14923-g5bef298 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Nov 27 20:02:56 PST 2017
+Linux version 4.15.0-rc1-next-20171129-15191-gd127129 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Tue Nov 28 19:17:25 PST 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -98,8 +98,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0ec2ba4   ( 779 kB)
-       .bss : 0xc0ec2ba4 - 0xc0f0d7ec   ( 300 kB)
+      .data : 0xc0e00000 - 0xc0ec2bb0   ( 779 kB)
+       .bss : 0xc0ec2bb0 - 0xc0f0d7ec   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
 	Tasks RCU enabled.
@@ -322,33 +322,33 @@
 Bluetooth: RFCOMM ver 1.11
 Bluetooth: BNEP (Ethernet Emulation) ver 1.3
 Bluetooth: BNEP socket layer initialized
-mmc1: new high speed SD card at address b368
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
- mmcblk1: p1 p2
 Bluetooth: HIDP (Human Interface Emulation) ver 1.2
 Bluetooth: HIDP socket layer initialized
 Key type dns_resolver registered
 Registering SWP/SWPB emulation handler
 Loading compiled-in X.509 certificates
+mmc1: new high speed SD card at address b368
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
+ mmcblk1: p1 p2
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
 rtc-em3027 2-0056: hctosys: unable to read the hardware clock
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
-hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
 ALSA device list:
   #0: tegra-trimslice
-Waiting for root device PARTUUID=11eb23c1-01...
 cfg80211: failed to load regulatory.db
+Waiting for root device PARTUUID=11eb23c1-01...
+hub 1-1:1.0: USB hub found
+hub 1-1:1.0: 4 ports detected
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig%jetson-tx1_log.txt
--- /tmp/fooa	2017-11-28 19:34:40.883926030 -0800
+++ /tmp/foob	2017-11-28 19:34:40.883926030 -0800
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 777085 us
-Basic modules init took 348555 us
-Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14368 us
-Carveout took 252664 us
-CPU initialization took 92463 us
-Total time taken by TegraBoot 708050 us
+Tegraboot started after 763811 us
+Basic modules init took 349376 us
+Sec Bootdevice Read Time = 203 ms, Read Size = 8459 KB
+Next stage binary read took 14343 us
+Carveout took 252667 us
+CPU initialization took 92453 us
+Total time taken by TegraBoot 708839 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.615] LPDDR4 Training: Number of tables = 10
+64b[0001.603] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig%jetson-tx1
-17039872 bytes read in 860 ms (18.9 MiB/s)
+17039872 bytes read in 871 ms (18.7 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
 44764 bytes read in 453 ms (95.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 494 ms (3.1 MiB/s)
+1627787 bytes read in 496 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0000000000 [0x411fd071]
-Linux version 4.15.0-rc1-next-20171128-216631-g5bef298 (jonathanh@sellafield) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Mon Nov 27 20:07:17 PST 2017
+Linux version 4.15.0-rc1-next-20171129-216899-gd127129 (jonathanh@sellafield) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Tue Nov 28 19:21:59 PST 2017
 Machine model: NVIDIA Tegra210 P2371 (P2530/P2595) reference design
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -171,7 +171,7 @@
 Policy zone: Normal
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffff80007a000000-ffff80007dffffff]
-Memory: 2982108K/3131392K available (9724K kernel code, 1152K rwdata, 4592K rodata, 1152K init, 397K bss, 132900K reserved, 16384K cma-reserved)
+Memory: 2982108K/3131392K available (9724K kernel code, 1152K rwdata, 4596K rodata, 1152K init, 397K bss, 132900K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
@@ -267,7 +267,7 @@
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 PCI: CLS 0 bytes, default 128
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 809eb000
+kvm [1]: IDMAP page: 809ec000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: vgic-v2@50044000
 kvm [1]: vgic interrupt IRQ1
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2017-11-28 19:34:40.891926067 -0800
+++ /tmp/foob	2017-11-28 19:34:40.891926067 -0800
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 731204 us
-Basic modules init took 348297 us
+Tegraboot started after 758038 us
+Basic modules init took 348474 us
 Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
 Next stage binary read took 14374 us
-Carveout took 252687 us
-CPU initialization took 92454 us
-Total time taken by TegraBoot 707812 us
+Carveout took 252667 us
+CPU initialization took 92457 us
+Total time taken by TegraBoot 707972 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.570] LPDDR4 Training: Number of tables = 10
+64b[0001.596] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-16843264 bytes read in 849 ms (18.9 MiB/s)
+16843264 bytes read in 835 ms (19.2 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-44764 bytes read in 454 ms (95.7 KiB/s)
+44764 bytes read in 453 ms (95.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 495 ms (3.1 MiB/s)
+1627787 bytes read in 493 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0000000000 [0x411fd071]
-Linux version 4.15.0-rc1-next-20171128-334872-g5bef2980adef (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Mon Nov 27 20:09:53 PST 2017
+Linux version 4.15.0-rc1-next-20171129-335140-gd127129e85a0 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Tue Nov 28 19:24:19 PST 2017
 Machine model: NVIDIA Tegra210 P2371 (P2530/P2595) reference design
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
Diff for ./tegra210-p2371-2180/tegra210-p2371-2180/defconfig%jetson-tx1_log.txt
Binary files /tmp/fooa and /tmp/foob differ
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2017-11-28 19:34:40.907926139 -0800
+++ /tmp/foob	2017-11-28 19:34:40.907926139 -0800
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0000000000 [0x411fd071]
-Linux version 4.15.0-rc1-next-20171128-334872-g5bef2980adef (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Mon Nov 27 20:09:53 PST 2017
+Linux version 4.15.0-rc1-next-20171129-335140-gd127129e85a0 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Tue Nov 28 19:24:19 PST 2017
 Machine model: Google Pixel C
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
@@ -265,7 +265,7 @@
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB, chardev (240:0)
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
-max77686-rtc max77620-rtc: setting system clock to 2001-09-08 09:40:01 UTC (999942001)
+max77686-rtc max77620-rtc: setting system clock to 2001-09-09 08:54:23 UTC (1000025663)
 PP1200_AVDD: disabling
 PP1200_RCAM: disabling
 PP2800L_RCAM: disabling
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-11-28 19:34:40.915926176 -0800
+++ /tmp/foob	2017-11-28 19:34:40.915926176 -0800
@@ -49,7 +49,7 @@
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.3 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 51470 (c90e hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-rc1-next-20171128-334872-g5bef2980adef (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Mon Nov 27 20:03:50 PST 2017
+Linux version 4.15.0-rc1-next-20171129-335140-gd127129e85a0 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Tue Nov 28 19:18:17 PST 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -93,8 +93,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0e00000   (12256 kB)
       .init : 0xc1300000 - 0xc1500000   (2048 kB)
-      .data : 0xc1500000 - 0xc1690b70   (1603 kB)
-       .bss : 0xc1690b70 - 0xc16f5de4   ( 405 kB)
+      .data : 0xc1500000 - 0xc1690af0   (1603 kB)
+       .bss : 0xc1690af0 - 0xc16f5d64   ( 405 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -366,16 +366,6 @@
 can: netlink gateway (rev 20170425) max_hops=1
 Key type dns_resolver registered
 ThumbEE CPU extension supported.
-Registering SWP/SWPB emulation handler
-Loading compiled-in X.509 certificates
-vdd_ddr: supplied by vdd_5v_in
-vdd_5v_sata: supplied by vdd_5v_in
-usb1_vbus: supplied by vdd_5v_in
-usb3_vbus: supplied by vdd_5v_in
-sys_3v3,vdd_3v3_alw: supplied by vdd_5v_in
-sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
-mmc0: new ultra high speed DDR50 SDHC card at address aaaa
-mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
@@ -383,7 +373,17 @@
 mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB, chardev (237:0)
 Primary GPT is invalid, using alternate GPT.
  mmcblk1: p1 p2 p3 p4 p5
+mmc0: new ultra high speed DDR50 SDHC card at address aaaa
+mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
+Registering SWP/SWPB emulation handler
+Loading compiled-in X.509 certificates
  mmcblk0: p1 p2 p3
+vdd_ddr: supplied by vdd_5v_in
+vdd_5v_sata: supplied by vdd_5v_in
+usb1_vbus: supplied by vdd_5v_in
+usb3_vbus: supplied by vdd_5v_in
+sys_3v3,vdd_3v3_alw: supplied by vdd_5v_in
+sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
 +VDD_5V_HDMI: supplied by sys_3v3,vdd_3v3_alw
 tegra-pcie 3000.pcie: 2x3 configuration
 tegra-pcie 3000.pcie: probing port 0, using 2 lanes
@@ -458,9 +458,9 @@
 EXT4-fs (mmcblk1p1): warning: mounting fs with errors, running e2fsck is recommended
 EXT4-fs (mmcblk1p1): recovery complete
 EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:17.
+VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.424096] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.423703] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2017-11-28 19:34:40.923926212 -0800
+++ /tmp/foob	2017-11-28 19:34:40.923926212 -0800
@@ -40,7 +40,7 @@
 	 ################################
 	 3.3 MiB/s
 done
-Bytes transferred = 6192624 (5e7df0 hex)
+Bytes transferred = 6193480 (5e8148 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -51,7 +51,7 @@
 done
 Bytes transferred = 51470 (c90e hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5e7df0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5e8148 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90d ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-rc1-next-20171128-14923-g5bef298 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Nov 27 20:02:56 PST 2017
+Linux version 4.15.0-rc1-next-20171129-15191-gd127129 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Tue Nov 28 19:17:25 PST 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -90,8 +90,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0ec2ba4   ( 779 kB)
-       .bss : 0xc0ec2ba4 - 0xc0f0d7ec   ( 300 kB)
+      .data : 0xc0e00000 - 0xc0ec2bb0   ( 779 kB)
+       .bss : 0xc0ec2bb0 - 0xc0f0d7ec   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -464,4 +464,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.677643] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.677650] usb 2-1: new high-speed USB device number 2 using tegra-ehci
