Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2017-07-20 21:43:04.256360117 -0700
+++ /tmp/foob	2017-07-20 21:43:04.260360133 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc1-next-20170720-305317-gf80addb (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Jul 19 22:33:30 PDT 2017
+Linux version 4.13.0-rc1-next-20170721-305622-g83d9512 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Thu Jul 20 21:40:59 PDT 2017
 Boot CPU: AArch64 Processor [411fd070]
 Machine model: linux,dummy-virt
 Memory limited to 1024MB
@@ -30,7 +30,7 @@
 Policy zone: DMA
 Kernel command line: root=/dev/vda2 rw rootwait mem=1024M console=ttyAMA0,38400n8 init=/bin/bash
 PID hash table entries: 4096 (order: 3, 32768 bytes)
-Memory: 999304K/1048576K available (9084K kernel code, 1118K rwdata, 4412K rodata, 1088K init, 401K bss, 32888K reserved, 16384K cma-reserved)
+Memory: 999304K/1048576K available (9084K kernel code, 1118K rwdata, 4416K rodata, 1088K init, 401K bss, 32888K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
@@ -116,7 +116,7 @@
 hw perfevents: enabled with armv8_pmuv3 PMU driver, 1 counters available
 kvm [1]: HYP mode not available
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.804:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.752:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -183,7 +183,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2017-07-20 08:44:09 UTC (1500540249)
+rtc-pl031 9010000.pl031: setting system clock to 2017-07-21 04:41:59 UTC (1500612119)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-07-20 21:43:04.296360282 -0700
+++ /tmp/foob	2017-07-20 21:43:04.300360299 -0700
@@ -42,18 +42,18 @@
 	 ###########################
 	 3 MiB/s
 done
-Bytes transferred = 8016272 (7a5190 hex)
+Bytes transferred = 8015960 (7a5058 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.8 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x7a5190 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x7a5058 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc1-next-20170720-187076-gf80addb (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Jul 19 22:28:18 PDT 2017
+Linux version 4.13.0-rc1-next-20170721-187381-g83d9512 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Jul 20 21:35:49 PDT 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -85,7 +85,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993552K/2096128K available (10240K kernel code, 1562K rwdata, 4460K rodata, 2048K init, 402K bss, 37040K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993552K/2096128K available (10240K kernel code, 1562K rwdata, 4464K rodata, 2048K init, 402K bss, 37040K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -96,7 +96,7 @@
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
       .data : 0xc1400000 - 0xc1586ba0   (1563 kB)
-       .bss : 0xc158fcb0 - 0xc15f46fc   ( 403 kB)
+       .bss : 0xc158fc98 - 0xc15f46fc   ( 403 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -320,18 +320,18 @@
  mmcblk0: p1 p2 p3
 tegra-ehci 7d008000.usb: EHCI Host Controller
 tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 1
-mmc1: new high speed MMC card at address 0001
 tegra-ehci 7d008000.usb: irq 339, io mem 0x7d008000
+mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
+mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
+ mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2001-05-13 17:07:30 UTC (989773650)
-mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
- mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2001-05-14 16:15:07 UTC (989856907)
 vdd_hdmi_5v0: disabling
 dvdd_ts: disabling
 usb1_vbus: disabling
@@ -345,4 +345,4 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.419283] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.459331] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2017-07-20 21:43:04.328360414 -0700
+++ /tmp/foob	2017-07-20 21:43:04.332360430 -0700
@@ -38,20 +38,20 @@
 	 #################################################################
 	 #################################################################
 	 #############################
-	 3.1 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 6149208 (5dd458 hex)
+Bytes transferred = 6149608 (5dd5e8 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.6 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5dd458 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5dd5e8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -60,7 +60,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc1-next-20170720-187076-gf80addb (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Jul 19 22:28:20 PDT 2017
+Linux version 4.13.0-rc1-next-20170721-187381-g83d9512 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Jul 20 21:35:46 PDT 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -70,7 +70,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0eafa00, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0eaf9c0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -93,8 +93,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0eba220   ( 745 kB)
-       .bss : 0xc0ec4604 - 0xc0f0f3bc   ( 300 kB)
+      .data : 0xc0e00000 - 0xc0eba1e0   ( 745 kB)
+       .bss : 0xc0ec45ec - 0xc0f0f37c   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -287,8 +287,8 @@
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
-mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
  mmcblk0: p1 p2 p3
+mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
  mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
 tegra30-i2s 70080400.i2s: DMA channels sourced from device 70080000.ahub
@@ -323,7 +323,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2001-05-13 17:10:06 UTC (989773806)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2001-05-14 16:17:50 UTC (989857070)
 vdd_hdmi_5v0: disabling
 dvdd_ts: disabling
 usb1_vbus: disabling
@@ -339,4 +339,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.677219] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.667666] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-07-20 21:43:04.364360562 -0700
+++ /tmp/foob	2017-07-20 21:43:04.368360578 -0700
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+MMC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -35,20 +35,20 @@
 	 #################################################################
 	 #################################################################
 	 ###########################
-	 3.9 MiB/s
+	 3.7 MiB/s
 done
-Bytes transferred = 8016272 (7a5190 hex)
+Bytes transferred = 8015960 (7a5058 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.8 MiB/s
+	 3.6 MiB/s
 done
 Bytes transferred = 67161 (10659 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x7a5190 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x7a5058 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013658
@@ -56,7 +56,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc1-next-20170720-187076-gf80addb (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Jul 19 22:28:18 PDT 2017
+Linux version 4.13.0-rc1-next-20170721-187381-g83d9512 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Jul 20 21:35:49 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -80,7 +80,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993428K/2096128K available (10240K kernel code, 1562K rwdata, 4460K rodata, 2048K init, 402K bss, 37164K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993428K/2096128K available (10240K kernel code, 1562K rwdata, 4464K rodata, 2048K init, 402K bss, 37164K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -91,7 +91,7 @@
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
       .data : 0xc1400000 - 0xc1586ba0   (1563 kB)
-       .bss : 0xc158fcb0 - 0xc15f46fc   ( 403 kB)
+       .bss : 0xc158fc98 - 0xc15f46fc   ( 403 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -318,12 +318,6 @@
 +5V_SATA: supplied by +5V_SYS
 +12V_SATA: supplied by +VDD_MUX
  usb2-0: usb2-0 supply vbus not found, using dummy regulator
-mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
- mmcblk0: p1
  usb2-1: usb2-1 supply vbus not found, using dummy regulator
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
 tegra-pcie 1003000.pcie: 2x1, 1x1 configuration
@@ -349,9 +343,15 @@
 +2.8V_RUN_CAM_AF: Bringing 0uV into 2800000-2800000uV
 as3722-ldo11: supplied by +3.3V_RUN
 +1.8V_RUN_VPP_FUSE: Bringing 0uV into 1800000-1800000uV
+mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
 tegra-ahci 70027000.sata: AHCI 0001.0301 32 slots 2 ports 3 Gbps 0x1 impl platform mode
 tegra-ahci 70027000.sata: flags: 64bit ncq sntf pm led pio slum part sadm sds apst 
 scsi host0: tegra-ahci
+ mmcblk0: p1
 scsi host1: tegra-ahci
 ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 345
 ata2: DUMMY
@@ -425,6 +425,9 @@
 pci 0000:00:02.0:   bridge window [io  0x1000-0x1fff]
 pci 0000:00:02.0:   bridge window [mem 0x13100000-0x131fffff]
 pci 0000:00:02.0:   bridge window [mem 0x20000000-0x200fffff 64bit pref]
+mmcblk1: error -5 sending stop command, original cmd response 0x900, card status 0x900
+mmcblk1: error -5 transferring data, sector 7744504, nr 8, cmd response 0x900, card status 0x80000b00
+mmcblk1: retrying using single block read
 pci 0000:00:01.0: nv_msi_ht_cap_quirk didn't locate host bridge
 pcieport 0000:00:01.0: enabling device (0140 -> 0142)
 pcieport 0000:00:01.0: Signaling PME with IRQ 393
@@ -440,7 +443,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:03 UTC (946684923)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:13 UTC (946684813)
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +USB0_VBUS_SW: disabling
 +1.05V_RUN_AVDD_HDMI_PLL: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2017-07-20 21:43:04.388360661 -0700
+++ /tmp/foob	2017-07-20 21:43:04.392360677 -0700
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -33,9 +33,9 @@
 	 #################################################################
 	 #################################################################
 	 #############################
-	 3.9 MiB/s
+	 3.7 MiB/s
 done
-Bytes transferred = 6149208 (5dd458 hex)
+Bytes transferred = 6149608 (5dd5e8 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -46,7 +46,7 @@
 done
 Bytes transferred = 67161 (10659 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5dd458 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5dd5e8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013658
@@ -55,7 +55,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc1-next-20170720-187076-gf80addb (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Jul 19 22:28:20 PDT 2017
+Linux version 4.13.0-rc1-next-20170721-187381-g83d9512 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Jul 20 21:35:46 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -65,7 +65,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0eafa00, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0eaf9c0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -88,8 +88,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0eba220   ( 745 kB)
-       .bss : 0xc0ec4604 - 0xc0f0f3bc   ( 300 kB)
+      .data : 0xc0e00000 - 0xc0eba1e0   ( 745 kB)
+       .bss : 0xc0ec45ec - 0xc0f0f37c   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -333,8 +333,8 @@
 as3722-ldo4: supplied by +3.3V_SYS
 as3722-ldo5: supplied by +1.8V_VDDIO
 as3722-ldo6: supplied by +3.3V_RUN
-as3722-ldo7: supplied by +1.8V_VDDIO
 input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input1
+as3722-ldo7: supplied by +1.8V_VDDIO
 +1.05V_RUN_CAM_REAR: Bringing 0uV into 1050000-1050000uV
 as3722-ldo9: supplied by +5V_SYS
 +3.3V_RUN_TOUCH: Bringing 0uV into 2800000-2800000uV
@@ -403,8 +403,8 @@
 pci 0000:01:00.0: reg 0x10: [mem 0x00000000-0x00001fff 64bit]
 pci 0000:01:00.0: PME# supported from D0 D3hot D3cold
 mmc1: new high speed SDHC card at address aaaa
-mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 PCI: bus1: Fast back to back transfers disabled
+mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
 pci 0000:02:00.0: [10ec:8168] type 00 class 0x020000
 pci 0000:02:00.0: reg 0x10: [io  0x0000-0x00ff]
@@ -430,7 +430,11 @@
 pci 0000:00:02.0:   bridge window [io  0x1000-0x1fff]
 pci 0000:00:02.0:   bridge window [mem 0x13100000-0x131fffff]
 pci 0000:00:02.0:   bridge window [mem 0x20000000-0x200fffff 64bit pref]
+mmcblk1: error -5 sending stop command, original cmd response 0x900, card status 0x900
+mmcblk1: error -5 transferring data, sector 7744504, nr 8, cmd response 0x900, card status 0x80000b00
+mmcblk1: retrying using single block read
 pci 0000:00:01.0: nv_msi_ht_cap_quirk didn't locate host bridge
+ mmcblk1: p1 p2 p3
 pcieport 0000:00:01.0: enabling device (0140 -> 0142)
 pcieport 0000:00:01.0: Signaling PME with IRQ 393
 pci 0000:00:02.0: nv_msi_ht_cap_quirk didn't locate host bridge
@@ -438,14 +442,13 @@
 pcieport 0000:00:02.0: Signaling PME with IRQ 394
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
- mmcblk1: p1 p2 p3
 r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0c01000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 395
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:13 UTC (946684813)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:01:52 UTC (946684912)
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +USB0_VBUS_SW: disabling
 +1.05V_RUN_AVDD_HDMI_PLL: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-07-20 21:43:04.404360727 -0700
+++ /tmp/foob	2017-07-20 21:43:04.408360744 -0700
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -45,7 +45,7 @@
 	 ###########################
 	 3.6 MiB/s
 done
-Bytes transferred = 8016272 (7a5190 hex)
+Bytes transferred = 8015960 (7a5058 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -56,7 +56,7 @@
 done
 Bytes transferred = 68183 (10a57 hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x7a5190 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x7a5058 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a56
@@ -64,7 +64,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc1-next-20170720-187076-gf80addb (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Jul 19 22:28:18 PDT 2017
+Linux version 4.13.0-rc1-next-20170721-187381-g83d9512 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Jul 20 21:35:49 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -89,7 +89,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1994448K/2097148K available (10240K kernel code, 1562K rwdata, 4460K rodata, 2048K init, 402K bss, 37164K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1994448K/2097148K available (10240K kernel code, 1562K rwdata, 4464K rodata, 2048K init, 402K bss, 37164K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -100,7 +100,7 @@
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
       .data : 0xc1400000 - 0xc1586ba0   (1563 kB)
-       .bss : 0xc158fcb0 - 0xc15f46fc   ( 403 kB)
+       .bss : 0xc158fc98 - 0xc15f46fc   ( 403 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2017-07-20 21:43:04.420360793 -0700
+++ /tmp/foob	2017-07-20 21:43:04.420360793 -0700
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -41,20 +41,20 @@
 	 #################################################################
 	 #################################################################
 	 #############################
-	 3.6 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 6149208 (5dd458 hex)
+Bytes transferred = 6149608 (5dd5e8 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.3 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 68183 (10a57 hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5dd458 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5dd5e8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a56
@@ -63,7 +63,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc1-next-20170720-187076-gf80addb (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Jul 19 22:28:20 PDT 2017
+Linux version 4.13.0-rc1-next-20170721-187381-g83d9512 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Jul 20 21:35:46 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -74,7 +74,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c0eafa00, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0eaf9c0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -97,8 +97,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0eba220   ( 745 kB)
-       .bss : 0xc0ec4604 - 0xc0f0f3bc   ( 300 kB)
+      .data : 0xc0e00000 - 0xc0eba1e0   ( 745 kB)
+       .bss : 0xc0ec45ec - 0xc0f0f37c   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -356,7 +356,7 @@
 mmc1: new high speed SDIO card at address 0001
 mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
 input: gpio-keys as /devices/soc0/gpio-keys/input/input3
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:22 UTC (946684822)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:21 UTC (946684821)
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +1.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2017-07-20 21:43:04.448360908 -0700
+++ /tmp/foob	2017-07-20 21:43:04.448360908 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1801496 us
+Bootloader-Cpu Init at (time stamp): 1802106 us
 
 Pinmux changes applied in kernel way
 
@@ -59,25 +59,25 @@
 The proc BoardInfo: 0x0176:0x0001:0x01:0x43:0x00
 FastbootAddAtag: setting ATAG (0)
 
-Board Id Read Time: 3 us
+Board Id Read Time: 4 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1801470  us
-Bootloader (AosInit) time: 733539 us
-Bootloader Main Init time: 180130 us
-BlockDevMgrInit time: 157550 us
+Miscellaneous avp time: 1802079  us
+Bootloader (AosInit) time: 734232 us
+Bootloader Main Init time: 180080 us
+BlockDevMgrInit time: 157545 us
 PartMgrInit time: 8 us
 StorMgrInit time: 31 us
 FsMgrInit time: 2 us
-Partition Table load: 77081 us 
-Display init time: 574478 us
-Kernel read time: 1319 msec
+Partition Table load: 76770 us 
+Display init time: 574707 us
+Kernel read time: 1318 msec
 Load Kernel Size:   17473536 Bytes
-Bl Kernel ReadSpeed: 13247000 Bytes/sec
-Pre-Kernel Jump time: 3771378 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 8710533 us
+Bl Kernel ReadSpeed: 13257000 Bytes/sec
+Pre-Kernel Jump time: 3771255 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 8709247 us
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc1-next-20170720-305317-gf80addb (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Jul 19 22:33:30 PDT 2017
+Linux version 4.13.0-rc1-next-20170721-305622-g83d9512 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Thu Jul 20 21:40:59 PDT 2017
 Boot CPU: AArch64 Processor [4e0f0000]
 Machine model: NVIDIA Tegra132 Norrin
 Memory limited to 1734MB
@@ -103,7 +103,7 @@
 Policy zone: DMA
 Kernel command line: tegraid=13.1.3.0.0 mem=1734M@2048M memtype=255 vpr=151M@3817M ddr_die=2048M@2048M section=256M commchip_id=0 usb_port_owner_info=0 lane_owner_info=0 emc_max_dvfs=0 touch_id=0@0 video=tegrafb no_console_suspend=1 console=ttyS0,115200n8 debug_uartport=lsport,0 earlyprintk=uart8250-32bit,0x70006000 usbcore.old_scheme_first=1 lp0_vec=0@0x0 tegra_fbmem=32899072@0xad012000 core_edp_mv=1150 core_edp_ma=4000 pmuboard=0x0176:0x0001:0x01:0x43:0x00 displayboard=0x0162:0x0001:0x00:0x42:0x00 power_supply=Adapter audio_codec=rt5640 board_info=0x0176:0x0001:0x01:0x43:0x00 tegraboot=sdmmc gpt gpt_sector=192511 modem_id=0 android.kerneltype=normal
 PID hash table entries: 4096 (order: 3, 32768 bytes)
-Memory: 1708524K/1775616K available (9084K kernel code, 1118K rwdata, 4412K rodata, 1088K init, 401K bss, 50708K reserved, 16384K cma-reserved)
+Memory: 1708524K/1775616K available (9084K kernel code, 1118K rwdata, 4416K rodata, 1088K init, 401K bss, 50708K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
@@ -202,7 +202,7 @@
 Freeing initrd memory: 1284K
 kvm [1]: HYP mode not available
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.931:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.923:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -267,15 +267,15 @@
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
 mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 NET: Registered protocol family 17
-9pnet: Installing 9P2000 support
-Key type dns_resolver registered
-registered taskstats version 1
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
+9pnet: Installing 9P2000 support
+Key type dns_resolver registered
+registered taskstats version 1
 +VDD_LED: supplied by +VDD_MUX
 +5V_USB_HS: supplied by +5V_SYS
 +5V_USB_SS: supplied by +5V_SYS
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-07-20 21:43:04.456360942 -0700
+++ /tmp/foob	2017-07-20 21:43:04.460360957 -0700
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -48,20 +48,20 @@
 	 #################################################################
 	 #################################################################
 	 ###########################
-	 3.5 MiB/s
+	 4 MiB/s
 done
-Bytes transferred = 8016272 (7a5190 hex)
+Bytes transferred = 8015960 (7a5058 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 4.1 MiB/s
 done
 Bytes transferred = 21454 (53ce hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x7a5190 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x7a5058 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103cd
@@ -69,7 +69,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc1-next-20170720-187076-gf80addb (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Jul 19 22:28:18 PDT 2017
+Linux version 4.13.0-rc1-next-20170721-187381-g83d9512 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Jul 20 21:35:49 PDT 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -92,7 +92,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 954380K/1048576K available (10240K kernel code, 1562K rwdata, 4460K rodata, 2048K init, 402K bss, 28660K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 954380K/1048576K available (10240K kernel code, 1562K rwdata, 4464K rodata, 2048K init, 402K bss, 28660K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -103,7 +103,7 @@
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
       .data : 0xc1400000 - 0xc1586ba0   (1563 kB)
-       .bss : 0xc158fcb0 - 0xc15f46fc   ( 403 kB)
+       .bss : 0xc158fc98 - 0xc15f46fc   ( 403 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -251,10 +251,9 @@
 tegra-hdmi 54280000.hdmi: 54280000.hdmi supply hdmi not found, using dummy regulator
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
-[drm] Cannot find any crtc or sizes - going 1024x768
-Console: switching to colour frame buffer device 128x48
-drm drm: fb0:  frame buffer device
+[drm] Cannot find any crtc or sizes
 [drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
+[drm] Cannot find any crtc or sizes
 brd: module loaded
 loop: module loaded
 m25p80 spi32766.0: found s25fl008k, expected w25q80bl
@@ -350,8 +349,8 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-hub 1-1:1.0: USB hub found
 rtc-em3027 2-0056: hctosys: unable to read the hardware clock
+hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
 mmcblk1: error -5 sending stop command, original cmd response 0x900, card status 0x900
 mmcblk1: error -5 transferring data, sector 3839992, nr 8, cmd response 0x900, card status 0x80000b00
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2017-07-20 21:43:04.484361057 -0700
+++ /tmp/foob	2017-07-20 21:43:04.484361057 -0700
@@ -48,18 +48,18 @@
 	 #############################
 	 4 MiB/s
 done
-Bytes transferred = 6149208 (5dd458 hex)
+Bytes transferred = 6149608 (5dd5e8 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 4.1 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21454 (53ce hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x5dd458 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x5dd5e8 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103cd
@@ -68,7 +68,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc1-next-20170720-187076-gf80addb (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Jul 19 22:28:20 PDT 2017
+Linux version 4.13.0-rc1-next-20170721-187381-g83d9512 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Jul 20 21:35:46 PDT 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -77,7 +77,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0x3c000000
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0eafa00, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0eaf9c0, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -100,8 +100,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0eba220   ( 745 kB)
-       .bss : 0xc0ec4604 - 0xc0f0f3bc   ( 300 kB)
+      .data : 0xc0e00000 - 0xc0eba1e0   ( 745 kB)
+       .bss : 0xc0ec45ec - 0xc0f0f37c   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
 	Tasks RCU enabled.
@@ -246,9 +246,8 @@
 tegra-hdmi 54280000.hdmi: 54280000.hdmi supply hdmi not found, using dummy regulator
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
-[drm] Cannot find any crtc or sizes - going 1024x768
-Console: switching to colour frame buffer device 128x48
-drm drm: fb0:  frame buffer device
+[drm] Cannot find any crtc or sizes
+[drm] Cannot find any crtc or sizes
 [drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
 loop: module loaded
 m25p80 spi32766.0: found s25fl008k, expected w25q80bl
@@ -333,23 +332,23 @@
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
+tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 mmcblk1: error -5 sending stop command, original cmd response 0x900, card status 0x900
 mmcblk1: error -5 transferring data, sector 3839992, nr 8, cmd response 0x900, card status 0x80000b00
-tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
-mmcblk1: error -5 sending stop command, original cmd response 0x900, card status 0x900
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
-input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: hctosys: unable to read the hardware clock
-ALSA device list:
-  #0: tegra-trimslice
+mmcblk1: error -5 sending stop command, original cmd response 0x900, card status 0x900
 mmcblk1: error -5 transferring data, sector 3839992, nr 8, cmd response 0x900, card status 0x80000b00
 mmcblk1: retrying using single block read
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
  mmcblk1: p1 p2
+input: gpio-keys as /devices/soc0/gpio-keys/input/input0
+rtc-em3027 2-0056: hctosys: unable to read the hardware clock
+ALSA device list:
+  #0: tegra-trimslice
 Waiting for root device PARTUUID=11eb23c1-01...
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 mmc0: host does not support reading read-only switch, assuming write-enable
@@ -357,11 +356,11 @@
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
 mmcblk0: error -5 sending stop command, original cmd response 0x900, card status 0x900
 mmcblk0: error -5 transferring data, sector 7759864, nr 8, cmd response 0x900, card status 0x80000b00
+asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
 mmcblk0: error -5 sending stop command, original cmd response 0x900, card status 0x900
 mmcblk0: error -5 transferring data, sector 7759864, nr 8, cmd response 0x900, card status 0x80000b00
 mmcblk0: retrying using single block read
  mmcblk0: p1
-asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
 mmcblk0: r/w command failed, status = 0x80000900
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk0p1): recovery complete
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2017-07-20 21:43:04.504361139 -0700
+++ /tmp/foob	2017-07-20 21:43:04.508361155 -0700
@@ -1,7 +1,7 @@
 [TegraBoot] (version 23.00.2015.14-mobile-1ef66670)
 Processing in cold boot mode Bootloader 2
 A01Q Bootrom Patch rev = 3
-Power-up reason: pmu watchdog
+Power-up reason: on button
 No Battery Present
 Platform has Ddr4 type ram
 max77620 disabling SD1 Remote Sense
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 825996 us
-Basic modules init took 350393 us
+Tegraboot started after 776646 us
+Basic modules init took 348729 us
 Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14387 us
-Carveout took 252726 us
-CPU initialization took 92304 us
-Total time taken by TegraBoot 709810 us
+Next stage binary read took 14366 us
+Carveout took 252727 us
+CPU initialization took 92291 us
+Total time taken by TegraBoot 708113 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.664] LPDDR4 Training: Number of tables = 10
+64b[0001.615] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-16153088 bytes read in 826 ms (18.6 MiB/s)
+16153088 bytes read in 822 ms (18.7 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-45170 bytes read in 454 ms (96.7 KiB/s)
+45170 bytes read in 455 ms (96.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 495 ms (3.1 MiB/s)
+1627787 bytes read in 496 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc1-next-20170720-305317-gf80addb (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Jul 19 22:33:30 PDT 2017
+Linux version 4.13.0-rc1-next-20170721-305622-g83d9512 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Thu Jul 20 21:40:59 PDT 2017
 Boot CPU: AArch64 Processor [411fd071]
 Machine model: NVIDIA Tegra210 P2371 (P2530/P2595) reference design
 debug: ignoring loglevel setting.
@@ -175,7 +175,7 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffff80007a000000-ffff80007dffffff]
-Memory: 2982856K/3131392K available (9084K kernel code, 1118K rwdata, 4412K rodata, 1088K init, 401K bss, 132152K reserved, 16384K cma-reserved)
+Memory: 2982856K/3131392K available (9084K kernel code, 1118K rwdata, 4416K rodata, 1088K init, 401K bss, 132152K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
@@ -278,7 +278,7 @@
 kvm [1]: vgic interrupt IRQ1
 kvm [1]: virtual timer IRQ4
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.491:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.495:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2017-07-20 21:43:04.524361221 -0700
+++ /tmp/foob	2017-07-20 21:43:04.528361238 -0700
@@ -1,4 +1,4 @@
-rameters from FDT:
+T:
 efi: UEFI not found.
 cma: Reserved 16 MiB at 0x00000000f5400000
 earlycon: uart0 at MMIO 0x0000000070006000 (options '115200n8')
@@ -30,7 +30,7 @@
 Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2838120K/2988032K available (9084K kernel code, 1118K rwdata, 4412K rodata, 1088K init, 401K bss, 133528K reserved, 16384K cma-reserved)
+Memory: 2838120K/2988032K available (9084K kernel code, 1118K rwdata, 4416K rodata, 1088K init, 401K bss, 133528K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
@@ -76,7 +76,7 @@
 console [tty0] enabled
 bootconsole [uart0] disabled
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc1-next-20170720-305317-gf80addb (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Jul 19 22:33:30 PDT 2017
+Linux version 4.13.0-rc1-next-20170721-305622-g83d9512 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Thu Jul 20 21:40:59 PDT 2017
 Boot CPU: AArch64 Processor [411fd071]
 Machine model: Google Pixel C
 efi: Getting EFI parameters from FDT:
@@ -111,7 +111,7 @@
 Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2838120K/2988032K available (9084K kernel code, 1118K rwdata, 4412K rodata, 1088K init, 401K bss, 133528K reserved, 16384K cma-reserved)
+Memory: 2838120K/2988032K available (9084K kernel code, 1118K rwdata, 4416K rodata, 1088K init, 401K bss, 133528K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
@@ -184,13 +184,14 @@
 clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 7645041785100000 ns
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 pinctrl core: initialized pinctrl subsystem
-random: fast init done
 DMI not present or invalid.
+random: get_random_u32 called from bucket_table_alloc+0x114/0x29c with crng_init=0
 NET: Registered protocol family 16
 cpuidle: using governor menu
 vdso: 2 pages (1 code @ ffff000008966000, 1 data @ ffff000008ed5000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
+random: fast init done
 Serial: AMBA PL011 UART driver
 HugeTLB registered 2.00 MiB page size, pre-allocated 0 pages
 ACPI: Interpreter disabled.
@@ -339,12 +340,11 @@
 tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 BGND3R 29.1 GiB 
-max77686-rtc max77620-rtc: setting system clock to 2001-04-30 10:58:35 UTC (988628315)
 mmcblk0boot0: mmc0:0001 BGND3R partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
-tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
+max77686-rtc max77620-rtc: setting system clock to 2001-05-01 10:06:18 UTC (988711578)
 PP1200_AVDD: disabling
 PP1200_RCAM: disabling
 PP2800L_RCAM: disabling
@@ -356,6 +356,7 @@
 ALSA device list:
   No soundcards found.
 Freeing unused kernel memory: 1088K
+tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 
 System initialization...
 
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-07-20 21:43:04.532361254 -0700
+++ /tmp/foob	2017-07-20 21:43:04.536361271 -0700
@@ -40,20 +40,20 @@
 	 #################################################################
 	 #################################################################
 	 ###########################
-	 3.1 MiB/s
+	 2.8 MiB/s
 done
-Bytes transferred = 8016272 (7a5190 hex)
+Bytes transferred = 8015960 (7a5058 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.6 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 51848 (ca88 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x7a5190 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x7a5058 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa87 ... OK
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc1-next-20170720-187076-gf80addb (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Jul 19 22:28:18 PDT 2017
+Linux version 4.13.0-rc1-next-20170721-187381-g83d9512 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Jul 20 21:35:49 PDT 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -84,7 +84,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993468K/2096128K available (10240K kernel code, 1562K rwdata, 4460K rodata, 2048K init, 402K bss, 37124K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993468K/2096128K available (10240K kernel code, 1562K rwdata, 4464K rodata, 2048K init, 402K bss, 37124K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -95,7 +95,7 @@
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
       .data : 0xc1400000 - 0xc1586ba0   (1563 kB)
-       .bss : 0xc158fcb0 - 0xc15f46fc   ( 403 kB)
+       .bss : 0xc158fc98 - 0xc15f46fc   ( 403 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -312,17 +312,17 @@
 sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
 +VDD_5V_HDMI: supplied by sys_3v3,vdd_3v3_alw
 tegra-pcie 3000.pcie: 2x3 configuration
-mmc1: new high speed MMC card at address 0001
 tegra-pcie 3000.pcie: probing port 0, using 2 lanes
+mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB
+mmc0: new ultra high speed DDR50 SDHC card at address aaaa
+mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
 Primary GPT is invalid, using alternate GPT.
  mmcblk1: p1 p2 p3 p4 p5
-mmc0: new ultra high speed DDR50 SDHC card at address aaaa
 tegra-pcie 3000.pcie: probing port 2, using 2 lanes
-mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
  mmcblk0: p1 p2 p3
 tegra-pcie 3000.pcie: link 2 down, retrying
 tegra-pcie 3000.pcie: link 2 down, retrying
@@ -395,5 +395,5 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.053711] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.074110] usb 2-1: new high-speed USB device number 2 using tegra-ehci
 asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0f:67:60
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2017-07-20 21:43:04.556361353 -0700
+++ /tmp/foob	2017-07-20 21:43:04.560361369 -0700
@@ -40,18 +40,18 @@
 	 #############################
 	 3.2 MiB/s
 done
-Bytes transferred = 6149208 (5dd458 hex)
+Bytes transferred = 6149608 (5dd5e8 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 51848 (ca88 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5dd458 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5dd5e8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa87 ... OK
@@ -60,7 +60,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc1-next-20170720-187076-gf80addb (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Jul 19 22:28:20 PDT 2017
+Linux version 4.13.0-rc1-next-20170721-187381-g83d9512 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Jul 20 21:35:46 PDT 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -69,7 +69,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0eafa00, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0eaf9c0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -92,8 +92,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0eba220   ( 745 kB)
-       .bss : 0xc0ec4604 - 0xc0f0f3bc   ( 300 kB)
+      .data : 0xc0e00000 - 0xc0eba1e0   ( 745 kB)
+       .bss : 0xc0ec45ec - 0xc0f0f37c   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -389,4 +389,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.350087] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.340091] usb 2-1: new high-speed USB device number 2 using tegra-ehci
