Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-08-10 21:13:14.503745377 -0700
+++ /tmp/foob	2016-08-10 21:13:14.507745380 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc1-next-20160809+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Aug 8 21:08:31 PDT 2016
+Linux version 4.8.0-rc1-next-20160811+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Aug 10 21:09:15 PDT 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -18,15 +18,15 @@
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
-Memory: 1000360K/1048576K available (7868K kernel code, 777K rwdata, 3692K rodata, 960K init, 271K bss, 31832K reserved, 16384K cma-reserved)
+Memory: 1000360K/1048576K available (7868K kernel code, 777K rwdata, 3696K rodata, 960K init, 270K bss, 31832K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008830000   (  7872 KB)
     .rodata : 0xffff000008830000 - 0xffff000008bd0000   (  3712 KB)
       .init : 0xffff000008bd0000 - 0xffff000008cc0000   (   960 KB)
-      .data : 0xffff000008cc0000 - 0xffff000008d82400   (   777 KB)
-       .bss : 0xffff000008d82400 - 0xffff000008dc6038   (   272 KB)
+      .data : 0xffff000008cc0000 - 0xffff000008d82600   (   778 KB)
+       .bss : 0xffff000008d82600 - 0xffff000008dc60b8   (   271 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -100,7 +100,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.716:1): initialized
+audit: type=2000 audit(0.784:1): initialized
 workingset: timestamp_bits=46 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -166,7 +166,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2016-08-09 04:09:09 UTC (1470715749)
+rtc-pl031 9010000.pl031: setting system clock to 2016-08-11 04:09:53 UTC (1470888593)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-08-10 21:13:14.515745387 -0700
+++ /tmp/foob	2016-08-10 21:13:14.515745387 -0700
@@ -38,10 +38,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #############################
-	 3.4 MiB/s
+	 ##############################
+	 3.1 MiB/s
 done
-Bytes transferred = 7095216 (6c43b0 hex)
+Bytes transferred = 7107304 (6c72e8 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -52,7 +52,7 @@
 done
 Bytes transferred = 34956 (888c hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6c43b0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6c72e8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff88b ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc1-next-20160809-110829-g09f0834 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Aug 8 21:05:02 PDT 2016
+Linux version 4.8.0-rc1-next-20160811-111289-gc0a5420 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Aug 10 21:05:04 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,12 +69,12 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141d2c0, node_mem_map eeffa000
+free_area_init_node: node 0, pgdat c141da80, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 14 pages/cpu @eef9d000 s26624 r8192 d22528 u57344
+percpu: Embedded 14 pages/cpu @eef9b000 s26624 r8192 d22528 u57344
 pcpu-alloc: s26624 r8192 d22528 u57344 alloc=14*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1995100K/2096128K available (9216K kernel code, 1148K rwdata, 3976K rodata, 2048K init, 343K bss, 35492K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1995092K/2096128K available (9216K kernel code, 1150K rwdata, 3976K rodata, 2048K init, 343K bss, 35500K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,8 +92,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141f140   (1149 kB)
-       .bss : 0xc1421000 - 0xc1476c08   ( 344 kB)
+      .data : 0xc1300000 - 0xc141f900   (1151 kB)
+       .bss : 0xc1421000 - 0xc1476cc8   ( 344 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -135,7 +135,8 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
+No ATAGs?
+hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 8 bytes.
 Serial: AMBA PL011 UART driver
 tegra-mc 70019000.memory-controller: no memory timings for RAM code 3 registered
@@ -281,13 +282,21 @@
 Registering SWP/SWPB emulation handler
 usb1_vbus: supplied by vdd-sys-5v0
 usb2_vbus: supplied by vdd-sys-5v0
-mmc0: new high speed SDHC card at address aaaa
 VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
+mmc0: new high speed SDHC card at address aaaa
+mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
 tegra-dsi 54300000.dsi: cannot get VDD supply
+ mmcblk0: p1 p2 p3
 tegra-ehci 7d008000.usb: EHCI Host Controller
 tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 1
 tegra-ehci 7d008000.usb: irq 339, io mem 0x7d008000
+mmc1: new high speed MMC card at address 0001
+mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
+mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
+mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
+mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
+ mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
@@ -301,26 +310,12 @@
 dvdd_ts: disabling
 usb1_vbus: disabling
 VDD_5V0_HDMI_CON: disabling
-mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
- mmcblk0: p1 p2 p3
-Waiting for root device PARTUUID=7369c667-ff51-ec4a-29cd-baabf2fbe346...
-tegra-hdmi 54280000.hdmi: failed to get PLL regulator
-tegra-dsi 54300000.dsi: cannot get VDD supply
-mmc1: new high speed MMC card at address 0001
-mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
-mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
-mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
-mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
- mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
-tegra-hdmi 54280000.hdmi: failed to get PLL regulator
-tegra-dsi 54300000.dsi: cannot get VDD supply
 EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk1p1): recovery complete
 EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:17.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# 
\ No newline at end of file
+root@(none):/# [    2.160710] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-08-10 21:13:14.523745393 -0700
+++ /tmp/foob	2016-08-10 21:13:14.523745393 -0700
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra114 (Dalmore) # 
 Tegra114 (Dalmore) # usb reset
 (Re)start USB...
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 ############################################
-	 3.1 MiB/s
+	 3 MiB/s
 done
-Bytes transferred = 5413280 (5299a0 hex)
+Bytes transferred = 5415608 (52a2b8 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3.3 MiB/s
+	 2.6 MiB/s
 done
 Bytes transferred = 34956 (888c hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5299a0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x52a2b8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff88b ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc1-next-20160809-110829-g09f0834 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Aug 8 21:04:56 PDT 2016
+Linux version 4.8.0-rc1-next-20160811-111289-gc0a5420 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Aug 10 21:04:56 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,12 +69,12 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0d5c180, node_mem_map eeff9000
+free_area_init_node: node 0, pgdat c0d5c3c0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 14 pages/cpu @eef9c000 s26176 r8192 d22976 u57344
+percpu: Embedded 14 pages/cpu @eef9a000 s26176 r8192 d22976 u57344
 pcpu-alloc: s26176 r8192 d22976 u57344 alloc=14*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999944K/2096128K available (8192K kernel code, 375K rwdata, 2444K rodata, 1024K init, 366K bss, 30648K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1999936K/2096128K available (8192K kernel code, 375K rwdata, 2432K rodata, 1024K init, 366K bss, 30656K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,8 +92,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d5dc00   ( 375 kB)
-       .bss : 0xc0d5f000 - 0xc0dbaad8   ( 367 kB)
+      .data : 0xc0d00000 - 0xc0d5de40   ( 376 kB)
+       .bss : 0xc0d5f000 - 0xc0dbab18   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -132,7 +132,8 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
+No ATAGs?
+hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 8 bytes.
 tegra-mc 70019000.memory-controller: no memory timings for RAM code 3 registered
 dvdd_ts: Failed to request enable GPIO61: -517
@@ -212,6 +213,7 @@
 tegra-hdmi 54280000.hdmi: failed to get HDMI regulator
 tegra-dsi 54300000.dsi: cannot get VDD supply
 loop: module loaded
+random: fast init done
 m25p80 spi32766.0: w25q32dw (4096 Kbytes)
 libphy: Fixed MDIO Bus: probed
 CAN device driver interface
@@ -244,7 +246,6 @@
 gspca_main: v2.14.0 registered
 sbs-battery 0-000b: sbs_probe: Failed to get device status
 sbs-battery: probe of 0-000b failed with error -121
-random: fast init done
 tegra-wdt 60005000.timer: initialized (heartbeat = 120 sec, nowayout = 0)
 sdhci: Secure Digital Host Controller Interface driver
 sdhci: Copyright(c) Pierre Ossman
@@ -259,8 +260,8 @@
 usbhid: USB HID core driver
 mmc0: new high speed SDHC card at address aaaa
 mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
-mmc1: new high speed MMC card at address 0001
  mmcblk0: p1 p2 p3
+mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-08-10 21:13:14.527745396 -0700
+++ /tmp/foob	2016-08-10 21:13:14.531745399 -0700
@@ -33,10 +33,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #############################
-	 3.9 MiB/s
+	 ##############################
+	 4 MiB/s
 done
-Bytes transferred = 7095216 (6c43b0 hex)
+Bytes transferred = 7107304 (6c72e8 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -45,17 +45,17 @@
 Loading: *#####
 	 3.7 MiB/s
 done
-Bytes transferred = 66405 (10365 hex)
+Bytes transferred = 66621 (1043d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6c43b0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x6c72e8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
-   Using Device Tree in place at 83000000, end 83013364
+   Using Device Tree in place at 83000000, end 8301343c
 
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc1-next-20160809-110829-g09f0834 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Aug 8 21:05:02 PDT 2016
+Linux version 4.8.0-rc1-next-20160811-111289-gc0a5420 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Aug 10 21:05:04 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -64,12 +64,12 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141d2c0, node_mem_map eeffa000
+free_area_init_node: node 0, pgdat c141da80, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 14 pages/cpu @eef87000 s26624 r8192 d22528 u57344
+percpu: Embedded 14 pages/cpu @eef85000 s26624 r8192 d22528 u57344
 pcpu-alloc: s26624 r8192 d22528 u57344 alloc=14*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
@@ -77,7 +77,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1994980K/2096128K available (9216K kernel code, 1148K rwdata, 3976K rodata, 2048K init, 343K bss, 35612K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1994972K/2096128K available (9216K kernel code, 1150K rwdata, 3976K rodata, 2048K init, 343K bss, 35620K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -87,8 +87,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141f140   (1149 kB)
-       .bss : 0xc1421000 - 0xc1476c08   ( 344 kB)
+      .data : 0xc1300000 - 0xc141f900   (1151 kB)
+       .bss : 0xc1421000 - 0xc1476cc8   ( 344 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -130,7 +130,8 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
+No ATAGs?
+hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 8 bytes.
 Serial: AMBA PL011 UART driver
 +5V_SYS: supplied by +VDD_MUX
@@ -193,13 +194,13 @@
 tegra-pmc 7000e400.pmc: emergency thermal reset enabled
 Serial: 8250/16550 driver, 4 ports, IRQ sharing disabled
 console [ttyS0] disabled
-70006300.serial: ttyS0 at MMIO 0x70006300 (irq = 335, base_baud = 25500000) is a Tegra
+70006300.serial: ttyS0 at MMIO 0x70006300 (irq = 333, base_baud = 25500000) is a Tegra
 console [ttyS0] enabled
 SuperH (H)SCI(F) driver initialized
 msm_serial: driver initialized
 STMicroelectronics ASC driver initialized
-70006000.serial: ttyTHS1 at MMIO 0x70006000 (irq = 333, base_baud = 0) is a TEGRA_UART
-70006040.serial: ttyTHS2 at MMIO 0x70006040 (irq = 334, base_baud = 0) is a TEGRA_UART
+serial-tegra serial@0,70006000: failed to get alias id, errno -19
+serial-tegra serial@0,70006040: failed to get alias id, errno -19
 [drm] Initialized drm 1.1.0 20060810
 tegra-hdmi 54280000.hdmi: failed to get HDMI regulator
 brd: module loaded
@@ -253,7 +254,7 @@
 tegra-ehci: Tegra EHCI driver
 tegra-ehci 7d004000.usb: EHCI Host Controller
 tegra-ehci 7d004000.usb: new USB bus registered, assigned bus number 1
-tegra-ehci 7d004000.usb: irq 354, io mem 0x7d004000
+tegra-ehci 7d004000.usb: irq 352, io mem 0x7d004000
 tegra-ehci 7d004000.usb: USB 2.0 started, EHCI 1.10
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
@@ -267,7 +268,7 @@
 usbcore: registered new interface driver usb-storage
 mousedev: PS/2 mouse device common for all mice
 as3722-rtc as3722-rtc: rtc core: registered as3722-rtc as rtc0
-as3722-rtc as3722-rtc: RTC interrupt 377
+as3722-rtc as3722-rtc: RTC interrupt 375
 tegra_rtc 7000e000.rtc: rtc core: registered 7000e000.rtc as rtc1
 tegra_rtc 7000e000.rtc: Tegra internal Real Time Clock
 i2c /dev entries driver
@@ -300,12 +301,6 @@
 +3.3V_AVDD_HDMI_AP_GATED: supplied by +3.3V_RUN
 +USB0_VBUS_SW: supplied by +5V_SYS
 +5V_USB_HS: supplied by +5V_SYS
-mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
- mmcblk0: p1
 +3.3V_LP0: supplied by +3.3V_SYS
 +5V_HDMI_CON: supplied by +5V_SYS
 +5V_SATA: supplied by +5V_SYS
@@ -316,6 +311,7 @@
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
 tegra-pcie 1003000.pcie-controller: Failed to get supply 'avddio-pex': -517
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
+mmc0: new high speed MMC card at address 0001
 as3722-sd2: supplied by +5V_SYS
 as3722-sd3: supplied by +5V_SYS
 as3722-sd4: supplied by +5V_SYS
@@ -340,11 +336,11 @@
 tegra-ahci 70027000.sata: flags: 64bit ncq sntf pm led pio slum part sadm sds apst 
 scsi host0: tegra-ahci
 scsi host1: tegra-ahci
-ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 345
+ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 343
 ata2: DUMMY
 tegra-ehci 7d008000.usb: EHCI Host Controller
 tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 2
-tegra-ehci 7d008000.usb: irq 355, io mem 0x7d008000
+tegra-ehci 7d008000.usb: irq 353, io mem 0x7d008000
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
@@ -373,7 +369,6 @@
 pci 0000:01:00.0: [8086:4232] type 00 class 0x028000
 pci 0000:01:00.0: reg 0x10: [mem 0x00000000-0x00001fff 64bit]
 pci 0000:01:00.0: PME# supported from D0 D3hot D3cold
-ata1: SATA link down (SStatus 0 SControl 300)
 PCI: bus1: Fast back to back transfers disabled
 pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
 pci 0000:02:00.0: [10ec:8168] type 00 class 0x020000
@@ -410,15 +405,21 @@
 pcie_pme 0000:00:02.0:pcie001: service driver pcie_pme loaded
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
-r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0a50000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 395
+r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0a46000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 393
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
 drm drm: No connectors reported connected with modes
 [drm] Cannot find any crtc or sizes - going 1024x768
+ata1: SATA link down (SStatus 0 SControl 300)
 mmc1: new high speed SDHC card at address aaaa
 mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
  mmcblk1: p1 p2 p3
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+ mmcblk0: p1
 Console: switching to colour frame buffer device 128x48
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
@@ -426,7 +427,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:12 UTC (946684812)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:28 UTC (946684948)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
@@ -438,7 +439,7 @@
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:1.
+VFS: Mounted root (ext3 filesystem) on device 179:17.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-08-10 21:13:14.535745403 -0700
+++ /tmp/foob	2016-08-10 21:13:14.539745406 -0700
@@ -32,29 +32,29 @@
 	 #################################################################
 	 #################################################################
 	 ############################################
-	 911.1 KiB/s
+	 4 MiB/s
 done
-Bytes transferred = 5413280 (5299a0 hex)
+Bytes transferred = 5415608 (52a2b8 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 913.1 KiB/s
+	 3.7 MiB/s
 done
-Bytes transferred = 66405 (10365 hex)
+Bytes transferred = 66621 (1043d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5299a0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x52a2b8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
-   Using Device Tree in place at 83000000, end 83013364
+   Using Device Tree in place at 83000000, end 8301343c
 
 Starting kernel ...
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc1-next-20160809-110829-g09f0834 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Aug 8 21:04:56 PDT 2016
+Linux version 4.8.0-rc1-next-20160811-111289-gc0a5420 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Aug 10 21:04:56 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -64,12 +64,12 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0d5c180, node_mem_map eeff9000
+free_area_init_node: node 0, pgdat c0d5c3c0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 14 pages/cpu @eef86000 s26176 r8192 d22976 u57344
+percpu: Embedded 14 pages/cpu @eef84000 s26176 r8192 d22976 u57344
 pcpu-alloc: s26176 r8192 d22976 u57344 alloc=14*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
@@ -77,7 +77,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999824K/2096128K available (8192K kernel code, 375K rwdata, 2444K rodata, 1024K init, 366K bss, 30768K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1999816K/2096128K available (8192K kernel code, 375K rwdata, 2432K rodata, 1024K init, 366K bss, 30776K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -87,8 +87,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d5dc00   ( 375 kB)
-       .bss : 0xc0d5f000 - 0xc0dbaad8   ( 367 kB)
+      .data : 0xc0d00000 - 0xc0d5de40   ( 376 kB)
+       .bss : 0xc0d5f000 - 0xc0dbab18   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -127,7 +127,8 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
+No ATAGs?
+hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 8 bytes.
 +5V_SYS: supplied by +VDD_MUX
 +3.3V_SYS: supplied by +VDD_MUX
@@ -191,13 +192,13 @@
 tegra-pmc 7000e400.pmc: emergency thermal reset enabled
 Serial: 8250/16550 driver, 4 ports, IRQ sharing disabled
 console [ttyS0] disabled
-70006300.serial: ttyS0 at MMIO 0x70006300 (irq = 335, base_baud = 25500000) is a Tegra
+70006300.serial: ttyS0 at MMIO 0x70006300 (irq = 333, base_baud = 25500000) is a Tegra
 console [ttyS0] enabled
 console [ttyS0] enabled
 bootconsole [earlycon0] disabled
 bootconsole [earlycon0] disabled
-70006000.serial: ttyTHS1 at MMIO 0x70006000 (irq = 333, base_baud = 0) is a TEGRA_UART
-70006040.serial: ttyTHS2 at MMIO 0x70006040 (irq = 334, base_baud = 0) is a TEGRA_UART
+serial-tegra serial@0,70006000: failed to get alias id, errno -19
+serial-tegra serial@0,70006040: failed to get alias id, errno -19
 [drm] Initialized drm 1.1.0 20060810
 tegra-hdmi 54280000.hdmi: failed to get HDMI regulator
 loop: module loaded
@@ -243,7 +244,7 @@
 tegra-ehci: Tegra EHCI driver
 tegra-ehci 7d004000.usb: EHCI Host Controller
 tegra-ehci 7d004000.usb: new USB bus registered, assigned bus number 1
-tegra-ehci 7d004000.usb: irq 354, io mem 0x7d004000
+tegra-ehci 7d004000.usb: irq 352, io mem 0x7d004000
 tegra-ehci 7d004000.usb: USB 2.0 started, EHCI 1.10
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
@@ -253,7 +254,7 @@
 usbcore: registered new interface driver usb-storage
 mousedev: PS/2 mouse device common for all mice
 as3722-rtc as3722-rtc: rtc core: registered as3722-rtc as rtc0
-as3722-rtc as3722-rtc: RTC interrupt 377
+as3722-rtc as3722-rtc: RTC interrupt 375
 random: fast init done
 tegra_rtc 7000e000.rtc: rtc core: registered 7000e000.rtc as rtc1
 tegra_rtc 7000e000.rtc: Tegra internal Real Time Clock
@@ -344,11 +345,11 @@
 tegra-ahci 70027000.sata: flags: 64bit ncq sntf pm led pio slum part sadm sds apst 
 scsi host0: tegra-ahci
 scsi host1: tegra-ahci
-ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 345
+ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 343
 ata2: DUMMY
 tegra-ehci 7d008000.usb: EHCI Host Controller
 tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 2
-tegra-ehci 7d008000.usb: irq 355, io mem 0x7d008000
+tegra-ehci 7d008000.usb: irq 353, io mem 0x7d008000
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
@@ -356,7 +357,7 @@
 tegra-xusb 70090000.usb: xHCI Host Controller
 tegra-xusb 70090000.usb: new USB bus registered, assigned bus number 3
 tegra-xusb 70090000.usb: hcc params 0x0184f525 hci version 0x100 quirks 0x00010010
-tegra-xusb 70090000.usb: irq 347, io mem 0x70090000
+tegra-xusb 70090000.usb: irq 345, io mem 0x70090000
 hub 3-0:1.0: USB hub found
 hub 3-0:1.0: 6 ports detected
 tegra-xusb 70090000.usb: xHCI Host Controller
@@ -369,9 +370,9 @@
 mmc1: Unknown controller version (3). You may experience problems.
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
+ata1: SATA link down (SStatus 0 SControl 300)
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
 tegra-pcie 1003000.pcie-controller: probing port 0, using 2 lanes
-ata1: SATA link down (SStatus 0 SControl 300)
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
 tegra-pcie 1003000.pcie-controller: probing port 1, using 1 lanes
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000000
@@ -407,9 +408,9 @@
 pci 0000:00:02.0: BAR 9: assigned [mem 0x20000000-0x200fffff 64bit pref]
 pci 0000:00:02.0: BAR 7: assigned [io  0x1000-0x1fff]
 pci 0000:01:00.0: BAR 0: assigned [mem 0x13000000-0x13001fff 64bit]
+ mmcblk1: p1 p2 p3
 pci 0000:00:01.0: PCI bridge to [bus 01]
 pci 0000:00:01.0:   bridge window [mem 0x13000000-0x130fffff]
- mmcblk1: p1 p2 p3
 pci 0000:02:00.0: BAR 4: assigned [mem 0x20000000-0x20003fff 64bit pref]
 pci 0000:02:00.0: BAR 2: assigned [mem 0x13100000-0x13100fff 64bit]
 pci 0000:02:00.0: BAR 0: assigned [io  0x1000-0x10ff]
@@ -429,7 +430,7 @@
 pcie_pme 0000:00:02.0:pcie001: service driver pcie_pme loaded
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
-r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf09fa000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 395
+r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf09fa000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 393
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
@@ -441,7 +442,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:22 UTC (946684822)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:16 UTC (946684816)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
@@ -451,7 +452,7 @@
 +2.8V_RUN_CAM_AF: disabling
 +1.8V_RUN_VPP_FUSE: disabling
 ALSA device list:
-  #0: tegra-hda at 0x70038000 irq 346
+  #0: tegra-hda at 0x70038000 irq 344
   #1: NVIDIA Tegra Jetson TK1
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk0p1): recovery complete
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-08-10 21:13:14.571745432 -0700
+++ /tmp/foob	2016-08-10 21:13:14.571745432 -0700
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -40,21 +40,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #############################
-	 3.7 MiB/s
+	 ##############################
+	 3.3 MiB/s
 done
-Bytes transferred = 7095216 (6c43b0 hex)
+Bytes transferred = 7107304 (6c72e8 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.4 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 66821 (10505 hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6c43b0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x6c72e8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013504
@@ -62,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc1-next-20160809-110829-g09f0834 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Aug 8 21:05:02 PDT 2016
+Linux version 4.8.0-rc1-next-20160811-111289-gc0a5420 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Aug 10 21:05:04 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -72,12 +72,12 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c141d2c0, node_mem_map eeffa000
+free_area_init_node: node 0, pgdat c141da80, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327679 pages, LIFO batch:31
-percpu: Embedded 14 pages/cpu @eef87000 s26624 r8192 d22528 u57344
+percpu: Embedded 14 pages/cpu @eef85000 s26624 r8192 d22528 u57344
 pcpu-alloc: s26624 r8192 d22528 u57344 alloc=14*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522751
@@ -85,7 +85,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1996000K/2097148K available (9216K kernel code, 1148K rwdata, 3976K rodata, 2048K init, 343K bss, 35612K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1995992K/2097148K available (9216K kernel code, 1150K rwdata, 3976K rodata, 2048K init, 343K bss, 35620K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -95,8 +95,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141f140   (1149 kB)
-       .bss : 0xc1421000 - 0xc1476c08   ( 344 kB)
+      .data : 0xc1300000 - 0xc141f900   (1151 kB)
+       .bss : 0xc1421000 - 0xc1476cc8   ( 344 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -138,7 +138,8 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
+No ATAGs?
+hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 8 bytes.
 Serial: AMBA PL011 UART driver
 tegra-mc 70019000.memory-controller: no memory timings for RAM code 0 registered
@@ -304,24 +305,23 @@
 +3.3V_RUN: supplied by +3.3V_SYS
 +3.3V_AVDD_HDMI_AP_GATED: supplied by +3.3V_RUN
 +1.8V_RUN_CAM: supplied by +3.3V_RUN
-input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 +VDDIO_SDMMC3: supplied by +3.3V_RUN
 +1.8V_RUN_VPP_FUSE: supplied by +3.3V_RUN
+input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 +VDD_LED: supplied by +VDD_MUX
 +5V_VDD_TS_SW: supplied by +5V_SYS
 +5V_USB_HS: supplied by +5V_SYS
 +5V_USB_SS: supplied by +5V_SYS
 mmc0: new high speed MMC card at address 0001
 +3.3V_PANEL: supplied by +3.3V_RUN
-+3.3V_LP0: supplied by +3.3V_SYS
 mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
 mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
 mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
++3.3V_LP0: supplied by +3.3V_SYS
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 elan_i2c 1-0015: failed to read report data: -121
-elan_i2c 1-0015: failed to read report data: -121
 +5V_HDMI_CON: supplied by +5V_SYS
 usb3-0 supply vbus not found, using dummy regulator
 usb3-1 supply vbus not found, using dummy regulator
@@ -353,7 +353,7 @@
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:22 UTC (946684822)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 19:21:17 UTC (946754477)
 +1.8V_RUN_CAM: disabling
 +3.3V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2016-08-10 21:13:14.591745447 -0700
+++ /tmp/foob	2016-08-10 21:13:14.591745447 -0700
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -36,14 +36,12 @@
 Loading: *#################################################################
 	 #################################################################
 	 #################################################################
-	 ################################################################EHCI timed out on TD - token=0x88008d80
-T EHCI timed out on TD - token=0x88008d80
-T #
+	 #################################################################
 	 #################################################################
 	 ############################################
-	 390.6 KiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 5413280 (5299a0 hex)
+Bytes transferred = 5415608 (52a2b8 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -54,7 +52,7 @@
 done
 Bytes transferred = 66821 (10505 hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5299a0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x52a2b8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013504
@@ -175,11 +173,11 @@
 +1.8V_RUN_CAM: supplied by +3.3V_RUN
 +VDDIO_SDMMC3: supplied by +3.3V_RUN
 +1.8V_RUN_VPP_FUSE: supplied by +3.3V_RUN
-input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input3
 +VDD_LED: supplied by +VDD_MUX
 +5V_VDD_TS_SW: supplied by +5V_SYS
 +5V_USB_HS: supplied by +5V_SYS
 +5V_USB_SS: supplied by +5V_SYS
+input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input3
 +3.3V_PANEL: supplied by +3.3V_RUN
 +3.3V_LP0: supplied by +3.3V_SYS
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
@@ -216,17 +214,17 @@
 usb 1-2: new high-speed USB device number 2 using tegra-xusb
 mmc2: new high speed SDHC card at address 0007
 mmcblk2: mmc2:0007 SD08G 7.42 GiB 
+Console: switching to colour frame buffer device 170x48
 GPT:Primary header alternate_lba != Alt. header my_lba
 GPT:3014591 != 15564799
 GPT:Primary header thinks Alt. header is not at the end of the disk.
 GPT:3014591 != 15564799
 GPT: Use GNU Parted to correct GPT errors.
  mmcblk2: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
-Console: switching to colour frame buffer device 170x48
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input4
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:36 UTC (946684836)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 19:20:14 UTC (946754414)
 +1.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
 +1.05V_RUN_CAM_REAR: disabling
@@ -240,7 +238,6 @@
 EXT4-fs (mmcblk0p3): couldn't mount as ext3 due to feature incompatibilities
 uvcvideo: Found UVC 1.00 device HD WebCam (1bcf:2c67)
 input: HD WebCam as /devices/soc0/70090000.usb/usb1/1-2/1-2:1.0/input/input5
-usb 1-3: new high-speed USB device number 3 using tegra-xusb
 EXT4-fs (mmcblk0p3): recovery complete
 EXT4-fs (mmcblk0p3): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:3.
@@ -248,11 +245,11 @@
 Freeing unused kernel memory: 1024K (c0c00000 - c0d00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    6.787559] asix 1-3:1.0 eth0: register 'asix' at usb-70090000.usb-3, ASIX AX88772 USB 2.0 Ethernet, c8:d7:19:90:fa:65
+root@(none):/# [    5.674743] usb 1-3: new high-speed USB device number 3 using tegra-xusb
+asix 1-3:1.0 eth0: register 'asix' at usb-70090000.usb-3, ASIX AX88772 USB 2.0 Ethernet, c8:d7:19:90:fa:65
 usb 1-1: new high-speed USB device number 4 using tegra-xusb
 usb-storage 1-1:1.0: USB Mass Storage device detected
 scsi host0: usb-storage 1-1:1.0
-scsi host0: runtime PM trying to activate child device host0 but parent (1-1:1.0) is not active
 scsi 0:0:0:0: Direct-Access     SanDisk  SDDR-113         9412 PQ: 0 ANSI: 0
 sd 0:0:0:0: [sda] 15523840 512-byte logical blocks: (7.95 GB/7.40 GiB)
 sd 0:0:0:0: [sda] Write Protect is off
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2016-08-10 21:13:14.615745466 -0700
+++ /tmp/foob	2016-08-10 21:13:14.619745469 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1800551 us
+Bootloader-Cpu Init at (time stamp): 1800200 us
 
 Pinmux changes applied in kernel way
 
@@ -59,25 +59,25 @@
 The proc BoardInfo: 0x0176:0x0001:0x01:0x43:0x00
 FastbootAddAtag: setting ATAG (0)
 
-Board Id Read Time: 3 us
+Board Id Read Time: 4 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1800525  us
-Bootloader (AosInit) time: 733470 us
-Bootloader Main Init time: 180194 us
-BlockDevMgrInit time: 157553 us
+Miscellaneous avp time: 1800173  us
+Bootloader (AosInit) time: 733499 us
+Bootloader Main Init time: 180279 us
+BlockDevMgrInit time: 157559 us
 PartMgrInit time: 8 us
 StorMgrInit time: 30 us
-FsMgrInit time: 3 us
-Partition Table load: 77109 us 
-Display init time: 574331 us
-Kernel read time: 370 msec
+FsMgrInit time: 2 us
+Partition Table load: 77030 us 
+Display init time: 574437 us
+Kernel read time: 369 msec
 Load Kernel Size:   16068608 Bytes
-Bl Kernel ReadSpeed: 43428000 Bytes/sec
-Pre-Kernel Jump time: 1717766 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5811901 us
+Bl Kernel ReadSpeed: 43546000 Bytes/sec
+Pre-Kernel Jump time: 1717005 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5811147 us
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc1-next-20160809+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Aug 8 21:08:31 PDT 2016
+Linux version 4.8.0-rc1-next-20160811+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Aug 10 21:09:15 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
 Memory limited to 1734MB
 efi: Getting EFI parameters from FDT:
@@ -91,15 +91,15 @@
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 Dentry cache hash table entries: 262144 (order: 9, 2097152 bytes)
 Inode-cache hash table entries: 131072 (order: 8, 1048576 bytes)
-Memory: 1706952K/1775616K available (7868K kernel code, 777K rwdata, 3692K rodata, 960K init, 271K bss, 52280K reserved, 16384K cma-reserved)
+Memory: 1706952K/1775616K available (7868K kernel code, 777K rwdata, 3696K rodata, 960K init, 270K bss, 52280K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008830000   (  7872 KB)
     .rodata : 0xffff000008830000 - 0xffff000008bd0000   (  3712 KB)
       .init : 0xffff000008bd0000 - 0xffff000008cc0000   (   960 KB)
-      .data : 0xffff000008cc0000 - 0xffff000008d82400   (   777 KB)
-       .bss : 0xffff000008d82400 - 0xffff000008dc6038   (   272 KB)
+      .data : 0xffff000008cc0000 - 0xffff000008d82600   (   778 KB)
+       .bss : 0xffff000008d82600 - 0xffff000008dc60b8   (   271 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -184,7 +184,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(1.107:1): initialized
+audit: type=2000 audit(1.099:1): initialized
 workingset: timestamp_bits=46 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -251,8 +251,6 @@
 mmc0: new high speed MMC card at address 0001
 registered taskstats version 1
 +VDD_LED: supplied by +VDD_MUX
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 +5V_USB_HS: supplied by +5V_SYS
 +5V_USB_SS: supplied by +5V_SYS
 +5V_HDMI_CON: supplied by +5V_SYS
@@ -271,6 +269,8 @@
 ALSA device list:
   No soundcards found.
 Freeing unused kernel memory: 960K (ffff800000bd0000 - ffff800000cc0000)
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-08-10 21:13:14.635745482 -0700
+++ /tmp/foob	2016-08-10 21:13:14.635745482 -0700
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -46,21 +46,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #############################
-	 909.2 KiB/s
+	 ##############################
+	 903.3 KiB/s
 done
-Bytes transferred = 7095216 (6c43b0 hex)
+Bytes transferred = 7107304 (6c72e8 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 898.4 KiB/s
+	 861.3 KiB/s
 done
 Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x6c43b0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x6c72e8 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102b3
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc1-next-20160809-110829-g09f0834 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Aug 8 21:05:02 PDT 2016
+Linux version 4.8.0-rc1-next-20160811-111289-gc0a5420 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Aug 10 21:05:04 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: Compulab TrimSlice board
@@ -76,12 +76,12 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c141d2c0, node_mem_map ef7fa000
+free_area_init_node: node 0, pgdat c141da80, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 65536 pages, LIFO batch:15
-percpu: Embedded 14 pages/cpu @ef7c9000 s26624 r8192 d22528 u57344
+percpu: Embedded 14 pages/cpu @ef7c7000 s26624 r8192 d22528 u57344
 pcpu-alloc: s26624 r8192 d22528 u57344 alloc=14*4096
 pcpu-alloc: [0] 0 [0] 1 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 260608
@@ -89,7 +89,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 955924K/1048576K available (9216K kernel code, 1148K rwdata, 3976K rodata, 2048K init, 343K bss, 27116K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 955916K/1048576K available (9216K kernel code, 1150K rwdata, 3976K rodata, 2048K init, 343K bss, 27124K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -99,8 +99,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141f140   (1149 kB)
-       .bss : 0xc1421000 - 0xc1476c08   ( 344 kB)
+      .data : 0xc1300000 - 0xc141f900   (1151 kB)
+       .bss : 0xc1421000 - 0xc1476cc8   ( 344 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -139,7 +139,8 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
+No ATAGs?
+hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 4 bytes.
 Serial: AMBA PL011 UART driver
 usb1_vbus: Failed to request enable GPIO170: -517
@@ -322,28 +323,29 @@
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 mmc0: host does not support reading read-only switch, assuming write-enable
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
  mmcblk0: p1
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
-mmc1: new high speed SD card at address b368
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
+rtc-em3027 2-0056: setting system clock to 2015-02-05 15:29:02 UTC (1423150142)
+EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
+mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
-rtc-em3027 2-0056: setting system clock to 2015-02-05 14:57:03 UTC (1423148223)
  mmcblk1: p1 p2
-EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 EXT4-fs (mmcblk0p1): recovery complete
+random: fast init done
+asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
-random: fast init done
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-08-10 21:13:14.643745488 -0700
+++ /tmp/foob	2016-08-10 21:13:14.643745488 -0700
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -45,20 +45,20 @@
 	 #################################################################
 	 #################################################################
 	 ############################################
-	 913.1 KiB/s
+	 916 KiB/s
 done
-Bytes transferred = 5413280 (5299a0 hex)
+Bytes transferred = 5415608 (52a2b8 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 861.3 KiB/s
+	 826.2 KiB/s
 done
 Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x5299a0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x52a2b8 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102b3
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc1-next-20160809-110829-g09f0834 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Aug 8 21:04:56 PDT 2016
+Linux version 4.8.0-rc1-next-20160811-111289-gc0a5420 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Aug 10 21:04:56 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: Compulab TrimSlice board
@@ -76,12 +76,12 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0d5c180, node_mem_map ef7f9000
+free_area_init_node: node 0, pgdat c0d5c3c0, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 65536 pages, LIFO batch:15
-percpu: Embedded 14 pages/cpu @ef7c8000 s26176 r8192 d22976 u57344
+percpu: Embedded 14 pages/cpu @ef7c6000 s26176 r8192 d22976 u57344
 pcpu-alloc: s26176 r8192 d22976 u57344 alloc=14*4096
 pcpu-alloc: [0] 0 [0] 1 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 260608
@@ -89,7 +89,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 960768K/1048576K available (8192K kernel code, 375K rwdata, 2444K rodata, 1024K init, 366K bss, 22272K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 960760K/1048576K available (8192K kernel code, 375K rwdata, 2432K rodata, 1024K init, 366K bss, 22280K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -99,8 +99,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d5dc00   ( 375 kB)
-       .bss : 0xc0d5f000 - 0xc0dbaad8   ( 367 kB)
+      .data : 0xc0d00000 - 0xc0d5de40   ( 376 kB)
+       .bss : 0xc0d5f000 - 0xc0dbab18   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
@@ -138,7 +138,8 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
+No ATAGs?
+hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 4 bytes.
 usb1_vbus: Failed to request enable GPIO170: -517
 reg-fixed-voltage regulators:regulator@2: Failed to register regulator: -517
@@ -322,10 +323,10 @@
 NET: Registered protocol family 17
 NET: Registered protocol family 15
 can: controller area network core (rev 20120528 abi 9)
-usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 NET: Registered protocol family 29
 can: raw protocol (rev 20120528)
 can: broadcast manager protocol (rev 20160617 t)
+usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 can: netlink gateway (rev 20130117) max_hops=1
 Bluetooth: RFCOMM socket layer initialized
 Bluetooth: RFCOMM ver 1.11
@@ -335,7 +336,6 @@
 Bluetooth: HIDP socket layer initialized
 Key type dns_resolver registered
 Registering SWP/SWPB emulation handler
-utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
@@ -343,7 +343,7 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2015-02-05 14:58:31 UTC (1423148311)
+rtc-em3027 2-0056: setting system clock to 2015-02-05 15:27:29 UTC (1423150049)
 ALSA device list:
   #0: tegra-trimslice
 asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2016-08-10 21:13:14.667745508 -0700
+++ /tmp/foob	2016-08-10 21:13:14.667745508 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 726723 us
-Basic modules init took 349207 us
-Sec Bootdevice Read Time = 205 ms, Read Size = 8459 KB
-Next stage binary read took 16522 us
-Carveout took 252725 us
-CPU initialization took 92793 us
-Total time taken by TegraBoot 711247 us
+Tegraboot started after 736094 us
+Basic modules init took 348461 us
+Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
+Next stage binary read took 14390 us
+Carveout took 253246 us
+CPU initialization took 92308 us
+Total time taken by TegraBoot 708405 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.569] LPDDR4 Training: Number of tables = 10
+64b[0001.575] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-13640704 bytes read in 764 ms (17 MiB/s)
+13641216 bytes read in 765 ms (17 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-41182 bytes read in 453 ms (87.9 KiB/s)
+41182 bytes read in 454 ms (87.9 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 495 ms (3.1 MiB/s)
+1627787 bytes read in 494 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc1-next-20160809+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Aug 8 21:08:31 PDT 2016
+Linux version 4.8.0-rc1-next-20160811+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Aug 10 21:09:15 PDT 2016
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -166,15 +166,15 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffff80007a000000-ffff80007dffffff]
-Memory: 2979312K/3131392K available (7868K kernel code, 777K rwdata, 3692K rodata, 960K init, 271K bss, 135696K reserved, 16384K cma-reserved)
+Memory: 2979312K/3131392K available (7868K kernel code, 777K rwdata, 3696K rodata, 960K init, 270K bss, 135696K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008830000   (  7872 KB)
     .rodata : 0xffff000008830000 - 0xffff000008bd0000   (  3712 KB)
       .init : 0xffff000008bd0000 - 0xffff000008cc0000   (   960 KB)
-      .data : 0xffff000008cc0000 - 0xffff000008d82400   (   777 KB)
-       .bss : 0xffff000008d82400 - 0xffff000008dc6038   (   272 KB)
+      .data : 0xffff000008cc0000 - 0xffff000008d82600   (   778 KB)
+       .bss : 0xffff000008d82600 - 0xffff000008dc60b8   (   271 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -346,6 +346,6 @@
 devtmpfs: mounted
 Freeing unused kernel memory: 960K (ffff800000bd0000 - ffff800000cc0000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
-bash: no job control in th[    1.342375] random: fast init done
-is shell
+bash: no job contr[    1.343877] random: fast init done
+ol in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2016-08-10 21:13:14.671745511 -0700
+++ /tmp/foob	2016-08-10 21:13:14.675745513 -0700
@@ -1,4 +1,4 @@
-: uart0 at MMIO 0x0000000070006000 (options '115200n8')
+0000] earlycon: uart0 at MMIO 0x0000000070006000 (options '115200n8')
 bootconsole [uart0] enabled
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
@@ -22,15 +22,15 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2833508K/2988032K available (7868K kernel code, 777K rwdata, 3692K rodata, 960K init, 271K bss, 138140K reserved, 16384K cma-reserved)
+Memory: 2833508K/2988032K available (7868K kernel code, 777K rwdata, 3696K rodata, 960K init, 270K bss, 138140K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008830000   (  7872 KB)
     .rodata : 0xffff000008830000 - 0xffff000008bd0000   (  3712 KB)
       .init : 0xffff000008bd0000 - 0xffff000008cc0000   (   960 KB)
-      .data : 0xffff000008cc0000 - 0xffff000008d82400   (   777 KB)
-       .bss : 0xffff000008d82400 - 0xffff000008dc6038   (   272 KB)
+      .data : 0xffff000008cc0000 - 0xffff000008d82600   (   778 KB)
+       .bss : 0xffff000008d82600 - 0xffff000008dc60b8   (   271 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -59,7 +59,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x1a10: expected 0x2004f400
+boot misc3 0x2bb8: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -72,7 +72,7 @@
 console [tty0] enabled
 bootconsole [uart0] disabled
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc1-next-20160809+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Aug 8 21:08:31 PDT 2016
+Linux version 4.8.0-rc1-next-20160811+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Aug 10 21:09:15 PDT 2016
 Boot CPU: AArch64 Processor [411fd071]
 earlycon: uart0 at MMIO 0x0000000070006000 (options '115200n8')
 bootconsole [uart0] enabled
@@ -98,15 +98,15 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2833508K/2988032K available (7868K kernel code, 777K rwdata, 3692K rodata, 960K init, 271K bss, 138140K reserved, 16384K cma-reserved)
+Memory: 2833508K/2988032K available (7868K kernel code, 777K rwdata, 3696K rodata, 960K init, 270K bss, 138140K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008830000   (  7872 KB)
     .rodata : 0xffff000008830000 - 0xffff000008bd0000   (  3712 KB)
       .init : 0xffff000008bd0000 - 0xffff000008cc0000   (   960 KB)
-      .data : 0xffff000008cc0000 - 0xffff000008d82400   (   777 KB)
-       .bss : 0xffff000008d82400 - 0xffff000008dc6038   (   272 KB)
+      .data : 0xffff000008cc0000 - 0xffff000008d82600   (   778 KB)
+       .bss : 0xffff000008d82600 - 0xffff000008dc60b8   (   271 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -135,7 +135,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x1a10: expected 0x2004f400
+boot misc3 0x2bb8: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -221,7 +221,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.287:1): initialized
+audit: type=2000 audit(0.279:1): initialized
 workingset: timestamp_bits=46 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -292,12 +292,12 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-max77686-rtc max77620-rtc: setting system clock to 2000-05-20 09:38:59 UTC (958815539)
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 BGND3R 29.1 GiB 
 mmcblk0boot0: mmc0:0001 BGND3R partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB
+max77686-rtc max77620-rtc: setting system clock to 2000-05-22 09:39:27 UTC (958988367)
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
 USBC_VBUS: disabling
 PP1200_AVDD: disabling
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-08-10 21:13:14.679745516 -0700
+++ /tmp/foob	2016-08-10 21:13:14.683745520 -0700
@@ -38,21 +38,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #############################
-	 3.4 MiB/s
+	 ##############################
+	 3.1 MiB/s
 done
-Bytes transferred = 7095216 (6c43b0 hex)
+Bytes transferred = 7107304 (6c72e8 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.3 MiB/s
+	 2.6 MiB/s
 done
 Bytes transferred = 51572 (c974 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6c43b0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6c72e8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff973 ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc1-next-20160809-110829-g09f0834 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Aug 8 21:05:02 PDT 2016
+Linux version 4.8.0-rc1-next-20160811-111289-gc0a5420 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Aug 10 21:05:04 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,12 +68,12 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141d2c0, node_mem_map eeffa000
+free_area_init_node: node 0, pgdat c141da80, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 14 pages/cpu @eef8d000 s26624 r8192 d22528 u57344
+percpu: Embedded 14 pages/cpu @eef8b000 s26624 r8192 d22528 u57344
 pcpu-alloc: s26624 r8192 d22528 u57344 alloc=14*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
@@ -81,7 +81,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1995020K/2096128K available (9216K kernel code, 1148K rwdata, 3976K rodata, 2048K init, 343K bss, 35572K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1995012K/2096128K available (9216K kernel code, 1150K rwdata, 3976K rodata, 2048K init, 343K bss, 35580K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -91,8 +91,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141f140   (1149 kB)
-       .bss : 0xc1421000 - 0xc1476c08   ( 344 kB)
+      .data : 0xc1300000 - 0xc141f900   (1151 kB)
+       .bss : 0xc1421000 - 0xc1476cc8   ( 344 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -133,7 +133,8 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
+No ATAGs?
+hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 4 bytes.
 Serial: AMBA PL011 UART driver
 tegra-mc 7000f000.memory-controller: no memory timings for RAM code 3 registered
@@ -299,19 +300,19 @@
 sys_3v3,vdd_3v3_alw: supplied by vdd_5v_in
 +VDD_5V_HDMI: supplied by sys_3v3,vdd_3v3_alw
 sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
+mmc1: new high speed MMC card at address 0001
+mmc0: new ultra high speed DDR50 SDHC card at address aaaa
 tegra-pcie 3000.pcie-controller: 2x3 configuration
 tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
-mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM16G 14.8 GiB 
+mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
 mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB
 Primary GPT is invalid, using alternate GPT.
  mmcblk1: p1 p2 p3 p4 p5
-mmc0: new ultra high speed DDR50 SDHC card at address aaaa
-mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
- mmcblk0: p1 p2 p3
 tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
+ mmcblk0: p1 p2 p3
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -382,4 +383,4 @@
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.869373] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.879020] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-08-10 21:13:14.687745523 -0700
+++ /tmp/foob	2016-08-10 21:13:14.691745526 -0700
@@ -37,9 +37,9 @@
 	 #################################################################
 	 #################################################################
 	 ############################################
-	 3 MiB/s
+	 2.9 MiB/s
 done
-Bytes transferred = 5413280 (5299a0 hex)
+Bytes transferred = 5415608 (52a2b8 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 51572 (c974 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5299a0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x52a2b8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff973 ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc1-next-20160809-110829-g09f0834 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Aug 8 21:04:56 PDT 2016
+Linux version 4.8.0-rc1-next-20160811-111289-gc0a5420 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Aug 10 21:04:56 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,12 +68,12 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0d5c180, node_mem_map eeff9000
+free_area_init_node: node 0, pgdat c0d5c3c0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 14 pages/cpu @eef8c000 s26176 r8192 d22976 u57344
+percpu: Embedded 14 pages/cpu @eef8a000 s26176 r8192 d22976 u57344
 pcpu-alloc: s26176 r8192 d22976 u57344 alloc=14*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
@@ -81,7 +81,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999864K/2096128K available (8192K kernel code, 375K rwdata, 2444K rodata, 1024K init, 366K bss, 30728K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1999856K/2096128K available (8192K kernel code, 375K rwdata, 2432K rodata, 1024K init, 366K bss, 30736K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -91,8 +91,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d5dc00   ( 375 kB)
-       .bss : 0xc0d5f000 - 0xc0dbaad8   ( 367 kB)
+      .data : 0xc0d00000 - 0xc0d5de40   ( 376 kB)
+       .bss : 0xc0d5f000 - 0xc0dbab18   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -130,7 +130,8 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
+No ATAGs?
+hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 4 bytes.
 tegra-mc 7000f000.memory-controller: no memory timings for RAM code 3 registered
 vdd_5v_sata: Failed to request enable GPIO30: -517
@@ -275,10 +276,10 @@
 mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB
-mmc0: new ultra high speed DDR50 SDHC card at address aaaa
-mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
 Primary GPT is invalid, using alternate GPT.
  mmcblk1: p1 p2 p3 p4 p5
+mmc0: new ultra high speed DDR50 SDHC card at address aaaa
+mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
  mmcblk0: p1 p2 p3
 tegra30-i2s 70080400.i2s: DMA channels sourced from device 70080000.ahub
 tegra-snd-rt5640 sound: rt5640-aif1 <-> 70080400.i2s mapping ok
@@ -381,7 +382,6 @@
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (c0c00000 - c0d00000)
-bash: cannot set terminal process group (-1): In[    7.640962] usb 2-1: new high-speed USB device number 2 using tegra-ehci
-appropriate ioctl for device
+bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# 
\ No newline at end of file
+root@(none):/# [    7.640974] usb 2-1: new high-speed USB device number 2 using tegra-ehci
