Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-06-29 00:41:54.612315577 -0700
+++ /tmp/foob	2016-06-29 00:41:54.616315581 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc5-next-20160628+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Jun 27 23:52:11 PDT 2016
+Linux version 4.7.0-rc5-next-20160629+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Jun 29 00:36:46 PDT 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -18,7 +18,7 @@
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
-Memory: 1000644K/1048576K available (7676K kernel code, 762K rwdata, 3580K rodata, 960K init, 269K bss, 31548K reserved, 16384K cma-reserved)
+Memory: 1000644K/1048576K available (7676K kernel code, 762K rwdata, 3584K rodata, 960K init, 269K bss, 31548K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
@@ -97,7 +97,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.616:1): initialized
+audit: type=2000 audit(0.596:1): initialized
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -127,6 +127,7 @@
 Unable to detect cache hierarchy from DT for CPU 0
 loop: module loaded
  vda: vda1 vda2
+libphy: Fixed MDIO Bus: probed
 tun: Universal TUN/TAP device driver, 1.6
 tun: (C) 1999-2004 Max Krasnyansky <maxk@qualcomm.com>
 e1000e: Intel(R) PRO/1000 Network Driver - 3.2.6-k
@@ -160,7 +161,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2016-06-28 06:52:46 UTC (1467096766)
+rtc-pl031 9010000.pl031: setting system clock to 2016-06-29 07:37:19 UTC (1467185839)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-06-29 00:41:54.620315583 -0700
+++ /tmp/foob	2016-06-29 00:41:54.624315586 -0700
@@ -41,7 +41,7 @@
 	 ##########################################
 	 3.2 MiB/s
 done
-Bytes transferred = 7283448 (6f22f8 hex)
+Bytes transferred = 7283728 (6f2410 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -52,7 +52,7 @@
 done
 Bytes transferred = 34956 (888c hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6f22f8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6f2410 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff88b ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc5-next-20160628-102956-g6f2a161 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jun 27 23:49:42 PDT 2016
+Linux version 4.7.0-rc5-next-20160629-103198-gb07695e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Jun 29 00:34:09 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -90,7 +90,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc10b6e5c   (15036 kB)
+      .text : 0xc0208000 - 0xc10b6e90   (15036 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
       .data : 0xc1300000 - 0xc141d5c0   (1142 kB)
        .bss : 0xc141f000 - 0xc1474d20   ( 344 kB)
@@ -286,8 +286,8 @@
 mmc0: new high speed SDHC card at address aaaa
 mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 tegra-ehci 7d008000.usb: EHCI Host Controller
-tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 1
  mmcblk0: p1 p2 p3
+tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 1
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-06-29 00:41:54.632315593 -0700
+++ /tmp/foob	2016-06-29 00:41:54.632315593 -0700
@@ -36,21 +36,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #######################################################
+	 ########################################################
 	 2.8 MiB/s
 done
-Bytes transferred = 5575288 (551278 hex)
+Bytes transferred = 5580456 (5526a8 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 34956 (888c hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x551278 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5526a8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff88b ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc5-next-20160628-102956-g6f2a161 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jun 27 23:49:51 PDT 2016
+Linux version 4.7.0-rc5-next-20160629-103198-gb07695e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Jun 29 00:34:09 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998936K/2096128K available (8441K kernel code, 373K rwdata, 2844K rodata, 1024K init, 366K bss, 31656K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998936K/2096128K available (8441K kernel code, 373K rwdata, 2848K rodata, 1024K init, 366K bss, 31656K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0c0570c   (12278 kB)
+      .text : 0xc0008000 - 0xc0c06788   (12282 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
       .data : 0xc0e00000 - 0xc0e5d540   ( 374 kB)
-       .bss : 0xc0e5f000 - 0xc0ebab14   ( 367 kB)
+       .bss : 0xc0e5f000 - 0xc0ebab54   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -213,6 +213,7 @@
 tegra-dsi 54300000.dsi: cannot get VDD supply
 loop: module loaded
 m25p80 spi32766.0: w25q32dw (4096 Kbytes)
+libphy: Fixed MDIO Bus: probed
 CAN device driver interface
 igb: Intel(R) Gigabit Ethernet Network Driver - version 5.3.0-k
 igb: Copyright (c) 2007-2014 Intel Corporation.
@@ -258,11 +259,11 @@
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
+mmc0: new high speed SDHC card at address aaaa
+mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
  mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
-mmc0: new high speed SDHC card at address aaaa
-mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
  mmcblk0: p1 p2 p3
 tegra30-i2s 70080400.i2s: DMA channels sourced from device 70080000.ahub
 tegra-snd-rt5640 sound: rt5640-aif1 <-> 70080400.i2s mapping ok
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-06-29 00:41:54.636315595 -0700
+++ /tmp/foob	2016-06-29 00:41:54.640315598 -0700
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -36,7 +36,7 @@
 	 ##########################################
 	 4 MiB/s
 done
-Bytes transferred = 7283448 (6f22f8 hex)
+Bytes transferred = 7283728 (6f2410 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -47,7 +47,7 @@
 done
 Bytes transferred = 66693 (10485 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6f22f8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x6f2410 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013484
@@ -55,7 +55,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc5-next-20160628-102956-g6f2a161 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jun 27 23:49:42 PDT 2016
+Linux version 4.7.0-rc5-next-20160629-103198-gb07695e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Jun 29 00:34:09 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -85,7 +85,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc10b6e5c   (15036 kB)
+      .text : 0xc0208000 - 0xc10b6e90   (15036 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
       .data : 0xc1300000 - 0xc141d5c0   (1142 kB)
        .bss : 0xc141f000 - 0xc1474d20   ( 344 kB)
@@ -305,12 +305,6 @@
 +5V_SATA: supplied by +5V_SYS
 +12V_SATA: supplied by +VDD_MUX
 usb2-0 supply vbus not found, using dummy regulator
-mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
- mmcblk0: p1
 usb2-1 supply vbus not found, using dummy regulator
 usb3-0 supply vbus not found, using dummy regulator
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
@@ -322,6 +316,7 @@
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 as3722-sd5: supplied by +5V_SYS
 as3722-ldo0: supplied by +1.35V_LP0(sd2)
+mmc0: new high speed MMC card at address 0001
 as3722-ldo1: supplied by +3.3V_RUN
 as3722-ldo2: supplied by +1.8V_VDDIO
 as3722-ldo3: supplied by +3.3V_SYS
@@ -376,7 +371,6 @@
 PCI: bus1: Fast back to back transfers disabled
 pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
 pci 0000:02:00.0: [10ec:8168] type 00 class 0x020000
-ata1: SATA link down (SStatus 0 SControl 300)
 pci 0000:02:00.0: reg 0x10: [io  0x0000-0x00ff]
 pci 0000:02:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit]
 pci 0000:02:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
@@ -410,15 +404,22 @@
 pcie_pme 0000:00:02.0:pcie001: service driver pcie_pme loaded
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
-r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0a52000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 393
+r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0a48000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 393
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
+drm drm: No connectors reported connected with modes
+[drm] Cannot find any crtc or sizes - going 1024x768
 mmc1: new high speed SDHC card at address aaaa
 mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
  mmcblk1: p1 p2 p3
-tegra-hdmi 54280000.hdmi: cannot set audio to 44100 at 297000000 pclk
-Console: switching to colour frame buffer device 512x135
+ata1: SATA link down (SStatus 0 SControl 300)
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+ mmcblk0: p1
+Console: switching to colour frame buffer device 128x48
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 of: dev_pm_opp_of_cpumask_add_table: couldn't find opp table for cpu:0, -19
@@ -437,7 +438,7 @@
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:1.
+VFS: Mounted root (ext3 filesystem) on device 179:17.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-06-29 00:41:54.644315602 -0700
+++ /tmp/foob	2016-06-29 00:41:54.648315605 -0700
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+MMC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -31,21 +31,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #######################################################
-	 3.7 MiB/s
+	 ########################################################
+	 4 MiB/s
 done
-Bytes transferred = 5575288 (551278 hex)
+Bytes transferred = 5580456 (5526a8 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.3 MiB/s
+	 3.7 MiB/s
 done
 Bytes transferred = 66693 (10485 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x551278 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5526a8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013484
@@ -54,7 +54,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc5-next-20160628-102956-g6f2a161 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jun 27 23:49:51 PDT 2016
+Linux version 4.7.0-rc5-next-20160629-103198-gb07695e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Jun 29 00:34:09 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -77,7 +77,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998816K/2096128K available (8441K kernel code, 373K rwdata, 2844K rodata, 1024K init, 366K bss, 31776K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998816K/2096128K available (8441K kernel code, 373K rwdata, 2848K rodata, 1024K init, 366K bss, 31776K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -85,10 +85,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0c0570c   (12278 kB)
+      .text : 0xc0008000 - 0xc0c06788   (12282 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
       .data : 0xc0e00000 - 0xc0e5d540   ( 374 kB)
-       .bss : 0xc0e5f000 - 0xc0ebab14   ( 367 kB)
+       .bss : 0xc0e5f000 - 0xc0ebab54   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -222,6 +222,7 @@
 as3722-regulator as3722-regulator: regulator 13 register failed -517
 tegra-ahci 70027000.sata: couldn't get PHY in node sata: -517
 m25p80 spi32765.0: w25q32dw (4096 Kbytes)
+libphy: Fixed MDIO Bus: probed
 CAN device driver interface
 igb: Intel(R) Gigabit Ethernet Network Driver - version 5.3.0-k
 igb: Copyright (c) 2007-2014 Intel Corporation.
@@ -441,7 +442,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:09 UTC (946684929)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:01:36 UTC (946684896)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-06-29 00:41:54.652315607 -0700
+++ /tmp/foob	2016-06-29 00:41:54.656315611 -0700
@@ -41,20 +41,20 @@
 	 #################################################################
 	 #################################################################
 	 ##########################################
-	 3.5 MiB/s
+	 3.1 MiB/s
 done
-Bytes transferred = 7283448 (6f22f8 hex)
+Bytes transferred = 7283728 (6f2410 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.2 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 66901 (10555 hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6f22f8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x6f2410 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013554
@@ -62,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc5-next-20160628-102956-g6f2a161 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jun 27 23:49:42 PDT 2016
+Linux version 4.7.0-rc5-next-20160629-103198-gb07695e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Jun 29 00:34:09 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -93,7 +93,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc10b6e5c   (15036 kB)
+      .text : 0xc0208000 - 0xc10b6e90   (15036 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
       .data : 0xc1300000 - 0xc141d5c0   (1142 kB)
        .bss : 0xc141f000 - 0xc1474d20   ( 344 kB)
@@ -289,6 +289,7 @@
 mmc0: Invalid maximum block size, assuming 512 bytes
 mmc0: SDHCI controller on 700b0600.sdhci [700b0600.sdhci] using ADMA 64-bit
 ledtrig-cpu: registered to indicate activity on CPUs
+input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
 NET: Registered protocol family 10
@@ -305,26 +306,25 @@
 +3.3V_RUN: supplied by +3.3V_SYS
 +3.3V_AVDD_HDMI_AP_GATED: supplied by +3.3V_RUN
 +1.8V_RUN_CAM: supplied by +3.3V_RUN
-input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 +VDDIO_SDMMC3: supplied by +3.3V_RUN
 +1.8V_RUN_VPP_FUSE: supplied by +3.3V_RUN
 +VDD_LED: supplied by +VDD_MUX
 +5V_VDD_TS_SW: supplied by +5V_SYS
 +5V_USB_HS: supplied by +5V_SYS
 +5V_USB_SS: supplied by +5V_SYS
++3.3V_PANEL: supplied by +3.3V_RUN
++3.3V_LP0: supplied by +3.3V_SYS
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
 mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
 mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
-+3.3V_PANEL: supplied by +3.3V_RUN
-+3.3V_LP0: supplied by +3.3V_SYS
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
-elan_i2c 1-0015: failed to read report data: -121
 +5V_HDMI_CON: supplied by +5V_SYS
 usb3-0 supply vbus not found, using dummy regulator
 usb3-1 supply vbus not found, using dummy regulator
+elan_i2c 1-0015: failed to read report data: -121
 tegra-sor 54540000.sor: failed to probe output: -517
 panel supply power not found, using dummy regulator
 sdhci-tegra 700b0000.sdhci: allocated mmc-pwrseq
@@ -344,17 +344,17 @@
 [drm] No driver support for vblank timestamp query.
 mmc2: new high speed SDHC card at address 0007
 mmcblk2: mmc2:0007 SD08G 7.42 GiB 
+Console: switching to colour frame buffer device 170x48
+drm drm: fb0:  frame buffer device
 GPT:Primary header alternate_lba != Alt. header my_lba
 GPT:3014591 != 15564799
 GPT:Primary header thinks Alt. header is not at the end of the disk.
 GPT:3014591 != 15564799
 GPT: Use GNU Parted to correct GPT errors.
  mmcblk2: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
-Console: switching to colour frame buffer device 170x48
-drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:22 UTC (946684822)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:03:10 UTC (946684990)
 +1.8V_RUN_CAM: disabling
 +3.3V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2016-06-29 00:41:54.660315614 -0700
+++ /tmp/foob	2016-06-29 00:41:54.664315617 -0700
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -38,10 +38,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #######################################################
-	 3.5 MiB/s
+	 ########################################################
+	 3.4 MiB/s
 done
-Bytes transferred = 5575288 (551278 hex)
+Bytes transferred = 5580456 (5526a8 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -52,7 +52,7 @@
 done
 Bytes transferred = 66901 (10555 hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x551278 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5526a8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013554
@@ -92,6 +92,7 @@
 +1.8V_RUN_VPP_FUSE: Bringing 0uV into 1800000-1800000uV
 cros-ec-spi spi32766.0: Chrome EC device registered
 m25p80 spi32765.0: w25q32dw (4096 Kbytes)
+libphy: Fixed MDIO Bus: probed
 CAN device driver interface
 igb: Intel(R) Gigabit Ethernet Network Driver - version 5.3.0-k
 igb: Copyright (c) 2007-2014 Intel Corporation.
@@ -159,8 +160,8 @@
 NET: Registered protocol family 29
 can: raw protocol (rev 20120528)
 can: broadcast manager protocol (rev 20160617 t)
-can: netlink gateway (rev 20130117) max_hops=1
 input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input3
+can: netlink gateway (rev 20130117) max_hops=1
 Bluetooth: RFCOMM socket layer initialized
 Bluetooth: RFCOMM ver 1.11
 Bluetooth: BNEP (Ethernet Emulation) ver 1.3
@@ -212,9 +213,9 @@
 mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
+usb 1-2: new high-speed USB device number 2 using tegra-xusb
 mmc2: new high speed SDHC card at address 0007
 mmcblk2: mmc2:0007 SD08G 7.42 GiB 
-usb 1-2: new high-speed USB device number 2 using tegra-xusb
 GPT:Primary header alternate_lba != Alt. header my_lba
 GPT:3014591 != 15564799
 GPT:Primary header thinks Alt. header is not at the end of the disk.
@@ -225,7 +226,7 @@
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input4
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:23 UTC (946684823)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:24 UTC (946684824)
 +1.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
 +1.05V_RUN_CAM_REAR: disabling
@@ -247,10 +248,11 @@
 Freeing unused kernel memory: 1024K (c0d00000 - c0e00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    5.767519] asix 1-3:1.0 eth0: register 'asix' at usb-70090000.usb-3, ASIX AX88772 USB 2.0 Ethernet, c8:d7:19:90:fa:65
+root@(none):/# [    5.777518] asix 1-3:1.0 eth0: register 'asix' at usb-70090000.usb-3, ASIX AX88772 USB 2.0 Ethernet, c8:d7:19:90:fa:65
 usb 1-1: new high-speed USB device number 4 using tegra-xusb
 usb-storage 1-1:1.0: USB Mass Storage device detected
 scsi host0: usb-storage 1-1:1.0
+scsi host0: runtime PM trying to activate child device host0 but parent (1-1:1.0) is not active
 scsi 0:0:0:0: Direct-Access     SanDisk  SDDR-113         9412 PQ: 0 ANSI: 0
 sd 0:0:0:0: [sda] 15523840 512-byte logical blocks: (7.95 GB/7.40 GiB)
 sd 0:0:0:0: [sda] Write Protect is off
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2016-06-29 00:41:54.668315620 -0700
+++ /tmp/foob	2016-06-29 00:41:54.672315623 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1800854 us
+Bootloader-Cpu Init at (time stamp): 1801272 us
 
 Pinmux changes applied in kernel way
 
@@ -59,25 +59,25 @@
 The proc BoardInfo: 0x0176:0x0001:0x01:0x43:0x00
 FastbootAddAtag: setting ATAG (0)
 
-Board Id Read Time: 4 us
+Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1800827  us
-Bootloader (AosInit) time: 734244 us
-Bootloader Main Init time: 180086 us
-BlockDevMgrInit time: 157517 us
+Miscellaneous avp time: 1801246  us
+Bootloader (AosInit) time: 733444 us
+Bootloader Main Init time: 180209 us
+BlockDevMgrInit time: 157562 us
 PartMgrInit time: 8 us
-StorMgrInit time: 30 us
+StorMgrInit time: 31 us
 FsMgrInit time: 2 us
-Partition Table load: 76714 us 
-Display init time: 574686 us
+Partition Table load: 77066 us 
+Display init time: 574643 us
 Kernel read time: 364 msec
 Load Kernel Size:   15792128 Bytes
 Bl Kernel ReadSpeed: 43384000 Bytes/sec
-Pre-Kernel Jump time: 1698033 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5783932 us
+Pre-Kernel Jump time: 1695920 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5781917 us
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc5-next-20160628+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Jun 27 23:52:11 PDT 2016
+Linux version 4.7.0-rc5-next-20160629+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Jun 29 00:36:46 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
 Memory limited to 1734MB
 efi: Getting EFI parameters from FDT:
@@ -91,7 +91,7 @@
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 Dentry cache hash table entries: 262144 (order: 9, 2097152 bytes)
 Inode-cache hash table entries: 131072 (order: 8, 1048576 bytes)
-Memory: 1707236K/1775616K available (7676K kernel code, 762K rwdata, 3580K rodata, 960K init, 269K bss, 51996K reserved, 16384K cma-reserved)
+Memory: 1707236K/1775616K available (7676K kernel code, 762K rwdata, 3584K rodata, 960K init, 269K bss, 51996K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
@@ -208,6 +208,7 @@
 msm_serial: driver initialized
 Unable to detect cache hierarchy from DT for CPU 0
 loop: module loaded
+libphy: Fixed MDIO Bus: probed
 tun: Universal TUN/TAP device driver, 1.6
 tun: (C) 1999-2004 Max Krasnyansky <maxk@qualcomm.com>
 e1000e: Intel(R) PRO/1000 Network Driver - 3.2.6-k
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-06-29 00:41:54.676315626 -0700
+++ /tmp/foob	2016-06-29 00:41:54.676315626 -0700
@@ -47,9 +47,9 @@
 	 #################################################################
 	 #################################################################
 	 ##########################################
-	 4.1 MiB/s
+	 4 MiB/s
 done
-Bytes transferred = 7283448 (6f22f8 hex)
+Bytes transferred = 7283728 (6f2410 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -60,7 +60,7 @@
 done
 Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x6f22f8 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x6f2410 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102b3
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc5-next-20160628-102956-g6f2a161 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jun 27 23:49:42 PDT 2016
+Linux version 4.7.0-rc5-next-20160629-103198-gb07695e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Jun 29 00:34:09 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -97,7 +97,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc10b6e5c   (15036 kB)
+      .text : 0xc0208000 - 0xc10b6e90   (15036 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
       .data : 0xc1300000 - 0xc141d5c0   (1142 kB)
        .bss : 0xc141f000 - 0xc1474d20   ( 344 kB)
@@ -323,25 +323,24 @@
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
-input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: hctosys: unable to read the hardware clock
-Waiting for root device PARTUUID=11eb23c1-01...
 mmc0: host does not support reading read-only switch, assuming write-enable
+input: gpio-keys as /devices/soc0/gpio-keys/input/input0
 mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
+rtc-em3027 2-0056: hctosys: unable to read the hardware clock
  mmcblk0: p1
+EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
-EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
-usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
+usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-06-29 00:41:54.684315632 -0700
+++ /tmp/foob	2016-06-29 00:41:54.684315632 -0700
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -44,21 +44,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #######################################################
-	 912.1 KiB/s
+	 ########################################################
+	 905.3 KiB/s
 done
-Bytes transferred = 5575288 (551278 hex)
+Bytes transferred = 5580456 (5526a8 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 898.4 KiB/s
+	 861.3 KiB/s
 done
 Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x551278 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x5526a8 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102b3
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc5-next-20160628-102956-g6f2a161 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jun 27 23:49:51 PDT 2016
+Linux version 4.7.0-rc5-next-20160629-103198-gb07695e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Jun 29 00:34:09 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -89,7 +89,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 959752K/1048576K available (8441K kernel code, 373K rwdata, 2844K rodata, 1024K init, 366K bss, 23288K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 959752K/1048576K available (8441K kernel code, 373K rwdata, 2848K rodata, 1024K init, 366K bss, 23288K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -97,10 +97,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0c0570c   (12278 kB)
+      .text : 0xc0008000 - 0xc0c06788   (12282 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
       .data : 0xc0e00000 - 0xc0e5d540   ( 374 kB)
-       .bss : 0xc0e5f000 - 0xc0ebab14   ( 367 kB)
+       .bss : 0xc0e5f000 - 0xc0ebab54   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
@@ -238,6 +238,7 @@
 loop: module loaded
 m25p80 spi32766.0: found s25fl008k, expected w25q80bl
 m25p80 spi32766.0: s25fl008k (1024 Kbytes)
+libphy: Fixed MDIO Bus: probed
 CAN device driver interface
 igb: Intel(R) Gigabit Ethernet Network Driver - version 5.3.0-k
 igb: Copyright (c) 2007-2014 Intel Corporation.
@@ -306,20 +307,20 @@
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
- mmcblk0: p1
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
  mmcblk1: p1 p2
+ mmcblk0: p1
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
 NET: Registered protocol family 10
 mip6: Mobile IPv6
 sit: IPv6 over IPv4 tunneling driver
-usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 NET: Registered protocol family 17
 NET: Registered protocol family 15
 can: controller area network core (rev 20120528 abi 9)
+usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 NET: Registered protocol family 29
 can: raw protocol (rev 20120528)
 can: broadcast manager protocol (rev 20160617 t)
@@ -346,7 +347,7 @@
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:1.
+VFS: Mounted root (ext3 filesystem) on device 179:17.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (c0d00000 - c0e00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2016-06-29 00:41:54.692315638 -0700
+++ /tmp/foob	2016-06-29 00:41:54.692315638 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 766902 us
-Basic modules init took 348602 us
+Tegraboot started after 797927 us
+Basic modules init took 349325 us
 Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14366 us
-Carveout took 252714 us
-CPU initialization took 92315 us
-Total time taken by TegraBoot 707997 us
+Next stage binary read took 14368 us
+Carveout took 252716 us
+CPU initialization took 92321 us
+Total time taken by TegraBoot 708730 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.605] LPDDR4 Training: Number of tables = 10
+64b[0001.636] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -119,7 +119,7 @@
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
 13363712 bytes read in 759 ms (16.8 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-39484 bytes read in 454 ms (84 KiB/s)
+39484 bytes read in 453 ms (85 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
 1627787 bytes read in 495 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc5-next-20160628+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Jun 27 23:52:11 PDT 2016
+Linux version 4.7.0-rc5-next-20160629+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Jun 29 00:36:46 PDT 2016
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -166,7 +166,7 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffff80007a000000-ffff80007dffffff]
-Memory: 2979620K/3131392K available (7676K kernel code, 762K rwdata, 3580K rodata, 960K init, 269K bss, 135388K reserved, 16384K cma-reserved)
+Memory: 2979620K/3131392K available (7676K kernel code, 762K rwdata, 3584K rodata, 960K init, 269K bss, 135388K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
@@ -264,7 +264,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.447:1): initialized
+audit: type=2000 audit(0.443:1): initialized
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -290,6 +290,7 @@
 msm_serial: driver initialized
 Unable to detect cache hierarchy from DT for CPU 0
 loop: module loaded
+libphy: Fixed MDIO Bus: probed
 tun: Universal TUN/TAP device driver, 1.6
 tun: (C) 1999-2004 Max Krasnyansky <maxk@qualcomm.com>
 e1000e: Intel(R) PRO/1000 Network Driver - 3.2.6-k
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2016-06-29 00:41:54.696315642 -0700
+++ /tmp/foob	2016-06-29 00:41:54.700315644 -0700
@@ -1,6 +1,65 @@
- 0.000000] boot misc2 0x0: expected 0xf400f0da
+0x0000000070006000 (options '115200n8')
+bootconsole [uart0] enabled
+efi: Getting EFI parameters from FDT:
+efi: UEFI not found.
+cma: Reserved 16 MiB at 0x00000000f5400000
+psci: probing for conduit method from DT.
+psci: PSCIv1.0 detected in firmware.
+psci: Using standard PSCI v0.2 function IDs
+psci: MIGRATE_INFO_UP_CPU reported invalid physical ID (0x80000000)
+percpu: Embedded 20 pages/cpu @ffff8000bff50000 s42880 r8192 d30848 u81920
+Detected PIPT I-cache on CPU0
+CPU features: enabling workaround for ARM erratum 832075
+CPU features: enabling workaround for ARM erratum 834220
+Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 735336
+Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
+log_buf_len individual max cpu contribution: 4096 bytes
+log_buf_len total cpu_extra contributions: 12288 bytes
+log_buf_len min size: 16384 bytes
+log_buf_len: 32768 bytes
+early log buf free: 14420(88%)
+PID hash table entries: 4096 (order: 3, 32768 bytes)
+Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
+Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
+software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
+Memory: 2833856K/2988032K available (7676K kernel code, 762K rwdata, 3584K rodata, 960K init, 269K bss, 137792K reserved, 16384K cma-reserved)
+Virtual kernel memory layout:
+    modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
+    vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
+      .text : 0xffff000008080000 - 0xffff000008800000   (  7680 KB)
+    .rodata : 0xffff000008800000 - 0xffff000008b90000   (  3648 KB)
+      .init : 0xffff000008b90000 - 0xffff000008c80000   (   960 KB)
+      .data : 0xffff000008c80000 - 0xffff000008d3ea00   (   763 KB)
+       .bss : 0xffff000008d3ea00 - 0xffff000008d81fb8   (   270 KB)
+    fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
+    PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
+    vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
+              0xffff7e0000000000 - 0xffff7e0003000000   (    48 MB actual)
+    memory  : 0xffff800000000000 - 0xffff8000c0000000   (  3072 MB)
+SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
+Preemptible hierarchical RCU implementation.
+	Build-time adjustment of leaf fanout to 64.
+	RCU restricting CPUs from NR_CPUS=64 to nr_cpu_ids=4.
+RCU: Adjusting geometry for rcu_fanout_leaf=64, nr_cpu_ids=4
+NR_IRQS:64 nr_irqs:64 0
+GIC: Using split EOI/Deactivate mode
+/interrupt-controller@60004000: 192 interrupts forwarded to /interrupt-controller@50041000
+boot misc0 0x0: expected 0x80000
+ (comparison mask = 0x400ffffb)
+boot misc2 0x1f720005: expected 0x1f720f05
+ (comparison mask = 0xffffff17)
+PLL_C already enabled. Postponing set full defaults
+PLL_MB already enabled. Postponing set full defaults
+PLL_U already enabled. Postponing set full defaults
+PLL_D already enabled. Postponing set full defaults
+PLL_DP already enabled. Postponing set full defaults
+boot misc0 0x0: expected 0x20
+ (comparison mask = 0x7ffffff)
+boot misc1 0x90000000: expected 0xc0000000
+ (comparison mask = 0xf8000000)
+boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x8fc0: expected 0x2004f400
+boot misc3 0x9330: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -13,7 +72,7 @@
 console [tty0] enabled
 bootconsole [uart0] disabled
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc5-next-20160628+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Jun 27 23:52:11 PDT 2016
+Linux version 4.7.0-rc5-next-20160629+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Jun 29 00:36:46 PDT 2016
 Boot CPU: AArch64 Processor [411fd071]
 earlycon: uart0 at MMIO 0x0000000070006000 (options '115200n8')
 bootconsole [uart0] enabled
@@ -39,7 +98,7 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2833856K/2988032K available (7676K kernel code, 762K rwdata, 3580K rodata, 960K init, 269K bss, 137792K reserved, 16384K cma-reserved)
+Memory: 2833856K/2988032K available (7676K kernel code, 762K rwdata, 3584K rodata, 960K init, 269K bss, 137792K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
@@ -76,7 +135,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x8fc0: expected 0x2004f400
+boot misc3 0x9330: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -154,7 +213,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.227:1): initialized
+audit: type=2000 audit(0.235:1): initialized
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -176,6 +235,7 @@
 msm_serial: driver initialized
 Unable to detect cache hierarchy from DT for CPU 0
 loop: module loaded
+libphy: Fixed MDIO Bus: probed
 tun: Universal TUN/TAP device driver, 1.6
 tun: (C) 1999-2004 Max Krasnyansky <maxk@qualcomm.com>
 e1000e: Intel(R) PRO/1000 Network Driver - 3.2.6-k
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-06-29 00:41:54.704315647 -0700
+++ /tmp/foob	2016-06-29 00:41:54.708315650 -0700
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 ##########################################
-	 3.2 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 7283448 (6f22f8 hex)
+Bytes transferred = 7283728 (6f2410 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.3 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 51511 (c937 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6f22f8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6f2410 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff936 ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc5-next-20160628-102956-g6f2a161 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jun 27 23:49:42 PDT 2016
+Linux version 4.7.0-rc5-next-20160629-103198-gb07695e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Jun 29 00:34:09 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -89,7 +89,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc10b6e5c   (15036 kB)
+      .text : 0xc0208000 - 0xc10b6e90   (15036 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
       .data : 0xc1300000 - 0xc141d5c0   (1142 kB)
        .bss : 0xc141f000 - 0xc1474d20   ( 344 kB)
@@ -376,5 +376,6 @@
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
-bash: no job control in this shell
-root@(none):/# [    3.898513] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+bash: no job control in this shell[    3.893508] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+
+root@(none):/# 
\ No newline at end of file
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-06-29 00:41:54.712315654 -0700
+++ /tmp/foob	2016-06-29 00:41:54.716315656 -0700
@@ -36,10 +36,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #######################################################
-	 3.2 MiB/s
+	 ########################################################
+	 3.1 MiB/s
 done
-Bytes transferred = 5575288 (551278 hex)
+Bytes transferred = 5580456 (5526a8 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 51511 (c937 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x551278 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5526a8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff936 ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc5-next-20160628-102956-g6f2a161 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jun 27 23:49:51 PDT 2016
+Linux version 4.7.0-rc5-next-20160629-103198-gb07695e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Jun 29 00:34:09 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -81,7 +81,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998856K/2096128K available (8441K kernel code, 373K rwdata, 2844K rodata, 1024K init, 366K bss, 31736K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998856K/2096128K available (8441K kernel code, 373K rwdata, 2848K rodata, 1024K init, 366K bss, 31736K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -89,10 +89,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0c0570c   (12278 kB)
+      .text : 0xc0008000 - 0xc0c06788   (12282 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
       .data : 0xc0e00000 - 0xc0e5d540   ( 374 kB)
-       .bss : 0xc0e5f000 - 0xc0ebab14   ( 367 kB)
+       .bss : 0xc0e5f000 - 0xc0ebab54   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -225,6 +225,7 @@
 tegra-hdmi 54280000.hdmi: failed to get HDMI regulator
 loop: module loaded
 m25p80 spi32766.1: w25q32 (4096 Kbytes)
+libphy: Fixed MDIO Bus: probed
 CAN device driver interface
 igb: Intel(R) Gigabit Ethernet Network Driver - version 5.3.0-k
 igb: Copyright (c) 2007-2014 Intel Corporation.
@@ -378,4 +379,4 @@
 usb 2-1: new high-speed USB device number 2 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    7.815095] asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0f:67:60
+root@(none):/# 
\ No newline at end of file
