Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-12-15 03:43:22.797372651 -0800
+++ /tmp/foob	2016-12-15 03:43:22.801372665 -0800
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-next-20161214-165388-g9aa17b2 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Dec 13 20:36:55 PST 2016
+Linux version 4.9.0-next-20161215-165673-gdf5462d (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Dec 15 03:36:19 PST 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -101,7 +101,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.692:1): initialized
+audit: type=2000 audit(0.660:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=46 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -167,7 +167,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2016-12-14 04:37:34 UTC (1481690254)
+rtc-pl031 9010000.pl031: setting system clock to 2016-12-15 11:37:02 UTC (1481801822)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-12-15 03:43:22.809372698 -0800
+++ /tmp/foob	2016-12-15 03:43:22.809372698 -0800
@@ -39,9 +39,9 @@
 	 #################################################################
 	 #################################################################
 	 ##################################################
-	 2.9 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 7398800 (70e590 hex)
+Bytes transferred = 7400056 (70ea78 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -52,7 +52,7 @@
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x70e590 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x70ea78 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-next-20161214-140955-g9aa17b2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Dec 13 20:34:21 PST 2016
+Linux version 4.9.0-next-20161215-141240-gdf5462d (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Dec 15 03:33:13 PST 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -305,14 +305,14 @@
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
 usb1_vbus: supplied by vdd-sys-5v0
-mmc0: new high speed SDHC card at address aaaa
 usb2_vbus: supplied by vdd-sys-5v0
+mmc0: new high speed SDHC card at address aaaa
+mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
-mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
-mmc1: new high speed MMC card at address 0001
  mmcblk0: p1 p2 p3
+mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
@@ -328,7 +328,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-10-07 16:05:50 UTC (970934750)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-10-08 23:02:15 UTC (971046135)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -340,4 +340,4 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.579742] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.549531] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-12-15 03:43:22.817372728 -0800
+++ /tmp/foob	2016-12-15 03:43:22.817372728 -0800
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 ##################################################
-	 3.2 MiB/s
+	 2.9 MiB/s
 done
-Bytes transferred = 5504608 (53fe60 hex)
+Bytes transferred = 5504088 (53fc58 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 2.4 MiB/s
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x53fe60 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x53fc58 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-next-20161214-140955-g9aa17b2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Dec 13 20:33:52 PST 2016
+Linux version 4.9.0-next-20161215-141240-gdf5462d (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Dec 15 03:33:23 PST 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -133,7 +133,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?[    0.524405] hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
+No ATAGs?[    0.515250] hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 8 bytes.
 tegra-mc 70019000.memory-controller: no memory timings for RAM code 3 registered
 dvdd_ts: Failed to request enable GPIO61: -517
@@ -321,7 +321,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-10-07 16:02:55 UTC (970934575)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-10-08 23:05:01 UTC (971046301)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -335,4 +335,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# 
\ No newline at end of file
+root@(none):/# [    2.806358] usb 1-1: new h
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-12-15 03:43:22.825372757 -0800
+++ /tmp/foob	2016-12-15 03:43:22.825372757 -0800
@@ -34,20 +34,20 @@
 	 #################################################################
 	 #################################################################
 	 ##################################################
-	 3.2 MiB/s
+	 4 MiB/s
 done
-Bytes transferred = 7398800 (70e590 hex)
+Bytes transferred = 7400056 (70ea78 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.2 MiB/s
+	 3.8 MiB/s
 done
 Bytes transferred = 67133 (1063d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x70e590 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x70ea78 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301363c
@@ -55,7 +55,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-next-20161214-140955-g9aa17b2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Dec 13 20:34:21 PST 2016
+Linux version 4.9.0-next-20161215-141240-gdf5462d (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Dec 15 03:33:13 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -163,6 +163,96 @@
 PTP clock support registered
 EDAC MC: Ver: 3.0.0
 clocksource: Switched to clocksource arch_sys_counter
+------------[ cut here ]------------
+WARNING: CPU: 1 PID: 1 at fs/sysfs/dir.c:31 sysfs_warn_dup+0x58/0x74
+sysfs: cannot create duplicate filename '/devices/virtual/thermal/thermal_zone1'
+Modules linked in:
+CPU: 1 PID: 1 Comm: swapper/0 Not tainted 4.9.0-next-20161215-141240-gdf5462d #1
+Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
+[<c030fb38>] (unwind_backtrace) from [<c030b198>] (show_stack+0x10/0x14)
+[<c030b198>] (show_stack) from [<c0596230>] (dump_stack+0x84/0xa4)
+[<c0596230>] (dump_stack) from [<c0341bdc>] (__warn+0xd4/0x100)
+[<c0341bdc>] (__warn) from [<c0341ca8>] (warn_slowpath_fmt+0x38/0x48)
+[<c0341ca8>] (warn_slowpath_fmt) from [<c0478280>] (sysfs_warn_dup+0x58/0x74)
+[<c0478280>] (sysfs_warn_dup) from [<c0478368>] (sysfs_create_dir_ns+0x88/0x90)
+[<c0478368>] (sysfs_create_dir_ns) from [<c0597d60>] (kobject_add_internal+0xac/0x2e0)
+[<c0597d60>] (kobject_add_internal) from [<c059815c>] (kobject_add+0x4c/0x9c)
+[<c059815c>] (kobject_add) from [<c07d6bb8>] (device_add+0xc0/0x54c)
+[<c07d6bb8>] (device_add) from [<c0a1ff08>] (thermal_zone_device_register+0x160/0x564)
+[<c0a1ff08>] (thermal_zone_device_register) from [<c117e2f4>] (of_parse_thermal_zones+0x58c/0x840)
+[<c117e2f4>] (of_parse_thermal_zones) from [<c117dcfc>] (thermal_init+0x38/0xa4)
+[<c117dcfc>] (thermal_init) from [<c0301e6c>] (do_one_initcall+0x3c/0x16c)
+[<c0301e6c>] (do_one_initcall) from [<c1100d98>] (kernel_init_freeable+0x124/0x1f0)
+[<c1100d98>] (kernel_init_freeable) from [<c0bdf484>] (kernel_init+0x8/0x118)
+[<c0bdf484>] (kernel_init) from [<c0307fb8>] (ret_from_fork+0x14/0x3c)
+---[ end trace 38bc6b264b067772 ]---
+------------[ cut here ]------------
+WARNING: CPU: 1 PID: 1 at lib/kobject.c:240 kobject_add_internal+0x258/0x2e0
+kobject_add_internal failed for thermal_zone1 with -EEXIST, don't try to register things with the same name in the same directory.
+Modules linked in:
+CPU: 1 PID: 1 Comm: swapper/0 Tainted: G        W       4.9.0-next-20161215-141240-gdf5462d #1
+Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
+[<c030fb38>] (unwind_backtrace) from [<c030b198>] (show_stack+0x10/0x14)
+[<c030b198>] (show_stack) from [<c0596230>] (dump_stack+0x84/0xa4)
+[<c0596230>] (dump_stack) from [<c0341bdc>] (__warn+0xd4/0x100)
+[<c0341bdc>] (__warn) from [<c0341ca8>] (warn_slowpath_fmt+0x38/0x48)
+[<c0341ca8>] (warn_slowpath_fmt) from [<c0597f0c>] (kobject_add_internal+0x258/0x2e0)
+[<c0597f0c>] (kobject_add_internal) from [<c059815c>] (kobject_add+0x4c/0x9c)
+[<c059815c>] (kobject_add) from [<c07d6bb8>] (device_add+0xc0/0x54c)
+[<c07d6bb8>] (device_add) from [<c0a1ff08>] (thermal_zone_device_register+0x160/0x564)
+[<c0a1ff08>] (thermal_zone_device_register) from [<c117e2f4>] (of_parse_thermal_zones+0x58c/0x840)
+[<c117e2f4>] (of_parse_thermal_zones) from [<c117dcfc>] (thermal_init+0x38/0xa4)
+[<c117dcfc>] (thermal_init) from [<c0301e6c>] (do_one_initcall+0x3c/0x16c)
+[<c0301e6c>] (do_one_initcall) from [<c1100d98>] (kernel_init_freeable+0x124/0x1f0)
+[<c1100d98>] (kernel_init_freeable) from [<c0bdf484>] (kernel_init+0x8/0x118)
+[<c0bdf484>] (kernel_init) from [<c0307fb8>] (ret_from_fork+0x14/0x3c)
+---[ end trace 38bc6b264b067773 ]---
+Failed to build gpu zone -17
+------------[ cut here ]------------
+WARNING: CPU: 1 PID: 1 at fs/sysfs/dir.c:31 sysfs_warn_dup+0x58/0x74
+sysfs: cannot create duplicate filename '/devices/virtual/thermal/thermal_zone1'
+Modules linked in:
+CPU: 1 PID: 1 Comm: swapper/0 Tainted: G        W       4.9.0-next-20161215-141240-gdf5462d #1
+Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
+[<c030fb38>] (unwind_backtrace) from [<c030b198>] (show_stack+0x10/0x14)
+[<c030b198>] (show_stack) from [<c0596230>] (dump_stack+0x84/0xa4)
+[<c0596230>] (dump_stack) from [<c0341bdc>] (__warn+0xd4/0x100)
+[<c0341bdc>] (__warn) from [<c0341ca8>] (warn_slowpath_fmt+0x38/0x48)
+[<c0341ca8>] (warn_slowpath_fmt) from [<c0478280>] (sysfs_warn_dup+0x58/0x74)
+[<c0478280>] (sysfs_warn_dup) from [<c0478368>] (sysfs_create_dir_ns+0x88/0x90)
+[<c0478368>] (sysfs_create_dir_ns) from [<c0597d60>] (kobject_add_internal+0xac/0x2e0)
+[<c0597d60>] (kobject_add_internal) from [<c059815c>] (kobject_add+0x4c/0x9c)
+[<c059815c>] (kobject_add) from [<c07d6bb8>] (device_add+0xc0/0x54c)
+[<c07d6bb8>] (device_add) from [<c0a1ff08>] (thermal_zone_device_register+0x160/0x564)
+[<c0a1ff08>] (thermal_zone_device_register) from [<c117e2f4>] (of_parse_thermal_zones+0x58c/0x840)
+[<c117e2f4>] (of_parse_thermal_zones) from [<c117dcfc>] (thermal_init+0x38/0xa4)
+[<c117dcfc>] (thermal_init) from [<c0301e6c>] (do_one_initcall+0x3c/0x16c)
+[<c0301e6c>] (do_one_initcall) from [<c1100d98>] (kernel_init_freeable+0x124/0x1f0)
+[<c1100d98>] (kernel_init_freeable) from [<c0bdf484>] (kernel_init+0x8/0x118)
+[<c0bdf484>] (kernel_init) from [<c0307fb8>] (ret_from_fork+0x14/0x3c)
+---[ end trace 38bc6b264b067774 ]---
+------------[ cut here ]------------
+WARNING: CPU: 1 PID: 1 at lib/kobject.c:240 kobject_add_internal+0x258/0x2e0
+kobject_add_internal failed for thermal_zone1 with -EEXIST, don't try to register things with the same name in the same directory.
+Modules linked in:
+CPU: 1 PID: 1 Comm: swapper/0 Tainted: G        W       4.9.0-next-20161215-141240-gdf5462d #1
+Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
+[<c030fb38>] (unwind_backtrace) from [<c030b198>] (show_stack+0x10/0x14)
+[<c030b198>] (show_stack) from [<c0596230>] (dump_stack+0x84/0xa4)
+[<c0596230>] (dump_stack) from [<c0341bdc>] (__warn+0xd4/0x100)
+[<c0341bdc>] (__warn) from [<c0341ca8>] (warn_slowpath_fmt+0x38/0x48)
+[<c0341ca8>] (warn_slowpath_fmt) from [<c0597f0c>] (kobject_add_internal+0x258/0x2e0)
+[<c0597f0c>] (kobject_add_internal) from [<c059815c>] (kobject_add+0x4c/0x9c)
+[<c059815c>] (kobject_add) from [<c07d6bb8>] (device_add+0xc0/0x54c)
+[<c07d6bb8>] (device_add) from [<c0a1ff08>] (thermal_zone_device_register+0x160/0x564)
+[<c0a1ff08>] (thermal_zone_device_register) from [<c117e2f4>] (of_parse_thermal_zones+0x58c/0x840)
+[<c117e2f4>] (of_parse_thermal_zones) from [<c117dcfc>] (thermal_init+0x38/0xa4)
+[<c117dcfc>] (thermal_init) from [<c0301e6c>] (do_one_initcall+0x3c/0x16c)
+[<c0301e6c>] (do_one_initcall) from [<c1100d98>] (kernel_init_freeable+0x124/0x1f0)
+[<c1100d98>] (kernel_init_freeable) from [<c0bdf484>] (kernel_init+0x8/0x118)
+[<c0bdf484>] (kernel_init) from [<c0307fb8>] (ret_from_fork+0x14/0x3c)
+---[ end trace 38bc6b264b067775 ]---
+Failed to build pllx zone -17
 NET: Registered protocol family 2
 TCP established hash table entries: 8192 (order: 3, 32768 bytes)
 TCP bind hash table entries: 8192 (order: 4, 65536 bytes)
@@ -309,7 +399,6 @@
 +USB0_VBUS_SW: supplied by +5V_SYS
 +5V_USB_HS: supplied by +5V_SYS
 +3.3V_LP0: supplied by +3.3V_SYS
-mmc0: new high speed MMC card at address 0001
 +5V_HDMI_CON: supplied by +5V_SYS
 +5V_SATA: supplied by +5V_SYS
 +12V_SATA: supplied by +VDD_MUX
@@ -318,6 +407,7 @@
 usb3-0 supply vbus not found, using dummy regulator
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
 tegra-pcie 1003000.pcie-controller: Failed to get supply 'avddio-pex': -517
+mmc0: new high speed MMC card at address 0001
 tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
 as3722-sd2: supplied by +5V_SYS
@@ -428,19 +518,19 @@
 mmc1: new high speed SDHC card at address aaaa
 mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
  mmcblk1: p1 p2 p3
-Console: switching to colour frame buffer device 128x48
-drm drm: fb0:  frame buffer device
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+Console: switching to colour frame buffer device 128x48
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
  mmcblk0: p1
+drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 of: dev_pm_opp_of_cpumask_add_table: couldn't find opp table for cpu:0, -19
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:14 UTC (946684814)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:13 UTC (946684813)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-12-15 03:43:22.833372789 -0800
+++ /tmp/foob	2016-12-15 03:43:22.833372789 -0800
@@ -34,7 +34,7 @@
 	 ##################################################
 	 4 MiB/s
 done
-Bytes transferred = 5504608 (53fe60 hex)
+Bytes transferred = 5504088 (53fc58 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -45,7 +45,7 @@
 done
 Bytes transferred = 67133 (1063d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x53fe60 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x53fc58 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301363c
@@ -54,7 +54,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-next-20161214-140955-g9aa17b2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Dec 13 20:33:52 PST 2016
+Linux version 4.9.0-next-20161215-141240-gdf5462d (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Dec 15 03:33:23 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -128,7 +128,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?[    0.571309] hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
+No ATAGs?[    0.558652] hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 8 bytes.
 +5V_SYS: supplied by +VDD_MUX
 +3.3V_SYS: supplied by +VDD_MUX
@@ -328,9 +328,9 @@
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 as3722-sd5: supplied by +5V_SYS
 as3722-ldo0: supplied by +1.35V_LP0(sd2)
+input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input1
 as3722-ldo1: supplied by +3.3V_RUN
 as3722-ldo2: supplied by +1.8V_VDDIO
-input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input1
 as3722-ldo3: supplied by +3.3V_SYS
 as3722-ldo4: supplied by +3.3V_SYS
 as3722-ldo5: supplied by +1.8V_VDDIO
@@ -450,7 +450,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:13 UTC (946684813)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:14 UTC (946684814)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-12-15 03:43:22.841372820 -0800
+++ /tmp/foob	2016-12-15 03:43:22.841372820 -0800
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -42,20 +42,20 @@
 	 #################################################################
 	 #################################################################
 	 ##################################################
-	 3.5 MiB/s
+	 3.6 MiB/s
 done
-Bytes transferred = 7398800 (70e590 hex)
+Bytes transferred = 7400056 (70ea78 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.4 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 67943 (10967 hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x70e590 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x70ea78 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013966
@@ -63,7 +63,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-next-20161214-140955-g9aa17b2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Dec 13 20:34:21 PST 2016
+Linux version 4.9.0-next-20161215-141240-gdf5462d (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Dec 15 03:33:13 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -175,6 +175,96 @@
 tegra-emc 7001b000.emc: no memory timings for RAM code 0 found in DT
 tegra-emc: probe of 7001b000.emc failed with error -2
 clocksource: Switched to clocksource arch_sys_counter
+------------[ cut here ]------------
+WARNING: CPU: 0 PID: 1 at fs/sysfs/dir.c:31 sysfs_warn_dup+0x58/0x74
+sysfs: cannot create duplicate filename '/devices/virtual/thermal/thermal_zone1'
+Modules linked in:
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.9.0-next-20161215-141240-gdf5462d #1
+Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
+[<c030fb38>] (unwind_backtrace) from [<c030b198>] (show_stack+0x10/0x14)
+[<c030b198>] (show_stack) from [<c0596230>] (dump_stack+0x84/0xa4)
+[<c0596230>] (dump_stack) from [<c0341bdc>] (__warn+0xd4/0x100)
+[<c0341bdc>] (__warn) from [<c0341ca8>] (warn_slowpath_fmt+0x38/0x48)
+[<c0341ca8>] (warn_slowpath_fmt) from [<c0478280>] (sysfs_warn_dup+0x58/0x74)
+[<c0478280>] (sysfs_warn_dup) from [<c0478368>] (sysfs_create_dir_ns+0x88/0x90)
+[<c0478368>] (sysfs_create_dir_ns) from [<c0597d60>] (kobject_add_internal+0xac/0x2e0)
+[<c0597d60>] (kobject_add_internal) from [<c059815c>] (kobject_add+0x4c/0x9c)
+[<c059815c>] (kobject_add) from [<c07d6bb8>] (device_add+0xc0/0x54c)
+[<c07d6bb8>] (device_add) from [<c0a1ff08>] (thermal_zone_device_register+0x160/0x564)
+[<c0a1ff08>] (thermal_zone_device_register) from [<c117e2f4>] (of_parse_thermal_zones+0x58c/0x840)
+[<c117e2f4>] (of_parse_thermal_zones) from [<c117dcfc>] (thermal_init+0x38/0xa4)
+[<c117dcfc>] (thermal_init) from [<c0301e6c>] (do_one_initcall+0x3c/0x16c)
+[<c0301e6c>] (do_one_initcall) from [<c1100d98>] (kernel_init_freeable+0x124/0x1f0)
+[<c1100d98>] (kernel_init_freeable) from [<c0bdf484>] (kernel_init+0x8/0x118)
+[<c0bdf484>] (kernel_init) from [<c0307fb8>] (ret_from_fork+0x14/0x3c)
+---[ end trace b343dbaf9f82349c ]---
+------------[ cut here ]------------
+WARNING: CPU: 0 PID: 1 at lib/kobject.c:240 kobject_add_internal+0x258/0x2e0
+kobject_add_internal failed for thermal_zone1 with -EEXIST, don't try to register things with the same name in the same directory.
+Modules linked in:
+CPU: 0 PID: 1 Comm: swapper/0 Tainted: G        W       4.9.0-next-20161215-141240-gdf5462d #1
+Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
+[<c030fb38>] (unwind_backtrace) from [<c030b198>] (show_stack+0x10/0x14)
+[<c030b198>] (show_stack) from [<c0596230>] (dump_stack+0x84/0xa4)
+[<c0596230>] (dump_stack) from [<c0341bdc>] (__warn+0xd4/0x100)
+[<c0341bdc>] (__warn) from [<c0341ca8>] (warn_slowpath_fmt+0x38/0x48)
+[<c0341ca8>] (warn_slowpath_fmt) from [<c0597f0c>] (kobject_add_internal+0x258/0x2e0)
+[<c0597f0c>] (kobject_add_internal) from [<c059815c>] (kobject_add+0x4c/0x9c)
+[<c059815c>] (kobject_add) from [<c07d6bb8>] (device_add+0xc0/0x54c)
+[<c07d6bb8>] (device_add) from [<c0a1ff08>] (thermal_zone_device_register+0x160/0x564)
+[<c0a1ff08>] (thermal_zone_device_register) from [<c117e2f4>] (of_parse_thermal_zones+0x58c/0x840)
+[<c117e2f4>] (of_parse_thermal_zones) from [<c117dcfc>] (thermal_init+0x38/0xa4)
+[<c117dcfc>] (thermal_init) from [<c0301e6c>] (do_one_initcall+0x3c/0x16c)
+[<c0301e6c>] (do_one_initcall) from [<c1100d98>] (kernel_init_freeable+0x124/0x1f0)
+[<c1100d98>] (kernel_init_freeable) from [<c0bdf484>] (kernel_init+0x8/0x118)
+[<c0bdf484>] (kernel_init) from [<c0307fb8>] (ret_from_fork+0x14/0x3c)
+---[ end trace b343dbaf9f82349d ]---
+Failed to build gpu zone -17
+------------[ cut here ]------------
+WARNING: CPU: 0 PID: 1 at fs/sysfs/dir.c:31 sysfs_warn_dup+0x58/0x74
+sysfs: cannot create duplicate filename '/devices/virtual/thermal/thermal_zone1'
+Modules linked in:
+CPU: 0 PID: 1 Comm: swapper/0 Tainted: G        W       4.9.0-next-20161215-141240-gdf5462d #1
+Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
+[<c030fb38>] (unwind_backtrace) from [<c030b198>] (show_stack+0x10/0x14)
+[<c030b198>] (show_stack) from [<c0596230>] (dump_stack+0x84/0xa4)
+[<c0596230>] (dump_stack) from [<c0341bdc>] (__warn+0xd4/0x100)
+[<c0341bdc>] (__warn) from [<c0341ca8>] (warn_slowpath_fmt+0x38/0x48)
+[<c0341ca8>] (warn_slowpath_fmt) from [<c0478280>] (sysfs_warn_dup+0x58/0x74)
+[<c0478280>] (sysfs_warn_dup) from [<c0478368>] (sysfs_create_dir_ns+0x88/0x90)
+[<c0478368>] (sysfs_create_dir_ns) from [<c0597d60>] (kobject_add_internal+0xac/0x2e0)
+[<c0597d60>] (kobject_add_internal) from [<c059815c>] (kobject_add+0x4c/0x9c)
+[<c059815c>] (kobject_add) from [<c07d6bb8>] (device_add+0xc0/0x54c)
+[<c07d6bb8>] (device_add) from [<c0a1ff08>] (thermal_zone_device_register+0x160/0x564)
+[<c0a1ff08>] (thermal_zone_device_register) from [<c117e2f4>] (of_parse_thermal_zones+0x58c/0x840)
+[<c117e2f4>] (of_parse_thermal_zones) from [<c117dcfc>] (thermal_init+0x38/0xa4)
+[<c117dcfc>] (thermal_init) from [<c0301e6c>] (do_one_initcall+0x3c/0x16c)
+[<c0301e6c>] (do_one_initcall) from [<c1100d98>] (kernel_init_freeable+0x124/0x1f0)
+[<c1100d98>] (kernel_init_freeable) from [<c0bdf484>] (kernel_init+0x8/0x118)
+[<c0bdf484>] (kernel_init) from [<c0307fb8>] (ret_from_fork+0x14/0x3c)
+---[ end trace b343dbaf9f82349e ]---
+------------[ cut here ]------------
+WARNING: CPU: 0 PID: 1 at lib/kobject.c:240 kobject_add_internal+0x258/0x2e0
+kobject_add_internal failed for thermal_zone1 with -EEXIST, don't try to register things with the same name in the same directory.
+Modules linked in:
+CPU: 0 PID: 1 Comm: swapper/0 Tainted: G        W       4.9.0-next-20161215-141240-gdf5462d #1
+Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
+[<c030fb38>] (unwind_backtrace) from [<c030b198>] (show_stack+0x10/0x14)
+[<c030b198>] (show_stack) from [<c0596230>] (dump_stack+0x84/0xa4)
+[<c0596230>] (dump_stack) from [<c0341bdc>] (__warn+0xd4/0x100)
+[<c0341bdc>] (__warn) from [<c0341ca8>] (warn_slowpath_fmt+0x38/0x48)
+[<c0341ca8>] (warn_slowpath_fmt) from [<c0597f0c>] (kobject_add_internal+0x258/0x2e0)
+[<c0597f0c>] (kobject_add_internal) from [<c059815c>] (kobject_add+0x4c/0x9c)
+[<c059815c>] (kobject_add) from [<c07d6bb8>] (device_add+0xc0/0x54c)
+[<c07d6bb8>] (device_add) from [<c0a1ff08>] (thermal_zone_device_register+0x160/0x564)
+[<c0a1ff08>] (thermal_zone_device_register) from [<c117e2f4>] (of_parse_thermal_zones+0x58c/0x840)
+[<c117e2f4>] (of_parse_thermal_zones) from [<c117dcfc>] (thermal_init+0x38/0xa4)
+[<c117dcfc>] (thermal_init) from [<c0301e6c>] (do_one_initcall+0x3c/0x16c)
+[<c0301e6c>] (do_one_initcall) from [<c1100d98>] (kernel_init_freeable+0x124/0x1f0)
+[<c1100d98>] (kernel_init_freeable) from [<c0bdf484>] (kernel_init+0x8/0x118)
+[<c0bdf484>] (kernel_init) from [<c0307fb8>] (ret_from_fork+0x14/0x3c)
+---[ end trace b343dbaf9f82349f ]---
+Failed to build pllx zone -17
 NET: Registered protocol family 2
 TCP established hash table entries: 8192 (order: 3, 32768 bytes)
 TCP bind hash table entries: 8192 (order: 4, 65536 bytes)
@@ -295,6 +385,8 @@
 mmc0: Unknown controller version (3). You may experience problems.
 mmc0: Invalid maximum block size, assuming 512 bytes
 mmc0: SDHCI controller on 700b0600.sdhci [700b0600.sdhci] using ADMA
+elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
+input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 ledtrig-cpu: registered to indicate activity on CPUs
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
@@ -308,8 +400,6 @@
 can: broadcast manager protocol (rev 20161123 t)
 can: netlink gateway (rev 20130117) max_hops=1
 Key type dns_resolver registered
-elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
-input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
 +3.3V_RUN: supplied by +3.3V_SYS
@@ -324,6 +414,13 @@
 +3.3V_PANEL: supplied by +3.3V_RUN
 +3.3V_LP0: supplied by +3.3V_SYS
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
+mmc0: new high speed MMC card at address 0001
+elan_i2c 1-0015: failed to read report data: -121
+mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
+mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
+mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
+mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 +5V_HDMI_CON: supplied by +5V_SYS
 usb3-0 supply vbus not found, using dummy regulator
 usb3-1 supply vbus not found, using dummy regulator
@@ -333,12 +430,7 @@
 mmc1: Unknown controller version (3). You may experience problems.
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: Switching to 3.3V signalling voltage failed
-mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
-mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
-mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
-mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
+elan_i2c 1-0015: invalid report id data (ff)
 mmc1: SDHCI controller on 700b0000.sdhci [700b0000.sdhci] using ADMA
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
@@ -360,7 +452,7 @@
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:24 UTC (946684824)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:23 UTC (946684823)
 +1.8V_RUN_CAM: disabling
 +3.3V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2016-12-15 03:43:22.849372850 -0800
+++ /tmp/foob	2016-12-15 03:43:22.849372850 -0800
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -42,18 +42,18 @@
 	 ##################################################
 	 3.5 MiB/s
 done
-Bytes transferred = 5504608 (53fe60 hex)
+Bytes transferred = 5504088 (53fc58 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.4 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 67943 (10967 hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x53fe60 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x53fc58 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013966
@@ -62,7 +62,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-next-20161214-140955-g9aa17b2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Dec 13 20:33:52 PST 2016
+Linux version 4.9.0-next-20161215-141240-gdf5462d (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Dec 15 03:33:23 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -137,7 +137,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?[    0.509515] hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
+No ATAGs?[    0.505304] hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 8 bytes.
 tegra-mc 70019000.memory-controller: no memory timings for RAM code 0 registered
 +5V_SYS: supplied by +VDD_MUX
@@ -284,52 +284,6 @@
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
 tegra30-i2s 70301100.i2s: DMA channels sourced from device 70300000.ahub
-BUG: scheduling while atomic: kworker/0:1/37/0x00000002
-Modules linked in:
-CPU: 0 PID: 37 Comm: kworker/0:1 Not tainted 4.9.0-next-20161214-140955-g9aa17b2 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-Workqueue: events_freezable mmc_rescan
-[<c010f308>] (unwind_backtrace) from [<c010a900>] (show_stack+0x10/0x14)
-[<c010a900>] (show_stack) from [<c034d43c>] (dump_stack+0x84/0xa4)
-[<c034d43c>] (dump_stack) from [<c01b5458>] (__schedule_bug+0x60/0x80)
-[<c01b5458>] (__schedule_bug) from [<c08281d8>] (__schedule+0x528/0x60c)
-[<c08281d8>] (__schedule) from [<c082848c>] (schedule+0x50/0xc0)
-[<c082848c>] (schedule) from [<c08286e8>] (schedule_preempt_disabled+0x24/0x40)
-[<c08286e8>] (schedule_preempt_disabled) from [<c082c650>] (__mutex_lock_slowpath+0x400/0x6f0)
-[<c082c650>] (__mutex_lock_slowpath) from [<c03c4800>] (clk_prepare_lock+0x3c/0xe4)
-[<c03c4800>] (clk_prepare_lock) from [<c03c6d08>] (clk_set_rate+0x14/0x30)
-[<c03c6d08>] (clk_set_rate) from [<c05dd1a0>] (tegra_sdhci_set_clock+0x28/0xdc)
-[<c05dd1a0>] (tegra_sdhci_set_clock) from [<c05dc308>] (sdhci_set_ios+0x2b8/0x57c)
-[<c05dc308>] (sdhci_set_ios) from [<c05c674c>] (mmc_set_bus_mode+0x18/0x1c)
-[<c05c674c>] (mmc_set_bus_mode) from [<c05c9650>] (mmc_init_card+0xb4/0x1824)
-[<c05c9650>] (mmc_init_card) from [<c05cb134>] (mmc_attach_mmc+0x8c/0x14c)
-[<c05cb134>] (mmc_attach_mmc) from [<c05c76b8>] (mmc_rescan+0x274/0x34c)
-[<c05c76b8>] (mmc_rescan) from [<c0134e74>] (process_one_work+0x120/0x31c)
-[<c0134e74>] (process_one_work) from [<c01350d4>] (worker_thread+0x2c/0x4b4)
-[<c01350d4>] (worker_thread) from [<c013a5e8>] (kthread+0xfc/0x134)
-[<c013a5e8>] (kthread) from [<c0107938>] (ret_from_fork+0x14/0x3c)
-BUG: scheduling while atomic: kworker/0:1/37/0x00000000
-Modules linked in:
-CPU: 0 PID: 37 Comm: kworker/0:1 Tainted: G        W       4.9.0-next-20161214-140955-g9aa17b2 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-Workqueue: events_freezable mmc_rescan
-[<c010f308>] (unwind_backtrace) from [<c010a900>] (show_stack+0x10/0x14)
-[<c010a900>] (show_stack) from [<c034d43c>] (dump_stack+0x84/0xa4)
-[<c034d43c>] (dump_stack) from [<c01b5458>] (__schedule_bug+0x60/0x80)
-[<c01b5458>] (__schedule_bug) from [<c08281d8>] (__schedule+0x528/0x60c)
-[<c08281d8>] (__schedule) from [<c082848c>] (schedule+0x50/0xc0)
-[<c082848c>] (schedule) from [<c082e02c>] (schedule_timeout+0x1c0/0x2e4)
-[<c082e02c>] (schedule_timeout) from [<c0828dd8>] (wait_for_common+0xbc/0x148)
-[<c0828dd8>] (wait_for_common) from [<c05c524c>] (mmc_wait_for_req_done+0x7c/0x104)
-[<c05c524c>] (mmc_wait_for_req_done) from [<c05c5410>] (mmc_wait_for_cmd+0x58/0x78)
-[<c05c5410>] (mmc_wait_for_cmd) from [<c05cbb04>] (mmc_go_idle+0x78/0xec)
-[<c05cbb04>] (mmc_go_idle) from [<c05c95d0>] (mmc_init_card+0x34/0x1824)
-[<c05c95d0>] (mmc_init_card) from [<c05cb134>] (mmc_attach_mmc+0x8c/0x14c)
-[<c05cb134>] (mmc_attach_mmc) from [<c05c76b8>] (mmc_rescan+0x274/0x34c)
-[<c05c76b8>] (mmc_rescan) from [<c0134e74>] (process_one_work+0x120/0x31c)
-[<c0134e74>] (process_one_work) from [<c01350d4>] (worker_thread+0x2c/0x4b4)
-[<c01350d4>] (worker_thread) from [<c013a5e8>] (kthread+0xfc/0x134)
-[<c013a5e8>] (kthread) from [<c0107938>] (ret_from_fork+0x14/0x3c)
 max98090 0-0010: MAX98090 REVID=0x43
 max98090 0-0010: use default 2.8v micbias
 tegra-snd-max98090 sound: HiFi <-> 70301100.i2s mapping ok
@@ -338,9 +292,9 @@
 NET: Registered protocol family 10
 Segment Routing with IPv6
 mip6: Mobile IPv6
-sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
+sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
 mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
 mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
@@ -416,7 +370,7 @@
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input4
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:22 UTC (946684822)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 07:58:03 UTC (946713483)
 +1.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
 +1.05V_RUN_CAM_REAR: disabling
@@ -437,4 +391,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# 
\ No newline at end of file
+root@(none):/# [    2.971520] usb 1-3: new high-speed USB device number 3 using tegra-xusb
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2016-12-15 03:43:22.857372881 -0800
+++ /tmp/foob	2016-12-15 03:43:22.857372881 -0800
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1800521 us
+Bootloader-Cpu Init at (time stamp): 1800695 us
 
 Pinmux changes applied in kernel way
 
@@ -59,23 +59,23 @@
 The proc BoardInfo: 0x0176:0x0001:0x01:0x43:0x00
 FastbootAddAtag: setting ATAG (0)
 
-Board Id Read Time: 4 us
+Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1800494  us
-Bootloader (AosInit) time: 733452 us
-Bootloader Main Init time: 180235 us
-BlockDevMgrInit time: 157533 us
-PartMgrInit time: 9 us
+Miscellaneous avp time: 1800669  us
+Bootloader (AosInit) time: 734240 us
+Bootloader Main Init time: 180096 us
+BlockDevMgrInit time: 157547 us
+PartMgrInit time: 8 us
 StorMgrInit time: 30 us
 FsMgrInit time: 2 us
-Partition Table load: 77119 us 
-Display init time: 574342 us
+Partition Table load: 76761 us 
+Display init time: 574665 us
 Kernel read time: 377 msec
 Load Kernel Size:   16304128 Bytes
 Bl Kernel ReadSpeed: 43247000 Bytes/sec
-Pre-Kernel Jump time: 1741131 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5844151 us
+Pre-Kernel Jump time: 1740907 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5845006 us
 Booting Linux on physical CPU 0x0
 Linux version 4.8.0-239303-g3477d16 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Fri Oct 7 01:12:37 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
@@ -184,7 +184,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(1.135:1): initialized
+audit: type=2000 audit(1.159:1): initialized
 workingset: timestamp_bits=46 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -251,10 +251,9 @@
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 +VDD_LED: supplied by +VDD_MUX
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 +5V_USB_HS: supplied by +5V_SYS
 +5V_USB_SS: supplied by +5V_SYS
 +5V_HDMI_CON: supplied by +5V_SYS
@@ -273,6 +272,9 @@
 Freeing unused kernel memory: 960K (ffff800000c00000 - ffff800000cf0000)
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
+sdhci-tegra 700b0400.sdhci: Got CD GPIO
+sdhci-tegra 700b0400.sdhci: Got WP GPIO
 
 System initialization...
 
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-12-15 03:43:22.865372911 -0800
+++ /tmp/foob	2016-12-15 03:43:22.865372911 -0800
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -49,7 +49,7 @@
 	 ##################################################
 	 4.1 MiB/s
 done
-Bytes transferred = 7398800 (70e590 hex)
+Bytes transferred = 7400056 (70ea78 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -60,7 +60,7 @@
 done
 Bytes transferred = 21404 (539c hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x70e590 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x70ea78 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 0241039b
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-next-20161214-140955-g9aa17b2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Dec 13 20:34:21 PST 2016
+Linux version 4.9.0-next-20161215-141240-gdf5462d (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Dec 15 03:33:13 PST 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: Compulab TrimSlice board
@@ -333,21 +333,21 @@
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
-tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
  mmcblk0: p1
+tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2015-02-06 02:12:55 UTC (1423188775)
-EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
+rtc-em3027 2-0056: setting system clock to 2015-02-06 02:20:41 UTC (1423189241)
+EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-12-15 03:43:22.873372942 -0800
+++ /tmp/foob	2016-12-15 03:43:22.873372942 -0800
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -45,9 +45,9 @@
 	 #################################################################
 	 #################################################################
 	 ##################################################
-	 4.1 MiB/s
+	 4 MiB/s
 done
-Bytes transferred = 5504608 (53fe60 hex)
+Bytes transferred = 5504088 (53fc58 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21404 (539c hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x53fe60 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x53fc58 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 0241039b
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-next-20161214-140955-g9aa17b2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Dec 13 20:33:52 PST 2016
+Linux version 4.9.0-next-20161215-141240-gdf5462d (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Dec 15 03:33:23 PST 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: Compulab TrimSlice board
@@ -139,7 +139,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?[    0.341151] hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
+No ATAGs?[    0.336410] hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 4 bytes.
 usb1_vbus: Failed to request enable GPIO170: -517
 reg-fixed-voltage regulators:regulator@2: Failed to register regulator: -517
@@ -309,20 +309,20 @@
 tlv320aic23-codec 2-001a: ASoC: no dapm match for RLINEIN --> NULL --> Line Input
 tlv320aic23-codec 2-001a: ASoC: Failed to add route RLINEIN -> NULL -> Line Input
 tlv320aic23-codec 2-001a: Control not supported for path MICIN -> [NULL] -> Mic Input
-tlv320aic23-codec 2-001a: ASoC: no dapm match for MICIN --> NULL --> Mic Input
-tlv320aic23-codec 2-001a: ASoC: Failed to add route MICIN -> NULL -> Mic Input
-tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
-NET: Registered protocol family 10
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
  mmcblk0: p1
+tlv320aic23-codec 2-001a: ASoC: no dapm match for MICIN --> NULL --> Mic Input
+tlv320aic23-codec 2-001a: ASoC: Failed to add route MICIN -> NULL -> Mic Input
+tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
+NET: Registered protocol family 10
 mmc1: new high speed SD card at address b368
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
- mmcblk1: p1 p2
 Segment Routing with IPv6
 mip6: Mobile IPv6
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
+ mmcblk1: p1 p2
 sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
 NET: Registered protocol family 17
 NET: Registered protocol family 15
@@ -349,7 +349,7 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2015-02-06 02:11:23 UTC (1423188683)
+rtc-em3027 2-0056: setting system clock to 2015-02-06 02:22:14 UTC (1423189334)
 ALSA device list:
   #0: tegra-trimslice
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2016-12-15 03:43:22.881372972 -0800
+++ /tmp/foob	2016-12-15 03:43:22.885372987 -0800
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 721344 us
-Basic modules init took 349429 us
+Tegraboot started after 858512 us
+Basic modules init took 350353 us
 Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14788 us
-Carveout took 252710 us
-CPU initialization took 92307 us
-Total time taken by TegraBoot 709234 us
+Next stage binary read took 14368 us
+Carveout took 252728 us
+CPU initialization took 92304 us
+Total time taken by TegraBoot 709753 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.562] LPDDR4 Training: Number of tables = 10
+64b[0001.696] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -119,9 +119,9 @@
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
 14987776 bytes read in 795 ms (18 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-44882 bytes read in 454 ms (95.7 KiB/s)
+44882 bytes read in 460 ms (94.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 500 ms (3.1 MiB/s)
+1627787 bytes read in 497 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-next-20161214-165388-g9aa17b2 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Dec 13 20:36:55 PST 2016
+Linux version 4.9.0-next-20161215-165673-gdf5462d (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Dec 15 03:36:19 PST 2016
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -241,6 +241,220 @@
 VFS: Disk quotas dquot_6.6.0
 VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
 pnp: PnP ACPI: disabled
+sysfs: cannot create duplicate filename '/devices/virtual/thermal/thermal_zone1'
+------------[ cut here ]------------
+WARNING: CPU: 0 PID: 1 at fs/sysfs/dir.c:31 sysfs_warn_dup+0x60/0x7c
+Modules linked in:
+
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.9.0-next-20161215-165673-gdf5462d #1
+Hardware name: NVIDIA Tegra210 P2371 (P2530/P2595) reference design (DT)
+task: ffff8000bc090000 task.stack: ffff8000bc098000
+PC is at sysfs_warn_dup+0x60/0x7c
+LR is at sysfs_warn_dup+0x60/0x7c
+pc : [<ffff000008256880>] lr : [<ffff000008256880>] pstate: 60000045
+sp : ffff8000bc09ba20
+x29: ffff8000bc09ba20 x28: ffff8000bc38d000 
+x27: ffff8000bc38d018 x26: ffff000008eb0178 
+x25: ffff000008eb0290 x24: ffff000008eb0250 
+x23: ffff000008f27000 x22: ffff8000bb9b1800 
+x21: ffff8000bb9b0528 x20: ffff8000bb9b4900 
+x19: ffff8000bb8ce000 x18: 000000000000123f 
+x17: 0000000000000001 x16: 0000000000000002 
+x15: 1f08282d15121b03 x14: 616d726568742f6c 
+x13: 616d726568742f6c x12: 6175747269762f73 
+x11: 6563697665642f27 x10: 20656d616e656c69 
+x9 : 0000000000000070 x8 : 7075642065746165 
+x7 : 726320746f6e6e61 x6 : ffff000008ed80e7 
+x5 : ffff000008d1bbb0 x4 : 0000000000000000 
+x3 : 0000000000000000 x2 : 0000000000000250 
+x1 : ffff8000bc090000 x0 : 0000000000000050 
+
+---[ end trace 42ee51915335d3fc ]---
+Call trace:
+Exception stack(0xffff8000bc09b850 to 0xffff8000bc09b980)
+b840:                                   ffff8000bb8ce000 0001000000000000
+b860: ffff8000bc09ba20 ffff000008256880 0000000000000006 0000000000000004
+b880: ffff000008bb1f18 0000000000000000 ffff000008de8268 0000000108ed7000
+b8a0: ffff8000bc09b940 ffff000008107b04 ffff8000bb8ce000 ffff8000bb9b4900
+b8c0: ffff8000bb9b0528 ffff8000bb9b1800 ffff000008f27000 ffff000008eb0250
+b8e0: ffff000008eb0290 ffff000008eb0178 0000000000000050 ffff8000bc090000
+b900: 0000000000000250 0000000000000000 0000000000000000 ffff000008d1bbb0
+b920: ffff000008ed80e7 726320746f6e6e61 7075642065746165 0000000000000070
+b940: 20656d616e656c69 6563697665642f27 6175747269762f73 616d726568742f6c
+b960: 616d726568742f6c 1f08282d15121b03 0000000000000002 0000000000000001
+[<ffff000008256880>] sysfs_warn_dup+0x60/0x7c
+[<ffff00000825699c>] sysfs_create_dir_ns+0x90/0xa0
+[<ffff00000837a458>] kobject_add_internal+0x94/0x298
+[<ffff00000837a8d4>] kobject_add+0x94/0xe0
+[<ffff00000851b848>] device_add+0xc0/0x578
+[<ffff00000851bd1c>] device_register+0x1c/0x28
+[<ffff00000871b044>] thermal_zone_device_register+0x1d4/0x5ac
+[<ffff000008d22090>] of_parse_thermal_zones+0x1f8/0x6d4
+[<ffff000008d21e2c>] thermal_init+0x68/0xd4
+[<ffff000008083144>] do_one_initcall+0x38/0x120
+[<ffff000008ce0d08>] kernel_init_freeable+0x1ac/0x24c
+[<ffff0000088b3468>] kernel_init+0x10/0xfc
+[<ffff000008082ec0>] ret_from_fork+0x10/0x50
+kobject_add_internal failed for thermal_zone1 with -EEXIST, don't try to register things with the same name in the same directory.
+------------[ cut here ]------------
+WARNING: CPU: 0 PID: 1 at lib/kobject.c:240 kobject_add_internal+0x174/0x298
+Modules linked in:
+
+CPU: 0 PID: 1 Comm: swapper/0 Tainted: G        W       4.9.0-next-20161215-165673-gdf5462d #1
+Hardware name: NVIDIA Tegra210 P2371 (P2530/P2595) reference design (DT)
+task: ffff8000bc090000 task.stack: ffff8000bc098000
+PC is at kobject_add_internal+0x174/0x298
+LR is at kobject_add_internal+0x174/0x298
+pc : [<ffff00000837a538>] lr : [<ffff00000837a538>] pstate: 60000045
+sp : ffff8000bc09ba80
+x29: ffff8000bc09ba80 x28: ffff8000bc38d000 
+x27: ffff8000bc38d018 x26: ffff000008eb0178 
+x25: ffff000008eb0290 x24: ffff000008eb0250 
+x23: ffff000008f27000 x22: ffff8000bb9b1800 
+x21: ffff8000bb9b1800 x20: 00000000ffffffef 
+x19: ffff8000bc38d028 x18: 000000000000123f 
+x17: 0000000000000001 x16: 0000000000000002 
+x15: 1f08282d15121b03 x14: 726f746365726964 
+x13: 20656d6173206568 x12: 74206e6920656d61 
+x11: 6e20656d61732065 x10: 6874206874697720 
+x9 : 00000000000000a6 x8 : 6574736967657220 
+x7 : 6f74207972742074 x6 : ffff000008ed8119 
+x5 : ffff000008d1bbb0 x4 : 0000000000000000 
+x3 : 0000000000000000 x2 : 00000000000003ca 
+x1 : ffff8000bc090000 x0 : 0000000000000082 
+
+---[ end trace 42ee51915335d3fd ]---
+Call trace:
+Exception stack(0xffff8000bc09b8b0 to 0xffff8000bc09b9e0)
+b8a0:                                   ffff8000bc38d028 0001000000000000
+b8c0: ffff8000bc09ba80 ffff00000837a538 0000000000000002 0000000000000004
+b8e0: ffff000008bb1f18 0000000000000000 ffff000008de8268 0000000108ed7000
+b900: ffff8000bc09b9a0 ffff000008107b04 ffff8000bc38d028 00000000ffffffef
+b920: ffff8000bb9b1800 ffff8000bb9b1800 ffff000008f27000 ffff000008eb0250
+b940: ffff000008eb0290 ffff000008eb0178 0000000000000082 ffff8000bc090000
+b960: 00000000000003ca 0000000000000000 0000000000000000 ffff000008d1bbb0
+b980: ffff000008ed8119 6f74207972742074 6574736967657220 00000000000000a6
+b9a0: 6874206874697720 6e20656d61732065 74206e6920656d61 20656d6173206568
+b9c0: 726f746365726964 1f08282d15121b03 0000000000000002 0000000000000001
+[<ffff00000837a538>] kobject_add_internal+0x174/0x298
+[<ffff00000837a8d4>] kobject_add+0x94/0xe0
+[<ffff00000851b848>] device_add+0xc0/0x578
+[<ffff00000851bd1c>] device_register+0x1c/0x28
+[<ffff00000871b044>] thermal_zone_device_register+0x1d4/0x5ac
+[<ffff000008d22090>] of_parse_thermal_zones+0x1f8/0x6d4
+[<ffff000008d21e2c>] thermal_init+0x68/0xd4
+[<ffff000008083144>] do_one_initcall+0x38/0x120
+[<ffff000008ce0d08>] kernel_init_freeable+0x1ac/0x24c
+[<ffff0000088b3468>] kernel_init+0x10/0xfc
+[<ffff000008082ec0>] ret_from_fork+0x10/0x50
+Failed to build gpu zone -17
+sysfs: cannot create duplicate filename '/devices/virtual/thermal/thermal_zone1'
+------------[ cut here ]------------
+WARNING: CPU: 0 PID: 1 at fs/sysfs/dir.c:31 sysfs_warn_dup+0x60/0x7c
+Modules linked in:
+
+CPU: 0 PID: 1 Comm: swapper/0 Tainted: G        W       4.9.0-next-20161215-165673-gdf5462d #1
+Hardware name: NVIDIA Tegra210 P2371 (P2530/P2595) reference design (DT)
+task: ffff8000bc090000 task.stack: ffff8000bc098000
+PC is at sysfs_warn_dup+0x60/0x7c
+LR is at sysfs_warn_dup+0x60/0x7c
+pc : [<ffff000008256880>] lr : [<ffff000008256880>] pstate: 60000045
+sp : ffff8000bc09ba20
+x29: ffff8000bc09ba20 x28: ffff8000bc38d000 
+x27: ffff8000bc38d018 x26: ffff000008eb0178 
+x25: ffff000008eb0290 x24: ffff000008eb0250 
+x23: ffff000008f27000 x22: ffff8000bb9b1800 
+x21: ffff8000bb9b0528 x20: ffff8000bb9b4b80 
+x19: ffff8000bb8ce000 x18: 000000000000123f 
+x17: 0000000000000001 x16: 0000000000000002 
+x15: 1f08282d15121b03 x14: 616d726568742f6c 
+x13: 6175747269762f73 x12: 6563697665642f27 
+x11: 20656d616e656c69 x10: 662065746163696c 
+x9 : 00000000000000db x8 : 2731656e6f7a5f6c 
+x7 : 616d726568742f6c x6 : ffff000008ed80e7 
+x5 : ffff000008d1bbb0 x4 : 0000000000000000 
+x3 : 0000000000000000 x2 : 000000000000053f 
+x1 : ffff8000bc090000 x0 : 0000000000000050 
+
+---[ end trace 42ee51915335d3fe ]---
+Call trace:
+Exception stack(0xffff8000bc09b850 to 0xffff8000bc09b980)
+b840:                                   ffff8000bb8ce000 0001000000000000
+b860: ffff8000bc09ba20 ffff000008256880 0000000000000006 0000000000000004
+b880: ffff000008bb1f18 0000000000000000 ffff000008de8268 0000000108ed7000
+b8a0: ffff8000bc09b940 ffff000008107b04 ffff8000bb8ce000 ffff8000bb9b4b80
+b8c0: ffff8000bb9b0528 ffff8000bb9b1800 ffff000008f27000 ffff000008eb0250
+b8e0: ffff000008eb0290 ffff000008eb0178 0000000000000050 ffff8000bc090000
+b900: 000000000000053f 0000000000000000 0000000000000000 ffff000008d1bbb0
+b920: ffff000008ed80e7 616d726568742f6c 2731656e6f7a5f6c 00000000000000db
+b940: 662065746163696c 20656d616e656c69 6563697665642f27 6175747269762f73
+b960: 616d726568742f6c 1f08282d15121b03 0000000000000002 0000000000000001
+[<ffff000008256880>] sysfs_warn_dup+0x60/0x7c
+[<ffff00000825699c>] sysfs_create_dir_ns+0x90/0xa0
+[<ffff00000837a458>] kobject_add_internal+0x94/0x298
+[<ffff00000837a8d4>] kobject_add+0x94/0xe0
+[<ffff00000851b848>] device_add+0xc0/0x578
+[<ffff00000851bd1c>] device_register+0x1c/0x28
+[<ffff00000871b044>] thermal_zone_device_register+0x1d4/0x5ac
+[<ffff000008d22090>] of_parse_thermal_zones+0x1f8/0x6d4
+[<ffff000008d21e2c>] thermal_init+0x68/0xd4
+[<ffff000008083144>] do_one_initcall+0x38/0x120
+[<ffff000008ce0d08>] kernel_init_freeable+0x1ac/0x24c
+[<ffff0000088b3468>] kernel_init+0x10/0xfc
+[<ffff000008082ec0>] ret_from_fork+0x10/0x50
+kobject_add_internal failed for thermal_zone1 with -EEXIST, don't try to register things with the same name in the same directory.
+------------[ cut here ]------------
+WARNING: CPU: 0 PID: 1 at lib/kobject.c:240 kobject_add_internal+0x174/0x298
+Modules linked in:
+
+CPU: 0 PID: 1 Comm: swapper/0 Tainted: G        W       4.9.0-next-20161215-165673-gdf5462d #1
+Hardware name: NVIDIA Tegra210 P2371 (P2530/P2595) reference design (DT)
+task: ffff8000bc090000 task.stack: ffff8000bc098000
+PC is at kobject_add_internal+0x174/0x298
+LR is at kobject_add_internal+0x174/0x298
+pc : [<ffff00000837a538>] lr : [<ffff00000837a538>] pstate: 60000045
+sp : ffff8000bc09ba80
+x29: ffff8000bc09ba80 x28: ffff8000bc38d000 
+x27: ffff8000bc38d018 x26: ffff000008eb0178 
+x25: ffff000008eb0290 x24: ffff000008eb0250 
+x23: ffff000008f27000 x22: ffff8000bb9b1800 
+x21: ffff8000bb9b1800 x20: 00000000ffffffef 
+x19: ffff8000bc38d028 x18: 000000000000123f 
+x17: 0000000000000001 x16: 0000000000000002 
+x15: 1f08282d15121b03 x14: 6f74207972742074 
+x13: 276e6f64202c5453 x12: 495845452d206874 
+x11: 69772031656e6f7a x10: 5f6c616d72656874 
+x9 : 0000000000000111 x8 : 20656d6173206568 
+x7 : 74206e6920656d61 x6 : ffff000008ed8119 
+x5 : ffff000008d1bbb0 x4 : 0000000000000000 
+x3 : 0000000000000000 x2 : 00000000000006b9 
+x1 : ffff8000bc090000 x0 : 0000000000000082 
+
+---[ end trace 42ee51915335d3ff ]---
+Call trace:
+Exception stack(0xffff8000bc09b8b0 to 0xffff8000bc09b9e0)
+b8a0:                                   ffff8000bc38d028 0001000000000000
+b8c0: ffff8000bc09ba80 ffff00000837a538 0000000000000002 0000000000000004
+b8e0: ffff000008bb1f18 0000000000000000 ffff000008de8268 0000000108ed7000
+b900: ffff8000bc09b9a0 ffff000008107b04 ffff8000bc38d028 00000000ffffffef
+b920: ffff8000bb9b1800 ffff8000bb9b1800 ffff000008f27000 ffff000008eb0250
+b940: ffff000008eb0290 ffff000008eb0178 0000000000000082 ffff8000bc090000
+b960: 00000000000006b9 0000000000000000 0000000000000000 ffff000008d1bbb0
+b980: ffff000008ed8119 74206e6920656d61 20656d6173206568 0000000000000111
+b9a0: 5f6c616d72656874 69772031656e6f7a 495845452d206874 276e6f64202c5453
+b9c0: 6f74207972742074 1f08282d15121b03 0000000000000002 0000000000000001
+[<ffff00000837a538>] kobject_add_internal+0x174/0x298
+[<ffff00000837a8d4>] kobject_add+0x94/0xe0
+[<ffff00000851b848>] device_add+0xc0/0x578
+[<ffff00000851bd1c>] device_register+0x1c/0x28
+[<ffff00000871b044>] thermal_zone_device_register+0x1d4/0x5ac
+[<ffff000008d22090>] of_parse_thermal_zones+0x1f8/0x6d4
+[<ffff000008d21e2c>] thermal_init+0x68/0xd4
+[<ffff000008083144>] do_one_initcall+0x38/0x120
+[<ffff000008ce0d08>] kernel_init_freeable+0x1ac/0x24c
+[<ffff0000088b3468>] kernel_init+0x10/0xfc
+[<ffff000008082ec0>] ret_from_fork+0x10/0x50
+Failed to build pllx zone -17
 NET: Registered protocol family 2
 TCP established hash table entries: 32768 (order: 6, 262144 bytes)
 TCP bind hash table entries: 32768 (order: 7, 524288 bytes)
@@ -262,7 +476,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.539:1): initialized
+audit: type=2000 audit(1.819:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=46 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -342,6 +556,6 @@
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
-bash: no job contr[    1.382018] random: fast init done
-ol in this shell
+bash: no job control in th[    2.664343] random: fast init done
+is shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2016-12-15 03:43:22.889373002 -0800
+++ /tmp/foob	2016-12-15 03:43:22.889373002 -0800
@@ -1,3 +1,9 @@
+ot CPU: AArch64 Processor [411fd071]
+efi: Getting EFI parameters from FDT:
+efi: UEFI not found.
+cma: Reserved 16 MiB at 0x00000000f5400000
+earlycon: uart0 at MMIO 0x0000000070006000 (options '115200n8')
+bootconsole [uart0] enabled
 psci: probing for conduit method from DT.
 psci: PSCIv1.0 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
@@ -49,7 +55,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x3e50: expected 0x2004f400
+boot misc3 0x3e70: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -62,7 +68,7 @@
 console [tty0] enabled
 bootconsole [uart0] disabled
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-next-20161214-165388-g9aa17b2 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Dec 13 20:36:55 PST 2016
+Linux version 4.9.0-next-20161215-165673-gdf5462d (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Dec 15 03:36:19 PST 2016
 Boot CPU: AArch64 Processor [411fd071]
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
@@ -120,7 +126,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x3e50: expected 0x2004f400
+boot misc3 0x3e70: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -186,6 +192,220 @@
 VFS: Disk quotas dquot_6.6.0
 VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
 pnp: PnP ACPI: disabled
+sysfs: cannot create duplicate filename '/devices/virtual/thermal/thermal_zone1'
+------------[ cut here ]------------
+WARNING: CPU: 3 PID: 1 at fs/sysfs/dir.c:31 sysfs_warn_dup+0x60/0x7c
+Modules linked in:
+
+CPU: 3 PID: 1 Comm: swapper/0 Not tainted 4.9.0-next-20161215-165673-gdf5462d #1
+Hardware name: Google Pixel C (DT)
+task: ffff8000bc098000 task.stack: ffff8000bc094000
+PC is at sysfs_warn_dup+0x60/0x7c
+LR is at sysfs_warn_dup+0x60/0x7c
+pc : [<ffff000008256880>] lr : [<ffff000008256880>] pstate: 60000045
+sp : ffff8000bc097a20
+x29: ffff8000bc097a20 x28: ffff8000bba02800 
+x27: ffff8000bba02818 x26: ffff000008eb0178 
+x25: ffff000008eb0290 x24: ffff000008eb0250 
+x23: ffff000008f27000 x22: ffff8000bba34780 
+x21: ffff8000bba33a50 x20: ffff8000bba36900 
+x19: ffff8000bc331000 x18: 000000000000123f 
+x17: 0000000000000001 x16: 0000000000000002 
+x15: 1f08282d15121b03 x14: 616d726568742f6c 
+x13: 6175747269762f73 x12: 6563697665642f27 
+x11: 20656d616e656c69 x10: 662065746163696c 
+x9 : 0000000000000087 x8 : 2731656e6f7a5f6c 
+x7 : 616d726568742f6c x6 : ffff8000bc0110a0 
+x5 : 0000000000000000 x4 : 0000000000000000 
+x3 : 0000000000000000 x2 : 00000000000002c4 
+x1 : ffff8000bc098000 x0 : 0000000000000050 
+
+---[ end trace f12c2ef5c791e66b ]---
+Call trace:
+Exception stack(0xffff8000bc097850 to 0xffff8000bc097980)
+7840:                                   ffff8000bc331000 0001000000000000
+7860: ffff8000bc097a20 ffff000008256880 0000000000000006 0000000000000004
+7880: ffff000008bb1f18 0000000000000000 ffff000008de8268 0000000108ed7000
+78a0: ffff8000bc097940 ffff000008107b04 ffff8000bc331000 ffff8000bba36900
+78c0: ffff8000bba33a50 ffff8000bba34780 ffff000008f27000 ffff000008eb0250
+78e0: ffff000008eb0290 ffff000008eb0178 0000000000000050 ffff8000bc098000
+7900: 00000000000002c4 0000000000000000 0000000000000000 0000000000000000
+7920: ffff8000bc0110a0 616d726568742f6c 2731656e6f7a5f6c 0000000000000087
+7940: 662065746163696c 20656d616e656c69 6563697665642f27 6175747269762f73
+7960: 616d726568742f6c 1f08282d15121b03 0000000000000002 0000000000000001
+[<ffff000008256880>] sysfs_warn_dup+0x60/0x7c
+[<ffff00000825699c>] sysfs_create_dir_ns+0x90/0xa0
+[<ffff00000837a458>] kobject_add_internal+0x94/0x298
+[<ffff00000837a8d4>] kobject_add+0x94/0xe0
+[<ffff00000851b848>] device_add+0xc0/0x578
+[<ffff00000851bd1c>] device_register+0x1c/0x28
+[<ffff00000871b044>] thermal_zone_device_register+0x1d4/0x5ac
+[<ffff000008d22090>] of_parse_thermal_zones+0x1f8/0x6d4
+[<ffff000008d21e2c>] thermal_init+0x68/0xd4
+[<ffff000008083144>] do_one_initcall+0x38/0x120
+[<ffff000008ce0d08>] kernel_init_freeable+0x1ac/0x24c
+[<ffff0000088b3468>] kernel_init+0x10/0xfc
+[<ffff000008082ec0>] ret_from_fork+0x10/0x50
+kobject_add_internal failed for thermal_zone1 with -EEXIST, don't try to register things with the same name in the same directory.
+------------[ cut here ]------------
+WARNING: CPU: 3 PID: 1 at lib/kobject.c:240 kobject_add_internal+0x174/0x298
+Modules linked in:
+
+CPU: 3 PID: 1 Comm: swapper/0 Tainted: G        W       4.9.0-next-20161215-165673-gdf5462d #1
+Hardware name: Google Pixel C (DT)
+task: ffff8000bc098000 task.stack: ffff8000bc094000
+PC is at kobject_add_internal+0x174/0x298
+LR is at kobject_add_internal+0x174/0x298
+pc : [<ffff00000837a538>] lr : [<ffff00000837a538>] pstate: 60000045
+sp : ffff8000bc097a80
+x29: ffff8000bc097a80 x28: ffff8000bba02800 
+x27: ffff8000bba02818 x26: ffff000008eb0178 
+x25: ffff000008eb0290 x24: ffff000008eb0250 
+x23: ffff000008f27000 x22: ffff8000bba34780 
+x21: ffff8000bba34780 x20: 00000000ffffffef 
+x19: ffff8000bba02828 x18: 000000000000123f 
+x17: 0000000000000001 x16: 0000000000000002 
+x15: 1f08282d15121b03 x14: 676572206f742079 
+x13: 72742074276e6f64 x12: 202c545349584545 
+x11: 2d20687469772031 x10: 656e6f7a5f6c616d 
+x9 : 00000000000000bd x8 : 6572696420656d61 
+x7 : 7320656874206e69 x6 : ffff8000bc0110a0 
+x5 : 0000000000000000 x4 : 0000000000000000 
+x3 : 0000000000000000 x2 : 000000000000043e 
+x1 : ffff8000bc098000 x0 : 0000000000000082 
+
+---[ end trace f12c2ef5c791e66c ]---
+Call trace:
+Exception stack(0xffff8000bc0978b0 to 0xffff8000bc0979e0)
+78a0:                                   ffff8000bba02828 0001000000000000
+78c0: ffff8000bc097a80 ffff00000837a538 0000000000000002 0000000000000004
+78e0: ffff000008bb1f18 0000000000000000 ffff000008de8268 0000000108ed7000
+7900: ffff8000bc0979a0 ffff000008107b04 ffff8000bba02828 00000000ffffffef
+7920: ffff8000bba34780 ffff8000bba34780 ffff000008f27000 ffff000008eb0250
+7940: ffff000008eb0290 ffff000008eb0178 0000000000000082 ffff8000bc098000
+7960: 000000000000043e 0000000000000000 0000000000000000 0000000000000000
+7980: ffff8000bc0110a0 7320656874206e69 6572696420656d61 00000000000000bd
+79a0: 656e6f7a5f6c616d 2d20687469772031 202c545349584545 72742074276e6f64
+79c0: 676572206f742079 1f08282d15121b03 0000000000000002 0000000000000001
+[<ffff00000837a538>] kobject_add_internal+0x174/0x298
+[<ffff00000837a8d4>] kobject_add+0x94/0xe0
+[<ffff00000851b848>] device_add+0xc0/0x578
+[<ffff00000851bd1c>] device_register+0x1c/0x28
+[<ffff00000871b044>] thermal_zone_device_register+0x1d4/0x5ac
+[<ffff000008d22090>] of_parse_thermal_zones+0x1f8/0x6d4
+[<ffff000008d21e2c>] thermal_init+0x68/0xd4
+[<ffff000008083144>] do_one_initcall+0x38/0x120
+[<ffff000008ce0d08>] kernel_init_freeable+0x1ac/0x24c
+[<ffff0000088b3468>] kernel_init+0x10/0xfc
+[<ffff000008082ec0>] ret_from_fork+0x10/0x50
+Failed to build gpu zone -17
+sysfs: cannot create duplicate filename '/devices/virtual/thermal/thermal_zone1'
+------------[ cut here ]------------
+WARNING: CPU: 3 PID: 1 at fs/sysfs/dir.c:31 sysfs_warn_dup+0x60/0x7c
+Modules linked in:
+
+CPU: 3 PID: 1 Comm: swapper/0 Tainted: G        W       4.9.0-next-20161215-165673-gdf5462d #1
+Hardware name: Google Pixel C (DT)
+task: ffff8000bc098000 task.stack: ffff8000bc094000
+PC is at sysfs_warn_dup+0x60/0x7c
+LR is at sysfs_warn_dup+0x60/0x7c
+pc : [<ffff000008256880>] lr : [<ffff000008256880>] pstate: 60000045
+sp : ffff8000bc097a20
+x29: ffff8000bc097a20 x28: ffff8000bba02800 
+x27: ffff8000bba02818 x26: ffff000008eb0178 
+x25: ffff000008eb0290 x24: ffff000008eb0250 
+x23: ffff000008f27000 x22: ffff8000bba34780 
+x21: ffff8000bba33a50 x20: ffff8000bba36b80 
+x19: ffff8000bc331000 x18: 000000000000123f 
+x17: 0000000000000001 x16: 0000000000000002 
+x15: 1f08282d15121b03 x14: 616d726568742f6c 
+x13: 6175747269762f73 x12: 6563697665642f27 
+x11: 20656d616e656c69 x10: 662065746163696c 
+x9 : 00000000000000f2 x8 : 2731656e6f7a5f6c 
+x7 : 616d726568742f6c x6 : ffff8000bc0110a0 
+x5 : 0000000000000000 x4 : 0000000000000000 
+x3 : 0000000000000000 x2 : 00000000000005b3 
+x1 : ffff8000bc098000 x0 : 0000000000000050 
+
+---[ end trace f12c2ef5c791e66d ]---
+Call trace:
+Exception stack(0xffff8000bc097850 to 0xffff8000bc097980)
+7840:                                   ffff8000bc331000 0001000000000000
+7860: ffff8000bc097a20 ffff000008256880 0000000000000006 0000000000000004
+7880: ffff000008bb1f18 0000000000000000 ffff000008de8268 0000000108ed7000
+78a0: ffff8000bc097940 ffff000008107b04 ffff8000bc331000 ffff8000bba36b80
+78c0: ffff8000bba33a50 ffff8000bba34780 ffff000008f27000 ffff000008eb0250
+78e0: ffff000008eb0290 ffff000008eb0178 0000000000000050 ffff8000bc098000
+7900: 00000000000005b3 0000000000000000 0000000000000000 0000000000000000
+7920: ffff8000bc0110a0 616d726568742f6c 2731656e6f7a5f6c 00000000000000f2
+7940: 662065746163696c 20656d616e656c69 6563697665642f27 6175747269762f73
+7960: 616d726568742f6c 1f08282d15121b03 0000000000000002 0000000000000001
+[<ffff000008256880>] sysfs_warn_dup+0x60/0x7c
+[<ffff00000825699c>] sysfs_create_dir_ns+0x90/0xa0
+[<ffff00000837a458>] kobject_add_internal+0x94/0x298
+[<ffff00000837a8d4>] kobject_add+0x94/0xe0
+[<ffff00000851b848>] device_add+0xc0/0x578
+[<ffff00000851bd1c>] device_register+0x1c/0x28
+[<ffff00000871b044>] thermal_zone_device_register+0x1d4/0x5ac
+[<ffff000008d22090>] of_parse_thermal_zones+0x1f8/0x6d4
+[<ffff000008d21e2c>] thermal_init+0x68/0xd4
+[<ffff000008083144>] do_one_initcall+0x38/0x120
+[<ffff000008ce0d08>] kernel_init_freeable+0x1ac/0x24c
+[<ffff0000088b3468>] kernel_init+0x10/0xfc
+[<ffff000008082ec0>] ret_from_fork+0x10/0x50
+kobject_add_internal failed for thermal_zone1 with -EEXIST, don't try to register things with the same name in the same directory.
+------------[ cut here ]------------
+WARNING: CPU: 3 PID: 1 at lib/kobject.c:240 kobject_add_internal+0x174/0x298
+Modules linked in:
+
+CPU: 3 PID: 1 Comm: swapper/0 Tainted: G        W       4.9.0-next-20161215-165673-gdf5462d #1
+Hardware name: Google Pixel C (DT)
+task: ffff8000bc098000 task.stack: ffff8000bc094000
+PC is at kobject_add_internal+0x174/0x298
+LR is at kobject_add_internal+0x174/0x298
+pc : [<ffff00000837a538>] lr : [<ffff00000837a538>] pstate: 60000045
+sp : ffff8000bc097a80
+x29: ffff8000bc097a80 x28: ffff8000bba02800 
+x27: ffff8000bba02818 x26: ffff000008eb0178 
+x25: ffff000008eb0290 x24: ffff000008eb0250 
+x23: ffff000008f27000 x22: ffff8000bba34780 
+x21: ffff8000bba34780 x20: 00000000ffffffef 
+x19: ffff8000bba02828 x18: 000000000000123f 
+x17: 0000000000000001 x16: 0000000000000002 
+x15: 1f08282d15121b03 x14: 676572206f742079 
+x13: 72742074276e6f64 x12: 202c545349584545 
+x11: 2d20687469772031 x10: 656e6f7a5f6c616d 
+x9 : 0000000000000128 x8 : 6572696420656d61 
+x7 : 7320656874206e69 x6 : ffff8000bc0110a0 
+x5 : 0000000000000000 x4 : 0000000000000000 
+x3 : 0000000000000000 x2 : 000000000000072d 
+x1 : ffff8000bc098000 x0 : 0000000000000082 
+
+---[ end trace f12c2ef5c791e66e ]---
+Call trace:
+Exception stack(0xffff8000bc0978b0 to 0xffff8000bc0979e0)
+78a0:                                   ffff8000bba02828 0001000000000000
+78c0: ffff8000bc097a80 ffff00000837a538 0000000000000002 0000000000000004
+78e0: ffff000008bb1f18 0000000000000000 ffff000008de8268 0000000108ed7000
+7900: ffff8000bc0979a0 ffff000008107b04 ffff8000bba02828 00000000ffffffef
+7920: ffff8000bba34780 ffff8000bba34780 ffff000008f27000 ffff000008eb0250
+7940: ffff000008eb0290 ffff000008eb0178 0000000000000082 ffff8000bc098000
+7960: 000000000000072d 0000000000000000 0000000000000000 0000000000000000
+7980: ffff8000bc0110a0 7320656874206e69 6572696420656d61 0000000000000128
+79a0: 656e6f7a5f6c616d 2d20687469772031 202c545349584545 72742074276e6f64
+79c0: 676572206f742079 1f08282d15121b03 0000000000000002 0000000000000001
+[<ffff00000837a538>] kobject_add_internal+0x174/0x298
+[<ffff00000837a8d4>] kobject_add+0x94/0xe0
+[<ffff00000851b848>] device_add+0xc0/0x578
+[<ffff00000851bd1c>] device_register+0x1c/0x28
+[<ffff00000871b044>] thermal_zone_device_register+0x1d4/0x5ac
+[<ffff000008d22090>] of_parse_thermal_zones+0x1f8/0x6d4
+[<ffff000008d21e2c>] thermal_init+0x68/0xd4
+[<ffff000008083144>] do_one_initcall+0x38/0x120
+[<ffff000008ce0d08>] kernel_init_freeable+0x1ac/0x24c
+[<ffff0000088b3468>] kernel_init+0x10/0xfc
+[<ffff000008082ec0>] ret_from_fork+0x10/0x50
+Failed to build pllx zone -17
 NET: Registered protocol family 2
 TCP established hash table entries: 32768 (order: 6, 262144 bytes)
 TCP bind hash table entries: 32768 (order: 7, 524288 bytes)
@@ -208,7 +428,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.363:1): initialized
+audit: type=2000 audit(0.371:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=46 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -277,8 +497,136 @@
 tegra_rtc 7000e000.rtc: rtc core: registered 7000e000.rtc as rtc1
 tegra_rtc 7000e000.rtc: Tegra internal Real Time Clock
 i2c /dev entries driver
-(NULL device *): hwmon_device_register() is deprecated. Please convert the driver to use hwmon_device_register_with_info().
-bq27xxx-battery 2-0055: support ver. 1.2.0 enabled
+sysfs: cannot create duplicate filename '/devices/virtual/thermal/thermal_zone1'
+------------[ cut here ]------------
+WARNING: CPU: 3 PID: 1 at fs/sysfs/dir.c:31 sysfs_warn_dup+0x60/0x7c
+Modules linked in:
+
+CPU: 3 PID: 1 Comm: swapper/0 Tainted: G        W       4.9.0-next-20161215-165673-gdf5462d #1
+Hardware name: Google Pixel C (DT)
+task: ffff8000bc098000 task.stack: ffff8000bc094000
+PC is at sysfs_warn_dup+0x60/0x7c
+LR is at sysfs_warn_dup+0x60/0x7c
+pc : [<ffff000008256880>] lr : [<ffff000008256880>] pstate: 60000145
+sp : ffff8000bc0978b0
+x29: ffff8000bc0978b0 x28: ffff8000bb257800 
+x27: ffff8000bb257818 x26: ffff000008eb0178 
+x25: ffff000008eb0290 x24: ffff000008eb0250 
+x23: ffff000008f27000 x22: ffff8000bba34780 
+x21: ffff8000bba33a50 x20: ffff8000bb2e0080 
+x19: ffff8000bb874000 x18: 0000000000000007 
+x17: 000000000000000e x16: ffff000008f77fff 
+x15: ffff800001000000 x14: 616d726568742f6c 
+x13: 616d726568742f6c x12: 6175747269762f73 
+x11: 6563697665642f27 x10: 20656d616e656c69 
+x9 : 00000000000001b9 x8 : 7075642065746165 
+x7 : 726320746f6e6e61 x6 : ffff8000bc0110a0 
+x5 : 0000000000000000 x4 : 0000000000000000 
+x3 : 0000000000000000 x2 : 0000000000000beb 
+x1 : ffff8000bc098000 x0 : 0000000000000050 
+
+---[ end trace f12c2ef5c791e66f ]---
+Call trace:
+Exception stack(0xffff8000bc0976e0 to 0xffff8000bc097810)
+76e0: ffff8000bb874000 0001000000000000 ffff8000bc0978b0 ffff000008256880
+7700: 0000000000000006 0000000000000004 ffff000008bb1f18 0000000000000000
+7720: ffff000008de8268 0000000108ed7000 ffff8000bc0977d0 ffff000008107b04
+7740: ffff8000bb874000 ffff8000bb2e0080 ffff8000bba33a50 ffff8000bba34780
+7760: ffff000008f27000 ffff000008eb0250 ffff000008eb0290 ffff000008eb0178
+7780: 0000000000000050 ffff8000bc098000 0000000000000beb 0000000000000000
+77a0: 0000000000000000 0000000000000000 ffff8000bc0110a0 726320746f6e6e61
+77c0: 7075642065746165 00000000000001b9 20656d616e656c69 6563697665642f27
+77e0: 6175747269762f73 616d726568742f6c 616d726568742f6c ffff800001000000
+7800: ffff000008f77fff 000000000000000e
+[<ffff000008256880>] sysfs_warn_dup+0x60/0x7c
+[<ffff00000825699c>] sysfs_create_dir_ns+0x90/0xa0
+[<ffff00000837a458>] kobject_add_internal+0x94/0x298
+[<ffff00000837a8d4>] kobject_add+0x94/0xe0
+[<ffff00000851b848>] device_add+0xc0/0x578
+[<ffff00000851bd1c>] device_register+0x1c/0x28
+[<ffff00000871b044>] thermal_zone_device_register+0x1d4/0x5ac
+[<ffff0000087168d8>] __power_supply_register+0x358/0x420
+[<ffff0000087169c8>] power_supply_register_no_ws+0x10/0x18
+[<ffff000008717dbc>] bq27xxx_battery_setup+0x114/0x1a0
+[<ffff0000087185ac>] bq27xxx_battery_i2c_probe+0xcc/0x1ac
+[<ffff000008700c8c>] i2c_device_probe+0x238/0x288
+[<ffff00000851e930>] driver_probe_device+0x220/0x2b8
+[<ffff00000851ea6c>] __driver_attach+0xa4/0xa8
+[<ffff00000851c984>] bus_for_each_dev+0x58/0x98
+[<ffff00000851e26c>] driver_attach+0x20/0x28
+[<ffff00000851de28>] bus_add_driver+0x1c8/0x22c
+[<ffff00000851f384>] driver_register+0x60/0xf4
+[<ffff0000087017ec>] i2c_register_driver+0x44/0x84
+[<ffff000008d21d54>] bq27xxx_battery_i2c_driver_init+0x18/0x20
+[<ffff000008083144>] do_one_initcall+0x38/0x120
+[<ffff000008ce0d08>] kernel_init_freeable+0x1ac/0x24c
+[<ffff0000088b3468>] kernel_init+0x10/0xfc
+[<ffff000008082ec0>] ret_from_fork+0x10/0x50
+kobject_add_internal failed for thermal_zone1 with -EEXIST, don't try to register things with the same name in the same directory.
+------------[ cut here ]------------
+WARNING: CPU: 3 PID: 1 at lib/kobject.c:240 kobject_add_internal+0x174/0x298
+Modules linked in:
+
+CPU: 3 PID: 1 Comm: swapper/0 Tainted: G        W       4.9.0-next-20161215-165673-gdf5462d #1
+Hardware name: Google Pixel C (DT)
+task: ffff8000bc098000 task.stack: ffff8000bc094000
+PC is at kobject_add_internal+0x174/0x298
+LR is at kobject_add_internal+0x174/0x298
+pc : [<ffff00000837a538>] lr : [<ffff00000837a538>] pstate: 60000145
+sp : ffff8000bc097910
+x29: ffff8000bc097910 x28: ffff8000bb257800 
+x27: ffff8000bb257818 x26: ffff000008eb0178 
+x25: ffff000008eb0290 x24: ffff000008eb0250 
+x23: ffff000008f27000 x22: ffff8000bba34780 
+x21: ffff8000bba34780 x20: 00000000ffffffef 
+x19: ffff8000bb257828 x18: 0000000000000007 
+x17: 000000000000000e x16: ffff000008f77fff 
+x15: ffff800001000000 x14: 676572206f742079 
+x13: 72742074276e6f64 x12: 202c545349584545 
+x11: 2d20687469772031 x10: 656e6f7a5f6c616d 
+x9 : 00000000000001fa x8 : 6572696420656d61 
+x7 : 7320656874206e69 x6 : ffff8000bc0110a0 
+x5 : 0000000000000000 x4 : 0000000000000000 
+x3 : 0000000000000000 x2 : 0000000000000d9c 
+x1 : ffff8000bc098000 x0 : 0000000000000082 
+
+---[ end trace f12c2ef5c791e670 ]---
+Call trace:
+Exception stack(0xffff8000bc097740 to 0xffff8000bc097870)
+7740: ffff8000bb257828 0001000000000000 ffff8000bc097910 ffff00000837a538
+7760: 0000000000000002 0000000000000004 ffff000008bb1f18 0000000000000000
+7780: ffff000008de8268 0000000108ed7000 ffff8000bc097830 ffff000008107b04
+77a0: ffff8000bb257828 00000000ffffffef ffff8000bba34780 ffff8000bba34780
+77c0: ffff000008f27000 ffff000008eb0250 ffff000008eb0290 ffff000008eb0178
+77e0: 0000000000000082 ffff8000bc098000 0000000000000d9c 0000000000000000
+7800: 0000000000000000 0000000000000000 ffff8000bc0110a0 7320656874206e69
+7820: 6572696420656d61 00000000000001fa 656e6f7a5f6c616d 2d20687469772031
+7840: 202c545349584545 72742074276e6f64 676572206f742079 ffff800001000000
+7860: ffff000008f77fff 000000000000000e
+[<ffff00000837a538>] kobject_add_internal+0x174/0x298
+[<ffff00000837a8d4>] kobject_add+0x94/0xe0
+[<ffff00000851b848>] device_add+0xc0/0x578
+[<ffff00000851bd1c>] device_register+0x1c/0x28
+[<ffff00000871b044>] thermal_zone_device_register+0x1d4/0x5ac
+[<ffff0000087168d8>] __power_supply_register+0x358/0x420
+[<ffff0000087169c8>] power_supply_register_no_ws+0x10/0x18
+[<ffff000008717dbc>] bq27xxx_battery_setup+0x114/0x1a0
+[<ffff0000087185ac>] bq27xxx_battery_i2c_probe+0xcc/0x1ac
+[<ffff000008700c8c>] i2c_device_probe+0x238/0x288
+[<ffff00000851e930>] driver_probe_device+0x220/0x2b8
+[<ffff00000851ea6c>] __driver_attach+0xa4/0xa8
+[<ffff00000851c984>] bus_for_each_dev+0x58/0x98
+[<ffff00000851e26c>] driver_attach+0x20/0x28
+[<ffff00000851de28>] bus_add_driver+0x1c8/0x22c
+[<ffff00000851f384>] driver_register+0x60/0xf4
+[<ffff0000087017ec>] i2c_register_driver+0x44/0x84
+[<ffff000008d21d54>] bq27xxx_battery_i2c_driver_init+0x18/0x20
+[<ffff000008083144>] do_one_initcall+0x38/0x120
+[<ffff000008ce0d08>] kernel_init_freeable+0x1ac/0x24c
+[<ffff0000088b3468>] kernel_init+0x10/0xfc
+[<ffff000008082ec0>] ret_from_fork+0x10/0x50
+bq27xxx-battery 2-0055: failed to register battery
+bq27xxx-battery: probe of 2-0055 failed with error -17
 sdhci: Secure Digital Host Controller Interface driver
 sdhci: Copyright(c) Pierre Ossman
 Synopsys Designware Multimedia Card Interface Driver
@@ -296,9 +644,9 @@
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
 tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 mmc0: new high speed MMC card at address 0001
+max77686-rtc max77620-rtc: setting system clock to 2000-09-25 17:04:49 UTC (969901489)
 mmcblk0: mmc0:0001 BGND3R 29.1 GiB 
 mmcblk0boot0: mmc0:0001 BGND3R partition 1 4.00 MiB
-max77686-rtc max77620-rtc: setting system clock to 2000-09-24 10:05:40 UTC (969789940)
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-12-15 03:43:22.897373034 -0800
+++ /tmp/foob	2016-12-15 03:43:22.897373034 -0800
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 ##################################################
-	 3.3 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 7398800 (70e590 hex)
+Bytes transferred = 7400056 (70ea78 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 51804 (ca5c hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x70e590 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x70ea78 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa5b ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-next-20161214-140955-g9aa17b2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Dec 13 20:34:21 PST 2016
+Linux version 4.9.0-next-20161215-141240-gdf5462d (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Dec 15 03:33:13 PST 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -306,18 +306,18 @@
 +VDD_5V_HDMI: supplied by sys_3v3,vdd_3v3_alw
 sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
 tegra-pcie 3000.pcie-controller: 2x3 configuration
-tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
+mmc0: new ultra high speed DDR50 SDHC card at address aaaa
+mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB
-mmc0: new ultra high speed DDR50 SDHC card at address aaaa
-mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
 Primary GPT is invalid, using alternate GPT.
-tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
  mmcblk1: p1 p2 p3 p4 p5
  mmcblk0: p1 p2 p3
+tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
+tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -390,10 +390,10 @@
 EXT4-fs (mmcblk1p1): warning: mounting fs with errors, running e2fsck is recommended
 EXT4-fs (mmcblk1p1): recovery complete
 EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:1.
+VFS: Mounted root (ext3 filesystem) on device 179:17.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.143613] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.213952] usb 2-1: new high-speed USB device number 2 using tegra-ehci
 asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0f:67:60
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-12-15 03:43:22.905373064 -0800
+++ /tmp/foob	2016-12-15 03:43:22.905373064 -0800
@@ -39,7 +39,7 @@
 	 ##################################################
 	 3.2 MiB/s
 done
-Bytes transferred = 5504608 (53fe60 hex)
+Bytes transferred = 5504088 (53fc58 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 51804 (ca5c hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x53fe60 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x53fc58 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa5b ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-next-20161214-140955-g9aa17b2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Dec 13 20:33:52 PST 2016
+Linux version 4.9.0-next-20161215-141240-gdf5462d (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Dec 15 03:33:23 PST 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -131,7 +131,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?[    0.493249] hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
+No ATAGs?[    0.499428] hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 4 bytes.
 tegra-mc 7000f000.memory-controller: no memory timings for RAM code 3 registered
 vdd_5v_sata: Failed to request enable GPIO30: -517
@@ -390,4 +390,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.460290] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.480138] usb 2-1: new high-speed USB device number 2 using tegra-ehci
