Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2015-12-18 00:24:16.345848351 -0800
+++ /tmp/foob	2015-12-18 00:24:16.349848347 -0800
@@ -1,6 +1,6 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
-Linux version 4.4.0-rc5-next-20151217+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Dec 16 23:19:09 PST 2015
+Linux version 4.4.0-rc5-next-20151218+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Fri Dec 18 00:19:21 PST 2015
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -20,7 +20,7 @@
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
 software IO TLB [mem 0x79c00000-0x7dc00000] (64MB) mapped at [ffffffc039c00000-ffffffc03dbfffff]
-Memory: 938088K/1048576K available (5924K kernel code, 558K rwdata, 2572K rodata, 676K init, 220K bss, 94104K reserved, 16384K cma-reserved)
+Memory: 938072K/1048576K available (5928K kernel code, 567K rwdata, 2572K rodata, 680K init, 219K bss, 94120K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     vmalloc : 0xffffff8000000000 - 0xffffffbdbfff0000   (   246 GB)
     vmemmap : 0xffffffbdc0000000 - 0xffffffbfc0000000   (     8 GB maximum)
@@ -29,9 +29,9 @@
     PCI I/O : 0xffffffbffae00000 - 0xffffffbffbe00000   (    16 MB)
     modules : 0xffffffbffc000000 - 0xffffffc000000000   (    64 MB)
     memory  : 0xffffffc000000000 - 0xffffffc040000000   (  1024 MB)
-      .init : 0xffffffc0008cf000 - 0xffffffc000978000   (   676 KB)
-      .text : 0xffffffc000080000 - 0xffffffc0008ce0b4   (  8505 KB)
-      .data : 0xffffffc000978000 - 0xffffffc000a03a00   (   559 KB)
+      .init : 0xffffffc0008d0000 - 0xffffffc00097a000   (   680 KB)
+      .text : 0xffffffc000080000 - 0xffffffc0008cf0b4   (  8509 KB)
+      .data : 0xffffffc00097a000 - 0xffffffc000a07c00   (   567 KB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 64.
@@ -62,12 +62,13 @@
 NET: Registered protocol family 16
 cpuidle: using governor ladder
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffffffc00097d000, 1 data @ ffffffc00097c000)
+vdso: 2 pages (1 code @ ffffffc000981000, 1 data @ ffffffc000980000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
 9000000.pl011: ttyAMA0 at MMIO 0x9000000 (irq = 38, base_baud = 0) is a PL011 rev1
 console [ttyAMA0] enabled
+HugeTLB registered 2 MB page size, pre-allocated 0 pages
 vgaarb: loaded
 SCSI subsystem initialized
 usbcore: registered new interface driver usbfs
@@ -95,8 +96,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.656:1): initialized
-HugeTLB registered 2 MB page size, pre-allocated 0 pages
+audit: type=2000 audit(0.676:1): initialized
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
 Key type id_resolver registered
@@ -142,7 +142,7 @@
 EXT4-fs (vda2): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 254:2.
 devtmpfs: mounted
-Freeing unused kernel memory: 676K (ffffffc0008cf000 - ffffffc000978000)
+Freeing unused kernel memory: 680K (ffffffc0008d0000 - ffffffc00097a000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 bash-4.3# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-12-18 00:24:16.353848344 -0800
+++ /tmp/foob	2015-12-18 00:24:16.357848340 -0800
@@ -38,9 +38,9 @@
 	 #################################################################
 	 #################################################################
 	 ################################################################
-	 3.1 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 6653024 (658460 hex)
+Bytes transferred = 6656384 (659180 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -51,7 +51,7 @@
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x658460 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x659180 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2015-12-18 00:24:16.361848336 -0800
+++ /tmp/foob	2015-12-18 00:24:16.365848333 -0800
@@ -37,9 +37,9 @@
 	 #################################################################
 	 #################################################################
 	 ################################
-	 3.1 MiB/s
+	 2.9 MiB/s
 done
-Bytes transferred = 5235024 (4fe150 hex)
+Bytes transferred = 5236472 (4fe6f8 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4fe150 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4fe6f8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.4.0-rc5-next-20151217-62145-g040e3da (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Dec 16 23:18:07 PST 2015
+Linux version 4.4.0-rc5-next-20151218-62459-gf7ac28a (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Dec 18 00:18:15 PST 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -71,7 +71,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0a26f00, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0a26f40, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -92,9 +92,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0976330   (9657 kB)
+      .text : 0xc0008000 - 0xc0976374   (9657 kB)
       .init : 0xc0977000 - 0xc09cc000   ( 340 kB)
-      .data : 0xc09cc000 - 0xc0a28a20   ( 371 kB)
+      .data : 0xc09cc000 - 0xc0a28a60   ( 371 kB)
        .bss : 0xc0a2b000 - 0xc0a86d90   ( 368 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -291,45 +291,45 @@
 WARNING: CPU: 0 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-rc5-next-20151217-62145-g040e3da #1
+CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-rc5-next-20151218-62459-gf7ac28a #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0017248>] (unwind_backtrace) from [<c0012870>] (show_stack+0x10/0x14)
-[<c0012870>] (show_stack) from [<c0239504>] (dump_stack+0x84/0xd0)
-[<c0239504>] (dump_stack) from [<c0026b60>] (warn_slowpath_common+0x74/0xb0)
+[<c0012870>] (show_stack) from [<c0239b84>] (dump_stack+0x84/0xd0)
+[<c0239b84>] (dump_stack) from [<c0026b60>] (warn_slowpath_common+0x74/0xb0)
 [<c0026b60>] (warn_slowpath_common) from [<c0026c30>] (warn_slowpath_fmt+0x30/0x40)
 [<c0026c30>] (warn_slowpath_fmt) from [<c00398dc>] (check_flush_dependency+0xc8/0x12c)
 [<c00398dc>] (check_flush_dependency) from [<c00399f8>] (flush_work+0x54/0x11c)
-[<c00399f8>] (flush_work) from [<c00c013c>] (lru_add_drain_all+0x124/0x180)
-[<c00c013c>] (lru_add_drain_all) from [<c00f0c58>] (migrate_prep+0x8/0x10)
-[<c00f0c58>] (migrate_prep) from [<c00bacb4>] (alloc_contig_range+0xc4/0x2f8)
-[<c00bacb4>] (alloc_contig_range) from [<c00f27c8>] (cma_alloc+0xd8/0x1a0)
-[<c00f27c8>] (cma_alloc) from [<c001c250>] (__alloc_from_contiguous+0x38/0xdc)
+[<c00399f8>] (flush_work) from [<c00c0764>] (lru_add_drain_all+0x124/0x180)
+[<c00c0764>] (lru_add_drain_all) from [<c00f12b8>] (migrate_prep+0x8/0x10)
+[<c00f12b8>] (migrate_prep) from [<c00bb2dc>] (alloc_contig_range+0xc4/0x2f8)
+[<c00bb2dc>] (alloc_contig_range) from [<c00f2e28>] (cma_alloc+0xd8/0x1a0)
+[<c00f2e28>] (cma_alloc) from [<c001c250>] (__alloc_from_contiguous+0x38/0xdc)
 [<c001c250>] (__alloc_from_contiguous) from [<c001cf18>] (__dma_alloc+0x228/0x2ac)
 [<c001cf18>] (__dma_alloc) from [<c001d020>] (arm_dma_alloc+0x48/0x54)
-[<c001d020>] (arm_dma_alloc) from [<c00ed064>] (dma_pool_alloc+0x28c/0x304)
-[<c00ed064>] (dma_pool_alloc) from [<c03f92e4>] (ehci_qh_alloc+0x3c/0xb4)
-[<c03f92e4>] (ehci_qh_alloc) from [<c03fd4d0>] (ehci_setup+0x154/0x478)
-[<c03fd4d0>] (ehci_setup) from [<c0401218>] (tegra_ehci_reset+0xc/0x50)
-[<c0401218>] (tegra_ehci_reset) from [<c03e4cd0>] (usb_add_hcd+0x2fc/0x804)
-[<c03e4cd0>] (usb_add_hcd) from [<c0400b90>] (tegra_ehci_probe+0x3b8/0x504)
-[<c0400b90>] (tegra_ehci_probe) from [<c0333708>] (platform_drv_probe+0x50/0xac)
-[<c0333708>] (platform_drv_probe) from [<c0332128>] (driver_probe_device+0x1f4/0x2b0)
-[<c0332128>] (driver_probe_device) from [<c03306dc>] (bus_for_each_drv+0x44/0x8c)
-[<c03306dc>] (bus_for_each_drv) from [<c0331e84>] (__device_attach+0xa0/0x104)
-[<c0331e84>] (__device_attach) from [<c0331558>] (bus_probe_device+0x84/0x8c)
-[<c0331558>] (bus_probe_device) from [<c033198c>] (deferred_probe_work_func+0x60/0x8c)
-[<c033198c>] (deferred_probe_work_func) from [<c003a804>] (process_one_work+0x120/0x328)
+[<c001d020>] (arm_dma_alloc) from [<c00ed6c4>] (dma_pool_alloc+0x28c/0x304)
+[<c00ed6c4>] (dma_pool_alloc) from [<c03f9978>] (ehci_qh_alloc+0x3c/0xb4)
+[<c03f9978>] (ehci_qh_alloc) from [<c03fdb64>] (ehci_setup+0x154/0x478)
+[<c03fdb64>] (ehci_setup) from [<c04018ac>] (tegra_ehci_reset+0xc/0x50)
+[<c04018ac>] (tegra_ehci_reset) from [<c03e5364>] (usb_add_hcd+0x2fc/0x804)
+[<c03e5364>] (usb_add_hcd) from [<c0401224>] (tegra_ehci_probe+0x3b8/0x504)
+[<c0401224>] (tegra_ehci_probe) from [<c0333d88>] (platform_drv_probe+0x50/0xac)
+[<c0333d88>] (platform_drv_probe) from [<c03327a8>] (driver_probe_device+0x1f4/0x2b0)
+[<c03327a8>] (driver_probe_device) from [<c0330d5c>] (bus_for_each_drv+0x44/0x8c)
+[<c0330d5c>] (bus_for_each_drv) from [<c0332504>] (__device_attach+0xa0/0x104)
+[<c0332504>] (__device_attach) from [<c0331bd8>] (bus_probe_device+0x84/0x8c)
+[<c0331bd8>] (bus_probe_device) from [<c033200c>] (deferred_probe_work_func+0x60/0x8c)
+[<c033200c>] (deferred_probe_work_func) from [<c003a804>] (process_one_work+0x120/0x328)
 [<c003a804>] (process_one_work) from [<c003aa70>] (worker_thread+0x2c/0x4b4)
 [<c003aa70>] (worker_thread) from [<c003fbac>] (kthread+0xd8/0xf4)
 [<c003fbac>] (kthread) from [<c000f938>] (ret_from_fork+0x14/0x3c)
----[ end trace f2777f27716c36bc ]---
+---[ end trace 20a950742dfa31be ]---
 tegra-ehci 7d008000.usb: irq 339, io mem 0x7d008000
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-06-28 16:18:07 UTC (962209087)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-06-29 17:18:21 UTC (962299101)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-12-18 00:24:16.369848329 -0800
+++ /tmp/foob	2015-12-18 00:24:16.373848325 -0800
@@ -40,20 +40,20 @@
 	 #################################################################
 	 #################################################################
 	 ################################################################
-	 3.3 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 6653024 (658460 hex)
+Bytes transferred = 6656384 (659180 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 2.8 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x658460 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x659180 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012a89
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2015-12-18 00:24:16.381848318 -0800
+++ /tmp/foob	2015-12-18 00:24:16.381848318 -0800
@@ -29,7 +29,6 @@
 Tegra124 (Jetson TK1) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash'
 Tegra124 (Jetson TK1) # setenv ipaddr 192.168.100.102; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 zImage.tegra_defconfig
-Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'zImage.tegra_defconfig'.
@@ -42,18 +41,18 @@
 	 ################################
 	 3.2 MiB/s
 done
-Bytes transferred = 5235024 (4fe150 hex)
+Bytes transferred = 5236472 (4fe6f8 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 2.7 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x4fe150 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x4fe6f8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012a89
@@ -64,7 +63,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.4.0-rc5-next-20151217-62145-g040e3da (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Dec 16 23:18:07 PST 2015
+Linux version 4.4.0-rc5-next-20151218-62459-gf7ac28a (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Dec 18 00:18:15 PST 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -74,7 +73,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0a26f00, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0a26f40, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -95,9 +94,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0976330   (9657 kB)
+      .text : 0xc0008000 - 0xc0976374   (9657 kB)
       .init : 0xc0977000 - 0xc09cc000   ( 340 kB)
-      .data : 0xc09cc000 - 0xc0a28a20   ( 371 kB)
+      .data : 0xc09cc000 - 0xc0a28a60   ( 371 kB)
        .bss : 0xc0a2b000 - 0xc0a86d90   ( 368 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -329,40 +328,40 @@
 WARNING: CPU: 0 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-rc5-next-20151217-62145-g040e3da #1
+CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-rc5-next-20151218-62459-gf7ac28a #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0017248>] (unwind_backtrace) from [<c0012870>] (show_stack+0x10/0x14)
-[<c0012870>] (show_stack) from [<c0239504>] (dump_stack+0x84/0xd0)
-[<c0239504>] (dump_stack) from [<c0026b60>] (warn_slowpath_common+0x74/0xb0)
+[<c0012870>] (show_stack) from [<c0239b84>] (dump_stack+0x84/0xd0)
+[<c0239b84>] (dump_stack) from [<c0026b60>] (warn_slowpath_common+0x74/0xb0)
 [<c0026b60>] (warn_slowpath_common) from [<c0026c30>] (warn_slowpath_fmt+0x30/0x40)
 [<c0026c30>] (warn_slowpath_fmt) from [<c00398dc>] (check_flush_dependency+0xc8/0x12c)
 [<c00398dc>] (check_flush_dependency) from [<c00399f8>] (flush_work+0x54/0x11c)
-[<c00399f8>] (flush_work) from [<c00c013c>] (lru_add_drain_all+0x124/0x180)
-[<c00c013c>] (lru_add_drain_all) from [<c00f0c58>] (migrate_prep+0x8/0x10)
-[<c00f0c58>] (migrate_prep) from [<c00bacb4>] (alloc_contig_range+0xc4/0x2f8)
-[<c00bacb4>] (alloc_contig_range) from [<c00f27c8>] (cma_alloc+0xd8/0x1a0)
-[<c00f27c8>] (cma_alloc) from [<c001c250>] (__alloc_from_contiguous+0x38/0xdc)
+[<c00399f8>] (flush_work) from [<c00c0764>] (lru_add_drain_all+0x124/0x180)
+[<c00c0764>] (lru_add_drain_all) from [<c00f12b8>] (migrate_prep+0x8/0x10)
+[<c00f12b8>] (migrate_prep) from [<c00bb2dc>] (alloc_contig_range+0xc4/0x2f8)
+[<c00bb2dc>] (alloc_contig_range) from [<c00f2e28>] (cma_alloc+0xd8/0x1a0)
+[<c00f2e28>] (cma_alloc) from [<c001c250>] (__alloc_from_contiguous+0x38/0xdc)
 [<c001c250>] (__alloc_from_contiguous) from [<c001cf18>] (__dma_alloc+0x228/0x2ac)
 [<c001cf18>] (__dma_alloc) from [<c001d020>] (arm_dma_alloc+0x48/0x54)
-[<c001d020>] (arm_dma_alloc) from [<c034231c>] (dmam_alloc_coherent+0x8c/0xf0)
-[<c034231c>] (dmam_alloc_coherent) from [<c0388148>] (ahci_port_start+0xd0/0x1e0)
-[<c0388148>] (ahci_port_start) from [<c036fe20>] (ata_host_start+0xf8/0x1d4)
-[<c036fe20>] (ata_host_start) from [<c0374db4>] (ata_host_activate+0x20/0x134)
-[<c0374db4>] (ata_host_activate) from [<c03875f0>] (ahci_host_activate+0x3c/0x11c)
-[<c03875f0>] (ahci_host_activate) from [<c0389998>] (ahci_platform_init_host+0x1cc/0x3c8)
-[<c0389998>] (ahci_platform_init_host) from [<c0389664>] (tegra_ahci_probe+0x404/0x4c4)
-[<c0389664>] (tegra_ahci_probe) from [<c0333708>] (platform_drv_probe+0x50/0xac)
-[<c0333708>] (platform_drv_probe) from [<c0332128>] (driver_probe_device+0x1f4/0x2b0)
-[<c0332128>] (driver_probe_device) from [<c03306dc>] (bus_for_each_drv+0x44/0x8c)
-[<c03306dc>] (bus_for_each_drv) from [<c0331e84>] (__device_attach+0xa0/0x104)
-[<c0331e84>] (__device_attach) from [<c0331558>] (bus_probe_device+0x84/0x8c)
-[<c0331558>] (bus_probe_device) from [<c033198c>] (deferred_probe_work_func+0x60/0x8c)
-[<c033198c>] (deferred_probe_work_func) from [<c003a804>] (process_one_work+0x120/0x328)
+[<c001d020>] (arm_dma_alloc) from [<c034299c>] (dmam_alloc_coherent+0x8c/0xf0)
+[<c034299c>] (dmam_alloc_coherent) from [<c03887dc>] (ahci_port_start+0xd0/0x1e0)
+[<c03887dc>] (ahci_port_start) from [<c03704b4>] (ata_host_start+0xf8/0x1d4)
+[<c03704b4>] (ata_host_start) from [<c0375448>] (ata_host_activate+0x20/0x134)
+[<c0375448>] (ata_host_activate) from [<c0387c84>] (ahci_host_activate+0x3c/0x11c)
+[<c0387c84>] (ahci_host_activate) from [<c038a02c>] (ahci_platform_init_host+0x1cc/0x3c8)
+[<c038a02c>] (ahci_platform_init_host) from [<c0389cf8>] (tegra_ahci_probe+0x404/0x4c4)
+[<c0389cf8>] (tegra_ahci_probe) from [<c0333d88>] (platform_drv_probe+0x50/0xac)
+[<c0333d88>] (platform_drv_probe) from [<c03327a8>] (driver_probe_device+0x1f4/0x2b0)
+[<c03327a8>] (driver_probe_device) from [<c0330d5c>] (bus_for_each_drv+0x44/0x8c)
+[<c0330d5c>] (bus_for_each_drv) from [<c0332504>] (__device_attach+0xa0/0x104)
+[<c0332504>] (__device_attach) from [<c0331bd8>] (bus_probe_device+0x84/0x8c)
+[<c0331bd8>] (bus_probe_device) from [<c033200c>] (deferred_probe_work_func+0x60/0x8c)
+[<c033200c>] (deferred_probe_work_func) from [<c003a804>] (process_one_work+0x120/0x328)
 [<c003a804>] (process_one_work) from [<c003aa70>] (worker_thread+0x2c/0x4b4)
 [<c003aa70>] (worker_thread) from [<c003fbac>] (kthread+0xd8/0xf4)
 [<c003fbac>] (kthread) from [<c000f938>] (ret_from_fork+0x14/0x3c)
----[ end trace c1348b3a5e11625b ]---
+---[ end trace bee99f4588bfdf87 ]---
 scsi host0: tegra-ahci
 scsi host1: tegra-ahci
 ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 341
@@ -384,7 +383,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:01:19 UTC (946684879)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:19 UTC (946684819)
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-12-18 00:24:16.385848314 -0800
+++ /tmp/foob	2015-12-18 00:24:16.389848311 -0800
@@ -48,18 +48,18 @@
 	 ################################################################
 	 4.1 MiB/s
 done
-Bytes transferred = 6653024 (658460 hex)
+Bytes transferred = 6656384 (659180 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 4 MiB/s
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x658460 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x659180 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2015-12-18 00:24:16.393848307 -0800
+++ /tmp/foob	2015-12-18 00:24:16.397848303 -0800
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -45,20 +45,20 @@
 	 #################################################################
 	 #################################################################
 	 ################################
-	 3.5 MiB/s
+	 3.6 MiB/s
 done
-Bytes transferred = 5235024 (4fe150 hex)
+Bytes transferred = 5236472 (4fe6f8 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 2.9 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x4fe150 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x4fe6f8 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -69,7 +69,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.4.0-rc5-next-20151217-62145-g040e3da (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Dec 16 23:18:07 PST 2015
+Linux version 4.4.0-rc5-next-20151218-62459-gf7ac28a (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Dec 18 00:18:15 PST 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -78,7 +78,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0a26f00, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0a26f40, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -99,9 +99,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0976330   (9657 kB)
+      .text : 0xc0008000 - 0xc0976374   (9657 kB)
       .init : 0xc0977000 - 0xc09cc000   ( 340 kB)
-      .data : 0xc09cc000 - 0xc0a28a20   ( 371 kB)
+      .data : 0xc09cc000 - 0xc0a28a60   ( 371 kB)
        .bss : 0xc0a2b000 - 0xc0a86d90   ( 368 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -302,17 +302,15 @@
 tlv320aic23-codec 2-001a: ASoC: no dapm match for MICIN --> NULL --> Mic Input
 tlv320aic23-codec 2-001a: ASoC: Failed to add route MICIN -> NULL -> Mic Input
 tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 NET: Registered protocol family 10
 mmc0: host does not support reading read-only switch, assuming write-enable
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
-mip6: Mobile IPv6
 mmc0: new high speed SDHC card at address b368
-mmc1: new high speed SD card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
+mip6: Mobile IPv6
 sit: IPv6 over IPv4 tunneling driver
+mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
- mmcblk0: p1
- mmcblk1: p1 p2
 NET: Registered protocol family 17
 NET: Registered protocol family 15
 can: controller area network core (rev 20120528 abi 9)
@@ -331,11 +329,50 @@
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
+------------[ cut here ]------------
+WARNING: CPU: 1 PID: 77 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
+workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
+Modules linked in:
+CPU: 1 PID: 77 Comm: kworker/u4:2 Not tainted 4.4.0-rc5-next-20151218-62459-gf7ac28a #1
+Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
+Workqueue: deferwq deferred_probe_work_func
+[<c0017248>] (unwind_backtrace) from [<c0012870>] (show_stack+0x10/0x14)
+[<c0012870>] (show_stack) from [<c0239b84>] (dump_stack+0x84/0xd0)
+[<c0239b84>] (dump_stack) from [<c0026b60>] (warn_slowpath_common+0x74/0xb0)
+[<c0026b60>] (warn_slowpath_common) from [<c0026c30>] (warn_slowpath_fmt+0x30/0x40)
+[<c0026c30>] (warn_slowpath_fmt) from [<c00398dc>] (check_flush_dependency+0xc8/0x12c)
+[<c00398dc>] (check_flush_dependency) from [<c00399f8>] (flush_work+0x54/0x11c)
+[<c00399f8>] (flush_work) from [<c00c0764>] (lru_add_drain_all+0x124/0x180)
+[<c00c0764>] (lru_add_drain_all) from [<c00f12b8>] (migrate_prep+0x8/0x10)
+[<c00f12b8>] (migrate_prep) from [<c00bb2dc>] (alloc_contig_range+0xc4/0x2f8)
+[<c00bb2dc>] (alloc_contig_range) from [<c00f2e28>] (cma_alloc+0xd8/0x1a0)
+[<c00f2e28>] (cma_alloc) from [<c001c250>] (__alloc_from_contiguous+0x38/0xdc)
+[<c001c250>] (__alloc_from_contiguous) from [<c001cf18>] (__dma_alloc+0x228/0x2ac)
+[<c001cf18>] (__dma_alloc) from [<c001d020>] (arm_dma_alloc+0x48/0x54)
+[<c001d020>] (arm_dma_alloc) from [<c00ed6c4>] (dma_pool_alloc+0x28c/0x304)
+[<c00ed6c4>] (dma_pool_alloc) from [<c03f9978>] (ehci_qh_alloc+0x3c/0xb4)
+[<c03f9978>] (ehci_qh_alloc) from [<c03fdb64>] (ehci_setup+0x154/0x478)
+[<c03fdb64>] (ehci_setup) from [<c04018ac>] (tegra_ehci_reset+0xc/0x50)
+[<c04018ac>] (tegra_ehci_reset) from [<c03e5364>] (usb_add_hcd+0x2fc/0x804)
+[<c03e5364>] (usb_add_hcd) from [<c0401224>] (tegra_ehci_probe+0x3b8/0x504)
+[<c0401224>] (tegra_ehci_probe) from [<c0333d88>] (platform_drv_probe+0x50/0xac)
+[<c0333d88>] (platform_drv_probe) from [<c03327a8>] (driver_probe_device+0x1f4/0x2b0)
+[<c03327a8>] (driver_probe_device) from [<c0330d5c>] (bus_for_each_drv+0x44/0x8c)
+[<c0330d5c>] (bus_for_each_drv) from [<c0332504>] (__device_attach+0xa0/0x104)
+[<c0332504>] (__device_attach) from [<c0331bd8>] (bus_probe_device+0x84/0x8c)
+[<c0331bd8>] (bus_probe_device) from [<c033200c>] (deferred_probe_work_func+0x60/0x8c)
+[<c033200c>] (deferred_probe_work_func) from [<c003a804>] (process_one_work+0x120/0x328)
+[<c003a804>] (process_one_work) from [<c003aa70>] (worker_thread+0x2c/0x4b4)
+[<c003aa70>] (worker_thread) from [<c003fbac>] (kthread+0xd8/0xf4)
+[<c003fbac>] (kthread) from [<c000f938>] (ret_from_fork+0x14/0x3c)
+---[ end trace 470a720e51ba6e65 ]---
+ mmcblk0: p1
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
-hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
+ mmcblk1: p1 p2
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
+hub 1-1:1.0: USB hub found
+hub 1-1:1.0: 4 ports detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-27 03:07:46 UTC (1419649666)
+rtc-em3027 2-0056: setting system clock to 2014-12-27 03:12:51 UTC (1419649971)
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-12-18 00:24:16.401848299 -0800
+++ /tmp/foob	2015-12-18 00:24:16.401848299 -0800
@@ -38,9 +38,9 @@
 	 #################################################################
 	 #################################################################
 	 ################################################################
-	 3.3 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 6653024 (658460 hex)
+Bytes transferred = 6656384 (659180 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -51,7 +51,7 @@
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x658460 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x659180 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2015-12-18 00:24:16.409848293 -0800
+++ /tmp/foob	2015-12-18 00:24:16.409848293 -0800
@@ -37,9 +37,9 @@
 	 #################################################################
 	 #################################################################
 	 ################################
-	 3.3 MiB/s
+	 3 MiB/s
 done
-Bytes transferred = 5235024 (4fe150 hex)
+Bytes transferred = 5236472 (4fe6f8 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4fe150 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4fe6f8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.4.0-rc5-next-20151217-62145-g040e3da (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Dec 16 23:18:07 PST 2015
+Linux version 4.4.0-rc5-next-20151218-62459-gf7ac28a (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Dec 18 00:18:15 PST 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -70,7 +70,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0a26f00, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0a26f40, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -91,9 +91,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0976330   (9657 kB)
+      .text : 0xc0008000 - 0xc0976374   (9657 kB)
       .init : 0xc0977000 - 0xc09cc000   ( 340 kB)
-      .data : 0xc09cc000 - 0xc0a28a20   ( 371 kB)
+      .data : 0xc09cc000 - 0xc0a28a60   ( 371 kB)
        .bss : 0xc0a2b000 - 0xc0a86d90   ( 368 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -340,39 +340,39 @@
 WARNING: CPU: 0 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-rc5-next-20151217-62145-g040e3da #1
+CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-rc5-next-20151218-62459-gf7ac28a #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0017248>] (unwind_backtrace) from [<c0012870>] (show_stack+0x10/0x14)
-[<c0012870>] (show_stack) from [<c0239504>] (dump_stack+0x84/0xd0)
-[<c0239504>] (dump_stack) from [<c0026b60>] (warn_slowpath_common+0x74/0xb0)
+[<c0012870>] (show_stack) from [<c0239b84>] (dump_stack+0x84/0xd0)
+[<c0239b84>] (dump_stack) from [<c0026b60>] (warn_slowpath_common+0x74/0xb0)
 [<c0026b60>] (warn_slowpath_common) from [<c0026c30>] (warn_slowpath_fmt+0x30/0x40)
 [<c0026c30>] (warn_slowpath_fmt) from [<c00398dc>] (check_flush_dependency+0xc8/0x12c)
 [<c00398dc>] (check_flush_dependency) from [<c00399f8>] (flush_work+0x54/0x11c)
-[<c00399f8>] (flush_work) from [<c00c013c>] (lru_add_drain_all+0x124/0x180)
-[<c00c013c>] (lru_add_drain_all) from [<c00f0c58>] (migrate_prep+0x8/0x10)
-[<c00f0c58>] (migrate_prep) from [<c00bacb4>] (alloc_contig_range+0xc4/0x2f8)
-[<c00bacb4>] (alloc_contig_range) from [<c00f27c8>] (cma_alloc+0xd8/0x1a0)
-[<c00f27c8>] (cma_alloc) from [<c001c250>] (__alloc_from_contiguous+0x38/0xdc)
+[<c00399f8>] (flush_work) from [<c00c0764>] (lru_add_drain_all+0x124/0x180)
+[<c00c0764>] (lru_add_drain_all) from [<c00f12b8>] (migrate_prep+0x8/0x10)
+[<c00f12b8>] (migrate_prep) from [<c00bb2dc>] (alloc_contig_range+0xc4/0x2f8)
+[<c00bb2dc>] (alloc_contig_range) from [<c00f2e28>] (cma_alloc+0xd8/0x1a0)
+[<c00f2e28>] (cma_alloc) from [<c001c250>] (__alloc_from_contiguous+0x38/0xdc)
 [<c001c250>] (__alloc_from_contiguous) from [<c001cf18>] (__dma_alloc+0x228/0x2ac)
 [<c001cf18>] (__dma_alloc) from [<c001d020>] (arm_dma_alloc+0x48/0x54)
-[<c001d020>] (arm_dma_alloc) from [<c00ed064>] (dma_pool_alloc+0x28c/0x304)
-[<c00ed064>] (dma_pool_alloc) from [<c03f92e4>] (ehci_qh_alloc+0x3c/0xb4)
-[<c03f92e4>] (ehci_qh_alloc) from [<c03fd4d0>] (ehci_setup+0x154/0x478)
-[<c03fd4d0>] (ehci_setup) from [<c0401218>] (tegra_ehci_reset+0xc/0x50)
-[<c0401218>] (tegra_ehci_reset) from [<c03e4cd0>] (usb_add_hcd+0x2fc/0x804)
-[<c03e4cd0>] (usb_add_hcd) from [<c0400b90>] (tegra_ehci_probe+0x3b8/0x504)
-[<c0400b90>] (tegra_ehci_probe) from [<c0333708>] (platform_drv_probe+0x50/0xac)
-[<c0333708>] (platform_drv_probe) from [<c0332128>] (driver_probe_device+0x1f4/0x2b0)
-[<c0332128>] (driver_probe_device) from [<c03306dc>] (bus_for_each_drv+0x44/0x8c)
-[<c03306dc>] (bus_for_each_drv) from [<c0331e84>] (__device_attach+0xa0/0x104)
-[<c0331e84>] (__device_attach) from [<c0331558>] (bus_probe_device+0x84/0x8c)
-[<c0331558>] (bus_probe_device) from [<c033198c>] (deferred_probe_work_func+0x60/0x8c)
-[<c033198c>] (deferred_probe_work_func) from [<c003a804>] (process_one_work+0x120/0x328)
+[<c001d020>] (arm_dma_alloc) from [<c00ed6c4>] (dma_pool_alloc+0x28c/0x304)
+[<c00ed6c4>] (dma_pool_alloc) from [<c03f9978>] (ehci_qh_alloc+0x3c/0xb4)
+[<c03f9978>] (ehci_qh_alloc) from [<c03fdb64>] (ehci_setup+0x154/0x478)
+[<c03fdb64>] (ehci_setup) from [<c04018ac>] (tegra_ehci_reset+0xc/0x50)
+[<c04018ac>] (tegra_ehci_reset) from [<c03e5364>] (usb_add_hcd+0x2fc/0x804)
+[<c03e5364>] (usb_add_hcd) from [<c0401224>] (tegra_ehci_probe+0x3b8/0x504)
+[<c0401224>] (tegra_ehci_probe) from [<c0333d88>] (platform_drv_probe+0x50/0xac)
+[<c0333d88>] (platform_drv_probe) from [<c03327a8>] (driver_probe_device+0x1f4/0x2b0)
+[<c03327a8>] (driver_probe_device) from [<c0330d5c>] (bus_for_each_drv+0x44/0x8c)
+[<c0330d5c>] (bus_for_each_drv) from [<c0332504>] (__device_attach+0xa0/0x104)
+[<c0332504>] (__device_attach) from [<c0331bd8>] (bus_probe_device+0x84/0x8c)
+[<c0331bd8>] (bus_probe_device) from [<c033200c>] (deferred_probe_work_func+0x60/0x8c)
+[<c033200c>] (deferred_probe_work_func) from [<c003a804>] (process_one_work+0x120/0x328)
 [<c003a804>] (process_one_work) from [<c003aa70>] (worker_thread+0x2c/0x4b4)
 [<c003aa70>] (worker_thread) from [<c003fbac>] (kthread+0xd8/0xf4)
 [<c003fbac>] (kthread) from [<c000f938>] (ret_from_fork+0x14/0x3c)
----[ end trace c1e380384c37b6c0 ]---
+---[ end trace 45532478ef8a4093 ]---
 tegra-ehci 7d004000.usb: irq 343, io mem 0x7d004000
 tegra-ehci 7d004000.usb: USB 2.0 started, EHCI 1.00
 hub 1-0:1.0: USB hub found
