Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2017-03-07 18:59:53.741075236 -0800
+++ /tmp/foob	2017-03-07 18:59:53.741075236 -0800
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc1-next-20170307-180993-g3c90632 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Mar 7 13:24:55 PST 2017
+Linux version 4.11.0-rc1-next-20170308-181595-g56b8bad (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Mar 7 18:52:09 PST 2017
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -21,7 +21,7 @@
 psci: PSCIv0.2 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: Trusted OS migration not required
-percpu: Embedded 23 pages/cpu @ffff80003efd0000 s56960 r8192 d29056 u94208
+percpu: Embedded 23 pages/cpu @ffff80003efd0000 s56832 r8192 d29184 u94208
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
 CPU features: enabling workaround for ARM erratum 834220
@@ -29,15 +29,15 @@
 Policy zone: DMA
 Kernel command line: root=/dev/vda2 rw rootwait mem=1024M console=ttyAMA0,38400n8 init=/bin/bash
 PID hash table entries: 4096 (order: 3, 32768 bytes)
-Memory: 1000044K/1048576K available (8700K kernel code, 982K rwdata, 4216K rodata, 1088K init, 395K bss, 32148K reserved, 16384K cma-reserved)
+Memory: 999972K/1048576K available (8764K kernel code, 990K rwdata, 4228K rodata, 1088K init, 395K bss, 32220K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff000008900000   (  8704 KB)
-    .rodata : 0xffff000008900000 - 0xffff000008d30000   (  4288 KB)
-      .init : 0xffff000008d30000 - 0xffff000008e40000   (  1088 KB)
-      .data : 0xffff000008e40000 - 0xffff000008f35a00   (   983 KB)
-       .bss : 0xffff000008f35a00 - 0xffff000008f98638   (   396 KB)
+      .text : 0xffff000008080000 - 0xffff000008910000   (  8768 KB)
+    .rodata : 0xffff000008910000 - 0xffff000008d40000   (  4288 KB)
+      .init : 0xffff000008d40000 - 0xffff000008e50000   (  1088 KB)
+      .data : 0xffff000008e50000 - 0xffff000008f47a00   (   991 KB)
+       .bss : 0xffff000008f47a00 - 0xffff000008faa638   (   396 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -45,9 +45,8 @@
     memory  : 0xffff800000000000 - 0xffff800040000000   (  1024 MB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
 Preemptible hierarchical RCU implementation.
-	Build-time adjustment of leaf fanout to 64.
 	RCU restricting CPUs from NR_CPUS=64 to nr_cpu_ids=1.
-RCU: Adjusting geometry for rcu_fanout_leaf=64, nr_cpu_ids=1
+RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=1
 NR_IRQS:64 nr_irqs:64 0
 GICv2m: range[mem 0x08020000-0x08020fff], SPI[80:143]
 arm_arch_timer: WARNING: Invalid trigger for IRQ3, assuming level low
@@ -78,7 +77,7 @@
 pinctrl core: initialized pinctrl subsystem
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffff000008907000, 1 data @ ffff000008e45000)
+vdso: 2 pages (1 code @ ffff000008917000, 1 data @ ffff000008e55000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -114,7 +113,7 @@
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 kvm [1]: HYP mode not available
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.956:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.744:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -180,7 +179,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2017-03-07 21:25:35 UTC (1488921935)
+rtc-pl031 9010000.pl031: setting system clock to 2017-03-08 02:52:49 UTC (1488941569)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-03-07 18:59:53.749075268 -0800
+++ /tmp/foob	2017-03-07 18:59:53.749075268 -0800
@@ -38,21 +38,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ############################################################
-	 2.8 MiB/s
+	 #########################################################
+	 3.2 MiB/s
 done
-Bytes transferred = 7547624 (732ae8 hex)
+Bytes transferred = 7509352 (729568 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.6 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x732ae8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x729568 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc1-next-20170307-156560-g3c90632 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Mar 7 13:21:37 PST 2017
+Linux version 4.11.0-rc1-next-20170308-157162-g56b8bad (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Mar 7 18:49:20 PST 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -71,20 +71,20 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c1539880, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1539800, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 16 pages/cpu @eef9c000 s34432 r8192 d22912 u65536
-pcpu-alloc: s34432 r8192 d22912 u65536 alloc=16*4096
+percpu: Embedded 16 pages/cpu @eef9c000 s34368 r8192 d22976 u65536
+pcpu-alloc: s34368 r8192 d22976 u65536 alloc=16*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=7369c667-ff51-ec4a-29cd-baabf2fbe346 rootwait gpt debug earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993764K/2096128K available (10240K kernel code, 1260K rwdata, 4240K rodata, 2048K init, 536K bss, 36828K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993896K/2096128K available (10240K kernel code, 1260K rwdata, 4224K rodata, 2048K init, 407K bss, 36696K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -94,14 +94,13 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc153b1e0   (1261 kB)
-       .bss : 0xc153d000 - 0xc15c32f8   ( 537 kB)
+      .data : 0xc1400000 - 0xc153b160   (1261 kB)
+       .bss : 0xc153d000 - 0xc15a2cb8   ( 408 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU debugfs-based tracing is enabled.
-	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=16 to nr_cpu_ids=4.
-RCU: Adjusting geometry for rcu_fanout_leaf=32, nr_cpu_ids=4
+RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=4
 NR_IRQS:16 nr_irqs:16 16
 /interrupt-controller@60004000: 160 interrupts forwarded to /interrupt-controller@50041000
 Tegra clk 127: register failed with -17
@@ -332,17 +331,17 @@
 mmc0: Failed to register LED device: -517
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
 sdhci-tegra 78000400.sdhci: Got CD GPIO
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-12-30 08:58:17 UTC (978166697)
-dvdd_ts: disabling
-usb1_vbus: disabling
-vdd_cam_1v8_reg: disabling
-[    2sdhci: Got WP GPIO
+sdhci-tegra 78000400.sdhci: Got WP GPIO
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-12-30 14:26:10 UTC (978186370)
 alloc_contig_range: [fb848, fb849) PFNs busy
 mmc0: Invalid maximum block size, assuming 512 bytes
 mmc0: Failed to register LED device: -517
 alloc_contig_range: [fb848, fb849) PFNs busy
 mmc0: Invalid maximum block size, assuming 512 bytes
 mmc0: Failed to register LED device: -517
+dvdd_ts: disabling
+usb1_vbus: disabling
+vdd_cam_1v8_reg: disabling
 Waiting for root device PARTUUID=7369c667-ff51-ec4a-29cd-baabf2fbe346...
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
 asix 1-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0d:b5:16
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2017-03-07 18:59:53.753075284 -0800
+++ /tmp/foob	2017-03-07 18:59:53.757075300 -0800
@@ -39,18 +39,18 @@
 	 ######################################################
 	 3.3 MiB/s
 done
-Bytes transferred = 5561776 (54ddb0 hex)
+Bytes transferred = 5563344 (54e3d0 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x54ddb0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x54e3d0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc1-next-20170307-156560-g3c90632 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Mar 7 13:21:03 PST 2017
+Linux version 4.11.0-rc1-next-20170308-157162-g56b8bad (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Mar 7 18:48:53 PST 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,13 +69,13 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0d5de80, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0d5dec0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 16 pages/cpu @eef9b000 s34368 r8192 d22976 u65536
-pcpu-alloc: s34368 r8192 d22976 u65536 alloc=16*4096
+percpu: Embedded 16 pages/cpu @eef9b000 s34304 r8192 d23040 u65536
+pcpu-alloc: s34304 r8192 d23040 u65536 alloc=16*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=7369c667-ff51-ec4a-29cd-baabf2fbe346 rootwait gpt debug earlyprintk init=/bin/bash
@@ -92,10 +92,9 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d5f360   ( 381 kB)
+      .data : 0xc0d00000 - 0xc0d5f3a0   ( 381 kB)
        .bss : 0xc0d61000 - 0xc0dad40c   ( 306 kB)
 Preemptible hierarchical RCU implementation.
-	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
 /interrupt-controller@60004000: 160 interrupts forwarded to /interrupt-controller@50041000
 Tegra clk 127: register failed with -17
@@ -321,7 +320,7 @@
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
 sdhci-tegra 78000400.sdhci: Got CD GPIO
 sdhci-tegra 78000400.sdhci: Got WP GPIO
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-12-30 08:54:02 UTC (978166442)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-12-30 14:21:51 UTC (978186111)
 alloc_contig_range: [fbc5b, fbc5c) PFNs busy
 alloc_contig_range: [fbc5c, fbc5d) PFNs busy
 mmc0: Invalid maximum block size, assuming 512 bytes
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-03-07 18:59:53.761075316 -0800
+++ /tmp/foob	2017-03-07 18:59:53.765075331 -0800
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+MMC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -33,21 +33,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ############################################################
-	 3.5 MiB/s
+	 #########################################################
+	 4 MiB/s
 done
-Bytes transferred = 7547624 (732ae8 hex)
+Bytes transferred = 7509352 (729568 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3 MiB/s
+	 3.8 MiB/s
 done
 Bytes transferred = 67133 (1063d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x732ae8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x729568 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301363c
@@ -55,7 +55,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc1-next-20170307-156560-g3c90632 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Mar 7 13:21:37 PST 2017
+Linux version 4.11.0-rc1-next-20170308-157162-g56b8bad (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Mar 7 18:49:20 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -66,20 +66,20 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c1539880, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1539800, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 16 pages/cpu @eef86000 s34432 r8192 d22912 u65536
-pcpu-alloc: s34432 r8192 d22912 u65536 alloc=16*4096
+percpu: Embedded 16 pages/cpu @eef86000 s34368 r8192 d22976 u65536
+pcpu-alloc: s34368 r8192 d22976 u65536 alloc=16*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=b1791faa-84e8-4e27-83a1-9fb0a77e8803 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993644K/2096128K available (10240K kernel code, 1260K rwdata, 4240K rodata, 2048K init, 536K bss, 36948K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993776K/2096128K available (10240K kernel code, 1260K rwdata, 4224K rodata, 2048K init, 407K bss, 36816K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -89,14 +89,13 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc153b1e0   (1261 kB)
-       .bss : 0xc153d000 - 0xc15c32f8   ( 537 kB)
+      .data : 0xc1400000 - 0xc153b160   (1261 kB)
+       .bss : 0xc153d000 - 0xc15a2cb8   ( 408 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU debugfs-based tracing is enabled.
-	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=16 to nr_cpu_ids=4.
-RCU: Adjusting geometry for rcu_fanout_leaf=32, nr_cpu_ids=4
+RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=4
 NR_IRQS:16 nr_irqs:16 16
 /interrupt-controller@60004000: 160 interrupts forwarded to /interrupt-controller@50041000
 Tegra clk 127: register failed with -17
@@ -460,8 +459,8 @@
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:14 UTC (946684814)
 mmc0: Unknown controller version (3). You may experience problems.
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:04:17 UTC (946685057)
 alloc_contig_range: [fb853, fb854) PFNs busy
 alloc_contig_range: [fb857, fb858) PFNs busy
 alloc_contig_range: [fb858, fb859) PFNs busy
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2017-03-07 18:59:53.769075347 -0800
+++ /tmp/foob	2017-03-07 18:59:53.773075363 -0800
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MMC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -32,20 +32,20 @@
 	 #################################################################
 	 #################################################################
 	 ######################################################
-	 4 MiB/s
+	 3.9 MiB/s
 done
-Bytes transferred = 5561776 (54ddb0 hex)
+Bytes transferred = 5563344 (54e3d0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 4 MiB/s
+	 3.8 MiB/s
 done
 Bytes transferred = 67133 (1063d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x54ddb0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x54e3d0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301363c
@@ -54,7 +54,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc1-next-20170307-156560-g3c90632 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Mar 7 13:21:03 PST 2017
+Linux version 4.11.0-rc1-next-20170308-157162-g56b8bad (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Mar 7 18:48:53 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -64,13 +64,13 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0d5de80, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0d5dec0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 16 pages/cpu @eef85000 s34368 r8192 d22976 u65536
-pcpu-alloc: s34368 r8192 d22976 u65536 alloc=16*4096
+percpu: Embedded 16 pages/cpu @eef85000 s34304 r8192 d23040 u65536
+pcpu-alloc: s34304 r8192 d23040 u65536 alloc=16*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=b1791faa-84e8-4e27-83a1-9fb0a77e8803 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
@@ -87,10 +87,9 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d5f360   ( 381 kB)
+      .data : 0xc0d00000 - 0xc0d5f3a0   ( 381 kB)
        .bss : 0xc0d61000 - 0xc0dad40c   ( 306 kB)
 Preemptible hierarchical RCU implementation.
-	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
 /interrupt-controller@60004000: 160 interrupts forwarded to /interrupt-controller@50041000
 Tegra clk 127: register failed with -17
@@ -465,8 +464,8 @@
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
-mmc0: Unknown controller version (3). You may experience problems.
 as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:13 UTC (946684813)
+mmc0: Unknown controller version (3). You may experience problems.
 alloc_contig_range: [fbc9d, fbc9e) PFNs busy
 alloc_contig_range: [fbc9e, fbc9f) PFNs busy
 mmc0: Invalid maximum block size, assuming 512 bytes
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-03-07 18:59:53.777075379 -0800
+++ /tmp/foob	2017-03-07 18:59:53.781075395 -0800
@@ -41,21 +41,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ############################################################
-	 3.5 MiB/s
+	 #########################################################
+	 2.8 MiB/s
 done
-Bytes transferred = 7547624 (732ae8 hex)
+Bytes transferred = 7509352 (729568 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.4 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 68155 (10a3b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x732ae8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x729568 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a3a
@@ -63,7 +63,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc1-next-20170307-156560-g3c90632 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Mar 7 13:21:37 PST 2017
+Linux version 4.11.0-rc1-next-20170308-157162-g56b8bad (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Mar 7 18:49:20 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -75,20 +75,20 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c1539880, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1539800, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327679 pages, LIFO batch:31
-percpu: Embedded 16 pages/cpu @eef85000 s34432 r8192 d22912 u65536
-pcpu-alloc: s34432 r8192 d22912 u65536 alloc=16*4096
+percpu: Embedded 16 pages/cpu @eef85000 s34368 r8192 d22976 u65536
+pcpu-alloc: s34368 r8192 d22976 u65536 alloc=16*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522751
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=4d36e82a-b3c0-4b4b-bfcd-afe5217ac023 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1994660K/2097148K available (10240K kernel code, 1260K rwdata, 4240K rodata, 2048K init, 536K bss, 36952K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1994792K/2097148K available (10240K kernel code, 1260K rwdata, 4224K rodata, 2048K init, 407K bss, 36820K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -98,14 +98,13 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc153b1e0   (1261 kB)
-       .bss : 0xc153d000 - 0xc15c32f8   ( 537 kB)
+      .data : 0xc1400000 - 0xc153b160   (1261 kB)
+       .bss : 0xc153d000 - 0xc15a2cb8   ( 408 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU debugfs-based tracing is enabled.
-	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=16 to nr_cpu_ids=4.
-RCU: Adjusting geometry for rcu_fanout_leaf=32, nr_cpu_ids=4
+RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=4
 NR_IRQS:16 nr_irqs:16 16
 /interrupt-controller@60004000: 160 interrupts forwarded to /interrupt-controller@50041000
 Tegra clk 127: register failed with -17
@@ -322,11 +321,13 @@
 +5V_USB_SS: supplied by +5V_SYS
 +3.3V_PANEL: supplied by +3.3V_RUN
 +3.3V_LP0: supplied by +3.3V_SYS
+elan_i2c 1-0015: reading cmd (0x0105) fail.
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
-elan_i2c 1-0015: fetching report descriptor failed.: -121
 +5V_HDMI_CON: supplied by +5V_SYS
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
  usb3-1: usb3-1 supply vbus not found, using dummy regulator
+elan_i2c 1-0015: failed to get trace info: -5
+elan_i2c: probe of 1-0015 failed with error -5
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
 sdhci-tegra 700b0000.sdhci: allocated mmc-pwrseq
@@ -349,7 +350,6 @@
 mmc0: Failed to register LED device: -517
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
-elan_i2c 1-0015: device initialize failed: -121
 Console: switching to colour frame buffer device 170x48
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
@@ -378,40 +378,23 @@
 mmc0: Invalid maximum block size, assuming 512 bytes
 mmc0: Failed to register LED device: -517
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:23 UTC (946684823)
 sdhci-tegra 700b0000.sdhci: allocated mmc-pwrseq
 mmc0: Unknown controller version (3). You may experience problems.
-alloc_contig_range: [fb850, fb851) PFNs busy
-alloc_contig_range: [fb851, fb852) PFNs busy
-alloc_contig_range: [fb852, fb853) PFNs busy
-alloc_contig_range: [fb853, fb854) PFNs busy
-mmc0: Invalid maximum block size, assuming 512 bytes
-mmc0: Failed to register LED device: -517
-sdhci-tegra 700b0400.sdhci: Got CD GPIO
-sdhci-tegra 700b0400.sdhci: Got WP GPIO
-mmc0: Unknown controller version (3). You may experience problems.
-alloc_contig_range: [fb850, fb851) PFNs busy
-alloc_contig_range: [fb851, fb852) PFNs busy
-alloc_contig_range: [fb852, fb853) PFNs busy
-alloc_contig_range: [fb853, fb854) PFNs busy
-mmc0: Invalid maximum block size, assuming 512 bytes
-mmc0: Failed to register LED device: -517
-mmc0: Unknown controller version (3). You may experience problems.
-alloc_contig_range: [fb850, fb851) PFNs busy
-alloc_contig_range: [fb851, fb852) PFNs busy
-alloc_contig_range: [fb852, fb853) PFNs busy
-alloc_contig_range: [fb853, fb854) PFNs busy
-mmc0: Invalid maximum block size, assuming 512 bytes
-mmc0: Failed to register LED device: -517
-elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
-input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input1
-sdhci-tegra 700b0000.sdhci: allocated mmc-pwrseq
-mmc0: Unknown controller version (3). You may experience problems.
-alloc_contig_range: [fb850, fb851) PFNs busy
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:23 UTC (946684823)
++1.8V_RUN_CAM: disabling
++3.3V_RUN_CAM: disabling
++1.2V_RUN_CAM_FRONT: disabling
++1.05V_RUN_CAM_REAR: disabling
++2.8V_RUN_TOUCH: disabling
++2.8V_RUN_CAM_AF: disabling
++1.8V_RUN_VPP_FUSE: disabling
++5V_VDD_TS_SW: disabling
++5V_USB_HS: disabling
++5V_USB_SS: disabling
+[  : [fb850, fb851) PFNs busy
 alloc_contig_range: [fb851, fb852) PFNs busy
 alloc_contig_range: [fb852, fb853) PFNs busy
 alloc_contig_range: [fb853, fb854) PFNs busy
-alloc_contig_range: [fb854, fb855) PFNs busy
 mmc0: Invalid maximum block size, assuming 512 bytes
 mmc0: Failed to register LED device: -517
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
@@ -421,7 +404,6 @@
 alloc_contig_range: [fb851, fb852) PFNs busy
 alloc_contig_range: [fb852, fb853) PFNs busy
 alloc_contig_range: [fb853, fb854) PFNs busy
-alloc_contig_range: [fb854, fb855) PFNs busy
 mmc0: Invalid maximum block size, assuming 512 bytes
 mmc0: Failed to register LED device: -517
 mmc0: Unknown controller version (3). You may experience problems.
@@ -429,17 +411,6 @@
 alloc_contig_range: [fb851, fb852) PFNs busy
 alloc_contig_range: [fb852, fb853) PFNs busy
 alloc_contig_range: [fb853, fb854) PFNs busy
-alloc_contig_range: [fb854, fb855) PFNs busy
 mmc0: Invalid maximum block size, assuming 512 bytes
 mmc0: Failed to register LED device: -517
-+1.8V_RUN_CAM: disabling
-+3.3V_RUN_CAM: disabling
-+1.2V_RUN_CAM_FRONT: disabling
-+1.05V_RUN_CAM_REAR: disabling
-+2.8V_RUN_TOUCH: disabling
-+2.8V_RUN_CAM_AF: disabling
-+1.8V_RUN_VPP_FUSE: disabling
-+5V_VDD_TS_SW: disabling
-+5V_USB_HS: disabling
-+5V_USB_SS: disabling
 Waiting for root device PARTUUID=4d36e82a-b3c0-4b4b-bfcd-afe5217ac023...
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2017-03-07 18:59:53.785075411 -0800
+++ /tmp/foob	2017-03-07 18:59:53.789075427 -0800
@@ -40,9 +40,9 @@
 	 #################################################################
 	 #################################################################
 	 ######################################################
-	 3.4 MiB/s
+	 3.5 MiB/s
 done
-Bytes transferred = 5561776 (54ddb0 hex)
+Bytes transferred = 5563344 (54e3d0 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -53,7 +53,7 @@
 done
 Bytes transferred = 68155 (10a3b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x54ddb0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x54e3d0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a3a
@@ -62,7 +62,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc1-next-20170307-156560-g3c90632 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Mar 7 13:21:03 PST 2017
+Linux version 4.11.0-rc1-next-20170308-157162-g56b8bad (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Mar 7 18:48:53 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -73,13 +73,13 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c0d5de80, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0d5dec0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327679 pages, LIFO batch:31
-percpu: Embedded 16 pages/cpu @eef84000 s34368 r8192 d22976 u65536
-pcpu-alloc: s34368 r8192 d22976 u65536 alloc=16*4096
+percpu: Embedded 16 pages/cpu @eef84000 s34304 r8192 d23040 u65536
+pcpu-alloc: s34304 r8192 d23040 u65536 alloc=16*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522751
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=4d36e82a-b3c0-4b4b-bfcd-afe5217ac023 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
@@ -96,10 +96,9 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d5f360   ( 381 kB)
+      .data : 0xc0d00000 - 0xc0d5f3a0   ( 381 kB)
        .bss : 0xc0d61000 - 0xc0dad40c   ( 306 kB)
 Preemptible hierarchical RCU implementation.
-	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
 /interrupt-controller@60004000: 160 interrupts forwarded to /interrupt-controller@50041000
 Tegra clk 127: register failed with -17
@@ -388,9 +387,9 @@
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc0: Unknown controller version (3). You may experience problems.
+input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input3
 alloc_contig_range: [fbc70, fbc71) PFNs busy
 alloc_contig_range: [fbc71, fbc72) PFNs busy
-input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input3
 alloc_contig_range: [fbc72, fbc73) PFNs busy
 mmc0: Invalid maximum block size, assuming 512 bytes
 mmc0: Failed to register LED device: -517
@@ -431,7 +430,7 @@
   #1: GoogleNyanBig
 alloc_contig_range: [fbc71, fbc72) PFNs busy
 alloc_contig_range: [fbc72, fbc73) PFNs busy
-alloc_contig_range: [fbc71, fbc72) PFNs busy
+alloc_contig_range: [fbc70, fbc71) PFNs busy
 alloc_contig_range: [fbc72, fbc73) PFNs busy
 alloc_contig_range: [fbc71, fbc72) PFNs busy
 alloc_contig_range: [fbc72, fbc73) PFNs busy
@@ -448,12 +447,12 @@
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc0: Unknown controller version (3). You may experience problems.
 alloc_contig_range: [fbc70, fbc71) PFNs busy
+uvcvideo: Found UVC 1.00 device HD WebCam (1bcf:2c67)
 alloc_contig_range: [fbc71, fbc72) PFNs busy
 alloc_contig_range: [fbc72, fbc73) PFNs busy
 mmc0: Invalid maximum block size, assuming 512 bytes
-mmc0: Failed to register LED device: -517
-uvcvideo: Found UVC 1.00 device HD WebCam (1bcf:2c67)
 input: HD WebCam as /devices/soc0/70090000.usb/usb1/1-2/1-2:1.0/input/input5
+mmc0: Failed to register LED device: -517
 mmc0: Unknown controller version (3). You may experience problems.
 alloc_contig_range: [fbc70, fbc71) PFNs busy
 alloc_contig_range: [fbc71, fbc72) PFNs busy
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2017-03-07 18:59:53.793075443 -0800
+++ /tmp/foob	2017-03-07 18:59:53.797075459 -0800
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1801353 us
+Bootloader-Cpu Init at (time stamp): 1800579 us
 
 Pinmux changes applied in kernel way
 
@@ -62,20 +62,20 @@
 Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1801327  us
-Bootloader (AosInit) time: 733446 us
-Bootloader Main Init time: 180042 us
-BlockDevMgrInit time: 157538 us
-PartMgrInit time: 8 us
+Miscellaneous avp time: 1800553  us
+Bootloader (AosInit) time: 734312 us
+Bootloader Main Init time: 180399 us
+BlockDevMgrInit time: 157480 us
+PartMgrInit time: 9 us
 StorMgrInit time: 30 us
 FsMgrInit time: 2 us
-Partition Table load: 77053 us 
-Display init time: 574439 us
+Partition Table load: 76824 us 
+Display init time: 574714 us
 Kernel read time: 377 msec
 Load Kernel Size:   16304128 Bytes
 Bl Kernel ReadSpeed: 43247000 Bytes/sec
-Pre-Kernel Jump time: 1740422 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5844561 us
+Pre-Kernel Jump time: 1740003 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5886999 us
 Booting Linux on physical CPU 0x0
 Linux version 4.8.0-239303-g3477d16 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Fri Oct 7 01:12:37 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
@@ -184,7 +184,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(1.159:1): initialized
+audit: type=2000 audit(1.175:1): initialized
 workingset: timestamp_bits=46 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -254,6 +254,7 @@
 +VDD_LED: supplied by +VDD_MUX
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 +5V_USB_HS: supplied by +5V_SYS
 +5V_USB_SS: supplied by +5V_SYS
 +5V_HDMI_CON: supplied by +5V_SYS
@@ -272,9 +273,6 @@
 Freeing unused kernel memory: 960K (ffff800000c00000 - ffff800000cf0000)
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
-sdhci-tegra 700b0400.sdhci: Got CD GPIO
-sdhci-tegra 700b0400.sdhci: Got WP GPIO
 
 System initialization...
 
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-03-07 18:59:53.801075475 -0800
+++ /tmp/foob	2017-03-07 18:59:53.805075491 -0800
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -46,21 +46,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ############################################################
-	 3.3 MiB/s
+	 #########################################################
+	 4.1 MiB/s
 done
-Bytes transferred = 7547624 (732ae8 hex)
+Bytes transferred = 7509352 (729568 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 2.9 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21426 (53b2 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x732ae8 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x729568 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103b1
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc1-next-20170307-156560-g3c90632 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Mar 7 13:21:37 PST 2017
+Linux version 4.11.0-rc1-next-20170308-157162-g56b8bad (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Mar 7 18:49:20 PST 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -78,20 +78,20 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c1539880, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c1539800, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 65536 pages, LIFO batch:15
-percpu: Embedded 16 pages/cpu @ef7c6000 s34432 r8192 d22912 u65536
-pcpu-alloc: s34432 r8192 d22912 u65536 alloc=16*4096
+percpu: Embedded 16 pages/cpu @ef7c6000 s34368 r8192 d22976 u65536
+pcpu-alloc: s34368 r8192 d22976 u65536 alloc=16*4096
 pcpu-alloc: [0] 0 [0] 1 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 260608
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=11eb23c1-01 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 954584K/1048576K available (10240K kernel code, 1260K rwdata, 4240K rodata, 2048K init, 536K bss, 28456K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 954716K/1048576K available (10240K kernel code, 1260K rwdata, 4224K rodata, 2048K init, 407K bss, 28324K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -101,14 +101,13 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc153b1e0   (1261 kB)
-       .bss : 0xc153d000 - 0xc15c32f8   ( 537 kB)
+      .data : 0xc1400000 - 0xc153b160   (1261 kB)
+       .bss : 0xc153d000 - 0xc15a2cb8   ( 408 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	RCU debugfs-based tracing is enabled.
-	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=16 to nr_cpu_ids=2.
-RCU: Adjusting geometry for rcu_fanout_leaf=32, nr_cpu_ids=2
+RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=2
 NR_IRQS:16 nr_irqs:16 16
 /interrupt-controller@60004000: 128 interrupts forwarded to /interrupt-controller@50041000
 L2C: platform modifies aux control register: 0x02080000 -> 0x3e480001
@@ -353,8 +352,8 @@
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
 alloc_contig_range: [3c045, 3c046) PFNs busy
 alloc_contig_range: [3c049, 3c04a) PFNs busy
-rtc-em3027 2-0056: hctosys: unable to read the hardware clock
 alloc_contig_range: [3c04a, 3c04b) PFNs busy
+rtc-em3027 2-0056: hctosys: unable to read the hardware clock
 mmc0: Invalid maximum block size, assuming 512 bytes
 mmc0: Failed to register LED device: -517
 sdhci-tegra c8000600.sdhci: Got CD GPIO
@@ -363,9 +362,9 @@
 alloc_contig_range: [3c049, 3c04a) PFNs busy
 alloc_contig_range: [3c04a, 3c04b) PFNs busy
 alloc_contig_range: [3c04b, 3c04c) PFNs busy
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 mmc0: Invalid maximum block size, assuming 512 bytes
 mmc0: Failed to register LED device: -517
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
 Waiting for root device PARTUUID=11eb23c1-01...
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2017-03-07 18:59:53.809075507 -0800
+++ /tmp/foob	2017-03-07 18:59:53.813075522 -0800
@@ -45,9 +45,9 @@
 	 #################################################################
 	 #################################################################
 	 ######################################################
-	 4.1 MiB/s
+	 4 MiB/s
 done
-Bytes transferred = 5561776 (54ddb0 hex)
+Bytes transferred = 5563344 (54e3d0 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21426 (53b2 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x54ddb0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x54e3d0 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103b1
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc1-next-20170307-156560-g3c90632 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Mar 7 13:21:03 PST 2017
+Linux version 4.11.0-rc1-next-20170308-157162-g56b8bad (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Mar 7 18:48:53 PST 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -76,13 +76,13 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0d5de80, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0d5dec0, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 65536 pages, LIFO batch:15
-percpu: Embedded 16 pages/cpu @ef7c5000 s34368 r8192 d22976 u65536
-pcpu-alloc: s34368 r8192 d22976 u65536 alloc=16*4096
+percpu: Embedded 16 pages/cpu @ef7c5000 s34304 r8192 d23040 u65536
+pcpu-alloc: s34304 r8192 d23040 u65536 alloc=16*4096
 pcpu-alloc: [0] 0 [0] 1 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 260608
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=11eb23c1-01 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
@@ -99,12 +99,11 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d5f360   ( 381 kB)
+      .data : 0xc0d00000 - 0xc0d5f3a0   ( 381 kB)
        .bss : 0xc0d61000 - 0xc0dad40c   ( 306 kB)
 Preemptible hierarchical RCU implementation.
-	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
-RCU: Adjusting geometry for rcu_fanout_leaf=32, nr_cpu_ids=2
+RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=2
 NR_IRQS:16 nr_irqs:16 16
 /interrupt-controller@60004000: 128 interrupts forwarded to /interrupt-controller@50041000
 L2C: platform modifies aux control register: 0x02080000 -> 0x3e480001
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2017-03-07 18:59:53.817075538 -0800
+++ /tmp/foob	2017-03-07 18:59:53.821075554 -0800
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 746567 us
-Basic modules init took 348892 us
+Tegraboot started after 764563 us
+Basic modules init took 348689 us
 Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14362 us
-Carveout took 252735 us
-CPU initialization took 92301 us
-Total time taken by TegraBoot 708290 us
+Next stage binary read took 14365 us
+Carveout took 252725 us
+CPU initialization took 92344 us
+Total time taken by TegraBoot 708123 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.585] LPDDR4 Training: Number of tables = 10
+64b[0001.603] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-15424000 bytes read in 807 ms (18.2 MiB/s)
+15497728 bytes read in 808 ms (18.3 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-44882 bytes read in 460 ms (94.7 KiB/s)
+44882 bytes read in 461 ms (94.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 495 ms (3.1 MiB/s)
+1627787 bytes read in 496 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc1-next-20170307-180993-g3c90632 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Mar 7 13:24:55 PST 2017
+Linux version 4.11.0-rc1-next-20170308-181595-g56b8bad (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Mar 7 18:52:09 PST 2017
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -163,8 +163,8 @@
 psci: PSCIv0.2 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: MIGRATE_INFO_TYPE not supported.
-percpu: Embedded 23 pages/cpu @ffff8000bff4e000 s56960 r8192 d29056 u94208
-pcpu-alloc: s56960 r8192 d29056 u94208 alloc=23*4096
+percpu: Embedded 23 pages/cpu @ffff8000bff4e000 s56832 r8192 d29184 u94208
+pcpu-alloc: s56832 r8192 d29184 u94208 alloc=23*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
@@ -174,15 +174,15 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffff80007a000000-ffff80007dffffff]
-Memory: 2983632K/3131392K available (8700K kernel code, 982K rwdata, 4216K rodata, 1088K init, 395K bss, 131376K reserved, 16384K cma-reserved)
+Memory: 2983560K/3131392K available (8764K kernel code, 990K rwdata, 4228K rodata, 1088K init, 395K bss, 131448K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff000008900000   (  8704 KB)
-    .rodata : 0xffff000008900000 - 0xffff000008d30000   (  4288 KB)
-      .init : 0xffff000008d30000 - 0xffff000008e40000   (  1088 KB)
-      .data : 0xffff000008e40000 - 0xffff000008f35a00   (   983 KB)
-       .bss : 0xffff000008f35a00 - 0xffff000008f98638   (   396 KB)
+      .text : 0xffff000008080000 - 0xffff000008910000   (  8768 KB)
+    .rodata : 0xffff000008910000 - 0xffff000008d40000   (  4288 KB)
+      .init : 0xffff000008d40000 - 0xffff000008e50000   (  1088 KB)
+      .data : 0xffff000008e50000 - 0xffff000008f47a00   (   991 KB)
+       .bss : 0xffff000008f47a00 - 0xffff000008faa638   (   396 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -190,9 +190,8 @@
     memory  : 0xffff800000000000 - 0xffff8000c0000000   (  3072 MB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Preemptible hierarchical RCU implementation.
-	Build-time adjustment of leaf fanout to 64.
 	RCU restricting CPUs from NR_CPUS=64 to nr_cpu_ids=4.
-RCU: Adjusting geometry for rcu_fanout_leaf=64, nr_cpu_ids=4
+RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=4
 NR_IRQS:64 nr_irqs:64 0
 GIC: Using split EOI/Deactivate mode
 /interrupt-controller@60004000: 192 interrupts forwarded to /interrupt-controller@50041000
@@ -236,7 +235,7 @@
 pinctrl core: initialized pinctrl subsystem
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffff000008907000, 1 data @ ffff000008e45000)
+vdso: 2 pages (1 code @ ffff000008917000, 1 data @ ffff000008e55000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -270,7 +269,7 @@
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 PCI: CLS 0 bytes, default 128
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 808f1000
+kvm [1]: IDMAP page: 808f9000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2017-03-07 18:59:53.825075570 -0800
+++ /tmp/foob	2017-03-07 18:59:53.825075570 -0800
@@ -1,11 +1,4 @@
-rocessor [411fd071]
-efi: Getting EFI parameters from FDT:
-efi: UEFI not found.
-cma: Reserved 16 MiB at 0x00000000f5400000
-earlycon: uart0 at MMIO 0x0000000070006000 (options '115200n8')
-bootconsole [uart0] enabled
-NUMA: No NUMA configuration found
-NUMA: Faking a node at [mem 0x0000000000000000-0x000000013fffffff]
+000000-0x000000013fffffff]
 NUMA: Adding memblock [0x80000000 - 0xf65fffff] on node 0
 NUMA: Adding memblock [0x100000000 - 0x13fffffff] on node 0
 NUMA: Initmem setup node 0 [mem 0x80000000-0x13fffffff]
@@ -22,7 +15,7 @@
 psci: PSCIv1.0 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: MIGRATE_INFO_UP_CPU reported invalid physical ID (0x80000000)
-percpu: Embedded 23 pages/cpu @ffff8000bff42000 s56960 r8192 d29056 u94208
+percpu: Embedded 23 pages/cpu @ffff8000bff42000 s56832 r8192 d29184 u94208
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
 CPU features: enabling workaround for ARM erratum 834220
@@ -31,15 +24,15 @@
 Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2837816K/2988032K available (8700K kernel code, 982K rwdata, 4216K rodata, 1088K init, 395K bss, 133832K reserved, 16384K cma-reserved)
+Memory: 2837744K/2988032K available (8764K kernel code, 990K rwdata, 4228K rodata, 1088K init, 395K bss, 133904K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff000008900000   (  8704 KB)
-    .rodata : 0xffff000008900000 - 0xffff000008d30000   (  4288 KB)
-      .init : 0xffff000008d30000 - 0xffff000008e40000   (  1088 KB)
-      .data : 0xffff000008e40000 - 0xffff000008f35a00   (   983 KB)
-       .bss : 0xffff000008f35a00 - 0xffff000008f98638   (   396 KB)
+      .text : 0xffff000008080000 - 0xffff000008910000   (  8768 KB)
+    .rodata : 0xffff000008910000 - 0xffff000008d40000   (  4288 KB)
+      .init : 0xffff000008d40000 - 0xffff000008e50000   (  1088 KB)
+      .data : 0xffff000008e50000 - 0xffff000008f47a00   (   991 KB)
+       .bss : 0xffff000008f47a00 - 0xffff000008faa638   (   396 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -47,9 +40,8 @@
     memory  : 0xffff800000000000 - 0xffff8000c0000000   (  3072 MB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Preemptible hierarchical RCU implementation.
-	Build-time adjustment of leaf fanout to 64.
 	RCU restricting CPUs from NR_CPUS=64 to nr_cpu_ids=4.
-RCU: Adjusting geometry for rcu_fanout_leaf=64, nr_cpu_ids=4
+RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=4
 NR_IRQS:64 nr_irqs:64 0
 GIC: Using split EOI/Deactivate mode
 /interrupt-controller@60004000: 192 interrupts forwarded to /interrupt-controller@50041000
@@ -68,7 +60,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x8ae8: expected 0x2004f400
+boot misc3 0xaad8: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -81,7 +73,7 @@
 console [tty0] enabled
 bootconsole [uart0] disabled
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc1-next-20170307-180993-g3c90632 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Mar 7 13:24:55 PST 2017
+Linux version 4.11.0-rc1-next-20170308-181595-g56b8bad (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Mar 7 18:52:09 PST 2017
 Boot CPU: AArch64 Processor [411fd071]
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
@@ -106,7 +98,7 @@
 psci: PSCIv1.0 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: MIGRATE_INFO_UP_CPU reported invalid physical ID (0x80000000)
-percpu: Embedded 23 pages/cpu @ffff8000bff42000 s56960 r8192 d29056 u94208
+percpu: Embedded 23 pages/cpu @ffff8000bff42000 s56832 r8192 d29184 u94208
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
 CPU features: enabling workaround for ARM erratum 834220
@@ -115,15 +107,15 @@
 Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2837816K/2988032K available (8700K kernel code, 982K rwdata, 4216K rodata, 1088K init, 395K bss, 133832K reserved, 16384K cma-reserved)
+Memory: 2837744K/2988032K available (8764K kernel code, 990K rwdata, 4228K rodata, 1088K init, 395K bss, 133904K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff000008900000   (  8704 KB)
-    .rodata : 0xffff000008900000 - 0xffff000008d30000   (  4288 KB)
-      .init : 0xffff000008d30000 - 0xffff000008e40000   (  1088 KB)
-      .data : 0xffff000008e40000 - 0xffff000008f35a00   (   983 KB)
-       .bss : 0xffff000008f35a00 - 0xffff000008f98638   (   396 KB)
+      .text : 0xffff000008080000 - 0xffff000008910000   (  8768 KB)
+    .rodata : 0xffff000008910000 - 0xffff000008d40000   (  4288 KB)
+      .init : 0xffff000008d40000 - 0xffff000008e50000   (  1088 KB)
+      .data : 0xffff000008e50000 - 0xffff000008f47a00   (   991 KB)
+       .bss : 0xffff000008f47a00 - 0xffff000008faa638   (   396 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -131,9 +123,8 @@
     memory  : 0xffff800000000000 - 0xffff8000c0000000   (  3072 MB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Preemptible hierarchical RCU implementation.
-	Build-time adjustment of leaf fanout to 64.
 	RCU restricting CPUs from NR_CPUS=64 to nr_cpu_ids=4.
-RCU: Adjusting geometry for rcu_fanout_leaf=64, nr_cpu_ids=4
+RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=4
 NR_IRQS:64 nr_irqs:64 0
 GIC: Using split EOI/Deactivate mode
 /interrupt-controller@60004000: 192 interrupts forwarded to /interrupt-controller@50041000
@@ -152,7 +143,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x8ae8: expected 0x2004f400
+boot misc3 0xaad8: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -194,7 +185,7 @@
 pinctrl core: initialized pinctrl subsystem
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffff000008907000, 1 data @ ffff000008e45000)
+vdso: 2 pages (1 code @ ffff000008917000, 1 data @ ffff000008e55000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -235,14 +226,14 @@
 Unpacking initramfs...
 Freeing initrd memory: 2364K
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 808f1000
+kvm [1]: IDMAP page: 808f9000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
 kvm [1]: vgic interrupt IRQ1
 kvm [1]: virtual timer IRQ4
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.331:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.327:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -336,7 +327,7 @@
 alloc_contig_range: [f5469, f546a) PFNs busy
 alloc_contig_range: [f546a, f546b) PFNs busy
 mmc0: Failed to register LED device: -517
-max77686-rtc max77620-rtc: setting system clock to 2000-12-17 02:53:51 UTC (977021631)
+max77686-rtc max77620-rtc: setting system clock to 2000-12-17 08:19:39 UTC (977041179)
 PP1200_AVDD: disabling
 PP1200_RCAM: disabling
 PP2800L_RCAM: disabling
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-03-07 18:59:53.833075602 -0800
+++ /tmp/foob	2017-03-07 18:59:53.833075602 -0800
@@ -38,21 +38,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ############################################################
-	 2.9 MiB/s
+	 #########################################################
+	 3.2 MiB/s
 done
-Bytes transferred = 7547624 (732ae8 hex)
+Bytes transferred = 7509352 (729568 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.6 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 51804 (ca5c hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x732ae8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x729568 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa5b ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc1-next-20170307-156560-g3c90632 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Mar 7 13:21:37 PST 2017
+Linux version 4.11.0-rc1-next-20170308-157162-g56b8bad (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Mar 7 18:49:20 PST 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -70,20 +70,20 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c1539880, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1539800, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 16 pages/cpu @eef8c000 s34432 r8192 d22912 u65536
-pcpu-alloc: s34432 r8192 d22912 u65536 alloc=16*4096
+percpu: Embedded 16 pages/cpu @eef8c000 s34368 r8192 d22976 u65536
+pcpu-alloc: s34368 r8192 d22976 u65536 alloc=16*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=edbe1840-3633-c751-7993-b0ac834f81e0 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993684K/2096128K available (10240K kernel code, 1260K rwdata, 4240K rodata, 2048K init, 536K bss, 36908K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993816K/2096128K available (10240K kernel code, 1260K rwdata, 4224K rodata, 2048K init, 407K bss, 36776K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -93,14 +93,13 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc153b1e0   (1261 kB)
-       .bss : 0xc153d000 - 0xc15c32f8   ( 537 kB)
+      .data : 0xc1400000 - 0xc153b160   (1261 kB)
+       .bss : 0xc153d000 - 0xc15a2cb8   ( 408 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU debugfs-based tracing is enabled.
-	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=16 to nr_cpu_ids=4.
-RCU: Adjusting geometry for rcu_fanout_leaf=32, nr_cpu_ids=4
+RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=4
 NR_IRQS:16 nr_irqs:16 16
 /interrupt-controller@60004000: 160 interrupts forwarded to /interrupt-controller@50041000
 L2C: platform modifies aux control register: 0x02080000 -> 0x3e480001
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2017-03-07 18:59:53.841075633 -0800
+++ /tmp/foob	2017-03-07 18:59:53.841075633 -0800
@@ -39,7 +39,7 @@
 	 ######################################################
 	 3.2 MiB/s
 done
-Bytes transferred = 5561776 (54ddb0 hex)
+Bytes transferred = 5563344 (54e3d0 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 51804 (ca5c hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x54ddb0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x54e3d0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa5b ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.11.0-rc1-next-20170307-156560-g3c90632 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Mar 7 13:21:03 PST 2017
+Linux version 4.11.0-rc1-next-20170308-157162-g56b8bad (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Mar 7 18:48:53 PST 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,13 +68,13 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0d5de80, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0d5dec0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 16 pages/cpu @eef8b000 s34368 r8192 d22976 u65536
-pcpu-alloc: s34368 r8192 d22976 u65536 alloc=16*4096
+percpu: Embedded 16 pages/cpu @eef8b000 s34304 r8192 d23040 u65536
+pcpu-alloc: s34304 r8192 d23040 u65536 alloc=16*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=edbe1840-3633-c751-7993-b0ac834f81e0 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
@@ -91,10 +91,9 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d5f360   ( 381 kB)
+      .data : 0xc0d00000 - 0xc0d5f3a0   ( 381 kB)
        .bss : 0xc0d61000 - 0xc0dad40c   ( 306 kB)
 Preemptible hierarchical RCU implementation.
-	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
 /interrupt-controller@60004000: 160 interrupts forwarded to /interrupt-controller@50041000
 L2C: platform modifies aux control register: 0x02080000 -> 0x3e480001
