Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-04-03 21:55:11.375588646 -0700
+++ /tmp/foob	2016-04-03 21:55:11.375588646 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1-next-20160401+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Mar 31 21:50:17 PDT 2016
+Linux version 4.6.0-rc2-next-20160404+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Apr 3 21:50:07 PDT 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -19,14 +19,14 @@
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
 software IO TLB [mem 0x79c00000-0x7dc00000] (64MB) mapped at [ffffffc039c00000-ffffffc03dbfffff]
-Memory: 936552K/1048576K available (6848K kernel code, 626K rwdata, 2896K rodata, 848K init, 243K bss, 95640K reserved, 16384K cma-reserved)
+Memory: 936308K/1048576K available (7004K kernel code, 627K rwdata, 2976K rodata, 856K init, 242K bss, 95884K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffffff8000000000 - 0xffffff8008000000   (   128 MB)
     vmalloc : 0xffffff8008000000 - 0xffffffbdbfff0000   (   246 GB)
-      .text : 0xffffff8008080000 - 0xffffff800872e000   (  6840 KB)
-    .rodata : 0xffffff800872e000 - 0xffffff8008a06000   (  2912 KB)
-      .init : 0xffffff8008a06000 - 0xffffff8008ada000   (   848 KB)
-      .data : 0xffffff8008ada000 - 0xffffff8008b76800   (   626 KB)
+      .text : 0xffffff8008080000 - 0xffffff8008755000   (  6996 KB)
+    .rodata : 0xffffff8008755000 - 0xffffff8008a41000   (  2992 KB)
+      .init : 0xffffff8008a41000 - 0xffffff8008b17000   (   856 KB)
+      .data : 0xffffff8008b17000 - 0xffffff8008bb3e00   (   628 KB)
     vmemmap : 0xffffffbdc0000000 - 0xffffffbfc0000000   (     8 GB maximum)
               0xffffffbdc0000000 - 0xffffffbdc1000000   (    16 MB actual)
     fixed   : 0xffffffbffe7fd000 - 0xffffffbffec00000   (  4108 KB)
@@ -60,7 +60,7 @@
 pinctrl core: initialized pinctrl subsystem
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffffff8008734000, 1 data @ ffffff8008ae0000)
+vdso: 2 pages (1 code @ ffffff800875b000, 1 data @ ffffff8008b1c000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -94,7 +94,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.540:1): initialized
+audit: type=2000 audit(0.808:1): initialized
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -154,7 +154,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2016-04-01 04:50:49 UTC (1459486249)
+rtc-pl031 9010000.pl031: setting system clock to 2016-04-04 04:50:37 UTC (1459745437)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
@@ -163,7 +163,7 @@
 EXT4-fs (vda2): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 254:2.
 devtmpfs: mounted
-Freeing unused kernel memory: 848K (ffffff8008a06000 - ffffff8008ada000)
+Freeing unused kernel memory: 856K (ffffff8008a41000 - ffffff8008b17000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 bash-4.3# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-04-03 21:55:11.383588645 -0700
+++ /tmp/foob	2016-04-03 21:55:11.383588645 -0700
@@ -38,21 +38,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##################
-	 2.9 MiB/s
+	 ###################
+	 3.4 MiB/s
 done
-Bytes transferred = 6936680 (69d868 hex)
+Bytes transferred = 6944200 (69f5c8 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.6 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 34912 (8860 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x69d868 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x69f5c8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff85f ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1-next-20160401-84376-g7d2277f (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Mar 31 21:48:44 PDT 2016
+Linux version 4.6.0-rc2-next-20160404-84632-g067a845 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Apr 3 21:48:16 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141c6c0, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c141ccc0, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1995128K/2096128K available (9523K kernel code, 1145K rwdata, 4120K rodata, 2048K init, 341K bss, 35464K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1995128K/2096128K available (9523K kernel code, 1147K rwdata, 4124K rodata, 2048K init, 341K bss, 35464K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,9 +90,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc1052ca4   (14636 kB)
+      .text : 0xc0208000 - 0xc1053e5c   (14640 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141e700   (1146 kB)
+      .data : 0xc1300000 - 0xc141ed20   (1148 kB)
        .bss : 0xc1420000 - 0xc1475558   ( 342 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -322,7 +322,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-24 16:07:23 UTC (948730043)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-27 16:05:54 UTC (948989154)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -334,4 +334,5 @@
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.345201] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-04-03 21:55:11.391588644 -0700
+++ /tmp/foob	2016-04-03 21:55:11.391588644 -0700
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 ######################################
-	 3.2 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 5315760 (511cb0 hex)
+Bytes transferred = 5318784 (512880 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 34912 (8860 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x511cb0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x512880 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff85f ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1-next-20160401-84376-g7d2277f (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Mar 31 21:48:30 PDT 2016
+Linux version 4.6.0-rc2-next-20160404-84632-g067a845 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Apr 3 21:48:19 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0c5c640, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0c5c6c0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -90,9 +90,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab2288   (10921 kB)
+      .text : 0xc0008000 - 0xc0ab22e0   (10921 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
-      .data : 0xc0c00000 - 0xc0c5e1c0   ( 377 kB)
+      .data : 0xc0c00000 - 0xc0c5e240   ( 377 kB)
        .bss : 0xc0c60000 - 0xc0cbaa88   ( 363 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -275,15 +275,15 @@
 mmc1: SDHCI controller on 78000600.sdhci [78000600.sdhci] using ADMA
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
-mmc1: new high speed MMC card at address 0001
-mmcblk0: mmc1:0001 SEM32G 29.1 GiB 
-mmcblk0boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmc0: new high speed SDHC card at address aaaa
-mmcblk1: mmc0:aaaa SL16G 14.8 GiB 
-mmcblk0boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
-mmcblk0rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
- mmcblk1: p1 p2 p3
+mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
+mmc1: new high speed MMC card at address 0001
+mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
+mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
+mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
+mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
+ mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
+ mmcblk0: p1 p2 p3
 tegra30-i2s 70080400.i2s: DMA channels sourced from device 70080000.ahub
 tegra-snd-rt5640 sound: rt5640-aif1 <-> 70080400.i2s mapping ok
 input: NVIDIA Tegra Dalmore Headphones as /devices/soc0/sound/sound/card0/input0
@@ -320,16 +320,16 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-24 16:06:02 UTC (948729962)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-27 16:08:19 UTC (948989299)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
 ALSA device list:
   #0: NVIDIA Tegra Dalmore
-EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
-EXT4-fs (mmcblk0p1): recovery complete
-EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:1.
+EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
+EXT4-fs (mmcblk1p1): recovery complete
+EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
+VFS: Mounted root (ext3 filesystem) on device 179:17.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-04-03 21:55:11.399588642 -0700
+++ /tmp/foob	2016-04-03 21:55:11.399588642 -0700
@@ -40,21 +40,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##################
-	 2.8 MiB/s
+	 ###################
+	 3.2 MiB/s
 done
-Bytes transferred = 6936680 (69d868 hex)
+Bytes transferred = 6944200 (69f5c8 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 2.7 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 64236 (faec hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x69d868 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x69f5c8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012aeb
@@ -62,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1-next-20160401-84376-g7d2277f (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Mar 31 21:48:44 PDT 2016
+Linux version 4.6.0-rc2-next-20160404-84632-g067a845 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Apr 3 21:48:16 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -71,7 +71,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141c6c0, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c141ccc0, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -84,7 +84,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1995020K/2096128K available (9523K kernel code, 1145K rwdata, 4120K rodata, 2048K init, 341K bss, 35572K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1995020K/2096128K available (9523K kernel code, 1147K rwdata, 4124K rodata, 2048K init, 341K bss, 35572K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,9 +92,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc1052ca4   (14636 kB)
+      .text : 0xc0208000 - 0xc1053e5c   (14640 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141e700   (1146 kB)
+      .data : 0xc1300000 - 0xc141ed20   (1148 kB)
        .bss : 0xc1420000 - 0xc1475558   ( 342 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -304,6 +304,12 @@
 +5V_HDMI_CON: supplied by +5V_SYS
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
 mmc0: new HS200 MMC card at address 0001
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+Primary GPT is invalid, using alternate GPT.
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9
 tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
 tegra-ahci 70027000.sata: Failed to get regulators
 tegra-ehci 7d008000.usb: EHCI Host Controller
@@ -336,6 +342,7 @@
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2014500000, volt: 1060000, enabled: 1. New: freq: 2014500000, volt: 1060000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2116500000, volt: 1100000, enabled: 1. New: freq: 2116500000, volt: 1100000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2218500000, volt: 1140000, enabled: 1. New: freq: 2218500000, volt: 1140000, enabled: 1
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2320500000, volt: 1180000, enabled: 1. New: freq: 2320500000, volt: 1180000, enabled: 1
 tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
@@ -344,6 +351,7 @@
 tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
 tegra-ahci 70027000.sata: Failed to get regulators
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:15 UTC (946684815)
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc1: Unknown controller version (3). You may experience problems.
@@ -375,20 +383,13 @@
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
 tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
 tegra-ahci 70027000.sata: Failed to get regulators
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 18:03:33 UTC (946749813)
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +USB0_VBUS_SW: disabling
 +1.05V_RUN_AVDD_HDMI_PLL: disabling
 +5V_HDMI_CON: disabling
 +5V_SATA: disabling
 +12V_SATA: disabling
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
-Primary GPT is invalid, using alternate GPT.
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9
+asix 2-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:11:31:3b
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc1: Unknown controller version (3). You may experience problems.
@@ -416,45 +417,13 @@
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2320500000, volt: 1180000, enabled: 1. New: freq: 2320500000, volt: 1180000, enabled: 1
 tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
-tegra-pcie 1003000.pcie-controller: Failed to get supply 'avddio-pex': -517
-tegra-hdmi 54280000.hdmi: failed to get PLL regulator
-tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
-tegra-ahci 70027000.sata: Failed to get regulators
-asix 2-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:11:31:3b
 EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
-sdhci-tegra 700b0400.sdhci: Got CD GPIO
-sdhci-tegra 700b0400.sdhci: Got WP GPIO
-mmc1: Unknown controller version (3). You may experience problems.
 EXT4-fs (mmcblk0p1): couldn't mount as ext2 due to feature incompatibilities
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 204000000, volt: 900000, enabled: 1. New: freq: 204000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 306000000, volt: 900000, enabled: 1. New: freq: 306000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 408000000, volt: 900000, enabled: 1. New: freq: 408000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 510000000, volt: 900000, enabled: 1. New: freq: 510000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 612000000, volt: 900000, enabled: 1. New: freq: 612000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 714000000, volt: 900000, enabled: 1. New: freq: 714000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 816000000, volt: 900000, enabled: 1. New: freq: 816000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 918000000, volt: 900000, enabled: 1. New: freq: 918000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1020000000, volt: 900000, enabled: 1. New: freq: 1020000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1122000000, volt: 900000, enabled: 1. New: freq: 1122000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1224000000, volt: 900000, enabled: 1. New: freq: 1224000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1326000000, volt: 900000, enabled: 1. New: freq: 1326000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1428000000, volt: 900000, enabled: 1. New: freq: 1428000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1530000000, volt: 900000, enabled: 1. New: freq: 1530000000, volt: 900000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1632000000, volt: 920000, enabled: 1. New: freq: 1632000000, volt: 920000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1734000000, volt: 960000, enabled: 1. New: freq: 1734000000, volt: 960000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1836000000, volt: 990000, enabled: 1. New: freq: 1836000000, volt: 990000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1938000000, volt: 1030000, enabled: 1. New: freq: 1938000000, volt: 1030000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2014500000, volt: 1060000, enabled: 1. New: freq: 2014500000, volt: 1060000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2116500000, volt: 1100000, enabled: 1. New: freq: 2116500000, volt: 1100000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2218500000, volt: 1140000, enabled: 1. New: freq: 2218500000, volt: 1140000, enabled: 1
-cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2320500000, volt: 1180000, enabled: 1. New: freq: 2320500000, volt: 1180000, enabled: 1
-tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
-tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
 tegra-pcie 1003000.pcie-controller: Failed to get supply 'avddio-pex': -517
+EXT4-fs (mmcblk0p1): warning: mounting fs with errors, running e2fsck is recommended
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
 tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
 tegra-ahci 70027000.sata: Failed to get regulators
-EXT4-fs (mmcblk0p1): warning: mounting fs with errors, running e2fsck is recommended
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-04-03 21:55:11.403588641 -0700
+++ /tmp/foob	2016-04-03 21:55:11.407588641 -0700
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 ######################################
-	 3.4 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 5315760 (511cb0 hex)
+Bytes transferred = 5318784 (512880 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.1 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 64236 (faec hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x511cb0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x512880 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012aeb
@@ -61,7 +61,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1-next-20160401-84376-g7d2277f (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Mar 31 21:48:30 PDT 2016
+Linux version 4.6.0-rc2-next-20160404-84632-g067a845 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Apr 3 21:48:19 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -71,7 +71,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0c5c640, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0c5c6c0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -92,9 +92,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab2288   (10921 kB)
+      .text : 0xc0008000 - 0xc0ab22e0   (10921 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
-      .data : 0xc0c00000 - 0xc0c5e1c0   ( 377 kB)
+      .data : 0xc0c00000 - 0xc0c5e240   ( 377 kB)
        .bss : 0xc0c60000 - 0xc0cbaa88   ( 363 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -355,10 +355,19 @@
 tegra-ahci 70027000.sata: Failed to get regulators
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 18:02:06 UTC (946749726)
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:16 UTC (946684816)
 mmc1: Unknown controller version (3). You may experience problems.
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 204000000, volt: 900000, enabled: 1. New: freq: 204000000, volt: 900000, enabled: 1
++3.3V_AVDD_HDMI_AP_GATED: disabling
++USB0_VBUS_SW: disabling
++1.05V_RUN_AVDD_HDMI_PLL: disabling
++5V_HDMI_CON: disabling
++5V_SATA: disabling
++12V_SATA: disabling
+ALSA device list:
+  #0: tegra-hda at 0x70038000 irq 344
+  #1: NVIDIA Tegra Jetson TK1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 306000000, volt: 900000, enabled: 1. New: freq: 306000000, volt: 900000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 408000000, volt: 900000, enabled: 1. New: freq: 408000000, volt: 900000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 510000000, volt: 900000, enabled: 1. New: freq: 510000000, volt: 900000, enabled: 1
@@ -386,15 +395,6 @@
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
 tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
 tegra-ahci 70027000.sata: Failed to get regulators
-+3.3V_AVDD_HDMI_AP_GATED: disabling
-+USB0_VBUS_SW: disabling
-+1.05V_RUN_AVDD_HDMI_PLL: disabling
-+5V_HDMI_CON: disabling
-+5V_SATA: disabling
-+12V_SATA: disabling
-ALSA device list:
-  #0: tegra-hda at 0x70038000 irq 344
-  #1: NVIDIA Tegra Jetson TK1
 asix 2-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:11:31:3b
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
@@ -415,18 +415,12 @@
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1428000000, volt: 900000, enabled: 1. New: freq: 1428000000, volt: 900000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1530000000, volt: 900000, enabled: 1. New: freq: 1530000000, volt: 900000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1632000000, volt: 920000, enabled: 1. New: freq: 1632000000, volt: 920000, enabled: 1
-EXT4-fs (mmcblk0p1): warning: mounting fs with errors, running e2fsck is recommended
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1734000000, volt: 960000, enabled: 1. New: freq: 1734000000, volt: 960000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1836000000, volt: 990000, enabled: 1. New: freq: 1836000000, volt: 990000, enabled: 1
-EXT4-fs (mmcblk0p1): recovery complete
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1938000000, volt: 1030000, enabled: 1. New: freq: 1938000000, volt: 1030000, enabled: 1
-EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext4 filesystem) on device 179:1.
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2014500000, volt: 1060000, enabled: 1. New: freq: 2014500000, volt: 1060000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2116500000, volt: 1100000, enabled: 1. New: freq: 2116500000, volt: 1100000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2218500000, volt: 1140000, enabled: 1. New: freq: 2218500000, volt: 1140000, enabled: 1
-devtmpfs: mounted
-Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2320500000, volt: 1180000, enabled: 1. New: freq: 2320500000, volt: 1180000, enabled: 1
 tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
@@ -435,8 +429,14 @@
 tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
 tegra-ahci 70027000.sata: Failed to get regulators
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
+EXT4-fs (mmcblk0p1): warning: mounting fs with errors, running e2fsck is recommended
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc1: Unknown controller version (3). You may experience problems.
+EXT4-fs (mmcblk0p1): recovery complete
+EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
+VFS: Mounted root (ext4 filesystem) on device 179:1.
+devtmpfs: mounted
+Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2016-04-03 21:55:11.411588640 -0700
+++ /tmp/foob	2016-04-03 21:55:11.415588640 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1799909 us
+Bootloader-Cpu Init at (time stamp): 1800618 us
 
 Pinmux changes applied in kernel way
 
@@ -62,22 +62,22 @@
 Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1799883  us
-Bootloader (AosInit) time: 734242 us
-Bootloader Main Init time: 180088 us
-BlockDevMgrInit time: 157539 us
+Miscellaneous avp time: 1800592  us
+Bootloader (AosInit) time: 733452 us
+Bootloader Main Init time: 180225 us
+BlockDevMgrInit time: 157494 us
 PartMgrInit time: 8 us
 StorMgrInit time: 30 us
 FsMgrInit time: 2 us
-Partition Table load: 76701 us 
-Display init time: 574840 us
-Kernel read time: 306 msec
-Load Kernel Size:   13125632 Bytes
-Bl Kernel ReadSpeed: 42894000 Bytes/sec
-Pre-Kernel Jump time: 1494954 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5507059 us
+Partition Table load: 77147 us 
+Display init time: 574412 us
+Kernel read time: 307 msec
+Load Kernel Size:   13377536 Bytes
+Bl Kernel ReadSpeed: 43575000 Bytes/sec
+Pre-Kernel Jump time: 1513916 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5525847 us
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1-next-20160401+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Mar 31 21:50:17 PDT 2016
+Linux version 4.6.0-rc2-next-20160404+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Apr 3 21:50:07 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
 Memory limited to 1734MB
 efi: Getting EFI parameters from FDT:
@@ -92,14 +92,14 @@
 Dentry cache hash table entries: 262144 (order: 9, 2097152 bytes)
 Inode-cache hash table entries: 131072 (order: 8, 1048576 bytes)
 software IO TLB [mem 0xe5200000-0xe9200000] (64MB) mapped at [ffffffc065200000-ffffffc0691fffff]
-Memory: 1643920K/1775616K available (6848K kernel code, 626K rwdata, 2896K rodata, 848K init, 243K bss, 115312K reserved, 16384K cma-reserved)
+Memory: 1643676K/1775616K available (7004K kernel code, 627K rwdata, 2976K rodata, 856K init, 242K bss, 115556K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffffff8000000000 - 0xffffff8008000000   (   128 MB)
     vmalloc : 0xffffff8008000000 - 0xffffffbdbfff0000   (   246 GB)
-      .text : 0xffffff8008080000 - 0xffffff800872e000   (  6840 KB)
-    .rodata : 0xffffff800872e000 - 0xffffff8008a06000   (  2912 KB)
-      .init : 0xffffff8008a06000 - 0xffffff8008ada000   (   848 KB)
-      .data : 0xffffff8008ada000 - 0xffffff8008b76800   (   626 KB)
+      .text : 0xffffff8008080000 - 0xffffff8008755000   (  6996 KB)
+    .rodata : 0xffffff8008755000 - 0xffffff8008a41000   (  2992 KB)
+      .init : 0xffffff8008a41000 - 0xffffff8008b17000   (   856 KB)
+      .data : 0xffffff8008b17000 - 0xffffff8008bb3e00   (   628 KB)
     vmemmap : 0xffffffbdc0000000 - 0xffffffbfc0000000   (     8 GB maximum)
               0xffffffbdc0000000 - 0xffffffbdc1b18000   (    27 MB actual)
     fixed   : 0xffffffbffe7fd000 - 0xffffffbffec00000   (  4108 KB)
@@ -133,7 +133,7 @@
 pinctrl core: initialized pinctrl subsystem
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffffff8008734000, 1 data @ ffffff8008ae0000)
+vdso: 2 pages (1 code @ ffffff800875b000, 1 data @ ffffff8008b1c000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -181,7 +181,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.835:1): initialized
+audit: type=2000 audit(0.839:1): initialized
 workingset: timestamp_bits=44 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -245,32 +245,32 @@
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 hctosys: unable to open rtc device (rtc0)
-mmc0: new HS200 MMC card at address 0001
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
+sdhci-tegra 700b0400.sdhci: Got WP GPIO
+mmc1: Unknown controller version (3). You may experience problems.
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +VDD_LED: disabling
+mmc0: new HS200 MMC card at address 0001
 +5V_USB_HS: disabling
 +5V_USB_SS: disabling
 +1.05V_RUN_AVDD_HDMI_PLL_AP_GATE: disabling
 +5V_HDMI_CON: disabling
 ALSA device list:
   No soundcards found.
-Freeing unused kernel memory: 848K (ffffff8008a06000 - ffffff8008ada000)
-sdhci-tegra 700b0400.sdhci: Got WP GPIO
-mmc1: Unknown controller version (3). You may experience problems.
-sdhci-tegra 700b0400.sdhci: Got CD GPIO
-sdhci-tegra 700b0400.sdhci: Got WP GPIO
-mmc1: Unknown controller version (3). You may experience problems.
+Freeing unused kernel memory: 856K (ffffff8008a41000 - ffffff8008b17000)
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 
 System initialization...
 
-Mounting /proc :[SUCCESS]
-Mounting /sys :[SUCCESS]
-Mounting /dev :[SUCCESS]
+Mounting /proc :[    2.591145] sdhci-tegra 700b0400.sdhci: Got CD GPIO
+[SUCCESS]
+Mounting /sys :[    2.599951] sdhci-tegra 700b0400.sdhci: Got WP GPIO
+[SUCCESS]
+Mounting /dev :[    2.612117] mmc1: Unknown controller version (3). You may experience problems.
+[SUCCESS]
 Mounting /dev/pts :[SUCCESS]
 Mounting /sys/kernel/debug :[SUCCESS]
 Mounting other filesystems :[SUCCESS]
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-04-03 21:55:11.419588639 -0700
+++ /tmp/foob	2016-04-03 21:55:11.423588638 -0700
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -46,21 +46,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##################
-	 3.3 MiB/s
+	 ###################
+	 903.3 KiB/s
 done
-Bytes transferred = 6936680 (69d868 hex)
+Bytes transferred = 6944200 (69f5c8 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 2.9 MiB/s
+	 859.4 KiB/s
 done
 Bytes transferred = 21128 (5288 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x69d868 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x69f5c8 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410287
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1-next-20160401-84376-g7d2277f (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Mar 31 21:48:44 PDT 2016
+Linux version 4.6.0-rc2-next-20160404-84632-g067a845 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Apr 3 21:48:16 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -76,7 +76,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c141c6c0, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c141ccc0, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -89,7 +89,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 955936K/1048576K available (9523K kernel code, 1145K rwdata, 4120K rodata, 2048K init, 341K bss, 27104K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 955936K/1048576K available (9523K kernel code, 1147K rwdata, 4124K rodata, 2048K init, 341K bss, 27104K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -97,9 +97,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc1052ca4   (14636 kB)
+      .text : 0xc0208000 - 0xc1053e5c   (14640 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141e700   (1146 kB)
+      .data : 0xc1300000 - 0xc141ed20   (1148 kB)
        .bss : 0xc1420000 - 0xc1475558   ( 342 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
@@ -320,23 +320,22 @@
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
+mmc0: host does not support reading read-only switch, assuming write-enable
+mmc0: new high speed SDHC card at address b368
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
+ mmcblk0: p1
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-28 13:35:07 UTC (1419773707)
-Waiting for root device PARTUUID=11eb23c1-01...
-mmc0: host does not support reading read-only switch, assuming write-enable
-mmc0: new high speed SDHC card at address b368
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
- mmcblk0: p1
 mmc1: new high speed SD card at address b368
+rtc-em3027 2-0056: setting system clock to 2014-12-28 13:39:19 UTC (1419773959)
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
+EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
-EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-04-03 21:55:11.427588637 -0700
+++ /tmp/foob	2016-04-03 21:55:11.431588637 -0700
@@ -45,20 +45,20 @@
 	 #################################################################
 	 #################################################################
 	 ######################################
-	 909.2 KiB/s
+	 910.2 KiB/s
 done
-Bytes transferred = 5315760 (511cb0 hex)
+Bytes transferred = 5318784 (512880 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 896.5 KiB/s
+	 859.4 KiB/s
 done
 Bytes transferred = 21128 (5288 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x511cb0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x512880 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410287
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1-next-20160401-84376-g7d2277f (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Mar 31 21:48:30 PDT 2016
+Linux version 4.6.0-rc2-next-20160404-84632-g067a845 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Apr 3 21:48:19 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -76,7 +76,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0c5c640, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0c5c6c0, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -97,9 +97,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab2288   (10921 kB)
+      .text : 0xc0008000 - 0xc0ab22e0   (10921 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
-      .data : 0xc0c00000 - 0xc0c5e1c0   ( 377 kB)
+      .data : 0xc0c00000 - 0xc0c5e240   ( 377 kB)
        .bss : 0xc0c60000 - 0xc0cbaa88   ( 363 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -340,7 +340,7 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-28 13:34:01 UTC (1419773641)
+rtc-em3027 2-0056: setting system clock to 2014-12-28 13:40:43 UTC (1419774043)
 ALSA device list:
   #0: tegra-trimslice
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2016-04-03 21:55:11.435588636 -0700
+++ /tmp/foob	2016-04-03 21:55:11.439588635 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 746199 us
-Basic modules init took 348499 us
-Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14368 us
-Carveout took 253209 us
-CPU initialization took 92337 us
-Total time taken by TegraBoot 708413 us
+Tegraboot started after 735875 us
+Basic modules init took 348535 us
+Sec Bootdevice Read Time = 204 ms, Read Size = 8459 KB
+Next stage binary read took 15920 us
+Carveout took 252778 us
+CPU initialization took 92297 us
+Total time taken by TegraBoot 709530 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.585] LPDDR4 Training: Number of tables = 10
+64b[0001.576] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=/dev/mmcblk0p1 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-11495424 bytes read in 732 ms (15 MiB/s)
+11746816 bytes read in 739 ms (15.2 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
 37136 bytes read in 454 ms (79.1 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 494 ms (3.1 MiB/s)
+1627787 bytes read in 495 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1-next-20160401+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Mar 31 21:50:17 PDT 2016
+Linux version 4.6.0-rc2-next-20160404+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Apr 3 21:50:07 PDT 2016
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -166,14 +166,14 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffffffc07a000000-ffffffc07dffffff]
-Memory: 2981492K/3131392K available (6848K kernel code, 626K rwdata, 2896K rodata, 848K init, 243K bss, 133516K reserved, 16384K cma-reserved)
+Memory: 2981248K/3131392K available (7004K kernel code, 627K rwdata, 2976K rodata, 856K init, 242K bss, 133760K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffffff8000000000 - 0xffffff8008000000   (   128 MB)
     vmalloc : 0xffffff8008000000 - 0xffffffbdbfff0000   (   246 GB)
-      .text : 0xffffff8008080000 - 0xffffff800872e000   (  6840 KB)
-    .rodata : 0xffffff800872e000 - 0xffffff8008a06000   (  2912 KB)
-      .init : 0xffffff8008a06000 - 0xffffff8008ada000   (   848 KB)
-      .data : 0xffffff8008ada000 - 0xffffff8008b76800   (   626 KB)
+      .text : 0xffffff8008080000 - 0xffffff8008755000   (  6996 KB)
+    .rodata : 0xffffff8008755000 - 0xffffff8008a41000   (  2992 KB)
+      .init : 0xffffff8008a41000 - 0xffffff8008b17000   (   856 KB)
+      .data : 0xffffff8008b17000 - 0xffffff8008bb3e00   (   628 KB)
     vmemmap : 0xffffffbdc0000000 - 0xffffffbfc0000000   (     8 GB maximum)
               0xffffffbdc0000000 - 0xffffffbdc3000000   (    48 MB actual)
     fixed   : 0xffffffbffe7fd000 - 0xffffffbffec00000   (  4108 KB)
@@ -222,7 +222,7 @@
 pinctrl core: initialized pinctrl subsystem
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffffff8008734000, 1 data @ ffffff8008ae0000)
+vdso: 2 pages (1 code @ ffffff800875b000, 1 data @ ffffff8008b1c000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -259,7 +259,7 @@
 kvm [1]: timer IRQ3
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.363:1): initialized
+audit: type=2000 audit(0.367:1): initialized
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -330,7 +330,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 848K (ffffff8008a06000 - ffffff8008ada000)
+Freeing unused kernel memory: 856K (ffffff8008a41000 - ffffff8008b17000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-04-03 21:55:11.443588634 -0700
+++ /tmp/foob	2016-04-03 21:55:11.447588634 -0700
@@ -38,21 +38,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##################
-	 2.9 MiB/s
+	 ###################
+	 3.3 MiB/s
 done
-Bytes transferred = 6936680 (69d868 hex)
+Bytes transferred = 6944200 (69f5c8 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.6 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x69d868 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x69f5c8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1-next-20160401-84376-g7d2277f (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Mar 31 21:48:44 PDT 2016
+Linux version 4.6.0-rc2-next-20160404-84632-g067a845 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Apr 3 21:48:16 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,7 +68,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141c6c0, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c141ccc0, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -81,7 +81,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1995048K/2096128K available (9523K kernel code, 1145K rwdata, 4120K rodata, 2048K init, 341K bss, 35544K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1995048K/2096128K available (9523K kernel code, 1147K rwdata, 4124K rodata, 2048K init, 341K bss, 35544K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -89,9 +89,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc1052ca4   (14636 kB)
+      .text : 0xc0208000 - 0xc1053e5c   (14640 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141e700   (1146 kB)
+      .data : 0xc1300000 - 0xc141ed20   (1148 kB)
        .bss : 0xc1420000 - 0xc1475558   ( 342 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -291,13 +291,6 @@
 vddio_sdmmc,avdd_vdac: supplied by vdd_5v_in
 avdd_dsi_csi,pwrdet_mipi: supplied by vdd_1v8_gen
 vdd_pllm,x,u,a_p_c_s: supplied by vdd_1v8_gen
-mmc1: new high speed MMC card at address 0001
-mmcblk0: mmc1:0001 SEM16G 14.8 GiB 
-mmcblk0boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
-mmcblk0boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
-mmcblk0rpmb: mmc1:0001 SEM16G partition 3 128 KiB
-Primary GPT is invalid, using alternate GPT.
- mmcblk0: p1 p2 p3 p4 p5
 vdd_ddr_hs: supplied by vdd_1v8_gen
 vdd_ddr: supplied by vdd_5v_in
 vdd_5v_sata: supplied by vdd_5v_in
@@ -306,6 +299,13 @@
 sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
 tegra-pcie 3000.pcie-controller: 2x3 configuration
 tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
+mmc1: new high speed MMC card at address 0001
+mmcblk0: mmc1:0001 SEM16G 14.8 GiB 
+mmcblk0boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
+mmcblk0boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
+mmcblk0rpmb: mmc1:0001 SEM16G partition 3 128 KiB
+Primary GPT is invalid, using alternate GPT.
+ mmcblk0: p1 p2 p3 p4 p5
 tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -377,4 +377,4 @@
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.858481] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.803432] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-04-03 21:55:11.451588633 -0700
+++ /tmp/foob	2016-04-03 21:55:11.455588632 -0700
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 ######################################
-	 3.3 MiB/s
+	 3 MiB/s
 done
-Bytes transferred = 5315760 (511cb0 hex)
+Bytes transferred = 5318784 (512880 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.3 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x511cb0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x512880 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1-next-20160401-84376-g7d2277f (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Mar 31 21:48:30 PDT 2016
+Linux version 4.6.0-rc2-next-20160404-84632-g067a845 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Apr 3 21:48:19 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,7 +68,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0c5c640, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0c5c6c0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -89,9 +89,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab2288   (10921 kB)
+      .text : 0xc0008000 - 0xc0ab22e0   (10921 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
-      .data : 0xc0c00000 - 0xc0c5e1c0   ( 377 kB)
+      .data : 0xc0c00000 - 0xc0c5e240   ( 377 kB)
        .bss : 0xc0c60000 - 0xc0cbaa88   ( 363 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -377,4 +377,5 @@
 Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.080624] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# 
\ No newline at end of file
