Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2015-08-12 07:21:58.403249073 -0700
+++ /tmp/foob	2015-08-12 07:21:58.407249070 -0700
@@ -1,6 +1,6 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
-Linux version 4.2.0-rc6-next-20150810+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Aug 10 00:19:03 PDT 2015
+Linux version 4.2.0-rc6-next-20150812+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Aug 12 07:19:00 PDT 2015
 CPU: AArch64 Processor [411fd070] revision 0
 Detected PIPT I-cache on CPU0
 alternatives: enabling workaround for ARM erratum 832075
@@ -19,7 +19,7 @@
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
 software IO TLB [mem 0x79c00000-0x7dc00000] (64MB) mapped at [ffffffc039c00000-ffffffc03dbfffff]
-Memory: 939192K/1048576K available (5377K kernel code, 470K rwdata, 2244K rodata, 508K init, 204K bss, 93000K reserved, 16384K cma-reserved)
+Memory: 939176K/1048576K available (5381K kernel code, 474K rwdata, 2248K rodata, 512K init, 204K bss, 93016K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     vmalloc : 0xffffff8000000000 - 0xffffffbdbfff0000   (   246 GB)
     vmemmap : 0xffffffbdc0000000 - 0xffffffbfc0000000   (     8 GB maximum)
@@ -28,9 +28,9 @@
     PCI I/O : 0xffffffbffae00000 - 0xffffffbffbe00000   (    16 MB)
     modules : 0xffffffbffc000000 - 0xffffffc000000000   (    64 MB)
     memory  : 0xffffffc000000000 - 0xffffffc040000000   (  1024 MB)
-      .init : 0xffffffc0007f4000 - 0xffffffc000873000   (   508 KB)
-      .text : 0xffffffc000080000 - 0xffffffc0007f3534   (  7630 KB)
-      .data : 0xffffffc000883000 - 0xffffffc0008f8800   (   470 KB)
+      .init : 0xffffffc0007f6000 - 0xffffffc000876000   (   512 KB)
+      .text : 0xffffffc000080000 - 0xffffffc0007f5534   (  7638 KB)
+      .data : 0xffffffc000886000 - 0xffffffc0008fca00   (   475 KB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 64.
@@ -61,7 +61,7 @@
 NET: Registered protocol family 16
 cpuidle: using governor ladder
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffffffc000889000, 1 data @ ffffffc000888000)
+vdso: 2 pages (1 code @ ffffffc00088d000, 1 data @ ffffffc00088c000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -88,7 +88,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 2, 16384 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.390:1): initialized
+audit: type=2000 audit(0.400:1): initialized
 HugeTLB registered 2 MB page size, pre-allocated 0 pages
 VFS: Disk quotas dquot_6.6.0
 VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
@@ -132,8 +132,8 @@
 EXT4-fs (vda2): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 254:2.
 devtmpfs: mounted
-Freeing unused kernel memory: 508K (ffffffc0007f4000 - ffffffc000873000)
-Freeing alternatives memory: 44K (ffffffc000873000 - ffffffc00087e000)
+Freeing unused kernel memory: 512K (ffffffc0007f6000 - ffffffc000876000)
+Freeing alternatives memory: 44K (ffffffc000876000 - ffffffc000881000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 bash-4.3# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-08-12 07:21:58.427249054 -0700
+++ /tmp/foob	2015-08-12 07:21:58.431249050 -0700
@@ -40,7 +40,7 @@
 	 #############################################
 	 3.3 MiB/s
 done
-Bytes transferred = 6378000 (615210 hex)
+Bytes transferred = 6383368 (616708 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -51,7 +51,7 @@
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x615210 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x616708 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc6-next-20150810-35016-gb195df5 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Aug 10 00:17:40 PDT 2015
+Linux version 4.2.0-rc6-next-20150812-35595-gc1a0c66 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Aug 12 07:17:44 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -80,7 +80,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999008K/2096128K available (8014K kernel code, 1016K rwdata, 3604K rodata, 1008K init, 340K bss, 31584K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1999008K/2096128K available (8018K kernel code, 1016K rwdata, 3604K rodata, 1004K init, 340K bss, 31584K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,8 +88,8 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d61b1c   (11623 kB)
-      .init : 0xc0d62000 - 0xc0e5e000   (1008 kB)
+      .text : 0xc0208000 - 0xc0d62b54   (11627 kB)
+      .init : 0xc0d63000 - 0xc0e5e000   (1004 kB)
       .data : 0xc0e5e000 - 0xc0f5c180   (1017 kB)
        .bss : 0xc0f5f000 - 0xc0fb4268   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
@@ -296,7 +296,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-20 16:11:02 UTC (951063062)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-22 23:11:17 UTC (951261077)
 vdd_cam_1v8_reg: disabling
 usb1_vbus: disabling
 dvdd_ts: disabling
@@ -305,7 +305,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1008K (c0d62000 - c0e5e000)
+Freeing unused kernel memory: 1004K (c0d63000 - c0e5e000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.198949] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.224192] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2015-08-12 07:21:58.435249047 -0700
+++ /tmp/foob	2015-08-12 07:21:58.439249045 -0700
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 ###################
-	 3.3 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 5039696 (4ce650 hex)
+Bytes transferred = 5041872 (4ceed0 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4ce650 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4ceed0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc6-next-20150810-35016-gb195df5 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Aug 10 00:18:05 PDT 2015
+Linux version 4.2.0-rc6-next-20150812-35595-gc1a0c66 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Aug 12 07:18:08 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c09b47c0, node_mem_map ee7f7000
+free_area_init_node: node 0, pgdat c09b67c0, node_mem_map ee7f7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002716K/2096128K available (6872K kernel code, 344K rwdata, 2380K rodata, 312K init, 363K bss, 27876K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 2002708K/2096128K available (6876K kernel code, 344K rwdata, 2380K rodata, 316K init, 363K bss, 27884K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0911660   (9254 kB)
-      .init : 0xc0912000 - 0xc0960000   ( 312 kB)
-      .data : 0xc0960000 - 0xc09b6340   ( 345 kB)
-       .bss : 0xc09b9000 - 0xc0a13da0   ( 364 kB)
+      .text : 0xc0008000 - 0xc0912698   (9258 kB)
+      .init : 0xc0913000 - 0xc0962000   ( 316 kB)
+      .data : 0xc0962000 - 0xc09b8340   ( 345 kB)
+       .bss : 0xc09bb000 - 0xc0a15da0   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -281,6 +281,58 @@
 vdd_hdmi_5v0: supplied by vdd-sys-5v0
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
+------------[ cut here ]------------
+WARNING: CPU: 0 PID: 70 at include/linux/ww_mutex.h:158 drm_atomic_check_only+0x4fc/0x574()
+DEBUG_LOCKS_WARN_ON(ctx->done_acquire)
+Modules linked in:
+CPU: 0 PID: 70 Comm: kworker/u8:1 Not tainted 4.2.0-rc6-next-20150812-35595-gc1a0c66 #1
+Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
+Workqueue: deferwq deferred_probe_work_func
+[<c0016898>] (unwind_backtrace) from [<c0012124>] (show_stack+0x10/0x14)
+[<c0012124>] (show_stack) from [<c067cc68>] (dump_stack+0x84/0xd0)
+[<c067cc68>] (dump_stack) from [<c00257a8>] (warn_slowpath_common+0x74/0xb0)
+[<c00257a8>] (warn_slowpath_common) from [<c0025878>] (warn_slowpath_fmt+0x30/0x40)
+[<c0025878>] (warn_slowpath_fmt) from [<c02e3eec>] (drm_atomic_check_only+0x4fc/0x574)
+[<c02e3eec>] (drm_atomic_check_only) from [<c02e3fd8>] (drm_atomic_commit+0x10/0x64)
+[<c02e3fd8>] (drm_atomic_commit) from [<c02c2ac8>] (drm_atomic_helper_set_config+0x394/0x3ec)
+[<c02c2ac8>] (drm_atomic_helper_set_config) from [<c02d4180>] (drm_mode_set_config_internal+0x58/0xd4)
+[<c02d4180>] (drm_mode_set_config_internal) from [<c02c50d8>] (restore_fbdev_mode+0xd4/0xf4)
+[<c02c50d8>] (restore_fbdev_mode) from [<c02c6940>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
+[<c02c6940>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c02c69ac>] (drm_fb_helper_set_par+0x2c/0x54)
+[<c02c69ac>] (drm_fb_helper_set_par) from [<c02632d4>] (fbcon_init+0x474/0x4dc)
+[<c02632d4>] (fbcon_init) from [<c0299e70>] (visual_init+0xb0/0xfc)
+[<c0299e70>] (visual_init) from [<c029be14>] (do_bind_con_driver+0x134/0x354)
+[<c029be14>] (do_bind_con_driver) from [<c029c1a4>] (do_take_over_console+0x170/0x1dc)
+[<c029c1a4>] (do_take_over_console) from [<c02633a0>] (do_fbcon_takeover+0x64/0xcc)
+[<c02633a0>] (do_fbcon_takeover) from [<c003e4ac>] (notifier_call_chain+0x44/0x84)
+[<c003e4ac>] (notifier_call_chain) from [<c003e680>] (__blocking_notifier_call_chain+0x48/0x60)
+[<c003e680>] (__blocking_notifier_call_chain) from [<c003e6b0>] (blocking_notifier_call_chain+0x18/0x20)
+[<c003e6b0>] (blocking_notifier_call_chain) from [<c026d290>] (register_framebuffer+0x184/0x264)
+[<c026d290>] (register_framebuffer) from [<c02c6c18>] (drm_fb_helper_initial_config+0x244/0x3e4)
+[<c02c6c18>] (drm_fb_helper_initial_config) from [<c02e8c70>] (tegra_drm_fb_init+0x48/0xb4)
+[<c02e8c70>] (tegra_drm_fb_init) from [<c02e6f08>] (tegra_drm_load+0x170/0x224)
+[<c02e6f08>] (tegra_drm_load) from [<c02cef0c>] (drm_dev_register+0xa0/0xb4)
+[<c02cef0c>] (drm_dev_register) from [<c02e6a64>] (host1x_drm_probe+0x44/0xb0)
+[<c02e6a64>] (host1x_drm_probe) from [<c02b1e90>] (host1x_device_probe+0x1c/0x28)
+[<c02b1e90>] (host1x_device_probe) from [<c02fd408>] (driver_probe_device+0x1e0/0x2a0)
+[<c02fd408>] (driver_probe_device) from [<c02fbaa8>] (bus_for_each_drv+0x44/0x8c)
+[<c02fbaa8>] (bus_for_each_drv) from [<c02fd19c>] (__device_attach+0xa0/0x104)
+[<c02fd19c>] (__device_attach) from [<c02fc918>] (bus_probe_device+0x84/0x8c)
+[<c02fc918>] (bus_probe_device) from [<c02fae6c>] (device_add+0x33c/0x524)
+[<c02fae6c>] (device_add) from [<c02b209c>] (host1x_subdev_register+0xb0/0xd4)
+[<c02b209c>] (host1x_subdev_register) from [<c02b2954>] (host1x_client_register+0xf4/0x11c)
+[<c02b2954>] (host1x_client_register) from [<c02f016c>] (tegra_hdmi_probe+0x170/0x250)
+[<c02f016c>] (tegra_hdmi_probe) from [<c02fe8e0>] (platform_drv_probe+0x48/0xa4)
+[<c02fe8e0>] (platform_drv_probe) from [<c02fd408>] (driver_probe_device+0x1e0/0x2a0)
+[<c02fd408>] (driver_probe_device) from [<c02fbaa8>] (bus_for_each_drv+0x44/0x8c)
+[<c02fbaa8>] (bus_for_each_drv) from [<c02fd19c>] (__device_attach+0xa0/0x104)
+[<c02fd19c>] (__device_attach) from [<c02fc918>] (bus_probe_device+0x84/0x8c)
+[<c02fc918>] (bus_probe_device) from [<c02fccf0>] (deferred_probe_work_func+0x5c/0x88)
+[<c02fccf0>] (deferred_probe_work_func) from [<c00385b0>] (process_one_work+0x120/0x328)
+[<c00385b0>] (process_one_work) from [<c0038814>] (worker_thread+0x28/0x490)
+[<c0038814>] (worker_thread) from [<c003d90c>] (kthread+0xd8/0xf4)
+[<c003d90c>] (kthread) from [<c000f628>] (ret_from_fork+0x14/0x2c)
+---[ end trace 402c977eafd4c8f8 ]---
 Console: switching to colour frame buffer device 240x75
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
@@ -292,7 +344,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-20 16:11:33 UTC (951063093)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-22 23:11:45 UTC (951261105)
 vdd_cam_1v8_reg: disabling
 usb1_vbus: disabling
 dvdd_ts: disabling
@@ -304,7 +356,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 312K (c0912000 - c0960000)
+Freeing unused kernel memory: 316K (c0913000 - c0962000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.730095] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.160079] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-08-12 07:21:58.443249041 -0700
+++ /tmp/foob	2015-08-12 07:21:58.447249037 -0700
@@ -42,18 +42,18 @@
 	 #############################################
 	 3.2 MiB/s
 done
-Bytes transferred = 6378000 (615210 hex)
+Bytes transferred = 6383368 (616708 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *#####
-	 3.2 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x615210 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x616708 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012a89
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc6-next-20150810-35016-gb195df5 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Aug 10 00:17:40 PDT 2015
+Linux version 4.2.0-rc6-next-20150812-35595-gc1a0c66 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Aug 12 07:17:44 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998900K/2096128K available (8014K kernel code, 1016K rwdata, 3604K rodata, 1008K init, 340K bss, 31692K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1998900K/2096128K available (8018K kernel code, 1016K rwdata, 3604K rodata, 1004K init, 340K bss, 31692K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,8 +90,8 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d61b1c   (11623 kB)
-      .init : 0xc0d62000 - 0xc0e5e000   (1008 kB)
+      .text : 0xc0208000 - 0xc0d62b54   (11627 kB)
+      .init : 0xc0d63000 - 0xc0e5e000   (1004 kB)
       .data : 0xc0e5e000 - 0xc0f5c180   (1017 kB)
        .bss : 0xc0f5f000 - 0xc0fb4268   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
@@ -286,12 +286,6 @@
 Registering SWP/SWPB emulation handler
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
 mmc1: MAN_BKOPS_EN bit is not set
-+3.3V_SYS: supplied by +VDD_MUX
-+3.3V_LP0: supplied by +3.3V_SYS
-+1.35V_LP0(sd2): supplied by +5V_SYS
-+1.05V_RUN_AVDD: supplied by +1.35V_LP0(sd2)
-tegra-pcie 1003000.pcie-controller: probing port 0, using 2 lanes
-tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
 mmc1: new high speed MMC card at address 0001
 mmcblk0: mmc1:0001 SEM16G 14.6 GiB 
 mmcblk0boot0: mmc1:0001 SEM16G partition 1 4.00 MiB
@@ -299,6 +293,12 @@
 mmcblk0rpmb: mmc1:0001 SEM16G partition 3 4.00 MiB
 Primary GPT is invalid, using alternate GPT.
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9
++3.3V_SYS: supplied by +VDD_MUX
++3.3V_LP0: supplied by +3.3V_SYS
++1.35V_LP0(sd2): supplied by +5V_SYS
++1.05V_RUN_AVDD: supplied by +1.35V_LP0(sd2)
+tegra-pcie 1003000.pcie-controller: probing port 0, using 2 lanes
+tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
 tegra-pcie 1003000.pcie-controller: link 0 down, retrying
 tegra-pcie 1003000.pcie-controller: link 0 down, retrying
 tegra-pcie 1003000.pcie-controller: link 0 down, retrying
@@ -361,7 +361,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1008K (c0d62000 - c0e5e000)
+Freeing unused kernel memory: 1004K (c0d63000 - c0e5e000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2015-08-12 07:21:58.451249035 -0700
+++ /tmp/foob	2015-08-12 07:21:58.455249032 -0700
@@ -41,18 +41,18 @@
 	 ###################
 	 3.2 MiB/s
 done
-Bytes transferred = 5039696 (4ce650 hex)
+Bytes transferred = 5041872 (4ceed0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *#####
-	 2.9 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4ce650 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4ceed0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012a89
@@ -63,7 +63,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc6-next-20150810-35016-gb195df5 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Aug 10 00:18:05 PDT 2015
+Linux version 4.2.0-rc6-next-20150812-35595-gc1a0c66 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Aug 12 07:18:08 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -71,7 +71,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c09b47c0, node_mem_map ee7f7000
+free_area_init_node: node 0, pgdat c09b67c0, node_mem_map ee7f7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -84,7 +84,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002608K/2096128K available (6872K kernel code, 344K rwdata, 2380K rodata, 312K init, 363K bss, 27984K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 2002600K/2096128K available (6876K kernel code, 344K rwdata, 2380K rodata, 316K init, 363K bss, 27992K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,10 +92,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0911660   (9254 kB)
-      .init : 0xc0912000 - 0xc0960000   ( 312 kB)
-      .data : 0xc0960000 - 0xc09b6340   ( 345 kB)
-       .bss : 0xc09b9000 - 0xc0a13da0   ( 364 kB)
+      .text : 0xc0008000 - 0xc0912698   (9258 kB)
+      .init : 0xc0913000 - 0xc0962000   ( 316 kB)
+      .data : 0xc0962000 - 0xc09b8340   ( 345 kB)
+       .bss : 0xc09bb000 - 0xc0a15da0   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -319,6 +319,58 @@
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
 tegra-hdmi 54280000.hdmi: cannot set audio to 44100 at 297000000 pclk
+------------[ cut here ]------------
+WARNING: CPU: 0 PID: 72 at include/linux/ww_mutex.h:158 drm_atomic_check_only+0x4fc/0x574()
+DEBUG_LOCKS_WARN_ON(ctx->done_acquire)
+Modules linked in:
+CPU: 0 PID: 72 Comm: kworker/u8:1 Not tainted 4.2.0-rc6-next-20150812-35595-gc1a0c66 #1
+Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
+Workqueue: deferwq deferred_probe_work_func
+[<c0016898>] (unwind_backtrace) from [<c0012124>] (show_stack+0x10/0x14)
+[<c0012124>] (show_stack) from [<c067cc68>] (dump_stack+0x84/0xd0)
+[<c067cc68>] (dump_stack) from [<c00257a8>] (warn_slowpath_common+0x74/0xb0)
+[<c00257a8>] (warn_slowpath_common) from [<c0025878>] (warn_slowpath_fmt+0x30/0x40)
+[<c0025878>] (warn_slowpath_fmt) from [<c02e3eec>] (drm_atomic_check_only+0x4fc/0x574)
+[<c02e3eec>] (drm_atomic_check_only) from [<c02e3fd8>] (drm_atomic_commit+0x10/0x64)
+[<c02e3fd8>] (drm_atomic_commit) from [<c02c2ac8>] (drm_atomic_helper_set_config+0x394/0x3ec)
+[<c02c2ac8>] (drm_atomic_helper_set_config) from [<c02d4180>] (drm_mode_set_config_internal+0x58/0xd4)
+[<c02d4180>] (drm_mode_set_config_internal) from [<c02c50d8>] (restore_fbdev_mode+0xd4/0xf4)
+[<c02c50d8>] (restore_fbdev_mode) from [<c02c6940>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
+[<c02c6940>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c02c69ac>] (drm_fb_helper_set_par+0x2c/0x54)
+[<c02c69ac>] (drm_fb_helper_set_par) from [<c02632d4>] (fbcon_init+0x474/0x4dc)
+[<c02632d4>] (fbcon_init) from [<c0299e70>] (visual_init+0xb0/0xfc)
+[<c0299e70>] (visual_init) from [<c029be14>] (do_bind_con_driver+0x134/0x354)
+[<c029be14>] (do_bind_con_driver) from [<c029c1a4>] (do_take_over_console+0x170/0x1dc)
+[<c029c1a4>] (do_take_over_console) from [<c02633a0>] (do_fbcon_takeover+0x64/0xcc)
+[<c02633a0>] (do_fbcon_takeover) from [<c003e4ac>] (notifier_call_chain+0x44/0x84)
+[<c003e4ac>] (notifier_call_chain) from [<c003e680>] (__blocking_notifier_call_chain+0x48/0x60)
+[<c003e680>] (__blocking_notifier_call_chain) from [<c003e6b0>] (blocking_notifier_call_chain+0x18/0x20)
+[<c003e6b0>] (blocking_notifier_call_chain) from [<c026d290>] (register_framebuffer+0x184/0x264)
+[<c026d290>] (register_framebuffer) from [<c02c6c18>] (drm_fb_helper_initial_config+0x244/0x3e4)
+[<c02c6c18>] (drm_fb_helper_initial_config) from [<c02e8c70>] (tegra_drm_fb_init+0x48/0xb4)
+[<c02e8c70>] (tegra_drm_fb_init) from [<c02e6f08>] (tegra_drm_load+0x170/0x224)
+[<c02e6f08>] (tegra_drm_load) from [<c02cef0c>] (drm_dev_register+0xa0/0xb4)
+[<c02cef0c>] (drm_dev_register) from [<c02e6a64>] (host1x_drm_probe+0x44/0xb0)
+[<c02e6a64>] (host1x_drm_probe) from [<c02b1e90>] (host1x_device_probe+0x1c/0x28)
+[<c02b1e90>] (host1x_device_probe) from [<c02fd408>] (driver_probe_device+0x1e0/0x2a0)
+[<c02fd408>] (driver_probe_device) from [<c02fbaa8>] (bus_for_each_drv+0x44/0x8c)
+[<c02fbaa8>] (bus_for_each_drv) from [<c02fd19c>] (__device_attach+0xa0/0x104)
+[<c02fd19c>] (__device_attach) from [<c02fc918>] (bus_probe_device+0x84/0x8c)
+[<c02fc918>] (bus_probe_device) from [<c02fae6c>] (device_add+0x33c/0x524)
+[<c02fae6c>] (device_add) from [<c02b209c>] (host1x_subdev_register+0xb0/0xd4)
+[<c02b209c>] (host1x_subdev_register) from [<c02b2954>] (host1x_client_register+0xf4/0x11c)
+[<c02b2954>] (host1x_client_register) from [<c02f016c>] (tegra_hdmi_probe+0x170/0x250)
+[<c02f016c>] (tegra_hdmi_probe) from [<c02fe8e0>] (platform_drv_probe+0x48/0xa4)
+[<c02fe8e0>] (platform_drv_probe) from [<c02fd408>] (driver_probe_device+0x1e0/0x2a0)
+[<c02fd408>] (driver_probe_device) from [<c02fbaa8>] (bus_for_each_drv+0x44/0x8c)
+[<c02fbaa8>] (bus_for_each_drv) from [<c02fd19c>] (__device_attach+0xa0/0x104)
+[<c02fd19c>] (__device_attach) from [<c02fc918>] (bus_probe_device+0x84/0x8c)
+[<c02fc918>] (bus_probe_device) from [<c02fccf0>] (deferred_probe_work_func+0x5c/0x88)
+[<c02fccf0>] (deferred_probe_work_func) from [<c00385b0>] (process_one_work+0x120/0x328)
+[<c00385b0>] (process_one_work) from [<c0038814>] (worker_thread+0x28/0x490)
+[<c0038814>] (worker_thread) from [<c003d90c>] (kthread+0xd8/0xf4)
+[<c003d90c>] (kthread) from [<c000f628>] (ret_from_fork+0x14/0x2c)
+---[ end trace b5ae9c672fd7bc21 ]---
 Console: switching to colour frame buffer device 512x135
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
@@ -341,7 +393,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-02-01 00:00:54 UTC (949363254)
+as3722-rtc as3722-rtc: setting system clock to 2000-02-01 00:00:53 UTC (949363253)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_VPP_FUSE: disabling
 +2.8V_RUN_CAM_AF: disabling
@@ -355,13 +407,13 @@
   #1: NVIDIA Tegra Jetson TK1
 ata1: SATA link down (SStatus 0 SControl 300)
 EXT2-fs (mmcblk0p1): error: couldn't mount because of unsupported optional features (244)
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 312K (c0912000 - c0960000)
+Freeing unused kernel memory: 316K (c0913000 - c0962000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# 
\ No newline at end of file
+root@(none):/# [    7.873270] asix 2-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:11:31:3b
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-08-12 07:21:58.459249029 -0700
+++ /tmp/foob	2015-08-12 07:21:58.463249025 -0700
@@ -48,18 +48,18 @@
 	 #############################################
 	 4.1 MiB/s
 done
-Bytes transferred = 6378000 (615210 hex)
+Bytes transferred = 6383368 (616708 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 4 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x615210 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x616708 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -67,7 +67,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc6-next-20150810-35016-gb195df5 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Aug 10 00:17:40 PDT 2015
+Linux version 4.2.0-rc6-next-20150812-35595-gc1a0c66 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Aug 12 07:17:44 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -88,7 +88,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 959816K/1048576K available (8014K kernel code, 1016K rwdata, 3604K rodata, 1008K init, 340K bss, 23224K reserved, 65536K cma-reserved, 204800K highmem)
+Memory: 959816K/1048576K available (8018K kernel code, 1016K rwdata, 3604K rodata, 1004K init, 340K bss, 23224K reserved, 65536K cma-reserved, 204800K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -96,8 +96,8 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d61b1c   (11623 kB)
-      .init : 0xc0d62000 - 0xc0e5e000   (1008 kB)
+      .text : 0xc0208000 - 0xc0d62b54   (11627 kB)
+      .init : 0xc0d63000 - 0xc0e5e000   (1004 kB)
       .data : 0xc0e5e000 - 0xc0f5c180   (1017 kB)
        .bss : 0xc0f5f000 - 0xc0fb4268   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
@@ -334,8 +334,8 @@
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
 hub 1-1:1.0: USB hub found
+rtc-em3027 2-0056: setting system clock to 2014-12-26 20:24:32 UTC (1419625472)
 hub 1-1:1.0: 4 ports detected
-rtc-em3027 2-0056: setting system clock to 2014-12-26 20:23:56 UTC (1419625436)
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
@@ -344,7 +344,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1008K (c0d62000 - c0e5e000)
+Freeing unused kernel memory: 1004K (c0d63000 - c0e5e000)
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2015-08-12 07:21:58.467249022 -0700
+++ /tmp/foob	2015-08-12 07:21:58.471249019 -0700
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -47,7 +47,7 @@
 	 ###################
 	 4.1 MiB/s
 done
-Bytes transferred = 5039696 (4ce650 hex)
+Bytes transferred = 5041872 (4ceed0 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x4ce650 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x4ceed0 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -69,7 +69,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc6-next-20150810-35016-gb195df5 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Aug 10 00:18:05 PDT 2015
+Linux version 4.2.0-rc6-next-20150812-35595-gc1a0c66 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Aug 12 07:18:08 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -78,7 +78,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c09b47c0, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c09b67c0, node_mem_map eeff7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -91,7 +91,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 963524K/1048576K available (6872K kernel code, 344K rwdata, 2380K rodata, 312K init, 363K bss, 19516K reserved, 65536K cma-reserved, 204800K highmem)
+Memory: 963516K/1048576K available (6876K kernel code, 344K rwdata, 2380K rodata, 316K init, 363K bss, 19524K reserved, 65536K cma-reserved, 204800K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -99,10 +99,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0911660   (9254 kB)
-      .init : 0xc0912000 - 0xc0960000   ( 312 kB)
-      .data : 0xc0960000 - 0xc09b6340   ( 345 kB)
-       .bss : 0xc09b9000 - 0xc0a13da0   ( 364 kB)
+      .text : 0xc0008000 - 0xc0912698   (9258 kB)
+      .init : 0xc0913000 - 0xc0962000   ( 316 kB)
+      .data : 0xc0962000 - 0xc09b8340   ( 345 kB)
+       .bss : 0xc09bb000 - 0xc0a15da0   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
@@ -237,6 +237,58 @@
 [drm] No driver support for vblank timestamp query.
 drm drm: No connectors reported connected with modes
 [drm] Cannot find any crtc or sizes - going 1024x768
+------------[ cut here ]------------
+WARNING: CPU: 1 PID: 1 at include/linux/ww_mutex.h:158 drm_atomic_check_only+0x4fc/0x574()
+DEBUG_LOCKS_WARN_ON(ctx->done_acquire)
+Modules linked in:
+CPU: 1 PID: 1 Comm: swapper/0 Not tainted 4.2.0-rc6-next-20150812-35595-gc1a0c66 #1
+Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
+[<c0016898>] (unwind_backtrace) from [<c0012124>] (show_stack+0x10/0x14)
+[<c0012124>] (show_stack) from [<c067cc68>] (dump_stack+0x84/0xd0)
+[<c067cc68>] (dump_stack) from [<c00257a8>] (warn_slowpath_common+0x74/0xb0)
+[<c00257a8>] (warn_slowpath_common) from [<c0025878>] (warn_slowpath_fmt+0x30/0x40)
+[<c0025878>] (warn_slowpath_fmt) from [<c02e3eec>] (drm_atomic_check_only+0x4fc/0x574)
+[<c02e3eec>] (drm_atomic_check_only) from [<c02e3fd8>] (drm_atomic_commit+0x10/0x64)
+[<c02e3fd8>] (drm_atomic_commit) from [<c02c2ac8>] (drm_atomic_helper_set_config+0x394/0x3ec)
+[<c02c2ac8>] (drm_atomic_helper_set_config) from [<c02d4180>] (drm_mode_set_config_internal+0x58/0xd4)
+[<c02d4180>] (drm_mode_set_config_internal) from [<c02c50d8>] (restore_fbdev_mode+0xd4/0xf4)
+[<c02c50d8>] (restore_fbdev_mode) from [<c02c6940>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
+[<c02c6940>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c02c69ac>] (drm_fb_helper_set_par+0x2c/0x54)
+[<c02c69ac>] (drm_fb_helper_set_par) from [<c02632d4>] (fbcon_init+0x474/0x4dc)
+[<c02632d4>] (fbcon_init) from [<c0299e70>] (visual_init+0xb0/0xfc)
+[<c0299e70>] (visual_init) from [<c029be14>] (do_bind_con_driver+0x134/0x354)
+[<c029be14>] (do_bind_con_driver) from [<c029c1a4>] (do_take_over_console+0x170/0x1dc)
+[<c029c1a4>] (do_take_over_console) from [<c02633a0>] (do_fbcon_takeover+0x64/0xcc)
+[<c02633a0>] (do_fbcon_takeover) from [<c003e4ac>] (notifier_call_chain+0x44/0x84)
+[<c003e4ac>] (notifier_call_chain) from [<c003e680>] (__blocking_notifier_call_chain+0x48/0x60)
+[<c003e680>] (__blocking_notifier_call_chain) from [<c003e6b0>] (blocking_notifier_call_chain+0x18/0x20)
+[<c003e6b0>] (blocking_notifier_call_chain) from [<c026d290>] (register_framebuffer+0x184/0x264)
+[<c026d290>] (register_framebuffer) from [<c02c6c18>] (drm_fb_helper_initial_config+0x244/0x3e4)
+[<c02c6c18>] (drm_fb_helper_initial_config) from [<c02e8c70>] (tegra_drm_fb_init+0x48/0xb4)
+[<c02e8c70>] (tegra_drm_fb_init) from [<c02e6f08>] (tegra_drm_load+0x170/0x224)
+[<c02e6f08>] (tegra_drm_load) from [<c02cef0c>] (drm_dev_register+0xa0/0xb4)
+[<c02cef0c>] (drm_dev_register) from [<c02e6a64>] (host1x_drm_probe+0x44/0xb0)
+[<c02e6a64>] (host1x_drm_probe) from [<c02b1e90>] (host1x_device_probe+0x1c/0x28)
+[<c02b1e90>] (host1x_device_probe) from [<c02fd408>] (driver_probe_device+0x1e0/0x2a0)
+[<c02fd408>] (driver_probe_device) from [<c02fbaa8>] (bus_for_each_drv+0x44/0x8c)
+[<c02fbaa8>] (bus_for_each_drv) from [<c02fd19c>] (__device_attach+0xa0/0x104)
+[<c02fd19c>] (__device_attach) from [<c02fc918>] (bus_probe_device+0x84/0x8c)
+[<c02fc918>] (bus_probe_device) from [<c02fae6c>] (device_add+0x33c/0x524)
+[<c02fae6c>] (device_add) from [<c02b209c>] (host1x_subdev_register+0xb0/0xd4)
+[<c02b209c>] (host1x_subdev_register) from [<c02b2954>] (host1x_client_register+0xf4/0x11c)
+[<c02b2954>] (host1x_client_register) from [<c02f6bc0>] (gr3d_probe+0x100/0x21c)
+[<c02f6bc0>] (gr3d_probe) from [<c02fe8e0>] (platform_drv_probe+0x48/0xa4)
+[<c02fe8e0>] (platform_drv_probe) from [<c02fd408>] (driver_probe_device+0x1e0/0x2a0)
+[<c02fd408>] (driver_probe_device) from [<c02fd554>] (__driver_attach+0x8c/0x90)
+[<c02fd554>] (__driver_attach) from [<c02fbb44>] (bus_for_each_dev+0x54/0x88)
+[<c02fbb44>] (bus_for_each_dev) from [<c02fcaf8>] (bus_add_driver+0xe4/0x1f0)
+[<c02fcaf8>] (bus_add_driver) from [<c02fdd20>] (driver_register+0x78/0xf4)
+[<c02fdd20>] (driver_register) from [<c092f704>] (host1x_drm_init+0x9c/0x10c)
+[<c092f704>] (host1x_drm_init) from [<c00096e4>] (do_one_initcall+0x80/0x1d0)
+[<c00096e4>] (do_one_initcall) from [<c0913db4>] (kernel_init_freeable+0x11c/0x1e8)
+[<c0913db4>] (kernel_init_freeable) from [<c0678c88>] (kernel_init+0x8/0xec)
+[<c0678c88>] (kernel_init) from [<c000f628>] (ret_from_fork+0x14/0x2c)
+---[ end trace 33a1ce386c1ae2c7 ]---
 Console: switching to colour frame buffer device 128x48
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
@@ -314,12 +366,12 @@
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
 tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
 NET: Registered protocol family 10
-mip6: Mobile IPv6
-sit: IPv6 over IPv4 tunneling driver
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
  mmcblk0: p1
+mip6: Mobile IPv6
+sit: IPv6 over IPv4 tunneling driver
 NET: Registered protocol family 17
 NET: Registered protocol family 15
 can: controller area network core (rev 20120528 abi 9)
@@ -341,13 +393,13 @@
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
 mmc1: new high speed SD card at address b368
-tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
+ mmcblk1: p1 p2
+tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
- mmcblk1: p1 p2
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-26 20:24:15 UTC (1419625455)
+rtc-em3027 2-0056: setting system clock to 2014-12-26 20:24:52 UTC (1419625492)
 ALSA device list:
   #0: tegra-trimslice
 EXT2-fs (mmcblk0p1): error: couldn't mount because of unsupported optional features (4)
@@ -356,7 +408,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 312K (c0912000 - c0960000)
+Freeing unused kernel memory: 316K (c0913000 - c0962000)
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-08-12 07:21:58.475249016 -0700
+++ /tmp/foob	2015-08-12 07:21:58.475249016 -0700
@@ -40,18 +40,18 @@
 	 #############################################
 	 3.3 MiB/s
 done
-Bytes transferred = 6378000 (615210 hex)
+Bytes transferred = 6383368 (616708 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x615210 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x616708 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc6-next-20150810-35016-gb195df5 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Aug 10 00:17:40 PDT 2015
+Linux version 4.2.0-rc6-next-20150812-35595-gc1a0c66 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Aug 12 07:17:44 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -80,7 +80,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998928K/2096128K available (8014K kernel code, 1016K rwdata, 3604K rodata, 1008K init, 340K bss, 31664K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1998928K/2096128K available (8018K kernel code, 1016K rwdata, 3604K rodata, 1004K init, 340K bss, 31664K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,8 +88,8 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d61b1c   (11623 kB)
-      .init : 0xc0d62000 - 0xc0e5e000   (1008 kB)
+      .text : 0xc0208000 - 0xc0d62b54   (11627 kB)
+      .init : 0xc0d63000 - 0xc0e5e000   (1004 kB)
       .data : 0xc0e5e000 - 0xc0f5c180   (1017 kB)
        .bss : 0xc0f5f000 - 0xc0fb4268   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
@@ -289,10 +289,10 @@
 mmcblk0: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk0boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk0boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
+tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 mmcblk0rpmb: mmc1:0001 SEM16G partition 3 128 KiB
 Primary GPT is invalid, using alternate GPT.
  mmcblk0: p1 p2 p3 p4 p5
-tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -363,7 +363,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1008K (c0d62000 - c0e5e000)
+Freeing unused kernel memory: 1004K (c0d63000 - c0e5e000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.753798] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.748791] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2015-08-12 07:21:58.483249009 -0700
+++ /tmp/foob	2015-08-12 07:21:58.483249009 -0700
@@ -37,9 +37,9 @@
 	 #################################################################
 	 #################################################################
 	 ###################
-	 3.4 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 5039696 (4ce650 hex)
+Bytes transferred = 5041872 (4ceed0 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4ce650 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4ceed0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc6-next-20150810-35016-gb195df5 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Aug 10 00:18:05 PDT 2015
+Linux version 4.2.0-rc6-next-20150812-35595-gc1a0c66 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Aug 12 07:18:08 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c09b47c0, node_mem_map ee7f7000
+free_area_init_node: node 0, pgdat c09b67c0, node_mem_map ee7f7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002636K/2096128K available (6872K kernel code, 344K rwdata, 2380K rodata, 312K init, 363K bss, 27956K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 2002628K/2096128K available (6876K kernel code, 344K rwdata, 2380K rodata, 316K init, 363K bss, 27964K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0911660   (9254 kB)
-      .init : 0xc0912000 - 0xc0960000   ( 312 kB)
-      .data : 0xc0960000 - 0xc09b6340   ( 345 kB)
-       .bss : 0xc09b9000 - 0xc0a13da0   ( 364 kB)
+      .text : 0xc0008000 - 0xc0912698   (9258 kB)
+      .init : 0xc0913000 - 0xc0962000   ( 316 kB)
+      .data : 0xc0962000 - 0xc09b8340   ( 345 kB)
+       .bss : 0xc09bb000 - 0xc0a15da0   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -334,6 +334,58 @@
 [drm] No driver support for vblank timestamp query.
 drm drm: No connectors reported connected with modes
 [drm] Cannot find any crtc or sizes - going 1024x768
+------------[ cut here ]------------
+WARNING: CPU: 1 PID: 82 at include/linux/ww_mutex.h:158 drm_atomic_check_only+0x4fc/0x574()
+DEBUG_LOCKS_WARN_ON(ctx->done_acquire)
+Modules linked in:
+CPU: 1 PID: 82 Comm: kworker/u8:2 Not tainted 4.2.0-rc6-next-20150812-35595-gc1a0c66 #1
+Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
+Workqueue: deferwq deferred_probe_work_func
+[<c0016898>] (unwind_backtrace) from [<c0012124>] (show_stack+0x10/0x14)
+[<c0012124>] (show_stack) from [<c067cc68>] (dump_stack+0x84/0xd0)
+[<c067cc68>] (dump_stack) from [<c00257a8>] (warn_slowpath_common+0x74/0xb0)
+[<c00257a8>] (warn_slowpath_common) from [<c0025878>] (warn_slowpath_fmt+0x30/0x40)
+[<c0025878>] (warn_slowpath_fmt) from [<c02e3eec>] (drm_atomic_check_only+0x4fc/0x574)
+[<c02e3eec>] (drm_atomic_check_only) from [<c02e3fd8>] (drm_atomic_commit+0x10/0x64)
+[<c02e3fd8>] (drm_atomic_commit) from [<c02c2ac8>] (drm_atomic_helper_set_config+0x394/0x3ec)
+[<c02c2ac8>] (drm_atomic_helper_set_config) from [<c02d4180>] (drm_mode_set_config_internal+0x58/0xd4)
+[<c02d4180>] (drm_mode_set_config_internal) from [<c02c50d8>] (restore_fbdev_mode+0xd4/0xf4)
+[<c02c50d8>] (restore_fbdev_mode) from [<c02c6940>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
+[<c02c6940>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c02c69ac>] (drm_fb_helper_set_par+0x2c/0x54)
+[<c02c69ac>] (drm_fb_helper_set_par) from [<c02632d4>] (fbcon_init+0x474/0x4dc)
+[<c02632d4>] (fbcon_init) from [<c0299e70>] (visual_init+0xb0/0xfc)
+[<c0299e70>] (visual_init) from [<c029be14>] (do_bind_con_driver+0x134/0x354)
+[<c029be14>] (do_bind_con_driver) from [<c029c1a4>] (do_take_over_console+0x170/0x1dc)
+[<c029c1a4>] (do_take_over_console) from [<c02633a0>] (do_fbcon_takeover+0x64/0xcc)
+[<c02633a0>] (do_fbcon_takeover) from [<c003e4ac>] (notifier_call_chain+0x44/0x84)
+[<c003e4ac>] (notifier_call_chain) from [<c003e680>] (__blocking_notifier_call_chain+0x48/0x60)
+[<c003e680>] (__blocking_notifier_call_chain) from [<c003e6b0>] (blocking_notifier_call_chain+0x18/0x20)
+[<c003e6b0>] (blocking_notifier_call_chain) from [<c026d290>] (register_framebuffer+0x184/0x264)
+[<c026d290>] (register_framebuffer) from [<c02c6c18>] (drm_fb_helper_initial_config+0x244/0x3e4)
+[<c02c6c18>] (drm_fb_helper_initial_config) from [<c02e8c70>] (tegra_drm_fb_init+0x48/0xb4)
+[<c02e8c70>] (tegra_drm_fb_init) from [<c02e6f08>] (tegra_drm_load+0x170/0x224)
+[<c02e6f08>] (tegra_drm_load) from [<c02cef0c>] (drm_dev_register+0xa0/0xb4)
+[<c02cef0c>] (drm_dev_register) from [<c02e6a64>] (host1x_drm_probe+0x44/0xb0)
+[<c02e6a64>] (host1x_drm_probe) from [<c02b1e90>] (host1x_device_probe+0x1c/0x28)
+[<c02b1e90>] (host1x_device_probe) from [<c02fd408>] (driver_probe_device+0x1e0/0x2a0)
+[<c02fd408>] (driver_probe_device) from [<c02fbaa8>] (bus_for_each_drv+0x44/0x8c)
+[<c02fbaa8>] (bus_for_each_drv) from [<c02fd19c>] (__device_attach+0xa0/0x104)
+[<c02fd19c>] (__device_attach) from [<c02fc918>] (bus_probe_device+0x84/0x8c)
+[<c02fc918>] (bus_probe_device) from [<c02fae6c>] (device_add+0x33c/0x524)
+[<c02fae6c>] (device_add) from [<c02b209c>] (host1x_subdev_register+0xb0/0xd4)
+[<c02b209c>] (host1x_subdev_register) from [<c02b2954>] (host1x_client_register+0xf4/0x11c)
+[<c02b2954>] (host1x_client_register) from [<c02f016c>] (tegra_hdmi_probe+0x170/0x250)
+[<c02f016c>] (tegra_hdmi_probe) from [<c02fe8e0>] (platform_drv_probe+0x48/0xa4)
+[<c02fe8e0>] (platform_drv_probe) from [<c02fd408>] (driver_probe_device+0x1e0/0x2a0)
+[<c02fd408>] (driver_probe_device) from [<c02fbaa8>] (bus_for_each_drv+0x44/0x8c)
+[<c02fbaa8>] (bus_for_each_drv) from [<c02fd19c>] (__device_attach+0xa0/0x104)
+[<c02fd19c>] (__device_attach) from [<c02fc918>] (bus_probe_device+0x84/0x8c)
+[<c02fc918>] (bus_probe_device) from [<c02fccf0>] (deferred_probe_work_func+0x5c/0x88)
+[<c02fccf0>] (deferred_probe_work_func) from [<c00385b0>] (process_one_work+0x120/0x328)
+[<c00385b0>] (process_one_work) from [<c0038814>] (worker_thread+0x28/0x490)
+[<c0038814>] (worker_thread) from [<c003d90c>] (kthread+0xd8/0xf4)
+[<c003d90c>] (kthread) from [<c000f628>] (ret_from_fork+0x14/0x2c)
+---[ end trace babe5256f050980d ]---
 Console: switching to colour frame buffer device 128x48
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
@@ -351,7 +403,7 @@
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
-tps65910-rtc tps65910-rtc: setting system clock to 2000-01-01 00:00:02 UTC (946684802)
+tps65910-rtc tps65910-rtc: setting system clock to 2000-01-01 00:00:03 UTC (946684803)
 usb1_vbus: disabling
 vdd_sata,avdd_plle: disabling
 ALSA device list:
@@ -363,7 +415,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 312K (c0912000 - c0960000)
+Freeing unused kernel memory: 316K (c0913000 - c0962000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.416763] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.826417] usb 2-1: new high-speed USB device number 2 using tegra-ehci
