Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2015-08-05 01:55:33.819307599 -0700
+++ /tmp/foob	2015-08-05 01:55:33.823307595 -0700
@@ -1,6 +1,6 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
-Linux version 4.2.0-rc5-next-20150804+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Aug 3 22:49:00 PDT 2015
+Linux version 4.2.0-rc5-next-20150805+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Aug 5 01:52:28 PDT 2015
 CPU: AArch64 Processor [411fd070] revision 0
 Detected PIPT I-cache on CPU0
 alternatives: enabling workaround for ARM erratum 832075
@@ -88,7 +88,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 2, 16384 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.410:1): initialized
+audit: type=2000 audit(0.430:1): initialized
 HugeTLB registered 2 MB page size, pre-allocated 0 pages
 VFS: Disk quotas dquot_6.6.0
 VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-08-05 01:55:33.827307593 -0700
+++ /tmp/foob	2015-08-05 01:55:33.831307590 -0700
@@ -37,21 +37,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ############################################
+	 #############################################
 	 3.3 MiB/s
 done
-Bytes transferred = 6369984 (6132c0 hex)
+Bytes transferred = 6371336 (613808 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.8 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6132c0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x613808 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc5-next-20150804-33837-gd7f6af1 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Aug 3 22:47:44 PDT 2015
+Linux version 4.2.0-rc5-next-20150805-34012-ga5b4661 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Aug 5 01:51:10 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -88,7 +88,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d5c8c0   (11603 kB)
+      .text : 0xc0208000 - 0xc0d5c8f8   (11603 kB)
       .init : 0xc0d5d000 - 0xc0e58000   (1004 kB)
       .data : 0xc0e58000 - 0xc0f56040   (1017 kB)
        .bss : 0xc0f59000 - 0xc0fae268   ( 341 kB)
@@ -296,7 +296,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-14 14:40:53 UTC (950539253)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-15 17:44:18 UTC (950636658)
 vdd_cam_1v8_reg: disabling
 usb1_vbus: disabling
 dvdd_ts: disabling
@@ -308,4 +308,4 @@
 Freeing unused kernel memory: 1004K (c0d5d000 - c0e58000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.209120] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.204128] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2015-08-05 01:55:33.839307583 -0700
+++ /tmp/foob	2015-08-05 01:55:33.839307583 -0700
@@ -36,21 +36,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ###################
+	 ##################
 	 3.2 MiB/s
 done
-Bytes transferred = 5035776 (4cd700 hex)
+Bytes transferred = 5035184 (4cd4b0 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3.3 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4cd700 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4cd4b0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc5-next-20150804-33837-gd7f6af1 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Aug 3 22:48:17 PDT 2015
+Linux version 4.2.0-rc5-next-20150805-34012-ga5b4661 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Aug 5 01:51:41 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c09b4700, node_mem_map ee7f7000
+free_area_init_node: node 0, pgdat c09b2700, node_mem_map ee7f7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002716K/2096128K available (6872K kernel code, 344K rwdata, 2376K rodata, 316K init, 363K bss, 27876K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 2002724K/2096128K available (6868K kernel code, 344K rwdata, 2376K rodata, 312K init, 363K bss, 27868K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09105b8   (9250 kB)
-      .init : 0xc0911000 - 0xc0960000   ( 316 kB)
-      .data : 0xc0960000 - 0xc09b62c0   ( 345 kB)
-       .bss : 0xc09b9000 - 0xc0a13da0   ( 364 kB)
+      .text : 0xc0008000 - 0xc090f58c   (9246 kB)
+      .init : 0xc0910000 - 0xc095e000   ( 312 kB)
+      .data : 0xc095e000 - 0xc09b42c0   ( 345 kB)
+       .bss : 0xc09b7000 - 0xc0a11da0   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -292,7 +292,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-14 14:41:25 UTC (950539285)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-15 17:44:56 UTC (950636696)
 vdd_cam_1v8_reg: disabling
 usb1_vbus: disabling
 dvdd_ts: disabling
@@ -304,7 +304,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 316K (c0911000 - c0960000)
+Freeing unused kernel memory: 312K (c0910000 - c095e000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.729927] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.749949] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-08-05 01:55:33.847307578 -0700
+++ /tmp/foob	2015-08-05 01:55:33.847307578 -0700
@@ -39,21 +39,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ############################################
+	 #############################################
 	 3.2 MiB/s
 done
-Bytes transferred = 6369984 (6132c0 hex)
+Bytes transferred = 6371336 (613808 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *#####
-	 3.4 MiB/s
+	 2.4 MiB/s
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6132c0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x613808 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012a89
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc5-next-20150804-33837-gd7f6af1 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Aug 3 22:47:44 PDT 2015
+Linux version 4.2.0-rc5-next-20150805-34012-ga5b4661 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Aug 5 01:51:10 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -90,7 +90,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d5c8c0   (11603 kB)
+      .text : 0xc0208000 - 0xc0d5c8f8   (11603 kB)
       .init : 0xc0d5d000 - 0xc0e58000   (1004 kB)
       .data : 0xc0e58000 - 0xc0f56040   (1017 kB)
        .bss : 0xc0f59000 - 0xc0fae268   ( 341 kB)
@@ -291,7 +291,6 @@
 +1.05V_RUN_AVDD: supplied by +1.35V_LP0(sd2)
 mmc1: MAN_BKOPS_EN bit is not set
 tegra-pcie 1003000.pcie-controller: probing port 0, using 2 lanes
-tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
 mmc1: new high speed MMC card at address 0001
 mmcblk0: mmc1:0001 SEM16G 14.6 GiB 
 mmcblk0boot0: mmc1:0001 SEM16G partition 1 4.00 MiB
@@ -299,6 +298,7 @@
 mmcblk0rpmb: mmc1:0001 SEM16G partition 3 4.00 MiB
 Primary GPT is invalid, using alternate GPT.
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9
+tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
 tegra-pcie 1003000.pcie-controller: link 0 down, retrying
 tegra-pcie 1003000.pcie-controller: link 0 down, retrying
 tegra-pcie 1003000.pcie-controller: link 0 down, retrying
@@ -355,9 +355,9 @@
 ata1: SATA link down (SStatus 0 SControl 300)
 EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
 EXT4-fs (mmcblk0p1): couldn't mount as ext2 due to feature incompatibilities
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 1004K (c0d5d000 - c0e58000)
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2015-08-05 01:55:33.855307571 -0700
+++ /tmp/foob	2015-08-05 01:55:33.855307571 -0700
@@ -38,21 +38,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ###################
+	 ##################
 	 3.1 MiB/s
 done
-Bytes transferred = 5035776 (4cd700 hex)
+Bytes transferred = 5035184 (4cd4b0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *#####
-	 3.1 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4cd700 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4cd4b0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012a89
@@ -63,7 +63,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc5-next-20150804-33837-gd7f6af1 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Aug 3 22:48:17 PDT 2015
+Linux version 4.2.0-rc5-next-20150805-34012-ga5b4661 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Aug 5 01:51:41 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -71,7 +71,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c09b4700, node_mem_map ee7f7000
+free_area_init_node: node 0, pgdat c09b2700, node_mem_map ee7f7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -84,7 +84,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002608K/2096128K available (6872K kernel code, 344K rwdata, 2376K rodata, 316K init, 363K bss, 27984K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 2002616K/2096128K available (6868K kernel code, 344K rwdata, 2376K rodata, 312K init, 363K bss, 27976K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,10 +92,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09105b8   (9250 kB)
-      .init : 0xc0911000 - 0xc0960000   ( 316 kB)
-      .data : 0xc0960000 - 0xc09b62c0   ( 345 kB)
-       .bss : 0xc09b9000 - 0xc0a13da0   ( 364 kB)
+      .text : 0xc0008000 - 0xc090f58c   (9246 kB)
+      .init : 0xc0910000 - 0xc095e000   ( 312 kB)
+      .data : 0xc095e000 - 0xc09b42c0   ( 345 kB)
+       .bss : 0xc09b7000 - 0xc0a11da0   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -340,7 +340,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-02-01 00:00:19 UTC (949363219)
+as3722-rtc as3722-rtc: setting system clock to 2000-02-01 00:00:54 UTC (949363254)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_VPP_FUSE: disabling
 +2.8V_RUN_CAM_AF: disabling
@@ -352,15 +352,15 @@
 ALSA device list:
   #0: tegra-hda at 0x70038000 irq 342
   #1: NVIDIA Tegra Jetson TK1
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 ata1: SATA link down (SStatus 0 SControl 300)
 EXT2-fs (mmcblk0p1): error: couldn't mount because of unsupported optional features (244)
 EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 316K (c0911000 - c0960000)
+Freeing unused kernel memory: 312K (c0910000 - c095e000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-08-05 01:55:33.863307565 -0700
+++ /tmp/foob	2015-08-05 01:55:33.863307565 -0700
@@ -45,10 +45,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ############################################
+	 #############################################
 	 4.1 MiB/s
 done
-Bytes transferred = 6369984 (6132c0 hex)
+Bytes transferred = 6371336 (613808 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -59,7 +59,7 @@
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x6132c0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x613808 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -67,7 +67,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc5-next-20150804-33837-gd7f6af1 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Aug 3 22:47:44 PDT 2015
+Linux version 4.2.0-rc5-next-20150805-34012-ga5b4661 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Aug 5 01:51:10 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -96,7 +96,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d5c8c0   (11603 kB)
+      .text : 0xc0208000 - 0xc0d5c8f8   (11603 kB)
       .init : 0xc0d5d000 - 0xc0e58000   (1004 kB)
       .data : 0xc0e58000 - 0xc0f56040   (1017 kB)
        .bss : 0xc0f59000 - 0xc0fae268   ( 341 kB)
@@ -188,7 +188,6 @@
 pci_bus 0000:00: root bus resource [bus 00-ff]
 pci_bus 0000:00: root bus resource [io  0x1000-0xffff]
 pci 0000:00:01.0: [10de:0bf0] type 01 class 0x060000
-pci 0000:00:01.0: IOMMU is currently not supported for PCI
 pci 0000:00:01.0: PME# supported from D0 D1 D2 D3hot D3cold
 PCI: bus0: Fast back to back transfers disabled
 pci 0000:00:01.0: bridge configuration invalid ([bus 00-00]), reconfiguring
@@ -197,7 +196,6 @@
 pci 0000:01:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit pref]
 pci 0000:01:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
 pci 0000:01:00.0: reg 0x30: [mem 0x00000000-0x0001ffff pref]
-pci 0000:01:00.0: IOMMU is currently not supported for PCI
 pci 0000:01:00.0: supports D1 D2
 pci 0000:01:00.0: PME# supported from D0 D1 D2 D3hot D3cold
 PCI: bus1: Fast back to back transfers disabled
@@ -323,28 +321,28 @@
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
-tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
-mmc0: host does not support reading read-only switch, assuming write-enable
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
-mmc0: new high speed SDHC card at address b368
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
+tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
- mmcblk0: p1
+mmc0: host does not support reading read-only switch, assuming write-enable
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
+mmc0: new high speed SDHC card at address b368
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-26 20:18:25 UTC (1419625105)
+rtc-em3027 2-0056: setting system clock to 2014-12-26 20:19:02 UTC (1419625142)
+ mmcblk0: p1
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
-mmc1: new high speed SD card at address b368
+hub 1-1:1.0: USB hub found
+hub 1-1:1.0: 4 ports detected
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 1004K (c0d5d000 - c0e58000)
-hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
+mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.427065] usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
+root@(none):/# [    2.417113] usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2015-08-05 01:55:33.867307562 -0700
+++ /tmp/foob	2015-08-05 01:55:33.871307560 -0700
@@ -44,10 +44,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ###################
+	 ##################
 	 4.1 MiB/s
 done
-Bytes transferred = 5035776 (4cd700 hex)
+Bytes transferred = 5035184 (4cd4b0 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x4cd700 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x4cd4b0 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -69,7 +69,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc5-next-20150804-33837-gd7f6af1 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Aug 3 22:48:17 PDT 2015
+Linux version 4.2.0-rc5-next-20150805-34012-ga5b4661 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Aug 5 01:51:41 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -78,7 +78,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c09b4700, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c09b2700, node_mem_map eeff7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -91,7 +91,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 963524K/1048576K available (6872K kernel code, 344K rwdata, 2376K rodata, 316K init, 363K bss, 19516K reserved, 65536K cma-reserved, 204800K highmem)
+Memory: 963532K/1048576K available (6868K kernel code, 344K rwdata, 2376K rodata, 312K init, 363K bss, 19508K reserved, 65536K cma-reserved, 204800K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -99,10 +99,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09105b8   (9250 kB)
-      .init : 0xc0911000 - 0xc0960000   ( 316 kB)
-      .data : 0xc0960000 - 0xc09b62c0   ( 345 kB)
-       .bss : 0xc09b9000 - 0xc0a13da0   ( 364 kB)
+      .text : 0xc0008000 - 0xc090f58c   (9246 kB)
+      .init : 0xc0910000 - 0xc095e000   ( 312 kB)
+      .data : 0xc095e000 - 0xc09b42c0   ( 345 kB)
+       .bss : 0xc09b7000 - 0xc0a11da0   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
@@ -191,7 +191,6 @@
 pci_bus 0000:00: root bus resource [bus 00-ff]
 pci_bus 0000:00: root bus resource [io  0x1000-0xffff]
 pci 0000:00:01.0: [10de:0bf0] type 01 class 0x060000
-pci 0000:00:01.0: IOMMU is currently not supported for PCI
 pci 0000:00:01.0: PME# supported from D0 D1 D2 D3hot D3cold
 PCI: bus0: Fast back to back transfers disabled
 pci 0000:00:01.0: bridge configuration invalid ([bus 00-00]), reconfiguring
@@ -200,7 +199,6 @@
 pci 0000:01:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit pref]
 pci 0000:01:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
 pci 0000:01:00.0: reg 0x30: [mem 0x00000000-0x0001ffff pref]
-pci 0000:01:00.0: IOMMU is currently not supported for PCI
 pci 0000:01:00.0: supports D1 D2
 pci 0000:01:00.0: PME# supported from D0 D1 D2 D3hot D3cold
 PCI: bus1: Fast back to back transfers disabled
@@ -311,15 +309,13 @@
 tlv320aic23-codec 2-001a: Control not supported for path MICIN -> [NULL] -> Mic Input
 tlv320aic23-codec 2-001a: ASoC: no dapm match for MICIN --> NULL --> Mic Input
 tlv320aic23-codec 2-001a: ASoC: Failed to add route MICIN -> NULL -> Mic Input
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
 tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 NET: Registered protocol family 10
 mip6: Mobile IPv6
-mmc0: host does not support reading read-only switch, assuming write-enable
 sit: IPv6 over IPv4 tunneling driver
-mmc0: new high speed SDHC card at address b368
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
 NET: Registered protocol family 17
+mmc0: host does not support reading read-only switch, assuming write-enable
 NET: Registered protocol family 15
 can: controller area network core (rev 20120528 abi 9)
 NET: Registered protocol family 29
@@ -337,16 +333,18 @@
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
+mmc0: new high speed SDHC card at address b368
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
  mmcblk0: p1
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-26 20:18:44 UTC (1419625124)
+rtc-em3027 2-0056: setting system clock to 2014-12-26 20:19:21 UTC (1419625161)
 ALSA device list:
   #0: tegra-trimslice
+hub 1-1:1.0: 4 ports detected
 EXT2-fs (mmcblk0p1): error: couldn't mount because of unsupported optional features (4)
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 mmc1: new high speed SD card at address b368
@@ -356,7 +354,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 316K (c0911000 - c0960000)
+Freeing unused kernel memory: 312K (c0910000 - c095e000)
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-08-05 01:55:33.875307557 -0700
+++ /tmp/foob	2015-08-05 01:55:33.879307553 -0700
@@ -37,10 +37,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ############################################
-	 3.3 MiB/s
+	 #############################################
+	 3.4 MiB/s
 done
-Bytes transferred = 6369984 (6132c0 hex)
+Bytes transferred = 6371336 (613808 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -51,7 +51,7 @@
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6132c0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x613808 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc5-next-20150804-33837-gd7f6af1 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Aug 3 22:47:44 PDT 2015
+Linux version 4.2.0-rc5-next-20150805-34012-ga5b4661 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Aug 5 01:51:10 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -88,7 +88,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d5c8c0   (11603 kB)
+      .text : 0xc0208000 - 0xc0d5c8f8   (11603 kB)
       .init : 0xc0d5d000 - 0xc0e58000   (1004 kB)
       .data : 0xc0e58000 - 0xc0f56040   (1017 kB)
        .bss : 0xc0f59000 - 0xc0fae268   ( 341 kB)
@@ -290,9 +290,9 @@
 mmcblk0boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk0boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk0rpmb: mmc1:0001 SEM16G partition 3 128 KiB
+tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 Primary GPT is invalid, using alternate GPT.
  mmcblk0: p1 p2 p3 p4 p5
-tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -303,7 +303,6 @@
 pci_bus 0000:00: root bus resource [bus 00-ff]
 pci_bus 0000:00: root bus resource [io  0x1000-0xffff]
 pci 0000:00:01.0: [10de:0e1c] type 01 class 0x060400
-pci 0000:00:01.0: IOMMU is currently not supported for PCI
 pci 0000:00:01.0: PME# supported from D0 D1 D2 D3hot D3cold
 PCI: bus0: Fast back to back transfers disabled
 pci 0000:00:01.0: bridge configuration invalid ([bus 00-00]), reconfiguring
@@ -311,7 +310,6 @@
 pci 0000:01:00.0: reg 0x10: [io  0x0000-0x00ff]
 pci 0000:01:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit pref]
 pci 0000:01:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
-pci 0000:01:00.0: IOMMU is currently not supported for PCI
 pci 0000:01:00.0: supports D1 D2
 pci 0000:01:00.0: PME# supported from D0 D1 D2 D3hot D3cold
 PCI: bus1: Fast back to back transfers disabled
@@ -366,4 +364,4 @@
 Freeing unused kernel memory: 1004K (c0d5d000 - c0e58000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.748392] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.738817] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2015-08-05 01:55:33.883307550 -0700
+++ /tmp/foob	2015-08-05 01:55:33.887307548 -0700
@@ -36,10 +36,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ###################
-	 3.2 MiB/s
+	 ##################
+	 3.3 MiB/s
 done
-Bytes transferred = 5035776 (4cd700 hex)
+Bytes transferred = 5035184 (4cd4b0 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4cd700 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4cd4b0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc5-next-20150804-33837-gd7f6af1 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Aug 3 22:48:17 PDT 2015
+Linux version 4.2.0-rc5-next-20150805-34012-ga5b4661 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Aug 5 01:51:41 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c09b4700, node_mem_map ee7f7000
+free_area_init_node: node 0, pgdat c09b2700, node_mem_map ee7f7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002636K/2096128K available (6872K kernel code, 344K rwdata, 2376K rodata, 316K init, 363K bss, 27956K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 2002644K/2096128K available (6868K kernel code, 344K rwdata, 2376K rodata, 312K init, 363K bss, 27948K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09105b8   (9250 kB)
-      .init : 0xc0911000 - 0xc0960000   ( 316 kB)
-      .data : 0xc0960000 - 0xc09b62c0   ( 345 kB)
-       .bss : 0xc09b9000 - 0xc0a13da0   ( 364 kB)
+      .text : 0xc0008000 - 0xc090f58c   (9246 kB)
+      .init : 0xc0910000 - 0xc095e000   ( 312 kB)
+      .data : 0xc095e000 - 0xc09b42c0   ( 345 kB)
+       .bss : 0xc09b7000 - 0xc0a11da0   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -300,7 +300,6 @@
 pci_bus 0000:00: root bus resource [bus 00-ff]
 pci_bus 0000:00: root bus resource [io  0x1000-0xffff]
 pci 0000:00:01.0: [10de:0e1c] type 01 class 0x060400
-pci 0000:00:01.0: IOMMU is currently not supported for PCI
 pci 0000:00:01.0: PME# supported from D0 D1 D2 D3hot D3cold
 PCI: bus0: Fast back to back transfers disabled
 pci 0000:00:01.0: bridge configuration invalid ([bus 00-00]), reconfiguring
@@ -308,7 +307,6 @@
 pci 0000:01:00.0: reg 0x10: [io  0x0000-0x00ff]
 pci 0000:01:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit pref]
 pci 0000:01:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
-pci 0000:01:00.0: IOMMU is currently not supported for PCI
 pci 0000:01:00.0: supports D1 D2
 pci 0000:01:00.0: PME# supported from D0 D1 D2 D3hot D3cold
 PCI: bus1: Fast back to back transfers disabled
@@ -363,7 +361,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 316K (c0911000 - c0960000)
+Freeing unused kernel memory: 312K (c0910000 - c095e000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.416749] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.386445] usb 2-1: new high-speed USB device number 2 using tegra-ehci
