Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2018-02-14 08:18:09.218903426 -0800
+++ /tmp/foob	2018-02-14 08:18:09.222903442 -0800
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0000000000 [0x411fd070]
-Linux version 4.16.0-rc1-next-20180212-349697-gc7434d8b9ebe (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sun Feb 11 19:23:19 PST 2018
+Linux version 4.16.0-rc1-next-20180213-350261-g1e4f57256a18 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Feb 14 08:08:59 PST 2018
 Machine model: linux,dummy-virt
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -65,9 +65,9 @@
 DMI not present or invalid.
 NET: Registered protocol family 16
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.200:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.208:1): state=initialized audit_enabled=0 res=1
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ 0000000004bb44e6, 1 data @ 0000000081aa141a)
+vdso: 2 pages (1 code @ 00000000ca403aac, 1 data @ 0000000018e6f039)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -176,7 +176,7 @@
 registered taskstats version 1
 Loading compiled-in X.509 certificates
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2018-02-12 03:30:30 UTC (1518406230)
+rtc-pl031 9010000.pl031: setting system clock to 2018-02-14 16:16:38 UTC (1518624998)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2018-02-14 08:18:09.294903745 -0800
+++ /tmp/foob	2018-02-14 08:18:09.298903762 -0800
@@ -49,7 +49,7 @@
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.5 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 34502 (86c6 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.16.0-rc1-next-20180212-349697-gc7434d8b9ebe (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Sun Feb 11 19:17:36 PST 2018
+Linux version 4.16.0-rc1-next-20180213-350261-g1e4f57256a18 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Feb 14 08:02:59 PST 2018
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -84,7 +84,7 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=7369c667-ff51-ec4a-29cd-baabf2fbe346 rootwait gpt debug earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1992580K/2096128K available (11264K kernel code, 1570K rwdata, 4352K rodata, 2048K init, 405K bss, 38012K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1992580K/2096128K available (11264K kernel code, 1569K rwdata, 4352K rodata, 2048K init, 405K bss, 38012K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -94,7 +94,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0x(ptrval) - 0x(ptrval)   (12256 kB)
       .init : 0x(ptrval) - 0x(ptrval)   (2048 kB)
-      .data : 0x(ptrval) - 0x(ptrval)   (1571 kB)
+      .data : 0x(ptrval) - 0x(ptrval)   (1570 kB)
        .bss : 0x(ptrval) - 0x(ptrval)   ( 406 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -324,10 +324,10 @@
 usb1_vbus: supplied by vdd-sys-5v0
 usb2_vbus: supplied by vdd-sys-5v0
 VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
- mmcblk0: p1 p2 p3
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
-mmc1: new high speed MMC card at address 0001
+ mmcblk0: p1 p2 p3
 [drm] No driver support for vblank timestamp query.
+mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
@@ -353,7 +353,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-07-05 10:24:20 UTC (962792660)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-07-07 23:08:07 UTC (963011287)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -365,4 +365,4 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.719200] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.759310] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2018-02-14 08:18:09.346903964 -0800
+++ /tmp/foob	2018-02-14 08:18:09.358904015 -0800
@@ -40,18 +40,18 @@
 	 ##################################
 	 2.9 MiB/s
 done
-Bytes transferred = 6222768 (5ef3b0 hex)
+Bytes transferred = 6223320 (5ef5d8 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.3 MiB/s
+	 2.5 MiB/s
 done
 Bytes transferred = 34502 (86c6 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5ef3b0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5ef5d8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff6c5 ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.16.0-rc1-next-20180212-29748-gc7434d8 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Sun Feb 11 19:16:48 PST 2018
+Linux version 4.16.0-rc1-next-20180213-30312-g1e4f572 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Feb 14 08:01:14 PST 2018
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -296,6 +296,7 @@
  mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
  mmcblk0: p1 p2 p3
 tegra30-i2s 70080400.i2s: DMA channels sourced from device 70080000.ahub
+tegra30-i2s 70080400.i2s: ASoC: Failed to create component debugfs directory
 tegra-snd-rt5640 sound: rt5640-aif1 <-> 70080400.i2s mapping ok
 input: NVIDIA Tegra Dalmore Headphones as /devices/soc0/sound/sound/card0/input0
 NET: Registered protocol family 10
@@ -343,7 +344,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-07-05 10:20:39 UTC (962792439)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-07-07 23:05:04 UTC (963011104)
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
@@ -361,4 +362,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.366945] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.396093] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2018-02-14 08:18:09.446904384 -0800
+++ /tmp/foob	2018-02-14 08:18:09.450904401 -0800
@@ -14,7 +14,7 @@
 Out:   serial
 Err:   serial
 Net:   No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra124 (Jetson TK1) # 
 Tegra124 (Jetson TK1) # pci enum
 Tegra124 (Jetson TK1) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=b1791faa-84e8-4e27-83a1-9fb0a77e8803 rootwait gpt debug noinitrd earlyprintk init=/bin/bash'
@@ -35,7 +35,7 @@
 	 #################################################################
 	 #################################################################
 	 #####################################
-	 4 MiB/s
+	 3.9 MiB/s
 done
 Bytes transferred = 8167936 (7ca200 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
@@ -56,7 +56,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.16.0-rc1-next-20180212-349697-gc7434d8b9ebe (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Sun Feb 11 19:17:36 PST 2018
+Linux version 4.16.0-rc1-next-20180213-350261-g1e4f57256a18 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Feb 14 08:02:59 PST 2018
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -79,7 +79,7 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=b1791faa-84e8-4e27-83a1-9fb0a77e8803 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1992472K/2096128K available (11264K kernel code, 1570K rwdata, 4352K rodata, 2048K init, 405K bss, 38120K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1992472K/2096128K available (11264K kernel code, 1569K rwdata, 4352K rodata, 2048K init, 405K bss, 38120K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -89,7 +89,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0x(ptrval) - 0x(ptrval)   (12256 kB)
       .init : 0x(ptrval) - 0x(ptrval)   (2048 kB)
-      .data : 0x(ptrval) - 0x(ptrval)   (1571 kB)
+      .data : 0x(ptrval) - 0x(ptrval)   (1570 kB)
        .bss : 0x(ptrval) - 0x(ptrval)   ( 406 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -322,27 +322,27 @@
 +USB0_VBUS_SW: supplied by +5V_SYS
 +3.3V_AVDD_HDMI_AP_GATED: supplied by +3.3V_RUN
 +5V_USB_HS: supplied by +5V_SYS
+mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 +3.3V_LP0: supplied by +3.3V_SYS
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 +5V_HDMI_CON: supplied by +5V_SYS
 +5V_SATA: supplied by +5V_SYS
 +12V_SATA: supplied by +VDD_MUX
  usb2-0: usb2-0 supply vbus not found, using dummy regulator
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB, chardev (236:0)
  usb2-1: usb2-1 supply vbus not found, using dummy regulator
+ mmcblk0: p1
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
-mmc0: new high speed MMC card at address 0001
 tegra-pcie 1003000.pcie: 2x1, 1x1 configuration
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 tegra-pcie 1003000.pcie: Failed to get supply 'avddio-pex': -517
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB, chardev (236:0)
 as3722-sd2: supplied by +5V_SYS
 as3722-sd3: supplied by +5V_SYS
 as3722-sd4: supplied by +5V_SYS
 as3722-sd5: supplied by +5V_SYS
- mmcblk0: p1
 as3722-ldo0: supplied by +1.35V_LP0(sd2)
 as3722-ldo1: supplied by +3.3V_RUN
 as3722-ldo2: supplied by +1.8V_VDDIO
@@ -398,8 +398,8 @@
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4 may corrupt adjacent RW1C bits
 pci 0000:00:02.0: enabling Extended Tags
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x88 may corrupt adjacent RW1C bits
-ata1: SATA link down (SStatus 0 SControl 300)
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x52 may corrupt adjacent RW1C bits
+ata1: SATA link down (SStatus 0 SControl 300)
 pci 0000:00:02.0: PME# supported from D0 D1 D2 D3hot D3cold
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4c may corrupt adjacent RW1C bits
 PCI: bus0: Fast back to back transfers disabled
@@ -443,7 +443,7 @@
 pcieport 0000:00:02.0: Signaling PME with IRQ 396
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
-r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xb28cf66e, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 397
+r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xa3c14396, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 397
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
@@ -455,7 +455,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:04:08 UTC (946685048)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:04:47 UTC (946685087)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2018-02-14 08:18:09.470904486 -0800
+++ /tmp/foob	2018-02-14 08:18:09.474904503 -0800
@@ -33,20 +33,20 @@
 	 #################################################################
 	 #################################################################
 	 ##################################
-	 3.5 MiB/s
+	 944.3 KiB/s
 done
-Bytes transferred = 6222768 (5ef3b0 hex)
+Bytes transferred = 6223320 (5ef5d8 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.5 MiB/s
+	 915 KiB/s
 done
 Bytes transferred = 66544 (103f0 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5ef3b0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5ef5d8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830133ef
@@ -54,7 +54,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.16.0-rc1-next-20180212-29748-gc7434d8 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Sun Feb 11 19:16:48 PST 2018
+Linux version 4.16.0-rc1-next-20180213-30312-g1e4f572 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Feb 14 08:01:14 PST 2018
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -292,6 +292,7 @@
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB, chardev (247:0)
  mmcblk0: p1
 tegra30-i2s 70301100.i2s: DMA channels sourced from device 70300000.ahub
+tegra30-i2s 70301100.i2s: ASoC: Failed to create component debugfs directory
 tegra-snd-rt5640 sound: rt5640-aif1 <-> 70301100.i2s mapping ok
 input: NVIDIA Tegra Jetson TK1 Headphones as /devices/soc0/sound/sound/card1/input0
 NET: Registered protocol family 10
@@ -338,10 +339,10 @@
 as3722-ldo2: supplied by +1.8V_VDDIO
 as3722-ldo3: supplied by +3.3V_SYS
 as3722-ldo4: supplied by +3.3V_SYS
+input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input1
 as3722-ldo5: supplied by +1.8V_VDDIO
 as3722-ldo6: supplied by +3.3V_RUN
 as3722-ldo7: supplied by +1.8V_VDDIO
-input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input1
 +1.05V_RUN_CAM_REAR: Bringing 0uV into 1050000-1050000uV
 as3722-ldo9: supplied by +5V_SYS
 +3.3V_RUN_TOUCH: Bringing 0uV into 2800000-2800000uV
@@ -438,7 +439,7 @@
 pcieport 0000:00:02.0: Signaling PME with IRQ 396
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
-r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0x62887841, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 397
+r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xb8da2619, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 397
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
@@ -450,7 +451,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:01:58 UTC (946684918)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:18 UTC (946684818)
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig%tegra-fw_log.txt
--- /tmp/fooa	2018-02-14 08:18:09.530904740 -0800
+++ /tmp/foob	2018-02-14 08:18:09.530904740 -0800
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MMC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -33,20 +33,20 @@
 	 #################################################################
 	 #################################################################
 	 ##############################################
-	 942.4 KiB/s
+	 3.7 MiB/s
 done
-Bytes transferred = 6386160 (6171f0 hex)
+Bytes transferred = 6386144 (6171e0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 927.7 KiB/s
+	 3.7 MiB/s
 done
 Bytes transferred = 66544 (103f0 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6171f0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x6171e0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830133ef
@@ -54,7 +54,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.16.0-rc1-next-20180212-231456-gc7434d8 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Sun Feb 11 19:17:10 PST 2018
+Linux version 4.16.0-rc1-next-20180213-232020-g1e4f572 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Feb 14 08:02:30 PST 2018
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -292,6 +292,7 @@
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB, chardev (247:0)
  mmcblk0: p1
 tegra30-i2s 70301100.i2s: DMA channels sourced from device 70300000.ahub
+tegra30-i2s 70301100.i2s: ASoC: Failed to create component debugfs directory
 tegra-snd-rt5640 sound: rt5640-aif1 <-> 70301100.i2s mapping ok
 input: NVIDIA Tegra Jetson TK1 Headphones as /devices/soc0/sound/sound/card1/input0
 NET: Registered protocol family 10
@@ -339,8 +340,8 @@
 as3722-ldo3: supplied by +3.3V_SYS
 as3722-ldo4: supplied by +3.3V_SYS
 as3722-ldo5: supplied by +1.8V_VDDIO
-input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input1
 as3722-ldo6: supplied by +3.3V_RUN
+input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input1
 as3722-ldo7: supplied by +1.8V_VDDIO
 +1.05V_RUN_CAM_REAR: Bringing 0uV into 1050000-1050000uV
 as3722-ldo9: supplied by +5V_SYS
@@ -446,7 +447,7 @@
 pcieport 0000:00:02.0: Signaling PME with IRQ 396
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
-r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0x645241a2, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 397
+r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0x5b287500, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 397
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
@@ -458,12 +459,12 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:18 UTC (946684818)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:35 UTC (946684955)
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
-+USB0_VBUS_SW: disabling
 cfg80211: failed to load regulatory.db
++USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2018-02-14 08:18:09.558904857 -0800
+++ /tmp/foob	2018-02-14 08:18:09.562904874 -0800
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -43,7 +43,7 @@
 	 #################################################################
 	 #################################################################
 	 #####################################
-	 3.3 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 8167936 (7ca200 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
@@ -52,7 +52,7 @@
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.1 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 67482 (1079a hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
@@ -64,7 +64,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.16.0-rc1-next-20180212-349697-gc7434d8b9ebe (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Sun Feb 11 19:17:36 PST 2018
+Linux version 4.16.0-rc1-next-20180213-350261-g1e4f57256a18 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Feb 14 08:02:59 PST 2018
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -88,7 +88,7 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=4d36e82a-b3c0-4b4b-bfcd-afe5217ac023 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993488K/2097148K available (11264K kernel code, 1570K rwdata, 4352K rodata, 2048K init, 405K bss, 38124K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1993488K/2097148K available (11264K kernel code, 1569K rwdata, 4352K rodata, 2048K init, 405K bss, 38124K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -98,7 +98,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0x(ptrval) - 0x(ptrval)   (12256 kB)
       .init : 0x(ptrval) - 0x(ptrval)   (2048 kB)
-      .data : 0x(ptrval) - 0x(ptrval)   (1571 kB)
+      .data : 0x(ptrval) - 0x(ptrval)   (1570 kB)
        .bss : 0x(ptrval) - 0x(ptrval)   ( 406 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -329,18 +329,18 @@
 +VDD_LED: supplied by +VDD_MUX
 +3.3V_AVDD_HDMI_AP_GATED: supplied by +3.3V_RUN
 +1.8V_RUN_CAM: supplied by +3.3V_RUN
-+VDDIO_SDMMC3: supplied by +3.3V_RUN
 mmc0: new high speed MMC card at address 0001
-+1.8V_RUN_VPP_FUSE: supplied by +3.3V_RUN
++VDDIO_SDMMC3: supplied by +3.3V_RUN
 mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
-+5V_VDD_TS_SW: supplied by +5V_SYS
++1.8V_RUN_VPP_FUSE: supplied by +3.3V_RUN
 mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
-+5V_USB_HS: supplied by +5V_SYS
++5V_VDD_TS_SW: supplied by +5V_SYS
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
-+5V_USB_SS: supplied by +5V_SYS
++5V_USB_HS: supplied by +5V_SYS
 mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB, chardev (236:0)
-+3.3V_PANEL: supplied by +3.3V_RUN
++5V_USB_SS: supplied by +5V_SYS
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
++3.3V_PANEL: supplied by +3.3V_RUN
 +3.3V_LP0: supplied by +3.3V_SYS
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 elan_i2c 1-0015: failed to read report data: -121
@@ -357,16 +357,16 @@
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc2: Unknown controller version (3). You may experience problems.
-mmc2: Invalid maximum block size, assuming 512 bytes
 elan_i2c 1-0015: invalid report id data (ff)
+mmc2: Invalid maximum block size, assuming 512 bytes
+mmc1: new high speed SDIO card at address 0001
 mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
-mmc1: new high speed SDIO card at address 0001
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
 tegra-sor 54540000.sor: failed to probe output: -517
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:23 UTC (946684823)
 panel-simple panel: panel supply power not found, using dummy regulator
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 04:41:08 UTC (946701668)
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +1.05V_RUN: disabling
 +1.8V_RUN_CAM: disabling
@@ -387,19 +387,19 @@
 EXT4-fs (mmcblk0p3): couldn't mount as ext2 due to feature incompatibilities
 mmc2: new high speed SDHC card at address 0007
 mmcblk2: mmc2:0007 SD08G 7.42 GiB 
-EXT4-fs (mmcblk0p3): recovery complete
-EXT4-fs (mmcblk0p3): mounted filesystem with ordered data mode. Opts: (null)
 GPT:Primary header alternate_lba != Alt. header my_lba
-VFS: Mounted root (ext4 filesystem) on device 179:3.
 GPT:3014591 != 15564799
 GPT:Primary header thinks Alt. header is not at the end of the disk.
-devtmpfs: mounted
 GPT:3014591 != 15564799
 GPT: Use GNU Parted to correct GPT errors.
  mmcblk2: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
-Freeing unused kernel memory: 2048K
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
+EXT4-fs (mmcblk0p3): recovery complete
+EXT4-fs (mmcblk0p3): mounted filesystem with ordered data mode. Opts: (null)
+VFS: Mounted root (ext4 filesystem) on device 179:3.
+devtmpfs: mounted
+Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2018-02-14 08:18:09.602905042 -0800
+++ /tmp/foob	2018-02-14 08:18:09.606905058 -0800
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -41,20 +41,20 @@
 	 #################################################################
 	 #################################################################
 	 ##################################
-	 3.2 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 6222768 (5ef3b0 hex)
+Bytes transferred = 6223320 (5ef5d8 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.104
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 2.9 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 67482 (1079a hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5ef3b0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5ef5d8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013799
@@ -62,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.16.0-rc1-next-20180212-29748-gc7434d8 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Sun Feb 11 19:16:48 PST 2018
+Linux version 4.16.0-rc1-next-20180213-30312-g1e4f572 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Feb 14 08:01:14 PST 2018
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -292,18 +292,19 @@
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
 tegra30-i2s 70301100.i2s: DMA channels sourced from device 70300000.ahub
-max98090 0-0010: MAX98090 REVID=0x43
-max98090 0-0010: use default 2.8v micbias
-tegra-snd-max98090 sound: HiFi <-> 70301100.i2s mapping ok
 mmc0: new high speed MMC card at address 0001
+max98090 0-0010: MAX98090 REVID=0x43
 mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
-input: GoogleNyanBig Headphones as /devices/soc0/sound/sound/card1/input1
+max98090 0-0010: use default 2.8v micbias
 mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
-input: GoogleNyanBig Mic Jack as /devices/soc0/sound/sound/card1/input2
+tegra30-i2s 70301100.i2s: ASoC: Failed to create component debugfs directory
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
 mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB, chardev (247:0)
-NET: Registered protocol family 10
+tegra-snd-max98090 sound: HiFi <-> 70301100.i2s mapping ok
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
+input: GoogleNyanBig Headphones as /devices/soc0/sound/sound/card1/input1
+input: GoogleNyanBig Mic Jack as /devices/soc0/sound/sound/card1/input2
+NET: Registered protocol family 10
 Segment Routing with IPv6
 mip6: Mobile IPv6
 sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
@@ -360,13 +361,13 @@
 input: gpio-keys as /devices/soc0/gpio-keys/input/input3
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:23 UTC (946684823)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 04:35:38 UTC (946701338)
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
 +3.3V_AVDD_HDMI_AP_GATED: disabling
-+1.8V_RUN_CAM: disabling
 cfg80211: failed to load regulatory.db
++1.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
 +1.05V_RUN_CAM_REAR: disabling
 +2.8V_RUN_TOUCH: disabling
@@ -383,17 +384,17 @@
   #1: GoogleNyanBig
 mmc2: new high speed SDHC card at address 0007
 mmcblk2: mmc2:0007 SD08G 7.42 GiB 
+tegra-sor 54540000.sor: failed to probe output: -517
+input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input4
+panel-simple panel: panel supply power not found, using dummy regulator
 GPT:Primary header alternate_lba != Alt. header my_lba
 GPT:3014591 != 15564799
 GPT:Primary header thinks Alt. header is not at the end of the disk.
 GPT:3014591 != 15564799
-tegra-sor 54540000.sor: failed to probe output: -517
 GPT: Use GNU Parted to correct GPT errors.
-panel-simple panel: panel supply power not found, using dummy regulator
  mmcblk2: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
-input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input4
-EXT4-fs (mmcblk0p3): couldn't mount as ext3 due to feature incompatibilities
 tegra-sor 54540000.sor: failed to probe output: -517
+EXT4-fs (mmcblk0p3): couldn't mount as ext3 due to feature incompatibilities
 panel-simple panel: panel supply power not found, using dummy regulator
 EXT4-fs (mmcblk0p3): recovery complete
 EXT4-fs (mmcblk0p3): mounted filesystem with ordered data mode. Opts: (null)
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig%tegra-fw_log.txt
--- /tmp/fooa	2018-02-14 08:18:09.678905362 -0800
+++ /tmp/foob	2018-02-14 08:18:09.678905362 -0800
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -43,18 +43,18 @@
 	 ##############################################
 	 3.4 MiB/s
 done
-Bytes transferred = 6386160 (6171f0 hex)
+Bytes transferred = 6386144 (6171e0 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.104
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.2 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 67482 (1079a hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6171f0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x6171e0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013799
@@ -62,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.16.0-rc1-next-20180212-231456-gc7434d8 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Sun Feb 11 19:17:10 PST 2018
+Linux version 4.16.0-rc1-next-20180213-232020-g1e4f572 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Feb 14 08:02:30 PST 2018
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -297,11 +297,12 @@
 mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
 max98090 0-0010: use default 2.8v micbias
 mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
+tegra30-i2s 70301100.i2s: ASoC: Failed to create component debugfs directory
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
 mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB, chardev (247:0)
 tegra-snd-max98090 sound: HiFi <-> 70301100.i2s mapping ok
-input: GoogleNyanBig Headphones as /devices/soc0/sound/sound/card1/input1
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
+input: GoogleNyanBig Headphones as /devices/soc0/sound/sound/card1/input1
 input: GoogleNyanBig Mic Jack as /devices/soc0/sound/sound/card1/input2
 NET: Registered protocol family 10
 Segment Routing with IPv6
@@ -368,12 +369,12 @@
 input: gpio-keys as /devices/soc0/gpio-keys/input/input3
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:23 UTC (946684823)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 04:38:27 UTC (946701507)
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
-cfg80211: failed to load regulatory.db
 +3.3V_AVDD_HDMI_AP_GATED: disabling
+cfg80211: failed to load regulatory.db
 +1.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
 +1.05V_RUN_CAM_REAR: disabling
@@ -387,6 +388,10 @@
 +5V_HDMI_CON: disabling
 ALSA device list:
   #1: GoogleNyanBig
+tegra-sor 54540000.sor: failed to probe output: -517
+panel-simple panel: panel supply power not found, using dummy regulator
+input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input4
+EXT4-fs (mmcblk0p3): couldn't mount as ext3 due to feature incompatibilities
 usb 1-2: new high-speed USB device number 2 using tegra-xusb
 mmc2: new high speed SDHC card at address 0007
 mmcblk2: mmc2:0007 SD08G 7.42 GiB 
@@ -400,26 +405,23 @@
 panel-simple panel: panel supply power not found, using dummy regulator
 uvcvideo: Found UVC 1.00 device HD WebCam (1bcf:2c67)
 input: HD WebCam: HD WebCam as /devices/soc0/70090000.usb/usb1/1-2/1-2:1.0/input/input5
-tegra-sor 54540000.sor: failed to probe output: -517
-panel-simple panel: panel supply power not found, using dummy regulator
-usb 1-3: new high-speed USB device number 3 using tegra-xusb
-tegra-sor 54540000.sor: failed to probe output: -517
-panel-simple panel: panel supply power not found, using dummy regulator
-input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input4
-asix 1-3:1.0 eth0: register 'asix' at usb-70090000.usb-3, ASIX AX88772 USB 2.0 Ethernet, c8:d7:19:90:fa:65
-tegra-sor 54540000.sor: failed to probe output: -517
-EXT4-fs (mmcblk0p3): couldn't mount as ext3 due to feature incompatibilities
-panel-simple panel: panel supply power not found, using dummy regulator
-usb 1-1: new high-speed USB device number 4 using tegra-xusb
 EXT4-fs (mmcblk0p3): recovery complete
-usb-storage 1-1:1.0: USB Mass Storage device detected
+tegra-sor 54540000.sor: failed to probe output: -517
 EXT4-fs (mmcblk0p3): mounted filesystem with ordered data mode. Opts: (null)
+panel-simple panel: panel supply power not found, using dummy regulator
 VFS: Mounted root (ext4 filesystem) on device 179:3.
-scsi host0: usb-storage 1-1:1.0
 devtmpfs: mounted
-tegra-sor 54540000.sor: failed to probe output: -517
-panel-simple panel: panel supply power not found, using dummy regulator
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# 
\ No newline at end of file
+root@(none):/# [    2.928236] alloc_contig_range: [fbc79, fbc7a) PFNs busy
+alloc_contig_range: [fbc79, fbc7a) PFNs busy
+alloc_contig_range: [fbc7b, fbc7c) PFNs busy
+alloc_contig_range: [fbc7c, fbc7d) PFNs busy
+alloc_contig_range: [fbc7d, fbc7e) PFNs busy
+alloc_contig_range: [fbc7e, fbc7f) PFNs busy
+alloc_contig_range: [fbc7f, fbc80) PFNs busy
+alloc_contig_range: [fbca1, fbca2) PFNs busy
+alloc_contig_range: [fbca2, fbca3) PFNs busy
+alloc_contig_range: [fbca3, fbca4) PFNs busy
+usb 1-3: new high-speed USB device number 3 using tegra-xusb
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2018-02-14 08:18:09.706905480 -0800
+++ /tmp/foob	2018-02-14 08:18:09.706905480 -0800
@@ -32,7 +32,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1800907 us
+Bootloader-Cpu Init at (time stamp): 1801353 us
 
 Pinmux changes applied in kernel way
 
@@ -63,22 +63,22 @@
 Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1800881  us
-Bootloader (AosInit) time: 734252 us
-Bootloader Main Init time: 180077 us
-BlockDevMgrInit time: 157520 us
-PartMgrInit time: 9 us
+Miscellaneous avp time: 1801327  us
+Bootloader (AosInit) time: 733426 us
+Bootloader Main Init time: 180039 us
+BlockDevMgrInit time: 157566 us
+PartMgrInit time: 8 us
 StorMgrInit time: 30 us
 FsMgrInit time: 2 us
-Partition Table load: 76762 us 
-Display init time: 574701 us
-Kernel read time: 1582 msec
+Partition Table load: 77040 us 
+Display init time: 574443 us
+Kernel read time: 1583 msec
 Load Kernel Size:   20969472 Bytes
-Bl Kernel ReadSpeed: 13255000 Bytes/sec
-Pre-Kernel Jump time: 4424632 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 9625587 us
+Bl Kernel ReadSpeed: 13246000 Bytes/sec
+Pre-Kernel Jump time: 4424009 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 9627178 us
 Booting Linux on physical CPU 0x0000000000 [0x4e0f0000]
-Linux version 4.16.0-rc1-next-20180212-349697-gc7434d8b9ebe (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sun Feb 11 19:23:19 PST 2018
+Linux version 4.16.0-rc1-next-20180213-350261-g1e4f57256a18 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Feb 14 08:08:59 PST 2018
 Machine model: NVIDIA Tegra132 Norrin
 Memory limited to 1734MB
 efi: Getting EFI parameters from FDT:
@@ -142,7 +142,7 @@
 audit: initializing netlink subsys (disabled)
 audit: type=2000 audit(0.140:1): state=initialized audit_enabled=0 res=1
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ 00000000773c02d5, 1 data @ 000000004f78ab21)
+vdso: 2 pages (1 code @ 000000009e3c9fb4, 1 data @ 00000000509ebe40)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -265,8 +265,8 @@
 9pnet: Installing 9P2000 support
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 Key type dns_resolver registered
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 registered taskstats version 1
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 Loading compiled-in X.509 certificates
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB, chardev (240:0)
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
Diff for ./tegra186-p2771-0000/tegra186-p2771-0000/defconfig%jetson-tx2_log.txt
Binary files /tmp/fooa and /tmp/foob differ
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2018-02-14 08:18:09.806905901 -0800
+++ /tmp/foob	2018-02-14 08:18:09.810905917 -0800
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -48,7 +48,7 @@
 	 #################################################################
 	 #################################################################
 	 #####################################
-	 4 MiB/s
+	 3.5 MiB/s
 done
 Bytes transferred = 8167936 (7ca200 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
@@ -57,7 +57,7 @@
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 4.1 MiB/s
+	 2.6 MiB/s
 done
 Bytes transferred = 21686 (54b6 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
@@ -69,7 +69,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.16.0-rc1-next-20180212-349697-gc7434d8b9ebe (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Sun Feb 11 19:17:36 PST 2018
+Linux version 4.16.0-rc1-next-20180213-350261-g1e4f57256a18 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Feb 14 08:02:59 PST 2018
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -91,7 +91,7 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=11eb23c1-01 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 953404K/1048576K available (11264K kernel code, 1570K rwdata, 4352K rodata, 2048K init, 405K bss, 29636K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 953404K/1048576K available (11264K kernel code, 1569K rwdata, 4352K rodata, 2048K init, 405K bss, 29636K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -101,7 +101,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0x(ptrval) - 0x(ptrval)   (12256 kB)
       .init : 0x(ptrval) - 0x(ptrval)   (2048 kB)
-      .data : 0x(ptrval) - 0x(ptrval)   (1571 kB)
+      .data : 0x(ptrval) - 0x(ptrval)   (1570 kB)
        .bss : 0x(ptrval) - 0x(ptrval)   ( 406 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
@@ -270,7 +270,7 @@
 igb: Copyright (c) 2007-2014 Intel Corporation.
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:01:00.0: enabling device (0140 -> 0143)
-r8169 0000:01:00.0 eth0: RTL8168d/8111d at 0xc5866eef, 00:01:c0:13:7d:63, XID 081000c0 IRQ 299
+r8169 0000:01:00.0 eth0: RTL8168d/8111d at 0x3c96c036, 00:01:c0:13:7d:63, XID 081000c0 IRQ 299
 r8169 0000:01:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 pegasus: v0.9.3 (2013/04/25), Pegasus/Pegasus II USB Ethernet driver
 usbcore: registered new interface driver pegasus
@@ -345,13 +345,13 @@
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 291, io mem 0xc5000000
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
-tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 mmc1: new high speed SD card at address b368
-hub 2-0:1.0: USB hub found
+tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
+hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
-input: gpio-keys as /devices/soc0/gpio-keys/input/input0
  mmcblk1: p1 p2
+input: gpio-keys as /devices/soc0/gpio-keys/input/input0
 rtc-em3027 2-0056: hctosys: unable to read the hardware clock
 Waiting for root device PARTUUID=11eb23c1-01...
 hub 1-1:1.0: USB hub found
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2018-02-14 08:18:09.834906018 -0800
+++ /tmp/foob	2018-02-14 08:18:09.838906035 -0800
@@ -46,9 +46,9 @@
 	 #################################################################
 	 #################################################################
 	 ##################################
-	 4.1 MiB/s
+	 4 MiB/s
 done
-Bytes transferred = 6222768 (5ef3b0 hex)
+Bytes transferred = 6223320 (5ef5d8 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -59,7 +59,7 @@
 done
 Bytes transferred = 21686 (54b6 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x5ef3b0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x5ef5d8 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024104b5
@@ -67,7 +67,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.16.0-rc1-next-20180212-29748-gc7434d8 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Sun Feb 11 19:16:48 PST 2018
+Linux version 4.16.0-rc1-next-20180213-30312-g1e4f572 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Feb 14 08:01:14 PST 2018
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -261,7 +261,7 @@
 igb: Copyright (c) 2007-2014 Intel Corporation.
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:01:00.0: enabling device (0140 -> 0143)
-r8169 0000:01:00.0 eth0: RTL8168d/8111d at 0x46dbfa4a, 00:01:c0:13:7d:63, XID 081000c0 IRQ 299
+r8169 0000:01:00.0 eth0: RTL8168d/8111d at 0x1872f311, 00:01:c0:13:7d:63, XID 081000c0 IRQ 299
 r8169 0000:01:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 pegasus: v0.9.3 (2013/04/25), Pegasus/Pegasus II USB Ethernet driver
 usbcore: registered new interface driver pegasus
@@ -310,6 +310,7 @@
 mmc1: SDHCI controller on c8000600.sdhci [c8000600.sdhci] using ADMA
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
+tegra20-i2s 70002800.i2s: ASoC: Failed to create component debugfs directory
 tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
 NET: Registered protocol family 10
 Segment Routing with IPv6
@@ -334,8 +335,8 @@
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
-tegra-ehci c5000000.usb: EHCI Host Controller
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
+tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
  mmcblk1: p1 p2
 tegra-ehci c5000000.usb: irq 291, io mem 0xc5000000
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig%jetson-tx1_log.txt
--- /tmp/fooa	2018-02-14 08:18:09.870906170 -0800
+++ /tmp/foob	2018-02-14 08:18:09.874906187 -0800
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 753789 us
-Basic modules init took 348478 us
+Tegraboot started after 766342 us
+Basic modules init took 348698 us
 Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14368 us
-Carveout took 252661 us
-CPU initialization took 92443 us
-Total time taken by TegraBoot 707950 us
+Next stage binary read took 14366 us
+Carveout took 252662 us
+CPU initialization took 92471 us
+Total time taken by TegraBoot 708197 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.592] LPDDR4 Training: Number of tables = 10
+64b[0001.605] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,9 +117,9 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig%jetson-tx1
-17601024 bytes read in 887 ms (18.9 MiB/s)
+17601024 bytes read in 879 ms (19.1 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-44914 bytes read in 453 ms (96.7 KiB/s)
+44914 bytes read in 455 ms (95.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
 1627787 bytes read in 495 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0000000000 [0x411fd071]
-Linux version 4.16.0-rc1-next-20180212-29748-gc7434d8 (jonathanh@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sun Feb 11 19:20:36 PST 2018
+Linux version 4.16.0-rc1-next-20180213-30312-g1e4f572 (jonathanh@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Feb 14 08:05:22 PST 2018
 Machine model: NVIDIA Tegra210 P2371 (P2530/P2595) reference design
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -223,7 +223,7 @@
 audit: initializing netlink subsys (disabled)
 audit: type=2000 audit(0.288:1): state=initialized audit_enabled=0 res=1
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ 000000006f2512ff, 1 data @ 00000000b34166f8)
+vdso: 2 pages (1 code @ 0000000068fabbd4, 1 data @ 000000007aaa685d)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2018-02-14 08:18:09.918906372 -0800
+++ /tmp/foob	2018-02-14 08:18:09.918906372 -0800
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 731418 us
-Basic modules init took 348260 us
+Tegraboot started after 765653 us
+Basic modules init took 349357 us
 Sec Bootdevice Read Time = 203 ms, Read Size = 8459 KB
-Next stage binary read took 15163 us
-Carveout took 252669 us
-CPU initialization took 92428 us
-Total time taken by TegraBoot 708520 us
+Next stage binary read took 14365 us
+Carveout took 252661 us
+CPU initialization took 92438 us
+Total time taken by TegraBoot 708821 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.571] LPDDR4 Training: Number of tables = 10
+64b[0001.605] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -117,9 +117,9 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-17469952 bytes read in 863 ms (19.3 MiB/s)
+17469952 bytes read in 864 ms (19.3 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-44914 bytes read in 453 ms (96.7 KiB/s)
+44914 bytes read in 455 ms (95.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
 1627787 bytes read in 494 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0000000000 [0x411fd071]
-Linux version 4.16.0-rc1-next-20180212-349697-gc7434d8b9ebe (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sun Feb 11 19:23:19 PST 2018
+Linux version 4.16.0-rc1-next-20180213-350261-g1e4f57256a18 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Feb 14 08:08:59 PST 2018
 Machine model: NVIDIA Tegra210 P2371 (P2530/P2595) reference design
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -223,7 +223,7 @@
 audit: initializing netlink subsys (disabled)
 audit: type=2000 audit(0.288:1): state=initialized audit_enabled=0 res=1
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ 00000000453f237a, 1 data @ 0000000065f14166)
+vdso: 2 pages (1 code @ 00000000c4a4e245, 1 data @ 0000000034458be7)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
Diff for ./tegra210-p2371-2180/tegra210-p2371-2180/defconfig%jetson-tx1_log.txt
Binary files /tmp/fooa and /tmp/foob differ
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2018-02-14 08:18:10.030906842 -0800
+++ /tmp/foob	2018-02-14 08:18:10.034906859 -0800
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0000000000 [0x411fd071]
-Linux version 4.16.0-rc1-next-20180212-349697-gc7434d8b9ebe (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sun Feb 11 19:23:19 PST 2018
+Linux version 4.16.0-rc1-next-20180213-350261-g1e4f57256a18 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Feb 14 08:08:59 PST 2018
 Machine model: Google Pixel C
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
@@ -100,7 +100,7 @@
 audit: initializing netlink subsys (disabled)
 audit: type=2000 audit(0.184:1): state=initialized audit_enabled=0 res=1
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ 00000000378cbfbe, 1 data @ 0000000086679a49)
+vdso: 2 pages (1 code @ 00000000fb3af46a, 1 data @ 0000000048f1c5d0)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -249,19 +249,19 @@
 Key type dns_resolver registered
 registered taskstats version 1
 Loading compiled-in X.509 certificates
-tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
-input: gpio-keys as /devices/platform/gpio-keys/input/input0
-tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 BGND3R 29.1 GiB 
+tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 mmcblk0boot0: mmc0:0001 BGND3R partition 1 4.00 MiB
+input: gpio-keys as /devices/platform/gpio-keys/input/input0
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
-max77686-rtc max77620-rtc: setting system clock to 2001-11-23 08:52:20 UTC (1006505540)
+tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB, chardev (240:0)
-PP1200_AVDD: disabling
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
-PP1200_RCAM: disabling
+max77686-rtc max77620-rtc: setting system clock to 2001-11-25 21:38:55 UTC (1006724335)
 tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
+PP1200_AVDD: disabling
+PP1200_RCAM: disabling
 PP2800L_RCAM: disabling
 PP2800L_FCAM: disabling
 PP_LDO6: disabling
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2018-02-14 08:18:10.066906995 -0800
+++ /tmp/foob	2018-02-14 08:18:10.070907011 -0800
@@ -40,7 +40,7 @@
 	 #################################################################
 	 #################################################################
 	 #####################################
-	 2.8 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 8167936 (7ca200 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.16.0-rc1-next-20180212-349697-gc7434d8b9ebe (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Sun Feb 11 19:17:36 PST 2018
+Linux version 4.16.0-rc1-next-20180213-350261-g1e4f57256a18 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Feb 14 08:02:59 PST 2018
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -83,7 +83,7 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=edbe1840-3633-c751-7993-b0ac834f81e0 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1992508K/2096128K available (11264K kernel code, 1570K rwdata, 4352K rodata, 2048K init, 405K bss, 38084K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1992508K/2096128K available (11264K kernel code, 1569K rwdata, 4352K rodata, 2048K init, 405K bss, 38084K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -93,7 +93,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0x(ptrval) - 0x(ptrval)   (12256 kB)
       .init : 0x(ptrval) - 0x(ptrval)   (2048 kB)
-      .data : 0x(ptrval) - 0x(ptrval)   (1571 kB)
+      .data : 0x(ptrval) - 0x(ptrval)   (1570 kB)
        .bss : 0x(ptrval) - 0x(ptrval)   ( 406 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -308,27 +308,27 @@
 can: broadcast manager protocol (rev 20170425 t)
 can: netlink gateway (rev 20170425) max_hops=1
 Key type dns_resolver registered
+mmc1: new high speed MMC card at address 0001
 ThumbEE CPU extension supported.
+mmcblk1: mmc1:0001 SEM16G 14.8 GiB 
 Registering SWP/SWPB emulation handler
+mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 Loading compiled-in X.509 certificates
+mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
+mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB, chardev (236:0)
+Primary GPT is invalid, using alternate GPT.
+ mmcblk1: p1 p2 p3 p4 p5
 vdd_ddr: supplied by vdd_5v_in
 vdd_5v_sata: supplied by vdd_5v_in
 usb1_vbus: supplied by vdd_5v_in
 usb3_vbus: supplied by vdd_5v_in
-sys_3v3,vdd_3v3_alw: supplied by vdd_5v_in
 mmc0: new ultra high speed DDR50 SDHC card at address aaaa
-sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
+sys_3v3,vdd_3v3_alw: supplied by vdd_5v_in
 mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
+sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
 +VDD_5V_HDMI: supplied by sys_3v3,vdd_3v3_alw
-mmc1: new high speed MMC card at address 0001
 tegra-pcie 3000.pcie: 2x3 configuration
-mmcblk1: mmc1:0001 SEM16G 14.8 GiB 
-mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 tegra-pcie 3000.pcie: probing port 0, using 2 lanes
-mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
-mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB, chardev (236:0)
-Primary GPT is invalid, using alternate GPT.
- mmcblk1: p1 p2 p3 p4 p5
  mmcblk0: p1 p2 p3
 tegra-pcie 3000.pcie: probing port 2, using 2 lanes
 tegra-pcie 3000.pcie: link 2 down, retrying
@@ -376,7 +376,7 @@
 pcieport 0000:00:01.0: Signaling PME with IRQ 375
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:01:00.0: enabling device (0140 -> 0143)
-r8169 0000:01:00.0 eth0: RTL8168evl/8111evl at 0xcce97645, 02:04:4b:01:60:a6, XID 0c900800 IRQ 376
+r8169 0000:01:00.0 eth0: RTL8168evl/8111evl at 0x6d93495e, 02:04:4b:01:60:a6, XID 0c900800 IRQ 376
 r8169 0000:01:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
@@ -402,10 +402,10 @@
 EXT4-fs (mmcblk1p1): warning: mounting fs with errors, running e2fsck is recommended
 EXT4-fs (mmcblk1p1): recovery complete
 EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:17.
+VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.104037] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.144111] usb 2-1: new high-speed USB device number 2 using tegra-ehci
 asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0f:67:60
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2018-02-14 08:18:10.086907079 -0800
+++ /tmp/foob	2018-02-14 08:18:10.086907079 -0800
@@ -38,20 +38,20 @@
 	 #################################################################
 	 #################################################################
 	 ##################################
-	 2.9 MiB/s
+	 2.8 MiB/s
 done
-Bytes transferred = 6222768 (5ef3b0 hex)
+Bytes transferred = 6223320 (5ef5d8 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.9 MiB/s
+	 2.6 MiB/s
 done
 Bytes transferred = 51470 (c90e hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5ef3b0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5ef5d8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90d ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.16.0-rc1-next-20180212-29748-gc7434d8 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Sun Feb 11 19:16:48 PST 2018
+Linux version 4.16.0-rc1-next-20180213-30312-g1e4f572 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Feb 14 08:01:14 PST 2018
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -289,6 +289,7 @@
 mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
  mmcblk0: p1 p2 p3
 tegra30-i2s 70080400.i2s: DMA channels sourced from device 70080000.ahub
+tegra30-i2s 70080400.i2s: ASoC: Failed to create component debugfs directory
 tegra-snd-rt5640 sound: rt5640-aif1 <-> 70080400.i2s mapping ok
 input: NVIDIA Tegra Beaver Headphones as /devices/soc0/sound/sound/card0/input0
 NET: Registered protocol family 10
@@ -366,7 +367,7 @@
 pcieport 0000:00:01.0: Signaling PME with IRQ 375
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:01:00.0: enabling device (0140 -> 0143)
-r8169 0000:01:00.0 eth0: RTL8168evl/8111evl at 0x99e87962, 02:04:4b:01:60:a6, XID 0c900800 IRQ 376
+r8169 0000:01:00.0 eth0: RTL8168evl/8111evl at 0x19323637, 02:04:4b:01:60:a6, XID 0c900800 IRQ 376
 r8169 0000:01:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
@@ -403,4 +404,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.388639] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.378650] usb 2-1: new high-speed USB device number 2 using tegra-ehci
