Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-11-14 01:19:57.765315301 -0800
+++ /tmp/foob	2016-11-14 01:19:57.769315312 -0800
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc4-next-20161111-158037-g37634f0 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Nov 10 20:21:58 PST 2016
+Linux version 4.9.0-rc5-next-20161114-158265-gb60de3e (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Nov 13 23:52:27 PST 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -18,15 +18,15 @@
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
-Memory: 999368K/1048576K available (8380K kernel code, 884K rwdata, 3976K rodata, 1024K init, 283K bss, 32824K reserved, 16384K cma-reserved)
+Memory: 999344K/1048576K available (8380K kernel code, 908K rwdata, 4012K rodata, 1024K init, 283K bss, 32848K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff0000088b0000   (  8384 KB)
     .rodata : 0xffff0000088b0000 - 0xffff000008ca0000   (  4032 KB)
       .init : 0xffff000008ca0000 - 0xffff000008da0000   (  1024 KB)
-      .data : 0xffff000008da0000 - 0xffff000008e7d200   (   885 KB)
-       .bss : 0xffff000008e7d200 - 0xffff000008ec40c0   (   284 KB)
+      .data : 0xffff000008da0000 - 0xffff000008e83200   (   909 KB)
+       .bss : 0xffff000008e83200 - 0xffff000008eca140   (   284 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -101,7 +101,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.896:1): initialized
+audit: type=2000 audit(0.656:1): initialized
 workingset: timestamp_bits=46 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -168,7 +168,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2016-11-11 04:22:36 UTC (1478838156)
+rtc-pl031 9010000.pl031: setting system clock to 2016-11-14 09:19:31 UTC (1479115171)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-11-14 01:19:57.801315413 -0800
+++ /tmp/foob	2016-11-14 01:19:57.801315413 -0800
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra114 (Dalmore) # 
 Tegra114 (Dalmore) # usb reset
 (Re)start USB...
@@ -38,10 +38,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ############################################
-	 2.9 MiB/s
+	 ##############################################
+	 3.1 MiB/s
 done
-Bytes transferred = 7313280 (6f9780 hex)
+Bytes transferred = 7354592 (7038e0 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -52,7 +52,7 @@
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6f9780 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x7038e0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc4-next-20161111-133604-g37634f0 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Nov 10 20:19:07 PST 2016
+Linux version 4.9.0-rc5-next-20161114-133832-gb60de3e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Nov 13 23:49:12 PST 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -71,7 +71,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c1428f80, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1429240, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -84,7 +84,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1995084K/2096128K available (9216K kernel code, 1194K rwdata, 4100K rodata, 2048K init, 343K bss, 35508K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1994952K/2096128K available (9216K kernel code, 1195K rwdata, 4116K rodata, 2048K init, 473K bss, 35640K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -94,8 +94,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc142aa80   (1195 kB)
-       .bss : 0xc142c000 - 0xc1481dc8   ( 344 kB)
+      .data : 0xc1300000 - 0xc142ad40   (1196 kB)
+       .bss : 0xc142c000 - 0xc14a2408   ( 474 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -229,22 +229,6 @@
 Serial: 8250/16550 driver, 4 ports, IRQ sharing disabled
 console [ttyS0] disabled
 70006300.serial: ttyS0 at MMIO 0x70006300 (irq = 329, base_baud = 25500000) is a Tegra
-------------[ cut here ]------------
-WARNING: CPU: 3 PID: 0 at kernel/rcu/tree.c:380 rcu_note_context_switch+0x24/0x2c
-Modules linked in:
-CPU: 3 PID: 0 Comm: swapper/3 Not tainted 4.9.0-rc4-next-20161111-133604-g37634f0 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c030fab8>] (unwind_backtrace) from [<c030b104>] (show_stack+0x10/0x14)
-[<c030b104>] (show_stack) from [<c0592150>] (dump_stack+0x84/0xa4)
-[<c0592150>] (dump_stack) from [<c0342be0>] (__warn+0xd4/0x100)
-[<c0342be0>] (__warn) from [<c0342c2c>] (warn_slowpath_null+0x20/0x28)
-[<c0342c2c>] (warn_slowpath_null) from [<c0393b00>] (rcu_note_context_switch+0x24/0x2c)
-[<c0393b00>] (rcu_note_context_switch) from [<c0bbef5c>] (__schedule+0x64/0x500)
-[<c0bbef5c>] (__schedule) from [<c0bbf51c>] (schedule+0x38/0x9c)
-[<c0bbf51c>] (schedule) from [<c0bbf790>] (schedule_preempt_disabled+0xc/0x10)
-[<c0bbf790>] (schedule_preempt_disabled) from [<c037ad94>] (cpu_startup_entry+0x180/0x1f4)
-[<c037ad94>] (cpu_startup_entry) from [<80301c2c>] (0x80301c2c)
----[ end trace 7d73673a641afab6 ]---
 console [ttyS0] enabled
 SuperH (H)SCI(F) driver initialized
 msm_serial: driver initialized
@@ -309,17 +293,17 @@
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
 NET: Registered protocol family 10
-mmc0: new high speed SDHC card at address aaaa
-mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 IPv6: Attempt to unregister permanent protocol 6
 IPv6: Attempt to unregister permanent protocol 136
 IPv6: Attempt to unregister permanent protocol 17
+mmc0: new high speed SDHC card at address aaaa
+mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
+ mmcblk0: p1 p2 p3
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
- mmcblk0: p1 p2 p3
  mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
 NET: Unregistered protocol family 10
 sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
@@ -347,7 +331,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-09-04 15:49:21 UTC (968082561)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-09-07 19:19:25 UTC (968354365)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -359,4 +343,4 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.700325] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.645243] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-11-14 01:19:57.817315465 -0800
+++ /tmp/foob	2016-11-14 01:19:57.821315477 -0800
@@ -37,9 +37,9 @@
 	 #################################################################
 	 #################################################################
 	 #################################################
-	 3.2 MiB/s
+	 3.1 MiB/s
 done
-Bytes transferred = 5480248 (539f38 hex)
+Bytes transferred = 5480000 (539e40 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x539f38 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x539e40 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc4-next-20161111-133604-g37634f0 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Nov 10 20:18:45 PST 2016
+Linux version 4.9.0-rc5-next-20161114-133832-gb60de3e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Nov 13 23:49:00 PST 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0d5e080, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0d5e1c0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999960K/2096128K available (8192K kernel code, 381K rwdata, 2468K rodata, 1024K init, 370K bss, 30632K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1999960K/2096128K available (8192K kernel code, 382K rwdata, 2468K rodata, 1024K init, 370K bss, 30632K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,7 +92,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d5f740   ( 382 kB)
+      .data : 0xc0d00000 - 0xc0d5f880   ( 383 kB)
        .bss : 0xc0d61000 - 0xc0dbdb1c   ( 371 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -133,7 +133,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?[    0.530160] hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
+No ATAGs?[    0.529471] hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 8 bytes.
 tegra-mc 70019000.memory-controller: no memory timings for RAM code 3 registered
 dvdd_ts: Failed to request enable GPIO61: -517
@@ -223,22 +223,6 @@
 Serial: 8250/16550 driver, 4 ports, IRQ sharing disabled
 console [ttyS0] disabled
 70006300.serial: ttyS0 at MMIO 0x70006300 (irq = 329, base_baud = 25500000) is a Tegra
-------------[ cut here ]------------
-WARNING: CPU: 2 PID: 0 at kernel/rcu/tree.c:380 rcu_note_context_switch+0x268/0x2c8
-Modules linked in:
-CPU: 2 PID: 0 Comm: swapper/2 Not tainted 4.9.0-rc4-next-20161111-133604-g37634f0 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c010f288>] (unwind_backtrace) from [<c010a86c>] (show_stack+0x10/0x14)
-[<c010a86c>] (show_stack) from [<c034677c>] (dump_stack+0x84/0xa4)
-[<c034677c>] (dump_stack) from [<c011f754>] (__warn+0xd4/0x100)
-[<c011f754>] (__warn) from [<c011f7a0>] (warn_slowpath_null+0x20/0x28)
-[<c011f7a0>] (warn_slowpath_null) from [<c0177a48>] (rcu_note_context_switch+0x268/0x2c8)
-[<c0177a48>] (rcu_note_context_switch) from [<c081d9f4>] (__schedule+0x70/0x60c)
-[<c081d9f4>] (__schedule) from [<c081e160>] (schedule+0x50/0xc0)
-[<c081e160>] (schedule) from [<c081e3bc>] (schedule_preempt_disabled+0x24/0x40)
-[<c081e3bc>] (schedule_preempt_disabled) from [<c015ab38>] (cpu_startup_entry+0x180/0x1f4)
-[<c015ab38>] (cpu_startup_entry) from [<801014cc>] (0x801014cc)
----[ end trace ce9171bc15acad38 ]---
 console [ttyS0] enabled
 console [ttyS0] enabled
 bootconsole [earlycon0] disabled
@@ -293,10 +277,10 @@
 usbhid: USB HID core driver
 mmc0: new high speed SDHC card at address aaaa
 mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
- mmcblk0: p1 p2 p3
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
+ mmcblk0: p1 p2 p3
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
  mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
@@ -344,7 +328,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-09-04 15:46:22 UTC (968082382)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-09-07 19:16:41 UTC (968354201)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -358,5 +342,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    5.120802] usb 1-1: new high-speed USB device number 2 using tegra-ehci
-asix 1-1:
\ No newline at end of file
+root@(none):/# [    3.269687] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-11-14 01:19:57.861315605 -0800
+++ /tmp/foob	2016-11-14 01:19:57.865315617 -0800
@@ -33,21 +33,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ############################################
-	 3.4 MiB/s
+	 ##############################################
+	 3.2 MiB/s
 done
-Bytes transferred = 7313280 (6f9780 hex)
+Bytes transferred = 7354592 (7038e0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.4 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 67133 (1063d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6f9780 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x7038e0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301363c
@@ -55,7 +55,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc4-next-20161111-133604-g37634f0 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Nov 10 20:19:07 PST 2016
+Linux version 4.9.0-rc5-next-20161114-133832-gb60de3e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Nov 13 23:49:12 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -66,7 +66,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c1428f80, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1429240, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -79,7 +79,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1994964K/2096128K available (9216K kernel code, 1194K rwdata, 4100K rodata, 2048K init, 343K bss, 35628K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1994832K/2096128K available (9216K kernel code, 1195K rwdata, 4116K rodata, 2048K init, 473K bss, 35760K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -89,8 +89,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc142aa80   (1195 kB)
-       .bss : 0xc142c000 - 0xc1481dc8   ( 344 kB)
+      .data : 0xc1300000 - 0xc142ad40   (1196 kB)
+       .bss : 0xc142c000 - 0xc14a2408   ( 474 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -197,23 +197,6 @@
 tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
 tegra-apbdma 60020000.dma: Tegra20 APB DMA driver register 32 channels
 tegra-pmc 7000e400.pmc: emergency thermal reset enabled
-------------[ cut here ]------------
-WARNING: CPU: 0 PID: 0 at kernel/rcu/tree.c:380 rcu_note_context_switch+0x24/0x2c
-Modules linked in:
-CPU: 0 PID: 0 Comm: swapper/0 Not tainted 4.9.0-rc4-next-20161111-133604-g37634f0 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c030fab8>] (unwind_backtrace) from [<c030b104>] (show_stack+0x10/0x14)
-[<c030b104>] (show_stack) from [<c0592150>] (dump_stack+0x84/0xa4)
-[<c0592150>] (dump_stack) from [<c0342be0>] (__warn+0xd4/0x100)
-[<c0342be0>] (__warn) from [<c0342c2c>] (warn_slowpath_null+0x20/0x28)
-[<c0342c2c>] (warn_slowpath_null) from [<c0393b00>] (rcu_note_context_switch+0x24/0x2c)
-[<c0393b00>] (rcu_note_context_switch) from [<c0bbef5c>] (__schedule+0x64/0x500)
-[<c0bbef5c>] (__schedule) from [<c0bbf51c>] (schedule+0x38/0x9c)
-[<c0bbf51c>] (schedule) from [<c0bbf790>] (schedule_preempt_disabled+0xc/0x10)
-[<c0bbf790>] (schedule_preempt_disabled) from [<c037ad94>] (cpu_startup_entry+0x180/0x1f4)
-[<c037ad94>] (cpu_startup_entry) from [<c1100c04>] (start_kernel+0x338/0x3a4)
-[<c1100c04>] (start_kernel) from [<8020807c>] (0x8020807c)
----[ end trace a0b8d9d5fffc8e0f ]---
 Serial: 8250/16550 driver, 4 ports, IRQ sharing disabled
 console [ttyS0] disabled
 70006300.serial: ttyS0 at MMIO 0x70006300 (irq = 335, base_baud = 25500000) is a Tegra
@@ -310,15 +293,15 @@
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
 NET: Registered protocol family 10
+IPv6: Attempt to unregister permanent protocol 6
+IPv6: Attempt to unregister permanent protocol 136
+IPv6: Attempt to unregister permanent protocol 17
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
  mmcblk0: p1
-IPv6: Attempt to unregister permanent protocol 6
-IPv6: Attempt to unregister permanent protocol 136
-IPv6: Attempt to unregister permanent protocol 17
 NET: Unregistered protocol family 10
 sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
 NET: Registered protocol family 17
@@ -403,8 +386,8 @@
 pci 0000:01:00.0: reg 0x10: [mem 0x00000000-0x00001fff 64bit]
 pci 0000:01:00.0: PME# supported from D0 D3hot D3cold
 PCI: bus1: Fast back to back transfers disabled
-ata1: SATA link down (SStatus 0 SControl 300)
 pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
+ata1: SATA link down (SStatus 0 SControl 300)
 pci 0000:02:00.0: [10ec:8168] type 00 class 0x020000
 pci 0000:02:00.0: reg 0x10: [io  0x0000-0x00ff]
 pci 0000:02:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit]
@@ -454,7 +437,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:16 UTC (946684816)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:15 UTC (946684815)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-11-14 01:19:57.897315720 -0800
+++ /tmp/foob	2016-11-14 01:19:57.901315733 -0800
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MMC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -34,18 +34,18 @@
 	 #################################################
 	 4 MiB/s
 done
-Bytes transferred = 5480248 (539f38 hex)
+Bytes transferred = 5480000 (539e40 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 4 MiB/s
+	 3.8 MiB/s
 done
 Bytes transferred = 67133 (1063d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x539f38 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x539e40 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301363c
@@ -54,7 +54,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc4-next-20161111-133604-g37634f0 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Nov 10 20:18:45 PST 2016
+Linux version 4.9.0-rc5-next-20161114-133832-gb60de3e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Nov 13 23:49:00 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -64,7 +64,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0d5e080, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0d5e1c0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -77,7 +77,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999840K/2096128K available (8192K kernel code, 381K rwdata, 2468K rodata, 1024K init, 370K bss, 30752K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1999840K/2096128K available (8192K kernel code, 382K rwdata, 2468K rodata, 1024K init, 370K bss, 30752K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -87,7 +87,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d5f740   ( 382 kB)
+      .data : 0xc0d00000 - 0xc0d5f880   ( 383 kB)
        .bss : 0xc0d61000 - 0xc0dbdb1c   ( 371 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -128,7 +128,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?[    0.585810] hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
+No ATAGs?[    0.584522] hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 8 bytes.
 +5V_SYS: supplied by +VDD_MUX
 +3.3V_SYS: supplied by +VDD_MUX
@@ -188,27 +188,11 @@
 tegra-xusb-padctl 7009f000.padctl: failed to setup XUSB ports: -517
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
 tegra-pcie 1003000.pcie-controller: Failed to get supply 'avddio-pex': -517
-------------[ cut here ]------------
-WARNING: CPU: 2 PID: 0 at kernel/rcu/tree.c:380 rcu_note_context_switch+0x268/0x2c8
 tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
-Modules linked in:
-CPU: 2 PID: 0 Comm: swapper/2 Not tainted 4.9.0-rc4-next-20161111-133604-g37634f0 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c010f288>] (unwind_backtrace) from [<c010a86c>] (show_stack+0x10/0x14)
-[<c010a86c>] (show_stack) from [<c034677c>] (dump_stack+0x84/0xa4)
 tegra-apbdma 60020000.dma: Tegra20 APB DMA driver register 32 channels
 tegra-pmc 7000e400.pmc: emergency thermal reset enabled
 Serial: 8250/16550 driver, 4 ports, IRQ sharing disabled
 console [ttyS0] disabled
-[<c034677c>] (dump_stack) from [<c011f754>] (__warn+0xd4/0x100)
-[<c011f754>] (__warn) from [<c011f7a0>] (warn_slowpath_null+0x20/0x28)
-[<c011f7a0>] (warn_slowpath_null) from [<c0177a48>] (rcu_note_context_switch+0x268/0x2c8)
-[<c0177a48>] (rcu_note_context_switch) from [<c081d9f4>] (__schedule+0x70/0x60c)
-[<c081d9f4>] (__schedule) from [<c081e160>] (schedule+0x50/0xc0)
-[<c081e160>] (schedule) from [<c081e3bc>] (schedule_preempt_disabled+0x24/0x40)
-[<c081e3bc>] (schedule_preempt_disabled) from [<c015ab38>] (cpu_startup_entry+0x180/0x1f4)
-[<c015ab38>] (cpu_startup_entry) from [<801014cc>] (0x801014cc)
----[ end trace b2185ed87737e2da ]---
 70006300.serial: ttyS0 at MMIO 0x70006300 (irq = 335, base_baud = 25500000) is a Tegra
 console [ttyS0] enabled
 console [ttyS0] enabled
@@ -396,8 +380,8 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
-ata1: SATA link down (SStatus 0 SControl 300)
 tegra-pcie 1003000.pcie-controller: probing port 0, using 2 lanes
+ata1: SATA link down (SStatus 0 SControl 300)
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
 tegra-pcie 1003000.pcie-controller: probing port 1, using 1 lanes
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000000
@@ -435,8 +419,8 @@
 pci 0000:01:00.0: BAR 0: assigned [mem 0x13000000-0x13001fff 64bit]
 pci 0000:00:01.0: PCI bridge to [bus 01]
 pci 0000:00:01.0:   bridge window [mem 0x13000000-0x130fffff]
- mmcblk1: p1 p2 p3
 pci 0000:02:00.0: BAR 4: assigned [mem 0x20000000-0x20003fff 64bit pref]
+ mmcblk1: p1 p2 p3
 pci 0000:02:00.0: BAR 2: assigned [mem 0x13100000-0x13100fff 64bit]
 pci 0000:02:00.0: BAR 0: assigned [io  0x1000-0x10ff]
 pci 0000:00:02.0: PCI bridge to [bus 02]
@@ -467,7 +451,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:15 UTC (946684815)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:14 UTC (946684814)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-11-14 01:19:57.933315835 -0800
+++ /tmp/foob	2016-11-14 01:19:57.937315846 -0800
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -41,21 +41,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ############################################
-	 3.6 MiB/s
+	 ##############################################
+	 3.5 MiB/s
 done
-Bytes transferred = 7313280 (6f9780 hex)
+Bytes transferred = 7354592 (7038e0 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.4 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 67943 (10967 hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6f9780 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x7038e0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013966
@@ -63,7 +63,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc4-next-20161111-133604-g37634f0 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Nov 10 20:19:07 PST 2016
+Linux version 4.9.0-rc5-next-20161114-133832-gb60de3e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Nov 13 23:49:12 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -75,7 +75,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c1428f80, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1429240, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -88,7 +88,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1995980K/2097148K available (9216K kernel code, 1194K rwdata, 4100K rodata, 2048K init, 343K bss, 35632K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1995848K/2097148K available (9216K kernel code, 1195K rwdata, 4116K rodata, 2048K init, 473K bss, 35764K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -98,8 +98,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc142aa80   (1195 kB)
-       .bss : 0xc142c000 - 0xc1481dc8   ( 344 kB)
+      .data : 0xc1300000 - 0xc142ad40   (1196 kB)
+       .bss : 0xc142c000 - 0xc14a2408   ( 474 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -207,22 +207,6 @@
 Serial: 8250/16550 driver, 4 ports, IRQ sharing disabled
 console [ttyS0] disabled
 70006000.serial: ttyS0 at MMIO 0x70006000 (irq = 333, base_baud = 25500000) is a Tegra
-------------[ cut here ]------------
-WARNING: CPU: 1 PID: 0 at kernel/rcu/tree.c:380 rcu_note_context_switch+0x24/0x2c
-Modules linked in:
-CPU: 1 PID: 0 Comm: swapper/1 Not tainted 4.9.0-rc4-next-20161111-133604-g37634f0 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c030fab8>] (unwind_backtrace) from [<c030b104>] (show_stack+0x10/0x14)
-[<c030b104>] (show_stack) from [<c0592150>] (dump_stack+0x84/0xa4)
-[<c0592150>] (dump_stack) from [<c0342be0>] (__warn+0xd4/0x100)
-[<c0342be0>] (__warn) from [<c0342c2c>] (warn_slowpath_null+0x20/0x28)
-[<c0342c2c>] (warn_slowpath_null) from [<c0393b00>] (rcu_note_context_switch+0x24/0x2c)
-[<c0393b00>] (rcu_note_context_switch) from [<c0bbef5c>] (__schedule+0x64/0x500)
-[<c0bbef5c>] (__schedule) from [<c0bbf51c>] (schedule+0x38/0x9c)
-[<c0bbf51c>] (schedule) from [<c0bbf790>] (schedule_preempt_disabled+0xc/0x10)
-[<c0bbf790>] (schedule_preempt_disabled) from [<c037ad94>] (cpu_startup_entry+0x180/0x1f4)
-[<c037ad94>] (cpu_startup_entry) from [<80301c2c>] (0x80301c2c)
----[ end trace b2cb96cb2482ccb3 ]---
 console [ttyS0] enabled
 SuperH (H)SCI(F) driver initialized
 msm_serial: driver initialized
@@ -315,17 +299,17 @@
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
 NET: Registered protocol family 10
+elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
+input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
+IPv6: Attempt to unregister permanent protocol 6
+IPv6: Attempt to unregister permanent protocol 136
+IPv6: Attempt to unregister permanent protocol 17
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
 mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
 mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
-IPv6: Attempt to unregister permanent protocol 6
-IPv6: Attempt to unregister permanent protocol 136
-IPv6: Attempt to unregister permanent protocol 17
-elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
-input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 NET: Unregistered protocol family 10
 sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
 NET: Registered protocol family 17
@@ -349,10 +333,10 @@
 +3.3V_PANEL: supplied by +3.3V_RUN
 +3.3V_LP0: supplied by +3.3V_SYS
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
+elan_i2c 1-0015: failed to read report data: -121
 +5V_HDMI_CON: supplied by +5V_SYS
 usb3-0 supply vbus not found, using dummy regulator
 usb3-1 supply vbus not found, using dummy regulator
-elan_i2c 1-0015: failed to read report data: -121
 tegra-sor 54540000.sor: failed to probe output: -517
 panel supply power not found, using dummy regulator
 sdhci-tegra 700b0000.sdhci: allocated mmc-pwrseq
@@ -364,9 +348,9 @@
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc2: Unknown controller version (3). You may experience problems.
 mmc2: Invalid maximum block size, assuming 512 bytes
-elan_i2c 1-0015: invalid report id data (ff)
 mmc1: new high speed SDIO card at address 0001
 mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
+elan_i2c 1-0015: invalid report id data (ff)
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
 mmc2: new high speed SDHC card at address 0007
@@ -381,7 +365,7 @@
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:25 UTC (946684825)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:23 UTC (946684823)
 +1.8V_RUN_CAM: disabling
 +3.3V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2016-11-14 01:19:57.941315858 -0800
+++ /tmp/foob	2016-11-14 01:19:57.941315858 -0800
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -42,18 +42,18 @@
 	 #################################################
 	 3.6 MiB/s
 done
-Bytes transferred = 5480248 (539f38 hex)
+Bytes transferred = 5480000 (539e40 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.1 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 67943 (10967 hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x539f38 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x539e40 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013966
@@ -62,7 +62,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc4-next-20161111-133604-g37634f0 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Nov 10 20:18:45 PST 2016
+Linux version 4.9.0-rc5-next-20161114-133832-gb60de3e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Nov 13 23:49:00 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -73,7 +73,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c0d5e080, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0d5e1c0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -86,7 +86,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2000856K/2097148K available (8192K kernel code, 381K rwdata, 2468K rodata, 1024K init, 370K bss, 30756K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 2000856K/2097148K available (8192K kernel code, 382K rwdata, 2468K rodata, 1024K init, 370K bss, 30756K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -96,7 +96,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d5f740   ( 382 kB)
+      .data : 0xc0d00000 - 0xc0d5f880   ( 383 kB)
        .bss : 0xc0d61000 - 0xc0dbdb1c   ( 371 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -137,7 +137,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?[    0.499271] hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
+No ATAGs?[    0.498875] hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 8 bytes.
 tegra-mc 70019000.memory-controller: no memory timings for RAM code 0 registered
 +5V_SYS: supplied by +VDD_MUX
@@ -201,38 +201,6 @@
 Serial: 8250/16550 driver, 4 ports, IRQ sharing disabled
 console [ttyS0] disabled
 70006000.serial: ttyS0 at MMIO 0x70006000 (irq = 333, base_baud = 25500000) is a Tegra
-------------[ cut here ]------------
-------------[ cut here ]------------
-WARNING: CPU: 2 PID: 0 at kernel/rcu/tree.c:380 rcu_note_context_switch+0x268/0x2c8
-WARNING: CPU: 3 PID: 0 at kernel/rcu/tree.c:380 rcu_note_context_switch+0x268/0x2c8
-Modules linked in:
-Modules linked in:
-CPU: 2 PID: 0 Comm: swapper/2 Not tainted 4.9.0-rc4-next-20161111-133604-g37634f0 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c010f288>] (unwind_backtrace) from [<c010a86c>] (show_stack+0x10/0x14)
-[<c010a86c>] (show_stack) from [<c034677c>] (dump_stack+0x84/0xa4)
-[<c034677c>] (dump_stack) from [<c011f754>] (__warn+0xd4/0x100)
-[<c011f754>] (__warn) from [<c011f7a0>] (warn_slowpath_null+0x20/0x28)
-[<c011f7a0>] (warn_slowpath_null) from [<c0177a48>] (rcu_note_context_switch+0x268/0x2c8)
-[<c0177a48>] (rcu_note_context_switch) from [<c081d9f4>] (__schedule+0x70/0x60c)
-[<c081d9f4>] (__schedule) from [<c081e160>] (schedule+0x50/0xc0)
-[<c081e160>] (schedule) from [<c081e3bc>] (schedule_preempt_disabled+0x24/0x40)
-[<c081e3bc>] (schedule_preempt_disabled) from [<c015ab38>] (cpu_startup_entry+0x180/0x1f4)
-[<c015ab38>] (cpu_startup_entry) from [<801014cc>] (0x801014cc)
-CPU: 3 PID: 0 Comm: swapper/3 Not tainted 4.9.0-rc4-next-20161111-133604-g37634f0 #1
----[ end trace 0b3e539468976c6a ]---
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c010f288>] (unwind_backtrace) from [<c010a86c>] (show_stack+0x10/0x14)
-[<c010a86c>] (show_stack) from [<c034677c>] (dump_stack+0x84/0xa4)
-[<c034677c>] (dump_stack) from [<c011f754>] (__warn+0xd4/0x100)
-[<c011f754>] (__warn) from [<c011f7a0>] (warn_slowpath_null+0x20/0x28)
-[<c011f7a0>] (warn_slowpath_null) from [<c0177a48>] (rcu_note_context_switch+0x268/0x2c8)
-[<c0177a48>] (rcu_note_context_switch) from [<c081d9f4>] (__schedule+0x70/0x60c)
-[<c081d9f4>] (__schedule) from [<c081e160>] (schedule+0x50/0xc0)
-[<c081e160>] (schedule) from [<c081e3bc>] (schedule_preempt_disabled+0x24/0x40)
-[<c081e3bc>] (schedule_preempt_disabled) from [<c015ab38>] (cpu_startup_entry+0x180/0x1f4)
-[<c015ab38>] (cpu_startup_entry) from [<801014cc>] (0x801014cc)
----[ end trace 0b3e539468976c6b ]---
 console [ttyS0] enabled
 console [ttyS0] enabled
 bootconsole [earlycon0] disabled
@@ -330,10 +298,10 @@
 IPv6: Attempt to unregister permanent protocol 6
 IPv6: Attempt to unregister permanent protocol 136
 IPv6: Attempt to unregister permanent protocol 17
-input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input3
 NET: Unregistered protocol family 10
 IPv6: ah6_init: can't add xfrm type
 IPv6: esp6_init: can't add xfrm type
+input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input3
 IPv6: ipcomp6_init: can't add xfrm type
 mip6: Mobile IPv6
 mip6: mip6_init: can't add xfrm type(destopt)
@@ -408,7 +376,7 @@
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input4
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:26 UTC (946684826)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:23 UTC (946684823)
 +1.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
 +1.05V_RUN_CAM_REAR: disabling
@@ -420,14 +388,13 @@
   #0: tegra-hda at 0x70038000 irq 343
   #1: GoogleNyanBig
 EXT4-fs (mmcblk0p3): couldn't mount as ext3 due to feature incompatibilities
-uvcvideo: Found UVC 1.00 device HD WebCam (1bcf:2c67)
-input: HD WebCam as /devices/soc0/70090000.usb/usb1/1-2/1-2:1.0/input/input5
 EXT4-fs (mmcblk0p3): recovery complete
 EXT4-fs (mmcblk0p3): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:3.
-usb 1-3: new high-speed USB device number 3 using tegra-xusb
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
-bash: no job control in this shell
-root@(none):/# [    6.231372] asix 1-3:1.0 eth0: register 'asix' at usb-70090000.usb-3, ASIX AX88772 USB 2.0 Ethernet, c8:d7:19:90:fa:65
+bash: no job control in th[    3.206038] uvcvideo: Found UVC 1.00 device HD WebCam (1bcf:2c67)
+is shell
+input: HD WebCam as /devices/soc0/70090000.usb/usb1/1-2/1-2:1.0/input/input5
+root@(none):/# 
\ No newline at end of file
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2016-11-14 01:19:57.977315975 -0800
+++ /tmp/foob	2016-11-14 01:19:57.981315988 -0800
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1801622 us
+Bootloader-Cpu Init at (time stamp): 1801585 us
 
 Pinmux changes applied in kernel way
 
@@ -62,20 +62,20 @@
 Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1801596  us
-Bootloader (AosInit) time: 733499 us
-Bootloader Main Init time: 180173 us
-BlockDevMgrInit time: 157536 us
+Miscellaneous avp time: 1801559  us
+Bootloader (AosInit) time: 733474 us
+Bootloader Main Init time: 180220 us
+BlockDevMgrInit time: 157547 us
 PartMgrInit time: 8 us
-StorMgrInit time: 31 us
-FsMgrInit time: 2 us
-Partition Table load: 77078 us 
-Display init time: 574344 us
+StorMgrInit time: 30 us
+FsMgrInit time: 3 us
+Partition Table load: 77082 us 
+Display init time: 574588 us
 Kernel read time: 378 msec
 Load Kernel Size:   16304128 Bytes
 Bl Kernel ReadSpeed: 43132000 Bytes/sec
-Pre-Kernel Jump time: 1740799 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5843698 us
+Pre-Kernel Jump time: 1740829 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5845965 us
 Booting Linux on physical CPU 0x0
 Linux version 4.8.0-239303-g3477d16 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Fri Oct 7 01:12:37 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
@@ -184,7 +184,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(1.159:1): initialized
+audit: type=2000 audit(1.131:1): initialized
 workingset: timestamp_bits=46 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -251,10 +251,11 @@
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-+VDD_LED: supplied by +VDD_MUX
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
++VDD_LED: supplied by +VDD_MUX
 +5V_USB_HS: supplied by +5V_SYS
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 +5V_USB_SS: supplied by +5V_SYS
 +5V_HDMI_CON: supplied by +5V_SYS
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
@@ -272,9 +273,6 @@
 Freeing unused kernel memory: 960K (ffff800000c00000 - ffff800000cf0000)
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
-sdhci-tegra 700b0400.sdhci: Got CD GPIO
-sdhci-tegra 700b0400.sdhci: Got WP GPIO
 
 System initialization...
 
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-11-14 01:19:57.985315999 -0800
+++ /tmp/foob	2016-11-14 01:19:57.985315999 -0800
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -46,10 +46,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ############################################
-	 4.1 MiB/s
+	 ##############################################
+	 3.9 MiB/s
 done
-Bytes transferred = 7313280 (6f9780 hex)
+Bytes transferred = 7354592 (7038e0 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -60,7 +60,7 @@
 done
 Bytes transferred = 21404 (539c hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x6f9780 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x7038e0 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 0241039b
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc4-next-20161111-133604-g37634f0 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Nov 10 20:19:07 PST 2016
+Linux version 4.9.0-rc5-next-20161114-133832-gb60de3e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Nov 13 23:49:12 PST 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: Compulab TrimSlice board
@@ -78,7 +78,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c1428f80, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c1429240, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -91,7 +91,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 955888K/1048576K available (9216K kernel code, 1194K rwdata, 4100K rodata, 2048K init, 343K bss, 27152K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 955756K/1048576K available (9216K kernel code, 1195K rwdata, 4116K rodata, 2048K init, 473K bss, 27284K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -101,8 +101,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc142aa80   (1195 kB)
-       .bss : 0xc142c000 - 0xc1481dc8   ( 344 kB)
+      .data : 0xc1300000 - 0xc142ad40   (1196 kB)
+       .bss : 0xc142c000 - 0xc14a2408   ( 474 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -316,21 +316,15 @@
 IPv6: Attempt to unregister permanent protocol 136
 IPv6: Attempt to unregister permanent protocol 17
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
-mmc1: new high speed SD card at address b368
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
- mmcblk1: p1 p2
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
  mmcblk0: p1
+mmc1: new high speed SD card at address b368
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
+ mmcblk1: p1 p2
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
-------------[ cut here ]------------
-WARNING: CPU: 1 PID: 0 at kernel/rcu/tree.c:380 rcu_note_context_switch+0x24/0x2c
-Modules linked in:
-CPU: 1 PID: 0 Comm: swapper/1 Not tainted 4.9.0-rc4-next-20161111-133604-g37634f0 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c030fab8>] (unwind_backtrace) from [<c030b104>] (show_stack+0x10/0x14)
 NET: Unregistered protocol family 10
 sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
 NET: Registered protocol family 17
@@ -342,32 +336,22 @@
 Key type dns_resolver registered
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
-[<c030b104>] (show_stack) from [<c0592150>] (dump_stack+0x84/0xa4)
-[<c0592150>] (dump_stack) from [<c0342be0>] (__warn+0xd4/0x100)
-[<c0342be0>] (__warn) from [<c0342c2c>] (warn_slowpath_null+0x20/0x28)
-[<c0342c2c>] (warn_slowpath_null) from [<c0393b00>] (rcu_note_context_switch+0x24/0x2c)
-[<c0393b00>] (rcu_note_context_switch) from [<c0bbef5c>] (__schedule+0x64/0x500)
-[<c0bbef5c>] (__schedule) from [<c0bbf51c>] (schedule+0x38/0x9c)
-[<c0bbf51c>] (schedule) from [<c0bbf790>] (schedule_preempt_disabled+0xc/0x10)
-[<c0bbf790>] (schedule_preempt_disabled) from [<c037ad94>] (cpu_startup_entry+0x180/0x1f4)
-[<c037ad94>] (cpu_startup_entry) from [<00301c2c>] (0x301c2c)
----[ end trace 5ef4bb43ef5cae44 ]---
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
-usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2015-02-05 23:11:21 UTC (1423177881)
+rtc-em3027 2-0056: setting system clock to 2015-02-05 23:17:56 UTC (1423178276)
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
+usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 random: fast init done
 asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:17.
+VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-11-14 01:19:58.017316101 -0800
+++ /tmp/foob	2016-11-14 01:19:58.021316113 -0800
@@ -47,7 +47,7 @@
 	 #################################################
 	 4.1 MiB/s
 done
-Bytes transferred = 5480248 (539f38 hex)
+Bytes transferred = 5480000 (539e40 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21404 (539c hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x539f38 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x539e40 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 0241039b
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc4-next-20161111-133604-g37634f0 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Nov 10 20:18:45 PST 2016
+Linux version 4.9.0-rc5-next-20161114-133832-gb60de3e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Nov 13 23:49:00 PST 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: Compulab TrimSlice board
@@ -76,7 +76,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0d5e080, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0d5e1c0, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -89,7 +89,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 960764K/1048576K available (8192K kernel code, 381K rwdata, 2468K rodata, 1024K init, 370K bss, 22276K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 960764K/1048576K available (8192K kernel code, 382K rwdata, 2468K rodata, 1024K init, 370K bss, 22276K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -99,7 +99,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d5f740   ( 382 kB)
+      .data : 0xc0d00000 - 0xc0d5f880   ( 383 kB)
        .bss : 0xc0d61000 - 0xc0dbdb1c   ( 371 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -139,7 +139,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?[    0.349365] hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
+No ATAGs?[    0.349195] hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 4 bytes.
 usb1_vbus: Failed to request enable GPIO170: -517
 reg-fixed-voltage regulators:regulator@2: Failed to register regulator: -517
@@ -192,7 +192,6 @@
 pci 0000:00:01.0: [10de:0bf0] type 01 class 0x060000
 pci 0000:00:01.0: PME# supported from D0 D1 D2 D3hot D3cold
 PCI: bus0: Fast back to back transfers disabled
-------------[ cut here ]------------
 pci 0000:00:01.0: bridge configuration invalid ([bus 00-00]), reconfiguring
 pci 0000:01:00.0: [10ec:8168] type 00 class 0x020000
 pci 0000:01:00.0: reg 0x10: [io  0x0000-0x00ff]
@@ -221,21 +220,6 @@
 pci 0000:01:00.0: Signaling PME through PCIe PME interrupt
 pcie_pme 0000:00:01.0:pcie001: service driver pcie_pme loaded
 tegra-apbdma 6000a000.dma: Tegra20 APB DMA driver register 16 channels
-WARNING: CPU: 1 PID: 0 at kernel/rcu/tree.c:380 rcu_note_context_switch+0x268/0x2c8
-Modules linked in:
-CPU: 1 PID: 0 Comm: swapper/1 Not tainted 4.9.0-rc4-next-20161111-133604-g37634f0 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c010f288>] (unwind_backtrace) from [<c010a86c>] (show_stack+0x10/0x14)
-[<c010a86c>] (show_stack) from [<c034677c>] (dump_stack+0x84/0xa4)
-[<c034677c>] (dump_stack) from [<c011f754>] (__warn+0xd4/0x100)
-[<c011f754>] (__warn) from [<c011f7a0>] (warn_slowpath_null+0x20/0x28)
-[<c011f7a0>] (warn_slowpath_null) from [<c0177a48>] (rcu_note_context_switch+0x268/0x2c8)
-[<c0177a48>] (rcu_note_context_switch) from [<c081d9f4>] (__schedule+0x70/0x60c)
-[<c081d9f4>] (__schedule) from [<c081e160>] (schedule+0x50/0xc0)
-[<c081e160>] (schedule) from [<c081e3bc>] (schedule_preempt_disabled+0x24/0x40)
-[<c081e3bc>] (schedule_preempt_disabled) from [<c015ab38>] (cpu_startup_entry+0x180/0x1f4)
-[<c015ab38>] (cpu_startup_entry) from [<001014cc>] (0x1014cc)
----[ end trace 63275f0fb23cd40a ]---
 Serial: 8250/16550 driver, 4 ports, IRQ sharing disabled
 console [ttyS0] disabled
 70006000.serial: ttyS0 at MMIO 0x70006000 (irq = 279, base_baud = 13500000) is a Tegra
@@ -330,10 +314,10 @@
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
-hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
 IPv6: Attempt to unregister permanent protocol 6
 IPv6: Attempt to unregister permanent protocol 136
+hub 1-1:1.0: USB hub found
+hub 1-1:1.0: 4 ports detected
 IPv6: Attempt to unregister permanent protocol 17
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 NET: Unregistered protocol family 10
@@ -343,7 +327,6 @@
 mip6: Mobile IPv6
 mip6: mip6_init: can't add xfrm type(destopt)
 sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
-asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
 NET: Registered protocol family 17
 NET: Registered protocol family 15
 can: controller area network core (rev 20120528 abi 9)
@@ -359,7 +342,6 @@
 Bluetooth: HIDP socket layer initialized
 Key type dns_resolver registered
 Registering SWP/SWPB emulation handler
-utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
@@ -367,14 +349,15 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2015-02-05 23:09:50 UTC (1423177790)
+rtc-em3027 2-0056: setting system clock to 2015-02-05 23:16:21 UTC (1423178181)
 ALSA device list:
   #0: tegra-trimslice
+asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
-random: fast init done
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
+random: fast init done
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2016-11-14 01:19:58.065316254 -0800
+++ /tmp/foob	2016-11-14 01:19:58.065316254 -0800
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 773209 us
-Basic modules init took 348701 us
+Tegraboot started after 728998 us
+Basic modules init took 349111 us
 Sec Bootdevice Read Time = 203 ms, Read Size = 8459 KB
-Next stage binary read took 14368 us
-Carveout took 252736 us
-CPU initialization took 92850 us
-Total time taken by TegraBoot 708655 us
+Next stage binary read took 14835 us
+Carveout took 252720 us
+CPU initialization took 92321 us
+Total time taken by TegraBoot 708987 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.612] LPDDR4 Training: Number of tables = 10
+64b[0001.569] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-14668288 bytes read in 788 ms (17.8 MiB/s)
+14692864 bytes read in 788 ms (17.8 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
 43755 bytes read in 454 ms (93.8 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 502 ms (3.1 MiB/s)
+1627787 bytes read in 499 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc4-next-20161111-158037-g37634f0 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Nov 10 20:21:58 PST 2016
+Linux version 4.9.0-rc5-next-20161114-158265-gb60de3e (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Nov 13 23:52:27 PST 2016
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -166,15 +166,15 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffff80007a000000-ffff80007dffffff]
-Memory: 2978348K/3131392K available (8380K kernel code, 884K rwdata, 3976K rodata, 1024K init, 283K bss, 136660K reserved, 16384K cma-reserved)
+Memory: 2978324K/3131392K available (8380K kernel code, 908K rwdata, 4012K rodata, 1024K init, 283K bss, 136684K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff0000088b0000   (  8384 KB)
     .rodata : 0xffff0000088b0000 - 0xffff000008ca0000   (  4032 KB)
       .init : 0xffff000008ca0000 - 0xffff000008da0000   (  1024 KB)
-      .data : 0xffff000008da0000 - 0xffff000008e7d200   (   885 KB)
-       .bss : 0xffff000008e7d200 - 0xffff000008ec40c0   (   284 KB)
+      .data : 0xffff000008da0000 - 0xffff000008e83200   (   909 KB)
+       .bss : 0xffff000008e83200 - 0xffff000008eca140   (   284 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -259,7 +259,7 @@
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 PCI: CLS 0 bytes, default 128
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 80896000
+kvm [1]: IDMAP page: 8089b000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
@@ -267,7 +267,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.543:1): initialized
+audit: type=2000 audit(0.527:1): initialized
 workingset: timestamp_bits=46 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -318,53 +318,6 @@
 mousedev: PS/2 mouse device common for all mice
 tegra_rtc 7000e000.rtc: rtc core: registered 7000e000.rtc as rtc1
 tegra_rtc 7000e000.rtc: Tegra internal Real Time Clock
-------------[ cut here ]------------
-WARNING: CPU: 3 PID: 0 at kernel/rcu/tree.c:380 rcu_momentary_dyntick_idle+0xbc/0xc4
-Modules linked in:
-
-CPU: 3 PID: 0 Comm: swapper/3 Not tainted 4.9.0-rc4-next-20161111-158037-g37634f0 #1
-Hardware name: NVIDIA Tegra210 P2371 (P2530/P2595) reference design (DT)
-task: ffff8000bc113200 task.stack: ffff8000bc124000
-PC is at rcu_momentary_dyntick_idle+0xbc/0xc4
-LR is at rcu_note_context_switch+0x1b8/0x2ac
-pc : [<ffff000008111be0>] lr : [<ffff000008116b68>] pstate: 600000c5
-sp : ffff8000bc127e90
-x29: ffff8000bc127e90 x28: ffff000008da8000 
-x27: 0000000000000001 x26: ffff8000bc124000 
-x25: 0000000000000000 x24: ffff8000bc124000 
-x23: ffff000008890a20 x22: ffff8000bc113200 
-x21: ffff8000bffa1880 x20: ffff000008da8000 
-x19: ffff8000bc113200 x18: 0000000000000007 
-x17: 000000000000000e x16: 0000000000000001 
-x15: 0000000000000019 x14: 0000000000000033 
-x13: 000000000000004c x12: 0000000000000000 
-x11: 0000000000000000 x10: 0000000000000870 
-x9 : ffff8000bc124000 x8 : ffff8000bc113ad0 
-x7 : 000000003142cd48 x6 : 003473bbffd0bc00 
-x5 : ffff8000bff9bbd0 x4 : 0000000000000004 
-x3 : ffff000008dc0680 x2 : ffff8000bff9bbdc 
-x1 : 0000000000000001 x0 : ffff000008dc4000 
-
----[ end trace 7e6e58e44d21a7ea ]---
-Call trace:
-Exception stack(0xffff8000bc127cc0 to 0xffff8000bc127df0)
-7cc0: ffff8000bc113200 0001000000000000 ffff8000bc127e90 ffff000008111be0
-7ce0: 00000000000003e0 ffff000008b6d13e ffff000008e8c410 ffff8000bc127f80
-7d00: ffff8000bc127d90 ffff00000837ded8 ffff000008e8c434 ffff8000bc127fd0
-7d20: 00000000ffffffd8 ffff0000088f0b64 00000000000003e0 ffff000008b6dcc1
-7d40: ffff8000bffa1880 0000000000000bb7 0000000000000002 0000000000000000
-7d60: ffff000008dc4000 0000000000000001 ffff8000bff9bbdc ffff000008dc0680
-7d80: 0000000000000004 ffff8000bff9bbd0 003473bbffd0bc00 000000003142cd48
-7da0: ffff8000bc113ad0 ffff8000bc124000 0000000000000870 0000000000000000
-7dc0: 0000000000000000 000000000000004c 0000000000000033 0000000000000019
-7de0: 0000000000000001 000000000000000e
-[<ffff000008111be0>] rcu_momentary_dyntick_idle+0xbc/0xc4
-[<ffff0000088904e8>] __schedule+0x68/0x564
-[<ffff000008890a20>] schedule+0x3c/0xa0
-[<ffff000008890e00>] schedule_preempt_disabled+0x20/0x38
-[<ffff0000080f6810>] cpu_startup_entry+0xdc/0x1d4
-[<ffff00000808e698>] secondary_start_kernel+0x158/0x18c
-[<000000008089b1a4>] 0x8089b1a4
 i2c /dev entries driver
 sdhci: Secure Digital Host Controller Interface driver
 sdhci: Copyright(c) Pierre Ossman
@@ -395,6 +348,6 @@
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
-bash: no job contr[    1.690051] random: fast init done
+bash: no job contr[    1.393931] random: fast init done
 ol in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2016-11-14 01:19:58.109316393 -0800
+++ /tmp/foob	2016-11-14 01:19:58.109316393 -0800
@@ -1,35 +1,8 @@
-.
-cma: Reserved 16 MiB at 0x00000000f5400000
-earlycon: uart0 at MMIO 0x0000000070006000 (options '115200n8')
-bootconsole [uart0] enabled
-psci: probing for conduit method from DT.
-psci: PSCIv1.0 detected in firmware.
-psci: Using standard PSCI v0.2 function IDs
-psci: MIGRATE_INFO_UP_CPU reported invalid physical ID (0x80000000)
-percpu: Embedded 21 pages/cpu @ffff8000bff4d000 s48128 r8192 d29696 u86016
-Detected PIPT I-cache on CPU0
-CPU features: enabling workaround for ARM erratum 832075
-CPU features: enabling workaround for ARM erratum 834220
-Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 735336
-Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
-log_buf_len individual max cpu contribution: 4096 bytes
-log_buf_len total cpu_extra contributions: 12288 bytes
-log_buf_len min size: 16384 bytes
-log_buf_len: 32768 bytes
-early log buf free: 14404(87%)
-PID hash table entries: 4096 (order: 3, 32768 bytes)
-Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
-Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
-software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2832536K/2988032K available (8380K kernel code, 884K rwdata, 3976K rodata, 1024K init, 283K bss, 139112K reserved, 16384K cma-reserved)
-Virtual kernel memory layout:
-    modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
-    vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff0000088b0000   (  8384 KB)
+0 - 0xffff0000088b0000   (  8384 KB)
     .rodata : 0xffff0000088b0000 - 0xffff000008ca0000   (  4032 KB)
       .init : 0xffff000008ca0000 - 0xffff000008da0000   (  1024 KB)
-      .data : 0xffff000008da0000 - 0xffff000008e7d200   (   885 KB)
-       .bss : 0xffff000008e7d200 - 0xffff000008ec40c0   (   284 KB)
+      .data : 0xffff000008da0000 - 0xffff000008e83200   (   909 KB)
+       .bss : 0xffff000008e83200 - 0xffff000008eca140   (   284 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -58,83 +31,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x55a8: expected 0x2004f400
- (comparison mask = 0xffffffff)
-PLL_P already enabled. Postponing set full defaults
-PLL_X already enabled. Postponing set full defaults
-boot misc1 0x0: expected 0x20
- (comparison mask = 0xffffff)
-arm_arch_timer: Architected cp15 timer(s) running at 19.20MHz (phys).
-clocksource: arch_sys_counter: mask: 0xffffffffffffff max_cycles: 0x46d987e47, max_idle_ns: 440795202767 ns
-sched_clock: 56 bits at 19MHz, resolution 52ns, wraps every 4398046511078ns
-Console: colour dummy device 80x25
-console [tty0] enabled
-bootconsole [uart0] disabled
-Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc4-next-20161111-158037-g37634f0 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Nov 10 20:21:58 PST 2016
-Boot CPU: AArch64 Processor [411fd071]
-efi: Getting EFI parameters from FDT:
-efi: UEFI not found.
-cma: Reserved 16 MiB at 0x00000000f5400000
-earlycon: uart0 at MMIO 0x0000000070006000 (options '115200n8')
-bootconsole [uart0] enabled
-psci: probing for conduit method from DT.
-psci: PSCIv1.0 detected in firmware.
-psci: Using standard PSCI v0.2 function IDs
-psci: MIGRATE_INFO_UP_CPU reported invalid physical ID (0x80000000)
-percpu: Embedded 21 pages/cpu @ffff8000bff4d000 s48128 r8192 d29696 u86016
-Detected PIPT I-cache on CPU0
-CPU features: enabling workaround for ARM erratum 832075
-CPU features: enabling workaround for ARM erratum 834220
-Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 735336
-Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
-log_buf_len individual max cpu contribution: 4096 bytes
-log_buf_len total cpu_extra contributions: 12288 bytes
-log_buf_len min size: 16384 bytes
-log_buf_len: 32768 bytes
-early log buf free: 14404(87%)
-PID hash table entries: 4096 (order: 3, 32768 bytes)
-Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
-Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
-software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2832536K/2988032K available (8380K kernel code, 884K rwdata, 3976K rodata, 1024K init, 283K bss, 139112K reserved, 16384K cma-reserved)
-Virtual kernel memory layout:
-    modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
-    vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff0000088b0000   (  8384 KB)
-    .rodata : 0xffff0000088b0000 - 0xffff000008ca0000   (  4032 KB)
-      .init : 0xffff000008ca0000 - 0xffff000008da0000   (  1024 KB)
-      .data : 0xffff000008da0000 - 0xffff000008e7d200   (   885 KB)
-       .bss : 0xffff000008e7d200 - 0xffff000008ec40c0   (   284 KB)
-    fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
-    PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
-    vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
-              0xffff7e0000000000 - 0xffff7e0003000000   (    48 MB actual)
-    memory  : 0xffff800000000000 - 0xffff8000c0000000   (  3072 MB)
-SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
-Preemptible hierarchical RCU implementation.
-	Build-time adjustment of leaf fanout to 64.
-	RCU restricting CPUs from NR_CPUS=64 to nr_cpu_ids=4.
-RCU: Adjusting geometry for rcu_fanout_leaf=64, nr_cpu_ids=4
-NR_IRQS:64 nr_irqs:64 0
-GIC: Using split EOI/Deactivate mode
-/interrupt-controller@60004000: 192 interrupts forwarded to /interrupt-controller@50041000
-boot misc0 0x0: expected 0x80000
- (comparison mask = 0x400ffffb)
-boot misc2 0x1f720005: expected 0x1f720f05
- (comparison mask = 0xffffff17)
-PLL_C already enabled. Postponing set full defaults
-PLL_MB already enabled. Postponing set full defaults
-PLL_U already enabled. Postponing set full defaults
-PLL_D already enabled. Postponing set full defaults
-PLL_DP already enabled. Postponing set full defaults
-boot misc0 0x0: expected 0x20
- (comparison mask = 0x7ffffff)
-boot misc1 0x90000000: expected 0xc0000000
- (comparison mask = 0xf8000000)
-boot misc2 0x0: expected 0xf400f0da
- (comparison mask = 0xffffffff)
-boot misc3 0x55a8: expected 0x2004f400
+boot misc3 0xec60: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -212,56 +109,9 @@
 RPC: Registered tcp transport module.
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 Unpacking initramfs...
-------------[ cut here ]------------
-WARNING: CPU: 2 PID: 0 at kernel/rcu/tree.c:380 rcu_momentary_dyntick_idle+0xbc/0xc4
-Modules linked in:
-
-CPU: 2 PID: 0 Comm: swapper/2 Not tainted 4.9.0-rc4-next-20161111-158037-g37634f0 #1
-Hardware name: Google Pixel C (DT)
-task: ffff8000bc112580 task.stack: ffff8000bc124000
-PC is at rcu_momentary_dyntick_idle+0xbc/0xc4
-LR is at rcu_note_context_switch+0x1b8/0x2ac
-pc : [<ffff000008111be0>] lr : [<ffff000008116b68>] pstate: 600000c5
-sp : ffff8000bc127e90
-x29: ffff8000bc127e90 x28: ffff000008da8000 
-x27: 0000000000000001 x26: ffff8000bc124000 
-x25: 0000000000000000 x24: ffff8000bc124000 
-x23: ffff000008890a20 x22: ffff8000bc112580 
-x21: ffff8000bff81880 x20: ffff000008da8000 
-x19: ffff8000bc112580 x18: 0000000000000007 
-x17: 000000000000000e x16: 0000000000000001 
-x15: 0000000000000019 x14: 0000000000000033 
-x13: 000000000000004c x12: 0000000000000000 
-x11: 0000000000000000 x10: 0000000000000870 
-x9 : ffff8000bc124000 x8 : ffff8000bc112e50 
-x7 : 000000000f842a64 x6 : 001312cffff7cc00 
-x5 : ffff8000bff7bbd0 x4 : 0000000000000004 
-x3 : ffff000008dc0680 x2 : ffff8000bff7bbdc 
-x1 : 0000000000000001 x0 : ffff000008dc4000 
-
----[ end trace 7c4f9f2af6e88cac ]---
-Call trace:
-Exception stack(0xffff8000bc127cc0 to 0xffff8000bc127df0)
-7cc0: ffff8000bc112580 0001000000000000 ffff8000bc127e90 ffff000008111be0
-7ce0: 00000000000003e0 ffff000008b6d13e ffff000008e8c410 ffff8000bc127f80
-7d00: ffff8000bc127d90 ffff00000837ded8 ffff000008e8c434 ffff8000bc127fd0
-7d20: 00000000ffffffd8 ffff0000088f0b64 00000000000003e0 ffff000008b6dcc1
-7d40: ffff000008e8c410 ffff8000bc127fd0 ffff000008e8c7f0 ffff000008b6dcc1
-7d60: ffff000008dc4000 0000000000000001 ffff8000bff7bbdc ffff000008dc0680
-7d80: 0000000000000004 ffff8000bff7bbd0 001312cffff7cc00 000000000f842a64
-7da0: ffff8000bc112e50 ffff8000bc124000 0000000000000870 0000000000000000
-7dc0: 0000000000000000 000000000000004c 0000000000000033 0000000000000019
-7de0: 0000000000000001 000000000000000e
-[<ffff000008111be0>] rcu_momentary_dyntick_idle+0xbc/0xc4
-[<ffff0000088904e8>] __schedule+0x68/0x564
-[<ffff000008890a20>] schedule+0x3c/0xa0
-[<ffff000008890e00>] schedule_preempt_disabled+0x20/0x38
-[<ffff0000080f6810>] cpu_startup_entry+0xdc/0x1d4
-[<ffff00000808e698>] secondary_start_kernel+0x158/0x18c
-[<000000008089b1a4>] 0x8089b1a4
 Freeing initrd memory: 2360K
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 80896000
+kvm [1]: IDMAP page: 8089b000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
@@ -269,7 +119,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.335:1): initialized
+audit: type=2000 audit(0.331:1): initialized
 workingset: timestamp_bits=46 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -356,10 +206,10 @@
 tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
 tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
-max77686-rtc max77620-rtc: setting system clock to 2000-08-22 09:51:03 UTC (966937863)
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 BGND3R 29.1 GiB 
 mmcblk0boot0: mmc0:0001 BGND3R partition 1 4.00 MiB
+max77686-rtc max77620-rtc: setting system clock to 2000-08-25 13:21:16 UTC (967209676)
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-11-14 01:19:58.113316406 -0800
+++ /tmp/foob	2016-11-14 01:19:58.117316420 -0800
@@ -38,10 +38,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ############################################
-	 3.4 MiB/s
+	 ##############################################
+	 3.3 MiB/s
 done
-Bytes transferred = 7313280 (6f9780 hex)
+Bytes transferred = 7354592 (7038e0 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -52,7 +52,7 @@
 done
 Bytes transferred = 51804 (ca5c hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6f9780 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x7038e0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa5b ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc4-next-20161111-133604-g37634f0 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Nov 10 20:19:07 PST 2016
+Linux version 4.9.0-rc5-next-20161114-133832-gb60de3e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Nov 13 23:49:12 PST 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -70,7 +70,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c1428f80, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1429240, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -83,7 +83,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1995004K/2096128K available (9216K kernel code, 1194K rwdata, 4100K rodata, 2048K init, 343K bss, 35588K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1994872K/2096128K available (9216K kernel code, 1195K rwdata, 4116K rodata, 2048K init, 473K bss, 35720K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -93,8 +93,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc142aa80   (1195 kB)
-       .bss : 0xc142c000 - 0xc1481dc8   ( 344 kB)
+      .data : 0xc1300000 - 0xc142ad40   (1196 kB)
+       .bss : 0xc142c000 - 0xc14a2408   ( 474 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -224,22 +224,6 @@
 Serial: 8250/16550 driver, 4 ports, IRQ sharing disabled
 console [ttyS0] disabled
 70006000.serial: ttyS0 at MMIO 0x70006000 (irq = 331, base_baud = 25500000) is a Tegra
-------------[ cut here ]------------
-WARNING: CPU: 2 PID: 0 at kernel/rcu/tree.c:380 rcu_note_context_switch+0x24/0x2c
-Modules linked in:
-CPU: 2 PID: 0 Comm: swapper/2 Not tainted 4.9.0-rc4-next-20161111-133604-g37634f0 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c030fab8>] (unwind_backtrace) from [<c030b104>] (show_stack+0x10/0x14)
-[<c030b104>] (show_stack) from [<c0592150>] (dump_stack+0x84/0xa4)
-[<c0592150>] (dump_stack) from [<c0342be0>] (__warn+0xd4/0x100)
-[<c0342be0>] (__warn) from [<c0342c2c>] (warn_slowpath_null+0x20/0x28)
-[<c0342c2c>] (warn_slowpath_null) from [<c0393b00>] (rcu_note_context_switch+0x24/0x2c)
-[<c0393b00>] (rcu_note_context_switch) from [<c0bbef5c>] (__schedule+0x64/0x500)
-[<c0bbef5c>] (__schedule) from [<c0bbf51c>] (schedule+0x38/0x9c)
-[<c0bbf51c>] (schedule) from [<c0bbf790>] (schedule_preempt_disabled+0xc/0x10)
-[<c0bbf790>] (schedule_preempt_disabled) from [<c037ad94>] (cpu_startup_entry+0x180/0x1f4)
-[<c037ad94>] (cpu_startup_entry) from [<80301c2c>] (0x80301c2c)
----[ end trace 9f5275067b57f46f ]---
 console [ttyS0] enabled
 SuperH (H)SCI(F) driver initialized
 msm_serial: driver initialized
@@ -303,13 +287,13 @@
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
 NET: Registered protocol family 10
+IPv6: Attempt to unregister permanent protocol 6
+IPv6: Attempt to unregister permanent protocol 136
+IPv6: Attempt to unregister permanent protocol 17
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
-IPv6: Attempt to unregister permanent protocol 6
-IPv6: Attempt to unregister permanent protocol 136
-IPv6: Attempt to unregister permanent protocol 17
 mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB
 Primary GPT is invalid, using alternate GPT.
  mmcblk1: p1 p2 p3 p4 p5
@@ -395,7 +379,7 @@
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
-tps65910-rtc tps65910-rtc: setting system clock to 2000-01-01 00:00:03 UTC (946684803)
+tps65910-rtc tps65910-rtc: setting system clock to 2000-01-01 00:00:02 UTC (946684802)
 vdd_sata,avdd_plle: disabling
 usb1_vbus: disabling
 EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
@@ -407,5 +391,5 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
-root@(none):/# [    5.693122] asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0f:67:60
+root@(none):/# [    4.118603] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0f:67:60
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-11-14 01:19:58.121316432 -0800
+++ /tmp/foob	2016-11-14 01:19:58.125316445 -0800
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 #################################################
-	 3.3 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 5480248 (539f38 hex)
+Bytes transferred = 5480000 (539e40 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.9 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 51804 (ca5c hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x539f38 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x539e40 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa5b ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc4-next-20161111-133604-g37634f0 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Nov 10 20:18:45 PST 2016
+Linux version 4.9.0-rc5-next-20161114-133832-gb60de3e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Nov 13 23:49:00 PST 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,7 +68,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0d5e080, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0d5e1c0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -81,7 +81,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999880K/2096128K available (8192K kernel code, 381K rwdata, 2468K rodata, 1024K init, 370K bss, 30712K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1999880K/2096128K available (8192K kernel code, 382K rwdata, 2468K rodata, 1024K init, 370K bss, 30712K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -91,7 +91,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d5f740   ( 382 kB)
+      .data : 0xc0d00000 - 0xc0d5f880   ( 383 kB)
        .bss : 0xc0d61000 - 0xc0dbdb1c   ( 371 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -131,7 +131,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?[    0.503815] hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
+No ATAGs?[    0.504121] hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 4 bytes.
 tegra-mc 7000f000.memory-controller: no memory timings for RAM code 3 registered
 vdd_5v_sata: Failed to request enable GPIO30: -517
@@ -201,7 +201,6 @@
 tegra30-pinctrl 70000868.pinmux: invalid group "vi_vsync_pd6" for function "rsvd1"
 tegra30-pinctrl 70000868.pinmux: invalid group "vi_hsync_pd7" for function "rsvd1"
 tegra30-pinctrl 70000868.pinmux: invalid group "hdmi_int_pn7" for function "rsvd1"
-------------[ cut here ]------------
 tegra30-pinctrl 70000868.pinmux: invalid group "ulpi_data3_po4" for function "rsvd1"
 tegra30-pinctrl 70000868.pinmux: invalid group "vi_d10_pt2" for function "rsvd1"
 tegra30-pinctrl 70000868.pinmux: invalid group "vi_d11_pt3" for function "rsvd1"
@@ -218,21 +217,6 @@
 tegra-pmc 7000e400.pmc: i2c-thermtrip node not found, emergency thermal reset disabled.
 Serial: 8250/16550 driver, 4 ports, IRQ sharing disabled
 console [ttyS0] disabled
-WARNING: CPU: 3 PID: 0 at kernel/rcu/tree.c:380 rcu_note_context_switch+0x268/0x2c8
-Modules linked in:
-CPU: 3 PID: 0 Comm: swapper/3 Not tainted 4.9.0-rc4-next-20161111-133604-g37634f0 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c010f288>] (unwind_backtrace) from [<c010a86c>] (show_stack+0x10/0x14)
-[<c010a86c>] (show_stack) from [<c034677c>] (dump_stack+0x84/0xa4)
-[<c034677c>] (dump_stack) from [<c011f754>] (__warn+0xd4/0x100)
-[<c011f754>] (__warn) from [<c011f7a0>] (warn_slowpath_null+0x20/0x28)
-[<c011f7a0>] (warn_slowpath_null) from [<c0177a48>] (rcu_note_context_switch+0x268/0x2c8)
-[<c0177a48>] (rcu_note_context_switch) from [<c081d9f4>] (__schedule+0x70/0x60c)
-[<c081d9f4>] (__schedule) from [<c081e160>] (schedule+0x50/0xc0)
-[<c081e160>] (schedule) from [<c081e3bc>] (schedule_preempt_disabled+0x24/0x40)
-[<c081e3bc>] (schedule_preempt_disabled) from [<c015ab38>] (cpu_startup_entry+0x180/0x1f4)
-[<c015ab38>] (cpu_startup_entry) from [<801014cc>] (0x801014cc)
----[ end trace 0fc97afa81fa6d27 ]---
 70006000.serial: ttyS0 at MMIO 0x70006000 (irq = 331, base_baud = 25500000) is a Tegra
 console [ttyS0] enabled
 console [ttyS0] enabled
@@ -392,7 +376,7 @@
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
-tps65910-rtc tps65910-rtc: setting system clock to 2000-01-01 00:00:04 UTC (946684804)
+tps65910-rtc tps65910-rtc: setting system clock to 2000-01-01 00:00:03 UTC (946684803)
 vdd_sata,avdd_plle: disabling
 usb1_vbus: disabling
 ALSA device list:
@@ -406,5 +390,5 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
-root@(none):/# [    6.832252] asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0f:67:60
+root@(none):/# [    4.911744] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0f:67:60
