Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2017-11-23 19:21:27.631903956 -0800
+++ /tmp/foob	2017-11-23 19:21:27.631903956 -0800
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0000000000 [0x411fd070]
-Linux version 4.14.0-next-20171123-334169-g279a24799865 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Nov 22 18:55:59 PST 2017
+Linux version 4.14.0-next-20171124-334250-g6fc478f80f68 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Thu Nov 23 19:11:09 PST 2017
 Machine model: linux,dummy-virt
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -182,7 +182,7 @@
 registered taskstats version 1
 Loading compiled-in X.509 certificates
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2017-11-23 03:02:54 UTC (1511406174)
+rtc-pl031 9010000.pl031: setting system clock to 2017-11-24 03:17:56 UTC (1511493476)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-11-23 19:21:27.663904099 -0800
+++ /tmp/foob	2017-11-23 19:21:27.663904099 -0800
@@ -40,7 +40,7 @@
 	 #################################################################
 	 #################################################################
 	 ###############################
-	 3 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 8086016 (7b6200 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
@@ -49,7 +49,7 @@
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.5 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 34502 (86c6 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-next-20171123-334169-g279a24799865 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Nov 22 18:49:34 PST 2017
+Linux version 4.14.0-next-20171124-334250-g6fc478f80f68 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Nov 23 19:04:53 PST 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -372,11 +372,11 @@
 mmc0: new high speed SDHC card at address aaaa
 usb1_vbus: supplied by vdd-sys-5v0
 usb2_vbus: supplied by vdd-sys-5v0
-mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
- mmcblk0: p1 p2 p3
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
+mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
+ mmcblk0: p1 p2 p3
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
@@ -393,7 +393,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-04-15 09:50:56 UTC (955792256)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-04-16 10:06:37 UTC (955879597)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -405,5 +405,4 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.928976] usb 1-1: new high-speed USB device number 2 using tegra-ehci
-asix 1-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0d:b5:16
+root@(none):/# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2017-11-23 19:21:27.675904153 -0800
+++ /tmp/foob	2017-11-23 19:21:27.679904170 -0800
@@ -40,18 +40,18 @@
 	 ################################
 	 3.2 MiB/s
 done
-Bytes transferred = 6192624 (5e7df0 hex)
+Bytes transferred = 6192952 (5e7f38 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.7 MiB/s
+	 2.5 MiB/s
 done
 Bytes transferred = 34502 (86c6 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5e7df0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5e7f38 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff6c5 ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-next-20171123-14220-g279a247 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Nov 22 18:47:53 PST 2017
+Linux version 4.14.0-next-20171124-14301-g6fc478f (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Nov 23 19:03:26 PST 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -388,7 +388,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-04-15 09:48:12 UTC (955792092)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-04-16 10:03:51 UTC (955879431)
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
@@ -406,4 +406,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.171198] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.171428] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-11-23 19:21:27.687904204 -0800
+++ /tmp/foob	2017-11-23 19:21:27.691904223 -0800
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -35,7 +35,7 @@
 	 #################################################################
 	 #################################################################
 	 ###############################
-	 3.8 MiB/s
+	 4 MiB/s
 done
 Bytes transferred = 8086016 (7b6200 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
@@ -44,7 +44,7 @@
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 1.2 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 66544 (103f0 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
@@ -56,7 +56,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-next-20171123-334169-g279a24799865 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Nov 22 18:49:34 PST 2017
+Linux version 4.14.0-next-20171124-334250-g6fc478f80f68 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Nov 23 19:04:53 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -397,6 +397,7 @@
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
 tegra-pcie 1003000.pcie: 2x1, 1x1 configuration
 tegra-pcie 1003000.pcie: Failed to get supply 'avddio-pex': -517
+mmc0: new high speed MMC card at address 0001
 tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
 as3722-sd2: supplied by +5V_SYS
@@ -418,14 +419,8 @@
 +2.8V_RUN_CAM_AF: Bringing 0uV into 2800000-2800000uV
 as3722-ldo11: supplied by +3.3V_RUN
 +1.8V_RUN_VPP_FUSE: Bringing 0uV into 1800000-1800000uV
-mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB, chardev (237:0)
 tegra-ahci 70027000.sata: AHCI 0001.0301 32 slots 2 ports 3 Gbps 0x1 impl platform mode
 tegra-ahci 70027000.sata: flags: 64bit ncq sntf pm led pio slum part sadm sds apst 
- mmcblk0: p1
 scsi host0: tegra-ahci
 scsi host1: tegra-ahci
 ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 345
@@ -459,7 +454,6 @@
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x52 may corrupt adjacent RW1C bits
 pci 0000:00:01.0: PME# supported from D0 D1 D2 D3hot D3cold
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x4c may corrupt adjacent RW1C bits
-ata1: SATA link down (SStatus 0 SControl 300)
 pci 0000:00:02.0: [10de:0e13] type 01 class 0x060400
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4 may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4 may corrupt adjacent RW1C bits
@@ -504,23 +498,18 @@
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
 r8169 0000:02:00.0 (unnamed net_device) (uninitialized): not PCI Express
-mmc1: new high speed SDHC card at address aaaa
-mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
-r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0a77000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 22
+r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0a6d000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 22
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
- mmcblk1: p1 p2 p3
-tegra-hdmi 54280000.hdmi: cannot set audio to 48000 Hz at 297000000 Hz pixel clock
-Console: switching to colour frame buffer device 512x135
-drm drm: fb0:  frame buffer device
+[drm] Cannot find any crtc or sizes
 [drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
 iommu: Adding device cpufreq-dt.0 to group 74
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:14 UTC (946684814)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:04:22 UTC (946685062)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
@@ -529,12 +518,24 @@
 +3.3V_RUN_TOUCH: disabling
 +2.8V_RUN_CAM_AF: disabling
 +1.8V_RUN_VPP_FUSE: disabling
+ata1: SATA link down (SStatus 0 SControl 300)
+mmc1: new high speed SDHC card at address aaaa
+mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
+ mmcblk1: p1 p2 p3
+tegra-hdmi 54280000.hdmi: cannot set audio to 48000 Hz at 297000000 Hz pixel clock
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB, chardev (237:0)
+ mmcblk0: p1
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:1.
+VFS: Mounted root (ext3 filesystem) on device 179:17.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K
+Console: switching to colour frame buffer device 512x135
+drm drm: fb0:  frame buffer device
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2017-11-23 19:21:27.727904382 -0800
+++ /tmp/foob	2017-11-23 19:21:27.731904401 -0800
@@ -33,20 +33,20 @@
 	 #################################################################
 	 #################################################################
 	 ################################
-	 943.4 KiB/s
+	 4 MiB/s
 done
-Bytes transferred = 6192624 (5e7df0 hex)
+Bytes transferred = 6192952 (5e7f38 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 927.7 KiB/s
+	 3.7 MiB/s
 done
 Bytes transferred = 66544 (103f0 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5e7df0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5e7f38 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830133ef
@@ -54,7 +54,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-next-20171123-14220-g279a247 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Nov 22 18:47:53 PST 2017
+Linux version 4.14.0-next-20171124-14301-g6fc478f (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Nov 23 19:03:26 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -476,9 +476,9 @@
 PCI: bus1: Fast back to back transfers disabled
 pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
 pci 0000:02:00.0: [10ec:8168] type 00 class 0x020000
+pci 0000:02:00.0: reg 0x10: [io  0x0000-0x00ff]
 mmc1: new high speed SDHC card at address aaaa
 mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
-pci 0000:02:00.0: reg 0x10: [io  0x0000-0x00ff]
 pci 0000:02:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit]
 pci 0000:02:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
 PCI: bus2: Fast back to back transfers disabled
@@ -497,8 +497,8 @@
 pci 0000:00:02.0:   bridge window [io  0x1000-0x1fff]
 pci 0000:00:02.0:   bridge window [mem 0x13100000-0x131fffff]
 pci 0000:00:02.0:   bridge window [mem 0x20000000-0x200fffff 64bit pref]
- mmcblk1: p1 p2 p3
 pci 0000:00:01.0: nv_msi_ht_cap_quirk didn't locate host bridge
+ mmcblk1: p1 p2 p3
 pcieport 0000:00:01.0: enabling device (0140 -> 0142)
 pcieport 0000:00:01.0: Signaling PME with IRQ 395
 pci 0000:00:02.0: nv_msi_ht_cap_quirk didn't locate host bridge
@@ -520,7 +520,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:19 UTC (946684819)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:14 UTC (946684814)
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig%tegra-fw_log.txt
--- /tmp/fooa	2017-11-23 19:21:27.759904526 -0800
+++ /tmp/foob	2017-11-23 19:21:27.763904543 -0800
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MMC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -32,21 +32,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ###########################################
-	 3.5 MiB/s
+	 ############################################
+	 3.6 MiB/s
 done
-Bytes transferred = 6356208 (60fcf0 hex)
+Bytes transferred = 6356480 (60fe00 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.5 MiB/s
+	 429.7 KiB/s
 done
 Bytes transferred = 66544 (103f0 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x60fcf0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x60fe00 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830133ef
@@ -54,7 +54,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-next-20171123-215928-g279a247 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Nov 22 18:48:21 PST 2017
+Linux version 4.14.0-next-20171124-216009-g6fc478f (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Nov 23 19:03:51 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -451,8 +451,8 @@
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
 tegra-pcie 1003000.pcie: 2x1, 1x1 configuration
 tegra-pcie 1003000.pcie: probing port 0, using 2 lanes
-tegra-pcie 1003000.pcie: Slot present pin change, signature: 00000008
 ata1: SATA link down (SStatus 0 SControl 300)
+tegra-pcie 1003000.pcie: Slot present pin change, signature: 00000008
 tegra-pcie 1003000.pcie: probing port 1, using 1 lanes
 tegra-pcie 1003000.pcie: Slot present pin change, signature: 00000000
 tegra-pcie 1003000.pcie: PCI host bridge to bus 0000:00
@@ -484,14 +484,14 @@
 PCI: bus1: Fast back to back transfers disabled
 pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
 pci 0000:02:00.0: [10ec:8168] type 00 class 0x020000
+mmc1: new high speed SDHC card at address aaaa
+mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 pci 0000:02:00.0: reg 0x10: [io  0x0000-0x00ff]
 pci 0000:02:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit]
 pci 0000:02:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
 PCI: bus2: Fast back to back transfers disabled
 pci_bus 0000:02: busn_res: [bus 02-ff] end is updated to 02
 pci 0000:00:01.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
-mmc1: new high speed SDHC card at address aaaa
-mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 pci 0000:00:02.0: BAR 8: assigned [mem 0x13100000-0x131fffff]
 pci 0000:00:02.0: BAR 9: assigned [mem 0x20000000-0x200fffff 64bit pref]
 pci 0000:00:02.0: BAR 7: assigned [io  0x1000-0x1fff]
@@ -504,12 +504,12 @@
 pci 0000:00:02.0: PCI bridge to [bus 02]
 pci 0000:00:02.0:   bridge window [io  0x1000-0x1fff]
 pci 0000:00:02.0:   bridge window [mem 0x13100000-0x131fffff]
+ mmcblk1: p1 p2 p3
 pci 0000:00:02.0:   bridge window [mem 0x20000000-0x200fffff 64bit pref]
 pci 0000:00:01.0: nv_msi_ht_cap_quirk didn't locate host bridge
 pcieport 0000:00:01.0: enabling device (0140 -> 0142)
 pcieport 0000:00:01.0: Signaling PME with IRQ 395
 pci 0000:00:02.0: nv_msi_ht_cap_quirk didn't locate host bridge
- mmcblk1: p1 p2 p3
 pcieport 0000:00:02.0: enabling device (0140 -> 0143)
 pcieport 0000:00:02.0: Signaling PME with IRQ 396
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
@@ -528,7 +528,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:08 UTC (946684928)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:10 UTC (946684930)
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-11-23 19:21:27.787904649 -0800
+++ /tmp/foob	2017-11-23 19:21:27.791904668 -0800
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -43,7 +43,7 @@
 	 #################################################################
 	 #################################################################
 	 ###############################
-	 3.6 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 8086016 (7b6200 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
@@ -64,7 +64,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-next-20171123-334169-g279a24799865 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Nov 22 18:49:34 PST 2017
+Linux version 4.14.0-next-20171124-334250-g6fc478f80f68 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Nov 23 19:04:53 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2017-11-23 19:21:27.827904827 -0800
+++ /tmp/foob	2017-11-23 19:21:27.827904827 -0800
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -41,20 +41,20 @@
 	 #################################################################
 	 #################################################################
 	 ################################
-	 3.6 MiB/s
+	 3.5 MiB/s
 done
-Bytes transferred = 6192624 (5e7df0 hex)
+Bytes transferred = 6192952 (5e7f38 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.4 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 67482 (1079a hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5e7df0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5e7f38 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013799
@@ -62,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-next-20171123-14220-g279a247 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Nov 22 18:47:53 PST 2017
+Linux version 4.14.0-next-20171124-14301-g6fc478f (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Nov 23 19:03:26 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -312,90 +312,90 @@
 kernel BUG at drivers/platform/chrome/cros_ec_proto.c:34!
 Internal error: Oops - BUG: 0 [#1] PREEMPT SMP ARM
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.14.0-next-20171123-14220-g279a247 #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.14.0-next-20171124-14301-g6fc478f #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 task: ee890740 task.stack: ee8a6000
 PC is at cros_ec_prepare_tx+0x168/0x16c
 LR is at cros_ec_pkt_xfer_spi+0x18/0x474
-pc : [<c06ebf20>]    lr : [<c0559068>]    psr: 20000013
+pc : [<c06ec100>]    lr : [<c0559124>]    psr: 20000013
 sp : ee8a7bb8  ip : 00000000  fp : 00000000
-r10: c0559050  r9 : 00000000  r8 : eefe7fb4
-r7 : ee2aaac0  r6 : ee2aaac0  r5 : ee2aaac0  r4 : ee28ee50
-r3 : 0000ffff  r2 : 00000000  r1 : 0000ffff  r0 : ee28ee50
+r10: c055910c  r9 : 00000000  r8 : eefe7fb4
+r7 : ee28cac0  r6 : ee28cac0  r5 : ee28cac0  r4 : ee270e50
+r3 : 0000ffff  r2 : 00000000  r1 : 0000ffff  r0 : ee270e50
 Flags: nzCv  IRQs on  FIQs on  Mode SVC_32  ISA ARM  Segment none
 Control: 10c5387d  Table: 8000406a  DAC: 00000051
 Process swapper/0 (pid: 1, stack limit = 0xee8a6210)
 Stack: (0xee8a7bb8 to 0xee8a8000)
-7ba0:                                                       ee28ee50 ee2aaac0
-7bc0: ee2aab50 c0559068 c0995a7c 00000004 ee8a7c3c ee8a7c3c ee2a9800 00000000
-7be0: c05a5d74 00000000 00000008 00000008 00000000 ee8a7bf4 ee8a7bf4 00000000
-7c00: ee8a7c00 ee8a7c00 ee2aab10 ee2aaa40 00000008 00000000 00000000 00000000
+7ba0:                                                       ee270e50 ee28cac0
+7bc0: ee28cb50 c0559124 c099613c 00000004 ee8a7c3c ee8a7c3c ee28b800 00000000
+7be0: c05a5e30 00000000 00000008 00000008 00000000 ee8a7bf4 ee8a7bf4 00000000
+7c00: ee8a7c00 ee8a7c00 ee28cb10 ee28ca40 00000008 00000000 00000000 00000000
 7c20: c0d91bd0 c023c0ec 40002800 00000540 eef94bd0 c023e550 002dc6c0 ee8a7bd0
-7c40: ee8a7bd0 00000000 c06ebd50 005c5540 3b9aca00 000f4240 00000000 ee8a7c88
-7c60: 00000001 00000501 c0559050 c099a418 00000000 ee2aaa80 a0000013 c06ebd50
-7c80: ee2aaac0 ee28ee50 ee2aaac0 ee2a9800 ee2aaac0 eefe7fb4 00000000 c0559050
-7ca0: 00000000 c06ebcac 0000400b ee28ee50 ee2aaac0 ee2a9800 0000000c eefe7fb4
-7cc0: c0eecefc c06ec0c8 00000000 ee28ee50 ee2a9800 00000000 00000000 eefe7fb4
-7ce0: c0e6c3c8 c0558af4 ee8a7cf4 ee2a9800 00000000 00000001 ee2a9800 ee28ee50
-7d00: ee2aab50 c0558ea4 00000000 00000000 00000000 000007d0 ee2a9800 00000000
-7d20: c0e6c3b8 c0efe1a8 00000000 c05a67b8 ee2a9800 c0efe1a4 00000000 c05347b8
-7d40: 00000000 ee8a7d78 c0534964 00000001 ee034000 00000000 c0efe180 c0532dc0
-7d60: ee9f1170 ee22e5b8 ee2a9800 ee2a9834 c0e72ef4 c05344e0 ee2a9800 00000001
-7d80: 00000000 ee2a9800 ee2a9800 c0e72ef4 c0e04c48 c0533b24 ee2a9800 ee2a9808
-7da0: 00000000 c0532160 40400000 00000000 ee034358 c05a97b4 ee2a9800 e416bf4f
-7dc0: 00000000 ee2a9800 ee034000 00000000 eea8be10 eefe8004 00000001 00000000
-7de0: 00000000 c05a660c 002dc6c0 ee034000 eefe7fb4 00000000 ee2a9800 c05a85ac
-7e00: 00000000 014000c0 00000000 c05a7f98 00000063 002dc6c0 ee034000 ee2aa610
-7e20: eea8be00 eea8be10 40000000 00000153 00000000 c05a8ad0 eefe7d0c ee034350
-7e40: ee034000 eea8be00 eea8be10 c05a965c 00000000 ee034350 ffffffed eea8be10
-7e60: fffffdfb c0e73850 00000000 00000000 c0e73850 c05360d0 c0536080 eea8be10
-7e80: c0efe1a4 00000000 c0efe1a8 c05347b8 eea8be10 c0e73850 eea8be44 00000000
-7ea0: 00000000 c0d6b83c c0d00594 c053492c 00000000 c0e73850 c0534880 c0532e5c
-7ec0: ee8cc55c eea14934 c0e73850 ee2a5e80 c0e6a9b8 c0533cf8 c0be09c0 c0e73850
-7ee0: 00000006 c0e73850 00000006 c0e04c48 c0d387f0 c05354ac c0535d24 c0d8434c
-7f00: 00000006 c0101bbc 00000000 ee8a7f24 00000000 c0161e00 00000000 c0c7f270
-7f20: 0000010e efffcc45 c0c7f270 0000010e efffcc53 c013ecb4 c0c7dd58 00000000
+7c40: ee8a7bd0 00000000 c06ebf30 005c5180 3b9aca00 000f4240 00000000 ee8a7c88
+7c60: 00000001 00000501 c055910c c099aad8 00000000 ee28ca80 a0000013 c06ebf30
+7c80: ee28cac0 ee270e50 ee28cac0 ee28b800 ee28cac0 eefe7fb4 00000000 c055910c
+7ca0: 00000000 c06ebe8c 0000400b ee270e50 ee28cac0 ee28b800 0000000c eefe7fb4
+7cc0: c0eecefc c06ec2a8 00000000 ee270e50 ee28b800 00000000 00000000 eefe7fb4
+7ce0: c0e6c3c8 c0558bb0 ee8a7cf4 ee28b800 00000000 00000001 ee28b800 ee270e50
+7d00: ee28cb50 c0558f60 00000000 00000000 00000000 000007d0 ee28b800 00000000
+7d20: c0e6c3b8 c0efe1a8 00000000 c05a6874 ee28b800 c0efe1a4 00000000 c0534874
+7d40: 00000000 ee8a7d78 c0534a20 00000001 ee03a000 00000000 c0efe180 c0532e7c
+7d60: ee9f1170 ee2605b8 ee28b800 ee28b834 c0e72ef4 c053459c ee28b800 00000001
+7d80: 00000000 ee28b800 ee28b800 c0e72ef4 c0e04c48 c0533be0 ee28b800 ee28b808
+7da0: 00000000 c053221c 40400000 00000000 ee03a358 c05a9870 ee28b800 e416bf4f
+7dc0: 00000000 ee28b800 ee03a000 00000000 eea8be10 eefe8004 00000001 00000000
+7de0: 00000000 c05a66c8 002dc6c0 ee03a000 eefe7fb4 00000000 ee28b800 c05a8668
+7e00: 00000000 014000c0 00000000 c05a8054 00000063 002dc6c0 ee03a000 ee28c610
+7e20: eea8be00 eea8be10 40000000 00000153 00000000 c05a8b8c eefe7d0c ee03a350
+7e40: ee03a000 eea8be00 eea8be10 c05a9718 00000000 ee03a350 ffffffed eea8be10
+7e60: fffffdfb c0e73850 00000000 00000000 c0e73850 c053618c c053613c eea8be10
+7e80: c0efe1a4 00000000 c0efe1a8 c0534874 eea8be10 c0e73850 eea8be44 00000000
+7ea0: 00000000 c0d6b83c c0d00594 c05349e8 00000000 c0e73850 c053493c c0532f18
+7ec0: ee8cc55c eea14934 c0e73850 ee287e80 c0e6a9b8 c0533db4 c0be0a38 c0e73850
+7ee0: 00000006 c0e73850 00000006 c0e04c48 c0d387f0 c0535568 c0535de0 c0d8434c
+7f00: 00000006 c0101bbc 00000000 ee8a7f24 00000000 c0161e00 00000000 c0c7f290
+7f20: 0000010e efffcc45 c0c7f290 0000010e efffcc53 c013ecb4 c0c7dd78 00000000
 7f40: 00000006 00000006 efffcc45 efffcc4a 00000000 e416bf4f 00000006 c0d8434c
 7f60: 00000006 c0d6b830 c0ec2880 0000010e c0d00594 c0d00dd0 00000006 00000006
-7f80: 00000000 c0d00594 00000000 c099342c 00000000 00000000 00000000 00000000
-7fa0: 00000000 c0993434 00000000 c0107d70 00000000 00000000 00000000 00000000
+7f80: 00000000 c0d00594 00000000 c0993aec 00000000 00000000 00000000 00000000
+7fa0: 00000000 c0993af4 00000000 c0107d70 00000000 00000000 00000000 00000000
 7fc0: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
 7fe0: 00000000 00000000 00000000 00000000 00000013 00000000 00000000 00000000
-[<c06ebf20>] (cros_ec_prepare_tx) from [<c0559068>] (cros_ec_pkt_xfer_spi+0x18/0x474)
-[<c0559068>] (cros_ec_pkt_xfer_spi) from [<c06ebcac>] (send_command+0x20/0xd8)
-[<c06ebcac>] (send_command) from [<c06ec0c8>] (cros_ec_query_all+0x1a4/0x398)
-[<c06ec0c8>] (cros_ec_query_all) from [<c0558af4>] (cros_ec_register+0x94/0x21c)
-[<c0558af4>] (cros_ec_register) from [<c0558ea4>] (cros_ec_spi_probe+0x108/0x150)
-[<c0558ea4>] (cros_ec_spi_probe) from [<c05a67b8>] (spi_drv_probe+0x7c/0xa8)
-[<c05a67b8>] (spi_drv_probe) from [<c05347b8>] (driver_probe_device+0x250/0x318)
-[<c05347b8>] (driver_probe_device) from [<c0532dc0>] (bus_for_each_drv+0x44/0x8c)
-[<c0532dc0>] (bus_for_each_drv) from [<c05344e0>] (__device_attach+0x9c/0x100)
-[<c05344e0>] (__device_attach) from [<c0533b24>] (bus_probe_device+0x84/0x8c)
-[<c0533b24>] (bus_probe_device) from [<c0532160>] (device_add+0x39c/0x54c)
-[<c0532160>] (device_add) from [<c05a660c>] (spi_add_device+0x88/0x130)
-[<c05a660c>] (spi_add_device) from [<c05a85ac>] (spi_register_controller+0x428/0x918)
-[<c05a85ac>] (spi_register_controller) from [<c05a8ad0>] (devm_spi_register_controller+0x34/0x70)
-[<c05a8ad0>] (devm_spi_register_controller) from [<c05a965c>] (tegra_spi_probe+0x298/0x35c)
-[<c05a965c>] (tegra_spi_probe) from [<c05360d0>] (platform_drv_probe+0x50/0xb0)
-[<c05360d0>] (platform_drv_probe) from [<c05347b8>] (driver_probe_device+0x250/0x318)
-[<c05347b8>] (driver_probe_device) from [<c053492c>] (__driver_attach+0xac/0xb0)
-[<c053492c>] (__driver_attach) from [<c0532e5c>] (bus_for_each_dev+0x54/0x88)
-[<c0532e5c>] (bus_for_each_dev) from [<c0533cf8>] (bus_add_driver+0xe4/0x1f0)
-[<c0533cf8>] (bus_add_driver) from [<c05354ac>] (driver_register+0x78/0xf4)
-[<c05354ac>] (driver_register) from [<c0101bbc>] (do_one_initcall+0x48/0x18c)
+[<c06ec100>] (cros_ec_prepare_tx) from [<c0559124>] (cros_ec_pkt_xfer_spi+0x18/0x474)
+[<c0559124>] (cros_ec_pkt_xfer_spi) from [<c06ebe8c>] (send_command+0x20/0xd8)
+[<c06ebe8c>] (send_command) from [<c06ec2a8>] (cros_ec_query_all+0x1a4/0x398)
+[<c06ec2a8>] (cros_ec_query_all) from [<c0558bb0>] (cros_ec_register+0x94/0x21c)
+[<c0558bb0>] (cros_ec_register) from [<c0558f60>] (cros_ec_spi_probe+0x108/0x150)
+[<c0558f60>] (cros_ec_spi_probe) from [<c05a6874>] (spi_drv_probe+0x7c/0xa8)
+[<c05a6874>] (spi_drv_probe) from [<c0534874>] (driver_probe_device+0x250/0x318)
+[<c0534874>] (driver_probe_device) from [<c0532e7c>] (bus_for_each_drv+0x44/0x8c)
+[<c0532e7c>] (bus_for_each_drv) from [<c053459c>] (__device_attach+0x9c/0x100)
+[<c053459c>] (__device_attach) from [<c0533be0>] (bus_probe_device+0x84/0x8c)
+[<c0533be0>] (bus_probe_device) from [<c053221c>] (device_add+0x39c/0x54c)
+[<c053221c>] (device_add) from [<c05a66c8>] (spi_add_device+0x88/0x130)
+[<c05a66c8>] (spi_add_device) from [<c05a8668>] (spi_register_controller+0x428/0x918)
+[<c05a8668>] (spi_register_controller) from [<c05a8b8c>] (devm_spi_register_controller+0x34/0x70)
+[<c05a8b8c>] (devm_spi_register_controller) from [<c05a9718>] (tegra_spi_probe+0x298/0x35c)
+[<c05a9718>] (tegra_spi_probe) from [<c053618c>] (platform_drv_probe+0x50/0xb0)
+[<c053618c>] (platform_drv_probe) from [<c0534874>] (driver_probe_device+0x250/0x318)
+[<c0534874>] (driver_probe_device) from [<c05349e8>] (__driver_attach+0xac/0xb0)
+[<c05349e8>] (__driver_attach) from [<c0532f18>] (bus_for_each_dev+0x54/0x88)
+[<c0532f18>] (bus_for_each_dev) from [<c0533db4>] (bus_add_driver+0xe4/0x1f0)
+[<c0533db4>] (bus_add_driver) from [<c0535568>] (driver_register+0x78/0xf4)
+[<c0535568>] (driver_register) from [<c0101bbc>] (do_one_initcall+0x48/0x18c)
 [<c0101bbc>] (do_one_initcall) from [<c0d00dd0>] (kernel_init_freeable+0x10c/0x1d0)
-[<c0d00dd0>] (kernel_init_freeable) from [<c0993434>] (kernel_init+0x8/0x114)
-[<c0993434>] (kernel_init) from [<c0107d70>] (ret_from_fork+0x14/0x24)
+[<c0d00dd0>] (kernel_init_freeable) from [<c0993af4>] (kernel_init+0x8/0x114)
+[<c0993af4>] (kernel_init) from [<c0107d70>] (ret_from_fork+0x14/0x24)
 Code: e2800008 e8bd8070 e7f001f2 e7f001f2 (e7f001f2) 
----[ end trace 289aef78feed09f9 ]---
+---[ end trace dd9bdaf6d0e24c22 ]---
 Kernel panic - not syncing: Attempted to kill init! exitcode=0x0000000b
 
 CPU3: stopping
-CPU: 3 PID: 0 Comm: swapper/3 Tainted: G      D          4.14.0-next-20171123-14220-g279a247 #1
+CPU: 3 PID: 0 Comm: swapper/3 Tainted: G      D          4.14.0-next-20171124-14301-g6fc478f #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c0111230>] (unwind_backtrace) from [<c010c168>] (show_stack+0x10/0x14)
-[<c010c168>] (show_stack) from [<c097f93c>] (dump_stack+0x84/0xa4)
-[<c097f93c>] (dump_stack) from [<c010f71c>] (handle_IPI+0x18c/0x39c)
+[<c010c168>] (show_stack) from [<c097fffc>] (dump_stack+0x84/0xa4)
+[<c097fffc>] (dump_stack) from [<c010f71c>] (handle_IPI+0x18c/0x39c)
 [<c010f71c>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cd4c>] (__irq_svc+0x6c/0xa8)
 Exception stack(0xee8e3f70 to 0xee8e3fb8)
@@ -406,34 +406,34 @@
 [<c01087dc>] (arch_cpu_idle) from [<c016290c>] (do_idle+0x140/0x204)
 [<c016290c>] (do_idle) from [<c0162c58>] (cpu_startup_entry+0x18/0x1c)
 [<c0162c58>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
-CPU2: stopping
-CPU: 2 PID: 0 Comm: swapper/2 Tainted: G      D          4.14.0-next-20171123-14220-g279a247 #1
+CPU1: stopping
+CPU: 1 PID: 0 Comm: swapper/1 Tainted: G      D          4.14.0-next-20171124-14301-g6fc478f #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c0111230>] (unwind_backtrace) from [<c010c168>] (show_stack+0x10/0x14)
-[<c010c168>] (show_stack) from [<c097f93c>] (dump_stack+0x84/0xa4)
-[<c097f93c>] (dump_stack) from [<c010f71c>] (handle_IPI+0x18c/0x39c)
+[<c010c168>] (show_stack) from [<c097fffc>] (dump_stack+0x84/0xa4)
+[<c097fffc>] (dump_stack) from [<c010f71c>] (handle_IPI+0x18c/0x39c)
 [<c010f71c>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cd4c>] (__irq_svc+0x6c/0xa8)
-Exception stack(0xee8e1f70 to 0xee8e1fb8)
-1f60:                                     00000001 00000000 ee8e1fc8 c011a560
-1f80: 00000004 c0e04ce0 00000000 c0e04c6c 00000000 c0d8d2b8 00000000 c0e04cf0
-1fa0: 2e225000 ee8e1fc0 c01087d8 c01087dc 60000013 ffffffff
+Exception stack(0xee8c1f70 to 0xee8c1fb8)
+1f60:                                     00000001 00000000 ee8c1fc8 c011a560
+1f80: 00000002 c0e04ce0 00000000 c0e04c6c 00000000 c0d8d2b8 00000000 c0e04cf0
+1fa0: 2e214000 ee8c1fc0 c01087d8 c01087dc 60000013 ffffffff
 [<c010cd4c>] (__irq_svc) from [<c01087dc>] (arch_cpu_idle+0x30/0x3c)
 [<c01087dc>] (arch_cpu_idle) from [<c016290c>] (do_idle+0x140/0x204)
 [<c016290c>] (do_idle) from [<c0162c58>] (cpu_startup_entry+0x18/0x1c)
 [<c0162c58>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
-CPU1: stopping
-CPU: 1 PID: 0 Comm: swapper/1 Tainted: G      D          4.14.0-next-20171123-14220-g279a247 #1
+CPU2: stopping
+CPU: 2 PID: 0 Comm: swapper/2 Tainted: G      D          4.14.0-next-20171124-14301-g6fc478f #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c0111230>] (unwind_backtrace) from [<c010c168>] (show_stack+0x10/0x14)
-[<c010c168>] (show_stack) from [<c097f93c>] (dump_stack+0x84/0xa4)
-[<c097f93c>] (dump_stack) from [<c010f71c>] (handle_IPI+0x18c/0x39c)
+[<c010c168>] (show_stack) from [<c097fffc>] (dump_stack+0x84/0xa4)
+[<c097fffc>] (dump_stack) from [<c010f71c>] (handle_IPI+0x18c/0x39c)
 [<c010f71c>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cd4c>] (__irq_svc+0x6c/0xa8)
-Exception stack(0xee8c1f70 to 0xee8c1fb8)
-1f60:                                     00000001 00000000 ee8c1fc8 c011a560
-1f80: 00000002 c0e04ce0 00000000 c0e04c6c 00000000 c0d8d2b8 00000000 c0e04cf0
-1fa0: 2e214000 ee8c1fc0 c01087d8 c01087dc 60000013 ffffffff
+Exception stack(0xee8e1f70 to 0xee8e1fb8)
+1f60:                                     00000001 00000000 ee8e1fc8 c011a560
+1f80: 00000004 c0e04ce0 00000000 c0e04c6c 00000000 c0d8d2b8 00000000 c0e04cf0
+1fa0: 2e225000 ee8e1fc0 c01087d8 c01087dc 60000013 ffffffff
 [<c010cd4c>] (__irq_svc) from [<c01087dc>] (arch_cpu_idle+0x30/0x3c)
 [<c01087dc>] (arch_cpu_idle) from [<c016290c>] (do_idle+0x140/0x204)
 [<c016290c>] (do_idle) from [<c0162c58>] (cpu_startup_entry+0x18/0x1c)
@@ -443,11 +443,11 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 8 at kernel/irq_work.c:68 irq_work_queue_on+0xbc/0xfc
 Modules linked in:
-CPU: 0 PID: 8 Comm: rcu_preempt Tainted: G      D          4.14.0-next-20171123-14220-g279a247 #1
+CPU: 0 PID: 8 Comm: rcu_preempt Tainted: G      D          4.14.0-next-20171124-14301-g6fc478f #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c0111230>] (unwind_backtrace) from [<c010c168>] (show_stack+0x10/0x14)
-[<c010c168>] (show_stack) from [<c097f93c>] (dump_stack+0x84/0xa4)
-[<c097f93c>] (dump_stack) from [<c0121c20>] (__warn+0xe4/0x110)
+[<c010c168>] (show_stack) from [<c097fffc>] (dump_stack+0x84/0xa4)
+[<c097fffc>] (dump_stack) from [<c0121c20>] (__warn+0xe4/0x110)
 [<c0121c20>] (__warn) from [<c0121c90>] (warn_slowpath_null+0x44/0x4c)
 [<c0121c90>] (warn_slowpath_null) from [<c01d7610>] (irq_work_queue_on+0xbc/0xfc)
 [<c01d7610>] (irq_work_queue_on) from [<c01841b8>] (rcu_implicit_dynticks_qs+0x280/0x2a0)
@@ -455,4 +455,4 @@
 [<c018336c>] (force_qs_rnp) from [<c0183940>] (rcu_gp_kthread+0x534/0xa58)
 [<c0183940>] (rcu_gp_kthread) from [<c013f55c>] (kthread+0x11c/0x154)
 [<c013f55c>] (kthread) from [<c0107d70>] (ret_from_fork+0x14/0x24)
----[ end trace 289aef78feed09fa ]---
+---[ end trace dd9bdaf6d0e24c23 ]---
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig%tegra-fw_log.txt
--- /tmp/fooa	2017-11-23 19:21:27.835904863 -0800
+++ /tmp/foob	2017-11-23 19:21:27.839904882 -0800
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -40,21 +40,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ###########################################
+	 ############################################
 	 3.5 MiB/s
 done
-Bytes transferred = 6356208 (60fcf0 hex)
+Bytes transferred = 6356480 (60fe00 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.4 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 67482 (1079a hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x60fcf0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x60fe00 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013799
@@ -62,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-next-20171123-215928-g279a247 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Nov 22 18:48:21 PST 2017
+Linux version 4.14.0-next-20171124-216009-g6fc478f (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Nov 23 19:03:51 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -312,14 +312,14 @@
 kernel BUG at drivers/platform/chrome/cros_ec_proto.c:34!
 Internal error: Oops - BUG: 0 [#1] PREEMPT SMP ARM
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.14.0-next-20171123-215928-g279a247 #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.14.0-next-20171124-216009-g6fc478f #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 task: ee890740 task.stack: ee8a6000
 PC is at cros_ec_prepare_tx+0x168/0x16c
 LR is at cros_ec_pkt_xfer_spi+0x18/0x474
-pc : [<c06ebf20>]    lr : [<c0559068>]    psr: 20000013
+pc : [<c06ec100>]    lr : [<c0559124>]    psr: 20000013
 sp : ee8a7bb8  ip : 00000000  fp : 00000000
-r10: c0559050  r9 : 00000000  r8 : eefe7fb4
+r10: c055910c  r9 : 00000000  r8 : eefe7fb4
 r7 : ee28cac0  r6 : ee28cac0  r5 : ee28cac0  r4 : ee270e50
 r3 : 0000ffff  r2 : 00000000  r1 : 0000ffff  r0 : ee270e50
 Flags: nzCv  IRQs on  FIQs on  Mode SVC_32  ISA ARM  Segment none
@@ -327,91 +327,75 @@
 Process swapper/0 (pid: 1, stack limit = 0xee8a6210)
 Stack: (0xee8a7bb8 to 0xee8a8000)
 7ba0:                                                       ee270e50 ee28cac0
-7bc0: ee28cb50 c0559068 c0995a7c 00000004 ee8a7c3c ee8a7c3c ee28b800 00000000
-7be0: c05a5d74 00000000 00000008 00000008 00000000 ee8a7bf4 ee8a7bf4 00000000
+7bc0: ee28cb50 c0559124 c099613c 00000004 ee8a7c3c ee8a7c3c ee28b800 00000000
+7be0: c05a5e30 00000000 00000008 00000008 00000000 ee8a7bf4 ee8a7bf4 00000000
 7c00: ee8a7c00 ee8a7c00 ee28cb10 ee28ca40 00000008 00000000 00000000 00000000
 7c20: c0e91bd0 c023c0ec 40002800 00000540 eef94bd0 c023e550 002dc6c0 ee8a7bd0
-7c40: ee8a7bd0 00000000 c06ebd50 005c5180 3b9aca00 000f4240 00000000 ee8a7c88
-7c60: 00000001 00000501 c0559050 c099a418 00000000 ee28ca80 a0000013 c06ebd50
-7c80: ee28cac0 ee270e50 ee28cac0 ee28b800 ee28cac0 eefe7fb4 00000000 c0559050
-7ca0: 00000000 c06ebcac 0000400b ee270e50 ee28cac0 ee28b800 0000000c eefe7fb4
-7cc0: c0fecefc c06ec0c8 00000000 ee270e50 ee28b800 00000000 00000000 eefe7fb4
-7ce0: c0f6c3c8 c0558af4 ee8a7cf4 ee28b800 00000000 00000001 ee28b800 ee270e50
-7d00: ee28cb50 c0558ea4 00000000 00000000 00000000 000007d0 ee28b800 00000000
-7d20: c0f6c3b8 c0ffe1a8 00000000 c05a67b8 ee28b800 c0ffe1a4 00000000 c05347b8
-7d40: 00000000 ee8a7d78 c0534964 00000001 ee03a000 00000000 c0ffe180 c0532dc0
-7d60: ee9f1170 ee2605b8 ee28b800 ee28b834 c0f72ef4 c05344e0 ee28b800 00000001
-7d80: 00000000 ee28b800 ee28b800 c0f72ef4 c0f04c48 c0533b24 ee28b800 ee28b808
-7da0: 00000000 c0532160 40400000 00000000 ee03a358 c05a97b4 ee28b800 e416bf4f
+7c40: ee8a7bd0 00000000 c06ebf30 005c5180 3b9aca00 000f4240 00000000 ee8a7c88
+7c60: 00000001 00000501 c055910c c099aad8 00000000 ee28ca80 a0000013 c06ebf30
+7c80: ee28cac0 ee270e50 ee28cac0 ee28b800 ee28cac0 eefe7fb4 00000000 c055910c
+7ca0: 00000000 c06ebe8c 0000400b ee270e50 ee28cac0 ee28b800 0000000c eefe7fb4
+7cc0: c0fecefc c06ec2a8 00000000 ee270e50 ee28b800 00000000 00000000 eefe7fb4
+7ce0: c0f6c3c8 c0558bb0 ee8a7cf4 ee28b800 00000000 00000001 ee28b800 ee270e50
+7d00: ee28cb50 c0558f60 00000000 00000000 00000000 000007d0 ee28b800 00000000
+7d20: c0f6c3b8 c0ffe1a8 00000000 c05a6874 ee28b800 c0ffe1a4 00000000 c0534874
+7d40: 00000000 ee8a7d78 c0534a20 00000001 ee03a000 00000000 c0ffe180 c0532e7c
+7d60: ee9f1170 ee2605b8 ee28b800 ee28b834 c0f72ef4 c053459c ee28b800 00000001
+7d80: 00000000 ee28b800 ee28b800 c0f72ef4 c0f04c48 c0533be0 ee28b800 ee28b808
+7da0: 00000000 c053221c 40400000 00000000 ee03a358 c05a9870 ee28b800 e416bf4f
 7dc0: 00000000 ee28b800 ee03a000 00000000 eea8be10 eefe8004 00000001 00000000
-7de0: 00000000 c05a660c 002dc6c0 ee03a000 eefe7fb4 00000000 ee28b800 c05a85ac
-7e00: 00000000 014000c0 00000000 c05a7f98 00000063 002dc6c0 ee03a000 ee28c610
-7e20: eea8be00 eea8be10 40000000 00000153 00000000 c05a8ad0 eefe7d0c ee03a350
-7e40: ee03a000 eea8be00 eea8be10 c05a965c 00000000 ee03a350 ffffffed eea8be10
-7e60: fffffdfb c0f73850 00000000 00000000 c0f73850 c05360d0 c0536080 eea8be10
-7e80: c0ffe1a4 00000000 c0ffe1a8 c05347b8 eea8be10 c0f73850 eea8be44 00000000
-7ea0: 00000000 c0e6b83c c0e00594 c053492c 00000000 c0f73850 c0534880 c0532e5c
-7ec0: ee8cc55c eea14934 c0f73850 ee287e80 c0f6a9b8 c0533cf8 c0c21e44 c0f73850
-7ee0: 00000006 c0f73850 00000006 c0f04c48 c0e387f0 c05354ac c0535d24 c0e8434c
-7f00: 00000006 c0101bbc 00000000 ee8a7f24 00000000 c0161e00 00000000 c0cc0288
-7f20: 0000010e efffcc45 c0cc0288 0000010e efffcc53 c013ecb4 c0cbed70 00000000
+7de0: 00000000 c05a66c8 002dc6c0 ee03a000 eefe7fb4 00000000 ee28b800 c05a8668
+7e00: 00000000 014000c0 00000000 c05a8054 00000063 002dc6c0 ee03a000 ee28c610
+7e20: eea8be00 eea8be10 40000000 00000153 00000000 c05a8b8c eefe7d0c ee03a350
+7e40: ee03a000 eea8be00 eea8be10 c05a9718 00000000 ee03a350 ffffffed eea8be10
+7e60: fffffdfb c0f73850 00000000 00000000 c0f73850 c053618c c053613c eea8be10
+7e80: c0ffe1a4 00000000 c0ffe1a8 c0534874 eea8be10 c0f73850 eea8be44 00000000
+7ea0: 00000000 c0e6b83c c0e00594 c05349e8 00000000 c0f73850 c053493c c0532f18
+7ec0: ee8cc55c eea14934 c0f73850 ee287e80 c0f6a9b8 c0533db4 c0c21ebc c0f73850
+7ee0: 00000006 c0f73850 00000006 c0f04c48 c0e387f0 c0535568 c0535de0 c0e8434c
+7f00: 00000006 c0101bbc 00000000 ee8a7f24 00000000 c0161e00 00000000 c0cc02a8
+7f20: 0000010e efffcc45 c0cc02a8 0000010e efffcc53 c013ecb4 c0cbed90 00000000
 7f40: 00000006 00000006 efffcc45 efffcc4a 00000000 e416bf4f 00000006 c0e8434c
 7f60: 00000006 c0e6b830 c0fc2880 0000010e c0e00594 c0e00dd0 00000006 00000006
-7f80: 00000000 c0e00594 00000000 c099342c 00000000 00000000 00000000 00000000
-7fa0: 00000000 c0993434 00000000 c0107d70 00000000 00000000 00000000 00000000
+7f80: 00000000 c0e00594 00000000 c0993aec 00000000 00000000 00000000 00000000
+7fa0: 00000000 c0993af4 00000000 c0107d70 00000000 00000000 00000000 00000000
 7fc0: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
 7fe0: 00000000 00000000 00000000 00000000 00000013 00000000 00000000 00000000
-[<c06ebf20>] (cros_ec_prepare_tx) from [<c0559068>] (cros_ec_pkt_xfer_spi+0x18/0x474)
-[<c0559068>] (cros_ec_pkt_xfer_spi) from [<c06ebcac>] (send_command+0x20/0xd8)
-[<c06ebcac>] (send_command) from [<c06ec0c8>] (cros_ec_query_all+0x1a4/0x398)
-[<c06ec0c8>] (cros_ec_query_all) from [<c0558af4>] (cros_ec_register+0x94/0x21c)
-[<c0558af4>] (cros_ec_register) from [<c0558ea4>] (cros_ec_spi_probe+0x108/0x150)
-[<c0558ea4>] (cros_ec_spi_probe) from [<c05a67b8>] (spi_drv_probe+0x7c/0xa8)
-[<c05a67b8>] (spi_drv_probe) from [<c05347b8>] (driver_probe_device+0x250/0x318)
-[<c05347b8>] (driver_probe_device) from [<c0532dc0>] (bus_for_each_drv+0x44/0x8c)
-[<c0532dc0>] (bus_for_each_drv) from [<c05344e0>] (__device_attach+0x9c/0x100)
-[<c05344e0>] (__device_attach) from [<c0533b24>] (bus_probe_device+0x84/0x8c)
-[<c0533b24>] (bus_probe_device) from [<c0532160>] (device_add+0x39c/0x54c)
-[<c0532160>] (device_add) from [<c05a660c>] (spi_add_device+0x88/0x130)
-[<c05a660c>] (spi_add_device) from [<c05a85ac>] (spi_register_controller+0x428/0x918)
-[<c05a85ac>] (spi_register_controller) from [<c05a8ad0>] (devm_spi_register_controller+0x34/0x70)
-[<c05a8ad0>] (devm_spi_register_controller) from [<c05a965c>] (tegra_spi_probe+0x298/0x35c)
-[<c05a965c>] (tegra_spi_probe) from [<c05360d0>] (platform_drv_probe+0x50/0xb0)
-[<c05360d0>] (platform_drv_probe) from [<c05347b8>] (driver_probe_device+0x250/0x318)
-[<c05347b8>] (driver_probe_device) from [<c053492c>] (__driver_attach+0xac/0xb0)
-[<c053492c>] (__driver_attach) from [<c0532e5c>] (bus_for_each_dev+0x54/0x88)
-[<c0532e5c>] (bus_for_each_dev) from [<c0533cf8>] (bus_add_driver+0xe4/0x1f0)
-[<c0533cf8>] (bus_add_driver) from [<c05354ac>] (driver_register+0x78/0xf4)
-[<c05354ac>] (driver_register) from [<c0101bbc>] (do_one_initcall+0x48/0x18c)
+[<c06ec100>] (cros_ec_prepare_tx) from [<c0559124>] (cros_ec_pkt_xfer_spi+0x18/0x474)
+[<c0559124>] (cros_ec_pkt_xfer_spi) from [<c06ebe8c>] (send_command+0x20/0xd8)
+[<c06ebe8c>] (send_command) from [<c06ec2a8>] (cros_ec_query_all+0x1a4/0x398)
+[<c06ec2a8>] (cros_ec_query_all) from [<c0558bb0>] (cros_ec_register+0x94/0x21c)
+[<c0558bb0>] (cros_ec_register) from [<c0558f60>] (cros_ec_spi_probe+0x108/0x150)
+[<c0558f60>] (cros_ec_spi_probe) from [<c05a6874>] (spi_drv_probe+0x7c/0xa8)
+[<c05a6874>] (spi_drv_probe) from [<c0534874>] (driver_probe_device+0x250/0x318)
+[<c0534874>] (driver_probe_device) from [<c0532e7c>] (bus_for_each_drv+0x44/0x8c)
+[<c0532e7c>] (bus_for_each_drv) from [<c053459c>] (__device_attach+0x9c/0x100)
+[<c053459c>] (__device_attach) from [<c0533be0>] (bus_probe_device+0x84/0x8c)
+[<c0533be0>] (bus_probe_device) from [<c053221c>] (device_add+0x39c/0x54c)
+[<c053221c>] (device_add) from [<c05a66c8>] (spi_add_device+0x88/0x130)
+[<c05a66c8>] (spi_add_device) from [<c05a8668>] (spi_register_controller+0x428/0x918)
+[<c05a8668>] (spi_register_controller) from [<c05a8b8c>] (devm_spi_register_controller+0x34/0x70)
+[<c05a8b8c>] (devm_spi_register_controller) from [<c05a9718>] (tegra_spi_probe+0x298/0x35c)
+[<c05a9718>] (tegra_spi_probe) from [<c053618c>] (platform_drv_probe+0x50/0xb0)
+[<c053618c>] (platform_drv_probe) from [<c0534874>] (driver_probe_device+0x250/0x318)
+[<c0534874>] (driver_probe_device) from [<c05349e8>] (__driver_attach+0xac/0xb0)
+[<c05349e8>] (__driver_attach) from [<c0532f18>] (bus_for_each_dev+0x54/0x88)
+[<c0532f18>] (bus_for_each_dev) from [<c0533db4>] (bus_add_driver+0xe4/0x1f0)
+[<c0533db4>] (bus_add_driver) from [<c0535568>] (driver_register+0x78/0xf4)
+[<c0535568>] (driver_register) from [<c0101bbc>] (do_one_initcall+0x48/0x18c)
 [<c0101bbc>] (do_one_initcall) from [<c0e00dd0>] (kernel_init_freeable+0x10c/0x1d0)
-[<c0e00dd0>] (kernel_init_freeable) from [<c0993434>] (kernel_init+0x8/0x114)
-[<c0993434>] (kernel_init) from [<c0107d70>] (ret_from_fork+0x14/0x24)
+[<c0e00dd0>] (kernel_init_freeable) from [<c0993af4>] (kernel_init+0x8/0x114)
+[<c0993af4>] (kernel_init) from [<c0107d70>] (ret_from_fork+0x14/0x24)
 Code: e2800008 e8bd8070 e7f001f2 e7f001f2 (e7f001f2) 
----[ end trace 1aac5ec07e2c1a34 ]---
+---[ end trace d153cebdc35afedc ]---
 Kernel panic - not syncing: Attempted to kill init! exitcode=0x0000000b
 
-CPU2: stopping
-CPU: 2 PID: 0 Comm: swapper/2 Tainted: G      D          4.14.0-next-20171123-215928-g279a247 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c0111230>] (unwind_backtrace) from [<c010c168>] (show_stack+0x10/0x14)
-[<c010c168>] (show_stack) from [<c097f93c>] (dump_stack+0x84/0xa4)
-[<c097f93c>] (dump_stack) from [<c010f71c>] (handle_IPI+0x18c/0x39c)
-[<c010f71c>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
-[<c0101430>] (gic_handle_irq) from [<c010cd4c>] (__irq_svc+0x6c/0xa8)
-Exception stack(0xee8e1f70 to 0xee8e1fb8)
-1f60:                                     00000001 00000000 ee8e1fc8 c011a560
-1f80: 00000004 c0f04ce0 00000000 c0f04c6c 00000000 c0e8d2b8 00000000 c0f04cf0
-1fa0: 2e125000 ee8e1fc0 c01087d8 c01087dc 60000013 ffffffff
-[<c010cd4c>] (__irq_svc) from [<c01087dc>] (arch_cpu_idle+0x30/0x3c)
-[<c01087dc>] (arch_cpu_idle) from [<c016290c>] (do_idle+0x140/0x204)
-[<c016290c>] (do_idle) from [<c0162c58>] (cpu_startup_entry+0x18/0x1c)
-[<c0162c58>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
 CPU1: stopping
-CPU: 1 PID: 0 Comm: swapper/1 Tainted: G      D          4.14.0-next-20171123-215928-g279a247 #1
+CPU: 1 PID: 0 Comm: swapper/1 Tainted: G      D          4.14.0-next-20171124-216009-g6fc478f #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c0111230>] (unwind_backtrace) from [<c010c168>] (show_stack+0x10/0x14)
-[<c010c168>] (show_stack) from [<c097f93c>] (dump_stack+0x84/0xa4)
-[<c097f93c>] (dump_stack) from [<c010f71c>] (handle_IPI+0x18c/0x39c)
+[<c010c168>] (show_stack) from [<c097fffc>] (dump_stack+0x84/0xa4)
+[<c097fffc>] (dump_stack) from [<c010f71c>] (handle_IPI+0x18c/0x39c)
 [<c010f71c>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cd4c>] (__irq_svc+0x6c/0xa8)
 Exception stack(0xee8c1f70 to 0xee8c1fb8)
@@ -423,11 +407,11 @@
 [<c016290c>] (do_idle) from [<c0162c58>] (cpu_startup_entry+0x18/0x1c)
 [<c0162c58>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
 CPU3: stopping
-CPU: 3 PID: 0 Comm: swapper/3 Tainted: G      D          4.14.0-next-20171123-215928-g279a247 #1
+CPU: 3 PID: 0 Comm: swapper/3 Tainted: G      D          4.14.0-next-20171124-216009-g6fc478f #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c0111230>] (unwind_backtrace) from [<c010c168>] (show_stack+0x10/0x14)
-[<c010c168>] (show_stack) from [<c097f93c>] (dump_stack+0x84/0xa4)
-[<c097f93c>] (dump_stack) from [<c010f71c>] (handle_IPI+0x18c/0x39c)
+[<c010c168>] (show_stack) from [<c097fffc>] (dump_stack+0x84/0xa4)
+[<c097fffc>] (dump_stack) from [<c010f71c>] (handle_IPI+0x18c/0x39c)
 [<c010f71c>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cd4c>] (__irq_svc+0x6c/0xa8)
 Exception stack(0xee8e3f70 to 0xee8e3fb8)
@@ -438,16 +422,32 @@
 [<c01087dc>] (arch_cpu_idle) from [<c016290c>] (do_idle+0x140/0x204)
 [<c016290c>] (do_idle) from [<c0162c58>] (cpu_startup_entry+0x18/0x1c)
 [<c0162c58>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
+CPU2: stopping
+CPU: 2 PID: 0 Comm: swapper/2 Tainted: G      D          4.14.0-next-20171124-216009-g6fc478f #1
+Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
+[<c0111230>] (unwind_backtrace) from [<c010c168>] (show_stack+0x10/0x14)
+[<c010c168>] (show_stack) from [<c097fffc>] (dump_stack+0x84/0xa4)
+[<c097fffc>] (dump_stack) from [<c010f71c>] (handle_IPI+0x18c/0x39c)
+[<c010f71c>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
+[<c0101430>] (gic_handle_irq) from [<c010cd4c>] (__irq_svc+0x6c/0xa8)
+Exception stack(0xee8e1f70 to 0xee8e1fb8)
+1f60:                                     00000001 00000000 ee8e1fc8 c011a560
+1f80: 00000004 c0f04ce0 00000000 c0f04c6c 00000000 c0e8d2b8 00000000 c0f04cf0
+1fa0: 2e125000 ee8e1fc0 c01087d8 c01087dc 60000013 ffffffff
+[<c010cd4c>] (__irq_svc) from [<c01087dc>] (arch_cpu_idle+0x30/0x3c)
+[<c01087dc>] (arch_cpu_idle) from [<c016290c>] (do_idle+0x140/0x204)
+[<c016290c>] (do_idle) from [<c0162c58>] (cpu_startup_entry+0x18/0x1c)
+[<c0162c58>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
 ---[ end Kernel panic - not syncing: Attempted to kill init! exitcode=0x0000000b
 
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 8 at kernel/irq_work.c:68 irq_work_queue_on+0xbc/0xfc
 Modules linked in:
-CPU: 0 PID: 8 Comm: rcu_preempt Tainted: G      D          4.14.0-next-20171123-215928-g279a247 #1
+CPU: 0 PID: 8 Comm: rcu_preempt Tainted: G      D          4.14.0-next-20171124-216009-g6fc478f #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c0111230>] (unwind_backtrace) from [<c010c168>] (show_stack+0x10/0x14)
-[<c010c168>] (show_stack) from [<c097f93c>] (dump_stack+0x84/0xa4)
-[<c097f93c>] (dump_stack) from [<c0121c20>] (__warn+0xe4/0x110)
+[<c010c168>] (show_stack) from [<c097fffc>] (dump_stack+0x84/0xa4)
+[<c097fffc>] (dump_stack) from [<c0121c20>] (__warn+0xe4/0x110)
 [<c0121c20>] (__warn) from [<c0121c90>] (warn_slowpath_null+0x44/0x4c)
 [<c0121c90>] (warn_slowpath_null) from [<c01d7610>] (irq_work_queue_on+0xbc/0xfc)
 [<c01d7610>] (irq_work_queue_on) from [<c01841b8>] (rcu_implicit_dynticks_qs+0x280/0x2a0)
@@ -455,4 +455,4 @@
 [<c018336c>] (force_qs_rnp) from [<c0183940>] (rcu_gp_kthread+0x534/0xa58)
 [<c0183940>] (rcu_gp_kthread) from [<c013f55c>] (kthread+0x11c/0x154)
 [<c013f55c>] (kthread) from [<c0107d70>] (ret_from_fork+0x14/0x24)
----[ end trace 1aac5ec07e2c1a35 ]---
+---[ end trace d153cebdc35afedd ]---
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2017-11-23 19:21:27.855904952 -0800
+++ /tmp/foob	2017-11-23 19:21:27.855904952 -0800
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1801195 us
+Bootloader-Cpu Init at (time stamp): 1801764 us
 
 Pinmux changes applied in kernel way
 
@@ -59,25 +59,25 @@
 The proc BoardInfo: 0x0176:0x0001:0x01:0x43:0x00
 FastbootAddAtag: setting ATAG (0)
 
-Board Id Read Time: 4 us
+Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1801168  us
-Bootloader (AosInit) time: 733473 us
-Bootloader Main Init time: 180252 us
-BlockDevMgrInit time: 157560 us
+Miscellaneous avp time: 1801738  us
+Bootloader (AosInit) time: 734237 us
+Bootloader Main Init time: 180077 us
+BlockDevMgrInit time: 157553 us
 PartMgrInit time: 8 us
-StorMgrInit time: 31 us
+StorMgrInit time: 30 us
 FsMgrInit time: 2 us
-Partition Table load: 77051 us 
-Display init time: 574603 us
+Partition Table load: 76690 us 
+Display init time: 574810 us
 Kernel read time: 1535 msec
 Load Kernel Size:   20342784 Bytes
 Bl Kernel ReadSpeed: 13252000 Bytes/sec
-Pre-Kernel Jump time: 4307369 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 9461419 us
+Pre-Kernel Jump time: 4307082 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 9464250 us
 Booting Linux on physical CPU 0x0000000000 [0x4e0f0000]
-Linux version 4.14.0-next-20171123-334169-g279a24799865 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Nov 22 18:55:59 PST 2017
+Linux version 4.14.0-next-20171124-334250-g6fc478f80f68 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Thu Nov 23 19:11:09 PST 2017
 Machine model: NVIDIA Tegra132 Norrin
 Memory limited to 1734MB
 efi: Getting EFI parameters from FDT:
@@ -148,7 +148,7 @@
 DMI not present or invalid.
 NET: Registered protocol family 16
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.136:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.132:1): state=initialized audit_enabled=0 res=1
 cpuidle: using governor menu
 vdso: 2 pages (1 code @ ffff0000089f6000, 1 data @ ffff000008f75000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
@@ -282,6 +282,7 @@
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
+sdhci-tegra 700b0400.sdhci: Got CD GPIO
 hctosys: unable to open rtc device (rtc0)
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +VDD_LED: disabling
@@ -292,7 +293,6 @@
 ALSA device list:
   No soundcards found.
 Freeing unused kernel memory: 1152K
-sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 
 System initialization...
Diff for ./tegra186-p2771-0000/tegra186-p2771-0000/defconfig%jetson-tx2_log.txt
Binary files /tmp/fooa and /tmp/foob differ
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-11-23 19:21:27.895905130 -0800
+++ /tmp/foob	2017-11-23 19:21:27.899905149 -0800
@@ -48,7 +48,7 @@
 	 #################################################################
 	 #################################################################
 	 ###############################
-	 3.5 MiB/s
+	 3.7 MiB/s
 done
 Bytes transferred = 8086016 (7b6200 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
@@ -69,7 +69,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-next-20171123-334169-g279a24799865 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Nov 22 18:49:34 PST 2017
+Linux version 4.14.0-next-20171124-334250-g6fc478f80f68 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Nov 23 19:04:53 PST 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2017-11-23 19:21:27.923905255 -0800
+++ /tmp/foob	2017-11-23 19:21:27.927905272 -0800
@@ -48,7 +48,7 @@
 	 ################################
 	 4.1 MiB/s
 done
-Bytes transferred = 6192624 (5e7df0 hex)
+Bytes transferred = 6192952 (5e7f38 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -59,7 +59,7 @@
 done
 Bytes transferred = 21160 (52a8 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x5e7df0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x5e7f38 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102a7
@@ -67,7 +67,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-next-20171123-14220-g279a247 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Nov 22 18:47:53 PST 2017
+Linux version 4.14.0-next-20171124-14301-g6fc478f (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Nov 23 19:03:26 PST 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig%jetson-tx1_log.txt
--- /tmp/fooa	2017-11-23 19:21:27.943905343 -0800
+++ /tmp/foob	2017-11-23 19:21:27.943905343 -0800
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 737064 us
-Basic modules init took 350287 us
+Tegraboot started after 796511 us
+Basic modules init took 348768 us
 Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14366 us
+Next stage binary read took 14373 us
 Carveout took 252669 us
-CPU initialization took 92465 us
-Total time taken by TegraBoot 709787 us
+CPU initialization took 92438 us
+Total time taken by TegraBoot 708248 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.578] LPDDR4 Training: Number of tables = 10
+64b[0001.634] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig%jetson-tx1
-17039872 bytes read in 877 ms (18.5 MiB/s)
+17039872 bytes read in 870 ms (18.7 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
 44764 bytes read in 454 ms (95.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 495 ms (3.1 MiB/s)
+1627787 bytes read in 494 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0000000000 [0x411fd071]
-Linux version 4.14.0-next-20171123-14220-g279a247 (jonathanh@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Nov 22 18:51:35 PST 2017
+Linux version 4.14.0-next-20171124-14301-g6fc478f (jonathanh@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Thu Nov 23 19:07:07 PST 2017
 Machine model: NVIDIA Tegra210 P2371 (P2530/P2595) reference design
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -267,7 +267,7 @@
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 PCI: CLS 0 bytes, default 128
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 809eb000
+kvm [1]: IDMAP page: 809ec000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: vgic-v2@50044000
 kvm [1]: vgic interrupt IRQ1
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2017-11-23 19:21:27.963905431 -0800
+++ /tmp/foob	2017-11-23 19:21:27.967905450 -0800
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 741214 us
-Basic modules init took 348393 us
+Tegraboot started after 721353 us
+Basic modules init took 348541 us
 Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14390 us
-Carveout took 252666 us
-CPU initialization took 92447 us
-Total time taken by TegraBoot 707896 us
+Next stage binary read took 14365 us
+Carveout took 252658 us
+CPU initialization took 92462 us
+Total time taken by TegraBoot 708026 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.580] LPDDR4 Training: Number of tables = 10
+64b[0001.561] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,7 +117,7 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-16843264 bytes read in 863 ms (18.6 MiB/s)
+16843264 bytes read in 846 ms (19 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
 44764 bytes read in 453 ms (95.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0000000000 [0x411fd071]
-Linux version 4.14.0-next-20171123-334169-g279a24799865 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Nov 22 18:55:59 PST 2017
+Linux version 4.14.0-next-20171124-334250-g6fc478f80f68 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Thu Nov 23 19:11:09 PST 2017
 Machine model: NVIDIA Tegra210 P2371 (P2530/P2595) reference design
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -267,7 +267,7 @@
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 PCI: CLS 0 bytes, default 128
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 809e3000
+kvm [1]: IDMAP page: 809e4000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: vgic-v2@50044000
 kvm [1]: vgic interrupt IRQ1
Diff for ./tegra210-p2371-2180/tegra210-p2371-2180/defconfig%jetson-tx1_log.txt
Binary files /tmp/fooa and /tmp/foob differ
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2017-11-23 19:21:27.991905557 -0800
+++ /tmp/foob	2017-11-23 19:21:27.995905575 -0800
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0000000000 [0x411fd071]
-Linux version 4.14.0-next-20171123-334169-g279a24799865 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Nov 22 18:55:59 PST 2017
+Linux version 4.14.0-next-20171124-334250-g6fc478f80f68 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Thu Nov 23 19:11:09 PST 2017
 Machine model: Google Pixel C
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
@@ -151,7 +151,7 @@
 Unpacking initramfs...
 Freeing initrd memory: 3408K
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 809e3000
+kvm [1]: IDMAP page: 809e4000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: vgic-v2@50044000
 kvm [1]: vgic interrupt IRQ1
@@ -265,7 +265,7 @@
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB, chardev (240:0)
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
-max77686-rtc max77620-rtc: setting system clock to 2001-09-03 08:25:45 UTC (999505545)
+max77686-rtc max77620-rtc: setting system clock to 2001-09-04 08:40:46 UTC (999592846)
 PP1200_AVDD: disabling
 PP1200_RCAM: disabling
 PP2800L_RCAM: disabling
@@ -276,8 +276,8 @@
 PP1800_CAM: disabling
 ALSA device list:
   No soundcards found.
-tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 Freeing unused kernel memory: 1152K
+tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 
 System initialization...
 
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-11-23 19:21:28.007905628 -0800
+++ /tmp/foob	2017-11-23 19:21:28.011905646 -0800
@@ -40,7 +40,7 @@
 	 #################################################################
 	 #################################################################
 	 ###############################
-	 2.8 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 8086016 (7b6200 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
@@ -49,7 +49,7 @@
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.7 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 51470 (c90e hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-next-20171123-334169-g279a24799865 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Nov 22 18:49:34 PST 2017
+Linux version 4.14.0-next-20171124-334250-g6fc478f80f68 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Nov 23 19:04:53 PST 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -463,4 +463,4 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.494053] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.484094] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2017-11-23 19:21:28.015905663 -0800
+++ /tmp/foob	2017-11-23 19:21:28.019905680 -0800
@@ -40,18 +40,18 @@
 	 ################################
 	 3.3 MiB/s
 done
-Bytes transferred = 6192624 (5e7df0 hex)
+Bytes transferred = 6192952 (5e7f38 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 51470 (c90e hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5e7df0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5e7f38 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90d ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-next-20171123-14220-g279a247 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Nov 22 18:47:53 PST 2017
+Linux version 4.14.0-next-20171124-14301-g6fc478f (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Nov 23 19:03:26 PST 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
