Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-11-23 07:27:15.975597890 -0800
+++ /tmp/foob	2016-11-23 07:27:15.979597903 -0800
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc6-next-20161122-160968-g637aaec (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Nov 23 03:36:31 PST 2016
+Linux version 4.9.0-rc6-next-20161123-161173-g272da04 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Nov 23 07:21:15 PST 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -101,7 +101,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.676:1): initialized
+audit: type=2000 audit(0.688:1): initialized
 workingset: timestamp_bits=46 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -167,7 +167,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2016-11-23 11:37:08 UTC (1479901028)
+rtc-pl031 9010000.pl031: setting system clock to 2016-11-23 15:21:51 UTC (1479914511)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-11-23 07:27:15.983597916 -0800
+++ /tmp/foob	2016-11-23 07:27:15.987597929 -0800
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra114 (Dalmore) # 
 Tegra114 (Dalmore) # usb reset
 (Re)start USB...
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 ###############################################
-	 3.1 MiB/s
+	 3.5 MiB/s
 done
-Bytes transferred = 7357624 (7044b8 hex)
+Bytes transferred = 7357464 (704418 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3.3 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x7044b8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x704418 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc6-next-20161122-136535-g637aaec (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Nov 23 03:33:28 PST 2016
+Linux version 4.9.0-rc6-next-20161123-136740-g272da04 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Nov 23 07:18:30 PST 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -306,11 +306,11 @@
 usb1_vbus: supplied by vdd-sys-5v0
 usb2_vbus: supplied by vdd-sys-5v0
 mmc0: new high speed SDHC card at address aaaa
-mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
- mmcblk0: p1 p2 p3
+mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 [drm] No driver support for vblank timestamp query.
+ mmcblk0: p1 p2 p3
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
@@ -327,7 +327,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-09-16 23:04:04 UTC (969145444)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-09-17 02:49:00 UTC (969158940)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -339,4 +339,4 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.400321] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.415246] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-11-23 07:27:15.991597942 -0800
+++ /tmp/foob	2016-11-23 07:27:15.995597955 -0800
@@ -36,10 +36,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #################################################
-	 3.3 MiB/s
+	 ##################################################
+	 3.1 MiB/s
 done
-Bytes transferred = 5489600 (53c3c0 hex)
+Bytes transferred = 5491024 (53c950 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x53c3c0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x53c950 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc6-next-20161122-136535-g637aaec (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Nov 23 03:33:12 PST 2016
+Linux version 4.9.0-rc6-next-20161123-136740-g272da04 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Nov 23 07:18:12 PST 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -133,7 +133,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?[    0.531586] hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
+No ATAGs?[    0.529872] hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 8 bytes.
 tegra-mc 70019000.memory-controller: no memory timings for RAM code 3 registered
 dvdd_ts: Failed to request enable GPIO61: -517
@@ -321,7 +321,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-09-16 23:01:20 UTC (969145280)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-09-17 02:46:17 UTC (969158777)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -335,4 +335,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.783148] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-11-23 07:27:15.999597968 -0800
+++ /tmp/foob	2016-11-23 07:27:16.003597981 -0800
@@ -34,20 +34,20 @@
 	 #################################################################
 	 #################################################################
 	 ###############################################
-	 3.9 MiB/s
+	 3.8 MiB/s
 done
-Bytes transferred = 7357624 (7044b8 hex)
+Bytes transferred = 7357464 (704418 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.6 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 67133 (1063d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x7044b8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x704418 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301363c
@@ -55,7 +55,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc6-next-20161122-136535-g637aaec (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Nov 23 03:33:28 PST 2016
+Linux version 4.9.0-rc6-next-20161123-136740-g272da04 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Nov 23 07:18:30 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -312,7 +312,6 @@
 +5V_SATA: supplied by +5V_SYS
 +12V_SATA: supplied by +VDD_MUX
 mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 usb2-0 supply vbus not found, using dummy regulator
 usb2-1 supply vbus not found, using dummy regulator
 usb3-0 supply vbus not found, using dummy regulator
@@ -423,8 +422,9 @@
 mmc1: new high speed SDHC card at address aaaa
 mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
  mmcblk1: p1 p2 p3
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 Console: switching to colour frame buffer device 128x48
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
  mmcblk0: p1
@@ -434,7 +434,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:14 UTC (946684814)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:18 UTC (946684938)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
@@ -446,7 +446,7 @@
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:1.
+VFS: Mounted root (ext3 filesystem) on device 179:17.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-11-23 07:27:16.007597995 -0800
+++ /tmp/foob	2016-11-23 07:27:16.011598007 -0800
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -31,21 +31,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #################################################
+	 ##################################################
 	 4 MiB/s
 done
-Bytes transferred = 5489600 (53c3c0 hex)
+Bytes transferred = 5491024 (53c950 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 4 MiB/s
+	 3.8 MiB/s
 done
 Bytes transferred = 67133 (1063d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x53c3c0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x53c950 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301363c
@@ -54,7 +54,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc6-next-20161122-136535-g637aaec (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Nov 23 03:33:12 PST 2016
+Linux version 4.9.0-rc6-next-20161123-136740-g272da04 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Nov 23 07:18:12 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -128,7 +128,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?[    0.587449] hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
+No ATAGs?[    0.585248] hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 8 bytes.
 +5V_SYS: supplied by +VDD_MUX
 +3.3V_SYS: supplied by +VDD_MUX
@@ -374,8 +374,8 @@
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
 tegra-pcie 1003000.pcie-controller: probing port 0, using 2 lanes
-tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
 ata1: SATA link down (SStatus 0 SControl 300)
+tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
 tegra-pcie 1003000.pcie-controller: probing port 1, using 1 lanes
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000000
 tegra-pcie 1003000.pcie-controller: PCI host bridge to bus 0000:00
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-11-23 07:27:16.015598020 -0800
+++ /tmp/foob	2016-11-23 07:27:16.015598020 -0800
@@ -44,7 +44,7 @@
 	 ###############################################
 	 3.6 MiB/s
 done
-Bytes transferred = 7357624 (7044b8 hex)
+Bytes transferred = 7357464 (704418 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -55,7 +55,7 @@
 done
 Bytes transferred = 67943 (10967 hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x7044b8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x704418 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013966
@@ -63,7 +63,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc6-next-20161122-136535-g637aaec (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Nov 23 03:33:28 PST 2016
+Linux version 4.9.0-rc6-next-20161123-136740-g272da04 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Nov 23 07:18:30 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -296,8 +296,6 @@
 mmc0: SDHCI controller on 700b0600.sdhci [700b0600.sdhci] using ADMA
 ledtrig-cpu: registered to indicate activity on CPUs
 usbcore: registered new interface driver usbhid
-elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
-input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 usbhid: USB HID core driver
 NET: Registered protocol family 10
 Segment Routing with IPv6
@@ -311,6 +309,7 @@
 Key type dns_resolver registered
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
+elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
 +3.3V_RUN: supplied by +3.3V_SYS
 +3.3V_AVDD_HDMI_AP_GATED: supplied by +3.3V_RUN
 +1.8V_RUN_CAM: supplied by +3.3V_RUN
@@ -321,34 +320,34 @@
 +5V_USB_HS: supplied by +5V_SYS
 +5V_USB_SS: supplied by +5V_SYS
 +3.3V_PANEL: supplied by +3.3V_RUN
-mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
-mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
-mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
-mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 +3.3V_LP0: supplied by +3.3V_SYS
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 +5V_HDMI_CON: supplied by +5V_SYS
 usb3-0 supply vbus not found, using dummy regulator
 usb3-1 supply vbus not found, using dummy regulator
-elan_i2c 1-0015: failed to read report data: -121
 tegra-sor 54540000.sor: failed to probe output: -517
 panel supply power not found, using dummy regulator
 sdhci-tegra 700b0000.sdhci: allocated mmc-pwrseq
 mmc1: Unknown controller version (3). You may experience problems.
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: Switching to 3.3V signalling voltage failed
+mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
+mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
+mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
+mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
 mmc1: SDHCI controller on 700b0000.sdhci [700b0000.sdhci] using ADMA
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc2: Unknown controller version (3). You may experience problems.
 mmc2: Invalid maximum block size, assuming 512 bytes
-elan_i2c 1-0015: invalid report id data (ff)
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 mmc1: new high speed SDIO card at address 0001
 mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
+input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
+elan_i2c 1-0015: invalid report id data (ff)
 mmc2: new high speed SDHC card at address 0007
 mmcblk2: mmc2:0007 SD08G 7.42 GiB 
 GPT:Primary header alternate_lba != Alt. header my_lba
@@ -361,7 +360,7 @@
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:24 UTC (946684824)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 03:30:18 UTC (946697418)
 +1.8V_RUN_CAM: disabling
 +3.3V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2016-11-23 07:27:16.023598047 -0800
+++ /tmp/foob	2016-11-23 07:27:16.027598060 -0800
@@ -39,21 +39,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #################################################
-	 3.5 MiB/s
+	 ##################################################
+	 3.6 MiB/s
 done
-Bytes transferred = 5489600 (53c3c0 hex)
+Bytes transferred = 5491024 (53c950 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.4 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 67943 (10967 hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x53c3c0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x53c950 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013966
@@ -62,7 +62,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc6-next-20161122-136535-g637aaec (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Nov 23 03:33:12 PST 2016
+Linux version 4.9.0-rc6-next-20161123-136740-g272da04 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Nov 23 07:18:12 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -137,7 +137,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?[    0.499882] hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
+No ATAGs?[    0.499025] hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 8 bytes.
 tegra-mc 70019000.memory-controller: no memory timings for RAM code 0 registered
 +5V_SYS: supplied by +VDD_MUX
@@ -289,10 +289,10 @@
 mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
 mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
-mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 tegra-snd-max98090 sound: HiFi <-> 70301100.i2s mapping ok
 input: GoogleNyanBig Headphones as /devices/soc0/sound/sound/card1/input1
+mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 input: GoogleNyanBig Mic Jack as /devices/soc0/sound/sound/card1/input2
 NET: Registered protocol family 10
 Segment Routing with IPv6
@@ -356,9 +356,9 @@
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
 usb 1-2: new high-speed USB device number 2 using tegra-xusb
-input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input3
 mmc2: new high speed SDHC card at address 0007
 mmcblk2: mmc2:0007 SD08G 7.42 GiB 
+input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input3
 GPT:Primary header alternate_lba != Alt. header my_lba
 GPT:3014591 != 15564799
 GPT:Primary header thinks Alt. header is not at the end of the disk.
@@ -369,7 +369,7 @@
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input4
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:22 UTC (946684822)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 03:29:34 UTC (946697374)
 +1.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
 +1.05V_RUN_CAM_REAR: disabling
@@ -390,4 +390,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.912877] usb 1-3: new high-speed USB device number 3 using tegra-xusb
+root@(none):/# [    2.901873] usb 1-3: new high-speed USB device number 3 using tegra-xusb
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2016-11-23 07:27:16.031598073 -0800
+++ /tmp/foob	2016-11-23 07:27:16.035598086 -0800
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1800674 us
+Bootloader-Cpu Init at (time stamp): 1800648 us
 
 Pinmux changes applied in kernel way
 
@@ -62,20 +62,20 @@
 Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1800648  us
-Bootloader (AosInit) time: 733462 us
-Bootloader Main Init time: 180249 us
-BlockDevMgrInit time: 157488 us
-PartMgrInit time: 9 us
+Miscellaneous avp time: 1800622  us
+Bootloader (AosInit) time: 733469 us
+Bootloader Main Init time: 180247 us
+BlockDevMgrInit time: 157492 us
+PartMgrInit time: 8 us
 StorMgrInit time: 30 us
 FsMgrInit time: 2 us
-Partition Table load: 77156 us 
-Display init time: 574459 us
-Kernel read time: 379 msec
+Partition Table load: 77170 us 
+Display init time: 574344 us
+Kernel read time: 377 msec
 Load Kernel Size:   16304128 Bytes
-Bl Kernel ReadSpeed: 43018000 Bytes/sec
-Pre-Kernel Jump time: 1740247 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5843151 us
+Bl Kernel ReadSpeed: 43247000 Bytes/sec
+Pre-Kernel Jump time: 1741319 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5844452 us
 Booting Linux on physical CPU 0x0
 Linux version 4.8.0-239303-g3477d16 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Fri Oct 7 01:12:37 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
@@ -184,7 +184,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(1.171:1): initialized
+audit: type=2000 audit(1.159:1): initialized
 workingset: timestamp_bits=46 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-11-23 07:27:16.039598099 -0800
+++ /tmp/foob	2016-11-23 07:27:16.039598099 -0800
@@ -47,20 +47,20 @@
 	 #################################################################
 	 #################################################################
 	 ###############################################
-	 4 MiB/s
+	 4.1 MiB/s
 done
-Bytes transferred = 7357624 (7044b8 hex)
+Bytes transferred = 7357464 (704418 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 4.1 MiB/s
 done
 Bytes transferred = 21404 (539c hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x7044b8 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x704418 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 0241039b
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc6-next-20161122-136535-g637aaec (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Nov 23 03:33:28 PST 2016
+Linux version 4.9.0-rc6-next-20161123-136740-g272da04 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Nov 23 07:18:30 PST 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: Compulab TrimSlice board
@@ -334,17 +334,17 @@
 mmc0: new high speed SDHC card at address b368
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
 hub 2-0:1.0: 1 port detected
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
  mmcblk0: p1
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2015-02-05 23:56:53 UTC (1423180613)
+rtc-em3027 2-0056: setting system clock to 2015-02-06 00:03:57 UTC (1423181037)
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
-random: fast init done
 asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
+random: fast init done
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:17.
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-11-23 07:27:16.047598125 -0800
+++ /tmp/foob	2016-11-23 07:27:16.051598138 -0800
@@ -44,21 +44,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #################################################
-	 4.1 MiB/s
+	 ##################################################
+	 4 MiB/s
 done
-Bytes transferred = 5489600 (53c3c0 hex)
+Bytes transferred = 5491024 (53c950 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 2 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21404 (539c hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x53c3c0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x53c950 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 0241039b
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc6-next-20161122-136535-g637aaec (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Nov 23 03:33:12 PST 2016
+Linux version 4.9.0-rc6-next-20161123-136740-g272da04 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Nov 23 07:18:12 PST 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: Compulab TrimSlice board
@@ -139,7 +139,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?[    0.350607] hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
+No ATAGs?[    0.350219] hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 4 bytes.
 usb1_vbus: Failed to request enable GPIO170: -517
 reg-fixed-voltage regulators:regulator@2: Failed to register regulator: -517
@@ -342,7 +342,7 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2015-02-05 23:55:23 UTC (1423180523)
+rtc-em3027 2-0056: setting system clock to 2015-02-06 00:02:24 UTC (1423180944)
 ALSA device list:
   #0: tegra-trimslice
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2016-11-23 07:27:16.055598151 -0800
+++ /tmp/foob	2016-11-23 07:27:16.059598165 -0800
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 805581 us
-Basic modules init took 348762 us
-Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14426 us
-Carveout took 252712 us
-CPU initialization took 92304 us
-Total time taken by TegraBoot 708204 us
+Tegraboot started after 734392 us
+Basic modules init took 349292 us
+Sec Bootdevice Read Time = 204 ms, Read Size = 8459 KB
+Next stage binary read took 16108 us
+Carveout took 252715 us
+CPU initialization took 92308 us
+Total time taken by TegraBoot 710423 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.643] LPDDR4 Training: Number of tables = 10
+64b[0001.576] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -119,9 +119,9 @@
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
 14834176 bytes read in 791 ms (17.9 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-43755 bytes read in 454 ms (93.8 KiB/s)
+43755 bytes read in 453 ms (93.8 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 495 ms (3.1 MiB/s)
+1627787 bytes read in 500 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc6-next-20161122-160968-g637aaec (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Nov 23 03:36:31 PST 2016
+Linux version 4.9.0-rc6-next-20161123-161173-g272da04 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Nov 23 07:21:15 PST 2016
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -259,7 +259,7 @@
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 PCI: CLS 0 bytes, default 128
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 808ac000
+kvm [1]: IDMAP page: 808ab000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
@@ -347,6 +347,6 @@
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
-bash: no job control in th[    1.408839] random: fast init done
+bash: no job control in th[    1.397274] random: fast init done
 is shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2016-11-23 07:27:16.063598177 -0800
+++ /tmp/foob	2016-11-23 07:27:16.063598177 -0800
@@ -1,5 +1,75 @@
+ MMIO 0x0000000070006000 (options '115200n8')
+bootconsole [uart0] enabled
+psci: probing for conduit method from DT.
+psci: PSCIv1.0 detected in firmware.
+psci: Using standard PSCI v0.2 function IDs
+psci: MIGRATE_INFO_UP_CPU reported invalid physical ID (0x80000000)
+percpu: Embedded 21 pages/cpu @ffff8000bff4d000 s48128 r8192 d29696 u86016
+Detected PIPT I-cache on CPU0
+CPU features: enabling workaround for ARM erratum 832075
+CPU features: enabling workaround for ARM erratum 834220
+Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 735336
+Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
+log_buf_len individual max cpu contribution: 4096 bytes
+log_buf_len total cpu_extra contributions: 12288 bytes
+log_buf_len min size: 16384 bytes
+log_buf_len: 32768 bytes
+early log buf free: 14404(87%)
+PID hash table entries: 4096 (order: 3, 32768 bytes)
+Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
+Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
+software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
+Memory: 2832372K/2988032K available (8444K kernel code, 918K rwdata, 4072K rodata, 1024K init, 282K bss, 139276K reserved, 16384K cma-reserved)
+Virtual kernel memory layout:
+    modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
+    vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
+      .text : 0xffff000008080000 - 0xffff0000088c0000   (  8448 KB)
+    .rodata : 0xffff0000088c0000 - 0xffff000008cc0000   (  4096 KB)
+      .init : 0xffff000008cc0000 - 0xffff000008dc0000   (  1024 KB)
+      .data : 0xffff000008dc0000 - 0xffff000008ea5a00   (   919 KB)
+       .bss : 0xffff000008ea5a00 - 0xffff000008eec240   (   283 KB)
+    fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
+    PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
+    vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
+              0xffff7e0000000000 - 0xffff7e0003000000   (    48 MB actual)
+    memory  : 0xffff800000000000 - 0xffff8000c0000000   (  3072 MB)
+SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
+Preemptible hierarchical RCU implementation.
+	Build-time adjustment of leaf fanout to 64.
+	RCU restricting CPUs from NR_CPUS=64 to nr_cpu_ids=4.
+RCU: Adjusting geometry for rcu_fanout_leaf=64, nr_cpu_ids=4
+NR_IRQS:64 nr_irqs:64 0
+GIC: Using split EOI/Deactivate mode
+/interrupt-controller@60004000: 192 interrupts forwarded to /interrupt-controller@50041000
+boot misc0 0x0: expected 0x80000
+ (comparison mask = 0x400ffffb)
+boot misc2 0x1f720005: expected 0x1f720f05
+ (comparison mask = 0xffffff17)
+PLL_C already enabled. Postponing set full defaults
+PLL_MB already enabled. Postponing set full defaults
+PLL_U already enabled. Postponing set full defaults
+PLL_D already enabled. Postponing set full defaults
+PLL_DP already enabled. Postponing set full defaults
+boot misc0 0x0: expected 0x20
+ (comparison mask = 0x7ffffff)
+boot misc1 0x90000000: expected 0xc0000000
+ (comparison mask = 0xf8000000)
+boot misc2 0x0: expected 0xf400f0da
+ (comparison mask = 0xffffffff)
+boot misc3 0xbc90: expected 0x2004f400
+ (comparison mask = 0xffffffff)
+PLL_P already enabled. Postponing set full defaults
+PLL_X already enabled. Postponing set full defaults
+boot misc1 0x0: expected 0x20
+ (comparison mask = 0xffffff)
+arm_arch_timer: Architected cp15 timer(s) running at 19.20MHz (phys).
+clocksource: arch_sys_counter: mask: 0xffffffffffffff max_cycles: 0x46d987e47, max_idle_ns: 440795202767 ns
+sched_clock: 56 bits at 19MHz, resolution 52ns, wraps every 4398046511078ns
+Console: colour dummy device 80x25
+console [tty0] enabled
+bootconsole [uart0] disabled
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc6-next-20161122-160968-g637aaec (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Nov 23 03:36:31 PST 2016
+Linux version 4.9.0-rc6-next-20161123-161173-g272da04 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Nov 23 07:21:15 PST 2016
 Boot CPU: AArch64 Processor [411fd071]
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
@@ -62,7 +132,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0xbb88: expected 0x2004f400
+boot misc3 0xbc90: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -142,7 +212,7 @@
 Unpacking initramfs...
 Freeing initrd memory: 2364K
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 808ac000
+kvm [1]: IDMAP page: 808ab000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
@@ -150,7 +220,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.331:1): initialized
+audit: type=2000 audit(0.327:1): initialized
 workingset: timestamp_bits=46 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -239,9 +309,10 @@
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 BGND3R 29.1 GiB 
 mmcblk0boot0: mmc0:0001 BGND3R partition 1 4.00 MiB
-max77686-rtc max77620-rtc: setting system clock to 2000-09-03 17:05:12 UTC (968000712)
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
+max77686-rtc max77620-rtc: setting system clock to 2000-09-03 20:50:05 UTC (968014205)
 PP1200_AVDD: disabling
 PP1200_RCAM: disabling
 PP2800L_RCAM: disabling
@@ -249,11 +320,10 @@
 PP_LDO6: disabling
 PPLCD_VDD: disabling
 PP1800_LCDIO: disabling
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
-tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 PP1800_CAM: disabling
 ALSA device list:
   No soundcards found.
+tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 Freeing unused kernel memory: 1024K
 
 System initialization...
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-11-23 07:27:16.071598204 -0800
+++ /tmp/foob	2016-11-23 07:27:16.071598204 -0800
@@ -41,18 +41,18 @@
 	 ###############################################
 	 3.3 MiB/s
 done
-Bytes transferred = 7357624 (7044b8 hex)
+Bytes transferred = 7357464 (704418 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.9 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 51804 (ca5c hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x7044b8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x704418 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa5b ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc6-next-20161122-136535-g637aaec (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Nov 23 03:33:28 PST 2016
+Linux version 4.9.0-rc6-next-20161123-136740-g272da04 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Nov 23 07:18:30 PST 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -311,11 +311,11 @@
 mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB
+mmc0: new ultra high speed DDR50 SDHC card at address aaaa
 Primary GPT is invalid, using alternate GPT.
  mmcblk1: p1 p2 p3 p4 p5
-tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
-mmc0: new ultra high speed DDR50 SDHC card at address aaaa
 mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
+tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
  mmcblk0: p1 p2 p3
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -387,4 +387,4 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.858689] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.889090] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-11-23 07:27:16.079598229 -0800
+++ /tmp/foob	2016-11-23 07:27:16.079598229 -0800
@@ -36,21 +36,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #################################################
-	 3.5 MiB/s
+	 ##################################################
+	 3.3 MiB/s
 done
-Bytes transferred = 5489600 (53c3c0 hex)
+Bytes transferred = 5491024 (53c950 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.9 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 51804 (ca5c hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x53c3c0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x53c950 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa5b ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-rc6-next-20161122-136535-g637aaec (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Nov 23 03:33:12 PST 2016
+Linux version 4.9.0-rc6-next-20161123-136740-g272da04 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Nov 23 07:18:12 PST 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -131,7 +131,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?[    0.504842] hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
+No ATAGs?[    0.504476] hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 4 bytes.
 tegra-mc 7000f000.memory-controller: no memory timings for RAM code 3 registered
 vdd_5v_sata: Failed to request enable GPIO30: -517
@@ -383,4 +383,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.381028] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.371011] usb 2-1: new high-speed USB device number 2 using tegra-ehci
