Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2017-09-20 21:34:06.961683857 -0700
+++ /tmp/foob	2017-09-20 21:34:06.961683857 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-next-20170920-200864-g0b093a5 (jonathanh@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Tue Sep 19 21:56:05 PDT 2017
+Linux version 4.14.0-rc1-next-20170921-319325-g43ec4ba69622 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Sep 20 21:32:00 PDT 2017
 Boot CPU: AArch64 Processor [411fd070]
 Machine model: linux,dummy-virt
 Memory limited to 1024MB
@@ -8,7 +8,7 @@
 cma: Reserved 16 MiB at 0x000000007f000000
 NUMA: No NUMA configuration found
 NUMA: Faking a node at [mem 0x0000000000000000-0x000000007fffffff]
-NUMA: NODE_DATA [mem 0x7efeaf00-0x7efec9ff]
+NUMA: NODE_DATA [mem 0x7efe9f00-0x7efeb9ff]
 Zone ranges:
   DMA      [mem 0x0000000040000000-0x000000007fffffff]
   Normal   empty
@@ -21,7 +21,7 @@
 psci: Using standard PSCI v0.2 function IDs
 psci: Trusted OS migration not required
 random: fast init done
-percpu: Embedded 23 pages/cpu @ffff80003efd0000 s55936 r8192 d30080 u94208
+percpu: Embedded 23 pages/cpu @ffff80003efcf000 s55936 r8192 d30080 u94208
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
 CPU features: enabling workaround for ARM erratum 834220
@@ -29,15 +29,15 @@
 Policy zone: DMA
 Kernel command line: root=/dev/vda2 rw rootwait mem=1024M console=ttyAMA0,38400n8 init=/bin/bash
 PID hash table entries: 4096 (order: 3, 32768 bytes)
-Memory: 998976K/1048576K available (9212K kernel code, 1142K rwdata, 4592K rodata, 1152K init, 402K bss, 33216K reserved, 16384K cma-reserved)
+Memory: 998908K/1048576K available (9276K kernel code, 1142K rwdata, 4592K rodata, 1152K init, 402K bss, 33284K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff000008980000   (  9216 KB)
-    .rodata : 0xffff000008980000 - 0xffff000008e00000   (  4608 KB)
-      .init : 0xffff000008e00000 - 0xffff000008f20000   (  1152 KB)
-      .data : 0xffff000008f20000 - 0xffff00000903da00   (  1143 KB)
-       .bss : 0xffff00000903da00 - 0xffff0000090a24f0   (   403 KB)
+      .text : 0xffff000008080000 - 0xffff000008990000   (  9280 KB)
+    .rodata : 0xffff000008990000 - 0xffff000008e10000   (  4608 KB)
+      .init : 0xffff000008e10000 - 0xffff000008f30000   (  1152 KB)
+      .data : 0xffff000008f30000 - 0xffff00000904da00   (  1143 KB)
+       .bss : 0xffff00000904da00 - 0xffff0000090b24f0   (   403 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -76,8 +76,10 @@
 pinctrl core: initialized pinctrl subsystem
 DMI not present or invalid.
 NET: Registered protocol family 16
+audit: initializing netlink subsys (disabled)
+audit: type=2000 audit(0.184:1): state=initialized audit_enabled=0 res=1
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffff000008986000, 1 data @ ffff000008f25000)
+vdso: 2 pages (1 code @ ffff000008996000, 1 data @ ffff000008f35000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -113,8 +115,6 @@
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 hw perfevents: enabled with armv8_pmuv3 PMU driver, 1 counters available
 kvm [1]: HYP mode not available
-audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.812:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -179,7 +179,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2017-09-20 04:56:43 UTC (1505883403)
+rtc-pl031 9010000.pl031: setting system clock to 2017-09-21 04:33:09 UTC (1505968389)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-09-20 21:34:06.985683960 -0700
+++ /tmp/foob	2017-09-20 21:34:06.989683977 -0700
@@ -40,7 +40,7 @@
 	 #################################################################
 	 #################################################################
 	 #####################################
-	 3 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 8167944 (7ca208 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
@@ -49,7 +49,7 @@
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.6 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 35036 (88dc hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-next-20170920-200864-g0b093a5 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Tue Sep 19 21:51:39 PDT 2017
+Linux version 4.14.0-rc1-next-20170921-201084-g43ec4ba (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Sep 20 21:26:23 PDT 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -72,7 +72,7 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c158e380, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c158f000, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -86,7 +86,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993480K/2096128K available (10240K kernel code, 1626K rwdata, 4508K rodata, 2048K init, 405K bss, 37112K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993476K/2096128K available (10240K kernel code, 1629K rwdata, 4508K rodata, 2048K init, 405K bss, 37116K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -96,8 +96,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1596828   (1627 kB)
-       .bss : 0xc159fbb4 - 0xc1605374   ( 406 kB)
+      .data : 0xc1400000 - 0xc15974a8   (1630 kB)
+       .bss : 0xc15a0bb4 - 0xc1606374   ( 406 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -401,7 +401,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-11 11:47:32 UTC (950269652)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-12 11:20:03 UTC (950354403)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -413,5 +413,5 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.979394] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.979127] usb 1-1: new high-speed USB device number 2 using tegra-ehci
 asix 1-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0d:b5:16
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2017-09-20 21:34:06.993683994 -0700
+++ /tmp/foob	2017-09-20 21:34:06.997684012 -0700
@@ -40,18 +40,18 @@
 	 #######################################
 	 3.1 MiB/s
 done
-Bytes transferred = 6293360 (600770 hex)
+Bytes transferred = 6294928 (600d90 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.8 MiB/s
+	 2.6 MiB/s
 done
 Bytes transferred = 35036 (88dc hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x600770 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x600d90 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8db ... OK
@@ -60,7 +60,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-next-20170920-200864-g0b093a5 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Tue Sep 19 21:51:20 PDT 2017
+Linux version 4.14.0-rc1-next-20170921-201084-g43ec4ba (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Sep 20 21:26:29 PDT 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -70,7 +70,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0fb8400, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0fb9480, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -84,7 +84,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1997496K/2096128K available (9216K kernel code, 781K rwdata, 2796K rodata, 1024K init, 300K bss, 33096K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1997492K/2096128K available (9216K kernel code, 785K rwdata, 2796K rodata, 1024K init, 300K bss, 33100K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -94,8 +94,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
-      .data : 0xc0f00000 - 0xc0fc3520   ( 782 kB)
-       .bss : 0xc0fcd514 - 0xc10186ac   ( 301 kB)
+      .data : 0xc0f00000 - 0xc0fc45a0   ( 786 kB)
+       .bss : 0xc0fce514 - 0xc10196ac   ( 301 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -351,10 +351,10 @@
 iommu: Adding device snd-soc-dummy to group 57
 mmc0: new high speed SDHC card at address aaaa
 mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
- mmcblk0: p1 p2 p3
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
+ mmcblk0: p1 p2 p3
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
  mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
@@ -396,7 +396,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-11 11:45:03 UTC (950269503)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-12 11:22:29 UTC (950354549)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -410,4 +410,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.220283] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.220550] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-09-20 21:34:07.001684029 -0700
+++ /tmp/foob	2017-09-20 21:34:07.005684046 -0700
@@ -35,7 +35,7 @@
 	 #################################################################
 	 #################################################################
 	 #####################################
-	 3.5 MiB/s
+	 3.7 MiB/s
 done
 Bytes transferred = 8167944 (7ca208 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
@@ -44,7 +44,7 @@
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.6 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 67173 (10665 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
@@ -56,7 +56,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-next-20170920-200864-g0b093a5 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Tue Sep 19 21:51:39 PDT 2017
+Linux version 4.14.0-rc1-next-20170921-201084-g43ec4ba (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Sep 20 21:26:23 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -67,7 +67,7 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c158e380, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c158f000, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -81,7 +81,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993360K/2096128K available (10240K kernel code, 1626K rwdata, 4508K rodata, 2048K init, 405K bss, 37232K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993356K/2096128K available (10240K kernel code, 1629K rwdata, 4508K rodata, 2048K init, 405K bss, 37236K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -91,8 +91,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1596828   (1627 kB)
-       .bss : 0xc159fbb4 - 0xc1605374   ( 406 kB)
+      .data : 0xc1400000 - 0xc15974a8   (1630 kB)
+       .bss : 0xc15a0bb4 - 0xc1606374   ( 406 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -467,9 +467,9 @@
 pci 0000:00:01.0: PME# supported from D0 D1 D2 D3hot D3cold
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x4c may corrupt adjacent RW1C bits
 pci 0000:00:02.0: [10de:0e13] type 01 class 0x060400
-pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4 may corrupt adjacent RW1C bits
 ata1: SATA link down (SStatus 0 SControl 300)
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4 may corrupt adjacent RW1C bits
+pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4 may corrupt adjacent RW1C bits
 pci 0000:00:02.0: enabling Extended Tags
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x88 may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x52 may corrupt adjacent RW1C bits
@@ -529,7 +529,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:38 UTC (946684958)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:14 UTC (946684814)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2017-09-20 21:34:07.013684081 -0700
+++ /tmp/foob	2017-09-20 21:34:07.013684081 -0700
@@ -29,24 +29,24 @@
 Loading: *#################################################################
 	 #################################################################
 	 #################################################################
-	 #################################################################
+	 ######T T ###########################################################
 	 #################################################################
 	 #################################################################
 	 #######################################
-	 3.5 MiB/s
+	 533.2 KiB/s
 done
-Bytes transferred = 6293360 (600770 hex)
+Bytes transferred = 6294928 (600d90 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.4 MiB/s
+	 3.8 MiB/s
 done
 Bytes transferred = 67173 (10665 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x600770 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x600d90 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013664
@@ -55,7 +55,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-next-20170920-200864-g0b093a5 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Tue Sep 19 21:51:20 PDT 2017
+Linux version 4.14.0-rc1-next-20170921-201084-g43ec4ba (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Sep 20 21:26:29 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -65,7 +65,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0fb8400, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0fb9480, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -79,7 +79,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1997376K/2096128K available (9216K kernel code, 781K rwdata, 2796K rodata, 1024K init, 300K bss, 33216K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1997372K/2096128K available (9216K kernel code, 785K rwdata, 2796K rodata, 1024K init, 300K bss, 33220K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -89,8 +89,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
-      .data : 0xc0f00000 - 0xc0fc3520   ( 782 kB)
-       .bss : 0xc0fcd514 - 0xc10186ac   ( 301 kB)
+      .data : 0xc0f00000 - 0xc0fc45a0   ( 786 kB)
+       .bss : 0xc0fce514 - 0xc10196ac   ( 301 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -458,8 +458,8 @@
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
 tegra-pcie 1003000.pcie: 2x1, 1x1 configuration
 tegra-pcie 1003000.pcie: probing port 0, using 2 lanes
-tegra-pcie 1003000.pcie: Slot present pin change, signature: 00000008
 ata1: SATA link down (SStatus 0 SControl 300)
+tegra-pcie 1003000.pcie: Slot present pin change, signature: 00000008
 tegra-pcie 1003000.pcie: probing port 1, using 1 lanes
 tegra-pcie 1003000.pcie: Slot present pin change, signature: 00000000
 tegra-pcie 1003000.pcie: PCI host bridge to bus 0000:00
@@ -498,8 +498,8 @@
 pci 0000:02:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit]
 pci 0000:02:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
 pci 0000:02:00.0: supports D1 D2
- mmcblk1: p1 p2 p3
 pci 0000:02:00.0: PME# supported from D0 D1 D2 D3hot D3cold
+ mmcblk1: p1 p2 p3
 PCI: bus2: Fast back to back transfers disabled
 pci_bus 0000:02: busn_res: [bus 02-ff] end is updated to 02
 pci 0000:00:01.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
@@ -537,7 +537,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:14 UTC (946684814)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:25 UTC (946684825)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-09-20 21:34:07.021684115 -0700
+++ /tmp/foob	2017-09-20 21:34:07.021684115 -0700
@@ -43,7 +43,7 @@
 	 #################################################################
 	 #################################################################
 	 #####################################
-	 3 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 8167944 (7ca208 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
@@ -52,7 +52,7 @@
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 2.7 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 68203 (10a6b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
@@ -64,7 +64,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-next-20170920-200864-g0b093a5 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Tue Sep 19 21:51:39 PDT 2017
+Linux version 4.14.0-rc1-next-20170921-201084-g43ec4ba (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Sep 20 21:26:23 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -76,7 +76,7 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c158e380, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c158f000, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -90,7 +90,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1994376K/2097148K available (10240K kernel code, 1626K rwdata, 4508K rodata, 2048K init, 405K bss, 37236K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1994372K/2097148K available (10240K kernel code, 1629K rwdata, 4508K rodata, 2048K init, 405K bss, 37240K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -100,8 +100,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1596828   (1627 kB)
-       .bss : 0xc159fbb4 - 0xc1605374   ( 406 kB)
+      .data : 0xc1400000 - 0xc15974a8   (1630 kB)
+       .bss : 0xc15a0bb4 - 0xc1606374   ( 406 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -377,6 +377,8 @@
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc0: Unknown controller version (3). You may experience problems.
 mmc0: Invalid maximum block size, assuming 512 bytes
+elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
+input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 mmc0: SDHCI controller on 700b0600.sdhci [700b0600.sdhci] using ADMA
 ledtrig-cpu: registered to indicate activity on CPUs
 usbcore: registered new interface driver usbhid
@@ -405,29 +407,29 @@
 +3.3V_PANEL: supplied by +3.3V_RUN
 +3.3V_LP0: supplied by +3.3V_SYS
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
-elan_i2c 1-0015: fetching report descriptor failed.: -121
+elan_i2c 1-0015: failed to read report data: -121
 +5V_HDMI_CON: supplied by +5V_SYS
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
  usb3-1: usb3-1 supply vbus not found, using dummy regulator
-mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
-mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
-mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
-mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
 sdhci-tegra 700b0000.sdhci: allocated mmc-pwrseq
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 mmc1: Unknown controller version (3). You may experience problems.
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: Switching to 3.3V signalling voltage failed
+mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
+mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
+mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
+mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
+elan_i2c 1-0015: invalid report id data (ff)
 mmc1: SDHCI controller on 700b0000.sdhci [700b0000.sdhci] using ADMA
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc2: Unknown controller version (3). You may experience problems.
 mmc2: Invalid maximum block size, assuming 512 bytes
 mmc1: new high speed SDIO card at address 0001
-elan_i2c 1-0015: device initialize failed: -121
 mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
@@ -440,12 +442,10 @@
 GPT: Use GNU Parted to correct GPT errors.
  mmcblk2: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 Console: switching to colour frame buffer device 170x48
-elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
-input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 16:54:50 UTC (946745690)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-02 16:29:57 UTC (946830597)
 +1.8V_RUN_CAM: disabling
 +3.3V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2017-09-20 21:34:07.029684150 -0700
+++ /tmp/foob	2017-09-20 21:34:07.029684150 -0700
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -41,20 +41,20 @@
 	 #################################################################
 	 #################################################################
 	 #######################################
-	 2.9 MiB/s
+	 3.6 MiB/s
 done
-Bytes transferred = 6293360 (600770 hex)
+Bytes transferred = 6294928 (600d90 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.1 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 68203 (10a6b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x600770 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x600d90 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a6a
@@ -63,7 +63,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-next-20170920-200864-g0b093a5 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Tue Sep 19 21:51:20 PDT 2017
+Linux version 4.14.0-rc1-next-20170921-201084-g43ec4ba (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Sep 20 21:26:29 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -74,7 +74,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c0fb8400, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0fb9480, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -88,7 +88,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998392K/2097148K available (9216K kernel code, 781K rwdata, 2796K rodata, 1024K init, 300K bss, 33220K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1998388K/2097148K available (9216K kernel code, 785K rwdata, 2796K rodata, 1024K init, 300K bss, 33224K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -98,8 +98,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
-      .data : 0xc0f00000 - 0xc0fc3520   ( 782 kB)
-       .bss : 0xc0fcd514 - 0xc10186ac   ( 301 kB)
+      .data : 0xc0f00000 - 0xc0fc45a0   ( 786 kB)
+       .bss : 0xc0fce514 - 0xc10196ac   ( 301 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -312,14 +312,14 @@
 kernel BUG at drivers/platform/chrome/cros_ec_proto.c:34!
 Internal error: Oops - BUG: 0 [#1] PREEMPT SMP ARM
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.14.0-rc1-next-20170920-200864-g0b093a5 #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.14.0-rc1-next-20170921-201084-g43ec4ba #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 task: ee891740 task.stack: ee8a6000
 PC is at cros_ec_prepare_tx+0x168/0x16c
 LR is at cros_ec_pkt_xfer_spi+0x18/0x474
-pc : [<c06d7d64>]    lr : [<c05459ec>]    psr: 20000013
+pc : [<c06d7ed0>]    lr : [<c0545a84>]    psr: 20000013
 sp : ee8a7bd8  ip : 00000000  fp : 00000000
-r10: c05459d4  r9 : 00000000  r8 : eefe503c
+r10: c0545a6c  r9 : 00000000  r8 : eefe503c
 r7 : ee2c6000  r6 : ee2c6000  r5 : ee2c6000  r4 : ee283d90
 r3 : 0000ffff  r2 : 00000000  r1 : 0000ffff  r0 : ee283d90
 Flags: nzCv  IRQs on  FIQs on  Mode SVC_32  ISA ARM  Segment none
@@ -327,131 +327,131 @@
 Process swapper/0 (pid: 1, stack limit = 0xee8a6210)
 Stack: (0xee8a7bd8 to 0xee8a8000)
 7bc0:                                                       ee283d90 ee2c6000
-7be0: ee2c6090 c05459ec c0976e94 00000004 ee8a7c5c ee8a7c5c ee2bbc00 00000000
-7c00: c0591dbc 00000000 00000008 00000008 00000000 ee8a7c14 ee8a7c14 00000000
+7be0: ee2c6090 c0545a84 c0977174 00000004 ee8a7c5c ee8a7c5c ee2bbc00 00000000
+7c00: c0591e54 00000000 00000008 00000008 00000000 ee8a7c14 ee8a7c14 00000000
 7c20: ee8a7c20 ee8a7c20 ee2c6050 ee2c6780 00000008 00000000 00000000 00000000
-7c40: c0e91d08 c0238b78 40002800 000003e0 eef8cd08 c023afe8 002dc6c0 ee8a7bf0
-7c60: ee8a7bf0 00000000 c06d7b94 005c58c0 3b9aca00 000f4240 00000000 ee8a7ca8
-7c80: 00000001 00000501 c05459d4 c097b738 00000000 ee2c67c0 a0000013 c06d7b94
-7ca0: ee2c6000 ee283d90 ee2c6000 ee2bbc00 ee2c6000 eefe503c 00000000 c05459d4
-7cc0: 00000000 c06d7af0 0000400b ee283d90 ee2c6000 ee2bbc00 0000000c eefe503c
-7ce0: c0ff8e7c c06d7f0c 00000000 ee283d90 ee2bbc00 00000000 00000000 eefe503c
-7d00: c0f6b7a4 c0545478 ee8a7d14 ee2bbc00 00000000 00000001 ee2bbc00 ee283d90
-7d20: ee2c6090 c0545828 00000000 00000000 00000000 000007d0 ee2bbc00 00000000
-7d40: c0f6b794 00000000 00000000 c0592800 ee2bbc00 c100a204 c100a208 c051e828
-7d60: 00000000 ee8a7d98 c051e9b8 00000001 c100a1e0 00000000 00000000 c051ce48
-7d80: ee9f6170 ee189db8 ee2bbc00 ee2bbc34 c0f72464 c051e568 ee2bbc00 00000001
-7da0: 00000000 ee2bbc00 ee2bbc00 c0f72464 ee0f6000 c051dbac ee2bbc00 00000000
-7dc0: ee2bbc08 c051c1f0 ee2bbc00 c0595714 00000000 ee2bbc00 ee2bbc00 ee0f6000
-7de0: 00000000 eea8fe10 eefe508c 00000001 00000000 c0592654 002dc6c0 ee0f6000
-7e00: eefe503c 00000000 ee2bbc00 c0594600 00000000 014000c0 00000000 c0593fec
-7e20: 00000063 002dc6c0 ee0f6000 ee2c6350 eea8fe00 eea8fe10 40000000 00000153
-7e40: 00000000 c0594ac0 eefe4d2c ee0f6348 ee0f6000 eea8fe00 eea8fe10 c059564c
-7e60: 00000000 ee0f6348 ffffffed eea8fe10 fffffdfb c0f72dc4 00000000 00000000
-7e80: c0f72dc4 c052011c c05200cc eea8fe10 c100a204 c100a208 00000000 c051e828
-7ea0: eea8fe10 c0f72dc4 eea8fe44 00000000 00000111 c0e6b83c c0e005a4 c051e980
-7ec0: 00000000 c0f72dc4 c051e8d4 c051cee4 ee8c955c eea18934 c0f72dc4 ee2b8680
-7ee0: c0f69d70 c051dd80 c0c1cdc4 c0f72dc4 00000006 c0f72dc4 00000006 c0e38804
-7f00: 00000000 c051f4f8 c051fd70 c0e83ca8 00000006 c0101b8c 00000000 c0cba3e8
+7c40: c0e91d08 c0238bfc 40002800 000003e0 eef8cd08 c023b06c 002dc6c0 ee8a7bf0
+7c60: ee8a7bf0 00000000 c06d7d00 005c58c0 3b9aca00 000f4240 00000000 ee8a7ca8
+7c80: 00000001 00000501 c0545a6c c097ba18 00000000 ee2c67c0 a0000013 c06d7d00
+7ca0: ee2c6000 ee283d90 ee2c6000 ee2bbc00 ee2c6000 eefe503c 00000000 c0545a6c
+7cc0: 00000000 c06d7c5c 0000400b ee283d90 ee2c6000 ee2bbc00 0000000c eefe503c
+7ce0: c0ff9e7c c06d8078 00000000 ee283d90 ee2bbc00 00000000 00000000 eefe503c
+7d00: c0f6c080 c0545510 ee8a7d14 ee2bbc00 00000000 00000001 ee2bbc00 ee283d90
+7d20: ee2c6090 c05458c0 00000000 00000000 00000000 000007d0 ee2bbc00 00000000
+7d40: c0f6c070 00000000 00000000 c0592898 ee2bbc00 c100b204 c100b208 c051e8c0
+7d60: 00000000 ee8a7d98 c051ea50 00000001 c100b1e0 00000000 00000000 c051cee0
+7d80: ee9f6170 ee189db8 ee2bbc00 ee2bbc34 c0f72d8c c051e600 ee2bbc00 00000001
+7da0: 00000000 ee2bbc00 ee2bbc00 c0f72d8c ee07e000 c051dc44 ee2bbc00 00000000
+7dc0: ee2bbc08 c051c288 ee2bbc00 c05957ac 00000000 ee2bbc00 ee2bbc00 ee07e000
+7de0: 00000000 eea8fe10 eefe508c 00000001 00000000 c05926ec 002dc6c0 ee07e000
+7e00: eefe503c 00000000 ee2bbc00 c0594698 00000000 014000c0 00000000 c0594084
+7e20: 00000063 002dc6c0 ee07e000 ee2c6350 eea8fe00 eea8fe10 40000000 00000153
+7e40: 00000000 c0594b58 eefe4d2c ee07e348 ee07e000 eea8fe00 eea8fe10 c05956e4
+7e60: 00000000 ee07e348 ffffffed eea8fe10 fffffdfb c0f73708 00000000 00000000
+7e80: c0f73708 c05201b4 c0520164 eea8fe10 c100b204 c100b208 00000000 c051e8c0
+7ea0: eea8fe10 c0f73708 eea8fe44 00000000 00000111 c0e6b83c c0e005a4 c051ea18
+7ec0: 00000000 c0f73708 c051e96c c051cf7c ee8c955c eea18934 c0f73708 ee2b8680
+7ee0: c0f6a610 c051de18 c0c1ce4c c0f73708 00000006 c0f73708 00000006 c0e38804
+7f00: 00000000 c051f590 c051fe08 c0e83ca8 00000006 c0101b8c 00000000 c0cba3e8
 7f20: 00000100 efffcc45 c0cba3e8 00000111 efffcc53 c013e3d4 c0cb8e94 00000000
 7f40: 00000006 00000006 efffcc45 efffcc4a 00000000 efffcbc0 00000006 c0e6b82c
-7f60: c0e83ca8 00000006 c0e6b830 c0fcd540 00000111 c0e00da8 00000006 00000006
-7f80: 00000000 c0e005a4 00000000 c0974c28 00000000 00000000 00000000 00000000
-7fa0: 00000000 c0974c30 00000000 c0107cf0 00000000 00000000 00000000 00000000
+7f60: c0e83ca8 00000006 c0e6b830 c0fce540 00000111 c0e00da8 00000006 00000006
+7f80: 00000000 c0e005a4 00000000 c0974f08 00000000 00000000 00000000 00000000
+7fa0: 00000000 c0974f10 00000000 c0107cf0 00000000 00000000 00000000 00000000
 7fc0: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
 7fe0: 00000000 00000000 00000000 00000000 00000013 00000000 00000000 00000000
-[<c06d7d64>] (cros_ec_prepare_tx) from [<c05459ec>] (cros_ec_pkt_xfer_spi+0x18/0x474)
-[<c05459ec>] (cros_ec_pkt_xfer_spi) from [<c06d7af0>] (send_command+0x20/0xd8)
-[<c06d7af0>] (send_command) from [<c06d7f0c>] (cros_ec_query_all+0x1a4/0x398)
-[<c06d7f0c>] (cros_ec_query_all) from [<c0545478>] (cros_ec_register+0x94/0x21c)
-[<c0545478>] (cros_ec_register) from [<c0545828>] (cros_ec_spi_probe+0x108/0x150)
-[<c0545828>] (cros_ec_spi_probe) from [<c0592800>] (spi_drv_probe+0x7c/0xa8)
-[<c0592800>] (spi_drv_probe) from [<c051e828>] (driver_probe_device+0x238/0x2e4)
-[<c051e828>] (driver_probe_device) from [<c051ce48>] (bus_for_each_drv+0x44/0x8c)
-[<c051ce48>] (bus_for_each_drv) from [<c051e568>] (__device_attach+0x9c/0x100)
-[<c051e568>] (__device_attach) from [<c051dbac>] (bus_probe_device+0x84/0x8c)
-[<c051dbac>] (bus_probe_device) from [<c051c1f0>] (device_add+0x380/0x528)
-[<c051c1f0>] (device_add) from [<c0592654>] (spi_add_device+0x88/0x130)
-[<c0592654>] (spi_add_device) from [<c0594600>] (spi_register_controller+0x428/0x8b4)
-[<c0594600>] (spi_register_controller) from [<c0594ac0>] (devm_spi_register_controller+0x34/0x70)
-[<c0594ac0>] (devm_spi_register_controller) from [<c059564c>] (tegra_spi_probe+0x298/0x35c)
-[<c059564c>] (tegra_spi_probe) from [<c052011c>] (platform_drv_probe+0x50/0xb0)
-[<c052011c>] (platform_drv_probe) from [<c051e828>] (driver_probe_device+0x238/0x2e4)
-[<c051e828>] (driver_probe_device) from [<c051e980>] (__driver_attach+0xac/0xb0)
-[<c051e980>] (__driver_attach) from [<c051cee4>] (bus_for_each_dev+0x54/0x88)
-[<c051cee4>] (bus_for_each_dev) from [<c051dd80>] (bus_add_driver+0xe4/0x1f0)
-[<c051dd80>] (bus_add_driver) from [<c051f4f8>] (driver_register+0x78/0xf4)
-[<c051f4f8>] (driver_register) from [<c0101b8c>] (do_one_initcall+0x3c/0x168)
+[<c06d7ed0>] (cros_ec_prepare_tx) from [<c0545a84>] (cros_ec_pkt_xfer_spi+0x18/0x474)
+[<c0545a84>] (cros_ec_pkt_xfer_spi) from [<c06d7c5c>] (send_command+0x20/0xd8)
+[<c06d7c5c>] (send_command) from [<c06d8078>] (cros_ec_query_all+0x1a4/0x398)
+[<c06d8078>] (cros_ec_query_all) from [<c0545510>] (cros_ec_register+0x94/0x21c)
+[<c0545510>] (cros_ec_register) from [<c05458c0>] (cros_ec_spi_probe+0x108/0x150)
+[<c05458c0>] (cros_ec_spi_probe) from [<c0592898>] (spi_drv_probe+0x7c/0xa8)
+[<c0592898>] (spi_drv_probe) from [<c051e8c0>] (driver_probe_device+0x238/0x2e4)
+[<c051e8c0>] (driver_probe_device) from [<c051cee0>] (bus_for_each_drv+0x44/0x8c)
+[<c051cee0>] (bus_for_each_drv) from [<c051e600>] (__device_attach+0x9c/0x100)
+[<c051e600>] (__device_attach) from [<c051dc44>] (bus_probe_device+0x84/0x8c)
+[<c051dc44>] (bus_probe_device) from [<c051c288>] (device_add+0x380/0x528)
+[<c051c288>] (device_add) from [<c05926ec>] (spi_add_device+0x88/0x130)
+[<c05926ec>] (spi_add_device) from [<c0594698>] (spi_register_controller+0x428/0x8b4)
+[<c0594698>] (spi_register_controller) from [<c0594b58>] (devm_spi_register_controller+0x34/0x70)
+[<c0594b58>] (devm_spi_register_controller) from [<c05956e4>] (tegra_spi_probe+0x298/0x35c)
+[<c05956e4>] (tegra_spi_probe) from [<c05201b4>] (platform_drv_probe+0x50/0xb0)
+[<c05201b4>] (platform_drv_probe) from [<c051e8c0>] (driver_probe_device+0x238/0x2e4)
+[<c051e8c0>] (driver_probe_device) from [<c051ea18>] (__driver_attach+0xac/0xb0)
+[<c051ea18>] (__driver_attach) from [<c051cf7c>] (bus_for_each_dev+0x54/0x88)
+[<c051cf7c>] (bus_for_each_dev) from [<c051de18>] (bus_add_driver+0xe4/0x1f0)
+[<c051de18>] (bus_add_driver) from [<c051f590>] (driver_register+0x78/0xf4)
+[<c051f590>] (driver_register) from [<c0101b8c>] (do_one_initcall+0x3c/0x168)
 [<c0101b8c>] (do_one_initcall) from [<c0e00da8>] (kernel_init_freeable+0x10c/0x1d0)
-[<c0e00da8>] (kernel_init_freeable) from [<c0974c30>] (kernel_init+0x8/0x114)
-[<c0974c30>] (kernel_init) from [<c0107cf0>] (ret_from_fork+0x14/0x24)
+[<c0e00da8>] (kernel_init_freeable) from [<c0974f10>] (kernel_init+0x8/0x114)
+[<c0974f10>] (kernel_init) from [<c0107cf0>] (ret_from_fork+0x14/0x24)
 Code: e2800008 e8bd8070 e7f001f2 e7f001f2 (e7f001f2) 
----[ end trace 3742da5a7ed46a52 ]---
+---[ end trace f65ce7b9da0e7b2e ]---
 Kernel panic - not syncing: Attempted to kill init! exitcode=0x0000000b
 
-CPU2: stopping
-CPU: 2 PID: 0 Comm: swapper/2 Tainted: G      D         4.14.0-rc1-next-20170920-200864-g0b093a5 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c01110d0>] (unwind_backtrace) from [<c010c08c>] (show_stack+0x10/0x14)
-[<c010c08c>] (show_stack) from [<c09614c0>] (dump_stack+0x84/0xa4)
-[<c09614c0>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
-[<c010f61c>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
-[<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
-Exception stack(0xee8e1f70 to 0xee8e1fb8)
-1f60:                                     00000001 00000000 ee8e1fc8 c011a3a0
-1f80: c0f03ccc c0f03c68 c0e8d378 c0f03cd8 c0fbe1fd 00000000 00000001 00000000
-1fa0: 2e11d000 ee8e1fc0 c0108758 c010875c 60000013 ffffffff
-[<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
-[<c010875c>] (arch_cpu_idle) from [<c0160c78>] (do_idle+0x148/0x204)
-[<c0160c78>] (do_idle) from [<c0160fbc>] (cpu_startup_entry+0x18/0x1c)
-[<c0160fbc>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
 CPU3: stopping
-CPU: 3 PID: 0 Comm: swapper/3 Tainted: G      D         4.14.0-rc1-next-20170920-200864-g0b093a5 #1
+CPU: 3 PID: 0 Comm: swapper/3 Tainted: G      D         4.14.0-rc1-next-20170921-201084-g43ec4ba #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c01110d0>] (unwind_backtrace) from [<c010c08c>] (show_stack+0x10/0x14)
-[<c010c08c>] (show_stack) from [<c09614c0>] (dump_stack+0x84/0xa4)
-[<c09614c0>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
+[<c010c08c>] (show_stack) from [<c09617a0>] (dump_stack+0x84/0xa4)
+[<c09617a0>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
 [<c010f61c>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
 Exception stack(0xee8e3f70 to 0xee8e3fb8)
 3f60:                                     00000001 00000000 ee8e3fc8 c011a3a0
-3f80: c0f03ccc c0f03c68 c0e8d378 c0f03cd8 c0fbe1fd 00000000 00000001 00000000
+3f80: c0f03ccc c0f03c68 c0e8d378 c0f03cd8 c0fbf27d 00000000 00000001 00000000
 3fa0: 2e12e000 ee8e3fc0 c0108758 c010875c 60000013 ffffffff
 [<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
 [<c010875c>] (arch_cpu_idle) from [<c0160c78>] (do_idle+0x148/0x204)
 [<c0160c78>] (do_idle) from [<c0160fbc>] (cpu_startup_entry+0x18/0x1c)
 [<c0160fbc>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
 CPU1: stopping
-CPU: 1 PID: 0 Comm: swapper/1 Tainted: G      D         4.14.0-rc1-next-20170920-200864-g0b093a5 #1
+CPU: 1 PID: 0 Comm: swapper/1 Tainted: G      D         4.14.0-rc1-next-20170921-201084-g43ec4ba #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c01110d0>] (unwind_backtrace) from [<c010c08c>] (show_stack+0x10/0x14)
-[<c010c08c>] (show_stack) from [<c09614c0>] (dump_stack+0x84/0xa4)
-[<c09614c0>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
+[<c010c08c>] (show_stack) from [<c09617a0>] (dump_stack+0x84/0xa4)
+[<c09617a0>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
 [<c010f61c>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
 Exception stack(0xee8dff70 to 0xee8dffb8)
 ff60:                                     00000001 00000000 ee8dffc8 c011a3a0
-ff80: c0f03ccc c0f03c68 c0e8d378 c0f03cd8 c0fbe1fd 00000000 00000001 00000000
+ff80: c0f03ccc c0f03c68 c0e8d378 c0f03cd8 c0fbf27d 00000000 00000001 00000000
 ffa0: 2e10c000 ee8dffc0 c0108758 c010875c 60000013 ffffffff
 [<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
 [<c010875c>] (arch_cpu_idle) from [<c0160c78>] (do_idle+0x148/0x204)
 [<c0160c78>] (do_idle) from [<c0160fbc>] (cpu_startup_entry+0x18/0x1c)
 [<c0160fbc>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
+CPU2: stopping
+CPU: 2 PID: 0 Comm: swapper/2 Tainted: G      D         4.14.0-rc1-next-20170921-201084-g43ec4ba #1
+Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
+[<c01110d0>] (unwind_backtrace) from [<c010c08c>] (show_stack+0x10/0x14)
+[<c010c08c>] (show_stack) from [<c09617a0>] (dump_stack+0x84/0xa4)
+[<c09617a0>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
+[<c010f61c>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
+[<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
+Exception stack(0xee8e1f70 to 0xee8e1fb8)
+1f60:                                     00000001 00000000 ee8e1fc8 c011a3a0
+1f80: c0f03ccc c0f03c68 c0e8d378 c0f03cd8 c0fbf27d 00000000 00000001 00000000
+1fa0: 2e11d000 ee8e1fc0 c0108758 c010875c 60000013 ffffffff
+[<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
+[<c010875c>] (arch_cpu_idle) from [<c0160c78>] (do_idle+0x148/0x204)
+[<c0160c78>] (do_idle) from [<c0160fbc>] (cpu_startup_entry+0x18/0x1c)
+[<c0160fbc>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
 ---[ end Kernel panic - not syncing: Attempted to kill init! exitcode=0x0000000b
 
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 8 at kernel/irq_work.c:68 irq_work_queue_on+0xbc/0xfc
 Modules linked in:
-CPU: 0 PID: 8 Comm: rcu_preempt Tainted: G      D         4.14.0-rc1-next-20170920-200864-g0b093a5 #1
+CPU: 0 PID: 8 Comm: rcu_preempt Tainted: G      D         4.14.0-rc1-next-20170921-201084-g43ec4ba #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c01110d0>] (unwind_backtrace) from [<c010c08c>] (show_stack+0x10/0x14)
-[<c010c08c>] (show_stack) from [<c09614c0>] (dump_stack+0x84/0xa4)
-[<c09614c0>] (dump_stack) from [<c0121908>] (__warn+0xd8/0x104)
+[<c010c08c>] (show_stack) from [<c09617a0>] (dump_stack+0x84/0xa4)
+[<c09617a0>] (dump_stack) from [<c0121908>] (__warn+0xd8/0x104)
 [<c0121908>] (__warn) from [<c0121954>] (warn_slowpath_null+0x20/0x28)
-[<c0121954>] (warn_slowpath_null) from [<c01d3f68>] (irq_work_queue_on+0xbc/0xfc)
-[<c01d3f68>] (irq_work_queue_on) from [<c0181fb4>] (rcu_implicit_dynticks_qs+0x280/0x2a0)
+[<c0121954>] (warn_slowpath_null) from [<c01d3fe0>] (irq_work_queue_on+0xbc/0xfc)
+[<c01d3fe0>] (irq_work_queue_on) from [<c0181fb4>] (rcu_implicit_dynticks_qs+0x280/0x2a0)
 [<c0181fb4>] (rcu_implicit_dynticks_qs) from [<c0181158>] (force_qs_rnp+0x140/0x1e0)
 [<c0181158>] (force_qs_rnp) from [<c0181730>] (rcu_gp_kthread+0x538/0xa6c)
 [<c0181730>] (rcu_gp_kthread) from [<c013ec84>] (kthread+0x120/0x158)
 [<c013ec84>] (kthread) from [<c0107cf0>] (ret_from_fork+0x14/0x24)
----[ end trace 3742da5a7ed46a53 ]---
+---[ end trace f65ce7b9da0e7b2f ]---
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2017-09-20 21:34:07.037684184 -0700
+++ /tmp/foob	2017-09-20 21:34:07.037684184 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1801006 us
+Bootloader-Cpu Init at (time stamp): 1800848 us
 
 Pinmux changes applied in kernel way
 
@@ -62,22 +62,22 @@
 Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1800980  us
-Bootloader (AosInit) time: 734240 us
-Bootloader Main Init time: 180084 us
-BlockDevMgrInit time: 157539 us
+Miscellaneous avp time: 1800822  us
+Bootloader (AosInit) time: 734237 us
+Bootloader Main Init time: 180058 us
+BlockDevMgrInit time: 157536 us
 PartMgrInit time: 8 us
-StorMgrInit time: 30 us
-FsMgrInit time: 3 us
-Partition Table load: 76733 us 
-Display init time: 574801 us
-Kernel read time: 1345 msec
-Load Kernel Size:   17825792 Bytes
-Bl Kernel ReadSpeed: 13253000 Bytes/sec
-Pre-Kernel Jump time: 3838893 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 8802869 us
+StorMgrInit time: 31 us
+FsMgrInit time: 2 us
+Partition Table load: 76661 us 
+Display init time: 574841 us
+Kernel read time: 1350 msec
+Load Kernel Size:   17891328 Bytes
+Bl Kernel ReadSpeed: 13252000 Bytes/sec
+Pre-Kernel Jump time: 3851607 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 8820554 us
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-next-20170920-200864-g0b093a5 (jonathanh@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Tue Sep 19 21:56:05 PDT 2017
+Linux version 4.14.0-rc1-next-20170921-319325-g43ec4ba69622 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Sep 20 21:32:00 PDT 2017
 Boot CPU: AArch64 Processor [4e0f0000]
 Machine model: NVIDIA Tegra132 Norrin
 Memory limited to 1734MB
@@ -86,7 +86,7 @@
 cma: Reserved 16 MiB at 0x00000000eb400000
 NUMA: No NUMA configuration found
 NUMA: Faking a node at [mem 0x0000000000000000-0x00000000ec5fffff]
-NUMA: NODE_DATA [mem 0xec5c0d80-0xec5c287f]
+NUMA: NODE_DATA [mem 0xec5bfd80-0xec5c187f]
 Zone ranges:
   DMA      [mem 0x0000000080000000-0x00000000ec5fffff]
   Normal   empty
@@ -96,21 +96,21 @@
 Initmem setup node 0 [mem 0x0000000080000000-0x00000000ec5fffff]
 /cpus/cpu@1: missing enable-method property
 random: fast init done
-percpu: Embedded 23 pages/cpu @ffff80006c5a4000 s55936 r8192 d30080 u94208
+percpu: Embedded 23 pages/cpu @ffff80006c5a3000 s55936 r8192 d30080 u94208
 Detected PIPT I-cache on CPU0
 Built 1 zonelists, mobility grouping on.  Total pages: 436968
 Policy zone: DMA
 Kernel command line: tegraid=13.1.3.0.0 mem=1734M@2048M memtype=255 vpr=151M@3817M ddr_die=2048M@2048M section=256M commchip_id=0 usb_port_owner_info=0 lane_owner_info=0 emc_max_dvfs=0 touch_id=0@0 video=tegrafb no_console_suspend=1 console=ttyS0,115200n8 debug_uartport=lsport,0 earlyprintk=uart8250-32bit,0x70006000 usbcore.old_scheme_first=1 lp0_vec=0@0x0 tegra_fbmem=32899072@0xad012000 core_edp_mv=1150 core_edp_ma=4000 pmuboard=0x0176:0x0001:0x01:0x43:0x00 displayboard=0x0162:0x0001:0x00:0x42:0x00 power_supply=Adapter audio_codec=rt5640 board_info=0x0176:0x0001:0x01:0x43:0x00 tegraboot=sdmmc gpt gpt_sector=192511 modem_id=0 android.kerneltype=normal
 PID hash table entries: 4096 (order: 3, 32768 bytes)
-Memory: 1708200K/1775616K available (9212K kernel code, 1142K rwdata, 4592K rodata, 1152K init, 402K bss, 51032K reserved, 16384K cma-reserved)
+Memory: 1708132K/1775616K available (9276K kernel code, 1142K rwdata, 4592K rodata, 1152K init, 402K bss, 51100K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff000008980000   (  9216 KB)
-    .rodata : 0xffff000008980000 - 0xffff000008e00000   (  4608 KB)
-      .init : 0xffff000008e00000 - 0xffff000008f20000   (  1152 KB)
-      .data : 0xffff000008f20000 - 0xffff00000903da00   (  1143 KB)
-       .bss : 0xffff00000903da00 - 0xffff0000090a24f0   (   403 KB)
+      .text : 0xffff000008080000 - 0xffff000008990000   (  9280 KB)
+    .rodata : 0xffff000008990000 - 0xffff000008e10000   (  4608 KB)
+      .init : 0xffff000008e10000 - 0xffff000008f30000   (  1152 KB)
+      .data : 0xffff000008f30000 - 0xffff00000904da00   (  1143 KB)
+       .bss : 0xffff00000904da00 - 0xffff0000090b24f0   (   403 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -149,8 +149,10 @@
 pinctrl core: initialized pinctrl subsystem
 DMI not present or invalid.
 NET: Registered protocol family 16
+audit: initializing netlink subsys (disabled)
+audit: type=2000 audit(0.128:1): state=initialized audit_enabled=0 res=1
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffff000008986000, 1 data @ ffff000008f25000)
+vdso: 2 pages (1 code @ ffff000008996000, 1 data @ ffff000008f35000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -199,8 +201,6 @@
 Unpacking initramfs...
 Freeing initrd memory: 1284K
 kvm [1]: HYP mode not available
-audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.975:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -271,14 +271,15 @@
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
-+VDD_LED: supplied by +VDD_MUX
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
++VDD_LED: supplied by +VDD_MUX
 +5V_USB_HS: supplied by +5V_SYS
 +5V_USB_SS: supplied by +5V_SYS
 +5V_HDMI_CON: supplied by +5V_SYS
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
+sdhci-tegra 700b0400.sdhci: Got CD GPIO
 hctosys: unable to open rtc device (rtc0)
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +VDD_LED: disabling
@@ -289,7 +290,6 @@
 ALSA device list:
   No soundcards found.
 Freeing unused kernel memory: 1152K
-sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 
 System initialization...
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-09-20 21:34:07.041684201 -0700
+++ /tmp/foob	2017-09-20 21:34:07.045684219 -0700
@@ -48,7 +48,7 @@
 	 #################################################################
 	 #################################################################
 	 #####################################
-	 3.6 MiB/s
+	 3.7 MiB/s
 done
 Bytes transferred = 8167944 (7ca208 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
@@ -57,7 +57,7 @@
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 21454 (53ce hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
@@ -69,7 +69,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-next-20170920-200864-g0b093a5 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Tue Sep 19 21:51:39 PDT 2017
+Linux version 4.14.0-rc1-next-20170921-201084-g43ec4ba (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Sep 20 21:26:23 PDT 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -79,7 +79,7 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0x3c000000
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c158e380, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c158f000, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -93,7 +93,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 954312K/1048576K available (10240K kernel code, 1626K rwdata, 4508K rodata, 2048K init, 405K bss, 28728K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 954308K/1048576K available (10240K kernel code, 1629K rwdata, 4508K rodata, 2048K init, 405K bss, 28732K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -103,8 +103,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1596828   (1627 kB)
-       .bss : 0xc159fbb4 - 0xc1605374   ( 406 kB)
+      .data : 0xc1400000 - 0xc15974a8   (1630 kB)
+       .bss : 0xc15a0bb4 - 0xc1606374   ( 406 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -350,17 +350,17 @@
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
-mmc1: new high speed SD card at address b368
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
- mmcblk1: p1 p2
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
 rtc-em3027 2-0056: hctosys: unable to read the hardware clock
+Waiting for root device PARTUUID=11eb23c1-01...
+mmc1: new high speed SD card at address b368
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
+ mmcblk1: p1 p2
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
-Waiting for root device PARTUUID=11eb23c1-01...
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2017-09-20 21:34:07.049684236 -0700
+++ /tmp/foob	2017-09-20 21:34:07.053684253 -0700
@@ -46,20 +46,20 @@
 	 #################################################################
 	 #################################################################
 	 #######################################
-	 3.5 MiB/s
+	 4 MiB/s
 done
-Bytes transferred = 6293360 (600770 hex)
+Bytes transferred = 6294928 (600d90 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 4.1 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21454 (53ce hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x600770 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x600d90 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103cd
@@ -68,7 +68,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-next-20170920-200864-g0b093a5 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Tue Sep 19 21:51:20 PDT 2017
+Linux version 4.14.0-rc1-next-20170921-201084-g43ec4ba (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Sep 20 21:26:29 PDT 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -77,7 +77,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0x3c000000
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0fb8400, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0fb9480, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -91,7 +91,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 958324K/1048576K available (9216K kernel code, 781K rwdata, 2796K rodata, 1024K init, 300K bss, 24716K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 958320K/1048576K available (9216K kernel code, 785K rwdata, 2796K rodata, 1024K init, 300K bss, 24720K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -101,8 +101,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
-      .data : 0xc0f00000 - 0xc0fc3520   ( 782 kB)
-       .bss : 0xc0fcd514 - 0xc10186ac   ( 301 kB)
+      .data : 0xc0f00000 - 0xc0fc45a0   ( 786 kB)
+       .bss : 0xc0fce514 - 0xc10196ac   ( 301 kB)
 Preemptible hierarchical RCU implementation.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
 	Tasks RCU enabled.
@@ -324,8 +324,8 @@
 NET: Registered protocol family 17
 NET: Registered protocol family 15
 can: controller area network core (rev 20170425 abi 9)
-NET: Registered protocol family 29
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
+NET: Registered protocol family 29
 can: raw protocol (rev 20170425)
 can: broadcast manager protocol (rev 20170425 t)
 can: netlink gateway (rev 20170425) max_hops=1
@@ -333,24 +333,24 @@
 Bluetooth: RFCOMM ver 1.11
 Bluetooth: BNEP (Ethernet Emulation) ver 1.3
 Bluetooth: BNEP socket layer initialized
+mmc1: new high speed SD card at address b368
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
+ mmcblk1: p1 p2
 Bluetooth: HIDP (Human Interface Emulation) ver 1.2
 Bluetooth: HIDP socket layer initialized
 Key type dns_resolver registered
 Registering SWP/SWPB emulation handler
-mmc1: new high speed SD card at address b368
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
- mmcblk1: p1 p2
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
+rtc-em3027 2-0056: hctosys: unable to read the hardware clock
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
-rtc-em3027 2-0056: hctosys: unable to read the hardware clock
 ALSA device list:
   #0: tegra-trimslice
 Waiting for root device PARTUUID=11eb23c1-01...
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2017-09-20 21:34:07.057684270 -0700
+++ /tmp/foob	2017-09-20 21:34:07.061684288 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 724487 us
-Basic modules init took 348838 us
-Sec Bootdevice Read Time = 204 ms, Read Size = 8459 KB
-Next stage binary read took 15904 us
-Carveout took 252664 us
-CPU initialization took 93139 us
-Total time taken by TegraBoot 710545 us
+Tegraboot started after 872506 us
+Basic modules init took 348568 us
+Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
+Next stage binary read took 14373 us
+Carveout took 252696 us
+CPU initialization took 92459 us
+Total time taken by TegraBoot 708096 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.566] LPDDR4 Training: Number of tables = 10
+64b[0001.708] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,9 +117,9 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-16505344 bytes read in 851 ms (18.5 MiB/s)
+16570880 bytes read in 847 ms (18.7 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-45170 bytes read in 453 ms (96.7 KiB/s)
+45170 bytes read in 454 ms (96.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
 1627787 bytes read in 495 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-next-20170920-200864-g0b093a5 (jonathanh@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Tue Sep 19 21:56:05 PDT 2017
+Linux version 4.14.0-rc1-next-20170921-319325-g43ec4ba69622 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Sep 20 21:32:00 PDT 2017
 Boot CPU: AArch64 Processor [411fd071]
 Machine model: NVIDIA Tegra210 P2371 (P2530/P2595) reference design
 debug: ignoring loglevel setting.
@@ -142,7 +142,7 @@
 cma: Reserved 16 MiB at 0x00000000fe000000
 NUMA: No NUMA configuration found
 NUMA: Faking a node at [mem 0x0000000000000000-0x000000013fffffff]
-NUMA: NODE_DATA [mem 0x13ffada80-0x13ffaf57f]
+NUMA: NODE_DATA [mem 0x13ffaca80-0x13ffae57f]
 Zone ranges:
   DMA      [mem 0x0000000080000000-0x00000000ffffffff]
   Normal   [mem 0x0000000100000000-0x000000013fffffff]
@@ -162,7 +162,7 @@
 psci: Using standard PSCI v0.2 function IDs
 psci: MIGRATE_INFO_TYPE not supported.
 random: fast init done
-percpu: Embedded 23 pages/cpu @ffff8000bff4e000 s55936 r8192 d30080 u94208
+percpu: Embedded 23 pages/cpu @ffff8000bff4d000 s55936 r8192 d30080 u94208
 pcpu-alloc: s55936 r8192 d30080 u94208 alloc=23*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Detected PIPT I-cache on CPU0
@@ -173,15 +173,15 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffff80007a000000-ffff80007dffffff]
-Memory: 2982564K/3131392K available (9212K kernel code, 1142K rwdata, 4592K rodata, 1152K init, 402K bss, 132444K reserved, 16384K cma-reserved)
+Memory: 2982496K/3131392K available (9276K kernel code, 1142K rwdata, 4592K rodata, 1152K init, 402K bss, 132512K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff000008980000   (  9216 KB)
-    .rodata : 0xffff000008980000 - 0xffff000008e00000   (  4608 KB)
-      .init : 0xffff000008e00000 - 0xffff000008f20000   (  1152 KB)
-      .data : 0xffff000008f20000 - 0xffff00000903da00   (  1143 KB)
-       .bss : 0xffff00000903da00 - 0xffff0000090a24f0   (   403 KB)
+      .text : 0xffff000008080000 - 0xffff000008990000   (  9280 KB)
+    .rodata : 0xffff000008990000 - 0xffff000008e10000   (  4608 KB)
+      .init : 0xffff000008e10000 - 0xffff000008f30000   (  1152 KB)
+      .data : 0xffff000008f30000 - 0xffff00000904da00   (  1143 KB)
+       .bss : 0xffff00000904da00 - 0xffff0000090b24f0   (   403 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -232,8 +232,10 @@
 pinctrl core: initialized pinctrl subsystem
 DMI not present or invalid.
 NET: Registered protocol family 16
+audit: initializing netlink subsys (disabled)
+audit: type=2000 audit(0.268:1): state=initialized audit_enabled=0 res=1
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffff000008986000, 1 data @ ffff000008f25000)
+vdso: 2 pages (1 code @ ffff000008996000, 1 data @ ffff000008f35000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -268,14 +270,12 @@
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 PCI: CLS 0 bytes, default 128
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 80974000
+kvm [1]: IDMAP page: 80976000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
 kvm [1]: vgic interrupt IRQ1
 kvm [1]: virtual timer IRQ4
-audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.507:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -340,14 +340,13 @@
 registered taskstats version 1
 hctosys: unable to open rtc device (rtc0)
 ALSA device list:
-  No soundcards found.
-Waiting for root device PARTUUID=00000000-0001-0000-788f-a75600000000...
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 SDW16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SDW16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SDW16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SDW16G partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12 p13 p14 p15 p16 p17 p18
+  No soundcards found.
 EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2017-09-20 21:34:07.065684305 -0700
+++ /tmp/foob	2017-09-20 21:34:07.069684322 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-next-20170920-200864-g0b093a5 (jonathanh@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Tue Sep 19 21:56:05 PDT 2017
+Linux version 4.14.0-rc1-next-20170921-319325-g43ec4ba69622 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Sep 20 21:32:00 PDT 2017
 Boot CPU: AArch64 Processor [411fd071]
 Machine model: Google Pixel C
 efi: Getting EFI parameters from FDT:
@@ -9,7 +9,7 @@
 bootconsole [uart0] enabled
 NUMA: No NUMA configuration found
 NUMA: Faking a node at [mem 0x0000000000000000-0x000000013fffffff]
-NUMA: NODE_DATA [mem 0x13ffa1c80-0x13ffa377f]
+NUMA: NODE_DATA [mem 0x13ffa0c80-0x13ffa277f]
 Zone ranges:
   DMA      [mem 0x0000000080000000-0x00000000ffffffff]
   Normal   [mem 0x0000000100000000-0x000000013fffffff]
@@ -22,7 +22,7 @@
 psci: PSCIv1.0 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: MIGRATE_INFO_UP_CPU reported invalid physical ID (0x80000000)
-percpu: Embedded 23 pages/cpu @ffff8000bff42000 s55936 r8192 d30080 u94208
+percpu: Embedded 23 pages/cpu @ffff8000bff41000 s55936 r8192 d30080 u94208
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
 CPU features: enabling workaround for ARM erratum 834220
@@ -31,15 +31,15 @@
 Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2837828K/2988032K available (9212K kernel code, 1142K rwdata, 4592K rodata, 1152K init, 402K bss, 133820K reserved, 16384K cma-reserved)
+Memory: 2837760K/2988032K available (9276K kernel code, 1142K rwdata, 4592K rodata, 1152K init, 402K bss, 133888K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff000008980000   (  9216 KB)
-    .rodata : 0xffff000008980000 - 0xffff000008e00000   (  4608 KB)
-      .init : 0xffff000008e00000 - 0xffff000008f20000   (  1152 KB)
-      .data : 0xffff000008f20000 - 0xffff00000903da00   (  1143 KB)
-       .bss : 0xffff00000903da00 - 0xffff0000090a24f0   (   403 KB)
+      .text : 0xffff000008080000 - 0xffff000008990000   (  9280 KB)
+    .rodata : 0xffff000008990000 - 0xffff000008e10000   (  4608 KB)
+      .init : 0xffff000008e10000 - 0xffff000008f30000   (  1152 KB)
+      .data : 0xffff000008f30000 - 0xffff00000904da00   (  1143 KB)
+       .bss : 0xffff00000904da00 - 0xffff0000090b24f0   (   403 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -107,8 +107,10 @@
 pinctrl core: initialized pinctrl subsystem
 DMI not present or invalid.
 NET: Registered protocol family 16
+audit: initializing netlink subsys (disabled)
+audit: type=2000 audit(0.180:1): state=initialized audit_enabled=0 res=1
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffff000008986000, 1 data @ ffff000008f25000)
+vdso: 2 pages (1 code @ ffff000008996000, 1 data @ ffff000008f35000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -150,14 +152,12 @@
 Unpacking initramfs...
 Freeing initrd memory: 1284K
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 80974000
+kvm [1]: IDMAP page: 80976000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
 kvm [1]: vgic interrupt IRQ1
 kvm [1]: virtual timer IRQ4
-audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.255:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -260,7 +260,7 @@
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
-max77686-rtc max77620-rtc: setting system clock to 2001-07-01 10:20:09 UTC (993982809)
+max77686-rtc max77620-rtc: setting system clock to 2001-07-02 09:56:36 UTC (994067796)
 PP1200_AVDD: disabling
 PP1200_RCAM: disabling
 PP2800L_RCAM: disabling
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-09-20 21:34:07.073684339 -0700
+++ /tmp/foob	2017-09-20 21:34:07.077684357 -0700
@@ -40,7 +40,7 @@
 	 #################################################################
 	 #################################################################
 	 #####################################
-	 2.9 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 8167944 (7ca208 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
@@ -49,7 +49,7 @@
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.7 MiB/s
+	 3.5 MiB/s
 done
 Bytes transferred = 51860 (ca94 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-next-20170920-200864-g0b093a5 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Tue Sep 19 21:51:39 PDT 2017
+Linux version 4.14.0-rc1-next-20170921-201084-g43ec4ba (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Sep 20 21:26:23 PDT 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -71,7 +71,7 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c158e380, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c158f000, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -85,7 +85,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993400K/2096128K available (10240K kernel code, 1626K rwdata, 4508K rodata, 2048K init, 405K bss, 37192K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993396K/2096128K available (10240K kernel code, 1629K rwdata, 4508K rodata, 2048K init, 405K bss, 37196K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -95,8 +95,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1596828   (1627 kB)
-       .bss : 0xc159fbb4 - 0xc1605374   ( 406 kB)
+      .data : 0xc1400000 - 0xc15974a8   (1630 kB)
+       .bss : 0xc15a0bb4 - 0xc1606374   ( 406 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -385,18 +385,18 @@
 sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
 +VDD_5V_HDMI: supplied by sys_3v3,vdd_3v3_alw
 mmc1: new high speed MMC card at address 0001
-tegra-pcie 3000.pcie: 2x3 configuration
 mmcblk1: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
-tegra-pcie 3000.pcie: probing port 0, using 2 lanes
 mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB
 Primary GPT is invalid, using alternate GPT.
  mmcblk1: p1 p2 p3 p4 p5
+tegra-pcie 3000.pcie: 2x3 configuration
 mmc0: new ultra high speed DDR50 SDHC card at address aaaa
 mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
-tegra-pcie 3000.pcie: probing port 2, using 2 lanes
+tegra-pcie 3000.pcie: probing port 0, using 2 lanes
  mmcblk0: p1 p2 p3
+tegra-pcie 3000.pcie: probing port 2, using 2 lanes
 tegra-pcie 3000.pcie: link 2 down, retrying
 tegra-pcie 3000.pcie: link 2 down, retrying
 tegra-pcie 3000.pcie: link 2 down, retrying
@@ -473,4 +473,4 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.483650] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.544461] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2017-09-20 21:34:07.081684374 -0700
+++ /tmp/foob	2017-09-20 21:34:07.085684391 -0700
@@ -38,20 +38,20 @@
 	 #################################################################
 	 #################################################################
 	 #######################################
-	 2.8 MiB/s
+	 3.1 MiB/s
 done
-Bytes transferred = 6293360 (600770 hex)
+Bytes transferred = 6294928 (600d90 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.6 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 51860 (ca94 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x600770 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x600d90 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa93 ... OK
@@ -60,7 +60,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-next-20170920-200864-g0b093a5 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Tue Sep 19 21:51:20 PDT 2017
+Linux version 4.14.0-rc1-next-20170921-201084-g43ec4ba (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Sep 20 21:26:29 PDT 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -69,7 +69,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0fb8400, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0fb9480, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -83,7 +83,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1997416K/2096128K available (9216K kernel code, 781K rwdata, 2796K rodata, 1024K init, 300K bss, 33176K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1997412K/2096128K available (9216K kernel code, 785K rwdata, 2796K rodata, 1024K init, 300K bss, 33180K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -93,8 +93,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
-      .data : 0xc0f00000 - 0xc0fc3520   ( 782 kB)
-       .bss : 0xc0fcd514 - 0xc10186ac   ( 301 kB)
+      .data : 0xc0f00000 - 0xc0fc45a0   ( 786 kB)
+       .bss : 0xc0fce514 - 0xc10196ac   ( 301 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -470,4 +470,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.804504] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# 
\ No newline at end of file
