Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2015-07-02 21:53:00.607139854 -0700
+++ /tmp/foob	2015-07-02 21:53:00.611139852 -0700
@@ -1,6 +1,6 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
-Linux version 4.1.0-next-20150702+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Jul 1 23:05:24 PDT 2015
+Linux version 4.1.0-next-20150703+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Jul 2 21:50:22 PDT 2015
 CPU: AArch64 Processor [411fd070] revision 0
 Detected PIPT I-cache on CPU0
 alternatives: enabling workaround for ARM erratum 832075
@@ -89,7 +89,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 2, 16384 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.380:1): initialized
+audit: type=2000 audit(0.390:1): initialized
 HugeTLB registered 2 MB page size, pre-allocated 0 pages
 VFS: Disk quotas dquot_6.6.0
 VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-07-02 21:53:00.615139850 -0700
+++ /tmp/foob	2015-07-02 21:53:00.619139848 -0700
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra114 (Dalmore) # 
 Tegra114 (Dalmore) # usb reset
 (Re)start USB...
@@ -40,18 +40,18 @@
 	 ###############################
 	 3.3 MiB/s
 done
-Bytes transferred = 6174856 (5e3888 hex)
+Bytes transferred = 6174592 (5e3780 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.8 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5e3888 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5e3780 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.1.0-next-20150702-27495-gc8a9ad2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Jul 1 23:03:47 PDT 2015
+Linux version 4.1.0-next-20150703-27645-ge58c243 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Jul 2 21:48:42 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -88,7 +88,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d1ac20   (11340 kB)
+      .text : 0xc0208000 - 0xc0d1ac10   (11340 kB)
       .init : 0xc0d1b000 - 0xc0e06000   ( 940 kB)
       .data : 0xc0e06000 - 0xc0f09b00   (1039 kB)
        .bss : 0xc0f0c000 - 0xc0f5dc28   ( 328 kB)
@@ -277,61 +277,6 @@
 vdd_hdmi_5v0: supplied by vdd-sys-5v0
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
-------------[ cut here ]------------
-WARNING: CPU: 1 PID: 58 at drivers/gpu/drm/drm_irq.c:1323 drm_vblank_on+0xf0/0x118()
-Modules linked in:
-CPU: 1 PID: 58 Comm: kworker/u8:1 Not tainted 4.1.0-next-20150702-27495-gc8a9ad2 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-Workqueue: deferwq deferred_probe_work_func
-[<c0217d68>] (unwind_backtrace) from [<c02136d0>] (show_stack+0x10/0x14)
-[<c02136d0>] (show_stack) from [<c0960504>] (dump_stack+0x78/0x94)
-[<c0960504>] (dump_stack) from [<c024c374>] (warn_slowpath_common+0x74/0xb0)
-[<c024c374>] (warn_slowpath_common) from [<c024c3cc>] (warn_slowpath_null+0x1c/0x24)
-[<c024c3cc>] (warn_slowpath_null) from [<c05c2ed8>] (drm_vblank_on+0xf0/0x118)
-[<c05c2ed8>] (drm_vblank_on) from [<c05b8938>] (drm_atomic_helper_commit_modeset_enables+0x8c/0x180)
-[<c05b8938>] (drm_atomic_helper_commit_modeset_enables) from [<c05ffcd0>] (tegra_atomic_complete+0x2c/0x50)
-[<c05ffcd0>] (tegra_atomic_complete) from [<c05ffd54>] (tegra_atomic_commit+0x60/0x90)
-[<c05ffd54>] (tegra_atomic_commit) from [<c05da334>] (drm_atomic_commit+0x40/0x64)
-[<c05da334>] (drm_atomic_commit) from [<c05b8698>] (drm_atomic_helper_set_config+0x394/0x3ec)
-[<c05b8698>] (drm_atomic_helper_set_config) from [<c05ca4d0>] (drm_mode_set_config_internal+0x58/0xd4)
-[<c05ca4d0>] (drm_mode_set_config_internal) from [<c05bab24>] (restore_fbdev_mode+0xd4/0xf4)
-[<c05bab24>] (restore_fbdev_mode) from [<c05bc260>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
-[<c05bc260>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c05bc2b8>] (drm_fb_helper_set_par+0x18/0x34)
-[<c05bc2b8>] (drm_fb_helper_set_par) from [<c050a404>] (fbcon_init+0x474/0x4dc)
-[<c050a404>] (fbcon_init) from [<c057339c>] (visual_init+0x9c/0xe8)
-[<c057339c>] (visual_init) from [<c0575354>] (do_bind_con_driver+0x134/0x354)
-[<c0575354>] (do_bind_con_driver) from [<c05756e4>] (do_take_over_console+0x170/0x1dc)
-[<c05756e4>] (do_take_over_console) from [<c050a4d0>] (do_fbcon_takeover+0x64/0xcc)
-[<c050a4d0>] (do_fbcon_takeover) from [<c0265254>] (notifier_call_chain+0x44/0x84)
-[<c0265254>] (notifier_call_chain) from [<c0265468>] (__blocking_notifier_call_chain+0x48/0x60)
-[<c0265468>] (__blocking_notifier_call_chain) from [<c0265498>] (blocking_notifier_call_chain+0x18/0x20)
-[<c0265498>] (blocking_notifier_call_chain) from [<c0514c68>] (register_framebuffer+0x184/0x264)
-[<c0514c68>] (register_framebuffer) from [<c05bc518>] (drm_fb_helper_initial_config+0x244/0x408)
-[<c05bc518>] (drm_fb_helper_initial_config) from [<c06018e4>] (tegra_drm_fb_init+0x48/0xb4)
-[<c06018e4>] (tegra_drm_fb_init) from [<c05ffbf0>] (tegra_drm_load+0x170/0x224)
-[<c05ffbf0>] (tegra_drm_load) from [<c05c51d4>] (drm_dev_register+0xa0/0x100)
-[<c05c51d4>] (drm_dev_register) from [<c05ff74c>] (host1x_drm_probe+0x44/0xb0)
-[<c05ff74c>] (host1x_drm_probe) from [<c05a7b2c>] (host1x_device_probe+0x1c/0x28)
-[<c05a7b2c>] (host1x_device_probe) from [<c0616e74>] (driver_probe_device+0x1c8/0x26c)
-[<c0616e74>] (driver_probe_device) from [<c061557c>] (bus_for_each_drv+0x44/0x8c)
-[<c061557c>] (bus_for_each_drv) from [<c0616c34>] (__device_attach+0x88/0xd8)
-[<c0616c34>] (__device_attach) from [<c06163f4>] (bus_probe_device+0x84/0x8c)
-[<c06163f4>] (bus_probe_device) from [<c0614920>] (device_add+0x33c/0x524)
-[<c0614920>] (device_add) from [<c05a7d38>] (host1x_subdev_register+0xb0/0xd4)
-[<c05a7d38>] (host1x_subdev_register) from [<c05a85f8>] (host1x_client_register+0xf4/0x11c)
-[<c05a85f8>] (host1x_client_register) from [<c0609584>] (tegra_hdmi_probe+0x170/0x250)
-[<c0609584>] (tegra_hdmi_probe) from [<c0618434>] (platform_drv_probe+0x48/0xa4)
-[<c0618434>] (platform_drv_probe) from [<c0616e74>] (driver_probe_device+0x1c8/0x26c)
-[<c0616e74>] (driver_probe_device) from [<c061557c>] (bus_for_each_drv+0x44/0x8c)
-[<c061557c>] (bus_for_each_drv) from [<c0616c34>] (__device_attach+0x88/0xd8)
-[<c0616c34>] (__device_attach) from [<c06163f4>] (bus_probe_device+0x84/0x8c)
-[<c06163f4>] (bus_probe_device) from [<c06167cc>] (deferred_probe_work_func+0x5c/0x88)
-[<c06167cc>] (deferred_probe_work_func) from [<c025f280>] (process_one_work+0x134/0x338)
-[<c025f280>] (process_one_work) from [<c025f4e0>] (worker_thread+0x28/0x4cc)
-[<c025f4e0>] (worker_thread) from [<c0264630>] (kthread+0xd8/0xf4)
-[<c0264630>] (kthread) from [<c0210a68>] (ret_from_fork+0x14/0x2c)
----[ end trace cb117e7ec896906c ]---
-Console: switching to colour frame buffer device 240x75
 mmc1: MAN_BKOPS_EN bit is not set
 mmc1: new high speed MMC card at address 0001
 mmcblk0: mmc1:0001 SEM32G 29.1 GiB 
@@ -339,6 +284,7 @@
 mmcblk0boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
 mmcblk0rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
+Console: switching to colour frame buffer device 240x75
 drm drm: fb0:  frame buffer device
 drm drm: registered panic notifier
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
@@ -350,7 +296,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-12 14:55:27 UTC (947688927)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-13 13:40:18 UTC (947770818)
 vdd_cam_1v8_reg: disabling
 usb1_vbus: disabling
 dvdd_ts: disabling
@@ -362,4 +308,4 @@
 Freeing unused kernel memory: 940K (c0d1b000 - c0e06000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.664028] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.198997] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2015-07-02 21:53:00.627139845 -0700
+++ /tmp/foob	2015-07-02 21:53:00.627139845 -0700
@@ -36,10 +36,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #######################
+	 ######################
 	 3.3 MiB/s
 done
-Bytes transferred = 5094168 (4dbb18 hex)
+Bytes transferred = 5093632 (4db900 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4dbb18 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4db900 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.1.0-next-20150702-27495-gc8a9ad2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Jul 1 23:04:31 PDT 2015
+Linux version 4.1.0-next-20150703-27645-ge58c243 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Jul 2 21:49:30 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -90,7 +90,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0930b48   (9379 kB)
+      .text : 0xc0008000 - 0xc0930b38   (9379 kB)
       .init : 0xc0931000 - 0xc097e000   ( 308 kB)
       .data : 0xc097e000 - 0xc09d49c0   ( 347 kB)
        .bss : 0xc09d7000 - 0xc0a31c94   ( 364 kB)
@@ -283,60 +283,6 @@
 vdd_hdmi_5v0: supplied by vdd-sys-5v0
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
-------------[ cut here ]------------
-WARNING: CPU: 0 PID: 70 at drivers/gpu/drm/drm_irq.c:1323 drm_vblank_on+0xf0/0x118()
-Modules linked in:
-CPU: 0 PID: 70 Comm: kworker/u8:1 Not tainted 4.1.0-next-20150702-27495-gc8a9ad2 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-Workqueue: deferwq deferred_probe_work_func
-[<c00167f8>] (unwind_backtrace) from [<c0012124>] (show_stack+0x10/0x14)
-[<c0012124>] (show_stack) from [<c0698e94>] (dump_stack+0x84/0xd0)
-[<c0698e94>] (dump_stack) from [<c002683c>] (warn_slowpath_common+0x74/0xb0)
-[<c002683c>] (warn_slowpath_common) from [<c0026894>] (warn_slowpath_null+0x1c/0x24)
-[<c0026894>] (warn_slowpath_null) from [<c02e79c8>] (drm_vblank_on+0xf0/0x118)
-[<c02e79c8>] (drm_vblank_on) from [<c02ddd3c>] (drm_atomic_helper_commit_modeset_enables+0x8c/0x180)
-[<c02ddd3c>] (drm_atomic_helper_commit_modeset_enables) from [<c0301f88>] (tegra_atomic_complete+0x2c/0x50)
-[<c0301f88>] (tegra_atomic_complete) from [<c030200c>] (tegra_atomic_commit+0x60/0x90)
-[<c030200c>] (tegra_atomic_commit) from [<c02ff098>] (drm_atomic_commit+0x40/0x64)
-[<c02ff098>] (drm_atomic_commit) from [<c02dda9c>] (drm_atomic_helper_set_config+0x394/0x3ec)
-[<c02dda9c>] (drm_atomic_helper_set_config) from [<c02ef0ac>] (drm_mode_set_config_internal+0x58/0xd4)
-[<c02ef0ac>] (drm_mode_set_config_internal) from [<c02dff28>] (restore_fbdev_mode+0xd4/0xf4)
-[<c02dff28>] (restore_fbdev_mode) from [<c02e1664>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
-[<c02e1664>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c02e16bc>] (drm_fb_helper_set_par+0x18/0x34)
-[<c02e16bc>] (drm_fb_helper_set_par) from [<c027f258>] (fbcon_init+0x474/0x4dc)
-[<c027f258>] (fbcon_init) from [<c02b4804>] (visual_init+0x9c/0xe8)
-[<c02b4804>] (visual_init) from [<c02b67a8>] (do_bind_con_driver+0x134/0x354)
-[<c02b67a8>] (do_bind_con_driver) from [<c02b6b38>] (do_take_over_console+0x170/0x1dc)
-[<c02b6b38>] (do_take_over_console) from [<c027f324>] (do_fbcon_takeover+0x64/0xcc)
-[<c027f324>] (do_fbcon_takeover) from [<c003f0d0>] (notifier_call_chain+0x44/0x84)
-[<c003f0d0>] (notifier_call_chain) from [<c003f2a4>] (__blocking_notifier_call_chain+0x48/0x60)
-[<c003f2a4>] (__blocking_notifier_call_chain) from [<c003f2d4>] (blocking_notifier_call_chain+0x18/0x20)
-[<c003f2d4>] (blocking_notifier_call_chain) from [<c0289218>] (register_framebuffer+0x184/0x264)
-[<c0289218>] (register_framebuffer) from [<c02e191c>] (drm_fb_helper_initial_config+0x244/0x408)
-[<c02e191c>] (drm_fb_helper_initial_config) from [<c0303c50>] (tegra_drm_fb_init+0x48/0xb4)
-[<c0303c50>] (tegra_drm_fb_init) from [<c0301ea8>] (tegra_drm_load+0x170/0x224)
-[<c0301ea8>] (tegra_drm_load) from [<c02e9cdc>] (drm_dev_register+0xa0/0x100)
-[<c02e9cdc>] (drm_dev_register) from [<c0301a04>] (host1x_drm_probe+0x44/0xb0)
-[<c0301a04>] (host1x_drm_probe) from [<c02ccf54>] (host1x_device_probe+0x1c/0x28)
-[<c02ccf54>] (host1x_device_probe) from [<c0319184>] (driver_probe_device+0x1c8/0x26c)
-[<c0319184>] (driver_probe_device) from [<c0317894>] (bus_for_each_drv+0x44/0x8c)
-[<c0317894>] (bus_for_each_drv) from [<c0318f44>] (__device_attach+0x88/0xd8)
-[<c0318f44>] (__device_attach) from [<c0318704>] (bus_probe_device+0x84/0x8c)
-[<c0318704>] (bus_probe_device) from [<c0316c58>] (device_add+0x33c/0x524)
-[<c0316c58>] (device_add) from [<c02cd160>] (host1x_subdev_register+0xb0/0xd4)
-[<c02cd160>] (host1x_subdev_register) from [<c02cda20>] (host1x_client_register+0xf4/0x11c)
-[<c02cda20>] (host1x_client_register) from [<c030b8e4>] (tegra_hdmi_probe+0x170/0x250)
-[<c030b8e4>] (tegra_hdmi_probe) from [<c031a644>] (platform_drv_probe+0x48/0xa4)
-[<c031a644>] (platform_drv_probe) from [<c0319184>] (driver_probe_device+0x1c8/0x26c)
-[<c0319184>] (driver_probe_device) from [<c0317894>] (bus_for_each_drv+0x44/0x8c)
-[<c0317894>] (bus_for_each_drv) from [<c0318f44>] (__device_attach+0x88/0xd8)
-[<c0318f44>] (__device_attach) from [<c0318704>] (bus_probe_device+0x84/0x8c)
-[<c0318704>] (bus_probe_device) from [<c0318adc>] (deferred_probe_work_func+0x5c/0x88)
-[<c0318adc>] (deferred_probe_work_func) from [<c00391bc>] (process_one_work+0x120/0x328)
-[<c00391bc>] (process_one_work) from [<c0039420>] (worker_thread+0x28/0x490)
-[<c0039420>] (worker_thread) from [<c003e510>] (kthread+0xd8/0xf4)
-[<c003e510>] (kthread) from [<c000f628>] (ret_from_fork+0x14/0x2c)
----[ end trace 1ac80ea143ac60d8 ]---
 Console: switching to colour frame buffer device 240x75
 drm drm: fb0:  frame buffer device
 drm drm: registered panic notifier
@@ -349,7 +295,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-12 14:56:09 UTC (947688969)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-13 13:41:10 UTC (947770870)
 vdd_cam_1v8_reg: disabling
 usb1_vbus: disabling
 dvdd_ts: disabling
@@ -364,5 +310,4 @@
 Freeing unused kernel memory: 308K (c0931000 - c097e000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.199949] usb 1-1: new high-speed USB device number 2 using tegra-ehci
-cfg80211: Calling CRDA to update world regulatory domain
+root@(none):/# [    2.739911] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-07-02 21:53:00.639139840 -0700
+++ /tmp/foob	2015-07-02 21:53:00.643139837 -0700
@@ -42,18 +42,18 @@
 	 ###############################
 	 3.3 MiB/s
 done
-Bytes transferred = 6174856 (5e3888 hex)
+Bytes transferred = 6174592 (5e3780 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *#####
-	 2.9 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 63791 (f92f hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5e3888 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5e3780 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 8201292e
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.1.0-next-20150702-27495-gc8a9ad2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Jul 1 23:03:47 PDT 2015
+Linux version 4.1.0-next-20150703-27645-ge58c243 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Jul 2 21:48:42 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -90,7 +90,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d1ac20   (11340 kB)
+      .text : 0xc0208000 - 0xc0d1ac10   (11340 kB)
       .init : 0xc0d1b000 - 0xc0e06000   ( 940 kB)
       .data : 0xc0e06000 - 0xc0f09b00   (1039 kB)
        .bss : 0xc0f0c000 - 0xc0f5dc28   ( 328 kB)
@@ -286,13 +286,13 @@
 +1.35V_LP0(sd2): supplied by +5V_SYS
 +1.05V_RUN_AVDD: supplied by +1.35V_LP0(sd2)
 tegra-pcie 1003000.pcie-controller: probing port 0, using 2 lanes
-tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
 mmc0: MAN_BKOPS_EN bit is not set
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 SEM16G 14.6 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
 Primary GPT is invalid, using alternate GPT.
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9
 tegra-pcie 1003000.pcie-controller: link 0 down, retrying
@@ -317,60 +317,6 @@
 +3.3V_AVDD_HDMI_AP_GATED: supplied by +3.3V_RUN
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
-------------[ cut here ]------------
-WARNING: CPU: 2 PID: 58 at drivers/gpu/drm/drm_irq.c:1323 drm_vblank_on+0xf0/0x118()
-Modules linked in:
-CPU: 2 PID: 58 Comm: kworker/u8:1 Not tainted 4.1.0-next-20150702-27495-gc8a9ad2 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-Workqueue: deferwq deferred_probe_work_func
-[<c0217d68>] (unwind_backtrace) from [<c02136d0>] (show_stack+0x10/0x14)
-[<c02136d0>] (show_stack) from [<c0960504>] (dump_stack+0x78/0x94)
-[<c0960504>] (dump_stack) from [<c024c374>] (warn_slowpath_common+0x74/0xb0)
-[<c024c374>] (warn_slowpath_common) from [<c024c3cc>] (warn_slowpath_null+0x1c/0x24)
-[<c024c3cc>] (warn_slowpath_null) from [<c05c2ed8>] (drm_vblank_on+0xf0/0x118)
-[<c05c2ed8>] (drm_vblank_on) from [<c05b8938>] (drm_atomic_helper_commit_modeset_enables+0x8c/0x180)
-[<c05b8938>] (drm_atomic_helper_commit_modeset_enables) from [<c05ffcd0>] (tegra_atomic_complete+0x2c/0x50)
-[<c05ffcd0>] (tegra_atomic_complete) from [<c05ffd54>] (tegra_atomic_commit+0x60/0x90)
-[<c05ffd54>] (tegra_atomic_commit) from [<c05da334>] (drm_atomic_commit+0x40/0x64)
-[<c05da334>] (drm_atomic_commit) from [<c05b8698>] (drm_atomic_helper_set_config+0x394/0x3ec)
-[<c05b8698>] (drm_atomic_helper_set_config) from [<c05ca4d0>] (drm_mode_set_config_internal+0x58/0xd4)
-[<c05ca4d0>] (drm_mode_set_config_internal) from [<c05bab24>] (restore_fbdev_mode+0xd4/0xf4)
-[<c05bab24>] (restore_fbdev_mode) from [<c05bc260>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
-[<c05bc260>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c05bc2b8>] (drm_fb_helper_set_par+0x18/0x34)
-[<c05bc2b8>] (drm_fb_helper_set_par) from [<c050a404>] (fbcon_init+0x474/0x4dc)
-[<c050a404>] (fbcon_init) from [<c057339c>] (visual_init+0x9c/0xe8)
-[<c057339c>] (visual_init) from [<c0575354>] (do_bind_con_driver+0x134/0x354)
-[<c0575354>] (do_bind_con_driver) from [<c05756e4>] (do_take_over_console+0x170/0x1dc)
-[<c05756e4>] (do_take_over_console) from [<c050a4d0>] (do_fbcon_takeover+0x64/0xcc)
-[<c050a4d0>] (do_fbcon_takeover) from [<c0265254>] (notifier_call_chain+0x44/0x84)
-[<c0265254>] (notifier_call_chain) from [<c0265468>] (__blocking_notifier_call_chain+0x48/0x60)
-[<c0265468>] (__blocking_notifier_call_chain) from [<c0265498>] (blocking_notifier_call_chain+0x18/0x20)
-[<c0265498>] (blocking_notifier_call_chain) from [<c0514c68>] (register_framebuffer+0x184/0x264)
-[<c0514c68>] (register_framebuffer) from [<c05bc518>] (drm_fb_helper_initial_config+0x244/0x408)
-[<c05bc518>] (drm_fb_helper_initial_config) from [<c06018e4>] (tegra_drm_fb_init+0x48/0xb4)
-[<c06018e4>] (tegra_drm_fb_init) from [<c05ffbf0>] (tegra_drm_load+0x170/0x224)
-[<c05ffbf0>] (tegra_drm_load) from [<c05c51d4>] (drm_dev_register+0xa0/0x100)
-[<c05c51d4>] (drm_dev_register) from [<c05ff74c>] (host1x_drm_probe+0x44/0xb0)
-[<c05ff74c>] (host1x_drm_probe) from [<c05a7b2c>] (host1x_device_probe+0x1c/0x28)
-[<c05a7b2c>] (host1x_device_probe) from [<c0616e74>] (driver_probe_device+0x1c8/0x26c)
-[<c0616e74>] (driver_probe_device) from [<c061557c>] (bus_for_each_drv+0x44/0x8c)
-[<c061557c>] (bus_for_each_drv) from [<c0616c34>] (__device_attach+0x88/0xd8)
-[<c0616c34>] (__device_attach) from [<c06163f4>] (bus_probe_device+0x84/0x8c)
-[<c06163f4>] (bus_probe_device) from [<c0614920>] (device_add+0x33c/0x524)
-[<c0614920>] (device_add) from [<c05a7d38>] (host1x_subdev_register+0xb0/0xd4)
-[<c05a7d38>] (host1x_subdev_register) from [<c05a85f8>] (host1x_client_register+0xf4/0x11c)
-[<c05a85f8>] (host1x_client_register) from [<c0609584>] (tegra_hdmi_probe+0x170/0x250)
-[<c0609584>] (tegra_hdmi_probe) from [<c0618434>] (platform_drv_probe+0x48/0xa4)
-[<c0618434>] (platform_drv_probe) from [<c0616e74>] (driver_probe_device+0x1c8/0x26c)
-[<c0616e74>] (driver_probe_device) from [<c061557c>] (bus_for_each_drv+0x44/0x8c)
-[<c061557c>] (bus_for_each_drv) from [<c0616c34>] (__device_attach+0x88/0xd8)
-[<c0616c34>] (__device_attach) from [<c06163f4>] (bus_probe_device+0x84/0x8c)
-[<c06163f4>] (bus_probe_device) from [<c06167cc>] (deferred_probe_work_func+0x5c/0x88)
-[<c06167cc>] (deferred_probe_work_func) from [<c025f280>] (process_one_work+0x134/0x338)
-[<c025f280>] (process_one_work) from [<c025f4e0>] (worker_thread+0x28/0x4cc)
-[<c025f4e0>] (worker_thread) from [<c0264630>] (kthread+0xd8/0xf4)
-[<c0264630>] (kthread) from [<c0210a68>] (ret_from_fork+0x14/0x2c)
----[ end trace 10d75373a5249ee2 ]---
 Console: switching to colour frame buffer device 512x135
 drm drm: fb0:  frame buffer device
 drm drm: registered panic notifier
@@ -412,9 +358,9 @@
 ata1: SATA link down (SStatus 0 SControl 300)
 EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
 EXT4-fs (mmcblk0p1): couldn't mount as ext2 due to feature incompatibilities
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 940K (c0d1b000 - c0e06000)
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2015-07-02 21:53:00.647139835 -0700
+++ /tmp/foob	2015-07-02 21:53:00.651139834 -0700
@@ -38,22 +38,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #######################
-	 760.7 KiB/s
+	 ######################
+	 3.2 MiB/s
 done
-Bytes transferred = 5094168 (4dbb18 hex)
+Bytes transferred = 5093632 (4db900 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
-Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *#####
-	 2.9 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 63791 (f92f hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4dbb18 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4db900 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 8201292e
@@ -64,7 +63,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.1.0-next-20150702-27495-gc8a9ad2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Jul 1 23:04:31 PDT 2015
+Linux version 4.1.0-next-20150703-27645-ge58c243 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Jul 2 21:49:30 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -93,7 +92,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0930b48   (9379 kB)
+      .text : 0xc0008000 - 0xc0930b38   (9379 kB)
       .init : 0xc0931000 - 0xc097e000   ( 308 kB)
       .data : 0xc097e000 - 0xc09d49c0   ( 347 kB)
        .bss : 0xc09d7000 - 0xc0a31c94   ( 364 kB)
@@ -319,60 +318,6 @@
 +3.3V_AVDD_HDMI_AP_GATED: supplied by +3.3V_RUN
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
-------------[ cut here ]------------
-WARNING: CPU: 1 PID: 73 at drivers/gpu/drm/drm_irq.c:1323 drm_vblank_on+0xf0/0x118()
-Modules linked in:
-CPU: 1 PID: 73 Comm: kworker/u8:1 Not tainted 4.1.0-next-20150702-27495-gc8a9ad2 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-Workqueue: deferwq deferred_probe_work_func
-[<c00167f8>] (unwind_backtrace) from [<c0012124>] (show_stack+0x10/0x14)
-[<c0012124>] (show_stack) from [<c0698e94>] (dump_stack+0x84/0xd0)
-[<c0698e94>] (dump_stack) from [<c002683c>] (warn_slowpath_common+0x74/0xb0)
-[<c002683c>] (warn_slowpath_common) from [<c0026894>] (warn_slowpath_null+0x1c/0x24)
-[<c0026894>] (warn_slowpath_null) from [<c02e79c8>] (drm_vblank_on+0xf0/0x118)
-[<c02e79c8>] (drm_vblank_on) from [<c02ddd3c>] (drm_atomic_helper_commit_modeset_enables+0x8c/0x180)
-[<c02ddd3c>] (drm_atomic_helper_commit_modeset_enables) from [<c0301f88>] (tegra_atomic_complete+0x2c/0x50)
-[<c0301f88>] (tegra_atomic_complete) from [<c030200c>] (tegra_atomic_commit+0x60/0x90)
-[<c030200c>] (tegra_atomic_commit) from [<c02ff098>] (drm_atomic_commit+0x40/0x64)
-[<c02ff098>] (drm_atomic_commit) from [<c02dda9c>] (drm_atomic_helper_set_config+0x394/0x3ec)
-[<c02dda9c>] (drm_atomic_helper_set_config) from [<c02ef0ac>] (drm_mode_set_config_internal+0x58/0xd4)
-[<c02ef0ac>] (drm_mode_set_config_internal) from [<c02dff28>] (restore_fbdev_mode+0xd4/0xf4)
-[<c02dff28>] (restore_fbdev_mode) from [<c02e1664>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
-[<c02e1664>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c02e16bc>] (drm_fb_helper_set_par+0x18/0x34)
-[<c02e16bc>] (drm_fb_helper_set_par) from [<c027f258>] (fbcon_init+0x474/0x4dc)
-[<c027f258>] (fbcon_init) from [<c02b4804>] (visual_init+0x9c/0xe8)
-[<c02b4804>] (visual_init) from [<c02b67a8>] (do_bind_con_driver+0x134/0x354)
-[<c02b67a8>] (do_bind_con_driver) from [<c02b6b38>] (do_take_over_console+0x170/0x1dc)
-[<c02b6b38>] (do_take_over_console) from [<c027f324>] (do_fbcon_takeover+0x64/0xcc)
-[<c027f324>] (do_fbcon_takeover) from [<c003f0d0>] (notifier_call_chain+0x44/0x84)
-[<c003f0d0>] (notifier_call_chain) from [<c003f2a4>] (__blocking_notifier_call_chain+0x48/0x60)
-[<c003f2a4>] (__blocking_notifier_call_chain) from [<c003f2d4>] (blocking_notifier_call_chain+0x18/0x20)
-[<c003f2d4>] (blocking_notifier_call_chain) from [<c0289218>] (register_framebuffer+0x184/0x264)
-[<c0289218>] (register_framebuffer) from [<c02e191c>] (drm_fb_helper_initial_config+0x244/0x408)
-[<c02e191c>] (drm_fb_helper_initial_config) from [<c0303c50>] (tegra_drm_fb_init+0x48/0xb4)
-[<c0303c50>] (tegra_drm_fb_init) from [<c0301ea8>] (tegra_drm_load+0x170/0x224)
-[<c0301ea8>] (tegra_drm_load) from [<c02e9cdc>] (drm_dev_register+0xa0/0x100)
-[<c02e9cdc>] (drm_dev_register) from [<c0301a04>] (host1x_drm_probe+0x44/0xb0)
-[<c0301a04>] (host1x_drm_probe) from [<c02ccf54>] (host1x_device_probe+0x1c/0x28)
-[<c02ccf54>] (host1x_device_probe) from [<c0319184>] (driver_probe_device+0x1c8/0x26c)
-[<c0319184>] (driver_probe_device) from [<c0317894>] (bus_for_each_drv+0x44/0x8c)
-[<c0317894>] (bus_for_each_drv) from [<c0318f44>] (__device_attach+0x88/0xd8)
-[<c0318f44>] (__device_attach) from [<c0318704>] (bus_probe_device+0x84/0x8c)
-[<c0318704>] (bus_probe_device) from [<c0316c58>] (device_add+0x33c/0x524)
-[<c0316c58>] (device_add) from [<c02cd160>] (host1x_subdev_register+0xb0/0xd4)
-[<c02cd160>] (host1x_subdev_register) from [<c02cda20>] (host1x_client_register+0xf4/0x11c)
-[<c02cda20>] (host1x_client_register) from [<c030b8e4>] (tegra_hdmi_probe+0x170/0x250)
-[<c030b8e4>] (tegra_hdmi_probe) from [<c031a644>] (platform_drv_probe+0x48/0xa4)
-[<c031a644>] (platform_drv_probe) from [<c0319184>] (driver_probe_device+0x1c8/0x26c)
-[<c0319184>] (driver_probe_device) from [<c0317894>] (bus_for_each_drv+0x44/0x8c)
-[<c0317894>] (bus_for_each_drv) from [<c0318f44>] (__device_attach+0x88/0xd8)
-[<c0318f44>] (__device_attach) from [<c0318704>] (bus_probe_device+0x84/0x8c)
-[<c0318704>] (bus_probe_device) from [<c0318adc>] (deferred_probe_work_func+0x5c/0x88)
-[<c0318adc>] (deferred_probe_work_func) from [<c00391bc>] (process_one_work+0x120/0x328)
-[<c00391bc>] (process_one_work) from [<c0039420>] (worker_thread+0x28/0x490)
-[<c0039420>] (worker_thread) from [<c003e510>] (kthread+0xd8/0xf4)
-[<c003e510>] (kthread) from [<c000f628>] (ret_from_fork+0x14/0x2c)
----[ end trace f5c4926337fa529f ]---
 Console: switching to colour frame buffer device 512x135
 drm drm: fb0:  frame buffer device
 drm drm: registered panic notifier
@@ -394,7 +339,6 @@
 hub 2-0:1.0: 1 port detected
 cpufreq: cpufreq_add_dev: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_add_dev: CPU0: Unlisted initial frequency changed to: 714000 KHz
-cfg80211: Calling CRDA to update world regulatory domain
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc1: Unknown controller version (3). You may experience problems.
@@ -403,7 +347,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-02-01 00:00:25 UTC (949363225)
+as3722-rtc as3722-rtc: setting system clock to 2000-02-01 00:00:19 UTC (949363219)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_VPP_FUSE: disabling
 +2.8V_RUN_CAM_AF: disabling
@@ -424,6 +368,7 @@
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 308K (c0931000 - c097e000)
+cfg80211: Calling CRDA to update world regulatory domain
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    7.893265] asix 2-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:11:31:3b
+root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-07-02 21:53:00.659139830 -0700
+++ /tmp/foob	2015-07-02 21:53:00.659139830 -0700
@@ -48,7 +48,7 @@
 	 ###############################
 	 4.1 MiB/s
 done
-Bytes transferred = 6174856 (5e3888 hex)
+Bytes transferred = 6174592 (5e3780 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -59,7 +59,7 @@
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x5e3888 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x5e3780 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -67,7 +67,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.1.0-next-20150702-27495-gc8a9ad2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Jul 1 23:03:47 PDT 2015
+Linux version 4.1.0-next-20150703-27645-ge58c243 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Jul 2 21:48:42 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -96,7 +96,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d1ac20   (11340 kB)
+      .text : 0xc0208000 - 0xc0d1ac10   (11340 kB)
       .init : 0xc0d1b000 - 0xc0e06000   ( 940 kB)
       .data : 0xc0e06000 - 0xc0f09b00   (1039 kB)
        .bss : 0xc0f0c000 - 0xc0f5dc28   ( 328 kB)
@@ -324,21 +324,20 @@
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
-tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
+tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 mmc0: host does not support reading read-only switch, assuming write-enable
+mmc0: new high speed SDHC card at address b368
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
+ mmcblk0: p1
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-26 17:20:12 UTC (1419614412)
-Waiting for root device /dev/mmcblk0p1...
-mmc0: new high speed SDHC card at address b368
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
- mmcblk0: p1
-EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
+rtc-em3027 2-0056: setting system clock to 2014-12-26 17:28:42 UTC (1419614922)
+EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2015-07-02 21:53:00.667139826 -0700
+++ /tmp/foob	2015-07-02 21:53:00.667139826 -0700
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -44,10 +44,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #######################
+	 ######################
 	 4.1 MiB/s
 done
-Bytes transferred = 5094168 (4dbb18 hex)
+Bytes transferred = 5093632 (4db900 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x4dbb18 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x4db900 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -69,7 +69,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.1.0-next-20150702-27495-gc8a9ad2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Jul 1 23:04:31 PDT 2015
+Linux version 4.1.0-next-20150703-27645-ge58c243 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Jul 2 21:49:30 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -99,7 +99,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0930b48   (9379 kB)
+      .text : 0xc0008000 - 0xc0930b38   (9379 kB)
       .init : 0xc0931000 - 0xc097e000   ( 308 kB)
       .data : 0xc097e000 - 0xc09d49c0   ( 347 kB)
        .bss : 0xc09d7000 - 0xc0a31c94   ( 364 kB)
@@ -351,7 +351,7 @@
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-26 17:20:33 UTC (1419614433)
+rtc-em3027 2-0056: setting system clock to 2014-12-26 17:29:01 UTC (1419614941)
 ALSA device list:
   #0: tegra-trimslice
 kjournald starting.  Commit interval 5 seconds
@@ -361,7 +361,7 @@
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 308K (c0931000 - c097e000)
-usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
-bash: cannot set terminal process group (-1): Inappropriate ioctl for device
+bash: ca[    2.242305] usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
+nnot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-07-02 21:53:00.683139819 -0700
+++ /tmp/foob	2015-07-02 21:53:00.687139817 -0700
@@ -38,20 +38,20 @@
 	 #################################################################
 	 #################################################################
 	 ###############################
-	 3.4 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 6174856 (5e3888 hex)
+Bytes transferred = 6174592 (5e3780 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5e3888 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5e3780 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.1.0-next-20150702-27495-gc8a9ad2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Jul 1 23:03:47 PDT 2015
+Linux version 4.1.0-next-20150703-27645-ge58c243 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Jul 2 21:48:42 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -88,7 +88,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d1ac20   (11340 kB)
+      .text : 0xc0208000 - 0xc0d1ac10   (11340 kB)
       .init : 0xc0d1b000 - 0xc0e06000   ( 940 kB)
       .data : 0xc0e06000 - 0xc0f09b00   (1039 kB)
        .bss : 0xc0f0c000 - 0xc0f5dc28   ( 328 kB)
@@ -368,4 +368,4 @@
 Freeing unused kernel memory: 940K (c0d1b000 - c0e06000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.773405] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.768795] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2015-07-02 21:53:00.695139813 -0700
+++ /tmp/foob	2015-07-02 21:53:00.695139813 -0700
@@ -36,10 +36,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #######################
-	 3.4 MiB/s
+	 ######################
+	 3.3 MiB/s
 done
-Bytes transferred = 5094168 (4dbb18 hex)
+Bytes transferred = 5093632 (4db900 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4dbb18 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4db900 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.1.0-next-20150702-27495-gc8a9ad2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Jul 1 23:04:31 PDT 2015
+Linux version 4.1.0-next-20150703-27645-ge58c243 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Jul 2 21:49:30 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -90,7 +90,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0930b48   (9379 kB)
+      .text : 0xc0008000 - 0xc0930b38   (9379 kB)
       .init : 0xc0931000 - 0xc097e000   ( 308 kB)
       .data : 0xc097e000 - 0xc09d49c0   ( 347 kB)
        .bss : 0xc09d7000 - 0xc0a31c94   ( 364 kB)
@@ -362,8 +362,8 @@
 vdd_sata,avdd_plle: disabling
 ALSA device list:
   #0: NVIDIA Tegra Beaver
-kjournald starting.  Commit interval 5 seconds
 EXT3-fs (mmcblk0p1): warning: mounting fs with errors, running e2fsck is recommended
+kjournald starting.  Commit interval 5 seconds
 EXT3-fs (mmcblk0p1): using internal journal
 EXT3-fs (mmcblk0p1): recovery complete
 EXT3-fs (mmcblk0p1): mounted filesystem with writeback data mode
@@ -372,4 +372,4 @@
 Freeing unused kernel memory: 308K (c0931000 - c097e000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.436391] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.426416] usb 2-1: new high-speed USB device number 2 using tegra-ehci
