Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2017-05-17 08:54:18.763130849 -0700
+++ /tmp/foob	2017-05-17 08:54:18.767130865 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.12.0-rc1-next-20170516-290450-g7f49246 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed May 17 08:37:18 PDT 2017
+Linux version 4.12.0-rc1-next-20170517-290675-g973d630 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed May 17 08:51:59 PDT 2017
 Boot CPU: AArch64 Processor [411fd070]
 Machine model: linux,dummy-virt
 Memory limited to 1024MB
@@ -30,15 +30,15 @@
 Policy zone: DMA
 Kernel command line: root=/dev/vda2 rw rootwait mem=1024M console=ttyAMA0,38400n8 init=/bin/bash
 PID hash table entries: 4096 (order: 3, 32768 bytes)
-Memory: 999804K/1048576K available (8828K kernel code, 1090K rwdata, 4260K rodata, 1088K init, 396K bss, 32388K reserved, 16384K cma-reserved)
+Memory: 999804K/1048576K available (8828K kernel code, 1088K rwdata, 4264K rodata, 1088K init, 398K bss, 32388K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008920000   (  8832 KB)
     .rodata : 0xffff000008920000 - 0xffff000008d50000   (  4288 KB)
       .init : 0xffff000008d50000 - 0xffff000008e60000   (  1088 KB)
-      .data : 0xffff000008e60000 - 0xffff000008f70a00   (  1091 KB)
-       .bss : 0xffff000008f70a00 - 0xffff000008fd3bc0   (   397 KB)
+      .data : 0xffff000008e60000 - 0xffff000008f70200   (  1089 KB)
+       .bss : 0xffff000008f70200 - 0xffff000008fd3bc0   (   399 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -64,6 +64,7 @@
 Mount-cache hash table entries: 2048 (order: 2, 16384 bytes)
 Mountpoint-cache hash table entries: 2048 (order: 2, 16384 bytes)
 ASID allocator initialised with 65536 entries
+Hierarchical SRCU implementation.
 EFI services will not be available.
 smp: Bringing up secondary CPUs ...
 smp: Brought up 1 node, 1 CPU
@@ -114,7 +115,7 @@
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 kvm [1]: HYP mode not available
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.711:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.720:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -181,7 +182,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2017-05-17 15:38:01 UTC (1495035481)
+rtc-pl031 9010000.pl031: setting system clock to 2017-05-17 15:52:41 UTC (1495036361)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-05-17 08:54:18.771130882 -0700
+++ /tmp/foob	2017-05-17 08:54:18.775130900 -0700
@@ -40,20 +40,20 @@
 	 #################################################################
 	 #################################################################
 	 ####################
-	 3.3 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 7918496 (78d3a0 hex)
+Bytes transferred = 7917152 (78ce60 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3.3 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x78d3a0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x78ce60 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.12.0-rc1-next-20170516-172209-g7f49246 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed May 17 08:33:25 PDT 2017
+Linux version 4.12.0-rc1-next-20170517-172434-g973d630 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed May 17 08:48:24 PDT 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -72,7 +72,7 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c157fcc0, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c157fc40, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -95,11 +95,11 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1587930   (1567 kB)
-       .bss : 0xc1590d04 - 0xc15f5458   ( 402 kB)
+      .data : 0xc1400000 - 0xc15878b0   (1567 kB)
+       .bss : 0xc1590d10 - 0xc15f5458   ( 402 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
-	RCU debugfs-based tracing is enabled.
+	RCU event tracing is enabled.
 	RCU restricting CPUs from NR_CPUS=16 to nr_cpu_ids=4.
 RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=4
 NR_IRQS:16 nr_irqs:16 16
@@ -124,6 +124,7 @@
 /cpus/cpu@3 missing clock-frequency property
 CPU0: thread -1, cpu 0, socket 0, mpidr 80000000
 Setting up static identity map for 0x80300000 - 0x803000a0
+Hierarchical SRCU implementation.
 Tegra Revision: A01 SKU: 0 CPU Process: 1 SoC Process: 0
 EFI services will not be available.
 smp: Bringing up secondary CPUs ...
@@ -328,7 +329,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2001-03-11 03:09:34 UTC (984280174)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2001-03-11 03:24:34 UTC (984281074)
 vdd_hdmi_5v0: disabling
 dvdd_ts: disabling
 usb1_vbus: disabling
@@ -342,4 +343,4 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.449140] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.438972] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2017-05-17 08:54:18.779130917 -0700
+++ /tmp/foob	2017-05-17 08:54:18.783130934 -0700
@@ -38,9 +38,9 @@
 	 #################################################################
 	 #################################################################
 	 ##########################
-	 3.3 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 6092400 (5cf670 hex)
+Bytes transferred = 6093560 (5cfaf8 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -51,7 +51,7 @@
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5cf670 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5cfaf8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -60,7 +60,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.12.0-rc1-next-20170516-172209-g7f49246 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed May 17 08:33:23 PDT 2017
+Linux version 4.12.0-rc1-next-20170517-172434-g973d630 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed May 17 08:48:28 PDT 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -70,7 +70,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0eaea00, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0eae940, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -83,7 +83,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998568K/2096128K available (9216K kernel code, 739K rwdata, 2692K rodata, 1024K init, 299K bss, 32024K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998568K/2096128K available (9216K kernel code, 739K rwdata, 2696K rodata, 1024K init, 299K bss, 32024K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -93,7 +93,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0eb8ff8   ( 740 kB)
+      .data : 0xc0e00000 - 0xc0eb8f38   ( 740 kB)
        .bss : 0xc0ec264c - 0xc0f0d44c   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 NR_IRQS:16 nr_irqs:16 16
@@ -118,6 +118,7 @@
 /cpus/cpu@3 missing clock-frequency property
 CPU0: thread -1, cpu 0, socket 0, mpidr 80000000
 Setting up static identity map for 0x80100000 - 0x801000a0
+Hierarchical SRCU implementation.
 Tegra Revision: A01 SKU: 0 CPU Process: 1 SoC Process: 0
 smp: Bringing up secondary CPUs ...
 CPU1: thread -1, cpu 1, socket 0, mpidr 80000001
@@ -320,7 +321,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2001-03-11 03:12:17 UTC (984280337)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2001-03-11 03:27:14 UTC (984281234)
 vdd_hdmi_5v0: disabling
 dvdd_ts: disabling
 usb1_vbus: disabling
@@ -336,4 +337,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.658852] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.669245] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-05-17 08:54:18.787130951 -0700
+++ /tmp/foob	2017-05-17 08:54:18.791130967 -0700
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+MMC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -37,7 +37,7 @@
 	 ####################
 	 3.9 MiB/s
 done
-Bytes transferred = 7918496 (78d3a0 hex)
+Bytes transferred = 7917152 (78ce60 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -48,7 +48,7 @@
 done
 Bytes transferred = 67133 (1063d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x78d3a0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x78ce60 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301363c
@@ -56,7 +56,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.12.0-rc1-next-20170516-172209-g7f49246 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed May 17 08:33:25 PDT 2017
+Linux version 4.12.0-rc1-next-20170517-172434-g973d630 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed May 17 08:48:24 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -67,7 +67,7 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c157fcc0, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c157fc40, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -90,11 +90,11 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1587930   (1567 kB)
-       .bss : 0xc1590d04 - 0xc15f5458   ( 402 kB)
+      .data : 0xc1400000 - 0xc15878b0   (1567 kB)
+       .bss : 0xc1590d10 - 0xc15f5458   ( 402 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
-	RCU debugfs-based tracing is enabled.
+	RCU event tracing is enabled.
 	RCU restricting CPUs from NR_CPUS=16 to nr_cpu_ids=4.
 RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=4
 NR_IRQS:16 nr_irqs:16 16
@@ -119,6 +119,7 @@
 /cpus/cpu@3 missing clock-frequency property
 CPU0: thread -1, cpu 0, socket 0, mpidr 80000000
 Setting up static identity map for 0x80300000 - 0x803000a0
+Hierarchical SRCU implementation.
 Tegra Revision: A01 SKU: 129 CPU Process: 1 SoC Process: 1
 EFI services will not be available.
 smp: Bringing up secondary CPUs ...
@@ -315,6 +316,12 @@
 +5V_SATA: supplied by +5V_SYS
 +12V_SATA: supplied by +VDD_MUX
  usb2-0: usb2-0 supply vbus not found, using dummy regulator
+mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+ mmcblk0: p1
  usb2-1: usb2-1 supply vbus not found, using dummy regulator
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
@@ -339,16 +346,10 @@
 as3722-ldo10: supplied by +5V_SYS
 +2.8V_RUN_CAM_AF: Bringing 0uV into 2800000-2800000uV
 as3722-ldo11: supplied by +3.3V_RUN
-mmc0: new high speed MMC card at address 0001
 +1.8V_RUN_VPP_FUSE: Bringing 0uV into 1800000-1800000uV
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 tegra-ahci 70027000.sata: AHCI 0001.0301 32 slots 2 ports 3 Gbps 0x1 impl platform mode
 tegra-ahci 70027000.sata: flags: 64bit ncq sntf pm led pio slum part sadm sds apst 
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
 scsi host0: tegra-ahci
- mmcblk0: p1
 scsi host1: tegra-ahci
 ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 345
 ata2: DUMMY
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2017-05-17 08:54:18.795130984 -0700
+++ /tmp/foob	2017-05-17 08:54:18.799131001 -0700
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+MMC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -33,9 +33,9 @@
 	 #################################################################
 	 #################################################################
 	 ##########################
-	 3.9 MiB/s
+	 4 MiB/s
 done
-Bytes transferred = 6092400 (5cf670 hex)
+Bytes transferred = 6093560 (5cfaf8 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -46,7 +46,7 @@
 done
 Bytes transferred = 67133 (1063d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5cf670 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5cfaf8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301363c
@@ -55,7 +55,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.12.0-rc1-next-20170516-172209-g7f49246 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed May 17 08:33:23 PDT 2017
+Linux version 4.12.0-rc1-next-20170517-172434-g973d630 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed May 17 08:48:28 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -65,7 +65,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0eaea00, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0eae940, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -78,7 +78,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998448K/2096128K available (9216K kernel code, 739K rwdata, 2692K rodata, 1024K init, 299K bss, 32144K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998448K/2096128K available (9216K kernel code, 739K rwdata, 2696K rodata, 1024K init, 299K bss, 32144K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,7 +88,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0eb8ff8   ( 740 kB)
+      .data : 0xc0e00000 - 0xc0eb8f38   ( 740 kB)
        .bss : 0xc0ec264c - 0xc0f0d44c   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 NR_IRQS:16 nr_irqs:16 16
@@ -113,6 +113,7 @@
 /cpus/cpu@3 missing clock-frequency property
 CPU0: thread -1, cpu 0, socket 0, mpidr 80000000
 Setting up static identity map for 0x80100000 - 0x801000a0
+Hierarchical SRCU implementation.
 Tegra Revision: A01 SKU: 129 CPU Process: 1 SoC Process: 1
 smp: Bringing up secondary CPUs ...
 CPU1: thread -1, cpu 1, socket 0, mpidr 80000001
@@ -441,7 +442,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:01:38 UTC (946684898)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:01:29 UTC (946684889)
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +USB0_VBUS_SW: disabling
 +1.05V_RUN_AVDD_HDMI_PLL: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-05-17 08:54:18.803131018 -0700
+++ /tmp/foob	2017-05-17 08:54:18.803131018 -0700
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -45,18 +45,18 @@
 	 ####################
 	 3.6 MiB/s
 done
-Bytes transferred = 7918496 (78d3a0 hex)
+Bytes transferred = 7917152 (78ce60 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.4 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 68155 (10a3b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x78d3a0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x78ce60 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a3a
@@ -64,7 +64,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.12.0-rc1-next-20170516-172209-g7f49246 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed May 17 08:33:25 PDT 2017
+Linux version 4.12.0-rc1-next-20170517-172434-g973d630 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed May 17 08:48:24 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -76,7 +76,7 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c157fcc0, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c157fc40, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -99,11 +99,11 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1587930   (1567 kB)
-       .bss : 0xc1590d04 - 0xc15f5458   ( 402 kB)
+      .data : 0xc1400000 - 0xc15878b0   (1567 kB)
+       .bss : 0xc1590d10 - 0xc15f5458   ( 402 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
-	RCU debugfs-based tracing is enabled.
+	RCU event tracing is enabled.
 	RCU restricting CPUs from NR_CPUS=16 to nr_cpu_ids=4.
 RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=4
 NR_IRQS:16 nr_irqs:16 16
@@ -128,6 +128,7 @@
 /cpus/cpu@3 missing clock-frequency property
 CPU0: thread -1, cpu 0, socket 0, mpidr 80000000
 Setting up static identity map for 0x80300000 - 0x803000a0
+Hierarchical SRCU implementation.
 Tegra Revision: A01 SKU: 131 CPU Process: 1 SoC Process: 1
 EFI services will not be available.
 smp: Bringing up secondary CPUs ...
@@ -297,6 +298,8 @@
 mmc0: Unknown controller version (3). You may experience problems.
 mmc0: Invalid maximum block size, assuming 512 bytes
 mmc0: SDHCI controller on 700b0600.sdhci [700b0600.sdhci] using ADMA
+elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
+input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 ledtrig-cpu: registered to indicate activity on CPUs
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
@@ -313,10 +316,14 @@
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
 +3.3V_RUN: supplied by +3.3V_SYS
-elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
-input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 +VDD_LED: supplied by +VDD_MUX
 +3.3V_AVDD_HDMI_AP_GATED: supplied by +3.3V_RUN
+mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
+mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
+mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
+mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 +1.8V_RUN_CAM: supplied by +3.3V_RUN
 +VDDIO_SDMMC3: supplied by +3.3V_RUN
 +1.8V_RUN_VPP_FUSE: supplied by +3.3V_RUN
@@ -326,6 +333,7 @@
 +3.3V_PANEL: supplied by +3.3V_RUN
 +3.3V_LP0: supplied by +3.3V_SYS
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
+elan_i2c 1-0015: failed to read report data: -121
 +5V_HDMI_CON: supplied by +5V_SYS
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
  usb3-1: usb3-1 supply vbus not found, using dummy regulator
@@ -335,13 +343,8 @@
 mmc1: Unknown controller version (3). You may experience problems.
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: Switching to 3.3V signalling voltage failed
-mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
-mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
-mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
-mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 mmc1: SDHCI controller on 700b0000.sdhci [700b0000.sdhci] using ADMA
+elan_i2c 1-0015: invalid report id data (ff)
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc2: Unknown controller version (3). You may experience problems.
@@ -349,7 +352,7 @@
 mmc1: new high speed SDIO card at address 0001
 mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:22 UTC (946684822)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:23 UTC (946684823)
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +1.05V_RUN: disabling
 +1.8V_RUN_CAM: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2017-05-17 08:54:18.811131053 -0700
+++ /tmp/foob	2017-05-17 08:54:18.815131069 -0700
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -41,20 +41,20 @@
 	 #################################################################
 	 #################################################################
 	 ##########################
-	 3.5 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 6092400 (5cf670 hex)
+Bytes transferred = 6093560 (5cfaf8 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.4 MiB/s
+	 3.6 MiB/s
 done
 Bytes transferred = 68155 (10a3b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5cf670 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5cfaf8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a3a
@@ -63,7 +63,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.12.0-rc1-next-20170516-172209-g7f49246 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed May 17 08:33:23 PDT 2017
+Linux version 4.12.0-rc1-next-20170517-172434-g973d630 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed May 17 08:48:28 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -74,7 +74,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c0eaea00, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0eae940, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -87,7 +87,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999464K/2097148K available (9216K kernel code, 739K rwdata, 2692K rodata, 1024K init, 299K bss, 32148K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1999464K/2097148K available (9216K kernel code, 739K rwdata, 2696K rodata, 1024K init, 299K bss, 32148K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -97,7 +97,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0eb8ff8   ( 740 kB)
+      .data : 0xc0e00000 - 0xc0eb8f38   ( 740 kB)
        .bss : 0xc0ec264c - 0xc0f0d44c   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 NR_IRQS:16 nr_irqs:16 16
@@ -122,6 +122,7 @@
 /cpus/cpu@3 missing clock-frequency property
 CPU0: thread -1, cpu 0, socket 0, mpidr 80000000
 Setting up static identity map for 0x80100000 - 0x801000a0
+Hierarchical SRCU implementation.
 Tegra Revision: A01 SKU: 131 CPU Process: 1 SoC Process: 1
 smp: Bringing up secondary CPUs ...
 CPU1: thread -1, cpu 1, socket 0, mpidr 80000001
@@ -294,12 +295,12 @@
 input: GoogleNyanBig Headphones as /devices/soc0/sound/sound/card1/input1
 input: GoogleNyanBig Mic Jack as /devices/soc0/sound/sound/card1/input2
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
-mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 NET: Registered protocol family 10
 Segment Routing with IPv6
 mip6: Mobile IPv6
 sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
+mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 NET: Registered protocol family 17
 NET: Registered protocol family 15
 can: controller area network core (rev 20170425 abi 9)
@@ -356,7 +357,7 @@
 mmc1: new high speed SDIO card at address 0001
 mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
 input: gpio-keys as /devices/soc0/gpio-keys/input/input3
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:23 UTC (946684823)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:22 UTC (946684822)
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +1.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
@@ -390,5 +391,6 @@
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
-bash: no job control in this shell
-root@(none):/# [    2.999249] usb 1-3: new high-speed USB device number 3 using tegra-xusb
+bash: no j[    2.979362] usb 1-3: new high-speed USB device number 3 using tegra-xusb
+ob control in this shell
+root@(none):/# 
\ No newline at end of file
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2017-05-17 08:54:18.819131085 -0700
+++ /tmp/foob	2017-05-17 08:54:18.819131085 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1801037 us
+Bootloader-Cpu Init at (time stamp): 1800986 us
 
 Pinmux changes applied in kernel way
 
@@ -62,22 +62,22 @@
 Board Id Read Time: 4 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1801010  us
-Bootloader (AosInit) time: 734227 us
-Bootloader Main Init time: 180082 us
-BlockDevMgrInit time: 157528 us
+Miscellaneous avp time: 1800959  us
+Bootloader (AosInit) time: 734247 us
+Bootloader Main Init time: 180076 us
+BlockDevMgrInit time: 157514 us
 PartMgrInit time: 8 us
-StorMgrInit time: 31 us
+StorMgrInit time: 30 us
 FsMgrInit time: 2 us
-Partition Table load: 76723 us 
-Display init time: 574800 us
-Kernel read time: 390 msec
-Load Kernel Size:   16986112 Bytes
-Bl Kernel ReadSpeed: 43554000 Bytes/sec
-Pre-Kernel Jump time: 1792715 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5914925 us
+Partition Table load: 77078 us 
+Display init time: 574442 us
+Kernel read time: 391 msec
+Load Kernel Size:   16984064 Bytes
+Bl Kernel ReadSpeed: 43437000 Bytes/sec
+Pre-Kernel Jump time: 1792106 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5915120 us
 Booting Linux on physical CPU 0x0
-Linux version 4.12.0-rc1-next-20170516-290450-g7f49246 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed May 17 08:37:18 PDT 2017
+Linux version 4.12.0-rc1-next-20170517-290675-g973d630 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed May 17 08:51:59 PDT 2017
 Boot CPU: AArch64 Processor [4e0f0000]
 Machine model: NVIDIA Tegra132 Norrin
 Memory limited to 1734MB
@@ -103,15 +103,15 @@
 Policy zone: DMA
 Kernel command line: tegraid=13.1.3.0.0 mem=1734M@2048M memtype=255 vpr=151M@3817M ddr_die=2048M@2048M section=256M commchip_id=0 usb_port_owner_info=0 lane_owner_info=0 emc_max_dvfs=0 touch_id=0@0 video=tegrafb no_console_suspend=1 console=ttyS0,115200n8 debug_uartport=lsport,0 earlyprintk=uart8250-32bit,0x70006000 usbcore.old_scheme_first=1 lp0_vec=0@0x0 tegra_fbmem=32899072@0xad012000 core_edp_mv=1150 core_edp_ma=4000 pmuboard=0x0176:0x0001:0x01:0x43:0x00 displayboard=0x0162:0x0001:0x00:0x42:0x00 power_supply=Adapter audio_codec=rt5640 board_info=0x0176:0x0001:0x01:0x43:0x00 tegraboot=sdmmc gpt gpt_sector=192511 modem_id=0 android.kerneltype=normal
 PID hash table entries: 4096 (order: 3, 32768 bytes)
-Memory: 1709020K/1775616K available (8828K kernel code, 1090K rwdata, 4260K rodata, 1088K init, 396K bss, 50212K reserved, 16384K cma-reserved)
+Memory: 1709020K/1775616K available (8828K kernel code, 1088K rwdata, 4264K rodata, 1088K init, 398K bss, 50212K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008920000   (  8832 KB)
     .rodata : 0xffff000008920000 - 0xffff000008d50000   (  4288 KB)
       .init : 0xffff000008d50000 - 0xffff000008e60000   (  1088 KB)
-      .data : 0xffff000008e60000 - 0xffff000008f70a00   (  1091 KB)
-       .bss : 0xffff000008f70a00 - 0xffff000008fd3bc0   (   397 KB)
+      .data : 0xffff000008e60000 - 0xffff000008f70200   (  1089 KB)
+       .bss : 0xffff000008f70200 - 0xffff000008fd3bc0   (   399 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -136,6 +136,7 @@
 Mount-cache hash table entries: 4096 (order: 3, 32768 bytes)
 Mountpoint-cache hash table entries: 4096 (order: 3, 32768 bytes)
 ASID allocator initialised with 256 entries
+Hierarchical SRCU implementation.
 EFI services will not be available.
 smp: Bringing up secondary CPUs ...
 smp: Brought up 1 node, 1 CPU
@@ -198,7 +199,7 @@
 Freeing initrd memory: 1284K
 kvm [1]: HYP mode not available
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.909:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.917:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-05-17 08:54:18.827131120 -0700
+++ /tmp/foob	2017-05-17 08:54:18.827131120 -0700
@@ -48,9 +48,9 @@
 	 #################################################################
 	 #################################################################
 	 ####################
-	 4.1 MiB/s
+	 4 MiB/s
 done
-Bytes transferred = 7918496 (78d3a0 hex)
+Bytes transferred = 7917152 (78ce60 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -61,7 +61,7 @@
 done
 Bytes transferred = 21426 (53b2 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x78d3a0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x78ce60 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103b1
@@ -69,7 +69,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.12.0-rc1-next-20170516-172209-g7f49246 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed May 17 08:33:25 PDT 2017
+Linux version 4.12.0-rc1-next-20170517-172434-g973d630 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed May 17 08:48:24 PDT 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -79,7 +79,7 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0x3c000000
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c157fcc0, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c157fc40, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -102,11 +102,11 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1587930   (1567 kB)
-       .bss : 0xc1590d04 - 0xc15f5458   ( 402 kB)
+      .data : 0xc1400000 - 0xc15878b0   (1567 kB)
+       .bss : 0xc1590d10 - 0xc15f5458   ( 402 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
-	RCU debugfs-based tracing is enabled.
+	RCU event tracing is enabled.
 	RCU restricting CPUs from NR_CPUS=16 to nr_cpu_ids=2.
 RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=2
 NR_IRQS:16 nr_irqs:16 16
@@ -130,6 +130,7 @@
 CPU: Testing write buffer coherency: ok
 CPU0: thread -1, cpu 0, socket 0, mpidr 80000000
 Setting up static identity map for 0x300000 - 0x3000a0
+Hierarchical SRCU implementation.
 Tegra Revision: A03 prime SKU: 8 CPU Process: 1 SoC Process: 2
 EFI services will not be available.
 smp: Bringing up secondary CPUs ...
@@ -345,8 +346,8 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-hub 1-1:1.0: USB hub found
 rtc-em3027 2-0056: hctosys: unable to read the hardware clock
+hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
 Waiting for root device PARTUUID=11eb23c1-01...
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2017-05-17 08:54:18.831131138 -0700
+++ /tmp/foob	2017-05-17 08:54:18.835131154 -0700
@@ -48,7 +48,7 @@
 	 ##########################
 	 4.1 MiB/s
 done
-Bytes transferred = 6092400 (5cf670 hex)
+Bytes transferred = 6093560 (5cfaf8 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -59,7 +59,7 @@
 done
 Bytes transferred = 21426 (53b2 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x5cf670 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x5cfaf8 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103b1
@@ -68,7 +68,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.12.0-rc1-next-20170516-172209-g7f49246 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed May 17 08:33:23 PDT 2017
+Linux version 4.12.0-rc1-next-20170517-172434-g973d630 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed May 17 08:48:28 PDT 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -77,7 +77,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0x3c000000
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0eaea00, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0eae940, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -90,7 +90,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 959396K/1048576K available (9216K kernel code, 739K rwdata, 2692K rodata, 1024K init, 299K bss, 23644K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 959396K/1048576K available (9216K kernel code, 739K rwdata, 2696K rodata, 1024K init, 299K bss, 23644K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -100,7 +100,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0eb8ff8   ( 740 kB)
+      .data : 0xc0e00000 - 0xc0eb8f38   ( 740 kB)
        .bss : 0xc0ec264c - 0xc0f0d44c   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
@@ -126,6 +126,7 @@
 CPU: Testing write buffer coherency: ok
 CPU0: thread -1, cpu 0, socket 0, mpidr 80000000
 Setting up static identity map for 0x100000 - 0x1000a0
+Hierarchical SRCU implementation.
 Tegra Revision: A03 prime SKU: 8 CPU Process: 1 SoC Process: 2
 smp: Bringing up secondary CPUs ...
 CPU1: thread -1, cpu 1, socket 0, mpidr 80000001
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2017-05-17 08:54:18.839131170 -0700
+++ /tmp/foob	2017-05-17 08:54:18.843131187 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 758713 us
-Basic modules init took 350236 us
+Tegraboot started after 777932 us
+Basic modules init took 350756 us
 Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14362 us
-Carveout took 252729 us
-CPU initialization took 92309 us
-Total time taken by TegraBoot 709636 us
+Next stage binary read took 14365 us
+Carveout took 252717 us
+CPU initialization took 92311 us
+Total time taken by TegraBoot 710149 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.599] LPDDR4 Training: Number of tables = 10
+64b[0001.618] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,9 +117,9 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-15665664 bytes read in 811 ms (18.4 MiB/s)
+15663616 bytes read in 811 ms (18.4 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-45142 bytes read in 461 ms (94.7 KiB/s)
+45142 bytes read in 460 ms (95.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
 1627787 bytes read in 495 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.12.0-rc1-next-20170516-290450-g7f49246 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed May 17 08:37:18 PDT 2017
+Linux version 4.12.0-rc1-next-20170517-290675-g973d630 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed May 17 08:51:59 PDT 2017
 Boot CPU: AArch64 Processor [411fd071]
 Machine model: NVIDIA Tegra210 P2371 (P2530/P2595) reference design
 debug: ignoring loglevel setting.
@@ -175,15 +175,15 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffff80007a000000-ffff80007dffffff]
-Memory: 2983376K/3131392K available (8828K kernel code, 1090K rwdata, 4260K rodata, 1088K init, 396K bss, 131632K reserved, 16384K cma-reserved)
+Memory: 2983376K/3131392K available (8828K kernel code, 1088K rwdata, 4264K rodata, 1088K init, 398K bss, 131632K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008920000   (  8832 KB)
     .rodata : 0xffff000008920000 - 0xffff000008d50000   (  4288 KB)
       .init : 0xffff000008d50000 - 0xffff000008e60000   (  1088 KB)
-      .data : 0xffff000008e60000 - 0xffff000008f70a00   (  1091 KB)
-       .bss : 0xffff000008f70a00 - 0xffff000008fd3bc0   (   397 KB)
+      .data : 0xffff000008e60000 - 0xffff000008f70200   (  1089 KB)
+       .bss : 0xffff000008f70200 - 0xffff000008fd3bc0   (   399 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -211,6 +211,7 @@
 Mount-cache hash table entries: 8192 (order: 4, 65536 bytes)
 Mountpoint-cache hash table entries: 8192 (order: 4, 65536 bytes)
 ASID allocator initialised with 65536 entries
+Hierarchical SRCU implementation.
 Speedo Revision 4
 Tegra Revision: A01 SKU: 1 CPU Process: -22 SoC Process: -22
 EFI services will not be available.
@@ -267,14 +268,14 @@
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 PCI: CLS 0 bytes, default 128
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 8090e000
+kvm [1]: IDMAP page: 8090f000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
 kvm [1]: vgic interrupt IRQ1
 kvm [1]: virtual timer IRQ4
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.531:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.535:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -355,6 +356,6 @@
 devtmpfs: mounted
 Freeing unused kernel memory: 1088K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
-bash: no job contr[    1.350469] random: fast init done
+bash: no job contr[    1.334971] random: fast init done
 ol in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2017-05-17 08:54:18.847131205 -0700
+++ /tmp/foob	2017-05-17 08:54:18.851131223 -0700
@@ -1,5 +1,4 @@
- Google Pixel C
-efi: Getting EFI parameters from FDT:
+g EFI parameters from FDT:
 efi: UEFI not found.
 cma: Reserved 16 MiB at 0x00000000f5400000
 earlycon: uart0 at MMIO 0x0000000070006000 (options '115200n8')
@@ -31,15 +30,15 @@
 Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2838640K/2988032K available (8828K kernel code, 1090K rwdata, 4260K rodata, 1088K init, 396K bss, 133008K reserved, 16384K cma-reserved)
+Memory: 2838644K/2988032K available (8828K kernel code, 1088K rwdata, 4264K rodata, 1088K init, 398K bss, 133004K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008920000   (  8832 KB)
     .rodata : 0xffff000008920000 - 0xffff000008d50000   (  4288 KB)
       .init : 0xffff000008d50000 - 0xffff000008e60000   (  1088 KB)
-      .data : 0xffff000008e60000 - 0xffff000008f70a00   (  1091 KB)
-       .bss : 0xffff000008f70a00 - 0xffff000008fd3bc0   (   397 KB)
+      .data : 0xffff000008e60000 - 0xffff000008f70200   (  1089 KB)
+       .bss : 0xffff000008f70200 - 0xffff000008fd3bc0   (   399 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -76,7 +75,7 @@
 console [tty0] enabled
 bootconsole [uart0] disabled
 Booting Linux on physical CPU 0x0
-Linux version 4.12.0-rc1-next-20170516-290450-g7f49246 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed May 17 08:37:18 PDT 2017
+Linux version 4.12.0-rc1-next-20170517-290675-g973d630 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed May 17 08:51:59 PDT 2017
 Boot CPU: AArch64 Processor [411fd071]
 Machine model: Google Pixel C
 efi: Getting EFI parameters from FDT:
@@ -111,15 +110,15 @@
 Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2838640K/2988032K available (8828K kernel code, 1090K rwdata, 4260K rodata, 1088K init, 396K bss, 133008K reserved, 16384K cma-reserved)
+Memory: 2838644K/2988032K available (8828K kernel code, 1088K rwdata, 4264K rodata, 1088K init, 398K bss, 133004K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008920000   (  8832 KB)
     .rodata : 0xffff000008920000 - 0xffff000008d50000   (  4288 KB)
       .init : 0xffff000008d50000 - 0xffff000008e60000   (  1088 KB)
-      .data : 0xffff000008e60000 - 0xffff000008f70a00   (  1091 KB)
-       .bss : 0xffff000008f70a00 - 0xffff000008fd3bc0   (   397 KB)
+      .data : 0xffff000008e60000 - 0xffff000008f70200   (  1089 KB)
+       .bss : 0xffff000008f70200 - 0xffff000008fd3bc0   (   399 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -163,6 +162,7 @@
 Mount-cache hash table entries: 8192 (order: 4, 65536 bytes)
 Mountpoint-cache hash table entries: 8192 (order: 4, 65536 bytes)
 ASID allocator initialised with 65536 entries
+Hierarchical SRCU implementation.
 Speedo Revision 4
 Tegra Revision: A02 SKU: 19 CPU Process: 0 SoC Process: 0
 EFI services will not be available.
@@ -224,9 +224,9 @@
 RPC: Registered tcp transport module.
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 Unpacking initramfs...
-Freeing initrd memory: 1284K
+Freeing initrd memory: 1280K
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 8090e000
+kvm [1]: IDMAP page: 8090f000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
@@ -332,15 +332,16 @@
 Key type dns_resolver registered
 registered taskstats version 1
 tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
-input: gpio-keys as /devices/platform/gpio-keys/input/input0
-tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 BGND3R 29.1 GiB 
 mmcblk0boot0: mmc0:0001 BGND3R partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
-max77686-rtc max77620-rtc: setting system clock to 2001-02-25 21:03:27 UTC (983135007)
+tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
+input: gpio-keys as /devices/platform/gpio-keys/input/input0
+tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
+max77686-rtc max77620-rtc: setting system clock to 2001-02-25 21:18:07 UTC (983135887)
 PP1200_AVDD: disabling
 PP1200_RCAM: disabling
 PP2800L_RCAM: disabling
@@ -351,7 +352,6 @@
 PP1800_CAM: disabling
 ALSA device list:
   No soundcards found.
-tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 Freeing unused kernel memory: 1088K
 
 System initialization...
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-05-17 08:54:18.855131239 -0700
+++ /tmp/foob	2017-05-17 08:54:18.859131256 -0700
@@ -40,20 +40,20 @@
 	 #################################################################
 	 #################################################################
 	 ####################
-	 3.3 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 7918496 (78d3a0 hex)
+Bytes transferred = 7917152 (78ce60 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.5 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 51804 (ca5c hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x78d3a0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x78ce60 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa5b ... OK
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.12.0-rc1-next-20170516-172209-g7f49246 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed May 17 08:33:25 PDT 2017
+Linux version 4.12.0-rc1-next-20170517-172434-g973d630 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed May 17 08:48:24 PDT 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -71,7 +71,7 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c157fcc0, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c157fc40, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -94,11 +94,11 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1587930   (1567 kB)
-       .bss : 0xc1590d04 - 0xc15f5458   ( 402 kB)
+      .data : 0xc1400000 - 0xc15878b0   (1567 kB)
+       .bss : 0xc1590d10 - 0xc15f5458   ( 402 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
-	RCU debugfs-based tracing is enabled.
+	RCU event tracing is enabled.
 	RCU restricting CPUs from NR_CPUS=16 to nr_cpu_ids=4.
 RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=4
 NR_IRQS:16 nr_irqs:16 16
@@ -122,6 +122,7 @@
 CPU: Testing write buffer coherency: ok
 CPU0: thread -1, cpu 0, socket 0, mpidr 80000000
 Setting up static identity map for 0x80300000 - 0x803000a0
+Hierarchical SRCU implementation.
 Tegra Revision: A03 SKU: 128 CPU Process: 4 SoC Process: 0
 EFI services will not be available.
 smp: Bringing up secondary CPUs ...
@@ -307,18 +308,18 @@
 usb3_vbus: supplied by vdd_5v_in
 sys_3v3,vdd_3v3_alw: supplied by vdd_5v_in
 sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
-+VDD_5V_HDMI: supplied by sys_3v3,vdd_3v3_alw
-tegra-pcie 3000.pcie-controller: 2x3 configuration
-tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB
-mmc0: new ultra high speed DDR50 SDHC card at address aaaa
-mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
++VDD_5V_HDMI: supplied by sys_3v3,vdd_3v3_alw
+tegra-pcie 3000.pcie-controller: 2x3 configuration
+tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
 Primary GPT is invalid, using alternate GPT.
  mmcblk1: p1 p2 p3 p4 p5
+mmc0: new ultra high speed DDR50 SDHC card at address aaaa
+mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
 tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
  mmcblk0: p1 p2 p3
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -392,5 +393,5 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.064012] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.083623] usb 2-1: new high-speed USB device number 2 using tegra-ehci
 asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0f:67:60
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2017-05-17 08:54:18.867131289 -0700
+++ /tmp/foob	2017-05-17 08:54:18.867131289 -0700
@@ -27,6 +27,7 @@
 Tegra30 (Beaver) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=edbe1840-3633-c751-7993-b0ac834f81e0 rootwait gpt debug noinitrd earlyprintk init=/bin/bash'
 Tegra30 (Beaver) # setenv ipaddr 192.168.100.101; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra30 (Beaver) # tftpboot 0x81000000 zImage.tegra_defconfig
+Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'zImage.tegra_defconfig'.
@@ -38,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 ##########################
-	 3.1 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 6092400 (5cf670 hex)
+Bytes transferred = 6093560 (5cfaf8 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 51804 (ca5c hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5cf670 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5cfaf8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa5b ... OK
@@ -60,7 +61,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.12.0-rc1-next-20170516-172209-g7f49246 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed May 17 08:33:23 PDT 2017
+Linux version 4.12.0-rc1-next-20170517-172434-g973d630 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed May 17 08:48:28 PDT 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -69,7 +70,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0eaea00, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0eae940, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -82,7 +83,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998488K/2096128K available (9216K kernel code, 739K rwdata, 2692K rodata, 1024K init, 299K bss, 32104K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998488K/2096128K available (9216K kernel code, 739K rwdata, 2696K rodata, 1024K init, 299K bss, 32104K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,7 +93,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0eb8ff8   ( 740 kB)
+      .data : 0xc0e00000 - 0xc0eb8f38   ( 740 kB)
        .bss : 0xc0ec264c - 0xc0f0d44c   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 NR_IRQS:16 nr_irqs:16 16
@@ -116,6 +117,7 @@
 CPU: Testing write buffer coherency: ok
 CPU0: thread -1, cpu 0, socket 0, mpidr 80000000
 Setting up static identity map for 0x80100000 - 0x801000a0
+Hierarchical SRCU implementation.
 Tegra Revision: A03 SKU: 128 CPU Process: 4 SoC Process: 0
 smp: Bringing up secondary CPUs ...
 CPU1: thread -1, cpu 1, socket 0, mpidr 80000001
@@ -386,4 +388,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.370164] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.370079] usb 2-1: new high-speed USB device number 2 using tegra-ehci
