Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2015-08-21 03:23:45.212206283 -0700
+++ /tmp/foob	2015-08-21 03:23:45.216206280 -0700
@@ -1,6 +1,6 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
-Linux version 4.2.0-rc7-next-20150820+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Aug 20 03:49:34 PDT 2015
+Linux version 4.2.0-rc7-next-20150821+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Fri Aug 21 03:19:04 PDT 2015
 CPU: AArch64 Processor [411fd070] revision 0
 Detected PIPT I-cache on CPU0
 alternatives: enabling workaround for ARM erratum 832075
@@ -91,7 +91,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 2, 16384 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.580:1): initialized
+audit: type=2000 audit(0.670:1): initialized
 HugeTLB registered 2 MB page size, pre-allocated 0 pages
 VFS: Disk quotas dquot_6.6.0
 VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-08-21 03:23:45.220206278 -0700
+++ /tmp/foob	2015-08-21 03:23:45.224206275 -0700
@@ -38,20 +38,20 @@
 	 #################################################################
 	 #################################################################
 	 ###############################################
-	 3.4 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 6400616 (61aa68 hex)
+Bytes transferred = 6404840 (61bae8 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.8 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 34966 (8896 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x61aa68 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x61bae8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff895 ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc7-next-20150820-38036-gf0359ea (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Aug 20 03:47:55 PDT 2015
+Linux version 4.2.0-rc7-next-20150821-38393-g1ef981b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Fri Aug 21 03:17:49 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -67,7 +67,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f6a640, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0f6c680, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -80,7 +80,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998944K/2096128K available (8043K kernel code, 1017K rwdata, 3620K rodata, 1028K init, 340K bss, 31648K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1998936K/2096128K available (8051K kernel code, 1017K rwdata, 3620K rodata, 1028K init, 340K bss, 31656K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,10 +88,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d6cdfc   (11668 kB)
-      .init : 0xc0d6d000 - 0xc0e6e000   (1028 kB)
-      .data : 0xc0e6e000 - 0xc0f6c620   (1018 kB)
-       .bss : 0xc0f6f000 - 0xc0fc42e8   ( 341 kB)
+      .text : 0xc0208000 - 0xc0d6eeb0   (11676 kB)
+      .init : 0xc0d6f000 - 0xc0e70000   (1028 kB)
+      .data : 0xc0e70000 - 0xc0f6e660   (1018 kB)
+       .bss : 0xc0f71000 - 0xc0fc62e8   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -259,6 +259,10 @@
 mmc0: MAN_BKOPS_EN bit is not set
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 SEM32G 29.1 GiB 
+mmcblk0boot0: mmc0:0001 SEM32G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM32G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM32G partition 3 4.00 MiB
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 vdd-lcd-bl: supplied by vdd_ac_bat
 vdd-sys-5v0: supplied by vdd_ac_bat
 VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
@@ -267,11 +271,7 @@
 [drm] No driver support for vblank timestamp query.
 drm drm: No connectors reported connected with modes
 [drm] Cannot find any crtc or sizes - going 1024x768
-mmcblk0boot0: mmc0:0001 SEM32G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM32G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM32G partition 3 4.00 MiB
 Console: switching to colour frame buffer device 128x48
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 vdd-sys-3v3: supplied by vdd_ac_bat
@@ -291,7 +291,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 78000400.sdhci [78000400.sdhci] using ADMA
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-03-01 19:41:45 UTC (951939705)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-03-02 19:11:42 UTC (952024302)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -300,7 +300,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1028K (c0d6d000 - c0e6e000)
+Freeing unused kernel memory: 1028K (c0d6f000 - c0e70000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2015-08-21 03:23:45.228206272 -0700
+++ /tmp/foob	2015-08-21 03:23:45.232206270 -0700
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 ###################
-	 3.5 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 5043848 (4cf688 hex)
+Bytes transferred = 5043840 (4cf680 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.8 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 34966 (8896 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4cf688 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4cf680 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff895 ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc7-next-20150820-38036-gf0359ea (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Aug 20 03:48:13 PDT 2015
+Linux version 4.2.0-rc7-next-20150821-38393-g1ef981b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Aug 21 03:18:05 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002684K/2096128K available (6884K kernel code, 344K rwdata, 2388K rodata, 324K init, 363K bss, 27908K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 2002684K/2096128K available (6884K kernel code, 344K rwdata, 2384K rodata, 328K init, 363K bss, 27908K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,8 +90,8 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09166d8   (9274 kB)
-      .init : 0xc0917000 - 0xc0968000   ( 324 kB)
+      .text : 0xc0008000 - 0xc09156bc   (9270 kB)
+      .init : 0xc0916000 - 0xc0968000   ( 328 kB)
       .data : 0xc0968000 - 0xc09be080   ( 345 kB)
        .bss : 0xc09c1000 - 0xc0a1bd20   ( 364 kB)
 Preemptible hierarchical RCU implementation.
@@ -239,13 +239,13 @@
 mmc0: MAN_BKOPS_EN bit is not set
 NET: Registered protocol family 17
 mmc0: new high speed MMC card at address 0001
+NET: Registered protocol family 15
+can: controller area network core (rev 20120528 abi 9)
 mmcblk0: mmc0:0001 SEM32G 29.1 GiB 
 mmcblk0boot0: mmc0:0001 SEM32G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM32G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM32G partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
-NET: Registered protocol family 15
-can: controller area network core (rev 20120528 abi 9)
 NET: Registered protocol family 29
 can: raw protocol (rev 20120528)
 can: broadcast manager protocol (rev 20120528 t)
@@ -270,54 +270,54 @@
 WARNING: CPU: 0 PID: 6 at include/linux/ww_mutex.h:158 drm_atomic_check_only+0x4fc/0x574()
 DEBUG_LOCKS_WARN_ON(ctx->done_acquire)
 Modules linked in:
-CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.2.0-rc7-next-20150820-38036-gf0359ea #1
+CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.2.0-rc7-next-20150821-38393-g1ef981b #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0016898>] (unwind_backtrace) from [<c0012124>] (show_stack+0x10/0x14)
-[<c0012124>] (show_stack) from [<c067e850>] (dump_stack+0x84/0xd0)
-[<c067e850>] (dump_stack) from [<c00257f8>] (warn_slowpath_common+0x74/0xb0)
-[<c00257f8>] (warn_slowpath_common) from [<c00258c8>] (warn_slowpath_fmt+0x30/0x40)
-[<c00258c8>] (warn_slowpath_fmt) from [<c02e5e98>] (drm_atomic_check_only+0x4fc/0x574)
-[<c02e5e98>] (drm_atomic_check_only) from [<c02e5f84>] (drm_atomic_commit+0x10/0x64)
-[<c02e5f84>] (drm_atomic_commit) from [<c02c49fc>] (drm_atomic_helper_set_config+0x394/0x3ec)
-[<c02c49fc>] (drm_atomic_helper_set_config) from [<c02d612c>] (drm_mode_set_config_internal+0x58/0xd4)
-[<c02d612c>] (drm_mode_set_config_internal) from [<c02c7004>] (restore_fbdev_mode+0xd4/0xf4)
-[<c02c7004>] (restore_fbdev_mode) from [<c02c886c>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
-[<c02c886c>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c02c88d8>] (drm_fb_helper_set_par+0x2c/0x54)
-[<c02c88d8>] (drm_fb_helper_set_par) from [<c026504c>] (fbcon_init+0x474/0x4dc)
-[<c026504c>] (fbcon_init) from [<c029bc20>] (visual_init+0xb0/0xfc)
-[<c029bc20>] (visual_init) from [<c029dbc4>] (do_bind_con_driver+0x134/0x354)
-[<c029dbc4>] (do_bind_con_driver) from [<c029df54>] (do_take_over_console+0x170/0x1dc)
-[<c029df54>] (do_take_over_console) from [<c0265118>] (do_fbcon_takeover+0x64/0xcc)
-[<c0265118>] (do_fbcon_takeover) from [<c003e590>] (notifier_call_chain+0x44/0x84)
-[<c003e590>] (notifier_call_chain) from [<c003e764>] (__blocking_notifier_call_chain+0x48/0x60)
-[<c003e764>] (__blocking_notifier_call_chain) from [<c003e794>] (blocking_notifier_call_chain+0x18/0x20)
-[<c003e794>] (blocking_notifier_call_chain) from [<c026f008>] (register_framebuffer+0x184/0x264)
-[<c026f008>] (register_framebuffer) from [<c02c8b44>] (drm_fb_helper_initial_config+0x244/0x3e4)
-[<c02c8b44>] (drm_fb_helper_initial_config) from [<c02eac20>] (tegra_drm_fb_init+0x48/0xb4)
-[<c02eac20>] (tegra_drm_fb_init) from [<c02e8eb8>] (tegra_drm_load+0x174/0x228)
-[<c02e8eb8>] (tegra_drm_load) from [<c02d0e84>] (drm_dev_register+0xa0/0xb4)
-[<c02d0e84>] (drm_dev_register) from [<c02e8a10>] (host1x_drm_probe+0x44/0xb0)
-[<c02e8a10>] (host1x_drm_probe) from [<c02b3b84>] (host1x_device_probe+0x1c/0x28)
-[<c02b3b84>] (host1x_device_probe) from [<c03008d0>] (driver_probe_device+0x1e0/0x2a0)
-[<c03008d0>] (driver_probe_device) from [<c02fef70>] (bus_for_each_drv+0x44/0x8c)
-[<c02fef70>] (bus_for_each_drv) from [<c0300664>] (__device_attach+0xa0/0x104)
-[<c0300664>] (__device_attach) from [<c02ffde0>] (bus_probe_device+0x84/0x8c)
-[<c02ffde0>] (bus_probe_device) from [<c02fe334>] (device_add+0x33c/0x524)
-[<c02fe334>] (device_add) from [<c02b3d90>] (host1x_subdev_register+0xb0/0xd4)
-[<c02b3d90>] (host1x_subdev_register) from [<c02b4648>] (host1x_client_register+0xf4/0x11c)
-[<c02b4648>] (host1x_client_register) from [<c02f22e0>] (tegra_hdmi_probe+0x170/0x250)
-[<c02f22e0>] (tegra_hdmi_probe) from [<c0301da8>] (platform_drv_probe+0x48/0xa4)
-[<c0301da8>] (platform_drv_probe) from [<c03008d0>] (driver_probe_device+0x1e0/0x2a0)
-[<c03008d0>] (driver_probe_device) from [<c02fef70>] (bus_for_each_drv+0x44/0x8c)
-[<c02fef70>] (bus_for_each_drv) from [<c0300664>] (__device_attach+0xa0/0x104)
-[<c0300664>] (__device_attach) from [<c02ffde0>] (bus_probe_device+0x84/0x8c)
-[<c02ffde0>] (bus_probe_device) from [<c03001b8>] (deferred_probe_work_func+0x5c/0x88)
-[<c03001b8>] (deferred_probe_work_func) from [<c0038650>] (process_one_work+0x120/0x328)
-[<c0038650>] (process_one_work) from [<c00388b4>] (worker_thread+0x28/0x490)
-[<c00388b4>] (worker_thread) from [<c003d9ac>] (kthread+0xd8/0xf4)
-[<c003d9ac>] (kthread) from [<c000f628>] (ret_from_fork+0x14/0x2c)
----[ end trace 9f5b6b2d89f1868a ]---
+[<c0012124>] (show_stack) from [<c0211d60>] (dump_stack+0x84/0xd0)
+[<c0211d60>] (dump_stack) from [<c0025e78>] (warn_slowpath_common+0x74/0xb0)
+[<c0025e78>] (warn_slowpath_common) from [<c0025f48>] (warn_slowpath_fmt+0x30/0x40)
+[<c0025f48>] (warn_slowpath_fmt) from [<c02e86b4>] (drm_atomic_check_only+0x4fc/0x574)
+[<c02e86b4>] (drm_atomic_check_only) from [<c02e87a0>] (drm_atomic_commit+0x10/0x64)
+[<c02e87a0>] (drm_atomic_commit) from [<c02c7218>] (drm_atomic_helper_set_config+0x394/0x3ec)
+[<c02c7218>] (drm_atomic_helper_set_config) from [<c02d8948>] (drm_mode_set_config_internal+0x58/0xd4)
+[<c02d8948>] (drm_mode_set_config_internal) from [<c02c9820>] (restore_fbdev_mode+0xd4/0xf4)
+[<c02c9820>] (restore_fbdev_mode) from [<c02cb088>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
+[<c02cb088>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c02cb0f4>] (drm_fb_helper_set_par+0x2c/0x54)
+[<c02cb0f4>] (drm_fb_helper_set_par) from [<c0267960>] (fbcon_init+0x474/0x4dc)
+[<c0267960>] (fbcon_init) from [<c029e414>] (visual_init+0xb0/0xfc)
+[<c029e414>] (visual_init) from [<c02a03b8>] (do_bind_con_driver+0x134/0x354)
+[<c02a03b8>] (do_bind_con_driver) from [<c02a0748>] (do_take_over_console+0x170/0x1dc)
+[<c02a0748>] (do_take_over_console) from [<c0267a2c>] (do_fbcon_takeover+0x64/0xcc)
+[<c0267a2c>] (do_fbcon_takeover) from [<c003ec10>] (notifier_call_chain+0x44/0x84)
+[<c003ec10>] (notifier_call_chain) from [<c003ede4>] (__blocking_notifier_call_chain+0x48/0x60)
+[<c003ede4>] (__blocking_notifier_call_chain) from [<c003ee14>] (blocking_notifier_call_chain+0x18/0x20)
+[<c003ee14>] (blocking_notifier_call_chain) from [<c027191c>] (register_framebuffer+0x184/0x264)
+[<c027191c>] (register_framebuffer) from [<c02cb360>] (drm_fb_helper_initial_config+0x244/0x3e4)
+[<c02cb360>] (drm_fb_helper_initial_config) from [<c02ed43c>] (tegra_drm_fb_init+0x48/0xb4)
+[<c02ed43c>] (tegra_drm_fb_init) from [<c02eb6d4>] (tegra_drm_load+0x174/0x228)
+[<c02eb6d4>] (tegra_drm_load) from [<c02d36a0>] (drm_dev_register+0xa0/0xb4)
+[<c02d36a0>] (drm_dev_register) from [<c02eb22c>] (host1x_drm_probe+0x44/0xb0)
+[<c02eb22c>] (host1x_drm_probe) from [<c02b63a0>] (host1x_device_probe+0x1c/0x28)
+[<c02b63a0>] (host1x_device_probe) from [<c03030ec>] (driver_probe_device+0x1e0/0x2a0)
+[<c03030ec>] (driver_probe_device) from [<c030178c>] (bus_for_each_drv+0x44/0x8c)
+[<c030178c>] (bus_for_each_drv) from [<c0302e80>] (__device_attach+0xa0/0x104)
+[<c0302e80>] (__device_attach) from [<c03025fc>] (bus_probe_device+0x84/0x8c)
+[<c03025fc>] (bus_probe_device) from [<c0300b50>] (device_add+0x33c/0x524)
+[<c0300b50>] (device_add) from [<c02b65ac>] (host1x_subdev_register+0xb0/0xd4)
+[<c02b65ac>] (host1x_subdev_register) from [<c02b6e64>] (host1x_client_register+0xf4/0x11c)
+[<c02b6e64>] (host1x_client_register) from [<c02f4afc>] (tegra_hdmi_probe+0x170/0x250)
+[<c02f4afc>] (tegra_hdmi_probe) from [<c03045c4>] (platform_drv_probe+0x48/0xa4)
+[<c03045c4>] (platform_drv_probe) from [<c03030ec>] (driver_probe_device+0x1e0/0x2a0)
+[<c03030ec>] (driver_probe_device) from [<c030178c>] (bus_for_each_drv+0x44/0x8c)
+[<c030178c>] (bus_for_each_drv) from [<c0302e80>] (__device_attach+0xa0/0x104)
+[<c0302e80>] (__device_attach) from [<c03025fc>] (bus_probe_device+0x84/0x8c)
+[<c03025fc>] (bus_probe_device) from [<c03029d4>] (deferred_probe_work_func+0x5c/0x88)
+[<c03029d4>] (deferred_probe_work_func) from [<c0038cd0>] (process_one_work+0x120/0x328)
+[<c0038cd0>] (process_one_work) from [<c0038f34>] (worker_thread+0x28/0x490)
+[<c0038f34>] (worker_thread) from [<c003e02c>] (kthread+0xd8/0xf4)
+[<c003e02c>] (kthread) from [<c000f628>] (ret_from_fork+0x14/0x2c)
+---[ end trace 58de100d22bd7739 ]---
 Console: switching to colour frame buffer device 128x48
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
@@ -341,7 +341,7 @@
 tegra-snd-rt5640 sound: rt5640-aif1 <-> 70080400.i2s mapping ok
 input: NVIDIA Tegra Dalmore Headphones as /devices/soc0/sound/sound/card0/input0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-03-01 19:42:19 UTC (951939739)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-03-02 19:12:11 UTC (952024331)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -354,7 +354,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 324K (c0917000 - c0968000)
+Freeing unused kernel memory: 328K (c0916000 - c0968000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-08-21 03:23:45.236206267 -0700
+++ /tmp/foob	2015-08-21 03:23:45.240206264 -0700
@@ -42,18 +42,18 @@
 	 ###############################################
 	 3.2 MiB/s
 done
-Bytes transferred = 6400616 (61aa68 hex)
+Bytes transferred = 6404840 (61bae8 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *#####
-	 2.9 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 64210 (fad2 hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x61aa68 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x61bae8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012ad1
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc7-next-20150820-38036-gf0359ea (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Aug 20 03:47:55 PDT 2015
+Linux version 4.2.0-rc7-next-20150821-38393-g1ef981b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Fri Aug 21 03:17:49 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f6a640, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0f6c680, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998836K/2096128K available (8043K kernel code, 1017K rwdata, 3620K rodata, 1028K init, 340K bss, 31756K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1998828K/2096128K available (8051K kernel code, 1017K rwdata, 3620K rodata, 1028K init, 340K bss, 31764K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d6cdfc   (11668 kB)
-      .init : 0xc0d6d000 - 0xc0e6e000   (1028 kB)
-      .data : 0xc0e6e000 - 0xc0f6c620   (1018 kB)
-       .bss : 0xc0f6f000 - 0xc0fc42e8   ( 341 kB)
+      .text : 0xc0208000 - 0xc0d6eeb0   (11676 kB)
+      .init : 0xc0d6f000 - 0xc0e70000   (1028 kB)
+      .data : 0xc0e70000 - 0xc0f6e660   (1018 kB)
+       .bss : 0xc0f71000 - 0xc0fc62e8   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -257,7 +257,6 @@
 usbhid: USB HID core driver
 NET: Registered protocol family 10
 sit: IPv6 over IPv4 tunneling driver
-sit0 uses DEPRECATED zero tx_queue_len - convert driver to use IFF_NO_QUEUE instead.
 mmc0: MAN_BKOPS_EN bit is not set
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 SEM16G 14.6 GiB 
@@ -266,6 +265,7 @@
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
 Primary GPT is invalid, using alternate GPT.
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9
+sit0 uses DEPRECATED zero tx_queue_len - convert driver to use IFF_NO_QUEUE instead.
 NET: Registered protocol family 17
 can: controller area network core (rev 20120528 abi 9)
 NET: Registered protocol family 29
@@ -335,7 +335,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-02-01 01:26:58 UTC (949368418)
+as3722-rtc as3722-rtc: setting system clock to 2000-02-01 00:00:18 UTC (949363218)
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
@@ -352,7 +352,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1028K (c0d6d000 - c0e6e000)
+Freeing unused kernel memory: 1028K (c0d6f000 - c0e70000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2015-08-21 03:23:45.244206261 -0700
+++ /tmp/foob	2015-08-21 03:23:45.248206259 -0700
@@ -29,6 +29,7 @@
 Tegra124 (Jetson TK1) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug init=/bin/bash noinitrd'
 Tegra124 (Jetson TK1) # setenv ipaddr 192.168.100.102; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra124 (Jetson TK1) # tftpboot 0x81000000 zImage.tegra_defconfig
+Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'zImage.tegra_defconfig'.
@@ -39,20 +40,20 @@
 	 #################################################################
 	 #################################################################
 	 ###################
-	 3.3 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 5043848 (4cf688 hex)
+Bytes transferred = 5043840 (4cf680 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *#####
-	 2.6 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 64210 (fad2 hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4cf688 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4cf680 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012ad1
@@ -63,7 +64,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc7-next-20150820-38036-gf0359ea (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Aug 20 03:48:13 PDT 2015
+Linux version 4.2.0-rc7-next-20150821-38393-g1ef981b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Aug 21 03:18:05 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -84,7 +85,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002576K/2096128K available (6884K kernel code, 344K rwdata, 2388K rodata, 324K init, 363K bss, 28016K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 2002576K/2096128K available (6884K kernel code, 344K rwdata, 2384K rodata, 328K init, 363K bss, 28016K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,8 +93,8 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09166d8   (9274 kB)
-      .init : 0xc0917000 - 0xc0968000   ( 324 kB)
+      .text : 0xc0008000 - 0xc09156bc   (9270 kB)
+      .init : 0xc0916000 - 0xc0968000   ( 328 kB)
       .data : 0xc0968000 - 0xc09be080   ( 345 kB)
        .bss : 0xc09c1000 - 0xc0a1bd20   ( 364 kB)
 Preemptible hierarchical RCU implementation.
@@ -305,54 +306,54 @@
 WARNING: CPU: 0 PID: 6 at include/linux/ww_mutex.h:158 drm_atomic_check_only+0x4fc/0x574()
 DEBUG_LOCKS_WARN_ON(ctx->done_acquire)
 Modules linked in:
-CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.2.0-rc7-next-20150820-38036-gf0359ea #1
+CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.2.0-rc7-next-20150821-38393-g1ef981b #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0016898>] (unwind_backtrace) from [<c0012124>] (show_stack+0x10/0x14)
-[<c0012124>] (show_stack) from [<c067e850>] (dump_stack+0x84/0xd0)
-[<c067e850>] (dump_stack) from [<c00257f8>] (warn_slowpath_common+0x74/0xb0)
-[<c00257f8>] (warn_slowpath_common) from [<c00258c8>] (warn_slowpath_fmt+0x30/0x40)
-[<c00258c8>] (warn_slowpath_fmt) from [<c02e5e98>] (drm_atomic_check_only+0x4fc/0x574)
-[<c02e5e98>] (drm_atomic_check_only) from [<c02e5f84>] (drm_atomic_commit+0x10/0x64)
-[<c02e5f84>] (drm_atomic_commit) from [<c02c49fc>] (drm_atomic_helper_set_config+0x394/0x3ec)
-[<c02c49fc>] (drm_atomic_helper_set_config) from [<c02d612c>] (drm_mode_set_config_internal+0x58/0xd4)
-[<c02d612c>] (drm_mode_set_config_internal) from [<c02c7004>] (restore_fbdev_mode+0xd4/0xf4)
-[<c02c7004>] (restore_fbdev_mode) from [<c02c886c>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
-[<c02c886c>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c02c88d8>] (drm_fb_helper_set_par+0x2c/0x54)
-[<c02c88d8>] (drm_fb_helper_set_par) from [<c026504c>] (fbcon_init+0x474/0x4dc)
-[<c026504c>] (fbcon_init) from [<c029bc20>] (visual_init+0xb0/0xfc)
-[<c029bc20>] (visual_init) from [<c029dbc4>] (do_bind_con_driver+0x134/0x354)
-[<c029dbc4>] (do_bind_con_driver) from [<c029df54>] (do_take_over_console+0x170/0x1dc)
-[<c029df54>] (do_take_over_console) from [<c0265118>] (do_fbcon_takeover+0x64/0xcc)
-[<c0265118>] (do_fbcon_takeover) from [<c003e590>] (notifier_call_chain+0x44/0x84)
-[<c003e590>] (notifier_call_chain) from [<c003e764>] (__blocking_notifier_call_chain+0x48/0x60)
-[<c003e764>] (__blocking_notifier_call_chain) from [<c003e794>] (blocking_notifier_call_chain+0x18/0x20)
-[<c003e794>] (blocking_notifier_call_chain) from [<c026f008>] (register_framebuffer+0x184/0x264)
-[<c026f008>] (register_framebuffer) from [<c02c8b44>] (drm_fb_helper_initial_config+0x244/0x3e4)
-[<c02c8b44>] (drm_fb_helper_initial_config) from [<c02eac20>] (tegra_drm_fb_init+0x48/0xb4)
-[<c02eac20>] (tegra_drm_fb_init) from [<c02e8eb8>] (tegra_drm_load+0x174/0x228)
-[<c02e8eb8>] (tegra_drm_load) from [<c02d0e84>] (drm_dev_register+0xa0/0xb4)
-[<c02d0e84>] (drm_dev_register) from [<c02e8a10>] (host1x_drm_probe+0x44/0xb0)
-[<c02e8a10>] (host1x_drm_probe) from [<c02b3b84>] (host1x_device_probe+0x1c/0x28)
-[<c02b3b84>] (host1x_device_probe) from [<c03008d0>] (driver_probe_device+0x1e0/0x2a0)
-[<c03008d0>] (driver_probe_device) from [<c02fef70>] (bus_for_each_drv+0x44/0x8c)
-[<c02fef70>] (bus_for_each_drv) from [<c0300664>] (__device_attach+0xa0/0x104)
-[<c0300664>] (__device_attach) from [<c02ffde0>] (bus_probe_device+0x84/0x8c)
-[<c02ffde0>] (bus_probe_device) from [<c02fe334>] (device_add+0x33c/0x524)
-[<c02fe334>] (device_add) from [<c02b3d90>] (host1x_subdev_register+0xb0/0xd4)
-[<c02b3d90>] (host1x_subdev_register) from [<c02b4648>] (host1x_client_register+0xf4/0x11c)
-[<c02b4648>] (host1x_client_register) from [<c02f22e0>] (tegra_hdmi_probe+0x170/0x250)
-[<c02f22e0>] (tegra_hdmi_probe) from [<c0301da8>] (platform_drv_probe+0x48/0xa4)
-[<c0301da8>] (platform_drv_probe) from [<c03008d0>] (driver_probe_device+0x1e0/0x2a0)
-[<c03008d0>] (driver_probe_device) from [<c02fef70>] (bus_for_each_drv+0x44/0x8c)
-[<c02fef70>] (bus_for_each_drv) from [<c0300664>] (__device_attach+0xa0/0x104)
-[<c0300664>] (__device_attach) from [<c02ffde0>] (bus_probe_device+0x84/0x8c)
-[<c02ffde0>] (bus_probe_device) from [<c03001b8>] (deferred_probe_work_func+0x5c/0x88)
-[<c03001b8>] (deferred_probe_work_func) from [<c0038650>] (process_one_work+0x120/0x328)
-[<c0038650>] (process_one_work) from [<c00388b4>] (worker_thread+0x28/0x490)
-[<c00388b4>] (worker_thread) from [<c003d9ac>] (kthread+0xd8/0xf4)
-[<c003d9ac>] (kthread) from [<c000f628>] (ret_from_fork+0x14/0x2c)
----[ end trace b09848768513f3a5 ]---
+[<c0012124>] (show_stack) from [<c0211d60>] (dump_stack+0x84/0xd0)
+[<c0211d60>] (dump_stack) from [<c0025e78>] (warn_slowpath_common+0x74/0xb0)
+[<c0025e78>] (warn_slowpath_common) from [<c0025f48>] (warn_slowpath_fmt+0x30/0x40)
+[<c0025f48>] (warn_slowpath_fmt) from [<c02e86b4>] (drm_atomic_check_only+0x4fc/0x574)
+[<c02e86b4>] (drm_atomic_check_only) from [<c02e87a0>] (drm_atomic_commit+0x10/0x64)
+[<c02e87a0>] (drm_atomic_commit) from [<c02c7218>] (drm_atomic_helper_set_config+0x394/0x3ec)
+[<c02c7218>] (drm_atomic_helper_set_config) from [<c02d8948>] (drm_mode_set_config_internal+0x58/0xd4)
+[<c02d8948>] (drm_mode_set_config_internal) from [<c02c9820>] (restore_fbdev_mode+0xd4/0xf4)
+[<c02c9820>] (restore_fbdev_mode) from [<c02cb088>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
+[<c02cb088>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c02cb0f4>] (drm_fb_helper_set_par+0x2c/0x54)
+[<c02cb0f4>] (drm_fb_helper_set_par) from [<c0267960>] (fbcon_init+0x474/0x4dc)
+[<c0267960>] (fbcon_init) from [<c029e414>] (visual_init+0xb0/0xfc)
+[<c029e414>] (visual_init) from [<c02a03b8>] (do_bind_con_driver+0x134/0x354)
+[<c02a03b8>] (do_bind_con_driver) from [<c02a0748>] (do_take_over_console+0x170/0x1dc)
+[<c02a0748>] (do_take_over_console) from [<c0267a2c>] (do_fbcon_takeover+0x64/0xcc)
+[<c0267a2c>] (do_fbcon_takeover) from [<c003ec10>] (notifier_call_chain+0x44/0x84)
+[<c003ec10>] (notifier_call_chain) from [<c003ede4>] (__blocking_notifier_call_chain+0x48/0x60)
+[<c003ede4>] (__blocking_notifier_call_chain) from [<c003ee14>] (blocking_notifier_call_chain+0x18/0x20)
+[<c003ee14>] (blocking_notifier_call_chain) from [<c027191c>] (register_framebuffer+0x184/0x264)
+[<c027191c>] (register_framebuffer) from [<c02cb360>] (drm_fb_helper_initial_config+0x244/0x3e4)
+[<c02cb360>] (drm_fb_helper_initial_config) from [<c02ed43c>] (tegra_drm_fb_init+0x48/0xb4)
+[<c02ed43c>] (tegra_drm_fb_init) from [<c02eb6d4>] (tegra_drm_load+0x174/0x228)
+[<c02eb6d4>] (tegra_drm_load) from [<c02d36a0>] (drm_dev_register+0xa0/0xb4)
+[<c02d36a0>] (drm_dev_register) from [<c02eb22c>] (host1x_drm_probe+0x44/0xb0)
+[<c02eb22c>] (host1x_drm_probe) from [<c02b63a0>] (host1x_device_probe+0x1c/0x28)
+[<c02b63a0>] (host1x_device_probe) from [<c03030ec>] (driver_probe_device+0x1e0/0x2a0)
+[<c03030ec>] (driver_probe_device) from [<c030178c>] (bus_for_each_drv+0x44/0x8c)
+[<c030178c>] (bus_for_each_drv) from [<c0302e80>] (__device_attach+0xa0/0x104)
+[<c0302e80>] (__device_attach) from [<c03025fc>] (bus_probe_device+0x84/0x8c)
+[<c03025fc>] (bus_probe_device) from [<c0300b50>] (device_add+0x33c/0x524)
+[<c0300b50>] (device_add) from [<c02b65ac>] (host1x_subdev_register+0xb0/0xd4)
+[<c02b65ac>] (host1x_subdev_register) from [<c02b6e64>] (host1x_client_register+0xf4/0x11c)
+[<c02b6e64>] (host1x_client_register) from [<c02f4afc>] (tegra_hdmi_probe+0x170/0x250)
+[<c02f4afc>] (tegra_hdmi_probe) from [<c03045c4>] (platform_drv_probe+0x48/0xa4)
+[<c03045c4>] (platform_drv_probe) from [<c03030ec>] (driver_probe_device+0x1e0/0x2a0)
+[<c03030ec>] (driver_probe_device) from [<c030178c>] (bus_for_each_drv+0x44/0x8c)
+[<c030178c>] (bus_for_each_drv) from [<c0302e80>] (__device_attach+0xa0/0x104)
+[<c0302e80>] (__device_attach) from [<c03025fc>] (bus_probe_device+0x84/0x8c)
+[<c03025fc>] (bus_probe_device) from [<c03029d4>] (deferred_probe_work_func+0x5c/0x88)
+[<c03029d4>] (deferred_probe_work_func) from [<c0038cd0>] (process_one_work+0x120/0x328)
+[<c0038cd0>] (process_one_work) from [<c0038f34>] (worker_thread+0x28/0x490)
+[<c0038f34>] (worker_thread) from [<c003e02c>] (kthread+0xd8/0xf4)
+[<c003e02c>] (kthread) from [<c000f628>] (ret_from_fork+0x14/0x2c)
+---[ end trace 12ed84c0511dd844 ]---
 Console: switching to colour frame buffer device 512x135
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
@@ -387,7 +388,7 @@
 tegra-snd-rt5640 sound: rt5640-aif1 <-> 70301100.i2s mapping ok
 input: NVIDIA Tegra Jetson TK1 Headphones as /devices/soc0/sound/sound/card1/input1
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-02-01 01:27:33 UTC (949368453)
+as3722-rtc as3722-rtc: setting system clock to 2000-02-01 00:00:56 UTC (949363256)
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
@@ -406,7 +407,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 324K (c0917000 - c0968000)
+Freeing unused kernel memory: 328K (c0916000 - c0968000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-08-21 03:23:45.252206256 -0700
+++ /tmp/foob	2015-08-21 03:23:45.252206256 -0700
@@ -48,18 +48,18 @@
 	 ###############################################
 	 4.1 MiB/s
 done
-Bytes transferred = 6400616 (61aa68 hex)
+Bytes transferred = 6404840 (61bae8 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 4 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21202 (52d2 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x61aa68 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x61bae8 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102d1
@@ -67,7 +67,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc7-next-20150820-38036-gf0359ea (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Aug 20 03:47:55 PDT 2015
+Linux version 4.2.0-rc7-next-20150821-38393-g1ef981b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Fri Aug 21 03:17:49 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -75,7 +75,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0f6a640, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c0f6c680, node_mem_map eeff8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -88,7 +88,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 959752K/1048576K available (8043K kernel code, 1017K rwdata, 3620K rodata, 1028K init, 340K bss, 23288K reserved, 65536K cma-reserved, 204800K highmem)
+Memory: 959744K/1048576K available (8051K kernel code, 1017K rwdata, 3620K rodata, 1028K init, 340K bss, 23296K reserved, 65536K cma-reserved, 204800K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -96,10 +96,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d6cdfc   (11668 kB)
-      .init : 0xc0d6d000 - 0xc0e6e000   (1028 kB)
-      .data : 0xc0e6e000 - 0xc0f6c620   (1018 kB)
-       .bss : 0xc0f6f000 - 0xc0fc42e8   ( 341 kB)
+      .text : 0xc0208000 - 0xc0d6eeb0   (11676 kB)
+      .init : 0xc0d6f000 - 0xc0e70000   (1028 kB)
+      .data : 0xc0e70000 - 0xc0f6e660   (1018 kB)
+       .bss : 0xc0f71000 - 0xc0fc62e8   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -334,7 +334,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on c8000600.sdhci [c8000600.sdhci] using ADMA
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-26 20:48:07 UTC (1419626887)
+rtc-em3027 2-0056: setting system clock to 2014-12-26 20:49:19 UTC (1419626959)
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
@@ -345,8 +345,8 @@
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
 devtmpfs: mounted
-Freeing unused kernel memory: 1028K (c0d6d000 - c0e6e000)
+Freeing unused kernel memory: 1028K (c0d6f000 - c0e70000)
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# 
\ No newline at end of file
+root@(none):/# [    3.290593] asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2015-08-21 03:23:45.260206251 -0700
+++ /tmp/foob	2015-08-21 03:23:45.260206251 -0700
@@ -47,7 +47,7 @@
 	 ###################
 	 4.1 MiB/s
 done
-Bytes transferred = 5043848 (4cf688 hex)
+Bytes transferred = 5043840 (4cf680 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21202 (52d2 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x4cf688 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x4cf680 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102d1
@@ -69,7 +69,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc7-next-20150820-38036-gf0359ea (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Aug 20 03:48:13 PDT 2015
+Linux version 4.2.0-rc7-next-20150821-38393-g1ef981b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Aug 21 03:18:05 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -91,7 +91,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 963492K/1048576K available (6884K kernel code, 344K rwdata, 2388K rodata, 324K init, 363K bss, 19548K reserved, 65536K cma-reserved, 204800K highmem)
+Memory: 963492K/1048576K available (6884K kernel code, 344K rwdata, 2384K rodata, 328K init, 363K bss, 19548K reserved, 65536K cma-reserved, 204800K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -99,8 +99,8 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09166d8   (9274 kB)
-      .init : 0xc0917000 - 0xc0968000   ( 324 kB)
+      .text : 0xc0008000 - 0xc09156bc   (9270 kB)
+      .init : 0xc0916000 - 0xc0968000   ( 328 kB)
       .data : 0xc0968000 - 0xc09be080   ( 345 kB)
        .bss : 0xc09c1000 - 0xc0a1bd20   ( 364 kB)
 Preemptible hierarchical RCU implementation.
@@ -240,54 +240,54 @@
 WARNING: CPU: 0 PID: 1 at include/linux/ww_mutex.h:158 drm_atomic_check_only+0x4fc/0x574()
 DEBUG_LOCKS_WARN_ON(ctx->done_acquire)
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.2.0-rc7-next-20150820-38036-gf0359ea #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.2.0-rc7-next-20150821-38393-g1ef981b #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c0016898>] (unwind_backtrace) from [<c0012124>] (show_stack+0x10/0x14)
-[<c0012124>] (show_stack) from [<c067e850>] (dump_stack+0x84/0xd0)
-[<c067e850>] (dump_stack) from [<c00257f8>] (warn_slowpath_common+0x74/0xb0)
-[<c00257f8>] (warn_slowpath_common) from [<c00258c8>] (warn_slowpath_fmt+0x30/0x40)
-[<c00258c8>] (warn_slowpath_fmt) from [<c02e5e98>] (drm_atomic_check_only+0x4fc/0x574)
-[<c02e5e98>] (drm_atomic_check_only) from [<c02e5f84>] (drm_atomic_commit+0x10/0x64)
-[<c02e5f84>] (drm_atomic_commit) from [<c02c49fc>] (drm_atomic_helper_set_config+0x394/0x3ec)
-[<c02c49fc>] (drm_atomic_helper_set_config) from [<c02d612c>] (drm_mode_set_config_internal+0x58/0xd4)
-[<c02d612c>] (drm_mode_set_config_internal) from [<c02c7004>] (restore_fbdev_mode+0xd4/0xf4)
-[<c02c7004>] (restore_fbdev_mode) from [<c02c886c>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
-[<c02c886c>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c02c88d8>] (drm_fb_helper_set_par+0x2c/0x54)
-[<c02c88d8>] (drm_fb_helper_set_par) from [<c026504c>] (fbcon_init+0x474/0x4dc)
-[<c026504c>] (fbcon_init) from [<c029bc20>] (visual_init+0xb0/0xfc)
-[<c029bc20>] (visual_init) from [<c029dbc4>] (do_bind_con_driver+0x134/0x354)
-[<c029dbc4>] (do_bind_con_driver) from [<c029df54>] (do_take_over_console+0x170/0x1dc)
-[<c029df54>] (do_take_over_console) from [<c0265118>] (do_fbcon_takeover+0x64/0xcc)
-[<c0265118>] (do_fbcon_takeover) from [<c003e590>] (notifier_call_chain+0x44/0x84)
-[<c003e590>] (notifier_call_chain) from [<c003e764>] (__blocking_notifier_call_chain+0x48/0x60)
-[<c003e764>] (__blocking_notifier_call_chain) from [<c003e794>] (blocking_notifier_call_chain+0x18/0x20)
-[<c003e794>] (blocking_notifier_call_chain) from [<c026f008>] (register_framebuffer+0x184/0x264)
-[<c026f008>] (register_framebuffer) from [<c02c8b44>] (drm_fb_helper_initial_config+0x244/0x3e4)
-[<c02c8b44>] (drm_fb_helper_initial_config) from [<c02eac20>] (tegra_drm_fb_init+0x48/0xb4)
-[<c02eac20>] (tegra_drm_fb_init) from [<c02e8eb8>] (tegra_drm_load+0x174/0x228)
-[<c02e8eb8>] (tegra_drm_load) from [<c02d0e84>] (drm_dev_register+0xa0/0xb4)
-[<c02d0e84>] (drm_dev_register) from [<c02e8a10>] (host1x_drm_probe+0x44/0xb0)
-[<c02e8a10>] (host1x_drm_probe) from [<c02b3b84>] (host1x_device_probe+0x1c/0x28)
-[<c02b3b84>] (host1x_device_probe) from [<c03008d0>] (driver_probe_device+0x1e0/0x2a0)
-[<c03008d0>] (driver_probe_device) from [<c02fef70>] (bus_for_each_drv+0x44/0x8c)
-[<c02fef70>] (bus_for_each_drv) from [<c0300664>] (__device_attach+0xa0/0x104)
-[<c0300664>] (__device_attach) from [<c02ffde0>] (bus_probe_device+0x84/0x8c)
-[<c02ffde0>] (bus_probe_device) from [<c02fe334>] (device_add+0x33c/0x524)
-[<c02fe334>] (device_add) from [<c02b3d90>] (host1x_subdev_register+0xb0/0xd4)
-[<c02b3d90>] (host1x_subdev_register) from [<c02b4648>] (host1x_client_register+0xf4/0x11c)
-[<c02b4648>] (host1x_client_register) from [<c02fa080>] (gr3d_probe+0x100/0x21c)
-[<c02fa080>] (gr3d_probe) from [<c0301da8>] (platform_drv_probe+0x48/0xa4)
-[<c0301da8>] (platform_drv_probe) from [<c03008d0>] (driver_probe_device+0x1e0/0x2a0)
-[<c03008d0>] (driver_probe_device) from [<c0300a1c>] (__driver_attach+0x8c/0x90)
-[<c0300a1c>] (__driver_attach) from [<c02ff00c>] (bus_for_each_dev+0x54/0x88)
-[<c02ff00c>] (bus_for_each_dev) from [<c02fffc0>] (bus_add_driver+0xe4/0x1f0)
-[<c02fffc0>] (bus_add_driver) from [<c03011e8>] (driver_register+0x78/0xf4)
-[<c03011e8>] (driver_register) from [<c093383c>] (host1x_drm_init+0x9c/0x10c)
-[<c093383c>] (host1x_drm_init) from [<c00096e4>] (do_one_initcall+0x80/0x1d0)
-[<c00096e4>] (do_one_initcall) from [<c0917db0>] (kernel_init_freeable+0x118/0x1e4)
-[<c0917db0>] (kernel_init_freeable) from [<c067a870>] (kernel_init+0x8/0xec)
-[<c067a870>] (kernel_init) from [<c000f628>] (ret_from_fork+0x14/0x2c)
----[ end trace 0f83b3a9e928cb4c ]---
+[<c0012124>] (show_stack) from [<c0211d60>] (dump_stack+0x84/0xd0)
+[<c0211d60>] (dump_stack) from [<c0025e78>] (warn_slowpath_common+0x74/0xb0)
+[<c0025e78>] (warn_slowpath_common) from [<c0025f48>] (warn_slowpath_fmt+0x30/0x40)
+[<c0025f48>] (warn_slowpath_fmt) from [<c02e86b4>] (drm_atomic_check_only+0x4fc/0x574)
+[<c02e86b4>] (drm_atomic_check_only) from [<c02e87a0>] (drm_atomic_commit+0x10/0x64)
+[<c02e87a0>] (drm_atomic_commit) from [<c02c7218>] (drm_atomic_helper_set_config+0x394/0x3ec)
+[<c02c7218>] (drm_atomic_helper_set_config) from [<c02d8948>] (drm_mode_set_config_internal+0x58/0xd4)
+[<c02d8948>] (drm_mode_set_config_internal) from [<c02c9820>] (restore_fbdev_mode+0xd4/0xf4)
+[<c02c9820>] (restore_fbdev_mode) from [<c02cb088>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
+[<c02cb088>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c02cb0f4>] (drm_fb_helper_set_par+0x2c/0x54)
+[<c02cb0f4>] (drm_fb_helper_set_par) from [<c0267960>] (fbcon_init+0x474/0x4dc)
+[<c0267960>] (fbcon_init) from [<c029e414>] (visual_init+0xb0/0xfc)
+[<c029e414>] (visual_init) from [<c02a03b8>] (do_bind_con_driver+0x134/0x354)
+[<c02a03b8>] (do_bind_con_driver) from [<c02a0748>] (do_take_over_console+0x170/0x1dc)
+[<c02a0748>] (do_take_over_console) from [<c0267a2c>] (do_fbcon_takeover+0x64/0xcc)
+[<c0267a2c>] (do_fbcon_takeover) from [<c003ec10>] (notifier_call_chain+0x44/0x84)
+[<c003ec10>] (notifier_call_chain) from [<c003ede4>] (__blocking_notifier_call_chain+0x48/0x60)
+[<c003ede4>] (__blocking_notifier_call_chain) from [<c003ee14>] (blocking_notifier_call_chain+0x18/0x20)
+[<c003ee14>] (blocking_notifier_call_chain) from [<c027191c>] (register_framebuffer+0x184/0x264)
+[<c027191c>] (register_framebuffer) from [<c02cb360>] (drm_fb_helper_initial_config+0x244/0x3e4)
+[<c02cb360>] (drm_fb_helper_initial_config) from [<c02ed43c>] (tegra_drm_fb_init+0x48/0xb4)
+[<c02ed43c>] (tegra_drm_fb_init) from [<c02eb6d4>] (tegra_drm_load+0x174/0x228)
+[<c02eb6d4>] (tegra_drm_load) from [<c02d36a0>] (drm_dev_register+0xa0/0xb4)
+[<c02d36a0>] (drm_dev_register) from [<c02eb22c>] (host1x_drm_probe+0x44/0xb0)
+[<c02eb22c>] (host1x_drm_probe) from [<c02b63a0>] (host1x_device_probe+0x1c/0x28)
+[<c02b63a0>] (host1x_device_probe) from [<c03030ec>] (driver_probe_device+0x1e0/0x2a0)
+[<c03030ec>] (driver_probe_device) from [<c030178c>] (bus_for_each_drv+0x44/0x8c)
+[<c030178c>] (bus_for_each_drv) from [<c0302e80>] (__device_attach+0xa0/0x104)
+[<c0302e80>] (__device_attach) from [<c03025fc>] (bus_probe_device+0x84/0x8c)
+[<c03025fc>] (bus_probe_device) from [<c0300b50>] (device_add+0x33c/0x524)
+[<c0300b50>] (device_add) from [<c02b65ac>] (host1x_subdev_register+0xb0/0xd4)
+[<c02b65ac>] (host1x_subdev_register) from [<c02b6e64>] (host1x_client_register+0xf4/0x11c)
+[<c02b6e64>] (host1x_client_register) from [<c02fc89c>] (gr3d_probe+0x100/0x21c)
+[<c02fc89c>] (gr3d_probe) from [<c03045c4>] (platform_drv_probe+0x48/0xa4)
+[<c03045c4>] (platform_drv_probe) from [<c03030ec>] (driver_probe_device+0x1e0/0x2a0)
+[<c03030ec>] (driver_probe_device) from [<c0303238>] (__driver_attach+0x8c/0x90)
+[<c0303238>] (__driver_attach) from [<c0301828>] (bus_for_each_dev+0x54/0x88)
+[<c0301828>] (bus_for_each_dev) from [<c03027dc>] (bus_add_driver+0xe4/0x1f0)
+[<c03027dc>] (bus_add_driver) from [<c0303a04>] (driver_register+0x78/0xf4)
+[<c0303a04>] (driver_register) from [<c093283c>] (host1x_drm_init+0x9c/0x10c)
+[<c093283c>] (host1x_drm_init) from [<c00096e4>] (do_one_initcall+0x80/0x1d0)
+[<c00096e4>] (do_one_initcall) from [<c0916db0>] (kernel_init_freeable+0x118/0x1e4)
+[<c0916db0>] (kernel_init_freeable) from [<c067f0dc>] (kernel_init+0x8/0xec)
+[<c067f0dc>] (kernel_init) from [<c000f628>] (ret_from_fork+0x14/0x2c)
+---[ end trace ab781d7c7f0e1522 ]---
 Console: switching to colour frame buffer device 128x48
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
@@ -398,20 +398,20 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on c8000600.sdhci [c8000600.sdhci] using ADMA
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-26 20:48:26 UTC (1419626906)
+rtc-em3027 2-0056: setting system clock to 2014-12-26 20:49:38 UTC (1419626978)
 ALSA device list:
   #0: tegra-trimslice
 EXT2-fs (mmcblk0p1): error: couldn't mount because of unsupported optional features (4)
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
- mmcblk1: p1 p2
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
+ mmcblk1: p1 p2
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 devtmpfs: mounted
-Freeing unused kernel memory: 324K (c0917000 - c0968000)
+Freeing unused kernel memory: 328K (c0916000 - c0968000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-08-21 03:23:45.264206248 -0700
+++ /tmp/foob	2015-08-21 03:23:45.268206245 -0700
@@ -40,18 +40,18 @@
 	 ###############################################
 	 3.3 MiB/s
 done
-Bytes transferred = 6400616 (61aa68 hex)
+Bytes transferred = 6404840 (61bae8 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 51539 (c953 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x61aa68 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x61bae8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff952 ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc7-next-20150820-38036-gf0359ea (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Aug 20 03:47:55 PDT 2015
+Linux version 4.2.0-rc7-next-20150821-38393-g1ef981b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Fri Aug 21 03:17:49 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -67,7 +67,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f6a640, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0f6c680, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -80,7 +80,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998864K/2096128K available (8043K kernel code, 1017K rwdata, 3620K rodata, 1028K init, 340K bss, 31728K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1998856K/2096128K available (8051K kernel code, 1017K rwdata, 3620K rodata, 1028K init, 340K bss, 31736K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,10 +88,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d6cdfc   (11668 kB)
-      .init : 0xc0d6d000 - 0xc0e6e000   (1028 kB)
-      .data : 0xc0e6e000 - 0xc0f6c620   (1018 kB)
-       .bss : 0xc0f6f000 - 0xc0fc42e8   ( 341 kB)
+      .text : 0xc0208000 - 0xc0d6eeb0   (11676 kB)
+      .init : 0xc0d6f000 - 0xc0e70000   (1028 kB)
+      .data : 0xc0e70000 - 0xc0f6e660   (1018 kB)
+       .bss : 0xc0f71000 - 0xc0fc62e8   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -357,7 +357,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1028K (c0d6d000 - c0e6e000)
+Freeing unused kernel memory: 1028K (c0d6f000 - c0e70000)
 usb 2-1: new high-speed USB device number 2 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2015-08-21 03:23:45.272206242 -0700
+++ /tmp/foob	2015-08-21 03:23:45.276206239 -0700
@@ -39,18 +39,18 @@
 	 ###################
 	 3.4 MiB/s
 done
-Bytes transferred = 5043848 (4cf688 hex)
+Bytes transferred = 5043840 (4cf680 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.3 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 51539 (c953 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4cf688 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4cf680 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff952 ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc7-next-20150820-38036-gf0359ea (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Aug 20 03:48:13 PDT 2015
+Linux version 4.2.0-rc7-next-20150821-38393-g1ef981b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Aug 21 03:18:05 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002604K/2096128K available (6884K kernel code, 344K rwdata, 2388K rodata, 324K init, 363K bss, 27988K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 2002604K/2096128K available (6884K kernel code, 344K rwdata, 2384K rodata, 328K init, 363K bss, 27988K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,8 +90,8 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09166d8   (9274 kB)
-      .init : 0xc0917000 - 0xc0968000   ( 324 kB)
+      .text : 0xc0008000 - 0xc09156bc   (9270 kB)
+      .init : 0xc0916000 - 0xc0968000   ( 328 kB)
       .data : 0xc0968000 - 0xc09be080   ( 345 kB)
        .bss : 0xc09c1000 - 0xc0a1bd20   ( 364 kB)
 Preemptible hierarchical RCU implementation.
@@ -260,22 +260,22 @@
 can: netlink gateway (rev 20130117) max_hops=1
 Bluetooth: RFCOMM socket layer initialized
 Bluetooth: RFCOMM ver 1.11
-Bluetooth: BNEP (Ethernet Emulation) ver 1.3
 mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 SEM16G 14.8 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 1.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 1.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 128 KiB
+Bluetooth: BNEP (Ethernet Emulation) ver 1.3
 Bluetooth: BNEP socket layer initialized
+Primary GPT is invalid, using alternate GPT.
+ mmcblk0: p1 p2 p3 p4 p5
 Bluetooth: HIDP (Human Interface Emulation) ver 1.2
 Bluetooth: HIDP socket layer initialized
-mmcblk0: mmc0:0001 SEM16G 14.8 GiB 
 Registering SWP/SWPB emulation handler
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 1.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 1.00 MiB
 tegra-pcie 3000.pcie-controller: 2x3 configuration
 sys_3v3,vdd_3v3_alw: supplied by vdd_5v_in
 sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
 tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 128 KiB
-Primary GPT is invalid, using alternate GPT.
- mmcblk0: p1 p2 p3 p4 p5
 tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -322,57 +322,57 @@
 drm drm: No connectors reported connected with modes
 [drm] Cannot find any crtc or sizes - going 1024x768
 ------------[ cut here ]------------
-WARNING: CPU: 1 PID: 76 at include/linux/ww_mutex.h:158 drm_atomic_check_only+0x4fc/0x574()
+WARNING: CPU: 2 PID: 6 at include/linux/ww_mutex.h:158 drm_atomic_check_only+0x4fc/0x574()
 DEBUG_LOCKS_WARN_ON(ctx->done_acquire)
 Modules linked in:
-CPU: 1 PID: 76 Comm: kworker/u8:1 Not tainted 4.2.0-rc7-next-20150820-38036-gf0359ea #1
+CPU: 2 PID: 6 Comm: kworker/u8:0 Not tainted 4.2.0-rc7-next-20150821-38393-g1ef981b #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0016898>] (unwind_backtrace) from [<c0012124>] (show_stack+0x10/0x14)
-[<c0012124>] (show_stack) from [<c067e850>] (dump_stack+0x84/0xd0)
-[<c067e850>] (dump_stack) from [<c00257f8>] (warn_slowpath_common+0x74/0xb0)
-[<c00257f8>] (warn_slowpath_common) from [<c00258c8>] (warn_slowpath_fmt+0x30/0x40)
-[<c00258c8>] (warn_slowpath_fmt) from [<c02e5e98>] (drm_atomic_check_only+0x4fc/0x574)
-[<c02e5e98>] (drm_atomic_check_only) from [<c02e5f84>] (drm_atomic_commit+0x10/0x64)
-[<c02e5f84>] (drm_atomic_commit) from [<c02c49fc>] (drm_atomic_helper_set_config+0x394/0x3ec)
-[<c02c49fc>] (drm_atomic_helper_set_config) from [<c02d612c>] (drm_mode_set_config_internal+0x58/0xd4)
-[<c02d612c>] (drm_mode_set_config_internal) from [<c02c7004>] (restore_fbdev_mode+0xd4/0xf4)
-[<c02c7004>] (restore_fbdev_mode) from [<c02c886c>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
-[<c02c886c>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c02c88d8>] (drm_fb_helper_set_par+0x2c/0x54)
-[<c02c88d8>] (drm_fb_helper_set_par) from [<c026504c>] (fbcon_init+0x474/0x4dc)
-[<c026504c>] (fbcon_init) from [<c029bc20>] (visual_init+0xb0/0xfc)
-[<c029bc20>] (visual_init) from [<c029dbc4>] (do_bind_con_driver+0x134/0x354)
-[<c029dbc4>] (do_bind_con_driver) from [<c029df54>] (do_take_over_console+0x170/0x1dc)
-[<c029df54>] (do_take_over_console) from [<c0265118>] (do_fbcon_takeover+0x64/0xcc)
-[<c0265118>] (do_fbcon_takeover) from [<c003e590>] (notifier_call_chain+0x44/0x84)
-[<c003e590>] (notifier_call_chain) from [<c003e764>] (__blocking_notifier_call_chain+0x48/0x60)
-[<c003e764>] (__blocking_notifier_call_chain) from [<c003e794>] (blocking_notifier_call_chain+0x18/0x20)
-[<c003e794>] (blocking_notifier_call_chain) from [<c026f008>] (register_framebuffer+0x184/0x264)
-[<c026f008>] (register_framebuffer) from [<c02c8b44>] (drm_fb_helper_initial_config+0x244/0x3e4)
-[<c02c8b44>] (drm_fb_helper_initial_config) from [<c02eac20>] (tegra_drm_fb_init+0x48/0xb4)
-[<c02eac20>] (tegra_drm_fb_init) from [<c02e8eb8>] (tegra_drm_load+0x174/0x228)
-[<c02e8eb8>] (tegra_drm_load) from [<c02d0e84>] (drm_dev_register+0xa0/0xb4)
-[<c02d0e84>] (drm_dev_register) from [<c02e8a10>] (host1x_drm_probe+0x44/0xb0)
-[<c02e8a10>] (host1x_drm_probe) from [<c02b3b84>] (host1x_device_probe+0x1c/0x28)
-[<c02b3b84>] (host1x_device_probe) from [<c03008d0>] (driver_probe_device+0x1e0/0x2a0)
-[<c03008d0>] (driver_probe_device) from [<c02fef70>] (bus_for_each_drv+0x44/0x8c)
-[<c02fef70>] (bus_for_each_drv) from [<c0300664>] (__device_attach+0xa0/0x104)
-[<c0300664>] (__device_attach) from [<c02ffde0>] (bus_probe_device+0x84/0x8c)
-[<c02ffde0>] (bus_probe_device) from [<c02fe334>] (device_add+0x33c/0x524)
-[<c02fe334>] (device_add) from [<c02b3d90>] (host1x_subdev_register+0xb0/0xd4)
-[<c02b3d90>] (host1x_subdev_register) from [<c02b4648>] (host1x_client_register+0xf4/0x11c)
-[<c02b4648>] (host1x_client_register) from [<c02f22e0>] (tegra_hdmi_probe+0x170/0x250)
-[<c02f22e0>] (tegra_hdmi_probe) from [<c0301da8>] (platform_drv_probe+0x48/0xa4)
-[<c0301da8>] (platform_drv_probe) from [<c03008d0>] (driver_probe_device+0x1e0/0x2a0)
-[<c03008d0>] (driver_probe_device) from [<c02fef70>] (bus_for_each_drv+0x44/0x8c)
-[<c02fef70>] (bus_for_each_drv) from [<c0300664>] (__device_attach+0xa0/0x104)
-[<c0300664>] (__device_attach) from [<c02ffde0>] (bus_probe_device+0x84/0x8c)
-[<c02ffde0>] (bus_probe_device) from [<c03001b8>] (deferred_probe_work_func+0x5c/0x88)
-[<c03001b8>] (deferred_probe_work_func) from [<c0038650>] (process_one_work+0x120/0x328)
-[<c0038650>] (process_one_work) from [<c00388b4>] (worker_thread+0x28/0x490)
-[<c00388b4>] (worker_thread) from [<c003d9ac>] (kthread+0xd8/0xf4)
-[<c003d9ac>] (kthread) from [<c000f628>] (ret_from_fork+0x14/0x2c)
----[ end trace af032e17ce34dd94 ]---
+[<c0012124>] (show_stack) from [<c0211d60>] (dump_stack+0x84/0xd0)
+[<c0211d60>] (dump_stack) from [<c0025e78>] (warn_slowpath_common+0x74/0xb0)
+[<c0025e78>] (warn_slowpath_common) from [<c0025f48>] (warn_slowpath_fmt+0x30/0x40)
+[<c0025f48>] (warn_slowpath_fmt) from [<c02e86b4>] (drm_atomic_check_only+0x4fc/0x574)
+[<c02e86b4>] (drm_atomic_check_only) from [<c02e87a0>] (drm_atomic_commit+0x10/0x64)
+[<c02e87a0>] (drm_atomic_commit) from [<c02c7218>] (drm_atomic_helper_set_config+0x394/0x3ec)
+[<c02c7218>] (drm_atomic_helper_set_config) from [<c02d8948>] (drm_mode_set_config_internal+0x58/0xd4)
+[<c02d8948>] (drm_mode_set_config_internal) from [<c02c9820>] (restore_fbdev_mode+0xd4/0xf4)
+[<c02c9820>] (restore_fbdev_mode) from [<c02cb088>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
+[<c02cb088>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c02cb0f4>] (drm_fb_helper_set_par+0x2c/0x54)
+[<c02cb0f4>] (drm_fb_helper_set_par) from [<c0267960>] (fbcon_init+0x474/0x4dc)
+[<c0267960>] (fbcon_init) from [<c029e414>] (visual_init+0xb0/0xfc)
+[<c029e414>] (visual_init) from [<c02a03b8>] (do_bind_con_driver+0x134/0x354)
+[<c02a03b8>] (do_bind_con_driver) from [<c02a0748>] (do_take_over_console+0x170/0x1dc)
+[<c02a0748>] (do_take_over_console) from [<c0267a2c>] (do_fbcon_takeover+0x64/0xcc)
+[<c0267a2c>] (do_fbcon_takeover) from [<c003ec10>] (notifier_call_chain+0x44/0x84)
+[<c003ec10>] (notifier_call_chain) from [<c003ede4>] (__blocking_notifier_call_chain+0x48/0x60)
+[<c003ede4>] (__blocking_notifier_call_chain) from [<c003ee14>] (blocking_notifier_call_chain+0x18/0x20)
+[<c003ee14>] (blocking_notifier_call_chain) from [<c027191c>] (register_framebuffer+0x184/0x264)
+[<c027191c>] (register_framebuffer) from [<c02cb360>] (drm_fb_helper_initial_config+0x244/0x3e4)
+[<c02cb360>] (drm_fb_helper_initial_config) from [<c02ed43c>] (tegra_drm_fb_init+0x48/0xb4)
+[<c02ed43c>] (tegra_drm_fb_init) from [<c02eb6d4>] (tegra_drm_load+0x174/0x228)
+[<c02eb6d4>] (tegra_drm_load) from [<c02d36a0>] (drm_dev_register+0xa0/0xb4)
+[<c02d36a0>] (drm_dev_register) from [<c02eb22c>] (host1x_drm_probe+0x44/0xb0)
+[<c02eb22c>] (host1x_drm_probe) from [<c02b63a0>] (host1x_device_probe+0x1c/0x28)
+[<c02b63a0>] (host1x_device_probe) from [<c03030ec>] (driver_probe_device+0x1e0/0x2a0)
+[<c03030ec>] (driver_probe_device) from [<c030178c>] (bus_for_each_drv+0x44/0x8c)
+[<c030178c>] (bus_for_each_drv) from [<c0302e80>] (__device_attach+0xa0/0x104)
+[<c0302e80>] (__device_attach) from [<c03025fc>] (bus_probe_device+0x84/0x8c)
+[<c03025fc>] (bus_probe_device) from [<c0300b50>] (device_add+0x33c/0x524)
+[<c0300b50>] (device_add) from [<c02b65ac>] (host1x_subdev_register+0xb0/0xd4)
+[<c02b65ac>] (host1x_subdev_register) from [<c02b6e64>] (host1x_client_register+0xf4/0x11c)
+[<c02b6e64>] (host1x_client_register) from [<c02f4afc>] (tegra_hdmi_probe+0x170/0x250)
+[<c02f4afc>] (tegra_hdmi_probe) from [<c03045c4>] (platform_drv_probe+0x48/0xa4)
+[<c03045c4>] (platform_drv_probe) from [<c03030ec>] (driver_probe_device+0x1e0/0x2a0)
+[<c03030ec>] (driver_probe_device) from [<c030178c>] (bus_for_each_drv+0x44/0x8c)
+[<c030178c>] (bus_for_each_drv) from [<c0302e80>] (__device_attach+0xa0/0x104)
+[<c0302e80>] (__device_attach) from [<c03025fc>] (bus_probe_device+0x84/0x8c)
+[<c03025fc>] (bus_probe_device) from [<c03029d4>] (deferred_probe_work_func+0x5c/0x88)
+[<c03029d4>] (deferred_probe_work_func) from [<c0038cd0>] (process_one_work+0x120/0x328)
+[<c0038cd0>] (process_one_work) from [<c0038f34>] (worker_thread+0x28/0x490)
+[<c0038f34>] (worker_thread) from [<c003e02c>] (kthread+0xd8/0xf4)
+[<c003e02c>] (kthread) from [<c000f628>] (ret_from_fork+0x14/0x2c)
+---[ end trace 0992cce3556a6eaf ]---
 Console: switching to colour frame buffer device 128x48
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
@@ -412,7 +412,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 324K (c0917000 - c0968000)
+Freeing unused kernel memory: 328K (c0916000 - c0968000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
