Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2017-09-26 01:04:58.517727393 -0700
+++ /tmp/foob	2017-09-26 01:04:58.517727393 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-next-20170922-201396-g7352731 (jonathanh@sellafield) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Thu Sep 21 22:23:12 PDT 2017
+Linux version 4.14.0-rc2-next-20170926-320346-gd35adcbe86c1 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Mon Sep 25 22:42:39 PDT 2017
 Boot CPU: AArch64 Processor [411fd070]
 Machine model: linux,dummy-virt
 Memory limited to 1024MB
@@ -21,7 +21,7 @@
 psci: Using standard PSCI v0.2 function IDs
 psci: Trusted OS migration not required
 random: fast init done
-percpu: Embedded 23 pages/cpu @ffff80003efcf000 s55936 r8192 d30080 u94208
+percpu: Embedded 23 pages/cpu @ffff80003efcf000 s56064 r8192 d29952 u94208
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
 CPU features: enabling workaround for ARM erratum 834220
@@ -29,15 +29,15 @@
 Policy zone: DMA
 Kernel command line: root=/dev/vda2 rw rootwait mem=1024M console=ttyAMA0,38400n8 init=/bin/bash
 PID hash table entries: 4096 (order: 3, 32768 bytes)
-Memory: 998908K/1048576K available (9276K kernel code, 1142K rwdata, 4592K rodata, 1152K init, 402K bss, 33284K reserved, 16384K cma-reserved)
+Memory: 998904K/1048576K available (9276K kernel code, 1148K rwdata, 4596K rodata, 1152K init, 400K bss, 33288K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008990000   (  9280 KB)
     .rodata : 0xffff000008990000 - 0xffff000008e10000   (  4608 KB)
       .init : 0xffff000008e10000 - 0xffff000008f30000   (  1152 KB)
-      .data : 0xffff000008f30000 - 0xffff00000904da00   (  1143 KB)
-       .bss : 0xffff00000904da00 - 0xffff0000090b2470   (   403 KB)
+      .data : 0xffff000008f30000 - 0xffff00000904f200   (  1149 KB)
+       .bss : 0xffff00000904f200 - 0xffff0000090b3208   (   401 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -77,7 +77,7 @@
 DMI not present or invalid.
 NET: Registered protocol family 16
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.172:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.180:1): state=initialized audit_enabled=0 res=1
 cpuidle: using governor menu
 vdso: 2 pages (1 code @ ffff000008996000, 1 data @ ffff000008f35000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
@@ -179,7 +179,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2017-09-22 05:24:06 UTC (1506057846)
+rtc-pl031 9010000.pl031: setting system clock to 2017-09-26 08:04:43 UTC (1506413083)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-09-26 01:04:58.561727578 -0700
+++ /tmp/foob	2017-09-26 01:04:58.565727594 -0700
@@ -40,20 +40,20 @@
 	 #################################################################
 	 #################################################################
 	 #####################################
-	 3 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 8167944 (7ca208 hex)
+Bytes transferred = 8172040 (7cb208 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 128.9 KiB/s
+	 3 MiB/s
 done
 Bytes transferred = 35036 (88dc hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x7ca200 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x7cb200 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8db ... OK
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-next-20170922-201396-g7352731 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Sep 21 22:20:12 PDT 2017
+Linux version 4.14.0-rc2-next-20170926-202105-gd35adcb (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Mon Sep 25 22:37:06 PDT 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -72,21 +72,21 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c158f000, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1590240, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
 random: fast init done
-percpu: Embedded 17 pages/cpu @eef98000 s39552 r8192 d21888 u69632
-pcpu-alloc: s39552 r8192 d21888 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @eef99000 s39616 r8192 d21824 u69632
+pcpu-alloc: s39616 r8192 d21824 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=7369c667-ff51-ec4a-29cd-baabf2fbe346 rootwait gpt debug earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993476K/2096128K available (10240K kernel code, 1629K rwdata, 4508K rodata, 2048K init, 405K bss, 37116K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993480K/2096128K available (10240K kernel code, 1633K rwdata, 4508K rodata, 2048K init, 401K bss, 37112K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -96,8 +96,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc15974a8   (1630 kB)
-       .bss : 0xc15a0bb4 - 0xc1606374   ( 406 kB)
+      .data : 0xc1400000 - 0xc15986e8   (1634 kB)
+       .bss : 0xc15a1bc0 - 0xc16062bc   ( 402 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -381,15 +381,15 @@
 mmc0: new high speed SDHC card at address aaaa
 mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 usb2_vbus: supplied by vdd-sys-5v0
- mmcblk0: p1 p2 p3
 VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
+ mmcblk0: p1 p2 p3
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
-mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
+mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB, chardev (237:0)
  mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
 Console: switching to colour frame buffer device 240x75
 drm drm: fb0:  frame buffer device
@@ -401,7 +401,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-13 12:13:50 UTC (950444030)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-17 12:33:33 UTC (950790813)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -413,5 +413,4 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.959159] usb 1-1: new high-speed USB device number 2 using tegra-ehci
-asix 1-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0d:b5:16
+root@(none):/# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2017-09-26 01:04:58.589727694 -0700
+++ /tmp/foob	2017-09-26 01:04:58.593727712 -0700
@@ -37,10 +37,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #######################################
-	 2.9 MiB/s
+	 ########################################
+	 3.3 MiB/s
 done
-Bytes transferred = 6293904 (600990 hex)
+Bytes transferred = 6298848 (601ce0 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -51,7 +51,7 @@
 done
 Bytes transferred = 35036 (88dc hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x600990 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x601ce0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8db ... OK
@@ -60,7 +60,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-next-20170922-201396-g7352731 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Sep 21 22:20:24 PDT 2017
+Linux version 4.14.0-rc2-next-20170926-202105-gd35adcb (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Sep 25 22:36:55 PDT 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -70,21 +70,21 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0fb9480, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0fba8c0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
 random: fast init done
-percpu: Embedded 17 pages/cpu @eef97000 s39680 r8192 d21760 u69632
-pcpu-alloc: s39680 r8192 d21760 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @eef98000 s39808 r8192 d21632 u69632
+pcpu-alloc: s39808 r8192 d21632 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=7369c667-ff51-ec4a-29cd-baabf2fbe346 rootwait gpt debug earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1997492K/2096128K available (9216K kernel code, 785K rwdata, 2796K rodata, 1024K init, 300K bss, 33100K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1997496K/2096128K available (9216K kernel code, 790K rwdata, 2800K rodata, 1024K init, 296K bss, 33096K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -94,8 +94,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
-      .data : 0xc0f00000 - 0xc0fc45a0   ( 786 kB)
-       .bss : 0xc0fce514 - 0xc10196ac   ( 301 kB)
+      .data : 0xc0f00000 - 0xc0fc59e0   ( 791 kB)
+       .bss : 0xc0fcf520 - 0xc101957c   ( 297 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -356,7 +356,7 @@
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
-mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
+mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB, chardev (247:0)
  mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
 iommu: Adding device 70080400.i2s to group 58
 tegra30-i2s 70080400.i2s: DMA channels sourced from device 70080000.ahub
@@ -396,7 +396,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-13 12:16:17 UTC (950444177)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-17 12:30:45 UTC (950790645)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -410,4 +410,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.188909] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.189295] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-09-26 01:04:58.653727963 -0700
+++ /tmp/foob	2017-09-26 01:04:58.653727963 -0700
@@ -27,7 +27,7 @@
 Filename 'zImage.multi_v7_defconfig'.
 Load address: 0x82000000
 Loading: *#################################################################
-	 #################################################################
+	 ################################################T T T #################
 	 #################################################################
 	 #################################################################
 	 #################################################################
@@ -35,20 +35,20 @@
 	 #################################################################
 	 #################################################################
 	 #####################################
-	 3.7 MiB/s
+	 468.8 KiB/s
 done
-Bytes transferred = 8167944 (7ca208 hex)
+Bytes transferred = 8172040 (7cb208 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.4 MiB/s
+	 3.8 MiB/s
 done
 Bytes transferred = 67173 (10665 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x7ca200 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x7cb200 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013664
@@ -56,7 +56,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-next-20170922-201396-g7352731 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Sep 21 22:20:12 PDT 2017
+Linux version 4.14.0-rc2-next-20170926-202105-gd35adcb (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Mon Sep 25 22:37:06 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -67,21 +67,21 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c158f000, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1590240, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
 random: fast init done
-percpu: Embedded 17 pages/cpu @eef82000 s39552 r8192 d21888 u69632
-pcpu-alloc: s39552 r8192 d21888 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @eef84000 s39616 r8192 d21824 u69632
+pcpu-alloc: s39616 r8192 d21824 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=b1791faa-84e8-4e27-83a1-9fb0a77e8803 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993356K/2096128K available (10240K kernel code, 1629K rwdata, 4508K rodata, 2048K init, 405K bss, 37236K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993364K/2096128K available (10240K kernel code, 1633K rwdata, 4508K rodata, 2048K init, 401K bss, 37228K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -91,8 +91,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc15974a8   (1630 kB)
-       .bss : 0xc15a0bb4 - 0xc1606374   ( 406 kB)
+      .data : 0xc1400000 - 0xc15986e8   (1634 kB)
+       .bss : 0xc15a1bc0 - 0xc16062bc   ( 402 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -404,7 +404,7 @@
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB, chardev (237:0)
  mmcblk0: p1
  usb2-1: usb2-1 supply vbus not found, using dummy regulator
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
@@ -467,8 +467,8 @@
 pci 0000:00:01.0: PME# supported from D0 D1 D2 D3hot D3cold
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x4c may corrupt adjacent RW1C bits
 pci 0000:00:02.0: [10de:0e13] type 01 class 0x060400
-ata1: SATA link down (SStatus 0 SControl 300)
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4 may corrupt adjacent RW1C bits
+ata1: SATA link down (SStatus 0 SControl 300)
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4 may corrupt adjacent RW1C bits
 pci 0000:00:02.0: enabling Extended Tags
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x88 may corrupt adjacent RW1C bits
@@ -529,7 +529,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:14 UTC (946684814)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:30 UTC (946684830)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2017-09-26 01:04:58.713728215 -0700
+++ /tmp/foob	2017-09-26 01:04:58.717728231 -0700
@@ -32,21 +32,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #######################################
-	 3.3 MiB/s
+	 ########################################
+	 3.6 MiB/s
 done
-Bytes transferred = 6293904 (600990 hex)
+Bytes transferred = 6298848 (601ce0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.6 MiB/s
+	 3.8 MiB/s
 done
 Bytes transferred = 67173 (10665 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x600990 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x601ce0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013664
@@ -55,7 +55,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-next-20170922-201396-g7352731 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Sep 21 22:20:24 PDT 2017
+Linux version 4.14.0-rc2-next-20170926-202105-gd35adcb (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Sep 25 22:36:55 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -65,21 +65,21 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0fb9480, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0fba8c0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
 random: fast init done
-percpu: Embedded 17 pages/cpu @eef81000 s39680 r8192 d21760 u69632
-pcpu-alloc: s39680 r8192 d21760 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @eef83000 s39808 r8192 d21632 u69632
+pcpu-alloc: s39808 r8192 d21632 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=b1791faa-84e8-4e27-83a1-9fb0a77e8803 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1997372K/2096128K available (9216K kernel code, 785K rwdata, 2796K rodata, 1024K init, 300K bss, 33220K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1997380K/2096128K available (9216K kernel code, 790K rwdata, 2800K rodata, 1024K init, 296K bss, 33212K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -89,8 +89,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
-      .data : 0xc0f00000 - 0xc0fc45a0   ( 786 kB)
-       .bss : 0xc0fce514 - 0xc10196ac   ( 301 kB)
+      .data : 0xc0f00000 - 0xc0fc59e0   ( 791 kB)
+       .bss : 0xc0fcf520 - 0xc101957c   ( 297 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -367,7 +367,7 @@
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB, chardev (247:0)
  mmcblk0: p1
 iommu: Adding device 70301100.i2s to group 75
 tegra30-i2s 70301100.i2s: DMA channels sourced from device 70300000.ahub
@@ -537,7 +537,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:16 UTC (946684816)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:14 UTC (946684814)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-09-26 01:04:58.757728399 -0700
+++ /tmp/foob	2017-09-26 01:04:58.761728416 -0700
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -43,20 +43,20 @@
 	 #################################################################
 	 #################################################################
 	 #####################################
-	 2.8 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 8167944 (7ca208 hex)
+Bytes transferred = 8172040 (7cb208 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 2.8 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 68203 (10a6b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x7ca200 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x7cb200 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a6a
@@ -64,7 +64,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-next-20170922-201396-g7352731 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Sep 21 22:20:12 PDT 2017
+Linux version 4.14.0-rc2-next-20170926-202105-gd35adcb (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Mon Sep 25 22:37:06 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -76,21 +76,21 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c158f000, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1590240, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327679 pages, LIFO batch:31
 random: fast init done
-percpu: Embedded 17 pages/cpu @eef81000 s39552 r8192 d21888 u69632
-pcpu-alloc: s39552 r8192 d21888 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @eef83000 s39616 r8192 d21824 u69632
+pcpu-alloc: s39616 r8192 d21824 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists, mobility grouping on.  Total pages: 522751
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=4d36e82a-b3c0-4b4b-bfcd-afe5217ac023 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1994372K/2097148K available (10240K kernel code, 1629K rwdata, 4508K rodata, 2048K init, 405K bss, 37240K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1994380K/2097148K available (10240K kernel code, 1633K rwdata, 4508K rodata, 2048K init, 401K bss, 37232K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -100,8 +100,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc15974a8   (1630 kB)
-       .bss : 0xc15a0bb4 - 0xc1606374   ( 406 kB)
+      .data : 0xc1400000 - 0xc15986e8   (1634 kB)
+       .bss : 0xc15a1bc0 - 0xc16062bc   ( 402 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -410,7 +410,7 @@
 mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
 mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
-mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
+mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB, chardev (237:0)
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 elan_i2c 1-0015: failed to read report data: -121
@@ -446,7 +446,7 @@
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-as3722-rtc as3722-rtc: setting system clock to 2000-01-03 17:26:29 UTC (946920389)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 14:32:29 UTC (946737149)
 +1.8V_RUN_CAM: disabling
 +3.3V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2017-09-26 01:04:58.809728618 -0700
+++ /tmp/foob	2017-09-26 01:04:58.813728634 -0700
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -40,21 +40,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #######################################
-	 2.9 MiB/s
+	 ########################################
+	 3.5 MiB/s
 done
-Bytes transferred = 6293904 (600990 hex)
+Bytes transferred = 6298848 (601ce0 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 811.5 KiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 68203 (10a6b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x600990 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x601ce0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a6a
@@ -63,7 +63,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-next-20170922-201396-g7352731 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Sep 21 22:20:24 PDT 2017
+Linux version 4.14.0-rc2-next-20170926-202105-gd35adcb (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Sep 25 22:36:55 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -74,21 +74,21 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c0fb9480, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0fba8c0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327679 pages, LIFO batch:31
 random: fast init done
-percpu: Embedded 17 pages/cpu @eef80000 s39680 r8192 d21760 u69632
-pcpu-alloc: s39680 r8192 d21760 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @eef82000 s39808 r8192 d21632 u69632
+pcpu-alloc: s39808 r8192 d21632 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists, mobility grouping on.  Total pages: 522751
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=4d36e82a-b3c0-4b4b-bfcd-afe5217ac023 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998388K/2097148K available (9216K kernel code, 785K rwdata, 2796K rodata, 1024K init, 300K bss, 33224K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1998396K/2097148K available (9216K kernel code, 790K rwdata, 2800K rodata, 1024K init, 296K bss, 33216K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -98,8 +98,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
-      .data : 0xc0f00000 - 0xc0fc45a0   ( 786 kB)
-       .bss : 0xc0fce514 - 0xc10196ac   ( 301 kB)
+      .data : 0xc0f00000 - 0xc0fc59e0   ( 791 kB)
+       .bss : 0xc0fcf520 - 0xc101957c   ( 297 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -312,146 +312,146 @@
 kernel BUG at drivers/platform/chrome/cros_ec_proto.c:34!
 Internal error: Oops - BUG: 0 [#1] PREEMPT SMP ARM
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.14.0-rc1-next-20170922-201396-g7352731 #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.14.0-rc2-next-20170926-202105-gd35adcb #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-task: ee891740 task.stack: ee8a6000
+task: ee891780 task.stack: ee8a6000
 PC is at cros_ec_prepare_tx+0x168/0x16c
 LR is at cros_ec_pkt_xfer_spi+0x18/0x474
-pc : [<c06d82b0>]    lr : [<c0545e0c>]    psr: 20000013
+pc : [<c06d91dc>]    lr : [<c0546c8c>]    psr: 20000013
 sp : ee8a7bd8  ip : 00000000  fp : 00000000
-r10: c0545df4  r9 : 00000000  r8 : eefe503c
-r7 : ee296000  r6 : ee296000  r5 : ee296000  r4 : ee276d90
+r10: c0546c74  r9 : 00000000  r8 : eefe59f8
+r7 : ee297200  r6 : ee297200  r5 : ee297200  r4 : ee276d90
 r3 : 0000ffff  r2 : 00000000  r1 : 0000ffff  r0 : ee276d90
 Flags: nzCv  IRQs on  FIQs on  Mode SVC_32  ISA ARM  Segment none
 Control: 10c5387d  Table: 8000406a  DAC: 00000051
 Process swapper/0 (pid: 1, stack limit = 0xee8a6210)
 Stack: (0xee8a7bd8 to 0xee8a8000)
-7bc0:                                                       ee276d90 ee296000
-7be0: ee296090 c0545e0c c0977554 00000004 ee8a7c5c ee8a7c5c ee2a6c00 00000000
-7c00: c05921dc 00000000 00000008 00000008 00000000 ee8a7c14 ee8a7c14 00000000
-7c20: ee8a7c20 ee8a7c20 ee296050 ee296780 00000008 00000000 00000000 00000000
-7c40: c0e91d08 c0238b40 40002800 000003e0 eef8cd08 c023afb0 002dc6c0 ee8a7bf0
-7c60: ee8a7bf0 00000000 c06d80e0 005c52c0 3b9aca00 000f4240 00000000 ee8a7ca8
-7c80: 00000001 00000501 c0545df4 c097bdf8 00000000 ee2967c0 a0000013 c06d80e0
-7ca0: ee296000 ee276d90 ee296000 ee2a6c00 ee296000 eefe503c 00000000 c0545df4
-7cc0: 00000000 c06d803c 0000400b ee276d90 ee296000 ee2a6c00 0000000c eefe503c
-7ce0: c0ff9e7c c06d8458 00000000 ee276d90 ee2a6c00 00000000 00000000 eefe503c
-7d00: c0f6c080 c0545898 ee8a7d14 ee2a6c00 00000000 00000001 ee2a6c00 ee276d90
-7d20: ee296090 c0545c48 00000000 00000000 00000000 000007d0 ee2a6c00 00000000
-7d40: c0f6c070 00000000 00000000 c0592c20 ee2a6c00 c100b204 c100b208 c051ec00
-7d60: 00000000 ee8a7d98 c051ed90 00000001 c100b1e0 00000000 00000000 c051d220
-7d80: ee9f6170 ee167db8 ee2a6c00 ee2a6c34 c0f72d8c c051e940 ee2a6c00 00000001
-7da0: 00000000 ee2a6c00 ee2a6c00 c0f72d8c ee038000 c051df84 ee2a6c00 00000000
-7dc0: ee2a6c08 c051c5c8 ee2a6c00 c0595b34 00000000 ee2a6c00 ee2a6c00 ee038000
-7de0: 00000000 eea8fe10 eefe508c 00000001 00000000 c0592a74 002dc6c0 ee038000
-7e00: eefe503c 00000000 ee2a6c00 c0594a20 00000000 014000c0 00000000 c059440c
-7e20: 00000063 002dc6c0 ee038000 ee296350 eea8fe00 eea8fe10 40000000 00000153
-7e40: 00000000 c0594ee0 eefe4d2c ee038348 ee038000 eea8fe00 eea8fe10 c0595a6c
-7e60: 00000000 ee038348 ffffffed eea8fe10 fffffdfb c0f73708 00000000 00000000
-7e80: c0f73708 c05204f4 c05204a4 eea8fe10 c100b204 c100b208 00000000 c051ec00
-7ea0: eea8fe10 c0f73708 eea8fe44 00000000 00000111 c0e6b83c c0e005a4 c051ed58
-7ec0: 00000000 c0f73708 c051ecac c051d2bc ee8c955c eea18934 c0f73708 ee293680
-7ee0: c0f6a610 c051e158 c0c1ced4 c0f73708 00000006 c0f73708 00000006 c0e38804
-7f00: 00000000 c051f8d0 c0520148 c0e84208 00000006 c0101b8c 00000000 c0cba3e8
-7f20: 00000100 efffcc45 c0cba3e8 00000111 efffcc53 c013e3d4 c0cb8e94 00000000
+7bc0:                                                       ee276d90 ee297200
+7be0: ee297290 c0546c8c c097891c 00000004 ee8a7c5c ee8a7c5c ee292800 00000000
+7c00: c05930bc 00000000 00000008 00000008 00000000 ee8a7c14 ee8a7c14 00000000
+7c20: ee8a7c20 ee8a7c20 ee297250 ee297180 00000008 00000000 00000000 00000000
+7c40: c0e91d90 c0239820 40002800 000000e0 eef8ed90 c023bc90 002dc6c0 ee8a7bf0
+7c60: ee8a7bf0 00000000 c06d900c 005c52e0 3b9aca00 000f4240 00000000 ee8a7ca8
+7c80: 00000001 00000501 c0546c74 c097d1ec 00000000 ee2971c0 a0000013 c06d900c
+7ca0: ee297200 ee276d90 ee297200 ee292800 ee297200 eefe59f8 00000000 c0546c74
+7cc0: 00000000 c06d8f68 0000400b ee276d90 ee297200 ee292800 0000000c eefe59f8
+7ce0: c0ff9cbc c06d9384 00000000 ee276d90 ee292800 00000000 00000000 eefe59f8
+7d00: c0f6d3f0 c0546718 ee8a7d14 ee292800 00000000 00000001 ee292800 ee276d90
+7d20: ee297290 c0546ac8 00000000 00000000 00000000 000007d0 ee292800 00000000
+7d40: c0f6d3e0 00000000 00000000 c0593b00 ee292800 c100b054 c100b058 c051fa80
+7d60: 00000000 ee8a7d98 c051fc10 00000001 c100b030 00000000 00000000 c051e0a0
+7d80: ee9f6170 ee17b9b8 ee292800 ee292834 c0f740fc c051f7c0 ee292800 00000001
+7da0: 00000000 ee292800 ee292800 c0f740fc ee107000 c051ee04 ee292800 00000000
+7dc0: ee292808 c051d448 ee292800 c0596a14 00000000 ee292800 ee292800 ee107000
+7de0: 00000000 eea8fe10 eefe5a48 00000001 00000000 c0593954 002dc6c0 ee107000
+7e00: eefe59f8 00000000 ee292800 c0595900 00000000 014000c0 00000000 c05952ec
+7e20: 00000063 002dc6c0 ee107000 ee290d50 eea8fe00 eea8fe10 40000000 00000153
+7e40: 00000000 c0595dc0 eefe56e8 ee107350 ee107000 eea8fe00 eea8fe10 c059694c
+7e60: 00000000 ee107350 ffffffed eea8fe10 fffffdfb c0f74a78 00000000 00000000
+7e80: c0f74a78 c0521374 c0521324 eea8fe10 c100b054 c100b058 00000000 c051fa80
+7ea0: eea8fe10 c0f74a78 eea8fe44 00000000 00000111 c0e6b83c c0e005a4 c051fbd8
+7ec0: 00000000 c0f74a78 c051fb2c c051e13c ee8c955c eea18934 c0f74a78 ee28f280
+7ee0: c0f6b968 c051efd8 c0c1d124 c0f74a78 00000006 c0f74a78 00000006 c0e38888
+7f00: 00000000 c0520750 c0520fc8 c0e84208 00000006 c0101b8c 00000000 c0cbb3d4
+7f20: 00000100 efffcc45 c0cbb3d4 00000111 efffcc53 c013e488 c0cb9e80 00000000
 7f40: 00000006 00000006 efffcc45 efffcc4a 00000000 efffcbc0 00000006 c0e6b82c
-7f60: c0e84208 00000006 c0e6b830 c0fce540 00000111 c0e00da8 00000006 00000006
-7f80: 00000000 c0e005a4 00000000 c09752e8 00000000 00000000 00000000 00000000
-7fa0: 00000000 c09752f0 00000000 c0107cf0 00000000 00000000 00000000 00000000
+7f60: c0e84208 00000006 c0e6b830 c0fcf540 00000111 c0e00da8 00000006 00000006
+7f80: 00000000 c0e005a4 00000000 c09766b0 00000000 00000000 00000000 00000000
+7fa0: 00000000 c09766b8 00000000 c0107cf0 00000000 00000000 00000000 00000000
 7fc0: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
 7fe0: 00000000 00000000 00000000 00000000 00000013 00000000 00000000 00000000
-[<c06d82b0>] (cros_ec_prepare_tx) from [<c0545e0c>] (cros_ec_pkt_xfer_spi+0x18/0x474)
-[<c0545e0c>] (cros_ec_pkt_xfer_spi) from [<c06d803c>] (send_command+0x20/0xd8)
-[<c06d803c>] (send_command) from [<c06d8458>] (cros_ec_query_all+0x1a4/0x398)
-[<c06d8458>] (cros_ec_query_all) from [<c0545898>] (cros_ec_register+0x94/0x21c)
-[<c0545898>] (cros_ec_register) from [<c0545c48>] (cros_ec_spi_probe+0x108/0x150)
-[<c0545c48>] (cros_ec_spi_probe) from [<c0592c20>] (spi_drv_probe+0x7c/0xa8)
-[<c0592c20>] (spi_drv_probe) from [<c051ec00>] (driver_probe_device+0x238/0x2e4)
-[<c051ec00>] (driver_probe_device) from [<c051d220>] (bus_for_each_drv+0x44/0x8c)
-[<c051d220>] (bus_for_each_drv) from [<c051e940>] (__device_attach+0x9c/0x100)
-[<c051e940>] (__device_attach) from [<c051df84>] (bus_probe_device+0x84/0x8c)
-[<c051df84>] (bus_probe_device) from [<c051c5c8>] (device_add+0x380/0x528)
-[<c051c5c8>] (device_add) from [<c0592a74>] (spi_add_device+0x88/0x130)
-[<c0592a74>] (spi_add_device) from [<c0594a20>] (spi_register_controller+0x428/0x8b4)
-[<c0594a20>] (spi_register_controller) from [<c0594ee0>] (devm_spi_register_controller+0x34/0x70)
-[<c0594ee0>] (devm_spi_register_controller) from [<c0595a6c>] (tegra_spi_probe+0x298/0x35c)
-[<c0595a6c>] (tegra_spi_probe) from [<c05204f4>] (platform_drv_probe+0x50/0xb0)
-[<c05204f4>] (platform_drv_probe) from [<c051ec00>] (driver_probe_device+0x238/0x2e4)
-[<c051ec00>] (driver_probe_device) from [<c051ed58>] (__driver_attach+0xac/0xb0)
-[<c051ed58>] (__driver_attach) from [<c051d2bc>] (bus_for_each_dev+0x54/0x88)
-[<c051d2bc>] (bus_for_each_dev) from [<c051e158>] (bus_add_driver+0xe4/0x1f0)
-[<c051e158>] (bus_add_driver) from [<c051f8d0>] (driver_register+0x78/0xf4)
-[<c051f8d0>] (driver_register) from [<c0101b8c>] (do_one_initcall+0x3c/0x168)
+[<c06d91dc>] (cros_ec_prepare_tx) from [<c0546c8c>] (cros_ec_pkt_xfer_spi+0x18/0x474)
+[<c0546c8c>] (cros_ec_pkt_xfer_spi) from [<c06d8f68>] (send_command+0x20/0xd8)
+[<c06d8f68>] (send_command) from [<c06d9384>] (cros_ec_query_all+0x1a4/0x398)
+[<c06d9384>] (cros_ec_query_all) from [<c0546718>] (cros_ec_register+0x94/0x21c)
+[<c0546718>] (cros_ec_register) from [<c0546ac8>] (cros_ec_spi_probe+0x108/0x150)
+[<c0546ac8>] (cros_ec_spi_probe) from [<c0593b00>] (spi_drv_probe+0x7c/0xa8)
+[<c0593b00>] (spi_drv_probe) from [<c051fa80>] (driver_probe_device+0x238/0x2e4)
+[<c051fa80>] (driver_probe_device) from [<c051e0a0>] (bus_for_each_drv+0x44/0x8c)
+[<c051e0a0>] (bus_for_each_drv) from [<c051f7c0>] (__device_attach+0x9c/0x100)
+[<c051f7c0>] (__device_attach) from [<c051ee04>] (bus_probe_device+0x84/0x8c)
+[<c051ee04>] (bus_probe_device) from [<c051d448>] (device_add+0x380/0x528)
+[<c051d448>] (device_add) from [<c0593954>] (spi_add_device+0x88/0x130)
+[<c0593954>] (spi_add_device) from [<c0595900>] (spi_register_controller+0x428/0x8b4)
+[<c0595900>] (spi_register_controller) from [<c0595dc0>] (devm_spi_register_controller+0x34/0x70)
+[<c0595dc0>] (devm_spi_register_controller) from [<c059694c>] (tegra_spi_probe+0x298/0x35c)
+[<c059694c>] (tegra_spi_probe) from [<c0521374>] (platform_drv_probe+0x50/0xb0)
+[<c0521374>] (platform_drv_probe) from [<c051fa80>] (driver_probe_device+0x238/0x2e4)
+[<c051fa80>] (driver_probe_device) from [<c051fbd8>] (__driver_attach+0xac/0xb0)
+[<c051fbd8>] (__driver_attach) from [<c051e13c>] (bus_for_each_dev+0x54/0x88)
+[<c051e13c>] (bus_for_each_dev) from [<c051efd8>] (bus_add_driver+0xe4/0x1f0)
+[<c051efd8>] (bus_add_driver) from [<c0520750>] (driver_register+0x78/0xf4)
+[<c0520750>] (driver_register) from [<c0101b8c>] (do_one_initcall+0x3c/0x168)
 [<c0101b8c>] (do_one_initcall) from [<c0e00da8>] (kernel_init_freeable+0x10c/0x1d0)
-[<c0e00da8>] (kernel_init_freeable) from [<c09752f0>] (kernel_init+0x8/0x114)
-[<c09752f0>] (kernel_init) from [<c0107cf0>] (ret_from_fork+0x14/0x24)
+[<c0e00da8>] (kernel_init_freeable) from [<c09766b8>] (kernel_init+0x8/0x114)
+[<c09766b8>] (kernel_init) from [<c0107cf0>] (ret_from_fork+0x14/0x24)
 Code: e2800008 e8bd8070 e7f001f2 e7f001f2 (e7f001f2) 
----[ end trace 9536c9ac31988c7c ]---
+---[ end trace 72f7e95ffc764221 ]---
 Kernel panic - not syncing: Attempted to kill init! exitcode=0x0000000b
 
 CPU1: stopping
-CPU: 1 PID: 0 Comm: swapper/1 Tainted: G      D         4.14.0-rc1-next-20170922-201396-g7352731 #1
+CPU: 1 PID: 0 Comm: swapper/1 Tainted: G      D         4.14.0-rc2-next-20170926-202105-gd35adcb #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c01110d0>] (unwind_backtrace) from [<c010c08c>] (show_stack+0x10/0x14)
-[<c010c08c>] (show_stack) from [<c0961b80>] (dump_stack+0x84/0xa4)
-[<c0961b80>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
+[<c010c08c>] (show_stack) from [<c0962f40>] (dump_stack+0x84/0xa4)
+[<c0962f40>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
 [<c010f61c>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
 Exception stack(0xee8dff70 to 0xee8dffb8)
-ff60:                                     00000001 00000000 ee8dffc8 c011a3a0
-ff80: c0f03ccc c0f03c68 c0e8d378 c0f03cd8 c0fbf27d 00000000 00000001 00000000
-ffa0: 2e10c000 ee8dffc0 c0108758 c010875c 60000013 ffffffff
+ff60:                                     00000001 00000000 ee8dffc8 c011a3c0
+ff80: c0f03ccc c0f03c68 c0e8d3f8 c0f03cd8 c0fc06bd 00000000 00000001 00000000
+ffa0: 2e10e000 ee8dffc0 c0108758 c010875c 60000013 ffffffff
 [<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
-[<c010875c>] (arch_cpu_idle) from [<c0160c78>] (do_idle+0x148/0x204)
-[<c0160c78>] (do_idle) from [<c0160fbc>] (cpu_startup_entry+0x18/0x1c)
-[<c0160fbc>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
-CPU3: stopping
-CPU: 3 PID: 0 Comm: swapper/3 Tainted: G      D         4.14.0-rc1-next-20170922-201396-g7352731 #1
+[<c010875c>] (arch_cpu_idle) from [<c0160e34>] (do_idle+0x148/0x204)
+[<c0160e34>] (do_idle) from [<c0161178>] (cpu_startup_entry+0x18/0x1c)
+[<c0161178>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
+CPU2: stopping
+CPU: 2 PID: 0 Comm: swapper/2 Tainted: G      D         4.14.0-rc2-next-20170926-202105-gd35adcb #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c01110d0>] (unwind_backtrace) from [<c010c08c>] (show_stack+0x10/0x14)
-[<c010c08c>] (show_stack) from [<c0961b80>] (dump_stack+0x84/0xa4)
-[<c0961b80>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
+[<c010c08c>] (show_stack) from [<c0962f40>] (dump_stack+0x84/0xa4)
+[<c0962f40>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
 [<c010f61c>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
-Exception stack(0xee8e3f70 to 0xee8e3fb8)
-3f60:                                     00000001 00000000 ee8e3fc8 c011a3a0
-3f80: c0f03ccc c0f03c68 c0e8d378 c0f03cd8 c0fbf27d 00000000 00000001 00000000
-3fa0: 2e12e000 ee8e3fc0 c0108758 c010875c 60000013 ffffffff
+Exception stack(0xee8e1f70 to 0xee8e1fb8)
+1f60:                                     00000001 00000000 ee8e1fc8 c011a3c0
+1f80: c0f03ccc c0f03c68 c0e8d3f8 c0f03cd8 c0fc06bd 00000000 00000001 00000000
+1fa0: 2e11f000 ee8e1fc0 c0108758 c010875c 60000013 ffffffff
 [<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
-[<c010875c>] (arch_cpu_idle) from [<c0160c78>] (do_idle+0x148/0x204)
-[<c0160c78>] (do_idle) from [<c0160fbc>] (cpu_startup_entry+0x18/0x1c)
-[<c0160fbc>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
-CPU2: stopping
-CPU: 2 PID: 0 Comm: swapper/2 Tainted: G      D         4.14.0-rc1-next-20170922-201396-g7352731 #1
+[<c010875c>] (arch_cpu_idle) from [<c0160e34>] (do_idle+0x148/0x204)
+[<c0160e34>] (do_idle) from [<c0161178>] (cpu_startup_entry+0x18/0x1c)
+[<c0161178>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
+CPU3: stopping
+CPU: 3 PID: 0 Comm: swapper/3 Tainted: G      D         4.14.0-rc2-next-20170926-202105-gd35adcb #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c01110d0>] (unwind_backtrace) from [<c010c08c>] (show_stack+0x10/0x14)
-[<c010c08c>] (show_stack) from [<c0961b80>] (dump_stack+0x84/0xa4)
-[<c0961b80>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
+[<c010c08c>] (show_stack) from [<c0962f40>] (dump_stack+0x84/0xa4)
+[<c0962f40>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
 [<c010f61c>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
-Exception stack(0xee8e1f70 to 0xee8e1fb8)
-1f60:                                     00000001 00000000 ee8e1fc8 c011a3a0
-1f80: c0f03ccc c0f03c68 c0e8d378 c0f03cd8 c0fbf27d 00000000 00000001 00000000
-1fa0: 2e11d000 ee8e1fc0 c0108758 c010875c 60000013 ffffffff
+Exception stack(0xee8e3f70 to 0xee8e3fb8)
+3f60:                                     00000001 00000000 ee8e3fc8 c011a3c0
+3f80: c0f03ccc c0f03c68 c0e8d3f8 c0f03cd8 c0fc06bd 00000000 00000001 00000000
+3fa0: 2e130000 ee8e3fc0 c0108758 c010875c 60000013 ffffffff
 [<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
-[<c010875c>] (arch_cpu_idle) from [<c0160c78>] (do_idle+0x148/0x204)
-[<c0160c78>] (do_idle) from [<c0160fbc>] (cpu_startup_entry+0x18/0x1c)
-[<c0160fbc>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
+[<c010875c>] (arch_cpu_idle) from [<c0160e34>] (do_idle+0x148/0x204)
+[<c0160e34>] (do_idle) from [<c0161178>] (cpu_startup_entry+0x18/0x1c)
+[<c0161178>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
 ---[ end Kernel panic - not syncing: Attempted to kill init! exitcode=0x0000000b
 
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 8 at kernel/irq_work.c:68 irq_work_queue_on+0xbc/0xfc
 Modules linked in:
-CPU: 0 PID: 8 Comm: rcu_preempt Tainted: G      D         4.14.0-rc1-next-20170922-201396-g7352731 #1
+CPU: 0 PID: 8 Comm: rcu_preempt Tainted: G      D         4.14.0-rc2-next-20170926-202105-gd35adcb #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c01110d0>] (unwind_backtrace) from [<c010c08c>] (show_stack+0x10/0x14)
-[<c010c08c>] (show_stack) from [<c0961b80>] (dump_stack+0x84/0xa4)
-[<c0961b80>] (dump_stack) from [<c0121908>] (__warn+0xd8/0x104)
-[<c0121908>] (__warn) from [<c0121954>] (warn_slowpath_null+0x20/0x28)
-[<c0121954>] (warn_slowpath_null) from [<c01d3f24>] (irq_work_queue_on+0xbc/0xfc)
-[<c01d3f24>] (irq_work_queue_on) from [<c0181fb4>] (rcu_implicit_dynticks_qs+0x280/0x2a0)
-[<c0181fb4>] (rcu_implicit_dynticks_qs) from [<c0181158>] (force_qs_rnp+0x140/0x1e0)
-[<c0181158>] (force_qs_rnp) from [<c0181730>] (rcu_gp_kthread+0x538/0xa6c)
-[<c0181730>] (rcu_gp_kthread) from [<c013ec84>] (kthread+0x120/0x158)
-[<c013ec84>] (kthread) from [<c0107cf0>] (ret_from_fork+0x14/0x24)
----[ end trace 9536c9ac31988c7d ]---
+[<c010c08c>] (show_stack) from [<c0962f40>] (dump_stack+0x84/0xa4)
+[<c0962f40>] (dump_stack) from [<c01219ac>] (__warn+0xd8/0x104)
+[<c01219ac>] (__warn) from [<c01219f8>] (warn_slowpath_null+0x20/0x28)
+[<c01219f8>] (warn_slowpath_null) from [<c01d4bc0>] (irq_work_queue_on+0xbc/0xfc)
+[<c01d4bc0>] (irq_work_queue_on) from [<c0182358>] (rcu_implicit_dynticks_qs+0x280/0x2a0)
+[<c0182358>] (rcu_implicit_dynticks_qs) from [<c01814fc>] (force_qs_rnp+0x140/0x1e0)
+[<c01814fc>] (force_qs_rnp) from [<c0181ad4>] (rcu_gp_kthread+0x538/0xa6c)
+[<c0181ad4>] (rcu_gp_kthread) from [<c013ed38>] (kthread+0x120/0x158)
+[<c013ed38>] (kthread) from [<c0107cf0>] (ret_from_fork+0x14/0x24)
+---[ end trace 72f7e95ffc764222 ]---
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2017-09-26 01:04:58.841728751 -0700
+++ /tmp/foob	2017-09-26 01:04:58.841728751 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1801320 us
+Bootloader-Cpu Init at (time stamp): 1801285 us
 
 Pinmux changes applied in kernel way
 
@@ -62,22 +62,22 @@
 Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1801294  us
-Bootloader (AosInit) time: 733430 us
-Bootloader Main Init time: 180051 us
-BlockDevMgrInit time: 157535 us
+Miscellaneous avp time: 1801259  us
+Bootloader (AosInit) time: 733472 us
+Bootloader Main Init time: 180301 us
+BlockDevMgrInit time: 157557 us
 PartMgrInit time: 8 us
 StorMgrInit time: 30 us
 FsMgrInit time: 2 us
-Partition Table load: 77039 us 
-Display init time: 574678 us
-Kernel read time: 1350 msec
-Load Kernel Size:   17891328 Bytes
-Bl Kernel ReadSpeed: 13252000 Bytes/sec
-Pre-Kernel Jump time: 3852062 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 8820081 us
+Partition Table load: 76998 us 
+Display init time: 574846 us
+Kernel read time: 1351 msec
+Load Kernel Size:   17897472 Bytes
+Bl Kernel ReadSpeed: 13247000 Bytes/sec
+Pre-Kernel Jump time: 3853008 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 8825168 us
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-next-20170922-201396-g7352731 (jonathanh@sellafield) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Thu Sep 21 22:23:12 PDT 2017
+Linux version 4.14.0-rc2-next-20170926-320346-gd35adcbe86c1 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Mon Sep 25 22:42:39 PDT 2017
 Boot CPU: AArch64 Processor [4e0f0000]
 Machine model: NVIDIA Tegra132 Norrin
 Memory limited to 1734MB
@@ -86,7 +86,7 @@
 cma: Reserved 16 MiB at 0x00000000eb400000
 NUMA: No NUMA configuration found
 NUMA: Faking a node at [mem 0x0000000000000000-0x00000000ec5fffff]
-NUMA: NODE_DATA [mem 0xec5bfd80-0xec5c187f]
+NUMA: NODE_DATA [mem 0xec5c0b80-0xec5c267f]
 Zone ranges:
   DMA      [mem 0x0000000080000000-0x00000000ec5fffff]
   Normal   empty
@@ -96,21 +96,21 @@
 Initmem setup node 0 [mem 0x0000000080000000-0x00000000ec5fffff]
 /cpus/cpu@1: missing enable-method property
 random: fast init done
-percpu: Embedded 23 pages/cpu @ffff80006c5a3000 s55936 r8192 d30080 u94208
+percpu: Embedded 23 pages/cpu @ffff80006c5a4000 s56064 r8192 d29952 u94208
 Detected PIPT I-cache on CPU0
 Built 1 zonelists, mobility grouping on.  Total pages: 436968
 Policy zone: DMA
 Kernel command line: tegraid=13.1.3.0.0 mem=1734M@2048M memtype=255 vpr=151M@3817M ddr_die=2048M@2048M section=256M commchip_id=0 usb_port_owner_info=0 lane_owner_info=0 emc_max_dvfs=0 touch_id=0@0 video=tegrafb no_console_suspend=1 console=ttyS0,115200n8 debug_uartport=lsport,0 earlyprintk=uart8250-32bit,0x70006000 usbcore.old_scheme_first=1 lp0_vec=0@0x0 tegra_fbmem=32899072@0xad012000 core_edp_mv=1150 core_edp_ma=4000 pmuboard=0x0176:0x0001:0x01:0x43:0x00 displayboard=0x0162:0x0001:0x00:0x42:0x00 power_supply=Adapter audio_codec=rt5640 board_info=0x0176:0x0001:0x01:0x43:0x00 tegraboot=sdmmc gpt gpt_sector=192511 modem_id=0 android.kerneltype=normal
 PID hash table entries: 4096 (order: 3, 32768 bytes)
-Memory: 1708132K/1775616K available (9276K kernel code, 1142K rwdata, 4592K rodata, 1152K init, 402K bss, 51100K reserved, 16384K cma-reserved)
+Memory: 1708128K/1775616K available (9276K kernel code, 1148K rwdata, 4596K rodata, 1152K init, 400K bss, 51104K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008990000   (  9280 KB)
     .rodata : 0xffff000008990000 - 0xffff000008e10000   (  4608 KB)
       .init : 0xffff000008e10000 - 0xffff000008f30000   (  1152 KB)
-      .data : 0xffff000008f30000 - 0xffff00000904da00   (  1143 KB)
-       .bss : 0xffff00000904da00 - 0xffff0000090b2470   (   403 KB)
+      .data : 0xffff000008f30000 - 0xffff00000904f200   (  1149 KB)
+       .bss : 0xffff00000904f200 - 0xffff0000090b3208   (   401 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -150,7 +150,7 @@
 DMI not present or invalid.
 NET: Registered protocol family 16
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.128:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.132:1): state=initialized audit_enabled=0 res=1
 cpuidle: using governor menu
 vdso: 2 pages (1 code @ ffff000008996000, 1 data @ ffff000008f35000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
@@ -263,22 +263,23 @@
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
 mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 NET: Registered protocol family 17
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB, chardev (240:0)
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 9pnet: Installing 9P2000 support
 Key type dns_resolver registered
 registered taskstats version 1
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
 +VDD_LED: supplied by +VDD_MUX
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 +5V_USB_HS: supplied by +5V_SYS
 +5V_USB_SS: supplied by +5V_SYS
 +5V_HDMI_CON: supplied by +5V_SYS
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
+sdhci-tegra 700b0400.sdhci: Got CD GPIO
 hctosys: unable to open rtc device (rtc0)
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +VDD_LED: disabling
@@ -289,7 +290,6 @@
 ALSA device list:
   No soundcards found.
 Freeing unused kernel memory: 1152K
-sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 
 System initialization...
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-09-26 01:04:58.885728935 -0700
+++ /tmp/foob	2017-09-26 01:04:58.889728952 -0700
@@ -48,20 +48,20 @@
 	 #################################################################
 	 #################################################################
 	 #####################################
-	 3.5 MiB/s
+	 4.1 MiB/s
 done
-Bytes transferred = 8167944 (7ca208 hex)
+Bytes transferred = 8172040 (7cb208 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 4.1 MiB/s
 done
 Bytes transferred = 21454 (53ce hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x7ca200 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x7cb200 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103cd
@@ -69,7 +69,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-next-20170922-201396-g7352731 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Sep 21 22:20:12 PDT 2017
+Linux version 4.14.0-rc2-next-20170926-202105-gd35adcb (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Mon Sep 25 22:37:06 PDT 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -79,21 +79,21 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0x3c000000
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c158f000, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c1590240, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 65536 pages, LIFO batch:15
 random: fast init done
-percpu: Embedded 17 pages/cpu @ef7c4000 s39552 r8192 d21888 u69632
-pcpu-alloc: s39552 r8192 d21888 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @ef7c5000 s39616 r8192 d21824 u69632
+pcpu-alloc: s39616 r8192 d21824 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 
 Built 1 zonelists, mobility grouping on.  Total pages: 260608
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=11eb23c1-01 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 954308K/1048576K available (10240K kernel code, 1629K rwdata, 4508K rodata, 2048K init, 405K bss, 28732K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 954312K/1048576K available (10240K kernel code, 1633K rwdata, 4508K rodata, 2048K init, 401K bss, 28728K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -103,8 +103,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc15974a8   (1630 kB)
-       .bss : 0xc15a0bb4 - 0xc1606374   ( 406 kB)
+      .data : 0xc1400000 - 0xc15986e8   (1634 kB)
+       .bss : 0xc15a1bc0 - 0xc16062bc   ( 402 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -346,9 +346,9 @@
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2017-09-26 01:04:58.917729070 -0700
+++ /tmp/foob	2017-09-26 01:04:58.921729086 -0700
@@ -45,21 +45,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #######################################
-	 3.3 MiB/s
+	 ########################################
+	 3.8 MiB/s
 done
-Bytes transferred = 6293904 (600990 hex)
+Bytes transferred = 6298848 (601ce0 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 2.6 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21454 (53ce hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x600990 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x601ce0 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103cd
@@ -68,7 +68,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-next-20170922-201396-g7352731 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Sep 21 22:20:24 PDT 2017
+Linux version 4.14.0-rc2-next-20170926-202105-gd35adcb (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Sep 25 22:36:55 PDT 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -77,21 +77,21 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0x3c000000
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0fb9480, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0fba8c0, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 65536 pages, LIFO batch:15
 random: fast init done
-percpu: Embedded 17 pages/cpu @ef7c3000 s39680 r8192 d21760 u69632
-pcpu-alloc: s39680 r8192 d21760 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @ef7c4000 s39808 r8192 d21632 u69632
+pcpu-alloc: s39808 r8192 d21632 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 
 Built 1 zonelists, mobility grouping on.  Total pages: 260608
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=11eb23c1-01 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 958320K/1048576K available (9216K kernel code, 785K rwdata, 2796K rodata, 1024K init, 300K bss, 24720K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 958324K/1048576K available (9216K kernel code, 790K rwdata, 2800K rodata, 1024K init, 296K bss, 24716K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -101,8 +101,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
-      .data : 0xc0f00000 - 0xc0fc45a0   ( 786 kB)
-       .bss : 0xc0fce514 - 0xc10196ac   ( 301 kB)
+      .data : 0xc0f00000 - 0xc0fc59e0   ( 791 kB)
+       .bss : 0xc0fcf520 - 0xc101957c   ( 297 kB)
 Preemptible hierarchical RCU implementation.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
 	Tasks RCU enabled.
@@ -324,12 +324,15 @@
 NET: Registered protocol family 17
 NET: Registered protocol family 15
 can: controller area network core (rev 20170425 abi 9)
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
 NET: Registered protocol family 29
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 can: raw protocol (rev 20170425)
 can: broadcast manager protocol (rev 20170425 t)
 can: netlink gateway (rev 20170425) max_hops=1
 Bluetooth: RFCOMM socket layer initialized
+mmc1: new high speed SD card at address b368
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
+ mmcblk1: p1 p2
 Bluetooth: RFCOMM ver 1.11
 Bluetooth: BNEP (Ethernet Emulation) ver 1.3
 Bluetooth: BNEP socket layer initialized
@@ -337,20 +340,17 @@
 Bluetooth: HIDP socket layer initialized
 Key type dns_resolver registered
 Registering SWP/SWPB emulation handler
-mmc1: new high speed SD card at address b368
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
- mmcblk1: p1 p2
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
+rtc-em3027 2-0056: hctosys: unable to read the hardware clock
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
-rtc-em3027 2-0056: hctosys: unable to read the hardware clock
 ALSA device list:
   #0: tegra-trimslice
 Waiting for root device PARTUUID=11eb23c1-01...
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2017-09-26 01:04:58.949729204 -0700
+++ /tmp/foob	2017-09-26 01:04:58.953729221 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 730495 us
-Basic modules init took 349216 us
+Tegraboot started after 811547 us
+Basic modules init took 350036 us
 Sec Bootdevice Read Time = 203 ms, Read Size = 8459 KB
-Next stage binary read took 14370 us
-Carveout took 252656 us
-CPU initialization took 92466 us
-Total time taken by TegraBoot 708708 us
+Next stage binary read took 14368 us
+Carveout took 253476 us
+CPU initialization took 92435 us
+Total time taken by TegraBoot 710315 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.570] LPDDR4 Training: Number of tables = 10
+64b[0001.651] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-16570880 bytes read in 836 ms (18.9 MiB/s)
+16577024 bytes read in 831 ms (19 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-45170 bytes read in 455 ms (96.7 KiB/s)
+45170 bytes read in 453 ms (96.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 496 ms (3.1 MiB/s)
+1627787 bytes read in 495 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-next-20170922-201396-g7352731 (jonathanh@sellafield) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Thu Sep 21 22:23:12 PDT 2017
+Linux version 4.14.0-rc2-next-20170926-320346-gd35adcbe86c1 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Mon Sep 25 22:42:39 PDT 2017
 Boot CPU: AArch64 Processor [411fd071]
 Machine model: NVIDIA Tegra210 P2371 (P2530/P2595) reference design
 debug: ignoring loglevel setting.
@@ -142,7 +142,7 @@
 cma: Reserved 16 MiB at 0x00000000fe000000
 NUMA: No NUMA configuration found
 NUMA: Faking a node at [mem 0x0000000000000000-0x000000013fffffff]
-NUMA: NODE_DATA [mem 0x13ffaca80-0x13ffae57f]
+NUMA: NODE_DATA [mem 0x13ffae300-0x13ffafdff]
 Zone ranges:
   DMA      [mem 0x0000000080000000-0x00000000ffffffff]
   Normal   [mem 0x0000000100000000-0x000000013fffffff]
@@ -162,8 +162,8 @@
 psci: Using standard PSCI v0.2 function IDs
 psci: MIGRATE_INFO_TYPE not supported.
 random: fast init done
-percpu: Embedded 23 pages/cpu @ffff8000bff4d000 s55936 r8192 d30080 u94208
-pcpu-alloc: s55936 r8192 d30080 u94208 alloc=23*4096
+percpu: Embedded 23 pages/cpu @ffff8000bff4f000 s56064 r8192 d29952 u94208
+pcpu-alloc: s56064 r8192 d29952 u94208 alloc=23*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
@@ -173,15 +173,15 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffff80007a000000-ffff80007dffffff]
-Memory: 2982496K/3131392K available (9276K kernel code, 1142K rwdata, 4592K rodata, 1152K init, 402K bss, 132512K reserved, 16384K cma-reserved)
+Memory: 2982500K/3131392K available (9276K kernel code, 1148K rwdata, 4596K rodata, 1152K init, 400K bss, 132508K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008990000   (  9280 KB)
     .rodata : 0xffff000008990000 - 0xffff000008e10000   (  4608 KB)
       .init : 0xffff000008e10000 - 0xffff000008f30000   (  1152 KB)
-      .data : 0xffff000008f30000 - 0xffff00000904da00   (  1143 KB)
-       .bss : 0xffff00000904da00 - 0xffff0000090b2470   (   403 KB)
+      .data : 0xffff000008f30000 - 0xffff00000904f200   (  1149 KB)
+       .bss : 0xffff00000904f200 - 0xffff0000090b3208   (   401 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -270,7 +270,7 @@
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 PCI: CLS 0 bytes, default 128
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 80976000
+kvm [1]: IDMAP page: 80977000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
@@ -346,7 +346,7 @@
 mmcblk0: mmc0:0001 SDW16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SDW16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SDW16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SDW16G partition 3 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SDW16G partition 3 4.00 MiB, chardev (240:0)
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12 p13 p14 p15 p16 p17 p18
 EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2017-09-26 01:04:58.961729254 -0700
+++ /tmp/foob	2017-09-26 01:04:58.965729271 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-next-20170922-201396-g7352731 (jonathanh@sellafield) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Thu Sep 21 22:23:12 PDT 2017
+Linux version 4.14.0-rc2-next-20170926-320346-gd35adcbe86c1 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Mon Sep 25 22:42:39 PDT 2017
 Boot CPU: AArch64 Processor [411fd071]
 Machine model: Google Pixel C
 efi: Getting EFI parameters from FDT:
@@ -9,7 +9,7 @@
 bootconsole [uart0] enabled
 NUMA: No NUMA configuration found
 NUMA: Faking a node at [mem 0x0000000000000000-0x000000013fffffff]
-NUMA: NODE_DATA [mem 0x13ffa0c80-0x13ffa277f]
+NUMA: NODE_DATA [mem 0x13ffa2900-0x13ffa43ff]
 Zone ranges:
   DMA      [mem 0x0000000080000000-0x00000000ffffffff]
   Normal   [mem 0x0000000100000000-0x000000013fffffff]
@@ -22,7 +22,7 @@
 psci: PSCIv1.0 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: MIGRATE_INFO_UP_CPU reported invalid physical ID (0x80000000)
-percpu: Embedded 23 pages/cpu @ffff8000bff41000 s55936 r8192 d30080 u94208
+percpu: Embedded 23 pages/cpu @ffff8000bff43000 s56064 r8192 d29952 u94208
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
 CPU features: enabling workaround for ARM erratum 834220
@@ -31,15 +31,15 @@
 Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2837760K/2988032K available (9276K kernel code, 1142K rwdata, 4592K rodata, 1152K init, 402K bss, 133888K reserved, 16384K cma-reserved)
+Memory: 2837768K/2988032K available (9276K kernel code, 1148K rwdata, 4596K rodata, 1152K init, 400K bss, 133880K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008990000   (  9280 KB)
     .rodata : 0xffff000008990000 - 0xffff000008e10000   (  4608 KB)
       .init : 0xffff000008e10000 - 0xffff000008f30000   (  1152 KB)
-      .data : 0xffff000008f30000 - 0xffff00000904da00   (  1143 KB)
-       .bss : 0xffff00000904da00 - 0xffff0000090b2470   (   403 KB)
+      .data : 0xffff000008f30000 - 0xffff00000904f200   (  1149 KB)
+       .bss : 0xffff00000904f200 - 0xffff0000090b3208   (   401 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -150,9 +150,9 @@
 RPC: Registered tcp transport module.
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 Unpacking initramfs...
-Freeing initrd memory: 1284K
+Freeing initrd memory: 1280K
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 80976000
+kvm [1]: IDMAP page: 80977000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
@@ -258,9 +258,9 @@
 mmcblk0: mmc0:0001 BGND3R 29.1 GiB 
 mmcblk0boot0: mmc0:0001 BGND3R partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB
+mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB, chardev (240:0)
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
-max77686-rtc max77620-rtc: setting system clock to 2001-07-03 10:47:31 UTC (994157251)
+max77686-rtc max77620-rtc: setting system clock to 2001-07-07 11:07:08 UTC (994504028)
 PP1200_AVDD: disabling
 PP1200_RCAM: disabling
 PP2800L_RCAM: disabling
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-09-26 01:04:59.013729472 -0700
+++ /tmp/foob	2017-09-26 01:04:59.013729472 -0700
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra30 (Beaver) # 
 Tegra30 (Beaver) # usb reset
 (Re)start USB...
@@ -40,20 +40,20 @@
 	 #################################################################
 	 #################################################################
 	 #####################################
-	 2.9 MiB/s
+	 3.1 MiB/s
 done
-Bytes transferred = 8167944 (7ca208 hex)
+Bytes transferred = 8172040 (7cb208 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.6 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 51860 (ca94 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x7ca200 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x7cb200 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa93 ... OK
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-next-20170922-201396-g7352731 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Sep 21 22:20:12 PDT 2017
+Linux version 4.14.0-rc2-next-20170926-202105-gd35adcb (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Mon Sep 25 22:37:06 PDT 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -71,21 +71,21 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c158f000, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1590240, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
 random: fast init done
-percpu: Embedded 17 pages/cpu @eef88000 s39552 r8192 d21888 u69632
-pcpu-alloc: s39552 r8192 d21888 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @eef89000 s39616 r8192 d21824 u69632
+pcpu-alloc: s39616 r8192 d21824 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=edbe1840-3633-c751-7993-b0ac834f81e0 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993396K/2096128K available (10240K kernel code, 1629K rwdata, 4508K rodata, 2048K init, 405K bss, 37196K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993400K/2096128K available (10240K kernel code, 1633K rwdata, 4508K rodata, 2048K init, 401K bss, 37192K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -95,8 +95,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc15974a8   (1630 kB)
-       .bss : 0xc15a0bb4 - 0xc1606374   ( 406 kB)
+      .data : 0xc1400000 - 0xc15986e8   (1634 kB)
+       .bss : 0xc15a1bc0 - 0xc16062bc   ( 402 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -388,9 +388,9 @@
 mmcblk1: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
-mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB
 mmc0: new ultra high speed DDR50 SDHC card at address aaaa
 mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
+mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB, chardev (237:0)
 Primary GPT is invalid, using alternate GPT.
  mmcblk1: p1 p2 p3 p4 p5
  mmcblk0: p1 p2 p3
@@ -473,4 +473,4 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.544431] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.544039] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2017-09-26 01:04:59.065729690 -0700
+++ /tmp/foob	2017-09-26 01:04:59.069729707 -0700
@@ -37,10 +37,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #######################################
+	 ########################################
 	 2.8 MiB/s
 done
-Bytes transferred = 6293904 (600990 hex)
+Bytes transferred = 6298848 (601ce0 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -51,7 +51,7 @@
 done
 Bytes transferred = 51860 (ca94 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x600990 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x601ce0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa93 ... OK
@@ -60,7 +60,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-rc1-next-20170922-201396-g7352731 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Sep 21 22:20:24 PDT 2017
+Linux version 4.14.0-rc2-next-20170926-202105-gd35adcb (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Sep 25 22:36:55 PDT 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -69,21 +69,21 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0fb9480, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0fba8c0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
 random: fast init done
-percpu: Embedded 17 pages/cpu @eef87000 s39680 r8192 d21760 u69632
-pcpu-alloc: s39680 r8192 d21760 u69632 alloc=17*4096
+percpu: Embedded 17 pages/cpu @eef88000 s39808 r8192 d21632 u69632
+pcpu-alloc: s39808 r8192 d21632 u69632 alloc=17*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=edbe1840-3633-c751-7993-b0ac834f81e0 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1997412K/2096128K available (9216K kernel code, 785K rwdata, 2796K rodata, 1024K init, 300K bss, 33180K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1997416K/2096128K available (9216K kernel code, 790K rwdata, 2800K rodata, 1024K init, 296K bss, 33176K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -93,8 +93,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
-      .data : 0xc0f00000 - 0xc0fc45a0   ( 786 kB)
-       .bss : 0xc0fce514 - 0xc10196ac   ( 301 kB)
+      .data : 0xc0f00000 - 0xc0fc59e0   ( 791 kB)
+       .bss : 0xc0fcf520 - 0xc101957c   ( 297 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -353,7 +353,7 @@
 mmcblk1: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
-mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB
+mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB, chardev (247:0)
 Primary GPT is invalid, using alternate GPT.
  mmcblk1: p1 p2 p3 p4 p5
 mmc0: new ultra high speed DDR50 SDHC card at address aaaa
@@ -470,4 +470,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.794469] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.794439] usb 2-1: new high-speed USB device number 2 using tegra-ehci
