Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2015-09-14 20:23:33.506207110 -0700
+++ /tmp/foob	2015-09-14 20:23:33.506207110 -0700
@@ -1,6 +1,6 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
-Linux version 4.3.0-rc1-next-20150914+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Sep 13 20:20:06 PDT 2015
+Linux version 4.3.0-rc1-next-20150915+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Sep 14 20:18:52 PDT 2015
 CPU: AArch64 Processor [411fd070] revision 0
 Detected PIPT I-cache on CPU0
 alternatives: enabling workaround for ARM erratum 832075
@@ -20,7 +20,7 @@
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
 software IO TLB [mem 0x79c00000-0x7dc00000] (64MB) mapped at [ffffffc039c00000-ffffffc03dbfffff]
-Memory: 939192K/1048576K available (5369K kernel code, 472K rwdata, 2248K rodata, 512K init, 202K bss, 93000K reserved, 16384K cma-reserved)
+Memory: 939192K/1048576K available (5369K kernel code, 468K rwdata, 2252K rodata, 512K init, 202K bss, 93000K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     vmalloc : 0xffffff8000000000 - 0xffffffbdbfff0000   (   246 GB)
     vmemmap : 0xffffffbdc0000000 - 0xffffffbfc0000000   (     8 GB maximum)
@@ -29,9 +29,9 @@
     PCI I/O : 0xffffffbffae00000 - 0xffffffbffbe00000   (    16 MB)
     modules : 0xffffffbffc000000 - 0xffffffc000000000   (    64 MB)
     memory  : 0xffffffc000000000 - 0xffffffc040000000   (  1024 MB)
-      .init : 0xffffffc0007f3000 - 0xffffffc000873000   (   512 KB)
-      .text : 0xffffffc000080000 - 0xffffffc0007f2524   (  7626 KB)
-      .data : 0xffffffc000883000 - 0xffffffc0008f9000   (   472 KB)
+      .init : 0xffffffc0007f4000 - 0xffffffc000874000   (   512 KB)
+      .text : 0xffffffc000080000 - 0xffffffc0007f3524   (  7630 KB)
+      .data : 0xffffffc000884000 - 0xffffffc0008f9000   (   468 KB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 64.
@@ -90,7 +90,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 2, 16384 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.590:1): initialized
+audit: type=2000 audit(0.580:1): initialized
 HugeTLB registered 2 MB page size, pre-allocated 0 pages
 VFS: Disk quotas dquot_6.6.0
 VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
@@ -134,8 +134,8 @@
 EXT4-fs (vda2): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 254:2.
 devtmpfs: mounted
-Freeing unused kernel memory: 512K (ffffffc0007f3000 - ffffffc000873000)
-Freeing alternatives memory: 44K (ffffffc000873000 - ffffffc00087e000)
+Freeing unused kernel memory: 512K (ffffffc0007f4000 - ffffffc000874000)
+Freeing alternatives memory: 44K (ffffffc000874000 - ffffffc00087f000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 bash-4.3# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-09-14 20:23:33.514207103 -0700
+++ /tmp/foob	2015-09-14 20:23:33.514207103 -0700
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra114 (Dalmore) # 
 Tegra114 (Dalmore) # usb reset
 (Re)start USB...
@@ -38,9 +38,9 @@
 	 #################################################################
 	 #################################################################
 	 ################################################
-	 3.3 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 6426312 (620ec8 hex)
+Bytes transferred = 6424608 (620820 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -51,7 +51,7 @@
 done
 Bytes transferred = 34966 (8896 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x620ec8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x620820 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff895 ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.3.0-rc1-next-20150914-41348-g91ad81b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Sep 13 20:18:12 PDT 2015
+Linux version 4.3.0-rc1-next-20150915-41518-gb1ab908 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Sep 14 20:17:36 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -67,7 +67,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f793c0, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0f770c0, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -80,7 +80,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998884K/2096128K available (8103K kernel code, 1012K rwdata, 3624K rodata, 1028K init, 340K bss, 31708K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1998892K/2096128K available (8103K kernel code, 1012K rwdata, 3624K rodata, 1020K init, 340K bss, 31700K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,10 +88,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d7cdf0   (11732 kB)
-      .init : 0xc0d7d000 - 0xc0e7e000   (1028 kB)
-      .data : 0xc0e7e000 - 0xc0f7b3a0   (1013 kB)
-       .bss : 0xc0f7e000 - 0xc0fd32e8   ( 341 kB)
+      .text : 0xc0208000 - 0xc0d7cd90   (11732 kB)
+      .init : 0xc0d7d000 - 0xc0e7c000   (1020 kB)
+      .data : 0xc0e7c000 - 0xc0f790a0   (1013 kB)
+       .bss : 0xc0f7c000 - 0xc0fd12e8   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -289,7 +289,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 78000400.sdhci [78000400.sdhci] using ADMA
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-03-26 12:13:17 UTC (954072797)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-03-27 12:12:40 UTC (954159160)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -298,7 +298,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1028K (c0d7d000 - c0e7e000)
+Freeing unused kernel memory: 1020K (c0d7d000 - c0e7c000)
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2015-09-14 20:23:33.518207100 -0700
+++ /tmp/foob	2015-09-14 20:23:33.522207096 -0700
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 ####################
-	 3.3 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 5054632 (4d20a8 hex)
+Bytes transferred = 5054608 (4d2090 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.8 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 34966 (8896 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4d20a8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4d2090 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff895 ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.3.0-rc1-next-20150914-41348-g91ad81b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Sep 13 20:18:30 PDT 2015
+Linux version 4.3.0-rc1-next-20150915-41518-gb1ab908 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Sep 14 20:17:52 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -90,7 +90,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc091e670   (9306 kB)
+      .text : 0xc0008000 - 0xc091e690   (9306 kB)
       .init : 0xc091f000 - 0xc0970000   ( 324 kB)
       .data : 0xc0970000 - 0xc09c5400   ( 341 kB)
        .bss : 0xc09c8000 - 0xc0a22d20   ( 364 kB)
@@ -286,7 +286,7 @@
 tegra-snd-rt5640 sound: rt5640-aif1 <-> 70080400.i2s mapping ok
 input: NVIDIA Tegra Dalmore Headphones as /devices/soc0/sound/sound/card0/input0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-03-26 12:13:47 UTC (954072827)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-03-27 12:13:11 UTC (954159191)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-09-14 20:23:33.526207093 -0700
+++ /tmp/foob	2015-09-14 20:23:33.530207089 -0700
@@ -42,18 +42,18 @@
 	 ################################################
 	 3.2 MiB/s
 done
-Bytes transferred = 6426312 (620ec8 hex)
+Bytes transferred = 6424608 (620820 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *#####
-	 2.8 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 64210 (fad2 hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x620ec8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x620820 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012ad1
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.3.0-rc1-next-20150914-41348-g91ad81b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Sep 13 20:18:12 PDT 2015
+Linux version 4.3.0-rc1-next-20150915-41518-gb1ab908 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Sep 14 20:17:36 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f793c0, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0f770c0, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998776K/2096128K available (8103K kernel code, 1012K rwdata, 3624K rodata, 1028K init, 340K bss, 31816K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1998784K/2096128K available (8103K kernel code, 1012K rwdata, 3624K rodata, 1020K init, 340K bss, 31808K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d7cdf0   (11732 kB)
-      .init : 0xc0d7d000 - 0xc0e7e000   (1028 kB)
-      .data : 0xc0e7e000 - 0xc0f7b3a0   (1013 kB)
-       .bss : 0xc0f7e000 - 0xc0fd32e8   ( 341 kB)
+      .text : 0xc0208000 - 0xc0d7cd90   (11732 kB)
+      .init : 0xc0d7d000 - 0xc0e7c000   (1020 kB)
+      .data : 0xc0e7c000 - 0xc0f790a0   (1013 kB)
+       .bss : 0xc0f7c000 - 0xc0fd12e8   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -350,7 +350,7 @@
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 usb 2-1: new high-speed USB device number 2 using tegra-ehci
 devtmpfs: mounted
-Freeing unused kernel memory: 1028K (c0d7d000 - c0e7e000)
+Freeing unused kernel memory: 1020K (c0d7d000 - c0e7c000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    5.985854] asix 2-1:1.0 eth0: register 'asix
\ No newline at end of file
+root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2015-09-14 20:23:33.534207086 -0700
+++ /tmp/foob	2015-09-14 20:23:33.538207082 -0700
@@ -29,7 +29,6 @@
 Tegra124 (Jetson TK1) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug init=/bin/bash noinitrd'
 Tegra124 (Jetson TK1) # setenv ipaddr 192.168.100.102; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra124 (Jetson TK1) # tftpboot 0x81000000 zImage.tegra_defconfig
-Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'zImage.tegra_defconfig'.
@@ -42,18 +41,18 @@
 	 ####################
 	 3.2 MiB/s
 done
-Bytes transferred = 5054632 (4d20a8 hex)
+Bytes transferred = 5054608 (4d2090 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *#####
-	 3.1 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 64210 (fad2 hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4d20a8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4d2090 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012ad1
@@ -64,7 +63,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.3.0-rc1-next-20150914-41348-g91ad81b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Sep 13 20:18:30 PDT 2015
+Linux version 4.3.0-rc1-next-20150915-41518-gb1ab908 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Sep 14 20:17:52 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -93,7 +92,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc091e670   (9306 kB)
+      .text : 0xc0008000 - 0xc091e690   (9306 kB)
       .init : 0xc091f000 - 0xc0970000   ( 324 kB)
       .data : 0xc0970000 - 0xc09c5400   ( 341 kB)
        .bss : 0xc09c8000 - 0xc0a22d20   ( 364 kB)
@@ -333,7 +332,7 @@
 tegra-snd-rt5640 sound: rt5640-aif1 <-> 70301100.i2s mapping ok
 input: NVIDIA Tegra Jetson TK1 Headphones as /devices/soc0/sound/sound/card1/input1
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:20 UTC (946684820)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:55 UTC (946684855)
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-09-14 20:23:33.542207079 -0700
+++ /tmp/foob	2015-09-14 20:23:33.542207079 -0700
@@ -46,20 +46,20 @@
 	 #################################################################
 	 #################################################################
 	 ################################################
-	 4.1 MiB/s
+	 3.9 MiB/s
 done
-Bytes transferred = 6426312 (620ec8 hex)
+Bytes transferred = 6424608 (620820 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 2.9 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21202 (52d2 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x620ec8 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x620820 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102d1
@@ -67,7 +67,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.3.0-rc1-next-20150914-41348-g91ad81b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Sep 13 20:18:12 PDT 2015
+Linux version 4.3.0-rc1-next-20150915-41518-gb1ab908 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Sep 14 20:17:36 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -75,7 +75,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0f793c0, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c0f770c0, node_mem_map eeff8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -88,7 +88,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 959692K/1048576K available (8103K kernel code, 1012K rwdata, 3624K rodata, 1028K init, 340K bss, 23348K reserved, 65536K cma-reserved, 204800K highmem)
+Memory: 959700K/1048576K available (8103K kernel code, 1012K rwdata, 3624K rodata, 1020K init, 340K bss, 23340K reserved, 65536K cma-reserved, 204800K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -96,10 +96,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d7cdf0   (11732 kB)
-      .init : 0xc0d7d000 - 0xc0e7e000   (1028 kB)
-      .data : 0xc0e7e000 - 0xc0f7b3a0   (1013 kB)
-       .bss : 0xc0f7e000 - 0xc0fd32e8   ( 341 kB)
+      .text : 0xc0208000 - 0xc0d7cd90   (11732 kB)
+      .init : 0xc0d7d000 - 0xc0e7c000   (1020 kB)
+      .data : 0xc0e7c000 - 0xc0f790a0   (1013 kB)
+       .bss : 0xc0f7c000 - 0xc0fd12e8   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -332,19 +332,19 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on c8000600.sdhci [c8000600.sdhci] using ADMA
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-26 22:41:06 UTC (1419633666)
+rtc-em3027 2-0056: setting system clock to 2014-12-26 22:42:28 UTC (1419633748)
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
-mmc1: new high speed SD card at address b368
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
- mmcblk1: p1 p2
+mmc1: new high speed SD card at address b368
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 devtmpfs: mounted
-Freeing unused kernel memory: 1028K (c0d7d000 - c0e7e000)
+Freeing unused kernel memory: 1020K (c0d7d000 - c0e7c000)
+ mmcblk1: p1 p2
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.290233] asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008
\ No newline at end of file
+root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2015-09-14 20:23:33.550207072 -0700
+++ /tmp/foob	2015-09-14 20:23:33.550207072 -0700
@@ -47,18 +47,18 @@
 	 ####################
 	 4 MiB/s
 done
-Bytes transferred = 5054632 (4d20a8 hex)
+Bytes transferred = 5054608 (4d2090 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 4 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21202 (52d2 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x4d20a8 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x4d2090 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102d1
@@ -69,7 +69,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.3.0-rc1-next-20150914-41348-g91ad81b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Sep 13 20:18:30 PDT 2015
+Linux version 4.3.0-rc1-next-20150915-41518-gb1ab908 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Sep 14 20:17:52 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -99,7 +99,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc091e670   (9306 kB)
+      .text : 0xc0008000 - 0xc091e690   (9306 kB)
       .init : 0xc091f000 - 0xc0970000   ( 324 kB)
       .data : 0xc0970000 - 0xc09c5400   ( 341 kB)
        .bss : 0xc09c8000 - 0xc0a22d20   ( 364 kB)
@@ -306,28 +306,28 @@
 sit: IPv6 over IPv4 tunneling driver
 NET: Registered protocol family 17
 NET: Registered protocol family 15
-mmc0: host does not support reading read-only switch, assuming write-enable
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
 can: controller area network core (rev 20120528 abi 9)
-mmc0: new high speed SDHC card at address b368
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 NET: Registered protocol family 29
 can: raw protocol (rev 20120528)
 can: broadcast manager protocol (rev 20120528 t)
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
 can: netlink gateway (rev 20130117) max_hops=1
- mmcblk0: p1
 Bluetooth: RFCOMM socket layer initialized
 Bluetooth: RFCOMM ver 1.11
 Bluetooth: BNEP (Ethernet Emulation) ver 1.3
 Bluetooth: BNEP socket layer initialized
 Bluetooth: HIDP (Human Interface Emulation) ver 1.2
+mmc0: host does not support reading read-only switch, assuming write-enable
 Bluetooth: HIDP socket layer initialized
 Registering SWP/SWPB emulation handler
+mmc0: new high speed SDHC card at address b368
 tegra-phy c5004000.usb-phy: no vbus regulator
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 64, io mem 0xc5000000
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
+ mmcblk0: p1
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
@@ -343,18 +343,18 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on c8000600.sdhci [c8000600.sdhci] using ADMA
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-26 22:41:24 UTC (1419633684)
+rtc-em3027 2-0056: setting system clock to 2014-12-26 22:42:47 UTC (1419633767)
 ALSA device list:
   #0: tegra-trimslice
 EXT2-fs (mmcblk0p1): error: couldn't mount because of unsupported optional features (4)
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
- mmcblk1: p1 p2
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 VFS: Mounted root (ext3 filesystem) on device 179:1.
+usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
+ mmcblk1: p1 p2
 devtmpfs: mounted
 Freeing unused kernel memory: 324K (c091f000 - c0970000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-09-14 20:23:33.558207065 -0700
+++ /tmp/foob	2015-09-14 20:23:33.558207065 -0700
@@ -38,20 +38,20 @@
 	 #################################################################
 	 #################################################################
 	 ################################################
-	 3.4 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 6426312 (620ec8 hex)
+Bytes transferred = 6424608 (620820 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.5 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 51539 (c953 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x620ec8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x620820 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff952 ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.3.0-rc1-next-20150914-41348-g91ad81b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Sep 13 20:18:12 PDT 2015
+Linux version 4.3.0-rc1-next-20150915-41518-gb1ab908 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Sep 14 20:17:36 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -67,7 +67,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f793c0, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0f770c0, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -80,7 +80,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998804K/2096128K available (8103K kernel code, 1012K rwdata, 3624K rodata, 1028K init, 340K bss, 31788K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1998812K/2096128K available (8103K kernel code, 1012K rwdata, 3624K rodata, 1020K init, 340K bss, 31780K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,10 +88,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d7cdf0   (11732 kB)
-      .init : 0xc0d7d000 - 0xc0e7e000   (1028 kB)
-      .data : 0xc0e7e000 - 0xc0f7b3a0   (1013 kB)
-       .bss : 0xc0f7e000 - 0xc0fd32e8   ( 341 kB)
+      .text : 0xc0208000 - 0xc0d7cd90   (11732 kB)
+      .init : 0xc0d7d000 - 0xc0e7c000   (1020 kB)
+      .data : 0xc0e7c000 - 0xc0f790a0   (1013 kB)
+       .bss : 0xc0f7c000 - 0xc0fd12e8   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -272,11 +272,11 @@
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 1.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 1.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 128 KiB
+Primary GPT is invalid, using alternate GPT.
+ mmcblk0: p1 p2 p3 p4 p5
 tegra-pcie 3000.pcie-controller: 2x3 configuration
 sys_3v3,vdd_3v3_alw: supplied by vdd_5v_in
 sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
-Primary GPT is invalid, using alternate GPT.
- mmcblk0: p1 p2 p3 p4 p5
 tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
 tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -355,7 +355,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1028K (c0d7d000 - c0e7e000)
+Freeing unused kernel memory: 1020K (c0d7d000 - c0e7c000)
 usb 2-1: new high-speed USB device number 2 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2015-09-14 20:23:33.562207062 -0700
+++ /tmp/foob	2015-09-14 20:23:33.566207058 -0700
@@ -39,18 +39,18 @@
 	 ####################
 	 3.3 MiB/s
 done
-Bytes transferred = 5054632 (4d20a8 hex)
+Bytes transferred = 5054608 (4d2090 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.3 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 51539 (c953 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4d20a8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4d2090 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff952 ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.3.0-rc1-next-20150914-41348-g91ad81b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Sep 13 20:18:30 PDT 2015
+Linux version 4.3.0-rc1-next-20150915-41518-gb1ab908 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Sep 14 20:17:52 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -90,7 +90,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc091e670   (9306 kB)
+      .text : 0xc0008000 - 0xc091e690   (9306 kB)
       .init : 0xc091f000 - 0xc0970000   ( 324 kB)
       .data : 0xc0970000 - 0xc09c5400   ( 341 kB)
        .bss : 0xc09c8000 - 0xc0a22d20   ( 364 kB)
@@ -253,8 +253,6 @@
 can: controller area network core (rev 20120528 abi 9)
 NET: Registered protocol family 29
 can: raw protocol (rev 20120528)
-can: broadcast manager protocol (rev 20120528 t)
-can: netlink gateway (rev 20130117) max_hops=1
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 SEM16G 14.8 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 1.00 MiB
@@ -262,6 +260,8 @@
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 128 KiB
 Primary GPT is invalid, using alternate GPT.
  mmcblk0: p1 p2 p3 p4 p5
+can: broadcast manager protocol (rev 20120528 t)
+can: netlink gateway (rev 20130117) max_hops=1
 Bluetooth: RFCOMM socket layer initialized
 Bluetooth: RFCOMM ver 1.11
 Bluetooth: BNEP (Ethernet Emulation) ver 1.3
