Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2017-09-12 21:43:31.355283803 -0700
+++ /tmp/foob	2017-09-12 21:43:31.355283803 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-next-20170912-317999-g0d71e2d4aa14 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Mon Sep 11 21:39:37 PDT 2017
+Linux version 4.13.0-next-20170913-318059-g6f20b7a58cb9 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Tue Sep 12 21:40:36 PDT 2017
 Boot CPU: AArch64 Processor [411fd070]
 Machine model: linux,dummy-virt
 Memory limited to 1024MB
@@ -181,7 +181,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2017-09-12 04:40:47 UTC (1505191247)
+rtc-pl031 9010000.pl031: setting system clock to 2017-09-13 04:41:42 UTC (1505277702)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-09-12 21:43:31.391283969 -0700
+++ /tmp/foob	2017-09-12 21:43:31.391283969 -0700
@@ -39,21 +39,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ####################################
-	 3.1 MiB/s
+	 #####################################
+	 3.2 MiB/s
 done
-Bytes transferred = 8159752 (7c8208 hex)
+Bytes transferred = 8163848 (7c9208 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3.3 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 35036 (88dc hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x7c8200 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x7c9200 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8db ... OK
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-next-20170912-199758-g0d71e2d (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Mon Sep 11 21:34:26 PDT 2017
+Linux version 4.13.0-next-20170913-199818-g6f20b7a (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Tue Sep 12 21:35:43 PDT 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -72,7 +72,7 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c158e2c0, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c158eb80, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -86,7 +86,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993480K/2096128K available (10240K kernel code, 1625K rwdata, 4508K rodata, 2048K init, 405K bss, 37112K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993476K/2096128K available (10240K kernel code, 1628K rwdata, 4508K rodata, 2048K init, 405K bss, 37116K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -96,8 +96,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1596610   (1626 kB)
-       .bss : 0xc159fbc0 - 0xc160537c   ( 406 kB)
+      .data : 0xc1400000 - 0xc1597028   (1629 kB)
+       .bss : 0xc15a0bc0 - 0xc160637c   ( 406 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -136,7 +136,7 @@
 SMP: Total of 4 processors activated (96.00 BogoMIPS).
 CPU: All CPU(s) started in SVC mode.
 devtmpfs: initialized
-Built 1 zonelists, mobility grouping on.  Total pages: 514754
+Built 1 zonelists, mobility grouping on.  Total pages: 514753
 VFP support v0.3: implementor 41 architecture 4 part 30 variant f rev 0
 clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 19112604462750000 ns
 futex hash table entries: 1024 (order: 4, 65536 bytes)
@@ -382,8 +382,8 @@
 mmc0: new high speed SDHC card at address aaaa
 mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 usb2_vbus: supplied by vdd-sys-5v0
-VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
  mmcblk0: p1 p2 p3
+VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
 mmc1: new high speed MMC card at address 0001
@@ -412,7 +412,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-03 11:27:35 UTC (949577255)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-04 11:28:53 UTC (949663733)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -424,5 +424,5 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.069434] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.089083] usb 1-1: new high-speed USB device number 2 using tegra-ehci
 asix 1-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0d:b5:16
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2017-09-12 21:43:31.415284080 -0700
+++ /tmp/foob	2017-09-12 21:43:31.415284080 -0700
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra114 (Dalmore) # 
 Tegra114 (Dalmore) # usb reset
 (Re)start USB...
@@ -27,29 +27,21 @@
 Tegra114 (Dalmore) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=7369c667-ff51-ec4a-29cd-baabf2fbe346 rootwait gpt debug earlyprintk init=/bin/bash'
 Tegra114 (Dalmore) # setenv ipaddr 192.168.100.100; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra114 (Dalmore) # tftpboot 0x81000000 zImage.tegra_defconfig
-Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'zImage.tegra_defconfig'.
 Load address: 0x81000000
-Loading: *EHCI timed out on TD - token=0x8008d80
-T EHCI timed out on TD - token=0x8008d80
-T EHCI timed out on TD - token=0x8008d80
-T EHCI timed out on TD - token=0x8008d80
-T EHCI timed out on TD - token=0x8008d80
-T EHCI timed out on TD - token=0x8008d80
-T #################################################################
+Loading: *#################################################################
 	 #################################################################
 	 #################################################################
 	 #################################################################
 	 #################################################################
 	 #################################################################
 	 #######################################
-	 192.4 KiB/s
+	 2.9 MiB/s
 done
-Bytes transferred = 6288104 (5ff2e8 hex)
+Bytes transferred = 6292568 (600458 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
-Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
@@ -59,7 +51,7 @@
 done
 Bytes transferred = 35036 (88dc hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5ff2e8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x600458 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8db ... OK
@@ -68,7 +60,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-next-20170912-199758-g0d71e2d (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Sep 11 21:34:53 PDT 2017
+Linux version 4.13.0-next-20170913-199818-g6f20b7a (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Tue Sep 12 21:35:41 PDT 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -78,7 +70,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0fb7b80, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0fb8440, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -92,7 +84,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1997500K/2096128K available (9216K kernel code, 778K rwdata, 2796K rodata, 1024K init, 300K bss, 33092K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1997496K/2096128K available (9216K kernel code, 781K rwdata, 2796K rodata, 1024K init, 300K bss, 33096K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -102,8 +94,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
-      .data : 0xc0f00000 - 0xc0fc2b48   ( 779 kB)
-       .bss : 0xc0fcc52c - 0xc10176bc   ( 301 kB)
+      .data : 0xc0f00000 - 0xc0fc3560   ( 782 kB)
+       .bss : 0xc0fcd52c - 0xc10186bc   ( 301 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -138,7 +130,7 @@
 SMP: Total of 4 processors activated (96.00 BogoMIPS).
 CPU: All CPU(s) started in SVC mode.
 devtmpfs: initialized
-Built 1 zonelists, mobility grouping on.  Total pages: 515759
+Built 1 zonelists, mobility grouping on.  Total pages: 515758
 VFP support v0.3: implementor 41 architecture 4 part 30 variant f rev 0
 clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 19112604462750000 ns
 futex hash table entries: 1024 (order: 4, 65536 bytes)
@@ -352,81 +344,3 @@
 sdhci-tegra 78000400.sdhci: Got CD GPIO
 sdhci-tegra 78000400.sdhci: Got WP GPIO
 mmc0: Invalid maximum block size, assuming 512 bytes
-mmc0: SDHCI controller on 78000400.sdhci [78000400.sdhci] using ADMA
-mmc1: Invalid maximum block size, assuming 512 bytes
-mmc1: SDHCI controller on 78000600.sdhci [78000600.sdhci] using ADMA
-usbcore: registered new interface driver usbhid
-usbhid: USB HID core driver
-iommu: Adding device snd-soc-dummy to group 57
-mmc0: new high speed SDHC card at address aaaa
-mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
- mmcblk0: p1 p2 p3
-mmc1: new high speed MMC card at address 0001
-mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
-mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
-mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
-mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
- mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
-iommu: Adding device 70080400.i2s to group 58
-tegra30-i2s 70080400.i2s: DMA channels sourced from device 70080000.ahub
-tegra-snd-rt5640 sound: rt5640-aif1 <-> 70080400.i2s mapping ok
-input: NVIDIA Tegra Dalmore Headphones as /devices/soc0/sound/sound/card0/input0
-NET: Registered protocol family 10
-Segment Routing with IPv6
-mip6: Mobile IPv6
-sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
-NET: Registered protocol family 17
-NET: Registered protocol family 15
-can: controller area network core (rev 20170425 abi 9)
-NET: Registered protocol family 29
-can: raw protocol (rev 20170425)
-can: broadcast manager protocol (rev 20170425 t)
-can: netlink gateway (rev 20170425) max_hops=1
-Bluetooth: RFCOMM socket layer initialized
-Bluetooth: RFCOMM ver 1.11
-Bluetooth: BNEP (Ethernet Emulation) ver 1.3
-Bluetooth: BNEP socket layer initialized
-Bluetooth: HIDP (Human Interface Emulation) ver 1.2
-Bluetooth: HIDP socket layer initialized
-Key type dns_resolver registered
-Registering SWP/SWPB emulation handler
-usb1_vbus: supplied by vdd-sys-5v0
-usb2_vbus: supplied by vdd-sys-5v0
-VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
-[drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
-[drm] No driver support for vblank timestamp query.
-Console: switching to colour frame buffer device 240x75
-drm drm: fb0:  frame buffer device
-[drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
-tegra-ehci 7d008000.usb: EHCI Host Controller
-tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 1
-alloc_contig_range: [fbc4b, fbc4c) PFNs busy
-alloc_contig_range: [fbc60, fbc61) PFNs busy
-alloc_contig_range: [fbc61, fbc62) PFNs busy
-alloc_contig_range: [fbc62, fbc63) PFNs busy
-alloc_contig_range: [fbc63, fbc64) PFNs busy
-alloc_contig_range: [fbc64, fbc65) PFNs busy
-alloc_contig_range: [fbc65, fbc66) PFNs busy
-alloc_contig_range: [fbc66, fbc67) PFNs busy
-alloc_contig_range: [fbc67, fbc68) PFNs busy
-alloc_contig_range: [fbc68, fbc69) PFNs busy
-tegra-ehci 7d008000.usb: irq 340, io mem 0x7d008000
-tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
-hub 1-0:1.0: USB hub found
-hub 1-0:1.0: 1 port detected
-input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-03 11:30:39 UTC (949577439)
-dvdd_ts: disabling
-usb1_vbus: disabling
-vdd_cam_1v8_reg: disabling
-ALSA device list:
-  #0: NVIDIA Tegra Dalmore
-EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
-EXT4-fs (mmcblk1p1): recovery complete
-EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:17.
-devtmpfs: mounted
-Freeing unused kernel memory: 1024K
-bash: cannot set terminal process group (-1): Inappropriate ioctl for device
-bash: no job control in this shell
-root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-09-12 21:43:31.451284247 -0700
+++ /tmp/foob	2017-09-12 21:43:31.451284247 -0700
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MMC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -34,21 +34,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ####################################
-	 3.7 MiB/s
+	 #####################################
+	 4 MiB/s
 done
-Bytes transferred = 8159752 (7c8208 hex)
+Bytes transferred = 8163848 (7c9208 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.4 MiB/s
+	 4 MiB/s
 done
 Bytes transferred = 67173 (10665 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x7c8200 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x7c9200 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013664
@@ -56,7 +56,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-next-20170912-199758-g0d71e2d (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Mon Sep 11 21:34:26 PDT 2017
+Linux version 4.13.0-next-20170913-199818-g6f20b7a (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Tue Sep 12 21:35:43 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -67,7 +67,7 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c158e2c0, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c158eb80, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -81,7 +81,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993360K/2096128K available (10240K kernel code, 1625K rwdata, 4508K rodata, 2048K init, 405K bss, 37232K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993356K/2096128K available (10240K kernel code, 1628K rwdata, 4508K rodata, 2048K init, 405K bss, 37236K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -91,8 +91,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1596610   (1626 kB)
-       .bss : 0xc159fbc0 - 0xc160537c   ( 406 kB)
+      .data : 0xc1400000 - 0xc1597028   (1629 kB)
+       .bss : 0xc15a0bc0 - 0xc160637c   ( 406 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -131,7 +131,7 @@
 SMP: Total of 4 processors activated (96.00 BogoMIPS).
 CPU: All CPU(s) started in SVC mode.
 devtmpfs: initialized
-Built 1 zonelists, mobility grouping on.  Total pages: 514724
+Built 1 zonelists, mobility grouping on.  Total pages: 514723
 VFP support v0.3: implementor 41 architecture 4 part 30 variant f rev 0
 clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 19112604462750000 ns
 futex hash table entries: 1024 (order: 4, 65536 bytes)
@@ -401,12 +401,6 @@
 +5V_SATA: supplied by +5V_SYS
 +12V_SATA: supplied by +VDD_MUX
  usb2-0: usb2-0 supply vbus not found, using dummy regulator
-mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
- mmcblk0: p1
  usb2-1: usb2-1 supply vbus not found, using dummy regulator
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
 tegra-pcie 1003000.pcie: 2x1, 1x1 configuration
@@ -427,11 +421,17 @@
 as3722-ldo7: supplied by +1.8V_VDDIO
 +1.05V_RUN_CAM_REAR: Bringing 0uV into 1050000-1050000uV
 as3722-ldo9: supplied by +5V_SYS
+mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 +3.3V_RUN_TOUCH: Bringing 0uV into 2800000-2800000uV
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 as3722-ldo10: supplied by +5V_SYS
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 +2.8V_RUN_CAM_AF: Bringing 0uV into 2800000-2800000uV
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
 as3722-ldo11: supplied by +3.3V_RUN
 +1.8V_RUN_VPP_FUSE: Bringing 0uV into 1800000-1800000uV
+ mmcblk0: p1
 tegra-ahci 70027000.sata: AHCI 0001.0301 32 slots 2 ports 3 Gbps 0x1 impl platform mode
 tegra-ahci 70027000.sata: flags: 64bit ncq sntf pm led pio slum part sadm sds apst 
 scsi host0: tegra-ahci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2017-09-12 21:43:31.463284302 -0700
+++ /tmp/foob	2017-09-12 21:43:31.463284302 -0700
@@ -33,9 +33,9 @@
 	 #################################################################
 	 #################################################################
 	 #######################################
-	 3.9 MiB/s
+	 4 MiB/s
 done
-Bytes transferred = 6288104 (5ff2e8 hex)
+Bytes transferred = 6292568 (600458 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -46,7 +46,7 @@
 done
 Bytes transferred = 67173 (10665 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5ff2e8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x600458 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013664
@@ -55,7 +55,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-next-20170912-199758-g0d71e2d (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Sep 11 21:34:53 PDT 2017
+Linux version 4.13.0-next-20170913-199818-g6f20b7a (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Tue Sep 12 21:35:41 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -65,7 +65,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0fb7b80, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0fb8440, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -79,7 +79,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1997380K/2096128K available (9216K kernel code, 778K rwdata, 2796K rodata, 1024K init, 300K bss, 33212K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1997376K/2096128K available (9216K kernel code, 781K rwdata, 2796K rodata, 1024K init, 300K bss, 33216K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -89,8 +89,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
-      .data : 0xc0f00000 - 0xc0fc2b48   ( 779 kB)
-       .bss : 0xc0fcc52c - 0xc10176bc   ( 301 kB)
+      .data : 0xc0f00000 - 0xc0fc3560   ( 782 kB)
+       .bss : 0xc0fcd52c - 0xc10186bc   ( 301 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -125,7 +125,7 @@
 SMP: Total of 4 processors activated (96.00 BogoMIPS).
 CPU: All CPU(s) started in SVC mode.
 devtmpfs: initialized
-Built 1 zonelists, mobility grouping on.  Total pages: 515729
+Built 1 zonelists, mobility grouping on.  Total pages: 515728
 VFP support v0.3: implementor 41 architecture 4 part 30 variant f rev 0
 clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 19112604462750000 ns
 futex hash table entries: 1024 (order: 4, 65536 bytes)
@@ -538,7 +538,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:11 UTC (946684931)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:37 UTC (946684957)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-09-12 21:43:31.487284411 -0700
+++ /tmp/foob	2017-09-12 21:43:31.487284411 -0700
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -42,21 +42,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ####################################
-	 3.4 MiB/s
+	 #####################################
+	 2.8 MiB/s
 done
-Bytes transferred = 8159752 (7c8208 hex)
+Bytes transferred = 8163848 (7c9208 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.4 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 68203 (10a6b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x7c8200 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x7c9200 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a6a
@@ -64,7 +64,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-next-20170912-199758-g0d71e2d (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Mon Sep 11 21:34:26 PDT 2017
+Linux version 4.13.0-next-20170913-199818-g6f20b7a (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Tue Sep 12 21:35:43 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -76,7 +76,7 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c158e2c0, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c158eb80, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -90,7 +90,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1994376K/2097148K available (10240K kernel code, 1625K rwdata, 4508K rodata, 2048K init, 405K bss, 37236K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1994372K/2097148K available (10240K kernel code, 1628K rwdata, 4508K rodata, 2048K init, 405K bss, 37240K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -100,8 +100,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1596610   (1626 kB)
-       .bss : 0xc159fbc0 - 0xc160537c   ( 406 kB)
+      .data : 0xc1400000 - 0xc1597028   (1629 kB)
+       .bss : 0xc15a0bc0 - 0xc160637c   ( 406 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -140,7 +140,7 @@
 SMP: Total of 4 processors activated (96.00 BogoMIPS).
 CPU: All CPU(s) started in SVC mode.
 devtmpfs: initialized
-Built 1 zonelists, mobility grouping on.  Total pages: 514978
+Built 1 zonelists, mobility grouping on.  Total pages: 514977
 VFP support v0.3: implementor 41 architecture 4 part 30 variant f rev 0
 clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 19112604462750000 ns
 futex hash table entries: 1024 (order: 4, 65536 bytes)
@@ -414,10 +414,11 @@
 mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
+elan_i2c 1-0015: failed to read report data: -121
+elan_i2c 1-0015: failed to read report data: -121
 +5V_HDMI_CON: supplied by +5V_SYS
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
  usb3-1: usb3-1 supply vbus not found, using dummy regulator
-elan_i2c 1-0015: failed to read report data: -121
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
 sdhci-tegra 700b0000.sdhci: allocated mmc-pwrseq
@@ -446,7 +447,7 @@
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 06:26:43 UTC (946708003)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:25 UTC (946684825)
 +1.8V_RUN_CAM: disabling
 +3.3V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2017-09-12 21:43:31.499284468 -0700
+++ /tmp/foob	2017-09-12 21:43:31.503284486 -0700
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -36,25 +36,26 @@
 Load address: 0x82000000
 Loading: *#################################################################
 	 #################################################################
-	 #################################################################
+	 ####EHCI timed out on TD - token=0x8008d80
+T T #############################################################
 	 #################################################################
 	 #################################################################
 	 #################################################################
 	 #######################################
-	 3.5 MiB/s
+	 421.9 KiB/s
 done
-Bytes transferred = 6288104 (5ff2e8 hex)
+Bytes transferred = 6292568 (600458 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.4 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 68203 (10a6b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5ff2e8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x600458 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a6a
@@ -63,7 +64,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-next-20170912-199758-g0d71e2d (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Sep 11 21:34:53 PDT 2017
+Linux version 4.13.0-next-20170913-199818-g6f20b7a (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Tue Sep 12 21:35:41 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -74,7 +75,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c0fb7b80, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0fb8440, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -88,7 +89,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998396K/2097148K available (9216K kernel code, 778K rwdata, 2796K rodata, 1024K init, 300K bss, 33216K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1998392K/2097148K available (9216K kernel code, 781K rwdata, 2796K rodata, 1024K init, 300K bss, 33220K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -98,8 +99,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
-      .data : 0xc0f00000 - 0xc0fc2b48   ( 779 kB)
-       .bss : 0xc0fcc52c - 0xc10176bc   ( 301 kB)
+      .data : 0xc0f00000 - 0xc0fc3560   ( 782 kB)
+       .bss : 0xc0fcd52c - 0xc10186bc   ( 301 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -134,7 +135,7 @@
 SMP: Total of 4 processors activated (96.00 BogoMIPS).
 CPU: All CPU(s) started in SVC mode.
 devtmpfs: initialized
-Built 1 zonelists, mobility grouping on.  Total pages: 515983
+Built 1 zonelists, mobility grouping on.  Total pages: 515982
 VFP support v0.3: implementor 41 architecture 4 part 30 variant f rev 0
 clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 19112604462750000 ns
 futex hash table entries: 1024 (order: 4, 65536 bytes)
@@ -313,130 +314,130 @@
 kernel BUG at drivers/platform/chrome/cros_ec_proto.c:34!
 Internal error: Oops - BUG: 0 [#1] PREEMPT SMP ARM
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.13.0-next-20170912-199758-g0d71e2d #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.13.0-next-20170913-199818-g6f20b7a #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 task: ee891740 task.stack: ee8a6000
 PC is at cros_ec_prepare_tx+0x168/0x16c
 LR is at cros_ec_pkt_xfer_spi+0x18/0x474
-pc : [<c06d6210>]    lr : [<c0543dd8>]    psr: 20000013
+pc : [<c06d763c>]    lr : [<c0545204>]    psr: 20000013
 sp : ee8a7bd8  ip : 00000000  fp : 00000000
-r10: c0543dc0  r9 : 00000000  r8 : eefe503c
-r7 : ee293000  r6 : ee293000  r5 : ee293000  r4 : ee249d90
-r3 : 0000ffff  r2 : 00000000  r1 : 0000ffff  r0 : ee249d90
+r10: c05451ec  r9 : 00000000  r8 : eefe503c
+r7 : ee291000  r6 : ee291000  r5 : ee291000  r4 : ee248d90
+r3 : 0000ffff  r2 : 00000000  r1 : 0000ffff  r0 : ee248d90
 Flags: nzCv  IRQs on  FIQs on  Mode SVC_32  ISA ARM  Segment none
 Control: 10c5387d  Table: 8000406a  DAC: 00000051
 Process swapper/0 (pid: 1, stack limit = 0xee8a6210)
 Stack: (0xee8a7bd8 to 0xee8a8000)
-7bc0:                                                       ee249d90 ee293000
-7be0: ee293090 c0543dd8 c097537c 00000004 ee8a7c5c ee8a7c5c ee295c00 00000000
-7c00: c059021c 00000000 00000008 00000008 00000000 ee8a7c14 ee8a7c14 00000000
-7c20: ee8a7c20 ee8a7c20 ee293050 ee293780 00000008 00000000 00000000 00000000
-7c40: c0e90c50 c02383d0 40002800 000003e0 eef8cc50 c023a840 002dc6c0 ee8a7bf0
-7c60: ee8a7bf0 00000000 c06d6040 005c5260 3b9aca00 000f4240 00000000 ee8a7ca8
-7c80: 00000001 00000501 c0543dc0 c0979c0c 00000000 ee2937c0 a0000013 c06d6040
-7ca0: ee293000 ee249d90 ee293000 ee295c00 ee293000 eefe503c 00000000 c0543dc0
-7cc0: 00000000 c06d5f9c 0000400b ee249d90 ee293000 ee295c00 0000000c eefe503c
-7ce0: c0ff7e7c c06d63b8 00000010 ee249d90 ee295c00 00000000 00000000 eefe503c
-7d00: c0f6af18 c0543864 ee8a7d14 ee295c00 00000000 00000001 ee295c00 ee249d90
-7d20: ee293090 c0543c14 00000000 00000000 00000000 000007d0 ee295c00 00000000
-7d40: c0f6af08 00000000 00000000 c0590c60 ee295c00 c1009204 c1009208 c051cb20
-7d60: 00000000 ee8a7d98 c051ccb0 00000001 c10091e0 00000000 00000000 c051b140
-7d80: ee9f7170 ee2385b8 ee295c00 ee295c34 c0f71bdc c051c860 ee295c00 00000001
-7da0: 00000000 ee295c00 ee295c00 c0f71bdc ee073000 c051bea4 ee295c00 00000000
-7dc0: ee295c08 c051a4e8 ee295c00 c0593b64 00000010 ee295c00 ee295c00 ee073000
-7de0: 00000000 eea8be10 eefe508c 00000001 00000000 c0590ab4 002dc6c0 ee073000
-7e00: eefe503c 00000000 ee295c00 c0592a50 00000000 014000c0 00000000 c059243c
-7e20: 00000063 002dc6c0 ee073000 ee293350 eea8be00 eea8be10 40000000 00000153
-7e40: 00000000 c0592f10 eefe4d2c ee073348 ee073000 eea8be00 eea8be10 c0593a9c
-7e60: 00000000 ee073348 ffffffed eea8be10 fffffdfb c0f7253c 00000000 00000000
-7e80: c0f7253c c051e410 c051e3c0 eea8be10 c1009204 c1009208 00000000 c051cb20
-7ea0: eea8be10 c0f7253c eea8be44 00000000 00000111 c0e6a83c c0e005a4 c051cc78
-7ec0: 00000000 c0f7253c c051cbcc c051b1dc ee8c955c eea1c934 c0f7253c ee28be80
-7ee0: c0f694c8 c051c078 c0c1c8e8 c0f7253c 00000006 c0f7253c 00000006 c0e380d0
-7f00: 00000000 c051d7f0 c051e068 c0e82c84 00000006 c0101b8c 00000000 c0cba2fc
-7f20: 00000100 efffcb45 c0cba2fc 00000111 efffcb53 c013e410 c0cb8da8 00000000
-7f40: 00000006 00000006 efffcb45 efffcb4a c0f0dcd0 efffcac0 00000006 c0e6a82c
-7f60: c0e82c84 00000006 c0e6a830 c0fcc540 00000111 c0e00da8 00000006 00000006
-7f80: 00000000 c0e005a4 00000000 c0973124 00000000 00000000 00000000 00000000
-7fa0: 00000000 c097312c 00000000 c0107ce0 00000000 00000000 00000000 00000000
+7bc0:                                                       ee248d90 ee291000
+7be0: ee291090 c0545204 c09768a4 00000004 ee8a7c5c ee8a7c5c ee293c00 00000000
+7c00: c0591648 00000000 00000008 00000008 00000000 ee8a7c14 ee8a7c14 00000000
+7c20: ee8a7c20 ee8a7c20 ee291050 ee291780 00000008 00000000 00000000 00000000
+7c40: c0e91c50 c02383c8 40002800 000003e0 eef8cc50 c023a838 002dc6c0 ee8a7bf0
+7c60: ee8a7bf0 00000000 c06d746c 005c5220 3b9aca00 000f4240 00000000 ee8a7ca8
+7c80: 00000001 00000501 c05451ec c097b134 00000000 ee2917c0 a0000013 c06d746c
+7ca0: ee291000 ee248d90 ee291000 ee293c00 ee291000 eefe503c 00000000 c05451ec
+7cc0: 00000000 c06d73c8 0000400b ee248d90 ee291000 ee293c00 0000000c eefe503c
+7ce0: c0ff8e7c c06d77e4 00000010 ee248d90 ee293c00 00000000 00000000 eefe503c
+7d00: c0f6b7d8 c0544c90 ee8a7d14 ee293c00 00000000 00000001 ee293c00 ee248d90
+7d20: ee291090 c0545040 00000000 00000000 00000000 000007d0 ee293c00 00000000
+7d40: c0f6b7c8 00000000 00000000 c059208c ee293c00 c100a204 c100a208 c051df4c
+7d60: 00000000 ee8a7d98 c051e0dc 00000001 c100a1e0 00000000 00000000 c051c56c
+7d80: ee9f7170 ee2385b8 ee293c00 ee293c34 c0f7249c c051dc8c ee293c00 00000001
+7da0: 00000000 ee293c00 ee293c00 c0f7249c ee071000 c051d2d0 ee293c00 00000000
+7dc0: ee293c08 c051b914 ee293c00 c0594f90 00000010 ee293c00 ee293c00 ee071000
+7de0: 00000000 eea8be10 eefe508c 00000001 00000000 c0591ee0 002dc6c0 ee071000
+7e00: eefe503c 00000000 ee293c00 c0593e7c 00000000 014000c0 00000000 c0593868
+7e20: 00000063 002dc6c0 ee071000 ee291350 eea8be00 eea8be10 40000000 00000153
+7e40: 00000000 c059433c eefe4d2c ee071348 ee071000 eea8be00 eea8be10 c0594ec8
+7e60: 00000000 ee071348 ffffffed eea8be10 fffffdfb c0f72dfc 00000000 00000000
+7e80: c0f72dfc c051f83c c051f7ec eea8be10 c100a204 c100a208 00000000 c051df4c
+7ea0: eea8be10 c0f72dfc eea8be44 00000000 00000111 c0e6b83c c0e005a4 c051e0a4
+7ec0: 00000000 c0f72dfc c051dff8 c051c608 ee8c955c eea1c934 c0f72dfc ee28ae80
+7ee0: c0f69d88 c051d4a4 c0c1cc6c c0f72dfc 00000006 c0f72dfc 00000006 c0e38818
+7f00: 00000000 c051ec1c c051f494 c0e83c84 00000006 c0101b8c 00000000 c0cba388
+7f20: 00000100 efffcb45 c0cba388 00000111 efffcb53 c013e410 c0cb8e34 00000000
+7f40: 00000006 00000006 efffcb45 efffcb4a c0f0dcd0 efffcac0 00000006 c0e6b82c
+7f60: c0e83c84 00000006 c0e6b830 c0fcd540 00000111 c0e00da8 00000006 00000006
+7f80: 00000000 c0e005a4 00000000 c0974684 00000000 00000000 00000000 00000000
+7fa0: 00000000 c097468c 00000000 c0107ce0 00000000 00000000 00000000 00000000
 7fc0: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
 7fe0: 00000000 00000000 00000000 00000000 00000013 00000000 00000000 00000000
-[<c06d6210>] (cros_ec_prepare_tx) from [<c0543dd8>] (cros_ec_pkt_xfer_spi+0x18/0x474)
-[<c0543dd8>] (cros_ec_pkt_xfer_spi) from [<c06d5f9c>] (send_command+0x20/0xd8)
-[<c06d5f9c>] (send_command) from [<c06d63b8>] (cros_ec_query_all+0x1a4/0x398)
-[<c06d63b8>] (cros_ec_query_all) from [<c0543864>] (cros_ec_register+0x94/0x21c)
-[<c0543864>] (cros_ec_register) from [<c0543c14>] (cros_ec_spi_probe+0x108/0x150)
-[<c0543c14>] (cros_ec_spi_probe) from [<c0590c60>] (spi_drv_probe+0x7c/0xa8)
-[<c0590c60>] (spi_drv_probe) from [<c051cb20>] (driver_probe_device+0x238/0x2e4)
-[<c051cb20>] (driver_probe_device) from [<c051b140>] (bus_for_each_drv+0x44/0x8c)
-[<c051b140>] (bus_for_each_drv) from [<c051c860>] (__device_attach+0x9c/0x100)
-[<c051c860>] (__device_attach) from [<c051bea4>] (bus_probe_device+0x84/0x8c)
-[<c051bea4>] (bus_probe_device) from [<c051a4e8>] (device_add+0x380/0x528)
-[<c051a4e8>] (device_add) from [<c0590ab4>] (spi_add_device+0x88/0x130)
-[<c0590ab4>] (spi_add_device) from [<c0592a50>] (spi_register_controller+0x428/0x8b4)
-[<c0592a50>] (spi_register_controller) from [<c0592f10>] (devm_spi_register_controller+0x34/0x70)
-[<c0592f10>] (devm_spi_register_controller) from [<c0593a9c>] (tegra_spi_probe+0x298/0x35c)
-[<c0593a9c>] (tegra_spi_probe) from [<c051e410>] (platform_drv_probe+0x50/0xb0)
-[<c051e410>] (platform_drv_probe) from [<c051cb20>] (driver_probe_device+0x238/0x2e4)
-[<c051cb20>] (driver_probe_device) from [<c051cc78>] (__driver_attach+0xac/0xb0)
-[<c051cc78>] (__driver_attach) from [<c051b1dc>] (bus_for_each_dev+0x54/0x88)
-[<c051b1dc>] (bus_for_each_dev) from [<c051c078>] (bus_add_driver+0xe4/0x1f0)
-[<c051c078>] (bus_add_driver) from [<c051d7f0>] (driver_register+0x78/0xf4)
-[<c051d7f0>] (driver_register) from [<c0101b8c>] (do_one_initcall+0x3c/0x168)
+[<c06d763c>] (cros_ec_prepare_tx) from [<c0545204>] (cros_ec_pkt_xfer_spi+0x18/0x474)
+[<c0545204>] (cros_ec_pkt_xfer_spi) from [<c06d73c8>] (send_command+0x20/0xd8)
+[<c06d73c8>] (send_command) from [<c06d77e4>] (cros_ec_query_all+0x1a4/0x398)
+[<c06d77e4>] (cros_ec_query_all) from [<c0544c90>] (cros_ec_register+0x94/0x21c)
+[<c0544c90>] (cros_ec_register) from [<c0545040>] (cros_ec_spi_probe+0x108/0x150)
+[<c0545040>] (cros_ec_spi_probe) from [<c059208c>] (spi_drv_probe+0x7c/0xa8)
+[<c059208c>] (spi_drv_probe) from [<c051df4c>] (driver_probe_device+0x238/0x2e4)
+[<c051df4c>] (driver_probe_device) from [<c051c56c>] (bus_for_each_drv+0x44/0x8c)
+[<c051c56c>] (bus_for_each_drv) from [<c051dc8c>] (__device_attach+0x9c/0x100)
+[<c051dc8c>] (__device_attach) from [<c051d2d0>] (bus_probe_device+0x84/0x8c)
+[<c051d2d0>] (bus_probe_device) from [<c051b914>] (device_add+0x380/0x528)
+[<c051b914>] (device_add) from [<c0591ee0>] (spi_add_device+0x88/0x130)
+[<c0591ee0>] (spi_add_device) from [<c0593e7c>] (spi_register_controller+0x428/0x8b4)
+[<c0593e7c>] (spi_register_controller) from [<c059433c>] (devm_spi_register_controller+0x34/0x70)
+[<c059433c>] (devm_spi_register_controller) from [<c0594ec8>] (tegra_spi_probe+0x298/0x35c)
+[<c0594ec8>] (tegra_spi_probe) from [<c051f83c>] (platform_drv_probe+0x50/0xb0)
+[<c051f83c>] (platform_drv_probe) from [<c051df4c>] (driver_probe_device+0x238/0x2e4)
+[<c051df4c>] (driver_probe_device) from [<c051e0a4>] (__driver_attach+0xac/0xb0)
+[<c051e0a4>] (__driver_attach) from [<c051c608>] (bus_for_each_dev+0x54/0x88)
+[<c051c608>] (bus_for_each_dev) from [<c051d4a4>] (bus_add_driver+0xe4/0x1f0)
+[<c051d4a4>] (bus_add_driver) from [<c051ec1c>] (driver_register+0x78/0xf4)
+[<c051ec1c>] (driver_register) from [<c0101b8c>] (do_one_initcall+0x3c/0x168)
 [<c0101b8c>] (do_one_initcall) from [<c0e00da8>] (kernel_init_freeable+0x10c/0x1d0)
-[<c0e00da8>] (kernel_init_freeable) from [<c097312c>] (kernel_init+0x8/0x114)
-[<c097312c>] (kernel_init) from [<c0107ce0>] (ret_from_fork+0x14/0x34)
+[<c0e00da8>] (kernel_init_freeable) from [<c097468c>] (kernel_init+0x8/0x114)
+[<c097468c>] (kernel_init) from [<c0107ce0>] (ret_from_fork+0x14/0x34)
 Code: e2800008 e8bd8070 e7f001f2 e7f001f2 (e7f001f2) 
----[ end trace 4e84febfbc04632e ]---
+---[ end trace 746d2d10b2ff2155 ]---
 Kernel panic - not syncing: Attempted to kill init! exitcode=0x0000000b
 
 CPU2: stopping
-CPU: 2 PID: 0 Comm: swapper/2 Tainted: G      D         4.13.0-next-20170912-199758-g0d71e2d #1
+CPU: 2 PID: 0 Comm: swapper/2 Tainted: G      D         4.13.0-next-20170913-199818-g6f20b7a #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c01110d0>] (unwind_backtrace) from [<c010c090>] (show_stack+0x10/0x14)
-[<c010c090>] (show_stack) from [<c095fa00>] (dump_stack+0x84/0xa4)
-[<c095fa00>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
+[<c010c090>] (show_stack) from [<c0960f60>] (dump_stack+0x84/0xa4)
+[<c0960f60>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
 [<c010f61c>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
 Exception stack(0xee8e1f70 to 0xee8e1fb8)
 1f60:                                     00000001 00000000 ee8e1fc8 c011a380
-1f80: c0f03ccc c0f03c68 c0e8c378 c0f03cd8 c0fbd8a4 00000000 00000001 00000000
-1fa0: 2e11e000 ee8e1fc0 c0108758 c010875c 60000013 ffffffff
+1f80: c0f03ccc c0f03c68 c0e8d378 c0f03cd8 c0fbe23c 00000000 00000001 00000000
+1fa0: 2e11d000 ee8e1fc0 c0108758 c010875c 60000013 ffffffff
 [<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
-[<c010875c>] (arch_cpu_idle) from [<c0160ac8>] (do_idle+0x148/0x204)
-[<c0160ac8>] (do_idle) from [<c0160e0c>] (cpu_startup_entry+0x18/0x1c)
-[<c0160e0c>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
+[<c010875c>] (arch_cpu_idle) from [<c0160ae4>] (do_idle+0x148/0x204)
+[<c0160ae4>] (do_idle) from [<c0160e28>] (cpu_startup_entry+0x18/0x1c)
+[<c0160e28>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
 CPU1: stopping
-CPU: 1 PID: 0 Comm: swapper/1 Tainted: G      D         4.13.0-next-20170912-199758-g0d71e2d #1
+CPU: 1 PID: 0 Comm: swapper/1 Tainted: G      D         4.13.0-next-20170913-199818-g6f20b7a #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c01110d0>] (unwind_backtrace) from [<c010c090>] (show_stack+0x10/0x14)
-[<c010c090>] (show_stack) from [<c095fa00>] (dump_stack+0x84/0xa4)
-[<c095fa00>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
+[<c010c090>] (show_stack) from [<c0960f60>] (dump_stack+0x84/0xa4)
+[<c0960f60>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
 [<c010f61c>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
 Exception stack(0xee8dff70 to 0xee8dffb8)
 ff60:                                     00000001 00000000 ee8dffc8 c011a380
-ff80: c0f03ccc c0f03c68 c0e8c378 c0f03cd8 c0fbd8a4 00000000 00000001 00000000
-ffa0: 2e10d000 ee8dffc0 c0108758 c010875c 60000013 ffffffff
+ff80: c0f03ccc c0f03c68 c0e8d378 c0f03cd8 c0fbe23c 00000000 00000001 00000000
+ffa0: 2e10c000 ee8dffc0 c0108758 c010875c 60000013 ffffffff
 [<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
-[<c010875c>] (arch_cpu_idle) from [<c0160ac8>] (do_idle+0x148/0x204)
-[<c0160ac8>] (do_idle) from [<c0160e0c>] (cpu_startup_entry+0x18/0x1c)
-[<c0160e0c>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
+[<c010875c>] (arch_cpu_idle) from [<c0160ae4>] (do_idle+0x148/0x204)
+[<c0160ae4>] (do_idle) from [<c0160e28>] (cpu_startup_entry+0x18/0x1c)
+[<c0160e28>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
 CPU3: stopping
-CPU: 3 PID: 0 Comm: swapper/3 Tainted: G      D         4.13.0-next-20170912-199758-g0d71e2d #1
+CPU: 3 PID: 0 Comm: swapper/3 Tainted: G      D         4.13.0-next-20170913-199818-g6f20b7a #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c01110d0>] (unwind_backtrace) from [<c010c090>] (show_stack+0x10/0x14)
-[<c010c090>] (show_stack) from [<c095fa00>] (dump_stack+0x84/0xa4)
-[<c095fa00>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
+[<c010c090>] (show_stack) from [<c0960f60>] (dump_stack+0x84/0xa4)
+[<c0960f60>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
 [<c010f61c>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
 Exception stack(0xee8e3f70 to 0xee8e3fb8)
 3f60:                                     00000001 00000000 ee8e3fc8 c011a380
-3f80: c0f03ccc c0f03c68 c0e8c378 c0f03cd8 c0fbd8a4 00000000 00000001 00000000
-3fa0: 2e12f000 ee8e3fc0 c0108758 c010875c 60000013 ffffffff
+3f80: c0f03ccc c0f03c68 c0e8d378 c0f03cd8 c0fbe23c 00000000 00000001 00000000
+3fa0: 2e12e000 ee8e3fc0 c0108758 c010875c 60000013 ffffffff
 [<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
-[<c010875c>] (arch_cpu_idle) from [<c0160ac8>] (do_idle+0x148/0x204)
-[<c0160ac8>] (do_idle) from [<c0160e0c>] (cpu_startup_entry+0x18/0x1c)
-[<c0160e0c>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
+[<c010875c>] (arch_cpu_idle) from [<c0160ae4>] (do_idle+0x148/0x204)
+[<c0160ae4>] (do_idle) from [<c0160e28>] (cpu_startup_entry+0x18/0x1c)
+[<c0160e28>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
 ---[ end Kernel panic - not syncing: Attempted to kill init! exitcode=0x0000000b
 
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2017-09-12 21:43:31.519284559 -0700
+++ /tmp/foob	2017-09-12 21:43:31.519284559 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1801040 us
+Bootloader-Cpu Init at (time stamp): 1800907 us
 
 Pinmux changes applied in kernel way
 
@@ -62,22 +62,22 @@
 Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1801014  us
-Bootloader (AosInit) time: 734234 us
-Bootloader Main Init time: 180084 us
-BlockDevMgrInit time: 157525 us
+Miscellaneous avp time: 1800881  us
+Bootloader (AosInit) time: 734250 us
+Bootloader Main Init time: 180088 us
+BlockDevMgrInit time: 157631 us
 PartMgrInit time: 8 us
-StorMgrInit time: 31 us
+StorMgrInit time: 30 us
 FsMgrInit time: 2 us
-Partition Table load: 76690 us 
-Display init time: 574794 us
+Partition Table load: 76661 us 
+Display init time: 574815 us
 Kernel read time: 1340 msec
 Load Kernel Size:   17760256 Bytes
 Bl Kernel ReadSpeed: 13253000 Bytes/sec
-Pre-Kernel Jump time: 3826762 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 8787989 us
+Pre-Kernel Jump time: 3827291 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 8786239 us
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-next-20170912-317999-g0d71e2d4aa14 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Mon Sep 11 21:39:37 PDT 2017
+Linux version 4.13.0-next-20170913-318059-g6f20b7a58cb9 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Tue Sep 12 21:40:36 PDT 2017
 Boot CPU: AArch64 Processor [4e0f0000]
 Machine model: NVIDIA Tegra132 Norrin
 Memory limited to 1734MB
@@ -202,7 +202,7 @@
 Freeing initrd memory: 1284K
 kvm [1]: HYP mode not available
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.979:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.991:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-09-12 21:43:31.527284597 -0700
+++ /tmp/foob	2017-09-12 21:43:31.527284597 -0700
@@ -47,10 +47,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ####################################
-	 3.9 MiB/s
+	 #####################################
+	 4.1 MiB/s
 done
-Bytes transferred = 8159752 (7c8208 hex)
+Bytes transferred = 8163848 (7c9208 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -61,7 +61,7 @@
 done
 Bytes transferred = 21454 (53ce hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x7c8200 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x7c9200 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103cd
@@ -69,7 +69,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-next-20170912-199758-g0d71e2d (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Mon Sep 11 21:34:26 PDT 2017
+Linux version 4.13.0-next-20170913-199818-g6f20b7a (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Tue Sep 12 21:35:43 PDT 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -79,7 +79,7 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0x3c000000
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c158e2c0, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c158eb80, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -93,7 +93,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 954308K/1048576K available (10240K kernel code, 1625K rwdata, 4508K rodata, 2048K init, 405K bss, 28732K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 954304K/1048576K available (10240K kernel code, 1628K rwdata, 4508K rodata, 2048K init, 405K bss, 28736K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -103,8 +103,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1596610   (1626 kB)
-       .bss : 0xc159fbc0 - 0xc160537c   ( 406 kB)
+      .data : 0xc1400000 - 0xc1597028   (1629 kB)
+       .bss : 0xc15a0bc0 - 0xc160637c   ( 406 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -140,7 +140,7 @@
 SMP: Total of 2 processors activated (4.00 BogoMIPS).
 CPU: All CPU(s) started in SVC mode.
 devtmpfs: initialized
-Built 1 zonelists, mobility grouping on.  Total pages: 254961
+Built 1 zonelists, mobility grouping on.  Total pages: 254960
 VFP support v0.3: implementor 41 architecture 3 part 30 variant 9 rev 1
 clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 19112604462750000 ns
 futex hash table entries: 512 (order: 3, 32768 bytes)
@@ -346,10 +346,10 @@
 Key type dns_resolver registered
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2017-09-12 21:43:31.551284707 -0700
+++ /tmp/foob	2017-09-12 21:43:31.555284725 -0700
@@ -46,9 +46,9 @@
 	 #################################################################
 	 #################################################################
 	 #######################################
-	 4 MiB/s
+	 4.1 MiB/s
 done
-Bytes transferred = 6288104 (5ff2e8 hex)
+Bytes transferred = 6292568 (600458 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -59,7 +59,7 @@
 done
 Bytes transferred = 21454 (53ce hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x5ff2e8 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x600458 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103cd
@@ -68,7 +68,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-next-20170912-199758-g0d71e2d (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Sep 11 21:34:53 PDT 2017
+Linux version 4.13.0-next-20170913-199818-g6f20b7a (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Tue Sep 12 21:35:41 PDT 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -77,7 +77,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0x3c000000
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0fb7b80, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0fb8440, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -91,7 +91,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 958328K/1048576K available (9216K kernel code, 778K rwdata, 2796K rodata, 1024K init, 300K bss, 24712K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 958324K/1048576K available (9216K kernel code, 781K rwdata, 2796K rodata, 1024K init, 300K bss, 24716K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -101,8 +101,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
-      .data : 0xc0f00000 - 0xc0fc2b48   ( 779 kB)
-       .bss : 0xc0fcc52c - 0xc10176bc   ( 301 kB)
+      .data : 0xc0f00000 - 0xc0fc3560   ( 782 kB)
+       .bss : 0xc0fcd52c - 0xc10186bc   ( 301 kB)
 Preemptible hierarchical RCU implementation.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
 	Tasks RCU enabled.
@@ -136,7 +136,7 @@
 SMP: Total of 2 processors activated (4.00 BogoMIPS).
 CPU: All CPU(s) started in SVC mode.
 devtmpfs: initialized
-Built 1 zonelists, mobility grouping on.  Total pages: 255966
+Built 1 zonelists, mobility grouping on.  Total pages: 255965
 VFP support v0.3: implementor 41 architecture 3 part 30 variant 9 rev 1
 clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 19112604462750000 ns
 futex hash table entries: 512 (order: 3, 32768 bytes)
@@ -325,9 +325,9 @@
 NET: Registered protocol family 17
 NET: Registered protocol family 15
 can: controller area network core (rev 20170425 abi 9)
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 NET: Registered protocol family 29
 can: raw protocol (rev 20170425)
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
 can: broadcast manager protocol (rev 20170425 t)
 can: netlink gateway (rev 20170425) max_hops=1
 Bluetooth: RFCOMM socket layer initialized
@@ -339,12 +339,12 @@
 Key type dns_resolver registered
 Registering SWP/SWPB emulation handler
 mmc1: new high speed SD card at address b368
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
+ mmcblk1: p1 p2
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
- mmcblk1: p1 p2
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2017-09-12 21:43:31.563284762 -0700
+++ /tmp/foob	2017-09-12 21:43:31.567284782 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 837495 us
-Basic modules init took 349254 us
+Tegraboot started after 789300 us
+Basic modules init took 350330 us
 Sec Bootdevice Read Time = 203 ms, Read Size = 8459 KB
-Next stage binary read took 14382 us
-Carveout took 252666 us
-CPU initialization took 92435 us
-Total time taken by TegraBoot 708737 us
+Next stage binary read took 14917 us
+Carveout took 252688 us
+CPU initialization took 92445 us
+Total time taken by TegraBoot 710380 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.675] LPDDR4 Training: Number of tables = 10
+64b[0001.629] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-16439808 bytes read in 835 ms (18.8 MiB/s)
+16439808 bytes read in 828 ms (18.9 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-45170 bytes read in 454 ms (96.7 KiB/s)
+45170 bytes read in 453 ms (96.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 494 ms (3.1 MiB/s)
+1627787 bytes read in 495 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-next-20170912-317999-g0d71e2d4aa14 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Mon Sep 11 21:39:37 PDT 2017
+Linux version 4.13.0-next-20170913-318059-g6f20b7a58cb9 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Tue Sep 12 21:40:36 PDT 2017
 Boot CPU: AArch64 Processor [411fd071]
 Machine model: NVIDIA Tegra210 P2371 (P2530/P2595) reference design
 debug: ignoring loglevel setting.
@@ -270,7 +270,7 @@
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 PCI: CLS 0 bytes, default 128
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 80974000
+kvm [1]: IDMAP page: 80973000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2017-09-12 21:43:31.571284800 -0700
+++ /tmp/foob	2017-09-12 21:43:31.575284817 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-next-20170912-317999-g0d71e2d4aa14 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Mon Sep 11 21:39:37 PDT 2017
+Linux version 4.13.0-next-20170913-318059-g6f20b7a58cb9 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Tue Sep 12 21:40:36 PDT 2017
 Boot CPU: AArch64 Processor [411fd071]
 Machine model: Google Pixel C
 efi: Getting EFI parameters from FDT:
@@ -152,14 +152,14 @@
 Unpacking initramfs...
 Freeing initrd memory: 1284K
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 80974000
+kvm [1]: IDMAP page: 80973000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
 kvm [1]: vgic interrupt IRQ1
 kvm [1]: virtual timer IRQ4
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.255:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.259:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -260,8 +260,9 @@
 mmcblk0: mmc0:0001 BGND3R 29.1 GiB 
 mmcblk0boot0: mmc0:0001 BGND3R partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
-max77686-rtc max77620-rtc: setting system clock to 2001-06-23 10:04:19 UTC (993290659)
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
+max77686-rtc max77620-rtc: setting system clock to 2001-06-24 10:05:27 UTC (993377127)
 PP1200_AVDD: disabling
 PP1200_RCAM: disabling
 PP2800L_RCAM: disabling
@@ -269,12 +270,11 @@
 PP_LDO6: disabling
 PPLCD_VDD: disabling
 PP1800_LCDIO: disabling
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
-tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 PP1800_CAM: disabling
 ALSA device list:
   No soundcards found.
 Freeing unused kernel memory: 1088K
+tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 
 System initialization...
 
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-09-12 21:43:31.587284873 -0700
+++ /tmp/foob	2017-09-12 21:43:31.587284873 -0700
@@ -39,21 +39,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ####################################
-	 2.7 MiB/s
+	 #####################################
+	 3.2 MiB/s
 done
-Bytes transferred = 8159752 (7c8208 hex)
+Bytes transferred = 8163848 (7c9208 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.6 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 51860 (ca94 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x7c8200 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x7c9200 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa93 ... OK
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-next-20170912-199758-g0d71e2d (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Mon Sep 11 21:34:26 PDT 2017
+Linux version 4.13.0-next-20170913-199818-g6f20b7a (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Tue Sep 12 21:35:43 PDT 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -71,7 +71,7 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c158e2c0, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c158eb80, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -85,7 +85,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993400K/2096128K available (10240K kernel code, 1625K rwdata, 4508K rodata, 2048K init, 405K bss, 37192K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993396K/2096128K available (10240K kernel code, 1628K rwdata, 4508K rodata, 2048K init, 405K bss, 37196K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -95,8 +95,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1596610   (1626 kB)
-       .bss : 0xc159fbc0 - 0xc160537c   ( 406 kB)
+      .data : 0xc1400000 - 0xc1597028   (1629 kB)
+       .bss : 0xc15a0bc0 - 0xc160637c   ( 406 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -134,7 +134,7 @@
 SMP: Total of 4 processors activated (8.00 BogoMIPS).
 CPU: All CPU(s) started in SVC mode.
 devtmpfs: initialized
-Built 1 zonelists, mobility grouping on.  Total pages: 514734
+Built 1 zonelists, mobility grouping on.  Total pages: 514733
 VFP support v0.3: implementor 41 architecture 3 part 30 variant 9 rev 4
 clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 19112604462750000 ns
 futex hash table entries: 1024 (order: 4, 65536 bytes)
@@ -474,4 +474,4 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.544422] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.554444] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2017-09-12 21:43:31.603284947 -0700
+++ /tmp/foob	2017-09-12 21:43:31.607284964 -0700
@@ -40,18 +40,18 @@
 	 #######################################
 	 3.2 MiB/s
 done
-Bytes transferred = 6288104 (5ff2e8 hex)
+Bytes transferred = 6292568 (600458 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.9 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 51860 (ca94 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5ff2e8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x600458 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa93 ... OK
@@ -60,7 +60,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-next-20170912-199758-g0d71e2d (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Sep 11 21:34:53 PDT 2017
+Linux version 4.13.0-next-20170913-199818-g6f20b7a (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Tue Sep 12 21:35:41 PDT 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -69,7 +69,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0fb7b80, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0fb8440, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -83,7 +83,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1997420K/2096128K available (9216K kernel code, 778K rwdata, 2796K rodata, 1024K init, 300K bss, 33172K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1997416K/2096128K available (9216K kernel code, 781K rwdata, 2796K rodata, 1024K init, 300K bss, 33176K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -93,8 +93,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
-      .data : 0xc0f00000 - 0xc0fc2b48   ( 779 kB)
-       .bss : 0xc0fcc52c - 0xc10176bc   ( 301 kB)
+      .data : 0xc0f00000 - 0xc0fc3560   ( 782 kB)
+       .bss : 0xc0fcd52c - 0xc10186bc   ( 301 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -128,7 +128,7 @@
 SMP: Total of 4 processors activated (8.00 BogoMIPS).
 CPU: All CPU(s) started in SVC mode.
 devtmpfs: initialized
-Built 1 zonelists, mobility grouping on.  Total pages: 515739
+Built 1 zonelists, mobility grouping on.  Total pages: 515738
 VFP support v0.3: implementor 41 architecture 3 part 30 variant 9 rev 4
 clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 19112604462750000 ns
 futex hash table entries: 1024 (order: 4, 65536 bytes)
@@ -471,4 +471,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.804442] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.804422] usb 2-1: new high-speed USB device number 2 using tegra-ehci
