Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-01-14 20:54:14.520607806 -0800
+++ /tmp/foob	2016-01-14 20:54:14.524607803 -0800
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-next-20160114+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Jan 13 20:49:17 PST 2016
+Linux version 4.4.0-next-20160115+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Jan 14 20:49:29 PST 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -93,7 +93,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.580:1): initialized
+audit: type=2000 audit(0.452:1): initialized
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
 Key type id_resolver registered
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-01-14 20:54:14.560607775 -0800
+++ /tmp/foob	2016-01-14 20:54:14.560607775 -0800
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 ######
-	 3.1 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 6760936 (6729e8 hex)
+Bytes transferred = 6761088 (672a80 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6729e8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x672a80 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-next-20160114-67327-g0908339 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Jan 13 20:47:59 PST 2016
+Linux version 4.4.0-next-20160115-67505-g39750fe (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Jan 14 20:47:56 PST 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -293,10 +293,10 @@
 tegra-ehci 7d008000.usb: EHCI Host Controller
 tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 1
 ------------[ cut here ]------------
-WARNING: CPU: 3 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
+WARNING: CPU: 2 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 3 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160114-67327-g0908339 #1
+CPU: 2 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160115-67505-g39750fe #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c02188b8>] (unwind_backtrace) from [<c0213f94>] (show_stack+0x10/0x14)
@@ -313,28 +313,28 @@
 [<c021e39c>] (__alloc_from_contiguous) from [<c021ff8c>] (__dma_alloc+0x21c/0x250)
 [<c021ff8c>] (__dma_alloc) from [<c0220044>] (arm_dma_alloc+0x48/0x54)
 [<c0220044>] (arm_dma_alloc) from [<c0306c38>] (dma_pool_alloc+0x1dc/0x210)
-[<c0306c38>] (dma_pool_alloc) from [<c07c7008>] (ehci_qh_alloc+0x3c/0xb4)
-[<c07c7008>] (ehci_qh_alloc) from [<c07cb600>] (ehci_setup+0x154/0x478)
-[<c07cb600>] (ehci_setup) from [<c07d16b8>] (tegra_ehci_reset+0xc/0x50)
-[<c07d16b8>] (tegra_ehci_reset) from [<c07a1a80>] (usb_add_hcd+0x300/0x830)
-[<c07a1a80>] (usb_add_hcd) from [<c07d1030>] (tegra_ehci_probe+0x3b8/0x504)
-[<c07d1030>] (tegra_ehci_probe) from [<c0659274>] (platform_drv_probe+0x50/0xac)
-[<c0659274>] (platform_drv_probe) from [<c0657be0>] (driver_probe_device+0x1f4/0x2b0)
-[<c0657be0>] (driver_probe_device) from [<c0656170>] (bus_for_each_drv+0x44/0x8c)
-[<c0656170>] (bus_for_each_drv) from [<c0657964>] (__device_attach+0x9c/0x100)
-[<c0657964>] (__device_attach) from [<c0656ff0>] (bus_probe_device+0x84/0x8c)
-[<c0656ff0>] (bus_probe_device) from [<c0657424>] (deferred_probe_work_func+0x60/0x8c)
-[<c0657424>] (deferred_probe_work_func) from [<c025ebb8>] (process_one_work+0x134/0x338)
+[<c0306c38>] (dma_pool_alloc) from [<c07c7024>] (ehci_qh_alloc+0x3c/0xb4)
+[<c07c7024>] (ehci_qh_alloc) from [<c07cb61c>] (ehci_setup+0x154/0x478)
+[<c07cb61c>] (ehci_setup) from [<c07d16d4>] (tegra_ehci_reset+0xc/0x50)
+[<c07d16d4>] (tegra_ehci_reset) from [<c07a1a9c>] (usb_add_hcd+0x300/0x830)
+[<c07a1a9c>] (usb_add_hcd) from [<c07d104c>] (tegra_ehci_probe+0x3b8/0x504)
+[<c07d104c>] (tegra_ehci_probe) from [<c0659290>] (platform_drv_probe+0x50/0xac)
+[<c0659290>] (platform_drv_probe) from [<c0657bfc>] (driver_probe_device+0x1f4/0x2b0)
+[<c0657bfc>] (driver_probe_device) from [<c065618c>] (bus_for_each_drv+0x44/0x8c)
+[<c065618c>] (bus_for_each_drv) from [<c0657980>] (__device_attach+0x9c/0x100)
+[<c0657980>] (__device_attach) from [<c065700c>] (bus_probe_device+0x84/0x8c)
+[<c065700c>] (bus_probe_device) from [<c0657440>] (deferred_probe_work_func+0x60/0x8c)
+[<c0657440>] (deferred_probe_work_func) from [<c025ebb8>] (process_one_work+0x134/0x338)
 [<c025ebb8>] (process_one_work) from [<c025ee20>] (worker_thread+0x2c/0x4f0)
 [<c025ee20>] (worker_thread) from [<c0263fac>] (kthread+0xd8/0xf4)
 [<c0263fac>] (kthread) from [<c0210e78>] (ret_from_fork+0x14/0x3c)
----[ end trace 3ec852d0d064256b ]---
+---[ end trace c5f1bc321ca377cd ]---
 tegra-ehci 7d008000.usb: irq 339, io mem 0x7d008000
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-07-26 13:48:37 UTC (964619317)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-07-27 13:48:42 UTC (964705722)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -346,4 +346,4 @@
 Freeing unused kernel memory: 1056K (c0e42000 - c0f4a000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.509242] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.524150] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-01-14 20:54:14.568607766 -0800
+++ /tmp/foob	2016-01-14 20:54:14.568607766 -0800
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 #################################
-	 3.2 MiB/s
+	 2.8 MiB/s
 done
-Bytes transferred = 5253840 (502ad0 hex)
+Bytes transferred = 5254048 (502ba0 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 2.6 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x502ad0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x502ba0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-next-20160114-67327-g0908339 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Jan 13 20:48:11 PST 2016
+Linux version 4.4.0-next-20160115-67505-g39750fe (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Jan 14 20:48:10 PST 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -290,10 +290,10 @@
 tegra-ehci 7d008000.usb: EHCI Host Controller
 tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 1
 ------------[ cut here ]------------
-WARNING: CPU: 0 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
+WARNING: CPU: 2 PID: 77 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160114-67327-g0908339 #1
+CPU: 2 PID: 77 Comm: kworker/u8:1 Not tainted 4.4.0-next-20160115-67505-g39750fe #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0017208>] (unwind_backtrace) from [<c0012874>] (show_stack+0x10/0x14)
@@ -310,28 +310,28 @@
 [<c001c270>] (__alloc_from_contiguous) from [<c001cf38>] (__dma_alloc+0x228/0x2ac)
 [<c001cf38>] (__dma_alloc) from [<c001d040>] (arm_dma_alloc+0x48/0x54)
 [<c001d040>] (arm_dma_alloc) from [<c00edb34>] (dma_pool_alloc+0x28c/0x304)
-[<c00edb34>] (dma_pool_alloc) from [<c03fd2d0>] (ehci_qh_alloc+0x3c/0xb4)
-[<c03fd2d0>] (ehci_qh_alloc) from [<c04014bc>] (ehci_setup+0x154/0x478)
-[<c04014bc>] (ehci_setup) from [<c0405204>] (tegra_ehci_reset+0xc/0x50)
-[<c0405204>] (tegra_ehci_reset) from [<c03e8cb4>] (usb_add_hcd+0x2fc/0x804)
-[<c03e8cb4>] (usb_add_hcd) from [<c0404b7c>] (tegra_ehci_probe+0x3b8/0x504)
-[<c0404b7c>] (tegra_ehci_probe) from [<c0336cb0>] (platform_drv_probe+0x50/0xac)
-[<c0336cb0>] (platform_drv_probe) from [<c0335718>] (driver_probe_device+0x1f4/0x2b0)
-[<c0335718>] (driver_probe_device) from [<c0333cac>] (bus_for_each_drv+0x44/0x8c)
-[<c0333cac>] (bus_for_each_drv) from [<c033549c>] (__device_attach+0x9c/0x100)
-[<c033549c>] (__device_attach) from [<c0334b28>] (bus_probe_device+0x84/0x8c)
-[<c0334b28>] (bus_probe_device) from [<c0334f5c>] (deferred_probe_work_func+0x60/0x8c)
-[<c0334f5c>] (deferred_probe_work_func) from [<c003a6fc>] (process_one_work+0x120/0x328)
+[<c00edb34>] (dma_pool_alloc) from [<c03fd2ec>] (ehci_qh_alloc+0x3c/0xb4)
+[<c03fd2ec>] (ehci_qh_alloc) from [<c04014d8>] (ehci_setup+0x154/0x478)
+[<c04014d8>] (ehci_setup) from [<c0405220>] (tegra_ehci_reset+0xc/0x50)
+[<c0405220>] (tegra_ehci_reset) from [<c03e8cd0>] (usb_add_hcd+0x2fc/0x804)
+[<c03e8cd0>] (usb_add_hcd) from [<c0404b98>] (tegra_ehci_probe+0x3b8/0x504)
+[<c0404b98>] (tegra_ehci_probe) from [<c0336ccc>] (platform_drv_probe+0x50/0xac)
+[<c0336ccc>] (platform_drv_probe) from [<c0335734>] (driver_probe_device+0x1f4/0x2b0)
+[<c0335734>] (driver_probe_device) from [<c0333cc8>] (bus_for_each_drv+0x44/0x8c)
+[<c0333cc8>] (bus_for_each_drv) from [<c03354b8>] (__device_attach+0x9c/0x100)
+[<c03354b8>] (__device_attach) from [<c0334b44>] (bus_probe_device+0x84/0x8c)
+[<c0334b44>] (bus_probe_device) from [<c0334f78>] (deferred_probe_work_func+0x60/0x8c)
+[<c0334f78>] (deferred_probe_work_func) from [<c003a6fc>] (process_one_work+0x120/0x328)
 [<c003a6fc>] (process_one_work) from [<c003a968>] (worker_thread+0x2c/0x4b4)
 [<c003a968>] (worker_thread) from [<c003faa0>] (kthread+0xd8/0xf4)
 [<c003faa0>] (kthread) from [<c000f938>] (ret_from_fork+0x14/0x3c)
----[ end trace 9046626d052ce852 ]---
+---[ end trace 941912c09d97bd3c ]---
 tegra-ehci 7d008000.usb: irq 339, io mem 0x7d008000
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-07-26 13:49:03 UTC (964619343)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-07-27 13:50:04 UTC (964705804)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-01-14 20:54:14.576607759 -0800
+++ /tmp/foob	2016-01-14 20:54:14.576607759 -0800
@@ -33,52 +33,6 @@
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'zImage.multi_v7_defconfig'.
 Load address: 0x82000000
-Loading: *EHCI timed out on TD - token=0x8008d80
-T EHCI timed out on TD - token=0x8008d80
-T EHCI timed out on TD - token=0x8008d80
-T EHCI timed out on TD - token=0x8008d80
-T EHCI timed out on TD - token=0x8008d80
-T EHCI timed out on TD - token=0x8008d80
-
-Abort
-Tegra124 (Jetson TK1) # 
-Tegra124 (Jetson TK1) # reset
-resetting ...
-
-U-Boot SPL 2014.07-00097-g772e173 (Jul 25 2014 - 20:02:52)
-
-
-U-Boot 2014.07-00097-g772e173 (Jul 25 2014 - 20:02:52)
-
-TEGRA124
-Board: NVIDIA Jetson TK1
-I2C:   ready
-DRAM:  2 GiB
-MMC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
-In:    serial
-Out:   serial
-Err:   serial
-Net:   Net Initialization Skipped
-No ethernet found.
-Hit any key to stop autoboot:  2  0 
-Tegra124 (Jetson TK1) # 
-Tegra124 (Jetson TK1) # usb reset
-(Re)start USB...
-USB0:   USB EHCI 1.10
-scanning bus 0 for devices... 1 USB Device(s) found
-USB1:   USB EHCI 1.10
-scanning bus 1 for devices... 2 USB Device(s) found
-       scanning usb for storage devices... 0 Storage Device(s) found
-       scanning usb for ethernet devices... 
-Warning: asx0 using MAC address from net device
-1 Ethernet Device(s) found
-Tegra124 (Jetson TK1) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash'
-Tegra124 (Jetson TK1) # setenv ipaddr 192.168.100.102; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
-Tegra124 (Jetson TK1) # tftpboot 0x82000000 zImage.multi_v7_defconfig
-Using asx0 device
-TFTP from server 192.168.100.1; our IP address is 192.168.100.102
-Filename 'zImage.multi_v7_defconfig'.
-Load address: 0x82000000
 Loading: *#################################################################
 	 #################################################################
 	 #################################################################
@@ -89,18 +43,18 @@
 	 ######
 	 3.2 MiB/s
 done
-Bytes transferred = 6760936 (6729e8 hex)
+Bytes transferred = 6761088 (672a80 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.2 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6729e8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x672a80 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012a89
@@ -108,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-next-20160114-67327-g0908339 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Jan 13 20:47:59 PST 2016
+Linux version 4.4.0-next-20160115-67505-g39750fe (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Jan 14 20:47:56 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -382,7 +336,7 @@
 WARNING: CPU: 2 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 2 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160114-67327-g0908339 #1
+CPU: 2 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160115-67505-g39750fe #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c02188b8>] (unwind_backtrace) from [<c0213f94>] (show_stack+0x10/0x14)
@@ -398,24 +352,24 @@
 [<c030e014>] (cma_alloc) from [<c021e39c>] (__alloc_from_contiguous+0x38/0xdc)
 [<c021e39c>] (__alloc_from_contiguous) from [<c021ff8c>] (__dma_alloc+0x21c/0x250)
 [<c021ff8c>] (__dma_alloc) from [<c0220044>] (arm_dma_alloc+0x48/0x54)
-[<c0220044>] (arm_dma_alloc) from [<c066a374>] (dmam_alloc_coherent+0x8c/0xf0)
-[<c066a374>] (dmam_alloc_coherent) from [<c06c4e3c>] (ahci_port_start+0xd0/0x1e0)
-[<c06c4e3c>] (ahci_port_start) from [<c06acce4>] (ata_host_start+0xf8/0x1d4)
-[<c06acce4>] (ata_host_start) from [<c06b1c00>] (ata_host_activate+0x20/0x134)
-[<c06b1c00>] (ata_host_activate) from [<c06c42b8>] (ahci_host_activate+0x3c/0x11c)
-[<c06c42b8>] (ahci_host_activate) from [<c06c61fc>] (ahci_platform_init_host+0x1cc/0x3c8)
-[<c06c61fc>] (ahci_platform_init_host) from [<c06c8460>] (tegra_ahci_probe+0x404/0x4c4)
-[<c06c8460>] (tegra_ahci_probe) from [<c0659274>] (platform_drv_probe+0x50/0xac)
-[<c0659274>] (platform_drv_probe) from [<c0657be0>] (driver_probe_device+0x1f4/0x2b0)
-[<c0657be0>] (driver_probe_device) from [<c0656170>] (bus_for_each_drv+0x44/0x8c)
-[<c0656170>] (bus_for_each_drv) from [<c0657964>] (__device_attach+0x9c/0x100)
-[<c0657964>] (__device_attach) from [<c0656ff0>] (bus_probe_device+0x84/0x8c)
-[<c0656ff0>] (bus_probe_device) from [<c0657424>] (deferred_probe_work_func+0x60/0x8c)
-[<c0657424>] (deferred_probe_work_func) from [<c025ebb8>] (process_one_work+0x134/0x338)
+[<c0220044>] (arm_dma_alloc) from [<c066a390>] (dmam_alloc_coherent+0x8c/0xf0)
+[<c066a390>] (dmam_alloc_coherent) from [<c06c4e58>] (ahci_port_start+0xd0/0x1e0)
+[<c06c4e58>] (ahci_port_start) from [<c06acd00>] (ata_host_start+0xf8/0x1d4)
+[<c06acd00>] (ata_host_start) from [<c06b1c1c>] (ata_host_activate+0x20/0x134)
+[<c06b1c1c>] (ata_host_activate) from [<c06c42d4>] (ahci_host_activate+0x3c/0x11c)
+[<c06c42d4>] (ahci_host_activate) from [<c06c6218>] (ahci_platform_init_host+0x1cc/0x3c8)
+[<c06c6218>] (ahci_platform_init_host) from [<c06c847c>] (tegra_ahci_probe+0x404/0x4c4)
+[<c06c847c>] (tegra_ahci_probe) from [<c0659290>] (platform_drv_probe+0x50/0xac)
+[<c0659290>] (platform_drv_probe) from [<c0657bfc>] (driver_probe_device+0x1f4/0x2b0)
+[<c0657bfc>] (driver_probe_device) from [<c065618c>] (bus_for_each_drv+0x44/0x8c)
+[<c065618c>] (bus_for_each_drv) from [<c0657980>] (__device_attach+0x9c/0x100)
+[<c0657980>] (__device_attach) from [<c065700c>] (bus_probe_device+0x84/0x8c)
+[<c065700c>] (bus_probe_device) from [<c0657440>] (deferred_probe_work_func+0x60/0x8c)
+[<c0657440>] (deferred_probe_work_func) from [<c025ebb8>] (process_one_work+0x134/0x338)
 [<c025ebb8>] (process_one_work) from [<c025ee20>] (worker_thread+0x2c/0x4f0)
 [<c025ee20>] (worker_thread) from [<c0263fac>] (kthread+0xd8/0xf4)
 [<c0263fac>] (kthread) from [<c0210e78>] (ret_from_fork+0x14/0x3c)
----[ end trace cd5d71cf0824c203 ]---
+---[ end trace 1dfed483db9ff408 ]---
 scsi host0: tegra-ahci
 scsi host1: tegra-ahci
 ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 341
@@ -437,7 +391,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:56 UTC (946684856)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:18 UTC (946684818)
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
@@ -450,8 +404,8 @@
 EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
 EXT4-fs (mmcblk0p1): couldn't mount as ext2 due to feature incompatibilities
 EXT4-fs (mmcblk0p1): recovery complete
-EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 usb 2-1: new high-speed USB device number 2 using tegra-ehci
+EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 1056K (c0e42000 - c0f4a000)
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-01-14 20:54:14.584607754 -0800
+++ /tmp/foob	2016-01-14 20:54:14.584607754 -0800
@@ -29,53 +29,6 @@
 Tegra124 (Jetson TK1) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash'
 Tegra124 (Jetson TK1) # setenv ipaddr 192.168.100.102; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 zImage.tegra_defconfig
-Waiting for Ethernet connection... done.
-Using asx0 device
-TFTP from server 192.168.100.1; our IP address is 192.168.100.102
-Filename 'zImage.tegra_defconfig'.
-Load address: 0x82000000
-Loading: *EHCI timed out on TD - token=0x8008d80
-T EHCI timed out on TD - token=0x8008d80
-T EHCI timed out on TD - token=0x8008d80
-T EHCI timed out on TD - token=0x8008d80
-T EHCI timed out on TD - token=0x8008d80
-T EHCI timed out on TD - token=0x8008d80
-
-Abort
-Tegra124 (Jetson TK1) # 
-Tegra124 (Jetson TK1) # reset
-resetting ...
-
-U-Boot SPL 2014.07-00097-g772e173 (Jul 25 2014 - 20:02:52)
-
-
-U-Boot 2014.07-00097-g772e173 (Jul 25 2014 - 20:02:52)
-
-TEGRA124
-Board: NVIDIA Jetson TK1
-I2C:   ready
-DRAM:  2 GiB
-MMC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
-In:    serial
-Out:   serial
-Err:   serial
-Net:   Net Initialization Skipped
-No ethernet found.
-Hit any key to stop autoboot:  2  0 
-Tegra124 (Jetson TK1) # 
-Tegra124 (Jetson TK1) # usb reset
-(Re)start USB...
-USB0:   USB EHCI 1.10
-scanning bus 0 for devices... 1 USB Device(s) found
-USB1:   USB EHCI 1.10
-scanning bus 1 for devices... 2 USB Device(s) found
-       scanning usb for storage devices... 0 Storage Device(s) found
-       scanning usb for ethernet devices... 
-Warning: asx0 using MAC address from net device
-1 Ethernet Device(s) found
-Tegra124 (Jetson TK1) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash'
-Tegra124 (Jetson TK1) # setenv ipaddr 192.168.100.102; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
-Tegra124 (Jetson TK1) # tftpboot 0x82000000 zImage.tegra_defconfig
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'zImage.tegra_defconfig'.
@@ -86,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 #################################
-	 3.2 MiB/s
+	 3.1 MiB/s
 done
-Bytes transferred = 5253840 (502ad0 hex)
+Bytes transferred = 5254048 (502ba0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.1 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x502ad0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x502ba0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012a89
@@ -108,7 +61,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-next-20160114-67327-g0908339 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Jan 13 20:48:11 PST 2016
+Linux version 4.4.0-next-20160115-67505-g39750fe (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Jan 14 20:48:10 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -349,42 +302,16 @@
 tegra-pcie 1003000.pcie-controller: link 0 down, ignoring
 tegra-pcie 1003000.pcie-controller: probing port 1, using 1 lanes
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000000
+tegra-pcie 1003000.pcie-controller: link 1 down, retrying
+tegra-pcie 1003000.pcie-controller: link 1 down, retrying
+tegra-pcie 1003000.pcie-controller: link 1 down, retrying
+tegra-pcie 1003000.pcie-controller: link 1 down, ignoring
 tegra-pcie 1003000.pcie-controller: PCI host bridge to bus 0000:00
 pci_bus 0000:00: root bus resource [mem 0x13000000-0x1fffffff]
 pci_bus 0000:00: root bus resource [mem 0x20000000-0x3fffffff pref]
 pci_bus 0000:00: root bus resource [bus 00-ff]
 pci_bus 0000:00: root bus resource [io  0x1000-0xffff]
-pci 0000:00:02.0: [10de:0e13] type 01 class 0x060400
-pci 0000:00:02.0: PME# supported from D0 D1 D2 D3hot D3cold
-PCI: bus0: Fast back to back transfers disabled
-pci 0000:00:02.0: bridge configuration invalid ([bus 00-00]), reconfiguring
-pci 0000:01:00.0: [10ec:8168] type 00 class 0x020000
-pci 0000:01:00.0: reg 0x10: [io  0x0000-0x00ff]
-pci 0000:01:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit]
-pci 0000:01:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
-pci 0000:01:00.0: supports D1 D2
-pci 0000:01:00.0: PME# supported from D0 D1 D2 D3hot D3cold
-PCI: bus1: Fast back to back transfers disabled
-pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
-pci 0000:00:02.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
-pci 0000:00:02.0: BAR 9: assigned [mem 0x20000000-0x200fffff 64bit pref]
-pci 0000:00:02.0: BAR 7: assigned [io  0x1000-0x1fff]
-pci 0000:01:00.0: BAR 4: assigned [mem 0x20000000-0x20003fff 64bit pref]
-pci 0000:01:00.0: BAR 2: assigned [mem 0x13000000-0x13000fff 64bit]
-pci 0000:01:00.0: BAR 0: assigned [io  0x1000-0x10ff]
-pci 0000:00:02.0: PCI bridge to [bus 01]
-pci 0000:00:02.0:   bridge window [io  0x1000-0x1fff]
-pci 0000:00:02.0:   bridge window [mem 0x13000000-0x130fffff]
-pci 0000:00:02.0:   bridge window [mem 0x20000000-0x200fffff 64bit pref]
-pci 0000:00:02.0: nv_msi_ht_cap_quirk didn't locate host bridge
-pcieport 0000:00:02.0: enabling device (0140 -> 0143)
-pcieport 0000:00:02.0: Signaling PME through PCIe PME interrupt
-pci 0000:01:00.0: Signaling PME through PCIe PME interrupt
-pcie_pme 0000:00:02.0:pcie01: service driver pcie_pme loaded
-r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
-r8169 0000:01:00.0: enabling device (0140 -> 0143)
-r8169 0000:01:00.0 eth0: RTL8168g/8111g at 0xf0952000, 00:04:4b:25:b3:b4, XID 0c000800 IRQ 388
-r8169 0000:01:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
+PCI: bus0: Fast back to back transfers enabled
 +5V_HDMI_CON: supplied by +5V_SYS
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 +3.3V_RUN: supplied by +3.3V_SYS
@@ -403,7 +330,7 @@
 WARNING: CPU: 0 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160114-67327-g0908339 #1
+CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160115-67505-g39750fe #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0017208>] (unwind_backtrace) from [<c0012874>] (show_stack+0x10/0x14)
@@ -419,24 +346,24 @@
 [<c00f3298>] (cma_alloc) from [<c001c270>] (__alloc_from_contiguous+0x38/0xdc)
 [<c001c270>] (__alloc_from_contiguous) from [<c001cf38>] (__dma_alloc+0x228/0x2ac)
 [<c001cf38>] (__dma_alloc) from [<c001d040>] (arm_dma_alloc+0x48/0x54)
-[<c001d040>] (arm_dma_alloc) from [<c0345c84>] (dmam_alloc_coherent+0x8c/0xf0)
-[<c0345c84>] (dmam_alloc_coherent) from [<c038bd2c>] (ahci_port_start+0xd0/0x1e0)
-[<c038bd2c>] (ahci_port_start) from [<c0373a04>] (ata_host_start+0xf8/0x1d4)
-[<c0373a04>] (ata_host_start) from [<c0378998>] (ata_host_activate+0x20/0x134)
-[<c0378998>] (ata_host_activate) from [<c038b1d4>] (ahci_host_activate+0x3c/0x11c)
-[<c038b1d4>] (ahci_host_activate) from [<c038d57c>] (ahci_platform_init_host+0x1cc/0x3c8)
-[<c038d57c>] (ahci_platform_init_host) from [<c038d248>] (tegra_ahci_probe+0x404/0x4c4)
-[<c038d248>] (tegra_ahci_probe) from [<c0336cb0>] (platform_drv_probe+0x50/0xac)
-[<c0336cb0>] (platform_drv_probe) from [<c0335718>] (driver_probe_device+0x1f4/0x2b0)
-[<c0335718>] (driver_probe_device) from [<c0333cac>] (bus_for_each_drv+0x44/0x8c)
-[<c0333cac>] (bus_for_each_drv) from [<c033549c>] (__device_attach+0x9c/0x100)
-[<c033549c>] (__device_attach) from [<c0334b28>] (bus_probe_device+0x84/0x8c)
-[<c0334b28>] (bus_probe_device) from [<c0334f5c>] (deferred_probe_work_func+0x60/0x8c)
-[<c0334f5c>] (deferred_probe_work_func) from [<c003a6fc>] (process_one_work+0x120/0x328)
+[<c001d040>] (arm_dma_alloc) from [<c0345ca0>] (dmam_alloc_coherent+0x8c/0xf0)
+[<c0345ca0>] (dmam_alloc_coherent) from [<c038bd48>] (ahci_port_start+0xd0/0x1e0)
+[<c038bd48>] (ahci_port_start) from [<c0373a20>] (ata_host_start+0xf8/0x1d4)
+[<c0373a20>] (ata_host_start) from [<c03789b4>] (ata_host_activate+0x20/0x134)
+[<c03789b4>] (ata_host_activate) from [<c038b1f0>] (ahci_host_activate+0x3c/0x11c)
+[<c038b1f0>] (ahci_host_activate) from [<c038d598>] (ahci_platform_init_host+0x1cc/0x3c8)
+[<c038d598>] (ahci_platform_init_host) from [<c038d264>] (tegra_ahci_probe+0x404/0x4c4)
+[<c038d264>] (tegra_ahci_probe) from [<c0336ccc>] (platform_drv_probe+0x50/0xac)
+[<c0336ccc>] (platform_drv_probe) from [<c0335734>] (driver_probe_device+0x1f4/0x2b0)
+[<c0335734>] (driver_probe_device) from [<c0333cc8>] (bus_for_each_drv+0x44/0x8c)
+[<c0333cc8>] (bus_for_each_drv) from [<c03354b8>] (__device_attach+0x9c/0x100)
+[<c03354b8>] (__device_attach) from [<c0334b44>] (bus_probe_device+0x84/0x8c)
+[<c0334b44>] (bus_probe_device) from [<c0334f78>] (deferred_probe_work_func+0x60/0x8c)
+[<c0334f78>] (deferred_probe_work_func) from [<c003a6fc>] (process_one_work+0x120/0x328)
 [<c003a6fc>] (process_one_work) from [<c003a968>] (worker_thread+0x2c/0x4b4)
 [<c003a968>] (worker_thread) from [<c003faa0>] (kthread+0xd8/0xf4)
 [<c003faa0>] (kthread) from [<c000f938>] (ret_from_fork+0x14/0x3c)
----[ end trace 6cd34ce53e19eed7 ]---
+---[ end trace 08aa520dd97eace2 ]---
 scsi host0: tegra-ahci
 scsi host1: tegra-ahci
 ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 341
@@ -458,7 +385,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:05 UTC (946684925)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:01:46 UTC (946684906)
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
@@ -475,9 +402,9 @@
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 devtmpfs: mounted
 Freeing unused kernel memory: 344K (c0982000 - c09d8000)
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    5.730727] asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:11:31:3b
+root@(none):/# [    6.720716] asix 2-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:11:31:3b
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-01-14 20:54:14.588607752 -0800
+++ /tmp/foob	2016-01-14 20:54:14.592607747 -0800
@@ -47,20 +47,20 @@
 	 #################################################################
 	 #################################################################
 	 ######
-	 4 MiB/s
+	 4.1 MiB/s
 done
-Bytes transferred = 6760936 (6729e8 hex)
+Bytes transferred = 6761088 (672a80 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 4 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x6729e8 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x672a80 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-next-20160114-67327-g0908339 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Jan 13 20:47:59 PST 2016
+Linux version 4.4.0-next-20160115-67505-g39750fe (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Jan 14 20:47:56 PST 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -324,24 +324,24 @@
 mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
  mmcblk0: p1
+mmc1: new high speed SD card at address b368
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 hub 2-0:1.0: USB hub found
+ mmcblk1: p1 p2
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-mmc1: new high speed SD card at address b368
-rtc-em3027 2-0056: setting system clock to 2014-12-28 03:43:42 UTC (1419738222)
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
- mmcblk1: p1 p2
+rtc-em3027 2-0056: setting system clock to 2014-12-28 03:51:24 UTC (1419738684)
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 hub 1-1:1.0: USB hub found
+hub 1-1:1.0: 4 ports detected
+usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 1056K (c0e42000 - c0f4a000)
-hub 1-1:1.0: 4 ports detected
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-01-14 20:54:14.596607743 -0800
+++ /tmp/foob	2016-01-14 20:54:14.600607738 -0800
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -45,20 +45,20 @@
 	 #################################################################
 	 #################################################################
 	 #################################
-	 4.1 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 5253840 (502ad0 hex)
+Bytes transferred = 5254048 (502ba0 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x502ad0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x502ba0 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-next-20160114-67327-g0908339 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Jan 13 20:48:11 PST 2016
+Linux version 4.4.0-next-20160115-67505-g39750fe (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Jan 14 20:48:10 PST 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -303,18 +303,21 @@
 tlv320aic23-codec 2-001a: Control not supported for path MICIN -> [NULL] -> Mic Input
 tlv320aic23-codec 2-001a: ASoC: no dapm match for MICIN --> NULL --> Mic Input
 tlv320aic23-codec 2-001a: ASoC: Failed to add route MICIN -> NULL -> Mic Input
-tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
-NET: Registered protocol family 10
-mip6: Mobile IPv6
 mmc0: host does not support reading read-only switch, assuming write-enable
-mmc1: new high speed SD card at address b368
 mmc0: new high speed SDHC card at address b368
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
+tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
+ mmcblk0: p1
+NET: Registered protocol family 10
+mip6: Mobile IPv6
 sit: IPv6 over IPv4 tunneling driver
+mmc1: new high speed SD card at address b368
 NET: Registered protocol family 17
-mmcblk0: mmc1:b368 NCard 1.83 GiB (ro)
 NET: Registered protocol family 15
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 can: controller area network core (rev 20120528 abi 9)
+ mmcblk1: p1 p2
 NET: Registered protocol family 29
 can: raw protocol (rev 20120528)
 can: broadcast manager protocol (rev 20120528 t)
@@ -330,26 +333,61 @@
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
+------------[ cut here ]------------
+WARNING: CPU: 0 PID: 78 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
+workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
+Modules linked in:
+CPU: 0 PID: 78 Comm: kworker/u4:2 Not tainted 4.4.0-next-20160115-67505-g39750fe #1
+Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
+Workqueue: deferwq deferred_probe_work_func
+[<c0017208>] (unwind_backtrace) from [<c0012874>] (show_stack+0x10/0x14)
+[<c0012874>] (show_stack) from [<c023c1c4>] (dump_stack+0x84/0xd0)
+[<c023c1c4>] (dump_stack) from [<c0026b70>] (warn_slowpath_common+0x74/0xb0)
+[<c0026b70>] (warn_slowpath_common) from [<c0026c40>] (warn_slowpath_fmt+0x30/0x40)
+[<c0026c40>] (warn_slowpath_fmt) from [<c00397d4>] (check_flush_dependency+0xc8/0x12c)
+[<c00397d4>] (check_flush_dependency) from [<c00398f0>] (flush_work+0x54/0x11c)
+[<c00398f0>] (flush_work) from [<c00c0884>] (lru_add_drain_all+0x124/0x180)
+[<c00c0884>] (lru_add_drain_all) from [<c00f1728>] (migrate_prep+0x8/0x10)
+[<c00f1728>] (migrate_prep) from [<c00bb3fc>] (alloc_contig_range+0xc4/0x2f8)
+[<c00bb3fc>] (alloc_contig_range) from [<c00f3298>] (cma_alloc+0xd8/0x1a0)
+[<c00f3298>] (cma_alloc) from [<c001c270>] (__alloc_from_contiguous+0x38/0xdc)
+[<c001c270>] (__alloc_from_contiguous) from [<c001cf38>] (__dma_alloc+0x228/0x2ac)
+[<c001cf38>] (__dma_alloc) from [<c001d040>] (arm_dma_alloc+0x48/0x54)
+[<c001d040>] (arm_dma_alloc) from [<c00edb34>] (dma_pool_alloc+0x28c/0x304)
+[<c00edb34>] (dma_pool_alloc) from [<c03fd2ec>] (ehci_qh_alloc+0x3c/0xb4)
+[<c03fd2ec>] (ehci_qh_alloc) from [<c04014d8>] (ehci_setup+0x154/0x478)
+[<c04014d8>] (ehci_setup) from [<c0405220>] (tegra_ehci_reset+0xc/0x50)
+[<c0405220>] (tegra_ehci_reset) from [<c03e8cd0>] (usb_add_hcd+0x2fc/0x804)
+[<c03e8cd0>] (usb_add_hcd) from [<c0404b98>] (tegra_ehci_probe+0x3b8/0x504)
+[<c0404b98>] (tegra_ehci_probe) from [<c0336ccc>] (platform_drv_probe+0x50/0xac)
+[<c0336ccc>] (platform_drv_probe) from [<c0335734>] (driver_probe_device+0x1f4/0x2b0)
+[<c0335734>] (driver_probe_device) from [<c0333cc8>] (bus_for_each_drv+0x44/0x8c)
+[<c0333cc8>] (bus_for_each_drv) from [<c03354b8>] (__device_attach+0x9c/0x100)
+[<c03354b8>] (__device_attach) from [<c0334b44>] (bus_probe_device+0x84/0x8c)
+[<c0334b44>] (bus_probe_device) from [<c0334f78>] (deferred_probe_work_func+0x60/0x8c)
+[<c0334f78>] (deferred_probe_work_func) from [<c003a6fc>] (process_one_work+0x120/0x328)
+[<c003a6fc>] (process_one_work) from [<c003a968>] (worker_thread+0x2c/0x4b4)
+[<c003a968>] (worker_thread) from [<c003faa0>] (kthread+0xd8/0xf4)
+[<c003faa0>] (kthread) from [<c000f938>] (ret_from_fork+0x14/0x3c)
+---[ end trace 3e050a61ffc4d5e1 ]---
+hub 1-1:1.0: USB hub found
+hub 1-1:1.0: 4 ports detected
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
-mmcblk1: mmc0:b368 NCard 3.70 GiB 
- mmcblk1: p1
- mmcblk0: p1 p2
+usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
-hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-28 03:44:01 UTC (1419738241)
+rtc-em3027 2-0056: setting system clock to 2014-12-28 03:52:16 UTC (1419738736)
 ALSA device list:
   #0: tegra-trimslice
-EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
-EXT4-fs (mmcblk1p1): recovery complete
-EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:17.
+EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
+EXT4-fs (mmcblk0p1): recovery complete
+EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
+VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 344K (c0982000 - c09d8000)
-usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
+asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-01-14 20:54:14.608607734 -0800
+++ /tmp/foob	2016-01-14 20:54:14.608607734 -0800
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 ######
-	 3.3 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 6760936 (6729e8 hex)
+Bytes transferred = 6761088 (672a80 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6729e8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x672a80 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-next-20160114-67327-g0908339 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Jan 13 20:47:59 PST 2016
+Linux version 4.4.0-next-20160115-67505-g39750fe (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Jan 14 20:47:56 PST 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -287,8 +287,8 @@
 mmcblk0boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk0rpmb: mmc1:0001 SEM16G partition 3 128 KiB
 Primary GPT is invalid, using alternate GPT.
-tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
  mmcblk0: p1 p2 p3 p4 p5
+tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -341,10 +341,10 @@
 tegra-ehci 7d004000.usb: EHCI Host Controller
 tegra-ehci 7d004000.usb: new USB bus registered, assigned bus number 1
 ------------[ cut here ]------------
-WARNING: CPU: 2 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
+WARNING: CPU: 1 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 2 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160114-67327-g0908339 #1
+CPU: 1 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160115-67505-g39750fe #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c02188b8>] (unwind_backtrace) from [<c0213f94>] (show_stack+0x10/0x14)
@@ -361,22 +361,22 @@
 [<c021e39c>] (__alloc_from_contiguous) from [<c021ff8c>] (__dma_alloc+0x21c/0x250)
 [<c021ff8c>] (__dma_alloc) from [<c0220044>] (arm_dma_alloc+0x48/0x54)
 [<c0220044>] (arm_dma_alloc) from [<c0306c38>] (dma_pool_alloc+0x1dc/0x210)
-[<c0306c38>] (dma_pool_alloc) from [<c07c7008>] (ehci_qh_alloc+0x3c/0xb4)
-[<c07c7008>] (ehci_qh_alloc) from [<c07cb600>] (ehci_setup+0x154/0x478)
-[<c07cb600>] (ehci_setup) from [<c07d16b8>] (tegra_ehci_reset+0xc/0x50)
-[<c07d16b8>] (tegra_ehci_reset) from [<c07a1a80>] (usb_add_hcd+0x300/0x830)
-[<c07a1a80>] (usb_add_hcd) from [<c07d1030>] (tegra_ehci_probe+0x3b8/0x504)
-[<c07d1030>] (tegra_ehci_probe) from [<c0659274>] (platform_drv_probe+0x50/0xac)
-[<c0659274>] (platform_drv_probe) from [<c0657be0>] (driver_probe_device+0x1f4/0x2b0)
-[<c0657be0>] (driver_probe_device) from [<c0656170>] (bus_for_each_drv+0x44/0x8c)
-[<c0656170>] (bus_for_each_drv) from [<c0657964>] (__device_attach+0x9c/0x100)
-[<c0657964>] (__device_attach) from [<c0656ff0>] (bus_probe_device+0x84/0x8c)
-[<c0656ff0>] (bus_probe_device) from [<c0657424>] (deferred_probe_work_func+0x60/0x8c)
-[<c0657424>] (deferred_probe_work_func) from [<c025ebb8>] (process_one_work+0x134/0x338)
+[<c0306c38>] (dma_pool_alloc) from [<c07c7024>] (ehci_qh_alloc+0x3c/0xb4)
+[<c07c7024>] (ehci_qh_alloc) from [<c07cb61c>] (ehci_setup+0x154/0x478)
+[<c07cb61c>] (ehci_setup) from [<c07d16d4>] (tegra_ehci_reset+0xc/0x50)
+[<c07d16d4>] (tegra_ehci_reset) from [<c07a1a9c>] (usb_add_hcd+0x300/0x830)
+[<c07a1a9c>] (usb_add_hcd) from [<c07d104c>] (tegra_ehci_probe+0x3b8/0x504)
+[<c07d104c>] (tegra_ehci_probe) from [<c0659290>] (platform_drv_probe+0x50/0xac)
+[<c0659290>] (platform_drv_probe) from [<c0657bfc>] (driver_probe_device+0x1f4/0x2b0)
+[<c0657bfc>] (driver_probe_device) from [<c065618c>] (bus_for_each_drv+0x44/0x8c)
+[<c065618c>] (bus_for_each_drv) from [<c0657980>] (__device_attach+0x9c/0x100)
+[<c0657980>] (__device_attach) from [<c065700c>] (bus_probe_device+0x84/0x8c)
+[<c065700c>] (bus_probe_device) from [<c0657440>] (deferred_probe_work_func+0x60/0x8c)
+[<c0657440>] (deferred_probe_work_func) from [<c025ebb8>] (process_one_work+0x134/0x338)
 [<c025ebb8>] (process_one_work) from [<c025ee20>] (worker_thread+0x2c/0x4f0)
 [<c025ee20>] (worker_thread) from [<c0263fac>] (kthread+0xd8/0xf4)
 [<c0263fac>] (kthread) from [<c0210e78>] (ret_from_fork+0x14/0x3c)
----[ end trace acc58cfb9cd77fab ]---
+---[ end trace a2202902597f48d2 ]---
 tegra-ehci 7d004000.usb: irq 343, io mem 0x7d004000
 tegra-ehci 7d004000.usb: USB 2.0 started, EHCI 1.00
 hub 1-0:1.0: USB hub found
@@ -399,4 +399,4 @@
 Freeing unused kernel memory: 1056K (c0e42000 - c0f4a000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.003515] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.013527] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-01-14 20:54:14.616607728 -0800
+++ /tmp/foob	2016-01-14 20:54:14.616607728 -0800
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 #################################
-	 3.4 MiB/s
+	 2.8 MiB/s
 done
-Bytes transferred = 5253840 (502ad0 hex)
+Bytes transferred = 5254048 (502ba0 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.3 MiB/s
+	 2.6 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x502ad0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x502ba0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-next-20160114-67327-g0908339 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Jan 13 20:48:11 PST 2016
+Linux version 4.4.0-next-20160115-67505-g39750fe (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Jan 14 20:48:10 PST 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -339,10 +339,10 @@
 tegra-ehci 7d004000.usb: EHCI Host Controller
 tegra-ehci 7d004000.usb: new USB bus registered, assigned bus number 1
 ------------[ cut here ]------------
-WARNING: CPU: 3 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
+WARNING: CPU: 0 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 3 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160114-67327-g0908339 #1
+CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160115-67505-g39750fe #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0017208>] (unwind_backtrace) from [<c0012874>] (show_stack+0x10/0x14)
@@ -359,22 +359,22 @@
 [<c001c270>] (__alloc_from_contiguous) from [<c001cf38>] (__dma_alloc+0x228/0x2ac)
 [<c001cf38>] (__dma_alloc) from [<c001d040>] (arm_dma_alloc+0x48/0x54)
 [<c001d040>] (arm_dma_alloc) from [<c00edb34>] (dma_pool_alloc+0x28c/0x304)
-[<c00edb34>] (dma_pool_alloc) from [<c03fd2d0>] (ehci_qh_alloc+0x3c/0xb4)
-[<c03fd2d0>] (ehci_qh_alloc) from [<c04014bc>] (ehci_setup+0x154/0x478)
-[<c04014bc>] (ehci_setup) from [<c0405204>] (tegra_ehci_reset+0xc/0x50)
-[<c0405204>] (tegra_ehci_reset) from [<c03e8cb4>] (usb_add_hcd+0x2fc/0x804)
-[<c03e8cb4>] (usb_add_hcd) from [<c0404b7c>] (tegra_ehci_probe+0x3b8/0x504)
-[<c0404b7c>] (tegra_ehci_probe) from [<c0336cb0>] (platform_drv_probe+0x50/0xac)
-[<c0336cb0>] (platform_drv_probe) from [<c0335718>] (driver_probe_device+0x1f4/0x2b0)
-[<c0335718>] (driver_probe_device) from [<c0333cac>] (bus_for_each_drv+0x44/0x8c)
-[<c0333cac>] (bus_for_each_drv) from [<c033549c>] (__device_attach+0x9c/0x100)
-[<c033549c>] (__device_attach) from [<c0334b28>] (bus_probe_device+0x84/0x8c)
-[<c0334b28>] (bus_probe_device) from [<c0334f5c>] (deferred_probe_work_func+0x60/0x8c)
-[<c0334f5c>] (deferred_probe_work_func) from [<c003a6fc>] (process_one_work+0x120/0x328)
+[<c00edb34>] (dma_pool_alloc) from [<c03fd2ec>] (ehci_qh_alloc+0x3c/0xb4)
+[<c03fd2ec>] (ehci_qh_alloc) from [<c04014d8>] (ehci_setup+0x154/0x478)
+[<c04014d8>] (ehci_setup) from [<c0405220>] (tegra_ehci_reset+0xc/0x50)
+[<c0405220>] (tegra_ehci_reset) from [<c03e8cd0>] (usb_add_hcd+0x2fc/0x804)
+[<c03e8cd0>] (usb_add_hcd) from [<c0404b98>] (tegra_ehci_probe+0x3b8/0x504)
+[<c0404b98>] (tegra_ehci_probe) from [<c0336ccc>] (platform_drv_probe+0x50/0xac)
+[<c0336ccc>] (platform_drv_probe) from [<c0335734>] (driver_probe_device+0x1f4/0x2b0)
+[<c0335734>] (driver_probe_device) from [<c0333cc8>] (bus_for_each_drv+0x44/0x8c)
+[<c0333cc8>] (bus_for_each_drv) from [<c03354b8>] (__device_attach+0x9c/0x100)
+[<c03354b8>] (__device_attach) from [<c0334b44>] (bus_probe_device+0x84/0x8c)
+[<c0334b44>] (bus_probe_device) from [<c0334f78>] (deferred_probe_work_func+0x60/0x8c)
+[<c0334f78>] (deferred_probe_work_func) from [<c003a6fc>] (process_one_work+0x120/0x328)
 [<c003a6fc>] (process_one_work) from [<c003a968>] (worker_thread+0x2c/0x4b4)
 [<c003a968>] (worker_thread) from [<c003faa0>] (kthread+0xd8/0xf4)
 [<c003faa0>] (kthread) from [<c000f938>] (ret_from_fork+0x14/0x3c)
----[ end trace 81dbbff40788701c ]---
+---[ end trace a4c29e583b2818e5 ]---
 tegra-ehci 7d004000.usb: irq 343, io mem 0x7d004000
 tegra-ehci 7d004000.usb: USB 2.0 started, EHCI 1.00
 hub 1-0:1.0: USB hub found
@@ -399,4 +399,4 @@
 Freeing unused kernel memory: 344K (c0982000 - c09d8000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.460537] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.440522] usb 2-1: new high-speed USB device number 2 using tegra-ehci
