Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-03-28 18:26:23.143209521 -0700
+++ /tmp/foob	2016-03-28 18:26:23.143209521 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1-next-20160327+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Mar 27 03:50:07 PDT 2016
+Linux version 4.6.0-rc1-next-20160329+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Mar 28 18:20:30 PDT 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -94,7 +94,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.632:1): initialized
+audit: type=2000 audit(0.712:1): initialized
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -154,7 +154,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2016-03-27 10:50:37 UTC (1459075837)
+rtc-pl031 9010000.pl031: setting system clock to 2016-03-29 01:21:00 UTC (1459214460)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-03-28 18:26:23.151209517 -0700
+++ /tmp/foob	2016-03-28 18:26:23.155209515 -0700
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 #################
-	 3.4 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 6918824 (6992a8 hex)
+Bytes transferred = 6926312 (69afe8 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3.3 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 34912 (8860 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6992a8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x69afe8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff85f ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1-next-20160327-82947-ga892f4b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Mar 27 03:48:43 PDT 2016
+Linux version 4.6.0-rc1-next-20160329-83440-g682b931 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Mar 28 18:19:35 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141b480, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c141b600, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1995132K/2096128K available (9522K kernel code, 1141K rwdata, 4108K rodata, 2048K init, 341K bss, 35460K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1995132K/2096128K available (9523K kernel code, 1141K rwdata, 4112K rodata, 2048K init, 341K bss, 35460K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,9 +90,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc104fabc   (14623 kB)
+      .text : 0xc0208000 - 0xc1050c04   (14628 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141d4c0   (1142 kB)
+      .data : 0xc1300000 - 0xc141d640   (1142 kB)
        .bss : 0xc141f000 - 0xc1474558   ( 342 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -158,6 +158,9 @@
 genirq: Setting trigger mode 8 for irq 341 failed (irq_chip_set_type_parent+0x0/0x28)
 palmas 2-0058: Irq flag is 0x00000000
 palmas 2-0058: Muxing GPIO ff, PWM 0, LED 0
+sdhci-vdd-sd-slot: ramp_delay not set
+rail-3v3: Voltage range but no REGULATOR_CHANGE_VOLTAGE
+rail-5v0: Voltage range but no REGULATOR_CHANGE_VOLTAGE
 pps_core: LinuxPPS API ver. 1 registered
 pps_core: Software ver. 5.3.6 - Copyright 2005-2007 Rodolfo Giometti <giometti@linux.it>
 PTP clock support registered
@@ -276,15 +279,15 @@
 VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
 vdd_hdmi_5v0: supplied by vdd-sys-5v0
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
-[drm] No driver support for vblank timestamp query.
 mmc0: new high speed SDHC card at address aaaa
 mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
+[drm] No driver support for vblank timestamp query.
+ mmcblk0: p1 p2 p3
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
- mmcblk0: p1 p2 p3
  mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
 Console: switching to colour frame buffer device 240x75
 drm drm: fb0:  frame buffer device
@@ -297,7 +300,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-19 22:08:15 UTC (948319695)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-21 12:39:01 UTC (948458341)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -309,5 +312,4 @@
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
-root@(none):/# 
\ No newline at end of file
+root@(none):/# [    2.234154] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-03-28 18:26:23.159209512 -0700
+++ /tmp/foob	2016-03-28 18:26:23.159209512 -0700
@@ -39,18 +39,18 @@
 	 ######################################
 	 3.3 MiB/s
 done
-Bytes transferred = 5316552 (511fc8 hex)
+Bytes transferred = 5315960 (511d78 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 34912 (8860 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x511fc8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x511d78 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff85f ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1-next-20160327-82947-ga892f4b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Mar 27 03:48:30 PDT 2016
+Linux version 4.6.0-rc1-next-20160329-83440-g682b931 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Mar 28 18:18:56 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -90,7 +90,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab220c   (10921 kB)
+      .text : 0xc0008000 - 0xc0ab2258   (10921 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
       .data : 0xc0c00000 - 0xc0c5df80   ( 376 kB)
        .bss : 0xc0c5f000 - 0xc0cb9a48   ( 363 kB)
@@ -154,6 +154,9 @@
 genirq: Setting trigger mode 8 for irq 341 failed (irq_chip_set_type_parent+0x0/0x28)
 palmas 2-0058: Irq flag is 0x00000000
 palmas 2-0058: Muxing GPIO ff, PWM 0, LED 0
+sdhci-vdd-sd-slot: ramp_delay not set
+rail-3v3: Voltage range but no REGULATOR_CHANGE_VOLTAGE
+rail-5v0: Voltage range but no REGULATOR_CHANGE_VOLTAGE
 Linux video capture interface: v2.00
 pps_core: LinuxPPS API ver. 1 registered
 pps_core: Software ver. 5.3.6 - Copyright 2005-2007 Rodolfo Giometti <giometti@linux.it>
@@ -249,15 +252,15 @@
 mmc1: SDHCI controller on 78000600.sdhci [78000600.sdhci] using ADMA
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
-mmc0: new high speed SDHC card at address aaaa
-mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 mmc1: new high speed MMC card at address 0001
-mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
-mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
-mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
-mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
- mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
- mmcblk0: p1 p2 p3
+mmcblk0: mmc1:0001 SEM32G 29.1 GiB 
+mmcblk0boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
+mmc0: new high speed SDHC card at address aaaa
+mmcblk1: mmc0:aaaa SL16G 14.8 GiB 
+mmcblk0boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
+mmcblk0rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
+ mmcblk1: p1 p2 p3
 tegra30-i2s 70080400.i2s: DMA channels sourced from device 70080000.ahub
 tegra-snd-rt5640 sound: rt5640-aif1 <-> 70080400.i2s mapping ok
 input: NVIDIA Tegra Dalmore Headphones as /devices/soc0/sound/sound/card0/input0
@@ -295,18 +298,19 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-19 22:05:53 UTC (948319553)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-21 12:36:23 UTC (948458183)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
 ALSA device list:
   #0: NVIDIA Tegra Dalmore
-EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
-EXT4-fs (mmcblk1p1): recovery complete
-EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:17.
+EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
+EXT4-fs (mmcblk0p1): recovery complete
+EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
+VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.375244] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-03-28 18:26:23.167209510 -0700
+++ /tmp/foob	2016-03-28 18:26:23.167209510 -0700
@@ -29,6 +29,7 @@
 Tegra124 (Jetson TK1) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash'
 Tegra124 (Jetson TK1) # setenv ipaddr 192.168.100.102; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 zImage.multi_v7_defconfig
+Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'zImage.multi_v7_defconfig'.
@@ -41,20 +42,20 @@
 	 #################################################################
 	 #################################################################
 	 #################
-	 3.2 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 6918824 (6992a8 hex)
+Bytes transferred = 6926312 (69afe8 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.1 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 64236 (faec hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6992a8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x69afe8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012aeb
@@ -62,7 +63,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1-next-20160327-82947-ga892f4b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Mar 27 03:48:43 PDT 2016
+Linux version 4.6.0-rc1-next-20160329-83440-g682b931 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Mar 28 18:19:35 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -71,7 +72,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141b480, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c141b600, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -84,7 +85,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1995024K/2096128K available (9522K kernel code, 1141K rwdata, 4108K rodata, 2048K init, 341K bss, 35568K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1995024K/2096128K available (9523K kernel code, 1141K rwdata, 4112K rodata, 2048K init, 341K bss, 35568K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,9 +93,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc104fabc   (14623 kB)
+      .text : 0xc0208000 - 0xc1050c04   (14628 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141d4c0   (1142 kB)
+      .data : 0xc1300000 - 0xc141d640   (1142 kB)
        .bss : 0xc141f000 - 0xc1474558   ( 342 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -293,14 +294,8 @@
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
 tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
 tegra-ahci 70027000.sata: Failed to get regulators
-+5V_USB_HS: supplied by +5V_SYS
 mmc0: new HS200 MMC card at address 0001
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
-Primary GPT is invalid, using alternate GPT.
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9
++5V_USB_HS: supplied by +5V_SYS
 tegra-ehci 7d008000.usb: EHCI Host Controller
 tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci 7d008000.usb: irq 351, io mem 0x7d008000
@@ -341,14 +336,52 @@
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:16 UTC (946684816)
 mmc1: Unknown controller version (3). You may experience problems.
+cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 204000000, volt: 900000, enabled: 1. New: freq: 204000000, volt: 900000, enabled: 1
+cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 306000000, volt: 900000, enabled: 1. New: freq: 306000000, volt: 900000, enabled: 1
+cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 408000000, volt: 900000, enabled: 1. New: freq: 408000000, volt: 900000, enabled: 1
+cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 510000000, volt: 900000, enabled: 1. New: freq: 510000000, volt: 900000, enabled: 1
+cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 612000000, volt: 900000, enabled: 1. New: freq: 612000000, volt: 900000, enabled: 1
+cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 714000000, volt: 900000, enabled: 1. New: freq: 714000000, volt: 900000, enabled: 1
+cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 816000000, volt: 900000, enabled: 1. New: freq: 816000000, volt: 900000, enabled: 1
+cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 918000000, volt: 900000, enabled: 1. New: freq: 918000000, volt: 900000, enabled: 1
+cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1020000000, volt: 900000, enabled: 1. New: freq: 1020000000, volt: 900000, enabled: 1
+cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1122000000, volt: 900000, enabled: 1. New: freq: 1122000000, volt: 900000, enabled: 1
+cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1224000000, volt: 900000, enabled: 1. New: freq: 1224000000, volt: 900000, enabled: 1
+cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1326000000, volt: 900000, enabled: 1. New: freq: 1326000000, volt: 900000, enabled: 1
+cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1428000000, volt: 900000, enabled: 1. New: freq: 1428000000, volt: 900000, enabled: 1
+cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1530000000, volt: 900000, enabled: 1. New: freq: 1530000000, volt: 900000, enabled: 1
+cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1632000000, volt: 920000, enabled: 1. New: freq: 1632000000, volt: 920000, enabled: 1
+cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1734000000, volt: 960000, enabled: 1. New: freq: 1734000000, volt: 960000, enabled: 1
+cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1836000000, volt: 990000, enabled: 1. New: freq: 1836000000, volt: 990000, enabled: 1
+cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1938000000, volt: 1030000, enabled: 1. New: freq: 1938000000, volt: 1030000, enabled: 1
+cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2014500000, volt: 1060000, enabled: 1. New: freq: 2014500000, volt: 1060000, enabled: 1
+cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2116500000, volt: 1100000, enabled: 1. New: freq: 2116500000, volt: 1100000, enabled: 1
+cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2218500000, volt: 1140000, enabled: 1. New: freq: 2218500000, volt: 1140000, enabled: 1
+cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 2320500000, volt: 1180000, enabled: 1. New: freq: 2320500000, volt: 1180000, enabled: 1
+tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
+tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
+tegra-pcie 1003000.pcie-controller: Failed to get supply 'avddio-pex': -517
+tegra-hdmi 54280000.hdmi: failed to get PLL regulator
+tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
+tegra-ahci 70027000.sata: Failed to get regulators
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:11 UTC (946684931)
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +USB0_VBUS_SW: disabling
 +1.05V_RUN_AVDD_HDMI_PLL: disabling
 +5V_HDMI_CON: disabling
 +5V_SATA: disabling
 +12V_SATA: disabling
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+Primary GPT is invalid, using alternate GPT.
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9
+sdhci-tegra 700b0400.sdhci: Got CD GPIO
+sdhci-tegra 700b0400.sdhci: Got WP GPIO
+mmc1: Unknown controller version (3). You may experience problems.
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 204000000, volt: 900000, enabled: 1. New: freq: 204000000, volt: 900000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 306000000, volt: 900000, enabled: 1. New: freq: 306000000, volt: 900000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 408000000, volt: 900000, enabled: 1. New: freq: 408000000, volt: 900000, enabled: 1
@@ -360,7 +393,6 @@
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1020000000, volt: 900000, enabled: 1. New: freq: 1020000000, volt: 900000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1122000000, volt: 900000, enabled: 1. New: freq: 1122000000, volt: 900000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1224000000, volt: 900000, enabled: 1. New: freq: 1224000000, volt: 900000, enabled: 1
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1326000000, volt: 900000, enabled: 1. New: freq: 1326000000, volt: 900000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1428000000, volt: 900000, enabled: 1. New: freq: 1428000000, volt: 900000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 1530000000, volt: 900000, enabled: 1. New: freq: 1530000000, volt: 900000, enabled: 1
@@ -379,11 +411,10 @@
 tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
 tegra-ahci 70027000.sata: Failed to get regulators
 asix 2-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:11:31:3b
+EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc1: Unknown controller version (3). You may experience problems.
-EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
-EXT4-fs (mmcblk0p1): couldn't mount as ext2 due to feature incompatibilities
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 204000000, volt: 900000, enabled: 1. New: freq: 204000000, volt: 900000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 306000000, volt: 900000, enabled: 1. New: freq: 306000000, volt: 900000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 408000000, volt: 900000, enabled: 1. New: freq: 408000000, volt: 900000, enabled: 1
@@ -412,6 +443,7 @@
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
 tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
 tegra-ahci 70027000.sata: Failed to get regulators
+EXT4-fs (mmcblk0p1): couldn't mount as ext2 due to feature incompatibilities
 EXT4-fs (mmcblk0p1): warning: mounting fs with errors, running e2fsck is recommended
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-03-28 18:26:23.175209505 -0700
+++ /tmp/foob	2016-03-28 18:26:23.175209505 -0700
@@ -41,18 +41,18 @@
 	 ######################################
 	 3.2 MiB/s
 done
-Bytes transferred = 5316552 (511fc8 hex)
+Bytes transferred = 5315960 (511d78 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.4 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 64236 (faec hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x511fc8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x511d78 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012aeb
@@ -61,7 +61,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1-next-20160327-82947-ga892f4b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Mar 27 03:48:30 PDT 2016
+Linux version 4.6.0-rc1-next-20160329-83440-g682b931 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Mar 28 18:18:56 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -92,7 +92,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab220c   (10921 kB)
+      .text : 0xc0008000 - 0xc0ab2258   (10921 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
       .data : 0xc0c00000 - 0xc0c5df80   ( 376 kB)
        .bss : 0xc0c5f000 - 0xc0cb9a48   ( 363 kB)
@@ -344,8 +344,8 @@
 tegra-ahci 70027000.sata: Failed to get regulators
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
-sdhci-tegra 700b0400.sdhci: Got WP GPIO
 as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:15 UTC (946684815)
+sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc1: Unknown controller version (3). You may experience problems.
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 204000000, volt: 900000, enabled: 1. New: freq: 204000000, volt: 900000, enabled: 1
 +3.3V_AVDD_HDMI_AP_GATED: disabling
@@ -386,11 +386,17 @@
 tegra-ahci 70027000.sata: Failed to get regulators
 asix 2-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:11:31:3b
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
+EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc1: Unknown controller version (3). You may experience problems.
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 204000000, volt: 900000, enabled: 1. New: freq: 204000000, volt: 900000, enabled: 1
+EXT4-fs (mmcblk0p1): warning: mounting fs with errors, running e2fsck is recommended
+EXT4-fs (mmcblk0p1): recovery complete
+EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
+VFS: Mounted root (ext4 filesystem) on device 179:1.
+devtmpfs: mounted
+Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 306000000, volt: 900000, enabled: 1. New: freq: 306000000, volt: 900000, enabled: 1
-EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 408000000, volt: 900000, enabled: 1. New: freq: 408000000, volt: 900000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 510000000, volt: 900000, enabled: 1. New: freq: 510000000, volt: 900000, enabled: 1
 cpu cpu0: _opp_add: duplicate OPPs detected. Existing: freq: 612000000, volt: 900000, enabled: 1. New: freq: 612000000, volt: 900000, enabled: 1
@@ -417,15 +423,6 @@
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
 tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
 tegra-ahci 70027000.sata: Failed to get regulators
-sdhci-tegra 700b0400.sdhci: Got CD GPIO
-EXT4-fs (mmcblk0p1): warning: mounting fs with errors, running e2fsck is recommended
-sdhci-tegra 700b0400.sdhci: Got WP GPIO
-mmc1: Unknown controller version (3). You may experience problems.
-EXT4-fs (mmcblk0p1): recovery complete
-EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext4 filesystem) on device 179:1.
-devtmpfs: mounted
-Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2016-03-28 18:26:23.183209499 -0700
+++ /tmp/foob	2016-03-28 18:26:23.183209499 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1800085 us
+Bootloader-Cpu Init at (time stamp): 1799696 us
 
 Pinmux changes applied in kernel way
 
@@ -62,22 +62,22 @@
 Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1800059  us
-Bootloader (AosInit) time: 734247 us
-Bootloader Main Init time: 180090 us
-BlockDevMgrInit time: 157534 us
+Miscellaneous avp time: 1799670  us
+Bootloader (AosInit) time: 734220 us
+Bootloader Main Init time: 183955 us
+BlockDevMgrInit time: 157609 us
 PartMgrInit time: 8 us
-StorMgrInit time: 30 us
+StorMgrInit time: 31 us
 FsMgrInit time: 2 us
-Partition Table load: 76692 us 
-Display init time: 574841 us
+Partition Table load: 77259 us 
+Display init time: 574600 us
 Kernel read time: 301 msec
 Load Kernel Size:   13053952 Bytes
 Bl Kernel ReadSpeed: 43368000 Bytes/sec
-Pre-Kernel Jump time: 1489537 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5493572 us
+Pre-Kernel Jump time: 1489166 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5497918 us
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1-next-20160327+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Mar 27 03:50:07 PDT 2016
+Linux version 4.6.0-rc1-next-20160329+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Mar 28 18:20:30 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
 Memory limited to 1734MB
 efi: Getting EFI parameters from FDT:
@@ -179,7 +179,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.835:1): initialized
+audit: type=2000 audit(0.827:1): initialized
 workingset: timestamp_bits=44 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -243,26 +243,26 @@
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 hctosys: unable to open rtc device (rtc0)
+sdhci-tegra 700b0400.sdhci: Got WP GPIO
+mmc1: Unknown controller version (3). You may experience problems.
++3.3V_AVDD_HDMI_AP_GATED: disabling
++VDD_LED: disabling
++5V_USB_HS: disabling
++5V_USB_SS: disabling
 mmc0: new HS200 MMC card at address 0001
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
-+3.3V_AVDD_HDMI_AP_GATED: disabling
-+VDD_LED: disabling
-+5V_USB_HS: disabling
-+5V_USB_SS: disabling
 +1.05V_RUN_AVDD_HDMI_PLL_AP_GATE: disabling
 +5V_HDMI_CON: disabling
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 ALSA device list:
-  No soundcards found.
-Freeing unused kernel memory: 812K (ffffff8008a01000 - ffffff8008acc000)
-sdhci-tegra 700b0400.sdhci: Got WP GPIO
-mmc1: Unknown controller version (3). You may experience problems.
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc1: Unknown controller version (3). You may experience problems.
+  No soundcards found.
+Freeing unused kernel memory: 812K (ffffff8008a01000 - ffffff8008acc000)
 
 System initialization...
 
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-03-28 18:26:23.187209498 -0700
+++ /tmp/foob	2016-03-28 18:26:23.191209496 -0700
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -47,20 +47,20 @@
 	 #################################################################
 	 #################################################################
 	 #################
-	 4 MiB/s
+	 906.3 KiB/s
 done
-Bytes transferred = 6918824 (6992a8 hex)
+Bytes transferred = 6926312 (69afe8 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 859.4 KiB/s
 done
 Bytes transferred = 21128 (5288 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x6992a8 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x69afe8 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410287
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1-next-20160327-82947-ga892f4b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Mar 27 03:48:43 PDT 2016
+Linux version 4.6.0-rc1-next-20160329-83440-g682b931 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Mar 28 18:19:35 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -76,7 +76,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c141b480, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c141b600, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -89,7 +89,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 955940K/1048576K available (9522K kernel code, 1141K rwdata, 4108K rodata, 2048K init, 341K bss, 27100K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 955940K/1048576K available (9523K kernel code, 1141K rwdata, 4112K rodata, 2048K init, 341K bss, 27100K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -97,9 +97,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc104fabc   (14623 kB)
+      .text : 0xc0208000 - 0xc1050c04   (14628 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141d4c0   (1142 kB)
+      .data : 0xc1300000 - 0xc141d640   (1142 kB)
        .bss : 0xc141f000 - 0xc1474558   ( 342 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
@@ -326,18 +326,17 @@
  mmcblk0: p1
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
-hub 2-0:1.0: USB hub found
-hub 2-0:1.0: 1 port detected
-input: gpio-keys as /devices/soc0/gpio-keys/input/input0
 mmc1: new high speed SD card at address b368
-rtc-em3027 2-0056: setting system clock to 2014-12-28 13:00:00 UTC (1419771600)
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
+hub 2-0:1.0: USB hub found
+hub 2-0:1.0: 1 port detected
+input: gpio-keys as /devices/soc0/gpio-keys/input/input0
+rtc-em3027 2-0056: setting system clock to 2014-12-28 13:05:36 UTC (1419771936)
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
-asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-03-28 18:26:23.195209495 -0700
+++ /tmp/foob	2016-03-28 18:26:23.199209494 -0700
@@ -45,9 +45,9 @@
 	 #################################################################
 	 #################################################################
 	 ######################################
-	 901.4 KiB/s
+	 908.2 KiB/s
 done
-Bytes transferred = 5316552 (511fc8 hex)
+Bytes transferred = 5315960 (511d78 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21128 (5288 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x511fc8 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x511d78 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410287
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1-next-20160327-82947-ga892f4b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Mar 27 03:48:30 PDT 2016
+Linux version 4.6.0-rc1-next-20160329-83440-g682b931 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Mar 28 18:18:56 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -97,7 +97,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab220c   (10921 kB)
+      .text : 0xc0008000 - 0xc0ab2258   (10921 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
       .data : 0xc0c00000 - 0xc0c5df80   ( 376 kB)
        .bss : 0xc0c5f000 - 0xc0cb9a48   ( 363 kB)
@@ -340,12 +340,11 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-28 12:58:37 UTC (1419771517)
+rtc-em3027 2-0056: setting system clock to 2014-12-28 13:04:09 UTC (1419771849)
 ALSA device list:
   #0: tegra-trimslice
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
-asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2016-03-28 18:26:23.203209490 -0700
+++ /tmp/foob	2016-03-28 18:26:23.207209488 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 804964 us
-Basic modules init took 348615 us
-Sec Bootdevice Read Time = 203 ms, Read Size = 8459 KB
-Next stage binary read took 14366 us
-Carveout took 252768 us
-CPU initialization took 92940 us
-Total time taken by TegraBoot 708689 us
+Tegraboot started after 750702 us
+Basic modules init took 348849 us
+Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
+Next stage binary read took 14361 us
+Carveout took 252713 us
+CPU initialization took 92333 us
+Total time taken by TegraBoot 708256 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.643] LPDDR4 Training: Number of tables = 10
+64b[0001.589] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=/dev/mmcblk0p1 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-11423744 bytes read in 718 ms (15.2 MiB/s)
+11423744 bytes read in 714 ms (15.3 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-37136 bytes read in 453 ms (79.1 KiB/s)
+37136 bytes read in 454 ms (79.1 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 494 ms (3.1 MiB/s)
+1627787 bytes read in 496 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1-next-20160327+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Mar 27 03:50:07 PDT 2016
+Linux version 4.6.0-rc1-next-20160329+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Mar 28 18:20:30 PDT 2016
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -259,7 +259,7 @@
 kvm [1]: timer IRQ3
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.359:1): initialized
+audit: type=2000 audit(0.363:1): initialized
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-03-28 18:26:23.211209486 -0700
+++ /tmp/foob	2016-03-28 18:26:23.215209484 -0700
@@ -41,18 +41,18 @@
 	 #################
 	 2.9 MiB/s
 done
-Bytes transferred = 6918824 (6992a8 hex)
+Bytes transferred = 6926312 (69afe8 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 1.5 MiB/s
+	 2.6 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6992a8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x69afe8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1-next-20160327-82947-ga892f4b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Mar 27 03:48:43 PDT 2016
+Linux version 4.6.0-rc1-next-20160329-83440-g682b931 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Mar 28 18:19:35 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,7 +68,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141b480, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c141b600, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -81,7 +81,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1995052K/2096128K available (9522K kernel code, 1141K rwdata, 4108K rodata, 2048K init, 341K bss, 35540K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1995052K/2096128K available (9523K kernel code, 1141K rwdata, 4112K rodata, 2048K init, 341K bss, 35540K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -89,9 +89,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc104fabc   (14623 kB)
+      .text : 0xc0208000 - 0xc1050c04   (14628 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141d4c0   (1142 kB)
+      .data : 0xc1300000 - 0xc141d640   (1142 kB)
        .bss : 0xc141f000 - 0xc1474558   ( 342 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -281,8 +281,8 @@
 sys_3v3,vdd_3v3_alw: supplied by vdd_5v_in
 sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
 tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
-tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 mmc1: new high speed MMC card at address 0001
+tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 mmcblk0: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk0boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk0boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
@@ -362,4 +362,5 @@
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.723403] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# 
\ No newline at end of file
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-03-28 18:26:23.219209483 -0700
+++ /tmp/foob	2016-03-28 18:26:23.219209483 -0700
@@ -39,7 +39,7 @@
 	 ######################################
 	 3.4 MiB/s
 done
-Bytes transferred = 5316552 (511fc8 hex)
+Bytes transferred = 5315960 (511d78 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x511fc8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x511d78 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-rc1-next-20160327-82947-ga892f4b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Mar 27 03:48:30 PDT 2016
+Linux version 4.6.0-rc1-next-20160329-83440-g682b931 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Mar 28 18:18:56 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -89,7 +89,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab220c   (10921 kB)
+      .text : 0xc0008000 - 0xc0ab2258   (10921 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
       .data : 0xc0c00000 - 0xc0c5df80   ( 376 kB)
        .bss : 0xc0c5f000 - 0xc0cb9a48   ( 363 kB)
@@ -362,4 +362,5 @@
 Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.020606] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# 
\ No newline at end of file
