Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2017-02-10 00:13:38.640887506 -0800
+++ /tmp/foob	2017-02-10 00:13:38.644887523 -0800
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc7-next-20170209-176687-g9f7e70f (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Feb 8 22:54:12 PST 2017
+Linux version 4.10.0-rc7-next-20170210-176877-g632571b (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Fri Feb 10 00:07:47 PST 2017
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -113,7 +113,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.720:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.724:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -179,7 +179,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2017-02-09 06:54:55 UTC (1486623295)
+rtc-pl031 9010000.pl031: setting system clock to 2017-02-10 08:08:33 UTC (1486714113)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-02-10 00:13:38.648887538 -0800
+++ /tmp/foob	2017-02-10 00:13:38.652887553 -0800
@@ -41,18 +41,18 @@
 	 #########################################################
 	 3.1 MiB/s
 done
-Bytes transferred = 7512216 (72a098 hex)
+Bytes transferred = 7510704 (729ab0 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.6 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x72a098 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x729ab0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -60,18 +60,18 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc7-next-20170209-152254-g9f7e70f (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Feb 8 22:50:29 PST 2017
+Linux version 4.10.0-rc7-next-20170210-152444-g632571b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Fri Feb 10 00:04:49 PST 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
-OF: fdt:Machine model: NVIDIA Tegra114 Dalmore evaluation board
+OF: fdt: Machine model: NVIDIA Tegra114 Dalmore evaluation board
 debug: ignoring loglevel setting.
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c153a840, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c153a800, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -94,7 +94,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc153c1e0   (1265 kB)
+      .data : 0xc1400000 - 0xc153c1a0   (1265 kB)
        .bss : 0xc153e000 - 0xc15c4178   ( 537 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -303,14 +303,14 @@
 Key type dns_resolver registered
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
-usb1_vbus: supplied by vdd-sys-5v0
 mmc0: new high speed SDHC card at address aaaa
-mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
+usb1_vbus: supplied by vdd-sys-5v0
 usb2_vbus: supplied by vdd-sys-5v0
 VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
- mmcblk0: p1 p2 p3
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
+mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
+ mmcblk0: p1 p2 p3
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
@@ -327,7 +327,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-12-03 18:24:24 UTC (975867864)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-12-04 19:38:57 UTC (975958737)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -339,4 +339,4 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.529580] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.509558] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2017-02-10 00:13:38.660887584 -0800
+++ /tmp/foob	2017-02-10 00:13:38.660887584 -0800
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 ######################################################
-	 3.3 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 5553864 (54bec8 hex)
+Bytes transferred = 5554352 (54c0b0 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x54bec8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x54c0b0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -59,11 +59,11 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc7-next-20170209-152254-g9f7e70f (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Feb 8 22:50:03 PST 2017
+Linux version 4.10.0-rc7-next-20170210-152444-g632571b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Feb 10 00:04:29 PST 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
-OF: fdt:Machine model: NVIDIA Tegra114 Dalmore evaluation board
+OF: fdt: Machine model: NVIDIA Tegra114 Dalmore evaluation board
 debug: ignoring loglevel setting.
 bootconsole [earlycon0] enabled
 cma: Reserved 64 MiB at 0xfbc00000
@@ -320,7 +320,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-12-03 18:21:46 UTC (975867706)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-12-04 19:36:15 UTC (975958575)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -334,4 +334,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.739097] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.749221] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-02-10 00:13:38.664887599 -0800
+++ /tmp/foob	2017-02-10 00:13:38.668887615 -0800
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MMC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -36,18 +36,18 @@
 	 #########################################################
 	 3.9 MiB/s
 done
-Bytes transferred = 7512216 (72a098 hex)
+Bytes transferred = 7510704 (729ab0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3 MiB/s
+	 3.8 MiB/s
 done
 Bytes transferred = 67133 (1063d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x72a098 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x729ab0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301363c
@@ -55,18 +55,18 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc7-next-20170209-152254-g9f7e70f (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Feb 8 22:50:29 PST 2017
+Linux version 4.10.0-rc7-next-20170210-152444-g632571b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Fri Feb 10 00:04:49 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
-OF: fdt:Machine model: NVIDIA Tegra124 Jetson TK1
+OF: fdt: Machine model: NVIDIA Tegra124 Jetson TK1
 debug: ignoring loglevel setting.
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c153a840, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c153a800, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -89,7 +89,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc153c1e0   (1265 kB)
+      .data : 0xc1400000 - 0xc153c1a0   (1265 kB)
        .bss : 0xc153e000 - 0xc15c4178   ( 537 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -380,8 +380,8 @@
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4c may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x3e may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x3e may corrupt adjacent RW1C bits
-ata1: SATA link down (SStatus 0 SControl 300)
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x4 may corrupt adjacent RW1C bits
+ata1: SATA link down (SStatus 0 SControl 300)
 pci_bus 0000:00: 1-byte config write to 0000:00:01.0 offset 0xc may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4 may corrupt adjacent RW1C bits
 pci_bus 0000:00: 1-byte config write to 0000:00:02.0 offset 0xc may corrupt adjacent RW1C bits
@@ -439,7 +439,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:40 UTC (946684960)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:05 UTC (946684925)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2017-02-10 00:13:38.672887631 -0800
+++ /tmp/foob	2017-02-10 00:13:38.676887645 -0800
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+MMC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -34,7 +34,7 @@
 	 ######################################################
 	 3.9 MiB/s
 done
-Bytes transferred = 5553864 (54bec8 hex)
+Bytes transferred = 5554352 (54c0b0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -45,7 +45,7 @@
 done
 Bytes transferred = 67133 (1063d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x54bec8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x54c0b0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301363c
@@ -54,11 +54,11 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc7-next-20170209-152254-g9f7e70f (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Feb 8 22:50:03 PST 2017
+Linux version 4.10.0-rc7-next-20170210-152444-g632571b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Feb 10 00:04:29 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
-OF: fdt:Machine model: NVIDIA Tegra124 Jetson TK1
+OF: fdt: Machine model: NVIDIA Tegra124 Jetson TK1
 debug: ignoring loglevel setting.
 bootconsole [earlycon0] enabled
 cma: Reserved 64 MiB at 0xfbc00000
@@ -328,8 +328,8 @@
 as3722-ldo4: supplied by +3.3V_SYS
 as3722-ldo5: supplied by +1.8V_VDDIO
 as3722-ldo6: supplied by +3.3V_RUN
-as3722-ldo7: supplied by +1.8V_VDDIO
 input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input1
+as3722-ldo7: supplied by +1.8V_VDDIO
 +1.05V_RUN_CAM_REAR: Bringing 0uV into 1050000-1050000uV
 as3722-ldo9: supplied by +5V_SYS
 +3.3V_RUN_TOUCH: Bringing 0uV into 2800000-2800000uV
@@ -368,8 +368,8 @@
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
 tegra-pcie 1003000.pcie-controller: probing port 0, using 2 lanes
-ata1: SATA link down (SStatus 0 SControl 300)
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
+ata1: SATA link down (SStatus 0 SControl 300)
 tegra-pcie 1003000.pcie-controller: probing port 1, using 1 lanes
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000000
 tegra-pcie 1003000.pcie-controller: PCI host bridge to bus 0000:00
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-02-10 00:13:38.680887662 -0800
+++ /tmp/foob	2017-02-10 00:13:38.684887677 -0800
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -42,9 +42,9 @@
 	 #################################################################
 	 #################################################################
 	 #########################################################
-	 3.6 MiB/s
+	 3.5 MiB/s
 done
-Bytes transferred = 7512216 (72a098 hex)
+Bytes transferred = 7510704 (729ab0 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -55,7 +55,7 @@
 done
 Bytes transferred = 68155 (10a3b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x72a098 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x729ab0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a3a
@@ -63,19 +63,19 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc7-next-20170209-152254-g9f7e70f (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Feb 8 22:50:29 PST 2017
+Linux version 4.10.0-rc7-next-20170210-152444-g632571b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Fri Feb 10 00:04:49 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
-OF: fdt:Machine model: Acer Chromebook 13 CB5-311
-OF: fdt:Ignoring memory range 0x100000000 - 0x17ffff000
+OF: fdt: Machine model: Acer Chromebook 13 CB5-311
+OF: fdt: Ignoring memory range 0x100000000 - 0x17ffff000
 debug: ignoring loglevel setting.
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c153a840, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c153a800, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -98,7 +98,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc153c1e0   (1265 kB)
+      .data : 0xc1400000 - 0xc153c1a0   (1265 kB)
        .bss : 0xc153e000 - 0xc15c4178   ( 537 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -294,6 +294,8 @@
 mmc0: Unknown controller version (3). You may experience problems.
 mmc0: Invalid maximum block size, assuming 512 bytes
 mmc0: SDHCI controller on 700b0600.sdhci [700b0600.sdhci] using ADMA
+elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
+input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 ledtrig-cpu: registered to indicate activity on CPUs
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
@@ -311,8 +313,6 @@
 Registering SWP/SWPB emulation handler
 +3.3V_RUN: supplied by +3.3V_SYS
 +VDD_LED: supplied by +VDD_MUX
-elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
-input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 +3.3V_AVDD_HDMI_AP_GATED: supplied by +3.3V_RUN
 +1.8V_RUN_CAM: supplied by +3.3V_RUN
 +VDDIO_SDMMC3: supplied by +3.3V_RUN
@@ -321,29 +321,29 @@
 +5V_USB_HS: supplied by +5V_SYS
 +5V_USB_SS: supplied by +5V_SYS
 +3.3V_PANEL: supplied by +3.3V_RUN
-mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
-mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
-mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
-mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 +3.3V_LP0: supplied by +3.3V_SYS
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
-elan_i2c 1-0015: failed to read report data: -121
 +5V_HDMI_CON: supplied by +5V_SYS
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
  usb3-1: usb3-1 supply vbus not found, using dummy regulator
+elan_i2c 1-0015: failed to read report data: -121
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
+mmc0: new high speed MMC card at address 0001
 sdhci-tegra 700b0000.sdhci: allocated mmc-pwrseq
 mmc1: Unknown controller version (3). You may experience problems.
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: Switching to 3.3V signalling voltage failed
+mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
+mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
+mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
+mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 mmc1: SDHCI controller on 700b0000.sdhci [700b0000.sdhci] using ADMA
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
-elan_i2c 1-0015: invalid report id data (ff)
 mmc2: Unknown controller version (3). You may experience problems.
+elan_i2c 1-0015: invalid report id data (ff)
 mmc2: Invalid maximum block size, assuming 512 bytes
 mmc1: new high speed SDIO card at address 0001
 mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2017-02-10 00:13:38.688887692 -0800
+++ /tmp/foob	2017-02-10 00:13:38.692887708 -0800
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -34,28 +34,26 @@
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'zImage.tegra_defconfig'.
 Load address: 0x82000000
-Loading: *#####EHCI timed out on TD - token=0x8008d80
-T EHCI timed out on TD - token=0x8008d80
-T ############################################################
+Loading: *#################################################################
 	 #################################################################
 	 #################################################################
 	 #################################################################
 	 #################################################################
 	 ######################################################
-	 402.3 KiB/s
+	 3.6 MiB/s
 done
-Bytes transferred = 5553864 (54bec8 hex)
+Bytes transferred = 5554352 (54c0b0 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.4 MiB/s
+	 3.6 MiB/s
 done
 Bytes transferred = 68155 (10a3b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x54bec8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x54c0b0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a3a
@@ -64,12 +62,12 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc7-next-20170209-152254-g9f7e70f (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Feb 8 22:50:03 PST 2017
+Linux version 4.10.0-rc7-next-20170210-152444-g632571b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Feb 10 00:04:29 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
-OF: fdt:Machine model: Acer Chromebook 13 CB5-311
-OF: fdt:Ignoring memory range 0x100000000 - 0x17ffff000
+OF: fdt: Machine model: Acer Chromebook 13 CB5-311
+OF: fdt: Ignoring memory range 0x100000000 - 0x17ffff000
 debug: ignoring loglevel setting.
 bootconsole [earlycon0] enabled
 cma: Reserved 64 MiB at 0xfbc00000
@@ -294,15 +292,15 @@
 Segment Routing with IPv6
 mip6: Mobile IPv6
 mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
 sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
-NET: Registered protocol family 17
+mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
 mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
+NET: Registered protocol family 17
+NET: Registered protocol family 15
+can: controller area network core (rev 20120528 abi 9)
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
 mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
-NET: Registered protocol family 15
-can: controller area network core (rev 20120528 abi 9)
 NET: Registered protocol family 29
 can: raw protocol (rev 20120528)
 can: broadcast manager protocol (rev 20161123 t)
@@ -371,7 +369,7 @@
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input4
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:35 UTC (946684835)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:23 UTC (946684823)
 +1.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
 +1.05V_RUN_CAM_REAR: disabling
@@ -383,14 +381,13 @@
   #0: tegra-hda at 0x70038000 irq 343
   #1: GoogleNyanBig
 EXT4-fs (mmcblk0p3): couldn't mount as ext3 due to feature incompatibilities
-uvcvideo: Found UVC 1.00 device HD WebCam (1bcf:2c67)
-input: HD WebCam as /devices/soc0/70090000.usb/usb1/1-2/1-2:1.0/input/input5
-usb 1-3: new high-speed USB device number 3 using tegra-xusb
 EXT4-fs (mmcblk0p3): recovery complete
 EXT4-fs (mmcblk0p3): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:3.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K
+uvcvideo: Found UVC 1.00 device HD WebCam (1bcf:2c67)
+input: HD WebCam as /devices/soc0/70090000.usb/usb1/1-2/1-2:1.0/input/input5
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# 
\ No newline at end of file
+root@(none):/# [    2.913638] usb 1-3: new high-speed USB device number 3 using tegra-xusb
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2017-02-10 00:13:38.696887723 -0800
+++ /tmp/foob	2017-02-10 00:13:38.700887738 -0800
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1800575 us
+Bootloader-Cpu Init at (time stamp): 1800688 us
 
 Pinmux changes applied in kernel way
 
@@ -62,20 +62,20 @@
 Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1800549  us
-Bootloader (AosInit) time: 734333 us
-Bootloader Main Init time: 180392 us
-BlockDevMgrInit time: 157438 us
-PartMgrInit time: 19 us
-StorMgrInit time: 30 us
+Miscellaneous avp time: 1800662  us
+Bootloader (AosInit) time: 733443 us
+Bootloader Main Init time: 180251 us
+BlockDevMgrInit time: 157500 us
+PartMgrInit time: 8 us
+StorMgrInit time: 31 us
 FsMgrInit time: 2 us
-Partition Table load: 76804 us 
-Display init time: 574847 us
-Kernel read time: 377 msec
+Partition Table load: 77135 us 
+Display init time: 574531 us
+Kernel read time: 378 msec
 Load Kernel Size:   16304128 Bytes
-Bl Kernel ReadSpeed: 43247000 Bytes/sec
-Pre-Kernel Jump time: 1740424 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5845591 us
+Bl Kernel ReadSpeed: 43132000 Bytes/sec
+Pre-Kernel Jump time: 1740583 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5842491 us
 Booting Linux on physical CPU 0x0
 Linux version 4.8.0-239303-g3477d16 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Fri Oct 7 01:12:37 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
@@ -184,7 +184,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(1.147:1): initialized
+audit: type=2000 audit(1.151:1): initialized
 workingset: timestamp_bits=46 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-02-10 00:13:38.704887755 -0800
+++ /tmp/foob	2017-02-10 00:13:38.708887770 -0800
@@ -49,7 +49,7 @@
 	 #########################################################
 	 4.1 MiB/s
 done
-Bytes transferred = 7512216 (72a098 hex)
+Bytes transferred = 7510704 (729ab0 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -60,7 +60,7 @@
 done
 Bytes transferred = 21426 (53b2 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x72a098 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x729ab0 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103b1
@@ -68,17 +68,17 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc7-next-20170209-152254-g9f7e70f (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Feb 8 22:50:29 PST 2017
+Linux version 4.10.0-rc7-next-20170210-152444-g632571b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Fri Feb 10 00:04:49 PST 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
-OF: fdt:Machine model: Compulab TrimSlice board
+OF: fdt: Machine model: Compulab TrimSlice board
 debug: ignoring loglevel setting.
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c153a840, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c153a800, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -101,7 +101,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc153c1e0   (1265 kB)
+      .data : 0xc1400000 - 0xc153c1a0   (1265 kB)
        .bss : 0xc153e000 - 0xc15c4178   ( 537 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
@@ -330,23 +330,23 @@
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
+tegra-ehci c5000000.usb: EHCI Host Controller
+tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
+tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
-tegra-ehci c5000000.usb: EHCI Host Controller
  mmcblk0: p1
-tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
-tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2015-02-07 05:44:04 UTC (1423287844)
-EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
+rtc-em3027 2-0056: setting system clock to 2015-02-07 05:47:37 UTC (1423288057)
+EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2017-02-10 00:13:38.712887785 -0800
+++ /tmp/foob	2017-02-10 00:13:38.716887801 -0800
@@ -45,20 +45,20 @@
 	 #################################################################
 	 #################################################################
 	 ######################################################
-	 4 MiB/s
+	 4.1 MiB/s
 done
-Bytes transferred = 5553864 (54bec8 hex)
+Bytes transferred = 5554352 (54c0b0 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 4.1 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21426 (53b2 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x54bec8 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x54c0b0 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103b1
@@ -67,10 +67,10 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc7-next-20170209-152254-g9f7e70f (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Feb 8 22:50:03 PST 2017
+Linux version 4.10.0-rc7-next-20170210-152444-g632571b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Feb 10 00:04:29 PST 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
-OF: fdt:Machine model: Compulab TrimSlice board
+OF: fdt: Machine model: Compulab TrimSlice board
 debug: ignoring loglevel setting.
 bootconsole [earlycon0] enabled
 cma: Reserved 64 MiB at 0x3c000000
@@ -312,21 +312,23 @@
 tlv320aic23-codec 2-001a: ASoC: Failed to add route MICIN -> NULL -> Mic Input
 tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
 NET: Registered protocol family 10
+Segment Routing with IPv6
 mmc0: host does not support reading read-only switch, assuming write-enable
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
 mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
  mmcblk0: p1
-mmc1: new high speed SD card at address b368
-Segment Routing with IPv6
 mip6: Mobile IPv6
 sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
 NET: Registered protocol family 17
 NET: Registered protocol family 15
+mmc1: new high speed SD card at address b368
 can: controller area network core (rev 20120528 abi 9)
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 NET: Registered protocol family 29
 can: raw protocol (rev 20120528)
 can: broadcast manager protocol (rev 20161123 t)
+ mmcblk1: p1 p2
 can: netlink gateway (rev 20130117) max_hops=1
 Bluetooth: RFCOMM socket layer initialized
 Bluetooth: RFCOMM ver 1.11
@@ -340,16 +342,14 @@
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
- mmcblk1: p1 p2
+hub 1-1:1.0: USB hub found
+hub 1-1:1.0: 4 ports detected
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2015-02-07 05:41:45 UTC (1423287705)
+rtc-em3027 2-0056: setting system clock to 2015-02-07 05:45:50 UTC (1423287950)
 ALSA device list:
-hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
   #0: tegra-trimslice
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2017-02-10 00:13:38.720887816 -0800
+++ /tmp/foob	2017-02-10 00:13:38.724887831 -0800
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 743907 us
-Basic modules init took 350121 us
+Tegraboot started after 769316 us
+Basic modules init took 348534 us
 Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14371 us
-Carveout took 252784 us
-CPU initialization took 92308 us
-Total time taken by TegraBoot 709584 us
+Next stage binary read took 14370 us
+Carveout took 252717 us
+CPU initialization took 92319 us
+Total time taken by TegraBoot 707940 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.584] LPDDR4 Training: Number of tables = 10
+64b[0001.607] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -119,9 +119,9 @@
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
 15346176 bytes read in 804 ms (18.2 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-44882 bytes read in 475 ms (91.8 KiB/s)
+44882 bytes read in 474 ms (91.8 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 497 ms (3.1 MiB/s)
+1627787 bytes read in 495 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc7-next-20170209-176687-g9f7e70f (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Feb 8 22:54:12 PST 2017
+Linux version 4.10.0-rc7-next-20170210-176877-g632571b (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Fri Feb 10 00:07:47 PST 2017
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -356,6 +356,6 @@
 devtmpfs: mounted
 Freeing unused kernel memory: 1088K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
-bash: no job contr[    1.367851] random: fast init done
+bash: no job contr[    1.344600] random: fast init done
 ol in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2017-02-10 00:13:38.728887847 -0800
+++ /tmp/foob	2017-02-10 00:13:38.728887847 -0800
@@ -1,4 +1,7 @@
- '115200n8')
+ EFI parameters from FDT:
+efi: UEFI not found.
+cma: Reserved 16 MiB at 0x00000000f5400000
+earlycon: uart0 at MMIO 0x0000000070006000 (options '115200n8')
 bootconsole [uart0] enabled
 NUMA: No NUMA configuration found
 NUMA: Faking a node at [mem 0x0000000000000000-0x000000013fffffff]
@@ -63,7 +66,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x4e28: expected 0x2004f400
+boot misc3 0x4f28: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -76,7 +79,7 @@
 console [tty0] enabled
 bootconsole [uart0] disabled
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc7-next-20170209-176687-g9f7e70f (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Feb 8 22:54:12 PST 2017
+Linux version 4.10.0-rc7-next-20170210-176877-g632571b (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Fri Feb 10 00:07:47 PST 2017
 Boot CPU: AArch64 Processor [411fd071]
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
@@ -146,7 +149,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x4e28: expected 0x2004f400
+boot misc3 0x4f28: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -236,7 +239,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.323:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.327:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -324,13 +327,12 @@
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
 tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 mmc0: new high speed MMC card at address 0001
-max77686-rtc max77620-rtc: setting system clock to 2000-11-20 12:21:55 UTC (974722915)
 mmcblk0: mmc0:0001 BGND3R 29.1 GiB 
 mmcblk0boot0: mmc0:0001 BGND3R partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
-tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
+max77686-rtc max77620-rtc: setting system clock to 2000-11-21 13:35:32 UTC (974813732)
 PP1200_AVDD: disabling
 PP1200_RCAM: disabling
 PP2800L_RCAM: disabling
@@ -341,6 +343,7 @@
 PP1800_CAM: disabling
 ALSA device list:
   No soundcards found.
+tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 Freeing unused kernel memory: 1088K
 
 System initialization...
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-02-10 00:13:38.736887878 -0800
+++ /tmp/foob	2017-02-10 00:13:38.736887878 -0800
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 #########################################################
-	 3.1 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 7512216 (72a098 hex)
+Bytes transferred = 7510704 (729ab0 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.7 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 51804 (ca5c hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x72a098 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x729ab0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa5b ... OK
@@ -60,17 +60,17 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc7-next-20170209-152254-g9f7e70f (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Feb 8 22:50:29 PST 2017
+Linux version 4.10.0-rc7-next-20170210-152444-g632571b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Fri Feb 10 00:04:49 PST 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
-OF: fdt:Machine model: NVIDIA Tegra30 Beaver evaluation board
+OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
 debug: ignoring loglevel setting.
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c153a840, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c153a800, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -93,7 +93,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc153c1e0   (1265 kB)
+      .data : 0xc1400000 - 0xc153c1a0   (1265 kB)
        .bss : 0xc153e000 - 0xc15c4178   ( 537 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -311,11 +311,11 @@
 mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB
+mmc0: new ultra high speed DDR50 SDHC card at address aaaa
+mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
 Primary GPT is invalid, using alternate GPT.
  mmcblk1: p1 p2 p3 p4 p5
-mmc0: new ultra high speed DDR50 SDHC card at address aaaa
 tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
-mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
  mmcblk0: p1 p2 p3
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -394,5 +394,5 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.073637] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.094010] usb 2-1: new high-speed USB device number 2 using tegra-ehci
 asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0f:67:60
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2017-02-10 00:13:38.744887909 -0800
+++ /tmp/foob	2017-02-10 00:13:38.744887909 -0800
@@ -37,9 +37,9 @@
 	 #################################################################
 	 #################################################################
 	 ######################################################
-	 3.4 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 5553864 (54bec8 hex)
+Bytes transferred = 5554352 (54c0b0 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 51804 (ca5c hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x54bec8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x54c0b0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa5b ... OK
@@ -59,10 +59,10 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc7-next-20170209-152254-g9f7e70f (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Feb 8 22:50:03 PST 2017
+Linux version 4.10.0-rc7-next-20170210-152444-g632571b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Feb 10 00:04:29 PST 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
-OF: fdt:Machine model: NVIDIA Tegra30 Beaver evaluation board
+OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
 debug: ignoring loglevel setting.
 bootconsole [earlycon0] enabled
 cma: Reserved 64 MiB at 0xfbc00000
@@ -389,4 +389,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.370062] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.370058] usb 2-1: new high-speed USB device number 2 using tegra-ehci
