Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2017-01-19 02:57:51.499541184 -0800
+++ /tmp/foob	2017-01-19 02:57:51.499541184 -0800
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc4-next-20170118-170839-g8fd170c (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Jan 17 21:52:27 PST 2017
+Linux version 4.10.0-rc4-next-20170119-171138-gfb62712 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Jan 19 02:51:27 PST 2017
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -18,15 +18,15 @@
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
-Memory: 998864K/1048576K available (8508K kernel code, 956K rwdata, 4144K rodata, 1088K init, 389K bss, 33328K reserved, 16384K cma-reserved)
+Memory: 998860K/1048576K available (8508K kernel code, 958K rwdata, 4144K rodata, 1088K init, 391K bss, 33332K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff0000088d0000   (  8512 KB)
     .rodata : 0xffff0000088d0000 - 0xffff000008ce0000   (  4160 KB)
       .init : 0xffff000008ce0000 - 0xffff000008df0000   (  1088 KB)
-      .data : 0xffff000008df0000 - 0xffff000008edf200   (   957 KB)
-       .bss : 0xffff000008edf200 - 0xffff000008f407b0   (   390 KB)
+      .data : 0xffff000008df0000 - 0xffff000008edfa00   (   959 KB)
+       .bss : 0xffff000008edfa00 - 0xffff000008f417b0   (   392 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -101,7 +101,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.684:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.668:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=46 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -168,7 +168,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2017-01-18 05:53:15 UTC (1484718795)
+rtc-pl031 9010000.pl031: setting system clock to 2017-01-19 10:52:11 UTC (1484823131)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-01-19 02:57:51.503541198 -0800
+++ /tmp/foob	2017-01-19 02:57:51.507541212 -0800
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra114 (Dalmore) # 
 Tegra114 (Dalmore) # usb reset
 (Re)start USB...
@@ -39,9 +39,9 @@
 	 #################################################################
 	 #################################################################
 	 #####################################################
-	 2.8 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 7453768 (71bc48 hex)
+Bytes transferred = 7456936 (71c8a8 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -52,7 +52,7 @@
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x71bc48 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x71c8a8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc4-next-20170118-146406-g8fd170c (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Jan 17 21:49:31 PST 2017
+Linux version 4.10.0-rc4-next-20170119-146705-gfb62712 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Jan 19 02:48:15 PST 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -71,7 +71,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c1432200, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1432600, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -84,7 +84,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1994828K/2096128K available (9216K kernel code, 1231K rwdata, 4176K rodata, 2048K init, 531K bss, 35764K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1994820K/2096128K available (9216K kernel code, 1232K rwdata, 4172K rodata, 2048K init, 535K bss, 35772K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -94,8 +94,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc1433d00   (1232 kB)
-       .bss : 0xc1435000 - 0xc14b9d28   ( 532 kB)
+      .data : 0xc1300000 - 0xc1434100   (1233 kB)
+       .bss : 0xc1436000 - 0xc14bbd28   ( 536 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU debugfs-based tracing is enabled.
@@ -305,32 +305,32 @@
 Key type dns_resolver registered
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
+mmc0: new high speed SDHC card at address aaaa
 usb1_vbus: supplied by vdd-sys-5v0
 usb2_vbus: supplied by vdd-sys-5v0
-mmc0: new high speed SDHC card at address aaaa
-mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
- mmcblk0: p1 p2 p3
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
 [drm] Cannot find any crtc or sizes - going 1024x768
+mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
+Console: switching to colour frame buffer device 128x48
+drm drm: fb0:  frame buffer device
+[drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
+ mmcblk0: p1 p2 p3
 mmc1: new high speed MMC card at address 0001
+tegra-ehci 7d008000.usb: EHCI Host Controller
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
- mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
-Console: switching to colour frame buffer device 128x48
-drm drm: fb0:  frame buffer device
-[drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
-tegra-ehci 7d008000.usb: EHCI Host Controller
 tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 1
 tegra-ehci 7d008000.usb: irq 339, io mem 0x7d008000
+ mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-11-11 17:22:40 UTC (973963360)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-11-12 22:18:56 UTC (974067536)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -342,4 +342,4 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.449541] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.439599] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2017-01-19 02:57:51.511541226 -0800
+++ /tmp/foob	2017-01-19 02:57:51.515541240 -0800
@@ -36,21 +36,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ####################################################
-	 3.2 MiB/s
+	 #####################################################
+	 2.8 MiB/s
 done
-Bytes transferred = 5528928 (545d60 hex)
+Bytes transferred = 5537072 (547d30 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 2.6 MiB/s
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x545d60 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x547d30 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc4-next-20170118-146406-g8fd170c (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Jan 17 21:49:10 PST 2017
+Linux version 4.10.0-rc4-next-20170119-146705-gfb62712 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Jan 19 02:48:15 PST 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0d5ed40, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0d5f100, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999928K/2096128K available (8192K kernel code, 385K rwdata, 2480K rodata, 1024K init, 364K bss, 30664K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1999924K/2096128K available (8192K kernel code, 385K rwdata, 2480K rodata, 1024K init, 368K bss, 30668K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,8 +92,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d60400   ( 385 kB)
-       .bss : 0xc0d62000 - 0xc0dbd34c   ( 365 kB)
+      .data : 0xc0d00000 - 0xc0d607c0   ( 386 kB)
+       .bss : 0xc0d62000 - 0xc0dbe34c   ( 369 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -322,7 +322,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-11-11 17:19:52 UTC (973963192)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-11-12 22:21:41 UTC (974067701)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -336,4 +336,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.749394] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.739365] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-01-19 02:57:51.519541253 -0800
+++ /tmp/foob	2017-01-19 02:57:51.523541268 -0800
@@ -34,20 +34,20 @@
 	 #################################################################
 	 #################################################################
 	 #####################################################
-	 3.3 MiB/s
+	 4 MiB/s
 done
-Bytes transferred = 7453768 (71bc48 hex)
+Bytes transferred = 7456936 (71c8a8 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 4 MiB/s
+	 1.3 MiB/s
 done
 Bytes transferred = 67133 (1063d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x71bc48 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x71c8a8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301363c
@@ -55,7 +55,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc4-next-20170118-146406-g8fd170c (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Jan 17 21:49:31 PST 2017
+Linux version 4.10.0-rc4-next-20170119-146705-gfb62712 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Jan 19 02:48:15 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -66,7 +66,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c1432200, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1432600, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -79,7 +79,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1994708K/2096128K available (9216K kernel code, 1231K rwdata, 4176K rodata, 2048K init, 531K bss, 35884K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1994700K/2096128K available (9216K kernel code, 1232K rwdata, 4172K rodata, 2048K init, 535K bss, 35892K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -89,8 +89,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc1433d00   (1232 kB)
-       .bss : 0xc1435000 - 0xc14b9d28   ( 532 kB)
+      .data : 0xc1300000 - 0xc1434100   (1233 kB)
+       .bss : 0xc1436000 - 0xc14bbd28   ( 536 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU debugfs-based tracing is enabled.
@@ -316,33 +316,36 @@
 usb2-1 supply vbus not found, using dummy regulator
 usb3-0 supply vbus not found, using dummy regulator
 mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+ mmcblk0: p1
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
 tegra-pcie 1003000.pcie-controller: Failed to get supply 'avddio-pex': -517
 tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
 ------------[ cut here ]------------
-WARNING: CPU: 1 PID: 87 at drivers/base/dd.c:349 driver_probe_device+0x288/0x2c0
+WARNING: CPU: 2 PID: 92 at drivers/base/dd.c:349 driver_probe_device+0x288/0x2c0
 Modules linked in:
-CPU: 1 PID: 87 Comm: kworker/1:1 Not tainted 4.10.0-rc4-next-20170118-146406-g8fd170c #1
+CPU: 2 PID: 92 Comm: kworker/2:1 Not tainted 4.10.0-rc4-next-20170119-146705-gfb62712 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: events deferred_probe_work_func
 [<c030fb48>] (unwind_backtrace) from [<c030b158>] (show_stack+0x10/0x14)
-[<c030b158>] (show_stack) from [<c059e8d4>] (dump_stack+0x84/0xa4)
-[<c059e8d4>] (dump_stack) from [<c0341b48>] (__warn+0xd4/0x100)
+[<c030b158>] (show_stack) from [<c059fa94>] (dump_stack+0x84/0xa4)
+[<c059fa94>] (dump_stack) from [<c0341b48>] (__warn+0xd4/0x100)
 [<c0341b48>] (__warn) from [<c0341b94>] (warn_slowpath_null+0x20/0x28)
-[<c0341b94>] (warn_slowpath_null) from [<c07e3adc>] (driver_probe_device+0x288/0x2c0)
-[<c07e3adc>] (driver_probe_device) from [<c07e201c>] (bus_for_each_drv+0x44/0x8c)
-[<c07e201c>] (bus_for_each_drv) from [<c07e37cc>] (__device_attach+0x9c/0x100)
-[<c07e37cc>] (__device_attach) from [<c07e2eb4>] (bus_probe_device+0x84/0x8c)
-[<c07e2eb4>] (bus_probe_device) from [<c07e32e8>] (deferred_probe_work_func+0x60/0x8c)
-[<c07e32e8>] (deferred_probe_work_func) from [<c0357038>] (process_one_work+0x134/0x33c)
-[<c0357038>] (process_one_work) from [<c0357268>] (process_scheduled_works+0x28/0x38)
-[<c0357268>] (process_scheduled_works) from [<c0357490>] (worker_thread+0x218/0x4f0)
-[<c0357490>] (worker_thread) from [<c035c808>] (kthread+0xfc/0x134)
-[<c035c808>] (kthread) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
----[ end trace f9c6d6c8b528bd51 ]---
-gpio gpiochip1: (as3722-gpio): could not create pin range
-as3722-pinctrl as3722-pinctrl: Couldn't add pin range, -517
+[<c0341b94>] (warn_slowpath_null) from [<c07e509c>] (driver_probe_device+0x288/0x2c0)
+[<c07e509c>] (driver_probe_device) from [<c07e35dc>] (bus_for_each_drv+0x44/0x8c)
+[<c07e35dc>] (bus_for_each_drv) from [<c07e4d8c>] (__device_attach+0x9c/0x100)
+[<c07e4d8c>] (__device_attach) from [<c07e4474>] (bus_probe_device+0x84/0x8c)
+[<c07e4474>] (bus_probe_device) from [<c07e48a8>] (deferred_probe_work_func+0x60/0x8c)
+[<c07e48a8>] (deferred_probe_work_func) from [<c0357078>] (process_one_work+0x134/0x33c)
+[<c0357078>] (process_one_work) from [<c03572a8>] (process_scheduled_works+0x28/0x38)
+[<c03572a8>] (process_scheduled_works) from [<c03574d0>] (worker_thread+0x218/0x4f0)
+[<c03574d0>] (worker_thread) from [<c035c848>] (kthread+0xfc/0x134)
+[<c035c848>] (kthread) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
+---[ end trace eab116d7d35f620d ]---
 as3722-sd2: supplied by +5V_SYS
 as3722-sd3: supplied by +5V_SYS
 as3722-sd4: supplied by +5V_SYS
@@ -365,146 +368,6 @@
 hub 2-0:1.0: 1 port detected
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
-tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
-tegra-pcie 1003000.pcie-controller: Failed to get supply 'avddio-pex': -517
-tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
-tegra-hdmi 54280000.hdmi: failed to get PLL regulator
-------------[ cut here ]------------
-WARNING: CPU: 1 PID: 87 at drivers/base/dd.c:349 driver_probe_device+0x288/0x2c0
-Modules linked in:
-CPU: 1 PID: 87 Comm: kworker/1:1 Tainted: G        W       4.10.0-rc4-next-20170118-146406-g8fd170c #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-Workqueue: events deferred_probe_work_func
-[<c030fb48>] (unwind_backtrace) from [<c030b158>] (show_stack+0x10/0x14)
-[<c030b158>] (show_stack) from [<c059e8d4>] (dump_stack+0x84/0xa4)
-[<c059e8d4>] (dump_stack) from [<c0341b48>] (__warn+0xd4/0x100)
-[<c0341b48>] (__warn) from [<c0341b94>] (warn_slowpath_null+0x20/0x28)
-[<c0341b94>] (warn_slowpath_null) from [<c07e3adc>] (driver_probe_device+0x288/0x2c0)
-[<c07e3adc>] (driver_probe_device) from [<c07e201c>] (bus_for_each_drv+0x44/0x8c)
-[<c07e201c>] (bus_for_each_drv) from [<c07e37cc>] (__device_attach+0x9c/0x100)
-[<c07e37cc>] (__device_attach) from [<c07e2eb4>] (bus_probe_device+0x84/0x8c)
-[<c07e2eb4>] (bus_probe_device) from [<c07e32e8>] (deferred_probe_work_func+0x60/0x8c)
-[<c07e32e8>] (deferred_probe_work_func) from [<c0357038>] (process_one_work+0x134/0x33c)
-[<c0357038>] (process_one_work) from [<c0357268>] (process_scheduled_works+0x28/0x38)
-[<c0357268>] (process_scheduled_works) from [<c0357490>] (worker_thread+0x218/0x4f0)
-[<c0357490>] (worker_thread) from [<c035c808>] (kthread+0xfc/0x134)
-[<c035c808>] (kthread) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
----[ end trace f9c6d6c8b528bd52 ]---
-gpio gpiochip1: (as3722-gpio): could not create pin range
-as3722-pinctrl as3722-pinctrl: Couldn't add pin range, -517
-as3722-sd2: supplied by +5V_SYS
-as3722-sd3: supplied by +5V_SYS
-as3722-sd4: supplied by +5V_SYS
-as3722-sd5: supplied by +5V_SYS
-as3722-ldo0: supplied by +1.35V_LP0(sd2)
-as3722-ldo2: supplied by +1.8V_VDDIO
-as3722-ldo3: supplied by +3.3V_SYS
-as3722-ldo4: supplied by +3.3V_SYS
-as3722-ldo5: supplied by +1.8V_VDDIO
-+VDDIO_SDMMC3: bypassed regulator has no supply!
-+VDDIO_SDMMC3: failed to get the current voltage(-517)
-as3722-regulator as3722-regulator: regulator 13 register failed -517
-tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
-tegra-ahci 70027000.sata: Failed to get regulators
-input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-sdhci-tegra 700b0400.sdhci: Got CD GPIO
-sdhci-tegra 700b0400.sdhci: Got WP GPIO
-tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
-tegra-pcie 1003000.pcie-controller: Failed to get supply 'avddio-pex': -517
-tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
-tegra-hdmi 54280000.hdmi: failed to get PLL regulator
-------------[ cut here ]------------
-WARNING: CPU: 1 PID: 87 at drivers/base/dd.c:349 driver_probe_device+0x288/0x2c0
-Modules linked in:
-CPU: 1 PID: 87 Comm: kworker/1:1 Tainted: G        W       4.10.0-rc4-next-20170118-146406-g8fd170c #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-Workqueue: events deferred_probe_work_func
-[<c030fb48>] (unwind_backtrace) from [<c030b158>] (show_stack+0x10/0x14)
-[<c030b158>] (show_stack) from [<c059e8d4>] (dump_stack+0x84/0xa4)
-[<c059e8d4>] (dump_stack) from [<c0341b48>] (__warn+0xd4/0x100)
-[<c0341b48>] (__warn) from [<c0341b94>] (warn_slowpath_null+0x20/0x28)
-[<c0341b94>] (warn_slowpath_null) from [<c07e3adc>] (driver_probe_device+0x288/0x2c0)
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:24 UTC (946684944)
-[<c07e3adc>] (driver_probe_device) from [<c07e201c>] (bus_for_each_drv+0x44/0x8c)
-[<c07e201c>] (bus_for_each_drv) from [<c07e37cc>] (__device_attach+0x9c/0x100)
-[<c07e37cc>] (__device_attach) from [<c07e2eb4>] (bus_probe_device+0x84/0x8c)
-[<c07e2eb4>] (bus_probe_device) from [<c07e32e8>] (deferred_probe_work_func+0x60/0x8c)
-[<c07e32e8>] (deferred_probe_work_func) from [<c0357038>] (process_one_work+0x134/0x33c)
-[<c0357038>] (process_one_work) from [<c0357268>] (process_scheduled_works+0x28/0x38)
-[<c0357268>] (process_scheduled_works) from [<c0357490>] (worker_thread+0x218/0x4f0)
-[<c0357490>] (worker_thread) from [<c035c808>] (kthread+0xfc/0x134)
-[<c035c808>] (kthread) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
----[ end trace f9c6d6c8b528bd53 ]---
-gpio gpiochip1: (as3722-gpio): could not create pin range
-as3722-pinctrl as3722-pinctrl: Couldn't add pin range, -517
-+3.3V_AVDD_HDMI_AP_GATED: disabling
-+USB0_VBUS_SW: disabling
-+1.05V_RUN_AVDD_HDMI_PLL: disabling
-+5V_HDMI_CON: disabling
-+5V_SATA: disabling
-+12V_SATA: disabling
-as3722-sd2: supplied by +5V_SYS
-as3722-sd3: supplied by +5V_SYS
-as3722-sd4: supplied by +5V_SYS
-as3722-sd5: supplied by +5V_SYS
-as3722-ldo0: supplied by +1.35V_LP0(sd2)
-as3722-ldo2: supplied by +1.8V_VDDIO
-as3722-ldo3: supplied by +3.3V_SYS
-as3722-ldo4: supplied by +3.3V_SYS
-as3722-ldo5: supplied by +1.8V_VDDIO
-+VDDIO_SDMMC3: bypassed regulator has no supply!
-+VDDIO_SDMMC3: failed to get the current voltage(-517)
-as3722-regulator as3722-regulator: regulator 13 register failed -517
-tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
-tegra-ahci 70027000.sata: Failed to get regulators
-[   SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
- mmcblk0: p1
-EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
-sdhci-tegra 700b0400.sdhci: Got CD GPIO
-sdhci-tegra 700b0400.sdhci: Got WP GPIO
-tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
-tegra-pcie 1003000.pcie-controller: Failed to get supply 'avddio-pex': -517
-tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
-tegra-hdmi 54280000.hdmi: failed to get PLL regulator
-------------[ cut here ]------------
-WARNING: CPU: 2 PID: 60 at drivers/base/dd.c:349 driver_probe_device+0x288/0x2c0
-Modules linked in:
-CPU: 2 PID: 60 Comm: kworker/2:1 Tainted: G        W       4.10.0-rc4-next-20170118-146406-g8fd170c #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-Workqueue: events deferred_probe_work_func
-[<c030fb48>] (unwind_backtrace) from [<c030b158>] (show_stack+0x10/0x14)
-[<c030b158>] (show_stack) from [<c059e8d4>] (dump_stack+0x84/0xa4)
-[<c059e8d4>] (dump_stack) from [<c0341b48>] (__warn+0xd4/0x100)
-[<c0341b48>] (__warn) from [<c0341b94>] (warn_slowpath_null+0x20/0x28)
-[<c0341b94>] (warn_slowpath_null) from [<c07e3adc>] (driver_probe_device+0x288/0x2c0)
-[<c07e3adc>] (driver_probe_device) from [<c07e201c>] (bus_for_each_drv+0x44/0x8c)
-[<c07e201c>] (bus_for_each_drv) from [<c07e37cc>] (__device_attach+0x9c/0x100)
-[<c07e37cc>] (__device_attach) from [<c07e2eb4>] (bus_probe_device+0x84/0x8c)
-[<c07e2eb4>] (bus_probe_device) from [<c07e32e8>] (deferred_probe_work_func+0x60/0x8c)
-[<c07e32e8>] (deferred_probe_work_func) from [<c0357038>] (process_one_work+0x134/0x33c)
-[<c0357038>] (process_one_work) from [<c03572a4>] (worker_thread+0x2c/0x4f0)
-[<c03572a4>] (worker_thread) from [<c035c808>] (kthread+0xfc/0x134)
-[<c035c808>] (kthread) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
----[ end trace f9c6d6c8b528bd54 ]---
-as3722-sd2: supplied by +5V_SYS
-as3722-sd3: supplied by +5V_SYS
-as3722-sd4: supplied by +5V_SYS
-as3722-sd5: supplied by +5V_SYS
-as3722-ldo0: supplied by +1.35V_LP0(sd2)
-as3722-ldo2: supplied by +1.8V_VDDIO
-as3722-ldo3: supplied by +3.3V_SYS
-as3722-ldo4: supplied by +3.3V_SYS
-as3722-ldo5: supplied by +1.8V_VDDIO
-+VDDIO_SDMMC3: bypassed regulator has no supply!
-+VDDIO_SDMMC3: failed to get the current voltage(-517)
-as3722-regulator as3722-regulator: regulator 13 register failed -517
-tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
-tegra-ahci 70027000.sata: Failed to get regulators
-sdhci-tegra 700b0400.sdhci: Got CD GPIO
-sdhci-tegra 700b0400.sdhci: Got WP GPIO
 +3.3V_RUN: supplied by +3.3V_SYS
 +3.3V_LP0: supplied by +3.3V_SYS
 +3.3V_AVDD_HDMI_AP_GATED: supplied by +3.3V_RUN
@@ -540,10 +403,6 @@
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc1: Unknown controller version (3). You may experience problems.
-EXT4-fs (mmcblk0p1): recovery complete
-EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:1.
-devtmpfs: mounted
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
@@ -569,16 +428,16 @@
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x4 may corrupt adjacent RW1C bits
 pci_bus 0000:00: 1-byte config write to 0000:00:01.0 offset 0xc may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4 may corrupt adjacent RW1C bits
-ata1: SATA link down (SStatus 0 SControl 300)
 pci_bus 0000:00: 1-byte config write to 0000:00:02.0 offset 0xc may corrupt adjacent RW1C bits
 PCI: bus0: Fast back to back transfers disabled
 pci 0000:00:01.0: bridge configuration invalid ([bus 00-00]), reconfiguring
+ata1: SATA link down (SStatus 0 SControl 300)
 pci 0000:00:02.0: bridge configuration invalid ([bus 00-00]), reconfiguring
 pci 0000:01:00.0: [8086:4232] type 00 class 0x028000
 pci 0000:01:00.0: reg 0x10: [mem 0x00000000-0x00001fff 64bit]
 pci 0000:01:00.0: PME# supported from D0 D3hot D3cold
-Freeing unused kernel memory: 2048K
 mmc1: new high speed SDHC card at address aaaa
+mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 PCI: bus1: Fast back to back transfers disabled
 pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
 pci 0000:02:00.0: [10ec:8168] type 00 class 0x020000
@@ -587,23 +446,59 @@
 pci 0000:02:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
 pci 0000:02:00.0: supports D1 D2
 pci 0000:02:00.0: PME# supported from D0 D1 D2 D3hot D3cold
-mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
  mmcblk1: p1 p2 p3
 PCI: bus2: Fast back to back transfers disabled
 pci_bus 0000:02: busn_res: [bus 02-ff] end is updated to 02
 pci 0000:00:01.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
-bash: ca[    3.851573] pci 0000:00:02.0: BAR 8: assigned [mem 0x13100000-0x131fffff]
-nnot set terminal process group [    3.859066] pci 0000:00:02.0: BAR 9: assigned [mem 0x20000000-0x200fffff 64bit pref]
-(-1): Inappropriate ioctl for de[    3.869581] pci 0000:00:02.0: BAR 7: assigned [io  0x1000-0x1fff]
-vice
-bash: no job control in th[    3.878453] pci 0000:01:00.0: BAR 0: assigned [mem 0x13000000-0x13001fff 64bit]
-is shell
+pci 0000:00:02.0: BAR 8: assigned [mem 0x13100000-0x131fffff]
+pci 0000:00:02.0: BAR 9: assigned [mem 0x20000000-0x200fffff 64bit pref]
+pci 0000:00:02.0: BAR 7: assigned [io  0x1000-0x1fff]
+pci 0000:01:00.0: BAR 0: assigned [mem 0x13000000-0x13001fff 64bit]
 pci 0000:00:01.0: PCI bridge to [bus 01]
 pci 0000:00:01.0:   bridge window [mem 0x13000000-0x130fffff]
 pci 0000:02:00.0: BAR 4: assigned [mem 0x20000000-0x20003fff 64bit pref]
-root@(none):/# [    3.908942] pci 0000:02:00.0: BAR 2: assigned [mem 0x13100000-0x13100fff 64bit]
+pci 0000:02:00.0: BAR 2: assigned [mem 0x13100000-0x13100fff 64bit]
 pci 0000:02:00.0: BAR 0: assigned [io  0x1000-0x10ff]
 pci 0000:00:02.0: PCI bridge to [bus 02]
 pci 0000:00:02.0:   bridge window [io  0x1000-0x1fff]
 pci 0000:00:02.0:   bridge window [mem 0x13100000-0x131fffff]
-pci 0000:00:02.0:   bridge window [mem 0x20000000-0x200fffff 
\ No newline at end of file
+pci 0000:00:02.0:   bridge window [mem 0x20000000-0x200fffff 64bit pref]
+pci 0000:00:01.0: nv_msi_ht_cap_quirk didn't locate host bridge
+pcieport 0000:00:01.0: enabling device (0140 -> 0142)
+pcieport 0000:00:01.0: Signaling PME with IRQ 393
+pci 0000:00:02.0: nv_msi_ht_cap_quirk didn't locate host bridge
+pcieport 0000:00:02.0: enabling device (0140 -> 0143)
+pcieport 0000:00:02.0: Signaling PME with IRQ 394
+r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
+r8169 0000:02:00.0: enabling device (0140 -> 0143)
+r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0b58000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 395
+r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
++1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
+[drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
+[drm] No driver support for vblank timestamp query.
+tegra-hdmi 54280000.hdmi: cannot set audio to 48000 Hz at 297000000 Hz pixel clock
+Console: switching to colour frame buffer device 512x135
+drm drm: fb0:  frame buffer device
+[drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
+of: dev_pm_opp_of_cpumask_add_table: couldn't find opp table for cpu:0, -19
+cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
+cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
+input: gpio-keys as /devices/soc0/gpio-keys/input/input0
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:14 UTC (946684814)
++USB0_VBUS_SW: disabling
++1.8V_RUN_CAM: disabling
++2.8V_RUN_CAM: disabling
++1.2V_RUN_CAM_FRONT: disabling
++1.05V_RUN_CAM_REAR: disabling
++3.3V_RUN_TOUCH: disabling
++2.8V_RUN_CAM_AF: disabling
++1.8V_RUN_VPP_FUSE: disabling
+EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
+EXT4-fs (mmcblk0p1): recovery complete
+EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
+VFS: Mounted root (ext3 filesystem) on device 179:1.
+devtmpfs: mounted
+Freeing unused kernel memory: 2048K
+bash: cannot set terminal process group (-1): Inappropriate ioctl for device
+bash: no job control in this shell
+root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2017-01-19 02:57:51.531541296 -0800
+++ /tmp/foob	2017-01-19 02:57:51.531541296 -0800
@@ -31,21 +31,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ####################################################
-	 4 MiB/s
+	 #####################################################
+	 3.9 MiB/s
 done
-Bytes transferred = 5528928 (545d60 hex)
+Bytes transferred = 5537072 (547d30 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.6 MiB/s
+	 3.8 MiB/s
 done
 Bytes transferred = 67133 (1063d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x545d60 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x547d30 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301363c
@@ -54,7 +54,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc4-next-20170118-146406-g8fd170c (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Jan 17 21:49:10 PST 2017
+Linux version 4.10.0-rc4-next-20170119-146705-gfb62712 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Jan 19 02:48:15 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -64,7 +64,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0d5ed40, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0d5f100, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -77,7 +77,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999808K/2096128K available (8192K kernel code, 385K rwdata, 2480K rodata, 1024K init, 364K bss, 30784K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1999804K/2096128K available (8192K kernel code, 385K rwdata, 2480K rodata, 1024K init, 368K bss, 30788K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -87,8 +87,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d60400   ( 385 kB)
-       .bss : 0xc0d62000 - 0xc0dbd34c   ( 365 kB)
+      .data : 0xc0d00000 - 0xc0d607c0   ( 386 kB)
+       .bss : 0xc0d62000 - 0xc0dbe34c   ( 369 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -318,6 +318,8 @@
 tegra-pcie 1003000.pcie-controller: Failed to get supply 'avddio-pex': -517
 tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
+gpio gpiochip1: (as3722-gpio): could not create pin range
+as3722-pinctrl as3722-pinctrl: Couldn't add pin range, -517
 as3722-sd2: supplied by +5V_SYS
 as3722-sd3: supplied by +5V_SYS
 as3722-sd4: supplied by +5V_SYS
@@ -330,9 +332,9 @@
 +VDDIO_SDMMC3: bypassed regulator has no supply!
 +VDDIO_SDMMC3: failed to get the current voltage(-517)
 as3722-regulator as3722-regulator: regulator 13 register failed -517
+input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input1
 tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
 tegra-ahci 70027000.sata: Failed to get regulators
-input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input1
 tegra-ehci 7d008000.usb: EHCI Host Controller
 tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci 7d008000.usb: irq 355, io mem 0x7d008000
@@ -343,6 +345,28 @@
 tegra-xusb 70090000.usb: failed to get regulators: -517
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
+tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
+tegra-pcie 1003000.pcie-controller: Failed to get supply 'avddio-pex': -517
+tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
+tegra-hdmi 54280000.hdmi: failed to get PLL regulator
+as3722-sd2: supplied by +5V_SYS
+as3722-sd3: supplied by +5V_SYS
+as3722-sd4: supplied by +5V_SYS
+as3722-sd5: supplied by +5V_SYS
+as3722-ldo0: supplied by +1.35V_LP0(sd2)
+as3722-ldo2: supplied by +1.8V_VDDIO
+as3722-ldo3: supplied by +3.3V_SYS
+as3722-ldo4: supplied by +3.3V_SYS
+as3722-ldo5: supplied by +1.8V_VDDIO
++VDDIO_SDMMC3: bypassed regulator has no supply!
++VDDIO_SDMMC3: failed to get the current voltage(-517)
+as3722-regulator as3722-regulator: regulator 13 register failed -517
+tegra-ahci 70027000.sata: Failed to get supply 'avdd': -517
+tegra-ahci 70027000.sata: Failed to get regulators
+tegra-xusb 70090000.usb: Failed to get supply 'avddio-pex': -517
+tegra-xusb 70090000.usb: failed to get regulators: -517
+sdhci-tegra 700b0400.sdhci: Got CD GPIO
+sdhci-tegra 700b0400.sdhci: Got WP GPIO
 +3.3V_RUN: supplied by +3.3V_SYS
 +3.3V_LP0: supplied by +3.3V_SYS
 +3.3V_AVDD_HDMI_AP_GATED: supplied by +3.3V_RUN
@@ -458,7 +482,7 @@
 pcieport 0000:00:02.0: Signaling PME with IRQ 394
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
-r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0afb000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 395
+r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0c0a000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 395
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
@@ -471,7 +495,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:13 UTC (946684813)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:01:56 UTC (946684916)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-01-19 02:57:51.535541310 -0800
+++ /tmp/foob	2017-01-19 02:57:51.539541324 -0800
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -42,20 +42,20 @@
 	 #################################################################
 	 #################################################################
 	 #####################################################
-	 3.6 MiB/s
+	 3.5 MiB/s
 done
-Bytes transferred = 7453768 (71bc48 hex)
+Bytes transferred = 7456936 (71c8a8 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.4 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 67943 (10967 hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x71bc48 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x71c8a8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013966
@@ -63,7 +63,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc4-next-20170118-146406-g8fd170c (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Jan 17 21:49:31 PST 2017
+Linux version 4.10.0-rc4-next-20170119-146705-gfb62712 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Jan 19 02:48:15 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -75,7 +75,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c1432200, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1432600, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -88,7 +88,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1995724K/2097148K available (9216K kernel code, 1231K rwdata, 4176K rodata, 2048K init, 531K bss, 35888K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1995716K/2097148K available (9216K kernel code, 1232K rwdata, 4172K rodata, 2048K init, 535K bss, 35896K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -98,8 +98,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc1433d00   (1232 kB)
-       .bss : 0xc1435000 - 0xc14b9d28   ( 532 kB)
+      .data : 0xc1300000 - 0xc1434100   (1233 kB)
+       .bss : 0xc1436000 - 0xc14bbd28   ( 536 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU debugfs-based tracing is enabled.
@@ -298,8 +298,6 @@
 mmc0: Unknown controller version (3). You may experience problems.
 mmc0: Invalid maximum block size, assuming 512 bytes
 mmc0: SDHCI controller on 700b0600.sdhci [700b0600.sdhci] using ADMA
-elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
-input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 ledtrig-cpu: registered to indicate activity on CPUs
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
@@ -327,26 +325,28 @@
 tegra-dpaux 545c0000.dpaux: failed to get VDD supply: -517
 panel supply power not found, using dummy regulator
 ------------[ cut here ]------------
-WARNING: CPU: 2 PID: 98 at drivers/base/dd.c:349 driver_probe_device+0x288/0x2c0
+WARNING: CPU: 3 PID: 97 at drivers/base/dd.c:349 driver_probe_device+0x288/0x2c0
+elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
+input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 Modules linked in:
-CPU: 2 PID: 98 Comm: kworker/2:2 Not tainted 4.10.0-rc4-next-20170118-146406-g8fd170c #1
+CPU: 3 PID: 97 Comm: kworker/3:2 Not tainted 4.10.0-rc4-next-20170119-146705-gfb62712 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: events deferred_probe_work_func
 [<c030fb48>] (unwind_backtrace) from [<c030b158>] (show_stack+0x10/0x14)
-[<c030b158>] (show_stack) from [<c059e8d4>] (dump_stack+0x84/0xa4)
-[<c059e8d4>] (dump_stack) from [<c0341b48>] (__warn+0xd4/0x100)
+[<c030b158>] (show_stack) from [<c059fa94>] (dump_stack+0x84/0xa4)
+[<c059fa94>] (dump_stack) from [<c0341b48>] (__warn+0xd4/0x100)
 [<c0341b48>] (__warn) from [<c0341b94>] (warn_slowpath_null+0x20/0x28)
-[<c0341b94>] (warn_slowpath_null) from [<c07e3adc>] (driver_probe_device+0x288/0x2c0)
-[<c07e3adc>] (driver_probe_device) from [<c07e201c>] (bus_for_each_drv+0x44/0x8c)
-[<c07e201c>] (bus_for_each_drv) from [<c07e37cc>] (__device_attach+0x9c/0x100)
-[<c07e37cc>] (__device_attach) from [<c07e2eb4>] (bus_probe_device+0x84/0x8c)
-[<c07e2eb4>] (bus_probe_device) from [<c07e32e8>] (deferred_probe_work_func+0x60/0x8c)
-[<c07e32e8>] (deferred_probe_work_func) from [<c0357038>] (process_one_work+0x134/0x33c)
-[<c0357038>] (process_one_work) from [<c0357268>] (process_scheduled_works+0x28/0x38)
-[<c0357268>] (process_scheduled_works) from [<c0357490>] (worker_thread+0x218/0x4f0)
-[<c0357490>] (worker_thread) from [<c035c808>] (kthread+0xfc/0x134)
-[<c035c808>] (kthread) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
----[ end trace 292e21496f28c34e ]---
+[<c0341b94>] (warn_slowpath_null) from [<c07e509c>] (driver_probe_device+0x288/0x2c0)
+[<c07e509c>] (driver_probe_device) from [<c07e35dc>] (bus_for_each_drv+0x44/0x8c)
+[<c07e35dc>] (bus_for_each_drv) from [<c07e4d8c>] (__device_attach+0x9c/0x100)
+[<c07e4d8c>] (__device_attach) from [<c07e4474>] (bus_probe_device+0x84/0x8c)
+[<c07e4474>] (bus_probe_device) from [<c07e48a8>] (deferred_probe_work_func+0x60/0x8c)
+[<c07e48a8>] (deferred_probe_work_func) from [<c0357078>] (process_one_work+0x134/0x33c)
+[<c0357078>] (process_one_work) from [<c03572a8>] (process_scheduled_works+0x28/0x38)
+[<c03572a8>] (process_scheduled_works) from [<c03574d0>] (worker_thread+0x218/0x4f0)
+[<c03574d0>] (worker_thread) from [<c035c848>] (kthread+0xfc/0x134)
+[<c035c848>] (kthread) from [<c0307f78>] (ret_from_fork+0x14/0x3c)
+---[ end trace 365f4a3cb4dce2d2 ]---
 sdhci-tegra 700b0000.sdhci: allocated mmc-pwrseq
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
@@ -357,25 +357,25 @@
 +VDDIO_SDMMC3: supplied by +3.3V_RUN
 +1.8V_RUN_VPP_FUSE: supplied by +3.3V_RUN
 +3.3V_LP0: supplied by +3.3V_SYS
+mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
 elan_i2c 1-0015: failed to read report data: -121
+mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
 tegra-sor 54540000.sor: failed to probe output: -517
 panel supply power not found, using dummy regulator
 sdhci-tegra 700b0000.sdhci: allocated mmc-pwrseq
 mmc1: Unknown controller version (3). You may experience problems.
-mmc1: Invalid maximum block size, assuming 512 bytes
-mmc1: Switching to 3.3V signalling voltage failed
-mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
-mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
+mmc1: Invalid maximum block size, assuming 512 bytes
 mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
+mmc1: Switching to 3.3V signalling voltage failed
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 mmc1: SDHCI controller on 700b0000.sdhci [700b0000.sdhci] using ADMA
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc2: Unknown controller version (3). You may experience problems.
-mmc2: Invalid maximum block size, assuming 512 bytes
 elan_i2c 1-0015: invalid report id data (ff)
+mmc2: Invalid maximum block size, assuming 512 bytes
 mmc1: new high speed SDIO card at address 0001
 mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2017-01-19 02:57:51.543541338 -0800
+++ /tmp/foob	2017-01-19 02:57:51.547541352 -0800
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -39,21 +39,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ####################################################
-	 3.5 MiB/s
+	 #####################################################
+	 3.6 MiB/s
 done
-Bytes transferred = 5528928 (545d60 hex)
+Bytes transferred = 5537072 (547d30 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 2.9 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 67943 (10967 hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x545d60 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x547d30 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013966
@@ -62,7 +62,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc4-next-20170118-146406-g8fd170c (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Jan 17 21:49:10 PST 2017
+Linux version 4.10.0-rc4-next-20170119-146705-gfb62712 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Jan 19 02:48:15 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -73,7 +73,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c0d5ed40, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0d5f100, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -86,7 +86,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2000824K/2097148K available (8192K kernel code, 385K rwdata, 2480K rodata, 1024K init, 364K bss, 30788K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 2000820K/2097148K available (8192K kernel code, 385K rwdata, 2480K rodata, 1024K init, 368K bss, 30792K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -96,8 +96,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d60400   ( 385 kB)
-       .bss : 0xc0d62000 - 0xc0dbd34c   ( 365 kB)
+      .data : 0xc0d00000 - 0xc0d607c0   ( 386 kB)
+       .bss : 0xc0d62000 - 0xc0dbe34c   ( 369 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -298,13 +298,13 @@
 mmc0: new high speed MMC card at address 0001
 sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
 mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
+mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
 NET: Registered protocol family 17
 NET: Registered protocol family 15
-mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
+can: controller area network core (rev 20120528 abi 9)
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
 mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
-can: controller area network core (rev 20120528 abi 9)
 NET: Registered protocol family 29
 can: raw protocol (rev 20120528)
 can: broadcast manager protocol (rev 20161123 t)
@@ -381,7 +381,7 @@
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input4
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:23 UTC (946684823)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:22 UTC (946684822)
 +1.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
 +1.05V_RUN_CAM_REAR: disabling
@@ -402,4 +402,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.983520] usb 1-3: new high-speed USB device number 3 using tegra-xusb
+root@(none):/# [    2.983528] usb 1-3: new high-speed USB device number 3 using tegra-xusb
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2017-01-19 02:57:51.551541366 -0800
+++ /tmp/foob	2017-01-19 02:57:51.555541380 -0800
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1800657 us
+Bootloader-Cpu Init at (time stamp): 1800658 us
 
 Pinmux changes applied in kernel way
 
@@ -62,20 +62,20 @@
 Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1800631  us
-Bootloader (AosInit) time: 733471 us
-Bootloader Main Init time: 180233 us
-BlockDevMgrInit time: 157497 us
+Miscellaneous avp time: 1800632  us
+Bootloader (AosInit) time: 733441 us
+Bootloader Main Init time: 180284 us
+BlockDevMgrInit time: 157500 us
 PartMgrInit time: 8 us
 StorMgrInit time: 31 us
 FsMgrInit time: 2 us
-Partition Table load: 77145 us 
-Display init time: 574495 us
+Partition Table load: 77161 us 
+Display init time: 574608 us
 Kernel read time: 378 msec
 Load Kernel Size:   16304128 Bytes
 Bl Kernel ReadSpeed: 43132000 Bytes/sec
-Pre-Kernel Jump time: 1740148 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5842036 us
+Pre-Kernel Jump time: 1740686 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5844800 us
 Booting Linux on physical CPU 0x0
 Linux version 4.8.0-239303-g3477d16 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Fri Oct 7 01:12:37 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
@@ -184,7 +184,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(1.167:1): initialized
+audit: type=2000 audit(1.179:1): initialized
 workingset: timestamp_bits=46 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -251,10 +251,11 @@
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-+VDD_LED: supplied by +VDD_MUX
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
++VDD_LED: supplied by +VDD_MUX
 +5V_USB_HS: supplied by +5V_SYS
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 +5V_USB_SS: supplied by +5V_SYS
 +5V_HDMI_CON: supplied by +5V_SYS
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
@@ -272,9 +273,6 @@
 Freeing unused kernel memory: 960K (ffff800000c00000 - ffff800000cf0000)
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
-sdhci-tegra 700b0400.sdhci: Got CD GPIO
-sdhci-tegra 700b0400.sdhci: Got WP GPIO
 
 System initialization...
 
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-01-19 02:57:51.559541394 -0800
+++ /tmp/foob	2017-01-19 02:57:51.563541409 -0800
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -49,7 +49,7 @@
 	 #####################################################
 	 4.1 MiB/s
 done
-Bytes transferred = 7453768 (71bc48 hex)
+Bytes transferred = 7456936 (71c8a8 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -60,7 +60,7 @@
 done
 Bytes transferred = 21404 (539c hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x71bc48 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x71c8a8 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 0241039b
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc4-next-20170118-146406-g8fd170c (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Jan 17 21:49:31 PST 2017
+Linux version 4.10.0-rc4-next-20170119-146705-gfb62712 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Jan 19 02:48:15 PST 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: Compulab TrimSlice board
@@ -78,7 +78,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c1432200, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c1432600, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -91,7 +91,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 955648K/1048576K available (9216K kernel code, 1231K rwdata, 4176K rodata, 2048K init, 531K bss, 27392K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 955640K/1048576K available (9216K kernel code, 1232K rwdata, 4172K rodata, 2048K init, 535K bss, 27400K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -101,8 +101,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc1433d00   (1232 kB)
-       .bss : 0xc1435000 - 0xc14b9d28   ( 532 kB)
+      .data : 0xc1300000 - 0xc1434100   (1233 kB)
+       .bss : 0xc1436000 - 0xc14bbd28   ( 536 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	RCU debugfs-based tracing is enabled.
@@ -344,7 +344,7 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2015-02-06 04:13:31 UTC (1423196011)
+rtc-em3027 2-0056: setting system clock to 2015-02-06 04:18:05 UTC (1423196285)
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
@@ -352,8 +352,8 @@
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
-random: fast init done
 asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
+random: fast init done
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2017-01-19 02:57:51.567541423 -0800
+++ /tmp/foob	2017-01-19 02:57:51.571541436 -0800
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -44,21 +44,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ####################################################
-	 4 MiB/s
+	 #####################################################
+	 4.1 MiB/s
 done
-Bytes transferred = 5528928 (545d60 hex)
+Bytes transferred = 5537072 (547d30 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 4.1 MiB/s
 done
 Bytes transferred = 21404 (539c hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x545d60 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x547d30 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 0241039b
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc4-next-20170118-146406-g8fd170c (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Jan 17 21:49:10 PST 2017
+Linux version 4.10.0-rc4-next-20170119-146705-gfb62712 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Jan 19 02:48:15 PST 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: Compulab TrimSlice board
@@ -76,7 +76,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0d5ed40, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0d5f100, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -89,7 +89,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 960748K/1048576K available (8192K kernel code, 385K rwdata, 2480K rodata, 1024K init, 364K bss, 22292K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 960744K/1048576K available (8192K kernel code, 385K rwdata, 2480K rodata, 1024K init, 368K bss, 22296K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -99,8 +99,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d60400   ( 385 kB)
-       .bss : 0xc0d62000 - 0xc0dbd34c   ( 365 kB)
+      .data : 0xc0d00000 - 0xc0d607c0   ( 386 kB)
+       .bss : 0xc0d62000 - 0xc0dbe34c   ( 369 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
@@ -313,18 +313,14 @@
 tlv320aic23-codec 2-001a: ASoC: no dapm match for MICIN --> NULL --> Mic Input
 tlv320aic23-codec 2-001a: ASoC: Failed to add route MICIN -> NULL -> Mic Input
 tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 NET: Registered protocol family 10
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
- mmcblk0: p1
-mmc1: new high speed SD card at address b368
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
- mmcblk1: p1 p2
 Segment Routing with IPv6
 mip6: Mobile IPv6
 sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
 NET: Registered protocol family 17
 NET: Registered protocol family 15
 can: controller area network core (rev 20120528 abi 9)
@@ -343,14 +339,18 @@
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
+ mmcblk0: p1
+mmc1: new high speed SD card at address b368
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
+ mmcblk1: p1 p2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
-hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2015-02-06 04:11:58 UTC (1423195918)
+hub 1-1:1.0: USB hub found
+hub 1-1:1.0: 4 ports detected
+rtc-em3027 2-0056: setting system clock to 2015-02-06 04:19:35 UTC (1423196375)
 ALSA device list:
   #0: tegra-trimslice
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2017-01-19 02:57:51.579541464 -0800
+++ /tmp/foob	2017-01-19 02:57:51.579541464 -0800
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 741700 us
-Basic modules init took 349132 us
+Tegraboot started after 739600 us
+Basic modules init took 349838 us
 Sec Bootdevice Read Time = 203 ms, Read Size = 8459 KB
-Next stage binary read took 15646 us
-Carveout took 252723 us
-CPU initialization took 92298 us
-Total time taken by TegraBoot 709799 us
+Next stage binary read took 14380 us
+Carveout took 252705 us
+CPU initialization took 92310 us
+Total time taken by TegraBoot 709233 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.582] LPDDR4 Training: Number of tables = 10
+64b[0001.580] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-15069696 bytes read in 799 ms (18 MiB/s)
+15071744 bytes read in 797 ms (18 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
 44882 bytes read in 454 ms (95.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 500 ms (3.1 MiB/s)
+1627787 bytes read in 501 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc4-next-20170118-170839-g8fd170c (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Jan 17 21:52:27 PST 2017
+Linux version 4.10.0-rc4-next-20170119-171138-gfb62712 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Jan 19 02:51:27 PST 2017
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -161,15 +161,15 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffff80007a000000-ffff80007dffffff]
-Memory: 2977844K/3131392K available (8508K kernel code, 956K rwdata, 4144K rodata, 1088K init, 389K bss, 137164K reserved, 16384K cma-reserved)
+Memory: 2977840K/3131392K available (8508K kernel code, 958K rwdata, 4144K rodata, 1088K init, 391K bss, 137168K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff0000088d0000   (  8512 KB)
     .rodata : 0xffff0000088d0000 - 0xffff000008ce0000   (  4160 KB)
       .init : 0xffff000008ce0000 - 0xffff000008df0000   (  1088 KB)
-      .data : 0xffff000008df0000 - 0xffff000008edf200   (   957 KB)
-       .bss : 0xffff000008edf200 - 0xffff000008f407b0   (   390 KB)
+      .data : 0xffff000008df0000 - 0xffff000008edfa00   (   959 KB)
+       .bss : 0xffff000008edfa00 - 0xffff000008f417b0   (   392 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -343,6 +343,6 @@
 devtmpfs: mounted
 Freeing unused kernel memory: 1088K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
-bash: no job control in th[    1.334319] random: fast init done
+bash: no job control in th[    1.336590] random: fast init done
 is shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2017-01-19 02:57:51.587541493 -0800
+++ /tmp/foob	2017-01-19 02:57:51.587541493 -0800
@@ -1,3 +1,9 @@
+U: AArch64 Processor [411fd071]
+efi: Getting EFI parameters from FDT:
+efi: UEFI not found.
+cma: Reserved 16 MiB at 0x00000000f5400000
+earlycon: uart0 at MMIO 0x0000000070006000 (options '115200n8')
+bootconsole [uart0] enabled
 psci: probing for conduit method from DT.
 psci: PSCIv1.0 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
@@ -12,15 +18,15 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2832032K/2988032K available (8508K kernel code, 956K rwdata, 4144K rodata, 1088K init, 389K bss, 139616K reserved, 16384K cma-reserved)
+Memory: 2832024K/2988032K available (8508K kernel code, 958K rwdata, 4144K rodata, 1088K init, 391K bss, 139624K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff0000088d0000   (  8512 KB)
     .rodata : 0xffff0000088d0000 - 0xffff000008ce0000   (  4160 KB)
       .init : 0xffff000008ce0000 - 0xffff000008df0000   (  1088 KB)
-      .data : 0xffff000008df0000 - 0xffff000008edf200   (   957 KB)
-       .bss : 0xffff000008edf200 - 0xffff000008f407b0   (   390 KB)
+      .data : 0xffff000008df0000 - 0xffff000008edfa00   (   959 KB)
+       .bss : 0xffff000008edfa00 - 0xffff000008f417b0   (   392 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -49,7 +55,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0xabc0: expected 0x2004f400
+boot misc3 0xaf90: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -62,7 +68,7 @@
 console [tty0] enabled
 bootconsole [uart0] disabled
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc4-next-20170118-170839-g8fd170c (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Jan 17 21:52:27 PST 2017
+Linux version 4.10.0-rc4-next-20170119-171138-gfb62712 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Jan 19 02:51:27 PST 2017
 Boot CPU: AArch64 Processor [411fd071]
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
@@ -83,15 +89,15 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2832032K/2988032K available (8508K kernel code, 956K rwdata, 4144K rodata, 1088K init, 389K bss, 139616K reserved, 16384K cma-reserved)
+Memory: 2832024K/2988032K available (8508K kernel code, 958K rwdata, 4144K rodata, 1088K init, 391K bss, 139624K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff0000088d0000   (  8512 KB)
     .rodata : 0xffff0000088d0000 - 0xffff000008ce0000   (  4160 KB)
       .init : 0xffff000008ce0000 - 0xffff000008df0000   (  1088 KB)
-      .data : 0xffff000008df0000 - 0xffff000008edf200   (   957 KB)
-       .bss : 0xffff000008edf200 - 0xffff000008f407b0   (   390 KB)
+      .data : 0xffff000008df0000 - 0xffff000008edfa00   (   959 KB)
+       .bss : 0xffff000008edfa00 - 0xffff000008f417b0   (   392 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -120,7 +126,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0xabc0: expected 0x2004f400
+boot misc3 0xaf90: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -198,7 +204,7 @@
 RPC: Registered tcp transport module.
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 Unpacking initramfs...
-Freeing initrd memory: 2360K
+Freeing initrd memory: 2364K
 kvm [1]: 8-bit VMID
 kvm [1]: IDMAP page: 808c2000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
@@ -208,7 +214,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.323:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.331:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=46 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -237,8 +243,8 @@
 PP2800L_RCAM: Bringing 3100000uV into 2800000-2800000uV
 PP1100_SOC_RTC: Bringing 1000000uV into 850000-850000uV
 PP2800L_FCAM: Bringing 3100000uV into 2800000-2800000uV
-PP_LDO6: Bringing 2800000uV into 1800000-1800000uV
 random: fast init done
+PP_LDO6: Bringing 2800000uV into 1800000-1800000uV
 hisi_sas: driver version v1.6
 libphy: Fixed MDIO Bus: probed
 tun: Universal TUN/TAP device driver, 1.6
@@ -298,18 +304,18 @@
 tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 BGND3R 29.1 GiB 
+max77686-rtc max77620-rtc: setting system clock to 2000-10-30 16:19:28 UTC (972922768)
 mmcblk0boot0: mmc0:0001 BGND3R partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
-max77686-rtc max77620-rtc: setting system clock to 2000-10-29 11:20:34 UTC (972818434)
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
-tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 PP1200_AVDD: disabling
 PP1200_RCAM: disabling
 PP2800L_RCAM: disabling
 PP2800L_FCAM: disabling
 PP_LDO6: disabling
 PPLCD_VDD: disabling
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
+tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 PP1800_LCDIO: disabling
 PP1800_CAM: disabling
 ALSA device list:
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-01-19 02:57:51.595541521 -0800
+++ /tmp/foob	2017-01-19 02:57:51.595541521 -0800
@@ -41,7 +41,7 @@
 	 #####################################################
 	 3.2 MiB/s
 done
-Bytes transferred = 7453768 (71bc48 hex)
+Bytes transferred = 7456936 (71c8a8 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -52,7 +52,7 @@
 done
 Bytes transferred = 51804 (ca5c hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x71bc48 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x71c8a8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa5b ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc4-next-20170118-146406-g8fd170c (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Jan 17 21:49:31 PST 2017
+Linux version 4.10.0-rc4-next-20170119-146705-gfb62712 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Jan 19 02:48:15 PST 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -70,7 +70,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c1432200, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1432600, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -83,7 +83,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1994748K/2096128K available (9216K kernel code, 1231K rwdata, 4176K rodata, 2048K init, 531K bss, 35844K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1994740K/2096128K available (9216K kernel code, 1232K rwdata, 4172K rodata, 2048K init, 535K bss, 35852K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -93,8 +93,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc1433d00   (1232 kB)
-       .bss : 0xc1435000 - 0xc14b9d28   ( 532 kB)
+      .data : 0xc1300000 - 0xc1434100   (1233 kB)
+       .bss : 0xc1436000 - 0xc14bbd28   ( 536 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU debugfs-based tracing is enabled.
@@ -396,5 +396,5 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.063582] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.083592] usb 2-1: new high-speed USB device number 2 using tegra-ehci
 asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0f:67:60
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2017-01-19 02:57:51.599541534 -0800
+++ /tmp/foob	2017-01-19 02:57:51.603541548 -0800
@@ -36,21 +36,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ####################################################
+	 #####################################################
 	 3.3 MiB/s
 done
-Bytes transferred = 5528928 (545d60 hex)
+Bytes transferred = 5537072 (547d30 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.9 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 51804 (ca5c hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x545d60 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x547d30 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa5b ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc4-next-20170118-146406-g8fd170c (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Jan 17 21:49:10 PST 2017
+Linux version 4.10.0-rc4-next-20170119-146705-gfb62712 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Jan 19 02:48:15 PST 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,7 +68,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0d5ed40, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0d5f100, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -81,7 +81,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999848K/2096128K available (8192K kernel code, 385K rwdata, 2480K rodata, 1024K init, 364K bss, 30744K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1999844K/2096128K available (8192K kernel code, 385K rwdata, 2480K rodata, 1024K init, 368K bss, 30748K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -91,8 +91,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d60400   ( 385 kB)
-       .bss : 0xc0d62000 - 0xc0dbd34c   ( 365 kB)
+      .data : 0xc0d00000 - 0xc0d607c0   ( 386 kB)
+       .bss : 0xc0d62000 - 0xc0dbe34c   ( 369 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -198,12 +198,8 @@
 bounce: pool size: 64 pages
 io scheduler noop registered (default)
 io scheduler mq-deadline registered
-tegra30-pinctrl 70000868.pinmux: invalid group "sdmmc3_dat6_pd3" for function "rsvd1"
 tegra-pcie 3000.pcie-controller: 2x3 configuration
-tegra-pcie 3000.pcie-controller: Failed to get supply 'hvdd-pex': -517
-tegra-apbdma 6000a000.dma: Tegra20 APB DMA driver register 32 channels
-tegra-pmc 7000e400.pmc: i2c-thermtrip node not found, emergency thermal reset disabled.
-Serial: 8250/16550 driver, 4 ports, IRQ sharing disabled
+tegra30-pinctrl 70000868.pinmux: invalid group "sdmmc3_dat6_pd3" for function "rsvd1"
 tegra30-pinctrl 70000868.pinmux: invalid group "sdmmc3_dat7_pd4" for function "rsvd1"
 tegra30-pinctrl 70000868.pinmux: invalid group "vi_vsync_pd6" for function "rsvd1"
 tegra30-pinctrl 70000868.pinmux: invalid group "vi_hsync_pd7" for function "rsvd1"
@@ -218,6 +214,10 @@
 tegra30-pinctrl 70000868.pinmux: invalid group "pv3" for function "rsvd1"
 tegra30-pinctrl 70000868.pinmux: invalid group "pbb0" for function "rsvd1"
 tegra30-pinctrl 70000868.pinmux: invalid group "pcc1" for function "rsvd1"
+tegra-pcie 3000.pcie-controller: Failed to get supply 'hvdd-pex': -517
+tegra-apbdma 6000a000.dma: Tegra20 APB DMA driver register 32 channels
+tegra-pmc 7000e400.pmc: i2c-thermtrip node not found, emergency thermal reset disabled.
+Serial: 8250/16550 driver, 4 ports, IRQ sharing disabled
 console [ttyS0] disabled
 70006000.serial: ttyS0 at MMIO 0x70006000 (irq = 331, base_baud = 25500000) is a Tegra
 console [ttyS0] enabled
@@ -391,4 +391,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.360422] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.350533] usb 2-1: new high-speed USB device number 2 using tegra-ehci
