Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2015-12-13 22:54:26.829305355 -0800
+++ /tmp/foob	2015-12-13 22:54:26.833305351 -0800
@@ -1,6 +1,6 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
-Linux version 4.4.0-rc4-next-20151211+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Dec 10 22:53:23 PST 2015
+Linux version 4.4.0-rc4-next-20151214+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Dec 13 22:49:37 PST 2015
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -20,7 +20,7 @@
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
 software IO TLB [mem 0x79c00000-0x7dc00000] (64MB) mapped at [ffffffc039c00000-ffffffc03dbfffff]
-Memory: 938124K/1048576K available (5912K kernel code, 559K rwdata, 2560K rodata, 660K init, 223K bss, 94068K reserved, 16384K cma-reserved)
+Memory: 938088K/1048576K available (5916K kernel code, 563K rwdata, 2572K rodata, 676K init, 223K bss, 94104K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     vmalloc : 0xffffff8000000000 - 0xffffffbdbfff0000   (   246 GB)
     vmemmap : 0xffffffbdc0000000 - 0xffffffbfc0000000   (     8 GB maximum)
@@ -29,9 +29,9 @@
     PCI I/O : 0xffffffbffae00000 - 0xffffffbffbe00000   (    16 MB)
     modules : 0xffffffbffc000000 - 0xffffffc000000000   (    64 MB)
     memory  : 0xffffffc000000000 - 0xffffffc040000000   (  1024 MB)
-      .init : 0xffffffc0008c9000 - 0xffffffc00096e000   (   660 KB)
-      .text : 0xffffffc000080000 - 0xffffffc0008c80b4   (  8481 KB)
-      .data : 0xffffffc00096e000 - 0xffffffc0009f9e00   (   560 KB)
+      .init : 0xffffffc0008cd000 - 0xffffffc000976000   (   676 KB)
+      .text : 0xffffffc000080000 - 0xffffffc0008cc0b4   (  8497 KB)
+      .data : 0xffffffc000976000 - 0xffffffc000a02e00   (   564 KB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 64.
@@ -62,7 +62,7 @@
 NET: Registered protocol family 16
 cpuidle: using governor ladder
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffffffc000975000, 1 data @ ffffffc000974000)
+vdso: 2 pages (1 code @ ffffffc00097d000, 1 data @ ffffffc00097c000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -93,7 +93,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.592:1): initialized
+audit: type=2000 audit(0.604:1): initialized
 HugeTLB registered 2 MB page size, pre-allocated 0 pages
 VFS: Disk quotas dquot_6.6.0
 VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
@@ -142,7 +142,7 @@
 EXT4-fs (vda2): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 254:2.
 devtmpfs: mounted
-Freeing unused kernel memory: 660K (ffffffc0008c9000 - ffffffc00096e000)
+Freeing unused kernel memory: 676K (ffffffc0008cd000 - ffffffc000976000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 bash-4.3# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-12-13 22:54:26.841305346 -0800
+++ /tmp/foob	2015-12-13 22:54:26.845305343 -0800
@@ -37,21 +37,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ############################################################
-	 3.3 MiB/s
+	 #############################################################
+	 3.2 MiB/s
 done
-Bytes transferred = 6599800 (64b478 hex)
+Bytes transferred = 6610128 (64dcd0 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.8 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x64b478 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x64dcd0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-rc4-next-20151211-60578-g9379b04 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Dec 10 22:51:56 PST 2015
+Linux version 4.4.0-rc4-next-20151214-61244-g7e8998e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Dec 13 22:48:09 PST 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -67,7 +67,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0ffa980, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1001640, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -80,7 +80,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998348K/2096128K available (8334K kernel code, 1090K rwdata, 3840K rodata, 1020K init, 344K bss, 32244K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998320K/2096128K available (8338K kernel code, 1093K rwdata, 3848K rodata, 1032K init, 344K bss, 32272K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,10 +88,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0dec824   (12179 kB)
-      .init : 0xc0ded000 - 0xc0eec000   (1020 kB)
-      .data : 0xc0eec000 - 0xc0ffc900   (1091 kB)
-       .bss : 0xc0fff000 - 0xc10550a8   ( 345 kB)
+      .text : 0xc0208000 - 0xc0def87c   (12191 kB)
+      .init : 0xc0df0000 - 0xc0ef2000   (1032 kB)
+      .data : 0xc0ef2000 - 0xc10035c0   (1094 kB)
+       .bss : 0xc1006000 - 0xc105c0a8   ( 345 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -293,12 +293,12 @@
 WARNING: CPU: 2 PID: 65 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 2 PID: 65 Comm: kworker/u8:1 Not tainted 4.4.0-rc4-next-20151211-60578-g9379b04 #1
+CPU: 2 PID: 65 Comm: kworker/u8:1 Not tainted 4.4.0-rc4-next-20151214-61244-g7e8998e #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0218878>] (unwind_backtrace) from [<c0213f10>] (show_stack+0x10/0x14)
-[<c0213f10>] (show_stack) from [<c04863ac>] (dump_stack+0x78/0x94)
-[<c04863ac>] (dump_stack) from [<c024ab28>] (warn_slowpath_common+0x74/0xb0)
+[<c0213f10>] (show_stack) from [<c04863cc>] (dump_stack+0x78/0x94)
+[<c04863cc>] (dump_stack) from [<c024ab28>] (warn_slowpath_common+0x74/0xb0)
 [<c024ab28>] (warn_slowpath_common) from [<c024abf8>] (warn_slowpath_fmt+0x30/0x40)
 [<c024abf8>] (warn_slowpath_fmt) from [<c025ddd4>] (check_flush_dependency+0xc8/0x124)
 [<c025ddd4>] (check_flush_dependency) from [<c025dee8>] (flush_work+0x54/0x144)
@@ -310,28 +310,28 @@
 [<c021e31c>] (__alloc_from_contiguous) from [<c021ff10>] (__dma_alloc+0x21c/0x250)
 [<c021ff10>] (__dma_alloc) from [<c021ffc8>] (arm_dma_alloc+0x48/0x54)
 [<c021ffc8>] (arm_dma_alloc) from [<c0305e78>] (dma_pool_alloc+0x1dc/0x210)
-[<c0305e78>] (dma_pool_alloc) from [<c07a3c30>] (ehci_qh_alloc+0x3c/0xb4)
-[<c07a3c30>] (ehci_qh_alloc) from [<c07a8228>] (ehci_setup+0x154/0x478)
-[<c07a8228>] (ehci_setup) from [<c07ae2e0>] (tegra_ehci_reset+0xc/0x50)
-[<c07ae2e0>] (tegra_ehci_reset) from [<c077e918>] (usb_add_hcd+0x300/0x830)
-[<c077e918>] (usb_add_hcd) from [<c07adc58>] (tegra_ehci_probe+0x3b8/0x504)
-[<c07adc58>] (tegra_ehci_probe) from [<c06446f8>] (platform_drv_probe+0x50/0xac)
-[<c06446f8>] (platform_drv_probe) from [<c064305c>] (driver_probe_device+0x1d4/0x290)
-[<c064305c>] (driver_probe_device) from [<c0641654>] (bus_for_each_drv+0x44/0x8c)
-[<c0641654>] (bus_for_each_drv) from [<c0642dd8>] (__device_attach+0xa0/0x104)
-[<c0642dd8>] (__device_attach) from [<c06424d4>] (bus_probe_device+0x84/0x8c)
-[<c06424d4>] (bus_probe_device) from [<c0642908>] (deferred_probe_work_func+0x60/0x8c)
-[<c0642908>] (deferred_probe_work_func) from [<c025ed30>] (process_one_work+0x134/0x338)
+[<c0305e78>] (dma_pool_alloc) from [<c07a58fc>] (ehci_qh_alloc+0x3c/0xb4)
+[<c07a58fc>] (ehci_qh_alloc) from [<c07a9ef4>] (ehci_setup+0x154/0x478)
+[<c07a9ef4>] (ehci_setup) from [<c07affac>] (tegra_ehci_reset+0xc/0x50)
+[<c07affac>] (tegra_ehci_reset) from [<c07805e4>] (usb_add_hcd+0x300/0x830)
+[<c07805e4>] (usb_add_hcd) from [<c07af924>] (tegra_ehci_probe+0x3b8/0x504)
+[<c07af924>] (tegra_ehci_probe) from [<c0645ae0>] (platform_drv_probe+0x50/0xac)
+[<c0645ae0>] (platform_drv_probe) from [<c0644444>] (driver_probe_device+0x1d4/0x290)
+[<c0644444>] (driver_probe_device) from [<c0642a3c>] (bus_for_each_drv+0x44/0x8c)
+[<c0642a3c>] (bus_for_each_drv) from [<c06441c0>] (__device_attach+0xa0/0x104)
+[<c06441c0>] (__device_attach) from [<c06438bc>] (bus_probe_device+0x84/0x8c)
+[<c06438bc>] (bus_probe_device) from [<c0643cf0>] (deferred_probe_work_func+0x60/0x8c)
+[<c0643cf0>] (deferred_probe_work_func) from [<c025ed30>] (process_one_work+0x134/0x338)
 [<c025ed30>] (process_one_work) from [<c025ef98>] (worker_thread+0x2c/0x4f0)
 [<c025ef98>] (worker_thread) from [<c0264128>] (kthread+0xd8/0xf4)
 [<c0264128>] (kthread) from [<c0210df8>] (ret_from_fork+0x14/0x3c)
----[ end trace 6c5737e6a9e0850a ]---
+---[ end trace 8b36e8c5121df2e7 ]---
 tegra-ehci 7d008000.usb: irq 339, io mem 0x7d008000
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-06-22 15:51:04 UTC (961689064)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-06-25 15:47:21 UTC (961948041)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -340,7 +340,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1020K (c0ded000 - c0eec000)
+Freeing unused kernel memory: 1032K (c0df0000 - c0ef2000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.484194] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.483995] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2015-12-13 22:54:26.849305339 -0800
+++ /tmp/foob	2015-12-13 22:54:26.853305335 -0800
@@ -39,18 +39,18 @@
 	 ################################
 	 3.3 MiB/s
 done
-Bytes transferred = 5231152 (4fd230 hex)
+Bytes transferred = 5231072 (4fd1e0 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.8 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4fd230 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4fd1e0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.4.0-rc4-next-20151211-60578-g9379b04 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Dec 10 22:52:12 PST 2015
+Linux version 4.4.0-rc4-next-20151214-61244-g7e8998e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Dec 13 22:48:25 PST 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -70,7 +70,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0a24d00, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0a24d40, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -83,7 +83,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002248K/2096128K available (7179K kernel code, 370K rwdata, 2464K rodata, 344K init, 367K bss, 28344K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2002248K/2096128K available (7183K kernel code, 370K rwdata, 2464K rodata, 340K init, 367K bss, 28344K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -91,9 +91,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09732f4   (9645 kB)
-      .init : 0xc0974000 - 0xc09ca000   ( 344 kB)
-      .data : 0xc09ca000 - 0xc0a26820   ( 371 kB)
+      .text : 0xc0008000 - 0xc09742d8   (9649 kB)
+      .init : 0xc0975000 - 0xc09ca000   ( 340 kB)
+      .data : 0xc09ca000 - 0xc0a26860   ( 371 kB)
        .bss : 0xc0a29000 - 0xc0a84d90   ( 368 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -296,12 +296,12 @@
 WARNING: CPU: 0 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-rc4-next-20151211-60578-g9379b04 #1
+CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-rc4-next-20151214-61244-g7e8998e #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0017248>] (unwind_backtrace) from [<c0012870>] (show_stack+0x10/0x14)
-[<c0012870>] (show_stack) from [<c02392e4>] (dump_stack+0x84/0xd0)
-[<c02392e4>] (dump_stack) from [<c0026b60>] (warn_slowpath_common+0x74/0xb0)
+[<c0012870>] (show_stack) from [<c0239324>] (dump_stack+0x84/0xd0)
+[<c0239324>] (dump_stack) from [<c0026b60>] (warn_slowpath_common+0x74/0xb0)
 [<c0026b60>] (warn_slowpath_common) from [<c0026c30>] (warn_slowpath_fmt+0x30/0x40)
 [<c0026c30>] (warn_slowpath_fmt) from [<c00398dc>] (check_flush_dependency+0xc8/0x12c)
 [<c00398dc>] (check_flush_dependency) from [<c00399f8>] (flush_work+0x54/0x11c)
@@ -313,28 +313,28 @@
 [<c001c250>] (__alloc_from_contiguous) from [<c001cf18>] (__dma_alloc+0x228/0x2ac)
 [<c001cf18>] (__dma_alloc) from [<c001d020>] (arm_dma_alloc+0x48/0x54)
 [<c001d020>] (arm_dma_alloc) from [<c00eceec>] (dma_pool_alloc+0x28c/0x304)
-[<c00eceec>] (dma_pool_alloc) from [<c03f7768>] (ehci_qh_alloc+0x3c/0xb4)
-[<c03f7768>] (ehci_qh_alloc) from [<c03fb954>] (ehci_setup+0x154/0x478)
-[<c03fb954>] (ehci_setup) from [<c03ff69c>] (tegra_ehci_reset+0xc/0x50)
-[<c03ff69c>] (tegra_ehci_reset) from [<c03e3154>] (usb_add_hcd+0x2fc/0x804)
-[<c03e3154>] (usb_add_hcd) from [<c03ff014>] (tegra_ehci_probe+0x3b8/0x504)
-[<c03ff014>] (tegra_ehci_probe) from [<c03322b0>] (platform_drv_probe+0x50/0xac)
-[<c03322b0>] (platform_drv_probe) from [<c0330d10>] (driver_probe_device+0x1d4/0x290)
-[<c0330d10>] (driver_probe_device) from [<c032f30c>] (bus_for_each_drv+0x44/0x8c)
-[<c032f30c>] (bus_for_each_drv) from [<c0330a8c>] (__device_attach+0xa0/0x104)
-[<c0330a8c>] (__device_attach) from [<c0330188>] (bus_probe_device+0x84/0x8c)
-[<c0330188>] (bus_probe_device) from [<c03305bc>] (deferred_probe_work_func+0x60/0x8c)
-[<c03305bc>] (deferred_probe_work_func) from [<c003a804>] (process_one_work+0x120/0x328)
+[<c00eceec>] (dma_pool_alloc) from [<c03f7d24>] (ehci_qh_alloc+0x3c/0xb4)
+[<c03f7d24>] (ehci_qh_alloc) from [<c03fbf10>] (ehci_setup+0x154/0x478)
+[<c03fbf10>] (ehci_setup) from [<c03ffc58>] (tegra_ehci_reset+0xc/0x50)
+[<c03ffc58>] (tegra_ehci_reset) from [<c03e3710>] (usb_add_hcd+0x2fc/0x804)
+[<c03e3710>] (usb_add_hcd) from [<c03ff5d0>] (tegra_ehci_probe+0x3b8/0x504)
+[<c03ff5d0>] (tegra_ehci_probe) from [<c0332578>] (platform_drv_probe+0x50/0xac)
+[<c0332578>] (platform_drv_probe) from [<c0330fd8>] (driver_probe_device+0x1d4/0x290)
+[<c0330fd8>] (driver_probe_device) from [<c032f5d4>] (bus_for_each_drv+0x44/0x8c)
+[<c032f5d4>] (bus_for_each_drv) from [<c0330d54>] (__device_attach+0xa0/0x104)
+[<c0330d54>] (__device_attach) from [<c0330450>] (bus_probe_device+0x84/0x8c)
+[<c0330450>] (bus_probe_device) from [<c0330884>] (deferred_probe_work_func+0x60/0x8c)
+[<c0330884>] (deferred_probe_work_func) from [<c003a804>] (process_one_work+0x120/0x328)
 [<c003a804>] (process_one_work) from [<c003aa70>] (worker_thread+0x2c/0x4b4)
 [<c003aa70>] (worker_thread) from [<c003fbac>] (kthread+0xd8/0xf4)
 [<c003fbac>] (kthread) from [<c000f938>] (ret_from_fork+0x14/0x3c)
----[ end trace e64ff063e5ffd2ef ]---
+---[ end trace 52eee0b6536e45b0 ]---
 tegra-ehci 7d008000.usb: irq 339, io mem 0x7d008000
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-06-22 15:51:41 UTC (961689101)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-06-25 15:47:53 UTC (961948073)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -345,7 +345,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 344K (c0974000 - c09ca000)
+Freeing unused kernel memory: 340K (c0975000 - c09ca000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-12-13 22:54:26.869305323 -0800
+++ /tmp/foob	2015-12-13 22:54:26.873305319 -0800
@@ -29,7 +29,6 @@
 Tegra124 (Jetson TK1) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash'
 Tegra124 (Jetson TK1) # setenv ipaddr 192.168.100.102; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 zImage.multi_v7_defconfig
-Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'zImage.multi_v7_defconfig'.
@@ -40,21 +39,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ############################################################
-	 3.3 MiB/s
+	 #############################################################
+	 3.2 MiB/s
 done
-Bytes transferred = 6599800 (64b478 hex)
+Bytes transferred = 6610128 (64dcd0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 2.9 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x64b478 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x64dcd0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012a89
@@ -62,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-rc4-next-20151211-60578-g9379b04 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Dec 10 22:51:56 PST 2015
+Linux version 4.4.0-rc4-next-20151214-61244-g7e8998e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Dec 13 22:48:09 PST 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -70,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0ffa980, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1001640, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -83,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998240K/2096128K available (8334K kernel code, 1090K rwdata, 3840K rodata, 1020K init, 344K bss, 32352K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998212K/2096128K available (8338K kernel code, 1093K rwdata, 3848K rodata, 1032K init, 344K bss, 32380K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -91,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0dec824   (12179 kB)
-      .init : 0xc0ded000 - 0xc0eec000   (1020 kB)
-      .data : 0xc0eec000 - 0xc0ffc900   (1091 kB)
-       .bss : 0xc0fff000 - 0xc10550a8   ( 345 kB)
+      .text : 0xc0208000 - 0xc0def87c   (12191 kB)
+      .init : 0xc0df0000 - 0xc0ef2000   (1032 kB)
+      .data : 0xc0ef2000 - 0xc10035c0   (1094 kB)
+       .bss : 0xc1006000 - 0xc105c0a8   ( 345 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -272,14 +271,6 @@
 usbhid: USB HID core driver
 NET: Registered protocol family 10
 sit: IPv6 over IPv4 tunneling driver
-mmc0: MAN_BKOPS_EN bit is not set
-mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
-Primary GPT is invalid, using alternate GPT.
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9
 NET: Registered protocol family 17
 can: controller area network core (rev 20120528 abi 9)
 NET: Registered protocol family 29
@@ -294,6 +285,14 @@
 +3.3V_LP0: supplied by +3.3V_SYS
 +1.35V_LP0(sd2): supplied by +5V_SYS
 +1.05V_RUN_AVDD: supplied by +1.35V_LP0(sd2)
+mmc0: MAN_BKOPS_EN bit is not set
+mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+Primary GPT is invalid, using alternate GPT.
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9
 tegra-pcie 1003000.pcie-controller: probing port 0, using 2 lanes
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
 tegra-pcie 1003000.pcie-controller: link 0 down, retrying
@@ -327,15 +326,15 @@
 tegra-ahci 70027000.sata: AHCI 0001.0301 32 slots 2 ports 3 Gbps 0x1 impl platform mode
 tegra-ahci 70027000.sata: flags: 64bit ncq sntf pm led pio slum part sadm sds apst 
 ------------[ cut here ]------------
-WARNING: CPU: 2 PID: 66 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
+WARNING: CPU: 1 PID: 66 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 2 PID: 66 Comm: kworker/u8:1 Not tainted 4.4.0-rc4-next-20151211-60578-g9379b04 #1
+CPU: 1 PID: 66 Comm: kworker/u8:1 Not tainted 4.4.0-rc4-next-20151214-61244-g7e8998e #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0218878>] (unwind_backtrace) from [<c0213f10>] (show_stack+0x10/0x14)
-[<c0213f10>] (show_stack) from [<c04863ac>] (dump_stack+0x78/0x94)
-[<c04863ac>] (dump_stack) from [<c024ab28>] (warn_slowpath_common+0x74/0xb0)
+[<c0213f10>] (show_stack) from [<c04863cc>] (dump_stack+0x78/0x94)
+[<c04863cc>] (dump_stack) from [<c024ab28>] (warn_slowpath_common+0x74/0xb0)
 [<c024ab28>] (warn_slowpath_common) from [<c024abf8>] (warn_slowpath_fmt+0x30/0x40)
 [<c024abf8>] (warn_slowpath_fmt) from [<c025ddd4>] (check_flush_dependency+0xc8/0x124)
 [<c025ddd4>] (check_flush_dependency) from [<c025dee8>] (flush_work+0x54/0x144)
@@ -346,24 +345,24 @@
 [<c030d254>] (cma_alloc) from [<c021e31c>] (__alloc_from_contiguous+0x38/0xdc)
 [<c021e31c>] (__alloc_from_contiguous) from [<c021ff10>] (__dma_alloc+0x21c/0x250)
 [<c021ff10>] (__dma_alloc) from [<c021ffc8>] (arm_dma_alloc+0x48/0x54)
-[<c021ffc8>] (arm_dma_alloc) from [<c0654edc>] (dmam_alloc_coherent+0x8c/0xf0)
-[<c0654edc>] (dmam_alloc_coherent) from [<c06acf8c>] (ahci_port_start+0xd0/0x1e0)
-[<c06acf8c>] (ahci_port_start) from [<c0694e34>] (ata_host_start+0xf8/0x1d4)
-[<c0694e34>] (ata_host_start) from [<c0699d50>] (ata_host_activate+0x20/0x134)
-[<c0699d50>] (ata_host_activate) from [<c06ac408>] (ahci_host_activate+0x3c/0x11c)
-[<c06ac408>] (ahci_host_activate) from [<c06ae34c>] (ahci_platform_init_host+0x1cc/0x3c8)
-[<c06ae34c>] (ahci_platform_init_host) from [<c06b05b0>] (tegra_ahci_probe+0x404/0x4c4)
-[<c06b05b0>] (tegra_ahci_probe) from [<c06446f8>] (platform_drv_probe+0x50/0xac)
-[<c06446f8>] (platform_drv_probe) from [<c064305c>] (driver_probe_device+0x1d4/0x290)
-[<c064305c>] (driver_probe_device) from [<c0641654>] (bus_for_each_drv+0x44/0x8c)
-[<c0641654>] (bus_for_each_drv) from [<c0642dd8>] (__device_attach+0xa0/0x104)
-[<c0642dd8>] (__device_attach) from [<c06424d4>] (bus_probe_device+0x84/0x8c)
-[<c06424d4>] (bus_probe_device) from [<c0642908>] (deferred_probe_work_func+0x60/0x8c)
-[<c0642908>] (deferred_probe_work_func) from [<c025ed30>] (process_one_work+0x134/0x338)
+[<c021ffc8>] (arm_dma_alloc) from [<c0656308>] (dmam_alloc_coherent+0x8c/0xf0)
+[<c0656308>] (dmam_alloc_coherent) from [<c06ae408>] (ahci_port_start+0xd0/0x1e0)
+[<c06ae408>] (ahci_port_start) from [<c06962b0>] (ata_host_start+0xf8/0x1d4)
+[<c06962b0>] (ata_host_start) from [<c069b1cc>] (ata_host_activate+0x20/0x134)
+[<c069b1cc>] (ata_host_activate) from [<c06ad884>] (ahci_host_activate+0x3c/0x11c)
+[<c06ad884>] (ahci_host_activate) from [<c06af7c8>] (ahci_platform_init_host+0x1cc/0x3c8)
+[<c06af7c8>] (ahci_platform_init_host) from [<c06b1a2c>] (tegra_ahci_probe+0x404/0x4c4)
+[<c06b1a2c>] (tegra_ahci_probe) from [<c0645ae0>] (platform_drv_probe+0x50/0xac)
+[<c0645ae0>] (platform_drv_probe) from [<c0644444>] (driver_probe_device+0x1d4/0x290)
+[<c0644444>] (driver_probe_device) from [<c0642a3c>] (bus_for_each_drv+0x44/0x8c)
+[<c0642a3c>] (bus_for_each_drv) from [<c06441c0>] (__device_attach+0xa0/0x104)
+[<c06441c0>] (__device_attach) from [<c06438bc>] (bus_probe_device+0x84/0x8c)
+[<c06438bc>] (bus_probe_device) from [<c0643cf0>] (deferred_probe_work_func+0x60/0x8c)
+[<c0643cf0>] (deferred_probe_work_func) from [<c025ed30>] (process_one_work+0x134/0x338)
 [<c025ed30>] (process_one_work) from [<c025ef98>] (worker_thread+0x2c/0x4f0)
 [<c025ef98>] (worker_thread) from [<c0264128>] (kthread+0xd8/0xf4)
 [<c0264128>] (kthread) from [<c0210df8>] (ret_from_fork+0x14/0x3c)
----[ end trace 82c57ca66d5d49e5 ]---
+---[ end trace ab67b1ac05ba4933 ]---
 scsi host0: tegra-ahci
 scsi host1: tegra-ahci
 ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 341
@@ -385,7 +384,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:19 UTC (946684819)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:18 UTC (946684818)
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
@@ -398,11 +397,11 @@
 EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
 EXT4-fs (mmcblk0p1): couldn't mount as ext2 due to feature incompatibilities
 EXT4-fs (mmcblk0p1): recovery complete
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 devtmpfs: mounted
-Freeing unused kernel memory: 1020K (c0ded000 - c0eec000)
+Freeing unused kernel memory: 1032K (c0df0000 - c0ef2000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2015-12-13 22:54:26.881305310 -0800
+++ /tmp/foob	2015-12-13 22:54:26.885305307 -0800
@@ -39,21 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 ################################
-	 782.2 KiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 5231152 (4fd230 hex)
+Bytes transferred = 5231072 (4fd1e0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
-Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 2.9 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x4fd230 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x4fd1e0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012a89
@@ -64,7 +63,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.4.0-rc4-next-20151211-60578-g9379b04 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Dec 10 22:52:12 PST 2015
+Linux version 4.4.0-rc4-next-20151214-61244-g7e8998e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Dec 13 22:48:25 PST 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -73,7 +72,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0a24d00, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0a24d40, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -86,7 +85,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002140K/2096128K available (7179K kernel code, 370K rwdata, 2464K rodata, 344K init, 367K bss, 28452K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2002140K/2096128K available (7183K kernel code, 370K rwdata, 2464K rodata, 340K init, 367K bss, 28452K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -94,9 +93,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09732f4   (9645 kB)
-      .init : 0xc0974000 - 0xc09ca000   ( 344 kB)
-      .data : 0xc09ca000 - 0xc0a26820   ( 371 kB)
+      .text : 0xc0008000 - 0xc09742d8   (9649 kB)
+      .init : 0xc0975000 - 0xc09ca000   ( 340 kB)
+      .data : 0xc09ca000 - 0xc0a26860   ( 371 kB)
        .bss : 0xc0a29000 - 0xc0a84d90   ( 368 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -306,42 +305,16 @@
 tegra-pcie 1003000.pcie-controller: link 0 down, ignoring
 tegra-pcie 1003000.pcie-controller: probing port 1, using 1 lanes
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000000
+tegra-pcie 1003000.pcie-controller: link 1 down, retrying
+tegra-pcie 1003000.pcie-controller: link 1 down, retrying
+tegra-pcie 1003000.pcie-controller: link 1 down, retrying
+tegra-pcie 1003000.pcie-controller: link 1 down, ignoring
 tegra-pcie 1003000.pcie-controller: PCI host bridge to bus 0000:00
 pci_bus 0000:00: root bus resource [mem 0x13000000-0x1fffffff]
 pci_bus 0000:00: root bus resource [mem 0x20000000-0x3fffffff pref]
 pci_bus 0000:00: root bus resource [bus 00-ff]
 pci_bus 0000:00: root bus resource [io  0x1000-0xffff]
-pci 0000:00:02.0: [10de:0e13] type 01 class 0x060400
-pci 0000:00:02.0: PME# supported from D0 D1 D2 D3hot D3cold
-PCI: bus0: Fast back to back transfers disabled
-pci 0000:00:02.0: bridge configuration invalid ([bus 00-00]), reconfiguring
-pci 0000:01:00.0: [10ec:8168] type 00 class 0x020000
-pci 0000:01:00.0: reg 0x10: [io  0x0000-0x00ff]
-pci 0000:01:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit]
-pci 0000:01:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
-pci 0000:01:00.0: supports D1 D2
-pci 0000:01:00.0: PME# supported from D0 D1 D2 D3hot D3cold
-PCI: bus1: Fast back to back transfers disabled
-pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
-pci 0000:00:02.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
-pci 0000:00:02.0: BAR 9: assigned [mem 0x20000000-0x200fffff 64bit pref]
-pci 0000:00:02.0: BAR 7: assigned [io  0x1000-0x1fff]
-pci 0000:01:00.0: BAR 4: assigned [mem 0x20000000-0x20003fff 64bit pref]
-pci 0000:01:00.0: BAR 2: assigned [mem 0x13000000-0x13000fff 64bit]
-pci 0000:01:00.0: BAR 0: assigned [io  0x1000-0x10ff]
-pci 0000:00:02.0: PCI bridge to [bus 01]
-pci 0000:00:02.0:   bridge window [io  0x1000-0x1fff]
-pci 0000:00:02.0:   bridge window [mem 0x13000000-0x130fffff]
-pci 0000:00:02.0:   bridge window [mem 0x20000000-0x200fffff 64bit pref]
-pci 0000:00:02.0: nv_msi_ht_cap_quirk didn't locate host bridge
-pcieport 0000:00:02.0: enabling device (0140 -> 0143)
-pcieport 0000:00:02.0: Signaling PME through PCIe PME interrupt
-pci 0000:01:00.0: Signaling PME through PCIe PME interrupt
-pcie_pme 0000:00:02.0:pcie01: service driver pcie_pme loaded
-r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
-r8169 0000:01:00.0: enabling device (0140 -> 0143)
-r8169 0000:01:00.0 eth0: RTL8168g/8111g at 0xf08f8000, 00:04:4b:25:b3:b4, XID 0c000800 IRQ 388
-r8169 0000:01:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
+PCI: bus0: Fast back to back transfers enabled
 +5V_HDMI_CON: supplied by +5V_SYS
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 +3.3V_RUN: supplied by +3.3V_SYS
@@ -360,12 +333,12 @@
 WARNING: CPU: 0 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-rc4-next-20151211-60578-g9379b04 #1
+CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-rc4-next-20151214-61244-g7e8998e #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0017248>] (unwind_backtrace) from [<c0012870>] (show_stack+0x10/0x14)
-[<c0012870>] (show_stack) from [<c02392e4>] (dump_stack+0x84/0xd0)
-[<c02392e4>] (dump_stack) from [<c0026b60>] (warn_slowpath_common+0x74/0xb0)
+[<c0012870>] (show_stack) from [<c0239324>] (dump_stack+0x84/0xd0)
+[<c0239324>] (dump_stack) from [<c0026b60>] (warn_slowpath_common+0x74/0xb0)
 [<c0026b60>] (warn_slowpath_common) from [<c0026c30>] (warn_slowpath_fmt+0x30/0x40)
 [<c0026c30>] (warn_slowpath_fmt) from [<c00398dc>] (check_flush_dependency+0xc8/0x12c)
 [<c00398dc>] (check_flush_dependency) from [<c00399f8>] (flush_work+0x54/0x11c)
@@ -376,24 +349,24 @@
 [<c00f2650>] (cma_alloc) from [<c001c250>] (__alloc_from_contiguous+0x38/0xdc)
 [<c001c250>] (__alloc_from_contiguous) from [<c001cf18>] (__dma_alloc+0x228/0x2ac)
 [<c001cf18>] (__dma_alloc) from [<c001d020>] (arm_dma_alloc+0x48/0x54)
-[<c001d020>] (arm_dma_alloc) from [<c03409c8>] (dmam_alloc_coherent+0x8c/0xf0)
-[<c03409c8>] (dmam_alloc_coherent) from [<c03867cc>] (ahci_port_start+0xd0/0x1e0)
-[<c03867cc>] (ahci_port_start) from [<c036e4a4>] (ata_host_start+0xf8/0x1d4)
-[<c036e4a4>] (ata_host_start) from [<c0373438>] (ata_host_activate+0x20/0x134)
-[<c0373438>] (ata_host_activate) from [<c0385c74>] (ahci_host_activate+0x3c/0x11c)
-[<c0385c74>] (ahci_host_activate) from [<c038801c>] (ahci_platform_init_host+0x1cc/0x3c8)
-[<c038801c>] (ahci_platform_init_host) from [<c0387ce8>] (tegra_ahci_probe+0x404/0x4c4)
-[<c0387ce8>] (tegra_ahci_probe) from [<c03322b0>] (platform_drv_probe+0x50/0xac)
-[<c03322b0>] (platform_drv_probe) from [<c0330d10>] (driver_probe_device+0x1d4/0x290)
-[<c0330d10>] (driver_probe_device) from [<c032f30c>] (bus_for_each_drv+0x44/0x8c)
-[<c032f30c>] (bus_for_each_drv) from [<c0330a8c>] (__device_attach+0xa0/0x104)
-[<c0330a8c>] (__device_attach) from [<c0330188>] (bus_probe_device+0x84/0x8c)
-[<c0330188>] (bus_probe_device) from [<c03305bc>] (deferred_probe_work_func+0x60/0x8c)
-[<c03305bc>] (deferred_probe_work_func) from [<c003a804>] (process_one_work+0x120/0x328)
+[<c001d020>] (arm_dma_alloc) from [<c0340c90>] (dmam_alloc_coherent+0x8c/0xf0)
+[<c0340c90>] (dmam_alloc_coherent) from [<c0386ae4>] (ahci_port_start+0xd0/0x1e0)
+[<c0386ae4>] (ahci_port_start) from [<c036e7bc>] (ata_host_start+0xf8/0x1d4)
+[<c036e7bc>] (ata_host_start) from [<c0373750>] (ata_host_activate+0x20/0x134)
+[<c0373750>] (ata_host_activate) from [<c0385f8c>] (ahci_host_activate+0x3c/0x11c)
+[<c0385f8c>] (ahci_host_activate) from [<c0388334>] (ahci_platform_init_host+0x1cc/0x3c8)
+[<c0388334>] (ahci_platform_init_host) from [<c0388000>] (tegra_ahci_probe+0x404/0x4c4)
+[<c0388000>] (tegra_ahci_probe) from [<c0332578>] (platform_drv_probe+0x50/0xac)
+[<c0332578>] (platform_drv_probe) from [<c0330fd8>] (driver_probe_device+0x1d4/0x290)
+[<c0330fd8>] (driver_probe_device) from [<c032f5d4>] (bus_for_each_drv+0x44/0x8c)
+[<c032f5d4>] (bus_for_each_drv) from [<c0330d54>] (__device_attach+0xa0/0x104)
+[<c0330d54>] (__device_attach) from [<c0330450>] (bus_probe_device+0x84/0x8c)
+[<c0330450>] (bus_probe_device) from [<c0330884>] (deferred_probe_work_func+0x60/0x8c)
+[<c0330884>] (deferred_probe_work_func) from [<c003a804>] (process_one_work+0x120/0x328)
 [<c003a804>] (process_one_work) from [<c003aa70>] (worker_thread+0x2c/0x4b4)
 [<c003aa70>] (worker_thread) from [<c003fbac>] (kthread+0xd8/0xf4)
 [<c003fbac>] (kthread) from [<c000f938>] (ret_from_fork+0x14/0x3c)
----[ end trace efa526c62bfad89a ]---
+---[ end trace d2b4ae76a7e97134 ]---
 scsi host0: tegra-ahci
 scsi host1: tegra-ahci
 ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 341
@@ -415,7 +388,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:59 UTC (946684859)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:55 UTC (946684855)
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
@@ -430,11 +403,11 @@
 ata1: SATA link down (SStatus 0 SControl 300)
 EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
 EXT4-fs (mmcblk0p1): recovery complete
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 344K (c0974000 - c09ca000)
+Freeing unused kernel memory: 340K (c0975000 - c09ca000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    5.712754] asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:11:31:3b
+root@(none):/# [    6.722472] asix 2-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:11:31:3b
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-12-13 22:54:26.897305299 -0800
+++ /tmp/foob	2015-12-13 22:54:26.897305299 -0800
@@ -45,21 +45,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ############################################################
+	 #############################################################
 	 4 MiB/s
 done
-Bytes transferred = 6599800 (64b478 hex)
+Bytes transferred = 6610128 (64dcd0 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 4 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x64b478 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x64dcd0 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -67,7 +67,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-rc4-next-20151211-60578-g9379b04 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Dec 10 22:51:56 PST 2015
+Linux version 4.4.0-rc4-next-20151214-61244-g7e8998e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Dec 13 22:48:09 PST 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -75,7 +75,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0ffa980, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c1001640, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -88,7 +88,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 959164K/1048576K available (8334K kernel code, 1090K rwdata, 3840K rodata, 1020K init, 344K bss, 23876K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 959136K/1048576K available (8338K kernel code, 1093K rwdata, 3848K rodata, 1032K init, 344K bss, 23904K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -96,10 +96,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0dec824   (12179 kB)
-      .init : 0xc0ded000 - 0xc0eec000   (1020 kB)
-      .data : 0xc0eec000 - 0xc0ffc900   (1091 kB)
-       .bss : 0xc0fff000 - 0xc10550a8   ( 345 kB)
+      .text : 0xc0208000 - 0xc0def87c   (12191 kB)
+      .init : 0xc0df0000 - 0xc0ef2000   (1032 kB)
+      .data : 0xc0ef2000 - 0xc10035c0   (1094 kB)
+       .bss : 0xc1006000 - 0xc105c0a8   ( 345 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -321,24 +321,24 @@
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
- mmcblk0: p1
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
+ mmcblk0: p1
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-27 02:56:44 UTC (1419649004)
+rtc-em3027 2-0056: setting system clock to 2014-12-27 02:57:59 UTC (1419649079)
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
-mmc1: new high speed SD card at address b368
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 EXT4-fs (mmcblk0p1): recovery complete
- mmcblk1: p1 p2
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1020K (c0ded000 - c0eec000)
+Freeing unused kernel memory: 1032K (c0df0000 - c0ef2000)
+mmc1: new high speed SD card at address b368
 hub 1-1:1.0: USB hub found
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 hub 1-1:1.0: 4 ports detected
+ mmcblk1: p1 p2
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2015-12-13 22:54:26.905305291 -0800
+++ /tmp/foob	2015-12-13 22:54:26.909305288 -0800
@@ -47,7 +47,7 @@
 	 ################################
 	 4 MiB/s
 done
-Bytes transferred = 5231152 (4fd230 hex)
+Bytes transferred = 5231072 (4fd1e0 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x4fd230 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x4fd1e0 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -69,7 +69,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.4.0-rc4-next-20151211-60578-g9379b04 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Dec 10 22:52:12 PST 2015
+Linux version 4.4.0-rc4-next-20151214-61244-g7e8998e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Dec 13 22:48:25 PST 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -78,7 +78,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0a24d00, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0a24d40, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -91,7 +91,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 963064K/1048576K available (7179K kernel code, 370K rwdata, 2464K rodata, 344K init, 367K bss, 19976K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 963064K/1048576K available (7183K kernel code, 370K rwdata, 2464K rodata, 340K init, 367K bss, 19976K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -99,9 +99,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09732f4   (9645 kB)
-      .init : 0xc0974000 - 0xc09ca000   ( 344 kB)
-      .data : 0xc09ca000 - 0xc0a26820   ( 371 kB)
+      .text : 0xc0008000 - 0xc09742d8   (9649 kB)
+      .init : 0xc0975000 - 0xc09ca000   ( 340 kB)
+      .data : 0xc09ca000 - 0xc0a26860   ( 371 kB)
        .bss : 0xc0a29000 - 0xc0a84d90   ( 368 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -308,12 +308,14 @@
 tlv320aic23-codec 2-001a: ASoC: no dapm match for MICIN --> NULL --> Mic Input
 tlv320aic23-codec 2-001a: ASoC: Failed to add route MICIN -> NULL -> Mic Input
 tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
-NET: Registered protocol family 10
+mmc0: host does not support reading read-only switch, assuming write-enable
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
+mmc0: new high speed SDHC card at address b368
+NET: Registered protocol family 10
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
 mip6: Mobile IPv6
 sit: IPv6 over IPv4 tunneling driver
-mmc0: host does not support reading read-only switch, assuming write-enable
-mmc0: new high speed SDHC card at address b368
+ mmcblk0: p1
 NET: Registered protocol family 17
 NET: Registered protocol family 15
 can: controller area network core (rev 20120528 abi 9)
@@ -332,64 +334,25 @@
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
-------------[ cut here ]------------
-WARNING: CPU: 1 PID: 74 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
-workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
-Modules linked in:
-CPU: 1 PID: 74 Comm: kworker/u4:2 Not tainted 4.4.0-rc4-next-20151211-60578-g9379b04 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-Workqueue: deferwq deferred_probe_work_func
-[<c0017248>] (unwind_backtrace) from [<c0012870>] (show_stack+0x10/0x14)
-[<c0012870>] (show_stack) from [<c02392e4>] (dump_stack+0x84/0xd0)
-[<c02392e4>] (dump_stack) from [<c0026b60>] (warn_slowpath_common+0x74/0xb0)
-[<c0026b60>] (warn_slowpath_common) from [<c0026c30>] (warn_slowpath_fmt+0x30/0x40)
-[<c0026c30>] (warn_slowpath_fmt) from [<c00398dc>] (check_flush_dependency+0xc8/0x12c)
-[<c00398dc>] (check_flush_dependency) from [<c00399f8>] (flush_work+0x54/0x11c)
-[<c00399f8>] (flush_work) from [<c00c008c>] (lru_add_drain_all+0x124/0x180)
-[<c00c008c>] (lru_add_drain_all) from [<c00f0ae0>] (migrate_prep+0x8/0x10)
-[<c00f0ae0>] (migrate_prep) from [<c00bac04>] (alloc_contig_range+0xc4/0x2f8)
-[<c00bac04>] (alloc_contig_range) from [<c00f2650>] (cma_alloc+0xd8/0x1a0)
-[<c00f2650>] (cma_alloc) from [<c001c250>] (__alloc_from_contiguous+0x38/0xdc)
-[<c001c250>] (__alloc_from_contiguous) from [<c001cf18>] (__dma_alloc+0x228/0x2ac)
-[<c001cf18>] (__dma_alloc) from [<c001d020>] (arm_dma_alloc+0x48/0x54)
-[<c001d020>] (arm_dma_alloc) from [<c00eceec>] (dma_pool_alloc+0x28c/0x304)
-[<c00eceec>] (dma_pool_alloc) from [<c03f7768>] (ehci_qh_alloc+0x3c/0xb4)
-[<c03f7768>] (ehci_qh_alloc) from [<c03fb954>] (ehci_setup+0x154/0x478)
-[<c03fb954>] (ehci_setup) from [<c03ff69c>] (tegra_ehci_reset+0xc/0x50)
-[<c03ff69c>] (tegra_ehci_reset) from [<c03e3154>] (usb_add_hcd+0x2fc/0x804)
-[<c03e3154>] (usb_add_hcd) from [<c03ff014>] (tegra_ehci_probe+0x3b8/0x504)
-[<c03ff014>] (tegra_ehci_probe) from [<c03322b0>] (platform_drv_probe+0x50/0xac)
-[<c03322b0>] (platform_drv_probe) from [<c0330d10>] (driver_probe_device+0x1d4/0x290)
-[<c0330d10>] (driver_probe_device) from [<c032f30c>] (bus_for_each_drv+0x44/0x8c)
-[<c032f30c>] (bus_for_each_drv) from [<c0330a8c>] (__device_attach+0xa0/0x104)
-[<c0330a8c>] (__device_attach) from [<c0330188>] (bus_probe_device+0x84/0x8c)
-[<c0330188>] (bus_probe_device) from [<c03305bc>] (deferred_probe_work_func+0x60/0x8c)
-[<c03305bc>] (deferred_probe_work_func) from [<c003a804>] (process_one_work+0x120/0x328)
-[<c003a804>] (process_one_work) from [<c003aa70>] (worker_thread+0x2c/0x4b4)
-[<c003aa70>] (worker_thread) from [<c003fbac>] (kthread+0xd8/0xf4)
-[<c003fbac>] (kthread) from [<c000f938>] (ret_from_fork+0x14/0x3c)
----[ end trace b8f898c23980c49f ]---
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
- mmcblk0: p1
+mmc1: new high speed SD card at address b368
 hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
+hub 1-1:1.0: 4 ports detected
 hub 2-0:1.0: 1 port detected
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
+ mmcblk1: p1 p2
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-27 02:57:04 UTC (1419649024)
+rtc-em3027 2-0056: setting system clock to 2014-12-27 02:58:18 UTC (1419649098)
 ALSA device list:
   #0: tegra-trimslice
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
-mmc1: new high speed SD card at address b368
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
- mmcblk1: p1 p2
 devtmpfs: mounted
-Freeing unused kernel memory: 344K (c0974000 - c09ca000)
+Freeing unused kernel memory: 340K (c0975000 - c09ca000)
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-12-13 22:54:26.921305278 -0800
+++ /tmp/foob	2015-12-13 22:54:26.925305274 -0800
@@ -37,10 +37,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ############################################################
-	 3.4 MiB/s
+	 #############################################################
+	 3.3 MiB/s
 done
-Bytes transferred = 6599800 (64b478 hex)
+Bytes transferred = 6610128 (64dcd0 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -51,7 +51,7 @@
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x64b478 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x64dcd0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-rc4-next-20151211-60578-g9379b04 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Dec 10 22:51:56 PST 2015
+Linux version 4.4.0-rc4-next-20151214-61244-g7e8998e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Dec 13 22:48:09 PST 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -67,7 +67,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0ffa980, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1001640, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -80,7 +80,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998268K/2096128K available (8334K kernel code, 1090K rwdata, 3840K rodata, 1020K init, 344K bss, 32324K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998240K/2096128K available (8338K kernel code, 1093K rwdata, 3848K rodata, 1032K init, 344K bss, 32352K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,10 +88,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0dec824   (12179 kB)
-      .init : 0xc0ded000 - 0xc0eec000   (1020 kB)
-      .data : 0xc0eec000 - 0xc0ffc900   (1091 kB)
-       .bss : 0xc0fff000 - 0xc10550a8   ( 345 kB)
+      .text : 0xc0208000 - 0xc0def87c   (12191 kB)
+      .init : 0xc0df0000 - 0xc0ef2000   (1032 kB)
+      .data : 0xc0ef2000 - 0xc10035c0   (1094 kB)
+       .bss : 0xc1006000 - 0xc105c0a8   ( 345 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -275,16 +275,16 @@
 Key type dns_resolver registered
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
+tegra-pcie 3000.pcie-controller: 2x3 configuration
+sys_3v3,vdd_3v3_alw: supplied by vdd_5v_in
+sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
+tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
 mmc1: new high speed MMC card at address 0001
 mmcblk0: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk0boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk0boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk0rpmb: mmc1:0001 SEM16G partition 3 128 KiB
 Primary GPT is invalid, using alternate GPT.
-tegra-pcie 3000.pcie-controller: 2x3 configuration
-sys_3v3,vdd_3v3_alw: supplied by vdd_5v_in
-sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
-tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
  mmcblk0: p1 p2 p3 p4 p5
 tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -339,15 +339,15 @@
 tegra-ehci 7d004000.usb: EHCI Host Controller
 tegra-ehci 7d004000.usb: new USB bus registered, assigned bus number 1
 ------------[ cut here ]------------
-WARNING: CPU: 0 PID: 63 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
+WARNING: CPU: 1 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 0 PID: 63 Comm: kworker/u8:1 Not tainted 4.4.0-rc4-next-20151211-60578-g9379b04 #1
+CPU: 1 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-rc4-next-20151214-61244-g7e8998e #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0218878>] (unwind_backtrace) from [<c0213f10>] (show_stack+0x10/0x14)
-[<c0213f10>] (show_stack) from [<c04863ac>] (dump_stack+0x78/0x94)
-[<c04863ac>] (dump_stack) from [<c024ab28>] (warn_slowpath_common+0x74/0xb0)
+[<c0213f10>] (show_stack) from [<c04863cc>] (dump_stack+0x78/0x94)
+[<c04863cc>] (dump_stack) from [<c024ab28>] (warn_slowpath_common+0x74/0xb0)
 [<c024ab28>] (warn_slowpath_common) from [<c024abf8>] (warn_slowpath_fmt+0x30/0x40)
 [<c024abf8>] (warn_slowpath_fmt) from [<c025ddd4>] (check_flush_dependency+0xc8/0x124)
 [<c025ddd4>] (check_flush_dependency) from [<c025dee8>] (flush_work+0x54/0x144)
@@ -359,22 +359,22 @@
 [<c021e31c>] (__alloc_from_contiguous) from [<c021ff10>] (__dma_alloc+0x21c/0x250)
 [<c021ff10>] (__dma_alloc) from [<c021ffc8>] (arm_dma_alloc+0x48/0x54)
 [<c021ffc8>] (arm_dma_alloc) from [<c0305e78>] (dma_pool_alloc+0x1dc/0x210)
-[<c0305e78>] (dma_pool_alloc) from [<c07a3c30>] (ehci_qh_alloc+0x3c/0xb4)
-[<c07a3c30>] (ehci_qh_alloc) from [<c07a8228>] (ehci_setup+0x154/0x478)
-[<c07a8228>] (ehci_setup) from [<c07ae2e0>] (tegra_ehci_reset+0xc/0x50)
-[<c07ae2e0>] (tegra_ehci_reset) from [<c077e918>] (usb_add_hcd+0x300/0x830)
-[<c077e918>] (usb_add_hcd) from [<c07adc58>] (tegra_ehci_probe+0x3b8/0x504)
-[<c07adc58>] (tegra_ehci_probe) from [<c06446f8>] (platform_drv_probe+0x50/0xac)
-[<c06446f8>] (platform_drv_probe) from [<c064305c>] (driver_probe_device+0x1d4/0x290)
-[<c064305c>] (driver_probe_device) from [<c0641654>] (bus_for_each_drv+0x44/0x8c)
-[<c0641654>] (bus_for_each_drv) from [<c0642dd8>] (__device_attach+0xa0/0x104)
-[<c0642dd8>] (__device_attach) from [<c06424d4>] (bus_probe_device+0x84/0x8c)
-[<c06424d4>] (bus_probe_device) from [<c0642908>] (deferred_probe_work_func+0x60/0x8c)
-[<c0642908>] (deferred_probe_work_func) from [<c025ed30>] (process_one_work+0x134/0x338)
+[<c0305e78>] (dma_pool_alloc) from [<c07a58fc>] (ehci_qh_alloc+0x3c/0xb4)
+[<c07a58fc>] (ehci_qh_alloc) from [<c07a9ef4>] (ehci_setup+0x154/0x478)
+[<c07a9ef4>] (ehci_setup) from [<c07affac>] (tegra_ehci_reset+0xc/0x50)
+[<c07affac>] (tegra_ehci_reset) from [<c07805e4>] (usb_add_hcd+0x300/0x830)
+[<c07805e4>] (usb_add_hcd) from [<c07af924>] (tegra_ehci_probe+0x3b8/0x504)
+[<c07af924>] (tegra_ehci_probe) from [<c0645ae0>] (platform_drv_probe+0x50/0xac)
+[<c0645ae0>] (platform_drv_probe) from [<c0644444>] (driver_probe_device+0x1d4/0x290)
+[<c0644444>] (driver_probe_device) from [<c0642a3c>] (bus_for_each_drv+0x44/0x8c)
+[<c0642a3c>] (bus_for_each_drv) from [<c06441c0>] (__device_attach+0xa0/0x104)
+[<c06441c0>] (__device_attach) from [<c06438bc>] (bus_probe_device+0x84/0x8c)
+[<c06438bc>] (bus_probe_device) from [<c0643cf0>] (deferred_probe_work_func+0x60/0x8c)
+[<c0643cf0>] (deferred_probe_work_func) from [<c025ed30>] (process_one_work+0x134/0x338)
 [<c025ed30>] (process_one_work) from [<c025ef98>] (worker_thread+0x2c/0x4f0)
 [<c025ef98>] (worker_thread) from [<c0264128>] (kthread+0xd8/0xf4)
 [<c0264128>] (kthread) from [<c0210df8>] (ret_from_fork+0x14/0x3c)
----[ end trace c4022478d9a0d461 ]---
+---[ end trace 43bc273d4645e913 ]---
 tegra-ehci 7d004000.usb: irq 343, io mem 0x7d004000
 tegra-ehci 7d004000.usb: USB 2.0 started, EHCI 1.00
 hub 1-0:1.0: USB hub found
@@ -394,7 +394,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1020K (c0ded000 - c0eec000)
+Freeing unused kernel memory: 1032K (c0df0000 - c0ef2000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.978480] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.973465] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2015-12-13 22:54:26.933305267 -0800
+++ /tmp/foob	2015-12-13 22:54:26.933305267 -0800
@@ -39,18 +39,18 @@
 	 ################################
 	 3.4 MiB/s
 done
-Bytes transferred = 5231152 (4fd230 hex)
+Bytes transferred = 5231072 (4fd1e0 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4fd230 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4fd1e0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.4.0-rc4-next-20151211-60578-g9379b04 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Dec 10 22:52:12 PST 2015
+Linux version 4.4.0-rc4-next-20151214-61244-g7e8998e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Dec 13 22:48:25 PST 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -70,7 +70,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0a24d00, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0a24d40, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -83,7 +83,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002168K/2096128K available (7179K kernel code, 370K rwdata, 2464K rodata, 344K init, 367K bss, 28424K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2002168K/2096128K available (7183K kernel code, 370K rwdata, 2464K rodata, 340K init, 367K bss, 28424K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -91,9 +91,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09732f4   (9645 kB)
-      .init : 0xc0974000 - 0xc09ca000   ( 344 kB)
-      .data : 0xc09ca000 - 0xc0a26820   ( 371 kB)
+      .text : 0xc0008000 - 0xc09742d8   (9649 kB)
+      .init : 0xc0975000 - 0xc09ca000   ( 340 kB)
+      .data : 0xc09ca000 - 0xc0a26860   ( 371 kB)
        .bss : 0xc0a29000 - 0xc0a84d90   ( 368 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -346,12 +346,12 @@
 WARNING: CPU: 0 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-rc4-next-20151211-60578-g9379b04 #1
+CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-rc4-next-20151214-61244-g7e8998e #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0017248>] (unwind_backtrace) from [<c0012870>] (show_stack+0x10/0x14)
-[<c0012870>] (show_stack) from [<c02392e4>] (dump_stack+0x84/0xd0)
-[<c02392e4>] (dump_stack) from [<c0026b60>] (warn_slowpath_common+0x74/0xb0)
+[<c0012870>] (show_stack) from [<c0239324>] (dump_stack+0x84/0xd0)
+[<c0239324>] (dump_stack) from [<c0026b60>] (warn_slowpath_common+0x74/0xb0)
 [<c0026b60>] (warn_slowpath_common) from [<c0026c30>] (warn_slowpath_fmt+0x30/0x40)
 [<c0026c30>] (warn_slowpath_fmt) from [<c00398dc>] (check_flush_dependency+0xc8/0x12c)
 [<c00398dc>] (check_flush_dependency) from [<c00399f8>] (flush_work+0x54/0x11c)
@@ -363,22 +363,22 @@
 [<c001c250>] (__alloc_from_contiguous) from [<c001cf18>] (__dma_alloc+0x228/0x2ac)
 [<c001cf18>] (__dma_alloc) from [<c001d020>] (arm_dma_alloc+0x48/0x54)
 [<c001d020>] (arm_dma_alloc) from [<c00eceec>] (dma_pool_alloc+0x28c/0x304)
-[<c00eceec>] (dma_pool_alloc) from [<c03f7768>] (ehci_qh_alloc+0x3c/0xb4)
-[<c03f7768>] (ehci_qh_alloc) from [<c03fb954>] (ehci_setup+0x154/0x478)
-[<c03fb954>] (ehci_setup) from [<c03ff69c>] (tegra_ehci_reset+0xc/0x50)
-[<c03ff69c>] (tegra_ehci_reset) from [<c03e3154>] (usb_add_hcd+0x2fc/0x804)
-[<c03e3154>] (usb_add_hcd) from [<c03ff014>] (tegra_ehci_probe+0x3b8/0x504)
-[<c03ff014>] (tegra_ehci_probe) from [<c03322b0>] (platform_drv_probe+0x50/0xac)
-[<c03322b0>] (platform_drv_probe) from [<c0330d10>] (driver_probe_device+0x1d4/0x290)
-[<c0330d10>] (driver_probe_device) from [<c032f30c>] (bus_for_each_drv+0x44/0x8c)
-[<c032f30c>] (bus_for_each_drv) from [<c0330a8c>] (__device_attach+0xa0/0x104)
-[<c0330a8c>] (__device_attach) from [<c0330188>] (bus_probe_device+0x84/0x8c)
-[<c0330188>] (bus_probe_device) from [<c03305bc>] (deferred_probe_work_func+0x60/0x8c)
-[<c03305bc>] (deferred_probe_work_func) from [<c003a804>] (process_one_work+0x120/0x328)
+[<c00eceec>] (dma_pool_alloc) from [<c03f7d24>] (ehci_qh_alloc+0x3c/0xb4)
+[<c03f7d24>] (ehci_qh_alloc) from [<c03fbf10>] (ehci_setup+0x154/0x478)
+[<c03fbf10>] (ehci_setup) from [<c03ffc58>] (tegra_ehci_reset+0xc/0x50)
+[<c03ffc58>] (tegra_ehci_reset) from [<c03e3710>] (usb_add_hcd+0x2fc/0x804)
+[<c03e3710>] (usb_add_hcd) from [<c03ff5d0>] (tegra_ehci_probe+0x3b8/0x504)
+[<c03ff5d0>] (tegra_ehci_probe) from [<c0332578>] (platform_drv_probe+0x50/0xac)
+[<c0332578>] (platform_drv_probe) from [<c0330fd8>] (driver_probe_device+0x1d4/0x290)
+[<c0330fd8>] (driver_probe_device) from [<c032f5d4>] (bus_for_each_drv+0x44/0x8c)
+[<c032f5d4>] (bus_for_each_drv) from [<c0330d54>] (__device_attach+0xa0/0x104)
+[<c0330d54>] (__device_attach) from [<c0330450>] (bus_probe_device+0x84/0x8c)
+[<c0330450>] (bus_probe_device) from [<c0330884>] (deferred_probe_work_func+0x60/0x8c)
+[<c0330884>] (deferred_probe_work_func) from [<c003a804>] (process_one_work+0x120/0x328)
 [<c003a804>] (process_one_work) from [<c003aa70>] (worker_thread+0x2c/0x4b4)
 [<c003aa70>] (worker_thread) from [<c003fbac>] (kthread+0xd8/0xf4)
 [<c003fbac>] (kthread) from [<c000f938>] (ret_from_fork+0x14/0x3c)
----[ end trace 4438c1d815908d44 ]---
+---[ end trace da116bcedf383893 ]---
 tegra-ehci 7d004000.usb: irq 343, io mem 0x7d004000
 tegra-ehci 7d004000.usb: USB 2.0 started, EHCI 1.00
 hub 1-0:1.0: USB hub found
@@ -400,7 +400,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 344K (c0974000 - c09ca000)
+Freeing unused kernel memory: 340K (c0975000 - c09ca000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.400555] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.420544] usb 2-1: new high-speed USB device number 2 using tegra-ehci
