Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2017-02-20 20:59:20.765977752 -0800
+++ /tmp/foob	2017-02-20 20:59:20.765977752 -0800
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc8-next-20170220-178568-gb349d1b (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Feb 19 21:07:20 PST 2017
+Linux version 4.10.0-next-20170221-178837-g3d4ffb9 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Feb 20 20:53:03 PST 2017
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -114,7 +114,7 @@
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 kvm [1]: HYP mode not available
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.692:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.712:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -180,7 +180,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2017-02-20 05:08:00 UTC (1487567280)
+rtc-pl031 9010000.pl031: setting system clock to 2017-02-21 04:53:45 UTC (1487652825)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-02-20 20:59:20.785977830 -0800
+++ /tmp/foob	2017-02-20 20:59:20.785977830 -0800
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 ###########################################################
-	 3.3 MiB/s
+	 2.8 MiB/s
 done
-Bytes transferred = 7543624 (731b48 hex)
+Bytes transferred = 7542392 (731678 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.8 MiB/s
+	 2.6 MiB/s
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x731b48 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x731678 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc8-next-20170220-154135-gb349d1b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Feb 19 21:03:52 PST 2017
+Linux version 4.10.0-next-20170221-154404-g3d4ffb9 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Feb 20 20:50:25 PST 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -71,7 +71,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c1538500, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c15384c0, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -84,7 +84,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993772K/2096128K available (10240K kernel code, 1255K rwdata, 4208K rodata, 2048K init, 536K bss, 36820K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993772K/2096128K available (10240K kernel code, 1255K rwdata, 4212K rodata, 2048K init, 536K bss, 36820K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -94,7 +94,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1539e20   (1256 kB)
+      .data : 0xc1400000 - 0xc1539de0   (1256 kB)
        .bss : 0xc153b000 - 0xc15c12b8   ( 537 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -146,6 +146,7 @@
 hw-breakpoint: maximum watchpoint size is 8 bytes.
 Serial: AMBA PL011 UART driver
 tegra-mc 70019000.memory-controller: no memory timings for RAM code 3 registered
+AT91: Could not find identification node
 dvdd_ts: Failed to request enable GPIO61: -517
 reg-fixed-voltage regulators:regulator@1: Failed to register regulator: -517
 usb1_vbus: Failed to request enable GPIO108: -517
@@ -317,8 +318,8 @@
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
-Console: switching to colour frame buffer device 240x75
  mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
+Console: switching to colour frame buffer device 240x75
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
 tegra-ehci 7d008000.usb: EHCI Host Controller
@@ -328,7 +329,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-12-14 16:36:00 UTC (976811760)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-12-15 16:24:54 UTC (976897494)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -340,4 +341,4 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.509359] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.519383] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2017-02-20 20:59:20.793977862 -0800
+++ /tmp/foob	2017-02-20 20:59:20.793977862 -0800
@@ -39,7 +39,7 @@
 	 #######################################################
 	 3.1 MiB/s
 done
-Bytes transferred = 5566304 (54ef60 hex)
+Bytes transferred = 5566800 (54f150 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x54ef60 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x54f150 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc8-next-20170220-154135-gb349d1b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Feb 19 21:03:56 PST 2017
+Linux version 4.10.0-next-20170221-154404-g3d4ffb9 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Feb 20 20:49:58 PST 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -321,7 +321,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-12-14 16:38:42 UTC (976811922)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-12-15 16:22:15 UTC (976897335)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -335,4 +335,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.747254] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.747155] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-02-20 20:59:20.797977877 -0800
+++ /tmp/foob	2017-02-20 20:59:20.801977893 -0800
@@ -34,20 +34,20 @@
 	 #################################################################
 	 #################################################################
 	 ###########################################################
-	 4 MiB/s
+	 3.9 MiB/s
 done
-Bytes transferred = 7543624 (731b48 hex)
+Bytes transferred = 7542392 (731678 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.6 MiB/s
+	 3.8 MiB/s
 done
 Bytes transferred = 67133 (1063d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x731b48 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x731678 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301363c
@@ -55,7 +55,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc8-next-20170220-154135-gb349d1b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Feb 19 21:03:52 PST 2017
+Linux version 4.10.0-next-20170221-154404-g3d4ffb9 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Feb 20 20:50:25 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -66,7 +66,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c1538500, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c15384c0, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -79,7 +79,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993652K/2096128K available (10240K kernel code, 1255K rwdata, 4208K rodata, 2048K init, 536K bss, 36940K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993652K/2096128K available (10240K kernel code, 1255K rwdata, 4212K rodata, 2048K init, 536K bss, 36940K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -89,7 +89,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1539e20   (1256 kB)
+      .data : 0xc1400000 - 0xc1539de0   (1256 kB)
        .bss : 0xc153b000 - 0xc15c12b8   ( 537 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -140,6 +140,7 @@
 hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 8 bytes.
 Serial: AMBA PL011 UART driver
+AT91: Could not find identification node
 +5V_SYS: supplied by +VDD_MUX
 +3.3V_SYS: supplied by +VDD_MUX
 +USB0_VBUS_SW: Failed to request enable GPIO108: -517
@@ -317,6 +318,7 @@
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
 tegra-pcie 1003000.pcie-controller: Failed to get supply 'avddio-pex': -517
+mmc0: new high speed MMC card at address 0001
 tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
 as3722-sd2: supplied by +5V_SYS
@@ -336,16 +338,10 @@
 +3.3V_RUN_TOUCH: Bringing 0uV into 2800000-2800000uV
 as3722-ldo10: supplied by +5V_SYS
 +2.8V_RUN_CAM_AF: Bringing 0uV into 2800000-2800000uV
-mmc0: new high speed MMC card at address 0001
 as3722-ldo11: supplied by +3.3V_RUN
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 +1.8V_RUN_VPP_FUSE: Bringing 0uV into 1800000-1800000uV
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
 tegra-ahci 70027000.sata: AHCI 0001.0301 32 slots 2 ports 3 Gbps 0x1 impl platform mode
 tegra-ahci 70027000.sata: flags: 64bit ncq sntf pm led pio slum part sadm sds apst 
- mmcblk0: p1
 scsi host0: tegra-ahci
 scsi host1: tegra-ahci
 ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 345
@@ -381,7 +377,6 @@
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x52 may corrupt adjacent RW1C bits
 pci 0000:00:02.0: PME# supported from D0 D1 D2 D3hot D3cold
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4c may corrupt adjacent RW1C bits
-ata1: SATA link down (SStatus 0 SControl 300)
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x3e may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x3e may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x4 may corrupt adjacent RW1C bits
@@ -392,7 +387,6 @@
 pci 0000:01:00.0: [8086:4232] type 00 class 0x028000
 pci 0000:01:00.0: reg 0x10: [mem 0x00000000-0x00001fff 64bit]
 pci 0000:01:00.0: PME# supported from D0 D3hot D3cold
-mmc1: new high speed SDHC card at address aaaa
 PCI: bus1: Fast back to back transfers disabled
 pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
 pci 0000:02:00.0: [10ec:8168] type 00 class 0x020000
@@ -401,8 +395,6 @@
 pci 0000:02:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
 pci 0000:02:00.0: supports D1 D2
 pci 0000:02:00.0: PME# supported from D0 D1 D2 D3hot D3cold
-mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
- mmcblk1: p1 p2 p3
 PCI: bus2: Fast back to back transfers disabled
 pci_bus 0000:02: busn_res: [bus 02-ff] end is updated to 02
 pci 0000:00:01.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
@@ -427,20 +419,29 @@
 pcieport 0000:00:02.0: Signaling PME with IRQ 394
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
-r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0b54000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 395
+r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0b41000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 395
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
-tegra-hdmi 54280000.hdmi: cannot set audio to 48000 Hz at 297000000 Hz pixel clock
-Console: switching to colour frame buffer device 512x135
+[drm] Cannot find any crtc or sizes - going 1024x768
+ata1: SATA link down (SStatus 0 SControl 300)
+mmc1: new high speed SDHC card at address aaaa
+mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
+ mmcblk1: p1 p2 p3
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+ mmcblk0: p1
+Console: switching to colour frame buffer device 128x48
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
 of: dev_pm_opp_of_cpumask_add_table: couldn't find opp table for cpu:0, -19
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:13 UTC (946684813)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:38 UTC (946684958)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
@@ -452,7 +453,7 @@
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:1.
+VFS: Mounted root (ext3 filesystem) on device 179:17.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2017-02-20 20:59:20.809977924 -0800
+++ /tmp/foob	2017-02-20 20:59:20.809977924 -0800
@@ -34,7 +34,7 @@
 	 #######################################################
 	 3.9 MiB/s
 done
-Bytes transferred = 5566304 (54ef60 hex)
+Bytes transferred = 5566800 (54f150 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -45,7 +45,7 @@
 done
 Bytes transferred = 67133 (1063d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x54ef60 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x54f150 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301363c
@@ -54,7 +54,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc8-next-20170220-154135-gb349d1b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Feb 19 21:03:56 PST 2017
+Linux version 4.10.0-next-20170221-154404-g3d4ffb9 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Feb 20 20:49:58 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -446,7 +446,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:01:35 UTC (946684895)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:13 UTC (946684813)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-02-20 20:59:20.817977956 -0800
+++ /tmp/foob	2017-02-20 20:59:20.821977971 -0800
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -42,20 +42,20 @@
 	 #################################################################
 	 #################################################################
 	 ###########################################################
-	 3.6 MiB/s
+	 3.5 MiB/s
 done
-Bytes transferred = 7543624 (731b48 hex)
+Bytes transferred = 7542392 (731678 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.2 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 68155 (10a3b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x731b48 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x731678 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a3a
@@ -63,7 +63,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc8-next-20170220-154135-gb349d1b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Feb 19 21:03:52 PST 2017
+Linux version 4.10.0-next-20170221-154404-g3d4ffb9 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Feb 20 20:50:25 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -75,7 +75,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c1538500, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c15384c0, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -88,7 +88,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1994668K/2097148K available (10240K kernel code, 1255K rwdata, 4208K rodata, 2048K init, 536K bss, 36944K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1994668K/2097148K available (10240K kernel code, 1255K rwdata, 4212K rodata, 2048K init, 536K bss, 36944K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -98,7 +98,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1539e20   (1256 kB)
+      .data : 0xc1400000 - 0xc1539de0   (1256 kB)
        .bss : 0xc153b000 - 0xc15c12b8   ( 537 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -150,6 +150,7 @@
 hw-breakpoint: maximum watchpoint size is 8 bytes.
 Serial: AMBA PL011 UART driver
 tegra-mc 70019000.memory-controller: no memory timings for RAM code 0 registered
+AT91: Could not find identification node
 +5V_SYS: supplied by +VDD_MUX
 +3.3V_SYS: supplied by +VDD_MUX
 +VDD_LED: Failed to request enable GPIO122: -517
@@ -294,9 +295,7 @@
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc0: Unknown controller version (3). You may experience problems.
 mmc0: Invalid maximum block size, assuming 512 bytes
-elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
 mmc0: SDHCI controller on 700b0600.sdhci [700b0600.sdhci] using ADMA
-input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 ledtrig-cpu: registered to indicate activity on CPUs
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
@@ -314,6 +313,8 @@
 Registering SWP/SWPB emulation handler
 +3.3V_RUN: supplied by +3.3V_SYS
 +VDD_LED: supplied by +VDD_MUX
+elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
+input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 +3.3V_AVDD_HDMI_AP_GATED: supplied by +3.3V_RUN
 +1.8V_RUN_CAM: supplied by +3.3V_RUN
 +VDDIO_SDMMC3: supplied by +3.3V_RUN
@@ -322,15 +323,15 @@
 +5V_USB_HS: supplied by +5V_SYS
 +5V_USB_SS: supplied by +5V_SYS
 +3.3V_PANEL: supplied by +3.3V_RUN
-+3.3V_LP0: supplied by +3.3V_SYS
-+1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
-elan_i2c 1-0015: failed to read report data: -121
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
 mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
 mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
++3.3V_LP0: supplied by +3.3V_SYS
++1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
+elan_i2c 1-0015: failed to read report data: -121
 +5V_HDMI_CON: supplied by +5V_SYS
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
  usb3-1: usb3-1 supply vbus not found, using dummy regulator
@@ -340,10 +341,10 @@
 mmc1: Unknown controller version (3). You may experience problems.
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: Switching to 3.3V signalling voltage failed
-elan_i2c 1-0015: invalid report id data (ff)
 mmc1: SDHCI controller on 700b0000.sdhci [700b0000.sdhci] using ADMA
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
+elan_i2c 1-0015: invalid report id data (ff)
 mmc2: Unknown controller version (3). You may experience problems.
 mmc2: Invalid maximum block size, assuming 512 bytes
 mmc1: new high speed SDIO card at address 0001
@@ -362,7 +363,7 @@
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:24 UTC (946684824)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:23 UTC (946684823)
 +1.8V_RUN_CAM: disabling
 +3.3V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2017-02-20 20:59:20.825977987 -0800
+++ /tmp/foob	2017-02-20 20:59:20.825977987 -0800
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -40,9 +40,9 @@
 	 #################################################################
 	 #################################################################
 	 #######################################################
-	 3.5 MiB/s
+	 3.6 MiB/s
 done
-Bytes transferred = 5566304 (54ef60 hex)
+Bytes transferred = 5566800 (54f150 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -53,7 +53,7 @@
 done
 Bytes transferred = 68155 (10a3b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x54ef60 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x54f150 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a3a
@@ -62,7 +62,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc8-next-20170220-154135-gb349d1b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Feb 19 21:03:56 PST 2017
+Linux version 4.10.0-next-20170221-154404-g3d4ffb9 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Feb 20 20:49:58 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -370,7 +370,7 @@
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input4
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:22 UTC (946684822)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:23 UTC (946684823)
 +1.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
 +1.05V_RUN_CAM_REAR: disabling
@@ -391,4 +391,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.910563] usb 1-3: new high-speed USB device number 3 using tegra-xusb
+root@(none):/# [    2.900383] usb 1-3: new high-speed USB device number 3 using tegra-xusb
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2017-02-20 20:59:20.833978018 -0800
+++ /tmp/foob	2017-02-20 20:59:20.833978018 -0800
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1800617 us
+Bootloader-Cpu Init at (time stamp): 1800688 us
 
 Pinmux changes applied in kernel way
 
@@ -62,20 +62,20 @@
 Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1800591  us
-Bootloader (AosInit) time: 733463 us
+Miscellaneous avp time: 1800662  us
+Bootloader (AosInit) time: 734280 us
 Bootloader Main Init time: 180253 us
-BlockDevMgrInit time: 157519 us
+BlockDevMgrInit time: 157530 us
 PartMgrInit time: 8 us
 StorMgrInit time: 30 us
-FsMgrInit time: 2 us
-Partition Table load: 77116 us 
-Display init time: 574501 us
-Kernel read time: 377 msec
+FsMgrInit time: 3 us
+Partition Table load: 76673 us 
+Display init time: 574827 us
+Kernel read time: 376 msec
 Load Kernel Size:   16304128 Bytes
-Bl Kernel ReadSpeed: 43247000 Bytes/sec
-Pre-Kernel Jump time: 1741255 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5844276 us
+Bl Kernel ReadSpeed: 43362000 Bytes/sec
+Pre-Kernel Jump time: 1741244 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5887139 us
 Booting Linux on physical CPU 0x0
 Linux version 4.8.0-239303-g3477d16 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Fri Oct 7 01:12:37 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
@@ -184,7 +184,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(1.131:1): initialized
+audit: type=2000 audit(1.175:1): initialized
 workingset: timestamp_bits=46 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -251,8 +251,8 @@
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 +VDD_LED: supplied by +VDD_MUX
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 +5V_USB_HS: supplied by +5V_SYS
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-02-20 20:59:20.841978050 -0800
+++ /tmp/foob	2017-02-20 20:59:20.841978050 -0800
@@ -47,9 +47,9 @@
 	 #################################################################
 	 #################################################################
 	 ###########################################################
-	 4.1 MiB/s
+	 4 MiB/s
 done
-Bytes transferred = 7543624 (731b48 hex)
+Bytes transferred = 7542392 (731678 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -60,7 +60,7 @@
 done
 Bytes transferred = 21426 (53b2 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x731b48 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x731678 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103b1
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc8-next-20170220-154135-gb349d1b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Feb 19 21:03:52 PST 2017
+Linux version 4.10.0-next-20170221-154404-g3d4ffb9 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Feb 20 20:50:25 PST 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -78,7 +78,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c1538500, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c15384c0, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -91,7 +91,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 954592K/1048576K available (10240K kernel code, 1255K rwdata, 4208K rodata, 2048K init, 536K bss, 28448K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 954592K/1048576K available (10240K kernel code, 1255K rwdata, 4212K rodata, 2048K init, 536K bss, 28448K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -101,7 +101,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1539e20   (1256 kB)
+      .data : 0xc1400000 - 0xc1539de0   (1256 kB)
        .bss : 0xc153b000 - 0xc15c12b8   ( 537 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
@@ -149,6 +149,7 @@
 hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 4 bytes.
 Serial: AMBA PL011 UART driver
+AT91: Could not find identification node
 usb1_vbus: Failed to request enable GPIO170: -517
 reg-fixed-voltage regulators:regulator@2: Failed to register regulator: -517
 vgaarb: loaded
@@ -335,18 +336,18 @@
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 mmc0: host does not support reading read-only switch, assuming write-enable
-mmc1: new high speed SD card at address b368
 mmc0: new high speed SDHC card at address b368
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
+ mmcblk0: p1
+mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
  mmcblk1: p1 p2
- mmcblk0: p1
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2000-11-01 00:00:16 UTC (973036816)
+rtc-em3027 2-0056: setting system clock to 2000-11-01 00:00:17 UTC (973036817)
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
@@ -355,7 +356,7 @@
 random: fast init done
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:17.
+VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2017-02-20 20:59:20.845978065 -0800
+++ /tmp/foob	2017-02-20 20:59:20.849978081 -0800
@@ -47,18 +47,18 @@
 	 #######################################################
 	 4 MiB/s
 done
-Bytes transferred = 5566304 (54ef60 hex)
+Bytes transferred = 5566800 (54f150 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 4.1 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21426 (53b2 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x54ef60 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x54f150 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103b1
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc8-next-20170220-154135-gb349d1b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Feb 19 21:03:56 PST 2017
+Linux version 4.10.0-next-20170221-154404-g3d4ffb9 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Feb 20 20:49:58 PST 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -314,22 +314,22 @@
 tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
 NET: Registered protocol family 10
 Segment Routing with IPv6
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
 mmc0: host does not support reading read-only switch, assuming write-enable
-mmc1: new high speed SD card at address b368
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 mmc0: new high speed SDHC card at address b368
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
- mmcblk0: p1
- mmcblk1: p1 p2
 mip6: Mobile IPv6
+ mmcblk0: p1
 sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
 NET: Registered protocol family 17
 NET: Registered protocol family 15
+mmc1: new high speed SD card at address b368
 can: controller area network core (rev 20120528 abi 9)
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 NET: Registered protocol family 29
 can: raw protocol (rev 20120528)
 can: broadcast manager protocol (rev 20161123 t)
+ mmcblk1: p1 p2
 can: netlink gateway (rev 20130117) max_hops=1
 Bluetooth: RFCOMM socket layer initialized
 Bluetooth: RFCOMM ver 1.11
@@ -343,13 +343,13 @@
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
+hub 1-1:1.0: USB hub found
+hub 1-1:1.0: 4 ports detected
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
-hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2000-11-01 00:00:16 UTC (973036816)
+rtc-em3027 2-0056: setting system clock to 2000-11-01 00:00:15 UTC (973036815)
 ALSA device list:
   #0: tegra-trimslice
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
@@ -358,7 +358,7 @@
 asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:17.
+VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2017-02-20 20:59:20.853978097 -0800
+++ /tmp/foob	2017-02-20 20:59:20.857978112 -0800
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 758875 us
-Basic modules init took 348771 us
-Sec Bootdevice Read Time = 203 ms, Read Size = 8459 KB
-Next stage binary read took 15145 us
-Carveout took 252717 us
-CPU initialization took 92329 us
-Total time taken by TegraBoot 708962 us
+Tegraboot started after 755725 us
+Basic modules init took 348506 us
+Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
+Next stage binary read took 14365 us
+Carveout took 252735 us
+CPU initialization took 92311 us
+Total time taken by TegraBoot 707917 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.598] LPDDR4 Training: Number of tables = 10
+64b[0001.594] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -121,7 +121,7 @@
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
 44882 bytes read in 460 ms (94.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 496 ms (3.1 MiB/s)
+1627787 bytes read in 495 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc8-next-20170220-178568-gb349d1b (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Feb 19 21:07:20 PST 2017
+Linux version 4.10.0-next-20170221-178837-g3d4ffb9 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Feb 20 20:53:03 PST 2017
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -357,6 +357,6 @@
 devtmpfs: mounted
 Freeing unused kernel memory: 1088K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
-bash: no job contr[    1.365368] random: fast init done
+bash: no job contr[    1.313913] random: fast init done
 ol in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2017-02-20 20:59:20.861978128 -0800
+++ /tmp/foob	2017-02-20 20:59:20.865978144 -0800
@@ -1,56 +1,4 @@
-2017
-Boot CPU: AArch64 Processor [411fd071]
-efi: Getting EFI parameters from FDT:
-efi: UEFI not found.
-cma: Reserved 16 MiB at 0x00000000f5400000
-earlycon: uart0 at MMIO 0x0000000070006000 (options '115200n8')
-bootconsole [uart0] enabled
-NUMA: No NUMA configuration found
-NUMA: Faking a node at [mem 0x0000000000000000-0x000000013fffffff]
-NUMA: Adding memblock [0x80000000 - 0xf65fffff] on node 0
-NUMA: Adding memblock [0x100000000 - 0x13fffffff] on node 0
-NUMA: Initmem setup node 0 [mem 0x80000000-0x13fffffff]
-NUMA: NODE_DATA [mem 0x13ffa1400-0x13ffa2eff]
-Zone ranges:
-  DMA      [mem 0x0000000080000000-0x00000000ffffffff]
-  Normal   [mem 0x0000000100000000-0x000000013fffffff]
-Movable zone start for each node
-Early memory node ranges
-  node   0: [mem 0x0000000080000000-0x00000000f65fffff]
-  node   0: [mem 0x0000000100000000-0x000000013fffffff]
-Initmem setup node 0 [mem 0x0000000080000000-0x000000013fffffff]
-psci: probing for conduit method from DT.
-psci: PSCIv1.0 detected in firmware.
-psci: Using standard PSCI v0.2 function IDs
-psci: MIGRATE_INFO_UP_CPU reported invalid physical ID (0x80000000)
-percpu: Embedded 23 pages/cpu @ffff8000bff42000 s56960 r8192 d29056 u94208
-Detected PIPT I-cache on CPU0
-CPU features: enabling workaround for ARM erratum 832075
-CPU features: enabling workaround for ARM erratum 834220
-Built 1 zonelists in Node order, mobility grouping on.  Total pages: 735336
-Policy zone: Normal
-Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
-PID hash table entries: 4096 (order: 3, 32768 bytes)
-software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2837900K/2988032K available (8700K kernel code, 964K rwdata, 4200K rodata, 1088K init, 397K bss, 133748K reserved, 16384K cma-reserved)
-Virtual kernel memory layout:
-    modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
-    vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff000008900000   (  8704 KB)
-    .rodata : 0xffff000008900000 - 0xffff000008d20000   (  4224 KB)
-      .init : 0xffff000008d20000 - 0xffff000008e30000   (  1088 KB)
-      .data : 0xffff000008e30000 - 0xffff000008f21200   (   965 KB)
-       .bss : 0xffff000008f21200 - 0xffff000008f84638   (   398 KB)
-    fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
-    PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
-    vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
-              0xffff7e0000000000 - 0xffff7e0003000000   (    48 MB actual)
-    memory  : 0xffff800000000000 - 0xffff8000c0000000   (  3072 MB)
-SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
-Preemptible hierarchical RCU implementation.
-	Build-time adjustment of leaf fanout to 64.
-	RCU restricting CPUs from NR_CPUS=64 to nr_cpu_ids=4.
-RCU: Adjusting geometry for rcu_fanout_leaf=64, nr_cpu_ids=4
+ometry for rcu_fanout_leaf=64, nr_cpu_ids=4
 NR_IRQS:64 nr_irqs:64 0
 GIC: Using split EOI/Deactivate mode
 /interrupt-controller@60004000: 192 interrupts forwarded to /interrupt-controller@50041000
@@ -69,7 +17,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x59c8: expected 0x2004f400
+boot misc3 0x5ae8: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -82,7 +30,7 @@
 console [tty0] enabled
 bootconsole [uart0] disabled
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc8-next-20170220-178568-gb349d1b (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Feb 19 21:07:20 PST 2017
+Linux version 4.10.0-next-20170221-178837-g3d4ffb9 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Feb 20 20:53:03 PST 2017
 Boot CPU: AArch64 Processor [411fd071]
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
@@ -153,7 +101,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x59c8: expected 0x2004f400
+boot misc3 0x5ae8: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -243,7 +191,7 @@
 kvm [1]: vgic interrupt IRQ1
 kvm [1]: virtual timer IRQ4
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.327:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.323:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -333,9 +281,11 @@
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 BGND3R 29.1 GiB 
 mmcblk0boot0: mmc0:0001 BGND3R partition 1 4.00 MiB
+max77686-rtc max77620-rtc: setting system clock to 2000-12-02 10:20:34 UTC (975752434)
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
-max77686-rtc max77620-rtc: setting system clock to 2000-12-01 10:34:50 UTC (975666890)
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
+tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 PP1200_AVDD: disabling
 PP1200_RCAM: disabling
 PP2800L_RCAM: disabling
@@ -343,8 +293,6 @@
 PP_LDO6: disabling
 PPLCD_VDD: disabling
 PP1800_LCDIO: disabling
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
-tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 PP1800_CAM: disabling
 ALSA device list:
   No soundcards found.
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-02-20 20:59:20.869978159 -0800
+++ /tmp/foob	2017-02-20 20:59:20.869978159 -0800
@@ -41,7 +41,7 @@
 	 ###########################################################
 	 3.3 MiB/s
 done
-Bytes transferred = 7543624 (731b48 hex)
+Bytes transferred = 7542392 (731678 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -52,7 +52,7 @@
 done
 Bytes transferred = 51804 (ca5c hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x731b48 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x731678 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa5b ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc8-next-20170220-154135-gb349d1b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Feb 19 21:03:52 PST 2017
+Linux version 4.10.0-next-20170221-154404-g3d4ffb9 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Feb 20 20:50:25 PST 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -70,7 +70,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c1538500, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c15384c0, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -83,7 +83,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993692K/2096128K available (10240K kernel code, 1255K rwdata, 4208K rodata, 2048K init, 536K bss, 36900K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993692K/2096128K available (10240K kernel code, 1255K rwdata, 4212K rodata, 2048K init, 536K bss, 36900K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -93,7 +93,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1539e20   (1256 kB)
+      .data : 0xc1400000 - 0xc1539de0   (1256 kB)
        .bss : 0xc153b000 - 0xc15c12b8   ( 537 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -144,6 +144,7 @@
 hw-breakpoint: maximum watchpoint size is 4 bytes.
 Serial: AMBA PL011 UART driver
 tegra-mc 7000f000.memory-controller: no memory timings for RAM code 3 registered
+AT91: Could not find identification node
 vdd_5v_sata: Failed to request enable GPIO30: -517
 reg-fixed-voltage regulators:regulator@3: Failed to register regulator: -517
 usb1_vbus: Failed to request enable GPIO238: -517
@@ -305,6 +306,8 @@
 sys_3v3,vdd_3v3_alw: supplied by vdd_5v_in
 sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
 +VDD_5V_HDMI: supplied by sys_3v3,vdd_3v3_alw
+tegra-pcie 3000.pcie-controller: 2x3 configuration
+tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
@@ -313,11 +316,9 @@
 Primary GPT is invalid, using alternate GPT.
  mmcblk1: p1 p2 p3 p4 p5
 mmc0: new ultra high speed DDR50 SDHC card at address aaaa
+tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
  mmcblk0: p1 p2 p3
-tegra-pcie 3000.pcie-controller: 2x3 configuration
-tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
-tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -395,5 +396,5 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.133962] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.083601] usb 2-1: new high-speed USB device number 2 using tegra-ehci
 asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0f:67:60
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2017-02-20 20:59:20.877978191 -0800
+++ /tmp/foob	2017-02-20 20:59:20.877978191 -0800
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 #######################################################
-	 3.3 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 5566304 (54ef60 hex)
+Bytes transferred = 5566800 (54f150 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.9 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 51804 (ca5c hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x54ef60 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x54f150 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa5b ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc8-next-20170220-154135-gb349d1b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Feb 19 21:03:56 PST 2017
+Linux version 4.10.0-next-20170221-154404-g3d4ffb9 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Feb 20 20:49:58 PST 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -390,4 +390,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.380142] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.380052] usb 2-1: new high-speed USB device number 2 using tegra-ehci
