Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2015-11-04 22:23:47.987894196 -0800
+++ /tmp/foob	2015-11-04 22:23:47.991894193 -0800
@@ -1,6 +1,6 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
-Linux version 4.3.0-next-20151104+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Nov 3 23:20:09 PST 2015
+Linux version 4.3.0-next-20151105+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Nov 4 22:19:10 PST 2015
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -19,7 +19,7 @@
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
 software IO TLB [mem 0x79c00000-0x7dc00000] (64MB) mapped at [ffffffc039c00000-ffffffc03dbfffff]
-Memory: 938852K/1048576K available (5554K kernel code, 491K rwdata, 2336K rodata, 544K init, 209K bss, 93340K reserved, 16384K cma-reserved)
+Memory: 938832K/1048576K available (5554K kernel code, 504K rwdata, 2340K rodata, 544K init, 211K bss, 93360K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     vmalloc : 0xffffff8000000000 - 0xffffffbdbfff0000   (   246 GB)
     vmemmap : 0xffffffbdc0000000 - 0xffffffbfc0000000   (     8 GB maximum)
@@ -28,9 +28,9 @@
     PCI I/O : 0xffffffbffae00000 - 0xffffffbffbe00000   (    16 MB)
     modules : 0xffffffbffc000000 - 0xffffffc000000000   (    64 MB)
     memory  : 0xffffffc000000000 - 0xffffffc040000000   (  1024 MB)
-      .init : 0xffffffc000837000 - 0xffffffc0008bf000   (   544 KB)
-      .text : 0xffffffc000080000 - 0xffffffc0008369f4   (  7899 KB)
-      .data : 0xffffffc0008d0000 - 0xffffffc00094ac00   (   491 KB)
+      .init : 0xffffffc000838000 - 0xffffffc0008c0000   (   544 KB)
+      .text : 0xffffffc000080000 - 0xffffffc0008379f4   (  7903 KB)
+      .data : 0xffffffc0008d1000 - 0xffffffc00094f200   (   505 KB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 64.
@@ -61,7 +61,7 @@
 NET: Registered protocol family 16
 cpuidle: using governor ladder
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffffffc0008d5000, 1 data @ ffffffc0008d4000)
+vdso: 2 pages (1 code @ ffffffc0008d9000, 1 data @ ffffffc0008d8000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -88,7 +88,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.484:1): initialized
+audit: type=2000 audit(0.564:1): initialized
 HugeTLB registered 2 MB page size, pre-allocated 0 pages
 VFS: Disk quotas dquot_6.6.0
 VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
@@ -133,8 +133,8 @@
 EXT4-fs (vda2): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 254:2.
 devtmpfs: mounted
-Freeing unused kernel memory: 544K (ffffffc000837000 - ffffffc0008bf000)
-Freeing alternatives memory: 48K (ffffffc0008bf000 - ffffffc0008cb000)
+Freeing unused kernel memory: 544K (ffffffc000838000 - ffffffc0008c0000)
+Freeing alternatives memory: 48K (ffffffc0008c0000 - ffffffc0008cc000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 bash-4.3# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-11-04 22:23:47.995894189 -0800
+++ /tmp/foob	2015-11-04 22:23:47.999894185 -0800
@@ -38,20 +38,20 @@
 	 #################################################################
 	 #################################################################
 	 #########################################################
-	 3.2 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 6551712 (63f8a0 hex)
+Bytes transferred = 6552416 (63fb60 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x63f8a0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x63fb60 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.3.0-next-20151104-53786-gccbc5c2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Nov 3 23:18:21 PST 2015
+Linux version 4.3.0-next-20151105-53968-gabac5a3 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Nov 4 22:17:44 PST 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -67,7 +67,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0fc9b80, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c0fcc180, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -80,7 +80,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998564K/2096128K available (8263K kernel code, 1038K rwdata, 3736K rodata, 1052K init, 341K bss, 32028K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998552K/2096128K available (8267K kernel code, 1040K rwdata, 3740K rodata, 1052K init, 341K bss, 32040K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,10 +88,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0dc0cfc   (12004 kB)
-      .init : 0xc0dc1000 - 0xc0ec8000   (1052 kB)
-      .data : 0xc0ec8000 - 0xc0fcbae0   (1039 kB)
-       .bss : 0xc0fce000 - 0xc1023468   ( 342 kB)
+      .text : 0xc0208000 - 0xc0dc2e78   (12012 kB)
+      .init : 0xc0dc3000 - 0xc0eca000   (1052 kB)
+      .data : 0xc0eca000 - 0xc0fce0e0   (1041 kB)
+       .bss : 0xc0fd1000 - 0xc1026468   ( 342 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -298,7 +298,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-05-16 16:15:42 UTC (958493742)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-05-17 15:15:07 UTC (958576507)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -307,7 +307,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1052K (c0dc1000 - c0ec8000)
+Freeing unused kernel memory: 1052K (c0dc3000 - c0eca000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.248992] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.244091] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2015-11-04 22:23:48.003894180 -0800
+++ /tmp/foob	2015-11-04 22:23:48.007894176 -0800
@@ -36,21 +36,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##############################
-	 3.3 MiB/s
+	 ###############################
+	 3.4 MiB/s
 done
-Bytes transferred = 5209088 (4f7c00 hex)
+Bytes transferred = 5211552 (4f85a0 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.8 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4f7c00 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4f85a0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.3.0-next-20151104-53786-gccbc5c2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Nov 3 23:18:38 PST 2015
+Linux version 4.3.0-next-20151105-53968-gabac5a3 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Nov 4 22:17:59 PST 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0a14280, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0a16640, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002332K/2096128K available (7150K kernel code, 351K rwdata, 2448K rodata, 340K init, 364K bss, 28260K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2002320K/2096128K available (7155K kernel code, 352K rwdata, 2452K rodata, 340K init, 364K bss, 28272K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0968008   (9601 kB)
-      .init : 0xc0969000 - 0xc09be000   ( 340 kB)
-      .data : 0xc09be000 - 0xc0a15da0   ( 352 kB)
-       .bss : 0xc0a18000 - 0xc0a73190   ( 365 kB)
+      .text : 0xc0008000 - 0xc096a170   (9609 kB)
+      .init : 0xc096b000 - 0xc09c0000   ( 340 kB)
+      .data : 0xc09c0000 - 0xc0a18160   ( 353 kB)
+       .bss : 0xc0a1b000 - 0xc0a76190   ( 365 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -297,7 +297,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-05-16 16:16:14 UTC (958493774)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-05-17 15:15:35 UTC (958576535)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -308,7 +308,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 340K (c0969000 - c09be000)
+Freeing unused kernel memory: 340K (c096b000 - c09c0000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.870064] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.880117] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-11-04 22:23:48.015894169 -0800
+++ /tmp/foob	2015-11-04 22:23:48.015894169 -0800
@@ -42,7 +42,7 @@
 	 #########################################################
 	 3.2 MiB/s
 done
-Bytes transferred = 6551712 (63f8a0 hex)
+Bytes transferred = 6552416 (63fb60 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -53,7 +53,7 @@
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x63f8a0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x63fb60 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012a89
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.3.0-next-20151104-53786-gccbc5c2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Nov 3 23:18:21 PST 2015
+Linux version 4.3.0-next-20151105-53968-gabac5a3 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Nov 4 22:17:44 PST 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0fc9b80, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c0fcc180, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998456K/2096128K available (8263K kernel code, 1038K rwdata, 3736K rodata, 1052K init, 341K bss, 32136K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998444K/2096128K available (8267K kernel code, 1040K rwdata, 3740K rodata, 1052K init, 341K bss, 32148K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0dc0cfc   (12004 kB)
-      .init : 0xc0dc1000 - 0xc0ec8000   (1052 kB)
-      .data : 0xc0ec8000 - 0xc0fcbae0   (1039 kB)
-       .bss : 0xc0fce000 - 0xc1023468   ( 342 kB)
+      .text : 0xc0208000 - 0xc0dc2e78   (12012 kB)
+      .init : 0xc0dc3000 - 0xc0eca000   (1052 kB)
+      .data : 0xc0eca000 - 0xc0fce0e0   (1041 kB)
+       .bss : 0xc0fd1000 - 0xc1026468   ( 342 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -275,6 +275,14 @@
 NET: Registered protocol family 10
 sit: IPv6 over IPv4 tunneling driver
 NET: Registered protocol family 17
+mmc0: MAN_BKOPS_EN bit is not set
+mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 SEM16G 14.6 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+Primary GPT is invalid, using alternate GPT.
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9
 can: controller area network core (rev 20120528 abi 9)
 NET: Registered protocol family 29
 can: raw protocol (rev 20120528)
@@ -288,14 +296,6 @@
 +3.3V_LP0: supplied by +3.3V_SYS
 +1.35V_LP0(sd2): supplied by +5V_SYS
 +1.05V_RUN_AVDD: supplied by +1.35V_LP0(sd2)
-mmc0: MAN_BKOPS_EN bit is not set
-mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 SEM16G 14.6 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
-Primary GPT is invalid, using alternate GPT.
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9
 tegra-pcie 1003000.pcie-controller: probing port 0, using 2 lanes
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
 tegra-pcie 1003000.pcie-controller: link 0 down, retrying
@@ -350,7 +350,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 13:50:00 UTC (946734600)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:18 UTC (946684818)
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
@@ -367,7 +367,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1052K (c0dc1000 - c0ec8000)
+Freeing unused kernel memory: 1052K (c0dc3000 - c0eca000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2015-11-04 22:23:48.023894160 -0800
+++ /tmp/foob	2015-11-04 22:23:48.023894160 -0800
@@ -38,21 +38,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##############################
-	 3.1 MiB/s
+	 ###############################
+	 3.2 MiB/s
 done
-Bytes transferred = 5209088 (4f7c00 hex)
+Bytes transferred = 5211552 (4f85a0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *#####
-	 2.8 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4f7c00 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4f85a0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012a89
@@ -63,7 +63,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.3.0-next-20151104-53786-gccbc5c2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Nov 3 23:18:38 PST 2015
+Linux version 4.3.0-next-20151105-53968-gabac5a3 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Nov 4 22:17:59 PST 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -71,7 +71,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0a14280, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0a16640, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -84,7 +84,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002224K/2096128K available (7150K kernel code, 351K rwdata, 2448K rodata, 340K init, 364K bss, 28368K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2002212K/2096128K available (7155K kernel code, 352K rwdata, 2452K rodata, 340K init, 364K bss, 28380K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,10 +92,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0968008   (9601 kB)
-      .init : 0xc0969000 - 0xc09be000   ( 340 kB)
-      .data : 0xc09be000 - 0xc0a15da0   ( 352 kB)
-       .bss : 0xc0a18000 - 0xc0a73190   ( 365 kB)
+      .text : 0xc0008000 - 0xc096a170   (9609 kB)
+      .init : 0xc096b000 - 0xc09c0000   ( 340 kB)
+      .data : 0xc09c0000 - 0xc0a18160   ( 353 kB)
+       .bss : 0xc0a1b000 - 0xc0a76190   ( 365 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -376,7 +376,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 13:50:33 UTC (946734633)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:49 UTC (946684849)
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
@@ -392,10 +392,10 @@
 EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 VFS: Mounted root (ext4 filesystem) on device 179:1.
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 devtmpfs: mounted
-Freeing unused kernel memory: 340K (c0969000 - c09be000)
+Freeing unused kernel memory: 340K (c096b000 - c09c0000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-11-04 22:23:48.031894152 -0800
+++ /tmp/foob	2015-11-04 22:23:48.031894152 -0800
@@ -46,9 +46,9 @@
 	 #################################################################
 	 #################################################################
 	 #########################################################
-	 4 MiB/s
+	 4.1 MiB/s
 done
-Bytes transferred = 6551712 (63f8a0 hex)
+Bytes transferred = 6552416 (63fb60 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -59,7 +59,7 @@
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x63f8a0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x63fb60 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -67,7 +67,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.3.0-next-20151104-53786-gccbc5c2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Nov 3 23:18:21 PST 2015
+Linux version 4.3.0-next-20151105-53968-gabac5a3 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Nov 4 22:17:44 PST 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -75,7 +75,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0fc9b80, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c0fcc180, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -88,7 +88,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 959372K/1048576K available (8263K kernel code, 1038K rwdata, 3736K rodata, 1052K init, 341K bss, 23668K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 959360K/1048576K available (8267K kernel code, 1040K rwdata, 3740K rodata, 1052K init, 341K bss, 23680K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -96,10 +96,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0dc0cfc   (12004 kB)
-      .init : 0xc0dc1000 - 0xc0ec8000   (1052 kB)
-      .data : 0xc0ec8000 - 0xc0fcbae0   (1039 kB)
-       .bss : 0xc0fce000 - 0xc1023468   ( 342 kB)
+      .text : 0xc0208000 - 0xc0dc2e78   (12012 kB)
+      .init : 0xc0dc3000 - 0xc0eca000   (1052 kB)
+      .data : 0xc0eca000 - 0xc0fce0e0   (1041 kB)
+       .bss : 0xc0fd1000 - 0xc1026468   ( 342 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -327,21 +327,21 @@
 mmc0: new high speed SDHC card at address b368
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
- mmcblk0: p1
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
+ mmcblk0: p1
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-26 23:19:01 UTC (1419635941)
+rtc-em3027 2-0056: setting system clock to 2014-12-26 23:26:43 UTC (1419636403)
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
+mmc1: new high speed SD card at address b368
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
 devtmpfs: mounted
-mmc1: new high speed SD card at address b368
-Freeing unused kernel memory: 1052K (c0dc1000 - c0ec8000)
+Freeing unused kernel memory: 1052K (c0dc3000 - c0eca000)
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2015-11-04 22:23:48.039894144 -0800
+++ /tmp/foob	2015-11-04 22:23:48.039894144 -0800
@@ -44,10 +44,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##############################
+	 ###############################
 	 4.1 MiB/s
 done
-Bytes transferred = 5209088 (4f7c00 hex)
+Bytes transferred = 5211552 (4f85a0 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x4f7c00 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x4f85a0 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -69,7 +69,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.3.0-next-20151104-53786-gccbc5c2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Nov 3 23:18:38 PST 2015
+Linux version 4.3.0-next-20151105-53968-gabac5a3 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Nov 4 22:17:59 PST 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -78,7 +78,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0a14280, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0a16640, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -91,7 +91,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 963140K/1048576K available (7150K kernel code, 351K rwdata, 2448K rodata, 340K init, 364K bss, 19900K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 963128K/1048576K available (7155K kernel code, 352K rwdata, 2452K rodata, 340K init, 364K bss, 19912K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -99,10 +99,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0968008   (9601 kB)
-      .init : 0xc0969000 - 0xc09be000   ( 340 kB)
-      .data : 0xc09be000 - 0xc0a15da0   ( 352 kB)
-       .bss : 0xc0a18000 - 0xc0a73190   ( 365 kB)
+      .text : 0xc0008000 - 0xc096a170   (9609 kB)
+      .init : 0xc096b000 - 0xc09c0000   ( 340 kB)
+      .data : 0xc09c0000 - 0xc0a18160   ( 353 kB)
+       .bss : 0xc0a1b000 - 0xc0a76190   ( 365 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
@@ -311,15 +311,12 @@
 tlv320aic23-codec 2-001a: ASoC: Failed to add route RLINEIN -> NULL -> Line Input
 tlv320aic23-codec 2-001a: Control not supported for path MICIN -> [NULL] -> Mic Input
 tlv320aic23-codec 2-001a: ASoC: no dapm match for MICIN --> NULL --> Mic Input
-tlv320aic23-codec 2-001a: ASoC: Failed to add route MICIN -> NULL -> Mic Input
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
+tlv320aic23-codec 2-001a: ASoC: Failed to add route MICIN -> NULL -> Mic Input
 tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
-mmc0: host does not support reading read-only switch, assuming write-enable
-mmc0: new high speed SDHC card at address b368
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
- mmcblk0: p1
 NET: Registered protocol family 10
 mip6: Mobile IPv6
+mmc0: host does not support reading read-only switch, assuming write-enable
 sit: IPv6 over IPv4 tunneling driver
 NET: Registered protocol family 17
 NET: Registered protocol family 15
@@ -331,33 +328,36 @@
 Bluetooth: RFCOMM socket layer initialized
 Bluetooth: RFCOMM ver 1.11
 Bluetooth: BNEP (Ethernet Emulation) ver 1.3
+mmc0: new high speed SDHC card at address b368
 Bluetooth: BNEP socket layer initialized
 Bluetooth: HIDP (Human Interface Emulation) ver 1.2
 Bluetooth: HIDP socket layer initialized
-mmc1: new high speed SD card at address b368
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
+ mmcblk0: p1
 Key type dns_resolver registered
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
- mmcblk1: p1 p2
 Registering SWP/SWPB emulation handler
-hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
+hub 1-1:1.0: USB hub found
+hub 1-1:1.0: 4 ports detected
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-26 23:19:20 UTC (1419635960)
+rtc-em3027 2-0056: setting system clock to 2014-12-26 23:27:01 UTC (1419636421)
 ALSA device list:
   #0: tegra-trimslice
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
+mmc1: new high speed SD card at address b368
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
+ mmcblk1: p1 p2
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 340K (c0969000 - c09be000)
+Freeing unused kernel memory: 340K (c096b000 - c09c0000)
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-11-04 22:23:48.047894136 -0800
+++ /tmp/foob	2015-11-04 22:23:48.047894136 -0800
@@ -38,20 +38,20 @@
 	 #################################################################
 	 #################################################################
 	 #########################################################
-	 3.3 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 6551712 (63f8a0 hex)
+Bytes transferred = 6552416 (63fb60 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x63f8a0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x63fb60 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.3.0-next-20151104-53786-gccbc5c2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Nov 3 23:18:21 PST 2015
+Linux version 4.3.0-next-20151105-53968-gabac5a3 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Nov 4 22:17:44 PST 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -67,7 +67,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0fc9b80, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c0fcc180, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -80,7 +80,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998484K/2096128K available (8263K kernel code, 1038K rwdata, 3736K rodata, 1052K init, 341K bss, 32108K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998472K/2096128K available (8267K kernel code, 1040K rwdata, 3740K rodata, 1052K init, 341K bss, 32120K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,10 +88,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0dc0cfc   (12004 kB)
-      .init : 0xc0dc1000 - 0xc0ec8000   (1052 kB)
-      .data : 0xc0ec8000 - 0xc0fcbae0   (1039 kB)
-       .bss : 0xc0fce000 - 0xc1023468   ( 342 kB)
+      .text : 0xc0208000 - 0xc0dc2e78   (12012 kB)
+      .init : 0xc0dc3000 - 0xc0eca000   (1052 kB)
+      .data : 0xc0eca000 - 0xc0fce0e0   (1041 kB)
+       .bss : 0xc0fd1000 - 0xc1026468   ( 342 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -288,8 +288,8 @@
 mmcblk0: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk0boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk0boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
-tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 mmcblk0rpmb: mmc1:0001 SEM16G partition 3 128 KiB
+tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 Primary GPT is invalid, using alternate GPT.
  mmcblk0: p1 p2 p3 p4 p5
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -362,7 +362,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1052K (c0dc1000 - c0ec8000)
+Freeing unused kernel memory: 1052K (c0dc3000 - c0eca000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.758394] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.778824] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2015-11-04 22:23:48.055894128 -0800
+++ /tmp/foob	2015-11-04 22:23:48.055894128 -0800
@@ -36,21 +36,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##############################
+	 ###############################
 	 3.3 MiB/s
 done
-Bytes transferred = 5209088 (4f7c00 hex)
+Bytes transferred = 5211552 (4f85a0 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.3 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4f7c00 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4f85a0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.3.0-next-20151104-53786-gccbc5c2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Nov 3 23:18:38 PST 2015
+Linux version 4.3.0-next-20151105-53968-gabac5a3 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Nov 4 22:17:59 PST 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0a14280, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0a16640, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002252K/2096128K available (7150K kernel code, 351K rwdata, 2448K rodata, 340K init, 364K bss, 28340K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2002240K/2096128K available (7155K kernel code, 352K rwdata, 2452K rodata, 340K init, 364K bss, 28352K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0968008   (9601 kB)
-      .init : 0xc0969000 - 0xc09be000   ( 340 kB)
-      .data : 0xc09be000 - 0xc0a15da0   ( 352 kB)
-       .bss : 0xc0a18000 - 0xc0a73190   ( 365 kB)
+      .text : 0xc0008000 - 0xc096a170   (9609 kB)
+      .init : 0xc096b000 - 0xc09c0000   ( 340 kB)
+      .data : 0xc09c0000 - 0xc0a18160   ( 353 kB)
+       .bss : 0xc0a1b000 - 0xc0a76190   ( 365 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -364,7 +364,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 340K (c0969000 - c09be000)
+Freeing unused kernel memory: 340K (c096b000 - c09c0000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.516063] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.516078] usb 2-1: new high-speed USB device number 2 using tegra-ehci
