Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-06-20 23:27:06.572340846 -0700
+++ /tmp/foob	2016-06-20 23:27:06.576340848 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-next-20160620+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Jun 20 00:06:14 PDT 2016
+Linux version 4.7.0-rc4-next-20160621+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Jun 20 23:21:45 PDT 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -9,7 +9,7 @@
 psci: PSCIv0.2 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: Trusted OS migration not required
-percpu: Embedded 20 pages/cpu @ffff80003efd0000 s42880 r8192 d30848 u81920
+percpu: Embedded 21 pages/cpu @ffff80003efcf000 s47104 r8192 d30720 u86016
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
 CPU features: enabling workaround for ARM erratum 834220
@@ -19,15 +19,15 @@
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
 software IO TLB [mem 0x79c00000-0x7dc00000] (64MB) mapped at [ffff800039c00000-ffff80003dbfffff]
-Memory: 934968K/1048576K available (7656K kernel code, 681K rwdata, 3476K rodata, 960K init, 262K bss, 97224K reserved, 16384K cma-reserved)
+Memory: 934968K/1048576K available (7656K kernel code, 679K rwdata, 3476K rodata, 960K init, 260K bss, 97224K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff0000087f0000   (  7616 KB)
     .rodata : 0xffff0000087f0000 - 0xffff000008b60000   (  3520 KB)
       .init : 0xffff000008b60000 - 0xffff000008c50000   (   960 KB)
-      .data : 0xffff000008c50000 - 0xffff000008cfa600   (   682 KB)
-       .bss : 0xffff000008cfa600 - 0xffff000008d3be38   (   263 KB)
+      .data : 0xffff000008c50000 - 0xffff000008cf9e00   (   680 KB)
+       .bss : 0xffff000008cf9e00 - 0xffff000008d3ae38   (   261 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -98,7 +98,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.596:1): initialized
+audit: type=2000 audit(0.600:1): initialized
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -112,64 +112,64 @@
 libphy: mdio_driver_register: phy-bcm-ns2-pci
 pl061_gpio 9030000.pl061: PL061 GPIO chip @0x0000000009030000 registered
 Unable to handle kernel NULL pointer dereference at virtual address 00000000
-pgd = ffff000008d3f000
+pgd = ffff000008d3e000
 [00000000] *pgd=000000007effe003, *pud=000000007effd003, *pmd=0000000000000000
 Internal error: Oops: 94000004 [#1] PREEMPT SMP
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.7.0-rc3-next-20160620+ #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.7.0-rc4-next-20160621+ #1
 Hardware name: linux,dummy-virt (DT)
 task: ffff800039860000 ti: ffff800039868000 task.ti: ffff800039868000
 PC is at pci_ecam_create+0x1c/0x1f0
-LR is at pci_host_common_probe+0x90/0x2b4
-pc : [<ffff0000083cbb94>] lr : [<ffff0000083ceb30>] pstate: 00000045
+LR is at pci_host_common_probe+0x90/0x2b8
+pc : [<ffff0000083cbf94>] lr : [<ffff0000083cef30>] pstate: 00000145
 sp : ffff80003986bb40
-x29: ffff80003986bb40 x28: ffff000008baf7e0 
-x27: ffff000008cff000 x26: ffff000008b6047c 
-x25: ffff000008baf840 x24: ffff000008cff000 
+x29: ffff80003986bb40 x28: ffff000008baf9c0 
+x27: ffff000008cfe000 x26: ffff000008b6047c 
+x25: ffff000008bafa20 x24: ffff000008cfe000 
 x23: ffff000008c93e30 x22: 0000000000000000 
 x21: ffff80003986bc00 x20: ffff80003998e010 
 x19: ffffffffffffffea x18: 0000000000000007 
 x17: 000000000000000e x16: 0000000000000001 
-x15: 0000000000000007 x14: 000000000000000e 
-x13: 0000000000000013 x12: 0000000000000028 
+x15: 0000000000000007 x14: ffffffffffffffff 
+x13: ffffffffffffffff x12: 0000000000000028 
 x11: 0000000000000028 x10: 0101010101010101 
 x9 : fffffffffffffffe x8 : 7f7f7f7f7f7f7f7f 
 x7 : 6462ff726b6b6462 x6 : ffff80003986baf8 
-x5 : 6564000000000000 x4 : ffff80003eff29c8 
+x5 : 6564000000000000 x4 : 0000000000000000 
 x3 : ffff000008c93e30 x2 : 0000000000000000 
 x1 : ffff80003986bc10 x0 : ffff80003998e010 
 
 Process swapper/0 (pid: 1, stack limit = 0xffff800039868020)
 Stack: (0xffff80003986bb40 to 0xffff80003986c000)
-bb40: ffff80003986bb90 ffff0000083ceb30 0000000000000000 ffff80003998e010
-bb60: ffff80003986bc00 0000000000000000 ffff000008c93e30 ffff000008cff000
-bb80: ffff000008baf840 0000000000000000 ffff80003986bc50 ffff0000083ced80
+bb40: ffff80003986bb90 ffff0000083cef30 ffff7dfffe801380 ffff80003998e010
+bb60: ffff80003986bc00 0000000000000000 ffff000008c93e30 ffff000008cfe000
+bb80: ffff000008bafa20 0000000000000000 ffff80003986bc50 ffff0000083cf184
 bba0: ffff80003998e000 ffff80003998e010 ffff000008c941c8 ffff000008c941f0
-bbc0: 0000000000000000 ffff000008cff000 ffff000008baf840 ffff000008b6047c
-bbe0: ffff000008cff000 ffff00000869b4e8 ffff000008d2ede8 ffff000008869238
+bbc0: 0000000000000000 ffff000008cfe000 ffff000008bafa20 ffff000008b6047c
+bbe0: ffff000008cfe000 ffff00000869b878 ffff000008d2dde8 ffff000008869238
 bc00: ffff80003986bc00 ffff80003986bc00 000000003f000000 000000003fffffff
 bc20: ffff80003eff29c8 0000000000000200 0000000000000000 0000000000000000
-bc40: 0000000000000000 0000000000000000 ffff80003986bc70 ffff0000084ace68
-bc60: ffff000008c941f0 ffff0000084aaaa0 ffff80003986bca0 ffff0000084ab33c
-bc80: ffff80003998e010 ffff000008d2b000 0000000000000000 ffff000008d2b000
-bca0: ffff80003986bce0 ffff0000084ab480 ffff80003998e010 ffff000008c941f0
-bcc0: ffff80003998e070 ffff000008caa4d8 ffff000008caa000 ffff0000084a93cc
-bce0: ffff80003986bd10 ffff0000084a93c0 0000000000000000 ffff000008c941f0
-bd00: ffff0000084ab3dc ffff80003986bd60 ffff80003986bd50 ffff0000084aac4c
-bd20: ffff000008c941f0 ffff800039b6d100 0000000000000000 ffff0000087d3348
-bd40: ffff80003981fea8 ffff80003995e768 ffff80003986bd60 ffff0000084aa864
-bd60: ffff80003986bda0 ffff0000084abd38 ffff000008c941f0 ffff000008b82dac
-bd80: 0000000000000000 ffff000008baf7b0 ffff000008b518b0 ffff0000084abd70
-bda0: ffff80003986bdc0 ffff0000084acdac ffff800039868000 ffff000008b82dac
-bdc0: ffff80003986bdd0 ffff000008b82dc4 ffff80003986bde0 ffff000008081a14
-bde0: ffff80003986be50 ffff000008b60cb0 ffff000008c163f0 0000000000000006
-be00: 0000000000000103 ffff000008baf7b0 ffff000008c66d00 ffff000008a54640
-be20: ffff80003986be30 ffff000008a53e70 0000000600000006 0000000000000000
-be40: 0000000000000000 ffff000008b518b0 ffff80003986beb0 ffff0000087d5250
-be60: ffff0000087d5240 0000000000000000 0000000000000000 0000000000000000
+bc40: 0000000000000000 0000000000000000 ffff80003986bc70 ffff0000084ad26c
+bc60: ffff000008c941f0 ffff0000084aaea4 ffff80003986bca0 ffff0000084ab740
+bc80: ffff80003998e010 ffff000008d2a000 0000000000000000 ffff000008d2a000
+bca0: ffff80003986bce0 ffff0000084ab884 ffff80003998e010 ffff000008c941f0
+bcc0: ffff80003998e070 ffff000008caa4d8 ffff000008caa000 ffff0000084a97d0
+bce0: ffff80003986bd10 ffff0000084a97c4 0000000000000000 ffff000008c941f0
+bd00: ffff0000084ab7e0 ffff80003986bd60 ffff80003986bd50 ffff0000084ab050
+bd20: ffff000008c941f0 ffff800039b4e900 0000000000000000 ffff0000087d39c4
+bd40: ffff80003981fea8 ffff8000398e2d68 ffff80003986bd60 ffff0000084aac68
+bd60: ffff80003986bda0 ffff0000084ac13c ffff000008c941f0 ffff000008b82dbc
+bd80: 0000000000000000 ffff000008baf990 ffff000008b519b0 ffff0000084ac174
+bda0: ffff80003986bdc0 ffff0000084ad1b0 ffff800039868000 ffff000008b82dbc
+bdc0: ffff80003986bdd0 ffff000008b82dd4 ffff80003986bde0 ffff000008081a14
+bde0: ffff80003986be50 ffff000008b60cb0 ffff000008c17010 0000000000000006
+be00: 0000000000000103 ffff000008baf990 ffff000008c66d00 ffff000008a54740
+be20: ffff80003986be30 ffff000008a53f70 0000000600000006 0000000000000000
+be40: 0000000000000000 ffff000008b519b0 ffff80003986beb0 ffff0000087d58cc
+be60: ffff0000087d58bc 0000000000000000 0000000000000000 0000000000000000
 be80: 0000000000000000 0000000000000000 0000000000000000 0000000000000000
 bea0: 0000000000000000 0000000000000000 0000000000000000 ffff000008084e10
-bec0: ffff0000087d5240 0000000000000000 0000000000000000 0000000000000000
+bec0: ffff0000087d58bc 0000000000000000 0000000000000000 0000000000000000
 bee0: 0000000000000000 0000000000000000 0000000000000000 0000000000000000
 bf00: 0000000000000000 0000000000000000 0000000000000000 0000000000000000
 bf20: 0000000000000000 0000000000000000 0000000000000000 0000000000000000
@@ -181,33 +181,33 @@
 bfe0: 0000000000000000 0000000000000000 0000000000000000 0000000000000000
 Call trace:
 Exception stack(0xffff80003986b980 to 0xffff80003986baa0)
-b980: ffffffffffffffea ffff80003998e010 ffff80003986bb40 ffff0000083cbb94
-b9a0: ffff80003986b9d0 ffff00000869a230 ffff000008d2ede8 ffff80003efec9e8
-b9c0: ffff80003986b9d0 ffff00000869a240 ffff80003986ba10 ffff00000869a338
+b980: ffffffffffffffea ffff80003998e010 ffff80003986bb40 ffff0000083cbf94
+b9a0: ffff80003986b9d0 ffff00000869a5c0 ffff000008d2dde8 ffff80003efec9e8
+b9c0: ffff80003986b9d0 ffff00000869a5d0 ffff80003986ba10 ffff00000869a6c8
 b9e0: ffff80003efec9e8 ffff000008a74000 0000000001000000 0000000000000000
-ba00: ffff000008a749d8 0000000000000000 ffff80003986ba30 ffff00000869ea9c
+ba00: ffff000008a74ac8 0000000000000000 ffff80003986ba30 ffff00000869ee2c
 ba20: ffff80003998e010 ffff80003986bc10 0000000000000000 ffff000008c93e30
-ba40: ffff80003eff29c8 6564000000000000 ffff80003986baf8 6462ff726b6b6462
+ba40: 0000000000000000 6564000000000000 ffff80003986baf8 6462ff726b6b6462
 ba60: 7f7f7f7f7f7f7f7f fffffffffffffffe 0101010101010101 0000000000000028
-ba80: 0000000000000028 0000000000000013 000000000000000e 0000000000000007
-[<ffff0000083cbb94>] pci_ecam_create+0x1c/0x1f0
-[<ffff0000083ceb30>] pci_host_common_probe+0x90/0x2b4
-[<ffff0000083ced80>] gen_pci_probe+0x2c/0x38
-[<ffff0000084ace68>] platform_drv_probe+0x50/0xbc
-[<ffff0000084ab33c>] driver_probe_device+0x200/0x2a0
-[<ffff0000084ab480>] __driver_attach+0xa4/0xa8
-[<ffff0000084a93c0>] bus_for_each_dev+0x58/0x98
-[<ffff0000084aac4c>] driver_attach+0x20/0x28
-[<ffff0000084aa864>] bus_add_driver+0x1c8/0x22c
-[<ffff0000084abd38>] driver_register+0x60/0xf4
-[<ffff0000084acdac>] __platform_driver_register+0x4c/0x54
-[<ffff000008b82dc4>] gen_pci_driver_init+0x18/0x20
+ba80: 0000000000000028 ffffffffffffffff ffffffffffffffff 0000000000000007
+[<ffff0000083cbf94>] pci_ecam_create+0x1c/0x1f0
+[<ffff0000083cef30>] pci_host_common_probe+0x90/0x2b8
+[<ffff0000083cf184>] gen_pci_probe+0x2c/0x38
+[<ffff0000084ad26c>] platform_drv_probe+0x50/0xbc
+[<ffff0000084ab740>] driver_probe_device+0x200/0x2a0
+[<ffff0000084ab884>] __driver_attach+0xa4/0xa8
+[<ffff0000084a97c4>] bus_for_each_dev+0x58/0x98
+[<ffff0000084ab050>] driver_attach+0x20/0x28
+[<ffff0000084aac68>] bus_add_driver+0x1c8/0x22c
+[<ffff0000084ac13c>] driver_register+0x60/0xf4
+[<ffff0000084ad1b0>] __platform_driver_register+0x4c/0x54
+[<ffff000008b82dd4>] gen_pci_driver_init+0x18/0x20
 [<ffff000008081a14>] do_one_initcall+0x38/0x11c
 [<ffff000008b60cb0>] kernel_init_freeable+0x14c/0x1ec
-[<ffff0000087d5250>] kernel_init+0x10/0xfc
+[<ffff0000087d58cc>] kernel_init+0x10/0xfc
 [<ffff000008084e10>] ret_from_fork+0x10/0x40
 Code: a9025bf5 a90363f7 f90023f9 928002b3 (f9400045) 
----[ end trace ab14d2ce9f6f1382 ]---
+---[ end trace cd1510edfd714a4d ]---
 Kernel panic - not syncing: Attempted to kill init! exitcode=0x0000000b
 
 Kernel Offset: disabled
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-06-20 23:27:06.596340858 -0700
+++ /tmp/foob	2016-06-20 23:27:06.600340860 -0700
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 #########################################
-	 3.3 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 7267360 (6ee420 hex)
+Bytes transferred = 7279344 (6f12f0 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3.3 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 34956 (888c hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6ee420 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6f12f0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff88b ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-next-20160620-101088-g6ab9759 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jun 20 00:03:31 PDT 2016
+Linux version 4.7.0-rc4-next-20160621-101370-g10698aa (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jun 20 23:19:35 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,20 +69,20 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141bb40, node_mem_map eeffa000
+free_area_init_node: node 0, pgdat c141b840, node_mem_map eeffa000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 13 pages/cpu @eefa1000 s23808 r8192 d21248 u53248
-pcpu-alloc: s23808 r8192 d21248 u53248 alloc=13*4096
+percpu: Embedded 14 pages/cpu @eef9d000 s26048 r8192 d23104 u57344
+pcpu-alloc: s26048 r8192 d23104 u57344 alloc=14*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=7369c667-ff51-ec4a-29cd-baabf2fbe346 rootwait gpt debug earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1995124K/2096128K available (9530K kernel code, 1142K rwdata, 4508K rodata, 2048K init, 342K bss, 35468K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1995108K/2096128K available (9530K kernel code, 1142K rwdata, 4516K rodata, 2048K init, 342K bss, 35484K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc10b5874   (15031 kB)
+      .text : 0xc0208000 - 0xc10b7b40   (15039 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141dba0   (1143 kB)
-       .bss : 0xc141f000 - 0xc1474918   ( 343 kB)
+      .data : 0xc1300000 - 0xc141d8a0   (1143 kB)
+       .bss : 0xc141f000 - 0xc1474a48   ( 343 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -282,13 +282,21 @@
 Registering SWP/SWPB emulation handler
 usb1_vbus: supplied by vdd-sys-5v0
 usb2_vbus: supplied by vdd-sys-5v0
-mmc0: new high speed SDHC card at address aaaa
 VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
+mmc0: new high speed SDHC card at address aaaa
+mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 tegra-dsi 54300000.dsi: cannot get VDD supply
+ mmcblk0: p1 p2 p3
 tegra-ehci 7d008000.usb: EHCI Host Controller
 tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 1
 tegra-ehci 7d008000.usb: irq 339, io mem 0x7d008000
+mmc1: new high speed MMC card at address 0001
+mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
+mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
+mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
+mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
+ mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
@@ -302,25 +310,12 @@
 dvdd_ts: disabling
 usb1_vbus: disabling
 VDD_5V0_HDMI_CON: disabling
-mmc1: new high speed MMC card at address 0001
-mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
-mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
-mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
-mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
- mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
-tegra-hdmi 54280000.hdmi: failed to get PLL regulator
-tegra-dsi 54300000.dsi: cannot get VDD supply
-mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
- mmcblk0: p1 p2 p3
 EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
-tegra-hdmi 54280000.hdmi: failed to get PLL regulator
-tegra-dsi 54300000.dsi: cannot get VDD supply
 EXT4-fs (mmcblk1p1): recovery complete
 EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:1.
+VFS: Mounted root (ext3 filesystem) on device 179:17.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# 
\ No newline at end of file
+root@(none):/# [    2.160627] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-06-20 23:27:06.604340863 -0700
+++ /tmp/foob	2016-06-20 23:27:06.604340863 -0700
@@ -37,9 +37,9 @@
 	 #################################################################
 	 #################################################################
 	 #######################################################
-	 3.1 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 5569224 (54fac8 hex)
+Bytes transferred = 5570448 (54ff90 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 34956 (888c hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x54fac8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x54ff90 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff88b ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-next-20160620-101088-g6ab9759 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jun 20 00:03:43 PDT 2016
+Linux version 4.7.0-rc4-next-20160621-101370-g10698aa (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jun 20 23:19:36 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,20 +69,20 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0e5bd80, node_mem_map eeff9000
+free_area_init_node: node 0, pgdat c0e5bd00, node_mem_map eeff9000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 13 pages/cpu @eefa0000 s23936 r8192 d21120 u53248
-pcpu-alloc: s23936 r8192 d21120 u53248 alloc=13*4096
+percpu: Embedded 14 pages/cpu @eef9c000 s26048 r8192 d23104 u57344
+pcpu-alloc: s26048 r8192 d23104 u57344 alloc=14*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=7369c667-ff51-ec4a-29cd-baabf2fbe346 rootwait gpt debug earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998936K/2096128K available (8441K kernel code, 374K rwdata, 2844K rodata, 1024K init, 366K bss, 31656K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998920K/2096128K available (8441K kernel code, 374K rwdata, 2844K rodata, 1024K init, 366K bss, 31672K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0c054e0   (12278 kB)
+      .text : 0xc0008000 - 0xc0c0553c   (12278 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0e5d920   ( 375 kB)
-       .bss : 0xc0e5f000 - 0xc0ebab14   ( 367 kB)
+      .data : 0xc0e00000 - 0xc0e5d8a0   ( 375 kB)
+       .bss : 0xc0e5f000 - 0xc0ebab04   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -256,15 +256,15 @@
 mmc1: SDHCI controller on 78000600.sdhci [78000600.sdhci] using ADMA
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
+mmc0: new high speed SDHC card at address aaaa
+mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
-mmc0: new high speed SDHC card at address aaaa
-mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
- mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
  mmcblk0: p1 p2 p3
+ mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
 tegra30-i2s 70080400.i2s: DMA channels sourced from device 70080000.ahub
 tegra-snd-rt5640 sound: rt5640-aif1 <-> 70080400.i2s mapping ok
 input: NVIDIA Tegra Dalmore Headphones as /devices/soc0/sound/sound/card0/input0
@@ -312,10 +312,10 @@
 EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk1p1): recovery complete
 EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:1.
+VFS: Mounted root (ext3 filesystem) on device 179:17.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (c0d00000 - c0e00000)
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-06-20 23:27:06.612340866 -0700
+++ /tmp/foob	2016-06-20 23:27:06.612340866 -0700
@@ -34,9 +34,9 @@
 	 #################################################################
 	 #################################################################
 	 #########################################
-	 3.9 MiB/s
+	 4 MiB/s
 done
-Bytes transferred = 7267360 (6ee420 hex)
+Bytes transferred = 7279344 (6f12f0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -47,7 +47,7 @@
 done
 Bytes transferred = 66737 (104b1 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6ee420 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x6f12f0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830134b0
@@ -55,7 +55,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-next-20160620-101088-g6ab9759 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jun 20 00:03:31 PDT 2016
+Linux version 4.7.0-rc4-next-20160621-101370-g10698aa (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jun 20 23:19:35 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -64,20 +64,20 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141bb40, node_mem_map eeffa000
+free_area_init_node: node 0, pgdat c141b840, node_mem_map eeffa000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 13 pages/cpu @eef8b000 s23808 r8192 d21248 u53248
-pcpu-alloc: s23808 r8192 d21248 u53248 alloc=13*4096
+percpu: Embedded 14 pages/cpu @eef87000 s26048 r8192 d23104 u57344
+pcpu-alloc: s26048 r8192 d23104 u57344 alloc=14*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=b1791faa-84e8-4e27-83a1-9fb0a77e8803 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1995004K/2096128K available (9530K kernel code, 1142K rwdata, 4508K rodata, 2048K init, 342K bss, 35588K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1994988K/2096128K available (9530K kernel code, 1142K rwdata, 4516K rodata, 2048K init, 342K bss, 35604K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -85,10 +85,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc10b5874   (15031 kB)
+      .text : 0xc0208000 - 0xc10b7b40   (15039 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141dba0   (1143 kB)
-       .bss : 0xc141f000 - 0xc1474918   ( 343 kB)
+      .data : 0xc1300000 - 0xc141d8a0   (1143 kB)
+       .bss : 0xc141f000 - 0xc1474a48   ( 343 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -307,13 +307,13 @@
 +5V_SATA: supplied by +5V_SYS
 +12V_SATA: supplied by +VDD_MUX
 usb2-0 supply vbus not found, using dummy regulator
+usb2-1 supply vbus not found, using dummy regulator
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
  mmcblk0: p1
-usb2-1 supply vbus not found, using dummy regulator
 usb3-0 supply vbus not found, using dummy regulator
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
 tegra-pcie 1003000.pcie-controller: Failed to get supply 'avddio-pex': -517
@@ -377,8 +377,8 @@
 pci 0000:01:00.0: PME# supported from D0 D3hot D3cold
 PCI: bus1: Fast back to back transfers disabled
 pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
-pci 0000:02:00.0: [10ec:8168] type 00 class 0x020000
 ata1: SATA link down (SStatus 0 SControl 300)
+pci 0000:02:00.0: [10ec:8168] type 00 class 0x020000
 pci 0000:02:00.0: reg 0x10: [io  0x0000-0x00ff]
 pci 0000:02:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit]
 pci 0000:02:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
@@ -388,8 +388,11 @@
 pci_bus 0000:02: busn_res: [bus 02-ff] end is updated to 02
 pci 0000:00:01.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
 pci 0000:00:02.0: BAR 8: assigned [mem 0x13100000-0x131fffff]
+mmc1: new high speed SDHC card at address aaaa
+mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 pci 0000:00:02.0: BAR 9: assigned [mem 0x20000000-0x200fffff 64bit pref]
 pci 0000:00:02.0: BAR 7: assigned [io  0x1000-0x1fff]
+ mmcblk1: p1 p2 p3
 pci 0000:01:00.0: BAR 0: assigned [mem 0x13000000-0x13001fff 64bit]
 pci 0000:00:01.0: PCI bridge to [bus 01]
 pci 0000:00:01.0:   bridge window [mem 0x13000000-0x130fffff]
@@ -412,13 +415,10 @@
 pcie_pme 0000:00:02.0:pcie001: service driver pcie_pme loaded
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
-r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0a52000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 393
+r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0a5c000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 393
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
-mmc1: new high speed SDHC card at address aaaa
-mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
- mmcblk1: p1 p2 p3
 tegra-hdmi 54280000.hdmi: cannot set audio to 44100 at 297000000 pclk
 Console: switching to colour frame buffer device 512x135
 drm drm: fb0:  frame buffer device
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-06-20 23:27:06.620340870 -0700
+++ /tmp/foob	2016-06-20 23:27:06.624340873 -0700
@@ -27,25 +27,25 @@
 Filename 'zImage.tegra_defconfig'.
 Load address: 0x82000000
 Loading: *#################################################################
-	 #################################################################
+	 ###################T T T ##############################################
 	 #################################################################
 	 #################################################################
 	 #################################################################
 	 #######################################################
-	 3.8 MiB/s
+	 257.8 KiB/s
 done
-Bytes transferred = 5569224 (54fac8 hex)
+Bytes transferred = 5570448 (54ff90 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.7 MiB/s
+	 904.3 KiB/s
 done
 Bytes transferred = 66737 (104b1 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x54fac8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x54ff90 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830134b0
@@ -54,7 +54,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-next-20160620-101088-g6ab9759 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jun 20 00:03:43 PDT 2016
+Linux version 4.7.0-rc4-next-20160621-101370-g10698aa (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jun 20 23:19:36 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -64,20 +64,20 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0e5bd80, node_mem_map eeff9000
+free_area_init_node: node 0, pgdat c0e5bd00, node_mem_map eeff9000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 13 pages/cpu @eef8a000 s23936 r8192 d21120 u53248
-pcpu-alloc: s23936 r8192 d21120 u53248 alloc=13*4096
+percpu: Embedded 14 pages/cpu @eef86000 s26048 r8192 d23104 u57344
+pcpu-alloc: s26048 r8192 d23104 u57344 alloc=14*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=b1791faa-84e8-4e27-83a1-9fb0a77e8803 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998816K/2096128K available (8441K kernel code, 374K rwdata, 2844K rodata, 1024K init, 366K bss, 31776K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998800K/2096128K available (8441K kernel code, 374K rwdata, 2844K rodata, 1024K init, 366K bss, 31792K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -85,10 +85,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0c054e0   (12278 kB)
+      .text : 0xc0008000 - 0xc0c0553c   (12278 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0e5d920   ( 375 kB)
-       .bss : 0xc0e5f000 - 0xc0ebab14   ( 367 kB)
+      .data : 0xc0e00000 - 0xc0e5d8a0   ( 375 kB)
+       .bss : 0xc0e5f000 - 0xc0ebab04   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -286,8 +286,8 @@
 rt5640 0-001c: ASoC: no source widget found for DAC R2 Power
 rt5640 0-001c: ASoC: Failed to add route DAC R2 Power -> direct -> IF2 DAC R
 tegra-snd-rt5640 sound: rt5640-aif1 <-> 70301100.i2s mapping ok
-input: NVIDIA Tegra Jetson TK1 Headphones as /devices/soc0/sound/sound/card1/input1
-input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input0
+input: NVIDIA Tegra Jetson TK1 Headphones as /devices/soc0/sound/sound/card1/input0
+input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input1
 NET: Registered protocol family 10
 mip6: Mobile IPv6
 sit: IPv6 over IPv4 tunneling driver
@@ -369,10 +369,10 @@
 mmc1: Unknown controller version (3). You may experience problems.
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
+ata1: SATA link down (SStatus 0 SControl 300)
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
 tegra-pcie 1003000.pcie-controller: probing port 0, using 2 lanes
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
-ata1: SATA link down (SStatus 0 SControl 300)
 tegra-pcie 1003000.pcie-controller: probing port 1, using 1 lanes
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000000
 tegra-pcie 1003000.pcie-controller: PCI host bridge to bus 0000:00
@@ -399,15 +399,15 @@
 pci 0000:02:00.0: supports D1 D2
 pci 0000:02:00.0: PME# supported from D0 D1 D2 D3hot D3cold
 mmc1: new high speed SDHC card at address aaaa
+mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
+ mmcblk1: p1 p2 p3
 PCI: bus2: Fast back to back transfers disabled
 pci_bus 0000:02: busn_res: [bus 02-ff] end is updated to 02
-mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 pci 0000:00:01.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
 pci 0000:00:02.0: BAR 8: assigned [mem 0x13100000-0x131fffff]
 pci 0000:00:02.0: BAR 9: assigned [mem 0x20000000-0x200fffff 64bit pref]
 pci 0000:00:02.0: BAR 7: assigned [io  0x1000-0x1fff]
 pci 0000:01:00.0: BAR 0: assigned [mem 0x13000000-0x13001fff 64bit]
- mmcblk1: p1 p2 p3
 pci 0000:00:01.0: PCI bridge to [bus 01]
 pci 0000:00:01.0:   bridge window [mem 0x13000000-0x130fffff]
 pci 0000:02:00.0: BAR 4: assigned [mem 0x20000000-0x20003fff 64bit pref]
@@ -441,7 +441,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:04 UTC (946684924)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:15 UTC (946684935)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-06-20 23:27:06.628340874 -0700
+++ /tmp/foob	2016-06-20 23:27:06.632340876 -0700
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -41,9 +41,9 @@
 	 #################################################################
 	 #################################################################
 	 #########################################
-	 3.4 MiB/s
+	 3.6 MiB/s
 done
-Bytes transferred = 7267360 (6ee420 hex)
+Bytes transferred = 7279344 (6f12f0 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -54,7 +54,7 @@
 done
 Bytes transferred = 66901 (10555 hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6ee420 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x6f12f0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013554
@@ -62,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-next-20160620-101088-g6ab9759 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jun 20 00:03:31 PDT 2016
+Linux version 4.7.0-rc4-next-20160621-101370-g10698aa (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jun 20 23:19:35 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -72,20 +72,20 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c141bb40, node_mem_map eeffa000
+free_area_init_node: node 0, pgdat c141b840, node_mem_map eeffa000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327679 pages, LIFO batch:31
-percpu: Embedded 13 pages/cpu @eef8b000 s23808 r8192 d21248 u53248
-pcpu-alloc: s23808 r8192 d21248 u53248 alloc=13*4096
+percpu: Embedded 14 pages/cpu @eef87000 s26048 r8192 d23104 u57344
+pcpu-alloc: s26048 r8192 d23104 u57344 alloc=14*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522751
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=4d36e82a-b3c0-4b4b-bfcd-afe5217ac023 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1996024K/2097148K available (9530K kernel code, 1142K rwdata, 4508K rodata, 2048K init, 342K bss, 35588K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1996008K/2097148K available (9530K kernel code, 1142K rwdata, 4516K rodata, 2048K init, 342K bss, 35604K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -93,10 +93,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc10b5874   (15031 kB)
+      .text : 0xc0208000 - 0xc10b7b40   (15039 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141dba0   (1143 kB)
-       .bss : 0xc141f000 - 0xc1474918   ( 343 kB)
+      .data : 0xc1300000 - 0xc141d8a0   (1143 kB)
+       .bss : 0xc141f000 - 0xc1474a48   ( 343 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -295,6 +295,7 @@
 usbhid: USB HID core driver
 NET: Registered protocol family 10
 sit: IPv6 over IPv4 tunneling driver
+input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 NET: Registered protocol family 17
 can: controller area network core (rev 20120528 abi 9)
 NET: Registered protocol family 29
@@ -307,23 +308,15 @@
 +3.3V_RUN: supplied by +3.3V_SYS
 +3.3V_AVDD_HDMI_AP_GATED: supplied by +3.3V_RUN
 +1.8V_RUN_CAM: supplied by +3.3V_RUN
-input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 +VDDIO_SDMMC3: supplied by +3.3V_RUN
 +1.8V_RUN_VPP_FUSE: supplied by +3.3V_RUN
 +VDD_LED: supplied by +VDD_MUX
 +5V_VDD_TS_SW: supplied by +5V_SYS
 +5V_USB_HS: supplied by +5V_SYS
 +5V_USB_SS: supplied by +5V_SYS
-mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
-mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
-mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
-mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 +3.3V_PANEL: supplied by +3.3V_RUN
 +3.3V_LP0: supplied by +3.3V_SYS
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
-elan_i2c 1-0015: failed to read report data: -121
 +5V_HDMI_CON: supplied by +5V_SYS
 usb3-0 supply vbus not found, using dummy regulator
 usb3-1 supply vbus not found, using dummy regulator
@@ -334,16 +327,23 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 +1.8V_VDDIO: voltage operation not allowed
 mmc1: Switching to 3.3V signalling voltage failed
+mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
+mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
+mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
+mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 mmc1: SDHCI controller on 700b0000.sdhci [700b0000.sdhci] using ADMA 64-bit
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc2: Unknown controller version (3). You may experience problems.
 mmc2: Invalid maximum block size, assuming 512 bytes
-elan_i2c 1-0015: invalid report id data (ff)
-mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
 mmc1: new high speed SDIO card at address 0001
+mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
+Console: switching to colour frame buffer device 170x48
+drm drm: fb0:  frame buffer device
 mmc2: new high speed SDHC card at address 0007
 mmcblk2: mmc2:0007 SD08G 7.42 GiB 
 GPT:Primary header alternate_lba != Alt. header my_lba
@@ -352,8 +352,6 @@
 GPT:3014591 != 15564799
 GPT: Use GNU Parted to correct GPT errors.
  mmcblk2: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
-Console: switching to colour frame buffer device 170x48
-drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
 as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:22 UTC (946684822)
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2016-06-20 23:27:06.636340878 -0700
+++ /tmp/foob	2016-06-20 23:27:06.640340881 -0700
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 #######################################################
-	 3.4 MiB/s
+	 3.5 MiB/s
 done
-Bytes transferred = 5569224 (54fac8 hex)
+Bytes transferred = 5570448 (54ff90 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.2 MiB/s
+	 3.5 MiB/s
 done
 Bytes transferred = 66901 (10555 hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x54fac8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x54ff90 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013554
@@ -141,11 +141,11 @@
 tegra30-i2s 70301100.i2s: DMA channels sourced from device 70300000.ahub
 max98090 0-0010: MAX98090 REVID=0x43
 max98090 0-0010: use default 2.8v micbias
-mmc0: new high speed MMC card at address 0001
 tegra-snd-max98090 sound: HiFi <-> 70301100.i2s mapping ok
-mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
 input: GoogleNyanBig Headphones as /devices/soc0/sound/sound/card1/input1
 input: GoogleNyanBig Mic Jack as /devices/soc0/sound/sound/card1/input2
+mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
 mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
 mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
@@ -160,7 +160,6 @@
 can: raw protocol (rev 20120528)
 can: broadcast manager protocol (rev 20160617 t)
 can: netlink gateway (rev 20130117) max_hops=1
-input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input3
 Bluetooth: RFCOMM socket layer initialized
 Bluetooth: RFCOMM ver 1.11
 Bluetooth: BNEP (Ethernet Emulation) ver 1.3
@@ -182,6 +181,7 @@
 +3.3V_LP0: supplied by +3.3V_SYS
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 +5V_HDMI_CON: supplied by +5V_SYS
+input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input3
 usb3-0 supply vbus not found, using dummy regulator
 usb3-1 supply vbus not found, using dummy regulator
 tegra-sor 54540000.sor: failed to probe output: -517
@@ -216,13 +216,13 @@
 mmc2: new high speed SDHC card at address 0007
 mmcblk2: mmc2:0007 SD08G 7.42 GiB 
 Console: switching to colour frame buffer device 170x48
+drm drm: fb0:  frame buffer device
 GPT:Primary header alternate_lba != Alt. header my_lba
 GPT:3014591 != 15564799
 GPT:Primary header thinks Alt. header is not at the end of the disk.
 GPT:3014591 != 15564799
 GPT: Use GNU Parted to correct GPT errors.
  mmcblk2: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
-drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input4
 as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:24 UTC (946684824)
@@ -246,7 +246,7 @@
 Freeing unused kernel memory: 1024K (c0d00000 - c0e00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.804656] usb 1-3: new high-speed USB device number 3 using tegra-xusb
+root@(none):/# [    5.770950] usb 1-3: new high-speed USB device number 3 using tegra-xusb
 asix 1-3:1.0 eth0: register 'asix' at usb-70090000.usb-3, ASIX AX88772 USB 2.0 Ethernet, c8:d7:19:90:fa:65
 usb 1-1: new high-speed USB device number 4 using tegra-xusb
 usb-storage 1-1:1.0: USB Mass Storage device detected
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2016-06-20 23:27:06.644340882 -0700
+++ /tmp/foob	2016-06-20 23:27:06.644340882 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1800547 us
+Bootloader-Cpu Init at (time stamp): 1801066 us
 
 Pinmux changes applied in kernel way
 
@@ -59,32 +59,32 @@
 The proc BoardInfo: 0x0176:0x0001:0x01:0x43:0x00
 FastbootAddAtag: setting ATAG (0)
 
-Board Id Read Time: 4 us
+Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1800520  us
-Bootloader (AosInit) time: 733470 us
-Bootloader Main Init time: 180230 us
-BlockDevMgrInit time: 157531 us
+Miscellaneous avp time: 1801040  us
+Bootloader (AosInit) time: 734253 us
+Bootloader Main Init time: 180103 us
+BlockDevMgrInit time: 157532 us
 PartMgrInit time: 8 us
-StorMgrInit time: 31 us
+StorMgrInit time: 30 us
 FsMgrInit time: 2 us
-Partition Table load: 77148 us 
-Display init time: 574453 us
-Kernel read time: 358 msec
-Load Kernel Size:   15511552 Bytes
-Bl Kernel ReadSpeed: 43328000 Bytes/sec
-Pre-Kernel Jump time: 1675053 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5754226 us
+Partition Table load: 76685 us 
+Display init time: 574691 us
+Kernel read time: 359 msec
+Load Kernel Size:   15509504 Bytes
+Bl Kernel ReadSpeed: 43201000 Bytes/sec
+Pre-Kernel Jump time: 1675257 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5796103 us
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-next-20160620+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Jun 20 00:06:14 PDT 2016
+Linux version 4.7.0-rc4-next-20160621+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Jun 20 23:21:45 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
 Memory limited to 1734MB
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
 cma: Reserved 16 MiB at 0x00000000eb400000
 /cpus/cpu@1: missing enable-method property
-percpu: Embedded 20 pages/cpu @ffff80006c59f000 s42880 r8192 d30848 u81920
+percpu: Embedded 21 pages/cpu @ffff80006c59e000 s47104 r8192 d30720 u86016
 Detected PIPT I-cache on CPU0
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 436968
 Kernel command line: tegraid=13.1.3.0.0 mem=1734M@2048M memtype=255 vpr=151M@3817M ddr_die=2048M@2048M section=256M commchip_id=0 usb_port_owner_info=0 lane_owner_info=0 emc_max_dvfs=0 touch_id=0@0 video=tegrafb no_console_suspend=1 console=ttyS0,115200n8 debug_uartport=lsport,0 earlyprintk=uart8250-32bit,0x70006000 usbcore.old_scheme_first=1 lp0_vec=0@0x0 tegra_fbmem=32899072@0xad012000 core_edp_mv=1150 core_edp_ma=4000 pmuboard=0x0176:0x0001:0x01:0x43:0x00 displayboard=0x0162:0x0001:0x00:0x42:0x00 power_supply=Adapter audio_codec=rt5640 board_info=0x0176:0x0001:0x01:0x43:0x00 tegraboot=sdmmc gpt gpt_sector=94207 modem_id=0 android.kerneltype=normal
@@ -92,15 +92,15 @@
 Dentry cache hash table entries: 262144 (order: 9, 2097152 bytes)
 Inode-cache hash table entries: 131072 (order: 8, 1048576 bytes)
 software IO TLB [mem 0xe5200000-0xe9200000] (64MB) mapped at [ffff800065200000-ffff8000691fffff]
-Memory: 1641560K/1775616K available (7656K kernel code, 681K rwdata, 3476K rodata, 960K init, 262K bss, 117672K reserved, 16384K cma-reserved)
+Memory: 1641560K/1775616K available (7656K kernel code, 679K rwdata, 3476K rodata, 960K init, 260K bss, 117672K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff0000087f0000   (  7616 KB)
     .rodata : 0xffff0000087f0000 - 0xffff000008b60000   (  3520 KB)
       .init : 0xffff000008b60000 - 0xffff000008c50000   (   960 KB)
-      .data : 0xffff000008c50000 - 0xffff000008cfa600   (   682 KB)
-       .bss : 0xffff000008cfa600 - 0xffff000008d3be38   (   263 KB)
+      .data : 0xffff000008c50000 - 0xffff000008cf9e00   (   680 KB)
+       .bss : 0xffff000008cf9e00 - 0xffff000008d3ae38   (   261 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -185,7 +185,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(1.083:1): initialized
+audit: type=2000 audit(1.091:1): initialized
 workingset: timestamp_bits=44 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-06-20 23:27:06.652340886 -0700
+++ /tmp/foob	2016-06-20 23:27:06.652340886 -0700
@@ -47,20 +47,20 @@
 	 #################################################################
 	 #################################################################
 	 #########################################
-	 905.3 KiB/s
+	 4.1 MiB/s
 done
-Bytes transferred = 7267360 (6ee420 hex)
+Bytes transferred = 7279344 (6f12f0 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 861.3 KiB/s
+	 4 MiB/s
 done
 Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x6ee420 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x6f12f0 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102b3
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-next-20160620-101088-g6ab9759 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jun 20 00:03:31 PDT 2016
+Linux version 4.7.0-rc4-next-20160621-101370-g10698aa (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jun 20 23:19:35 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -76,20 +76,20 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c141bb40, node_mem_map ef7fa000
+free_area_init_node: node 0, pgdat c141b840, node_mem_map ef7fa000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 65536 pages, LIFO batch:15
-percpu: Embedded 13 pages/cpu @ef7cb000 s23808 r8192 d21248 u53248
-pcpu-alloc: s23808 r8192 d21248 u53248 alloc=13*4096
+percpu: Embedded 14 pages/cpu @ef7c9000 s26048 r8192 d23104 u57344
+pcpu-alloc: s26048 r8192 d23104 u57344 alloc=14*4096
 pcpu-alloc: [0] 0 [0] 1 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 260608
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=11eb23c1-01 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 955940K/1048576K available (9530K kernel code, 1142K rwdata, 4508K rodata, 2048K init, 342K bss, 27100K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 955932K/1048576K available (9530K kernel code, 1142K rwdata, 4516K rodata, 2048K init, 342K bss, 27108K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -97,10 +97,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc10b5874   (15031 kB)
+      .text : 0xc0208000 - 0xc10b7b40   (15039 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141dba0   (1143 kB)
-       .bss : 0xc141f000 - 0xc1474918   ( 343 kB)
+      .data : 0xc1300000 - 0xc141d8a0   (1143 kB)
+       .bss : 0xc141f000 - 0xc1474a48   ( 343 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -322,29 +322,27 @@
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
+mmc0: host does not support reading read-only switch, assuming write-enable
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: hctosys: unable to read the hardware clock
-Waiting for root device PARTUUID=11eb23c1-01...
-mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
+rtc-em3027 2-0056: hctosys: unable to read the hardware clock
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
  mmcblk0: p1
+EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
-EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
-usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
-asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
+usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-06-20 23:27:06.660340891 -0700
+++ /tmp/foob	2016-06-20 23:27:06.660340891 -0700
@@ -45,20 +45,20 @@
 	 #################################################################
 	 #################################################################
 	 #######################################################
-	 904.3 KiB/s
+	 915 KiB/s
 done
-Bytes transferred = 5569224 (54fac8 hex)
+Bytes transferred = 5570448 (54ff90 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 861.3 KiB/s
+	 898.4 KiB/s
 done
 Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x54fac8 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x54ff90 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102b3
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-next-20160620-101088-g6ab9759 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jun 20 00:03:43 PDT 2016
+Linux version 4.7.0-rc4-next-20160621-101370-g10698aa (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jun 20 23:19:36 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -76,20 +76,20 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0e5bd80, node_mem_map ef7f9000
+free_area_init_node: node 0, pgdat c0e5bd00, node_mem_map ef7f9000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 65536 pages, LIFO batch:15
-percpu: Embedded 13 pages/cpu @ef7ca000 s23936 r8192 d21120 u53248
-pcpu-alloc: s23936 r8192 d21120 u53248 alloc=13*4096
+percpu: Embedded 14 pages/cpu @ef7c8000 s26048 r8192 d23104 u57344
+pcpu-alloc: s26048 r8192 d23104 u57344 alloc=14*4096
 pcpu-alloc: [0] 0 [0] 1 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 260608
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=11eb23c1-01 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 959752K/1048576K available (8441K kernel code, 374K rwdata, 2844K rodata, 1024K init, 366K bss, 23288K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 959744K/1048576K available (8441K kernel code, 374K rwdata, 2844K rodata, 1024K init, 366K bss, 23296K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -97,10 +97,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0c054e0   (12278 kB)
+      .text : 0xc0008000 - 0xc0c0553c   (12278 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0e5d920   ( 375 kB)
-       .bss : 0xc0e5f000 - 0xc0ebab14   ( 367 kB)
+      .data : 0xc0e00000 - 0xc0e5d8a0   ( 375 kB)
+       .bss : 0xc0e5f000 - 0xc0ebab04   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
@@ -307,11 +307,11 @@
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
+ mmcblk0: p1
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
  mmcblk1: p1 p2
- mmcblk0: p1
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
 NET: Registered protocol family 10
@@ -347,7 +347,7 @@
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:17.
+VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (c0d00000 - c0e00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2016-06-20 23:27:06.668340894 -0700
+++ /tmp/foob	2016-06-20 23:27:06.668340894 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 846368 us
-Basic modules init took 348541 us
+Tegraboot started after 766749 us
+Basic modules init took 348447 us
 Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14365 us
-Carveout took 252755 us
-CPU initialization took 92307 us
-Total time taken by TegraBoot 707968 us
+Next stage binary read took 14382 us
+Carveout took 252740 us
+CPU initialization took 92842 us
+Total time taken by TegraBoot 708411 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.683] LPDDR4 Training: Number of tables = 10
+64b[0001.605] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-13084160 bytes read in 752 ms (16.6 MiB/s)
+13082112 bytes read in 759 ms (16.4 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-39484 bytes read in 453 ms (85 KiB/s)
+39484 bytes read in 454 ms (84 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 496 ms (3.1 MiB/s)
+1627787 bytes read in 494 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-next-20160620+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Jun 20 00:06:14 PDT 2016
+Linux version 4.7.0-rc4-next-20160621+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Jun 20 23:21:45 PDT 2016
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -149,8 +149,8 @@
 psci: PSCIv0.2 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: MIGRATE_INFO_TYPE not supported.
-percpu: Embedded 20 pages/cpu @ffff8000bff51000 s42880 r8192 d30848 u81920
-pcpu-alloc: s42880 r8192 d30848 u81920 alloc=20*4096
+percpu: Embedded 21 pages/cpu @ffff8000bff4d000 s47104 r8192 d30720 u86016
+pcpu-alloc: s47104 r8192 d30720 u86016 alloc=21*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
@@ -166,15 +166,15 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffff80007a000000-ffff80007dffffff]
-Memory: 2979896K/3131392K available (7656K kernel code, 681K rwdata, 3476K rodata, 960K init, 262K bss, 135112K reserved, 16384K cma-reserved)
+Memory: 2979884K/3131392K available (7656K kernel code, 679K rwdata, 3476K rodata, 960K init, 260K bss, 135124K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff0000087f0000   (  7616 KB)
     .rodata : 0xffff0000087f0000 - 0xffff000008b60000   (  3520 KB)
       .init : 0xffff000008b60000 - 0xffff000008c50000   (   960 KB)
-      .data : 0xffff000008c50000 - 0xffff000008cfa600   (   682 KB)
-       .bss : 0xffff000008cfa600 - 0xffff000008d3be38   (   263 KB)
+      .data : 0xffff000008c50000 - 0xffff000008cf9e00   (   680 KB)
+       .bss : 0xffff000008cf9e00 - 0xffff000008d3ae38   (   261 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -264,7 +264,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.451:1): initialized
+audit: type=2000 audit(0.463:1): initialized
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2016-06-20 23:27:06.672340896 -0700
+++ /tmp/foob	2016-06-20 23:27:06.676340899 -0700
@@ -1,12 +1,8 @@
- bootconsole [uart0] enabled
-efi: Getting EFI parameters from FDT:
-efi: UEFI not found.
-cma: Reserved 16 MiB at 0x00000000f5400000
 psci: probing for conduit method from DT.
 psci: PSCIv1.0 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: MIGRATE_INFO_UP_CPU reported invalid physical ID (0x80000000)
-percpu: Embedded 20 pages/cpu @ffff8000bff4f000 s42880 r8192 d30848 u81920
+percpu: Embedded 21 pages/cpu @ffff8000bff4b000 s47104 r8192 d30720 u86016
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
 CPU features: enabling workaround for ARM erratum 834220
@@ -21,15 +17,15 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2834128K/2988032K available (7656K kernel code, 681K rwdata, 3476K rodata, 960K init, 262K bss, 137520K reserved, 16384K cma-reserved)
+Memory: 2834120K/2988032K available (7656K kernel code, 679K rwdata, 3476K rodata, 960K init, 260K bss, 137528K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff0000087f0000   (  7616 KB)
     .rodata : 0xffff0000087f0000 - 0xffff000008b60000   (  3520 KB)
       .init : 0xffff000008b60000 - 0xffff000008c50000   (   960 KB)
-      .data : 0xffff000008c50000 - 0xffff000008cfa600   (   682 KB)
-       .bss : 0xffff000008cfa600 - 0xffff000008d3be38   (   263 KB)
+      .data : 0xffff000008c50000 - 0xffff000008cf9e00   (   680 KB)
+       .bss : 0xffff000008cf9e00 - 0xffff000008d3ae38   (   261 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -58,7 +54,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0xefb0: expected 0x2004f400
+boot misc3 0xf0a0: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -71,7 +67,7 @@
 console [tty0] enabled
 bootconsole [uart0] disabled
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-next-20160620+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Jun 20 00:06:14 PDT 2016
+Linux version 4.7.0-rc4-next-20160621+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Jun 20 23:21:45 PDT 2016
 Boot CPU: AArch64 Processor [411fd071]
 earlycon: uart0 at MMIO 0x0000000070006000 (options '115200n8')
 bootconsole [uart0] enabled
@@ -82,7 +78,7 @@
 psci: PSCIv1.0 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: MIGRATE_INFO_UP_CPU reported invalid physical ID (0x80000000)
-percpu: Embedded 20 pages/cpu @ffff8000bff4f000 s42880 r8192 d30848 u81920
+percpu: Embedded 21 pages/cpu @ffff8000bff4b000 s47104 r8192 d30720 u86016
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
 CPU features: enabling workaround for ARM erratum 834220
@@ -97,15 +93,15 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2834128K/2988032K available (7656K kernel code, 681K rwdata, 3476K rodata, 960K init, 262K bss, 137520K reserved, 16384K cma-reserved)
+Memory: 2834120K/2988032K available (7656K kernel code, 679K rwdata, 3476K rodata, 960K init, 260K bss, 137528K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff0000087f0000   (  7616 KB)
     .rodata : 0xffff0000087f0000 - 0xffff000008b60000   (  3520 KB)
       .init : 0xffff000008b60000 - 0xffff000008c50000   (   960 KB)
-      .data : 0xffff000008c50000 - 0xffff000008cfa600   (   682 KB)
-       .bss : 0xffff000008cfa600 - 0xffff000008d3be38   (   263 KB)
+      .data : 0xffff000008c50000 - 0xffff000008cf9e00   (   680 KB)
+       .bss : 0xffff000008cf9e00 - 0xffff000008d3ae38   (   261 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -134,7 +130,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0xefb0: expected 0x2004f400
+boot misc3 0xf0a0: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -204,7 +200,7 @@
 RPC: Registered tcp transport module.
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 Unpacking initramfs...
-Freeing initrd memory: 2364K (ffff800009f21000 - ffff80000a170000)
+Freeing initrd memory: 2360K (ffff800009f21000 - ffff80000a16f000)
 kvm [1]: 8-bit VMID
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-06-20 23:27:06.680340901 -0700
+++ /tmp/foob	2016-06-20 23:27:06.684340902 -0700
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 #########################################
-	 3.3 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 7267360 (6ee420 hex)
+Bytes transferred = 7279344 (6f12f0 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.7 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 51511 (c937 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6ee420 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6f12f0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff936 ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-next-20160620-101088-g6ab9759 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jun 20 00:03:31 PDT 2016
+Linux version 4.7.0-rc4-next-20160621-101370-g10698aa (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jun 20 23:19:35 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,20 +68,20 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141bb40, node_mem_map eeffa000
+free_area_init_node: node 0, pgdat c141b840, node_mem_map eeffa000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 13 pages/cpu @eef91000 s23808 r8192 d21248 u53248
-pcpu-alloc: s23808 r8192 d21248 u53248 alloc=13*4096
+percpu: Embedded 14 pages/cpu @eef8d000 s26048 r8192 d23104 u57344
+pcpu-alloc: s26048 r8192 d23104 u57344 alloc=14*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=edbe1840-3633-c751-7993-b0ac834f81e0 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1995044K/2096128K available (9530K kernel code, 1142K rwdata, 4508K rodata, 2048K init, 342K bss, 35548K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1995028K/2096128K available (9530K kernel code, 1142K rwdata, 4516K rodata, 2048K init, 342K bss, 35564K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -89,10 +89,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc10b5874   (15031 kB)
+      .text : 0xc0208000 - 0xc10b7b40   (15039 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141dba0   (1143 kB)
-       .bss : 0xc141f000 - 0xc1474918   ( 343 kB)
+      .data : 0xc1300000 - 0xc141d8a0   (1143 kB)
+       .bss : 0xc141f000 - 0xc1474a48   ( 343 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -303,10 +303,10 @@
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
-mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
+tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 mmc0: error -110 whilst initialising SD card
+mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB
-tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 Primary GPT is invalid, using alternate GPT.
  mmcblk1: p1 p2 p3 p4 p5
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-06-20 23:27:06.688340904 -0700
+++ /tmp/foob	2016-06-20 23:27:06.692340907 -0700
@@ -39,18 +39,18 @@
 	 #######################################################
 	 3.2 MiB/s
 done
-Bytes transferred = 5569224 (54fac8 hex)
+Bytes transferred = 5570448 (54ff90 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 51511 (c937 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x54fac8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x54ff90 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff936 ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-next-20160620-101088-g6ab9759 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jun 20 00:03:43 PDT 2016
+Linux version 4.7.0-rc4-next-20160621-101370-g10698aa (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jun 20 23:19:36 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,20 +68,20 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0e5bd80, node_mem_map eeff9000
+free_area_init_node: node 0, pgdat c0e5bd00, node_mem_map eeff9000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 13 pages/cpu @eef90000 s23936 r8192 d21120 u53248
-pcpu-alloc: s23936 r8192 d21120 u53248 alloc=13*4096
+percpu: Embedded 14 pages/cpu @eef8c000 s26048 r8192 d23104 u57344
+pcpu-alloc: s26048 r8192 d23104 u57344 alloc=14*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=edbe1840-3633-c751-7993-b0ac834f81e0 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998856K/2096128K available (8441K kernel code, 374K rwdata, 2844K rodata, 1024K init, 366K bss, 31736K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998840K/2096128K available (8441K kernel code, 374K rwdata, 2844K rodata, 1024K init, 366K bss, 31752K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -89,10 +89,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0c054e0   (12278 kB)
+      .text : 0xc0008000 - 0xc0c0553c   (12278 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0e5d920   ( 375 kB)
-       .bss : 0xc0e5f000 - 0xc0ebab14   ( 367 kB)
+      .data : 0xc0e00000 - 0xc0e5d8a0   ( 375 kB)
+       .bss : 0xc0e5f000 - 0xc0ebab04   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -364,7 +364,7 @@
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
-tps65910-rtc tps65910-rtc: setting system clock to 2000-01-01 00:00:03 UTC (946684803)
+tps65910-rtc tps65910-rtc: setting system clock to 2000-01-01 00:00:04 UTC (946684804)
 vdd_sata,avdd_plle: disabling
 usb1_vbus: disabling
 ALSA device list:
@@ -376,7 +376,6 @@
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (c0d00000 - c0e00000)
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# 
\ No newline at end of file
+root@(none):/# [    8.691182] usb 2-1: new high-speed USB device number 2 using tegra-ehci
