Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-08-16 01:41:08.410456097 -0700
+++ /tmp/foob	2016-08-16 01:41:08.414456101 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc1-next-20160815-136279-g6442b07 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Aug 14 20:37:37 PDT 2016
+Linux version 4.8.0-rc2-next-20160816-136740-g0c9a343 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Aug 16 01:37:05 PDT 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -18,15 +18,15 @@
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
-Memory: 1000208K/1048576K available (7932K kernel code, 797K rwdata, 3712K rodata, 960K init, 275K bss, 31984K reserved, 16384K cma-reserved)
+Memory: 1000200K/1048576K available (7932K kernel code, 806K rwdata, 3720K rodata, 960K init, 273K bss, 31992K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008840000   (  7936 KB)
     .rodata : 0xffff000008840000 - 0xffff000008bf0000   (  3776 KB)
       .init : 0xffff000008bf0000 - 0xffff000008ce0000   (   960 KB)
-      .data : 0xffff000008ce0000 - 0xffff000008da7400   (   797 KB)
-       .bss : 0xffff000008da7400 - 0xffff000008dec0b8   (   276 KB)
+      .data : 0xffff000008ce0000 - 0xffff000008da9a00   (   807 KB)
+       .bss : 0xffff000008da9a00 - 0xffff000008dee0b8   (   274 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -100,7 +100,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.748:1): initialized
+audit: type=2000 audit(0.788:1): initialized
 workingset: timestamp_bits=46 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -165,7 +165,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2016-08-15 03:38:14 UTC (1471232294)
+rtc-pl031 9010000.pl031: setting system clock to 2016-08-16 08:37:42 UTC (1471336662)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-08-16 01:41:08.418456105 -0700
+++ /tmp/foob	2016-08-16 01:41:08.418456105 -0700
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 ##############################
-	 3.1 MiB/s
+	 3 MiB/s
 done
-Bytes transferred = 7108752 (6c7890 hex)
+Bytes transferred = 7106848 (6c7120 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.8 MiB/s
+	 2.6 MiB/s
 done
 Bytes transferred = 34956 (888c hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6c7890 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6c7120 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff88b ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc1-next-20160815-111846-g6442b07 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Aug 14 20:33:39 PDT 2016
+Linux version 4.8.0-rc2-next-20160816-112307-g0c9a343 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Aug 16 01:33:24 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141db80, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c141bbc0, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1995092K/2096128K available (9216K kernel code, 1150K rwdata, 3980K rodata, 2048K init, 343K bss, 35500K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1995100K/2096128K available (9216K kernel code, 1142K rwdata, 3980K rodata, 2048K init, 342K bss, 35492K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,8 +92,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141fa20   (1151 kB)
-       .bss : 0xc1421000 - 0xc1476cc8   ( 344 kB)
+      .data : 0xc1300000 - 0xc141da60   (1143 kB)
+       .bss : 0xc141f000 - 0xc1474bc8   ( 343 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-08-16 01:41:08.422456107 -0700
+++ /tmp/foob	2016-08-16 01:41:08.426456111 -0700
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra114 (Dalmore) # 
 Tegra114 (Dalmore) # usb reset
 (Re)start USB...
@@ -37,9 +37,9 @@
 	 #################################################################
 	 #################################################################
 	 #############################################
-	 3 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 5417512 (52aa28 hex)
+Bytes transferred = 5417120 (52a8a0 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 34956 (888c hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x52aa28 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x52a8a0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff88b ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc1-next-20160815-111846-g6442b07 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Aug 14 20:33:43 PDT 2016
+Linux version 4.8.0-rc2-next-20160816-112307-g0c9a343 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Aug 16 01:33:10 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0d5c440, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0d5c240, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999936K/2096128K available (8192K kernel code, 375K rwdata, 2436K rodata, 1024K init, 366K bss, 30656K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1999936K/2096128K available (8192K kernel code, 375K rwdata, 2440K rodata, 1024K init, 366K bss, 30656K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,8 +92,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d5dec0   ( 376 kB)
-       .bss : 0xc0d5f000 - 0xc0dbab18   ( 367 kB)
+      .data : 0xc0d00000 - 0xc0d5dcc0   ( 376 kB)
+       .bss : 0xc0d5f000 - 0xc0dbaa98   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -260,8 +260,8 @@
 usbhid: USB HID core driver
 mmc0: new high speed SDHC card at address aaaa
 mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
-mmc1: new high speed MMC card at address 0001
  mmcblk0: p1 p2 p3
+mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-08-16 01:41:08.430456115 -0700
+++ /tmp/foob	2016-08-16 01:41:08.430456115 -0700
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+MMC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -34,20 +34,20 @@
 	 #################################################################
 	 #################################################################
 	 ##############################
-	 3.9 MiB/s
+	 4 MiB/s
 done
-Bytes transferred = 7108752 (6c7890 hex)
+Bytes transferred = 7106848 (6c7120 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 4 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 66621 (1043d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6c7890 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x6c7120 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301343c
@@ -55,7 +55,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc1-next-20160815-111846-g6442b07 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Aug 14 20:33:39 PDT 2016
+Linux version 4.8.0-rc2-next-20160816-112307-g0c9a343 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Aug 16 01:33:24 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -64,7 +64,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141db80, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c141bbc0, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -77,7 +77,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1994972K/2096128K available (9216K kernel code, 1150K rwdata, 3980K rodata, 2048K init, 343K bss, 35620K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1994980K/2096128K available (9216K kernel code, 1142K rwdata, 3980K rodata, 2048K init, 342K bss, 35612K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -87,8 +87,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141fa20   (1151 kB)
-       .bss : 0xc1421000 - 0xc1476cc8   ( 344 kB)
+      .data : 0xc1300000 - 0xc141da60   (1143 kB)
+       .bss : 0xc141f000 - 0xc1474bc8   ( 343 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -294,12 +294,6 @@
 can: raw protocol (rev 20120528)
 can: broadcast manager protocol (rev 20160617 t)
 can: netlink gateway (rev 20130117) max_hops=1
-mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
- mmcblk0: p1
 Key type dns_resolver registered
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
@@ -312,6 +306,12 @@
 +5V_SATA: supplied by +5V_SYS
 +12V_SATA: supplied by +VDD_MUX
 usb2-0 supply vbus not found, using dummy regulator
+mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+ mmcblk0: p1
 usb2-1 supply vbus not found, using dummy regulator
 usb3-0 supply vbus not found, using dummy regulator
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
@@ -374,8 +374,8 @@
 pci 0000:01:00.0: [8086:4232] type 00 class 0x028000
 pci 0000:01:00.0: reg 0x10: [mem 0x00000000-0x00001fff 64bit]
 pci 0000:01:00.0: PME# supported from D0 D3hot D3cold
-ata1: SATA link down (SStatus 0 SControl 300)
 PCI: bus1: Fast back to back transfers disabled
+ata1: SATA link down (SStatus 0 SControl 300)
 pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
 pci 0000:02:00.0: [10ec:8168] type 00 class 0x020000
 pci 0000:02:00.0: reg 0x10: [io  0x0000-0x00ff]
@@ -411,23 +411,22 @@
 pcie_pme 0000:00:02.0:pcie001: service driver pcie_pme loaded
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
+mmc1: new high speed SDHC card at address aaaa
 r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0a4d000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 393
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
+mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
-drm drm: No connectors reported connected with modes
-[drm] Cannot find any crtc or sizes - going 1024x768
-mmc1: new high speed SDHC card at address aaaa
-mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
  mmcblk1: p1 p2 p3
-Console: switching to colour frame buffer device 128x48
+tegra-hdmi 54280000.hdmi: cannot set audio to 48000 Hz at 297000000 Hz pixel clock
+Console: switching to colour frame buffer device 512x135
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 of: dev_pm_opp_of_cpumask_add_table: couldn't find opp table for cpu:0, -19
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:12 UTC (946684812)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:59 UTC (946684979)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-08-16 01:41:08.438456121 -0700
+++ /tmp/foob	2016-08-16 01:41:08.438456121 -0700
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MMC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -32,20 +32,20 @@
 	 #################################################################
 	 #################################################################
 	 #############################################
-	 3.9 MiB/s
+	 915 KiB/s
 done
-Bytes transferred = 5417512 (52aa28 hex)
+Bytes transferred = 5417120 (52a8a0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.7 MiB/s
+	 916 KiB/s
 done
 Bytes transferred = 66621 (1043d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x52aa28 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x52a8a0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301343c
@@ -54,7 +54,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc1-next-20160815-111846-g6442b07 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Aug 14 20:33:43 PDT 2016
+Linux version 4.8.0-rc2-next-20160816-112307-g0c9a343 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Aug 16 01:33:10 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -64,7 +64,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0d5c440, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0d5c240, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -77,7 +77,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999816K/2096128K available (8192K kernel code, 375K rwdata, 2436K rodata, 1024K init, 366K bss, 30776K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1999816K/2096128K available (8192K kernel code, 375K rwdata, 2440K rodata, 1024K init, 366K bss, 30776K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -87,8 +87,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d5dec0   ( 376 kB)
-       .bss : 0xc0d5f000 - 0xc0dbab18   ( 367 kB)
+      .data : 0xc0d00000 - 0xc0d5dcc0   ( 376 kB)
+       .bss : 0xc0d5f000 - 0xc0dbaa98   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -222,7 +222,6 @@
 +VDDIO_SDMMC3: failed to get the current voltage(-517)
 as3722-regulator as3722-regulator: regulator 13 register failed -517
 tegra-ahci 70027000.sata: couldn't get PHY in node sata: -517
-random: fast init done
 m25p80 spi32765.0: w25q32dw (4096 Kbytes)
 libphy: Fixed MDIO Bus: probed
 CAN device driver interface
@@ -254,6 +253,7 @@
 usbcore: registered new interface driver cdc_wdm
 usbcore: registered new interface driver usb-storage
 mousedev: PS/2 mouse device common for all mice
+random: fast init done
 as3722-rtc as3722-rtc: rtc core: registered as3722-rtc as rtc0
 as3722-rtc as3722-rtc: RTC interrupt 375
 tegra_rtc 7000e000.rtc: rtc core: registered 7000e000.rtc as rtc1
@@ -370,8 +370,8 @@
 mmc1: Unknown controller version (3). You may experience problems.
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
-tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
 ata1: SATA link down (SStatus 0 SControl 300)
+tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
 tegra-pcie 1003000.pcie-controller: probing port 0, using 2 lanes
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
 tegra-pcie 1003000.pcie-controller: probing port 1, using 1 lanes
@@ -407,10 +407,10 @@
 pci 0000:00:02.0: BAR 8: assigned [mem 0x13100000-0x131fffff]
 pci 0000:00:02.0: BAR 9: assigned [mem 0x20000000-0x200fffff 64bit pref]
 pci 0000:00:02.0: BAR 7: assigned [io  0x1000-0x1fff]
- mmcblk1: p1 p2 p3
 pci 0000:01:00.0: BAR 0: assigned [mem 0x13000000-0x13001fff 64bit]
 pci 0000:00:01.0: PCI bridge to [bus 01]
 pci 0000:00:01.0:   bridge window [mem 0x13000000-0x130fffff]
+ mmcblk1: p1 p2 p3
 pci 0000:02:00.0: BAR 4: assigned [mem 0x20000000-0x20003fff 64bit pref]
 pci 0000:02:00.0: BAR 2: assigned [mem 0x13100000-0x13100fff 64bit]
 pci 0000:02:00.0: BAR 0: assigned [io  0x1000-0x10ff]
@@ -442,7 +442,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:05 UTC (946684925)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:21 UTC (946684821)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-08-16 01:41:08.442456125 -0700
+++ /tmp/foob	2016-08-16 01:41:08.446456130 -0700
@@ -41,20 +41,20 @@
 	 #################################################################
 	 #################################################################
 	 ##############################
-	 3.2 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 7108752 (6c7890 hex)
+Bytes transferred = 7106848 (6c7120 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.4 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 66821 (10505 hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6c7890 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x6c7120 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013504
@@ -62,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc1-next-20160815-111846-g6442b07 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Aug 14 20:33:39 PDT 2016
+Linux version 4.8.0-rc2-next-20160816-112307-g0c9a343 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Aug 16 01:33:24 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -72,7 +72,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c141db80, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c141bbc0, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -85,7 +85,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1995992K/2097148K available (9216K kernel code, 1150K rwdata, 3980K rodata, 2048K init, 343K bss, 35620K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1996000K/2097148K available (9216K kernel code, 1142K rwdata, 3980K rodata, 2048K init, 342K bss, 35612K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -95,8 +95,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141fa20   (1151 kB)
-       .bss : 0xc1421000 - 0xc1476cc8   ( 344 kB)
+      .data : 0xc1300000 - 0xc141da60   (1143 kB)
+       .bss : 0xc141f000 - 0xc1474bc8   ( 343 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -312,8 +312,8 @@
 +5V_VDD_TS_SW: supplied by +5V_SYS
 +5V_USB_HS: supplied by +5V_SYS
 +5V_USB_SS: supplied by +5V_SYS
-mmc0: new high speed MMC card at address 0001
 +3.3V_PANEL: supplied by +3.3V_RUN
+mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
 mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2016-08-16 01:41:08.450456133 -0700
+++ /tmp/foob	2016-08-16 01:41:08.450456133 -0700
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -41,18 +41,18 @@
 	 #############################################
 	 3.4 MiB/s
 done
-Bytes transferred = 5417512 (52aa28 hex)
+Bytes transferred = 5417120 (52a8a0 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.2 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 66821 (10505 hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x52aa28 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x52a8a0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013504
@@ -172,8 +172,8 @@
 +3.3V_AVDD_HDMI_AP_GATED: supplied by +3.3V_RUN
 +1.8V_RUN_CAM: supplied by +3.3V_RUN
 +VDDIO_SDMMC3: supplied by +3.3V_RUN
-input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input3
 +1.8V_RUN_VPP_FUSE: supplied by +3.3V_RUN
+input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input3
 +VDD_LED: supplied by +VDD_MUX
 +5V_VDD_TS_SW: supplied by +5V_SYS
 +5V_USB_HS: supplied by +5V_SYS
@@ -213,17 +213,18 @@
 [drm] No driver support for vblank timestamp query.
 usb 1-2: new high-speed USB device number 2 using tegra-xusb
 mmc2: new high speed SDHC card at address 0007
-Console: switching to colour frame buffer device 170x48
 mmcblk2: mmc2:0007 SD08G 7.42 GiB 
-drm drm: fb0:  frame buffer device
+Console: switching to colour frame buffer device 170x48
 GPT:Primary header alternate_lba != Alt. header my_lba
 GPT:3014591 != 15564799
 GPT:Primary header thinks Alt. header is not at the end of the disk.
 GPT:3014591 != 15564799
 GPT: Use GNU Parted to correct GPT errors.
+ mmcblk2: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
+drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input4
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:24 UTC (946684824)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:23 UTC (946684823)
 +1.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
 +1.05V_RUN_CAM_REAR: disabling
@@ -234,19 +235,18 @@
 ALSA device list:
   #0: tegra-hda at 0x70038000 irq 341
   #1: GoogleNyanBig
-[ 3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 EXT4-fs (mmcblk0p3): couldn't mount as ext3 due to feature incompatibilities
 uvcvideo: Found UVC 1.00 device HD WebCam (1bcf:2c67)
 input: HD WebCam as /devices/soc0/70090000.usb/usb1/1-2/1-2:1.0/input/input5
+usb 1-3: new high-speed USB device number 3 using tegra-xusb
 EXT4-fs (mmcblk0p3): recovery complete
 EXT4-fs (mmcblk0p3): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:3.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (c0c00000 - c0d00000)
-usb 1-3: new high-speed USB device number 3 using tegra-xusb
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    6.897534] asix 1-3:1.0 eth0: register 'asix' at usb-70090000.usb-3, ASIX AX88772 USB 2.0 Ethernet, c8:d7:19:90:fa:65
+root@(none):/# [    6.887660] asix 1-3:1.0 eth0: register 'asix' at usb-70090000.usb-3, ASIX AX88772 USB 2.0 Ethernet, c8:d7:19:90:fa:65
 usb 1-1: new high-speed USB device number 4 using tegra-xusb
 usb-storage 1-1:1.0: USB Mass Storage device detected
 scsi host0: usb-storage 1-1:1.0
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2016-08-16 01:41:08.458456139 -0700
+++ /tmp/foob	2016-08-16 01:41:08.458456139 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1800711 us
+Bootloader-Cpu Init at (time stamp): 1801334 us
 
 Pinmux changes applied in kernel way
 
@@ -62,22 +62,22 @@
 Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1800685  us
-Bootloader (AosInit) time: 734235 us
-Bootloader Main Init time: 180087 us
-BlockDevMgrInit time: 157539 us
+Miscellaneous avp time: 1801308  us
+Bootloader (AosInit) time: 733465 us
+Bootloader Main Init time: 180055 us
+BlockDevMgrInit time: 157568 us
 PartMgrInit time: 8 us
 StorMgrInit time: 30 us
 FsMgrInit time: 2 us
-Partition Table load: 76754 us 
-Display init time: 574676 us
-Kernel read time: 373 msec
-Load Kernel Size:   16220160 Bytes
-Bl Kernel ReadSpeed: 43485000 Bytes/sec
-Pre-Kernel Jump time: 1729514 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5829627 us
+Partition Table load: 76980 us 
+Display init time: 574434 us
+Kernel read time: 375 msec
+Load Kernel Size:   16230400 Bytes
+Bl Kernel ReadSpeed: 43281000 Bytes/sec
+Pre-Kernel Jump time: 1729231 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5828183 us
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc1-next-20160815-136279-g6442b07 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Aug 14 20:37:37 PDT 2016
+Linux version 4.8.0-rc2-next-20160816-136740-g0c9a343 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Aug 16 01:37:05 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
 Memory limited to 1734MB
 efi: Getting EFI parameters from FDT:
@@ -91,15 +91,15 @@
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 Dentry cache hash table entries: 262144 (order: 9, 2097152 bytes)
 Inode-cache hash table entries: 131072 (order: 8, 1048576 bytes)
-Memory: 1706800K/1775616K available (7932K kernel code, 797K rwdata, 3712K rodata, 960K init, 275K bss, 52432K reserved, 16384K cma-reserved)
+Memory: 1706792K/1775616K available (7932K kernel code, 806K rwdata, 3720K rodata, 960K init, 273K bss, 52440K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008840000   (  7936 KB)
     .rodata : 0xffff000008840000 - 0xffff000008bf0000   (  3776 KB)
       .init : 0xffff000008bf0000 - 0xffff000008ce0000   (   960 KB)
-      .data : 0xffff000008ce0000 - 0xffff000008da7400   (   797 KB)
-       .bss : 0xffff000008da7400 - 0xffff000008dec0b8   (   276 KB)
+      .data : 0xffff000008ce0000 - 0xffff000008da9a00   (   807 KB)
+       .bss : 0xffff000008da9a00 - 0xffff000008dee0b8   (   274 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -249,9 +249,11 @@
 Key type dns_resolver registered
 mmc0: new high speed MMC card at address 0001
 registered taskstats version 1
-+VDD_LED: supplied by +VDD_MUX
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
++VDD_LED: supplied by +VDD_MUX
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
 +5V_USB_HS: supplied by +5V_SYS
 +5V_USB_SS: supplied by +5V_SYS
 +5V_HDMI_CON: supplied by +5V_SYS
@@ -270,8 +272,6 @@
 ALSA device list:
   No soundcards found.
 Freeing unused kernel memory: 960K (ffff800000bf0000 - ffff800000ce0000)
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-08-16 01:41:08.462456143 -0700
+++ /tmp/foob	2016-08-16 01:41:08.466456148 -0700
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -47,9 +47,9 @@
 	 #################################################################
 	 #################################################################
 	 ##############################
-	 914.1 KiB/s
+	 905.3 KiB/s
 done
-Bytes transferred = 7108752 (6c7890 hex)
+Bytes transferred = 7106848 (6c7120 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -60,7 +60,7 @@
 done
 Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x6c7890 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x6c7120 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102b3
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc1-next-20160815-111846-g6442b07 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Aug 14 20:33:39 PDT 2016
+Linux version 4.8.0-rc2-next-20160816-112307-g0c9a343 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Aug 16 01:33:24 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: Compulab TrimSlice board
@@ -76,7 +76,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c141db80, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c141bbc0, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -89,7 +89,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 955916K/1048576K available (9216K kernel code, 1150K rwdata, 3980K rodata, 2048K init, 343K bss, 27124K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 955924K/1048576K available (9216K kernel code, 1142K rwdata, 3980K rodata, 2048K init, 342K bss, 27116K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -99,8 +99,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141fa20   (1151 kB)
-       .bss : 0xc1421000 - 0xc1476cc8   ( 344 kB)
+      .data : 0xc1300000 - 0xc141da60   (1143 kB)
+       .bss : 0xc141f000 - 0xc1474bc8   ( 343 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -321,14 +321,14 @@
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
-tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 mmc0: host does not support reading read-only switch, assuming write-enable
+tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
-mmc0: new high speed SDHC card at address b368
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
+mmc0: new high speed SDHC card at address b368
+rtc-em3027 2-0056: setting system clock to 2015-02-05 16:08:44 UTC (1423152524)
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
-rtc-em3027 2-0056: setting system clock to 2015-02-05 15:51:55 UTC (1423151515)
  mmcblk0: p1
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 mmc1: new high speed SD card at address b368
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-08-16 01:41:08.470456151 -0700
+++ /tmp/foob	2016-08-16 01:41:08.470456151 -0700
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -45,9 +45,9 @@
 	 #################################################################
 	 #################################################################
 	 #############################################
-	 901.4 KiB/s
+	 916 KiB/s
 done
-Bytes transferred = 5417512 (52aa28 hex)
+Bytes transferred = 5417120 (52a8a0 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x52aa28 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x52a8a0 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102b3
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc1-next-20160815-111846-g6442b07 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Aug 14 20:33:43 PDT 2016
+Linux version 4.8.0-rc2-next-20160816-112307-g0c9a343 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Aug 16 01:33:10 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: Compulab TrimSlice board
@@ -76,7 +76,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0d5c440, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0d5c240, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -89,7 +89,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 960760K/1048576K available (8192K kernel code, 375K rwdata, 2436K rodata, 1024K init, 366K bss, 22280K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 960760K/1048576K available (8192K kernel code, 375K rwdata, 2440K rodata, 1024K init, 366K bss, 22280K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -99,8 +99,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d5dec0   ( 376 kB)
-       .bss : 0xc0d5f000 - 0xc0dbab18   ( 367 kB)
+      .data : 0xc0d00000 - 0xc0d5dcc0   ( 376 kB)
+       .bss : 0xc0d5f000 - 0xc0dbaa98   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
@@ -303,13 +303,13 @@
 tlv320aic23-codec 2-001a: ASoC: Failed to add route RLINEIN -> NULL -> Line Input
 tlv320aic23-codec 2-001a: Control not supported for path MICIN -> [NULL] -> Mic Input
 tlv320aic23-codec 2-001a: ASoC: no dapm match for MICIN --> NULL --> Mic Input
-tlv320aic23-codec 2-001a: ASoC: Failed to add route MICIN -> NULL -> Mic Input
-tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
  mmcblk0: p1
+tlv320aic23-codec 2-001a: ASoC: Failed to add route MICIN -> NULL -> Mic Input
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
+tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
@@ -342,7 +342,7 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2015-02-05 15:53:29 UTC (1423151609)
+rtc-em3027 2-0056: setting system clock to 2015-02-05 16:07:12 UTC (1423152432)
 ALSA device list:
   #0: tegra-trimslice
 asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2016-08-16 01:41:08.474456154 -0700
+++ /tmp/foob	2016-08-16 01:41:08.478456157 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 726090 us
-Basic modules init took 348643 us
+Tegraboot started after 771374 us
+Basic modules init took 350063 us
 Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14371 us
-Carveout took 252725 us
-CPU initialization took 92333 us
-Total time taken by TegraBoot 708072 us
+Next stage binary read took 14366 us
+Carveout took 252754 us
+CPU initialization took 92309 us
+Total time taken by TegraBoot 709492 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.565] LPDDR4 Training: Number of tables = 10
+64b[0001.611] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-13792256 bytes read in 769 ms (17.1 MiB/s)
+13801984 bytes read in 768 ms (17.1 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-41182 bytes read in 453 ms (87.9 KiB/s)
+41182 bytes read in 454 ms (87.9 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 495 ms (3.1 MiB/s)
+1627787 bytes read in 502 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc1-next-20160815-136279-g6442b07 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Aug 14 20:37:37 PDT 2016
+Linux version 4.8.0-rc2-next-20160816-136740-g0c9a343 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Aug 16 01:37:05 PDT 2016
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -166,15 +166,15 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffff80007a000000-ffff80007dffffff]
-Memory: 2979160K/3131392K available (7932K kernel code, 797K rwdata, 3712K rodata, 960K init, 275K bss, 135848K reserved, 16384K cma-reserved)
+Memory: 2979152K/3131392K available (7932K kernel code, 806K rwdata, 3720K rodata, 960K init, 273K bss, 135856K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008840000   (  7936 KB)
     .rodata : 0xffff000008840000 - 0xffff000008bf0000   (  3776 KB)
       .init : 0xffff000008bf0000 - 0xffff000008ce0000   (   960 KB)
-      .data : 0xffff000008ce0000 - 0xffff000008da7400   (   797 KB)
-       .bss : 0xffff000008da7400 - 0xffff000008dec0b8   (   276 KB)
+      .data : 0xffff000008ce0000 - 0xffff000008da9a00   (   807 KB)
+       .bss : 0xffff000008da9a00 - 0xffff000008dee0b8   (   274 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -257,7 +257,7 @@
 RPC: Registered tcp transport module.
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 PCI: CLS 0 bytes, default 128
-kvm [1]: IDMAP page: 8082d000
+kvm [1]: IDMAP page: 8082f000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: 8-bit VMID
 kvm [1]: Hyp mode initialized successfully
@@ -266,7 +266,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.515:1): initialized
+audit: type=2000 audit(0.531:1): initialized
 workingset: timestamp_bits=46 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -345,6 +345,6 @@
 devtmpfs: mounted
 Freeing unused kernel memory: 960K (ffff800000bf0000 - ffff800000ce0000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
-bash: no job contr[    1.389273] random: fast init done
+bash: no job contr[    1.389239] random: fast init done
 ol in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2016-08-16 01:41:08.482456160 -0700
+++ /tmp/foob	2016-08-16 01:41:08.486456164 -0700
@@ -1,4 +1,5 @@
-enabled
+ earlycon: uart0 at MMIO 0x0000000070006000 (options '115200n8')
+bootconsole [uart0] enabled
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
 cma: Reserved 16 MiB at 0x00000000f5400000
@@ -21,15 +22,15 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2833356K/2988032K available (7932K kernel code, 797K rwdata, 3712K rodata, 960K init, 275K bss, 138292K reserved, 16384K cma-reserved)
+Memory: 2833344K/2988032K available (7932K kernel code, 806K rwdata, 3720K rodata, 960K init, 273K bss, 138304K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff000008840000   (  7936 KB)
     .rodata : 0xffff000008840000 - 0xffff000008bf0000   (  3776 KB)
       .init : 0xffff000008bf0000 - 0xffff000008ce0000   (   960 KB)
-      .data : 0xffff000008ce0000 - 0xffff000008da7400   (   797 KB)
-       .bss : 0xffff000008da7400 - 0xffff000008dec0b8   (   276 KB)
+      .data : 0xffff000008ce0000 - 0xffff000008da9a00   (   807 KB)
+       .bss : 0xffff000008da9a00 - 0xffff000008dee0b8   (   274 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -58,7 +59,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x58b0: expected 0x2004f400
+boot misc3 0x7400: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -132,8 +133,8 @@
 RPC: Registered tcp transport module.
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 Unpacking initramfs...
-Freeing initrd memory: 2360K (ffff800009fd0000 - ffff80000a21e000)
-kvm [1]: IDMAP page: 8082d000
+Freeing initrd memory: 2364K (ffff800009fd2000 - ffff80000a221000)
+kvm [1]: IDMAP page: 8082f000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: 8-bit VMID
 kvm [1]: Hyp mode initialized successfully
@@ -142,7 +143,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.659:1): initialized
+audit: type=2000 audit(0.695:1): initialized
 workingset: timestamp_bits=46 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -216,7 +217,7 @@
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
 mmc0: new high speed MMC card at address 0001
-max77686-rtc max77620-rtc: setting system clock to 2000-05-26 09:07:45 UTC (959332065)
+max77686-rtc max77620-rtc: setting system clock to 2000-05-27 14:07:12 UTC (959436432)
 mmcblk0: mmc0:0001 BGND3R 29.1 GiB 
 mmcblk0boot0: mmc0:0001 BGND3R partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-08-16 01:41:08.490456169 -0700
+++ /tmp/foob	2016-08-16 01:41:08.490456169 -0700
@@ -39,9 +39,9 @@
 	 #################################################################
 	 #################################################################
 	 ##############################
-	 3 MiB/s
+	 2.9 MiB/s
 done
-Bytes transferred = 7108752 (6c7890 hex)
+Bytes transferred = 7106848 (6c7120 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -52,7 +52,7 @@
 done
 Bytes transferred = 51572 (c974 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6c7890 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6c7120 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff973 ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc1-next-20160815-111846-g6442b07 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Aug 14 20:33:39 PDT 2016
+Linux version 4.8.0-rc2-next-20160816-112307-g0c9a343 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Aug 16 01:33:24 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,7 +68,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141db80, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c141bbc0, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -81,7 +81,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1995012K/2096128K available (9216K kernel code, 1150K rwdata, 3980K rodata, 2048K init, 343K bss, 35580K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1995020K/2096128K available (9216K kernel code, 1142K rwdata, 3980K rodata, 2048K init, 342K bss, 35572K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -91,8 +91,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141fa20   (1151 kB)
-       .bss : 0xc1421000 - 0xc1476cc8   ( 344 kB)
+      .data : 0xc1300000 - 0xc141da60   (1143 kB)
+       .bss : 0xc141f000 - 0xc1474bc8   ( 343 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -299,18 +299,18 @@
 +VDD_5V_HDMI: supplied by sys_3v3,vdd_3v3_alw
 sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
 mmc1: new high speed MMC card at address 0001
-tegra-pcie 3000.pcie-controller: 2x3 configuration
-tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
 mmcblk1: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
-mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB
 mmc0: new ultra high speed DDR50 SDHC card at address aaaa
+mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB
 mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
 Primary GPT is invalid, using alternate GPT.
  mmcblk1: p1 p2 p3 p4 p5
-tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
  mmcblk0: p1 p2 p3
+tegra-pcie 3000.pcie-controller: 2x3 configuration
+tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
+tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -381,4 +381,4 @@
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.853958] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.923985] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-08-16 01:41:08.494456172 -0700
+++ /tmp/foob	2016-08-16 01:41:08.498456175 -0700
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 #############################################
-	 2.9 MiB/s
+	 3.1 MiB/s
 done
-Bytes transferred = 5417512 (52aa28 hex)
+Bytes transferred = 5417120 (52a8a0 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.7 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 51572 (c974 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x52aa28 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x52a8a0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff973 ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc1-next-20160815-111846-g6442b07 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Aug 14 20:33:43 PDT 2016
+Linux version 4.8.0-rc2-next-20160816-112307-g0c9a343 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Aug 16 01:33:10 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,7 +68,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0d5c440, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0d5c240, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -81,7 +81,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999856K/2096128K available (8192K kernel code, 375K rwdata, 2436K rodata, 1024K init, 366K bss, 30736K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1999856K/2096128K available (8192K kernel code, 375K rwdata, 2440K rodata, 1024K init, 366K bss, 30736K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -91,8 +91,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d5dec0   ( 376 kB)
-       .bss : 0xc0d5f000 - 0xc0dbab18   ( 367 kB)
+      .data : 0xc0d00000 - 0xc0d5dcc0   ( 376 kB)
+       .bss : 0xc0d5f000 - 0xc0dbaa98   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -382,5 +382,4 @@
 Freeing unused kernel memory: 1024K (c0c00000 - c0d00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
-root@(none):/# 
\ No newline at end of file
+root@(none):/# [    7.740967] usb 2-1: new high-speed USB device number 2 using tegra-ehci
