Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-10-04 22:58:03.289483559 -0700
+++ /tmp/foob	2016-10-04 22:58:03.293483567 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-next-20161004-126151-gc7d3b91 (pwalmsley@sellafield) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Oct 3 21:22:51 PDT 2016
+Linux version 4.8.0-next-20161005-126395-g0a766d8 (pwalmsley@sellafield) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Oct 4 22:52:13 PDT 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -18,7 +18,7 @@
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
-Memory: 999572K/1048576K available (8252K kernel code, 852K rwdata, 3900K rodata, 1024K init, 283K bss, 32620K reserved, 16384K cma-reserved)
+Memory: 999572K/1048576K available (8252K kernel code, 852K rwdata, 3904K rodata, 1024K init, 283K bss, 32620K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
@@ -100,7 +100,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(1.068:1): initialized
+audit: type=2000 audit(1.132:1): initialized
 workingset: timestamp_bits=46 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -167,7 +167,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2016-10-04 04:23:33 UTC (1475555013)
+rtc-pl031 9010000.pl031: setting system clock to 2016-10-05 05:52:47 UTC (1475646767)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-10-04 22:58:03.305483590 -0700
+++ /tmp/foob	2016-10-04 22:58:03.309483598 -0700
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra114 (Dalmore) # 
 Tegra114 (Dalmore) # usb reset
 (Re)start USB...
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 ###############################################
-	 3.2 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 5450600 (532b68 hex)
+Bytes transferred = 5449824 (532860 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x532b68 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x532860 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-next-20161004-126151-gc7d3b91 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Oct 3 21:20:41 PDT 2016
+Linux version 4.8.0-next-20161005-126395-g0a766d8 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Oct 4 22:50:02 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -311,171 +311,30 @@
 usb1_vbus: supplied by vdd-sys-5v0
 usb2_vbus: supplied by vdd-sys-5v0
 VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
-Unable to handle kernel paging request at virtual address fffffff8
-pgd = c0004000
-[fffffff8] *pgd=afffd861, *pte=00000000, *ppte=00000000
-Internal error: Oops: 37 [#1] PREEMPT SMP ARM
-Modules linked in:
-CPU: 0 PID: 37 Comm: kworker/0:1 Not tainted 4.8.0-next-20161004-126151-gc7d3b91 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-Workqueue: events deferred_probe_work_func
-task: eea0a280 task.stack: eea32000
-PC is at drm_debugfs_crtc_add+0x8/0x70
-LR is at drm_minor_register+0xa4/0x1b0
-pc : [<c0440b80>]    lr : [<c0428a60>]    psr: a0000013
-sp : eea33d20  ip : ee1a5d14  fp : c0d02100
-r10: 00000001  r9 : 00000000  r8 : 00000000
-r7 : ee1cd800  r6 : ee1cda4c  r5 : ee1a5d00  r4 : fffffff8
-r3 : 00000001  r2 : 00000000  r1 : eea33c70  r0 : fffffff8
-Flags: NzCv  IRQs on  FIQs on  Mode SVC_32  ISA ARM  Segment none
-Control: 10c5387d  Table: 8000406a  DAC: 00000051
-Process kworker/0:1 (pid: 37, stack limit = 0xeea32210)
-Stack: (0xeea33d20 to 0xeea34000)
-3d20: fffffff8 ee1a5d00 ee1cda4c c0428a60 eea0a280 ee1cd800 00000000 00000000
-3d40: c0d34794 00000006 00000000 c0428be8 ee1cd800 eeb73200 c0d3467c c0442008
-3d60: eeb73218 00000000 c0d34794 00000006 eeb73210 c0dabfec 00000000 c0407578
-3d80: c040755c c045ac3c 00000000 eea33db8 c045af10 00000001 c0dabfc8 c045922c
-3da0: eeb65f70 eeb50738 eeb73210 eeb73244 c0d33fa0 c045a9c4 eeb73210 00000001
-3dc0: c0d02100 eeb73210 eeb73210 c0d33fa0 ee959010 c045a0b4 eeb73210 00000000
-3de0: eeb73218 c0458624 c0d6404c 60000013 c0d02100 c0817664 eeb73200 ee1cdc10
-3e00: eeb67180 eeb733d8 eeb733b0 eef9f580 eeb73210 c0407744 eefd5538 eeb73594
-3e20: c0d3400c c0d33f68 ee1cdc10 eeb73410 00000000 c0408058 ee1cdc10 ee965610
-3e40: 00000000 ee965600 00000000 c044bb4c 00000000 ee968380 ee1cdc10 00000000
-3e60: ffffffed ee965610 fffffdfb c0d348f8 00000006 c045c20c c045c1bc ee965610
-3e80: c0dabfec 00000000 c0d348f8 c045ac3c 00000000 eea33ec0 c045af10 00000001
-3ea0: 00000000 c045922c ee8a1e70 eeb50638 ee965610 ee965644 c0d34de8 c045a9c4
-3ec0: ee965610 00000001 c0d02100 ee965610 ee965610 c0d34de8 eefa2800 c045a0b4
-3ee0: ee965610 c0d34d70 c0d34d90 c045a4e8 eea12a00 c0d34d98 eef9f580 c0134890
-3f00: eef9f580 00000000 eea12a00 eea12a18 00000001 eef9f580 eea12a18 eef9f580
-3f20: 00000008 c0134ab4 eefa28f5 eea12a00 eef9f598 c0134cc8 00000000 eea190c0
-3f40: 00000000 eea190c0 00000000 eea12a00 c0134ac4 00000000 00000000 00000000
-3f60: 00000000 c0139d68 7ffffaff 00000000 ffffffff eea12a00 00000000 00000000
-3f80: eea33f80 eea33f80 00000000 00000000 eea33f90 eea33f90 eea33fac eea190c0
-3fa0: c0139c90 00000000 00000000 c0107938 00000000 00000000 00000000 00000000
-3fc0: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
-3fe0: 00000000 00000000 00000000 00000000 00000013 00000000 ffffbffb bfffffff
-[<c0440b80>] (drm_debugfs_crtc_add) from [<c0428a60>] (drm_minor_register+0xa4/0x1b0)
-[<c0428a60>] (drm_minor_register) from [<c0428be8>] (drm_dev_register+0x7c/0xd0)
-[<c0428be8>] (drm_dev_register) from [<c0442008>] (host1x_drm_probe+0x38/0x90)
-[<c0442008>] (host1x_drm_probe) from [<c0407578>] (host1x_device_probe+0x1c/0x28)
-[<c0407578>] (host1x_device_probe) from [<c045ac3c>] (driver_probe_device+0x1f0/0x2a8)
-[<c045ac3c>] (driver_probe_device) from [<c045922c>] (bus_for_each_drv+0x44/0x8c)
-[<c045922c>] (bus_for_each_drv) from [<c045a9c4>] (__device_attach+0x9c/0x100)
-[<c045a9c4>] (__device_attach) from [<c045a0b4>] (bus_probe_device+0x84/0x8c)
-[<c045a0b4>] (bus_probe_device) from [<c0458624>] (device_add+0x380/0x528)
-[<c0458624>] (device_add) from [<c0407744>] (host1x_subdev_register+0xb0/0xd4)
-[<c0407744>] (host1x_subdev_register) from [<c0408058>] (host1x_client_register+0xf4/0x11c)
-[<c0408058>] (host1x_client_register) from [<c044bb4c>] (tegra_hdmi_probe+0x1c8/0x2c8)
-[<c044bb4c>] (tegra_hdmi_probe) from [<c045c20c>] (platform_drv_probe+0x50/0xb0)
-[<c045c20c>] (platform_drv_probe) from [<c045ac3c>] (driver_probe_device+0x1f0/0x2a8)
-[<c045ac3c>] (driver_probe_device) from [<c045922c>] (bus_for_each_drv+0x44/0x8c)
-[<c045922c>] (bus_for_each_drv) from [<c045a9c4>] (__device_attach+0x9c/0x100)
-[<c045a9c4>] (__device_attach) from [<c045a0b4>] (bus_probe_device+0x84/0x8c)
-[<c045a0b4>] (bus_probe_device) from [<c045a4e8>] (deferred_probe_work_func+0x60/0x8c)
-[<c045a4e8>] (deferred_probe_work_func) from [<c0134890>] (process_one_work+0x120/0x31c)
-[<c0134890>] (process_one_work) from [<c0134ab4>] (process_scheduled_works+0x28/0x38)
-[<c0134ab4>] (process_scheduled_works) from [<c0134cc8>] (worker_thread+0x204/0x4b4)
-[<c0134cc8>] (worker_thread) from [<c0139d68>] (kthread+0xd8/0xf4)
-[<c0139d68>] (kthread) from [<c0107938>] (ret_from_fork+0x14/0x3c)
-Code: e584334c e8bd8010 e92d4070 e1a04000 (e5943000) 
----[ end trace b76cd3785b81e186 ]---
-Unable to handle kernel paging request at virtual address ffffffec
-pgd = c0004000
-[ffffffec] *pgd=afffd861, *pte=00000000, *ppte=00000000
-Internal error: Oops: 37 [#2] PREEMPT SMP ARM
-Modules linked in:
-CPU: 0 PID: 37 Comm: kworker/0:1 Tainted: G      D         4.8.0-next-20161004-126151-gc7d3b91 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-task: eea0a280 task.stack: eea32000
-PC is at kthread_data+0x4/0xc
-LR is at wq_worker_sleeping+0x8/0xc8
-pc : [<c013a35c>]    lr : [<c0135930>]    psr: 20000193
-sp : eea33ae0  ip : eef9ff28  fp : eea33b2c
-r10: c0d02e08  r9 : c0d62040  r8 : c0c59a00
-r7 : eea0a6c8  r6 : eea0a6c0  r5 : eea0a280  r4 : eef9fa00
-r3 : 00000000  r2 : 00000000  r1 : eea0a280  r0 : eea0a280
-Flags: nzCv  IRQs off  FIQs on  Mode SVC_32  ISA ARM  Segment none
-Control: 10c5387d  Table: 8000406a  DAC: 00000051
-Process kworker/0:1 (pid: 37, stack limit = 0xeea32210)
-Stack: (0xeea33ae0 to 0xeea34000)
-3ae0: 00000020 c0814be8 635688c0 d84156c5 635688c0 d84156c5 745d1746 c01420e4
-3b00: eea1ca80 00000008 00000000 eea3375c ee85b3c0 eea0a640 eea33b38 0000000b
-3b20: 00000001 00000001 eea33b34 c01420e4 eea0a280 c0122e64 eea33b38 eea33b38
-3b40: 00000001 c0d602c4 eea33cd0 c0d07638 60000113 0000000b 00000001 c0440b82
-3b60: c0440b84 c010ac8c eea32210 0000000b 00000002 c0d07638 bf000000 00000008
-3b80: 65a33cd0 33343835 20633433 64623865 30313038 32396520 37303464 31652030
-3ba0: 30343061 28203030 34393565 30303033 a0002029 c01b1e74 000001ff fffffff8
-3bc0: 00000037 00000000 eea33cd0 fffffff8 eea32000 00000001 c0d02100 c011a00c
-3be0: 00000037 c01157c0 d84156c5 c01f3b10 00000000 ee580238 00000000 ee808f00
-3c00: a0000013 024000c0 c02150e0 00000037 c01154cc fffffff8 c0d07f14 eea33cd0
-3c20: eea32000 00000001 c0d02100 c010128c 00000002 ee581710 19a1477f c0216748
-3c40: 00000000 00000000 00000001 ee5801e8 00008124 c09528b0 c09188a0 ee1d0040
-3c60: ee580508 ee1cd800 ee1a5d1c eea33c80 c0d6404c 60000013 00000002 c0817664
-3c80: 00000001 eea33c80 ee580508 ee1d0040 c0d34794 ee1a5d00 00000002 c04408b8
-3ca0: c09528b0 00000000 ee1a5d14 c0d34774 00000000 ee1a5d00 c0440b80 a0000013
-3cc0: ffffffff eea33d04 00000000 c010b338 fffffff8 eea33c70 00000000 00000001
-3ce0: fffffff8 ee1a5d00 ee1cda4c ee1cd800 00000000 00000000 00000001 c0d02100
-3d00: ee1a5d14 eea33d20 c0428a60 c0440b80 a0000013 ffffffff 00000051 00000000
-3d20: fffffff8 ee1a5d00 ee1cda4c c0428a60 eea0a280 ee1cd800 00000000 00000000
-3d40: c0d34794 00000006 00000000 c0428be8 ee1cd800 eeb73200 c0d3467c c0442008
-3d60: eeb73218 00000000 c0d34794 00000006 eeb73210 c0dabfec 00000000 c0407578
-3d80: c040755c c045ac3c 00000000 eea33db8 c045af10 00000001 c0dabfc8 c045922c
-3da0: eeb65f70 eeb50738 eeb73210 eeb73244 c0d33fa0 c045a9c4 eeb73210 00000001
-3dc0: c0d02100 eeb73210 eeb73210 c0d33fa0 ee959010 c045a0b4 eeb73210 00000000
-3de0: eeb73218 c0458624 c0d6404c 60000013 c0d02100 c0817664 eeb73200 ee1cdc10
-3e00: eeb67180 eeb733d8 eeb733b0 eef9f580 eeb73210 c0407744 eefd5538 eeb73594
-3e20: c0d3400c c0d33f68 ee1cdc10 eeb73410 00000000 c0408058 ee1cdc10 ee965610
-3e40: 00000000 ee965600 00000000 c044bb4c 00000000 ee968380 ee1cdc10 00000000
-3e60: ffffffed ee965610 fffffdfb c0d348f8 00000006 c045c20c c045c1bc ee965610
-3e80: c0dabfec 00000000 c0d348f8 c045ac3c 00000000 eea33ec0 c045af10 00000001
-3ea0: 00000000 c045922c ee8a1e70 eeb50638 ee965610 ee965644 c0d34de8 c045a9c4
-3ec0: ee965610 00000001 c0d02100 ee965610 ee965610 c0d34de8 eefa2800 c045a0b4
-3ee0: ee965610 c0d34d70 c0d34d90 c045a4e8 eea12a00 c0d34d98 eef9f580 c0134890
-3f00: eef9f580 00000000 eea12a00 eea12a18 00000001 eef9f580 eea12a18 eef9f580
-3f20: 00000008 c0134ab4 eefa28f5 eea12a00 eef9f598 c0134cc8 00000000 eea190c0
-3f40: 00000000 eea190c0 00000000 eea12a00 c0134ac4 00000000 00000000 00000000
-3f60: 00000000 c0139d68 7ffffaff 00000000 ffffffff eea12a00 00000000 00000000
-3f80: eea33f80 eea33f80 00000001 00010001 eea33f90 eea33f90 eea33fac eea190c0
-3fa0: c0139c90 00000000 00000000 c0107938 00000000 00000000 00000000 00000000
-3fc0: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
-3fe0: 00000000 00000000 00000000 00000000 00000013 00000000 ffffbffb bfffffff
-[<c013a35c>] (kthread_data) from [<c0135930>] (wq_worker_sleeping+0x8/0xc8)
-[<c0135930>] (wq_worker_sleeping) from [<c0814be8>] (__schedule+0x37c/0x60c)
-[<c0814be8>] (__schedule) from [<c01420e4>] (do_task_dead+0x8c/0x90)
-[<c01420e4>] (do_task_dead) from [<c0122e64>] (do_exit+0x688/0xa04)
-[<c0122e64>] (do_exit) from [<c010ac8c>] (die+0x43c/0x484)
-[<c010ac8c>] (die) from [<c011a00c>] (__do_kernel_fault.part.0+0x54/0x1e4)
-[<c011a00c>] (__do_kernel_fault.part.0) from [<c01157c0>] (do_page_fault+0x2f4/0x380)
-[<c01157c0>] (do_page_fault) from [<c010128c>] (do_DataAbort+0x34/0xb8)
-[<c010128c>] (do_DataAbort) from [<c010b338>] (__dabt_svc+0x58/0x80)
-Exception stack(0xeea33cd0 to 0xeea33d18)
-3cc0:                                     fffffff8 eea33c70 00000000 00000001
-3ce0: fffffff8 ee1a5d00 ee1cda4c ee1cd800 00000000 00000000 00000001 c0d02100
-3d00: ee1a5d14 eea33d20 c0428a60 c0440b80 a0000013 ffffffff
-[<c010b338>] (__dabt_svc) from [<c0440b80>] (drm_debugfs_crtc_add+0x8/0x70)
-[<c0440b80>] (drm_debugfs_crtc_add) from [<c0428a60>] (drm_minor_register+0xa4/0x1b0)
-[<c0428a60>] (drm_minor_register) from [<c0428be8>] (drm_dev_register+0x7c/0xd0)
-[<c0428be8>] (drm_dev_register) from [<c0442008>] (host1x_drm_probe+0x38/0x90)
-[<c0442008>] (host1x_drm_probe) from [<c0407578>] (host1x_device_probe+0x1c/0x28)
-[<c0407578>] (host1x_device_probe) from [<c045ac3c>] (driver_probe_device+0x1f0/0x2a8)
-[<c045ac3c>] (driver_probe_device) from [<c045922c>] (bus_for_each_drv+0x44/0x8c)
-[<c045922c>] (bus_for_each_drv) from [<c045a9c4>] (__device_attach+0x9c/0x100)
-[<c045a9c4>] (__device_attach) from [<c045a0b4>] (bus_probe_device+0x84/0x8c)
-[<c045a0b4>] (bus_probe_device) from [<c0458624>] (device_add+0x380/0x528)
-[<c0458624>] (device_add) from [<c0407744>] (host1x_subdev_register+0xb0/0xd4)
-[<c0407744>] (host1x_subdev_register) from [<c0408058>] (host1x_client_register+0xf4/0x11c)
-[<c0408058>] (host1x_client_register) from [<c044bb4c>] (tegra_hdmi_probe+0x1c8/0x2c8)
-[<c044bb4c>] (tegra_hdmi_probe) from [<c045c20c>] (platform_drv_probe+0x50/0xb0)
-[<c045c20c>] (platform_drv_probe) from [<c045ac3c>] (driver_probe_device+0x1f0/0x2a8)
-[<c045ac3c>] (driver_probe_device) from [<c045922c>] (bus_for_each_drv+0x44/0x8c)
-[<c045922c>] (bus_for_each_drv) from [<c045a9c4>] (__device_attach+0x9c/0x100)
-[<c045a9c4>] (__device_attach) from [<c045a0b4>] (bus_probe_device+0x84/0x8c)
-[<c045a0b4>] (bus_probe_device) from [<c045a4e8>] (deferred_probe_work_func+0x60/0x8c)
-[<c045a4e8>] (deferred_probe_work_func) from [<c0134890>] (process_one_work+0x120/0x31c)
-[<c0134890>] (process_one_work) from [<c0134ab4>] (process_scheduled_works+0x28/0x38)
-[<c0134ab4>] (process_scheduled_works) from [<c0134cc8>] (worker_thread+0x204/0x4b4)
-[<c0134cc8>] (worker_thread) from [<c0139d68>] (kthread+0xd8/0xf4)
-[<c0139d68>] (kthread) from [<c0107938>] (ret_from_fork+0x14/0x3c)
-Code: c01399f8 c0a8a84c c0d60f94 e5903418 (e5130014) 
----[ end trace b76cd3785b81e187 ]---
-Fixing recursive fault but reboot is needed!
+[drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
+[drm] No driver support for vblank timestamp query.
+Console: switching to colour frame buffer device 240x75
+drm drm: fb0:  frame buffer device
+[drm] Initialized tegra 0.0.0 20120330 on minor 0
+tegra-ehci 7d008000.usb: EHCI Host Controller
+tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 1
+tegra-ehci 7d008000.usb: irq 339, io mem 0x7d008000
+tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
+hub 1-0:1.0: USB hub found
+hub 1-0:1.0: 1 port detected
+input: gpio-keys as /devices/soc0/gpio-keys/input/input1
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-07-29 17:15:57 UTC (964890957)
+dvdd_ts: disabling
+usb1_vbus: disabling
+vdd_cam_1v8_reg: disabling
+ALSA device list:
+  #0: NVIDIA Tegra Dalmore
+EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
+EXT4-fs (mmcblk1p1): recovery complete
+EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
+VFS: Mounted root (ext3 filesystem) on device 179:17.
+devtmpfs: mounted
+Freeing unused kernel memory: 1024K (c0c00000 - c0d00000)
+bash: cannot set terminal process group (-1): Inappropriate ioctl for device
+bash: no job control in this shell
+root@(none):/# [    6.146464] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-10-04 22:58:03.325483630 -0700
+++ /tmp/foob	2016-10-04 22:58:03.329483638 -0700
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MMC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -34,7 +34,7 @@
 	 ###############################################
 	 4 MiB/s
 done
-Bytes transferred = 5450600 (532b68 hex)
+Bytes transferred = 5449824 (532860 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -45,7 +45,7 @@
 done
 Bytes transferred = 67133 (1063d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x532b68 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x532860 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301363c
@@ -54,7 +54,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-next-20161004-126151-gc7d3b91 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Oct 3 21:20:41 PDT 2016
+Linux version 4.8.0-next-20161005-126395-g0a766d8 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Oct 4 22:50:02 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -382,8 +382,8 @@
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
 tegra-pcie 1003000.pcie-controller: probing port 0, using 2 lanes
-tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
 ata1: SATA link down (SStatus 0 SControl 300)
+tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
 tegra-pcie 1003000.pcie-controller: probing port 1, using 1 lanes
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000000
 tegra-pcie 1003000.pcie-controller: PCI host bridge to bus 0000:00
@@ -410,9 +410,9 @@
 pci 0000:02:00.0: supports D1 D2
 pci 0000:02:00.0: PME# supported from D0 D1 D2 D3hot D3cold
 mmc1: new high speed SDHC card at address aaaa
-mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 PCI: bus2: Fast back to back transfers disabled
 pci_bus 0000:02: busn_res: [bus 02-ff] end is updated to 02
+mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 pci 0000:00:01.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
 pci 0000:00:02.0: BAR 8: assigned [mem 0x13100000-0x131fffff]
 pci 0000:00:02.0: BAR 9: assigned [mem 0x20000000-0x200fffff 64bit pref]
@@ -421,7 +421,6 @@
 pci 0000:00:01.0: PCI bridge to [bus 01]
 pci 0000:00:01.0:   bridge window [mem 0x13000000-0x130fffff]
 pci 0000:02:00.0: BAR 4: assigned [mem 0x20000000-0x20003fff 64bit pref]
- mmcblk1: p1 p2 p3
 pci 0000:02:00.0: BAR 2: assigned [mem 0x13100000-0x13100fff 64bit]
 pci 0000:02:00.0: BAR 0: assigned [io  0x1000-0x10ff]
 pci 0000:00:02.0: PCI bridge to [bus 02]
@@ -432,6 +431,7 @@
 pcieport 0000:00:01.0: enabling device (0140 -> 0142)
 pcieport 0000:00:01.0: Signaling PME through PCIe PME interrupt
 pci 0000:01:00.0: Signaling PME through PCIe PME interrupt
+ mmcblk1: p1 p2 p3
 pcie_pme 0000:00:01.0:pcie001: service driver pcie_pme loaded
 pci 0000:00:02.0: nv_msi_ht_cap_quirk didn't locate host bridge
 pcieport 0000:00:02.0: enabling device (0140 -> 0143)
@@ -442,171 +442,34 @@
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
 r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf09fa000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 395
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
-Unable to handle kernel paging request at virtual address fffffff8
-pgd = c0004000
-[fffffff8] *pgd=afffd861, *pte=00000000, *ppte=00000000
-Internal error: Oops: 37 [#1] PREEMPT SMP ARM
-Modules linked in:
-CPU: 2 PID: 72 Comm: kworker/2:1 Not tainted 4.8.0-next-20161004-126151-gc7d3b91 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-Workqueue: events deferred_probe_work_func
-task: ee00f880 task.stack: ee010000
-PC is at drm_debugfs_crtc_add+0x8/0x70
-LR is at drm_minor_register+0xa4/0x1b0
-pc : [<c0440b80>]    lr : [<c0428a60>]    psr: a0000113
-sp : ee011d20  ip : ee2f4914  fp : c0d02100
-r10: 00000001  r9 : 00000000  r8 : 00000000
-r7 : ee2f3800  r6 : ee2f3a4c  r5 : ee2f4900  r4 : fffffff8
-r3 : 00000001  r2 : 00000000  r1 : ee011c70  r0 : fffffff8
-Flags: NzCv  IRQs on  FIQs on  Mode SVC_32  ISA ARM  Segment none
-Control: 10c5387d  Table: 8000406a  DAC: 00000051
-Process kworker/2:1 (pid: 72, stack limit = 0xee010210)
-Stack: (0xee011d20 to 0xee012000)
-1d20: fffffff8 ee2f4900 ee2f3a4c c0428a60 ee00f880 ee2f3800 00000000 00000000
-1d40: c0d34794 0000001e 00000000 c0428be8 ee2f3800 eebae800 c0d3467c c0442008
-1d60: eebae818 00000000 c0d34794 0000001e eebae810 c0dabfec 00000000 c0407578
-1d80: c040755c c045ac3c 00000000 ee011db8 c045af10 00000001 c0dabfc8 c045922c
-1da0: eeaaa370 eebac0b8 eebae810 eebae844 c0d33fa0 c045a9c4 eebae810 00000001
-1dc0: c0d02100 eebae810 eebae810 c0d33fa0 ee9b6e10 c045a0b4 eebae810 00000000
-1de0: eebae818 c0458624 c0d6404c 60000113 c0d02100 c0817664 eebae800 ee2f3c10
-1e00: ee034ec0 eebae9d8 eebae9b0 eefa5580 eebae810 c0407744 eefbf974 eebaeb94
-1e20: c0d3400c c0d33f68 ee2f3c10 eebaea10 00000000 c0408058 ee2f3c10 ee9b6810
-1e40: 00000000 ee9b6800 00000000 c044bb4c 00000000 ee9b4940 ee2f3c10 00000000
-1e60: ffffffed ee9b6810 fffffdfb c0d348f8 0000001e c045c20c c045c1bc ee9b6810
-1e80: c0dabfec 00000000 c0d348f8 c045ac3c 00000000 ee011ec0 c045af10 00000001
-1ea0: 00000000 c045922c ee8a3d70 eebacfb8 ee9b6810 ee9b6844 c0d34de8 c045a9c4
-1ec0: ee9b6810 00000001 c0d02100 ee9b6810 ee9b6810 c0d34de8 eefa8800 c045a0b4
-1ee0: ee9b6810 c0d34d70 c0d34d90 c045a4e8 eeb78f00 c0d34d98 eefa5580 c0134890
-1f00: ee171484 eefa5580 eeb78f00 eeb78f18 00000001 eefa5580 eeb78f18 eefa5580
-1f20: 00000008 c0134ab4 eefa88f5 eeb78f00 eefa5598 c0134cc8 00000000 eeaf6fc0
-1f40: 00000000 eeaf6fc0 00000000 eeb78f00 c0134ac4 00000000 00000000 00000000
-1f60: 00000000 c0139d68 fffcffff 00000000 ffefffff eeb78f00 00000000 00000000
-1f80: ee011f80 ee011f80 00000000 00000000 ee011f90 ee011f90 ee011fac eeaf6fc0
-1fa0: c0139c90 00000000 00000000 c0107938 00000000 00000000 00000000 00000000
-1fc0: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
-1fe0: 00000000 00000000 00000000 00000000 00000013 00000000 fffffff3 ffffefbf
-[<c0440b80>] (drm_debugfs_crtc_add) from [<c0428a60>] (drm_minor_register+0xa4/0x1b0)
-[<c0428a60>] (drm_minor_register) from [<c0428be8>] (drm_dev_register+0x7c/0xd0)
-[<c0428be8>] (drm_dev_register) from [<c0442008>] (host1x_drm_probe+0x38/0x90)
-[<c0442008>] (host1x_drm_probe) from [<c0407578>] (host1x_device_probe+0x1c/0x28)
-[<c0407578>] (host1x_device_probe) from [<c045ac3c>] (driver_probe_device+0x1f0/0x2a8)
-[<c045ac3c>] (driver_probe_device) from [<c045922c>] (bus_for_each_drv+0x44/0x8c)
-[<c045922c>] (bus_for_each_drv) from [<c045a9c4>] (__device_attach+0x9c/0x100)
-[<c045a9c4>] (__device_attach) from [<c045a0b4>] (bus_probe_device+0x84/0x8c)
-[<c045a0b4>] (bus_probe_device) from [<c0458624>] (device_add+0x380/0x528)
-[<c0458624>] (device_add) from [<c0407744>] (host1x_subdev_register+0xb0/0xd4)
-[<c0407744>] (host1x_subdev_register) from [<c0408058>] (host1x_client_register+0xf4/0x11c)
-[<c0408058>] (host1x_client_register) from [<c044bb4c>] (tegra_hdmi_probe+0x1c8/0x2c8)
-[<c044bb4c>] (tegra_hdmi_probe) from [<c045c20c>] (platform_drv_probe+0x50/0xb0)
-[<c045c20c>] (platform_drv_probe) from [<c045ac3c>] (driver_probe_device+0x1f0/0x2a8)
-[<c045ac3c>] (driver_probe_device) from [<c045922c>] (bus_for_each_drv+0x44/0x8c)
-[<c045922c>] (bus_for_each_drv) from [<c045a9c4>] (__device_attach+0x9c/0x100)
-[<c045a9c4>] (__device_attach) from [<c045a0b4>] (bus_probe_device+0x84/0x8c)
-[<c045a0b4>] (bus_probe_device) from [<c045a4e8>] (deferred_probe_work_func+0x60/0x8c)
-[<c045a4e8>] (deferred_probe_work_func) from [<c0134890>] (process_one_work+0x120/0x31c)
-[<c0134890>] (process_one_work) from [<c0134ab4>] (process_scheduled_works+0x28/0x38)
-[<c0134ab4>] (process_scheduled_works) from [<c0134cc8>] (worker_thread+0x204/0x4b4)
-[<c0134cc8>] (worker_thread) from [<c0139d68>] (kthread+0xd8/0xf4)
-[<c0139d68>] (kthread) from [<c0107938>] (ret_from_fork+0x14/0x3c)
-Code: e584334c e8bd8010 e92d4070 e1a04000 (e5943000) 
----[ end trace 988a54919c1729f9 ]---
-Unable to handle kernel paging request at virtual address ffffffec
-pgd = c0004000
-[ffffffec] *pgd=afffd861, *pte=00000000, *ppte=00000000
-Internal error: Oops: 37 [#2] PREEMPT SMP ARM
-Modules linked in:
-CPU: 2 PID: 72 Comm: kworker/2:1 Tainted: G      D         4.8.0-next-20161004-126151-gc7d3b91 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-task: ee00f880 task.stack: ee010000
-PC is at kthread_data+0x4/0xc
-LR is at wq_worker_sleeping+0x8/0xc8
-pc : [<c013a35c>]    lr : [<c0135930>]    psr: 20000193
-sp : ee011ae0  ip : eefa5f28  fp : ee011b2c
-r10: c0d02e08  r9 : c0d62040  r8 : c0c59a00
-r7 : ee00fcc8  r6 : ee00fcc0  r5 : ee00f880  r4 : eefa5a00
-r3 : 00000000  r2 : 00000000  r1 : ee00f880  r0 : ee00f880
-Flags: nzCv  IRQs off  FIQs on  Mode SVC_32  ISA ARM  Segment none
-Control: 10c5387d  Table: 8000406a  DAC: 00000051
-Process kworker/2:1 (pid: 72, stack limit = 0xee010210)
-Stack: (0xee011ae0 to 0xee012000)
-1ae0: 00000020 c0814be8 ee012ea8 ee012d80 c0c54280 c01f50c0 ee8fb504 c01420e4
-1b00: ee8fb000 00000051 00000000 ee01175c ee85d3c0 ee00fc40 ee011b38 0000000b
-1b20: 00000001 00000001 ee011b34 c01420e4 ee00f880 c0122e64 ee011b38 ee011b38
-1b40: 00000001 c0d602c4 ee011cd0 c0d07638 60000113 0000000b 00000001 c0440b82
-1b60: c0440b84 c010ac8c ee010210 0000000b 00000002 c0d07638 bf000000 00000008
-1b80: 65011cd0 33343835 20633433 64623865 30313038 32396520 37303464 31652030
-1ba0: 30343061 28203030 34393565 30303033 c0002029 c01b1e74 000001ff fffffff8
-1bc0: 00000037 00000000 ee011cd0 fffffff8 ee010000 00000001 c0d02100 c011a00c
-1be0: 00000037 c01157c0 d84156c5 c01f3b10 00000000 ee808f00 a0000113 ee808f00
-1c00: a0000113 024000c0 c02150e0 00000037 c01154cc fffffff8 c0d07f14 ee011cd0
-1c20: ee010000 00000001 c0d02100 c010128c 00000002 ee5ace18 27ef637e c0216748
-1c40: 00000000 00000000 00000001 ee5ab6e8 00008124 c09528b0 c09188a0 ee2f4c40
-1c60: ee5a8a08 ee2f3800 ee2f491c ee011c80 c0d6404c 60000113 00000002 c0817664
-1c80: 00000001 ee011c80 ee5a8a08 ee2f4c40 c0d34794 ee2f4900 00000002 c04408b8
-1ca0: c09528b0 00000000 ee2f4914 c0d34774 00000000 ee2f4900 c0440b80 a0000113
-1cc0: ffffffff ee011d04 00000000 c010b338 fffffff8 ee011c70 00000000 00000001
-1ce0: fffffff8 ee2f4900 ee2f3a4c ee2f3800 00000000 00000000 00000001 c0d02100
-1d00: ee2f4914 ee011d20 c0428a60 c0440b80 a0000113 ffffffff 00000051 00000000
-1d20: fffffff8 ee2f4900 ee2f3a4c c0428a60 ee00f880 ee2f3800 00000000 00000000
-1d40: c0d34794 0000001e 00000000 c0428be8 ee2f3800 eebae800 c0d3467c c0442008
-1d60: eebae818 00000000 c0d34794 0000001e eebae810 c0dabfec 00000000 c0407578
-1d80: c040755c c045ac3c 00000000 ee011db8 c045af10 00000001 c0dabfc8 c045922c
-1da0: eeaaa370 eebac0b8 eebae810 eebae844 c0d33fa0 c045a9c4 eebae810 00000001
-1dc0: c0d02100 eebae810 eebae810 c0d33fa0 ee9b6e10 c045a0b4 eebae810 00000000
-1de0: eebae818 c0458624 c0d6404c 60000113 c0d02100 c0817664 eebae800 ee2f3c10
-1e00: ee034ec0 eebae9d8 eebae9b0 eefa5580 eebae810 c0407744 eefbf974 eebaeb94
-1e20: c0d3400c c0d33f68 ee2f3c10 eebaea10 00000000 c0408058 ee2f3c10 ee9b6810
-1e40: 00000000 ee9b6800 00000000 c044bb4c 00000000 ee9b4940 ee2f3c10 00000000
-1e60: ffffffed ee9b6810 fffffdfb c0d348f8 0000001e c045c20c c045c1bc ee9b6810
-1e80: c0dabfec 00000000 c0d348f8 c045ac3c 00000000 ee011ec0 c045af10 00000001
-1ea0: 00000000 c045922c ee8a3d70 eebacfb8 ee9b6810 ee9b6844 c0d34de8 c045a9c4
-1ec0: ee9b6810 00000001 c0d02100 ee9b6810 ee9b6810 c0d34de8 eefa8800 c045a0b4
-1ee0: ee9b6810 c0d34d70 c0d34d90 c045a4e8 eeb78f00 c0d34d98 eefa5580 c0134890
-1f00: ee171484 eefa5580 eeb78f00 eeb78f18 00000001 eefa5580 eeb78f18 eefa5580
-1f20: 00000008 c0134ab4 eefa88f5 eeb78f00 eefa5598 c0134cc8 00000000 eeaf6fc0
-1f40: 00000000 eeaf6fc0 00000000 eeb78f00 c0134ac4 00000000 00000000 00000000
-1f60: 00000000 c0139d68 fffcffff 00000000 ffefffff eeb78f00 00000000 00000000
-1f80: ee011f80 ee011f80 00000001 00010001 ee011f90 ee011f90 ee011fac eeaf6fc0
-1fa0: c0139c90 00000000 00000000 c0107938 00000000 00000000 00000000 00000000
-1fc0: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
-1fe0: 00000000 00000000 00000000 00000000 00000013 00000000 fffffff3 ffffefbf
-[<c013a35c>] (kthread_data) from [<c0135930>] (wq_worker_sleeping+0x8/0xc8)
-[<c0135930>] (wq_worker_sleeping) from [<c0814be8>] (__schedule+0x37c/0x60c)
-[<c0814be8>] (__schedule) from [<c01420e4>] (do_task_dead+0x8c/0x90)
-[<c01420e4>] (do_task_dead) from [<c0122e64>] (do_exit+0x688/0xa04)
-[<c0122e64>] (do_exit) from [<c010ac8c>] (die+0x43c/0x484)
-[<c010ac8c>] (die) from [<c011a00c>] (__do_kernel_fault.part.0+0x54/0x1e4)
-[<c011a00c>] (__do_kernel_fault.part.0) from [<c01157c0>] (do_page_fault+0x2f4/0x380)
-[<c01157c0>] (do_page_fault) from [<c010128c>] (do_DataAbort+0x34/0xb8)
-[<c010128c>] (do_DataAbort) from [<c010b338>] (__dabt_svc+0x58/0x80)
-Exception stack(0xee011cd0 to 0xee011d18)
-1cc0:                                     fffffff8 ee011c70 00000000 00000001
-1ce0: fffffff8 ee2f4900 ee2f3a4c ee2f3800 00000000 00000000 00000001 c0d02100
-1d00: ee2f4914 ee011d20 c0428a60 c0440b80 a0000113 ffffffff
-[<c010b338>] (__dabt_svc) from [<c0440b80>] (drm_debugfs_crtc_add+0x8/0x70)
-[<c0440b80>] (drm_debugfs_crtc_add) from [<c0428a60>] (drm_minor_register+0xa4/0x1b0)
-[<c0428a60>] (drm_minor_register) from [<c0428be8>] (drm_dev_register+0x7c/0xd0)
-[<c0428be8>] (drm_dev_register) from [<c0442008>] (host1x_drm_probe+0x38/0x90)
-[<c0442008>] (host1x_drm_probe) from [<c0407578>] (host1x_device_probe+0x1c/0x28)
-[<c0407578>] (host1x_device_probe) from [<c045ac3c>] (driver_probe_device+0x1f0/0x2a8)
-[<c045ac3c>] (driver_probe_device) from [<c045922c>] (bus_for_each_drv+0x44/0x8c)
-[<c045922c>] (bus_for_each_drv) from [<c045a9c4>] (__device_attach+0x9c/0x100)
-[<c045a9c4>] (__device_attach) from [<c045a0b4>] (bus_probe_device+0x84/0x8c)
-[<c045a0b4>] (bus_probe_device) from [<c0458624>] (device_add+0x380/0x528)
-[<c0458624>] (device_add) from [<c0407744>] (host1x_subdev_register+0xb0/0xd4)
-[<c0407744>] (host1x_subdev_register) from [<c0408058>] (host1x_client_register+0xf4/0x11c)
-[<c0408058>] (host1x_client_register) from [<c044bb4c>] (tegra_hdmi_probe+0x1c8/0x2c8)
-[<c044bb4c>] (tegra_hdmi_probe) from [<c045c20c>] (platform_drv_probe+0x50/0xb0)
-[<c045c20c>] (platform_drv_probe) from [<c045ac3c>] (driver_probe_device+0x1f0/0x2a8)
-[<c045ac3c>] (driver_probe_device) from [<c045922c>] (bus_for_each_drv+0x44/0x8c)
-[<c045922c>] (bus_for_each_drv) from [<c045a9c4>] (__device_attach+0x9c/0x100)
-[<c045a9c4>] (__device_attach) from [<c045a0b4>] (bus_probe_device+0x84/0x8c)
-[<c045a0b4>] (bus_probe_device) from [<c045a4e8>] (deferred_probe_work_func+0x60/0x8c)
-[<c045a4e8>] (deferred_probe_work_func) from [<c0134890>] (process_one_work+0x120/0x31c)
-[<c0134890>] (process_one_work) from [<c0134ab4>] (process_scheduled_works+0x28/0x38)
-[<c0134ab4>] (process_scheduled_works) from [<c0134cc8>] (worker_thread+0x204/0x4b4)
-[<c0134cc8>] (worker_thread) from [<c0139d68>] (kthread+0xd8/0xf4)
-[<c0139d68>] (kthread) from [<c0107938>] (ret_from_fork+0x14/0x3c)
-Code: c01399f8 c0a8a84c c0d60f94 e5903418 (e5130014) 
----[ end trace 988a54919c1729fa ]---
-Fixing recursive fault but reboot is needed!
+[drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
+[drm] No driver support for vblank timestamp query.
+tegra-hdmi 54280000.hdmi: cannot set audio to 48000 Hz at 297000000 Hz pixel clock
+Console: switching to colour frame buffer device 512x135
+drm drm: fb0:  frame buffer device
+[drm] Initialized tegra 0.0.0 20120330 on minor 0
+of: dev_pm_opp_of_cpumask_add_table: couldn't find opp table for cpu:0, -19
+cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
+cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
+input: gpio-keys as /devices/soc0/gpio-keys/input/input2
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:17 UTC (946684817)
++USB0_VBUS_SW: disabling
++1.8V_RUN_CAM: disabling
++2.8V_RUN_CAM: disabling
++1.2V_RUN_CAM_FRONT: disabling
++1.05V_RUN_CAM_REAR: disabling
++3.3V_RUN_TOUCH: disabling
++2.8V_RUN_CAM_AF: disabling
++1.8V_RUN_VPP_FUSE: disabling
+ALSA device list:
+  #0: tegra-hda at 0x70038000 irq 346
+  #1: NVIDIA Tegra Jetson TK1
+EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
+EXT4-fs (mmcblk0p1): recovery complete
+EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
+VFS: Mounted root (ext3 filesystem) on device 179:1.
+devtmpfs: mounted
+Freeing unused kernel memory: 1024K (c0c00000 - c0d00000)
+bash: cannot set terminal process group (-1): Inappropriate ioctl for device
+bash: no job control in this shell
+root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2016-10-04 22:58:03.349483677 -0700
+++ /tmp/foob	2016-10-04 22:58:03.349483677 -0700
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -36,24 +36,23 @@
 Loading: *#################################################################
 	 #################################################################
 	 #################################################################
-	 #################################################################EHCI timed out on TD - token=0x88008d80
-T 
+	 #################################################################
 	 #################################################################
 	 ###############################################
-	 422.9 KiB/s
+	 3.6 MiB/s
 done
-Bytes transferred = 5450600 (532b68 hex)
+Bytes transferred = 5449824 (532860 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.4 MiB/s
+	 3.6 MiB/s
 done
 Bytes transferred = 67899 (1093b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x532b68 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x532860 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301393a
@@ -222,173 +221,52 @@
 +VDDIO_SDMMC3: ramp_delay not set
 mmc1: new high speed SDIO card at address 0001
 mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
-Unable to handle kernel paging request at virtual address fffffff8
-pgd = c0004000
-[fffffff8] *pgd=afffd861, *pte=00000000, *ppte=00000000
-Internal error: Oops: 37 [#1] PREEMPT SMP ARM
-Modules linked in:
-CPU: 2 PID: 83 Comm: kworker/2:1 Not tainted 4.8.0-next-20161004-126151-gc7d3b91 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-Workqueue: events deferred_probe_work_func
-task: ee0fc500 task.stack: ee144000
-PC is at drm_debugfs_crtc_add+0x8/0x70
-LR is at drm_minor_register+0xa4/0x1b0
-pc : [<c0440b80>]    lr : [<c0428a60>]    psr: a0000113
-sp : ee145d18  ip : ee298d54  fp : c0d02100
-r10: 00000001  r9 : 00000000  r8 : 00000000
-r7 : ee29c800  r6 : ee29ca4c  r5 : ee298d40  r4 : fffffff8
-r3 : 00000001  r2 : 00000000  r1 : ee145c68  r0 : fffffff8
-Flags: NzCv  IRQs on  FIQs on  Mode SVC_32  ISA ARM  Segment none
-Control: 10c5387d  Table: 8000406a  DAC: 00000051
-Process kworker/2:1 (pid: 83, stack limit = 0xee144210)
-Stack: (0xee145d18 to 0xee146000)
-5d00:                                                       fffffff8 ee298d40
-5d20: ee29ca4c c0428a60 ee0fc500 ee29c800 00000000 00000000 c0d34794 00000017
-5d40: 00000000 c0428be8 ee29c800 ee00be00 c0d3467c c0442008 ee00be18 00000000
-5d60: c0d34794 00000017 ee00be10 c0dabfec 00000000 c0407578 c040755c c045ac3c
-5d80: 00000000 ee145db0 c045af10 00000001 c0dabfc8 c045922c eeaf5a70 ee009038
-5da0: ee00be10 ee00be44 c0d33fa0 c045a9c4 ee00be10 00000001 c0d02100 ee00be10
-5dc0: ee00be10 c0d33fa0 ee9ad010 c045a0b4 ee00be10 00000000 ee00be18 c0458624
-5de0: c0d6404c 60000113 c0d02100 c0817664 ee00be00 ee25a010 eeba5f80 ee00bfd8
-5e00: ee00bfb0 eefa5580 ee00be10 c0407744 eefc02ec eeba2194 c0d3400c c0d33f68
-5e20: ee25a010 eeba2010 00000000 c0408058 ee25a010 ee9b8810 ee298850 ee298ec0
-5e40: ee25a038 c0452c44 00000000 c0ab5d34 c095586c c0955864 ee9b8802 00000000
-5e60: ffffffed ee9b8810 fffffdfb c0d349c0 00000017 c045c20c c045c1bc ee9b8810
-5e80: c0dabfec 00000000 c0d349c0 c045ac3c 00000000 ee145ec0 c045af10 00000001
-5ea0: 00000000 c045922c ee8a3d70 ee009db8 ee9b8810 ee9b8844 c0d34de8 c045a9c4
-5ec0: ee9b8810 00000001 c0d02100 ee9b8810 ee9b8810 c0d34de8 eefa8800 c045a0b4
-5ee0: ee9b8810 c0d34d70 c0d34d90 c045a4e8 ee11f700 c0d34d98 eefa5580 c0134890
-5f00: eefa5580 00000000 ee11f700 ee11f718 00000001 eefa5580 ee11f718 eefa5580
-5f20: 00000008 c0134ab4 eefa88f5 ee11f700 eefa5598 c0134cc8 00000000 ee108fc0
-5f40: 00000000 ee108fc0 00000000 ee11f700 c0134ac4 00000000 00000000 00000000
-5f60: 00000000 c0139d68 00000000 00000000 00000000 ee11f700 00000000 00000000
-5f80: ee145f80 ee145f80 00000000 00000000 ee145f90 ee145f90 ee145fac ee108fc0
-5fa0: c0139c90 00000000 00000000 c0107938 00000000 00000000 00000000 00000000
-5fc0: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
-5fe0: 00000000 00000000 00000000 00000000 00000013 00000000 00000000 00000000
-[<c0440b80>] (drm_debugfs_crtc_add) from [<c0428a60>] (drm_minor_register+0xa4/0x1b0)
-[<c0428a60>] (drm_minor_register) from [<c0428be8>] (drm_dev_register+0x7c/0xd0)
-[<c0428be8>] (drm_dev_register) from [<c0442008>] (host1x_drm_probe+0x38/0x90)
-[<c0442008>] (host1x_drm_probe) from [<c0407578>] (host1x_device_probe+0x1c/0x28)
-[<c0407578>] (host1x_device_probe) from [<c045ac3c>] (driver_probe_device+0x1f0/0x2a8)
-[<c045ac3c>] (driver_probe_device) from [<c045922c>] (bus_for_each_drv+0x44/0x8c)
-[<c045922c>] (bus_for_each_drv) from [<c045a9c4>] (__device_attach+0x9c/0x100)
-[<c045a9c4>] (__device_attach) from [<c045a0b4>] (bus_probe_device+0x84/0x8c)
-[<c045a0b4>] (bus_probe_device) from [<c0458624>] (device_add+0x380/0x528)
-[<c0458624>] (device_add) from [<c0407744>] (host1x_subdev_register+0xb0/0xd4)
-[<c0407744>] (host1x_subdev_register) from [<c0408058>] (host1x_client_register+0xf4/0x11c)
-[<c0408058>] (host1x_client_register) from [<c0452c44>] (tegra_sor_probe+0x26c/0x4ac)
-[<c0452c44>] (tegra_sor_probe) from [<c045c20c>] (platform_drv_probe+0x50/0xb0)
-[<c045c20c>] (platform_drv_probe) from [<c045ac3c>] (driver_probe_device+0x1f0/0x2a8)
-[<c045ac3c>] (driver_probe_device) from [<c045922c>] (bus_for_each_drv+0x44/0x8c)
-[<c045922c>] (bus_for_each_drv) from [<c045a9c4>] (__device_attach+0x9c/0x100)
-[<c045a9c4>] (__device_attach) from [<c045a0b4>] (bus_probe_device+0x84/0x8c)
-[<c045a0b4>] (bus_probe_device) from [<c045a4e8>] (deferred_probe_work_func+0x60/0x8c)
-[<c045a4e8>] (deferred_probe_work_func) from [<c0134890>] (process_one_work+0x120/0x31c)
-[<c0134890>] (process_one_work) from [<c0134ab4>] (process_scheduled_works+0x28/0x38)
-[<c0134ab4>] (process_scheduled_works) from [<c0134cc8>] (worker_thread+0x204/0x4b4)
-[<c0134cc8>] (worker_thread) from [<c0139d68>] (kthread+0xd8/0xf4)
-[<c0139d68>] (kthread) from [<c0107938>] (ret_from_fork+0x14/0x3c)
-Code: e584334c e8bd8010 e92d4070 e1a04000 (e5943000) 
----[ end trace b4c40e72f416899a ]---
-Unable to handle kernel paging request at virtual address ffffffec
-pgd = c0004000
-[ffffffec] *pgd=afffd861, *pte=00000000, *ppte=00000000
-Internal error: Oops: 37 [#2] PREEMPT SMP ARM
-Modules linked in:
-CPU: 2 PID: 83 Comm: kworker/2:1 Tainted: G      D         4.8.0-next-20161004-126151-gc7d3b91 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-task: ee0fc500 task.stack: ee144000
-PC is at kthread_data+0x4/0xc
-LR is at wq_worker_sleeping+0x8/0xc8
-pc : [<c013a35c>]    lr : [<c0135930>]    psr: 20000193
-sp : ee145ad8  ip : eefa5f28  fp : ee145b24
-r10: c0d02e08  r9 : c0d62040  r8 : c0c59a00
-r7 : ee0fc948  r6 : ee0fc940  r5 : ee0fc500  r4 : eefa5a00
-r3 : 00000000  r2 : 00000000  r1 : ee0fc500  r0 : ee0fc500
-Flags: nzCv  IRQs off  FIQs on  Mode SVC_32  ISA ARM  Segment none
-Control: 10c5387d  Table: 8000406a  DAC: 00000051
-Process kworker/2:1 (pid: 83, stack limit = 0xee144210)
-Stack: (0xee145ad8 to 0xee146000)
-5ac0:                                                       00000020 c0814be8
-5ae0: ee8ef7e8 ee8ef6c0 c0c54280 c01f50c0 ee8f0f84 c01420e4 ee8f0a80 0000001a
-5b00: 00000000 ee145754 ee85d3c0 ee0fc8c0 ee145b30 0000000b 00000001 00000001
-5b20: ee145b2c c01420e4 ee0fc500 c0122e64 ee145b30 ee145b30 00000001 c0d602c4
-5b40: ee145cc8 c0d07638 60000113 0000000b 00000001 c0440b82 c0440b84 c010ac8c
-5b60: ee144210 0000000b 00000002 c0d07638 bf000000 00000008 65145cc8 33343835
-5b80: 20633433 64623865 30313038 32396520 37303464 31652030 30343061 28203030
-5ba0: 34393565 30303033 ee002029 c01b1e74 ee144000 fffffff8 00000037 00000000
-5bc0: ee145cc8 fffffff8 ee144000 00000001 c0d02100 c011a00c 00000037 c01157c0
-5be0: d84156c5 c01f3b10 00000000 ee597418 00000000 ee808f00 a0000113 024000c0
-5c00: c02150e0 00000037 c01154cc fffffff8 c0d07f14 ee145cc8 ee144000 00000001
-5c20: c0d02100 c010128c 00000002 ee5982d8 380742ff c0216748 00000000 00000000
-5c40: 00000001 ee5973c8 00008124 c09528b0 c09188a0 ee29e080 ee5976e8 ee29c800
-5c60: ee298d5c ee145c78 c0d6404c 60000113 00000002 c0817664 00000001 ee145c78
-5c80: ee5976e8 ee29e080 c0d34794 ee298d40 00000002 c04408b8 c09528b0 00000000
-5ca0: ee298d54 c0d34774 00000000 ee298d40 c0440b80 a0000113 ffffffff ee145cfc
-5cc0: 00000000 c010b338 fffffff8 ee145c68 00000000 00000001 fffffff8 ee298d40
-5ce0: ee29ca4c ee29c800 00000000 00000000 00000001 c0d02100 ee298d54 ee145d18
-5d00: c0428a60 c0440b80 a0000113 ffffffff 00000051 00000000 fffffff8 ee298d40
-5d20: ee29ca4c c0428a60 ee0fc500 ee29c800 00000000 00000000 c0d34794 00000017
-5d40: 00000000 c0428be8 ee29c800 ee00be00 c0d3467c c0442008 ee00be18 00000000
-5d60: c0d34794 00000017 ee00be10 c0dabfec 00000000 c0407578 c040755c c045ac3c
-5d80: 00000000 ee145db0 c045af10 00000001 c0dabfc8 c045922c eeaf5a70 ee009038
-5da0: ee00be10 ee00be44 c0d33fa0 c045a9c4 ee00be10 00000001 c0d02100 ee00be10
-5dc0: ee00be10 c0d33fa0 ee9ad010 c045a0b4 ee00be10 00000000 ee00be18 c0458624
-5de0: c0d6404c 60000113 c0d02100 c0817664 ee00be00 ee25a010 eeba5f80 ee00bfd8
-5e00: ee00bfb0 eefa5580 ee00be10 c0407744 eefc02ec eeba2194 c0d3400c c0d33f68
-5e20: ee25a010 eeba2010 00000000 c0408058 ee25a010 ee9b8810 ee298850 ee298ec0
-5e40: ee25a038 c0452c44 00000000 c0ab5d34 c095586c c0955864 ee9b8802 00000000
-5e60: ffffffed ee9b8810 fffffdfb c0d349c0 00000017 c045c20c c045c1bc ee9b8810
-5e80: c0dabfec 00000000 c0d349c0 c045ac3c 00000000 ee145ec0 c045af10 00000001
-5ea0: 00000000 c045922c ee8a3d70 ee009db8 ee9b8810 ee9b8844 c0d34de8 c045a9c4
-5ec0: ee9b8810 00000001 c0d02100 ee9b8810 ee9b8810 c0d34de8 eefa8800 c045a0b4
-5ee0: ee9b8810 c0d34d70 c0d34d90 c045a4e8 ee11f700 c0d34d98 eefa5580 c0134890
-5f00: eefa5580 00000000 ee11f700 ee11f718 00000001 eefa5580 ee11f718 eefa5580
-5f20: 00000008 c0134ab4 eefa88f5 ee11f700 eefa5598 c0134cc8 00000000 ee108fc0
-5f40: 00000000 ee108fc0 00000000 ee11f700 c0134ac4 00000000 00000000 00000000
-5f60: 00000000 c0139d68 00000000 00000000 00000000 ee11f700 00000000 00000000
-5f80: ee145f80 ee145f80 00000001 00010001 ee145f90 ee145f90 ee145fac ee108fc0
-5fa0: c0139c90 00000000 00000000 c0107938 00000000 00000000 00000000 00000000
-5fc0: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
-5fe0: 00000000 00000000 00000000 00000000 00000013 00000000 00000000 00000000
-[<c013a35c>] (kthread_data) from [<c0135930>] (wq_worker_sleeping+0x8/0xc8)
-[<c0135930>] (wq_worker_sleeping) from [<c0814be8>] (__schedule+0x37c/0x60c)
-[<c0814be8>] (__schedule) from [<c01420e4>] (do_task_dead+0x8c/0x90)
-[<c01420e4>] (do_task_dead) from [<c0122e64>] (do_exit+0x688/0xa04)
-[<c0122e64>] (do_exit) from [<c010ac8c>] (die+0x43c/0x484)
-[<c010ac8c>] (die) from [<c011a00c>] (__do_kernel_fault.part.0+0x54/0x1e4)
-[<c011a00c>] (__do_kernel_fault.part.0) from [<c01157c0>] (do_page_fault+0x2f4/0x380)
-[<c01157c0>] (do_page_fault) from [<c010128c>] (do_DataAbort+0x34/0xb8)
-[<c010128c>] (do_DataAbort) from [<c010b338>] (__dabt_svc+0x58/0x80)
-Exception stack(0xee145cc8 to 0xee145d10)
-5cc0:                   fffffff8 ee145c68 00000000 00000001 fffffff8 ee298d40
-5ce0: ee29ca4c ee29c800 00000000 00000000 00000001 c0d02100 ee298d54 ee145d18
-5d00: c0428a60 c0440b80 a0000113 ffffffff
-[<c010b338>] (__dabt_svc) from [<c0440b80>] (drm_debugfs_crtc_add+0x8/0x70)
-[<c0440b80>] (drm_debugfs_crtc_add) from [<c0428a60>] (drm_minor_register+0xa4/0x1b0)
-[<c0428a60>] (drm_minor_register) from [<c0428be8>] (drm_dev_register+0x7c/0xd0)
-[<c0428be8>] (drm_dev_register) from [<c0442008>] (host1x_drm_probe+0x38/0x90)
-[<c0442008>] (host1x_drm_probe) from [<c0407578>] (host1x_device_probe+0x1c/0x28)
-[<c0407578>] (host1x_device_probe) from [<c045ac3c>] (driver_probe_device+0x1f0/0x2a8)
-[<c045ac3c>] (driver_probe_device) from [<c045922c>] (bus_for_each_drv+0x44/0x8c)
-[<c045922c>] (bus_for_each_drv) from [<c045a9c4>] (__device_attach+0x9c/0x100)
-[<c045a9c4>] (__device_attach) from [<c045a0b4>] (bus_probe_device+0x84/0x8c)
-[<c045a0b4>] (bus_probe_device) from [<c0458624>] (device_add+0x380/0x528)
-[<c0458624>] (device_add) from [<c0407744>] (host1x_subdev_register+0xb0/0xd4)
-[<c0407744>] (host1x_subdev_register) from [<c0408058>] (host1x_client_register+0xf4/0x11c)
-[<c0408058>] (host1x_client_register) from [<c0452c44>] (tegra_sor_probe+0x26c/0x4ac)
-[<c0452c44>] (tegra_sor_probe) from [<c045c20c>] (platform_drv_probe+0x50/0xb0)
-[<c045c20c>] (platform_drv_probe) from [<c045ac3c>] (driver_probe_device+0x1f0/0x2a8)
-[<c045ac3c>] (driver_probe_device) from [<c045922c>] (bus_for_each_drv+0x44/0x8c)
-[<c045922c>] (bus_for_each_drv) from [<c045a9c4>] (__device_attach+0x9c/0x100)
-[<c045a9c4>] (__device_attach) from [<c045a0b4>] (bus_probe_device+0x84/0x8c)
-[<c045a0b4>] (bus_probe_device) from [<c045a4e8>] (deferred_probe_work_func+0x60/0x8c)
-[<c045a4e8>] (deferred_probe_work_func) from [<c0134890>] (process_one_work+0x120/0x31c)
-[<c0134890>] (process_one_work) from [<c0134ab4>] (process_scheduled_works+0x28/0x38)
-[<c0134ab4>] (process_scheduled_works) from [<c0134cc8>] (worker_thread+0x204/0x4b4)
-[<c0134cc8>] (worker_thread) from [<c0139d68>] (kthread+0xd8/0xf4)
-[<c0139d68>] (kthread) from [<c0107938>] (ret_from_fork+0x14/0x3c)
-Code: c01399f8 c0a8a84c c0d60f94 e5903418 (e5130014) 
----[ end trace b4c40e72f416899b ]---
-Fixing recursive fault but reboot is needed!
+[drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
+[drm] No driver support for vblank timestamp query.
+usb 1-2: new high-speed USB device number 2 using tegra-xusb
+mmc2: new high speed SDHC card at address 0007
+mmcblk2: mmc2:0007 SD08G 7.42 GiB 
+GPT:Primary header alternate_lba != Alt. header my_lba
+GPT:3014591 != 15564799
+GPT:Primary header thinks Alt. header is not at the end of the disk.
+GPT:3014591 != 15564799
+GPT: Use GNU Parted to correct GPT errors.
+ mmcblk2: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
+Console: switching to colour frame buffer device 170x48
+drm drm: fb0:  frame buffer device
+[drm] Initialized tegra 0.0.0 20120330 on minor 0
+input: gpio-keys as /devices/soc0/gpio-keys/input/input4
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 19:47:14 UTC (946756034)
++1.8V_RUN_CAM: disabling
++1.2V_RUN_CAM_FRONT: disabling
++1.05V_RUN_CAM_REAR: disabling
++2.8V_RUN_TOUCH: disabling
++2.8V_RUN_CAM_AF: disabling
++1.8V_RUN_VPP_FUSE: disabling
++5V_VDD_TS_SW: disabling
+ALSA device list:
+  #0: tegra-hda at 0x70038000 irq 341
+  #1: GoogleNyanBig
+EXT4-fs (mmcblk0p3): couldn't mount as ext3 due to feature incompatibilities
+uvcvideo: Found UVC 1.00 device HD WebCam (1bcf:2c67)
+input: HD WebCam as /devices/soc0/70090000.usb/usb1/1-2/1-2:1.0/input/input5
+usb 1-3: new high-speed USB device number 3 using tegra-xusb
+EXT4-fs (mmcblk0p3): recovery complete
+EXT4-fs (mmcblk0p3): mounted filesystem with ordered data mode. Opts: (null)
+VFS: Mounted root (ext4 filesystem) on device 179:3.
+devtmpfs: mounted
+Freeing unused kernel memory: 1024K (c0c00000 - c0d00000)
+bash: cannot set terminal process group (-1): Inappropriate ioctl for device
+bash: no job control in this shell
+root@(none):/# [    6.524537] asix 1-3:1.0 eth0: register 'asix' at usb-70090000.usb-3, ASIX AX88772 USB 2.0 Ethernet, c8:d7:19:90:fa:65
+usb 1-1: new high-speed USB device number 4 using tegra-xusb
+usb-storage 1-1:1.0: USB Mass Storage device detected
+scsi host0: usb-storage 1-1:1.0
+scsi 0:0:0:0: Direct-Access     SanDisk  SDDR-113         9412 PQ: 0 ANSI: 0
+sd 0:0:0:0: [sda] 15523840 512-byte logical blocks: (7.95 GB/7.40 GiB)
+sd 0:0:0:0: [sda] Write Protect is off
+sd 0:0:0:0: [sda] Mode Sense: 03 00 00 00
+sd 0:0:0:0: [sda] No Caching mode page found
+sd 0:0:0:0: [sda] Assuming drive cache: write through
+ sda: sda1 sda2 sda3 sda4 sda5 sda6 sda7 sda8 sda9 sda10 sda11 sda12 sda13
+sd 0:0:0:0: [sda] Attached SCSI removable disk
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2016-10-04 22:58:03.369483717 -0700
+++ /tmp/foob	2016-10-04 22:58:03.373483725 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1801380 us
+Bootloader-Cpu Init at (time stamp): 1800582 us
 
 Pinmux changes applied in kernel way
 
@@ -59,47 +59,47 @@
 The proc BoardInfo: 0x0176:0x0001:0x01:0x43:0x00
 FastbootAddAtag: setting ATAG (0)
 
-Board Id Read Time: 3 us
+Board Id Read Time: 4 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1801354  us
-Bootloader (AosInit) time: 733439 us
-Bootloader Main Init time: 180454 us
-BlockDevMgrInit time: 157653 us
-PartMgrInit time: 8 us
+Miscellaneous avp time: 1800555  us
+Bootloader (AosInit) time: 734395 us
+Bootloader Main Init time: 180348 us
+BlockDevMgrInit time: 157486 us
+PartMgrInit time: 9 us
 StorMgrInit time: 31 us
 FsMgrInit time: 2 us
-Partition Table load: 76989 us 
-Display init time: 574378 us
-Kernel read time: 368 msec
-Load Kernel Size:   16001024 Bytes
-Bl Kernel ReadSpeed: 43481000 Bytes/sec
-Pre-Kernel Jump time: 1717611 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5810745 us
+Partition Table load: 76693 us 
+Display init time: 574935 us
+Kernel read time: 375 msec
+Load Kernel Size:   16224256 Bytes
+Bl Kernel ReadSpeed: 43264000 Bytes/sec
+Pre-Kernel Jump time: 1734659 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5836801 us
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-rc1+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Oct 3 08:27:54 PDT 2016
+Linux version 4.8.0-231430-g21f54dd (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Oct 4 09:03:02 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
 Memory limited to 1734MB
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
 cma: Reserved 16 MiB at 0x00000000eb400000
 /cpus/cpu@1: missing enable-method property
-percpu: Embedded 21 pages/cpu @ffff80006c59b000 s48256 r8192 d29568 u86016
+percpu: Embedded 21 pages/cpu @ffff80006c59b000 s47488 r8192 d30336 u86016
 Detected PIPT I-cache on CPU0
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 436968
 Kernel command line: tegraid=13.1.3.0.0 mem=1734M@2048M memtype=255 vpr=151M@3817M ddr_die=2048M@2048M section=256M commchip_id=0 usb_port_owner_info=0 lane_owner_info=0 emc_max_dvfs=0 touch_id=0@0 video=tegrafb no_console_suspend=1 console=ttyS0,115200n8 debug_uartport=lsport,0 earlyprintk=uart8250-32bit,0x70006000 usbcore.old_scheme_first=1 lp0_vec=0@0x0 tegra_fbmem=32899072@0xad012000 core_edp_mv=1150 core_edp_ma=4000 pmuboard=0x0176:0x0001:0x01:0x43:0x00 displayboard=0x0162:0x0001:0x00:0x42:0x00 power_supply=Adapter audio_codec=rt5640 board_info=0x0176:0x0001:0x01:0x43:0x00 tegraboot=sdmmc gpt gpt_sector=94207 modem_id=0 android.kerneltype=normal
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 Dentry cache hash table entries: 262144 (order: 9, 2097152 bytes)
 Inode-cache hash table entries: 131072 (order: 8, 1048576 bytes)
-Memory: 1707008K/1775616K available (7804K kernel code, 775K rwdata, 3684K rodata, 960K init, 269K bss, 52224K reserved, 16384K cma-reserved)
+Memory: 1706788K/1775616K available (7932K kernel code, 800K rwdata, 3716K rodata, 960K init, 275K bss, 52444K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff000008820000   (  7808 KB)
-    .rodata : 0xffff000008820000 - 0xffff000008bc0000   (  3712 KB)
-      .init : 0xffff000008bc0000 - 0xffff000008cb0000   (   960 KB)
-      .data : 0xffff000008cb0000 - 0xffff000008d71c00   (   775 KB)
-       .bss : 0xffff000008d71c00 - 0xffff000008db50b8   (   270 KB)
+      .text : 0xffff000008080000 - 0xffff000008840000   (  7936 KB)
+    .rodata : 0xffff000008840000 - 0xffff000008bf0000   (  3776 KB)
+      .init : 0xffff000008bf0000 - 0xffff000008ce0000   (   960 KB)
+      .data : 0xffff000008ce0000 - 0xffff000008da8200   (   801 KB)
+       .bss : 0xffff000008da8200 - 0xffff000008decf38   (   276 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -113,7 +113,7 @@
 NR_IRQS:64 nr_irqs:64 0
 /interrupt-controller@60004000: 160 interrupts forwarded to /interrupt-controller@50041000
 Tegra clk 127: register failed with -17
-Architected cp15 timer(s) running at 12.00MHz (virt).
+arm_arch_timer: Architected cp15 timer(s) running at 12.00MHz (virt).
 clocksource: arch_sys_counter: mask: 0xffffffffffffff max_cycles: 0x2c47f4ee7, max_idle_ns: 440795202497 ns
 sched_clock: 56 bits at 12MHz, resolution 83ns, wraps every 4398046511096ns
 Console: colour dummy device 80x25
@@ -134,7 +134,7 @@
 pinctrl core: initialized pinctrl subsystem
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffff000008826000, 1 data @ ffff000008cb4000)
+vdso: 2 pages (1 code @ ffff000008847000, 1 data @ ffff000008ce4000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -184,14 +184,13 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(1.107:1): initialized
+audit: type=2000 audit(1.179:1): initialized
 workingset: timestamp_bits=46 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
 Key type id_resolver registered
 Key type id_legacy registered
 nfs4filelayout_init: NFSv4 File Layout Driver Registering...
-fuse init (API version 7.25)
 9p: Installing v9fs 9p2000 file system support
 io scheduler noop registered
 io scheduler cfq registered (default)
@@ -248,8 +247,8 @@
 NET: Registered protocol family 17
 9pnet: Installing 9P2000 support
 Key type dns_resolver registered
-mmc0: new high speed MMC card at address 0001
 registered taskstats version 1
+mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 +VDD_LED: supplied by +VDD_MUX
@@ -262,8 +261,6 @@
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
 hctosys: unable to open rtc device (rtc0)
-sdhci-tegra 700b0400.sdhci: Got CD GPIO
-sdhci-tegra 700b0400.sdhci: Got WP GPIO
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +VDD_LED: disabling
 +5V_USB_HS: disabling
@@ -272,7 +269,9 @@
 +5V_HDMI_CON: disabling
 ALSA device list:
   No soundcards found.
-Freeing unused kernel memory: 960K (ffff800000bc0000 - ffff800000cb0000)
+Freeing unused kernel memory: 960K (ffff800000bf0000 - ffff800000ce0000)
+sdhci-tegra 700b0400.sdhci: Got CD GPIO
+sdhci-tegra 700b0400.sdhci: Got WP GPIO
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-10-04 22:58:03.389483757 -0700
+++ /tmp/foob	2016-10-04 22:58:03.393483764 -0700
@@ -45,20 +45,20 @@
 	 #################################################################
 	 #################################################################
 	 ###############################################
-	 911.1 KiB/s
+	 4.1 MiB/s
 done
-Bytes transferred = 5450600 (532b68 hex)
+Bytes transferred = 5449824 (532860 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 861.3 KiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x532b68 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x532860 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102b3
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-next-20161004-126151-gc7d3b91 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Oct 3 21:20:41 PDT 2016
+Linux version 4.8.0-next-20161005-126395-g0a766d8 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Oct 4 22:50:02 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: Compulab TrimSlice board
@@ -229,91 +229,130 @@
 bootconsole [earlycon0] disabled
 [drm] Initialized
 54280000.hdmi supply hdmi not found, using dummy regulator
-Unable to handle kernel paging request at virtual address fffffff8
-pgd = c0004000
-[fffffff8] *pgd=2fffd861, *pte=00000000, *ppte=00000000
-Internal error: Oops: 37 [#1] PREEMPT SMP ARM
-Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.8.0-next-20161004-126151-gc7d3b91 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-task: ef0513c0 task.stack: ef052000
-PC is at drm_debugfs_crtc_add+0x8/0x70
-LR is at drm_minor_register+0xa4/0x1b0
-pc : [<c0440b80>]    lr : [<c0428a60>]    psr: a0000013
-sp : ef053d08  ip : ef33bbd4  fp : 00000000
-r10: 00000001  r9 : 00000000  r8 : 00000000
-r7 : ef33a400  r6 : ef33a64c  r5 : ef33bbc0  r4 : fffffff8
-r3 : 00000001  r2 : 00000000  r1 : ef053c58  r0 : fffffff8
-Flags: NzCv  IRQs on  FIQs on  Mode SVC_32  ISA ARM  Segment none
-Control: 10c5387d  Table: 0000404a  DAC: 00000051
-Process swapper/0 (pid: 1, stack limit = 0xef052210)
-Stack: (0xef053d08 to 0xef054000)
-3d00:                   fffffff8 ef33bbc0 ef33a64c c0428a60 ef0513c0 ef33a400
-3d20: 00000000 00000000 c0d34794 00000000 00000000 c0428be8 ef33a400 ef2d0600
-3d40: c0d3467c c0442008 ef2d0618 00000000 c0d34794 00000000 ef2d0610 c0dabfec
-3d60: 00000000 c0407578 c040755c c045ac3c 00000000 ef053da0 c045af10 00000001
-3d80: c0dabfc8 c045922c ef220670 ef2ddc38 ef2d0610 ef2d0644 c0d33fa0 c045a9c4
-3da0: ef2d0610 00000001 00000000 ef2d0610 ef2d0610 c0d33fa0 ef0f9610 c045a0b4
-3dc0: ef2d0610 00000000 ef2d0618 c0458624 c0d6404c 60000013 00000000 c0817664
-3de0: ef2d0600 ef33a810 ef2d3480 ef2d07d8 ef2d07b0 c0952c4c ef2d0610 c0407744
-3e00: ef7e6040 ef2d0994 c0d3400c c0d33f68 ef33a810 ef2d0810 ef33b650 c0408058
-3e20: ef105a10 00000000 c0955f18 ef33a810 ef105a00 c04540cc c0453fbc ffffffed
-3e40: ef105a10 fffffdfb c0d34af4 00000000 00000007 c045c20c c045c1bc ef105a10
-3e60: c0dabfec 00000000 c0d34af4 c045ac3c ef105a10 c0d34af4 ef105a44 00000000
-3e80: 00000000 c045ada0 00000000 c0d34af4 c045acf4 c04592c8 ef009c5c ef0f7b34
-3ea0: c0d34af4 ef33c100 c0d34de8 c045a298 c0ac7004 c0d34af4 c0952c64 c0d34af4
-3ec0: c0952c64 c0952c60 00000000 c045b59c c045bea0 00000006 c0952c64 c045c124
-3ee0: 00000000 00000006 c0c3c830 c0d60000 c0c22650 c0c3c83c 00000000 c0c22678
-3f00: c0c53784 c01017ac c0d6404c 60000013 00000000 c0817664 00000000 ef053f18
-3f20: 00000000 c0d0b860 efffc12c c0920acc 00000103 c0139748 00000000 c0ae2530
-3f40: c0b6111c 00000000 00000006 00000006 c0d0b828 efffc0c0 c0c53784 00000006
-3f60: c0c3c830 c0d60000 00000103 c0c3c83c c0c005a4 c0c00d9c 00000006 00000006
-3f80: 00000000 c0c005a4 00000000 c0812f0c 00000000 00000000 00000000 00000000
-3fa0: 00000000 c0812f14 00000000 c0107938 00000000 00000000 00000000 00000000
-3fc0: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
-3fe0: 00000000 00000000 00000000 00000000 00000013 00000000 51c7bfff c7ef7bff
-[<c0440b80>] (drm_debugfs_crtc_add) from [<c0428a60>] (drm_minor_register+0xa4/0x1b0)
-[<c0428a60>] (drm_minor_register) from [<c0428be8>] (drm_dev_register+0x7c/0xd0)
-[<c0428be8>] (drm_dev_register) from [<c0442008>] (host1x_drm_probe+0x38/0x90)
-[<c0442008>] (host1x_drm_probe) from [<c0407578>] (host1x_device_probe+0x1c/0x28)
-[<c0407578>] (host1x_device_probe) from [<c045ac3c>] (driver_probe_device+0x1f0/0x2a8)
-[<c045ac3c>] (driver_probe_device) from [<c045922c>] (bus_for_each_drv+0x44/0x8c)
-[<c045922c>] (bus_for_each_drv) from [<c045a9c4>] (__device_attach+0x9c/0x100)
-[<c045a9c4>] (__device_attach) from [<c045a0b4>] (bus_probe_device+0x84/0x8c)
-[<c045a0b4>] (bus_probe_device) from [<c0458624>] (device_add+0x380/0x528)
-[<c0458624>] (device_add) from [<c0407744>] (host1x_subdev_register+0xb0/0xd4)
-[<c0407744>] (host1x_subdev_register) from [<c0408058>] (host1x_client_register+0xf4/0x11c)
-[<c0408058>] (host1x_client_register) from [<c04540cc>] (gr3d_probe+0x110/0x23c)
-[<c04540cc>] (gr3d_probe) from [<c045c20c>] (platform_drv_probe+0x50/0xb0)
-[<c045c20c>] (platform_drv_probe) from [<c045ac3c>] (driver_probe_device+0x1f0/0x2a8)
-[<c045ac3c>] (driver_probe_device) from [<c045ada0>] (__driver_attach+0xac/0xb0)
-[<c045ada0>] (__driver_attach) from [<c04592c8>] (bus_for_each_dev+0x54/0x88)
-[<c04592c8>] (bus_for_each_dev) from [<c045a298>] (bus_add_driver+0xe8/0x1f4)
-[<c045a298>] (bus_add_driver) from [<c045b59c>] (driver_register+0x78/0xf4)
-[<c045b59c>] (driver_register) from [<c045c124>] (__platform_register_drivers+0x28/0x84)
-[<c045c124>] (__platform_register_drivers) from [<c0c22678>] (host1x_drm_init+0x28/0x50)
-[<c0c22678>] (host1x_drm_init) from [<c01017ac>] (do_one_initcall+0x3c/0x16c)
-[<c01017ac>] (do_one_initcall) from [<c0c00d9c>] (kernel_init_freeable+0x120/0x1ec)
-[<c0c00d9c>] (kernel_init_freeable) from [<c0812f14>] (kernel_init+0x8/0x118)
-[<c0812f14>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
-Code: e584334c e8bd8010 e92d4070 e1a04000 (e5943000) 
----[ end trace d4b9fa67a3662aaa ]---
-Kernel panic - not syncing: Attempted to kill init! exitcode=0x0000000b
-
-CPU1: stopping
-CPU: 1 PID: 0 Comm: swapper/1 Tainted: G      D         4.8.0-next-20161004-126151-gc7d3b91 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c010f248>] (unwind_backtrace) from [<c010a84c>] (show_stack+0x10/0x14)
-[<c010a84c>] (show_stack) from [<c034469c>] (dump_stack+0x84/0xa4)
-[<c034469c>] (dump_stack) from [<c010d99c>] (handle_IPI+0x154/0x184)
-[<c010d99c>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x84/0x90)
-[<c0101430>] (gic_handle_irq) from [<c010b3cc>] (__irq_svc+0x6c/0xa8)
-Exception stack(0xef077f90 to 0xef077fd8)
-7f80:                                     00000001 00000000 ef077fe8 c0118e00
-7fa0: 00000000 00000002 c0d02494 c0d0242c 00000000 c0c58e78 00000000 c0d0249c
-7fc0: c0c563c0 ef077fe0 c0108374 c0108378 60000013 ffffffff
-[<c010b3cc>] (__irq_svc) from [<c0108378>] (arch_cpu_idle+0x30/0x3c)
-[<c0108378>] (arch_cpu_idle) from [<c015a334>] (cpu_startup_entry+0x13c/0x1f4)
-[<c015a334>] (cpu_startup_entry) from [<001014cc>] (0x1014cc)
----[ end Kernel panic - not syncing: Attempted to kill init! exitcode=0x0000000b
-
+[drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
+[drm] No driver support for vblank timestamp query.
+drm drm: No connectors reported connected with modes
+[drm] Cannot find any crtc or sizes - going 1024x768
+Console: switching to colour frame buffer device 128x48
+drm drm: fb0:  frame buffer device
+[drm] Initialized tegra 0.0.0 20120330 on minor 0
+loop: module loaded
+m25p80 spi32766.0: found s25fl008k, expected w25q80bl
+m25p80 spi32766.0: s25fl008k (1024 Kbytes)
+libphy: Fixed MDIO Bus: probed
+CAN device driver interface
+igb: Intel(R) Gigabit Ethernet Network Driver - version 5.4.0-k
+igb: Copyright (c) 2007-2014 Intel Corporation.
+r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
+r8169 0000:01:00.0: enabling device (0140 -> 0143)
+r8169 0000:01:00.0 eth0: RTL8168d/8111d at 0xf085d000, 00:01:c0:13:7d:63, XID 081000c0 IRQ 296
+r8169 0000:01:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
+pegasus: v0.9.3 (2013/04/25), Pegasus/Pegasus II USB Ethernet driver
+usbcore: registered new interface driver pegasus
+usbcore: registered new interface driver asix
+usbcore: registered new interface driver ax88179_178a
+usbcore: registered new interface driver cdc_ether
+usbcore: registered new interface driver smsc75xx
+usbcore: registered new interface driver smsc95xx
+usbcore: registered new interface driver net1080
+usbcore: registered new interface driver cdc_subset
+usbcore: registered new interface driver zaurus
+usbcore: registered new interface driver cdc_ncm
+tegra-phy c5004000.usb-phy: no vbus regulator
+tegra-phy c5008000.usb-phy: no vbus regulator
+ehci_hcd: USB 2.0 'Enhanced' Host Controller (EHCI) Driver
+ehci-pci: EHCI PCI platform driver
+tegra-ehci: Tegra EHCI driver
+ulpi_phy_power_on: ulpi write failed
+tegra-ehci c5004000.usb: Failed to power on the phy
+tegra-ehci: probe of c5004000.usb failed with error -110
+tegra-ehci c5008000.usb: EHCI Host Controller
+tegra-ehci c5008000.usb: new USB bus registered, assigned bus number 1
+tegra-ehci c5008000.usb: irq 290, io mem 0xc5008000
+tegra-ehci c5008000.usb: USB 2.0 started, EHCI 1.00
+hub 1-0:1.0: USB hub found
+hub 1-0:1.0: 1 port detected
+usbcore: registered new interface driver cdc_acm
+cdc_acm: USB Abstract Control Model driver for USB modems and ISDN adapters
+usbcore: registered new interface driver cdc_wdm
+usbcore: registered new interface driver usb-storage
+mousedev: PS/2 mouse device common for all mice
+rtc-em3027 2-0056: rtc core: registered rtc-em3027 as rtc0
+tegra_rtc 7000e000.rtc: rtc core: registered 7000e000.rtc as rtc1
+tegra_rtc 7000e000.rtc: Tegra internal Real Time Clock
+i2c /dev entries driver
+usbcore: registered new interface driver uvcvideo
+USB Video Class driver (1.1.1)
+gspca_main: v2.14.0 registered
+sdhci: Secure Digital Host Controller Interface driver
+sdhci: Copyright(c) Pierre Ossman
+sdhci-pltfm: SDHCI platform and OF driver helper
+mmc0: Invalid maximum block size, assuming 512 bytes
+mmc0: SDHCI controller on c8000000.sdhci [c8000000.sdhci] using ADMA
+sdhci-tegra c8000600.sdhci: Got CD GPIO
+sdhci-tegra c8000600.sdhci: Got WP GPIO
+mmc1: Invalid maximum block size, assuming 512 bytes
+mmc1: SDHCI controller on c8000600.sdhci [c8000600.sdhci] using ADMA
+usbcore: registered new interface driver usbhid
+usbhid: USB HID core driver
+tlv320aic23-codec 2-001a: Control not supported for path LLINEIN -> [NULL] -> Line Input
+tlv320aic23-codec 2-001a: ASoC: no dapm match for LLINEIN --> NULL --> Line Input
+tlv320aic23-codec 2-001a: ASoC: Failed to add route LLINEIN -> NULL -> Line Input
+tlv320aic23-codec 2-001a: Control not supported for path RLINEIN -> [NULL] -> Line Input
+tlv320aic23-codec 2-001a: ASoC: no dapm match for RLINEIN --> NULL --> Line Input
+tlv320aic23-codec 2-001a: ASoC: Failed to add route RLINEIN -> NULL -> Line Input
+tlv320aic23-codec 2-001a: Control not supported for path MICIN -> [NULL] -> Mic Input
+tlv320aic23-codec 2-001a: ASoC: no dapm match for MICIN --> NULL --> Mic Input
+tlv320aic23-codec 2-001a: ASoC: Failed to add route MICIN -> NULL -> Mic Input
+tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
+mmc0: host does not support reading read-only switch, assuming write-enable
+mmc0: new high speed SDHC card at address b368
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
+ mmcblk0: p1
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
+mmc1: new high speed SD card at address b368
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
+ mmcblk1: p1 p2
+hub 1-1:1.0: USB hub found
+hub 1-1:1.0: 4 ports detected
+NET: Registered protocol family 10
+mip6: Mobile IPv6
+sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
+NET: Registered protocol family 17
+NET: Registered protocol family 15
+can: controller area network core (rev 20120528 abi 9)
+usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
+NET: Registered protocol family 29
+can: raw protocol (rev 20120528)
+can: broadcast manager protocol (rev 20160617 t)
+can: netlink gateway (rev 20130117) max_hops=1
+Bluetooth: RFCOMM socket layer initialized
+Bluetooth: RFCOMM ver 1.11
+Bluetooth: BNEP (Ethernet Emulation) ver 1.3
+Bluetooth: BNEP socket layer initialized
+Bluetooth: HIDP (Human Interface Emulation) ver 1.2
+Bluetooth: HIDP socket layer initialized
+Key type dns_resolver registered
+Registering SWP/SWPB emulation handler
+utmi_phy_clk_enable: timeout waiting for phy to stabilize
+tegra-ehci c5000000.usb: EHCI Host Controller
+tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
+tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
+tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
+hub 2-0:1.0: USB hub found
+hub 2-0:1.0: 1 port detected
+input: gpio-keys as /devices/soc0/gpio-keys/input/input0
+rtc-em3027 2-0056: setting system clock to 2015-02-05 19:02:52 UTC (1423162972)
+ALSA device list:
+  #0: tegra-trimslice
+asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
+EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
+random: fast init done
+EXT4-fs (mmcblk0p1): recovery complete
+EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
+VFS: Mounted root (ext3 filesystem) on device 179:1.
+devtmpfs: mounted
+Freeing unused kernel memory: 1024K (c0c00000 - c0d00000)
+bash: cannot set terminal process group (-1): Inappropriate ioctl for device
+bash: no job control in this shell
+root@(none):/# 
\ No newline at end of file
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2016-10-04 22:58:03.409483796 -0700
+++ /tmp/foob	2016-10-04 22:58:03.413483804 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 731246 us
-Basic modules init took 348697 us
-Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14390 us
-Carveout took 252726 us
-CPU initialization took 92350 us
-Total time taken by TegraBoot 708163 us
+Tegraboot started after 742870 us
+Basic modules init took 348965 us
+Sec Bootdevice Read Time = 203 ms, Read Size = 8459 KB
+Next stage binary read took 15907 us
+Carveout took 252721 us
+CPU initialization took 92312 us
+Total time taken by TegraBoot 709905 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.570] LPDDR4 Training: Number of tables = 10
+64b[0001.583] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -121,7 +121,7 @@
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
 43755 bytes read in 454 ms (93.8 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 502 ms (3.1 MiB/s)
+1627787 bytes read in 500 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-next-20161004-126151-gc7d3b91 (pwalmsley@sellafield) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Oct 3 21:22:51 PDT 2016
+Linux version 4.8.0-next-20161005-126395-g0a766d8 (pwalmsley@sellafield) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Oct 4 22:52:13 PDT 2016
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -166,7 +166,7 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffff80007a000000-ffff80007dffffff]
-Memory: 2978504K/3131392K available (8252K kernel code, 852K rwdata, 3900K rodata, 1024K init, 283K bss, 136504K reserved, 16384K cma-reserved)
+Memory: 2978504K/3131392K available (8252K kernel code, 852K rwdata, 3904K rodata, 1024K init, 283K bss, 136504K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
@@ -258,7 +258,7 @@
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 PCI: CLS 0 bytes, default 128
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 8087c000
+kvm [1]: IDMAP page: 8087d000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
@@ -266,7 +266,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.463:1): initialized
+audit: type=2000 audit(0.547:1): initialized
 workingset: timestamp_bits=46 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -347,6 +347,6 @@
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (ffff800000c70000 - ffff800000d70000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
-bash: no job contr[    1.381230] random: fast init done
+bash: no job contr[    1.427035] random: fast init done
 ol in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2016-10-04 22:58:03.425483828 -0700
+++ /tmp/foob	2016-10-04 22:58:03.429483835 -0700
@@ -17,7 +17,7 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2832692K/2988032K available (8252K kernel code, 852K rwdata, 3900K rodata, 1024K init, 283K bss, 138956K reserved, 16384K cma-reserved)
+Memory: 2832692K/2988032K available (8252K kernel code, 852K rwdata, 3904K rodata, 1024K init, 283K bss, 138956K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
@@ -54,7 +54,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x4b80: expected 0x2004f400
+boot misc3 0x4eb8: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -131,7 +131,7 @@
 Unpacking initramfs...
 Freeing initrd memory: 2360K (ffff80000a06f000 - ffff80000a2bd000)
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 8087c000
+kvm [1]: IDMAP page: 8087d000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
@@ -139,7 +139,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.603:1): initialized
+audit: type=2000 audit(0.711:1): initialized
 workingset: timestamp_bits=46 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -227,16 +227,16 @@
 Key type dns_resolver registered
 registered taskstats version 1
 tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
-input: gpio-keys as /devices/platform/gpio-keys/input/input0
-tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 BGND3R 29.1 GiB 
 mmcblk0boot0: mmc0:0001 BGND3R partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
-max77686-rtc max77620-rtc: setting system clock to 2000-07-15 09:52:35 UTC (963654755)
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
 tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
+input: gpio-keys as /devices/platform/gpio-keys/input/input0
+tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
+max77686-rtc max77620-rtc: setting system clock to 2000-07-16 11:21:33 UTC (963746493)
 PP1200_AVDD: disabling
 PP1200_RCAM: disabling
 PP2800L_RCAM: disabling
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-10-04 22:58:03.437483851 -0700
+++ /tmp/foob	2016-10-04 22:58:03.437483851 -0700
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 ###############################################
-	 3.2 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 5450600 (532b68 hex)
+Bytes transferred = 5449824 (532860 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 51572 (c974 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x532b68 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x532860 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff973 ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.8.0-next-20161004-126151-gc7d3b91 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Oct 3 21:20:41 PDT 2016
+Linux version 4.8.0-next-20161005-126395-g0a766d8 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Oct 4 22:50:02 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -351,171 +351,38 @@
 r8169 0000:01:00.0: enabling device (0140 -> 0143)
 r8169 0000:01:00.0 eth0: RTL8168evl/8111evl at 0xf087e000, 02:04:4b:01:60:a6, XID 0c900800 IRQ 375
 r8169 0000:01:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
-Unable to handle kernel paging request at virtual address fffffff8
-pgd = c0004000
-[fffffff8] *pgd=afffd861, *pte=00000000, *ppte=00000000
-Internal error: Oops: 37 [#1] PREEMPT SMP ARM
-Modules linked in:
-CPU: 0 PID: 37 Comm: kworker/0:1 Not tainted 4.8.0-next-20161004-126151-gc7d3b91 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-Workqueue: events deferred_probe_work_func
-task: eea66280 task.stack: eea70000
-PC is at drm_debugfs_crtc_add+0x8/0x70
-LR is at drm_minor_register+0xa4/0x1b0
-pc : [<c0440b80>]    lr : [<c0428a60>]    psr: a0000013
-sp : eea71d20  ip : eeb51a94  fp : c0d02100
-r10: 00000001  r9 : 00000000  r8 : 00000000
-r7 : eeb56400  r6 : eeb5664c  r5 : eeb51a80  r4 : fffffff8
-r3 : 00000001  r2 : 00000000  r1 : eea71c70  r0 : fffffff8
-Flags: NzCv  IRQs on  FIQs on  Mode SVC_32  ISA ARM  Segment none
-Control: 10c5387d  Table: 8000404a  DAC: 00000051
-Process kworker/0:1 (pid: 37, stack limit = 0xeea70210)
-Stack: (0xeea71d20 to 0xeea72000)
-1d20: fffffff8 eeb51a80 eeb5664c c0428a60 eea66280 eeb56400 00000000 00000000
-1d40: c0d34794 0000000c 00000000 c0428be8 eeb56400 eeb8d200 c0d3467c c0442008
-1d60: eeb8d218 00000000 c0d34794 0000000c eeb8d210 c0dabfec 00000000 c0407578
-1d80: c040755c c045ac3c 00000000 eea71db8 c045af10 00000001 c0dabfc8 c045922c
-1da0: eeaa6470 eeb8fd38 eeb8d210 eeb8d244 c0d33fa0 c045a9c4 eeb8d210 00000001
-1dc0: c0d02100 eeb8d210 eeb8d210 c0d33fa0 ee99be10 c045a0b4 eeb8d210 00000000
-1de0: eeb8d218 c0458624 c0d6404c 60000013 c0d02100 c0817664 eeb8d200 eeb56810
-1e00: ee00b180 eeb8d3d8 eeb8d3b0 eef8f580 eeb8d210 c0407744 eefc6c60 eeb8d594
-1e20: c0d3400c c0d33f68 eeb56810 eeb8d410 00000000 c0408058 eeb56810 ee9a1c10
-1e40: 00000000 ee9a1c00 00000000 c044bb4c 00000000 ee99e840 eeb56810 00000000
-1e60: ffffffed ee9a1c10 fffffdfb c0d348f8 0000000c c045c20c c045c1bc ee9a1c10
-1e80: c0dabfec 00000000 c0d348f8 c045ac3c 00000000 eea71ec0 c045af10 00000001
-1ea0: 00000000 c045922c ee809a70 eeb8fc38 ee9a1c10 ee9a1c44 c0d34de8 c045a9c4
-1ec0: ee9a1c10 00000001 c0d02100 ee9a1c10 ee9a1c10 c0d34de8 eef92800 c045a0b4
-1ee0: ee9a1c10 c0d34d70 c0d34d90 c045a4e8 eea4b880 c0d34d98 eef8f580 c0134890
-1f00: eef8f580 00000000 eea4b880 eea4b898 00000001 eef8f580 eea4b898 eef8f580
-1f20: 00000008 c0134ab4 eef928f5 eea4b880 eef8f598 c0134cc8 00000000 eea4f400
-1f40: 00000000 eea4f400 00000000 eea4b880 c0134ac4 00000000 00000000 00000000
-1f60: 00000000 c0139d68 ffffffff 00000000 7ffffbff eea4b880 00000000 00000000
-1f80: eea71f80 eea71f80 00000000 00000000 eea71f90 eea71f90 eea71fac eea4f400
-1fa0: c0139c90 00000000 00000000 c0107938 00000000 00000000 00000000 00000000
-1fc0: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
-1fe0: 00000000 00000000 00000000 00000000 00000013 00000000 ffbfff7b fff6fffd
-[<c0440b80>] (drm_debugfs_crtc_add) from [<c0428a60>] (drm_minor_register+0xa4/0x1b0)
-[<c0428a60>] (drm_minor_register) from [<c0428be8>] (drm_dev_register+0x7c/0xd0)
-[<c0428be8>] (drm_dev_register) from [<c0442008>] (host1x_drm_probe+0x38/0x90)
-[<c0442008>] (host1x_drm_probe) from [<c0407578>] (host1x_device_probe+0x1c/0x28)
-[<c0407578>] (host1x_device_probe) from [<c045ac3c>] (driver_probe_device+0x1f0/0x2a8)
-[<c045ac3c>] (driver_probe_device) from [<c045922c>] (bus_for_each_drv+0x44/0x8c)
-[<c045922c>] (bus_for_each_drv) from [<c045a9c4>] (__device_attach+0x9c/0x100)
-[<c045a9c4>] (__device_attach) from [<c045a0b4>] (bus_probe_device+0x84/0x8c)
-[<c045a0b4>] (bus_probe_device) from [<c0458624>] (device_add+0x380/0x528)
-[<c0458624>] (device_add) from [<c0407744>] (host1x_subdev_register+0xb0/0xd4)
-[<c0407744>] (host1x_subdev_register) from [<c0408058>] (host1x_client_register+0xf4/0x11c)
-[<c0408058>] (host1x_client_register) from [<c044bb4c>] (tegra_hdmi_probe+0x1c8/0x2c8)
-[<c044bb4c>] (tegra_hdmi_probe) from [<c045c20c>] (platform_drv_probe+0x50/0xb0)
-[<c045c20c>] (platform_drv_probe) from [<c045ac3c>] (driver_probe_device+0x1f0/0x2a8)
-[<c045ac3c>] (driver_probe_device) from [<c045922c>] (bus_for_each_drv+0x44/0x8c)
-[<c045922c>] (bus_for_each_drv) from [<c045a9c4>] (__device_attach+0x9c/0x100)
-[<c045a9c4>] (__device_attach) from [<c045a0b4>] (bus_probe_device+0x84/0x8c)
-[<c045a0b4>] (bus_probe_device) from [<c045a4e8>] (deferred_probe_work_func+0x60/0x8c)
-[<c045a4e8>] (deferred_probe_work_func) from [<c0134890>] (process_one_work+0x120/0x31c)
-[<c0134890>] (process_one_work) from [<c0134ab4>] (process_scheduled_works+0x28/0x38)
-[<c0134ab4>] (process_scheduled_works) from [<c0134cc8>] (worker_thread+0x204/0x4b4)
-[<c0134cc8>] (worker_thread) from [<c0139d68>] (kthread+0xd8/0xf4)
-[<c0139d68>] (kthread) from [<c0107938>] (ret_from_fork+0x14/0x3c)
-Code: e584334c e8bd8010 e92d4070 e1a04000 (e5943000) 
----[ end trace 7b4686ebd9062dc8 ]---
-Unable to handle kernel paging request at virtual address ffffffec
-pgd = c0004000
-[ffffffec] *pgd=afffd861, *pte=00000000, *ppte=00000000
-Internal error: Oops: 37 [#2] PREEMPT SMP ARM
-Modules linked in:
-CPU: 0 PID: 37 Comm: kworker/0:1 Tainted: G      D         4.8.0-next-20161004-126151-gc7d3b91 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-task: eea66280 task.stack: eea70000
-PC is at kthread_data+0x4/0xc
-LR is at wq_worker_sleeping+0x8/0xc8
-pc : [<c013a35c>]    lr : [<c0135930>]    psr: 20000193
-sp : eea71ae0  ip : eef8ff28  fp : eea71b2c
-r10: c0d02e08  r9 : c0d62040  r8 : c0c59a00
-r7 : eea666c8  r6 : eea666c0  r5 : eea66280  r4 : eef8fa00
-r3 : 00000000  r2 : 00000000  r1 : eea66280  r0 : eea66280
-Flags: nzCv  IRQs off  FIQs on  Mode SVC_32  ISA ARM  Segment none
-Control: 10c5387d  Table: 8000404a  DAC: 00000051
-Process kworker/0:1 (pid: 37, stack limit = 0xeea70210)
-Stack: (0xeea71ae0 to 0xeea72000)
-1ae0: 00000020 c0814be8 635688c0 d84156c5 635688c0 d84156c5 745d1746 c01420e4
-1b00: eea54a80 00000026 00000000 eea7175c ee8593c0 eea66640 eea71b38 0000000b
-1b20: 00000001 00000001 eea71b34 c01420e4 eea66280 c0122e64 eea71b38 eea71b38
-1b40: 00000001 c0d602c4 eea71cd0 c0d07638 60000113 0000000b 00000001 c0440b82
-1b60: c0440b84 c010ac8c eea70210 0000000b 00000002 c0d07638 bf000000 00000008
-1b80: 65a71cd0 33343835 20633433 64623865 30313038 32396520 37303464 31652030
-1ba0: 30343061 28203030 34393565 30303033 a0002029 c01b1e74 000001ff fffffff8
-1bc0: 00000037 00000000 eea71cd0 fffffff8 eea70000 00000001 c0d02100 c011a00c
-1be0: 00000037 c01157c0 d84156c5 c01f3b10 00000000 ee53d5f8 00000000 ee808f00
-1c00: a0000013 024000c0 c02150e0 00000037 c01154cc fffffff8 c0d07f14 eea71cd0
-1c20: eea70000 00000001 c0d02100 c010128c 00000002 ee53e5a8 01c9c380 c0216748
-1c40: 00000000 00000000 00000001 ee53d5a8 00008124 c09528b0 c09188a0 eeb51dc0
-1c60: ee53d8c8 eeb56400 eeb51a9c eea71c80 c0d6404c 60000013 00000002 c0817664
-1c80: 00000001 eea71c80 ee53d8c8 eeb51dc0 c0d34794 eeb51a80 00000002 c04408b8
-1ca0: c09528b0 00000000 eeb51a94 c0d34774 00000000 eeb51a80 c0440b80 a0000013
-1cc0: ffffffff eea71d04 00000000 c010b338 fffffff8 eea71c70 00000000 00000001
-1ce0: fffffff8 eeb51a80 eeb5664c eeb56400 00000000 00000000 00000001 c0d02100
-1d00: eeb51a94 eea71d20 c0428a60 c0440b80 a0000013 ffffffff 00000051 00000000
-1d20: fffffff8 eeb51a80 eeb5664c c0428a60 eea66280 eeb56400 00000000 00000000
-1d40: c0d34794 0000000c 00000000 c0428be8 eeb56400 eeb8d200 c0d3467c c0442008
-1d60: eeb8d218 00000000 c0d34794 0000000c eeb8d210 c0dabfec 00000000 c0407578
-1d80: c040755c c045ac3c 00000000 eea71db8 c045af10 00000001 c0dabfc8 c045922c
-1da0: eeaa6470 eeb8fd38 eeb8d210 eeb8d244 c0d33fa0 c045a9c4 eeb8d210 00000001
-1dc0: c0d02100 eeb8d210 eeb8d210 c0d33fa0 ee99be10 c045a0b4 eeb8d210 00000000
-1de0: eeb8d218 c0458624 c0d6404c 60000013 c0d02100 c0817664 eeb8d200 eeb56810
-1e00: ee00b180 eeb8d3d8 eeb8d3b0 eef8f580 eeb8d210 c0407744 eefc6c60 eeb8d594
-1e20: c0d3400c c0d33f68 eeb56810 eeb8d410 00000000 c0408058 eeb56810 ee9a1c10
-1e40: 00000000 ee9a1c00 00000000 c044bb4c 00000000 ee99e840 eeb56810 00000000
-1e60: ffffffed ee9a1c10 fffffdfb c0d348f8 0000000c c045c20c c045c1bc ee9a1c10
-1e80: c0dabfec 00000000 c0d348f8 c045ac3c 00000000 eea71ec0 c045af10 00000001
-1ea0: 00000000 c045922c ee809a70 eeb8fc38 ee9a1c10 ee9a1c44 c0d34de8 c045a9c4
-1ec0: ee9a1c10 00000001 c0d02100 ee9a1c10 ee9a1c10 c0d34de8 eef92800 c045a0b4
-1ee0: ee9a1c10 c0d34d70 c0d34d90 c045a4e8 eea4b880 c0d34d98 eef8f580 c0134890
-1f00: eef8f580 00000000 eea4b880 eea4b898 00000001 eef8f580 eea4b898 eef8f580
-1f20: 00000008 c0134ab4 eef928f5 eea4b880 eef8f598 c0134cc8 00000000 eea4f400
-1f40: 00000000 eea4f400 00000000 eea4b880 c0134ac4 00000000 00000000 00000000
-1f60: 00000000 c0139d68 ffffffff 00000000 7ffffbff eea4b880 00000000 00000000
-1f80: eea71f80 eea71f80 00000001 00010001 eea71f90 eea71f90 eea71fac eea4f400
-1fa0: c0139c90 00000000 00000000 c0107938 00000000 00000000 00000000 00000000
-1fc0: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
-1fe0: 00000000 00000000 00000000 00000000 00000013 00000000 ffbfff7b fff6fffd
-[<c013a35c>] (kthread_data) from [<c0135930>] (wq_worker_sleeping+0x8/0xc8)
-[<c0135930>] (wq_worker_sleeping) from [<c0814be8>] (__schedule+0x37c/0x60c)
-[<c0814be8>] (__schedule) from [<c01420e4>] (do_task_dead+0x8c/0x90)
-[<c01420e4>] (do_task_dead) from [<c0122e64>] (do_exit+0x688/0xa04)
-[<c0122e64>] (do_exit) from [<c010ac8c>] (die+0x43c/0x484)
-[<c010ac8c>] (die) from [<c011a00c>] (__do_kernel_fault.part.0+0x54/0x1e4)
-[<c011a00c>] (__do_kernel_fault.part.0) from [<c01157c0>] (do_page_fault+0x2f4/0x380)
-[<c01157c0>] (do_page_fault) from [<c010128c>] (do_DataAbort+0x34/0xb8)
-[<c010128c>] (do_DataAbort) from [<c010b338>] (__dabt_svc+0x58/0x80)
-Exception stack(0xeea71cd0 to 0xeea71d18)
-1cc0:                                     fffffff8 eea71c70 00000000 00000001
-1ce0: fffffff8 eeb51a80 eeb5664c eeb56400 00000000 00000000 00000001 c0d02100
-1d00: eeb51a94 eea71d20 c0428a60 c0440b80 a0000013 ffffffff
-[<c010b338>] (__dabt_svc) from [<c0440b80>] (drm_debugfs_crtc_add+0x8/0x70)
-[<c0440b80>] (drm_debugfs_crtc_add) from [<c0428a60>] (drm_minor_register+0xa4/0x1b0)
-[<c0428a60>] (drm_minor_register) from [<c0428be8>] (drm_dev_register+0x7c/0xd0)
-[<c0428be8>] (drm_dev_register) from [<c0442008>] (host1x_drm_probe+0x38/0x90)
-[<c0442008>] (host1x_drm_probe) from [<c0407578>] (host1x_device_probe+0x1c/0x28)
-[<c0407578>] (host1x_device_probe) from [<c045ac3c>] (driver_probe_device+0x1f0/0x2a8)
-[<c045ac3c>] (driver_probe_device) from [<c045922c>] (bus_for_each_drv+0x44/0x8c)
-[<c045922c>] (bus_for_each_drv) from [<c045a9c4>] (__device_attach+0x9c/0x100)
-[<c045a9c4>] (__device_attach) from [<c045a0b4>] (bus_probe_device+0x84/0x8c)
-[<c045a0b4>] (bus_probe_device) from [<c0458624>] (device_add+0x380/0x528)
-[<c0458624>] (device_add) from [<c0407744>] (host1x_subdev_register+0xb0/0xd4)
-[<c0407744>] (host1x_subdev_register) from [<c0408058>] (host1x_client_register+0xf4/0x11c)
-[<c0408058>] (host1x_client_register) from [<c044bb4c>] (tegra_hdmi_probe+0x1c8/0x2c8)
-[<c044bb4c>] (tegra_hdmi_probe) from [<c045c20c>] (platform_drv_probe+0x50/0xb0)
-[<c045c20c>] (platform_drv_probe) from [<c045ac3c>] (driver_probe_device+0x1f0/0x2a8)
-[<c045ac3c>] (driver_probe_device) from [<c045922c>] (bus_for_each_drv+0x44/0x8c)
-[<c045922c>] (bus_for_each_drv) from [<c045a9c4>] (__device_attach+0x9c/0x100)
-[<c045a9c4>] (__device_attach) from [<c045a0b4>] (bus_probe_device+0x84/0x8c)
-[<c045a0b4>] (bus_probe_device) from [<c045a4e8>] (deferred_probe_work_func+0x60/0x8c)
-[<c045a4e8>] (deferred_probe_work_func) from [<c0134890>] (process_one_work+0x120/0x31c)
-[<c0134890>] (process_one_work) from [<c0134ab4>] (process_scheduled_works+0x28/0x38)
-[<c0134ab4>] (process_scheduled_works) from [<c0134cc8>] (worker_thread+0x204/0x4b4)
-[<c0134cc8>] (worker_thread) from [<c0139d68>] (kthread+0xd8/0xf4)
-[<c0139d68>] (kthread) from [<c0107938>] (ret_from_fork+0x14/0x3c)
-Code: c01399f8 c0a8a84c c0d60f94 e5903418 (e5130014) 
----[ end trace 7b4686ebd9062dc9 ]---
-Fixing recursive fault but reboot is needed!
+[drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
+[drm] No driver support for vblank timestamp query.
+drm drm: No connectors reported connected with modes
+[drm] Cannot find any crtc or sizes - going 1024x768
+Console: switching to colour frame buffer device 128x48
+drm drm: fb0:  frame buffer device
+[drm] Initialized tegra 0.0.0 20120330 on minor 0
+utmi_phy_clk_enable: timeout waiting for phy to stabilize
+tegra-ehci 7d004000.usb: EHCI Host Controller
+tegra-ehci 7d004000.usb: new USB bus registered, assigned bus number 1
+tegra-ehci 7d004000.usb: irq 343, io mem 0x7d004000
+tegra-ehci 7d004000.usb: USB 2.0 started, EHCI 1.00
+hub 1-0:1.0: USB hub found
+hub 1-0:1.0: 1 port detected
+tegra-ehci 7d008000.usb: EHCI Host Controller
+tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 2
+tegra-ehci 7d008000.usb: irq 344, io mem 0x7d008000
+tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.00
+hub 2-0:1.0: USB hub found
+hub 2-0:1.0: 1 port detected
+tps65910-rtc tps65910-rtc: setting system clock to 2000-01-01 00:00:03 UTC (946684803)
+vdd_sata,avdd_plle: disabling
+usb1_vbus: disabling
+ALSA device list:
+  #0: NVIDIA Tegra Beaver
+EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
+EXT4-fs (mmcblk1p1): warning: mounting fs with errors, running e2fsck is recommended
+EXT4-fs (mmcblk1p1): recovery complete
+EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
+VFS: Mounted root (ext3 filesystem) on device 179:1.
+devtmpfs: mounted
+Freeing unused kernel memory: 1024K (c0c00000 - c0d00000)
+bash: cannot set terminal process group (-1): Inappropriate ioctl for device
+bash: no job control in this shell
+root@(none):/# 
\ No newline at end of file
