Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-06-27 00:26:40.368632995 -0700
+++ /tmp/foob	2016-06-27 00:26:40.372632997 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc4-next-20160624+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Fri Jun 24 00:21:21 PDT 2016
+Linux version 4.7.0-rc4-next-20160627+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Jun 27 00:21:24 PDT 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -9,7 +9,7 @@
 psci: PSCIv0.2 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: Trusted OS migration not required
-percpu: Embedded 21 pages/cpu @ffff80003efcf000 s47104 r8192 d30720 u86016
+percpu: Embedded 20 pages/cpu @ffff80003efd1000 s42880 r8192 d30848 u81920
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
 CPU features: enabling workaround for ARM erratum 834220
@@ -18,15 +18,15 @@
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
-Memory: 1000836K/1048576K available (7684K kernel code, 696K rwdata, 3512K rodata, 960K init, 263K bss, 31356K reserved, 16384K cma-reserved)
+Memory: 1000652K/1048576K available (7748K kernel code, 762K rwdata, 3576K rodata, 960K init, 261K bss, 31540K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff0000087f0000   (  7616 KB)
-    .rodata : 0xffff0000087f0000 - 0xffff000008b70000   (  3584 KB)
-      .init : 0xffff000008b70000 - 0xffff000008c60000   (   960 KB)
-      .data : 0xffff000008c60000 - 0xffff000008d0e200   (   697 KB)
-       .bss : 0xffff000008d0e200 - 0xffff000008d4fe38   (   264 KB)
+      .text : 0xffff000008080000 - 0xffff000008800000   (  7680 KB)
+    .rodata : 0xffff000008800000 - 0xffff000008b90000   (  3648 KB)
+      .init : 0xffff000008b90000 - 0xffff000008c80000   (   960 KB)
+      .data : 0xffff000008c80000 - 0xffff000008d3e800   (   762 KB)
+       .bss : 0xffff000008d3e800 - 0xffff000008d7ff38   (   262 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -61,7 +61,7 @@
 pinctrl core: initialized pinctrl subsystem
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffff0000087f6000, 1 data @ ffff000008c64000)
+vdso: 2 pages (1 code @ ffff000008806000, 1 data @ ffff000008c84000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -97,7 +97,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.692:1): initialized
+audit: type=2000 audit(0.604:1): initialized
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -160,7 +160,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2016-06-24 07:21:56 UTC (1466752916)
+rtc-pl031 9010000.pl031: setting system clock to 2016-06-27 07:21:57 UTC (1467012117)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
@@ -169,7 +169,7 @@
 EXT4-fs (vda2): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 254:2.
 devtmpfs: mounted
-Freeing unused kernel memory: 960K (ffff800000b70000 - ffff800000c60000)
+Freeing unused kernel memory: 960K (ffff800000b90000 - ffff800000c80000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 bash-4.3# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-06-27 00:26:40.400633013 -0700
+++ /tmp/foob	2016-06-27 00:26:40.404633015 -0700
@@ -38,10 +38,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##########################################
-	 3.2 MiB/s
+	 #########################################
+	 3.3 MiB/s
 done
-Bytes transferred = 7285368 (6f2a78 hex)
+Bytes transferred = 7280728 (6f1858 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -52,7 +52,7 @@
 done
 Bytes transferred = 34956 (888c hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6f2a78 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6f1858 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff88b ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc4-next-20160624-102247-g2cf991d (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Fri Jun 24 00:19:03 PDT 2016
+Linux version 4.7.0-rc4-next-20160627-102768-gaa20c9a (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jun 27 00:18:47 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,20 +69,20 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141b440, node_mem_map eeffa000
+free_area_init_node: node 0, pgdat c141b500, node_mem_map eeffa000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 14 pages/cpu @eef9d000 s26048 r8192 d23104 u57344
-pcpu-alloc: s26048 r8192 d23104 u57344 alloc=14*4096
+percpu: Embedded 13 pages/cpu @eefa1000 s23808 r8192 d21248 u53248
+pcpu-alloc: s23808 r8192 d21248 u53248 alloc=13*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=7369c667-ff51-ec4a-29cd-baabf2fbe346 rootwait gpt debug earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1995108K/2096128K available (9531K kernel code, 1141K rwdata, 4520K rodata, 2048K init, 342K bss, 35484K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1995124K/2096128K available (9531K kernel code, 1141K rwdata, 4512K rodata, 2048K init, 343K bss, 35468K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc10b8cec   (15044 kB)
+      .text : 0xc0208000 - 0xc10b6d94   (15036 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141d4a0   (1142 kB)
-       .bss : 0xc141f000 - 0xc1474b48   ( 343 kB)
+      .data : 0xc1300000 - 0xc141d580   (1142 kB)
+       .bss : 0xc141f000 - 0xc1474d20   ( 344 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -135,8 +135,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?
-hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
+No ATAGs?hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 8 bytes.
 Serial: AMBA PL011 UART driver
 tegra-mc 70019000.memory-controller: no memory timings for RAM code 3 registered
@@ -287,19 +286,23 @@
 tegra-ehci 7d008000.usb: EHCI Host Controller
 tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 1
 tegra-ehci 7d008000.usb: irq 339, io mem 0x7d008000
-mmc0: new high speed SDHC card at address aaaa
+mmc1: new high speed MMC card at address 0001
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
-mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
+mmc0: new high speed SDHC card at address aaaa
+mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
 tegra-dsi 54300000.dsi: cannot get VDD supply
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
- mmcblk0: p1 p2 p3
 hctosys: unable to open rtc device (rtc0)
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
 tegra-dsi 54300000.dsi: cannot get VDD supply
-mmc1: new high speed MMC card at address 0001
+vdd_hdmi_5v0: disabling
+dvdd_ts: disabling
+usb1_vbus: disabling
+VDD_5V0_HDMI_CON: disabling
+ mmcblk0: p1 p2 p3
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
 tegra-dsi 54300000.dsi: cannot get VDD supply
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
@@ -307,13 +310,9 @@
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
  mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
+EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
 tegra-dsi 54300000.dsi: cannot get VDD supply
-vdd_hdmi_5v0: disabling
-dvdd_ts: disabling
-usb1_vbus: disabling
-VDD_5V0_HDMI_CON: disabling
-EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk1p1): recovery complete
 EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:17.
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-06-27 00:26:40.444633037 -0700
+++ /tmp/foob	2016-06-27 00:26:40.444633037 -0700
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 #######################################################
-	 3 MiB/s
+	 2.9 MiB/s
 done
-Bytes transferred = 5572104 (550608 hex)
+Bytes transferred = 5572016 (5505b0 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 2.6 MiB/s
 done
 Bytes transferred = 34956 (888c hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x550608 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5505b0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff88b ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc4-next-20160624-102247-g2cf991d (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Jun 24 00:18:54 PDT 2016
+Linux version 4.7.0-rc4-next-20160627-102768-gaa20c9a (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jun 27 00:18:51 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,20 +69,20 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0e5b900, node_mem_map eeff9000
+free_area_init_node: node 0, pgdat c0e5b980, node_mem_map eeff9000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 14 pages/cpu @eef9c000 s26048 r8192 d23104 u57344
-pcpu-alloc: s26048 r8192 d23104 u57344 alloc=14*4096
+percpu: Embedded 13 pages/cpu @eefa0000 s23936 r8192 d21120 u53248
+pcpu-alloc: s23936 r8192 d21120 u53248 alloc=13*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=7369c667-ff51-ec4a-29cd-baabf2fbe346 rootwait gpt debug earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998920K/2096128K available (8441K kernel code, 373K rwdata, 2848K rodata, 1024K init, 366K bss, 31672K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998936K/2096128K available (8441K kernel code, 373K rwdata, 2844K rodata, 1024K init, 366K bss, 31656K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0c06630   (12282 kB)
+      .text : 0xc0008000 - 0xc0c056b8   (12278 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0e5d4a0   ( 374 kB)
-       .bss : 0xc0e5f000 - 0xc0ebab04   ( 367 kB)
+      .data : 0xc0e00000 - 0xc0e5d540   ( 374 kB)
+       .bss : 0xc0e5f000 - 0xc0ebab14   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -132,8 +132,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?
-hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
+No ATAGs?hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 8 bytes.
 tegra-mc 70019000.memory-controller: no memory timings for RAM code 3 registered
 dvdd_ts: Failed to request enable GPIO61: -517
@@ -256,15 +255,15 @@
 mmc1: SDHCI controller on 78000600.sdhci [78000600.sdhci] using ADMA
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
-mmc0: new high speed SDHC card at address aaaa
-mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
+mmc0: new high speed SDHC card at address aaaa
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
- mmcblk0: p1 p2 p3
+mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
  mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
+ mmcblk0: p1 p2 p3
 tegra30-i2s 70080400.i2s: DMA channels sourced from device 70080000.ahub
 tegra-snd-rt5640 sound: rt5640-aif1 <-> 70080400.i2s mapping ok
 input: NVIDIA Tegra Dalmore Headphones as /devices/soc0/sound/sound/card0/input0
@@ -312,9 +311,10 @@
 EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk1p1): recovery complete
 EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:17.
+VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (c0d00000 - c0e00000)
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    5.983557] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-06-27 00:26:40.480633057 -0700
+++ /tmp/foob	2016-06-27 00:26:40.484633059 -0700
@@ -33,21 +33,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##########################################
-	 3.7 MiB/s
+	 #########################################
+	 3.9 MiB/s
 done
-Bytes transferred = 7285368 (6f2a78 hex)
+Bytes transferred = 7280728 (6f1858 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.5 MiB/s
+	 4 MiB/s
 done
 Bytes transferred = 66693 (10485 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6f2a78 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x6f1858 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013484
@@ -55,7 +55,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc4-next-20160624-102247-g2cf991d (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Fri Jun 24 00:19:03 PDT 2016
+Linux version 4.7.0-rc4-next-20160627-102768-gaa20c9a (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jun 27 00:18:47 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -64,20 +64,20 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141b440, node_mem_map eeffa000
+free_area_init_node: node 0, pgdat c141b500, node_mem_map eeffa000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 14 pages/cpu @eef87000 s26048 r8192 d23104 u57344
-pcpu-alloc: s26048 r8192 d23104 u57344 alloc=14*4096
+percpu: Embedded 13 pages/cpu @eef8b000 s23808 r8192 d21248 u53248
+pcpu-alloc: s23808 r8192 d21248 u53248 alloc=13*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=b1791faa-84e8-4e27-83a1-9fb0a77e8803 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1994988K/2096128K available (9531K kernel code, 1141K rwdata, 4520K rodata, 2048K init, 342K bss, 35604K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1995004K/2096128K available (9531K kernel code, 1141K rwdata, 4512K rodata, 2048K init, 343K bss, 35588K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -85,10 +85,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc10b8cec   (15044 kB)
+      .text : 0xc0208000 - 0xc10b6d94   (15036 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141d4a0   (1142 kB)
-       .bss : 0xc141f000 - 0xc1474b48   ( 343 kB)
+      .data : 0xc1300000 - 0xc141d580   (1142 kB)
+       .bss : 0xc141f000 - 0xc1474d20   ( 344 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -130,8 +130,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?
-hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
+No ATAGs?hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 8 bytes.
 Serial: AMBA PL011 UART driver
 +5V_SYS: supplied by +VDD_MUX
@@ -306,6 +305,12 @@
 +5V_SATA: supplied by +5V_SYS
 +12V_SATA: supplied by +VDD_MUX
 usb2-0 supply vbus not found, using dummy regulator
+mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+ mmcblk0: p1
 usb2-1 supply vbus not found, using dummy regulator
 usb3-0 supply vbus not found, using dummy regulator
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
@@ -315,7 +320,6 @@
 as3722-sd3: supplied by +5V_SYS
 as3722-sd4: supplied by +5V_SYS
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
-mmc0: new high speed MMC card at address 0001
 as3722-sd5: supplied by +5V_SYS
 as3722-ldo0: supplied by +1.35V_LP0(sd2)
 as3722-ldo1: supplied by +3.3V_RUN
@@ -372,6 +376,7 @@
 PCI: bus1: Fast back to back transfers disabled
 pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
 pci 0000:02:00.0: [10ec:8168] type 00 class 0x020000
+ata1: SATA link down (SStatus 0 SControl 300)
 pci 0000:02:00.0: reg 0x10: [io  0x0000-0x00ff]
 pci 0000:02:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit]
 pci 0000:02:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
@@ -405,29 +410,22 @@
 pcie_pme 0000:00:02.0:pcie001: service driver pcie_pme loaded
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
-r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0a48000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 393
+r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0a52000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 393
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
-drm drm: No connectors reported connected with modes
-[drm] Cannot find any crtc or sizes - going 1024x768
-ata1: SATA link down (SStatus 0 SControl 300)
 mmc1: new high speed SDHC card at address aaaa
 mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
  mmcblk1: p1 p2 p3
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-Console: switching to colour frame buffer device 128x48
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
- mmcblk0: p1
+tegra-hdmi 54280000.hdmi: cannot set audio to 44100 at 297000000 pclk
+Console: switching to colour frame buffer device 512x135
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 of: dev_pm_opp_of_cpumask_add_table: couldn't find opp table for cpu:0, -19
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:09 UTC (946684929)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:13 UTC (946684813)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
@@ -439,7 +437,7 @@
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:17.
+VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-06-27 00:26:40.528633083 -0700
+++ /tmp/foob	2016-06-27 00:26:40.528633083 -0700
@@ -28,24 +28,24 @@
 Load address: 0x82000000
 Loading: *#################################################################
 	 #################################################################
-	 #################################################################
+	 #############T T T ####################################################
 	 #################################################################
 	 #################################################################
 	 #######################################################
-	 3.9 MiB/s
+	 261.7 KiB/s
 done
-Bytes transferred = 5572104 (550608 hex)
+Bytes transferred = 5572016 (5505b0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.7 MiB/s
+	 929.7 KiB/s
 done
 Bytes transferred = 66693 (10485 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x550608 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5505b0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013484
@@ -54,7 +54,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc4-next-20160624-102247-g2cf991d (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Jun 24 00:18:54 PDT 2016
+Linux version 4.7.0-rc4-next-20160627-102768-gaa20c9a (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jun 27 00:18:51 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -64,20 +64,20 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0e5b900, node_mem_map eeff9000
+free_area_init_node: node 0, pgdat c0e5b980, node_mem_map eeff9000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 14 pages/cpu @eef86000 s26048 r8192 d23104 u57344
-pcpu-alloc: s26048 r8192 d23104 u57344 alloc=14*4096
+percpu: Embedded 13 pages/cpu @eef8a000 s23936 r8192 d21120 u53248
+pcpu-alloc: s23936 r8192 d21120 u53248 alloc=13*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=b1791faa-84e8-4e27-83a1-9fb0a77e8803 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998800K/2096128K available (8441K kernel code, 373K rwdata, 2848K rodata, 1024K init, 366K bss, 31792K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998816K/2096128K available (8441K kernel code, 373K rwdata, 2844K rodata, 1024K init, 366K bss, 31776K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -85,10 +85,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0c06630   (12282 kB)
+      .text : 0xc0008000 - 0xc0c056b8   (12278 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0e5d4a0   ( 374 kB)
-       .bss : 0xc0e5f000 - 0xc0ebab04   ( 367 kB)
+      .data : 0xc0e00000 - 0xc0e5d540   ( 374 kB)
+       .bss : 0xc0e5f000 - 0xc0ebab14   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -127,8 +127,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?
-hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
+No ATAGs?hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 8 bytes.
 +5V_SYS: supplied by +VDD_MUX
 +3.3V_SYS: supplied by +VDD_MUX
@@ -286,8 +285,8 @@
 rt5640 0-001c: ASoC: no source widget found for DAC R2 Power
 rt5640 0-001c: ASoC: Failed to add route DAC R2 Power -> direct -> IF2 DAC R
 tegra-snd-rt5640 sound: rt5640-aif1 <-> 70301100.i2s mapping ok
-input: NVIDIA Tegra Jetson TK1 Headphones as /devices/soc0/sound/sound/card1/input0
-input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input1
+input: NVIDIA Tegra Jetson TK1 Headphones as /devices/soc0/sound/sound/card1/input1
+input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input0
 NET: Registered protocol family 10
 mip6: Mobile IPv6
 sit: IPv6 over IPv4 tunneling driver
@@ -369,10 +368,10 @@
 mmc1: Unknown controller version (3). You may experience problems.
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
-ata1: SATA link down (SStatus 0 SControl 300)
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
 tegra-pcie 1003000.pcie-controller: probing port 0, using 2 lanes
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
+ata1: SATA link down (SStatus 0 SControl 300)
 tegra-pcie 1003000.pcie-controller: probing port 1, using 1 lanes
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000000
 tegra-pcie 1003000.pcie-controller: PCI host bridge to bus 0000:00
@@ -398,11 +397,10 @@
 pci 0000:02:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
 pci 0000:02:00.0: supports D1 D2
 pci 0000:02:00.0: PME# supported from D0 D1 D2 D3hot D3cold
-mmc1: new high speed SDHC card at address aaaa
-mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
- mmcblk1: p1 p2 p3
 PCI: bus2: Fast back to back transfers disabled
 pci_bus 0000:02: busn_res: [bus 02-ff] end is updated to 02
+mmc1: new high speed SDHC card at address aaaa
+mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 pci 0000:00:01.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
 pci 0000:00:02.0: BAR 8: assigned [mem 0x13100000-0x131fffff]
 pci 0000:00:02.0: BAR 9: assigned [mem 0x20000000-0x200fffff 64bit pref]
@@ -415,6 +413,7 @@
 pci 0000:02:00.0: BAR 0: assigned [io  0x1000-0x10ff]
 pci 0000:00:02.0: PCI bridge to [bus 02]
 pci 0000:00:02.0:   bridge window [io  0x1000-0x1fff]
+ mmcblk1: p1 p2 p3
 pci 0000:00:02.0:   bridge window [mem 0x13100000-0x131fffff]
 pci 0000:00:02.0:   bridge window [mem 0x20000000-0x200fffff 64bit pref]
 pci 0000:00:01.0: nv_msi_ht_cap_quirk didn't locate host bridge
@@ -441,7 +440,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:17 UTC (946684817)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:16 UTC (946684936)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-06-27 00:26:40.572633106 -0700
+++ /tmp/foob	2016-06-27 00:26:40.572633106 -0700
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -40,21 +40,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##########################################
-	 3.5 MiB/s
+	 #########################################
+	 3.2 MiB/s
 done
-Bytes transferred = 7285368 (6f2a78 hex)
+Bytes transferred = 7280728 (6f1858 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.4 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 66901 (10555 hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6f2a78 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x6f1858 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013554
@@ -62,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc4-next-20160624-102247-g2cf991d (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Fri Jun 24 00:19:03 PDT 2016
+Linux version 4.7.0-rc4-next-20160627-102768-gaa20c9a (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jun 27 00:18:47 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -72,20 +72,20 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c141b440, node_mem_map eeffa000
+free_area_init_node: node 0, pgdat c141b500, node_mem_map eeffa000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327679 pages, LIFO batch:31
-percpu: Embedded 14 pages/cpu @eef87000 s26048 r8192 d23104 u57344
-pcpu-alloc: s26048 r8192 d23104 u57344 alloc=14*4096
+percpu: Embedded 13 pages/cpu @eef8b000 s23808 r8192 d21248 u53248
+pcpu-alloc: s23808 r8192 d21248 u53248 alloc=13*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522751
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=4d36e82a-b3c0-4b4b-bfcd-afe5217ac023 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1996008K/2097148K available (9531K kernel code, 1141K rwdata, 4520K rodata, 2048K init, 342K bss, 35604K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1996024K/2097148K available (9531K kernel code, 1141K rwdata, 4512K rodata, 2048K init, 343K bss, 35588K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -93,10 +93,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc10b8cec   (15044 kB)
+      .text : 0xc0208000 - 0xc10b6d94   (15036 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141d4a0   (1142 kB)
-       .bss : 0xc141f000 - 0xc1474b48   ( 343 kB)
+      .data : 0xc1300000 - 0xc141d580   (1142 kB)
+       .bss : 0xc141f000 - 0xc1474d20   ( 344 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -138,8 +138,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?
-hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
+No ATAGs?hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 8 bytes.
 Serial: AMBA PL011 UART driver
 tegra-mc 70019000.memory-controller: no memory timings for RAM code 0 registered
@@ -289,6 +288,7 @@
 mmc0: Unknown controller version (3). You may experience problems.
 mmc0: Invalid maximum block size, assuming 512 bytes
 mmc0: SDHCI controller on 700b0600.sdhci [700b0600.sdhci] using ADMA 64-bit
+input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 ledtrig-cpu: registered to indicate activity on CPUs
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
@@ -318,6 +318,7 @@
 +5V_HDMI_CON: supplied by +5V_SYS
 usb3-0 supply vbus not found, using dummy regulator
 usb3-1 supply vbus not found, using dummy regulator
+elan_i2c 1-0015: failed to read report data: -121
 tegra-sor 54540000.sor: failed to probe output: -517
 panel supply power not found, using dummy regulator
 sdhci-tegra 700b0000.sdhci: allocated mmc-pwrseq
@@ -330,14 +331,15 @@
 mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
 mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 mmc1: SDHCI controller on 700b0000.sdhci [700b0000.sdhci] using ADMA 64-bit
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc2: Unknown controller version (3). You may experience problems.
 mmc2: Invalid maximum block size, assuming 512 bytes
-mmc1: new high speed SDIO card at address 0001
+elan_i2c 1-0015: invalid report id data (ff)
 mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
+mmc1: new high speed SDIO card at address 0001
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
 mmc2: new high speed SDHC card at address 0007
@@ -351,7 +353,7 @@
 Console: switching to colour frame buffer device 170x48
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
-input: gpio-keys as /devices/soc0/gpio-keys/input/input0
+input: gpio-keys as /devices/soc0/gpio-keys/input/input1
 as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:22 UTC (946684822)
 +1.8V_RUN_CAM: disabling
 +3.3V_RUN_CAM: disabling
@@ -363,7 +365,6 @@
 +5V_VDD_TS_SW: disabling
 +5V_USB_HS: disabling
 +5V_USB_SS: disabling
-input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input1
 EXT4-fs (mmcblk0p3): couldn't mount as ext3 due to feature incompatibilities
 EXT4-fs (mmcblk0p3): couldn't mount as ext2 due to feature incompatibilities
 EXT4-fs (mmcblk0p3): recovery complete
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2016-06-27 00:26:40.596633119 -0700
+++ /tmp/foob	2016-06-27 00:26:40.600633122 -0700
@@ -41,7 +41,7 @@
 	 #######################################################
 	 3.6 MiB/s
 done
-Bytes transferred = 5572104 (550608 hex)
+Bytes transferred = 5572016 (5505b0 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -52,7 +52,7 @@
 done
 Bytes transferred = 66901 (10555 hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x550608 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5505b0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013554
@@ -141,17 +141,18 @@
 tegra30-i2s 70301100.i2s: DMA channels sourced from device 70300000.ahub
 max98090 0-0010: MAX98090 REVID=0x43
 max98090 0-0010: use default 2.8v micbias
+tegra-snd-max98090 sound: HiFi <-> 70301100.i2s mapping ok
+input: GoogleNyanBig Headphones as /devices/soc0/sound/sound/card1/input1
+input: GoogleNyanBig Mic Jack as /devices/soc0/sound/sound/card1/input2
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
 mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
 mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
-tegra-snd-max98090 sound: HiFi <-> 70301100.i2s mapping ok
-input: GoogleNyanBig Headphones as /devices/soc0/sound/sound/card1/input1
-input: GoogleNyanBig Mic Jack as /devices/soc0/sound/sound/card1/input2
 NET: Registered protocol family 10
 mip6: Mobile IPv6
+input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input3
 sit: IPv6 over IPv4 tunneling driver
 NET: Registered protocol family 17
 NET: Registered protocol family 15
@@ -173,7 +174,6 @@
 +1.8V_RUN_CAM: supplied by +3.3V_RUN
 +VDDIO_SDMMC3: supplied by +3.3V_RUN
 +1.8V_RUN_VPP_FUSE: supplied by +3.3V_RUN
-input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input3
 +VDD_LED: supplied by +VDD_MUX
 +5V_VDD_TS_SW: supplied by +5V_SYS
 +5V_USB_HS: supplied by +5V_SYS
@@ -214,8 +214,8 @@
 [drm] No driver support for vblank timestamp query.
 usb 1-2: new high-speed USB device number 2 using tegra-xusb
 mmc2: new high speed SDHC card at address 0007
-Console: switching to colour frame buffer device 170x48
 mmcblk2: mmc2:0007 SD08G 7.42 GiB 
+Console: switching to colour frame buffer device 170x48
 drm drm: fb0:  frame buffer device
 GPT:Primary header alternate_lba != Alt. header my_lba
 GPT:3014591 != 15564799
@@ -225,7 +225,7 @@
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input4
  mmcblk2: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:24 UTC (946684824)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:23 UTC (946684823)
 +1.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
 +1.05V_RUN_CAM_REAR: disabling
@@ -244,10 +244,10 @@
 VFS: Mounted root (ext4 filesystem) on device 179:3.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (c0d00000 - c0e00000)
+usb 1-3: new high-speed USB device number 3 using tegra-xusb
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    5.774772] usb 1-3: new high-speed USB device number 3 using tegra-xusb
-asix 1-3:1.0 eth0: register 'asix' at usb-70090000.usb-3, ASIX AX88772 USB 2.0 Ethernet, c8:d7:19:90:fa:65
+root@(none):/# [    5.827313] asix 1-3:1.0 eth0: register 'asix' at usb-70090000.usb-3, ASIX AX88772 USB 2.0 Ethernet, c8:d7:19:90:fa:65
 usb 1-1: new high-speed USB device number 4 using tegra-xusb
 usb-storage 1-1:1.0: USB Mass Storage device detected
 scsi host0: usb-storage 1-1:1.0
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2016-06-27 00:26:40.632633139 -0700
+++ /tmp/foob	2016-06-27 00:26:40.636633141 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1800442 us
+Bootloader-Cpu Init at (time stamp): 1800248 us
 
 Pinmux changes applied in kernel way
 
@@ -59,47 +59,47 @@
 The proc BoardInfo: 0x0176:0x0001:0x01:0x43:0x00
 FastbootAddAtag: setting ATAG (0)
 
-Board Id Read Time: 4 us
+Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1800415  us
+Miscellaneous avp time: 1800222  us
 Bootloader (AosInit) time: 733453 us
-Bootloader Main Init time: 180149 us
-BlockDevMgrInit time: 157509 us
-PartMgrInit time: 9 us
+Bootloader Main Init time: 180318 us
+BlockDevMgrInit time: 157549 us
+PartMgrInit time: 8 us
 StorMgrInit time: 30 us
 FsMgrInit time: 2 us
-Partition Table load: 77037 us 
-Display init time: 574432 us
-Kernel read time: 360 msec
-Load Kernel Size:   15593472 Bytes
-Bl Kernel ReadSpeed: 43315000 Bytes/sec
-Pre-Kernel Jump time: 1681651 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5760548 us
+Partition Table load: 77126 us 
+Display init time: 574431 us
+Kernel read time: 364 msec
+Load Kernel Size:   15790080 Bytes
+Bl Kernel ReadSpeed: 43379000 Bytes/sec
+Pre-Kernel Jump time: 1696187 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5783320 us
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc4-next-20160624+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Fri Jun 24 00:21:21 PDT 2016
+Linux version 4.7.0-rc4-next-20160627+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Jun 27 00:21:24 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
 Memory limited to 1734MB
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
 cma: Reserved 16 MiB at 0x00000000eb400000
 /cpus/cpu@1: missing enable-method property
-percpu: Embedded 21 pages/cpu @ffff80006c59e000 s47104 r8192 d30720 u86016
+percpu: Embedded 20 pages/cpu @ffff80006c5a0000 s42880 r8192 d30848 u81920
 Detected PIPT I-cache on CPU0
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 436968
 Kernel command line: tegraid=13.1.3.0.0 mem=1734M@2048M memtype=255 vpr=151M@3817M ddr_die=2048M@2048M section=256M commchip_id=0 usb_port_owner_info=0 lane_owner_info=0 emc_max_dvfs=0 touch_id=0@0 video=tegrafb no_console_suspend=1 console=ttyS0,115200n8 debug_uartport=lsport,0 earlyprintk=uart8250-32bit,0x70006000 usbcore.old_scheme_first=1 lp0_vec=0@0x0 tegra_fbmem=32899072@0xad012000 core_edp_mv=1150 core_edp_ma=4000 pmuboard=0x0176:0x0001:0x01:0x43:0x00 displayboard=0x0162:0x0001:0x00:0x42:0x00 power_supply=Adapter audio_codec=rt5640 board_info=0x0176:0x0001:0x01:0x43:0x00 tegraboot=sdmmc gpt gpt_sector=94207 modem_id=0 android.kerneltype=normal
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 Dentry cache hash table entries: 262144 (order: 9, 2097152 bytes)
 Inode-cache hash table entries: 131072 (order: 8, 1048576 bytes)
-Memory: 1707428K/1775616K available (7684K kernel code, 696K rwdata, 3512K rodata, 960K init, 263K bss, 51804K reserved, 16384K cma-reserved)
+Memory: 1707244K/1775616K available (7748K kernel code, 762K rwdata, 3576K rodata, 960K init, 261K bss, 51988K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff0000087f0000   (  7616 KB)
-    .rodata : 0xffff0000087f0000 - 0xffff000008b70000   (  3584 KB)
-      .init : 0xffff000008b70000 - 0xffff000008c60000   (   960 KB)
-      .data : 0xffff000008c60000 - 0xffff000008d0e200   (   697 KB)
-       .bss : 0xffff000008d0e200 - 0xffff000008d4fe38   (   264 KB)
+      .text : 0xffff000008080000 - 0xffff000008800000   (  7680 KB)
+    .rodata : 0xffff000008800000 - 0xffff000008b90000   (  3648 KB)
+      .init : 0xffff000008b90000 - 0xffff000008c80000   (   960 KB)
+      .data : 0xffff000008c80000 - 0xffff000008d3e800   (   762 KB)
+       .bss : 0xffff000008d3e800 - 0xffff000008d7ff38   (   262 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -134,7 +134,7 @@
 pinctrl core: initialized pinctrl subsystem
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffff0000087f6000, 1 data @ ffff000008c64000)
+vdso: 2 pages (1 code @ ffff000008806000, 1 data @ ffff000008c84000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -184,7 +184,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(1.111:1): initialized
+audit: type=2000 audit(1.123:1): initialized
 workingset: timestamp_bits=44 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -246,9 +246,13 @@
 NET: Registered protocol family 17
 9pnet: Installing 9P2000 support
 Key type dns_resolver registered
+mmc0: new high speed MMC card at address 0001
 registered taskstats version 1
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 +VDD_LED: supplied by +VDD_MUX
-mmc0: new high speed MMC card at address 0001
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
 +5V_USB_HS: supplied by +5V_SYS
 +5V_USB_SS: supplied by +5V_SYS
 +5V_HDMI_CON: supplied by +5V_SYS
@@ -268,11 +272,7 @@
 +5V_HDMI_CON: disabling
 ALSA device list:
   No soundcards found.
-Freeing unused kernel memory: 960K (ffff800000b70000 - ffff800000c60000)
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+Freeing unused kernel memory: 960K (ffff800000b90000 - ffff800000c80000)
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-06-27 00:26:40.680633166 -0700
+++ /tmp/foob	2016-06-27 00:26:40.684633168 -0700
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -46,10 +46,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##########################################
-	 908.2 KiB/s
+	 #########################################
+	 909.2 KiB/s
 done
-Bytes transferred = 7285368 (6f2a78 hex)
+Bytes transferred = 7280728 (6f1858 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -60,7 +60,7 @@
 done
 Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x6f2a78 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x6f1858 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102b3
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc4-next-20160624-102247-g2cf991d (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Fri Jun 24 00:19:03 PDT 2016
+Linux version 4.7.0-rc4-next-20160627-102768-gaa20c9a (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jun 27 00:18:47 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -76,20 +76,20 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c141b440, node_mem_map ef7fa000
+free_area_init_node: node 0, pgdat c141b500, node_mem_map ef7fa000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 65536 pages, LIFO batch:15
-percpu: Embedded 14 pages/cpu @ef7c9000 s26048 r8192 d23104 u57344
-pcpu-alloc: s26048 r8192 d23104 u57344 alloc=14*4096
+percpu: Embedded 13 pages/cpu @ef7cb000 s23808 r8192 d21248 u53248
+pcpu-alloc: s23808 r8192 d21248 u53248 alloc=13*4096
 pcpu-alloc: [0] 0 [0] 1 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 260608
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=11eb23c1-01 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 955932K/1048576K available (9531K kernel code, 1141K rwdata, 4520K rodata, 2048K init, 342K bss, 27108K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 955940K/1048576K available (9531K kernel code, 1141K rwdata, 4512K rodata, 2048K init, 343K bss, 27100K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -97,10 +97,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc10b8cec   (15044 kB)
+      .text : 0xc0208000 - 0xc10b6d94   (15036 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141d4a0   (1142 kB)
-       .bss : 0xc141f000 - 0xc1474b48   ( 343 kB)
+      .data : 0xc1300000 - 0xc141d580   (1142 kB)
+       .bss : 0xc141f000 - 0xc1474d20   ( 344 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -139,8 +139,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?
-hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
+No ATAGs?hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 4 bytes.
 Serial: AMBA PL011 UART driver
 usb1_vbus: Failed to request enable GPIO170: -517
@@ -337,7 +336,6 @@
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
-asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-06-27 00:26:40.716633186 -0700
+++ /tmp/foob	2016-06-27 00:26:40.716633186 -0700
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -45,20 +45,20 @@
 	 #################################################################
 	 #################################################################
 	 #######################################################
-	 4.1 MiB/s
+	 919.9 KiB/s
 done
-Bytes transferred = 5572104 (550608 hex)
+Bytes transferred = 5572016 (5505b0 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 861.3 KiB/s
 done
 Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x550608 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x5505b0 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102b3
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc4-next-20160624-102247-g2cf991d (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Jun 24 00:18:54 PDT 2016
+Linux version 4.7.0-rc4-next-20160627-102768-gaa20c9a (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jun 27 00:18:51 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -76,20 +76,20 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0e5b900, node_mem_map ef7f9000
+free_area_init_node: node 0, pgdat c0e5b980, node_mem_map ef7f9000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 65536 pages, LIFO batch:15
-percpu: Embedded 14 pages/cpu @ef7c8000 s26048 r8192 d23104 u57344
-pcpu-alloc: s26048 r8192 d23104 u57344 alloc=14*4096
+percpu: Embedded 13 pages/cpu @ef7ca000 s23936 r8192 d21120 u53248
+pcpu-alloc: s23936 r8192 d21120 u53248 alloc=13*4096
 pcpu-alloc: [0] 0 [0] 1 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 260608
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=11eb23c1-01 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 959744K/1048576K available (8441K kernel code, 373K rwdata, 2848K rodata, 1024K init, 366K bss, 23296K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 959752K/1048576K available (8441K kernel code, 373K rwdata, 2844K rodata, 1024K init, 366K bss, 23288K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -97,10 +97,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0c06630   (12282 kB)
+      .text : 0xc0008000 - 0xc0c056b8   (12278 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0e5d4a0   ( 374 kB)
-       .bss : 0xc0e5f000 - 0xc0ebab04   ( 367 kB)
+      .data : 0xc0e00000 - 0xc0e5d540   ( 374 kB)
+       .bss : 0xc0e5f000 - 0xc0ebab14   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
@@ -138,8 +138,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?
-hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
+No ATAGs?hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 4 bytes.
 usb1_vbus: Failed to request enable GPIO170: -517
 reg-fixed-voltage regulators:regulator@2: Failed to register regulator: -517
@@ -307,11 +306,11 @@
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
- mmcblk0: p1
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
+ mmcblk0: p1
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
 NET: Registered protocol family 10
@@ -320,11 +319,11 @@
 NET: Registered protocol family 17
 NET: Registered protocol family 15
 can: controller area network core (rev 20120528 abi 9)
-usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 NET: Registered protocol family 29
 can: raw protocol (rev 20120528)
 can: broadcast manager protocol (rev 20160617 t)
 can: netlink gateway (rev 20130117) max_hops=1
+usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 Bluetooth: RFCOMM socket layer initialized
 Bluetooth: RFCOMM ver 1.11
 Bluetooth: BNEP (Ethernet Emulation) ver 1.3
@@ -333,7 +332,6 @@
 Bluetooth: HIDP socket layer initialized
 Key type dns_resolver registered
 Registering SWP/SWPB emulation handler
-utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
@@ -348,7 +346,7 @@
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:1.
+VFS: Mounted root (ext3 filesystem) on device 179:17.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (c0d00000 - c0e00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2016-06-27 00:26:40.760633210 -0700
+++ /tmp/foob	2016-06-27 00:26:40.764633212 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 749178 us
-Basic modules init took 348895 us
-Sec Bootdevice Read Time = 203 ms, Read Size = 8459 KB
-Next stage binary read took 15082 us
-Carveout took 252712 us
-CPU initialization took 92306 us
-Total time taken by TegraBoot 708995 us
+Tegraboot started after 726549 us
+Basic modules init took 348530 us
+Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
+Next stage binary read took 14366 us
+Carveout took 252731 us
+CPU initialization took 92316 us
+Total time taken by TegraBoot 707943 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.589] LPDDR4 Training: Number of tables = 10
+64b[0001.566] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-13165056 bytes read in 761 ms (16.5 MiB/s)
+13363200 bytes read in 758 ms (16.8 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
 39484 bytes read in 454 ms (84 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 496 ms (3.1 MiB/s)
+1627787 bytes read in 495 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc4-next-20160624+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Fri Jun 24 00:21:21 PDT 2016
+Linux version 4.7.0-rc4-next-20160627+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Jun 27 00:21:24 PDT 2016
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -149,8 +149,8 @@
 psci: PSCIv0.2 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: MIGRATE_INFO_TYPE not supported.
-percpu: Embedded 21 pages/cpu @ffff8000bff4d000 s47104 r8192 d30720 u86016
-pcpu-alloc: s47104 r8192 d30720 u86016 alloc=21*4096
+percpu: Embedded 20 pages/cpu @ffff8000bff52000 s42880 r8192 d30848 u81920
+pcpu-alloc: s42880 r8192 d30848 u81920 alloc=20*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
@@ -166,15 +166,15 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffff80007a000000-ffff80007dffffff]
-Memory: 2979800K/3131392K available (7684K kernel code, 696K rwdata, 3512K rodata, 960K init, 263K bss, 135208K reserved, 16384K cma-reserved)
+Memory: 2979628K/3131392K available (7748K kernel code, 762K rwdata, 3576K rodata, 960K init, 261K bss, 135380K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff0000087f0000   (  7616 KB)
-    .rodata : 0xffff0000087f0000 - 0xffff000008b70000   (  3584 KB)
-      .init : 0xffff000008b70000 - 0xffff000008c60000   (   960 KB)
-      .data : 0xffff000008c60000 - 0xffff000008d0e200   (   697 KB)
-       .bss : 0xffff000008d0e200 - 0xffff000008d4fe38   (   264 KB)
+      .text : 0xffff000008080000 - 0xffff000008800000   (  7680 KB)
+    .rodata : 0xffff000008800000 - 0xffff000008b90000   (  3648 KB)
+      .init : 0xffff000008b90000 - 0xffff000008c80000   (   960 KB)
+      .data : 0xffff000008c80000 - 0xffff000008d3e800   (   762 KB)
+       .bss : 0xffff000008d3e800 - 0xffff000008d7ff38   (   262 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -224,7 +224,7 @@
 pinctrl core: initialized pinctrl subsystem
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffff0000087f6000, 1 data @ ffff000008c64000)
+vdso: 2 pages (1 code @ ffff000008806000, 1 data @ ffff000008c84000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -264,7 +264,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.463:1): initialized
+audit: type=2000 audit(0.459:1): initialized
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -339,7 +339,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 960K (ffff800000b70000 - ffff800000c60000)
+Freeing unused kernel memory: 960K (ffff800000b90000 - ffff800000c80000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2016-06-27 00:26:40.780633220 -0700
+++ /tmp/foob	2016-06-27 00:26:40.780633220 -0700
@@ -1,4 +1,5 @@
-06000 (options '115200n8')
+Processor [411fd071]
+earlycon: uart0 at MMIO 0x0000000070006000 (options '115200n8')
 bootconsole [uart0] enabled
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
@@ -7,7 +8,7 @@
 psci: PSCIv1.0 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: MIGRATE_INFO_UP_CPU reported invalid physical ID (0x80000000)
-percpu: Embedded 21 pages/cpu @ffff8000bff4b000 s47104 r8192 d30720 u86016
+percpu: Embedded 20 pages/cpu @ffff8000bff50000 s42880 r8192 d30848 u81920
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
 CPU features: enabling workaround for ARM erratum 834220
@@ -22,15 +23,15 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2834032K/2988032K available (7684K kernel code, 696K rwdata, 3512K rodata, 960K init, 263K bss, 137616K reserved, 16384K cma-reserved)
+Memory: 2833860K/2988032K available (7748K kernel code, 762K rwdata, 3576K rodata, 960K init, 261K bss, 137788K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff0000087f0000   (  7616 KB)
-    .rodata : 0xffff0000087f0000 - 0xffff000008b70000   (  3584 KB)
-      .init : 0xffff000008b70000 - 0xffff000008c60000   (   960 KB)
-      .data : 0xffff000008c60000 - 0xffff000008d0e200   (   697 KB)
-       .bss : 0xffff000008d0e200 - 0xffff000008d4fe38   (   264 KB)
+      .text : 0xffff000008080000 - 0xffff000008800000   (  7680 KB)
+    .rodata : 0xffff000008800000 - 0xffff000008b90000   (  3648 KB)
+      .init : 0xffff000008b90000 - 0xffff000008c80000   (   960 KB)
+      .data : 0xffff000008c80000 - 0xffff000008d3e800   (   762 KB)
+       .bss : 0xffff000008d3e800 - 0xffff000008d7ff38   (   262 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -59,7 +60,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x8008: expected 0x2004f400
+boot misc3 0x81d0: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -72,7 +73,7 @@
 console [tty0] enabled
 bootconsole [uart0] disabled
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc4-next-20160624+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Fri Jun 24 00:21:21 PDT 2016
+Linux version 4.7.0-rc4-next-20160627+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Jun 27 00:21:24 PDT 2016
 Boot CPU: AArch64 Processor [411fd071]
 earlycon: uart0 at MMIO 0x0000000070006000 (options '115200n8')
 bootconsole [uart0] enabled
@@ -83,7 +84,7 @@
 psci: PSCIv1.0 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: MIGRATE_INFO_UP_CPU reported invalid physical ID (0x80000000)
-percpu: Embedded 21 pages/cpu @ffff8000bff4b000 s47104 r8192 d30720 u86016
+percpu: Embedded 20 pages/cpu @ffff8000bff50000 s42880 r8192 d30848 u81920
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
 CPU features: enabling workaround for ARM erratum 834220
@@ -98,15 +99,15 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2834032K/2988032K available (7684K kernel code, 696K rwdata, 3512K rodata, 960K init, 263K bss, 137616K reserved, 16384K cma-reserved)
+Memory: 2833860K/2988032K available (7748K kernel code, 762K rwdata, 3576K rodata, 960K init, 261K bss, 137788K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff0000087f0000   (  7616 KB)
-    .rodata : 0xffff0000087f0000 - 0xffff000008b70000   (  3584 KB)
-      .init : 0xffff000008b70000 - 0xffff000008c60000   (   960 KB)
-      .data : 0xffff000008c60000 - 0xffff000008d0e200   (   697 KB)
-       .bss : 0xffff000008d0e200 - 0xffff000008d4fe38   (   264 KB)
+      .text : 0xffff000008080000 - 0xffff000008800000   (  7680 KB)
+    .rodata : 0xffff000008800000 - 0xffff000008b90000   (  3648 KB)
+      .init : 0xffff000008b90000 - 0xffff000008c80000   (   960 KB)
+      .data : 0xffff000008c80000 - 0xffff000008d3e800   (   762 KB)
+       .bss : 0xffff000008d3e800 - 0xffff000008d7ff38   (   262 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -135,7 +136,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x8008: expected 0x2004f400
+boot misc3 0x81d0: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -173,7 +174,7 @@
 pinctrl core: initialized pinctrl subsystem
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffff0000087f6000, 1 data @ ffff000008c64000)
+vdso: 2 pages (1 code @ ffff000008806000, 1 data @ ffff000008c84000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -205,7 +206,7 @@
 RPC: Registered tcp transport module.
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 Unpacking initramfs...
-Freeing initrd memory: 2364K (ffff800009f35000 - ffff80000a184000)
+Freeing initrd memory: 2364K (ffff800009f65000 - ffff80000a1b4000)
 kvm [1]: 8-bit VMID
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
@@ -213,7 +214,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.231:1): initialized
+audit: type=2000 audit(0.227:1): initialized
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -271,10 +272,10 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-tegra_rtc 7000e000.rtc: setting system clock to 1970-01-01 00:00:09 UTC (9)
+tegra_rtc 7000e000.rtc: setting system clock to 1970-01-01 00:00:08 UTC (8)
 ALSA device list:
   No soundcards found.
-Freeing unused kernel memory: 960K (ffff800000b70000 - ffff800000c60000)
+Freeing unused kernel memory: 960K (ffff800000b90000 - ffff800000c80000)
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 BGND3R 29.1 GiB 
 mmcblk0boot0: mmc0:0001 BGND3R partition 1 4.00 MiB
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-06-27 00:26:40.816633240 -0700
+++ /tmp/foob	2016-06-27 00:26:40.820633242 -0700
@@ -38,21 +38,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##########################################
-	 3.2 MiB/s
+	 #########################################
+	 3.3 MiB/s
 done
-Bytes transferred = 7285368 (6f2a78 hex)
+Bytes transferred = 7280728 (6f1858 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 51511 (c937 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6f2a78 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6f1858 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff936 ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc4-next-20160624-102247-g2cf991d (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Fri Jun 24 00:19:03 PDT 2016
+Linux version 4.7.0-rc4-next-20160627-102768-gaa20c9a (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jun 27 00:18:47 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,20 +68,20 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141b440, node_mem_map eeffa000
+free_area_init_node: node 0, pgdat c141b500, node_mem_map eeffa000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 14 pages/cpu @eef8d000 s26048 r8192 d23104 u57344
-pcpu-alloc: s26048 r8192 d23104 u57344 alloc=14*4096
+percpu: Embedded 13 pages/cpu @eef91000 s23808 r8192 d21248 u53248
+pcpu-alloc: s23808 r8192 d21248 u53248 alloc=13*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=edbe1840-3633-c751-7993-b0ac834f81e0 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1995028K/2096128K available (9531K kernel code, 1141K rwdata, 4520K rodata, 2048K init, 342K bss, 35564K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1995044K/2096128K available (9531K kernel code, 1141K rwdata, 4512K rodata, 2048K init, 343K bss, 35548K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -89,10 +89,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc10b8cec   (15044 kB)
+      .text : 0xc0208000 - 0xc10b6d94   (15036 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141d4a0   (1142 kB)
-       .bss : 0xc141f000 - 0xc1474b48   ( 343 kB)
+      .data : 0xc1300000 - 0xc141d580   (1142 kB)
+       .bss : 0xc141f000 - 0xc1474d20   ( 344 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -133,8 +133,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?
-hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
+No ATAGs?hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 4 bytes.
 Serial: AMBA PL011 UART driver
 tegra-mc 7000f000.memory-controller: no memory timings for RAM code 3 registered
@@ -301,10 +300,10 @@
 tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
 mmc0: error -110 whilst initialising SD card
 mmc1: new high speed MMC card at address 0001
-tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 mmcblk1: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
+tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB
 Primary GPT is invalid, using alternate GPT.
  mmcblk1: p1 p2 p3 p4 p5
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-06-27 00:26:40.828633247 -0700
+++ /tmp/foob	2016-06-27 00:26:40.828633247 -0700
@@ -39,7 +39,7 @@
 	 #######################################################
 	 3.2 MiB/s
 done
-Bytes transferred = 5572104 (550608 hex)
+Bytes transferred = 5572016 (5505b0 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 51511 (c937 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x550608 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5505b0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff936 ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc4-next-20160624-102247-g2cf991d (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Jun 24 00:18:54 PDT 2016
+Linux version 4.7.0-rc4-next-20160627-102768-gaa20c9a (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jun 27 00:18:51 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,20 +68,20 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0e5b900, node_mem_map eeff9000
+free_area_init_node: node 0, pgdat c0e5b980, node_mem_map eeff9000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-percpu: Embedded 14 pages/cpu @eef8c000 s26048 r8192 d23104 u57344
-pcpu-alloc: s26048 r8192 d23104 u57344 alloc=14*4096
+percpu: Embedded 13 pages/cpu @eef90000 s23936 r8192 d21120 u53248
+pcpu-alloc: s23936 r8192 d21120 u53248 alloc=13*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=edbe1840-3633-c751-7993-b0ac834f81e0 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998840K/2096128K available (8441K kernel code, 373K rwdata, 2848K rodata, 1024K init, 366K bss, 31752K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998856K/2096128K available (8441K kernel code, 373K rwdata, 2844K rodata, 1024K init, 366K bss, 31736K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -89,10 +89,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0c06630   (12282 kB)
+      .text : 0xc0008000 - 0xc0c056b8   (12278 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0e5d4a0   ( 374 kB)
-       .bss : 0xc0e5f000 - 0xc0ebab04   ( 367 kB)
+      .data : 0xc0e00000 - 0xc0e5d540   ( 374 kB)
+       .bss : 0xc0e5f000 - 0xc0ebab14   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -130,8 +130,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?
-hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
+No ATAGs?hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 4 bytes.
 tegra-mc 7000f000.memory-controller: no memory timings for RAM code 3 registered
 vdd_5v_sata: Failed to request enable GPIO30: -517
@@ -364,7 +363,7 @@
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
-tps65910-rtc tps65910-rtc: setting system clock to 2000-01-01 00:00:04 UTC (946684804)
+tps65910-rtc tps65910-rtc: setting system clock to 2000-01-01 00:00:03 UTC (946684803)
 vdd_sata,avdd_plle: disabling
 usb1_vbus: disabling
 ALSA device list:
@@ -376,6 +375,7 @@
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (c0d00000 - c0e00000)
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    8.481741] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# 
\ No newline at end of file
