Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2015-08-13 00:52:06.638883732 -0700
+++ /tmp/foob	2015-08-13 00:52:06.638883732 -0700
@@ -1,6 +1,6 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
-Linux version 4.2.0-rc6-next-20150812+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Aug 12 07:19:00 PDT 2015
+Linux version 4.2.0-rc6-next-20150813+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Aug 13 00:48:57 PDT 2015
 CPU: AArch64 Processor [411fd070] revision 0
 Detected PIPT I-cache on CPU0
 alternatives: enabling workaround for ARM erratum 832075
@@ -19,7 +19,7 @@
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
 software IO TLB [mem 0x79c00000-0x7dc00000] (64MB) mapped at [ffffffc039c00000-ffffffc03dbfffff]
-Memory: 939176K/1048576K available (5381K kernel code, 474K rwdata, 2248K rodata, 512K init, 204K bss, 93016K reserved, 16384K cma-reserved)
+Memory: 939192K/1048576K available (5385K kernel code, 470K rwdata, 2244K rodata, 500K init, 204K bss, 93000K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     vmalloc : 0xffffff8000000000 - 0xffffffbdbfff0000   (   246 GB)
     vmemmap : 0xffffffbdc0000000 - 0xffffffbfc0000000   (     8 GB maximum)
@@ -28,9 +28,9 @@
     PCI I/O : 0xffffffbffae00000 - 0xffffffbffbe00000   (    16 MB)
     modules : 0xffffffbffc000000 - 0xffffffc000000000   (    64 MB)
     memory  : 0xffffffc000000000 - 0xffffffc040000000   (  1024 MB)
-      .init : 0xffffffc0007f6000 - 0xffffffc000876000   (   512 KB)
+      .init : 0xffffffc0007f6000 - 0xffffffc000873000   (   500 KB)
       .text : 0xffffffc000080000 - 0xffffffc0007f5534   (  7638 KB)
-      .data : 0xffffffc000886000 - 0xffffffc0008fca00   (   475 KB)
+      .data : 0xffffffc000883000 - 0xffffffc0008f8a00   (   471 KB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 64.
@@ -61,7 +61,7 @@
 NET: Registered protocol family 16
 cpuidle: using governor ladder
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffffffc00088d000, 1 data @ ffffffc00088c000)
+vdso: 2 pages (1 code @ ffffffc000889000, 1 data @ ffffffc000888000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -88,7 +88,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 2, 16384 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.400:1): initialized
+audit: type=2000 audit(0.490:1): initialized
 HugeTLB registered 2 MB page size, pre-allocated 0 pages
 VFS: Disk quotas dquot_6.6.0
 VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
@@ -132,8 +132,8 @@
 EXT4-fs (vda2): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 254:2.
 devtmpfs: mounted
-Freeing unused kernel memory: 512K (ffffffc0007f6000 - ffffffc000876000)
-Freeing alternatives memory: 44K (ffffffc000876000 - ffffffc000881000)
+Freeing unused kernel memory: 500K (ffffffc0007f6000 - ffffffc000873000)
+Freeing alternatives memory: 44K (ffffffc000873000 - ffffffc00087e000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 bash-4.3# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-08-13 00:52:06.646883728 -0700
+++ /tmp/foob	2015-08-13 00:52:06.646883728 -0700
@@ -37,21 +37,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #############################################
+	 ##############################################
 	 3.3 MiB/s
 done
-Bytes transferred = 6383368 (616708 hex)
+Bytes transferred = 6386256 (617250 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.8 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x616708 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x617250 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc6-next-20150812-35595-gc1a0c66 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Aug 12 07:17:44 PDT 2015
+Linux version 4.2.0-rc6-next-20150813-36072-g30b42f4 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Aug 13 00:47:41 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -67,7 +67,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f5a1c0, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0f5e380, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -80,7 +80,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999008K/2096128K available (8018K kernel code, 1016K rwdata, 3604K rodata, 1004K init, 340K bss, 31584K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1998992K/2096128K available (8026K kernel code, 1016K rwdata, 3608K rodata, 1008K init, 340K bss, 31600K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,10 +88,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d62b54   (11627 kB)
-      .init : 0xc0d63000 - 0xc0e5e000   (1004 kB)
-      .data : 0xc0e5e000 - 0xc0f5c180   (1017 kB)
-       .bss : 0xc0f5f000 - 0xc0fb4268   ( 341 kB)
+      .text : 0xc0208000 - 0xc0d65bcc   (11639 kB)
+      .init : 0xc0d66000 - 0xc0e62000   (1008 kB)
+      .data : 0xc0e62000 - 0xc0f60340   (1017 kB)
+       .bss : 0xc0f63000 - 0xc0fb82e8   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -296,16 +296,16 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-22 23:11:17 UTC (951261077)
-vdd_cam_1v8_reg: disabling
-usb1_vbus: disabling
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-23 16:41:12 UTC (951324072)
 dvdd_ts: disabling
+usb1_vbus: disabling
+vdd_cam_1v8_reg: disabling
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1004K (c0d63000 - c0e5e000)
+Freeing unused kernel memory: 1008K (c0d66000 - c0e62000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.224192] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.219193] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2015-08-13 00:52:06.650883725 -0700
+++ /tmp/foob	2015-08-13 00:52:06.654883722 -0700
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 ###################
-	 3.4 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 5041872 (4ceed0 hex)
+Bytes transferred = 5044744 (4cfa08 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3.3 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4ceed0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4cfa08 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc6-next-20150812-35595-gc1a0c66 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Aug 12 07:18:08 PDT 2015
+Linux version 4.2.0-rc6-next-20150813-36072-g30b42f4 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Aug 13 00:48:06 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c09b67c0, node_mem_map ee7f7000
+free_area_init_node: node 0, pgdat c09b87c0, node_mem_map ee7f7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002708K/2096128K available (6876K kernel code, 344K rwdata, 2380K rodata, 316K init, 363K bss, 27884K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 2002700K/2096128K available (6888K kernel code, 344K rwdata, 2380K rodata, 312K init, 363K bss, 27892K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0912698   (9258 kB)
-      .init : 0xc0913000 - 0xc0962000   ( 316 kB)
-      .data : 0xc0962000 - 0xc09b8340   ( 345 kB)
-       .bss : 0xc09bb000 - 0xc0a15da0   ( 364 kB)
+      .text : 0xc0008000 - 0xc09156a8   (9270 kB)
+      .init : 0xc0916000 - 0xc0964000   ( 312 kB)
+      .data : 0xc0964000 - 0xc09ba340   ( 345 kB)
+       .bss : 0xc09bd000 - 0xc0a17da0   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -282,57 +282,57 @@
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
 ------------[ cut here ]------------
-WARNING: CPU: 0 PID: 70 at include/linux/ww_mutex.h:158 drm_atomic_check_only+0x4fc/0x574()
+WARNING: CPU: 0 PID: 6 at include/linux/ww_mutex.h:158 drm_atomic_check_only+0x4fc/0x574()
 DEBUG_LOCKS_WARN_ON(ctx->done_acquire)
 Modules linked in:
-CPU: 0 PID: 70 Comm: kworker/u8:1 Not tainted 4.2.0-rc6-next-20150812-35595-gc1a0c66 #1
+CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.2.0-rc6-next-20150813-36072-g30b42f4 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0016898>] (unwind_backtrace) from [<c0012124>] (show_stack+0x10/0x14)
-[<c0012124>] (show_stack) from [<c067cc68>] (dump_stack+0x84/0xd0)
-[<c067cc68>] (dump_stack) from [<c00257a8>] (warn_slowpath_common+0x74/0xb0)
-[<c00257a8>] (warn_slowpath_common) from [<c0025878>] (warn_slowpath_fmt+0x30/0x40)
-[<c0025878>] (warn_slowpath_fmt) from [<c02e3eec>] (drm_atomic_check_only+0x4fc/0x574)
-[<c02e3eec>] (drm_atomic_check_only) from [<c02e3fd8>] (drm_atomic_commit+0x10/0x64)
-[<c02e3fd8>] (drm_atomic_commit) from [<c02c2ac8>] (drm_atomic_helper_set_config+0x394/0x3ec)
-[<c02c2ac8>] (drm_atomic_helper_set_config) from [<c02d4180>] (drm_mode_set_config_internal+0x58/0xd4)
-[<c02d4180>] (drm_mode_set_config_internal) from [<c02c50d8>] (restore_fbdev_mode+0xd4/0xf4)
-[<c02c50d8>] (restore_fbdev_mode) from [<c02c6940>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
-[<c02c6940>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c02c69ac>] (drm_fb_helper_set_par+0x2c/0x54)
-[<c02c69ac>] (drm_fb_helper_set_par) from [<c02632d4>] (fbcon_init+0x474/0x4dc)
-[<c02632d4>] (fbcon_init) from [<c0299e70>] (visual_init+0xb0/0xfc)
-[<c0299e70>] (visual_init) from [<c029be14>] (do_bind_con_driver+0x134/0x354)
-[<c029be14>] (do_bind_con_driver) from [<c029c1a4>] (do_take_over_console+0x170/0x1dc)
-[<c029c1a4>] (do_take_over_console) from [<c02633a0>] (do_fbcon_takeover+0x64/0xcc)
-[<c02633a0>] (do_fbcon_takeover) from [<c003e4ac>] (notifier_call_chain+0x44/0x84)
-[<c003e4ac>] (notifier_call_chain) from [<c003e680>] (__blocking_notifier_call_chain+0x48/0x60)
-[<c003e680>] (__blocking_notifier_call_chain) from [<c003e6b0>] (blocking_notifier_call_chain+0x18/0x20)
-[<c003e6b0>] (blocking_notifier_call_chain) from [<c026d290>] (register_framebuffer+0x184/0x264)
-[<c026d290>] (register_framebuffer) from [<c02c6c18>] (drm_fb_helper_initial_config+0x244/0x3e4)
-[<c02c6c18>] (drm_fb_helper_initial_config) from [<c02e8c70>] (tegra_drm_fb_init+0x48/0xb4)
-[<c02e8c70>] (tegra_drm_fb_init) from [<c02e6f08>] (tegra_drm_load+0x170/0x224)
-[<c02e6f08>] (tegra_drm_load) from [<c02cef0c>] (drm_dev_register+0xa0/0xb4)
-[<c02cef0c>] (drm_dev_register) from [<c02e6a64>] (host1x_drm_probe+0x44/0xb0)
-[<c02e6a64>] (host1x_drm_probe) from [<c02b1e90>] (host1x_device_probe+0x1c/0x28)
-[<c02b1e90>] (host1x_device_probe) from [<c02fd408>] (driver_probe_device+0x1e0/0x2a0)
-[<c02fd408>] (driver_probe_device) from [<c02fbaa8>] (bus_for_each_drv+0x44/0x8c)
-[<c02fbaa8>] (bus_for_each_drv) from [<c02fd19c>] (__device_attach+0xa0/0x104)
-[<c02fd19c>] (__device_attach) from [<c02fc918>] (bus_probe_device+0x84/0x8c)
-[<c02fc918>] (bus_probe_device) from [<c02fae6c>] (device_add+0x33c/0x524)
-[<c02fae6c>] (device_add) from [<c02b209c>] (host1x_subdev_register+0xb0/0xd4)
-[<c02b209c>] (host1x_subdev_register) from [<c02b2954>] (host1x_client_register+0xf4/0x11c)
-[<c02b2954>] (host1x_client_register) from [<c02f016c>] (tegra_hdmi_probe+0x170/0x250)
-[<c02f016c>] (tegra_hdmi_probe) from [<c02fe8e0>] (platform_drv_probe+0x48/0xa4)
-[<c02fe8e0>] (platform_drv_probe) from [<c02fd408>] (driver_probe_device+0x1e0/0x2a0)
-[<c02fd408>] (driver_probe_device) from [<c02fbaa8>] (bus_for_each_drv+0x44/0x8c)
-[<c02fbaa8>] (bus_for_each_drv) from [<c02fd19c>] (__device_attach+0xa0/0x104)
-[<c02fd19c>] (__device_attach) from [<c02fc918>] (bus_probe_device+0x84/0x8c)
-[<c02fc918>] (bus_probe_device) from [<c02fccf0>] (deferred_probe_work_func+0x5c/0x88)
-[<c02fccf0>] (deferred_probe_work_func) from [<c00385b0>] (process_one_work+0x120/0x328)
-[<c00385b0>] (process_one_work) from [<c0038814>] (worker_thread+0x28/0x490)
-[<c0038814>] (worker_thread) from [<c003d90c>] (kthread+0xd8/0xf4)
-[<c003d90c>] (kthread) from [<c000f628>] (ret_from_fork+0x14/0x2c)
----[ end trace 402c977eafd4c8f8 ]---
+[<c0012124>] (show_stack) from [<c067f600>] (dump_stack+0x84/0xd0)
+[<c067f600>] (dump_stack) from [<c00257b8>] (warn_slowpath_common+0x74/0xb0)
+[<c00257b8>] (warn_slowpath_common) from [<c0025888>] (warn_slowpath_fmt+0x30/0x40)
+[<c0025888>] (warn_slowpath_fmt) from [<c02e6758>] (drm_atomic_check_only+0x4fc/0x574)
+[<c02e6758>] (drm_atomic_check_only) from [<c02e6844>] (drm_atomic_commit+0x10/0x64)
+[<c02e6844>] (drm_atomic_commit) from [<c02c531c>] (drm_atomic_helper_set_config+0x394/0x3ec)
+[<c02c531c>] (drm_atomic_helper_set_config) from [<c02d69ec>] (drm_mode_set_config_internal+0x58/0xd4)
+[<c02d69ec>] (drm_mode_set_config_internal) from [<c02c7924>] (restore_fbdev_mode+0xd4/0xf4)
+[<c02c7924>] (restore_fbdev_mode) from [<c02c918c>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
+[<c02c918c>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c02c91f8>] (drm_fb_helper_set_par+0x2c/0x54)
+[<c02c91f8>] (drm_fb_helper_set_par) from [<c0265a10>] (fbcon_init+0x474/0x4dc)
+[<c0265a10>] (fbcon_init) from [<c029c6c8>] (visual_init+0xb0/0xfc)
+[<c029c6c8>] (visual_init) from [<c029e66c>] (do_bind_con_driver+0x134/0x354)
+[<c029e66c>] (do_bind_con_driver) from [<c029e9fc>] (do_take_over_console+0x170/0x1dc)
+[<c029e9fc>] (do_take_over_console) from [<c0265adc>] (do_fbcon_takeover+0x64/0xcc)
+[<c0265adc>] (do_fbcon_takeover) from [<c003e500>] (notifier_call_chain+0x44/0x84)
+[<c003e500>] (notifier_call_chain) from [<c003e6d4>] (__blocking_notifier_call_chain+0x48/0x60)
+[<c003e6d4>] (__blocking_notifier_call_chain) from [<c003e704>] (blocking_notifier_call_chain+0x18/0x20)
+[<c003e704>] (blocking_notifier_call_chain) from [<c026f9cc>] (register_framebuffer+0x184/0x264)
+[<c026f9cc>] (register_framebuffer) from [<c02c9464>] (drm_fb_helper_initial_config+0x244/0x3e4)
+[<c02c9464>] (drm_fb_helper_initial_config) from [<c02eb4dc>] (tegra_drm_fb_init+0x48/0xb4)
+[<c02eb4dc>] (tegra_drm_fb_init) from [<c02e9774>] (tegra_drm_load+0x170/0x224)
+[<c02e9774>] (tegra_drm_load) from [<c02d1778>] (drm_dev_register+0xa0/0xb4)
+[<c02d1778>] (drm_dev_register) from [<c02e92d0>] (host1x_drm_probe+0x44/0xb0)
+[<c02e92d0>] (host1x_drm_probe) from [<c02b46e8>] (host1x_device_probe+0x1c/0x28)
+[<c02b46e8>] (host1x_device_probe) from [<c02ffc7c>] (driver_probe_device+0x1e0/0x2a0)
+[<c02ffc7c>] (driver_probe_device) from [<c02fe31c>] (bus_for_each_drv+0x44/0x8c)
+[<c02fe31c>] (bus_for_each_drv) from [<c02ffa10>] (__device_attach+0xa0/0x104)
+[<c02ffa10>] (__device_attach) from [<c02ff18c>] (bus_probe_device+0x84/0x8c)
+[<c02ff18c>] (bus_probe_device) from [<c02fd6e0>] (device_add+0x33c/0x524)
+[<c02fd6e0>] (device_add) from [<c02b48f4>] (host1x_subdev_register+0xb0/0xd4)
+[<c02b48f4>] (host1x_subdev_register) from [<c02b51ac>] (host1x_client_register+0xf4/0x11c)
+[<c02b51ac>] (host1x_client_register) from [<c02f29d8>] (tegra_hdmi_probe+0x170/0x250)
+[<c02f29d8>] (tegra_hdmi_probe) from [<c0301154>] (platform_drv_probe+0x48/0xa4)
+[<c0301154>] (platform_drv_probe) from [<c02ffc7c>] (driver_probe_device+0x1e0/0x2a0)
+[<c02ffc7c>] (driver_probe_device) from [<c02fe31c>] (bus_for_each_drv+0x44/0x8c)
+[<c02fe31c>] (bus_for_each_drv) from [<c02ffa10>] (__device_attach+0xa0/0x104)
+[<c02ffa10>] (__device_attach) from [<c02ff18c>] (bus_probe_device+0x84/0x8c)
+[<c02ff18c>] (bus_probe_device) from [<c02ff564>] (deferred_probe_work_func+0x5c/0x88)
+[<c02ff564>] (deferred_probe_work_func) from [<c00385c0>] (process_one_work+0x120/0x328)
+[<c00385c0>] (process_one_work) from [<c0038824>] (worker_thread+0x28/0x490)
+[<c0038824>] (worker_thread) from [<c003d91c>] (kthread+0xd8/0xf4)
+[<c003d91c>] (kthread) from [<c000f628>] (ret_from_fork+0x14/0x2c)
+---[ end trace 56e216c3ab03e430 ]---
 Console: switching to colour frame buffer device 240x75
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
@@ -344,10 +344,10 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-22 23:11:45 UTC (951261105)
-vdd_cam_1v8_reg: disabling
-usb1_vbus: disabling
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-23 16:41:42 UTC (951324102)
 dvdd_ts: disabling
+usb1_vbus: disabling
+vdd_cam_1v8_reg: disabling
 ALSA device list:
   #0: NVIDIA Tegra Dalmore
 EXT2-fs (mmcblk0p1): error: couldn't mount because of unsupported optional features (4)
@@ -356,7 +356,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 316K (c0913000 - c0962000)
+Freeing unused kernel memory: 312K (c0916000 - c0964000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.160079] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.269983] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-08-13 00:52:06.658883720 -0700
+++ /tmp/foob	2015-08-13 00:52:06.662883718 -0700
@@ -39,10 +39,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #############################################
+	 ##############################################
 	 3.2 MiB/s
 done
-Bytes transferred = 6383368 (616708 hex)
+Bytes transferred = 6386256 (617250 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -53,7 +53,7 @@
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x616708 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x617250 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012a89
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc6-next-20150812-35595-gc1a0c66 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Aug 12 07:17:44 PDT 2015
+Linux version 4.2.0-rc6-next-20150813-36072-g30b42f4 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Aug 13 00:47:41 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f5a1c0, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0f5e380, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998900K/2096128K available (8018K kernel code, 1016K rwdata, 3604K rodata, 1004K init, 340K bss, 31692K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1998884K/2096128K available (8026K kernel code, 1016K rwdata, 3608K rodata, 1008K init, 340K bss, 31708K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d62b54   (11627 kB)
-      .init : 0xc0d63000 - 0xc0e5e000   (1004 kB)
-      .data : 0xc0e5e000 - 0xc0f5c180   (1017 kB)
-       .bss : 0xc0f5f000 - 0xc0fb4268   ( 341 kB)
+      .text : 0xc0208000 - 0xc0d65bcc   (11639 kB)
+      .init : 0xc0d66000 - 0xc0e62000   (1008 kB)
+      .data : 0xc0e62000 - 0xc0f60340   (1017 kB)
+       .bss : 0xc0f63000 - 0xc0fb82e8   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -285,20 +285,20 @@
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
++3.3V_SYS: supplied by +VDD_MUX
++3.3V_LP0: supplied by +3.3V_SYS
++1.35V_LP0(sd2): supplied by +5V_SYS
++1.05V_RUN_AVDD: supplied by +1.35V_LP0(sd2)
 mmc1: MAN_BKOPS_EN bit is not set
 mmc1: new high speed MMC card at address 0001
+tegra-pcie 1003000.pcie-controller: probing port 0, using 2 lanes
+tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
 mmcblk0: mmc1:0001 SEM16G 14.6 GiB 
 mmcblk0boot0: mmc1:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc1:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc1:0001 SEM16G partition 3 4.00 MiB
 Primary GPT is invalid, using alternate GPT.
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9
-+3.3V_SYS: supplied by +VDD_MUX
-+3.3V_LP0: supplied by +3.3V_SYS
-+1.35V_LP0(sd2): supplied by +5V_SYS
-+1.05V_RUN_AVDD: supplied by +1.35V_LP0(sd2)
-tegra-pcie 1003000.pcie-controller: probing port 0, using 2 lanes
-tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
 tegra-pcie 1003000.pcie-controller: link 0 down, retrying
 tegra-pcie 1003000.pcie-controller: link 0 down, retrying
 tegra-pcie 1003000.pcie-controller: link 0 down, retrying
@@ -345,14 +345,14 @@
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
 as3722-rtc as3722-rtc: setting system clock to 2000-02-01 00:00:18 UTC (949363218)
-+USB0_VBUS_SW: disabling
-+1.8V_RUN_VPP_FUSE: disabling
-+2.8V_RUN_CAM_AF: disabling
-+3.3V_RUN_TOUCH: disabling
-+1.05V_RUN_CAM_REAR: disabling
-+1.2V_RUN_CAM_FRONT: disabling
-+2.8V_RUN_CAM: disabling
 +1.8V_RUN_CAM: disabling
++2.8V_RUN_CAM: disabling
++1.2V_RUN_CAM_FRONT: disabling
++1.05V_RUN_CAM_REAR: disabling
++3.3V_RUN_TOUCH: disabling
++2.8V_RUN_CAM_AF: disabling
++1.8V_RUN_VPP_FUSE: disabling
++USB0_VBUS_SW: disabling
 ata1: SATA link down (SStatus 0 SControl 300)
 EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
 EXT4-fs (mmcblk0p1): couldn't mount as ext2 due to feature incompatibilities
@@ -361,7 +361,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1004K (c0d63000 - c0e5e000)
+Freeing unused kernel memory: 1008K (c0d66000 - c0e62000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2015-08-13 00:52:06.666883715 -0700
+++ /tmp/foob	2015-08-13 00:52:06.670883713 -0700
@@ -41,7 +41,7 @@
 	 ###################
 	 3.2 MiB/s
 done
-Bytes transferred = 5041872 (4ceed0 hex)
+Bytes transferred = 5044744 (4cfa08 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -52,7 +52,7 @@
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4ceed0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4cfa08 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012a89
@@ -63,7 +63,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc6-next-20150812-35595-gc1a0c66 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Aug 12 07:18:08 PDT 2015
+Linux version 4.2.0-rc6-next-20150813-36072-g30b42f4 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Aug 13 00:48:06 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -71,7 +71,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c09b67c0, node_mem_map ee7f7000
+free_area_init_node: node 0, pgdat c09b87c0, node_mem_map ee7f7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -84,7 +84,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002600K/2096128K available (6876K kernel code, 344K rwdata, 2380K rodata, 316K init, 363K bss, 27992K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 2002592K/2096128K available (6888K kernel code, 344K rwdata, 2380K rodata, 312K init, 363K bss, 28000K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,10 +92,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0912698   (9258 kB)
-      .init : 0xc0913000 - 0xc0962000   ( 316 kB)
-      .data : 0xc0962000 - 0xc09b8340   ( 345 kB)
-       .bss : 0xc09bb000 - 0xc0a15da0   ( 364 kB)
+      .text : 0xc0008000 - 0xc09156a8   (9270 kB)
+      .init : 0xc0916000 - 0xc0964000   ( 312 kB)
+      .data : 0xc0964000 - 0xc09ba340   ( 345 kB)
+       .bss : 0xc09bd000 - 0xc0a17da0   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -302,16 +302,44 @@
 tegra-pcie 1003000.pcie-controller: link 0 down, ignoring
 tegra-pcie 1003000.pcie-controller: probing port 1, using 1 lanes
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000000
-tegra-pcie 1003000.pcie-controller: link 1 down, retrying
-tegra-pcie 1003000.pcie-controller: link 1 down, retrying
-tegra-pcie 1003000.pcie-controller: link 1 down, retrying
-tegra-pcie 1003000.pcie-controller: link 1 down, ignoring
 tegra-pcie 1003000.pcie-controller: PCI host bridge to bus 0000:00
 pci_bus 0000:00: root bus resource [mem 0x13000000-0x1fffffff]
 pci_bus 0000:00: root bus resource [mem 0x20000000-0x3fffffff pref]
 pci_bus 0000:00: root bus resource [bus 00-ff]
 pci_bus 0000:00: root bus resource [io  0x1000-0xffff]
-PCI: bus0: Fast back to back transfers enabled
+pci 0000:00:02.0: [10de:0e13] type 01 class 0x060400
+pci 0000:00:02.0: PME# supported from D0 D1 D2 D3hot D3cold
+PCI: bus0: Fast back to back transfers disabled
+pci 0000:00:02.0: bridge configuration invalid ([bus 00-00]), reconfiguring
+pci 0000:00:02.0: PCI bridge to [bus 01-ff]
+pci 0000:00:02.0:   bridge window [io  0x0000-0x0fff]
+pci 0000:01:00.0: [10ec:8168] type 00 class 0x020000
+pci 0000:01:00.0: reg 0x10: [io  0x0000-0x00ff]
+pci 0000:01:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit]
+pci 0000:01:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
+pci 0000:01:00.0: supports D1 D2
+pci 0000:01:00.0: PME# supported from D0 D1 D2 D3hot D3cold
+PCI: bus1: Fast back to back transfers disabled
+pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
+pci 0000:00:02.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
+pci 0000:00:02.0: BAR 9: assigned [mem 0x20000000-0x200fffff 64bit pref]
+pci 0000:00:02.0: BAR 7: assigned [io  0x1000-0x1fff]
+pci 0000:01:00.0: BAR 4: assigned [mem 0x20000000-0x20003fff 64bit pref]
+pci 0000:01:00.0: BAR 2: assigned [mem 0x13000000-0x13000fff 64bit]
+pci 0000:01:00.0: BAR 0: assigned [io  0x1000-0x10ff]
+pci 0000:00:02.0: PCI bridge to [bus 01]
+pci 0000:00:02.0:   bridge window [io  0x1000-0x1fff]
+pci 0000:00:02.0:   bridge window [mem 0x13000000-0x130fffff]
+pci 0000:00:02.0:   bridge window [mem 0x20000000-0x200fffff 64bit pref]
+pci 0000:00:02.0: nv_msi_ht_cap_quirk didn't locate host bridge
+pcieport 0000:00:02.0: enabling device (0140 -> 0143)
+pcieport 0000:00:02.0: Signaling PME through PCIe PME interrupt
+pci 0000:01:00.0: Signaling PME through PCIe PME interrupt
+pcie_pme 0000:00:02.0:pcie01: service driver pcie_pme loaded
+r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
+r8169 0000:01:00.0: enabling device (0140 -> 0143)
+r8169 0000:01:00.0 eth0: RTL8168g/8111g at 0xf014c000, 00:04:4b:25:b3:b4, XID 0c000800 IRQ 388
+r8169 0000:01:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 +5V_HDMI_CON: supplied by +5V_SYS
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 +3.3V_RUN: supplied by +3.3V_SYS
@@ -320,57 +348,57 @@
 [drm] No driver support for vblank timestamp query.
 tegra-hdmi 54280000.hdmi: cannot set audio to 44100 at 297000000 pclk
 ------------[ cut here ]------------
-WARNING: CPU: 0 PID: 72 at include/linux/ww_mutex.h:158 drm_atomic_check_only+0x4fc/0x574()
+WARNING: CPU: 0 PID: 6 at include/linux/ww_mutex.h:158 drm_atomic_check_only+0x4fc/0x574()
 DEBUG_LOCKS_WARN_ON(ctx->done_acquire)
 Modules linked in:
-CPU: 0 PID: 72 Comm: kworker/u8:1 Not tainted 4.2.0-rc6-next-20150812-35595-gc1a0c66 #1
+CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.2.0-rc6-next-20150813-36072-g30b42f4 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0016898>] (unwind_backtrace) from [<c0012124>] (show_stack+0x10/0x14)
-[<c0012124>] (show_stack) from [<c067cc68>] (dump_stack+0x84/0xd0)
-[<c067cc68>] (dump_stack) from [<c00257a8>] (warn_slowpath_common+0x74/0xb0)
-[<c00257a8>] (warn_slowpath_common) from [<c0025878>] (warn_slowpath_fmt+0x30/0x40)
-[<c0025878>] (warn_slowpath_fmt) from [<c02e3eec>] (drm_atomic_check_only+0x4fc/0x574)
-[<c02e3eec>] (drm_atomic_check_only) from [<c02e3fd8>] (drm_atomic_commit+0x10/0x64)
-[<c02e3fd8>] (drm_atomic_commit) from [<c02c2ac8>] (drm_atomic_helper_set_config+0x394/0x3ec)
-[<c02c2ac8>] (drm_atomic_helper_set_config) from [<c02d4180>] (drm_mode_set_config_internal+0x58/0xd4)
-[<c02d4180>] (drm_mode_set_config_internal) from [<c02c50d8>] (restore_fbdev_mode+0xd4/0xf4)
-[<c02c50d8>] (restore_fbdev_mode) from [<c02c6940>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
-[<c02c6940>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c02c69ac>] (drm_fb_helper_set_par+0x2c/0x54)
-[<c02c69ac>] (drm_fb_helper_set_par) from [<c02632d4>] (fbcon_init+0x474/0x4dc)
-[<c02632d4>] (fbcon_init) from [<c0299e70>] (visual_init+0xb0/0xfc)
-[<c0299e70>] (visual_init) from [<c029be14>] (do_bind_con_driver+0x134/0x354)
-[<c029be14>] (do_bind_con_driver) from [<c029c1a4>] (do_take_over_console+0x170/0x1dc)
-[<c029c1a4>] (do_take_over_console) from [<c02633a0>] (do_fbcon_takeover+0x64/0xcc)
-[<c02633a0>] (do_fbcon_takeover) from [<c003e4ac>] (notifier_call_chain+0x44/0x84)
-[<c003e4ac>] (notifier_call_chain) from [<c003e680>] (__blocking_notifier_call_chain+0x48/0x60)
-[<c003e680>] (__blocking_notifier_call_chain) from [<c003e6b0>] (blocking_notifier_call_chain+0x18/0x20)
-[<c003e6b0>] (blocking_notifier_call_chain) from [<c026d290>] (register_framebuffer+0x184/0x264)
-[<c026d290>] (register_framebuffer) from [<c02c6c18>] (drm_fb_helper_initial_config+0x244/0x3e4)
-[<c02c6c18>] (drm_fb_helper_initial_config) from [<c02e8c70>] (tegra_drm_fb_init+0x48/0xb4)
-[<c02e8c70>] (tegra_drm_fb_init) from [<c02e6f08>] (tegra_drm_load+0x170/0x224)
-[<c02e6f08>] (tegra_drm_load) from [<c02cef0c>] (drm_dev_register+0xa0/0xb4)
-[<c02cef0c>] (drm_dev_register) from [<c02e6a64>] (host1x_drm_probe+0x44/0xb0)
-[<c02e6a64>] (host1x_drm_probe) from [<c02b1e90>] (host1x_device_probe+0x1c/0x28)
-[<c02b1e90>] (host1x_device_probe) from [<c02fd408>] (driver_probe_device+0x1e0/0x2a0)
-[<c02fd408>] (driver_probe_device) from [<c02fbaa8>] (bus_for_each_drv+0x44/0x8c)
-[<c02fbaa8>] (bus_for_each_drv) from [<c02fd19c>] (__device_attach+0xa0/0x104)
-[<c02fd19c>] (__device_attach) from [<c02fc918>] (bus_probe_device+0x84/0x8c)
-[<c02fc918>] (bus_probe_device) from [<c02fae6c>] (device_add+0x33c/0x524)
-[<c02fae6c>] (device_add) from [<c02b209c>] (host1x_subdev_register+0xb0/0xd4)
-[<c02b209c>] (host1x_subdev_register) from [<c02b2954>] (host1x_client_register+0xf4/0x11c)
-[<c02b2954>] (host1x_client_register) from [<c02f016c>] (tegra_hdmi_probe+0x170/0x250)
-[<c02f016c>] (tegra_hdmi_probe) from [<c02fe8e0>] (platform_drv_probe+0x48/0xa4)
-[<c02fe8e0>] (platform_drv_probe) from [<c02fd408>] (driver_probe_device+0x1e0/0x2a0)
-[<c02fd408>] (driver_probe_device) from [<c02fbaa8>] (bus_for_each_drv+0x44/0x8c)
-[<c02fbaa8>] (bus_for_each_drv) from [<c02fd19c>] (__device_attach+0xa0/0x104)
-[<c02fd19c>] (__device_attach) from [<c02fc918>] (bus_probe_device+0x84/0x8c)
-[<c02fc918>] (bus_probe_device) from [<c02fccf0>] (deferred_probe_work_func+0x5c/0x88)
-[<c02fccf0>] (deferred_probe_work_func) from [<c00385b0>] (process_one_work+0x120/0x328)
-[<c00385b0>] (process_one_work) from [<c0038814>] (worker_thread+0x28/0x490)
-[<c0038814>] (worker_thread) from [<c003d90c>] (kthread+0xd8/0xf4)
-[<c003d90c>] (kthread) from [<c000f628>] (ret_from_fork+0x14/0x2c)
----[ end trace b5ae9c672fd7bc21 ]---
+[<c0012124>] (show_stack) from [<c067f600>] (dump_stack+0x84/0xd0)
+[<c067f600>] (dump_stack) from [<c00257b8>] (warn_slowpath_common+0x74/0xb0)
+[<c00257b8>] (warn_slowpath_common) from [<c0025888>] (warn_slowpath_fmt+0x30/0x40)
+[<c0025888>] (warn_slowpath_fmt) from [<c02e6758>] (drm_atomic_check_only+0x4fc/0x574)
+[<c02e6758>] (drm_atomic_check_only) from [<c02e6844>] (drm_atomic_commit+0x10/0x64)
+[<c02e6844>] (drm_atomic_commit) from [<c02c531c>] (drm_atomic_helper_set_config+0x394/0x3ec)
+[<c02c531c>] (drm_atomic_helper_set_config) from [<c02d69ec>] (drm_mode_set_config_internal+0x58/0xd4)
+[<c02d69ec>] (drm_mode_set_config_internal) from [<c02c7924>] (restore_fbdev_mode+0xd4/0xf4)
+[<c02c7924>] (restore_fbdev_mode) from [<c02c918c>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
+[<c02c918c>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c02c91f8>] (drm_fb_helper_set_par+0x2c/0x54)
+[<c02c91f8>] (drm_fb_helper_set_par) from [<c0265a10>] (fbcon_init+0x474/0x4dc)
+[<c0265a10>] (fbcon_init) from [<c029c6c8>] (visual_init+0xb0/0xfc)
+[<c029c6c8>] (visual_init) from [<c029e66c>] (do_bind_con_driver+0x134/0x354)
+[<c029e66c>] (do_bind_con_driver) from [<c029e9fc>] (do_take_over_console+0x170/0x1dc)
+[<c029e9fc>] (do_take_over_console) from [<c0265adc>] (do_fbcon_takeover+0x64/0xcc)
+[<c0265adc>] (do_fbcon_takeover) from [<c003e500>] (notifier_call_chain+0x44/0x84)
+[<c003e500>] (notifier_call_chain) from [<c003e6d4>] (__blocking_notifier_call_chain+0x48/0x60)
+[<c003e6d4>] (__blocking_notifier_call_chain) from [<c003e704>] (blocking_notifier_call_chain+0x18/0x20)
+[<c003e704>] (blocking_notifier_call_chain) from [<c026f9cc>] (register_framebuffer+0x184/0x264)
+[<c026f9cc>] (register_framebuffer) from [<c02c9464>] (drm_fb_helper_initial_config+0x244/0x3e4)
+[<c02c9464>] (drm_fb_helper_initial_config) from [<c02eb4dc>] (tegra_drm_fb_init+0x48/0xb4)
+[<c02eb4dc>] (tegra_drm_fb_init) from [<c02e9774>] (tegra_drm_load+0x170/0x224)
+[<c02e9774>] (tegra_drm_load) from [<c02d1778>] (drm_dev_register+0xa0/0xb4)
+[<c02d1778>] (drm_dev_register) from [<c02e92d0>] (host1x_drm_probe+0x44/0xb0)
+[<c02e92d0>] (host1x_drm_probe) from [<c02b46e8>] (host1x_device_probe+0x1c/0x28)
+[<c02b46e8>] (host1x_device_probe) from [<c02ffc7c>] (driver_probe_device+0x1e0/0x2a0)
+[<c02ffc7c>] (driver_probe_device) from [<c02fe31c>] (bus_for_each_drv+0x44/0x8c)
+[<c02fe31c>] (bus_for_each_drv) from [<c02ffa10>] (__device_attach+0xa0/0x104)
+[<c02ffa10>] (__device_attach) from [<c02ff18c>] (bus_probe_device+0x84/0x8c)
+[<c02ff18c>] (bus_probe_device) from [<c02fd6e0>] (device_add+0x33c/0x524)
+[<c02fd6e0>] (device_add) from [<c02b48f4>] (host1x_subdev_register+0xb0/0xd4)
+[<c02b48f4>] (host1x_subdev_register) from [<c02b51ac>] (host1x_client_register+0xf4/0x11c)
+[<c02b51ac>] (host1x_client_register) from [<c02f29d8>] (tegra_hdmi_probe+0x170/0x250)
+[<c02f29d8>] (tegra_hdmi_probe) from [<c0301154>] (platform_drv_probe+0x48/0xa4)
+[<c0301154>] (platform_drv_probe) from [<c02ffc7c>] (driver_probe_device+0x1e0/0x2a0)
+[<c02ffc7c>] (driver_probe_device) from [<c02fe31c>] (bus_for_each_drv+0x44/0x8c)
+[<c02fe31c>] (bus_for_each_drv) from [<c02ffa10>] (__device_attach+0xa0/0x104)
+[<c02ffa10>] (__device_attach) from [<c02ff18c>] (bus_probe_device+0x84/0x8c)
+[<c02ff18c>] (bus_probe_device) from [<c02ff564>] (deferred_probe_work_func+0x5c/0x88)
+[<c02ff564>] (deferred_probe_work_func) from [<c00385c0>] (process_one_work+0x120/0x328)
+[<c00385c0>] (process_one_work) from [<c0038824>] (worker_thread+0x28/0x490)
+[<c0038824>] (worker_thread) from [<c003d91c>] (kthread+0xd8/0xf4)
+[<c003d91c>] (kthread) from [<c000f628>] (ret_from_fork+0x14/0x2c)
+---[ end trace 9da4cc0f34b5f9e3 ]---
 Console: switching to colour frame buffer device 512x135
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
@@ -393,15 +421,15 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-02-01 00:00:53 UTC (949363253)
-+USB0_VBUS_SW: disabling
-+1.8V_RUN_VPP_FUSE: disabling
-+2.8V_RUN_CAM_AF: disabling
-+3.3V_RUN_TOUCH: disabling
-+1.05V_RUN_CAM_REAR: disabling
-+1.2V_RUN_CAM_FRONT: disabling
-+2.8V_RUN_CAM: disabling
+as3722-rtc as3722-rtc: setting system clock to 2000-02-01 00:00:51 UTC (949363251)
 +1.8V_RUN_CAM: disabling
++2.8V_RUN_CAM: disabling
++1.2V_RUN_CAM_FRONT: disabling
++1.05V_RUN_CAM_REAR: disabling
++3.3V_RUN_TOUCH: disabling
++2.8V_RUN_CAM_AF: disabling
++1.8V_RUN_VPP_FUSE: disabling
++USB0_VBUS_SW: disabling
 ALSA device list:
   #0: tegra-hda at 0x70038000 irq 342
   #1: NVIDIA Tegra Jetson TK1
@@ -413,7 +441,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 316K (c0913000 - c0962000)
+Freeing unused kernel memory: 312K (c0916000 - c0964000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    7.873270] asix 2-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:11:31:3b
+root@(none):/# [    6.955379] asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:11:31:3b
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-08-13 00:52:06.674883711 -0700
+++ /tmp/foob	2015-08-13 00:52:06.678883708 -0700
@@ -45,21 +45,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #############################################
+	 ##############################################
 	 4.1 MiB/s
 done
-Bytes transferred = 6383368 (616708 hex)
+Bytes transferred = 6386256 (617250 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 2.5 MiB/s
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x616708 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x617250 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -67,7 +67,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc6-next-20150812-35595-gc1a0c66 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Aug 12 07:17:44 PDT 2015
+Linux version 4.2.0-rc6-next-20150813-36072-g30b42f4 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Aug 13 00:47:41 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -75,7 +75,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0f5a1c0, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c0f5e380, node_mem_map eeff8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -88,7 +88,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 959816K/1048576K available (8018K kernel code, 1016K rwdata, 3604K rodata, 1004K init, 340K bss, 23224K reserved, 65536K cma-reserved, 204800K highmem)
+Memory: 959800K/1048576K available (8026K kernel code, 1016K rwdata, 3608K rodata, 1008K init, 340K bss, 23240K reserved, 65536K cma-reserved, 204800K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -96,10 +96,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d62b54   (11627 kB)
-      .init : 0xc0d63000 - 0xc0e5e000   (1004 kB)
-      .data : 0xc0e5e000 - 0xc0f5c180   (1017 kB)
-       .bss : 0xc0f5f000 - 0xc0fb4268   ( 341 kB)
+      .text : 0xc0208000 - 0xc0d65bcc   (11639 kB)
+      .init : 0xc0d66000 - 0xc0e62000   (1008 kB)
+      .data : 0xc0e62000 - 0xc0f60340   (1017 kB)
+       .bss : 0xc0f63000 - 0xc0fb82e8   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -323,29 +323,28 @@
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
  mmcblk0: p1
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-hub 1-1:1.0: USB hub found
-rtc-em3027 2-0056: setting system clock to 2014-12-26 20:24:32 UTC (1419625472)
-hub 1-1:1.0: 4 ports detected
+rtc-em3027 2-0056: setting system clock to 2014-12-26 20:25:09 UTC (1419625509)
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
-mmc1: new high speed SD card at address b368
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
- mmcblk1: p1 p2
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
+mmc1: new high speed SD card at address b368
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
+hub 1-1:1.0: USB hub found
+ mmcblk1: p1 p2
 devtmpfs: mounted
-Freeing unused kernel memory: 1004K (c0d63000 - c0e5e000)
-usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
+Freeing unused kernel memory: 1008K (c0d66000 - c0e62000)
+hub 1-1:1.0: 4 ports detected
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# 
\ No newline at end of file
+root@(none):/# [    2.447200] usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2015-08-13 00:52:06.682883705 -0700
+++ /tmp/foob	2015-08-13 00:52:06.682883705 -0700
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -47,7 +47,7 @@
 	 ###################
 	 4.1 MiB/s
 done
-Bytes transferred = 5041872 (4ceed0 hex)
+Bytes transferred = 5044744 (4cfa08 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x4ceed0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x4cfa08 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -69,7 +69,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc6-next-20150812-35595-gc1a0c66 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Aug 12 07:18:08 PDT 2015
+Linux version 4.2.0-rc6-next-20150813-36072-g30b42f4 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Aug 13 00:48:06 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -78,7 +78,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c09b67c0, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c09b87c0, node_mem_map eeff7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -91,7 +91,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 963516K/1048576K available (6876K kernel code, 344K rwdata, 2380K rodata, 316K init, 363K bss, 19524K reserved, 65536K cma-reserved, 204800K highmem)
+Memory: 963508K/1048576K available (6888K kernel code, 344K rwdata, 2380K rodata, 312K init, 363K bss, 19532K reserved, 65536K cma-reserved, 204800K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -99,10 +99,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0912698   (9258 kB)
-      .init : 0xc0913000 - 0xc0962000   ( 316 kB)
-      .data : 0xc0962000 - 0xc09b8340   ( 345 kB)
-       .bss : 0xc09bb000 - 0xc0a15da0   ( 364 kB)
+      .text : 0xc0008000 - 0xc09156a8   (9270 kB)
+      .init : 0xc0916000 - 0xc0964000   ( 312 kB)
+      .data : 0xc0964000 - 0xc09ba340   ( 345 kB)
+       .bss : 0xc09bd000 - 0xc0a17da0   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
@@ -238,57 +238,57 @@
 drm drm: No connectors reported connected with modes
 [drm] Cannot find any crtc or sizes - going 1024x768
 ------------[ cut here ]------------
-WARNING: CPU: 1 PID: 1 at include/linux/ww_mutex.h:158 drm_atomic_check_only+0x4fc/0x574()
+WARNING: CPU: 0 PID: 1 at include/linux/ww_mutex.h:158 drm_atomic_check_only+0x4fc/0x574()
 DEBUG_LOCKS_WARN_ON(ctx->done_acquire)
 Modules linked in:
-CPU: 1 PID: 1 Comm: swapper/0 Not tainted 4.2.0-rc6-next-20150812-35595-gc1a0c66 #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.2.0-rc6-next-20150813-36072-g30b42f4 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c0016898>] (unwind_backtrace) from [<c0012124>] (show_stack+0x10/0x14)
-[<c0012124>] (show_stack) from [<c067cc68>] (dump_stack+0x84/0xd0)
-[<c067cc68>] (dump_stack) from [<c00257a8>] (warn_slowpath_common+0x74/0xb0)
-[<c00257a8>] (warn_slowpath_common) from [<c0025878>] (warn_slowpath_fmt+0x30/0x40)
-[<c0025878>] (warn_slowpath_fmt) from [<c02e3eec>] (drm_atomic_check_only+0x4fc/0x574)
-[<c02e3eec>] (drm_atomic_check_only) from [<c02e3fd8>] (drm_atomic_commit+0x10/0x64)
-[<c02e3fd8>] (drm_atomic_commit) from [<c02c2ac8>] (drm_atomic_helper_set_config+0x394/0x3ec)
-[<c02c2ac8>] (drm_atomic_helper_set_config) from [<c02d4180>] (drm_mode_set_config_internal+0x58/0xd4)
-[<c02d4180>] (drm_mode_set_config_internal) from [<c02c50d8>] (restore_fbdev_mode+0xd4/0xf4)
-[<c02c50d8>] (restore_fbdev_mode) from [<c02c6940>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
-[<c02c6940>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c02c69ac>] (drm_fb_helper_set_par+0x2c/0x54)
-[<c02c69ac>] (drm_fb_helper_set_par) from [<c02632d4>] (fbcon_init+0x474/0x4dc)
-[<c02632d4>] (fbcon_init) from [<c0299e70>] (visual_init+0xb0/0xfc)
-[<c0299e70>] (visual_init) from [<c029be14>] (do_bind_con_driver+0x134/0x354)
-[<c029be14>] (do_bind_con_driver) from [<c029c1a4>] (do_take_over_console+0x170/0x1dc)
-[<c029c1a4>] (do_take_over_console) from [<c02633a0>] (do_fbcon_takeover+0x64/0xcc)
-[<c02633a0>] (do_fbcon_takeover) from [<c003e4ac>] (notifier_call_chain+0x44/0x84)
-[<c003e4ac>] (notifier_call_chain) from [<c003e680>] (__blocking_notifier_call_chain+0x48/0x60)
-[<c003e680>] (__blocking_notifier_call_chain) from [<c003e6b0>] (blocking_notifier_call_chain+0x18/0x20)
-[<c003e6b0>] (blocking_notifier_call_chain) from [<c026d290>] (register_framebuffer+0x184/0x264)
-[<c026d290>] (register_framebuffer) from [<c02c6c18>] (drm_fb_helper_initial_config+0x244/0x3e4)
-[<c02c6c18>] (drm_fb_helper_initial_config) from [<c02e8c70>] (tegra_drm_fb_init+0x48/0xb4)
-[<c02e8c70>] (tegra_drm_fb_init) from [<c02e6f08>] (tegra_drm_load+0x170/0x224)
-[<c02e6f08>] (tegra_drm_load) from [<c02cef0c>] (drm_dev_register+0xa0/0xb4)
-[<c02cef0c>] (drm_dev_register) from [<c02e6a64>] (host1x_drm_probe+0x44/0xb0)
-[<c02e6a64>] (host1x_drm_probe) from [<c02b1e90>] (host1x_device_probe+0x1c/0x28)
-[<c02b1e90>] (host1x_device_probe) from [<c02fd408>] (driver_probe_device+0x1e0/0x2a0)
-[<c02fd408>] (driver_probe_device) from [<c02fbaa8>] (bus_for_each_drv+0x44/0x8c)
-[<c02fbaa8>] (bus_for_each_drv) from [<c02fd19c>] (__device_attach+0xa0/0x104)
-[<c02fd19c>] (__device_attach) from [<c02fc918>] (bus_probe_device+0x84/0x8c)
-[<c02fc918>] (bus_probe_device) from [<c02fae6c>] (device_add+0x33c/0x524)
-[<c02fae6c>] (device_add) from [<c02b209c>] (host1x_subdev_register+0xb0/0xd4)
-[<c02b209c>] (host1x_subdev_register) from [<c02b2954>] (host1x_client_register+0xf4/0x11c)
-[<c02b2954>] (host1x_client_register) from [<c02f6bc0>] (gr3d_probe+0x100/0x21c)
-[<c02f6bc0>] (gr3d_probe) from [<c02fe8e0>] (platform_drv_probe+0x48/0xa4)
-[<c02fe8e0>] (platform_drv_probe) from [<c02fd408>] (driver_probe_device+0x1e0/0x2a0)
-[<c02fd408>] (driver_probe_device) from [<c02fd554>] (__driver_attach+0x8c/0x90)
-[<c02fd554>] (__driver_attach) from [<c02fbb44>] (bus_for_each_dev+0x54/0x88)
-[<c02fbb44>] (bus_for_each_dev) from [<c02fcaf8>] (bus_add_driver+0xe4/0x1f0)
-[<c02fcaf8>] (bus_add_driver) from [<c02fdd20>] (driver_register+0x78/0xf4)
-[<c02fdd20>] (driver_register) from [<c092f704>] (host1x_drm_init+0x9c/0x10c)
-[<c092f704>] (host1x_drm_init) from [<c00096e4>] (do_one_initcall+0x80/0x1d0)
-[<c00096e4>] (do_one_initcall) from [<c0913db4>] (kernel_init_freeable+0x11c/0x1e8)
-[<c0913db4>] (kernel_init_freeable) from [<c0678c88>] (kernel_init+0x8/0xec)
-[<c0678c88>] (kernel_init) from [<c000f628>] (ret_from_fork+0x14/0x2c)
----[ end trace 33a1ce386c1ae2c7 ]---
+[<c0012124>] (show_stack) from [<c067f600>] (dump_stack+0x84/0xd0)
+[<c067f600>] (dump_stack) from [<c00257b8>] (warn_slowpath_common+0x74/0xb0)
+[<c00257b8>] (warn_slowpath_common) from [<c0025888>] (warn_slowpath_fmt+0x30/0x40)
+[<c0025888>] (warn_slowpath_fmt) from [<c02e6758>] (drm_atomic_check_only+0x4fc/0x574)
+[<c02e6758>] (drm_atomic_check_only) from [<c02e6844>] (drm_atomic_commit+0x10/0x64)
+[<c02e6844>] (drm_atomic_commit) from [<c02c531c>] (drm_atomic_helper_set_config+0x394/0x3ec)
+[<c02c531c>] (drm_atomic_helper_set_config) from [<c02d69ec>] (drm_mode_set_config_internal+0x58/0xd4)
+[<c02d69ec>] (drm_mode_set_config_internal) from [<c02c7924>] (restore_fbdev_mode+0xd4/0xf4)
+[<c02c7924>] (restore_fbdev_mode) from [<c02c918c>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
+[<c02c918c>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c02c91f8>] (drm_fb_helper_set_par+0x2c/0x54)
+[<c02c91f8>] (drm_fb_helper_set_par) from [<c0265a10>] (fbcon_init+0x474/0x4dc)
+[<c0265a10>] (fbcon_init) from [<c029c6c8>] (visual_init+0xb0/0xfc)
+[<c029c6c8>] (visual_init) from [<c029e66c>] (do_bind_con_driver+0x134/0x354)
+[<c029e66c>] (do_bind_con_driver) from [<c029e9fc>] (do_take_over_console+0x170/0x1dc)
+[<c029e9fc>] (do_take_over_console) from [<c0265adc>] (do_fbcon_takeover+0x64/0xcc)
+[<c0265adc>] (do_fbcon_takeover) from [<c003e500>] (notifier_call_chain+0x44/0x84)
+[<c003e500>] (notifier_call_chain) from [<c003e6d4>] (__blocking_notifier_call_chain+0x48/0x60)
+[<c003e6d4>] (__blocking_notifier_call_chain) from [<c003e704>] (blocking_notifier_call_chain+0x18/0x20)
+[<c003e704>] (blocking_notifier_call_chain) from [<c026f9cc>] (register_framebuffer+0x184/0x264)
+[<c026f9cc>] (register_framebuffer) from [<c02c9464>] (drm_fb_helper_initial_config+0x244/0x3e4)
+[<c02c9464>] (drm_fb_helper_initial_config) from [<c02eb4dc>] (tegra_drm_fb_init+0x48/0xb4)
+[<c02eb4dc>] (tegra_drm_fb_init) from [<c02e9774>] (tegra_drm_load+0x170/0x224)
+[<c02e9774>] (tegra_drm_load) from [<c02d1778>] (drm_dev_register+0xa0/0xb4)
+[<c02d1778>] (drm_dev_register) from [<c02e92d0>] (host1x_drm_probe+0x44/0xb0)
+[<c02e92d0>] (host1x_drm_probe) from [<c02b46e8>] (host1x_device_probe+0x1c/0x28)
+[<c02b46e8>] (host1x_device_probe) from [<c02ffc7c>] (driver_probe_device+0x1e0/0x2a0)
+[<c02ffc7c>] (driver_probe_device) from [<c02fe31c>] (bus_for_each_drv+0x44/0x8c)
+[<c02fe31c>] (bus_for_each_drv) from [<c02ffa10>] (__device_attach+0xa0/0x104)
+[<c02ffa10>] (__device_attach) from [<c02ff18c>] (bus_probe_device+0x84/0x8c)
+[<c02ff18c>] (bus_probe_device) from [<c02fd6e0>] (device_add+0x33c/0x524)
+[<c02fd6e0>] (device_add) from [<c02b48f4>] (host1x_subdev_register+0xb0/0xd4)
+[<c02b48f4>] (host1x_subdev_register) from [<c02b51ac>] (host1x_client_register+0xf4/0x11c)
+[<c02b51ac>] (host1x_client_register) from [<c02f942c>] (gr3d_probe+0x100/0x21c)
+[<c02f942c>] (gr3d_probe) from [<c0301154>] (platform_drv_probe+0x48/0xa4)
+[<c0301154>] (platform_drv_probe) from [<c02ffc7c>] (driver_probe_device+0x1e0/0x2a0)
+[<c02ffc7c>] (driver_probe_device) from [<c02ffdc8>] (__driver_attach+0x8c/0x90)
+[<c02ffdc8>] (__driver_attach) from [<c02fe3b8>] (bus_for_each_dev+0x54/0x88)
+[<c02fe3b8>] (bus_for_each_dev) from [<c02ff36c>] (bus_add_driver+0xe4/0x1f0)
+[<c02ff36c>] (bus_add_driver) from [<c0300594>] (driver_register+0x78/0xf4)
+[<c0300594>] (driver_register) from [<c09325fc>] (host1x_drm_init+0x9c/0x10c)
+[<c09325fc>] (host1x_drm_init) from [<c00096e4>] (do_one_initcall+0x80/0x1d0)
+[<c00096e4>] (do_one_initcall) from [<c0916db4>] (kernel_init_freeable+0x11c/0x1e8)
+[<c0916db4>] (kernel_init_freeable) from [<c067b620>] (kernel_init+0x8/0xec)
+[<c067b620>] (kernel_init) from [<c000f628>] (ret_from_fork+0x14/0x2c)
+---[ end trace 41ceb974c3ed1d8c ]---
 Console: switching to colour frame buffer device 128x48
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
@@ -366,12 +366,12 @@
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
 tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
 NET: Registered protocol family 10
+mip6: Mobile IPv6
+sit: IPv6 over IPv4 tunneling driver
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
  mmcblk0: p1
-mip6: Mobile IPv6
-sit: IPv6 over IPv4 tunneling driver
 NET: Registered protocol family 17
 NET: Registered protocol family 15
 can: controller area network core (rev 20120528 abi 9)
@@ -389,17 +389,17 @@
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
-tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
+tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
+tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
-tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-26 20:24:52 UTC (1419625492)
+rtc-em3027 2-0056: setting system clock to 2014-12-26 20:25:27 UTC (1419625527)
 ALSA device list:
   #0: tegra-trimslice
 EXT2-fs (mmcblk0p1): error: couldn't mount because of unsupported optional features (4)
@@ -408,7 +408,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 316K (c0913000 - c0962000)
+Freeing unused kernel memory: 312K (c0916000 - c0964000)
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-08-13 00:52:06.690883701 -0700
+++ /tmp/foob	2015-08-13 00:52:06.690883701 -0700
@@ -37,21 +37,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #############################################
+	 ##############################################
 	 3.3 MiB/s
 done
-Bytes transferred = 6383368 (616708 hex)
+Bytes transferred = 6386256 (617250 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.3 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x616708 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x617250 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc6-next-20150812-35595-gc1a0c66 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Aug 12 07:17:44 PDT 2015
+Linux version 4.2.0-rc6-next-20150813-36072-g30b42f4 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Aug 13 00:47:41 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -67,7 +67,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f5a1c0, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0f5e380, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -80,7 +80,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998928K/2096128K available (8018K kernel code, 1016K rwdata, 3604K rodata, 1004K init, 340K bss, 31664K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1998912K/2096128K available (8026K kernel code, 1016K rwdata, 3608K rodata, 1008K init, 340K bss, 31680K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,10 +88,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d62b54   (11627 kB)
-      .init : 0xc0d63000 - 0xc0e5e000   (1004 kB)
-      .data : 0xc0e5e000 - 0xc0f5c180   (1017 kB)
-       .bss : 0xc0f5f000 - 0xc0fb4268   ( 341 kB)
+      .text : 0xc0208000 - 0xc0d65bcc   (11639 kB)
+      .init : 0xc0d66000 - 0xc0e62000   (1008 kB)
+      .data : 0xc0e62000 - 0xc0f60340   (1017 kB)
+       .bss : 0xc0f63000 - 0xc0fb82e8   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -281,18 +281,18 @@
 Key type dns_resolver registered
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
-tegra-pcie 3000.pcie-controller: 2x3 configuration
-sys_3v3,vdd_3v3_alw: supplied by vdd_5v_in
-sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
-tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
 mmc1: new high speed MMC card at address 0001
 mmcblk0: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk0boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk0boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
-tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 mmcblk0rpmb: mmc1:0001 SEM16G partition 3 128 KiB
+tegra-pcie 3000.pcie-controller: 2x3 configuration
+sys_3v3,vdd_3v3_alw: supplied by vdd_5v_in
+sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
 Primary GPT is invalid, using alternate GPT.
+tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
  mmcblk0: p1 p2 p3 p4 p5
+tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -355,15 +355,15 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 tps65910-rtc tps65910-rtc: setting system clock to 2000-01-01 00:00:02 UTC (946684802)
-usb1_vbus: disabling
 vdd_sata,avdd_plle: disabling
+usb1_vbus: disabling
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk0p1): warning: mounting fs with errors, running e2fsck is recommended
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1004K (c0d63000 - c0e5e000)
+Freeing unused kernel memory: 1008K (c0d66000 - c0e62000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.748791] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.748499] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2015-08-13 00:52:06.698883695 -0700
+++ /tmp/foob	2015-08-13 00:52:06.698883695 -0700
@@ -39,18 +39,18 @@
 	 ###################
 	 3.3 MiB/s
 done
-Bytes transferred = 5041872 (4ceed0 hex)
+Bytes transferred = 5044744 (4cfa08 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.5 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4ceed0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4cfa08 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc6-next-20150812-35595-gc1a0c66 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Aug 12 07:18:08 PDT 2015
+Linux version 4.2.0-rc6-next-20150813-36072-g30b42f4 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Aug 13 00:48:06 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c09b67c0, node_mem_map ee7f7000
+free_area_init_node: node 0, pgdat c09b87c0, node_mem_map ee7f7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002628K/2096128K available (6876K kernel code, 344K rwdata, 2380K rodata, 316K init, 363K bss, 27964K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 2002620K/2096128K available (6888K kernel code, 344K rwdata, 2380K rodata, 312K init, 363K bss, 27972K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0912698   (9258 kB)
-      .init : 0xc0913000 - 0xc0962000   ( 316 kB)
-      .data : 0xc0962000 - 0xc09b8340   ( 345 kB)
-       .bss : 0xc09bb000 - 0xc0a15da0   ( 364 kB)
+      .text : 0xc0008000 - 0xc09156a8   (9270 kB)
+      .init : 0xc0916000 - 0xc0964000   ( 312 kB)
+      .data : 0xc0964000 - 0xc09ba340   ( 345 kB)
+       .bss : 0xc09bd000 - 0xc0a17da0   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -335,57 +335,57 @@
 drm drm: No connectors reported connected with modes
 [drm] Cannot find any crtc or sizes - going 1024x768
 ------------[ cut here ]------------
-WARNING: CPU: 1 PID: 82 at include/linux/ww_mutex.h:158 drm_atomic_check_only+0x4fc/0x574()
+WARNING: CPU: 0 PID: 69 at include/linux/ww_mutex.h:158 drm_atomic_check_only+0x4fc/0x574()
 DEBUG_LOCKS_WARN_ON(ctx->done_acquire)
 Modules linked in:
-CPU: 1 PID: 82 Comm: kworker/u8:2 Not tainted 4.2.0-rc6-next-20150812-35595-gc1a0c66 #1
+CPU: 0 PID: 69 Comm: kworker/u8:1 Not tainted 4.2.0-rc6-next-20150813-36072-g30b42f4 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0016898>] (unwind_backtrace) from [<c0012124>] (show_stack+0x10/0x14)
-[<c0012124>] (show_stack) from [<c067cc68>] (dump_stack+0x84/0xd0)
-[<c067cc68>] (dump_stack) from [<c00257a8>] (warn_slowpath_common+0x74/0xb0)
-[<c00257a8>] (warn_slowpath_common) from [<c0025878>] (warn_slowpath_fmt+0x30/0x40)
-[<c0025878>] (warn_slowpath_fmt) from [<c02e3eec>] (drm_atomic_check_only+0x4fc/0x574)
-[<c02e3eec>] (drm_atomic_check_only) from [<c02e3fd8>] (drm_atomic_commit+0x10/0x64)
-[<c02e3fd8>] (drm_atomic_commit) from [<c02c2ac8>] (drm_atomic_helper_set_config+0x394/0x3ec)
-[<c02c2ac8>] (drm_atomic_helper_set_config) from [<c02d4180>] (drm_mode_set_config_internal+0x58/0xd4)
-[<c02d4180>] (drm_mode_set_config_internal) from [<c02c50d8>] (restore_fbdev_mode+0xd4/0xf4)
-[<c02c50d8>] (restore_fbdev_mode) from [<c02c6940>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
-[<c02c6940>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c02c69ac>] (drm_fb_helper_set_par+0x2c/0x54)
-[<c02c69ac>] (drm_fb_helper_set_par) from [<c02632d4>] (fbcon_init+0x474/0x4dc)
-[<c02632d4>] (fbcon_init) from [<c0299e70>] (visual_init+0xb0/0xfc)
-[<c0299e70>] (visual_init) from [<c029be14>] (do_bind_con_driver+0x134/0x354)
-[<c029be14>] (do_bind_con_driver) from [<c029c1a4>] (do_take_over_console+0x170/0x1dc)
-[<c029c1a4>] (do_take_over_console) from [<c02633a0>] (do_fbcon_takeover+0x64/0xcc)
-[<c02633a0>] (do_fbcon_takeover) from [<c003e4ac>] (notifier_call_chain+0x44/0x84)
-[<c003e4ac>] (notifier_call_chain) from [<c003e680>] (__blocking_notifier_call_chain+0x48/0x60)
-[<c003e680>] (__blocking_notifier_call_chain) from [<c003e6b0>] (blocking_notifier_call_chain+0x18/0x20)
-[<c003e6b0>] (blocking_notifier_call_chain) from [<c026d290>] (register_framebuffer+0x184/0x264)
-[<c026d290>] (register_framebuffer) from [<c02c6c18>] (drm_fb_helper_initial_config+0x244/0x3e4)
-[<c02c6c18>] (drm_fb_helper_initial_config) from [<c02e8c70>] (tegra_drm_fb_init+0x48/0xb4)
-[<c02e8c70>] (tegra_drm_fb_init) from [<c02e6f08>] (tegra_drm_load+0x170/0x224)
-[<c02e6f08>] (tegra_drm_load) from [<c02cef0c>] (drm_dev_register+0xa0/0xb4)
-[<c02cef0c>] (drm_dev_register) from [<c02e6a64>] (host1x_drm_probe+0x44/0xb0)
-[<c02e6a64>] (host1x_drm_probe) from [<c02b1e90>] (host1x_device_probe+0x1c/0x28)
-[<c02b1e90>] (host1x_device_probe) from [<c02fd408>] (driver_probe_device+0x1e0/0x2a0)
-[<c02fd408>] (driver_probe_device) from [<c02fbaa8>] (bus_for_each_drv+0x44/0x8c)
-[<c02fbaa8>] (bus_for_each_drv) from [<c02fd19c>] (__device_attach+0xa0/0x104)
-[<c02fd19c>] (__device_attach) from [<c02fc918>] (bus_probe_device+0x84/0x8c)
-[<c02fc918>] (bus_probe_device) from [<c02fae6c>] (device_add+0x33c/0x524)
-[<c02fae6c>] (device_add) from [<c02b209c>] (host1x_subdev_register+0xb0/0xd4)
-[<c02b209c>] (host1x_subdev_register) from [<c02b2954>] (host1x_client_register+0xf4/0x11c)
-[<c02b2954>] (host1x_client_register) from [<c02f016c>] (tegra_hdmi_probe+0x170/0x250)
-[<c02f016c>] (tegra_hdmi_probe) from [<c02fe8e0>] (platform_drv_probe+0x48/0xa4)
-[<c02fe8e0>] (platform_drv_probe) from [<c02fd408>] (driver_probe_device+0x1e0/0x2a0)
-[<c02fd408>] (driver_probe_device) from [<c02fbaa8>] (bus_for_each_drv+0x44/0x8c)
-[<c02fbaa8>] (bus_for_each_drv) from [<c02fd19c>] (__device_attach+0xa0/0x104)
-[<c02fd19c>] (__device_attach) from [<c02fc918>] (bus_probe_device+0x84/0x8c)
-[<c02fc918>] (bus_probe_device) from [<c02fccf0>] (deferred_probe_work_func+0x5c/0x88)
-[<c02fccf0>] (deferred_probe_work_func) from [<c00385b0>] (process_one_work+0x120/0x328)
-[<c00385b0>] (process_one_work) from [<c0038814>] (worker_thread+0x28/0x490)
-[<c0038814>] (worker_thread) from [<c003d90c>] (kthread+0xd8/0xf4)
-[<c003d90c>] (kthread) from [<c000f628>] (ret_from_fork+0x14/0x2c)
----[ end trace babe5256f050980d ]---
+[<c0012124>] (show_stack) from [<c067f600>] (dump_stack+0x84/0xd0)
+[<c067f600>] (dump_stack) from [<c00257b8>] (warn_slowpath_common+0x74/0xb0)
+[<c00257b8>] (warn_slowpath_common) from [<c0025888>] (warn_slowpath_fmt+0x30/0x40)
+[<c0025888>] (warn_slowpath_fmt) from [<c02e6758>] (drm_atomic_check_only+0x4fc/0x574)
+[<c02e6758>] (drm_atomic_check_only) from [<c02e6844>] (drm_atomic_commit+0x10/0x64)
+[<c02e6844>] (drm_atomic_commit) from [<c02c531c>] (drm_atomic_helper_set_config+0x394/0x3ec)
+[<c02c531c>] (drm_atomic_helper_set_config) from [<c02d69ec>] (drm_mode_set_config_internal+0x58/0xd4)
+[<c02d69ec>] (drm_mode_set_config_internal) from [<c02c7924>] (restore_fbdev_mode+0xd4/0xf4)
+[<c02c7924>] (restore_fbdev_mode) from [<c02c918c>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
+[<c02c918c>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c02c91f8>] (drm_fb_helper_set_par+0x2c/0x54)
+[<c02c91f8>] (drm_fb_helper_set_par) from [<c0265a10>] (fbcon_init+0x474/0x4dc)
+[<c0265a10>] (fbcon_init) from [<c029c6c8>] (visual_init+0xb0/0xfc)
+[<c029c6c8>] (visual_init) from [<c029e66c>] (do_bind_con_driver+0x134/0x354)
+[<c029e66c>] (do_bind_con_driver) from [<c029e9fc>] (do_take_over_console+0x170/0x1dc)
+[<c029e9fc>] (do_take_over_console) from [<c0265adc>] (do_fbcon_takeover+0x64/0xcc)
+[<c0265adc>] (do_fbcon_takeover) from [<c003e500>] (notifier_call_chain+0x44/0x84)
+[<c003e500>] (notifier_call_chain) from [<c003e6d4>] (__blocking_notifier_call_chain+0x48/0x60)
+[<c003e6d4>] (__blocking_notifier_call_chain) from [<c003e704>] (blocking_notifier_call_chain+0x18/0x20)
+[<c003e704>] (blocking_notifier_call_chain) from [<c026f9cc>] (register_framebuffer+0x184/0x264)
+[<c026f9cc>] (register_framebuffer) from [<c02c9464>] (drm_fb_helper_initial_config+0x244/0x3e4)
+[<c02c9464>] (drm_fb_helper_initial_config) from [<c02eb4dc>] (tegra_drm_fb_init+0x48/0xb4)
+[<c02eb4dc>] (tegra_drm_fb_init) from [<c02e9774>] (tegra_drm_load+0x170/0x224)
+[<c02e9774>] (tegra_drm_load) from [<c02d1778>] (drm_dev_register+0xa0/0xb4)
+[<c02d1778>] (drm_dev_register) from [<c02e92d0>] (host1x_drm_probe+0x44/0xb0)
+[<c02e92d0>] (host1x_drm_probe) from [<c02b46e8>] (host1x_device_probe+0x1c/0x28)
+[<c02b46e8>] (host1x_device_probe) from [<c02ffc7c>] (driver_probe_device+0x1e0/0x2a0)
+[<c02ffc7c>] (driver_probe_device) from [<c02fe31c>] (bus_for_each_drv+0x44/0x8c)
+[<c02fe31c>] (bus_for_each_drv) from [<c02ffa10>] (__device_attach+0xa0/0x104)
+[<c02ffa10>] (__device_attach) from [<c02ff18c>] (bus_probe_device+0x84/0x8c)
+[<c02ff18c>] (bus_probe_device) from [<c02fd6e0>] (device_add+0x33c/0x524)
+[<c02fd6e0>] (device_add) from [<c02b48f4>] (host1x_subdev_register+0xb0/0xd4)
+[<c02b48f4>] (host1x_subdev_register) from [<c02b51ac>] (host1x_client_register+0xf4/0x11c)
+[<c02b51ac>] (host1x_client_register) from [<c02f29d8>] (tegra_hdmi_probe+0x170/0x250)
+[<c02f29d8>] (tegra_hdmi_probe) from [<c0301154>] (platform_drv_probe+0x48/0xa4)
+[<c0301154>] (platform_drv_probe) from [<c02ffc7c>] (driver_probe_device+0x1e0/0x2a0)
+[<c02ffc7c>] (driver_probe_device) from [<c02fe31c>] (bus_for_each_drv+0x44/0x8c)
+[<c02fe31c>] (bus_for_each_drv) from [<c02ffa10>] (__device_attach+0xa0/0x104)
+[<c02ffa10>] (__device_attach) from [<c02ff18c>] (bus_probe_device+0x84/0x8c)
+[<c02ff18c>] (bus_probe_device) from [<c02ff564>] (deferred_probe_work_func+0x5c/0x88)
+[<c02ff564>] (deferred_probe_work_func) from [<c00385c0>] (process_one_work+0x120/0x328)
+[<c00385c0>] (process_one_work) from [<c0038824>] (worker_thread+0x28/0x490)
+[<c0038824>] (worker_thread) from [<c003d91c>] (kthread+0xd8/0xf4)
+[<c003d91c>] (kthread) from [<c000f628>] (ret_from_fork+0x14/0x2c)
+---[ end trace 63e8dea1285d1537 ]---
 Console: switching to colour frame buffer device 128x48
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
@@ -404,8 +404,8 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 tps65910-rtc tps65910-rtc: setting system clock to 2000-01-01 00:00:03 UTC (946684803)
-usb1_vbus: disabling
 vdd_sata,avdd_plle: disabling
+usb1_vbus: disabling
 ALSA device list:
   #0: NVIDIA Tegra Beaver
 EXT2-fs (mmcblk0p1): error: couldn't mount because of unsupported optional features (4)
@@ -415,7 +415,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 316K (c0913000 - c0962000)
+Freeing unused kernel memory: 312K (c0916000 - c0964000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.826417] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.966310] usb 2-1: new high-speed USB device number 2 using tegra-ehci
