Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-05-22 22:26:20.257686768 -0700
+++ /tmp/foob	2016-05-22 22:26:20.257686768 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-next-20160520+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu May 19 22:42:58 PDT 2016
+Linux version 4.6.0-next-20160523+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun May 22 22:21:13 PDT 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -27,7 +27,7 @@
     .rodata : 0xffff0000087c0000 - 0xffff000008ae0000   (  3200 KB)
       .init : 0xffff000008ae0000 - 0xffff000008bd0000   (   960 KB)
       .data : 0xffff000008bd0000 - 0xffff000008c71e00   (   648 KB)
-       .bss : 0xffff000008c71e00 - 0xffff000008cb2bc0   (   260 KB)
+       .bss : 0xffff000008c71e00 - 0xffff000008cb2b40   (   260 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -98,7 +98,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.648:1): initialized
+audit: type=2000 audit(0.676:1): initialized
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -159,7 +159,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2016-05-20 05:43:31 UTC (1463723011)
+rtc-pl031 9010000.pl031: setting system clock to 2016-05-23 05:21:46 UTC (1463980906)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-05-22 22:26:20.293686783 -0700
+++ /tmp/foob	2016-05-22 22:26:20.297686785 -0700
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 ############################
-	 3.3 MiB/s
+	 3.1 MiB/s
 done
-Bytes transferred = 7079272 (6c0568 hex)
+Bytes transferred = 7078048 (6c00a0 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 34956 (888c hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6c0568 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6c00a0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff88b ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-next-20160520-95196-g89949ab (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu May 19 22:40:28 PDT 2016
+Linux version 4.6.0-next-20160523-95466-g9ef1954 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun May 22 22:19:07 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141bac0, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c141bb00, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -90,9 +90,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc106c540   (14738 kB)
+      .text : 0xc0208000 - 0xc106c574   (14738 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141db20   (1143 kB)
+      .data : 0xc1300000 - 0xc141db60   (1143 kB)
        .bss : 0xc141f000 - 0xc1474ca0   ( 344 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -260,27 +260,27 @@
 ehci-atmel: EHCI Atmel driver
 tegra-ehci: Tegra EHCI driver
 ------------[ cut here ]------------
-WARNING: CPU: 1 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
+WARNING: CPU: 3 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
 Modules linked in:
-CPU: 1 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160520-95196-g89949ab #1
+CPU: 3 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160523-95466-g9ef1954 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c030f9d0>] (unwind_backtrace) from [<c030b0c8>] (show_stack+0x10/0x14)
-[<c030b0c8>] (show_stack) from [<c05864a8>] (dump_stack+0x84/0xa4)
-[<c05864a8>] (dump_stack) from [<c0341b88>] (__warn+0xd4/0x100)
-[<c0341b88>] (__warn) from [<c0341bd4>] (warn_slowpath_null+0x20/0x28)
-[<c0341bd4>] (warn_slowpath_null) from [<c06a0974>] (__of_reset_control_get+0x1bc/0x228)
-[<c06a0974>] (__of_reset_control_get) from [<c0912314>] (tegra_ehci_probe+0x2d0/0x51c)
-[<c0912314>] (tegra_ehci_probe) from [<c07794a4>] (platform_drv_probe+0x50/0xb0)
-[<c07794a4>] (platform_drv_probe) from [<c0777e08>] (driver_probe_device+0x1f4/0x2b0)
-[<c0777e08>] (driver_probe_device) from [<c0777f70>] (__driver_attach+0xac/0xb0)
-[<c0777f70>] (__driver_attach) from [<c077641c>] (bus_for_each_dev+0x54/0x88)
-[<c077641c>] (bus_for_each_dev) from [<c07773fc>] (bus_add_driver+0xe8/0x1f4)
-[<c07773fc>] (bus_add_driver) from [<c077878c>] (driver_register+0x78/0xf4)
-[<c077878c>] (driver_register) from [<c0301e7c>] (do_one_initcall+0x3c/0x16c)
+[<c030f9d0>] (unwind_backtrace) from [<c030b0bc>] (show_stack+0x10/0x14)
+[<c030b0bc>] (show_stack) from [<c0586328>] (dump_stack+0x84/0xa4)
+[<c0586328>] (dump_stack) from [<c0341b7c>] (__warn+0xd4/0x100)
+[<c0341b7c>] (__warn) from [<c0341bc8>] (warn_slowpath_null+0x20/0x28)
+[<c0341bc8>] (warn_slowpath_null) from [<c06a0e7c>] (__of_reset_control_get+0x1bc/0x228)
+[<c06a0e7c>] (__of_reset_control_get) from [<c0912a08>] (tegra_ehci_probe+0x2d0/0x51c)
+[<c0912a08>] (tegra_ehci_probe) from [<c07799ac>] (platform_drv_probe+0x50/0xb0)
+[<c07799ac>] (platform_drv_probe) from [<c0778310>] (driver_probe_device+0x1f4/0x2b0)
+[<c0778310>] (driver_probe_device) from [<c0778478>] (__driver_attach+0xac/0xb0)
+[<c0778478>] (__driver_attach) from [<c0776924>] (bus_for_each_dev+0x54/0x88)
+[<c0776924>] (bus_for_each_dev) from [<c0777904>] (bus_add_driver+0xe8/0x1f4)
+[<c0777904>] (bus_add_driver) from [<c0778c94>] (driver_register+0x78/0xf4)
+[<c0778c94>] (driver_register) from [<c0301e7c>] (do_one_initcall+0x3c/0x16c)
 [<c0301e7c>] (do_one_initcall) from [<c1100d80>] (kernel_init_freeable+0x120/0x1ec)
-[<c1100d80>] (kernel_init_freeable) from [<c0b5b960>] (kernel_init+0x8/0x118)
-[<c0b5b960>] (kernel_init) from [<c0307fb8>] (ret_from_fork+0x14/0x3c)
----[ end trace e773221ad97da0f5 ]---
+[<c1100d80>] (kernel_init_freeable) from [<c0b5c0b0>] (kernel_init+0x8/0x118)
+[<c0b5c0b0>] (kernel_init) from [<c0307fb8>] (ret_from_fork+0x14/0x3c)
+---[ end trace 71a6800edd50c2c2 ]---
 tegra-ehci 7d008000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d008000.usb: continuing, but please update your DT
 ohci_hcd: USB 1.1 'Open' Host Controller (OHCI) Driver
@@ -329,9 +329,9 @@
 [drm] No driver support for vblank timestamp query.
 mmc0: new high speed SDHC card at address aaaa
 mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
+ mmcblk0: p1 p2 p3
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
- mmcblk0: p1 p2 p3
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
@@ -346,7 +346,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-03-13 17:00:05 UTC (952966805)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-03-16 16:38:53 UTC (953224733)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -358,4 +358,4 @@
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.545139] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.525371] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-05-22 22:26:20.301686787 -0700
+++ /tmp/foob	2016-05-22 22:26:20.305686789 -0700
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra114 (Dalmore) # 
 Tegra114 (Dalmore) # usb reset
 (Re)start USB...
@@ -39,18 +39,18 @@
 	 ##########################################
 	 3.2 MiB/s
 done
-Bytes transferred = 5377272 (520cf8 hex)
+Bytes transferred = 5377944 (520f98 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.6 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 34956 (888c hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x520cf8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x520f98 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff88b ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-next-20160520-95196-g89949ab (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu May 19 22:40:40 PDT 2016
+Linux version 4.6.0-next-20160523-95466-g9ef1954 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun May 22 22:19:07 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0c5bd40, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0c5bd80, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -90,9 +90,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0abe014   (10969 kB)
+      .text : 0xc0008000 - 0xc0abe058   (10969 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
-      .data : 0xc0c00000 - 0xc0c5d8c0   ( 375 kB)
+      .data : 0xc0c00000 - 0xc0c5d900   ( 375 kB)
        .bss : 0xc0c5f000 - 0xc0cbab18   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -252,25 +252,25 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1b4/0x218
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160520-95196-g89949ab #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160523-95466-g9ef1954 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c010f1e8>] (unwind_backtrace) from [<c010a864>] (show_stack+0x10/0x14)
-[<c010a864>] (show_stack) from [<c033dd54>] (dump_stack+0x84/0xa4)
-[<c033dd54>] (dump_stack) from [<c011f320>] (__warn+0xd4/0x100)
-[<c011f320>] (__warn) from [<c011f36c>] (warn_slowpath_null+0x20/0x28)
-[<c011f36c>] (warn_slowpath_null) from [<c03c129c>] (__of_reset_control_get+0x1b4/0x218)
-[<c03c129c>] (__of_reset_control_get) from [<c0513d84>] (tegra_ehci_probe+0x2c8/0x514)
-[<c0513d84>] (tegra_ehci_probe) from [<c044165c>] (platform_drv_probe+0x50/0xb0)
-[<c044165c>] (platform_drv_probe) from [<c0440088>] (driver_probe_device+0x1f4/0x2b0)
-[<c0440088>] (driver_probe_device) from [<c04401f0>] (__driver_attach+0xac/0xb0)
-[<c04401f0>] (__driver_attach) from [<c043e6ac>] (bus_for_each_dev+0x54/0x88)
-[<c043e6ac>] (bus_for_each_dev) from [<c043f67c>] (bus_add_driver+0xe8/0x1f4)
-[<c043f67c>] (bus_add_driver) from [<c04409ec>] (driver_register+0x78/0xf4)
-[<c04409ec>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
+[<c010f1a8>] (unwind_backtrace) from [<c010a858>] (show_stack+0x10/0x14)
+[<c010a858>] (show_stack) from [<c033db74>] (dump_stack+0x84/0xa4)
+[<c033db74>] (dump_stack) from [<c011f2d4>] (__warn+0xd4/0x100)
+[<c011f2d4>] (__warn) from [<c011f320>] (warn_slowpath_null+0x20/0x28)
+[<c011f320>] (warn_slowpath_null) from [<c03c10c4>] (__of_reset_control_get+0x1b4/0x218)
+[<c03c10c4>] (__of_reset_control_get) from [<c0513d94>] (tegra_ehci_probe+0x2c8/0x514)
+[<c0513d94>] (tegra_ehci_probe) from [<c0441488>] (platform_drv_probe+0x50/0xb0)
+[<c0441488>] (platform_drv_probe) from [<c043feb4>] (driver_probe_device+0x1f4/0x2b0)
+[<c043feb4>] (driver_probe_device) from [<c044001c>] (__driver_attach+0xac/0xb0)
+[<c044001c>] (__driver_attach) from [<c043e4d8>] (bus_for_each_dev+0x54/0x88)
+[<c043e4d8>] (bus_for_each_dev) from [<c043f4a8>] (bus_add_driver+0xe8/0x1f4)
+[<c043f4a8>] (bus_add_driver) from [<c0440818>] (driver_register+0x78/0xf4)
+[<c0440818>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
 [<c010179c>] (do_one_initcall) from [<c0b00d94>] (kernel_init_freeable+0x11c/0x1e8)
-[<c0b00d94>] (kernel_init_freeable) from [<c07f1510>] (kernel_init+0x8/0x118)
-[<c07f1510>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
----[ end trace 88b8fbe7de456e80 ]---
+[<c0b00d94>] (kernel_init_freeable) from [<c07f1984>] (kernel_init+0x8/0x118)
+[<c07f1984>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
+---[ end trace 4238f2f5b163fe5a ]---
 tegra-ehci 7d008000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d008000.usb: continuing, but please update your DT
 usbcore: registered new interface driver cdc_acm
@@ -302,11 +302,11 @@
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
+mmc0: new high speed SDHC card at address aaaa
+mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
  mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
-mmc0: new high speed SDHC card at address aaaa
-mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
  mmcblk0: p1 p2 p3
 tegra30-i2s 70080400.i2s: DMA channels sourced from device 70080000.ahub
 tegra-snd-rt5640 sound: rt5640-aif1 <-> 70080400.i2s mapping ok
@@ -344,7 +344,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-03-13 17:02:31 UTC (952966951)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-03-16 16:41:26 UTC (953224886)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-05-22 22:26:20.313686792 -0700
+++ /tmp/foob	2016-05-22 22:26:20.313686792 -0700
@@ -36,7 +36,7 @@
 	 ############################
 	 4 MiB/s
 done
-Bytes transferred = 7079272 (6c0568 hex)
+Bytes transferred = 7078048 (6c00a0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -47,7 +47,7 @@
 done
 Bytes transferred = 66737 (104b1 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6c0568 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x6c00a0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830134b0
@@ -55,7 +55,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-next-20160520-95196-g89949ab (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu May 19 22:40:28 PDT 2016
+Linux version 4.6.0-next-20160523-95466-g9ef1954 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun May 22 22:19:07 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -64,7 +64,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141bac0, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c141bb00, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -85,9 +85,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc106c540   (14738 kB)
+      .text : 0xc0208000 - 0xc106c574   (14738 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141db20   (1143 kB)
+      .data : 0xc1300000 - 0xc141db60   (1143 kB)
        .bss : 0xc141f000 - 0xc1474ca0   ( 344 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -254,25 +254,25 @@
 ------------[ cut here ]------------
 WARNING: CPU: 1 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
 Modules linked in:
-CPU: 1 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160520-95196-g89949ab #1
+CPU: 1 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160523-95466-g9ef1954 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c030f9d0>] (unwind_backtrace) from [<c030b0c8>] (show_stack+0x10/0x14)
-[<c030b0c8>] (show_stack) from [<c05864a8>] (dump_stack+0x84/0xa4)
-[<c05864a8>] (dump_stack) from [<c0341b88>] (__warn+0xd4/0x100)
-[<c0341b88>] (__warn) from [<c0341bd4>] (warn_slowpath_null+0x20/0x28)
-[<c0341bd4>] (warn_slowpath_null) from [<c06a0974>] (__of_reset_control_get+0x1bc/0x228)
-[<c06a0974>] (__of_reset_control_get) from [<c0912314>] (tegra_ehci_probe+0x2d0/0x51c)
-[<c0912314>] (tegra_ehci_probe) from [<c07794a4>] (platform_drv_probe+0x50/0xb0)
-[<c07794a4>] (platform_drv_probe) from [<c0777e08>] (driver_probe_device+0x1f4/0x2b0)
-[<c0777e08>] (driver_probe_device) from [<c0777f70>] (__driver_attach+0xac/0xb0)
-[<c0777f70>] (__driver_attach) from [<c077641c>] (bus_for_each_dev+0x54/0x88)
-[<c077641c>] (bus_for_each_dev) from [<c07773fc>] (bus_add_driver+0xe8/0x1f4)
-[<c07773fc>] (bus_add_driver) from [<c077878c>] (driver_register+0x78/0xf4)
-[<c077878c>] (driver_register) from [<c0301e7c>] (do_one_initcall+0x3c/0x16c)
+[<c030f9d0>] (unwind_backtrace) from [<c030b0bc>] (show_stack+0x10/0x14)
+[<c030b0bc>] (show_stack) from [<c0586328>] (dump_stack+0x84/0xa4)
+[<c0586328>] (dump_stack) from [<c0341b7c>] (__warn+0xd4/0x100)
+[<c0341b7c>] (__warn) from [<c0341bc8>] (warn_slowpath_null+0x20/0x28)
+[<c0341bc8>] (warn_slowpath_null) from [<c06a0e7c>] (__of_reset_control_get+0x1bc/0x228)
+[<c06a0e7c>] (__of_reset_control_get) from [<c0912a08>] (tegra_ehci_probe+0x2d0/0x51c)
+[<c0912a08>] (tegra_ehci_probe) from [<c07799ac>] (platform_drv_probe+0x50/0xb0)
+[<c07799ac>] (platform_drv_probe) from [<c0778310>] (driver_probe_device+0x1f4/0x2b0)
+[<c0778310>] (driver_probe_device) from [<c0778478>] (__driver_attach+0xac/0xb0)
+[<c0778478>] (__driver_attach) from [<c0776924>] (bus_for_each_dev+0x54/0x88)
+[<c0776924>] (bus_for_each_dev) from [<c0777904>] (bus_add_driver+0xe8/0x1f4)
+[<c0777904>] (bus_add_driver) from [<c0778c94>] (driver_register+0x78/0xf4)
+[<c0778c94>] (driver_register) from [<c0301e7c>] (do_one_initcall+0x3c/0x16c)
 [<c0301e7c>] (do_one_initcall) from [<c1100d80>] (kernel_init_freeable+0x120/0x1ec)
-[<c1100d80>] (kernel_init_freeable) from [<c0b5b960>] (kernel_init+0x8/0x118)
-[<c0b5b960>] (kernel_init) from [<c0307fb8>] (ret_from_fork+0x14/0x3c)
----[ end trace 10600d3acc926463 ]---
+[<c1100d80>] (kernel_init_freeable) from [<c0b5c0b0>] (kernel_init+0x8/0x118)
+[<c0b5c0b0>] (kernel_init) from [<c0307fb8>] (ret_from_fork+0x14/0x3c)
+---[ end trace 28c3fdb0e4b18381 ]---
 tegra-ehci 7d004000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d004000.usb: continuing, but please update your DT
 tegra-ehci 7d004000.usb: EHCI Host Controller
@@ -330,13 +330,13 @@
 +5V_SATA: supplied by +5V_SYS
 +12V_SATA: supplied by +VDD_MUX
 usb2-0 supply vbus not found, using dummy regulator
+usb2-1 supply vbus not found, using dummy regulator
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
  mmcblk0: p1
-usb2-1 supply vbus not found, using dummy regulator
 usb3-0 supply vbus not found, using dummy regulator
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
 tegra-pcie 1003000.pcie-controller: Failed to get supply 'avddio-pex': -517
@@ -402,11 +402,11 @@
 pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
 pci 0000:02:00.0: [10ec:8168] type 00 class 0x020000
 pci 0000:02:00.0: reg 0x10: [io  0x0000-0x00ff]
+ata1: SATA link down (SStatus 0 SControl 300)
 pci 0000:02:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit]
 pci 0000:02:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
 pci 0000:02:00.0: supports D1 D2
 pci 0000:02:00.0: PME# supported from D0 D1 D2 D3hot D3cold
-ata1: SATA link down (SStatus 0 SControl 300)
 PCI: bus2: Fast back to back transfers disabled
 pci_bus 0000:02: busn_res: [bus 02-ff] end is updated to 02
 pci 0000:00:01.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-05-22 22:26:20.317686794 -0700
+++ /tmp/foob	2016-05-22 22:26:20.321686795 -0700
@@ -32,20 +32,20 @@
 	 #################################################################
 	 #################################################################
 	 ##########################################
-	 4 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 5377272 (520cf8 hex)
+Bytes transferred = 5377944 (520f98 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.7 MiB/s
+	 3.5 MiB/s
 done
 Bytes transferred = 66737 (104b1 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x520cf8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x520f98 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830134b0
@@ -54,7 +54,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-next-20160520-95196-g89949ab (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu May 19 22:40:40 PDT 2016
+Linux version 4.6.0-next-20160523-95466-g9ef1954 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun May 22 22:19:07 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -64,7 +64,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0c5bd40, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0c5bd80, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -85,9 +85,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0abe014   (10969 kB)
+      .text : 0xc0008000 - 0xc0abe058   (10969 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
-      .data : 0xc0c00000 - 0xc0c5d8c0   ( 375 kB)
+      .data : 0xc0c00000 - 0xc0c5d900   ( 375 kB)
        .bss : 0xc0c5f000 - 0xc0cbab18   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -244,25 +244,25 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1b4/0x218
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160520-95196-g89949ab #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160523-95466-g9ef1954 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c010f1e8>] (unwind_backtrace) from [<c010a864>] (show_stack+0x10/0x14)
-[<c010a864>] (show_stack) from [<c033dd54>] (dump_stack+0x84/0xa4)
-[<c033dd54>] (dump_stack) from [<c011f320>] (__warn+0xd4/0x100)
-[<c011f320>] (__warn) from [<c011f36c>] (warn_slowpath_null+0x20/0x28)
-[<c011f36c>] (warn_slowpath_null) from [<c03c129c>] (__of_reset_control_get+0x1b4/0x218)
-[<c03c129c>] (__of_reset_control_get) from [<c0513d84>] (tegra_ehci_probe+0x2c8/0x514)
-[<c0513d84>] (tegra_ehci_probe) from [<c044165c>] (platform_drv_probe+0x50/0xb0)
-[<c044165c>] (platform_drv_probe) from [<c0440088>] (driver_probe_device+0x1f4/0x2b0)
-[<c0440088>] (driver_probe_device) from [<c04401f0>] (__driver_attach+0xac/0xb0)
-[<c04401f0>] (__driver_attach) from [<c043e6ac>] (bus_for_each_dev+0x54/0x88)
-[<c043e6ac>] (bus_for_each_dev) from [<c043f67c>] (bus_add_driver+0xe8/0x1f4)
-[<c043f67c>] (bus_add_driver) from [<c04409ec>] (driver_register+0x78/0xf4)
-[<c04409ec>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
+[<c010f1a8>] (unwind_backtrace) from [<c010a858>] (show_stack+0x10/0x14)
+[<c010a858>] (show_stack) from [<c033db74>] (dump_stack+0x84/0xa4)
+[<c033db74>] (dump_stack) from [<c011f2d4>] (__warn+0xd4/0x100)
+[<c011f2d4>] (__warn) from [<c011f320>] (warn_slowpath_null+0x20/0x28)
+[<c011f320>] (warn_slowpath_null) from [<c03c10c4>] (__of_reset_control_get+0x1b4/0x218)
+[<c03c10c4>] (__of_reset_control_get) from [<c0513d94>] (tegra_ehci_probe+0x2c8/0x514)
+[<c0513d94>] (tegra_ehci_probe) from [<c0441488>] (platform_drv_probe+0x50/0xb0)
+[<c0441488>] (platform_drv_probe) from [<c043feb4>] (driver_probe_device+0x1f4/0x2b0)
+[<c043feb4>] (driver_probe_device) from [<c044001c>] (__driver_attach+0xac/0xb0)
+[<c044001c>] (__driver_attach) from [<c043e4d8>] (bus_for_each_dev+0x54/0x88)
+[<c043e4d8>] (bus_for_each_dev) from [<c043f4a8>] (bus_add_driver+0xe8/0x1f4)
+[<c043f4a8>] (bus_add_driver) from [<c0440818>] (driver_register+0x78/0xf4)
+[<c0440818>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
 [<c010179c>] (do_one_initcall) from [<c0b00d94>] (kernel_init_freeable+0x11c/0x1e8)
-[<c0b00d94>] (kernel_init_freeable) from [<c07f1510>] (kernel_init+0x8/0x118)
-[<c07f1510>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
----[ end trace e9805e14fa1c73b3 ]---
+[<c0b00d94>] (kernel_init_freeable) from [<c07f1984>] (kernel_init+0x8/0x118)
+[<c07f1984>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
+---[ end trace 743418de6e44ad05 ]---
 tegra-ehci 7d004000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d004000.usb: continuing, but please update your DT
 tegra-ehci 7d004000.usb: EHCI Host Controller
@@ -460,7 +460,7 @@
 cpu cpu2: cpufreq_init: failed to get clk: -2
 cpu cpu3: cpufreq_init: failed to get clk: -2
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:05 UTC (946684925)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:01:37 UTC (946684897)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2016-05-22 22:26:20.325686797 -0700
+++ /tmp/foob	2016-05-22 22:26:20.329686799 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1800487 us
+Bootloader-Cpu Init at (time stamp): 1800867 us
 
 Pinmux changes applied in kernel way
 
@@ -62,22 +62,22 @@
 Board Id Read Time: 4 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1800460  us
-Bootloader (AosInit) time: 733520 us
-Bootloader Main Init time: 180156 us
-BlockDevMgrInit time: 157559 us
+Miscellaneous avp time: 1800840  us
+Bootloader (AosInit) time: 734227 us
+Bootloader Main Init time: 183858 us
+BlockDevMgrInit time: 157610 us
 PartMgrInit time: 8 us
-StorMgrInit time: 31 us
+StorMgrInit time: 30 us
 FsMgrInit time: 2 us
-Partition Table load: 77088 us 
-Display init time: 574387 us
-Kernel read time: 344 msec
+Partition Table load: 77208 us 
+Display init time: 574350 us
+Kernel read time: 345 msec
 Load Kernel Size:   14952448 Bytes
-Bl Kernel ReadSpeed: 43466000 Bytes/sec
-Pre-Kernel Jump time: 1633001 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5694093 us
+Bl Kernel ReadSpeed: 43340000 Bytes/sec
+Pre-Kernel Jump time: 1633858 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5698590 us
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-next-20160520+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu May 19 22:42:58 PDT 2016
+Linux version 4.6.0-next-20160523+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun May 22 22:21:13 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
 Memory limited to 1734MB
 efi: Getting EFI parameters from FDT:
@@ -100,7 +100,7 @@
     .rodata : 0xffff0000087c0000 - 0xffff000008ae0000   (  3200 KB)
       .init : 0xffff000008ae0000 - 0xffff000008bd0000   (   960 KB)
       .data : 0xffff000008bd0000 - 0xffff000008c71e00   (   648 KB)
-       .bss : 0xffff000008c71e00 - 0xffff000008cb2bc0   (   260 KB)
+       .bss : 0xffff000008c71e00 - 0xffff000008cb2b40   (   260 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -185,7 +185,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(1.079:1): initialized
+audit: type=2000 audit(1.091:1): initialized
 workingset: timestamp_bits=44 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -242,9 +242,11 @@
 NET: Registered protocol family 17
 9pnet: Installing 9P2000 support
 Key type dns_resolver registered
+mmc0: new high speed MMC card at address 0001
 registered taskstats version 1
 +VDD_LED: supplied by +VDD_MUX
-mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 +5V_USB_HS: supplied by +5V_SYS
 +5V_USB_SS: supplied by +5V_SYS
 +5V_HDMI_CON: supplied by +5V_SYS
@@ -265,8 +267,6 @@
 ALSA device list:
   No soundcards found.
 Freeing unused kernel memory: 960K (ffff800000ae0000 - ffff800000bd0000)
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-05-22 22:26:20.333686800 -0700
+++ /tmp/foob	2016-05-22 22:26:20.337686802 -0700
@@ -49,18 +49,18 @@
 	 ############################
 	 4.1 MiB/s
 done
-Bytes transferred = 7079272 (6c0568 hex)
+Bytes transferred = 7078048 (6c00a0 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 4 MiB/s
 done
 Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x6c0568 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x6c00a0 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102b3
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-next-20160520-95196-g89949ab (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu May 19 22:40:28 PDT 2016
+Linux version 4.6.0-next-20160523-95466-g9ef1954 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun May 22 22:19:07 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -76,7 +76,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c141bac0, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c141bb00, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -97,9 +97,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc106c540   (14738 kB)
+      .text : 0xc0208000 - 0xc106c574   (14738 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141db20   (1143 kB)
+      .data : 0xc1300000 - 0xc141db60   (1143 kB)
        .bss : 0xc141f000 - 0xc1474ca0   ( 344 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
@@ -273,25 +273,25 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160520-95196-g89949ab #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160523-95466-g9ef1954 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c030f9d0>] (unwind_backtrace) from [<c030b0c8>] (show_stack+0x10/0x14)
-[<c030b0c8>] (show_stack) from [<c05864a8>] (dump_stack+0x84/0xa4)
-[<c05864a8>] (dump_stack) from [<c0341b88>] (__warn+0xd4/0x100)
-[<c0341b88>] (__warn) from [<c0341bd4>] (warn_slowpath_null+0x20/0x28)
-[<c0341bd4>] (warn_slowpath_null) from [<c06a0974>] (__of_reset_control_get+0x1bc/0x228)
-[<c06a0974>] (__of_reset_control_get) from [<c0912314>] (tegra_ehci_probe+0x2d0/0x51c)
-[<c0912314>] (tegra_ehci_probe) from [<c07794a4>] (platform_drv_probe+0x50/0xb0)
-[<c07794a4>] (platform_drv_probe) from [<c0777e08>] (driver_probe_device+0x1f4/0x2b0)
-[<c0777e08>] (driver_probe_device) from [<c0777f70>] (__driver_attach+0xac/0xb0)
-[<c0777f70>] (__driver_attach) from [<c077641c>] (bus_for_each_dev+0x54/0x88)
-[<c077641c>] (bus_for_each_dev) from [<c07773fc>] (bus_add_driver+0xe8/0x1f4)
-[<c07773fc>] (bus_add_driver) from [<c077878c>] (driver_register+0x78/0xf4)
-[<c077878c>] (driver_register) from [<c0301e7c>] (do_one_initcall+0x3c/0x16c)
+[<c030f9d0>] (unwind_backtrace) from [<c030b0bc>] (show_stack+0x10/0x14)
+[<c030b0bc>] (show_stack) from [<c0586328>] (dump_stack+0x84/0xa4)
+[<c0586328>] (dump_stack) from [<c0341b7c>] (__warn+0xd4/0x100)
+[<c0341b7c>] (__warn) from [<c0341bc8>] (warn_slowpath_null+0x20/0x28)
+[<c0341bc8>] (warn_slowpath_null) from [<c06a0e7c>] (__of_reset_control_get+0x1bc/0x228)
+[<c06a0e7c>] (__of_reset_control_get) from [<c0912a08>] (tegra_ehci_probe+0x2d0/0x51c)
+[<c0912a08>] (tegra_ehci_probe) from [<c07799ac>] (platform_drv_probe+0x50/0xb0)
+[<c07799ac>] (platform_drv_probe) from [<c0778310>] (driver_probe_device+0x1f4/0x2b0)
+[<c0778310>] (driver_probe_device) from [<c0778478>] (__driver_attach+0xac/0xb0)
+[<c0778478>] (__driver_attach) from [<c0776924>] (bus_for_each_dev+0x54/0x88)
+[<c0776924>] (bus_for_each_dev) from [<c0777904>] (bus_add_driver+0xe8/0x1f4)
+[<c0777904>] (bus_add_driver) from [<c0778c94>] (driver_register+0x78/0xf4)
+[<c0778c94>] (driver_register) from [<c0301e7c>] (do_one_initcall+0x3c/0x16c)
 [<c0301e7c>] (do_one_initcall) from [<c1100d80>] (kernel_init_freeable+0x120/0x1ec)
-[<c1100d80>] (kernel_init_freeable) from [<c0b5b960>] (kernel_init+0x8/0x118)
-[<c0b5b960>] (kernel_init) from [<c0307fb8>] (ret_from_fork+0x14/0x3c)
----[ end trace d62cf2f56eeea8c1 ]---
+[<c1100d80>] (kernel_init_freeable) from [<c0b5c0b0>] (kernel_init+0x8/0x118)
+[<c0b5c0b0>] (kernel_init) from [<c0307fb8>] (ret_from_fork+0x14/0x3c)
+---[ end trace bbaf02479b8803d0 ]---
 tegra-ehci c5000000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci c5000000.usb: continuing, but please update your DT
 ulpi_phy_power_on: ulpi write failed
@@ -345,27 +345,28 @@
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
-mmc0: host does not support reading read-only switch, assuming write-enable
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
-mmc0: new high speed SDHC card at address b368
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
- mmcblk0: p1
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
 rtc-em3027 2-0056: hctosys: unable to read the hardware clock
-EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
+Waiting for root device PARTUUID=11eb23c1-01...
+mmc0: host does not support reading read-only switch, assuming write-enable
+mmc0: new high speed SDHC card at address b368
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
+ mmcblk0: p1
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
-usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
+EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
+usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-05-22 22:26:20.341686804 -0700
+++ /tmp/foob	2016-05-22 22:26:20.345686806 -0700
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -45,9 +45,9 @@
 	 #################################################################
 	 #################################################################
 	 ##########################################
-	 912.1 KiB/s
+	 906.3 KiB/s
 done
-Bytes transferred = 5377272 (520cf8 hex)
+Bytes transferred = 5377944 (520f98 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x520cf8 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x520f98 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102b3
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-next-20160520-95196-g89949ab (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu May 19 22:40:40 PDT 2016
+Linux version 4.6.0-next-20160523-95466-g9ef1954 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun May 22 22:19:07 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -76,7 +76,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0c5bd40, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0c5bd80, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -97,9 +97,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0abe014   (10969 kB)
+      .text : 0xc0008000 - 0xc0abe058   (10969 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
-      .data : 0xc0c00000 - 0xc0c5d8c0   ( 375 kB)
+      .data : 0xc0c00000 - 0xc0c5d900   ( 375 kB)
        .bss : 0xc0c5f000 - 0xc0cbab18   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -265,25 +265,25 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1b4/0x218
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160520-95196-g89949ab #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160523-95466-g9ef1954 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c010f1e8>] (unwind_backtrace) from [<c010a864>] (show_stack+0x10/0x14)
-[<c010a864>] (show_stack) from [<c033dd54>] (dump_stack+0x84/0xa4)
-[<c033dd54>] (dump_stack) from [<c011f320>] (__warn+0xd4/0x100)
-[<c011f320>] (__warn) from [<c011f36c>] (warn_slowpath_null+0x20/0x28)
-[<c011f36c>] (warn_slowpath_null) from [<c03c129c>] (__of_reset_control_get+0x1b4/0x218)
-[<c03c129c>] (__of_reset_control_get) from [<c0513d84>] (tegra_ehci_probe+0x2c8/0x514)
-[<c0513d84>] (tegra_ehci_probe) from [<c044165c>] (platform_drv_probe+0x50/0xb0)
-[<c044165c>] (platform_drv_probe) from [<c0440088>] (driver_probe_device+0x1f4/0x2b0)
-[<c0440088>] (driver_probe_device) from [<c04401f0>] (__driver_attach+0xac/0xb0)
-[<c04401f0>] (__driver_attach) from [<c043e6ac>] (bus_for_each_dev+0x54/0x88)
-[<c043e6ac>] (bus_for_each_dev) from [<c043f67c>] (bus_add_driver+0xe8/0x1f4)
-[<c043f67c>] (bus_add_driver) from [<c04409ec>] (driver_register+0x78/0xf4)
-[<c04409ec>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
+[<c010f1a8>] (unwind_backtrace) from [<c010a858>] (show_stack+0x10/0x14)
+[<c010a858>] (show_stack) from [<c033db74>] (dump_stack+0x84/0xa4)
+[<c033db74>] (dump_stack) from [<c011f2d4>] (__warn+0xd4/0x100)
+[<c011f2d4>] (__warn) from [<c011f320>] (warn_slowpath_null+0x20/0x28)
+[<c011f320>] (warn_slowpath_null) from [<c03c10c4>] (__of_reset_control_get+0x1b4/0x218)
+[<c03c10c4>] (__of_reset_control_get) from [<c0513d94>] (tegra_ehci_probe+0x2c8/0x514)
+[<c0513d94>] (tegra_ehci_probe) from [<c0441488>] (platform_drv_probe+0x50/0xb0)
+[<c0441488>] (platform_drv_probe) from [<c043feb4>] (driver_probe_device+0x1f4/0x2b0)
+[<c043feb4>] (driver_probe_device) from [<c044001c>] (__driver_attach+0xac/0xb0)
+[<c044001c>] (__driver_attach) from [<c043e4d8>] (bus_for_each_dev+0x54/0x88)
+[<c043e4d8>] (bus_for_each_dev) from [<c043f4a8>] (bus_add_driver+0xe8/0x1f4)
+[<c043f4a8>] (bus_add_driver) from [<c0440818>] (driver_register+0x78/0xf4)
+[<c0440818>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
 [<c010179c>] (do_one_initcall) from [<c0b00d94>] (kernel_init_freeable+0x11c/0x1e8)
-[<c0b00d94>] (kernel_init_freeable) from [<c07f1510>] (kernel_init+0x8/0x118)
-[<c07f1510>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
----[ end trace a478de30e818b26f ]---
+[<c0b00d94>] (kernel_init_freeable) from [<c07f1984>] (kernel_init+0x8/0x118)
+[<c07f1984>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
+---[ end trace c1348525b3e8ed7f ]---
 tegra-ehci c5000000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci c5000000.usb: continuing, but please update your DT
 ulpi_phy_power_on: ulpi write failed
@@ -331,10 +331,10 @@
 mmc0: host does not support reading read-only switch, assuming write-enable
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
 mmc0: new high speed SDHC card at address b368
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
- mmcblk0: p1
 mmc1: new high speed SD card at address b368
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
+ mmcblk0: p1
  mmcblk1: p1 p2
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
@@ -348,12 +348,12 @@
 can: raw protocol (rev 20120528)
 can: broadcast manager protocol (rev 20120528 t)
 can: netlink gateway (rev 20130117) max_hops=1
-usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 Bluetooth: RFCOMM socket layer initialized
 Bluetooth: RFCOMM ver 1.11
 Bluetooth: BNEP (Ethernet Emulation) ver 1.3
 Bluetooth: BNEP socket layer initialized
 Bluetooth: HIDP (Human Interface Emulation) ver 1.2
+usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 Bluetooth: HIDP socket layer initialized
 Key type dns_resolver registered
 Registering SWP/SWPB emulation handler
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2016-05-22 22:26:20.349686808 -0700
+++ /tmp/foob	2016-05-22 22:26:20.353686810 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 739556 us
-Basic modules init took 350226 us
+Tegraboot started after 746902 us
+Basic modules init took 348556 us
 Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14368 us
-Carveout took 252716 us
-CPU initialization took 92313 us
-Total time taken by TegraBoot 709623 us
+Next stage binary read took 14395 us
+Carveout took 252724 us
+CPU initialization took 92328 us
+Total time taken by TegraBoot 708003 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.580] LPDDR4 Training: Number of tables = 10
+64b[0001.585] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,9 +117,9 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-12525056 bytes read in 746 ms (16 MiB/s)
+12525056 bytes read in 744 ms (16.1 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-37108 bytes read in 453 ms (79.1 KiB/s)
+37108 bytes read in 454 ms (79.1 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
 1627787 bytes read in 495 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-next-20160520+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu May 19 22:42:58 PDT 2016
+Linux version 4.6.0-next-20160523+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun May 22 22:21:13 PDT 2016
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -174,7 +174,7 @@
     .rodata : 0xffff0000087c0000 - 0xffff000008ae0000   (  3200 KB)
       .init : 0xffff000008ae0000 - 0xffff000008bd0000   (   960 KB)
       .data : 0xffff000008bd0000 - 0xffff000008c71e00   (   648 KB)
-       .bss : 0xffff000008c71e00 - 0xffff000008cb2bc0   (   260 KB)
+       .bss : 0xffff000008c71e00 - 0xffff000008cb2b40   (   260 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -259,11 +259,12 @@
 PCI: CLS 0 bytes, default 128
 kvm [1]: 8-bit VMID
 kvm [1]: Hyp mode initialized successfully
-kvm [1]: GICH base=0x50044000, GICV base=0x50046000, IRQ=1
+kvm [1]: vgic-v2@50044000
+kvm [1]: vgic interrupt IRQ1
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.443:1): initialized
+audit: type=2000 audit(0.447:1): initialized
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2016-05-22 22:26:20.357686811 -0700
+++ /tmp/foob	2016-05-22 22:26:20.361686813 -0700
@@ -1,8 +1,3 @@
-   0.000000] earlycon: uart0 at MMIO 0x0000000070006000 (options '115200n8')
-bootconsole [uart0] enabled
-efi: Getting EFI parameters from FDT:
-efi: UEFI not found.
-cma: Reserved 16 MiB at 0x00000000f5400000
 psci: probing for conduit method from DT.
 psci: PSCIv1.0 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
@@ -30,7 +25,7 @@
     .rodata : 0xffff0000087c0000 - 0xffff000008ae0000   (  3200 KB)
       .init : 0xffff000008ae0000 - 0xffff000008bd0000   (   960 KB)
       .data : 0xffff000008bd0000 - 0xffff000008c71e00   (   648 KB)
-       .bss : 0xffff000008c71e00 - 0xffff000008cb2bc0   (   260 KB)
+       .bss : 0xffff000008c71e00 - 0xffff000008cb2b40   (   260 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -59,7 +54,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x9620: expected 0x2004f400
+boot misc3 0x94b8: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -72,7 +67,7 @@
 console [tty0] enabled
 bootconsole [uart0] disabled
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-next-20160520+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu May 19 22:42:58 PDT 2016
+Linux version 4.6.0-next-20160523+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun May 22 22:21:13 PDT 2016
 Boot CPU: AArch64 Processor [411fd071]
 earlycon: uart0 at MMIO 0x0000000070006000 (options '115200n8')
 bootconsole [uart0] enabled
@@ -106,7 +101,7 @@
     .rodata : 0xffff0000087c0000 - 0xffff000008ae0000   (  3200 KB)
       .init : 0xffff000008ae0000 - 0xffff000008bd0000   (   960 KB)
       .data : 0xffff000008bd0000 - 0xffff000008c71e00   (   648 KB)
-       .bss : 0xffff000008c71e00 - 0xffff000008cb2bc0   (   260 KB)
+       .bss : 0xffff000008c71e00 - 0xffff000008cb2b40   (   260 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -135,7 +130,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x9620: expected 0x2004f400
+boot misc3 0x94b8: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -208,11 +203,12 @@
 Freeing initrd memory: 2364K (ffff800009e98000 - ffff80000a0e7000)
 kvm [1]: 8-bit VMID
 kvm [1]: Hyp mode initialized successfully
-kvm [1]: GICH base=0x50044000, GICV base=0x50046000, IRQ=1
+kvm [1]: vgic-v2@50044000
+kvm [1]: vgic interrupt IRQ1
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.231:1): initialized
+audit: type=2000 audit(0.219:1): initialized
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-05-22 22:26:20.365686814 -0700
+++ /tmp/foob	2016-05-22 22:26:20.369686816 -0700
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 ############################
-	 3.4 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 7079272 (6c0568 hex)
+Bytes transferred = 7078048 (6c00a0 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 51627 (c9ab hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6c0568 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6c00a0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff9aa ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-next-20160520-95196-g89949ab (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu May 19 22:40:28 PDT 2016
+Linux version 4.6.0-next-20160523-95466-g9ef1954 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun May 22 22:19:07 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,7 +68,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141bac0, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c141bb00, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -89,9 +89,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc106c540   (14738 kB)
+      .text : 0xc0208000 - 0xc106c574   (14738 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141db20   (1143 kB)
+      .data : 0xc1300000 - 0xc141db60   (1143 kB)
        .bss : 0xc141f000 - 0xc1474ca0   ( 344 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -239,27 +239,27 @@
 ehci-atmel: EHCI Atmel driver
 tegra-ehci: Tegra EHCI driver
 ------------[ cut here ]------------
-WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
+WARNING: CPU: 1 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160520-95196-g89949ab #1
+CPU: 1 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160523-95466-g9ef1954 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c030f9d0>] (unwind_backtrace) from [<c030b0c8>] (show_stack+0x10/0x14)
-[<c030b0c8>] (show_stack) from [<c05864a8>] (dump_stack+0x84/0xa4)
-[<c05864a8>] (dump_stack) from [<c0341b88>] (__warn+0xd4/0x100)
-[<c0341b88>] (__warn) from [<c0341bd4>] (warn_slowpath_null+0x20/0x28)
-[<c0341bd4>] (warn_slowpath_null) from [<c06a0974>] (__of_reset_control_get+0x1bc/0x228)
-[<c06a0974>] (__of_reset_control_get) from [<c0912314>] (tegra_ehci_probe+0x2d0/0x51c)
-[<c0912314>] (tegra_ehci_probe) from [<c07794a4>] (platform_drv_probe+0x50/0xb0)
-[<c07794a4>] (platform_drv_probe) from [<c0777e08>] (driver_probe_device+0x1f4/0x2b0)
-[<c0777e08>] (driver_probe_device) from [<c0777f70>] (__driver_attach+0xac/0xb0)
-[<c0777f70>] (__driver_attach) from [<c077641c>] (bus_for_each_dev+0x54/0x88)
-[<c077641c>] (bus_for_each_dev) from [<c07773fc>] (bus_add_driver+0xe8/0x1f4)
-[<c07773fc>] (bus_add_driver) from [<c077878c>] (driver_register+0x78/0xf4)
-[<c077878c>] (driver_register) from [<c0301e7c>] (do_one_initcall+0x3c/0x16c)
+[<c030f9d0>] (unwind_backtrace) from [<c030b0bc>] (show_stack+0x10/0x14)
+[<c030b0bc>] (show_stack) from [<c0586328>] (dump_stack+0x84/0xa4)
+[<c0586328>] (dump_stack) from [<c0341b7c>] (__warn+0xd4/0x100)
+[<c0341b7c>] (__warn) from [<c0341bc8>] (warn_slowpath_null+0x20/0x28)
+[<c0341bc8>] (warn_slowpath_null) from [<c06a0e7c>] (__of_reset_control_get+0x1bc/0x228)
+[<c06a0e7c>] (__of_reset_control_get) from [<c0912a08>] (tegra_ehci_probe+0x2d0/0x51c)
+[<c0912a08>] (tegra_ehci_probe) from [<c07799ac>] (platform_drv_probe+0x50/0xb0)
+[<c07799ac>] (platform_drv_probe) from [<c0778310>] (driver_probe_device+0x1f4/0x2b0)
+[<c0778310>] (driver_probe_device) from [<c0778478>] (__driver_attach+0xac/0xb0)
+[<c0778478>] (__driver_attach) from [<c0776924>] (bus_for_each_dev+0x54/0x88)
+[<c0776924>] (bus_for_each_dev) from [<c0777904>] (bus_add_driver+0xe8/0x1f4)
+[<c0777904>] (bus_add_driver) from [<c0778c94>] (driver_register+0x78/0xf4)
+[<c0778c94>] (driver_register) from [<c0301e7c>] (do_one_initcall+0x3c/0x16c)
 [<c0301e7c>] (do_one_initcall) from [<c1100d80>] (kernel_init_freeable+0x120/0x1ec)
-[<c1100d80>] (kernel_init_freeable) from [<c0b5b960>] (kernel_init+0x8/0x118)
-[<c0b5b960>] (kernel_init) from [<c0307fb8>] (ret_from_fork+0x14/0x3c)
----[ end trace b4d11744c20cafcf ]---
+[<c1100d80>] (kernel_init_freeable) from [<c0b5c0b0>] (kernel_init+0x8/0x118)
+[<c0b5c0b0>] (kernel_init) from [<c0307fb8>] (ret_from_fork+0x14/0x3c)
+---[ end trace 1942d62a8de9890a ]---
 tegra-ehci 7d004000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d004000.usb: continuing, but please update your DT
 ohci_hcd: USB 1.1 'Open' Host Controller (OHCI) Driver
@@ -308,13 +308,13 @@
 sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
 tegra-pcie 3000.pcie-controller: 2x3 configuration
 tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
+mmc0: error -110 whilst initialising SD card
+tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
-mmc0: error -110 whilst initialising SD card
 mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB
-tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 Primary GPT is invalid, using alternate GPT.
  mmcblk1: p1 p2 p3 p4 p5
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -385,7 +385,7 @@
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
-bash: cannot set terminal process group (-1): Inappropriate ioct[    3.973530] usb 2-1: new high-speed USB device number 2 using tegra-ehci
-l for device
+bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-05-22 22:26:20.373686818 -0700
+++ /tmp/foob	2016-05-22 22:26:20.377686820 -0700
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 ##########################################
-	 3.3 MiB/s
+	 3.1 MiB/s
 done
-Bytes transferred = 5377272 (520cf8 hex)
+Bytes transferred = 5377944 (520f98 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 51627 (c9ab hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x520cf8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x520f98 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff9aa ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-next-20160520-95196-g89949ab (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu May 19 22:40:40 PDT 2016
+Linux version 4.6.0-next-20160523-95466-g9ef1954 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun May 22 22:19:07 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,7 +68,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0c5bd40, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0c5bd80, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -89,9 +89,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0abe014   (10969 kB)
+      .text : 0xc0008000 - 0xc0abe058   (10969 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
-      .data : 0xc0c00000 - 0xc0c5d8c0   ( 375 kB)
+      .data : 0xc0c00000 - 0xc0c5d900   ( 375 kB)
        .bss : 0xc0c5f000 - 0xc0cbab18   ( 367 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -231,25 +231,25 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1b4/0x218
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160520-95196-g89949ab #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160523-95466-g9ef1954 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-[<c010f1e8>] (unwind_backtrace) from [<c010a864>] (show_stack+0x10/0x14)
-[<c010a864>] (show_stack) from [<c033dd54>] (dump_stack+0x84/0xa4)
-[<c033dd54>] (dump_stack) from [<c011f320>] (__warn+0xd4/0x100)
-[<c011f320>] (__warn) from [<c011f36c>] (warn_slowpath_null+0x20/0x28)
-[<c011f36c>] (warn_slowpath_null) from [<c03c129c>] (__of_reset_control_get+0x1b4/0x218)
-[<c03c129c>] (__of_reset_control_get) from [<c0513d84>] (tegra_ehci_probe+0x2c8/0x514)
-[<c0513d84>] (tegra_ehci_probe) from [<c044165c>] (platform_drv_probe+0x50/0xb0)
-[<c044165c>] (platform_drv_probe) from [<c0440088>] (driver_probe_device+0x1f4/0x2b0)
-[<c0440088>] (driver_probe_device) from [<c04401f0>] (__driver_attach+0xac/0xb0)
-[<c04401f0>] (__driver_attach) from [<c043e6ac>] (bus_for_each_dev+0x54/0x88)
-[<c043e6ac>] (bus_for_each_dev) from [<c043f67c>] (bus_add_driver+0xe8/0x1f4)
-[<c043f67c>] (bus_add_driver) from [<c04409ec>] (driver_register+0x78/0xf4)
-[<c04409ec>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
+[<c010f1a8>] (unwind_backtrace) from [<c010a858>] (show_stack+0x10/0x14)
+[<c010a858>] (show_stack) from [<c033db74>] (dump_stack+0x84/0xa4)
+[<c033db74>] (dump_stack) from [<c011f2d4>] (__warn+0xd4/0x100)
+[<c011f2d4>] (__warn) from [<c011f320>] (warn_slowpath_null+0x20/0x28)
+[<c011f320>] (warn_slowpath_null) from [<c03c10c4>] (__of_reset_control_get+0x1b4/0x218)
+[<c03c10c4>] (__of_reset_control_get) from [<c0513d94>] (tegra_ehci_probe+0x2c8/0x514)
+[<c0513d94>] (tegra_ehci_probe) from [<c0441488>] (platform_drv_probe+0x50/0xb0)
+[<c0441488>] (platform_drv_probe) from [<c043feb4>] (driver_probe_device+0x1f4/0x2b0)
+[<c043feb4>] (driver_probe_device) from [<c044001c>] (__driver_attach+0xac/0xb0)
+[<c044001c>] (__driver_attach) from [<c043e4d8>] (bus_for_each_dev+0x54/0x88)
+[<c043e4d8>] (bus_for_each_dev) from [<c043f4a8>] (bus_add_driver+0xe8/0x1f4)
+[<c043f4a8>] (bus_add_driver) from [<c0440818>] (driver_register+0x78/0xf4)
+[<c0440818>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
 [<c010179c>] (do_one_initcall) from [<c0b00d94>] (kernel_init_freeable+0x11c/0x1e8)
-[<c0b00d94>] (kernel_init_freeable) from [<c07f1510>] (kernel_init+0x8/0x118)
-[<c07f1510>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
----[ end trace 6a715040bcd08336 ]---
+[<c0b00d94>] (kernel_init_freeable) from [<c07f1984>] (kernel_init+0x8/0x118)
+[<c07f1984>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
+---[ end trace d15ae1b6fa99956e ]---
 tegra-ehci 7d004000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d004000.usb: continuing, but please update your DT
 usbcore: registered new interface driver cdc_acm
@@ -388,4 +388,4 @@
 usb 2-1: new high-speed USB device number 2 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    7.844402] asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0f:67:60
+root@(none):/# 
\ No newline at end of file
