Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2015-08-10 00:21:54.262299907 -0700
+++ /tmp/foob	2015-08-10 00:21:54.262299907 -0700
@@ -1,6 +1,6 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
-Linux version 4.2.0-rc5-next-20150807+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Fri Aug 7 02:19:00 PDT 2015
+Linux version 4.2.0-rc6-next-20150810+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Aug 10 00:19:03 PDT 2015
 CPU: AArch64 Processor [411fd070] revision 0
 Detected PIPT I-cache on CPU0
 alternatives: enabling workaround for ARM erratum 832075
@@ -19,7 +19,7 @@
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
 software IO TLB [mem 0x79c00000-0x7dc00000] (64MB) mapped at [ffffffc039c00000-ffffffc03dbfffff]
-Memory: 939144K/1048576K available (5413K kernel code, 475K rwdata, 2252K rodata, 508K init, 203K bss, 93048K reserved, 16384K cma-reserved)
+Memory: 939192K/1048576K available (5377K kernel code, 470K rwdata, 2244K rodata, 508K init, 204K bss, 93000K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     vmalloc : 0xffffff8000000000 - 0xffffffbdbfff0000   (   246 GB)
     vmemmap : 0xffffffbdc0000000 - 0xffffffbfc0000000   (     8 GB maximum)
@@ -28,9 +28,9 @@
     PCI I/O : 0xffffffbffae00000 - 0xffffffbffbe00000   (    16 MB)
     modules : 0xffffffbffc000000 - 0xffffffc000000000   (    64 MB)
     memory  : 0xffffffc000000000 - 0xffffffc040000000   (  1024 MB)
-      .init : 0xffffffc0007ff000 - 0xffffffc00087e000   (   508 KB)
-      .text : 0xffffffc000080000 - 0xffffffc0007fe534   (  7674 KB)
-      .data : 0xffffffc00088e000 - 0xffffffc000904c00   (   475 KB)
+      .init : 0xffffffc0007f4000 - 0xffffffc000873000   (   508 KB)
+      .text : 0xffffffc000080000 - 0xffffffc0007f3534   (  7630 KB)
+      .data : 0xffffffc000883000 - 0xffffffc0008f8800   (   470 KB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 64.
@@ -61,7 +61,7 @@
 NET: Registered protocol family 16
 cpuidle: using governor ladder
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffffffc000895000, 1 data @ ffffffc000894000)
+vdso: 2 pages (1 code @ ffffffc000889000, 1 data @ ffffffc000888000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -106,9 +106,6 @@
  vda: vda1 vda2
 tun: Universal TUN/TAP device driver, 1.6
 tun: (C) 1999-2004 Max Krasnyansky <maxk@qualcomm.com>
-thunder-BGX, ver 1.0
-thunder-nic, ver 1.0
-thunder-nicvf, ver 1.0
 sky2: driver version 1.30
 ehci_hcd: USB 2.0 'Enhanced' Host Controller (EHCI) Driver
 ehci-pci: EHCI PCI platform driver
@@ -135,8 +132,8 @@
 EXT4-fs (vda2): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 254:2.
 devtmpfs: mounted
-Freeing unused kernel memory: 508K (ffffffc0007ff000 - ffffffc00087e000)
-Freeing alternatives memory: 44K (ffffffc00087e000 - ffffffc000889000)
+Freeing unused kernel memory: 508K (ffffffc0007f4000 - ffffffc000873000)
+Freeing alternatives memory: 44K (ffffffc000873000 - ffffffc00087e000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 bash-4.3# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-08-10 00:21:54.270299900 -0700
+++ /tmp/foob	2015-08-10 00:21:54.270299900 -0700
@@ -40,18 +40,18 @@
 	 #############################################
 	 3.3 MiB/s
 done
-Bytes transferred = 6373128 (613f08 hex)
+Bytes transferred = 6378000 (615210 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x613f08 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x615210 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc5-next-20150807-34667-ge6455bc (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Fri Aug 7 02:17:40 PDT 2015
+Linux version 4.2.0-rc6-next-20150810-35016-gb195df5 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Aug 10 00:17:40 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -67,7 +67,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f56140, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0f5a1c0, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -80,7 +80,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999024K/2096128K available (8006K kernel code, 1016K rwdata, 3596K rodata, 1008K init, 340K bss, 31568K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1999008K/2096128K available (8014K kernel code, 1016K rwdata, 3604K rodata, 1008K init, 340K bss, 31584K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,10 +88,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d5d9c8   (11607 kB)
-      .init : 0xc0d5e000 - 0xc0e5a000   (1008 kB)
-      .data : 0xc0e5a000 - 0xc0f58100   (1017 kB)
-       .bss : 0xc0f5b000 - 0xc0fb0268   ( 341 kB)
+      .text : 0xc0208000 - 0xc0d61b1c   (11623 kB)
+      .init : 0xc0d62000 - 0xc0e5e000   (1008 kB)
+      .data : 0xc0e5e000 - 0xc0f5c180   (1017 kB)
+       .bss : 0xc0f5f000 - 0xc0fb4268   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -296,7 +296,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-17 18:10:53 UTC (950811053)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-20 16:11:02 UTC (951063062)
 vdd_cam_1v8_reg: disabling
 usb1_vbus: disabling
 dvdd_ts: disabling
@@ -305,7 +305,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1008K (c0d5e000 - c0e5a000)
+Freeing unused kernel memory: 1008K (c0d62000 - c0e5e000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.208997] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.198949] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2015-08-10 00:21:54.278299893 -0700
+++ /tmp/foob	2015-08-10 00:21:54.278299893 -0700
@@ -39,18 +39,18 @@
 	 ###################
 	 3.3 MiB/s
 done
-Bytes transferred = 5036736 (4cdac0 hex)
+Bytes transferred = 5039696 (4ce650 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3.3 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4cdac0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4ce650 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc5-next-20150807-34667-ge6455bc (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Aug 7 02:18:09 PDT 2015
+Linux version 4.2.0-rc6-next-20150810-35016-gb195df5 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Aug 10 00:18:05 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -90,7 +90,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09115ec   (9254 kB)
+      .text : 0xc0008000 - 0xc0911660   (9254 kB)
       .init : 0xc0912000 - 0xc0960000   ( 312 kB)
       .data : 0xc0960000 - 0xc09b6340   ( 345 kB)
        .bss : 0xc09b9000 - 0xc0a13da0   ( 364 kB)
@@ -292,7 +292,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-17 18:11:30 UTC (950811090)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-20 16:11:33 UTC (951063093)
 vdd_cam_1v8_reg: disabling
 usb1_vbus: disabling
 dvdd_ts: disabling
@@ -307,4 +307,4 @@
 Freeing unused kernel memory: 312K (c0912000 - c0960000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.729933] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.730095] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-08-10 00:21:54.282299889 -0700
+++ /tmp/foob	2015-08-10 00:21:54.286299887 -0700
@@ -40,20 +40,20 @@
 	 #################################################################
 	 #################################################################
 	 #############################################
-	 3.1 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 6373128 (613f08 hex)
+Bytes transferred = 6378000 (615210 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *#####
-	 2.8 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x613f08 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x615210 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012a89
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc5-next-20150807-34667-ge6455bc (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Fri Aug 7 02:17:40 PDT 2015
+Linux version 4.2.0-rc6-next-20150810-35016-gb195df5 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Aug 10 00:17:40 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f56140, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0f5a1c0, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998916K/2096128K available (8006K kernel code, 1016K rwdata, 3596K rodata, 1008K init, 340K bss, 31676K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1998900K/2096128K available (8014K kernel code, 1016K rwdata, 3604K rodata, 1008K init, 340K bss, 31692K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d5d9c8   (11607 kB)
-      .init : 0xc0d5e000 - 0xc0e5a000   (1008 kB)
-      .data : 0xc0e5a000 - 0xc0f58100   (1017 kB)
-       .bss : 0xc0f5b000 - 0xc0fb0268   ( 341 kB)
+      .text : 0xc0208000 - 0xc0d61b1c   (11623 kB)
+      .init : 0xc0d62000 - 0xc0e5e000   (1008 kB)
+      .data : 0xc0e5e000 - 0xc0f5c180   (1017 kB)
+       .bss : 0xc0f5f000 - 0xc0fb4268   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -285,14 +285,14 @@
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
+mmc1: MAN_BKOPS_EN bit is not set
 +3.3V_SYS: supplied by +VDD_MUX
 +3.3V_LP0: supplied by +3.3V_SYS
 +1.35V_LP0(sd2): supplied by +5V_SYS
 +1.05V_RUN_AVDD: supplied by +1.35V_LP0(sd2)
-mmc1: MAN_BKOPS_EN bit is not set
-mmc1: new high speed MMC card at address 0001
 tegra-pcie 1003000.pcie-controller: probing port 0, using 2 lanes
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
+mmc1: new high speed MMC card at address 0001
 mmcblk0: mmc1:0001 SEM16G 14.6 GiB 
 mmcblk0boot0: mmc1:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc1:0001 SEM16G partition 2 4.00 MiB
@@ -340,6 +340,7 @@
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
+of_cpumask_init_opp_table: couldn't find opp table for cpu:0, -19
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
@@ -355,12 +356,12 @@
 ata1: SATA link down (SStatus 0 SControl 300)
 EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
 EXT4-fs (mmcblk0p1): couldn't mount as ext2 due to feature incompatibilities
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 devtmpfs: mounted
-Freeing unused kernel memory: 1008K (c0d5e000 - c0e5a000)
+Freeing unused kernel memory: 1008K (c0d62000 - c0e5e000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2015-08-10 00:21:54.290299883 -0700
+++ /tmp/foob	2015-08-10 00:21:54.294299880 -0700
@@ -29,7 +29,6 @@
 Tegra124 (Jetson TK1) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug init=/bin/bash noinitrd'
 Tegra124 (Jetson TK1) # setenv ipaddr 192.168.100.102; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra124 (Jetson TK1) # tftpboot 0x81000000 zImage.tegra_defconfig
-Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'zImage.tegra_defconfig'.
@@ -42,18 +41,18 @@
 	 ###################
 	 3.2 MiB/s
 done
-Bytes transferred = 5036736 (4cdac0 hex)
+Bytes transferred = 5039696 (4ce650 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *#####
-	 3.1 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4cdac0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4ce650 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012a89
@@ -64,7 +63,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc5-next-20150807-34667-ge6455bc (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Aug 7 02:18:09 PDT 2015
+Linux version 4.2.0-rc6-next-20150810-35016-gb195df5 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Aug 10 00:18:05 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -93,7 +92,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09115ec   (9254 kB)
+      .text : 0xc0008000 - 0xc0911660   (9254 kB)
       .init : 0xc0912000 - 0xc0960000   ( 312 kB)
       .data : 0xc0960000 - 0xc09b6340   ( 345 kB)
        .bss : 0xc09b9000 - 0xc0a13da0   ( 364 kB)
@@ -338,10 +337,11 @@
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
+of_cpumask_init_opp_table: couldn't find opp table for cpu:0, -19
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-02-01 00:00:57 UTC (949363257)
+as3722-rtc as3722-rtc: setting system clock to 2000-02-01 00:00:54 UTC (949363254)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_VPP_FUSE: disabling
 +2.8V_RUN_CAM_AF: disabling
@@ -353,9 +353,9 @@
 ALSA device list:
   #0: tegra-hda at 0x70038000 irq 342
   #1: NVIDIA Tegra Jetson TK1
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 ata1: SATA link down (SStatus 0 SControl 300)
 EXT2-fs (mmcblk0p1): error: couldn't mount because of unsupported optional features (244)
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-08-10 00:21:54.298299876 -0700
+++ /tmp/foob	2015-08-10 00:21:54.302299873 -0700
@@ -48,18 +48,18 @@
 	 #############################################
 	 4.1 MiB/s
 done
-Bytes transferred = 6373128 (613f08 hex)
+Bytes transferred = 6378000 (615210 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 4 MiB/s
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x613f08 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x615210 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -67,7 +67,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc5-next-20150807-34667-ge6455bc (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Fri Aug 7 02:17:40 PDT 2015
+Linux version 4.2.0-rc6-next-20150810-35016-gb195df5 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Aug 10 00:17:40 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -75,7 +75,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0f56140, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c0f5a1c0, node_mem_map eeff8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -88,7 +88,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 959832K/1048576K available (8006K kernel code, 1016K rwdata, 3596K rodata, 1008K init, 340K bss, 23208K reserved, 65536K cma-reserved, 204800K highmem)
+Memory: 959816K/1048576K available (8014K kernel code, 1016K rwdata, 3604K rodata, 1008K init, 340K bss, 23224K reserved, 65536K cma-reserved, 204800K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -96,10 +96,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d5d9c8   (11607 kB)
-      .init : 0xc0d5e000 - 0xc0e5a000   (1008 kB)
-      .data : 0xc0e5a000 - 0xc0f58100   (1017 kB)
-       .bss : 0xc0f5b000 - 0xc0fb0268   ( 341 kB)
+      .text : 0xc0208000 - 0xc0d61b1c   (11623 kB)
+      .init : 0xc0d62000 - 0xc0e5e000   (1008 kB)
+      .data : 0xc0e5e000 - 0xc0f5c180   (1017 kB)
+       .bss : 0xc0f5f000 - 0xc0fb4268   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -323,28 +323,29 @@
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 mmc0: host does not support reading read-only switch, assuming write-enable
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
 mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
-tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
  mmcblk0: p1
+tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-26 20:22:06 UTC (1419625326)
+hub 1-1:1.0: USB hub found
+hub 1-1:1.0: 4 ports detected
+rtc-em3027 2-0056: setting system clock to 2014-12-26 20:23:56 UTC (1419625436)
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 mmc1: new high speed SD card at address b368
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
+ mmcblk1: p1 p2
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1008K (c0d5e000 - c0e5a000)
-hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
- mmcblk1: p1 p2
+Freeing unused kernel memory: 1008K (c0d62000 - c0e5e000)
+usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.416955] usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
+root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2015-08-10 00:21:54.306299869 -0700
+++ /tmp/foob	2015-08-10 00:21:54.310299866 -0700
@@ -47,7 +47,7 @@
 	 ###################
 	 4.1 MiB/s
 done
-Bytes transferred = 5036736 (4cdac0 hex)
+Bytes transferred = 5039696 (4ce650 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x4cdac0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x4ce650 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -69,7 +69,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc5-next-20150807-34667-ge6455bc (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Aug 7 02:18:09 PDT 2015
+Linux version 4.2.0-rc6-next-20150810-35016-gb195df5 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Aug 10 00:18:05 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -99,7 +99,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09115ec   (9254 kB)
+      .text : 0xc0008000 - 0xc0911660   (9254 kB)
       .init : 0xc0912000 - 0xc0960000   ( 312 kB)
       .data : 0xc0960000 - 0xc09b6340   ( 345 kB)
        .bss : 0xc09b9000 - 0xc0a13da0   ( 364 kB)
@@ -314,12 +314,12 @@
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
 tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
 NET: Registered protocol family 10
+mip6: Mobile IPv6
+sit: IPv6 over IPv4 tunneling driver
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
-mip6: Mobile IPv6
  mmcblk0: p1
-sit: IPv6 over IPv4 tunneling driver
 NET: Registered protocol family 17
 NET: Registered protocol family 15
 can: controller area network core (rev 20120528 abi 9)
@@ -347,7 +347,7 @@
 hub 2-0:1.0: 1 port detected
  mmcblk1: p1 p2
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-26 20:22:25 UTC (1419625345)
+rtc-em3027 2-0056: setting system clock to 2014-12-26 20:24:15 UTC (1419625455)
 ALSA device list:
   #0: tegra-trimslice
 EXT2-fs (mmcblk0p1): error: couldn't mount because of unsupported optional features (4)
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-08-10 00:21:54.314299862 -0700
+++ /tmp/foob	2015-08-10 00:21:54.318299860 -0700
@@ -40,18 +40,18 @@
 	 #############################################
 	 3.3 MiB/s
 done
-Bytes transferred = 6373128 (613f08 hex)
+Bytes transferred = 6378000 (615210 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 836.9 KiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x613f08 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x615210 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc5-next-20150807-34667-ge6455bc (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Fri Aug 7 02:17:40 PDT 2015
+Linux version 4.2.0-rc6-next-20150810-35016-gb195df5 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Aug 10 00:17:40 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -67,7 +67,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f56140, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0f5a1c0, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -80,7 +80,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998944K/2096128K available (8006K kernel code, 1016K rwdata, 3596K rodata, 1008K init, 340K bss, 31648K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1998928K/2096128K available (8014K kernel code, 1016K rwdata, 3604K rodata, 1008K init, 340K bss, 31664K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,10 +88,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d5d9c8   (11607 kB)
-      .init : 0xc0d5e000 - 0xc0e5a000   (1008 kB)
-      .data : 0xc0e5a000 - 0xc0f58100   (1017 kB)
-       .bss : 0xc0f5b000 - 0xc0fb0268   ( 341 kB)
+      .text : 0xc0208000 - 0xc0d61b1c   (11623 kB)
+      .init : 0xc0d62000 - 0xc0e5e000   (1008 kB)
+      .data : 0xc0e5e000 - 0xc0f5c180   (1017 kB)
+       .bss : 0xc0f5f000 - 0xc0fb4268   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -363,7 +363,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1008K (c0d5e000 - c0e5a000)
+Freeing unused kernel memory: 1008K (c0d62000 - c0e5e000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.758796] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.753798] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2015-08-10 00:21:54.322299856 -0700
+++ /tmp/foob	2015-08-10 00:21:54.326299853 -0700
@@ -39,18 +39,18 @@
 	 ###################
 	 3.4 MiB/s
 done
-Bytes transferred = 5036736 (4cdac0 hex)
+Bytes transferred = 5039696 (4ce650 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 3.5 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4cdac0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4ce650 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc5-next-20150807-34667-ge6455bc (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Aug 7 02:18:09 PDT 2015
+Linux version 4.2.0-rc6-next-20150810-35016-gb195df5 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Aug 10 00:18:05 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -90,7 +90,7 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09115ec   (9254 kB)
+      .text : 0xc0008000 - 0xc0911660   (9254 kB)
       .init : 0xc0912000 - 0xc0960000   ( 312 kB)
       .data : 0xc0960000 - 0xc09b6340   ( 345 kB)
        .bss : 0xc09b9000 - 0xc0a13da0   ( 364 kB)
@@ -366,4 +366,4 @@
 Freeing unused kernel memory: 312K (c0912000 - c0960000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.416744] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.416763] usb 2-1: new high-speed USB device number 2 using tegra-ehci
