Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2015-07-23 23:22:16.901255622 -0700
+++ /tmp/foob	2015-07-23 23:22:16.905255620 -0700
@@ -1,6 +1,6 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
-Linux version 4.2.0-rc3-next-20150723+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Jul 23 00:18:50 PDT 2015
+Linux version 4.2.0-rc3-next-20150724+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Jul 23 23:19:33 PDT 2015
 CPU: AArch64 Processor [411fd070] revision 0
 Detected PIPT I-cache on CPU0
 alternatives: enabling workaround for ARM erratum 832075
@@ -19,7 +19,7 @@
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
 software IO TLB [mem 0x79c00000-0x7dc00000] (64MB) mapped at [ffffffc039c00000-ffffffc03dbfffff]
-Memory: 939132K/1048576K available (5473K kernel code, 476K rwdata, 2240K rodata, 512K init, 202K bss, 93060K reserved, 16384K cma-reserved)
+Memory: 939132K/1048576K available (5477K kernel code, 472K rwdata, 2240K rodata, 512K init, 202K bss, 93060K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     vmalloc : 0xffffff8000000000 - 0xffffffbdbfff0000   (   246 GB)
     vmemmap : 0xffffffbdc0000000 - 0xffffffbfc0000000   (     8 GB maximum)
@@ -28,9 +28,9 @@
     PCI I/O : 0xffffffbffae00000 - 0xffffffbffbe00000   (    16 MB)
     modules : 0xffffffbffc000000 - 0xffffffc000000000   (    64 MB)
     memory  : 0xffffffc000000000 - 0xffffffc040000000   (  1024 MB)
-      .init : 0xffffffc00080b000 - 0xffffffc00088b000   (   512 KB)
-      .text : 0xffffffc000080000 - 0xffffffc00080a5b4   (  7722 KB)
-      .data : 0xffffffc000891000 - 0xffffffc000908000   (   476 KB)
+      .init : 0xffffffc00080c000 - 0xffffffc00088c000   (   512 KB)
+      .text : 0xffffffc000080000 - 0xffffffc00080b5c4   (  7726 KB)
+      .data : 0xffffffc000892000 - 0xffffffc000908000   (   472 KB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 64.
@@ -88,7 +88,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 2, 16384 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.380:1): initialized
+audit: type=2000 audit(0.430:1): initialized
 HugeTLB registered 2 MB page size, pre-allocated 0 pages
 VFS: Disk quotas dquot_6.6.0
 VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
@@ -135,8 +135,8 @@
 EXT4-fs (vda2): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 254:2.
 devtmpfs: mounted
-Freeing unused kernel memory: 512K (ffffffc00080b000 - ffffffc00088b000)
-Freeing alternatives memory: 16K (ffffffc00088b000 - ffffffc00088f000)
+Freeing unused kernel memory: 512K (ffffffc00080c000 - ffffffc00088c000)
+Freeing alternatives memory: 16K (ffffffc00088c000 - ffffffc000890000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 bash-4.3# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-07-23 23:22:16.909255618 -0700
+++ /tmp/foob	2015-07-23 23:22:16.913255617 -0700
@@ -38,20 +38,20 @@
 	 #################################################################
 	 #################################################################
 	 #########################################
-	 3.3 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 6320792 (607298 hex)
+Bytes transferred = 6324552 (608148 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.8 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x607298 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x608148 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc3-next-20150723-31160-g4e8d04d (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Jul 23 00:17:34 PDT 2015
+Linux version 4.2.0-rc3-next-20150724-31452-ge8e9cc3 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Jul 23 23:17:42 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -67,7 +67,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f33d00, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0f35d80, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -80,7 +80,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999184K/2096128K available (7964K kernel code, 1007K rwdata, 3556K rodata, 960K init, 337K bss, 31408K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1999176K/2096128K available (7968K kernel code, 1007K rwdata, 3560K rodata, 960K init, 337K bss, 31416K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,10 +88,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d49204   (11525 kB)
-      .init : 0xc0d4a000 - 0xc0e3a000   ( 960 kB)
-      .data : 0xc0e3a000 - 0xc0f35d00   (1008 kB)
-       .bss : 0xc0f38000 - 0xc0f8c660   ( 338 kB)
+      .text : 0xc0208000 - 0xc0d4b2e8   (11533 kB)
+      .init : 0xc0d4c000 - 0xc0e3c000   ( 960 kB)
+      .data : 0xc0e3c000 - 0xc0f37d80   (1008 kB)
+       .bss : 0xc0f3a000 - 0xc0f8e760   ( 338 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -296,7 +296,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-02 16:10:07 UTC (949507807)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-03 15:10:21 UTC (949590621)
 vdd_cam_1v8_reg: disabling
 usb1_vbus: disabling
 dvdd_ts: disabling
@@ -305,7 +305,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 960K (c0d4a000 - c0e3a000)
+Freeing unused kernel memory: 960K (c0d4c000 - c0e3c000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.194156] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.204065] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2015-07-23 23:22:16.917255614 -0700
+++ /tmp/foob	2015-07-23 23:22:16.921255611 -0700
@@ -39,18 +39,18 @@
 	 ##################
 	 3.2 MiB/s
 done
-Bytes transferred = 5030376 (4cc1e8 hex)
+Bytes transferred = 5033824 (4ccf60 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4cc1e8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4ccf60 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc3-next-20150723-31160-g4e8d04d (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Jul 23 00:18:11 PDT 2015
+Linux version 4.2.0-rc3-next-20150724-31452-ge8e9cc3 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Jul 23 23:18:15 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c09af0c0, node_mem_map ee7f7000
+free_area_init_node: node 0, pgdat c09b10c0, node_mem_map ee7f7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002756K/2096128K available (6859K kernel code, 347K rwdata, 2372K rodata, 308K init, 363K bss, 27836K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 2002748K/2096128K available (6863K kernel code, 347K rwdata, 2372K rodata, 312K init, 363K bss, 27844K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc090c360   (9233 kB)
-      .init : 0xc090d000 - 0xc095a000   ( 308 kB)
-      .data : 0xc095a000 - 0xc09b0c80   ( 348 kB)
-       .bss : 0xc09b3000 - 0xc0a0dc58   ( 364 kB)
+      .text : 0xc0008000 - 0xc090d418   (9238 kB)
+      .init : 0xc090e000 - 0xc095c000   ( 312 kB)
+      .data : 0xc095c000 - 0xc09b2c80   ( 348 kB)
+       .bss : 0xc09b5000 - 0xc0a0fd5c   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -292,7 +292,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-02 16:10:46 UTC (949507846)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-03 15:10:54 UTC (949590654)
 vdd_cam_1v8_reg: disabling
 usb1_vbus: disabling
 dvdd_ts: disabling
@@ -304,7 +304,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 308K (c090d000 - c095a000)
+Freeing unused kernel memory: 312K (c090e000 - c095c000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.729958] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.729961] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-07-23 23:22:16.925255608 -0700
+++ /tmp/foob	2015-07-23 23:22:16.929255605 -0700
@@ -40,9 +40,9 @@
 	 #################################################################
 	 #################################################################
 	 #########################################
-	 3.2 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 6320792 (607298 hex)
+Bytes transferred = 6324552 (608148 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -53,7 +53,7 @@
 done
 Bytes transferred = 63859 (f973 hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x607298 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x608148 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012972
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc3-next-20150723-31160-g4e8d04d (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Jul 23 00:17:34 PDT 2015
+Linux version 4.2.0-rc3-next-20150724-31452-ge8e9cc3 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Jul 23 23:17:42 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f33d00, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0f35d80, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999080K/2096128K available (7964K kernel code, 1007K rwdata, 3556K rodata, 960K init, 337K bss, 31512K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1999072K/2096128K available (7968K kernel code, 1007K rwdata, 3560K rodata, 960K init, 337K bss, 31520K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d49204   (11525 kB)
-      .init : 0xc0d4a000 - 0xc0e3a000   ( 960 kB)
-      .data : 0xc0e3a000 - 0xc0f35d00   (1008 kB)
-       .bss : 0xc0f38000 - 0xc0f8c660   ( 338 kB)
+      .text : 0xc0208000 - 0xc0d4b2e8   (11533 kB)
+      .init : 0xc0d4c000 - 0xc0e3c000   ( 960 kB)
+      .data : 0xc0e3c000 - 0xc0f37d80   (1008 kB)
+       .bss : 0xc0f3a000 - 0xc0f8e760   ( 338 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -359,7 +359,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 960K (c0d4a000 - c0e3a000)
+Freeing unused kernel memory: 960K (c0d4c000 - c0e3c000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2015-07-23 23:22:16.933255604 -0700
+++ /tmp/foob	2015-07-23 23:22:16.933255604 -0700
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 ##################
-	 3.2 MiB/s
+	 3.1 MiB/s
 done
-Bytes transferred = 5030376 (4cc1e8 hex)
+Bytes transferred = 5033824 (4ccf60 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *#####
-	 2.8 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 63859 (f973 hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4cc1e8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4ccf60 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012972
@@ -63,7 +63,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc3-next-20150723-31160-g4e8d04d (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Jul 23 00:18:11 PDT 2015
+Linux version 4.2.0-rc3-next-20150724-31452-ge8e9cc3 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Jul 23 23:18:15 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -71,7 +71,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c09af0c0, node_mem_map ee7f7000
+free_area_init_node: node 0, pgdat c09b10c0, node_mem_map ee7f7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -84,7 +84,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002652K/2096128K available (6859K kernel code, 347K rwdata, 2372K rodata, 308K init, 363K bss, 27940K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 2002644K/2096128K available (6863K kernel code, 347K rwdata, 2372K rodata, 312K init, 363K bss, 27948K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,10 +92,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc090c360   (9233 kB)
-      .init : 0xc090d000 - 0xc095a000   ( 308 kB)
-      .data : 0xc095a000 - 0xc09b0c80   ( 348 kB)
-       .bss : 0xc09b3000 - 0xc0a0dc58   ( 364 kB)
+      .text : 0xc0008000 - 0xc090d418   (9238 kB)
+      .init : 0xc090e000 - 0xc095c000   ( 312 kB)
+      .data : 0xc095c000 - 0xc09b2c80   ( 348 kB)
+       .bss : 0xc09b5000 - 0xc0a0fd5c   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -359,7 +359,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 308K (c090d000 - c095a000)
+Freeing unused kernel memory: 312K (c090e000 - c095c000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-07-23 23:22:16.941255598 -0700
+++ /tmp/foob	2015-07-23 23:22:16.941255598 -0700
@@ -46,20 +46,20 @@
 	 #################################################################
 	 #################################################################
 	 #########################################
-	 4.1 MiB/s
+	 3.9 MiB/s
 done
-Bytes transferred = 6320792 (607298 hex)
+Bytes transferred = 6324552 (608148 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 2 MiB/s
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x607298 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x608148 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -67,7 +67,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc3-next-20150723-31160-g4e8d04d (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Jul 23 00:17:34 PDT 2015
+Linux version 4.2.0-rc3-next-20150724-31452-ge8e9cc3 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Jul 23 23:17:42 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -75,7 +75,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0f33d00, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c0f35d80, node_mem_map eeff8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -88,7 +88,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 959984K/1048576K available (7964K kernel code, 1007K rwdata, 3556K rodata, 960K init, 337K bss, 23056K reserved, 65536K cma-reserved, 204800K highmem)
+Memory: 959976K/1048576K available (7968K kernel code, 1007K rwdata, 3560K rodata, 960K init, 337K bss, 23064K reserved, 65536K cma-reserved, 204800K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -96,10 +96,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d49204   (11525 kB)
-      .init : 0xc0d4a000 - 0xc0e3a000   ( 960 kB)
-      .data : 0xc0e3a000 - 0xc0f35d00   (1008 kB)
-       .bss : 0xc0f38000 - 0xc0f8c660   ( 338 kB)
+      .text : 0xc0208000 - 0xc0d4b2e8   (11533 kB)
+      .init : 0xc0d4c000 - 0xc0e3c000   ( 960 kB)
+      .data : 0xc0e3c000 - 0xc0f37d80   (1008 kB)
+       .bss : 0xc0f3a000 - 0xc0f8e760   ( 338 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -323,26 +323,26 @@
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
-tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
-mmc0: host does not support reading read-only switch, assuming write-enable
-mmc0: new high speed SDHC card at address b368
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
+tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
 hub 2-0:1.0: USB hub found
- mmcblk0: p1
+mmc0: host does not support reading read-only switch, assuming write-enable
 hub 2-0:1.0: 1 port detected
+mmc0: new high speed SDHC card at address b368
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-26 19:22:58 UTC (1419621778)
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
+rtc-em3027 2-0056: setting system clock to 2014-12-26 19:37:43 UTC (1419622663)
+ mmcblk0: p1
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
+hub 1-1:1.0: USB hub found
+hub 1-1:1.0: 4 ports detected
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
-hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
 devtmpfs: mounted
-Freeing unused kernel memory: 960K (c0d4a000 - c0e3a000)
 mmc1: new high speed SD card at address b368
+Freeing unused kernel memory: 960K (c0d4c000 - c0e3c000)
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2015-07-23 23:22:16.945255595 -0700
+++ /tmp/foob	2015-07-23 23:22:16.949255593 -0700
@@ -45,9 +45,9 @@
 	 #################################################################
 	 #################################################################
 	 ##################
-	 4.1 MiB/s
+	 3.8 MiB/s
 done
-Bytes transferred = 5030376 (4cc1e8 hex)
+Bytes transferred = 5033824 (4ccf60 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x4cc1e8 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x4ccf60 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -69,7 +69,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc3-next-20150723-31160-g4e8d04d (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Jul 23 00:18:11 PDT 2015
+Linux version 4.2.0-rc3-next-20150724-31452-ge8e9cc3 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Jul 23 23:18:15 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -78,7 +78,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c09af0c0, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c09b10c0, node_mem_map eeff7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -91,7 +91,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 963556K/1048576K available (6859K kernel code, 347K rwdata, 2372K rodata, 308K init, 363K bss, 19484K reserved, 65536K cma-reserved, 204800K highmem)
+Memory: 963548K/1048576K available (6863K kernel code, 347K rwdata, 2372K rodata, 312K init, 363K bss, 19492K reserved, 65536K cma-reserved, 204800K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -99,10 +99,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc090c360   (9233 kB)
-      .init : 0xc090d000 - 0xc095a000   ( 308 kB)
-      .data : 0xc095a000 - 0xc09b0c80   ( 348 kB)
-       .bss : 0xc09b3000 - 0xc0a0dc58   ( 364 kB)
+      .text : 0xc0008000 - 0xc090d418   (9238 kB)
+      .init : 0xc090e000 - 0xc095c000   ( 312 kB)
+      .data : 0xc095c000 - 0xc09b2c80   ( 348 kB)
+       .bss : 0xc09b5000 - 0xc0a0fd5c   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
@@ -315,9 +315,11 @@
 tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
 NET: Registered protocol family 10
 mmc0: host does not support reading read-only switch, assuming write-enable
+mmc0: new high speed SDHC card at address b368
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
 mip6: Mobile IPv6
+ mmcblk0: p1
 sit: IPv6 over IPv4 tunneling driver
-mmc0: new high speed SDHC card at address b368
 NET: Registered protocol family 17
 NET: Registered protocol family 15
 can: controller area network core (rev 20120528 abi 9)
@@ -326,9 +328,7 @@
 can: broadcast manager protocol (rev 20120528 t)
 can: netlink gateway (rev 20130117) max_hops=1
 Bluetooth: RFCOMM socket layer initialized
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
 Bluetooth: RFCOMM ver 1.11
- mmcblk0: p1
 Bluetooth: BNEP (Ethernet Emulation) ver 1.3
 Bluetooth: BNEP socket layer initialized
 Bluetooth: HIDP (Human Interface Emulation) ver 1.2
@@ -340,23 +340,23 @@
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
+mmc1: new high speed SD card at address b368
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
+ mmcblk1: p1 p2
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-26 19:23:17 UTC (1419621797)
-mmc1: new high speed SD card at address b368
+rtc-em3027 2-0056: setting system clock to 2014-12-26 19:38:03 UTC (1419622683)
 ALSA device list:
   #0: tegra-trimslice
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 EXT2-fs (mmcblk0p1): error: couldn't mount because of unsupported optional features (4)
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
- mmcblk1: p1 p2
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 308K (c090d000 - c095a000)
+Freeing unused kernel memory: 312K (c090e000 - c095c000)
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-07-23 23:22:16.953255591 -0700
+++ /tmp/foob	2015-07-23 23:22:16.957255588 -0700
@@ -40,18 +40,18 @@
 	 #########################################
 	 3.3 MiB/s
 done
-Bytes transferred = 6320792 (607298 hex)
+Bytes transferred = 6324552 (608148 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.3 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x607298 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x608148 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc3-next-20150723-31160-g4e8d04d (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Jul 23 00:17:34 PDT 2015
+Linux version 4.2.0-rc3-next-20150724-31452-ge8e9cc3 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Jul 23 23:17:42 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -67,7 +67,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f33d00, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0f35d80, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -80,7 +80,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999104K/2096128K available (7964K kernel code, 1007K rwdata, 3556K rodata, 960K init, 337K bss, 31488K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1999096K/2096128K available (7968K kernel code, 1007K rwdata, 3560K rodata, 960K init, 337K bss, 31496K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,10 +88,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d49204   (11525 kB)
-      .init : 0xc0d4a000 - 0xc0e3a000   ( 960 kB)
-      .data : 0xc0e3a000 - 0xc0f35d00   (1008 kB)
-       .bss : 0xc0f38000 - 0xc0f8c660   ( 338 kB)
+      .text : 0xc0208000 - 0xc0d4b2e8   (11533 kB)
+      .init : 0xc0d4c000 - 0xc0e3c000   ( 960 kB)
+      .data : 0xc0e3c000 - 0xc0f37d80   (1008 kB)
+       .bss : 0xc0f3a000 - 0xc0f8e760   ( 338 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -290,9 +290,9 @@
 mmcblk0boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk0boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk0rpmb: mmc1:0001 SEM16G partition 3 128 KiB
+tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 Primary GPT is invalid, using alternate GPT.
  mmcblk0: p1 p2 p3 p4 p5
-tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -363,7 +363,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 960K (c0d4a000 - c0e3a000)
+Freeing unused kernel memory: 960K (c0d4c000 - c0e3c000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.743375] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.768822] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2015-07-23 23:22:16.961255585 -0700
+++ /tmp/foob	2015-07-23 23:22:16.965255583 -0700
@@ -37,9 +37,9 @@
 	 #################################################################
 	 #################################################################
 	 ##################
-	 3.3 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 5030376 (4cc1e8 hex)
+Bytes transferred = 5033824 (4ccf60 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4cc1e8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4ccf60 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc3-next-20150723-31160-g4e8d04d (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Jul 23 00:18:11 PDT 2015
+Linux version 4.2.0-rc3-next-20150724-31452-ge8e9cc3 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Jul 23 23:18:15 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c09af0c0, node_mem_map ee7f7000
+free_area_init_node: node 0, pgdat c09b10c0, node_mem_map ee7f7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002676K/2096128K available (6859K kernel code, 347K rwdata, 2372K rodata, 308K init, 363K bss, 27916K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 2002668K/2096128K available (6863K kernel code, 347K rwdata, 2372K rodata, 312K init, 363K bss, 27924K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc090c360   (9233 kB)
-      .init : 0xc090d000 - 0xc095a000   ( 308 kB)
-      .data : 0xc095a000 - 0xc09b0c80   ( 348 kB)
-       .bss : 0xc09b3000 - 0xc0a0dc58   ( 364 kB)
+      .text : 0xc0008000 - 0xc090d418   (9238 kB)
+      .init : 0xc090e000 - 0xc095c000   ( 312 kB)
+      .data : 0xc095c000 - 0xc09b2c80   ( 348 kB)
+       .bss : 0xc09b5000 - 0xc0a0fd5c   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -363,7 +363,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 308K (c090d000 - c095a000)
+Freeing unused kernel memory: 312K (c090e000 - c095c000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.416703] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.406409] usb 2-1: new high-speed USB device number 2 using tegra-ehci
