Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-03-16 22:55:57.477263285 -0700
+++ /tmp/foob	2016-03-16 22:55:57.477263285 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.5.0-next-20160316+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Mar 15 23:50:37 PDT 2016
+Linux version 4.5.0-next-20160317+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Mar 16 22:50:49 PDT 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -9,7 +9,7 @@
 psci: PSCIv0.2 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: Trusted OS migration not required
-percpu: Embedded 20 pages/cpu @ffffffc03efd4000 s43008 r8192 d30720 u81920
+percpu: Embedded 20 pages/cpu @ffffffc03efd3000 s43008 r8192 d30720 u81920
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
 CPU features: enabling workaround for ARM erratum 834220
@@ -19,14 +19,14 @@
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
 software IO TLB [mem 0x79c00000-0x7dc00000] (64MB) mapped at [ffffffc039c00000-ffffffc03dbfffff]
-Memory: 936628K/1048576K available (6840K kernel code, 616K rwdata, 2880K rodata, 812K init, 241K bss, 95564K reserved, 16384K cma-reserved)
+Memory: 936624K/1048576K available (6844K kernel code, 612K rwdata, 2880K rodata, 812K init, 241K bss, 95568K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffffff8000000000 - 0xffffff8008000000   (   128 MB)
     vmalloc : 0xffffff8008000000 - 0xffffffbdbfff0000   (   246 GB)
-      .text : 0xffffff8008080000 - 0xffffff800872c000   (  6832 KB)
-    .rodata : 0xffffff800872c000 - 0xffffff8008a00000   (  2896 KB)
-      .init : 0xffffff8008a00000 - 0xffffff8008acb000   (   812 KB)
-      .data : 0xffffff8008acb000 - 0xffffff8008b65000   (   616 KB)
+      .text : 0xffffff8008080000 - 0xffffff800872d000   (  6836 KB)
+    .rodata : 0xffffff800872d000 - 0xffffff8008a01000   (  2896 KB)
+      .init : 0xffffff8008a01000 - 0xffffff8008acc000   (   812 KB)
+      .data : 0xffffff8008acc000 - 0xffffff8008b65000   (   612 KB)
     vmemmap : 0xffffffbdc0000000 - 0xffffffbfc0000000   (     8 GB maximum)
               0xffffffbdc0000000 - 0xffffffbdc1000000   (    16 MB actual)
     fixed   : 0xffffffbffe7fd000 - 0xffffffbffec00000   (  4108 KB)
@@ -60,7 +60,7 @@
 pinctrl core: initialized pinctrl subsystem
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffffff8008732000, 1 data @ ffffff8008ad0000)
+vdso: 2 pages (1 code @ ffffff8008733000, 1 data @ ffffff8008ad0000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -94,7 +94,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.644:1): initialized
+audit: type=2000 audit(0.560:1): initialized
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -154,7 +154,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2016-03-16 06:51:10 UTC (1458111070)
+rtc-pl031 9010000.pl031: setting system clock to 2016-03-17 05:51:18 UTC (1458193878)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
@@ -163,7 +163,7 @@
 EXT4-fs (vda2): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 254:2.
 devtmpfs: mounted
-Freeing unused kernel memory: 812K (ffffff8008a00000 - ffffff8008acb000)
+Freeing unused kernel memory: 812K (ffffff8008a01000 - ffffff8008acc000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 bash-4.3# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-03-16 22:55:57.485263279 -0700
+++ /tmp/foob	2016-03-16 22:55:57.485263279 -0700
@@ -41,7 +41,7 @@
 	 #################
 	 3.2 MiB/s
 done
-Bytes transferred = 6926120 (69af28 hex)
+Bytes transferred = 6926984 (69b288 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -52,7 +52,7 @@
 done
 Bytes transferred = 34912 (8860 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x69af28 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x69b288 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff85f ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.5.0-next-20160316-81627-g8559bac (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Mar 15 23:49:14 PDT 2016
+Linux version 4.5.0-next-20160317-81903-g0a2dc466 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Mar 16 22:48:59 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -276,8 +276,8 @@
 vdd-sys-5v0: supplied by vdd_ac_bat
 VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
 vdd_hdmi_5v0: supplied by vdd-sys-5v0
-mmc0: new high speed SDHC card at address aaaa
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
+mmc0: new high speed SDHC card at address aaaa
 mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 [drm] No driver support for vblank timestamp query.
  mmcblk0: p1 p2 p3
@@ -298,7 +298,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-08 18:08:25 UTC (947354905)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-09 17:08:17 UTC (947437697)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -310,4 +310,5 @@
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.239243] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-03-16 22:55:57.489263276 -0700
+++ /tmp/foob	2016-03-16 22:55:57.493263272 -0700
@@ -36,21 +36,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #####################################
-	 3.5 MiB/s
+	 ######################################
+	 3.3 MiB/s
 done
-Bytes transferred = 5312000 (510e00 hex)
+Bytes transferred = 5314952 (511988 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 34912 (8860 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x510e00 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x511988 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff85f ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.5.0-next-20160316-81627-g8559bac (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Mar 15 23:48:58 PDT 2016
+Linux version 4.5.0-next-20160317-81903-g0a2dc466 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Mar 16 22:48:53 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -90,7 +90,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab2164   (10921 kB)
+      .text : 0xc0008000 - 0xc0ab2180   (10921 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
       .data : 0xc0c00000 - 0xc0c5df80   ( 376 kB)
        .bss : 0xc0c5f000 - 0xc0cb9a88   ( 363 kB)
@@ -250,15 +250,15 @@
 mmc1: SDHCI controller on 78000600.sdhci [78000600.sdhci] using ADMA
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
-mmc1: new high speed MMC card at address 0001
-mmcblk0: mmc1:0001 SEM32G 29.1 GiB 
-mmcblk0boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmc0: new high speed SDHC card at address aaaa
-mmcblk0boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
-mmcblk1: mmc0:aaaa SL16G 14.8 GiB 
-mmcblk0rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
- mmcblk1: p1 p2 p3
+mmc1: new high speed MMC card at address 0001
+mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
+mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
+mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
+mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
+ mmcblk0: p1 p2 p3
+mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
+ mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
 tegra30-i2s 70080400.i2s: DMA channels sourced from device 70080000.ahub
 tegra-snd-rt5640 sound: rt5640-aif1 <-> 70080400.i2s mapping ok
 input: NVIDIA Tegra Dalmore Headphones as /devices/soc0/sound/sound/card0/input0
@@ -296,16 +296,16 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-08 18:05:48 UTC (947354748)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-09 17:05:47 UTC (947437547)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
 ALSA device list:
   #0: NVIDIA Tegra Dalmore
-EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
-EXT4-fs (mmcblk0p1): recovery complete
-EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:1.
+EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
+EXT4-fs (mmcblk1p1): recovery complete
+EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
+VFS: Mounted root (ext3 filesystem) on device 179:17.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-03-16 22:55:57.497263269 -0700
+++ /tmp/foob	2016-03-16 22:55:57.501263266 -0700
@@ -43,18 +43,18 @@
 	 #################
 	 3.2 MiB/s
 done
-Bytes transferred = 6926120 (69af28 hex)
+Bytes transferred = 6926984 (69b288 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 2.3 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 64236 (faec hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x69af28 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x69b288 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012aeb
@@ -62,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.5.0-next-20160316-81627-g8559bac (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Mar 15 23:49:14 PDT 2016
+Linux version 4.5.0-next-20160317-81903-g0a2dc466 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Mar 16 22:48:59 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-03-16 22:55:57.505263263 -0700
+++ /tmp/foob	2016-03-16 22:55:57.509263259 -0700
@@ -38,21 +38,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #####################################
+	 ######################################
 	 3.2 MiB/s
 done
-Bytes transferred = 5312000 (510e00 hex)
+Bytes transferred = 5314952 (511988 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.1 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 64236 (faec hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x510e00 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x511988 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012aeb
@@ -61,7 +61,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.5.0-next-20160316-81627-g8559bac (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Mar 15 23:48:58 PDT 2016
+Linux version 4.5.0-next-20160317-81903-g0a2dc466 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Mar 16 22:48:53 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -92,7 +92,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab2164   (10921 kB)
+      .text : 0xc0008000 - 0xc0ab2180   (10921 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
       .data : 0xc0c00000 - 0xc0c5df80   ( 376 kB)
        .bss : 0xc0c5f000 - 0xc0cb9a88   ( 363 kB)
@@ -353,7 +353,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 14:32:23 UTC (946737143)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:18 UTC (946684818)
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
@@ -367,13 +367,13 @@
   #1: NVIDIA Tegra Jetson TK1
 ata1: SATA link down (SStatus 0 SControl 300)
 EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 EXT4-fs (mmcblk0p1): warning: mounting fs with errors, running e2fsck is recommended
 EXT4-fs (mmcblk0p1): recovery complete
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    6.707486] asix 2-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:11:31:3b
+root@(none):/# [    6.657124] asix 2-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:11:31:3b
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2016-03-16 22:55:57.513263257 -0700
+++ /tmp/foob	2016-03-16 22:55:57.517263253 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1799989 us
+Bootloader-Cpu Init at (time stamp): 1800282 us
 
 Pinmux changes applied in kernel way
 
@@ -62,29 +62,29 @@
 Board Id Read Time: 4 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1799962  us
-Bootloader (AosInit) time: 734228 us
-Bootloader Main Init time: 180087 us
-BlockDevMgrInit time: 157535 us
-PartMgrInit time: 9 us
-StorMgrInit time: 30 us
+Miscellaneous avp time: 1800255  us
+Bootloader (AosInit) time: 733468 us
+Bootloader Main Init time: 180349 us
+BlockDevMgrInit time: 157566 us
+PartMgrInit time: 8 us
+StorMgrInit time: 31 us
 FsMgrInit time: 2 us
-Partition Table load: 76773 us 
-Display init time: 574664 us
+Partition Table load: 77042 us 
+Display init time: 574430 us
 Kernel read time: 301 msec
 Load Kernel Size:   13053952 Bytes
 Bl Kernel ReadSpeed: 43368000 Bytes/sec
-Pre-Kernel Jump time: 1489573 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5493483 us
+Pre-Kernel Jump time: 1489331 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5493227 us
 Booting Linux on physical CPU 0x0
-Linux version 4.5.0-next-20160316+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Mar 15 23:50:37 PDT 2016
+Linux version 4.5.0-next-20160317+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Mar 16 22:50:49 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
 Memory limited to 1734MB
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
 cma: Reserved 16 MiB at 0x00000000eb400000
 /cpus/cpu@1: missing enable-method property
-percpu: Embedded 20 pages/cpu @ffffffc06c5a4000 s43008 r8192 d30720 u81920
+percpu: Embedded 20 pages/cpu @ffffffc06c5a3000 s43008 r8192 d30720 u81920
 Detected PIPT I-cache on CPU0
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 436968
 Kernel command line: tegraid=13.1.3.0.0 mem=1734M@2048M memtype=255 vpr=151M@3817M ddr_die=2048M@2048M section=256M commchip_id=0 usb_port_owner_info=0 lane_owner_info=0 emc_max_dvfs=0 touch_id=0@0 video=tegrafb no_console_suspend=1 console=ttyS0,115200n8 debug_uartport=lsport,0 earlyprintk=uart8250-32bit,0x70006000 usbcore.old_scheme_first=1 lp0_vec=0@0x0 tegra_fbmem=32899072@0xad012000 core_edp_mv=1150 core_edp_ma=4000 pmuboard=0x0176:0x0001:0x01:0x43:0x00 displayboard=0x0162:0x0001:0x00:0x42:0x00 power_supply=Adapter audio_codec=rt5640 board_info=0x0176:0x0001:0x01:0x43:0x00 tegraboot=sdmmc gpt gpt_sector=94207 modem_id=0 android.kerneltype=normal
@@ -92,14 +92,14 @@
 Dentry cache hash table entries: 262144 (order: 9, 2097152 bytes)
 Inode-cache hash table entries: 131072 (order: 8, 1048576 bytes)
 software IO TLB [mem 0xe5200000-0xe9200000] (64MB) mapped at [ffffffc065200000-ffffffc0691fffff]
-Memory: 1643996K/1775616K available (6840K kernel code, 616K rwdata, 2880K rodata, 812K init, 241K bss, 115236K reserved, 16384K cma-reserved)
+Memory: 1643992K/1775616K available (6844K kernel code, 612K rwdata, 2880K rodata, 812K init, 241K bss, 115240K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffffff8000000000 - 0xffffff8008000000   (   128 MB)
     vmalloc : 0xffffff8008000000 - 0xffffffbdbfff0000   (   246 GB)
-      .text : 0xffffff8008080000 - 0xffffff800872c000   (  6832 KB)
-    .rodata : 0xffffff800872c000 - 0xffffff8008a00000   (  2896 KB)
-      .init : 0xffffff8008a00000 - 0xffffff8008acb000   (   812 KB)
-      .data : 0xffffff8008acb000 - 0xffffff8008b65000   (   616 KB)
+      .text : 0xffffff8008080000 - 0xffffff800872d000   (  6836 KB)
+    .rodata : 0xffffff800872d000 - 0xffffff8008a01000   (  2896 KB)
+      .init : 0xffffff8008a01000 - 0xffffff8008acc000   (   812 KB)
+      .data : 0xffffff8008acc000 - 0xffffff8008b65000   (   612 KB)
     vmemmap : 0xffffffbdc0000000 - 0xffffffbfc0000000   (     8 GB maximum)
               0xffffffbdc0000000 - 0xffffffbdc1b18000   (    27 MB actual)
     fixed   : 0xffffffbffe7fd000 - 0xffffffbffec00000   (  4108 KB)
@@ -133,7 +133,7 @@
 pinctrl core: initialized pinctrl subsystem
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffffff8008732000, 1 data @ ffffff8008ad0000)
+vdso: 2 pages (1 code @ ffffff8008733000, 1 data @ ffffff8008ad0000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -179,7 +179,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.859:1): initialized
+audit: type=2000 audit(0.871:1): initialized
 workingset: timestamp_bits=44 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -254,25 +254,25 @@
 +5V_HDMI_CON: disabling
 ALSA device list:
   No soundcards found.
-Freeing unused kernel memory: 812K (ffffff8008a00000 - ffffff8008acb000)
+Freeing unused kernel memory: 812K (ffffff8008a01000 - ffffff8008acc000)
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 
 System initialization...
 
-Mounting /proc :[    2.583525] mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+Mounting /proc :[    2.589336] mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 [SUCCESS]
-Mounting /sys :[    2.598773] mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+Mounting /sys :[SUCCESS]
+Mounting /dev :[    2.609994] mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 [SUCCESS]
-Mounting /dev :[SUCCESS]
-Mounting /dev/pts :[    2.615036] mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+Mounting /dev/pts :[    2.625248] mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 [SUCCESS]
-Mounting /sys/kernel/debug :[    2.667169] sdhci-tegra 700b0400.sdhci: Got CD GPIO
+Mounting /sys/kernel/debug :[SUCCESS]
+Mounting other filesystems :[    2.673542] sdhci-tegra 700b0400.sdhci: Got CD GPIO
+sdhci-tegra 700b0400.sdhci: Got WP GPIO
 [SUCCESS]
-Mounting other filesystems :[    2.676330] sdhci-tegra 700b0400.sdhci: Got WP GPIO
-mmc1: Unknown controller version (3). You may experience problems.
+Enabling hot-plug :[    2.694197] mmc1: Unknown controller version (3). You may experience problems.
 [SUCCESS]
-Enabling hot-plug :[SUCCESS]
 Starting syslogd :[SUCCESS]
 
 System initialization complete.
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-03-16 22:55:57.521263250 -0700
+++ /tmp/foob	2016-03-16 22:55:57.525263247 -0700
@@ -47,20 +47,20 @@
 	 #################################################################
 	 #################################################################
 	 #################
-	 906.3 KiB/s
+	 907.2 KiB/s
 done
-Bytes transferred = 6926120 (69af28 hex)
+Bytes transferred = 6926984 (69b288 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 896.5 KiB/s
+	 859.4 KiB/s
 done
 Bytes transferred = 21128 (5288 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x69af28 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x69b288 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410287
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.5.0-next-20160316-81627-g8559bac (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Mar 15 23:49:14 PDT 2016
+Linux version 4.5.0-next-20160317-81903-g0a2dc466 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Mar 16 22:48:59 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -325,9 +325,9 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-28 11:49:23 UTC (1419767363)
-mmc0: host does not support reading read-only switch, assuming write-enable
+rtc-em3027 2-0056: setting system clock to 2014-12-28 12:06:23 UTC (1419768383)
 Waiting for root device PARTUUID=11eb23c1-01...
+mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
  mmcblk0: p1
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-03-16 22:55:57.529263243 -0700
+++ /tmp/foob	2016-03-16 22:55:57.529263243 -0700
@@ -44,10 +44,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #####################################
-	 905.3 KiB/s
+	 ######################################
+	 912.1 KiB/s
 done
-Bytes transferred = 5312000 (510e00 hex)
+Bytes transferred = 5314952 (511988 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21128 (5288 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x510e00 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x511988 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410287
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.5.0-next-20160316-81627-g8559bac (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Mar 15 23:48:58 PDT 2016
+Linux version 4.5.0-next-20160317-81903-g0a2dc466 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Mar 16 22:48:53 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -97,7 +97,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab2164   (10921 kB)
+      .text : 0xc0008000 - 0xc0ab2180   (10921 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
       .data : 0xc0c00000 - 0xc0c5df80   ( 376 kB)
        .bss : 0xc0c5f000 - 0xc0cb9a88   ( 363 kB)
@@ -307,17 +307,16 @@
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
 NET: Registered protocol family 10
 mip6: Mobile IPv6
+mmc1: new high speed SD card at address b368
 sit: IPv6 over IPv4 tunneling driver
+mmcblk0: mmc1:b368 NCard 1.83 GiB (ro)
+ mmcblk0: p1 p2
 NET: Registered protocol family 17
-NET: Registered protocol family 15
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
-mmc1: new high speed SD card at address b368
+mmcblk1: mmc0:b368 NCard 3.70 GiB 
+NET: Registered protocol family 15
 can: controller area network core (rev 20120528 abi 9)
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
- mmcblk0: p1
- mmcblk1: p1 p2
 NET: Registered protocol family 29
 can: raw protocol (rev 20120528)
 can: broadcast manager protocol (rev 20120528 t)
@@ -330,7 +329,6 @@
 Bluetooth: HIDP socket layer initialized
 Key type dns_resolver registered
 Registering SWP/SWPB emulation handler
-hub 1-1:1.0: USB hub found
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
@@ -339,15 +337,17 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-28 11:47:52 UTC (1419767272)
+rtc-em3027 2-0056: setting system clock to 2014-12-28 12:04:56 UTC (1419768296)
 ALSA device list:
   #0: tegra-trimslice
+hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
-EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
+ mmcblk1: p1
+EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
-EXT4-fs (mmcblk0p1): recovery complete
-EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:1.
+EXT4-fs (mmcblk1p1): recovery complete
+EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
+VFS: Mounted root (ext3 filesystem) on device 179:17.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2016-03-16 22:55:57.537263238 -0700
+++ /tmp/foob	2016-03-16 22:55:57.537263238 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 775754 us
-Basic modules init took 349171 us
-Sec Bootdevice Read Time = 203 ms, Read Size = 8459 KB
-Next stage binary read took 14912 us
-Carveout took 252775 us
-CPU initialization took 92482 us
-Total time taken by TegraBoot 709340 us
+Tegraboot started after 857665 us
+Basic modules init took 350386 us
+Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
+Next stage binary read took 14446 us
+Carveout took 252768 us
+CPU initialization took 92624 us
+Total time taken by TegraBoot 710224 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.615] LPDDR4 Training: Number of tables = 10
+64b[0001.696] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=/dev/mmcblk0p1 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-11423744 bytes read in 716 ms (15.2 MiB/s)
+11423744 bytes read in 718 ms (15.2 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
 37136 bytes read in 453 ms (79.1 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 494 ms (3.1 MiB/s)
+1627787 bytes read in 495 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.5.0-next-20160316+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Mar 15 23:50:37 PDT 2016
+Linux version 4.5.0-next-20160317+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Mar 16 22:50:49 PDT 2016
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -149,7 +149,7 @@
 psci: PSCIv0.2 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: MIGRATE_INFO_TYPE not supported.
-percpu: Embedded 20 pages/cpu @ffffffc0bff5b000 s43008 r8192 d30720 u81920
+percpu: Embedded 20 pages/cpu @ffffffc0bff5a000 s43008 r8192 d30720 u81920
 pcpu-alloc: s43008 r8192 d30720 u81920 alloc=20*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Detected PIPT I-cache on CPU0
@@ -166,14 +166,14 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffffffc07a000000-ffffffc07dffffff]
-Memory: 2981568K/3131392K available (6840K kernel code, 616K rwdata, 2880K rodata, 812K init, 241K bss, 133440K reserved, 16384K cma-reserved)
+Memory: 2981564K/3131392K available (6844K kernel code, 612K rwdata, 2880K rodata, 812K init, 241K bss, 133444K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffffff8000000000 - 0xffffff8008000000   (   128 MB)
     vmalloc : 0xffffff8008000000 - 0xffffffbdbfff0000   (   246 GB)
-      .text : 0xffffff8008080000 - 0xffffff800872c000   (  6832 KB)
-    .rodata : 0xffffff800872c000 - 0xffffff8008a00000   (  2896 KB)
-      .init : 0xffffff8008a00000 - 0xffffff8008acb000   (   812 KB)
-      .data : 0xffffff8008acb000 - 0xffffff8008b65000   (   616 KB)
+      .text : 0xffffff8008080000 - 0xffffff800872d000   (  6836 KB)
+    .rodata : 0xffffff800872d000 - 0xffffff8008a01000   (  2896 KB)
+      .init : 0xffffff8008a01000 - 0xffffff8008acc000   (   812 KB)
+      .data : 0xffffff8008acc000 - 0xffffff8008b65000   (   612 KB)
     vmemmap : 0xffffffbdc0000000 - 0xffffffbfc0000000   (     8 GB maximum)
               0xffffffbdc0000000 - 0xffffffbdc3000000   (    48 MB actual)
     fixed   : 0xffffffbffe7fd000 - 0xffffffbffec00000   (  4108 KB)
@@ -222,7 +222,7 @@
 pinctrl core: initialized pinctrl subsystem
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffffff8008732000, 1 data @ ffffff8008ad0000)
+vdso: 2 pages (1 code @ ffffff8008733000, 1 data @ ffffff8008ad0000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -259,7 +259,7 @@
 kvm [1]: timer IRQ3
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.439:1): initialized
+audit: type=2000 audit(0.435:1): initialized
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -330,7 +330,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 812K (ffffff8008a00000 - ffffff8008acb000)
+Freeing unused kernel memory: 812K (ffffff8008a01000 - ffffff8008acc000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-03-16 22:55:57.545263231 -0700
+++ /tmp/foob	2016-03-16 22:55:57.545263231 -0700
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 #################
-	 3.4 MiB/s
+	 2.8 MiB/s
 done
-Bytes transferred = 6926120 (69af28 hex)
+Bytes transferred = 6926984 (69b288 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.5 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x69af28 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x69b288 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.5.0-next-20160316-81627-g8559bac (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Mar 15 23:49:14 PDT 2016
+Linux version 4.5.0-next-20160317-81903-g0a2dc466 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Mar 16 22:48:59 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -285,10 +285,10 @@
 mmcblk0: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk0boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk0boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
+tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 mmcblk0rpmb: mmc1:0001 SEM16G partition 3 128 KiB
 Primary GPT is invalid, using alternate GPT.
  mmcblk0: p1 p2 p3 p4 p5
-tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -362,4 +362,5 @@
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.743565] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# 
\ No newline at end of file
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-03-16 22:55:57.553263225 -0700
+++ /tmp/foob	2016-03-16 22:55:57.553263225 -0700
@@ -36,21 +36,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #####################################
-	 3.4 MiB/s
+	 ######################################
+	 3.3 MiB/s
 done
-Bytes transferred = 5312000 (510e00 hex)
+Bytes transferred = 5314952 (511988 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.3 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x510e00 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x511988 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.5.0-next-20160316-81627-g8559bac (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Mar 15 23:48:58 PDT 2016
+Linux version 4.5.0-next-20160317-81903-g0a2dc466 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Mar 16 22:48:53 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -89,7 +89,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0ab2164   (10921 kB)
+      .text : 0xc0008000 - 0xc0ab2180   (10921 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
       .data : 0xc0c00000 - 0xc0c5df80   ( 376 kB)
        .bss : 0xc0c5f000 - 0xc0cb9a88   ( 363 kB)
@@ -360,7 +360,7 @@
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K (c0b00000 - c0c00000)
-bash: cannot set terminal process group (-1): In[    4.170466] usb 2-1: new high-speed USB device number 2 using tegra-ehci
-appropriate ioctl for device
-bash: no job control in this shell
+bash: cannot set terminal process group (-1): Inappropriate ioctl for device
+ba[    4.190495] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+sh: no job control in this shell
 root@(none):/# 
\ No newline at end of file
