Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2017-02-15 20:58:04.211851381 -0800
+++ /tmp/foob	2017-02-15 20:58:04.215851397 -0800
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc8-next-20170215-177550-gfee922e (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Feb 14 22:22:23 PST 2017
+Linux version 4.10.0-rc8-next-20170216-177814-g38161d2 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Feb 15 20:52:00 PST 2017
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -29,7 +29,7 @@
 Policy zone: DMA
 Kernel command line: root=/dev/vda2 rw rootwait mem=1024M console=ttyAMA0,38400n8 init=/bin/bash
 PID hash table entries: 4096 (order: 3, 32768 bytes)
-Memory: 1000124K/1048576K available (8700K kernel code, 964K rwdata, 4200K rodata, 1088K init, 398K bss, 32068K reserved, 16384K cma-reserved)
+Memory: 1000124K/1048576K available (8700K kernel code, 964K rwdata, 4200K rodata, 1088K init, 396K bss, 32068K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
@@ -37,7 +37,7 @@
     .rodata : 0xffff000008900000 - 0xffff000008d20000   (  4224 KB)
       .init : 0xffff000008d20000 - 0xffff000008e30000   (  1088 KB)
       .data : 0xffff000008e30000 - 0xffff000008f21200   (   965 KB)
-       .bss : 0xffff000008f21200 - 0xffff000008f84a38   (   399 KB)
+       .bss : 0xffff000008f21200 - 0xffff000008f845b8   (   397 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -114,7 +114,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.688:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.664:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -180,7 +180,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2017-02-15 06:23:03 UTC (1487139783)
+rtc-pl031 9010000.pl031: setting system clock to 2017-02-16 04:52:41 UTC (1487220761)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-02-15 20:58:04.219851412 -0800
+++ /tmp/foob	2017-02-15 20:58:04.223851428 -0800
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra114 (Dalmore) # 
 Tegra114 (Dalmore) # usb reset
 (Re)start USB...
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 ##########################################################
-	 3.3 MiB/s
+	 2.8 MiB/s
 done
-Bytes transferred = 7518048 (72b760 hex)
+Bytes transferred = 7517848 (72b698 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.6 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x72b760 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x72b698 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc8-next-20170215-153117-gfee922e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Feb 14 22:19:05 PST 2017
+Linux version 4.10.0-rc8-next-20170216-153381-g38161d2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Feb 15 20:49:00 PST 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -71,7 +71,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c153a680, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c153a640, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -84,7 +84,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993764K/2096128K available (10240K kernel code, 1263K rwdata, 4204K rodata, 2048K init, 536K bss, 36828K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993764K/2096128K available (10240K kernel code, 1263K rwdata, 4208K rodata, 2048K init, 536K bss, 36828K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -94,8 +94,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc153bfa0   (1264 kB)
-       .bss : 0xc153d000 - 0xc15c31f8   ( 537 kB)
+      .data : 0xc1400000 - 0xc153bf60   (1264 kB)
+       .bss : 0xc153d000 - 0xc15c3238   ( 537 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU debugfs-based tracing is enabled.
@@ -304,19 +304,19 @@
 Key type dns_resolver registered
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
-mmc0: new high speed SDHC card at address aaaa
 usb1_vbus: supplied by vdd-sys-5v0
 usb2_vbus: supplied by vdd-sys-5v0
+mmc0: new high speed SDHC card at address aaaa
 VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
-mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
+mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
+ mmcblk0: p1 p2 p3
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
- mmcblk0: p1 p2 p3
  mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
 Console: switching to colour frame buffer device 240x75
 drm drm: fb0:  frame buffer device
@@ -328,7 +328,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-12-09 17:53:48 UTC (976384428)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-12-10 16:23:30 UTC (976465410)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -340,4 +340,4 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.519444] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.499439] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2017-02-15 20:58:04.227851443 -0800
+++ /tmp/foob	2017-02-15 20:58:04.231851458 -0800
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 ######################################################
-	 3.1 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 5555424 (54c4e0 hex)
+Bytes transferred = 5558128 (54cf70 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.8 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x54c4e0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x54cf70 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc8-next-20170215-153117-gfee922e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Feb 14 22:18:59 PST 2017
+Linux version 4.10.0-rc8-next-20170216-153381-g38161d2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Feb 15 20:48:49 PST 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0d5ec00, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0d5ebc0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -92,8 +92,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d600e0   ( 385 kB)
-       .bss : 0xc0d62000 - 0xc0dbe66c   ( 370 kB)
+      .data : 0xc0d00000 - 0xc0d600a0   ( 385 kB)
+       .bss : 0xc0d62000 - 0xc0dbe6ac   ( 370 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -321,7 +321,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-12-09 17:51:03 UTC (976384263)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-12-10 16:20:51 UTC (976465251)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -335,4 +335,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.750588] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.761335] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-02-15 20:58:04.235851474 -0800
+++ /tmp/foob	2017-02-15 20:58:04.239851490 -0800
@@ -36,7 +36,7 @@
 	 ##########################################################
 	 4 MiB/s
 done
-Bytes transferred = 7518048 (72b760 hex)
+Bytes transferred = 7517848 (72b698 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -47,7 +47,7 @@
 done
 Bytes transferred = 67133 (1063d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x72b760 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x72b698 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301363c
@@ -55,7 +55,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc8-next-20170215-153117-gfee922e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Feb 14 22:19:05 PST 2017
+Linux version 4.10.0-rc8-next-20170216-153381-g38161d2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Feb 15 20:49:00 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -66,7 +66,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c153a680, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c153a640, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -79,7 +79,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993644K/2096128K available (10240K kernel code, 1263K rwdata, 4204K rodata, 2048K init, 536K bss, 36948K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993644K/2096128K available (10240K kernel code, 1263K rwdata, 4208K rodata, 2048K init, 536K bss, 36948K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -89,8 +89,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc153bfa0   (1264 kB)
-       .bss : 0xc153d000 - 0xc15c31f8   ( 537 kB)
+      .data : 0xc1400000 - 0xc153bf60   (1264 kB)
+       .bss : 0xc153d000 - 0xc15c3238   ( 537 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU debugfs-based tracing is enabled.
@@ -313,12 +313,6 @@
 +5V_SATA: supplied by +5V_SYS
 +12V_SATA: supplied by +VDD_MUX
  usb2-0: usb2-0 supply vbus not found, using dummy regulator
-mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
- mmcblk0: p1
  usb2-1: usb2-1 supply vbus not found, using dummy regulator
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
@@ -343,16 +337,22 @@
 as3722-ldo10: supplied by +5V_SYS
 +2.8V_RUN_CAM_AF: Bringing 0uV into 2800000-2800000uV
 as3722-ldo11: supplied by +3.3V_RUN
+mmc0: new high speed MMC card at address 0001
 +1.8V_RUN_VPP_FUSE: Bringing 0uV into 1800000-1800000uV
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 tegra-ahci 70027000.sata: AHCI 0001.0301 32 slots 2 ports 3 Gbps 0x1 impl platform mode
 tegra-ahci 70027000.sata: flags: 64bit ncq sntf pm led pio slum part sadm sds apst 
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 scsi host0: tegra-ahci
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
 scsi host1: tegra-ahci
 ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 345
 ata2: DUMMY
 tegra-ehci 7d008000.usb: EHCI Host Controller
 tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci 7d008000.usb: irq 355, io mem 0x7d008000
+ mmcblk0: p1
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
@@ -372,10 +372,12 @@
 pci_bus 0000:00: root bus resource [mem 0x20000000-0x3fffffff pref]
 pci_bus 0000:00: root bus resource [bus 00-ff]
 pci 0000:00:01.0: [10de:0e12] type 01 class 0x060400
+pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x88 may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x52 may corrupt adjacent RW1C bits
 pci 0000:00:01.0: PME# supported from D0 D1 D2 D3hot D3cold
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x4c may corrupt adjacent RW1C bits
 pci 0000:00:02.0: [10de:0e13] type 01 class 0x060400
+pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x88 may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x52 may corrupt adjacent RW1C bits
 pci 0000:00:02.0: PME# supported from D0 D1 D2 D3hot D3cold
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4c may corrupt adjacent RW1C bits
@@ -384,8 +386,6 @@
 ata1: SATA link down (SStatus 0 SControl 300)
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x4 may corrupt adjacent RW1C bits
 pci_bus 0000:00: 1-byte config write to 0000:00:01.0 offset 0xc may corrupt adjacent RW1C bits
-pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4 may corrupt adjacent RW1C bits
-pci_bus 0000:00: 1-byte config write to 0000:00:02.0 offset 0xc may corrupt adjacent RW1C bits
 PCI: bus0: Fast back to back transfers disabled
 pci 0000:00:01.0: bridge configuration invalid ([bus 00-00]), reconfiguring
 pci 0000:00:02.0: bridge configuration invalid ([bus 00-00]), reconfiguring
@@ -393,7 +393,6 @@
 pci 0000:01:00.0: reg 0x10: [mem 0x00000000-0x00001fff 64bit]
 pci 0000:01:00.0: PME# supported from D0 D3hot D3cold
 mmc1: new high speed SDHC card at address aaaa
-mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 PCI: bus1: Fast back to back transfers disabled
 pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
 pci 0000:02:00.0: [10ec:8168] type 00 class 0x020000
@@ -402,6 +401,7 @@
 pci 0000:02:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
 pci 0000:02:00.0: supports D1 D2
 pci 0000:02:00.0: PME# supported from D0 D1 D2 D3hot D3cold
+mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
  mmcblk1: p1 p2 p3
 PCI: bus2: Fast back to back transfers disabled
 pci_bus 0000:02: busn_res: [bus 02-ff] end is updated to 02
@@ -427,7 +427,7 @@
 pcieport 0000:00:02.0: Signaling PME with IRQ 394
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
-r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0b54000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 395
+r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0b53000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 395
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
@@ -440,7 +440,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:35 UTC (946684955)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:08 UTC (946684928)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2017-02-15 20:58:04.243851505 -0800
+++ /tmp/foob	2017-02-15 20:58:04.247851520 -0800
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MMC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -32,9 +32,9 @@
 	 #################################################################
 	 #################################################################
 	 ######################################################
-	 3.9 MiB/s
+	 4 MiB/s
 done
-Bytes transferred = 5555424 (54c4e0 hex)
+Bytes transferred = 5558128 (54cf70 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -45,7 +45,7 @@
 done
 Bytes transferred = 67133 (1063d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x54c4e0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x54cf70 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301363c
@@ -54,7 +54,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc8-next-20170215-153117-gfee922e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Feb 14 22:18:59 PST 2017
+Linux version 4.10.0-rc8-next-20170216-153381-g38161d2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Feb 15 20:48:49 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -64,7 +64,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0d5ec00, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0d5ebc0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -87,8 +87,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d600e0   ( 385 kB)
-       .bss : 0xc0d62000 - 0xc0dbe66c   ( 370 kB)
+      .data : 0xc0d00000 - 0xc0d600a0   ( 385 kB)
+       .bss : 0xc0d62000 - 0xc0dbe6ac   ( 370 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -379,10 +379,12 @@
 pci_bus 0000:00: root bus resource [mem 0x20000000-0x3fffffff pref]
 pci_bus 0000:00: root bus resource [bus 00-ff]
 pci 0000:00:01.0: [10de:0e12] type 01 class 0x060400
+pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x88 may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x52 may corrupt adjacent RW1C bits
 pci 0000:00:01.0: PME# supported from D0 D1 D2 D3hot D3cold
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x4c may corrupt adjacent RW1C bits
 pci 0000:00:02.0: [10de:0e13] type 01 class 0x060400
+pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x88 may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x52 may corrupt adjacent RW1C bits
 pci 0000:00:02.0: PME# supported from D0 D1 D2 D3hot D3cold
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4c may corrupt adjacent RW1C bits
@@ -390,8 +392,6 @@
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x3e may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x4 may corrupt adjacent RW1C bits
 pci_bus 0000:00: 1-byte config write to 0000:00:01.0 offset 0xc may corrupt adjacent RW1C bits
-pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4 may corrupt adjacent RW1C bits
-pci_bus 0000:00: 1-byte config write to 0000:00:02.0 offset 0xc may corrupt adjacent RW1C bits
 PCI: bus0: Fast back to back transfers disabled
 pci 0000:00:01.0: bridge configuration invalid ([bus 00-00]), reconfiguring
 pci 0000:00:02.0: bridge configuration invalid ([bus 00-00]), reconfiguring
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-02-15 20:58:04.251851536 -0800
+++ /tmp/foob	2017-02-15 20:58:04.255851550 -0800
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -42,9 +42,9 @@
 	 #################################################################
 	 #################################################################
 	 ##########################################################
-	 3.5 MiB/s
+	 3.6 MiB/s
 done
-Bytes transferred = 7518048 (72b760 hex)
+Bytes transferred = 7517848 (72b698 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -55,7 +55,7 @@
 done
 Bytes transferred = 68155 (10a3b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x72b760 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x72b698 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a3a
@@ -63,7 +63,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc8-next-20170215-153117-gfee922e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Feb 14 22:19:05 PST 2017
+Linux version 4.10.0-rc8-next-20170216-153381-g38161d2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Feb 15 20:49:00 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -75,7 +75,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c153a680, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c153a640, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -88,7 +88,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1994660K/2097148K available (10240K kernel code, 1263K rwdata, 4204K rodata, 2048K init, 536K bss, 36952K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1994660K/2097148K available (10240K kernel code, 1263K rwdata, 4208K rodata, 2048K init, 536K bss, 36952K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -98,8 +98,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc153bfa0   (1264 kB)
-       .bss : 0xc153d000 - 0xc15c31f8   ( 537 kB)
+      .data : 0xc1400000 - 0xc153bf60   (1264 kB)
+       .bss : 0xc153d000 - 0xc15c3238   ( 537 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU debugfs-based tracing is enabled.
@@ -294,7 +294,9 @@
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc0: Unknown controller version (3). You may experience problems.
 mmc0: Invalid maximum block size, assuming 512 bytes
+elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
 mmc0: SDHCI controller on 700b0600.sdhci [700b0600.sdhci] using ADMA
+input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 ledtrig-cpu: registered to indicate activity on CPUs
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
@@ -303,7 +305,6 @@
 sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
 NET: Registered protocol family 17
 can: controller area network core (rev 20120528 abi 9)
-elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
 NET: Registered protocol family 29
 can: raw protocol (rev 20120528)
 can: broadcast manager protocol (rev 20161123 t)
@@ -323,6 +324,7 @@
 +3.3V_PANEL: supplied by +3.3V_RUN
 +3.3V_LP0: supplied by +3.3V_SYS
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
+elan_i2c 1-0015: failed to read report data: -121
 +5V_HDMI_CON: supplied by +5V_SYS
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
  usb3-1: usb3-1 supply vbus not found, using dummy regulator
@@ -334,18 +336,18 @@
 mmc1: Switching to 3.3V signalling voltage failed
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
-mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
-mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
-mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
 mmc1: SDHCI controller on 700b0000.sdhci [700b0000.sdhci] using ADMA
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc2: Unknown controller version (3). You may experience problems.
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 mmc2: Invalid maximum block size, assuming 512 bytes
+mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
+mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
+mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
+elan_i2c 1-0015: invalid report id data (ff)
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 mmc1: new high speed SDIO card at address 0001
 mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
-input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
 mmc2: new high speed SDHC card at address 0007
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2017-02-15 20:58:04.259851566 -0800
+++ /tmp/foob	2017-02-15 20:58:04.263851583 -0800
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -40,9 +40,9 @@
 	 #################################################################
 	 #################################################################
 	 ######################################################
-	 3.4 MiB/s
+	 3.5 MiB/s
 done
-Bytes transferred = 5555424 (54c4e0 hex)
+Bytes transferred = 5558128 (54cf70 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -53,7 +53,7 @@
 done
 Bytes transferred = 68155 (10a3b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x54c4e0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x54cf70 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a3a
@@ -62,7 +62,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc8-next-20170215-153117-gfee922e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Feb 14 22:18:59 PST 2017
+Linux version 4.10.0-rc8-next-20170216-153381-g38161d2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Feb 15 20:48:49 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -73,7 +73,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c0d5ec00, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0d5ebc0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -96,8 +96,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d600e0   ( 385 kB)
-       .bss : 0xc0d62000 - 0xc0dbe66c   ( 370 kB)
+      .data : 0xc0d00000 - 0xc0d600a0   ( 385 kB)
+       .bss : 0xc0d62000 - 0xc0dbe6ac   ( 370 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -293,11 +293,13 @@
 Segment Routing with IPv6
 mip6: Mobile IPv6
 mmc0: new high speed MMC card at address 0001
-sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
 mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
+sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
+NET: Registered protocol family 17
 mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
-NET: Registered protocol family 17
+mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 NET: Registered protocol family 15
 can: controller area network core (rev 20120528 abi 9)
 NET: Registered protocol family 29
@@ -329,9 +331,7 @@
  usb3-1: usb3-1 supply vbus not found, using dummy regulator
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
-mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
 tegra-xusb 70090000.usb: Firmware timestamp: 2014-09-16 02:10:07 UTC
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 tegra-xusb 70090000.usb: xHCI Host Controller
 tegra-xusb 70090000.usb: new USB bus registered, assigned bus number 1
 tegra-xusb 70090000.usb: hcc params 0x0184f525 hci version 0x100 quirks 0x00010010
@@ -356,6 +356,8 @@
 mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
+usb 1-2: new high-speed USB device number 2 using tegra-xusb
+input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input3
 mmc2: new high speed SDHC card at address 0007
 mmcblk2: mmc2:0007 SD08G 7.42 GiB 
 GPT:Primary header alternate_lba != Alt. header my_lba
@@ -364,9 +366,7 @@
 GPT:3014591 != 15564799
 GPT: Use GNU Parted to correct GPT errors.
  mmcblk2: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
-usb 1-2: new high-speed USB device number 2 using tegra-xusb
 Console: switching to colour frame buffer device 170x48
-input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input3
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input4
@@ -383,12 +383,12 @@
   #1: GoogleNyanBig
 EXT4-fs (mmcblk0p3): couldn't mount as ext3 due to feature incompatibilities
 uvcvideo: Found UVC 1.00 device HD WebCam (1bcf:2c67)
-EXT4-fs (mmcblk0p3): recovery complete
 input: HD WebCam as /devices/soc0/70090000.usb/usb1/1-2/1-2:1.0/input/input5
+EXT4-fs (mmcblk0p3): recovery complete
 EXT4-fs (mmcblk0p3): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:3.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.894054] usb 1-3: new high-speed USB device number 3 using tegra-xusb
+root@(none):/# [    2.914550] usb 1-3: new high-speed USB device number 3 using tegra-xusb
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2017-02-15 20:58:04.267851598 -0800
+++ /tmp/foob	2017-02-15 20:58:04.271851613 -0800
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1800568 us
+Bootloader-Cpu Init at (time stamp): 1801350 us
 
 Pinmux changes applied in kernel way
 
@@ -62,20 +62,20 @@
 Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1800542  us
-Bootloader (AosInit) time: 733490 us
-Bootloader Main Init time: 180181 us
-BlockDevMgrInit time: 157552 us
+Miscellaneous avp time: 1801324  us
+Bootloader (AosInit) time: 733471 us
+Bootloader Main Init time: 180040 us
+BlockDevMgrInit time: 157519 us
 PartMgrInit time: 8 us
 StorMgrInit time: 30 us
-FsMgrInit time: 3 us
-Partition Table load: 77008 us 
-Display init time: 574612 us
+FsMgrInit time: 2 us
+Partition Table load: 77039 us 
+Display init time: 574506 us
 Kernel read time: 378 msec
 Load Kernel Size:   16304128 Bytes
 Bl Kernel ReadSpeed: 43132000 Bytes/sec
-Pre-Kernel Jump time: 1740612 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5844632 us
+Pre-Kernel Jump time: 1740284 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5845359 us
 Booting Linux on physical CPU 0x0
 Linux version 4.8.0-239303-g3477d16 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Fri Oct 7 01:12:37 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
@@ -184,7 +184,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(1.147:1): initialized
+audit: type=2000 audit(1.131:1): initialized
 workingset: timestamp_bits=46 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -253,9 +253,9 @@
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 +VDD_LED: supplied by +VDD_MUX
-+5V_USB_HS: supplied by +5V_SYS
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
++5V_USB_HS: supplied by +5V_SYS
 +5V_USB_SS: supplied by +5V_SYS
 +5V_HDMI_CON: supplied by +5V_SYS
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-02-15 20:58:04.275851628 -0800
+++ /tmp/foob	2017-02-15 20:58:04.279851643 -0800
@@ -49,7 +49,7 @@
 	 ##########################################################
 	 4.1 MiB/s
 done
-Bytes transferred = 7518048 (72b760 hex)
+Bytes transferred = 7517848 (72b698 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -60,7 +60,7 @@
 done
 Bytes transferred = 21426 (53b2 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x72b760 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x72b698 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103b1
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc8-next-20170215-153117-gfee922e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Feb 14 22:19:05 PST 2017
+Linux version 4.10.0-rc8-next-20170216-153381-g38161d2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Feb 15 20:49:00 PST 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -78,7 +78,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c153a680, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c153a640, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -91,7 +91,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 954584K/1048576K available (10240K kernel code, 1263K rwdata, 4204K rodata, 2048K init, 536K bss, 28456K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 954584K/1048576K available (10240K kernel code, 1263K rwdata, 4208K rodata, 2048K init, 536K bss, 28456K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -101,8 +101,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc153bfa0   (1264 kB)
-       .bss : 0xc153d000 - 0xc15c31f8   ( 537 kB)
+      .data : 0xc1400000 - 0xc153bf60   (1264 kB)
+       .bss : 0xc153d000 - 0xc15c3238   ( 537 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	RCU debugfs-based tracing is enabled.
@@ -195,6 +195,7 @@
 pci_bus 0000:00: root bus resource [mem 0xa8000000-0xbfffffff pref]
 pci_bus 0000:00: root bus resource [bus 00-ff]
 pci 0000:00:01.0: [10de:0bf0] type 01 class 0x060000
+pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x88 may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x52 may corrupt adjacent RW1C bits
 pci 0000:00:01.0: PME# supported from D0 D1 D2 D3hot D3cold
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x4c may corrupt adjacent RW1C bits
@@ -212,7 +213,6 @@
 pci 0000:01:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit pref]
 pci 0000:01:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
 pci 0000:01:00.0: reg 0x30: [mem 0x00000000-0x0001ffff pref]
-pci_bus 0000:01: 2-byte config write to 0000:01:00.0 offset 0x52 may corrupt adjacent RW1C bits
 pci 0000:01:00.0: supports D1 D2
 pci 0000:01:00.0: PME# supported from D0 D1 D2 D3hot D3cold
 PCI: bus1: Fast back to back transfers disabled
@@ -343,7 +343,7 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2028-02-01 01:01:16 UTC (1832979676)
+rtc-em3027 2-0056: hctosys: unable to read the hardware clock
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2017-02-15 20:58:04.283851659 -0800
+++ /tmp/foob	2017-02-15 20:58:04.287851675 -0800
@@ -47,7 +47,7 @@
 	 ######################################################
 	 4.1 MiB/s
 done
-Bytes transferred = 5555424 (54c4e0 hex)
+Bytes transferred = 5558128 (54cf70 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21426 (53b2 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x54c4e0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x54cf70 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103b1
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc8-next-20170215-153117-gfee922e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Feb 14 22:18:59 PST 2017
+Linux version 4.10.0-rc8-next-20170216-153381-g38161d2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Feb 15 20:48:49 PST 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -76,7 +76,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0d5ec00, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0d5ebc0, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -99,8 +99,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d600e0   ( 385 kB)
-       .bss : 0xc0d62000 - 0xc0dbe66c   ( 370 kB)
+      .data : 0xc0d00000 - 0xc0d600a0   ( 385 kB)
+       .bss : 0xc0d62000 - 0xc0dbe6ac   ( 370 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
@@ -192,6 +192,7 @@
 pci_bus 0000:00: root bus resource [mem 0xa8000000-0xbfffffff pref]
 pci_bus 0000:00: root bus resource [bus 00-ff]
 pci 0000:00:01.0: [10de:0bf0] type 01 class 0x060000
+pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x88 may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x52 may corrupt adjacent RW1C bits
 pci 0000:00:01.0: PME# supported from D0 D1 D2 D3hot D3cold
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x4c may corrupt adjacent RW1C bits
@@ -209,7 +210,6 @@
 pci 0000:01:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit pref]
 pci 0000:01:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
 pci 0000:01:00.0: reg 0x30: [mem 0x00000000-0x0001ffff pref]
-pci_bus 0000:01: 2-byte config write to 0000:01:00.0 offset 0x52 may corrupt adjacent RW1C bits
 pci 0000:01:00.0: supports D1 D2
 pci 0000:01:00.0: PME# supported from D0 D1 D2 D3hot D3cold
 PCI: bus1: Fast back to back transfers disabled
@@ -319,9 +319,9 @@
 mmc1: new high speed SD card at address b368
 mmc0: new high speed SDHC card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
- mmcblk1: p1 p2
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
  mmcblk0: p1
+ mmcblk1: p1 p2
 mip6: Mobile IPv6
 sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
 NET: Registered protocol family 17
@@ -349,7 +349,7 @@
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2028-02-10 00:00:15 UTC (1833753615)
+rtc-em3027 2-0056: hctosys: unable to read the hardware clock
 ALSA device list:
   #0: tegra-trimslice
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2017-02-15 20:58:04.291851691 -0800
+++ /tmp/foob	2017-02-15 20:58:04.295851706 -0800
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 755756 us
-Basic modules init took 349512 us
-Sec Bootdevice Read Time = 204 ms, Read Size = 8459 KB
-Next stage binary read took 15703 us
-Carveout took 252732 us
-CPU initialization took 92328 us
-Total time taken by TegraBoot 710275 us
+Tegraboot started after 848777 us
+Basic modules init took 348886 us
+Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
+Next stage binary read took 14381 us
+Carveout took 252717 us
+CPU initialization took 92332 us
+Total time taken by TegraBoot 708316 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.596] LPDDR4 Training: Number of tables = 10
+64b[0001.685] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-15340032 bytes read in 804 ms (18.2 MiB/s)
+15340032 bytes read in 803 ms (18.2 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-44882 bytes read in 476 ms (91.8 KiB/s)
+44882 bytes read in 474 ms (91.8 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 495 ms (3.1 MiB/s)
+1627787 bytes read in 494 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc8-next-20170215-177550-gfee922e (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Feb 14 22:22:23 PST 2017
+Linux version 4.10.0-rc8-next-20170216-177814-g38161d2 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Feb 15 20:52:00 PST 2017
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -174,7 +174,7 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffff80007a000000-ffff80007dffffff]
-Memory: 2983712K/3131392K available (8700K kernel code, 964K rwdata, 4200K rodata, 1088K init, 398K bss, 131296K reserved, 16384K cma-reserved)
+Memory: 2983712K/3131392K available (8700K kernel code, 964K rwdata, 4200K rodata, 1088K init, 396K bss, 131296K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
@@ -182,7 +182,7 @@
     .rodata : 0xffff000008900000 - 0xffff000008d20000   (  4224 KB)
       .init : 0xffff000008d20000 - 0xffff000008e30000   (  1088 KB)
       .data : 0xffff000008e30000 - 0xffff000008f21200   (   965 KB)
-       .bss : 0xffff000008f21200 - 0xffff000008f84a38   (   399 KB)
+       .bss : 0xffff000008f21200 - 0xffff000008f845b8   (   397 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -269,7 +269,7 @@
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 PCI: CLS 0 bytes, default 128
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 808ec000
+kvm [1]: IDMAP page: 808ed000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
@@ -277,7 +277,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.535:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.539:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -357,6 +357,6 @@
 devtmpfs: mounted
 Freeing unused kernel memory: 1088K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
-bash: no job contr[    1.351629] random: fast init done
+bash: no job contr[    1.375837] random: fast init done
 ol in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2017-02-15 20:58:04.299851721 -0800
+++ /tmp/foob	2017-02-15 20:58:04.299851721 -0800
@@ -1,65 +1,4 @@
-2:23 PST 2017
-Boot CPU: AArch64 Processor [411fd071]
-efi: Getting EFI parameters from FDT:
-efi: UEFI not found.
-cma: Reserved 16 MiB at 0x00000000f5400000
-earlycon: uart0 at MMIO 0x0000000070006000 (options '115200n8')
-bootconsole [uart0] enabled
-NUMA: No NUMA configuration found
-NUMA: Faking a node at [mem 0x0000000000000000-0x000000013fffffff]
-NUMA: Adding memblock [0x80000000 - 0xf65fffff] on node 0
-NUMA: Adding memblock [0x100000000 - 0x13fffffff] on node 0
-NUMA: Initmem setup node 0 [mem 0x80000000-0x13fffffff]
-NUMA: NODE_DATA [mem 0x13ffa1400-0x13ffa2eff]
-Zone ranges:
-  DMA      [mem 0x0000000080000000-0x00000000ffffffff]
-  Normal   [mem 0x0000000100000000-0x000000013fffffff]
-Movable zone start for each node
-Early memory node ranges
-  node   0: [mem 0x0000000080000000-0x00000000f65fffff]
-  node   0: [mem 0x0000000100000000-0x000000013fffffff]
-Initmem setup node 0 [mem 0x0000000080000000-0x000000013fffffff]
-psci: probing for conduit method from DT.
-psci: PSCIv1.0 detected in firmware.
-psci: Using standard PSCI v0.2 function IDs
-psci: MIGRATE_INFO_UP_CPU reported invalid physical ID (0x80000000)
-percpu: Embedded 23 pages/cpu @ffff8000bff42000 s56960 r8192 d29056 u94208
-Detected PIPT I-cache on CPU0
-CPU features: enabling workaround for ARM erratum 832075
-CPU features: enabling workaround for ARM erratum 834220
-Built 1 zonelists in Node order, mobility grouping on.  Total pages: 735336
-Policy zone: Normal
-Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
-PID hash table entries: 4096 (order: 3, 32768 bytes)
-software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2837900K/2988032K available (8700K kernel code, 964K rwdata, 4200K rodata, 1088K init, 398K bss, 133748K reserved, 16384K cma-reserved)
-Virtual kernel memory layout:
-    modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
-    vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff000008900000   (  8704 KB)
-    .rodata : 0xffff000008900000 - 0xffff000008d20000   (  4224 KB)
-      .init : 0xffff000008d20000 - 0xffff000008e30000   (  1088 KB)
-      .data : 0xffff000008e30000 - 0xffff000008f21200   (   965 KB)
-       .bss : 0xffff000008f21200 - 0xffff000008f84a38   (   399 KB)
-    fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
-    PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
-    vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
-              0xffff7e0000000000 - 0xffff7e0003000000   (    48 MB actual)
-    memory  : 0xffff800000000000 - 0xffff8000c0000000   (  3072 MB)
-SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
-Preemptible hierarchical RCU implementation.
-	Build-time adjustment of leaf fanout to 64.
-	RCU restricting CPUs from NR_CPUS=64 to nr_cpu_ids=4.
-RCU: Adjusting geometry for rcu_fanout_leaf=64, nr_cpu_ids=4
-NR_IRQS:64 nr_irqs:64 0
-GIC: Using split EOI/Deactivate mode
-/interrupt-controller@60004000: 192 interrupts forwarded to /interrupt-controller@50041000
-boot misc0 0x0: expected 0x80000
- (comparison mask = 0x400ffffb)
-boot misc2 0x1f720005: expected 0x1f720f05
- (comparison mask = 0xffffff17)
-PLL_C already enabled. Postponing set full defaults
-PLL_MB already enabled. Postponing set full defaults
+ts
 PLL_U already enabled. Postponing set full defaults
 PLL_D already enabled. Postponing set full defaults
 PLL_DP already enabled. Postponing set full defaults
@@ -69,7 +8,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x5588: expected 0x2004f400
+boot misc3 0x59a8: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -82,7 +21,7 @@
 console [tty0] enabled
 bootconsole [uart0] disabled
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc8-next-20170215-177550-gfee922e (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Feb 14 22:22:23 PST 2017
+Linux version 4.10.0-rc8-next-20170216-177814-g38161d2 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Feb 15 20:52:00 PST 2017
 Boot CPU: AArch64 Processor [411fd071]
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
@@ -116,7 +55,7 @@
 Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2837900K/2988032K available (8700K kernel code, 964K rwdata, 4200K rodata, 1088K init, 398K bss, 133748K reserved, 16384K cma-reserved)
+Memory: 2837900K/2988032K available (8700K kernel code, 964K rwdata, 4200K rodata, 1088K init, 396K bss, 133748K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
@@ -124,7 +63,7 @@
     .rodata : 0xffff000008900000 - 0xffff000008d20000   (  4224 KB)
       .init : 0xffff000008d20000 - 0xffff000008e30000   (  1088 KB)
       .data : 0xffff000008e30000 - 0xffff000008f21200   (   965 KB)
-       .bss : 0xffff000008f21200 - 0xffff000008f84a38   (   399 KB)
+       .bss : 0xffff000008f21200 - 0xffff000008f845b8   (   397 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -153,7 +92,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x5588: expected 0x2004f400
+boot misc3 0x59a8: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -235,7 +174,7 @@
 Unpacking initramfs...
 Freeing initrd memory: 2360K
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 808ec000
+kvm [1]: IDMAP page: 808ed000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
@@ -243,7 +182,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.323:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.327:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -272,8 +211,8 @@
 PP2800L_RCAM: Bringing 3100000uV into 2800000-2800000uV
 PP1100_SOC_RTC: Bringing 1000000uV into 850000-850000uV
 PP2800L_FCAM: Bringing 3100000uV into 2800000-2800000uV
-PP_LDO6: Bringing 2800000uV into 1800000-1800000uV
 random: fast init done
+PP_LDO6: Bringing 2800000uV into 1800000-1800000uV
 hisi_sas: driver version v1.6
 libphy: Fixed MDIO Bus: probed
 tun: Universal TUN/TAP device driver, 1.6
@@ -329,15 +268,15 @@
 registered taskstats version 1
 tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 BGND3R 29.1 GiB 
 mmcblk0boot0: mmc0:0001 BGND3R partition 1 4.00 MiB
-max77686-rtc max77620-rtc: setting system clock to 2000-11-26 11:49:58 UTC (975239398)
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
 tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
+tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
+max77686-rtc max77620-rtc: setting system clock to 2000-11-27 10:19:35 UTC (975320375)
 PP1200_AVDD: disabling
 PP1200_RCAM: disabling
 PP2800L_RCAM: disabling
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-02-15 20:58:04.307851752 -0800
+++ /tmp/foob	2017-02-15 20:58:04.307851752 -0800
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 ##########################################################
-	 3.2 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 7518048 (72b760 hex)
+Bytes transferred = 7517848 (72b698 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.3 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 51804 (ca5c hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x72b760 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x72b698 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa5b ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc8-next-20170215-153117-gfee922e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Feb 14 22:19:05 PST 2017
+Linux version 4.10.0-rc8-next-20170216-153381-g38161d2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Feb 15 20:49:00 PST 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -70,7 +70,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c153a680, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c153a640, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -83,7 +83,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993684K/2096128K available (10240K kernel code, 1263K rwdata, 4204K rodata, 2048K init, 536K bss, 36908K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993684K/2096128K available (10240K kernel code, 1263K rwdata, 4208K rodata, 2048K init, 536K bss, 36908K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -93,8 +93,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc153bfa0   (1264 kB)
-       .bss : 0xc153d000 - 0xc15c31f8   ( 537 kB)
+      .data : 0xc1400000 - 0xc153bf60   (1264 kB)
+       .bss : 0xc153d000 - 0xc15c3238   ( 537 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU debugfs-based tracing is enabled.
@@ -306,18 +306,18 @@
 sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
 +VDD_5V_HDMI: supplied by sys_3v3,vdd_3v3_alw
 tegra-pcie 3000.pcie-controller: 2x3 configuration
-tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
 mmc1: new high speed MMC card at address 0001
+mmc0: new ultra high speed DDR50 SDHC card at address aaaa
+mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
 mmcblk1: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB
-mmc0: new ultra high speed DDR50 SDHC card at address aaaa
-mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
 Primary GPT is invalid, using alternate GPT.
  mmcblk1: p1 p2 p3 p4 p5
-tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
  mmcblk0: p1 p2 p3
+tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
+tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -328,6 +328,7 @@
 pci_bus 0000:00: root bus resource [mem 0x28000000-0x3fffffff pref]
 pci_bus 0000:00: root bus resource [bus 00-ff]
 pci 0000:00:01.0: [10de:0e1c] type 01 class 0x060400
+pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x88 may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x52 may corrupt adjacent RW1C bits
 pci 0000:00:01.0: PME# supported from D0 D1 D2 D3hot D3cold
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x4c may corrupt adjacent RW1C bits
@@ -344,7 +345,6 @@
 pci 0000:01:00.0: reg 0x10: [io  0x0000-0x00ff]
 pci 0000:01:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit pref]
 pci 0000:01:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
-pci_bus 0000:01: 2-byte config write to 0000:01:00.0 offset 0x52 may corrupt adjacent RW1C bits
 pci 0000:01:00.0: supports D1 D2
 pci 0000:01:00.0: PME# supported from D0 D1 D2 D3hot D3cold
 PCI: bus1: Fast back to back transfers disabled
@@ -395,5 +395,5 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.104057] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.154001] usb 2-1: new high-speed USB device number 2 using tegra-ehci
 asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0f:67:60
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2017-02-15 20:58:04.315851784 -0800
+++ /tmp/foob	2017-02-15 20:58:04.315851784 -0800
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 ######################################################
-	 3.4 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 5555424 (54c4e0 hex)
+Bytes transferred = 5558128 (54cf70 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.3 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 51804 (ca5c hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x54c4e0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x54cf70 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa5b ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.10.0-rc8-next-20170215-153117-gfee922e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Feb 14 22:18:59 PST 2017
+Linux version 4.10.0-rc8-next-20170216-153381-g38161d2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Feb 15 20:48:49 PST 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,7 +68,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0d5ec00, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0d5ebc0, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -91,8 +91,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
-      .data : 0xc0d00000 - 0xc0d600e0   ( 385 kB)
-       .bss : 0xc0d62000 - 0xc0dbe66c   ( 370 kB)
+      .data : 0xc0d00000 - 0xc0d600a0   ( 385 kB)
+       .bss : 0xc0d62000 - 0xc0dbe6ac   ( 370 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -321,6 +321,7 @@
 pci_bus 0000:00: root bus resource [mem 0x28000000-0x3fffffff pref]
 pci_bus 0000:00: root bus resource [bus 00-ff]
 pci 0000:00:01.0: [10de:0e1c] type 01 class 0x060400
+pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x88 may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x52 may corrupt adjacent RW1C bits
 pci 0000:00:01.0: PME# supported from D0 D1 D2 D3hot D3cold
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x4c may corrupt adjacent RW1C bits
@@ -337,7 +338,6 @@
 pci 0000:01:00.0: reg 0x10: [io  0x0000-0x00ff]
 pci 0000:01:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit pref]
 pci 0000:01:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
-pci_bus 0000:01: 2-byte config write to 0000:01:00.0 offset 0x52 may corrupt adjacent RW1C bits
 pci 0000:01:00.0: supports D1 D2
 pci 0000:01:00.0: PME# supported from D0 D1 D2 D3hot D3cold
 PCI: bus1: Fast back to back transfers disabled
@@ -390,4 +390,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.370203] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.370204] usb 2-1: new high-speed USB device number 2 using tegra-ehci
