Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-01-21 19:54:25.149130486 -0800
+++ /tmp/foob	2016-01-21 19:54:25.149130486 -0800
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-next-20160121+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Jan 20 19:24:15 PST 2016
+Linux version 4.4.0-next-20160122+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Jan 21 19:49:18 PST 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -92,7 +92,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.456:1): initialized
+audit: type=2000 audit(0.448:1): initialized
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
 Key type id_resolver registered
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-01-21 19:54:25.153130483 -0800
+++ /tmp/foob	2016-01-21 19:54:25.157130482 -0800
@@ -41,18 +41,18 @@
 	 ######
 	 3.4 MiB/s
 done
-Bytes transferred = 6760680 (6728e8 hex)
+Bytes transferred = 6761576 (672c68 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6728e8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x672c68 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-next-20160121-67729-g8771b78 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Jan 20 19:22:43 PST 2016
+Linux version 4.4.0-next-20160122-67820-gcccc613 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Jan 21 19:47:55 PST 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -90,7 +90,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0e41de4   (12520 kB)
+      .text : 0xc0208000 - 0xc0e41db4   (12520 kB)
       .init : 0xc0e42000 - 0xc0f4a000   (1056 kB)
       .data : 0xc0f4a000 - 0xc105fc40   (1112 kB)
        .bss : 0xc1062000 - 0xc10b85a0   ( 346 kB)
@@ -295,45 +295,45 @@
 WARNING: CPU: 2 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 2 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160121-67729-g8771b78 #1
+CPU: 2 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160122-67820-gcccc613 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c02188b8>] (unwind_backtrace) from [<c0213f94>] (show_stack+0x10/0x14)
-[<c0213f94>] (show_stack) from [<c04893ec>] (dump_stack+0x78/0x94)
-[<c04893ec>] (dump_stack) from [<c024ab38>] (warn_slowpath_common+0x74/0xb0)
+[<c0213f94>] (show_stack) from [<c048946c>] (dump_stack+0x78/0x94)
+[<c048946c>] (dump_stack) from [<c024ab38>] (warn_slowpath_common+0x74/0xb0)
 [<c024ab38>] (warn_slowpath_common) from [<c024ac08>] (warn_slowpath_fmt+0x30/0x40)
 [<c024ac08>] (warn_slowpath_fmt) from [<c025dc5c>] (check_flush_dependency+0xc8/0x124)
 [<c025dc5c>] (check_flush_dependency) from [<c025dd70>] (flush_work+0x54/0x144)
-[<c025dd70>] (flush_work) from [<c02dab88>] (lru_add_drain_all+0x124/0x180)
-[<c02dab88>] (lru_add_drain_all) from [<c030c6c0>] (migrate_prep+0x8/0x10)
-[<c030c6c0>] (migrate_prep) from [<c02d5ba0>] (alloc_contig_range+0xc4/0x2f8)
-[<c02d5ba0>] (alloc_contig_range) from [<c030dfec>] (cma_alloc+0xd8/0x1a0)
-[<c030dfec>] (cma_alloc) from [<c021e39c>] (__alloc_from_contiguous+0x38/0xdc)
+[<c025dd70>] (flush_work) from [<c02dabfc>] (lru_add_drain_all+0x124/0x180)
+[<c02dabfc>] (lru_add_drain_all) from [<c030c74c>] (migrate_prep+0x8/0x10)
+[<c030c74c>] (migrate_prep) from [<c02d5c14>] (alloc_contig_range+0xc4/0x2f8)
+[<c02d5c14>] (alloc_contig_range) from [<c030e078>] (cma_alloc+0xd8/0x1a0)
+[<c030e078>] (cma_alloc) from [<c021e39c>] (__alloc_from_contiguous+0x38/0xdc)
 [<c021e39c>] (__alloc_from_contiguous) from [<c021ff8c>] (__dma_alloc+0x21c/0x250)
 [<c021ff8c>] (__dma_alloc) from [<c0220044>] (arm_dma_alloc+0x48/0x54)
-[<c0220044>] (arm_dma_alloc) from [<c0306c10>] (dma_pool_alloc+0x1dc/0x210)
-[<c0306c10>] (dma_pool_alloc) from [<c07c7084>] (ehci_qh_alloc+0x3c/0xb4)
-[<c07c7084>] (ehci_qh_alloc) from [<c07cb67c>] (ehci_setup+0x154/0x478)
-[<c07cb67c>] (ehci_setup) from [<c07d1734>] (tegra_ehci_reset+0xc/0x50)
-[<c07d1734>] (tegra_ehci_reset) from [<c07a1afc>] (usb_add_hcd+0x300/0x830)
-[<c07a1afc>] (usb_add_hcd) from [<c07d10ac>] (tegra_ehci_probe+0x3b8/0x504)
-[<c07d10ac>] (tegra_ehci_probe) from [<c0659218>] (platform_drv_probe+0x50/0xac)
-[<c0659218>] (platform_drv_probe) from [<c0657b84>] (driver_probe_device+0x1f4/0x2b0)
-[<c0657b84>] (driver_probe_device) from [<c0656114>] (bus_for_each_drv+0x44/0x8c)
-[<c0656114>] (bus_for_each_drv) from [<c0657908>] (__device_attach+0x9c/0x100)
-[<c0657908>] (__device_attach) from [<c0656f94>] (bus_probe_device+0x84/0x8c)
-[<c0656f94>] (bus_probe_device) from [<c06573c8>] (deferred_probe_work_func+0x60/0x8c)
-[<c06573c8>] (deferred_probe_work_func) from [<c025ebb8>] (process_one_work+0x134/0x338)
+[<c0220044>] (arm_dma_alloc) from [<c0306c9c>] (dma_pool_alloc+0x1dc/0x210)
+[<c0306c9c>] (dma_pool_alloc) from [<c07c7274>] (ehci_qh_alloc+0x3c/0xb4)
+[<c07c7274>] (ehci_qh_alloc) from [<c07cb86c>] (ehci_setup+0x154/0x478)
+[<c07cb86c>] (ehci_setup) from [<c07d1924>] (tegra_ehci_reset+0xc/0x50)
+[<c07d1924>] (tegra_ehci_reset) from [<c07a1cec>] (usb_add_hcd+0x300/0x830)
+[<c07a1cec>] (usb_add_hcd) from [<c07d129c>] (tegra_ehci_probe+0x3b8/0x504)
+[<c07d129c>] (tegra_ehci_probe) from [<c0659308>] (platform_drv_probe+0x50/0xac)
+[<c0659308>] (platform_drv_probe) from [<c0657c74>] (driver_probe_device+0x1f4/0x2b0)
+[<c0657c74>] (driver_probe_device) from [<c0656204>] (bus_for_each_drv+0x44/0x8c)
+[<c0656204>] (bus_for_each_drv) from [<c06579f8>] (__device_attach+0x9c/0x100)
+[<c06579f8>] (__device_attach) from [<c0657084>] (bus_probe_device+0x84/0x8c)
+[<c0657084>] (bus_probe_device) from [<c06574b8>] (deferred_probe_work_func+0x60/0x8c)
+[<c06574b8>] (deferred_probe_work_func) from [<c025ebb8>] (process_one_work+0x134/0x338)
 [<c025ebb8>] (process_one_work) from [<c025ee20>] (worker_thread+0x2c/0x4f0)
 [<c025ee20>] (worker_thread) from [<c0263fac>] (kthread+0xd8/0xf4)
 [<c0263fac>] (kthread) from [<c0210e78>] (ret_from_fork+0x14/0x3c)
----[ end trace 60d12a293cc4b90f ]---
+---[ end trace 208ebf5728283675 ]---
 tegra-ehci 7d008000.usb: irq 339, io mem 0x7d008000
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-01 08:32:09 UTC (946715529)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-02 08:57:24 UTC (946803444)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -345,4 +345,4 @@
 Freeing unused kernel memory: 1056K (c0e42000 - c0f4a000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.499345] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.504221] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-01-21 19:54:25.161130480 -0800
+++ /tmp/foob	2016-01-21 19:54:25.165130478 -0800
@@ -39,18 +39,18 @@
 	 #################################
 	 3.3 MiB/s
 done
-Bytes transferred = 5254600 (502dc8 hex)
+Bytes transferred = 5255208 (503028 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 2.6 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x502dc8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x503028 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-next-20160121-67729-g8771b78 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Jan 20 19:23:04 PST 2016
+Linux version 4.4.0-next-20160122-67820-gcccc613 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Jan 21 19:48:13 PST 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -90,7 +90,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0981738   (9702 kB)
+      .text : 0xc0008000 - 0xc098171c   (9702 kB)
       .init : 0xc0982000 - 0xc09d8000   ( 344 kB)
       .data : 0xc09d8000 - 0xc0a34f20   ( 372 kB)
        .bss : 0xc0a37000 - 0xc0a92d88   ( 368 kB)
@@ -292,45 +292,45 @@
 WARNING: CPU: 0 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160121-67729-g8771b78 #1
+CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160122-67820-gcccc613 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0017208>] (unwind_backtrace) from [<c0012874>] (show_stack+0x10/0x14)
-[<c0012874>] (show_stack) from [<c023c164>] (dump_stack+0x84/0xd0)
-[<c023c164>] (dump_stack) from [<c0026b70>] (warn_slowpath_common+0x74/0xb0)
+[<c0012874>] (show_stack) from [<c023c1a4>] (dump_stack+0x84/0xd0)
+[<c023c1a4>] (dump_stack) from [<c0026b70>] (warn_slowpath_common+0x74/0xb0)
 [<c0026b70>] (warn_slowpath_common) from [<c0026c40>] (warn_slowpath_fmt+0x30/0x40)
 [<c0026c40>] (warn_slowpath_fmt) from [<c00397d4>] (check_flush_dependency+0xc8/0x12c)
 [<c00397d4>] (check_flush_dependency) from [<c00398f0>] (flush_work+0x54/0x11c)
-[<c00398f0>] (flush_work) from [<c00c0884>] (lru_add_drain_all+0x124/0x180)
-[<c00c0884>] (lru_add_drain_all) from [<c00f16fc>] (migrate_prep+0x8/0x10)
-[<c00f16fc>] (migrate_prep) from [<c00bb3fc>] (alloc_contig_range+0xc4/0x2f8)
-[<c00bb3fc>] (alloc_contig_range) from [<c00f326c>] (cma_alloc+0xd8/0x1a0)
-[<c00f326c>] (cma_alloc) from [<c001c270>] (__alloc_from_contiguous+0x38/0xdc)
+[<c00398f0>] (flush_work) from [<c00c08d4>] (lru_add_drain_all+0x124/0x180)
+[<c00c08d4>] (lru_add_drain_all) from [<c00f1764>] (migrate_prep+0x8/0x10)
+[<c00f1764>] (migrate_prep) from [<c00bb44c>] (alloc_contig_range+0xc4/0x2f8)
+[<c00bb44c>] (alloc_contig_range) from [<c00f32d4>] (cma_alloc+0xd8/0x1a0)
+[<c00f32d4>] (cma_alloc) from [<c001c270>] (__alloc_from_contiguous+0x38/0xdc)
 [<c001c270>] (__alloc_from_contiguous) from [<c001cf38>] (__dma_alloc+0x228/0x2ac)
 [<c001cf38>] (__dma_alloc) from [<c001d040>] (arm_dma_alloc+0x48/0x54)
-[<c001d040>] (arm_dma_alloc) from [<c00edb08>] (dma_pool_alloc+0x28c/0x304)
-[<c00edb08>] (dma_pool_alloc) from [<c03fd27c>] (ehci_qh_alloc+0x3c/0xb4)
-[<c03fd27c>] (ehci_qh_alloc) from [<c0401468>] (ehci_setup+0x154/0x478)
-[<c0401468>] (ehci_setup) from [<c04051b0>] (tegra_ehci_reset+0xc/0x50)
-[<c04051b0>] (tegra_ehci_reset) from [<c03e8c60>] (usb_add_hcd+0x2fc/0x804)
-[<c03e8c60>] (usb_add_hcd) from [<c0404b28>] (tegra_ehci_probe+0x3b8/0x504)
-[<c0404b28>] (tegra_ehci_probe) from [<c0336c54>] (platform_drv_probe+0x50/0xac)
-[<c0336c54>] (platform_drv_probe) from [<c03356bc>] (driver_probe_device+0x1f4/0x2b0)
-[<c03356bc>] (driver_probe_device) from [<c0333c50>] (bus_for_each_drv+0x44/0x8c)
-[<c0333c50>] (bus_for_each_drv) from [<c0335440>] (__device_attach+0x9c/0x100)
-[<c0335440>] (__device_attach) from [<c0334acc>] (bus_probe_device+0x84/0x8c)
-[<c0334acc>] (bus_probe_device) from [<c0334f00>] (deferred_probe_work_func+0x60/0x8c)
-[<c0334f00>] (deferred_probe_work_func) from [<c003a6fc>] (process_one_work+0x120/0x328)
+[<c001d040>] (arm_dma_alloc) from [<c00edb70>] (dma_pool_alloc+0x28c/0x304)
+[<c00edb70>] (dma_pool_alloc) from [<c03fd304>] (ehci_qh_alloc+0x3c/0xb4)
+[<c03fd304>] (ehci_qh_alloc) from [<c04014f0>] (ehci_setup+0x154/0x478)
+[<c04014f0>] (ehci_setup) from [<c0405238>] (tegra_ehci_reset+0xc/0x50)
+[<c0405238>] (tegra_ehci_reset) from [<c03e8ce8>] (usb_add_hcd+0x2fc/0x804)
+[<c03e8ce8>] (usb_add_hcd) from [<c0404bb0>] (tegra_ehci_probe+0x3b8/0x504)
+[<c0404bb0>] (tegra_ehci_probe) from [<c0336cd0>] (platform_drv_probe+0x50/0xac)
+[<c0336cd0>] (platform_drv_probe) from [<c0335738>] (driver_probe_device+0x1f4/0x2b0)
+[<c0335738>] (driver_probe_device) from [<c0333ccc>] (bus_for_each_drv+0x44/0x8c)
+[<c0333ccc>] (bus_for_each_drv) from [<c03354bc>] (__device_attach+0x9c/0x100)
+[<c03354bc>] (__device_attach) from [<c0334b48>] (bus_probe_device+0x84/0x8c)
+[<c0334b48>] (bus_probe_device) from [<c0334f7c>] (deferred_probe_work_func+0x60/0x8c)
+[<c0334f7c>] (deferred_probe_work_func) from [<c003a6fc>] (process_one_work+0x120/0x328)
 [<c003a6fc>] (process_one_work) from [<c003a968>] (worker_thread+0x2c/0x4b4)
 [<c003a968>] (worker_thread) from [<c003faa0>] (kthread+0xd8/0xf4)
 [<c003faa0>] (kthread) from [<c000f938>] (ret_from_fork+0x14/0x3c)
----[ end trace 45c18807e738dda8 ]---
+---[ end trace 02f0688c2ab90aea ]---
 tegra-ehci 7d008000.usb: irq 339, io mem 0x7d008000
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-01 08:33:49 UTC (946715629)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-02 08:59:06 UTC (946803546)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-01-21 19:54:25.169130476 -0800
+++ /tmp/foob	2016-01-21 19:54:25.173130473 -0800
@@ -29,53 +29,6 @@
 Tegra124 (Jetson TK1) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash'
 Tegra124 (Jetson TK1) # setenv ipaddr 192.168.100.102; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 zImage.multi_v7_defconfig
-Waiting for Ethernet connection... done.
-Using asx0 device
-TFTP from server 192.168.100.1; our IP address is 192.168.100.102
-Filename 'zImage.multi_v7_defconfig'.
-Load address: 0x82000000
-Loading: *####EHCI timed out on TD - token=0x8008d80
-T EHCI timed out on TD - token=0x8008d80
-T EHCI timed out on TD - token=0x8008d80
-T EHCI timed out on TD - token=0x8008d80
-T EHCI timed out on TD - token=0x8008d80
-T EHCI timed out on TD - token=0x8008d80
-
-Abort
-Tegra124 (Jetson TK1) # 
-Tegra124 (Jetson TK1) # reset
-resetting ...
-
-U-Boot SPL 2014.07-00097-g772e173 (Jul 25 2014 - 20:02:52)
-
-
-U-Boot 2014.07-00097-g772e173 (Jul 25 2014 - 20:02:52)
-
-TEGRA124
-Board: NVIDIA Jetson TK1
-I2C:   ready
-DRAM:  2 GiB
-MMC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
-In:    serial
-Out:   serial
-Err:   serial
-Net:   Net Initialization Skipped
-No ethernet found.
-Hit any key to stop autoboot:  2  0 
-Tegra124 (Jetson TK1) # 
-Tegra124 (Jetson TK1) # usb reset
-(Re)start USB...
-USB0:   USB EHCI 1.10
-scanning bus 0 for devices... 1 USB Device(s) found
-USB1:   USB EHCI 1.10
-scanning bus 1 for devices... 2 USB Device(s) found
-       scanning usb for storage devices... 0 Storage Device(s) found
-       scanning usb for ethernet devices... 
-Warning: asx0 using MAC address from net device
-1 Ethernet Device(s) found
-Tegra124 (Jetson TK1) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash'
-Tegra124 (Jetson TK1) # setenv ipaddr 192.168.100.102; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
-Tegra124 (Jetson TK1) # tftpboot 0x82000000 zImage.multi_v7_defconfig
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'zImage.multi_v7_defconfig'.
@@ -88,20 +41,20 @@
 	 #################################################################
 	 #################################################################
 	 ######
-	 3.2 MiB/s
+	 3.1 MiB/s
 done
-Bytes transferred = 6760680 (6728e8 hex)
+Bytes transferred = 6761576 (672c68 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.1 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6728e8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x672c68 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012a89
@@ -109,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-next-20160121-67729-g8771b78 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Jan 20 19:22:43 PST 2016
+Linux version 4.4.0-next-20160122-67820-gcccc613 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Jan 21 19:47:55 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -139,7 +92,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0e41de4   (12520 kB)
+      .text : 0xc0208000 - 0xc0e41db4   (12520 kB)
       .init : 0xc0e42000 - 0xc0f4a000   (1056 kB)
       .data : 0xc0f4a000 - 0xc105fc40   (1112 kB)
        .bss : 0xc1062000 - 0xc10b85a0   ( 346 kB)
@@ -382,40 +335,40 @@
 WARNING: CPU: 2 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 2 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160121-67729-g8771b78 #1
+CPU: 2 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160122-67820-gcccc613 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c02188b8>] (unwind_backtrace) from [<c0213f94>] (show_stack+0x10/0x14)
-[<c0213f94>] (show_stack) from [<c04893ec>] (dump_stack+0x78/0x94)
-[<c04893ec>] (dump_stack) from [<c024ab38>] (warn_slowpath_common+0x74/0xb0)
+[<c0213f94>] (show_stack) from [<c048946c>] (dump_stack+0x78/0x94)
+[<c048946c>] (dump_stack) from [<c024ab38>] (warn_slowpath_common+0x74/0xb0)
 [<c024ab38>] (warn_slowpath_common) from [<c024ac08>] (warn_slowpath_fmt+0x30/0x40)
 [<c024ac08>] (warn_slowpath_fmt) from [<c025dc5c>] (check_flush_dependency+0xc8/0x124)
 [<c025dc5c>] (check_flush_dependency) from [<c025dd70>] (flush_work+0x54/0x144)
-[<c025dd70>] (flush_work) from [<c02dab88>] (lru_add_drain_all+0x124/0x180)
-[<c02dab88>] (lru_add_drain_all) from [<c030c6c0>] (migrate_prep+0x8/0x10)
-[<c030c6c0>] (migrate_prep) from [<c02d5ba0>] (alloc_contig_range+0xc4/0x2f8)
-[<c02d5ba0>] (alloc_contig_range) from [<c030dfec>] (cma_alloc+0xd8/0x1a0)
-[<c030dfec>] (cma_alloc) from [<c021e39c>] (__alloc_from_contiguous+0x38/0xdc)
+[<c025dd70>] (flush_work) from [<c02dabfc>] (lru_add_drain_all+0x124/0x180)
+[<c02dabfc>] (lru_add_drain_all) from [<c030c74c>] (migrate_prep+0x8/0x10)
+[<c030c74c>] (migrate_prep) from [<c02d5c14>] (alloc_contig_range+0xc4/0x2f8)
+[<c02d5c14>] (alloc_contig_range) from [<c030e078>] (cma_alloc+0xd8/0x1a0)
+[<c030e078>] (cma_alloc) from [<c021e39c>] (__alloc_from_contiguous+0x38/0xdc)
 [<c021e39c>] (__alloc_from_contiguous) from [<c021ff8c>] (__dma_alloc+0x21c/0x250)
 [<c021ff8c>] (__dma_alloc) from [<c0220044>] (arm_dma_alloc+0x48/0x54)
-[<c0220044>] (arm_dma_alloc) from [<c066a31c>] (dmam_alloc_coherent+0x8c/0xf0)
-[<c066a31c>] (dmam_alloc_coherent) from [<c06c4de4>] (ahci_port_start+0xd0/0x1e0)
-[<c06c4de4>] (ahci_port_start) from [<c06acc8c>] (ata_host_start+0xf8/0x1d4)
-[<c06acc8c>] (ata_host_start) from [<c06b1ba8>] (ata_host_activate+0x20/0x134)
-[<c06b1ba8>] (ata_host_activate) from [<c06c4260>] (ahci_host_activate+0x3c/0x11c)
-[<c06c4260>] (ahci_host_activate) from [<c06c61a4>] (ahci_platform_init_host+0x1cc/0x3c8)
-[<c06c61a4>] (ahci_platform_init_host) from [<c06c8408>] (tegra_ahci_probe+0x404/0x4c4)
-[<c06c8408>] (tegra_ahci_probe) from [<c0659218>] (platform_drv_probe+0x50/0xac)
-[<c0659218>] (platform_drv_probe) from [<c0657b84>] (driver_probe_device+0x1f4/0x2b0)
-[<c0657b84>] (driver_probe_device) from [<c0656114>] (bus_for_each_drv+0x44/0x8c)
-[<c0656114>] (bus_for_each_drv) from [<c0657908>] (__device_attach+0x9c/0x100)
-[<c0657908>] (__device_attach) from [<c0656f94>] (bus_probe_device+0x84/0x8c)
-[<c0656f94>] (bus_probe_device) from [<c06573c8>] (deferred_probe_work_func+0x60/0x8c)
-[<c06573c8>] (deferred_probe_work_func) from [<c025ebb8>] (process_one_work+0x134/0x338)
+[<c0220044>] (arm_dma_alloc) from [<c066a410>] (dmam_alloc_coherent+0x8c/0xf0)
+[<c066a410>] (dmam_alloc_coherent) from [<c06c4ed4>] (ahci_port_start+0xd0/0x1e0)
+[<c06c4ed4>] (ahci_port_start) from [<c06acd7c>] (ata_host_start+0xf8/0x1d4)
+[<c06acd7c>] (ata_host_start) from [<c06b1c98>] (ata_host_activate+0x20/0x134)
+[<c06b1c98>] (ata_host_activate) from [<c06c4350>] (ahci_host_activate+0x3c/0x11c)
+[<c06c4350>] (ahci_host_activate) from [<c06c6294>] (ahci_platform_init_host+0x1cc/0x3c8)
+[<c06c6294>] (ahci_platform_init_host) from [<c06c84f8>] (tegra_ahci_probe+0x404/0x4c4)
+[<c06c84f8>] (tegra_ahci_probe) from [<c0659308>] (platform_drv_probe+0x50/0xac)
+[<c0659308>] (platform_drv_probe) from [<c0657c74>] (driver_probe_device+0x1f4/0x2b0)
+[<c0657c74>] (driver_probe_device) from [<c0656204>] (bus_for_each_drv+0x44/0x8c)
+[<c0656204>] (bus_for_each_drv) from [<c06579f8>] (__device_attach+0x9c/0x100)
+[<c06579f8>] (__device_attach) from [<c0657084>] (bus_probe_device+0x84/0x8c)
+[<c0657084>] (bus_probe_device) from [<c06574b8>] (deferred_probe_work_func+0x60/0x8c)
+[<c06574b8>] (deferred_probe_work_func) from [<c025ebb8>] (process_one_work+0x134/0x338)
 [<c025ebb8>] (process_one_work) from [<c025ee20>] (worker_thread+0x2c/0x4f0)
 [<c025ee20>] (worker_thread) from [<c0263fac>] (kthread+0xd8/0xf4)
 [<c0263fac>] (kthread) from [<c0210e78>] (ret_from_fork+0x14/0x3c)
----[ end trace bce3e8878b83146a ]---
+---[ end trace 1228346256aca919 ]---
 scsi host0: tegra-ahci
 scsi host1: tegra-ahci
 ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 341
@@ -437,7 +390,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:57 UTC (946684857)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:18 UTC (946684818)
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-01-21 19:54:25.177130472 -0800
+++ /tmp/foob	2016-01-21 19:54:25.181130470 -0800
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 #################################
-	 3.2 MiB/s
+	 3.1 MiB/s
 done
-Bytes transferred = 5254600 (502dc8 hex)
+Bytes transferred = 5255208 (503028 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.2 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x502dc8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x503028 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012a89
@@ -61,7 +61,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-next-20160121-67729-g8771b78 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Jan 20 19:23:04 PST 2016
+Linux version 4.4.0-next-20160122-67820-gcccc613 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Jan 21 19:48:13 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -92,7 +92,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0981738   (9702 kB)
+      .text : 0xc0008000 - 0xc098171c   (9702 kB)
       .init : 0xc0982000 - 0xc09d8000   ( 344 kB)
       .data : 0xc09d8000 - 0xc0a34f20   ( 372 kB)
        .bss : 0xc0a37000 - 0xc0a92d88   ( 368 kB)
@@ -278,8 +278,8 @@
 NET: Registered protocol family 29
 can: raw protocol (rev 20120528)
 can: broadcast manager protocol (rev 20120528 t)
-input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input1
 can: netlink gateway (rev 20130117) max_hops=1
+input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input1
 Bluetooth: RFCOMM socket layer initialized
 Bluetooth: RFCOMM ver 1.11
 Bluetooth: BNEP (Ethernet Emulation) ver 1.3
@@ -301,42 +301,16 @@
 tegra-pcie 1003000.pcie-controller: link 0 down, ignoring
 tegra-pcie 1003000.pcie-controller: probing port 1, using 1 lanes
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000000
+tegra-pcie 1003000.pcie-controller: link 1 down, retrying
+tegra-pcie 1003000.pcie-controller: link 1 down, retrying
+tegra-pcie 1003000.pcie-controller: link 1 down, retrying
+tegra-pcie 1003000.pcie-controller: link 1 down, ignoring
 tegra-pcie 1003000.pcie-controller: PCI host bridge to bus 0000:00
 pci_bus 0000:00: root bus resource [mem 0x13000000-0x1fffffff]
 pci_bus 0000:00: root bus resource [mem 0x20000000-0x3fffffff pref]
 pci_bus 0000:00: root bus resource [bus 00-ff]
 pci_bus 0000:00: root bus resource [io  0x1000-0xffff]
-pci 0000:00:02.0: [10de:0e13] type 01 class 0x060400
-pci 0000:00:02.0: PME# supported from D0 D1 D2 D3hot D3cold
-PCI: bus0: Fast back to back transfers disabled
-pci 0000:00:02.0: bridge configuration invalid ([bus 00-00]), reconfiguring
-pci 0000:01:00.0: [10ec:8168] type 00 class 0x020000
-pci 0000:01:00.0: reg 0x10: [io  0x0000-0x00ff]
-pci 0000:01:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit]
-pci 0000:01:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
-pci 0000:01:00.0: supports D1 D2
-pci 0000:01:00.0: PME# supported from D0 D1 D2 D3hot D3cold
-PCI: bus1: Fast back to back transfers disabled
-pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
-pci 0000:00:02.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
-pci 0000:00:02.0: BAR 9: assigned [mem 0x20000000-0x200fffff 64bit pref]
-pci 0000:00:02.0: BAR 7: assigned [io  0x1000-0x1fff]
-pci 0000:01:00.0: BAR 4: assigned [mem 0x20000000-0x20003fff 64bit pref]
-pci 0000:01:00.0: BAR 2: assigned [mem 0x13000000-0x13000fff 64bit]
-pci 0000:01:00.0: BAR 0: assigned [io  0x1000-0x10ff]
-pci 0000:00:02.0: PCI bridge to [bus 01]
-pci 0000:00:02.0:   bridge window [io  0x1000-0x1fff]
-pci 0000:00:02.0:   bridge window [mem 0x13000000-0x130fffff]
-pci 0000:00:02.0:   bridge window [mem 0x20000000-0x200fffff 64bit pref]
-pci 0000:00:02.0: nv_msi_ht_cap_quirk didn't locate host bridge
-pcieport 0000:00:02.0: enabling device (0140 -> 0143)
-pcieport 0000:00:02.0: Signaling PME through PCIe PME interrupt
-pci 0000:01:00.0: Signaling PME through PCIe PME interrupt
-pcie_pme 0000:00:02.0:pcie01: service driver pcie_pme loaded
-r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
-r8169 0000:01:00.0: enabling device (0140 -> 0143)
-r8169 0000:01:00.0 eth0: RTL8168g/8111g at 0xf0952000, 00:04:4b:25:b3:b4, XID 0c000800 IRQ 388
-r8169 0000:01:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
+PCI: bus0: Fast back to back transfers enabled
 +5V_HDMI_CON: supplied by +5V_SYS
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 +3.3V_RUN: supplied by +3.3V_SYS
@@ -355,40 +329,40 @@
 WARNING: CPU: 0 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160121-67729-g8771b78 #1
+CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160122-67820-gcccc613 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0017208>] (unwind_backtrace) from [<c0012874>] (show_stack+0x10/0x14)
-[<c0012874>] (show_stack) from [<c023c164>] (dump_stack+0x84/0xd0)
-[<c023c164>] (dump_stack) from [<c0026b70>] (warn_slowpath_common+0x74/0xb0)
+[<c0012874>] (show_stack) from [<c023c1a4>] (dump_stack+0x84/0xd0)
+[<c023c1a4>] (dump_stack) from [<c0026b70>] (warn_slowpath_common+0x74/0xb0)
 [<c0026b70>] (warn_slowpath_common) from [<c0026c40>] (warn_slowpath_fmt+0x30/0x40)
 [<c0026c40>] (warn_slowpath_fmt) from [<c00397d4>] (check_flush_dependency+0xc8/0x12c)
 [<c00397d4>] (check_flush_dependency) from [<c00398f0>] (flush_work+0x54/0x11c)
-[<c00398f0>] (flush_work) from [<c00c0884>] (lru_add_drain_all+0x124/0x180)
-[<c00c0884>] (lru_add_drain_all) from [<c00f16fc>] (migrate_prep+0x8/0x10)
-[<c00f16fc>] (migrate_prep) from [<c00bb3fc>] (alloc_contig_range+0xc4/0x2f8)
-[<c00bb3fc>] (alloc_contig_range) from [<c00f326c>] (cma_alloc+0xd8/0x1a0)
-[<c00f326c>] (cma_alloc) from [<c001c270>] (__alloc_from_contiguous+0x38/0xdc)
+[<c00398f0>] (flush_work) from [<c00c08d4>] (lru_add_drain_all+0x124/0x180)
+[<c00c08d4>] (lru_add_drain_all) from [<c00f1764>] (migrate_prep+0x8/0x10)
+[<c00f1764>] (migrate_prep) from [<c00bb44c>] (alloc_contig_range+0xc4/0x2f8)
+[<c00bb44c>] (alloc_contig_range) from [<c00f32d4>] (cma_alloc+0xd8/0x1a0)
+[<c00f32d4>] (cma_alloc) from [<c001c270>] (__alloc_from_contiguous+0x38/0xdc)
 [<c001c270>] (__alloc_from_contiguous) from [<c001cf38>] (__dma_alloc+0x228/0x2ac)
 [<c001cf38>] (__dma_alloc) from [<c001d040>] (arm_dma_alloc+0x48/0x54)
-[<c001d040>] (arm_dma_alloc) from [<c0345c28>] (dmam_alloc_coherent+0x8c/0xf0)
-[<c0345c28>] (dmam_alloc_coherent) from [<c038bcd0>] (ahci_port_start+0xd0/0x1e0)
-[<c038bcd0>] (ahci_port_start) from [<c03739a8>] (ata_host_start+0xf8/0x1d4)
-[<c03739a8>] (ata_host_start) from [<c037893c>] (ata_host_activate+0x20/0x134)
-[<c037893c>] (ata_host_activate) from [<c038b178>] (ahci_host_activate+0x3c/0x11c)
-[<c038b178>] (ahci_host_activate) from [<c038d520>] (ahci_platform_init_host+0x1cc/0x3c8)
-[<c038d520>] (ahci_platform_init_host) from [<c038d1ec>] (tegra_ahci_probe+0x404/0x4c4)
-[<c038d1ec>] (tegra_ahci_probe) from [<c0336c54>] (platform_drv_probe+0x50/0xac)
-[<c0336c54>] (platform_drv_probe) from [<c03356bc>] (driver_probe_device+0x1f4/0x2b0)
-[<c03356bc>] (driver_probe_device) from [<c0333c50>] (bus_for_each_drv+0x44/0x8c)
-[<c0333c50>] (bus_for_each_drv) from [<c0335440>] (__device_attach+0x9c/0x100)
-[<c0335440>] (__device_attach) from [<c0334acc>] (bus_probe_device+0x84/0x8c)
-[<c0334acc>] (bus_probe_device) from [<c0334f00>] (deferred_probe_work_func+0x60/0x8c)
-[<c0334f00>] (deferred_probe_work_func) from [<c003a6fc>] (process_one_work+0x120/0x328)
+[<c001d040>] (arm_dma_alloc) from [<c0345ca8>] (dmam_alloc_coherent+0x8c/0xf0)
+[<c0345ca8>] (dmam_alloc_coherent) from [<c038bd50>] (ahci_port_start+0xd0/0x1e0)
+[<c038bd50>] (ahci_port_start) from [<c0373a28>] (ata_host_start+0xf8/0x1d4)
+[<c0373a28>] (ata_host_start) from [<c03789bc>] (ata_host_activate+0x20/0x134)
+[<c03789bc>] (ata_host_activate) from [<c038b1f8>] (ahci_host_activate+0x3c/0x11c)
+[<c038b1f8>] (ahci_host_activate) from [<c038d5a0>] (ahci_platform_init_host+0x1cc/0x3c8)
+[<c038d5a0>] (ahci_platform_init_host) from [<c038d26c>] (tegra_ahci_probe+0x404/0x4c4)
+[<c038d26c>] (tegra_ahci_probe) from [<c0336cd0>] (platform_drv_probe+0x50/0xac)
+[<c0336cd0>] (platform_drv_probe) from [<c0335738>] (driver_probe_device+0x1f4/0x2b0)
+[<c0335738>] (driver_probe_device) from [<c0333ccc>] (bus_for_each_drv+0x44/0x8c)
+[<c0333ccc>] (bus_for_each_drv) from [<c03354bc>] (__device_attach+0x9c/0x100)
+[<c03354bc>] (__device_attach) from [<c0334b48>] (bus_probe_device+0x84/0x8c)
+[<c0334b48>] (bus_probe_device) from [<c0334f7c>] (deferred_probe_work_func+0x60/0x8c)
+[<c0334f7c>] (deferred_probe_work_func) from [<c003a6fc>] (process_one_work+0x120/0x328)
 [<c003a6fc>] (process_one_work) from [<c003a968>] (worker_thread+0x2c/0x4b4)
 [<c003a968>] (worker_thread) from [<c003faa0>] (kthread+0xd8/0xf4)
 [<c003faa0>] (kthread) from [<c000f938>] (ret_from_fork+0x14/0x3c)
----[ end trace 13a9c7a412a347a3 ]---
+---[ end trace 524e5f07c3ea2c55 ]---
 scsi host0: tegra-ahci
 scsi host1: tegra-ahci
 ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 341
@@ -410,7 +384,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:03:02 UTC (946684982)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:01:55 UTC (946684915)
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
@@ -425,11 +399,11 @@
 ata1: SATA link down (SStatus 0 SControl 300)
 EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
 EXT4-fs (mmcblk0p1): recovery complete
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 devtmpfs: mounted
 Freeing unused kernel memory: 344K (c0982000 - c09d8000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    5.788248] asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:11:31:3b
+root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-01-21 19:54:25.185130468 -0800
+++ /tmp/foob	2016-01-21 19:54:25.189130466 -0800
@@ -49,7 +49,7 @@
 	 ######
 	 4 MiB/s
 done
-Bytes transferred = 6760680 (6728e8 hex)
+Bytes transferred = 6761576 (672c68 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -60,7 +60,7 @@
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x6728e8 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x672c68 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-next-20160121-67729-g8771b78 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Jan 20 19:22:43 PST 2016
+Linux version 4.4.0-next-20160122-67820-gcccc613 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Jan 21 19:47:55 PST 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -97,7 +97,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0e41de4   (12520 kB)
+      .text : 0xc0208000 - 0xc0e41db4   (12520 kB)
       .init : 0xc0e42000 - 0xc0f4a000   (1056 kB)
       .data : 0xc0f4a000 - 0xc105fc40   (1112 kB)
        .bss : 0xc1062000 - 0xc10b85a0   ( 346 kB)
@@ -329,18 +329,18 @@
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
 mmc1: new high speed SD card at address b368
-rtc-em3027 2-0056: setting system clock to 2014-12-28 04:14:19 UTC (1419740059)
+rtc-em3027 2-0056: setting system clock to 2014-12-28 04:16:16 UTC (1419740176)
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
+hub 1-1:1.0: USB hub found
+hub 1-1:1.0: 4 ports detected
+usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 1056K (c0e42000 - c0f4a000)
-hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-01-21 19:54:25.193130463 -0800
+++ /tmp/foob	2016-01-21 19:54:25.197130461 -0800
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -45,9 +45,9 @@
 	 #################################################################
 	 #################################################################
 	 #################################
-	 3.4 MiB/s
+	 4.1 MiB/s
 done
-Bytes transferred = 5254600 (502dc8 hex)
+Bytes transferred = 5255208 (503028 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x502dc8 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x503028 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-next-20160121-67729-g8771b78 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Jan 20 19:23:04 PST 2016
+Linux version 4.4.0-next-20160122-67820-gcccc613 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Jan 21 19:48:13 PST 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -97,7 +97,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0981738   (9702 kB)
+      .text : 0xc0008000 - 0xc098171c   (9702 kB)
       .init : 0xc0982000 - 0xc09d8000   ( 344 kB)
       .data : 0xc09d8000 - 0xc0a34f20   ( 372 kB)
        .bss : 0xc0a37000 - 0xc0a92d88   ( 368 kB)
@@ -307,13 +307,10 @@
 NET: Registered protocol family 10
 mmc0: host does not support reading read-only switch, assuming write-enable
 mip6: Mobile IPv6
-mmc0: new high speed SDHC card at address b368
 mmc1: new high speed SD card at address b368
 sit: IPv6 over IPv4 tunneling driver
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
- mmcblk0: p1
- mmcblk1: p1 p2
+mmc0: new high speed SDHC card at address b368
+mmcblk0: mmc1:b368 NCard 1.83 GiB (ro)
 NET: Registered protocol family 17
 NET: Registered protocol family 15
 can: controller area network core (rev 20120528 abi 9)
@@ -331,22 +328,25 @@
 Registering SWP/SWPB emulation handler
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
-hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
+mmcblk1: mmc0:b368 NCard 3.70 GiB 
+ mmcblk1: p1
+ mmcblk0: p1 p2
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
+hub 1-1:1.0: USB hub found
+hub 1-1:1.0: 4 ports detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-28 04:15:18 UTC (1419740118)
+rtc-em3027 2-0056: setting system clock to 2014-12-28 04:17:16 UTC (1419740236)
 ALSA device list:
   #0: tegra-trimslice
-EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
+EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
-EXT4-fs (mmcblk0p1): recovery complete
-EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:1.
+EXT4-fs (mmcblk1p1): recovery complete
+EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
+VFS: Mounted root (ext3 filesystem) on device 179:17.
 devtmpfs: mounted
 Freeing unused kernel memory: 344K (c0982000 - c09d8000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-01-21 19:54:25.201130460 -0800
+++ /tmp/foob	2016-01-21 19:54:25.205130458 -0800
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 ######
-	 3.5 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 6760680 (6728e8 hex)
+Bytes transferred = 6761576 (672c68 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.3 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6728e8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x672c68 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-next-20160121-67729-g8771b78 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Jan 20 19:22:43 PST 2016
+Linux version 4.4.0-next-20160122-67820-gcccc613 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Jan 21 19:47:55 PST 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -89,7 +89,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0e41de4   (12520 kB)
+      .text : 0xc0208000 - 0xc0e41db4   (12520 kB)
       .init : 0xc0e42000 - 0xc0f4a000   (1056 kB)
       .data : 0xc0f4a000 - 0xc105fc40   (1112 kB)
        .bss : 0xc1062000 - 0xc10b85a0   ( 346 kB)
@@ -286,8 +286,8 @@
 mmcblk0boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk0rpmb: mmc1:0001 SEM16G partition 3 128 KiB
 Primary GPT is invalid, using alternate GPT.
-tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
  mmcblk0: p1 p2 p3 p4 p5
+tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -340,42 +340,42 @@
 tegra-ehci 7d004000.usb: EHCI Host Controller
 tegra-ehci 7d004000.usb: new USB bus registered, assigned bus number 1
 ------------[ cut here ]------------
-WARNING: CPU: 3 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
+WARNING: CPU: 2 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 3 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160121-67729-g8771b78 #1
+CPU: 2 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160122-67820-gcccc613 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c02188b8>] (unwind_backtrace) from [<c0213f94>] (show_stack+0x10/0x14)
-[<c0213f94>] (show_stack) from [<c04893ec>] (dump_stack+0x78/0x94)
-[<c04893ec>] (dump_stack) from [<c024ab38>] (warn_slowpath_common+0x74/0xb0)
+[<c0213f94>] (show_stack) from [<c048946c>] (dump_stack+0x78/0x94)
+[<c048946c>] (dump_stack) from [<c024ab38>] (warn_slowpath_common+0x74/0xb0)
 [<c024ab38>] (warn_slowpath_common) from [<c024ac08>] (warn_slowpath_fmt+0x30/0x40)
 [<c024ac08>] (warn_slowpath_fmt) from [<c025dc5c>] (check_flush_dependency+0xc8/0x124)
 [<c025dc5c>] (check_flush_dependency) from [<c025dd70>] (flush_work+0x54/0x144)
-[<c025dd70>] (flush_work) from [<c02dab88>] (lru_add_drain_all+0x124/0x180)
-[<c02dab88>] (lru_add_drain_all) from [<c030c6c0>] (migrate_prep+0x8/0x10)
-[<c030c6c0>] (migrate_prep) from [<c02d5ba0>] (alloc_contig_range+0xc4/0x2f8)
-[<c02d5ba0>] (alloc_contig_range) from [<c030dfec>] (cma_alloc+0xd8/0x1a0)
-[<c030dfec>] (cma_alloc) from [<c021e39c>] (__alloc_from_contiguous+0x38/0xdc)
+[<c025dd70>] (flush_work) from [<c02dabfc>] (lru_add_drain_all+0x124/0x180)
+[<c02dabfc>] (lru_add_drain_all) from [<c030c74c>] (migrate_prep+0x8/0x10)
+[<c030c74c>] (migrate_prep) from [<c02d5c14>] (alloc_contig_range+0xc4/0x2f8)
+[<c02d5c14>] (alloc_contig_range) from [<c030e078>] (cma_alloc+0xd8/0x1a0)
+[<c030e078>] (cma_alloc) from [<c021e39c>] (__alloc_from_contiguous+0x38/0xdc)
 [<c021e39c>] (__alloc_from_contiguous) from [<c021ff8c>] (__dma_alloc+0x21c/0x250)
 [<c021ff8c>] (__dma_alloc) from [<c0220044>] (arm_dma_alloc+0x48/0x54)
-[<c0220044>] (arm_dma_alloc) from [<c0306c10>] (dma_pool_alloc+0x1dc/0x210)
-[<c0306c10>] (dma_pool_alloc) from [<c07c7084>] (ehci_qh_alloc+0x3c/0xb4)
-[<c07c7084>] (ehci_qh_alloc) from [<c07cb67c>] (ehci_setup+0x154/0x478)
-[<c07cb67c>] (ehci_setup) from [<c07d1734>] (tegra_ehci_reset+0xc/0x50)
-[<c07d1734>] (tegra_ehci_reset) from [<c07a1afc>] (usb_add_hcd+0x300/0x830)
-[<c07a1afc>] (usb_add_hcd) from [<c07d10ac>] (tegra_ehci_probe+0x3b8/0x504)
-[<c07d10ac>] (tegra_ehci_probe) from [<c0659218>] (platform_drv_probe+0x50/0xac)
-[<c0659218>] (platform_drv_probe) from [<c0657b84>] (driver_probe_device+0x1f4/0x2b0)
-[<c0657b84>] (driver_probe_device) from [<c0656114>] (bus_for_each_drv+0x44/0x8c)
-[<c0656114>] (bus_for_each_drv) from [<c0657908>] (__device_attach+0x9c/0x100)
-[<c0657908>] (__device_attach) from [<c0656f94>] (bus_probe_device+0x84/0x8c)
-[<c0656f94>] (bus_probe_device) from [<c06573c8>] (deferred_probe_work_func+0x60/0x8c)
-[<c06573c8>] (deferred_probe_work_func) from [<c025ebb8>] (process_one_work+0x134/0x338)
+[<c0220044>] (arm_dma_alloc) from [<c0306c9c>] (dma_pool_alloc+0x1dc/0x210)
+[<c0306c9c>] (dma_pool_alloc) from [<c07c7274>] (ehci_qh_alloc+0x3c/0xb4)
+[<c07c7274>] (ehci_qh_alloc) from [<c07cb86c>] (ehci_setup+0x154/0x478)
+[<c07cb86c>] (ehci_setup) from [<c07d1924>] (tegra_ehci_reset+0xc/0x50)
+[<c07d1924>] (tegra_ehci_reset) from [<c07a1cec>] (usb_add_hcd+0x300/0x830)
+[<c07a1cec>] (usb_add_hcd) from [<c07d129c>] (tegra_ehci_probe+0x3b8/0x504)
+[<c07d129c>] (tegra_ehci_probe) from [<c0659308>] (platform_drv_probe+0x50/0xac)
+[<c0659308>] (platform_drv_probe) from [<c0657c74>] (driver_probe_device+0x1f4/0x2b0)
+[<c0657c74>] (driver_probe_device) from [<c0656204>] (bus_for_each_drv+0x44/0x8c)
+[<c0656204>] (bus_for_each_drv) from [<c06579f8>] (__device_attach+0x9c/0x100)
+[<c06579f8>] (__device_attach) from [<c0657084>] (bus_probe_device+0x84/0x8c)
+[<c0657084>] (bus_probe_device) from [<c06574b8>] (deferred_probe_work_func+0x60/0x8c)
+[<c06574b8>] (deferred_probe_work_func) from [<c025ebb8>] (process_one_work+0x134/0x338)
 [<c025ebb8>] (process_one_work) from [<c025ee20>] (worker_thread+0x2c/0x4f0)
 [<c025ee20>] (worker_thread) from [<c0263fac>] (kthread+0xd8/0xf4)
 [<c0263fac>] (kthread) from [<c0210e78>] (ret_from_fork+0x14/0x3c)
----[ end trace f5cad7005d88ec50 ]---
+---[ end trace bd091358915b4f0c ]---
 tegra-ehci 7d004000.usb: irq 343, io mem 0x7d004000
 tegra-ehci 7d004000.usb: USB 2.0 started, EHCI 1.00
 hub 1-0:1.0: USB hub found
@@ -398,4 +398,4 @@
 Freeing unused kernel memory: 1056K (c0e42000 - c0f4a000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.998532] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.018923] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-01-21 19:54:25.209130455 -0800
+++ /tmp/foob	2016-01-21 19:54:25.213130453 -0800
@@ -37,9 +37,9 @@
 	 #################################################################
 	 #################################################################
 	 #################################
-	 3.5 MiB/s
+	 3 MiB/s
 done
-Bytes transferred = 5254600 (502dc8 hex)
+Bytes transferred = 5255208 (503028 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x502dc8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x503028 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-next-20160121-67729-g8771b78 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Jan 20 19:23:04 PST 2016
+Linux version 4.4.0-next-20160122-67820-gcccc613 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Jan 21 19:48:13 PST 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -89,7 +89,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0981738   (9702 kB)
+      .text : 0xc0008000 - 0xc098171c   (9702 kB)
       .init : 0xc0982000 - 0xc09d8000   ( 344 kB)
       .data : 0xc09d8000 - 0xc0a34f20   ( 372 kB)
        .bss : 0xc0a37000 - 0xc0a92d88   ( 368 kB)
@@ -341,39 +341,39 @@
 WARNING: CPU: 0 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160121-67729-g8771b78 #1
+CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160122-67820-gcccc613 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0017208>] (unwind_backtrace) from [<c0012874>] (show_stack+0x10/0x14)
-[<c0012874>] (show_stack) from [<c023c164>] (dump_stack+0x84/0xd0)
-[<c023c164>] (dump_stack) from [<c0026b70>] (warn_slowpath_common+0x74/0xb0)
+[<c0012874>] (show_stack) from [<c023c1a4>] (dump_stack+0x84/0xd0)
+[<c023c1a4>] (dump_stack) from [<c0026b70>] (warn_slowpath_common+0x74/0xb0)
 [<c0026b70>] (warn_slowpath_common) from [<c0026c40>] (warn_slowpath_fmt+0x30/0x40)
 [<c0026c40>] (warn_slowpath_fmt) from [<c00397d4>] (check_flush_dependency+0xc8/0x12c)
 [<c00397d4>] (check_flush_dependency) from [<c00398f0>] (flush_work+0x54/0x11c)
-[<c00398f0>] (flush_work) from [<c00c0884>] (lru_add_drain_all+0x124/0x180)
-[<c00c0884>] (lru_add_drain_all) from [<c00f16fc>] (migrate_prep+0x8/0x10)
-[<c00f16fc>] (migrate_prep) from [<c00bb3fc>] (alloc_contig_range+0xc4/0x2f8)
-[<c00bb3fc>] (alloc_contig_range) from [<c00f326c>] (cma_alloc+0xd8/0x1a0)
-[<c00f326c>] (cma_alloc) from [<c001c270>] (__alloc_from_contiguous+0x38/0xdc)
+[<c00398f0>] (flush_work) from [<c00c08d4>] (lru_add_drain_all+0x124/0x180)
+[<c00c08d4>] (lru_add_drain_all) from [<c00f1764>] (migrate_prep+0x8/0x10)
+[<c00f1764>] (migrate_prep) from [<c00bb44c>] (alloc_contig_range+0xc4/0x2f8)
+[<c00bb44c>] (alloc_contig_range) from [<c00f32d4>] (cma_alloc+0xd8/0x1a0)
+[<c00f32d4>] (cma_alloc) from [<c001c270>] (__alloc_from_contiguous+0x38/0xdc)
 [<c001c270>] (__alloc_from_contiguous) from [<c001cf38>] (__dma_alloc+0x228/0x2ac)
 [<c001cf38>] (__dma_alloc) from [<c001d040>] (arm_dma_alloc+0x48/0x54)
-[<c001d040>] (arm_dma_alloc) from [<c00edb08>] (dma_pool_alloc+0x28c/0x304)
-[<c00edb08>] (dma_pool_alloc) from [<c03fd27c>] (ehci_qh_alloc+0x3c/0xb4)
-[<c03fd27c>] (ehci_qh_alloc) from [<c0401468>] (ehci_setup+0x154/0x478)
-[<c0401468>] (ehci_setup) from [<c04051b0>] (tegra_ehci_reset+0xc/0x50)
-[<c04051b0>] (tegra_ehci_reset) from [<c03e8c60>] (usb_add_hcd+0x2fc/0x804)
-[<c03e8c60>] (usb_add_hcd) from [<c0404b28>] (tegra_ehci_probe+0x3b8/0x504)
-[<c0404b28>] (tegra_ehci_probe) from [<c0336c54>] (platform_drv_probe+0x50/0xac)
-[<c0336c54>] (platform_drv_probe) from [<c03356bc>] (driver_probe_device+0x1f4/0x2b0)
-[<c03356bc>] (driver_probe_device) from [<c0333c50>] (bus_for_each_drv+0x44/0x8c)
-[<c0333c50>] (bus_for_each_drv) from [<c0335440>] (__device_attach+0x9c/0x100)
-[<c0335440>] (__device_attach) from [<c0334acc>] (bus_probe_device+0x84/0x8c)
-[<c0334acc>] (bus_probe_device) from [<c0334f00>] (deferred_probe_work_func+0x60/0x8c)
-[<c0334f00>] (deferred_probe_work_func) from [<c003a6fc>] (process_one_work+0x120/0x328)
+[<c001d040>] (arm_dma_alloc) from [<c00edb70>] (dma_pool_alloc+0x28c/0x304)
+[<c00edb70>] (dma_pool_alloc) from [<c03fd304>] (ehci_qh_alloc+0x3c/0xb4)
+[<c03fd304>] (ehci_qh_alloc) from [<c04014f0>] (ehci_setup+0x154/0x478)
+[<c04014f0>] (ehci_setup) from [<c0405238>] (tegra_ehci_reset+0xc/0x50)
+[<c0405238>] (tegra_ehci_reset) from [<c03e8ce8>] (usb_add_hcd+0x2fc/0x804)
+[<c03e8ce8>] (usb_add_hcd) from [<c0404bb0>] (tegra_ehci_probe+0x3b8/0x504)
+[<c0404bb0>] (tegra_ehci_probe) from [<c0336cd0>] (platform_drv_probe+0x50/0xac)
+[<c0336cd0>] (platform_drv_probe) from [<c0335738>] (driver_probe_device+0x1f4/0x2b0)
+[<c0335738>] (driver_probe_device) from [<c0333ccc>] (bus_for_each_drv+0x44/0x8c)
+[<c0333ccc>] (bus_for_each_drv) from [<c03354bc>] (__device_attach+0x9c/0x100)
+[<c03354bc>] (__device_attach) from [<c0334b48>] (bus_probe_device+0x84/0x8c)
+[<c0334b48>] (bus_probe_device) from [<c0334f7c>] (deferred_probe_work_func+0x60/0x8c)
+[<c0334f7c>] (deferred_probe_work_func) from [<c003a6fc>] (process_one_work+0x120/0x328)
 [<c003a6fc>] (process_one_work) from [<c003a968>] (worker_thread+0x2c/0x4b4)
 [<c003a968>] (worker_thread) from [<c003faa0>] (kthread+0xd8/0xf4)
 [<c003faa0>] (kthread) from [<c000f938>] (ret_from_fork+0x14/0x3c)
----[ end trace c38af1ee3d3fe6a4 ]---
+---[ end trace f84d69cc87b69db6 ]---
 tegra-ehci 7d004000.usb: irq 343, io mem 0x7d004000
 tegra-ehci 7d004000.usb: USB 2.0 started, EHCI 1.00
 hub 1-0:1.0: USB hub found
