Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-01-12 00:10:07.422461915 -0800
+++ /tmp/foob	2016-01-12 00:10:07.422461915 -0800
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-rc8-next-20160111+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Jan 11 04:19:07 PST 2016
+Linux version 4.4.0-next-20160112+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Jan 12 00:05:35 PST 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -93,7 +93,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.536:1): initialized
+audit: type=2000 audit(0.472:1): initialized
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
 Key type id_resolver registered
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-01-12 00:10:07.458461882 -0800
+++ /tmp/foob	2016-01-12 00:10:07.462461879 -0800
@@ -41,18 +41,18 @@
 	 ######
 	 3.3 MiB/s
 done
-Bytes transferred = 6763968 (6735c0 hex)
+Bytes transferred = 6761880 (672d98 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.6 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6735c0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x672d98 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-rc8-next-20160111-66728-gbf976ab (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jan 11 04:17:41 PST 2016
+Linux version 4.4.0-next-20160112-66957-gd299f5e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Jan 12 00:03:39 PST 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c105dc40, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c105dd00, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1997952K/2096128K available (8495K kernel code, 1110K rwdata, 4024K rodata, 1052K init, 345K bss, 32640K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1997952K/2096128K available (8495K kernel code, 1111K rwdata, 4024K rodata, 1052K init, 345K bss, 32640K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,9 +90,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0e42df0   (12524 kB)
+      .text : 0xc0208000 - 0xc0e42de8   (12524 kB)
       .init : 0xc0e43000 - 0xc0f4a000   (1052 kB)
-      .data : 0xc0f4a000 - 0xc105fbc0   (1111 kB)
+      .data : 0xc0f4a000 - 0xc105fc80   (1112 kB)
        .bss : 0xc1062000 - 0xc10b85a0   ( 346 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -293,48 +293,48 @@
 tegra-ehci 7d008000.usb: EHCI Host Controller
 tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 1
 ------------[ cut here ]------------
-WARNING: CPU: 3 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
+WARNING: CPU: 2 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 3 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-rc8-next-20160111-66728-gbf976ab #1
+CPU: 2 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160112-66957-gd299f5e #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c02188b8>] (unwind_backtrace) from [<c0213f94>] (show_stack+0x10/0x14)
-[<c0213f94>] (show_stack) from [<c048972c>] (dump_stack+0x78/0x94)
-[<c048972c>] (dump_stack) from [<c024ab38>] (warn_slowpath_common+0x74/0xb0)
+[<c0213f94>] (show_stack) from [<c048918c>] (dump_stack+0x78/0x94)
+[<c048918c>] (dump_stack) from [<c024ab38>] (warn_slowpath_common+0x74/0xb0)
 [<c024ab38>] (warn_slowpath_common) from [<c024ac08>] (warn_slowpath_fmt+0x30/0x40)
 [<c024ac08>] (warn_slowpath_fmt) from [<c025dc58>] (check_flush_dependency+0xc8/0x124)
 [<c025dc58>] (check_flush_dependency) from [<c025dd6c>] (flush_work+0x54/0x144)
-[<c025dd6c>] (flush_work) from [<c02dae8c>] (lru_add_drain_all+0x124/0x180)
-[<c02dae8c>] (lru_add_drain_all) from [<c030c7ec>] (migrate_prep+0x8/0x10)
-[<c030c7ec>] (migrate_prep) from [<c02d5ea4>] (alloc_contig_range+0xc4/0x2f8)
-[<c02d5ea4>] (alloc_contig_range) from [<c030e118>] (cma_alloc+0xd8/0x1a0)
-[<c030e118>] (cma_alloc) from [<c021e39c>] (__alloc_from_contiguous+0x38/0xdc)
+[<c025dd6c>] (flush_work) from [<c02dab88>] (lru_add_drain_all+0x124/0x180)
+[<c02dab88>] (lru_add_drain_all) from [<c030c4e8>] (migrate_prep+0x8/0x10)
+[<c030c4e8>] (migrate_prep) from [<c02d5ba0>] (alloc_contig_range+0xc4/0x2f8)
+[<c02d5ba0>] (alloc_contig_range) from [<c030de14>] (cma_alloc+0xd8/0x1a0)
+[<c030de14>] (cma_alloc) from [<c021e39c>] (__alloc_from_contiguous+0x38/0xdc)
 [<c021e39c>] (__alloc_from_contiguous) from [<c021ff8c>] (__dma_alloc+0x21c/0x250)
 [<c021ff8c>] (__dma_alloc) from [<c0220044>] (arm_dma_alloc+0x48/0x54)
-[<c0220044>] (arm_dma_alloc) from [<c0306d3c>] (dma_pool_alloc+0x1dc/0x210)
-[<c0306d3c>] (dma_pool_alloc) from [<c07c767c>] (ehci_qh_alloc+0x3c/0xb4)
-[<c07c767c>] (ehci_qh_alloc) from [<c07cbc74>] (ehci_setup+0x154/0x478)
-[<c07cbc74>] (ehci_setup) from [<c07d1d2c>] (tegra_ehci_reset+0xc/0x50)
-[<c07d1d2c>] (tegra_ehci_reset) from [<c07a20f4>] (usb_add_hcd+0x300/0x830)
-[<c07a20f4>] (usb_add_hcd) from [<c07d16a4>] (tegra_ehci_probe+0x3b8/0x504)
-[<c07d16a4>] (tegra_ehci_probe) from [<c06598b4>] (platform_drv_probe+0x50/0xac)
-[<c06598b4>] (platform_drv_probe) from [<c0658210>] (driver_probe_device+0x1f4/0x2b0)
-[<c0658210>] (driver_probe_device) from [<c06567a8>] (bus_for_each_drv+0x44/0x8c)
-[<c06567a8>] (bus_for_each_drv) from [<c0657f94>] (__device_attach+0x9c/0x100)
-[<c0657f94>] (__device_attach) from [<c0657628>] (bus_probe_device+0x84/0x8c)
-[<c0657628>] (bus_probe_device) from [<c0657a5c>] (deferred_probe_work_func+0x60/0x8c)
-[<c0657a5c>] (deferred_probe_work_func) from [<c025ebb4>] (process_one_work+0x134/0x338)
+[<c0220044>] (arm_dma_alloc) from [<c0306a38>] (dma_pool_alloc+0x1dc/0x210)
+[<c0306a38>] (dma_pool_alloc) from [<c07c7128>] (ehci_qh_alloc+0x3c/0xb4)
+[<c07c7128>] (ehci_qh_alloc) from [<c07cb720>] (ehci_setup+0x154/0x478)
+[<c07cb720>] (ehci_setup) from [<c07d17d8>] (tegra_ehci_reset+0xc/0x50)
+[<c07d17d8>] (tegra_ehci_reset) from [<c07a1ba0>] (usb_add_hcd+0x300/0x830)
+[<c07a1ba0>] (usb_add_hcd) from [<c07d1150>] (tegra_ehci_probe+0x3b8/0x504)
+[<c07d1150>] (tegra_ehci_probe) from [<c065938c>] (platform_drv_probe+0x50/0xac)
+[<c065938c>] (platform_drv_probe) from [<c0657ce8>] (driver_probe_device+0x1f4/0x2b0)
+[<c0657ce8>] (driver_probe_device) from [<c0656280>] (bus_for_each_drv+0x44/0x8c)
+[<c0656280>] (bus_for_each_drv) from [<c0657a6c>] (__device_attach+0x9c/0x100)
+[<c0657a6c>] (__device_attach) from [<c0657100>] (bus_probe_device+0x84/0x8c)
+[<c0657100>] (bus_probe_device) from [<c0657534>] (deferred_probe_work_func+0x60/0x8c)
+[<c0657534>] (deferred_probe_work_func) from [<c025ebb4>] (process_one_work+0x134/0x338)
 [<c025ebb4>] (process_one_work) from [<c025ee1c>] (worker_thread+0x2c/0x4f0)
 [<c025ee1c>] (worker_thread) from [<c0263fa8>] (kthread+0xd8/0xf4)
 [<c0263fa8>] (kthread) from [<c0210e78>] (ret_from_fork+0x14/0x3c)
----[ end trace af07882d720d7e8e ]---
+---[ end trace 8d72c5f8634a454f ]---
 tegra-ehci 7d008000.usb: irq 339, io mem 0x7d008000
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-07-23 21:18:12 UTC (964387092)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-07-24 17:04:16 UTC (964458256)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -346,4 +346,4 @@
 Freeing unused kernel memory: 1052K (c0e43000 - c0f4a000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.514135] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.524097] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-01-12 00:10:07.466461875 -0800
+++ /tmp/foob	2016-01-12 00:10:07.470461872 -0800
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 #################################
-	 3.2 MiB/s
+	 3.1 MiB/s
 done
-Bytes transferred = 5254872 (502ed8 hex)
+Bytes transferred = 5254512 (502d70 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x502ed8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x502d70 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-rc8-next-20160111-66728-gbf976ab (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jan 11 04:17:55 PST 2016
+Linux version 4.4.0-next-20160112-66957-gd299f5e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Jan 12 00:03:56 PST 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -90,7 +90,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0982734   (9706 kB)
+      .text : 0xc0008000 - 0xc0982720   (9706 kB)
       .init : 0xc0983000 - 0xc09d8000   ( 340 kB)
       .data : 0xc09d8000 - 0xc0a34fa0   ( 372 kB)
        .bss : 0xc0a37000 - 0xc0a92d88   ( 368 kB)
@@ -290,48 +290,48 @@
 tegra-ehci 7d008000.usb: EHCI Host Controller
 tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 1
 ------------[ cut here ]------------
-WARNING: CPU: 2 PID: 77 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
+WARNING: CPU: 0 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 2 PID: 77 Comm: kworker/u8:1 Not tainted 4.4.0-rc8-next-20160111-66728-gbf976ab #1
+CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160112-66957-gd299f5e #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0017208>] (unwind_backtrace) from [<c0012874>] (show_stack+0x10/0x14)
-[<c0012874>] (show_stack) from [<c023c444>] (dump_stack+0x84/0xd0)
-[<c023c444>] (dump_stack) from [<c0026b70>] (warn_slowpath_common+0x74/0xb0)
+[<c0012874>] (show_stack) from [<c023bf24>] (dump_stack+0x84/0xd0)
+[<c023bf24>] (dump_stack) from [<c0026b70>] (warn_slowpath_common+0x74/0xb0)
 [<c0026b70>] (warn_slowpath_common) from [<c0026c40>] (warn_slowpath_fmt+0x30/0x40)
 [<c0026c40>] (warn_slowpath_fmt) from [<c00397d0>] (check_flush_dependency+0xc8/0x12c)
 [<c00397d0>] (check_flush_dependency) from [<c00398ec>] (flush_work+0x54/0x11c)
-[<c00398ec>] (flush_work) from [<c00c0b78>] (lru_add_drain_all+0x124/0x180)
-[<c00c0b78>] (lru_add_drain_all) from [<c00f1814>] (migrate_prep+0x8/0x10)
-[<c00f1814>] (migrate_prep) from [<c00bb6f0>] (alloc_contig_range+0xc4/0x2f8)
-[<c00bb6f0>] (alloc_contig_range) from [<c00f3384>] (cma_alloc+0xd8/0x1a0)
-[<c00f3384>] (cma_alloc) from [<c001c270>] (__alloc_from_contiguous+0x38/0xdc)
+[<c00398ec>] (flush_work) from [<c00c08b0>] (lru_add_drain_all+0x124/0x180)
+[<c00c08b0>] (lru_add_drain_all) from [<c00f154c>] (migrate_prep+0x8/0x10)
+[<c00f154c>] (migrate_prep) from [<c00bb428>] (alloc_contig_range+0xc4/0x2f8)
+[<c00bb428>] (alloc_contig_range) from [<c00f30bc>] (cma_alloc+0xd8/0x1a0)
+[<c00f30bc>] (cma_alloc) from [<c001c270>] (__alloc_from_contiguous+0x38/0xdc)
 [<c001c270>] (__alloc_from_contiguous) from [<c001cf38>] (__dma_alloc+0x228/0x2ac)
 [<c001cf38>] (__dma_alloc) from [<c001d040>] (arm_dma_alloc+0x48/0x54)
-[<c001d040>] (arm_dma_alloc) from [<c00edc20>] (dma_pool_alloc+0x28c/0x304)
-[<c00edc20>] (dma_pool_alloc) from [<c03fd538>] (ehci_qh_alloc+0x3c/0xb4)
-[<c03fd538>] (ehci_qh_alloc) from [<c0401724>] (ehci_setup+0x154/0x478)
-[<c0401724>] (ehci_setup) from [<c040546c>] (tegra_ehci_reset+0xc/0x50)
-[<c040546c>] (tegra_ehci_reset) from [<c03e8f1c>] (usb_add_hcd+0x2fc/0x804)
-[<c03e8f1c>] (usb_add_hcd) from [<c0404de4>] (tegra_ehci_probe+0x3b8/0x504)
-[<c0404de4>] (tegra_ehci_probe) from [<c0336f10>] (platform_drv_probe+0x50/0xac)
-[<c0336f10>] (platform_drv_probe) from [<c0335968>] (driver_probe_device+0x1f4/0x2b0)
-[<c0335968>] (driver_probe_device) from [<c0333f04>] (bus_for_each_drv+0x44/0x8c)
-[<c0333f04>] (bus_for_each_drv) from [<c03356ec>] (__device_attach+0x9c/0x100)
-[<c03356ec>] (__device_attach) from [<c0334d80>] (bus_probe_device+0x84/0x8c)
-[<c0334d80>] (bus_probe_device) from [<c03351b4>] (deferred_probe_work_func+0x60/0x8c)
-[<c03351b4>] (deferred_probe_work_func) from [<c003a6f8>] (process_one_work+0x120/0x328)
+[<c001d040>] (arm_dma_alloc) from [<c00ed958>] (dma_pool_alloc+0x28c/0x304)
+[<c00ed958>] (dma_pool_alloc) from [<c03fd1a4>] (ehci_qh_alloc+0x3c/0xb4)
+[<c03fd1a4>] (ehci_qh_alloc) from [<c0401390>] (ehci_setup+0x154/0x478)
+[<c0401390>] (ehci_setup) from [<c04050d8>] (tegra_ehci_reset+0xc/0x50)
+[<c04050d8>] (tegra_ehci_reset) from [<c03e8b88>] (usb_add_hcd+0x2fc/0x804)
+[<c03e8b88>] (usb_add_hcd) from [<c0404a50>] (tegra_ehci_probe+0x3b8/0x504)
+[<c0404a50>] (tegra_ehci_probe) from [<c0336b7c>] (platform_drv_probe+0x50/0xac)
+[<c0336b7c>] (platform_drv_probe) from [<c03355d4>] (driver_probe_device+0x1f4/0x2b0)
+[<c03355d4>] (driver_probe_device) from [<c0333b70>] (bus_for_each_drv+0x44/0x8c)
+[<c0333b70>] (bus_for_each_drv) from [<c0335358>] (__device_attach+0x9c/0x100)
+[<c0335358>] (__device_attach) from [<c03349ec>] (bus_probe_device+0x84/0x8c)
+[<c03349ec>] (bus_probe_device) from [<c0334e20>] (deferred_probe_work_func+0x60/0x8c)
+[<c0334e20>] (deferred_probe_work_func) from [<c003a6f8>] (process_one_work+0x120/0x328)
 [<c003a6f8>] (process_one_work) from [<c003a964>] (worker_thread+0x2c/0x4b4)
 [<c003a964>] (worker_thread) from [<c003fa9c>] (kthread+0xd8/0xf4)
 [<c003fa9c>] (kthread) from [<c000f938>] (ret_from_fork+0x14/0x3c)
----[ end trace b3f6ad3f2a9fbcb4 ]---
+---[ end trace fc42758130390674 ]---
 tegra-ehci 7d008000.usb: irq 339, io mem 0x7d008000
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-07-23 21:18:40 UTC (964387120)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-07-24 17:04:45 UTC (964458285)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-01-12 00:10:07.474461868 -0800
+++ /tmp/foob	2016-01-12 00:10:07.478461864 -0800
@@ -43,7 +43,7 @@
 	 ######
 	 3.2 MiB/s
 done
-Bytes transferred = 6763968 (6735c0 hex)
+Bytes transferred = 6761880 (672d98 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -54,7 +54,7 @@
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6735c0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x672d98 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012a89
@@ -62,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-rc8-next-20160111-66728-gbf976ab (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jan 11 04:17:41 PST 2016
+Linux version 4.4.0-next-20160112-66957-gd299f5e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Jan 12 00:03:39 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -71,7 +71,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c105dc40, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c105dd00, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -84,7 +84,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1997844K/2096128K available (8495K kernel code, 1110K rwdata, 4024K rodata, 1052K init, 345K bss, 32748K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1997844K/2096128K available (8495K kernel code, 1111K rwdata, 4024K rodata, 1052K init, 345K bss, 32748K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,9 +92,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0e42df0   (12524 kB)
+      .text : 0xc0208000 - 0xc0e42de8   (12524 kB)
       .init : 0xc0e43000 - 0xc0f4a000   (1052 kB)
-      .data : 0xc0f4a000 - 0xc105fbc0   (1111 kB)
+      .data : 0xc0f4a000 - 0xc105fc80   (1112 kB)
        .bss : 0xc1062000 - 0xc10b85a0   ( 346 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -333,43 +333,43 @@
 tegra-ahci 70027000.sata: AHCI 0001.0301 32 slots 2 ports 3 Gbps 0x1 impl platform mode
 tegra-ahci 70027000.sata: flags: 64bit ncq sntf pm led pio slum part sadm sds apst 
 ------------[ cut here ]------------
-WARNING: CPU: 1 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
+WARNING: CPU: 2 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 1 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-rc8-next-20160111-66728-gbf976ab #1
+CPU: 2 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160112-66957-gd299f5e #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c02188b8>] (unwind_backtrace) from [<c0213f94>] (show_stack+0x10/0x14)
-[<c0213f94>] (show_stack) from [<c048972c>] (dump_stack+0x78/0x94)
-[<c048972c>] (dump_stack) from [<c024ab38>] (warn_slowpath_common+0x74/0xb0)
+[<c0213f94>] (show_stack) from [<c048918c>] (dump_stack+0x78/0x94)
+[<c048918c>] (dump_stack) from [<c024ab38>] (warn_slowpath_common+0x74/0xb0)
 [<c024ab38>] (warn_slowpath_common) from [<c024ac08>] (warn_slowpath_fmt+0x30/0x40)
 [<c024ac08>] (warn_slowpath_fmt) from [<c025dc58>] (check_flush_dependency+0xc8/0x124)
 [<c025dc58>] (check_flush_dependency) from [<c025dd6c>] (flush_work+0x54/0x144)
-[<c025dd6c>] (flush_work) from [<c02dae8c>] (lru_add_drain_all+0x124/0x180)
-[<c02dae8c>] (lru_add_drain_all) from [<c030c7ec>] (migrate_prep+0x8/0x10)
-[<c030c7ec>] (migrate_prep) from [<c02d5ea4>] (alloc_contig_range+0xc4/0x2f8)
-[<c02d5ea4>] (alloc_contig_range) from [<c030e118>] (cma_alloc+0xd8/0x1a0)
-[<c030e118>] (cma_alloc) from [<c021e39c>] (__alloc_from_contiguous+0x38/0xdc)
+[<c025dd6c>] (flush_work) from [<c02dab88>] (lru_add_drain_all+0x124/0x180)
+[<c02dab88>] (lru_add_drain_all) from [<c030c4e8>] (migrate_prep+0x8/0x10)
+[<c030c4e8>] (migrate_prep) from [<c02d5ba0>] (alloc_contig_range+0xc4/0x2f8)
+[<c02d5ba0>] (alloc_contig_range) from [<c030de14>] (cma_alloc+0xd8/0x1a0)
+[<c030de14>] (cma_alloc) from [<c021e39c>] (__alloc_from_contiguous+0x38/0xdc)
 [<c021e39c>] (__alloc_from_contiguous) from [<c021ff8c>] (__dma_alloc+0x21c/0x250)
 [<c021ff8c>] (__dma_alloc) from [<c0220044>] (arm_dma_alloc+0x48/0x54)
-[<c0220044>] (arm_dma_alloc) from [<c066a9b4>] (dmam_alloc_coherent+0x8c/0xf0)
-[<c066a9b4>] (dmam_alloc_coherent) from [<c06c549c>] (ahci_port_start+0xd0/0x1e0)
-[<c06c549c>] (ahci_port_start) from [<c06ad344>] (ata_host_start+0xf8/0x1d4)
-[<c06ad344>] (ata_host_start) from [<c06b2260>] (ata_host_activate+0x20/0x134)
-[<c06b2260>] (ata_host_activate) from [<c06c4918>] (ahci_host_activate+0x3c/0x11c)
-[<c06c4918>] (ahci_host_activate) from [<c06c685c>] (ahci_platform_init_host+0x1cc/0x3c8)
-[<c06c685c>] (ahci_platform_init_host) from [<c06c8ac0>] (tegra_ahci_probe+0x404/0x4c4)
-[<c06c8ac0>] (tegra_ahci_probe) from [<c06598b4>] (platform_drv_probe+0x50/0xac)
-[<c06598b4>] (platform_drv_probe) from [<c0658210>] (driver_probe_device+0x1f4/0x2b0)
-[<c0658210>] (driver_probe_device) from [<c06567a8>] (bus_for_each_drv+0x44/0x8c)
-[<c06567a8>] (bus_for_each_drv) from [<c0657f94>] (__device_attach+0x9c/0x100)
-[<c0657f94>] (__device_attach) from [<c0657628>] (bus_probe_device+0x84/0x8c)
-[<c0657628>] (bus_probe_device) from [<c0657a5c>] (deferred_probe_work_func+0x60/0x8c)
-[<c0657a5c>] (deferred_probe_work_func) from [<c025ebb4>] (process_one_work+0x134/0x338)
+[<c0220044>] (arm_dma_alloc) from [<c066a48c>] (dmam_alloc_coherent+0x8c/0xf0)
+[<c066a48c>] (dmam_alloc_coherent) from [<c06c4f54>] (ahci_port_start+0xd0/0x1e0)
+[<c06c4f54>] (ahci_port_start) from [<c06acdfc>] (ata_host_start+0xf8/0x1d4)
+[<c06acdfc>] (ata_host_start) from [<c06b1d18>] (ata_host_activate+0x20/0x134)
+[<c06b1d18>] (ata_host_activate) from [<c06c43d0>] (ahci_host_activate+0x3c/0x11c)
+[<c06c43d0>] (ahci_host_activate) from [<c06c6314>] (ahci_platform_init_host+0x1cc/0x3c8)
+[<c06c6314>] (ahci_platform_init_host) from [<c06c8578>] (tegra_ahci_probe+0x404/0x4c4)
+[<c06c8578>] (tegra_ahci_probe) from [<c065938c>] (platform_drv_probe+0x50/0xac)
+[<c065938c>] (platform_drv_probe) from [<c0657ce8>] (driver_probe_device+0x1f4/0x2b0)
+[<c0657ce8>] (driver_probe_device) from [<c0656280>] (bus_for_each_drv+0x44/0x8c)
+[<c0656280>] (bus_for_each_drv) from [<c0657a6c>] (__device_attach+0x9c/0x100)
+[<c0657a6c>] (__device_attach) from [<c0657100>] (bus_probe_device+0x84/0x8c)
+[<c0657100>] (bus_probe_device) from [<c0657534>] (deferred_probe_work_func+0x60/0x8c)
+[<c0657534>] (deferred_probe_work_func) from [<c025ebb4>] (process_one_work+0x134/0x338)
 [<c025ebb4>] (process_one_work) from [<c025ee1c>] (worker_thread+0x2c/0x4f0)
 [<c025ee1c>] (worker_thread) from [<c0263fa8>] (kthread+0xd8/0xf4)
 [<c0263fa8>] (kthread) from [<c0210e78>] (ret_from_fork+0x14/0x3c)
----[ end trace 9bb0737ca220652e ]---
+---[ end trace d0cb35dcd0a0d9af ]---
 scsi host0: tegra-ahci
 scsi host1: tegra-ahci
 ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 341
@@ -403,9 +403,9 @@
 ata1: SATA link down (SStatus 0 SControl 300)
 EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
 EXT4-fs (mmcblk0p1): couldn't mount as ext2 due to feature incompatibilities
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 1052K (c0e43000 - c0f4a000)
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-01-12 00:10:07.482461861 -0800
+++ /tmp/foob	2016-01-12 00:10:07.486461857 -0800
@@ -41,18 +41,18 @@
 	 #################################
 	 3.2 MiB/s
 done
-Bytes transferred = 5254872 (502ed8 hex)
+Bytes transferred = 5254512 (502d70 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 2.8 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x502ed8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x502d70 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012a89
@@ -61,7 +61,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-rc8-next-20160111-66728-gbf976ab (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jan 11 04:17:55 PST 2016
+Linux version 4.4.0-next-20160112-66957-gd299f5e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Jan 12 00:03:56 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -92,7 +92,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0982734   (9706 kB)
+      .text : 0xc0008000 - 0xc0982720   (9706 kB)
       .init : 0xc0983000 - 0xc09d8000   ( 340 kB)
       .data : 0xc09d8000 - 0xc0a34fa0   ( 372 kB)
        .bss : 0xc0a37000 - 0xc0a92d88   ( 368 kB)
@@ -356,40 +356,40 @@
 WARNING: CPU: 0 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-rc8-next-20160111-66728-gbf976ab #1
+CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160112-66957-gd299f5e #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0017208>] (unwind_backtrace) from [<c0012874>] (show_stack+0x10/0x14)
-[<c0012874>] (show_stack) from [<c023c444>] (dump_stack+0x84/0xd0)
-[<c023c444>] (dump_stack) from [<c0026b70>] (warn_slowpath_common+0x74/0xb0)
+[<c0012874>] (show_stack) from [<c023bf24>] (dump_stack+0x84/0xd0)
+[<c023bf24>] (dump_stack) from [<c0026b70>] (warn_slowpath_common+0x74/0xb0)
 [<c0026b70>] (warn_slowpath_common) from [<c0026c40>] (warn_slowpath_fmt+0x30/0x40)
 [<c0026c40>] (warn_slowpath_fmt) from [<c00397d0>] (check_flush_dependency+0xc8/0x12c)
 [<c00397d0>] (check_flush_dependency) from [<c00398ec>] (flush_work+0x54/0x11c)
-[<c00398ec>] (flush_work) from [<c00c0b78>] (lru_add_drain_all+0x124/0x180)
-[<c00c0b78>] (lru_add_drain_all) from [<c00f1814>] (migrate_prep+0x8/0x10)
-[<c00f1814>] (migrate_prep) from [<c00bb6f0>] (alloc_contig_range+0xc4/0x2f8)
-[<c00bb6f0>] (alloc_contig_range) from [<c00f3384>] (cma_alloc+0xd8/0x1a0)
-[<c00f3384>] (cma_alloc) from [<c001c270>] (__alloc_from_contiguous+0x38/0xdc)
+[<c00398ec>] (flush_work) from [<c00c08b0>] (lru_add_drain_all+0x124/0x180)
+[<c00c08b0>] (lru_add_drain_all) from [<c00f154c>] (migrate_prep+0x8/0x10)
+[<c00f154c>] (migrate_prep) from [<c00bb428>] (alloc_contig_range+0xc4/0x2f8)
+[<c00bb428>] (alloc_contig_range) from [<c00f30bc>] (cma_alloc+0xd8/0x1a0)
+[<c00f30bc>] (cma_alloc) from [<c001c270>] (__alloc_from_contiguous+0x38/0xdc)
 [<c001c270>] (__alloc_from_contiguous) from [<c001cf38>] (__dma_alloc+0x228/0x2ac)
 [<c001cf38>] (__dma_alloc) from [<c001d040>] (arm_dma_alloc+0x48/0x54)
-[<c001d040>] (arm_dma_alloc) from [<c0345ee4>] (dmam_alloc_coherent+0x8c/0xf0)
-[<c0345ee4>] (dmam_alloc_coherent) from [<c038bf8c>] (ahci_port_start+0xd0/0x1e0)
-[<c038bf8c>] (ahci_port_start) from [<c0373c64>] (ata_host_start+0xf8/0x1d4)
-[<c0373c64>] (ata_host_start) from [<c0378bf8>] (ata_host_activate+0x20/0x134)
-[<c0378bf8>] (ata_host_activate) from [<c038b434>] (ahci_host_activate+0x3c/0x11c)
-[<c038b434>] (ahci_host_activate) from [<c038d7dc>] (ahci_platform_init_host+0x1cc/0x3c8)
-[<c038d7dc>] (ahci_platform_init_host) from [<c038d4a8>] (tegra_ahci_probe+0x404/0x4c4)
-[<c038d4a8>] (tegra_ahci_probe) from [<c0336f10>] (platform_drv_probe+0x50/0xac)
-[<c0336f10>] (platform_drv_probe) from [<c0335968>] (driver_probe_device+0x1f4/0x2b0)
-[<c0335968>] (driver_probe_device) from [<c0333f04>] (bus_for_each_drv+0x44/0x8c)
-[<c0333f04>] (bus_for_each_drv) from [<c03356ec>] (__device_attach+0x9c/0x100)
-[<c03356ec>] (__device_attach) from [<c0334d80>] (bus_probe_device+0x84/0x8c)
-[<c0334d80>] (bus_probe_device) from [<c03351b4>] (deferred_probe_work_func+0x60/0x8c)
-[<c03351b4>] (deferred_probe_work_func) from [<c003a6f8>] (process_one_work+0x120/0x328)
+[<c001d040>] (arm_dma_alloc) from [<c0345b50>] (dmam_alloc_coherent+0x8c/0xf0)
+[<c0345b50>] (dmam_alloc_coherent) from [<c038bbf8>] (ahci_port_start+0xd0/0x1e0)
+[<c038bbf8>] (ahci_port_start) from [<c03738d0>] (ata_host_start+0xf8/0x1d4)
+[<c03738d0>] (ata_host_start) from [<c0378864>] (ata_host_activate+0x20/0x134)
+[<c0378864>] (ata_host_activate) from [<c038b0a0>] (ahci_host_activate+0x3c/0x11c)
+[<c038b0a0>] (ahci_host_activate) from [<c038d448>] (ahci_platform_init_host+0x1cc/0x3c8)
+[<c038d448>] (ahci_platform_init_host) from [<c038d114>] (tegra_ahci_probe+0x404/0x4c4)
+[<c038d114>] (tegra_ahci_probe) from [<c0336b7c>] (platform_drv_probe+0x50/0xac)
+[<c0336b7c>] (platform_drv_probe) from [<c03355d4>] (driver_probe_device+0x1f4/0x2b0)
+[<c03355d4>] (driver_probe_device) from [<c0333b70>] (bus_for_each_drv+0x44/0x8c)
+[<c0333b70>] (bus_for_each_drv) from [<c0335358>] (__device_attach+0x9c/0x100)
+[<c0335358>] (__device_attach) from [<c03349ec>] (bus_probe_device+0x84/0x8c)
+[<c03349ec>] (bus_probe_device) from [<c0334e20>] (deferred_probe_work_func+0x60/0x8c)
+[<c0334e20>] (deferred_probe_work_func) from [<c003a6f8>] (process_one_work+0x120/0x328)
 [<c003a6f8>] (process_one_work) from [<c003a964>] (worker_thread+0x2c/0x4b4)
 [<c003a964>] (worker_thread) from [<c003fa9c>] (kthread+0xd8/0xf4)
 [<c003fa9c>] (kthread) from [<c000f938>] (ret_from_fork+0x14/0x3c)
----[ end trace 9f7d714f8afa6ee8 ]---
+---[ end trace e0a9b659aa52320d ]---
 scsi host0: tegra-ahci
 scsi host1: tegra-ahci
 ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 341
@@ -433,4 +433,4 @@
 usb 2-1: new high-speed USB device number 2 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    5.760819] asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:11:31:3b
+root@(none):/# [    5.719985] asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:11:31:3b
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-01-12 00:10:07.490461854 -0800
+++ /tmp/foob	2016-01-12 00:10:07.494461849 -0800
@@ -47,20 +47,20 @@
 	 #################################################################
 	 #################################################################
 	 ######
-	 4.1 MiB/s
+	 4 MiB/s
 done
-Bytes transferred = 6763968 (6735c0 hex)
+Bytes transferred = 6761880 (672d98 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 2.5 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x6735c0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x672d98 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-rc8-next-20160111-66728-gbf976ab (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jan 11 04:17:41 PST 2016
+Linux version 4.4.0-next-20160112-66957-gd299f5e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Jan 12 00:03:39 PST 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -76,7 +76,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c105dc40, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c105dd00, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -89,7 +89,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 958768K/1048576K available (8495K kernel code, 1110K rwdata, 4024K rodata, 1052K init, 345K bss, 24272K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 958768K/1048576K available (8495K kernel code, 1111K rwdata, 4024K rodata, 1052K init, 345K bss, 24272K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -97,9 +97,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0e42df0   (12524 kB)
+      .text : 0xc0208000 - 0xc0e42de8   (12524 kB)
       .init : 0xc0e43000 - 0xc0f4a000   (1052 kB)
-      .data : 0xc0f4a000 - 0xc105fbc0   (1111 kB)
+      .data : 0xc0f4a000 - 0xc105fc80   (1112 kB)
        .bss : 0xc1062000 - 0xc10b85a0   ( 346 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
@@ -325,23 +325,109 @@
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
  mmcblk0: p1
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
-tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
-hub 2-0:1.0: USB hub found
-hub 2-0:1.0: 1 port detected
-input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-mmc1: new high speed SD card at address b368
-rtc-em3027 2-0056: setting system clock to 2014-12-27 06:48:54 UTC (1419662934)
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
- mmcblk1: p1 p2
-EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
-hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
-usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
-EXT4-fs (mmcblk0p1): recovery complete
-EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:1.
-devtmpfs: mounted
-Freeing unused kernel memory: 1052K (c0e43000 - c0f4a000)
-bash: cannot set terminal process group (-1): Inappropriate ioctl for device
-bash: no job control in this shell
-root@(none):/# 
\ No newline at end of file
+NMI watchdog: BUG: soft lockup - CPU#1 stuck for 23s! [kworker/1:1:67]
+Modules linked in:
+CPU: 1 PID: 67 Comm: kworker/1:1 Not tainted 4.4.0-next-20160112-66957-gd299f5e #1
+Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
+Workqueue: events dbs_work_handler
+task: eed7c000 ti: eed7a000 task.ti: eed7a000
+PC is at smp_call_function_single+0xfc/0x15c
+LR is at arch_send_call_function_single_ipi+0x34/0x38
+pc : [<c02adc0c>]    lr : [<c0216e50>]    psr: 00000013
+sp : eed7bd10  ip : 00000001  fp : eed7be30
+r10: eed7bdb4  r9 : 00000002  r8 : 00000002
+r7 : 00000000  r6 : c0217c98  r5 : eed7bdbc  r4 : 00000001
+r3 : 00000003  r2 : c0f4c864  r1 : 20000013  r0 : 00000000
+Flags: nzcv  IRQs on  FIQs on  Mode SVC_32  ISA ARM  Segment none
+Control: 10c5387d  Table: 0020404a  DAC: 00000051
+CPU: 1 PID: 67 Comm: kworker/1:1 Not tainted 4.4.0-next-20160112-66957-gd299f5e #1
+Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
+Workqueue: events dbs_work_handler
+[<c02188b8>] (unwind_backtrace) from [<c0213f94>] (show_stack+0x10/0x14)
+[<c0213f94>] (show_stack) from [<c048918c>] (dump_stack+0x78/0x94)
+[<c048918c>] (dump_stack) from [<c02bae3c>] (watchdog_timer_fn+0x1b4/0x230)
+[<c02bae3c>] (watchdog_timer_fn) from [<c029a5bc>] (__hrtimer_run_queues+0x114/0x198)
+[<c029a5bc>] (__hrtimer_run_queues) from [<c029ac78>] (hrtimer_interrupt+0xb0/0x214)
+[<c029ac78>] (hrtimer_interrupt) from [<c0217ce8>] (twd_handler+0x2c/0x40)
+[<c0217ce8>] (twd_handler) from [<c028e034>] (handle_percpu_devid_irq+0x68/0x84)
+[<c028e034>] (handle_percpu_devid_irq) from [<c028a0d4>] (generic_handle_irq+0x18/0x28)
+[<c028a0d4>] (generic_handle_irq) from [<c028a1d0>] (__handle_domain_irq+0x54/0xb4)
+[<c028a1d0>] (__handle_domain_irq) from [<c020a718>] (gic_handle_irq+0x50/0x98)
+[<c020a718>] (gic_handle_irq) from [<c0214ad4>] (__irq_svc+0x54/0x70)
+Exception stack(0xeed7bcc0 to 0xeed7bd08)
+bcc0: 00000000 20000013 c0f4c864 00000003 00000001 eed7bdbc c0217c98 00000000
+bce0: 00000002 00000002 eed7bdb4 eed7be30 00000001 eed7bd10 c0216e50 c02adc0c
+bd00: 00000013 ffffffff
+[<c0214ad4>] (__irq_svc) from [<c02adc0c>] (smp_call_function_single+0xfc/0x15c)
+[<c02adc0c>] (smp_call_function_single) from [<c02ae03c>] (smp_call_function+0x2c/0x34)
+[<c02ae03c>] (smp_call_function) from [<c02ae06c>] (on_each_cpu+0x10/0x30)
+[<c02ae06c>] (on_each_cpu) from [<c0217d20>] (twd_rate_change+0x24/0x30)
+[<c0217d20>] (twd_rate_change) from [<c0264c00>] (notifier_call_chain+0x44/0x84)
+[<c0264c00>] (notifier_call_chain) from [<c0264d78>] (__srcu_notifier_call_chain+0x44/0x60)
+[<c0264d78>] (__srcu_notifier_call_chain) from [<c0264dac>] (srcu_notifier_call_chain+0x18/0x20)
+[<c0264dac>] (srcu_notifier_call_chain) from [<c08bb7bc>] (__clk_notify+0x6c/0x74)
+[<c08bb7bc>] (__clk_notify) from [<c08bb858>] (__clk_recalc_rates+0x94/0x98)
+[<c08bb858>] (__clk_recalc_rates) from [<c08bb830>] (__clk_recalc_rates+0x6c/0x98)
+[<c08bb830>] (__clk_recalc_rates) from [<c08bd4f8>] (clk_core_set_parent+0x104/0x16c)
+[<c08bd4f8>] (clk_core_set_parent) from [<c085ac88>] (tegra_target+0x94/0x130)
+[<c085ac88>] (tegra_target) from [<c0854b90>] (__cpufreq_driver_target+0x178/0x298)
+[<c0854b90>] (__cpufreq_driver_target) from [<c08589f8>] (dbs_check_cpu+0x1a0/0x1ec)
+[<c08589f8>] (dbs_check_cpu) from [<c08584ac>] (od_dbs_timer+0x4c/0xdc)
+[<c08584ac>] (od_dbs_timer) from [<c0858d38>] (dbs_work_handler+0x154/0x1a0)
+[<c0858d38>] (dbs_work_handler) from [<c025ebb4>] (process_one_work+0x134/0x338)
+[<c025ebb4>] (process_one_work) from [<c025ee1c>] (worker_thread+0x2c/0x4f0)
+[<c025ee1c>] (worker_thread) from [<c0263fa8>] (kthread+0xd8/0xf4)
+[<c0263fa8>] (kthread) from [<c0210e78>] (ret_from_fork+0x14/0x3c)
+NMI watchdog: BUG: soft lockup - CPU#1 stuck for 23s! [kworker/1:1:67]
+Modules linked in:
+CPU: 1 PID: 67 Comm: kworker/1:1 Tainted: G             L  4.4.0-next-20160112-66957-gd299f5e #1
+Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
+Workqueue: events dbs_work_handler
+task: eed7c000 ti: eed7a000 task.ti: eed7a000
+PC is at smp_call_function_single+0xfc/0x15c
+LR is at arch_send_call_function_single_ipi+0x34/0x38
+pc : [<c02adc0c>]    lr : [<c0216e50>]    psr: 00000013
+sp : eed7bd10  ip : 00000001  fp : eed7be30
+r10: eed7bdb4  r9 : 00000002  r8 : 00000002
+r7 : 00000000  r6 : c0217c98  r5 : eed7bdbc  r4 : 00000001
+r3 : 00000003  r2 : c0f4c864  r1 : 20000013  r0 : 00000000
+Flags: nzcv  IRQs on  FIQs on  Mode SVC_32  ISA ARM  Segment none
+Control: 10c5387d  Table: 0020404a  DAC: 00000051
+CPU: 1 PID: 67 Comm: kworker/1:1 Tainted: G             L  4.4.0-next-20160112-66957-gd299f5e #1
+Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
+Workqueue: events dbs_work_handler
+[<c02188b8>] (unwind_backtrace) from [<c0213f94>] (show_stack+0x10/0x14)
+[<c0213f94>] (show_stack) from [<c048918c>] (dump_stack+0x78/0x94)
+[<c048918c>] (dump_stack) from [<c02bae3c>] (watchdog_timer_fn+0x1b4/0x230)
+[<c02bae3c>] (watchdog_timer_fn) from [<c029a5bc>] (__hrtimer_run_queues+0x114/0x198)
+[<c029a5bc>] (__hrtimer_run_queues) from [<c029ac78>] (hrtimer_interrupt+0xb0/0x214)
+[<c029ac78>] (hrtimer_interrupt) from [<c0217ce8>] (twd_handler+0x2c/0x40)
+[<c0217ce8>] (twd_handler) from [<c028e034>] (handle_percpu_devid_irq+0x68/0x84)
+[<c028e034>] (handle_percpu_devid_irq) from [<c028a0d4>] (generic_handle_irq+0x18/0x28)
+[<c028a0d4>] (generic_handle_irq) from [<c028a1d0>] (__handle_domain_irq+0x54/0xb4)
+[<c028a1d0>] (__handle_domain_irq) from [<c020a718>] (gic_handle_irq+0x50/0x98)
+[<c020a718>] (gic_handle_irq) from [<c0214ad4>] (__irq_svc+0x54/0x70)
+Exception stack(0xeed7bcc0 to 0xeed7bd08)
+bcc0: 00000000 20000013 c0f4c864 00000003 00000001 eed7bdbc c0217c98 00000000
+bce0: 00000002 00000002 eed7bdb4 eed7be30 00000001 eed7bd10 c0216e50 c02adc0c
+bd00: 00000013 ffffffff
+[<c0214ad4>] (__irq_svc) from [<c02adc0c>] (smp_call_function_single+0xfc/0x15c)
+[<c02adc0c>] (smp_call_function_single) from [<c02ae03c>] (smp_call_function+0x2c/0x34)
+[<c02ae03c>] (smp_call_function) from [<c02ae06c>] (on_each_cpu+0x10/0x30)
+[<c02ae06c>] (on_each_cpu) from [<c0217d20>] (twd_rate_change+0x24/0x30)
+[<c0217d20>] (twd_rate_change) from [<c0264c00>] (notifier_call_chain+0x44/0x84)
+[<c0264c00>] (notifier_call_chain) from [<c0264d78>] (__srcu_notifier_call_chain+0x44/0x60)
+[<c0264d78>] (__srcu_notifier_call_chain) from [<c0264dac>] (srcu_notifier_call_chain+0x18/0x20)
+[<c0264dac>] (srcu_notifier_call_chain) from [<c08bb7bc>] (__clk_notify+0x6c/0x74)
+[<c08bb7bc>] (__clk_notify) from [<c08bb858>] (__clk_recalc_rates+0x94/0x98)
+[<c08bb858>] (__clk_recalc_rates) from [<c08bb830>] (__clk_recalc_rates+0x6c/0x98)
+[<c08bb830>] (__clk_recalc_rates) from [<c08bd4f8>] (clk_core_set_parent+0x104/0x16c)
+[<c08bd4f8>] (clk_core_set_parent) from [<c085ac88>] (tegra_target+0x94/0x130)
+[<c085ac88>] (tegra_target) from [<c0854b90>] (__cpufreq_driver_target+0x178/0x298)
+[<c0854b90>] (__cpufreq_driver_target) from [<c08589f8>] (dbs_check_cpu+0x1a0/0x1ec)
+[<c08589f8>] (dbs_check_cpu) from [<c08584ac>] (od_dbs_timer+0x4c/0xdc)
+[<c08584ac>] (od_dbs_timer) from [<c0858d38>] (dbs_work_handler+0x154/0x1a0)
+[<c0858d38>] (dbs_work_handler) from [<c025ebb4>] (process_one_work+0x134/0x338)
+[<c025ebb4>] (process_one_work) from [<c025ee1c>] (worker_thread+0x2c/0x4f0)
+[<c025ee1c>] (worker_thread) from [<c0263fa8>] (kthread+0xd8/0xf4)
+[<c0263fa8>] (kthread) from [<c0210e78>] (ret_from_fork+0x14/0x3c)
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-01-12 00:10:07.502461842 -0800
+++ /tmp/foob	2016-01-12 00:10:07.502461842 -0800
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -45,20 +45,20 @@
 	 #################################################################
 	 #################################################################
 	 #################################
-	 4 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 5254872 (502ed8 hex)
+Bytes transferred = 5254512 (502d70 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x502ed8 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x502d70 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-rc8-next-20160111-66728-gbf976ab (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jan 11 04:17:55 PST 2016
+Linux version 4.4.0-next-20160112-66957-gd299f5e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Jan 12 00:03:56 PST 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -97,7 +97,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0982734   (9706 kB)
+      .text : 0xc0008000 - 0xc0982720   (9706 kB)
       .init : 0xc0983000 - 0xc09d8000   ( 340 kB)
       .data : 0xc09d8000 - 0xc0a34fa0   ( 372 kB)
        .bss : 0xc0a37000 - 0xc0a92d88   ( 368 kB)
@@ -308,13 +308,15 @@
 NET: Registered protocol family 10
 mmc0: host does not support reading read-only switch, assuming write-enable
 mip6: Mobile IPv6
-mmc1: new high speed SD card at address b368
-mmc0: new high speed SDHC card at address b368
-mmcblk0: mmc1:b368 NCard 1.83 GiB (ro)
-mmcblk1: mmc0:b368 NCard 3.70 GiB 
 sit: IPv6 over IPv4 tunneling driver
+mmc0: new high speed SDHC card at address b368
 NET: Registered protocol family 17
+mmc1: new high speed SD card at address b368
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
+ mmcblk0: p1
 NET: Registered protocol family 15
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
+ mmcblk1: p1 p2
 can: controller area network core (rev 20120528 abi 9)
 NET: Registered protocol family 29
 can: raw protocol (rev 20120528)
@@ -328,28 +330,26 @@
 Bluetooth: HIDP socket layer initialized
 Key type dns_resolver registered
 Registering SWP/SWPB emulation handler
+hub 1-1:1.0: USB hub found
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
- mmcblk0: p1 p2
- mmcblk1: p1
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
-hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-27 06:49:12 UTC (1419662952)
+rtc-em3027 2-0056: setting system clock to 2014-12-27 06:54:16 UTC (1419663256)
 ALSA device list:
   #0: tegra-trimslice
-EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
-EXT4-fs (mmcblk1p1): recovery complete
-EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:17.
+hub 1-1:1.0: 4 ports detected
+EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
+usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
+EXT4-fs (mmcblk0p1): recovery complete
+EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
+VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 340K (c0983000 - c09d8000)
-usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-01-12 00:10:07.510461836 -0800
+++ /tmp/foob	2016-01-12 00:10:07.510461836 -0800
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 ######
-	 3.4 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 6763968 (6735c0 hex)
+Bytes transferred = 6761880 (672d98 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.9 MiB/s
+	 3.5 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6735c0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x672d98 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-rc8-next-20160111-66728-gbf976ab (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jan 11 04:17:41 PST 2016
+Linux version 4.4.0-next-20160112-66957-gd299f5e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Jan 12 00:03:39 PST 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,7 +68,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c105dc40, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c105dd00, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -81,7 +81,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1997872K/2096128K available (8495K kernel code, 1110K rwdata, 4024K rodata, 1052K init, 345K bss, 32720K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1997872K/2096128K available (8495K kernel code, 1111K rwdata, 4024K rodata, 1052K init, 345K bss, 32720K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -89,9 +89,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0e42df0   (12524 kB)
+      .text : 0xc0208000 - 0xc0e42de8   (12524 kB)
       .init : 0xc0e43000 - 0xc0f4a000   (1052 kB)
-      .data : 0xc0f4a000 - 0xc105fbc0   (1111 kB)
+      .data : 0xc0f4a000 - 0xc105fc80   (1112 kB)
        .bss : 0xc1062000 - 0xc10b85a0   ( 346 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -287,8 +287,8 @@
 mmcblk0boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk0rpmb: mmc1:0001 SEM16G partition 3 128 KiB
 Primary GPT is invalid, using alternate GPT.
-tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
  mmcblk0: p1 p2 p3 p4 p5
+tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -344,39 +344,39 @@
 WARNING: CPU: 1 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 1 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-rc8-next-20160111-66728-gbf976ab #1
+CPU: 1 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160112-66957-gd299f5e #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c02188b8>] (unwind_backtrace) from [<c0213f94>] (show_stack+0x10/0x14)
-[<c0213f94>] (show_stack) from [<c048972c>] (dump_stack+0x78/0x94)
-[<c048972c>] (dump_stack) from [<c024ab38>] (warn_slowpath_common+0x74/0xb0)
+[<c0213f94>] (show_stack) from [<c048918c>] (dump_stack+0x78/0x94)
+[<c048918c>] (dump_stack) from [<c024ab38>] (warn_slowpath_common+0x74/0xb0)
 [<c024ab38>] (warn_slowpath_common) from [<c024ac08>] (warn_slowpath_fmt+0x30/0x40)
 [<c024ac08>] (warn_slowpath_fmt) from [<c025dc58>] (check_flush_dependency+0xc8/0x124)
 [<c025dc58>] (check_flush_dependency) from [<c025dd6c>] (flush_work+0x54/0x144)
-[<c025dd6c>] (flush_work) from [<c02dae8c>] (lru_add_drain_all+0x124/0x180)
-[<c02dae8c>] (lru_add_drain_all) from [<c030c7ec>] (migrate_prep+0x8/0x10)
-[<c030c7ec>] (migrate_prep) from [<c02d5ea4>] (alloc_contig_range+0xc4/0x2f8)
-[<c02d5ea4>] (alloc_contig_range) from [<c030e118>] (cma_alloc+0xd8/0x1a0)
-[<c030e118>] (cma_alloc) from [<c021e39c>] (__alloc_from_contiguous+0x38/0xdc)
+[<c025dd6c>] (flush_work) from [<c02dab88>] (lru_add_drain_all+0x124/0x180)
+[<c02dab88>] (lru_add_drain_all) from [<c030c4e8>] (migrate_prep+0x8/0x10)
+[<c030c4e8>] (migrate_prep) from [<c02d5ba0>] (alloc_contig_range+0xc4/0x2f8)
+[<c02d5ba0>] (alloc_contig_range) from [<c030de14>] (cma_alloc+0xd8/0x1a0)
+[<c030de14>] (cma_alloc) from [<c021e39c>] (__alloc_from_contiguous+0x38/0xdc)
 [<c021e39c>] (__alloc_from_contiguous) from [<c021ff8c>] (__dma_alloc+0x21c/0x250)
 [<c021ff8c>] (__dma_alloc) from [<c0220044>] (arm_dma_alloc+0x48/0x54)
-[<c0220044>] (arm_dma_alloc) from [<c0306d3c>] (dma_pool_alloc+0x1dc/0x210)
-[<c0306d3c>] (dma_pool_alloc) from [<c07c767c>] (ehci_qh_alloc+0x3c/0xb4)
-[<c07c767c>] (ehci_qh_alloc) from [<c07cbc74>] (ehci_setup+0x154/0x478)
-[<c07cbc74>] (ehci_setup) from [<c07d1d2c>] (tegra_ehci_reset+0xc/0x50)
-[<c07d1d2c>] (tegra_ehci_reset) from [<c07a20f4>] (usb_add_hcd+0x300/0x830)
-[<c07a20f4>] (usb_add_hcd) from [<c07d16a4>] (tegra_ehci_probe+0x3b8/0x504)
-[<c07d16a4>] (tegra_ehci_probe) from [<c06598b4>] (platform_drv_probe+0x50/0xac)
-[<c06598b4>] (platform_drv_probe) from [<c0658210>] (driver_probe_device+0x1f4/0x2b0)
-[<c0658210>] (driver_probe_device) from [<c06567a8>] (bus_for_each_drv+0x44/0x8c)
-[<c06567a8>] (bus_for_each_drv) from [<c0657f94>] (__device_attach+0x9c/0x100)
-[<c0657f94>] (__device_attach) from [<c0657628>] (bus_probe_device+0x84/0x8c)
-[<c0657628>] (bus_probe_device) from [<c0657a5c>] (deferred_probe_work_func+0x60/0x8c)
-[<c0657a5c>] (deferred_probe_work_func) from [<c025ebb4>] (process_one_work+0x134/0x338)
+[<c0220044>] (arm_dma_alloc) from [<c0306a38>] (dma_pool_alloc+0x1dc/0x210)
+[<c0306a38>] (dma_pool_alloc) from [<c07c7128>] (ehci_qh_alloc+0x3c/0xb4)
+[<c07c7128>] (ehci_qh_alloc) from [<c07cb720>] (ehci_setup+0x154/0x478)
+[<c07cb720>] (ehci_setup) from [<c07d17d8>] (tegra_ehci_reset+0xc/0x50)
+[<c07d17d8>] (tegra_ehci_reset) from [<c07a1ba0>] (usb_add_hcd+0x300/0x830)
+[<c07a1ba0>] (usb_add_hcd) from [<c07d1150>] (tegra_ehci_probe+0x3b8/0x504)
+[<c07d1150>] (tegra_ehci_probe) from [<c065938c>] (platform_drv_probe+0x50/0xac)
+[<c065938c>] (platform_drv_probe) from [<c0657ce8>] (driver_probe_device+0x1f4/0x2b0)
+[<c0657ce8>] (driver_probe_device) from [<c0656280>] (bus_for_each_drv+0x44/0x8c)
+[<c0656280>] (bus_for_each_drv) from [<c0657a6c>] (__device_attach+0x9c/0x100)
+[<c0657a6c>] (__device_attach) from [<c0657100>] (bus_probe_device+0x84/0x8c)
+[<c0657100>] (bus_probe_device) from [<c0657534>] (deferred_probe_work_func+0x60/0x8c)
+[<c0657534>] (deferred_probe_work_func) from [<c025ebb4>] (process_one_work+0x134/0x338)
 [<c025ebb4>] (process_one_work) from [<c025ee1c>] (worker_thread+0x2c/0x4f0)
 [<c025ee1c>] (worker_thread) from [<c0263fa8>] (kthread+0xd8/0xf4)
 [<c0263fa8>] (kthread) from [<c0210e78>] (ret_from_fork+0x14/0x3c)
----[ end trace d5e9d232ec3bdcd9 ]---
+---[ end trace 660ddcc4cd4e2d09 ]---
 tegra-ehci 7d004000.usb: irq 343, io mem 0x7d004000
 tegra-ehci 7d004000.usb: USB 2.0 started, EHCI 1.00
 hub 1-0:1.0: USB hub found
@@ -399,4 +399,4 @@
 Freeing unused kernel memory: 1052K (c0e43000 - c0f4a000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.003533] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.008518] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-01-12 00:10:07.518461828 -0800
+++ /tmp/foob	2016-01-12 00:10:07.518461828 -0800
@@ -39,18 +39,18 @@
 	 #################################
 	 3.3 MiB/s
 done
-Bytes transferred = 5254872 (502ed8 hex)
+Bytes transferred = 5254512 (502d70 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x502ed8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x502d70 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-rc8-next-20160111-66728-gbf976ab (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jan 11 04:17:55 PST 2016
+Linux version 4.4.0-next-20160112-66957-gd299f5e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Jan 12 00:03:56 PST 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -89,7 +89,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0982734   (9706 kB)
+      .text : 0xc0008000 - 0xc0982720   (9706 kB)
       .init : 0xc0983000 - 0xc09d8000   ( 340 kB)
       .data : 0xc09d8000 - 0xc0a34fa0   ( 372 kB)
        .bss : 0xc0a37000 - 0xc0a92d88   ( 368 kB)
@@ -339,42 +339,42 @@
 tegra-ehci 7d004000.usb: EHCI Host Controller
 tegra-ehci 7d004000.usb: new USB bus registered, assigned bus number 1
 ------------[ cut here ]------------
-WARNING: CPU: 0 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
+WARNING: CPU: 2 PID: 79 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-rc8-next-20160111-66728-gbf976ab #1
+CPU: 2 PID: 79 Comm: kworker/u8:1 Not tainted 4.4.0-next-20160112-66957-gd299f5e #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0017208>] (unwind_backtrace) from [<c0012874>] (show_stack+0x10/0x14)
-[<c0012874>] (show_stack) from [<c023c444>] (dump_stack+0x84/0xd0)
-[<c023c444>] (dump_stack) from [<c0026b70>] (warn_slowpath_common+0x74/0xb0)
+[<c0012874>] (show_stack) from [<c023bf24>] (dump_stack+0x84/0xd0)
+[<c023bf24>] (dump_stack) from [<c0026b70>] (warn_slowpath_common+0x74/0xb0)
 [<c0026b70>] (warn_slowpath_common) from [<c0026c40>] (warn_slowpath_fmt+0x30/0x40)
 [<c0026c40>] (warn_slowpath_fmt) from [<c00397d0>] (check_flush_dependency+0xc8/0x12c)
 [<c00397d0>] (check_flush_dependency) from [<c00398ec>] (flush_work+0x54/0x11c)
-[<c00398ec>] (flush_work) from [<c00c0b78>] (lru_add_drain_all+0x124/0x180)
-[<c00c0b78>] (lru_add_drain_all) from [<c00f1814>] (migrate_prep+0x8/0x10)
-[<c00f1814>] (migrate_prep) from [<c00bb6f0>] (alloc_contig_range+0xc4/0x2f8)
-[<c00bb6f0>] (alloc_contig_range) from [<c00f3384>] (cma_alloc+0xd8/0x1a0)
-[<c00f3384>] (cma_alloc) from [<c001c270>] (__alloc_from_contiguous+0x38/0xdc)
+[<c00398ec>] (flush_work) from [<c00c08b0>] (lru_add_drain_all+0x124/0x180)
+[<c00c08b0>] (lru_add_drain_all) from [<c00f154c>] (migrate_prep+0x8/0x10)
+[<c00f154c>] (migrate_prep) from [<c00bb428>] (alloc_contig_range+0xc4/0x2f8)
+[<c00bb428>] (alloc_contig_range) from [<c00f30bc>] (cma_alloc+0xd8/0x1a0)
+[<c00f30bc>] (cma_alloc) from [<c001c270>] (__alloc_from_contiguous+0x38/0xdc)
 [<c001c270>] (__alloc_from_contiguous) from [<c001cf38>] (__dma_alloc+0x228/0x2ac)
 [<c001cf38>] (__dma_alloc) from [<c001d040>] (arm_dma_alloc+0x48/0x54)
-[<c001d040>] (arm_dma_alloc) from [<c00edc20>] (dma_pool_alloc+0x28c/0x304)
-[<c00edc20>] (dma_pool_alloc) from [<c03fd538>] (ehci_qh_alloc+0x3c/0xb4)
-[<c03fd538>] (ehci_qh_alloc) from [<c0401724>] (ehci_setup+0x154/0x478)
-[<c0401724>] (ehci_setup) from [<c040546c>] (tegra_ehci_reset+0xc/0x50)
-[<c040546c>] (tegra_ehci_reset) from [<c03e8f1c>] (usb_add_hcd+0x2fc/0x804)
-[<c03e8f1c>] (usb_add_hcd) from [<c0404de4>] (tegra_ehci_probe+0x3b8/0x504)
-[<c0404de4>] (tegra_ehci_probe) from [<c0336f10>] (platform_drv_probe+0x50/0xac)
-[<c0336f10>] (platform_drv_probe) from [<c0335968>] (driver_probe_device+0x1f4/0x2b0)
-[<c0335968>] (driver_probe_device) from [<c0333f04>] (bus_for_each_drv+0x44/0x8c)
-[<c0333f04>] (bus_for_each_drv) from [<c03356ec>] (__device_attach+0x9c/0x100)
-[<c03356ec>] (__device_attach) from [<c0334d80>] (bus_probe_device+0x84/0x8c)
-[<c0334d80>] (bus_probe_device) from [<c03351b4>] (deferred_probe_work_func+0x60/0x8c)
-[<c03351b4>] (deferred_probe_work_func) from [<c003a6f8>] (process_one_work+0x120/0x328)
+[<c001d040>] (arm_dma_alloc) from [<c00ed958>] (dma_pool_alloc+0x28c/0x304)
+[<c00ed958>] (dma_pool_alloc) from [<c03fd1a4>] (ehci_qh_alloc+0x3c/0xb4)
+[<c03fd1a4>] (ehci_qh_alloc) from [<c0401390>] (ehci_setup+0x154/0x478)
+[<c0401390>] (ehci_setup) from [<c04050d8>] (tegra_ehci_reset+0xc/0x50)
+[<c04050d8>] (tegra_ehci_reset) from [<c03e8b88>] (usb_add_hcd+0x2fc/0x804)
+[<c03e8b88>] (usb_add_hcd) from [<c0404a50>] (tegra_ehci_probe+0x3b8/0x504)
+[<c0404a50>] (tegra_ehci_probe) from [<c0336b7c>] (platform_drv_probe+0x50/0xac)
+[<c0336b7c>] (platform_drv_probe) from [<c03355d4>] (driver_probe_device+0x1f4/0x2b0)
+[<c03355d4>] (driver_probe_device) from [<c0333b70>] (bus_for_each_drv+0x44/0x8c)
+[<c0333b70>] (bus_for_each_drv) from [<c0335358>] (__device_attach+0x9c/0x100)
+[<c0335358>] (__device_attach) from [<c03349ec>] (bus_probe_device+0x84/0x8c)
+[<c03349ec>] (bus_probe_device) from [<c0334e20>] (deferred_probe_work_func+0x60/0x8c)
+[<c0334e20>] (deferred_probe_work_func) from [<c003a6f8>] (process_one_work+0x120/0x328)
 [<c003a6f8>] (process_one_work) from [<c003a964>] (worker_thread+0x2c/0x4b4)
 [<c003a964>] (worker_thread) from [<c003fa9c>] (kthread+0xd8/0xf4)
 [<c003fa9c>] (kthread) from [<c000f938>] (ret_from_fork+0x14/0x3c)
----[ end trace a1b236e9945d411a ]---
+---[ end trace eba1bfc7e0f73b9e ]---
 tegra-ehci 7d004000.usb: irq 343, io mem 0x7d004000
 tegra-ehci 7d004000.usb: USB 2.0 started, EHCI 1.00
 hub 1-0:1.0: USB hub found
@@ -399,4 +399,4 @@
 Freeing unused kernel memory: 340K (c0983000 - c09d8000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.480550] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.410381] usb 2-1: new high-speed USB device number 2 using tegra-ehci
