Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2015-07-22 05:21:31.552622653 -0700
+++ /tmp/foob	2015-07-22 05:21:31.556622649 -0700
@@ -1,6 +1,6 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
-Linux version 4.2.0-rc3-next-20150721+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Jul 22 05:03:57 PDT 2015
+Linux version 4.2.0-rc3-next-20150722+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Jul 22 05:18:54 PDT 2015
 CPU: AArch64 Processor [411fd070] revision 0
 Detected PIPT I-cache on CPU0
 alternatives: enabling workaround for ARM erratum 832075
@@ -88,7 +88,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 2, 16384 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.420:1): initialized
+audit: type=2000 audit(0.410:1): initialized
 HugeTLB registered 2 MB page size, pre-allocated 0 pages
 VFS: Disk quotas dquot_6.6.0
 VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-07-22 05:21:31.560622647 -0700
+++ /tmp/foob	2015-07-22 05:21:31.564622645 -0700
@@ -38,20 +38,20 @@
 	 #################################################################
 	 #################################################################
 	 #########################################
-	 3.2 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 6322624 (6079c0 hex)
+Bytes transferred = 6320144 (607010 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.8 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6079c0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x607010 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc3-next-20150721-30813-ge844ab8 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Jul 22 05:02:39 PDT 2015
+Linux version 4.2.0-rc3-next-20150722-31033-g72a68fd (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Jul 22 05:17:38 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -67,7 +67,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f39a80, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0f33cc0, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -80,7 +80,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999160K/2096128K available (7964K kernel code, 1022K rwdata, 3556K rodata, 968K init, 337K bss, 31432K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1999184K/2096128K available (7964K kernel code, 1007K rwdata, 3556K rodata, 960K init, 337K bss, 31408K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,10 +88,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d4926c   (11525 kB)
-      .init : 0xc0d4a000 - 0xc0e3c000   ( 968 kB)
-      .data : 0xc0e3c000 - 0xc0f3ba80   (1023 kB)
-       .bss : 0xc0f3e000 - 0xc0f92660   ( 338 kB)
+      .text : 0xc0208000 - 0xc0d49214   (11525 kB)
+      .init : 0xc0d4a000 - 0xc0e3a000   ( 960 kB)
+      .data : 0xc0e3a000 - 0xc0f35cc0   (1008 kB)
+       .bss : 0xc0f38000 - 0xc0f8c660   ( 338 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -269,6 +269,13 @@
 NET: Registered protocol family 29
 can: raw protocol (rev 20120528)
 can: broadcast manager protocol (rev 20120528 t)
+mmc1: MAN_BKOPS_EN bit is not set
+mmc1: new high speed MMC card at address 0001
+mmcblk0: mmc1:0001 SEM32G 29.1 GiB 
+mmcblk0boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
+mmcblk0boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
+mmcblk0rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 can: netlink gateway (rev 20130117) max_hops=1
 Key type dns_resolver registered
 ThumbEE CPU extension supported.
@@ -278,13 +285,6 @@
 vdd_hdmi_5v0: supplied by vdd-sys-5v0
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
-mmc1: MAN_BKOPS_EN bit is not set
-mmc1: new high speed MMC card at address 0001
-mmcblk0: mmc1:0001 SEM32G 29.1 GiB 
-mmcblk0boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
-mmcblk0boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
-mmcblk0rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 Console: switching to colour frame buffer device 240x75
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
@@ -296,7 +296,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-01 20:55:09 UTC (949438509)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-01 21:10:07 UTC (949439407)
 vdd_cam_1v8_reg: disabling
 usb1_vbus: disabling
 dvdd_ts: disabling
@@ -305,7 +305,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 968K (c0d4a000 - c0e3c000)
+Freeing unused kernel memory: 960K (c0d4a000 - c0e3a000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.214214] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.214217] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2015-07-22 05:21:31.568622644 -0700
+++ /tmp/foob	2015-07-22 05:21:31.572622643 -0700
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 #######################
-	 3.4 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 5103008 (4ddda0 hex)
+Bytes transferred = 5102512 (4ddbb0 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 2.6 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4ddda0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4ddbb0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc3-next-20150721-30813-ge844ab8 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Jul 22 05:03:16 PDT 2015
+Linux version 4.2.0-rc3-next-20150722-31033-g72a68fd (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Jul 22 05:18:16 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c09d0f80, node_mem_map ee7f7000
+free_area_init_node: node 0, pgdat c09d2f80, node_mem_map ee7f7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002620K/2096128K available (6978K kernel code, 346K rwdata, 2392K rodata, 308K init, 363K bss, 27972K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 2002612K/2096128K available (6978K kernel code, 346K rwdata, 2396K rodata, 312K init, 363K bss, 27980K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc092ec58   (9372 kB)
-      .init : 0xc092f000 - 0xc097c000   ( 308 kB)
-      .data : 0xc097c000 - 0xc09d2b40   ( 347 kB)
-       .bss : 0xc09d5000 - 0xc0a2fc58   ( 364 kB)
+      .text : 0xc0008000 - 0xc092fc7c   (9376 kB)
+      .init : 0xc0930000 - 0xc097e000   ( 312 kB)
+      .data : 0xc097e000 - 0xc09d4b40   ( 347 kB)
+       .bss : 0xc09d7000 - 0xc0a31c58   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -168,7 +168,6 @@
 Bluetooth: HCI socket layer initialized
 Bluetooth: L2CAP socket layer initialized
 Bluetooth: SCO socket layer initialized
-cfg80211: Calling CRDA to update world regulatory domain
 clocksource: Switched to clocksource arch_sys_counter
 NET: Registered protocol family 2
 TCP established hash table entries: 8192 (order: 3, 32768 bytes)
@@ -293,7 +292,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-01 20:55:51 UTC (949438551)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-02-01 21:10:48 UTC (949439448)
 vdd_cam_1v8_reg: disabling
 usb1_vbus: disabling
 dvdd_ts: disabling
@@ -305,7 +304,7 @@
 EXT3-fs (mmcblk0p1): mounted filesystem with writeback data mode
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 308K (c092f000 - c097c000)
+Freeing unused kernel memory: 312K (c0930000 - c097e000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.759943] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.729921] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-07-22 05:21:31.576622640 -0700
+++ /tmp/foob	2015-07-22 05:21:31.580622636 -0700
@@ -42,18 +42,18 @@
 	 #########################################
 	 3.2 MiB/s
 done
-Bytes transferred = 6322624 (6079c0 hex)
+Bytes transferred = 6320144 (607010 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *#####
-	 3 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 63859 (f973 hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6079c0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x607010 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012972
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc3-next-20150721-30813-ge844ab8 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Jul 22 05:02:39 PDT 2015
+Linux version 4.2.0-rc3-next-20150722-31033-g72a68fd (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Jul 22 05:17:38 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f39a80, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0f33cc0, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999056K/2096128K available (7964K kernel code, 1022K rwdata, 3556K rodata, 968K init, 337K bss, 31536K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1999080K/2096128K available (7964K kernel code, 1007K rwdata, 3556K rodata, 960K init, 337K bss, 31512K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d4926c   (11525 kB)
-      .init : 0xc0d4a000 - 0xc0e3c000   ( 968 kB)
-      .data : 0xc0e3c000 - 0xc0f3ba80   (1023 kB)
-       .bss : 0xc0f3e000 - 0xc0f92660   ( 338 kB)
+      .text : 0xc0208000 - 0xc0d49214   (11525 kB)
+      .init : 0xc0d4a000 - 0xc0e3a000   ( 960 kB)
+      .data : 0xc0e3a000 - 0xc0f35cc0   (1008 kB)
+       .bss : 0xc0f38000 - 0xc0f8c660   ( 338 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -359,7 +359,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 968K (c0d4a000 - c0e3c000)
+Freeing unused kernel memory: 960K (c0d4a000 - c0e3a000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2015-07-22 05:21:31.584622634 -0700
+++ /tmp/foob	2015-07-22 05:21:31.588622632 -0700
@@ -29,31 +29,31 @@
 Tegra124 (Jetson TK1) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug init=/bin/bash noinitrd'
 Tegra124 (Jetson TK1) # setenv ipaddr 192.168.100.102; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra124 (Jetson TK1) # tftpboot 0x81000000 zImage.tegra_defconfig
-Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'zImage.tegra_defconfig'.
 Load address: 0x81000000
-Loading: *#################################################################
+Loading: *T #################################################################
 	 #################################################################
 	 #################################################################
 	 #################################################################
 	 #################################################################
 	 #######################
-	 3.1 MiB/s
+	 740.2 KiB/s
 done
-Bytes transferred = 5103008 (4ddda0 hex)
+Bytes transferred = 5102512 (4ddbb0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
+Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *#####
-	 3.2 MiB/s
+	 3.6 MiB/s
 done
 Bytes transferred = 63859 (f973 hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4ddda0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4ddbb0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012972
@@ -64,7 +64,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc3-next-20150721-30813-ge844ab8 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Jul 22 05:03:16 PDT 2015
+Linux version 4.2.0-rc3-next-20150722-31033-g72a68fd (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Jul 22 05:18:16 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -72,7 +72,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c09d0f80, node_mem_map ee7f7000
+free_area_init_node: node 0, pgdat c09d2f80, node_mem_map ee7f7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -85,7 +85,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002516K/2096128K available (6978K kernel code, 346K rwdata, 2392K rodata, 308K init, 363K bss, 28076K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 2002508K/2096128K available (6978K kernel code, 346K rwdata, 2396K rodata, 312K init, 363K bss, 28084K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -93,10 +93,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc092ec58   (9372 kB)
-      .init : 0xc092f000 - 0xc097c000   ( 308 kB)
-      .data : 0xc097c000 - 0xc09d2b40   ( 347 kB)
-       .bss : 0xc09d5000 - 0xc0a2fc58   ( 364 kB)
+      .text : 0xc0008000 - 0xc092fc7c   (9376 kB)
+      .init : 0xc0930000 - 0xc097e000   ( 312 kB)
+      .data : 0xc097e000 - 0xc09d4b40   ( 347 kB)
+       .bss : 0xc09d7000 - 0xc0a31c58   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -170,7 +170,6 @@
 Bluetooth: HCI socket layer initialized
 Bluetooth: L2CAP socket layer initialized
 Bluetooth: SCO socket layer initialized
-cfg80211: Calling CRDA to update world regulatory domain
 clocksource: Switched to clocksource arch_sys_counter
 NET: Registered protocol family 2
 TCP established hash table entries: 8192 (order: 3, 32768 bytes)
@@ -298,7 +297,6 @@
 tegra-pcie 1003000.pcie-controller: probing port 0, using 2 lanes
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
 tegra-pcie 1003000.pcie-controller: link 0 down, retrying
-cfg80211: Calling CRDA to update world regulatory domain
 tegra-pcie 1003000.pcie-controller: link 0 down, retrying
 tegra-pcie 1003000.pcie-controller: link 0 down, retrying
 tegra-pcie 1003000.pcie-controller: link 0 down, ignoring
@@ -342,7 +340,7 @@
 cpufreq: cpufreq_add_dev: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_add_dev: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-02-01 00:00:21 UTC (949363221)
+as3722-rtc as3722-rtc: setting system clock to 2000-02-01 00:00:24 UTC (949363224)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_VPP_FUSE: disabling
 +2.8V_RUN_CAM_AF: disabling
@@ -362,8 +360,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 308K (c092f000 - c097c000)
-cfg80211: Calling CRDA to update world regulatory domain
+Freeing unused kernel memory: 312K (c0930000 - c097e000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-07-22 05:21:31.592622630 -0700
+++ /tmp/foob	2015-07-22 05:21:31.592622630 -0700
@@ -48,18 +48,18 @@
 	 #########################################
 	 4.1 MiB/s
 done
-Bytes transferred = 6322624 (6079c0 hex)
+Bytes transferred = 6320144 (607010 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 4 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x6079c0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x607010 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -67,7 +67,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc3-next-20150721-30813-ge844ab8 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Jul 22 05:02:39 PDT 2015
+Linux version 4.2.0-rc3-next-20150722-31033-g72a68fd (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Jul 22 05:17:38 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -75,7 +75,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0f39a80, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c0f33cc0, node_mem_map eeff8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -88,7 +88,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 959960K/1048576K available (7964K kernel code, 1022K rwdata, 3556K rodata, 968K init, 337K bss, 23080K reserved, 65536K cma-reserved, 204800K highmem)
+Memory: 959984K/1048576K available (7964K kernel code, 1007K rwdata, 3556K rodata, 960K init, 337K bss, 23056K reserved, 65536K cma-reserved, 204800K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -96,10 +96,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d4926c   (11525 kB)
-      .init : 0xc0d4a000 - 0xc0e3c000   ( 968 kB)
-      .data : 0xc0e3c000 - 0xc0f3ba80   (1023 kB)
-       .bss : 0xc0f3e000 - 0xc0f92660   ( 338 kB)
+      .text : 0xc0208000 - 0xc0d49214   (11525 kB)
+      .init : 0xc0d4a000 - 0xc0e3a000   ( 960 kB)
+      .data : 0xc0e3a000 - 0xc0f35cc0   (1008 kB)
+       .bss : 0xc0f38000 - 0xc0f8c660   ( 338 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -334,15 +334,15 @@
  mmcblk0: p1
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-26 19:21:44 UTC (1419621704)
+rtc-em3027 2-0056: setting system clock to 2014-12-26 19:22:22 UTC (1419621742)
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
-hub 1-1:1.0: USB hub found
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
-devtmpfs: mounted
-Freeing unused kernel memory: 968K (c0d4a000 - c0e3c000)
+hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
+devtmpfs: mounted
+Freeing unused kernel memory: 960K (c0d4a000 - c0e3a000)
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2015-07-22 05:21:31.600622626 -0700
+++ /tmp/foob	2015-07-22 05:21:31.600622626 -0700
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -45,9 +45,9 @@
 	 #################################################################
 	 #################################################################
 	 #######################
-	 4.1 MiB/s
+	 3.8 MiB/s
 done
-Bytes transferred = 5103008 (4ddda0 hex)
+Bytes transferred = 5102512 (4ddbb0 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x4ddda0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x4ddbb0 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -69,7 +69,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc3-next-20150721-30813-ge844ab8 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Jul 22 05:03:16 PDT 2015
+Linux version 4.2.0-rc3-next-20150722-31033-g72a68fd (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Jul 22 05:18:16 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -78,7 +78,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c09d0f80, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c09d2f80, node_mem_map eeff7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -91,7 +91,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 963420K/1048576K available (6978K kernel code, 346K rwdata, 2392K rodata, 308K init, 363K bss, 19620K reserved, 65536K cma-reserved, 204800K highmem)
+Memory: 963412K/1048576K available (6978K kernel code, 346K rwdata, 2396K rodata, 312K init, 363K bss, 19628K reserved, 65536K cma-reserved, 204800K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -99,10 +99,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc092ec58   (9372 kB)
-      .init : 0xc092f000 - 0xc097c000   ( 308 kB)
-      .data : 0xc097c000 - 0xc09d2b40   ( 347 kB)
-       .bss : 0xc09d5000 - 0xc0a2fc58   ( 364 kB)
+      .text : 0xc0008000 - 0xc092fc7c   (9376 kB)
+      .init : 0xc0930000 - 0xc097e000   ( 312 kB)
+      .data : 0xc097e000 - 0xc09d4b40   ( 347 kB)
+       .bss : 0xc09d7000 - 0xc0a31c58   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
@@ -165,7 +165,6 @@
 Bluetooth: HCI socket layer initialized
 Bluetooth: L2CAP socket layer initialized
 Bluetooth: SCO socket layer initialized
-cfg80211: Calling CRDA to update world regulatory domain
 clocksource: Switched to clocksource timer_us
 NET: Registered protocol family 2
 TCP established hash table entries: 8192 (order: 3, 32768 bytes)
@@ -316,12 +315,12 @@
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
 tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
 NET: Registered protocol family 10
+mip6: Mobile IPv6
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
- mmcblk0: p1
-mip6: Mobile IPv6
 sit: IPv6 over IPv4 tunneling driver
+ mmcblk0: p1
 NET: Registered protocol family 17
 NET: Registered protocol family 15
 can: controller area network core (rev 20120528 abi 9)
@@ -343,13 +342,13 @@
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
 mmc1: new high speed SD card at address b368
-tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
- mmcblk1: p1 p2
+tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
+ mmcblk1: p1 p2
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-26 19:22:03 UTC (1419621723)
+rtc-em3027 2-0056: setting system clock to 2014-12-26 19:22:41 UTC (1419621761)
 ALSA device list:
   #0: tegra-trimslice
 kjournald starting.  Commit interval 5 seconds
@@ -358,7 +357,7 @@
 EXT3-fs (mmcblk0p1): mounted filesystem with writeback data mode
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 308K (c092f000 - c097c000)
+Freeing unused kernel memory: 312K (c0930000 - c097e000)
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-07-22 05:21:31.608622622 -0700
+++ /tmp/foob	2015-07-22 05:21:31.608622622 -0700
@@ -38,20 +38,20 @@
 	 #################################################################
 	 #################################################################
 	 #########################################
-	 3.4 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 6322624 (6079c0 hex)
+Bytes transferred = 6320144 (607010 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6079c0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x607010 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.2.0-rc3-next-20150721-30813-ge844ab8 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Jul 22 05:02:39 PDT 2015
+Linux version 4.2.0-rc3-next-20150722-31033-g72a68fd (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Jul 22 05:17:38 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -67,7 +67,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f39a80, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0f33cc0, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -80,7 +80,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999080K/2096128K available (7964K kernel code, 1022K rwdata, 3556K rodata, 968K init, 337K bss, 31512K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 1999104K/2096128K available (7964K kernel code, 1007K rwdata, 3556K rodata, 960K init, 337K bss, 31488K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,10 +88,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0d4926c   (11525 kB)
-      .init : 0xc0d4a000 - 0xc0e3c000   ( 968 kB)
-      .data : 0xc0e3c000 - 0xc0f3ba80   (1023 kB)
-       .bss : 0xc0f3e000 - 0xc0f92660   ( 338 kB)
+      .text : 0xc0208000 - 0xc0d49214   (11525 kB)
+      .init : 0xc0d4a000 - 0xc0e3a000   ( 960 kB)
+      .data : 0xc0e3a000 - 0xc0f35cc0   (1008 kB)
+       .bss : 0xc0f38000 - 0xc0f8c660   ( 338 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -364,7 +364,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 968K (c0d4a000 - c0e3c000)
+Freeing unused kernel memory: 960K (c0d4a000 - c0e3a000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.773758] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.773778] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2015-07-22 05:21:31.616622616 -0700
+++ /tmp/foob	2015-07-22 05:21:31.616622616 -0700
@@ -39,18 +39,18 @@
 	 #######################
 	 3.4 MiB/s
 done
-Bytes transferred = 5103008 (4ddda0 hex)
+Bytes transferred = 5102512 (4ddbb0 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4ddda0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4ddbb0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.2.0-rc3-next-20150721-30813-ge844ab8 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Jul 22 05:03:16 PDT 2015
+Linux version 4.2.0-rc3-next-20150722-31033-g72a68fd (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Jul 22 05:18:16 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c09d0f80, node_mem_map ee7f7000
+free_area_init_node: node 0, pgdat c09d2f80, node_mem_map ee7f7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002540K/2096128K available (6978K kernel code, 346K rwdata, 2392K rodata, 308K init, 363K bss, 28052K reserved, 65536K cma-reserved, 1252352K highmem)
+Memory: 2002532K/2096128K available (6978K kernel code, 346K rwdata, 2396K rodata, 312K init, 363K bss, 28060K reserved, 65536K cma-reserved, 1252352K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc092ec58   (9372 kB)
-      .init : 0xc092f000 - 0xc097c000   ( 308 kB)
-      .data : 0xc097c000 - 0xc09d2b40   ( 347 kB)
-       .bss : 0xc09d5000 - 0xc0a2fc58   ( 364 kB)
+      .text : 0xc0008000 - 0xc092fc7c   (9376 kB)
+      .init : 0xc0930000 - 0xc097e000   ( 312 kB)
+      .data : 0xc097e000 - 0xc09d4b40   ( 347 kB)
+       .bss : 0xc09d7000 - 0xc0a31c58   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -163,7 +163,6 @@
 Bluetooth: HCI socket layer initialized
 Bluetooth: L2CAP socket layer initialized
 Bluetooth: SCO socket layer initialized
-cfg80211: Calling CRDA to update world regulatory domain
 clocksource: Switched to clocksource timer_us
 NET: Registered protocol family 2
 TCP established hash table entries: 8192 (order: 3, 32768 bytes)
@@ -294,7 +293,6 @@
 tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
-cfg80211: Calling CRDA to update world regulatory domain
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, ignoring
 tegra-pcie 3000.pcie-controller: PCI host bridge to bus 0000:00
@@ -366,7 +364,7 @@
 EXT3-fs (mmcblk0p1): mounted filesystem with writeback data mode
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 308K (c092f000 - c097c000)
+Freeing unused kernel memory: 312K (c0930000 - c097e000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.436671] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.426680] usb 2-1: new high-speed USB device number 2 using tegra-ehci
