Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2015-10-19 00:23:53.222989680 -0700
+++ /tmp/foob	2015-10-19 00:23:53.226989676 -0700
@@ -1,6 +1,6 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
-Linux version 4.3.0-rc5-next-20151016+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Fri Oct 16 01:19:05 PDT 2015
+Linux version 4.3.0-rc5-next-20151019+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Oct 19 00:19:19 PDT 2015
 CPU: AArch64 Processor [411fd070] revision 0
 Detected PIPT I-cache on CPU0
 alternatives: enabling workaround for ARM erratum 832075
@@ -19,7 +19,7 @@
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
 software IO TLB [mem 0x79c00000-0x7dc00000] (64MB) mapped at [ffffffc039c00000-ffffffc03dbfffff]
-Memory: 938904K/1048576K available (5538K kernel code, 485K rwdata, 2324K rodata, 536K init, 206K bss, 93288K reserved, 16384K cma-reserved)
+Memory: 938904K/1048576K available (5538K kernel code, 481K rwdata, 2328K rodata, 536K init, 206K bss, 93288K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     vmalloc : 0xffffff8000000000 - 0xffffffbdbfff0000   (   246 GB)
     vmemmap : 0xffffffbdc0000000 - 0xffffffbfc0000000   (     8 GB maximum)
@@ -28,9 +28,9 @@
     PCI I/O : 0xffffffbffae00000 - 0xffffffbffbe00000   (    16 MB)
     modules : 0xffffffbffc000000 - 0xffffffc000000000   (    64 MB)
     memory  : 0xffffffc000000000 - 0xffffffc040000000   (  1024 MB)
-      .init : 0xffffffc000830000 - 0xffffffc0008b6000   (   536 KB)
-      .text : 0xffffffc000080000 - 0xffffffc00082f9e4   (  7871 KB)
-      .data : 0xffffffc0008c7000 - 0xffffffc000940400   (   485 KB)
+      .init : 0xffffffc000831000 - 0xffffffc0008b7000   (   536 KB)
+      .text : 0xffffffc000080000 - 0xffffffc0008309e4   (  7875 KB)
+      .data : 0xffffffc0008c8000 - 0xffffffc000940400   (   481 KB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 64.
@@ -86,7 +86,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 2, 16384 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.630:1): initialized
+audit: type=2000 audit(0.550:1): initialized
 HugeTLB registered 2 MB page size, pre-allocated 0 pages
 VFS: Disk quotas dquot_6.6.0
 VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
@@ -134,10 +134,8 @@
 EXT4-fs (vda2): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 254:2.
 devtmpfs: mounted
- 
-Freeing unused kernel memory: 536K (ffffffc000830000 - ffffffc0008b6000)
-Freeing alternatives memory: 48K (ffffffc0008b6000 - ffffffc0008c2000)
+Freeing unused kernel memory: 536K (ffffffc000831000 - ffffffc0008b7000)
+Freeing alternatives memory: 48K (ffffffc0008b7000 - ffffffc0008c3000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-bash-4.3#  
 bash-4.3# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-10-19 00:23:53.250989654 -0700
+++ /tmp/foob	2015-10-19 00:23:53.254989650 -0700
@@ -27,6 +27,7 @@
 Tegra114 (Dalmore) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug init=/bin/bash'
 Tegra114 (Dalmore) # setenv ipaddr 192.168.100.100; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra114 (Dalmore) # tftpboot 0x81000000 zImage.multi_v7_defconfig
+Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'zImage.multi_v7_defconfig'.
@@ -38,9 +39,9 @@
 	 #################################################################
 	 #################################################################
 	 ######################################################
-	 3.2 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 6510488 (635798 hex)
+Bytes transferred = 6516864 (637080 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -51,7 +52,7 @@
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x635798 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x637080 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -59,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.3.0-rc5-next-20151016-49772-gcd685d8 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Fri Oct 16 01:17:47 PDT 2015
+Linux version 4.3.0-rc5-next-20151019-50529-gfb68b25 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Oct 19 00:17:46 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -67,7 +68,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0fb1d80, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c0fb59c0, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -80,7 +81,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998660K/2096128K available (8225K kernel code, 1031K rwdata, 3696K rodata, 1040K init, 340K bss, 31932K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998644K/2096128K available (8226K kernel code, 1038K rwdata, 3708K rodata, 1036K init, 340K bss, 31948K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,10 +89,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0dad7f4   (11926 kB)
-      .init : 0xc0dae000 - 0xc0eb2000   (1040 kB)
-      .data : 0xc0eb2000 - 0xc0fb3d60   (1032 kB)
-       .bss : 0xc0fb6000 - 0xc100b3e8   ( 341 kB)
+      .text : 0xc0208000 - 0xc0db0814   (11939 kB)
+      .init : 0xc0db1000 - 0xc0eb4000   (1036 kB)
+      .data : 0xc0eb4000 - 0xc0fb7920   (1039 kB)
+       .bss : 0xc0fba000 - 0xc100f3e8   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -286,7 +287,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 78000600.sdhci [78000600.sdhci] using ADMA
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-04-27 17:14:16 UTC (956855656)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-04-30 16:14:28 UTC (957111268)
 vdd_cam_1v8_reg: disabling
 dvdd_ts: disabling
 usb1_vbus: disabling
@@ -298,13 +299,13 @@
 mmcblk0boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
 mmcblk0rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
-EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
+EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1040K (c0dae000 - c0eb2000)
+Freeing unused kernel memory: 1036K (c0db1000 - c0eb4000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2015-10-19 00:23:53.258989645 -0700
+++ /tmp/foob	2015-10-19 00:23:53.262989642 -0700
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 ##############################
-	 3.3 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 5204560 (4f6a50 hex)
+Bytes transferred = 5204504 (4f6a18 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.8 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4f6a50 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4f6a18 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.3.0-rc5-next-20151016-49772-gcd685d8 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Oct 16 01:18:04 PDT 2015
+Linux version 4.3.0-rc5-next-20151019-50529-gfb68b25 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Oct 19 00:18:06 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0a12400, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0a12480, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002340K/2096128K available (7151K kernel code, 351K rwdata, 2444K rodata, 336K init, 364K bss, 28252K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2002340K/2096128K available (7147K kernel code, 351K rwdata, 2444K rodata, 340K init, 364K bss, 28252K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,8 +90,8 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09670ec   (9597 kB)
-      .init : 0xc0968000 - 0xc09bc000   ( 336 kB)
+      .text : 0xc0008000 - 0xc0966114   (9593 kB)
+      .init : 0xc0967000 - 0xc09bc000   ( 340 kB)
       .data : 0xc09bc000 - 0xc0a13fa0   ( 352 kB)
        .bss : 0xc0a16000 - 0xc0a710c0   ( 365 kB)
 Preemptible hierarchical RCU implementation.
@@ -296,7 +296,7 @@
 tegra-snd-rt5640 sound: rt5640-aif1 <-> 70080400.i2s mapping ok
 input: NVIDIA Tegra Dalmore Headphones as /devices/soc0/sound/sound/card0/input0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-04-27 17:14:47 UTC (956855687)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-04-30 16:14:57 UTC (957111297)
 vdd_cam_1v8_reg: disabling
 dvdd_ts: disabling
 usb1_vbus: disabling
@@ -309,7 +309,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 336K (c0968000 - c09bc000)
+Freeing unused kernel memory: 340K (c0967000 - c09bc000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-10-19 00:23:53.266989639 -0700
+++ /tmp/foob	2015-10-19 00:23:53.270989634 -0700
@@ -42,18 +42,18 @@
 	 ######################################################
 	 3.3 MiB/s
 done
-Bytes transferred = 6510488 (635798 hex)
+Bytes transferred = 6516864 (637080 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *#####
-	 2.9 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x635798 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x637080 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012a89
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.3.0-rc5-next-20151016-49772-gcd685d8 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Fri Oct 16 01:17:47 PDT 2015
+Linux version 4.3.0-rc5-next-20151019-50529-gfb68b25 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Oct 19 00:17:46 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0fb1d80, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c0fb59c0, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998552K/2096128K available (8225K kernel code, 1031K rwdata, 3696K rodata, 1040K init, 340K bss, 32040K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998536K/2096128K available (8226K kernel code, 1038K rwdata, 3708K rodata, 1036K init, 340K bss, 32056K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0dad7f4   (11926 kB)
-      .init : 0xc0dae000 - 0xc0eb2000   (1040 kB)
-      .data : 0xc0eb2000 - 0xc0fb3d60   (1032 kB)
-       .bss : 0xc0fb6000 - 0xc100b3e8   ( 341 kB)
+      .text : 0xc0208000 - 0xc0db0814   (11939 kB)
+      .init : 0xc0db1000 - 0xc0eb4000   (1036 kB)
+      .data : 0xc0eb4000 - 0xc0fb7920   (1039 kB)
+       .bss : 0xc0fba000 - 0xc100f3e8   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -356,7 +356,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1040K (c0dae000 - c0eb2000)
+Freeing unused kernel memory: 1036K (c0db1000 - c0eb4000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2015-10-19 00:23:53.274989630 -0700
+++ /tmp/foob	2015-10-19 00:23:53.278989627 -0700
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 ##############################
-	 3.1 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 5204560 (4f6a50 hex)
+Bytes transferred = 5204504 (4f6a18 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *#####
-	 3.1 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4f6a50 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4f6a18 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012a89
@@ -63,7 +63,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.3.0-rc5-next-20151016-49772-gcd685d8 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Oct 16 01:18:04 PDT 2015
+Linux version 4.3.0-rc5-next-20151019-50529-gfb68b25 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Oct 19 00:18:06 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -71,7 +71,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0a12400, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0a12480, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -84,7 +84,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002232K/2096128K available (7151K kernel code, 351K rwdata, 2444K rodata, 336K init, 364K bss, 28360K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2002232K/2096128K available (7147K kernel code, 351K rwdata, 2444K rodata, 340K init, 364K bss, 28360K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,8 +92,8 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09670ec   (9597 kB)
-      .init : 0xc0968000 - 0xc09bc000   ( 336 kB)
+      .text : 0xc0008000 - 0xc0966114   (9593 kB)
+      .init : 0xc0967000 - 0xc09bc000   ( 340 kB)
       .data : 0xc09bc000 - 0xc0a13fa0   ( 352 kB)
        .bss : 0xc0a16000 - 0xc0a710c0   ( 365 kB)
 Preemptible hierarchical RCU implementation.
@@ -362,7 +362,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 336K (c0968000 - c09bc000)
+Freeing unused kernel memory: 340K (c0967000 - c09bc000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-10-19 00:23:53.282989623 -0700
+++ /tmp/foob	2015-10-19 00:23:53.286989619 -0700
@@ -48,18 +48,18 @@
 	 ######################################################
 	 4 MiB/s
 done
-Bytes transferred = 6510488 (635798 hex)
+Bytes transferred = 6516864 (637080 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 4 MiB/s
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x635798 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x637080 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -67,7 +67,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.3.0-rc5-next-20151016-49772-gcd685d8 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Fri Oct 16 01:17:47 PDT 2015
+Linux version 4.3.0-rc5-next-20151019-50529-gfb68b25 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Oct 19 00:17:46 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -75,7 +75,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0fb1d80, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c0fb59c0, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -88,7 +88,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 959468K/1048576K available (8225K kernel code, 1031K rwdata, 3696K rodata, 1040K init, 340K bss, 23572K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 959452K/1048576K available (8226K kernel code, 1038K rwdata, 3708K rodata, 1036K init, 340K bss, 23588K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -96,10 +96,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0dad7f4   (11926 kB)
-      .init : 0xc0dae000 - 0xc0eb2000   (1040 kB)
-      .data : 0xc0eb2000 - 0xc0fb3d60   (1032 kB)
-       .bss : 0xc0fb6000 - 0xc100b3e8   ( 341 kB)
+      .text : 0xc0208000 - 0xc0db0814   (11939 kB)
+      .init : 0xc0db1000 - 0xc0eb4000   (1036 kB)
+      .data : 0xc0eb4000 - 0xc0fb7920   (1039 kB)
+       .bss : 0xc0fba000 - 0xc100f3e8   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -241,21 +241,21 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 0 at drivers/soc/tegra/pmc.c:607 tegra_pmc_enter_suspend_mode+0x194/0x1ac()
 Modules linked in:
-CPU: 0 PID: 0 Comm: swapper/0 Not tainted 4.3.0-rc5-next-20151016-49772-gcd685d8 #1
+CPU: 0 PID: 0 Comm: swapper/0 Not tainted 4.3.0-rc5-next-20151019-50529-gfb68b25 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c0218728>] (unwind_backtrace) from [<c0213e40>] (show_stack+0x10/0x14)
-[<c0213e40>] (show_stack) from [<c0482eac>] (dump_stack+0x78/0x94)
-[<c0482eac>] (dump_stack) from [<c024a98c>] (warn_slowpath_common+0x74/0xb0)
+[<c0213e40>] (show_stack) from [<c0482f2c>] (dump_stack+0x78/0x94)
+[<c0482f2c>] (dump_stack) from [<c024a98c>] (warn_slowpath_common+0x74/0xb0)
 [<c024a98c>] (warn_slowpath_common) from [<c024a9e4>] (warn_slowpath_null+0x1c/0x24)
-[<c024a9e4>] (warn_slowpath_null) from [<c0562c80>] (tegra_pmc_enter_suspend_mode+0x194/0x1ac)
-[<c0562c80>] (tegra_pmc_enter_suspend_mode) from [<c0241e3c>] (tegra_idle_lp2_last+0xc/0x40)
+[<c024a9e4>] (warn_slowpath_null) from [<c0562f18>] (tegra_pmc_enter_suspend_mode+0x194/0x1ac)
+[<c0562f18>] (tegra_pmc_enter_suspend_mode) from [<c0241e3c>] (tegra_idle_lp2_last+0xc/0x40)
 [<c0241e3c>] (tegra_idle_lp2_last) from [<c02429a8>] (tegra20_idle_lp2_coupled+0xf0/0x1e8)
-[<c02429a8>] (tegra20_idle_lp2_coupled) from [<c0823b64>] (cpuidle_enter_state+0xbc/0x260)
-[<c0823b64>] (cpuidle_enter_state) from [<c0825b30>] (cpuidle_enter_state_coupled+0x33c/0x370)
-[<c0825b30>] (cpuidle_enter_state_coupled) from [<c027ee2c>] (cpu_startup_entry+0x174/0x24c)
-[<c027ee2c>] (cpu_startup_entry) from [<c0daec1c>] (start_kernel+0x35c/0x3c4)
-[<c0daec1c>] (start_kernel) from [<0020807c>] (0x20807c)
----[ end trace e4ef53ee06970cc1 ]---
+[<c02429a8>] (tegra20_idle_lp2_coupled) from [<c0823da4>] (cpuidle_enter_state+0xbc/0x260)
+[<c0823da4>] (cpuidle_enter_state) from [<c0825d70>] (cpuidle_enter_state_coupled+0x33c/0x370)
+[<c0825d70>] (cpuidle_enter_state_coupled) from [<c027ee2c>] (cpu_startup_entry+0x174/0x24c)
+[<c027ee2c>] (cpu_startup_entry) from [<c0db1c1c>] (start_kernel+0x35c/0x3c4)
+[<c0db1c1c>] (start_kernel) from [<0020807c>] (0x20807c)
+---[ end trace db3c547d67cf1cce ]---
 tegra-i2c 7000c500.i2c: i2c transfer timed out
 rtc-em3027 2-0056: em3027_get_time: read error
 rtc-em3027 2-0056: rtc core: registered rtc-em3027 as rtc0
@@ -344,24 +344,25 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on c8000600.sdhci [c8000600.sdhci] using ADMA
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-26 23:01:47 UTC (1419634907)
+rtc-em3027 2-0056: setting system clock to 2014-12-26 23:05:37 UTC (1419635137)
 Waiting for root device /dev/mmcblk0p1...
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
  mmcblk0: p1
-EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 usb 2-1: new high-speed USB device number 2 using tegra-ehci
-EXT4-fs (mmcblk0p1): recovery complete
-EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:1.
-devtmpfs: mounted
-Freeing unused kernel memory: 1040K (c0dae000 - c0eb2000)
+EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
 hub 2-1:1.0: USB hub found
+EXT4-fs (mmcblk0p1): recovery complete
+EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
+VFS: Mounted root (ext3 filesystem) on device 179:1.
+devtmpfs: mounted
+Freeing unused kernel memory: 1036K (c0db1000 - c0eb4000)
 hub 2-1:1.0: 4 ports detected
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.749427] usb 2-1.2: new high-speed USB device number 3 using tegra-ehci
+usb 2-1.2: new high-speed USB device number 3 using tegra-ehci
+root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2015-10-19 00:23:53.290989615 -0700
+++ /tmp/foob	2015-10-19 00:23:53.294989611 -0700
@@ -45,20 +45,20 @@
 	 #################################################################
 	 #################################################################
 	 ##############################
-	 4 MiB/s
+	 4.1 MiB/s
 done
-Bytes transferred = 5204560 (4f6a50 hex)
+Bytes transferred = 5204504 (4f6a18 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 4 MiB/s
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x4f6a50 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x4f6a18 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -69,7 +69,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.3.0-rc5-next-20151016-49772-gcd685d8 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Oct 16 01:18:04 PDT 2015
+Linux version 4.3.0-rc5-next-20151019-50529-gfb68b25 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Oct 19 00:18:06 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -78,7 +78,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0a12400, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0a12480, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -91,7 +91,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 963148K/1048576K available (7151K kernel code, 351K rwdata, 2444K rodata, 336K init, 364K bss, 19892K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 963148K/1048576K available (7147K kernel code, 351K rwdata, 2444K rodata, 340K init, 364K bss, 19892K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -99,8 +99,8 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09670ec   (9597 kB)
-      .init : 0xc0968000 - 0xc09bc000   ( 336 kB)
+      .text : 0xc0008000 - 0xc0966114   (9593 kB)
+      .init : 0xc0967000 - 0xc09bc000   ( 340 kB)
       .data : 0xc09bc000 - 0xc0a13fa0   ( 352 kB)
        .bss : 0xc0a16000 - 0xc0a710c0   ( 365 kB)
 Preemptible hierarchical RCU implementation.
@@ -239,7 +239,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.3.0-rc5-next-20151016-49772-gcd685d8 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Oct 16 01:18:04 PDT 2015
+Linux version 4.3.0-rc5-next-20151019-50529-gfb68b25 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Oct 19 00:18:06 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -248,7 +248,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0a12400, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0a12480, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -261,7 +261,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 963148K/1048576K available (7151K kernel code, 351K rwdata, 2444K rodata, 336K init, 364K bss, 19892K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 963148K/1048576K available (7147K kernel code, 351K rwdata, 2444K rodata, 340K init, 364K bss, 19892K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -269,8 +269,8 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09670ec   (9597 kB)
-      .init : 0xc0968000 - 0xc09bc000   ( 336 kB)
+      .text : 0xc0008000 - 0xc0966114   (9593 kB)
+      .init : 0xc0967000 - 0xc09bc000   ( 340 kB)
       .data : 0xc09bc000 - 0xc0a13fa0   ( 352 kB)
        .bss : 0xc0a16000 - 0xc0a710c0   ( 365 kB)
 Preemptible hierarchical RCU implementation.
@@ -508,11 +508,11 @@
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-26 23:02:09 UTC (1419634929)
+rtc-em3027 2-0056: setting system clock to 2014-12-26 23:05:58 UTC (1419635158)
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
- mmcblk0: p1
 ALSA device list:
   #0: tegra-trimslice
+ mmcblk0: p1
 EXT2-fs (mmcblk0p1): error: couldn't mount because of unsupported optional features (4)
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 usb 2-1: new high-speed USB device number 2 using tegra-ehci
@@ -521,12 +521,11 @@
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 mmc1: new high speed SD card at address b368
 devtmpfs: mounted
+Freeing unused kernel memory: 340K (c0967000 - c09bc000)
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
-Freeing unused kernel memory: 336K (c0968000 - c09bc000)
  mmcblk1: p1 p2
 hub 2-1:1.0: USB hub found
 hub 2-1:1.0: 4 ports detected
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-usb 2-1.2: new high-speed USB device number 3 using tegra-ehci
-root@(none):/# 
\ No newline at end of file
+root@(none):/# [    4.467279] usb 2-1.2: new high-speed USB device number 3 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-10-19 00:23:53.302989604 -0700
+++ /tmp/foob	2015-10-19 00:23:53.302989604 -0700
@@ -38,20 +38,20 @@
 	 #################################################################
 	 #################################################################
 	 ######################################################
-	 3.3 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 6510488 (635798 hex)
+Bytes transferred = 6516864 (637080 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.6 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x635798 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x637080 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.3.0-rc5-next-20151016-49772-gcd685d8 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Fri Oct 16 01:17:47 PDT 2015
+Linux version 4.3.0-rc5-next-20151019-50529-gfb68b25 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Oct 19 00:17:46 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -67,7 +67,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0fb1d80, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c0fb59c0, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -80,7 +80,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998580K/2096128K available (8225K kernel code, 1031K rwdata, 3696K rodata, 1040K init, 340K bss, 32012K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998564K/2096128K available (8226K kernel code, 1038K rwdata, 3708K rodata, 1036K init, 340K bss, 32028K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,10 +88,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0dad7f4   (11926 kB)
-      .init : 0xc0dae000 - 0xc0eb2000   (1040 kB)
-      .data : 0xc0eb2000 - 0xc0fb3d60   (1032 kB)
-       .bss : 0xc0fb6000 - 0xc100b3e8   ( 341 kB)
+      .text : 0xc0208000 - 0xc0db0814   (11939 kB)
+      .init : 0xc0db1000 - 0xc0eb4000   (1036 kB)
+      .data : 0xc0eb4000 - 0xc0fb7920   (1039 kB)
+       .bss : 0xc0fba000 - 0xc100f3e8   ( 341 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -350,14 +350,14 @@
 mmcblk0rpmb: mmc1:0001 SEM16G partition 3 128 KiB
 Primary GPT is invalid, using alternate GPT.
  mmcblk0: p1 p2 p3 p4 p5
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk0p1): warning: mounting fs with errors, running e2fsck is recommended
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1040K (c0dae000 - c0eb2000)
+Freeing unused kernel memory: 1036K (c0db1000 - c0eb4000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2015-10-19 00:23:53.310989596 -0700
+++ /tmp/foob	2015-10-19 00:23:53.310989596 -0700
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 ##############################
-	 3.3 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 5204560 (4f6a50 hex)
+Bytes transferred = 5204504 (4f6a18 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.9 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4f6a50 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4f6a18 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.3.0-rc5-next-20151016-49772-gcd685d8 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Oct 16 01:18:04 PDT 2015
+Linux version 4.3.0-rc5-next-20151019-50529-gfb68b25 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Oct 19 00:18:06 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0a12400, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0a12480, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002260K/2096128K available (7151K kernel code, 351K rwdata, 2444K rodata, 336K init, 364K bss, 28332K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2002260K/2096128K available (7147K kernel code, 351K rwdata, 2444K rodata, 340K init, 364K bss, 28332K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,8 +90,8 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09670ec   (9597 kB)
-      .init : 0xc0968000 - 0xc09bc000   ( 336 kB)
+      .text : 0xc0008000 - 0xc0966114   (9593 kB)
+      .init : 0xc0967000 - 0xc09bc000   ( 340 kB)
       .data : 0xc09bc000 - 0xc0a13fa0   ( 352 kB)
        .bss : 0xc0a16000 - 0xc0a710c0   ( 365 kB)
 Preemptible hierarchical RCU implementation.
@@ -343,9 +343,9 @@
 mmc1: new high speed MMC card at address 0001
 mmcblk0: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk0boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 mmcblk0boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk0rpmb: mmc1:0001 SEM16G partition 3 128 KiB
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 Primary GPT is invalid, using alternate GPT.
  mmcblk0: p1 p2 p3 p4 p5
 tegra-snd-rt5640 sound: rt5640-aif1 <-> 70080400.i2s mapping ok
@@ -363,7 +363,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 336K (c0968000 - c09bc000)
+Freeing unused kernel memory: 340K (c0967000 - c09bc000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
