Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	1969-12-31 16:00:00.000000000 -0800
+++ /tmp/foob	2015-12-10 22:58:23.127265655 -0800
@@ -0,0 +1,148 @@
+Booting Linux on physical CPU 0x0
+Initializing cgroup subsys cpu
+Linux version 4.4.0-rc4-next-20151211+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Dec 10 22:53:23 PST 2015
+Boot CPU: AArch64 Processor [411fd070]
+Memory limited to 1024MB
+efi: Getting EFI parameters from FDT:
+efi: UEFI not found.
+cma: Reserved 16 MiB at 0x000000007f000000
+psci: probing for conduit method from DT.
+psci: PSCIv0.2 detected in firmware.
+psci: Using standard PSCI v0.2 function IDs
+psci: Trusted OS migration not required
+PERCPU: Embedded 20 pages/cpu @ffffffc03efdb000 s41472 r8192 d32256 u81920
+Detected PIPT I-cache on CPU0
+CPU features: enabling workaround for ARM erratum 832075
+CPU features: enabling workaround for ARM erratum 834220
+Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 258048
+Kernel command line: root=/dev/vda2 rw rootwait mem=1024M console=ttyAMA0,38400n8 init=/bin/bash
+PID hash table entries: 4096 (order: 3, 32768 bytes)
+Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
+Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
+software IO TLB [mem 0x79c00000-0x7dc00000] (64MB) mapped at [ffffffc039c00000-ffffffc03dbfffff]
+Memory: 938124K/1048576K available (5912K kernel code, 559K rwdata, 2560K rodata, 660K init, 223K bss, 94068K reserved, 16384K cma-reserved)
+Virtual kernel memory layout:
+    vmalloc : 0xffffff8000000000 - 0xffffffbdbfff0000   (   246 GB)
+    vmemmap : 0xffffffbdc0000000 - 0xffffffbfc0000000   (     8 GB maximum)
+              0xffffffbdc1000000 - 0xffffffbdc2000000   (    16 MB actual)
+    fixed   : 0xffffffbffa7fd000 - 0xffffffbffac00000   (  4108 KB)
+    PCI I/O : 0xffffffbffae00000 - 0xffffffbffbe00000   (    16 MB)
+    modules : 0xffffffbffc000000 - 0xffffffc000000000   (    64 MB)
+    memory  : 0xffffffc000000000 - 0xffffffc040000000   (  1024 MB)
+      .init : 0xffffffc0008c9000 - 0xffffffc00096e000   (   660 KB)
+      .text : 0xffffffc000080000 - 0xffffffc0008c80b4   (  8481 KB)
+      .data : 0xffffffc00096e000 - 0xffffffc0009f9e00   (   560 KB)
+SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
+Preemptible hierarchical RCU implementation.
+	Build-time adjustment of leaf fanout to 64.
+	RCU restricting CPUs from NR_CPUS=64 to nr_cpu_ids=1.
+RCU: Adjusting geometry for rcu_fanout_leaf=64, nr_cpu_ids=1
+NR_IRQS:64 nr_irqs:64 0
+Architected cp15 timer(s) running at 62.50MHz (virt).
+clocksource: arch_sys_counter: mask: 0xffffffffffffff max_cycles: 0x1cd42e208c, max_idle_ns: 881590405314 ns
+sched_clock: 56 bits at 62MHz, resolution 16ns, wraps every 4398046511096ns
+Console: colour dummy device 80x25
+Calibrating delay loop (skipped), value calculated using timer frequency.. 125.00 BogoMIPS (lpj=250000)
+pid_max: default: 32768 minimum: 301
+Security Framework initialized
+Mount-cache hash table entries: 2048 (order: 2, 16384 bytes)
+Mountpoint-cache hash table entries: 2048 (order: 2, 16384 bytes)
+Initializing cgroup subsys memory
+Initializing cgroup subsys hugetlb
+EFI services will not be available.
+ASID allocator initialised with 65536 entries
+Brought up 1 CPUs
+SMP: Total of 1 processors activated.
+CPU: All CPU(s) started at EL1
+alternatives: patching kernel code
+devtmpfs: initialized
+DMI not present or invalid.
+clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 7645041785100000 ns
+pinctrl core: initialized pinctrl subsystem
+NET: Registered protocol family 16
+cpuidle: using governor ladder
+cpuidle: using governor menu
+vdso: 2 pages (1 code @ ffffffc000975000, 1 data @ ffffffc000974000)
+hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
+DMA: preallocated 256 KiB pool for atomic allocations
+Serial: AMBA PL011 UART driver
+9000000.pl011: ttyAMA0 at MMIO 0x9000000 (irq = 38, base_baud = 0) is a PL011 rev1
+console [ttyAMA0] enabled
+vgaarb: loaded
+SCSI subsystem initialized
+usbcore: registered new interface driver usbfs
+usbcore: registered new interface driver hub
+usbcore: registered new device driver usb
+pps_core: LinuxPPS API ver. 1 registered
+pps_core: Software ver. 5.3.6 - Copyright 2005-2007 Rodolfo Giometti <giometti@linux.it>
+PTP clock support registered
+dmi: Firmware registration failed.
+Advanced Linux Sound Architecture Driver Initialized.
+clocksource: Switched to clocksource arch_sys_counter
+NET: Registered protocol family 2
+TCP established hash table entries: 8192 (order: 4, 65536 bytes)
+TCP bind hash table entries: 8192 (order: 5, 131072 bytes)
+TCP: Hash tables configured (established 8192 bind 8192)
+UDP hash table entries: 512 (order: 2, 16384 bytes)
+UDP-Lite hash table entries: 512 (order: 2, 16384 bytes)
+NET: Registered protocol family 1
+RPC: Registered named UNIX socket transport module.
+RPC: Registered udp transport module.
+RPC: Registered tcp transport module.
+RPC: Registered tcp NFSv4.1 backchannel transport module.
+kvm [1]: HYP mode not available
+futex hash table entries: 256 (order: 3, 32768 bytes)
+audit: initializing netlink subsys (disabled)
+audit: type=2000 audit(0.592:1): initialized
+HugeTLB registered 2 MB page size, pre-allocated 0 pages
+VFS: Disk quotas dquot_6.6.0
+VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
+squashfs: version 4.0 (2009/01/31) Phillip Lougher
+NFS: Registering the id_resolver key type
+Key type id_resolver registered
+Key type id_legacy registered
+fuse init (API version 7.24)
+9p: Installing v9fs 9p2000 file system support
+io scheduler noop registered
+io scheduler cfq registered (default)
+Serial: 8250/16550 driver, 4 ports, IRQ sharing disabled
+SuperH (H)SCI(F) driver initialized
+msm_serial: driver initialized
+Unable to detect cache hierarchy from DT for CPU 0
+loop: module loaded
+ vda: vda1 vda2
+tun: Universal TUN/TAP device driver, 1.6
+tun: (C) 1999-2004 Max Krasnyansky <maxk@qualcomm.com>
+sky2: driver version 1.30
+ehci_hcd: USB 2.0 'Enhanced' Host Controller (EHCI) Driver
+ehci-pci: EHCI PCI platform driver
+ehci-platform: EHCI generic platform driver
+ohci_hcd: USB 1.1 'Open' Host Controller (OHCI) Driver
+ohci-pci: OHCI PCI platform driver
+ohci-platform: OHCI generic platform driver
+usbcore: registered new interface driver usb-storage
+mousedev: PS/2 mouse device common for all mice
+sdhci: Secure Digital Host Controller Interface driver
+sdhci: Copyright(c) Pierre Ossman
+Synopsys Designware Multimedia Card Interface Driver
+sdhci-pltfm: SDHCI platform and OF driver helper
+ledtrig-cpu: registered to indicate activity on CPUs
+usbcore: registered new interface driver usbhid
+usbhid: USB HID core driver
+NET: Registered protocol family 17
+9pnet: Installing 9P2000 support
+Key type dns_resolver registered
+registered taskstats version 1
+hctosys: unable to open rtc device (rtc0)
+ALSA device list:
+  No soundcards found.
+uart-pl011 9000000.pl011: no DMA platform data
+EXT4-fs (vda2): couldn't mount as ext3 due to feature incompatibilities
+EXT4-fs (vda2): recovery complete
+EXT4-fs (vda2): mounted filesystem with ordered data mode. Opts: (null)
+VFS: Mounted root (ext4 filesystem) on device 254:2.
+devtmpfs: mounted
+Freeing unused kernel memory: 660K (ffffffc0008c9000 - ffffffc00096e000)
+bash: cannot set terminal process group (-1): Inappropriate ioctl for device
+bash: no job control in this shell
+bash-4.3# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-12-10 22:58:23.151265635 -0800
+++ /tmp/foob	2015-12-10 22:58:23.151265635 -0800
@@ -38,9 +38,9 @@
 	 #################################################################
 	 #################################################################
 	 ############################################################
-	 3.4 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 6598280 (64ae88 hex)
+Bytes transferred = 6599800 (64b478 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -51,7 +51,7 @@
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x64ae88 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x64b478 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-rc4-next-20151210-60419-g8225f4e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Dec 9 22:18:44 PST 2015
+Linux version 4.4.0-rc4-next-20151211-60578-g9379b04 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Dec 10 22:51:56 PST 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -67,20 +67,20 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0ff8880, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c0ffa980, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-PERCPU: Embedded 12 pages/cpu @eefa3000 s19136 r8192 d21824 u49152
-pcpu-alloc: s19136 r8192 d21824 u49152 alloc=12*4096
+PERCPU: Embedded 13 pages/cpu @eef9f000 s23232 r8192 d21824 u53248
+pcpu-alloc: s23232 r8192 d21824 u53248 alloc=13*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998372K/2096128K available (8330K kernel code, 1090K rwdata, 3840K rodata, 1016K init, 344K bss, 32220K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998348K/2096128K available (8334K kernel code, 1090K rwdata, 3840K rodata, 1020K init, 344K bss, 32244K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,10 +88,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0deb814   (12175 kB)
-      .init : 0xc0dec000 - 0xc0eea000   (1016 kB)
-      .data : 0xc0eea000 - 0xc0ffa800   (1090 kB)
-       .bss : 0xc0ffd000 - 0xc10530a8   ( 345 kB)
+      .text : 0xc0208000 - 0xc0dec824   (12179 kB)
+      .init : 0xc0ded000 - 0xc0eec000   (1020 kB)
+      .data : 0xc0eec000 - 0xc0ffc900   (1091 kB)
+       .bss : 0xc0fff000 - 0xc10550a8   ( 345 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -293,45 +293,45 @@
 WARNING: CPU: 2 PID: 65 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 2 PID: 65 Comm: kworker/u8:1 Not tainted 4.4.0-rc4-next-20151210-60419-g8225f4e #1
+CPU: 2 PID: 65 Comm: kworker/u8:1 Not tainted 4.4.0-rc4-next-20151211-60578-g9379b04 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
-[<c0218870>] (unwind_backtrace) from [<c0213f10>] (show_stack+0x10/0x14)
-[<c0213f10>] (show_stack) from [<c0485e2c>] (dump_stack+0x78/0x94)
-[<c0485e2c>] (dump_stack) from [<c024ab28>] (warn_slowpath_common+0x74/0xb0)
+[<c0218878>] (unwind_backtrace) from [<c0213f10>] (show_stack+0x10/0x14)
+[<c0213f10>] (show_stack) from [<c04863ac>] (dump_stack+0x78/0x94)
+[<c04863ac>] (dump_stack) from [<c024ab28>] (warn_slowpath_common+0x74/0xb0)
 [<c024ab28>] (warn_slowpath_common) from [<c024abf8>] (warn_slowpath_fmt+0x30/0x40)
 [<c024abf8>] (warn_slowpath_fmt) from [<c025ddd4>] (check_flush_dependency+0xc8/0x124)
 [<c025ddd4>] (check_flush_dependency) from [<c025dee8>] (flush_work+0x54/0x144)
-[<c025dee8>] (flush_work) from [<c02d9e0c>] (lru_add_drain_all+0x124/0x180)
-[<c02d9e0c>] (lru_add_drain_all) from [<c030b50c>] (migrate_prep+0x8/0x10)
-[<c030b50c>] (migrate_prep) from [<c02d4e24>] (alloc_contig_range+0xc4/0x2f8)
-[<c02d4e24>] (alloc_contig_range) from [<c030ce38>] (cma_alloc+0xd8/0x1a0)
-[<c030ce38>] (cma_alloc) from [<c021e31c>] (__alloc_from_contiguous+0x38/0xdc)
+[<c025dee8>] (flush_work) from [<c02da1c8>] (lru_add_drain_all+0x124/0x180)
+[<c02da1c8>] (lru_add_drain_all) from [<c030b928>] (migrate_prep+0x8/0x10)
+[<c030b928>] (migrate_prep) from [<c02d51e0>] (alloc_contig_range+0xc4/0x2f8)
+[<c02d51e0>] (alloc_contig_range) from [<c030d254>] (cma_alloc+0xd8/0x1a0)
+[<c030d254>] (cma_alloc) from [<c021e31c>] (__alloc_from_contiguous+0x38/0xdc)
 [<c021e31c>] (__alloc_from_contiguous) from [<c021ff10>] (__dma_alloc+0x21c/0x250)
 [<c021ff10>] (__dma_alloc) from [<c021ffc8>] (arm_dma_alloc+0x48/0x54)
-[<c021ffc8>] (arm_dma_alloc) from [<c0305a5c>] (dma_pool_alloc+0x1dc/0x210)
-[<c0305a5c>] (dma_pool_alloc) from [<c07a3798>] (ehci_qh_alloc+0x3c/0xb4)
-[<c07a3798>] (ehci_qh_alloc) from [<c07a7d90>] (ehci_setup+0x154/0x478)
-[<c07a7d90>] (ehci_setup) from [<c07ade48>] (tegra_ehci_reset+0xc/0x50)
-[<c07ade48>] (tegra_ehci_reset) from [<c077e480>] (usb_add_hcd+0x300/0x830)
-[<c077e480>] (usb_add_hcd) from [<c07ad7c0>] (tegra_ehci_probe+0x3b8/0x504)
-[<c07ad7c0>] (tegra_ehci_probe) from [<c064474c>] (platform_drv_probe+0x50/0xac)
-[<c064474c>] (platform_drv_probe) from [<c06430b0>] (driver_probe_device+0x1d4/0x290)
-[<c06430b0>] (driver_probe_device) from [<c06416a8>] (bus_for_each_drv+0x44/0x8c)
-[<c06416a8>] (bus_for_each_drv) from [<c0642e2c>] (__device_attach+0xa0/0x104)
-[<c0642e2c>] (__device_attach) from [<c0642528>] (bus_probe_device+0x84/0x8c)
-[<c0642528>] (bus_probe_device) from [<c064295c>] (deferred_probe_work_func+0x60/0x8c)
-[<c064295c>] (deferred_probe_work_func) from [<c025ed30>] (process_one_work+0x134/0x338)
+[<c021ffc8>] (arm_dma_alloc) from [<c0305e78>] (dma_pool_alloc+0x1dc/0x210)
+[<c0305e78>] (dma_pool_alloc) from [<c07a3c30>] (ehci_qh_alloc+0x3c/0xb4)
+[<c07a3c30>] (ehci_qh_alloc) from [<c07a8228>] (ehci_setup+0x154/0x478)
+[<c07a8228>] (ehci_setup) from [<c07ae2e0>] (tegra_ehci_reset+0xc/0x50)
+[<c07ae2e0>] (tegra_ehci_reset) from [<c077e918>] (usb_add_hcd+0x300/0x830)
+[<c077e918>] (usb_add_hcd) from [<c07adc58>] (tegra_ehci_probe+0x3b8/0x504)
+[<c07adc58>] (tegra_ehci_probe) from [<c06446f8>] (platform_drv_probe+0x50/0xac)
+[<c06446f8>] (platform_drv_probe) from [<c064305c>] (driver_probe_device+0x1d4/0x290)
+[<c064305c>] (driver_probe_device) from [<c0641654>] (bus_for_each_drv+0x44/0x8c)
+[<c0641654>] (bus_for_each_drv) from [<c0642dd8>] (__device_attach+0xa0/0x104)
+[<c0642dd8>] (__device_attach) from [<c06424d4>] (bus_probe_device+0x84/0x8c)
+[<c06424d4>] (bus_probe_device) from [<c0642908>] (deferred_probe_work_func+0x60/0x8c)
+[<c0642908>] (deferred_probe_work_func) from [<c025ed30>] (process_one_work+0x134/0x338)
 [<c025ed30>] (process_one_work) from [<c025ef98>] (worker_thread+0x2c/0x4f0)
 [<c025ef98>] (worker_thread) from [<c0264128>] (kthread+0xd8/0xf4)
 [<c0264128>] (kthread) from [<c0210df8>] (ret_from_fork+0x14/0x3c)
----[ end trace faed461176012dc3 ]---
+---[ end trace 6c5737e6a9e0850a ]---
 tegra-ehci 7d008000.usb: irq 339, io mem 0x7d008000
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-06-21 15:17:50 UTC (961600670)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-06-22 15:51:04 UTC (961689064)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -340,7 +340,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1016K (c0dec000 - c0eea000)
+Freeing unused kernel memory: 1020K (c0ded000 - c0eec000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.494290] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.484194] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2015-12-10 22:58:23.159265629 -0800
+++ /tmp/foob	2015-12-10 22:58:23.159265629 -0800
@@ -39,7 +39,7 @@
 	 ################################
 	 3.3 MiB/s
 done
-Bytes transferred = 5230384 (4fcf30 hex)
+Bytes transferred = 5231152 (4fd230 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4fcf30 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4fd230 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.4.0-rc4-next-20151210-60419-g8225f4e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Dec 9 22:19:01 PST 2015
+Linux version 4.4.0-rc4-next-20151211-60578-g9379b04 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Dec 10 22:52:12 PST 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -70,20 +70,20 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0a22cc0, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0a24d00, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-PERCPU: Embedded 12 pages/cpu @eefa2000 s19456 r8192 d21504 u49152
-pcpu-alloc: s19456 r8192 d21504 u49152 alloc=12*4096
+PERCPU: Embedded 13 pages/cpu @eef9e000 s23488 r8192 d21568 u53248
+pcpu-alloc: s23488 r8192 d21568 u53248 alloc=13*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002272K/2096128K available (7179K kernel code, 369K rwdata, 2464K rodata, 336K init, 367K bss, 28320K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2002248K/2096128K available (7179K kernel code, 370K rwdata, 2464K rodata, 344K init, 367K bss, 28344K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -91,10 +91,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09732fc   (9645 kB)
-      .init : 0xc0974000 - 0xc09c8000   ( 336 kB)
-      .data : 0xc09c8000 - 0xc0a247e0   ( 370 kB)
-       .bss : 0xc0a27000 - 0xc0a82d90   ( 368 kB)
+      .text : 0xc0008000 - 0xc09732f4   (9645 kB)
+      .init : 0xc0974000 - 0xc09ca000   ( 344 kB)
+      .data : 0xc09ca000 - 0xc0a26820   ( 371 kB)
+       .bss : 0xc0a29000 - 0xc0a84d90   ( 368 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -296,45 +296,45 @@
 WARNING: CPU: 0 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-rc4-next-20151210-60419-g8225f4e #1
+CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-rc4-next-20151211-60578-g9379b04 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
-[<c0017240>] (unwind_backtrace) from [<c0012870>] (show_stack+0x10/0x14)
-[<c0012870>] (show_stack) from [<c0238d84>] (dump_stack+0x84/0xd0)
-[<c0238d84>] (dump_stack) from [<c0026b60>] (warn_slowpath_common+0x74/0xb0)
+[<c0017248>] (unwind_backtrace) from [<c0012870>] (show_stack+0x10/0x14)
+[<c0012870>] (show_stack) from [<c02392e4>] (dump_stack+0x84/0xd0)
+[<c02392e4>] (dump_stack) from [<c0026b60>] (warn_slowpath_common+0x74/0xb0)
 [<c0026b60>] (warn_slowpath_common) from [<c0026c30>] (warn_slowpath_fmt+0x30/0x40)
 [<c0026c30>] (warn_slowpath_fmt) from [<c00398dc>] (check_flush_dependency+0xc8/0x12c)
 [<c00398dc>] (check_flush_dependency) from [<c00399f8>] (flush_work+0x54/0x11c)
-[<c00399f8>] (flush_work) from [<c00bfce4>] (lru_add_drain_all+0x124/0x180)
-[<c00bfce4>] (lru_add_drain_all) from [<c00f06d8>] (migrate_prep+0x8/0x10)
-[<c00f06d8>] (migrate_prep) from [<c00ba85c>] (alloc_contig_range+0xc4/0x2f8)
-[<c00ba85c>] (alloc_contig_range) from [<c00f2248>] (cma_alloc+0xd8/0x1a0)
-[<c00f2248>] (cma_alloc) from [<c001c250>] (__alloc_from_contiguous+0x38/0xdc)
+[<c00399f8>] (flush_work) from [<c00c008c>] (lru_add_drain_all+0x124/0x180)
+[<c00c008c>] (lru_add_drain_all) from [<c00f0ae0>] (migrate_prep+0x8/0x10)
+[<c00f0ae0>] (migrate_prep) from [<c00bac04>] (alloc_contig_range+0xc4/0x2f8)
+[<c00bac04>] (alloc_contig_range) from [<c00f2650>] (cma_alloc+0xd8/0x1a0)
+[<c00f2650>] (cma_alloc) from [<c001c250>] (__alloc_from_contiguous+0x38/0xdc)
 [<c001c250>] (__alloc_from_contiguous) from [<c001cf18>] (__dma_alloc+0x228/0x2ac)
 [<c001cf18>] (__dma_alloc) from [<c001d020>] (arm_dma_alloc+0x48/0x54)
-[<c001d020>] (arm_dma_alloc) from [<c00ecae4>] (dma_pool_alloc+0x28c/0x304)
-[<c00ecae4>] (dma_pool_alloc) from [<c03f74ec>] (ehci_qh_alloc+0x3c/0xb4)
-[<c03f74ec>] (ehci_qh_alloc) from [<c03fb6d8>] (ehci_setup+0x154/0x478)
-[<c03fb6d8>] (ehci_setup) from [<c03ff420>] (tegra_ehci_reset+0xc/0x50)
-[<c03ff420>] (tegra_ehci_reset) from [<c03e2ed8>] (usb_add_hcd+0x2fc/0x804)
-[<c03e2ed8>] (usb_add_hcd) from [<c03fed98>] (tegra_ehci_probe+0x3b8/0x504)
-[<c03fed98>] (tegra_ehci_probe) from [<c03323c8>] (platform_drv_probe+0x50/0xac)
-[<c03323c8>] (platform_drv_probe) from [<c0330e28>] (driver_probe_device+0x1d4/0x290)
-[<c0330e28>] (driver_probe_device) from [<c032f424>] (bus_for_each_drv+0x44/0x8c)
-[<c032f424>] (bus_for_each_drv) from [<c0330ba4>] (__device_attach+0xa0/0x104)
-[<c0330ba4>] (__device_attach) from [<c03302a0>] (bus_probe_device+0x84/0x8c)
-[<c03302a0>] (bus_probe_device) from [<c03306d4>] (deferred_probe_work_func+0x60/0x8c)
-[<c03306d4>] (deferred_probe_work_func) from [<c003a804>] (process_one_work+0x120/0x328)
+[<c001d020>] (arm_dma_alloc) from [<c00eceec>] (dma_pool_alloc+0x28c/0x304)
+[<c00eceec>] (dma_pool_alloc) from [<c03f7768>] (ehci_qh_alloc+0x3c/0xb4)
+[<c03f7768>] (ehci_qh_alloc) from [<c03fb954>] (ehci_setup+0x154/0x478)
+[<c03fb954>] (ehci_setup) from [<c03ff69c>] (tegra_ehci_reset+0xc/0x50)
+[<c03ff69c>] (tegra_ehci_reset) from [<c03e3154>] (usb_add_hcd+0x2fc/0x804)
+[<c03e3154>] (usb_add_hcd) from [<c03ff014>] (tegra_ehci_probe+0x3b8/0x504)
+[<c03ff014>] (tegra_ehci_probe) from [<c03322b0>] (platform_drv_probe+0x50/0xac)
+[<c03322b0>] (platform_drv_probe) from [<c0330d10>] (driver_probe_device+0x1d4/0x290)
+[<c0330d10>] (driver_probe_device) from [<c032f30c>] (bus_for_each_drv+0x44/0x8c)
+[<c032f30c>] (bus_for_each_drv) from [<c0330a8c>] (__device_attach+0xa0/0x104)
+[<c0330a8c>] (__device_attach) from [<c0330188>] (bus_probe_device+0x84/0x8c)
+[<c0330188>] (bus_probe_device) from [<c03305bc>] (deferred_probe_work_func+0x60/0x8c)
+[<c03305bc>] (deferred_probe_work_func) from [<c003a804>] (process_one_work+0x120/0x328)
 [<c003a804>] (process_one_work) from [<c003aa70>] (worker_thread+0x2c/0x4b4)
 [<c003aa70>] (worker_thread) from [<c003fbac>] (kthread+0xd8/0xf4)
 [<c003fbac>] (kthread) from [<c000f938>] (ret_from_fork+0x14/0x3c)
----[ end trace 886f9f1541da9266 ]---
+---[ end trace e64ff063e5ffd2ef ]---
 tegra-ehci 7d008000.usb: irq 339, io mem 0x7d008000
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-06-21 15:18:27 UTC (961600707)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-06-22 15:51:41 UTC (961689101)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -345,7 +345,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 336K (c0974000 - c09c8000)
+Freeing unused kernel memory: 344K (c0974000 - c09ca000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-12-10 22:58:23.167265622 -0800
+++ /tmp/foob	2015-12-10 22:58:23.171265619 -0800
@@ -29,6 +29,7 @@
 Tegra124 (Jetson TK1) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash'
 Tegra124 (Jetson TK1) # setenv ipaddr 192.168.100.102; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 zImage.multi_v7_defconfig
+Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'zImage.multi_v7_defconfig'.
@@ -40,20 +41,20 @@
 	 #################################################################
 	 #################################################################
 	 ############################################################
-	 3.4 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 6598280 (64ae88 hex)
+Bytes transferred = 6599800 (64b478 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.1 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x64ae88 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x64b478 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012a89
@@ -61,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-rc4-next-20151210-60419-g8225f4e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Dec 9 22:18:44 PST 2015
+Linux version 4.4.0-rc4-next-20151211-60578-g9379b04 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Dec 10 22:51:56 PST 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -69,20 +70,20 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0ff8880, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c0ffa980, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-PERCPU: Embedded 12 pages/cpu @eef8f000 s19136 r8192 d21824 u49152
-pcpu-alloc: s19136 r8192 d21824 u49152 alloc=12*4096
+PERCPU: Embedded 13 pages/cpu @eef8b000 s23232 r8192 d21824 u53248
+pcpu-alloc: s23232 r8192 d21824 u53248 alloc=13*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998264K/2096128K available (8330K kernel code, 1090K rwdata, 3840K rodata, 1016K init, 344K bss, 32328K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998240K/2096128K available (8334K kernel code, 1090K rwdata, 3840K rodata, 1020K init, 344K bss, 32352K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +91,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0deb814   (12175 kB)
-      .init : 0xc0dec000 - 0xc0eea000   (1016 kB)
-      .data : 0xc0eea000 - 0xc0ffa800   (1090 kB)
-       .bss : 0xc0ffd000 - 0xc10530a8   ( 345 kB)
+      .text : 0xc0208000 - 0xc0dec824   (12179 kB)
+      .init : 0xc0ded000 - 0xc0eec000   (1020 kB)
+      .data : 0xc0eec000 - 0xc0ffc900   (1091 kB)
+       .bss : 0xc0fff000 - 0xc10550a8   ( 345 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -326,43 +327,43 @@
 tegra-ahci 70027000.sata: AHCI 0001.0301 32 slots 2 ports 3 Gbps 0x1 impl platform mode
 tegra-ahci 70027000.sata: flags: 64bit ncq sntf pm led pio slum part sadm sds apst 
 ------------[ cut here ]------------
-WARNING: CPU: 2 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
+WARNING: CPU: 2 PID: 66 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 2 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-rc4-next-20151210-60419-g8225f4e #1
+CPU: 2 PID: 66 Comm: kworker/u8:1 Not tainted 4.4.0-rc4-next-20151211-60578-g9379b04 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
-[<c0218870>] (unwind_backtrace) from [<c0213f10>] (show_stack+0x10/0x14)
-[<c0213f10>] (show_stack) from [<c0485e2c>] (dump_stack+0x78/0x94)
-[<c0485e2c>] (dump_stack) from [<c024ab28>] (warn_slowpath_common+0x74/0xb0)
+[<c0218878>] (unwind_backtrace) from [<c0213f10>] (show_stack+0x10/0x14)
+[<c0213f10>] (show_stack) from [<c04863ac>] (dump_stack+0x78/0x94)
+[<c04863ac>] (dump_stack) from [<c024ab28>] (warn_slowpath_common+0x74/0xb0)
 [<c024ab28>] (warn_slowpath_common) from [<c024abf8>] (warn_slowpath_fmt+0x30/0x40)
 [<c024abf8>] (warn_slowpath_fmt) from [<c025ddd4>] (check_flush_dependency+0xc8/0x124)
 [<c025ddd4>] (check_flush_dependency) from [<c025dee8>] (flush_work+0x54/0x144)
-[<c025dee8>] (flush_work) from [<c02d9e0c>] (lru_add_drain_all+0x124/0x180)
-[<c02d9e0c>] (lru_add_drain_all) from [<c030b50c>] (migrate_prep+0x8/0x10)
-[<c030b50c>] (migrate_prep) from [<c02d4e24>] (alloc_contig_range+0xc4/0x2f8)
-[<c02d4e24>] (alloc_contig_range) from [<c030ce38>] (cma_alloc+0xd8/0x1a0)
-[<c030ce38>] (cma_alloc) from [<c021e31c>] (__alloc_from_contiguous+0x38/0xdc)
+[<c025dee8>] (flush_work) from [<c02da1c8>] (lru_add_drain_all+0x124/0x180)
+[<c02da1c8>] (lru_add_drain_all) from [<c030b928>] (migrate_prep+0x8/0x10)
+[<c030b928>] (migrate_prep) from [<c02d51e0>] (alloc_contig_range+0xc4/0x2f8)
+[<c02d51e0>] (alloc_contig_range) from [<c030d254>] (cma_alloc+0xd8/0x1a0)
+[<c030d254>] (cma_alloc) from [<c021e31c>] (__alloc_from_contiguous+0x38/0xdc)
 [<c021e31c>] (__alloc_from_contiguous) from [<c021ff10>] (__dma_alloc+0x21c/0x250)
 [<c021ff10>] (__dma_alloc) from [<c021ffc8>] (arm_dma_alloc+0x48/0x54)
-[<c021ffc8>] (arm_dma_alloc) from [<c0654f30>] (dmam_alloc_coherent+0x8c/0xf0)
-[<c0654f30>] (dmam_alloc_coherent) from [<c06accb4>] (ahci_port_start+0xd0/0x1e0)
-[<c06accb4>] (ahci_port_start) from [<c0694b5c>] (ata_host_start+0xf8/0x1d4)
-[<c0694b5c>] (ata_host_start) from [<c0699a78>] (ata_host_activate+0x20/0x134)
-[<c0699a78>] (ata_host_activate) from [<c06ac130>] (ahci_host_activate+0x3c/0x11c)
-[<c06ac130>] (ahci_host_activate) from [<c06ae074>] (ahci_platform_init_host+0x1cc/0x3c8)
-[<c06ae074>] (ahci_platform_init_host) from [<c06b02d8>] (tegra_ahci_probe+0x404/0x4c4)
-[<c06b02d8>] (tegra_ahci_probe) from [<c064474c>] (platform_drv_probe+0x50/0xac)
-[<c064474c>] (platform_drv_probe) from [<c06430b0>] (driver_probe_device+0x1d4/0x290)
-[<c06430b0>] (driver_probe_device) from [<c06416a8>] (bus_for_each_drv+0x44/0x8c)
-[<c06416a8>] (bus_for_each_drv) from [<c0642e2c>] (__device_attach+0xa0/0x104)
-[<c0642e2c>] (__device_attach) from [<c0642528>] (bus_probe_device+0x84/0x8c)
-[<c0642528>] (bus_probe_device) from [<c064295c>] (deferred_probe_work_func+0x60/0x8c)
-[<c064295c>] (deferred_probe_work_func) from [<c025ed30>] (process_one_work+0x134/0x338)
+[<c021ffc8>] (arm_dma_alloc) from [<c0654edc>] (dmam_alloc_coherent+0x8c/0xf0)
+[<c0654edc>] (dmam_alloc_coherent) from [<c06acf8c>] (ahci_port_start+0xd0/0x1e0)
+[<c06acf8c>] (ahci_port_start) from [<c0694e34>] (ata_host_start+0xf8/0x1d4)
+[<c0694e34>] (ata_host_start) from [<c0699d50>] (ata_host_activate+0x20/0x134)
+[<c0699d50>] (ata_host_activate) from [<c06ac408>] (ahci_host_activate+0x3c/0x11c)
+[<c06ac408>] (ahci_host_activate) from [<c06ae34c>] (ahci_platform_init_host+0x1cc/0x3c8)
+[<c06ae34c>] (ahci_platform_init_host) from [<c06b05b0>] (tegra_ahci_probe+0x404/0x4c4)
+[<c06b05b0>] (tegra_ahci_probe) from [<c06446f8>] (platform_drv_probe+0x50/0xac)
+[<c06446f8>] (platform_drv_probe) from [<c064305c>] (driver_probe_device+0x1d4/0x290)
+[<c064305c>] (driver_probe_device) from [<c0641654>] (bus_for_each_drv+0x44/0x8c)
+[<c0641654>] (bus_for_each_drv) from [<c0642dd8>] (__device_attach+0xa0/0x104)
+[<c0642dd8>] (__device_attach) from [<c06424d4>] (bus_probe_device+0x84/0x8c)
+[<c06424d4>] (bus_probe_device) from [<c0642908>] (deferred_probe_work_func+0x60/0x8c)
+[<c0642908>] (deferred_probe_work_func) from [<c025ed30>] (process_one_work+0x134/0x338)
 [<c025ed30>] (process_one_work) from [<c025ef98>] (worker_thread+0x2c/0x4f0)
 [<c025ef98>] (worker_thread) from [<c0264128>] (kthread+0xd8/0xf4)
 [<c0264128>] (kthread) from [<c0210df8>] (ret_from_fork+0x14/0x3c)
----[ end trace 89109f1209420c33 ]---
+---[ end trace 82c57ca66d5d49e5 ]---
 scsi host0: tegra-ahci
 scsi host1: tegra-ahci
 ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 341
@@ -384,7 +385,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:18 UTC (946684818)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:19 UTC (946684819)
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
@@ -398,10 +399,10 @@
 EXT4-fs (mmcblk0p1): couldn't mount as ext2 due to feature incompatibilities
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 VFS: Mounted root (ext4 filesystem) on device 179:1.
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 devtmpfs: mounted
-Freeing unused kernel memory: 1016K (c0dec000 - c0eea000)
+Freeing unused kernel memory: 1020K (c0ded000 - c0eec000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2015-12-10 22:58:23.175265616 -0800
+++ /tmp/foob	2015-12-10 22:58:23.175265616 -0800
@@ -29,7 +29,6 @@
 Tegra124 (Jetson TK1) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash'
 Tegra124 (Jetson TK1) # setenv ipaddr 192.168.100.102; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 zImage.tegra_defconfig
-Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'zImage.tegra_defconfig'.
@@ -40,20 +39,21 @@
 	 #################################################################
 	 #################################################################
 	 ################################
-	 3.1 MiB/s
+	 782.2 KiB/s
 done
-Bytes transferred = 5230384 (4fcf30 hex)
+Bytes transferred = 5231152 (4fd230 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
+Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.2 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x4fcf30 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x4fd230 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012a89
@@ -64,7 +64,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.4.0-rc4-next-20151210-60419-g8225f4e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Dec 9 22:19:01 PST 2015
+Linux version 4.4.0-rc4-next-20151211-60578-g9379b04 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Dec 10 22:52:12 PST 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -73,20 +73,20 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0a22cc0, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0a24d00, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-PERCPU: Embedded 12 pages/cpu @eef8e000 s19456 r8192 d21504 u49152
-pcpu-alloc: s19456 r8192 d21504 u49152 alloc=12*4096
+PERCPU: Embedded 13 pages/cpu @eef8a000 s23488 r8192 d21568 u53248
+pcpu-alloc: s23488 r8192 d21568 u53248 alloc=13*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002164K/2096128K available (7179K kernel code, 369K rwdata, 2464K rodata, 336K init, 367K bss, 28428K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2002140K/2096128K available (7179K kernel code, 370K rwdata, 2464K rodata, 344K init, 367K bss, 28452K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -94,10 +94,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09732fc   (9645 kB)
-      .init : 0xc0974000 - 0xc09c8000   ( 336 kB)
-      .data : 0xc09c8000 - 0xc0a247e0   ( 370 kB)
-       .bss : 0xc0a27000 - 0xc0a82d90   ( 368 kB)
+      .text : 0xc0008000 - 0xc09732f4   (9645 kB)
+      .init : 0xc0974000 - 0xc09ca000   ( 344 kB)
+      .data : 0xc09ca000 - 0xc0a26820   ( 371 kB)
+       .bss : 0xc0a29000 - 0xc0a84d90   ( 368 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -306,16 +306,42 @@
 tegra-pcie 1003000.pcie-controller: link 0 down, ignoring
 tegra-pcie 1003000.pcie-controller: probing port 1, using 1 lanes
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000000
-tegra-pcie 1003000.pcie-controller: link 1 down, retrying
-tegra-pcie 1003000.pcie-controller: link 1 down, retrying
-tegra-pcie 1003000.pcie-controller: link 1 down, retrying
-tegra-pcie 1003000.pcie-controller: link 1 down, ignoring
 tegra-pcie 1003000.pcie-controller: PCI host bridge to bus 0000:00
 pci_bus 0000:00: root bus resource [mem 0x13000000-0x1fffffff]
 pci_bus 0000:00: root bus resource [mem 0x20000000-0x3fffffff pref]
 pci_bus 0000:00: root bus resource [bus 00-ff]
 pci_bus 0000:00: root bus resource [io  0x1000-0xffff]
-PCI: bus0: Fast back to back transfers enabled
+pci 0000:00:02.0: [10de:0e13] type 01 class 0x060400
+pci 0000:00:02.0: PME# supported from D0 D1 D2 D3hot D3cold
+PCI: bus0: Fast back to back transfers disabled
+pci 0000:00:02.0: bridge configuration invalid ([bus 00-00]), reconfiguring
+pci 0000:01:00.0: [10ec:8168] type 00 class 0x020000
+pci 0000:01:00.0: reg 0x10: [io  0x0000-0x00ff]
+pci 0000:01:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit]
+pci 0000:01:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
+pci 0000:01:00.0: supports D1 D2
+pci 0000:01:00.0: PME# supported from D0 D1 D2 D3hot D3cold
+PCI: bus1: Fast back to back transfers disabled
+pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
+pci 0000:00:02.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
+pci 0000:00:02.0: BAR 9: assigned [mem 0x20000000-0x200fffff 64bit pref]
+pci 0000:00:02.0: BAR 7: assigned [io  0x1000-0x1fff]
+pci 0000:01:00.0: BAR 4: assigned [mem 0x20000000-0x20003fff 64bit pref]
+pci 0000:01:00.0: BAR 2: assigned [mem 0x13000000-0x13000fff 64bit]
+pci 0000:01:00.0: BAR 0: assigned [io  0x1000-0x10ff]
+pci 0000:00:02.0: PCI bridge to [bus 01]
+pci 0000:00:02.0:   bridge window [io  0x1000-0x1fff]
+pci 0000:00:02.0:   bridge window [mem 0x13000000-0x130fffff]
+pci 0000:00:02.0:   bridge window [mem 0x20000000-0x200fffff 64bit pref]
+pci 0000:00:02.0: nv_msi_ht_cap_quirk didn't locate host bridge
+pcieport 0000:00:02.0: enabling device (0140 -> 0143)
+pcieport 0000:00:02.0: Signaling PME through PCIe PME interrupt
+pci 0000:01:00.0: Signaling PME through PCIe PME interrupt
+pcie_pme 0000:00:02.0:pcie01: service driver pcie_pme loaded
+r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
+r8169 0000:01:00.0: enabling device (0140 -> 0143)
+r8169 0000:01:00.0 eth0: RTL8168g/8111g at 0xf08f8000, 00:04:4b:25:b3:b4, XID 0c000800 IRQ 388
+r8169 0000:01:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 +5V_HDMI_CON: supplied by +5V_SYS
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 +3.3V_RUN: supplied by +3.3V_SYS
@@ -334,40 +360,40 @@
 WARNING: CPU: 0 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-rc4-next-20151210-60419-g8225f4e #1
+CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-rc4-next-20151211-60578-g9379b04 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
-[<c0017240>] (unwind_backtrace) from [<c0012870>] (show_stack+0x10/0x14)
-[<c0012870>] (show_stack) from [<c0238d84>] (dump_stack+0x84/0xd0)
-[<c0238d84>] (dump_stack) from [<c0026b60>] (warn_slowpath_common+0x74/0xb0)
+[<c0017248>] (unwind_backtrace) from [<c0012870>] (show_stack+0x10/0x14)
+[<c0012870>] (show_stack) from [<c02392e4>] (dump_stack+0x84/0xd0)
+[<c02392e4>] (dump_stack) from [<c0026b60>] (warn_slowpath_common+0x74/0xb0)
 [<c0026b60>] (warn_slowpath_common) from [<c0026c30>] (warn_slowpath_fmt+0x30/0x40)
 [<c0026c30>] (warn_slowpath_fmt) from [<c00398dc>] (check_flush_dependency+0xc8/0x12c)
 [<c00398dc>] (check_flush_dependency) from [<c00399f8>] (flush_work+0x54/0x11c)
-[<c00399f8>] (flush_work) from [<c00bfce4>] (lru_add_drain_all+0x124/0x180)
-[<c00bfce4>] (lru_add_drain_all) from [<c00f06d8>] (migrate_prep+0x8/0x10)
-[<c00f06d8>] (migrate_prep) from [<c00ba85c>] (alloc_contig_range+0xc4/0x2f8)
-[<c00ba85c>] (alloc_contig_range) from [<c00f2248>] (cma_alloc+0xd8/0x1a0)
-[<c00f2248>] (cma_alloc) from [<c001c250>] (__alloc_from_contiguous+0x38/0xdc)
+[<c00399f8>] (flush_work) from [<c00c008c>] (lru_add_drain_all+0x124/0x180)
+[<c00c008c>] (lru_add_drain_all) from [<c00f0ae0>] (migrate_prep+0x8/0x10)
+[<c00f0ae0>] (migrate_prep) from [<c00bac04>] (alloc_contig_range+0xc4/0x2f8)
+[<c00bac04>] (alloc_contig_range) from [<c00f2650>] (cma_alloc+0xd8/0x1a0)
+[<c00f2650>] (cma_alloc) from [<c001c250>] (__alloc_from_contiguous+0x38/0xdc)
 [<c001c250>] (__alloc_from_contiguous) from [<c001cf18>] (__dma_alloc+0x228/0x2ac)
 [<c001cf18>] (__dma_alloc) from [<c001d020>] (arm_dma_alloc+0x48/0x54)
-[<c001d020>] (arm_dma_alloc) from [<c0340ae0>] (dmam_alloc_coherent+0x8c/0xf0)
-[<c0340ae0>] (dmam_alloc_coherent) from [<c038657c>] (ahci_port_start+0xd0/0x1e0)
-[<c038657c>] (ahci_port_start) from [<c036e254>] (ata_host_start+0xf8/0x1d4)
-[<c036e254>] (ata_host_start) from [<c03731e8>] (ata_host_activate+0x20/0x134)
-[<c03731e8>] (ata_host_activate) from [<c0385a24>] (ahci_host_activate+0x3c/0x11c)
-[<c0385a24>] (ahci_host_activate) from [<c0387dcc>] (ahci_platform_init_host+0x1cc/0x3c8)
-[<c0387dcc>] (ahci_platform_init_host) from [<c0387a98>] (tegra_ahci_probe+0x404/0x4c4)
-[<c0387a98>] (tegra_ahci_probe) from [<c03323c8>] (platform_drv_probe+0x50/0xac)
-[<c03323c8>] (platform_drv_probe) from [<c0330e28>] (driver_probe_device+0x1d4/0x290)
-[<c0330e28>] (driver_probe_device) from [<c032f424>] (bus_for_each_drv+0x44/0x8c)
-[<c032f424>] (bus_for_each_drv) from [<c0330ba4>] (__device_attach+0xa0/0x104)
-[<c0330ba4>] (__device_attach) from [<c03302a0>] (bus_probe_device+0x84/0x8c)
-[<c03302a0>] (bus_probe_device) from [<c03306d4>] (deferred_probe_work_func+0x60/0x8c)
-[<c03306d4>] (deferred_probe_work_func) from [<c003a804>] (process_one_work+0x120/0x328)
+[<c001d020>] (arm_dma_alloc) from [<c03409c8>] (dmam_alloc_coherent+0x8c/0xf0)
+[<c03409c8>] (dmam_alloc_coherent) from [<c03867cc>] (ahci_port_start+0xd0/0x1e0)
+[<c03867cc>] (ahci_port_start) from [<c036e4a4>] (ata_host_start+0xf8/0x1d4)
+[<c036e4a4>] (ata_host_start) from [<c0373438>] (ata_host_activate+0x20/0x134)
+[<c0373438>] (ata_host_activate) from [<c0385c74>] (ahci_host_activate+0x3c/0x11c)
+[<c0385c74>] (ahci_host_activate) from [<c038801c>] (ahci_platform_init_host+0x1cc/0x3c8)
+[<c038801c>] (ahci_platform_init_host) from [<c0387ce8>] (tegra_ahci_probe+0x404/0x4c4)
+[<c0387ce8>] (tegra_ahci_probe) from [<c03322b0>] (platform_drv_probe+0x50/0xac)
+[<c03322b0>] (platform_drv_probe) from [<c0330d10>] (driver_probe_device+0x1d4/0x290)
+[<c0330d10>] (driver_probe_device) from [<c032f30c>] (bus_for_each_drv+0x44/0x8c)
+[<c032f30c>] (bus_for_each_drv) from [<c0330a8c>] (__device_attach+0xa0/0x104)
+[<c0330a8c>] (__device_attach) from [<c0330188>] (bus_probe_device+0x84/0x8c)
+[<c0330188>] (bus_probe_device) from [<c03305bc>] (deferred_probe_work_func+0x60/0x8c)
+[<c03305bc>] (deferred_probe_work_func) from [<c003a804>] (process_one_work+0x120/0x328)
 [<c003a804>] (process_one_work) from [<c003aa70>] (worker_thread+0x2c/0x4b4)
 [<c003aa70>] (worker_thread) from [<c003fbac>] (kthread+0xd8/0xf4)
 [<c003fbac>] (kthread) from [<c000f938>] (ret_from_fork+0x14/0x3c)
----[ end trace c7502cfd5f2fae80 ]---
+---[ end trace efa526c62bfad89a ]---
 scsi host0: tegra-ahci
 scsi host1: tegra-ahci
 ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 341
@@ -389,7 +415,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:56 UTC (946684856)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:59 UTC (946684859)
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
@@ -404,11 +430,11 @@
 ata1: SATA link down (SStatus 0 SControl 300)
 EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
 EXT4-fs (mmcblk0p1): recovery complete
-EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 usb 2-1: new high-speed USB device number 2 using tegra-ehci
+EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 336K (c0974000 - c09c8000)
+Freeing unused kernel memory: 344K (c0974000 - c09ca000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    6.732788] asix 2-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:11:31:3b
+root@(none):/# [    5.712754] asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:11:31:3b
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-12-10 22:58:23.183265609 -0800
+++ /tmp/foob	2015-12-10 22:58:23.183265609 -0800
@@ -46,20 +46,20 @@
 	 #################################################################
 	 #################################################################
 	 ############################################################
-	 4.1 MiB/s
+	 4 MiB/s
 done
-Bytes transferred = 6598280 (64ae88 hex)
+Bytes transferred = 6599800 (64b478 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 4 MiB/s
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x64ae88 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x64b478 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -67,7 +67,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-rc4-next-20151210-60419-g8225f4e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Dec 9 22:18:44 PST 2015
+Linux version 4.4.0-rc4-next-20151211-60578-g9379b04 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Dec 10 22:51:56 PST 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -75,20 +75,20 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0ff8880, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c0ffa980, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 65536 pages, LIFO batch:15
-PERCPU: Embedded 12 pages/cpu @ef7cb000 s19136 r8192 d21824 u49152
-pcpu-alloc: s19136 r8192 d21824 u49152 alloc=12*4096
+PERCPU: Embedded 13 pages/cpu @ef7c9000 s23232 r8192 d21824 u53248
+pcpu-alloc: s23232 r8192 d21824 u53248 alloc=13*4096
 pcpu-alloc: [0] 0 [0] 1 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 260608
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 959180K/1048576K available (8330K kernel code, 1090K rwdata, 3840K rodata, 1016K init, 344K bss, 23860K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 959164K/1048576K available (8334K kernel code, 1090K rwdata, 3840K rodata, 1020K init, 344K bss, 23876K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -96,10 +96,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0deb814   (12175 kB)
-      .init : 0xc0dec000 - 0xc0eea000   (1016 kB)
-      .data : 0xc0eea000 - 0xc0ffa800   (1090 kB)
-       .bss : 0xc0ffd000 - 0xc10530a8   ( 345 kB)
+      .text : 0xc0208000 - 0xc0dec824   (12179 kB)
+      .init : 0xc0ded000 - 0xc0eec000   (1020 kB)
+      .data : 0xc0eec000 - 0xc0ffc900   (1091 kB)
+       .bss : 0xc0fff000 - 0xc10550a8   ( 345 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -321,13 +321,13 @@
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
  mmcblk0: p1
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-27 02:47:00 UTC (1419648420)
+rtc-em3027 2-0056: setting system clock to 2014-12-27 02:56:44 UTC (1419649004)
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
@@ -335,10 +335,10 @@
  mmcblk1: p1 p2
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
+devtmpfs: mounted
+Freeing unused kernel memory: 1020K (c0ded000 - c0eec000)
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
-devtmpfs: mounted
-Freeing unused kernel memory: 1016K (c0dec000 - c0eea000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2015-12-10 22:58:23.191265602 -0800
+++ /tmp/foob	2015-12-10 22:58:23.191265602 -0800
@@ -47,7 +47,7 @@
 	 ################################
 	 4 MiB/s
 done
-Bytes transferred = 5230384 (4fcf30 hex)
+Bytes transferred = 5231152 (4fd230 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x4fcf30 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x4fd230 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -69,7 +69,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.4.0-rc4-next-20151210-60419-g8225f4e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Dec 9 22:19:01 PST 2015
+Linux version 4.4.0-rc4-next-20151211-60578-g9379b04 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Dec 10 22:52:12 PST 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -78,20 +78,20 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0a22cc0, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0a24d00, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 65536 pages, LIFO batch:15
-PERCPU: Embedded 12 pages/cpu @ef7ca000 s19456 r8192 d21504 u49152
-pcpu-alloc: s19456 r8192 d21504 u49152 alloc=12*4096
+PERCPU: Embedded 13 pages/cpu @ef7c8000 s23488 r8192 d21568 u53248
+pcpu-alloc: s23488 r8192 d21568 u53248 alloc=13*4096
 pcpu-alloc: [0] 0 [0] 1 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 260608
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 963080K/1048576K available (7179K kernel code, 369K rwdata, 2464K rodata, 336K init, 367K bss, 19960K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 963064K/1048576K available (7179K kernel code, 370K rwdata, 2464K rodata, 344K init, 367K bss, 19976K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -99,10 +99,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09732fc   (9645 kB)
-      .init : 0xc0974000 - 0xc09c8000   ( 336 kB)
-      .data : 0xc09c8000 - 0xc0a247e0   ( 370 kB)
-       .bss : 0xc0a27000 - 0xc0a82d90   ( 368 kB)
+      .text : 0xc0008000 - 0xc09732f4   (9645 kB)
+      .init : 0xc0974000 - 0xc09ca000   ( 344 kB)
+      .data : 0xc09ca000 - 0xc0a26820   ( 371 kB)
+       .bss : 0xc0a29000 - 0xc0a84d90   ( 368 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
@@ -308,14 +308,12 @@
 tlv320aic23-codec 2-001a: ASoC: no dapm match for MICIN --> NULL --> Mic Input
 tlv320aic23-codec 2-001a: ASoC: Failed to add route MICIN -> NULL -> Mic Input
 tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
-mmc0: host does not support reading read-only switch, assuming write-enable
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
-mmc0: new high speed SDHC card at address b368
 NET: Registered protocol family 10
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 mip6: Mobile IPv6
- mmcblk0: p1
 sit: IPv6 over IPv4 tunneling driver
+mmc0: host does not support reading read-only switch, assuming write-enable
+mmc0: new high speed SDHC card at address b368
 NET: Registered protocol family 17
 NET: Registered protocol family 15
 can: controller area network core (rev 20120528 abi 9)
@@ -334,25 +332,64 @@
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
-mmc1: new high speed SD card at address b368
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
+------------[ cut here ]------------
+WARNING: CPU: 1 PID: 74 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
+workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
+Modules linked in:
+CPU: 1 PID: 74 Comm: kworker/u4:2 Not tainted 4.4.0-rc4-next-20151211-60578-g9379b04 #1
+Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
+Workqueue: deferwq deferred_probe_work_func
+[<c0017248>] (unwind_backtrace) from [<c0012870>] (show_stack+0x10/0x14)
+[<c0012870>] (show_stack) from [<c02392e4>] (dump_stack+0x84/0xd0)
+[<c02392e4>] (dump_stack) from [<c0026b60>] (warn_slowpath_common+0x74/0xb0)
+[<c0026b60>] (warn_slowpath_common) from [<c0026c30>] (warn_slowpath_fmt+0x30/0x40)
+[<c0026c30>] (warn_slowpath_fmt) from [<c00398dc>] (check_flush_dependency+0xc8/0x12c)
+[<c00398dc>] (check_flush_dependency) from [<c00399f8>] (flush_work+0x54/0x11c)
+[<c00399f8>] (flush_work) from [<c00c008c>] (lru_add_drain_all+0x124/0x180)
+[<c00c008c>] (lru_add_drain_all) from [<c00f0ae0>] (migrate_prep+0x8/0x10)
+[<c00f0ae0>] (migrate_prep) from [<c00bac04>] (alloc_contig_range+0xc4/0x2f8)
+[<c00bac04>] (alloc_contig_range) from [<c00f2650>] (cma_alloc+0xd8/0x1a0)
+[<c00f2650>] (cma_alloc) from [<c001c250>] (__alloc_from_contiguous+0x38/0xdc)
+[<c001c250>] (__alloc_from_contiguous) from [<c001cf18>] (__dma_alloc+0x228/0x2ac)
+[<c001cf18>] (__dma_alloc) from [<c001d020>] (arm_dma_alloc+0x48/0x54)
+[<c001d020>] (arm_dma_alloc) from [<c00eceec>] (dma_pool_alloc+0x28c/0x304)
+[<c00eceec>] (dma_pool_alloc) from [<c03f7768>] (ehci_qh_alloc+0x3c/0xb4)
+[<c03f7768>] (ehci_qh_alloc) from [<c03fb954>] (ehci_setup+0x154/0x478)
+[<c03fb954>] (ehci_setup) from [<c03ff69c>] (tegra_ehci_reset+0xc/0x50)
+[<c03ff69c>] (tegra_ehci_reset) from [<c03e3154>] (usb_add_hcd+0x2fc/0x804)
+[<c03e3154>] (usb_add_hcd) from [<c03ff014>] (tegra_ehci_probe+0x3b8/0x504)
+[<c03ff014>] (tegra_ehci_probe) from [<c03322b0>] (platform_drv_probe+0x50/0xac)
+[<c03322b0>] (platform_drv_probe) from [<c0330d10>] (driver_probe_device+0x1d4/0x290)
+[<c0330d10>] (driver_probe_device) from [<c032f30c>] (bus_for_each_drv+0x44/0x8c)
+[<c032f30c>] (bus_for_each_drv) from [<c0330a8c>] (__device_attach+0xa0/0x104)
+[<c0330a8c>] (__device_attach) from [<c0330188>] (bus_probe_device+0x84/0x8c)
+[<c0330188>] (bus_probe_device) from [<c03305bc>] (deferred_probe_work_func+0x60/0x8c)
+[<c03305bc>] (deferred_probe_work_func) from [<c003a804>] (process_one_work+0x120/0x328)
+[<c003a804>] (process_one_work) from [<c003aa70>] (worker_thread+0x2c/0x4b4)
+[<c003aa70>] (worker_thread) from [<c003fbac>] (kthread+0xd8/0xf4)
+[<c003fbac>] (kthread) from [<c000f938>] (ret_from_fork+0x14/0x3c)
+---[ end trace b8f898c23980c49f ]---
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
+ mmcblk0: p1
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
- mmcblk1: p1 p2
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-27 02:47:19 UTC (1419648439)
+rtc-em3027 2-0056: setting system clock to 2014-12-27 02:57:04 UTC (1419649024)
 ALSA device list:
   #0: tegra-trimslice
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
+mmc1: new high speed SD card at address b368
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
+ mmcblk1: p1 p2
 devtmpfs: mounted
-Freeing unused kernel memory: 336K (c0974000 - c09c8000)
+Freeing unused kernel memory: 344K (c0974000 - c09ca000)
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-12-10 22:58:23.199265596 -0800
+++ /tmp/foob	2015-12-10 22:58:23.199265596 -0800
@@ -40,18 +40,18 @@
 	 ############################################################
 	 3.4 MiB/s
 done
-Bytes transferred = 6598280 (64ae88 hex)
+Bytes transferred = 6599800 (64b478 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.5 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x64ae88 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x64b478 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-rc4-next-20151210-60419-g8225f4e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Dec 9 22:18:44 PST 2015
+Linux version 4.4.0-rc4-next-20151211-60578-g9379b04 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Dec 10 22:51:56 PST 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -67,20 +67,20 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0ff8880, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c0ffa980, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-PERCPU: Embedded 12 pages/cpu @eef93000 s19136 r8192 d21824 u49152
-pcpu-alloc: s19136 r8192 d21824 u49152 alloc=12*4096
+PERCPU: Embedded 13 pages/cpu @eef8f000 s23232 r8192 d21824 u53248
+pcpu-alloc: s23232 r8192 d21824 u53248 alloc=13*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998292K/2096128K available (8330K kernel code, 1090K rwdata, 3840K rodata, 1016K init, 344K bss, 32300K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998268K/2096128K available (8334K kernel code, 1090K rwdata, 3840K rodata, 1020K init, 344K bss, 32324K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,10 +88,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0deb814   (12175 kB)
-      .init : 0xc0dec000 - 0xc0eea000   (1016 kB)
-      .data : 0xc0eea000 - 0xc0ffa800   (1090 kB)
-       .bss : 0xc0ffd000 - 0xc10530a8   ( 345 kB)
+      .text : 0xc0208000 - 0xc0dec824   (12179 kB)
+      .init : 0xc0ded000 - 0xc0eec000   (1020 kB)
+      .data : 0xc0eec000 - 0xc0ffc900   (1091 kB)
+       .bss : 0xc0fff000 - 0xc10550a8   ( 345 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -281,11 +281,11 @@
 mmcblk0boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk0rpmb: mmc1:0001 SEM16G partition 3 128 KiB
 Primary GPT is invalid, using alternate GPT.
- mmcblk0: p1 p2 p3 p4 p5
 tegra-pcie 3000.pcie-controller: 2x3 configuration
 sys_3v3,vdd_3v3_alw: supplied by vdd_5v_in
 sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
 tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
+ mmcblk0: p1 p2 p3 p4 p5
 tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -339,42 +339,42 @@
 tegra-ehci 7d004000.usb: EHCI Host Controller
 tegra-ehci 7d004000.usb: new USB bus registered, assigned bus number 1
 ------------[ cut here ]------------
-WARNING: CPU: 0 PID: 64 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
+WARNING: CPU: 0 PID: 63 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 0 PID: 64 Comm: kworker/u8:1 Not tainted 4.4.0-rc4-next-20151210-60419-g8225f4e #1
+CPU: 0 PID: 63 Comm: kworker/u8:1 Not tainted 4.4.0-rc4-next-20151211-60578-g9379b04 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
-[<c0218870>] (unwind_backtrace) from [<c0213f10>] (show_stack+0x10/0x14)
-[<c0213f10>] (show_stack) from [<c0485e2c>] (dump_stack+0x78/0x94)
-[<c0485e2c>] (dump_stack) from [<c024ab28>] (warn_slowpath_common+0x74/0xb0)
+[<c0218878>] (unwind_backtrace) from [<c0213f10>] (show_stack+0x10/0x14)
+[<c0213f10>] (show_stack) from [<c04863ac>] (dump_stack+0x78/0x94)
+[<c04863ac>] (dump_stack) from [<c024ab28>] (warn_slowpath_common+0x74/0xb0)
 [<c024ab28>] (warn_slowpath_common) from [<c024abf8>] (warn_slowpath_fmt+0x30/0x40)
 [<c024abf8>] (warn_slowpath_fmt) from [<c025ddd4>] (check_flush_dependency+0xc8/0x124)
 [<c025ddd4>] (check_flush_dependency) from [<c025dee8>] (flush_work+0x54/0x144)
-[<c025dee8>] (flush_work) from [<c02d9e0c>] (lru_add_drain_all+0x124/0x180)
-[<c02d9e0c>] (lru_add_drain_all) from [<c030b50c>] (migrate_prep+0x8/0x10)
-[<c030b50c>] (migrate_prep) from [<c02d4e24>] (alloc_contig_range+0xc4/0x2f8)
-[<c02d4e24>] (alloc_contig_range) from [<c030ce38>] (cma_alloc+0xd8/0x1a0)
-[<c030ce38>] (cma_alloc) from [<c021e31c>] (__alloc_from_contiguous+0x38/0xdc)
+[<c025dee8>] (flush_work) from [<c02da1c8>] (lru_add_drain_all+0x124/0x180)
+[<c02da1c8>] (lru_add_drain_all) from [<c030b928>] (migrate_prep+0x8/0x10)
+[<c030b928>] (migrate_prep) from [<c02d51e0>] (alloc_contig_range+0xc4/0x2f8)
+[<c02d51e0>] (alloc_contig_range) from [<c030d254>] (cma_alloc+0xd8/0x1a0)
+[<c030d254>] (cma_alloc) from [<c021e31c>] (__alloc_from_contiguous+0x38/0xdc)
 [<c021e31c>] (__alloc_from_contiguous) from [<c021ff10>] (__dma_alloc+0x21c/0x250)
 [<c021ff10>] (__dma_alloc) from [<c021ffc8>] (arm_dma_alloc+0x48/0x54)
-[<c021ffc8>] (arm_dma_alloc) from [<c0305a5c>] (dma_pool_alloc+0x1dc/0x210)
-[<c0305a5c>] (dma_pool_alloc) from [<c07a3798>] (ehci_qh_alloc+0x3c/0xb4)
-[<c07a3798>] (ehci_qh_alloc) from [<c07a7d90>] (ehci_setup+0x154/0x478)
-[<c07a7d90>] (ehci_setup) from [<c07ade48>] (tegra_ehci_reset+0xc/0x50)
-[<c07ade48>] (tegra_ehci_reset) from [<c077e480>] (usb_add_hcd+0x300/0x830)
-[<c077e480>] (usb_add_hcd) from [<c07ad7c0>] (tegra_ehci_probe+0x3b8/0x504)
-[<c07ad7c0>] (tegra_ehci_probe) from [<c064474c>] (platform_drv_probe+0x50/0xac)
-[<c064474c>] (platform_drv_probe) from [<c06430b0>] (driver_probe_device+0x1d4/0x290)
-[<c06430b0>] (driver_probe_device) from [<c06416a8>] (bus_for_each_drv+0x44/0x8c)
-[<c06416a8>] (bus_for_each_drv) from [<c0642e2c>] (__device_attach+0xa0/0x104)
-[<c0642e2c>] (__device_attach) from [<c0642528>] (bus_probe_device+0x84/0x8c)
-[<c0642528>] (bus_probe_device) from [<c064295c>] (deferred_probe_work_func+0x60/0x8c)
-[<c064295c>] (deferred_probe_work_func) from [<c025ed30>] (process_one_work+0x134/0x338)
+[<c021ffc8>] (arm_dma_alloc) from [<c0305e78>] (dma_pool_alloc+0x1dc/0x210)
+[<c0305e78>] (dma_pool_alloc) from [<c07a3c30>] (ehci_qh_alloc+0x3c/0xb4)
+[<c07a3c30>] (ehci_qh_alloc) from [<c07a8228>] (ehci_setup+0x154/0x478)
+[<c07a8228>] (ehci_setup) from [<c07ae2e0>] (tegra_ehci_reset+0xc/0x50)
+[<c07ae2e0>] (tegra_ehci_reset) from [<c077e918>] (usb_add_hcd+0x300/0x830)
+[<c077e918>] (usb_add_hcd) from [<c07adc58>] (tegra_ehci_probe+0x3b8/0x504)
+[<c07adc58>] (tegra_ehci_probe) from [<c06446f8>] (platform_drv_probe+0x50/0xac)
+[<c06446f8>] (platform_drv_probe) from [<c064305c>] (driver_probe_device+0x1d4/0x290)
+[<c064305c>] (driver_probe_device) from [<c0641654>] (bus_for_each_drv+0x44/0x8c)
+[<c0641654>] (bus_for_each_drv) from [<c0642dd8>] (__device_attach+0xa0/0x104)
+[<c0642dd8>] (__device_attach) from [<c06424d4>] (bus_probe_device+0x84/0x8c)
+[<c06424d4>] (bus_probe_device) from [<c0642908>] (deferred_probe_work_func+0x60/0x8c)
+[<c0642908>] (deferred_probe_work_func) from [<c025ed30>] (process_one_work+0x134/0x338)
 [<c025ed30>] (process_one_work) from [<c025ef98>] (worker_thread+0x2c/0x4f0)
 [<c025ef98>] (worker_thread) from [<c0264128>] (kthread+0xd8/0xf4)
 [<c0264128>] (kthread) from [<c0210df8>] (ret_from_fork+0x14/0x3c)
----[ end trace 56fcd2ef13f1081d ]---
+---[ end trace c4022478d9a0d461 ]---
 tegra-ehci 7d004000.usb: irq 343, io mem 0x7d004000
 tegra-ehci 7d004000.usb: USB 2.0 started, EHCI 1.00
 hub 1-0:1.0: USB hub found
@@ -394,7 +394,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1016K (c0dec000 - c0eea000)
+Freeing unused kernel memory: 1020K (c0ded000 - c0eec000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.008490] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.978480] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2015-12-10 22:58:23.203265592 -0800
+++ /tmp/foob	2015-12-10 22:58:23.207265590 -0800
@@ -39,7 +39,7 @@
 	 ################################
 	 3.4 MiB/s
 done
-Bytes transferred = 5230384 (4fcf30 hex)
+Bytes transferred = 5231152 (4fd230 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4fcf30 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4fd230 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.4.0-rc4-next-20151210-60419-g8225f4e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Dec 9 22:19:01 PST 2015
+Linux version 4.4.0-rc4-next-20151211-60578-g9379b04 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Dec 10 22:52:12 PST 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -70,20 +70,20 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0a22cc0, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0a24d00, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-PERCPU: Embedded 12 pages/cpu @eef92000 s19456 r8192 d21504 u49152
-pcpu-alloc: s19456 r8192 d21504 u49152 alloc=12*4096
+PERCPU: Embedded 13 pages/cpu @eef8e000 s23488 r8192 d21568 u53248
+pcpu-alloc: s23488 r8192 d21568 u53248 alloc=13*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002192K/2096128K available (7179K kernel code, 369K rwdata, 2464K rodata, 336K init, 367K bss, 28400K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2002168K/2096128K available (7179K kernel code, 370K rwdata, 2464K rodata, 344K init, 367K bss, 28424K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -91,10 +91,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09732fc   (9645 kB)
-      .init : 0xc0974000 - 0xc09c8000   ( 336 kB)
-      .data : 0xc09c8000 - 0xc0a247e0   ( 370 kB)
-       .bss : 0xc0a27000 - 0xc0a82d90   ( 368 kB)
+      .text : 0xc0008000 - 0xc09732f4   (9645 kB)
+      .init : 0xc0974000 - 0xc09ca000   ( 344 kB)
+      .data : 0xc09ca000 - 0xc0a26820   ( 371 kB)
+       .bss : 0xc0a29000 - 0xc0a84d90   ( 368 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -346,39 +346,39 @@
 WARNING: CPU: 0 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-rc4-next-20151210-60419-g8225f4e #1
+CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-rc4-next-20151211-60578-g9379b04 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
-[<c0017240>] (unwind_backtrace) from [<c0012870>] (show_stack+0x10/0x14)
-[<c0012870>] (show_stack) from [<c0238d84>] (dump_stack+0x84/0xd0)
-[<c0238d84>] (dump_stack) from [<c0026b60>] (warn_slowpath_common+0x74/0xb0)
+[<c0017248>] (unwind_backtrace) from [<c0012870>] (show_stack+0x10/0x14)
+[<c0012870>] (show_stack) from [<c02392e4>] (dump_stack+0x84/0xd0)
+[<c02392e4>] (dump_stack) from [<c0026b60>] (warn_slowpath_common+0x74/0xb0)
 [<c0026b60>] (warn_slowpath_common) from [<c0026c30>] (warn_slowpath_fmt+0x30/0x40)
 [<c0026c30>] (warn_slowpath_fmt) from [<c00398dc>] (check_flush_dependency+0xc8/0x12c)
 [<c00398dc>] (check_flush_dependency) from [<c00399f8>] (flush_work+0x54/0x11c)
-[<c00399f8>] (flush_work) from [<c00bfce4>] (lru_add_drain_all+0x124/0x180)
-[<c00bfce4>] (lru_add_drain_all) from [<c00f06d8>] (migrate_prep+0x8/0x10)
-[<c00f06d8>] (migrate_prep) from [<c00ba85c>] (alloc_contig_range+0xc4/0x2f8)
-[<c00ba85c>] (alloc_contig_range) from [<c00f2248>] (cma_alloc+0xd8/0x1a0)
-[<c00f2248>] (cma_alloc) from [<c001c250>] (__alloc_from_contiguous+0x38/0xdc)
+[<c00399f8>] (flush_work) from [<c00c008c>] (lru_add_drain_all+0x124/0x180)
+[<c00c008c>] (lru_add_drain_all) from [<c00f0ae0>] (migrate_prep+0x8/0x10)
+[<c00f0ae0>] (migrate_prep) from [<c00bac04>] (alloc_contig_range+0xc4/0x2f8)
+[<c00bac04>] (alloc_contig_range) from [<c00f2650>] (cma_alloc+0xd8/0x1a0)
+[<c00f2650>] (cma_alloc) from [<c001c250>] (__alloc_from_contiguous+0x38/0xdc)
 [<c001c250>] (__alloc_from_contiguous) from [<c001cf18>] (__dma_alloc+0x228/0x2ac)
 [<c001cf18>] (__dma_alloc) from [<c001d020>] (arm_dma_alloc+0x48/0x54)
-[<c001d020>] (arm_dma_alloc) from [<c00ecae4>] (dma_pool_alloc+0x28c/0x304)
-[<c00ecae4>] (dma_pool_alloc) from [<c03f74ec>] (ehci_qh_alloc+0x3c/0xb4)
-[<c03f74ec>] (ehci_qh_alloc) from [<c03fb6d8>] (ehci_setup+0x154/0x478)
-[<c03fb6d8>] (ehci_setup) from [<c03ff420>] (tegra_ehci_reset+0xc/0x50)
-[<c03ff420>] (tegra_ehci_reset) from [<c03e2ed8>] (usb_add_hcd+0x2fc/0x804)
-[<c03e2ed8>] (usb_add_hcd) from [<c03fed98>] (tegra_ehci_probe+0x3b8/0x504)
-[<c03fed98>] (tegra_ehci_probe) from [<c03323c8>] (platform_drv_probe+0x50/0xac)
-[<c03323c8>] (platform_drv_probe) from [<c0330e28>] (driver_probe_device+0x1d4/0x290)
-[<c0330e28>] (driver_probe_device) from [<c032f424>] (bus_for_each_drv+0x44/0x8c)
-[<c032f424>] (bus_for_each_drv) from [<c0330ba4>] (__device_attach+0xa0/0x104)
-[<c0330ba4>] (__device_attach) from [<c03302a0>] (bus_probe_device+0x84/0x8c)
-[<c03302a0>] (bus_probe_device) from [<c03306d4>] (deferred_probe_work_func+0x60/0x8c)
-[<c03306d4>] (deferred_probe_work_func) from [<c003a804>] (process_one_work+0x120/0x328)
+[<c001d020>] (arm_dma_alloc) from [<c00eceec>] (dma_pool_alloc+0x28c/0x304)
+[<c00eceec>] (dma_pool_alloc) from [<c03f7768>] (ehci_qh_alloc+0x3c/0xb4)
+[<c03f7768>] (ehci_qh_alloc) from [<c03fb954>] (ehci_setup+0x154/0x478)
+[<c03fb954>] (ehci_setup) from [<c03ff69c>] (tegra_ehci_reset+0xc/0x50)
+[<c03ff69c>] (tegra_ehci_reset) from [<c03e3154>] (usb_add_hcd+0x2fc/0x804)
+[<c03e3154>] (usb_add_hcd) from [<c03ff014>] (tegra_ehci_probe+0x3b8/0x504)
+[<c03ff014>] (tegra_ehci_probe) from [<c03322b0>] (platform_drv_probe+0x50/0xac)
+[<c03322b0>] (platform_drv_probe) from [<c0330d10>] (driver_probe_device+0x1d4/0x290)
+[<c0330d10>] (driver_probe_device) from [<c032f30c>] (bus_for_each_drv+0x44/0x8c)
+[<c032f30c>] (bus_for_each_drv) from [<c0330a8c>] (__device_attach+0xa0/0x104)
+[<c0330a8c>] (__device_attach) from [<c0330188>] (bus_probe_device+0x84/0x8c)
+[<c0330188>] (bus_probe_device) from [<c03305bc>] (deferred_probe_work_func+0x60/0x8c)
+[<c03305bc>] (deferred_probe_work_func) from [<c003a804>] (process_one_work+0x120/0x328)
 [<c003a804>] (process_one_work) from [<c003aa70>] (worker_thread+0x2c/0x4b4)
 [<c003aa70>] (worker_thread) from [<c003fbac>] (kthread+0xd8/0xf4)
 [<c003fbac>] (kthread) from [<c000f938>] (ret_from_fork+0x14/0x3c)
----[ end trace c471d447807b5a19 ]---
+---[ end trace 4438c1d815908d44 ]---
 tegra-ehci 7d004000.usb: irq 343, io mem 0x7d004000
 tegra-ehci 7d004000.usb: USB 2.0 started, EHCI 1.00
 hub 1-0:1.0: USB hub found
@@ -400,7 +400,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 336K (c0974000 - c09c8000)
+Freeing unused kernel memory: 344K (c0974000 - c09ca000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.430567] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.400555] usb 2-1: new high-speed USB device number 2 using tegra-ehci
