Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2017-11-19 20:19:36.653422094 -0800
+++ /tmp/foob	2017-11-19 20:19:36.657422111 -0800
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0000000000 [0x411fd070]
-Linux version 4.14.0-next-20171117-333860-gee92513eae01 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Fri Nov 17 02:22:41 PST 2017
+Linux version 4.14.0-next-20171120-333971-ge6016f98372b (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sun Nov 19 20:11:05 PST 2017
 Machine model: linux,dummy-virt
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -182,7 +182,7 @@
 registered taskstats version 1
 Loading compiled-in X.509 certificates
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2017-11-17 10:29:10 UTC (1510914550)
+rtc-pl031 9010000.pl031: setting system clock to 2017-11-20 04:18:15 UTC (1511151495)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-11-19 20:19:36.677422203 -0800
+++ /tmp/foob	2017-11-19 20:19:36.677422203 -0800
@@ -40,7 +40,7 @@
 	 #################################################################
 	 #################################################################
 	 #############################
-	 2.8 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 8057344 (7af200 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
@@ -49,7 +49,7 @@
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 126 KiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 34502 (86c6 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-next-20171117-333860-gee92513eae01 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Fri Nov 17 02:16:35 PST 2017
+Linux version 4.14.0-next-20171120-333971-ge6016f98372b (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Sun Nov 19 20:05:02 PST 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -94,8 +94,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0e00000   (12256 kB)
       .init : 0xc1300000 - 0xc1500000   (2048 kB)
-      .data : 0xc1500000 - 0xc1690800   (1602 kB)
-       .bss : 0xc1690800 - 0xc16f5bb4   ( 405 kB)
+      .data : 0xc1500000 - 0xc16908e0   (1603 kB)
+       .bss : 0xc16908e0 - 0xc16f5cb4   ( 405 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -372,11 +372,11 @@
 mmc0: new high speed SDHC card at address aaaa
 usb1_vbus: supplied by vdd-sys-5v0
 usb2_vbus: supplied by vdd-sys-5v0
-VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
 mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
+VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
+ mmcblk0: p1 p2 p3
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
- mmcblk0: p1 p2 p3
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
@@ -393,7 +393,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-04-09 17:17:44 UTC (955300664)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-04-12 11:06:12 UTC (955537572)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -405,5 +405,5 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.939263] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.929296] usb 1-1: new high-speed USB device number 2 using tegra-ehci
 asix 1-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0d:b5:16
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2017-11-19 20:19:36.717422382 -0800
+++ /tmp/foob	2017-11-19 20:19:36.717422382 -0800
@@ -37,21 +37,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##############################
-	 3.3 MiB/s
+	 ###############################
+	 3.2 MiB/s
 done
-Bytes transferred = 6164560 (5e1050 hex)
+Bytes transferred = 6166776 (5e18f8 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 34502 (86c6 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5e1050 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5e18f8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff6c5 ... OK
@@ -60,7 +60,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-next-20171117-13911-gee92513 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Fri Nov 17 02:15:48 PST 2017
+Linux version 4.14.0-next-20171120-14022-ge6016f9 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Sun Nov 19 20:03:22 PST 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -82,7 +82,7 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=7369c667-ff51-ec4a-29cd-baabf2fbe346 rootwait gpt debug earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998600K/2096128K available (9216K kernel code, 777K rwdata, 2556K rodata, 1024K init, 299K bss, 31992K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998600K/2096128K available (9216K kernel code, 778K rwdata, 2560K rodata, 1024K init, 299K bss, 31992K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,8 +92,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0ec2728   ( 778 kB)
-       .bss : 0xc0ec2728 - 0xc0f0d3b4   ( 300 kB)
+      .data : 0xc0e00000 - 0xc0ec2820   ( 779 kB)
+       .bss : 0xc0ec2820 - 0xc0f0d4b4   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -389,7 +389,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-04-09 17:15:06 UTC (955300506)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-04-12 11:03:34 UTC (955537414)
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
@@ -407,4 +407,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.171438] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.171495] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-11-19 20:19:36.725422420 -0800
+++ /tmp/foob	2017-11-19 20:19:36.729422438 -0800
@@ -35,7 +35,7 @@
 	 #################################################################
 	 #################################################################
 	 #############################
-	 4 MiB/s
+	 3.9 MiB/s
 done
 Bytes transferred = 8057344 (7af200 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
@@ -56,7 +56,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-next-20171117-333860-gee92513eae01 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Fri Nov 17 02:16:35 PST 2017
+Linux version 4.14.0-next-20171120-333971-ge6016f98372b (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Sun Nov 19 20:05:02 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -89,8 +89,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0e00000   (12256 kB)
       .init : 0xc1300000 - 0xc1500000   (2048 kB)
-      .data : 0xc1500000 - 0xc1690800   (1602 kB)
-       .bss : 0xc1690800 - 0xc16f5bb4   ( 405 kB)
+      .data : 0xc1500000 - 0xc16908e0   (1603 kB)
+       .bss : 0xc16908e0 - 0xc16f5cb4   ( 405 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -397,6 +397,7 @@
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
 tegra-pcie 1003000.pcie: 2x1, 1x1 configuration
 tegra-pcie 1003000.pcie: Failed to get supply 'avddio-pex': -517
+mmc0: new high speed MMC card at address 0001
 tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
 as3722-sd2: supplied by +5V_SYS
@@ -418,15 +419,9 @@
 +2.8V_RUN_CAM_AF: Bringing 0uV into 2800000-2800000uV
 as3722-ldo11: supplied by +3.3V_RUN
 +1.8V_RUN_VPP_FUSE: Bringing 0uV into 1800000-1800000uV
-mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB, chardev (237:0)
 tegra-ahci 70027000.sata: AHCI 0001.0301 32 slots 2 ports 3 Gbps 0x1 impl platform mode
 tegra-ahci 70027000.sata: flags: 64bit ncq sntf pm led pio slum part sadm sds apst 
 scsi host0: tegra-ahci
- mmcblk0: p1
 scsi host1: tegra-ahci
 ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 345
 ata2: DUMMY
@@ -459,7 +454,6 @@
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x52 may corrupt adjacent RW1C bits
 pci 0000:00:01.0: PME# supported from D0 D1 D2 D3hot D3cold
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x4c may corrupt adjacent RW1C bits
-ata1: SATA link down (SStatus 0 SControl 300)
 pci 0000:00:02.0: [10de:0e13] type 01 class 0x060400
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4 may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4 may corrupt adjacent RW1C bits
@@ -504,23 +498,18 @@
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
 r8169 0000:02:00.0 (unnamed net_device) (uninitialized): not PCI Express
-mmc1: new high speed SDHC card at address aaaa
-mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
-r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0a77000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 22
+r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0a6d000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 22
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
- mmcblk1: p1 p2 p3
-tegra-hdmi 54280000.hdmi: cannot set audio to 48000 Hz at 297000000 Hz pixel clock
-Console: switching to colour frame buffer device 512x135
-drm drm: fb0:  frame buffer device
+[drm] Cannot find any crtc or sizes
 [drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
 iommu: Adding device cpufreq-dt.0 to group 74
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:26 UTC (946684946)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:41 UTC (946684961)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
@@ -529,12 +518,24 @@
 +3.3V_RUN_TOUCH: disabling
 +2.8V_RUN_CAM_AF: disabling
 +1.8V_RUN_VPP_FUSE: disabling
+mmc1: new high speed SDHC card at address aaaa
+mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
+ata1: SATA link down (SStatus 0 SControl 300)
+ mmcblk1: p1 p2 p3
+tegra-hdmi 54280000.hdmi: cannot set audio to 48000 Hz at 297000000 Hz pixel clock
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB, chardev (237:0)
+ mmcblk0: p1
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:1.
+VFS: Mounted root (ext3 filesystem) on device 179:17.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K
+Console: switching to colour frame buffer device 512x135
+drm drm: fb0:  frame buffer device
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2017-11-19 20:19:36.737422473 -0800
+++ /tmp/foob	2017-11-19 20:19:36.741422491 -0800
@@ -32,10 +32,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##############################
-	 3.9 MiB/s
+	 ###############################
+	 4 MiB/s
 done
-Bytes transferred = 6164560 (5e1050 hex)
+Bytes transferred = 6166776 (5e18f8 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -46,7 +46,7 @@
 done
 Bytes transferred = 66544 (103f0 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5e1050 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5e18f8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830133ef
@@ -55,7 +55,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-next-20171117-13911-gee92513 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Fri Nov 17 02:15:48 PST 2017
+Linux version 4.14.0-next-20171120-14022-ge6016f9 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Sun Nov 19 20:03:22 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -77,7 +77,7 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=b1791faa-84e8-4e27-83a1-9fb0a77e8803 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998492K/2096128K available (9216K kernel code, 777K rwdata, 2556K rodata, 1024K init, 299K bss, 32100K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998492K/2096128K available (9216K kernel code, 778K rwdata, 2560K rodata, 1024K init, 299K bss, 32100K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -87,8 +87,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0ec2728   ( 778 kB)
-       .bss : 0xc0ec2728 - 0xc0f0d3b4   ( 300 kB)
+      .data : 0xc0e00000 - 0xc0ec2820   ( 779 kB)
+       .bss : 0xc0ec2820 - 0xc0f0d4b4   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -413,8 +413,8 @@
 as3722-ldo5: supplied by +1.8V_VDDIO
 as3722-ldo6: supplied by +3.3V_RUN
 as3722-ldo7: supplied by +1.8V_VDDIO
-+1.05V_RUN_CAM_REAR: Bringing 0uV into 1050000-1050000uV
 input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input1
++1.05V_RUN_CAM_REAR: Bringing 0uV into 1050000-1050000uV
 as3722-ldo9: supplied by +5V_SYS
 +3.3V_RUN_TOUCH: Bringing 0uV into 2800000-2800000uV
 as3722-ldo10: supplied by +5V_SYS
@@ -478,13 +478,13 @@
 pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
 pci 0000:02:00.0: [10ec:8168] type 00 class 0x020000
 pci 0000:02:00.0: reg 0x10: [io  0x0000-0x00ff]
+mmc1: new high speed SDHC card at address aaaa
+mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 pci 0000:02:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit]
 pci 0000:02:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
 PCI: bus2: Fast back to back transfers disabled
 pci_bus 0000:02: busn_res: [bus 02-ff] end is updated to 02
 pci 0000:00:01.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
-mmc1: new high speed SDHC card at address aaaa
-mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 pci 0000:00:02.0: BAR 8: assigned [mem 0x13100000-0x131fffff]
 pci 0000:00:02.0: BAR 9: assigned [mem 0x20000000-0x200fffff 64bit pref]
 pci 0000:00:02.0: BAR 7: assigned [io  0x1000-0x1fff]
@@ -499,10 +499,10 @@
 pci 0000:00:02.0:   bridge window [mem 0x13100000-0x131fffff]
 pci 0000:00:02.0:   bridge window [mem 0x20000000-0x200fffff 64bit pref]
 pci 0000:00:01.0: nv_msi_ht_cap_quirk didn't locate host bridge
+ mmcblk1: p1 p2 p3
 pcieport 0000:00:01.0: enabling device (0140 -> 0142)
 pcieport 0000:00:01.0: Signaling PME with IRQ 395
 pci 0000:00:02.0: nv_msi_ht_cap_quirk didn't locate host bridge
- mmcblk1: p1 p2 p3
 pcieport 0000:00:02.0: enabling device (0140 -> 0143)
 pcieport 0000:00:02.0: Signaling PME with IRQ 396
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig%tegra-fw_log.txt
--- /tmp/fooa	2017-11-19 20:19:36.765422600 -0800
+++ /tmp/foob	2017-11-19 20:19:36.769422618 -0800
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MMC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -32,21 +32,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##########################################
-	 3.5 MiB/s
+	 ##########################################T T T 
+	 368.2 KiB/s
 done
-Bytes transferred = 6329816 (6095d8 hex)
+Bytes transferred = 6331560 (609ca8 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.3 MiB/s
+	 3.5 MiB/s
 done
 Bytes transferred = 66544 (103f0 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6095d8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x609ca8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830133ef
@@ -55,7 +55,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-next-20171117-215619-gee92513 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Fri Nov 17 02:16:04 PST 2017
+Linux version 4.14.0-next-20171120-215730-ge6016f9 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Sun Nov 19 20:03:46 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -77,7 +77,7 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=b1791faa-84e8-4e27-83a1-9fb0a77e8803 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1997468K/2096128K available (9216K kernel code, 777K rwdata, 2820K rodata, 1024K init, 299K bss, 33124K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1997468K/2096128K available (9216K kernel code, 778K rwdata, 2820K rodata, 1024K init, 299K bss, 33124K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -87,8 +87,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
-      .data : 0xc0f00000 - 0xc0fc2728   ( 778 kB)
-       .bss : 0xc0fc2728 - 0xc100d3b4   ( 300 kB)
+      .data : 0xc0f00000 - 0xc0fc2820   ( 779 kB)
+       .bss : 0xc0fc2820 - 0xc100d4b4   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -529,10 +529,11 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:15 UTC (946684815)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:30 UTC (946684830)
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
+cfg80211: failed to load regulatory.db
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
@@ -544,7 +545,6 @@
 ALSA device list:
   #0: tegra-hda at 0x70038000 irq 346
   #1: NVIDIA Tegra Jetson TK1
-cfg80211: failed to load regulatory.db
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-11-19 20:19:36.781422673 -0800
+++ /tmp/foob	2017-11-19 20:19:36.785422690 -0800
@@ -43,7 +43,7 @@
 	 #################################################################
 	 #################################################################
 	 #############################
-	 2.8 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 8057344 (7af200 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
@@ -52,7 +52,7 @@
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 2.7 MiB/s
+	 44.9 KiB/s
 done
 Bytes transferred = 67482 (1079a hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
@@ -64,7 +64,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-next-20171117-333860-gee92513eae01 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Fri Nov 17 02:16:35 PST 2017
+Linux version 4.14.0-next-20171120-333971-ge6016f98372b (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Sun Nov 19 20:05:02 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -98,8 +98,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0e00000   (12256 kB)
       .init : 0xc1300000 - 0xc1500000   (2048 kB)
-      .data : 0xc1500000 - 0xc1690800   (1602 kB)
-       .bss : 0xc1690800 - 0xc16f5bb4   ( 405 kB)
+      .data : 0xc1500000 - 0xc16908e0   (1603 kB)
+       .bss : 0xc16908e0 - 0xc16f5cb4   ( 405 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -369,9 +369,9 @@
 mmc0: Unknown controller version (3). You may experience problems.
 mmc0: Invalid maximum block size, assuming 512 bytes
 elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
-input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 mmc0: SDHCI controller on 700b0600.sdhci [700b0600.sdhci] using ADMA
 ledtrig-cpu: registered to indicate activity on CPUs
+input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
 NET: Registered protocol family 10
@@ -380,6 +380,12 @@
 NET: Registered protocol family 17
 can: controller area network core (rev 20170425 abi 9)
 NET: Registered protocol family 29
+mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
+mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
+mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
+mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB, chardev (237:0)
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 can: raw protocol (rev 20170425)
 can: broadcast manager protocol (rev 20170425 t)
 can: netlink gateway (rev 20170425) max_hops=1
@@ -387,7 +393,6 @@
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
 Loading compiled-in X.509 certificates
-mmc0: new high speed MMC card at address 0001
 +3.3V_RUN: supplied by +3.3V_SYS
 +VDD_LED: supplied by +VDD_MUX
 +3.3V_AVDD_HDMI_AP_GATED: supplied by +3.3V_RUN
@@ -403,56 +408,19 @@
 +5V_HDMI_CON: supplied by +5V_SYS
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
  usb3-1: usb3-1 supply vbus not found, using dummy regulator
+elan_i2c 1-0015: failed to read report data: -121
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
 sdhci-tegra 700b0000.sdhci: allocated mmc-pwrseq
 mmc1: Unknown controller version (3). You may experience problems.
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: Switching to 3.3V signalling voltage failed
+elan_i2c 1-0015: invalid report id data (ff)
 mmc1: SDHCI controller on 700b0000.sdhci [700b0000.sdhci] using ADMA
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc2: Unknown controller version (3). You may experience problems.
 mmc2: Invalid maximum block size, assuming 512 bytes
 mmc1: new high speed SDIO card at address 0001
-mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
-mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
-mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
 mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
-mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB, chardev (237:0)
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
-[drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
-[drm] No driver support for vblank timestamp query.
-mmc2: new high speed SDHC card at address 0007
-mmcblk2: mmc2:0007 SD08G 7.42 GiB 
-GPT:Primary header alternate_lba != Alt. header my_lba
-GPT:3014591 != 15564799
-GPT:Primary header thinks Alt. header is not at the end of the disk.
-GPT:3014591 != 15564799
-GPT: Use GNU Parted to correct GPT errors.
- mmcblk2: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
-Console: switching to colour frame buffer device 170x48
-drm drm: fb0:  frame buffer device
-[drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
-input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-as3722-rtc as3722-rtc: setting system clock to 2000-01-17 19:43:00 UTC (948138180)
-+1.8V_RUN_CAM: disabling
-+3.3V_RUN_CAM: disabling
-+1.2V_RUN_CAM_FRONT: disabling
-+1.05V_RUN_CAM_REAR: disabling
-+2.8V_RUN_TOUCH: disabling
-+2.8V_RUN_CAM_AF: disabling
-+1.8V_RUN_VPP_FUSE: disabling
-+5V_VDD_TS_SW: disabling
-+5V_USB_HS: disabling
-+5V_USB_SS: disabling
-EXT4-fs (mmcblk0p3): couldn't mount as ext3 due to feature incompatibilities
-EXT4-fs (mmcblk0p3): couldn't mount as ext2 due to feature incompatibilities
-EXT4-fs (mmcblk0p3): recovery complete
-EXT4-fs (mmcblk0p3): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext4 filesystem) on device 179:3.
-devtmpfs: mounted
-Freeing unused kernel memory: 2048K
-bash: cannot set terminal process group (-1): Inappropriate ioctl for device
-bash: no job control in this shell
-root@(none):/# 
\ No newline at end of file
+[d
\ No newline at end of file
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2017-11-19 20:19:36.797422744 -0800
+++ /tmp/foob	2017-11-19 20:19:36.797422744 -0800
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -34,18 +34,16 @@
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'zImage.tegra_defconfig'.
 Load address: 0x82000000
-Loading: *##########################EHCI timed out on TD - token=0x88008d80
-T EHCI timed out on TD - token=0x88008d80
-T #######################################
+Loading: *#################################################################
 	 #################################################################
 	 #################################################################
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##############################
-	 388.7 KiB/s
+	 ###############################
+	 3.5 MiB/s
 done
-Bytes transferred = 6164560 (5e1050 hex)
+Bytes transferred = 6166776 (5e18f8 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -56,7 +54,7 @@
 done
 Bytes transferred = 67482 (1079a hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5e1050 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5e18f8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013799
@@ -65,7 +63,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-next-20171117-13911-gee92513 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Fri Nov 17 02:15:48 PST 2017
+Linux version 4.14.0-next-20171120-14022-ge6016f9 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Sun Nov 19 20:03:22 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -88,7 +86,7 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=4d36e82a-b3c0-4b4b-bfcd-afe5217ac023 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999512K/2097148K available (9216K kernel code, 777K rwdata, 2556K rodata, 1024K init, 299K bss, 32100K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1999512K/2097148K available (9216K kernel code, 778K rwdata, 2560K rodata, 1024K init, 299K bss, 32100K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -98,8 +96,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0ec2728   ( 778 kB)
-       .bss : 0xc0ec2728 - 0xc0f0d3b4   ( 300 kB)
+      .data : 0xc0e00000 - 0xc0ec2820   ( 779 kB)
+       .bss : 0xc0ec2820 - 0xc0f0d4b4   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -315,130 +313,146 @@
 kernel BUG at drivers/platform/chrome/cros_ec_proto.c:34!
 Internal error: Oops - BUG: 0 [#1] PREEMPT SMP ARM
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.14.0-next-20171117-13911-gee92513 #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.14.0-next-20171120-14022-ge6016f9 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 task: ee890740 task.stack: ee8a6000
 PC is at cros_ec_prepare_tx+0x168/0x16c
 LR is at cros_ec_pkt_xfer_spi+0x18/0x474
-pc : [<c06e7398>]    lr : [<c0555d98>]    psr: 20000013
+pc : [<c06e76b4>]    lr : [<c0556054>]    psr: 20000013
 sp : ee8a7bd8  ip : 00000000  fp : 00000000
-r10: c0555d80  r9 : 00000000  r8 : eefe7fb4
-r7 : ee28db00  r6 : ee28db00  r5 : ee28db00  r4 : ee270e50
+r10: c055603c  r9 : 00000000  r8 : eefe7fb4
+r7 : ee28cac0  r6 : ee28cac0  r5 : ee28cac0  r4 : ee270e50
 r3 : 0000ffff  r2 : 00000000  r1 : 0000ffff  r0 : ee270e50
 Flags: nzCv  IRQs on  FIQs on  Mode SVC_32  ISA ARM  Segment none
 Control: 10c5387d  Table: 8000406a  DAC: 00000051
 Process swapper/0 (pid: 1, stack limit = 0xee8a6210)
 Stack: (0xee8a7bd8 to 0xee8a8000)
-7bc0:                                                       ee270e50 ee28db00
-7be0: ee28db90 c0555d98 c098ba74 00000004 ee8a7c5c ee8a7c5c ee28b800 00000000
-7c00: c05a232c 00000000 00000008 00000008 00000000 ee8a7c14 ee8a7c14 00000000
-7c20: ee8a7c20 ee8a7c20 ee28db50 ee28da80 00000008 00000000 00000000 00000000
-7c40: c0d90bd0 c023b620 40002800 00000560 eef94bd0 c023da84 002dc6c0 ee8a7bf0
-7c60: ee8a7bf0 00000000 c06e71c8 005c51a0 3b9aca00 000f4240 00000000 ee8a7ca8
-7c80: 00000001 00000501 c0555d80 c0990410 00000000 ee28dac0 a0000013 c06e71c8
-7ca0: ee28db00 ee270e50 ee28db00 ee28b800 ee28db00 eefe7fb4 00000000 c0555d80
-7cc0: 00000000 c06e7124 0000400b ee270e50 ee28db00 ee28b800 0000000c eefe7fb4
-7ce0: c0eecdbc c06e7540 00000000 ee270e50 ee28b800 00000000 00000000 eefe7fb4
-7d00: c0e6c308 c0555824 ee8a7d14 ee28b800 00000000 00000001 ee28b800 ee270e50
-7d20: ee28db90 c0555bd4 00000000 00000000 00000000 000007d0 ee28b800 00000000
-7d40: c0e6c2f8 c0efe068 00000000 c05a2d70 ee28b800 c0efe064 00000000 c05315b0
-7d60: 00000000 ee8a7d98 c053175c 00000001 c0efe040 00000000 00000000 c052fbb8
-7d80: ee9f1170 ee2605b8 ee28b800 ee28b834 c0e72e34 c05312d8 ee28b800 00000001
-7da0: 00000000 ee28b800 ee28b800 c0e72e34 ee03a000 c053091c ee28b800 00000000
-7dc0: ee28b808 c052ef60 ee28b800 c05a5cdc 00000000 ee28b800 ee28b800 ee03a000
-7de0: 00000000 eea8be10 eefe8004 00000001 00000000 c05a2bc4 002dc6c0 ee03a000
-7e00: eefe7fb4 00000000 ee28b800 c05a4b64 00000000 014000c0 00000000 c05a4550
-7e20: 00000063 002dc6c0 ee03a000 ee28d650 eea8be00 eea8be10 40000000 00000153
-7e40: 00000000 c05a5088 eefe7d0c ee03a350 ee03a000 eea8be00 eea8be10 c05a5c14
-7e60: 00000000 ee03a350 ffffffed eea8be10 fffffdfb c0e73790 00000000 00000000
-7e80: c0e73790 c0532ec8 c0532e78 eea8be10 c0efe064 00000000 c0efe068 c05315b0
-7ea0: eea8be10 c0e73790 eea8be44 00000000 0000010e c0d6a83c c0d00594 c0531724
-7ec0: 00000000 c0e73790 c0531678 c052fc54 ee8cc55c eea14934 c0e73790 ee287e80
-7ee0: c0e6a8f8 c0530af0 c0be0844 c0e73790 00000006 c0e73790 00000006 c0d3866c
-7f00: 00000000 c05322a4 c0532b1c c0d8332c 00000006 c0101b8c 00000000 c0c7e1c8
-7f20: 00000100 efffcc45 c0c7e1c8 0000010e efffcc53 c013e8f4 c0c7ccb0 00000000
-7f40: 00000006 00000006 efffcc45 efffcc4a 00000000 efffcbc0 00000006 c0d6a82c
-7f60: c0d8332c 00000006 c0d6a830 c0ec2740 0000010e c0d00d94 00000006 00000006
-7f80: 00000000 c0d00594 00000000 c09894a8 00000000 00000000 00000000 00000000
-7fa0: 00000000 c09894b0 00000000 c0107cf0 00000000 00000000 00000000 00000000
+7bc0:                                                       ee270e50 ee28cac0
+7be0: ee28cb50 c0556054 c098c2f4 00000004 ee8a7c5c ee8a7c5c ee28b800 00000000
+7c00: c05a25e4 00000000 00000008 00000008 00000000 ee8a7c14 ee8a7c14 00000000
+7c20: ee8a7c20 ee8a7c20 ee28cb10 ee28ca40 00000008 00000000 00000000 00000000
+7c40: c0d91bd0 c023b654 40002800 00000540 eef94bd0 c023dab8 002dc6c0 ee8a7bf0
+7c60: ee8a7bf0 00000000 c06e74e4 005c5180 3b9aca00 000f4240 00000000 ee8a7ca8
+7c80: 00000001 00000501 c055603c c0990c90 00000000 ee28ca80 a0000013 c06e74e4
+7ca0: ee28cac0 ee270e50 ee28cac0 ee28b800 ee28cac0 eefe7fb4 00000000 c055603c
+7cc0: 00000000 c06e7440 0000400b ee270e50 ee28cac0 ee28b800 0000000c eefe7fb4
+7ce0: c0eecebc c06e785c 00000000 ee270e50 ee28b800 00000000 00000000 eefe7fb4
+7d00: c0e6c3c8 c0555ae0 ee8a7d14 ee28b800 00000000 00000001 ee28b800 ee270e50
+7d20: ee28cb50 c0555e90 00000000 00000000 00000000 000007d0 ee28b800 00000000
+7d40: c0e6c3b8 c0efe168 00000000 c05a3028 ee28b800 c0efe164 00000000 c053186c
+7d60: 00000000 ee8a7d98 c0531a18 00000001 c0efe140 00000000 00000000 c052fe74
+7d80: ee9f1170 ee2605b8 ee28b800 ee28b834 c0e72ef4 c0531594 ee28b800 00000001
+7da0: 00000000 ee28b800 ee28b800 c0e72ef4 ee03a000 c0530bd8 ee28b800 00000000
+7dc0: ee28b808 c052f21c ee28b800 c05a5f94 00000000 ee28b800 ee28b800 ee03a000
+7de0: 00000000 eea8be10 eefe8004 00000001 00000000 c05a2e7c 002dc6c0 ee03a000
+7e00: eefe7fb4 00000000 ee28b800 c05a4e1c 00000000 014000c0 00000000 c05a4808
+7e20: 00000063 002dc6c0 ee03a000 ee28c610 eea8be00 eea8be10 40000000 00000153
+7e40: 00000000 c05a5340 eefe7d0c ee03a350 ee03a000 eea8be00 eea8be10 c05a5ecc
+7e60: 00000000 ee03a350 ffffffed eea8be10 fffffdfb c0e73850 00000000 00000000
+7e80: c0e73850 c0533184 c0533134 eea8be10 c0efe164 00000000 c0efe168 c053186c
+7ea0: eea8be10 c0e73850 eea8be44 00000000 0000010e c0d6b83c c0d00594 c05319e0
+7ec0: 00000000 c0e73850 c0531934 c052ff10 ee8cc55c eea14934 c0e73850 ee287e80
+7ee0: c0e6a9b8 c0530dac c0be0910 c0e73850 00000006 c0e73850 00000006 c0d38678
+7f00: 00000000 c0532560 c0532dd8 c0d8434c 00000006 c0101b8c 00000000 c0c7f21c
+7f20: 00000100 efffcc45 c0c7f21c 0000010e efffcc53 c013e920 c0c7dd04 00000000
+7f40: 00000006 00000006 efffcc45 efffcc4a 00000000 efffcbc0 00000006 c0d6b82c
+7f60: c0d8434c 00000006 c0d6b830 c0ec2840 0000010e c0d00d94 00000006 00000006
+7f80: 00000000 c0d00594 00000000 c0989cc8 00000000 00000000 00000000 00000000
+7fa0: 00000000 c0989cd0 00000000 c0107cf0 00000000 00000000 00000000 00000000
 7fc0: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
 7fe0: 00000000 00000000 00000000 00000000 00000013 00000000 00000000 00000000
-[<c06e7398>] (cros_ec_prepare_tx) from [<c0555d98>] (cros_ec_pkt_xfer_spi+0x18/0x474)
-[<c0555d98>] (cros_ec_pkt_xfer_spi) from [<c06e7124>] (send_command+0x20/0xd8)
-[<c06e7124>] (send_command) from [<c06e7540>] (cros_ec_query_all+0x1a4/0x398)
-[<c06e7540>] (cros_ec_query_all) from [<c0555824>] (cros_ec_register+0x94/0x21c)
-[<c0555824>] (cros_ec_register) from [<c0555bd4>] (cros_ec_spi_probe+0x108/0x150)
-[<c0555bd4>] (cros_ec_spi_probe) from [<c05a2d70>] (spi_drv_probe+0x7c/0xa8)
-[<c05a2d70>] (spi_drv_probe) from [<c05315b0>] (driver_probe_device+0x250/0x318)
-[<c05315b0>] (driver_probe_device) from [<c052fbb8>] (bus_for_each_drv+0x44/0x8c)
-[<c052fbb8>] (bus_for_each_drv) from [<c05312d8>] (__device_attach+0x9c/0x100)
-[<c05312d8>] (__device_attach) from [<c053091c>] (bus_probe_device+0x84/0x8c)
-[<c053091c>] (bus_probe_device) from [<c052ef60>] (device_add+0x380/0x528)
-[<c052ef60>] (device_add) from [<c05a2bc4>] (spi_add_device+0x88/0x130)
-[<c05a2bc4>] (spi_add_device) from [<c05a4b64>] (spi_register_controller+0x428/0x918)
-[<c05a4b64>] (spi_register_controller) from [<c05a5088>] (devm_spi_register_controller+0x34/0x70)
-[<c05a5088>] (devm_spi_register_controller) from [<c05a5c14>] (tegra_spi_probe+0x298/0x35c)
-[<c05a5c14>] (tegra_spi_probe) from [<c0532ec8>] (platform_drv_probe+0x50/0xb0)
-[<c0532ec8>] (platform_drv_probe) from [<c05315b0>] (driver_probe_device+0x250/0x318)
-[<c05315b0>] (driver_probe_device) from [<c0531724>] (__driver_attach+0xac/0xb0)
-[<c0531724>] (__driver_attach) from [<c052fc54>] (bus_for_each_dev+0x54/0x88)
-[<c052fc54>] (bus_for_each_dev) from [<c0530af0>] (bus_add_driver+0xe4/0x1f0)
-[<c0530af0>] (bus_add_driver) from [<c05322a4>] (driver_register+0x78/0xf4)
-[<c05322a4>] (driver_register) from [<c0101b8c>] (do_one_initcall+0x3c/0x168)
+[<c06e76b4>] (cros_ec_prepare_tx) from [<c0556054>] (cros_ec_pkt_xfer_spi+0x18/0x474)
+[<c0556054>] (cros_ec_pkt_xfer_spi) from [<c06e7440>] (send_command+0x20/0xd8)
+[<c06e7440>] (send_command) from [<c06e785c>] (cros_ec_query_all+0x1a4/0x398)
+[<c06e785c>] (cros_ec_query_all) from [<c0555ae0>] (cros_ec_register+0x94/0x21c)
+[<c0555ae0>] (cros_ec_register) from [<c0555e90>] (cros_ec_spi_probe+0x108/0x150)
+[<c0555e90>] (cros_ec_spi_probe) from [<c05a3028>] (spi_drv_probe+0x7c/0xa8)
+[<c05a3028>] (spi_drv_probe) from [<c053186c>] (driver_probe_device+0x250/0x318)
+[<c053186c>] (driver_probe_device) from [<c052fe74>] (bus_for_each_drv+0x44/0x8c)
+[<c052fe74>] (bus_for_each_drv) from [<c0531594>] (__device_attach+0x9c/0x100)
+[<c0531594>] (__device_attach) from [<c0530bd8>] (bus_probe_device+0x84/0x8c)
+[<c0530bd8>] (bus_probe_device) from [<c052f21c>] (device_add+0x380/0x528)
+[<c052f21c>] (device_add) from [<c05a2e7c>] (spi_add_device+0x88/0x130)
+[<c05a2e7c>] (spi_add_device) from [<c05a4e1c>] (spi_register_controller+0x428/0x918)
+[<c05a4e1c>] (spi_register_controller) from [<c05a5340>] (devm_spi_register_controller+0x34/0x70)
+[<c05a5340>] (devm_spi_register_controller) from [<c05a5ecc>] (tegra_spi_probe+0x298/0x35c)
+[<c05a5ecc>] (tegra_spi_probe) from [<c0533184>] (platform_drv_probe+0x50/0xb0)
+[<c0533184>] (platform_drv_probe) from [<c053186c>] (driver_probe_device+0x250/0x318)
+[<c053186c>] (driver_probe_device) from [<c05319e0>] (__driver_attach+0xac/0xb0)
+[<c05319e0>] (__driver_attach) from [<c052ff10>] (bus_for_each_dev+0x54/0x88)
+[<c052ff10>] (bus_for_each_dev) from [<c0530dac>] (bus_add_driver+0xe4/0x1f0)
+[<c0530dac>] (bus_add_driver) from [<c0532560>] (driver_register+0x78/0xf4)
+[<c0532560>] (driver_register) from [<c0101b8c>] (do_one_initcall+0x3c/0x168)
 [<c0101b8c>] (do_one_initcall) from [<c0d00d94>] (kernel_init_freeable+0x10c/0x1d0)
-[<c0d00d94>] (kernel_init_freeable) from [<c09894b0>] (kernel_init+0x8/0x114)
-[<c09894b0>] (kernel_init) from [<c0107cf0>] (ret_from_fork+0x14/0x24)
+[<c0d00d94>] (kernel_init_freeable) from [<c0989cd0>] (kernel_init+0x8/0x114)
+[<c0989cd0>] (kernel_init) from [<c0107cf0>] (ret_from_fork+0x14/0x24)
 Code: e2800008 e8bd8070 e7f001f2 e7f001f2 (e7f001f2) 
----[ end trace f453c19b0ae474b7 ]---
+---[ end trace 7b74035030905d8a ]---
 Kernel panic - not syncing: Attempted to kill init! exitcode=0x0000000b
 
-CPU1: stopping
-CPU: 1 PID: 0 Comm: swapper/1 Tainted: G      D          4.14.0-next-20171117-13911-gee92513 #1
+CPU3: stopping
+CPU: 3 PID: 0 Comm: swapper/3 Tainted: G      D          4.14.0-next-20171120-14022-ge6016f9 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c0111130>] (unwind_backtrace) from [<c010c074>] (show_stack+0x10/0x14)
-[<c010c074>] (show_stack) from [<c0975b7c>] (dump_stack+0x84/0xa4)
-[<c0975b7c>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
+[<c010c074>] (show_stack) from [<c097639c>] (dump_stack+0x84/0xa4)
+[<c097639c>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
 [<c010f61c>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
-Exception stack(0xee8c1f70 to 0xee8c1fb8)
-1f60:                                     00000001 00000000 ee8c1fc8 c011a420
-1f80: 00000002 c0e04cdc 00000000 c0e04c68 00000000 c0d8c2b8 00000000 c0e04cec
-1fa0: 2e215000 ee8c1fc0 c0108758 c010875c 60000013 ffffffff
+Exception stack(0xee8e3f70 to 0xee8e3fb8)
+3f60:                                     00000001 00000000 ee8e3fc8 c011a420
+3f80: 00000008 c0e04cdc 00000000 c0e04c68 00000000 c0d8d2b8 00000000 c0e04cec
+3fa0: 2e236000 ee8e3fc0 c0108758 c010875c 60000013 ffffffff
 [<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
-[<c010875c>] (arch_cpu_idle) from [<c01624dc>] (do_idle+0x140/0x204)
-[<c01624dc>] (do_idle) from [<c0162828>] (cpu_startup_entry+0x18/0x1c)
-[<c0162828>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
+[<c010875c>] (arch_cpu_idle) from [<c0162508>] (do_idle+0x140/0x204)
+[<c0162508>] (do_idle) from [<c0162854>] (cpu_startup_entry+0x18/0x1c)
+[<c0162854>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
 CPU2: stopping
-CPU: 2 PID: 0 Comm: swapper/2 Tainted: G      D          4.14.0-next-20171117-13911-gee92513 #1
+CPU: 2 PID: 0 Comm: swapper/2 Tainted: G      D          4.14.0-next-20171120-14022-ge6016f9 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c0111130>] (unwind_backtrace) from [<c010c074>] (show_stack+0x10/0x14)
-[<c010c074>] (show_stack) from [<c0975b7c>] (dump_stack+0x84/0xa4)
-[<c0975b7c>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
+[<c010c074>] (show_stack) from [<c097639c>] (dump_stack+0x84/0xa4)
+[<c097639c>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
 [<c010f61c>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
 Exception stack(0xee8e1f70 to 0xee8e1fb8)
 1f60:                                     00000001 00000000 ee8e1fc8 c011a420
-1f80: 00000004 c0e04cdc 00000000 c0e04c68 00000000 c0d8c2b8 00000000 c0e04cec
-1fa0: 2e226000 ee8e1fc0 c0108758 c010875c 60000013 ffffffff
+1f80: 00000004 c0e04cdc 00000000 c0e04c68 00000000 c0d8d2b8 00000000 c0e04cec
+1fa0: 2e225000 ee8e1fc0 c0108758 c010875c 60000013 ffffffff
 [<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
-[<c010875c>] (arch_cpu_idle) from [<c01624dc>] (do_idle+0x140/0x204)
-[<c01624dc>] (do_idle) from [<c0162828>] (cpu_startup_entry+0x18/0x1c)
-[<c0162828>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
-CPU3: stopping
-CPU: 3 PID: 0 Comm: swapper/3 Tainted: G      D          4.14.0-next-20171117-13911-gee92513 #1
+[<c010875c>] (arch_cpu_idle) from [<c0162508>] (do_idle+0x140/0x204)
+[<c0162508>] (do_idle) from [<c0162854>] (cpu_startup_entry+0x18/0x1c)
+[<c0162854>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
+CPU1: stopping
+CPU: 1 PID: 0 Comm: swapper/1 Tainted: G      D          4.14.0-next-20171120-14022-ge6016f9 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c0111130>] (unwind_backtrace) from [<c010c074>] (show_stack+0x10/0x14)
-[<c010c074>] (show_stack) from [<c0975b7c>] (dump_stack+0x84/0xa4)
-[<c0975b7c>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
+[<c010c074>] (show_stack) from [<c097639c>] (dump_stack+0x84/0xa4)
+[<c097639c>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
 [<c010f61c>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
-Exception stack(0xee8e3f70 to 0xee8e3fb8)
-3f60:                                     00000001 00000000 ee8e3fc8 c011a420
-3f80: 00000008 c0e04cdc 00000000 c0e04c68 00000000 c0d8c2b8 00000000 c0e04cec
-3fa0: 2e237000 ee8e3fc0 c0108758 c010875c 60000013 ffffffff
+Exception stack(0xee8c1f70 to 0xee8c1fb8)
+1f60:                                     00000001 00000000 ee8c1fc8 c011a420
+1f80: 00000002 c0e04cdc 00000000 c0e04c68 00000000 c0d8d2b8 00000000 c0e04cec
+1fa0: 2e214000 ee8c1fc0 c0108758 c010875c 60000013 ffffffff
 [<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
-[<c010875c>] (arch_cpu_idle) from [<c01624dc>] (do_idle+0x140/0x204)
-[<c01624dc>] (do_idle) from [<c0162828>] (cpu_startup_entry+0x18/0x1c)
-[<c0162828>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
+[<c010875c>] (arch_cpu_idle) from [<c0162508>] (do_idle+0x140/0x204)
+[<c0162508>] (do_idle) from [<c0162854>] (cpu_startup_entry+0x18/0x1c)
+[<c0162854>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
 ---[ end Kernel panic - not syncing: Attempted to kill init! exitcode=0x0000000b
 
+------------[ cut here ]------------
+WARNING: CPU: 0 PID: 8 at kernel/irq_work.c:68 irq_work_queue_on+0xbc/0xfc
+Modules linked in:
+CPU: 0 PID: 8 Comm: rcu_preempt Tainted: G      D          4.14.0-next-20171120-14022-ge6016f9 #1
+Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
+[<c0111130>] (unwind_backtrace) from [<c010c074>] (show_stack+0x10/0x14)
+[<c010c074>] (show_stack) from [<c097639c>] (dump_stack+0x84/0xa4)
+[<c097639c>] (dump_stack) from [<c0121ab0>] (__warn+0xe4/0x110)
+[<c0121ab0>] (__warn) from [<c0121b20>] (warn_slowpath_null+0x44/0x4c)
+[<c0121b20>] (warn_slowpath_null) from [<c01d6b74>] (irq_work_queue_on+0xbc/0xfc)
+[<c01d6b74>] (irq_work_queue_on) from [<c0183c80>] (rcu_implicit_dynticks_qs+0x280/0x2a0)
+[<c0183c80>] (rcu_implicit_dynticks_qs) from [<c0182e34>] (force_qs_rnp+0x13c/0x1dc)
+[<c0182e34>] (force_qs_rnp) from [<c0183408>] (rcu_gp_kthread+0x534/0xa58)
+[<c0183408>] (rcu_gp_kthread) from [<c013f1c8>] (kthread+0x11c/0x154)
+[<c013f1c8>] (kthread) from [<c0107cf0>] (ret_from_fork+0x14/0x24)
+---[ end trace 7b74035030905d8b ]---
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig%tegra-fw_log.txt
--- /tmp/fooa	2017-11-19 20:19:36.825422871 -0800
+++ /tmp/foob	2017-11-19 20:19:36.829422889 -0800
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -43,18 +43,18 @@
 	 ##########################################
 	 3.6 MiB/s
 done
-Bytes transferred = 6329816 (6095d8 hex)
+Bytes transferred = 6331560 (609ca8 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.4 MiB/s
+	 3.6 MiB/s
 done
 Bytes transferred = 67482 (1079a hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6095d8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x609ca8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013799
@@ -63,7 +63,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-next-20171117-215619-gee92513 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Fri Nov 17 02:16:04 PST 2017
+Linux version 4.14.0-next-20171120-215730-ge6016f9 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Sun Nov 19 20:03:46 PST 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -86,7 +86,7 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=4d36e82a-b3c0-4b4b-bfcd-afe5217ac023 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998488K/2097148K available (9216K kernel code, 777K rwdata, 2820K rodata, 1024K init, 299K bss, 33124K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1998488K/2097148K available (9216K kernel code, 778K rwdata, 2820K rodata, 1024K init, 299K bss, 33124K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -96,8 +96,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0e00000 - 0xc0f00000   (1024 kB)
-      .data : 0xc0f00000 - 0xc0fc2728   ( 778 kB)
-       .bss : 0xc0fc2728 - 0xc100d3b4   ( 300 kB)
+      .data : 0xc0f00000 - 0xc0fc2820   ( 779 kB)
+       .bss : 0xc0fc2820 - 0xc100d4b4   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -313,130 +313,131 @@
 kernel BUG at drivers/platform/chrome/cros_ec_proto.c:34!
 Internal error: Oops - BUG: 0 [#1] PREEMPT SMP ARM
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.14.0-next-20171117-215619-gee92513 #1
+CPU: 2 PID: 1 Comm: swapper/0 Not tainted 4.14.0-next-20171120-215730-ge6016f9 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 task: ee890740 task.stack: ee8a6000
 PC is at cros_ec_prepare_tx+0x168/0x16c
 LR is at cros_ec_pkt_xfer_spi+0x18/0x474
-pc : [<c06e7398>]    lr : [<c0555d98>]    psr: 20000013
+pc : [<c06e76b4>]    lr : [<c0556054>]    psr: 20000013
 sp : ee8a7bd8  ip : 00000000  fp : 00000000
-r10: c0555d80  r9 : 00000000  r8 : eefe7fb4
-r7 : ee28db00  r6 : ee28db00  r5 : ee28db00  r4 : ee270e50
-r3 : 0000ffff  r2 : 00000000  r1 : 0000ffff  r0 : ee270e50
+r10: c055603c  r9 : 00000000  r8 : eefe7fb4
+r7 : ee284b40  r6 : ee284b40  r5 : ee284b40  r4 : ee0afcd0
+r3 : 0000ffff  r2 : 00000000  r1 : 0000ffff  r0 : ee0afcd0
 Flags: nzCv  IRQs on  FIQs on  Mode SVC_32  ISA ARM  Segment none
 Control: 10c5387d  Table: 8000406a  DAC: 00000051
 Process swapper/0 (pid: 1, stack limit = 0xee8a6210)
 Stack: (0xee8a7bd8 to 0xee8a8000)
-7bc0:                                                       ee270e50 ee28db00
-7be0: ee28db90 c0555d98 c098ba74 00000004 ee8a7c5c ee8a7c5c ee28b800 00000000
-7c00: c05a232c 00000000 00000008 00000008 00000000 ee8a7c14 ee8a7c14 00000000
-7c20: ee8a7c20 ee8a7c20 ee28db50 ee28da80 00000008 00000000 00000000 00000000
-7c40: c0e90bd0 c023b620 40002800 00000560 eef94bd0 c023da84 002dc6c0 ee8a7bf0
-7c60: ee8a7bf0 00000000 c06e71c8 005c51a0 3b9aca00 000f4240 00000000 ee8a7ca8
-7c80: 00000001 00000501 c0555d80 c0990410 00000000 ee28dac0 a0000013 c06e71c8
-7ca0: ee28db00 ee270e50 ee28db00 ee28b800 ee28db00 eefe7fb4 00000000 c0555d80
-7cc0: 00000000 c06e7124 0000400b ee270e50 ee28db00 ee28b800 0000000c eefe7fb4
-7ce0: c0fecdbc c06e7540 00000000 ee270e50 ee28b800 00000000 00000000 eefe7fb4
-7d00: c0f6c308 c0555824 ee8a7d14 ee28b800 00000000 00000001 ee28b800 ee270e50
-7d20: ee28db90 c0555bd4 00000000 00000000 00000000 000007d0 ee28b800 00000000
-7d40: c0f6c2f8 c0ffe068 00000000 c05a2d70 ee28b800 c0ffe064 00000000 c05315b0
-7d60: 00000000 ee8a7d98 c053175c 00000001 c0ffe040 00000000 00000000 c052fbb8
-7d80: ee9f1170 ee2605b8 ee28b800 ee28b834 c0f72e34 c05312d8 ee28b800 00000001
-7da0: 00000000 ee28b800 ee28b800 c0f72e34 ee03a000 c053091c ee28b800 00000000
-7dc0: ee28b808 c052ef60 ee28b800 c05a5cdc 00000000 ee28b800 ee28b800 ee03a000
-7de0: 00000000 eea8be10 eefe8004 00000001 00000000 c05a2bc4 002dc6c0 ee03a000
-7e00: eefe7fb4 00000000 ee28b800 c05a4b64 00000000 014000c0 00000000 c05a4550
-7e20: 00000063 002dc6c0 ee03a000 ee28d650 eea8be00 eea8be10 40000000 00000153
-7e40: 00000000 c05a5088 eefe7d0c ee03a350 ee03a000 eea8be00 eea8be10 c05a5c14
-7e60: 00000000 ee03a350 ffffffed eea8be10 fffffdfb c0f73790 00000000 00000000
-7e80: c0f73790 c0532ec8 c0532e78 eea8be10 c0ffe064 00000000 c0ffe068 c05315b0
-7ea0: eea8be10 c0f73790 eea8be44 00000000 0000010e c0e6a83c c0e00594 c0531724
-7ec0: 00000000 c0f73790 c0531678 c052fc54 ee8cc55c eea14934 c0f73790 ee287e80
-7ee0: c0f6a8f8 c0530af0 c0c21c88 c0f73790 00000006 c0f73790 00000006 c0e3866c
-7f00: 00000000 c05322a4 c0532b1c c0e8332c 00000006 c0101b8c 00000000 c0cc01e0
-7f20: 00000100 efffcc45 c0cc01e0 0000010e efffcc53 c013e8f4 c0cbecc8 00000000
-7f40: 00000006 00000006 efffcc45 efffcc4a 00000000 efffcbc0 00000006 c0e6a82c
-7f60: c0e8332c 00000006 c0e6a830 c0fc2740 0000010e c0e00d94 00000006 00000006
-7f80: 00000000 c0e00594 00000000 c09894a8 00000000 00000000 00000000 00000000
-7fa0: 00000000 c09894b0 00000000 c0107cf0 00000000 00000000 00000000 00000000
+7bc0:                                                       ee0afcd0 ee284b40
+7be0: ee284bd0 c0556054 c098c2f4 00000004 ee8a7c5c ee8a7c5c ee281800 00000000
+7c00: c05a25e4 00000000 00000008 00000008 00000000 ee8a7c14 ee8a7c14 00000000
+7c20: ee8a7c20 ee8a7c20 ee284b90 ee284ac0 00000008 00000000 00000000 00000000
+7c40: c0e91bd0 c023b654 40002800 00000580 eefb6bd0 c023dab8 002dc6c0 ee8a7bf0
+7c60: ee8a7bf0 00000000 c06e74e4 005c5080 3b9aca00 000f4240 00000000 ee8a7ca8
+7c80: 00000001 00000501 c055603c c0990c90 00000000 ee284b00 a0000013 c06e74e4
+7ca0: ee284b40 ee0afcd0 ee284b40 ee281800 ee284b40 eefe7fb4 00000000 c055603c
+7cc0: 00000000 c06e7440 0000400b ee0afcd0 ee284b40 ee281800 0000000c eefe7fb4
+7ce0: c0fecebc c06e785c 00000000 ee0afcd0 ee281800 00000000 00000000 eefe7fb4
+7d00: c0f6c3c8 c0555ae0 ee8a7d14 ee281800 00000000 00000001 ee281800 ee0afcd0
+7d20: ee284bd0 c0555e90 00000000 00000000 00000000 000007d0 ee281800 00000000
+7d40: c0f6c3b8 c0ffe168 00000000 c05a3028 ee281800 c0ffe164 00000000 c053186c
+7d60: 00000000 ee8a7d98 c0531a18 00000001 c0ffe140 00000000 00000000 c052fe74
+7d80: ee9f1170 ee029838 ee281800 ee281834 c0f72ef4 c0531594 ee281800 00000001
+7da0: 00000000 ee281800 ee281800 c0f72ef4 ee107000 c0530bd8 ee281800 00000000
+7dc0: ee281808 c052f21c ee281800 c05a5f94 00000000 ee281800 ee281800 ee107000
+7de0: 00000000 eea8be10 eefe8004 00000001 00000000 c05a2e7c 002dc6c0 ee107000
+7e00: eefe7fb4 00000000 ee281800 c05a4e1c 00000000 014000c0 00000000 c05a4808
+7e20: 00000063 002dc6c0 ee107000 ee284690 eea8be00 eea8be10 40000000 00000153
+7e40: 00000000 c05a5340 eefe7d0c ee107350 ee107000 eea8be00 eea8be10 c05a5ecc
+7e60: 00000000 ee107350 ffffffed eea8be10 fffffdfb c0f73850 00000000 00000000
+7e80: c0f73850 c0533184 c0533134 eea8be10 c0ffe164 00000000 c0ffe168 c053186c
+7ea0: eea8be10 c0f73850 eea8be44 00000000 0000010e c0e6b83c c0e00594 c05319e0
+7ec0: 00000000 c0f73850 c0531934 c052ff10 ee8cc55c eea14934 c0f73850 ee27d100
+7ee0: c0f6a9b8 c0530dac c0c21d54 c0f73850 00000006 c0f73850 00000006 c0e38678
+7f00: 00000000 c0532560 c0532dd8 c0e8434c 00000006 c0101b8c 00000000 c0cc0234
+7f20: 00000100 efffcc45 c0cc0234 0000010e efffcc53 c013e920 c0cbed1c 00000000
+7f40: 00000006 00000006 efffcc45 efffcc4a 00000000 efffcbc0 00000006 c0e6b82c
+7f60: c0e8434c 00000006 c0e6b830 c0fc2840 0000010e c0e00d94 00000006 00000006
+7f80: 00000000 c0e00594 00000000 c0989cc8 00000000 00000000 00000000 00000000
+7fa0: 00000000 c0989cd0 00000000 c0107cf0 00000000 00000000 00000000 00000000
 7fc0: 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
 7fe0: 00000000 00000000 00000000 00000000 00000013 00000000 00000000 00000000
-[<c06e7398>] (cros_ec_prepare_tx) from [<c0555d98>] (cros_ec_pkt_xfer_spi+0x18/0x474)
-[<c0555d98>] (cros_ec_pkt_xfer_spi) from [<c06e7124>] (send_command+0x20/0xd8)
-[<c06e7124>] (send_command) from [<c06e7540>] (cros_ec_query_all+0x1a4/0x398)
-[<c06e7540>] (cros_ec_query_all) from [<c0555824>] (cros_ec_register+0x94/0x21c)
-[<c0555824>] (cros_ec_register) from [<c0555bd4>] (cros_ec_spi_probe+0x108/0x150)
-[<c0555bd4>] (cros_ec_spi_probe) from [<c05a2d70>] (spi_drv_probe+0x7c/0xa8)
-[<c05a2d70>] (spi_drv_probe) from [<c05315b0>] (driver_probe_device+0x250/0x318)
-[<c05315b0>] (driver_probe_device) from [<c052fbb8>] (bus_for_each_drv+0x44/0x8c)
-[<c052fbb8>] (bus_for_each_drv) from [<c05312d8>] (__device_attach+0x9c/0x100)
-[<c05312d8>] (__device_attach) from [<c053091c>] (bus_probe_device+0x84/0x8c)
-[<c053091c>] (bus_probe_device) from [<c052ef60>] (device_add+0x380/0x528)
-[<c052ef60>] (device_add) from [<c05a2bc4>] (spi_add_device+0x88/0x130)
-[<c05a2bc4>] (spi_add_device) from [<c05a4b64>] (spi_register_controller+0x428/0x918)
-[<c05a4b64>] (spi_register_controller) from [<c05a5088>] (devm_spi_register_controller+0x34/0x70)
-[<c05a5088>] (devm_spi_register_controller) from [<c05a5c14>] (tegra_spi_probe+0x298/0x35c)
-[<c05a5c14>] (tegra_spi_probe) from [<c0532ec8>] (platform_drv_probe+0x50/0xb0)
-[<c0532ec8>] (platform_drv_probe) from [<c05315b0>] (driver_probe_device+0x250/0x318)
-[<c05315b0>] (driver_probe_device) from [<c0531724>] (__driver_attach+0xac/0xb0)
-[<c0531724>] (__driver_attach) from [<c052fc54>] (bus_for_each_dev+0x54/0x88)
-[<c052fc54>] (bus_for_each_dev) from [<c0530af0>] (bus_add_driver+0xe4/0x1f0)
-[<c0530af0>] (bus_add_driver) from [<c05322a4>] (driver_register+0x78/0xf4)
-[<c05322a4>] (driver_register) from [<c0101b8c>] (do_one_initcall+0x3c/0x168)
+[<c06e76b4>] (cros_ec_prepare_tx) from [<c0556054>] (cros_ec_pkt_xfer_spi+0x18/0x474)
+[<c0556054>] (cros_ec_pkt_xfer_spi) from [<c06e7440>] (send_command+0x20/0xd8)
+[<c06e7440>] (send_command) from [<c06e785c>] (cros_ec_query_all+0x1a4/0x398)
+[<c06e785c>] (cros_ec_query_all) from [<c0555ae0>] (cros_ec_register+0x94/0x21c)
+[<c0555ae0>] (cros_ec_register) from [<c0555e90>] (cros_ec_spi_probe+0x108/0x150)
+[<c0555e90>] (cros_ec_spi_probe) from [<c05a3028>] (spi_drv_probe+0x7c/0xa8)
+[<c05a3028>] (spi_drv_probe) from [<c053186c>] (driver_probe_device+0x250/0x318)
+[<c053186c>] (driver_probe_device) from [<c052fe74>] (bus_for_each_drv+0x44/0x8c)
+[<c052fe74>] (bus_for_each_drv) from [<c0531594>] (__device_attach+0x9c/0x100)
+[<c0531594>] (__device_attach) from [<c0530bd8>] (bus_probe_device+0x84/0x8c)
+[<c0530bd8>] (bus_probe_device) from [<c052f21c>] (device_add+0x380/0x528)
+[<c052f21c>] (device_add) from [<c05a2e7c>] (spi_add_device+0x88/0x130)
+[<c05a2e7c>] (spi_add_device) from [<c05a4e1c>] (spi_register_controller+0x428/0x918)
+[<c05a4e1c>] (spi_register_controller) from [<c05a5340>] (devm_spi_register_controller+0x34/0x70)
+[<c05a5340>] (devm_spi_register_controller) from [<c05a5ecc>] (tegra_spi_probe+0x298/0x35c)
+[<c05a5ecc>] (tegra_spi_probe) from [<c0533184>] (platform_drv_probe+0x50/0xb0)
+[<c0533184>] (platform_drv_probe) from [<c053186c>] (driver_probe_device+0x250/0x318)
+[<c053186c>] (driver_probe_device) from [<c05319e0>] (__driver_attach+0xac/0xb0)
+[<c05319e0>] (__driver_attach) from [<c052ff10>] (bus_for_each_dev+0x54/0x88)
+[<c052ff10>] (bus_for_each_dev) from [<c0530dac>] (bus_add_driver+0xe4/0x1f0)
+[<c0530dac>] (bus_add_driver) from [<c0532560>] (driver_register+0x78/0xf4)
+[<c0532560>] (driver_register) from [<c0101b8c>] (do_one_initcall+0x3c/0x168)
 [<c0101b8c>] (do_one_initcall) from [<c0e00d94>] (kernel_init_freeable+0x10c/0x1d0)
-[<c0e00d94>] (kernel_init_freeable) from [<c09894b0>] (kernel_init+0x8/0x114)
-[<c09894b0>] (kernel_init) from [<c0107cf0>] (ret_from_fork+0x14/0x24)
+[<c0e00d94>] (kernel_init_freeable) from [<c0989cd0>] (kernel_init+0x8/0x114)
+[<c0989cd0>] (kernel_init) from [<c0107cf0>] (ret_from_fork+0x14/0x24)
 Code: e2800008 e8bd8070 e7f001f2 e7f001f2 (e7f001f2) 
----[ end trace 87f997ff11d7f60c ]---
+---[ end trace b99f9d6ab4c89ad0 ]---
 Kernel panic - not syncing: Attempted to kill init! exitcode=0x0000000b
 
-CPU2: stopping
-CPU: 2 PID: 0 Comm: swapper/2 Tainted: G      D          4.14.0-next-20171117-215619-gee92513 #1
+CPU3: stopping
+CPU: 3 PID: 0 Comm: swapper/3 Tainted: G      D          4.14.0-next-20171120-215730-ge6016f9 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c0111130>] (unwind_backtrace) from [<c010c074>] (show_stack+0x10/0x14)
-[<c010c074>] (show_stack) from [<c0975b7c>] (dump_stack+0x84/0xa4)
-[<c0975b7c>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
+[<c010c074>] (show_stack) from [<c097639c>] (dump_stack+0x84/0xa4)
+[<c097639c>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
 [<c010f61c>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
-Exception stack(0xee8e1f70 to 0xee8e1fb8)
-1f60:                                     00000001 00000000 ee8e1fc8 c011a420
-1f80: 00000004 c0f04cdc 00000000 c0f04c68 00000000 c0e8c2b8 00000000 c0f04cec
-1fa0: 2e126000 ee8e1fc0 c0108758 c010875c 60000013 ffffffff
+Exception stack(0xee8e3f70 to 0xee8e3fb8)
+3f60:                                     00000001 00000000 ee8e3fc8 c011a420
+3f80: 00000008 c0f04cdc 00000000 c0f04c68 00000000 c0e8d2b8 00000000 c0f04cec
+3fa0: 2e136000 ee8e3fc0 c0108758 c010875c 60000013 ffffffff
 [<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
-[<c010875c>] (arch_cpu_idle) from [<c01624dc>] (do_idle+0x140/0x204)
-[<c01624dc>] (do_idle) from [<c0162828>] (cpu_startup_entry+0x18/0x1c)
-[<c0162828>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
-CPU3: stopping
-CPU: 3 PID: 0 Comm: swapper/3 Tainted: G      D          4.14.0-next-20171117-215619-gee92513 #1
+[<c010875c>] (arch_cpu_idle) from [<c0162508>] (do_idle+0x140/0x204)
+[<c0162508>] (do_idle) from [<c0162854>] (cpu_startup_entry+0x18/0x1c)
+[<c0162854>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
+CPU0: stopping
+CPU: 0 PID: 0 Comm: swapper/0 Tainted: G      D          4.14.0-next-20171120-215730-ge6016f9 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c0111130>] (unwind_backtrace) from [<c010c074>] (show_stack+0x10/0x14)
-[<c010c074>] (show_stack) from [<c0975b7c>] (dump_stack+0x84/0xa4)
-[<c0975b7c>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
+[<c010c074>] (show_stack) from [<c097639c>] (dump_stack+0x84/0xa4)
+[<c097639c>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
 [<c010f61c>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
-Exception stack(0xee8e3f70 to 0xee8e3fb8)
-3f60:                                     00000001 00000000 ee8e3fc8 c011a420
-3f80: 00000008 c0f04cdc 00000000 c0f04c68 00000000 c0e8c2b8 00000000 c0f04cec
-3fa0: 2e137000 ee8e3fc0 c0108758 c010875c 60000013 ffffffff
+Exception stack(0xc0f01f38 to 0xc0f01f80)
+1f20:                                                       00000001 00000000
+1f40: c0f01f90 c011a420 00000001 c0f04cdc 00000000 c0f04c68 00000000 c0e8d2b8
+1f60: 00000000 c0f04cec 2e103000 c0f01f88 c0108758 c010875c 60000013 ffffffff
 [<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
-[<c010875c>] (arch_cpu_idle) from [<c01624dc>] (do_idle+0x140/0x204)
-[<c01624dc>] (do_idle) from [<c0162828>] (cpu_startup_entry+0x18/0x1c)
-[<c0162828>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
+[<c010875c>] (arch_cpu_idle) from [<c0162508>] (do_idle+0x140/0x204)
+[<c0162508>] (do_idle) from [<c0162854>] (cpu_startup_entry+0x18/0x1c)
+[<c0162854>] (cpu_startup_entry) from [<c0e00c20>] (start_kernel+0x35c/0x3c4)
+[<c0e00c20>] (start_kernel) from [<00000000>] (  (null))
 CPU1: stopping
-CPU: 1 PID: 0 Comm: swapper/1 Tainted: G      D          4.14.0-next-20171117-215619-gee92513 #1
+CPU: 1 PID: 0 Comm: swapper/1 Tainted: G      D          4.14.0-next-20171120-215730-ge6016f9 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c0111130>] (unwind_backtrace) from [<c010c074>] (show_stack+0x10/0x14)
-[<c010c074>] (show_stack) from [<c0975b7c>] (dump_stack+0x84/0xa4)
-[<c0975b7c>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
+[<c010c074>] (show_stack) from [<c097639c>] (dump_stack+0x84/0xa4)
+[<c097639c>] (dump_stack) from [<c010f61c>] (handle_IPI+0x18c/0x39c)
 [<c010f61c>] (handle_IPI) from [<c0101430>] (gic_handle_irq+0x88/0x94)
 [<c0101430>] (gic_handle_irq) from [<c010cc4c>] (__irq_svc+0x6c/0xa8)
 Exception stack(0xee8c1f70 to 0xee8c1fb8)
 1f60:                                     00000001 00000000 ee8c1fc8 c011a420
-1f80: 00000002 c0f04cdc 00000000 c0f04c68 00000000 c0e8c2b8 00000000 c0f04cec
-1fa0: 2e115000 ee8c1fc0 c0108758 c010875c 60000013 ffffffff
+1f80: 00000002 c0f04cdc 00000000 c0f04c68 00000000 c0e8d2b8 00000000 c0f04cec
+1fa0: 2e114000 ee8c1fc0 c0108758 c010875c 60000013 ffffffff
 [<c010cc4c>] (__irq_svc) from [<c010875c>] (arch_cpu_idle+0x30/0x3c)
-[<c010875c>] (arch_cpu_idle) from [<c01624dc>] (do_idle+0x140/0x204)
-[<c01624dc>] (do_idle) from [<c0162828>] (cpu_startup_entry+0x18/0x1c)
-[<c0162828>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
+[<c010875c>] (arch_cpu_idle) from [<c0162508>] (do_idle+0x140/0x204)
+[<c0162508>] (do_idle) from [<c0162854>] (cpu_startup_entry+0x18/0x1c)
+[<c0162854>] (cpu_startup_entry) from [<801018ac>] (0x801018ac)
 ---[ end Kernel panic - not syncing: Attempted to kill init! exitcode=0x0000000b
 
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2017-11-19 20:19:36.833422907 -0800
+++ /tmp/foob	2017-11-19 20:19:36.833422907 -0800
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1801586 us
+Bootloader-Cpu Init at (time stamp): 1801307 us
 
 Pinmux changes applied in kernel way
 
@@ -59,25 +59,25 @@
 The proc BoardInfo: 0x0176:0x0001:0x01:0x43:0x00
 FastbootAddAtag: setting ATAG (0)
 
-Board Id Read Time: 3 us
+Board Id Read Time: 4 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1801560  us
-Bootloader (AosInit) time: 733498 us
-Bootloader Main Init time: 180195 us
-BlockDevMgrInit time: 157548 us
+Miscellaneous avp time: 1801280  us
+Bootloader (AosInit) time: 733464 us
+Bootloader Main Init time: 180298 us
+BlockDevMgrInit time: 157551 us
 PartMgrInit time: 8 us
-StorMgrInit time: 30 us
+StorMgrInit time: 31 us
 FsMgrInit time: 2 us
-Partition Table load: 77075 us 
-Display init time: 574517 us
-Kernel read time: 1510 msec
+Partition Table load: 77101 us 
+Display init time: 574684 us
+Kernel read time: 1509 msec
 Load Kernel Size:   20013056 Bytes
-Bl Kernel ReadSpeed: 13253000 Bytes/sec
-Pre-Kernel Jump time: 4246329 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 9374322 us
+Bl Kernel ReadSpeed: 13262000 Bytes/sec
+Pre-Kernel Jump time: 4245514 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 9375599 us
 Booting Linux on physical CPU 0x0000000000 [0x4e0f0000]
-Linux version 4.14.0-next-20171117-333860-gee92513eae01 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Fri Nov 17 02:22:41 PST 2017
+Linux version 4.14.0-next-20171120-333971-ge6016f98372b (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sun Nov 19 20:11:05 PST 2017
 Machine model: NVIDIA Tegra132 Norrin
 Memory limited to 1734MB
 efi: Getting EFI parameters from FDT:
@@ -148,7 +148,7 @@
 DMI not present or invalid.
 NET: Registered protocol family 16
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.132:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.128:1): state=initialized audit_enabled=0 res=1
 cpuidle: using governor menu
 vdso: 2 pages (1 code @ ffff0000089b6000, 1 data @ ffff000008f25000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
@@ -282,6 +282,7 @@
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
+sdhci-tegra 700b0400.sdhci: Got CD GPIO
 hctosys: unable to open rtc device (rtc0)
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +VDD_LED: disabling
@@ -292,7 +293,6 @@
 ALSA device list:
   No soundcards found.
 Freeing unused kernel memory: 1088K
-sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 
 System initialization...
Diff for ./tegra186-p2771-0000/tegra186-p2771-0000/defconfig%jetson-tx2_log.txt
Binary files /tmp/fooa and /tmp/foob differ
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-11-19 20:19:36.865423051 -0800
+++ /tmp/foob	2017-11-19 20:19:36.869423070 -0800
@@ -48,7 +48,7 @@
 	 #################################################################
 	 #################################################################
 	 #############################
-	 3.6 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 8057344 (7af200 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
@@ -69,7 +69,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-next-20171117-333860-gee92513eae01 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Fri Nov 17 02:16:35 PST 2017
+Linux version 4.14.0-next-20171120-333971-ge6016f98372b (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Sun Nov 19 20:05:02 PST 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -101,8 +101,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0e00000   (12256 kB)
       .init : 0xc1300000 - 0xc1500000   (2048 kB)
-      .data : 0xc1500000 - 0xc1690800   (1602 kB)
-       .bss : 0xc1690800 - 0xc16f5bb4   ( 405 kB)
+      .data : 0xc1500000 - 0xc16908e0   (1603 kB)
+       .bss : 0xc16908e0 - 0xc16f5cb4   ( 405 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2017-11-19 20:19:36.897423196 -0800
+++ /tmp/foob	2017-11-19 20:19:36.897423196 -0800
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -45,21 +45,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##############################
-	 4 MiB/s
+	 ###############################
+	 4.1 MiB/s
 done
-Bytes transferred = 6164560 (5e1050 hex)
+Bytes transferred = 6166776 (5e18f8 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 21160 (52a8 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x5e1050 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x5e18f8 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102a7
@@ -68,7 +68,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-next-20171117-13911-gee92513 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Fri Nov 17 02:15:48 PST 2017
+Linux version 4.14.0-next-20171120-14022-ge6016f9 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Sun Nov 19 20:03:22 PST 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -89,7 +89,7 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=11eb23c1-01 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 959424K/1048576K available (9216K kernel code, 777K rwdata, 2556K rodata, 1024K init, 299K bss, 23616K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 959424K/1048576K available (9216K kernel code, 778K rwdata, 2560K rodata, 1024K init, 299K bss, 23616K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -99,8 +99,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0ec2728   ( 778 kB)
-       .bss : 0xc0ec2728 - 0xc0f0d3b4   ( 300 kB)
+      .data : 0xc0e00000 - 0xc0ec2820   ( 779 kB)
+       .bss : 0xc0ec2820 - 0xc0f0d4b4   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
 	Tasks RCU enabled.
@@ -323,27 +323,27 @@
 Bluetooth: RFCOMM ver 1.11
 Bluetooth: BNEP (Ethernet Emulation) ver 1.3
 Bluetooth: BNEP socket layer initialized
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
 Bluetooth: HIDP (Human Interface Emulation) ver 1.2
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 Bluetooth: HIDP socket layer initialized
 Key type dns_resolver registered
 Registering SWP/SWPB emulation handler
 Loading compiled-in X.509 certificates
-utmi_phy_clk_enable: timeout waiting for phy to stabilize
 mmc1: new high speed SD card at address b368
-tegra-ehci c5000000.usb: EHCI Host Controller
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
+utmi_phy_clk_enable: timeout waiting for phy to stabilize
+tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
- mmcblk1: p1 p2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
+ mmcblk1: p1 p2
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: hctosys: unable to read the hardware clock
-cfg80211: Loading compiled-in X.509 certificates for regulatory database
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
+rtc-em3027 2-0056: hctosys: unable to read the hardware clock
+cfg80211: Loading compiled-in X.509 certificates for regulatory database
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
 cfg80211: failed to load regulatory.db
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig%jetson-tx1_log.txt
--- /tmp/fooa	2017-11-19 20:19:36.921423305 -0800
+++ /tmp/foob	2017-11-19 20:19:36.925423323 -0800
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 741644 us
-Basic modules init took 348704 us
+Tegraboot started after 772084 us
+Basic modules init took 350129 us
 Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14365 us
-Carveout took 252663 us
-CPU initialization took 92433 us
-Total time taken by TegraBoot 708165 us
+Next stage binary read took 14366 us
+Carveout took 252721 us
+CPU initialization took 92455 us
+Total time taken by TegraBoot 709671 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.580] LPDDR4 Training: Number of tables = 10
+64b[0001.612] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,9 +117,9 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig%jetson-tx1
-16644608 bytes read in 857 ms (18.5 MiB/s)
+16644608 bytes read in 864 ms (18.4 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-44764 bytes read in 453 ms (95.7 KiB/s)
+44764 bytes read in 454 ms (95.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
 1627787 bytes read in 495 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0000000000 [0x411fd071]
-Linux version 4.14.0-next-20171117-215619-gee92513 (jonathanh@sellafield) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Fri Nov 17 02:19:39 PST 2017
+Linux version 4.14.0-next-20171120-14022-ge6016f9 (jonathanh@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sun Nov 19 20:07:34 PST 2017
 Machine model: NVIDIA Tegra210 P2371 (P2530/P2595) reference design
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -267,7 +267,7 @@
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 PCI: CLS 0 bytes, default 128
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 809a4000
+kvm [1]: IDMAP page: 809a5000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: vgic-v2@50044000
 kvm [1]: vgic interrupt IRQ1
@@ -343,9 +343,10 @@
 registered taskstats version 1
 Loading compiled-in X.509 certificates
 hctosys: unable to open rtc device (rtc0)
-mmc0: new high speed MMC card at address 0001
 ALSA device list:
   No soundcards found.
+Waiting for root device PARTUUID=00000000-0001-0000-788f-a75600000000...
+mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 SDW16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SDW16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SDW16G partition 2 4.00 MiB
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2017-11-19 20:19:36.953423449 -0800
+++ /tmp/foob	2017-11-19 20:19:36.953423449 -0800
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 741149 us
-Basic modules init took 348867 us
+Tegraboot started after 799017 us
+Basic modules init took 348580 us
 Sec Bootdevice Read Time = 203 ms, Read Size = 8459 KB
-Next stage binary read took 14365 us
-Carveout took 252672 us
-CPU initialization took 93709 us
-Total time taken by TegraBoot 709613 us
+Next stage binary read took 14398 us
+Carveout took 253356 us
+CPU initialization took 92444 us
+Total time taken by TegraBoot 708778 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.581] LPDDR4 Training: Number of tables = 10
+64b[0001.637] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-16513536 bytes read in 829 ms (19 MiB/s)
+16513536 bytes read in 835 ms (18.9 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
 44764 bytes read in 454 ms (95.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 494 ms (3.1 MiB/s)
+1627787 bytes read in 495 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0000000000 [0x411fd071]
-Linux version 4.14.0-next-20171117-333860-gee92513eae01 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Fri Nov 17 02:22:41 PST 2017
+Linux version 4.14.0-next-20171120-333971-ge6016f98372b (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sun Nov 19 20:11:05 PST 2017
 Machine model: NVIDIA Tegra210 P2371 (P2530/P2595) reference design
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
Diff for ./tegra210-p2371-2180/tegra210-p2371-2180/defconfig%jetson-tx1_log.txt
Binary files /tmp/fooa and /tmp/foob differ
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2017-11-19 20:19:36.981423576 -0800
+++ /tmp/foob	2017-11-19 20:19:36.981423576 -0800
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0000000000 [0x411fd071]
-Linux version 4.14.0-next-20171117-333860-gee92513eae01 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Fri Nov 17 02:22:41 PST 2017
+Linux version 4.14.0-next-20171120-333971-ge6016f98372b (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Sun Nov 19 20:11:05 PST 2017
 Machine model: Google Pixel C
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
@@ -263,8 +263,9 @@
 mmcblk0boot0: mmc0:0001 BGND3R partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB, chardev (240:0)
+max77686-rtc max77620-rtc: setting system clock to 2001-08-31 09:40:59 UTC (999250859)
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
-max77686-rtc max77620-rtc: setting system clock to 2001-08-28 15:52:19 UTC (999013939)
+tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 PP1200_AVDD: disabling
 PP1200_RCAM: disabling
 PP2800L_RCAM: disabling
@@ -276,7 +277,6 @@
 ALSA device list:
   No soundcards found.
 Freeing unused kernel memory: 1088K
-tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 
 System initialization...
 
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-11-19 20:19:37.001423666 -0800
+++ /tmp/foob	2017-11-19 20:19:37.005423684 -0800
@@ -27,7 +27,6 @@
 Tegra30 (Beaver) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=edbe1840-3633-c751-7993-b0ac834f81e0 rootwait gpt debug noinitrd earlyprintk init=/bin/bash'
 Tegra30 (Beaver) # setenv ipaddr 192.168.100.101; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra30 (Beaver) # tftpboot 0x81000000 zImage.multi_v7_defconfig
-Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'zImage.multi_v7_defconfig'.
@@ -41,7 +40,7 @@
 	 #################################################################
 	 #################################################################
 	 #############################
-	 2.8 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 8057344 (7af200 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
@@ -50,7 +49,7 @@
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.6 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 51470 (c90e hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
@@ -62,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-next-20171117-333860-gee92513eae01 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Fri Nov 17 02:16:35 PST 2017
+Linux version 4.14.0-next-20171120-333971-ge6016f98372b (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Sun Nov 19 20:05:02 PST 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -94,8 +93,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0e00000   (12256 kB)
       .init : 0xc1300000 - 0xc1500000   (2048 kB)
-      .data : 0xc1500000 - 0xc1690800   (1602 kB)
-       .bss : 0xc1690800 - 0xc16f5bb4   ( 405 kB)
+      .data : 0xc1500000 - 0xc16908e0   (1603 kB)
+       .bss : 0xc16908e0 - 0xc16f5cb4   ( 405 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -464,4 +463,4 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.494051] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.494050] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2017-11-19 20:19:37.009423702 -0800
+++ /tmp/foob	2017-11-19 20:19:37.013423720 -0800
@@ -37,21 +37,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##############################
+	 ###############################
 	 3.3 MiB/s
 done
-Bytes transferred = 6164560 (5e1050 hex)
+Bytes transferred = 6166776 (5e18f8 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.3 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 51470 (c90e hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5e1050 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5e18f8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90d ... OK
@@ -60,7 +60,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.14.0-next-20171117-13911-gee92513 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Fri Nov 17 02:15:48 PST 2017
+Linux version 4.14.0-next-20171120-14022-ge6016f9 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Sun Nov 19 20:03:22 PST 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -81,7 +81,7 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=edbe1840-3633-c751-7993-b0ac834f81e0 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998528K/2096128K available (9216K kernel code, 777K rwdata, 2556K rodata, 1024K init, 299K bss, 32064K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998528K/2096128K available (9216K kernel code, 778K rwdata, 2560K rodata, 1024K init, 299K bss, 32064K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -91,8 +91,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0ec2728   ( 778 kB)
-       .bss : 0xc0ec2728 - 0xc0f0d3b4   ( 300 kB)
+      .data : 0xc0e00000 - 0xc0ec2820   ( 779 kB)
+       .bss : 0xc0ec2820 - 0xc0f0d4b4   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -465,4 +465,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# 
\ No newline at end of file
+root@(none):/# [    4.727578] usb 2-1: new high-speed USB device number 2 using tegra-ehci
