Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2015-12-16 23:24:09.061694404 -0800
+++ /tmp/foob	2015-12-16 23:24:09.065694401 -0800
@@ -1,6 +1,6 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
-Linux version 4.4.0-rc5-next-20151216+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Dec 15 22:19:13 PST 2015
+Linux version 4.4.0-rc5-next-20151217+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Dec 16 23:19:09 PST 2015
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -20,7 +20,7 @@
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
 software IO TLB [mem 0x79c00000-0x7dc00000] (64MB) mapped at [ffffffc039c00000-ffffffc03dbfffff]
-Memory: 938088K/1048576K available (5924K kernel code, 562K rwdata, 2568K rodata, 676K init, 220K bss, 94104K reserved, 16384K cma-reserved)
+Memory: 938088K/1048576K available (5924K kernel code, 558K rwdata, 2572K rodata, 676K init, 220K bss, 94104K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     vmalloc : 0xffffff8000000000 - 0xffffffbdbfff0000   (   246 GB)
     vmemmap : 0xffffffbdc0000000 - 0xffffffbfc0000000   (     8 GB maximum)
@@ -29,9 +29,9 @@
     PCI I/O : 0xffffffbffae00000 - 0xffffffbffbe00000   (    16 MB)
     modules : 0xffffffbffc000000 - 0xffffffc000000000   (    64 MB)
     memory  : 0xffffffc000000000 - 0xffffffc040000000   (  1024 MB)
-      .init : 0xffffffc0008ce000 - 0xffffffc000977000   (   676 KB)
-      .text : 0xffffffc000080000 - 0xffffffc0008cd0b4   (  8501 KB)
-      .data : 0xffffffc000977000 - 0xffffffc000a03a00   (   563 KB)
+      .init : 0xffffffc0008cf000 - 0xffffffc000978000   (   676 KB)
+      .text : 0xffffffc000080000 - 0xffffffc0008ce0b4   (  8505 KB)
+      .data : 0xffffffc000978000 - 0xffffffc000a03a00   (   559 KB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 64.
@@ -95,7 +95,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.624:1): initialized
+audit: type=2000 audit(0.656:1): initialized
 HugeTLB registered 2 MB page size, pre-allocated 0 pages
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -142,7 +142,7 @@
 EXT4-fs (vda2): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 254:2.
 devtmpfs: mounted
-Freeing unused kernel memory: 676K (ffffffc0008ce000 - ffffffc000977000)
+Freeing unused kernel memory: 676K (ffffffc0008cf000 - ffffffc000978000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 bash-4.3# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-12-16 23:24:09.069694397 -0800
+++ /tmp/foob	2015-12-16 23:24:09.069694397 -0800
@@ -37,10 +37,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #############################################################
-	 3.3 MiB/s
+	 ################################################################
+	 3.1 MiB/s
 done
-Bytes transferred = 6615680 (64f280 hex)
+Bytes transferred = 6653024 (658460 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -51,7 +51,7 @@
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x64f280 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x658460 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2015-12-16 23:24:09.073694394 -0800
+++ /tmp/foob	2015-12-16 23:24:09.077694390 -0800
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 ################################
-	 3.3 MiB/s
+	 3.1 MiB/s
 done
-Bytes transferred = 5235232 (4fe220 hex)
+Bytes transferred = 5235024 (4fe150 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 2.6 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4fe220 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4fe150 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.4.0-rc5-next-20151216-61915-g5703684 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Dec 15 22:18:09 PST 2015
+Linux version 4.4.0-rc5-next-20151217-62145-g040e3da (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Dec 16 23:18:07 PST 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -71,7 +71,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0a26dc0, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0a26f00, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -92,9 +92,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0976310   (9657 kB)
+      .text : 0xc0008000 - 0xc0976330   (9657 kB)
       .init : 0xc0977000 - 0xc09cc000   ( 340 kB)
-      .data : 0xc09cc000 - 0xc0a288e0   ( 371 kB)
+      .data : 0xc09cc000 - 0xc0a28a20   ( 371 kB)
        .bss : 0xc0a2b000 - 0xc0a86d90   ( 368 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -291,7 +291,7 @@
 WARNING: CPU: 0 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-rc5-next-20151216-61915-g5703684 #1
+CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-rc5-next-20151217-62145-g040e3da #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0017248>] (unwind_backtrace) from [<c0012870>] (show_stack+0x10/0x14)
@@ -308,28 +308,28 @@
 [<c001c250>] (__alloc_from_contiguous) from [<c001cf18>] (__dma_alloc+0x228/0x2ac)
 [<c001cf18>] (__dma_alloc) from [<c001d020>] (arm_dma_alloc+0x48/0x54)
 [<c001d020>] (arm_dma_alloc) from [<c00ed064>] (dma_pool_alloc+0x28c/0x304)
-[<c00ed064>] (dma_pool_alloc) from [<c03f92d0>] (ehci_qh_alloc+0x3c/0xb4)
-[<c03f92d0>] (ehci_qh_alloc) from [<c03fd4bc>] (ehci_setup+0x154/0x478)
-[<c03fd4bc>] (ehci_setup) from [<c0401204>] (tegra_ehci_reset+0xc/0x50)
-[<c0401204>] (tegra_ehci_reset) from [<c03e4cbc>] (usb_add_hcd+0x2fc/0x804)
-[<c03e4cbc>] (usb_add_hcd) from [<c0400b7c>] (tegra_ehci_probe+0x3b8/0x504)
-[<c0400b7c>] (tegra_ehci_probe) from [<c03336c4>] (platform_drv_probe+0x50/0xac)
-[<c03336c4>] (platform_drv_probe) from [<c03320e4>] (driver_probe_device+0x1f4/0x2b0)
-[<c03320e4>] (driver_probe_device) from [<c0330698>] (bus_for_each_drv+0x44/0x8c)
-[<c0330698>] (bus_for_each_drv) from [<c0331e40>] (__device_attach+0xa0/0x104)
-[<c0331e40>] (__device_attach) from [<c0331514>] (bus_probe_device+0x84/0x8c)
-[<c0331514>] (bus_probe_device) from [<c0331948>] (deferred_probe_work_func+0x60/0x8c)
-[<c0331948>] (deferred_probe_work_func) from [<c003a804>] (process_one_work+0x120/0x328)
+[<c00ed064>] (dma_pool_alloc) from [<c03f92e4>] (ehci_qh_alloc+0x3c/0xb4)
+[<c03f92e4>] (ehci_qh_alloc) from [<c03fd4d0>] (ehci_setup+0x154/0x478)
+[<c03fd4d0>] (ehci_setup) from [<c0401218>] (tegra_ehci_reset+0xc/0x50)
+[<c0401218>] (tegra_ehci_reset) from [<c03e4cd0>] (usb_add_hcd+0x2fc/0x804)
+[<c03e4cd0>] (usb_add_hcd) from [<c0400b90>] (tegra_ehci_probe+0x3b8/0x504)
+[<c0400b90>] (tegra_ehci_probe) from [<c0333708>] (platform_drv_probe+0x50/0xac)
+[<c0333708>] (platform_drv_probe) from [<c0332128>] (driver_probe_device+0x1f4/0x2b0)
+[<c0332128>] (driver_probe_device) from [<c03306dc>] (bus_for_each_drv+0x44/0x8c)
+[<c03306dc>] (bus_for_each_drv) from [<c0331e84>] (__device_attach+0xa0/0x104)
+[<c0331e84>] (__device_attach) from [<c0331558>] (bus_probe_device+0x84/0x8c)
+[<c0331558>] (bus_probe_device) from [<c033198c>] (deferred_probe_work_func+0x60/0x8c)
+[<c033198c>] (deferred_probe_work_func) from [<c003a804>] (process_one_work+0x120/0x328)
 [<c003a804>] (process_one_work) from [<c003aa70>] (worker_thread+0x2c/0x4b4)
 [<c003aa70>] (worker_thread) from [<c003fbac>] (kthread+0xd8/0xf4)
 [<c003fbac>] (kthread) from [<c000f938>] (ret_from_fork+0x14/0x3c)
----[ end trace 0d5bc3c05d2e4dae ]---
+---[ end trace f2777f27716c36bc ]---
 tegra-ehci 7d008000.usb: irq 339, io mem 0x7d008000
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-06-27 15:18:09 UTC (962119089)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-06-28 16:18:07 UTC (962209087)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-12-16 23:24:09.081694387 -0800
+++ /tmp/foob	2015-12-16 23:24:09.081694387 -0800
@@ -29,37 +29,31 @@
 Tegra124 (Jetson TK1) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash'
 Tegra124 (Jetson TK1) # setenv ipaddr 192.168.100.102; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 zImage.multi_v7_defconfig
-Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'zImage.multi_v7_defconfig'.
 Load address: 0x82000000
-Loading: *EHCI timed out on TD - token=0x8008d80
-T EHCI timed out on TD - token=0x8008d80
-T EHCI timed out on TD - token=0x8008d80
-T EHCI timed out on TD - token=0x8008d80
-T #################################################################
+Loading: *#################################################################
 	 #################################################################
 	 #################################################################
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #############################################################
-	 293.9 KiB/s
+	 ################################################################
+	 3.3 MiB/s
 done
-Bytes transferred = 6615680 (64f280 hex)
+Bytes transferred = 6653024 (658460 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
-Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.1 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x64f280 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x658460 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012a89
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2015-12-16 23:24:09.089694379 -0800
+++ /tmp/foob	2015-12-16 23:24:09.089694379 -0800
@@ -29,6 +29,7 @@
 Tegra124 (Jetson TK1) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash'
 Tegra124 (Jetson TK1) # setenv ipaddr 192.168.100.102; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 zImage.tegra_defconfig
+Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'zImage.tegra_defconfig'.
@@ -39,20 +40,20 @@
 	 #################################################################
 	 #################################################################
 	 ################################
-	 2.8 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 5235232 (4fe220 hex)
+Bytes transferred = 5235024 (4fe150 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 2.8 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x4fe220 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x4fe150 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012a89
@@ -63,7 +64,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.4.0-rc5-next-20151216-61915-g5703684 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Dec 15 22:18:09 PST 2015
+Linux version 4.4.0-rc5-next-20151217-62145-g040e3da (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Dec 16 23:18:07 PST 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -73,7 +74,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0a26dc0, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0a26f00, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -94,9 +95,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0976310   (9657 kB)
+      .text : 0xc0008000 - 0xc0976330   (9657 kB)
       .init : 0xc0977000 - 0xc09cc000   ( 340 kB)
-      .data : 0xc09cc000 - 0xc0a288e0   ( 371 kB)
+      .data : 0xc09cc000 - 0xc0a28a20   ( 371 kB)
        .bss : 0xc0a2b000 - 0xc0a86d90   ( 368 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -300,42 +301,16 @@
 tegra-pcie 1003000.pcie-controller: link 0 down, ignoring
 tegra-pcie 1003000.pcie-controller: probing port 1, using 1 lanes
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000000
+tegra-pcie 1003000.pcie-controller: link 1 down, retrying
+tegra-pcie 1003000.pcie-controller: link 1 down, retrying
+tegra-pcie 1003000.pcie-controller: link 1 down, retrying
+tegra-pcie 1003000.pcie-controller: link 1 down, ignoring
 tegra-pcie 1003000.pcie-controller: PCI host bridge to bus 0000:00
 pci_bus 0000:00: root bus resource [mem 0x13000000-0x1fffffff]
 pci_bus 0000:00: root bus resource [mem 0x20000000-0x3fffffff pref]
 pci_bus 0000:00: root bus resource [bus 00-ff]
 pci_bus 0000:00: root bus resource [io  0x1000-0xffff]
-pci 0000:00:02.0: [10de:0e13] type 01 class 0x060400
-pci 0000:00:02.0: PME# supported from D0 D1 D2 D3hot D3cold
-PCI: bus0: Fast back to back transfers disabled
-pci 0000:00:02.0: bridge configuration invalid ([bus 00-00]), reconfiguring
-pci 0000:01:00.0: [10ec:8168] type 00 class 0x020000
-pci 0000:01:00.0: reg 0x10: [io  0x0000-0x00ff]
-pci 0000:01:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit]
-pci 0000:01:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
-pci 0000:01:00.0: supports D1 D2
-pci 0000:01:00.0: PME# supported from D0 D1 D2 D3hot D3cold
-PCI: bus1: Fast back to back transfers disabled
-pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
-pci 0000:00:02.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
-pci 0000:00:02.0: BAR 9: assigned [mem 0x20000000-0x200fffff 64bit pref]
-pci 0000:00:02.0: BAR 7: assigned [io  0x1000-0x1fff]
-pci 0000:01:00.0: BAR 4: assigned [mem 0x20000000-0x20003fff 64bit pref]
-pci 0000:01:00.0: BAR 2: assigned [mem 0x13000000-0x13000fff 64bit]
-pci 0000:01:00.0: BAR 0: assigned [io  0x1000-0x10ff]
-pci 0000:00:02.0: PCI bridge to [bus 01]
-pci 0000:00:02.0:   bridge window [io  0x1000-0x1fff]
-pci 0000:00:02.0:   bridge window [mem 0x13000000-0x130fffff]
-pci 0000:00:02.0:   bridge window [mem 0x20000000-0x200fffff 64bit pref]
-pci 0000:00:02.0: nv_msi_ht_cap_quirk didn't locate host bridge
-pcieport 0000:00:02.0: enabling device (0140 -> 0143)
-pcieport 0000:00:02.0: Signaling PME through PCIe PME interrupt
-pci 0000:01:00.0: Signaling PME through PCIe PME interrupt
-pcie_pme 0000:00:02.0:pcie01: service driver pcie_pme loaded
-r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
-r8169 0000:01:00.0: enabling device (0140 -> 0143)
-r8169 0000:01:00.0 eth0: RTL8168g/8111g at 0xf0952000, 00:04:4b:25:b3:b4, XID 0c000800 IRQ 388
-r8169 0000:01:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
+PCI: bus0: Fast back to back transfers enabled
 +5V_HDMI_CON: supplied by +5V_SYS
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 +3.3V_RUN: supplied by +3.3V_SYS
@@ -354,7 +329,7 @@
 WARNING: CPU: 0 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-rc5-next-20151216-61915-g5703684 #1
+CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-rc5-next-20151217-62145-g040e3da #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0017248>] (unwind_backtrace) from [<c0012870>] (show_stack+0x10/0x14)
@@ -370,24 +345,24 @@
 [<c00f27c8>] (cma_alloc) from [<c001c250>] (__alloc_from_contiguous+0x38/0xdc)
 [<c001c250>] (__alloc_from_contiguous) from [<c001cf18>] (__dma_alloc+0x228/0x2ac)
 [<c001cf18>] (__dma_alloc) from [<c001d020>] (arm_dma_alloc+0x48/0x54)
-[<c001d020>] (arm_dma_alloc) from [<c03422d8>] (dmam_alloc_coherent+0x8c/0xf0)
-[<c03422d8>] (dmam_alloc_coherent) from [<c038812c>] (ahci_port_start+0xd0/0x1e0)
-[<c038812c>] (ahci_port_start) from [<c036fe04>] (ata_host_start+0xf8/0x1d4)
-[<c036fe04>] (ata_host_start) from [<c0374d98>] (ata_host_activate+0x20/0x134)
-[<c0374d98>] (ata_host_activate) from [<c03875d4>] (ahci_host_activate+0x3c/0x11c)
-[<c03875d4>] (ahci_host_activate) from [<c038997c>] (ahci_platform_init_host+0x1cc/0x3c8)
-[<c038997c>] (ahci_platform_init_host) from [<c0389648>] (tegra_ahci_probe+0x404/0x4c4)
-[<c0389648>] (tegra_ahci_probe) from [<c03336c4>] (platform_drv_probe+0x50/0xac)
-[<c03336c4>] (platform_drv_probe) from [<c03320e4>] (driver_probe_device+0x1f4/0x2b0)
-[<c03320e4>] (driver_probe_device) from [<c0330698>] (bus_for_each_drv+0x44/0x8c)
-[<c0330698>] (bus_for_each_drv) from [<c0331e40>] (__device_attach+0xa0/0x104)
-[<c0331e40>] (__device_attach) from [<c0331514>] (bus_probe_device+0x84/0x8c)
-[<c0331514>] (bus_probe_device) from [<c0331948>] (deferred_probe_work_func+0x60/0x8c)
-[<c0331948>] (deferred_probe_work_func) from [<c003a804>] (process_one_work+0x120/0x328)
+[<c001d020>] (arm_dma_alloc) from [<c034231c>] (dmam_alloc_coherent+0x8c/0xf0)
+[<c034231c>] (dmam_alloc_coherent) from [<c0388148>] (ahci_port_start+0xd0/0x1e0)
+[<c0388148>] (ahci_port_start) from [<c036fe20>] (ata_host_start+0xf8/0x1d4)
+[<c036fe20>] (ata_host_start) from [<c0374db4>] (ata_host_activate+0x20/0x134)
+[<c0374db4>] (ata_host_activate) from [<c03875f0>] (ahci_host_activate+0x3c/0x11c)
+[<c03875f0>] (ahci_host_activate) from [<c0389998>] (ahci_platform_init_host+0x1cc/0x3c8)
+[<c0389998>] (ahci_platform_init_host) from [<c0389664>] (tegra_ahci_probe+0x404/0x4c4)
+[<c0389664>] (tegra_ahci_probe) from [<c0333708>] (platform_drv_probe+0x50/0xac)
+[<c0333708>] (platform_drv_probe) from [<c0332128>] (driver_probe_device+0x1f4/0x2b0)
+[<c0332128>] (driver_probe_device) from [<c03306dc>] (bus_for_each_drv+0x44/0x8c)
+[<c03306dc>] (bus_for_each_drv) from [<c0331e84>] (__device_attach+0xa0/0x104)
+[<c0331e84>] (__device_attach) from [<c0331558>] (bus_probe_device+0x84/0x8c)
+[<c0331558>] (bus_probe_device) from [<c033198c>] (deferred_probe_work_func+0x60/0x8c)
+[<c033198c>] (deferred_probe_work_func) from [<c003a804>] (process_one_work+0x120/0x328)
 [<c003a804>] (process_one_work) from [<c003aa70>] (worker_thread+0x2c/0x4b4)
 [<c003aa70>] (worker_thread) from [<c003fbac>] (kthread+0xd8/0xf4)
 [<c003fbac>] (kthread) from [<c000f938>] (ret_from_fork+0x14/0x3c)
----[ end trace 1e4adeba59919648 ]---
+---[ end trace c1348b3a5e11625b ]---
 scsi host0: tegra-ahci
 scsi host1: tegra-ahci
 ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 341
@@ -409,7 +384,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:01:33 UTC (946684893)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:01:19 UTC (946684879)
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
@@ -417,4 +392,4 @@
 +3.3V_RUN_TOUCH: disabling
 +2.8V_RUN_CAM_AF: disabling
 +1.8V_RUN_VPP_FUSE: disabling
-+USB0
\ No newline at end of file
++USB0ÿ
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-12-16 23:24:09.097694372 -0800
+++ /tmp/foob	2015-12-16 23:24:09.097694372 -0800
@@ -45,10 +45,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #############################################################
+	 ################################################################
 	 4.1 MiB/s
 done
-Bytes transferred = 6615680 (64f280 hex)
+Bytes transferred = 6653024 (658460 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -59,7 +59,7 @@
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x64f280 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x658460 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2015-12-16 23:24:09.105694365 -0800
+++ /tmp/foob	2015-12-16 23:24:09.105694365 -0800
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -45,9 +45,9 @@
 	 #################################################################
 	 #################################################################
 	 ################################
-	 3.7 MiB/s
+	 3.5 MiB/s
 done
-Bytes transferred = 5235232 (4fe220 hex)
+Bytes transferred = 5235024 (4fe150 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x4fe220 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x4fe150 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -69,7 +69,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.4.0-rc5-next-20151216-61915-g5703684 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Dec 15 22:18:09 PST 2015
+Linux version 4.4.0-rc5-next-20151217-62145-g040e3da (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Dec 16 23:18:07 PST 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -78,7 +78,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0a26dc0, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0a26f00, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -99,9 +99,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0976310   (9657 kB)
+      .text : 0xc0008000 - 0xc0976330   (9657 kB)
       .init : 0xc0977000 - 0xc09cc000   ( 340 kB)
-      .data : 0xc09cc000 - 0xc0a288e0   ( 371 kB)
+      .data : 0xc09cc000 - 0xc0a28a20   ( 371 kB)
        .bss : 0xc0a2b000 - 0xc0a86d90   ( 368 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -305,11 +305,11 @@
 NET: Registered protocol family 10
 mmc0: host does not support reading read-only switch, assuming write-enable
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
-mmc0: new high speed SDHC card at address b368
 mip6: Mobile IPv6
-sit: IPv6 over IPv4 tunneling driver
+mmc0: new high speed SDHC card at address b368
 mmc1: new high speed SD card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
+sit: IPv6 over IPv4 tunneling driver
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk0: p1
  mmcblk1: p1 p2
@@ -338,4 +338,4 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-27 03:03:54 UTC (1419649434)
+rtc-em3027 2-0056: setting system clock to 2014-12-27 03:07:46 UTC (1419649666)
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-12-16 23:24:09.109694362 -0800
+++ /tmp/foob	2015-12-16 23:24:09.113694358 -0800
@@ -37,21 +37,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #############################################################
+	 ################################################################
 	 3.3 MiB/s
 done
-Bytes transferred = 6615680 (64f280 hex)
+Bytes transferred = 6653024 (658460 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.5 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x64f280 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x658460 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2015-12-16 23:24:09.117694353 -0800
+++ /tmp/foob	2015-12-16 23:24:09.121694350 -0800
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 ################################
-	 3.4 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 5235232 (4fe220 hex)
+Bytes transferred = 5235024 (4fe150 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.3 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4fe220 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4fe150 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.4.0-rc5-next-20151216-61915-g5703684 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Dec 15 22:18:09 PST 2015
+Linux version 4.4.0-rc5-next-20151217-62145-g040e3da (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Dec 16 23:18:07 PST 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -70,7 +70,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0a26dc0, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0a26f00, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -91,9 +91,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0976310   (9657 kB)
+      .text : 0xc0008000 - 0xc0976330   (9657 kB)
       .init : 0xc0977000 - 0xc09cc000   ( 340 kB)
-      .data : 0xc09cc000 - 0xc0a288e0   ( 371 kB)
+      .data : 0xc09cc000 - 0xc0a28a20   ( 371 kB)
        .bss : 0xc0a2b000 - 0xc0a86d90   ( 368 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -340,7 +340,7 @@
 WARNING: CPU: 0 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-rc5-next-20151216-61915-g5703684 #1
+CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-rc5-next-20151217-62145-g040e3da #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0017248>] (unwind_backtrace) from [<c0012870>] (show_stack+0x10/0x14)
@@ -357,22 +357,22 @@
 [<c001c250>] (__alloc_from_contiguous) from [<c001cf18>] (__dma_alloc+0x228/0x2ac)
 [<c001cf18>] (__dma_alloc) from [<c001d020>] (arm_dma_alloc+0x48/0x54)
 [<c001d020>] (arm_dma_alloc) from [<c00ed064>] (dma_pool_alloc+0x28c/0x304)
-[<c00ed064>] (dma_pool_alloc) from [<c03f92d0>] (ehci_qh_alloc+0x3c/0xb4)
-[<c03f92d0>] (ehci_qh_alloc) from [<c03fd4bc>] (ehci_setup+0x154/0x478)
-[<c03fd4bc>] (ehci_setup) from [<c0401204>] (tegra_ehci_reset+0xc/0x50)
-[<c0401204>] (tegra_ehci_reset) from [<c03e4cbc>] (usb_add_hcd+0x2fc/0x804)
-[<c03e4cbc>] (usb_add_hcd) from [<c0400b7c>] (tegra_ehci_probe+0x3b8/0x504)
-[<c0400b7c>] (tegra_ehci_probe) from [<c03336c4>] (platform_drv_probe+0x50/0xac)
-[<c03336c4>] (platform_drv_probe) from [<c03320e4>] (driver_probe_device+0x1f4/0x2b0)
-[<c03320e4>] (driver_probe_device) from [<c0330698>] (bus_for_each_drv+0x44/0x8c)
-[<c0330698>] (bus_for_each_drv) from [<c0331e40>] (__device_attach+0xa0/0x104)
-[<c0331e40>] (__device_attach) from [<c0331514>] (bus_probe_device+0x84/0x8c)
-[<c0331514>] (bus_probe_device) from [<c0331948>] (deferred_probe_work_func+0x60/0x8c)
-[<c0331948>] (deferred_probe_work_func) from [<c003a804>] (process_one_work+0x120/0x328)
+[<c00ed064>] (dma_pool_alloc) from [<c03f92e4>] (ehci_qh_alloc+0x3c/0xb4)
+[<c03f92e4>] (ehci_qh_alloc) from [<c03fd4d0>] (ehci_setup+0x154/0x478)
+[<c03fd4d0>] (ehci_setup) from [<c0401218>] (tegra_ehci_reset+0xc/0x50)
+[<c0401218>] (tegra_ehci_reset) from [<c03e4cd0>] (usb_add_hcd+0x2fc/0x804)
+[<c03e4cd0>] (usb_add_hcd) from [<c0400b90>] (tegra_ehci_probe+0x3b8/0x504)
+[<c0400b90>] (tegra_ehci_probe) from [<c0333708>] (platform_drv_probe+0x50/0xac)
+[<c0333708>] (platform_drv_probe) from [<c0332128>] (driver_probe_device+0x1f4/0x2b0)
+[<c0332128>] (driver_probe_device) from [<c03306dc>] (bus_for_each_drv+0x44/0x8c)
+[<c03306dc>] (bus_for_each_drv) from [<c0331e84>] (__device_attach+0xa0/0x104)
+[<c0331e84>] (__device_attach) from [<c0331558>] (bus_probe_device+0x84/0x8c)
+[<c0331558>] (bus_probe_device) from [<c033198c>] (deferred_probe_work_func+0x60/0x8c)
+[<c033198c>] (deferred_probe_work_func) from [<c003a804>] (process_one_work+0x120/0x328)
 [<c003a804>] (process_one_work) from [<c003aa70>] (worker_thread+0x2c/0x4b4)
 [<c003aa70>] (worker_thread) from [<c003fbac>] (kthread+0xd8/0xf4)
 [<c003fbac>] (kthread) from [<c000f938>] (ret_from_fork+0x14/0x3c)
----[ end trace 920c7986aab23888 ]---
+---[ end trace c1e380384c37b6c0 ]---
 tegra-ehci 7d004000.usb: irq 343, io mem 0x7d004000
 tegra-ehci 7d004000.usb: USB 2.0 started, EHCI 1.00
 hub 1-0:1.0: USB hub found
