Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-02-04 20:24:26.681018989 -0800
+++ /tmp/foob	2016-02-04 20:24:26.685018988 -0800
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.5.0-rc2-next-20160204+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Feb 3 20:19:45 PST 2016
+Linux version 4.5.0-rc2-next-20160205+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Feb 4 20:19:40 PST 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -92,7 +92,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.488:1): initialized
+audit: type=2000 audit(0.452:1): initialized
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -139,7 +139,7 @@
 9pnet: Installing 9P2000 support
 Key type dns_resolver registered
 registered taskstats version 1
-rtc-pl031 9010000.pl031: setting system clock to 2016-02-04 04:20:14 UTC (1454559614)
+rtc-pl031 9010000.pl031: setting system clock to 2016-02-05 04:20:09 UTC (1454646009)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-02-04 20:24:26.689018985 -0800
+++ /tmp/foob	2016-02-04 20:24:26.693018984 -0800
@@ -39,9 +39,9 @@
 	 #################################################################
 	 #################################################################
 	 ########
-	 3.4 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 6789992 (679b68 hex)
+Bytes transferred = 6790424 (679d18 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -52,7 +52,7 @@
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x679b68 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x679d18 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.5.0-rc2-next-20160204-70112-g6d3d7d4 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Feb 3 20:17:52 PST 2016
+Linux version 4.5.0-rc2-next-20160205-70441-gbc91a31 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Feb 4 20:17:51 PST 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c10746c0, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1074740, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0e4cf10   (12564 kB)
+      .text : 0xc0208000 - 0xc0e4cf80   (12564 kB)
       .init : 0xc0e4d000 - 0xc0f60000   (1100 kB)
-      .data : 0xc0f60000 - 0xc1076680   (1114 kB)
-       .bss : 0xc1079000 - 0xc10cf620   ( 346 kB)
+      .data : 0xc0f60000 - 0xc1076700   (1114 kB)
+       .bss : 0xc1079000 - 0xc10cf6a0   ( 346 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -295,7 +295,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-15 09:27:56 UTC (947928476)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-16 09:27:58 UTC (948014878)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -307,4 +307,4 @@
 Freeing unused kernel memory: 1100K (c0e4d000 - c0f60000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.214141] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.224364] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-02-04 20:24:26.697018982 -0800
+++ /tmp/foob	2016-02-04 20:24:26.701018981 -0800
@@ -39,18 +39,18 @@
 	 ##################################
 	 2.9 MiB/s
 done
-Bytes transferred = 5264584 (5054c8 hex)
+Bytes transferred = 5264704 (505540 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.6 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5054c8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x505540 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.5.0-rc2-next-20160204-70112-g6d3d7d4 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Feb 3 20:18:14 PST 2016
+Linux version 4.5.0-rc2-next-20160205-70441-gbc91a31 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Feb 4 20:18:13 PST 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002172K/2096128K available (7229K kernel code, 372K rwdata, 2492K rodata, 340K init, 367K bss, 28420K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2002172K/2096128K available (7225K kernel code, 372K rwdata, 2492K rodata, 344K init, 367K bss, 28420K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc098686c   (9723 kB)
-      .init : 0xc0987000 - 0xc09dc000   ( 340 kB)
+      .text : 0xc0008000 - 0xc098588c   (9719 kB)
+      .init : 0xc0986000 - 0xc09dc000   ( 344 kB)
       .data : 0xc09dc000 - 0xc0a39220   ( 373 kB)
-       .bss : 0xc0a3c000 - 0xc0a97d88   ( 368 kB)
+       .bss : 0xc0a3c000 - 0xc0a97dcc   ( 368 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -292,7 +292,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-15 09:29:40 UTC (947928580)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-16 09:29:40 UTC (948014980)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -303,7 +303,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 340K (c0987000 - c09dc000)
+Freeing unused kernel memory: 344K (c0986000 - c09dc000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.547458] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.527528] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-02-04 20:24:26.705018979 -0800
+++ /tmp/foob	2016-02-04 20:24:26.709018977 -0800
@@ -41,20 +41,20 @@
 	 #################################################################
 	 #################################################################
 	 ########
-	 3.4 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 6789992 (679b68 hex)
+Bytes transferred = 6790424 (679d18 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.2 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 64218 (fada hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x679b68 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x679d18 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012ad9
@@ -62,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.5.0-rc2-next-20160204-70112-g6d3d7d4 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Feb 3 20:17:52 PST 2016
+Linux version 4.5.0-rc2-next-20160205-70441-gbc91a31 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Feb 4 20:17:51 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -71,7 +71,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c10746c0, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1074740, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -92,10 +92,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0e4cf10   (12564 kB)
+      .text : 0xc0208000 - 0xc0e4cf80   (12564 kB)
       .init : 0xc0e4d000 - 0xc0f60000   (1100 kB)
-      .data : 0xc0f60000 - 0xc1076680   (1114 kB)
-       .bss : 0xc1079000 - 0xc10cf620   ( 346 kB)
+      .data : 0xc0f60000 - 0xc1076700   (1114 kB)
+       .bss : 0xc1079000 - 0xc10cf6a0   ( 346 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-02-04 20:24:26.713018976 -0800
+++ /tmp/foob	2016-02-04 20:24:26.717018974 -0800
@@ -39,9 +39,9 @@
 	 #################################################################
 	 #################################################################
 	 ##################################
-	 3.2 MiB/s
+	 2.9 MiB/s
 done
-Bytes transferred = 5264584 (5054c8 hex)
+Bytes transferred = 5264704 (505540 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -52,7 +52,7 @@
 done
 Bytes transferred = 64218 (fada hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5054c8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x505540 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012ad9
@@ -61,7 +61,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.5.0-rc2-next-20160204-70112-g6d3d7d4 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Feb 3 20:18:14 PST 2016
+Linux version 4.5.0-rc2-next-20160205-70441-gbc91a31 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Feb 4 20:18:13 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -84,7 +84,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002064K/2096128K available (7229K kernel code, 372K rwdata, 2492K rodata, 340K init, 367K bss, 28528K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2002064K/2096128K available (7225K kernel code, 372K rwdata, 2492K rodata, 344K init, 367K bss, 28528K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,10 +92,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc098686c   (9723 kB)
-      .init : 0xc0987000 - 0xc09dc000   ( 340 kB)
+      .text : 0xc0008000 - 0xc098588c   (9719 kB)
+      .init : 0xc0986000 - 0xc09dc000   ( 344 kB)
       .data : 0xc09dc000 - 0xc0a39220   ( 373 kB)
-       .bss : 0xc0a3c000 - 0xc0a97d88   ( 368 kB)
+       .bss : 0xc0a3c000 - 0xc0a97dcc   ( 368 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -347,7 +347,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:33 UTC (946684953)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:01:57 UTC (946684917)
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
@@ -361,12 +361,12 @@
   #1: NVIDIA Tegra Jetson TK1
 ata1: SATA link down (SStatus 0 SControl 300)
 EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 340K (c0987000 - c09dc000)
+Freeing unused kernel memory: 344K (c0986000 - c09dc000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-02-04 20:24:26.721018972 -0800
+++ /tmp/foob	2016-02-04 20:24:26.721018972 -0800
@@ -49,18 +49,18 @@
 	 ########
 	 4.1 MiB/s
 done
-Bytes transferred = 6789992 (679b68 hex)
+Bytes transferred = 6790424 (679d18 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 4 MiB/s
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x679b68 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x679d18 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.5.0-rc2-next-20160204-70112-g6d3d7d4 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Feb 3 20:17:52 PST 2016
+Linux version 4.5.0-rc2-next-20160205-70441-gbc91a31 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Feb 4 20:17:51 PST 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -76,7 +76,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c10746c0, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c1074740, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -97,10 +97,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0e4cf10   (12564 kB)
+      .text : 0xc0208000 - 0xc0e4cf80   (12564 kB)
       .init : 0xc0e4d000 - 0xc0f60000   (1100 kB)
-      .data : 0xc0f60000 - 0xc1076680   (1114 kB)
-       .bss : 0xc1079000 - 0xc10cf620   ( 346 kB)
+      .data : 0xc0f60000 - 0xc1076700   (1114 kB)
+       .bss : 0xc1079000 - 0xc10cf6a0   ( 346 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -322,27 +322,28 @@
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
-mmc0: host does not support reading read-only switch, assuming write-enable
-mmc0: new high speed SDHC card at address b368
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
- mmcblk0: p1
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
-mmc1: new high speed SD card at address b368
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-28 06:15:12 UTC (1419747312)
- mmcblk1: p1 p2
-EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
+rtc-em3027 2-0056: setting system clock to 2014-12-28 06:17:18 UTC (1419747438)
+mmc0: host does not support reading read-only switch, assuming write-enable
+Waiting for root device PARTUUID=11eb23c1-01...
+mmc1: new high speed SD card at address b368
+mmc0: new high speed SDHC card at address b368
+mmcblk0: mmc1:b368 NCard 1.83 GiB (ro)
+mmcblk1: mmc0:b368 NCard 3.70 GiB 
+ mmcblk1: p1
+ mmcblk0: p1 p2
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
+EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
-EXT4-fs (mmcblk0p1): recovery complete
-EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:1.
+EXT4-fs (mmcblk1p1): recovery complete
+EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
+VFS: Mounted root (ext3 filesystem) on device 179:17.
 devtmpfs: mounted
 Freeing unused kernel memory: 1100K (c0e4d000 - c0f60000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-02-04 20:24:26.729018969 -0800
+++ /tmp/foob	2016-02-04 20:24:26.729018969 -0800
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -45,20 +45,20 @@
 	 #################################################################
 	 #################################################################
 	 ##################################
-	 3.5 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 5264584 (5054c8 hex)
+Bytes transferred = 5264704 (505540 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x5054c8 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x505540 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.5.0-rc2-next-20160204-70112-g6d3d7d4 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Feb 3 20:18:14 PST 2016
+Linux version 4.5.0-rc2-next-20160205-70441-gbc91a31 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Feb 4 20:18:13 PST 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -89,7 +89,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 962988K/1048576K available (7229K kernel code, 372K rwdata, 2492K rodata, 340K init, 367K bss, 20052K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 962988K/1048576K available (7225K kernel code, 372K rwdata, 2492K rodata, 344K init, 367K bss, 20052K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -97,10 +97,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc098686c   (9723 kB)
-      .init : 0xc0987000 - 0xc09dc000   ( 340 kB)
+      .text : 0xc0008000 - 0xc098588c   (9719 kB)
+      .init : 0xc0986000 - 0xc09dc000   ( 344 kB)
       .data : 0xc09dc000 - 0xc0a39220   ( 373 kB)
-       .bss : 0xc0a3c000 - 0xc0a97d88   ( 368 kB)
+       .bss : 0xc0a3c000 - 0xc0a97dcc   ( 368 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
@@ -306,8 +306,8 @@
 tlv320aic23-codec 2-001a: ASoC: no dapm match for MICIN --> NULL --> Mic Input
 tlv320aic23-codec 2-001a: ASoC: Failed to add route MICIN -> NULL -> Mic Input
 tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
 NET: Registered protocol family 10
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 mip6: Mobile IPv6
 sit: IPv6 over IPv4 tunneling driver
 NET: Registered protocol family 17
@@ -317,10 +317,10 @@
 mmc1: new high speed SD card at address b368
 mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc1:b368 NCard 1.83 GiB (ro)
-mmcblk1: mmc0:b368 NCard 3.70 GiB 
- mmcblk1: p1
  mmcblk0: p1 p2
+mmcblk1: mmc0:b368 NCard 3.70 GiB 
 NET: Registered protocol family 29
+ mmcblk1: p1
 can: raw protocol (rev 20120528)
 can: broadcast manager protocol (rev 20120528 t)
 can: netlink gateway (rev 20130117) max_hops=1
@@ -334,15 +334,15 @@
 Registering SWP/SWPB emulation handler
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
+tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
-tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-28 06:16:14 UTC (1419747374)
+rtc-em3027 2-0056: setting system clock to 2014-12-28 06:18:19 UTC (1419747499)
 ALSA device list:
   #0: tegra-trimslice
 EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
@@ -352,7 +352,7 @@
 EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:17.
 devtmpfs: mounted
-Freeing unused kernel memory: 340K (c0987000 - c09dc000)
+Freeing unused kernel memory: 344K (c0986000 - c09dc000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2016-02-04 20:24:26.737018966 -0800
+++ /tmp/foob	2016-02-04 20:24:26.737018966 -0800
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 841180 us
-Basic modules init took 350303 us
+Tegraboot started after 797949 us
+Basic modules init took 348945 us
 Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14392 us
-Carveout took 252727 us
-CPU initialization took 92315 us
-Total time taken by TegraBoot 709737 us
+Next stage binary read took 14366 us
+Carveout took 252716 us
+CPU initialization took 92304 us
+Total time taken by TegraBoot 708331 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.679] LPDDR4 Training: Number of tables = 10
+64b[0001.636] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -119,9 +119,9 @@
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
 10762752 bytes read in 697 ms (14.7 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-37136 bytes read in 453 ms (79.1 KiB/s)
+37136 bytes read in 460 ms (78.1 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 494 ms (3.1 MiB/s)
+1627787 bytes read in 495 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.5.0-rc2-next-20160204+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Feb 3 20:19:45 PST 2016
+Linux version 4.5.0-rc2-next-20160205+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Feb 4 20:19:40 PST 2016
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: Early serial console at MMIO32 0x70006000 (options '')
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-02-04 20:24:26.745018962 -0800
+++ /tmp/foob	2016-02-04 20:24:26.745018962 -0800
@@ -41,18 +41,18 @@
 	 ########
 	 3.4 MiB/s
 done
-Bytes transferred = 6789992 (679b68 hex)
+Bytes transferred = 6790424 (679d18 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.3 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x679b68 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x679d18 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.5.0-rc2-next-20160204-70112-g6d3d7d4 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Feb 3 20:17:52 PST 2016
+Linux version 4.5.0-rc2-next-20160205-70441-gbc91a31 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Feb 4 20:17:51 PST 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,7 +68,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c10746c0, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1074740, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -89,10 +89,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0e4cf10   (12564 kB)
+      .text : 0xc0208000 - 0xc0e4cf80   (12564 kB)
       .init : 0xc0e4d000 - 0xc0f60000   (1100 kB)
-      .data : 0xc0f60000 - 0xc1076680   (1114 kB)
-       .bss : 0xc1079000 - 0xc10cf620   ( 346 kB)
+      .data : 0xc0f60000 - 0xc1076700   (1114 kB)
+       .bss : 0xc1079000 - 0xc10cf6a0   ( 346 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -282,13 +282,13 @@
 sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
 tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
 mmc1: new high speed MMC card at address 0001
-tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 mmcblk0: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk0boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk0boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk0rpmb: mmc1:0001 SEM16G partition 3 128 KiB
 Primary GPT is invalid, using alternate GPT.
  mmcblk0: p1 p2 p3 p4 p5
+tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -362,4 +362,4 @@
 Freeing unused kernel memory: 1100K (c0e4d000 - c0f60000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.703904] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.733457] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-02-04 20:24:26.749018961 -0800
+++ /tmp/foob	2016-02-04 20:24:26.753018960 -0800
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 ##################################
-	 3 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 5264584 (5054c8 hex)
+Bytes transferred = 5264704 (505540 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.7 MiB/s
+	 3.5 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5054c8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x505540 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.5.0-rc2-next-20160204-70112-g6d3d7d4 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Feb 3 20:18:14 PST 2016
+Linux version 4.5.0-rc2-next-20160205-70441-gbc91a31 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Feb 4 20:18:13 PST 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -81,7 +81,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002092K/2096128K available (7229K kernel code, 372K rwdata, 2492K rodata, 340K init, 367K bss, 28500K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2002092K/2096128K available (7225K kernel code, 372K rwdata, 2492K rodata, 344K init, 367K bss, 28500K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -89,10 +89,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc098686c   (9723 kB)
-      .init : 0xc0987000 - 0xc09dc000   ( 340 kB)
+      .text : 0xc0008000 - 0xc098588c   (9719 kB)
+      .init : 0xc0986000 - 0xc09dc000   ( 344 kB)
       .data : 0xc09dc000 - 0xc0a39220   ( 373 kB)
-       .bss : 0xc0a3c000 - 0xc0a97d88   ( 368 kB)
+       .bss : 0xc0a3c000 - 0xc0a97dcc   ( 368 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -359,7 +359,8 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 340K (c0987000 - c09dc000)
+Freeing unused kernel memory: 344K (c0986000 - c09dc000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.100386] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# 
\ No newline at end of file
