Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2015-11-09 05:08:49.518245700 -0800
+++ /tmp/foob	2015-11-09 05:08:49.522245696 -0800
@@ -1,6 +1,6 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
-Linux version 4.3.0-next-20151109+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Nov 8 19:49:47 PST 2015
+Linux version 4.3.0-next-20151106+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Nov 9 05:03:58 PST 2015
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -10,7 +10,7 @@
 psci: PSCIv0.2 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
 psci: Trusted OS migration not required
-PERCPU: Embedded 15 pages/cpu @ffffffc03efe0000 s24576 r8192 d28672 u61440
+PERCPU: Embedded 16 pages/cpu @ffffffc03efdf000 s24704 r8192 d32640 u65536
 Detected PIPT I-cache on CPU0
 CPU features: enabling workaround for ARM erratum 832075
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 258048
@@ -19,7 +19,7 @@
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
 software IO TLB [mem 0x79c00000-0x7dc00000] (64MB) mapped at [ffffffc039c00000-ffffffc03dbfffff]
-Memory: 938884K/1048576K available (5518K kernel code, 496K rwdata, 2340K rodata, 540K init, 211K bss, 93308K reserved, 16384K cma-reserved)
+Memory: 938832K/1048576K available (5554K kernel code, 504K rwdata, 2340K rodata, 544K init, 211K bss, 93360K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     vmalloc : 0xffffff8000000000 - 0xffffffbdbfff0000   (   246 GB)
     vmemmap : 0xffffffbdc0000000 - 0xffffffbfc0000000   (     8 GB maximum)
@@ -28,9 +28,9 @@
     PCI I/O : 0xffffffbffae00000 - 0xffffffbffbe00000   (    16 MB)
     modules : 0xffffffbffc000000 - 0xffffffc000000000   (    64 MB)
     memory  : 0xffffffc000000000 - 0xffffffc040000000   (  1024 MB)
-      .init : 0xffffffc00082f000 - 0xffffffc0008b6000   (   540 KB)
-      .text : 0xffffffc000080000 - 0xffffffc00082e9f4   (  7867 KB)
-      .data : 0xffffffc0008c7000 - 0xffffffc000943200   (   497 KB)
+      .init : 0xffffffc000838000 - 0xffffffc0008c0000   (   544 KB)
+      .text : 0xffffffc000080000 - 0xffffffc0008379f4   (  7903 KB)
+      .data : 0xffffffc0008d1000 - 0xffffffc00094f200   (   505 KB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 64.
@@ -61,7 +61,7 @@
 NET: Registered protocol family 16
 cpuidle: using governor ladder
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffffffc0008cd000, 1 data @ ffffffc0008cc000)
+vdso: 2 pages (1 code @ ffffffc0008d9000, 1 data @ ffffffc0008d8000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -88,7 +88,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.540:1): initialized
+audit: type=2000 audit(0.580:1): initialized
 HugeTLB registered 2 MB page size, pre-allocated 0 pages
 VFS: Disk quotas dquot_6.6.0
 VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
@@ -134,8 +134,8 @@
 EXT4-fs (vda2): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 254:2.
 devtmpfs: mounted
-Freeing unused kernel memory: 540K (ffffffc00082f000 - ffffffc0008b6000)
-Freeing alternatives memory: 48K (ffffffc0008b6000 - ffffffc0008c2000)
+Freeing unused kernel memory: 544K (ffffffc000838000 - ffffffc0008c0000)
+Freeing alternatives memory: 48K (ffffffc0008c0000 - ffffffc0008cc000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 bash-4.3# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-11-09 05:08:49.526245691 -0800
+++ /tmp/foob	2015-11-09 05:08:49.530245688 -0800
@@ -37,21 +37,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ########################################################
+	 #########################################################
 	 3.3 MiB/s
 done
-Bytes transferred = 6543176 (63d748 hex)
+Bytes transferred = 6552488 (63fba8 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x63d748 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x63fba8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.3.0-next-20151109-54253-g6958584 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Nov 8 19:48:05 PST 2015
+Linux version 4.3.0-next-20151106-54161-gc57d628 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Nov 9 05:02:34 PST 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -67,20 +67,20 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0fc4180, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c0fcc180, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-PERCPU: Embedded 12 pages/cpu @eefa3000 s19136 r8192 d21824 u49152
-pcpu-alloc: s19136 r8192 d21824 u49152 alloc=12*4096
+PERCPU: Embedded 12 pages/cpu @eefa3000 s19200 r8192 d21760 u49152
+pcpu-alloc: s19200 r8192 d21760 u49152 alloc=12*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998584K/2096128K available (8243K kernel code, 1040K rwdata, 3740K rodata, 1044K init, 341K bss, 32008K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998552K/2096128K available (8267K kernel code, 1040K rwdata, 3740K rodata, 1052K init, 341K bss, 32040K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,10 +88,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0dbce14   (11988 kB)
-      .init : 0xc0dbd000 - 0xc0ec2000   (1044 kB)
-      .data : 0xc0ec2000 - 0xc0fc60e0   (1041 kB)
-       .bss : 0xc0fc9000 - 0xc101e468   ( 342 kB)
+      .text : 0xc0208000 - 0xc0dc2e98   (12012 kB)
+      .init : 0xc0dc3000 - 0xc0eca000   (1052 kB)
+      .data : 0xc0eca000 - 0xc0fce0e0   (1041 kB)
+       .bss : 0xc0fd1000 - 0xc1026468   ( 342 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -277,10 +277,10 @@
 Registering SWP/SWPB emulation handler
 vdd-sys-5v0: supplied by vdd_ac_bat
 VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
-mmc1: MAN_BKOPS_EN bit is not set
 vdd_hdmi_5v0: supplied by vdd-sys-5v0
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
+mmc1: MAN_BKOPS_EN bit is not set
 mmc1: new high speed MMC card at address 0001
 mmcblk0: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk0boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
@@ -298,7 +298,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-05-21 12:45:45 UTC (958913145)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-05-21 22:00:09 UTC (958946409)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -307,7 +307,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1044K (c0dbd000 - c0ec2000)
+Freeing unused kernel memory: 1052K (c0dc3000 - c0eca000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.258984] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.254115] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2015-11-09 05:08:49.534245684 -0800
+++ /tmp/foob	2015-11-09 05:08:49.538245681 -0800
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra114 (Dalmore) # 
 Tegra114 (Dalmore) # usb reset
 (Re)start USB...
@@ -36,10 +36,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##############################
+	 ###############################
 	 3.3 MiB/s
 done
-Bytes transferred = 5201568 (4f5ea0 hex)
+Bytes transferred = 5213800 (4f8e68 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4f5ea0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4f8e68 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.3.0-next-20151109-54253-g6958584 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Nov 8 19:48:24 PST 2015
+Linux version 4.3.0-next-20151106-54161-gc57d628 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Nov 9 05:02:59 PST 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -69,20 +69,20 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0a0c640, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0a16640, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-PERCPU: Embedded 12 pages/cpu @eefa2000 s19456 r8192 d21504 u49152
-pcpu-alloc: s19456 r8192 d21504 u49152 alloc=12*4096
+PERCPU: Embedded 12 pages/cpu @eefa2000 s19520 r8192 d21440 u49152
+pcpu-alloc: s19520 r8192 d21440 u49152 alloc=12*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002360K/2096128K available (7123K kernel code, 352K rwdata, 2452K rodata, 332K init, 364K bss, 28232K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2002320K/2096128K available (7155K kernel code, 352K rwdata, 2452K rodata, 340K init, 364K bss, 28272K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0962104   (9577 kB)
-      .init : 0xc0963000 - 0xc09b6000   ( 332 kB)
-      .data : 0xc09b6000 - 0xc0a0e160   ( 353 kB)
-       .bss : 0xc0a11000 - 0xc0a6c190   ( 365 kB)
+      .text : 0xc0008000 - 0xc096a194   (9609 kB)
+      .init : 0xc096b000 - 0xc09c0000   ( 340 kB)
+      .data : 0xc09c0000 - 0xc0a18160   ( 353 kB)
+       .bss : 0xc0a1b000 - 0xc0a76190   ( 365 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -298,7 +298,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-05-21 12:46:18 UTC (958913178)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-05-21 22:00:39 UTC (958946439)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -309,7 +309,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 332K (c0963000 - c09b6000)
+Freeing unused kernel memory: 340K (c096b000 - c09c0000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.900158] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.880129] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-11-09 05:08:49.542245677 -0800
+++ /tmp/foob	2015-11-09 05:08:49.542245677 -0800
@@ -39,21 +39,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ########################################################
-	 3.1 MiB/s
+	 #########################################################
+	 3.2 MiB/s
 done
-Bytes transferred = 6543176 (63d748 hex)
+Bytes transferred = 6552488 (63fba8 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *#####
-	 3.1 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x63d748 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x63fba8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012a89
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.3.0-next-20151109-54253-g6958584 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Nov 8 19:48:05 PST 2015
+Linux version 4.3.0-next-20151106-54161-gc57d628 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Nov 9 05:02:34 PST 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -69,20 +69,20 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0fc4180, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c0fcc180, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-PERCPU: Embedded 12 pages/cpu @eef8f000 s19136 r8192 d21824 u49152
-pcpu-alloc: s19136 r8192 d21824 u49152 alloc=12*4096
+PERCPU: Embedded 12 pages/cpu @eef8f000 s19200 r8192 d21760 u49152
+pcpu-alloc: s19200 r8192 d21760 u49152 alloc=12*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug init=/bin/bash noinitrd
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998476K/2096128K available (8243K kernel code, 1040K rwdata, 3740K rodata, 1044K init, 341K bss, 32116K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998444K/2096128K available (8267K kernel code, 1040K rwdata, 3740K rodata, 1052K init, 341K bss, 32148K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0dbce14   (11988 kB)
-      .init : 0xc0dbd000 - 0xc0ec2000   (1044 kB)
-      .data : 0xc0ec2000 - 0xc0fc60e0   (1041 kB)
-       .bss : 0xc0fc9000 - 0xc101e468   ( 342 kB)
+      .text : 0xc0208000 - 0xc0dc2e98   (12012 kB)
+      .init : 0xc0dc3000 - 0xc0eca000   (1052 kB)
+      .data : 0xc0eca000 - 0xc0fce0e0   (1041 kB)
+       .bss : 0xc0fd1000 - 0xc1026468   ( 342 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -339,7 +339,7 @@
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
-cpu: dev_pm_opp_of_cpumask_add_table: couldn't find opp table for cpu:0, -19
+dev_pm_opp_of_cpumask_add_table: couldn't find opp table for cpu:0, -19
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
@@ -364,10 +364,10 @@
 EXT4-fs (mmcblk0p1): couldn't mount as ext2 due to feature incompatibilities
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 VFS: Mounted root (ext4 filesystem) on device 179:1.
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 devtmpfs: mounted
-Freeing unused kernel memory: 1044K (c0dbd000 - c0ec2000)
+Freeing unused kernel memory: 1052K (c0dc3000 - c0eca000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2015-11-09 05:08:49.550245669 -0800
+++ /tmp/foob	2015-11-09 05:08:49.550245669 -0800
@@ -38,21 +38,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##############################
-	 3.3 MiB/s
+	 ###############################
+	 3.1 MiB/s
 done
-Bytes transferred = 5201568 (4f5ea0 hex)
+Bytes transferred = 5213800 (4f8e68 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *#####
-	 3.1 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4f5ea0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4f8e68 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 82012a89
@@ -63,7 +63,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.3.0-next-20151109-54253-g6958584 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Nov 8 19:48:24 PST 2015
+Linux version 4.3.0-next-20151106-54161-gc57d628 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Nov 9 05:02:59 PST 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -71,20 +71,20 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0a0c640, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0a16640, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-PERCPU: Embedded 12 pages/cpu @eef8e000 s19456 r8192 d21504 u49152
-pcpu-alloc: s19456 r8192 d21504 u49152 alloc=12*4096
+PERCPU: Embedded 12 pages/cpu @eef8e000 s19520 r8192 d21440 u49152
+pcpu-alloc: s19520 r8192 d21440 u49152 alloc=12*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug init=/bin/bash noinitrd
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002252K/2096128K available (7123K kernel code, 352K rwdata, 2452K rodata, 332K init, 364K bss, 28340K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2002212K/2096128K available (7155K kernel code, 352K rwdata, 2452K rodata, 340K init, 364K bss, 28380K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,10 +92,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0962104   (9577 kB)
-      .init : 0xc0963000 - 0xc09b6000   ( 332 kB)
-      .data : 0xc09b6000 - 0xc0a0e160   ( 353 kB)
-       .bss : 0xc0a11000 - 0xc0a6c190   ( 365 kB)
+      .text : 0xc0008000 - 0xc096a194   (9609 kB)
+      .init : 0xc096b000 - 0xc09c0000   ( 340 kB)
+      .data : 0xc09c0000 - 0xc0a18160   ( 353 kB)
+       .bss : 0xc0a1b000 - 0xc0a76190   ( 365 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -279,11 +279,11 @@
 NET: Registered protocol family 17
 NET: Registered protocol family 15
 can: controller area network core (rev 20120528 abi 9)
-input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input1
 NET: Registered protocol family 29
 can: raw protocol (rev 20120528)
 can: broadcast manager protocol (rev 20120528 t)
 can: netlink gateway (rev 20130117) max_hops=1
+input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input1
 Bluetooth: RFCOMM socket layer initialized
 Bluetooth: RFCOMM ver 1.11
 Bluetooth: BNEP (Ethernet Emulation) ver 1.3
@@ -366,7 +366,7 @@
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
-cpu: dev_pm_opp_of_cpumask_add_table: couldn't find opp table for cpu:0, -19
+dev_pm_opp_of_cpumask_add_table: couldn't find opp table for cpu:0, -19
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
@@ -377,7 +377,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:50 UTC (946684850)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:49 UTC (946684849)
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
@@ -391,12 +391,12 @@
   #1: NVIDIA Tegra Jetson TK1
 ata1: SATA link down (SStatus 0 SControl 300)
 EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 devtmpfs: mounted
-Freeing unused kernel memory: 332K (c0963000 - c09b6000)
+Freeing unused kernel memory: 340K (c096b000 - c09c0000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-11-09 05:08:49.558245661 -0800
+++ /tmp/foob	2015-11-09 05:08:49.558245661 -0800
@@ -45,10 +45,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ########################################################
+	 #########################################################
 	 4.1 MiB/s
 done
-Bytes transferred = 6543176 (63d748 hex)
+Bytes transferred = 6552488 (63fba8 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -59,7 +59,7 @@
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x63d748 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x63fba8 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -67,7 +67,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.3.0-next-20151109-54253-g6958584 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Nov 8 19:48:05 PST 2015
+Linux version 4.3.0-next-20151106-54161-gc57d628 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Nov 9 05:02:34 PST 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -75,20 +75,20 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0fc4180, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c0fcc180, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 65536 pages, LIFO batch:15
-PERCPU: Embedded 12 pages/cpu @ef7cb000 s19136 r8192 d21824 u49152
-pcpu-alloc: s19136 r8192 d21824 u49152 alloc=12*4096
+PERCPU: Embedded 12 pages/cpu @ef7cb000 s19200 r8192 d21760 u49152
+pcpu-alloc: s19200 r8192 d21760 u49152 alloc=12*4096
 pcpu-alloc: [0] 0 [0] 1 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 260608
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 959392K/1048576K available (8243K kernel code, 1040K rwdata, 3740K rodata, 1044K init, 341K bss, 23648K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 959360K/1048576K available (8267K kernel code, 1040K rwdata, 3740K rodata, 1052K init, 341K bss, 23680K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -96,10 +96,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0dbce14   (11988 kB)
-      .init : 0xc0dbd000 - 0xc0ec2000   (1044 kB)
-      .data : 0xc0ec2000 - 0xc0fc60e0   (1041 kB)
-       .bss : 0xc0fc9000 - 0xc101e468   ( 342 kB)
+      .text : 0xc0208000 - 0xc0dc2e98   (12012 kB)
+      .init : 0xc0dc3000 - 0xc0eca000   (1052 kB)
+      .data : 0xc0eca000 - 0xc0fce0e0   (1041 kB)
+       .bss : 0xc0fd1000 - 0xc1026468   ( 342 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -330,7 +330,7 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-26 23:34:32 UTC (1419636872)
+rtc-em3027 2-0056: setting system clock to 2014-12-26 23:36:23 UTC (1419636983)
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
  mmcblk0: p1
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
@@ -340,7 +340,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1044K (c0dbd000 - c0ec2000)
+Freeing unused kernel memory: 1052K (c0dc3000 - c0eca000)
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2015-11-09 05:08:49.562245657 -0800
+++ /tmp/foob	2015-11-09 05:08:49.566245653 -0800
@@ -44,10 +44,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##############################
+	 ###############################
 	 4.1 MiB/s
 done
-Bytes transferred = 5201568 (4f5ea0 hex)
+Bytes transferred = 5213800 (4f8e68 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x4f5ea0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x4f8e68 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -69,7 +69,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.3.0-next-20151109-54253-g6958584 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Nov 8 19:48:24 PST 2015
+Linux version 4.3.0-next-20151106-54161-gc57d628 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Nov 9 05:02:59 PST 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -78,20 +78,20 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0a0c640, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0a16640, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 65536 pages, LIFO batch:15
-PERCPU: Embedded 12 pages/cpu @ef7ca000 s19456 r8192 d21504 u49152
-pcpu-alloc: s19456 r8192 d21504 u49152 alloc=12*4096
+PERCPU: Embedded 12 pages/cpu @ef7ca000 s19520 r8192 d21440 u49152
+pcpu-alloc: s19520 r8192 d21440 u49152 alloc=12*4096
 pcpu-alloc: [0] 0 [0] 1 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 260608
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 963168K/1048576K available (7123K kernel code, 352K rwdata, 2452K rodata, 332K init, 364K bss, 19872K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 963128K/1048576K available (7155K kernel code, 352K rwdata, 2452K rodata, 340K init, 364K bss, 19912K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -99,10 +99,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0962104   (9577 kB)
-      .init : 0xc0963000 - 0xc09b6000   ( 332 kB)
-      .data : 0xc09b6000 - 0xc0a0e160   ( 353 kB)
-       .bss : 0xc0a11000 - 0xc0a6c190   ( 365 kB)
+      .text : 0xc0008000 - 0xc096a194   (9609 kB)
+      .init : 0xc096b000 - 0xc09c0000   ( 340 kB)
+      .data : 0xc09c0000 - 0xc0a18160   ( 353 kB)
+       .bss : 0xc0a1b000 - 0xc0a76190   ( 365 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
@@ -311,16 +311,16 @@
 tlv320aic23-codec 2-001a: ASoC: Failed to add route RLINEIN -> NULL -> Line Input
 tlv320aic23-codec 2-001a: Control not supported for path MICIN -> [NULL] -> Mic Input
 tlv320aic23-codec 2-001a: ASoC: no dapm match for MICIN --> NULL --> Mic Input
-tlv320aic23-codec 2-001a: ASoC: Failed to add route MICIN -> NULL -> Mic Input
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
+tlv320aic23-codec 2-001a: ASoC: Failed to add route MICIN -> NULL -> Mic Input
 tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
- mmcblk0: p1
 NET: Registered protocol family 10
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
 mip6: Mobile IPv6
 sit: IPv6 over IPv4 tunneling driver
+ mmcblk0: p1
 ip6tnl0: set_features() failed (-1); wanted 0x0000000000006000, left 0x0000000000006800
 NET: Registered protocol family 17
 NET: Registered protocol family 15
@@ -334,12 +334,7 @@
 Bluetooth: BNEP (Ethernet Emulation) ver 1.3
 Bluetooth: BNEP socket layer initialized
 Bluetooth: HIDP (Human Interface Emulation) ver 1.2
-mmc1: new high speed SD card at address b368
 Bluetooth: HIDP socket layer initialized
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
-hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
- mmcblk1: p1 p2
 Key type dns_resolver registered
 Registering SWP/SWPB emulation handler
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
@@ -347,18 +342,23 @@
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
+hub 1-1:1.0: USB hub found
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-26 23:34:52 UTC (1419636892)
+rtc-em3027 2-0056: setting system clock to 2014-12-26 23:36:44 UTC (1419637004)
+hub 1-1:1.0: 4 ports detected
 ALSA device list:
   #0: tegra-trimslice
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
+mmc1: new high speed SD card at address b368
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
+ mmcblk1: p1 p2
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 332K (c0963000 - c09b6000)
+Freeing unused kernel memory: 340K (c096b000 - c09c0000)
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-11-09 05:08:49.570245650 -0800
+++ /tmp/foob	2015-11-09 05:08:49.574245646 -0800
@@ -37,10 +37,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ########################################################
+	 #########################################################
 	 3.4 MiB/s
 done
-Bytes transferred = 6543176 (63d748 hex)
+Bytes transferred = 6552488 (63fba8 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -51,7 +51,7 @@
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x63d748 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x63fba8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.3.0-next-20151109-54253-g6958584 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Nov 8 19:48:05 PST 2015
+Linux version 4.3.0-next-20151106-54161-gc57d628 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Nov 9 05:02:34 PST 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -67,20 +67,20 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0fc4180, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c0fcc180, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-PERCPU: Embedded 12 pages/cpu @eef93000 s19136 r8192 d21824 u49152
-pcpu-alloc: s19136 r8192 d21824 u49152 alloc=12*4096
+PERCPU: Embedded 12 pages/cpu @eef93000 s19200 r8192 d21760 u49152
+pcpu-alloc: s19200 r8192 d21760 u49152 alloc=12*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug init=/bin/bash noinitrd
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998504K/2096128K available (8243K kernel code, 1040K rwdata, 3740K rodata, 1044K init, 341K bss, 32088K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998472K/2096128K available (8267K kernel code, 1040K rwdata, 3740K rodata, 1052K init, 341K bss, 32120K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,10 +88,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0dbce14   (11988 kB)
-      .init : 0xc0dbd000 - 0xc0ec2000   (1044 kB)
-      .data : 0xc0ec2000 - 0xc0fc60e0   (1041 kB)
-       .bss : 0xc0fc9000 - 0xc101e468   ( 342 kB)
+      .text : 0xc0208000 - 0xc0dc2e98   (12012 kB)
+      .init : 0xc0dc3000 - 0xc0eca000   (1052 kB)
+      .data : 0xc0eca000 - 0xc0fce0e0   (1041 kB)
+       .bss : 0xc0fd1000 - 0xc1026468   ( 342 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -280,18 +280,18 @@
 Key type dns_resolver registered
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
+tegra-pcie 3000.pcie-controller: 2x3 configuration
+sys_3v3,vdd_3v3_alw: supplied by vdd_5v_in
+sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
+tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
 mmc1: new high speed MMC card at address 0001
 mmcblk0: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk0boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk0boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk0rpmb: mmc1:0001 SEM16G partition 3 128 KiB
 Primary GPT is invalid, using alternate GPT.
-tegra-pcie 3000.pcie-controller: 2x3 configuration
-sys_3v3,vdd_3v3_alw: supplied by vdd_5v_in
-sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
-tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
- mmcblk0: p1 p2 p3 p4 p5
 tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
+ mmcblk0: p1 p2 p3 p4 p5
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -362,7 +362,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1044K (c0dbd000 - c0ec2000)
+Freeing unused kernel memory: 1052K (c0dc3000 - c0eca000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.743472] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.748437] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2015-11-09 05:08:49.578245642 -0800
+++ /tmp/foob	2015-11-09 05:08:49.582245638 -0800
@@ -36,10 +36,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##############################
-	 3.4 MiB/s
+	 ###############################
+	 3.3 MiB/s
 done
-Bytes transferred = 5201568 (4f5ea0 hex)
+Bytes transferred = 5213800 (4f8e68 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4f5ea0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4f8e68 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.3.0-next-20151109-54253-g6958584 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Nov 8 19:48:24 PST 2015
+Linux version 4.3.0-next-20151106-54161-gc57d628 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Nov 9 05:02:59 PST 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -69,20 +69,20 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0a0c640, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0a16640, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-PERCPU: Embedded 12 pages/cpu @eef92000 s19456 r8192 d21504 u49152
-pcpu-alloc: s19456 r8192 d21504 u49152 alloc=12*4096
+PERCPU: Embedded 12 pages/cpu @eef92000 s19520 r8192 d21440 u49152
+pcpu-alloc: s19520 r8192 d21440 u49152 alloc=12*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug init=/bin/bash noinitrd
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002280K/2096128K available (7123K kernel code, 352K rwdata, 2452K rodata, 332K init, 364K bss, 28312K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2002240K/2096128K available (7155K kernel code, 352K rwdata, 2452K rodata, 340K init, 364K bss, 28352K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0962104   (9577 kB)
-      .init : 0xc0963000 - 0xc09b6000   ( 332 kB)
-      .data : 0xc09b6000 - 0xc0a0e160   ( 353 kB)
-       .bss : 0xc0a11000 - 0xc0a6c190   ( 365 kB)
+      .text : 0xc0008000 - 0xc096a194   (9609 kB)
+      .init : 0xc096b000 - 0xc09c0000   ( 340 kB)
+      .data : 0xc09c0000 - 0xc0a18160   ( 353 kB)
+       .bss : 0xc0a1b000 - 0xc0a76190   ( 365 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -365,7 +365,8 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 332K (c0963000 - c09b6000)
+Freeing unused kernel memory: 340K (c096b000 - c09c0000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.576779] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# 
\ No newline at end of file
