Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2017-07-31 00:11:52.548973973 -0700
+++ /tmp/foob	2017-07-31 00:11:52.552973988 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc2-next-20170728-306891-gb2cf822 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Thu Jul 27 22:54:57 PDT 2017
+Linux version 4.13.0-rc3-next-20170731-307516-g742f220 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Mon Jul 31 00:09:12 PDT 2017
 Boot CPU: AArch64 Processor [411fd070]
 Machine model: linux,dummy-virt
 Memory limited to 1024MB
@@ -8,8 +8,6 @@
 cma: Reserved 16 MiB at 0x000000007f000000
 NUMA: No NUMA configuration found
 NUMA: Faking a node at [mem 0x0000000000000000-0x000000007fffffff]
-NUMA: Adding memblock [0x40000000 - 0x7fffffff] on node 0
-NUMA: Initmem setup node 0 [mem 0x40000000-0x7fffffff]
 NUMA: NODE_DATA [mem 0x7efe8f00-0x7efea9ff]
 Zone ranges:
   DMA      [mem 0x0000000040000000-0x000000007fffffff]
@@ -30,15 +28,15 @@
 Policy zone: DMA
 Kernel command line: root=/dev/vda2 rw rootwait mem=1024M console=ttyAMA0,38400n8 init=/bin/bash
 PID hash table entries: 4096 (order: 3, 32768 bytes)
-Memory: 999428K/1048576K available (9020K kernel code, 1116K rwdata, 4396K rodata, 1088K init, 403K bss, 32764K reserved, 16384K cma-reserved)
+Memory: 999304K/1048576K available (9084K kernel code, 1114K rwdata, 4440K rodata, 1088K init, 401K bss, 32888K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff000008950000   (  9024 KB)
-    .rodata : 0xffff000008950000 - 0xffff000008da0000   (  4416 KB)
-      .init : 0xffff000008da0000 - 0xffff000008eb0000   (  1088 KB)
-      .data : 0xffff000008eb0000 - 0xffff000008fc7200   (  1117 KB)
-       .bss : 0xffff000008fc7200 - 0xffff00000902c0f0   (   404 KB)
+      .text : 0xffff000008080000 - 0xffff000008960000   (  9088 KB)
+    .rodata : 0xffff000008960000 - 0xffff000008dc0000   (  4480 KB)
+      .init : 0xffff000008dc0000 - 0xffff000008ed0000   (  1088 KB)
+      .data : 0xffff000008ed0000 - 0xffff000008fe6a00   (  1115 KB)
+       .bss : 0xffff000008fe6a00 - 0xffff00000904b0f0   (   402 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -79,7 +77,7 @@
 DMI not present or invalid.
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffff000008956000, 1 data @ ffff000008eb5000)
+vdso: 2 pages (1 code @ ffff000008966000, 1 data @ ffff000008ed5000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -116,7 +114,7 @@
 hw perfevents: enabled with armv8_pmuv3 PMU driver, 1 counters available
 kvm [1]: HYP mode not available
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.772:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.820:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -182,7 +180,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2017-07-28 05:55:57 UTC (1501221357)
+rtc-pl031 9010000.pl031: setting system clock to 2017-07-31 07:10:17 UTC (1501485017)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-07-31 00:11:52.588974121 -0700
+++ /tmp/foob	2017-07-31 00:11:52.592974135 -0700
@@ -40,9 +40,9 @@
 	 #################################################################
 	 #################################################################
 	 ###########################
-	 3.4 MiB/s
+	 3 MiB/s
 done
-Bytes transferred = 8025640 (7a7628 hex)
+Bytes transferred = 8026896 (7a7b10 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -53,7 +53,7 @@
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x7a7628 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x7a7b10 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc2-next-20170728-188650-gb2cf822 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Jul 27 22:49:59 PDT 2017
+Linux version 4.13.0-rc3-next-20170731-189275-g742f220 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Mon Jul 31 00:05:02 PDT 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -72,7 +72,7 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c157f080, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c157c640, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -85,7 +85,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993544K/2096128K available (10240K kernel code, 1563K rwdata, 4464K rodata, 2048K init, 404K bss, 37048K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993552K/2096128K available (10240K kernel code, 1553K rwdata, 4476K rodata, 2048K init, 404K bss, 37040K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -95,8 +95,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1586ea0   (1564 kB)
-       .bss : 0xc158fc98 - 0xc15f507c   ( 405 kB)
+      .data : 0xc1400000 - 0xc1584460   (1554 kB)
+       .bss : 0xc158dc98 - 0xc15f307c   ( 405 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -314,12 +314,12 @@
 Registering SWP/SWPB emulation handler
 usb1_vbus: supplied by vdd-sys-5v0
 mmc0: new high speed SDHC card at address aaaa
-mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 usb2_vbus: supplied by vdd-sys-5v0
 VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
- mmcblk0: p1 p2 p3
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
+mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
+ mmcblk0: p1 p2 p3
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
@@ -336,7 +336,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2001-05-21 17:29:35 UTC (990466175)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2001-05-24 18:47:13 UTC (990730033)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -348,4 +348,4 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.599340] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.568934] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2017-07-31 00:11:52.596974150 -0700
+++ /tmp/foob	2017-07-31 00:11:52.596974150 -0700
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra114 (Dalmore) # 
 Tegra114 (Dalmore) # usb reset
 (Re)start USB...
@@ -40,18 +40,18 @@
 	 ##############################
 	 3.2 MiB/s
 done
-Bytes transferred = 6152792 (5de258 hex)
+Bytes transferred = 6154216 (5de7e8 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.8 MiB/s
+	 2.6 MiB/s
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5de258 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5de7e8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -60,7 +60,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc2-next-20170728-188650-gb2cf822 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Jul 27 22:49:58 PDT 2017
+Linux version 4.13.0-rc3-next-20170731-189275-g742f220 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Jul 31 00:04:51 PDT 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -70,7 +70,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0eafb40, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0ead140, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -83,7 +83,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998556K/2096128K available (9216K kernel code, 744K rwdata, 2736K rodata, 1024K init, 299K bss, 32036K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998568K/2096128K available (9216K kernel code, 734K rwdata, 2744K rodata, 1024K init, 299K bss, 32024K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -93,8 +93,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0eba360   ( 745 kB)
-       .bss : 0xc0ec45e0 - 0xc0f0f3bc   ( 300 kB)
+      .data : 0xc0e00000 - 0xc0eb7960   ( 735 kB)
+       .bss : 0xc0ec15e0 - 0xc0f0c3bc   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -328,7 +328,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2001-05-21 17:32:13 UTC (990466333)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2001-05-24 18:44:43 UTC (990729883)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -342,5 +342,5 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.828404] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.798960] usb 1-1: new high-speed USB device number 2 using tegra-ehci
 asix 1-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0d:b5:16
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-07-31 00:11:52.604974180 -0700
+++ /tmp/foob	2017-07-31 00:11:52.608974194 -0700
@@ -27,7 +27,7 @@
 Filename 'zImage.multi_v7_defconfig'.
 Load address: 0x82000000
 Loading: *#################################################################
-	 #################################################################
+	 ##################################T T T ###############################
 	 #################################################################
 	 #################################################################
 	 #################################################################
@@ -35,20 +35,20 @@
 	 #################################################################
 	 #################################################################
 	 ###########################
-	 939.5 KiB/s
+	 461.9 KiB/s
 done
-Bytes transferred = 8025640 (7a7628 hex)
+Bytes transferred = 8026896 (7a7b10 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 936.5 KiB/s
+	 3.8 MiB/s
 done
 Bytes transferred = 67161 (10659 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x7a7628 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x7a7b10 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013658
@@ -56,7 +56,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc2-next-20170728-188650-gb2cf822 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Jul 27 22:49:59 PDT 2017
+Linux version 4.13.0-rc3-next-20170731-189275-g742f220 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Mon Jul 31 00:05:02 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -67,7 +67,7 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c157f080, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c157c640, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -80,7 +80,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993424K/2096128K available (10240K kernel code, 1563K rwdata, 4464K rodata, 2048K init, 404K bss, 37168K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993432K/2096128K available (10240K kernel code, 1553K rwdata, 4476K rodata, 2048K init, 404K bss, 37160K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,8 +90,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1586ea0   (1564 kB)
-       .bss : 0xc158fc98 - 0xc15f507c   ( 405 kB)
+      .data : 0xc1400000 - 0xc1584460   (1554 kB)
+       .bss : 0xc158dc98 - 0xc15f307c   ( 405 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -318,12 +318,6 @@
 +5V_SATA: supplied by +5V_SYS
 +12V_SATA: supplied by +VDD_MUX
  usb2-0: usb2-0 supply vbus not found, using dummy regulator
-mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
- mmcblk0: p1
  usb2-1: usb2-1 supply vbus not found, using dummy regulator
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
 tegra-pcie 1003000.pcie: 2x1, 1x1 configuration
@@ -349,9 +343,15 @@
 +2.8V_RUN_CAM_AF: Bringing 0uV into 2800000-2800000uV
 as3722-ldo11: supplied by +3.3V_RUN
 +1.8V_RUN_VPP_FUSE: Bringing 0uV into 1800000-1800000uV
+mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
 tegra-ahci 70027000.sata: AHCI 0001.0301 32 slots 2 ports 3 Gbps 0x1 impl platform mode
 tegra-ahci 70027000.sata: flags: 64bit ncq sntf pm led pio slum part sadm sds apst 
 scsi host0: tegra-ahci
+ mmcblk0: p1
 scsi host1: tegra-ahci
 ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 345
 ata2: DUMMY
@@ -425,9 +425,6 @@
 pci 0000:00:02.0:   bridge window [io  0x1000-0x1fff]
 pci 0000:00:02.0:   bridge window [mem 0x13100000-0x131fffff]
 pci 0000:00:02.0:   bridge window [mem 0x20000000-0x200fffff 64bit pref]
-mmcblk1: error -5 sending stop command, original cmd response 0x900, card status 0x900
-mmcblk1: error -5 transferring data, sector 7744504, nr 8, cmd response 0x900, card status 0x80000b00
-mmcblk1: retrying using single block read
 pci 0000:00:01.0: nv_msi_ht_cap_quirk didn't locate host bridge
 pcieport 0000:00:01.0: enabling device (0140 -> 0142)
 pcieport 0000:00:01.0: Signaling PME with IRQ 393
@@ -436,20 +433,17 @@
 pcieport 0000:00:02.0: Signaling PME with IRQ 394
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
- mmcblk1: p1 p2 p3
 r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0b72000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 395
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
-tegra-hdmi 54280000.hdmi: cannot set audio to 48000 Hz at 297000000 Hz pixel clock
-Console: switching to colour frame buffer device 512x135
-drm drm: fb0:  frame buffer device
+[drm] Cannot find any crtc or sizes
 [drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:20 UTC (946684820)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:54 UTC (946684974)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
@@ -458,12 +452,16 @@
 +3.3V_RUN_TOUCH: disabling
 +2.8V_RUN_CAM_AF: disabling
 +1.8V_RUN_VPP_FUSE: disabling
+tegra-hdmi 54280000.hdmi: cannot set audio to 48000 Hz at 297000000 Hz pixel clock
+Console: switching to colour frame buffer device 512x135
+ mmcblk1: p1 p2 p3
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K
+drm drm: fb0:  frame buffer device
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2017-07-31 00:11:52.612974209 -0700
+++ /tmp/foob	2017-07-31 00:11:52.616974223 -0700
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -33,9 +33,9 @@
 	 #################################################################
 	 #################################################################
 	 ##############################
-	 3.9 MiB/s
+	 4 MiB/s
 done
-Bytes transferred = 6152792 (5de258 hex)
+Bytes transferred = 6154216 (5de7e8 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -46,7 +46,7 @@
 done
 Bytes transferred = 67161 (10659 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5de258 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5de7e8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013658
@@ -55,7 +55,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc2-next-20170728-188650-gb2cf822 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Jul 27 22:49:58 PDT 2017
+Linux version 4.13.0-rc3-next-20170731-189275-g742f220 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Jul 31 00:04:51 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -65,7 +65,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0eafb40, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0ead140, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -78,7 +78,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998436K/2096128K available (9216K kernel code, 744K rwdata, 2736K rodata, 1024K init, 299K bss, 32156K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998448K/2096128K available (9216K kernel code, 734K rwdata, 2744K rodata, 1024K init, 299K bss, 32144K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -88,8 +88,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0eba360   ( 745 kB)
-       .bss : 0xc0ec45e0 - 0xc0f0f3bc   ( 300 kB)
+      .data : 0xc0e00000 - 0xc0eb7960   ( 735 kB)
+       .bss : 0xc0ec15e0 - 0xc0f0c3bc   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -402,18 +402,17 @@
 pci 0000:01:00.0: [8086:4232] type 00 class 0x028000
 pci 0000:01:00.0: reg 0x10: [mem 0x00000000-0x00001fff 64bit]
 pci 0000:01:00.0: PME# supported from D0 D3hot D3cold
-mmc1: new high speed SDHC card at address aaaa
 PCI: bus1: Fast back to back transfers disabled
-mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
 pci 0000:02:00.0: [10ec:8168] type 00 class 0x020000
 pci 0000:02:00.0: reg 0x10: [io  0x0000-0x00ff]
+mmc1: new high speed SDHC card at address aaaa
+mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 pci 0000:02:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit]
 pci 0000:02:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
 pci 0000:02:00.0: supports D1 D2
 pci 0000:02:00.0: PME# supported from D0 D1 D2 D3hot D3cold
-mmcblk1: error -5 sending stop command, original cmd response 0x900, card status 0x900
-mmcblk1: error -5 transferring data, sector 7744504, nr 8, cmd response 0x900, card status 0x80000b00
+ mmcblk1: p1 p2 p3
 PCI: bus2: Fast back to back transfers disabled
 pci_bus 0000:02: busn_res: [bus 02-ff] end is updated to 02
 pci 0000:00:01.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
@@ -430,11 +429,7 @@
 pci 0000:00:02.0:   bridge window [io  0x1000-0x1fff]
 pci 0000:00:02.0:   bridge window [mem 0x13100000-0x131fffff]
 pci 0000:00:02.0:   bridge window [mem 0x20000000-0x200fffff 64bit pref]
-mmcblk1: error -5 sending stop command, original cmd response 0x900, card status 0x900
-mmcblk1: error -5 transferring data, sector 7744504, nr 8, cmd response 0x900, card status 0x80000b00
-mmcblk1: retrying using single block read
 pci 0000:00:01.0: nv_msi_ht_cap_quirk didn't locate host bridge
- mmcblk1: p1 p2 p3
 pcieport 0000:00:01.0: enabling device (0140 -> 0142)
 pcieport 0000:00:01.0: Signaling PME with IRQ 393
 pci 0000:00:02.0: nv_msi_ht_cap_quirk didn't locate host bridge
@@ -454,7 +449,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:01:30 UTC (946684890)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:13 UTC (946684813)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-07-31 00:11:52.644974327 -0700
+++ /tmp/foob	2017-07-31 00:11:52.644974327 -0700
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -45,18 +45,18 @@
 	 ###########################
 	 3.6 MiB/s
 done
-Bytes transferred = 8025640 (7a7628 hex)
+Bytes transferred = 8026896 (7a7b10 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.3 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 68183 (10a57 hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x7a7628 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x7a7b10 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a56
@@ -64,7 +64,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc2-next-20170728-188650-gb2cf822 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Jul 27 22:49:59 PDT 2017
+Linux version 4.13.0-rc3-next-20170731-189275-g742f220 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Mon Jul 31 00:05:02 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -76,7 +76,7 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c157f080, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c157c640, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -89,7 +89,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1994440K/2097148K available (10240K kernel code, 1563K rwdata, 4464K rodata, 2048K init, 404K bss, 37172K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1994448K/2097148K available (10240K kernel code, 1553K rwdata, 4476K rodata, 2048K init, 404K bss, 37164K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -99,8 +99,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1586ea0   (1564 kB)
-       .bss : 0xc158fc98 - 0xc15f507c   ( 405 kB)
+      .data : 0xc1400000 - 0xc1584460   (1554 kB)
+       .bss : 0xc158dc98 - 0xc15f307c   ( 405 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -300,9 +300,9 @@
 mmc0: Unknown controller version (3). You may experience problems.
 mmc0: Invalid maximum block size, assuming 512 bytes
 elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
+input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 mmc0: SDHCI controller on 700b0600.sdhci [700b0600.sdhci] using ADMA
 ledtrig-cpu: registered to indicate activity on CPUs
-input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
 NET: Registered protocol family 10
@@ -345,8 +345,8 @@
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
 mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
-mmc1: SDHCI controller on 700b0000.sdhci [700b0000.sdhci] using ADMA
 elan_i2c 1-0015: invalid report id data (ff)
+mmc1: SDHCI controller on 700b0000.sdhci [700b0000.sdhci] using ADMA
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc2: Unknown controller version (3). You may experience problems.
@@ -367,7 +367,7 @@
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:24 UTC (946684824)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:23 UTC (946684823)
 +1.8V_RUN_CAM: disabling
 +3.3V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2017-07-31 00:11:52.668974416 -0700
+++ /tmp/foob	2017-07-31 00:11:52.672974430 -0700
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -41,20 +41,20 @@
 	 #################################################################
 	 #################################################################
 	 ##############################
-	 3.5 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 6152792 (5de258 hex)
+Bytes transferred = 6154216 (5de7e8 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.4 MiB/s
+	 3.6 MiB/s
 done
 Bytes transferred = 68183 (10a57 hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5de258 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5de7e8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a56
@@ -63,7 +63,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc2-next-20170728-188650-gb2cf822 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Jul 27 22:49:58 PDT 2017
+Linux version 4.13.0-rc3-next-20170731-189275-g742f220 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Jul 31 00:04:51 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -74,7 +74,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c0eafb40, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0ead140, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -87,7 +87,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1999452K/2097148K available (9216K kernel code, 744K rwdata, 2736K rodata, 1024K init, 299K bss, 32160K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1999464K/2097148K available (9216K kernel code, 734K rwdata, 2744K rodata, 1024K init, 299K bss, 32148K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -97,8 +97,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0eba360   ( 745 kB)
-       .bss : 0xc0ec45e0 - 0xc0f0f3bc   ( 300 kB)
+      .data : 0xc0e00000 - 0xc0eb7960   ( 735 kB)
+       .bss : 0xc0ec15e0 - 0xc0f0c3bc   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -293,9 +293,12 @@
 tegra-snd-max98090 sound: HiFi <-> 70301100.i2s mapping ok
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
-mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
 input: GoogleNyanBig Headphones as /devices/soc0/sound/sound/card1/input1
+mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
 input: GoogleNyanBig Mic Jack as /devices/soc0/sound/sound/card1/input2
+mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
+mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 NET: Registered protocol family 10
 Segment Routing with IPv6
 mip6: Mobile IPv6
@@ -357,9 +360,8 @@
 mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
-mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
-mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
+input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input3
+usb 1-2: new high-speed USB device number 2 using tegra-xusb
 mmc2: new high speed SDHC card at address 0007
 mmcblk2: mmc2:0007 SD08G 7.42 GiB 
 GPT:Primary header alternate_lba != Alt. header my_lba
@@ -369,11 +371,10 @@
 GPT: Use GNU Parted to correct GPT errors.
  mmcblk2: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 Console: switching to colour frame buffer device 170x48
-usb 1-2: new high-speed USB device number 2 using tegra-xusb
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
-input: gpio-keys as /devices/soc0/gpio-keys/input/input3
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:22 UTC (946684822)
+input: gpio-keys as /devices/soc0/gpio-keys/input/input4
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:23 UTC (946684823)
 +1.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
 +1.05V_RUN_CAM_REAR: disabling
@@ -382,13 +383,11 @@
 +1.8V_RUN_VPP_FUSE: disabling
 +5V_VDD_TS_SW: disabling
 ALSA device list:
+  #0: tegra-hda at 0x70038000 irq 343
   #1: GoogleNyanBig
+EXT4-fs (mmcblk0p3): couldn't mount as ext3 due to feature incompatibilities
 uvcvideo: Found UVC 1.00 device HD WebCam (1bcf:2c67)
 input: HD WebCam: HD WebCam as /devices/soc0/70090000.usb/usb1/1-2/1-2:1.0/input/input5
-usb 1-3: new high-speed USB device number 3 using tegra-xusb
-input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input4
-asix 1-3:1.0 eth0: register 'asix' at usb-70090000.usb-3, ASIX AX88772 USB 2.0 Ethernet, c8:d7:19:90:fa:65
-EXT4-fs (mmcblk0p3): couldn't mount as ext3 due to feature incompatibilities
 EXT4-fs (mmcblk0p3): recovery complete
 EXT4-fs (mmcblk0p3): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:3.
@@ -396,6 +395,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.767910] usb 1-1: new high-speed USB device number 4 using tegra-xusb
-usb-storage 1-1:1.0: USB Mass Storage device detected
-scsi host0: usb-storage 1-1:1.0
+root@(none):/# [    3.004299] usb 1-3: new high-speed USB device number 3 using tegra-xusb
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2017-07-31 00:11:52.688974490 -0700
+++ /tmp/foob	2017-07-31 00:11:52.688974490 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1800763 us
+Bootloader-Cpu Init at (time stamp): 1800939 us
 
 Pinmux changes applied in kernel way
 
@@ -62,22 +62,22 @@
 Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1800737  us
-Bootloader (AosInit) time: 734229 us
-Bootloader Main Init time: 183629 us
-BlockDevMgrInit time: 157538 us
+Miscellaneous avp time: 1800913  us
+Bootloader (AosInit) time: 734241 us
+Bootloader Main Init time: 180072 us
+BlockDevMgrInit time: 157548 us
 PartMgrInit time: 8 us
 StorMgrInit time: 30 us
-FsMgrInit time: 3 us
-Partition Table load: 76983 us 
-Display init time: 574510 us
-Kernel read time: 1309 msec
-Load Kernel Size:   17340416 Bytes
-Bl Kernel ReadSpeed: 13247000 Bytes/sec
-Pre-Kernel Jump time: 3748692 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 8681913 us
+FsMgrInit time: 2 us
+Partition Table load: 76693 us 
+Display init time: 574841 us
+Kernel read time: 1318 msec
+Load Kernel Size:   17469440 Bytes
+Bl Kernel ReadSpeed: 13254000 Bytes/sec
+Pre-Kernel Jump time: 3770839 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 8707754 us
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc2-next-20170728-306891-gb2cf822 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Thu Jul 27 22:54:57 PDT 2017
+Linux version 4.13.0-rc3-next-20170731-307516-g742f220 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Mon Jul 31 00:09:12 PDT 2017
 Boot CPU: AArch64 Processor [4e0f0000]
 Machine model: NVIDIA Tegra132 Norrin
 Memory limited to 1734MB
@@ -86,8 +86,6 @@
 cma: Reserved 16 MiB at 0x00000000eb400000
 NUMA: No NUMA configuration found
 NUMA: Faking a node at [mem 0x0000000000000000-0x00000000ec5fffff]
-NUMA: Adding memblock [0x80000000 - 0xec5fffff] on node 0
-NUMA: Initmem setup node 0 [mem 0x80000000-0xec5fffff]
 NUMA: NODE_DATA [mem 0xec5bed80-0xec5c087f]
 Zone ranges:
   DMA      [mem 0x0000000080000000-0x00000000ec5fffff]
@@ -103,15 +101,15 @@
 Policy zone: DMA
 Kernel command line: tegraid=13.1.3.0.0 mem=1734M@2048M memtype=255 vpr=151M@3817M ddr_die=2048M@2048M section=256M commchip_id=0 usb_port_owner_info=0 lane_owner_info=0 emc_max_dvfs=0 touch_id=0@0 video=tegrafb no_console_suspend=1 console=ttyS0,115200n8 debug_uartport=lsport,0 earlyprintk=uart8250-32bit,0x70006000 usbcore.old_scheme_first=1 lp0_vec=0@0x0 tegra_fbmem=32899072@0xad012000 core_edp_mv=1150 core_edp_ma=4000 pmuboard=0x0176:0x0001:0x01:0x43:0x00 displayboard=0x0162:0x0001:0x00:0x42:0x00 power_supply=Adapter audio_codec=rt5640 board_info=0x0176:0x0001:0x01:0x43:0x00 tegraboot=sdmmc gpt gpt_sector=192511 modem_id=0 android.kerneltype=normal
 PID hash table entries: 4096 (order: 3, 32768 bytes)
-Memory: 1708652K/1775616K available (9020K kernel code, 1116K rwdata, 4396K rodata, 1088K init, 403K bss, 50580K reserved, 16384K cma-reserved)
+Memory: 1708528K/1775616K available (9084K kernel code, 1114K rwdata, 4440K rodata, 1088K init, 401K bss, 50704K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff000008950000   (  9024 KB)
-    .rodata : 0xffff000008950000 - 0xffff000008da0000   (  4416 KB)
-      .init : 0xffff000008da0000 - 0xffff000008eb0000   (  1088 KB)
-      .data : 0xffff000008eb0000 - 0xffff000008fc7200   (  1117 KB)
-       .bss : 0xffff000008fc7200 - 0xffff00000902c0f0   (   404 KB)
+      .text : 0xffff000008080000 - 0xffff000008960000   (  9088 KB)
+    .rodata : 0xffff000008960000 - 0xffff000008dc0000   (  4480 KB)
+      .init : 0xffff000008dc0000 - 0xffff000008ed0000   (  1088 KB)
+      .data : 0xffff000008ed0000 - 0xffff000008fe6a00   (  1115 KB)
+       .bss : 0xffff000008fe6a00 - 0xffff00000904b0f0   (   402 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -152,7 +150,7 @@
 DMI not present or invalid.
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffff000008956000, 1 data @ ffff000008eb5000)
+vdso: 2 pages (1 code @ ffff000008966000, 1 data @ ffff000008ed5000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -202,7 +200,7 @@
 Freeing initrd memory: 1284K
 kvm [1]: HYP mode not available
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.923:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.935:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -266,15 +264,15 @@
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
 mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 NET: Registered protocol family 17
-9pnet: Installing 9P2000 support
-Key type dns_resolver registered
-registered taskstats version 1
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+9pnet: Installing 9P2000 support
+Key type dns_resolver registered
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
+registered taskstats version 1
 +VDD_LED: supplied by +VDD_MUX
 +5V_USB_HS: supplied by +5V_SYS
 +5V_USB_SS: supplied by +5V_SYS
@@ -292,8 +290,8 @@
 +5V_HDMI_CON: disabling
 ALSA device list:
   No soundcards found.
-sdhci-tegra 700b0400.sdhci: Got WP GPIO
 Freeing unused kernel memory: 1088K
+sdhci-tegra 700b0400.sdhci: Got WP GPIO
 
 System initialization...
 
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-07-31 00:11:52.700974533 -0700
+++ /tmp/foob	2017-07-31 00:11:52.704974549 -0700
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -48,9 +48,9 @@
 	 #################################################################
 	 #################################################################
 	 ###########################
-	 4.1 MiB/s
+	 4 MiB/s
 done
-Bytes transferred = 8025640 (7a7628 hex)
+Bytes transferred = 8026896 (7a7b10 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -61,7 +61,7 @@
 done
 Bytes transferred = 21454 (53ce hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x7a7628 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x7a7b10 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103cd
@@ -69,7 +69,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc2-next-20170728-188650-gb2cf822 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Jul 27 22:49:59 PDT 2017
+Linux version 4.13.0-rc3-next-20170731-189275-g742f220 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Mon Jul 31 00:05:02 PDT 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -79,7 +79,7 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0x3c000000
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c157f080, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c157c640, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -92,7 +92,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 954376K/1048576K available (10240K kernel code, 1563K rwdata, 4464K rodata, 2048K init, 404K bss, 28664K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 954384K/1048576K available (10240K kernel code, 1553K rwdata, 4476K rodata, 2048K init, 404K bss, 28656K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -102,8 +102,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1586ea0   (1564 kB)
-       .bss : 0xc158fc98 - 0xc15f507c   ( 405 kB)
+      .data : 0xc1400000 - 0xc1584460   (1554 kB)
+       .bss : 0xc158dc98 - 0xc15f307c   ( 405 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -348,8 +348,8 @@
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
-input: gpio-keys as /devices/soc0/gpio-keys/input/input0
 hub 1-1:1.0: USB hub found
+input: gpio-keys as /devices/soc0/gpio-keys/input/input0
 rtc-em3027 2-0056: hctosys: unable to read the hardware clock
 hub 1-1:1.0: 4 ports detected
 mmcblk1: error -5 sending stop command, original cmd response 0x900, card status 0x900
@@ -377,5 +377,5 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/#  
+root@(none):/# 
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2017-07-31 00:11:52.720974608 -0700
+++ /tmp/foob	2017-07-31 00:11:52.720974608 -0700
@@ -46,9 +46,9 @@
 	 #################################################################
 	 #################################################################
 	 ##############################
-	 4.1 MiB/s
+	 4 MiB/s
 done
-Bytes transferred = 6152792 (5de258 hex)
+Bytes transferred = 6154216 (5de7e8 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -59,7 +59,7 @@
 done
 Bytes transferred = 21454 (53ce hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x5de258 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x5de7e8 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103cd
@@ -68,7 +68,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc2-next-20170728-188650-gb2cf822 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Jul 27 22:49:58 PDT 2017
+Linux version 4.13.0-rc3-next-20170731-189275-g742f220 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Jul 31 00:04:51 PDT 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -77,7 +77,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0x3c000000
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0eafb40, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0ead140, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -90,7 +90,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 959384K/1048576K available (9216K kernel code, 744K rwdata, 2736K rodata, 1024K init, 299K bss, 23656K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 959396K/1048576K available (9216K kernel code, 734K rwdata, 2744K rodata, 1024K init, 299K bss, 23644K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -100,8 +100,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0eba360   ( 745 kB)
-       .bss : 0xc0ec45e0 - 0xc0f0f3bc   ( 300 kB)
+      .data : 0xc0e00000 - 0xc0eb7960   ( 735 kB)
+       .bss : 0xc0ec15e0 - 0xc0f0c3bc   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
 	Tasks RCU enabled.
@@ -332,22 +332,22 @@
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
+tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 mmcblk1: error -5 sending stop command, original cmd response 0x900, card status 0x900
 mmcblk1: error -5 transferring data, sector 3839992, nr 8, cmd response 0x900, card status 0x80000b00
-tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
-mmcblk1: error -5 sending stop command, original cmd response 0x900, card status 0x900
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
+mmcblk1: error -5 sending stop command, original cmd response 0x900, card status 0x900
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
+mmcblk1: error -5 transferring data, sector 3839992, nr 8, cmd response 0x900, card status 0x80000b00
 rtc-em3027 2-0056: hctosys: unable to read the hardware clock
 ALSA device list:
   #0: tegra-trimslice
-mmcblk1: error -5 transferring data, sector 3839992, nr 8, cmd response 0x900, card status 0x80000b00
-mmcblk1: retrying using single block read
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
+mmcblk1: retrying using single block read
  mmcblk1: p1 p2
 Waiting for root device PARTUUID=11eb23c1-01...
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
@@ -356,8 +356,8 @@
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
 mmcblk0: error -5 sending stop command, original cmd response 0x900, card status 0x900
 mmcblk0: error -5 transferring data, sector 7759864, nr 8, cmd response 0x900, card status 0x80000b00
-mmcblk0: error -5 sending stop command, original cmd response 0x900, card status 0x900
 asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
+mmcblk0: error -5 sending stop command, original cmd response 0x900, card status 0x900
 mmcblk0: error -5 transferring data, sector 7759864, nr 8, cmd response 0x900, card status 0x80000b00
 mmcblk0: retrying using single block read
  mmcblk0: p1
@@ -370,5 +370,5 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/#  
+root@(none):/# 
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2017-07-31 00:11:52.736974667 -0700
+++ /tmp/foob	2017-07-31 00:11:52.740974681 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 745107 us
-Basic modules init took 348663 us
+Tegraboot started after 765073 us
+Basic modules init took 348821 us
 Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14366 us
-Carveout took 252670 us
-CPU initialization took 92438 us
-Total time taken by TegraBoot 708137 us
+Next stage binary read took 14390 us
+Carveout took 252651 us
+CPU initialization took 92408 us
+Total time taken by TegraBoot 708270 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.584] LPDDR4 Training: Number of tables = 10
+64b[0001.603] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-16019968 bytes read in 814 ms (18.8 MiB/s)
+16148992 bytes read in 837 ms (18.4 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-45170 bytes read in 460 ms (95.7 KiB/s)
+45170 bytes read in 454 ms (96.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 494 ms (3.1 MiB/s)
+1627787 bytes read in 497 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc2-next-20170728-306891-gb2cf822 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Thu Jul 27 22:54:57 PDT 2017
+Linux version 4.13.0-rc3-next-20170731-307516-g742f220 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Mon Jul 31 00:09:12 PDT 2017
 Boot CPU: AArch64 Processor [411fd071]
 Machine model: NVIDIA Tegra210 P2371 (P2530/P2595) reference design
 debug: ignoring loglevel setting.
@@ -142,9 +142,6 @@
 cma: Reserved 16 MiB at 0x00000000fe000000
 NUMA: No NUMA configuration found
 NUMA: Faking a node at [mem 0x0000000000000000-0x000000013fffffff]
-NUMA: Adding memblock [0x80000000 - 0xff1fffff] on node 0
-NUMA: Adding memblock [0x100000000 - 0x13fffffff] on node 0
-NUMA: Initmem setup node 0 [mem 0x80000000-0x13fffffff]
 NUMA: NODE_DATA [mem 0x13ffaba80-0x13ffad57f]
 Zone ranges:
   DMA      [mem 0x0000000080000000-0x00000000ffffffff]
@@ -175,15 +172,15 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffff80007a000000-ffff80007dffffff]
-Memory: 2982980K/3131392K available (9020K kernel code, 1116K rwdata, 4396K rodata, 1088K init, 403K bss, 132028K reserved, 16384K cma-reserved)
+Memory: 2982856K/3131392K available (9084K kernel code, 1114K rwdata, 4440K rodata, 1088K init, 401K bss, 132152K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff000008950000   (  9024 KB)
-    .rodata : 0xffff000008950000 - 0xffff000008da0000   (  4416 KB)
-      .init : 0xffff000008da0000 - 0xffff000008eb0000   (  1088 KB)
-      .data : 0xffff000008eb0000 - 0xffff000008fc7200   (  1117 KB)
-       .bss : 0xffff000008fc7200 - 0xffff00000902c0f0   (   404 KB)
+      .text : 0xffff000008080000 - 0xffff000008960000   (  9088 KB)
+    .rodata : 0xffff000008960000 - 0xffff000008dc0000   (  4480 KB)
+      .init : 0xffff000008dc0000 - 0xffff000008ed0000   (  1088 KB)
+      .data : 0xffff000008ed0000 - 0xffff000008fe6a00   (  1115 KB)
+       .bss : 0xffff000008fe6a00 - 0xffff00000904b0f0   (   402 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -236,7 +233,7 @@
 DMI not present or invalid.
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffff000008956000, 1 data @ ffff000008eb5000)
+vdso: 2 pages (1 code @ ffff000008966000, 1 data @ ffff000008ed5000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -271,14 +268,14 @@
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 PCI: CLS 0 bytes, default 128
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 80942000
+kvm [1]: IDMAP page: 80948000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
 kvm [1]: vgic interrupt IRQ1
 kvm [1]: virtual timer IRQ4
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.507:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.519:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2017-07-31 00:11:52.744974697 -0700
+++ /tmp/foob	2017-07-31 00:11:52.744974697 -0700
@@ -1,14 +1,9 @@
-
-efi: Getting EFI parameters from FDT:
-efi: UEFI not found.
+00] efi: UEFI not found.
 cma: Reserved 16 MiB at 0x00000000f5400000
 earlycon: uart0 at MMIO 0x0000000070006000 (options '115200n8')
 bootconsole [uart0] enabled
 NUMA: No NUMA configuration found
 NUMA: Faking a node at [mem 0x0000000000000000-0x000000013fffffff]
-NUMA: Adding memblock [0x80000000 - 0xf65fffff] on node 0
-NUMA: Adding memblock [0x100000000 - 0x13fffffff] on node 0
-NUMA: Initmem setup node 0 [mem 0x80000000-0x13fffffff]
 NUMA: NODE_DATA [mem 0x13ff9fc80-0x13ffa177f]
 Zone ranges:
   DMA      [mem 0x0000000080000000-0x00000000ffffffff]
@@ -31,15 +26,15 @@
 Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2838248K/2988032K available (9020K kernel code, 1116K rwdata, 4396K rodata, 1088K init, 403K bss, 133400K reserved, 16384K cma-reserved)
+Memory: 2838120K/2988032K available (9084K kernel code, 1114K rwdata, 4440K rodata, 1088K init, 401K bss, 133528K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff000008950000   (  9024 KB)
-    .rodata : 0xffff000008950000 - 0xffff000008da0000   (  4416 KB)
-      .init : 0xffff000008da0000 - 0xffff000008eb0000   (  1088 KB)
-      .data : 0xffff000008eb0000 - 0xffff000008fc7200   (  1117 KB)
-       .bss : 0xffff000008fc7200 - 0xffff00000902c0f0   (   404 KB)
+      .text : 0xffff000008080000 - 0xffff000008960000   (  9088 KB)
+    .rodata : 0xffff000008960000 - 0xffff000008dc0000   (  4480 KB)
+      .init : 0xffff000008dc0000 - 0xffff000008ed0000   (  1088 KB)
+      .data : 0xffff000008ed0000 - 0xffff000008fe6a00   (  1115 KB)
+       .bss : 0xffff000008fe6a00 - 0xffff00000904b0f0   (   402 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -65,7 +60,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x8cd: expected 0x2004f400
+boot misc3 0x8ce: expected 0x2004f400
  (comparison mask = 0xffffffff)
 boot misc1 0x0: expected 0x20
  (comparison mask = 0xffffff)
@@ -77,7 +72,7 @@
 console [tty0] enabled
 bootconsole [uart0] disabled
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc2-next-20170728-306891-gb2cf822 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Thu Jul 27 22:54:57 PDT 2017
+Linux version 4.13.0-rc3-next-20170731-307516-g742f220 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Mon Jul 31 00:09:12 PDT 2017
 Boot CPU: AArch64 Processor [411fd071]
 Machine model: Google Pixel C
 efi: Getting EFI parameters from FDT:
@@ -87,9 +82,6 @@
 bootconsole [uart0] enabled
 NUMA: No NUMA configuration found
 NUMA: Faking a node at [mem 0x0000000000000000-0x000000013fffffff]
-NUMA: Adding memblock [0x80000000 - 0xf65fffff] on node 0
-NUMA: Adding memblock [0x100000000 - 0x13fffffff] on node 0
-NUMA: Initmem setup node 0 [mem 0x80000000-0x13fffffff]
 NUMA: NODE_DATA [mem 0x13ff9fc80-0x13ffa177f]
 Zone ranges:
   DMA      [mem 0x0000000080000000-0x00000000ffffffff]
@@ -112,15 +104,15 @@
 Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2838248K/2988032K available (9020K kernel code, 1116K rwdata, 4396K rodata, 1088K init, 403K bss, 133400K reserved, 16384K cma-reserved)
+Memory: 2838120K/2988032K available (9084K kernel code, 1114K rwdata, 4440K rodata, 1088K init, 401K bss, 133528K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
-      .text : 0xffff000008080000 - 0xffff000008950000   (  9024 KB)
-    .rodata : 0xffff000008950000 - 0xffff000008da0000   (  4416 KB)
-      .init : 0xffff000008da0000 - 0xffff000008eb0000   (  1088 KB)
-      .data : 0xffff000008eb0000 - 0xffff000008fc7200   (  1117 KB)
-       .bss : 0xffff000008fc7200 - 0xffff00000902c0f0   (   404 KB)
+      .text : 0xffff000008080000 - 0xffff000008960000   (  9088 KB)
+    .rodata : 0xffff000008960000 - 0xffff000008dc0000   (  4480 KB)
+      .init : 0xffff000008dc0000 - 0xffff000008ed0000   (  1088 KB)
+      .data : 0xffff000008ed0000 - 0xffff000008fe6a00   (  1115 KB)
+       .bss : 0xffff000008fe6a00 - 0xffff00000904b0f0   (   402 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -146,7 +138,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x8cd: expected 0x2004f400
+boot misc3 0x8ce: expected 0x2004f400
  (comparison mask = 0xffffffff)
 boot misc1 0x0: expected 0x20
  (comparison mask = 0xffffff)
@@ -189,7 +181,7 @@
 DMI not present or invalid.
 NET: Registered protocol family 16
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ ffff000008956000, 1 data @ ffff000008eb5000)
+vdso: 2 pages (1 code @ ffff000008966000, 1 data @ ffff000008ed5000)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -229,16 +221,16 @@
 RPC: Registered tcp transport module.
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 Unpacking initramfs...
-Freeing initrd memory: 1280K
+Freeing initrd memory: 1284K
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 80942000
+kvm [1]: IDMAP page: 80948000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
 kvm [1]: vgic interrupt IRQ1
 kvm [1]: virtual timer IRQ4
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.267:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.263:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -334,15 +326,16 @@
 9pnet: Installing 9P2000 support
 Key type dns_resolver registered
 registered taskstats version 1
-mmc0: new high speed MMC card at address 0001
 tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
 tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
-max77686-rtc max77620-rtc: setting system clock to 2001-05-08 11:20:09 UTC (989320809)
+mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 BGND3R 29.1 GiB 
 mmcblk0boot0: mmc0:0001 BGND3R partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
+max77686-rtc max77620-rtc: setting system clock to 2001-05-11 12:34:25 UTC (989584465)
 PP1200_AVDD: disabling
 PP1200_RCAM: disabling
 PP2800L_RCAM: disabling
@@ -353,7 +346,6 @@
 PP1800_CAM: disabling
 ALSA device list:
   No soundcards found.
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
 tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 Freeing unused kernel memory: 1088K
 
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-07-31 00:11:52.768974784 -0700
+++ /tmp/foob	2017-07-31 00:11:52.768974784 -0700
@@ -40,20 +40,20 @@
 	 #################################################################
 	 #################################################################
 	 ###########################
-	 3.1 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 8025640 (7a7628 hex)
+Bytes transferred = 8026896 (7a7b10 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.7 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 51848 (ca88 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x7a7628 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x7a7b10 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa87 ... OK
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc2-next-20170728-188650-gb2cf822 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Jul 27 22:49:59 PDT 2017
+Linux version 4.13.0-rc3-next-20170731-189275-g742f220 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Mon Jul 31 00:05:02 PDT 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -71,7 +71,7 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c157f080, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c157c640, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -84,7 +84,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993464K/2096128K available (10240K kernel code, 1563K rwdata, 4464K rodata, 2048K init, 404K bss, 37128K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993472K/2096128K available (10240K kernel code, 1553K rwdata, 4476K rodata, 2048K init, 404K bss, 37120K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -94,8 +94,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1586ea0   (1564 kB)
-       .bss : 0xc158fc98 - 0xc15f507c   ( 405 kB)
+      .data : 0xc1400000 - 0xc1584460   (1554 kB)
+       .bss : 0xc158dc98 - 0xc15f307c   ( 405 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	RCU event tracing is enabled.
@@ -400,5 +400,4 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.083755] usb 2-1: new high-speed USB device number 2 using tegra-ehci
-asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0f:67:60
+root@(none):/# 
\ No newline at end of file
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2017-07-31 00:11:52.776974814 -0700
+++ /tmp/foob	2017-07-31 00:11:52.776974814 -0700
@@ -38,9 +38,9 @@
 	 #################################################################
 	 #################################################################
 	 ##############################
-	 3.2 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 6152792 (5de258 hex)
+Bytes transferred = 6154216 (5de7e8 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -51,7 +51,7 @@
 done
 Bytes transferred = 51848 (ca88 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5de258 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5de7e8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa87 ... OK
@@ -60,7 +60,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc2-next-20170728-188650-gb2cf822 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Jul 27 22:49:58 PDT 2017
+Linux version 4.13.0-rc3-next-20170731-189275-g742f220 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Jul 31 00:04:51 PDT 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -69,7 +69,7 @@
 Memory policy: Data cache writealloc
 cma: Reserved 64 MiB at 0xfbc00000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0eafb40, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0ead140, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1998476K/2096128K available (9216K kernel code, 744K rwdata, 2736K rodata, 1024K init, 299K bss, 32116K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1998488K/2096128K available (9216K kernel code, 734K rwdata, 2744K rodata, 1024K init, 299K bss, 32104K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,8 +92,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0008000 - 0xc0a00000   (10208 kB)
       .init : 0xc0d00000 - 0xc0e00000   (1024 kB)
-      .data : 0xc0e00000 - 0xc0eba360   ( 745 kB)
-       .bss : 0xc0ec45e0 - 0xc0f0f3bc   ( 300 kB)
+      .data : 0xc0e00000 - 0xc0eb7960   ( 735 kB)
+       .bss : 0xc0ec15e0 - 0xc0f0c3bc   ( 300 kB)
 Preemptible hierarchical RCU implementation.
 	Tasks RCU enabled.
 NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
@@ -394,4 +394,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.390694] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.370505] usb 2-1: new high-speed USB device number 2 using tegra-ehci
