Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2015-06-29 21:51:42.508757523 -0700
+++ /tmp/foob	2015-06-29 21:51:42.508757523 -0700
@@ -1,6 +1,6 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
-Linux version 4.1.0-next-20150626+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Jun 29 21:33:56 PDT 2015
+Linux version 4.1.0-next-20150629+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Jun 29 21:48:51 PDT 2015
 CPU: AArch64 Processor [411fd070] revision 0
 Detected PIPT I-cache on CPU0
 alternatives: enabling workaround for ARM erratum 832075
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-06-29 21:51:42.516757522 -0700
+++ /tmp/foob	2015-06-29 21:51:42.516757522 -0700
@@ -38,20 +38,20 @@
 	 #################################################################
 	 #################################################################
 	 ###############################
-	 3.3 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 6174880 (5e38a0 hex)
+Bytes transferred = 6174856 (5e3888 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3.3 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5e38a0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5e3888 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.1.0-next-20150626-26998-g043831b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jun 29 21:32:41 PDT 2015
+Linux version 4.1.0-next-20150629-27202-ge398431 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jun 29 21:47:34 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -67,7 +67,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f07ac0, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0f07b00, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -90,7 +90,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d1ac08   (11340 kB)
       .init : 0xc0d1b000 - 0xc0e06000   ( 940 kB)
-      .data : 0xc0e06000 - 0xc0f09ac0   (1039 kB)
+      .data : 0xc0e06000 - 0xc0f09b00   (1039 kB)
        .bss : 0xc0f0c000 - 0xc0f5dc28   ( 328 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -278,67 +278,67 @@
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
 ------------[ cut here ]------------
-WARNING: CPU: 1 PID: 58 at drivers/gpu/drm/drm_irq.c:1323 drm_vblank_on+0xf0/0x118()
+WARNING: CPU: 0 PID: 58 at drivers/gpu/drm/drm_irq.c:1323 drm_vblank_on+0xf0/0x118()
 Modules linked in:
-CPU: 1 PID: 58 Comm: kworker/u8:1 Not tainted 4.1.0-next-20150626-26998-g043831b #1
+CPU: 0 PID: 58 Comm: kworker/u8:1 Not tainted 4.1.0-next-20150629-27202-ge398431 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0217d68>] (unwind_backtrace) from [<c02136d0>] (show_stack+0x10/0x14)
-[<c02136d0>] (show_stack) from [<c09605bc>] (dump_stack+0x78/0x94)
-[<c09605bc>] (dump_stack) from [<c024c374>] (warn_slowpath_common+0x74/0xb0)
+[<c02136d0>] (show_stack) from [<c0960648>] (dump_stack+0x78/0x94)
+[<c0960648>] (dump_stack) from [<c024c374>] (warn_slowpath_common+0x74/0xb0)
 [<c024c374>] (warn_slowpath_common) from [<c024c3cc>] (warn_slowpath_null+0x1c/0x24)
-[<c024c3cc>] (warn_slowpath_null) from [<c05c312c>] (drm_vblank_on+0xf0/0x118)
-[<c05c312c>] (drm_vblank_on) from [<c05b8b8c>] (drm_atomic_helper_commit_modeset_enables+0x8c/0x180)
-[<c05b8b8c>] (drm_atomic_helper_commit_modeset_enables) from [<c05fff24>] (tegra_atomic_complete+0x2c/0x50)
-[<c05fff24>] (tegra_atomic_complete) from [<c05fffa8>] (tegra_atomic_commit+0x60/0x90)
-[<c05fffa8>] (tegra_atomic_commit) from [<c05da588>] (drm_atomic_commit+0x40/0x64)
-[<c05da588>] (drm_atomic_commit) from [<c05b88ec>] (drm_atomic_helper_set_config+0x394/0x3ec)
-[<c05b88ec>] (drm_atomic_helper_set_config) from [<c05ca724>] (drm_mode_set_config_internal+0x58/0xd4)
-[<c05ca724>] (drm_mode_set_config_internal) from [<c05bad78>] (restore_fbdev_mode+0xd4/0xf4)
-[<c05bad78>] (restore_fbdev_mode) from [<c05bc4b4>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
-[<c05bc4b4>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c05bc50c>] (drm_fb_helper_set_par+0x18/0x34)
-[<c05bc50c>] (drm_fb_helper_set_par) from [<c050a608>] (fbcon_init+0x474/0x4dc)
-[<c050a608>] (fbcon_init) from [<c05735f8>] (visual_init+0x9c/0xe8)
-[<c05735f8>] (visual_init) from [<c05755b0>] (do_bind_con_driver+0x134/0x354)
-[<c05755b0>] (do_bind_con_driver) from [<c0575940>] (do_take_over_console+0x170/0x1dc)
-[<c0575940>] (do_take_over_console) from [<c050a6d4>] (do_fbcon_takeover+0x64/0xcc)
-[<c050a6d4>] (do_fbcon_takeover) from [<c0265254>] (notifier_call_chain+0x44/0x84)
+[<c024c3cc>] (warn_slowpath_null) from [<c05c30b4>] (drm_vblank_on+0xf0/0x118)
+[<c05c30b4>] (drm_vblank_on) from [<c05b8b14>] (drm_atomic_helper_commit_modeset_enables+0x8c/0x180)
+[<c05b8b14>] (drm_atomic_helper_commit_modeset_enables) from [<c05ffeac>] (tegra_atomic_complete+0x2c/0x50)
+[<c05ffeac>] (tegra_atomic_complete) from [<c05fff30>] (tegra_atomic_commit+0x60/0x90)
+[<c05fff30>] (tegra_atomic_commit) from [<c05da510>] (drm_atomic_commit+0x40/0x64)
+[<c05da510>] (drm_atomic_commit) from [<c05b8874>] (drm_atomic_helper_set_config+0x394/0x3ec)
+[<c05b8874>] (drm_atomic_helper_set_config) from [<c05ca6ac>] (drm_mode_set_config_internal+0x58/0xd4)
+[<c05ca6ac>] (drm_mode_set_config_internal) from [<c05bad00>] (restore_fbdev_mode+0xd4/0xf4)
+[<c05bad00>] (restore_fbdev_mode) from [<c05bc43c>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
+[<c05bc43c>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c05bc494>] (drm_fb_helper_set_par+0x18/0x34)
+[<c05bc494>] (drm_fb_helper_set_par) from [<c050a5e8>] (fbcon_init+0x474/0x4dc)
+[<c050a5e8>] (fbcon_init) from [<c0573580>] (visual_init+0x9c/0xe8)
+[<c0573580>] (visual_init) from [<c0575538>] (do_bind_con_driver+0x134/0x354)
+[<c0575538>] (do_bind_con_driver) from [<c05758c8>] (do_take_over_console+0x170/0x1dc)
+[<c05758c8>] (do_take_over_console) from [<c050a6b4>] (do_fbcon_takeover+0x64/0xcc)
+[<c050a6b4>] (do_fbcon_takeover) from [<c0265254>] (notifier_call_chain+0x44/0x84)
 [<c0265254>] (notifier_call_chain) from [<c0265468>] (__blocking_notifier_call_chain+0x48/0x60)
 [<c0265468>] (__blocking_notifier_call_chain) from [<c0265498>] (blocking_notifier_call_chain+0x18/0x20)
-[<c0265498>] (blocking_notifier_call_chain) from [<c0514e6c>] (register_framebuffer+0x184/0x264)
-[<c0514e6c>] (register_framebuffer) from [<c05bc76c>] (drm_fb_helper_initial_config+0x244/0x408)
-[<c05bc76c>] (drm_fb_helper_initial_config) from [<c0601b38>] (tegra_drm_fb_init+0x48/0xb4)
-[<c0601b38>] (tegra_drm_fb_init) from [<c05ffe44>] (tegra_drm_load+0x170/0x224)
-[<c05ffe44>] (tegra_drm_load) from [<c05c5428>] (drm_dev_register+0xa0/0x100)
-[<c05c5428>] (drm_dev_register) from [<c05ff9a0>] (host1x_drm_probe+0x44/0xb0)
-[<c05ff9a0>] (host1x_drm_probe) from [<c05a7d80>] (host1x_device_probe+0x1c/0x28)
-[<c05a7d80>] (host1x_device_probe) from [<c06170c8>] (driver_probe_device+0x1c8/0x26c)
-[<c06170c8>] (driver_probe_device) from [<c06157d0>] (bus_for_each_drv+0x44/0x8c)
-[<c06157d0>] (bus_for_each_drv) from [<c0616e88>] (__device_attach+0x88/0xd8)
-[<c0616e88>] (__device_attach) from [<c0616648>] (bus_probe_device+0x84/0x8c)
-[<c0616648>] (bus_probe_device) from [<c0614b74>] (device_add+0x33c/0x524)
-[<c0614b74>] (device_add) from [<c05a7f8c>] (host1x_subdev_register+0xb0/0xd4)
-[<c05a7f8c>] (host1x_subdev_register) from [<c05a884c>] (host1x_client_register+0xf4/0x11c)
-[<c05a884c>] (host1x_client_register) from [<c06097d8>] (tegra_hdmi_probe+0x170/0x250)
-[<c06097d8>] (tegra_hdmi_probe) from [<c0618688>] (platform_drv_probe+0x48/0xa4)
-[<c0618688>] (platform_drv_probe) from [<c06170c8>] (driver_probe_device+0x1c8/0x26c)
-[<c06170c8>] (driver_probe_device) from [<c06157d0>] (bus_for_each_drv+0x44/0x8c)
-[<c06157d0>] (bus_for_each_drv) from [<c0616e88>] (__device_attach+0x88/0xd8)
-[<c0616e88>] (__device_attach) from [<c0616648>] (bus_probe_device+0x84/0x8c)
-[<c0616648>] (bus_probe_device) from [<c0616a20>] (deferred_probe_work_func+0x5c/0x88)
-[<c0616a20>] (deferred_probe_work_func) from [<c025f280>] (process_one_work+0x134/0x338)
+[<c0265498>] (blocking_notifier_call_chain) from [<c0514e4c>] (register_framebuffer+0x184/0x264)
+[<c0514e4c>] (register_framebuffer) from [<c05bc6f4>] (drm_fb_helper_initial_config+0x244/0x408)
+[<c05bc6f4>] (drm_fb_helper_initial_config) from [<c0601ac0>] (tegra_drm_fb_init+0x48/0xb4)
+[<c0601ac0>] (tegra_drm_fb_init) from [<c05ffdcc>] (tegra_drm_load+0x170/0x224)
+[<c05ffdcc>] (tegra_drm_load) from [<c05c53b0>] (drm_dev_register+0xa0/0x100)
+[<c05c53b0>] (drm_dev_register) from [<c05ff928>] (host1x_drm_probe+0x44/0xb0)
+[<c05ff928>] (host1x_drm_probe) from [<c05a7d08>] (host1x_device_probe+0x1c/0x28)
+[<c05a7d08>] (host1x_device_probe) from [<c0617050>] (driver_probe_device+0x1c8/0x26c)
+[<c0617050>] (driver_probe_device) from [<c0615758>] (bus_for_each_drv+0x44/0x8c)
+[<c0615758>] (bus_for_each_drv) from [<c0616e10>] (__device_attach+0x88/0xd8)
+[<c0616e10>] (__device_attach) from [<c06165d0>] (bus_probe_device+0x84/0x8c)
+[<c06165d0>] (bus_probe_device) from [<c0614afc>] (device_add+0x33c/0x524)
+[<c0614afc>] (device_add) from [<c05a7f14>] (host1x_subdev_register+0xb0/0xd4)
+[<c05a7f14>] (host1x_subdev_register) from [<c05a87d4>] (host1x_client_register+0xf4/0x11c)
+[<c05a87d4>] (host1x_client_register) from [<c0609760>] (tegra_hdmi_probe+0x170/0x250)
+[<c0609760>] (tegra_hdmi_probe) from [<c0618610>] (platform_drv_probe+0x48/0xa4)
+[<c0618610>] (platform_drv_probe) from [<c0617050>] (driver_probe_device+0x1c8/0x26c)
+[<c0617050>] (driver_probe_device) from [<c0615758>] (bus_for_each_drv+0x44/0x8c)
+[<c0615758>] (bus_for_each_drv) from [<c0616e10>] (__device_attach+0x88/0xd8)
+[<c0616e10>] (__device_attach) from [<c06165d0>] (bus_probe_device+0x84/0x8c)
+[<c06165d0>] (bus_probe_device) from [<c06169a8>] (deferred_probe_work_func+0x5c/0x88)
+[<c06169a8>] (deferred_probe_work_func) from [<c025f280>] (process_one_work+0x134/0x338)
 [<c025f280>] (process_one_work) from [<c025f4e0>] (worker_thread+0x28/0x4cc)
 [<c025f4e0>] (worker_thread) from [<c0264630>] (kthread+0xd8/0xf4)
 [<c0264630>] (kthread) from [<c0210a68>] (ret_from_fork+0x14/0x2c)
----[ end trace 5b8be6e8f6a64b3d ]---
+---[ end trace 950486941571397e ]---
 mmc1: MAN_BKOPS_EN bit is not set
 mmc1: new high speed MMC card at address 0001
 mmcblk0: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk0boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmcblk0boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
 mmcblk0rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
-Console: switching to colour frame buffer device 240x75
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
+Console: switching to colour frame buffer device 240x75
 drm drm: fb0:  frame buffer device
 drm drm: registered panic notifier
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
@@ -350,7 +350,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-10 13:24:08 UTC (947510648)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-10 13:39:06 UTC (947511546)
 vdd_cam_1v8_reg: disabling
 usb1_vbus: disabling
 dvdd_ts: disabling
@@ -362,4 +362,4 @@
 Freeing unused kernel memory: 940K (c0d1b000 - c0e06000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.658959] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.678940] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2015-06-29 21:51:42.524757519 -0700
+++ /tmp/foob	2015-06-29 21:51:42.524757519 -0700
@@ -36,10 +36,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #######################
+	 ######################
 	 3.3 MiB/s
 done
-Bytes transferred = 5094160 (4dbb10 hex)
+Bytes transferred = 5093936 (4dba30 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4dbb10 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4dba30 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.1.0-next-20150626-26998-g043831b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jun 29 21:33:09 PDT 2015
+Linux version 4.1.0-next-20150629-27202-ge398431 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jun 29 21:47:59 PDT 2015
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra114 Dalmore evaluation board
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c09d2dc0, node_mem_map ee7f7000
+free_area_init_node: node 0, pgdat c09d2e00, node_mem_map ee7f7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -90,9 +90,9 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0930b60   (9379 kB)
+      .text : 0xc0008000 - 0xc0930b68   (9379 kB)
       .init : 0xc0931000 - 0xc097e000   ( 308 kB)
-      .data : 0xc097e000 - 0xc09d4980   ( 347 kB)
+      .data : 0xc097e000 - 0xc09d49c0   ( 347 kB)
        .bss : 0xc09d7000 - 0xc0a31c94   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Additional per-CPU info printed with stalls.
@@ -286,57 +286,57 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 6 at drivers/gpu/drm/drm_irq.c:1323 drm_vblank_on+0xf0/0x118()
 Modules linked in:
-CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.1.0-next-20150626-26998-g043831b #1
+CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.1.0-next-20150629-27202-ge398431 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c00167f8>] (unwind_backtrace) from [<c0012124>] (show_stack+0x10/0x14)
-[<c0012124>] (show_stack) from [<c0698ef0>] (dump_stack+0x84/0xd0)
-[<c0698ef0>] (dump_stack) from [<c002683c>] (warn_slowpath_common+0x74/0xb0)
+[<c0012124>] (show_stack) from [<c0698f98>] (dump_stack+0x84/0xd0)
+[<c0698f98>] (dump_stack) from [<c002683c>] (warn_slowpath_common+0x74/0xb0)
 [<c002683c>] (warn_slowpath_common) from [<c0026894>] (warn_slowpath_null+0x1c/0x24)
-[<c0026894>] (warn_slowpath_null) from [<c02e7bc0>] (drm_vblank_on+0xf0/0x118)
-[<c02e7bc0>] (drm_vblank_on) from [<c02ddf34>] (drm_atomic_helper_commit_modeset_enables+0x8c/0x180)
-[<c02ddf34>] (drm_atomic_helper_commit_modeset_enables) from [<c0302180>] (tegra_atomic_complete+0x2c/0x50)
-[<c0302180>] (tegra_atomic_complete) from [<c0302204>] (tegra_atomic_commit+0x60/0x90)
-[<c0302204>] (tegra_atomic_commit) from [<c02ff290>] (drm_atomic_commit+0x40/0x64)
-[<c02ff290>] (drm_atomic_commit) from [<c02ddc94>] (drm_atomic_helper_set_config+0x394/0x3ec)
-[<c02ddc94>] (drm_atomic_helper_set_config) from [<c02ef2a4>] (drm_mode_set_config_internal+0x58/0xd4)
-[<c02ef2a4>] (drm_mode_set_config_internal) from [<c02e0120>] (restore_fbdev_mode+0xd4/0xf4)
-[<c02e0120>] (restore_fbdev_mode) from [<c02e185c>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
-[<c02e185c>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c02e18b4>] (drm_fb_helper_set_par+0x18/0x34)
-[<c02e18b4>] (drm_fb_helper_set_par) from [<c027f458>] (fbcon_init+0x474/0x4dc)
-[<c027f458>] (fbcon_init) from [<c02b4a04>] (visual_init+0x9c/0xe8)
-[<c02b4a04>] (visual_init) from [<c02b69a8>] (do_bind_con_driver+0x134/0x354)
-[<c02b69a8>] (do_bind_con_driver) from [<c02b6d38>] (do_take_over_console+0x170/0x1dc)
-[<c02b6d38>] (do_take_over_console) from [<c027f524>] (do_fbcon_takeover+0x64/0xcc)
-[<c027f524>] (do_fbcon_takeover) from [<c003f0d0>] (notifier_call_chain+0x44/0x84)
+[<c0026894>] (warn_slowpath_null) from [<c02e7b40>] (drm_vblank_on+0xf0/0x118)
+[<c02e7b40>] (drm_vblank_on) from [<c02ddeb4>] (drm_atomic_helper_commit_modeset_enables+0x8c/0x180)
+[<c02ddeb4>] (drm_atomic_helper_commit_modeset_enables) from [<c0302100>] (tegra_atomic_complete+0x2c/0x50)
+[<c0302100>] (tegra_atomic_complete) from [<c0302184>] (tegra_atomic_commit+0x60/0x90)
+[<c0302184>] (tegra_atomic_commit) from [<c02ff210>] (drm_atomic_commit+0x40/0x64)
+[<c02ff210>] (drm_atomic_commit) from [<c02ddc14>] (drm_atomic_helper_set_config+0x394/0x3ec)
+[<c02ddc14>] (drm_atomic_helper_set_config) from [<c02ef224>] (drm_mode_set_config_internal+0x58/0xd4)
+[<c02ef224>] (drm_mode_set_config_internal) from [<c02e00a0>] (restore_fbdev_mode+0xd4/0xf4)
+[<c02e00a0>] (restore_fbdev_mode) from [<c02e17dc>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
+[<c02e17dc>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c02e1834>] (drm_fb_helper_set_par+0x18/0x34)
+[<c02e1834>] (drm_fb_helper_set_par) from [<c027f3d8>] (fbcon_init+0x474/0x4dc)
+[<c027f3d8>] (fbcon_init) from [<c02b4984>] (visual_init+0x9c/0xe8)
+[<c02b4984>] (visual_init) from [<c02b6928>] (do_bind_con_driver+0x134/0x354)
+[<c02b6928>] (do_bind_con_driver) from [<c02b6cb8>] (do_take_over_console+0x170/0x1dc)
+[<c02b6cb8>] (do_take_over_console) from [<c027f4a4>] (do_fbcon_takeover+0x64/0xcc)
+[<c027f4a4>] (do_fbcon_takeover) from [<c003f0d0>] (notifier_call_chain+0x44/0x84)
 [<c003f0d0>] (notifier_call_chain) from [<c003f2a4>] (__blocking_notifier_call_chain+0x48/0x60)
 [<c003f2a4>] (__blocking_notifier_call_chain) from [<c003f2d4>] (blocking_notifier_call_chain+0x18/0x20)
-[<c003f2d4>] (blocking_notifier_call_chain) from [<c0289418>] (register_framebuffer+0x184/0x264)
-[<c0289418>] (register_framebuffer) from [<c02e1b14>] (drm_fb_helper_initial_config+0x244/0x408)
-[<c02e1b14>] (drm_fb_helper_initial_config) from [<c0303e48>] (tegra_drm_fb_init+0x48/0xb4)
-[<c0303e48>] (tegra_drm_fb_init) from [<c03020a0>] (tegra_drm_load+0x170/0x224)
-[<c03020a0>] (tegra_drm_load) from [<c02e9ed4>] (drm_dev_register+0xa0/0x100)
-[<c02e9ed4>] (drm_dev_register) from [<c0301bfc>] (host1x_drm_probe+0x44/0xb0)
-[<c0301bfc>] (host1x_drm_probe) from [<c02cd14c>] (host1x_device_probe+0x1c/0x28)
-[<c02cd14c>] (host1x_device_probe) from [<c031937c>] (driver_probe_device+0x1c8/0x26c)
-[<c031937c>] (driver_probe_device) from [<c0317a8c>] (bus_for_each_drv+0x44/0x8c)
-[<c0317a8c>] (bus_for_each_drv) from [<c031913c>] (__device_attach+0x88/0xd8)
-[<c031913c>] (__device_attach) from [<c03188fc>] (bus_probe_device+0x84/0x8c)
-[<c03188fc>] (bus_probe_device) from [<c0316e50>] (device_add+0x33c/0x524)
-[<c0316e50>] (device_add) from [<c02cd358>] (host1x_subdev_register+0xb0/0xd4)
-[<c02cd358>] (host1x_subdev_register) from [<c02cdc18>] (host1x_client_register+0xf4/0x11c)
-[<c02cdc18>] (host1x_client_register) from [<c030badc>] (tegra_hdmi_probe+0x170/0x250)
-[<c030badc>] (tegra_hdmi_probe) from [<c031a83c>] (platform_drv_probe+0x48/0xa4)
-[<c031a83c>] (platform_drv_probe) from [<c031937c>] (driver_probe_device+0x1c8/0x26c)
-[<c031937c>] (driver_probe_device) from [<c0317a8c>] (bus_for_each_drv+0x44/0x8c)
-[<c0317a8c>] (bus_for_each_drv) from [<c031913c>] (__device_attach+0x88/0xd8)
-[<c031913c>] (__device_attach) from [<c03188fc>] (bus_probe_device+0x84/0x8c)
-[<c03188fc>] (bus_probe_device) from [<c0318cd4>] (deferred_probe_work_func+0x5c/0x88)
-[<c0318cd4>] (deferred_probe_work_func) from [<c00391bc>] (process_one_work+0x120/0x328)
+[<c003f2d4>] (blocking_notifier_call_chain) from [<c0289398>] (register_framebuffer+0x184/0x264)
+[<c0289398>] (register_framebuffer) from [<c02e1a94>] (drm_fb_helper_initial_config+0x244/0x408)
+[<c02e1a94>] (drm_fb_helper_initial_config) from [<c0303dc8>] (tegra_drm_fb_init+0x48/0xb4)
+[<c0303dc8>] (tegra_drm_fb_init) from [<c0302020>] (tegra_drm_load+0x170/0x224)
+[<c0302020>] (tegra_drm_load) from [<c02e9e54>] (drm_dev_register+0xa0/0x100)
+[<c02e9e54>] (drm_dev_register) from [<c0301b7c>] (host1x_drm_probe+0x44/0xb0)
+[<c0301b7c>] (host1x_drm_probe) from [<c02cd0cc>] (host1x_device_probe+0x1c/0x28)
+[<c02cd0cc>] (host1x_device_probe) from [<c03192fc>] (driver_probe_device+0x1c8/0x26c)
+[<c03192fc>] (driver_probe_device) from [<c0317a0c>] (bus_for_each_drv+0x44/0x8c)
+[<c0317a0c>] (bus_for_each_drv) from [<c03190bc>] (__device_attach+0x88/0xd8)
+[<c03190bc>] (__device_attach) from [<c031887c>] (bus_probe_device+0x84/0x8c)
+[<c031887c>] (bus_probe_device) from [<c0316dd0>] (device_add+0x33c/0x524)
+[<c0316dd0>] (device_add) from [<c02cd2d8>] (host1x_subdev_register+0xb0/0xd4)
+[<c02cd2d8>] (host1x_subdev_register) from [<c02cdb98>] (host1x_client_register+0xf4/0x11c)
+[<c02cdb98>] (host1x_client_register) from [<c030ba5c>] (tegra_hdmi_probe+0x170/0x250)
+[<c030ba5c>] (tegra_hdmi_probe) from [<c031a7bc>] (platform_drv_probe+0x48/0xa4)
+[<c031a7bc>] (platform_drv_probe) from [<c03192fc>] (driver_probe_device+0x1c8/0x26c)
+[<c03192fc>] (driver_probe_device) from [<c0317a0c>] (bus_for_each_drv+0x44/0x8c)
+[<c0317a0c>] (bus_for_each_drv) from [<c03190bc>] (__device_attach+0x88/0xd8)
+[<c03190bc>] (__device_attach) from [<c031887c>] (bus_probe_device+0x84/0x8c)
+[<c031887c>] (bus_probe_device) from [<c0318c54>] (deferred_probe_work_func+0x5c/0x88)
+[<c0318c54>] (deferred_probe_work_func) from [<c00391bc>] (process_one_work+0x120/0x328)
 [<c00391bc>] (process_one_work) from [<c0039420>] (worker_thread+0x28/0x490)
 [<c0039420>] (worker_thread) from [<c003e510>] (kthread+0xd8/0xf4)
 [<c003e510>] (kthread) from [<c000f628>] (ret_from_fork+0x14/0x2c)
----[ end trace ac67ef9eb7eebac6 ]---
+---[ end trace ab9ae497181f5ff0 ]---
 Console: switching to colour frame buffer device 240x75
 drm drm: fb0:  frame buffer device
 drm drm: registered panic notifier
@@ -349,7 +349,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-10 13:24:39 UTC (947510679)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-10 13:39:34 UTC (947511574)
 vdd_cam_1v8_reg: disabling
 usb1_vbus: disabling
 dvdd_ts: disabling
@@ -364,5 +364,5 @@
 Freeing unused kernel memory: 308K (c0931000 - c097e000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.219945] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.199944] usb 1-1: new high-speed USB device number 2 using tegra-ehci
 cfg80211: Calling CRDA to update world regulatory domain
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-06-29 21:51:42.532757514 -0700
+++ /tmp/foob	2015-06-29 21:51:42.532757514 -0700
@@ -40,20 +40,20 @@
 	 #################################################################
 	 #################################################################
 	 ###############################
-	 3.3 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 6174880 (5e38a0 hex)
+Bytes transferred = 6174856 (5e3888 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x82000000
 Loading: *#####
-	 2 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 63791 (f92f hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5e38a0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5e3888 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 8201292e
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.1.0-next-20150626-26998-g043831b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jun 29 21:32:41 PDT 2015
+Linux version 4.1.0-next-20150629-27202-ge398431 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jun 29 21:47:34 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f07ac0, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0f07b00, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -92,7 +92,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d1ac08   (11340 kB)
       .init : 0xc0d1b000 - 0xc0e06000   ( 940 kB)
-      .data : 0xc0e06000 - 0xc0f09ac0   (1039 kB)
+      .data : 0xc0e06000 - 0xc0f09b00   (1039 kB)
        .bss : 0xc0f0c000 - 0xc0f5dc28   ( 328 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -292,9 +292,9 @@
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
 Primary GPT is invalid, using alternate GPT.
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9
-tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
 tegra-pcie 1003000.pcie-controller: link 0 down, retrying
 tegra-pcie 1003000.pcie-controller: link 0 down, retrying
 tegra-pcie 1003000.pcie-controller: link 0 down, retrying
@@ -318,59 +318,59 @@
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
 ------------[ cut here ]------------
-WARNING: CPU: 2 PID: 58 at drivers/gpu/drm/drm_irq.c:1323 drm_vblank_on+0xf0/0x118()
+WARNING: CPU: 1 PID: 58 at drivers/gpu/drm/drm_irq.c:1323 drm_vblank_on+0xf0/0x118()
 Modules linked in:
-CPU: 2 PID: 58 Comm: kworker/u8:1 Not tainted 4.1.0-next-20150626-26998-g043831b #1
+CPU: 1 PID: 58 Comm: kworker/u8:1 Not tainted 4.1.0-next-20150629-27202-ge398431 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0217d68>] (unwind_backtrace) from [<c02136d0>] (show_stack+0x10/0x14)
-[<c02136d0>] (show_stack) from [<c09605bc>] (dump_stack+0x78/0x94)
-[<c09605bc>] (dump_stack) from [<c024c374>] (warn_slowpath_common+0x74/0xb0)
+[<c02136d0>] (show_stack) from [<c0960648>] (dump_stack+0x78/0x94)
+[<c0960648>] (dump_stack) from [<c024c374>] (warn_slowpath_common+0x74/0xb0)
 [<c024c374>] (warn_slowpath_common) from [<c024c3cc>] (warn_slowpath_null+0x1c/0x24)
-[<c024c3cc>] (warn_slowpath_null) from [<c05c312c>] (drm_vblank_on+0xf0/0x118)
-[<c05c312c>] (drm_vblank_on) from [<c05b8b8c>] (drm_atomic_helper_commit_modeset_enables+0x8c/0x180)
-[<c05b8b8c>] (drm_atomic_helper_commit_modeset_enables) from [<c05fff24>] (tegra_atomic_complete+0x2c/0x50)
-[<c05fff24>] (tegra_atomic_complete) from [<c05fffa8>] (tegra_atomic_commit+0x60/0x90)
-[<c05fffa8>] (tegra_atomic_commit) from [<c05da588>] (drm_atomic_commit+0x40/0x64)
-[<c05da588>] (drm_atomic_commit) from [<c05b88ec>] (drm_atomic_helper_set_config+0x394/0x3ec)
-[<c05b88ec>] (drm_atomic_helper_set_config) from [<c05ca724>] (drm_mode_set_config_internal+0x58/0xd4)
-[<c05ca724>] (drm_mode_set_config_internal) from [<c05bad78>] (restore_fbdev_mode+0xd4/0xf4)
-[<c05bad78>] (restore_fbdev_mode) from [<c05bc4b4>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
-[<c05bc4b4>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c05bc50c>] (drm_fb_helper_set_par+0x18/0x34)
-[<c05bc50c>] (drm_fb_helper_set_par) from [<c050a608>] (fbcon_init+0x474/0x4dc)
-[<c050a608>] (fbcon_init) from [<c05735f8>] (visual_init+0x9c/0xe8)
-[<c05735f8>] (visual_init) from [<c05755b0>] (do_bind_con_driver+0x134/0x354)
-[<c05755b0>] (do_bind_con_driver) from [<c0575940>] (do_take_over_console+0x170/0x1dc)
-[<c0575940>] (do_take_over_console) from [<c050a6d4>] (do_fbcon_takeover+0x64/0xcc)
-[<c050a6d4>] (do_fbcon_takeover) from [<c0265254>] (notifier_call_chain+0x44/0x84)
+[<c024c3cc>] (warn_slowpath_null) from [<c05c30b4>] (drm_vblank_on+0xf0/0x118)
+[<c05c30b4>] (drm_vblank_on) from [<c05b8b14>] (drm_atomic_helper_commit_modeset_enables+0x8c/0x180)
+[<c05b8b14>] (drm_atomic_helper_commit_modeset_enables) from [<c05ffeac>] (tegra_atomic_complete+0x2c/0x50)
+[<c05ffeac>] (tegra_atomic_complete) from [<c05fff30>] (tegra_atomic_commit+0x60/0x90)
+[<c05fff30>] (tegra_atomic_commit) from [<c05da510>] (drm_atomic_commit+0x40/0x64)
+[<c05da510>] (drm_atomic_commit) from [<c05b8874>] (drm_atomic_helper_set_config+0x394/0x3ec)
+[<c05b8874>] (drm_atomic_helper_set_config) from [<c05ca6ac>] (drm_mode_set_config_internal+0x58/0xd4)
+[<c05ca6ac>] (drm_mode_set_config_internal) from [<c05bad00>] (restore_fbdev_mode+0xd4/0xf4)
+[<c05bad00>] (restore_fbdev_mode) from [<c05bc43c>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
+[<c05bc43c>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c05bc494>] (drm_fb_helper_set_par+0x18/0x34)
+[<c05bc494>] (drm_fb_helper_set_par) from [<c050a5e8>] (fbcon_init+0x474/0x4dc)
+[<c050a5e8>] (fbcon_init) from [<c0573580>] (visual_init+0x9c/0xe8)
+[<c0573580>] (visual_init) from [<c0575538>] (do_bind_con_driver+0x134/0x354)
+[<c0575538>] (do_bind_con_driver) from [<c05758c8>] (do_take_over_console+0x170/0x1dc)
+[<c05758c8>] (do_take_over_console) from [<c050a6b4>] (do_fbcon_takeover+0x64/0xcc)
+[<c050a6b4>] (do_fbcon_takeover) from [<c0265254>] (notifier_call_chain+0x44/0x84)
 [<c0265254>] (notifier_call_chain) from [<c0265468>] (__blocking_notifier_call_chain+0x48/0x60)
 [<c0265468>] (__blocking_notifier_call_chain) from [<c0265498>] (blocking_notifier_call_chain+0x18/0x20)
-[<c0265498>] (blocking_notifier_call_chain) from [<c0514e6c>] (register_framebuffer+0x184/0x264)
-[<c0514e6c>] (register_framebuffer) from [<c05bc76c>] (drm_fb_helper_initial_config+0x244/0x408)
-[<c05bc76c>] (drm_fb_helper_initial_config) from [<c0601b38>] (tegra_drm_fb_init+0x48/0xb4)
-[<c0601b38>] (tegra_drm_fb_init) from [<c05ffe44>] (tegra_drm_load+0x170/0x224)
-[<c05ffe44>] (tegra_drm_load) from [<c05c5428>] (drm_dev_register+0xa0/0x100)
-[<c05c5428>] (drm_dev_register) from [<c05ff9a0>] (host1x_drm_probe+0x44/0xb0)
-[<c05ff9a0>] (host1x_drm_probe) from [<c05a7d80>] (host1x_device_probe+0x1c/0x28)
-[<c05a7d80>] (host1x_device_probe) from [<c06170c8>] (driver_probe_device+0x1c8/0x26c)
-[<c06170c8>] (driver_probe_device) from [<c06157d0>] (bus_for_each_drv+0x44/0x8c)
-[<c06157d0>] (bus_for_each_drv) from [<c0616e88>] (__device_attach+0x88/0xd8)
-[<c0616e88>] (__device_attach) from [<c0616648>] (bus_probe_device+0x84/0x8c)
-[<c0616648>] (bus_probe_device) from [<c0614b74>] (device_add+0x33c/0x524)
-[<c0614b74>] (device_add) from [<c05a7f8c>] (host1x_subdev_register+0xb0/0xd4)
-[<c05a7f8c>] (host1x_subdev_register) from [<c05a884c>] (host1x_client_register+0xf4/0x11c)
-[<c05a884c>] (host1x_client_register) from [<c06097d8>] (tegra_hdmi_probe+0x170/0x250)
-[<c06097d8>] (tegra_hdmi_probe) from [<c0618688>] (platform_drv_probe+0x48/0xa4)
-[<c0618688>] (platform_drv_probe) from [<c06170c8>] (driver_probe_device+0x1c8/0x26c)
-[<c06170c8>] (driver_probe_device) from [<c06157d0>] (bus_for_each_drv+0x44/0x8c)
-[<c06157d0>] (bus_for_each_drv) from [<c0616e88>] (__device_attach+0x88/0xd8)
-[<c0616e88>] (__device_attach) from [<c0616648>] (bus_probe_device+0x84/0x8c)
-[<c0616648>] (bus_probe_device) from [<c0616a20>] (deferred_probe_work_func+0x5c/0x88)
-[<c0616a20>] (deferred_probe_work_func) from [<c025f280>] (process_one_work+0x134/0x338)
+[<c0265498>] (blocking_notifier_call_chain) from [<c0514e4c>] (register_framebuffer+0x184/0x264)
+[<c0514e4c>] (register_framebuffer) from [<c05bc6f4>] (drm_fb_helper_initial_config+0x244/0x408)
+[<c05bc6f4>] (drm_fb_helper_initial_config) from [<c0601ac0>] (tegra_drm_fb_init+0x48/0xb4)
+[<c0601ac0>] (tegra_drm_fb_init) from [<c05ffdcc>] (tegra_drm_load+0x170/0x224)
+[<c05ffdcc>] (tegra_drm_load) from [<c05c53b0>] (drm_dev_register+0xa0/0x100)
+[<c05c53b0>] (drm_dev_register) from [<c05ff928>] (host1x_drm_probe+0x44/0xb0)
+[<c05ff928>] (host1x_drm_probe) from [<c05a7d08>] (host1x_device_probe+0x1c/0x28)
+[<c05a7d08>] (host1x_device_probe) from [<c0617050>] (driver_probe_device+0x1c8/0x26c)
+[<c0617050>] (driver_probe_device) from [<c0615758>] (bus_for_each_drv+0x44/0x8c)
+[<c0615758>] (bus_for_each_drv) from [<c0616e10>] (__device_attach+0x88/0xd8)
+[<c0616e10>] (__device_attach) from [<c06165d0>] (bus_probe_device+0x84/0x8c)
+[<c06165d0>] (bus_probe_device) from [<c0614afc>] (device_add+0x33c/0x524)
+[<c0614afc>] (device_add) from [<c05a7f14>] (host1x_subdev_register+0xb0/0xd4)
+[<c05a7f14>] (host1x_subdev_register) from [<c05a87d4>] (host1x_client_register+0xf4/0x11c)
+[<c05a87d4>] (host1x_client_register) from [<c0609760>] (tegra_hdmi_probe+0x170/0x250)
+[<c0609760>] (tegra_hdmi_probe) from [<c0618610>] (platform_drv_probe+0x48/0xa4)
+[<c0618610>] (platform_drv_probe) from [<c0617050>] (driver_probe_device+0x1c8/0x26c)
+[<c0617050>] (driver_probe_device) from [<c0615758>] (bus_for_each_drv+0x44/0x8c)
+[<c0615758>] (bus_for_each_drv) from [<c0616e10>] (__device_attach+0x88/0xd8)
+[<c0616e10>] (__device_attach) from [<c06165d0>] (bus_probe_device+0x84/0x8c)
+[<c06165d0>] (bus_probe_device) from [<c06169a8>] (deferred_probe_work_func+0x5c/0x88)
+[<c06169a8>] (deferred_probe_work_func) from [<c025f280>] (process_one_work+0x134/0x338)
 [<c025f280>] (process_one_work) from [<c025f4e0>] (worker_thread+0x28/0x4cc)
 [<c025f4e0>] (worker_thread) from [<c0264630>] (kthread+0xd8/0xf4)
 [<c0264630>] (kthread) from [<c0210a68>] (ret_from_fork+0x14/0x2c)
----[ end trace 96846f66e56aa44c ]---
+---[ end trace 480fa9e134bc60c7 ]---
 Console: switching to colour frame buffer device 512x135
 drm drm: fb0:  frame buffer device
 drm drm: registered panic notifier
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2015-06-29 21:51:42.540757511 -0700
+++ /tmp/foob	2015-06-29 21:51:42.540757511 -0700
@@ -39,10 +39,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #######################
+	 ######################
 	 3.2 MiB/s
 done
-Bytes transferred = 5094160 (4dbb10 hex)
+Bytes transferred = 5093936 (4dba30 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x82000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -53,7 +53,7 @@
 done
 Bytes transferred = 63791 (f92f hex)
 Tegra124 (Jetson TK1) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4dbb10 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4dba30 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Using Device Tree in place at 82000000, end 8201292e
@@ -64,7 +64,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.1.0-next-20150626-26998-g043831b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jun 29 21:33:09 PDT 2015
+Linux version 4.1.0-next-20150629-27202-ge398431 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jun 29 21:47:59 PDT 2015
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
 Machine model: NVIDIA Tegra124 Jetson TK1
@@ -72,7 +72,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c09d2dc0, node_mem_map ee7f7000
+free_area_init_node: node 0, pgdat c09d2e00, node_mem_map ee7f7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -93,9 +93,9 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0930b60   (9379 kB)
+      .text : 0xc0008000 - 0xc0930b68   (9379 kB)
       .init : 0xc0931000 - 0xc097e000   ( 308 kB)
-      .data : 0xc097e000 - 0xc09d4980   ( 347 kB)
+      .data : 0xc097e000 - 0xc09d49c0   ( 347 kB)
        .bss : 0xc09d7000 - 0xc0a31c94   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Additional per-CPU info printed with stalls.
@@ -322,57 +322,57 @@
 ------------[ cut here ]------------
 WARNING: CPU: 1 PID: 73 at drivers/gpu/drm/drm_irq.c:1323 drm_vblank_on+0xf0/0x118()
 Modules linked in:
-CPU: 1 PID: 73 Comm: kworker/u8:1 Not tainted 4.1.0-next-20150626-26998-g043831b #1
+CPU: 1 PID: 73 Comm: kworker/u8:1 Not tainted 4.1.0-next-20150629-27202-ge398431 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c00167f8>] (unwind_backtrace) from [<c0012124>] (show_stack+0x10/0x14)
-[<c0012124>] (show_stack) from [<c0698ef0>] (dump_stack+0x84/0xd0)
-[<c0698ef0>] (dump_stack) from [<c002683c>] (warn_slowpath_common+0x74/0xb0)
+[<c0012124>] (show_stack) from [<c0698f98>] (dump_stack+0x84/0xd0)
+[<c0698f98>] (dump_stack) from [<c002683c>] (warn_slowpath_common+0x74/0xb0)
 [<c002683c>] (warn_slowpath_common) from [<c0026894>] (warn_slowpath_null+0x1c/0x24)
-[<c0026894>] (warn_slowpath_null) from [<c02e7bc0>] (drm_vblank_on+0xf0/0x118)
-[<c02e7bc0>] (drm_vblank_on) from [<c02ddf34>] (drm_atomic_helper_commit_modeset_enables+0x8c/0x180)
-[<c02ddf34>] (drm_atomic_helper_commit_modeset_enables) from [<c0302180>] (tegra_atomic_complete+0x2c/0x50)
-[<c0302180>] (tegra_atomic_complete) from [<c0302204>] (tegra_atomic_commit+0x60/0x90)
-[<c0302204>] (tegra_atomic_commit) from [<c02ff290>] (drm_atomic_commit+0x40/0x64)
-[<c02ff290>] (drm_atomic_commit) from [<c02ddc94>] (drm_atomic_helper_set_config+0x394/0x3ec)
-[<c02ddc94>] (drm_atomic_helper_set_config) from [<c02ef2a4>] (drm_mode_set_config_internal+0x58/0xd4)
-[<c02ef2a4>] (drm_mode_set_config_internal) from [<c02e0120>] (restore_fbdev_mode+0xd4/0xf4)
-[<c02e0120>] (restore_fbdev_mode) from [<c02e185c>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
-[<c02e185c>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c02e18b4>] (drm_fb_helper_set_par+0x18/0x34)
-[<c02e18b4>] (drm_fb_helper_set_par) from [<c027f458>] (fbcon_init+0x474/0x4dc)
-[<c027f458>] (fbcon_init) from [<c02b4a04>] (visual_init+0x9c/0xe8)
-[<c02b4a04>] (visual_init) from [<c02b69a8>] (do_bind_con_driver+0x134/0x354)
-[<c02b69a8>] (do_bind_con_driver) from [<c02b6d38>] (do_take_over_console+0x170/0x1dc)
-[<c02b6d38>] (do_take_over_console) from [<c027f524>] (do_fbcon_takeover+0x64/0xcc)
-[<c027f524>] (do_fbcon_takeover) from [<c003f0d0>] (notifier_call_chain+0x44/0x84)
+[<c0026894>] (warn_slowpath_null) from [<c02e7b40>] (drm_vblank_on+0xf0/0x118)
+[<c02e7b40>] (drm_vblank_on) from [<c02ddeb4>] (drm_atomic_helper_commit_modeset_enables+0x8c/0x180)
+[<c02ddeb4>] (drm_atomic_helper_commit_modeset_enables) from [<c0302100>] (tegra_atomic_complete+0x2c/0x50)
+[<c0302100>] (tegra_atomic_complete) from [<c0302184>] (tegra_atomic_commit+0x60/0x90)
+[<c0302184>] (tegra_atomic_commit) from [<c02ff210>] (drm_atomic_commit+0x40/0x64)
+[<c02ff210>] (drm_atomic_commit) from [<c02ddc14>] (drm_atomic_helper_set_config+0x394/0x3ec)
+[<c02ddc14>] (drm_atomic_helper_set_config) from [<c02ef224>] (drm_mode_set_config_internal+0x58/0xd4)
+[<c02ef224>] (drm_mode_set_config_internal) from [<c02e00a0>] (restore_fbdev_mode+0xd4/0xf4)
+[<c02e00a0>] (restore_fbdev_mode) from [<c02e17dc>] (drm_fb_helper_restore_fbdev_mode_unlocked+0x1c/0x5c)
+[<c02e17dc>] (drm_fb_helper_restore_fbdev_mode_unlocked) from [<c02e1834>] (drm_fb_helper_set_par+0x18/0x34)
+[<c02e1834>] (drm_fb_helper_set_par) from [<c027f3d8>] (fbcon_init+0x474/0x4dc)
+[<c027f3d8>] (fbcon_init) from [<c02b4984>] (visual_init+0x9c/0xe8)
+[<c02b4984>] (visual_init) from [<c02b6928>] (do_bind_con_driver+0x134/0x354)
+[<c02b6928>] (do_bind_con_driver) from [<c02b6cb8>] (do_take_over_console+0x170/0x1dc)
+[<c02b6cb8>] (do_take_over_console) from [<c027f4a4>] (do_fbcon_takeover+0x64/0xcc)
+[<c027f4a4>] (do_fbcon_takeover) from [<c003f0d0>] (notifier_call_chain+0x44/0x84)
 [<c003f0d0>] (notifier_call_chain) from [<c003f2a4>] (__blocking_notifier_call_chain+0x48/0x60)
 [<c003f2a4>] (__blocking_notifier_call_chain) from [<c003f2d4>] (blocking_notifier_call_chain+0x18/0x20)
-[<c003f2d4>] (blocking_notifier_call_chain) from [<c0289418>] (register_framebuffer+0x184/0x264)
-[<c0289418>] (register_framebuffer) from [<c02e1b14>] (drm_fb_helper_initial_config+0x244/0x408)
-[<c02e1b14>] (drm_fb_helper_initial_config) from [<c0303e48>] (tegra_drm_fb_init+0x48/0xb4)
-[<c0303e48>] (tegra_drm_fb_init) from [<c03020a0>] (tegra_drm_load+0x170/0x224)
-[<c03020a0>] (tegra_drm_load) from [<c02e9ed4>] (drm_dev_register+0xa0/0x100)
-[<c02e9ed4>] (drm_dev_register) from [<c0301bfc>] (host1x_drm_probe+0x44/0xb0)
-[<c0301bfc>] (host1x_drm_probe) from [<c02cd14c>] (host1x_device_probe+0x1c/0x28)
-[<c02cd14c>] (host1x_device_probe) from [<c031937c>] (driver_probe_device+0x1c8/0x26c)
-[<c031937c>] (driver_probe_device) from [<c0317a8c>] (bus_for_each_drv+0x44/0x8c)
-[<c0317a8c>] (bus_for_each_drv) from [<c031913c>] (__device_attach+0x88/0xd8)
-[<c031913c>] (__device_attach) from [<c03188fc>] (bus_probe_device+0x84/0x8c)
-[<c03188fc>] (bus_probe_device) from [<c0316e50>] (device_add+0x33c/0x524)
-[<c0316e50>] (device_add) from [<c02cd358>] (host1x_subdev_register+0xb0/0xd4)
-[<c02cd358>] (host1x_subdev_register) from [<c02cdc18>] (host1x_client_register+0xf4/0x11c)
-[<c02cdc18>] (host1x_client_register) from [<c030badc>] (tegra_hdmi_probe+0x170/0x250)
-[<c030badc>] (tegra_hdmi_probe) from [<c031a83c>] (platform_drv_probe+0x48/0xa4)
-[<c031a83c>] (platform_drv_probe) from [<c031937c>] (driver_probe_device+0x1c8/0x26c)
-[<c031937c>] (driver_probe_device) from [<c0317a8c>] (bus_for_each_drv+0x44/0x8c)
-[<c0317a8c>] (bus_for_each_drv) from [<c031913c>] (__device_attach+0x88/0xd8)
-[<c031913c>] (__device_attach) from [<c03188fc>] (bus_probe_device+0x84/0x8c)
-[<c03188fc>] (bus_probe_device) from [<c0318cd4>] (deferred_probe_work_func+0x5c/0x88)
-[<c0318cd4>] (deferred_probe_work_func) from [<c00391bc>] (process_one_work+0x120/0x328)
+[<c003f2d4>] (blocking_notifier_call_chain) from [<c0289398>] (register_framebuffer+0x184/0x264)
+[<c0289398>] (register_framebuffer) from [<c02e1a94>] (drm_fb_helper_initial_config+0x244/0x408)
+[<c02e1a94>] (drm_fb_helper_initial_config) from [<c0303dc8>] (tegra_drm_fb_init+0x48/0xb4)
+[<c0303dc8>] (tegra_drm_fb_init) from [<c0302020>] (tegra_drm_load+0x170/0x224)
+[<c0302020>] (tegra_drm_load) from [<c02e9e54>] (drm_dev_register+0xa0/0x100)
+[<c02e9e54>] (drm_dev_register) from [<c0301b7c>] (host1x_drm_probe+0x44/0xb0)
+[<c0301b7c>] (host1x_drm_probe) from [<c02cd0cc>] (host1x_device_probe+0x1c/0x28)
+[<c02cd0cc>] (host1x_device_probe) from [<c03192fc>] (driver_probe_device+0x1c8/0x26c)
+[<c03192fc>] (driver_probe_device) from [<c0317a0c>] (bus_for_each_drv+0x44/0x8c)
+[<c0317a0c>] (bus_for_each_drv) from [<c03190bc>] (__device_attach+0x88/0xd8)
+[<c03190bc>] (__device_attach) from [<c031887c>] (bus_probe_device+0x84/0x8c)
+[<c031887c>] (bus_probe_device) from [<c0316dd0>] (device_add+0x33c/0x524)
+[<c0316dd0>] (device_add) from [<c02cd2d8>] (host1x_subdev_register+0xb0/0xd4)
+[<c02cd2d8>] (host1x_subdev_register) from [<c02cdb98>] (host1x_client_register+0xf4/0x11c)
+[<c02cdb98>] (host1x_client_register) from [<c030ba5c>] (tegra_hdmi_probe+0x170/0x250)
+[<c030ba5c>] (tegra_hdmi_probe) from [<c031a7bc>] (platform_drv_probe+0x48/0xa4)
+[<c031a7bc>] (platform_drv_probe) from [<c03192fc>] (driver_probe_device+0x1c8/0x26c)
+[<c03192fc>] (driver_probe_device) from [<c0317a0c>] (bus_for_each_drv+0x44/0x8c)
+[<c0317a0c>] (bus_for_each_drv) from [<c03190bc>] (__device_attach+0x88/0xd8)
+[<c03190bc>] (__device_attach) from [<c031887c>] (bus_probe_device+0x84/0x8c)
+[<c031887c>] (bus_probe_device) from [<c0318c54>] (deferred_probe_work_func+0x5c/0x88)
+[<c0318c54>] (deferred_probe_work_func) from [<c00391bc>] (process_one_work+0x120/0x328)
 [<c00391bc>] (process_one_work) from [<c0039420>] (worker_thread+0x28/0x490)
 [<c0039420>] (worker_thread) from [<c003e510>] (kthread+0xd8/0xf4)
 [<c003e510>] (kthread) from [<c000f628>] (ret_from_fork+0x14/0x2c)
----[ end trace 31b295eebb105597 ]---
+---[ end trace 527a940aef61216a ]---
 Console: switching to colour frame buffer device 512x135
 drm drm: fb0:  frame buffer device
 drm drm: registered panic notifier
@@ -396,14 +396,14 @@
 cpufreq: cpufreq_add_dev: CPU0: Unlisted initial frequency changed to: 714000 KHz
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
-cfg80211: Calling CRDA to update world regulatory domain
 mmc1: Unknown controller version (3). You may experience problems.
 +VDDIO_SDMMC3: supplied by +3.3V_RUN
 sdhci-tegra 700b0400.sdhci: No vmmc regulator found
+cfg80211: Calling CRDA to update world regulatory domain
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-02-01 00:00:21 UTC (949363221)
+as3722-rtc as3722-rtc: setting system clock to 2000-02-01 00:00:55 UTC (949363255)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_VPP_FUSE: disabling
 +2.8V_RUN_CAM_AF: disabling
@@ -426,4 +426,4 @@
 Freeing unused kernel memory: 308K (c0931000 - c097e000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    7.878146] asix 2-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:11:31:3b
+root@(none):/# [    7.863544] asix 2-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:11:31:3b
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-06-29 21:51:42.548757508 -0700
+++ /tmp/foob	2015-06-29 21:51:42.548757508 -0700
@@ -48,18 +48,18 @@
 	 ###############################
 	 4.1 MiB/s
 done
-Bytes transferred = 6174880 (5e38a0 hex)
+Bytes transferred = 6174856 (5e3888 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 4 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x5e38a0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x5e3888 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -67,7 +67,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.1.0-next-20150626-26998-g043831b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jun 29 21:32:41 PDT 2015
+Linux version 4.1.0-next-20150629-27202-ge398431 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jun 29 21:47:34 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -75,7 +75,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0f07ac0, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c0f07b00, node_mem_map eeff8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -98,7 +98,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d1ac08   (11340 kB)
       .init : 0xc0d1b000 - 0xc0e06000   ( 940 kB)
-      .data : 0xc0e06000 - 0xc0f09ac0   (1039 kB)
+      .data : 0xc0e06000 - 0xc0f09b00   (1039 kB)
        .bss : 0xc0f0c000 - 0xc0f5dc28   ( 328 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
@@ -336,7 +336,7 @@
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
-rtc-em3027 2-0056: setting system clock to 2014-12-26 16:47:16 UTC (1419612436)
+rtc-em3027 2-0056: setting system clock to 2014-12-26 16:47:54 UTC (1419612474)
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2015-06-29 21:51:42.556757506 -0700
+++ /tmp/foob	2015-06-29 21:51:42.556757506 -0700
@@ -44,10 +44,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #######################
+	 ######################
 	 4.1 MiB/s
 done
-Bytes transferred = 5094160 (4dbb10 hex)
+Bytes transferred = 5093936 (4dba30 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x4dbb10 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x4dba30 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -69,7 +69,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.1.0-next-20150626-26998-g043831b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jun 29 21:33:09 PDT 2015
+Linux version 4.1.0-next-20150629-27202-ge398431 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jun 29 21:47:59 PDT 2015
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -78,7 +78,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c09d2dc0, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c09d2e00, node_mem_map eeff7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -99,9 +99,9 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0930b60   (9379 kB)
+      .text : 0xc0008000 - 0xc0930b68   (9379 kB)
       .init : 0xc0931000 - 0xc097e000   ( 308 kB)
-      .data : 0xc097e000 - 0xc09d4980   ( 347 kB)
+      .data : 0xc097e000 - 0xc09d49c0   ( 347 kB)
        .bss : 0xc09d7000 - 0xc0a31c94   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Additional per-CPU info printed with stalls.
@@ -321,8 +321,8 @@
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
-mip6: Mobile IPv6
  mmcblk0: p1
+mip6: Mobile IPv6
 sit: IPv6 over IPv4 tunneling driver
 NET: Registered protocol family 17
 NET: Registered protocol family 15
@@ -346,12 +346,12 @@
 hub 1-1:1.0: 4 ports detected
 mmc1: new high speed SD card at address b368
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
-hub 2-0:1.0: USB hub found
-hub 2-0:1.0: 1 port detected
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
+hub 2-0:1.0: USB hub found
+hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-26 16:47:37 UTC (1419612457)
+rtc-em3027 2-0056: setting system clock to 2014-12-26 16:48:13 UTC (1419612493)
 ALSA device list:
   #0: tegra-trimslice
 kjournald starting.  Commit interval 5 seconds
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2015-06-29 21:51:42.560757504 -0700
+++ /tmp/foob	2015-06-29 21:51:42.564757501 -0700
@@ -38,20 +38,20 @@
 	 #################################################################
 	 #################################################################
 	 ###############################
-	 3.4 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 6174880 (5e38a0 hex)
+Bytes transferred = 6174856 (5e3888 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5e38a0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5e3888 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.1.0-next-20150626-26998-g043831b (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jun 29 21:32:41 PDT 2015
+Linux version 4.1.0-next-20150629-27202-ge398431 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jun 29 21:47:34 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -67,7 +67,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0f07ac0, node_mem_map ee7f8000
+free_area_init_node: node 0, pgdat c0f07b00, node_mem_map ee7f8000
   DMA zone: 1520 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 194560 pages, LIFO batch:31
@@ -90,7 +90,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d1ac08   (11340 kB)
       .init : 0xc0d1b000 - 0xc0e06000   ( 940 kB)
-      .data : 0xc0e06000 - 0xc0f09ac0   (1039 kB)
+      .data : 0xc0e06000 - 0xc0f09b00   (1039 kB)
        .bss : 0xc0f0c000 - 0xc0f5dc28   ( 328 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -288,8 +288,8 @@
 mmcblk0: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk0boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk0boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
-tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 mmcblk0rpmb: mmc1:0001 SEM16G partition 3 128 KiB
+tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 Primary GPT is invalid, using alternate GPT.
  mmcblk0: p1 p2 p3 p4 p5
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -368,4 +368,4 @@
 Freeing unused kernel memory: 940K (c0d1b000 - c0e06000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.763798] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.773783] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2015-06-29 21:51:42.568757499 -0700
+++ /tmp/foob	2015-06-29 21:51:42.572757498 -0700
@@ -36,21 +36,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #######################
-	 3.2 MiB/s
+	 ######################
+	 3.3 MiB/s
 done
-Bytes transferred = 5094160 (4dbb10 hex)
+Bytes transferred = 5093936 (4dba30 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.3 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x4dbb10 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x4dba30 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -61,7 +61,7 @@
 Booting Linux on physical CPU 0x0
 Initializing cgroup subsys cpu
 Initializing cgroup subsys cpuacct
-Linux version 4.1.0-next-20150626-26998-g043831b (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jun 29 21:33:09 PDT 2015
+Linux version 4.1.0-next-20150629-27202-ge398431 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jun 29 21:47:59 PDT 2015
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c09d2dc0, node_mem_map ee7f7000
+free_area_init_node: node 0, pgdat c09d2e00, node_mem_map ee7f7000
   Normal zone: 1520 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 194560 pages, LIFO batch:31
@@ -90,9 +90,9 @@
     lowmem  : 0xc0000000 - 0xef800000   ( 760 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0930b60   (9379 kB)
+      .text : 0xc0008000 - 0xc0930b68   (9379 kB)
       .init : 0xc0931000 - 0xc097e000   ( 308 kB)
-      .data : 0xc097e000 - 0xc09d4980   ( 347 kB)
+      .data : 0xc097e000 - 0xc09d49c0   ( 347 kB)
        .bss : 0xc09d7000 - 0xc0a31c94   ( 364 kB)
 Preemptible hierarchical RCU implementation.
 	Additional per-CPU info printed with stalls.
@@ -362,8 +362,8 @@
 vdd_sata,avdd_plle: disabling
 ALSA device list:
   #0: NVIDIA Tegra Beaver
-EXT3-fs (mmcblk0p1): warning: mounting fs with errors, running e2fsck is recommended
 kjournald starting.  Commit interval 5 seconds
+EXT3-fs (mmcblk0p1): warning: mounting fs with errors, running e2fsck is recommended
 EXT3-fs (mmcblk0p1): using internal journal
 EXT3-fs (mmcblk0p1): recovery complete
 EXT3-fs (mmcblk0p1): mounted filesystem with writeback data mode
@@ -372,4 +372,4 @@
 Freeing unused kernel memory: 308K (c0931000 - c097e000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.436384] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.456698] usb 2-1: new high-speed USB device number 2 using tegra-ehci
