Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-01-31 22:54:26.671784920 -0800
+++ /tmp/foob	2016-01-31 22:54:26.675784920 -0800
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.5.0-rc1-next-20160129+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Fri Jan 29 01:19:25 PST 2016
+Linux version 4.5.0-rc1-next-20160201+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Jan 31 22:49:42 PST 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -92,7 +92,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.676:1): initialized
+audit: type=2000 audit(0.464:1): initialized
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
 Key type id_resolver registered
@@ -138,7 +138,7 @@
 9pnet: Installing 9P2000 support
 Key type dns_resolver registered
 registered taskstats version 1
-rtc-pl031 9010000.pl031: setting system clock to 2016-01-29 09:19:54 UTC (1454059194)
+rtc-pl031 9010000.pl031: setting system clock to 2016-02-01 06:50:11 UTC (1454309411)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-01-31 22:54:26.679784920 -0800
+++ /tmp/foob	2016-01-31 22:54:26.683784920 -0800
@@ -41,7 +41,7 @@
 	 ########
 	 3.2 MiB/s
 done
-Bytes transferred = 6784824 (678738 hex)
+Bytes transferred = 6788728 (679678 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -52,7 +52,7 @@
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x678738 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x679678 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.5.0-rc1-next-20160129-68987-g735cfa5 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Fri Jan 29 01:17:35 PST 2016
+Linux version 4.5.0-rc1-next-20160201-69474-g99e8324 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Jan 31 22:47:50 PST 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,20 +69,20 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c1072500, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1074580, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-PERCPU: Embedded 13 pages/cpu @eef9f000 s22912 r8192 d22144 u53248
-pcpu-alloc: s22912 r8192 d22144 u53248 alloc=13*4096
+PERCPU: Embedded 13 pages/cpu @eef9f000 s22976 r8192 d22080 u53248
+pcpu-alloc: s22976 r8192 d22080 u53248 alloc=13*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1997868K/2096128K available (8503K kernel code, 1113K rwdata, 4048K rodata, 1100K init, 345K bss, 32724K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1997860K/2096128K available (8507K kernel code, 1113K rwdata, 4052K rodata, 1100K init, 345K bss, 32732K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0e4ae44   (12556 kB)
-      .init : 0xc0e4b000 - 0xc0f5e000   (1100 kB)
-      .data : 0xc0f5e000 - 0xc1074480   (1114 kB)
-       .bss : 0xc1077000 - 0xc10cd5e0   ( 346 kB)
+      .text : 0xc0208000 - 0xc0e4cf00   (12564 kB)
+      .init : 0xc0e4d000 - 0xc0f60000   (1100 kB)
+      .data : 0xc0f60000 - 0xc1076540   (1114 kB)
+       .bss : 0xc1079000 - 0xc10cf5e0   ( 346 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -277,7 +277,6 @@
 vdd_hdmi_5v0: supplied by vdd-sys-5v0
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
-mmc1: MAN_BKOPS_EN bit is not set
 mmc1: new HS200 MMC card at address 0001
 mmcblk0: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk0boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
@@ -290,49 +289,12 @@
 usb2_vbus: supplied by vdd-sys-5v0
 tegra-ehci 7d008000.usb: EHCI Host Controller
 tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 1
-------------[ cut here ]------------
-WARNING: CPU: 3 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
-workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
-Modules linked in:
-CPU: 3 PID: 6 Comm: kworker/u8:0 Not tainted 4.5.0-rc1-next-20160129-68987-g735cfa5 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-Workqueue: deferwq deferred_probe_work_func
-[<c02188b8>] (unwind_backtrace) from [<c0213f94>] (show_stack+0x10/0x14)
-[<c0213f94>] (show_stack) from [<c0489bf4>] (dump_stack+0x78/0x94)
-[<c0489bf4>] (dump_stack) from [<c024ab38>] (warn_slowpath_common+0x74/0xb0)
-[<c024ab38>] (warn_slowpath_common) from [<c024ac08>] (warn_slowpath_fmt+0x30/0x40)
-[<c024ac08>] (warn_slowpath_fmt) from [<c025dc80>] (check_flush_dependency+0xc8/0x124)
-[<c025dc80>] (check_flush_dependency) from [<c025dd94>] (flush_work+0x54/0x144)
-[<c025dd94>] (flush_work) from [<c02db314>] (lru_add_drain_all+0x124/0x180)
-[<c02db314>] (lru_add_drain_all) from [<c030cda4>] (migrate_prep+0x8/0x10)
-[<c030cda4>] (migrate_prep) from [<c02d6310>] (alloc_contig_range+0xc4/0x2f8)
-[<c02d6310>] (alloc_contig_range) from [<c030e6d0>] (cma_alloc+0xd8/0x1a0)
-[<c030e6d0>] (cma_alloc) from [<c021e39c>] (__alloc_from_contiguous+0x38/0xdc)
-[<c021e39c>] (__alloc_from_contiguous) from [<c021ff8c>] (__dma_alloc+0x21c/0x250)
-[<c021ff8c>] (__dma_alloc) from [<c0220044>] (arm_dma_alloc+0x48/0x54)
-[<c0220044>] (arm_dma_alloc) from [<c03072b8>] (dma_pool_alloc+0x1dc/0x210)
-[<c03072b8>] (dma_pool_alloc) from [<c07c8590>] (ehci_qh_alloc+0x3c/0xb4)
-[<c07c8590>] (ehci_qh_alloc) from [<c07ccb88>] (ehci_setup+0x154/0x484)
-[<c07ccb88>] (ehci_setup) from [<c07d2c64>] (tegra_ehci_reset+0xc/0x50)
-[<c07d2c64>] (tegra_ehci_reset) from [<c07a3010>] (usb_add_hcd+0x300/0x840)
-[<c07a3010>] (usb_add_hcd) from [<c07d25dc>] (tegra_ehci_probe+0x3b8/0x504)
-[<c07d25dc>] (tegra_ehci_probe) from [<c065a41c>] (platform_drv_probe+0x50/0xb0)
-[<c065a41c>] (platform_drv_probe) from [<c0658d88>] (driver_probe_device+0x1f4/0x2b0)
-[<c0658d88>] (driver_probe_device) from [<c0657318>] (bus_for_each_drv+0x44/0x8c)
-[<c0657318>] (bus_for_each_drv) from [<c0658b0c>] (__device_attach+0x9c/0x100)
-[<c0658b0c>] (__device_attach) from [<c0658198>] (bus_probe_device+0x84/0x8c)
-[<c0658198>] (bus_probe_device) from [<c06585cc>] (deferred_probe_work_func+0x60/0x8c)
-[<c06585cc>] (deferred_probe_work_func) from [<c025ebdc>] (process_one_work+0x134/0x338)
-[<c025ebdc>] (process_one_work) from [<c025ee44>] (worker_thread+0x2c/0x4f0)
-[<c025ee44>] (worker_thread) from [<c0263fd0>] (kthread+0xd8/0xf4)
-[<c0263fd0>] (kthread) from [<c0210e78>] (ret_from_fork+0x14/0x3c)
----[ end trace aed83d94d5173667 ]---
 tegra-ehci 7d008000.usb: irq 339, io mem 0x7d008000
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-09 14:27:29 UTC (947428049)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-12 11:57:47 UTC (947678267)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -341,7 +303,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1100K (c0e4b000 - c0f5e000)
+Freeing unused kernel memory: 1100K (c0e4d000 - c0f60000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.499504] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.209159] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-01-31 22:54:26.699784921 -0800
+++ /tmp/foob	2016-01-31 22:54:26.703784921 -0800
@@ -37,9 +37,9 @@
 	 #################################################################
 	 #################################################################
 	 ##################################
-	 3 MiB/s
+	 2.9 MiB/s
 done
-Bytes transferred = 5261256 (5047c8 hex)
+Bytes transferred = 5262304 (504be0 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5047c8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x504be0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.5.0-rc1-next-20160129-68987-g735cfa5 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Jan 29 01:18:00 PST 2016
+Linux version 4.5.0-rc1-next-20160201-69474-g99e8324 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Jan 31 22:48:12 PST 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,20 +69,20 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0a35480, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0a37500, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-PERCPU: Embedded 13 pages/cpu @eef9e000 s23232 r8192 d21824 u53248
-pcpu-alloc: s23232 r8192 d21824 u53248 alloc=13*4096
+PERCPU: Embedded 13 pages/cpu @eef9e000 s23296 r8192 d21760 u53248
+pcpu-alloc: s23296 r8192 d21760 u53248 alloc=13*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002184K/2096128K available (7224K kernel code, 371K rwdata, 2488K rodata, 340K init, 367K bss, 28408K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2002172K/2096128K available (7224K kernel code, 372K rwdata, 2492K rodata, 344K init, 367K bss, 28420K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,10 +90,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09847c4   (9714 kB)
-      .init : 0xc0985000 - 0xc09da000   ( 340 kB)
-      .data : 0xc09da000 - 0xc0a36fa0   ( 372 kB)
-       .bss : 0xc0a39000 - 0xc0a94d88   ( 368 kB)
+      .text : 0xc0008000 - 0xc0985818   (9719 kB)
+      .init : 0xc0986000 - 0xc09dc000   ( 344 kB)
+      .data : 0xc09dc000 - 0xc0a39060   ( 373 kB)
+       .bss : 0xc0a3c000 - 0xc0a97d88   ( 368 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -248,7 +248,6 @@
 mmc1: SDHCI controller on 78000600.sdhci [78000600.sdhci] using ADMA
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
-mmc1: MAN_BKOPS_EN bit is not set
 mmc1: new HS200 MMC card at address 0001
 mmcblk0: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk0boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
@@ -287,49 +286,12 @@
 usb2_vbus: supplied by vdd-sys-5v0
 tegra-ehci 7d008000.usb: EHCI Host Controller
 tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 1
-------------[ cut here ]------------
-WARNING: CPU: 0 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
-workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
-Modules linked in:
-CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.5.0-rc1-next-20160129-68987-g735cfa5 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-Workqueue: deferwq deferred_probe_work_func
-[<c0017208>] (unwind_backtrace) from [<c0012874>] (show_stack+0x10/0x14)
-[<c0012874>] (show_stack) from [<c023d1ac>] (dump_stack+0x84/0xd0)
-[<c023d1ac>] (dump_stack) from [<c0026b70>] (warn_slowpath_common+0x74/0xb0)
-[<c0026b70>] (warn_slowpath_common) from [<c0026c40>] (warn_slowpath_fmt+0x30/0x40)
-[<c0026c40>] (warn_slowpath_fmt) from [<c00397f8>] (check_flush_dependency+0xc8/0x12c)
-[<c00397f8>] (check_flush_dependency) from [<c0039914>] (flush_work+0x54/0x11c)
-[<c0039914>] (flush_work) from [<c00c1084>] (lru_add_drain_all+0x124/0x180)
-[<c00c1084>] (lru_add_drain_all) from [<c00f2498>] (migrate_prep+0x8/0x10)
-[<c00f2498>] (migrate_prep) from [<c00bbbe0>] (alloc_contig_range+0xc4/0x2f8)
-[<c00bbbe0>] (alloc_contig_range) from [<c00f4008>] (cma_alloc+0xd8/0x1a0)
-[<c00f4008>] (cma_alloc) from [<c001c270>] (__alloc_from_contiguous+0x38/0xdc)
-[<c001c270>] (__alloc_from_contiguous) from [<c001cf38>] (__dma_alloc+0x228/0x2ac)
-[<c001cf38>] (__dma_alloc) from [<c001d040>] (arm_dma_alloc+0x48/0x54)
-[<c001d040>] (arm_dma_alloc) from [<c00ee1f8>] (dma_pool_alloc+0x28c/0x304)
-[<c00ee1f8>] (dma_pool_alloc) from [<c03fe770>] (ehci_qh_alloc+0x3c/0xb4)
-[<c03fe770>] (ehci_qh_alloc) from [<c040295c>] (ehci_setup+0x154/0x484)
-[<c040295c>] (ehci_setup) from [<c04066c8>] (tegra_ehci_reset+0xc/0x50)
-[<c04066c8>] (tegra_ehci_reset) from [<c03ea15c>] (usb_add_hcd+0x2fc/0x814)
-[<c03ea15c>] (usb_add_hcd) from [<c0406040>] (tegra_ehci_probe+0x3b8/0x504)
-[<c0406040>] (tegra_ehci_probe) from [<c0337efc>] (platform_drv_probe+0x50/0xb0)
-[<c0337efc>] (platform_drv_probe) from [<c0336964>] (driver_probe_device+0x1f4/0x2b0)
-[<c0336964>] (driver_probe_device) from [<c0334ef8>] (bus_for_each_drv+0x44/0x8c)
-[<c0334ef8>] (bus_for_each_drv) from [<c03366e8>] (__device_attach+0x9c/0x100)
-[<c03366e8>] (__device_attach) from [<c0335d74>] (bus_probe_device+0x84/0x8c)
-[<c0335d74>] (bus_probe_device) from [<c03361a8>] (deferred_probe_work_func+0x60/0x8c)
-[<c03361a8>] (deferred_probe_work_func) from [<c003a720>] (process_one_work+0x120/0x328)
-[<c003a720>] (process_one_work) from [<c003a98c>] (worker_thread+0x2c/0x4b4)
-[<c003a98c>] (worker_thread) from [<c003fac4>] (kthread+0xd8/0xf4)
-[<c003fac4>] (kthread) from [<c000f938>] (ret_from_fork+0x14/0x3c)
----[ end trace 6863fa8340092cb2 ]---
 tegra-ehci 7d008000.usb: irq 339, io mem 0x7d008000
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-09 14:29:18 UTC (947428158)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-12 11:59:37 UTC (947678377)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -340,7 +302,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 340K (c0985000 - c09da000)
+Freeing unused kernel memory: 344K (c0986000 - c09dc000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# 
\ No newline at end of file
+root@(none):/# [    2.516968] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-01-31 22:54:26.707784921 -0800
+++ /tmp/foob	2016-01-31 22:54:26.711784921 -0800
@@ -41,20 +41,20 @@
 	 #################################################################
 	 #################################################################
 	 ########
-	 3.2 MiB/s
+	 3.1 MiB/s
 done
-Bytes transferred = 6784824 (678738 hex)
+Bytes transferred = 6788728 (679678 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.2 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 64218 (fada hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x678738 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x679678 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012ad9
@@ -62,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.5.0-rc1-next-20160129-68987-g735cfa5 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Fri Jan 29 01:17:35 PST 2016
+Linux version 4.5.0-rc1-next-20160201-69474-g99e8324 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Jan 31 22:47:50 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -71,20 +71,20 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c1072500, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1074580, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-PERCPU: Embedded 13 pages/cpu @eef8b000 s22912 r8192 d22144 u53248
-pcpu-alloc: s22912 r8192 d22144 u53248 alloc=13*4096
+PERCPU: Embedded 13 pages/cpu @eef8b000 s22976 r8192 d22080 u53248
+pcpu-alloc: s22976 r8192 d22080 u53248 alloc=13*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1997760K/2096128K available (8503K kernel code, 1113K rwdata, 4048K rodata, 1100K init, 345K bss, 32832K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1997752K/2096128K available (8507K kernel code, 1113K rwdata, 4052K rodata, 1100K init, 345K bss, 32840K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,10 +92,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0e4ae44   (12556 kB)
-      .init : 0xc0e4b000 - 0xc0f5e000   (1100 kB)
-      .data : 0xc0f5e000 - 0xc1074480   (1114 kB)
-       .bss : 0xc1077000 - 0xc10cd5e0   ( 346 kB)
+      .text : 0xc0208000 - 0xc0e4cf00   (12564 kB)
+      .init : 0xc0e4d000 - 0xc0f60000   (1100 kB)
+      .data : 0xc0f60000 - 0xc1076540   (1114 kB)
+       .bss : 0xc1079000 - 0xc10cf5e0   ( 346 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -290,7 +290,6 @@
 +1.05V_RUN_AVDD: supplied by +1.35V_LP0(sd2)
 tegra-pcie 1003000.pcie-controller: probing port 0, using 2 lanes
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
-mmc0: MAN_BKOPS_EN bit is not set
 mmc0: new HS200 MMC card at address 0001
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
@@ -328,44 +327,6 @@
 +12V_SATA: supplied by +VDD_MUX
 tegra-ahci 70027000.sata: AHCI 0001.0301 32 slots 2 ports 3 Gbps 0x1 impl platform mode
 tegra-ahci 70027000.sata: flags: 64bit ncq sntf pm led pio slum part sadm sds apst 
-------------[ cut here ]------------
-WARNING: CPU: 2 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
-workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
-Modules linked in:
-CPU: 2 PID: 6 Comm: kworker/u8:0 Not tainted 4.5.0-rc1-next-20160129-68987-g735cfa5 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-Workqueue: deferwq deferred_probe_work_func
-[<c02188b8>] (unwind_backtrace) from [<c0213f94>] (show_stack+0x10/0x14)
-[<c0213f94>] (show_stack) from [<c0489bf4>] (dump_stack+0x78/0x94)
-[<c0489bf4>] (dump_stack) from [<c024ab38>] (warn_slowpath_common+0x74/0xb0)
-[<c024ab38>] (warn_slowpath_common) from [<c024ac08>] (warn_slowpath_fmt+0x30/0x40)
-[<c024ac08>] (warn_slowpath_fmt) from [<c025dc80>] (check_flush_dependency+0xc8/0x124)
-[<c025dc80>] (check_flush_dependency) from [<c025dd94>] (flush_work+0x54/0x144)
-[<c025dd94>] (flush_work) from [<c02db314>] (lru_add_drain_all+0x124/0x180)
-[<c02db314>] (lru_add_drain_all) from [<c030cda4>] (migrate_prep+0x8/0x10)
-[<c030cda4>] (migrate_prep) from [<c02d6310>] (alloc_contig_range+0xc4/0x2f8)
-[<c02d6310>] (alloc_contig_range) from [<c030e6d0>] (cma_alloc+0xd8/0x1a0)
-[<c030e6d0>] (cma_alloc) from [<c021e39c>] (__alloc_from_contiguous+0x38/0xdc)
-[<c021e39c>] (__alloc_from_contiguous) from [<c021ff8c>] (__dma_alloc+0x21c/0x250)
-[<c021ff8c>] (__dma_alloc) from [<c0220044>] (arm_dma_alloc+0x48/0x54)
-[<c0220044>] (arm_dma_alloc) from [<c066b53c>] (dmam_alloc_coherent+0x8c/0xf0)
-[<c066b53c>] (dmam_alloc_coherent) from [<c06c60cc>] (ahci_port_start+0xd0/0x1e0)
-[<c06c60cc>] (ahci_port_start) from [<c06adf20>] (ata_host_start+0xf8/0x1d4)
-[<c06adf20>] (ata_host_start) from [<c06b2e3c>] (ata_host_activate+0x20/0x134)
-[<c06b2e3c>] (ata_host_activate) from [<c06c5548>] (ahci_host_activate+0x3c/0x11c)
-[<c06c5548>] (ahci_host_activate) from [<c06c7498>] (ahci_platform_init_host+0x1cc/0x3c8)
-[<c06c7498>] (ahci_platform_init_host) from [<c06c96fc>] (tegra_ahci_probe+0x404/0x4c4)
-[<c06c96fc>] (tegra_ahci_probe) from [<c065a41c>] (platform_drv_probe+0x50/0xb0)
-[<c065a41c>] (platform_drv_probe) from [<c0658d88>] (driver_probe_device+0x1f4/0x2b0)
-[<c0658d88>] (driver_probe_device) from [<c0657318>] (bus_for_each_drv+0x44/0x8c)
-[<c0657318>] (bus_for_each_drv) from [<c0658b0c>] (__device_attach+0x9c/0x100)
-[<c0658b0c>] (__device_attach) from [<c0658198>] (bus_probe_device+0x84/0x8c)
-[<c0658198>] (bus_probe_device) from [<c06585cc>] (deferred_probe_work_func+0x60/0x8c)
-[<c06585cc>] (deferred_probe_work_func) from [<c025ebdc>] (process_one_work+0x134/0x338)
-[<c025ebdc>] (process_one_work) from [<c025ee44>] (worker_thread+0x2c/0x4f0)
-[<c025ee44>] (worker_thread) from [<c0263fd0>] (kthread+0xd8/0xf4)
-[<c0263fd0>] (kthread) from [<c0210e78>] (ret_from_fork+0x14/0x3c)
----[ end trace 46ec1d90eb975fea ]---
 scsi host0: tegra-ahci
 scsi host1: tegra-ahci
 ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 343
@@ -404,7 +365,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1100K (c0e4b000 - c0f5e000)
+Freeing unused kernel memory: 1100K (c0e4d000 - c0f60000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-01-31 22:54:26.715784920 -0800
+++ /tmp/foob	2016-01-31 22:54:26.719784921 -0800
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 ##################################
-	 3.1 MiB/s
+	 2.9 MiB/s
 done
-Bytes transferred = 5261256 (5047c8 hex)
+Bytes transferred = 5262304 (504be0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 2.8 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 64218 (fada hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5047c8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x504be0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012ad9
@@ -61,7 +61,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.5.0-rc1-next-20160129-68987-g735cfa5 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Jan 29 01:18:00 PST 2016
+Linux version 4.5.0-rc1-next-20160201-69474-g99e8324 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Jan 31 22:48:12 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -71,20 +71,20 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0a35480, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0a37500, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-PERCPU: Embedded 13 pages/cpu @eef8a000 s23232 r8192 d21824 u53248
-pcpu-alloc: s23232 r8192 d21824 u53248 alloc=13*4096
+PERCPU: Embedded 13 pages/cpu @eef8a000 s23296 r8192 d21760 u53248
+pcpu-alloc: s23296 r8192 d21760 u53248 alloc=13*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002076K/2096128K available (7224K kernel code, 371K rwdata, 2488K rodata, 340K init, 367K bss, 28516K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2002064K/2096128K available (7224K kernel code, 372K rwdata, 2492K rodata, 344K init, 367K bss, 28528K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -92,10 +92,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09847c4   (9714 kB)
-      .init : 0xc0985000 - 0xc09da000   ( 340 kB)
-      .data : 0xc09da000 - 0xc0a36fa0   ( 372 kB)
-       .bss : 0xc0a39000 - 0xc0a94d88   ( 368 kB)
+      .text : 0xc0008000 - 0xc0985818   (9719 kB)
+      .init : 0xc0986000 - 0xc09dc000   ( 344 kB)
+      .data : 0xc09dc000 - 0xc0a39060   ( 373 kB)
+       .bss : 0xc0a3c000 - 0xc0a97d88   ( 368 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -259,7 +259,6 @@
 mmc0: SDHCI controller on 700b0600.sdhci [700b0600.sdhci] using ADMA 64-bit
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
-mmc0: MAN_BKOPS_EN bit is not set
 mmc0: new HS200 MMC card at address 0001
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
@@ -352,44 +351,6 @@
 +12V_SATA: supplied by +VDD_MUX
 tegra-ahci 70027000.sata: AHCI 0001.0301 32 slots 2 ports 3 Gbps 0x1 impl platform mode
 tegra-ahci 70027000.sata: flags: 64bit ncq sntf pm led pio slum part sadm sds apst 
-------------[ cut here ]------------
-WARNING: CPU: 0 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
-workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
-Modules linked in:
-CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.5.0-rc1-next-20160129-68987-g735cfa5 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-Workqueue: deferwq deferred_probe_work_func
-[<c0017208>] (unwind_backtrace) from [<c0012874>] (show_stack+0x10/0x14)
-[<c0012874>] (show_stack) from [<c023d1ac>] (dump_stack+0x84/0xd0)
-[<c023d1ac>] (dump_stack) from [<c0026b70>] (warn_slowpath_common+0x74/0xb0)
-[<c0026b70>] (warn_slowpath_common) from [<c0026c40>] (warn_slowpath_fmt+0x30/0x40)
-[<c0026c40>] (warn_slowpath_fmt) from [<c00397f8>] (check_flush_dependency+0xc8/0x12c)
-[<c00397f8>] (check_flush_dependency) from [<c0039914>] (flush_work+0x54/0x11c)
-[<c0039914>] (flush_work) from [<c00c1084>] (lru_add_drain_all+0x124/0x180)
-[<c00c1084>] (lru_add_drain_all) from [<c00f2498>] (migrate_prep+0x8/0x10)
-[<c00f2498>] (migrate_prep) from [<c00bbbe0>] (alloc_contig_range+0xc4/0x2f8)
-[<c00bbbe0>] (alloc_contig_range) from [<c00f4008>] (cma_alloc+0xd8/0x1a0)
-[<c00f4008>] (cma_alloc) from [<c001c270>] (__alloc_from_contiguous+0x38/0xdc)
-[<c001c270>] (__alloc_from_contiguous) from [<c001cf38>] (__dma_alloc+0x228/0x2ac)
-[<c001cf38>] (__dma_alloc) from [<c001d040>] (arm_dma_alloc+0x48/0x54)
-[<c001d040>] (arm_dma_alloc) from [<c0346ed8>] (dmam_alloc_coherent+0x8c/0xf0)
-[<c0346ed8>] (dmam_alloc_coherent) from [<c038d04c>] (ahci_port_start+0xd0/0x1e0)
-[<c038d04c>] (ahci_port_start) from [<c0374ccc>] (ata_host_start+0xf8/0x1d4)
-[<c0374ccc>] (ata_host_start) from [<c0379c60>] (ata_host_activate+0x20/0x134)
-[<c0379c60>] (ata_host_activate) from [<c038c4f4>] (ahci_host_activate+0x3c/0x11c)
-[<c038c4f4>] (ahci_host_activate) from [<c038e8a8>] (ahci_platform_init_host+0x1cc/0x3c8)
-[<c038e8a8>] (ahci_platform_init_host) from [<c038e574>] (tegra_ahci_probe+0x404/0x4c4)
-[<c038e574>] (tegra_ahci_probe) from [<c0337efc>] (platform_drv_probe+0x50/0xb0)
-[<c0337efc>] (platform_drv_probe) from [<c0336964>] (driver_probe_device+0x1f4/0x2b0)
-[<c0336964>] (driver_probe_device) from [<c0334ef8>] (bus_for_each_drv+0x44/0x8c)
-[<c0334ef8>] (bus_for_each_drv) from [<c03366e8>] (__device_attach+0x9c/0x100)
-[<c03366e8>] (__device_attach) from [<c0335d74>] (bus_probe_device+0x84/0x8c)
-[<c0335d74>] (bus_probe_device) from [<c03361a8>] (deferred_probe_work_func+0x60/0x8c)
-[<c03361a8>] (deferred_probe_work_func) from [<c003a720>] (process_one_work+0x120/0x328)
-[<c003a720>] (process_one_work) from [<c003a98c>] (worker_thread+0x2c/0x4b4)
-[<c003a98c>] (worker_thread) from [<c003fac4>] (kthread+0xd8/0xf4)
-[<c003fac4>] (kthread) from [<c000f938>] (ret_from_fork+0x14/0x3c)
----[ end trace d63ce72123ae798a ]---
 scsi host0: tegra-ahci
 scsi host1: tegra-ahci
 ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 343
@@ -411,7 +372,7 @@
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA 64-bit
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:01:56 UTC (946684916)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:01:53 UTC (946684913)
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
@@ -426,11 +387,11 @@
 ata1: SATA link down (SStatus 0 SControl 300)
 EXT4-fs (mmcblk0p1): couldn't mount as ext3 due to feature incompatibilities
 EXT4-fs (mmcblk0p1): recovery complete
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:1.
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 devtmpfs: mounted
-Freeing unused kernel memory: 340K (c0985000 - c09da000)
+Freeing unused kernel memory: 344K (c0986000 - c09dc000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-01-31 22:54:26.723784921 -0800
+++ /tmp/foob	2016-01-31 22:54:26.727784920 -0800
@@ -49,7 +49,7 @@
 	 ########
 	 4.1 MiB/s
 done
-Bytes transferred = 6784824 (678738 hex)
+Bytes transferred = 6788728 (679678 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -60,7 +60,7 @@
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x678738 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x679678 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.5.0-rc1-next-20160129-68987-g735cfa5 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Fri Jan 29 01:17:35 PST 2016
+Linux version 4.5.0-rc1-next-20160201-69474-g99e8324 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Jan 31 22:47:50 PST 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -76,20 +76,20 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c1072500, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c1074580, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 65536 pages, LIFO batch:15
-PERCPU: Embedded 13 pages/cpu @ef7c9000 s22912 r8192 d22144 u53248
-pcpu-alloc: s22912 r8192 d22144 u53248 alloc=13*4096
+PERCPU: Embedded 13 pages/cpu @ef7c9000 s22976 r8192 d22080 u53248
+pcpu-alloc: s22976 r8192 d22080 u53248 alloc=13*4096
 pcpu-alloc: [0] 0 [0] 1 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 260608
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=11eb23c1-01 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 958684K/1048576K available (8503K kernel code, 1113K rwdata, 4048K rodata, 1100K init, 345K bss, 24356K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 958676K/1048576K available (8507K kernel code, 1113K rwdata, 4052K rodata, 1100K init, 345K bss, 24364K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -97,10 +97,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0e4ae44   (12556 kB)
-      .init : 0xc0e4b000 - 0xc0f5e000   (1100 kB)
-      .data : 0xc0f5e000 - 0xc1074480   (1114 kB)
-       .bss : 0xc1077000 - 0xc10cd5e0   ( 346 kB)
+      .text : 0xc0208000 - 0xc0e4cf00   (12564 kB)
+      .init : 0xc0e4d000 - 0xc0f60000   (1100 kB)
+      .data : 0xc0f60000 - 0xc1076540   (1114 kB)
+       .bss : 0xc1079000 - 0xc10cf5e0   ( 346 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -329,18 +329,19 @@
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
 mmc1: new high speed SD card at address b368
-rtc-em3027 2-0056: setting system clock to 2014-12-28 06:00:15 UTC (1419746415)
+rtc-em3027 2-0056: setting system clock to 2014-12-28 06:04:12 UTC (1419746652)
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
+asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1100K (c0e4b000 - c0f5e000)
+Freeing unused kernel memory: 1100K (c0e4d000 - c0f60000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-01-31 22:54:26.731784921 -0800
+++ /tmp/foob	2016-01-31 22:54:26.735784921 -0800
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -45,20 +45,20 @@
 	 #################################################################
 	 #################################################################
 	 ##################################
-	 3.4 MiB/s
+	 3.9 MiB/s
 done
-Bytes transferred = 5261256 (5047c8 hex)
+Bytes transferred = 5262304 (504be0 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 2.9 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x5047c8 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x504be0 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.5.0-rc1-next-20160129-68987-g735cfa5 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Jan 29 01:18:00 PST 2016
+Linux version 4.5.0-rc1-next-20160201-69474-g99e8324 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Jan 31 22:48:12 PST 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -76,20 +76,20 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c0a35480, node_mem_map ef7f7000
+free_area_init_node: node 0, pgdat c0a37500, node_mem_map ef7f7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 65536 pages, LIFO batch:15
-PERCPU: Embedded 13 pages/cpu @ef7c8000 s23232 r8192 d21824 u53248
-pcpu-alloc: s23232 r8192 d21824 u53248 alloc=13*4096
+PERCPU: Embedded 13 pages/cpu @ef7c8000 s23296 r8192 d21760 u53248
+pcpu-alloc: s23296 r8192 d21760 u53248 alloc=13*4096
 pcpu-alloc: [0] 0 [0] 1 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 260608
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=11eb23c1-01 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 963000K/1048576K available (7224K kernel code, 371K rwdata, 2488K rodata, 340K init, 367K bss, 20040K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 962988K/1048576K available (7224K kernel code, 372K rwdata, 2492K rodata, 344K init, 367K bss, 20052K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -97,10 +97,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09847c4   (9714 kB)
-      .init : 0xc0985000 - 0xc09da000   ( 340 kB)
-      .data : 0xc09da000 - 0xc0a36fa0   ( 372 kB)
-       .bss : 0xc0a39000 - 0xc0a94d88   ( 368 kB)
+      .text : 0xc0008000 - 0xc0985818   (9719 kB)
+      .init : 0xc0986000 - 0xc09dc000   ( 344 kB)
+      .data : 0xc09dc000 - 0xc0a39060   ( 373 kB)
+       .bss : 0xc0a3c000 - 0xc0a97d88   ( 368 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
@@ -303,16 +303,16 @@
 tlv320aic23-codec 2-001a: ASoC: no dapm match for MICIN --> NULL --> Mic Input
 tlv320aic23-codec 2-001a: ASoC: Failed to add route MICIN -> NULL -> Mic Input
 tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
 NET: Registered protocol family 10
-mmc0: host does not support reading read-only switch, assuming write-enable
 mip6: Mobile IPv6
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
+mmc0: host does not support reading read-only switch, assuming write-enable
+sit: IPv6 over IPv4 tunneling driver
 mmc0: new high speed SDHC card at address b368
 mmc1: new high speed SD card at address b368
-sit: IPv6 over IPv4 tunneling driver
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk0: p1
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
 NET: Registered protocol family 17
 NET: Registered protocol family 15
@@ -331,15 +331,15 @@
 Registering SWP/SWPB emulation handler
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
-hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
+hub 1-1:1.0: USB hub found
+hub 1-1:1.0: 4 ports detected
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-28 06:01:15 UTC (1419746475)
+rtc-em3027 2-0056: setting system clock to 2014-12-28 06:05:13 UTC (1419746713)
 ALSA device list:
   #0: tegra-trimslice
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
@@ -349,7 +349,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 340K (c0985000 - c09da000)
+Freeing unused kernel memory: 344K (c0986000 - c09dc000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2016-01-31 22:54:26.739784921 -0800
+++ /tmp/foob	2016-01-31 22:54:26.743784920 -0800
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 806908 us
-Basic modules init took 348525 us
-Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14376 us
-Carveout took 252778 us
-CPU initialization took 92304 us
-Total time taken by TegraBoot 707983 us
+Tegraboot started after 829697 us
+Basic modules init took 349225 us
+Sec Bootdevice Read Time = 203 ms, Read Size = 8459 KB
+Next stage binary read took 14384 us
+Carveout took 252707 us
+CPU initialization took 93244 us
+Total time taken by TegraBoot 709560 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.644] LPDDR4 Training: Number of tables = 10
+64b[0001.668] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=/dev/mmcblk0p1 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-10762240 bytes read in 698 ms (14.7 MiB/s)
+10762240 bytes read in 697 ms (14.7 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-37136 bytes read in 454 ms (79.1 KiB/s)
+37136 bytes read in 453 ms (79.1 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 495 ms (3.1 MiB/s)
+1627787 bytes read in 494 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.5.0-rc1-next-20160129+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Fri Jan 29 01:19:25 PST 2016
+Linux version 4.5.0-rc1-next-20160201+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Jan 31 22:49:42 PST 2016
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: Early serial console at MMIO32 0x70006000 (options '')
@@ -258,7 +258,7 @@
 kvm [1]: Hyp mode initialized successfully
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.411:1): initialized
+audit: type=2000 audit(0.415:1): initialized
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
 Key type id_resolver registered
@@ -316,7 +316,6 @@
 ALSA device list:
   No soundcards found.
 Waiting for root device /dev/mmcblk0p1...
-mmc0: MAN_BKOPS_EN bit is not set
 mmc0: new HS200 MMC card at address 0001
 mmcblk0: mmc0:0001 SDW16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SDW16G partition 1 4.00 MiB
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-01-31 22:54:26.747784921 -0800
+++ /tmp/foob	2016-01-31 22:54:26.747784921 -0800
@@ -41,18 +41,18 @@
 	 ########
 	 3.3 MiB/s
 done
-Bytes transferred = 6784824 (678738 hex)
+Bytes transferred = 6788728 (679678 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x678738 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x679678 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.5.0-rc1-next-20160129-68987-g735cfa5 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Fri Jan 29 01:17:35 PST 2016
+Linux version 4.5.0-rc1-next-20160201-69474-g99e8324 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Jan 31 22:47:50 PST 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,20 +68,20 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c1072500, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c1074580, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-PERCPU: Embedded 13 pages/cpu @eef8f000 s22912 r8192 d22144 u53248
-pcpu-alloc: s22912 r8192 d22144 u53248 alloc=13*4096
+PERCPU: Embedded 13 pages/cpu @eef8f000 s22976 r8192 d22080 u53248
+pcpu-alloc: s22976 r8192 d22080 u53248 alloc=13*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1997788K/2096128K available (8503K kernel code, 1113K rwdata, 4048K rodata, 1100K init, 345K bss, 32804K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1997780K/2096128K available (8507K kernel code, 1113K rwdata, 4052K rodata, 1100K init, 345K bss, 32812K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -89,10 +89,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0e4ae44   (12556 kB)
-      .init : 0xc0e4b000 - 0xc0f5e000   (1100 kB)
-      .data : 0xc0f5e000 - 0xc1074480   (1114 kB)
-       .bss : 0xc1077000 - 0xc10cd5e0   ( 346 kB)
+      .text : 0xc0208000 - 0xc0e4cf00   (12564 kB)
+      .init : 0xc0e4d000 - 0xc0f60000   (1100 kB)
+      .data : 0xc0f60000 - 0xc1076540   (1114 kB)
+       .bss : 0xc1079000 - 0xc10cf5e0   ( 346 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -339,43 +339,6 @@
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci 7d004000.usb: EHCI Host Controller
 tegra-ehci 7d004000.usb: new USB bus registered, assigned bus number 1
-------------[ cut here ]------------
-WARNING: CPU: 3 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
-workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
-Modules linked in:
-CPU: 3 PID: 6 Comm: kworker/u8:0 Not tainted 4.5.0-rc1-next-20160129-68987-g735cfa5 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-Workqueue: deferwq deferred_probe_work_func
-[<c02188b8>] (unwind_backtrace) from [<c0213f94>] (show_stack+0x10/0x14)
-[<c0213f94>] (show_stack) from [<c0489bf4>] (dump_stack+0x78/0x94)
-[<c0489bf4>] (dump_stack) from [<c024ab38>] (warn_slowpath_common+0x74/0xb0)
-[<c024ab38>] (warn_slowpath_common) from [<c024ac08>] (warn_slowpath_fmt+0x30/0x40)
-[<c024ac08>] (warn_slowpath_fmt) from [<c025dc80>] (check_flush_dependency+0xc8/0x124)
-[<c025dc80>] (check_flush_dependency) from [<c025dd94>] (flush_work+0x54/0x144)
-[<c025dd94>] (flush_work) from [<c02db314>] (lru_add_drain_all+0x124/0x180)
-[<c02db314>] (lru_add_drain_all) from [<c030cda4>] (migrate_prep+0x8/0x10)
-[<c030cda4>] (migrate_prep) from [<c02d6310>] (alloc_contig_range+0xc4/0x2f8)
-[<c02d6310>] (alloc_contig_range) from [<c030e6d0>] (cma_alloc+0xd8/0x1a0)
-[<c030e6d0>] (cma_alloc) from [<c021e39c>] (__alloc_from_contiguous+0x38/0xdc)
-[<c021e39c>] (__alloc_from_contiguous) from [<c021ff8c>] (__dma_alloc+0x21c/0x250)
-[<c021ff8c>] (__dma_alloc) from [<c0220044>] (arm_dma_alloc+0x48/0x54)
-[<c0220044>] (arm_dma_alloc) from [<c03072b8>] (dma_pool_alloc+0x1dc/0x210)
-[<c03072b8>] (dma_pool_alloc) from [<c07c8590>] (ehci_qh_alloc+0x3c/0xb4)
-[<c07c8590>] (ehci_qh_alloc) from [<c07ccb88>] (ehci_setup+0x154/0x484)
-[<c07ccb88>] (ehci_setup) from [<c07d2c64>] (tegra_ehci_reset+0xc/0x50)
-[<c07d2c64>] (tegra_ehci_reset) from [<c07a3010>] (usb_add_hcd+0x300/0x840)
-[<c07a3010>] (usb_add_hcd) from [<c07d25dc>] (tegra_ehci_probe+0x3b8/0x504)
-[<c07d25dc>] (tegra_ehci_probe) from [<c065a41c>] (platform_drv_probe+0x50/0xb0)
-[<c065a41c>] (platform_drv_probe) from [<c0658d88>] (driver_probe_device+0x1f4/0x2b0)
-[<c0658d88>] (driver_probe_device) from [<c0657318>] (bus_for_each_drv+0x44/0x8c)
-[<c0657318>] (bus_for_each_drv) from [<c0658b0c>] (__device_attach+0x9c/0x100)
-[<c0658b0c>] (__device_attach) from [<c0658198>] (bus_probe_device+0x84/0x8c)
-[<c0658198>] (bus_probe_device) from [<c06585cc>] (deferred_probe_work_func+0x60/0x8c)
-[<c06585cc>] (deferred_probe_work_func) from [<c025ebdc>] (process_one_work+0x134/0x338)
-[<c025ebdc>] (process_one_work) from [<c025ee44>] (worker_thread+0x2c/0x4f0)
-[<c025ee44>] (worker_thread) from [<c0263fd0>] (kthread+0xd8/0xf4)
-[<c0263fd0>] (kthread) from [<c0210e78>] (ret_from_fork+0x14/0x3c)
----[ end trace 9757aacb7749fc08 ]---
 tegra-ehci 7d004000.usb: irq 343, io mem 0x7d004000
 tegra-ehci 7d004000.usb: USB 2.0 started, EHCI 1.00
 hub 1-0:1.0: USB hub found
@@ -395,7 +358,7 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 1100K (c0e4b000 - c0f5e000)
+Freeing unused kernel memory: 1100K (c0e4d000 - c0f60000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.018914] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.723937] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-01-31 22:54:26.755784922 -0800
+++ /tmp/foob	2016-01-31 22:54:26.755784922 -0800
@@ -39,18 +39,18 @@
 	 ##################################
 	 3.4 MiB/s
 done
-Bytes transferred = 5261256 (5047c8 hex)
+Bytes transferred = 5262304 (504be0 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5047c8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x504be0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.5.0-rc1-next-20160129-68987-g735cfa5 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Fri Jan 29 01:18:00 PST 2016
+Linux version 4.5.0-rc1-next-20160201-69474-g99e8324 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Jan 31 22:48:12 PST 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,20 +68,20 @@
 cma: Reserved 64 MiB at 0xfbc00000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c0a35480, node_mem_map eeff7000
+free_area_init_node: node 0, pgdat c0a37500, node_mem_map eeff7000
   Normal zone: 1536 pages used for memmap
   Normal zone: 0 pages reserved
   Normal zone: 196608 pages, LIFO batch:31
   HighMem zone: 327424 pages, LIFO batch:31
-PERCPU: Embedded 13 pages/cpu @eef8e000 s23232 r8192 d21824 u53248
-pcpu-alloc: s23232 r8192 d21824 u53248 alloc=13*4096
+PERCPU: Embedded 13 pages/cpu @eef8e000 s23296 r8192 d21760 u53248
+pcpu-alloc: s23296 r8192 d21760 u53248 alloc=13*4096
 pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
 Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 522496
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=/dev/mmcblk0p1 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2002104K/2096128K available (7224K kernel code, 371K rwdata, 2488K rodata, 340K init, 367K bss, 28488K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2002092K/2096128K available (7224K kernel code, 372K rwdata, 2492K rodata, 344K init, 367K bss, 28500K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -89,10 +89,10 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc09847c4   (9714 kB)
-      .init : 0xc0985000 - 0xc09da000   ( 340 kB)
-      .data : 0xc09da000 - 0xc0a36fa0   ( 372 kB)
-       .bss : 0xc0a39000 - 0xc0a94d88   ( 368 kB)
+      .text : 0xc0008000 - 0xc0985818   (9719 kB)
+      .init : 0xc0986000 - 0xc09dc000   ( 344 kB)
+      .data : 0xc09dc000 - 0xc0a39060   ( 373 kB)
+       .bss : 0xc0a3c000 - 0xc0a97d88   ( 368 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -337,43 +337,6 @@
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci 7d004000.usb: EHCI Host Controller
 tegra-ehci 7d004000.usb: new USB bus registered, assigned bus number 1
-------------[ cut here ]------------
-WARNING: CPU: 0 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
-workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
-Modules linked in:
-CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.5.0-rc1-next-20160129-68987-g735cfa5 #1
-Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
-Workqueue: deferwq deferred_probe_work_func
-[<c0017208>] (unwind_backtrace) from [<c0012874>] (show_stack+0x10/0x14)
-[<c0012874>] (show_stack) from [<c023d1ac>] (dump_stack+0x84/0xd0)
-[<c023d1ac>] (dump_stack) from [<c0026b70>] (warn_slowpath_common+0x74/0xb0)
-[<c0026b70>] (warn_slowpath_common) from [<c0026c40>] (warn_slowpath_fmt+0x30/0x40)
-[<c0026c40>] (warn_slowpath_fmt) from [<c00397f8>] (check_flush_dependency+0xc8/0x12c)
-[<c00397f8>] (check_flush_dependency) from [<c0039914>] (flush_work+0x54/0x11c)
-[<c0039914>] (flush_work) from [<c00c1084>] (lru_add_drain_all+0x124/0x180)
-[<c00c1084>] (lru_add_drain_all) from [<c00f2498>] (migrate_prep+0x8/0x10)
-[<c00f2498>] (migrate_prep) from [<c00bbbe0>] (alloc_contig_range+0xc4/0x2f8)
-[<c00bbbe0>] (alloc_contig_range) from [<c00f4008>] (cma_alloc+0xd8/0x1a0)
-[<c00f4008>] (cma_alloc) from [<c001c270>] (__alloc_from_contiguous+0x38/0xdc)
-[<c001c270>] (__alloc_from_contiguous) from [<c001cf38>] (__dma_alloc+0x228/0x2ac)
-[<c001cf38>] (__dma_alloc) from [<c001d040>] (arm_dma_alloc+0x48/0x54)
-[<c001d040>] (arm_dma_alloc) from [<c00ee1f8>] (dma_pool_alloc+0x28c/0x304)
-[<c00ee1f8>] (dma_pool_alloc) from [<c03fe770>] (ehci_qh_alloc+0x3c/0xb4)
-[<c03fe770>] (ehci_qh_alloc) from [<c040295c>] (ehci_setup+0x154/0x484)
-[<c040295c>] (ehci_setup) from [<c04066c8>] (tegra_ehci_reset+0xc/0x50)
-[<c04066c8>] (tegra_ehci_reset) from [<c03ea15c>] (usb_add_hcd+0x2fc/0x814)
-[<c03ea15c>] (usb_add_hcd) from [<c0406040>] (tegra_ehci_probe+0x3b8/0x504)
-[<c0406040>] (tegra_ehci_probe) from [<c0337efc>] (platform_drv_probe+0x50/0xb0)
-[<c0337efc>] (platform_drv_probe) from [<c0336964>] (driver_probe_device+0x1f4/0x2b0)
-[<c0336964>] (driver_probe_device) from [<c0334ef8>] (bus_for_each_drv+0x44/0x8c)
-[<c0334ef8>] (bus_for_each_drv) from [<c03366e8>] (__device_attach+0x9c/0x100)
-[<c03366e8>] (__device_attach) from [<c0335d74>] (bus_probe_device+0x84/0x8c)
-[<c0335d74>] (bus_probe_device) from [<c03361a8>] (deferred_probe_work_func+0x60/0x8c)
-[<c03361a8>] (deferred_probe_work_func) from [<c003a720>] (process_one_work+0x120/0x328)
-[<c003a720>] (process_one_work) from [<c003a98c>] (worker_thread+0x2c/0x4b4)
-[<c003a98c>] (worker_thread) from [<c003fac4>] (kthread+0xd8/0xf4)
-[<c003fac4>] (kthread) from [<c000f938>] (ret_from_fork+0x14/0x3c)
----[ end trace 55c05a7b8cd18b9d ]---
 tegra-ehci 7d004000.usb: irq 343, io mem 0x7d004000
 tegra-ehci 7d004000.usb: USB 2.0 started, EHCI 1.00
 hub 1-0:1.0: USB hub found
@@ -395,7 +358,8 @@
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
-Freeing unused kernel memory: 340K (c0985000 - c09da000)
+Freeing unused kernel memory: 344K (c0986000 - c09dc000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.390429] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# 
\ No newline at end of file
