Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-01-18 19:54:10.569239042 -0800
+++ /tmp/foob	2016-01-18 19:54:10.573239040 -0800
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-next-20160118+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Sun Jan 17 20:21:00 PST 2016
+Linux version 4.4.0-next-20160119+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Jan 18 19:49:10 PST 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -19,7 +19,7 @@
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
 software IO TLB [mem 0x79c00000-0x7dc00000] (64MB) mapped at [ffffffc039c00000-ffffffc03dbfffff]
-Memory: 937852K/1048576K available (5952K kernel code, 572K rwdata, 2748K rodata, 692K init, 221K bss, 94340K reserved, 16384K cma-reserved)
+Memory: 937852K/1048576K available (5956K kernel code, 568K rwdata, 2748K rodata, 692K init, 221K bss, 94340K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     vmalloc : 0xffffff8000000000 - 0xffffffbdbfff0000   (   246 GB)
     vmemmap : 0xffffffbdc0000000 - 0xffffffbfc0000000   (     8 GB maximum)
@@ -28,9 +28,9 @@
     PCI I/O : 0xffffffbffae00000 - 0xffffffbffbe00000   (    16 MB)
     modules : 0xffffffbffc000000 - 0xffffffc000000000   (    64 MB)
     memory  : 0xffffffc000000000 - 0xffffffc040000000   (  1024 MB)
-      .init : 0xffffffc000902000 - 0xffffffc0009af000   (   692 KB)
-      .text : 0xffffffc000080000 - 0xffffffc000901224   (  8709 KB)
-      .data : 0xffffffc0009af000 - 0xffffffc000a3e000   (   572 KB)
+      .init : 0xffffffc000903000 - 0xffffffc0009b0000   (   692 KB)
+      .text : 0xffffffc000080000 - 0xffffffc000902224   (  8713 KB)
+      .data : 0xffffffc0009b0000 - 0xffffffc000a3e000   (   568 KB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 64.
@@ -92,7 +92,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.432:1): initialized
+audit: type=2000 audit(0.660:1): initialized
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
 Key type id_resolver registered
@@ -138,7 +138,7 @@
 EXT4-fs (vda2): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 254:2.
 devtmpfs: mounted
-Freeing unused kernel memory: 692K (ffffffc000902000 - ffffffc0009af000)
+Freeing unused kernel memory: 692K (ffffffc000903000 - ffffffc0009b0000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 bash-4.3# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-01-18 19:54:10.577239037 -0800
+++ /tmp/foob	2016-01-18 19:54:10.581239034 -0800
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 ######
-	 3.2 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 6761776 (672d30 hex)
+Bytes transferred = 6761352 (672b88 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x672d30 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x672b88 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-next-20160118-67399-g92704bd (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Jan 17 20:18:41 PST 2016
+Linux version 4.4.0-next-20160119-67464-g7fa8f5d (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jan 18 19:47:50 PST 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -90,7 +90,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0e41d90   (12520 kB)
+      .text : 0xc0208000 - 0xc0e41da0   (12520 kB)
       .init : 0xc0e42000 - 0xc0f48000   (1048 kB)
       .data : 0xc0f48000 - 0xc105dcc0   (1112 kB)
        .bss : 0xc1060000 - 0xc10b65a0   ( 346 kB)
@@ -292,48 +292,48 @@
 tegra-ehci 7d008000.usb: EHCI Host Controller
 tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 1
 ------------[ cut here ]------------
-WARNING: CPU: 2 PID: 81 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
+WARNING: CPU: 3 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 2 PID: 81 Comm: kworker/u8:1 Not tainted 4.4.0-next-20160118-67399-g92704bd #1
+CPU: 3 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160119-67464-g7fa8f5d #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c02188e8>] (unwind_backtrace) from [<c0213fb8>] (show_stack+0x10/0x14)
-[<c0213fb8>] (show_stack) from [<c0488a8c>] (dump_stack+0x78/0x94)
-[<c0488a8c>] (dump_stack) from [<c024ab48>] (warn_slowpath_common+0x74/0xb0)
+[<c0213fb8>] (show_stack) from [<c0488a6c>] (dump_stack+0x78/0x94)
+[<c0488a6c>] (dump_stack) from [<c024ab48>] (warn_slowpath_common+0x74/0xb0)
 [<c024ab48>] (warn_slowpath_common) from [<c024ac18>] (warn_slowpath_fmt+0x30/0x40)
 [<c024ac18>] (warn_slowpath_fmt) from [<c025dcb0>] (check_flush_dependency+0xc8/0x124)
 [<c025dcb0>] (check_flush_dependency) from [<c025ddc4>] (flush_work+0x54/0x144)
-[<c025ddc4>] (flush_work) from [<c02da630>] (lru_add_drain_all+0x130/0x18c)
-[<c02da630>] (lru_add_drain_all) from [<c030c250>] (migrate_prep+0x8/0x10)
-[<c030c250>] (migrate_prep) from [<c02d5630>] (alloc_contig_range+0xc4/0x2f8)
-[<c02d5630>] (alloc_contig_range) from [<c030db7c>] (cma_alloc+0xd8/0x1a0)
-[<c030db7c>] (cma_alloc) from [<c021e3dc>] (__alloc_from_contiguous+0x38/0xdc)
+[<c025ddc4>] (flush_work) from [<c02da614>] (lru_add_drain_all+0x130/0x18c)
+[<c02da614>] (lru_add_drain_all) from [<c030c234>] (migrate_prep+0x8/0x10)
+[<c030c234>] (migrate_prep) from [<c02d5614>] (alloc_contig_range+0xc4/0x2f8)
+[<c02d5614>] (alloc_contig_range) from [<c030db60>] (cma_alloc+0xd8/0x1a0)
+[<c030db60>] (cma_alloc) from [<c021e3dc>] (__alloc_from_contiguous+0x38/0xdc)
 [<c021e3dc>] (__alloc_from_contiguous) from [<c021ffcc>] (__dma_alloc+0x21c/0x250)
 [<c021ffcc>] (__dma_alloc) from [<c0220084>] (arm_dma_alloc+0x48/0x54)
-[<c0220084>] (arm_dma_alloc) from [<c0306768>] (dma_pool_alloc+0x1dc/0x210)
-[<c0306768>] (dma_pool_alloc) from [<c07c7034>] (ehci_qh_alloc+0x3c/0xb4)
-[<c07c7034>] (ehci_qh_alloc) from [<c07cb62c>] (ehci_setup+0x154/0x478)
-[<c07cb62c>] (ehci_setup) from [<c07d16e4>] (tegra_ehci_reset+0xc/0x50)
-[<c07d16e4>] (tegra_ehci_reset) from [<c07a1aac>] (usb_add_hcd+0x300/0x830)
-[<c07a1aac>] (usb_add_hcd) from [<c07d105c>] (tegra_ehci_probe+0x3b8/0x504)
-[<c07d105c>] (tegra_ehci_probe) from [<c065914c>] (platform_drv_probe+0x50/0xac)
-[<c065914c>] (platform_drv_probe) from [<c0657ab8>] (driver_probe_device+0x1f4/0x2b0)
-[<c0657ab8>] (driver_probe_device) from [<c0656048>] (bus_for_each_drv+0x44/0x8c)
-[<c0656048>] (bus_for_each_drv) from [<c065783c>] (__device_attach+0x9c/0x100)
-[<c065783c>] (__device_attach) from [<c0656ec8>] (bus_probe_device+0x84/0x8c)
-[<c0656ec8>] (bus_probe_device) from [<c06572fc>] (deferred_probe_work_func+0x60/0x8c)
-[<c06572fc>] (deferred_probe_work_func) from [<c025ec0c>] (process_one_work+0x134/0x338)
+[<c0220084>] (arm_dma_alloc) from [<c030674c>] (dma_pool_alloc+0x1dc/0x210)
+[<c030674c>] (dma_pool_alloc) from [<c07c7030>] (ehci_qh_alloc+0x3c/0xb4)
+[<c07c7030>] (ehci_qh_alloc) from [<c07cb628>] (ehci_setup+0x154/0x478)
+[<c07cb628>] (ehci_setup) from [<c07d16e0>] (tegra_ehci_reset+0xc/0x50)
+[<c07d16e0>] (tegra_ehci_reset) from [<c07a1aa8>] (usb_add_hcd+0x300/0x830)
+[<c07a1aa8>] (usb_add_hcd) from [<c07d1058>] (tegra_ehci_probe+0x3b8/0x504)
+[<c07d1058>] (tegra_ehci_probe) from [<c06590d0>] (platform_drv_probe+0x50/0xac)
+[<c06590d0>] (platform_drv_probe) from [<c0657a3c>] (driver_probe_device+0x1f4/0x2b0)
+[<c0657a3c>] (driver_probe_device) from [<c0655fcc>] (bus_for_each_drv+0x44/0x8c)
+[<c0655fcc>] (bus_for_each_drv) from [<c06577c0>] (__device_attach+0x9c/0x100)
+[<c06577c0>] (__device_attach) from [<c0656e4c>] (bus_probe_device+0x84/0x8c)
+[<c0656e4c>] (bus_probe_device) from [<c0657280>] (deferred_probe_work_func+0x60/0x8c)
+[<c0657280>] (deferred_probe_work_func) from [<c025ec0c>] (process_one_work+0x134/0x338)
 [<c025ec0c>] (process_one_work) from [<c025ee74>] (worker_thread+0x2c/0x4f0)
 [<c025ee74>] (worker_thread) from [<c0264048>] (kthread+0xd8/0xf4)
 [<c0264048>] (kthread) from [<c0210e78>] (ret_from_fork+0x14/0x3c)
----[ end trace 73307e5e222f4db9 ]---
+---[ end trace 8720cb8d11710c70 ]---
 tegra-ehci 7d008000.usb: irq 339, io mem 0x7d008000
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-07-30 13:19:37 UTC (964963177)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-07-31 12:48:48 UTC (965047728)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -345,4 +345,4 @@
 Freeing unused kernel memory: 1048K (c0e42000 - c0f48000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.504219] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.504134] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-01-18 19:54:10.589239027 -0800
+++ /tmp/foob	2016-01-18 19:54:10.589239027 -0800
@@ -37,9 +37,9 @@
 	 #################################################################
 	 #################################################################
 	 #################################
-	 2.9 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 5253992 (502b68 hex)
+Bytes transferred = 5253704 (502a48 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 34894 (884e hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x502b68 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x502a48 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff84d ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-next-20160118-67399-g92704bd (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Jan 17 20:18:58 PST 2016
+Linux version 4.4.0-next-20160119-67464-g7fa8f5d (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jan 18 19:48:04 PST 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -90,7 +90,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc098172c   (9702 kB)
+      .text : 0xc0008000 - 0xc0981724   (9702 kB)
       .init : 0xc0982000 - 0xc09d6000   ( 336 kB)
       .data : 0xc09d6000 - 0xc0a32f20   ( 372 kB)
        .bss : 0xc0a35000 - 0xc0a90d88   ( 368 kB)
@@ -289,48 +289,48 @@
 tegra-ehci 7d008000.usb: EHCI Host Controller
 tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 1
 ------------[ cut here ]------------
-WARNING: CPU: 0 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
+WARNING: CPU: 2 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160118-67399-g92704bd #1
+CPU: 2 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160119-67464-g7fa8f5d #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0017200>] (unwind_backtrace) from [<c0012888>] (show_stack+0x10/0x14)
-[<c0012888>] (show_stack) from [<c023b884>] (dump_stack+0x84/0xd0)
-[<c023b884>] (dump_stack) from [<c0026bfc>] (warn_slowpath_common+0x74/0xb0)
+[<c0012888>] (show_stack) from [<c023b864>] (dump_stack+0x84/0xd0)
+[<c023b864>] (dump_stack) from [<c0026bfc>] (warn_slowpath_common+0x74/0xb0)
 [<c0026bfc>] (warn_slowpath_common) from [<c0026ccc>] (warn_slowpath_fmt+0x30/0x40)
 [<c0026ccc>] (warn_slowpath_fmt) from [<c00398a4>] (check_flush_dependency+0xc8/0x12c)
 [<c00398a4>] (check_flush_dependency) from [<c00399c0>] (flush_work+0x54/0x11c)
-[<c00399c0>] (flush_work) from [<c00c043c>] (lru_add_drain_all+0x130/0x18c)
-[<c00c043c>] (lru_add_drain_all) from [<c00f1394>] (migrate_prep+0x8/0x10)
-[<c00f1394>] (migrate_prep) from [<c00baf9c>] (alloc_contig_range+0xc4/0x2f8)
-[<c00baf9c>] (alloc_contig_range) from [<c00f2f04>] (cma_alloc+0xd8/0x1a0)
-[<c00f2f04>] (cma_alloc) from [<c001c290>] (__alloc_from_contiguous+0x38/0xdc)
+[<c00399c0>] (flush_work) from [<c00c0420>] (lru_add_drain_all+0x130/0x18c)
+[<c00c0420>] (lru_add_drain_all) from [<c00f1378>] (migrate_prep+0x8/0x10)
+[<c00f1378>] (migrate_prep) from [<c00baf80>] (alloc_contig_range+0xc4/0x2f8)
+[<c00baf80>] (alloc_contig_range) from [<c00f2ee8>] (cma_alloc+0xd8/0x1a0)
+[<c00f2ee8>] (cma_alloc) from [<c001c290>] (__alloc_from_contiguous+0x38/0xdc)
 [<c001c290>] (__alloc_from_contiguous) from [<c001cf58>] (__dma_alloc+0x228/0x2ac)
 [<c001cf58>] (__dma_alloc) from [<c001d060>] (arm_dma_alloc+0x48/0x54)
-[<c001d060>] (arm_dma_alloc) from [<c00ed76c>] (dma_pool_alloc+0x28c/0x304)
-[<c00ed76c>] (dma_pool_alloc) from [<c03fd24c>] (ehci_qh_alloc+0x3c/0xb4)
-[<c03fd24c>] (ehci_qh_alloc) from [<c0401438>] (ehci_setup+0x154/0x478)
-[<c0401438>] (ehci_setup) from [<c0405180>] (tegra_ehci_reset+0xc/0x50)
-[<c0405180>] (tegra_ehci_reset) from [<c03e8c30>] (usb_add_hcd+0x2fc/0x804)
-[<c03e8c30>] (usb_add_hcd) from [<c0404af8>] (tegra_ehci_probe+0x3b8/0x504)
-[<c0404af8>] (tegra_ehci_probe) from [<c0336bac>] (platform_drv_probe+0x50/0xac)
-[<c0336bac>] (platform_drv_probe) from [<c0335614>] (driver_probe_device+0x1f4/0x2b0)
-[<c0335614>] (driver_probe_device) from [<c0333ba8>] (bus_for_each_drv+0x44/0x8c)
-[<c0333ba8>] (bus_for_each_drv) from [<c0335398>] (__device_attach+0x9c/0x100)
-[<c0335398>] (__device_attach) from [<c0334a24>] (bus_probe_device+0x84/0x8c)
-[<c0334a24>] (bus_probe_device) from [<c0334e58>] (deferred_probe_work_func+0x60/0x8c)
-[<c0334e58>] (deferred_probe_work_func) from [<c003a7cc>] (process_one_work+0x120/0x328)
+[<c001d060>] (arm_dma_alloc) from [<c00ed750>] (dma_pool_alloc+0x28c/0x304)
+[<c00ed750>] (dma_pool_alloc) from [<c03fd204>] (ehci_qh_alloc+0x3c/0xb4)
+[<c03fd204>] (ehci_qh_alloc) from [<c04013f0>] (ehci_setup+0x154/0x478)
+[<c04013f0>] (ehci_setup) from [<c0405138>] (tegra_ehci_reset+0xc/0x50)
+[<c0405138>] (tegra_ehci_reset) from [<c03e8be8>] (usb_add_hcd+0x2fc/0x804)
+[<c03e8be8>] (usb_add_hcd) from [<c0404ab0>] (tegra_ehci_probe+0x3b8/0x504)
+[<c0404ab0>] (tegra_ehci_probe) from [<c0336b64>] (platform_drv_probe+0x50/0xac)
+[<c0336b64>] (platform_drv_probe) from [<c03355cc>] (driver_probe_device+0x1f4/0x2b0)
+[<c03355cc>] (driver_probe_device) from [<c0333b60>] (bus_for_each_drv+0x44/0x8c)
+[<c0333b60>] (bus_for_each_drv) from [<c0335350>] (__device_attach+0x9c/0x100)
+[<c0335350>] (__device_attach) from [<c03349dc>] (bus_probe_device+0x84/0x8c)
+[<c03349dc>] (bus_probe_device) from [<c0334e10>] (deferred_probe_work_func+0x60/0x8c)
+[<c0334e10>] (deferred_probe_work_func) from [<c003a7cc>] (process_one_work+0x120/0x328)
 [<c003a7cc>] (process_one_work) from [<c003aa38>] (worker_thread+0x2c/0x4b4)
 [<c003aa38>] (worker_thread) from [<c003fbac>] (kthread+0xd8/0xf4)
 [<c003fbac>] (kthread) from [<c000f938>] (ret_from_fork+0x14/0x3c)
----[ end trace 68d29687d2fd93ee ]---
+---[ end trace 1b874460d101b9de ]---
 tegra-ehci 7d008000.usb: irq 339, io mem 0x7d008000
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-07-30 13:21:25 UTC (964963285)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-07-31 12:50:40 UTC (965047840)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-01-18 19:54:10.593239024 -0800
+++ /tmp/foob	2016-01-18 19:54:10.597239022 -0800
@@ -41,20 +41,20 @@
 	 #################################################################
 	 #################################################################
 	 ######
-	 3.1 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 6761776 (672d30 hex)
+Bytes transferred = 6761352 (672b88 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 2.9 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x672d30 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x672b88 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012a89
@@ -62,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-next-20160118-67399-g92704bd (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Jan 17 20:18:41 PST 2016
+Linux version 4.4.0-next-20160119-67464-g7fa8f5d (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jan 18 19:47:50 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -92,7 +92,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0e41d90   (12520 kB)
+      .text : 0xc0208000 - 0xc0e41da0   (12520 kB)
       .init : 0xc0e42000 - 0xc0f48000   (1048 kB)
       .data : 0xc0f48000 - 0xc105dcc0   (1112 kB)
        .bss : 0xc1060000 - 0xc10b65a0   ( 346 kB)
@@ -332,43 +332,43 @@
 tegra-ahci 70027000.sata: AHCI 0001.0301 32 slots 2 ports 3 Gbps 0x1 impl platform mode
 tegra-ahci 70027000.sata: flags: 64bit ncq sntf pm led pio slum part sadm sds apst 
 ------------[ cut here ]------------
-WARNING: CPU: 1 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
+WARNING: CPU: 2 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 1 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160118-67399-g92704bd #1
+CPU: 2 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160119-67464-g7fa8f5d #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c02188e8>] (unwind_backtrace) from [<c0213fb8>] (show_stack+0x10/0x14)
-[<c0213fb8>] (show_stack) from [<c0488a8c>] (dump_stack+0x78/0x94)
-[<c0488a8c>] (dump_stack) from [<c024ab48>] (warn_slowpath_common+0x74/0xb0)
+[<c0213fb8>] (show_stack) from [<c0488a6c>] (dump_stack+0x78/0x94)
+[<c0488a6c>] (dump_stack) from [<c024ab48>] (warn_slowpath_common+0x74/0xb0)
 [<c024ab48>] (warn_slowpath_common) from [<c024ac18>] (warn_slowpath_fmt+0x30/0x40)
 [<c024ac18>] (warn_slowpath_fmt) from [<c025dcb0>] (check_flush_dependency+0xc8/0x124)
 [<c025dcb0>] (check_flush_dependency) from [<c025ddc4>] (flush_work+0x54/0x144)
-[<c025ddc4>] (flush_work) from [<c02da630>] (lru_add_drain_all+0x130/0x18c)
-[<c02da630>] (lru_add_drain_all) from [<c030c250>] (migrate_prep+0x8/0x10)
-[<c030c250>] (migrate_prep) from [<c02d5630>] (alloc_contig_range+0xc4/0x2f8)
-[<c02d5630>] (alloc_contig_range) from [<c030db7c>] (cma_alloc+0xd8/0x1a0)
-[<c030db7c>] (cma_alloc) from [<c021e3dc>] (__alloc_from_contiguous+0x38/0xdc)
+[<c025ddc4>] (flush_work) from [<c02da614>] (lru_add_drain_all+0x130/0x18c)
+[<c02da614>] (lru_add_drain_all) from [<c030c234>] (migrate_prep+0x8/0x10)
+[<c030c234>] (migrate_prep) from [<c02d5614>] (alloc_contig_range+0xc4/0x2f8)
+[<c02d5614>] (alloc_contig_range) from [<c030db60>] (cma_alloc+0xd8/0x1a0)
+[<c030db60>] (cma_alloc) from [<c021e3dc>] (__alloc_from_contiguous+0x38/0xdc)
 [<c021e3dc>] (__alloc_from_contiguous) from [<c021ffcc>] (__dma_alloc+0x21c/0x250)
 [<c021ffcc>] (__dma_alloc) from [<c0220084>] (arm_dma_alloc+0x48/0x54)
-[<c0220084>] (arm_dma_alloc) from [<c066a288>] (dmam_alloc_coherent+0x8c/0xf0)
-[<c066a288>] (dmam_alloc_coherent) from [<c06c4d50>] (ahci_port_start+0xd0/0x1e0)
-[<c06c4d50>] (ahci_port_start) from [<c06acbf8>] (ata_host_start+0xf8/0x1d4)
-[<c06acbf8>] (ata_host_start) from [<c06b1b14>] (ata_host_activate+0x20/0x134)
-[<c06b1b14>] (ata_host_activate) from [<c06c41cc>] (ahci_host_activate+0x3c/0x11c)
-[<c06c41cc>] (ahci_host_activate) from [<c06c6110>] (ahci_platform_init_host+0x1cc/0x3c8)
-[<c06c6110>] (ahci_platform_init_host) from [<c06c8374>] (tegra_ahci_probe+0x404/0x4c4)
-[<c06c8374>] (tegra_ahci_probe) from [<c065914c>] (platform_drv_probe+0x50/0xac)
-[<c065914c>] (platform_drv_probe) from [<c0657ab8>] (driver_probe_device+0x1f4/0x2b0)
-[<c0657ab8>] (driver_probe_device) from [<c0656048>] (bus_for_each_drv+0x44/0x8c)
-[<c0656048>] (bus_for_each_drv) from [<c065783c>] (__device_attach+0x9c/0x100)
-[<c065783c>] (__device_attach) from [<c0656ec8>] (bus_probe_device+0x84/0x8c)
-[<c0656ec8>] (bus_probe_device) from [<c06572fc>] (deferred_probe_work_func+0x60/0x8c)
-[<c06572fc>] (deferred_probe_work_func) from [<c025ec0c>] (process_one_work+0x134/0x338)
+[<c0220084>] (arm_dma_alloc) from [<c066a20c>] (dmam_alloc_coherent+0x8c/0xf0)
+[<c066a20c>] (dmam_alloc_coherent) from [<c06c4cd4>] (ahci_port_start+0xd0/0x1e0)
+[<c06c4cd4>] (ahci_port_start) from [<c06acb7c>] (ata_host_start+0xf8/0x1d4)
+[<c06acb7c>] (ata_host_start) from [<c06b1a98>] (ata_host_activate+0x20/0x134)
+[<c06b1a98>] (ata_host_activate) from [<c06c4150>] (ahci_host_activate+0x3c/0x11c)
+[<c06c4150>] (ahci_host_activate) from [<c06c6094>] (ahci_platform_init_host+0x1cc/0x3c8)
+[<c06c6094>] (ahci_platform_init_host) from [<c06c82f8>] (tegra_ahci_probe+0x404/0x4c4)
+[<c06c82f8>] (tegra_ahci_probe) from [<c06590d0>] (platform_drv_probe+0x50/0xac)
+[<c06590d0>] (platform_drv_probe) from [<c0657a3c>] (driver_probe_device+0x1f4/0x2b0)
+[<c0657a3c>] (driver_probe_device) from [<c0655fcc>] (bus_for_each_drv+0x44/0x8c)
+[<c0655fcc>] (bus_for_each_drv) from [<c06577c0>] (__device_attach+0x9c/0x100)
+[<c06577c0>] (__device_attach) from [<c0656e4c>] (bus_probe_device+0x84/0x8c)
+[<c0656e4c>] (bus_probe_device) from [<c0657280>] (deferred_probe_work_func+0x60/0x8c)
+[<c0657280>] (deferred_probe_work_func) from [<c025ec0c>] (process_one_work+0x134/0x338)
 [<c025ec0c>] (process_one_work) from [<c025ee74>] (worker_thread+0x2c/0x4f0)
 [<c025ee74>] (worker_thread) from [<c0264048>] (kthread+0xd8/0xf4)
 [<c0264048>] (kthread) from [<c0210e78>] (ret_from_fork+0x14/0x3c)
----[ end trace c9660768f1ea9573 ]---
+---[ end trace 8df03c7e849b50bf ]---
 scsi host0: tegra-ahci
 scsi host1: tegra-ahci
 ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 341
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-01-18 19:54:10.601239018 -0800
+++ /tmp/foob	2016-01-18 19:54:10.605239014 -0800
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 #################################
-	 3.2 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 5253992 (502b68 hex)
+Bytes transferred = 5253704 (502a48 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.1 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 64138 (fa8a hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x502b68 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x502a48 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83012a89
@@ -61,7 +61,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-next-20160118-67399-g92704bd (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Jan 17 20:18:58 PST 2016
+Linux version 4.4.0-next-20160119-67464-g7fa8f5d (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jan 18 19:48:04 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -92,7 +92,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc098172c   (9702 kB)
+      .text : 0xc0008000 - 0xc0981724   (9702 kB)
       .init : 0xc0982000 - 0xc09d6000   ( 336 kB)
       .data : 0xc09d6000 - 0xc0a32f20   ( 372 kB)
        .bss : 0xc0a35000 - 0xc0a90d88   ( 368 kB)
@@ -301,42 +301,16 @@
 tegra-pcie 1003000.pcie-controller: link 0 down, ignoring
 tegra-pcie 1003000.pcie-controller: probing port 1, using 1 lanes
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000000
+tegra-pcie 1003000.pcie-controller: link 1 down, retrying
+tegra-pcie 1003000.pcie-controller: link 1 down, retrying
+tegra-pcie 1003000.pcie-controller: link 1 down, retrying
+tegra-pcie 1003000.pcie-controller: link 1 down, ignoring
 tegra-pcie 1003000.pcie-controller: PCI host bridge to bus 0000:00
 pci_bus 0000:00: root bus resource [mem 0x13000000-0x1fffffff]
 pci_bus 0000:00: root bus resource [mem 0x20000000-0x3fffffff pref]
 pci_bus 0000:00: root bus resource [bus 00-ff]
 pci_bus 0000:00: root bus resource [io  0x1000-0xffff]
-pci 0000:00:02.0: [10de:0e13] type 01 class 0x060400
-pci 0000:00:02.0: PME# supported from D0 D1 D2 D3hot D3cold
-PCI: bus0: Fast back to back transfers disabled
-pci 0000:00:02.0: bridge configuration invalid ([bus 00-00]), reconfiguring
-pci 0000:01:00.0: [10ec:8168] type 00 class 0x020000
-pci 0000:01:00.0: reg 0x10: [io  0x0000-0x00ff]
-pci 0000:01:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit]
-pci 0000:01:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
-pci 0000:01:00.0: supports D1 D2
-pci 0000:01:00.0: PME# supported from D0 D1 D2 D3hot D3cold
-PCI: bus1: Fast back to back transfers disabled
-pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
-pci 0000:00:02.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
-pci 0000:00:02.0: BAR 9: assigned [mem 0x20000000-0x200fffff 64bit pref]
-pci 0000:00:02.0: BAR 7: assigned [io  0x1000-0x1fff]
-pci 0000:01:00.0: BAR 4: assigned [mem 0x20000000-0x20003fff 64bit pref]
-pci 0000:01:00.0: BAR 2: assigned [mem 0x13000000-0x13000fff 64bit]
-pci 0000:01:00.0: BAR 0: assigned [io  0x1000-0x10ff]
-pci 0000:00:02.0: PCI bridge to [bus 01]
-pci 0000:00:02.0:   bridge window [io  0x1000-0x1fff]
-pci 0000:00:02.0:   bridge window [mem 0x13000000-0x130fffff]
-pci 0000:00:02.0:   bridge window [mem 0x20000000-0x200fffff 64bit pref]
-pci 0000:00:02.0: nv_msi_ht_cap_quirk didn't locate host bridge
-pcieport 0000:00:02.0: enabling device (0140 -> 0143)
-pcieport 0000:00:02.0: Signaling PME through PCIe PME interrupt
-pci 0000:01:00.0: Signaling PME through PCIe PME interrupt
-pcie_pme 0000:00:02.0:pcie01: service driver pcie_pme loaded
-r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
-r8169 0000:01:00.0: enabling device (0140 -> 0143)
-r8169 0000:01:00.0 eth0: RTL8168g/8111g at 0xf0952000, 00:04:4b:25:b3:b4, XID 0c000800 IRQ 388
-r8169 0000:01:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
+PCI: bus0: Fast back to back transfers enabled
 +5V_HDMI_CON: supplied by +5V_SYS
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 +3.3V_RUN: supplied by +3.3V_SYS
@@ -355,40 +329,40 @@
 WARNING: CPU: 0 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160118-67399-g92704bd #1
+CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160119-67464-g7fa8f5d #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0017200>] (unwind_backtrace) from [<c0012888>] (show_stack+0x10/0x14)
-[<c0012888>] (show_stack) from [<c023b884>] (dump_stack+0x84/0xd0)
-[<c023b884>] (dump_stack) from [<c0026bfc>] (warn_slowpath_common+0x74/0xb0)
+[<c0012888>] (show_stack) from [<c023b864>] (dump_stack+0x84/0xd0)
+[<c023b864>] (dump_stack) from [<c0026bfc>] (warn_slowpath_common+0x74/0xb0)
 [<c0026bfc>] (warn_slowpath_common) from [<c0026ccc>] (warn_slowpath_fmt+0x30/0x40)
 [<c0026ccc>] (warn_slowpath_fmt) from [<c00398a4>] (check_flush_dependency+0xc8/0x12c)
 [<c00398a4>] (check_flush_dependency) from [<c00399c0>] (flush_work+0x54/0x11c)
-[<c00399c0>] (flush_work) from [<c00c043c>] (lru_add_drain_all+0x130/0x18c)
-[<c00c043c>] (lru_add_drain_all) from [<c00f1394>] (migrate_prep+0x8/0x10)
-[<c00f1394>] (migrate_prep) from [<c00baf9c>] (alloc_contig_range+0xc4/0x2f8)
-[<c00baf9c>] (alloc_contig_range) from [<c00f2f04>] (cma_alloc+0xd8/0x1a0)
-[<c00f2f04>] (cma_alloc) from [<c001c290>] (__alloc_from_contiguous+0x38/0xdc)
+[<c00399c0>] (flush_work) from [<c00c0420>] (lru_add_drain_all+0x130/0x18c)
+[<c00c0420>] (lru_add_drain_all) from [<c00f1378>] (migrate_prep+0x8/0x10)
+[<c00f1378>] (migrate_prep) from [<c00baf80>] (alloc_contig_range+0xc4/0x2f8)
+[<c00baf80>] (alloc_contig_range) from [<c00f2ee8>] (cma_alloc+0xd8/0x1a0)
+[<c00f2ee8>] (cma_alloc) from [<c001c290>] (__alloc_from_contiguous+0x38/0xdc)
 [<c001c290>] (__alloc_from_contiguous) from [<c001cf58>] (__dma_alloc+0x228/0x2ac)
 [<c001cf58>] (__dma_alloc) from [<c001d060>] (arm_dma_alloc+0x48/0x54)
-[<c001d060>] (arm_dma_alloc) from [<c0345bb8>] (dmam_alloc_coherent+0x8c/0xf0)
-[<c0345bb8>] (dmam_alloc_coherent) from [<c038bc60>] (ahci_port_start+0xd0/0x1e0)
-[<c038bc60>] (ahci_port_start) from [<c0373938>] (ata_host_start+0xf8/0x1d4)
-[<c0373938>] (ata_host_start) from [<c03788cc>] (ata_host_activate+0x20/0x134)
-[<c03788cc>] (ata_host_activate) from [<c038b108>] (ahci_host_activate+0x3c/0x11c)
-[<c038b108>] (ahci_host_activate) from [<c038d4b0>] (ahci_platform_init_host+0x1cc/0x3c8)
-[<c038d4b0>] (ahci_platform_init_host) from [<c038d17c>] (tegra_ahci_probe+0x404/0x4c4)
-[<c038d17c>] (tegra_ahci_probe) from [<c0336bac>] (platform_drv_probe+0x50/0xac)
-[<c0336bac>] (platform_drv_probe) from [<c0335614>] (driver_probe_device+0x1f4/0x2b0)
-[<c0335614>] (driver_probe_device) from [<c0333ba8>] (bus_for_each_drv+0x44/0x8c)
-[<c0333ba8>] (bus_for_each_drv) from [<c0335398>] (__device_attach+0x9c/0x100)
-[<c0335398>] (__device_attach) from [<c0334a24>] (bus_probe_device+0x84/0x8c)
-[<c0334a24>] (bus_probe_device) from [<c0334e58>] (deferred_probe_work_func+0x60/0x8c)
-[<c0334e58>] (deferred_probe_work_func) from [<c003a7cc>] (process_one_work+0x120/0x328)
+[<c001d060>] (arm_dma_alloc) from [<c0345b70>] (dmam_alloc_coherent+0x8c/0xf0)
+[<c0345b70>] (dmam_alloc_coherent) from [<c038bc18>] (ahci_port_start+0xd0/0x1e0)
+[<c038bc18>] (ahci_port_start) from [<c03738f0>] (ata_host_start+0xf8/0x1d4)
+[<c03738f0>] (ata_host_start) from [<c0378884>] (ata_host_activate+0x20/0x134)
+[<c0378884>] (ata_host_activate) from [<c038b0c0>] (ahci_host_activate+0x3c/0x11c)
+[<c038b0c0>] (ahci_host_activate) from [<c038d468>] (ahci_platform_init_host+0x1cc/0x3c8)
+[<c038d468>] (ahci_platform_init_host) from [<c038d134>] (tegra_ahci_probe+0x404/0x4c4)
+[<c038d134>] (tegra_ahci_probe) from [<c0336b64>] (platform_drv_probe+0x50/0xac)
+[<c0336b64>] (platform_drv_probe) from [<c03355cc>] (driver_probe_device+0x1f4/0x2b0)
+[<c03355cc>] (driver_probe_device) from [<c0333b60>] (bus_for_each_drv+0x44/0x8c)
+[<c0333b60>] (bus_for_each_drv) from [<c0335350>] (__device_attach+0x9c/0x100)
+[<c0335350>] (__device_attach) from [<c03349dc>] (bus_probe_device+0x84/0x8c)
+[<c03349dc>] (bus_probe_device) from [<c0334e10>] (deferred_probe_work_func+0x60/0x8c)
+[<c0334e10>] (deferred_probe_work_func) from [<c003a7cc>] (process_one_work+0x120/0x328)
 [<c003a7cc>] (process_one_work) from [<c003aa38>] (worker_thread+0x2c/0x4b4)
 [<c003aa38>] (worker_thread) from [<c003fbac>] (kthread+0xd8/0xf4)
 [<c003fbac>] (kthread) from [<c000f938>] (ret_from_fork+0x14/0x3c)
----[ end trace 95ce28e445afe67f ]---
+---[ end trace d39f67b1b57f21ab ]---
 scsi host0: tegra-ahci
 scsi host1: tegra-ahci
 ata1: SATA max UDMA/133 mmio [mem 0x70027000-0x70028fff] port 0x100 irq 341
@@ -432,4 +406,4 @@
 Freeing unused kernel memory: 336K (c0982000 - c09d6000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    5.773621] asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:11:31:3b
+root@(none):/# 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-01-18 19:54:10.609239011 -0800
+++ /tmp/foob	2016-01-18 19:54:10.613239009 -0800
@@ -49,7 +49,7 @@
 	 ######
 	 4.1 MiB/s
 done
-Bytes transferred = 6761776 (672d30 hex)
+Bytes transferred = 6761352 (672b88 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -60,7 +60,7 @@
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x672d30 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x672b88 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-next-20160118-67399-g92704bd (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Jan 17 20:18:41 PST 2016
+Linux version 4.4.0-next-20160119-67464-g7fa8f5d (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jan 18 19:47:50 PST 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -97,7 +97,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0e41d90   (12520 kB)
+      .text : 0xc0208000 - 0xc0e41da0   (12520 kB)
       .init : 0xc0e42000 - 0xc0f48000   (1048 kB)
       .data : 0xc0f48000 - 0xc105dcc0   (1112 kB)
        .bss : 0xc1060000 - 0xc10b65a0   ( 346 kB)
@@ -329,7 +329,7 @@
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
 mmc1: new high speed SD card at address b368
-rtc-em3027 2-0056: setting system clock to 2014-12-28 04:02:17 UTC (1419739337)
+rtc-em3027 2-0056: setting system clock to 2014-12-28 04:04:16 UTC (1419739456)
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-01-18 19:54:10.617239006 -0800
+++ /tmp/foob	2016-01-18 19:54:10.621239003 -0800
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -45,20 +45,20 @@
 	 #################################################################
 	 #################################################################
 	 #################################
-	 4.1 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 5253992 (502b68 hex)
+Bytes transferred = 5253704 (502a48 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 21130 (528a hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x502b68 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x502a48 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 02410289
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-next-20160118-67399-g92704bd (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Jan 17 20:18:58 PST 2016
+Linux version 4.4.0-next-20160119-67464-g7fa8f5d (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jan 18 19:48:04 PST 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -97,7 +97,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc098172c   (9702 kB)
+      .text : 0xc0008000 - 0xc0981724   (9702 kB)
       .init : 0xc0982000 - 0xc09d6000   ( 336 kB)
       .data : 0xc09d6000 - 0xc0a32f20   ( 372 kB)
        .bss : 0xc0a35000 - 0xc0a90d88   ( 368 kB)
@@ -302,15 +302,14 @@
 tlv320aic23-codec 2-001a: Control not supported for path MICIN -> [NULL] -> Mic Input
 tlv320aic23-codec 2-001a: ASoC: no dapm match for MICIN --> NULL --> Mic Input
 tlv320aic23-codec 2-001a: ASoC: Failed to add route MICIN -> NULL -> Mic Input
-tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
+tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
 NET: Registered protocol family 10
 mmc0: host does not support reading read-only switch, assuming write-enable
 mip6: Mobile IPv6
-mmc1: new high speed SD card at address b368
 sit: IPv6 over IPv4 tunneling driver
+mmc1: new high speed SD card at address b368
 mmc0: new high speed SDHC card at address b368
-mmcblk0: mmc1:b368 NCard 1.83 GiB (ro)
 NET: Registered protocol family 17
 NET: Registered protocol family 15
 can: controller area network core (rev 20120528 abi 9)
@@ -330,16 +329,17 @@
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
+mmcblk0: mmc1:b368 NCard 1.83 GiB (ro)
 mmcblk1: mmc0:b368 NCard 3.70 GiB 
- mmcblk1: p1
  mmcblk0: p1 p2
+ mmcblk1: p1
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2014-12-28 04:03:18 UTC (1419739398)
+rtc-em3027 2-0056: setting system clock to 2014-12-28 04:05:14 UTC (1419739514)
 ALSA device list:
   #0: tegra-trimslice
 EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
@@ -351,4 +351,4 @@
 Freeing unused kernel memory: 336K (c0982000 - c09d6000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# 
\ No newline at end of file
+root@(none):/# [    3.287648] asix 1-1.2:1.0 eth1: register 'asix' at usb-c5008000.usb-1.2, ASIX AX88772 USB 2.0 Ethernet, 00:60:6e:60:6c:68
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-01-18 19:54:10.625238999 -0800
+++ /tmp/foob	2016-01-18 19:54:10.629238996 -0800
@@ -41,7 +41,7 @@
 	 ######
 	 3.3 MiB/s
 done
-Bytes transferred = 6761776 (672d30 hex)
+Bytes transferred = 6761352 (672b88 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -52,7 +52,7 @@
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x672d30 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x672b88 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-next-20160118-67399-g92704bd (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Sun Jan 17 20:18:41 PST 2016
+Linux version 4.4.0-next-20160119-67464-g7fa8f5d (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Jan 18 19:47:50 PST 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -89,7 +89,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc0e41d90   (12520 kB)
+      .text : 0xc0208000 - 0xc0e41da0   (12520 kB)
       .init : 0xc0e42000 - 0xc0f48000   (1048 kB)
       .data : 0xc0f48000 - 0xc105dcc0   (1112 kB)
        .bss : 0xc1060000 - 0xc10b65a0   ( 346 kB)
@@ -280,13 +280,13 @@
 sys_3v3,vdd_3v3_alw: supplied by vdd_5v_in
 sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
 tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
-tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 mmc1: new high speed MMC card at address 0001
 mmcblk0: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk0boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk0boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk0rpmb: mmc1:0001 SEM16G partition 3 128 KiB
 Primary GPT is invalid, using alternate GPT.
+tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
  mmcblk0: p1 p2 p3 p4 p5
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -340,42 +340,42 @@
 tegra-ehci 7d004000.usb: EHCI Host Controller
 tegra-ehci 7d004000.usb: new USB bus registered, assigned bus number 1
 ------------[ cut here ]------------
-WARNING: CPU: 0 PID: 80 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
+WARNING: CPU: 1 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x124()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 0 PID: 80 Comm: kworker/u8:1 Not tainted 4.4.0-next-20160118-67399-g92704bd #1
+CPU: 1 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160119-67464-g7fa8f5d #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c02188e8>] (unwind_backtrace) from [<c0213fb8>] (show_stack+0x10/0x14)
-[<c0213fb8>] (show_stack) from [<c0488a8c>] (dump_stack+0x78/0x94)
-[<c0488a8c>] (dump_stack) from [<c024ab48>] (warn_slowpath_common+0x74/0xb0)
+[<c0213fb8>] (show_stack) from [<c0488a6c>] (dump_stack+0x78/0x94)
+[<c0488a6c>] (dump_stack) from [<c024ab48>] (warn_slowpath_common+0x74/0xb0)
 [<c024ab48>] (warn_slowpath_common) from [<c024ac18>] (warn_slowpath_fmt+0x30/0x40)
 [<c024ac18>] (warn_slowpath_fmt) from [<c025dcb0>] (check_flush_dependency+0xc8/0x124)
 [<c025dcb0>] (check_flush_dependency) from [<c025ddc4>] (flush_work+0x54/0x144)
-[<c025ddc4>] (flush_work) from [<c02da630>] (lru_add_drain_all+0x130/0x18c)
-[<c02da630>] (lru_add_drain_all) from [<c030c250>] (migrate_prep+0x8/0x10)
-[<c030c250>] (migrate_prep) from [<c02d5630>] (alloc_contig_range+0xc4/0x2f8)
-[<c02d5630>] (alloc_contig_range) from [<c030db7c>] (cma_alloc+0xd8/0x1a0)
-[<c030db7c>] (cma_alloc) from [<c021e3dc>] (__alloc_from_contiguous+0x38/0xdc)
+[<c025ddc4>] (flush_work) from [<c02da614>] (lru_add_drain_all+0x130/0x18c)
+[<c02da614>] (lru_add_drain_all) from [<c030c234>] (migrate_prep+0x8/0x10)
+[<c030c234>] (migrate_prep) from [<c02d5614>] (alloc_contig_range+0xc4/0x2f8)
+[<c02d5614>] (alloc_contig_range) from [<c030db60>] (cma_alloc+0xd8/0x1a0)
+[<c030db60>] (cma_alloc) from [<c021e3dc>] (__alloc_from_contiguous+0x38/0xdc)
 [<c021e3dc>] (__alloc_from_contiguous) from [<c021ffcc>] (__dma_alloc+0x21c/0x250)
 [<c021ffcc>] (__dma_alloc) from [<c0220084>] (arm_dma_alloc+0x48/0x54)
-[<c0220084>] (arm_dma_alloc) from [<c0306768>] (dma_pool_alloc+0x1dc/0x210)
-[<c0306768>] (dma_pool_alloc) from [<c07c7034>] (ehci_qh_alloc+0x3c/0xb4)
-[<c07c7034>] (ehci_qh_alloc) from [<c07cb62c>] (ehci_setup+0x154/0x478)
-[<c07cb62c>] (ehci_setup) from [<c07d16e4>] (tegra_ehci_reset+0xc/0x50)
-[<c07d16e4>] (tegra_ehci_reset) from [<c07a1aac>] (usb_add_hcd+0x300/0x830)
-[<c07a1aac>] (usb_add_hcd) from [<c07d105c>] (tegra_ehci_probe+0x3b8/0x504)
-[<c07d105c>] (tegra_ehci_probe) from [<c065914c>] (platform_drv_probe+0x50/0xac)
-[<c065914c>] (platform_drv_probe) from [<c0657ab8>] (driver_probe_device+0x1f4/0x2b0)
-[<c0657ab8>] (driver_probe_device) from [<c0656048>] (bus_for_each_drv+0x44/0x8c)
-[<c0656048>] (bus_for_each_drv) from [<c065783c>] (__device_attach+0x9c/0x100)
-[<c065783c>] (__device_attach) from [<c0656ec8>] (bus_probe_device+0x84/0x8c)
-[<c0656ec8>] (bus_probe_device) from [<c06572fc>] (deferred_probe_work_func+0x60/0x8c)
-[<c06572fc>] (deferred_probe_work_func) from [<c025ec0c>] (process_one_work+0x134/0x338)
+[<c0220084>] (arm_dma_alloc) from [<c030674c>] (dma_pool_alloc+0x1dc/0x210)
+[<c030674c>] (dma_pool_alloc) from [<c07c7030>] (ehci_qh_alloc+0x3c/0xb4)
+[<c07c7030>] (ehci_qh_alloc) from [<c07cb628>] (ehci_setup+0x154/0x478)
+[<c07cb628>] (ehci_setup) from [<c07d16e0>] (tegra_ehci_reset+0xc/0x50)
+[<c07d16e0>] (tegra_ehci_reset) from [<c07a1aa8>] (usb_add_hcd+0x300/0x830)
+[<c07a1aa8>] (usb_add_hcd) from [<c07d1058>] (tegra_ehci_probe+0x3b8/0x504)
+[<c07d1058>] (tegra_ehci_probe) from [<c06590d0>] (platform_drv_probe+0x50/0xac)
+[<c06590d0>] (platform_drv_probe) from [<c0657a3c>] (driver_probe_device+0x1f4/0x2b0)
+[<c0657a3c>] (driver_probe_device) from [<c0655fcc>] (bus_for_each_drv+0x44/0x8c)
+[<c0655fcc>] (bus_for_each_drv) from [<c06577c0>] (__device_attach+0x9c/0x100)
+[<c06577c0>] (__device_attach) from [<c0656e4c>] (bus_probe_device+0x84/0x8c)
+[<c0656e4c>] (bus_probe_device) from [<c0657280>] (deferred_probe_work_func+0x60/0x8c)
+[<c0657280>] (deferred_probe_work_func) from [<c025ec0c>] (process_one_work+0x134/0x338)
 [<c025ec0c>] (process_one_work) from [<c025ee74>] (worker_thread+0x2c/0x4f0)
 [<c025ee74>] (worker_thread) from [<c0264048>] (kthread+0xd8/0xf4)
 [<c0264048>] (kthread) from [<c0210e78>] (ret_from_fork+0x14/0x3c)
----[ end trace baf9a99254816b3c ]---
+---[ end trace 5d0676211630d6e8 ]---
 tegra-ehci 7d004000.usb: irq 343, io mem 0x7d004000
 tegra-ehci 7d004000.usb: USB 2.0 started, EHCI 1.00
 hub 1-0:1.0: USB hub found
@@ -398,5 +398,4 @@
 Freeing unused kernel memory: 1048K (c0e42000 - c0f48000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-usb 2-1: new high-speed USB device number 2 using tegra-ehci
-root@(none):/# 
\ No newline at end of file
+root@(none):/# [    4.008924] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-01-18 19:54:10.633238993 -0800
+++ /tmp/foob	2016-01-18 19:54:10.637238990 -0800
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 #################################
-	 3.4 MiB/s
+	 2.8 MiB/s
 done
-Bytes transferred = 5253992 (502b68 hex)
+Bytes transferred = 5253704 (502a48 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 51467 (c90b hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x502b68 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x502a48 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90a ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.4.0-next-20160118-67399-g92704bd (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Sun Jan 17 20:18:58 PST 2016
+Linux version 4.4.0-next-20160119-67464-g7fa8f5d (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Jan 18 19:48:04 PST 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -89,7 +89,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc098172c   (9702 kB)
+      .text : 0xc0008000 - 0xc0981724   (9702 kB)
       .init : 0xc0982000 - 0xc09d6000   ( 336 kB)
       .data : 0xc09d6000 - 0xc0a32f20   ( 372 kB)
        .bss : 0xc0a35000 - 0xc0a90d88   ( 368 kB)
@@ -341,39 +341,39 @@
 WARNING: CPU: 0 PID: 6 at kernel/workqueue.c:2361 check_flush_dependency+0xc8/0x12c()
 workqueue: WQ_MEM_RECLAIM deferwq:deferred_probe_work_func is flushing !WQ_MEM_RECLAIM events:lru_add_drain_per_cpu
 Modules linked in:
-CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160118-67399-g92704bd #1
+CPU: 0 PID: 6 Comm: kworker/u8:0 Not tainted 4.4.0-next-20160119-67464-g7fa8f5d #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: deferwq deferred_probe_work_func
 [<c0017200>] (unwind_backtrace) from [<c0012888>] (show_stack+0x10/0x14)
-[<c0012888>] (show_stack) from [<c023b884>] (dump_stack+0x84/0xd0)
-[<c023b884>] (dump_stack) from [<c0026bfc>] (warn_slowpath_common+0x74/0xb0)
+[<c0012888>] (show_stack) from [<c023b864>] (dump_stack+0x84/0xd0)
+[<c023b864>] (dump_stack) from [<c0026bfc>] (warn_slowpath_common+0x74/0xb0)
 [<c0026bfc>] (warn_slowpath_common) from [<c0026ccc>] (warn_slowpath_fmt+0x30/0x40)
 [<c0026ccc>] (warn_slowpath_fmt) from [<c00398a4>] (check_flush_dependency+0xc8/0x12c)
 [<c00398a4>] (check_flush_dependency) from [<c00399c0>] (flush_work+0x54/0x11c)
-[<c00399c0>] (flush_work) from [<c00c043c>] (lru_add_drain_all+0x130/0x18c)
-[<c00c043c>] (lru_add_drain_all) from [<c00f1394>] (migrate_prep+0x8/0x10)
-[<c00f1394>] (migrate_prep) from [<c00baf9c>] (alloc_contig_range+0xc4/0x2f8)
-[<c00baf9c>] (alloc_contig_range) from [<c00f2f04>] (cma_alloc+0xd8/0x1a0)
-[<c00f2f04>] (cma_alloc) from [<c001c290>] (__alloc_from_contiguous+0x38/0xdc)
+[<c00399c0>] (flush_work) from [<c00c0420>] (lru_add_drain_all+0x130/0x18c)
+[<c00c0420>] (lru_add_drain_all) from [<c00f1378>] (migrate_prep+0x8/0x10)
+[<c00f1378>] (migrate_prep) from [<c00baf80>] (alloc_contig_range+0xc4/0x2f8)
+[<c00baf80>] (alloc_contig_range) from [<c00f2ee8>] (cma_alloc+0xd8/0x1a0)
+[<c00f2ee8>] (cma_alloc) from [<c001c290>] (__alloc_from_contiguous+0x38/0xdc)
 [<c001c290>] (__alloc_from_contiguous) from [<c001cf58>] (__dma_alloc+0x228/0x2ac)
 [<c001cf58>] (__dma_alloc) from [<c001d060>] (arm_dma_alloc+0x48/0x54)
-[<c001d060>] (arm_dma_alloc) from [<c00ed76c>] (dma_pool_alloc+0x28c/0x304)
-[<c00ed76c>] (dma_pool_alloc) from [<c03fd24c>] (ehci_qh_alloc+0x3c/0xb4)
-[<c03fd24c>] (ehci_qh_alloc) from [<c0401438>] (ehci_setup+0x154/0x478)
-[<c0401438>] (ehci_setup) from [<c0405180>] (tegra_ehci_reset+0xc/0x50)
-[<c0405180>] (tegra_ehci_reset) from [<c03e8c30>] (usb_add_hcd+0x2fc/0x804)
-[<c03e8c30>] (usb_add_hcd) from [<c0404af8>] (tegra_ehci_probe+0x3b8/0x504)
-[<c0404af8>] (tegra_ehci_probe) from [<c0336bac>] (platform_drv_probe+0x50/0xac)
-[<c0336bac>] (platform_drv_probe) from [<c0335614>] (driver_probe_device+0x1f4/0x2b0)
-[<c0335614>] (driver_probe_device) from [<c0333ba8>] (bus_for_each_drv+0x44/0x8c)
-[<c0333ba8>] (bus_for_each_drv) from [<c0335398>] (__device_attach+0x9c/0x100)
-[<c0335398>] (__device_attach) from [<c0334a24>] (bus_probe_device+0x84/0x8c)
-[<c0334a24>] (bus_probe_device) from [<c0334e58>] (deferred_probe_work_func+0x60/0x8c)
-[<c0334e58>] (deferred_probe_work_func) from [<c003a7cc>] (process_one_work+0x120/0x328)
+[<c001d060>] (arm_dma_alloc) from [<c00ed750>] (dma_pool_alloc+0x28c/0x304)
+[<c00ed750>] (dma_pool_alloc) from [<c03fd204>] (ehci_qh_alloc+0x3c/0xb4)
+[<c03fd204>] (ehci_qh_alloc) from [<c04013f0>] (ehci_setup+0x154/0x478)
+[<c04013f0>] (ehci_setup) from [<c0405138>] (tegra_ehci_reset+0xc/0x50)
+[<c0405138>] (tegra_ehci_reset) from [<c03e8be8>] (usb_add_hcd+0x2fc/0x804)
+[<c03e8be8>] (usb_add_hcd) from [<c0404ab0>] (tegra_ehci_probe+0x3b8/0x504)
+[<c0404ab0>] (tegra_ehci_probe) from [<c0336b64>] (platform_drv_probe+0x50/0xac)
+[<c0336b64>] (platform_drv_probe) from [<c03355cc>] (driver_probe_device+0x1f4/0x2b0)
+[<c03355cc>] (driver_probe_device) from [<c0333b60>] (bus_for_each_drv+0x44/0x8c)
+[<c0333b60>] (bus_for_each_drv) from [<c0335350>] (__device_attach+0x9c/0x100)
+[<c0335350>] (__device_attach) from [<c03349dc>] (bus_probe_device+0x84/0x8c)
+[<c03349dc>] (bus_probe_device) from [<c0334e10>] (deferred_probe_work_func+0x60/0x8c)
+[<c0334e10>] (deferred_probe_work_func) from [<c003a7cc>] (process_one_work+0x120/0x328)
 [<c003a7cc>] (process_one_work) from [<c003aa38>] (worker_thread+0x2c/0x4b4)
 [<c003aa38>] (worker_thread) from [<c003fbac>] (kthread+0xd8/0xf4)
 [<c003fbac>] (kthread) from [<c000f938>] (ret_from_fork+0x14/0x3c)
----[ end trace bdb44eaa50daeba0 ]---
+---[ end trace c85cdc3a01426c84 ]---
 tegra-ehci 7d004000.usb: irq 343, io mem 0x7d004000
 tegra-ehci 7d004000.usb: USB 2.0 started, EHCI 1.00
 hub 1-0:1.0: USB hub found
