Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-05-19 00:42:25.037704535 -0700
+++ /tmp/foob	2016-05-19 00:42:25.041704536 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-next-20160518+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue May 17 22:07:23 PDT 2016
+Linux version 4.6.0-next-20160519+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu May 19 00:37:04 PDT 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -19,15 +19,15 @@
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
 software IO TLB [mem 0x79c00000-0x7dc00000] (64MB) mapped at [ffff800039c00000-ffff80003dbfffff]
-Memory: 935520K/1048576K available (7436K kernel code, 644K rwdata, 3184K rodata, 960K init, 258K bss, 96672K reserved, 16384K cma-reserved)
+Memory: 935516K/1048576K available (7436K kernel code, 647K rwdata, 3184K rodata, 960K init, 259K bss, 96676K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff0000087c0000   (  7424 KB)
     .rodata : 0xffff0000087c0000 - 0xffff000008ae0000   (  3200 KB)
       .init : 0xffff000008ae0000 - 0xffff000008bd0000   (   960 KB)
-      .data : 0xffff000008bd0000 - 0xffff000008c71000   (   644 KB)
-       .bss : 0xffff000008c71000 - 0xffff000008cb1b40   (   259 KB)
+      .data : 0xffff000008bd0000 - 0xffff000008c71e00   (   648 KB)
+       .bss : 0xffff000008c71e00 - 0xffff000008cb2b40   (   260 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -98,7 +98,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.608:1): initialized
+audit: type=2000 audit(0.580:1): initialized
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -159,7 +159,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2016-05-18 05:07:58 UTC (1463548078)
+rtc-pl031 9010000.pl031: setting system clock to 2016-05-19 07:37:37 UTC (1463643457)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-05-19 00:42:25.053704538 -0700
+++ /tmp/foob	2016-05-19 00:42:25.057704539 -0700
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra114 (Dalmore) # 
 Tegra114 (Dalmore) # usb reset
 (Re)start USB...
@@ -27,7 +27,6 @@
 Tegra114 (Dalmore) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=7369c667-ff51-ec4a-29cd-baabf2fbe346 rootwait gpt debug earlyprintk init=/bin/bash'
 Tegra114 (Dalmore) # setenv ipaddr 192.168.100.100; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra114 (Dalmore) # tftpboot 0x81000000 zImage.multi_v7_defconfig
-Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'zImage.multi_v7_defconfig'.
@@ -42,18 +41,18 @@
 	 ############################
 	 3.1 MiB/s
 done
-Bytes transferred = 7079112 (6c04c8 hex)
+Bytes transferred = 7077600 (6bfee0 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 34956 (888c hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6c04c8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6bfee0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff88b ... OK
@@ -61,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-next-20160518-94993-g621a2a0 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue May 17 22:04:51 PDT 2016
+Linux version 4.6.0-next-20160519-95140-g3c60f7e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu May 19 00:34:41 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -70,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141bb40, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c141bac0, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -91,9 +90,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc106c538   (14738 kB)
+      .text : 0xc0208000 - 0xc106c520   (14738 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141dba0   (1143 kB)
+      .data : 0xc1300000 - 0xc141db20   (1143 kB)
        .bss : 0xc141f000 - 0xc1474ca0   ( 344 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -262,25 +261,25 @@
 ------------[ cut here ]------------
 WARNING: CPU: 1 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
 Modules linked in:
-CPU: 1 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160518-94993-g621a2a0 #1
+CPU: 1 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160519-95140-g3c60f7e #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c030f9d0>] (unwind_backtrace) from [<c030b0c8>] (show_stack+0x10/0x14)
-[<c030b0c8>] (show_stack) from [<c0585fc8>] (dump_stack+0x84/0xa4)
-[<c0585fc8>] (dump_stack) from [<c0341b6c>] (__warn+0xd4/0x100)
+[<c030b0c8>] (show_stack) from [<c05863a8>] (dump_stack+0x84/0xa4)
+[<c05863a8>] (dump_stack) from [<c0341b6c>] (__warn+0xd4/0x100)
 [<c0341b6c>] (__warn) from [<c0341bb8>] (warn_slowpath_null+0x20/0x28)
-[<c0341bb8>] (warn_slowpath_null) from [<c06a0494>] (__of_reset_control_get+0x1bc/0x228)
-[<c06a0494>] (__of_reset_control_get) from [<c0911eb8>] (tegra_ehci_probe+0x2d0/0x51c)
-[<c0911eb8>] (tegra_ehci_probe) from [<c0778f9c>] (platform_drv_probe+0x50/0xb0)
-[<c0778f9c>] (platform_drv_probe) from [<c0777900>] (driver_probe_device+0x1f4/0x2b0)
-[<c0777900>] (driver_probe_device) from [<c0777a68>] (__driver_attach+0xac/0xb0)
-[<c0777a68>] (__driver_attach) from [<c0775f14>] (bus_for_each_dev+0x54/0x88)
-[<c0775f14>] (bus_for_each_dev) from [<c0776ef4>] (bus_add_driver+0xe8/0x1f4)
-[<c0776ef4>] (bus_add_driver) from [<c0778284>] (driver_register+0x78/0xf4)
-[<c0778284>] (driver_register) from [<c0301e7c>] (do_one_initcall+0x3c/0x16c)
+[<c0341bb8>] (warn_slowpath_null) from [<c06a0874>] (__of_reset_control_get+0x1bc/0x228)
+[<c06a0874>] (__of_reset_control_get) from [<c091235c>] (tegra_ehci_probe+0x2d0/0x51c)
+[<c091235c>] (tegra_ehci_probe) from [<c07793a4>] (platform_drv_probe+0x50/0xb0)
+[<c07793a4>] (platform_drv_probe) from [<c0777d08>] (driver_probe_device+0x1f4/0x2b0)
+[<c0777d08>] (driver_probe_device) from [<c0777e70>] (__driver_attach+0xac/0xb0)
+[<c0777e70>] (__driver_attach) from [<c077631c>] (bus_for_each_dev+0x54/0x88)
+[<c077631c>] (bus_for_each_dev) from [<c07772fc>] (bus_add_driver+0xe8/0x1f4)
+[<c07772fc>] (bus_add_driver) from [<c077868c>] (driver_register+0x78/0xf4)
+[<c077868c>] (driver_register) from [<c0301e7c>] (do_one_initcall+0x3c/0x16c)
 [<c0301e7c>] (do_one_initcall) from [<c1100d80>] (kernel_init_freeable+0x120/0x1ec)
-[<c1100d80>] (kernel_init_freeable) from [<c0b5bc84>] (kernel_init+0x8/0x118)
-[<c0b5bc84>] (kernel_init) from [<c0307fb8>] (ret_from_fork+0x14/0x3c)
----[ end trace d5d3855d7da811e1 ]---
+[<c1100d80>] (kernel_init_freeable) from [<c0b5b7fc>] (kernel_init+0x8/0x118)
+[<c0b5b7fc>] (kernel_init) from [<c0307fb8>] (ret_from_fork+0x14/0x3c)
+---[ end trace 5d734b73e1b4c758 ]---
 tegra-ehci 7d008000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d008000.usb: continuing, but please update your DT
 ohci_hcd: USB 1.1 'Open' Host Controller (OHCI) Driver
@@ -334,19 +333,19 @@
 mmcblk0rpmb: mmc0:0001 SEM32G partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 Console: switching to colour frame buffer device 240x75
-drm drm: fb0:  frame buffer device
 mmc1: new high speed SDHC card at address aaaa
 mmcblk1: mmc1:aaaa SL16G 14.8 GiB 
+ mmcblk1: p1 p2 p3
+drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 tegra-ehci 7d008000.usb: EHCI Host Controller
 tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 1
- mmcblk1: p1 p2 p3
 tegra-ehci 7d008000.usb: irq 22, io mem 0x7d008000
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-03-11 16:24:23 UTC (952791863)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-03-12 18:54:19 UTC (952887259)
 vdd_cam_1v8_reg: disabling
 usb1_vbus: disabling
 dvdd_ts: disabling
@@ -358,4 +357,4 @@
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.527684] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.547697] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-05-19 00:42:25.069704540 -0700
+++ /tmp/foob	2016-05-19 00:42:25.073704541 -0700
@@ -37,9 +37,9 @@
 	 #################################################################
 	 #################################################################
 	 ##########################################
-	 3.1 MiB/s
+	 2.9 MiB/s
 done
-Bytes transferred = 5377160 (520c88 hex)
+Bytes transferred = 5377384 (520d68 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 34956 (888c hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x520c88 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x520d68 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff88b ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-next-20160518-94993-g621a2a0 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue May 17 22:04:51 PDT 2016
+Linux version 4.6.0-next-20160519-95140-g3c60f7e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu May 19 00:34:38 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -82,7 +82,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2000976K/2096128K available (7415K kernel code, 374K rwdata, 2560K rodata, 1024K init, 366K bss, 29616K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2000976K/2096128K available (7416K kernel code, 374K rwdata, 2560K rodata, 1024K init, 366K bss, 29616K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -90,7 +90,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0abdff4   (10968 kB)
+      .text : 0xc0008000 - 0xc0abe028   (10969 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
       .data : 0xc0c00000 - 0xc0c5d8c0   ( 375 kB)
        .bss : 0xc0c5f000 - 0xc0cbab18   ( 367 kB)
@@ -251,25 +251,25 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1b4/0x218
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160518-94993-g621a2a0 #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160519-95140-g3c60f7e #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c010f1e8>] (unwind_backtrace) from [<c010a864>] (show_stack+0x10/0x14)
-[<c010a864>] (show_stack) from [<c033d8d4>] (dump_stack+0x84/0xa4)
-[<c033d8d4>] (dump_stack) from [<c011f308>] (__warn+0xd4/0x100)
+[<c010a864>] (show_stack) from [<c033dc54>] (dump_stack+0x84/0xa4)
+[<c033dc54>] (dump_stack) from [<c011f308>] (__warn+0xd4/0x100)
 [<c011f308>] (__warn) from [<c011f354>] (warn_slowpath_null+0x20/0x28)
-[<c011f354>] (warn_slowpath_null) from [<c03c0e1c>] (__of_reset_control_get+0x1b4/0x218)
-[<c03c0e1c>] (__of_reset_control_get) from [<c0513a94>] (tegra_ehci_probe+0x2c8/0x514)
-[<c0513a94>] (tegra_ehci_probe) from [<c04411b4>] (platform_drv_probe+0x50/0xb0)
-[<c04411b4>] (platform_drv_probe) from [<c043fbe0>] (driver_probe_device+0x1f4/0x2b0)
-[<c043fbe0>] (driver_probe_device) from [<c043fd48>] (__driver_attach+0xac/0xb0)
-[<c043fd48>] (__driver_attach) from [<c043e204>] (bus_for_each_dev+0x54/0x88)
-[<c043e204>] (bus_for_each_dev) from [<c043f1d4>] (bus_add_driver+0xe8/0x1f4)
-[<c043f1d4>] (bus_add_driver) from [<c0440544>] (driver_register+0x78/0xf4)
-[<c0440544>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
+[<c011f354>] (warn_slowpath_null) from [<c03c119c>] (__of_reset_control_get+0x1b4/0x218)
+[<c03c119c>] (__of_reset_control_get) from [<c0513dcc>] (tegra_ehci_probe+0x2c8/0x514)
+[<c0513dcc>] (tegra_ehci_probe) from [<c044155c>] (platform_drv_probe+0x50/0xb0)
+[<c044155c>] (platform_drv_probe) from [<c043ff88>] (driver_probe_device+0x1f4/0x2b0)
+[<c043ff88>] (driver_probe_device) from [<c04400f0>] (__driver_attach+0xac/0xb0)
+[<c04400f0>] (__driver_attach) from [<c043e5ac>] (bus_for_each_dev+0x54/0x88)
+[<c043e5ac>] (bus_for_each_dev) from [<c043f57c>] (bus_add_driver+0xe8/0x1f4)
+[<c043f57c>] (bus_add_driver) from [<c04408ec>] (driver_register+0x78/0xf4)
+[<c04408ec>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
 [<c010179c>] (do_one_initcall) from [<c0b00d94>] (kernel_init_freeable+0x11c/0x1e8)
-[<c0b00d94>] (kernel_init_freeable) from [<c07f107c>] (kernel_init+0x8/0x118)
-[<c07f107c>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
----[ end trace bfe7e25800ceda30 ]---
+[<c0b00d94>] (kernel_init_freeable) from [<c07f14f4>] (kernel_init+0x8/0x118)
+[<c07f14f4>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
+---[ end trace ada2752f271b16c4 ]---
 tegra-ehci 7d008000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d008000.usb: continuing, but please update your DT
 usbcore: registered new interface driver cdc_acm
@@ -343,7 +343,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-03-11 16:25:52 UTC (952791952)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-03-12 18:56:58 UTC (952887418)
 vdd_cam_1v8_reg: disabling
 usb1_vbus: disabling
 dvdd_ts: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-05-19 00:42:25.101704546 -0700
+++ /tmp/foob	2016-05-19 00:42:25.101704546 -0700
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+MMC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -36,7 +36,7 @@
 	 ############################
 	 4 MiB/s
 done
-Bytes transferred = 7079112 (6c04c8 hex)
+Bytes transferred = 7077600 (6bfee0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -47,7 +47,7 @@
 done
 Bytes transferred = 66737 (104b1 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6c04c8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x6bfee0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830134b0
@@ -55,7 +55,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-next-20160518-94993-g621a2a0 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue May 17 22:04:51 PDT 2016
+Linux version 4.6.0-next-20160519-95140-g3c60f7e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu May 19 00:34:41 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -64,7 +64,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141bb40, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c141bac0, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -85,9 +85,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc106c538   (14738 kB)
+      .text : 0xc0208000 - 0xc106c520   (14738 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141dba0   (1143 kB)
+      .data : 0xc1300000 - 0xc141db20   (1143 kB)
        .bss : 0xc141f000 - 0xc1474ca0   ( 344 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -253,25 +253,25 @@
 ------------[ cut here ]------------
 WARNING: CPU: 1 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
 Modules linked in:
-CPU: 1 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160518-94993-g621a2a0 #1
+CPU: 1 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160519-95140-g3c60f7e #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c030f9d0>] (unwind_backtrace) from [<c030b0c8>] (show_stack+0x10/0x14)
-[<c030b0c8>] (show_stack) from [<c0585fc8>] (dump_stack+0x84/0xa4)
-[<c0585fc8>] (dump_stack) from [<c0341b6c>] (__warn+0xd4/0x100)
+[<c030b0c8>] (show_stack) from [<c05863a8>] (dump_stack+0x84/0xa4)
+[<c05863a8>] (dump_stack) from [<c0341b6c>] (__warn+0xd4/0x100)
 [<c0341b6c>] (__warn) from [<c0341bb8>] (warn_slowpath_null+0x20/0x28)
-[<c0341bb8>] (warn_slowpath_null) from [<c06a0494>] (__of_reset_control_get+0x1bc/0x228)
-[<c06a0494>] (__of_reset_control_get) from [<c0911eb8>] (tegra_ehci_probe+0x2d0/0x51c)
-[<c0911eb8>] (tegra_ehci_probe) from [<c0778f9c>] (platform_drv_probe+0x50/0xb0)
-[<c0778f9c>] (platform_drv_probe) from [<c0777900>] (driver_probe_device+0x1f4/0x2b0)
-[<c0777900>] (driver_probe_device) from [<c0777a68>] (__driver_attach+0xac/0xb0)
-[<c0777a68>] (__driver_attach) from [<c0775f14>] (bus_for_each_dev+0x54/0x88)
-[<c0775f14>] (bus_for_each_dev) from [<c0776ef4>] (bus_add_driver+0xe8/0x1f4)
-[<c0776ef4>] (bus_add_driver) from [<c0778284>] (driver_register+0x78/0xf4)
-[<c0778284>] (driver_register) from [<c0301e7c>] (do_one_initcall+0x3c/0x16c)
+[<c0341bb8>] (warn_slowpath_null) from [<c06a0874>] (__of_reset_control_get+0x1bc/0x228)
+[<c06a0874>] (__of_reset_control_get) from [<c091235c>] (tegra_ehci_probe+0x2d0/0x51c)
+[<c091235c>] (tegra_ehci_probe) from [<c07793a4>] (platform_drv_probe+0x50/0xb0)
+[<c07793a4>] (platform_drv_probe) from [<c0777d08>] (driver_probe_device+0x1f4/0x2b0)
+[<c0777d08>] (driver_probe_device) from [<c0777e70>] (__driver_attach+0xac/0xb0)
+[<c0777e70>] (__driver_attach) from [<c077631c>] (bus_for_each_dev+0x54/0x88)
+[<c077631c>] (bus_for_each_dev) from [<c07772fc>] (bus_add_driver+0xe8/0x1f4)
+[<c07772fc>] (bus_add_driver) from [<c077868c>] (driver_register+0x78/0xf4)
+[<c077868c>] (driver_register) from [<c0301e7c>] (do_one_initcall+0x3c/0x16c)
 [<c0301e7c>] (do_one_initcall) from [<c1100d80>] (kernel_init_freeable+0x120/0x1ec)
-[<c1100d80>] (kernel_init_freeable) from [<c0b5bc84>] (kernel_init+0x8/0x118)
-[<c0b5bc84>] (kernel_init) from [<c0307fb8>] (ret_from_fork+0x14/0x3c)
----[ end trace e9fa34d629cc0a6c ]---
+[<c1100d80>] (kernel_init_freeable) from [<c0b5b7fc>] (kernel_init+0x8/0x118)
+[<c0b5b7fc>] (kernel_init) from [<c0307fb8>] (ret_from_fork+0x14/0x3c)
+---[ end trace 39f65bbfb1f5d33a ]---
 tegra-ehci 7d008000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d008000.usb: continuing, but please update your DT
 tegra-ehci 7d004000.usb: EHCI Host Controller
@@ -330,13 +330,9 @@
 +3.3V_AVDD_HDMI_AP_GATED: supplied by +3.3V_RUN
 usb2-0 supply vbus not found, using dummy regulator
 mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
- mmcblk0: p1
 usb2-1 supply vbus not found, using dummy regulator
 usb3-0 supply vbus not found, using dummy regulator
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
 tegra-pcie 1003000.pcie-controller: Failed to get supply 'avddio-pex': -517
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
@@ -400,7 +396,6 @@
 PCI: bus1: Fast back to back transfers disabled
 pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
 pci 0000:02:00.0: [10ec:8168] type 00 class 0x020000
-ata1: SATA link down (SStatus 0 SControl 300)
 pci 0000:02:00.0: reg 0x10: [io  0x0000-0x00ff]
 pci 0000:02:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit]
 pci 0000:02:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
@@ -434,15 +429,21 @@
 pcie_pme 0000:00:02.0:pcie001: service driver pcie_pme loaded
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
-r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0a52000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 393
+r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0a48000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 393
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
+drm drm: No connectors reported connected with modes
+[drm] Cannot find any crtc or sizes - going 1024x768
 mmc1: new high speed SDHC card at address aaaa
 mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
  mmcblk1: p1 p2 p3
-tegra-hdmi 54280000.hdmi: cannot set audio to 44100 at 297000000 pclk
-Console: switching to colour frame buffer device 512x135
+ata1: SATA link down (SStatus 0 SControl 300)
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+Console: switching to colour frame buffer device 128x48
+ mmcblk0: p1
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 of: dev_pm_opp_of_cpumask_add_table: couldn't find opp table for cpu:0, -19
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-05-19 00:42:25.109704547 -0700
+++ /tmp/foob	2016-05-19 00:42:25.109704547 -0700
@@ -32,20 +32,20 @@
 	 #################################################################
 	 #################################################################
 	 ##########################################
-	 3.9 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 5377160 (520c88 hex)
+Bytes transferred = 5377384 (520d68 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 4 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 66737 (104b1 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x520c88 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x520d68 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830134b0
@@ -54,7 +54,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-next-20160518-94993-g621a2a0 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue May 17 22:04:51 PDT 2016
+Linux version 4.6.0-next-20160519-95140-g3c60f7e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu May 19 00:34:38 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -77,7 +77,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2000856K/2096128K available (7415K kernel code, 374K rwdata, 2560K rodata, 1024K init, 366K bss, 29736K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2000856K/2096128K available (7416K kernel code, 374K rwdata, 2560K rodata, 1024K init, 366K bss, 29736K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -85,7 +85,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0abdff4   (10968 kB)
+      .text : 0xc0008000 - 0xc0abe028   (10969 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
       .data : 0xc0c00000 - 0xc0c5d8c0   ( 375 kB)
        .bss : 0xc0c5f000 - 0xc0cbab18   ( 367 kB)
@@ -243,25 +243,25 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1b4/0x218
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160518-94993-g621a2a0 #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160519-95140-g3c60f7e #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c010f1e8>] (unwind_backtrace) from [<c010a864>] (show_stack+0x10/0x14)
-[<c010a864>] (show_stack) from [<c033d8d4>] (dump_stack+0x84/0xa4)
-[<c033d8d4>] (dump_stack) from [<c011f308>] (__warn+0xd4/0x100)
+[<c010a864>] (show_stack) from [<c033dc54>] (dump_stack+0x84/0xa4)
+[<c033dc54>] (dump_stack) from [<c011f308>] (__warn+0xd4/0x100)
 [<c011f308>] (__warn) from [<c011f354>] (warn_slowpath_null+0x20/0x28)
-[<c011f354>] (warn_slowpath_null) from [<c03c0e1c>] (__of_reset_control_get+0x1b4/0x218)
-[<c03c0e1c>] (__of_reset_control_get) from [<c0513a94>] (tegra_ehci_probe+0x2c8/0x514)
-[<c0513a94>] (tegra_ehci_probe) from [<c04411b4>] (platform_drv_probe+0x50/0xb0)
-[<c04411b4>] (platform_drv_probe) from [<c043fbe0>] (driver_probe_device+0x1f4/0x2b0)
-[<c043fbe0>] (driver_probe_device) from [<c043fd48>] (__driver_attach+0xac/0xb0)
-[<c043fd48>] (__driver_attach) from [<c043e204>] (bus_for_each_dev+0x54/0x88)
-[<c043e204>] (bus_for_each_dev) from [<c043f1d4>] (bus_add_driver+0xe8/0x1f4)
-[<c043f1d4>] (bus_add_driver) from [<c0440544>] (driver_register+0x78/0xf4)
-[<c0440544>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
+[<c011f354>] (warn_slowpath_null) from [<c03c119c>] (__of_reset_control_get+0x1b4/0x218)
+[<c03c119c>] (__of_reset_control_get) from [<c0513dcc>] (tegra_ehci_probe+0x2c8/0x514)
+[<c0513dcc>] (tegra_ehci_probe) from [<c044155c>] (platform_drv_probe+0x50/0xb0)
+[<c044155c>] (platform_drv_probe) from [<c043ff88>] (driver_probe_device+0x1f4/0x2b0)
+[<c043ff88>] (driver_probe_device) from [<c04400f0>] (__driver_attach+0xac/0xb0)
+[<c04400f0>] (__driver_attach) from [<c043e5ac>] (bus_for_each_dev+0x54/0x88)
+[<c043e5ac>] (bus_for_each_dev) from [<c043f57c>] (bus_add_driver+0xe8/0x1f4)
+[<c043f57c>] (bus_add_driver) from [<c04408ec>] (driver_register+0x78/0xf4)
+[<c04408ec>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
 [<c010179c>] (do_one_initcall) from [<c0b00d94>] (kernel_init_freeable+0x11c/0x1e8)
-[<c0b00d94>] (kernel_init_freeable) from [<c07f107c>] (kernel_init+0x8/0x118)
-[<c07f107c>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
----[ end trace 0e2fb8489f093cb1 ]---
+[<c0b00d94>] (kernel_init_freeable) from [<c07f14f4>] (kernel_init+0x8/0x118)
+[<c07f14f4>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
+---[ end trace 1556d40b22ca6b46 ]---
 tegra-ehci 7d008000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d008000.usb: continuing, but please update your DT
 tegra-ehci 7d004000.usb: EHCI Host Controller
@@ -399,8 +399,8 @@
 pci 0000:00:02.0: [10de:0e13] type 01 class 0x060400
 pci 0000:00:02.0: PME# supported from D0 D1 D2 D3hot D3cold
 PCI: bus0: Fast back to back transfers disabled
-pci 0000:00:01.0: bridge configuration invalid ([bus 00-00]), reconfiguring
 ata1: SATA link down (SStatus 0 SControl 300)
+pci 0000:00:01.0: bridge configuration invalid ([bus 00-00]), reconfiguring
 pci 0000:00:02.0: bridge configuration invalid ([bus 00-00]), reconfiguring
 pci 0000:01:00.0: [8086:4232] type 00 class 0x028000
 pci 0000:01:00.0: reg 0x10: [mem 0x00000000-0x00001fff 64bit]
@@ -414,6 +414,7 @@
 pci 0000:02:00.0: supports D1 D2
 pci 0000:02:00.0: PME# supported from D0 D1 D2 D3hot D3cold
 mmc1: new high speed SDHC card at address aaaa
+mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 PCI: bus2: Fast back to back transfers disabled
 pci_bus 0000:02: busn_res: [bus 02-ff] end is updated to 02
 pci 0000:00:01.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
@@ -423,7 +424,6 @@
 pci 0000:01:00.0: BAR 0: assigned [mem 0x13000000-0x13001fff 64bit]
 pci 0000:00:01.0: PCI bridge to [bus 01]
 pci 0000:00:01.0:   bridge window [mem 0x13000000-0x130fffff]
-mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 pci 0000:02:00.0: BAR 4: assigned [mem 0x20000000-0x20003fff 64bit pref]
 pci 0000:02:00.0: BAR 2: assigned [mem 0x13100000-0x13100fff 64bit]
 pci 0000:02:00.0: BAR 0: assigned [io  0x1000-0x10ff]
@@ -460,7 +460,7 @@
 cpu cpu2: cpufreq_init: failed to get clk: -2
 cpu cpu3: cpufreq_init: failed to get clk: -2
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:07 UTC (946684927)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:06 UTC (946684926)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	1969-12-31 16:00:00.000000000 -0800
+++ /tmp/foob	2016-05-19 00:42:25.117704548 -0700
@@ -0,0 +1,292 @@
+Rebooting device after flashing.
+TegraBoot Init
+Processing in cold boot mode
+NVRM Initialized shmoo database
+NVRM CLOCKS: PLLX0:      0 Khz
+NVRM CLOCKS: PLLM0:      792000 Khz
+NVRM CLOCKS: PLLC0:      0 Khz
+NVRM CLOCKS: PLLP0:      408000 Khz
+NVRM CLOCKS: PLLA0:      11289 Khz
+NVRM CLOCKS: CPU:        12000 Khz
+NVRM CLOCKS: AVP:        48000 Khz
+NVRM CLOCKS: System Bus: 48000 Khz
+NVRM CLOCKS: Memory Controller: 792000
+NVRM CLOCKS: External Memory Controller: 792000
+0x00 BoardInfo: 0x0001:0x0007:0x065c:0x03e8:0x00:0x42:0x00:0xff:0xff:0xff:0xff:0xff:0xff
+ 0x01 BoardInfo: 0x0001:0x0007:0x0162:0x0001:0x00:0x42:0x00:0x00:0x00:0x00:0x00:0x00:0x00
+ 0x02 BoardInfo: 0x0000:0x0000:0x0000:0x0000:0x00:0x00:0x00:0x00:0x00:0x00:0x00:0x00:0x00
+ 0x03 BoardInfo: 0x0000:0x0000:0x0000:0x0000:0x00:0x00:0x00:0x00:0x00:0x00:0x00:0x00:0x00
+ 0x04 BoardInfo: 0x0001:0x0007:0x0713:0x03e8:0x00:0x42:0x00:0xff:0xff:0xff:0xff:0xff:0xff
+ 0x05 BoardInfo: 0x0000:0x0000:0x0000:0x0000:0x00:0x00:0x00:0x00:0x00:0x00:0x00:0x00:0x00
+ 0x06 BoardInfo: 0x0001:0x0007:0x0176:0x0001:0x01:0x43:0x00:0xff:0xff:0xff:0xff:0xff:0xff
+ 0x07 BoardInfo: 0x0001:0x0007:0x0156:0x0000:0x00:0x46:0x05:0xff:0xff:0xff:0xff:0xff:0xff
+ ADJUSTED CLOCKS:
+MC clock is set to 792000 KHz
+EMC clock is set to 792000 KHz (DDR clock is at 396000 KHz)
+PLLX0 clock is set to      0 KHz
+PLLC0 clock is set to      0 KHz
+CPU clock is set to  12000 KHz
+System and AVP clock is set to  48000 KHz
+GraphicsHost clock is set to 163200 KHz
+MSENC clock is set to  79200 KHz
+Vde clock is set to 204000 KHz
+
+Bootloader-Cpu Init at (time stamp): 1801201 us
+
+Pinmux changes applied in kernel way
+
+(version UNDEF_BUILD)
+Platform Pre Boot configuration...
+The proc BoardInfo: 0x0176:0x0001:0x01:0x43:0x00
+Initializing Display
+This Pmu Module is not present.
+
+(version UNDEF_BUILD)
+
+Device - Unlocked 
+
+
+
+Serial Number  17418A202140000000A078300 
+
+Checking for android ota recovery 
+[NvPartMgrGetIdByName]:Could not find a match for given partition name[NvPartMgrGetIdByName]:Could not find a match for given partition name(FastbootBMPRead) Unsuccesful attempt to read BMP image
+Could not render BMP on screen
+Cold-booting Linux
+Platform Pre OS Boot configuration...
+Setting VddId=2 to 1150 mV
+FastbootAddAtag: setting ATAG (1413545985)
+The proc BoardInfo: 0x0176:0x0001:0x01:0x43:0x00
+FastbootAddAtag: setting ATAG (0)
+
+Board Id Read Time: 3 us
+Power Rail enable Time: 1 us
+
+Miscellaneous avp time: 1801175  us
+Bootloader (AosInit) time: 733468 us
+Bootloader Main Init time: 180276 us
+BlockDevMgrInit time: 157563 us
+PartMgrInit time: 8 us
+StorMgrInit time: 31 us
+FsMgrInit time: 2 us
+Partition Table load: 77109 us 
+Display init time: 574359 us
+Kernel read time: 344 msec
+Load Kernel Size:   14952448 Bytes
+Bl Kernel ReadSpeed: 43466000 Bytes/sec
+Pre-Kernel Jump time: 1632822 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5694922 us
+Booting Linux on physical CPU 0x0
+Linux version 4.6.0-next-20160519+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu May 19 00:37:04 PDT 2016
+Boot CPU: AArch64 Processor [4e0f0000]
+Memory limited to 1734MB
+efi: Getting EFI parameters from FDT:
+efi: UEFI not found.
+cma: Reserved 16 MiB at 0x00000000eb400000
+/cpus/cpu@1: missing enable-method property
+percpu: Embedded 20 pages/cpu @ffff80006c59f000 s42752 r8192 d30976 u81920
+Detected PIPT I-cache on CPU0
+Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 436968
+Kernel command line: tegraid=13.1.3.0.0 mem=1734M@2048M memtype=255 vpr=151M@3817M ddr_die=2048M@2048M section=256M commchip_id=0 usb_port_owner_info=0 lane_owner_info=0 emc_max_dvfs=0 touch_id=0@0 video=tegrafb no_console_suspend=1 console=ttyS0,115200n8 debug_uartport=lsport,0 earlyprintk=uart8250-32bit,0x70006000 usbcore.old_scheme_first=1 lp0_vec=0@0x0 tegra_fbmem=32899072@0xad012000 core_edp_mv=1150 core_edp_ma=4000 pmuboard=0x0176:0x0001:0x01:0x43:0x00 displayboard=0x0162:0x0001:0x00:0x42:0x00 power_supply=Adapter audio_codec=rt5640 board_info=0x0176:0x0001:0x01:0x43:0x00 tegraboot=sdmmc gpt gpt_sector=94207 modem_id=0 android.kerneltype=normal
+PID hash table entries: 4096 (order: 3, 32768 bytes)
+Dentry cache hash table entries: 262144 (order: 9, 2097152 bytes)
+Inode-cache hash table entries: 131072 (order: 8, 1048576 bytes)
+software IO TLB [mem 0xe5200000-0xe9200000] (64MB) mapped at [ffff800065200000-ffff8000691fffff]
+Memory: 1642108K/1775616K available (7436K kernel code, 647K rwdata, 3184K rodata, 960K init, 259K bss, 117124K reserved, 16384K cma-reserved)
+Virtual kernel memory layout:
+    modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
+    vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
+      .text : 0xffff000008080000 - 0xffff0000087c0000   (  7424 KB)
+    .rodata : 0xffff0000087c0000 - 0xffff000008ae0000   (  3200 KB)
+      .init : 0xffff000008ae0000 - 0xffff000008bd0000   (   960 KB)
+      .data : 0xffff000008bd0000 - 0xffff000008c71e00   (   648 KB)
+       .bss : 0xffff000008c71e00 - 0xffff000008cb2b40   (   260 KB)
+    fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
+    PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
+    vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
+              0xffff7e0000000000 - 0xffff7e0001b18000   (    27 MB actual)
+    memory  : 0xffff800000000000 - 0xffff80006c600000   (  1734 MB)
+SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
+Preemptible hierarchical RCU implementation.
+	Build-time adjustment of leaf fanout to 64.
+	RCU restricting CPUs from NR_CPUS=64 to nr_cpu_ids=1.
+RCU: Adjusting geometry for rcu_fanout_leaf=64, nr_cpu_ids=1
+NR_IRQS:64 nr_irqs:64 0
+/interrupt-controller@60004000: 160 interrupts forwarded to /interrupt-controller@50041000
+Tegra clk 127: register failed with -17
+Architected cp15 timer(s) running at 12.00MHz (virt).
+clocksource: arch_sys_counter: mask: 0xffffffffffffff max_cycles: 0x2c47f4ee7, max_idle_ns: 440795202497 ns
+sched_clock: 56 bits at 12MHz, resolution 83ns, wraps every 4398046511096ns
+Console: colour dummy device 80x25
+Calibrating delay loop (skipped), value calculated using timer frequency.. 24.00 BogoMIPS (lpj=48000)
+pid_max: default: 32768 minimum: 301
+Security Framework initialized
+Mount-cache hash table entries: 4096 (order: 3, 32768 bytes)
+Mountpoint-cache hash table entries: 4096 (order: 3, 32768 bytes)
+ASID allocator initialised with 256 entries
+EFI services will not be available.
+Brought up 1 CPUs
+SMP: Total of 1 processors activated.
+CPU features: detected feature: 32-bit EL0 Support
+CPU: All CPU(s) started at EL1
+devtmpfs: initialized
+DMI not present or invalid.
+clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 7645041785100000 ns
+pinctrl core: initialized pinctrl subsystem
+NET: Registered protocol family 16
+cpuidle: using governor menu
+vdso: 2 pages (1 code @ ffff0000087c6000, 1 data @ ffff000008bd4000)
+hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
+DMA: preallocated 256 KiB pool for atomic allocations
+Serial: AMBA PL011 UART driver
+HugeTLB registered 2 MB page size, pre-allocated 0 pages
+ACPI: Interpreter disabled.
++5V_VDD_TS: Failed to request enable GPIO81: -517
+reg-fixed-voltage regulators:regulator@11: Failed to register regulator: -517
++5V_HDMI_CON: Failed to request enable GPIO86: -517
+reg-fixed-voltage regulators:regulator@10: Failed to register regulator: -517
++1.05V_RUN_AVDD_HDMI_PLL_AP_GATE: Failed to request enable GPIO63: -517
+reg-fixed-voltage regulators:regulator@9: Failed to register regulator: -517
++5V_USB_SS: Failed to request enable GPIO109: -517
+reg-fixed-voltage regulators:regulator@7: Failed to register regulator: -517
++5V_USB_HS: Failed to request enable GPIO108: -517
+reg-fixed-voltage regulators:regulator@6: Failed to register regulator: -517
++VDD_LED: Failed to request enable GPIO122: -517
+reg-fixed-voltage regulators:regulator@5: Failed to register regulator: -517
++3.3V_SYS: supplied by +VDD_MUX
++5V_SYS: supplied by +VDD_MUX
+vgaarb: loaded
+SCSI subsystem initialized
+usbcore: registered new interface driver usbfs
+usbcore: registered new interface driver hub
+usbcore: registered new device driver usb
+pps_core: LinuxPPS API ver. 1 registered
+pps_core: Software ver. 5.3.6 - Copyright 2005-2007 Rodolfo Giometti <giometti@linux.it>
+PTP clock support registered
+dmi: Firmware registration failed.
+Advanced Linux Sound Architecture Driver Initialized.
+clocksource: Switched to clocksource arch_sys_counter
+VFS: Disk quotas dquot_6.6.0
+VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
+pnp: PnP ACPI: disabled
+NET: Registered protocol family 2
+TCP established hash table entries: 16384 (order: 5, 131072 bytes)
+TCP bind hash table entries: 16384 (order: 6, 262144 bytes)
+TCP: Hash tables configured (established 16384 bind 16384)
+UDP hash table entries: 1024 (order: 3, 32768 bytes)
+UDP-Lite hash table entries: 1024 (order: 3, 32768 bytes)
+NET: Registered protocol family 1
+RPC: Registered named UNIX socket transport module.
+RPC: Registered udp transport module.
+RPC: Registered tcp transport module.
+RPC: Registered tcp NFSv4.1 backchannel transport module.
+Unpacking initramfs...
+Freeing initrd memory: 2364K (ffff800001680000 - ffff8000018cf000)
+kvm [1]: HYP mode not available
+futex hash table entries: 256 (order: 3, 32768 bytes)
+audit: initializing netlink subsys (disabled)
+audit: type=2000 audit(1.087:1): initialized
+workingset: timestamp_bits=44 max_order=19 bucket_order=0
+squashfs: version 4.0 (2009/01/31) Phillip Lougher
+NFS: Registering the id_resolver key type
+Key type id_resolver registered
+Key type id_legacy registered
+fuse init (API version 7.24)
+9p: Installing v9fs 9p2000 file system support
+io scheduler noop registered
+io scheduler cfq registered (default)
+tegra-apbdma 60020000.dma: Tegra20 APB DMA driver register 32 channels
+tegra-pmc 7000e400.pmc: i2c-thermtrip node not found, emergency thermal reset disabled.
+xenfs: not registering filesystem on non-xen platform
+Serial: 8250/16550 driver, 4 ports, IRQ sharing disabled
+console [ttyS0] disabled
+70006000.serial: ttyS0 at MMIO 0x70006000 (irq = 18, base_baud = 25500000) is a Tegra
+console [ttyS0] enabled
+SuperH (H)SCI(F) driver initialized
+msm_serial: driver initialized
+Unable to detect cache hierarchy from DT for CPU 0
+loop: module loaded
+tun: Universal TUN/TAP device driver, 1.6
+tun: (C) 1999-2004 Max Krasnyansky <maxk@qualcomm.com>
+e1000e: Intel(R) PRO/1000 Network Driver - 3.2.6-k
+e1000e: Copyright(c) 1999 - 2015 Intel Corporation.
+igb: Intel(R) Gigabit Ethernet Network Driver - version 5.3.0-k
+igb: Copyright (c) 2007-2014 Intel Corporation.
+igbvf: Intel(R) Gigabit Virtual Function Network Driver - version 2.0.2-k
+igbvf: Copyright (c) 2009 - 2012 Intel Corporation.
+sky2: driver version 1.30
+VFIO - User Level meta-driver version: 0.3
+ehci_hcd: USB 2.0 'Enhanced' Host Controller (EHCI) Driver
+ehci-pci: EHCI PCI platform driver
+ehci-platform: EHCI generic platform driver
+ehci-msm: Qualcomm On-Chip EHCI Host Controller
+ohci_hcd: USB 1.1 'Open' Host Controller (OHCI) Driver
+ohci-pci: OHCI PCI platform driver
+ohci-platform: OHCI generic platform driver
+usbcore: registered new interface driver usb-storage
+mousedev: PS/2 mouse device common for all mice
+i2c /dev entries driver
+sdhci: Secure Digital Host Controller Interface driver
+sdhci: Copyright(c) Pierre Ossman
+Synopsys Designware Multimedia Card Interface Driver
+sdhci-pltfm: SDHCI platform and OF driver helper
+mmc0: Unknown controller version (3). You may experience problems.
+mmc0: Invalid maximum block size, assuming 512 bytes
+mmc0: SDHCI controller on 700b0600.sdhci [700b0600.sdhci] using ADMA 64-bit
+sdhci-tegra 700b0400.sdhci: Got CD GPIO
+sdhci-tegra 700b0400.sdhci: Got WP GPIO
+mmc1: Unknown controller version (3). You may experience problems.
+ledtrig-cpu: registered to indicate activity on CPUs
+usbcore: registered new interface driver usbhid
+usbhid: USB HID core driver
+NET: Registered protocol family 17
+9pnet: Installing 9P2000 support
+mmc0: new high speed MMC card at address 0001
+Key type dns_resolver registered
+registered taskstats version 1
++5V_HDMI_CON: supplied by +5V_SYS
++5V_USB_SS: supplied by +5V_SYS
++5V_USB_HS: supplied by +5V_SYS
++VDD_LED: supplied by +VDD_MUX
+sdhci-tegra 700b0400.sdhci: Got CD GPIO
+sdhci-tegra 700b0400.sdhci: Got WP GPIO
+mmc1: Unknown controller version (3). You may experience problems.
+input: gpio-keys as /devices/platform/gpio-keys/input/input0
+hctosys: unable to open rtc device (rtc0)
+sdhci-tegra 700b0400.sdhci: Got CD GPIO
+sdhci-tegra 700b0400.sdhci: Got WP GPIO
+mmc1: Unknown controller version (3). You may experience problems.
++3.3V_AVDD_HDMI_AP_GATED: disabling
++5V_HDMI_CON: disabling
++1.05V_RUN_AVDD_HDMI_PLL_AP_GATE: disabling
++5V_USB_SS: disabling
++5V_USB_HS: disabling
++VDD_LED: disabling
+ALSA device list:
+  No soundcards found.
+Freeing unused kernel memory: 960K (ffff800000ae0000 - ffff800000bd0000)
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
+sdhci-tegra 700b0400.sdhci: Got CD GPIO
+sdhci-tegra 700b0400.sdhci: Got WP GPIO
+mmc1: Unknown controller version (3). You may experience problems.
+
+System initialization...
+
+Mounting /proc :[SUCCESS]
+Mounting /sys :[SUCCESS]
+Mounting /dev :[SUCCESS]
+Mounting /dev/pts :[SUCCESS]
+Mounting /sys/kernel/debug :[SUCCESS]
+Mounting other filesystems :[SUCCESS]
+Enabling hot-plug :[SUCCESS]
+Populating /dev :[SUCCESS]
+Starting syslogd :[SUCCESS]
+Starting telnetd :[SUCCESS]
+
+System initialization complete.
+
+Please press Enter to activate this console. 
\ No newline at end of file
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-05-19 00:42:25.133704551 -0700
+++ /tmp/foob	2016-05-19 00:42:25.133704551 -0700
@@ -47,20 +47,20 @@
 	 #################################################################
 	 #################################################################
 	 ############################
-	 913.1 KiB/s
+	 898.4 KiB/s
 done
-Bytes transferred = 7079112 (6c04c8 hex)
+Bytes transferred = 7077600 (6bfee0 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 898.4 KiB/s
+	 861.3 KiB/s
 done
 Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x6c04c8 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x6bfee0 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102b3
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-next-20160518-94993-g621a2a0 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue May 17 22:04:51 PDT 2016
+Linux version 4.6.0-next-20160519-95140-g3c60f7e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu May 19 00:34:41 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -76,7 +76,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c141bb40, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c141bac0, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -97,9 +97,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc106c538   (14738 kB)
+      .text : 0xc0208000 - 0xc106c520   (14738 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141dba0   (1143 kB)
+      .data : 0xc1300000 - 0xc141db20   (1143 kB)
        .bss : 0xc141f000 - 0xc1474ca0   ( 344 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
@@ -271,27 +271,27 @@
 ehci-atmel: EHCI Atmel driver
 tegra-ehci: Tegra EHCI driver
 ------------[ cut here ]------------
-WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
+WARNING: CPU: 1 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160518-94993-g621a2a0 #1
+CPU: 1 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160519-95140-g3c60f7e #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c030f9d0>] (unwind_backtrace) from [<c030b0c8>] (show_stack+0x10/0x14)
-[<c030b0c8>] (show_stack) from [<c0585fc8>] (dump_stack+0x84/0xa4)
-[<c0585fc8>] (dump_stack) from [<c0341b6c>] (__warn+0xd4/0x100)
+[<c030b0c8>] (show_stack) from [<c05863a8>] (dump_stack+0x84/0xa4)
+[<c05863a8>] (dump_stack) from [<c0341b6c>] (__warn+0xd4/0x100)
 [<c0341b6c>] (__warn) from [<c0341bb8>] (warn_slowpath_null+0x20/0x28)
-[<c0341bb8>] (warn_slowpath_null) from [<c06a0494>] (__of_reset_control_get+0x1bc/0x228)
-[<c06a0494>] (__of_reset_control_get) from [<c0911eb8>] (tegra_ehci_probe+0x2d0/0x51c)
-[<c0911eb8>] (tegra_ehci_probe) from [<c0778f9c>] (platform_drv_probe+0x50/0xb0)
-[<c0778f9c>] (platform_drv_probe) from [<c0777900>] (driver_probe_device+0x1f4/0x2b0)
-[<c0777900>] (driver_probe_device) from [<c0777a68>] (__driver_attach+0xac/0xb0)
-[<c0777a68>] (__driver_attach) from [<c0775f14>] (bus_for_each_dev+0x54/0x88)
-[<c0775f14>] (bus_for_each_dev) from [<c0776ef4>] (bus_add_driver+0xe8/0x1f4)
-[<c0776ef4>] (bus_add_driver) from [<c0778284>] (driver_register+0x78/0xf4)
-[<c0778284>] (driver_register) from [<c0301e7c>] (do_one_initcall+0x3c/0x16c)
+[<c0341bb8>] (warn_slowpath_null) from [<c06a0874>] (__of_reset_control_get+0x1bc/0x228)
+[<c06a0874>] (__of_reset_control_get) from [<c091235c>] (tegra_ehci_probe+0x2d0/0x51c)
+[<c091235c>] (tegra_ehci_probe) from [<c07793a4>] (platform_drv_probe+0x50/0xb0)
+[<c07793a4>] (platform_drv_probe) from [<c0777d08>] (driver_probe_device+0x1f4/0x2b0)
+[<c0777d08>] (driver_probe_device) from [<c0777e70>] (__driver_attach+0xac/0xb0)
+[<c0777e70>] (__driver_attach) from [<c077631c>] (bus_for_each_dev+0x54/0x88)
+[<c077631c>] (bus_for_each_dev) from [<c07772fc>] (bus_add_driver+0xe8/0x1f4)
+[<c07772fc>] (bus_add_driver) from [<c077868c>] (driver_register+0x78/0xf4)
+[<c077868c>] (driver_register) from [<c0301e7c>] (do_one_initcall+0x3c/0x16c)
 [<c0301e7c>] (do_one_initcall) from [<c1100d80>] (kernel_init_freeable+0x120/0x1ec)
-[<c1100d80>] (kernel_init_freeable) from [<c0b5bc84>] (kernel_init+0x8/0x118)
-[<c0b5bc84>] (kernel_init) from [<c0307fb8>] (ret_from_fork+0x14/0x3c)
----[ end trace 4cd109e5fddbd80c ]---
+[<c1100d80>] (kernel_init_freeable) from [<c0b5b7fc>] (kernel_init+0x8/0x118)
+[<c0b5b7fc>] (kernel_init) from [<c0307fb8>] (ret_from_fork+0x14/0x3c)
+---[ end trace 799125ef333756b7 ]---
 tegra-ehci c5008000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci c5008000.usb: continuing, but please update your DT
 tegra-ehci c5008000.usb: EHCI Host Controller
@@ -340,24 +340,24 @@
 Key type dns_resolver registered
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 tegra-ehci c5000000.usb: irq 24, io mem 0xc5000000
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
 rtc-em3027 0-0056: hctosys: unable to read the hardware clock
-Waiting for root device PARTUUID=11eb23c1-01...
 mmc0: new high speed SD card at address b368
 mmcblk0: mmc0:b368 NCard 1.83 GiB (ro)
  mmcblk0: p1 p2
-mmc1: host does not support reading read-only switch, assuming write-enable
+Waiting for root device PARTUUID=11eb23c1-01...
 hub 1-1:1.0: USB hub found
-mmc1: new high speed SDHC card at address b368
 hub 1-1:1.0: 4 ports detected
+mmc1: host does not support reading read-only switch, assuming write-enable
+mmc1: new high speed SDHC card at address b368
 mmcblk1: mmc1:b368 NCard 3.70 GiB 
  mmcblk1: p1
 EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-05-19 00:42:25.141704552 -0700
+++ /tmp/foob	2016-05-19 00:42:25.141704552 -0700
@@ -45,9 +45,9 @@
 	 #################################################################
 	 #################################################################
 	 ##########################################
-	 909.2 KiB/s
+	 905.3 KiB/s
 done
-Bytes transferred = 5377160 (520c88 hex)
+Bytes transferred = 5377384 (520d68 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -58,7 +58,7 @@
 done
 Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x520c88 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x520d68 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102b3
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-next-20160518-94993-g621a2a0 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue May 17 22:04:51 PDT 2016
+Linux version 4.6.0-next-20160519-95140-g3c60f7e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu May 19 00:34:38 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -89,7 +89,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 961792K/1048576K available (7415K kernel code, 374K rwdata, 2560K rodata, 1024K init, 366K bss, 21248K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 961792K/1048576K available (7416K kernel code, 374K rwdata, 2560K rodata, 1024K init, 366K bss, 21248K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -97,7 +97,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0abdff4   (10968 kB)
+      .text : 0xc0008000 - 0xc0abe028   (10969 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
       .data : 0xc0c00000 - 0xc0c5d8c0   ( 375 kB)
        .bss : 0xc0c5f000 - 0xc0cbab18   ( 367 kB)
@@ -265,25 +265,25 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1b4/0x218
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160518-94993-g621a2a0 #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160519-95140-g3c60f7e #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c010f1e8>] (unwind_backtrace) from [<c010a864>] (show_stack+0x10/0x14)
-[<c010a864>] (show_stack) from [<c033d8d4>] (dump_stack+0x84/0xa4)
-[<c033d8d4>] (dump_stack) from [<c011f308>] (__warn+0xd4/0x100)
+[<c010a864>] (show_stack) from [<c033dc54>] (dump_stack+0x84/0xa4)
+[<c033dc54>] (dump_stack) from [<c011f308>] (__warn+0xd4/0x100)
 [<c011f308>] (__warn) from [<c011f354>] (warn_slowpath_null+0x20/0x28)
-[<c011f354>] (warn_slowpath_null) from [<c03c0e1c>] (__of_reset_control_get+0x1b4/0x218)
-[<c03c0e1c>] (__of_reset_control_get) from [<c0513a94>] (tegra_ehci_probe+0x2c8/0x514)
-[<c0513a94>] (tegra_ehci_probe) from [<c04411b4>] (platform_drv_probe+0x50/0xb0)
-[<c04411b4>] (platform_drv_probe) from [<c043fbe0>] (driver_probe_device+0x1f4/0x2b0)
-[<c043fbe0>] (driver_probe_device) from [<c043fd48>] (__driver_attach+0xac/0xb0)
-[<c043fd48>] (__driver_attach) from [<c043e204>] (bus_for_each_dev+0x54/0x88)
-[<c043e204>] (bus_for_each_dev) from [<c043f1d4>] (bus_add_driver+0xe8/0x1f4)
-[<c043f1d4>] (bus_add_driver) from [<c0440544>] (driver_register+0x78/0xf4)
-[<c0440544>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
+[<c011f354>] (warn_slowpath_null) from [<c03c119c>] (__of_reset_control_get+0x1b4/0x218)
+[<c03c119c>] (__of_reset_control_get) from [<c0513dcc>] (tegra_ehci_probe+0x2c8/0x514)
+[<c0513dcc>] (tegra_ehci_probe) from [<c044155c>] (platform_drv_probe+0x50/0xb0)
+[<c044155c>] (platform_drv_probe) from [<c043ff88>] (driver_probe_device+0x1f4/0x2b0)
+[<c043ff88>] (driver_probe_device) from [<c04400f0>] (__driver_attach+0xac/0xb0)
+[<c04400f0>] (__driver_attach) from [<c043e5ac>] (bus_for_each_dev+0x54/0x88)
+[<c043e5ac>] (bus_for_each_dev) from [<c043f57c>] (bus_add_driver+0xe8/0x1f4)
+[<c043f57c>] (bus_add_driver) from [<c04408ec>] (driver_register+0x78/0xf4)
+[<c04408ec>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
 [<c010179c>] (do_one_initcall) from [<c0b00d94>] (kernel_init_freeable+0x11c/0x1e8)
-[<c0b00d94>] (kernel_init_freeable) from [<c07f107c>] (kernel_init+0x8/0x118)
-[<c07f107c>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
----[ end trace 38d3f79f200bff69 ]---
+[<c0b00d94>] (kernel_init_freeable) from [<c07f14f4>] (kernel_init+0x8/0x118)
+[<c07f14f4>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
+---[ end trace 2e5deb6e81cea0d6 ]---
 tegra-ehci c5008000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci c5008000.usb: continuing, but please update your DT
 tegra-ehci c5008000.usb: EHCI Host Controller
@@ -325,19 +325,19 @@
 tlv320aic23-codec 0-001a: ASoC: no dapm match for RLINEIN --> NULL --> Line Input
 tlv320aic23-codec 0-001a: ASoC: Failed to add route RLINEIN -> NULL -> Line Input
 tlv320aic23-codec 0-001a: Control not supported for path MICIN -> [NULL] -> Mic Input
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
 tlv320aic23-codec 0-001a: ASoC: no dapm match for MICIN --> NULL --> Mic Input
 tlv320aic23-codec 0-001a: ASoC: Failed to add route MICIN -> NULL -> Mic Input
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
 mmc0: new high speed SD card at address b368
 mmcblk0: mmc0:b368 NCard 1.83 GiB (ro)
  mmcblk0: p1 p2
+hub 1-1:1.0: USB hub found
+hub 1-1:1.0: 4 ports detected
 mmc1: host does not support reading read-only switch, assuming write-enable
 mmc1: new high speed SDHC card at address b368
 mmcblk1: mmc1:b368 NCard 3.70 GiB 
  mmcblk1: p1
-hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
 NET: Registered protocol family 10
 mip6: Mobile IPv6
 sit: IPv6 over IPv4 tunneling driver
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2016-05-19 00:42:25.165704555 -0700
+++ /tmp/foob	2016-05-19 00:42:25.169704556 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 726207 us
-Basic modules init took 348648 us
-Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14365 us
-Carveout took 252720 us
-CPU initialization took 92332 us
-Total time taken by TegraBoot 708065 us
+Tegraboot started after 778077 us
+Basic modules init took 348561 us
+Sec Bootdevice Read Time = 204 ms, Read Size = 8459 KB
+Next stage binary read took 15926 us
+Carveout took 252752 us
+CPU initialization took 92324 us
+Total time taken by TegraBoot 709563 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.565] LPDDR4 Training: Number of tables = 10
+64b[0001.617] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,9 +117,9 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-12521472 bytes read in 744 ms (16 MiB/s)
+12525056 bytes read in 745 ms (16 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-37108 bytes read in 452 ms (80.1 KiB/s)
+37108 bytes read in 453 ms (79.1 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
 1627787 bytes read in 496 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-next-20160518+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue May 17 22:07:23 PDT 2016
+Linux version 4.6.0-next-20160519+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu May 19 00:37:04 PDT 2016
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -166,15 +166,15 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffff80007a000000-ffff80007dffffff]
-Memory: 2980468K/3131392K available (7436K kernel code, 644K rwdata, 3184K rodata, 960K init, 258K bss, 134540K reserved, 16384K cma-reserved)
+Memory: 2980464K/3131392K available (7436K kernel code, 647K rwdata, 3184K rodata, 960K init, 259K bss, 134544K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff0000087c0000   (  7424 KB)
     .rodata : 0xffff0000087c0000 - 0xffff000008ae0000   (  3200 KB)
       .init : 0xffff000008ae0000 - 0xffff000008bd0000   (   960 KB)
-      .data : 0xffff000008bd0000 - 0xffff000008c71000   (   644 KB)
-       .bss : 0xffff000008c71000 - 0xffff000008cb1b40   (   259 KB)
+      .data : 0xffff000008bd0000 - 0xffff000008c71e00   (   648 KB)
+       .bss : 0xffff000008c71e00 - 0xffff000008cb2b40   (   260 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -263,7 +263,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.451:1): initialized
+audit: type=2000 audit(0.459:1): initialized
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2016-05-19 00:42:25.173704557 -0700
+++ /tmp/foob	2016-05-19 00:42:25.177704557 -0700
@@ -1,4 +1,4 @@
-4 Processor [411fd071]
+fd071]
 earlycon: uart0 at MMIO 0x0000000070006000 (options '115200n8')
 bootconsole [uart0] enabled
 efi: Getting EFI parameters from FDT:
@@ -23,15 +23,15 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2834704K/2988032K available (7436K kernel code, 644K rwdata, 3184K rodata, 960K init, 258K bss, 136944K reserved, 16384K cma-reserved)
+Memory: 2834700K/2988032K available (7436K kernel code, 647K rwdata, 3184K rodata, 960K init, 259K bss, 136948K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff0000087c0000   (  7424 KB)
     .rodata : 0xffff0000087c0000 - 0xffff000008ae0000   (  3200 KB)
       .init : 0xffff000008ae0000 - 0xffff000008bd0000   (   960 KB)
-      .data : 0xffff000008bd0000 - 0xffff000008c71000   (   644 KB)
-       .bss : 0xffff000008c71000 - 0xffff000008cb1b40   (   259 KB)
+      .data : 0xffff000008bd0000 - 0xffff000008c71e00   (   648 KB)
+       .bss : 0xffff000008c71e00 - 0xffff000008cb2b40   (   260 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -60,7 +60,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x9518: expected 0x2004f400
+boot misc3 0x9408: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -73,7 +73,7 @@
 console [tty0] enabled
 bootconsole [uart0] disabled
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-next-20160518+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue May 17 22:07:23 PDT 2016
+Linux version 4.6.0-next-20160519+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu May 19 00:37:04 PDT 2016
 Boot CPU: AArch64 Processor [411fd071]
 earlycon: uart0 at MMIO 0x0000000070006000 (options '115200n8')
 bootconsole [uart0] enabled
@@ -99,15 +99,15 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2834704K/2988032K available (7436K kernel code, 644K rwdata, 3184K rodata, 960K init, 258K bss, 136944K reserved, 16384K cma-reserved)
+Memory: 2834700K/2988032K available (7436K kernel code, 647K rwdata, 3184K rodata, 960K init, 259K bss, 136948K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff0000087c0000   (  7424 KB)
     .rodata : 0xffff0000087c0000 - 0xffff000008ae0000   (  3200 KB)
       .init : 0xffff000008ae0000 - 0xffff000008bd0000   (   960 KB)
-      .data : 0xffff000008bd0000 - 0xffff000008c71000   (   644 KB)
-       .bss : 0xffff000008c71000 - 0xffff000008cb1b40   (   259 KB)
+      .data : 0xffff000008bd0000 - 0xffff000008c71e00   (   648 KB)
+       .bss : 0xffff000008c71e00 - 0xffff000008cb2b40   (   260 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -136,7 +136,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x9518: expected 0x2004f400
+boot misc3 0x9408: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -206,14 +206,14 @@
 RPC: Registered tcp transport module.
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 Unpacking initramfs...
-Freeing initrd memory: 2364K (ffff800009e97000 - ffff80000a0e6000)
+Freeing initrd memory: 2364K (ffff800009e98000 - ffff80000a0e7000)
 kvm [1]: 8-bit VMID
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: GICH base=0x50044000, GICV base=0x50046000, IRQ=1
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.239:1): initialized
+audit: type=2000 audit(0.227:1): initialized
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-05-19 00:42:25.201704561 -0700
+++ /tmp/foob	2016-05-19 00:42:25.205704561 -0700
@@ -27,7 +27,6 @@
 Tegra30 (Beaver) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=edbe1840-3633-c751-7993-b0ac834f81e0 rootwait gpt debug noinitrd earlyprintk init=/bin/bash'
 Tegra30 (Beaver) # setenv ipaddr 192.168.100.101; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra30 (Beaver) # tftpboot 0x81000000 zImage.multi_v7_defconfig
-Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'zImage.multi_v7_defconfig'.
@@ -40,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 ############################
-	 3.3 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 7079112 (6c04c8 hex)
+Bytes transferred = 7077600 (6bfee0 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 51627 (c9ab hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6c04c8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6bfee0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff9aa ... OK
@@ -61,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-next-20160518-94993-g621a2a0 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue May 17 22:04:51 PDT 2016
+Linux version 4.6.0-next-20160519-95140-g3c60f7e (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu May 19 00:34:41 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -69,7 +68,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141bb40, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c141bac0, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -90,9 +89,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc106c538   (14738 kB)
+      .text : 0xc0208000 - 0xc106c520   (14738 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141dba0   (1143 kB)
+      .data : 0xc1300000 - 0xc141db20   (1143 kB)
        .bss : 0xc141f000 - 0xc1474ca0   ( 344 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -242,25 +241,25 @@
 ------------[ cut here ]------------
 WARNING: CPU: 1 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1bc/0x228
 Modules linked in:
-CPU: 1 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160518-94993-g621a2a0 #1
+CPU: 1 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160519-95140-g3c60f7e #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c030f9d0>] (unwind_backtrace) from [<c030b0c8>] (show_stack+0x10/0x14)
-[<c030b0c8>] (show_stack) from [<c0585fc8>] (dump_stack+0x84/0xa4)
-[<c0585fc8>] (dump_stack) from [<c0341b6c>] (__warn+0xd4/0x100)
+[<c030b0c8>] (show_stack) from [<c05863a8>] (dump_stack+0x84/0xa4)
+[<c05863a8>] (dump_stack) from [<c0341b6c>] (__warn+0xd4/0x100)
 [<c0341b6c>] (__warn) from [<c0341bb8>] (warn_slowpath_null+0x20/0x28)
-[<c0341bb8>] (warn_slowpath_null) from [<c06a0494>] (__of_reset_control_get+0x1bc/0x228)
-[<c06a0494>] (__of_reset_control_get) from [<c0911eb8>] (tegra_ehci_probe+0x2d0/0x51c)
-[<c0911eb8>] (tegra_ehci_probe) from [<c0778f9c>] (platform_drv_probe+0x50/0xb0)
-[<c0778f9c>] (platform_drv_probe) from [<c0777900>] (driver_probe_device+0x1f4/0x2b0)
-[<c0777900>] (driver_probe_device) from [<c0777a68>] (__driver_attach+0xac/0xb0)
-[<c0777a68>] (__driver_attach) from [<c0775f14>] (bus_for_each_dev+0x54/0x88)
-[<c0775f14>] (bus_for_each_dev) from [<c0776ef4>] (bus_add_driver+0xe8/0x1f4)
-[<c0776ef4>] (bus_add_driver) from [<c0778284>] (driver_register+0x78/0xf4)
-[<c0778284>] (driver_register) from [<c0301e7c>] (do_one_initcall+0x3c/0x16c)
+[<c0341bb8>] (warn_slowpath_null) from [<c06a0874>] (__of_reset_control_get+0x1bc/0x228)
+[<c06a0874>] (__of_reset_control_get) from [<c091235c>] (tegra_ehci_probe+0x2d0/0x51c)
+[<c091235c>] (tegra_ehci_probe) from [<c07793a4>] (platform_drv_probe+0x50/0xb0)
+[<c07793a4>] (platform_drv_probe) from [<c0777d08>] (driver_probe_device+0x1f4/0x2b0)
+[<c0777d08>] (driver_probe_device) from [<c0777e70>] (__driver_attach+0xac/0xb0)
+[<c0777e70>] (__driver_attach) from [<c077631c>] (bus_for_each_dev+0x54/0x88)
+[<c077631c>] (bus_for_each_dev) from [<c07772fc>] (bus_add_driver+0xe8/0x1f4)
+[<c07772fc>] (bus_add_driver) from [<c077868c>] (driver_register+0x78/0xf4)
+[<c077868c>] (driver_register) from [<c0301e7c>] (do_one_initcall+0x3c/0x16c)
 [<c0301e7c>] (do_one_initcall) from [<c1100d80>] (kernel_init_freeable+0x120/0x1ec)
-[<c1100d80>] (kernel_init_freeable) from [<c0b5bc84>] (kernel_init+0x8/0x118)
-[<c0b5bc84>] (kernel_init) from [<c0307fb8>] (ret_from_fork+0x14/0x3c)
----[ end trace a9d9d336791fc9e0 ]---
+[<c1100d80>] (kernel_init_freeable) from [<c0b5b7fc>] (kernel_init+0x8/0x118)
+[<c0b5b7fc>] (kernel_init) from [<c0307fb8>] (ret_from_fork+0x14/0x3c)
+---[ end trace 7f49f071cd535254 ]---
 tegra-ehci 7d008000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d008000.usb: continuing, but please update your DT
 ohci_hcd: USB 1.1 'Open' Host Controller (OHCI) Driver
@@ -305,17 +304,17 @@
 sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
 usb3_vbus: supplied by vdd_5v_in
 usb1_vbus: supplied by vdd_5v_in
-vdd_5v_sata: supplied by vdd_5v_in
-vdd_ddr: supplied by vdd_5v_in
 mmc0: new high speed MMC card at address 0001
-tegra-pcie 3000.pcie-controller: 2x3 configuration
-tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 mmcblk0: mmc0:0001 SEM16G 14.8 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 1.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 1.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 128 KiB
 Primary GPT is invalid, using alternate GPT.
  mmcblk0: p1 p2 p3 p4 p5
+vdd_5v_sata: supplied by vdd_5v_in
+vdd_ddr: supplied by vdd_5v_in
+tegra-pcie 3000.pcie-controller: 2x3 configuration
+tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 mmc1: error -110 whilst initialising SD card
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -386,7 +385,7 @@
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
-bash: cannot set terminal process group (-1): Inappropriate ioctl for device
+bash: cannot set terminal process group (-1): Inappropriate ioct[    3.938255] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+l for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-05-19 00:42:25.209704562 -0700
+++ /tmp/foob	2016-05-19 00:42:25.213704563 -0700
@@ -37,9 +37,9 @@
 	 #################################################################
 	 #################################################################
 	 ##########################################
-	 3.2 MiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 5377160 (520c88 hex)
+Bytes transferred = 5377384 (520d68 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 51627 (c9ab hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x520c88 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x520d68 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff9aa ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.6.0-next-20160518-94993-g621a2a0 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue May 17 22:04:51 PDT 2016
+Linux version 4.6.0-next-20160519-95140-g3c60f7e (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu May 19 00:34:38 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -81,7 +81,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 2000896K/2096128K available (7415K kernel code, 374K rwdata, 2560K rodata, 1024K init, 366K bss, 29696K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 2000896K/2096128K available (7416K kernel code, 374K rwdata, 2560K rodata, 1024K init, 366K bss, 29696K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -89,7 +89,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0abdff4   (10968 kB)
+      .text : 0xc0008000 - 0xc0abe028   (10969 kB)
       .init : 0xc0b00000 - 0xc0c00000   (1024 kB)
       .data : 0xc0c00000 - 0xc0c5d8c0   ( 375 kB)
        .bss : 0xc0c5f000 - 0xc0cbab18   ( 367 kB)
@@ -231,25 +231,25 @@
 ------------[ cut here ]------------
 WARNING: CPU: 0 PID: 1 at drivers/reset/core.c:187 __of_reset_control_get+0x1b4/0x218
 Modules linked in:
-CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160518-94993-g621a2a0 #1
+CPU: 0 PID: 1 Comm: swapper/0 Not tainted 4.6.0-next-20160519-95140-g3c60f7e #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 [<c010f1e8>] (unwind_backtrace) from [<c010a864>] (show_stack+0x10/0x14)
-[<c010a864>] (show_stack) from [<c033d8d4>] (dump_stack+0x84/0xa4)
-[<c033d8d4>] (dump_stack) from [<c011f308>] (__warn+0xd4/0x100)
+[<c010a864>] (show_stack) from [<c033dc54>] (dump_stack+0x84/0xa4)
+[<c033dc54>] (dump_stack) from [<c011f308>] (__warn+0xd4/0x100)
 [<c011f308>] (__warn) from [<c011f354>] (warn_slowpath_null+0x20/0x28)
-[<c011f354>] (warn_slowpath_null) from [<c03c0e1c>] (__of_reset_control_get+0x1b4/0x218)
-[<c03c0e1c>] (__of_reset_control_get) from [<c0513a94>] (tegra_ehci_probe+0x2c8/0x514)
-[<c0513a94>] (tegra_ehci_probe) from [<c04411b4>] (platform_drv_probe+0x50/0xb0)
-[<c04411b4>] (platform_drv_probe) from [<c043fbe0>] (driver_probe_device+0x1f4/0x2b0)
-[<c043fbe0>] (driver_probe_device) from [<c043fd48>] (__driver_attach+0xac/0xb0)
-[<c043fd48>] (__driver_attach) from [<c043e204>] (bus_for_each_dev+0x54/0x88)
-[<c043e204>] (bus_for_each_dev) from [<c043f1d4>] (bus_add_driver+0xe8/0x1f4)
-[<c043f1d4>] (bus_add_driver) from [<c0440544>] (driver_register+0x78/0xf4)
-[<c0440544>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
+[<c011f354>] (warn_slowpath_null) from [<c03c119c>] (__of_reset_control_get+0x1b4/0x218)
+[<c03c119c>] (__of_reset_control_get) from [<c0513dcc>] (tegra_ehci_probe+0x2c8/0x514)
+[<c0513dcc>] (tegra_ehci_probe) from [<c044155c>] (platform_drv_probe+0x50/0xb0)
+[<c044155c>] (platform_drv_probe) from [<c043ff88>] (driver_probe_device+0x1f4/0x2b0)
+[<c043ff88>] (driver_probe_device) from [<c04400f0>] (__driver_attach+0xac/0xb0)
+[<c04400f0>] (__driver_attach) from [<c043e5ac>] (bus_for_each_dev+0x54/0x88)
+[<c043e5ac>] (bus_for_each_dev) from [<c043f57c>] (bus_add_driver+0xe8/0x1f4)
+[<c043f57c>] (bus_add_driver) from [<c04408ec>] (driver_register+0x78/0xf4)
+[<c04408ec>] (driver_register) from [<c010179c>] (do_one_initcall+0x3c/0x16c)
 [<c010179c>] (do_one_initcall) from [<c0b00d94>] (kernel_init_freeable+0x11c/0x1e8)
-[<c0b00d94>] (kernel_init_freeable) from [<c07f107c>] (kernel_init+0x8/0x118)
-[<c07f107c>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
----[ end trace 51ba94f4f390921c ]---
+[<c0b00d94>] (kernel_init_freeable) from [<c07f14f4>] (kernel_init+0x8/0x118)
+[<c07f14f4>] (kernel_init) from [<c0107938>] (ret_from_fork+0x14/0x3c)
+---[ end trace b2982a42cf303bca ]---
 tegra-ehci 7d008000.usb: can't get utmi-pads reset from the PHY
 tegra-ehci 7d008000.usb: continuing, but please update your DT
 usbcore: registered new interface driver cdc_acm
@@ -388,4 +388,4 @@
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    7.828608] asix 1-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0f:67:60
+root@(none):/# [    7.768665] asix 1-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0f:67:60
