Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-06-16 21:56:25.515954061 -0700
+++ /tmp/foob	2016-06-16 21:56:25.519954064 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-next-20160616+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Jun 15 23:21:07 PDT 2016
+Linux version 4.7.0-rc3-next-20160617+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Jun 16 21:51:00 PDT 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -19,15 +19,15 @@
 Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
 Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
 software IO TLB [mem 0x79c00000-0x7dc00000] (64MB) mapped at [ffff800039c00000-ffff80003dbfffff]
-Memory: 934968K/1048576K available (7660K kernel code, 682K rwdata, 3472K rodata, 960K init, 261K bss, 97224K reserved, 16384K cma-reserved)
+Memory: 934968K/1048576K available (7660K kernel code, 683K rwdata, 3472K rodata, 960K init, 260K bss, 97224K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff0000087f0000   (  7616 KB)
     .rodata : 0xffff0000087f0000 - 0xffff000008b60000   (  3520 KB)
       .init : 0xffff000008b60000 - 0xffff000008c50000   (   960 KB)
-      .data : 0xffff000008c50000 - 0xffff000008cfaa00   (   683 KB)
-       .bss : 0xffff000008cfaa00 - 0xffff000008d3be38   (   262 KB)
+      .data : 0xffff000008c50000 - 0xffff000008cfac00   (   683 KB)
+       .bss : 0xffff000008cfac00 - 0xffff000008d3be38   (   261 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -98,7 +98,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.644:1): initialized
+audit: type=2000 audit(0.592:1): initialized
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -161,7 +161,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2016-06-16 06:21:40 UTC (1466058100)
+rtc-pl031 9010000.pl031: setting system clock to 2016-06-17 04:51:33 UTC (1466139093)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-06-16 21:56:25.543954075 -0700
+++ /tmp/foob	2016-06-16 21:56:25.547954077 -0700
@@ -39,20 +39,20 @@
 	 #################################################################
 	 #################################################################
 	 ########################################
-	 3 MiB/s
+	 3.4 MiB/s
 done
-Bytes transferred = 7264992 (6edae0 hex)
+Bytes transferred = 7264112 (6ed770 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3 MiB/s
+	 2.6 MiB/s
 done
 Bytes transferred = 34956 (888c hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6edae0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6ed770 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff88b ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-next-20160616-100112-g649a371 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Jun 15 23:18:47 PDT 2016
+Linux version 4.7.0-rc3-next-20160617-100452-gce24ed9 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Jun 16 21:48:39 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -69,7 +69,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141bac0, node_mem_map eeffa000
+free_area_init_node: node 0, pgdat c141bb00, node_mem_map eeffa000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -90,9 +90,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc10b47d4   (15026 kB)
+      .text : 0xc0208000 - 0xc10b47f4   (15026 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141db20   (1143 kB)
+      .data : 0xc1300000 - 0xc141db60   (1143 kB)
        .bss : 0xc141f000 - 0xc14748d8   ( 343 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -283,19 +283,11 @@
 usb1_vbus: supplied by vdd-sys-5v0
 usb2_vbus: supplied by vdd-sys-5v0
 VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
-mmc0: new high speed SDHC card at address aaaa
-mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
 tegra-dsi 54300000.dsi: cannot get VDD supply
- mmcblk0: p1 p2 p3
+mmc0: new high speed SDHC card at address aaaa
 tegra-ehci 7d008000.usb: EHCI Host Controller
 tegra-ehci 7d008000.usb: new USB bus registered, assigned bus number 1
-mmc1: new high speed MMC card at address 0001
-mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
-mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
-mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
-mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
- mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
 tegra-ehci 7d008000.usb: irq 339, io mem 0x7d008000
 tegra-ehci 7d008000.usb: USB 2.0 started, EHCI 1.10
 hub 1-0:1.0: USB hub found
@@ -310,13 +302,25 @@
 dvdd_ts: disabling
 usb1_vbus: disabling
 VDD_5V0_HDMI_CON: disabling
+mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
+mmc1: new high speed MMC card at address 0001
+mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
+mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
+mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
+mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
+ mmcblk0: p1 p2 p3
+ mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
+tegra-hdmi 54280000.hdmi: failed to get PLL regulator
+tegra-dsi 54300000.dsi: cannot get VDD supply
 EXT4-fs (mmcblk1p1): mounting ext3 file system using the ext4 subsystem
+tegra-hdmi 54280000.hdmi: failed to get PLL regulator
+tegra-dsi 54300000.dsi: cannot get VDD supply
 EXT4-fs (mmcblk1p1): recovery complete
 EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext3 filesystem) on device 179:17.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-06-16 21:56:25.551954079 -0700
+++ /tmp/foob	2016-06-16 21:56:25.555954081 -0700
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra114 (Dalmore) # 
 Tegra114 (Dalmore) # usb reset
 (Re)start USB...
@@ -39,7 +39,7 @@
 	 ######################################################
 	 3.2 MiB/s
 done
-Bytes transferred = 5560616 (54d928 hex)
+Bytes transferred = 5562352 (54dff0 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 34956 (888c hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x54d928 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x54dff0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff88b ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-next-20160616-100112-g649a371 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Jun 15 23:18:40 PDT 2016
+Linux version 4.7.0-rc3-next-20160617-100452-gce24ed9 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Jun 16 21:48:34 PDT 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -90,7 +90,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0b052e8   (11253 kB)
+      .text : 0xc0008000 - 0xc0b05318   (11253 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
       .data : 0xc0d00000 - 0xc0d5d8e0   ( 375 kB)
        .bss : 0xc0d5f000 - 0xc0dbaad4   ( 367 kB)
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-06-16 21:56:25.559954083 -0700
+++ /tmp/foob	2016-06-16 21:56:25.563954084 -0700
@@ -31,23 +31,23 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ##################T T T ###############################################
+	 #################################################################
 	 #################################################################
 	 ########################################
-	 420.9 KiB/s
+	 3.3 MiB/s
 done
-Bytes transferred = 7264992 (6edae0 hex)
+Bytes transferred = 7264112 (6ed770 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.7 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 66737 (104b1 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6edae0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x6ed770 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830134b0
@@ -55,7 +55,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-next-20160616-100112-g649a371 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Jun 15 23:18:47 PDT 2016
+Linux version 4.7.0-rc3-next-20160617-100452-gce24ed9 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Jun 16 21:48:39 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -64,7 +64,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141bac0, node_mem_map eeffa000
+free_area_init_node: node 0, pgdat c141bb00, node_mem_map eeffa000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -85,9 +85,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc10b47d4   (15026 kB)
+      .text : 0xc0208000 - 0xc10b47f4   (15026 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141db20   (1143 kB)
+      .data : 0xc1300000 - 0xc141db60   (1143 kB)
        .bss : 0xc141f000 - 0xc14748d8   ( 343 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -379,20 +379,20 @@
 pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
 pci 0000:02:00.0: [10ec:8168] type 00 class 0x020000
 pci 0000:02:00.0: reg 0x10: [io  0x0000-0x00ff]
+ata1: SATA link down (SStatus 0 SControl 300)
 pci 0000:02:00.0: reg 0x18: [mem 0x00000000-0x00000fff 64bit]
 pci 0000:02:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
-ata1: SATA link down (SStatus 0 SControl 300)
 pci 0000:02:00.0: supports D1 D2
 pci 0000:02:00.0: PME# supported from D0 D1 D2 D3hot D3cold
 PCI: bus2: Fast back to back transfers disabled
 pci_bus 0000:02: busn_res: [bus 02-ff] end is updated to 02
 pci 0000:00:01.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
-pci 0000:00:02.0: BAR 8: assigned [mem 0x13100000-0x131fffff]
-pci 0000:00:02.0: BAR 9: assigned [mem 0x20000000-0x200fffff 64bit pref]
 mmc1: new high speed SDHC card at address aaaa
 mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
-pci 0000:00:02.0: BAR 7: assigned [io  0x1000-0x1fff]
  mmcblk1: p1 p2 p3
+pci 0000:00:02.0: BAR 8: assigned [mem 0x13100000-0x131fffff]
+pci 0000:00:02.0: BAR 9: assigned [mem 0x20000000-0x200fffff 64bit pref]
+pci 0000:00:02.0: BAR 7: assigned [io  0x1000-0x1fff]
 pci 0000:01:00.0: BAR 0: assigned [mem 0x13000000-0x13001fff 64bit]
 pci 0000:00:01.0: PCI bridge to [bus 01]
 pci 0000:00:01.0:   bridge window [mem 0x13000000-0x130fffff]
@@ -430,7 +430,7 @@
 cpu cpu2: cpufreq_init: failed to get clk: -2
 cpu cpu3: cpufreq_init: failed to get clk: -2
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:08 UTC (946684928)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:13 UTC (946684933)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-06-16 21:56:25.567954086 -0700
+++ /tmp/foob	2016-06-16 21:56:25.571954088 -0700
@@ -34,18 +34,18 @@
 	 ######################################################
 	 4 MiB/s
 done
-Bytes transferred = 5560616 (54d928 hex)
+Bytes transferred = 5562352 (54dff0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 4 MiB/s
+	 3.7 MiB/s
 done
 Bytes transferred = 66737 (104b1 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x54d928 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x54dff0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830134b0
@@ -54,7 +54,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-next-20160616-100112-g649a371 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Jun 15 23:18:40 PDT 2016
+Linux version 4.7.0-rc3-next-20160617-100452-gce24ed9 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Jun 16 21:48:34 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -85,7 +85,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0b052e8   (11253 kB)
+      .text : 0xc0008000 - 0xc0b05318   (11253 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
       .data : 0xc0d00000 - 0xc0d5d8e0   ( 375 kB)
        .bss : 0xc0d5f000 - 0xc0dbaad4   ( 367 kB)
@@ -399,15 +399,14 @@
 pci 0000:02:00.0: supports D1 D2
 pci 0000:02:00.0: PME# supported from D0 D1 D2 D3hot D3cold
 mmc1: new high speed SDHC card at address aaaa
+mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 PCI: bus2: Fast back to back transfers disabled
 pci_bus 0000:02: busn_res: [bus 02-ff] end is updated to 02
-mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 pci 0000:00:01.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
 pci 0000:00:02.0: BAR 8: assigned [mem 0x13100000-0x131fffff]
 pci 0000:00:02.0: BAR 9: assigned [mem 0x20000000-0x200fffff 64bit pref]
 pci 0000:00:02.0: BAR 7: assigned [io  0x1000-0x1fff]
 pci 0000:01:00.0: BAR 0: assigned [mem 0x13000000-0x13001fff 64bit]
- mmcblk1: p1 p2 p3
 pci 0000:00:01.0: PCI bridge to [bus 01]
 pci 0000:00:01.0:   bridge window [mem 0x13000000-0x130fffff]
 pci 0000:02:00.0: BAR 4: assigned [mem 0x20000000-0x20003fff 64bit pref]
@@ -433,8 +432,10 @@
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
-tegra-hdmi 54280000.hdmi: cannot set audio to 44100 at 297000000 pclk
-Console: switching to colour frame buffer device 512x135
+drm drm: No connectors reported connected with modes
+[drm] Cannot find any crtc or sizes - going 1024x768
+ mmcblk1: p1 p2 p3
+Console: switching to colour frame buffer device 128x48
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 of: dev_pm_opp_of_cpumask_add_table: couldn't find opp table for cpu:0, -19
@@ -444,7 +445,7 @@
 cpu cpu2: cpufreq_init: failed to get clk: -2
 cpu cpu3: cpufreq_init: failed to get clk: -2
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:16 UTC (946684816)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:15 UTC (946684815)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-06-16 21:56:25.575954090 -0700
+++ /tmp/foob	2016-06-16 21:56:25.579954092 -0700
@@ -41,9 +41,9 @@
 	 #################################################################
 	 #################################################################
 	 ########################################
-	 3.3 MiB/s
+	 3.6 MiB/s
 done
-Bytes transferred = 7264992 (6edae0 hex)
+Bytes transferred = 7264112 (6ed770 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -54,7 +54,7 @@
 done
 Bytes transferred = 66901 (10555 hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x6edae0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x6ed770 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013554
@@ -62,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-next-20160616-100112-g649a371 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Jun 15 23:18:47 PDT 2016
+Linux version 4.7.0-rc3-next-20160617-100452-gce24ed9 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Jun 16 21:48:39 PDT 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -72,7 +72,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c141bac0, node_mem_map eeffa000
+free_area_init_node: node 0, pgdat c141bb00, node_mem_map eeffa000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -93,9 +93,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc10b47d4   (15026 kB)
+      .text : 0xc0208000 - 0xc10b47f4   (15026 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141db20   (1143 kB)
+      .data : 0xc1300000 - 0xc141db60   (1143 kB)
        .bss : 0xc141f000 - 0xc14748d8   ( 343 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -330,8 +330,8 @@
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc2: Unknown controller version (3). You may experience problems.
-mmc0: new high speed MMC card at address 0001
 mmc2: Invalid maximum block size, assuming 512 bytes
+mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
 mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2016-06-16 21:56:25.583954093 -0700
+++ /tmp/foob	2016-06-16 21:56:25.583954093 -0700
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -41,7 +41,7 @@
 	 ######################################################
 	 3.6 MiB/s
 done
-Bytes transferred = 5560616 (54d928 hex)
+Bytes transferred = 5562352 (54dff0 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -52,7 +52,7 @@
 done
 Bytes transferred = 66901 (10555 hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x54d928 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x54dff0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013554
@@ -141,23 +141,23 @@
 tegra30-i2s 70301100.i2s: DMA channels sourced from device 70300000.ahub
 max98090 0-0010: MAX98090 REVID=0x43
 max98090 0-0010: use default 2.8v micbias
+tegra-snd-max98090 sound: HiFi <-> 70301100.i2s mapping ok
+input: GoogleNyanBig Headphones as /devices/soc0/sound/sound/card1/input1
+input: GoogleNyanBig Mic Jack as /devices/soc0/sound/sound/card1/input2
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
 mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
 mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
-tegra-snd-max98090 sound: HiFi <-> 70301100.i2s mapping ok
-input: GoogleNyanBig Headphones as /devices/soc0/sound/sound/card1/input1
-input: GoogleNyanBig Mic Jack as /devices/soc0/sound/sound/card1/input2
 NET: Registered protocol family 10
 mip6: Mobile IPv6
 sit: IPv6 over IPv4 tunneling driver
 NET: Registered protocol family 17
 NET: Registered protocol family 15
 can: controller area network core (rev 20120528 abi 9)
-NET: Registered protocol family 29
 input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input3
+NET: Registered protocol family 29
 can: raw protocol (rev 20120528)
 can: broadcast manager protocol (rev 20120528 t)
 can: netlink gateway (rev 20130117) max_hops=1
@@ -215,17 +215,17 @@
 usb 1-2: new high-speed USB device number 2 using tegra-xusb
 mmc2: new high speed SDHC card at address 0007
 mmcblk2: mmc2:0007 SD08G 7.42 GiB 
-Console: switching to colour frame buffer device 170x48
 GPT:Primary header alternate_lba != Alt. header my_lba
 GPT:3014591 != 15564799
 GPT:Primary header thinks Alt. header is not at the end of the disk.
 GPT:3014591 != 15564799
 GPT: Use GNU Parted to correct GPT errors.
  mmcblk2: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
+Console: switching to colour frame buffer device 170x48
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input4
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:24 UTC (946684824)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:23 UTC (946684823)
 +1.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
 +1.05V_RUN_CAM_REAR: disabling
@@ -246,7 +246,7 @@
 Freeing unused kernel memory: 1024K (c0c00000 - c0d00000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.844663] usb 1-3: new high-speed USB device number 3 using tegra-xusb
+root@(none):/# [    4.834658] usb 1-3: new high-speed USB device number 3 using tegra-xusb
 asix 1-3:1.0 eth0: register 'asix' at usb-70090000.usb-3, ASIX AX88772 USB 2.0 Ethernet, c8:d7:19:90:fa:65
 usb 1-1: new high-speed USB device number 4 using tegra-xusb
 usb-storage 1-1:1.0: USB Mass Storage device detected
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2016-06-16 21:56:25.591954098 -0700
+++ /tmp/foob	2016-06-16 21:56:25.591954098 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1800819 us
+Bootloader-Cpu Init at (time stamp): 1801287 us
 
 Pinmux changes applied in kernel way
 
@@ -62,22 +62,22 @@
 Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1800793  us
-Bootloader (AosInit) time: 734249 us
-Bootloader Main Init time: 180088 us
-BlockDevMgrInit time: 157545 us
+Miscellaneous avp time: 1801261  us
+Bootloader (AosInit) time: 733454 us
+Bootloader Main Init time: 180324 us
+BlockDevMgrInit time: 157568 us
 PartMgrInit time: 9 us
 StorMgrInit time: 30 us
 FsMgrInit time: 2 us
-Partition Table load: 76732 us 
-Display init time: 574698 us
+Partition Table load: 77038 us 
+Display init time: 574642 us
 Kernel read time: 359 msec
 Load Kernel Size:   15513600 Bytes
 Bl Kernel ReadSpeed: 43213000 Bytes/sec
-Pre-Kernel Jump time: 1675409 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5754316 us
+Pre-Kernel Jump time: 1674974 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5756075 us
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-next-20160616+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Jun 15 23:21:07 PDT 2016
+Linux version 4.7.0-rc3-next-20160617+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Jun 16 21:51:00 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
 Memory limited to 1734MB
 efi: Getting EFI parameters from FDT:
@@ -92,15 +92,15 @@
 Dentry cache hash table entries: 262144 (order: 9, 2097152 bytes)
 Inode-cache hash table entries: 131072 (order: 8, 1048576 bytes)
 software IO TLB [mem 0xe5200000-0xe9200000] (64MB) mapped at [ffff800065200000-ffff8000691fffff]
-Memory: 1641560K/1775616K available (7660K kernel code, 682K rwdata, 3472K rodata, 960K init, 261K bss, 117672K reserved, 16384K cma-reserved)
+Memory: 1641560K/1775616K available (7660K kernel code, 683K rwdata, 3472K rodata, 960K init, 260K bss, 117672K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff0000087f0000   (  7616 KB)
     .rodata : 0xffff0000087f0000 - 0xffff000008b60000   (  3520 KB)
       .init : 0xffff000008b60000 - 0xffff000008c50000   (   960 KB)
-      .data : 0xffff000008c50000 - 0xffff000008cfaa00   (   683 KB)
-       .bss : 0xffff000008cfaa00 - 0xffff000008d3be38   (   262 KB)
+      .data : 0xffff000008c50000 - 0xffff000008cfac00   (   683 KB)
+       .bss : 0xffff000008cfac00 - 0xffff000008d3be38   (   261 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -185,7 +185,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(1.107:1): initialized
+audit: type=2000 audit(1.103:1): initialized
 workingset: timestamp_bits=44 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -245,10 +245,14 @@
 NET: Registered protocol family 17
 9pnet: Installing 9P2000 support
 Key type dns_resolver registered
+mmc0: new high speed MMC card at address 0001
 registered taskstats version 1
 +VDD_LED: supplied by +VDD_MUX
-mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 +5V_USB_HS: supplied by +5V_SYS
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
 +5V_USB_SS: supplied by +5V_SYS
 +5V_HDMI_CON: supplied by +5V_SYS
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
@@ -268,10 +272,6 @@
 ALSA device list:
   No soundcards found.
 Freeing unused kernel memory: 960K (ffff800000b60000 - ffff800000c50000)
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-06-16 21:56:25.595954100 -0700
+++ /tmp/foob	2016-06-16 21:56:25.599954102 -0700
@@ -49,18 +49,18 @@
 	 ########################################
 	 906.3 KiB/s
 done
-Bytes transferred = 7264992 (6edae0 hex)
+Bytes transferred = 7264112 (6ed770 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 861.3 KiB/s
+	 898.4 KiB/s
 done
 Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x6edae0 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x6ed770 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102b3
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-next-20160616-100112-g649a371 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Jun 15 23:18:47 PDT 2016
+Linux version 4.7.0-rc3-next-20160617-100452-gce24ed9 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Jun 16 21:48:39 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -76,7 +76,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c141bac0, node_mem_map ef7fa000
+free_area_init_node: node 0, pgdat c141bb00, node_mem_map ef7fa000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -97,9 +97,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc10b47d4   (15026 kB)
+      .text : 0xc0208000 - 0xc10b47f4   (15026 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141db20   (1143 kB)
+      .data : 0xc1300000 - 0xc141db60   (1143 kB)
        .bss : 0xc141f000 - 0xc14748d8   ( 343 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
@@ -321,20 +321,20 @@
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
 mmc0: host does not support reading read-only switch, assuming write-enable
-tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 mmc0: new high speed SDHC card at address b368
 mmcblk0: mmc0:b368 NCard 3.70 GiB 
+ mmcblk0: p1
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
+tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: hctosys: unable to read the hardware clock
- mmcblk0: p1
-EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 mmc1: new high speed SD card at address b368
+rtc-em3027 2-0056: hctosys: unable to read the hardware clock
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
  mmcblk1: p1 p2
+EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 hub 1-1:1.0: USB hub found
 hub 1-1:1.0: 4 ports detected
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-06-16 21:56:25.603954104 -0700
+++ /tmp/foob	2016-06-16 21:56:25.607954106 -0700
@@ -47,18 +47,18 @@
 	 ######################################################
 	 4.1 MiB/s
 done
-Bytes transferred = 5560616 (54d928 hex)
+Bytes transferred = 5562352 (54dff0 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 4 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21172 (52b4 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x54d928 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x54dff0 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024102b3
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-next-20160616-100112-g649a371 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Jun 15 23:18:40 PDT 2016
+Linux version 4.7.0-rc3-next-20160617-100452-gce24ed9 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Jun 16 21:48:34 PDT 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: Compulab TrimSlice board
@@ -97,7 +97,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0b052e8   (11253 kB)
+      .text : 0xc0008000 - 0xc0b05318   (11253 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
       .data : 0xc0d00000 - 0xc0d5d8e0   ( 375 kB)
        .bss : 0xc0d5f000 - 0xc0dbaad4   ( 367 kB)
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2016-06-16 21:56:25.611954107 -0700
+++ /tmp/foob	2016-06-16 21:56:25.615954109 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 736723 us
-Basic modules init took 349102 us
-Sec Bootdevice Read Time = 203 ms, Read Size = 8459 KB
-Next stage binary read took 14942 us
-Carveout took 253616 us
-CPU initialization took 92305 us
-Total time taken by TegraBoot 709965 us
+Tegraboot started after 753388 us
+Basic modules init took 348517 us
+Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
+Next stage binary read took 14388 us
+Carveout took 252726 us
+CPU initialization took 92569 us
+Total time taken by TegraBoot 708200 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.577] LPDDR4 Training: Number of tables = 10
+64b[0001.592] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-13085184 bytes read in 759 ms (16.4 MiB/s)
+13085696 bytes read in 758 ms (16.5 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
 39484 bytes read in 453 ms (85 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 494 ms (3.1 MiB/s)
+1627787 bytes read in 495 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-next-20160616+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Jun 15 23:21:07 PDT 2016
+Linux version 4.7.0-rc3-next-20160617+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Jun 16 21:51:00 PDT 2016
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -166,15 +166,15 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffff80007a000000-ffff80007dffffff]
-Memory: 2979896K/3131392K available (7660K kernel code, 682K rwdata, 3472K rodata, 960K init, 261K bss, 135112K reserved, 16384K cma-reserved)
+Memory: 2979896K/3131392K available (7660K kernel code, 683K rwdata, 3472K rodata, 960K init, 260K bss, 135112K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff0000087f0000   (  7616 KB)
     .rodata : 0xffff0000087f0000 - 0xffff000008b60000   (  3520 KB)
       .init : 0xffff000008b60000 - 0xffff000008c50000   (   960 KB)
-      .data : 0xffff000008c50000 - 0xffff000008cfaa00   (   683 KB)
-       .bss : 0xffff000008cfaa00 - 0xffff000008d3be38   (   262 KB)
+      .data : 0xffff000008c50000 - 0xffff000008cfac00   (   683 KB)
+       .bss : 0xffff000008cfac00 - 0xffff000008d3be38   (   261 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -264,7 +264,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.459:1): initialized
+audit: type=2000 audit(0.451:1): initialized
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2016-06-16 21:56:25.619954111 -0700
+++ /tmp/foob	2016-06-16 21:56:25.619954111 -0700
@@ -17,15 +17,15 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2834132K/2988032K available (7660K kernel code, 682K rwdata, 3472K rodata, 960K init, 261K bss, 137516K reserved, 16384K cma-reserved)
+Memory: 2834132K/2988032K available (7660K kernel code, 683K rwdata, 3472K rodata, 960K init, 260K bss, 137516K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff0000087f0000   (  7616 KB)
     .rodata : 0xffff0000087f0000 - 0xffff000008b60000   (  3520 KB)
       .init : 0xffff000008b60000 - 0xffff000008c50000   (   960 KB)
-      .data : 0xffff000008c50000 - 0xffff000008cfaa00   (   683 KB)
-       .bss : 0xffff000008cfaa00 - 0xffff000008d3be38   (   262 KB)
+      .data : 0xffff000008c50000 - 0xffff000008cfac00   (   683 KB)
+       .bss : 0xffff000008cfac00 - 0xffff000008d3be38   (   261 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -54,7 +54,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0xe4e0: expected 0x2004f400
+boot misc3 0xe990: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -67,7 +67,7 @@
 console [tty0] enabled
 bootconsole [uart0] disabled
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-next-20160616+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Wed Jun 15 23:21:07 PDT 2016
+Linux version 4.7.0-rc3-next-20160617+ (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Thu Jun 16 21:51:00 PDT 2016
 Boot CPU: AArch64 Processor [411fd071]
 earlycon: uart0 at MMIO 0x0000000070006000 (options '115200n8')
 bootconsole [uart0] enabled
@@ -93,15 +93,15 @@
 Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
 Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2834132K/2988032K available (7660K kernel code, 682K rwdata, 3472K rodata, 960K init, 261K bss, 137516K reserved, 16384K cma-reserved)
+Memory: 2834132K/2988032K available (7660K kernel code, 683K rwdata, 3472K rodata, 960K init, 260K bss, 137516K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
       .text : 0xffff000008080000 - 0xffff0000087f0000   (  7616 KB)
     .rodata : 0xffff0000087f0000 - 0xffff000008b60000   (  3520 KB)
       .init : 0xffff000008b60000 - 0xffff000008c50000   (   960 KB)
-      .data : 0xffff000008c50000 - 0xffff000008cfaa00   (   683 KB)
-       .bss : 0xffff000008cfaa00 - 0xffff000008d3be38   (   262 KB)
+      .data : 0xffff000008c50000 - 0xffff000008cfac00   (   683 KB)
+       .bss : 0xffff000008cfac00 - 0xffff000008d3be38   (   261 KB)
     fixed   : 0xffff7dfffe7fd000 - 0xffff7dfffec00000   (  4108 KB)
     PCI I/O : 0xffff7dfffee00000 - 0xffff7dffffe00000   (    16 MB)
     vmemmap : 0xffff7e0000000000 - 0xffff800000000000   (  2048 GB maximum)
@@ -130,7 +130,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0xe4e0: expected 0x2004f400
+boot misc3 0xe990: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -208,7 +208,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.231:1): initialized
+audit: type=2000 audit(0.223:1): initialized
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-06-16 21:56:25.627954115 -0700
+++ /tmp/foob	2016-06-16 21:56:25.627954115 -0700
@@ -39,9 +39,9 @@
 	 #################################################################
 	 #################################################################
 	 ########################################
-	 2.8 MiB/s
+	 3 MiB/s
 done
-Bytes transferred = 7264992 (6edae0 hex)
+Bytes transferred = 7264112 (6ed770 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -52,7 +52,7 @@
 done
 Bytes transferred = 51511 (c937 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x6edae0 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x6ed770 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff936 ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-next-20160616-100112-g649a371 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Wed Jun 15 23:18:47 PDT 2016
+Linux version 4.7.0-rc3-next-20160617-100452-gce24ed9 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Thu Jun 16 21:48:39 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -68,7 +68,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c141bac0, node_mem_map eeffa000
+free_area_init_node: node 0, pgdat c141bb00, node_mem_map eeffa000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -89,9 +89,9 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0208000 - 0xc10b47d4   (15026 kB)
+      .text : 0xc0208000 - 0xc10b47f4   (15026 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc141db20   (1143 kB)
+      .data : 0xc1300000 - 0xc141db60   (1143 kB)
        .bss : 0xc141f000 - 0xc14748d8   ( 343 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -300,13 +300,13 @@
 sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
 tegra-pcie 3000.pcie-controller: 2x3 configuration
 tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
-mmc0: error -110 whilst initialising SD card
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
-tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB
+mmc0: error -110 whilst initialising SD card
+tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 Primary GPT is invalid, using alternate GPT.
  mmcblk1: p1 p2 p3 p4 p5
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -379,4 +379,4 @@
 Freeing unused kernel memory: 2048K (c1100000 - c1300000)
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.903571] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.903598] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-06-16 21:56:25.631954117 -0700
+++ /tmp/foob	2016-06-16 21:56:25.635954119 -0700
@@ -37,20 +37,20 @@
 	 #################################################################
 	 #################################################################
 	 ######################################################
-	 3.3 MiB/s
+	 3.1 MiB/s
 done
-Bytes transferred = 5560616 (54d928 hex)
+Bytes transferred = 5562352 (54dff0 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 2.9 MiB/s
 done
 Bytes transferred = 51511 (c937 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x54d928 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x54dff0 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff936 ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.7.0-rc3-next-20160616-100112-g649a371 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Wed Jun 15 23:18:40 PDT 2016
+Linux version 4.7.0-rc3-next-20160617-100452-gce24ed9 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Thu Jun 16 21:48:34 PDT 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -89,7 +89,7 @@
     lowmem  : 0xc0000000 - 0xf0000000   ( 768 MB)
     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
-      .text : 0xc0008000 - 0xc0b052e8   (11253 kB)
+      .text : 0xc0008000 - 0xc0b05318   (11253 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
       .data : 0xc0d00000 - 0xc0d5d8e0   ( 375 kB)
        .bss : 0xc0d5f000 - 0xc0dbaad4   ( 367 kB)
