Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2018-01-31 21:04:56.034876847 -0800
+++ /tmp/foob	2018-01-31 21:04:56.034876847 -0800
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0000000000 [0x411fd070]
-Linux version 4.15.0-next-20180131-348617-g537659c08a7d (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Tue Jan 30 22:09:13 PST 2018
+Linux version 4.15.0-next-20180201-348720-gf26e52e08ab8 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Jan 31 20:55:13 PST 2018
 Machine model: linux,dummy-virt
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -65,9 +65,9 @@
 DMI not present or invalid.
 NET: Registered protocol family 16
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.192:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.188:1): state=initialized audit_enabled=0 res=1
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ 00000000a9bf8b2c, 1 data @ 0000000035eb8c8b)
+vdso: 2 pages (1 code @ 00000000e5bace30, 1 data @ 000000006da9a7f9)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -176,7 +176,7 @@
 registered taskstats version 1
 Loading compiled-in X.509 certificates
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2018-01-31 06:16:21 UTC (1517379381)
+rtc-pl031 9010000.pl031: setting system clock to 2018-02-01 05:02:37 UTC (1517461357)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2018-01-31 21:04:56.070877006 -0800
+++ /tmp/foob	2018-01-31 21:04:56.070877006 -0800
@@ -27,11 +27,13 @@
 Tegra114 (Dalmore) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=7369c667-ff51-ec4a-29cd-baabf2fbe346 rootwait gpt debug earlyprintk init=/bin/bash'
 Tegra114 (Dalmore) # setenv ipaddr 192.168.100.100; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra114 (Dalmore) # tftpboot 0x81000000 zImage.multi_v7_defconfig
+Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'zImage.multi_v7_defconfig'.
 Load address: 0x81000000
-Loading: *#################################################################
+Loading: *EHCI timed out on TD - token=0x8008d80
+T #################################################################
 	 #################################################################
 	 #################################################################
 	 #################################################################
@@ -39,21 +41,22 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ####################################
-	 2.8 MiB/s
+	 #####################################
+	 1 MiB/s
 done
-Bytes transferred = 8159744 (7c8200 hex)
+Bytes transferred = 8167936 (7ca200 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
+Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.7 MiB/s
+	 2.5 MiB/s
 done
 Bytes transferred = 34502 (86c6 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x7c8200 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x7ca200 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff6c5 ... OK
@@ -61,7 +64,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-next-20180131-348617-g537659c08a7d (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Tue Jan 30 22:03:09 PST 2018
+Linux version 4.15.0-next-20180201-348720-gf26e52e08ab8 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Jan 31 20:49:02 PST 2018
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -84,7 +87,7 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=7369c667-ff51-ec4a-29cd-baabf2fbe346 rootwait gpt debug earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1992580K/2096128K available (11264K kernel code, 1570K rwdata, 4348K rodata, 2048K init, 405K bss, 38012K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1992580K/2096128K available (11264K kernel code, 1570K rwdata, 4352K rodata, 2048K init, 405K bss, 38012K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -353,7 +356,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-06-23 13:09:57 UTC (961765797)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-06-24 11:53:33 UTC (961847613)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -365,4 +368,4 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.769340] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.729340] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2018-01-31 21:04:56.082877059 -0800
+++ /tmp/foob	2018-01-31 21:04:56.086877076 -0800
@@ -38,20 +38,20 @@
 	 #################################################################
 	 #################################################################
 	 ##################################
-	 3.2 MiB/s
+	 3.1 MiB/s
 done
-Bytes transferred = 6218640 (5ee390 hex)
+Bytes transferred = 6220384 (5eea60 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 3.3 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 34502 (86c6 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5ee390 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5eea60 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff6c5 ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-next-20180131-28668-g537659c (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Tue Jan 30 22:02:18 PST 2018
+Linux version 4.15.0-next-20180201-28771-gf26e52e (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Jan 31 20:47:32 PST 2018
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -288,10 +288,10 @@
 usbhid: USB HID core driver
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
-mmc0: new high speed SDHC card at address aaaa
-mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
+mmc0: new high speed SDHC card at address aaaa
+mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB, chardev (247:0)
  mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
  mmcblk0: p1 p2 p3
@@ -343,7 +343,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-06-23 13:07:13 UTC (961765633)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-06-24 11:50:38 UTC (961847438)
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
@@ -361,4 +361,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.363057] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.396234] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2018-01-31 21:04:56.114877200 -0800
+++ /tmp/foob	2018-01-31 21:04:56.118877218 -0800
@@ -34,10 +34,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ####################################
-	 3.8 MiB/s
+	 #####################################
+	 4 MiB/s
 done
-Bytes transferred = 8159744 (7c8200 hex)
+Bytes transferred = 8167936 (7ca200 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -48,7 +48,7 @@
 done
 Bytes transferred = 66544 (103f0 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x7c8200 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x7ca200 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830133ef
@@ -56,7 +56,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-next-20180131-348617-g537659c08a7d (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Tue Jan 30 22:03:09 PST 2018
+Linux version 4.15.0-next-20180201-348720-gf26e52e08ab8 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Jan 31 20:49:02 PST 2018
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -79,7 +79,7 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=b1791faa-84e8-4e27-83a1-9fb0a77e8803 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1992472K/2096128K available (11264K kernel code, 1570K rwdata, 4348K rodata, 2048K init, 405K bss, 38120K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1992472K/2096128K available (11264K kernel code, 1570K rwdata, 4352K rodata, 2048K init, 405K bss, 38120K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -325,20 +325,20 @@
 +3.3V_LP0: supplied by +3.3V_SYS
 +5V_HDMI_CON: supplied by +5V_SYS
 +5V_SATA: supplied by +5V_SYS
+mmc0: new high speed MMC card at address 0001
 +12V_SATA: supplied by +VDD_MUX
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
  usb2-0: usb2-0 supply vbus not found, using dummy regulator
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
  usb2-1: usb2-1 supply vbus not found, using dummy regulator
-mmc0: new high speed MMC card at address 0001
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB, chardev (236:0)
 tegra-pcie 1003000.pcie: 2x1, 1x1 configuration
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 tegra-pcie 1003000.pcie: Failed to get supply 'avddio-pex': -517
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB, chardev (236:0)
-tegra-hdmi 54280000.hdmi: failed to get PLL regulator
  mmcblk0: p1
+tegra-hdmi 54280000.hdmi: failed to get PLL regulator
 as3722-sd2: supplied by +5V_SYS
 as3722-sd3: supplied by +5V_SYS
 as3722-sd4: supplied by +5V_SYS
@@ -398,8 +398,8 @@
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4 may corrupt adjacent RW1C bits
 pci 0000:00:02.0: enabling Extended Tags
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x88 may corrupt adjacent RW1C bits
-pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x52 may corrupt adjacent RW1C bits
 ata1: SATA link down (SStatus 0 SControl 300)
+pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x52 may corrupt adjacent RW1C bits
 pci 0000:00:02.0: PME# supported from D0 D1 D2 D3hot D3cold
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4c may corrupt adjacent RW1C bits
 PCI: bus0: Fast back to back transfers disabled
@@ -443,7 +443,7 @@
 pcieport 0000:00:02.0: Signaling PME with IRQ 396
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
-r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0x0e017d5b, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 397
+r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xcb2dba36, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 397
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
@@ -455,7 +455,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:04:11 UTC (946685051)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:04:12 UTC (946685052)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2018-01-31 21:04:56.138877306 -0800
+++ /tmp/foob	2018-01-31 21:04:56.138877306 -0800
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+MMC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -33,20 +33,20 @@
 	 #################################################################
 	 #################################################################
 	 ##################################
-	 3.8 MiB/s
+	 4 MiB/s
 done
-Bytes transferred = 6218640 (5ee390 hex)
+Bytes transferred = 6220384 (5eea60 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.7 MiB/s
+	 2 MiB/s
 done
 Bytes transferred = 66544 (103f0 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5ee390 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5eea60 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830133ef
@@ -54,7 +54,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-next-20180131-28668-g537659c (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Tue Jan 30 22:02:18 PST 2018
+Linux version 4.15.0-next-20180201-28771-gf26e52e (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Jan 31 20:47:32 PST 2018
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -388,8 +388,8 @@
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x52 may corrupt adjacent RW1C bits
 pci 0000:00:01.0: PME# supported from D0 D1 D2 D3hot D3cold
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x4c may corrupt adjacent RW1C bits
-pci 0000:00:02.0: [10de:0e13] type 01 class 0x060400
 ata1: SATA link down (SStatus 0 SControl 300)
+pci 0000:00:02.0: [10de:0e13] type 01 class 0x060400
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4 may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4 may corrupt adjacent RW1C bits
 pci 0000:00:02.0: enabling Extended Tags
@@ -438,7 +438,7 @@
 pcieport 0000:00:02.0: Signaling PME with IRQ 396
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
-r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0x8ef8ab92, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 397
+r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0x96a2d71f, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 397
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig%tegra-fw_log.txt
--- /tmp/fooa	2018-01-31 21:04:56.146877341 -0800
+++ /tmp/foob	2018-01-31 21:04:56.150877358 -0800
@@ -33,9 +33,9 @@
 	 #################################################################
 	 #################################################################
 	 #############################################
-	 3.9 MiB/s
+	 3.5 MiB/s
 done
-Bytes transferred = 6384624 (616bf0 hex)
+Bytes transferred = 6383832 (6168d8 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -46,7 +46,7 @@
 done
 Bytes transferred = 66544 (103f0 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x616bf0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x6168d8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830133ef
@@ -54,7 +54,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-next-20180131-230376-g537659c (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Tue Jan 30 22:03:05 PST 2018
+Linux version 4.15.0-next-20180201-230479-gf26e52e (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Jan 31 20:48:34 PST 2018
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -382,8 +382,8 @@
 tegra-pcie 1003000.pcie: probing port 0, using 2 lanes
 tegra-pcie 1003000.pcie: Slot present pin change, signature: 00000008
 tegra-pcie 1003000.pcie: probing port 1, using 1 lanes
-ata1: SATA link down (SStatus 0 SControl 300)
 tegra-pcie 1003000.pcie: Slot present pin change, signature: 00000000
+ata1: SATA link down (SStatus 0 SControl 300)
 tegra-pcie 1003000.pcie: PCI host bridge to bus 0000:00
 pci_bus 0000:00: root bus resource [io  0x0000-0xffff]
 pci_bus 0000:00: root bus resource [mem 0x13000000-0x1fffffff]
@@ -446,7 +446,7 @@
 pcieport 0000:00:02.0: Signaling PME with IRQ 396
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
-r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0x916e758d, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 397
+r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0x659202b6, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 397
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
@@ -458,7 +458,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:01:57 UTC (946684917)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:01:58 UTC (946684918)
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2018-01-31 21:04:56.178877482 -0800
+++ /tmp/foob	2018-01-31 21:04:56.178877482 -0800
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -42,10 +42,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ####################################
+	 #####################################
 	 3.3 MiB/s
 done
-Bytes transferred = 8159744 (7c8200 hex)
+Bytes transferred = 8167936 (7ca200 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.104
@@ -56,7 +56,7 @@
 done
 Bytes transferred = 67482 (1079a hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x7c8200 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x7ca200 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013799
@@ -64,7 +64,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-next-20180131-348617-g537659c08a7d (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Tue Jan 30 22:03:09 PST 2018
+Linux version 4.15.0-next-20180201-348720-gf26e52e08ab8 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Jan 31 20:49:02 PST 2018
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -88,7 +88,7 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=4d36e82a-b3c0-4b4b-bfcd-afe5217ac023 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993488K/2097148K available (11264K kernel code, 1570K rwdata, 4348K rodata, 2048K init, 405K bss, 38124K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1993488K/2097148K available (11264K kernel code, 1570K rwdata, 4352K rodata, 2048K init, 405K bss, 38124K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -308,7 +308,9 @@
 mmc0: Invalid maximum block size, assuming 512 bytes
 mmc0: SDHCI controller on 700b0600.sdhci [700b0600.sdhci] using ADMA
 ledtrig-cpu: registered to indicate activity on CPUs
+elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
 usbcore: registered new interface driver usbhid
+input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 usbhid: USB HID core driver
 NET: Registered protocol family 10
 Segment Routing with IPv6
@@ -324,29 +326,27 @@
 Registering SWP/SWPB emulation handler
 Loading compiled-in X.509 certificates
 +3.3V_RUN: supplied by +3.3V_SYS
-elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
-input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 +VDD_LED: supplied by +VDD_MUX
 +3.3V_AVDD_HDMI_AP_GATED: supplied by +3.3V_RUN
 +1.8V_RUN_CAM: supplied by +3.3V_RUN
 +VDDIO_SDMMC3: supplied by +3.3V_RUN
 +1.8V_RUN_VPP_FUSE: supplied by +3.3V_RUN
-mmc0: new high speed MMC card at address 0001
 +5V_VDD_TS_SW: supplied by +5V_SYS
-mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
 +5V_USB_HS: supplied by +5V_SYS
-mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
+mmc0: new high speed MMC card at address 0001
 +5V_USB_SS: supplied by +5V_SYS
-mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
-mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB, chardev (236:0)
+mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
 +3.3V_PANEL: supplied by +3.3V_RUN
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
+mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
 +3.3V_LP0: supplied by +3.3V_SYS
+mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
-elan_i2c 1-0015: failed to read report data: -121
 +5V_HDMI_CON: supplied by +5V_SYS
+mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB, chardev (236:0)
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
  usb3-1: usb3-1 supply vbus not found, using dummy regulator
+elan_i2c 1-0015: failed to read report data: -121
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
 sdhci-tegra 700b0000.sdhci: allocated mmc-pwrseq
@@ -365,7 +365,7 @@
 panel-simple panel: panel supply power not found, using dummy regulator
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
 tegra-sor 54540000.sor: failed to probe output: -517
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:23 UTC (946684823)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:01:52 UTC (946684912)
 panel-simple panel: panel supply power not found, using dummy regulator
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +1.05V_RUN: disabling
@@ -385,11 +385,6 @@
 +5V_HDMI_CON: disabling
 EXT4-fs (mmcblk0p3): couldn't mount as ext3 due to feature incompatibilities
 EXT4-fs (mmcblk0p3): couldn't mount as ext2 due to feature incompatibilities
-EXT4-fs (mmcblk0p3): recovery complete
-EXT4-fs (mmcblk0p3): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext4 filesystem) on device 179:3.
-devtmpfs: mounted
-Freeing unused kernel memory: 2048K
 mmc2: new high speed SDHC card at address 0007
 mmcblk2: mmc2:0007 SD08G 7.42 GiB 
 GPT:Primary header alternate_lba != Alt. header my_lba
@@ -400,6 +395,11 @@
  mmcblk2: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
+EXT4-fs (mmcblk0p3): recovery complete
+EXT4-fs (mmcblk0p3): mounted filesystem with ordered data mode. Opts: (null)
+VFS: Mounted root (ext4 filesystem) on device 179:3.
+devtmpfs: mounted
+Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2018-01-31 21:04:56.186877517 -0800
+++ /tmp/foob	2018-01-31 21:04:56.190877535 -0800
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -43,18 +43,18 @@
 	 ##################################
 	 3.3 MiB/s
 done
-Bytes transferred = 6218640 (5ee390 hex)
+Bytes transferred = 6220384 (5eea60 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.104
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.2 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 67482 (1079a hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5ee390 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5eea60 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013799
@@ -62,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-next-20180131-28668-g537659c (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Tue Jan 30 22:02:18 PST 2018
+Linux version 4.15.0-next-20180201-28771-gf26e52e (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Jan 31 20:47:32 PST 2018
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -292,18 +292,18 @@
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
 tegra30-i2s 70301100.i2s: DMA channels sourced from device 70300000.ahub
-mmc0: new high speed MMC card at address 0001
 max98090 0-0010: MAX98090 REVID=0x43
-mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
 max98090 0-0010: use default 2.8v micbias
-mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
-mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
-mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB, chardev (247:0)
 tegra-snd-max98090 sound: HiFi <-> 70301100.i2s mapping ok
+mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
 input: GoogleNyanBig Headphones as /devices/soc0/sound/sound/card1/input1
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
+mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
 input: GoogleNyanBig Mic Jack as /devices/soc0/sound/sound/card1/input2
+mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
+mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB, chardev (247:0)
 NET: Registered protocol family 10
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 Segment Routing with IPv6
 mip6: Mobile IPv6
 sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
@@ -364,8 +364,8 @@
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
-+3.3V_AVDD_HDMI_AP_GATED: disabling
 cfg80211: failed to load regulatory.db
++3.3V_AVDD_HDMI_AP_GATED: disabling
 +1.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
 +1.05V_RUN_CAM_REAR: disabling
@@ -387,14 +387,14 @@
 GPT:3014591 != 15564799
 GPT:Primary header thinks Alt. header is not at the end of the disk.
 GPT:3014591 != 15564799
+tegra-sor 54540000.sor: failed to probe output: -517
 GPT: Use GNU Parted to correct GPT errors.
+input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input4
  mmcblk2: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
-tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
 tegra-sor 54540000.sor: failed to probe output: -517
-input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input4
-panel-simple panel: panel supply power not found, using dummy regulator
 EXT4-fs (mmcblk0p3): couldn't mount as ext3 due to feature incompatibilities
+panel-simple panel: panel supply power not found, using dummy regulator
 EXT4-fs (mmcblk0p3): recovery complete
 EXT4-fs (mmcblk0p3): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:3.
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig%tegra-fw_log.txt
--- /tmp/fooa	2018-01-31 21:04:56.214877640 -0800
+++ /tmp/foob	2018-01-31 21:04:56.214877640 -0800
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -43,7 +43,7 @@
 	 #############################################
 	 3.3 MiB/s
 done
-Bytes transferred = 6384624 (616bf0 hex)
+Bytes transferred = 6383832 (6168d8 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.104
@@ -54,7 +54,7 @@
 done
 Bytes transferred = 67482 (1079a hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x616bf0 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x6168d8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013799
@@ -62,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-next-20180131-230376-g537659c (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Tue Jan 30 22:03:05 PST 2018
+Linux version 4.15.0-next-20180201-230479-gf26e52e (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Jan 31 20:48:34 PST 2018
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -292,18 +292,18 @@
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
 tegra30-i2s 70301100.i2s: DMA channels sourced from device 70300000.ahub
-mmc0: new high speed MMC card at address 0001
 max98090 0-0010: MAX98090 REVID=0x43
-mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
 max98090 0-0010: use default 2.8v micbias
-mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
-mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
-mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB, chardev (247:0)
 tegra-snd-max98090 sound: HiFi <-> 70301100.i2s mapping ok
+mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
 input: GoogleNyanBig Headphones as /devices/soc0/sound/sound/card1/input1
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
+mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
 input: GoogleNyanBig Mic Jack as /devices/soc0/sound/sound/card1/input2
+mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
+mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB, chardev (247:0)
 NET: Registered protocol family 10
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 Segment Routing with IPv6
 mip6: Mobile IPv6
 sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
@@ -388,9 +388,9 @@
 ALSA device list:
   #1: GoogleNyanBig
 tegra-sor 54540000.sor: failed to probe output: -517
-EXT4-fs (mmcblk0p3): couldn't mount as ext3 due to feature incompatibilities
 panel-simple panel: panel supply power not found, using dummy regulator
 input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input4
+EXT4-fs (mmcblk0p3): couldn't mount as ext3 due to feature incompatibilities
 usb 1-2: new high-speed USB device number 2 using tegra-xusb
 mmc2: new high speed SDHC card at address 0007
 mmcblk2: mmc2:0007 SD08G 7.42 GiB 
@@ -402,25 +402,25 @@
  mmcblk2: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
-uvcvideo: Found UVC 1.00 device HD WebCam (1bcf:2c67)
-input: HD WebCam: HD WebCam as /devices/soc0/70090000.usb/usb1/1-2/1-2:1.0/input/input5
-tegra-sor 54540000.sor: failed to probe output: -517
-panel-simple panel: panel supply power not found, using dummy regulator
 EXT4-fs (mmcblk0p3): recovery complete
 EXT4-fs (mmcblk0p3): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:3.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K
-alloc_contig_range: [fbc79, fbc7a) PFNs busy
-alloc_contig_range: [fbc79, fbc7a) PFNs busy
+alloc_contig_range: [fbc77, fbc78) PFNs busy
+bash: cannot set terminal process group [    2.587805] alloc_contig_range: [fbc78, fbc79) PFNs busy
+(-1): Inappropriate ioctl for device
+ba[    2.596510] alloc_contig_range: [fbc79, fbc7a) PFNs busy
+sh: no job control in this shell
+alloc_contig_range: [fbc7a, fbc7b) PFNs busy
 alloc_contig_range: [fbc7b, fbc7c) PFNs busy
 alloc_contig_range: [fbc7c, fbc7d) PFNs busy
 alloc_contig_range: [fbc7d, fbc7e) PFNs busy
 alloc_contig_range: [fbc7e, fbc7f) PFNs busy
-alloc_contig_range: [fbc7f, fbc80) PFNs busy
+root@(none):/# [    2.634891] alloc_contig_range: [fbc7f, fbc80) PFNs busy
 alloc_contig_range: [fbca1, fbca2) PFNs busy
-alloc_contig_range: [fbca2, fbca3) PFNs busy
-alloc_contig_range: [fbca3, fbca4) PFNs busy
-bash: cannot set terminal process group (-1): Inappropriate ioctl for device
-bash: no job control in this shell
-root@(none):/# [    3.077621] usb 1-3: new high-speed USB device number 3 using tegra-xusb
+uvcvideo: Found UVC 1.00 device HD WebCam (1bcf:2c67)
+input: HD WebCam: HD WebCam as /devices/soc0/70090000.usb/usb1/1-2/1-2:1.0/input/input5
+tegra-sor 54540000.sor: failed to probe output: -517
+panel-simple panel: panel supply power not found, using dummy regulator
+usb 1-3: new high-speed USB device number 3 using tegra-xusb
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2018-01-31 21:04:56.230877711 -0800
+++ /tmp/foob	2018-01-31 21:04:56.234877728 -0800
@@ -32,7 +32,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1800964 us
+Bootloader-Cpu Init at (time stamp): 1801455 us
 
 Pinmux changes applied in kernel way
 
@@ -60,25 +60,25 @@
 The proc BoardInfo: 0x0176:0x0001:0x01:0x43:0x00
 FastbootAddAtag: setting ATAG (0)
 
-Board Id Read Time: 4 us
+Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1800937  us
-Bootloader (AosInit) time: 734251 us
-Bootloader Main Init time: 180083 us
-BlockDevMgrInit time: 157533 us
-PartMgrInit time: 8 us
+Miscellaneous avp time: 1801429  us
+Bootloader (AosInit) time: 733457 us
+Bootloader Main Init time: 180114 us
+BlockDevMgrInit time: 157534 us
+PartMgrInit time: 9 us
 StorMgrInit time: 30 us
-FsMgrInit time: 3 us
-Partition Table load: 76691 us 
-Display init time: 574789 us
+FsMgrInit time: 2 us
+Partition Table load: 77100 us 
+Display init time: 574593 us
 Kernel read time: 1577 msec
 Load Kernel Size:   20895744 Bytes
 Bl Kernel ReadSpeed: 13250000 Bytes/sec
-Pre-Kernel Jump time: 4410471 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 9606731 us
+Pre-Kernel Jump time: 4410696 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 9605792 us
 Booting Linux on physical CPU 0x0000000000 [0x4e0f0000]
-Linux version 4.15.0-next-20180131-348617-g537659c08a7d (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Tue Jan 30 22:09:13 PST 2018
+Linux version 4.15.0-next-20180201-348720-gf26e52e08ab8 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Jan 31 20:55:13 PST 2018
 Machine model: NVIDIA Tegra132 Norrin
 Memory limited to 1734MB
 efi: Getting EFI parameters from FDT:
@@ -140,9 +140,9 @@
 DMI not present or invalid.
 NET: Registered protocol family 16
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.136:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.140:1): state=initialized audit_enabled=0 res=1
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ 000000004a8f2ff8, 1 data @ 00000000f1dd4229)
+vdso: 2 pages (1 code @ 00000000b5181e2e, 1 data @ 0000000001d5e413)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -263,8 +263,8 @@
 NET: Registered protocol family 17
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 9pnet: Installing 9P2000 support
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 Key type dns_resolver registered
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 registered taskstats version 1
 Loading compiled-in X.509 certificates
Diff for ./tegra186-p2771-0000/tegra186-p2771-0000/defconfig%jetson-tx2_log.txt
Binary files /tmp/fooa and /tmp/foob differ
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2018-01-31 21:04:56.286877957 -0800
+++ /tmp/foob	2018-01-31 21:04:56.290877976 -0800
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -47,21 +47,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ####################################
-	 4 MiB/s
+	 #####################################
+	 3.5 MiB/s
 done
-Bytes transferred = 8159744 (7c8200 hex)
+Bytes transferred = 8167936 (7ca200 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 21686 (54b6 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x7c8200 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x7ca200 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024104b5
@@ -69,7 +69,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-next-20180131-348617-g537659c08a7d (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Tue Jan 30 22:03:09 PST 2018
+Linux version 4.15.0-next-20180201-348720-gf26e52e08ab8 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Jan 31 20:49:02 PST 2018
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -91,7 +91,7 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=11eb23c1-01 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 953404K/1048576K available (11264K kernel code, 1570K rwdata, 4348K rodata, 2048K init, 405K bss, 29636K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 953404K/1048576K available (11264K kernel code, 1570K rwdata, 4352K rodata, 2048K init, 405K bss, 29636K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -270,7 +270,7 @@
 igb: Copyright (c) 2007-2014 Intel Corporation.
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:01:00.0: enabling device (0140 -> 0143)
-r8169 0000:01:00.0 eth0: RTL8168d/8111d at 0x4a59fa4c, 00:01:c0:13:7d:63, XID 081000c0 IRQ 299
+r8169 0000:01:00.0 eth0: RTL8168d/8111d at 0x77c897d6, 00:01:c0:13:7d:63, XID 081000c0 IRQ 299
 r8169 0000:01:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 pegasus: v0.9.3 (2013/04/25), Pegasus/Pegasus II USB Ethernet driver
 usbcore: registered new interface driver pegasus
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2018-01-31 21:04:56.306878046 -0800
+++ /tmp/foob	2018-01-31 21:04:56.310878064 -0800
@@ -46,9 +46,9 @@
 	 #################################################################
 	 #################################################################
 	 ##################################
-	 3.7 MiB/s
+	 4.1 MiB/s
 done
-Bytes transferred = 6218640 (5ee390 hex)
+Bytes transferred = 6220384 (5eea60 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -59,7 +59,7 @@
 done
 Bytes transferred = 21686 (54b6 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x5ee390 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x5eea60 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024104b5
@@ -67,7 +67,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-next-20180131-28668-g537659c (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Tue Jan 30 22:02:18 PST 2018
+Linux version 4.15.0-next-20180201-28771-gf26e52e (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Jan 31 20:47:32 PST 2018
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -261,7 +261,7 @@
 igb: Copyright (c) 2007-2014 Intel Corporation.
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:01:00.0: enabling device (0140 -> 0143)
-r8169 0000:01:00.0 eth0: RTL8168d/8111d at 0x84575651, 00:01:c0:13:7d:63, XID 081000c0 IRQ 299
+r8169 0000:01:00.0 eth0: RTL8168d/8111d at 0x683749d4, 00:01:c0:13:7d:63, XID 081000c0 IRQ 299
 r8169 0000:01:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 pegasus: v0.9.3 (2013/04/25), Pegasus/Pegasus II USB Ethernet driver
 usbcore: registered new interface driver pegasus
@@ -335,8 +335,8 @@
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
-tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
+tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 291, io mem 0xc5000000
  mmcblk1: p1 p2
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
@@ -346,8 +346,8 @@
 rtc-em3027 2-0056: hctosys: unable to read the hardware clock
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
 hub 1-1:1.0: USB hub found
-cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 hub 1-1:1.0: 4 ports detected
+cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
 ALSA device list:
 cfg80211: failed to load regulatory.db
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig%jetson-tx1_log.txt
--- /tmp/fooa	2018-01-31 21:04:56.314878081 -0800
+++ /tmp/foob	2018-01-31 21:04:56.318878099 -0800
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 727040 us
-Basic modules init took 349085 us
-Sec Bootdevice Read Time = 204 ms, Read Size = 8459 KB
-Next stage binary read took 15533 us
-Carveout took 252671 us
-CPU initialization took 93149 us
-Total time taken by TegraBoot 710438 us
+Tegraboot started after 743142 us
+Basic modules init took 348676 us
+Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
+Next stage binary read took 14368 us
+Carveout took 252663 us
+CPU initialization took 92465 us
+Total time taken by TegraBoot 708172 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.568] LPDDR4 Training: Number of tables = 10
+64b[0001.582] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig%jetson-tx1
-17527296 bytes read in 884 ms (18.9 MiB/s)
+17527296 bytes read in 894 ms (18.7 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
 44914 bytes read in 454 ms (95.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 497 ms (3.1 MiB/s)
+1627787 bytes read in 495 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0000000000 [0x411fd071]
-Linux version 4.15.0-next-20180131-28668-g537659c (jonathanh@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Tue Jan 30 22:06:32 PST 2018
+Linux version 4.15.0-next-20180201-28771-gf26e52e (jonathanh@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Jan 31 20:51:35 PST 2018
 Machine model: NVIDIA Tegra210 P2371 (P2530/P2595) reference design
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -223,7 +223,7 @@
 audit: initializing netlink subsys (disabled)
 audit: type=2000 audit(0.284:1): state=initialized audit_enabled=0 res=1
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ 000000007fb813df, 1 data @ 00000000396aba3a)
+vdso: 2 pages (1 code @ 00000000d8e6fc2a, 1 data @ 000000008a2c6757)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2018-01-31 21:04:56.334878169 -0800
+++ /tmp/foob	2018-01-31 21:04:56.338878187 -0800
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 749921 us
-Basic modules init took 348481 us
-Sec Bootdevice Read Time = 203 ms, Read Size = 8459 KB
-Next stage binary read took 14365 us
-Carveout took 253494 us
-CPU initialization took 92432 us
-Total time taken by TegraBoot 708772 us
+Tegraboot started after 754175 us
+Basic modules init took 348317 us
+Sec Bootdevice Read Time = 204 ms, Read Size = 8459 KB
+Next stage binary read took 15935 us
+Carveout took 252707 us
+CPU initialization took 92444 us
+Total time taken by TegraBoot 709403 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.589] LPDDR4 Training: Number of tables = 10
+64b[0001.594] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,9 +117,9 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-17396224 bytes read in 859 ms (19.3 MiB/s)
+17396224 bytes read in 865 ms (19.2 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-44914 bytes read in 455 ms (95.7 KiB/s)
+44914 bytes read in 454 ms (95.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
 1627787 bytes read in 494 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0000000000 [0x411fd071]
-Linux version 4.15.0-next-20180131-348617-g537659c08a7d (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Tue Jan 30 22:09:13 PST 2018
+Linux version 4.15.0-next-20180201-348720-gf26e52e08ab8 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Jan 31 20:55:13 PST 2018
 Machine model: NVIDIA Tegra210 P2371 (P2530/P2595) reference design
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -223,7 +223,7 @@
 audit: initializing netlink subsys (disabled)
 audit: type=2000 audit(0.284:1): state=initialized audit_enabled=0 res=1
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ 000000008863994d, 1 data @ 000000004e3889f1)
+vdso: 2 pages (1 code @ 00000000a07d09ae, 1 data @ 00000000d3d5b2a9)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
Diff for ./tegra210-p2371-2180/tegra210-p2371-2180/defconfig%jetson-tx1_log.txt
Binary files /tmp/fooa and /tmp/foob differ
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2018-01-31 21:04:56.362878293 -0800
+++ /tmp/foob	2018-01-31 21:04:56.362878293 -0800
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0000000000 [0x411fd071]
-Linux version 4.15.0-next-20180131-348617-g537659c08a7d (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Tue Jan 30 22:09:13 PST 2018
+Linux version 4.15.0-next-20180201-348720-gf26e52e08ab8 (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Jan 31 20:55:13 PST 2018
 Machine model: Google Pixel C
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
@@ -99,7 +99,7 @@
 audit: initializing netlink subsys (disabled)
 audit: type=2000 audit(0.180:1): state=initialized audit_enabled=0 res=1
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ 00000000928ece51, 1 data @ 00000000f153ad05)
+vdso: 2 pages (1 code @ 00000000efb19f0e, 1 data @ 000000004cb954dc)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -256,7 +256,7 @@
 mmcblk0boot0: mmc0:0001 BGND3R partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB, chardev (240:0)
-max77686-rtc max77620-rtc: setting system clock to 2001-11-11 11:39:01 UTC (1005478741)
+max77686-rtc max77620-rtc: setting system clock to 2001-11-12 10:25:17 UTC (1005560717)
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
 PP1200_AVDD: disabling
 tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2018-01-31 21:04:56.386878398 -0800
+++ /tmp/foob	2018-01-31 21:04:56.386878398 -0800
@@ -39,21 +39,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 ####################################
-	 2.8 MiB/s
+	 #####################################
+	 2.9 MiB/s
 done
-Bytes transferred = 8159744 (7c8200 hex)
+Bytes transferred = 8167936 (7ca200 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.7 MiB/s
+	 1.1 MiB/s
 done
 Bytes transferred = 51470 (c90e hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x7c8200 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x7ca200 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90d ... OK
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-next-20180131-348617-g537659c08a7d (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Tue Jan 30 22:03:09 PST 2018
+Linux version 4.15.0-next-20180201-348720-gf26e52e08ab8 (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Jan 31 20:49:02 PST 2018
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -83,7 +83,7 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=edbe1840-3633-c751-7993-b0ac834f81e0 rootwait gpt debug noinitrd earlyprintk init=/bin/bash
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1992508K/2096128K available (11264K kernel code, 1570K rwdata, 4348K rodata, 2048K init, 405K bss, 38084K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1992508K/2096128K available (11264K kernel code, 1570K rwdata, 4352K rodata, 2048K init, 405K bss, 38084K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -376,7 +376,7 @@
 pcieport 0000:00:01.0: Signaling PME with IRQ 375
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:01:00.0: enabling device (0140 -> 0143)
-r8169 0000:01:00.0 eth0: RTL8168evl/8111evl at 0x29f97731, 02:04:4b:01:60:a6, XID 0c900800 IRQ 376
+r8169 0000:01:00.0 eth0: RTL8168evl/8111evl at 0xf8efa659, 02:04:4b:01:60:a6, XID 0c900800 IRQ 376
 r8169 0000:01:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
@@ -407,5 +407,5 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.094036] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.094010] usb 2-1: new high-speed USB device number 2 using tegra-ehci
 asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0f:67:60
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2018-01-31 21:04:56.398878451 -0800
+++ /tmp/foob	2018-01-31 21:04:56.402878468 -0800
@@ -38,20 +38,20 @@
 	 #################################################################
 	 #################################################################
 	 ##################################
-	 2.8 MiB/s
+	 3.1 MiB/s
 done
-Bytes transferred = 6218640 (5ee390 hex)
+Bytes transferred = 6220384 (5eea60 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.1 MiB/s
+	 195.3 KiB/s
 done
 Bytes transferred = 51470 (c90e hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5ee390 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5eea60 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90d ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-next-20180131-28668-g537659c (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Tue Jan 30 22:02:18 PST 2018
+Linux version 4.15.0-next-20180201-28771-gf26e52e (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Jan 31 20:47:32 PST 2018
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -366,7 +366,7 @@
 pcieport 0000:00:01.0: Signaling PME with IRQ 375
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:01:00.0: enabling device (0140 -> 0143)
-r8169 0000:01:00.0 eth0: RTL8168evl/8111evl at 0x094b3ecd, 02:04:4b:01:60:a6, XID 0c900800 IRQ 376
+r8169 0000:01:00.0 eth0: RTL8168evl/8111evl at 0x34ee4b9f, 02:04:4b:01:60:a6, XID 0c900800 IRQ 376
 r8169 0000:01:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
@@ -403,4 +403,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.358574] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.378589] usb 2-1: new high-speed USB device number 2 using tegra-ehci
