Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2016-12-13 20:43:07.281325029 -0800
+++ /tmp/foob	2016-12-13 20:43:07.285325047 -0800
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-next-20161213-165275-gc791961 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Dec 12 22:08:19 PST 2016
+Linux version 4.9.0-next-20161214-165388-g9aa17b2 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Dec 13 20:36:55 PST 2016
 Boot CPU: AArch64 Processor [411fd070]
 Memory limited to 1024MB
 efi: Getting EFI parameters from FDT:
@@ -101,7 +101,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.680:1): initialized
+audit: type=2000 audit(0.692:1): initialized
 workingset: timestamp_bits=46 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -167,7 +167,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2016-12-13 06:09:01 UTC (1481609341)
+rtc-pl031 9010000.pl031: setting system clock to 2016-12-14 04:37:34 UTC (1481690254)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-12-13 20:43:07.289325064 -0800
+++ /tmp/foob	2016-12-13 20:43:07.289325064 -0800
@@ -38,21 +38,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #################################################
-	 3 MiB/s
+	 ##################################################
+	 2.9 MiB/s
 done
-Bytes transferred = 7396656 (70dd30 hex)
+Bytes transferred = 7398800 (70e590 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.8 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x70dd30 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x70e590 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -60,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-next-20161213-140842-gc791961 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Dec 12 22:05:23 PST 2016
+Linux version 4.9.0-next-20161214-140955-g9aa17b2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Dec 13 20:34:21 PST 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -71,7 +71,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c142a5c0, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c142a640, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -84,7 +84,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1994916K/2096128K available (9216K kernel code, 1200K rwdata, 4136K rodata, 2048K init, 503K bss, 35676K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1994916K/2096128K available (9216K kernel code, 1200K rwdata, 4140K rodata, 2048K init, 503K bss, 35676K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -94,8 +94,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc142c0c0   (1201 kB)
-       .bss : 0xc142e000 - 0xc14abc70   ( 504 kB)
+      .data : 0xc1300000 - 0xc142c140   (1201 kB)
+       .bss : 0xc142e000 - 0xc14abc90   ( 504 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -305,20 +305,20 @@
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
 usb1_vbus: supplied by vdd-sys-5v0
-usb2_vbus: supplied by vdd-sys-5v0
 mmc0: new high speed SDHC card at address aaaa
+usb2_vbus: supplied by vdd-sys-5v0
 VDD_5V0_HDMI_CON: supplied by vdd-sys-5v0
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
 mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
- mmcblk0: p1 p2 p3
-Console: switching to colour frame buffer device 240x75
 mmc1: new high speed MMC card at address 0001
+ mmcblk0: p1 p2 p3
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
  mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
+Console: switching to colour frame buffer device 240x75
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 tegra-ehci 7d008000.usb: EHCI Host Controller
@@ -328,7 +328,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-10-06 17:36:58 UTC (970853818)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-10-07 16:05:50 UTC (970934750)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -340,4 +340,4 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.479732] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    2.579742] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2016-12-13 20:43:07.297325097 -0800
+++ /tmp/foob	2016-12-13 20:43:07.297325097 -0800
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra114 (Dalmore) # 
 Tegra114 (Dalmore) # usb reset
 (Re)start USB...
@@ -37,9 +37,9 @@
 	 #################################################################
 	 #################################################################
 	 ##################################################
-	 3.3 MiB/s
+	 3.2 MiB/s
 done
-Bytes transferred = 5503824 (53fb50 hex)
+Bytes transferred = 5504608 (53fe60 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
@@ -50,7 +50,7 @@
 done
 Bytes transferred = 35044 (88e4 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x53fb50 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x53fe60 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8e3 ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-next-20161213-140842-gc791961 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Dec 12 22:05:05 PST 2016
+Linux version 4.9.0-next-20161214-140955-g9aa17b2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Dec 13 20:33:52 PST 2016
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -93,7 +93,7 @@
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
       .data : 0xc0d00000 - 0xc0d5fd00   ( 384 kB)
-       .bss : 0xc0d61000 - 0xc0dc5388   ( 401 kB)
+       .bss : 0xc0d61000 - 0xc0dc53a8   ( 401 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -133,7 +133,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?[    0.503230] hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
+No ATAGs?[    0.524405] hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 8 bytes.
 tegra-mc 70019000.memory-controller: no memory timings for RAM code 3 registered
 dvdd_ts: Failed to request enable GPIO61: -517
@@ -321,7 +321,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-10-06 17:34:07 UTC (970853647)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-10-07 16:02:55 UTC (970934575)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -335,4 +335,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.743008] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# 
\ No newline at end of file
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-12-13 20:43:07.305325129 -0800
+++ /tmp/foob	2016-12-13 20:43:07.309325147 -0800
@@ -33,21 +33,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #################################################
-	 3.5 MiB/s
+	 ##################################################
+	 3.2 MiB/s
 done
-Bytes transferred = 7396656 (70dd30 hex)
+Bytes transferred = 7398800 (70e590 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3 MiB/s
+	 3.2 MiB/s
 done
 Bytes transferred = 67133 (1063d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x70dd30 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x70e590 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301363c
@@ -55,7 +55,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-next-20161213-140842-gc791961 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Dec 12 22:05:23 PST 2016
+Linux version 4.9.0-next-20161214-140955-g9aa17b2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Dec 13 20:34:21 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -66,7 +66,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c142a5c0, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c142a640, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -79,7 +79,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1994796K/2096128K available (9216K kernel code, 1200K rwdata, 4136K rodata, 2048K init, 503K bss, 35796K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1994796K/2096128K available (9216K kernel code, 1200K rwdata, 4140K rodata, 2048K init, 503K bss, 35796K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -89,8 +89,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc142c0c0   (1201 kB)
-       .bss : 0xc142e000 - 0xc14abc70   ( 504 kB)
+      .data : 0xc1300000 - 0xc142c140   (1201 kB)
+       .bss : 0xc142e000 - 0xc14abc90   ( 504 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -309,16 +309,11 @@
 +USB0_VBUS_SW: supplied by +5V_SYS
 +5V_USB_HS: supplied by +5V_SYS
 +3.3V_LP0: supplied by +3.3V_SYS
+mmc0: new high speed MMC card at address 0001
 +5V_HDMI_CON: supplied by +5V_SYS
 +5V_SATA: supplied by +5V_SYS
 +12V_SATA: supplied by +VDD_MUX
 usb2-0 supply vbus not found, using dummy regulator
-mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
- mmcblk0: p1
 usb2-1 supply vbus not found, using dummy regulator
 usb3-0 supply vbus not found, using dummy regulator
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
@@ -382,7 +377,6 @@
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4c may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x3e may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x3e may corrupt adjacent RW1C bits
-ata1: SATA link down (SStatus 0 SControl 300)
 pci_bus 0000:00: 2-byte config write to 0000:00:01.0 offset 0x4 may corrupt adjacent RW1C bits
 pci_bus 0000:00: 1-byte config write to 0000:00:01.0 offset 0xc may corrupt adjacent RW1C bits
 pci_bus 0000:00: 2-byte config write to 0000:00:02.0 offset 0x4 may corrupt adjacent RW1C bits
@@ -393,8 +387,6 @@
 pci 0000:01:00.0: [8086:4232] type 00 class 0x028000
 pci 0000:01:00.0: reg 0x10: [mem 0x00000000-0x00001fff 64bit]
 pci 0000:01:00.0: PME# supported from D0 D3hot D3cold
-mmc1: new high speed SDHC card at address aaaa
-mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
 PCI: bus1: Fast back to back transfers disabled
 pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
 pci 0000:02:00.0: [10ec:8168] type 00 class 0x020000
@@ -403,7 +395,6 @@
 pci 0000:02:00.0: reg 0x20: [mem 0x00000000-0x00003fff 64bit pref]
 pci 0000:02:00.0: supports D1 D2
 pci 0000:02:00.0: PME# supported from D0 D1 D2 D3hot D3cold
- mmcblk1: p1 p2 p3
 PCI: bus2: Fast back to back transfers disabled
 pci_bus 0000:02: busn_res: [bus 02-ff] end is updated to 02
 pci 0000:00:01.0: BAR 8: assigned [mem 0x13000000-0x130fffff]
@@ -428,19 +419,28 @@
 pcieport 0000:00:02.0: Signaling PME with IRQ 394
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
-r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0b54000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 395
+r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xf0b41000, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 395
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
-tegra-hdmi 54280000.hdmi: cannot set audio to 48000 Hz at 297000000 Hz pixel clock
-Console: switching to colour frame buffer device 512x135
+[drm] Cannot find any crtc or sizes - going 1024x768
+ata1: SATA link down (SStatus 0 SControl 300)
+mmc1: new high speed SDHC card at address aaaa
+mmcblk1: mmc1:aaaa SD04G 3.69 GiB 
+ mmcblk1: p1 p2 p3
+Console: switching to colour frame buffer device 128x48
 drm drm: fb0:  frame buffer device
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+ mmcblk0: p1
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 of: dev_pm_opp_of_cpumask_add_table: couldn't find opp table for cpu:0, -19
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:15 UTC (946684815)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:14 UTC (946684814)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
@@ -452,7 +452,7 @@
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 EXT4-fs (mmcblk0p1): recovery complete
 EXT4-fs (mmcblk0p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:1.
+VFS: Mounted root (ext3 filesystem) on device 179:17.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2016-12-13 20:43:07.313325165 -0800
+++ /tmp/foob	2016-12-13 20:43:07.317325182 -0800
@@ -32,9 +32,9 @@
 	 #################################################################
 	 #################################################################
 	 ##################################################
-	 3.9 MiB/s
+	 4 MiB/s
 done
-Bytes transferred = 5503824 (53fb50 hex)
+Bytes transferred = 5504608 (53fe60 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -45,7 +45,7 @@
 done
 Bytes transferred = 67133 (1063d hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x53fb50 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x53fe60 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 8301363c
@@ -54,7 +54,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-next-20161213-140842-gc791961 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Dec 12 22:05:05 PST 2016
+Linux version 4.9.0-next-20161214-140955-g9aa17b2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Dec 13 20:33:52 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -88,7 +88,7 @@
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
       .data : 0xc0d00000 - 0xc0d5fd00   ( 384 kB)
-       .bss : 0xc0d61000 - 0xc0dc5388   ( 401 kB)
+       .bss : 0xc0d61000 - 0xc0dc53a8   ( 401 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -128,7 +128,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?[    0.549661] hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
+No ATAGs?[    0.571309] hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 8 bytes.
 +5V_SYS: supplied by +VDD_MUX
 +3.3V_SYS: supplied by +VDD_MUX
@@ -374,8 +374,8 @@
 mmc1: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
 tegra-pcie 1003000.pcie-controller: 2x1, 1x1 configuration
 tegra-pcie 1003000.pcie-controller: probing port 0, using 2 lanes
-ata1: SATA link down (SStatus 0 SControl 300)
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000008
+ata1: SATA link down (SStatus 0 SControl 300)
 tegra-pcie 1003000.pcie-controller: probing port 1, using 1 lanes
 tegra-pcie 1003000.pcie-controller: Slot present pin change, signature: 00000000
 tegra-pcie 1003000.pcie-controller: PCI host bridge to bus 0000:00
@@ -444,6 +444,7 @@
 [drm] Cannot find any crtc or sizes - going 1024x768
 Console: switching to colour frame buffer device 128x48
 drm drm: fb0:  frame buffer device
+ mmcblk1: p1 p2 p3
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 of: dev_pm_opp_of_cpumask_add_table: couldn't find opp table for cpu:0, -19
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
@@ -451,7 +452,6 @@
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
 as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:13 UTC (946684813)
 +USB0_VBUS_SW: disabling
- mmcblk1: p1 p2 p3
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-12-13 20:43:07.321325197 -0800
+++ /tmp/foob	2016-12-13 20:43:07.325325213 -0800
@@ -41,10 +41,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #################################################
-	 3.6 MiB/s
+	 ##################################################
+	 3.5 MiB/s
 done
-Bytes transferred = 7396656 (70dd30 hex)
+Bytes transferred = 7398800 (70e590 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -55,7 +55,7 @@
 done
 Bytes transferred = 67943 (10967 hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x70dd30 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x70e590 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013966
@@ -63,7 +63,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-next-20161213-140842-gc791961 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Dec 12 22:05:23 PST 2016
+Linux version 4.9.0-next-20161214-140955-g9aa17b2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Dec 13 20:34:21 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -75,7 +75,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c142a5c0, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c142a640, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -88,7 +88,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1995812K/2097148K available (9216K kernel code, 1200K rwdata, 4136K rodata, 2048K init, 503K bss, 35800K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1995812K/2097148K available (9216K kernel code, 1200K rwdata, 4140K rodata, 2048K init, 503K bss, 35800K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -98,8 +98,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc142c0c0   (1201 kB)
-       .bss : 0xc142e000 - 0xc14abc70   ( 504 kB)
+      .data : 0xc1300000 - 0xc142c140   (1201 kB)
+       .bss : 0xc142e000 - 0xc14abc90   ( 504 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -308,6 +308,8 @@
 can: broadcast manager protocol (rev 20161123 t)
 can: netlink gateway (rev 20130117) max_hops=1
 Key type dns_resolver registered
+elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
+input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
 +3.3V_RUN: supplied by +3.3V_SYS
@@ -325,8 +327,6 @@
 +5V_HDMI_CON: supplied by +5V_SYS
 usb3-0 supply vbus not found, using dummy regulator
 usb3-1 supply vbus not found, using dummy regulator
-elan_i2c 1-0015: reading cmd (0x0101) fail.
-elan_i2c 1-0015: failed to get product ID: -121
 tegra-sor 54540000.sor: failed to probe output: -517
 panel supply power not found, using dummy regulator
 sdhci-tegra 700b0000.sdhci: allocated mmc-pwrseq
@@ -346,22 +346,22 @@
 mmc2: Invalid maximum block size, assuming 512 bytes
 mmc1: new high speed SDIO card at address 0001
 mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
-elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
-input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
 mmc2: new high speed SDHC card at address 0007
 mmcblk2: mmc2:0007 SD08G 7.42 GiB 
+GPT:Primary header alternate_lba != Alt. header my_lba
+GPT:3014591 != 15564799
+GPT:Primary header thinks Alt. header is not at the end of the disk.
+GPT:3014591 != 15564799
+GPT: Use GNU Parted to correct GPT errors.
+ mmcblk2: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 Console: switching to colour frame buffer device 170x48
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
 as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:24 UTC (946684824)
 +1.8V_RUN_CAM: disabling
-GPT:Primary header alternate_lba != Alt. header my_lba
-GPT:3014591 != 15564799
-GPT:Primary header thinks Alt. header is not at the end of the disk.
-GPT:3014591 != 15564799
 +3.3V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
 +1.05V_RUN_CAM_REAR: disabling
@@ -371,8 +371,6 @@
 +5V_VDD_TS_SW: disabling
 +5V_USB_HS: disabling
 +5V_USB_SS: disabling
-GPT: Use GNU Parted to correct GPT errors.
- mmcblk2: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 EXT4-fs (mmcblk0p3): couldn't mount as ext3 due to feature incompatibilities
 EXT4-fs (mmcblk0p3): couldn't mount as ext2 due to feature incompatibilities
 EXT4-fs (mmcblk0p3): recovery complete
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2016-12-13 20:43:07.329325230 -0800
+++ /tmp/foob	2016-12-13 20:43:07.333325248 -0800
@@ -40,9 +40,9 @@
 	 #################################################################
 	 #################################################################
 	 ##################################################
-	 3.6 MiB/s
+	 3.5 MiB/s
 done
-Bytes transferred = 5503824 (53fb50 hex)
+Bytes transferred = 5504608 (53fe60 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -53,7 +53,7 @@
 done
 Bytes transferred = 67943 (10967 hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x53fb50 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x53fe60 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013966
@@ -62,7 +62,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-next-20161213-140842-gc791961 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Dec 12 22:05:05 PST 2016
+Linux version 4.9.0-next-20161214-140955-g9aa17b2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Dec 13 20:33:52 PST 2016
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -97,7 +97,7 @@
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
       .data : 0xc0d00000 - 0xc0d5fd00   ( 384 kB)
-       .bss : 0xc0d61000 - 0xc0dc5388   ( 401 kB)
+       .bss : 0xc0d61000 - 0xc0dc53a8   ( 401 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -137,7 +137,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?[    0.487831] hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
+No ATAGs?[    0.509515] hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 8 bytes.
 tegra-mc 70019000.memory-controller: no memory timings for RAM code 0 registered
 +5V_SYS: supplied by +VDD_MUX
@@ -284,52 +284,52 @@
 usbcore: registered new interface driver usbhid
 usbhid: USB HID core driver
 tegra30-i2s 70301100.i2s: DMA channels sourced from device 70300000.ahub
-BUG: scheduling while atomic: kworker/0:1/29/0x00000002
+BUG: scheduling while atomic: kworker/0:1/37/0x00000002
 Modules linked in:
-CPU: 0 PID: 29 Comm: kworker/0:1 Not tainted 4.9.0-next-20161213-140842-gc791961 #1
+CPU: 0 PID: 37 Comm: kworker/0:1 Not tainted 4.9.0-next-20161214-140955-g9aa17b2 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: events_freezable mmc_rescan
 [<c010f308>] (unwind_backtrace) from [<c010a900>] (show_stack+0x10/0x14)
-[<c010a900>] (show_stack) from [<c034d8dc>] (dump_stack+0x84/0xa4)
-[<c034d8dc>] (dump_stack) from [<c01b5548>] (__schedule_bug+0x60/0x80)
-[<c01b5548>] (__schedule_bug) from [<c0827f00>] (__schedule+0x528/0x60c)
-[<c0827f00>] (__schedule) from [<c08281b4>] (schedule+0x50/0xc0)
-[<c08281b4>] (schedule) from [<c0828410>] (schedule_preempt_disabled+0x24/0x40)
-[<c0828410>] (schedule_preempt_disabled) from [<c082c378>] (__mutex_lock_slowpath+0x400/0x6f0)
-[<c082c378>] (__mutex_lock_slowpath) from [<c03c4c74>] (clk_prepare_lock+0x3c/0xe4)
-[<c03c4c74>] (clk_prepare_lock) from [<c03c717c>] (clk_set_rate+0x14/0x30)
-[<c03c717c>] (clk_set_rate) from [<c05dd4a8>] (tegra_sdhci_set_clock+0x28/0xdc)
-[<c05dd4a8>] (tegra_sdhci_set_clock) from [<c05dc610>] (sdhci_set_ios+0x2b8/0x57c)
-[<c05dc610>] (sdhci_set_ios) from [<c05c6a28>] (mmc_set_bus_mode+0x18/0x1c)
-[<c05c6a28>] (mmc_set_bus_mode) from [<c05c9940>] (mmc_init_card+0xb4/0x1824)
-[<c05c9940>] (mmc_init_card) from [<c05cb424>] (mmc_attach_mmc+0x8c/0x14c)
-[<c05cb424>] (mmc_attach_mmc) from [<c05c79a4>] (mmc_rescan+0x274/0x34c)
-[<c05c79a4>] (mmc_rescan) from [<c0134e7c>] (process_one_work+0x120/0x320)
-[<c0134e7c>] (process_one_work) from [<c01350e0>] (worker_thread+0x2c/0x4b4)
-[<c01350e0>] (worker_thread) from [<c013a5f8>] (kthread+0xfc/0x134)
-[<c013a5f8>] (kthread) from [<c0107938>] (ret_from_fork+0x14/0x3c)
-BUG: scheduling while atomic: kworker/0:1/29/0x00000000
+[<c010a900>] (show_stack) from [<c034d43c>] (dump_stack+0x84/0xa4)
+[<c034d43c>] (dump_stack) from [<c01b5458>] (__schedule_bug+0x60/0x80)
+[<c01b5458>] (__schedule_bug) from [<c08281d8>] (__schedule+0x528/0x60c)
+[<c08281d8>] (__schedule) from [<c082848c>] (schedule+0x50/0xc0)
+[<c082848c>] (schedule) from [<c08286e8>] (schedule_preempt_disabled+0x24/0x40)
+[<c08286e8>] (schedule_preempt_disabled) from [<c082c650>] (__mutex_lock_slowpath+0x400/0x6f0)
+[<c082c650>] (__mutex_lock_slowpath) from [<c03c4800>] (clk_prepare_lock+0x3c/0xe4)
+[<c03c4800>] (clk_prepare_lock) from [<c03c6d08>] (clk_set_rate+0x14/0x30)
+[<c03c6d08>] (clk_set_rate) from [<c05dd1a0>] (tegra_sdhci_set_clock+0x28/0xdc)
+[<c05dd1a0>] (tegra_sdhci_set_clock) from [<c05dc308>] (sdhci_set_ios+0x2b8/0x57c)
+[<c05dc308>] (sdhci_set_ios) from [<c05c674c>] (mmc_set_bus_mode+0x18/0x1c)
+[<c05c674c>] (mmc_set_bus_mode) from [<c05c9650>] (mmc_init_card+0xb4/0x1824)
+[<c05c9650>] (mmc_init_card) from [<c05cb134>] (mmc_attach_mmc+0x8c/0x14c)
+[<c05cb134>] (mmc_attach_mmc) from [<c05c76b8>] (mmc_rescan+0x274/0x34c)
+[<c05c76b8>] (mmc_rescan) from [<c0134e74>] (process_one_work+0x120/0x31c)
+[<c0134e74>] (process_one_work) from [<c01350d4>] (worker_thread+0x2c/0x4b4)
+[<c01350d4>] (worker_thread) from [<c013a5e8>] (kthread+0xfc/0x134)
+[<c013a5e8>] (kthread) from [<c0107938>] (ret_from_fork+0x14/0x3c)
+BUG: scheduling while atomic: kworker/0:1/37/0x00000000
 Modules linked in:
-CPU: 0 PID: 29 Comm: kworker/0:1 Tainted: G        W       4.9.0-next-20161213-140842-gc791961 #1
+CPU: 0 PID: 37 Comm: kworker/0:1 Tainted: G        W       4.9.0-next-20161214-140955-g9aa17b2 #1
 Hardware name: NVIDIA Tegra SoC (Flattened Device Tree)
 Workqueue: events_freezable mmc_rescan
 [<c010f308>] (unwind_backtrace) from [<c010a900>] (show_stack+0x10/0x14)
-[<c010a900>] (show_stack) from [<c034d8dc>] (dump_stack+0x84/0xa4)
-[<c034d8dc>] (dump_stack) from [<c01b5548>] (__schedule_bug+0x60/0x80)
-[<c01b5548>] (__schedule_bug) from [<c0827f00>] (__schedule+0x528/0x60c)
-[<c0827f00>] (__schedule) from [<c08281b4>] (schedule+0x50/0xc0)
-[<c08281b4>] (schedule) from [<c082dd68>] (schedule_timeout+0x1c0/0x2e4)
-[<c082dd68>] (schedule_timeout) from [<c0828b00>] (wait_for_common+0xbc/0x148)
-[<c0828b00>] (wait_for_common) from [<c05c5528>] (mmc_wait_for_req_done+0x7c/0x104)
-[<c05c5528>] (mmc_wait_for_req_done) from [<c05c56ec>] (mmc_wait_for_cmd+0x58/0x78)
-[<c05c56ec>] (mmc_wait_for_cmd) from [<c05cbdf4>] (mmc_go_idle+0x78/0xfc)
-[<c05cbdf4>] (mmc_go_idle) from [<c05c98c0>] (mmc_init_card+0x34/0x1824)
-[<c05c98c0>] (mmc_init_card) from [<c05cb424>] (mmc_attach_mmc+0x8c/0x14c)
-[<c05cb424>] (mmc_attach_mmc) from [<c05c79a4>] (mmc_rescan+0x274/0x34c)
-[<c05c79a4>] (mmc_rescan) from [<c0134e7c>] (process_one_work+0x120/0x320)
-[<c0134e7c>] (process_one_work) from [<c01350e0>] (worker_thread+0x2c/0x4b4)
-[<c01350e0>] (worker_thread) from [<c013a5f8>] (kthread+0xfc/0x134)
-[<c013a5f8>] (kthread) from [<c0107938>] (ret_from_fork+0x14/0x3c)
+[<c010a900>] (show_stack) from [<c034d43c>] (dump_stack+0x84/0xa4)
+[<c034d43c>] (dump_stack) from [<c01b5458>] (__schedule_bug+0x60/0x80)
+[<c01b5458>] (__schedule_bug) from [<c08281d8>] (__schedule+0x528/0x60c)
+[<c08281d8>] (__schedule) from [<c082848c>] (schedule+0x50/0xc0)
+[<c082848c>] (schedule) from [<c082e02c>] (schedule_timeout+0x1c0/0x2e4)
+[<c082e02c>] (schedule_timeout) from [<c0828dd8>] (wait_for_common+0xbc/0x148)
+[<c0828dd8>] (wait_for_common) from [<c05c524c>] (mmc_wait_for_req_done+0x7c/0x104)
+[<c05c524c>] (mmc_wait_for_req_done) from [<c05c5410>] (mmc_wait_for_cmd+0x58/0x78)
+[<c05c5410>] (mmc_wait_for_cmd) from [<c05cbb04>] (mmc_go_idle+0x78/0xec)
+[<c05cbb04>] (mmc_go_idle) from [<c05c95d0>] (mmc_init_card+0x34/0x1824)
+[<c05c95d0>] (mmc_init_card) from [<c05cb134>] (mmc_attach_mmc+0x8c/0x14c)
+[<c05cb134>] (mmc_attach_mmc) from [<c05c76b8>] (mmc_rescan+0x274/0x34c)
+[<c05c76b8>] (mmc_rescan) from [<c0134e74>] (process_one_work+0x120/0x31c)
+[<c0134e74>] (process_one_work) from [<c01350d4>] (worker_thread+0x2c/0x4b4)
+[<c01350d4>] (worker_thread) from [<c013a5e8>] (kthread+0xfc/0x134)
+[<c013a5e8>] (kthread) from [<c0107938>] (ret_from_fork+0x14/0x3c)
 max98090 0-0010: MAX98090 REVID=0x43
 max98090 0-0010: use default 2.8v micbias
 tegra-snd-max98090 sound: HiFi <-> 70301100.i2s mapping ok
@@ -416,7 +416,7 @@
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input4
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 17:58:46 UTC (946749526)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:22 UTC (946684822)
 +1.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
 +1.05V_RUN_CAM_REAR: disabling
@@ -437,4 +437,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.208461] usb 1-3: new high-speed USB device number 3 using tegra-xusb
+root@(none):/# 
\ No newline at end of file
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2016-12-13 20:43:07.337325265 -0800
+++ /tmp/foob	2016-12-13 20:43:07.337325265 -0800
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1800663 us
+Bootloader-Cpu Init at (time stamp): 1800521 us
 
 Pinmux changes applied in kernel way
 
@@ -62,20 +62,20 @@
 Board Id Read Time: 4 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1800636  us
-Bootloader (AosInit) time: 733446 us
-Bootloader Main Init time: 180266 us
-BlockDevMgrInit time: 157491 us
-PartMgrInit time: 8 us
+Miscellaneous avp time: 1800494  us
+Bootloader (AosInit) time: 733452 us
+Bootloader Main Init time: 180235 us
+BlockDevMgrInit time: 157533 us
+PartMgrInit time: 9 us
 StorMgrInit time: 30 us
-FsMgrInit time: 3 us
-Partition Table load: 77048 us 
-Display init time: 574773 us
-Kernel read time: 378 msec
+FsMgrInit time: 2 us
+Partition Table load: 77119 us 
+Display init time: 574342 us
+Kernel read time: 377 msec
 Load Kernel Size:   16304128 Bytes
-Bl Kernel ReadSpeed: 43132000 Bytes/sec
-Pre-Kernel Jump time: 1740361 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 5842270 us
+Bl Kernel ReadSpeed: 43247000 Bytes/sec
+Pre-Kernel Jump time: 1741131 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 5844151 us
 Booting Linux on physical CPU 0x0
 Linux version 4.8.0-239303-g3477d16 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Fri Oct 7 01:12:37 PDT 2016
 Boot CPU: AArch64 Processor [4e0f0000]
@@ -184,7 +184,7 @@
 kvm [1]: HYP mode not available
 futex hash table entries: 256 (order: 3, 32768 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(1.163:1): initialized
+audit: type=2000 audit(1.135:1): initialized
 workingset: timestamp_bits=46 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -251,9 +251,10 @@
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
-+VDD_LED: supplied by +VDD_MUX
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
++VDD_LED: supplied by +VDD_MUX
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 +5V_USB_HS: supplied by +5V_SYS
 +5V_USB_SS: supplied by +5V_SYS
 +5V_HDMI_CON: supplied by +5V_SYS
@@ -272,9 +273,6 @@
 Freeing unused kernel memory: 960K (ffff800000c00000 - ffff800000cf0000)
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
-sdhci-tegra 700b0400.sdhci: Got CD GPIO
-sdhci-tegra 700b0400.sdhci: Got WP GPIO
 
 System initialization...
 
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-12-13 20:43:07.345325297 -0800
+++ /tmp/foob	2016-12-13 20:43:07.345325297 -0800
@@ -46,10 +46,10 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #################################################
+	 ##################################################
 	 4.1 MiB/s
 done
-Bytes transferred = 7396656 (70dd30 hex)
+Bytes transferred = 7398800 (70e590 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -60,7 +60,7 @@
 done
 Bytes transferred = 21404 (539c hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x70dd30 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x70e590 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 0241039b
@@ -68,7 +68,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-next-20161213-140842-gc791961 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Dec 12 22:05:23 PST 2016
+Linux version 4.9.0-next-20161214-140955-g9aa17b2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Dec 13 20:34:21 PST 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: Compulab TrimSlice board
@@ -78,7 +78,7 @@
 cma: Reserved 64 MiB at 0x3c000000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c142a5c0, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c142a640, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -91,7 +91,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 955720K/1048576K available (9216K kernel code, 1200K rwdata, 4136K rodata, 2048K init, 503K bss, 27320K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 955720K/1048576K available (9216K kernel code, 1200K rwdata, 4140K rodata, 2048K init, 503K bss, 27320K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -101,8 +101,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc142c0c0   (1201 kB)
-       .bss : 0xc142e000 - 0xc14abc70   ( 504 kB)
+      .data : 0xc1300000 - 0xc142c140   (1201 kB)
+       .bss : 0xc142e000 - 0xc14abc90   ( 504 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -343,7 +343,7 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2015-02-06 02:09:51 UTC (1423188591)
+rtc-em3027 2-0056: setting system clock to 2015-02-06 02:12:55 UTC (1423188775)
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2016-12-13 20:43:07.353325332 -0800
+++ /tmp/foob	2016-12-13 20:43:07.353325332 -0800
@@ -45,20 +45,20 @@
 	 #################################################################
 	 #################################################################
 	 ##################################################
-	 4 MiB/s
+	 4.1 MiB/s
 done
-Bytes transferred = 5503824 (53fb50 hex)
+Bytes transferred = 5504608 (53fe60 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 2.9 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 21404 (539c hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x53fb50 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x53fe60 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 0241039b
@@ -67,7 +67,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-next-20161213-140842-gc791961 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Dec 12 22:05:05 PST 2016
+Linux version 4.9.0-next-20161214-140955-g9aa17b2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Dec 13 20:33:52 PST 2016
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: Compulab TrimSlice board
@@ -100,7 +100,7 @@
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
       .data : 0xc0d00000 - 0xc0d5fd00   ( 384 kB)
-       .bss : 0xc0d61000 - 0xc0dc5388   ( 401 kB)
+       .bss : 0xc0d61000 - 0xc0dc53a8   ( 401 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 	RCU restricting CPUs from NR_CPUS=4 to nr_cpu_ids=2.
@@ -139,7 +139,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?[    0.316912] hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
+No ATAGs?[    0.341151] hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 4 bytes.
 usb1_vbus: Failed to request enable GPIO170: -517
 reg-fixed-voltage regulators:regulator@2: Failed to register regulator: -517
@@ -309,17 +309,17 @@
 tlv320aic23-codec 2-001a: ASoC: no dapm match for RLINEIN --> NULL --> Line Input
 tlv320aic23-codec 2-001a: ASoC: Failed to add route RLINEIN -> NULL -> Line Input
 tlv320aic23-codec 2-001a: Control not supported for path MICIN -> [NULL] -> Mic Input
-mmc0: host does not support reading read-only switch, assuming write-enable
-mmc0: new high speed SDHC card at address b368
-mmcblk0: mmc0:b368 NCard 3.70 GiB 
- mmcblk0: p1
 tlv320aic23-codec 2-001a: ASoC: no dapm match for MICIN --> NULL --> Mic Input
 tlv320aic23-codec 2-001a: ASoC: Failed to add route MICIN -> NULL -> Mic Input
 tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
+NET: Registered protocol family 10
+mmc0: host does not support reading read-only switch, assuming write-enable
+mmc0: new high speed SDHC card at address b368
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
+mmcblk0: mmc0:b368 NCard 3.70 GiB 
+ mmcblk0: p1
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
-NET: Registered protocol family 10
  mmcblk1: p1 p2
 Segment Routing with IPv6
 mip6: Mobile IPv6
@@ -349,7 +349,7 @@
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-rtc-em3027 2-0056: setting system clock to 2015-02-06 02:08:18 UTC (1423188498)
+rtc-em3027 2-0056: setting system clock to 2015-02-06 02:11:23 UTC (1423188683)
 ALSA device list:
   #0: tegra-trimslice
 EXT4-fs (mmcblk0p1): mounting ext3 file system using the ext4 subsystem
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2016-12-13 20:43:07.357325348 -0800
+++ /tmp/foob	2016-12-13 20:43:07.361325365 -0800
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 758565 us
-Basic modules init took 348903 us
+Tegraboot started after 721344 us
+Basic modules init took 349429 us
 Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14384 us
-Carveout took 252714 us
-CPU initialization took 92320 us
-Total time taken by TegraBoot 708321 us
+Next stage binary read took 14788 us
+Carveout took 252710 us
+CPU initialization took 92307 us
+Total time taken by TegraBoot 709234 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.597] LPDDR4 Training: Number of tables = 10
+64b[0001.562] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-14987776 bytes read in 794 ms (18 MiB/s)
+14987776 bytes read in 795 ms (18 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-44882 bytes read in 453 ms (96.7 KiB/s)
+44882 bytes read in 454 ms (95.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 503 ms (3.1 MiB/s)
+1627787 bytes read in 500 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-next-20161213-165275-gc791961 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Dec 12 22:08:19 PST 2016
+Linux version 4.9.0-next-20161214-165388-g9aa17b2 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Dec 13 20:36:55 PST 2016
 Boot CPU: AArch64 Processor [411fd071]
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -262,7 +262,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.535:1): initialized
+audit: type=2000 audit(0.539:1): initialized
 workingset: timestamp_bits=46 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -342,6 +342,6 @@
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
-bash: no job control in th[    1.338751] random: fast init done
-is shell
+bash: no job contr[    1.382018] random: fast init done
+ol in this shell
 root@(none):/# 
\ No newline at end of file
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2016-12-13 20:43:07.365325381 -0800
+++ /tmp/foob	2016-12-13 20:43:07.369325398 -0800
@@ -1,5 +1,3 @@
-ptions '115200n8')
-bootconsole [uart0] enabled
 psci: probing for conduit method from DT.
 psci: PSCIv1.0 detected in firmware.
 psci: Using standard PSCI v0.2 function IDs
@@ -51,7 +49,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x3f50: expected 0x2004f400
+boot misc3 0x3e50: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -64,7 +62,7 @@
 console [tty0] enabled
 bootconsole [uart0] disabled
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-next-20161213-165275-gc791961 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Mon Dec 12 22:08:19 PST 2016
+Linux version 4.9.0-next-20161214-165388-g9aa17b2 (pwalmsley@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09) ) #1 SMP PREEMPT Tue Dec 13 20:36:55 PST 2016
 Boot CPU: AArch64 Processor [411fd071]
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
@@ -122,7 +120,7 @@
  (comparison mask = 0xf8000000)
 boot misc2 0x0: expected 0xf400f0da
  (comparison mask = 0xffffffff)
-boot misc3 0x3f50: expected 0x2004f400
+boot misc3 0x3e50: expected 0x2004f400
  (comparison mask = 0xffffffff)
 PLL_P already enabled. Postponing set full defaults
 PLL_X already enabled. Postponing set full defaults
@@ -210,7 +208,7 @@
 kvm [1]: virtual timer IRQ4
 futex hash table entries: 1024 (order: 5, 131072 bytes)
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.323:1): initialized
+audit: type=2000 audit(0.363:1): initialized
 workingset: timestamp_bits=46 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -300,9 +298,9 @@
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 BGND3R 29.1 GiB 
 mmcblk0boot0: mmc0:0001 BGND3R partition 1 4.00 MiB
+max77686-rtc max77620-rtc: setting system clock to 2000-09-24 10:05:40 UTC (969789940)
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB
-max77686-rtc max77620-rtc: setting system clock to 2000-09-23 11:36:48 UTC (969709008)
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
 tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 PP1200_AVDD: disabling
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2016-12-13 20:43:07.373325414 -0800
+++ /tmp/foob	2016-12-13 20:43:07.377325431 -0800
@@ -27,7 +27,6 @@
 Tegra30 (Beaver) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=edbe1840-3633-c751-7993-b0ac834f81e0 rootwait gpt debug noinitrd earlyprintk init=/bin/bash'
 Tegra30 (Beaver) # setenv ipaddr 192.168.100.101; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra30 (Beaver) # tftpboot 0x81000000 zImage.multi_v7_defconfig
-Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'zImage.multi_v7_defconfig'.
@@ -39,21 +38,21 @@
 	 #################################################################
 	 #################################################################
 	 #################################################################
-	 #################################################
-	 3.2 MiB/s
+	 ##################################################
+	 3.3 MiB/s
 done
-Bytes transferred = 7396656 (70dd30 hex)
+Bytes transferred = 7398800 (70e590 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.9 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 51804 (ca5c hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x70dd30 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x70e590 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa5b ... OK
@@ -61,7 +60,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-next-20161213-140842-gc791961 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Mon Dec 12 22:05:23 PST 2016
+Linux version 4.9.0-next-20161214-140955-g9aa17b2 (pwalmsley@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP Tue Dec 13 20:34:21 PST 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -71,7 +70,7 @@
 cma: Reserved 64 MiB at 0xfb800000
 Memory policy: Data cache writealloc
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c142a5c0, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c142a640, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -84,7 +83,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1994836K/2096128K available (9216K kernel code, 1200K rwdata, 4136K rodata, 2048K init, 503K bss, 35756K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1994836K/2096128K available (9216K kernel code, 1200K rwdata, 4140K rodata, 2048K init, 503K bss, 35756K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -94,8 +93,8 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0c00000   (10208 kB)
       .init : 0xc1100000 - 0xc1300000   (2048 kB)
-      .data : 0xc1300000 - 0xc142c0c0   (1201 kB)
-       .bss : 0xc142e000 - 0xc14abc70   ( 504 kB)
+      .data : 0xc1300000 - 0xc142c140   (1201 kB)
+       .bss : 0xc142e000 - 0xc14abc90   ( 504 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
@@ -307,18 +306,18 @@
 +VDD_5V_HDMI: supplied by sys_3v3,vdd_3v3_alw
 sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
 tegra-pcie 3000.pcie-controller: 2x3 configuration
-mmc0: new ultra high speed DDR50 SDHC card at address aaaa
-mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
+tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB
+mmc0: new ultra high speed DDR50 SDHC card at address aaaa
+mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
 Primary GPT is invalid, using alternate GPT.
+tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
  mmcblk1: p1 p2 p3 p4 p5
  mmcblk0: p1 p2 p3
-tegra-pcie 3000.pcie-controller: probing port 0, using 2 lanes
-tegra-pcie 3000.pcie-controller: probing port 2, using 2 lanes
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
 tegra-pcie 3000.pcie-controller: link 2 down, retrying
@@ -391,10 +390,10 @@
 EXT4-fs (mmcblk1p1): warning: mounting fs with errors, running e2fsck is recommended
 EXT4-fs (mmcblk1p1): recovery complete
 EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:17.
+VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.144057] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.143613] usb 2-1: new high-speed USB device number 2 using tegra-ehci
 asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0f:67:60
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2016-12-13 20:43:07.381325449 -0800
+++ /tmp/foob	2016-12-13 20:43:07.385325466 -0800
@@ -39,18 +39,18 @@
 	 ##################################################
 	 3.2 MiB/s
 done
-Bytes transferred = 5503824 (53fb50 hex)
+Bytes transferred = 5504608 (53fe60 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 3.3 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 51804 (ca5c hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x53fb50 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x53fe60 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa5b ... OK
@@ -59,7 +59,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.9.0-next-20161213-140842-gc791961 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Mon Dec 12 22:05:05 PST 2016
+Linux version 4.9.0-next-20161214-140955-g9aa17b2 (pwalmsley@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04) ) #1 SMP PREEMPT Tue Dec 13 20:33:52 PST 2016
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt:Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -92,7 +92,7 @@
       .text : 0xc0008000 - 0xc0900000   (9184 kB)
       .init : 0xc0c00000 - 0xc0d00000   (1024 kB)
       .data : 0xc0d00000 - 0xc0d5fd00   ( 384 kB)
-       .bss : 0xc0d61000 - 0xc0dc5388   ( 401 kB)
+       .bss : 0xc0d61000 - 0xc0dc53a8   ( 401 kB)
 Preemptible hierarchical RCU implementation.
 	Build-time adjustment of leaf fanout to 32.
 NR_IRQS:16 nr_irqs:16 16
@@ -131,7 +131,7 @@
 NET: Registered protocol family 16
 DMA: preallocated 256 KiB pool for atomic coherent allocations
 cpuidle: using governor menu
-No ATAGs?[    0.469343] hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
+No ATAGs?[    0.493249] hw-breakpoint: found 5 (+1 reserved) breakpoint and 1 watchpoint registers.
 hw-breakpoint: maximum watchpoint size is 4 bytes.
 tegra-mc 7000f000.memory-controller: no memory timings for RAM code 3 registered
 vdd_5v_sata: Failed to request enable GPIO30: -517
@@ -390,4 +390,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.390267] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.460290] usb 2-1: new high-speed USB device number 2 using tegra-ehci
