Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2017-09-04 02:43:29.441863445 -0700
+++ /tmp/foob	2017-09-04 02:43:29.441863445 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc7-next-20170901-198696-g1d53d90 (jonathanh@sellafield) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Fri Sep 1 00:08:57 PDT 2017
+Linux version 4.13.0-next-20170904-317377-g9829d9f31f6c (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Mon Sep 4 02:41:15 PDT 2017
 Boot CPU: AArch64 Processor [411fd070]
 Machine model: linux,dummy-virt
 Memory limited to 1024MB
@@ -29,7 +29,7 @@
 Policy zone: DMA
 Kernel command line: root=/dev/vda2 rw rootwait mem=1024M console=ttyAMA0,38400n8 init=/bin/bash
 PID hash table entries: 4096 (order: 3, 32768 bytes)
-Memory: 998972K/1048576K available (9212K kernel code, 1144K rwdata, 4560K rodata, 1152K init, 404K bss, 33220K reserved, 16384K cma-reserved)
+Memory: 998972K/1048576K available (9212K kernel code, 1144K rwdata, 4568K rodata, 1152K init, 404K bss, 33220K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
@@ -116,7 +116,7 @@
 hw perfevents: enabled with armv8_pmuv3 PMU driver, 1 counters available
 kvm [1]: HYP mode not available
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.808:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.816:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=18 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -181,7 +181,7 @@
 Key type dns_resolver registered
 registered taskstats version 1
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2017-09-01 07:09:50 UTC (1504249790)
+rtc-pl031 9010000.pl031: setting system clock to 2017-09-04 09:42:23 UTC (1504518143)
 ALSA device list:
   No soundcards found.
 uart-pl011 9000000.pl011: no DMA platform data
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-09-04 02:43:29.481863617 -0700
+++ /tmp/foob	2017-09-04 02:43:29.481863617 -0700
@@ -27,7 +27,6 @@
 Tegra114 (Dalmore) # setenv bootargs 'ignore_loglevel console=ttyS0,115200n8 rw root=PARTUUID=7369c667-ff51-ec4a-29cd-baabf2fbe346 rootwait gpt debug earlyprintk init=/bin/bash'
 Tegra114 (Dalmore) # setenv ipaddr 192.168.100.100; setenv serverip 192.168.100.1; setenv tftpserverip 192.168.100.1
 Tegra114 (Dalmore) # tftpboot 0x81000000 zImage.multi_v7_defconfig
-Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'zImage.multi_v7_defconfig'.
@@ -41,20 +40,20 @@
 	 #################################################################
 	 #################################################################
 	 ####################################
-	 2.9 MiB/s
+	 3.1 MiB/s
 done
-Bytes transferred = 8151560 (7c6208 hex)
+Bytes transferred = 8159752 (7c8208 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.6 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 35036 (88dc hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x7c6200 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x7c8200 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8db ... OK
@@ -62,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc7-next-20170901-198696-g1d53d90 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Fri Sep 1 00:05:43 PDT 2017
+Linux version 4.13.0-next-20170904-199136-g9829d9f (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Mon Sep 4 02:36:12 PDT 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -73,7 +72,7 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c158e880, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c158e440, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -87,7 +86,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993480K/2096128K available (10240K kernel code, 1626K rwdata, 4488K rodata, 2048K init, 405K bss, 37112K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993480K/2096128K available (10240K kernel code, 1625K rwdata, 4500K rodata, 2048K init, 405K bss, 37112K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -97,7 +96,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1596bd0   (1627 kB)
+      .data : 0xc1400000 - 0xc1596790   (1626 kB)
        .bss : 0xc159fbc0 - 0xc16052fc   ( 406 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -379,8 +378,8 @@
 mmc0: new high speed SDHC card at address aaaa
 mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 Key type dns_resolver registered
-ThumbEE CPU extension supported.
  mmcblk0: p1 p2 p3
+ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
 usb1_vbus: supplied by vdd-sys-5v0
 usb2_vbus: supplied by vdd-sys-5v0
@@ -413,7 +412,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-23 13:58:35 UTC (948635915)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-26 16:31:01 UTC (948904261)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -425,5 +424,5 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.119074] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.129133] usb 1-1: new high-speed USB device number 2 using tegra-ehci
 asix 1-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0d:b5:16
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2017-09-04 02:43:29.489863651 -0700
+++ /tmp/foob	2017-09-04 02:43:29.489863651 -0700
@@ -38,20 +38,20 @@
 	 #################################################################
 	 #################################################################
 	 #######################################
-	 2.9 MiB/s
+	 3.1 MiB/s
 done
-Bytes transferred = 6287120 (5fef10 hex)
+Bytes transferred = 6291280 (5fff50 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.6 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 35036 (88dc hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5fef10 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5fff50 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff8db ... OK
@@ -60,7 +60,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc7-next-20170901-198696-g1d53d90 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Fri Sep 1 00:05:47 PDT 2017
+Linux version 4.13.0-next-20170904-199136-g9829d9f (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Sep 4 02:35:36 PDT 2017
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -354,8 +354,8 @@
 mmcblk0: mmc0:aaaa SL16G 14.8 GiB 
 mmc1: new high speed MMC card at address 0001
 mmcblk1: mmc1:0001 SEM32G 29.1 GiB 
- mmcblk0: p1 p2 p3
 mmcblk1boot0: mmc1:0001 SEM32G partition 1 4.00 MiB
+ mmcblk0: p1 p2 p3
 mmcblk1boot1: mmc1:0001 SEM32G partition 2 4.00 MiB
 mmcblk1rpmb: mmc1:0001 SEM32G partition 3 4.00 MiB
  mmcblk1: p1 p2 p3 p4 p5 p6 p7 p8
@@ -407,7 +407,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-23 14:01:08 UTC (948636068)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-01-26 16:28:33 UTC (948904113)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -421,4 +421,5 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# 
\ No newline at end of file
+root@(none):/# [    3.768760] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+asix 1-1:1.0 eth0: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0d:b5:16
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-09-04 02:43:29.497863685 -0700
+++ /tmp/foob	2017-09-04 02:43:29.497863685 -0700
@@ -28,27 +28,27 @@
 Load address: 0x82000000
 Loading: *#################################################################
 	 #################################################################
-	 #################################################################
+	 ###############################################T T T ##################
 	 #################################################################
 	 #################################################################
 	 #################################################################
 	 #################################################################
 	 #################################################################
 	 ####################################
-	 3.5 MiB/s
+	 468.8 KiB/s
 done
-Bytes transferred = 8151560 (7c6208 hex)
+Bytes transferred = 8159752 (7c8208 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.6 MiB/s
+	 3.8 MiB/s
 done
 Bytes transferred = 67173 (10665 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x7c6200 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x7c8200 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013664
@@ -56,7 +56,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc7-next-20170901-198696-g1d53d90 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Fri Sep 1 00:05:43 PDT 2017
+Linux version 4.13.0-next-20170904-199136-g9829d9f (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Mon Sep 4 02:36:12 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -67,7 +67,7 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c158e880, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c158e440, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -81,7 +81,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993360K/2096128K available (10240K kernel code, 1626K rwdata, 4488K rodata, 2048K init, 405K bss, 37232K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993360K/2096128K available (10240K kernel code, 1625K rwdata, 4500K rodata, 2048K init, 405K bss, 37232K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -91,7 +91,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1596bd0   (1627 kB)
+      .data : 0xc1400000 - 0xc1596790   (1626 kB)
        .bss : 0xc159fbc0 - 0xc16052fc   ( 406 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -401,13 +401,13 @@
 +5V_SATA: supplied by +5V_SYS
 +12V_SATA: supplied by +VDD_MUX
  usb2-0: usb2-0 supply vbus not found, using dummy regulator
+ usb2-1: usb2-1 supply vbus not found, using dummy regulator
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
  mmcblk0: p1
- usb2-1: usb2-1 supply vbus not found, using dummy regulator
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
 tegra-pcie 1003000.pcie: 2x1, 1x1 configuration
 tegra-pcie 1003000.pcie: Failed to get supply 'avddio-pex': -517
@@ -530,7 +530,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:14 UTC (946684814)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:30 UTC (946684830)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2017-09-04 02:43:29.517863771 -0700
+++ /tmp/foob	2017-09-04 02:43:29.517863771 -0700
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MMC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -35,18 +35,18 @@
 	 #######################################
 	 3.6 MiB/s
 done
-Bytes transferred = 6287120 (5fef10 hex)
+Bytes transferred = 6291280 (5fff50 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 67173 (10665 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5fef10 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5fff50 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013664
@@ -55,7 +55,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc7-next-20170901-198696-g1d53d90 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Fri Sep 1 00:05:47 PDT 2017
+Linux version 4.13.0-next-20170904-199136-g9829d9f (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Sep 4 02:35:36 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -417,8 +417,8 @@
 as3722-ldo2: supplied by +1.8V_VDDIO
 as3722-ldo3: supplied by +3.3V_SYS
 as3722-ldo4: supplied by +3.3V_SYS
-input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input1
 as3722-ldo5: supplied by +1.8V_VDDIO
+input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input1
 as3722-ldo6: supplied by +3.3V_RUN
 as3722-ldo7: supplied by +1.8V_VDDIO
 +1.05V_RUN_CAM_REAR: Bringing 0uV into 1050000-1050000uV
@@ -538,7 +538,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:15 UTC (946684815)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:14 UTC (946684814)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-09-04 02:43:29.545863892 -0700
+++ /tmp/foob	2017-09-04 02:43:29.549863908 -0700
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -43,20 +43,20 @@
 	 #################################################################
 	 #################################################################
 	 ####################################
-	 2.8 MiB/s
+	 3.5 MiB/s
 done
-Bytes transferred = 8151560 (7c6208 hex)
+Bytes transferred = 8159752 (7c8208 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 2.7 MiB/s
+	 3.3 MiB/s
 done
 Bytes transferred = 68203 (10a6b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x7c6200 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x7c8200 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a6a
@@ -64,7 +64,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc7-next-20170901-198696-g1d53d90 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Fri Sep 1 00:05:43 PDT 2017
+Linux version 4.13.0-next-20170904-199136-g9829d9f (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Mon Sep 4 02:36:12 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -76,7 +76,7 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 On node 0 totalpages: 524287
-free_area_init_node: node 0, pgdat c158e880, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c158e440, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -90,7 +90,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1994376K/2097148K available (10240K kernel code, 1626K rwdata, 4488K rodata, 2048K init, 405K bss, 37236K reserved, 65536K cma-reserved, 1245180K highmem)
+Memory: 1994376K/2097148K available (10240K kernel code, 1625K rwdata, 4500K rodata, 2048K init, 405K bss, 37236K reserved, 65536K cma-reserved, 1245180K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -100,7 +100,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1596bd0   (1627 kB)
+      .data : 0xc1400000 - 0xc1596790   (1626 kB)
        .bss : 0xc159fbc0 - 0xc16052fc   ( 406 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -407,8 +407,15 @@
 +5V_USB_SS: supplied by +5V_SYS
 +3.3V_PANEL: supplied by +3.3V_RUN
 +3.3V_LP0: supplied by +3.3V_SYS
+mmc0: new high speed MMC card at address 0001
+mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
+mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
+mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
+mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 elan_i2c 1-0015: failed to read report data: -121
+elan_i2c 1-0015: failed to read report data: -121
 +5V_HDMI_CON: supplied by +5V_SYS
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
  usb3-1: usb3-1 supply vbus not found, using dummy regulator
@@ -418,18 +425,12 @@
 mmc1: Unknown controller version (3). You may experience problems.
 mmc1: Invalid maximum block size, assuming 512 bytes
 mmc1: Switching to 3.3V signalling voltage failed
-mmc0: new high speed MMC card at address 0001
-mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
-mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
-mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
-mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 mmc1: SDHCI controller on 700b0000.sdhci [700b0000.sdhci] using ADMA
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
+elan_i2c 1-0015: invalid report id data (ff)
 mmc2: Unknown controller version (3). You may experience problems.
 mmc2: Invalid maximum block size, assuming 512 bytes
-elan_i2c 1-0015: invalid report id data (ff)
 mmc1: new high speed SDIO card at address 0001
 mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
@@ -446,7 +447,7 @@
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-as3722-rtc as3722-rtc: setting system clock to 2000-01-02 16:49:45 UTC (946831785)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:23 UTC (946684823)
 +1.8V_RUN_CAM: disabling
 +3.3V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2017-09-04 02:43:29.565863977 -0700
+++ /tmp/foob	2017-09-04 02:43:29.569863995 -0700
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -41,20 +41,20 @@
 	 #################################################################
 	 #################################################################
 	 #######################################
-	 3 MiB/s
+	 3.5 MiB/s
 done
-Bytes transferred = 6287120 (5fef10 hex)
+Bytes transferred = 6291280 (5fff50 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.1 MiB/s
+	 2.8 MiB/s
 done
 Bytes transferred = 68203 (10a6b hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5fef10 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5fff50 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013a6a
@@ -63,7 +63,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc7-next-20170901-198696-g1d53d90 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Fri Sep 1 00:05:47 PDT 2017
+Linux version 4.13.0-next-20170904-199136-g9829d9f (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Sep 4 02:35:36 PDT 2017
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -378,6 +378,7 @@
 tegra-snd-max98090 sound: HiFi <-> 70301100.i2s mapping ok
 mmc0: new high speed MMC card at address 0001
 input: GoogleNyanBig Headphones as /devices/soc0/sound/sound/card1/input1
+mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
 input: GoogleNyanBig Mic Jack as /devices/soc0/sound/sound/card1/input2
 NET: Registered protocol family 10
 Segment Routing with IPv6
@@ -415,9 +416,18 @@
  usb3-1: usb3-1 supply vbus not found, using dummy regulator
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
-tegra-xusb 70090000.usb: Falcon failed to start, state: 0x0
-tegra-xusb 70090000.usb: failed to load firmware: -5
-tegra-xusb: probe of 70090000.usb failed with error -5
+tegra-xusb 70090000.usb: Firmware timestamp: 2014-09-16 02:10:07 UTC
+tegra-xusb 70090000.usb: xHCI Host Controller
+tegra-xusb 70090000.usb: new USB bus registered, assigned bus number 1
+tegra-xusb 70090000.usb: hcc params 0x0184f525 hci version 0x100 quirks 0x00010010
+tegra-xusb 70090000.usb: irq 344, io mem 0x70090000
+hub 1-0:1.0: USB hub found
+hub 1-0:1.0: 6 ports detected
+tegra-xusb 70090000.usb: xHCI Host Controller
+tegra-xusb 70090000.usb: new USB bus registered, assigned bus number 2
+usb usb2: We don't know the algorithms for LPM for this host, disabling LPM.
+hub 2-0:1.0: USB hub found
+hub 2-0:1.0: 2 ports detected
 sdhci-tegra 700b0000.sdhci: allocated mmc-pwrseq
 mmc1: Unknown controller version (3). You may experience problems.
 mmc1: Invalid maximum block size, assuming 512 bytes
@@ -429,7 +439,6 @@
 mmc2: Invalid maximum block size, assuming 512 bytes
 mmc1: new high speed SDIO card at address 0001
 mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
-mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
 mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
 mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
 mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB
@@ -445,10 +454,11 @@
 GPT: Use GNU Parted to correct GPT errors.
  mmcblk2: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 Console: switching to colour frame buffer device 170x48
+usb 1-2: new high-speed USB device number 2 using tegra-xusb
 drm drm: fb0:  frame buffer device
 [drm] Initialized tegra 0.0.0 20120330 for drm on minor 0
 input: gpio-keys as /devices/soc0/gpio-keys/input/input3
-as3722-rtc as3722-rtc: setting system clock to 2000-01-02 16:47:35 UTC (946831655)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:23 UTC (946684823)
 +1.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
 +1.05V_RUN_CAM_REAR: disabling
@@ -456,11 +466,8 @@
 +2.8V_RUN_CAM_AF: disabling
 +1.8V_RUN_VPP_FUSE: disabling
 +5V_VDD_TS_SW: disabling
-+5V_USB_HS: disabling
-+5V_USB_SS: disabling
 ALSA device list:
   #1: GoogleNyanBig
-alloc_contig_range: [fbc70, fbc80) PFNs busy
 alloc_contig_range: [fbcb0, fbcc0) PFNs busy
 alloc_contig_range: [fbcc0, fbcd0) PFNs busy
 alloc_contig_range: [fbcd0, fbce0) PFNs busy
@@ -470,8 +477,12 @@
 alloc_contig_range: [fbd10, fbd20) PFNs busy
 alloc_contig_range: [fbd20, fbd30) PFNs busy
 alloc_contig_range: [fbd30, fbd40) PFNs busy
-EXT4-fs (mmcblk0p3): couldn't mount as ext3 due to feature incompatibilities
+alloc_contig_range: [fbd40, fbd50) PFNs busy
 input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input4
+uvcvideo: Found UVC 1.00 device HD WebCam (1bcf:2c67)
+input: HD WebCam: HD WebCam as /devices/soc0/70090000.usb/usb1/1-2/1-2:1.0/input/input5
+EXT4-fs (mmcblk0p3): couldn't mount as ext3 due to feature incompatibilities
+usb 1-3: new high-speed USB device number 3 using tegra-xusb
 EXT4-fs (mmcblk0p3): recovery complete
 EXT4-fs (mmcblk0p3): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:3.
@@ -479,4 +490,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# 
\ No newline at end of file
+root@(none):/# [    4.047301] asix 1-3:1.0 eth0: register 'asix' at usb-70090000.usb-3, ASIX AX88772 USB 2.0 Ethernet, c8:d7:19:90:fa:65
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2017-09-04 02:43:29.581864046 -0700
+++ /tmp/foob	2017-09-04 02:43:29.585864063 -0700
@@ -31,7 +31,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1801390 us
+Bootloader-Cpu Init at (time stamp): 1801012 us
 
 Pinmux changes applied in kernel way
 
@@ -62,22 +62,22 @@
 Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1801364  us
-Bootloader (AosInit) time: 733439 us
-Bootloader Main Init time: 180515 us
-BlockDevMgrInit time: 157637 us
+Miscellaneous avp time: 1800986  us
+Bootloader (AosInit) time: 734238 us
+Bootloader Main Init time: 180102 us
+BlockDevMgrInit time: 157532 us
 PartMgrInit time: 8 us
-StorMgrInit time: 31 us
+StorMgrInit time: 30 us
 FsMgrInit time: 2 us
-Partition Table load: 77018 us 
-Display init time: 574463 us
+Partition Table load: 76635 us 
+Display init time: 574967 us
 Kernel read time: 1345 msec
 Load Kernel Size:   17827840 Bytes
 Bl Kernel ReadSpeed: 13254000 Bytes/sec
-Pre-Kernel Jump time: 3840048 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 8806262 us
+Pre-Kernel Jump time: 3839895 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 8806110 us
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc7-next-20170901-198696-g1d53d90 (jonathanh@sellafield) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Fri Sep 1 00:08:57 PDT 2017
+Linux version 4.13.0-next-20170904-317377-g9829d9f31f6c (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Mon Sep 4 02:41:15 PDT 2017
 Boot CPU: AArch64 Processor [4e0f0000]
 Machine model: NVIDIA Tegra132 Norrin
 Memory limited to 1734MB
@@ -102,7 +102,7 @@
 Policy zone: DMA
 Kernel command line: tegraid=13.1.3.0.0 mem=1734M@2048M memtype=255 vpr=151M@3817M ddr_die=2048M@2048M section=256M commchip_id=0 usb_port_owner_info=0 lane_owner_info=0 emc_max_dvfs=0 touch_id=0@0 video=tegrafb no_console_suspend=1 console=ttyS0,115200n8 debug_uartport=lsport,0 earlyprintk=uart8250-32bit,0x70006000 usbcore.old_scheme_first=1 lp0_vec=0@0x0 tegra_fbmem=32899072@0xad012000 core_edp_mv=1150 core_edp_ma=4000 pmuboard=0x0176:0x0001:0x01:0x43:0x00 displayboard=0x0162:0x0001:0x00:0x42:0x00 power_supply=Adapter audio_codec=rt5640 board_info=0x0176:0x0001:0x01:0x43:0x00 tegraboot=sdmmc gpt gpt_sector=192511 modem_id=0 android.kerneltype=normal
 PID hash table entries: 4096 (order: 3, 32768 bytes)
-Memory: 1708196K/1775616K available (9212K kernel code, 1144K rwdata, 4560K rodata, 1152K init, 404K bss, 51036K reserved, 16384K cma-reserved)
+Memory: 1708196K/1775616K available (9212K kernel code, 1144K rwdata, 4568K rodata, 1152K init, 404K bss, 51036K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
@@ -202,7 +202,7 @@
 Freeing initrd memory: 1284K
 kvm [1]: HYP mode not available
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.983:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.991:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=19 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -273,15 +273,14 @@
 mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 +VDD_LED: supplied by +VDD_MUX
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8
 +5V_USB_HS: supplied by +5V_SYS
 +5V_USB_SS: supplied by +5V_SYS
 +5V_HDMI_CON: supplied by +5V_SYS
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
-sdhci-tegra 700b0400.sdhci: Got CD GPIO
 hctosys: unable to open rtc device (rtc0)
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +VDD_LED: disabling
@@ -291,8 +290,9 @@
 +5V_HDMI_CON: disabling
 ALSA device list:
   No soundcards found.
-sdhci-tegra 700b0400.sdhci: Got WP GPIO
 Freeing unused kernel memory: 1152K
+sdhci-tegra 700b0400.sdhci: Got CD GPIO
+sdhci-tegra 700b0400.sdhci: Got WP GPIO
 
 System initialization...
 
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-09-04 02:43:29.589864080 -0700
+++ /tmp/foob	2017-09-04 02:43:29.593864097 -0700
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -48,9 +48,9 @@
 	 #################################################################
 	 #################################################################
 	 ####################################
-	 3.6 MiB/s
+	 4.1 MiB/s
 done
-Bytes transferred = 8151560 (7c6208 hex)
+Bytes transferred = 8159752 (7c8208 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -61,7 +61,7 @@
 done
 Bytes transferred = 21454 (53ce hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x7c6200 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x7c8200 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103cd
@@ -69,7 +69,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc7-next-20170901-198696-g1d53d90 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Fri Sep 1 00:05:43 PDT 2017
+Linux version 4.13.0-next-20170904-199136-g9829d9f (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Mon Sep 4 02:36:12 PDT 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -79,7 +79,7 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0x3c000000
 On node 0 totalpages: 262144
-free_area_init_node: node 0, pgdat c158e880, node_mem_map ef7f8000
+free_area_init_node: node 0, pgdat c158e440, node_mem_map ef7f8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -93,7 +93,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 954308K/1048576K available (10240K kernel code, 1626K rwdata, 4488K rodata, 2048K init, 405K bss, 28732K reserved, 65536K cma-reserved, 196608K highmem)
+Memory: 954308K/1048576K available (10240K kernel code, 1625K rwdata, 4500K rodata, 2048K init, 405K bss, 28732K reserved, 65536K cma-reserved, 196608K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -103,7 +103,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1596bd0   (1627 kB)
+      .data : 0xc1400000 - 0xc1596790   (1626 kB)
        .bss : 0xc159fbc0 - 0xc16052fc   ( 406 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
 Hierarchical RCU implementation.
@@ -346,22 +346,22 @@
 Key type dns_resolver registered
 ThumbEE CPU extension supported.
 Registering SWP/SWPB emulation handler
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
-mmc1: new high speed SD card at address b368
-mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
- mmcblk1: p1 p2
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-hub 1-1:1.0: USB hub found
-hub 1-1:1.0: 4 ports detected
 rtc-em3027 2-0056: hctosys: unable to read the hardware clock
 Waiting for root device PARTUUID=11eb23c1-01...
+mmc1: new high speed SD card at address b368
+mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
+ mmcblk1: p1 p2
+hub 1-1:1.0: USB hub found
+hub 1-1:1.0: 4 ports detected
 usb 1-1.2: new high-speed USB device number 3 using tegra-ehci
 mmc0: host does not support reading read-only switch, assuming write-enable
 mmc0: new high speed SDHC card at address b368
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2017-09-04 02:43:29.613864183 -0700
+++ /tmp/foob	2017-09-04 02:43:29.617864200 -0700
@@ -46,9 +46,9 @@
 	 #################################################################
 	 #################################################################
 	 #######################################
-	 3.4 MiB/s
+	 3.7 MiB/s
 done
-Bytes transferred = 6287120 (5fef10 hex)
+Bytes transferred = 6291280 (5fff50 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -59,7 +59,7 @@
 done
 Bytes transferred = 21454 (53ce hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x5fef10 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x5fff50 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024103cd
@@ -68,7 +68,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc7-next-20170901-198696-g1d53d90 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Fri Sep 1 00:05:47 PDT 2017
+Linux version 4.13.0-next-20170904-199136-g9829d9f (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Sep 4 02:35:36 PDT 2017
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -326,10 +326,10 @@
 NET: Registered protocol family 15
 can: controller area network core (rev 20170425 abi 9)
 NET: Registered protocol family 29
-usb 1-1: new high-speed USB device number 2 using tegra-ehci
 can: raw protocol (rev 20170425)
 can: broadcast manager protocol (rev 20170425 t)
 can: netlink gateway (rev 20170425) max_hops=1
+usb 1-1: new high-speed USB device number 2 using tegra-ehci
 Bluetooth: RFCOMM socket layer initialized
 Bluetooth: RFCOMM ver 1.11
 Bluetooth: BNEP (Ethernet Emulation) ver 1.3
@@ -340,16 +340,16 @@
 Registering SWP/SWPB emulation handler
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
+ mmcblk1: p1 p2
 utmi_phy_clk_enable: timeout waiting for phy to stabilize
 tegra-ehci c5000000.usb: EHCI Host Controller
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 288, io mem 0xc5000000
- mmcblk1: p1 p2
 tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
 hub 2-0:1.0: 1 port detected
-input: gpio-keys as /devices/soc0/gpio-keys/input/input0
 hub 1-1:1.0: USB hub found
+input: gpio-keys as /devices/soc0/gpio-keys/input/input0
 hub 1-1:1.0: 4 ports detected
 rtc-em3027 2-0056: hctosys: unable to read the hardware clock
 ALSA device list:
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2017-09-04 02:43:29.629864252 -0700
+++ /tmp/foob	2017-09-04 02:43:29.629864252 -0700
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 798943 us
-Basic modules init took 348783 us
+Tegraboot started after 751468 us
+Basic modules init took 350889 us
 Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14362 us
-Carveout took 252716 us
-CPU initialization took 92444 us
-Total time taken by TegraBoot 708305 us
+Next stage binary read took 14373 us
+Carveout took 252710 us
+CPU initialization took 92426 us
+Total time taken by TegraBoot 710398 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.637] LPDDR4 Training: Number of tables = 10
+64b[0001.592] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-16507392 bytes read in 837 ms (18.8 MiB/s)
+16507392 bytes read in 835 ms (18.9 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-45170 bytes read in 454 ms (96.7 KiB/s)
+45170 bytes read in 456 ms (96.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 496 ms (3.1 MiB/s)
+1627787 bytes read in 495 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc7-next-20170901-198696-g1d53d90 (jonathanh@sellafield) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Fri Sep 1 00:08:57 PDT 2017
+Linux version 4.13.0-next-20170904-317377-g9829d9f31f6c (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Mon Sep 4 02:41:15 PDT 2017
 Boot CPU: AArch64 Processor [411fd071]
 Machine model: NVIDIA Tegra210 P2371 (P2530/P2595) reference design
 debug: ignoring loglevel setting.
@@ -173,7 +173,7 @@
 Kernel command line: ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xfa000000-0xfe000000] (64MB) mapped at [ffff80007a000000-ffff80007dffffff]
-Memory: 2982560K/3131392K available (9212K kernel code, 1144K rwdata, 4560K rodata, 1152K init, 404K bss, 132448K reserved, 16384K cma-reserved)
+Memory: 2982560K/3131392K available (9212K kernel code, 1144K rwdata, 4568K rodata, 1152K init, 404K bss, 132448K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
@@ -270,14 +270,14 @@
 RPC: Registered tcp NFSv4.1 backchannel transport module.
 PCI: CLS 0 bytes, default 128
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 80973000
+kvm [1]: IDMAP page: 80974000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
 kvm [1]: vgic interrupt IRQ1
 kvm [1]: virtual timer IRQ4
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.515:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.539:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2017-09-04 02:43:29.637864286 -0700
+++ /tmp/foob	2017-09-04 02:43:29.641864304 -0700
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc7-next-20170901-198696-g1d53d90 (jonathanh@sellafield) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Fri Sep 1 00:08:57 PDT 2017
+Linux version 4.13.0-next-20170904-317377-g9829d9f31f6c (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Mon Sep 4 02:41:15 PDT 2017
 Boot CPU: AArch64 Processor [411fd071]
 Machine model: Google Pixel C
 efi: Getting EFI parameters from FDT:
@@ -31,7 +31,7 @@
 Kernel command line: cros_secure vpr=0x08000000@0xf6800000 earlycon
 PID hash table entries: 4096 (order: 3, 32768 bytes)
 software IO TLB [mem 0xf1400000-0xf5400000] (64MB) mapped at [ffff800071400000-ffff8000753fffff]
-Memory: 2837828K/2988032K available (9212K kernel code, 1144K rwdata, 4560K rodata, 1152K init, 404K bss, 133820K reserved, 16384K cma-reserved)
+Memory: 2837828K/2988032K available (9212K kernel code, 1144K rwdata, 4568K rodata, 1152K init, 404K bss, 133820K reserved, 16384K cma-reserved)
 Virtual kernel memory layout:
     modules : 0xffff000000000000 - 0xffff000008000000   (   128 MB)
     vmalloc : 0xffff000008000000 - 0xffff7dffbfff0000   (129022 GB)
@@ -152,14 +152,14 @@
 Unpacking initramfs...
 Freeing initrd memory: 1280K
 kvm [1]: 8-bit VMID
-kvm [1]: IDMAP page: 80973000
+kvm [1]: IDMAP page: 80974000
 kvm [1]: HYP VA range: 800000000000:ffffffffffff
 kvm [1]: Hyp mode initialized successfully
 kvm [1]: vgic-v2@50044000
 kvm [1]: vgic interrupt IRQ1
 kvm [1]: virtual timer IRQ4
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.255:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.259:1): state=initialized audit_enabled=0 res=1
 workingset: timestamp_bits=44 max_order=20 bucket_order=0
 squashfs: version 4.0 (2009/01/31) Phillip Lougher
 NFS: Registering the id_resolver key type
@@ -262,7 +262,7 @@
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
-max77686-rtc max77620-rtc: setting system clock to 2001-06-12 12:33:33 UTC (992349213)
+max77686-rtc max77620-rtc: setting system clock to 2001-06-15 15:06:02 UTC (992617562)
 PP1200_AVDD: disabling
 PP1200_RCAM: disabling
 PP2800L_RCAM: disabling
@@ -273,8 +273,8 @@
 PP1800_CAM: disabling
 ALSA device list:
   No soundcards found.
-tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 Freeing unused kernel memory: 1152K
+tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
 
 System initialization...
 
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2017-09-04 02:43:29.645864320 -0700
+++ /tmp/foob	2017-09-04 02:43:29.649864338 -0700
@@ -40,20 +40,21 @@
 	 #################################################################
 	 #################################################################
 	 ####################################
-	 3.3 MiB/s
+	 1 MiB/s
 done
-Bytes transferred = 8151560 (7c6208 hex)
+Bytes transferred = 8159752 (7c8208 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
+Waiting for Ethernet connection... done.
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.9 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 51860 (ca94 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x7c6200 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x7c8200 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa93 ... OK
@@ -61,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc7-next-20170901-198696-g1d53d90 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Fri Sep 1 00:05:43 PDT 2017
+Linux version 4.13.0-next-20170904-199136-g9829d9f (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Mon Sep 4 02:36:12 PDT 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -71,7 +72,7 @@
 efi: UEFI not found.
 cma: Reserved 64 MiB at 0xfb800000
 On node 0 totalpages: 524032
-free_area_init_node: node 0, pgdat c158e880, node_mem_map eeff8000
+free_area_init_node: node 0, pgdat c158e440, node_mem_map eeff8000
   DMA zone: 1536 pages used for memmap
   DMA zone: 0 pages reserved
   DMA zone: 196608 pages, LIFO batch:31
@@ -85,7 +86,7 @@
 PID hash table entries: 4096 (order: 2, 16384 bytes)
 Dentry cache hash table entries: 131072 (order: 7, 524288 bytes)
 Inode-cache hash table entries: 65536 (order: 6, 262144 bytes)
-Memory: 1993400K/2096128K available (10240K kernel code, 1626K rwdata, 4488K rodata, 2048K init, 405K bss, 37192K reserved, 65536K cma-reserved, 1244160K highmem)
+Memory: 1993400K/2096128K available (10240K kernel code, 1625K rwdata, 4500K rodata, 2048K init, 405K bss, 37192K reserved, 65536K cma-reserved, 1244160K highmem)
 Virtual kernel memory layout:
     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
@@ -95,7 +96,7 @@
     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
       .text : 0xc0208000 - 0xc0d00000   (11232 kB)
       .init : 0xc1200000 - 0xc1400000   (2048 kB)
-      .data : 0xc1400000 - 0xc1596bd0   (1627 kB)
+      .data : 0xc1400000 - 0xc1596790   (1626 kB)
        .bss : 0xc159fbc0 - 0xc16052fc   ( 406 kB)
 SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
 Hierarchical RCU implementation.
@@ -386,18 +387,18 @@
 sys_3v3_pexs: supplied by sys_3v3,vdd_3v3_alw
 +VDD_5V_HDMI: supplied by sys_3v3,vdd_3v3_alw
 tegra-pcie 3000.pcie: 2x3 configuration
+tegra-pcie 3000.pcie: probing port 0, using 2 lanes
 mmc1: new high speed MMC card at address 0001
+mmc0: new ultra high speed DDR50 SDHC card at address aaaa
+mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
 mmcblk1: mmc1:0001 SEM16G 14.8 GiB 
 mmcblk1boot0: mmc1:0001 SEM16G partition 1 1.00 MiB
 mmcblk1boot1: mmc1:0001 SEM16G partition 2 1.00 MiB
+ mmcblk0: p1 p2 p3
 mmcblk1rpmb: mmc1:0001 SEM16G partition 3 128 KiB
-tegra-pcie 3000.pcie: probing port 0, using 2 lanes
+tegra-pcie 3000.pcie: probing port 2, using 2 lanes
 Primary GPT is invalid, using alternate GPT.
  mmcblk1: p1 p2 p3 p4 p5
-mmc0: new ultra high speed DDR50 SDHC card at address aaaa
-mmcblk0: mmc0:aaaa SS04G 3.69 GiB 
- mmcblk0: p1 p2 p3
-tegra-pcie 3000.pcie: probing port 2, using 2 lanes
 tegra-pcie 3000.pcie: link 2 down, retrying
 tegra-pcie 3000.pcie: link 2 down, retrying
 tegra-pcie 3000.pcie: link 2 down, retrying
@@ -469,9 +470,9 @@
 EXT4-fs (mmcblk1p1): warning: mounting fs with errors, running e2fsck is recommended
 EXT4-fs (mmcblk1p1): recovery complete
 EXT4-fs (mmcblk1p1): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext3 filesystem) on device 179:1.
+VFS: Mounted root (ext3 filesystem) on device 179:17.
 devtmpfs: mounted
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.544567] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.493639] usb 2-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2017-09-04 02:43:29.653864355 -0700
+++ /tmp/foob	2017-09-04 02:43:29.657864372 -0700
@@ -38,9 +38,9 @@
 	 #################################################################
 	 #################################################################
 	 #######################################
-	 2.9 MiB/s
+	 2.8 MiB/s
 done
-Bytes transferred = 6287120 (5fef10 hex)
+Bytes transferred = 6291280 (5fff50 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
@@ -51,7 +51,7 @@
 done
 Bytes transferred = 51860 (ca94 hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5fef10 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5fff50 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffffa93 ... OK
@@ -60,7 +60,7 @@
 
 Uncompressing Linux... done, booting the kernel.
 Booting Linux on physical CPU 0x0
-Linux version 4.13.0-rc7-next-20170901-198696-g1d53d90 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Fri Sep 1 00:05:47 PDT 2017
+Linux version 4.13.0-next-20170904-199136-g9829d9f (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Mon Sep 4 02:35:36 PDT 2017
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -469,6 +469,7 @@
 VFS: Mounted root (ext3 filesystem) on device 179:1.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K
+usb 2-1: new high-speed USB device number 2 using tegra-ehci
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# 
\ No newline at end of file
+root@(none):/# [    5.506822] asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0f:67:60
