Diff for ./qemu-vexpress64/__internal/defconfig_log.txt
--- /tmp/fooa	2018-02-08 19:48:23.012896665 -0800
+++ /tmp/foob	2018-02-08 19:48:23.012896665 -0800
@@ -1,191 +0,0 @@
-Booting Linux on physical CPU 0x0000000000 [0x411fd070]
-Linux version 4.15.0-next-20180208-349377-g957f77188c8b (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Feb 7 19:23:46 PST 2018
-Machine model: linux,dummy-virt
-Memory limited to 1024MB
-efi: Getting EFI parameters from FDT:
-efi: UEFI not found.
-cma: Reserved 16 MiB at 0x000000007f000000
-NUMA: No NUMA configuration found
-NUMA: Faking a node at [mem 0x0000000000000000-0x000000007fffffff]
-NUMA: NODE_DATA [mem 0x7efeb000-0x7efecaff]
-Zone ranges:
-  DMA32    [mem 0x0000000040000000-0x000000007fffffff]
-  Normal   empty
-Movable zone start for each node
-Early memory node ranges
-  node   0: [mem 0x0000000040000000-0x000000007fffffff]
-Initmem setup node 0 [mem 0x0000000040000000-0x000000007fffffff]
-psci: probing for conduit method from DT.
-psci: PSCIv0.2 detected in firmware.
-psci: Using standard PSCI v0.2 function IDs
-psci: Trusted OS migration not required
-random: fast init done
-percpu: Embedded 24 pages/cpu @        (ptrval) s57984 r8192 d32128 u98304
-Detected PIPT I-cache on CPU0
-CPU features: enabling workaround for ARM erratum 832075
-CPU features: enabling workaround for ARM erratum 834220
-Built 1 zonelists, mobility grouping on.  Total pages: 258048
-Policy zone: DMA32
-Kernel command line: root=/dev/vda2 rw rootwait mem=1024M console=ttyAMA0,38400n8 init=/bin/bash
-Memory: 998052K/1048576K available (9980K kernel code, 1252K rwdata, 4628K rodata, 1152K init, 404K bss, 34140K reserved, 16384K cma-reserved)
-SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
-Preemptible hierarchical RCU implementation.
-	RCU restricting CPUs from NR_CPUS=64 to nr_cpu_ids=1.
-	Tasks RCU enabled.
-RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=1
-NR_IRQS: 64, nr_irqs: 64, preallocated irqs: 0
-GICv2m: range[mem 0x08020000-0x08020fff], SPI[80:143]
-arch_timer: cp15 timer(s) running at 62.50MHz (virt).
-clocksource: arch_sys_counter: mask: 0xffffffffffffff max_cycles: 0x1cd42e208c, max_idle_ns: 881590405314 ns
-sched_clock: 56 bits at 62MHz, resolution 16ns, wraps every 4398046511096ns
-Console: colour dummy device 80x25
-Calibrating delay loop (skipped), value calculated using timer frequency.. 125.00 BogoMIPS (lpj=250000)
-pid_max: default: 32768 minimum: 301
-Security Framework initialized
-Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
-Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
-Mount-cache hash table entries: 2048 (order: 2, 16384 bytes)
-Mountpoint-cache hash table entries: 2048 (order: 2, 16384 bytes)
-ASID allocator initialised with 32768 entries
-Hierarchical SRCU implementation.
-EFI services will not be available.
-smp: Bringing up secondary CPUs ...
-smp: Brought up 1 node, 1 CPU
-SMP: Total of 1 processors activated.
-CPU features: detected feature: 32-bit EL0 Support
-CPU features: detected feature: Kernel page table isolation (KPTI)
-CPU: All CPU(s) started at EL1
-alternatives: patching kernel code
-devtmpfs: initialized
-Built 1 zonelists, mobility grouping on.  Total pages: 253609
-Policy zone: DMA32
-clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 7645041785100000 ns
-futex hash table entries: 256 (order: 3, 32768 bytes)
-pinctrl core: initialized pinctrl subsystem
-DMI not present or invalid.
-NET: Registered protocol family 16
-audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.196:1): state=initialized audit_enabled=0 res=1
-cpuidle: using governor menu
-vdso: 2 pages (1 code @ 000000008fc570bd, 1 data @ 00000000f0084c20)
-hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
-DMA: preallocated 256 KiB pool for atomic allocations
-Serial: AMBA PL011 UART driver
-9000000.pl011: ttyAMA0 at MMIO 0x9000000 (irq = 39, base_baud = 0) is a PL011 rev1
-console [ttyAMA0] enabled
-HugeTLB registered 2.00 MiB page size, pre-allocated 0 pages
-cryptd: max_cpu_qlen set to 1000
-ACPI: Interpreter disabled.
-vgaarb: loaded
-SCSI subsystem initialized
-usbcore: registered new interface driver usbfs
-usbcore: registered new interface driver hub
-usbcore: registered new device driver usb
-pps_core: LinuxPPS API ver. 1 registered
-pps_core: Software ver. 5.3.6 - Copyright 2005-2007 Rodolfo Giometti <giometti@linux.it>
-PTP clock support registered
-EDAC MC: Ver: 3.0.0
-Advanced Linux Sound Architecture Driver Initialized.
-clocksource: Switched to clocksource arch_sys_counter
-VFS: Disk quotas dquot_6.6.0
-VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
-pnp: PnP ACPI: disabled
-NET: Registered protocol family 2
-tcp_listen_portaddr_hash hash table entries: 512 (order: 1, 8192 bytes)
-TCP established hash table entries: 8192 (order: 4, 65536 bytes)
-TCP bind hash table entries: 8192 (order: 5, 131072 bytes)
-TCP: Hash tables configured (established 8192 bind 8192)
-UDP hash table entries: 512 (order: 2, 16384 bytes)
-UDP-Lite hash table entries: 512 (order: 2, 16384 bytes)
-NET: Registered protocol family 1
-RPC: Registered named UNIX socket transport module.
-RPC: Registered udp transport module.
-RPC: Registered tcp transport module.
-RPC: Registered tcp NFSv4.1 backchannel transport module.
-hw perfevents: enabled with armv8_pmuv3 PMU driver, 1 counters available
-kvm [1]: HYP mode not available
-Initialise system trusted keyrings
-workingset: timestamp_bits=44 max_order=18 bucket_order=0
-squashfs: version 4.0 (2009/01/31) Phillip Lougher
-NFS: Registering the id_resolver key type
-Key type id_resolver registered
-Key type id_legacy registered
-nfs4filelayout_init: NFSv4 File Layout Driver Registering...
-9p: Installing v9fs 9p2000 file system support
-Key type asymmetric registered
-Asymmetric key parser 'x509' registered
-Block layer SCSI generic (bsg) driver version 0.4 loaded (major 245)
-io scheduler noop registered
-io scheduler cfq registered (default)
-io scheduler mq-deadline registered
-io scheduler kyber registered
-pl061_gpio 9030000.pl061: PL061 GPIO chip @0x0000000009030000 registered
-PCI: OF: host bridge /pcie@10000000 ranges:
-PCI: OF:    IO 0x3eff0000..0x3effffff -> 0x00000000
-PCI: OF:   MEM 0x10000000..0x3efeffff -> 0x10000000
-PCI: OF:   MEM 0x8000000000..0xffffffffff -> 0x8000000000
-pci-host-generic 3f000000.pcie: ECAM at [mem 0x3f000000-0x3fffffff] for [bus 00-0f]
-pci-host-generic 3f000000.pcie: PCI host bridge to bus 0000:00
-pci_bus 0000:00: root bus resource [bus 00-0f]
-pci_bus 0000:00: root bus resource [io  0x0000-0xffff]
-pci_bus 0000:00: root bus resource [mem 0x10000000-0x3efeffff]
-pci_bus 0000:00: root bus resource [mem 0x8000000000-0xffffffffff]
-EINJ: ACPI disabled.
-Serial: 8250/16550 driver, 4 ports, IRQ sharing enabled
-SuperH (H)SCI(F) driver initialized
-msm_serial: driver initialized
-cacheinfo: Unable to detect cache hierarchy for CPU 0
-loop: module loaded
-virtio_blk virtio0: [vda] 6291456 512-byte logical blocks (3.22 GB/3.00 GiB)
- vda: vda1 vda2
-libphy: Fixed MDIO Bus: probed
-tun: Universal TUN/TAP device driver, 1.6
-thunder_xcv, ver 1.0
-thunder_bgx, ver 1.0
-nicpf, ver 1.0
-e1000e: Intel(R) PRO/1000 Network Driver - 3.2.6-k
-e1000e: Copyright(c) 1999 - 2015 Intel Corporation.
-igb: Intel(R) Gigabit Ethernet Network Driver - version 5.4.0-k
-igb: Copyright (c) 2007-2014 Intel Corporation.
-igbvf: Intel(R) Gigabit Virtual Function Network Driver - version 2.4.0-k
-igbvf: Copyright (c) 2009 - 2012 Intel Corporation.
-sky2: driver version 1.30
-VFIO - User Level meta-driver version: 0.3
-ehci_hcd: USB 2.0 'Enhanced' Host Controller (EHCI) Driver
-ehci-pci: EHCI PCI platform driver
-ehci-platform: EHCI generic platform driver
-ehci-orion: EHCI orion driver
-ehci-exynos: EHCI EXYNOS driver
-ohci_hcd: USB 1.1 'Open' Host Controller (OHCI) Driver
-ohci-pci: OHCI PCI platform driver
-ohci-platform: OHCI generic platform driver
-ohci-exynos: OHCI EXYNOS driver
-usbcore: registered new interface driver usb-storage
-rtc-pl031 9010000.pl031: rtc core: registered pl031 as rtc0
-i2c /dev entries driver
-sdhci: Secure Digital Host Controller Interface driver
-sdhci: Copyright(c) Pierre Ossman
-Synopsys Designware Multimedia Card Interface Driver
-sdhci-pltfm: SDHCI platform and OF driver helper
-ledtrig-cpu: registered to indicate activity on CPUs
-usbcore: registered new interface driver usbhid
-usbhid: USB HID core driver
-NET: Registered protocol family 17
-9pnet: Installing 9P2000 support
-Key type dns_resolver registered
-registered taskstats version 1
-Loading compiled-in X.509 certificates
-input: gpio-keys as /devices/platform/gpio-keys/input/input0
-rtc-pl031 9010000.pl031: setting system clock to 2018-02-08 03:30:49 UTC (1518060649)
-ALSA device list:
-  No soundcards found.
-uart-pl011 9000000.pl011: no DMA platform data
-EXT4-fs (vda2): couldn't mount as ext3 due to feature incompatibilities
-EXT4-fs (vda2): recovery complete
-EXT4-fs (vda2): mounted filesystem with ordered data mode. Opts: (null)
-VFS: Mounted root (ext4 filesystem) on device 254:2.
-devtmpfs: mounted
-Freeing unused kernel memory: 1152K
-bash: cannot set terminal process group (-1): Inappropriate ioctl for device
-bash: no job control in this shell
-bash-4.3# 
\ No newline at end of file
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/multi_v7_defconfig_log.txt
--- /tmp/fooa	2018-02-08 19:48:23.056896864 -0800
+++ /tmp/foob	2018-02-08 19:48:23.056896864 -0800
@@ -40,7 +40,7 @@
 	 #################################################################
 	 #################################################################
 	 #####################################
-	 2.9 MiB/s
+	 3 MiB/s
 done
 Bytes transferred = 8167936 (7ca200 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-next-20180208-349377-g957f77188c8b (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Feb 7 19:17:40 PST 2018
+Linux version 4.15.0-next-20180209-349478-g611f09f4907a (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Feb 8 19:04:12 PST 2018
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -353,7 +353,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-07-01 10:23:35 UTC (962447015)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-07-02 10:09:08 UTC (962532548)
 dvdd_ts: disabling
 usb1_vbus: disabling
 vdd_cam_1v8_reg: disabling
@@ -365,4 +365,4 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.739329] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.749267] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra114-dalmore-a04/tegra114-dalmore/tegra_defconfig_log.txt
--- /tmp/fooa	2018-02-08 19:48:23.092897026 -0800
+++ /tmp/foob	2018-02-08 19:48:23.092897026 -0800
@@ -38,20 +38,20 @@
 	 #################################################################
 	 #################################################################
 	 ##################################
-	 2.9 MiB/s
+	 3 MiB/s
 done
-Bytes transferred = 6221496 (5eeeb8 hex)
+Bytes transferred = 6221752 (5eefb8 hex)
 Tegra114 (Dalmore) # tftpboot 0x82000000 tegra114-dalmore.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.100
 Filename 'tegra114-dalmore.dtb'.
 Load address: 0x82000000
 Loading: *###
-	 2.5 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 34502 (86c6 hex)
 Tegra114 (Dalmore) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5eeeb8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5eefb8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff4000, end 8ffff6c5 ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-next-20180208-29428-g957f771 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Feb 7 19:16:46 PST 2018
+Linux version 4.15.0-next-20180209-29529-g611f09f (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Feb 8 19:02:48 PST 2018
 CPU: ARMv7 Processor [412fc0f2] revision 2 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -343,7 +343,7 @@
 hub 1-0:1.0: USB hub found
 hub 1-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
-palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-07-01 10:20:55 UTC (962446855)
+palmas-rtc 7000d000.i2c:tps65913@58:rtc: setting system clock to 2000-07-02 10:06:21 UTC (962532381)
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
@@ -361,4 +361,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    3.386626] usb 1-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    3.363852] usb 1-1: new high-speed USB device number 2 using tegra-ehci
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/multi_v7_defconfig_log.txt
--- /tmp/fooa	2018-02-08 19:48:23.132897208 -0800
+++ /tmp/foob	2018-02-08 19:48:23.136897226 -0800
@@ -35,7 +35,7 @@
 	 #################################################################
 	 #################################################################
 	 #####################################
-	 3.7 MiB/s
+	 3.9 MiB/s
 done
 Bytes transferred = 8167936 (7ca200 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
@@ -56,7 +56,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-next-20180208-349377-g957f77188c8b (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Feb 7 19:17:40 PST 2018
+Linux version 4.15.0-next-20180209-349478-g611f09f4907a (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Feb 8 19:04:12 PST 2018
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -325,25 +325,25 @@
 +3.3V_LP0: supplied by +3.3V_SYS
 +5V_HDMI_CON: supplied by +5V_SYS
 +5V_SATA: supplied by +5V_SYS
-mmc0: new high speed MMC card at address 0001
 +12V_SATA: supplied by +VDD_MUX
-mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
  usb2-0: usb2-0 supply vbus not found, using dummy regulator
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
  usb2-1: usb2-1 supply vbus not found, using dummy regulator
-mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
-mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB, chardev (236:0)
+mmc0: new high speed MMC card at address 0001
 tegra-pcie 1003000.pcie: 2x1, 1x1 configuration
+mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 tegra-pcie 1003000.pcie: Failed to get supply 'avddio-pex': -517
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 tegra124-dfll 70110000.clock: couldn't get vdd_cpu regulator
- mmcblk0: p1
+mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 tegra-hdmi 54280000.hdmi: failed to get PLL regulator
+mmcblk0rpmb: mmc0:0001 SEM16G partition 3 4.00 MiB, chardev (236:0)
 as3722-sd2: supplied by +5V_SYS
 as3722-sd3: supplied by +5V_SYS
 as3722-sd4: supplied by +5V_SYS
 as3722-sd5: supplied by +5V_SYS
 as3722-ldo0: supplied by +1.35V_LP0(sd2)
+ mmcblk0: p1
 as3722-ldo1: supplied by +3.3V_RUN
 as3722-ldo2: supplied by +1.8V_VDDIO
 as3722-ldo3: supplied by +3.3V_SYS
@@ -443,7 +443,7 @@
 pcieport 0000:00:02.0: Signaling PME with IRQ 396
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
-r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xb211beca, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 397
+r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0x1ef666a3, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 397
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
@@ -455,7 +455,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:04:14 UTC (946685054)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:26 UTC (946684946)
 +USB0_VBUS_SW: disabling
 +1.8V_RUN_CAM: disabling
 +2.8V_RUN_CAM: disabling
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig_log.txt
--- /tmp/fooa	2018-02-08 19:48:23.144897263 -0800
+++ /tmp/foob	2018-02-08 19:48:23.148897281 -0800
@@ -9,7 +9,7 @@
 Model: NVIDIA Jetson TK1
 Board: NVIDIA Jetson TK1
 DRAM:  2 GiB
-MMC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
+MC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
 In:    serial
 Out:   serial
 Err:   serial
@@ -33,20 +33,20 @@
 	 #################################################################
 	 #################################################################
 	 ##################################
-	 945.3 KiB/s
+	 4 MiB/s
 done
-Bytes transferred = 6221496 (5eeeb8 hex)
+Bytes transferred = 6221752 (5eefb8 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
 Filename 'tegra124-jetson-tk1.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 927.7 KiB/s
+	 3.5 MiB/s
 done
 Bytes transferred = 66544 (103f0 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5eeeb8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5eefb8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830133ef
@@ -54,7 +54,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-next-20180208-29428-g957f771 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Feb 7 19:16:46 PST 2018
+Linux version 4.15.0-next-20180209-29529-g611f09f (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Feb 8 19:02:48 PST 2018
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -438,7 +438,7 @@
 pcieport 0000:00:02.0: Signaling PME with IRQ 396
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
-r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0x3a0cf871, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 397
+r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0x635530e2, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 397
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
@@ -450,7 +450,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:18 UTC (946684818)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:13 UTC (946684813)
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
Diff for ./tegra124-jetson-tk1/tegra124-jetson-tk1/tegra_defconfig%tegra-fw_log.txt
--- /tmp/fooa	2018-02-08 19:48:23.168897370 -0800
+++ /tmp/foob	2018-02-08 19:48:23.168897370 -0800
@@ -33,9 +33,9 @@
 	 #################################################################
 	 #################################################################
 	 #############################################
-	 3.9 MiB/s
+	 3.5 MiB/s
 done
-Bytes transferred = 6384488 (616b68 hex)
+Bytes transferred = 6384368 (616af0 hex)
 Tegra124 (Jetson TK1) # tftpboot 0x83000000 tegra124-jetson-tk1.dtb
 Using eth_rtl8169 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.102
@@ -46,7 +46,7 @@
 done
 Bytes transferred = 66544 (103f0 hex)
 Tegra124 (Jetson TK1) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x616b68 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x616af0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 830133ef
@@ -54,7 +54,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-next-20180208-231136-g957f771 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Feb 7 19:17:06 PST 2018
+Linux version 4.15.0-next-20180209-231237-g611f09f (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Feb 8 19:03:11 PST 2018
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -446,7 +446,7 @@
 pcieport 0000:00:02.0: Signaling PME with IRQ 396
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:02:00.0: enabling device (0140 -> 0143)
-r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0xc30b036a, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 397
+r8169 0000:02:00.0 eth0: RTL8168g/8111g at 0x37c85f99, 00:04:4b:3a:d9:81, XID 0c000800 IRQ 397
 r8169 0000:02:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
@@ -458,7 +458,7 @@
 cpufreq: cpufreq_online: CPU0: Running at unlisted freq: 696000 KHz
 cpufreq: cpufreq_online: CPU0: Unlisted initial frequency changed to: 714000 KHz
 input: gpio-keys as /devices/soc0/gpio-keys/input/input2
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:02:02 UTC (946684922)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:14 UTC (946684814)
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
Diff for ./tegra124-nyan-big/tegra124-nyan-big/multi_v7_defconfig_log.txt
--- /tmp/fooa	2018-02-08 19:48:23.200897515 -0800
+++ /tmp/foob	2018-02-08 19:48:23.204897534 -0800
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -52,7 +52,7 @@
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.2 MiB/s
+	 3.4 MiB/s
 done
 Bytes transferred = 67482 (1079a hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
@@ -64,7 +64,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-next-20180208-349377-g957f77188c8b (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Feb 7 19:17:40 PST 2018
+Linux version 4.15.0-next-20180209-349478-g611f09f4907a (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Feb 8 19:04:12 PST 2018
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -308,7 +308,9 @@
 mmc0: Invalid maximum block size, assuming 512 bytes
 mmc0: SDHCI controller on 700b0600.sdhci [700b0600.sdhci] using ADMA
 ledtrig-cpu: registered to indicate activity on CPUs
+elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
 usbcore: registered new interface driver usbhid
+input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 usbhid: USB HID core driver
 NET: Registered protocol family 10
 Segment Routing with IPv6
@@ -324,27 +326,26 @@
 Registering SWP/SWPB emulation handler
 Loading compiled-in X.509 certificates
 +3.3V_RUN: supplied by +3.3V_SYS
-elan_i2c 1-0015: Elan Touchpad: Module ID: 0x002a, Firmware: 0x0005, Sample: 0x0007, IAP: 0x0009
-input: Elan Touchpad as /devices/soc0/7000c400.i2c/i2c-1/1-0015/input/input0
 +VDD_LED: supplied by +VDD_MUX
 +3.3V_AVDD_HDMI_AP_GATED: supplied by +3.3V_RUN
 +1.8V_RUN_CAM: supplied by +3.3V_RUN
 +VDDIO_SDMMC3: supplied by +3.3V_RUN
 +1.8V_RUN_VPP_FUSE: supplied by +3.3V_RUN
-mmc0: new high speed MMC card at address 0001
 +5V_VDD_TS_SW: supplied by +5V_SYS
-mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
 +5V_USB_HS: supplied by +5V_SYS
-mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
+mmc0: new high speed MMC card at address 0001
 +5V_USB_SS: supplied by +5V_SYS
-mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
-mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB, chardev (236:0)
+mmcblk0: mmc0:0001 MMC16G 14.4 GiB 
 +3.3V_PANEL: supplied by +3.3V_RUN
- mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
+mmcblk0boot0: mmc0:0001 MMC16G partition 1 2.00 MiB
 +3.3V_LP0: supplied by +3.3V_SYS
+mmcblk0boot1: mmc0:0001 MMC16G partition 2 2.00 MiB
 +1.05V_RUN_AVDD_HDMI_PLL: supplied by +1.05V_RUN
 elan_i2c 1-0015: failed to read report data: -121
+elan_i2c 1-0015: failed to read report data: -121
+mmcblk0rpmb: mmc0:0001 MMC16G partition 3 128 KiB, chardev (236:0)
 +5V_HDMI_CON: supplied by +5V_SYS
+ mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
  usb3-0: usb3-0 supply vbus not found, using dummy regulator
  usb3-1: usb3-1 supply vbus not found, using dummy regulator
 tegra-sor 54540000.sor: failed to probe output: -517
@@ -355,22 +356,22 @@
 mmc1: Switching to 3.3V signalling voltage failed
 mmc1: SDHCI controller on 700b0000.sdhci [700b0000.sdhci] using ADMA
 sdhci-tegra 700b0400.sdhci: Got CD GPIO
+elan_i2c 1-0015: invalid report id data (ff)
 sdhci-tegra 700b0400.sdhci: Got WP GPIO
 mmc2: Unknown controller version (3). You may experience problems.
-elan_i2c 1-0015: invalid report id data (ff)
 mmc2: Invalid maximum block size, assuming 512 bytes
-mmc1: new high speed SDIO card at address 0001
 mmc2: SDHCI controller on 700b0400.sdhci [700b0400.sdhci] using ADMA
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
 input: gpio-keys as /devices/soc0/gpio-keys/input/input1
 tegra-sor 54540000.sor: failed to probe output: -517
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:24 UTC (946684824)
 panel-simple panel: panel supply power not found, using dummy regulator
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:24 UTC (946684824)
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +1.05V_RUN: disabling
 +1.8V_RUN_CAM: disabling
 +3.3V_RUN_CAM: disabling
+mmc1: new high speed SDIO card at address 0001
 +1.2V_RUN_CAM_FRONT: disabling
 +1.05V_RUN_CAM_REAR: disabling
 +2.8V_RUN_TOUCH: disabling
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig_log.txt
--- /tmp/fooa	2018-02-08 19:48:23.216897587 -0800
+++ /tmp/foob	2018-02-08 19:48:23.220897606 -0800
@@ -5,7 +5,7 @@
 TEGRA124
 Model: Acer Chromebook 13 CB5-311
 DRAM:  4 GiB
-C:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
+:   Tegra SD/MMC: 0, Tegra SD/MMC: 1, Tegra SD/MMC: 2
 MMC init failed
 Using default environment
 
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -43,7 +43,7 @@
 	 ##################################
 	 3.4 MiB/s
 done
-Bytes transferred = 6221496 (5eeeb8 hex)
+Bytes transferred = 6221752 (5eefb8 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.104
@@ -54,7 +54,7 @@
 done
 Bytes transferred = 67482 (1079a hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x5eeeb8 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x5eefb8 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013799
@@ -62,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-next-20180208-29428-g957f771 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Feb 7 19:16:46 PST 2018
+Linux version 4.15.0-next-20180209-29529-g611f09f (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Feb 8 19:02:48 PST 2018
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -364,8 +364,8 @@
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
-cfg80211: failed to load regulatory.db
 +3.3V_AVDD_HDMI_AP_GATED: disabling
+cfg80211: failed to load regulatory.db
 +1.8V_RUN_CAM: disabling
 +1.2V_RUN_CAM_FRONT: disabling
 +1.05V_RUN_CAM_REAR: disabling
@@ -392,9 +392,9 @@
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
 tegra-sor 54540000.sor: failed to probe output: -517
-input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input4
 EXT4-fs (mmcblk0p3): couldn't mount as ext3 due to feature incompatibilities
 panel-simple panel: panel supply power not found, using dummy regulator
+input: tegra-hda HDMI/DP,pcm=3 as /devices/soc0/70030000.hda/sound/card0/input4
 EXT4-fs (mmcblk0p3): recovery complete
 EXT4-fs (mmcblk0p3): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:3.
Diff for ./tegra124-nyan-big/tegra124-nyan-big/tegra_defconfig%tegra-fw_log.txt
--- /tmp/fooa	2018-02-08 19:48:23.228897642 -0800
+++ /tmp/foob	2018-02-08 19:48:23.232897660 -0800
@@ -14,7 +14,7 @@
 Err:   serial
 Net:   Net Initialization Skipped
 No ethernet found.
-Hit any key to stop autoboot:  2  0 
+Hit any key to stop autoboot:  2  1  0 
 Tegra124 (Nyan-big) # 
 Tegra124 (Nyan-big) # usb start
 starting USB...
@@ -43,18 +43,18 @@
 	 #############################################
 	 3.4 MiB/s
 done
-Bytes transferred = 6384488 (616b68 hex)
+Bytes transferred = 6384368 (616af0 hex)
 Tegra124 (Nyan-big) # tftpboot 0x83000000 tegra124-nyan-big.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.104
 Filename 'tegra124-nyan-big.dtb'.
 Load address: 0x83000000
 Loading: *#####
-	 3.1 MiB/s
+	 3.6 MiB/s
 done
 Bytes transferred = 67482 (1079a hex)
 Tegra124 (Nyan-big) # bootz 0x82000000 - 0x83000000
-Kernel image @ 0x82000000 [ 0x000000 - 0x616b68 ]
+Kernel image @ 0x82000000 [ 0x000000 - 0x616af0 ]
 ## Flattened Device Tree blob at 83000000
    Booting using the fdt blob at 0x83000000
    Using Device Tree in place at 83000000, end 83013799
@@ -62,7 +62,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-next-20180208-231136-g957f771 (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Feb 7 19:17:06 PST 2018
+Linux version 4.15.0-next-20180209-231237-g611f09f (jonathanh@sellafield) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Feb 8 19:03:11 PST 2018
 CPU: ARMv7 Processor [413fc0f3] revision 3 (ARMv7), cr=10c5387d
 CPU: div instructions available: patching division code
 CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
@@ -368,13 +368,13 @@
 input: gpio-keys as /devices/soc0/gpio-keys/input/input3
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
-as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:23 UTC (946684823)
+as3722-rtc as3722-rtc: setting system clock to 2000-01-01 00:00:24 UTC (946684824)
 cfg80211: Loading compiled-in X.509 certificates for regulatory database
 cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
 platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
-cfg80211: failed to load regulatory.db
 +3.3V_AVDD_HDMI_AP_GATED: disabling
 +1.8V_RUN_CAM: disabling
+cfg80211: failed to load regulatory.db
 +1.2V_RUN_CAM_FRONT: disabling
 +1.05V_RUN_CAM_REAR: disabling
 +2.8V_RUN_TOUCH: disabling
@@ -402,18 +402,18 @@
  mmcblk2: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12
 tegra-sor 54540000.sor: failed to probe output: -517
 panel-simple panel: panel supply power not found, using dummy regulator
+uvcvideo: Found UVC 1.00 device HD WebCam (1bcf:2c67)
+input: HD WebCam: HD WebCam as /devices/soc0/70090000.usb/usb1/1-2/1-2:1.0/input/input5
+tegra-sor 54540000.sor: failed to probe output: -517
+panel-simple panel: panel supply power not found, using dummy regulator
 EXT4-fs (mmcblk0p3): recovery complete
 EXT4-fs (mmcblk0p3): mounted filesystem with ordered data mode. Opts: (null)
 VFS: Mounted root (ext4 filesystem) on device 179:3.
 devtmpfs: mounted
 Freeing unused kernel memory: 1024K
-uvcvideo: Found UVC 1.00 device HD WebCam (1bcf:2c67)
-input: HD WebCam: HD WebCam as /devices/soc0/70090000.usb/usb1/1-2/1-2:1.0/input/input5
-tegra-sor 54540000.sor: failed to probe output: -517
-panel-simple panel: panel supply power not found, using dummy regulator
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    2.898104] alloc_contig_range: [fbc79, fbc7a) PFNs busy
+root@(none):/# [    2.888451] alloc_contig_range: [fbc79, fbc7a) PFNs busy
 alloc_contig_range: [fbc79, fbc7a) PFNs busy
 alloc_contig_range: [fbc7b, fbc7c) PFNs busy
 alloc_contig_range: [fbc7c, fbc7d) PFNs busy
Diff for ./tegra132-norrin/tegra132-norrin/defconfig_log.txt
--- /tmp/fooa	2018-02-08 19:48:23.236897679 -0800
+++ /tmp/foob	2018-02-08 19:48:23.240897697 -0800
@@ -32,7 +32,7 @@
 MSENC clock is set to  79200 KHz
 Vde clock is set to 204000 KHz
 
-Bootloader-Cpu Init at (time stamp): 1801108 us
+Bootloader-Cpu Init at (time stamp): 1801091 us
 
 Pinmux changes applied in kernel way
 
@@ -60,25 +60,25 @@
 The proc BoardInfo: 0x0176:0x0001:0x01:0x43:0x00
 FastbootAddAtag: setting ATAG (0)
 
-Board Id Read Time: 4 us
+Board Id Read Time: 3 us
 Power Rail enable Time: 1 us
 
-Miscellaneous avp time: 1801081  us
-Bootloader (AosInit) time: 734234 us
-Bootloader Main Init time: 180082 us
-BlockDevMgrInit time: 157543 us
+Miscellaneous avp time: 1801065  us
+Bootloader (AosInit) time: 734252 us
+Bootloader Main Init time: 180071 us
+BlockDevMgrInit time: 157545 us
 PartMgrInit time: 8 us
-StorMgrInit time: 30 us
+StorMgrInit time: 31 us
 FsMgrInit time: 2 us
-Partition Table load: 76702 us 
-Display init time: 574690 us
+Partition Table load: 76688 us 
+Display init time: 574806 us
 Kernel read time: 1582 msec
 Load Kernel Size:   20969472 Bytes
 Bl Kernel ReadSpeed: 13255000 Bytes/sec
-Pre-Kernel Jump time: 4424730 us
-Clearing 32 Bit State and Warm resetting to 64bit mode: 9627907 us
+Pre-Kernel Jump time: 4424533 us
+Clearing 32 Bit State and Warm resetting to 64bit mode: 9627648 us
 Booting Linux on physical CPU 0x0000000000 [0x4e0f0000]
-Linux version 4.15.0-next-20180208-349377-g957f77188c8b (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Feb 7 19:23:46 PST 2018
+Linux version 4.15.0-next-20180209-349478-g611f09f4907a (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Thu Feb 8 19:10:18 PST 2018
 Machine model: NVIDIA Tegra132 Norrin
 Memory limited to 1734MB
 efi: Getting EFI parameters from FDT:
@@ -140,9 +140,9 @@
 DMI not present or invalid.
 NET: Registered protocol family 16
 audit: initializing netlink subsys (disabled)
-audit: type=2000 audit(0.136:1): state=initialized audit_enabled=0 res=1
+audit: type=2000 audit(0.144:1): state=initialized audit_enabled=0 res=1
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ 000000000704d01e, 1 data @ 00000000d0372ca0)
+vdso: 2 pages (1 code @ 00000000fd40a0e4, 1 data @ 000000007072ab7d)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -263,8 +263,8 @@
 NET: Registered protocol family 17
 mmcblk0: mmc0:0001 SEM16G 14.7 GiB 
 9pnet: Installing 9P2000 support
-mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 Key type dns_resolver registered
+mmcblk0boot0: mmc0:0001 SEM16G partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 SEM16G partition 2 4.00 MiB
 registered taskstats version 1
 Loading compiled-in X.509 certificates
Diff for ./tegra186-p2771-0000/tegra186-p2771-0000/defconfig%jetson-tx2_log.txt
--- /tmp/fooa	2018-02-08 19:48:23.264897804 -0800
+++ /tmp/foob	2018-02-08 19:48:23.268897823 -0800
@@ -2,14 +2,14 @@
 
 Pass: Linux Boot Test 1
 	Test Owner(s): N/A
-	Execution Time 56.73 sec
+	Execution Time 56.87 sec
 	Percent passed so far: 50.0
 	
 
 
 Pass: Linux Boot Test 2
 	Test Owner(s): N/A
-	Execution Time 56.18 sec
+	Execution Time 56.12 sec
 	Percent passed so far: 100.0
 	
 
@@ -32,7 +32,7 @@
 I> Version of mb1 bct binary 12
 W> MB1 BCT does not have controller prod config data
 I> Bootrom patch version : 31 (correctly patched)
-I> ATE fuse revision : 0x30880
+I> ATE fuse revision : 0x40080
 I> chip revision : A02P
 I> disabled_core_mask: 0xffffff0c
 I> Enabled Cores: 0xf3
@@ -97,8 +97,8 @@
 I> Loading preboot
 I> Binary(1) of size 59024 is loaded @ 0x40000000
 I> Restrict GSC#29 access to DPMU only
-I> Updated LSR_DVCOMP_PARAMS: 0xc07e0fc5c
-I> Updated LSR_DVCOMP_PARAMS: 0xc07c0f85c
+I> Updated LSR_DVCOMP_PARAMS: 0xc0760ec5c
+I> Updated LSR_DVCOMP_PARAMS: 0xc0780f05c
 I> WP0 triggered
 I> Waypoint-0 ACK received :)
 I> WP0.5 triggered
@@ -269,7 +269,7 @@
 I> Total write size is 512 bytes
 I> Write successful.
 I> Writing APP partition
-I> Total write size is 28517628 bytes
+I> Total write size is 28509436 bytes
 I> Write successful.
 I> Writing mts-bootpack partition
 I> Total write size is 2077344 bytes
@@ -515,24 +515,24 @@
 I> Kernel DTB @ 0x92000000
 I> Verified boot state = Orange
 E> tegrabl_display_clear: display is not initialized
-I> Checking boot.img header magic ... [0001.776] I> [OK]
+I> Checking boot.img header magic ... [0001.775] I> [OK]
 I> Valid boot.img @ 0xa8000000
 I> decompressor handler not found
-I> Copying kernel image (465664 bytes) from 0xa8000800 to 0x80080000 ... [0001.793] I> Done
-I> Move ramdisk (len: 6264763) from 0xa8072800 to 0x98000000
+I> Copying kernel image (465664 bytes) from 0xa8000800 to 0x80080000 ... [0001.792] I> Done
+I> Move ramdisk (len: 6264534) from 0xa8072800 to 0x98000000
 I> Updated bpmp info to DTB
-I> Ramdisk: Base: 0x98000000; Size: 0x5f97bb
+I> Ramdisk: Base: 0x98000000; Size: 0x5f96d6
 I> Updated initrd info to DTB
 I> disabled_core_mask: 0xffffff0c
 I> Kernel type = Normal
-I> add_serialno: Serial Num = 0320817135469
+I> add_serialno: Serial Num = 0320817139044
 W> No valid slot number is found in scratch register
 W> Return default slot: _a
-I> Linux Cmdline: root=/dev/nfs rw netdevwait ip=192.168.99.2:192.168.99.1:192.168.99.1:255.255.255.0::eth0:off nfsroot=192.168.99.1:/home/ausvrl2574/nfsroot fbcon=map:0 net.ifnames=0 console=tty0 OS=l4t console=ttyS0,115200n8 memtype=0 video=tegrafb no_console_suspend=1 earlycon=uart8250,mmio32,0x03100000 nvdumper_reserved=0xf7fc0000 gpt androidboot.secureos=none tegraid=18.1.2.0.0 tegra_keep_boot_clocks maxcpus=6 android.kerneltype=normal androidboot.serialno=0320817135469 androidboot.security=non-secure sdhci_tegra.en_boot_part_access=1 
+I> Linux Cmdline: root=/dev/nfs rw netdevwait ip=192.168.99.2:192.168.99.1:192.168.99.1:255.255.255.0::eth0:off nfsroot=192.168.99.1:/home/ausvrl2248/nfsroot fbcon=map:0 net.ifnames=0 console=tty0 OS=l4t console=ttyS0,115200n8 memtype=0 video=tegrafb no_console_suspend=1 earlycon=uart8250,mmio32,0x03100000 nvdumper_reserved=0xf7fc0000 gpt androidboot.secureos=none tegraid=18.1.2.0.0 tegra_keep_boot_clocks maxcpus=6 android.kerneltype=normal androidboot.serialno=0320817139044 androidboot.security=non-secure sdhci_tegra.en_boot_part_access=1 
 I> Updated bootarg info to DTB
-I> eeprom_get_mac_addr: MAC (type: 0): 00:04:4b:8c:59:49
-I> eeprom_get_mac_addr: MAC (type: 1): 00:04:4b:8c:59:4a
-I> eeprom_get_mac_addr: MAC (type: 2): 00:04:4b:8c:59:4b
+I> eeprom_get_mac_addr: MAC (type: 0): 00:04:4b:8c:60:c3
+I> eeprom_get_mac_addr: MAC (type: 1): 00:04:4b:8c:60:c4
+I> eeprom_get_mac_addr: MAC (type: 2): 00:04:4b:8c:60:c5
 E> "ids" doesn't exist, creating 
 E> "connection" doesn't exist, creating 
 E> "configs" doesn't exist, creating 
@@ -564,22 +564,7 @@
 E> "pmc-reset-reason" doesn't exist, creating 
 E> "pmic-reset-reason" doesn't exist, creating 
 I> disabled_core_mask: 0xffffff0c
-I> tegrabl_load_kernel_and_dtb: Done
-
-
-U-Boot 2017.03-g8537ddd769f4 (Oct 04 2017 - 08:46:37 -0700)
-
-TEGRA186
-Model: NVIDIA P2771-0000-500
-DRAM:  7.8 GiB
-MMC:   Tegra SD/MMC: 0, Tegra SD/MMC: 1
-*** Warning - bad CRC, using default environment
-
-In:    serial
-Out:   serial
-Err:   serial
-Net:   eth0: ethernet@2490000
-Hit any key to stop autoboot:  2 [0000.184] I> Welcome to MB2(TBoot-BPMP)(version: 01.00.160913-t186-M-00.00-mobile-f4c42291)
+I> Welcome to MB2(TBoot-BPMP)(version: 01.00.160913-t186-M-00.00-mobile-f4c42291)
 I> Default Heap @ [0xd486400 - 0xd488400]
 I> DMA Heap @ [0x84900000 - 0x85300000]
 I> bit @ 0xd480000
@@ -781,24 +766,24 @@
 I> Kernel DTB @ 0x92000000
 I> Verified boot state = Orange
 E> tegrabl_display_clear: display is not initialized
-I> Checking boot.img header magic ... [0001.779] I> [OK]
+I> Checking boot.img header magic ... [0001.778] I> [OK]
 I> Valid boot.img @ 0xa8000000
 I> decompressor handler not found
-I> Copying kernel image (465664 bytes) from 0xa8000800 to 0x80080000 ... [0001.796] I> Done
-I> Move ramdisk (len: 6264763) from 0xa8072800 to 0x98000000
+I> Copying kernel image (465664 bytes) from 0xa8000800 to 0x80080000 ... [0001.795] I> Done
+I> Move ramdisk (len: 6264534) from 0xa8072800 to 0x98000000
 I> Updated bpmp info to DTB
-I> Ramdisk: Base: 0x98000000; Size: 0x5f97bb
+I> Ramdisk: Base: 0x98000000; Size: 0x5f96d6
 I> Updated initrd info to DTB
 I> disabled_core_mask: 0xffffff0c
 I> Kernel type = Normal
-I> add_serialno: Serial Num = 0320817135469
+I> add_serialno: Serial Num = 0320817139044
 W> No valid slot number is found in scratch register
 W> Return default slot: _a
-I> Linux Cmdline: root=/dev/nfs rw netdevwait ip=192.168.99.2:192.168.99.1:192.168.99.1:255.255.255.0::eth0:off nfsroot=192.168.99.1:/home/ausvrl2574/nfsroot fbcon=map:0 net.ifnames=0 console=tty0 OS=l4t console=ttyS0,115200n8 memtype=0 video=tegrafb no_console_suspend=1 earlycon=uart8250,mmio32,0x03100000 nvdumper_reserved=0xf7fc0000 gpt androidboot.secureos=none tegraid=18.1.2.0.0 tegra_keep_boot_clocks maxcpus=6 android.kerneltype=normal androidboot.serialno=0320817135469 androidboot.security=non-secure sdhci_tegra.en_boot_part_access=1 
+I> Linux Cmdline: root=/dev/nfs rw netdevwait ip=192.168.99.2:192.168.99.1:192.168.99.1:255.255.255.0::eth0:off nfsroot=192.168.99.1:/home/ausvrl2248/nfsroot fbcon=map:0 net.ifnames=0 console=tty0 OS=l4t console=ttyS0,115200n8 memtype=0 video=tegrafb no_console_suspend=1 earlycon=uart8250,mmio32,0x03100000 nvdumper_reserved=0xf7fc0000 gpt androidboot.secureos=none tegraid=18.1.2.0.0 tegra_keep_boot_clocks maxcpus=6 android.kerneltype=normal androidboot.serialno=0320817139044 androidboot.security=non-secure sdhci_tegra.en_boot_part_access=1 
 I> Updated bootarg info to DTB
-I> eeprom_get_mac_addr: MAC (type: 0): 00:04:4b:8c:59:49
-I> eeprom_get_mac_addr: MAC (type: 1): 00:04:4b:8c:59:4a
-I> eeprom_get_mac_addr: MAC (type: 2): 00:04:4b:8c:59:4b
+I> eeprom_get_mac_addr: MAC (type: 0): 00:04:4b:8c:60:c3
+I> eeprom_get_mac_addr: MAC (type: 1): 00:04:4b:8c:60:c4
+I> eeprom_get_mac_addr: MAC (type: 2): 00:04:4b:8c:60:c5
 E> "ids" doesn't exist, creating 
 E> "connection" doesn't exist, creating 
 E> "configs" doesn't exist, creating 
@@ -853,15 +838,15 @@
 Scanning mmc 0:1...
 Found /boot/extlinux/extlinux.conf
 Retrieving file: /boot/extlinux/extlinux.conf
-431 bytes read in 69 ms (5.9 KiB/s)
+431 bytes read in 74 ms (4.9 KiB/s)
 p2771-0000 NFS boot options
 1:	primary kernel
 Enter choice: 1:	primary kernel
 Retrieving file: /boot/initrd
-6264763 bytes read in 210 ms (28.5 MiB/s)
+6264534 bytes read in 205 ms (29.1 MiB/s)
 Retrieving file: /boot/Image
 17601024 bytes read in 467 ms (35.9 MiB/s)
-append: console=ttyS0,115200n8 earlycon ignore_loglevel root=/dev/nfs rw netdevwait ip=192.168.99.2:192.168.99.1:192.168.99.1:255.255.255.0::eth0:off nfsroot=192.168.99.1:/home/ausvrl2574/nfsroot,nolock,nfsvers=2 rootwait
+append: console=ttyS0,115200n8 earlycon ignore_loglevel root=/dev/nfs rw netdevwait ip=192.168.99.2:192.168.99.1:192.168.99.1:255.255.255.0::eth0:off nfsroot=192.168.99.1:/home/ausvrl2248/nfsroot,nolock,nfsvers=2 rootwait
 Retrieving file: /boot/tegra186-p2771-0000.dtb
 23528 bytes read in 52 ms (441.4 KiB/s)
 ## Flattened Device Tree blob at 82000000
@@ -985,9 +970,9 @@
 VDD_HDMI_5V0: supplied by VDD_5V0_SYS
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
 mmc2: new high speed MMC card at address 0001
-max77686-rtc max77620-rtc: setting system clock to 2016-10-30 13:36:12 UTC (1477834572)
+max77686-rtc max77620-rtc: setting system clock to 2016-10-24 04:33:24 UTC (1477283604)
 mmcblk2: mmc2:0001 032G34 29.1 GiB 
-dwc-eth-dwmac 2490000.ethernet eth0: device MAC address 56:44:24:f6:90:a8
+dwc-eth-dwmac 2490000.ethernet eth0: device MAC address 3a:8e:29:f0:19:10
 mmcblk2boot0: mmc2:0001 032G34 partition 1 4.00 MiB
 Generic PHY stmmac-0:00: attached PHY driver [Generic PHY] (mii_bus:phy_addr=stmmac-0:00, irq=POLL)
 mmcblk2boot1: mmc2:0001 032G34 partition 2 4.00 MiB
@@ -996,34 +981,35 @@
 dwc-eth-dwmac 2490000.ethernet eth0: IEEE 1588-2008 Advanced Timestamp supported
  mmcblk2: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12 p13 p14 p15 p16 p17
 dwc-eth-dwmac 2490000.ethernet eth0: registered PTP clock
-WARNING: CPU: 4 PID: 40 at drivers/firmware/tegra/bpmp.c:360 tegra_bpmp_transfer+0x2dc/0x368
+WARNING: CPU: 3 PID: 5 at drivers/firmware/tegra/bpmp.c:360 tegra_bpmp_transfer+0x2dc/0x368
 Modules linked in:
-CPU: 4 PID: 40 Comm: kworker/u12:1 Tainted: G S               4.15.0-next-20180208-231136-g957f771 #1
+CPU: 3 PID: 5 Comm: kworker/u12:0 Tainted: G S               4.15.0-next-20180209-231237-g611f09f #1
 Hardware name: NVIDIA Tegra186 P2771-0000 Development Board (DT)
 IP-Config: Complete:
 Workqueue: events_power_efficient phy_state_machine
-     device=eth0, hwaddr=56:44:24:f6:90:a8, ipaddr=192.168.99.2, mask=255.255.255.0, gw=192.168.99.1
+     device=eth0, hwaddr=3a:8e:29:f0:19:10, ipaddr=192.168.99.2, mask=255.255.255.0, gw=192.168.99.1
 pstate: 80000085 (Nzcv daIf -PAN -UAO)
 pc : tegra_bpmp_transfer+0x2dc/0x368
 lr : tegra_bpmp_clk_transfer+0x94/0xd0
      host=192.168.99.2, domain=, nis-domain=(none)
-sp : ffff000009f73ab0
-x29: ffff000009f73ab0 x28: ffff8001f7808070 
+sp : ffff000009ddbab0
+x29: ffff000009ddbab0 x28: ffff8001f7808070 
+x27: ffff8001f7808020 
      bootserver=192.168.99.1, rootserver=192.168.99.1, rootpath=
-x27: ffff8001f7808020 x26: ffff00000902fda8 
+x26: ffff00000902fda8 
 x25: 0000000000000000 x24: 0000000000000001 
-x23: ffff8001f4e5c878 x22: 0000000000000080 
-x21: ffff000009f73b58 x20: ffff000009019000 
-x19: ffff000009f73bd8 x18: 0000000000000000 
-x17: 00000000621d5db4 x16: 000000003902a2ab 
-x15: 00000000711f23c3 x14: 00000000000c8000 
+x23: ffff8001f5c88878 x22: 0000000000000080 
+x21: ffff000009ddbb58 x20: ffff000009019000 
+x19: ffff000009ddbbd8 x18: 0000000000000007 
+x17: 000000000000000e x16: 0000000000000001 
+x15: 0000000000000019 x14: 00000000000c8000 
 x13: 00000000000079de x12: 00000000000079de 
-x11: ffffffffffffffff x10: 0000000000000003 
-x9 : 00000000ffffffff x8 : ffff8001fff976e0 
-x7 : ffff00000be50000 x6 : ffff000009f73b68 
+x11: ffffffffffffffff x10: 0000000000000005 
+x9 : 00000000ffffffff x8 : ffff8001fffc76e0 
+x7 : ffff00000c9f0000 x6 : ffff000009ddbb68 
 x5 : ffffffffffffffd0 x4 : 0000000000000010 
-x3 : ffff000009f73bc0 x2 : 0000000000000080 
-x1 : ffff000009f73b68 x0 : ffff8001f65efa18 
+x3 : ffff000009ddbbc0 x2 : 0000000000000080 
+x1 : ffff000009ddbb68 x0 : ffff8001f65b5c18 
 Call trace:
  tegra_bpmp_transfer+0x2dc/0x368
  tegra_bpmp_clk_transfer+0x94/0xd0
@@ -1040,31 +1026,31 @@
  worker_thread+0x13c/0x3f8
  kthread+0xfc/0x128
  ret_from_fork+0x10/0x18
----[ end trace 6e9860e6d42d1b1c ]---
-WARNING: CPU: 4 PID: 40 at drivers/firmware/tegra/bpmp.c:360 tegra_bpmp_transfer+0x2dc/0x368
+---[ end trace 0e00c8bd045097b0 ]---
+WARNING: CPU: 3 PID: 5 at drivers/firmware/tegra/bpmp.c:360 tegra_bpmp_transfer+0x2dc/0x368
 Modules linked in:
-CPU: 4 PID: 40 Comm: kworker/u12:1 Tainted: G S      W        4.15.0-next-20180208-231136-g957f771 #1
+CPU: 3 PID: 5 Comm: kworker/u12:0 Tainted: G S      W        4.15.0-next-20180209-231237-g611f09f #1
 Hardware name: NVIDIA Tegra186 P2771-0000 Development Board (DT)
 Workqueue: events_power_efficient phy_state_machine
 pstate: 80000085 (Nzcv daIf -PAN -UAO)
 pc : tegra_bpmp_transfer+0x2dc/0x368
 lr : tegra_bpmp_clk_transfer+0x94/0xd0
-sp : ffff000009f73a40
-x29: ffff000009f73a40 x28: ffff8001f7808070 
+sp : ffff000009ddba40
+x29: ffff000009ddba40 x28: ffff8001f7808070 
 x27: ffff8001f7808020 x26: ffff00000902fda8 
 x25: 0000000000000000 x24: 0000000000000001 
 x23: 00000000ffffffff x22: 0000000000000080 
-x21: ffff000009f73ae8 x20: ffff000009019000 
-x19: ffff000009f73b68 x18: 0000000000000000 
-x17: 00000000621d5db4 x16: 000000003902a2ab 
-x15: 00000000711f23c3 x14: 00000000000c8000 
+x21: ffff000009ddbae8 x20: ffff000009019000 
+x19: ffff000009ddbb68 x18: 0000000000000007 
+x17: 000000000000000e x16: 0000000000000001 
+x15: 0000000000000019 x14: 00000000000c8000 
 x13: 00000000000079de x12: 00000000000079de 
-x11: ffffffffffffffff x10: 0000000000000003 
-x9 : 00000000ffffffff x8 : ffff8001fff976e0 
-x7 : ffff00000be50000 x6 : ffff000009f73af8 
+x11: ffffffffffffffff x10: 0000000000000005 
+x9 : 00000000ffffffff x8 : ffff8001fffc76e0 
+x7 : ffff00000c9f0000 x6 : ffff000009ddbaf8 
 x5 : ffffffffffffffd0 x4 : 0000000000000010 
-x3 : ffff000009f73b50 x2 : 0000000000000080 
-x1 : ffff000009f73af8 x0 : ffff8001f65efa18 
+x3 : ffff000009ddbb50 x2 : 0000000000000080 
+x1 : ffff000009ddbaf8 x0 : ffff8001f65b5c18 
 Call trace:
  tegra_bpmp_transfer+0x2dc/0x368
  tegra_bpmp_clk_transfer+0x94/0xd0
@@ -1081,7 +1067,7 @@
  worker_thread+0x13c/0x3f8
  kthread+0xfc/0x128
  ret_from_fork+0x10/0x18
----[ end trace 6e9860e6d42d1b1d ]---
+---[ end trace 0e00c8bd045097b1 ]---
 dwc-eth-dwmac 2490000.ethernet: failed to set TX rate: -22
 dwc-eth-dwmac 2490000.ethernet eth0: Link is Up - 1Gbps/Full - flow control rx/tx
 VDD_RTC: disabling
@@ -1092,7 +1078,7 @@
   No soundcards found.
 Freeing unused kernel memory: 1152K
 /init: line 58: chmod: command not found
-L4T-INITRD Build DATE: Sun Oct 30 13:36:16 UTC 2016
+L4T-INITRD Build DATE: Mon Oct 24 04:33:28 UTC 2016
 Root device found: nfs
 Ethernet interfaces: eth0
 IP Address: 192.168.99.2
@@ -1104,106 +1090,106 @@
 Welcome to [1mUbuntu 16.04 LTS[0m!
 
 systemd[1]: Set hostname to <tegra-ubuntu>.
-systemd[1]: Reached target Swap.
-Reached target Swap.
+systemd[1]: Started Forward Password Requests to Wall Directory Watch.
+Started Forward Password Requests to Wall Directory Watch.
+systemd[1]: Listening on Journal Audit Socket.
+Listening on Journal Audit Socket.
 systemd[1]: Listening on udev Kernel Socket.
 Listening on udev Kernel Socket.
-systemd[1]: Listening on Syslog Socket.
-Listening on Syslog Socket.
-systemd[1]: Created slice User and Session Slice.
+systemd[1]: Reached target Swap.
+Reached target Swap.
+systemd[1]: Listening on udev Control Socket.
+Listening on udev Control Socket.
+systemd[1]: Reached target Encrypted Volumes.
+Reached target Encrypted Volumes.
+systemd[1]: Listening on Journal Socket (/dev/log).
+Listening on Journal Socket (/dev/log).
+Listening on /dev/initctl Compatibility Named Pipe.
 Created slice User and Session Slice.
-systemd[1]: Created slice System Slice.
+Started Dispatch Password Requests to Console Directory Watch.
+Listening on Syslog Socket.
+Reached target Paths.
 Created slice System Slice.
-systemd[1]: Created slice system-serial\x2dgetty.slice.
+Created slice system-getty.slice.
+Reached target Slices.
 Created slice system-serial\x2dgetty.slice.
-systemd[1]: Listening on /dev/initctl Compatibility Named Pipe.
-Listening on /dev/initctl Compatibility Named Pipe.
 Listening on Journal Socket.
-         Mounting POSIX Message Queue File System...
-         Mounting RPC Pipe File System...
          Starting Remount Root and Kernel File Systems...
-         Mounting Debug File System...
-Started Forward Password Requests to Wall Directory Watch.
-Reached target Encrypted Volumes.
-         Starting Load Kernel Modules...
+         Mounting RPC Pipe File System...
          Mounting Huge Pages File System...
+         Mounting POSIX Message Queue File System...
+         Mounting Debug File System...
          Starting Create Static Device Nodes in /dev...
-Created slice system-getty.slice.
-Listening on udev Control Socket.
-Started Dispatch Password Requests to Console Directory Watch.
-Reached target Paths.
-Reached target Slices.
-Listening on Journal Socket (/dev/log).
-Listening on Journal Audit Socket.
          Starting Journal Service...
+         Starting Load Kernel Modules...
+Mounted Debug File System.
 Mounted POSIX Message Queue File System.
 Mounted Huge Pages File System.
-Mounted Debug File System.
 Mounted RPC Pipe File System.
 Started Remount Root and Kernel File Systems.
+         Starting udev Coldplug all Devices...
+         Starting Load/Save Random Seed...
+Started Journal Service.
 Started Load Kernel Modules.
          Starting Apply Kernel Variables...
          Mounting Configuration File System...
-         Starting Load/Save Random Seed...
-         Starting udev Coldplug all Devices...
+         Starting Flush Journal to Persistent Storage...
 Mounted Configuration File System.
-Started Journal Service.
+systemd-journald[1530]: Received request to flush runtime journal from PID 1
+Started Flush Journal to Persistent Storage.
+Started Load/Save Random Seed.
 Started Create Static Device Nodes in /dev.
+         Starting udev Kernel Device Manager...
 Reached target Local File Systems (Pre).
 Reached target Local File Systems.
          Starting Set console keymap...
          Starting Preprocess NFS configuration...
-         Starting udev Kernel Device Manager...
-         Starting Flush Journal to Persistent Storage...
+         Starting Create Volatile Files and Directories...
 Started Apply Kernel Variables.
-Started Load/Save Random Seed.
 Failed to start Set console keymap.
 See 'systemctl status console-setup.service' for details.
-systemd-journald[1535]: Received request to flush runtime journal from PID 1
          Starting Raise network interfaces...
+Started udev Coldplug all Devices.
 Started Preprocess NFS configuration.
 Reached target NFS client services.
 Reached target Remote File Systems (Pre).
 Reached target Remote File Systems.
-Started Flush Journal to Persistent Storage.
-Started udev Coldplug all Devices.
-         Starting Create Volatile Files and Directories...
-Started udev Kernel Device Manager.
 Started Create Volatile Files and Directories.
+         Starting Update UTMP about System Boot/Shutdown...
+Reached target System Time Synchronized.
+Started udev Kernel Device Manager.
 Found device /dev/ttyS0.
+Started Update UTMP about System Boot/Shutdown.
 Started Raise network interfaces.
 Reached target Network.
 Reached target Network is Online.
-Reached target System Time Synchronized.
-         Starting Update UTMP about System Boot/Shutdown...
-Started Update UTMP about System Boot/Shutdown.
 Reached target System Initialization.
-Listening on D-Bus System Message Bus Socket.
+Started Daily Cleanup of Temporary Directories.
 Listening on RPCbind Server Activation Socket.
+Listening on D-Bus System Message Bus Socket.
 Reached target Sockets.
-Started Daily Cleanup of Temporary Directories.
-Started Daily apt activities.
-Reached target Timers.
 Reached target Basic System.
-         Starting System Logging Service...
-Started Regular background program processing daemon.
-         Starting /etc/rc.local Compatibility...
 Started D-Bus System Message Bus.
          Starting LSB: Start NTP daemon...
+         Starting /etc/rc.local Compatibility...
          Starting Permit User Sessions...
-         Starting Login Service...
-         Starting LSB: Set the CPU Frequency Scaling governor to "ondemand"...
          Starting OpenBSD Secure Shell server...
-Started System Logging Service.
+         Starting LSB: Set the CPU Frequency Scaling governor to "ondemand"...
+         Starting Login Service...
+Started Daily apt activities.
+Reached target Timers.
+         Starting System Logging Service...
+Started Regular background program processing daemon.
 Started /etc/rc.local Compatibility.
 Started Permit User Sessions.
-Started Serial Getty on ttyS0.
 Started Getty on tty1.
+Started Serial Getty on ttyS0.
 Reached target Login Prompts.
 Started Login Service.
 Started LSB: Set the CPU Frequency Scaling governor to "ondemand".
-Started OpenBSD Secure Shell server.
+Started System Logging Service.
 Started LSB: Start NTP daemon.
+Started OpenBSD Secure Shell server.
 Reached target Multi-User System.
 Reached target Graphical Interface.
          Starting Update UTMP about System Runlevel Changes...
@@ -1211,7 +1197,7 @@
 
 Ubuntu 16.04 LTS tegra-ubuntu ttyS0
 
-tegra-ubuntu login: [0000.184] I> Welcome to MB2(TBoot-BPMP)(version: 01.00.160913-t186-M-00.00-mobile-f4c42291)
+tegra-ubuntu login: [0000.194] I> Welcome to MB2(TBoot-BPMP)(version: 01.00.160913-t186-M-00.00-mobile-f4c42291)
 I> Default Heap @ [0xd486400 - 0xd488400]
 I> DMA Heap @ [0x84900000 - 0x85300000]
 I> bit @ 0xd480000
@@ -1413,24 +1399,24 @@
 I> Kernel DTB @ 0x92000000
 I> Verified boot state = Orange
 E> tegrabl_display_clear: display is not initialized
-I> Checking boot.img header magic ... [0001.779] I> [OK]
+I> Checking boot.img header magic ... [0001.789] I> [OK]
 I> Valid boot.img @ 0xa8000000
 I> decompressor handler not found
-I> Copying kernel image (465664 bytes) from 0xa8000800 to 0x80080000 ... [0001.796] I> Done
-I> Move ramdisk (len: 6264763) from 0xa8072800 to 0x98000000
+I> Copying kernel image (465664 bytes) from 0xa8000800 to 0x80080000 ... [0001.806] I> Done
+I> Move ramdisk (len: 6264534) from 0xa8072800 to 0x98000000
 I> Updated bpmp info to DTB
-I> Ramdisk: Base: 0x98000000; Size: 0x5f97bb
+I> Ramdisk: Base: 0x98000000; Size: 0x5f96d6
 I> Updated initrd info to DTB
 I> disabled_core_mask: 0xffffff0c
 I> Kernel type = Normal
-I> add_serialno: Serial Num = 0320817135469
+I> add_serialno: Serial Num = 0320817139044
 W> No valid slot number is found in scratch register
 W> Return default slot: _a
-I> Linux Cmdline: root=/dev/nfs rw netdevwait ip=192.168.99.2:192.168.99.1:192.168.99.1:255.255.255.0::eth0:off nfsroot=192.168.99.1:/home/ausvrl2574/nfsroot fbcon=map:0 net.ifnames=0 console=tty0 OS=l4t console=ttyS0,115200n8 memtype=0 video=tegrafb no_console_suspend=1 earlycon=uart8250,mmio32,0x03100000 nvdumper_reserved=0xf7fc0000 gpt androidboot.secureos=none tegraid=18.1.2.0.0 tegra_keep_boot_clocks maxcpus=6 android.kerneltype=normal androidboot.serialno=0320817135469 androidboot.security=non-secure sdhci_tegra.en_boot_part_access=1 
+I> Linux Cmdline: root=/dev/nfs rw netdevwait ip=192.168.99.2:192.168.99.1:192.168.99.1:255.255.255.0::eth0:off nfsroot=192.168.99.1:/home/ausvrl2248/nfsroot fbcon=map:0 net.ifnames=0 console=tty0 OS=l4t console=ttyS0,115200n8 memtype=0 video=tegrafb no_console_suspend=1 earlycon=uart8250,mmio32,0x03100000 nvdumper_reserved=0xf7fc0000 gpt androidboot.secureos=none tegraid=18.1.2.0.0 tegra_keep_boot_clocks maxcpus=6 android.kerneltype=normal androidboot.serialno=0320817139044 androidboot.security=non-secure sdhci_tegra.en_boot_part_access=1 
 I> Updated bootarg info to DTB
-I> eeprom_get_mac_addr: MAC (type: 0): 00:04:4b:8c:59:49
-I> eeprom_get_mac_addr: MAC (type: 1): 00:04:4b:8c:59:4a
-I> eeprom_get_mac_addr: MAC (type: 2): 00:04:4b:8c:59:4b
+I> eeprom_get_mac_addr: MAC (type: 0): 00:04:4b:8c:60:c3
+I> eeprom_get_mac_addr: MAC (type: 1): 00:04:4b:8c:60:c4
+I> eeprom_get_mac_addr: MAC (type: 2): 00:04:4b:8c:60:c5
 E> "ids" doesn't exist, creating 
 E> "connection" doesn't exist, creating 
 E> "configs" doesn't exist, creating 
@@ -1479,21 +1465,21 @@
 Net:   eth0: ethernet@2490000
 Hit any key to stop autoboot:  2  1  0 
 MMC: no card present
-mmc_init: -123, time 0
+mmc_init: -123, time 1
 switch to partitions #0, OK
 mmc0(part 0) is current device
 Scanning mmc 0:1...
 Found /boot/extlinux/extlinux.conf
 Retrieving file: /boot/extlinux/extlinux.conf
-431 bytes read in 69 ms (5.9 KiB/s)
+431 bytes read in 74 ms (4.9 KiB/s)
 p2771-0000 NFS boot options
 1:	primary kernel
 Enter choice: 1:	primary kernel
 Retrieving file: /boot/initrd
-6264763 bytes read in 209 ms (28.6 MiB/s)
+6264534 bytes read in 205 ms (29.1 MiB/s)
 Retrieving file: /boot/Image
 17601024 bytes read in 467 ms (35.9 MiB/s)
-append: console=ttyS0,115200n8 earlycon ignore_loglevel root=/dev/nfs rw netdevwait ip=192.168.99.2:192.168.99.1:192.168.99.1:255.255.255.0::eth0:off nfsroot=192.168.99.1:/home/ausvrl2574/nfsroot,nolock,nfsvers=2 rootwait
+append: console=ttyS0,115200n8 earlycon ignore_loglevel root=/dev/nfs rw netdevwait ip=192.168.99.2:192.168.99.1:192.168.99.1:255.255.255.0::eth0:off nfsroot=192.168.99.1:/home/ausvrl2248/nfsroot,nolock,nfsvers=2 rootwait
 Retrieving file: /boot/tegra186-p2771-0000.dtb
 23528 bytes read in 52 ms (441.4 KiB/s)
 ## Flattened Device Tree blob at 82000000
@@ -1617,9 +1603,9 @@
 VDD_HDMI_5V0: supplied by VDD_5V0_SYS
 input: gpio-keys as /devices/platform/gpio-keys/input/input0
 mmc2: new high speed MMC card at address 0001
-max77686-rtc max77620-rtc: setting system clock to 2016-10-30 13:37:14 UTC (1477834634)
+max77686-rtc max77620-rtc: setting system clock to 2016-10-24 04:34:26 UTC (1477283666)
 mmcblk2: mmc2:0001 032G34 29.1 GiB 
-dwc-eth-dwmac 2490000.ethernet eth0: device MAC address 86:c0:68:02:d1:7d
+dwc-eth-dwmac 2490000.ethernet eth0: device MAC address be:d4:67:dd:85:d2
 mmcblk2boot0: mmc2:0001 032G34 partition 1 4.00 MiB
 Generic PHY stmmac-0:00: attached PHY driver [Generic PHY] (mii_bus:phy_addr=stmmac-0:00, irq=POLL)
 mmcblk2boot1: mmc2:0001 032G34 partition 2 4.00 MiB
@@ -1628,35 +1614,35 @@
 dwc-eth-dwmac 2490000.ethernet eth0: IEEE 1588-2008 Advanced Timestamp supported
  mmcblk2: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12 p13 p14 p15 p16 p17
 dwc-eth-dwmac 2490000.ethernet eth0: registered PTP clock
-WARNING: CPU: 3 PID: 185 at drivers/firmware/tegra/bpmp.c:360 tegra_bpmp_transfer+0x2dc/0x368
+WARNING: CPU: 3 PID: 5 at drivers/firmware/tegra/bpmp.c:360 tegra_bpmp_transfer+0x2dc/0x368
 Modules linked in:
-CPU: 3 PID: 185 Comm: kworker/u12:3 Tainted: G S               4.15.0-next-20180208-231136-g957f771 #1
+CPU: 3 PID: 5 Comm: kworker/u12:0 Tainted: G S               4.15.0-next-20180209-231237-g611f09f #1
 Hardware name: NVIDIA Tegra186 P2771-0000 Development Board (DT)
-IP-Config: Complete:
 Workqueue: events_power_efficient phy_state_machine
+IP-Config: Complete:
 pstate: 80000085 (Nzcv daIf -PAN -UAO)
-     device=eth0, hwaddr=86:c0:68:02:d1:7d, ipaddr=192.168.99.2, mask=255.255.255.0, gw=192.168.99.1
 pc : tegra_bpmp_transfer+0x2dc/0x368
 lr : tegra_bpmp_clk_transfer+0x94/0xd0
-sp : ffff00000a423ab0
+     device=eth0, hwaddr=be:d4:67:dd:85:d2, ipaddr=192.168.99.2, mask=255.255.255.0, gw=192.168.99.1
+sp : ffff000009ddbab0
+x29: ffff000009ddbab0 x28: ffff8001f7808070 
+x27: ffff8001f7808020 x26: ffff00000902fda8 
      host=192.168.99.2, domain=, nis-domain=(none)
-x29: ffff00000a423ab0 x28: 0000000000000000 
-x27: ffff000009ddbce8 x26: ffff00000902fda8 
-     bootserver=192.168.99.1, rootserver=192.168.99.1, rootpath=
 x25: 0000000000000000 x24: 0000000000000001 
-x23: ffff8001f4e38878 
+x23: ffff8001f4ea8878 
+     bootserver=192.168.99.1, rootserver=192.168.99.1, rootpath=
 x22: 0000000000000080 
-x21: ffff00000a423b58 x20: ffff000009019000 
-x19: ffff00000a423bd8 x18: 0000000000000007 
-x17: 000000000000000e x16: 0000000000000001 
-x15: 0000000000000019 x14: 00000000000c8000 
+x21: ffff000009ddbb58 x20: ffff000009019000 
+x19: ffff000009ddbbd8 x18: 0000000000000000 
+x17: 00000000f42afc13 x16: 00000000bc4b37e4 
+x15: 000000001af58a14 x14: 00000000000c8000 
 x13: 00000000000079de x12: 00000000000079de 
 x11: ffffffffffffffff x10: 0000000000000005 
 x9 : 00000000ffffffff x8 : ffff8001fffc76e0 
-x7 : ffff00000c8d0000 x6 : ffff00000a423b68 
+x7 : ffff00000c8b0000 x6 : ffff000009ddbb68 
 x5 : ffffffffffffffd0 x4 : 0000000000000010 
-x3 : ffff00000a423bc0 x2 : 0000000000000080 
-x1 : ffff00000a423b68 x0 : ffff8001f6533c18 
+x3 : ffff000009ddbbc0 x2 : 0000000000000080 
+x1 : ffff000009ddbb68 x0 : ffff8001f6619c18 
 Call trace:
  tegra_bpmp_transfer+0x2dc/0x368
  tegra_bpmp_clk_transfer+0x94/0xd0
@@ -1673,31 +1659,31 @@
  worker_thread+0x13c/0x3f8
  kthread+0xfc/0x128
  ret_from_fork+0x10/0x18
----[ end trace 9ada84ef45d3bac3 ]---
-WARNING: CPU: 3 PID: 185 at drivers/firmware/tegra/bpmp.c:360 tegra_bpmp_transfer+0x2dc/0x368
+---[ end trace a9f6323c41f45376 ]---
+WARNING: CPU: 3 PID: 5 at drivers/firmware/tegra/bpmp.c:360 tegra_bpmp_transfer+0x2dc/0x368
 Modules linked in:
-CPU: 3 PID: 185 Comm: kworker/u12:3 Tainted: G S      W        4.15.0-next-20180208-231136-g957f771 #1
+CPU: 3 PID: 5 Comm: kworker/u12:0 Tainted: G S      W        4.15.0-next-20180209-231237-g611f09f #1
 Hardware name: NVIDIA Tegra186 P2771-0000 Development Board (DT)
 Workqueue: events_power_efficient phy_state_machine
 pstate: 80000085 (Nzcv daIf -PAN -UAO)
 pc : tegra_bpmp_transfer+0x2dc/0x368
 lr : tegra_bpmp_clk_transfer+0x94/0xd0
-sp : ffff00000a423a40
-x29: ffff00000a423a40 x28: 0000000000000000 
-x27: ffff000009ddbce8 x26: ffff00000902fda8 
+sp : ffff000009ddba40
+x29: ffff000009ddba40 x28: ffff8001f7808070 
+x27: ffff8001f7808020 x26: ffff00000902fda8 
 x25: 0000000000000000 x24: 0000000000000001 
 x23: 00000000ffffffff x22: 0000000000000080 
-x21: ffff00000a423ae8 x20: ffff000009019000 
-x19: ffff00000a423b68 x18: 0000000000000007 
-x17: 000000000000000e x16: 0000000000000001 
-x15: 0000000000000019 x14: 00000000000c8000 
+x21: ffff000009ddbae8 x20: ffff000009019000 
+x19: ffff000009ddbb68 x18: 0000000000000000 
+x17: 00000000f42afc13 x16: 00000000bc4b37e4 
+x15: 000000001af58a14 x14: 00000000000c8000 
 x13: 00000000000079de x12: 00000000000079de 
 x11: ffffffffffffffff x10: 0000000000000005 
 x9 : 00000000ffffffff x8 : ffff8001fffc76e0 
-x7 : ffff00000c8d0000 x6 : ffff00000a423af8 
+x7 : ffff00000c8b0000 x6 : ffff000009ddbaf8 
 x5 : ffffffffffffffd0 x4 : 0000000000000010 
-x3 : ffff00000a423b50 x2 : 0000000000000080 
-x1 : ffff00000a423af8 x0 : ffff8001f6533c18 
+x3 : ffff000009ddbb50 x2 : 0000000000000080 
+x1 : ffff000009ddbaf8 x0 : ffff8001f6619c18 
 Call trace:
  tegra_bpmp_transfer+0x2dc/0x368
  tegra_bpmp_clk_transfer+0x94/0xd0
@@ -1714,7 +1700,7 @@
  worker_thread+0x13c/0x3f8
  kthread+0xfc/0x128
  ret_from_fork+0x10/0x18
----[ end trace 9ada84ef45d3bac4 ]---
+---[ end trace a9f6323c41f45377 ]---
 dwc-eth-dwmac 2490000.ethernet: failed to set TX rate: -22
 dwc-eth-dwmac 2490000.ethernet eth0: Link is Up - 1Gbps/Full - flow control rx/tx
 VDD_RTC: disabling
@@ -1725,7 +1711,7 @@
   No soundcards found.
 Freeing unused kernel memory: 1152K
 /init: line 58: chmod: command not found
-L4T-INITRD Build DATE: Sun Oct 30 13:37:18 UTC 2016
+L4T-INITRD Build DATE: Mon Oct 24 04:34:31 UTC 2016
 Root device found: nfs
 Ethernet interfaces: eth0
 IP Address: 192.168.99.2
@@ -1737,105 +1723,105 @@
 Welcome to [1mUbuntu 16.04 LTS[0m!
 
 systemd[1]: Set hostname to <tegra-ubuntu>.
+systemd[1]: Started Forward Password Requests to Wall Directory Watch.
+Started Forward Password Requests to Wall Directory Watch.
 systemd[1]: Listening on /dev/initctl Compatibility Named Pipe.
 Listening on /dev/initctl Compatibility Named Pipe.
-systemd[1]: Created slice User and Session Slice.
-Created slice User and Session Slice.
-systemd[1]: Reached target Encrypted Volumes.
-Reached target Encrypted Volumes.
-systemd[1]: Listening on udev Kernel Socket.
-Listening on udev Kernel Socket.
-systemd[1]: Listening on Syslog Socket.
-Listening on Syslog Socket.
+systemd[1]: Listening on Journal Socket.
+Listening on Journal Socket.
+systemd[1]: Listening on udev Control Socket.
+Listening on udev Control Socket.
 systemd[1]: Listening on Journal Audit Socket.
 Listening on Journal Audit Socket.
-systemd[1]: Started Forward Password Requests to Wall Directory Watch.
-Started Forward Password Requests to Wall Directory Watch.
+systemd[1]: Created slice System Slice.
 Created slice System Slice.
+systemd[1]: Mounting POSIX Message Queue File System...
+         Mounting POSIX Message Queue File System...
+         Mounting Huge Pages File System...
+         Starting Remount Root and Kernel File Systems...
 Created slice system-serial\x2dgetty.slice.
-Listening on udev Control Socket.
+         Starting Load Kernel Modules...
+         Mounting RPC Pipe File System...
 Reached target Swap.
+         Mounting Debug File System...
 Listening on Journal Socket (/dev/log).
+Reached target Encrypted Volumes.
 Created slice system-getty.slice.
-Reached target Slices.
+Created slice User and Session Slice.
 Started Dispatch Password Requests to Console Directory Watch.
 Reached target Paths.
-Listening on Journal Socket.
-         Starting Load Kernel Modules...
-         Starting Remount Root and Kernel File Systems...
-         Starting Journal Service...
-         Mounting RPC Pipe File System...
+Listening on udev Kernel Socket.
+Reached target Slices.
          Starting Create Static Device Nodes in /dev...
-         Mounting Huge Pages File System...
-         Mounting Debug File System...
-         Mounting POSIX Message Queue File System...
+Listening on Syslog Socket.
+         Starting Journal Service...
+Mounted Debug File System.
 Mounted POSIX Message Queue File System.
 Mounted Huge Pages File System.
-Mounted Debug File System.
 Mounted RPC Pipe File System.
-Started Journal Service.
-Started Load Kernel Modules.
 Started Remount Root and Kernel File Systems.
+Started Load Kernel Modules.
 Started Create Static Device Nodes in /dev.
+Started Journal Service.
          Starting udev Kernel Device Manager...
+         Starting Apply Kernel Variables...
+         Mounting Configuration File System...
          Starting udev Coldplug all Devices...
+         Starting Flush Journal to Persistent Storage...
+         Starting Load/Save Random Seed...
 Reached target Local File Systems (Pre).
 Reached target Local File Systems.
          Starting Set console keymap...
          Starting Preprocess NFS configuration...
-         Starting Load/Save Random Seed...
-         Starting Apply Kernel Variables...
-         Mounting Configuration File System...
-         Starting Flush Journal to Persistent Storage...
 Mounted Configuration File System.
+Started Apply Kernel Variables.
+Started Load/Save Random Seed.
 Failed to start Set console keymap.
 See 'systemctl status console-setup.service' for details.
 Started Preprocess NFS configuration.
-Started Load/Save Random Seed.
-Started Apply Kernel Variables.
 Started udev Kernel Device Manager.
-systemd-journald[1521]: Received request to flush runtime journal from PID 1
-Started Flush Journal to Persistent Storage.
-         Starting Create Volatile Files and Directories...
-         Starting Raise network interfaces...
+systemd-journald[1533]: Received request to flush runtime journal from PID 1
 Reached target NFS client services.
 Reached target Remote File Systems (Pre).
 Reached target Remote File Systems.
-Started udev Coldplug all Devices.
+         Starting Raise network interfaces...
+Started Flush Journal to Persistent Storage.
+         Starting Create Volatile Files and Directories...
 Found device /dev/ttyS0.
+Started udev Coldplug all Devices.
 Started Create Volatile Files and Directories.
 Started Raise network interfaces.
 Reached target Network.
 Reached target Network is Online.
-         Starting Update UTMP about System Boot/Shutdown...
 Reached target System Time Synchronized.
+         Starting Update UTMP about System Boot/Shutdown...
 Started Update UTMP about System Boot/Shutdown.
 Reached target System Initialization.
-Listening on RPCbind Server Activation Socket.
 Listening on D-Bus System Message Bus Socket.
+Started Daily apt activities.
+Started Daily Cleanup of Temporary Directories.
+Reached target Timers.
+Listening on RPCbind Server Activation Socket.
 Reached target Sockets.
 Reached target Basic System.
-         Starting LSB: Start NTP daemon...
-Started D-Bus System Message Bus.
          Starting OpenBSD Secure Shell server...
+Started D-Bus System Message Bus.
+         Starting LSB: Set the CPU Frequency Scaling governor to "ondemand"...
          Starting Login Service...
          Starting /etc/rc.local Compatibility...
          Starting Permit User Sessions...
-Started Daily Cleanup of Temporary Directories.
 Started Regular background program processing daemon.
-         Starting LSB: Set the CPU Frequency Scaling governor to "ondemand"...
-Started Daily apt activities.
-Reached target Timers.
          Starting System Logging Service...
+         Starting LSB: Start NTP daemon...
 Started /etc/rc.local Compatibility.
 Started Permit User Sessions.
+Started System Logging Service.
 Started Getty on tty1.
 Started Serial Getty on ttyS0.
 Reached target Login Prompts.
-Started System Logging Service.
+Started OpenBSD Secure Shell server.
 Started LSB: Set the CPU Frequency Scaling governor to "ondemand".
 Started Login Service.
-Started OpenBSD Secure Shell server.
 Started LSB: Start NTP daemon.
 Reached target Multi-User System.
 Reached target Graphical Interface.
Diff for ./tegra20-trimslice/tegra20-trimslice/multi_v7_defconfig_log.txt
--- /tmp/fooa	2018-02-08 19:48:23.296897949 -0800
+++ /tmp/foob	2018-02-08 19:48:23.296897949 -0800
@@ -19,7 +19,7 @@
 Out:   serial
 Err:   serial
 Net:   RTL8169#0
-Hit any key to stop autoboot:  2  1  0 
+Hit any key to stop autoboot:  2  0 
 Tegra20 (TrimSlice) # 
 Tegra20 (TrimSlice) # usb reset
 resetting USB...
@@ -57,7 +57,7 @@
 Filename 'tegra20-trimslice.dtb'.
 Load address: 0x2408000
 Loading: *##
-	 3.4 MiB/s
+	 4.1 MiB/s
 done
 Bytes transferred = 21686 (54b6 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
@@ -69,7 +69,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-next-20180208-349377-g957f77188c8b (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Feb 7 19:17:40 PST 2018
+Linux version 4.15.0-next-20180209-349478-g611f09f4907a (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Feb 8 19:04:12 PST 2018
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -270,7 +270,7 @@
 igb: Copyright (c) 2007-2014 Intel Corporation.
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:01:00.0: enabling device (0140 -> 0143)
-r8169 0000:01:00.0 eth0: RTL8168d/8111d at 0x9a7be569, 00:01:c0:13:7d:63, XID 081000c0 IRQ 299
+r8169 0000:01:00.0 eth0: RTL8168d/8111d at 0xd5010aa3, 00:01:c0:13:7d:63, XID 081000c0 IRQ 299
 r8169 0000:01:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 pegasus: v0.9.3 (2013/04/25), Pegasus/Pegasus II USB Ethernet driver
 usbcore: registered new interface driver pegasus
@@ -345,13 +345,13 @@
 tegra-ehci c5000000.usb: new USB bus registered, assigned bus number 2
 tegra-ehci c5000000.usb: irq 291, io mem 0xc5000000
 usb 1-1: new high-speed USB device number 2 using tegra-ehci
-tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 mmc1: new high speed SD card at address b368
 mmcblk1: mmc1:b368 NCard 1.83 GiB (ro)
+tegra-ehci c5000000.usb: USB 2.0 started, EHCI 1.00
 hub 2-0:1.0: USB hub found
+ mmcblk1: p1 p2
 hub 2-0:1.0: 1 port detected
 input: gpio-keys as /devices/soc0/gpio-keys/input/input0
- mmcblk1: p1 p2
 rtc-em3027 2-0056: hctosys: unable to read the hardware clock
 Waiting for root device PARTUUID=11eb23c1-01...
 hub 1-1:1.0: USB hub found
Diff for ./tegra20-trimslice/tegra20-trimslice/tegra_defconfig_log.txt
--- /tmp/fooa	2018-02-08 19:48:23.316898040 -0800
+++ /tmp/foob	2018-02-08 19:48:23.320898058 -0800
@@ -48,7 +48,7 @@
 	 ##################################
 	 4.1 MiB/s
 done
-Bytes transferred = 6221496 (5eeeb8 hex)
+Bytes transferred = 6221752 (5eefb8 hex)
 Tegra20 (TrimSlice) # tftpboot 0x02408000 tegra20-trimslice.dtb
 Using RTL8169#0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.103
@@ -59,7 +59,7 @@
 done
 Bytes transferred = 21686 (54b6 hex)
 Tegra20 (TrimSlice) # bootz 0x00408000 - 0x02408000
-Kernel image @ 0x408000 [ 0x000000 - 0x5eeeb8 ]
+Kernel image @ 0x408000 [ 0x000000 - 0x5eefb8 ]
 ## Flattened Device Tree blob at 02408000
    Booting using the fdt blob at 0x2408000
    Using Device Tree in place at 02408000, end 024104b5
@@ -67,7 +67,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-next-20180208-29428-g957f771 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Feb 7 19:16:46 PST 2018
+Linux version 4.15.0-next-20180209-29529-g611f09f (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Feb 8 19:02:48 PST 2018
 CPU: ARMv7 Processor [411fc090] revision 0 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: Compulab TrimSlice board
@@ -261,7 +261,7 @@
 igb: Copyright (c) 2007-2014 Intel Corporation.
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:01:00.0: enabling device (0140 -> 0143)
-r8169 0000:01:00.0 eth0: RTL8168d/8111d at 0x55c2a063, 00:01:c0:13:7d:63, XID 081000c0 IRQ 299
+r8169 0000:01:00.0 eth0: RTL8168d/8111d at 0xcfb90bfa, 00:01:c0:13:7d:63, XID 081000c0 IRQ 299
 r8169 0000:01:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 pegasus: v0.9.3 (2013/04/25), Pegasus/Pegasus II USB Ethernet driver
 usbcore: registered new interface driver pegasus
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig%jetson-tx1_log.txt
--- /tmp/fooa	2018-02-08 19:48:23.340898149 -0800
+++ /tmp/foob	2018-02-08 19:48:23.344898166 -0800
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 755549 us
-Basic modules init took 348526 us
+Tegraboot started after 753298 us
+Basic modules init took 349423 us
 Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
 Next stage binary read took 14368 us
-Carveout took 252665 us
-CPU initialization took 92456 us
-Total time taken by TegraBoot 708015 us
+Carveout took 252677 us
+CPU initialization took 92438 us
+Total time taken by TegraBoot 708906 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.594] LPDDR4 Training: Number of tables = 10
+64b[0001.593] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,9 +117,9 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig%jetson-tx1
-17601024 bytes read in 886 ms (18.9 MiB/s)
+17601024 bytes read in 879 ms (19.1 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
-44914 bytes read in 453 ms (96.7 KiB/s)
+44914 bytes read in 454 ms (95.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
 1627787 bytes read in 495 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0000000000 [0x411fd071]
-Linux version 4.15.0-next-20180208-29428-g957f771 (jonathanh@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Feb 7 19:20:41 PST 2018
+Linux version 4.15.0-next-20180209-29529-g611f09f (jonathanh@kaivo) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Thu Feb 8 19:06:59 PST 2018
 Machine model: NVIDIA Tegra210 P2371 (P2530/P2595) reference design
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -223,7 +223,7 @@
 audit: initializing netlink subsys (disabled)
 audit: type=2000 audit(0.288:1): state=initialized audit_enabled=0 res=1
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ 0000000047f43233, 1 data @ 0000000041cb16fd)
+vdso: 2 pages (1 code @ 00000000365f6cff, 1 data @ 00000000b0f4b05c)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -338,6 +338,7 @@
 hctosys: unable to open rtc device (rtc0)
 ALSA device list:
   No soundcards found.
+Waiting for root device PARTUUID=00000000-0001-0000-788f-a75600000000...
 mmc0: new high speed MMC card at address 0001
 mmcblk0: mmc0:0001 SDW16G 14.7 GiB 
 mmcblk0boot0: mmc0:0001 SDW16G partition 1 4.00 MiB
Diff for ./tegra210-p2371-0000/tegra210-p2371-0000/defconfig_log.txt
--- /tmp/fooa	2018-02-08 19:48:23.348898185 -0800
+++ /tmp/foob	2018-02-08 19:48:23.352898203 -0800
@@ -64,17 +64,17 @@
 Loading and Validation of Secure OS Successful
 NvTbootPackSdramParams: start. 
 NvTbootPackSdramParams: done. 
-Tegraboot started after 801141 us
-Basic modules init took 349910 us
-Sec Bootdevice Read Time = 202 ms, Read Size = 8459 KB
-Next stage binary read took 14370 us
-Carveout took 252726 us
-CPU initialization took 92447 us
-Total time taken by TegraBoot 709453 us
+Tegraboot started after 759430 us
+Basic modules init took 348266 us
+Sec Bootdevice Read Time = 203 ms, Read Size = 8459 KB
+Next stage binary read took 14384 us
+Carveout took 252664 us
+CPU initialization took 93049 us
+Total time taken by TegraBoot 708363 us
 
 Starting CPU & Halting co-processor 
 
-64b[0001.640] LPDDR4 Training: Number of tables = 10
+64b[0001.598] LPDDR4 Training: Number of tables = 10
 EMC Training (SRC-freq: 204000; DST-freq: 408000)
 EMC Training Successful
 EMC Training (SRC-freq: 204000; DST-freq: 665600)
@@ -108,7 +108,7 @@
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # ums 0 mmc 0
 UMS: disk start sector: 0x0, count: 0x1d5a000
-|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-CTRL+C - Operation aborted
+|/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\/-\|/-\CTRL+C - Operation aborted
 Tegra210 (P2371-0000) # 
 Tegra210 (P2371-0000) # env set fdt_copy_node_names /cpus:/psci
 Tegra210 (P2371-0000) # setenv rootpart 1
@@ -117,11 +117,11 @@
 Tegra210 (P2371-0000) # part uuid ${devtype} ${devnum}:${rootpart} uuid
 Tegra210 (P2371-0000) # setenv bootargs ignore_loglevel console=ttyS0,115200n8 earlycon=uart8250,mmio32,0x70006000 rw root=PARTUUID=00000000-0001-0000-788f-a75600000000 rootwait debug earlyprintk init=/bin/bash
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${kernel_addr_r} /boot/Image.defconfig
-17469952 bytes read in 864 ms (19.3 MiB/s)
+17469952 bytes read in 863 ms (19.3 MiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${fdt_addr_r} /boot/tegra210-p2371-0000.dtb
 44914 bytes read in 453 ms (96.7 KiB/s)
 Tegra210 (P2371-0000) # load ${devtype} ${devnum}:${rootpart} ${ramdisk_addr_r} /boot/tegra-ramdisk.img
-1627787 bytes read in 494 ms (3.1 MiB/s)
+1627787 bytes read in 496 ms (3.1 MiB/s)
 Tegra210 (P2371-0000) # ramdisk_size=${filesize}
 Tegra210 (P2371-0000) # booti ${kernel_addr_r} - ${fdt_addr_r}
 ## Flattened Device Tree blob at 82000000
@@ -131,7 +131,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0000000000 [0x411fd071]
-Linux version 4.15.0-next-20180208-349377-g957f77188c8b (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Feb 7 19:23:46 PST 2018
+Linux version 4.15.0-next-20180209-349478-g611f09f4907a (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Thu Feb 8 19:10:18 PST 2018
 Machine model: NVIDIA Tegra210 P2371 (P2530/P2595) reference design
 debug: ignoring loglevel setting.
 earlycon: uart8250 at MMIO32 0x0000000070006000 (options '')
@@ -223,7 +223,7 @@
 audit: initializing netlink subsys (disabled)
 audit: type=2000 audit(0.288:1): state=initialized audit_enabled=0 res=1
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ 0000000033405a75, 1 data @ 000000003aa5a234)
+vdso: 2 pages (1 code @ 00000000faddc330, 1 data @ 0000000027397086)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
Diff for ./tegra210-p2371-2180/tegra210-p2371-2180/defconfig%jetson-tx1_log.txt
Binary files /tmp/fooa and /tmp/foob differ
Diff for ./tegra210-smaug/tegra210-smaug/defconfig_log.txt
--- /tmp/fooa	2018-02-08 19:48:23.376898312 -0800
+++ /tmp/foob	2018-02-08 19:48:23.380898330 -0800
@@ -1,5 +1,5 @@
 Booting Linux on physical CPU 0x0000000000 [0x411fd071]
-Linux version 4.15.0-next-20180208-349377-g957f77188c8b (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Wed Feb 7 19:23:46 PST 2018
+Linux version 4.15.0-next-20180209-349478-g611f09f4907a (jonathanh@tamien) (gcc version 4.9.2 20140904 (prerelease) (crosstool-NG linaro-1.13.1-4.9-2014.09 - Linaro GCC 4.9-2014.09)) #1 SMP PREEMPT Thu Feb 8 19:10:18 PST 2018
 Machine model: Google Pixel C
 efi: Getting EFI parameters from FDT:
 efi: UEFI not found.
@@ -100,7 +100,7 @@
 audit: initializing netlink subsys (disabled)
 audit: type=2000 audit(0.184:1): state=initialized audit_enabled=0 res=1
 cpuidle: using governor menu
-vdso: 2 pages (1 code @ 00000000cbebe8c2, 1 data @ 000000001d739148)
+vdso: 2 pages (1 code @ 00000000bd85bdfc, 1 data @ 00000000b818d152)
 hw-breakpoint: found 6 breakpoint and 4 watchpoint registers.
 DMA: preallocated 256 KiB pool for atomic allocations
 Serial: AMBA PL011 UART driver
@@ -256,8 +256,8 @@
 mmcblk0: mmc0:0001 BGND3R 29.1 GiB 
 mmcblk0boot0: mmc0:0001 BGND3R partition 1 4.00 MiB
 mmcblk0boot1: mmc0:0001 BGND3R partition 2 4.00 MiB
-max77686-rtc max77620-rtc: setting system clock to 2001-11-19 08:52:36 UTC (1006159956)
 mmcblk0rpmb: mmc0:0001 BGND3R partition 3 4.00 MiB, chardev (240:0)
+max77686-rtc max77620-rtc: setting system clock to 2001-11-20 08:39:52 UTC (1006245592)
  mmcblk0: p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11
 PP1200_AVDD: disabling
 tegra-i2c 7000d100.i2c: could not find pctldev for node /host1x@50000000/dpaux@545c0000/pinmux-i2c, deferring probe
Diff for ./tegra30-beaver/tegra30-beaver/multi_v7_defconfig_log.txt
--- /tmp/fooa	2018-02-08 19:48:23.408898457 -0800
+++ /tmp/foob	2018-02-08 19:48:23.412898474 -0800
@@ -40,7 +40,7 @@
 	 #################################################################
 	 #################################################################
 	 #####################################
-	 2.9 MiB/s
+	 3.1 MiB/s
 done
 Bytes transferred = 8167936 (7ca200 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
@@ -61,7 +61,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-next-20180208-349377-g957f77188c8b (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Wed Feb 7 19:17:40 PST 2018
+Linux version 4.15.0-next-20180209-349478-g611f09f4907a (jonathanh@tamien) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP Thu Feb 8 19:04:12 PST 2018
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -376,7 +376,7 @@
 pcieport 0000:00:01.0: Signaling PME with IRQ 375
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:01:00.0: enabling device (0140 -> 0143)
-r8169 0000:01:00.0 eth0: RTL8168evl/8111evl at 0x5969d770, 02:04:4b:01:60:a6, XID 0c900800 IRQ 376
+r8169 0000:01:00.0 eth0: RTL8168evl/8111evl at 0xc37b5c98, 02:04:4b:01:60:a6, XID 0c900800 IRQ 376
 r8169 0000:01:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
@@ -407,5 +407,5 @@
 Freeing unused kernel memory: 2048K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.104065] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.114076] usb 2-1: new high-speed USB device number 2 using tegra-ehci
 asix 2-1:1.0 eth1: register 'asix' at usb-7d008000.usb-1, ASIX AX88772 USB 2.0 Ethernet, 00:50:b6:0f:67:60
Diff for ./tegra30-beaver/tegra30-beaver/tegra_defconfig_log.txt
--- /tmp/fooa	2018-02-08 19:48:23.436898582 -0800
+++ /tmp/foob	2018-02-08 19:48:23.436898582 -0800
@@ -40,18 +40,18 @@
 	 ##################################
 	 2.9 MiB/s
 done
-Bytes transferred = 6221496 (5eeeb8 hex)
+Bytes transferred = 6221752 (5eefb8 hex)
 Tegra30 (Beaver) # tftpboot 0x82000000 tegra30-beaver.dtb
 Using asx0 device
 TFTP from server 192.168.100.1; our IP address is 192.168.100.101
 Filename 'tegra30-beaver.dtb'.
 Load address: 0x82000000
 Loading: *####
-	 2.5 MiB/s
+	 2.7 MiB/s
 done
 Bytes transferred = 51470 (c90e hex)
 Tegra30 (Beaver) # bootz 0x81000000 - 0x82000000
-Kernel image @ 0x81000000 [ 0x000000 - 0x5eeeb8 ]
+Kernel image @ 0x81000000 [ 0x000000 - 0x5eefb8 ]
 ## Flattened Device Tree blob at 82000000
    Booting using the fdt blob at 0x82000000
    Loading Device Tree to 8fff0000, end 8ffff90d ... OK
@@ -59,7 +59,7 @@
 Starting kernel ...
 
 Booting Linux on physical CPU 0x0
-Linux version 4.15.0-next-20180208-29428-g957f771 (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Wed Feb 7 19:16:46 PST 2018
+Linux version 4.15.0-next-20180209-29529-g611f09f (jonathanh@kaivo) (gcc version 4.7.3 20130328 (prerelease) (crosstool-NG linaro-1.13.1-4.7-2013.04-20130415 - Linaro GCC 2013.04)) #1 SMP PREEMPT Thu Feb 8 19:02:48 PST 2018
 CPU: ARMv7 Processor [412fc099] revision 9 (ARMv7), cr=10c5387d
 CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
 OF: fdt: Machine model: NVIDIA Tegra30 Beaver evaluation board
@@ -366,7 +366,7 @@
 pcieport 0000:00:01.0: Signaling PME with IRQ 375
 r8169 Gigabit Ethernet driver 2.3LK-NAPI loaded
 r8169 0000:01:00.0: enabling device (0140 -> 0143)
-r8169 0000:01:00.0 eth0: RTL8168evl/8111evl at 0x3a5b3e65, 02:04:4b:01:60:a6, XID 0c900800 IRQ 376
+r8169 0000:01:00.0 eth0: RTL8168evl/8111evl at 0x4c7d4b4e, 02:04:4b:01:60:a6, XID 0c900800 IRQ 376
 r8169 0000:01:00.0 eth0: jumbo features [frames: 9200 bytes, tx checksumming: ko]
 [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
 [drm] No driver support for vblank timestamp query.
@@ -403,4 +403,4 @@
 Freeing unused kernel memory: 1024K
 bash: cannot set terminal process group (-1): Inappropriate ioctl for device
 bash: no job control in this shell
-root@(none):/# [    4.368603] usb 2-1: new high-speed USB device number 2 using tegra-ehci
+root@(none):/# [    4.388586] usb 2-1: new high-speed USB device number 2 using tegra-ehci
